<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/11/3443" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/11/3443.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/11/3443.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/11/3443.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/6/11/3443/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/11/3443.atom" />
<meta name="HW.pisa" content="ggg;6/11/3443" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genomic Prediction of Single Crosses in the Early Stages of a Maize Hybrid Breeding Pipeline" />
<meta name="DC.Identifier" content="10.1534/g3.116.031286" />
<meta name="DC.Date" content="2016-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Kadam et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="10.1534/g3.117.300143" />
<meta name="DC.Description" content="Prediction of single-cross performance has been a major goal of plant breeders since the beginning of hybrid breeding. Recently, genomic prediction has shown to be a promising approach, but only limited studies have examined the accuracy of predicting single-cross performance. Moreover, no studies have examined the potential of predicting single crosses among random inbreds derived from a series of biparental families, which resembles the structure of germplasm comprising the initial stages of a hybrid maize breeding pipeline. The main objectives of this study were to evaluate the potential of genomic prediction for identifying superior single crosses early in the hybrid breeding pipeline and optimize its application. To accomplish these objectives, we designed and analyzed a novel population of single crosses representing the Iowa Stiff Stalk synthetic/non-Stiff Stalk heterotic pattern commonly used in the development of North American commercial maize hybrids. The performance of single crosses was predicted using parental combining ability and covariance among single crosses. Prediction accuracies were estimated using cross-validation and ranged from 0.28 to 0.77 for grain yield, 0.53 to 0.91 for plant height, and 0.49 to 0.94 for staygreen, depending on the number of tested parents of the single cross and genomic prediction method used. The genomic estimated general and specific combining abilities showed an advantage over genomic covariances among single crosses when one or both parents of the single cross were untested. Overall, our results suggest that genomic prediction of single crosses in the early stages of a hybrid breeding pipeline holds great potential to redesign hybrid breeding and increase its efficiency." />
<meta name="DC.Contributor" content="Dnyaneshwar C. Kadam" />
<meta name="DC.Contributor" content="Sarah M. Potts" />
<meta name="DC.Contributor" content="Martin O. Bohn" />
<meta name="DC.Contributor" content="Alexander E. Lipka" />
<meta name="DC.Contributor" content="Aaron J. Lorenz" />
<meta name="article:published_time" content="2016-11-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="Genomic Prediction of Single Crosses in the Early Stages of a Maize Hybrid Breeding Pipeline" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Prediction of single-cross performance has been a major goal of plant breeders since the beginning of hybrid breeding. Recently, genomic prediction has shown to be a promising approach, but only limited studies have examined the accuracy of predicting single-cross performance. Moreover, no studies have examined the potential of predicting single crosses among random inbreds derived from a series of biparental families, which resembles the structure of germplasm comprising the initial stages of a hybrid maize breeding pipeline. The main objectives of this study were to evaluate the potential of genomic prediction for identifying superior single crosses early in the hybrid breeding pipeline and optimize its application. To accomplish these objectives, we designed and analyzed a novel population of single crosses representing the Iowa Stiff Stalk synthetic/non-Stiff Stalk heterotic pattern commonly used in the development of North American commercial maize hybrids. The performance of single crosses was predicted using parental combining ability and covariance among single crosses. Prediction accuracies were estimated using cross-validation and ranged from 0.28 to 0.77 for grain yield, 0.53 to 0.91 for plant height, and 0.49 to 0.94 for staygreen, depending on the number of tested parents of the single cross and genomic prediction method used. The genomic estimated general and specific combining abilities showed an advantage over genomic covariances among single crosses when one or both parents of the single cross were untested. Overall, our results suggest that genomic prediction of single crosses in the early stages of a hybrid breeding pipeline holds great potential to redesign hybrid breeding and increase its efficiency.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/11/01" />
<meta name="citation_mjid" content="ggg;6/11/3443" />
<meta name="citation_id" content="6/11/3443" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/11/3443" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/11/3443.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/11/3443.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/11/3443.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.031286" />
<meta name="citation_pmid" content="27646704" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="3443" />
<meta name="citation_lastpage" content="3453" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Dnyaneshwar C. Kadam" />
<meta name="citation_author_institution" content="Department of Agronomy and Horticulture, University of Nebraska, Lincoln, Nebraska 68583" />
<meta name="citation_author" content="Sarah M. Potts" />
<meta name="citation_author_institution" content="Department of Crop Sciences, University of Illinois at Urbana-Champaign, Urbana, Illinois 61801" />
<meta name="citation_author" content="Martin O. Bohn" />
<meta name="citation_author_institution" content="Department of Crop Sciences, University of Illinois at Urbana-Champaign, Urbana, Illinois 61801" />
<meta name="citation_author" content="Alexander E. Lipka" />
<meta name="citation_author_institution" content="Department of Crop Sciences, University of Illinois at Urbana-Champaign, Urbana, Illinois 61801" />
<meta name="citation_author" content="Aaron J. Lorenz" />
<meta name="citation_author_institution" content="Department of Agronomy and Horticulture, University of Nebraska, Lincoln, Nebraska 68583" />
<meta name="citation_author_email" content="lore0149@umn.edu" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=T. Albrecht;citation_author=V. Wimmer;citation_author=H. Auinger;citation_author=M. Erbe;citation_author=C. Knaak;citation_title=Genome-based prediction of testcross values in maize.;citation_pages=339-350;citation_volume=123;citation_year=2011;citation_issue=2;citation_pmid=21505832;citation_doi=10.1007/s00122-011-1587-7" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=T. Albrecht;citation_author=H. J. Auinger;citation_author=V. Wimmer;citation_author=J. O. Ogutu;citation_author=C. Knaak;citation_title=Genome-based prediction of maize hybrid performance across genetic groups, testers, locations, and years.;citation_pages=1375-1386;citation_volume=127;citation_year=2014;citation_pmid=24723140;citation_doi=10.1007/s00122-014-2305-z" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=R. Bernardo;citation_title=Relationship between single-cross performance and molecular marker heterozygosity.;citation_pages=628-634;citation_volume=83;citation_year=1992;citation_pmid=24202681" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=R. Bernardo;citation_title=Prediction of Maize Single-Cross Performance Using RFLPs and Information from Related Hybrids;citation_pages=20-25;citation_volume=34;citation_year=1994;citation_issue=1;citation_doi=10.2135/cropsci1994.0011183X003400010003x" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=R. Bernardo;citation_title=Best Linear Unbiased Prediction of Maize Single-Cross Performance;citation_pages=50-56;citation_volume=36;citation_year=1996;citation_issue=1;citation_doi=10.2135/cropsci1996.0011183X003600010009x" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=R. Bernardo;citation_title=Best Linear Unbiased Prediction of the Performance of Crosses between Untested Maize Inbreds;citation_pages=872-876;citation_volume=36;citation_year=1996;citation_issue=4;citation_doi=10.2135/cropsci1996.0011183X003600040009x" />
<meta name="citation_reference" content="citation_title=Breeding for Quantitative Traits in Plants;citation_author=R. Bernardo;citation_year=2002" />
<meta name="citation_reference" content="citation_title=ASReml-R Reference Manual. ;citation_author=D. Butler;citation_author=B. R. Cullis;citation_author=A. Gilmour;citation_author=B. Gogel;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=R package version 1;citation_title=boot: Bootstrap R (S-Plus) Functions;citation_author=A. Canty;citation_author=B. Ripley;citation_pages=3-18;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=A. Charcosset;citation_author=M. Lefort-Buson;citation_author=A. Gallais;citation_title=Relationship between heterosis and heterozygosity at marker loci: a theoretical computation.;citation_pages=571-575;citation_volume=81;citation_year=1991;citation_pmid=24221369" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_journal_abbrev=Biometrics;citation_author=R. E. Comstock;citation_author=H. F. Robinson;citation_title=The components of genetic variance in populations of biparental progenies and their use in estimating the average degree of dominance.;citation_pages=254-266;citation_volume=4;citation_year=1948;citation_issue=4;citation_pmid=18108899;citation_doi=10.2307/3001412" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Sci.;citation_author=J. Dekkers;citation_title=Marker-assisted selection for commercial crossbred performance.;citation_volume=85;citation_year=2007;citation_pmid=17504955;citation_doi=10.2527/jas.2006-683" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=J. M. Hickey;citation_author=R. Pong-Wong;citation_author=H. D. Daetwyler;citation_author=M. P. L. Calus;citation_title=Whole-genome regression and prediction methods applied to plant and animal breeding.;citation_volume=193;citation_year=2013;citation_pmid=22745228;citation_doi=10.1534/genetics.112.143313" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_author=D. S. Falconer;citation_author=T. F. C. Mackay;citation_year=1996" />
<meta name="citation_reference" content="citation_title=Principles of Cultivar Development: Theory and Technique;citation_author=W. Fehr;citation_year=1987" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. C. Glaubitz;citation_author=T. M. Casstevens;citation_author=F. Lu;citation_author=J. Harriman;citation_author=R. J. Elshire;citation_title=TASSEL-GBS: a high capacity genotyping by sequencing analysis pipeline.;citation_pages=e90346;citation_volume=9;citation_year=2014;citation_pmid=24587335;citation_doi=10.1371/journal.pone.0090346" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=M. Gowda;citation_author=Y. Zhao;citation_author=H. P. Maurer;citation_author=E. A. Weissmann;citation_author=T. Würschum;citation_title=Best linear unbiased prediction of triticale hybrid performance.;citation_pages=223-230;citation_volume=191;citation_year=2013;citation_doi=10.1007/s10681-012-0784-z" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=A. R. Hallauer;citation_title=Relation Between Inbred and Hybrid Traits in Maize;citation_pages=703-706;citation_volume=17;citation_year=1977;citation_issue=5;citation_doi=10.2135/cropsci1977.0011183X001700050007x" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=J. He;citation_author=X. Zhao;citation_author=A. Laroche;citation_author=Z. Lu;citation_author=H. Liu;citation_title=Genotyping-by-sequencing (GBS), an ultimate marker-assisted selection (MAS) tool to accelerate plant breeding.;citation_pages=484;citation_volume=5;citation_year=2014;citation_pmid=25324846" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=E. L. Heffner;citation_author=M. E. Sorrells;citation_author=J. Jannink;citation_title=Genomic Selection for Crop Improvement;citation_pages=1-12;citation_volume=49;citation_year=2009;citation_issue=1;citation_doi=10.2135/cropsci2008.08.0512" />
<meta name="citation_reference" content="citation_journal_title=Plant Breed. Rev.;citation_author=J. B. Holland;citation_author=W. E. Nyquist;citation_author=C. T. Cervantes-Martínez;citation_title=Estimating and interpreting heritability for plant breeding: an update.;citation_pages=9-112;citation_volume=22;citation_year=2003" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=A. Jacobson;citation_author=L. Lian;citation_author=S. Zhong;citation_author=R. Bernardo;citation_title=General combining ability model for genomewide selection in a biparental cross.;citation_pages=895-905;citation_volume=54;citation_year=2014;citation_doi=10.2135/cropsci2013.11.0774" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Soc. Agron.;citation_author=M. T. Jenkins;citation_author=A. M. Brunson;citation_title=Methods of testing inbred lines of maize in crossbred combinations.;citation_pages=523-530;citation_volume=24;citation_year=1932;citation_doi=10.2134/agronj1932.00021962002400070004x" />
<meta name="citation_reference" content="citation_author=R. Kohavi;citation_pages=1137-1143;citation_year=1995" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=E. Lee;citation_author=M. Ash;citation_author=B. Good;citation_title=Re-examining the Relationship between Degree of Relatedness, Genetic Effects, and Heterosis in Maize;citation_pages=629-635;citation_volume=47;citation_year=2007;citation_issue=2;citation_doi=10.2135/cropsci2006.04.0275" />
<meta name="citation_reference" content="citation_journal_title=Crop Pasture Sci.;citation_author=Z. Lin;citation_author=B. Hayes;citation_author=H. Daetwyler;citation_title=Genomic selection in crops, trees and forages: a review.;citation_pages=1177-1191;citation_volume=65;citation_year=2014;citation_doi=10.1071/CP13363" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=C. F. H. Longin;citation_author=M. Gowda;citation_author=J. Mühleisen;citation_author=E. Ebmeyer;citation_author=E. Kazman;citation_title=Hybrid wheat: quantitative genetic parameters and consequences for the design of breeding programs.;citation_pages=2791-2801;citation_volume=126;citation_year=2013;citation_pmid=23913277;citation_doi=10.1007/s00122-013-2172-z" />
<meta name="citation_reference" content="citation_journal_title=Laboratory Papers.;citation_author=H. Love;citation_author=B. Wentz;citation_title=Correlations between ear characters and yield in corn.;citation_pages=3;citation_volume=1;citation_year=1914" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=B. D. Mansfield;citation_author=R. H. Mumm;citation_title=Survey of plant density tolerance in US maize germplasm.;citation_pages=157-173;citation_volume=54;citation_year=2014;citation_doi=10.2135/cropsci2013.04.0252" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=J. M. Massman;citation_author=A. Gordillo;citation_author=R. E. Lorenzana;citation_author=R. Bernardo;citation_title=Genomewide predictions from maize single-cross data.;citation_pages=13-22;citation_volume=126;citation_year=2013;citation_pmid=22886355;citation_doi=10.1007/s00122-012-1955-y" />
<meta name="citation_reference" content="citation_title=The Genetics and Exploitation of Heterosis in Crops;citation_title=Genetic diversity and heterosis;citation_author=A. Melchinger;citation_pages=99-118;citation_year=1999" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733" />
<meta name="citation_reference" content="citation_journal_title=Biometrika;citation_journal_abbrev=Biometrika;citation_author=H. Patterson;citation_author=E. Williams;citation_title=A new class of resolvable incomplete block designs;citation_pages=83-92;citation_volume=63;citation_year=1976;citation_issue=1;citation_doi=10.1093/biomet/63.1.83" />
<meta name="citation_reference" content="citation_journal_title=Plant Breed.;citation_author=J. C. Reif;citation_author=Y. Zhao;citation_author=T. Würschum;citation_author=M. Gowda;citation_author=V. Hahn;citation_title=Genomic prediction of sunflower hybrid performance.;citation_pages=107-114;citation_volume=132;citation_year=2013;citation_doi=10.1111/pbr.12007" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=C. Riedelsheimer;citation_author=A. Czedik-Eysenberg;citation_author=C. Grieder;citation_author=J. Lisec;citation_author=F. Technow;citation_title=Genomic and metabolic prediction of complex heterotic traits in hybrid maize.;citation_pages=217-220;citation_volume=44;citation_year=2012;citation_issue=2;citation_pmid=22246502;citation_doi=10.1038/ng.1033" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=T. Schrag;citation_author=A. Melchinger;citation_author=A. Sørensen;citation_author=M. Frisch;citation_title=Prediction of single-cross hybrid performance for grain yield and grain dry matter content in maize using AFLP markers associated with QTL.;citation_pages=1037-1047;citation_volume=113;citation_year=2006;citation_issue=6;citation_pmid=16896712;citation_doi=10.1007/s00122-006-0363-6" />
<meta name="citation_reference" content="citation_journal_title=Maydica;citation_author=T. Schrag;citation_author=M. Frish;citation_author=B. Dhillon;citation_author=A. Melchinger;citation_title=Marker-based prediction of hybrid performance in maize single-crosses involving doubled haploids.;citation_pages=353;citation_volume=54;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=T. A. Schrag;citation_author=H. P. Maurer;citation_author=A. E. Melchinger;citation_author=H. Piepho;citation_author=J. Peleman;citation_title=Prediction of single-cross hybrid performance in maize using haplotype blocks associated with QTL for grain yield.;citation_pages=1345-1355;citation_volume=114;citation_year=2007;citation_issue=8;citation_pmid=17323040;citation_doi=10.1007/s00122-007-0521-5" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=T. A. Schrag;citation_author=J. Möhring;citation_author=A. E. Melchinger;citation_author=B. Kusterer;citation_author=B. S. Dhillon;citation_title=Prediction of hybrid performance in maize using molecular markers and joint analyses of hybrids and parental inbreds.;citation_pages=451-461;citation_volume=120;citation_year=2010;citation_issue=2;citation_pmid=19916002;citation_doi=10.1007/s00122-009-1208-x" />
<meta name="citation_reference" content="citation_journal_title=J. Hered.;citation_author=G. H. Shull;citation_title=A pure-line method in corn breeding.;citation_pages=51-58;citation_volume=5;citation_year=1909" />
<meta name="citation_reference" content="citation_title=Corn: Origin, History, Technology, and Production.;citation_author=C. W. Smith;citation_author=J. Betrán;citation_year=2004" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=O. Smith;citation_title=Covariance between Line per se and Testcross Performance;citation_pages=540-543;citation_volume=26;citation_year=1986;citation_issue=3;citation_doi=10.2135/cropsci1986.0011183X002600030023x" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=F. Technow;citation_author=C. Riedelsheimer;citation_author=T. A. Schrag;citation_author=A. E. Melchinger;citation_title=Genomic prediction of hybrid performance in maize with models incorporating dominance and population specific marker effects.;citation_pages=1181-1194;citation_volume=125;citation_year=2012;citation_issue=6;citation_pmid=22733443;citation_doi=10.1007/s00122-012-1905-8" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=F. Technow;citation_author=T. A. Schrag;citation_author=W. Schipprack;citation_author=E. Bauer;citation_author=H. Simianer;citation_title=Genome properties and prospects of genomic prediction of hybrid performance in a breeding program of maize.;citation_volume=197;citation_year=2014;citation_pmid=24850820;citation_doi=10.1534/genetics.114.165860" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=M. Vuylsteke;citation_author=M. Kuiper;citation_author=P. Stam;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=85;citation_year=2000;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=V. S. Windhausen;citation_author=G. N. Atlin;citation_author=J. M. Hickey;citation_author=J. Crossa;citation_author=J. L. Jannink;citation_title=Effectiveness of genomic prediction of maize hybrid performance in different breeding populations and environments.;citation_pages=208-1436;citation_volume=2;citation_year=2012;citation_pmid=11012724;citation_doi=10.1046/j.1365-2540.2000.00747.x" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=Y. Zhao;citation_author=J. Zeng;citation_author=R. Fernando;citation_author=J. C. Reif;citation_title=Genomic prediction of hybrid wheat performance.;citation_pages=1427-810;citation_volume=53;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Y. Zhao;citation_author=Z. Li;citation_author=G. Liu;citation_author=Y. Jiang;citation_author=H. P. Maurer;citation_title=Genome-based establishment of a high-yielding heterotic pattern for hybrid wheat breeding.;citation_pages=802-15629;citation_volume=112;citation_year=2015;citation_doi=10.2135/cropsci2012.08.0463" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genomic Prediction of Single Crosses in the Early Stages of a Maize Hybrid Breeding Pipeline" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11.cover-source.jpg" />
<meta name="twitter:description" content="Prediction of single-cross performance has been a major goal of plant breeders since the beginning of hybrid breeding. Recently, genomic prediction has shown to be a promising approach, but only limited studies have examined the accuracy of predicting single-cross performance. Moreover, no studies have examined the potential of predicting single crosses among random inbreds derived from a series of biparental families, which resembles the structure of germplasm comprising the initial stages of a hybrid maize breeding pipeline. The main objectives of this study were to evaluate the potential of genomic prediction for identifying superior single crosses early in the hybrid breeding pipeline and optimize its application. To accomplish these objectives, we designed and analyzed a novel population of single crosses representing the Iowa Stiff Stalk synthetic/non-Stiff Stalk heterotic pattern commonly used in the development of North American commercial maize hybrids. The performance of single crosses was predicted using parental combining ability and covariance among single crosses. Prediction accuracies were estimated using cross-validation and ranged from 0.28 to 0.77 for grain yield, 0.53 to 0.91 for plant height, and 0.49 to 0.94 for staygreen, depending on the number of tested parents of the single cross and genomic prediction method used. The genomic estimated general and specific combining abilities showed an advantage over genomic covariances among single crosses when one or both parents of the single cross were untested. Overall, our results suggest that genomic prediction of single crosses in the early stages of a hybrid breeding pipeline holds great potential to redesign hybrid breeding and increase its efficiency." />
<meta name="og-title" property="og:title" content="Genomic Prediction of Single Crosses in the Early Stages of a Maize Hybrid Breeding Pipeline" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/11/3443" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Prediction of single-cross performance has been a major goal of plant breeders since the beginning of hybrid breeding. Recently, genomic prediction has shown to be a promising approach, but only limited studies have examined the accuracy of predicting single-cross performance. Moreover, no studies have examined the potential of predicting single crosses among random inbreds derived from a series of biparental families, which resembles the structure of germplasm comprising the initial stages of a hybrid maize breeding pipeline. The main objectives of this study were to evaluate the potential of genomic prediction for identifying superior single crosses early in the hybrid breeding pipeline and optimize its application. To accomplish these objectives, we designed and analyzed a novel population of single crosses representing the Iowa Stiff Stalk synthetic/non-Stiff Stalk heterotic pattern commonly used in the development of North American commercial maize hybrids. The performance of single crosses was predicted using parental combining ability and covariance among single crosses. Prediction accuracies were estimated using cross-validation and ranged from 0.28 to 0.77 for grain yield, 0.53 to 0.91 for plant height, and 0.49 to 0.94 for staygreen, depending on the number of tested parents of the single cross and genomic prediction method used. The genomic estimated general and specific combining abilities showed an advantage over genomic covariances among single crosses when one or both parents of the single cross were untested. Overall, our results suggest that genomic prediction of single crosses in the early stages of a hybrid breeding pipeline holds great potential to redesign hybrid breeding and increase its efficiency." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11.cover-source.jpg" />
    <title>Genomic Prediction of Single Crosses in the Early Stages of a Maize Hybrid Breeding Pipeline | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__MRoKQICmX-Ek5ZIyBlwXzUtbKmCmIVqNnsCLjO1o65o__abNV7fJaqFhsjvAynloQDGCCstgzs17fQnVdDEt9-NA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"2zx1MpRnZlw2wY8w_kPjoHVwnWITAvxu6QuO1WJYALI","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460465","apath":"\/ggg\/6\/11\/3443.atom","pisa":"ggg;6\/11\/3443","ac":{"\/ggg\/6\/11\/3443.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/11\/3443.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/11\/3443"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/11\/3443"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/11\/3443.supplemental":true,"\/content\/6\/11\/3443":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022kvd3QETH9k1wO6Jtnw9g5hJJN+vDA20wm3UqTMv5ExoKLA9l5Qs\\\\\\\/0nghN6NcfnJTGIMtBvVwHmPGXOY68jB0EO2JTn+mhXEjfz5kgcbZJRHBAvSKASssGnptMOIByLfKPl94ebFqL7dqMqwr1CSZAf3Hab+2dtUlsp3Y\\\\\\\/YdJ0PmSFuwGnvpdCFeEBjdbwDRkWsHSdbtYKfKOONIwVNgI6L8wa4EDHWkFQSDUSAeUkvljGEaRrlwof8SOrOABQNckHLi8u\\\\\\\/vhQf6sz7oPHgBnwyw5vbRE6jN6LiUEn4+YnKHa5Tu8KC09gVvabrH0OsWWHCnP\\\\\\\/Huo8doXWfd5d3XHstCvGtFYyXCM9ghkqbNXyi0aXs5hiGH0MaIVJGkAwnMY5SwBePGXbwfoZJXxMf9BsnNnQVvw5jn4XN6twg1T1XNYDPcIZa1YI1DKBNZ2dXnf+bXhDz\\\\\\\/1wvMxHeqIHhJnGWzTzxbVoJ4YrfTHHHlpVhSt7sSMU7\\\\\\\/mY3Zv6q9zyltNDWRBJ3k2aq0uwo9SiCDsSTBn19bpuBVRgKii1Pdzqd1\\\\\\\/Ip9qfTaie9+xIh0XElouVMW\\\\\\\/k6bzxu47JAo3NWlE9ZgCnUB+bGcTGbcFE6q56r4elibzdt2djBiC9WGKBFyYEXdbvuzJUft32AxwHhLeDiEXyt+xrdv6ieEUxodaKtNDoWRqq4n3vmi2IiQGEUjYFAJxT+46zNJ6oxBkJ1NFNUQyj8LXx5ebeoPU0L234iCElF3kb7cy9thX+QMxdHjlFIcWegMKcqxjEd3VxhE7ddWyrGu\\\\\\\/lk8wIuC9fKouroAvNR6cbiKyQQfIV6NN5wfbNl+uP0hbHQrIGAaf9CID2ODbs2ekjBjKCADd\\\\\\\/MOI+B+1VjPDWgSOwlYzZ3wsGcLEY4P5LyM8LnK0zwiAb7M57sgh+XPQyGE8elH7YO1SP3aUuBWbSzn4I1MxQXNiJxSE4TkFZ803JuCb\\\\\\\/uHuC0jSZC7oDerENEqoYz8dl0gwooYLIMEjqCFAJnwlDv8pXI\\\\\\\/DOs4fXpLopIgcP2xMZVKs7N\\\\\\\/tQ2dcsmQIUnecSoCb5XrT1Ytq8\\\\\\\/TDbnt+b+76zgjfpUrtq1yEuvJCTw\\\\\\\/wk6GuCm3zh5cwtP86myK8\\\\\\\/k5R6Sv9wlIv\\\\\\\/I7RIrDf2OMKNWCPw4sDwexWBLmzk62iW3XvwTm1MTLOLUevQVtM7HoDTvknJhXszXjNC7w+HKK1UtfuBoJwFWnYqKLeU71jhExQ3OyQLtU1DRqcF8sGfxtCpAna+kC3LS7kJDOqbtHdbKsW\\\u0022,\\\u0022iv\\\u0022:\\\u0022vlS3oEHiu8mUf5bgAAj2gw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022189b6116105dac54044209e8af7a91c3994ee9ebce96eb1908cd48aceb9cd6b0\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022n6tWzfAe9HTWtJ1bizUe118gjVT0o+0Kenm5Lr9jTHS\\\\\\\/IQdZfB3VB\\\\\\\/1\\\\\\\/2he9GFs3nqEw2rvfbaNOaLJZWBqrHFtXHWI4al1QBrCtGxU6cPTdaIFiSP4Z8DM0ePND+OytD+PNLIlF+PGgq0cForPgJupoVntRU\\\\\\\/kkTI0MGL8OkIVB5IlB0LNE4TRgvFgfqFX6W+Ty7LJPspu6iSbqYG8udXOhmAQkOHly3244xmtUiYAt+luMeFTJ96DofCbJU\\\\\\\/fbltwNugRIn\\\\\\\/ZIkD79ioNcXcFRoYcH0sdVG1h3YQ7T4LZl3PZDb8yQcVnWp18KEzt01tdOKjEAUPxhrYlVQqkIOH41MxoOvVglFhLDpzM9xyOivosF5yF0\\\\\\\/LpcfMG9iq2uFZBILvU7cMlOxnHMqOvyicp6vZ53AKQ0DYqAtrx8AsVD2rnd73xjILfrcl7awkCyklB5dyDKxYSdAOYS643b5ra47bN0MfttxmP\\\\\\\/8nhEDg6R+hTyz7DC6zLed54Izm7mfv98SUoMkEzU6I9RRn0NAPLNttILzVHRO7UaRnwJefzLdkZDVGn1Tv\\\\\\\/6p1B2v1\\\\\\\/E05Z0ExsCkhtYchTiR1eeuF22Gv+giJs+Fq5hRmhFy8xzt9b\\\\\\\/el+UlBkY7KN5PQT54rhKPzFVm6heivdCR79IELVSZ7jV96\\\\\\\/uiAiCqrFyGPBZ806vZwVBfCq1cHAbi+we\\\\\\\/YTKalMbwUPya3D4MTKuAmIK\\\\\\\/r8tSAMgb0eBhVFz998CKLBa6wpFGPrqQtu1ELNzbwp9JpadSi47wUGuZEAC28yV4DlTMOyDrYuYGxf62mBZsAheOTG82tIdYaO1ZOM6z\\\\\\\/+nXbDIZtmN8Ef1YEEqXeBdErm8h5ullqnY7Ea5WcJUY6hKbtM62EDNglf14zs\\\\\\\/BW0KzC845vj8yajYOWV59A\\\\\\\/L\\\\\\\/pVvrhpUG3R03ZgDJN+SKOJcCBc2UujwmbyINrURS1xDwlBR0WBEpQDMOArGRPqrPPlIk09RX5kXSlYhdcoaydAvb4+pygOx3pXg2UjWx6W6Qz74oAPVN+nAfMZUkQ6lIgPqDCGrfFVrkBiUGyIXgaX6ZX8QQpG7uzhscbigf8P\\\\\\\/fl5WWsgY9z\\\\\\\/b4RZYsMK83KYdAiw4L8SEt0KOxSmSy0VmxrtnTnEL7G19eXQcOf+IL76aGCBljW7HBOKNiy3ShAj96ITTXIxlrsGJopT\\\\\\\/UBHnbEKpOQX5YozZYXoS7icMqthmk5MC8zr4pVUrIsvlRNFo5HXnYUMz+UUatsQo06maBAI4kIFscOlR4tVKcpMGzfFj\\\\\\\/ehrtD4Y6ashWZ7kCkVCQnDdJNo\\\\\\\/ZwrkiluUJi\\\\\\\/pP4KBJ0UzodlTq6m2Nzty9wWNHPmAe0GrhVoU2szaK128nF+oSu1b872VCINBUYrai200bYrYEpHBk6OTjkgEkUEH9phcqHl33QtLrafXThAxJeL1zvT5+1e4AwIxguv6g6mjlmmk8h+hsXPePzHF6gm+M2LxTqjNnPmVdO5aJO5gVgAr074TOh\\\\\\\/wd1jC+a6DCYyYw20CkuFnTg3G9daTR38lbTM5tXVbX8pajt2KR90\\\\\\\/pntKE8Bwksch1XutjErmYNQD\\\\\\\/\\\\\\\/phnR5gxPEGgWq+e7wFyh+eibv7jLhJxdJcOjssEZpE0JuHoDTDlShJVjFKlpzsSPFctiBu4yyoBXWl2tvcjpZG+wE68PzJzUxFpSBVtJhTOk+0CnIbYCBdkfk3i2Jv5o+gcvGJgGldeytn\\\\\\\/b4dt+R2SQiIOf4g2HJUHCTyTtePdasY0dmzpds7BOZjIp9J9+FAl+uOZYvqyPDm95nV4whus9JuH7R1Q7rtYa6mKMcMF9DmTllVpP8TmgFcy8lVKjTjz9RyfEEGw3dKX4cV+60api56077Z77v5Ao8Czdy\\\\\\\/VJU1LAYJlNb7IQ9rShPgIWXImE41WTbfLmh\\\\\\\/qEUelv5NxgcfMYnZIIfZ5iSNiao8cVLGVIsMuihzmJ7dh3jj3VbXlAd+Mrx7j7QfM1C8qFx452zfEKCiUMwVWAR+QmwV2dG21IEBHVZtE2YQQUnrlGs6\\\\\\\/5uef5T4qVFdSxcJ26lxw4aZlpawzzyVer7NNTVV7cSabcv+gLdDF+szST+XGthVNJJP0IIr+3vM8ch6y5KgCMvJ+odoCQY\\\\\\\/A2aV3UCLBtdt14uAocHeE3X0aezcK1VQ1JmhGzDZNnqkc2aVkzNsSQWVJpxZ3MRV72XrRVxLgzDOS7SDBFGMTiu4Pbx9YaJT+AUEPWnMa3Kprg7daQZQyhPTu8Tw0z8YOrP5h1\\\\\\\/n9G664yNnfwA7amoFtP1nj8Ct7x4FrrfiQ3ZCOMIYMqenKSWE\\\\\\\/PLVuFWcDya67fuxHdJlGFw68SZwCJoRRjUKuHrSAocAM7YvBeWnBpSlRJFt0E6AoD9dcTJBUnCF4TyZfZf1CU8aEkymmqq9ln0oMDCL3f\\\\\\\/QQGV4vjsB8sX\\\\\\\/B0ndYRGe3jmQ3uuohiHynwGwMZE\\\\\\\/iZS2OlxOXz7waL6iOlQJCsavcBToq0GkQDjZGl6qui2+5OMgcH6jJlWWHVR0YctToEiKR+YGskVB6FoWu7XaVmsHP8FxYUEYjxItVK196kGzJJTXMpsiejeM1zZ3tiBQgIAQ59wIvaBgciB6K6bQxMVtdT\\\\\\\/ALPsP3vZa8XquNbG6tXSnTNCdovWrjApUHX4H200QjsfiJeDQidoMs16yTveAaQ5KOSy0ZlSv2BcA0an4eDmeOrbGXLimKKV04zLnm1NW8MPxwccyAVpTgoQRs8TuoK6X3z2MEnvB9z6FalTICjR\\\\\\\/Z78aeQpZwc8D+ZtvdPw4w8KGMPiczFBaNrhUwtf1lFxg4Lm8fgwZH+2EqpCiCgHDKZtcZ5xVhutmeUjU\\\\\\\/lU9yre6vuVX5r+v5S1aPLe+wJT3vA\\\\\\\/1thgPVMQfy\\\\\\\/JZmg1XQ9Arb0FWmGEE0g4duZLJ8yfyyXnVDpk\\\\\\\/Nd0cbrLAyQ5mf7p9GQO+yAPDi8Pt3RuCAyzmPtLZFuRqwDM\\\\\\\/pCrmfu1BxTkLM14pNO\\\\\\\/QfluDPaaBhxCoL+ARFpp1exshjfgHOS8z2RRzEOi3g7W2iOPnsmD+lwvpkFni7r7JNEMnCngjQvA3wX58\\\\\\\/fEQDeYz2lXpoVzus7Vp6QrxdtdOPh3xPYsV3jlTFCLj6ELhnxbKoFhyajayK7k+HH+ZOep2koiuuQuDwWc7aWwq09ZgJy5x39Je7HQYmtPOjKvLG7VMuKyYcAGu8Tr9hp1t4TvUbaqm6JLyWf2h6wBKudwJEoHBdXPAROIdDlruBNO9RbDkfH5PZruSWHcqK9U2m\\\\\\\/DGLGl\\\\\\\/i0gQgj7iYHPcWuFZ0sgo+ri1KFIf+PyKQx8FetLEArwpRzrc\\\\\\\/LyapeENUmxO4r39rgLrxKkEiZzOxbgk86z7QgD1ImmhvGCznlqYHDT0CU4s8XvgBOhpTQbQoirej+bfUHn1\\\\\\\/vjA4uPm5RoPrvKjXiU3XR8Cvvh94ig9xfLcik1WNkW7ZfV+KKeHGy0Vl6SQR99D6PHUk9MH9MSHTHK1coZBmqLSg20XE\\\\\\\/aMtINVwRa2OEuIt8RAwyxc7kSpaFC5uEDQPURyBE4WMFl6sfhE3PUkr4o3gVY7qEOus432Ufi0Ar4u1IUPw5Q4+4al\\\\\\\/i2c8difoCze5S8PPvJQHf4UeiWNSTViiv\\\\\\\/bsWMCuxDih0+02BpKDHOjxX31OuT7RKTHVv6rqVLimJtMm6xMkhhqv+rVjVfU347irXe6nGnl75iWr7BKSy17NNEkokLG6+KrpQYYxdkjwklejYAnIgj0bbaKvN2H3VtMJq0tgvHuleYuW34NNoQYyo2kpXL23FrusRZhOPh4awlIaQSeXPwOPfMp\\\\\\\/l0d13g2h\\\\\\\/qAfgUS1zer4rj\\\\\\\/mggt+IKix1Uqd5OZ1i35mXepgea93sN+wDMe7\\\\\\\/N1lMl\\\\\\\/Q4lSOCWiMwNZLth43nC4EK\\\\\\\/5NgCNd0aDhqb8oIrEuuly9G31bZMKeSzB7rzg2AK9YhparFw2y++QYLRRyetqO8cC1slwaNoNjmdmKRjoARMkh4vOeXvPZuvsuyA5scv3xFNGkuBMOGo1CjoaM\\\\\\\/qtxDVJXIrwpfrJpOUuN7pNBY1J3t8qZVRVapILtWci\\\\\\\/ZbD8HPAMFfyJHsfERKfqFpRh1VRi3kpPRL2C\\\\\\\/fgEc6kcTC3XsubhglXs9ep6uydME\\\\\\\/lfZlW1s0MQN3DYnkM7RIQmHrsnKYcFu7SBDw3OOc+6fNPZNWoqtnrGMpOPJBDS8\\\\\\\/rTOA1pQoehwgaye2B8Z+jR+QigD8WmMp+K07a50u5ziS0VUy6V7NIiyybPqyt0nKnDGTsK1yL\\\\\\\/jzKN36+wg9mescNfdLWqiLd56T60kLY3L69\\\\\\\/erXN+g1qTwuTgcXHUYPNWPnp5m5t7RmgdRKu\\\\\\\/J3S88urA8l8HLyEgo4dUESuznwQWKe4WK85tnwQvixHDB054nB4LbBQdX0mK9Mta8g\\\\\\\/VNtatElMHtg+SAc0hhnTIKkCPmGWdMHBPmDrdwQBcEwTwsvE1LziBfx1k4TaZlrnCFxvI0EVHFmgRP06Nk58OwEoDTn5SIsEj4TUM3O0yVvnkFr+2VJjixbUS4gXrYPBdGPI0hRF3rruAdiwEMpKlsd+nANlcGJWeown2VxtehvQ3DdoISN0yHS8gRrNb9E39uY+xz\\\\\\\/iQEcBWtHjwO5MFZIJzT0Q+O8YU19jcCfusOQJYAFMIWzrfryARTumNAJ3Xuyl\\\\\\\/70W8V2L3p6dv6d8j2xcyMgJXfrZCs5yVOJDDvQVpFE3Yrl6ykXUbOc3mjZ7iGpsx\\\\\\\/Qhxqygf5mTZUgNwZ1dAPfLZzSFtTle7F1yJ127s2WSd9iGDxYCzU6xaWsrCNjV75LvOR71XthCsax4Yx0DkfPvVHWfdO20LlYZnI3QBZ1BgkgDyT3CxJXH+HEUp5Q6eGLtpAxEdt\\\\\\\/P2hx\\\\\\\/9e0J4l6AzPYc\\\\\\\/rmPGZV83aym+p7+i\\\\\\\/AbsGWfpAXdNR+Pt4nIIqXwR8Ig2YSG1H5nP51DpfQMrdE1MoY8A8ZZyyivfy4cviguRpAqWrEYoeYNllgmeo6jXJWuLGaOzmxdZmkR7bY0sVPd05U4S4wihGmjUtFHfUjDXMc3qWu8TsrM8g7q9o41ozngPpic84+OewlyI7+zL3u7E12wWH3Hj5TCCnekyAPw0wcvFDpzv7ErhfP6\\\\\\\/RhaM43ZDAdz1BMuTre74kBm9xycvg0ElShZSHBF1rVwr+SkJoXVogWRNUlqFtIEEco+9L\\\\\\\/XUi1GRm\\\\\\\/mTHI723HlR4koiLYPHlHt7NJMWCAdqQTEAJZ09lQxVGwhxPv2QwUf4SIojBQB+NmEf1BuP\\\\\\\/u8P+dg5sa1zAnMZ+MSG1bniSxUKKtJn9iDhPF5DpzNo4gIMvdJjd9xVjWcA1SXtcnANNaTrAjT9pgn7FHgZSg\\\\\\\/nXWEJDVjmnQFZPP6O31XWQkGBQZ1vDA3wYqL3hXSSN3Ky\\\\\\\/F\\\\\\\/ckcpkq\\\\\\\/EbhbPFgv3YPX5oVc2dnTZsjRynyponauc7P0yeqYc6SbAMEFTL+Uf1jvE9KU66tB9E\\\\\\\/gOxUZO5X\\\\\\\/S7p1aWrvE66xzDMNxnQbL8Vm4Z6bIRGBPrJHLhMlSEy+wXXn3vfMj2wovx6NL6Coqoi2iEbMov7peClTQjiWuXZ3J40zB8FkMeLdxJzcUApPydKzd2HM14CtA5jdyCTuKWkLnbr+0F1z99WWUWX9yXB+\\\\\\\/U0VNEOGdjJU5bW9\\\\\\\/La5Mf6q\\\\\\\/gNKWJDyjRVL7uf7CDsthc7szAfYjhrS6VCUpIrOChpW5qngJWlo+dl3kkN\\\\\\\/cvlOBYLFZCAVAeuQYlJGJ3DmABJu3uVC6T9wRYRcez4rmNw3YfMBI3eCftTb1+zs5X6KeVBdbdTTmIj84d\\\\\\\/ig87+cTqGP5wyhoszdK8MNNU2d17mDUvpHW9UcLqfwsOopofkuWr1C7wKluvGIJpZ5MO9AaT6x9iZSWVGF0JTVlW3xMbVaj7jP5B0lY3hsITnRFXZN8Nl1z\\\\\\\/bcki5pj7QuxPlFgRu3ZvOAiQCDNvFMX3NYMNcqrCS2IymUo339Y8VdO+ukk2dGmJH1Bn2U4qwYm8jXdsIMMOWiMMEbipO23MM\\\\\\\/WYbgLiSDCz59998MUyNbU7CDJhniqt96DMWTAxoOaJXcAIH14bIJuO1BwTLluQC1Og9Q0MvHlYIs8Xs59keBjbWl\\\\\\\/2AZBF8PwWw+0ZtYVZsT+UQannlrCV0vXMC1mFUBzvjHtWOtBnx\\\\\\\/BuKnleLGKdVe+mMpIvBkXEOfvyxLd\\\\\\\/golvOR0bOpbO2l+gkw\\\\\\\/3REII2GEBexL\\\\\\\/TktyXmNBca77pmrtv2XjqCjPIcOBqXK+H7qN23Beos0qEYLzceaC0JDbO\\\\\\\/EgV1ZUXqbu0mw55iC7Vb\\\\\\\/NW1vtJqK4hXl9zK0Anzw4mRRAxmiafKslcWBhAALdBaShzt5uSptIALohsdbZvotLNrkvl7qNjH2o+IreqUmsrC7yjqZjf59BlWDtXuT13AKLgdpLVPV24zbPgp6Z+ZEfa25ChaUorxE+zXVXDi3oqBEx2UCHktgjt9kSYJ5\\\\\\\/4nX33zgfphlujh58+hCS6l+yXrO0yM7sz8izbbxUHtJTrkc1l\\\\\\\/3cafvVKc\\\\\\\/Q2CExzMb\\\\\\\/bYgvBkpMGKMStl9OafmCmpxNfvzfRDCBiCzRHjqz4DNHlqouDc6EOAhemjp2d1XyjgFzFkK1rGTopfuBREc3S7oz3vz7O8un2r+vFrfXirNlN95zrOWeDOlbuLGfu1ZlsdxNu2yuypFR2QmFJXymgaWIhFJ5MhtAFtwl6Q7JlyaKyoRSmJ2bLrVIj1AKAvd34lVLFzXnqSzKE\\\\\\\/WQT9xRGtgpFFyd8oI1n7TwZZaPE\\\\\\\/frdENuHJlbaoit17zAabqKo++mbish8Ja592Qlfg9fypb5CO\\\\\\\/qGOyAO5dIgmJkSoFcRHV94O0oRB\\\\\\\/jKoDl808RX\\\\\\\/s5WHvZXicMf7MgwsU\\\\\\\/0yGtI4jK\\\\\\\/SXUBJuKskToh2hsQkhzw8r0v+zmEd1i2cm56PV8Cpz+iR\\\\\\\/F+tk\\\\\\\/Y353AtVmGq4aiqYfItTKxXEfrSnw8AFn\\\\\\\/KMtnB3vjO3MgVJbGfeRDtBKQFGkgQHIdaa4AkdqHtwKRnVWopi7k6hrPuRExLnchBb1mrVWzslKrVYwCSgSbd+3pO8l\\\\\\\/4kEdZG6l3w8ZYAKRXHaVsQjJTVxWLRHhMivFE20n\\\\\\\/zAg2pTkjhnfx8dhnimu0yDiqfYxo3PtfSSUcn7SSLiYqy4vUO5\\\\\\\/LzWmJX1BGgOQl8tEHgHUWSf8CXw4GNKrIACPT0IHtJavoRruCT97lbxgWOs0Ep4JVDQKTtLPL6\\\\\\\/D+BJrUUHDWoc2DYKB4nxt+Ion1Ztb2D2IeCApXX8QXiWmrPxnqze17QdUHGvt2840tLT8LIcV1fMlUBhGVuRNhburlRS1CHs2k+AhAurdk21tEJ7NHph3xAHqCWOhIbRO6T0Ta\\\\\\\/jZ55Fuey3DNKIMcrLu7mu3WL5uLq5XOaj9USehrz+Kst4WGnZJ8JJMqBKXjm6MTwYufpd8YslScTJcJ4m9ptgUmoLTsANl8Etnr7J2jijtIk6YnXPTNETUv+y91m7s6k1DCaAF+Hlcl+XjiBx5\\\\\\\/rTLurpYzcg2nZ1Bs+9a5MQo3j7poR9j10QZawCCSSJm2O6lKoCK93TX3j5q7ILGzH2WTUuvJbWT9wG+x7VteIHwzN8\\\\\\\/C3qeovV6jeSCEy0RjeZOOcb8mEOwtvHS6eDPv8PyBVIUQWQblu8ACSLmPdBBCGlCfY3C3T4FAaCLlOAb+\\\\\\\/41VH0vG3noGOlGFpz1HXppT9uoiYqLT3tQAqHmDQfudJFemVPj+v4qyLIZm+oZA5azCESdCafq11vYRzuMXUPBv7cRzypcyoOzhZb7tu7SudrSQro4u7Y6irs75Zn941Q7\\\\\\\/tuzkXrOBo9hge8u\\\\\\\/9QGxrMvy0O6bEuwPYcBfeX9KgkuFuvki9tAjWP0+2HlNSKDa8+MDfTboZTpoqIPNvFCx2MhIsirMwjmnlmLFShVMAyUzI4+dKVcSjo6+cWZvyT72icVjJZCCUsVI0fqCJuM42fapZJ6IDYITmfd7C\\\\\\\/RODYWG0Mg0faLncoOjKYLVV2MOE4WUK5N5dnvSyeueaSBZIh4FSN+lIQ1nLtx3NsoVR75xDmalQq6XxO4mFhUXT3zy9XFTeucIp+xF5wtjz62W1qNI3cEWy6hMuxQPBu+YL\\\\\\\/2rgL5+5Ev53l7p55YQ039XV+YqMlM6P9VTaS+vl5wR87Ahun0\\\\\\\/2\\\\\\\/PEEItRL6gs0A+gPKh4uHa+UHpL0N0Uy6NFYAkzUFSRjeK72mI3UC0UGdNmQONlOJK3KwzAh79akPwV+Nov8hgJoVGspbm6bbkJ2jfl2SOLTPxBMxPYatELz2q26L0VMYGhOwn5nPIH1pXbOIPx2GBP6oZNURgxmhGF2JE7YFJYCtsCJBXj7kV+C7wKSgYmEZJN4EoHPPEM3G4HRyNiruA9XCQjto1N2ziF0lo+7sHuiX21oWtOi2sl1yX52HvPWl\\\\\\\/BKnBrqsw+BSEgQIMyXTo6dgnV4A3N+BB8kS4Jm\\\\\\\/jvctusyEtZL3cPv00xVgGR1R2nrToFiISHVgEfYb\\\\\\\/RI1xsCLPJmApwGGWiPI05NFszaxbRE9fKoJYWZX4a+q1Rs5kK7wX6gWrFm4DDWZoK7isF1PBzgbdPJCFCXaZBOEiP4A2Bow4jUiO1IHJqi12IpBDXUgEykH0vjr1DAMK5KBPq+huWy8LI3GLZt9YfwGilXdezYEMOY+jqq6o2g+si2njpWhPs7N2y0D2EZsNPJTGljY8zMUsO++zpaUcYrFNUEKf9b3RCcvrzuLHG3yZO2niyAqFRwMI4b6bRA2GG6dFn4OZ6f0dOKksDtbKlz9Pw63BF1Vf0wYMl0+L45+ImTe7pMrld7Uy5bSi1sjFfpDKimU9Uy2WV0H4OX5sdC5TM5PV8vP0+c1hbFogHgsKS3wpoE6va2vRWK\\\\\\\/OT8IJ\\\\\\\/wx2fMHYq1rDuDIJlS9vHG0Y2guQBq5kskak4jp\\\\\\\/g1ICzDGrZsVJGUWML+Ag\\\\\\\/FFQLJspA7R\\\\\\\/G3rdDKBh+9xxzbYk\\\\\\\/Q2ZXpvI1OHqy0CBK33\\\\\\\/ooYAS4SEA\\\\\\\/sIgRjRw8JtssvHG\\\\\\\/Kvpw1Vpz5TQZwV1K1Bp3hDMCC3b0J61A0KbGV0\\\\\\\/4VwzEWrmbHXNQLy6pb1K9NaibLovDDwgUR6Eq2d5alsoH4T3QPB2ja8glV2CovRhnSsJJcOZKWJXDMoRZ9BFkNlVXfM90hM\\\\\\\/+bjXvlE7OKAY\\\\\\\/A5YZLnM7h90iu4ObLjcF9ziF4LG3Op6JSx9KM72iTxTWE3O6CyN\\\\\\\/Wp17ungh0YQrHuyNYR+UhauErzJSjEnRRd\\\\\\\/XIWHJI8\\\\\\\/Z0dm9RKZE9nUOl\\\\\\\/r71oehgO3UPhZuAnxRAgFup9AuGBGuq6h9xOk06gi0ldBWMWcn25aEQOAsvzOI2kBMI0bDnz4WTznkwE1\\\\\\\/faF3FOxcmnLPlCd6lYwji9dFpwGjyp\\\\\\\/i+dXhsnGI+AUkqGDRTwno3weAOACO+F3elL0VLVYRSBiILSoS2rkG4VM9CzK38Dmbm6n+hzgxUfjDNwbtH3CIxDdueJ1eQV\\\\\\\/NtsK2zrfKqSfkIJWxIwxjQNAcLKMFZIe7fSZn\\\\\\\/5EoHroeWCtUssJFAMkYjhXMszJipEpZDQYYtaMMjhmym4F3XYG0LON5l607r3ItOSLPv2NA0czIdFKQG4GFJWKwRLpB\\\\\\\/dRoL7adiuiMWtsbn6Xxj+VUoNa\\\\\\\/RTML5G0TQzk2cl2AIzNp0Ogp4G1QgLY0Zziap+f4Sra+QdQQCncOqnqU8nHW\\\\\\\/Az+LuWx31MSGlbGBC2t\\\\\\\/ZrddKcydVNuc6WLA6Gc56zmkVGZkeETrq+v8wQjH6kGKnqC1ZvKwS294mvqw8yYCrAhu32MrIQQqcCkhkrQPH5ozf5ttcp8Or4xBUU3t0rai0rebPhI+X+JWmKQZ1+MQMRJzBr5rg\\\\\\\/hRaXI4bjp5jAp7oUTV0Tynk3mOiHE4laLhe6lbZEWVXkH\\\\\\\/Aen+jmugr+aoKbPyRJXqbIzPC5RAIqQ4ZhiSgzyw8T0noyWcsE16gyrLkm3gkUiB07FJKbyWmiNnO8Qh4XF99C9RhNU0vStlY7rsaytlOm5B24F1Oa3aHrtKlUI\\\\\\\/HUNgBeINOo8bPqIbGa2FAzcs95cgRdAJy+lk3CNQtgIyEp9afwgXwN7S8SBiXyXZa+LRYcgcNA3ncjovHYvVqlGT0qSjMTGEuqqUqpG4bvGFt3AkYomXpTX7nHjqXrSQnn0EC8pv2NBXleQOMiRRGeLaabnGl2cbiDNPeGXHjQjULTYZhBNNg88YF8xteDp70MfeWitqZw9WUCE6+gQ\\\\\\\/v2QfPkzaocZ\\\\\\\/2SFzGVVq+PRx6DBhFtgmAxq5HPQI8t3FJ+77mXhSn3xRU9Qwzhg3I4AsfUkvb2ioHsns\\\\\\\/oxuFJIuIdIZP+hKFKfDCnPGzgsipK\\\\\\\/\\\\\\\/1APJRLTI04H5Rv9sJzM4L9WLngS\\\\\\\/eO6BpbMJdBwsiBnxmnYAx59RZ5xnId6k8pvxCJAk6cNqFF3EET8kjQ81RDf6T10qFtXPUOTDUVmK36+fSz7s4mLzvgoHSTCiKVQH\\\\\\\/VYp2\\\\\\\/AaRRLTrJbe9qxuAujY28sL0yMMZcLEw0FBqJ9g\\\\\\\/lQyGrm\\\\\\\/kn76JrR67yy8MLwb9FQqMWchIRChO9UO3tA+D3JyYS4nLcXnu6IGZmBmsKHCySI7QCUZagYbg0AI6ONkenHcltBGFeMGfqaYZFZtVbSMLc3nei7TT7o7OAwXNytQrUVZljv\\\\\\\/6pLRpWmfoxRjalJkCZjlXhcAB9cpzZIbohtZQeOLf78e6vzmjkNDE10As5s3rOUnW2lIGr2sBEyMKgh9ZeMmE4wcCQm6YROYQMkUDmwoCR4zpGdW1HyvmdKduZgu4OyNjjuNNNdJqfIh8g7GVru60UlrjcZun0JrUdR7OqBOrr9uL8chw2ZV3huktOL9TtTeH5NjeZfI9JyDos2gW2hCgnWZ+wbgM9h5l1eUY+0za\\\\\\\/1Rh1mPKNZtSNgxTBvkoUT7ocTgfL6xoKnOGTH\\\\\\\/sCl8yRRDQsI4FWiYFA+m1TL8CRK0TGBx9TNrXTPij7PTmAywGspVT1lD1H2ucerDiUhRt30sR03GeZgmj0sh3LaynDKkWFs63DvVYgLX\\\\\\\/pkvrZgZ2XxLJwRbv0izXc9JTVht1ZLtUn9d8IQ5PMop0R9C1rEPk72BpjotdJpxeEFyCtdTgpkrA6jSgGyP+HPcfmdmaKjDJl5EFDA3lm5dfuK49FdUVvb3UynIsS+DFBnl52Ccy+RW7MCCzbHxOzlA9zJbRZjhX14F6MLVTOA6mvmz7uUBYd8Z60UUE98HGCIw4TvjKUmnO5gp99t4EJAwplJtoDcQuFtzMnBwzfepe1UGPoq9FCnVX4VcbmQ1TOS4scMMmNYOxAjn81NU07uJ+RjnY72kkWgzz0vNh1ga93+Yq5tDLpwukxbZLcLJqm05VqZld5aC1ToZD\\\\\\\/f+o5eBEVcVY70EehH02Bz0CmGwaT2Qze916WNbQfmMs3lXHgEqGWBtMEcWCR\\\\\\\/kGK9oSZGAMY2yH3euguvQ+bU0wukJlcyZcGJgrq2XmMLbSBW8Hxvoy2o7W+vjrQY8l627uPUeKK\\\\\\\/ijMYctV3SPbUJWzy8L5OwYUwm+9ueF2oDVegHBCzPqlnEzxLJAOHdNDZTF6TqM5d375jhoAVtE\\\\\\\/TSiJiA77m2LK05tXszdFmWYoz7SS6kmAOP\\\\\\\/C6fsDe1rl5qk4lJFCLBUVbTH1P1IYkceWZiAd3AzQtm7CUtUCJ2JEuogMFe4qRbFIDugVc4qaT0elw46xVTsExBk6dXSmStfFgf+DnWucV2foznMT\\\\\\\/Me4vobH22Qsko7+sUm37uOM+ER96F3Y7ufoOArL9UNWbJ3Ek+NtLv9oM4nG1dkUdk\\\\\\\/Bwv0\\\\\\\/Rafz1QcY2ZxFsVuMzJuIeZsuUV8BGVBfMjtULHVwabWFJFKB9Bbsvanjd0l7ClMny2sI6BfnCd4hN4Vs713h4lgaWgzAoGjMQ8l6tRvo\\\\\\\/PGvaaxE1AaWs\\\\\\\/23ymnfer\\\\\\\/2aYXpCCOzEmf4KW8W9Xoj5++jPOIDeIBOcRLSwEn49dc1F7KrhI9zlGhCwmJrUxTnaCUkhfgl9tApgl63L3xj6lAe9gVvUrn+hMMoQ+PERuC7cnsjJi\\\\\\\/ZnmhBa2ifywNjYXHlu20EjIUe3EU1qOgj5\\\\\\\/hqOg6Pdqilj7XiQMT8BCFzFBkiBrSfSFXNVfA+\\\\\\\/7ApR1r\\\\\\\/BxOTH+say7Qp57y8yMWzST7NlwJd9ApJN5yzptAWrI+gXnpDfelRts5ftB2vw7hgqpgDClcONMIRjck9ljRV5LaPcAI5DJPmvMsVj+4TaU0qnK6xkPzPfmIYtPHsbLTuKuj2IqKSf17GttKySM+JqyUDkhwFoR55brR2ObOdVoKyDwj1S2KhaGh4juTSs36nNjlZZjlbey89NYpMWx1wwEusQNh+8\\\\\\\/NcX4GzRaKOa50OsKQHSpFJdgJ5v9dy3AEzLfAQH8kLA3P04vtJ11jZshpsoVv5ZeZB5itTazXCH9Lr4ajq53+W4uNUbR5FL4+ab3mY9SwXbOpV+2jDAsh4lSYAPAxYsoDWcoRQoQ1P+TTD8Wug3Tk0kzM7LVSQIa7x2qFWy7+O50jtqRsblEyMHp1qeSkmxGQkeLFGOvQEZbkSbJXWXCpUkmTqUgoWxy6gbh4qAqPdURJc8JXV9TxGDP2llHcCpEDR5CcNUm3ljMN98mE3LK2FMMydOBS3zS8EXB6dBbd\\\\\\\/payYaxHk+xjAuaGuaUdwH8qxVHKzLCmU9DEbyH6OndzSuIJZAxEzdI9aFduP5BiUFcAD0vmbPAg4UBmM6dG4sPZwY1IEQNcB+5Dl35WeBdmF8Q0+VoXX\\\\\\\/7V1n849jrByFoSpsu3bH+pVgimmyIjSZH1qzt7kvFehsrA5NP+zRDs5HUr8KxizCR+4ql2QUUB80nah5RILLiCME6StuJOJD2+QBSC4sDRLe49clw8jng5HwhN2kvf57\\\\\\\/iMvnIjY784KxLKZuRbgRWzm+dbrERKbnGI7he9kDKKyZRC0t8MnOVjIZHy3Nh\\\\\\\/lOj2cNDOabx58kOnbtQAH1NpWjLF50MUOk9ZB3IZdeumZq2ji2JJvdcdFmHH0WgeLYi\\\\\\\/TL88xuc14kvBvZra8i5kCML8a1R6SjXfk6TdbWdbWr8qoxacDvqZKgjdub6aFM4nngQRw5+8lKFAj3BUgzb1jiwgLG8B3S1deGJ6\\\\\\\/S7MvFBrM7\\\\\\\/WW3cYSyAj+sinFrI0rXJGJtfmmMEIxig1EkeZ7BS2Wduv7F3c3s44lab6l\\\\\\\/unwmLcmdP9pcyBTWjRrQqq+SBmJ4VJK+yiMCaRBxiIACGF2xxpyTWRm+il\\\\\\\/3IZPHPVposG9DueHoW+QX\\\\\\\/4efHVnEckldU3RHKGWV7qox9s5+x8+eVIV+WBkAooe9+Gq\\\\\\\/hCOtinYOuJzkiUB5C6sicWpTsrpijaE1lBsacqUMnENGWeIvWRL9V0isEQPSljBdplGPI7sLpJVfaGmWqc6+Y7rUhdGLK2q1vm1Zafo4gc6pZyqCds6QK+PxuHSr7o41iOypnqXDKBXML+kUAZGHzn1lyKFJlpukmfvrdWRtMI7Hhs8i7eDfUu2iicVUtQKl2tpxRqA6FJcEM+6uK2VN3oQtSSbhLcYgK2eieu6noSh7vMkgEp28Xayr5shW7PAQA+dBWbLvRlKmetXa5PP643WbbKYmgoQVmpqC+UcrH24+3SEtq13SsCq6adh8idhHkCaFYrIGAIeDvTJl+9U59VkFyK6hy59Ndmeiz+ca9KH+zJVeUtNZfKoIgRP7\\\\\\\/1T\\\\\\\/ofogDkiqrgrHUbq0aVgMLg+rXvwUGBr6wW2N3V86BmhnP7RrPk7oU7Ke6CaJObvgI5a9adS3uQlky3P6Ky25z\\\\\\\/JLM7Jnzyyl5L7kXhKTM2+IU55aiiSZX7llfFMP6CXglJ+B7zsugVifrtsmyEvUn9ZzghggfZXTK1eempOibN5TcKlwJ5LSgiX8wMVEdtx1YfRZm6nFz+4pG9E1hXWBmqxKMM3nsjJ2DsjTk+nrqpcCpRumVuwHud0pFBe5\\\\\\\/ykJ6WAo+y2C1LZlaQgj7zHJ2qs0OlOl+XUSyxVDOpKIJmJD2QIy4drFKUovxXR2EyU8R7JO65cRAWd9fPRNGC1kmmtPI0twSn\\\\\\\/bwE1buIlEcG7qQELRx3NcUlv38XSoJ0KDozp8xZyK8+764l7XHts9AEnj\\\\\\\/3GL2f2K5HTNlMK+AWk+SvgL4DzS1FrXEKB5N\\\\\\\/qu++0xycUmpnxmtHUQsD2jyu5x5UJlBEtFw5G\\\\\\\/ymce\\\\\\\/YsxCV3aWQ4zye8VRAdDtBXTRB0kx4g32ZbJLeFKJ8MIEd9\\\\\\\/C6ipDE22C0W31Vm3xVKOckIyY9amJLoxqy4HP0tc8CYAIswDJOkZfg0TKergfYp9UXw9WbOA5N\\\\\\\/3ZsM3KILLDhoWEyWZUJEuCdCMUm8B3HpMNHcwJWZPra67D6mL3VPfZe88QgUXE4XyYSaUm5VCU\\\\\\\/Wng+p9+U7dhyPtEBTsFMzkI1B3Izfc9XBm65EuGpGrWvtgyhYV4X53TMUa7uN3xUi1WKb86hT5w+4rgl\\\\\\\/CrzbgRiwY++ZigtzqmXvsqo0VKCEFwmyhYJyZFKIhn9hOYNnUHXYod7EJDPfq6KJqFLA+WxK3zNuaooswB9HPf9BeyHkGHh8i8TUTzFGFnSZiCbE7FDEmZzQRJIwQP0X6du8gSiMwPUH\\\\\\\/1yMv7rUqfdFq+G6Dlc7eNq+Py7fxwNAK8lQgLYoKzm7uj7NkFYoggFfbn3us5Y2o9lh1cqYJD5qOLDlufiXLTguKw0hIUvk2gck7FhYA26hnjYUvqkNK7k0pKRTnx1qUefQiRYiPP\\\\\\\/KfslouG4D8FE0EFMeOjoNZJgOxzPdBhQCBYQdi6OxDcKxbmGuZtoqPpXkVqxHBjPieoqR8kmxwdwx9Vxou1Z0tGaNYjeSD4nrAvKX1Q+zt93AkcfkgO5ZwjMqAXktDvk88qiirBeTWp3YacGPsYam\\\\\\\/0kaDE3Sgy3Lt5lhLQ+5Scb1KMaGb\\\\\\\/0Q24K8SoI9fNdODSGFAqmqTpkZKSUVuJ9ZDwoagGTGwEV9Iu2oQBgzfBybQ4bcpmkXOoai8CmSLGBVzQfVah5knevFGS8ifHOhzxV5dP5yQ1EbEfQbmE3Y8wraOwjsb9ejxquqcwLBJbUrBrWPgWnW7TrKAExNep8M8wLgvtjCEXPu9Rci7IztvMbobHdIOyBvk80tKv21uU5r3ZwtbhVBnMCYTx7s7yhnYC4h70SurGFSFrKV94qrfUXD8DjzhFg3um5hboNvpYbFs4CnbUWNBEDMEc\\\\\\\/21NQtjTnkMjODz\\\\\\\/GzNwv8+\\\\\\\/nTQCRT2OzUozfCTZsq6z7c2a6s77bFq5y3LwM3v4U3VSE3SI0TSBS9vvUpSnMTvaU196pGT7sRRi9uoODNnYXKiR0uyNS8SDOaGcaAlAwnAKKeviZFcepXxRVQnW19D3vmTGUfd59teaUq\\\\\\\/VuYyQALlwBBb3EKE3NpZCN84wb2WmGYpcp6Itr9w36kCAmi5B8JQgluiEpJielCIUrc+g+D5ZZoLBQ1LnwVsvfhdYDS76ONG0a\\\\\\\/WuZqIkw5\\\\\\\/yIbc0QgyFv9xo20WpRLWKov2R2o0o0HaXVeeWNGWCSZ13v475k1c9HIQ+oZdkkOm\\\\\\\/QTl6AlSi2JSAqiMhuH8eAPQoSPQyn1bWgfDDpQcydxXvF1cbgefpQCTPLykdw70xot0PZmlpBi3hk3i3RlpUXZdvYu7V5zS14Jm5LiuuzLfolw4x46IZwl0j3aWMfauKDwIFZOtPiFPnQKazd9O36FxeBxRWnA9qVZ7MBdu0ik8qd9ZorCabjfSOZhSrbgS108bQghrPhhS4zrwb860f6Y52Wy3w8QCmMaKupzfmdNW0If1XQgKnE+5p7XaryCdSNa9WdK76S5S+B5pDIHX3UsOl9JEd9o3j+XAFWSI2e0Gt+6GYqN6A1aNalaiTx6J+lfCtBO5UaTByhxAryyYE4Gsft9jz4A+RYEbzUsyKOH92EEcl3\\\\\\\/9l9xk0924nSp+lT5mnOaX9zp+EHtGd7JzjUUoGLNxAAk2o4VlFIO\\\\\\\/hTLjdROOUJueZ4TuBjxE+8\\\\\\\/gN16wdXjrAMf\\\\\\\/wT775K8HyB55B8DPp8aUg4TVzze8MKkr\\\\\\\/tUjoA3wbzcz\\\\\\\/yiShrUomWm0HR0JA47+dx8V5I9ZPKw5ZCKaW5gzikg3\\\\\\\/jcqgcUSsozzGM1UIVzg978EJnGn71797IKY9TgWW7j1THqjI+yN5vvxzxg8LHLy1ZkEJ8f3QmCWbINVtF17KhdFKZk5C0nL9f2Ub25x8I6wPLXtewTMUccE3wsFSDMuFGMNT+ygCzfL1dllG\\\\\\\/ykshOlEavp96PkkfJ+HuL7OnY6P46UDY4l9Kh2sD2x4tnN4\\\\\\\/3IEiGJH9aZDLisvqt2fBAOk6fYYT9KSIVQiM2MM8hEs641X\\\\\\\/7JN0J1650qKKSu+QU8XXD9D\\\\\\\/P9Tewo6pWSVMsVPpr4qLT3K1XT0m6bCXvwdAZ5JhNwoOcO\\\\\\\/ss5PimAqW0moI7dlkUimiVLpIO\\\\\\\/Mrs4dc4zqMdlnWzkn66EWmoLNpmCQFQmlZWaproj21bvcWCn+9ylfr5Ao0FlVnX0BUSjUQPZQ9CTS3Kh4IqbdkE4ASkLR\\\\\\\/qthkyV2EvSwv6BKyhp6kNk2JpqaY8WxFtMGCPsk3mZiOmY3z8QMKVngqgOz2d419EqC0HhfHROZZaxpC91mqb4xz8jVMQfgb\\\\\\\/EQjDLQBvRDguIim\\\\\\\/y75kjvWJrmabrmyMmDmZMn6as\\\\\\\/LE6l3ko1JLangtmd+p\\\\\\\/xvFkzsjUJZAtp9jW8PCFju0wSftNTCyO4az2W0g5EpnnzWAkppKwwuP6b4joxIzsTDTo35lc8Bv0f1FTGebBMTMRfr896fycXmBhfll4vqs+DU8On8s3ZumBKk3KjQvzGF6+YNKFSfOl\\\\\\\/OT6apAxy6iZq8NM0i7yoEz8xUgnK3Pnk4ZnhUIGL9VJ6x01uItZowp92k+LQv32py1o9Vheu8pBC1OKTanZZ4cOoK4Vy\\\\\\\/IiifoTOF\\\\\\\/eCOkbxQ0011NbTujeQMVOS89BD75JdxfKm3nEoagA3RpVHaGOxyjyr6ylT3Lh3n6BXX84zvcNNrFjEHFpxVtOH7dTxf\\\\\\\/CYWpVQ+b6svfrbt\\\\\\\/IlPg9h0SImIWigCH8z2GPPBPA6qhtCM5ddCc3gwdkvV1E\\\\\\\/35TZsYLOYbr8wJDuoKAj\\\\\\\/mQBJ7NzwCy57Ntt+6zbq8TqNhwzBmu51SgrBAC1Is3KQz8Rv01RR+KNMlnekQJ4Jmo6O5EL+GoCnKWLSrU+vBleyJAmHj2bK8s+5nq0XvkDXLIRIaa+gCUURzMpreBSAZ+AH17CZh3LT+q8+acbT56xgTh0qfkKweiZmVlWqS63vWMlxZO4Cy5D2KY2j3fOINNIOhNVdk3K89Y0vONGtEb\\\\\\\/ReqSqHNSj8LPhV0bSF76gFv0DmqRtsVHRCsbFDFzlflf8Ud3ELYbUbPedq5+hDYjHRO8qH8oJIgYUHCWabKjsC\\\\\\\/D6xEnyDpWkpb6AUi7mmYMQbmEfSdC2RRBZJ72FxzF+EA5DkQMBZfAmiC5Uiq3RXtjMJQ9axYzi6NJXyMiXt758\\\\\\\/44elvTkInilf0nvYOz0kYgIEt2ujRjeFL90sWv3IFF2KmovGFUeQ\\\\\\\/U1THoXLP8NpLiAomjJ0b2bvRM73k7CGWKoCoLu5vu3GqrXMBJ1rXJo+fME\\\\\\\/jEuIKOA51Nfz5+ceuwadKW2b00KsNjZwOoAJV0mn08ZiqhUQHQFrQZX1sHmSFHAQPiSdBVmftsUQcNL2hNIFVqJlY+U5L4rxGN\\\\\\\/gFPbiXGG7U4lH4g7eEgAkVzQXCpVhg\\\\\\\/WJ2RSP0top391SWwYWkBn+2cKQGazSj6vunEBeARkhW8Gv\\\\\\\/77MI+Y1qJf9YBPZ9Ep1ZAr5N2PhymdFf7pbj9mi+N9E\\\\\\\/sMTWmDZv8Lexz0aF1EiNg4m7ga0PaaKFxN18swaXy+1RadishBbf1yYiH5jp+9uc5dxAunMP2MucZ0vqXmdeXEJJJd+PvWKLrVwhSUldHI8oHImgOHfia3RrOvVFB9+KZyjiHfO2yw7rU1FiMW1BL9wl+3Yx1Kdmi7GU0gz8fYeC6RM2oBHNm2WuoDyi+uyEWoIMVMmLDetCQYM9oja0Q04Q+NupM1cZpwe4fLVbMqegLHaZcQjho1pJw\\\\\\\/PQwnI7F5Qe6wNbehyriVroIv5Vf5ais1bmHZ7K6yLR6UadZDOZVE9HcqdnAlfaWXU4Kpr4EqKb4rRWk7Dzu0naJf0POPhHGTUwI7bLA+ZOLMsKILYS4+vYFbTwNiZQG2S7FxwPpNi2SlwvCXD7bxTN82N77CjAg++OAUgisCQWgIEPcKMXAZ6GnwbFsf9NDvj3IK7Uzrl1aeMQG2wUrZwUQOABt23Dc8BIrYZdRy6Sb6w+gtdNMHibYUoJiZXx0wO93tFJCD50EngqT9vgXiNw46Oq35rV1k5ffWXQyXPAjhGCj+nk0vF70AFsaMIW2wlOsvBrPiIpjdR8t0L2KjraJNmSzJLr+FmB72evJzDczDiaBSeJ+OC9OTDWRXlJ78BROtaPDdTFlcxdtjc3GJzj4\\\\\\\/PHq2Zr1vi1JDZwyjMQ+nIr4LObFSqlLSVesNZzSIdMHTfYn8Jf4hb4Hjucw0Q4fsZMMVIuEhBXytERwxmKU50uYgSWHqxxBTIXD98zPsT7j6rt21RjIxcH\\\\\\\/EJwRXZYN+q6r2d6cBnhsOKwNoUbpKVEIEZx2Xyj0fEz1wQtHdBhPHN02jlziGzVNsUHcFUcFph3bl7bOdegcYVlBnOo58ajsHnERCdGwqFOVrwQNQR86Y3BQ+eBrlWoLSyMW\\\\\\\/aGcfxmDwFkQbGvJ2TpAujMhA3OhFSlgZmVuQUVAechnILNOvNp34PxtkwxNuXh29cAPkSVFez68hrbv2LCu\\\\\\\/Xt\\\\\\\/001cWYynEdPnohHAToG51VcBoQ0WqZvdmV8vCkiqwsFLhtKijX9PhFj02iSs3aaGnB30hoUvUWu4UWMS1zWhpP70kIo1Bkk+qNfXUDomCgMdx29QSrorEpz9OsvjpunMIyuv0n4oksDCcIxuF393GGF9uJNvEHvTOqA+gDjz8jtuDIFBgfmq7FcmH+Ef7JEblVHtaaMViq6Ys\\\\\\\/v\\\\\\\/YbsGx4YhQ+44J6OlZfONbKrd8UzFk3\\\\\\\/86bWM\\\\\\\/6eAXvQtPuEaCRjTN2Nwh3hSoy8RfW6TEeVXUxd9xxY9kSz036Hy0fR2jyDkzdTBJwQmhH71cGHdTxZBg1CV8mn9k5fSniFuDddXSCRUZEcBn8dWeZEbCTzfeCB8sqYi2iKO8a8vesflTCaPau0AAw1bqLept9pv3S25kGRlTE1GGOEMPC4m\\\\\\\/Y8tt8qlaCjCVTkb9AZKpKN8WSmBwOtfF28hfLTWWWlE8obQ0AUeqW7hThiTgUVQSHsGBxxI74YGgjcXRnlFXp4PiFKBHUys8Y4A5hF\\\\\\\/Uc0qPuyuR9FQqIAQZbJPBgikB4yhQp0ysTrP62PTqSEYnKUQWQwhl\\\\\\\/PV4o7qlzL5WHJ5Rbkfp9\\\\\\\/8X+Lc\\\\\\\/b+boLcE+6z08\\\\\\\/zFntFkvNprKS12\\\\\\\/TzfJURrtvRKGRtyNCZHDnv5tZuoZenfveBIt6ndemkZ807qJBIpCoz3TPKBPgaCwuea0ONBr8DCBbg0rHNeXIgntgK4L\\\\\\\/DaVX3NNjReo3K\\\\\\\/PzgcC75UbUpVG\\\\\\\/24sXTRKU0FzYvsePpwKrCvN6Nzsjh0dkawAAGS0MtUD6RYvoJN3BfNFXj4P8p7Z29CcxefDis6ofzLZKhwhF+01zb8cWLHGzsWsR3dL83pU20BMGSlmA\\\\\\\/YlOkKb\\\\\\\/6WSZODCE4wleIO0NH62MVu+Zke9hrjf0mssdRIdBuBaVDDm7C62E4bAds+eZ0u7iiIqSyF8mj1mw+xCGSDPU6fGMUGkKAg3+rv5k0SFIXTYlb5DpXoYqbBFkbt8mZgRtnQyHdxeXiQis+Myj+v3DMazyBDakdhK4i5C2APTYEkIRQJS2rqIyxiTQ+kpAIZ7xdP11tsi8nMOUKATkxk9yd3wFI2omOaQXraigjAnwTMLkyQ94IHypaHcTBKfX\\\\\\\/589Ge6K\\\\\\\/hBDvh\\\\\\\/pU8mTfIP7RKCyk4ldyjtTw9oM2PD8NJYKnTAcXw2Ce1JlF3mS+rz3SzXaRxwohMHN7WK3ncJVhmd5TviIbpK+genDH1kfoLzalfe8BpAsvrmFAemvMnfc1folHqqyuAjexBOHKOme1snFS1Fzaj69EEd8kYVIsGrrJeejLynHe8+cNp6mVIEFEHk9vTNnDnxdJv1muY3jCFo5s2qxDyD4P4GGLY6Pw4qFRQjg+4P6k4VJpm5HssvITpzAL8kx0eawqTaernB7DWLGo84\\\\\\\/sgmuDvpyk5PCCV2BBM2d2zJ5+JWq6PV0JqBVYJWK81UZohzs81mBGMK6p1k7rq8dmv2Taog1WY3IkK7Vupgwm9mGM3WpBSd\\\\\\\/BOMofC4kUTFBg5G9zXxAiEPF4KDpUB3yud\\\\\\\/IH9Yq9lwyxp1BDkcHj1hJvTz6GtHyuQeW0hrGCwfOGIIZbqllP2wO9ecI8xmlb+CgyIhkhMsTZ5ToVoERvYlm4K4qRJPdKej0WZ+EVL3hb85yXuzhbGCfHa01\\\\\\\/9DopCNbwISkge3H\\\\\\\/s1CmnBbsp\\\\\\\/AXRFRRhSrjg9ea7wTbwiBCluyuHiaErkue00Bu9Im\\\\\\\/gi61X\\\\\\\/NFGG+cVqKMUpVfYMZVdH64dNWsZrHe27kjijjZEBoSGh9XHKFEzgN5+2RociFdqRPl\\\\\\\/IX9e0pyW8fUMHIDjQMSQqIKyk9VdSpHG9LbLKNG2gRLtxYBbGl6aU10orMspcir1xmRurG03UXY6Lc0o+ViTouvwoD+ccEp\\\\\\\/OF\\\\\\\/+ZJ1srE74YYayG2KWnXbmlB0\\\\\\\/lC4qgTRhAzkxvObTN4\\\\\\\/p+zzSEJadiI5+2vsGEYGWSaxLlEOQeQ\\\\\\\/2MDlKoqG2KStbL29gtqVC7FlGmf4gIgr0tYSDJj6ugXMh2orlNsSQT35w5CWEaecjZZXRewjfhm8wBkn7TEU69U3RWXLqHVUtruVEvf+UyEKZ7xeEd0mNzAYvg+UmKXb+e8pUrgv+Z6QZfjFStfYpSO14iYsbzLcVslB4wSW4TPrVWI2EwHcKQ0QPp1G6ELZf\\\\\\\/Tq6gZHgpZnOrBIXOeWI+RCl+yXjCSlKnTdzC+HJPeSz1yWaQ5Z4kcEFiR1q6O7sJKsja6VGjZYev7hecF82XPDescA64W8cSCgdnk+C17hqGMzSm75nSnMuvehcBAHN6TH9\\\\\\\/AFTUi+lEbxz7jiUUiIDybeIXhJ2hZaW\\\\\\\/7OIIUJkHoTVHNQRnEDVw1Q\\\\\\\/LXisjU0eDC8zFF12s25AkQR7C7GYKTyTTRFN6BYwssYQd8HDZ+7MNaScVtwE9QxWLEepIBJmi4iLExVhE6fMZqYlIkqHvDuVQ20kQosYkY6wrimaZD5JZc72vHI3VvG85zGAob0VaVldD8ldk4Xtpbg5Av73sI0ChE\\\\\\\/OMpHp2sYnSvqNmnc0Y+9Lq4abWRHEp+cXqpQHlJBJ3OwS2BXfB6cB2lpIvoxVrj6TzlQNUp3oWDUsrm9bPymDXP0pKAjNZEnW9Al3wqkJROhEHsj3Qs7QQxiE4Tr53fAcLxn29\\\\\\\/YXrz1KoUbzW66rmpvgdFNW4ScA0jhob281m5Uf31jc4IcV3e5lHAYGi5BEX17GwOaGSDmU9IMZ08gF8oYHUkj8D6TGN9BHSdt7KeqjKEv+ElDAI8La7OvPqxDVU4q88jt0cZRb7RMddtKys30cdvpNAH47D3UWw2vmOmkFv2bzdKIJ+Zbgwdnckn+u4Jv6AVu2JB91lpM\\\\\\\/brImhXrvttuJMG2IQioQWw6nePDh2McYXqm44PiPC5XAXREalqVKc\\\\\\\/EeQd0q+on40x8SIYP2La7GQwIGnftVAPpDMmalQdzYfZShUTOwu8Xg0aYl9zaPEey1X1YSzyvIY80DTMUM+ZA82mm1PQWv5QCD0as9cvn3gL17xo9V\\\\\\\/MDJEonUQOvUd+fTmCbLXEG4+Wq+9SJdPjE4bq8LjNkKvg6S2ha51qrlH162Fq\\\\\\\/slNXgs1jj03DX2CfCcfEui2KIYDEpkrt\\\\\\\/PhpAIiQGN1IReZDtRaTy45pZ\\\\\\\/Ms8lKLQRN8ZRK3cx+UjvmYn+B86IHrdCDwht0LcLMqArcBr6IrPeBc\\\\\\\/v1nVSPmrnRTuLr6hMTLSpRGuYAirJg94OOl1Yqszdpe6l9UqUo37IV7WEoqN9sQQhDQrMsfnSilsS6So45yLbmUE2KrwnQAPzIFTMr\\\\\\\/a14ITd2a+Ln7ubPvn0BOm9WRHlwV4CbvMTvjpQOL9GsxuSy\\\\\\\/QnfOG\\\\\\\/Xo5HtKoPpF3f2g7fIZkUY\\\\\\\/1lsaLmrSi+pMaoZCZba9r4xo+F2pMudXTD1hZhBjh1XAW8hMZy9TY2VNU48sX8lKnys4q0t8Ytbtf8pHTshtvkJoZxK\\\\\\\/AqBHHeuOQtbraHXbyg8u8N7SS5spTcyWNoVzEyUcxsUcDODGIaz1YQ+yQ7SHTiWjS3i704CamTsRtmoLE54YjSSr5pk5vFuzecIm41ynBYDNY6XpD+ILS0CWYdJEczUb505XNv492UOozpgF6bU6N\\\\\\\/8jEn9mEErTBpTduuxs+Z2gd1XSDRdHApK9pJ3e8QawM\\\\\\\/BMTYTlVjakftekvtDAOstvG5LJdTECfL6Soosm9C3shNMlSoRnSqKsCB+mwUh+h4SfOJkFQQ4nkPkiV7l92p0GyUtOGTXB9ehIKmgiXcBP7\\\\\\\/oav5n+HVdQ1yq7ryaZ+cFokwHNL5UD1i3hOP3ZiQh4is\\\\\\\/qCyjbGk5UOtvC0WsZDFH17jGvN3hOAa0zoU4lUYk3NNrai0++rEWzMOXrpzGBTQlM9gDX0GE92AvHA4aGGze\\\\\\\/HBhXwxxLq+Viv7hZHgcQP\\\\\\\/JICAkOQUCyjic88YUEut7P9poPP+7ms0PbM1CFvoQAbrNhkJMlJLwdPVGmIctqYprhVa9WDp\\\\\\\/yOBMJDAwuU7p3isNsyq0\\\\\\\/jP62p0lmlP39j0S4\\\\\\\/yrlDWYTwT0HNWsJJoSbMC6ohmrLVH1sGRmtKKiOS7s+6C5t9\\\\\\\/y9kNqiqlq03+MFphfvFtG+3mJDRt7qAOOrSTt60M24DJUIKaUIaxvydGsp\\\\\\\/3wCDt2jawPYSbaNTYqbd+q513b4IruEynMVaUds03\\\\\\\/iAtmTja5HLVB\\\\\\\/qQ\\\\\\\/PzUZuOGjxeShuKd094gEtiP6JahdvKnJMXG2RKuiA3DbLntNknalwjTzyqlOD3T66s9dPZkJRCkMUd5xsvMw+iUnH3CEQWPTP1F4DG+s\\\\\\\/l4dWE68QTlzUxADG8IOXbEp0J0Sq0VGQHVWpgPPhD\\\\\\\/2fbkxoZtirNA1pxlgfLiPUQv3N5JZYBiVNZUvwcVw+rM9raictiicTXNAfulAmlR27EUs6Tk9mHjtECHJlPQgFH4MmNRvQqmGy9pfCQpwi4bvdSneba29RR4OXQwI+3aea9McR01cOL+51YFive0OKQ+bXImqRtX9CZ4YvAJPDCRW8jkXJKc5om8\\\\\\\/vxrJXKseqlSNUnJkrd6X+dUrvRmTSE1I8cIkG5+Qq2Y1Bf18wam0QRuLMChGTFE5NJw664Mg9j7zQoi1Av7BlXGoePs\\\\\\\/YughPAaAsVM3oHsr1Zk8LgLrK0iBZq7+komfInMvbZW\\\\\\\/dIHPJmYc9wJI1r2TzM40Fr5LJ+B3+gujmR52A6CWA1\\\\\\\/qg77ejJe3KIc2u3Dp2ChCjVe7gJkwSErA\\\\\\\/BL9GpeyiGp7VBS3cNUrbrQplL55+RU1IsMTosr\\\\\\\/Te1vglOeRerVBwaNHKuPwsQu0I7lBcKvVNMF5GTHf0enFSydWdHoTPxCFtxucff3+cOe1nZuuOcCf78aVRof4SHZo07zQ\\\\\\\/dEtQ7TmhNyeKtVDu4zcZ1mUXh2Sd9CGji0SeRm4x5BJkizVETx4NTUP+5QzBUcToOH+yxI9r9JpopeCLTe58pnwc18KXdM5sTcKLAF0lmNZ4+1qE3UCrVn9EnsPda7HOnxK\\\\\\\/bU8+qdepsqdDqTVugOQTtgxg2+UIoBqRpvvHs1lE7iFcdLz0Ypwel8sXtHoadnhTz36vKmxaPc\\\\\\\/d1YSNz7ZKYVxnRpH5McZg1\\\\\\\/WXhse4HlTTtUj6s4VLZzaCqO0c\\\\\\\/NKbnYr1INy6AwWu83Sx5aPpctnZAMF\\\\\\\/ieMWpavkTo4ZIIVbd7KiNQiVxv01KHComGCbrB\\\\\\\/aKoF8ehK71TDC0Ipiy9avKN7JsAMr2lUwmrxk+Uc\\\\\\\/GoEibE\\\\\\\/wXiyNeZUzcxpZIoadRN2sL7hjqo9sPETypeNJ1TxqdykKd\\\\\\\/L58\\\\\\\/ZrCFGmhJ8ntG+1sL6ouQvJbUPpYOnvSEt\\\\\\\/FC7YK7BUjh2e1ghS8kioavaxoEjzG5EvnxUMFK2m4YsyvtIqcr8+QlDStyTMwFgqvWMCK2W2ggdoU\\\\\\\/H5TZou5sz0\\\\\\\/rY+vaozqPPUdMyGa+52ax9GIDOLumtwkBQez\\\\\\\/rfvxbf6iTdBCEUSGDFVAv4pZzBW9dAUHJDMQ4vMetoirfE5YwR73rqmTBI+MGHHNxNKd0CGlS+HUMM7\\\\\\\/dpvLG+rnebtxTtUjSfE1C5RK2JRIPYfOjQxPCoIK1jr4WSNqMFj6Go9LXWmf7x\\\\\\\/2xoB2W3YV4\\\\\\\/2smJNRSWTZdoZZXIcuHVV2Xc9ZMj1e3aPFOCQbDomKlHjCM4VJUm\\\\\\\/NAbyKsC3mDdS1WUWAGUs4baDAcfUzWaDtBpY8pJZymhww2bmmiiMbflmkDSDpVThRmw\\\\\\\/S\\\\\\\/7IcpNToZNVW6feR8jLaw8JiqDF5AFTcCV\\\\\\\/g6rtH9z7nZ7yh1QN+S68Q9fDoYd2atyUGKoDmQGdL3B9kYxiddeT3VRrN30EbWZWfUWfn1gCM5SfM8HBCzTSxStD0FSqoGAQBu9xHdefr49xcvnH8mVTFFcphSpOEj6m4b2zYZrkWDGNW0UNJ2q5c5kA\\\\\\\/OGeYlsNoQUaaO30isGTVQpLLvil84SqSjtBXYp4uD77dWui1NUYcTMSyKyO2vdXmUguE5DJy3o8n9jlbpkmgl3LF570Pgvl9ClAVsvw01y02aukyIwj0O2sD54oewTfzhW0pwiarBwfxJBVRFq47ewaylYVyiKjVkcWvolUrja299sJ7UmBKlt04+0etKD2Ffnr6me3S2Udm+miQWCcKmm7hyJJIn5rcNVztdADIF9QjYuqLtrFh23U91CgszYa\\\\\\\/lye6X6o78eCdXJrx01Fhd+P2+dAyg1Jv55s6cwq9J5aeaDggOgk09S2sIVbHkXKIU2geFbb9EhtTQRUU1O5+7trexc0hucgO1HW987ArIoJb63v+Szg+GbTiPFhGTgN7l7amDCpW9FQdTJ79rch\\\\\\\/42MkHr8XuDUO8RSlGqdeML3Yn\\\\\\\/c+f\\\\\\\/4c9APkiTEYBIQ3oAvMXaS7\\\\\\\/mZzTadmxFJtDOJ3Mk9nVzYN0psZk49elVVs1FX1G10q1fR\\\\\\\/xsTtFZQUQKPFh8EyS+um55e8VDk8D9E6qou5JhZLGE1tDX99cmfpl5yHWpPTf3gBIppObNPAlor399ap7O+pJugvdinS4O\\\\\\\/cJ84dvEQnLcDAA8qIIFoGyv\\\\\\\/uZKOiEnzpVMs+wM3ERlIrdCYLrK9BZlt3bd7MMYyn+PmMH0MSA4YFAuntKhLT0jDNyllJK\\\\\\\/4mHKDdumOczPeIVhrmbdjD7cwjv23i9o55W0BH1dr\\\\\\\/8qAtGdAYFkdrHRueu0wjlc2urcYabYY8OqUGauUVSXZq587mKD6jkjvri\\\\\\\/di3d2idinARqPT3lZVI67vZ9pD8s6l2H9Puy9jdb6EE5mPyAaUoM4CjgrueYMlo31JphZU\\\\\\\/rV+DEy6j9ap+ffxMM3ro\\\\\\\/FD7jJ5MPgtXmgaAbC\\\\\\\/1B2FVYEDms85JzTlUrKSttQnZCe8Sf2iHTgct6ImNwVVOeERZ1ep7JdLqm2vNXGl1RFJ7gLN5+sSsyrorEorPEKfO91F+s7xw9EJUYE+T+0s0d358pp1mlyAjdAMpSi1OKcvQ+ldJQOhGE3g5j\\\\\\\/PMbRGsPetnSyeICw+0eBBdXCyy55OLlFbV69UCOkc5x\\\\\\\/r63sE6yoJlvCtZKS4P0ZpJ48+rgOtf8QSIQK1c1TphGUitcZp9BFWopyf8S37d594DHn7LtZvzX0vt7GbB2Q48KBmxQxaFEX7xmKvGjXocgVDoSlsCXoUiBkAmg2Uwhg0kcHQTdCwoQfkXLQE+y7RNiehd3afrqS+4CSiB7OQF7dMgflvJzO8q4ChN4EZfFmIsnWCaFfgxBC0xrLu2BFumnommpXJQH638kMS23zvEnV9pEVCFMiwq4606eB3H\\\\\\\/2mCzYhRPoFn\\\\\\\/6ZHBTO3jGzSZI+bqRsUsZZBL1na0SqmPImJ0Tp5Ji44TphiX09wYzBIttTQR4O\\\\\\\/v9YSzsUsyILl9UHXgkvF34NX6Xqo7A7X3tm7IFXo6lDRcakU85CyCwEoP\\\\\\\/2dLBhz6NzfjZtpsCX152Kv5h43IcbU9O5xcbt0NM+cPoLwgeYz0AILPBn35tFCRbY+Bu7tzm54bZMLqNJZkWBzFwoTxATMAVf3YqZ0HHYtBSgR2xTiBAeYrXQ6BIb816A+UX4wFlDxmEULcUf5GvzXjTtcEj\\\\\\\/6mSAmK4bqmgxN6cbkcJUnktwX6xqzrZKsQRflv2bVji6Id\\\\\\\/RaOMCQshTHyhJJtPnh7btejMGuP5fTukCXu+eqtGKyiNVnjU0tzvNrxSANGFEBuI6+a+IjSasJeZZUBx6z24Yf4GurSpVGL0GJrjXjncIYBuzFcjGjg3++ypSDFL2w07oh\\\\\\\/Mmavl4+tQy155xcQrdET8xVIoxYrSvxz8TNz+UT9H8F\\\\\\\/UCkqHUsiWPqcz+YMyvYsQbRYmHRQ7d2QgbPVuGxFNQAOiEByPWEyT8UTIJRuK4dbI3VJa5nHQ3rcDReCgQkETvq19AiHg0Tu2mwWamFU14xhqrey\\\\\\\/qY09MOBQCItVK+0ZMlKYcvajG+0PvhrMq4fnk7fQoLmYtJ45oYjC+ttVZX+WzdicXO4+rxsVG5I5tcsoAcI6UbejJrNaApJCvcNMwr3t6rhK9ygqjL16jNko4IGx1ZtwAYpDXsKE2e1NYadmaQAbQ0q9zAte\\\\\\\/BeUUmQM1ufNiatsvAFBP\\\\\\\/2x5AFT6KqJFOcSI0MxVeHcazWTFGnf+\\\\\\\/W71f726uW8TJ9cQnNK5iIwVPR+kPqlg1\\\\\\\/C9iMDo8HTPzsuayJ\\\\\\\/8w29ynsiTsfxPzccj\\\\\\\/CNyjtWW84WrCwENT0TCQjvzIt2sjPsuIk1jFG\\\\\\\/i8ZzGj6SIcUZHMRPFR5NTxixh\\\\\\\/X7naD\\\\\\\/mPw5UWQxA+CHWHWJKTP3yCP\\\\\\\/yIGOnsbCABxwLYmWvdqpZT++2DYjTowZ8UiJ6pzABaHMyroVPV72WpTiJ08sig8gg2nV7y1N84KgnQkWg1B7LrSIelBe4VYFN4GholeTCZqmVaUjVOhlOha9SvblmOlD6f80zPFzuL5RFAhX6EwtklOiyWZ92Cl4WKDTbvljiT2bMqTJVHcvR8LnHez1XSa\\\\\\\/ts6lGUOMTQKrbDtuuk1q2iYqEsjhHLTcRCeWdUr3WRx0HsGUiuQr8YjQUZo27xTjTSBOpkZfpGLeX4AUKcc8BBiLZC\\\\\\\/+7egK6EXg5phTFnQvIcedXriejt8mKLCR3rtTFDOZvtSz876uoHpfA6sX4ezpd6c8bjmJMbNIDExqOoNiXeMeZFU5x14SPO9KM+qti7fDBXbIGuz3UKI8eKo8Rfta3+n3MKj3KfoNHoG5Z7pfPu6FnujlaWH008xi3XDCS5JLhwiRednWSIbrd2S7PTsL6we01ZYYUlbhFaUqk3R24CaMctpb+WqyjsguWb5rvcDFwQhPM9WwPriqvfPQrsTjH47mBRZ6Q0YfC5b5OtUB0bs8ZwQYet0ZD+JSYUFm0OTFylcuBPp\\\\\\\/BLJki\\\\\\\/XZ47lzgoCEsUuKbni1VKmaij6QacVuOPPC1efiTd\\\\\\\/GuUNzBJEhcuTX2Eq6+w6kpby4T1JZe8s2hWZGW4dmJ\\\\\\\/piksKg9R2Coognpqtki5bsPJXURSB7xIX\\\\\\\/Q==\\\u0022,\\\u0022iv\\\u0022:\\\u0022V18h6gMxRr\\\\\\\/Uj6GxeRCSDQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022fab5dccc5482f7c31cf32dc755dc206695e398cd6ae2e28ce038db4b1513d589\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022KiukeFC9uujpCXwvyQP4XFE9PutDJX8YPtvF+kaZpPWYmCLfnTYzrT853ZcGSCIwdE\\\\\\\/4c8hdM3l4XEF9WN1+zshY9Qh4laWVZUQNbSOb+Pl7jfxswGaoOE88u7JZIY473hYBgM25nGoudoNhim6DQZq0yei18QfV2UEjgAVhCUesKqrplby9kbVmWrj65AwYYXUco5MMZ+AAyFQhr3bA6YSDREOt5o3m94MMGGNOQH1FU4oiG9\\\\\\\/6DofcIamca+OOckDJLeRpiCJBRo4ne1+eJMu43x+zPeQPVwVmZnxs2LpQ+58FoVxlwab6nYwmM5skuz7OuMlKJQfq5+1Kc7nv0929iQpvX\\\\\\\/FA6EB8BPVNdlGA2nhPHn9V5zICIKPAmacRK0HZbTY4Vn5lWC7zl8JcmD0cN6GmEAXeUQhiCcTKj08lZB0NOgcVQHL+MR2FuJgynlo0Ij5+yzWVR77Es3cA7vFVed0T8hm9rdNRXcS8rso+OA4MFhec7Xe732zXGCMQR5EKmgdjO+HNzMOdxUTHtZ5VWVMe6PakyfLvp6OXOfSvBON+LakLXcbEUV0L6Ipw0+xFGJqWgl0T3R\\\\\\\/UWC53WK9U0wsBD6BsFa28wlFUg8aD9fxWPyVl3BtgFAi2oeWuIUWcmmrrEifYY0d3nbn+H6001W5B\\\\\\\/se31QHQskrANurnJyps+omoVRuexcjHS8wd\\\\\\\/AFdJCF4q2nADz1oxbnb3zq9zu9yVJWWaQOjSvKc70Cq4X6c2T\\\\\\\/2yetkYp9UNuTo5ck\\\\\\\/1HclBD3v9EJcvpmScf591wOiYDltnMmU+u3JrWowamfJf8RMglJ3ExJp8aWRNdocSRuJ6Xkul\\\\\\\/G7LpJtcqqFfjQhkDIheIm0GMwdFl9KHniprYvkbZVHVW8y+jpvT5RngU1kegCNbrDA0StPqogIwD3mFEdML8J5DigxQKvJdFwPnKFGk6CO+Z6yx67YESW9C4icg8JB2NzQMp5LR1aHO8Gxoz0tnGbqXuoWPJM0iL5G3yaI+LHaZFclMJHD4+woJiO83Z6iIJuJLi9eOBVBgireqzq+Tps735W+az8OABKEXLp81SF5Ms0+6NVRdhbh4UvD9F6lQe2DUeJ9UsknOLvcz10OvurOVBKdY1rFZf3yIuoyWe2DmkyPDNEOUl\\\\\\\/9h\\\\\\\/OOo4sb2BNqWufL9fdldQAubYRFIK1MoRFX9ALuPX9\\\\\\\/BkwL3q\\\\\\\/qD\\\\\\\/Lv+jH2JHj369bsTn\\\\\\\/f\\\\\\\/tVke27bh6Mn4vUco1jad14AjJIGnHdbR3BeFMD3aT+2vHp4PcKMTl94tMhyehJHmtExTAKk3K\\\\\\\/WxSOF1DbW6lGCDfby82hlk0Ack3sK+ixgKnW6vE7r9AOUE1R1X6yWuK7qb\\\\\\\/kuZra1g\\\\\\\/myxyqZFJtU7J4gAb1NJwj1qPyy3AtSVm1SIFD4kofr4deB8Ha7uNke1U6xSIP3PLgmE0B4yW0GlqsvmBetfxpyAGLYBoksoUiyMP9VADrrOwTS1IirN5fdoUvrCE6URlkBT84LyVWsHPXylzC2zzBaK\\\\\\\/m5mdTSd+1HVD6P5vzKqjQWRWfc4twosl3K2NwSPJSCJQ89F9er87opzLu5FUL215KVUHhFlBbRP+8qn\\\\\\\/X3KnlfQVJuI52KKjBdAiy7YQXTUnRbx71e3Sw32\\\\\\\/\\\\\\\/JzOB\\\\\\\/4YB8Lo1D6oFyYsWwyRkCB2P6UxWI8eMGukTjhyjhjn62EpxNau5KG5AY9kvGyGiLtxOcqbBu1+oODkAeYHEnFdJFyjDdQ+qt6ojdVNWUOkf406ONMhhw410mK2+EdlVGnD9aqhXhL4QAifBO0pdfmzXkyTpVEZBJyZn9QxM+SFwz1GWrXqYKIzIS1Y+xvEAMzqTjBQqQ4Fegj9hCH6o7aUQW+oMErAy+NAnD2+QXXcyw6SuKAziNkTa7W3LaQ4x0VSl27niyOwVEGoUpbx2ek6lIiy9K0DCkmUZa9wfHnJwK7HUO69FUaf1NedCG7WtamMafHfrAOCP+gwsLeZtltENyAjNQi2s5EvQlyQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022gDkrtOf6l0xf\\\\\\\/rZSeOxvnA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225bbc3f44d7dc4c79b84def345831283a7bae7c4748765a11d661e9a5f21b3537\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460465 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/11/3443" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-yle9RQE6Yovs2eplT02di73WZZZybIWTtwoVqcB8ZnQ" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/11/3443" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-_MLmUXCnemRy4XVqj46T89ruZgxppXLGv54Ns5YX0hs" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/10/3431" title="RNAi-Based Suppressor Screens Reveal Genetic Interactions Between the CRL2LRR-1 E3-Ligase and the DNA Replication Machinery in Caenorhabditis elegans" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/11/3455" title="Proper Sterol Distribution Is Required for Candida albicans Hyphal Formation and Virulence" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460465" id="node460465" data-pisa="ggg;6/11/3443" data-pisa-master="ggg;g3.116.031286" data-apath="/ggg/6/11/3443.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Prediction of Single Crosses in the Early Stages of a Maize Hybrid Breeding Pipeline</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Dnyaneshwar C. Kadam</span>, <span class="highwire-citation-author" data-delta="1">Sarah M. Potts</span>, <span class="highwire-citation-author" data-delta="2">Martin O. Bohn</span>, <span class="highwire-citation-author" data-delta="3">Alexander E. Lipka</span> and <span class="highwire-citation-author" data-delta="4">Aaron J. Lorenz</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3443-3453;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.031286 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460465" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Dnyaneshwar C. Kadam </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Horticulture, University of Nebraska, Lincoln, Nebraska 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dnyaneshwar%2BC.%2BKadam%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kadam%20DC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADnyaneshwar%2BC.%2BKadam%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Sarah M. Potts </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Crop Sciences, University of Illinois at Urbana-Champaign, Urbana, Illinois 61801</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sarah%2BM.%2BPotts%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Potts%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASarah%2BM.%2BPotts%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Martin O. Bohn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Crop Sciences, University of Illinois at Urbana-Champaign, Urbana, Illinois 61801</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BO.%2BBohn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bohn%20MO&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMartin%2BO.%2BBohn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Alexander E. Lipka </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Crop Sciences, University of Illinois at Urbana-Champaign, Urbana, Illinois 61801</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexander%2BE.%2BLipka%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lipka%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlexander%2BE.%2BLipka%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Aaron J. Lorenz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Horticulture, University of Nebraska, Lincoln, Nebraska 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aaron%2BJ.%2BLorenz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lorenz%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAaron%2BJ.%2BLorenz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:lore0149@umn.edu" class="" data-icon-position="" data-hide-link-title="0">lore0149@umn.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/11/3443" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460465" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460465/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/11/3443.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460465" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460465/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/11/3443.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460465" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460465/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/11/3443.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460465" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460465/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/11/3443.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460465" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460465/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-corrections" >
  
      
  
  <div class="pane-content">
    <div class="messages highwire-corrections highwire-correction-type-correction"><h3 class="highwire-corrections-title">This article has a correction. Please see:</h3><ul class="highwire-corrections-list"><li class="relation-73067 first last"><a href="/content/7/10/3557" class="" data-icon-position="" data-hide-link-title="0">Corrigendum - October 01, 2017</a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Prediction of single-cross performance has been a major goal of plant breeders since the beginning of hybrid breeding. Recently, genomic prediction has shown to be a promising approach, but only limited studies have examined the accuracy of predicting single-cross performance. Moreover, no studies have examined the potential of predicting single crosses among random inbreds derived from a series of biparental families, which resembles the structure of germplasm comprising the initial stages of a hybrid maize breeding pipeline. The main objectives of this study were to evaluate the potential of genomic prediction for identifying superior single crosses early in the hybrid breeding pipeline and optimize its application. To accomplish these objectives, we designed and analyzed a novel population of single crosses representing the Iowa Stiff Stalk synthetic/non-Stiff Stalk heterotic pattern commonly used in the development of North American commercial maize hybrids. The performance of single crosses was predicted using parental combining ability and covariance among single crosses. Prediction accuracies were estimated using cross-validation and ranged from 0.28 to 0.77 for grain yield, 0.53 to 0.91 for plant height, and 0.49 to 0.94 for staygreen, depending on the number of tested parents of the single cross and genomic prediction method used. The genomic estimated general and specific combining abilities showed an advantage over genomic covariances among single crosses when one or both parents of the single cross were untested. Overall, our results suggest that genomic prediction of single crosses in the early stages of a hybrid breeding pipeline holds great potential to redesign hybrid breeding and increase its efficiency.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/hybrid-breeding" class="hw-term hw-article-keyword hw-article-keyword-hybrid-breeding" rel="nofollow">hybrid breeding</a></li><li class="kwd"><a href="/keyword/genomic-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-prediction" rel="nofollow">genomic prediction</a></li><li class="kwd"><a href="/keyword/genotyping-sequencing-0" class="hw-term hw-article-keyword hw-article-keyword-genotyping-by-sequencing" rel="nofollow">genotyping by sequencing</a></li><li class="kwd"><a href="/keyword/general-combining-ability" class="hw-term hw-article-keyword hw-article-keyword-general-combining-ability" rel="nofollow">general combining ability</a></li><li class="kwd"><a href="/keyword/specific-combining-ability" class="hw-term hw-article-keyword hw-article-keyword-specific-combining-ability" rel="nofollow">specific combining ability</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared Data Resources</a></li><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">Genomic Selection</a></li></ul><p id="p-3">Contemporary hybrid breeding programs are based on the pure-line method of corn breeding proposed by <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Shull (1909)</a>. This method includes the development of inbreds by self-pollination, followed by evaluation of selected inbreds for single-cross performance when crossed to other inbreds. A major challenge with this method is achieving adequate testing of the inbreds to evaluate performance in single-cross combinations (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hallauer <em>et al.</em> 1988</a>). In maize, heterotic groups are well defined, and single crosses are almost exclusively made between heterotic groups. The fullest assessment of single-cross performance in maize, therefore, would be a complete factorial mating design achieved by making all between-heterotic groups single crosses. This would provide complete information on both general combining ability (GCA) and specific combining ability (SCA) (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Comstock and Robinson 1948</a>). However, a full factorial design among inbreds can be cost prohibitive, as advanced hybrid breeding programs typically have many inbreds to evaluate, making the number of all possible single crosses extremely large. For this reason, predicting single-cross performance has always been a major issue for hybrid breeding programs (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Schrag <em>et al.</em> 2009</a>).</p><p id="p-4">Several approaches have been used to evaluate the genetic merit of inbreds for single-cross performance with variable success. These approaches include inbred <em>per se</em> performance, performance when crossed to testers (topcross test), best linear unbiased prediction (BLUP) using pedigrees, and molecular marker-assisted prediction. Many of these approaches have been reviewed in detail elsewhere (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Schrag <em>et al.</em> 2009</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Smith and Betrán 2004</a>). <em>Per se</em> performance of inbreds is typically found to be a very poor predictor of single-cross performance, especially for traits such as grain yield (GY), where strong dominance effects underlie the genetic variance (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hallauer 1977</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Love and Wentz 1914</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Smith 1986</a>). A topcross test is an established and simple approach to assess the genetic worth of inbreds in single-cross combinations (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Jenkins and Brunson 1932</a>). However, topcross evaluation of a large number of inbreds is difficult (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Albrecht <em>et al.</em> 2011</a>), and selections based on single-cross performances are carried out in later stages, increasing the time required for commercial hybrid development. <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bernardo (1996a)</a> showed that pedigree-based BLUP is useful for prediction of untested single crosses. He used pedigree-based covariance matrices among tested and untested single crosses to obtain BLUPs for untested single crosses. The correlations between observed and predicted performance were moderate (0.43–0.76) for single crosses where both parents were tested in single-cross combinations. However, correlations were severely decreased when one or both of parents were untested (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bernardo 1996b</a>).</p><p id="p-5">The relationship between genetic distance (GD) of parental inbreds, measured by molecular markers, and heterosis has been extensively studied in maize. While it is possible to predict single-cross performance using marker-based GD for hybrid sets composed of both intra- and inter-heterotic group single crosses, correlations for predicting interheterotic group single crosses only were reported to be very low (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Lee <em>et al.</em> 2007</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Melchinger 1999</a>). Two possible causes of these low prediction accuracies include (1) loose association between heterotic quantitative trait loci (QTL) and the molecular markers used to estimate GD, and (2) opposite linkage phases between the QTL and marker alleles across heterotic groups (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bernardo 1992</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Charcosset <em>et al.</em> 1991</a>). Commercial hybrids consist of only interheterotic group single crosses, making them the only type relevant for prediction in breeding programs. In a modified approach, prediction of single-cross performance and SCA based on only significant markers was suggested (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Vuylsteke <em>et al.</em> 2000</a>), but this approach was found to be inferior to an established GCA method. Also, extending the GCA predictions with SCA estimates from associated markers did not improve the prediction accuracy (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Schrag <em>et al.</em> 2006</a>, <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">2007</a>).</p><p id="p-6">Genomic prediction is an approach that uses markers to predict the genetic value of complex traits in progeny for selection and breeding (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Meuwissen <em>et al.</em> 2001</a>). When genomic predictions are used to make selections, it is referred to as genomic selection (GS). The primary difference between GS and traditional forms of marker-assisted selection (MAS) is the simultaneous use of a large number of markers distributed genome-wide, as opposed to a small set of markers linked to QTL (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Heffner <em>et al.</em> 2009</a>). Implementation of genomic prediction and selection requires the development of training (calibration) sets consisting of individuals that have been both phenotyped and genotyped, followed by model calibration. A whole suite of genomic prediction models have been developed, each deploying different strategies to estimate genome-wide marker effects or model genomic relationships between individuals (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">de los Campos <em>et al.</em> 2013</a>).</p><p id="p-7">Recent results from simulation and experimental studies have indicated the usefulness of genomic prediction models to predict hybrid performance in maize (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Albrecht <em>et al.</em> 2011</a>, <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2014</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Jacobson <em>et al.</em> 2014</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Massman <em>et al.</em> 2013</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Riedelsheimer <em>et al.</em> 2012</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Technow <em>et al.</em> 2012</a>, <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">2014</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Windhausen <em>et al.</em> 2012</a>). However, most of the experimental studies were focused on prediction of topcross performance under a single tester scenario (<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Albrecht <em>et al.</em> 2011</a>, <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">2014</a>; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Jacobson <em>et al.</em> 2014</a>; <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Riedelsheimer <em>et al.</em> 2012</a>; <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Windhausen <em>et al.</em> 2012</a>). Experimental studies on genomic prediction of single-cross performance have been based on historical data consisting of established inbred parents with mixed and complex ancestry (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Massman <em>et al.</em> 2013</a>; <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Technow <em>et al.</em> 2014</a>). These studies used covariances among tested and untested single crosses estimated from realized genomic relationship matrices to predict the performance of untested single crosses. The prediction accuracies were high, often exceeding 0.75, even when both parents of the single cross were untested.</p><p id="p-8">Identification of superior single crosses early in the hybrid breeding pipeline would be beneficial to developing commercial hybrids more quickly. The current practice of initial selection among available inbreds based on their topcross performance, followed by evaluation of single crosses made among selected inbreds, increases time required for commercial hybrid development. Moreover, this approach does not allow evaluation of all possible single-cross combinations among available inbreds. It is important, therefore, to study the potential of genomic prediction of early-stage single crosses (<em>i.e.</em>, single crosses between sets of random inbreds derived from each heterotic group).</p><p id="p-9">With this in mind, the main objective of this study was to evaluate the potential of genomic prediction for identifying superior single crosses early in the breeding pipeline. Also, the effect of model and training set composition on single-cross prediction accuracy was evaluated. To accomplish these objectives, we designed and analyzed a novel population of single crosses. The parental recombinant inbred lines (RILs) and doubled haploid lines (DHLs) were randomly selected from three Iowa Stiff Stalk synthetic (SSS) and three non-Stiff Stalk synthetic (NSS) biparental populations. All single crosses, therefore, represented the SSS/NSS heterotic pattern commonly used in the development of North American commercial maize hybrids. All RILs and DHLs were genotyped using genotyping by sequencing (GBS) (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Elshire <em>et al.</em> 2011</a>); an affordable genotyping option which is critical to the routine use of these methods in a breeding program.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Germplasm</h3><p id="p-10">Three SSS inbred parents (PHG39, PHJ40, and B73) and three NSS inbred parents (LH82, PHG47, and PHG84) were used to create six biparental families by making each of the three possible crosses between the three SSS inbreds, and also between the three NSS inbreds (Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031286/-/DC1/FigureS1.tiff">Figure S1</a>). The parents were selected to be both genetically diverse and superior in GCA for GY under high planting density (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Mansfield and Mumm 2014</a>). A total of 217 lines were developed from crosses between these parents. Approximately 10% of these lines were RILs and 90% were DHLs. RILs and DHLs will hereafter be referred to collectively as “inbred progenies.” The number of inbred progenies in each of the six biparental families ranged from two to 69 (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Random crosses among the inbred progenies were made between heterotic groups to produce 312 single crosses (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). Single crosses representing each biparental family were balanced to the extent possible while maximizing the number of inbred progenies used in the single crosses. Completely balanced representation was not achieved due to seed limitations and comparatively fewer inbred progenies available for certain biparental families. Single crosses were grouped into nine single-cross families, which we define as a group of single crosses created using inbred progenies from the same biparental family on each side of the heterotic pattern (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). For example, a single cross with pedigree (PHJ40 × PHG39)DH-1/(PHG47 × PHG84)DH-1 belongs to the same single-cross family as a single cross with pedigree (PHJ40 × PHG39)DH-2/(PHG47 × PHG84)DH-2. The mean number of times an individual SSS inbred progeny was used in a cross was 6.9. The mean number of times an individual NSS inbred progeny was used in a cross was 1.8. Number of single crosses per single-cross family ranged from 19 to 51 (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469717/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469717/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Family designations of nine single-cross families and number of single crosses belonging to each of the nine families</span><div class="sb-div caption-clear"></div></div></div><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/11/3443/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Crossing scheme between RILs or DHLs derived from three biparental families representing the SSS (y-axis) and NSS (x-axis) heterotic groups. Colored boxes indicate the presence while unfilled boxes indicate absence of a particular single cross. Bold lines delineate single-cross families." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-934584868" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Crossing scheme between RILs or DHLs derived from three biparental families representing the SSS (&lt;em&gt;y&lt;/em&gt;-axis) and NSS (&lt;em&gt;x&lt;/em&gt;-axis) heterotic groups. Colored boxes indicate the presence while unfilled boxes indicate absence of a particular single cross. Bold lines delineate single-cross families.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/11/3443/F1.medium.gif" width="440" height="174"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/11/3443/F1.medium.gif" width="440" height="174"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/11/3443/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/11/3443/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469647" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-12" class="first-child">Crossing scheme between RILs or DHLs derived from three biparental families representing the SSS (<em>y</em>-axis) and NSS (<em>x</em>-axis) heterotic groups. Colored boxes indicate the presence while unfilled boxes indicate absence of a particular single cross. Bold lines delineate single-cross families.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>Field experiments</h3><p id="p-13">The 312 single crosses were evaluated at two locations in 2012 and three locations in 2013. Two locations were common between years. The locations were as follows: South Farms (Urbana, IL; 2012 and 2013), Maxwell Farms (Urbana, IL; 2012 and 2013), and Monmouth (IL; 2013 only). The five location–year combinations were defined as separate environments. The experimental design was an α(0, 1)-incomplete block design (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Patterson and Williams 1976</a>) with three replications at each environment. All trials were planted with an Almaco Seed Pro 360 planter set at 0.64 m row spacing and 4.46 m long row. Entries were grown in small plots consisting of two rows. Plots were overplanted by 15% to compensate for germination failure and later thinned to the target plant density of 116,000 plants/ha<sup>–1</sup>. All fields were controlled for weeds. Nitrogen (N) was applied before planting as 28% urea-ammonium nitrate at a rate of 336.4 kg/ha to all fields. Phosphorous and potassium were each applied at 112 kg/ha according to recommended levels determined by soil tests performed by the University of Illinois Crop Science Research and Education Center. Stand counts were recorded and plots with planting densities lower than 106,000 plants/ha<sup>–1</sup> discarded. Additionally, issues with seed production resulted in fewer single crosses being planted at all locations in 2013 (South Farms, 260; Maxwell Farms, 259; and Monmouth, 258). Plots were machine harvested and data were recorded for several agronomic traits. For this study, data on GY, plant height (PH), and staygreen (SG) were used for downstream analyses. GY was converted to Mt/ha on a 155 g/kg moisture basis. PH was measured postanthesis on a single representative plant determined by visually surveying the entire plot before measurement. SG was evaluated visually as a percentage of total dry down, where a rating of 1 represented complete senescence and a rating of 10 represented fully green leaves.</p></div><div id="sec-4" class="subsection"><h3>GBS</h3><p id="p-14">Five plants of each inbred progeny were germinated. A total of 0.1 g of tissue was sampled from leaf tips and pooled across the five plants. DNA was extracted using the Qiagen DNeasy Plant 96 kit following the DNeasy Plant Handbook. DNA samples were sent to the Institute for Genomic Diversity at Cornell University for GBS, where library construction and sequencing was performed as described by <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Elshire <em>et al.</em> (2011)</a>. Single nucleotide polymorphisms (SNPs) were scored from the raw sequence data using the TASSEL GBS Pipeline version 3.0 (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Glaubitz <em>et al.</em> 2014</a>). SNPs with &gt;20% missing values and &lt;5% minor allele frequency were removed from the dataset. Heterozygotes were treated as missing data. Missing data were replaced by the mean coded value for the marker (<em>i.e.</em>, naïve imputation). Of the markers remaining after filtration, markers that were polymorphic among both SSS and NSS inbred progenies were retained for analysis. The final marker data set consisted of 2296 high-quality SNPs. The distribution of SNPs among the 10 chromosomes of the B73 reference genome is displayed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031286/-/DC1/FigureS2.tiff">Figure S2</a>.</p></div><div id="sec-5" class="subsection"><h3>Phenotypic data analysis</h3><p id="p-15">The phenotypic data were unbalanced due to missing observations. We used the following statistical model for the analysis of the data across the five environments:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-1.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-2.gif"/></noscript></span></span> is the phenotypic observation for <em>i</em><sup>th</sup> single cross evaluated in the <em>k</em><sup>th</sup> environment in the <em>l</em><sup>th</sup> complete block (<em>i.e.</em>, replicate) and <em>q</em><sup>th</sup> incomplete block. The effects in the model are as follows: <em>μ</em> is the grand mean; <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-3.gif"/></noscript></span></span> represents effect of the <em>i</em><sup>th</sup> single cross; <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-4.gif"/></noscript></span></span> represents the effect of the <em>k</em><sup>th</sup> environment; <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-5.gif"/></noscript></span></span> represents the interaction effect between single cross and environment; <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-6.gif"/></noscript></span></span> represents the effect of the <em>l</em><sup>th</sup> complete block nested within the <em>k</em><sup>th</sup> environment; <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-7.gif"/></noscript></span></span> represents the effect of the <em>q</em><sup>th</sup> incomplete block nested within the <em>l</em><sup>th</sup> complete block in the <em>k</em><sup>th</sup> environment; and <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-8.gif"/></noscript></span></span> represents the residual. Environment and replication nested within environment effects were modeled as fixed effects while all other effects were treated as random. In model (1), <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-9.gif"/></noscript></span></span> Error and block variances were allowed to be heterogeneous among environments.</p><p id="p-16">The above model was implemented using ASReml-R software (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Butler <em>et al.</em> 2009</a>) to obtain restricted maximum likelihood estimates of all variance components and solve the mixed linear model equations. Significance of the variance components was determined using likelihood ratio tests at 0.001 level of significance. The entry-mean heritability of each trait was computed according to <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Holland <em>et al.</em> (2003)</a> as: <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-10.gif"/></noscript></span></span> where, <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-11.gif"/></noscript></span></span> represents the variance among single crosses, <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-12.gif"/></noscript></span></span> represents the variance of interaction effects of single crosses with environments, <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-13.gif"/></noscript></span></span> is the residual variance, <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-14.gif"/></noscript></span></span> is the harmonic mean of number of observations per single cross within an environment, and <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-15.gif"/></noscript></span></span> is the harmonic mean of total number of observations per single cross. Similarly, model (1) was used to estimate the genetic variance and broad-sense heritability for an individual single-cross family. Finally, we calculated BLUPs of single crosses using model (1), which were used as validation to estimate genomic prediction accuracy in downstream analyses.</p></div><div id="sec-6" class="subsection"><h3>Single-cross prediction methods</h3><p id="p-17">The linear model used for single-cross performance was as follows:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-16.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>where <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-17.gif"/></noscript></span></span> is the phenotypic observation on a single cross between the <em>i</em><sup>th</sup> and <em>j</em><sup>th</sup> inbred progeny, evaluated in the <em>k</em><sup>th</sup> environment in the <em>l</em><sup>th</sup> complete block and <em>q</em><sup>th</sup> incomplete block. The effects in the model are as follows: <em>μ</em> is the grand mean; <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-18.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-19.gif"/></noscript></span></span> represents the GCA effects of the females (SSS inbred progenies) and males (NSS inbred progenies), respectively; <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-20.gif"/></noscript></span></span> represents the SCA effect of the single cross; and <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-21.gif"/></noscript></span></span> represent the interaction effects of respective terms with the <em>k</em><sup>th</sup> environment. The remaining terms were as described in model (1).</p><p id="p-18">The random effect vectors <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-22.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-23.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-24.gif"/></noscript></span></span> were assumed to have the following multivariate normal (MVN) distributions: <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-25.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-26.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-27.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-28.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-29.gif"/></noscript></span></span> were additive genomic relationship matrices of females and males, calculated as simple marker similarity coefficients. The dominance relationship matrix, <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-30.gif"/></noscript></span></span> was computed according to <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bernardo (2002)</a>, using the corresponding elements from matrices <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-31.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-32.gif"/></noscript></span></span> Model (2) was implemented using ASReml-R (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Butler <em>et al.</em> 2009</a>).</p><p id="p-19">We evaluated four methods to predict single-cross performance using model (2). Broadly, these methods can be grouped into two categories: (1) parent GCA and SCA effects, and (2) additive and dominance covariances among single crosses.</p><div id="sec-7" class="subsection"><h4>Method 1a: Parent GCA:</h4><p id="p-20">Performance of untested single crosses <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-33.gif"/></noscript></span></span> was predicted from the GCA of the corresponding parents, <em>i</em> and <em>j</em>, estimated from model (2) as:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-34.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>GCA of females and/or males with no performance data were estimated from related inbred progenies using information from relatives through <em>G<sub>m</sub></em> and/or <em>G<sub>f</sub></em> in model (2).</p></div><div id="sec-8" class="subsection"><h4>Method 1b: Parent GCA and single-cross SCA:</h4><p id="p-21">Performance of untested single crosses <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-35.gif"/></noscript></span></span> was predicted using the sum of the parent GCA and SCA of the single crosses as:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-36.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>As for the GCA effects, the SCA effects for untested single crosses were estimated using the dominance genomic relationship matrix in model (2).</p></div><div id="sec-9" class="subsection"><h4>Method 2a: Additive covariance among single crosses:</h4><p id="p-22">The performance of untested single crosses <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-37.gif"/></noscript></span></span> was predicted based on the covariance among tested and untested single crosses as:<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-38.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>where <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-39.gif"/></noscript></span></span> is the genetic covariance matrix of untested and tested single crosses, <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-40.gif"/></noscript></span></span> is the phenotypic covariance matrix of the tested single crosses, and <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-41.gif"/></noscript></span></span> is a vector of tested single-cross BLUPs obtained from model (1). The elements of <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-42.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-43.gif"/></noscript></span></span> were computed according to <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Bernardo (2002)</a>, using the genomic relationship matrices <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-44.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-45.gif"/></noscript></span></span> Briefly, let <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-46.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-47.gif"/></noscript></span></span> denote any two female (F) inbred progenies and <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-48.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-49.gif"/></noscript></span></span> any two male (M) inbred progenies. For a given pair of single crosses, (<span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-50.gif"/></noscript></span></span> × <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-51.gif"/></noscript></span></span>) and (<span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-52.gif"/></noscript></span></span> × <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-53.gif"/></noscript></span></span>), the elements of <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-54.gif"/></noscript></span></span> and the off-diagonal elements of <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-55.gif"/></noscript></span></span> were calculated as <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-56.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-57.gif"/></noscript></span></span> + <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-58.gif"/></noscript></span></span> The diagonal elements of <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-59.gif"/></noscript></span></span> were estimated as <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-60.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-61.gif"/></noscript></span></span> + <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-62.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-63.gif"/></noscript></span></span> was equal to <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-64.gif"/></noscript></span></span> divided by the total number of observations for single cross <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-65.gif"/></noscript></span></span> The estimates of <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-66.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-67.gif"/></noscript></span></span> were obtained from model (2).</p></div><div id="sec-10" class="subsection"><h4>Method 2b: Additive and dominance covariance among single crosses:</h4><p id="p-23">The method described in 2a was extended by including dominance covariance among the tested and untested single crosses. Specifically, the elements of <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-68.gif"/></noscript></span></span> and off diagonal elements of <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-69.gif"/></noscript></span></span> were computed as <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-70.gif"/></noscript></span></span> + <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-71.gif"/></noscript></span></span> The diagonal elements of <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-72.gif"/></noscript></span></span> were estimated as <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-73.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-74.gif"/></noscript></span></span> + <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-75.gif"/></noscript></span></span> The estimates of <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-76.gif"/></noscript></span></span> was obtained from model (2).</p></div></div><div id="sec-11" class="subsection"><h3>Cross-validation and prediction accuracy estimation</h3><p id="p-24">Accuracy of single-cross prediction was evaluated using leave-one-out cross-validation (LOOCV). LOOCV is a particular case of <em>k</em>-fold cross-validation with <em>k</em> = <em>n</em>. We chose LOOCV because the greater number of folds minimizes bias in the estimator (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Kohavi 1995</a>). Five different LOOCV scenarios involving varying degrees of relationship between training and validation set single crosses were considered (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). The cross-validation scenarios were as follows: (1) T2, both parents of a single cross contained in the validation set were tested; (2) T1F, only the female parent of a single cross contained in the validation set was tested; (3) T1M, only the male parent of a single cross contained in the validation set was tested; (4) T0, neither of the parents of a single cross contained in the validation set was tested; and (5) novel single-cross family, all single crosses belonging to one single-cross family were removed from the training set and thus formed the validation set. The conventional LOOCV was slightly modified to maintain constant training set size for each of the five cross-validation scenarios considered. The common maximum possible training set size across the five scenarios was 261. With this in mind, we decided to set the training set size to 250 for all five cross-validation scenarios in order to remove the confounding effect of population size. For the first four scenarios, the cross-validation is repeated such that each of the 312 single crosses was placed into the validation set exactly one time (<em>i.e.</em>, leave-one-individual-out cross-validation). For each of the 312 rounds, a random sample of 250 single crosses from the remaining single crosses was drawn without replacement and formed the training set. This was repeated 30 times to allow for sufficient resampling of the training set for a total of 9360 (30 × 312) resampled training sets. For each of the 30 repetitions, the predictions were integrated into a single vector and correlated with the phenotypic observations, as described below. For scenario 5, the cross-validation was repeated so that each of the nine single-cross families was entered into the validation set one time (<em>i.e.</em>, leave-one-family-out cross-validation). This was repeated 30 times by resampling 250 single crosses without replacement from the training set. The prediction accuracy, however, was evaluated only for the six largest families because size of the three families (f7, f8, and f9) was too small to accurately estimate correlation coefficients (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/11/3443/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Schematic visualization of T2, T1F, T1M, and T0 cross-validation scenarios. Each small square represents one single cross. Completely filled squares (T2) indicate that both male and female parents of a single cross contained in the validation set were tested, half-filled squares indicate either the female (T1F) or male parent (T1M) of single cross contained in the validation set was tested, and unfilled squares (T0) indicate that neither parent of a single cross contained in the validation set was tested." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-934584868" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Schematic visualization of T2, T1F, T1M, and T0 cross-validation scenarios. Each small square represents one single cross. Completely filled squares (T2) indicate that both male and female parents of a single cross contained in the validation set were tested, half-filled squares indicate either the female (T1F) or male parent (T1M) of single cross contained in the validation set was tested, and unfilled squares (T0) indicate that neither parent of a single cross contained in the validation set was tested.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/11/3443/F2.medium.gif" width="440" height="335"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/11/3443/F2.medium.gif" width="440" height="335"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/11/3443/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/11/3443/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469583" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-25" class="first-child">Schematic visualization of T2, T1F, T1M, and T0 cross-validation scenarios. Each small square represents one single cross. Completely filled squares (T2) indicate that both male and female parents of a single cross contained in the validation set were tested, half-filled squares indicate either the female (T1F) or male parent (T1M) of single cross contained in the validation set was tested, and unfilled squares (T0) indicate that neither parent of a single cross contained in the validation set was tested.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">The single-cross BLUPs returned from model (1) were treated as the observed single-cross performance and used for validation. Prediction accuracy was expressed as the Pearson’s correlation coefficient between the observed and predicted single-cross performance divided by the square root of the broad-sense heritability on an entry-mean basis (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Dekkers 2007</a>). The mean prediction accuracy across the 30 repetitions was reported. SEs of the prediction accuracy were calculated using the bootstrap method implemented in the <em>R</em> package, <em>boot</em> (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Canty 2016</a>). For each of the 30 repetitions, the predicted and observed values were resampled with replacement 200 times and the resulting distribution of 200 correlation coefficient estimates was used to estimate the bootstrap SE. The mean SE across the 30 repetitions was reported.</p></div><div id="sec-12" class="subsection"><h3>Data availability</h3><p id="p-27"><a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031286/-/DC1/FileS1.csv">File S1</a> contains phenotypic data. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031286/-/DC1/FileS2.csv">File S2</a> contains BLUPs of single crosses. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031286/-/DC1/FileS3.csv">File S3</a> contains data for 2296 SNPs scored on SSS inbred progenies. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031286/-/DC1/FileS4.csv">File S4</a> contains data for 2296 SNPs scored on NSS inbred progenies. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031286/-/DC1/FileS5.csv">File S5</a> contains an additive genomic relationship matrix for SSS inbred progenies. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031286/-/DC1/FileS6.csv">File S6</a> contains an additive genomic relationship matrix for NSS inbred progenies. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031286/-/DC1/FileS7.csv">File S7</a> contains a dominance genomic relationship matrix for the single crosses.</p></div></div><div class="section results" id="sec-13"><h2 class="">Results</h2><div id="sec-14" class="subsection"><h3>Variance components and broad-sense heritability</h3><p id="p-28">Variance among single crosses <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-77.gif"/></noscript></span></span> was significantly different from zero <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-78.gif"/></noscript></span></span> in the whole population as well as within individual single-cross families for all three traits (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). For GY, the entry-mean heritability was 0.58 across the whole population of single crosses, and it ranged from 0.53 to 0.83 within the individual single-cross families. Similarly, for PH and SG, the entry-mean heritability was 0.89 and 0.81 in the whole population, respectively, and ranged from 0.88 to 0.91 and 0.67 to 0.80 within individual single-cross families, respectively. The sum of parent <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-79.gif"/></noscript></span></span> was greater than <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-80.gif"/></noscript></span></span> for all traits. The proportion of <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-81.gif"/></noscript></span></span> was highest for GY, followed by PH and SG (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469586/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469586/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Mean, range, genetic variance, and broad-sense heritability estimates in whole population as well as individual single-cross families for grain yield (GY; Mt/ha), plant height (PH; cm), and staygreen (SG; 1–10 rating)</span><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469563/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469563/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <p id="p-29" class="first-child">General combining ability variance of stiff stalk synthetic (<span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/T3/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/T3/embed/mml-math-88.gif"/></noscript></span></span>) and non-stiff stalk (<span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/T3/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/T3/embed/mml-math-89.gif"/></noscript></span></span>) inbred progenies and specific combining ability variance (<span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/T3/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/T3/embed/mml-math-90.gif"/></noscript></span></span>) of single crosses between them.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Prediction accuracy for T2, T1F, T1M, and T0 scenarios</h3><p id="p-31">We first evaluated the prediction accuracy for T2, T1F, T1M, and T0 scenarios in the whole population using leave-one-individual-out cross-validation. Higher prediction accuracies were observed for SG and PH compared with GY for all scenarios (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). Prediction accuracies were highest for T2, followed by T1F, T1M, and T0. The four methods were similar in accuracy when applied to the T2 and T1F scenarios. However, methods 1a and 1b were mostly better than methods 2a and 2b for predicting single-cross performance in the T1M and T0 scenarios. Modeling SCA led to small increases in prediction accuracy for GY and PH, with a maximum increase in the T0 scenario (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031286/-/DC1/TableS1.docx">Table S1</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/11/3443/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Prediction accuracy for T2, T1F, T1M and T0 cross-validation scenarios for traits grain yield (GY), plant height (PH) and staygreen (SG) obtained using the four methods 1a (Parent GCA), 1b (Parent GCA plus single-cross SCA), 2a (Additive genetic covariance among single crosses) and 2b (Additive plus dominance covariance among single crosses) as evaluated with training set of 250 and leave-one-individual-out cross-validation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-934584868" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Prediction accuracy for T2, T1F, T1M and T0 cross-validation scenarios for traits grain yield (GY), plant height (PH) and staygreen (SG) obtained using the four methods 1a (Parent GCA), 1b (Parent GCA plus single-cross SCA), 2a (Additive genetic covariance among single crosses) and 2b (Additive plus dominance covariance among single crosses) as evaluated with training set of 250 and leave-one-individual-out cross-validation.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/11/3443/F3.medium.gif" width="440" height="173"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/11/3443/F3.medium.gif" width="440" height="173"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/11/3443/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/11/3443/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469507" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-32" class="first-child">Prediction accuracy for T2, T1F, T1M and T0 cross-validation scenarios for traits grain yield (GY), plant height (PH) and staygreen (SG) obtained using the four methods 1a (Parent GCA), 1b (Parent GCA plus single-cross SCA), 2a (Additive genetic covariance among single crosses) and 2b (Additive plus dominance covariance among single crosses) as evaluated with training set of 250 and leave-one-individual-out cross-validation.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>Prediction accuracy for novel single-cross family</h3><p id="p-33">We next investigated the potential to predict the performance of single crosses in a new single-cross family, using the phenotypic and genotypic information on the single crosses from related single-cross families (leave-one-family-out cross-validation). When eight of the families were used as a training set to predict performances of single crosses from the remaining family, prediction accuracies were generally moderate for GY and high for PH and SG (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). The mean accuracies of methods 1a and 1b for prediction of novel single-cross families were 0.67 and 0.62 for GY, 0.85 and 0.76 for PH, and 0.78 and 0.78 for SG, respectively (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031286/-/DC1/TableS2.docx">Table S2</a>). Variation in prediction accuracy across families was observed, especially for GY. We also evaluated the effect of adding single crosses from the family being predicted to the training set by comparing prediction accuracy of individual family with leave-one-individual-out and leave-one-family-out cross-validations. The goal of this analysis was to measure the benefit of including information from the same single-cross family to accurately separate single crosses within the same family. Although the prediction accuracies were increased slightly for some families, they were decreased for other families (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). The mean prediction accuracies of methods 1a and 1b were 0.60 and 0.61 for GY, 0.84 and 0.85 for PH, and 0.79 and 0.78 for SG, respectively. Adding individuals from the family being predicted benefited method 1b more than method 1a (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031286/-/DC1/TableS2.docx">Table S2</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/11/3443/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Mean prediction accuracy and SE of methods 1a (orange) and 1b (blue) in predicting performance of novel single-cross families. Two cross-validation schemes were used: leave-one-family out (bottom panel) and leave-one-individual out (top panel). Traits analyzed were grain yield (GY), plant height (PH), and staygreen (SG). SEs were estimated using the bootstrap method." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-934584868" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Mean prediction accuracy and SE of methods 1a (orange) and 1b (blue) in predicting performance of novel single-cross families. Two cross-validation schemes were used: leave-one-family out (bottom panel) and leave-one-individual out (top panel). Traits analyzed were grain yield (GY), plant height (PH), and staygreen (SG). SEs were estimated using the bootstrap method.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/11/3443/F4.medium.gif" width="440" height="208"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/11/3443/F4.medium.gif" width="440" height="208"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/11/3443/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/11/3443/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469576" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-34" class="first-child">Mean prediction accuracy and SE of methods 1a (orange) and 1b (blue) in predicting performance of novel single-cross families. Two cross-validation schemes were used: leave-one-family out (bottom panel) and leave-one-individual out (top panel). Traits analyzed were grain yield (GY), plant height (PH), and staygreen (SG). SEs were estimated using the bootstrap method.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h3>Genomic predictions of GY of all possible single crosses</h3><p id="p-35">Genomic predictions were calculated for all possible 7866 single crosses between 46 SSS and 171 NSS inbred progenies based on the prediction model, including parent GCA and cross SCA effects (<em>i.e.</em>, method 1b). The genomic predictions for GY ranged from 7.5 to 9.5 Mt/ha. The top 100 single crosses based on genomic predictions included only one single cross that was actually made and tested; the remaining 99 single crosses were never made. Moreover, &gt;50 untested single-cross combinations surpassed the highest genomic prediction of any tested single cross (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/11/3443/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of genomic predictions for grain yield (GY) for all 7866 possible single crosses between the 46 SSS inbred progenies and 171 NSS inbred progenies." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-934584868" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Distribution of genomic predictions for grain yield (GY) for all 7866 possible single crosses between the 46 SSS inbred progenies and 171 NSS inbred progenies.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/11/3443/F5.medium.gif" width="440" height="334"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/11/3443/F5.medium.gif" width="440" height="334"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/11/3443/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/11/3443/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469584" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-36" class="first-child">Distribution of genomic predictions for grain yield (GY) for all 7866 possible single crosses between the 46 SSS inbred progenies and 171 NSS inbred progenies.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-18"><h2 class="">Discussion</h2><p id="p-37">Typical hybrid maize breeding programs involve the creation of large biparental families for topcrossing to elite testers early in the breeding pipeline. Early-stage selections are performed on the basis of topcross performance with a single elite tester, which is the sum of the candidate line GCA effect and any SCA effect between the candidate line and tester. While this is a very convenient and routine method, it has long been recognized that it would be ideal to test all combinations of possible parents immediately in the hybrid breeding pipeline (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Fehr 1987</a>). There are two main advantages of early evaluation of all potential single crosses. First, it could identify the best parental combination immediately after progeny development. Selection of inbred progenies only on the basis of topcross evaluation leaves open the possibility that some unique parental combinations never made and tested could be superior in performance and become commercial hybrids. Second, early evaluation based on single-cross performance could enable commercialization of hybrids in less time by essentially skipping the topcross stages. Despite these advantages, field testing of all potential single crosses of inbred progenies is completely impractical for a mature hybrid maize breeding program.</p><p id="p-38">Advances in genotyping technology, such as GBS, have made it practical to genotype all parental candidate lines with dense, genome-wide markers (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">He <em>et al.</em> 2014</a>). Genomic prediction models can predict the performance of all possible single-cross combinations, allowing <em>in silico</em> evaluation of all parental combinations, as in the ideal scenario described above. In the present study, GBS and yield trial data were used to build genomic prediction models for predicting single-cross performance. The accuracy of predicting single crosses estimated using cross-validation ranged from 0.28 to 0.77 for GY, 0.53 to 0.91 for PH, and 0.49 to 0.94 for SG, depending on the model and cross-validation scenario (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031286/-/DC1/TableS2.docx">Table S2</a>). These prediction accuracies were 37–97, 56–96, and 54–100% of the accuracy of phenotypes <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-95.gif"/></noscript></span></span> for GY, PH, and SG, respectively. Therefore, the prediction accuracies of single-cross performance achieved in this study indicate that this approach holds great potential for increasing the efficiency of a hybrid breeding program by enabling the effective evaluation of all single-cross combinations.</p><div id="sec-19" class="subsection"><h3>Prediction accuracy for T2, T1F, T1M, and T0 single crosses</h3><p id="p-39">In order to understand the effect of parent testing on the accuracy of single cross predictions, we evaluated the accuracies of prediction of single crosses having both (T2), either female (T1F) or male (T1M), or no (T0) parent tested for single-cross performance. Observed differences in prediction accuracies between these scenarios were considerable, with the highest prediction accuracy for T2 single crosses followed by T1 (T1F or T1M) and T0 single crosses. The T0 scenario was the most difficult to predict. Similar trends have been observed using simulations (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Technow <em>et al.</em> 2012</a>), as well as experimental studies based on historical data in maize (<a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Massman <em>et al.</em> 2013</a>; <a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">Technow <em>et al.</em> 2014</a>) and wheat (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Zhao <em>et al.</em> 2015</a>). This finding can be explained by the representation of parents among a differing number of single-cross combinations in the training set. The information shared between the single cross being predicted and the training set increases as the number of times the parents are tested in different single cross combinations. As a result, the GCA and SCA effects are estimated with high accuracy, as indicated by decreasing SEs along with increased representation of single-cross parents in the training set (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). In the T2 scenario, both parents are tested in multiple single-cross combinations within the training set, enabling accurate estimation of parent GCA effects. With a preponderance of GCA variance over SCA variance, genotypic values of T2 single crosses can, therefore, be predicted with higher accuracy. In the T1 scenario, however, only one of the parents is tested in single-cross combination and consequently the prediction accuracy of T1 single crosses is lower than for T2 single crosses. The mean of T1 single-cross prediction accuracies observed in this study were 78, 86, and 80% of the T2 single-cross prediction for GY, PH, and SG, respectively. The prediction accuracy of the T1F single crosses is greater than that of the T1M single crosses. This finding can be explained by the smaller total number of females than males, which increases the number of times each female is tested in a single-cross combination. The mean accuracies of T0 single-cross prediction were 53, 75, and 59% of the mean accuracies of T2 single-cross prediction for GY, PH, and SG, respectively. This suggests that performance of single crosses having at least one tested parent can be effectively predicted using genomic-estimated GCA and SCA effects, but prediction accuracies suffer considerably if neither of the parents of a single cross are tested. This issue should be studied using larger population sizes, both in terms of more interconnected biparental populations and progenies per population, to determine if population size can overcome parent representation in the training set.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/11/3443/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="SEs of predicted GCA and SCA effects estimated using differing numbers of single crosses per parental inbred progeny." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-934584868" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;SEs of predicted GCA and SCA effects estimated using differing numbers of single crosses per parental inbred progeny.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/11/3443/F6.medium.gif" width="440" height="163"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/6/11/3443/F6.medium.gif" width="440" height="163"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/11/3443/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/11/3443/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469578" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-40" class="first-child">SEs of predicted GCA and SCA effects estimated using differing numbers of single crosses per parental inbred progeny.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-20" class="subsection"><h3>Comparison of single-cross prediction methods</h3><p id="p-41">The published studies on prediction of single-cross performance have used covariance among tested and untested single crosses (methods 2a and 2b) to predict the performance of untested single crosses (<a id="xref-ref-32-4" class="xref-bibr" href="#ref-32">Massman <em>et al.</em> 2013</a>; <a id="xref-ref-46-4" class="xref-bibr" href="#ref-46">Technow <em>et al.</em> 2014</a>). In an alternative approach, we used genomic-estimated GCA and SCA (methods 1a and 1b) to predict the performance of untested single crosses. The comparison of prediction accuracies showed that the four methods achieved comparable accuracies for predicting T2 and T1F single crosses. For T1M and T0 single crosses, however, methods 1a and 1b provided higher prediction accuracies compared with methods 2a and 2b. Although the same information is input into the two different types of methods (<em>i.e.</em>, additive genomic relationship matrices of parents, dominance relationship matrix of the single crosses), the methods differ in their underlying assumptions and in the way in which resulting predictions are calculated. Methods 1a and 1b use the three genomic relationship matrices separately in order to model the covariances of GCA effects of females, males, and SCA effects of single crosses. Methods 2a and 2b, on the other hand, combine the covariance matrices to estimate the single-cross covariance through summation of the covariance between the female parents and the covariance between the male parents. This difference can explain the improved performance of method 1 over method 2 under the T1M and T0 cross-validation scenarios. In general, the amount of information used from the tested genotypes for BLUP of untested genotypes depends upon the covariance between the tested and untested genotypes. The genetic covariance is a function of the population genetic variance and genetic relationship. In the case of a single population, the genetic variance is constant and thus, in essence, most of the information used in calculating BLUP of an untested genotype is derived from the most closely related tested genotypes. However, in the case of two separate populations, two individuals with higher covariance from the first population would not necessarily be more closely related than two individuals with a lower covariance from a second population. This is due to the fact that the genetic covariance estimate is population specific, as it depends on the genetic variance within a population (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Falconer and Mackay 1996</a>). As a result, in single crosses where two parental populations are involved (female population and male population), having higher covariance (as estimated in this and previous studies) may not mean they are more related than single crosses having lower covariance. Methods 2a and 2b, however, invariably assume that single crosses having higher covariance are more related than single crosses with lower covariance. This could result in wider use of information from comparatively less related single crosses, which could ultimately affect prediction accuracy.</p><p id="p-42">To test this hypothesis for differences in prediction accuracy of two groups of methods, we randomly sampled a balanced (equal number of females and males) subset of single crosses among 40 females and 40 males. Methods 2a and 2b were modified to correct the discrepancies in weighting of relationship between females and males by using the average of GCA variance of females and males. Specifically, for method 2a, the elements of <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-96.gif"/></noscript></span></span> and off-diagonal elements of <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-97.gif"/></noscript></span></span> were computed as:<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-98.gif"/></noscript></span></span>The diagonal elements of <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-99.gif"/></noscript></span></span> were estimated as:<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3443/embed/mml-math-100.gif"/></noscript></span></span>Method 2b was similarly modified. The GY prediction accuracy of the four methods for the T2, T1F, T1M, and T0 scenarios was evaluated using leave-one-individual-out cross-validations. The modified methods 2a and 2b obtained higher accuracies for T1M and T0 compared with the original methods 2a and 2b (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). The accuracies were comparable to methods 1a and 1b. For T2 and T1F scenarios, however, the modified methods 2a and 2b obtained lower accuracies than the original methods. This could also be explained based on our hypothesis. The use of average GCA variance enabled the model to extract information from more closely related tested single crosses. However, the amount of information extracted is related to the genetic covariance, which, as mentioned above, is a function of the genetic relationship and genetic variance. The T2 and T1F scenarios include tested female parents in the training set. Because GCA variance of the female population is larger in our study, use of the average GCA variance lowered the amount of information from the female parents relative to what would have been achieved if the populations were treated separately.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469625/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469625/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Correlation between observed and predicted grain yield (GY) in a random balanced subset of hybrids for three groups of hybrid prediction methods, as evaluated by leave-one-individual-out cross-validation</span><div class="sb-div caption-clear"></div></div></div><p id="p-44">Overall, these results indicate that single-cross covariance-based methods (2a and 2b) confound the genetic relationship and genetic variance in using information from tested single crosses. The combining ability-based methods (1a and 1b) correctly use genetic relationships and variances by separately estimating female and male GCA. The previous studies on single-cross prediction in maize (<a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Schrag <em>et al.</em> 2009</a>, <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">2010</a>; <a id="xref-ref-46-5" class="xref-bibr" href="#ref-46">Technow <em>et al.</em> 2014</a>), wheat (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Longin <em>et al.</em> 2013</a>), sunflower (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Reif <em>et al.</em> 2013</a>), and triticale (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Gowda <em>et al.</em> 2013</a>) have reported different estimates of GCA variance between two parental populations of single crosses. This suggests that single-cross prediction based on genomic estimated GCA and SCA is a better approach compared with the commonly used method using genomic covariances among single crosses.</p><p id="p-45">The prediction accuracies for T2, T1, and T0 single crosses reported by <a id="xref-ref-46-6" class="xref-bibr" href="#ref-46">Technow <em>et al.</em> (2014)</a> and T2 and T1 single crosses reported by <a id="xref-ref-32-5" class="xref-bibr" href="#ref-32">Massman <em>et al.</em> (2013)</a> are higher than the corresponding accuracies observed in the present study. This discrepancy is likely due to differences in population and family structure between the present study and those previously reported. <a id="xref-ref-32-6" class="xref-bibr" href="#ref-32">Massman <em>et al.</em> (2013)</a> and <a id="xref-ref-46-7" class="xref-bibr" href="#ref-46">Technow <em>et al.</em> (2014)</a> used single crosses made among a diverse set of established inbred parents that could be clustered into genetic groups whereas the present study consisted of many inbred progenies from only three families on each side of the heterotic pattern. Moreover, the families shared parents, resulting in less variation among families. As <a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">Windhausen <em>et al.</em> (2012)</a> reported, when population structure is present and not accounted for, the prediction accuracy can derive mostly from differences in mean performances between subpopulations. Our population contained less genetic variation and fewer subpopulations compared with previously reported studies. In addition, the average number of single-cross combinations per parental line was higher in previously reported studies.</p></div><div id="sec-21" class="subsection"><h3>The benefit of modeling SCA</h3><p id="p-46">We observed an increase in single-cross prediction accuracy for GY by modeling and estimating SCA effects and subsequently summing GCA and SCA effects. The increase in accuracy by modeling SCA was highest for T0 single crosses, followed by T1 and T2 single crosses. This result suggests that modeling SCA can be more beneficial for single crosses with untested parents compared with those with one or two tested parents. These findings could possibly be explained by a small number of single-cross combinations per parental line. If parents are tested in a small number of single-cross combinations, as in the present study, their GCA effect predictions could capture a significant portion of the SCA effect as well. The increase in prediction accuracy achieved by adding SCA would clearly depend on the magnitude of the SCA bias of the predicted GCA effect. We do not have the ability to estimate this bias, but in our study the ratio of SCA <em>vs.</em> GCA variance was small for all traits (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). When a parent has no performance data in single-cross combination, its GCA is predicted based on all tested relatives, resulting in a predicted GCA effect less biased by SCA. Hence, SCA is expected to improve the predictions for single crosses with untested parents.</p><p id="p-47">When predicting the GY of single crosses from a novel family (<em>i.e.</em>, leave-one-family-out cross-validations), the accuracy was similar or slightly lower when both GCA and SCA were included compared with GCA only. When information is not shared between family members, SCA is determined from single-cross combinations of relatives in other families, which are expected to be less accurate as SCA depends on specific parental combinations. This suggests that information from closely related single crosses is beneficial to SCA effect estimation, and that if this information is not available overall prediction accuracy can be reduced through error in the SCA effect estimation. Previously reported studies on hybrid genomic prediction in maize (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bernardo 1994</a>), wheat (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Zhao <em>et al.</em> 2013</a>), triticale (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Gowda <em>et al.</em> 2013</a>), and sunflower (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Reif <em>et al.</em> 2013</a>) reported a small decrease in prediction accuracy by modeling an SCA effect. These studies used diversity panels as their experimental material. This decrease in prediction accuracy could be attributed to an inability to accurately predict SCA from distantly related single-cross combinations. The accuracy of SCA effect estimation should be tested in every unique situation (<em>e.g.</em>, within-family selection <em>vs.</em> selection across families) and population where single-cross genomic prediction is being used, in order to prevent a possible detrimental outcome by adding SCA effects.</p></div><div id="sec-22" class="subsection"><h3>Prospects for early-stage single-cross prediction</h3><p id="p-48">Overall, this study indicates that breeders should consider redesigning hybrid breeding programs to take advantage of genomic prediction. The early stages of maize hybrid development consist of the generation of RILs or DHs from biparental families on each side of a heterotic pattern, followed by evaluation of their potential to serve as parents of hybrids. Traditionally, initial selections are conducted on the basis of topcross tests. Single crosses among selected inbred progenies are evaluated in later stages in the breeding pipeline. While this method has many advantages, one major disadvantage is that not all potential single crosses among breeding lines can be evaluated. Moreover, the addition of multiple years of topcross testing increases the time to hybrid release. The use of genomic prediction to identify superior single crosses could both shorten the length of time to hybrid release, and prevent the discarding of superior single crosses that just never happened to be phenotypically evaluated in the topcross system. We believe this can be achieved given the high prediction accuracies observed when both parents (T2) are included in the training set. Additionally, opportunity exists to optimize the genomic prediction of early-stage single crosses. Pedigree selection and frequent use of successful parents creates a family structure within typical hybrid maize breeding programs consisting of interconnected biparental families. The results from this study demonstrate that single-cross genomic prediction methods even hold potential for separating single crosses from a common family background (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>). The prediction accuracy for novel single-cross families was moderate to high, and the addition of single crosses from the same family to the training set only minimally improved accuracy. Further study of the optimization of larger training sets through leveraging family structure could further improve the accuracy of genomic prediction of single crosses.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-49">We are indebted to Rita H. Mumm for her vision in the design of the unique germplasm used in this study. We thank the editor and four anonymous reviewers of the earlier versions of this manuscript for their helpful input and suggestions. This research was supported by United States Department of Agriculture National Institute of Food and Agriculture grant ILLU-802-614. D.C.K. was supported by an Indian Council of Agricultural Research International Fellowship, and S.M.P. was supported by an Illinois Corn Marketing Board Fellowship in Plant Breeding. </p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-4"><p id="p-50">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031286/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031286/-/DC1</a>.</p></li><li class="fn" id="fn-5"><p id="p-51"><em>Communicating editor: J. B. Holland</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-05-17"><span class="received-label">Received </span>May 17, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-08-12"><span class="accepted-label">Accepted </span>August 12, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Kadam <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3443.1" data-doi="10.1007/s00122-011-1587-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Albrecht</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wimmer</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auinger</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erbe</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knaak</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-based prediction of testcross values in maize.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">339</span>–<span class="cit-lpage">350</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DAlbrecht%26rft.auinit1%253DT.%26rft.volume%253D123%26rft.issue%253D2%26rft.spage%253D339%26rft.epage%253D350%26rft.atitle%253DGenome-based%2Bprediction%2Bof%2Btestcross%2Bvalues%2Bin%2Bmaize.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1587-7%26rft_id%253Dinfo%253Apmid%252F21505832%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1587-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21505832&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3443.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3443.2" data-doi="10.1007/s00122-014-2305-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Albrecht</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auinger</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wimmer</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ogutu</span>  <span class="cit-name-given-names">J. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knaak</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-based prediction of maize hybrid performance across genetic groups, testers, locations, and years.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">1375</span>–<span class="cit-lpage">1386</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D1375%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-014-2305-z%26rft_id%253Dinfo%253Apmid%252F24723140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-014-2305-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24723140&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3443.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3443.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Relationship between single-cross performance and molecular marker heterozygosity.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">628</span>–<span class="cit-lpage">634</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D83%26rft.spage%253D628%26rft_id%253Dinfo%253Apmid%252F24202681%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24202681&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3443.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992HK21600013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3443.4" data-doi="10.2135/cropsci1994.0011183X003400010003x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Prediction of maize single-cross performance using RFLPs and information from related hybrids.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">20</span>–<span class="cit-lpage">25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBernardo%26rft.auinit1%253DR.%26rft.volume%253D34%26rft.issue%253D1%26rft.spage%253D20%26rft.epage%253D25%26rft.atitle%253DPrediction%2Bof%2BMaize%2BSingle-Cross%2BPerformance%2BUsing%2BRFLPs%2Band%2BInformation%2Bfrom%2BRelated%2BHybrids%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1994.0011183X003400010003x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1994.0011183X003400010003x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1994MV45600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3443.5" data-doi="10.2135/cropsci1996.0011183X003600010009x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span><span class="cit-comment">a</span> <span class="cit-article-title">Best linear unbiased prediction of maize single-cross performance.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">50</span>–<span class="cit-lpage">56</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBernardo%26rft.auinit1%253DR.%26rft.volume%253D36%26rft.issue%253D1%26rft.spage%253D50%26rft.epage%253D56%26rft.atitle%253DBest%2BLinear%2BUnbiased%2BPrediction%2Bof%2BMaize%2BSingle-Cross%2BPerformance%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1996.0011183X003600010009x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1996.0011183X003600010009x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1996TN28200009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3443.6" data-doi="10.2135/cropsci1996.0011183X003600040009x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span><span class="cit-comment">b</span> <span class="cit-article-title">Best linear unbiased prediction of the performance of crosses between untested maize inbreds.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">872</span>–<span class="cit-lpage">876</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBernardo%26rft.auinit1%253DR.%26rft.volume%253D36%26rft.issue%253D4%26rft.spage%253D872%26rft.epage%253D876%26rft.atitle%253DBest%2BLinear%2BUnbiased%2BPrediction%2Bof%2Bthe%2BPerformance%2Bof%2BCrosses%2Bbetween%2BUntested%2BMaize%2BInbreds%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1996.0011183X003600040009x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1996.0011183X003600040009x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-book" id="cit-6.11.3443.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-source">Breeding for Quantitative Traits in Plants</span>. <span class="cit-publ-name">Stemma Press</span>, <span class="cit-publ-loc">Woodbury</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-book" id="cit-6.11.3443.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Butler</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilmour</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gogel</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-source"><em>ASReml-R Reference Manual</em>. </span><span class="cit-publ-name">The State of Queensland, Department of Primary Industries and Fisheries</span>, <span class="cit-publ-loc">Brisbane</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3443.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Canty</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ripley</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">boot: Bootstrap R (S-Plus) Functions</span>. <abbr class="cit-jnl-abbrev">R package version 1</abbr>.<span class="cit-fpage">3</span>–<span class="cit-lpage">18</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3443.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charcosset</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lefort-Buson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gallais</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Relationship between heterosis and heterozygosity at marker loci: a theoretical computation.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">571</span>–<span class="cit-lpage">575</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D81%26rft.spage%253D571%26rft_id%253Dinfo%253Apmid%252F24221369%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24221369&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3443.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3443.11" data-doi="10.2307/3001412"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Comstock</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">H. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1948</span> <span class="cit-article-title">The components of genetic variance in populations of biparental progenies and their use in estimating the average degree of dominance.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">254</span>–<span class="cit-lpage">266</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.stitle%253DBiometrics%26rft.aulast%253DComstock%26rft.auinit1%253DR.%2BE.%26rft.volume%253D4%26rft.issue%253D4%26rft.spage%253D254%26rft.epage%253D266%26rft.atitle%253DThe%2Bcomponents%2Bof%2Bgenetic%2Bvariance%2Bin%2Bpopulations%2Bof%2Bbiparental%2Bprogenies%2Band%2Btheir%2Buse%2Bin%2Bestimating%2Bthe%2Baverage%2Bdegree%2Bof%2Bdominance.%26rft_id%253Dinfo%253Adoi%252F10.2307%252F3001412%26rft_id%253Dinfo%253Apmid%252F18108899%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/3001412&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18108899&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3443.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1948UG24700005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3443.12" data-doi="10.2527/jas.2006-683"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Marker-assisted selection for commercial crossbred performance.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">2104</span>–<span class="cit-lpage">2114</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas.2006-683%26rft_id%253Dinfo%253Apmid%252F17504955%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas.2006-683&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17504955&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3443.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3443.13" data-doi="10.1534/genetics.112.143313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Whole-genome regression and prediction methods applied to plant and animal breeding.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">327</span>–<span class="cit-lpage">345</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.143313%26rft_id%253Dinfo%253Apmid%252F22745228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvMzI3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNDQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3443.14" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3443.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3443.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falconer</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-comment">Introduction to Quantitative Genetics, Ed 4. Longmans Green, Harlow, Essex, UK</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-book" id="cit-6.11.3443.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fehr</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-source">Principles of Cultivar Development: Theory and Technique</span>. <span class="cit-publ-name">Macmillan</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3443.17" data-doi="10.1371/journal.pone.0090346"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casstevens</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harriman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">TASSEL-GBS: a high capacity genotyping by sequencing analysis pipeline.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e90346</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De90346%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0090346%26rft_id%253Dinfo%253Apmid%252F24587335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0090346&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24587335&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3443.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3443.18" data-doi="10.1007/s10681-012-0784-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gowda</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maurer</span>  <span class="cit-name-given-names">H. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weissmann</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Würschum</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Best linear unbiased prediction of triticale hybrid performance.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">191</span>: <span class="cit-fpage">223</span>–<span class="cit-lpage">230</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D191%26rft.spage%253D223%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-012-0784-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-012-0784-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3443.19" data-doi="10.2135/cropsci1977.0011183X001700050007x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hallauer</span>  <span class="cit-name-given-names">A. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1977</span> <span class="cit-article-title">Relation between inbred and hybrid traits in maize.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">703</span>–<span class="cit-lpage">706</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253De%2BGama%26rft.auinit1%253DE.%2BE.%2BG.%26rft.volume%253D17%26rft.issue%253D5%26rft.spage%253D703%26rft.epage%253D706%26rft.atitle%253DRelation%2BBetween%2BInbred%2Band%2BHybrid%2BTraits%2Bin%2BMaize%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1977.0011183X001700050007x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1977.0011183X001700050007x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-book" id="cit-6.11.3443.20"><div class="cit-metadata unstructured">Hallauer, R., W. Russell, and K. Lamkey, 1988 Corn breeding, pp. 463–564 in <em>Corn and Corn Improvement</em>, Agronomy Monograph 18, Ed. 3. American Society of Agronomy, Crop Science Society of America, and Soil Science Society of America, Madison, WI.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3443.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laroche</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genotyping-by-sequencing (GBS), an ultimate marker-assisted selection (MAS) tool to accelerate plant breeding.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">484</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D5%26rft.spage%253D484%26rft_id%253Dinfo%253Apmid%252F25324846%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=25324846&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3443.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3443.22" data-doi="10.2135/cropsci2008.08.0512"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomic selection for crop improvement.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DHeffner%26rft.auinit1%253DE.%2BL.%26rft.volume%253D49%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D12%26rft.atitle%253DGenomic%2BSelection%2Bfor%2BCrop%2BImprovement%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2008.08.0512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2008.08.0512&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000263344300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3443.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nyquist</span>  <span class="cit-name-given-names">W. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cervantes-Martínez</span>  <span class="cit-name-given-names">C. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Estimating and interpreting heritability for plant breeding: an update.</span> <abbr class="cit-jnl-abbrev">Plant Breed. Rev.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">9</span>–<span class="cit-lpage">112</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBreed.%2BRev.%26rft.volume%253D22%26rft.spage%253D9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3443.24" data-doi="10.2135/cropsci2013.11.0774"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jacobson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lian</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">General combining ability model for genomewide selection in a biparental cross.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">895</span>–<span class="cit-lpage">905</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D54%26rft.spage%253D895%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2013.11.0774%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2013.11.0774&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3443.25" data-doi="10.2134/agronj1932.00021962002400070004x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jenkins</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brunson</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1932</span> <span class="cit-article-title">Methods of testing inbred lines of maize in crossbred combinations.</span> <abbr class="cit-jnl-abbrev">J. Am. Soc. Agron.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">523</span>–<span class="cit-lpage">530</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BSoc.%2BAgron.%26rft.volume%253D24%26rft.spage%253D523%26rft_id%253Dinfo%253Adoi%252F10.2134%252Fagronj1932.00021962002400070004x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2134/agronj1932.00021962002400070004x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-confproc" id="cit-6.11.3443.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kohavi</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <em>A study of cross-validation and bootstrap for accuracy estimation and model selection</em>, pp. <span class="cit-fpage">1137</span>–<span class="cit-lpage">1143</span> in Proceedings of the Fourteenth International Joint Conference on Artificial Intelligence, edited by <span class="cit-ed"><span class="cit-name-surname">Mellish</span>  <span class="cit-name-given-names">C. S.</span></span>. <span class="cit-publ-name">Morgan Kaufmann</span>, <span class="cit-publ-loc">San Francisco</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3443.27" data-doi="10.2135/cropsci2006.04.0275"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ash</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Good</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Re-examining the relationship between degree of relatedness, genetic effects, and heterosis in maize.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">629</span>–<span class="cit-lpage">635</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DLee%26rft.auinit1%253DE.%2BA.%26rft.volume%253D47%26rft.issue%253D2%26rft.spage%253D629%26rft.epage%253D635%26rft.atitle%253DRe-examining%2Bthe%2BRelationship%2Bbetween%2BDegree%2Bof%2BRelatedness%252C%2BGenetic%2BEffects%252C%2Band%2BHeterosis%2Bin%2BMaize%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2006.04.0275%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2006.04.0275&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000245735800018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.11.3443.28" data-doi="10.1071/CP13363"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic selection in crops, trees and forages: a review.</span> <abbr class="cit-jnl-abbrev">Crop Pasture Sci.</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">1177</span>–<span class="cit-lpage">1191</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BPasture%2BSci.%26rft.volume%253D65%26rft.spage%253D1177%26rft_id%253Dinfo%253Adoi%252F10.1071%252FCP13363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1071/CP13363&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3443.29" data-doi="10.1007/s00122-013-2172-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Longin</span>  <span class="cit-name-given-names">C. F. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gowda</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mühleisen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ebmeyer</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kazman</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Hybrid wheat: quantitative genetic parameters and consequences for the design of breeding programs.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">126</span>: <span class="cit-fpage">2791</span>–<span class="cit-lpage">2801</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D126%26rft.spage%253D2791%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2172-z%26rft_id%253Dinfo%253Apmid%252F23913277%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2172-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23913277&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3443.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3443.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Love</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wentz</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1914</span> <span class="cit-article-title">Correlations between ear characters and yield in corn.</span> <abbr class="cit-jnl-abbrev">Laboratory Papers.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">3</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLaboratory%2BPapers.%26rft.volume%253D1%26rft.spage%253D3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3443.31" data-doi="10.2135/cropsci2013.04.0252"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mansfield</span>  <span class="cit-name-given-names">B. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mumm</span>  <span class="cit-name-given-names">R. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Survey of plant density tolerance in US maize germplasm.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">157</span>–<span class="cit-lpage">173</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D54%26rft.spage%253D157%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2013.04.0252%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2013.04.0252&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3443.32" data-doi="10.1007/s00122-012-1955-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Massman</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gordillo</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenzana</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomewide predictions from maize single-cross data.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">126</span>: <span class="cit-fpage">13</span>–<span class="cit-lpage">22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D126%26rft.spage%253D13%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-012-1955-y%26rft_id%253Dinfo%253Apmid%252F22886355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-012-1955-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22886355&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3443.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-book" id="cit-6.11.3443.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Genetic diversity and heterosis</span>, pp. <span class="cit-fpage">99</span>–<span class="cit-lpage">118</span>, in <span class="cit-source">The Genetics and Exploitation of Heterosis in Crops</span>. <span class="cit-publ-name">American Society of Agronomy, Crop Science Society of America, and Soil Science Society of America</span><span class="cit-publ-loc">Madison, WI</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3443.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzExLzM0NDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3443.35" data-doi="10.1093/biomet/63.1.83"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">1976</span> <span class="cit-article-title">A new class of resolvable incomplete block designs.</span> <abbr class="cit-jnl-abbrev">Biometrika</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">83</span>–<span class="cit-lpage">92</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrika%26rft.stitle%253DBiometrika%26rft.aulast%253DPATTERSON%26rft.auinit1%253DH.%2BD.%26rft.volume%253D63%26rft.issue%253D1%26rft.spage%253D83%26rft.epage%253D92%26rft.atitle%253DA%2Bnew%2Bclass%2Bof%2Bresolvable%2Bincomplete%2Bblock%2Bdesigns%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiomet%252F63.1.83%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/biomet/63.1.83&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1976BL81800010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3443.36" data-doi="10.1111/pbr.12007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reif</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Würschum</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gowda</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hahn</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic prediction of sunflower hybrid performance.</span> <abbr class="cit-jnl-abbrev">Plant Breed.</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">107</span>–<span class="cit-lpage">114</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBreed.%26rft.volume%253D132%26rft.spage%253D107%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpbr.12007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pbr.12007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3443.37" data-doi="10.1038/ng.1033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Riedelsheimer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Czedik-Eysenberg</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grieder</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lisec</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Technow</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic and metabolic prediction of complex heterotic traits in hybrid maize.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">217</span>–<span class="cit-lpage">220</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DRiedelsheimer%26rft.auinit1%253DC.%26rft.volume%253D44%26rft.issue%253D2%26rft.spage%253D217%26rft.epage%253D220%26rft.atitle%253DGenomic%2Band%2Bmetabolic%2Bprediction%2Bof%2Bcomplex%2Bheterotic%2Btraits%2Bin%2Bhybrid%2Bmaize.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.1033%26rft_id%253Dinfo%253Apmid%252F22246502%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.1033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22246502&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3443.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3443.38" data-doi="10.1007/s00122-006-0363-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schrag</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sørensen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frisch</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Prediction of single-cross hybrid performance for grain yield and grain dry matter content in maize using AFLP markers associated with QTL.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">1037</span>–<span class="cit-lpage">1047</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DSchrag%26rft.auinit1%253DT.%2BA.%26rft.volume%253D113%26rft.issue%253D6%26rft.spage%253D1037%26rft.epage%253D1047%26rft.atitle%253DPrediction%2Bof%2Bsingle-cross%2Bhybrid%2Bperformance%2Bfor%2Bgrain%2Byield%2Band%2Bgrain%2Bdry%2Bmatter%2Bcontent%2Bin%2Bmaize%2Busing%2BAFLP%2Bmarkers%2Bassociated%2Bwith%2BQTL.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-006-0363-6%26rft_id%253Dinfo%253Apmid%252F16896712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-006-0363-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16896712&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3443.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000240831100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3443.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schrag</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frish</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dhillon</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Marker-based prediction of hybrid performance in maize single-crosses involving doubled haploids.</span> <abbr class="cit-jnl-abbrev">Maydica</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">353</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMaydica%26rft.volume%253D54%26rft.spage%253D353%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3443.40" data-doi="10.1007/s00122-007-0521-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schrag</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maurer</span>  <span class="cit-name-given-names">H. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piepho</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peleman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Prediction of single-cross hybrid performance in maize using haplotype blocks associated with QTL for grain yield.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">1345</span>–<span class="cit-lpage">1355</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DSchrag%26rft.auinit1%253DT.%2BA.%26rft.volume%253D114%26rft.issue%253D8%26rft.spage%253D1345%26rft.epage%253D1355%26rft.atitle%253DPrediction%2Bof%2Bsingle-cross%2Bhybrid%2Bperformance%2Bin%2Bmaize%2Busing%2Bhaplotype%2Bblocks%2Bassociated%2Bwith%2BQTL%2Bfor%2Bgrain%2Byield.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-007-0521-5%26rft_id%253Dinfo%253Apmid%252F17323040%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-007-0521-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17323040&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3443.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3443.41" data-doi="10.1007/s00122-009-1208-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schrag</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Möhring</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kusterer</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dhillon</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Prediction of hybrid performance in maize using molecular markers and joint analyses of hybrids and parental inbreds.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">120</span>: <span class="cit-fpage">451</span>–<span class="cit-lpage">461</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DSchrag%26rft.auinit1%253DT.%2BA.%26rft.volume%253D120%26rft.issue%253D2%26rft.spage%253D451%26rft.epage%253D461%26rft.atitle%253DPrediction%2Bof%2Bhybrid%2Bperformance%2Bin%2Bmaize%2Busing%2Bmolecular%2Bmarkers%2Band%2Bjoint%2Banalyses%2Bof%2Bhybrids%2Band%2Bparental%2Binbreds.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-009-1208-x%26rft_id%253Dinfo%253Apmid%252F19916002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-009-1208-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19916002&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3443.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3443.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shull</span>  <span class="cit-name-given-names">G. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1909</span> <span class="cit-article-title">A pure-line method in corn breeding.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">51</span>–<span class="cit-lpage">58</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BHered.%26rft.volume%253D5%26rft.spage%253D51%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-book" id="cit-6.11.3443.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Betrán</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-source"><em>Corn: Origin</em>, <em>History</em>, <em>Technology</em>, <em>and Production</em>.</span> <span class="cit-publ-name">John Wiley &amp; Sons</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3443.44" data-doi="10.2135/cropsci1986.0011183X002600030023x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Covariance between line per se and testcross performance.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">540</span>–<span class="cit-lpage">543</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DSmith%26rft.auinit1%253DO.%2BS.%26rft.volume%253D26%26rft.issue%253D3%26rft.spage%253D540%26rft.epage%253D543%26rft.atitle%253DCovariance%2Bbetween%2BLine%2Bper%2Bse%2Band%2BTestcross%2BPerformance%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1986.0011183X002600030023x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1986.0011183X002600030023x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1986C310500023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3443.45" data-doi="10.1007/s00122-012-1905-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Technow</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riedelsheimer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schrag</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic prediction of hybrid performance in maize with models incorporating dominance and population specific marker effects.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">1181</span>–<span class="cit-lpage">1194</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DTechnow%26rft.auinit1%253DF.%26rft.volume%253D125%26rft.issue%253D6%26rft.spage%253D1181%26rft.epage%253D1194%26rft.atitle%253DGenomic%2Bprediction%2Bof%2Bhybrid%2Bperformance%2Bin%2Bmaize%2Bwith%2Bmodels%2Bincorporating%2Bdominance%2Band%2Bpopulation%2Bspecific%2Bmarker%2Beffects.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-012-1905-8%26rft_id%253Dinfo%253Apmid%252F22733443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-012-1905-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22733443&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3443.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3443.46" data-doi="10.1534/genetics.114.165860"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Technow</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schrag</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schipprack</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bauer</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simianer</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome properties and prospects of genomic prediction of hybrid performance in a breeding program of maize.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">197</span>: <span class="cit-fpage">1343</span>–<span class="cit-lpage">1355</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.165860%26rft_id%253Dinfo%253Apmid%252F24850820%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Ny80LzEzNDMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzExLzM0NDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3443.47" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vuylsteke</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuiper</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stam</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Chromosomal regions involved in hybrid performance and heterosis: their AFLP-based identification and practical use in prediction models.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">208</span>–<span class="cit-lpage">218</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3443.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3443.48" data-doi="10.1046/j.1365-2540.2000.00747.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Windhausen</span>  <span class="cit-name-given-names">V. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atlin</span>  <span class="cit-name-given-names">G. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Effectiveness of genomic prediction of maize hybrid performance in different breeding populations and environments.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">1427</span>–<span class="cit-lpage">1436</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D85%26rft.spage%253D208%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2540.2000.00747.x%26rft_id%253Dinfo%253Apmid%252F11012724%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2540.2000.00747.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11012724&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3443.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3443.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reif</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic prediction of hybrid wheat performance.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">802</span>–<span class="cit-lpage">810</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D2%26rft.spage%253D1427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3443.50" data-doi="10.2135/cropsci2012.08.0463"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maurer</span>  <span class="cit-name-given-names">H. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genome-based establishment of a high-yielding heterotic pattern for hybrid wheat breeding.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">15624</span>–<span class="cit-lpage">15629</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.volume%253D53%26rft.spage%253D802%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2012.08.0463%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2012.08.0463&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000319527000008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/11/3443.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/10/3431" title="RNAi-Based Suppressor Screens Reveal Genetic Interactions Between the CRL2LRR-1 E3-Ligase and the DNA Replication Machinery in Caenorhabditis elegans" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/11/3455" title="Proper Sterol Distribution Is Required for Candida albicans Hyphal Formation and Virulence" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/11"><span>Volume 6 Issue 11, November 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/11.cover-source.jpg?itok=hTYn4PRy" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/11/3443.supplemental" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-quw5egULtmv6dfGYjsZyE1MsrdM-0nz9vOXz6zVFb7U" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/11/3443" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/11/3443" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/11/3443" class="active" data-icon-position="" data-hide-link-title="0">Genomic Prediction of Single Crosses in the Early Stages of a Maize Hybrid Breeding Pipeline</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460465" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-cHKNQkFOyAHezn7GyLGBc9rbrZRibyn7jEOeH0CI860" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/11/3443.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/11/3443" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-OfNtQ01nZZXk1xNaqz9apCLQecHlVpAUPCH9wXagJ8o" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/11/3443" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/11/3443.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460465" id="node460465--2" data-pisa="ggg;6/11/3443" data-pisa-master="ggg;g3.116.031286" data-apath="/ggg/6/11/3443.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Prediction of Single Crosses in the Early Stages of a Maize Hybrid Breeding Pipeline</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Dnyaneshwar C. Kadam</span>, <span class="highwire-citation-author" data-delta="1">Sarah M. Potts</span>, <span class="highwire-citation-author" data-delta="2">Martin O. Bohn</span>, <span class="highwire-citation-author" data-delta="3">Alexander E. Lipka</span> and <span class="highwire-citation-author" data-delta="4">Aaron J. Lorenz</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3443-3453;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.031286 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460465--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Dnyaneshwar C. Kadam </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Horticulture, University of Nebraska, Lincoln, Nebraska 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dnyaneshwar%2BC.%2BKadam%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kadam%20DC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADnyaneshwar%2BC.%2BKadam%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Sarah M. Potts </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Crop Sciences, University of Illinois at Urbana-Champaign, Urbana, Illinois 61801</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sarah%2BM.%2BPotts%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Potts%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASarah%2BM.%2BPotts%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Martin O. Bohn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Crop Sciences, University of Illinois at Urbana-Champaign, Urbana, Illinois 61801</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BO.%2BBohn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bohn%20MO&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMartin%2BO.%2BBohn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Alexander E. Lipka </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Crop Sciences, University of Illinois at Urbana-Champaign, Urbana, Illinois 61801</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexander%2BE.%2BLipka%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lipka%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlexander%2BE.%2BLipka%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Aaron J. Lorenz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Horticulture, University of Nebraska, Lincoln, Nebraska 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aaron%2BJ.%2BLorenz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lorenz%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAaron%2BJ.%2BLorenz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:lore0149@umn.edu" class="" data-icon-position="" data-hide-link-title="0">lore0149@umn.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/11/3443" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/11/3443&amp;title=Genomic%20Prediction%20of%20Single%20Crosses%20in%20the%20Early%20Stages%20of%20a%20Maize%20Hybrid%20Breeding%20Pipeline" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/11/3443&amp;title=Genomic%20Prediction%20of%20Single%20Crosses%20in%20the%20Early%20Stages%20of%20a%20Maize%20Hybrid%20Breeding%20Pipeline" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/11/3443&amp;title=Genomic%20Prediction%20of%20Single%20Crosses%20in%20the%20Early%20Stages%20of%20a%20Maize%20Hybrid%20Breeding%20Pipeline" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/11/3443&amp;text=Genomic%20Prediction%20of%20Single%20Crosses%20in%20the%20Early%20Stages%20of%20a%20Maize%20Hybrid%20Breeding%20Pipeline" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/11/3443&amp;title=Genomic%20Prediction%20of%20Single%20Crosses%20in%20the%20Early%20Stages%20of%20a%20Maize%20Hybrid%20Breeding%20Pipeline" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/11/3443&amp;t=Genomic%20Prediction%20of%20Single%20Crosses%20in%20the%20Early%20Stages%20of%20a%20Maize%20Hybrid%20Breeding%20Pipeline" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/11/3443&amp;title=Genomic%20Prediction%20of%20Single%20Crosses%20in%20the%20Early%20Stages%20of%20a%20Maize%20Hybrid%20Breeding%20Pipeline" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/11/3443&amp;title=Genomic%20Prediction%20of%20Single%20Crosses%20in%20the%20Early%20Stages%20of%20a%20Maize%20Hybrid%20Breeding%20Pipeline" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460465/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460465" id="node460465--3" data-pisa="ggg;6/11/3443" data-pisa-master="ggg;g3.116.031286" data-apath="/ggg/6/11/3443.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Prediction of Single Crosses in the Early Stages of a Maize Hybrid Breeding Pipeline</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Dnyaneshwar C. Kadam</span>, <span class="highwire-citation-author" data-delta="1">Sarah M. Potts</span>, <span class="highwire-citation-author" data-delta="2">Martin O. Bohn</span>, <span class="highwire-citation-author" data-delta="3">Alexander E. Lipka</span> and <span class="highwire-citation-author" data-delta="4">Aaron J. Lorenz</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3443-3453;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.031286 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460465--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Dnyaneshwar C. Kadam </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Horticulture, University of Nebraska, Lincoln, Nebraska 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dnyaneshwar%2BC.%2BKadam%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kadam%20DC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADnyaneshwar%2BC.%2BKadam%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Sarah M. Potts </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Crop Sciences, University of Illinois at Urbana-Champaign, Urbana, Illinois 61801</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sarah%2BM.%2BPotts%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Potts%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASarah%2BM.%2BPotts%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Martin O. Bohn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Crop Sciences, University of Illinois at Urbana-Champaign, Urbana, Illinois 61801</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BO.%2BBohn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bohn%20MO&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMartin%2BO.%2BBohn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Alexander E. Lipka </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Crop Sciences, University of Illinois at Urbana-Champaign, Urbana, Illinois 61801</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexander%2BE.%2BLipka%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lipka%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlexander%2BE.%2BLipka%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Aaron J. Lorenz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Horticulture, University of Nebraska, Lincoln, Nebraska 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aaron%2BJ.%2BLorenz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lorenz%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAaron%2BJ.%2BLorenz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:lore0149@umn.edu" class="" data-icon-position="" data-hide-link-title="0">lore0149@umn.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3443&amp;rft.epage=3453&amp;rft.atitle=Genomic%20Prediction%20of%20Single%20Crosses%20in%20the%20Early%20Stages%20of%20a%20Maize%20Hybrid%20Breeding%20Pipeline&amp;rft.volume=6&amp;rft.issue=11&amp;rft.date=2016-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Kadam%2C+Dnyaneshwar+C.&amp;rft.au=Potts%2C+Sarah+M.&amp;rft.au=Bohn%2C+Martin+O.&amp;rft.au=Lipka%2C+Alexander+E.&amp;rft.au=Lorenz%2C+Aaron+J."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460465/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460465/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460465/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460465/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460465/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460465/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460465/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460465/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460465/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460465/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460465/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460465/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/11/3443" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/11/3443.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/11/3443.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/11/3443.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/11/3443.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
