<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/11/3571" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/11/3571.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/11/3571.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/11/3571.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/11/3571.atom" />
<meta name="HW.pisa" content="ggg;6/11/3571" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genome Sequencing of Chromosome 1 Substitution Lines Derived from Chinese Wild Mice Revealed a Unique Resource for Genetic Studies of Complex Traits" />
<meta name="DC.Identifier" content="10.1534/g3.116.033902" />
<meta name="DC.Date" content="2016-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright Â© 2016 Xu et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Mouse resources such as Collaborative Cross, outbred stocks, Hybrid Mouse Diversity Panel, and chromosome substitution strains have been instrumental to many progresses in the studies of complex traits genetics. We have established a population of chromosome 1 (Chr 1) substitution lines (C1SLs) in which donor chromosomes were derived from Chinese wild mice. Genome sequencing of 18 lines of this population showed that Chr 1 had been replaced by the donor chromosome. About 4.5 million unique single nucleotide polymorphisms and indels were discovered on Chr 1, of which 1.3 million were novel. Compared with sequenced classical inbred strains, Chr 1 of each C1SL had fivefold more variants, and more loss of function and potentially regulatory variants. Further haplotype analysis showed that the donor chromosome accumulated more historical recombination events, with the largest haplotype block being only 100 kb, and about 57% of the blocks were &lt;1 kb. Subspecies origin analysis showed that these chromosomes had a mosaic genome structure that dominantly originated from Mus musculus musculus and M. m. castaneus subspecies, except for the C57BL/6J-Chr1KM line from M. m. domesticus . In addition, phenotyping four of these lines on blood biochemistry suggested that there were substantial phenotypic variations among our lines, especially line C57BL/6J-Chr1HZ and donor strain C57BL/6J. Further gene ontology enrichment revealed that the differentially expressed genes among liver-expressed genes between C57BL/6J and C57BL/6J-Chr1HZ were enriched in lipid metabolism biological processes. All these characteristics enable C1SLs to be a unique resource for identifying and fine mapping quantitative trait loci on mouse Chr 1, and carrying out systems genetics studies of complex traits." />
<meta name="DC.Contributor" content="Fuyi Xu" />
<meta name="DC.Contributor" content="Tianzhu Chao" />
<meta name="DC.Contributor" content="Yingming Liang" />
<meta name="DC.Contributor" content="Kai Li" />
<meta name="DC.Contributor" content="Shixian Hu" />
<meta name="DC.Contributor" content="Maochun Wang" />
<meta name="DC.Contributor" content="Yuxun Zhou" />
<meta name="DC.Contributor" content="Hongyan Xu" />
<meta name="DC.Contributor" content="Junhua Xiao" />
<meta name="article:published_time" content="2016-11-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Genome Sequencing of Chromosome 1 Substitution Lines Derived from Chinese Wild Mice Revealed a Unique Resource for Genetic Studies of Complex Traits" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Mouse resources such as Collaborative Cross, outbred stocks, Hybrid Mouse Diversity Panel, and chromosome substitution strains have been instrumental to many progresses in the studies of complex traits genetics. We have established a population of chromosome 1 (Chr 1) substitution lines (C1SLs) in which donor chromosomes were derived from Chinese wild mice. Genome sequencing of 18 lines of this population showed that Chr 1 had been replaced by the donor chromosome. About 4.5 million unique single nucleotide polymorphisms and indels were discovered on Chr 1, of which 1.3 million were novel. Compared with sequenced classical inbred strains, Chr 1 of each C1SL had fivefold more variants, and more loss of function and potentially regulatory variants. Further haplotype analysis showed that the donor chromosome accumulated more historical recombination events, with the largest haplotype block being only 100 kb, and about 57% of the blocks were &amp;lt;1 kb. Subspecies origin analysis showed that these chromosomes had a mosaic genome structure that dominantly originated from &lt;i&gt;Mus musculus musculus&lt;/i&gt; and &lt;i&gt;M. m. castaneus&lt;/i&gt; subspecies, except for the C57BL/6J-Chr1&lt;sup&gt;KM&lt;/sup&gt; line from &lt;i&gt;M. m. domesticus&lt;/i&gt;. In addition, phenotyping four of these lines on blood biochemistry suggested that there were substantial phenotypic variations among our lines, especially line C57BL/6J-Chr1&lt;sup&gt;HZ&lt;/sup&gt; and donor strain C57BL/6J. Further gene ontology enrichment revealed that the differentially expressed genes among liver-expressed genes between C57BL/6J and C57BL/6J-Chr1&lt;sup&gt;HZ&lt;/sup&gt; were enriched in lipid metabolism biological processes. All these characteristics enable C1SLs to be a unique resource for identifying and fine mapping quantitative trait loci on mouse Chr 1, and carrying out systems genetics studies of complex traits.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/11/01" />
<meta name="citation_mjid" content="ggg;6/11/3571" />
<meta name="citation_id" content="6/11/3571" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/11/3571" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/11/3571.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/11/3571.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/11/3571.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.033902" />
<meta name="citation_pmid" content="27605517" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3571" />
<meta name="citation_lastpage" content="3580" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Fuyi Xu" />
<meta name="citation_author_institution" content="College of Chemistry, Chemical Engineering, and Biotechnology, Donghua University, Shanghai 201620, China" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5952-5771" />
<meta name="citation_author" content="Tianzhu Chao" />
<meta name="citation_author_institution" content="College of Chemistry, Chemical Engineering, and Biotechnology, Donghua University, Shanghai 201620, China" />
<meta name="citation_author" content="Yingming Liang" />
<meta name="citation_author_institution" content="The Laboratory of Genetic Regulators in the Immune System, Xinxiang Medical University, Henan 453003, China" />
<meta name="citation_author" content="Kai Li" />
<meta name="citation_author_institution" content="College of Chemistry, Chemical Engineering, and Biotechnology, Donghua University, Shanghai 201620, China" />
<meta name="citation_author" content="Shixian Hu" />
<meta name="citation_author_institution" content="College of Chemistry, Chemical Engineering, and Biotechnology, Donghua University, Shanghai 201620, China" />
<meta name="citation_author" content="Maochun Wang" />
<meta name="citation_author_institution" content="College of Chemistry, Chemical Engineering, and Biotechnology, Donghua University, Shanghai 201620, China" />
<meta name="citation_author" content="Yuxun Zhou" />
<meta name="citation_author_institution" content="College of Chemistry, Chemical Engineering, and Biotechnology, Donghua University, Shanghai 201620, China" />
<meta name="citation_author" content="Hongyan Xu" />
<meta name="citation_author_institution" content="Department of Biostatistics and Epidemiology, Medical College of Georgia, Augusta University, Georgia 30912" />
<meta name="citation_author_email" content="hxu@augusta.edu" />
<meta name="citation_author_email" content="xiaojunhua@dhu.edu.cn" />
<meta name="citation_author" content="Junhua Xiao" />
<meta name="citation_author_institution" content="College of Chemistry, Chemical Engineering, and Biotechnology, Donghua University, Shanghai 201620, China" />
<meta name="citation_author_email" content="hxu@augusta.edu" />
<meta name="citation_author_email" content="xiaojunhua@dhu.edu.cn" />
<meta name="citation_reference" content="citation_journal_title=Hum. Mol. Genet.;citation_author=E. Ahlqvist;citation_author=D. Ekman;citation_author=T. Lindvall;citation_author=M. Popovic;citation_author=M. Forster;citation_title=High-resolution mapping of a complex disease, a model for rheumatoid arthritis, using heterogeneous stock mice.;citation_volume=20;citation_year=2011;citation_pmid=21565963;citation_doi=10.1093/hmg/ddr206" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. L. Aylor;citation_author=W. Valdar;citation_author=W. Foulds-Mathes;citation_author=R. J. Buus;citation_author=R. A. Verdugo;citation_title=Genetic analysis of complex traits in the emerging collaborative cross.;citation_volume=21;citation_year=2011;citation_pmid=21406540;citation_doi=10.1101/gr.111310.110" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=J. C. Barrett;citation_author=B. Fry;citation_author=J. Maller;citation_author=M. J. Daly;citation_title=Haploview: analysis and visualization of LD and haplotype maps.;citation_volume=21;citation_year=2005;citation_pmid=15297300;citation_doi=10.1093/bioinformatics/bth457" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=B. J. Bennett;citation_author=C. R. Farber;citation_author=L. Orozco;citation_author=H. M. Kang;citation_author=A. Ghazalpour;citation_title=A high-resolution association mapping panel for the dissection of complex traits in mice.;citation_volume=20;citation_year=2010;citation_pmid=20054062;citation_doi=10.1101/gr.099234.109" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=B. J. Bennett;citation_author=R. C. Davis;citation_author=M. Civelek;citation_author=L. Orozco;citation_author=J. Wu;citation_title=Genetic architecture of atherosclerosis in mice: a systems genetics analysis of common inbred strains.;citation_pages=e1005711;citation_volume=11;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Annual Review of Ecology and Systematics;citation_journal_abbrev=Annu. Rev. Ecol. Syst.;citation_author=P. Boursot;citation_author=J. C. Auffray;citation_author=J. Brittondavidian;citation_author=F. Bonhomme;citation_title=The Evolution of House Mice;citation_pages=119-152;citation_volume=24;citation_year=1993;citation_issue=1;citation_doi=10.1146/annurev.es.24.110193.001003" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. A. Buchner;citation_author=J. H. Nadeau;citation_title=Contrasting genetic architectures in different mouse reference populations used for studying complex traits.;citation_volume=25;citation_year=2015;citation_pmid=25953951;citation_doi=10.1101/gr.187450.114" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=C. Chiang;citation_author=R. M. Layer;citation_author=G. G. Faust;citation_author=M. R. Lindberg;citation_author=D. B. Rose;citation_title=SpeedSeq: ultra-fast personal genome analysis and interpretation.;citation_pages=966-968;citation_volume=12;citation_year=2015;citation_pmid=26258291;citation_doi=10.1038/nmeth.3505" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Danecek;citation_author=A. Auton;citation_author=G. Abecasis;citation_author=C. A. Albers;citation_author=E. Banks;citation_title=The variant call format and VCFtools.;citation_volume=27;citation_year=2011;citation_pmid=21653522;citation_doi=10.1093/bioinformatics/btr330" />
<meta name="citation_reference" content="citation_journal_title=Alcohol. Clin. Exp. Res.;citation_author=D. A. DeSantis;citation_author=P. Lee;citation_author=S. K. Doerner;citation_author=C. W. Ko;citation_author=J. H. Kawasoe;citation_title=Genetic resistance to liver fibrosis on A/J mouse chromosome 17.;citation_pages=1668-1679;citation_volume=37;citation_year=2013;citation_pmid=23763294" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. T. Eppig;citation_author=J. A. Blake;citation_author=C. J. Bult;citation_author=J. A. Kadin;citation_author=J. E. Richardson;citation_title=The Mouse Genome Database (MGD): facilitating mouse as a model for human biology and disease.;citation_pages=D726-D736;citation_volume=43;citation_year=2014;citation_pmid=25348401" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. Flint;citation_author=E. Eskin;citation_title=Genome-wide association studies in mice.;citation_pages=807-817;citation_volume=13;citation_year=2012;citation_issue=11;citation_pmid=23044826;citation_doi=10.1038/nrg3335" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. Flint;citation_author=W. Valdar;citation_author=S. Shifman;citation_author=R. Mott;citation_title=Strategies for mapping and cloning quantitative trait genes in rodents.;citation_pages=271-286;citation_volume=6;citation_year=2005;citation_issue=4;citation_pmid=15803197;citation_doi=10.1038/nrg1576" />
<meta name="citation_reference" content="citation_journal_title=Environ. Health Perspect.;citation_author=J. E. French;citation_author=D. M. Gatti;citation_author=D. L. Morgan;citation_author=G. E. Kissling;citation_author=K. R. Shockley;citation_title=Diversity outbred mice identify population-based exposure thresholds and genetic factors that influence benzene-induced genotoxicity.;citation_pages=237-245;citation_volume=123;citation_year=2014;citation_pmid=25376053" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=A. Ghazalpour;citation_author=C. D. Rau;citation_author=C. R. Farber;citation_author=B. J. Bennett;citation_author=L. D. Orozco;citation_title=Hybrid mouse diversity panel: a panel of inbred mouse strains suitable for analysis of complex genetic traits.;citation_pages=680-692;citation_volume=23;citation_year=2012;citation_issue=9-10;citation_pmid=22892838;citation_doi=10.1007/s00335-012-9411-5" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=S. Gregorova;citation_author=P. Divina;citation_author=R. Storchova;citation_author=Z. Trachtulec;citation_author=V. Fotopulosova;citation_title=Mouse consomic strains: exploiting genetic divergence between Mus m. musculus and Mus m. domesticus subspecies.;citation_volume=18;citation_year=2008;citation_pmid=18256238;citation_doi=10.1101/gr.7160508" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Protoc;citation_author=D. W. Huang;citation_author=B. T. Sherman;citation_author=R. A. Lempicki;citation_title=Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources.;citation_pages=44-57;citation_volume=4;citation_year=2009;citation_issue=1;citation_pmid=19131956;citation_doi=10.1038/nprot.2008.211" />
<meta name="citation_reference" content="citation_journal_title=Heredity (Edinb);citation_author=S. Kato;citation_author=A. Ishii;citation_author=A. Nishi;citation_author=S. Kuriki;citation_author=T. Koide;citation_title=Segregation of a QTL cluster for home-cage activity using a new mapping method based on regression analysis of congenic mouse strains.;citation_pages=416-423;citation_volume=113;citation_year=2014;citation_pmid=24781804;citation_doi=10.1038/hdy.2014.42" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=T. M. Keane;citation_author=L. Goodstadt;citation_author=P. Danecek;citation_author=M. A. White;citation_author=K. Wong;citation_title=Mouse genomic variation and its effect on phenotypes and gene regulation.;citation_pages=289-294;citation_volume=477;citation_year=2011;citation_issue=7364;citation_pmid=21921910;citation_doi=10.1038/nature10413" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_title=Linkage disequilibrium in wild mice.;citation_author=C. C. Laurie;citation_author=D. A. Nickerson;citation_author=A. D. Anderson;citation_author=B.S. Weir;citation_author=R. J. Livingston;citation_volume=3;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The sequence alignment/map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Cell Metab.;citation_author=H. Ma;citation_author=N. Marti Gutierrez;citation_author=R. Morey;citation_author=C. Van Dyken;citation_author=E. Kang;citation_title=Incompatibility between nuclear and mitochondrial genomes contributes to an interspecies reproductive barrier.;citation_pages=283-294;citation_volume=24;citation_year=2016;citation_pmid=27425585;citation_doi=10.1016/j.cmet.2016.06.012" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=W. McLaren;citation_author=B. Pritchard;citation_author=D. Rios;citation_author=Y. Chen;citation_author=P. Flicek;citation_title=Deriving the consequences of genomic variants with the Ensembl API and SNP Effect Predictor.;citation_volume=26;citation_year=2010;citation_pmid=20562413;citation_doi=10.1093/bioinformatics/btq330" />
<meta name="citation_reference" content="citation_journal_title=J. Clin. Invest.;citation_author=L. D. Mesner;citation_author=B. Ray;citation_author=Y. H. Hsu;citation_author=A. Manichaikul;citation_author=E. Lum;citation_title=Bicc1 is a genetic determinant of osteoblastogenesis and bone mineral density.;citation_pages=2736-2749;citation_volume=124;citation_year=2014;citation_pmid=24789909;citation_doi=10.1172/JCI73072" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. H. Nadeau;citation_author=J. B. Singer;citation_author=A. Matin;citation_author=E. S. Lander;citation_title=Analysing complex genetic traits with chromosome substitution strains.;citation_pages=221-225;citation_volume=24;citation_year=2000;citation_issue=3;citation_pmid=10700173;citation_doi=10.1038/73427" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=J. H. Nadeau;citation_author=J. Forejt;citation_author=T. Takada;citation_author=T. Shiroishi;citation_title=Chromosome substitution strains: gene discovery, functional analysis, and systems studies.;citation_pages=693-705;citation_volume=23;citation_year=2012;citation_issue=9-10;citation_pmid=22961226;citation_doi=10.1007/s00335-012-9426-y" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. K. Patel;citation_author=J. Mukesh;citation_title=NGS QC Toolkit: a toolkit for quality control of next generation sequencing data.;citation_pages=e30619-e30619;citation_volume=7;citation_year=2012;citation_issue=2;citation_pmid=22312429;citation_doi=10.1371/journal.pone.0030619" />
<meta name="citation_reference" content="citation_journal_title=eLife;citation_title=Insights into mammalian biology from the wild house mouse Mus musculus.;citation_author=M. Phifer-Rixey;citation_author=M. W. Nachman;citation_volume=4;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=G3;citation_author=C. D. Rau;citation_author=B. Parks;citation_author=Y. Wang;citation_author=E. Eskin;citation_author=P. Simecek;citation_title=High-density genotypes of inbred mouse strains: improved power and precision of association mapping.;citation_volume=5;citation_year=2015;citation_pmid=26224782;citation_doi=10.1534/g3.115.020784" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. Salcedo;citation_author=A. Geraldes;citation_author=M. W. Nachman;citation_title=Nucleotide Variation in Wild and Inbred Mice;citation_pages=2277-2291;citation_volume=177;citation_year=2007;citation_issue=4;citation_pmid=18073432;citation_doi=10.1534/genetics.107.079988" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=K. L. Svenson;citation_author=D. M. Gatti;citation_author=W. Valdar;citation_author=C. E. Welsh;citation_author=R. Cheng;citation_title=High-Resolution Genetic Mapping Using the Mouse Diversity Outbred Population;citation_pages=437-447;citation_volume=190;citation_year=2012;citation_issue=2;citation_pmid=22345611;citation_doi=10.1534/genetics.111.132597" />
<meta name="citation_reference" content="citation_journal_title=Eur. J. Hum. Genet.;citation_author=B. Tim;citation_author=R. K. Hastings;citation_author=G. Sirisha;citation_author=R. C. Free;citation_author=A. J. Brookes;citation_title=GWAS central: a comprehensive resource for the comparison and interrogation of genome-wide association studies.;citation_pages=949-952;citation_volume=22;citation_year=2014;citation_pmid=24301061;citation_doi=10.1038/ejhg.2013.274" />
<meta name="citation_reference" content="citation_journal_title=Parasite Immunol.;citation_author=A. K. Turner;citation_author=S. Paterson;citation_title=Wild rodents as a model to discover genes and pathways underlying natural variation in infectious disease susceptibility.;citation_pages=386-395;citation_volume=35;citation_year=2013;citation_pmid=23550923" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=W. Valdar;citation_author=L. C. Solberg;citation_author=D. Gauguier;citation_author=S. Burnett;citation_author=P. Klenerman;citation_title=Genome-wide genetic association of complex traits in heterogeneous stock mice.;citation_pages=879-887;citation_volume=38;citation_year=2006;citation_issue=8;citation_pmid=16832355;citation_doi=10.1038/ng1840" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=N. Wang;citation_author=J. M. Akey;citation_author=K. Zhang;citation_author=R. Chakraborty;citation_author=L. Jin;citation_title=Distribution of recombination crossovers and the origin of haplotype blocks: the interplay of population history, recombination, and mutation.;citation_pages=1227-1234;citation_volume=71;citation_year=2002;citation_issue=5;citation_pmid=12384857;citation_doi=10.1086/344398" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=J. Xiao;citation_author=Y. Liang;citation_author=K. Li;citation_author=Y. Zhou;citation_author=W. Cai;citation_title=A novel strategy for genetic dissection of complex traits: the population of specific chromosome substitution strains from laboratory and wild mice.;citation_pages=370-376;citation_volume=21;citation_year=2010;citation_issue=7-8;citation_pmid=20623355;citation_doi=10.1007/s00335-010-9270-x" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=B. Yalcin;citation_author=J. Flint;citation_title=Association studies in outbred mice in a new era of full-genome sequencing.;citation_pages=719-726;citation_volume=23;citation_year=2012;citation_issue=9-10;citation_pmid=22847376;citation_doi=10.1007/s00335-012-9409-z" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=H. Yang;citation_author=T. A. Bell;citation_author=G. A. Churchill;citation_author=F. Pardo-Manuel de Villena;citation_title=On the subspecific origin of the laboratory mouse.;citation_pages=1100-1107;citation_volume=39;citation_year=2007;citation_issue=9;citation_pmid=17660819;citation_doi=10.1038/ng2087" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=H. Yang;citation_author=J. R. Wang;citation_author=J. P. Didion;citation_author=R. J. Buus;citation_author=T. A. Bell;citation_title=Subspecific origin and haplotype diversity in the laboratory mouse.;citation_pages=648-655;citation_volume=43;citation_year=2011;citation_issue=7;citation_pmid=21623374;citation_doi=10.1038/ng.847" />
<meta name="citation_reference" content="citation_journal_abbrev=J Genet Genomics;citation_author=X. Zhang;citation_author=Z. Zhu;citation_author=Z. Huang;citation_author=P. Tan;citation_title=Microsatellite genotyping for four expected inbred mouse strains from KM mice.;citation_pages=214-222;citation_volume=34;citation_year=2007;citation_issue=3;citation_pmid=17498618;citation_doi=10.1016/S1673-8527(07)60022-8" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genome Sequencing of Chromosome 1 Substitution Lines Derived from Chinese Wild Mice Revealed a Unique Resource for Genetic Studies of Complex Traits" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11.cover-source.jpg" />
<meta name="twitter:description" content="Mouse resources such as Collaborative Cross, outbred stocks, Hybrid Mouse Diversity Panel, and chromosome substitution strains have been instrumental to many progresses in the studies of complex traits genetics. We have established a population of chromosome 1 (Chr 1) substitution lines (C1SLs) in which donor chromosomes were derived from Chinese wild mice. Genome sequencing of 18 lines of this population showed that Chr 1 had been replaced by the donor chromosome. About 4.5 million unique single nucleotide polymorphisms and indels were discovered on Chr 1, of which 1.3 million were novel. Compared with sequenced classical inbred strains, Chr 1 of each C1SL had fivefold more variants, and more loss of function and potentially regulatory variants. Further haplotype analysis showed that the donor chromosome accumulated more historical recombination events, with the largest haplotype block being only 100 kb, and about 57% of the blocks were &lt;1 kb. Subspecies origin analysis showed that these chromosomes had a mosaic genome structure that dominantly originated from Mus musculus musculus and M. m. castaneus subspecies, except for the C57BL/6J-Chr1KM line from M. m. domesticus . In addition, phenotyping four of these lines on blood biochemistry suggested that there were substantial phenotypic variations among our lines, especially line C57BL/6J-Chr1HZ and donor strain C57BL/6J. Further gene ontology enrichment revealed that the differentially expressed genes among liver-expressed genes between C57BL/6J and C57BL/6J-Chr1HZ were enriched in lipid metabolism biological processes. All these characteristics enable C1SLs to be a unique resource for identifying and fine mapping quantitative trait loci on mouse Chr 1, and carrying out systems genetics studies of complex traits." />
<meta name="og-title" property="og:title" content="Genome Sequencing of Chromosome 1 Substitution Lines Derived from Chinese Wild Mice Revealed a Unique Resource for Genetic Studies of Complex Traits" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/11/3571" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Mouse resources such as Collaborative Cross, outbred stocks, Hybrid Mouse Diversity Panel, and chromosome substitution strains have been instrumental to many progresses in the studies of complex traits genetics. We have established a population of chromosome 1 (Chr 1) substitution lines (C1SLs) in which donor chromosomes were derived from Chinese wild mice. Genome sequencing of 18 lines of this population showed that Chr 1 had been replaced by the donor chromosome. About 4.5 million unique single nucleotide polymorphisms and indels were discovered on Chr 1, of which 1.3 million were novel. Compared with sequenced classical inbred strains, Chr 1 of each C1SL had fivefold more variants, and more loss of function and potentially regulatory variants. Further haplotype analysis showed that the donor chromosome accumulated more historical recombination events, with the largest haplotype block being only 100 kb, and about 57% of the blocks were &lt;1 kb. Subspecies origin analysis showed that these chromosomes had a mosaic genome structure that dominantly originated from Mus musculus musculus and M. m. castaneus subspecies, except for the C57BL/6J-Chr1KM line from M. m. domesticus . In addition, phenotyping four of these lines on blood biochemistry suggested that there were substantial phenotypic variations among our lines, especially line C57BL/6J-Chr1HZ and donor strain C57BL/6J. Further gene ontology enrichment revealed that the differentially expressed genes among liver-expressed genes between C57BL/6J and C57BL/6J-Chr1HZ were enriched in lipid metabolism biological processes. All these characteristics enable C1SLs to be a unique resource for identifying and fine mapping quantitative trait loci on mouse Chr 1, and carrying out systems genetics studies of complex traits." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11.cover-source.jpg" />
    <title>Genome Sequencing of Chromosome 1 Substitution Lines Derived from Chinese Wild Mice Revealed a Unique Resource for Genetic Studies of Complex Traits | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"qVt1Jo32n_upMEnjq9sXZbXC9soCbR9eIMQO0IBt3Xk","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460453","apath":"\/ggg\/6\/11\/3571.atom","pisa":"ggg;6\/11\/3571","ac":{"\/ggg\/6\/11\/3571.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/11\/3571.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/11\/3571"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/11\/3571"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/11\/3571.supplemental":true,"\/content\/6\/11\/3571":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022CGNacucJooRVHIU8YnjrgCf\\\\\\\/m9R7FJRWvopg01BHvMz8jGalIh\\\\\\\/C+FMAayfYI6eb6mDxK9NdUI9XDzi3g69Ya\\\\\\\/NRSE9vQPr68EPKmZsmK1yZqj+eJCetBeuqWVoGCVVY6kxzRC7E6XfhEJnzS92Wv6ab2ozjZkVs4EqUfkJt5qhn44HVRMFQv2koFV57DF0MKdv7jYh+b8+nnohRJ+rxv3723OSJhcpp6EH3g13gStnE\\\\\\\/OeZsp3nb4nblHvG+0PnBC097LecLpLXiwkn1V2oukRMAVTC0ksYrcSkSCXl5lKKC7UvcSvTAD6+FS08ji05KcOEJtnK2RrsHMtxQQwZMQSZYtgS4\\\\\\\/TPPrDE4J84LY3ZuryseRKimBo8YD9IDlEQT+k6\\\\\\\/GfsDaYz1+P2An79Ujpug1psWRpBHrrLAUqjCKBrUZO5xgLSQAHfWe6aoHvQqwuVvlArBCw1QIZAr4VtbVb8wu7kFKbmIhJfrhQ3iC2KNK1HXfGyS8dEbhytmF\\\\\\\/B8Fr9JweAATAyvWrbCk5Ba9FdkV7X1BF3FhbDA3AnZudZqlbnVAsXxjRyT9uZ5yXpu7uan8HaeusAIbUByRJA1u4ThMbqdDpsQMmtM7a1TBOeEGhkf9BEvvE0QEi3cEVF6RKwMjlo+A95PariOeoDYaAJb4JhAl34RaJV6UxTmp5qtSmqdDk8TjRpC3CrYnZWLgPpR02lGC83YbVmieQo4FfAWSjWlY5ejJI9zX33L\\\\\\\/FjvrXoNWP\\\\\\\/cxN43F3ioXKBLHhWdwe8xqbGMr2k7mftKLKIeURKUAlDX30vc\\\\\\\/xkZ4IMSzAntypVg4HNR\\\\\\\/rOdT+7aDB\\\\\\\/gf4p9wgofEj7kIDzYMCXXrgy+RrFJi+63O7sz99h4KB26tAvyHq0U79IkO7hq+xdvJX3ZfhvURZvn2sIEq89uxXtwZOLm1IQO0LCWtn1QgYQ0Hm8tQid1GDR+ihtrM6VyAZ4n2tIfxqBTlF0t4H36PKW2E9FLnPImS9zLzHAcvxNVhWE0i06xAlTVwA\\\\\\\/ptZeu2HG9k+UrChLo5OGpno2dWn1yNk\\\\\\\/EvrQGt6UL5RN9N4pyPyWHEf\\\\\\\/YYx+mzOaecy2KruPrNSnRsG3B3YJqWcITc9VE+RBqAZofeWuiL1dR3IJ+2XiA05GN8whzuU\\\\\\\/msJ8dTlleRMb2EBQkM9qo9J\\\\\\\/FPTlEnwDLSYhtNGTi9MNUp+760\\\\\\\/1UQZcck8K9xzrDfyjObpdD264FPhVx4viveO3oJJPs3Z1FlspUdkzoY7QQb3+tnlHKLYbe2v\\\\\\\/TJLO\\\u0022,\\\u0022iv\\\u0022:\\\u0022YvZomyIr1yQLCIdSSQg\\\\\\\/8Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002217ba07a5242966cb0b11f41e368bdba08ef4b540f3cbc07d35730c778ca5454e\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022duRPpzJhX+xpS6\\\\\\\/lGwnVOCHCghdF6DNrFiDaVeJBNevQ1J\\\\\\\/fshDNMff\\\\\\\/NZio7\\\\\\\/vllWy19c19Y8zQ+PWieOmm1HMFn2oifmyO\\\\\\\/PUFGSxDorEzCOfP3rptsz8PIigbXn+A3tLcI7QjbWoJHV6JxmBT3CBVgpVqx9hY3oqUZ7mc2Zrwuc7EGBK9t0Q8k\\\\\\\/+cQTeN1VQLi9O6NR7qDVurWUkACZlZRDSkr6fx3e518DE5VqGxlNKqMebX1FhLU7Mv+WsyYlc9WX3IeZ9oeWhzIseullKRBYrAztY5lOczeifYyc8PAd3KZ2wafCD6E56888qBRqYALWydmu+46zqxJtM+N5Jchf43fTb2XksDTKrBtdIyw71hZoejujG2MktJhj1Kg2iqDKNAHHtIl4\\\\\\\/nrl8123NCNM\\\\\\\/vD9XfhuA4elyPUKSC9rGFEGhVErbxlUngl3flfPbGXRvzkjiO8QO6bUGmgFbc8n3JmR1ev86UO5vK4DPGqoEp1Xf\\\\\\\/KcLiXReLHBlwb60fqnk9mniaaeHxMxGVAQnoILWoq2IfERb2t1YKzykQx0tLtX7S0oP7pAapYBZ7QE3oQOTSiuwkKFij9NREsbJgi6gzCjfBA91ckQIjZ5CZFxzxxFNGFCi6rT26nZtIUcALgxYRYN7ZlLGDZmtCmHHZg7e7ecF7dVfV8CPhW594nJUtfPWcgTxzQCqkKBWkTw578rRWTQNThoyooTFEfY+uFe+xuYeddrErMgXsm574FybKi5tkd8B8U3QUmjPU8aLW7ONsTyS05yKcrZo4EqEZLDEwj\\\\\\\/e4xRwQ0yvkNN5NA+W7XPd7Mu01qww96JJEDWhV2\\\\\\\/Ldw6CP1dyhSN4whHTT\\\\\\\/ht8WrxwwL3qEIg9U+lHiQRdtGn\\\\\\\/qni8CbazMFxY1i0HS0Y5EoryIial9ygpxOfOqHynB2UXOKAwjwhfNeqym3nMp0JW16Sknm8OhuJ7I\\\\\\\/Z9Rjm5X6rgT+aZfkKOT90\\\\\\\/R9v+WZoYWg3TX\\\\\\\/OFOa2IUFYI8K4nV8rICWTvxLqDv86BzDEvm9NQj3M1lUzjc1bNscZ30D6NNe+SE1zIej4r35n1r6\\\\\\\/pBnuA+r6AjTYhnZbtUXAzWL2MidV0Neqlb0H9DbahTdKV4JacKvp8gAiCnQIU1Au6MrnVsUmAP4LhkfUbdrTZE6Cxsxv2bwSpmzbeBTBFMRj4zJz8Zv3\\\\\\\/eqT65RcHUstgewZTAk+Y7m\\\\\\\/cEGv5CXvm8YFNeCLsvkTTAfBZnaBbHv74q05opf2z00yMa4voqcwSG6ERRp8XLQ2Eek83ouqLxWawDBYxXo7EY8Q2BE74PSEQ\\\\\\\/ckru8b0AxF\\\\\\\/O2DDRVXcky6SKcbV1dAL1EmwELFhlH8U1Nqtch0YEcKUKNmediqiNdVV2q4Z29KfBFH3hzaCV91+heL3\\\\\\\/+bav+0ZaxPa4QqCtsUIEMWRJ+WGI6Gg6MQdXAdHmKhCn5bPKql\\\\\\\/tAvsDr2pU3wSZdQrzdjG5XXH99aH6ZWktYY5gLMyvcc1V4BPIkIJ9d+JoMCnYCFYKcx2meay6aCggF6iZsHqCZNiS+NyR+rDNGDn59wGB+9rZKuQymsIoGpfnAJ2DE0VqtwTsoDCG4IGFmf7lCRPKJuZvBpqOljrPamJALbVw+K4vu+hRhtyVotWFzys1wpeXkc5\\\\\\\/MBG8HrlsmtJ9lojfbtZGZiJZQ\\\\\\\/\\\\\\\/H0Vv7MgaT\\\\\\\/RYmWreVKNu+oaIHnTB9lPeWT0l6mFkTaDOlJuocO\\\\\\\/Scq3iwaVf\\\\\\\/qzFdTKz1ocFv4hq\\\\\\\/9H\\\\\\\/C7D0o7vAwA7RGtt3NDeUG\\\\\\\/35WXQs2neOiseV8yynsi4BzeLE38XM+s1y4Pb+ym\\\\\\\/zVl7be+URC3E7dJ4II5R8XsN7se2G2gF81CvIK9dbTpv6d+tDohIAizguEhwWrmS0YvH7V8TWfi2yXjtddSMQ+dlE0o4n5tCvwftaqGleNVru7CYVVCBrMq7MWwLVYUwR1wvgSLQTu3+b+RtGcWIhThd1cuyp6KoiUv2mXCyai\\\\\\\/m66EsLQOytv1ePooWEEJSLUihArU3Q8ZY9hN1RtKhhMOzdQENXdUaLAoVArIDDo2r8AF7Fs7PIS4yVQ45SAVQrhJVs70jPIqysbtWeHgdzo5LuSxCIqzzlxdVLmQYvJOsaKsRxlPwwF7z\\\\\\\/w5lxcGmkGnsRy7vbdrCyKwErevVkFYz\\\\\\\/AwY\\\\\\\/qxe5Qe\\\\\\\/U0fPbtseP20ZYJ5xJmjkotaYG8f58\\\\\\\/ec48T7qxdvW5gBizsUy9U7Y4b8zFXfcXgGa1DVU5vDpU4H5Sz+HT7wp+wJubiAghrteYiKZDq2ylC\\\\\\\/FeWK0lVVLHUNGIxC1AYSm8iknBChcQ0apXJu0IAQK9tu3l2N3er8NG40VWwGvYfQJPm0lRjoP+2H2FiXdQh2TYqjo28r\\\\\\\/XvYs6Gde1djsBpGv3kQx0F756Uzs\\\\\\\/as8rJ293t+Y5RfWcmq8o1JfPSpGl6WaRGUZzEHSEGnNdUwTuAdEtZ\\\\\\\/28vwBfUir70FNNaVoTBGJ2FN3JHKb2X+BYgrgzROPDsHhPck\\\\\\\/uXICAex87dOVGsfydCu\\\\\\\/atrfqHGv6ADqA1B7ymSusNyx93pPgJjxlE3arAVWxF4+H0SSBUJh\\\\\\\/R4W8fHEmTc7R1K5jD9eLH6DssX\\\\\\\/fjgNLTrC8fBGe4DZcPsQJ+vCbTyaJIfD1UPJSjmmQOE8Ew34bE8wjGQDi4y2opxdGcpaJkL67m5LNJVPipSeYv\\\\\\\/vAGfxwBla2KO6iemq98hoqtXKEiXZPuk1WrVVMqf3Lo+e+PNrv+RIM21vHTL7xruewHLLYc1kUEtgt+Lp3fQ6x\\\\\\\/DzCOPDWX45u8YdZg1gEkzTbFR5yhnMFbPnRRXWvHe\\\\\\\/T2WpMmpFLl90B+TCJSumPR438JfovzyiUa\\\\\\\/ogH5M\\\\\\\/Y+qOeersF5yskuQhS5RmQXkUqIkaUoKfsrt+sPwFPamZCzo7KhcecvMbUq3U9isfVP7jM1NbxHO2JJuixXsI0j7PSB1n1487yhvgoN+Uy2gXXs9xCpmoIH\\\\\\\/s4qIsinXzxhWiks+v6Ut\\\\\\\/MA\\\\\\\/YVkIdkgCSZyTY+df4ZqF3TN9+L\\\\\\\/A6rvQoK3wtUs8lfgJF0k4ythtCGLIz\\\\\\\/KnlrNvS9d7PMlUr1jBBJeBdeszWrOtlM9xHVpYgMoF58t16Xrzc8Fnpm22QfHhbvB1svOthxKzpB5jJu2IkS3ePiIu0K5ye8k+eeVfeVtl7nmaT4t+vJMStnanVgM+N27d9Pzx+X3XyytHExvexYQjHVYRdi64l2HPvTlvjSr+wrV0zC7g9A7LfyDayiJes8JGCNH9rDXeBvCPTls3bnYS3OU+Rye1SA4XFVmVGyZ4akTpl8FzNVT3NnKffwjkQVwHqdatc8MRf+TXbddcde3War8Ek28d8BZ0sEKdSBHJQv1eBd+HvVuW+W\\\\\\\/kJSZp3WCYp8131y0SGnUizjjvs3rxWcqbXpIuyJpJia4x20ppK\\\\\\\/jaHwvduSzecrs80jHsiG7Rkx\\\\\\\/FM0jV242xQIEYDTBGnO\\\\\\\/4PANofqWMQmJTFi2cZhHdAHuVXFpsoW6igyPirKOa9I5U7UGGsCmn0vcNWKsNm51b6HOEiMSA9jvI2C3nRlgl4bilf0R9L\\\\\\\/LIFm42IQcFQfpAzZYwc\\\\\\\/n4EXm1C1Bl3EUyOcuyIsHpkmXtaNI9F2ake1fHhH\\\\\\\/zsBoynStgjiT5kBqVDSeeuwRUCzcYqGdJvd30tTi4cziJp8FjC0hI9gicEvLDovF6jo2A0aoukXKJdIiZByLIXQAtA3qWjW9uAXlCcPFNeGnLTN\\\\\\\/bAQzm2H545bUhUReM+TNkaiWbWbkWi1gzgTLQ67prTBjvNagqCocey2OTg9DkV3CD+azgkTQAUzYP7C\\\\\\\/s98KTe2cy8VFeD\\\\\\\/cJnQcjQ9lgMSPq42LSme9KLU+MnNNsRsypiNTCf5UzD8y\\\\\\\/h2QQVFih1+L3OMhyNHpQqpYz6ibMadPyK8HNDvIEuB\\\\\\\/RNGn8nVM81a6z\\\\\\\/kYzcoBILrCk4oVdEhk5T9al4Rxva+E51ch42c3+3M3HQBxReGt\\\\\\\/+pfeWd7SWs1LgMaNZ2SwrXI4\\\\\\\/B27GiFQy5m3Jf\\\\\\\/kVUMlpSGfDpiXQGfpB1ERT\\\\\\\/e9YivzPng8VJgg8sBIpqTn7tqGIzuSwaISN6hG9E+3CE\\\\\\\/s9mxnTJkrTsSqe6QtGk5DtwQL21c4kw9HC9I2lm37y6pwKx47WTDsLxcVwneI+hUUHMCAAeS5hV3lJ7mrhR7r7beznJyPFpSbV8AbbzIySm+criUMY3DmePOIBQlYZsZRJAPugIz+ybBkuY\\\\\\\/GlCbYds8E2aZkGtRKT31YgSg4S4dO0TABMr\\\\\\\/abFqkcuIsGfzzCCaH3Iln8Ty9kq8AV1aIdnZBzt65SKvcCV7B9XvPCpModDavZu\\\\\\\/gH5vEL\\\\\\\/OqhAyx9EvfuWvCYklxb+Mar59\\\\\\\/aBnM9vWy48539iK\\\\\\\/fXDfjBurjwbX0yBmpPKeZSJ6BiyYcorL\\\\\\\/QT77LW87IEI4Fyuuy0X5rbhNok0YQjRhuINcIxCnW5kfgX8ah7AUU+L+rk2Bj+7JTbbDMTsx9HoZWshqVmxLQMq8GehbcgzH46raspJXHMbBT7XLXL\\\\\\\/8jlSRCyMA\\\\\\\/iDEnmzXo++rl6oHSffEHkaRNjQTRuJ\\\\\\\/mvAqB2f9oXCRXceJq5seWXPnX2IlOHkj\\\\\\\/c1QYbbbazCprI\\\\\\\/UZAqbNzK3oJ4+yYoHFa\\\\\\\/Ctin7gRC\\\\\\\/6Wk5LN6TsZKJCVvFzsfjwSNWisjRVLvTJUL4VXbM1wsguWO6168oqTP787Ct3m\\\\\\\/2+g3jRqhFptP4bkyr\\\\\\\/dYdRy2D5TCUsXUs9S0\\\\\\\/Rf+nEkMETFsvVy0gP7TEPIJTstFU4mMC4PPJlheW\\\\\\\/2iT\\\\\\\/iE3\\\\\\\/bUAlbKQNMCiFFZcgBwLfoEYe1QwHmjGdyL39s2DzKyCuq1SfH8d69vTIBltZ1GvM1UNmB5agtaBahok3nM2tuEg6M\\\\\\\/5DIlLddbUqsyr50SafeJeqcG2ZUohsiUpdeukZMg7sxt\\\\\\\/vWdhXbnWwqWAHvfD\\\\\\\/Hq2gbzYY2llu2FTnkLb+K6mFUHgswYzXQFTxwdlLUhhW6ayckywUKvc6wMB0t8+u3Txk0SSgT9YOBFy0x6tnrHdwNyC0Pu92wZ4S8XkVOHmXwtW2djkhfPhnqjnSTFnkc\\\\\\\/zh9oNWMwgqk4fP+6SGYZVZywrXq4EmH+v0U4hR4io6PkOjGqCYimzYAJN0ltgAn7w\\\\\\\/VipXm9jn3ELuSbh488E72zUylt8oaExuphlE+bCoj4X0Sb5VxON3hticDn1ilZN\\\\\\\/BYV9NpRSwJuolv0V2PP08UHtcHtV8lpOFpTTLAXZAdbYMlz1UOn4oUE\\\\\\\/15Re4J3wvO7+dRb0ub6WL+nW+K5SATbsbi7mUTTHExrZADN54pPv\\\\\\\/hCROzxC21xSLpLXsUI1cm\\\\\\\/jS8PnjLkdslZv2b4r6cVqNwqJoCTKaf+1fxvPLIVNH2D2lMMzpzh+2q0cEpYFhan31P50Z3z6fMnJ4Ki1qmOxFjDbDokD6EyEYCgZ0z7bAF4h2OH05v1hSE2R0BJEXgyfBleGR9WKsrgwPLNPQWxRdXv0PxY1yRNEookRXkyIt00QvJ5gGSv6SRvDH0CplF3cPlrGLs86a1z+NFgLgFnzlbtV2vet4Ep6BwdAhd6XBtNz2Vv9mzqS4PkS4rVgXoeDcXsMDBOZKbzrbgUuC2nEEHq1P0OuN2lxPapIKKma\\\\\\\/70KJ4uwVb4a+BTlLb62wDyrSlY60bHNqCeOrGUgBq\\\\\\\/6FYefFeuhqVFCM0\\\\\\\/S8wjESQHDhrNN0js6wk0NnqQKrfUmXa\\\\\\\/bSx9seHzWfEjF\\\\\\\/cGHTO2zrqWFNjF8U9aYigFoZ\\\\\\\/krMAUcffhCe07VApxG7Zsqhh7itkoQqHk9ggqa9XW\\\\\\\/wFuuoiwZFPFRxhQ6lwew9efNemAxzieB+7P5ZZ4H5R+crhPBZU4C4Trtl3z1rU6EXl5v6hV17R6VAbe0GqyS4929\\\\\\\/uupvX96Hg07V3POzZWsPqOrRP1qHBmn9WG\\\\\\\/se65QzlIiI3Sfq2prjEcvrX6xJomEf0ZblLvgz0fEsxWavdnWxMPQeVW9H33mAuuFFUDLJY+t0XVM5YDUZNKZFAMjCnzQ85vPZBZIvtTBRH1U2HdNDyIjUWwa0LvDju\\\\\\\/2b7S4XvbhFHrlbgpIjAljruYf66LGcfwtAvOZKAjh1lE78Ffc04dZSEiXUH+Fs+0qumV4XPhFliytGpHNQvzeo4TGs+DZMChqoWhknb8aynVmpG+b1KH1+zdbAy8ssbofvjHg5NXliIy+gJpQOfaoM4SEQf1RyN8xxWfbeKUQVNwwWfEbHHL2+5o3AEk32orT5a6NQcaijvxT44ZVj6RD1EshhrCdCEcb3E4EroSc0EsVdWTZAoT7Lmsh2PGCQhFsvW0rtSH4UXQX7oSe1ahGxq6izA3ufQ9yLbA7xHk8P+GAS3XKGv\\\\\\\/irfRV9\\\\\\\/t7dbH1mCqd38Dijx+XiUNnKzK+T5QZg2tzscMQGj4WO7Uqh4NpkcLiD809zPn6JgdUa\\\\\\\/q1vbsEMlX4SY4GoCRUQc3ATPjfjgy3qSyTsOp9TKq62N0Z5NNLfCmD9TNPV4lSLR6fwfjuTUbzHeS7phPduKJ1Sh4LGwkV4HaEHFmZz1TaQRLk+mpVTjrjThdjwRsINd2dupfVIB5eCB8e27PY3FSRiCYfS9MdeSxGGCZDvD8oDdrwk8jYpdmzo9dkZN838fMe1rQg\\\\\\\/rCoA3XBDY\\\\\\\/1H4rDeyCxtEuAidyLjq8Bn4ksCjSHEUjIlD4D0I48wpW45wdkbqElE+5bi6o+vepsybwDIZfk4csJlwbKnpxfh6OqWCXWtrVblYJlcYskrhEgu1YvqIdr3QIfxMcar3CY2tO5DGRqrtfO+ga\\\\\\\/6TDN3Yxqt3Jsf6K5oO8ror9tZFJfIoE\\\\\\\/gkOkqOoIpmaVsUQ5vKD9BZ968WngPNJMBOsfazO1uUrmt0zZU4PXmlkc7Q23tedGtCwEItFtIIHW2uR5awGF\\\\\\\/kMIzaB1DhfOHM7fYagqwna2nkP6Ya5ij\\\\\\\/QOkUOnXjnpQzmXJu0NSASapWdnf3Z7Lt9WywvsGqZ\\\\\\\/CahaKuCw2MD3XA6md6NXCUlGVXhmQxgVX731X5uQwodYR63gf1PQlx0wUO8saA7NlPEwLHRZaGhIXsNz9HrLWZD0Omvmsda77\\\\\\\/Kd6Hb7zbkD4h0URE\\\\\\\/77lagXAND0rtpx2wD1n9jHZUvsvwFzz368f7+UV3BdWePeBBU1VoaZwqr9rk7I7BscUHB6SVTH3DtzWM9oa7iNnuZVlak\\\\\\\/WYeQcpWbxkm+gYtQCo+z8Pl2Ygr6vO7s989ERBVrYl6ombzi9Hu6UIkh8G5TAQLGd8STyeKTOxDkrQq6FScBs+P8TZvyvejF3I9lRrv0iG99yj2MdckXDCRX5prSHTnJaxHOyPzsmNgUn98Q2U1nHDLMM++Hv7sspwaAYzbY816EXoL70MaNfMHEvw9juE6ABzEWa0JpOb6X8ZbPveQ5HrS2p6OS+7tbo9gBT1qeWJrvAQb0PI2p6dE4kxtXD5GqgXbTTJA3aSEF4tLF4ARtR9JZWI4yxtEdLTKs69KEdwo5k6srOPvRzu2vcIeQEph+24i1UBWlUJdJexyouI+7ZCGH8QKCMpt1Dd9niR+MgO3VbFZSH1OBxE+FL5QRdXS1nzTNpgvgX5Bv+gcJzPujdh9ox0x0r0jXZENKEBuAOaZK7MbaWAk1+oXyURtVW16GkwotQ3NvVtX7L\\\\\\\/shKSL0Z+dBt8S1QDS+i5UoC\\\\\\\/iGHjT4jkO2yC7IvKQluTfcmEQnnFi94yI6qcjxLaaC1Sepbvr5oQJySst\\\\\\\/Uhcnol8hgaR3rf52aNctfO2tSyFH7qAD0spwO3YdZwbPeYupDHzwghx8+W3gcW8dwZs+iMX1H6FIqbdC19oS8kMSnLrlO5MUR1XYhvC0rwI0vl6TBoyqx6USfZPpPOrIYCJlGSmF2sr1WIPceJfBF5O\\\\\\\/kz78hQWle9GWTeRQsO4ioKT0f5GgA0xlBRLi68gdIzFup5Tm0CMVoHkbCwF8fmAio2AekeBiKgWtD8RLz6nTGyUpaSsbivRFqCx3DQjrCgQIiNBTNwkEr5eEYaC\\\\\\\/ptsD+ZfNVFOOjRDeWw\\\\\\\/BsW50O0j4iABfypU\\\\\\\/3W3UmYSXkFENK\\\\\\\/8on+Y3LWSXjy9MnlEm6sFIkLes5n+1OBguJMm89747sachkNBmXLlro5MlTXycgrpWFEEmdCBSPS4CJrMcMoDqPNY+rtEUIqqHjp9DwhNRjEzOSF2IbS2OrCO51mHLt+G6yvfQo23qqbQ\\\\\\\/+vUb3oSZdhXOt+PP3Pdrs5DusVwe\\\\\\\/oXm+PLwa6x7Ze2n3o9uvUZz2gMDXafB47H8HU1fG3y4SKvt2MDnje4dR3yg4ii41wJ4sRiMVhdHRPvZPgX1TMVyAGUkuMbq+8+myImvBsJ4apG1nW+GuilOxACoxSiWsPK9cYYCfMBbLGxkUp5td9h0tIZrNOjOHHeTshV85g\\\\\\\/KQ9WF5b5nYfOZMMdx3RFnjHZPvKlJqNXkwsZNrlz9ov0F37eoejCEFcglIykKh1AgjGulklgMgXDnJ77Bn9lmM2nffLnmpaT7QmBv64vZoTl5lbXyJ3++Mp3mRmNoK0oeDWOCp209RF\\\\\\\/P0g2e1Gt6HbUgPdIZ5PQt6WvL3OZx+UQv+q94CkruCKr\\\\\\\/THLbfORWIAfn5+QNCBr511hDSTjqfds7A\\\\\\\/8B8hH6MzU4jSiAtJKehKRkZGmhFufm7bY8sTIqNzXAkPHCy3EVtsjeAehGqn+NokSrNnrQJosKgmg5JS9wD\\\\\\\/EbeSD6ELPDZxBasl5+jVeQGq\\\\\\\/dvjcc8m4mkIML2NylQNkwsOIn3RH+MxPpfw6lg73FQrk6OvcCby07e33aqaJRrNYUhN3N7+HMN9Z1N82YrnbamNhPLGd+vHVNZdbHO74cUBjrhStDj03fMNZ6lJU9P3dXXKP2njnH2Z\\\\\\\/oKJfSrUfCFZZODDHqzLI\\\\\\\/FOGfiTeoD4aU1dA7kXq1BnXat3mjB05ZADIFVgJcniJL4blvt7Ok8aApT5CNckSm0WmkF8jX\\\\\\\/8Eux1YiXgTZBCrmGbzFm8Simo2Qw6sJyiC5XYcnQWwF1uLx7aTvMs7zGxX\\\\\\\/7rrqgLkNqYcmpHMvCvghqmtbaBlj2drRlWUiy74oDU53x6b7ltlipI+p3A\\\\\\\/HitbDXFV\\\\\\\/2+CcLkxq2HWgxz\\\\\\\/cn97+PKr5HlqrC87TtRUXXb7cXdBoAe7yifastasOn+zL+oBEf22sXPJ6pj\\\\\\\/TblJ\\\\\\\/miZXeC88bPQr9xXBcMCvfHBFdo2DtIDyFbqgzZtcpLPvEh8sT+XzAgcxne0ea7YsO7K4veXsGdguVVVMiZTiTDR9\\\\\\\/UR\\\\\\\/aUdKlBmPHMHTljv3aNnkogvdydBuVWrvwB71mBgc+4cyXE9cQMxEtPnyNXhrcm8\\\\\\\/DBrYYzKcCHc0ahe+3S0zmSddB5vW+V15M+Nx1IcqimnNPSbjokiHBddJk6OVh2w4itrbyXblsP14bTV6o0vpTfUmBCXwPQ8ZNd5T\\\\\\\/t8jjGFeQ2chRX81vC2THeo\\\\\\\/DPf6xUHnqUDkBpoS2CzIvn\\\\\\\/O9PVWIwyGdtpwjBjckrrhuCmNWfbzMn3yHjVgConY2nGxrlRtJGustGfYJEeHmJrLOQg\\\\\\\/JZePrp5izxLfRwYECnhEnzoZBCTgMqX1iiaM6rL+LLgEhTeBnaV4i\\\\\\\/P8m9h2xfE2twpwrbWJ7q1gc8j9GnbWoGuVXxIWkRZovZ7GqFVsBApf2B36MMvYaK3aLpKH2OJAH3feAk1azEbaZ42GWZOuoBRXbOx37jgGQnsm4NzxnWtAU8BtVbwI8QAhqLogw8wZ\\\\\\\/3gtOjkqxNrWZfOV6totIl4sQJSapRsxUjeLpIjrM8P\\\\\\\/IQTK1OIxNdXNEMXGxrR9kEkL0GdjEKJD43FVSZV5I+YWnvdXAXVwYSRHP\\\\\\\/kSdJiFObaiTplLUepEMwtmpzX0cm26\\\\\\\/tiSNPLBfpbqt0FBaP6E+Sk+3yVdDSUXPl4fg+U4ssWOuYXhGH5QUPTjDEyp0oSCJectYPp50ij1fT+TZ\\\\\\\/2EY9797r4cOxTbZbGXuktZ2nTrV8OyQ4U69gZLR58ViS2WWY8qQC\\\\\\\/Ytr2TuOeJsp2UpGl7wljB5xTY\\\\\\\/nZyfZhPGHqYhqdmF1EWwy2wH7LobBK+5RscaR8dCiDXJ7Z4K8w\\\\\\\/SpplMlYOUPyxlY6fv2sVFs32DOrnedV+IKoN\\\\\\\/sj8TGZha+rGdzo65jbaKR8DC64kX3i2lL+JoQFZlXvkbpGRNhZAK+nJ2JjCW4m0KqGwgqoj4HzvkfNQ47zz55nTXvn3jQeHPF88anOzoJGA642GJdTCD3X7J5PS7cH\\\\\\\/YKVxjCPp6duojIv7Q40VQ5CmU0bLum9i0NcvXCFxZnJKHTFCw042LN6OqJld2IYWtZUKX0nw++gYoBRbIvrdtGRgHwSFGvSh6FiNOodgAarY8lPFpj5ZeoHopFrnwUbZXUhDbrR2YDB\\\\\\\/6c4tW0EnzhSERl1QTy4JYOTvKcQN5O33gQBWhPPYiNhLNmAoniy4g9ncWF8YrB4lQbtXpW90Akh0SXYh64nfZxgJTKDjZg2GxE\\\\\\\/5tWS6YuXaZSdceowjU4R5jmSRUI9X5m\\\\\\\/5yvRCxge7walxrCaRiFQmpoCsZyj09mvk69CbtvTLdVTY7V4oxEfAPqD4wvrlfKMbdj8s2KBFVkLUzHzjWyUllrlDF17PjcwbqGY\\\\\\\/clPUj7fRNoBYd5ek7V25pIfBBHqixELdg00hOBaLY37bTHU3Q70Hof50KjhCWlIyllGniz1KVHYaSBkF3xKmsXcUPiW3DrsIWx7LecXakDY5w79sjcU3ps1edJ2MFbsXRr2OoUfac6PzX3E8Q0pcYP1ZLNahrT064MXMbvV+Tq6Mt+mGZSKIZqFAHq6JzE50SC4LTvBBQs\\\\\\\/Lk+xsgJcS0x7C1NFGRnhP04YTKZf0EXLgyiH3dRGK6WBl1yA0BsYgO3KbsT2iKVpVt1PXCWtkDKjCruW2Q1cEiytwtAVapBAeCdbge1T1tUX2mou4hP30VUj+hcfuDvzrukspF\\\\\\\/wJ+Ypg1vJGOlWowEz3gYX6hxADG1KROVhUYI9sOI1HN+uWfl2dpC3bGULQdDCl55IHJ3IIJ\\\\\\\/pilfGUFuBx889gKIbyZu03WDF06VnpFwDDluD0XymOV3QQqJyqEQW0wAjKCOI00RUyuhVB2yKIIvWWINULzTO1caDFEPZlJTM3MuTxdbMPVAH1gauj26\\\\\\\/GrHijFEI5VGgO0YhDlbmshUys1EYhWTn88hEeGHY9cZuaCtTpYukzVK+CNBmho2+KqVEASTr0QD9hJfeRlH57yKeWtNvhRGGoimZe4GTwRqoOmvuBBE7xHjiCU\\\\\\\/+QQv4G5Fk43+67rJB2KV4cQU9aC7B3ozNjwkhJI1ncWYByMj9cmuRIoHwsJMmZNbWKQ3VVQaAXuyhG87HWqFFSg8TgtCmFKkbWMmd65Fdd5F6XrP63Yk55oDuFeBCpJZtRtEfhW6xEgWDVzzirPPfBkUPI4dWXxxsIin97o+Kk\\\\\\\/B77ke0SrMIEyN\\\\\\\/zZhj2bkqbJHnELOW84j0FwknYwjEn2oF6nsUiMyvE0k0JEYa1AN8zK2vzrLPlsa0uZPvkFuWTC4vLqwNRjsWhKt+c1x0wQe5shBfZ0ovMWELru5ZXOYeJm1m4MTZRNdMdgCrO8tN\\\\\\\/qb6IvLtyeGExsycemTy9vCepHdAtMoS49N2GvdJySLMmSTzrQc8RxZmUnpox7XPc6zMckwNLf5zd63kCdR\\\\\\\/3yVs1xngty338XztDHJ\\\\\\\/hi1ok8h1THJLEFAJUP+zaZPKxenuJaK3cqhLUNxgYD3cgTcWibwHjdbdat09LhkwJ48e4mlYzHLsF\\\\\\\/ZLEnVmo7pqpkIo5r61uaPVKuuvDTdl+78c4qqHDPBPYtw60gNNSdmBYfqlm4vY9+tGpW5CU++3KGE8a8XOBwfMXJdL+aCtdwK\\\\\\\/n1oYWmkdxP798H3ShgrrM6coTLbG0uWxNM4xLODl8owvQIdPzh\\\\\\\/Oi3EfEehS+rTQuH0j8udSOS79Ux0CEOmCpymMzqbH9sJa+kDrrvzKb2uWiGl6MQGHu75YwJEM\\\\\\\/BlriuO8\\\\\\\/F2+JVHu6hOTu+CP3Zz6uPbkQq1jXfCihBO2dkAU4l2JYAXQdYUGkb+R2NqTRWd3Pf261Pc5T+rwzpVMPkPQZa5xGrTlqzAWkbyz0a+Vr21HEj68DfhQZ0ca5\\\\\\\/qGcnC\\\\\\\/sUnDwJZVraqHTionSQoYURiIqzCy3TkA1JNYNeRuEjjW6o77hCdZu3YZAfrAjDGUzgTO1E51kx6ufad3zjGnzApe4nHzXm1P+YMTr9ppSwQNRhaDb7iOa8vA61zl9JWCzWv3Lz5nempZ3yLYpkFivVCybgqtGdGkvHZwKOKukVBhn0gRcP6SVtuv8C2vveZd4oUFjzUdkmv6vqZ38bxEBuL1Ec4C4p70PnCBC3AdZkcpCVutZ2hCiYgbXG3482S5zpWALGHrUUr8tYDY2z5\\\\\\\/5JkYPiHiJh9LlY7fffwxXJ4yZneqyd0VwMADUz6OamzVuyvKkUHg7\\\\\\\/sdJjuN9jSi00h9nRm9GpT\\\\\\\/KL\\\\\\\/a7qbhOnK3cIDBMXfV\\\\\\\/0vgqtm3MkvBkGyb8gCCli477H0nymStKKM7xAfwvC0XVAKGTTTTKGwOeurgqk7DXC940tTOtMb5eTwsbkdK2hwYlFGmnTwHMUIFCLP+kgdJIa5oGthdOnct4dA3\\\\\\\/N22Bmnc6Yp00Vi9i+aw3uHnXFTVU+zkTOQh+tNroXOldfZVURWWvEjwqysCFA\\\\\\\/\\\\\\\/8\\\\\\\/IHWvOQH5mvlRA9OqlOOxmq5ads5op5if0hgHVtjE3WbI0mkxscIhhr6\\\\\\\/J\\\\\\\/NN4P1hYQx4gHufdeqCIC7NA+O1L6m4qqJyGyVaBoZO+DN9e29FDtIX8H\\\\\\\/PVLmF94zpXZzXV5vZ+pjjlnxJt5uu+5TmwqBUbp5JdG4F\\\\\\\/ukDFD7DfbYYmAt2ppzyqk9NBLOlYgGr0OeXz56NaXCzQmbqfgm9koMvnm0+sst+APDq2VyV2rhUg1YM3VSaf5IxkFGtw4v8ccaLy\\\\\\\/xGX7QOC4yKmvGGmhkhKbfGXaApCM4GeyslLCau2Z+W0P9ovnz3b\\\\\\\/kazKEHVrIqMOISVF10tt7Vp8eFMEYWt74HLLqVza+boYc3h8uj5svJMdrkGmid3tT5zfwME0fp6K5t\\\\\\\/jud7s+vd6+5oIwSXUgbxY3QhCYHVe2Jgdjx0a+0SDPC1IZ7SVPVYI5G43SASvFo\\\\\\\/SZkP\\\\\\\/96WCXMfOWj3RA2VWMv+5F6OLRKiRNPAT\\\\\\\/SWAcRmOsHNq7U+2EoBv3EssATd4zl62tkwveUWeCFBOLHaXvzLqryUd9CeWda51YJje\\\\\\\/ZVUpCJQgOc\\\\\\\/2A6J47a+JXOZ2QnH1IiY5IfFZ0uSY+88zPfVjTUuQC9c11yD4QD43+YscIQaXSo+KQ3pkJWInVWOsO11Rjw2c32cNjKheUW\\\\\\\/R\\\\\\\/gmnk2ayEFbxgMeXQrk0Bn2QYEKIbGb380QFNYNzy\\\\\\\/esJ88hkSQkPzva16SY94aZrj53pJ6YaTKahnv9lvlY\\\\\\\/DoHkb\\\\\\\/cE1lBEz8aZjnHk\\\\\\\/EbWdwMF+Ai5IHkw1WA2gEjtx1uwdBqtAoaIutzSno7UBGsxK6JqHK1sJyufOdsxvXG3GDHrrzYFPVv+WAG4KFrvW2KN2lLV2mEyoplkcWxNneK0mrtgD1xt4lh9JRiqWXsIrKPT24bcpepMT6p5+3nh1Atiqw4tME\\\\\\\/60ZT8j5ZX9CSdI9ftFqNAq7Jh+TGfNFToNqlvS\\\\\\\/EKPzxToYjWT4ZIQCHtwllLWgO\\\\\\\/0OLmOMXfq8GKAPFe4RTHdEaeSToovrsqlQ2DDUieTDo5jn+3NWkzjy0tN2h3pXiuWbnfDuPxT9nd6H7bqOtBFv36JYgG1U9QhAMMxaCKDHq5WKa3WnR06SlGlEW7bEgfYq59lKr5lC1eUZVexDDS141Srv8q1vAgVR+SMP\\\\\\\/M67dL7dds\\\\\\\/8xMNnjDJR3quAw4JHs00LGNblGN2zE3HX4PX8XyrX9uTYPS5NOXyhA72fUqXQCSedk9KwXTd7rNyCsfNjgiEc3IwPaFVCcjIXZqE94O3t4TlpcEEpFnloHGrcPny3+J10YzHgtBLBuL7skzCq9vD9Li62p9COB\\\\\\\/q0\\\\\\\/UYLsO+FYQChYgzoZo\\\\\\\/B66XEmeztXKHE\\\\\\\/tUAhLAcUBKfPL0R8A5Qqq9\\\\\\\/wNkNK4VxfyRZCLcXzv86dIR4kWkBoH1Z1SKp37IJiVpTTsuvtg3Ai0zFIX0bQsdx1FGRisP8R78ZVQwbaFKbHP++ntUQO428SR+OAqklL65\\\\\\\/CIv3VG2iEMyrCBXH0oftU\\\\\\\/X5XhkC27VLgiVDNZs4+KVjX1KDkY44Oa73OwI\\\\\\\/uA1E\\\\\\\/yLylfahXRThcHRLaamW2QkMWIlkIS+UFz4B7ZLbVJ\\\\\\\/EEKQCzwwW+S6Vz\\\\\\\/yxFAPQ56EQD58jibP82BYsQ2zoDWxvN0F0QRb4KUCRnnI+as79AuCxPWdwA9zAuMyS4x8bBvpwtTrgeBrDm+kDL+\\\\\\\/xw+IGcd4IiAQ05TUD9tNniquC4Gx1nlTyw\\\\\\\/l0\\\\\\\/GMe+jAbDWGmf0\\\\\\\/lkYGwLEpAHt8srqEttM6NVgAocxXVJWvZxMaEL1o8laqRK84f1NpYlfNaazRV7acclVpvHnbxk17i82+K6I+vApI6mMhi91sLZRWLJgIZuq7l\\\\\\\/J2yCHfTJ0Qi91AgRLwNjl7tSWmOwBTl0UZ6E1iewwo6mzfoO8bP4spQJdJOJPJzBurNCI4ektZhmybymGf6iZo2yjsi\\\\\\\/LENxbks475Kjwyby\\\\\\\/GaL1bqTHyxTuU\\\\\\\/aHqBWZoEGpcsjKOrnbxR+Ve7X8evzIRHo5E0U\\\\\\\/yekNJfanDcNq5HLLZ9YSOSpLDKMskyq4\\\\\\\/joc\\\\\\\/7eboR+WqQTRnA7pliyPUUKEUrH3GVbUI6Cdab\\\\\\\/o07BjBPIhjvQqowBouqjnwHkJ7nKcuJPTQbmBib4jlViRb7FqLDDzGLk9AWtqYmuhzKjWaxuasJ3Fs5riY9GDyny\\\\\\\/IsrdjbGg3g92h05S2K2Xg02OMRKAy4gIY3RENdwXFVVaYbO2WifF69fOvGvjcddFldjI4pcNha8e\\\\\\\/546QSowqVhoTm\\\\\\\/WujoSGP7WBj\\\\\\\/7MhtRefdaAlp9O9bTLDxvld1mV+C+tshp2l+nySFZUh9rsK5hWpeTPQzlmF9VxIO8x2GxJ3ePzpW\\\\\\\/DcK7f3JUmxV1yO7Qn2SgEajaHC7v8t04GdzwANmvgVs11ZiLqRCx13+g3H\\\\\\\/yRsu7auaLK2eJg4YiV+9h9twUmcWSxpTOu99A0y3af9RPbWKo\\\\\\\/NvnxlQeoTA8q4d+ff8N8aoKcnL\\\\\\\/\\\\\\\/6\\\\\\\/KK56e2fH+K\\\\\\\/UL1JDWppgSe6sy5tB+p39gx0azABDcWXuxo1HsQTnLBQFVQd4DelCMqvYlG1xK\\\\\\\/VMuniO4XzuMQjl+1r\\\\\\\/4NmHX7RThSqnQqk4l0pWpdhQyVcAMV6jBj6HYi\\\\\\\/qmgdmdDd4\\\\\\\/jX\\\\\\\/Ez3x16NO52r3JNIw6Ye2bQHcispzc\\\\\\\/xsVH6sEyblv5+IGsw1KvbUldCgy6x86gprnxSAhdeEjasU6qaSi00r7uTz4mq4IzK4iBn9Pmr\\\\\\\/twXoZRn2lB1wAML+LOmmKOe3zV4pYF8iHO8CmoUaZazHseufruxfF\\\\\\\/Jecru+vuoTNOVGdssvI9TwKowUvx4V28BKRzl2E\\\\\\\/DKq0wn3uvU9DUJNLv5sFaeKbxiO\\\\\\\/qR+X+jN17UzncGB1iHYJHqWqETIVHBzjdOFpH98SNpptmVYAZPIhPfAan4XFS1Ty6750IKkm2sFxgvww1Je4DALi\\\\\\\/IhPc4SkYlZBjKjX5sat8poPHWKJSDVcbxSg2dIvel9V1XPwDFkoNxZRsnjlNM7LOWTZTE5dmyHT\\\\\\\/RugvgQA2nloIR+0m+cNQxrtcVN4UdvQ51Vd+\\\\\\\/5J636eEUH\\\\\\\/MBLzq2nUanfsDjfbr8giVPDYb2P+xATFmZP72IEFSKexBMIWJmO43k1iOHV197C41on5Bl2WFxsqT96LkXOWwCOLYGyzjo+uVrQ\\\\\\\/gjxNE+jzhYf9UtyD8mly4c4HQqKjL8slG9terio7X43uEArlyrRSFvsWlou2qwQaujWdhW+\\\\\\\/qPFCRuq+CQpO80CPM5rTwYqIiTo4LWifaw7uh89hkTQjkeJUOpr+Eea28veZs4lFBM5T01Htl0gPhPk1iAgT+HY1x2ofktT3zgjsSd\\\\\\\/6crylwxrWZ56kjlsASxh6mU2r8fbo5UVgCo2tXYczam7NJNMpW3k9eq\\\\\\\/YnTstfETgEmUg1l7K8hjehYlEfGgkuRJPkY\\\\\\\/\\\\\\\/iFyZFEunYVv6QUqd\\\\\\\/PeR\\\\\\\/nHgl1xGUhe1sKbNqG+Jr\\\\\\\/A0j2BhZbdXSW0wgcOLgtMIlitp29eP\\\\\\\/J9JP9VbcE3XrnRuNF3SAlymXHBfnvSSvUwP6Vv0FzwAgfhAdVW9IeZ7FR99D2smQUqsena3FQ7hC9fvINc73sfnw1zeY\\\\\\\/81l1WQEg\\\\\\\/A9CSLwtmctRKkmTd4B4zpwSO4U\\\\\\\/zkn5DYiSpXGjJgaLs5wMzYAk\\\\\\\/kvDB50OcSWlFjo83nXOKJdIst5JU+tDLSi0FYUL+\\\\\\\/9Nv\\\\\\\/qKzw9RvczbE2UXjQKurLcedP85I8d5B2uTppE5\\\\\\\/lBzudtm\\\\\\\/h4mkByKrI\\\\\\\/oRdXm43UKiGIBvSOeJ4wOyIlEDwBravZ72CpdsvZKJJfp+1XwV3bfpNhZTqN1crMChzrfLIodHkoRaa62T21nm9ciovL1r87XyW2yUHBezCMKbV2sK0hKlXGdIkbjCj3tUkz+jODu2f4NAkeaE6R\\\\\\\/cMiOlLipe82+7WXViKZAD6U0vrlTwXrmxYiDgNtpGdVUDfVbjqsDLMUEveqijZ22Jn0OMmV49ydBwoKpoHXnJpzRoauqWf4p9bUSPWFEXccsnseXFmhQBNMjFxrP0SfuBVs1Zlkzn5GHs4QhCgeozNWfhKbkBuz3t7ICjWDEHuZF56b1RGN\\\\\\\/RfHogvJGK14fwEcOID6DhDRWMwvTpKf3X6W95+EBjkbtm9qFBesR8dy\\\\\\\/BmfBDJAIknljzqGUneNzzFTyV5QRuVuzmWX+nA7u\\\\\\\/d59qZkwVVKLWUns8bdAHkfBAgizBsn9x7Pfi1nDycFnmLQkJcSzlDIyjtcn5hqyYAZ6fltalcBSqRVIJu091nOcQF1DbpFa7iO3WaZn+WVQb9oagcxEq+ePcSxkCDNwtf17Kt2gXB9uOc+ITGp9Odns3rSWJyNIGe0aGTJwghGDvmJxLqYQl9VHMioB5U1yYE32MSQ8+2Mt4b\\\\\\\/AegHBblfV0L0ZT+Zek1bSeTYPpcmmefK6fylPtwXz8uwHFvfQ5lH1vqiU2uYQwN7m4oafFIicrQg8\\\\\\\/uFcFPYysJ\\\\\\\/up+cDaqhhObGEi3yd0p0tfUJ07+nJDuOu9TB95M+WbrBrWdB+MB1JYSZZrjw9UH8pODKSXcnLo07TxymwHU29SiXmiVKU8p3pXq3LrYRNu89cpIsS19g9BXugHv0M4iYHDSYCj\\\\\\\/di8NPcNNneMJYfCXv67rzb66P2gTxvD+BtiQ6hjm1OQjCTJYULIUS7fMm2538nYPrdUcudU1kgyAvXknUZ5jS9jnjQSAAq\\\\\\\/RXlVZtIdx8cHPiZHQNWqDq\\\\\\\/9jfYJWtqBG923+N4BAZNfbL1iKr2Dz2oCezc8L5hxSKlRFW05GQCtrYSMQrTOEFZY5ZxB2xdElxNHHdFxEUqdUJDrAcF4ffgapZ\\\\\\\/vP47YBO8L8kW978oAi+EATEBcNg17imjwRqFFsO+TgodN7XHTdS6r410f0UzSAaCYvQZoGpX40omqU49Br\\\\\\\/Akj+rFWX3yp2frr4Vdm6jfC3rN52xdgBFZDso3ec\\\\\\\/hvj5lbTxSipt82CcSBcJ\\\\\\\/iS7U4VZZ54qdI96gm1Tbx6OKiVELRw\\\\\\\/kjk5T0vCzZfdqp5nPaQC\\\\\\\/PfFqq3sTFPMItwiwY3ZMZhLI5\\\\\\\/AcKzgl\\\\\\\/sf0KleLkfGjnRDA5zQidz43eZ3kLv1BFcBh\\\\\\\/fGG4oditRXm8BRmJ+pZKh6UAUTdnuXETvheYlVocAkdz2Rib9arAzpaVJ+wQ1VtV8AIic2qSv3oYq7lgJQvN6fotF5G+EOBdm8va0l4Kjo69jzqPDyU7PpdKAoX53lcfNpC2c4uikDSVDrSmDbTOUJhmyG2UAJpbg0DXZE8GeK1IborcolIcR0464sWhor3zamZCOGifKtAGKbyCH\\\\\\\/Gh7HeZqL5RDBvB5kSlfPpX7UEegFzjVzpR9Mu3yc9txl2j3eKI61qd4+5DEkJuzmSKN3UBVeZ7GDjkIifTBp0M1rAtABlXzTwynTWDjZcQSWH57TUwqfWdrAQ2+D830n7VXoeZx0jTFMv+rhjfGb8H+b2BsMfy\\\\\\\/Sip97Qhggb7ZHsuFgS1mXTm90s8lUWb8H6It5zJKqd\\\\\\\/Y5VdZww3kK8nczdqnrNBt801OZIR8vFhpGNPmsG0W7Vybuaths0i21xXTGfWiCetj5MgxapXLF69wh7445vkXD\\\\\\\/yMcAjgZ4TgsMlExI3aHOgFwftQzmuB2Kfrk9yA8XkzYXOx2hB\\\\\\\/sXKAOy7sfvD3rURH4h96PJK\\\\\\\/88anIznaUc62KkwwL4kYh7fdi+1Ia3WwUt\\\\\\\/gQtewrYQp3jtCcUJWlPmj1W2k0\\\\\\\/H5zT2nORnU81eXWEgz8pjM\\\\\\\/6AhiYkl4ivSgAey3gojlVyf34kVkCCZDsySCDNcrdFMmJ34LvQvLTgud6ZUrQU0wJrRk2UVa52oOseBfw8pnfthl00ee7aWvx\\\\\\\/UXTZtMfiWFW8jCB9JR4oFr+LA0vFjmwDRrEK3mF03e+yPDCtj1AMjUbM2F50BQM8Tz1JQEFJHwtbt5OiStmeyfZhrv2lvM0mN86n36tmBxsK+mgWlf0y9JE8nyDrQgGPlhNaA2Hm2n4uJ+XLXiE3uXXreVHpJ3VKQbICZT1paRjeN13vJlG92w1vV0TCbHTgclJ89LOFlG0yb4z7\\\\\\\/auL3otcOjE3neNt3TyStlETy0TJ8UlfcF1lEDpbSI\\\\\\\/SB\\\\\\\/uQZz4Ux0usi5v15a0zvof3P9BNuRj7sZIQTZSSRONWKWxvhJL\\\\\\\/jYwo5iUwu0rNXCMxBl1rJH7tRd5i8plPK\\\\\\\/Yovh8H1+dn35b8dnE2xF0oerqQ\\\\\\\/8\\\\\\\/4WOz8AtAr2lZ7tX2IeHh3R9lMcE\\\\\\\/KoRHJvFqZrmAOvk1ijeU1MKJ0G\\\\\\\/f3A25ehmX0bDVidwfDxFJKIObQD5Js2ckc3eWPekgCqlmsQr9jpxT1yINezhMpK1dvfMl6EZ0bVJH8mG7lJl9udQewlHm7BlZohQE2wnpZ5RO+EsW97jaAGCUz1LzRkqV3+e4quIAXH2UNIFdc9jGXs34W6Knl2c9K+OkYGJlGXoND1Bg3TwWuuHj5mfIPzyMD9vXbUr1KEKddpdNkhLQJV0NJ7e4hkE5rmwlMhp6xmr2\\\\\\\/NMjDRAMwczoR91iwYRwVkKkt2uIM0oauCk+6xv\\\\\\\/ZjjPJUmtd+3HOj7rlCy8RC7Z9qQqcXAgm2vBeaDoiqDmNaKQaM4iinYCcJPCV9qYocZzBuiq378tNQnOxoBC46PFi07lDw0KYhjmVhsO9IMJ3vMK9o0TMaJQy5mbggGP1j1hbOij36M4SFVlQYXOxNy85RkSCPLyazx0Ka6usI3S8E39miofojCPp\\\\\\\/fQC7l1upCz55Iv7uqv3qvcXQ0r6RGxKkG4abEXH1JrDZC41NHoS0mBF6kgr7B1jaP4osXkskOVRUMwkAtFPgOduSLeIAfQh97Fgft9pKyjznjo4x5SlBOvRrgTPxcnNcDofKLgPik7\\\\\\\/Jps7Adt4DI8t5lxJ\\\\\\\/94k1Iq9zbtRe6W32zQcz99o+0cSu7AD9S6MfqlnMDYSQN924kX+XVI\\\\\\\/qYc8ohRGtHwiZDOgbVqdyENCYr7F66dPOTmR3ibeDFcD6RfXzYFwULb9qcMxidObpIwbgt7wLwGLfKST1SRb11ZH3xGWVPWMfWjJt650a8UbJCItj7KYVswqUWGwRTusH5HLHyezqPcD90G+7\\\\\\\/rXRogeyG+Tx1D\\\\\\\/zp2mYENhXE14vXSILOi3T60YQsirYy5OQ5+N1hX2WZbu07eawKqjdp64HF7L9OGlpLYJpZMMmNtaIhzFkg7PBi\\\\\\\/gm9naJHTXTHVlv2O\\\\\\\/S8oMYRw4laVw2WNfFncdYy0LxomVQHzPz6eR2ishIkhTZEefnoNKPnXxt7vK8Y5SkDxmpX9EZBKDMG1yFOZMmDjtmODgGiII0ConGMfNh8hyf4gtRmEFCsyzpoGiZtElo4V6pvvdNA9ncHYtz4QYbJk3U3h675JBJNM9vfN4Vb+nOktMmJ8O6aJmw93ePD2l7X1RHnU3V66w6Hx8bQdLihzBuqnZjzwdxUC3KVP+aK6xwIvrYqdRkac+fFcVoIS6rQg0ssmF9enJdOXBXlPCwfBPwPNQnWtVYlufnbi0Hf+MZwRbioF+3rtr9oR4a58oLLl8KoDmjEyq0qt3E9XcNKlShUUgRks7YWeiOeToxlZsK9pqiqIKWNkdtQiuAVHDXt+S6ramARSbywiYf0QfLgcY47n7wmdrMEC5ZsmM+inq6sh6iYGbeviFMuJjRMwKFwLYDJ5xgQE+hi3f5MGL0z2bwQuzGlChhrWNNO0DNkZ3jh7FW4Qc6BkCd+E7ZK1j8E6L5x7JG0yF7VoX8sZpETdKCiXewNDRuDTNkIZIGqxthK92t8Rt4qN5AY12VqkNkxiXtcM5MneT+MSyGWi398egTwehYnp2qnWeo1UpgGCOF9aN+GV4Nvaj\\\\\\\/SjdSMg7hLgZO7xFrjH\\\\\\\/oB4PU6GkzSU3+WtsUMFe6rViQZM6zYK87FOBhICNBuBmWepffiY0G1j\\\\\\\/w2Wv5Ff5GKT7dMaZQRvSUjI4U7pnAqXSkSdy\\\\\\\/of\\\\\\\/VyattAEqWKgN6\\\\\\\/3bEW939PRGuPBfdG3M4sxIVJaoF2KRd0nKr1DWorpOzTQunlB0PzO4UVAPKGUxZvr3pIIIUFQEyACSWVB7DG7Y15LiqAjcUPYTqj03wAqMl1Jp+6aQMWkBHuuGB2bTkGX7VKt5Ow3Yx7zRFfGOmICBwql39vBwN2VkPekCQRcAeGhxzb7BlRMOmuM5aGocy\\\\\\\/yBrPkz1AKYrRm425cZcWa349xuBQ+QGTvcjc+fiSaS39DG7zEbH9Y7ZwiwjDtjAFD4sq1hPLBKEavCMs1WE976LxylypT72H4LVw1sCXssodpGJJDFK3Q7C8ZISoSmTVrsW5DOewHsCIpptXOcfRgld7vvvolC2IDIDknIYHQo5ygmre28oV9AgZdHzk6+VcMVGosgRQLlOgFVsGXoRBzXpubbPIdEcBIFwcRzAq9WHb\\\\\\\/9VXOxzjW0A1yALJIgiitVRz3OKzRU7b6Zxfikr5RAaZAPtMSVNs1MwDduwirPFR45RQ\\\\\\\/q2BTnVyJRaELXvi6NGj\\\\\\\/eKgzqxQ+HfpmLqST+sqoFpyphfWeLKZbTX21uQ\\\\\\\/2UpvpV9Zs+JmHqr6kaLMHqvRu1RpwldD1nQODya465URuJ0bLjQG+wA3Rger0iQZb\\\\\\\/HvBauT+VbQckKbiDwwGUNeMU99Slra0E1JrTmWz4+AcZZzXIL9l+65gcyn4fIChjX5hE\\\\\\\/30Ct\\\\\\\/3BOSXn\\\\\\\/8SSJs\\\\\\\/AgI8SREaiyTfuSbZn8L+pC9Qx6nLWCGQKiudYk37RAaQZzmFDIwY3Qo4BgHXU0w8XBYMkOlx+8yOS6bCSynlNE2li6OeocaRjatNEU6zihZ2z9Zmu\\\\\\\/mUMW+hW2Q\\\\\\\/F3e9Nln6i2WJQeiLmdVZxh+SFcAZlQ37a0Ol4CVa+8WDzoS0RLMR4pcBWdq5DVsNsQeGbpedkT1CJ6arkqZ8ld2G4bN3Lh9Zej4xJtTS7ioGIEofQ9+O44peqA\\\\\\\/Dzuw4TIWWowLoQGsluuKQAlRR\\\\\\\/veLPXDMADv2gRMxwFz0BXvGq\\\\\\\/KvegTuEDXWZbyxqvK+LquTAUpVFLDm1eC\\\\\\\/63SURfO3+mW+afm4rFfk+XX9rIE0HWDXdOOC\\\\\\\/1NDMfXheu5OMVOiMn9gmwXGwhhlFW+87MeEd+Sy4jLrx3eqcLU6DpFLp42Bngo93zE4Aexsgp5h1YrW0j3SuP1jA94kOofG90GntuemocNPm\\\\\\\/9ycDcx+m72DuHIwtJKjXAK7tfMJctzhKOunH+LFmT\\\\\\\/7mxD2cjEiTGYE6Y98P5KH3tb0A2qJ8+E4a5zfCr\\\\\\\/i2qYe7rpy3GdoWAEvzbBgl6TCvVCanJ0shU4w1sKdHznDWGgx6WK+taTMFHzSg0IO0dmrYZwpdC7dDoKhTF0MjpryB\\\\\\\/Lb5b22VtMIq9z1cJaINY8QaTxQWgESVPkFCKH5tdIyw1SUNO3z80yel5ktEY7MMm1gOe7q1uZM\\\\\\\/QnVi8vR5FxqDN2HGG84MiKrlr7ZiamwSl471aXfdWZoAGUvQT4NdCoGRg4mr4tSx\\\\\\\/85nWX1u8QKMQ2P93qY0rqocx9e0mn+gjNes5XqQUdGG+heTaJTJYivsk2Uj7zow5vQHSX+UqnLKl5SIRrNvlJ2gQGLSv3nnNIm7SCz9XRotm12hWoQfhEfFHrcUUPqPBUqwSZtjGYfFOU9f6qkbQVM7TvR1JQJsfkKlBjx7rpA2E0QZK19otNQCgY676cz\\\\\\\/JWVJF2Lm8YZ9+\\\\\\\/PYl2KIHpuWa6hIC9FKZZ3UsrswDyf\\\\\\\/6FVyXxCjq54R75ZBVVCvoUMnPr0gzlTjw0EYqObpnk3+guHl59vgVuLaVIALBD+ZWNQPpjLIs+U9YNaQ5utwakV0MORB+9dKZxPF9PA0TZcflTZMMwJH5zIcwbzWvjWeTpSq6KdPyeWcld7k+V5T2Y3YG+PB4AlvRt8LKq0BkGSJtmnmIqI+JQZ1a2Pt7zhY64E81alpOUk4lED7W1PDW59BKUiggwwjH8O1t8jJDGTt+A7Ayie3Cq4Cdp5gyXZcln\\\\\\\/C790WLP4Hq7OlrXkYlUiKRwFoFtQl\\\\\\\/EKgcK092u2NYUGBqQqQLKc6oyeYC+r1SktpqJqBqPjUsbfCAU7419EHxYYZeh29GKioif8UB4j4GONGB2qYbxfKC5Blq5PB1wC\\\\\\\/72AdXFJN+COi1mIDxdTjLqlBYJjP0xwHh4Azi+bdafl4cDFXv80Aer2TpUADFjyebHA9PkXwDYDKYh6lEzmWzqf55Ugo7CxQu\\\\\\\/R3V4ExokP65YbcU9JpLqgrXdPDlr6983QPBDeYXhg99qA0yc5mGli5U7aYDps+9GzzUHZg7NEIm5\\\\\\\/oPKOkhntNXAXT51HG8NFQDtE+b6nV1yU9pEVqPxYjVggRc4HsvPQ0n3RODE3ApNzuAg+yY6CakTC7ytyBqiLOYgeO5LqFDBs6X9MBqQXoSMq+h51C5yMKXZ0QRYnr4wj78gslkxYTdZIf0MbZXekeYxddLJpkPK+r8n3\\\\\\\/AtImL5\\\\\\\/3GfvjPtkQMsIoHdbOfKvTCjber2Rb2pGhlVFFR4mkG5qc8CHWKZ2xUjcwrvbr5XJMe9hdLyDK75G276gCVPV\\\\\\\/Pcz6QL0sD9zeI9JcH6PXSMKmL4LLeXwp\\\\\\\/9Wu7VnMhHypfz2h6K8IV+mXJiEMu56Rg4vrPBeiG5uxEup34IkTxEM5IpgmXAI0HITJurM2BUNTsoKl49Am\\\\\\\/ltAbwTGJL0vULn\\\\\\\/Y9o5RXwHLtkmLZ14UwC7um81zlZPqSaAQCUr+R7mhW2n73Ga+6cjmhki4Yz7BN5GM8ue+iBNBBjpci\\\\\\\/qY2KooUoacJyvN3DTSd7RpOmF4wCp+tzQDuTtZJsFWByqSTq\\\\\\\/250G2\\\\\\\/5AF99w7V6urAtEXOHjjgKdMr3zx6AU6zCcfdO4NaASOCXpXGUssIZo7GWZz6JZu+4hA+EpouZ8kSJ\\\\\\\/bAm+HCUdf6sPaXzTAuAUJyTBO5hzHkYrjU9XXEkuI2gTwMWURz9PI9qjqd5wfyVs9OZX5NWcpU3ZSubMef16G2b1LXqTk8662oFb+39d4Za3Wz800+s\\\\\\\/5urXKHlqEr3NtG6a2AHoP7LHELUfmsBcJcbz7tEICYwbQRHtyTRawHE7py6VdiLaBAC5VbKLnhCShnlXT2MTRw9mrm19jBgyKBamvotJPss0TBAwRSwA83+11LttCaZ+ypAra8qw8NMaRD7L1xB1JnajZLqlRWpUp+og5FmRQ63iBiGYnrgOp7fcd3b8uIOXDGtNieqf200OQXs3HjtLbcy\\\\\\\/mMGimkbIxuG5z13z660D+ED8eWK9I1sD9CtEj2\\\\\\\/Tsl9UJi\\\\\\\/zOnAE35ZXXwdackXhBXuzbiaue+I5DOCsoV\\\\\\\/Dvk+9ZmGF\\\\\\\/HFJDDvBKAjl+CGvdyk6CxiwwVcmogiigQYwJSHgssrg2dDNcPRY0wNLMXH5en1emn1MPajuIbI3kP47\\\\\\\/t9pweYBqxkLN+fBNNxI\\\\\\\/JJUPiQej1pEmRIGUFnAVY+e+kBagul8RsTYWxTYZYy647QdvHxIV7nGvD5QTRLzoq\\\\\\\/YHiV2hVDeO0gQqmp0hgPMlGseWoScVR42JFokkiSmZ8bzO694PQ+TZ\\\\\\\/\\\\\\\/EYmwDOCHEEvZiqyCKqIJyl+gyqNeYlgFClLQw5Hfa3l82s4HIkZw8JTtOVus42vV6X8NDWXC4qpHcRYAFEauauuMI0cMlvK1irWWl0XZzh7\\\\\\\/A3lMtQBcswLGFfnTSNMYZdoj9KJuD1aaaD5f4whPn4x219pUHsLSJbig32IjM3IKUedx+bJLJSDLEhSe19UoVMvIyeHaDvrzQqMpQOnupJdYkuussBSIlXt5yRHXQMgX\\\\\\\/ta21x4YG0XZkl5r5771zA5XBgVLz3tK4ZRl5t3ZulfVJ40ehkMsyHZQDYu6hTS29BH4fnkPoBoESEWu0k9xYogL4bXDTHOO0Xvp5iecDNb1ed3uE4ZSHasi52CZkGeI4kPhDOTIPSELfLtIoGm2u+SdxsIa6C5Xh0BwOjAKo9zo0GBB5t8oM\\\\\\\/QMj8b5B5VjSZcXu7jN1VIQTCgGCfcDYHaXakFQcZYFe\\\\\\\/5ApF9z\\\\\\\/U0igT99NDNt\\\\\\\/3i3pWojEXK90DpeQfX4kdTHTXHpjJbEyuAePy1mRidIGaJutIu3OpRfuQvRWw5LdXpvLJglQVGPYt5+0oJD7J3Cl6ToJettVwAk9JRRJxFAYnkHhPs2xu1hvqjxVGpXzYspzuhdGN5Cd\\\\\\\/8SuoPiOSlHQEECBFIEQx3iG0jUGIn3Ek1X08oXKDCuBavm026CbKnBtKn9f2fKCGCZOwriT\\\\\\\/WDSvfS9ytJGxpX0VBn+ZzxRN\\\\\\\/zBDvgGdmCYYVLqKkWQmCRlTAjDkAX\\\\\\\/cI5hpB5nX9NOhJY5Y72fuMsLG5rEDqF8mNvrJsLjoDJnryvgRCnifDSm5158B7QPwEYibGBVC3zjjeTXEbocJd17DIdhsmMLQCs8GuHjwvqGR\\\\\\\/SysaafPPml\\\\\\\/f0eIGdL3u1ThpmKQGVA4p2qnJDQrpdWhVgcbnys81CYYJgFi3x4gbzIBZ1dB4yIiJeiW3j1\\\\\\\/DNx5yCoTMZeEdsWwr19Xk9cWHamOgxzAUanuRiSV37v4q3DvQhuBr\\\\\\\/fqyWX1j4YtXWdYSZmSDTkbgKRUuwdeMtgbEUWGoREE06KDLWrHfRPXTC9X8X3cM2Kfkz0NBggdhuyQE1jY1rC2hkyZel9nNVMRrGvBxaiQMvF57LNOM1jBTYG8lfqlTZjRJ8nWT9yC7KobyXM1GMKHxs6WMjeP+OdhGC+X951Ezu7zbtaW+55W+k3UP9zFnOX+S8EcrtkKcVW8BJMVuaC2dcl9huOULOJIZ7FqorTK7GX8\\\\\\\/00cFo\\\\\\\/rAiLiQzEEOWZIjvxf6R06bRJFGzY42z5gesAHJ7RyEdlpJoyeHRMzS4RTGt7f70132X8vubU1I77njVp3myBQsUYZ+SQNTi\\\\\\\/Gmw\\\\\\\/xAPemFF6RkabmmRXaBkMC3xWELacfmNg0vzJ2L+\\\\\\\/gjvSoT2RF21iHWqBYHtiKO6nEHMzk782LehJlTJEWZlAQWQYRffxyCpxJVZSnw5cKjxWGVQ2pS3htzZUVfQQHmGSptR9dLGKStUwYjWbn84vdVhHlPzhZe+ZGlkKIGgdhJDJiPd6WoAOdbpKWKx0S0utxYW85L6O8Iy25s37slDS29X1fqGzo3t3QdqHfkHEX28jeHsmvCxitsZjqMqLhtlgItVhmhlnpAHgTJyvpUKZgvJwcxk10eBZx9CcSsaU7igTsHLOvPUNesMqq7rcAL791lvl1pZKWU6Pi4SuigCZC77pTQZA1dqqqyZz5Acm12hOn3WrNJg+LIseSB9jAtEcIupBGQ6xZASY1oTb\\\\\\\/WsmO2wRquXBhHWHjCds6TNAfSSpWmYFvqSYY6GUIXyJMKmdSBfixTpAig6+jCpKKjU1paLIK0LgcGfUV8or258ATCtqDStYRTsTMrAJSwTA\\\\\\\/tbJGUEWq2ztQTqBs\\\\\\\/YmC5DScW5p3c64IumnOVDqSxKagyp3F8Tmf8+8uPOqkNBhCX94Ci79gZZpRHZnnnfNbItDEdRGXM8t1d5stN6Hz88ddhHfvXBwmYz3TIJNGAGF1YzCD6qx+MXcxoOOAZ0Hdo7Ph7NmklMV99Yu05wqxLTNGHdZ8uwkHBkJ+CefjsVkDR8dBHCAPMFPU9Af+WgfGK8+86q8zbZqWK72Hy6bM4YJpOlcyqqmteUn2G44J\\\\\\\/YyH3VhY\\\\\\\/JQQbgT5DBsFW98DUba8GVosuaNgCqOCx24jEcG4m92NyirFB9o1GEOhEy7V4672ACONIKiArFNxS6Wpgvr0h3StsY8Iuh4tHOsFhkC4qY7teK\\\\\\\/Guvz9IAnJpFX46M8vjZDsN9aUT4AprKH8l7VlU2oJS9yCA0YBSAy17HSukZMwr\\\\\\\/oP0GQqqL2IuglOu7s2ZuylJXnCnAJ7GpijBzbu6HqS1rG7pj88NfW3s7X7L92vS+8GX\\\\\\\/XuSyaIPMlopSSBrhclpG5zv3kf+if0CHS4rmxLGgXdGQfqVZ8BNdbHcNMeEN4i68nuFcstrcjmQAqVwJt66jB+lV9G7QqVUPeedAqRwzEbJfsnmirhz5WET\\\\\\\/07y9jhTYipjSHz45YujZ4NBR7Azv6uK60JIBFl5kmFQvIUi\\\\\\\/onmTH1aI+CKjL2ZL57lcnDry+dAE6SDuZlzxjI4U0JgYKmzq\\\\\\\/BV8yLL813bTy+i7csdWNsa11l1AEUuTdO9Ah\\\\\\\/D7gnk0E8Ap\\\\\\\/T5z3kSU0655ZBYv1DYxDLyH\\\\\\\/MxmrJNoyuYFyVMRcUDhVuG9qa\\\\\\\/yaEJ93GU6uIBNHDg7mUaeQHA5ApoYpNAhm8YfoIYdw8yhJxUmjUbIDSSW63Av5Rot3UwuideC4iJx1CfvyNpPsB1sW+GCacwfB93OdJo54G7P2UBkR3q26XLKAuTWtLmS6qlXBbUv9OuK5ET8o\\\\\\\/GDT4uDbikPzkx9Sbo2OP6tPJtPyYWGI1sVTooESZACi\\\\\\\/LB56JunBNddZcuLUxSg\\\\\\\/CdN37QIu+R3\\\\\\\/5bM85jQWZJLgJoYAcoWTjHHGPZ\\\\\\\/bT9CYXXjHT2y5i4V6Almk8n1Z+Xg46EPGXp0Vbd49GeSoL\\\\\\\/CpGVHMGlm8loTnzDlW3srTfXKake4bq9B69ajnvCx\\\\\\\/3QPeKexeGp1XEOEjpRXypRJYUZOVWElz4dCy7dDbwarRs7g53Teu9w+ncrk50T9zRijuX6IjuGJs3hjFw6gbNOWGqw6VjF5\\\\\\\/grTD9mGdj9fcbLS2GkUTwO8NGMRK8lEvEo6Zh14AMv5ZmTxkKawMGKhbV1FvAM7RcxG0Vubtz63XeFLPOCIWDLEjzr86PUqS8LOA6Kp1DSCoylIFQmqnZqt+FNLWgVYYTNGjfcCr8C+ejX16eqMds1T\\\\\\\/jYehzFe\\\\\\\/OmWi+Hpig\\\\\\\/UOf+ZiAPdyqtw2G7HNtC7mqELxBizokOTZlT3zyWVgl\\\\\\\/g9SIk1QfBnvmsOAyhX1wrf7c+UBTZ6QLDeCqtFPpQN+1fX5YmLRqZLifgjFdh8HdVnQqvE1\\\\\\\/NL1AG3hgMGCLvnpgnC3q2eHj7Y3UfEpMytPfdwqnO9YvQeVa6chOwxTSBBu5YxZGl\\\\\\\/ZgdKiSscmApsTTy0PkkX66QAskl+tzAAMNLo9vNRJh3t3fnsB72IorLsab\\\\\\\/hDdckrlwGPH7kF9avAeh8YMdO75HT3LQeuqVeJT7vM5tQ57IU3cs23hhsEYJYi5XNcg67jIIiHRI7wAs2YxEnBUnTOTGANbXRWnaKYWneeHcbhuBZ84xAnqa6dm5XAb+VFWfQUBtehOfCq22pllMQwcMetNXAGyj+zDlLtFOn2Pc713f7DGuvetcu7KF3sMV6Us8ByvC1IETiwJgS7xoCmXVv6TUgrVk=\\\u0022,\\\u0022iv\\\u0022:\\\u0022LwUxGwFdkVSuXu+tKBl1Kw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022bd268c8b2b7766f867d5290736d69383363cc5425e1b5e1dda744fd7d18a167c\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022tpHojLguMEaGvOujvrqR6bImocvDR4CgH4U36lnLhDmD3jpDsKEbZABuJ13xh7nIe053LlncyboZmcSfGQn6YBSefjgyAtWb0EI\\\\\\\/nnJcDoac2yAImb4WAjFLqsvRbmJLoI9ZcRooZyGfrp+D0fepPdNb0p+8mPOsuBh3FKbAcVVYTBuEWfEVqvxIgau3Z9zUGMaMumEYPD\\\\\\\/cu7tYkpBlkGnPkvBBl\\\\\\\/Fgl6qDJRaGggrRD4IMEYP20Sz0EVJcbPLl8uKP1i6L8R2rwHPVROpguPLbrRVHwEYatSb9+m4U1ERphppyt8BMR1IF3hNZElr0UUIW3o1GsNQkLQG9Qk1+RPr6C2+xd5fFugas\\\\\\\/YnM2p6uzlKAUeadNZQLcveaGzfG\\\\\\\/TkuSxxzTJLjdBsNpPtkgzeifF3pJKvZretNn8SaN2pgRlOa\\\\\\\/T5dC3HfD5EOsaBQr3xQtdSxHWqlqmjb2QkDGkE+jIjWtMIEOx7CpQYgzvVGUWYZ\\\\\\\/b+34AMltel3QPIfAyCy6EijSAYUIwcgifLdo1k0Z3bJgqVcKaCFRskyDpqZ4HmobChb7ATmd3S54rEf5kBXXLMBbnrRi8+XUuYlw0nvSF7d5Yz0vEjUNZMGlwMkyjCGUR61k3sXkX327jzEKAilY9KRjYDfhGgBAeaOnLPuoqlnaCG6H0aUZ5NAyWGmi64P7r7dlQoAVd4phib17iwpgDX0AGOylto0v9PbJgs8RfHcAW81JcynBbgmKL02Lp8VlLc3SYvtFxH18hGZxAA5q2hXQQMKqSuGDx97zzDgfjD8k96zRJviMtPUsc2FIpD690wQ47B0UNKIOMWtJxALAfwKsd3Fa2NIOb4SREH83xrUk3UrMNlspNcXbyd81ZaQvaLNcOxW0spxq0lPMm0iX0uO81TdBOegKQu1xLLzHMQdtdEx3eQR3MQU7uCGqaccBQDlljLGe3dh6KIQjbOAn4CZolXgODfimHZFx\\\\\\\/W+VgJYegL++Pgjdzu4cibiJIq4ClAu5SRexBIt7DHZ9e8F7095e1gKMkyrTBB\\\\\\\/YFtazwj4en6bUdSzEYIsGKud5bVYId+2\\\\\\\/wIUhHfFeHywXHYsqfKYt4hhribj2sn25HLrpUOE+OaCIdh5m75TUZFkcb0qhEQ7pcaoUbGMSIpqFuU8uRDCJXaxJYwHulqnbQSEFPEZAWgqLEST64im+kBpf2u9jf0UXCzffVsqhxdkuvHxpM85uvTyHfh8YPgkDX42zXQmUtLnzf5+pw3FD6uC7f8NFp0PrjJD0vgVeHDgQ5JrUBvFJdkDlW9YaSVbdjAbAabbEhhhxxNKdOO5FM7bDuPxBo1lyMfiCsn15NJavTYqMdcu8BuG\\\\\\\/Tdmke7MIBXQFshwydBC9Q8Pxzyw72MlXyM0asVS1Uj\\\\\\\/HHPMBTu0OGMPnnwvW3HT7K2k1dLjKzAvGHDL0gpB2H4bHTUeXRRKz6UP1valP0RA1zjk2itVH+t7iY5qVpWwHj+S19E6zYC0g\\\\\\\/cwLGzgdRm90TQ3PhB53Q1TzWp+1S9hGAwWK1K1tOQqkC409\\\\\\\/pO8354GGmjqMeXDqYicuPBlbaZWrFX\\\\\\\/d0JVJM1WuNgyEEBlIDhOwXIWEc0Z4gGBibnxbDeQhx3xJSPUf6ZNgETTu+b3m4JmpI9KZURGMSH05FNioNueP63UbR6voLJSCS\\\\\\\/iaV7hUEHVOrffeUg+K6FOMAApOKm+HArCA5nx\\\\\\\/s3hRJ\\\\\\\/aA3rmyw7hsYKaTeF5Q8ViJhoM6Dd13CT6z3+8vFl6BCl1I7kgiyTLju58eOWCRUAKDDZ\\\\\\\/Xp07iWlWu9fhs+2\\\\\\\/Gv6MFnYhxgb+2aee6TnRrniAdsaLt5aUtPNrWUETGESZ9mkA\\\\\\\/PheaBKTq3q3LwA5dWFKpAhoPM\\\\\\\/PQVD923APqiIZw7H+NzKEFXxGZN8SeZ1ZqjhYB8BUgsNhZlHa+KQIXw6+Jlt8LxUUaMkbtzbbKeiMTMDk3a+sFFHFNRg7gNMsujPUtHSIJ2p00MMEi8McryFy59onvG3gqtKk0tDvZ+WZF31J2g7x7Pupc\\\\\\\/q18JibS8J\\\\\\\/JokESAsd9SCdd4uEgWMDT3q\\\\\\\/g9fnBwX9OnWtrqn23QxxsmWR1\\\\\\\/PwXMiiIWsUtB1CdPGy\\\\\\\/WCCnzyS\\\\\\\/X\\\\\\\/8tE8NqC10A==\\\u0022,\\\u0022iv\\\u0022:\\\u0022\\\\\\\/OAuS1RKvYc\\\\\\\/3AYWUK9Q4A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022618c550d6a072824800d23d5336a310bec095056b7cdaa6fc070125f8cb2fde1\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460453 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/11/3571" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-2IQkL6Y_bjdO7HCknoUQH7C6LHfs0Bp5OW9Hhgha9DU" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/11/3571" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-YIP069ikaozHK8OMgtdQH7nzpkPyt6xYKMj0yynoBHA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/11/3561" title="The Genetic Basis of Upland/Lowland Ecotype Divergence in Switchgrass (Panicum virgatum)" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/11/3581" title="Divergent DNA Methylation Provides Insights into the Evolution of Duplicate Genes in Zebrafish" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460453" id="node460453" data-pisa="ggg;6/11/3571" data-pisa-master="ggg;g3.116.033902" data-apath="/ggg/6/11/3571.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome Sequencing of Chromosome 1 Substitution Lines Derived from Chinese Wild Mice Revealed a Unique Resource for Genetic Studies of Complex Traits</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-5952-5771" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Fuyi Xu</span>, <span class="highwire-citation-author" data-delta="1">Tianzhu Chao</span>, <span class="highwire-citation-author" data-delta="2">Yingming Liang</span>, <span class="highwire-citation-author" data-delta="3">Kai Li</span>, <span class="highwire-citation-author" data-delta="4">Shixian Hu</span>, <span class="highwire-citation-author" data-delta="5">Maochun Wang</span>, <span class="highwire-citation-author" data-delta="6">Yuxun Zhou</span>, <span class="highwire-citation-author" data-delta="7">Hongyan Xu</span> and <span class="highwire-citation-author" data-delta="8">Junhua Xiao</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3571-3580;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.033902 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460453" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Fuyi Xu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Chemistry, Chemical Engineering, and Biotechnology, Donghua University, Shanghai 201620, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fuyi%2BXu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xu%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFuyi%2BXu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5952-5771" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Fuyi Xu</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Tianzhu Chao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Chemistry, Chemical Engineering, and Biotechnology, Donghua University, Shanghai 201620, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tianzhu%2BChao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chao%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATianzhu%2BChao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Yingming Liang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>The Laboratory of Genetic Regulators in the Immune System, Xinxiang Medical University, Henan 453003, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yingming%2BLiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liang%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYingming%2BLiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Kai Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Chemistry, Chemical Engineering, and Biotechnology, Donghua University, Shanghai 201620, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kai%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKai%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Shixian Hu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Chemistry, Chemical Engineering, and Biotechnology, Donghua University, Shanghai 201620, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shixian%2BHu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hu%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShixian%2BHu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Maochun Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Chemistry, Chemical Engineering, and Biotechnology, Donghua University, Shanghai 201620, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maochun%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMaochun%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Yuxun Zhou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Chemistry, Chemical Engineering, and Biotechnology, Donghua University, Shanghai 201620, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yuxun%2BZhou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhou%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYuxun%2BZhou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Hongyan Xu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Biostatistics and Epidemiology, Medical College of Georgia, Augusta University, Georgia 30912</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hongyan%2BXu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xu%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHongyan%2BXu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:hxu@augusta.edu" class="" data-icon-position="" data-hide-link-title="0">hxu@augusta.edu</a>
<a href="mailto:xiaojunhua@dhu.edu.cn" class="" data-icon-position="" data-hide-link-title="0">xiaojunhua@dhu.edu.cn</a></span></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Junhua Xiao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Chemistry, Chemical Engineering, and Biotechnology, Donghua University, Shanghai 201620, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Junhua%2BXiao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xiao%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJunhua%2BXiao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:hxu@augusta.edu" class="" data-icon-position="" data-hide-link-title="0">hxu@augusta.edu</a>
<a href="mailto:xiaojunhua@dhu.edu.cn" class="" data-icon-position="" data-hide-link-title="0">xiaojunhua@dhu.edu.cn</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/11/3571" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460453" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460453/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/11/3571.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460453" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460453/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/11/3571.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460453" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460453/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/11/3571.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460453" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460453/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/11/3571.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460453" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460453/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Mouse resources such as Collaborative Cross, outbred stocks, Hybrid Mouse Diversity Panel, and chromosome substitution strains have been instrumental to many progresses in the studies of complex traits genetics. We have established a population of chromosome 1 (Chr 1) substitution lines (C1SLs) in which donor chromosomes were derived from Chinese wild mice. Genome sequencing of 18 lines of this population showed that Chr 1 had been replaced by the donor chromosome. About 4.5 million unique single nucleotide polymorphisms and indels were discovered on Chr 1, of which 1.3 million were novel. Compared with sequenced classical inbred strains, Chr 1 of each C1SL had fivefold more variants, and more loss of function and potentially regulatory variants. Further haplotype analysis showed that the donor chromosome accumulated more historical recombination events, with the largest haplotype block being only 100 kb, and about 57% of the blocks were &lt;1 kb. Subspecies origin analysis showed that these chromosomes had a mosaic genome structure that dominantly originated from <em>Mus musculus musculus</em> and <em>M. m. castaneus</em> subspecies, except for the C57BL/6J-Chr1<sup>KM</sup> line from <em>M. m. domesticus</em>. In addition, phenotyping four of these lines on blood biochemistry suggested that there were substantial phenotypic variations among our lines, especially line C57BL/6J-Chr1<sup>HZ</sup> and donor strain C57BL/6J. Further gene ontology enrichment revealed that the differentially expressed genes among liver-expressed genes between C57BL/6J and C57BL/6J-Chr1<sup>HZ</sup> were enriched in lipid metabolism biological processes. All these characteristics enable C1SLs to be a unique resource for identifying and fine mapping quantitative trait loci on mouse Chr 1, and carrying out systems genetics studies of complex traits.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/chinese-wild-mice" class="hw-term hw-article-keyword hw-article-keyword-chinese-wild-mice" rel="nofollow">Chinese wild mice</a></li><li class="kwd"><a href="/keyword/chromosome-substitution" class="hw-term hw-article-keyword hw-article-keyword-chromosome-substitution" rel="nofollow">chromosome substitution</a></li><li class="kwd"><a href="/keyword/whole-genome-sequencing-0" class="hw-term hw-article-keyword hw-article-keyword-whole-genome-sequencing" rel="nofollow">whole genome sequencing</a></li><li class="kwd"><a href="/keyword/genetic-polymorphisms" class="hw-term hw-article-keyword hw-article-keyword-genetic-polymorphisms" rel="nofollow">genetic polymorphisms</a></li><li class="kwd"><a href="/keyword/haplotype" class="hw-term hw-article-keyword hw-article-keyword-haplotype" rel="nofollow">haplotype</a></li></ul><p id="p-3">Mouse as a model organism plays an important role in the dissection of the genetic basis of complex traits involving both genetic and environmental factors. Traditional F2 population for linkage studies with two extreme opposite phenotype strains could have sufficient power but generally lack resolution (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Flint <em>et al.</em> 2005</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Flint and Eskin 2012</a>). Therefore, new methods and resources have been developed to improve the efficiency of quantitative trait loci (QTL) mapping and gene identification.</p><p id="p-4">Over the last several decades, genome-wide association studies (GWAS) in humans have discovered thousands of genetic loci related to complex diseases or traits (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Tim <em>et al.</em> 2014</a>), and a similar strategy can be implemented in mouse populations. The Hybrid Mouse Diversity Panel (HMDP), which consists of â¼100 inbred strains, including 29 classical inbred (CI) strains and 71 recombinant inbred strains, has been used in mouse GWAS and systems genetics (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Ghazalpour <em>et al.</em> 2012</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bennett <em>et al.</em> 2015</a>). It has been estimated that this panel can detect a single nucleotide polymorphism (SNP) that contributes to 5% of the overall phenotypic variation in a 2.6â2.7 Mb genomic region (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bennett <em>et al.</em> 2010</a>). Recent studies have shown that expanding mice strains and genotyping density would further improve its power and resolution (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Rau <em>et al.</em> 2015</a>). The Collaborative Cross (CC) panel aims to generate &gt;1000 recombinant inbred strains derived from eight diverse mouse strains, which could provide a mapping resolution of â¼2 Mb (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Buchner and Nadeau 2015</a>). It is noteworthy that CC parental strains include three wild-derived strains, thereby dramatically increasing the genetic and phenotypic diversity of this resource. However, only 73 strains have been made publicly available so far (<a href="http://csbio.unc.edu/CCstatus/">http://csbio.unc.edu/CCstatus/</a>). Despite this limitation, investigators have successfully used pre-CC and completed CC strains in genetic studies of complex traits (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Aylor <em>et al.</em> 2011</a>; <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Buchner and Nadeau 2015</a>). Another valuable resource is outbred stocks (OS). Different from HMDP and CC, this resource is intended to maximize genetic diversity through carefully planned breeding. The accumulation of recombination events over time in OS markedly decreases linkage disequilibrium (LD) compared to inbred strains (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Yalcin and Flint 2012</a>). It should be noted that individual mice within an OS are not genetically identical. Therefore, every individual needs to be genotyped when performing association studies. Some currently available OS include heterogeneous stock (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Valdar <em>et al.</em> 2006</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Ahlqvist <em>et al.</em> 2011</a>) and the diversity outbred stock (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Svenson <em>et al.</em> 2012</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">French <em>et al.</em> 2014</a>).</p><p id="p-5">Unlike HMDP, CC, and OS, in which genotypeâphenotype associations are conducted on a heterogeneous genetic background, chromosome substitution strains (CSSs), constructed by transferring a single full-length chromosome from the donor strain onto the genetic background of the recipient strain by repeated backcrossing, are pursued to construct an homogeneous genetic background, except for the donor chromosome (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Nadeau <em>et al.</em> 2000</a>). Therefore, QTL on the donor chromosome can be relatively easier to detect. However, because a CSS panel is derived from just two mouse strains, they have less genetic diversity relative to CC, HMDP, and OS. Another problem is that if a QTL has been identified in a particular chromosome, further fine mapping is generally needed to narrow down the QTL, through the construction of congenic strains (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">DeSantis <em>et al.</em> 2013</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Kato <em>et al.</em> 2014</a>).</p><p id="p-6">Wild mice are another promising recourse for fine mapping and gene identification. Previous studies have shown that they harbor more genetic variations, accumulate more historical recombination events, and LD decays considerably more rapidly with physical distance than laboratory populations. Therefore, strong associations can be detected within regions &lt;100 kb, which can help to achieve single gene resolution (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Laurie <em>et al.</em> 2007</a>). Combining the respective advantages of CSSs and the wild mice resource depicted above, we proposed a new strategy to fine mapping and gene identification using a population of specific CSSs (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Xiao <em>et al.</em> 2010</a>). Several years ago, we began constructing a population of chromosome 1 substitution lines (C1SLs). Briefly, 30 mice were used as chromosome 1 (Chr 1) donors, including 24 wild mice captured across China, one C57BL/6J-Chr1<sup>KM</sup> (KM) line mouse and five inbred strains, with C57BL/6J (B6) as the recipient strain. The high genetic diversity on the replaced chromosome and the homogeneous background enable promising applications for fine mapping and gene identification. In this study, we carried out the whole genome sequence analysis for 18 lines of this population. As expected, there were large amounts of genetic variations and smaller haplotype blocks distributed on Chr 1, and a homogeneous genome background on other chromosomes. In addition, phenotyping some of these lines also revealed that they had high phenotypic diversity. All these results indicated that this population could serve as a unique and complementary resource for identifying and fine mapping QTL on mouse Chr 1, and carrying out systems genetics studies of complex traits.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Mice</h3><p id="p-7">In the 18 sequenced C1SLs, 17 donors were wild mice captured from different localities in China, the other donor KM mouse and recipient strain B6 were purchased from Shanghai SLAC Laboratory Animal Co., Ltd. (Shanghai, China). Briefly, the donor male was mated with a B6 female to produce hybrid F1, followed by seven generations of backcrossing using 29 SNPs evenly spaced on Chr 1 to assist selection, then brother/sister mating to create C1SLs. All mice were housed in a climate-controlled facility (temperature: 18â22Â°, humidity: 40â60%) with a 12 hr light/12 hr dark cycle, and allowed <em>ad libitum</em> access to water and a chow diet (M01-F25; Shanghai SLAC Laboratory Animal Co., Ltd.).</p></div><div id="sec-3" class="subsection"><h3>High-throughput sequencing</h3><p id="p-8">Genomic DNA was extracted from the tail tissue of each mouse using an AxyPrep Multisource Genomic DNA Miniprep Kit (Axygen) as the manufacturerâs default protocol described. Purified genomic DNA was sheared and size selected (300â500 bp). Paired end sequencing (2 Ã 150 bp) was carried out with an Illumina X Ten instrument (Illumina Inc., San Diego, CA) by WuXi AppTec (Shanghai, China) according to the manufacturerâs protocol, except that the KM line was sequenced with Illumina Hiseq2500 (PE125) on two lanes.</p></div><div id="sec-4" class="subsection"><h3>Quality control and read alignment</h3><p id="p-9">Raw reads were quality filtered with NGS QC Toolkit v2.3 (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Patel and Mukesh 2012</a>) to remove reads containing &gt;30% low-quality (<em>Q</em> &lt; 20) bases. Filtered reads were aligned to the mouse reference sequence (GRCm38/mm10) using speedseq v0.03a (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Chiang <em>et al.</em> 2015</a>) align module, which implemented BWA-MEM for read alignment, SAMBLASTER for in-memory duplicate marking, and Sambamba for multithreaded sorting and indexing to produce BAM alignment files for downstream analyses.</p></div><div id="sec-5" class="subsection"><h3>Variants identification and annotation</h3><p id="p-10">Variants identification were conducted individually for each line. SAMtools v1.2 mpileup and BCFtools v1.2 call function (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Li <em>et al.</em> 2009</a>) were used for SNP and indel calling, and the parameters â-t DP,DV,DP4,SP,DPR,INFO/DPR -E -Q 0 -pm3 -F0.25 âugf â and â-mv -f GQ,GP -p 0.99â were used for mpileup and call function. In order to identify high-quality variants data, variants were filtered using the vcf-annotate function in vcftools v0.1.12b (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Danecek <em>et al.</em> 2011</a>). The following parameters were used for all lines: â<em>H</em> â<em>f</em> +/<em>d</em> = 5/<em>q</em> = 20/<em>w</em> = 2/<em>a</em> = 5. For maximum depth, BLD, DX, FX, HZ, JD, QP, SMX, SY, TW, and TZ lines were set to 100; CM, KM, YP, ZC, and ZZ1 were set to 120; and the remaining lines, including YX, PD, and ZZ2, were set to 140.</p><p id="p-11">We used the sv module, which implemented lumpy-sv in speedseq software, to identify structural variants (SVs). Raw SVs were then filtered using the following criteria: calling quality &gt;100, and length between 50 and 100,000 bp for deletion and &lt;3,000,000 bp for duplication and inversion.</p><p id="p-12">Ensembl Variant Effect Predictor (VEP) v80 (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">McLaren <em>et al.</em> 2010</a>) was used to annotate the variants, and the SIFT algorithm was used to predict whether a missense variants would have a deleterious effect in a protein-coding gene.</p></div><div id="sec-6" class="subsection"><h3>Sensitivity and specificity of variant calls</h3><p id="p-13">We randomly selected 81 genomic segments spread over Chr 1 on ZZ2. Primers were designed by NCBI primer designing tool, Primer-BLAST. Each segment was PCR amplified and sequenced by capillary sequencing technology. After trimming the low-quality bases from two sides of each sequence manually, the trimmed sequences were mapped to the mouse reference genome to call SNP. This SNP data set was compared with SNP calls from the same regions using next generation sequencing. If an SNP was present in the Sanger-sequenced SNP calls but not present in our next generation sequencing SNP calls, this SNP was considered to be false negative. The reverse was considered to be false positive.</p></div><div id="sec-7" class="subsection"><h3>Subspecies origin analysis</h3><p id="p-14">SNP and indel information of 36 sequenced inbred strains were downloaded from the mouse genome project (MGP) at Sanger institute (<a href="http://www.sanger.ac.uk/science/data/mouse-genomes-project">http://www.sanger.ac.uk/science/data/mouse-genomes-project</a>). Chr 1 consensus sequences for three wild-derived strains [CAST/EiJ (CAST), PWK/PhJ (PWK), and WSB/EiJ (WSB)] and 18 C1SLs were constructed using SAMtools consensus parameter. The Chr 1 sequence was divided into 19,548 10-kb blocks. The percent similarity of each block of the Chr 1 sequence of each C1SL to the sequences of WSB, CAST, and PWK was then calculated. The subspecies with the highest percentage similarity was inferred as the subspecies origin for each block.</p></div><div id="sec-8" class="subsection"><h3>Haplotype analysis</h3><p id="p-15">We used the haplotype block definition based on the four gamete rule (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Wang <em>et al.</em> 2002</a>) to infer haplotype blocks. A total of 127 randomly selected Chr 1 regions were used in the analysis. For each region, haplotype analysis was performed with haploview v4.2 (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Barrett <em>et al.</em> 2005</a>) with the four gamete block (GAM) output option.</p></div><div id="sec-9" class="subsection"><h3>Blood biochemistry analysis</h3><p id="p-16">Blood biochemistry was measured in four C1SLs (CM: <em>n</em> = 8, HZ: <em>n</em> = 6, KM: <em>n</em> = 11, and ZZ2: <em>n</em> = 4) and recipient strain B6 (<em>n</em> = 15). Male mice were first maintained on a chow diet (M01-F25; Shanghai SLAC Laboratory Animal Co., Ltd.) for 8 wk, then fed with 10% fat diet (D12450B; Research Diets, New Brunswick, NJ) until killed at 20 wk of age. All mice were fasted for 4 hr in the morning before blood collection.</p><p id="p-17">Blood from the retro-orbital sinus was collected in 1.5-ml tubes containing EDTA, and centrifuged at 2500 Ã <em>g</em> for 15 min. Blood serum was frozen at â20Â° until assay. Low-density lipoprotein cholesterol (LDL-C), high-density lipoprotein cholesterol (HDL-C), triglycerides, cholesterol (CHOL), total protein (TP), albumin, total bilirubin, alanine aminotransferase (ALT), aspartate aminotransferase (AST), alkaline phosphatase, glucose (GLU), and creatinine concentrations were measured using a biochemical blood analyzer (Hitachi 7180; Hitachi, Tokyo, Japan) by Sino-British SIPPR/B&amp;K Lab Animal Ltd. (Shanghai, China). Mice were killed by cervical dislocation.</p></div><div id="sec-10" class="subsection"><h3>RNA isolation and expression profiling</h3><p id="p-18">Four HZ and four B6 male mice, examined for blood biochemistry were also used to perform transcription profiling. Liver tissues were collected and total RNAs were immediately extracted with RNAiso Plus reagent [Takara Biotechnology (Dalian) Co., Ltd, Liaoning, China], according to the manufacturerâs protocol, followed by quantification with NanoDrop 2000c. They had an A260/A280 of 1.8â2.0. For each line, RNA from two mice were pooled and assayed on one Affymetrix Mouse Transcriptome Assay 1.0 chip. CEL files were imported into Expression Console v1.4 (Affymetrix). The SST-RMA algorithm was used to background correct, normalize, and summarize all expression values. Then, the output CHP files were imported into Transcriptome Analysis Console v3.0 (Affymetrix) to conduct gene-level differential expression analysis.</p></div><div id="sec-11" class="subsection"><h3>Evaluation of genome background contamination</h3><p id="p-19">For each line, the genome was divided into 100-kb nonoverlapping intervals. We used SNP density to estimate contaminations on other chromosomes. We calculated the mean and SD of the number of SNPs per 100-kb region on Chr 1, and established a lower boundary of mean â2 SD. If the number of SNPs per 100-kb region on other chromosomes was larger than this boundary, this interval was considered to be a wild-type residual contamination.</p></div><div id="sec-12" class="subsection"><h3>Ethics statement</h3><p id="p-20">All mice were maintained under specific pathogen-free conditions, according to the Peopleâs Republic of China Laboratory Animal Regulations, and the study was conducted in accordance with the recommendations of, and was approved by the Laboratory Animal Committee of Donghua University.</p></div><div id="sec-13" class="subsection"><h3>Data availability</h3><p id="p-21">All raw reads were submitted to NCBI Sequence Read Archive under the accession number SRP066591. Expression data can be obtained from the Gene Expression Omnibus database (accession no. GSE81078). This mouse resource is available to the community of mouse geneticists and can be obtained upon request.</p></div></div><div class="section results" id="sec-14"><h2 class="">Results</h2><div id="sec-15" class="subsection"><h3>Data generation and variants discovery</h3><p id="p-22">In this study, C1SLs from 17 Chinese wild mice and one KM mouse were whole genomeâsequenced with Illumina sequencing-by-synthesis technology, with a target insert size of 300â400 bp, and 1965 Gb of data were obtained. Sequence reads were filtered for quality and then aligned to the mouse reference genome (GRCm38/mm10) using the BurrowsâWheeler Aligner implemented in speedseq software, which achieved at least 30Ã coverage for all lines (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> and Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033902/-/DC1/TableS1.xlsx">Table S1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469708/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469708/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">An overview of the mapping statistics and Chr 1 variants called from C1SLs</span><div class="sb-div caption-clear"></div></div></div><p id="p-24">We used SAMtools/BCFtools to call SNPs and indels from the reads of each sample, and these variants were custom filtered (see <em>Materials and Methods</em>). Overall, â¼29 million SNPs and 4.5 million indels (1â50 bp) were identified from the 18 Chr 1 sequences (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> and <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). For SNP genotypes, we found that 97.4% were homozygous and 2.6% were heterozygous on average (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033902/-/DC1/TableS2.xlsx">Table S2</a>), the latter most likely arising from true heterozygote loci due to incomplete fixation of the inbred strain, false-positive calls due to regions of copy number variation, sequencing errors, and mapping errors in highly repetitive genomic regions, as previously described (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Keane <em>et al.</em> 2011</a>). The sensitivity and specificity of our variant calls were established using 23-kb bases of DNA from the ZZ2 line. We sequenced 81 PCR segments spread over Chr 1 using Sanger sequencing. A total of 352 SNPs were detected. None of the SNPs were missed by whole genome sequencing, resulting in a false negative rate of 0% in our next-generation SNP calls. Two SNPs identified by whole genome sequencing were not found in Sanger sequencing, which gave a false positive rate of 0.57%.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/11/3571/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Overview of variants called from C1SLs. (A) Distribution of SNPs, indels, private variants, and structural variants on Chr 1 in C1SLs. (B) Venn diagram of comparison between C1SLs and MGP and dbSNP142 data sets of identified SNPs and indels on Chr 1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1267766082" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Overview of variants called from C1SLs. (A) Distribution of SNPs, indels, private variants, and structural variants on Chr 1 in C1SLs. (B) Venn diagram of comparison between C1SLs and MGP and dbSNP142 data sets of identified SNPs and indels on Chr 1.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/11/3571/F1.medium.gif" width="440" height="358"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/11/3571/F1.medium.gif" width="440" height="358"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/11/3571/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/11/3571/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469683" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-25" class="first-child">Overview of variants called from C1SLs. (A) Distribution of SNPs, indels, private variants, and structural variants on Chr 1 in C1SLs. (B) Venn diagram of comparison between C1SLs and MGP and dbSNP142 data sets of identified SNPs and indels on Chr 1.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">Compared with CI strains, we found large amounts of genetic variants in each line. There were, on average, 1.68 million SNPs and 0.25 million indels in our sequenced lines, except for strain KM (0.5 million SNPs and 0.09 million indels). However, for sequenced 36 strains from MGP, only 0.31 million SNPs were called for each strain, except for seven wild-derived inbred strains (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033902/-/DC1/TableS3.xlsx">Table S3</a>) (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Keane <em>et al.</em> 2011</a>). SNP density across Chr 1 was 8.2 SNP/kb (SD = 1.2), which was fivefold more than CI strains (1.6 SNP/kb, SD = 0.5) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033902/-/DC1/TableS3.xlsx">Table S3</a>). When the 18 lines were compared, each line carried relatively few private variants (mainly &lt;3% of all variants called in each line), except for KM (27%) and DX (8%) (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>), indicating that these two had large divergence from the other lines. Next, we merged all the variants from these lines and identified â¼4.5 million (4,523,955) unique variants, including 3,826,868 SNPs and 697,087 indels. Further comparison with entries in MGP and NCBI dbSNP142 variant data sets identified 2.7 million SNPs (71%) as known, and the remaining SNPs (1,099,486; 29%) were classified as novel. For indels, 450,043 (64.5%) had been described previously (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>).</p><p id="p-27">To identify SVs from the information of split and discordant reads and changes in mapped read depth, lumpy-sv implemented in speedseq software was used. We called 0.09 million SVs, including 83,146 deletions (&gt;50 bp), 1461 duplications, and 655 inversions (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a> and <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1A</a>).</p></div><div id="sec-16" class="subsection"><h3>Variants annotation</h3><p id="p-28">To understand the functional consequences of the variants called in these 18 donor chromosomes, we annotated these variants using the VEP from Ensembl (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). Within the total set of 4.5 million variants, 0.47% (21,335) were in protein-coding regions, among which 38.3% (8181) were annotated to be amino acid (AA)-altering from 1002 protein-coding genes. Of the 8181 AA-altering variants, 130 indels caused a disruption of the translational reading frame (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033902/-/DC1/TableS4.xlsx">Table S4</a>); 64 were predicted as premature truncation of the protein due to gain of stop codons, whereas 24 were predicted as loss of stop codons (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033902/-/DC1/TableS5.xlsx">Table S5</a>). Apart from the AA-altering variants, the other potentially functional categories included: variants with the potential to disrupt splicing events (3,841; 0.08%); variants in 3â² and 5â² untranslated regions (UTRs) with the potential to regulate protein translation (49,163; 1.1%); those within 5 kb upstream or downstream of transcription start or end sites with possible roles on transcriptional regulation (1,038,410; 23%). As shown in <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>, we also found that a large amount of novel variants fell into coding sequences and potentially regulatory regions, including 3009 missense variants, 45 stop gains, six stop loss, and 65 frameshift variants potentially having loss of function effects, and those including 1036 splicing variants, 14,154 3â² and 5â² UTR variants, and 303,537 5-kb upstream or downstream variants with potentially regulatory effect.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469650/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469650/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Predicted functional consequence of SNPs and indels</span><div class="sb-div caption-clear"></div></div></div><p id="p-30">Another important question was whether there were more functional variants on Chr 1 for each line than the previously sequenced CI strains. To address this question, we compared variant distribution and number of involved genes with published data in those potentially functional categories. The results from this analysis showed that C1SLs had 4.9-fold (SD = 1.0) more variants in each category compared with MGP-sequenced CI strains. The number of genes involved were 3.8-fold (SD = 1.8) more (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). For the number of variants per gene in each category, we found that there were more variants in potentially regulatory regions, especially in upstream or downstream gene regions and 3â² UTR regions.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/11/3571/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of potentially functional variants on chromosome between C1SLs and 31 MGP-sequenced CI strains. (A) Number of short variants in each categories. (B) Number of genes involved in each categories. All comparisons were statistically significant using a two-sample t-test (P &lt; 0.01)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1267766082" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison of potentially functional variants on chromosome between C1SLs and 31 MGP-sequenced CI strains. (A) Number of short variants in each categories. (B) Number of genes involved in each categories. All comparisons were statistically significant using a two-sample &lt;em&gt;t&lt;/em&gt;-test (&lt;em&gt;P&lt;/em&gt; &lt; 0.01).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/11/3571/F2.medium.gif" width="440" height="298"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/11/3571/F2.medium.gif" width="440" height="298"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/11/3571/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/11/3571/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469739" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-31" class="first-child">Comparison of potentially functional variants on chromosome between C1SLs and 31 MGP-sequenced CI strains. (A) Number of short variants in each categories. (B) Number of genes involved in each categories. All comparisons were statistically significant using a two-sample <em>t</em>-test (<em>P</em> &lt; 0.01).</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">Among the protein-coding genes with AA-altering SNPs, 483 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033902/-/DC1/TableS6.xlsx">Table S6</a>) genes were predicted to have deleterious missense SNPs (SIFT score &lt; 0.05). We annotated these genes and those with stop gain or loss and frameshift genes together to the KEGG pathway using the Database for Annotation, Visualization, and Integrated Discovery (DAVID) web-based tools (Huang da <em>et al.</em> 2009). This analysis resulted in 138 genes from 60 known pathways (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033902/-/DC1/TableS7.xlsx">Table S7</a>). The first interesting finding was that 13 genes were involved in immune-related pathways, including the complement and coagulation cascades pathway and natural killer cellâmediated cytotoxicity pathway. We also found that there were 19 genes participating in olfactory transduction, most of which were olfactory receptor genes. Other pathways involving relatively more genes were cytokineâcytokine receptor interaction (seven genes) and adhesion molecules (seven genes).</p><p id="p-33">There were 268 large deletions involving 157 genes. However, among them, only 30 were protein-coding genes, and others were pseudogenes or predicted genes. We also found 24 stop loss deletions, affecting 11 protein-coding genes.</p></div><div id="sec-17" class="subsection"><h3>Haplotype block analysis</h3><p id="p-34">To capture the haplotype structure of the donor chromosomes, we randomly selected 126 nonoverlapping intervals in which each interval contained about 10,000 SNPs, on average 0.5 Mb per region, with sizes ranging from 0.3 to 1 Mb, and completely encompassing a 64 Mb region of the Chr 1 sequences. For each region, the haplotype block analysis was performed according to the principle of the four gamete rule (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Wang <em>et al.</em> 2002</a>).</p><p id="p-35">A total of 28,105 blocks were identified on these regions. The number of blocks varied for each region, ranging from 127 to 982. The average number of blocks was 221. There were six regions with an extremely large number of blocks (&gt;88; mean + 2 SD of block number per 100 kb), indicating that these regions had many recombination events and may be recombination hotspots. From the block length distribution, nearly 56% of blocks were &lt;1 kb in length, only 4% (1081 blocks) were â¥10 kb, and the largest block was 103.8 kb, while other blocks were between 1 and 10 kb (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). These results suggest that the haplotype blocks of the donor chromosomes are consistent with the previous study, where the haplotype blocks were mainly &lt;100 kb (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Laurie <em>et al.</em> 2007</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/11/3571/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of haplotype block length across Chr 1 of C1SLs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1267766082" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Distribution of haplotype block length across Chr 1 of C1SLs.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/11/3571/F3.medium.gif" width="440" height="244"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/11/3571/F3.medium.gif" width="440" height="244"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/11/3571/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/11/3571/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469707" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-36" class="first-child">Distribution of haplotype block length across Chr 1 of C1SLs.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-18" class="subsection"><h3>Subspecies origin</h3><p id="p-37">We examined the ancestral subspecies origin of the 18 donor chromosomes, expecting to identify a mosaic of segments that could be assigned to one of three distinct lineages: <em>Mus musculus domesticus</em>, <em>M. m. musculus</em>, and <em>M. m. castaneus</em>. To assign subspecies origins to genomic intervals of the 18 lines, three wild-derived inbred strains, WSB, PWK and CAST, respectively, were chosen to represent the three lineages. We segmented each lineâs Chr 1 sequence into 19,548 10-kb blocks and compared sequence similarity for each block with the corresponding sequence from WSB, PWK, and CAST to infer the subspecies origin for each segment. In this study, for each C1SL, we considered the regions with &lt;5 SNP differences between this line and any two wild-derived inbred strains to be of undetermined origin, and excluded regions were considered to be subspecies that introgressed among those three wild-derived strains (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Yang <em>et al.</em> 2007</a>) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033902/-/DC1/TableS8.xlsx">Table S8</a>). This left an average of 63.2% (SD = 1.8%) of Chr 1 regions (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033902/-/DC1/TableS9.xlsx">Table S9</a>) in each line for similarity analysis.</p><p id="p-38">The results showed that all lines were a mosaic of regions that had all three subspecies origins (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). However, each subspeciesâ contribution was quite different (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>). KM had a large number of blocks with high sequence similarity to WSB, consistent with a previous report that KM mice are derived from Swiss mice (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Zhang <em>et al.</em> 2007</a>), which originated from <em>M. m. domesticus</em>. For the other 17 lines captured from different locations in China (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4C</a>), the sequences were mainly originated from <em>M. m. musculus</em> (68%, SD = 15%) and <em>M. m. castaneus</em> (25%, SD = 13%), which explained 93% (SD = 2%) of the Chr 1 sequence. A previous study found that there were two mouse subspecies, <em>M. m. musculus</em> and <em>M. m. castaneus</em>, in China, with the former mainly located in the north and the latter in the south (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Xiao <em>et al.</em> 2010</a>). However, we did not observe this distinct geographic distribution. Nonetheless, we found a consistent trend in the subspecies distribution. For example, the sequence of SY, which was from north of China, was largely contributed by <em>M. m. musculus</em> (89%) whereas the sequence of DX, which was from south of China, was largely contributed by <em>M. m. castaneus</em> (66%).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/11/3571/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Subspecies origin of Chr 1 of C1SLs. (A) Subspecies assignments for the 18 donor chromosomes. Each 10-kb interval is shown as a vertical bar of a color that reflects its subspecies origin. Intervals without color are regions of intersubspecies introgression or of undetermined status. (B) Contribution of the three main subspecies lineages in Chr 1 of C1SLs. (C) Wild mice collection sites in China." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1267766082" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Subspecies origin of Chr 1 of C1SLs. (A) Subspecies assignments for the 18 donor chromosomes. Each 10-kb interval is shown as a vertical bar of a color that reflects its subspecies origin. Intervals without color are regions of intersubspecies introgression or of undetermined status. (B) Contribution of the three main subspecies lineages in Chr 1 of C1SLs. (C) Wild mice collection sites in China.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/11/3571/F4.medium.gif" width="400" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/11/3571/F4.medium.gif" width="400" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/11/3571/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/11/3571/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469622" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-39" class="first-child">Subspecies origin of Chr 1 of C1SLs. (A) Subspecies assignments for the 18 donor chromosomes. Each 10-kb interval is shown as a vertical bar of a color that reflects its subspecies origin. Intervals without color are regions of intersubspecies introgression or of undetermined status. (B) Contribution of the three main subspecies lineages in Chr 1 of C1SLs. (C) Wild mice collection sites in China.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-19" class="subsection"><h3>Phenotypic diversity</h3><p id="p-40">An ideal mouse resource for QTL mapping should not only have extensive genetic diversity, but also high phenotypic diversity. For CSSs, they should have some phenotypic differences from the recipient strain. In order to determine the phenotypic diversity, we examined blood chemistry indexes in four lines (CM, HZ, KM, and ZZ2) and recipient strain B6. Of the 12 indexes assayed, all except GLU had some significant differences between B6 and these four lines (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>). HZ exhibited differences in six indexes (TP, ALT, AST, CHOL, HDL-C, and LDL-C), the most among these four lines (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a>). Interestingly, three blood lipid indexes (CHOL, HDL-C, and LDL-C) were significantly higher in HZ than B6, suggesting that HZ may have lipid metabolism disorders. We assayed gene expression levels in liver tissues to determine if these differences were reflected on intermediate phenotypes (gene expression levels). This assay contained 65,956 mouse genes, including 36,703 noncoding RNAs. After quality control and normalization, we identified 434 differentially expressed genes (fold change &gt; 2 and <em>P</em> &lt; 0.05), in which 130 genes were upregulated and 304 were downregulated compared to B6. Among these genes, 94 (22%) were located on Chr 1 whereas others were distributed on other chromosomes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033902/-/DC1/FigureS1.tif">Figure S1</a>). Next, we performed gene ontology enrichment analysis using DAVID online tools (Huang da <em>et al.</em> 2009). This analysis yielded a list of 183 genes (excluding noncoding and unmapped genes). The results showed a significant enrichment of 32 PANTHER biological process categories. <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> lists the top 10 enrichment categories, most of which were associated with lipid metabolism, such as CHOL biosynthetic and metabolic process, sterol biosynthetic and metabolic process, and lipid biosynthetic process.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/11/3571/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of phenotypes between C1SLs and recipient strain B6. (A) Comparison of 12 biochemistry indexes between CM, HZ, KM, and ZZ2, and recipient strain B6. Red: P &lt; 0.01; green: P &lt; 0.05. (B) Comparisons of six significant indexes between HZ and B6." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1267766082" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison of phenotypes between C1SLs and recipient strain B6. (A) Comparison of 12 biochemistry indexes between CM, HZ, KM, and ZZ2, and recipient strain B6. Red: &lt;em&gt;P&lt;/em&gt; &lt; 0.01; green: &lt;em&gt;P&lt;/em&gt; &lt; 0.05. (B) Comparisons of six significant indexes between HZ and B6.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/11/3571/F5.medium.gif" width="407" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/11/3571/F5.medium.gif" width="407" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/11/3571/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/11/3571/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469634" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-41" class="first-child">Comparison of phenotypes between C1SLs and recipient strain B6. (A) Comparison of 12 biochemistry indexes between CM, HZ, KM, and ZZ2, and recipient strain B6. Red: <em>P</em> &lt; 0.01; green: <em>P</em> &lt; 0.05. (B) Comparisons of six significant indexes between HZ and B6.</p><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469561/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469561/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Top 10 enriched gene ontology terms from a list of 183 genes differentially expressed between B6 and HZ</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-20" class="subsection"><h3>Quality control of Chr 1 and genetic background</h3><p id="p-43">An ideal CSS should carry an intact donor chromosome and &gt;99% of other genetic background identical to the recipient strain. Theoretically, with more generations of backcrosses, the genome background becomes increasingly pure while the probability of recombination event on the donor chromosome also increases (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gregorova <em>et al.</em> 2008</a>). Among C1SLs, two lines, ZC and HZ, had a recombination on Chr 1 proximal sequence. As a result, these two linesâ Chr 1 proximal sequences became the B6 sequence. Similarly, the Chr 1 distal sequences of four lines (PD, TZ, FX, and BLD) were replaced by the B6 sequence (details in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033902/-/DC1/TableS10.xlsx">Table S10</a>). For JD, we found a 5.7 Mb B6 sequence in the middle of Chr 1 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033902/-/DC1/TableS10.xlsx">Table S10</a>). For the other chromosomes from the genetic background, the residual contamination varied among these lines, ranging from 0.5 to 5.1% with an average of 1.7%, and six lines had &gt;2% contamination (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>). As shown in <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6B</a>, the wild island distributed across all autosomes, but some chromosomes such as chromosomes 4, 8, and 18 harbored more islands. Some islands were very large in certain lines. For example, BLD had a 7-Mb island on chromosome 3, DX had a 54-Mb island on chromosome 8, and ZZ2 had a 66-Mb island on chromosome 2. In addition, we also found that there were some overlapping islands in all of the 18 lines.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/11/3571/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Wild island contamination across genome background for C1SLs. (A) Overall wild island contamination for each line. (B) Wild island distribution across the genome background (chromosomes 2 - 19). Brown bars represent wild islands. The C1SLs from inside out are ZZ2, ZZ1, ZC, YX, YP, TZ, TW, SY, SMX, QP, PD, KM, JD, HZ, FX, DX, CM, and BLD." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1267766082" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Wild island contamination across genome background for C1SLs. (A) Overall wild island contamination for each line. (B) Wild island distribution across the genome background (chromosomes 2 - 19). Brown bars represent wild islands. The C1SLs from inside out are ZZ2, ZZ1, ZC, YX, YP, TZ, TW, SY, SMX, QP, PD, KM, JD, HZ, FX, DX, CM, and BLD.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/11/3571/F6.medium.gif" width="353" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/6/11/3571/F6.medium.gif" width="353" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/11/3571/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/11/3571/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469674" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-44" class="first-child">Wild island contamination across genome background for C1SLs. (A) Overall wild island contamination for each line. (B) Wild island distribution across the genome background (chromosomes 2 - 19). Brown bars represent wild islands. The C1SLs from inside out are ZZ2, ZZ1, ZC, YX, YP, TZ, TW, SY, SMX, QP, PD, KM, JD, HZ, FX, DX, CM, and BLD.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-21"><h2 class="">Discussion</h2><p id="p-45">In 2008, we began a project to construct a population of C1SLs. After years of hard work, this project is near completion. In the present study, we performed whole genome sequencing on C1SLs from 17 Chinese wild mice and one KM mouse. We detected the genetic polymorphisms and examined their genetic structure, including their subspecies origin and haplotype structure. We also selected some lines to test the blood biochemical indexes. With all these genome sequences and limited phenotype data, we assessed the value of these lines for the genetic studies of complex traits, and for QTL fine mapping and detecting new QTL on Chr 1.</p><p id="p-46">For the genetic studies of complex traits, high genetic diversity is essential for identifying causative genes. However, CI strains are originated from a limited progenitor, which constraints their genetic diversity (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Yang <em>et al.</em> 2011</a>). Recent whole genome sequencing of 36 inbred strains showed that most of the identified variants were from wild-derived strains (<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Keane <em>et al.</em> 2011</a>). The SNP density for CI strains was only 1.6 SNP/kb (SD = 0.5) which was far less than wild-derived strains. Consistent with previous studies that wild mice harbor more genetic variants (<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Laurie <em>et al.</em> 2007</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Salcedo <em>et al.</em> 2007</a>), in our sequenced lines, we completely identified 4.5 million SNPs and indels on donor chromosomes, of which 1.3 million were not in the MGP and NCBI mouse dbSNP142 data sets. We identified nearly 1.6 million SNPs for each line, which was fivefold more than those sequenced CI strains (0.31 million). Further annotation also identified many loss of functional variants, especially among the novel variants (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>).</p><p id="p-47">Another essential factor for the dissection of complex traits is mapping resolution. Recent progress in mouse resources have achieved a resolution of a â¼2 Mb interval containing roughly 10â20 genes (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Buchner and Nadeau 2015</a>), although the actual resolution varies with other factors, such as gene density. Other complementary methods, such as gene expression analysis and coexpression networks, can sometimes reduce the number of candidates to just one or a few genes and inform the underlying pathways (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Mesner <em>et al.</em> 2014</a>). Through haplotype analysis across these 18 lines, we found that the sizes of most haplotype blocks in our lines were &lt;100 kb, with nearly 57% blocks &lt;1 kb. This indicates that single gene resolution may be achieved when conducting association studies with C1SLs.</p><p id="p-48">Unlike laboratory mice maintained under well controlled environments, wild mice have spread around, and adapted to, many different environmental pressures. Therefore, they should exhibit considerably more phenotypic variations (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Phifer-Rixey and Nachman 2015</a>). In this study, we found most of our assayed blood chemistry indexes had significant differences with B6. The HZ line showed significantly higher blood lipids (HDL-C, LDL-C, and CHOL) and liver damageârelated indexes (TP, ALT, and AST). This was also reflected on expression levels of blood lipid metabolismârelated genes. Through functional annotations of the identified genetic variants, we found 13 genes with deleterious coding sequence variants associated with immune-related pathways, including the complement and coagulation cascades pathway and natural killer cellâmediated cytotoxicity pathway, indicating that they have some differences in susceptibility of infectious diseases with B6. Indeed, there are reports that wild mice have different infectious disease susceptibility (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Turner and Paterson 2013</a>). In addition, these lines also harbored more functional variants on olfactory transduction genes, which may be related to their adaption to the environment. All these findings suggest that these newly built C1SLs can be used for the genetic analysis of many different phenotypes.</p><p id="p-49">Our C1SLs also provide a potentially new resource for gene regulation and geneâgene interaction studies. Due to their special genetic makeup, for each gene or regulatory locus, there are at least two haplotypes and may be up to 19 from these mice lines, which may lead to improved power in gene regulation studies. In our sequenced lines, we found large amount of variants located in regulatory regions, suggesting <em>cis</em> regulation may exist for these nearby genes. Another finding was that nearly two-thirds of differentially expressed genes in the liver (between B6 and HZ) were located outside Chr 1. This difference may be triggered by the genetic variants on Chr 1 through <em>trans</em> regulations. Thus, this unique resource may provide more regulation information on the genes located on Chr 1.</p><p id="p-50">The house mouse, <em>M. musculus</em>, consists of three principal subspecies, with native populations of <em>M. m. musculus</em> in Eastern Europe and Asia, <em>M. m. castaneus</em> in Southeast Asia and India, and <em>M. m. domesticus</em> in Western Europe and the Middle East (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Boursot <em>et al.</em> 1993</a>). CI strains are predominantly derived from <em>M. m. domesticus</em>, which accounted for about 94.3% of genome sequences (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Yang <em>et al.</em> 2011</a>). Our subspecies origin analysis on Chr 1 confirmed that nearly 93% sequences were from <em>M. m. musculus</em> and <em>M. m. castaneus</em>. The overall contribution for two subspecies for each line are consistent with the geographical distribution of subspecies in China. Thus, these newly constructed C1SLs have three subspecies of origin: for Chr 1, they predominantly originated from <em>M. m. musculus</em> and <em>M. m. castaneus</em>, while other chromosomes were mainly of <em>M. m. domesticus</em> origin.</p><p id="p-51">Theoretically, â¥10 generations of backcrossing are required for purifying the CSS genome background (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Nadeau <em>et al.</em> 2012</a>), which is very time-consuming and laborious. In order to shorten the inbreeding time, we backcrossed seven generations in our lines. Through whole genome sequencing, we found that some lines were nearly purified with B6 background (CM, FX, JD, QP, SY, YX, and ZC), whereas others had &lt;5% residual contamination in the genome background with wild islands. The residual contamination is mainly caused by the limited number of backcrosses. It may also be due to positive selection for these wild islands, which means that once these wild islands are replaced by B6 sequences, it may cause genetic incompatibility, leading to problems such as reproductive isolation (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Ma <em>et al.</em> 2016</a>). However, these limited islands can be easily traced through SNP genotyping to determine if the fixation status is B6 or wild allele in the future intercrossing, which can be used to guide further research.</p><p id="p-52">Until now, nearly 1000 QTL have been mapped onto mouse Chr 1. However, only a small number of the underlying causative genes have been identified (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Eppig <em>et al.</em> 2014</a>). As a unique mouse resource, together with the accumulated phenotypic information, the sequence of the 18 mouse genomes will serve as a basis for understanding trait differences. They can be also used for fine mapping known QTL or identifying new QTL on mouse Chr 1, and will help dissect the genetic basis of complex traits.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-53">This work was supported by the National Natural Science Foundation of China (grant no. 31371257) to J.X. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-4"><p id="p-54">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033902/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033902/-/DC1</a>.</p></li><li class="fn" id="fn-5"><p id="p-55"><em>Communicating editor: A. S. McCallion</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-05-24"><span class="received-label">Received </span>May 24, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-08-30"><span class="accepted-label">Accepted </span>August 30, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright Â© 2016 Xu <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited. </p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.11.3571.1" data-doi="10.1093/hmg/ddr206"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ahlqvist</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ekman</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lindvall</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Popovic</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forster</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">High-resolution mapping of a complex disease, a model for rheumatoid arthritis, using heterogeneous stock mice.</span> <abbr class="cit-jnl-abbrev">Hum. Mol. Genet.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">3031</span>â<span class="cit-lpage">3041</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHum.%2BMol.%2BGenet.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fhmg%252Fddr206%26rft_id%253Dinfo%253Apmid%252F21565963%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiaG1nIjtzOjU6InJlc2lkIjtzOjEwOiIyMC8xNS8zMDMxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNTcxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.11.3571.2" data-doi="10.1101/gr.111310.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aylor</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foulds-Mathes</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buus</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verdugo</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Genetic analysis of complex traits in the emerging collaborative cross.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1213</span>â<span class="cit-lpage">1222</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.111310.110%26rft_id%253Dinfo%253Apmid%252F21406540%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIxLzgvMTIxMyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzU3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.11.3571.3" data-doi="10.1093/bioinformatics/bth457"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barrett</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fry</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maller</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daly</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">Haploview: analysis and visualization of LD and haplotype maps.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">263</span>â<span class="cit-lpage">265</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbth457%26rft_id%253Dinfo%253Apmid%252F15297300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyMS8yLzI2MyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzU3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.11.3571.4" data-doi="10.1101/gr.099234.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bennett</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farber</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Orozco</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ghazalpour</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">A high-resolution association mapping panel for the dissection of complex traits in mice.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">281</span>â<span class="cit-lpage">290</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.099234.109%26rft_id%253Dinfo%253Apmid%252F20054062%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIwLzIvMjgxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNTcxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.11.3571.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bennett</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Civelek</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Orozco</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">Genetic architecture of atherosclerosis in mice: a systems genetics analysis of common inbred strains.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e1005711</span> (erratum: PLoS Genet <span class="cit-vol">12</span>: e1005913).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D12%26rft.spage%253De1005711%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.11.3571.6" data-doi="10.1146/annurev.es.24.110193.001003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boursot</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auffray</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brittondavidian</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonhomme</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span>â<span class="cit-article-title">The evolution of house mice.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Ecol. Syst.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">119</span>â<span class="cit-lpage">152</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BEcology%2Band%2BSystematics%26rft.stitle%253DAnnu.%2BRev.%2BEcol.%2BSyst.%26rft.aulast%253DBoursot%26rft.auinit1%253DP.%26rft.volume%253D24%26rft.issue%253D1%26rft.spage%253D119%26rft.epage%253D152%26rft.atitle%253DThe%2BEvolution%2Bof%2BHouse%2BMice%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.es.24.110193.001003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.es.24.110193.001003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1993MJ37100005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.11.3571.7" data-doi="10.1101/gr.187450.114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buchner</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nadeau</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">Contrasting genetic architectures in different mouse reference populations used for studying complex traits.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">775</span>â<span class="cit-lpage">791</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.187450.114%26rft_id%253Dinfo%253Apmid%252F25953951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI1LzYvNzc1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNTcxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.11.3571.8" data-doi="10.1038/nmeth.3505"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chiang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Layer</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faust</span>  <span class="cit-name-given-names">G. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lindberg</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rose</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">SpeedSeq: ultra-fast personal genome analysis and interpretation.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">966</span>â<span class="cit-lpage">968</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D12%26rft.spage%253D966%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.3505%26rft_id%253Dinfo%253Apmid%252F26258291%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.3505&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26258291&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3571.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.11.3571.9" data-doi="10.1093/bioinformatics/btr330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Danecek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albers</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">The variant call format and VCFtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2156</span>â<span class="cit-lpage">2158</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr330%26rft_id%253Dinfo%253Apmid%252F21653522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjcvMTUvMjE1NiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzU3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.11.3571.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DeSantis</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doerner</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ko</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawasoe</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Genetic resistance to liver fibrosis on A/J mouse chromosome 17.</span> <abbr class="cit-jnl-abbrev">Alcohol. Clin. Exp. Res.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">1668</span>â<span class="cit-lpage">1679</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAlcohol.%2BClin.%2BExp.%2BRes.%26rft.volume%253D37%26rft.spage%253D1668%26rft_id%253Dinfo%253Apmid%252F23763294%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23763294&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3571.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.11.3571.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eppig</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blake</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bult</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kadin</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richardson</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">The Mouse Genome Database (MGD): facilitating mouse as a model for human biology and disease.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">D726</span>â<span class="cit-lpage">D736</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D43%26rft.spage%253DD726%26rft_id%253Dinfo%253Apmid%252F25348401%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=25348401&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3571.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.11.3571.12" data-doi="10.1038/nrg3335"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flint</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eskin</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Genome-wide association studies in mice.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">807</span>â<span class="cit-lpage">817</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DFlint%26rft.auinit1%253DJ.%26rft.volume%253D13%26rft.issue%253D11%26rft.spage%253D807%26rft.epage%253D817%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudies%2Bin%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3335%26rft_id%253Dinfo%253Apmid%252F23044826%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3335&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23044826&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3571.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.11.3571.13" data-doi="10.1038/nrg1576"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flint</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shifman</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mott</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">Strategies for mapping and cloning quantitative trait genes in rodents.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">271</span>â<span class="cit-lpage">286</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DFlint%26rft.auinit1%253DJ.%26rft.volume%253D6%26rft.issue%253D4%26rft.spage%253D271%26rft.epage%253D286%26rft.atitle%253DStrategies%2Bfor%2Bmapping%2Band%2Bcloning%2Bquantitative%2Btrait%2Bgenes%2Bin%2Brodents.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1576%26rft_id%253Dinfo%253Apmid%252F15803197%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1576&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15803197&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3571.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228093000013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.11.3571.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">French</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gatti</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kissling</span>  <span class="cit-name-given-names">G. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shockley</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">Diversity outbred mice identify population-based exposure thresholds and genetic factors that influence benzene-induced genotoxicity.</span> <abbr class="cit-jnl-abbrev">Environ. Health Perspect.</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">237</span>â<span class="cit-lpage">245</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEnviron.%2BHealth%2BPerspect.%26rft.volume%253D123%26rft.spage%253D237%26rft_id%253Dinfo%253Apmid%252F25376053%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=25376053&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3571.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.11.3571.15" data-doi="10.1007/s00335-012-9411-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ghazalpour</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rau</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farber</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bennett</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Orozco</span>  <span class="cit-name-given-names">L. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Hybrid mouse diversity panel: a panel of inbred mouse strains suitable for analysis of complex genetic traits.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">680</span>â<span class="cit-lpage">692</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DGhazalpour%26rft.auinit1%253DA.%26rft.volume%253D23%26rft.issue%253D9-10%26rft.spage%253D680%26rft.epage%253D692%26rft.atitle%253DHybrid%2Bmouse%2Bdiversity%2Bpanel%253A%2Ba%2Bpanel%2Bof%2Binbred%2Bmouse%2Bstrains%2Bsuitable%2Bfor%2Banalysis%2Bof%2Bcomplex%2Bgenetic%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-012-9411-5%26rft_id%253Dinfo%253Apmid%252F22892838%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-012-9411-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22892838&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3571.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309543200019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.11.3571.16" data-doi="10.1101/gr.7160508"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gregorova</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Divina</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Storchova</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trachtulec</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fotopulosova</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Mouse consomic strains: exploiting genetic divergence between Mus m. musculus and Mus m. domesticus subspecies.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">509</span>â<span class="cit-lpage">515</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.7160508%26rft_id%253Dinfo%253Apmid%252F18256238%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE4LzMvNTA5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNTcxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.11.3571.17" data-doi="10.1038/nprot.2008.211"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherman</span>  <span class="cit-name-given-names">B. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lempicki</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">44</span>â<span class="cit-lpage">57</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DHuang%26rft.auinit1%253Dd.%2Ba.%2B%2B.%2BW.%26rft.volume%253D4%26rft.issue%253D1%26rft.spage%253D44%26rft.epage%253D57%26rft.atitle%253DSystematic%2Band%2Bintegrative%2Banalysis%2Bof%2Blarge%2Bgene%2Blists%2Busing%2BDAVID%2Bbioinformatics%2Bresources.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2008.211%26rft_id%253Dinfo%253Apmid%252F19131956%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2008.211&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19131956&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3571.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265781800006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.11.3571.18" data-doi="10.1038/hdy.2014.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kato</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ishii</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nishi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuriki</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koide</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">Segregation of a QTL cluster for home-cage activity using a new mapping method based on regression analysis of congenic mouse strains.</span> <abbr class="cit-jnl-abbrev">Heredity (Edinb)</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">416</span>â<span class="cit-lpage">423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%2B%2528Edinb%2529%26rft.volume%253D113%26rft.spage%253D416%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2014.42%26rft_id%253Dinfo%253Apmid%252F24781804%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2014.42&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24781804&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3571.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.11.3571.19" data-doi="10.1038/nature10413"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keane</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodstadt</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danecek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Mouse genomic variation and its effect on phenotypes and gene regulation.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">477</span>: <span class="cit-fpage">289</span>â<span class="cit-lpage">294</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKeane%26rft.auinit1%253DT.%2BM.%26rft.volume%253D477%26rft.issue%253D7364%26rft.spage%253D289%26rft.epage%253D294%26rft.atitle%253DMouse%2Bgenomic%2Bvariation%2Band%2Bits%2Beffect%2Bon%2Bphenotypes%2Band%2Bgene%2Bregulation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10413%26rft_id%253Dinfo%253Apmid%252F21921910%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10413&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21921910&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3571.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000294852400022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.11.3571.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Laurie</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nickerson</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">B.S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Livingston</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Linkage disequilibrium in wild mice.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">3</span>: e144.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.11.3571.21" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">The sequence alignment/map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>â<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTYvMjA3OCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzU3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.11.3571.22" data-doi="10.1016/j.cmet.2016.06.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marti Gutierrez</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morey</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Dyken</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span>â<span class="cit-article-title">Incompatibility between nuclear and mitochondrial genomes contributes to an interspecies reproductive barrier.</span> <abbr class="cit-jnl-abbrev">Cell Metab.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">283</span>â<span class="cit-lpage">294</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BMetab.%26rft.volume%253D24%26rft.spage%253D283%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cmet.2016.06.012%26rft_id%253Dinfo%253Apmid%252F27425585%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cmet.2016.06.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27425585&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3571.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.11.3571.23" data-doi="10.1093/bioinformatics/btq330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McLaren</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rios</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flicek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Deriving the consequences of genomic variants with the Ensembl API and SNP Effect Predictor.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">2069</span>â<span class="cit-lpage">2070</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq330%26rft_id%253Dinfo%253Apmid%252F20562413%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjYvMTYvMjA2OSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzU3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.11.3571.24" data-doi="10.1172/JCI73072"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mesner</span>  <span class="cit-name-given-names">L. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ray</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hsu</span>  <span class="cit-name-given-names">Y. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manichaikul</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lum</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title"><em>Bicc1</em> is a genetic determinant of osteoblastogenesis and bone mineral density.</span> <abbr class="cit-jnl-abbrev">J. Clin. Invest.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">2736</span>â<span class="cit-lpage">2749</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BClin.%2BInvest.%26rft.volume%253D124%26rft.spage%253D2736%26rft_id%253Dinfo%253Adoi%252F10.1172%252FJCI73072%26rft_id%253Dinfo%253Apmid%252F24789909%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1172/JCI73072&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24789909&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3571.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.11.3571.25" data-doi="10.1038/73427"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nadeau</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singer</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span>â<span class="cit-article-title">Analysing complex genetic traits with chromosome substitution strains.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">221</span>â<span class="cit-lpage">225</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DNadeau%26rft.auinit1%253DJ.%2BH.%26rft.volume%253D24%26rft.issue%253D3%26rft.spage%253D221%26rft.epage%253D225%26rft.atitle%253DAnalysing%2Bcomplex%2Bgenetic%2Btraits%2Bwith%2Bchromosome%2Bsubstitution%2Bstrains.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F73427%26rft_id%253Dinfo%253Apmid%252F10700173%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/73427&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10700173&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3571.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000085590600010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.11.3571.26" data-doi="10.1007/s00335-012-9426-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nadeau</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forejt</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takada</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shiroishi</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Chromosome substitution strains: gene discovery, functional analysis, and systems studies.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">693</span>â<span class="cit-lpage">705</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DNadeau%26rft.auinit1%253DJ.%2BH.%26rft.volume%253D23%26rft.issue%253D9-10%26rft.spage%253D693%26rft.epage%253D705%26rft.atitle%253DChromosome%2Bsubstitution%2Bstrains%253A%2Bgene%2Bdiscovery%252C%2Bfunctional%2Banalysis%252C%2Band%2Bsystems%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-012-9426-y%26rft_id%253Dinfo%253Apmid%252F22961226%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-012-9426-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22961226&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3571.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309543200020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.11.3571.27" data-doi="10.1371/journal.pone.0030619"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Patel</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mukesh</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">NGS QC toolkit: a toolkit for quality control of next generation sequencing data.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e30619</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPatel%26rft.auinit1%253DR.%2BK.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253De30619%26rft.epage%253De30619%26rft.atitle%253DNGS%2BQC%2BToolkit%253A%2Ba%2Btoolkit%2Bfor%2Bquality%2Bcontrol%2Bof%2Bnext%2Bgeneration%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0030619%26rft_id%253Dinfo%253Apmid%252F22312429%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0030619&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22312429&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3571.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.11.3571.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Phifer-Rixey</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nachman</span>  <span class="cit-name-given-names">M. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">Insights into mammalian biology from the wild house mouse <em>Mus musculus</em>.</span> <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">4</span>: e05959.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.11.3571.29" data-doi="10.1534/g3.115.020784"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rau</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parks</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eskin</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simecek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">High-density genotypes of inbred mouse strains: improved power and precision of association mapping.</span> <abbr class="cit-jnl-abbrev">G3</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">2021</span>â<span class="cit-lpage">2026</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.115.020784%26rft_id%253Dinfo%253Apmid%252F26224782%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjUvMTAvMjAyMSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzU3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.11.3571.30" data-doi="10.1534/genetics.107.079988"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salcedo</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geraldes</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nachman</span>  <span class="cit-name-given-names">M. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Nucleotide variation in wild and inbred mice.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">2277</span>â<span class="cit-lpage">2291</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSalcedo%26rft.auinit1%253DT.%26rft.volume%253D177%26rft.issue%253D4%26rft.spage%253D2277%26rft.epage%253D2291%26rft.atitle%253DNucleotide%2BVariation%2Bin%2BWild%2Band%2BInbred%2BMice%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.079988%26rft_id%253Dinfo%253Apmid%252F18073432%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny80LzIyNzciO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzExLzM1NzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.11.3571.31" data-doi="10.1534/genetics.111.132597"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Svenson</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gatti</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Welsh</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">High-resolution genetic mapping using the mouse diversity outbred population.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">437</span>â<span class="cit-lpage">447</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSvenson%26rft.auinit1%253DK.%2BL.%26rft.volume%253D190%26rft.issue%253D2%26rft.spage%253D437%26rft.epage%253D447%26rft.atitle%253DHigh-Resolution%2BGenetic%2BMapping%2BUsing%2Bthe%2BMouse%2BDiversity%2BOutbred%2BPopulation%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.132597%26rft_id%253Dinfo%253Apmid%252F22345611%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzIvNDM3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNTcxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.11.3571.32" data-doi="10.1038/ejhg.2013.274"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tim</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hastings</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sirisha</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Free</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brookes</span>  <span class="cit-name-given-names">A. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">GWAS central: a comprehensive resource for the comparison and interrogation of genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Eur. J. Hum. Genet.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">949</span>â<span class="cit-lpage">952</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEur.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D22%26rft.spage%253D949%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fejhg.2013.274%26rft_id%253Dinfo%253Apmid%252F24301061%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ejhg.2013.274&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24301061&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3571.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.11.3571.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Wild rodents as a model to discover genes and pathways underlying natural variation in infectious disease susceptibility.</span> <abbr class="cit-jnl-abbrev">Parasite Immunol.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">386</span>â<span class="cit-lpage">395</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DParasite%2BImmunol.%26rft.volume%253D35%26rft.spage%253D386%26rft_id%253Dinfo%253Apmid%252F23550923%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23550923&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3571.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000325542100008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.11.3571.34" data-doi="10.1038/ng1840"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Solberg</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gauguier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burnett</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klenerman</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Genome-wide genetic association of complex traits in heterogeneous stock mice.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">879</span>â<span class="cit-lpage">887</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DValdar%26rft.auinit1%253DW.%26rft.volume%253D38%26rft.issue%253D8%26rft.spage%253D879%26rft.epage%253D887%26rft.atitle%253DGenome-wide%2Bgenetic%2Bassociation%2Bof%2Bcomplex%2Btraits%2Bin%2Bheterogeneous%2Bstock%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1840%26rft_id%253Dinfo%253Apmid%252F16832355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1840&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16832355&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3571.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239325700016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.11.3571.35" data-doi="10.1086/344398"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chakraborty</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span>â<span class="cit-article-title">Distribution of recombination crossovers and the origin of haplotype blocks: the interplay of population history, recombination, and mutation.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">1227</span>â<span class="cit-lpage">1234</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DWang%26rft.auinit1%253DN.%26rft.volume%253D71%26rft.issue%253D5%26rft.spage%253D1227%26rft.epage%253D1234%26rft.atitle%253DDistribution%2Bof%2Brecombination%2Bcrossovers%2Band%2Bthe%2Borigin%2Bof%2Bhaplotype%2Bblocks%253A%2Bthe%2Binterplay%2Bof%2Bpopulation%2Bhistory%252C%2Brecombination%252C%2Band%2Bmutation.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F344398%26rft_id%253Dinfo%253Apmid%252F12384857%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/344398&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12384857&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3571.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000178884300023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.11.3571.36" data-doi="10.1007/s00335-010-9270-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xiao</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cai</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">A novel strategy for genetic dissection of complex traits: the population of specific chromosome substitution strains from laboratory and wild mice.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">370</span>â<span class="cit-lpage">376</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DXiao%26rft.auinit1%253DJ.%26rft.volume%253D21%26rft.issue%253D7-8%26rft.spage%253D370%26rft.epage%253D376%26rft.atitle%253DA%2Bnovel%2Bstrategy%2Bfor%2Bgenetic%2Bdissection%2Bof%2Bcomplex%2Btraits%253A%2Bthe%2Bpopulation%2Bof%2Bspecific%2Bchromosome%2Bsubstitution%2Bstrains%2Bfrom%2Blaboratory%2Band%2Bwild%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-010-9270-x%26rft_id%253Dinfo%253Apmid%252F20623355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-010-9270-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20623355&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3571.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.11.3571.37" data-doi="10.1007/s00335-012-9409-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yalcin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flint</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Association studies in outbred mice in a new era of full-genome sequencing.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">719</span>â<span class="cit-lpage">726</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DYalcin%26rft.auinit1%253DB.%26rft.volume%253D23%26rft.issue%253D9-10%26rft.spage%253D719%26rft.epage%253D726%26rft.atitle%253DAssociation%2Bstudies%2Bin%2Boutbred%2Bmice%2Bin%2Ba%2Bnew%2Bera%2Bof%2Bfull-genome%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-012-9409-z%26rft_id%253Dinfo%253Apmid%252F22847376%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-012-9409-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22847376&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3571.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309543200023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.11.3571.38" data-doi="10.1038/ng2087"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pardo-Manuel de Villena</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">On the subspecific origin of the laboratory mouse.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">1100</span>â<span class="cit-lpage">1107</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYang%26rft.auinit1%253DH.%26rft.volume%253D39%26rft.issue%253D9%26rft.spage%253D1100%26rft.epage%253D1107%26rft.atitle%253DOn%2Bthe%2Bsubspecific%2Borigin%2Bof%2Bthe%2Blaboratory%2Bmouse.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng2087%26rft_id%253Dinfo%253Apmid%252F17660819%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng2087&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17660819&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3571.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249122400019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.11.3571.39" data-doi="10.1038/ng.847"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Didion</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buus</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Subspecific origin and haplotype diversity in the laboratory mouse.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">648</span>â<span class="cit-lpage">655</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYang%26rft.auinit1%253DH.%26rft.volume%253D43%26rft.issue%253D7%26rft.spage%253D648%26rft.epage%253D655%26rft.atitle%253DSubspecific%2Borigin%2Band%2Bhaplotype%2Bdiversity%2Bin%2Bthe%2Blaboratory%2Bmouse.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.847%26rft_id%253Dinfo%253Apmid%252F21623374%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.847&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21623374&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3571.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.11.3571.40" data-doi="10.1016/S1673-8527(07)60022-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Microsatellite genotyping for four expected inbred mouse strains from KM Mice.</span> <abbr class="cit-jnl-abbrev">J. Genet. Genomics</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">214</span>â<span class="cit-lpage">222</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BGenet%2BGenomics%26rft.aulast%253DZhang%26rft.auinit1%253DX.%26rft.volume%253D34%26rft.issue%253D3%26rft.spage%253D214%26rft.epage%253D222%26rft.atitle%253DMicrosatellite%2Bgenotyping%2Bfor%2Bfour%2Bexpected%2Binbred%2Bmouse%2Bstrains%2Bfrom%2BKM%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1673-8527%252807%252960022-8%26rft_id%253Dinfo%253Apmid%252F17498618%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1673-8527(07)60022-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17498618&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3571.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/11/3571.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/11/3561" title="The Genetic Basis of Upland/Lowland Ecotype Divergence in Switchgrass (Panicum virgatum)" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/11/3581" title="Divergent DNA Methylation Provides Insights into the Evolution of Duplicate Genes in Zebrafish" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/11"><span>Volume 6 Issue 11, November 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/11.cover-source.jpg?itok=hTYn4PRy" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/11/3571.supplemental" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-MzYEa4CcevLx7_QSb9ricNFw-lmbdPdmQ4TDrKdJSHU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/11/3571" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/11/3571" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/11/3571" class="active" data-icon-position="" data-hide-link-title="0">Genome Sequencing of Chromosome 1 Substitution Lines Derived from Chinese Wild Mice Revealed a Unique Resource for Genetic Studies of Complex Traits</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460453" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-TVfIGF2yz0EDtBZ6Q5epNEy5iW2MUzuKsgt97a3pkV8" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/11/3571.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/11/3571" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-2IirINcE08CY1TD_bcViJFsplSccr5EzoTAvcEle4Is" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/11/3571" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/11/3571.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460453" id="node460453--2" data-pisa="ggg;6/11/3571" data-pisa-master="ggg;g3.116.033902" data-apath="/ggg/6/11/3571.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome Sequencing of Chromosome 1 Substitution Lines Derived from Chinese Wild Mice Revealed a Unique Resource for Genetic Studies of Complex Traits</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-5952-5771" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Fuyi Xu</span>, <span class="highwire-citation-author" data-delta="1">Tianzhu Chao</span>, <span class="highwire-citation-author" data-delta="2">Yingming Liang</span>, <span class="highwire-citation-author" data-delta="3">Kai Li</span>, <span class="highwire-citation-author" data-delta="4">Shixian Hu</span>, <span class="highwire-citation-author" data-delta="5">Maochun Wang</span>, <span class="highwire-citation-author" data-delta="6">Yuxun Zhou</span>, <span class="highwire-citation-author" data-delta="7">Hongyan Xu</span> and <span class="highwire-citation-author" data-delta="8">Junhua Xiao</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3571-3580;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.033902 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460453--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Fuyi Xu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Chemistry, Chemical Engineering, and Biotechnology, Donghua University, Shanghai 201620, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fuyi%2BXu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xu%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFuyi%2BXu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5952-5771" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Fuyi Xu</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Tianzhu Chao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Chemistry, Chemical Engineering, and Biotechnology, Donghua University, Shanghai 201620, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tianzhu%2BChao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chao%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATianzhu%2BChao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Yingming Liang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>The Laboratory of Genetic Regulators in the Immune System, Xinxiang Medical University, Henan 453003, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yingming%2BLiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liang%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYingming%2BLiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Kai Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Chemistry, Chemical Engineering, and Biotechnology, Donghua University, Shanghai 201620, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kai%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKai%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Shixian Hu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Chemistry, Chemical Engineering, and Biotechnology, Donghua University, Shanghai 201620, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shixian%2BHu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hu%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShixian%2BHu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Maochun Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Chemistry, Chemical Engineering, and Biotechnology, Donghua University, Shanghai 201620, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maochun%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMaochun%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Yuxun Zhou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Chemistry, Chemical Engineering, and Biotechnology, Donghua University, Shanghai 201620, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yuxun%2BZhou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhou%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYuxun%2BZhou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Hongyan Xu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Biostatistics and Epidemiology, Medical College of Georgia, Augusta University, Georgia 30912</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hongyan%2BXu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xu%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHongyan%2BXu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:hxu@augusta.edu" class="" data-icon-position="" data-hide-link-title="0">hxu@augusta.edu</a>
<a href="mailto:xiaojunhua@dhu.edu.cn" class="" data-icon-position="" data-hide-link-title="0">xiaojunhua@dhu.edu.cn</a></span></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Junhua Xiao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Chemistry, Chemical Engineering, and Biotechnology, Donghua University, Shanghai 201620, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Junhua%2BXiao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xiao%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJunhua%2BXiao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:hxu@augusta.edu" class="" data-icon-position="" data-hide-link-title="0">hxu@augusta.edu</a>
<a href="mailto:xiaojunhua@dhu.edu.cn" class="" data-icon-position="" data-hide-link-title="0">xiaojunhua@dhu.edu.cn</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/11/3571" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/11/3571&amp;title=Genome%20Sequencing%20of%20Chromosome%201%20Substitution%20Lines%20Derived%20from%20Chinese%20Wild%20Mice%20Revealed%20a%20Unique%20Resource%20for%20Genetic%20Studies%20of%20Complex%20Traits" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/11/3571&amp;title=Genome%20Sequencing%20of%20Chromosome%201%20Substitution%20Lines%20Derived%20from%20Chinese%20Wild%20Mice%20Revealed%20a%20Unique%20Resource%20for%20Genetic%20Studies%20of%20Complex%20Traits" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/11/3571&amp;title=Genome%20Sequencing%20of%20Chromosome%201%20Substitution%20Lines%20Derived%20from%20Chinese%20Wild%20Mice%20Revealed%20a%20Unique%20Resource%20for%20Genetic%20Studies%20of%20Complex%20Traits" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/11/3571&amp;text=Genome%20Sequencing%20of%20Chromosome%201%20Substitution%20Lines%20Derived%20from%20Chinese%20Wild%20Mice%20Revealed%20a%20Unique%20Resource%20for%20Genetic%20Studies%20of%20Complex%20Traits" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/11/3571&amp;title=Genome%20Sequencing%20of%20Chromosome%201%20Substitution%20Lines%20Derived%20from%20Chinese%20Wild%20Mice%20Revealed%20a%20Unique%20Resource%20for%20Genetic%20Studies%20of%20Complex%20Traits" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/11/3571&amp;t=Genome%20Sequencing%20of%20Chromosome%201%20Substitution%20Lines%20Derived%20from%20Chinese%20Wild%20Mice%20Revealed%20a%20Unique%20Resource%20for%20Genetic%20Studies%20of%20Complex%20Traits" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/11/3571&amp;title=Genome%20Sequencing%20of%20Chromosome%201%20Substitution%20Lines%20Derived%20from%20Chinese%20Wild%20Mice%20Revealed%20a%20Unique%20Resource%20for%20Genetic%20Studies%20of%20Complex%20Traits" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/11/3571&amp;title=Genome%20Sequencing%20of%20Chromosome%201%20Substitution%20Lines%20Derived%20from%20Chinese%20Wild%20Mice%20Revealed%20a%20Unique%20Resource%20for%20Genetic%20Studies%20of%20Complex%20Traits" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460453/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460453" id="node460453--3" data-pisa="ggg;6/11/3571" data-pisa-master="ggg;g3.116.033902" data-apath="/ggg/6/11/3571.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome Sequencing of Chromosome 1 Substitution Lines Derived from Chinese Wild Mice Revealed a Unique Resource for Genetic Studies of Complex Traits</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-5952-5771" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Fuyi Xu</span>, <span class="highwire-citation-author" data-delta="1">Tianzhu Chao</span>, <span class="highwire-citation-author" data-delta="2">Yingming Liang</span>, <span class="highwire-citation-author" data-delta="3">Kai Li</span>, <span class="highwire-citation-author" data-delta="4">Shixian Hu</span>, <span class="highwire-citation-author" data-delta="5">Maochun Wang</span>, <span class="highwire-citation-author" data-delta="6">Yuxun Zhou</span>, <span class="highwire-citation-author" data-delta="7">Hongyan Xu</span> and <span class="highwire-citation-author" data-delta="8">Junhua Xiao</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3571-3580;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.033902 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460453--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Fuyi Xu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Chemistry, Chemical Engineering, and Biotechnology, Donghua University, Shanghai 201620, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fuyi%2BXu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xu%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFuyi%2BXu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5952-5771" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Fuyi Xu</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Tianzhu Chao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Chemistry, Chemical Engineering, and Biotechnology, Donghua University, Shanghai 201620, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tianzhu%2BChao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chao%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATianzhu%2BChao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Yingming Liang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>The Laboratory of Genetic Regulators in the Immune System, Xinxiang Medical University, Henan 453003, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yingming%2BLiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liang%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYingming%2BLiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Kai Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Chemistry, Chemical Engineering, and Biotechnology, Donghua University, Shanghai 201620, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kai%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKai%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Shixian Hu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Chemistry, Chemical Engineering, and Biotechnology, Donghua University, Shanghai 201620, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shixian%2BHu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hu%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShixian%2BHu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Maochun Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Chemistry, Chemical Engineering, and Biotechnology, Donghua University, Shanghai 201620, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maochun%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMaochun%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Yuxun Zhou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Chemistry, Chemical Engineering, and Biotechnology, Donghua University, Shanghai 201620, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yuxun%2BZhou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhou%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYuxun%2BZhou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Hongyan Xu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Biostatistics and Epidemiology, Medical College of Georgia, Augusta University, Georgia 30912</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hongyan%2BXu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xu%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHongyan%2BXu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:hxu@augusta.edu" class="" data-icon-position="" data-hide-link-title="0">hxu@augusta.edu</a>
<a href="mailto:xiaojunhua@dhu.edu.cn" class="" data-icon-position="" data-hide-link-title="0">xiaojunhua@dhu.edu.cn</a></span></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Junhua Xiao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>College of Chemistry, Chemical Engineering, and Biotechnology, Donghua University, Shanghai 201620, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Junhua%2BXiao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xiao%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJunhua%2BXiao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:hxu@augusta.edu" class="" data-icon-position="" data-hide-link-title="0">hxu@augusta.edu</a>
<a href="mailto:xiaojunhua@dhu.edu.cn" class="" data-icon-position="" data-hide-link-title="0">xiaojunhua@dhu.edu.cn</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3571&amp;rft.epage=3580&amp;rft.atitle=Genome%20Sequencing%20of%20Chromosome%201%20Substitution%20Lines%20Derived%20from%20Chinese%20Wild%20Mice%20Revealed%20a%20Unique%20Resource%20for%20Genetic%20Studies%20of%20Complex%20Traits&amp;rft.volume=6&amp;rft.issue=11&amp;rft.date=2016-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Xu%2C+Fuyi&amp;rft.au=Chao%2C+Tianzhu&amp;rft.au=Liang%2C+Yingming&amp;rft.au=Li%2C+Kai&amp;rft.au=Hu%2C+Shixian&amp;rft.au=Wang%2C+Maochun&amp;rft.au=Zhou%2C+Yuxun&amp;rft.au=Xu%2C+Hongyan&amp;rft.au=Xiao%2C+Junhua"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460453/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460453/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460453/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460453/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460453/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460453/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460453/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460453/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460453/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460453/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460453/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460453/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/11/3571" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-21" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/11/3571.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/11/3571.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/11/3571.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/11/3571.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
