<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/11/3475" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/11/3475.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/11/3475.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/11/3475.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/11/3475.atom" />
<meta name="HW.pisa" content="ggg;6/11/3475" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Multiple Transcript Properties Related to Translation Affect mRNA Degradation Rates in Saccharomyces cerevisiae" />
<meta name="DC.Identifier" content="10.1534/g3.116.032276" />
<meta name="DC.Date" content="2016-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Neymotin et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Degradation of mRNA contributes to variation in transcript abundance. Studies of individual mRNAs have shown that both cis and trans factors affect mRNA degradation rates. However, the factors underlying transcriptome-wide variation in mRNA degradation rates are poorly understood. We investigated the contribution of different transcript properties to transcriptome-wide degradation rate variation in the budding yeast, Saccharomyces cerevisiae , using multiple regression analysis. We find that multiple transcript properties are significantly associated with variation in mRNA degradation rates, and that a model incorporating these properties explains ∼50% of the genome-wide variance. Predictors of mRNA degradation rates include transcript length, ribosome density, biased codon usage, and GC content of the third position in codons. To experimentally validate these factors, we studied individual transcripts expressed from identical promoters. We find that decreasing ribosome density by mutating the first translational start site of a transcript increases its degradation rate. Using coding sequence variants of green fluorescent protein (GFP) that differ only at synonymous sites, we show that increased GC content of the third position of codons results in decreased rates of mRNA degradation. Thus, in steady-state conditions, a large fraction of genome-wide variation in mRNA degradation rates is determined by inherent properties of transcripts, many of which are related to translation, rather than specific regulatory mechanisms." />
<meta name="DC.Contributor" content="Benjamin Neymotin" />
<meta name="DC.Contributor" content="Victoria Ettorre" />
<meta name="DC.Contributor" content="David Gresham" />
<meta name="article:published_time" content="2016-11-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Multiple Transcript Properties Related to Translation Affect mRNA Degradation Rates in Saccharomyces cerevisiae" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Degradation of mRNA contributes to variation in transcript abundance. Studies of individual mRNAs have shown that both &lt;i&gt;cis&lt;/i&gt; and &lt;i&gt;trans&lt;/i&gt; factors affect mRNA degradation rates. However, the factors underlying transcriptome-wide variation in mRNA degradation rates are poorly understood. We investigated the contribution of different transcript properties to transcriptome-wide degradation rate variation in the budding yeast, &lt;i&gt;Saccharomyces cerevisiae&lt;/i&gt;, using multiple regression analysis. We find that multiple transcript properties are significantly associated with variation in mRNA degradation rates, and that a model incorporating these properties explains ∼50% of the genome-wide variance. Predictors of mRNA degradation rates include transcript length, ribosome density, biased codon usage, and GC content of the third position in codons. To experimentally validate these factors, we studied individual transcripts expressed from identical promoters. We find that decreasing ribosome density by mutating the first translational start site of a transcript increases its degradation rate. Using coding sequence variants of green fluorescent protein (GFP) that differ only at synonymous sites, we show that increased GC content of the third position of codons results in decreased rates of mRNA degradation. Thus, in steady-state conditions, a large fraction of genome-wide variation in mRNA degradation rates is determined by inherent properties of transcripts, many of which are related to translation, rather than specific regulatory mechanisms.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/11/01" />
<meta name="citation_mjid" content="ggg;6/11/3475" />
<meta name="citation_id" content="6/11/3475" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/11/3475" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/11/3475.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/11/3475.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/11/3475.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.032276" />
<meta name="citation_pmid" content="27633789" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3475" />
<meta name="citation_lastpage" content="3483" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Benjamin Neymotin" />
<meta name="citation_author_institution" content="Department of Biology, Center for Genomics and Systems Biology, New York University, New York 10003" />
<meta name="citation_author" content="Victoria Ettorre" />
<meta name="citation_author_institution" content="Department of Biology, Center for Genomics and Systems Biology, New York University, New York 10003" />
<meta name="citation_author" content="David Gresham" />
<meta name="citation_author_institution" content="Department of Biology, Center for Genomics and Systems Biology, New York University, New York 10003" />
<meta name="citation_author_email" content="dgresham@nyu.edu" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO J.;citation_author=J. S. Anderson;citation_author=R. P. Parker;citation_title=The 3' to 5' degradation of yeast mRNAs is a general mechanism for mRNA turnover that requires the SKI2 DEVH box protein and 3' to 5' exonucleases of the exosome complex;citation_pages=1497-1506;citation_volume=17;citation_year=1998;citation_issue=5;citation_pmid=9482746;citation_doi=10.1093/emboj/17.5.1497" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=C. A. Beelman;citation_author=R. Parker;citation_title=Differential effects of translational inhibition in cis and in trans on the decay of the unstable yeast MFA2 mRNA.;citation_pages=9687-9692;citation_volume=269;citation_year=1994;citation_issue=13;citation_pmid=8144558" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=C. A. Beelman;citation_author=A. Stevens;citation_author=G. Caponigro;citation_author=T. E. LaGrandeur;citation_author=L. Hatfield;citation_title=An essential component of the decapping enzyme required for normal rates of mRNA turnover.;citation_pages=642-646;citation_volume=382;citation_year=1996;citation_issue=6592;citation_pmid=8757137;citation_doi=10.1038/382642a0" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. Belle;citation_author=A. Tanay;citation_author=L. Bitincka;citation_author=R. Shamir;citation_author=E. K. O’Shea;citation_title=Quantification of protein half-lives in the budding yeast proteome.;citation_volume=103;citation_year=2006;citation_pmid=16916930;citation_doi=10.1073/pnas.0605420103" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=G. Caponigro;citation_author=D. Muhlrad;citation_author=R. Parker;citation_title=A small segment of the MAT alpha 1 transcript promotes mRNA decay in Saccharomyces cerevisiae: a stimulatory role for rare codons.;citation_pages=5141-5148;citation_volume=13;citation_year=1993;citation_issue=9;citation_pmid=8355674;citation_doi=10.1128/MCB.13.9.5141" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=D. B. Carlini;citation_title=Context-dependent codon bias and messenger RNA longevity in the yeast transcriptome.;citation_volume=22;citation_year=2005;citation_pmid=15772378;citation_doi=10.1093/molbev/msi135" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=C. Y. Chen;citation_author=R. Gherzi;citation_author=S. E. Ong;citation_author=E. L. Chan;citation_author=R. Raijmakers;citation_title=AU binding proteins recruit the exosome to degrade ARE-containing mRNAs.;citation_pages=451-464;citation_volume=107;citation_year=2001;citation_issue=4;citation_pmid=11719186;citation_doi=10.1016/S0092-8674(01)00578-5" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=C. J. Decker;citation_author=R. Parker;citation_title=A turnover pathway for both stable and unstable mRNAs in yeast: evidence for a requirement for deadenylation.;citation_pages=1632-1643;citation_volume=7;citation_year=1993;citation_issue=8;citation_pmid=8393418;citation_doi=10.1101/gad.7.8.1632" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=C. Dressaire;citation_author=F. Picard;citation_author=E. Redon;citation_author=P. Loubière;citation_author=I. Queinnec;citation_title=Role of mRNA stability during bacterial adaptation.;citation_pages=e59059;citation_volume=8;citation_year=2013;citation_pmid=23516597;citation_doi=10.1371/journal.pone.0059059" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=J. Duan;citation_author=J. Shi;citation_author=X. Ge;citation_author=L. Dölken;citation_author=W. Moy;citation_title=Genome-wide survey of interindividual differences of RNA stability in human lymphoblastoid cell lines.;citation_pages=1318;citation_volume=3;citation_year=2013;citation_pmid=23422947" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=S. Eck;citation_author=W. Stephan;citation_title=Determining the relationship of gene expression and global mRNA stability in Drosophila melanogaster and Escherichia coli using linear models.;citation_pages=102-107;citation_volume=424;citation_year=2008;citation_issue=1-2;citation_pmid=18755255;citation_doi=10.1016/j.gene.2008.07.033" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=S. Edri;citation_author=T. Tuller;citation_title=Quantifying the effect of ribosomal density on mRNA stability.;citation_pages=e102308;citation_volume=9;citation_year=2014;citation_pmid=25020060;citation_doi=10.1371/journal.pone.0102308" />
<meta name="citation_reference" content="citation_journal_title=Biochemical genetics;citation_journal_abbrev=Biochem Genet;citation_author=L. Feng;citation_author=D.-K. Niu;citation_title=Relationship between mRNA stability and length: an old question with a new twist.;citation_pages=131-137;citation_volume=45;citation_year=2007;citation_issue=1-2;citation_pmid=17221301;citation_doi=10.1007/s10528-006-9059-5" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=J. García-Martínez;citation_author=A. Aranda;citation_author=J. E. Pérez-Ortín;citation_title=Genomic run-on evaluates transcription rates for all yeast genes and identifies gene regulatory mechanisms.;citation_pages=303-313;citation_volume=15;citation_year=2004;citation_issue=2;citation_pmid=15260981;citation_doi=10.1016/j.molcel.2004.06.004" />
<meta name="citation_reference" content="citation_journal_title=eLife;citation_title=Measurement of average decoding rates of the 61 sense codons in vivo.;citation_author=J. Gardin;citation_author=R. Yeasmin;citation_author=A. Yurovsky;citation_author=Y. Cai;citation_author=S. Skiena;citation_volume=3;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=E. Garí;citation_author=L. Piedrafita;citation_author=M. Aldea;citation_author=E. Herrero;citation_title=A set of vectors with a tetracycline-regulatable promoter system for modulated gene expression in Saccharomyces cerevisiae.;citation_pages=837-848;citation_volume=13;citation_year=1997;citation_issue=9;citation_pmid=9234672;citation_doi=10.1002/(SICI)1097-0061(199707)13:9&lt;837::AID-YEA145&gt;3.0.CO;2-T" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=J. V. Geisberg;citation_author=Z. Moqtaderi;citation_author=X. Fan;citation_author=F. Ozsolak;citation_author=K. Struhl;citation_title=Global analysis of mRNA isoform half-lives reveals stabilizing and destabilizing elements in yeast.;citation_pages=812-824;citation_volume=156;citation_year=2014;citation_pmid=24529382;citation_doi=10.1016/j.cell.2013.12.026" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=S. Ghaemmaghami;citation_author=W.-K. Huh;citation_author=K. Bower;citation_author=R. W. Howson;citation_author=A. Belle;citation_title=Global analysis of protein expression in yeast.;citation_pages=737-741;citation_volume=425;citation_year=2003;citation_issue=6959;citation_pmid=14562106;citation_doi=10.1038/nature02046" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=J. Grigull;citation_author=S. Mnaimneh;citation_author=J. Pootoolal;citation_author=M. D. Robinson;citation_author=T. R. Hughes;citation_title=Genome-Wide Analysis of mRNA Stability Using Transcription Inhibitors and Microarrays Reveals Posttranscriptional Control of Ribosome Biogenesis Factors;citation_pages=5534-5547;citation_volume=24;citation_year=2004;citation_issue=12;citation_pmid=15169913;citation_doi=10.1128/MCB.24.12.5534-5547.2004" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=G. L. Hager;citation_author=J. G. McNally;citation_author=T. Misteli;citation_title=Transcription dynamics.;citation_pages=741-753;citation_volume=35;citation_year=2009;citation_issue=6;citation_pmid=19782025;citation_doi=10.1016/j.molcel.2009.09.005" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=D. J. Hogan;citation_author=D. P. Riordan;citation_author=A. P. Gerber;citation_author=D. Herschlag;citation_author=P. O. Brown;citation_title=Diverse RNA-binding proteins interact with functionally related sets of RNAs, suggesting an extensive regulatory system.;citation_pages=e255-e255;citation_volume=6;citation_year=2008;citation_issue=10;citation_pmid=18959479;citation_doi=10.1371/journal.pbio.0060255" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. Hong;citation_author=D. Gresham;citation_title=Molecular specificity, convergence and constraint shape adaptive evolution in nutrient-poor environments.;citation_pages=e1004041;citation_volume=10;citation_year=2014;citation_pmid=24415948;citation_doi=10.1371/journal.pgen.1004041" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=N. T. Ingolia;citation_author=S. Ghaemmaghami;citation_author=J. R. S. Newman;citation_author=J. S. Weissman;citation_title=Genome-wide analysis in vivo of translation with nucleotide resolution using ribosome profiling.;citation_volume=324;citation_year=2009;citation_pmid=19213877;citation_doi=10.1126/science.1168978" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=Q. Jing;citation_author=S. Huang;citation_author=S. Guth;citation_author=T. Zarubin;citation_author=A. Motoyama;citation_title=Involvement of microRNA in AU-rich element-mediated mRNA instability.;citation_pages=623-634;citation_volume=120;citation_year=2005;citation_issue=5;citation_pmid=15766526;citation_doi=10.1016/j.cell.2004.12.038" />
<meta name="citation_reference" content="citation_journal_title=RNA;citation_author=S. E. Kolitz;citation_author=J. E. Takacs;citation_author=J. R. Lorsch;citation_title=Kinetic and thermodynamic analysis of the role of start codon/anticodon base pairing during eukaryotic translation initiation.;citation_volume=15;citation_year=2009;citation_pmid=19029312;citation_doi=10.1261/rna.1318509" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=G. Kudla;citation_author=L. Lipinski;citation_author=F. Caffin;citation_author=A. Helwak;citation_author=M. Zylicz;citation_title=High guanine and cytosine content increases mRNA levels in mammalian cells.;citation_pages=e180-e180;citation_volume=4;citation_year=2006;citation_issue=6;citation_pmid=16700628;citation_doi=10.1371/journal.pbio.0040180" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=G. Kudla;citation_author=A. W. Murray;citation_author=D. Tollervey;citation_author=J. B. Plotkin;citation_title=Coding-Sequence Determinants of Gene Expression in Escherichia coli;citation_pages=255-258;citation_volume=324;citation_year=2009;citation_issue=5924;citation_pmid=19359587;citation_doi=10.1126/science.1170160" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=D. Lipson;citation_author=T. Raz;citation_author=A. Kieu;citation_author=D. R. Jones;citation_author=E. Giladi;citation_title=Quantification of the yeast transcriptome by single-molecule sequencing.;citation_pages=652-658;citation_volume=27;citation_year=2009;citation_issue=7;citation_pmid=19581875;citation_doi=10.1038/nbt.1551" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=A. Merhi;citation_author=N. Gérard;citation_author=E. Lauwers;citation_author=M. Prévost;citation_author=B. André;citation_title=Systematic mutational analysis of the intracellular regions of yeast Gap1 permease.;citation_pages=e18457-e18457;citation_volume=6;citation_year=2011;citation_issue=4;citation_pmid=21526172;citation_doi=10.1371/journal.pone.0018457" />
<meta name="citation_reference" content="citation_journal_title=Molecular Systems Biology;citation_journal_abbrev=Mol Syst Biol;citation_author=C. Miller;citation_author=B. Schwalb;citation_author=K. Maier;citation_author=D. Schulz;citation_author=S. Dümcke;citation_title=Dynamic transcriptome analysis measures rates of mRNA synthesis and decay in yeast;citation_pages=458-458;citation_volume=7;citation_year=2011;citation_issue=1;citation_pmid=21206491;citation_doi=10.1038/msb.2010.112" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=D. Muhlrad;citation_author=R. Parker;citation_title=Mutations affecting stability and deadenylation of the yeast MFA2 transcript.;citation_pages=2100-2111;citation_volume=6;citation_year=1992;citation_issue=11;citation_pmid=1427074;citation_doi=10.1101/gad.6.11.2100" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=D. Muhlrad;citation_author=C. J. Decker;citation_author=R. Parker;citation_title=Turnover mechanisms of the stable yeast PGK1 mRNA;citation_pages=2145-2156;citation_volume=15;citation_year=1995;citation_issue=4;citation_pmid=7891709;citation_doi=10.1128/MCB.15.4.2145" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=S. E. Munchel;citation_author=R. K. Shultzaberger;citation_author=N. Takizawa;citation_author=K. Weis;citation_title=Dynamic profiling of mRNA turnover reveals gene-specific and system-wide regulation of mRNA decay.;citation_volume=22;citation_year=2011;citation_pmid=21680716;citation_doi=10.1091/mbc.E11-01-0028" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=U. Nagalakshmi;citation_author=Z. Wang;citation_author=K. Waern;citation_author=C. Shou;citation_author=D. Raha;citation_title=The transcriptional landscape of the yeast genome defined by RNA sequencing.;citation_volume=320;citation_year=2008;citation_pmid=18451266;citation_doi=10.1126/science.1158441" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=R. Narsai;citation_author=K. A. Howell;citation_author=A. H. Millar;citation_author=N. O’Toole;citation_author=I. Small;citation_title=Genome-wide analysis of mRNA decay rates and their determinants in Arabidopsis thaliana.;citation_volume=19;citation_year=2007;citation_pmid=18024567;citation_doi=10.1105/tpc.107.055046" />
<meta name="citation_reference" content="citation_journal_title=RNA;citation_author=B. Neymotin;citation_author=R. Athanasiadou;citation_author=D. Gresham;citation_title=Determination of in vivo RNA kinetics using RATE-seq.;citation_volume=20;citation_year=2014;citation_pmid=25161313;citation_doi=10.1261/rna.045104.114" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=M. Nonet;citation_author=C. Scafe;citation_author=J. Sexton;citation_author=R. Young;citation_title=Eucaryotic RNA polymerase conditional mutant that rapidly ceases mRNA synthesis.;citation_pages=1602-1611;citation_volume=7;citation_year=1987;citation_issue=5;citation_pmid=3299050;citation_doi=10.1128/MCB.7.5.1602" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO J.;citation_author=W. Olivas;citation_author=R. Parker;citation_title=The Puf3 protein is a transcript-specific regulator of mRNA degradation in yeast;citation_pages=6602-6611;citation_volume=19;citation_year=2000;citation_issue=23;citation_pmid=11101532;citation_doi=10.1093/emboj/19.23.6602" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. Parker;citation_title=RNA Degradation in Saccharomyces cerevisae;citation_pages=671-702;citation_volume=191;citation_year=2012;citation_issue=3;citation_pmid=22785621;citation_doi=10.1534/genetics.111.137265" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=R. Parker;citation_author=A. Jacobson;citation_title=Translation and a 42-nucleotide segment within the coding region of the mRNA encoded by the MAT alpha 1 gene are involved in promoting rapid mRNA decay in yeast.;citation_pages=2780-2784;citation_volume=87;citation_year=1990;citation_issue=7;citation_pmid=2181450;citation_doi=10.1073/pnas.87.7.2780" />
<meta name="citation_reference" content="citation_journal_title=Nat. Struct. Mol. Biol.;citation_author=S. Pechmann;citation_author=J. Frydman;citation_title=Evolutionary conservation of codon optimality reveals hidden signatures of cotranslational folding.;citation_pages=237-243;citation_volume=20;citation_year=2013;citation_pmid=23262490;citation_doi=10.1038/nsmb.2466" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=V. Pelechano;citation_author=S. Chávez;citation_author=J. E. Pérez-Ortín;citation_title=A complete set of nascent transcription rates for yeast genes.;citation_pages=e15442-e15442;citation_volume=5;citation_year=2010;citation_issue=11;citation_pmid=21103382;citation_doi=10.1371/journal.pone.0015442" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. B. Plotkin;citation_author=G. Kudla;citation_title=Synonymous but not the same: the causes and consequences of codon bias.;citation_pages=32-42;citation_volume=12;citation_year=2011;citation_issue=1;citation_pmid=21102527;citation_doi=10.1038/nrg2899" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=V. Presnyak;citation_author=N. Alhusaini;citation_author=Y.-H. Chen;citation_author=S. Martin;citation_author=N. Morris;citation_title=Codon optimality is a major determinant of mRNA stability.;citation_pages=1111-1124;citation_volume=160;citation_year=2015;citation_pmid=25768907;citation_doi=10.1016/j.cell.2015.02.029" />
<meta name="citation_reference" content="citation_title=R: A language and environment for statistical computing.;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=M. Rabani;citation_author=J. Z. Levin;citation_author=L. Fan;citation_author=X. Adiconis;citation_author=R. Raychowdhury;citation_title=Metabolic labeling of RNA uncovers principles of RNA production and degradation dynamics in mammalian cells.;citation_pages=436-442;citation_volume=29;citation_year=2011;citation_issue=5;citation_pmid=21516085;citation_doi=10.1038/nbt.1861" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. W. Selinger;citation_author=R. M. Saxena;citation_author=K. J. Cheung;citation_author=G. M. Church;citation_author=C. Rosenow;citation_title=Global RNA half-life analysis in Escherichia coli reveals positional patterns of transcript degradation.;citation_volume=13;citation_year=2003;citation_pmid=12566399;citation_doi=10.1101/gr.912603" />
<meta name="citation_reference" content="citation_journal_title=Molecular Systems Biology;citation_journal_abbrev=Mol Syst Biol;citation_author=O. Shalem;citation_author=O. Dahan;citation_author=M. Levo;citation_author=M. R. Martinez;citation_author=I. Furman;citation_title=Transient transcriptional responses to stress are generated by opposing effects of mRNA production and degradation.;citation_pages=223-223;citation_volume=4;citation_year=2008;citation_pmid=18854817;citation_doi=10.1038/msb.2008.59" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=P. M. Sharp;citation_author=W. H. Li;citation_title=The codon adaptation index-a measure of directional synonymous codon usage bias, and its potential applications;citation_pages=1281-1295;citation_volume=15;citation_year=1987;citation_issue=3;citation_pmid=3547335;citation_doi=10.1093/nar/15.3.1281" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=G. Shaw;citation_author=R. Kamen;citation_title=A conserved AU sequence from the 3' untranslated region of GM-CSF mRNA mediates selective mRNA degradation.;citation_pages=659-667;citation_volume=46;citation_year=1986;citation_issue=5;citation_pmid=3488815;citation_doi=10.1016/0092-8674(86)90341-7" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=S. Thomsen;citation_author=S. Anders;citation_author=S. C. Janga;citation_author=W. Huber;citation_author=C. R. Alonso;citation_title=Genome-wide analysis of mRNA decay patterns during early Drosophila development.;citation_pages=R93-R93;citation_volume=11;citation_year=2010;citation_issue=9;citation_pmid=20858238;citation_doi=10.1186/gb-2010-11-9-r93" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Y. Wang;citation_author=C. L. Liu;citation_author=J. D. Storey;citation_author=R. J. Tibshirani;citation_author=D. Herschlag;citation_title=Precision and functional specificity in mRNA decay.;citation_volume=99;citation_year=2002;citation_pmid=11972065;citation_doi=10.1073/pnas.092538799" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=R. Wisdom;citation_author=W. Lee;citation_title=The protein-coding region of c-myc mRNA contains a sequence that specifies rapid mRNA turnover and induction by protein synthesis inhibitors.;citation_pages=232-243;citation_volume=5;citation_year=1991;citation_issue=2;citation_pmid=1995415;citation_doi=10.1101/gad.5.2.232" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=J. A. Wishart;citation_author=A. Hayes;citation_author=L. Wardleworth;citation_author=N. Zhang;citation_author=S. G. Oliver;citation_title=Doxycycline, the drug used to control the tet-regulatable promoter system, has no effect on global gene expression in Saccharomyces cerevisiae.;citation_pages=565-569;citation_volume=22;citation_year=2005;citation_issue=7;citation_pmid=15942933;citation_doi=10.1002/yea.1225" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=E. Yang;citation_author=E. van Nimwegen;citation_author=M. Zavolan;citation_author=N. Rajewsky;citation_author=M. Schroeder;citation_title=Decay Rates of Human mRNAs: Correlation With Functional Characteristics and Sequence Attributes;citation_pages=1863-1872;citation_volume=13;citation_year=2003;citation_issue=8;citation_pmid=12902380;citation_doi=10.1101/gr.1272403" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=D.-F. Yun;citation_author=F. Sherman;citation_title=Degradation of CYC1 mRNA in the yeast Saccharomyces cerevisiae does not require translation.;citation_pages=8895-8900;citation_volume=93;citation_year=1996;citation_issue=17;citation_pmid=8799124;citation_doi=10.1073/pnas.93.17.8895" />
<meta name="citation_reference" content="citation_journal_title=Nature structural &amp; molecular biology;citation_journal_abbrev=Nat Struct Mol Biol;citation_author=D. Zenklusen;citation_author=D. R. Larson;citation_author=R. H. Singer;citation_title=Single-RNA counting reveals alternative modes of gene expression in yeast.;citation_pages=1263-1271;citation_volume=15;citation_year=2008;citation_issue=12;citation_pmid=19011635;citation_doi=10.1038/nsmb.1514" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Multiple Transcript Properties Related to Translation Affect mRNA Degradation Rates in Saccharomyces cerevisiae" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11.cover-source.jpg" />
<meta name="twitter:description" content="Degradation of mRNA contributes to variation in transcript abundance. Studies of individual mRNAs have shown that both cis and trans factors affect mRNA degradation rates. However, the factors underlying transcriptome-wide variation in mRNA degradation rates are poorly understood. We investigated the contribution of different transcript properties to transcriptome-wide degradation rate variation in the budding yeast, Saccharomyces cerevisiae , using multiple regression analysis. We find that multiple transcript properties are significantly associated with variation in mRNA degradation rates, and that a model incorporating these properties explains ∼50% of the genome-wide variance. Predictors of mRNA degradation rates include transcript length, ribosome density, biased codon usage, and GC content of the third position in codons. To experimentally validate these factors, we studied individual transcripts expressed from identical promoters. We find that decreasing ribosome density by mutating the first translational start site of a transcript increases its degradation rate. Using coding sequence variants of green fluorescent protein (GFP) that differ only at synonymous sites, we show that increased GC content of the third position of codons results in decreased rates of mRNA degradation. Thus, in steady-state conditions, a large fraction of genome-wide variation in mRNA degradation rates is determined by inherent properties of transcripts, many of which are related to translation, rather than specific regulatory mechanisms." />
<meta name="og-title" property="og:title" content="Multiple Transcript Properties Related to Translation Affect mRNA Degradation Rates in Saccharomyces cerevisiae" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/11/3475" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Degradation of mRNA contributes to variation in transcript abundance. Studies of individual mRNAs have shown that both cis and trans factors affect mRNA degradation rates. However, the factors underlying transcriptome-wide variation in mRNA degradation rates are poorly understood. We investigated the contribution of different transcript properties to transcriptome-wide degradation rate variation in the budding yeast, Saccharomyces cerevisiae , using multiple regression analysis. We find that multiple transcript properties are significantly associated with variation in mRNA degradation rates, and that a model incorporating these properties explains ∼50% of the genome-wide variance. Predictors of mRNA degradation rates include transcript length, ribosome density, biased codon usage, and GC content of the third position in codons. To experimentally validate these factors, we studied individual transcripts expressed from identical promoters. We find that decreasing ribosome density by mutating the first translational start site of a transcript increases its degradation rate. Using coding sequence variants of green fluorescent protein (GFP) that differ only at synonymous sites, we show that increased GC content of the third position of codons results in decreased rates of mRNA degradation. Thus, in steady-state conditions, a large fraction of genome-wide variation in mRNA degradation rates is determined by inherent properties of transcripts, many of which are related to translation, rather than specific regulatory mechanisms." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11.cover-source.jpg" />
    <title>Multiple Transcript Properties Related to Translation Affect mRNA Degradation Rates in Saccharomyces cerevisiae | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"SZHmfBq7ArFpcLYKNHpRLhJr_2MRih5bQUwBsVsFwYI","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460457","apath":"\/ggg\/6\/11\/3475.atom","pisa":"ggg;6\/11\/3475","ac":{"\/ggg\/6\/11\/3475.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/11\/3475.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/11\/3475"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/11\/3475"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/11\/3475":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022UjHYnKyJs81SzMwkG68IrpJrxK4MXs7B5T+ZNwMfIvh\\\\\\\/KcGc1IDFPjr\\\\\\\/i\\\\\\\/G\\\\\\\/Ljp9Us3djAJL7IfjYsdajcAG6YiOs\\\\\\\/QN76QvTzk+frmISNtQp5Vw2qKYgZKdirxebgzpBQXB8kvLLHEKex5gYt2ysetC9hY7T2rpGNrEogK48\\\\\\\/g9uR1MLrPiwen12VIZ6mZTu7TPtCUCcoIQJb+ID\\\\\\\/sCaxSldFdwHrZfk+aeDgGIA\\\\\\\/2L5L15b3Vvn4AvQ5Q222UCmW+oJKu+Wsl9oN8xVNr69XoccTZuhw25MoubGSjbhyi5WbmSOxVv03mvW5IXH5umKvc4748+IZtexn8174TM83UBxsM+tQvEs8jjSqhaaIchcHNmAJO1o15yFVWqsNFIERNzAP4oNcscbeFQ3sUq4cf7\\\\\\\/Cj+5b10sjmpUfB0ssOsqHHw59Cc9sL\\\\\\\/Ebbn7sStTwZfDGzK7bXyLhlnJj6nB7RmR5UOnjiHzohxlZhrRupJqwfEzORT+4IuImr5n+B0HaHGM5MUPMEJqsTUWWBgRp2uVuLeORfVw66jz1\\\\\\\/ALGc091tCeyx7aETEfLrZV\\\\\\\/XbLtY5NJ9VullZrcxBp4\\\\\\\/dJAx8TdYhSmwr4ms2G\\\\\\\/4VFFTqLAfLTM6DEQA2mI7pKw3NjQXcgF2eJ0g\\\\\\\/t32BeX48peODWBHkV+dNbiAvAp3gclP24otSRFeDB94dFXfbuEBbvEpFqtPXOoaOxgLsTvAkDdytgPolDUBO9GPg3VwooRr3axG72EwhnQp1TRFlmgE9\\\\\\\/YhKI+xuutc7Wc7pWYbcFHAm3ijEuBjHHOjMhHg7QerDwz3jtBnq1bemF7Z3Wa2SI1vRcnO52OtBp\\\\\\\/j867\\\\\\\/hTDhEyUkiUiV4PdUXedWUwjc\\\\\\\/8fhW+Vy\\\\\\\/svKLECKT9ocJWEb3D\\\\\\\/GwrsGN1Zwn+rOCirhU1NMgYPqv95NdOnx6anAbYEz1+TiTOf13E190ApVAUg4wN9gd9NmonZTcVJp62ZCIA6urO4q40ynIwnIRDp3S486i2c7eBjIMnrHdyOe6f0rs4flDWmq6d2pJQ2heTJO8cnmuX5R1diSRYNRolL0G6aFygFAO4g5Zl4cjKwHTo+BkE3u6vBVS7l7rr+\\\\\\\/i2BBbbaRcjdCNYeNmkmS1h6bw\\\\\\\/oSrF0n9KXJdHGJmgyt8F0tyPHMlmNN\\\\\\\/cjivGYA322z1ToAYMpywL2QBi31lCP\\\\\\\/QBOV50CjZG3o5YcmfbjWJwm8CD8ejuscgVW8UIANIXdnvS\\\\\\\/bZcqsC5on65J45k1F36MOA7f6yfxwVKdRTt1mS\\\u0022,\\\u0022iv\\\u0022:\\\u0022t3qZ4j6lO3wJaOiaNlZomw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e7207dbfdc1978bf8fc440107165d871a522564519ecac0ec153438116f890ce\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022AYFdmIZR06wwP16gXBqFr0QP3e7vnQjJGcXLmDbrlv7fWe2vnbiJDFbksssyVo736\\\\\\\/WUEjSThK11qET\\\\\\\/40vedlUJDYsDHT+Ck08KVw\\\\\\\/nSlwMW0FlVL6jx6TTcIPtxBv0sJzpZOvBPHujToAYdRbH+cCds1Hi8P5WV8Jae7EsZLzZIc0ujD7oRAYmjVjU\\\\\\\/wVryR1Lkx86eU3DSahOBPH\\\\\\\/SVFHatSZSGXys2jZcv\\\\\\\/4RbittVpes\\\\\\\/V8z45lPz+J44CR6EEr4LNUirio8OMvSfuM5OvGfMnccSsn9mMivwFDZv5CDePtT\\\\\\\/WXTMacIxdWaQA9SbALblfnl17IrJb9qJzvH0espeYYstnc\\\\\\\/9bb2XVXPiwf727hSLUsn9BpL9YxYTFnXqEyI\\\\\\\/xPbHcGo0yNNlbBuCbmKAd1y60XhCLUgIiMjznFQuovKt10I5raFO3poyZqVjP9wVerc3yJzbXDjxky\\\\\\\/mtf+KMKvCWhFb5VqXX0r\\\\\\\/amTau8vXGd6RaQ+479MPfb3cJiuzucpJWBmVeeINFqwGHUCM8j5y91NofRssqVLt+eTsKCR57TGxNjLDyf9vCqIDqvFx3Iog3YeTDS\\\\\\\/T5jlS1kkhpZkUuWv3uIeZJcXqDbAgEFPO\\\\\\\/6lYHe4rHjE6KD+ZRQxscJ3bxiLXyWJGro3AuNdiwW9pL2F0zPBdjyx97B3Vt14nch+hQlUKtw\\\\\\\/oNQb9KkF0t6z8BvKTYFY9UTjFIgPCpjVQ1OHFRuG5BcunQh4UZoEp7R1lWamgX\\\\\\\/ZSYg0k3JRMfsmksjwORjrbo44QFfo2CuZi1JJyAZKCQmlq1iXEmz9jyGiQkLVz8+w63ki+++0LshrPVw\\\\\\\/NWV6JTR+aFxQgEhkgJrSvf44Cvx9dxNhW8llKw45HTHNUEzW63KQvRFZRcoZ9WEzmAyM4fqa6r+8ylLIiMUsWgY0IGEywRrKmn+Z6X6ODHglmO+ukl4cAQC4LqD5xqmJ8LLZ1ibsvvm2VYebIfvWyjDFAG1Rvs1NJWWPcjF75NqZeJbRr\\\\\\\/7dfsAY1bMk3pNcoH26UnvJeMgb3dSDYtlXuaaaxwyY0xX2VbDkTcM+a7qe+CLzs3Tr4CoJqoeq\\\\\\\/aOsRLQiFrX02pny1+w+z5ftFlKDnRqMq4bc9HuDdz05lTLp0WYgvP9ugRsJgIOeUUpiioudujSVqvpL8B2MvGGXGAmQZV0N7yclL9u7XtWLor0uDi4E+\\\\\\\/IBjAeD0aFXgKpglUJGT9e+i1polcpwR\\\\\\\/RSbPPwTIgdRLLGV\\\\\\\/dOx2MX6TMd\\\\\\\/LJuV7zP8Iml4IKt5DRGu2hTiABzVA4AXN4x0xAZ7ZuqLGopH2yqV73kKdGtpzHmB4A+RkzL8me2umYGGlj\\\\\\\/UvaPeTr4w3gPou461A\\\\\\\/FATeB3OGoT9qUN8Jg96lAX6FWGpNQQHXRZ3BV9uj3njtBRuanrVeKAGOWnRoQddpjKfzRIHA7T0\\\\\\\/RJPRz+\\\\\\\/iDX4eGeiHd\\\\\\\/4FA8L4Ue37MPNKdkxpEyDGk0i5E2\\\\\\\/jF0J9V2yHw+Py7+w8TQKT1KeaxkAZTO9rRcQs\\\\\\\/jwE2PJoAAx\\\\\\\/kmc6VtTvNMPFLrTCF+I3XS0TnmPfgBEdTvoz2NWYRS3HlA5IpASqMUwAlEdqpmEFQJmiYQrLErLmn1s59bM0mzoOcVZXq1GTMKpgbflgFse69rfUSypiqTcghdqvir83Gten5RJZjn2sQQlBuuZm8UdBhyArrXiPn0XOxag9G6+Jy7gfKWskz5g0W5irTply2LhhhNCa29n2nLqthdcmI\\\\\\\/3ohlwAvpEl0PsvSCSvOYwMkfYO9DnJ3aWK6OLKrXwDY7jVifiwCL8P8y29AtenDhFyMESx3rOE5rhl4bJixK3a+6CTmFjsiWtQ9Q29ZNb8LlHegyesd6ERpOKqdTDwCu0swxxC0V6l1Dgp0Lnh8CJOhNh0\\\\\\\/owFF7A5lGV+STW1yNSqLNOb8OfDsKMSNjqfc9Idb0eLb2pXqNVqibNmGnGGnlSSX26ErpQREUtt1xmqH1Of+JQvjLp1\\\\\\\/ViQZBjqNw2CbieuPIijZ5iE3x8dnO4B+SsaYld3K6H6q+ESaq5Fp9LXGLhs1hqqcC8\\\\\\\/JaLsQk3ycw+O7iKaFE9GJtTZkqMbNbBHZcRAJ8otL+CLo4xg+UBqr2X7VdbhCeZxjCyrwJoTI+fUehDSk0USo\\\\\\\/245GdgCIWjzcVw6VkS076XCsbSW0rCiW\\\\\\\/xxsYT2beWqQdAbq3bmqc5JTuI96K6vcWccFul94X5dVZj9TvAibXJGqisnCpGlnGb\\\\\\\/NbuppEokjANsgDnMBi6Q+AI19YJ6+w2QJg20KtLBtsBle8DDByWsg8OR2\\\\\\\/yydboZzo4gutWoJv\\\\\\\/QmKaPw+LkI2Duy0oP3Aj8EERzy6utwr5q9O+0CNuXKjB\\\\\\\/AbnGqLyeCGjWPTwtE271SnJUrnCcydKf4jwX4SlURMY7ofPcFHvI8jl8ZyqeugipAZp2iIEmg+owpIX8\\\\\\\/TiT913gWfcixFJ73Xxl\\\\\\\/QlaRD+sNXbHwkqOLV8vl9+akdUH3qghWGjZ9mS1nhO5e4R4LD5yjm3iTKaetAzcDPKcThDRYPSlAULvWoqeR+Lj7B0gm\\\\\\\/xrlSgkv7QzsfvuZqx7DhCG08JV6\\\\\\\/aM3Y7+pqF2DlbtFSIKshTA65SpnYZp+sdoKDWeI872zCOQydl5Kn9TjBqy\\\\\\\/5uH\\\\\\\/YYI\\\\\\\/w8P\\\\\\\/2VSXlt6VVk\\\\\\\/lzLZSq8QyvgwKM\\\\\\\/STxtW7oQDZXyt1hfrMstZoeUAEXuFrqxxUn8Vp5nbdr0KW3ry0fuO8A3EJCVXq6ZkkizpLHTN46GrQuXMTTuNqP\\\\\\\/rmljAbW+KkwPxv7UCN1t2jAwF6glH8LhRMOJQpfePBo1MYG\\\\\\\/f86ueOcrbJLfYpcjrIUohjXC3CTutBvjyPO6SbPc1\\\\\\\/qr8u9j3L4ICiLMGk02Qc+aWdHCcmFv+OgM13DXoEMYmVf0H88rkx0uj0BD3UlB6J4BrTNqnuFjxb+EjqD0l0pGAImILG4pPfxLQxhnOH9\\\\\\\/Hiky4n7OFX\\\\\\\/Z4PO5ZsT6w3zLglDCG5jZs1qv9C7Pzr\\\\\\\/sNVfLs08ghUOHEJOxqHR0ujUDMox4suBP\\\\\\\/fz9x4XuC8URk3r9Z\\\\\\\/y+RR1V7LQkBD635zpPzqtKpiVtbsuG2LGhz9573sGGjcDkreP+K\\\\\\\/0efrEZg\\\\\\\/Scv4uc74qm8n8TcftCc9Q6sjLSa2FiMUNXl6C3rBfwefHiaXIiQ8WaPGnDqvmYM98c7Kwenuop42PamMSHy75sBkB2UE2r\\\\\\\/MYWelaM83zYKsVEs\\\\\\\/CZAldrqKiBe0AIuyw3EulM9qukY26ZvkYGjI27Z930tRvn0KyUtxuUNQW\\\\\\\/HK4RXUD9+ZbLA+hQNI+UMy\\\\\\\/t9hZ0x0bgISWEq96RBWW2zzquTC3RCn\\\\\\\/qeqdjKJdNElJ+EAgvU+c7rx8akwnkHQAvu6FolW6nZizPl1bYIotlVj+09vuKWUqjjHkNTGzfl4M7Y1TDZgP8fTgdIKmaA5deE8PQZ8mQwn6uX5\\\\\\\/dPT2HIoYbXS5nxoGnVYY4pV4lhf2QD4\\\\\\\/rLMvFe1Mwlvj6Q1cr2+WNH6kf9dB\\\\\\\/cCJeTOdsmi7HHuGwsGloI8upeifozeE+lRwH7G8XhFbHWvGkyQkWf0RUTILCOzR84FHV8SPs6mchtym9nipsekEaE1zPVUztjxKqA+Kn7mR8lRErrODONibNZ8kKibkWsF3i\\\\\\\/9f\\\\\\\/RyPHLrQtpSwWZ9gxWn4sYo7zsrPK1VeTJW4\\\\\\\/y\\\\\\\/5SWMwt7OqCtkg52zOOIaWewB+dcX6BN4pEw77SszsFJcOuJNRT6qK0NgRi9q8BqKfyd4o4VBj5zVG5m8FteWfxNN0zJavndzkwk\\\\\\\/8oDBoNfgYUo3R6huWyVLVA4xqsQyE0VItTyobrcOWGQFhWWhpb+cwLTWNx9KRWdMssLjBLnR40vx5a0BBimLKb2hpYOqZc7GoLBtw2Ao5hrS4OQHDhxydjiH\\\\\\\/Ouv8mUhsBsXWfqHZMSF1yU10DyN53NC2CsaRnNXQhGKLhzVkkf0yzWQpmqoArRREAatdN4OQg70b7EHJ26PCkHd92lOYRCM\\\\\\\/jrmkGGIhA8nErVZgsOFVhtVCSPUlbX6NxkiwGy7hgbz0\\\\\\\/TmTXSBKOpM56tj4Ux\\\\\\\/M2oH6Zp7uRY4Mym6b6Kl+0Hs6xzIAkxnnVBXXZb2JFZ7hyZc0w+jHg3RpmEpgWcafuBrovVKAEm+Jv53JZoTYe0cr\\\\\\\/W2GJyja6Vl5QbSP1hCK3vWHIcZYCQAlB9RA5hO1WiFUvr9LgP8nH5PPq0hFteG0tlOrU3eW1IlBuU1GxrXCoO0xR+BMy+Kalnn5sNzjZfbPAvWYuQEJeWT9fgkzof\\\\\\\/scR\\\\\\\/a9bkZ7BdREWFRWfRsanIBn2ynZ6lDXkCyCPe\\\\\\\/EeL1wIybwunlxVflyZBpiHn2VP2mT5Zr9hYbILtkrK4rpQFhd20gKBgsgFwzvDOlZDdMsgLw4L5s050pqYB20aHxlrV15SNFfzSqiWTwIAZYdHPDQFaoeBikGd9io\\\\\\\/vcrtBhsNZdHbPIBN8hUiX0T6+pmqEVG5X3gYXdLGg1GCozMH7VQ1dIpMv63WpTUF2nxaZGBCIk36XltOC4PesGREPOfM9+zoUPXyt4tleHvF8c5U6fKLh2OPZT0AZnqOw8knp7oAFuiq7y19aabM28YIOvaFbqwUGdsqStHQX7SMf57RYNl0AdIj\\\\\\\/pG4Jlqt\\\\\\\/HZBIJd183S6ZNhmG5gsYH0ThbSdSQ4kw02gVFm01sZ+2INNwFpokcAdZPP4vrVzmiD3FeiW157Wyupbno8zwFuaSHyhj2pe64NvKeQTtZQ5xPXF50oG57KprAzrB4egMvCdL5e0dxSlxJwijHB9z7yvLgehr2YdQI0Pd4UQmGuqGX8svULGUk1NoL4RJzZqXqlIX9JeNCj+IEzhYxUIfBdXlc2J2UgPv7A5hAGJnAopAa2J64JlVuNjyd0mAdsLu\\\\\\\/TogXD4ixm+fcj2UnAEOujw\\\\\\\/+WWX9UkCgXZ5Wk6aodm5O+BnQmJOIoI6VS5GnBsliE+sQcK6QWvSv5\\\\\\\/c01IMrqogFK9pL8Fq4+DdFCYmy8En8cY9CdUfzwryZHjJpgB0Lzx9Ah5O5zMEGne7iNMyeS5aSGQbxzd9tFsV+Bx9iiN5d1iqbGPiFKXvjVUDKnjA9NDW\\\\\\\/WFIigqd\\\\\\\/T7xvFGLpOCt7vrnUw\\\\\\\/ntXSRPIIGWYK3lnLSHN796myEZLEPjyLrKx1ELb8wZmR02JdxYpn43sQK1\\\\\\\/dmZ\\\\\\\/zU6\\\\\\\/cbXEQqh1WYeSqYhQhhQqSD6SXYiEKv2lhZw8j2V6RH0t4\\\\\\\/aBcJOqzljyKDJq7PSMTvBITxWt\\\\\\\/Ov1NLmTTNTmZRYe3EJEEhohfMdhumDQSoDHNs\\\\\\\/vhfkfDmTJVRhr4Qj7bhNAVs1NdJtcpagkBeMnaQnsNYKcZNt1E7hMPFQLreBA4pqt64E3NmH2esAGGYQlNSttoGLr9ud7FqxUCP3XaUjyhmC3VRUctjgaZ9eTgj\\\\\\\/6pzkicYwVkiecg8gGUjsy2B80L5Cs14JrHE43eG+pQq+x4FSSkW2nggcFPAIgsX9AqsoEAHQekG60T4EYQzFW0ICOpLH1mIVEJgHehhwAn71kQSjEPUHeVZ66HUyNVWDXTkGUY7WqQV2EBM0PMPCI9tUvdYdCRz6zakcpK2HOxvQa\\\\\\\/NSMRy25B1QNI+HW7PVjn313af1zRqo4YOdthrF2nhAHnR1abJRGB3KbAmpqmc\\\\\\\/dISUNkhcb1mXHZPWRxX1h1nFrOGXjMAibMPnEQz+KXTqImshxAJ83xC34buB3XwnitYVBtAva+ydTB6NsuYa49vld\\\\\\\/XdpY79XkfT1k0mxAP3VcDV9QWSzHaoOKeM22Cs8D09tU1TAYljde0znVuAYdNsOpNE5A\\\\\\\/1+Tw8OyON2AmjuDhtOtuYDCBFWBb60rt6WdZ5zMmHwDTzidCb6+ogoDrJBVha4tj6PMgbDG8sw\\\\\\\/1IVkkSyh51CDFoaPlVnStLiVbtzntOis\\\\\\\/DIgRVJ+i0gtw9FMDTmDS0koNYGxlqqVEko+gJF7wxwtle0k64uhFRzpt36H\\\\\\\/NeH9zoZDpSO0eRe8D\\\\\\\/AJxI+aVqpiRi+dqGb6Vi9o13sWGff72aggFjmXn8UcqmO0ouofH6ZgQJUW0fbO\\\\\\\/sJJX4Gopfy1SxyHgp6vRDDoHXuGmBB6+fd7Nq6JV1ATB3O2noUv4Y\\\\\\\/WBfoBI+FIP2+4yLRHH0TpnLNURBy5hKSSiYhBZSFp6u1iYjkUhaRyqSpYyy9fLPgmFXuaTEb3qSXBU9FvdStasUnOW\\\\\\\/xYsAOXJyXzNsjcaddwz+lCN0+MNuldaQnXIGop2TZYRuYE8+OVQEvcThOFXEPu4HI02yfqe2i\\\\\\\/oP3VFYH5R1LPurc5fXdChZjA4rk\\\\\\\/GPzU546PQ1AcBSA5NbfZacRTJ+K7hRAh4Ei8kA1v\\\\\\\/AxpBH5lxXEcx9Fmq2z5G5bn2wFBVPQDYNHPXZ5kJjBF4cqx9ARRgXM9YNwpcXMP0nQ3llVdxKlGvfQt3h09XW4+lY8+k2\\\\\\\/M+1n2OtfdzFoecwN6ZXiMmB6lUJ6\\\\\\\/wKrwOIYUrQJXynONYXUn85eM0i+dacY1RBp5HOfVUBaFCGLIt4t3M\\\\\\\/FSjfWZOPyUbtuXOlG52OBOJp2Ad+dxVnwrzqC5wOLMX+9nWjSBkDZKQC4xYKvR0VGKgGMjG8gnVlyDTeTvWPhOopiudkpCuM73kBBgCbTaCD+F2mmWN+ND4eBPPC4l6J5k7YyKwlaS5Jhye3ncNSLXDS\\\\\\\/v+v4yjF4DGJ7dIvQPAH0ZXSUVKNeiUkJnDKUbWKBsFeXf2LBQqKZL+zf6d9K3yeCiqxDTQU8LY8HaQOu18zjmi0UQjFN9rdV9cvarHepheNOyZqvTIn7rtcIokKKdXy3\\\\\\\/fRFoMFtYpxO5i6AKW9BoiIE3ofuLKs1ovzzxSVo3WAoH8B4BLgsJM3zsqUCb3S+KQCIsIx+BcovuatJsMVAV7EMod76llrdt368\\\\\\\/o0DK69FsZB1hQhrWwqIC10BFYg9TeuYOHmRzfN9Yt3GYqc6\\\\\\\/nvXG+wIjZAV52WXz0paY1rk9+5fP5i7qXnguzmVljO1pHyAPPGuMFMN+C1VdwWdTgZGU4PmudYTtUIoDlMgPc7o\\\\\\\/Ob1nRvAmnHzhvGx\\\\\\\/uN2M6hp7g\\\\\\\/x3kpaqDrU640DwcyO52iGkTrjmacWw8WWrTbHClRuFhEXkRAY5dybku+jG2lVDzY0FMI4Ltq80QOs7VywHKl\\\\\\\/YskDgw53TikYKLfGFwBEAxl4YbL3Dz1QQF0RV07d7c0assTlaZ4jtJ6O\\\\\\\/O+cfzL83RdISkMXnrFtdHeCxziALytnZ3BV+qHT68hFQiAoe+Lm4kVz8So+RUGA4VVCmQUdA6ok3sI0zYT09ut7uZi93k5F\\\\\\\/CpiwYNHMetDmqq4HDT8x3nr\\\\\\\/r4qm+Rs7jxU9g6H4K\\\\\\\/cub2s0waS2I7iiP5KAfMtCRvG87f\\\\\\\/zvSJsJvbg3Fw3nQH2QXsfvjz2Vvho30WUYSDBu6K4WbwZMRYWiuxHjjW8fBTnMk+qa0EF+3\\\\\\\/HTlWKh79b0i0a4fDys8qyrhUAa9tAp4100edW\\\\\\\/gKGHL9mYnyaXnW4e2ZmgZ603YnB+8ZKEZsRTF4D1pAKVPsHZRT9GcFZT3604YqbwIHgJRmmWZNTmdo8O8fmxpdv9NjCk18BxqVtsDzND8dhQ5cgGlGdi\\\\\\\/2P5Vv8MBgbfC9cFTkN9uj8PpNlVK9kxJvs+CrYGeizgukl7526NT\\\\\\\/\\\\\\\/Nvhjbc6rsVsk14zutIhfBcJykO5AANZPjND0npCxrGzINzxpfubD+cjS47DCroEvCTiDb1fVh1EagqWMitWHT26mUCTjeOGizm+J0lrCZw7y4EXo6eCpPnDyhYmcg+CdKkcfzsuN804v+KThFM8RxNMIf18gjfElhTTE3fKmkQ7BMNURjdb+JVd9ArHWwwIy0zp28ouD61MtJZslJXOocgqA837V0F8vCiMVbbJ7OCag00IgHDRAp3vceNGAwGCjc9vzleWSEU2mBd8bR2GRMt7AYW9xhyhQfiflO2KRxCBpiD749JX0eaUWzKir1w6tmlTkv9fVEOZqvJ8cG\\\\\\\/n9c9LL9FA3oP0o\\\\\\\/Pa1APyjbcaV6sm5uvZfsJN7801UczNUejeVhcHJgNEANQ6nRTzHKziUPs+5mLTVqv1EUY4Yh1kSF8GtvwUQu3V7D\\\\\\\/cmwsKm+WDWx6klF+te9O7aYNO35sNtbzc+2MRy2RibQZx0hnNTbZR5AvSRmfOyQF94Nx22FSn3TseTe3yw4dWv5kRJBP0PM58XV7oWm7V5T8hchVjIh4peWgr83oAxKvsI2DPlRLMYXDed8MYs6EgNGHHq8doe79gcPL5yVRW4UR+7QpoEFXl572gx3yV+SqQ73tk6NSZvoc3GNhr8uBFX7J1Zy91lDw2WLBHOpH0ssXfixuVRv9s3PA\\\\\\\/QaIhGjB1F25jB1n1k\\\\\\\/iBru6ZLQMssYKF55+ez\\\\\\\/oiI7XU13f0DUFJYJl492tkUNnCoh3ToTSOsun9kTWsrqbaR1dxuPjmGTYTCgHEuWPx6HKIg8Jmv1OC5Eel4vaqCKXT\\\\\\\/eBtaSHTiEQaGVHeuRL\\\\\\\/J3FqeZub4yJkLyn7tfcW9twyTYDnCn8eONd+4fCTqu+lB2eOoedLFPGUZikiqw0LOX8CImAwFi64MzntqxOq25gtojgzKrSASFoh\\\\\\\/wWK9tDFO1fgxrv28CO9cY+EFGEO3mYKuqWlENzu2E2hx0G607yTg5GBd9Zs9kXtBPttjt6KEMRm0wZFoFCSlkZiC05JiSDgB3hi0ZG+uYFQwJ4S+ufAvR1TDDK7eBkY1CmEXknn6izgTJvxWiNU\\\\\\\/LQqgZTjlWo8rRXwZFNCvtF\\\\\\\/hf2EFXuZcjmq+SCxgijkW2FJvDuSUsKmjcND2FSXTW7r9qc9kYL7qkefuAVWmamW5aqCPU\\\\\\\/J3Qo3CvDNUutShzaqRCqGV9UkL+\\\\\\\/q0YgwT0uNEvNouZlav9qGgfrSNhV+5QZb0lLIsVMvCyVXZZZ\\\\\\\/WWWEJb2crZOqadpcPntv02+y9QeGNwcyw3V72mEuqN0h0czhfypEuKMxFPPA14FCqG7DIA3RJhuvz\\\\\\\/fiCmLnr8mOeFpzon7EiyNiA\\\\\\\/39psV\\\\\\\/g9vsONaKAfmJHSAgFeC\\\\\\\/vLTnigDyTGkheXrzxcexjcZPIxq0HC0y+YaekNsCL0lqaZ3apStKl+P3MRYikFXIJOYBurgW7oVnTk2lsMq8HHrO5Dtm4X299ih9nm6O6yWa8sbfnwk28fFjcgpw\\\\\\\/1kBtWAXkqbNoHrif9Jjk0m16bvYmOJLdWns125YjickE7Rg73DDFT\\\\\\\/YOvAkNUVqM7b0T8ZKXHOwar7TqKls8f2jsEz\\\\\\\/k9NHwKybDaBNpD60TGRRzPI301boT8wqb5U2gSI1ykgjAQAmEvFNuQ3mKp1seGfoD\\\\\\\/Q1s5B+M2tVdA3XUqgHnh9SU0Gje0BWwZbcne6JaWaXCR\\\\\\\/mBVnZXhqvJMHP6fRW\\\\\\\/C33iHPdx0FCZPmAg9nlbi5Ro6nu+0qwd0h4EHiwOEnnVMi27j\\\\\\\/2ee1gzhxF\\\\\\\/p0PfL8sml4s1ow5Ms0sW0nEocqdWVBhBS0KeyJq7C69b5pSHNgXWEdDi9lcSQeKhPG9vdsEfZcGECIAGglt1DgTdoVKWmcbkJtZWvXZcnUIaS5l+8VLymJVUZ+bMaCddNTynT2XwomVnqiJXLRD6ojRuiIs4dSoz9MbWnyezEDU07Jo+Ei0NBygRuOFfq7CMmRPjYtCJdratV2Wiqj3NFlp+olvyYLuhYWJpN4M4huJc31X2Pl9Um3XFlA4nOupV3zv42VlOK\\\\\\\/r2iysPLb84pK9ADzqdO7Cl9oIJ3zOK8Fmtz8EYuFKVTGl1CqCcU0mn8poz\\\\\\\/jyJ7s1HURrxZknONmWglF7FoW+Y8SZLpr59j9hMeXdwzJMypnDPk4dXiZdTFjVMDKwaHelIdPs7UvtSFJxzMuEktOw\\\\\\\/bf3OKndN0LMRRhf6XGmdROFJ+eZeG2Yibyg8SO8PD1k+h7HP1fc7gRkshGiRRasVfx0p6DpQ9FmozRxIWW1o+CnvpRJ8LPMdNY9OIqaHrPSD2RTyn89GSVNjC\\\\\\\/ylr1cVM4asOFxxeO5JLY0rXsWCN4Q0DkxuAzWBFz2hLG2DgsVAz1iVUZIGpi8zYhfi46eH+aRvrtzC9qX2AcSBo1vZ0eoGwjxx4yCr+UuCaochtgUQ7tcku+vba9n8UY7wXIMsgaYBL7FdNcmkIqG5j+V9Uz9lJJJd1SkEpDhqk274uA2fTJbCxWIUDXOP534U7Ki8jubKDmmVWFPE8\\\\\\\/aRtOsETgni0+eord9wGpx\\\\\\\/D6Gd6I88yyoH5tnBLfun1Bo\\\\\\\/iThVoN+0imoQtS70+qP9cH+2srNmDAtp5bYUWus0DLF8v5l3ipBFJ0Vx\\\\\\\/BeSbMQRFx2WLi5ccYexaTNeqdZm\\\\\\\/cquj+djo470SKbmFNOPTQX8zqJ9MRmu4Dp\\\\\\\/TcZWZg1VN0WGkqnc6aJ+NhB78431+lNpJ7AUjL\\\\\\\/BYILL5PuvcWcoHZ7lwUm1cQt0zOr9a+7k2hA3HM5R0FsHDT0YF5ETlQAD9nMcjXcK6KKJOgPz8OLWz5AmhSWi3Jprgz\\\\\\\/lodU3ZKawX6\\\\\\\/dQrFGyDJujWJVDi+oQdrxoQzoZPOrLSbYPSRmOJmISffHKWvKsGSKmhvmu6SNIv5zcsDmJGUvfsJvNJ+JRpX+FuqK7BQfluwH\\\\\\\/dHxVsiQgCwe65l+msdD0G+8v9lRJ2346TIof7qvxRat9nJSgIVJ+mYTyRMdG+Xotx\\\\\\\/cTU2uXs1KiH5j32Pc4TNluuMRDLCkH5F1mDpQJ82xCcmf0kDAPyIZBbjnPn6KZMyjweOEIRy72uUOzdrUSziyDxCOoFGZDT8DF4lbka0Wd4AvNHlRnv\\\\\\\/1e3N6SZCn7Kho+D\\\\\\\/6qf90hUuLpQNBn389RdqAC4bnvAx3W17yflZKFkvBOSEY0JNMfCN1zVwnRqLpTGMFztN2VSpeUUa7uRdoQfQG+0VJNINGkpGCfSdwPALN39mn76v6Zu+\\\\\\\/XK2z9H26cEr4j0xUW3Uc9J9dcuQ3UvLLDcCwC8JurD47\\\\\\\/ujdC5ms2+CVX8sxJPYo6PIPNbbnVr+OPlP0kyR4I+6LlF6tE2B+KDOOhtw+YgLKa09drfTC2javmtXMmB2xqfcOIpRRS0hfi8qJeELSDWxketerIvuqUstj28oPRW3ayC2VZxJ28Y0YKlbMklE2b8pqcDuuqVUO3IWI8cEKkfnc8vFW7b6NxHHryKbWaIiI2Tx5ppTElb\\\\\\\/bDZ\\\\\\\/aTDpxoTB8uHkLUyDOxQ5aUKm4gvt4JHXeysekLYDHzp86TAUjEBpWuwonsvHvA+yCrFFYEI+KlK3dTTB56BHwnYyMt1U\\\\\\\/Qa3kiQfTZE5KBRSV9CJlcuBzMISqRjrvrwakPlPXaeMYXv6rStVqGcY\\\\\\\/+jYoKErSkOu2fSyCuWH4w5pdc4CMGoBRZWge4GtYb2gGWu6ORn+FybuGIHdCG1O8zIKG5dioPg4fLmmalOMbNw8DgTuXJ7IbsfpDrHN4KDtv9UZqWoYs0oYgI2ep\\\\\\\/1FxPp4TFUpzyM0r+u1ddUihM\\\\\\\/LyvHoDuZfRGWdoFVgJNUwMkiwIgQ+UeY+SB+ybNQJ2V29G0Nkq7UmKmWQE3eS8e8QhVTxkphs4eixFV9E8jpT5eCSIFRZyRE8BJiqqw8C5Q0pazL+OKlnGhKRz4YtCELrwJT+cuowSv9y\\\\\\\/tNtOZlorl8xFeOkk0w6mRQ0y23gDw\\\\\\\/WmfWXq2cnQNCzjKKsd72FNQeW2V\\\\\\\/VmdMaBKTQ0clE+hxINqwbQ0HwkobE6lYjFXOGqG+Ebp4XQcjiY73l79mAphCYUvMK3vg1ZjdMuzr8dbNPC6CB4f6CL\\\\\\\/pB4aG3uoNApGpieutVK99TKYAYVCWAJQqhZleG7NxaQbUhCiMistlMUpd7ck2z9a3z1FnXX27Yq98XALTAakEgBOI+a\\\\\\\/SYkBpTuXC23Tp34YvqQvu+sFUaVsW48P+AvdtzRWa9Fm9m2qKml4SyYBi4mJ0ZfzQgleMf+Q\\\\\\\/G7gb+hsH\\\\\\\/dyQ+EhUyhi3phRtWfn9f8cFG6bE1HN8Umuu\\\\\\\/pALtr5GmgEp4ZxSPCDzuc6oXL1nlcuEgHtfVKNYi7OL6fE\\\\\\\/C3oKJhHUFvvjMF9G1zn4PtOFWFNpDByeH5M7QSgWdITUE89BQmLc1q1AHyLI+ISfrPF7pgbvnPAbyAHOClli20+sqCNExV7k6K0x7nPJlVHIznvUcYgOAzd8rP8+bHrvNyZeHNbMwbnP25D9b2yg9NRwaKBPPkLgMX7\\\\\\\/zxb\\\\\\\/H1El5kaAenfCi8IQOPAsZtEQVL0kQ2eBF4reIGCjXB07bSic7O4wn7\\\\\\\/WDaLZ3wOar9cFDoZjABHt+ZQxJQXYZr7XoMRLuxdDG+OFlCyxJmpylbHqfi5zKZy8LJUGjS0W+5MmE6qqqLfCsDqCzeyQhuvDbdob1CZ\\\\\\\/MFbm5pJUnCg4sDJnnV7zM+ImvqwW\\\\\\\/bTo1oxHOzs+BG4J4EdhTW4MwOHNjzvd0twU0UjIE81sI9dkZ2cFBMUrq9SLegl7N9rZkUH6YRgWJtHH+rqU5NAfGyp4a3VuL2H2zW2XzGQn1w1lrrO1R\\\\\\\/ejtPM6JywrVoRHjIXhKUVKH0YBzhH7oBJNfh60HGUABIEGf3xqjZcSPX6W7Ky+4GeflanSfP+3TYhE3vwljh66RgwJZ5OdwY7vTTi32mvX9IlPBeq5Uz6yeJyTQOnXnRAPYtPzLKrqa08WS8w3L6clCnUrRW7ibe0iplGPzN+i6uEe9ndpE+6NpgY+trtjC\\\\\\\/bhqGtag4Cd5rCsz+xvxM1aEMR\\\\\\\/rdJW1MO4xJPm+eDH3xklZZiYDj5nvmqnes70V3\\\\\\\/kq5FoIuNkMFPi5RUbouxhKZHjrsISw1sHv\\\\\\\/czLAO1QUhCgNLUe3y\\\\\\\/\\\\\\\/XTNTf98iWEO6tWHkYVtIY6jvfuWssYpdTStsUS9bXmJOQD6z+0En\\\\\\\/tKj5+pGvRs6\\\\\\\/Osm0jR5yCK5Sbf6x5+b5lpRoIB8vmNnfnbgeRQuG1reRF1xvA8fwIuhK0tyDuroqergWZriOhPLWQzYbA1nXsEdk6e6IhyndcorT\\\\\\\/fpPeIWFeFKvJgCKsyYaGXEZtEHHfI5NnnYXfKZ9Od6GDi+sS+SqTvZ6BeEoYoVCVc5sDcYHi8imqJh7D2Vx6WMTfQCEUtcpwqOiqMIrA\\\\\\\/S3o74EnctUzIkQsmt5habU0tGS8SzUUhkit9VY+3l9lfqHravh4dReXhgNUL+lgih1BmLuX48QCFBIHa0dMdV67FK4qErTv2PZiUnYRihR02\\\\\\\/qIFmr4QjZHUOSPEI3BQG2JUo\\\\\\\/dBfZzX4vGHGetukNmUdkg99Bt1LqkJJPaFo413jWNQLXDp73QOImBCImHkCHEeC7bI7v97gylPYZIhUmA0Q2VPhRtxJz\\\\\\\/AimuFQNXxWlta5zEfMLhNoGeciNhhzDB+kXhpjRN7PgtkQC\\\\\\\/MVnJCW\\\\\\\/gSJGImW5eqXKw8l+\\\\\\\/7sCWWAlubDDfT3O+unnE4y6\\\\\\\/F1o4TF9AxULH9Io\\\\\\\/1TxMXZVmn4QIYWwXhUYg3j0nYdyj3Iu2A7zW1pws3Eff8kRQsQQv3Np9C89MTqXPdksO9Lh8AYWBOzGLGdCgDiS3wrjV871GR2gh4dU9\\\\\\\/jPAl3KJCMa6552hmbe4T4UdA1kKc+UAnClFJR2eixRtr43ISX2o3n3ZHxNi1zmT9lw0AoCFJz5CE7jFoFXdkfCQd2MEqOKOEkw3XlYC\\\\\\\/SGe0z1G5oUmfVe1EiUoX1CS9LrntH2iSvi1niQIJ2+n8NVOFXtSk1gZULPR3WviedMeIQabW+th+UbQ+V3ql\\\\\\\/IaIF\\\\\\\/GRDkKh+QhPKQqpMrz16OrArKVNDINpt4eE2+213JROfOHtwaYqR+jH3gV\\\\\\\/4CIiFklzJXCeaHCw4ZCyKmwx3ZmWyMOncJPRTB6+EKLl\\\\\\\/SQrahTPh66kyOPdDcHlQhiW\\\\\\\/XSeZjas9a5HzWV2McO8Jvh+SabptCWcuToV8Cz7RQonebPlyaTed5tlu2fshpxmuVqCfJDvt0XDGA1FniBnYHSENgpXLB+a8y7UOgA9j8uWyA9wbFe4cSwmzOR2h+MuQ69+dHi9lBc700xG77gjTb2tVYfMmkpNXlEE9GoElC2Y5W7VzmsgdZq1UcX5brJ4f6MjjdbrhTxs20r5faae5JQ3XDmPs9Ja5z581MYhlvsBcckVEIb5\\\\\\\/yrOiTSmA2E7cNLVJo5GVCXW5NfuW0HYmNq4u+5xPsEu5szkQCQDFVVfrzEpa\\\\\\\/Vi+SM+PiLrljBbIqPaYmnjmGfNFIXolZgo8ikbz4Sk7hzvaRvSRJyxjOfokYLnIwTmMKFcaFqiZHY2ijd9XD\\\\\\\/RNV63ltmsGyQOIpNI6GXnGmx3w1il9J7W\\\\\\\/f6iGcYG3MBbHVVR4er86lSC0EvTmtzJ0f8TVEf8TXPrt4gf00bTlkDIVCpo9qrywPUiXlX6TsostlkMQY7fjXdUmIjL6sVZj55KFocjC0FQXvBXFqZ9eH7EuZCtW4lFjTHGibi2c62Ix9N6rgPU1Hd39ZwlwJCVH7CCZUDzoLzktsTj11PurdB\\\\\\\/p7QMP52tzXzaXQXE7LbPgfBthr9gPnxK4Jbn135IY2OMVczM3WtlaPFGaxfR875+FwqETWyoI1a4yxvSfeDIPw3JV+okcHKK2xcmhGz8an1btQ0+n3uB38PpNK3Na2aza2rwsK+RffG1WL8eae\\\\\\\/hxfvpacuUB8pxnKzx4\\\\\\\/BKaZnwcYtNm6M3034V3WUSkcJWYJoadJlyQHq3zo\\\\\\\/E6lQlV6BYo6RQb3TAnPgre2CeoUYptQBE3NPx3muTMEzfqAuHKI0sc6nu8kLiC\\\\\\\/CtdfhcOmTUp9tgM\\\\\\\/d+TwvTmjEU6naGtwwMtArbZpEdfp\\\\\\\/6IIFC4UFsF5budoAnFL59Azx7qpckNHUR1bMf8en9Q7YRAz7wUtC7R7ZEPsWGu+6FxiUCe2yu+vgzlyKeGLT1wVF6sUxK54j6jctvuqLJQVPGA\\\\\\\/TVAWfoI7hYDJSrNI1EwZP\\\\\\\/mM4rh+CnfcPnXqMBuFcglpYaTyayHcwSX152iPIn2XRbaPAOUm5p6EAOUu77Frd9g8bmvUKwRulIrVAMDHLft4FngK4ZYDlKB9Sdg\\\\\\\/3T0RgvEqFS15kWNvJDSt+EOBJDtE0pwd1jHI1rNA4RHBSPTLHiri6faSekQ9B7Ja\\\\\\\/h9yWus8Bw5neP5AR5wBsgfgaaJbfvcaTesL8GXEcy3QJZXiiNrGdepTnvK2+fKUunkDMLg964UYIgV\\\\\\\/ZMNKAbrmxX0N+rXCp8nQ1sZt4VtK5BiOmqt0gQdfgJ3VjKZplPUghRhMcI7RHW1ERm5ML11ix6ufHglmiLnpgmQoMGl3iHTHIEYhLc9kIFS6spF46GuwwT+JU6MpOpEOX2eGgw0ywQ5SZq3pB0PAwoRVOQ61f4\\\\\\\/W7qquAlux1sMNP3mn\\\\\\\/JamHxj4WXeLTZ+FnqnR3KBxnYKN9pFLojiMUidZvovUSXyNVJbG46sY1YWhEOrWJ6+bfL3fUaY72AN50PaGyWkr\\\\\\\/zkrppc\\\\\\\/pVTxUzHhUedCSiVaZPjSEyk1OVKLiXG7+59IuRVh4lYGvEWgskSonlnjGldCE9FAcYvUzKhqI7yskA8hCePu5K5Aj7YWXp+GsJALMqxekgFbBBJq9DloPP0HXM16EjFNBw4FMwTr3TGOMuEoVcFeR8b4qrq64rk6EQbehdPZTd7tlzzbsCu9TgQCMJxPWa5orJdLlyTtrokm0oKb4CDRnDqRUbLWhf0qMTyXbz8o+DZPa11viGsEd3oQuG9sY\\\\\\\/QspYTwCM3MMSklXmf0EeqVMyi+Pol3EGB9OkbsuzOx0XbUhOlQ\\\\\\\/4CebbM92kBX+wh91YyDiJFxC0BQebZ\\\\\\\/dj+7lFiw4nPnTlenv7XRpAiCiPbol1O2zML3+rVAavHBLCl7dMpphgzSuqYj\\\\\\\/NuHOpUzDS8tfwufRWTCDZnZCGd2rGwb+gnKFxJTAkJYsiA8vSzg4q6Ay6UPXmJtu2pvhjEzT+6Gn8Shji2n2LSL7P3OWZeaROHkOH2I9HTwqek9+eE\\\\\\\/kk+yu9nmDTvxEc7Q7Kkgmmjm\\\\\\\/lRua00lznusNficTHBlv96LB1Zl0vjYN5e+hkxhnvljaTenKXRaFz2PFZcGl6ZSBrD1zovAiKOFA+JnyR2WTeNBtEYacEuamlenBP6aXkmd0rRpbkWUJaC4PS0gDm4kjvFIXqQA0Y2OcIYmGUolt29G8IMWKa4yEO\\\\\\\/qrP0cg0NU7y1pHS2ijUW6q4naDO0JYMVTuDEbTDIxx2T\\\\\\\/j5OVqXJhVLrpaPrO1jJEkidh+KVcmnfHxjaiWE4Xyv\\\\\\\/\\\\\\\/I3R2nb22dNsryRb7qZTGlvdBOMk0kgnSrDoYBBcXdbD9j0sOWFy2BVMqOsck5iHiFjUpjqcmrNQcFTUoB1JrIfThve3iLOuvnfjzGrjYJHN50Uxfh\\\\\\\/R8EBQQSYEBLTX\\\\\\\/CYlojnvvdKYd\\\\\\\/XWeF1vIJkyCcMVM\\\\\\\/R0LhRKSKxsBqYaeFgKYXwOpKkG0ZbEzzuMCRpS5dEVSFJXFU4hCc4v8jbD1nHbhnjM3Oq1xEIvhNZQVFvyQH0MHmHevHxZN7Y66JrkS8x2e3Bf3DnaXYQjyxaz53gJ1TPceD1W4SvvtKc3ASDYELysEjqfQh17xbjwT1qRDAlHIyHaoFrE0OJQaUbYb9bHKCvPoRwLbpRhgCpcm6Qq0znl2WyMwx69WKqhnhtC3RWDrgtJwPva7Na\\\\\\\/z4lq2vph2IuCpw05QAx2AHD5vfVp8BgWmrJT2k2ZICtpam7BMzksC9xbLQWC51Z34q5bkc2b+9CJ6oqE7nCIbsFLPdzXyJiOxqcXLvUsMhTriZQnMN6krmsnL13wUebJn25x0kOnlRDeprZ+svkGyaG9y4CpVFFVQ0dIXDm\\\\\\\/PZHxwiWIiapW7zRKacaaodTlrvTEhMbdWCvKdl5qMQQdgfBOgEneLYbZfdHZJiyrwQ8+yMrLpzjQGNTBH5rgvT\\\\\\\/0Quu9\\\\\\\/gDogZzApVcQWQumDTDn7RIM5w8MYt2RIEI6qCMzaqlm6yVQGQZv10zq5zCX1\\\\\\\/lbUYrTY9wBvVKqqFWou7oBoDL4IrnzmcCVR0lflTse8EJicOTck3aUnMW+0Mvxgg+qdI2IT1hFOV5iDvbdUe2dCnw4xCGcLeLZcAsUQfbRkgM4s00djKFV0WYQQuPYMr5qKVJKy6kPQOkYCBseQ5hIXrP+cwhghQb5TwIZL+T4aREu4IG1jAg+yY+1R6bKlkWrfb+P0GhUQuE8H8p05l7RZG8Sp9YNzJA7gidrxzP2GWEqjY5yQSJl2hLh+BC1CisTeVe96uQitOB8kxSqYy5SmZHizBH6xpvbR5vBMu2lxXhBTG0L7+6eeu3ICYCSu8Dw0zhWmfjHMmvzGmnEBnTJ4OqfB\\\\\\\/fte7Zd8k3jZZhszxUXMsemBrnAQXt9FFl0VhwYAsbueZ+qK5zgWRzIFVRL4VCdIsHNUYQxhX2W0MYAqZuZ+bz0CJqekV3HPa62aUBJ4OZLiwEiEEoDD2BbYHDrnnejpvq+OVBfw8SXjMLhWYar85yZECJpW8TRAH7O+vVDNoaDNmZfEoGJ8zB489hnAcjw2pqj31Biio3V\\\\\\\/CXl4pG9s560un0H7DSfAvkJyVea+HkmkB3wnM8y\\\\\\\/ZTPhNnJdd9eDEcykz7V\\\\\\\/VyMzuP1w9srY\\\\\\\/7iZ8fKzEs\\\\\\\/vLKhUH8FGPbY9NUuXjuSCNMMPogVBEBfvRayLb\\\\\\\/idC90Riz1tUoM++EPr3GEqVjKsWF+8FyzgY7mx1IKGv4YmJCvR5Er4Ay8jRrKXl0tUFbsid8HWCK7bDvEmkM+Mhg0ZOIM\\\\\\\/86eJyHzmVAlKCV3PaU619bkvDU9iTWJeaiiYlBtFE\\\\\\\/0vbWz\\\\\\\/L9G7fBlkLgWIKyRpTk8WOzd3jG7rlHKzUzUmNMzdtnLtIQ6ri6NqsUg0CGGM2CC\\\\\\\/0Z9RCymxp5DhzBd2qMcbSfvUktz\\\\\\\/lFKxfFR+0xJ+P4gPKBEvmjc7RsYgo4KoxJxaKxzWjJxYhyOmBUjXhLJI0rqYGaoOegoHOF+zTSSIVx9eFRCyHOvBM1RTlF3wQ72o+47eYfCVwTS1Umam5+z3CEcIJ5ziJPPWK2\\\\\\\/nQ1WDiFnG6IRaqO7gELiQBfKui6r7DseekqZDPb0\\\\\\\/X+9kQVx9ZGGkWjA8pU9IQh1O00kHEf4Kfs7kaDkSismSLADdnZDTHt8NbT4+ieK3orgKQdHTutBxpNRbhXMu5FxO3WgZshfYhjLxeIR5kFTtZfjcmiqPnOTG3NCaqIar\\\\\\\/m8qLkzUBfYhqHJl5D4Dpa+GJOIkA7q8xGLzKj4LHYfla8Pym2pzH+1RJWUrbdQw6Fvz+B9MP203wlcCPkU74lnx9Z7ICWyHph9UcRaWhgROWdn82cqxE2ZQ1MZOOdRN0Teyy1xCKgdTfnxsJ5erD6UlvN2dz0ltlyS7stJyp3AMZ56czeEfB01iDu0AuuFz38dG7Z3O\\\\\\\/fSvT+eQoncpfJgOi3aP3a5h\\\\\\\/8+cYcZkuB69irDXa2EPAKnG04fF4OSEgMxV\\\\\\\/L10NDABHdincFs8mlnJ3ehVTwjk46ES+rm+dsDaqgwwhy6UoJtudtroYBz5zCNZhl8pLnyUuVAUz5LkxNgygjRJb1u5tHGxiWBw+4vwZ9yNjViSeuXFljZmM11TC3341EbUvpzEeB5WsMrGJAOsPvpESJyuBcCXLeRIln13kcnt9R6eq2EIzffaeevP3HYXvuRad0XQ2fdqVN6ALwhVUwaWzHe+41j29qFt8TfQYwJ4QOVKjLj4dumC9f3X\\\\\\\/KNA08HWBddNZPcr5D3bNB4URH0hmiVV0bjJXO4VNfAq3xTsEbYE669Uovorq4JRmF9dRf\\\\\\\/0TTj73YhJyqSOfIcX18UbpReGwC7Qh54ch146Y6ph2C\\\\\\\/GfMtqLltfX5+Gga\\\\\\\/5P7\\\\\\\/4JtZS42FZZaXfOamhsiEgMAZA8Kh\\\\\\\/InMvlI8yJZHjJStUEKHsBhiIvl7xRyPLj0GD4U+AkyhlQqVO0SJNzJDau\\\\\\\/e7BOi2avHtMre\\\\\\\/NVXFQtA1N4QetXgysWm3VHBHt25569WQ0n7TIv\\\\\\\/uZoP7XcCpF\\\\\\\/JnHeV4kNrtDCZzy7CNDWtXOpuMi1\\\\\\\/pkIcg8O4gTU758N4UqcN3mSsdbjD6n97OC1e0tfDOtk0iHgbIod\\\\\\\/jn3U\\\\\\\/B0yUE3rvYmkSWW8qwiW8OOH\\\\\\\/I5odKgFodcv9+Uo\\\\\\\/njhGRB6pVQljP\\\\\\\/k+UxkyEsaZdAxtF\\\\\\\/WwKIvFkumbSLNIY7Ggki9smROt94awCxNMrgOwybrYucl8osnkZz6jFFzFHecaSwknUOvdAAHArolFYO3SMztwl9AmUolqe5lo3Yy\\\\\\\/PIAKwJystEeSBBiV4EcSzMMOG+Hzh9nfPNC8Fm0ocuLrMJv3oyCwAHTUxN22XxbclPYf53cEVvECs0EncAO4Xq9Wo0HvmgNHECE+NKyOIgXgQi4CkN9F8fBuCgUASNcCZBvbVbnkjWmOqB+XkR0SkHSBzcUf9kNXcXLm34tr23lg9hUedEI8V\\\\\\\/zinhAz+NVos9AwGlRVqnXhRhQ+CTMKk+DJr4zDPdJ3BoVllCMc+Ue+m4ceHKf\\\\\\\/rnjEx62QuL\\\\\\\/0jHvddOZ6\\\\\\\/LCE8mv22dyrReiGfiw46wHFPtVz+BAiIJhSxd+GayxtSrJQksoiJkRKlp13mw0LpHuCfZdgpiM01yDvDTwlhO39ZkBUOGcTS7K6WwV3TwMN6DDmwHe10v60CwbwKsS5wYwNsB3qC+XpsIgD18jxZv1BjOhlid2qkNyjth23MXGTnTfuJ2ixVakp55IrkZfr6VUvDwNGUQduEawd+0V4DVPGhwh7RpZnUx6o8X5ZkZbWOAwzz2JIHIYXCphS1CZvW2FGvH9EFfUVfIe\\\\\\\/FXgrTh2hib5SeTuUgsbQ53NjqJiYymx\\\\\\\/quPtoD6pgBeEZbUDoRdpaZmDH4Bjui8c1h6FNd8S6g5hMc+jvGUdR7SS1123Qza9ykmmt4tadEdJpJTBFK+Z9Twa2M+gMHOuKAETn16M9Y1UOtLNXwWdKTMMYLdzQnRWpywuKg6wC7bN7Y812\\\\\\\/zqMkqYV048tcOiNXk5510pq9DkUGuqD+ckpe5sV1Gphyh9PPXRgnTTCqSlZ9XCX1X6Glcs8ukAWhRDNClpjfK6+kji9GCFoE+fHanTpLS3cv0nY3F1lGHA9x0nDu37cJPw+Mam3dmjhxe1jLvXAm\\\\\\\/VPAMngcFuWLg8aqMXcvjzhgRYAZnMUfwyboNyJQQOKFN\\\\\\\/ORiRWNLwEabtK7AX6no5pq1LnfqixNOy8rzLjO07b4VDX7CP3RTJIe+xTeoM\\\\\\\/34aW5z+bx2pLtgk5sXMwwmAVp6D0km9LQXtqrYLNK9dsysgx+fl7zrg\\\\\\\/VltNZNdUd6zSvaAloAiflxsB9foxLQ1lOKbenE5K4UIJ2GGnI7aaXwPW2fkxNz+KUGcr56vpWPAZv+SwytyB9ZT00nPHaCX+2jztWZ8aCY2upuKPqXT6I6Kh9xRsj8TtLG9ksjus+ZssC1IKVDl82AMnq15CMKnAvwBXow3fSeSjl6c4Jb\\\\\\\/1+NP0oTJO9PZBv7PoD\\\\\\\/jZ2hmeBsTCxWjiOhQoUxmT1d3+1JtaIjtKMFjtFmT7L+pPFeIRUZ0GeSfXDsUZdUKpBwyKSRcUxkFrZv5GaHWu0EhxaSjM0M5lFkqEtaNKU2uk3RYsSvUXG5sLYKeSEqV\\\\\\\/WqKJOh6zdgAdX1YpUqigIhiijXDqcv6bkrPjHdqZVKATxZ+cssMgFi7y38IIPQ3swAncLWZoPnmyVTC8Jmh6r2RYpgf65LO\\\\\\\/eplRTqrbN1hEKKppZ7yHY4M0qAb+A+L39vu31t+AfcgU3+fGwsY8IfbW99wkLh3JWZOPNFc\\\\\\\/sgEQYhynkbyQPffRiGPUTz0sPfNyHb8xdhOtyCHeMax83Tyc\\\\\\\/0szzigsaF75S+RF1nHnk5W19H8mkjnfdIcBRXCMrGVgeWZ5pmQ23Uz5vzeht23duW2Uu82RyKGLSDndhekduDyl5e2Zd6smBdo3foL0Qev86NTk4RHB\\\\\\\/H+pEiwoBdbCrxYt4\\\\\\\/NG9dE8GS3tdEHXKlp8HieJ1B4ShNqe9IeJcMiMWb7hHB\\\\\\\/URvK\\\\\\\/ONHSYhR0CcSEcbvHaHeDpV0rLb\\\\\\\/0Nx9FhalwOrkdRYVCLNfkVzdD6G5\\\\\\\/m3yGUKK2D9Z9osO8bWxI0SkSdtBSQuA8l2M23K6cn94rWIl00qH475ftQpI54cxGA+NqKS6EBYjaQTqR28N3VcAIMGKcQVTpGDLHKTII+eqiAdDk9rzDvA4zy+pr7ewqz4GepR9nwPpJQejRPFu82QANJNjmhjvKYs1KbkxHOREpDLLuRmFrkJRYoTHof4WH\\\\\\\/TcQ79oXYO6Ws92MT5gg3Yu7sTanHN7fo121tEyGpD+FofxJZIbT5xd72o0WpGpaSfzFd7IaGG7RKMDObc\\\\\\\/TXKj3dfdc8ngX\\\\\\\/9+f4wyTvpw2y+7N8qgBUGItMsx6mDaIvhYWxLrcp\\\\\\\/2o\\\\\\\/FEkQP8lz9EW2GVZfTm7pKRz4y6cRFNWkh+9iIbB4LT6btfztpH2bfMMjJAf+jcSaG2qxnfxxDjJGMfWp8TRuhNcdcKdkkKk6BkwJoTbOC8PycAp\\\\\\\/G+b8OpVnJJGZjgFpLN0wDG+chbH7g1+Vgm64B2N3Bv3dIh9GRdM4xu5vGru3GvP7jHM5Hk6XOIFlIkHWjXUBUWBCPqYvPebVAGU0O79eGgsXXO08qiKDUhYoGre6tz7RgF6TCQI77LEpJs46RqR8D6nwvJgAFckifbJr8CB6qGNS3DaKIziyW69rH6J37Zkb5b\\\\\\\/5AJu7IsFrmE8l3PjpPowCW2kACGiOHfGLRHvLaZ4ssJPpg0jyaEBVkBQyBUcI0xBRo8A0mz\\\\\\\/ndx4MwAJJTLnJE3D\\\\\\\/Y0A6oBhrjWFII\\\\\\\/lDjOOuzCwJDpIRK\\\\\\\/QP1kZGcC\\\\\\\/zonLRyxIhPiZPdxU4hRI8BFy8JaCIkIMIerHaPqMQMijbHOwzDKaLK0ezj6dmAD35RWeXMbxJH7rV9i5Nz2xCHLiRF\\\\\\\/l9xB74Q4YcBlPy26YwkNcvZTbZMAPdwkCParA1JmBTdU9bady8B83BAsguBnj0AK+Isc1WRNyEPX3QlDY+TlKg\\\\\\\/E3\\\\\\\/pQfaIjpehp8VVZSvTlatoTC1sXQdhVv0dxsUyK7yS3i6tovVLEoMZuR4K4+R4HotNS07YEM4+QDjxEKLsnNjnMQqumfjcptlDA+iD+aUao69Cm4xktc4pz9S1jJlTCjPMvee18fLrQ6bsjLHgL+3ZCH+fLN7sDOwJjo3zv9hPfnFaf845VwOdTln6t5C+\\\\\\\/QjZoYcZHFBBLpw6pupZmi1D31vRWk2c0DwGTpeE9\\\\\\\/ej92VV2BycDNVbSE2EiyXM870QvGWC3laWEWTBRTDpo0GP+Rz60j3MndaA9wHKlYju9RbCBpjwzxi3AaB7pmuXxtCwlNKlVjTU6JDA\\\\\\\/kAjmCzii3+EV1pPZUTvyRwPr0U0vPnPIpd2KlhDPLLhEvMr3I66Vpjfc6GYdZLo\\\\\\\/Gw9TTp1\\\\\\\/DRerxQ3AI3p\\\\\\\/BtnUiKEsQngr6q0UozLda+F2mAtS9vDhAS3Nl2du1X0GjB3NTPmAMWdK1d00sBTB3Jg7t7lLgv7Qe5W50hJ2Kh8e\\\\\\\/0V8CNqL+eeniMC3zN8Aiakp1mpRpClqWEilRajXoRAJ79upzWEKF+8PulXX9abPXNxSY6aSvoRBaYRacbhEgQKEnoLUQs7XTzcEcdUUmMIcTRNf1EZxlDhZjyH1ftY6CTI6hnGM5SZSLhUBCHjfMPL\\\\\\\/+3FuolpK1wYtpovRqvzoF3FZ5uF4JNLOsY+5rLOmXn+2MROLajvrAa4o+L44MpMKGYLnMXOEsET9LUc7zEG2Qhe6H8pgQ5bqn1MV9cHRem6UhxVQknB3fWQUZFlo2OjKz8snb3y0BqZ1\\\\\\\/3eoSx1vRiocFjb06BERFEfyZcK9DDJsVI+FqJdJ7OUBRCtjKXWA0hQ\\\\\\\/E\\\\\\\/E8r6LrXG+Q5IiHtEGMyLXNFWv231QL2zirI3hoKLBfF8A3CNGJmI3t1Fyh\\\\\\\/csOV7KcerKYeUvT0dgsW6nCiXYRMApoq5RbFNVd2ExYj2oFYAZX8Z3ZKC1lSzxBSpJTdWAcmATY4B4HQRdvUHaDAJc5zOuCE8NKtJ3k+ZXGaYh2WA5zsWGaXispYa7rMVBqnaReOSS605g0Iusu8MbP5OEElaIXQZcDl7nadRmLGjJCfE7yj1TZsMR9qyR5hNUTrxe3gdyVAiQKOGuyzh5Y4FjQQf46Q8mwvatpEwbe7+wNg4+O1tdSHmpw8\\\\\\\/q2uLn9wRmQSrus9m\\\\\\\/BJiwc5\\\\\\\/kknv9\\\\\\\/bTgW6\\\\\\\/Gu5yy3S75Mn8SFqRThN4ksTSfNT1utmrdXAwKetZMYXMK\\\\\\\/0RtYvrBnwxmvWlHN5fOHKKrGN9QffvfrI2l0hHkm27nH9zqmGXe38EV0OTi0XpGlimaMzl34tseuJ4\\\\\\\/OA2T8vEgZazT8pzpH7bOnfD4GNZf2D5D7nrxfmuV8GsAbs7WGymbwhkY4zLWFxwWPvSaeAV8jq1n03xVjUn3wgf+sf+Nei+RWMFgYzpfRmpSi4+2a6PhPJ4lJCEfvfFGJ+bX+p+KEDtgOU83jkyLcf7JNuI6MztMvcXJZt4BeWFgtA3+e6W9Ux4dlxR7RdBgDoPXdNEppMBLa6i0fVP4lTOrP+aMQkPUqc19ayPxUhFFtcu5EAIDIMpWtgVpx+9vCmR9Li0FVO1XZBQaVt56kGqBfLVkN\\\\\\\/IwcSvb+Fe70b3yK4hH3DoQVQ\\\\\\\/aJXUaaSVTHnpT2\\\\\\\/rrFOS4oRJPEIC65dDVft3yBtnBz5rKMRpaK1U+xjktUCEZJSCvDO9f6fyS\\\\\\\/zyS\\\\\\\/BBLF8\\\\\\\/0H57mTgt3Lb9arsHtkklQAfxfjZBD552fS+Q1ZxHMeeoX3M3hXITAJpO\\\\\\\/8P+U\\\\\\\/nRSGpnNaUapuD2XyVm86ktWZCAPAQGfX7fyScZdBLjDh66iNVZbkVGjMeYNyHg+aEmg89lliYi7zclKHgxq69jGADcGe5Skk\\\\\\\/ucxGGLKGwCjnU23C\\\\\\\/cZPNt95PDAHrVzKQ3+OzC2ox4MS0eRPXJN4HindIU6bS9+tFtOaEuRN1KuQ8CRgJGyXaJgSFkIDncvEsqFqyYElQnu2ULbkQ2J4iaJz+p726zhVSj31\\\\\\\/DSzASklvNjBJhVut6N7eAlyEM3J2bJLaGPAEGdUIHBsM3VN5xaYNj7mDxjUwb2ui2P2n7yS3ZTp3HfpHwTv4OmbGwxHJc8PCqUlllmpnUTKrJ8ijljDzXdIOaoPTOQZYe69zHsSPFVMNAl3M6TXLG9B0WQaQ\\\\\\\/bIZMZWEN9bvOcW27mmwUiKAcAq1Izs7cswLtQxMHL9oB01hOChHQaKCKfLx9aGqAwUunq5XvpC\\\\\\\/Mcs5isCntfQVNRsVia74qji8BJ1K85lEs9fu8FavCUc9xHlfeYbIm5ocm\\\\\\\/bYpFoqiOLY\\\\\\\/UZ8p27CUQhXBzpxUmkX1m3v+LM2KeI0He\\\\\\\/2QMbr7iA1NHFvE15gBq5TarNSGuB4g7L0YWSynk+BMFIepiI2ueq07HPEMMy31Ccm2RmGPCeXHmEf\\\\\\\/vikw\\\\\\\/Yy3BotqBtbehUN\\\\\\\/pbCRbtZ1U61Wt2yKEcaqRNL61wxuEVgsViHFOeEka7WwNib\\\\\\\/MM7BjlBdn4+pxNnkq\\\\\\\/gAd6RyTLrFXDO3MyfX8IftkSSBtacokTrhsmw8nArSPR+8oy8O3GQm1fqQasQapm\\\\\\\/xwhPzQyVpCl77OFQDcVMHdhnit8bEmJkfPQpBC4A8GoRtigk4mzh5ZvteQRnutF6hTS7O5onSeo7dkBkj0Bju9ZGmX0x9VyTFgyzBFbZTSjUknl+ElCtLav1ARswgI9XtWVdnc+Tv1kZzghO5xJL9IxhWe7u4iBidFvqMJDeSh2xKr4YrhMoZc+ldWu30BwHnUMoh2r5dyvTqL0gzbn8HU35kYyLuHTlVVaH9FNeat8+Ut4h9uATmNNLOwaAAPcmYrHI\\\\\\\/XAB6VnzeMDD7Ps6Zxmutviba9T7hxtBNnGTjmU\\\\\\\/x97T2az+RBaMyeMG0wFMqxyIap8zevmSCk0gvLWvo80EDKaPUdMuLwHKFk7RwRIqkoLsfP4CiMGV7iprJ\\\\\\\/KVnAbhFN8\\\\\\\/4Rn\\\\\\\/+ysZalfwc5KFNLmgXmhKQ7u5nOS+ZfM0jROOvjsTMLEOgLhSmY3XVWokZt8\\\\\\\/Dw6rjvVakxTUsj2vnd1NLfslQHVmFVCMSFnqLOEz\\\\\\\/bJp7NTeBQM42iv0wYAzAMhMl+5uV\\\\\\\/tLHwBLAS6f\\\\\\\/vFP85YVoklHiAVCAEKpLSjgP5L0wNwOSGr5XVgUpzID2MF\\\\\\\/SuXatPKmGwfGQ7Q2DBykt1UDSIz3FYBloGPryGpWAbN7Eqj2lkp6OK5we4Y2bYiGjM2u8tKRU8eQT5vF\\\\\\\/gfwNwZCCYQ+SAg7vGGmFjt7lQ352qGdVn\\\\\\\/84BRE90tbh5Wa+zLDTSW6+9d1llh52B2EDTacJBzVkfKqQ6HPAs5kvUSz6BpjNBjbnj71RQUbHG0NIzyfHvmFORnr4zoWIXeAiUbCyrVjKtgU56GQT3jzw+YiedMSY+6HW4QIE+sEti3S16Sl7aQfE+SHTsvdBxkeks\\\\\\\/Epd1NiqJ\\\\\\\/bl6OIsxhVi792+k03euPX9RQ2xmo3wWqR07XExzKLC5yMYpXbFDpTPsHiYR98ACeB5oIjN\\\\\\\/KJUJb7EhoA7desMJbYLl6A7CeXRWGuO3DdNybCc5baDnYE4l1dC14Z88oRbe+JHZXW\\\\\\\/MuzD\\\\\\\/ZUA09uVTjUKtOMI3ZleYH1K1HsRWA\\\\\\\/Izpn4cDCSLN54HJBqU95DOmFc9G2OLBeit0NlnJvmXXKMk5EcZByi1Xf\\\\\\\/bXqTbje+aS1mQoOX6aIcMSf6yCfqmPVK2A9rmSALqwHh1CqDv\\\\\\\/NDL3T6O+CS80CH9CiSGqK5hYl+vmeIHVu8gEG7iSi9meiTFUvX4i3nV++9XzFpvjsPBg8Vy1kaRtxH89JceS7a92jk2rXpDfpn3NUfRUHrc2dKeMUeJCEerqqtLKYFNnsi\\\\\\\/Q4RDM1Zi66law2FYeYRJWtxWt4uYfLsDx1W9WP8R2qWDPk39nuzOVGlqoZMNE5JuV4Rw9YM3SHfHl5K6vWMnT582F0dbrU+bLLr7sfFnm2XAbzgpzsPTkCUHSneos5Bo7amd9DtI4BsKSaV15pSnpT3xNmM6\\\\\\\/MHhZ5cMiATQswhoVZ+ZouA4nkFXqaU608kGUwDJsHXShA2dLGx\\\\\\\/rpCFuckaS9Z900GGjy4+zY5I6ebbB7JNQRxcXJFbtQMxjhdSwGhK7uPgnNgYfPsxSj+Rhpa3PFFwU5\\\\\\\/Q9MLw7ZAgv5iV4cJ5XGD0ZlxE6Dd95XLw0I03+ie\\\\\\\/ehFObigqoyB1xUuh1bU3EWR1f2lEVG6KFKT919M6nm66FnLkTXDEB3vZvck9VCgKO1OWvk8FkyV8GyHOgGI\\\\\\\/eJqVTrWXx\\\\\\\/E5Ht9+f4gYrzsXq5WZDEuW7wkJXLgh92uwfDmmZxAO5boXng7\\\\\\\/1\\\\\\\/4GAJYAKpN+u5ND56J9OGr0EuS2PD+fVW+RymLABnELuDgeWtFbNA6NxBgFC71bx9xCl5VYP64LAfb1vEVusbXb\\\\\\\/GwWl+0Ra7NzJYVESny7SJTGPTU1z7Z5yxVXSdY8u6JO7IpV2pBcPmMqu\\\\\\\/FvVeq8cv585P\\\\\\\/6DKsq\\\\\\\/ygvj5cpIuk\\\\\\\/VIGZ5lFXLrQhqrbSoJt71Cdwq4ey8R0cd+Jiadx4YV8Sk\\\\\\\/7i9yz9ndSBmN6wY7BUVtFJJI4kG+lMZqJWw6bzf4Hn\\\\\\\/8COgraKdmM8d\\\\\\\/mMRTj57h\\\\\\\/zmVMlxFv\\\\\\\/ItHUvwnElXMdJmfBZW\\\\\\\/UmOiveToooomyMj3vW\\\\\\\/FoiKD7kDFGV+tB0+t9NkZ8JhSKhYrk2Ck4jkcABEA8IB7pAXeGIDyFvg5XT4SkZEH6y2z9IsrKTBo5Xrq3zW8UMcQmnDkmk3X1SpQ7SdKj7iclS0MgbAvzXz8Tx16WoDUSiWAwMYyAJgmHBrLCXdRSeySNPiEAPDV5Hn1qQTcYB6KsKGK3ZtJbuPS1dyQrE5Zwr9HmM7EoZie5q2t9QtbfBZ+ROKG5xntLrWEcUoGemT9\\\\\\\/cawXETnaP30Vgb3nNi1lePQlFZqg6PiQmM\\\\\\\/p73Gj5vNdK8TidTc1pCMmYxq14XMdsa8dU3iK9ao8pQQJU4llnitBu7cMeJMbzPU2H8O7keDMCbgRPA2Eqlx22HyL7oyd4Crf3x9EhFJR7VSk812hSN344nzQzN8Iwiw0xtneP0Qzue6qjM+pwnXE6RKMapQ5Pw8Aaol2Qw2FuleE3Ds1eGC4\\\\\\\/9epB5KGTIMF7ncJM7YEoIH79OtZ5th2MS+qSZaNzSMjUbnYUS5k6\\\\\\\/VVSMVaqRr6YjlkRuvbT\\\\\\\/Ug3jj9hJM2uBil8LBtPb6FjhFNEOczoGP+dK1pEseZfeVef8EYHSKMXlgvnbpkVQdHBs9\\\\\\\/i019FzZZ4SldFAoMih2\\\\\\\/\\\\\\\/PVBPrzV4R7yyBvXxalHGHx8a3swdusMxeeXRU\\\\\\\/hpoi+cngi+DxJLIS2JsKmvHeRo7Q2W8HOxDaDa2D+qWcVqH2rDK92r6FDochdZnl8U3LSAgJK+BK9C39ePH6FWn2Au4I55LAnfnTFoCuEPn4NxilRqsHURskN2Z5cJaxdLJYC\\\\\\\/mIVTS6+fkEgNQxqECrsfWUQn2VK9MFN+rt5Qz8S98S83FwHL73Lrz7SjkW5mpxv8FQ1tVT7ZQwv9\\\\\\\/9gWO5xD6iuRwnwDGcXIY\\\\\\\/l+wLCyv+cJIT+kd6xENTeihC0DB5aKFevjWz+Ri4awWrNY7tt6p6zCUgBpeRP1KsDdBOGexHAfmaKRlAa2wIi\\\\\\\/4GgHsoyCI9VZKtQRgDmru3MSYOgG+9ubNQ\\\\\\\/syB9s4lbxvNBLpmOuDoixBoUGoVcUdSWh0Wf9BCsaewV0DDjs2zjZ71lQWUT5cslV8NUtlMEKnXDpDYsbnm3Pd6UH4jE6Oz09T9KKIIsrNO7i1w0KpcbBs5CbcCfVU6vHLkMtvc9ryQqk2nO0nN26B10AToskFxqVYZl\\\\\\\/mMS09z\\\\\\\/cgfKqPBHlta73KnjCHpCUUGNe5g==\\\u0022,\\\u0022iv\\\u0022:\\\u0022G\\\\\\\/w4a053nfcXG98zwwxhsg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022bd6e24df3dc93dddaa3377ae6081d79be7fd6696615c6abe85f26a077350a66a\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022EuGIWLLb7uHthmK7ddWVo+82hhTzUw9olA296ko0vyjNPuzhv1C+KMv8LyEeW7QeavQ9Wm1p1x7dc0usFh5bk98\\\\\\\/RHLVF\\\\\\\/p6SPZONw0x135tIH2aQTseC5gwtw0f1OPWU0SpSQ5oUKaBcaxATG2mtoRPfVqWoLHLyOItqaI56H8Hy5UwTE8Fj+7san5Do3K3tLdoRcsuGsF7uR7BpYKzkzeeyk6mTa\\\\\\\/+mpnfLDyHS1Q0arkpxLb3F7zXNE828O2n5OPYDrAv+4VnPbYpn87GM6Q4swtlyPV90+fsItesmlTKQ3iAjrU85HYgNe99F3L7rkzvMJwIcm5qpkZI9PsJ9ZGjQzgrbihTxF56mYufJPEs4ncAvAsQGcUfmjE9fF9OQOWa1maU8NIaF\\\\\\\/Adgz96SWNL0TcZRl+atB8ai4\\\\\\\/1m\\\\\\\/GnTsbZgltnqoFzeFGFaFIGM3xYRHK7Uzbi4wnzq2y0QhVIunxiiHBIyris+S8NgvdGJeq9S3K748HfIftBlNJHGszghh17RG+sEBGvFoSbwyWpbejVKJw\\\\\\\/OcxVwnlrO4S2hoqSacNf3AwwCjYar6MTELmz8IssnqjDIXkuY\\\\\\\/k3ynFUC0DmROiH4KwAAoLNOkE+oCmFPOgKznitpsQA2xhaCQqtTuSKGgNy2vxRylV2F+Yu4+TaIS3xpgT8naYmIfvq9\\\\\\\/4OWCGcMzJNAK1Nn2XWq2r+giohutrCPOtJ\\\\\\\/xyvHmVZAkypm35tv4VdVXJqt6WDq21ONVuNxQ0UMu9mwTAXtLoLhSYZUUI7xNwvYLQnJG14ZoqGZUxwTaW0AFOj\\\\\\\/ynS4Nu1NMn8dqLoSg9jml\\\\\\\/8qzZ7DSS9k4lef8Q1MjunlHsjWCbCp6FxjJRyBhYByN7ZMS8Gf1nVej+B2b1Asdu0+EDImJo\\\\\\\/6i9yfQAYfen1Gc1aIcDwJlRD48UIzmdOyK2KeAAXOi4dZds+HpoUtZjRsbdXTCSqPfhJbkV88Km23B19tRr\\\\\\\/lW7tcYfqgVjrEFQvaKxzoo1s\\\\\\\/0CpF55ofzi69WurVvhJCau4RT6R0Goxf1kY6A0OYnxPShFw5UOsYQER0Bp+fTcdtYc9JOeP+n1e56RskT4Okh94ooGZw2FoUJql\\\\\\\/n3xMKMDszJFTLhxStnMd7R751YU56tDufdS6NXUEfLFQDf3+WVLAfYYTBI\\\\\\\/JlnFECSSrI2Dvg7AgF96betY\\\\\\\/Xh+kdijwFDa5WN73HyMK\\\\\\\/JCVZf90mkwXdfcDLFoiMGwnh5K0WIzpq9EKk3g4RzAH4AWuJ57CY19gZ3E+twjyc7cOySWGdTpVMwYwtdnFzPlW1JsDKq3jMR374hKBBjoisLpYjcsEQGkmhZhI4xCJ0L1ce3EMi1mUB3PU7\\\\\\\/ZMdpmVPlCCiEdQe1ifjmIXOlr0lGYGFIhfMyVeDgFH1dgOamQCatb0gpyWU07IMfts30pmgH\\\\\\\/a08CIZ\\\\\\\/8WIMTbi3PvdMoXSHTIM6vaE3qJkMLN6HsVP6KJBOrjUQLkUZhee+3BR30UKKq2SHI\\\\\\\/dKmxeoZJUuQ9SEsLGMHqx37J5UiMV6OJayioVTNhDqJrwG\\\\\\\/sKDz0SPxBY1YYiwzWHMobuyLWc6RalR4P5B2Q5IYTzTVcq8rd5Fiul64tyb+gXqIAlcmP1wKEARWQXAWOYAzpCJeIufJtBNdL4RzvdA1oLPofwmhRK968k9S42OSlsFILRCB0e8WqhHo8MOgTDMHMDG0oDpBCruFeGaRvge9Csg9lI\\\\\\\/+dMFPbMpXu47izNWLxGz0DUrkAvfnrvncy4WX9T5TMY6zeJdZr9hucVtgH38V0eEkxWOkvIR5DGEIabssylnbR9DZlqgYxyofj4Z7MKiFg7wCoGEAur0tgkaHCWdiQN8BG3KcCvg\\\\\\\/8qVTrpF6w7uXcNYUxQ1IcB2aBO9DiHh05IlUUFFFK7dqk5EExWw7MSvsW4PfW\\\\\\\/1RbnjoPJOIcPKrJtcyIXNpErvPPtVRDOBkGCoWcaanFVibnTuoyA9ERfbjIh0RlF+X4U1z4\\\\\\\/2Hrg==\\\u0022,\\\u0022iv\\\u0022:\\\u002204hU4UyQdKJ6PIu3swKqoQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002286ee96b557a1b58ddfe7f36fe36923c09e4a57f564f0e027530cc36a24bf0b38\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460457 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/11/3475" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-poHiWyGTYlwQSg4IHXoWHFZLrjmC95MMA-d-i23Rwg8" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/11/3475" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-NiCwglM0nhSVbnOWMYQzvt_aNoOa5dR19cefEP3uvJQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/11/3467" title="Depletion of Shine-Dalgarno Sequences Within Bacterial Coding Regions Is Expression Dependent" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/11/3485" title="First Draft Assembly and Annotation of the Genome of a California Endemic Oak Quercus lobata Née (Fagaceae)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460457" id="node460457" data-pisa="ggg;6/11/3475" data-pisa-master="ggg;g3.116.032276" data-apath="/ggg/6/11/3475.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Multiple Transcript Properties Related to Translation Affect mRNA Degradation Rates in <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Benjamin Neymotin</span>, <span class="highwire-citation-author" data-delta="1">Victoria Ettorre</span> and <span class="highwire-citation-author" data-delta="2">David Gresham</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3475-3483;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.032276 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460457" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Benjamin Neymotin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Center for Genomics and Systems Biology, New York University, New York 10003</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Benjamin%2BNeymotin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Neymotin%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABenjamin%2BNeymotin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Victoria Ettorre </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Center for Genomics and Systems Biology, New York University, New York 10003</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Victoria%2BEttorre%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ettorre%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVictoria%2BEttorre%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">David Gresham </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Center for Genomics and Systems Biology, New York University, New York 10003</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BGresham%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gresham%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BGresham%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dgresham@nyu.edu" class="" data-icon-position="" data-hide-link-title="0">dgresham@nyu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/11/3475" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460457" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460457/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/11/3475.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460457" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460457/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/11/3475.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460457" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460457/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/11/3475.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460457" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460457/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/11/3475.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460457" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460457/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Degradation of mRNA contributes to variation in transcript abundance. Studies of individual mRNAs have shown that both <em>cis</em> and <em>trans</em> factors affect mRNA degradation rates. However, the factors underlying transcriptome-wide variation in mRNA degradation rates are poorly understood. We investigated the contribution of different transcript properties to transcriptome-wide degradation rate variation in the budding yeast, <em>Saccharomyces cerevisiae</em>, using multiple regression analysis. We find that multiple transcript properties are significantly associated with variation in mRNA degradation rates, and that a model incorporating these properties explains ∼50% of the genome-wide variance. Predictors of mRNA degradation rates include transcript length, ribosome density, biased codon usage, and GC content of the third position in codons. To experimentally validate these factors, we studied individual transcripts expressed from identical promoters. We find that decreasing ribosome density by mutating the first translational start site of a transcript increases its degradation rate. Using coding sequence variants of green fluorescent protein (GFP) that differ only at synonymous sites, we show that increased GC content of the third position of codons results in decreased rates of mRNA degradation. Thus, in steady-state conditions, a large fraction of genome-wide variation in mRNA degradation rates is determined by inherent properties of transcripts, many of which are related to translation, rather than specific regulatory mechanisms.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/codon-usage" class="hw-term hw-article-keyword hw-article-keyword-codon-usage" rel="nofollow">codon usage</a></li><li class="kwd"><a href="/keyword/mrna-degradation" class="hw-term hw-article-keyword hw-article-keyword-mrna-degradation" rel="nofollow">mRNA degradation</a></li><li class="kwd"><a href="/keyword/4-thiouracil" class="hw-term hw-article-keyword hw-article-keyword-4-thiouracil" rel="nofollow">4-thiouracil</a></li><li class="kwd"><a href="/keyword/translation" class="hw-term hw-article-keyword hw-article-keyword-translation" rel="nofollow">translation</a></li></ul><p id="p-3">Alterations in the abundance of mRNA result from changes in both the rate of transcript synthesis and the rate of transcript degradation. Synthesis and degradation of mRNAs is critical for control of gene expression and cell survival, as ablation of either process results in rapid loss of viability (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Nonet <em>et al.</em> 1987</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Anderson and Parker 1998</a>). The <em>cis</em> and <em>trans</em> factors that control rates of mRNA synthesis have been studied extensively in many systems (reviewed in <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hager <em>et al.</em> 2009</a>). By comparison, far less is known about factors that control rates of mRNA degradation. A complete understanding of gene expression regulation requires identification of the sources of variation in mRNA degradation.</p><p id="p-4">Our understanding of the mechanisms by which mRNAs are degraded (reviewed in <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Parker 2012</a>) is largely the result of studies of specific transcripts (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Decker and Parker 1993</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Muhlrad <em>et al.</em> 1995</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Beelman <em>et al.</em> 1996</a>). These studies have shown that mRNA degradation is controlled by <em>cis</em> factors, including sequence elements in the coding (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Parker and Jacobson 1990</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Wisdom and Lee 1991</a>) and untranslated (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Shaw and Kamen 1986</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Muhlrad and Parker 1992</a>) regions, as well as <em>trans</em> factors, including RNA binding proteins (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Olivas and Parker 2000</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Chen <em>et al.</em> 2001</a>) and noncoding RNAs (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Jing <em>et al.</em> 2005</a>). However, the extent to which these different factors impact global patterns of mRNA degradation remains unclear.</p><p id="p-5">Genome-wide mRNA degradation rates have been determined for a number of organisms including bacteria (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Selinger <em>et al.</em> 2003</a>), plants (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Narsai <em>et al.</em> 2007</a>), flies (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Thomsen <em>et al.</em> 2010</a>), mouse (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Rabani <em>et al.</em> 2011</a>) and human (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Duan <em>et al.</em> 2013</a>) cell lines. In the budding yeast, <em>Saccharomyces cerevisiae</em>, genome-wide mRNA degradation rates have been measured using a variety of methods, including transcriptional inhibition (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Wang <em>et al.</em> 2002</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Grigull <em>et al.</em> 2004</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Shalem <em>et al.</em> 2008</a>), genomic-run-on (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">García-Martínez <em>et al.</em> 2004</a>), and metabolic labeling (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Munchel <em>et al.</em> 2011</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Miller <em>et al.</em> 2011</a>). In general, the concordance between different global studies of mRNA degradation rates is poor, likely due to a combination of technical and biological sources of variation. Recently, we introduced RNA Approach to Equilibrium Sequencing (RATE-seq), which combines 4-thiouracil (4-tU) labeling and RNA-seq for determination of genome-wide <em>in vivo</em> mRNA degradation rates (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Neymotin <em>et al.</em> 2014</a>). Using approach to equilibrium labeling kinetics and nonlinear regression, RATE-seq overcomes several problems with existing methods, providing improved accuracy of mRNA degradation rates estimates in steady-state conditions.</p><p id="p-6">Despite discrepancies in estimates of mRNA degradation rates among different studies, three consistent features have been demonstrated across multiple transcriptome-wide datasets. First, there is variation in the rates at which different transcripts are degraded: some vary by as much as an order of magnitude. Second, transcripts for genes encoding functionally related products have similar degradation rates (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Selinger <em>et al.</em> 2003</a>; <a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Wang <em>et al.</em> 2002</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Neymotin <em>et al.</em> 2014</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Yang <em>et al.</em> 2003</a>). Third, no single property of transcripts explains the observed variation (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Narsai <em>et al.</em> 2007</a>; <a id="xref-ref-52-3" class="xref-bibr" href="#ref-52">Wang <em>et al.</em> 2002</a>; <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Munchel <em>et al.</em> 2011</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Miller <em>et al.</em> 2011</a>). This latter point suggests that either causative factors are obscured in genome-wide studies, or a combination of different factors affect rates of degradation that have transcript-specific effects. Potential properties of transcripts that might impact their rate of degradation include transcript length (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Dressaire <em>et al.</em> 2013</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Duan <em>et al.</em> 2013</a>), GC content (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Kudla <em>et al.</em> 2009</a>), transcript abundance (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Dressaire <em>et al.</em> 2013</a>), codon usage (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Carlini 2005</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Presnyak <em>et al.</em> 2015</a>), and folding properties. However, testing the effect of any single property of transcripts on global degradation rates is inherently challenging as each parameter can vary independently across transcripts. At the same time, many transcript properties are correlated with each other, making it difficult to identify causative factors. Thus, to identify determinants of variation in mRNA degradation rates, all known properties of transcripts must be considered simultaneously, and experimental designs that modulate a single property are required for validation experiments.</p><p id="p-7">Here, we analyzed factors that affect mRNA degradation rates in <em>S. cerevisiae</em> using multiple regression analysis (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Eck and Stephan 2008</a>; <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Duan <em>et al.</em> 2013</a>). We examined the contribution to genome-wide variation in mRNA degradation rates of multiple transcript properties for which genome-wide measurements exist, or can be calculated, including protein levels (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Ghaemmaghami <em>et al.</em> 2003</a>), protein half-life (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Belle <em>et al.</em> 2006</a>), RNA abundance (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Lipson <em>et al.</em> 2009</a>), transcription rates (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Pelechano <em>et al.</em> 2010</a>), UTR lengths (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Nagalakshmi <em>et al.</em> 2008</a>), ribosome density (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Ingolia <em>et al.</em> 2009</a>), association with RNA binding proteins (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hogan <em>et al.</em> 2008</a>), the codon adaptation index (CAI) (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Sharp and Li 1987</a>), and normalized translational efficiency (nTE) (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Pechmann and Frydman 2013</a>). A multiple regression model applied to mRNA degradation rates determined using RATE-seq accounts for 50% of the variation in mRNA degradation rates. Although less variation is explained when multiple regression models are applied to other genome-wide mRNA degradation datasets, many predictors are significant in multiple datasets, suggesting that they are reproducible transcript properties that impact degradation rates. These features include ribosome density, CAI, nTE, and GC content of the wobble position in codons (GC3), suggesting that translation and mRNA degradation rates are interdependent.</p><p id="p-8">Using experimental studies of individual transcripts, we show that decreasing ribosome density of a transcript increases its mRNA degradation rate resulting in decreased steady state transcript levels. Using GFP coding sequence variants that differ only in their GC3 content, we show that coding sequence affects mRNA degradation. Increasing the GC3 content results in decreased mRNA degradation rates and increased steady-state levels. Our experimental results confirm that mRNA degradation rates are determined by multiple factors, many of which are linked to translation.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Plasmid construction</h3><p id="p-9">pCM188 (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Garí <em>et al.</em> 1997</a>) was used as the backbone for all plasmids. This low copy CEN4 plasmid contains the <em><a href="http://www.yeastgenome.org/locus/S000000747/overview">URA3</a></em> gene, a constitutively expressed tetracycline transactivator, and a multiple cloning site with a <em><a href="http://www.yeastgenome.org/locus/S000003809/overview">CYC1</a></em> TATA region upstream, all under control of two copies of the tetracycline operator. Transcription of the gene is repressed in the presence of tetracycline or its derivatives, including doxycycline. Plasmids DGP148, DGP149, DGP231, and DGP147 are pCM188 with degenerate forms of GFP (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Kudla <em>et al.</em> 2009</a>), in which the proportion of GC content in the third position of each codon is 0.38 (GFP1), 0.60 (GFP2), 0.67 (GFP3), and 0.71 (GFP4). The coding sequence of each GFP was cloned into the <em>Bam</em>HI and <em>Not</em>I sites. Plasmid DGP217 is pCM188 with the <em><a href="http://www.yeastgenome.org/locus/S000001747/overview">GAP1</a></em> coding sequence and 3′UTR cloned into the <em>Bam</em>HI and <em>Not</em>I sites. Plasmid DGP218 contains the same insert as DGP217, except the start codon of <em><a href="http://www.yeastgenome.org/locus/S000001747/overview">GAP1</a></em> was mutated to GTG.</p></div><div id="sec-3" class="subsection"><h3>Strains and growth conditions</h3><p id="p-10">The laboratory strain, FY3 (<em>MATa <a href="http://www.yeastgenome.org/locus/S000000747/overview">ura3</a>-52</em>), which is isogenic to S288C, was used for all experiments. DGY696, DGY697, DGY698, and DGY1281 contain plasmids DGP147, DGP148, DGP149, and DGP231, respectively. DGY1193 and DGY1194 contain plasmids DGP217 and DGP218, respectively, with the <em><a href="http://www.yeastgenome.org/locus/S000001747/overview">GAP1</a></em> locus deleted from start to stop with the KanMX4 cassette.</p><p id="p-11">For all experiments, a single colony for each strain was inoculated in synthetic complete medium without uracil, to maintain selection of the plasmids. Studies of the <em><a href="http://www.yeastgenome.org/locus/S000001747/overview">GAP1</a></em> transcript were performed in nitrogen-limiting medium, with proline as the limiting nitrogen source, as described in <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hong and Gresham (2014)</a>. Saturated cultures from overnight cultures were back-diluted 1:50 into medium of the same composition. Cells were allowed to grow for 5.5 hr (∼2.5 doublings) before transcription was inhibited with doxycycline at a final concentration of 10 μg/ml. Cells were collected by filtration and snap frozen in liquid nitrogen.</p></div><div id="sec-4" class="subsection"><h3>RNA processing and qRT-PCR analysis</h3><p id="p-12">RNA was extracted using the hot acid phenol-chloroform method as in <a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Neymotin <em>et al.</em> (2014)</a>. Purified RNA was then treated with RQ1 DNAse according to manufacturer recommendations. Reverse transcription was performed using random hexamers and M-MLV reverse transcriptase. Quantitative Reverse Transcription PCR (qRT-PCR) was performed using the SYBR Green system and a Roche Light Cycler. RNA levels were quantified relative to <em><a href="http://www.yeastgenome.org/locus/S000002633/overview">HTA1</a></em>, a housekeeping gene that is unaffected by doxycycline addition. Ratios were calculated using the formula: <em>Y</em> = 2<sup>[−(HTA1<sub>ct</sub> - Gene<sub><sup>ct</sup></sub>)]</sup> where ct is the calculated cycle threshold. RNA levels from each time point were normalized to <em>t</em> = 0, which was set to 1. All analyses with error bars are the mean ± SE for 3–6 biological replicates. Values without error bars are the average of two replicates. Primer sequences had amplification efficiencies of at least 95% on RT products. The amplified products for all GFP strains begin between positions 463 and 586 of the transcripts and were between 80 and 120 bp long. The sequences used for qRT-PCR analysis are as follows: <em><a href="http://www.yeastgenome.org/locus/S000002633/overview">HTA1</a></em> (forward: 5′-GCTGGTAATGCTGCTAGGGATA-3′, reverse: 5′-TTACCCAATAGCTTGTTCAATT-3′), <em>GFP2</em> and <em>GFP4</em> (forward: 5′-TTGCCGGATAACCACTACCT-3′, reverse: 5′-CCTGCTGCAGTCACAAACTC-3′), <em>GFP3</em> (forward: 5′-GCCGATAAGCAGAAGAATGG-3′, reverse: 5′-TGTTGATAATGGTCCGCAAG-3′), GFP1 (forward: 5′-CGACCATTACCAGCAGAACA-3′, reverse: 5′-GGGTCCTTTGACAGAGCAGA-3′), <em>GAP1-ATG</em>, and <em>GAP1-GTG</em> (forward: 5′-TTTGTTCTGTCTTCGTCAC-3′, reverse: 5′-CTCTACGGATTCACTGGCAGCA-3′). Raw data from qRT-PCR experiments are provided in Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032276/-/DC1/TableS3.xlsx">Table S3</a>.</p></div><div id="sec-5" class="subsection"><h3>Multiple regression analysis</h3><p id="p-13">For multiple regression analysis, we used degradation rate constants (min<sup>−1</sup>), which are approximately normally distributed. To minimize the effects of extreme outliers on regression analysis, we removed mRNA degradation rates &gt; 1.5 times the interquartile range, which resulted in the exclusion of &lt; 5% of all genes for most datasets. Degradation rates for most datasets were calculated as ln(2)/<em>t</em><sub>half-life</sub>, except in <a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Munchel <em>et al.</em> (2011)</a> and <a id="xref-ref-36-4" class="xref-bibr" href="#ref-36">Neymotin <em>et al.</em> (2014)</a>, where the effects of dilution as a result of cellular growth was also considered. Transcript counts were from <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Lipson <em>et al.</em> (2009)</a>, and estimated based on an assumption of ∼60,000 mRNA/cell (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Zenklusen <em>et al.</em> 2008</a>). Protein per mRNA was calculated as the values from <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Ghaemmaghami <em>et al.</em> (2003)</a> divided by the values for counts. CAI (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Sharp and Li 1987</a>) and nTE (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Pechmann and Frydman 2013</a>) was calculated for each transcript based on codon frequency tables in the seqinr package in R (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">R Core Team 2013</a>) or in <a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">Pechmann and Frydman (2013)</a>. As many predictors appear log-normal, we log-transformed predictor variables [log<sub>10</sub>(Variable) or log<sub>10</sub>(Variable + 1)], except for GC content of each codon position and nTE, which are approximately normally distributed, and <strong>Δ</strong><em>G</em>, which is negative in value. For categorical variables, transcripts were classified as bound by an RNA-binding protein based on data from <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Hogan <em>et al.</em> (2008)</a>, and functionally annotated based on GO SLIM files downloaded from the Saccharomyces Genome Database (SGD; <a href="http://www.yeastgenome.org/">http://www.yeastgenome.org/</a>).</p><p id="p-14">In a linear multiple regression model, a parameter of interest is modeled as being dependent on two or more predictors. We consider the degradation rate constant as the modeled variable, and all genome-wide measurements as predictors. To build the model, we followed two separate approaches. In the first approach, we first determined the <em>P</em>-value of the pairwise correlation of each predictor with degradation rate. This indicates whether the regression coefficient is significantly different from zero, and whether or not the predictor has any affect on degradation rate. Next we included all predictors that have a <em>P</em>-value &lt; 0.05 into the multiple regression models. We then performed stepwise removal of terms by removing sequentially the predictors with the highest <em>P</em>-value. The final model is then the reduced model, where only significant terms remained. We obtained the same result using the step function in R, which reduces models based on Akaike’s Information Criterion (AIC). In a second approach, we first calculated the significance of each predictor when it is the only one in the model, as above. We then started adding to the model based on the predictors with the lowest <em>P</em>-value. With each additional term, we checked that all of the terms in the model remained significant. If a new term was added and it was not significant, we removed it from the model. If a new term was added and a different existing predictor was no longer significant, we tested models containing either the new predictor or the one that was no longer significant, and retained the one that explained more variation. We did not add terms that were insignificant in the pairwise correlation with degradation rate. Both approaches gave similar results. Model diagnostics suggest that there is no obvious curvature or patterns in terms of increase or decrease in variance as a function of fitted values (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032276/-/DC1/FigureS6.pdf">Figure S6</a>). There is also minimal curvature in the normal Q-Q plot, suggesting the model follows linearity (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032276/-/DC1/FigureS6.pdf">Figure S6</a>).</p></div><div id="sec-6" class="subsection"><h3>Data availability</h3><p id="p-15">We performed all analyses using R (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">R Core Team 2013</a>). In addition to custom written functions in R, we also used functions from the following packages: <em>TeachingDemos</em>, <em>Biostrings</em>, <em>LSD</em>, <em>stringi</em>, <em>GeneRfold</em>, and <em>seqinr</em>. The complete dataset is available as an R workspace, along with the code for performing the analysis and generating all figures at the Open Science Framework: <a href="https://osf.io/kf4u5/">https://osf.io/kf4u5/</a> (doi:<a href="http://dx.doi.org/10.17605/OSF.IO/KF4U5">10.17605/OSF.IO/KF4U5</a>).</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><div id="sec-8" class="subsection"><h3>Multiple transcript properties affect global mRNA degradation rates</h3><p id="p-16">Previous studies have found evidence for the effect of specific properties of transcripts on the degradation rate of individual transcripts (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Shaw and Kamen 1986</a>; <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Parker and Jacobson 1990</a>; <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Muhlrad and Parker 1992</a>; <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Olivas and Parker 2000</a>). We tested the relationship between globally measured transcript features (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>) and genome-wide mRNA degradation rates determined using RATE-seq (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032276/-/DC1/TableS1.xlsx">Table S1</a>). We found that several transcript features are significantly correlated with mRNA degradation rates (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032276/-/DC1/TableS2.xlsx">Table S2</a>). The most significant single feature predictive of mRNA degradation rates measured using RATE-seq was the length of the coding sequence, which explains almost 30% of the variance. The folding energy (<strong>Δ</strong><em>G</em>) is also significantly associated with mRNA degradation rates, which may be due to the fact that folding energy and coding sequence length are highly correlated. Several features related to the translation of transcripts were also significantly associated with mRNA degradation rates, including ribosome density, the CAI, nTE, and the GC3 content. We also tested whether the function of the encoded product is predictive of mRNA degradation rate, and found that functional assignment using gene ontology (GO) terms explains a significant fraction of the variation (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>). This is consistent with the observation that transcripts encoding proteins in similar functional categories degrade with similar kinetics (<a id="xref-ref-52-4" class="xref-bibr" href="#ref-52">Wang <em>et al.</em> 2002</a>; <a id="xref-ref-36-5" class="xref-bibr" href="#ref-36">Neymotin <em>et al.</em> 2014</a>; <a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Yang <em>et al.</em> 2003</a>). In addition, association with specific mRNA-binding proteins also explains a significant fraction of variation in mRNA degradation rates (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1A</a>). These results suggest a relationship between several features of transcripts and the rate at which they are degraded. We observed similar relationships between some predictors and mRNA degradation rates that have been measured using other methods (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032276/-/DC1/FigureS1.pdf">Figure S1</a>), suggesting that some of these relationships are reproducible despite the poor agreement in mRNA degradation rates among studies.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469750/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469750/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Genome-wide parameters included in model testing</span><div class="sb-div caption-clear"></div></div></div><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/11/3475/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Multiple factors are associated with variation in mRNA degradation rates. (A) Individual predictors explain different amounts of the variation in mRNA degradation rates determined using RATE-seq. Multiple regression models including continuous variables (model), and continuous and categorical variables (model++), explain around half the variation in mRNA degradation rates. Positive correlations with mRNA degradation rates are indicated in red, whereas negative correlations are indicated in green. Values corresponding to ANOVA and multiple regression are in gray. Predictors explaining " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1413068476" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Multiple factors are associated with variation in mRNA degradation rates. (A) Individual predictors explain different amounts of the variation in mRNA degradation rates determined using RATE-seq. Multiple regression models including continuous variables (model), and continuous and categorical variables (model++), explain around half the variation in mRNA degradation rates. Positive correlations with mRNA degradation rates are indicated in red, whereas negative correlations are indicated in green. Values corresponding to ANOVA and multiple regression are in gray. Predictors explaining &lt;&#xA0;1% of the variation are not shown, but are retained in the final model. (B) A comparison of modeled mRNA degradation rates with measured mRNA degradation rates shows that the model behaves similarly across the entire range of mRNA degradation rates. (C) Multiple regression models applied to other published mRNA degradation rates explain different amounts of the variation. Less variance can be explained for mRNA degradation rates that rely on transcriptional inhibition, with the exception of Presnyak &lt;em&gt;et al.&lt;/em&gt; (2015). Labeling studies using thiouracil or thiouridine are indicated in parentheses with 4tU and 4sU, respectively. GRO, genomic run on; DTA, dynamic transcriptome analysis.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/11/3475/F1.medium.gif" width="440" height="140"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/11/3475/F1.medium.gif" width="440" height="140"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/11/3475/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/11/3475/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469642" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-18" class="first-child">Multiple factors are associated with variation in mRNA degradation rates. (A) Individual predictors explain different amounts of the variation in mRNA degradation rates determined using RATE-seq. Multiple regression models including continuous variables (model), and continuous and categorical variables (model++), explain around half the variation in mRNA degradation rates. Positive correlations with mRNA degradation rates are indicated in red, whereas negative correlations are indicated in green. Values corresponding to ANOVA and multiple regression are in gray. Predictors explaining &lt; 1% of the variation are not shown, but are retained in the final model. (B) A comparison of modeled mRNA degradation rates with measured mRNA degradation rates shows that the model behaves similarly across the entire range of mRNA degradation rates. (C) Multiple regression models applied to other published mRNA degradation rates explain different amounts of the variation. Less variance can be explained for mRNA degradation rates that rely on transcriptional inhibition, with the exception of <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Presnyak <em>et al.</em> (2015)</a>. Labeling studies using thiouracil or thiouridine are indicated in parentheses with 4tU and 4sU, respectively. GRO, genomic run on; DTA, dynamic transcriptome analysis.</p><div class="sb-div caption-clear"></div></div></div><p id="p-19">Although many transcript properties are correlated with each other (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032276/-/DC1/FigureS2.pdf">Figure S2</a>), some properties show no correlation and therefore may exert independent and differential effects on the rate of mRNA degradation. Therefore, we used multiple regression analysis to model the contribution of multiple transcript features to variation in mRNA degradation rates simultaneously (<em>Materials and Methods</em>). We initially built a model incorporating all factors, and used sequential reduction to arrive at a minimal model (<em>Materials and Methods</em>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032276/-/DC1/FigureS7.pdf">Figure S7</a>). We find that the explained variation when multiple transcript properties are included exceeds the variance explained by any single factor, suggesting that degradation rates are determined by a combination of transcript features (“model” in <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1A</a>). The final model includes the following covariates: coding length, Δ<em>G</em>, ribosome density, 5′ UTR length, GC3, nTE, and protein half-life. In addition, it includes GC2 and transcription rate, despite the relatively small contribution of each of these predictors (<em>i.e.</em>, individually, they explain &lt; 1% of the variance). When the categorical factors of gene function and association with specific RNA-binding proteins are included, 50% of the variance in mRNA degradation rates can be explained (“model++” in <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1A</a>). Thus, rates predicted by a multiple regression model are in good agreement with experimentally determined rates (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1B</a>). Models incorporating all features explain significant fractions of the variation reported in other mRNA degradation datasets, albeit with reduced explanatory power (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1C</a>). Interestingly, we find that, in general, models applied to mRNA degradation rates measured using transcriptional inhibition tend to explain much less variation than models applied to RNA degradation rates measured with metabolic labeling methods.</p><p id="p-20">Our model suggests a negative relationship between mRNA degradation rates and translation elongation rates, as estimated by CAI, nTE, and ribosome density. Translation elongation rates are slowed during peptide bond formation for proline residues (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gardin <em>et al.</em> 2014</a>), and particularly when multiple prolines are encoded sequentially. To further investigate the role of translation elongation rates on mRNA degradation, we classified transcripts based on presence or absence of at least four sequential proline codons. Consistent with our multiple regression model, transcripts rich in proline degrade more rapidly than the rest of the transcriptome (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). Interestingly, stretches of proline codons are also associated with overall lower protein expression levels (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/11/3475/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The presence of multiple proline codons affects degradation rates and protein production. (A) Box plots of the mRNA degradation rates of proline-rich and proline-poor proteins. Transcripts that contain sequential proline codons, which slow translation elongation rates, tend to be less stable than other transcripts (Wilcoxon rank sum test, P&#xA0;=&#xA0;7.5&#xA0;&#xD7;&#xA0;10&#x2212;32). (B) The abundance of poly-proline containing proteins is reduced compared to the global distribution of protein abundances." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1413068476" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The presence of multiple proline codons affects degradation rates and protein production. (A) Box plots of the mRNA degradation rates of proline-rich and proline-poor proteins. Transcripts that contain sequential proline codons, which slow translation elongation rates, tend to be less stable than other transcripts (Wilcoxon rank sum test, &lt;em&gt;P&lt;/em&gt;&#xA0;=&#xA0;7.5&#xA0;&#xD7;&#xA0;10&lt;sup&gt;&#x2212;32&lt;/sup&gt;). (B) The abundance of poly-proline containing proteins is reduced compared to the global distribution of protein abundances.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/11/3475/F2.medium.gif" width="440" height="185"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/11/3475/F2.medium.gif" width="440" height="185"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/11/3475/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/11/3475/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469632" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-21" class="first-child">The presence of multiple proline codons affects degradation rates and protein production. (A) Box plots of the mRNA degradation rates of proline-rich and proline-poor proteins. Transcripts that contain sequential proline codons, which slow translation elongation rates, tend to be less stable than other transcripts (Wilcoxon rank sum test, <em>P</em> = 7.5 × 10<sup>−32</sup>). (B) The abundance of poly-proline containing proteins is reduced compared to the global distribution of protein abundances.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h3>Ribosome density affects mRNA degradation rate</h3><p id="p-22">Our computational analysis suggests that different aspects of translation affect mRNA degradation rates. Ribosomes are generally thought to protect mRNAs from degradation (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Parker 2012</a>). Consistent with previous analyses (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Edri and Tuller 2014</a>), we find that increased ribosome density is associated with decreased rates of mRNA degradation (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032276/-/DC1/FigureS3.pdf">Figure S3</a>). To experimentally reduce the density of ribosomes on specific transcripts, we mutated the start codon of an endogenous transcript, <em><a href="http://www.yeastgenome.org/locus/S000001747/overview">GAP1</a></em>, which encodes the general amino acid permease, from ATG to GTG, and placed it on a low-copy plasmid under control of doxycycline-repressible promoters (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Garí <em>et al.</em> 1997</a>). Addition of doxycycline has little affect on cellular physiology, and no detectable affect on global gene expression (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Wishart <em>et al.</em> 2005</a>). ATG start codons are required for the small ribosomal subunit to recruit the large ribosomal subunit for fully formed ribosomes. Mutation of the <em><a href="http://www.yeastgenome.org/locus/S000001747/overview">GAP1</a></em> start codon to GTG is expected to reduce the number of ribosomes bound to <em><a href="http://www.yeastgenome.org/locus/S000001747/overview">GAP1</a></em> mRNAs as translation initiation at GTG occurs with a frequency of ∼5% compared to ATG start codons (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Kolitz <em>et al.</em> 2009</a>).</p><p id="p-23">We tested the <em><a href="http://www.yeastgenome.org/locus/S000001747/overview">GAP1</a></em> transcripts for alteration in degradation kinetics as a function of start codon mutation. In the absence of the wildtype start codon, we find that the <em><a href="http://www.yeastgenome.org/locus/S000001747/overview">GAP1</a></em> transcript is decreased significantly in steady-state mRNA abundance (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>), and that the transcript degrades more rapidly upon addition of doxycycline to repress transcription initiation (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3, B and C</a>). This suggests that a decrease in ribosome density results in an increase in the degradation rate of the <em><a href="http://www.yeastgenome.org/locus/S000001747/overview">GAP1</a></em> transcript, consistent with the global trend identified in our computational analysis.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/11/3475/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The effect of ribosome density on GAP1 mRNA degradation rate. (A) Mutation of the wild-type start codon in GAP1 from ATG to GTG results in (A) a reduced steady state transcript level, and (B) an increased mRNA degradation rate. In (B) we show the average value for each time point &#xB1;&#xA0;SEM. In (C) we show bootstrapped CI for the regression of all data points. Solid lines indicate the line of best fit, and dotted lines indicate CI. In blue is the transcript with a wildtype start codon. In red is the transcript with a mutated start codon." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1413068476" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The effect of ribosome density on &lt;em&gt;GAP1&lt;/em&gt; mRNA degradation rate. (A) Mutation of the wild-type start codon in GAP1 from ATG to GTG results in (A) a reduced steady state transcript level, and (B) an increased mRNA degradation rate. In (B) we show the average value for each time point &#xB1;&#xA0;SEM. In (C) we show bootstrapped CI for the regression of all data points. Solid lines indicate the line of best fit, and dotted lines indicate CI. In blue is the transcript with a wildtype start codon. In red is the transcript with a mutated start codon.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/11/3475/F3.medium.gif" width="440" height="129"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/11/3475/F3.medium.gif" width="440" height="129"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/11/3475/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/11/3475/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469577" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-24" class="first-child">The effect of ribosome density on <em>GAP1</em> mRNA degradation rate. (A) Mutation of the wild-type start codon in GAP1 from ATG to GTG results in (A) a reduced steady state transcript level, and (B) an increased mRNA degradation rate. In (B) we show the average value for each time point ± SEM. In (C) we show bootstrapped CI for the regression of all data points. Solid lines indicate the line of best fit, and dotted lines indicate CI. In blue is the transcript with a wildtype start codon. In red is the transcript with a mutated start codon.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Decreased GC content of the third codon position increases the rate of mRNA degradation</h3><p id="p-25">Our multiple regression model predicts that factors involved in translation, including ribosome density, CAI, nTE, and the GC3 content, contribute to variation in mRNA degradation rates (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1A</a>). GC3 content has been reported to affect mRNA abundance and mRNA degradation in <em>Escherichia. coli</em> (<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Kudla <em>et al.</em> 2009</a>). In mammalian cells, GC3 content was also found to affect mRNA levels, but not degradation rates (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Kudla <em>et al.</em> 2006</a>), implying that mRNA synthesis or processing must underlie differences in mRNA levels. However, a more recent genome-wide study found evidence that decreased GC3 content is correlated with increased mRNA degradation rates (<a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Duan <em>et al.</em> 2013</a>).</p><p id="p-26">To study the contribution of GC3 content to variation in mRNA degradation rates, we used GFP constructs that differ in sequence at synonymous sites only (<a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">Kudla <em>et al.</em> 2009</a>). We studied four GFP transcripts that span a range of GC3 content from 38 to 71% (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). Changes in GC3 content also result in overall variation in total GC content (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A</a>). Coding sequences were placed under control of the identical doxycycline-regulated promoter on low copy plasmids, and engineered to have the same UTRs. We confirmed that all four constructs result in functional GFP expression (data not shown).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/11/3475/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="GC3 content affects degradation kinetics and steady state levels. (A) Four GFP transcripts containing synonymous mutations span a range of GC3 content (left) and overall GC (right). Nucleotide-level alignment of each GFP-encoding transcript relative to GFP2. Positions of similarity in sequence are depicted by a gray line, and sites of synonymous mutations are indicated with colored triangles. (B) Differences in GC3 content affect steady state levels of transcripts. (C, D) GFP2 (red), GFP3 (yellow), and GFP4 (blue) degrade in a GC3-dependent manner. In (C) we show the average value for each time point &#xB1;&#xA0;SEM. In (D) we show bootstrapped 95% confidence intervals for the regression of all data points. Solid lines indicate the line of best fit, and dotted lines indicate confidence intervals." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1413068476" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;GC3 content affects degradation kinetics and steady state levels. (A) Four GFP transcripts containing synonymous mutations span a range of GC3 content (left) and overall GC (right). Nucleotide-level alignment of each GFP-encoding transcript relative to GFP2. Positions of similarity in sequence are depicted by a gray line, and sites of synonymous mutations are indicated with colored triangles. (B) Differences in GC3 content affect steady state levels of transcripts. (C, D) GFP2 (red), GFP3 (yellow), and GFP4 (blue) degrade in a GC3-dependent manner. In (C) we show the average value for each time point &#xB1;&#xA0;SEM. In (D) we show bootstrapped 95% confidence intervals for the regression of all data points. Solid lines indicate the line of best fit, and dotted lines indicate confidence intervals.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/11/3475/F4.medium.gif" width="440" height="366"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/11/3475/F4.medium.gif" width="440" height="366"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/11/3475/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/11/3475/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469640" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-27" class="first-child">GC3 content affects degradation kinetics and steady state levels. (A) Four GFP transcripts containing synonymous mutations span a range of GC3 content (left) and overall GC (right). Nucleotide-level alignment of each GFP-encoding transcript relative to GFP2. Positions of similarity in sequence are depicted by a gray line, and sites of synonymous mutations are indicated with colored triangles. (B) Differences in GC3 content affect steady state levels of transcripts. (C, D) GFP2 (red), GFP3 (yellow), and GFP4 (blue) degrade in a GC3-dependent manner. In (C) we show the average value for each time point ± SEM. In (D) we show bootstrapped 95% confidence intervals for the regression of all data points. Solid lines indicate the line of best fit, and dotted lines indicate confidence intervals.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">As all coding sequences are expressed from an identical promoter, differences in steady-state mRNA abundance must result from differences in rates of degradation, synthesis, or processing. We find that steady-state mRNA levels vary with GC3 content, with the highest GC3 content resulting in the highest steady-state mRNA abundance (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4B</a>), consistent with observations in mammalian cells (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Kudla <em>et al.</em> 2006</a>). Following addition of doxycycline to repress transcription initiation, we confirmed that three of the four GFP-encoding transcripts degrade differentially in a GC3-dependent manner (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4, C and D</a>), consistent with our multiple regression prediction. Using the measured steady-state abundances and degradation rates for the three transcripts, we estimated synthesis rates. All three strains have similar estimated rates of synthesis, consistent with differences in degradation rates being the primary determinant of differences in steady-state mRNA levels. We find that a fourth construct, which has much lower GC3 content, and the lowest steady-state abundance, does not significantly differ in its mRNA degradation rate from the second lowest GC3 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032276/-/DC1/FigureS4.pdf">Figure S4</a>). This may reflect a limitation of the sensitivity of our assay, or the fact that other factors are likely to interact with the effect of GC3 content.</p></div></div><div class="section discussion" id="sec-11"><h2 class="">Discussion</h2><p id="p-29">The abundance of mRNAs is determined by both the rate at which they are synthesized and the rate at which they are degraded. In this study, we sought to construct a comprehensive model that predicts genome-wide variation in mRNA degradation rates. Using multiple regression analysis, we find that 43% of the variation in mRNA degradation rates determined using RATE-seq can be explained by considering multiple properties of transcripts in a single model. By including association with specific RNA-binding proteins and the function of the encoded product, ∼50% of the genome-wide variation in mRNA degradation rates can be explained. Interestingly, we find that, in general, methods for measuring RNA degradation that use transcriptional inhibition tend to explain less variation than less disruptive methods. This may reflect that fact that metabolic labeling methods, which perturb the cell only minimally, yield more physiologically relevant degradation rates than transcriptional inhibition, which results in cell death.</p><p id="p-30">In our analysis of mRNA degradation rates measured using RATE-seq, coding sequence length is the strongest single predictor of mRNA degradation rates: in general, the longer a transcript, the more rapidly it is degraded. Other genome-wide investigations have shown a positive relationship between the length of the mature mRNA and its rate of degradation (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Feng and Niu 2007</a>; <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Dressaire <em>et al.</em> 2013</a>; <a id="xref-ref-10-5" class="xref-bibr" href="#ref-10">Duan <em>et al.</em> 2013</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Geisberg <em>et al.</em> 2014</a>). In our original study using RATE-seq, we showed that estimates of mRNA degradation rates are not biased by transcript length (<a id="xref-ref-36-6" class="xref-bibr" href="#ref-36">Neymotin <em>et al.</em> 2014</a>). In the related DTA method of <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Miller <em>et al.</em>(2011)</a>, coding length is regressed out of estimates of mRNA degradation, and, therefore, the effect of coding length is no longer apparent in the resulting data. Studies of individual transcripts have shown that increasing transcript length by addition of specific sequences containing “instability elements” enhances a transcript’s rate of degradation (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Caponigro <em>et al.</em> 1993</a>). Therefore, it is possible that it is not the length of the transcript that affects its degradation, but the presence of additional regulatory elements, to which <em>trans</em> factors such as RNA-binding proteins can bind, promoting their degradation.</p><div id="sec-12" class="subsection"><h3>Association with ribosomes increases mRNA stability</h3><p id="p-31">Our multiple regression analysis provided evidence that protein translation impacts mRNA degradation rates. Previous studies regarding the role of translation initiation rates on mRNA degradation have shown differing results. In a detailed study of the <em><a href="http://www.yeastgenome.org/locus/S000003809/overview">CYC1</a></em> transcript, all start codons were removed from its coding region, thereby preventing ribosome binding and translation (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Yun and Sherman 1996</a>). Following global transcriptional inhibition with thiolutin, the untranslated transcript degraded with similar kinetics to the translated transcript. Similarly, when translation of the <em><a href="http://www.yeastgenome.org/locus/S000005089/overview">MFA2</a></em> transcript was inhibited by introducing a strong secondary structure in the 5′ region of the transcript, it was not found to alter the degradation kinetics following transcriptional inhibition using the GAL system (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Beelman and Parker 1994</a>). By contrast, using the same method to reduce translation of the <em><a href="http://www.yeastgenome.org/locus/S000000605/overview">PGK1</a></em> transcript results in an increased mRNA degradation rate (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Muhlrad <em>et al.</em> 1995</a>). Consistent with a recent report (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Edri and Tuller 2014</a>), our analysis shows that genome-wide increased association with translation machinery, as measured by ribosome density, is correlated with decreased mRNA degradation rates (<a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032276/-/DC1/FigureS3.pdf">Figure S3</a>).</p><p id="p-32">To validate experimentally the effect of translation initiation rates and ribosome density on the rate of mRNA degradation, we mutated the start codon of <em><a href="http://www.yeastgenome.org/locus/S000001747/overview">GAP1</a></em> from ATG to GTG. Consistent with results from our genome-wide study, loss of the wild-type start codon results in reduced mRNA levels, and an increased rate of mRNA degradation. Using a functional assay for growth inhibition in the presence of <span class="sc">d</span>-amino-acids, which are toxic to yeast cells and transported by <a href="http://www.yeastgenome.org/locus/S000001747/overview">GAP1</a>, we found that mutation of the start codon to GTG does not result in a complete loss of protein function (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032276/-/DC1/FigureS5.pdf">Figure S5</a>). This may be attributable to sufficient full-length protein expression from translation initiated at the GTG start codon to confer <span class="sc">d</span>-amino-acid sensitivity. In addition, a potential inframe start codon is located 288 nucleotides (96 amino acids) downstream of the wild-type start codon, and its use may result in a protein product that retains much of the wild-type <a href="http://www.yeastgenome.org/locus/S000001747/overview">GAP1</a> activity although the first 96 amino acids of <a href="http://www.yeastgenome.org/locus/S000001747/overview">GAP1</a> are known to affect the functionality and localization of the encoded permease (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Merhi <em>et al.</em> 2011</a>). An out-of-frame ORF between codons 110 and 187 could potentially promote nonsense mediated decay (NMD) of mutant transcripts. Therefore, in addition to a decrease in ribosome density, we cannot exclude the possibility that alterations in protein function and/or NMD contribute to the decreased stability of the mutant transcript.</p><p id="p-33">We searched for common transcript features to understand why a subset of the transcripts (∼4%) fit the model poorly (&gt;2 SD greater than the mean residual value) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032276/-/DC1/FigureS8.pdf">Figure S8</a>). We find that the outlier transcripts, which are degraded more rapidly than expected, are shorter, have lower ribosome density, and are less translationally efficient (based on normalized translational efficiency score). Thus, although shorter transcripts tend to be more stable, the properties of these outliers suggest that reduced translation of short transcripts results in faster degradation, consistent with our overall model.</p></div><div id="sec-13" class="subsection"><h3>Synonymous coding mutations affect mRNA degradation rates</h3><p id="p-34">Regression analysis suggested genome-wide relationships between codon usage and mRNA degradation rates. We find a negative correlation between two different measures of biased codon usage, CAI and nTE, and mRNA degradation rate. Bias toward more frequent codons in a transcript may increase the elongation rate during translation (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Plotkin and Kudla 2011</a>). Therefore, this negative correlation suggests that faster elongation by ribosomes may result in decreased rates of mRNA degradation. This observation is consistent with the recent report by <a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Presnyak <em>et al.</em> (2015)</a>, who showed that increased occurrence of optimized codons, as defined by nTE, in transcripts, results in decreased degradation rates. We note that, although the effect of nTE on mRNA degradation rates is significant, it explains only ∼7% of the genome-wide variance in both RATE-seq data and the data generated by <a id="xref-ref-44-4" class="xref-bibr" href="#ref-44">Presnyak <em>et al.</em> (2015)</a>, who used a RNA polymerase II temperature sensitive mutant for transcriptional shut-off (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032276/-/DC1/TableS2.xlsx">Table S2</a>). We also find evidence that the presence of multiple sequential proline codons, which slows translation elongation, results in faster mRNA degradation, further supporting an effect of translation elongation rates on mRNA degradation rates.</p><p id="p-35">By studying genome-wide degradation rates, we find evidence that the GC3 content is correlated negatively with rates of mRNA degradation. Despite the comparatively small amount of variance explained by GC3, the effect is significant and consistent with an earlier study that found a positive correlation between synonymous A|T dinucleotides spanning codon boundaries and mRNA degradation rates (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Carlini 2005</a>). To validate this effect experimentally, we tested the effect of GC3 content on mRNA degradation using GFP-encoding transcripts that differ in GC3 content. Consistent with our genome-wide analysis, we found that decreasing the GC3 content resulted in increased mRNA degradation rates, and lowered steady-state abundances. We surmise that the effect is not related to overall GC content of transcripts, as the GC content of the first (GC1) and second (GC2) position in codons explains much less variance. However, the effect of GC3 on mRNA degradation rate appears to be independent of the effect of codon bias, as the four GFP transcripts investigated do not show a systematic difference in CAI or nTE. Recently, increased GC3 content has been suggested to decrease mRNA degradation rates in human lymphoblastoid cells, and possibly explain variation in mRNA degradation rates between individuals (<a id="xref-ref-10-6" class="xref-bibr" href="#ref-10">Duan <em>et al.</em> 2013</a>). Thus, the relationship between GC3 content and mRNA degradation rates may be widely conserved in eukaryotes.</p></div><div id="sec-14" class="subsection"><h3>Conclusion</h3><p id="p-36">Our study shows that genome-wide variation in mRNA degradation rates is best explained by a combination of different transcript features, as suggested more than two decades ago (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Caponigro <em>et al.</em> 1993</a>). Many of the factors that affect genome-wide patterns of mRNA degradation rates are related to protein production, highlighting the close relationship between mRNA degradation and translation. Our statistical analysis and experimental validation confirm the effect of ribosome density and GC3 content on mRNA degradation rates. Whereas mutation of a start codon likely changes translation initiation rates, modifying GC3 within a transcript is likely to alter translation elongation rates. Thus, both translation initiation and elongation rates may impact mRNA decay rates, but perhaps through different degradation pathways.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-37">We thank Joshua Plotkin and Grzegorz Kudla for providing the degenerate GFP constructs for our studies. We thank members of the Gresham Lab for helpful discussions and suggestions. This work was supported by the National Institutes of Health (GM107466), the National Science Foundation (MCB-1244219), and a DuPont Young Professor award.</p><p id="p-38">Author contributions: B.N. performed the computational analysis and biological validation of GC content. V.E. performed biological validation of ribosome density. B.N. and D.G. conceived of the project and wrote the article.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-39">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032276/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032276/-/DC1</a>.</p></li><li class="fn" id="fn-3"><p id="p-40"><em>Communicating editor: Timothy R. Hughes</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-06-09"><span class="received-label">Received </span>June 9, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-08-20"><span class="accepted-label">Accepted </span>August 20, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Neymotin <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.1" data-doi="10.1093/emboj/17.5.1497"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parker</span>  <span class="cit-name-given-names">R. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">The 3′ to 5′ degradation of yeast mRNAs is a general mechanism for mRNA turnover that requires the SKI2 DEVH box protein and 3′ to 5′ exonucleases of the exosome complex.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1497</span>–<span class="cit-lpage">1506</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DAnderson%26rft.auinit1%253DJ.%2BS.%2BJ.%26rft.volume%253D17%26rft.issue%253D5%26rft.spage%253D1497%26rft.epage%253D1506%26rft.atitle%253DThe%2B3%2527%2Bto%2B5%2527%2Bdegradation%2Bof%2Byeast%2BmRNAs%2Bis%2Ba%2Bgeneral%2Bmechanism%2Bfor%2BmRNA%2Bturnover%2Bthat%2Brequires%2Bthe%2BSKI2%2BDEVH%2Bbox%2Bprotein%2Band%2B3%2527%2Bto%2B5%2527%2Bexonucleases%2Bof%2Bthe%2Bexosome%2Bcomplex%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252F17.5.1497%26rft_id%253Dinfo%253Apmid%252F9482746%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czo5OiIxNy81LzE0OTciO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzExLzM0NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beelman</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parker</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Differential effects of translational inhibition in cis and in trans on the decay of the unstable yeast MFA2 mRNA.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">269</span>: <span class="cit-fpage">9687</span>–<span class="cit-lpage">9692</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DBeelman%26rft.auinit1%253DC.%2BA.%26rft.volume%253D269%26rft.issue%253D13%26rft.spage%253D9687%26rft.epage%253D9692%26rft.atitle%253DDifferential%2Beffects%2Bof%2Btranslational%2Binhibition%2Bin%2Bcis%2Band%2Bin%2Btrans%2Bon%2Bthe%2Bdecay%2Bof%2Bthe%2Bunstable%2Byeast%2BMFA2%2BmRNA.%26rft_id%253Dinfo%253Apmid%252F8144558%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjExOiIyNjkvMTMvOTY4NyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzQ3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.3" data-doi="10.1038/382642a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beelman</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caponigro</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">LaGrandeur</span>  <span class="cit-name-given-names">T. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hatfield</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">An essential component of the decapping enzyme required for normal rates of mRNA turnover.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">382</span>: <span class="cit-fpage">642</span>–<span class="cit-lpage">646</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBeelman%26rft.auinit1%253DC.%2BA.%26rft.volume%253D382%26rft.issue%253D6592%26rft.spage%253D642%26rft.epage%253D646%26rft.atitle%253DAn%2Bessential%2Bcomponent%2Bof%2Bthe%2Bdecapping%2Benzyme%2Brequired%2Bfor%2Bnormal%2Brates%2Bof%2BmRNA%2Bturnover.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F382642a0%26rft_id%253Dinfo%253Apmid%252F8757137%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/382642a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8757137&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996VC30300057&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.4" data-doi="10.1073/pnas.0605420103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Belle</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanay</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bitincka</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shamir</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Shea</span>  <span class="cit-name-given-names">E. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Quantification of protein half-lives in the budding yeast proteome.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">13004</span>–<span class="cit-lpage">13009</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0605420103%26rft_id%253Dinfo%253Apmid%252F16916930%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAzLzM1LzEzMDA0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNDc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.5" data-doi="10.1128/MCB.13.9.5141"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Caponigro</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muhlrad</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parker</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">A small segment of the MAT alpha 1 transcript promotes mRNA decay in <em>Saccharomyces cerevisiae</em>: a stimulatory role for rare codons.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">5141</span>–<span class="cit-lpage">5148</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DCaponigro%26rft.auinit1%253DG.%26rft.volume%253D13%26rft.issue%253D9%26rft.spage%253D5141%26rft.epage%253D5148%26rft.atitle%253DA%2Bsmall%2Bsegment%2Bof%2Bthe%2BMAT%2Balpha%2B1%2Btranscript%2Bpromotes%2BmRNA%2Bdecay%2Bin%2BSaccharomyces%2Bcerevisiae%253A%2Ba%2Bstimulatory%2Brole%2Bfor%2Brare%2Bcodons.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.13.9.5141%26rft_id%253Dinfo%253Apmid%252F8355674%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjEzLzkvNTE0MSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzQ3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.6" data-doi="10.1093/molbev/msi135"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carlini</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Context-dependent codon bias and messenger RNA longevity in the yeast transcriptome.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1403</span>–<span class="cit-lpage">1411</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsi135%26rft_id%253Dinfo%253Apmid%252F15772378%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjIyLzYvMTQwMyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzQ3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.7" data-doi="10.1016/S0092-8674(01)00578-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gherzi</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ong</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raijmakers</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">AU binding proteins recruit the exosome to degrade ARE-containing mRNAs.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">451</span>–<span class="cit-lpage">464</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DChen%26rft.auinit1%253DC.%2BY.%26rft.volume%253D107%26rft.issue%253D4%26rft.spage%253D451%26rft.epage%253D464%26rft.atitle%253DAU%2Bbinding%2Bproteins%2Brecruit%2Bthe%2Bexosome%2Bto%2Bdegrade%2BARE-containing%2BmRNAs.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252801%252900578-5%26rft_id%253Dinfo%253Apmid%252F11719186%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(01)00578-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11719186&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000172272900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.8" data-doi="10.1101/gad.7.8.1632"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Decker</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parker</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">A turnover pathway for both stable and unstable mRNAs in yeast: evidence for a requirement for deadenylation.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1632</span>–<span class="cit-lpage">1643</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DDecker%26rft.auinit1%253DC%2BJ%26rft.volume%253D7%26rft.issue%253D8%26rft.spage%253D1632%26rft.epage%253D1643%26rft.atitle%253DA%2Bturnover%2Bpathway%2Bfor%2Bboth%2Bstable%2Band%2Bunstable%2BmRNAs%2Bin%2Byeast%253A%2Bevidence%2Bfor%2Ba%2Brequirement%2Bfor%2Bdeadenylation.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.7.8.1632%26rft_id%253Dinfo%253Apmid%252F8393418%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6ODoiNy84LzE2MzIiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzExLzM0NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.9" data-doi="10.1371/journal.pone.0059059"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dressaire</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Picard</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Redon</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loubière</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Queinnec</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Role of mRNA stability during bacterial adaptation.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e59059</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De59059%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0059059%26rft_id%253Dinfo%253Apmid%252F23516597%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0059059&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23516597&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ge</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dölken</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moy</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome-wide survey of interindividual differences of RNA stability in human lymphoblastoid cell lines.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">1318</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D3%26rft.spage%253D1318%26rft_id%253Dinfo%253Apmid%252F23422947%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23422947&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.11" data-doi="10.1016/j.gene.2008.07.033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eck</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephan</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Determining the relationship of gene expression and global mRNA stability in <em>Drosophila melanogaster</em> and <em>Escherichia coli</em> using linear models.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">424</span>: <span class="cit-fpage">102</span>–<span class="cit-lpage">107</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DEck%26rft.auinit1%253DS.%26rft.volume%253D424%26rft.issue%253D1-2%26rft.spage%253D102%26rft.epage%253D107%26rft.atitle%253DDetermining%2Bthe%2Brelationship%2Bof%2Bgene%2Bexpression%2Band%2Bglobal%2BmRNA%2Bstability%2Bin%2BDrosophila%2Bmelanogaster%2Band%2BEscherichia%2Bcoli%2Busing%2Blinear%2Bmodels.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gene.2008.07.033%26rft_id%253Dinfo%253Apmid%252F18755255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gene.2008.07.033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18755255&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.12" data-doi="10.1371/journal.pone.0102308"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edri</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tuller</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Quantifying the effect of ribosomal density on mRNA stability.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e102308</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De102308%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0102308%26rft_id%253Dinfo%253Apmid%252F25020060%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0102308&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25020060&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.13" data-doi="10.1007/s10528-006-9059-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Niu</span>  <span class="cit-name-given-names">D.-K.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Relationship between mRNA stability and length: an old question with a new twist.</span> <abbr class="cit-jnl-abbrev">Biochem. Genet.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">131</span>–<span class="cit-lpage">137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemical%2Bgenetics%26rft.stitle%253DBiochem%2BGenet%26rft.aulast%253DFeng%26rft.auinit1%253DL.%26rft.volume%253D45%26rft.issue%253D1-2%26rft.spage%253D131%26rft.epage%253D137%26rft.atitle%253DRelationship%2Bbetween%2BmRNA%2Bstability%2Band%2Blength%253A%2Ban%2Bold%2Bquestion%2Bwith%2Ba%2Bnew%2Btwist.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10528-006-9059-5%26rft_id%253Dinfo%253Apmid%252F17221301%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10528-006-9059-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17221301&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.14" data-doi="10.1016/j.molcel.2004.06.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">García-Martínez</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aranda</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Ortín</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Genomic run-on evaluates transcription rates for all yeast genes and identifies gene regulatory mechanisms.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">303</span>–<span class="cit-lpage">313</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DGarc%25C3%25ADa-Mart%25C3%25ADnez%26rft.auinit1%253DJ.%26rft.volume%253D15%26rft.issue%253D2%26rft.spage%253D303%26rft.epage%253D313%26rft.atitle%253DGenomic%2Brun-on%2Bevaluates%2Btranscription%2Brates%2Bfor%2Ball%2Byeast%2Bgenes%2Band%2Bidentifies%2Bgene%2Bregulatory%2Bmechanisms.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2004.06.004%26rft_id%253Dinfo%253Apmid%252F15260981%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2004.06.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15260981&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222958300018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gardin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yeasmin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yurovsky</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cai</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skiena</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Measurement of average decoding rates of the 61 sense codons in vivo.</span> <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">3</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.16" data-doi="10.1002/(SICI)1097-0061(199707)13:9&lt;837::AID-YEA145&gt;3.0.CO;2-T"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garí</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piedrafita</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aldea</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herrero</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">A set of vectors with a tetracycline-regulatable promoter system for modulated gene expression in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">837</span>–<span class="cit-lpage">848</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DGar%25C3%25AD%26rft.auinit1%253DE.%26rft.volume%253D13%26rft.issue%253D9%26rft.spage%253D837%26rft.epage%253D848%26rft.atitle%253DA%2Bset%2Bof%2Bvectors%2Bwith%2Ba%2Btetracycline-regulatable%2Bpromoter%2Bsystem%2Bfor%2Bmodulated%2Bgene%2Bexpression%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F%2528SICI%25291097-0061%2528199707%252913%253A9%253C837%253A%253AAID-YEA145%253E3.0.CO%253B2-T%26rft_id%253Dinfo%253Apmid%252F9234672%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/(SICI)1097-0061(199707)13:9&lt;837::AID-YEA145&gt;3.0.CO;2-T&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9234672&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XL37200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.17" data-doi="10.1016/j.cell.2013.12.026"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Geisberg</span>  <span class="cit-name-given-names">J. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moqtaderi</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ozsolak</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Struhl</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Global analysis of mRNA isoform half-lives reveals stabilizing and destabilizing elements in yeast.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">156</span>: <span class="cit-fpage">812</span>–<span class="cit-lpage">824</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D156%26rft.spage%253D812%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2013.12.026%26rft_id%253Dinfo%253Apmid%252F24529382%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2013.12.026&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24529382&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000331379800018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.18" data-doi="10.1038/nature02046"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ghaemmaghami</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huh</span>  <span class="cit-name-given-names">W.-K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bower</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howson</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Belle</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Global analysis of protein expression in yeast.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">425</span>: <span class="cit-fpage">737</span>–<span class="cit-lpage">741</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGhaemmaghami%26rft.auinit1%253DS.%26rft.volume%253D425%26rft.issue%253D6959%26rft.spage%253D737%26rft.epage%253D741%26rft.atitle%253DGlobal%2Banalysis%2Bof%2Bprotein%2Bexpression%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02046%26rft_id%253Dinfo%253Apmid%252F14562106%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02046&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14562106&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185924500046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.19" data-doi="10.1128/MCB.24.12.5534-5547.2004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grigull</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mnaimneh</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pootoolal</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hughes</span>  <span class="cit-name-given-names">T. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Genome-wide analysis of mRNA stability using transcription inhibitors and microarrays reveals posttranscriptional control of ribosome biogenesis factors.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">5534</span>–<span class="cit-lpage">5547</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DGrigull%26rft.auinit1%253DJ.%26rft.volume%253D24%26rft.issue%253D12%26rft.spage%253D5534%26rft.epage%253D5547%26rft.atitle%253DGenome-Wide%2BAnalysis%2Bof%2BmRNA%2BStability%2BUsing%2BTranscription%2BInhibitors%2Band%2BMicroarrays%2BReveals%2BPosttranscriptional%2BControl%2Bof%2BRibosome%2BBiogenesis%2BFactors%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.24.12.5534-5547.2004%26rft_id%253Dinfo%253Apmid%252F15169913%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyNC8xMi81NTM0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNDc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.20" data-doi="10.1016/j.molcel.2009.09.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hager</span>  <span class="cit-name-given-names">G. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McNally</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misteli</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Transcription dynamics.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">741</span>–<span class="cit-lpage">753</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DHager%26rft.auinit1%253DG.%2BL.%26rft.volume%253D35%26rft.issue%253D6%26rft.spage%253D741%26rft.epage%253D753%26rft.atitle%253DTranscription%2Bdynamics.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2009.09.005%26rft_id%253Dinfo%253Apmid%252F19782025%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2009.09.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19782025&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270559100009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.21" data-doi="10.1371/journal.pbio.0060255"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hogan</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riordan</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerber</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herschlag</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Diverse RNA-binding proteins interact with functionally related sets of RNAs, suggesting an extensive regulatory system.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e255</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DHogan%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D6%26rft.issue%253D10%26rft.spage%253De255%26rft.epage%253De255%26rft.atitle%253DDiverse%2BRNA-binding%2Bproteins%2Binteract%2Bwith%2Bfunctionally%2Brelated%2Bsets%2Bof%2BRNAs%252C%2Bsuggesting%2Ban%2Bextensive%2Bregulatory%2Bsystem.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0060255%26rft_id%253Dinfo%253Apmid%252F18959479%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0060255&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18959479&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.22" data-doi="10.1371/journal.pgen.1004041"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hong</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gresham</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Molecular specificity, convergence and constraint shape adaptive evolution in nutrient-poor environments.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1004041</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004041%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004041%26rft_id%253Dinfo%253Apmid%252F24415948%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004041&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24415948&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.23" data-doi="10.1126/science.1168978"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ingolia</span>  <span class="cit-name-given-names">N. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ghaemmaghami</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Newman</span>  <span class="cit-name-given-names">J. R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weissman</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genome-wide analysis in vivo of translation with nucleotide resolution using ribosome profiling.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">324</span>: <span class="cit-fpage">218</span>–<span class="cit-lpage">223</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1168978%26rft_id%253Dinfo%253Apmid%252F19213877%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjQvNTkyNC8yMTgiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzExLzM0NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.24" data-doi="10.1016/j.cell.2004.12.038"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jing</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guth</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zarubin</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Motoyama</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Involvement of microRNA in AU-rich element-mediated mRNA instability.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">120</span>: <span class="cit-fpage">623</span>–<span class="cit-lpage">634</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DJing%26rft.auinit1%253DQ.%26rft.volume%253D120%26rft.issue%253D5%26rft.spage%253D623%26rft.epage%253D634%26rft.atitle%253DInvolvement%2Bof%2BmicroRNA%2Bin%2BAU-rich%2Belement-mediated%2BmRNA%2Binstability.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2004.12.038%26rft_id%253Dinfo%253Apmid%252F15766526%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2004.12.038&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15766526&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227771400009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.25" data-doi="10.1261/rna.1318509"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kolitz</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takacs</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorsch</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Kinetic and thermodynamic analysis of the role of start codon/anticodon base pairing during eukaryotic translation initiation.</span> <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">138</span>–<span class="cit-lpage">152</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.1318509%26rft_id%253Dinfo%253Apmid%252F19029312%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjg6IjE1LzEvMTM4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNDc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.26" data-doi="10.1371/journal.pbio.0040180"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kudla</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipinski</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caffin</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Helwak</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zylicz</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">High guanine and cytosine content increases mRNA levels in mammalian cells.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e180</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DKudla%26rft.auinit1%253DG.%26rft.volume%253D4%26rft.issue%253D6%26rft.spage%253De180%26rft.epage%253De180%26rft.atitle%253DHigh%2Bguanine%2Band%2Bcytosine%2Bcontent%2Bincreases%2BmRNA%2Blevels%2Bin%2Bmammalian%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0040180%26rft_id%253Dinfo%253Apmid%252F16700628%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0040180&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16700628&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.27" data-doi="10.1126/science.1170160"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kudla</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tollervey</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plotkin</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Coding-sequence determinants of gene expression in <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">324</span>: <span class="cit-fpage">255</span>–<span class="cit-lpage">258</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DKudla%26rft.auinit1%253DG.%26rft.volume%253D324%26rft.issue%253D5924%26rft.spage%253D255%26rft.epage%253D258%26rft.atitle%253DCoding-Sequence%2BDeterminants%2Bof%2BGene%2BExpression%2Bin%2BEscherichia%2Bcoli%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1170160%26rft_id%253Dinfo%253Apmid%252F19359587%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjQvNTkyNC8yNTUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzExLzM0NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.28" data-doi="10.1038/nbt.1551"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lipson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raz</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kieu</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giladi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Quantification of the yeast transcriptome by single-molecule sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">652</span>–<span class="cit-lpage">658</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DLipson%26rft.auinit1%253DD.%26rft.volume%253D27%26rft.issue%253D7%26rft.spage%253D652%26rft.epage%253D658%26rft.atitle%253DQuantification%2Bof%2Bthe%2Byeast%2Btranscriptome%2Bby%2Bsingle-molecule%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1551%26rft_id%253Dinfo%253Apmid%252F19581875%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1551&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19581875&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267908500022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.29" data-doi="10.1371/journal.pone.0018457"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Merhi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gérard</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lauwers</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prévost</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">André</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Systematic mutational analysis of the intracellular regions of yeast Gap1 permease.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e18457</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DMerhi%26rft.auinit1%253DA.%26rft.volume%253D6%26rft.issue%253D4%26rft.spage%253De18457%26rft.epage%253De18457%26rft.atitle%253DSystematic%2Bmutational%2Banalysis%2Bof%2Bthe%2Bintracellular%2Bregions%2Bof%2Byeast%2BGap1%2Bpermease.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0018457%26rft_id%253Dinfo%253Apmid%252F21526172%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0018457&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21526172&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.30" data-doi="10.1038/msb.2010.112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwalb</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maier</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schulz</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dümcke</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Dynamic transcriptome analysis measures rates of mRNA synthesis and decay in yeast.</span> <abbr class="cit-jnl-abbrev">Mol. Syst. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">458</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BSystems%2BBiology%26rft.stitle%253DMol%2BSyst%2BBiol%26rft.aulast%253DMiller%26rft.auinit1%253DC.%26rft.volume%253D7%26rft.issue%253D1%26rft.spage%253D458%26rft.epage%253D458%26rft.atitle%253DDynamic%2Btranscriptome%2Banalysis%2Bmeasures%2Brates%2Bof%2BmRNA%2Bsynthesis%2Band%2Bdecay%2Bin%2Byeast%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fmsb.2010.112%26rft_id%253Dinfo%253Apmid%252F21206491%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibXNiIjtzOjU6InJlc2lkIjtzOjc6IjcvMS80NTgiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzExLzM0NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.31" data-doi="10.1101/gad.6.11.2100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muhlrad</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parker</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Mutations affecting stability and deadenylation of the yeast MFA2 transcript.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">2100</span>–<span class="cit-lpage">2111</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DMuhlrad%26rft.auinit1%253DD%26rft.volume%253D6%26rft.issue%253D11%26rft.spage%253D2100%26rft.epage%253D2111%26rft.atitle%253DMutations%2Baffecting%2Bstability%2Band%2Bdeadenylation%2Bof%2Bthe%2Byeast%2BMFA2%2Btranscript.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.6.11.2100%26rft_id%253Dinfo%253Apmid%252F1427074%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6OToiNi8xMS8yMTAwIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNDc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.32" data-doi="10.1128/MCB.15.4.2145"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muhlrad</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Decker</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parker</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Turnover mechanisms of the stable yeast PGK1 mRNA.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">2145</span>–<span class="cit-lpage">2156</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DMuhlrad%26rft.auinit1%253DD.%26rft.volume%253D15%26rft.issue%253D4%26rft.spage%253D2145%26rft.epage%253D2156%26rft.atitle%253DTurnover%2Bmechanisms%2Bof%2Bthe%2Bstable%2Byeast%2BPGK1%2BmRNA%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.15.4.2145%26rft_id%253Dinfo%253Apmid%252F7891709%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjE1LzQvMjE0NSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzQ3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.33" data-doi="10.1091/mbc.E11-01-0028"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Munchel</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shultzaberger</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takizawa</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weis</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Dynamic profiling of mRNA turnover reveals gene-specific and system-wide regulation of mRNA decay.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">2787</span>–<span class="cit-lpage">2795</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E11-01-0028%26rft_id%253Dinfo%253Apmid%252F21680716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIyMi8xNS8yNzg3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNDc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.34" data-doi="10.1126/science.1158441"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nagalakshmi</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waern</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shou</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raha</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The transcriptional landscape of the yeast genome defined by RNA sequencing.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">320</span>: <span class="cit-fpage">1344</span>–<span class="cit-lpage">1349</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1158441%26rft_id%253Dinfo%253Apmid%252F18451266%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjAvNTg4MS8xMzQ0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNDc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.35" data-doi="10.1105/tpc.107.055046"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Narsai</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howell</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Millar</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Toole</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Small</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genome-wide analysis of mRNA decay rates and their determinants in <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">3418</span>–<span class="cit-lpage">3436</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.107.055046%26rft_id%253Dinfo%253Apmid%252F18024567%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIxOS8xMS8zNDE4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNDc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.36" data-doi="10.1261/rna.045104.114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neymotin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Athanasiadou</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gresham</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Determination of in vivo RNA kinetics using RATE-seq.</span> <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1645</span>–<span class="cit-lpage">1652</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.045104.114%26rft_id%253Dinfo%253Apmid%252F25161313%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjEwOiIyMC8xMC8xNjQ1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNDc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.37" data-doi="10.1128/MCB.7.5.1602"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nonet</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scafe</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sexton</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Eucaryotic RNA polymerase conditional mutant that rapidly ceases mRNA synthesis.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1602</span>–<span class="cit-lpage">1611</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DNonet%26rft.auinit1%253DM%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253D1602%26rft.epage%253D1611%26rft.atitle%253DEucaryotic%2BRNA%2Bpolymerase%2Bconditional%2Bmutant%2Bthat%2Brapidly%2Bceases%2BmRNA%2Bsynthesis.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.7.5.1602%26rft_id%253Dinfo%253Apmid%252F3299050%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjg6IjcvNS8xNjAyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNDc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.38" data-doi="10.1093/emboj/19.23.6602"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Olivas</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parker</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The Puf3 protein is a transcript-specific regulator of mRNA degradation in yeast.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">6602</span>–<span class="cit-lpage">6611</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DOlivas%26rft.auinit1%253DW.%26rft.volume%253D19%26rft.issue%253D23%26rft.spage%253D6602%26rft.epage%253D6611%26rft.atitle%253DThe%2BPuf3%2Bprotein%2Bis%2Ba%2Btranscript-specific%2Bregulator%2Bof%2BmRNA%2Bdegradation%2Bin%2Byeast%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252F19.23.6602%26rft_id%253Dinfo%253Apmid%252F11101532%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMTkvMjMvNjYwMiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzQ3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.39" data-doi="10.1534/genetics.111.137265"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parker</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">RNA degradation in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">191</span>: <span class="cit-fpage">671</span>–<span class="cit-lpage">702</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DParker%26rft.auinit1%253DR.%26rft.volume%253D191%26rft.issue%253D3%26rft.spage%253D671%26rft.epage%253D702%26rft.atitle%253DRNA%2BDegradation%2Bin%2BSaccharomyces%2Bcerevisae%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.137265%26rft_id%253Dinfo%253Apmid%252F22785621%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkxLzMvNjcxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNDc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.40" data-doi="10.1073/pnas.87.7.2780"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parker</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacobson</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Translation and a 42-nucleotide segment within the coding region of the mRNA encoded by the MAT alpha 1 gene are involved in promoting rapid mRNA decay in yeast.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">87</span>: <span class="cit-fpage">2780</span>–<span class="cit-lpage">2784</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DParker%26rft.auinit1%253DR.%26rft.volume%253D87%26rft.issue%253D7%26rft.spage%253D2780%26rft.epage%253D2784%26rft.atitle%253DTranslation%2Band%2Ba%2B42-nucleotide%2Bsegment%2Bwithin%2Bthe%2Bcoding%2Bregion%2Bof%2Bthe%2BmRNA%2Bencoded%2Bby%2Bthe%2BMAT%2Balpha%2B1%2Bgene%2Bare%2Binvolved%2Bin%2Bpromoting%2Brapid%2BmRNA%2Bdecay%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.87.7.2780%26rft_id%253Dinfo%253Apmid%252F2181450%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI4Ny83LzI3ODAiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzExLzM0NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.41" data-doi="10.1038/nsmb.2466"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pechmann</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frydman</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Evolutionary conservation of codon optimality reveals hidden signatures of cotranslational folding.</span> <abbr class="cit-jnl-abbrev">Nat. Struct. Mol. Biol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">237</span>–<span class="cit-lpage">243</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BStruct.%2BMol.%2BBiol.%26rft.volume%253D20%26rft.spage%253D237%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb.2466%26rft_id%253Dinfo%253Apmid%252F23262490%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsmb.2466&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23262490&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.42" data-doi="10.1371/journal.pone.0015442"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pelechano</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chávez</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Ortín</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A complete set of nascent transcription rates for yeast genes.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e15442</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPelechano%26rft.auinit1%253DV.%26rft.volume%253D5%26rft.issue%253D11%26rft.spage%253De15442%26rft.epage%253De15442%26rft.atitle%253DA%2Bcomplete%2Bset%2Bof%2Bnascent%2Btranscription%2Brates%2Bfor%2Byeast%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0015442%26rft_id%253Dinfo%253Apmid%252F21103382%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0015442&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21103382&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.43" data-doi="10.1038/nrg2899"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Plotkin</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kudla</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Synonymous but not the same: the causes and consequences of codon bias.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">32</span>–<span class="cit-lpage">42</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DPlotkin%26rft.auinit1%253DJ.%2BB.%26rft.volume%253D12%26rft.issue%253D1%26rft.spage%253D32%26rft.epage%253D42%26rft.atitle%253DSynonymous%2Bbut%2Bnot%2Bthe%2Bsame%253A%2Bthe%2Bcauses%2Band%2Bconsequences%2Bof%2Bcodon%2Bbias.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2899%26rft_id%253Dinfo%253Apmid%252F21102527%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2899&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21102527&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000285410500009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.44" data-doi="10.1016/j.cell.2015.02.029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Presnyak</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alhusaini</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Y.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morris</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Codon optimality is a major determinant of mRNA stability.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">160</span>: <span class="cit-fpage">1111</span>–<span class="cit-lpage">1124</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D160%26rft.spage%253D1111%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2015.02.029%26rft_id%253Dinfo%253Apmid%252F25768907%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2015.02.029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25768907&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-book" id="cit-6.11.3475.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Core Team</span></li></ol><cite> <span class="cit-pub-date">2013</span> <span class="cit-source">R: A language and environment for statistical computing.</span> <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>. URL <a href="http://www.R-project.org/">http://www.R-project.org/</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.46" data-doi="10.1038/nbt.1861"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rabani</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levin</span>  <span class="cit-name-given-names">J. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adiconis</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raychowdhury</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Metabolic labeling of RNA uncovers principles of RNA production and degradation dynamics in mammalian cells.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">436</span>–<span class="cit-lpage">442</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DRabani%26rft.auinit1%253DM.%26rft.volume%253D29%26rft.issue%253D5%26rft.spage%253D436%26rft.epage%253D442%26rft.atitle%253DMetabolic%2Blabeling%2Bof%2BRNA%2Buncovers%2Bprinciples%2Bof%2BRNA%2Bproduction%2Band%2Bdegradation%2Bdynamics%2Bin%2Bmammalian%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1861%26rft_id%253Dinfo%253Apmid%252F21516085%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1861&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21516085&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.47" data-doi="10.1101/gr.912603"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Selinger</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saxena</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheung</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Church</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosenow</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Global RNA half-life analysis in <em>Escherichia coli</em> reveals positional patterns of transcript degradation.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">216</span>–<span class="cit-lpage">223</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.912603%26rft_id%253Dinfo%253Apmid%252F12566399%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEzLzIvMjE2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNDc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.48" data-doi="10.1038/msb.2008.59"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shalem</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dahan</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martinez</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Furman</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Transient transcriptional responses to stress are generated by opposing effects of mRNA production and degradation.</span> <abbr class="cit-jnl-abbrev">Mol. Syst. Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">223</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BSystems%2BBiology%26rft.stitle%253DMol%2BSyst%2BBiol%26rft.aulast%253DShalem%26rft.auinit1%253DO.%26rft.volume%253D4%26rft.spage%253D223%26rft.epage%253D223%26rft.atitle%253DTransient%2Btranscriptional%2Bresponses%2Bto%2Bstress%2Bare%2Bgenerated%2Bby%2Bopposing%2Beffects%2Bof%2BmRNA%2Bproduction%2Band%2Bdegradation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fmsb.2008.59%26rft_id%253Dinfo%253Apmid%252F18854817%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/msb.2008.59&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18854817&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.49" data-doi="10.1093/nar/15.3.1281"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sharp</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">The Codon Adaptation Index—a measure of directional synonymous codon usage bias, and its potential applications.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1281</span>–<span class="cit-lpage">1295</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DSharp%26rft.auinit1%253DP.%2BM.%26rft.volume%253D15%26rft.issue%253D3%26rft.spage%253D1281%26rft.epage%253D1295%26rft.atitle%253DThe%2Bcodon%2Badaptation%2Bindex-a%2Bmeasure%2Bof%2Bdirectional%2Bsynonymous%2Bcodon%2Busage%2Bbias%252C%2Band%2Bits%2Bpotential%2Bapplications%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F15.3.1281%26rft_id%253Dinfo%253Apmid%252F3547335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjE1LzMvMTI4MSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzQ3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.50" data-doi="10.1016/0092-8674(86)90341-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shaw</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kamen</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">A conserved AU sequence from the 3′ untranslated region of GM-CSF mRNA mediates selective mRNA degradation.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">659</span>–<span class="cit-lpage">667</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DShaw%26rft.auinit1%253DG.%26rft.volume%253D46%26rft.issue%253D5%26rft.spage%253D659%26rft.epage%253D667%26rft.atitle%253DA%2Bconserved%2BAU%2Bsequence%2Bfrom%2Bthe%2B3%2527%2Buntranslated%2Bregion%2Bof%2BGM-CSF%2BmRNA%2Bmediates%2Bselective%2BmRNA%2Bdegradation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252886%252990341-7%26rft_id%253Dinfo%253Apmid%252F3488815%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(86)90341-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3488815&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1986D837600004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.51" data-doi="10.1186/gb-2010-11-9-r93"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thomsen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anders</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Janga</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alonso</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-wide analysis of mRNA decay patterns during early <em>Drosophila</em> development.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">R93</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DThomsen%26rft.auinit1%253DS.%26rft.volume%253D11%26rft.issue%253D9%26rft.spage%253DR93%26rft.epage%253DR93%26rft.atitle%253DGenome-wide%2Banalysis%2Bof%2BmRNA%2Bdecay%2Bpatterns%2Bduring%2Bearly%2BDrosophila%2Bdevelopment.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2010-11-9-r93%26rft_id%253Dinfo%253Apmid%252F20858238%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2010-11-9-r93&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20858238&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.52" data-doi="10.1073/pnas.092538799"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Storey</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tibshirani</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herschlag</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Precision and functional specificity in mRNA decay.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">5860</span>–<span class="cit-lpage">5865</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.092538799%26rft_id%253Dinfo%253Apmid%252F11972065%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5OS85LzU4NjAiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzExLzM0NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.53" data-doi="10.1101/gad.5.2.232"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wisdom</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">The protein-coding region of c-myc mRNA contains a sequence that specifies rapid mRNA turnover and induction by protein synthesis inhibitors.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">232</span>–<span class="cit-lpage">243</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DWisdom%26rft.auinit1%253DR%26rft.volume%253D5%26rft.issue%253D2%26rft.spage%253D232%26rft.epage%253D243%26rft.atitle%253DThe%2Bprotein-coding%2Bregion%2Bof%2Bc-myc%2BmRNA%2Bcontains%2Ba%2Bsequence%2Bthat%2Bspecifies%2Brapid%2BmRNA%2Bturnover%2Band%2Binduction%2Bby%2Bprotein%2Bsynthesis%2Binhibitors.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.5.2.232%26rft_id%253Dinfo%253Apmid%252F1995415%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6NzoiNS8yLzIzMiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzQ3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.54" data-doi="10.1002/yea.1225"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wishart</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wardleworth</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliver</span>  <span class="cit-name-given-names">S. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Doxycycline, the drug used to control the tet-regulatable promoter system, has no effect on global gene expression in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">565</span>–<span class="cit-lpage">569</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DWishart%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D22%26rft.issue%253D7%26rft.spage%253D565%26rft.epage%253D569%26rft.atitle%253DDoxycycline%252C%2Bthe%2Bdrug%2Bused%2Bto%2Bcontrol%2Bthe%2Btet-regulatable%2Bpromoter%2Bsystem%252C%2Bhas%2Bno%2Beffect%2Bon%2Bglobal%2Bgene%2Bexpression%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.1225%26rft_id%253Dinfo%253Apmid%252F15942933%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.1225&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15942933&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230421500005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.55" data-doi="10.1101/gr.1272403"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Nimwegen</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zavolan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rajewsky</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schroeder</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Decay rates of human mRNAs: correlation with functional characteristics and sequence attributes.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">1863</span>–<span class="cit-lpage">1872</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DYang%26rft.auinit1%253DE.%26rft.volume%253D13%26rft.issue%253D8%26rft.spage%253D1863%26rft.epage%253D1872%26rft.atitle%253DDecay%2BRates%2Bof%2BHuman%2BmRNAs%253A%2BCorrelation%2BWith%2BFunctional%2BCharacteristics%2Band%2BSequence%2BAttributes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.1272403%26rft_id%253Dinfo%253Apmid%252F12902380%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjEzLzgvMTg2MyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzQ3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.56" data-doi="10.1073/pnas.93.17.8895"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yun</span>  <span class="cit-name-given-names">D.-F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherman</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Degradation of CYC1 mRNA in the yeast <em>Saccharomyces cerevisiae</em> does not require translation.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">8895</span>–<span class="cit-lpage">8900</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DYun%26rft.auinit1%253DD.%2BF.%26rft.volume%253D93%26rft.issue%253D17%26rft.spage%253D8895%26rft.epage%253D8900%26rft.atitle%253DDegradation%2Bof%2BCYC1%2BmRNA%2Bin%2Bthe%2Byeast%2BSaccharomyces%2Bcerevisiae%2Bdoes%2Bnot%2Brequire%2Btranslation.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.93.17.8895%26rft_id%253Dinfo%253Apmid%252F8799124%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTMvMTcvODg5NSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzQ3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3475.57" data-doi="10.1038/nsmb.1514"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zenklusen</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larson</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singer</span>  <span class="cit-name-given-names">R. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Single-RNA counting reveals alternative modes of gene expression in yeast.</span> <abbr class="cit-jnl-abbrev">Nat. Struct. Mol. Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1263</span>–<span class="cit-lpage">1271</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bstructural%2B%2526%2Bmolecular%2Bbiology%26rft.stitle%253DNat%2BStruct%2BMol%2BBiol%26rft.aulast%253DZenklusen%26rft.auinit1%253DD.%26rft.volume%253D15%26rft.issue%253D12%26rft.spage%253D1263%26rft.epage%253D1271%26rft.atitle%253DSingle-RNA%2Bcounting%2Breveals%2Balternative%2Bmodes%2Bof%2Bgene%2Bexpression%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb.1514%26rft_id%253Dinfo%253Apmid%252F19011635%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsmb.1514&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19011635&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3475.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261383900012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/11/3475.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/11/3467" title="Depletion of Shine-Dalgarno Sequences Within Bacterial Coding Regions Is Expression Dependent" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/11/3485" title="First Draft Assembly and Annotation of the Genome of a California Endemic Oak Quercus lobata Née (Fagaceae)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/11"><span>Volume 6 Issue 11, November 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/11.cover-source.jpg?itok=hTYn4PRy" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/11/3475" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-jTNzDubx41kKrld0KZb9l12V6Y8AHmpciKjc_0qq79Q" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/11/3475" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/11/3475" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/11/3475" class="active" data-icon-position="" data-hide-link-title="0">Multiple Transcript Properties Related to Translation Affect mRNA Degradation Rates in Saccharomyces cerevisiae</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460457" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-TdeJKCTdrAXzP9Im0JgqqPF90FkW3kmYF4v1izRRhdQ" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/11/3475.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/11/3475" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-74GXevtZBRrJ1MMQlprNWJMAEMmPcvuGC1g6G3n4Wh8" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/11/3475" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/11/3475.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460457" id="node460457--2" data-pisa="ggg;6/11/3475" data-pisa-master="ggg;g3.116.032276" data-apath="/ggg/6/11/3475.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Multiple Transcript Properties Related to Translation Affect mRNA Degradation Rates in <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Benjamin Neymotin</span>, <span class="highwire-citation-author" data-delta="1">Victoria Ettorre</span> and <span class="highwire-citation-author" data-delta="2">David Gresham</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3475-3483;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.032276 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460457--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Benjamin Neymotin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Center for Genomics and Systems Biology, New York University, New York 10003</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Benjamin%2BNeymotin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Neymotin%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABenjamin%2BNeymotin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Victoria Ettorre </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Center for Genomics and Systems Biology, New York University, New York 10003</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Victoria%2BEttorre%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ettorre%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVictoria%2BEttorre%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">David Gresham </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Center for Genomics and Systems Biology, New York University, New York 10003</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BGresham%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gresham%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BGresham%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dgresham@nyu.edu" class="" data-icon-position="" data-hide-link-title="0">dgresham@nyu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/11/3475" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/11/3475&amp;title=Multiple%20Transcript%20Properties%20Related%20to%20Translation%20Affect%20mRNA%20Degradation%20Rates%20in%20Saccharomyces%20cerevisiae" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/11/3475&amp;title=Multiple%20Transcript%20Properties%20Related%20to%20Translation%20Affect%20mRNA%20Degradation%20Rates%20in%20Saccharomyces%20cerevisiae" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/11/3475&amp;title=Multiple%20Transcript%20Properties%20Related%20to%20Translation%20Affect%20mRNA%20Degradation%20Rates%20in%20Saccharomyces%20cerevisiae" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/11/3475&amp;text=Multiple%20Transcript%20Properties%20Related%20to%20Translation%20Affect%20mRNA%20Degradation%20Rates%20in%20Saccharomyces%20cerevisiae" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/11/3475&amp;title=Multiple%20Transcript%20Properties%20Related%20to%20Translation%20Affect%20mRNA%20Degradation%20Rates%20in%20Saccharomyces%20cerevisiae" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/11/3475&amp;t=Multiple%20Transcript%20Properties%20Related%20to%20Translation%20Affect%20mRNA%20Degradation%20Rates%20in%20Saccharomyces%20cerevisiae" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/11/3475&amp;title=Multiple%20Transcript%20Properties%20Related%20to%20Translation%20Affect%20mRNA%20Degradation%20Rates%20in%20Saccharomyces%20cerevisiae" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/11/3475&amp;title=Multiple%20Transcript%20Properties%20Related%20to%20Translation%20Affect%20mRNA%20Degradation%20Rates%20in%20Saccharomyces%20cerevisiae" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460457/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460457" id="node460457--3" data-pisa="ggg;6/11/3475" data-pisa-master="ggg;g3.116.032276" data-apath="/ggg/6/11/3475.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Multiple Transcript Properties Related to Translation Affect mRNA Degradation Rates in <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Benjamin Neymotin</span>, <span class="highwire-citation-author" data-delta="1">Victoria Ettorre</span> and <span class="highwire-citation-author" data-delta="2">David Gresham</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3475-3483;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.032276 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460457--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Benjamin Neymotin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Center for Genomics and Systems Biology, New York University, New York 10003</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Benjamin%2BNeymotin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Neymotin%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABenjamin%2BNeymotin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Victoria Ettorre </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Center for Genomics and Systems Biology, New York University, New York 10003</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Victoria%2BEttorre%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ettorre%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVictoria%2BEttorre%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">David Gresham </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Center for Genomics and Systems Biology, New York University, New York 10003</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BGresham%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gresham%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BGresham%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dgresham@nyu.edu" class="" data-icon-position="" data-hide-link-title="0">dgresham@nyu.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3475&amp;rft.epage=3483&amp;rft.atitle=Multiple%20Transcript%20Properties%20Related%20to%20Translation%20Affect%20mRNA%20Degradation%20Rates%20in%20Saccharomyces%20cerevisiae&amp;rft.volume=6&amp;rft.issue=11&amp;rft.date=2016-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Neymotin%2C+Benjamin&amp;rft.au=Ettorre%2C+Victoria&amp;rft.au=Gresham%2C+David"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460457/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460457/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460457/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460457/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460457/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460457/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460457/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460457/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460457/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460457/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460457/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460457/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/11/3475" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/11/3475.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/11/3475.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/11/3475.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/11/3475.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
