<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/11/3507" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/11/3507.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/11/3507.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/11/3507.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/6/11/3507/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/11/3507.atom" />
<meta name="HW.pisa" content="ggg;6/11/3507" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Suitability of Different Mapping Algorithms for Genome-Wide Polymorphism Scans with Pool-Seq Data" />
<meta name="DC.Identifier" content="10.1534/g3.116.034488" />
<meta name="DC.Date" content="2016-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Kofler et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The cost-effectiveness of sequencing pools of individuals (Pool-Seq) provides the basis for the popularity and widespread use of this method for many research questions, ranging from unraveling the genetic basis of complex traits, to the clonal evolution of cancer cells. Because the accuracy of Pool-Seq could be affected by many potential sources of error, several studies have determined, for example, the influence of sequencing technology, the library preparation protocol, and mapping parameters. Nevertheless, the impact of the mapping tools has not yet been evaluated. Using simulated and real Pool-Seq data, we demonstrate a substantial impact of the mapping tools, leading to characteristic false positives in genome-wide scans. The problem of false positives was particularly pronounced when data with different read lengths and insert sizes were compared. Out of 14 evaluated algorithms novoalign, bwa mem and clc4 are most suitable for mapping Pool-Seq data. Nevertheless, no single algorithm is sufficient for avoiding all false positives. We show that the intersection of the results of two mapping algorithms provides a simple, yet effective, strategy to eliminate false positives. We propose that the implementation of a consistent Pool-Seq bioinformatics pipeline, building on the recommendations of this study, can substantially increase the reliability of Pool-Seq results, in particular when libraries generated with different protocols are being compared." />
<meta name="DC.Contributor" content="Robert Kofler" />
<meta name="DC.Contributor" content="Anna Maria Langmüller" />
<meta name="DC.Contributor" content="Pierre Nouhaud" />
<meta name="DC.Contributor" content="Kathrin Anna Otte" />
<meta name="DC.Contributor" content="Christian Schlötterer" />
<meta name="article:published_time" content="2016-11-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Suitability of Different Mapping Algorithms for Genome-Wide Polymorphism Scans with Pool-Seq Data" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The cost-effectiveness of sequencing pools of individuals (Pool-Seq) provides the basis for the popularity and widespread use of this method for many research questions, ranging from unraveling the genetic basis of complex traits, to the clonal evolution of cancer cells. Because the accuracy of Pool-Seq could be affected by many potential sources of error, several studies have determined, for example, the influence of sequencing technology, the library preparation protocol, and mapping parameters. Nevertheless, the impact of the mapping tools has not yet been evaluated. Using simulated and real Pool-Seq data, we demonstrate a substantial impact of the mapping tools, leading to characteristic false positives in genome-wide scans. The problem of false positives was particularly pronounced when data with different read lengths and insert sizes were compared. Out of 14 evaluated algorithms novoalign, bwa mem and clc4 are most suitable for mapping Pool-Seq data. Nevertheless, no single algorithm is sufficient for avoiding all false positives. We show that the intersection of the results of two mapping algorithms provides a simple, yet effective, strategy to eliminate false positives. We propose that the implementation of a consistent Pool-Seq bioinformatics pipeline, building on the recommendations of this study, can substantially increase the reliability of Pool-Seq results, in particular when libraries generated with different protocols are being compared.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/11/01" />
<meta name="citation_mjid" content="ggg;6/11/3507" />
<meta name="citation_id" content="6/11/3507" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/11/3507" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/11/3507.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/11/3507.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/11/3507.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.034488" />
<meta name="citation_pmid" content="27613752" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3507" />
<meta name="citation_lastpage" content="3515" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Robert Kofler" />
<meta name="citation_author_institution" content="Institut für Populationsgenetik, Vetmeduni Vienna, Veterinärplatz 1, 1210 Wien 1210, Austria" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9960-7248" />
<meta name="citation_author" content="Anna Maria Langmüller" />
<meta name="citation_author_institution" content="Institut für Populationsgenetik, Vetmeduni Vienna, Veterinärplatz 1, 1210 Wien 1210, Austria" />
<meta name="citation_author_institution" content="Vienna Graduate School of Population Genetics, 1210, Austria" />
<meta name="citation_author" content="Pierre Nouhaud" />
<meta name="citation_author_institution" content="Institut für Populationsgenetik, Vetmeduni Vienna, Veterinärplatz 1, 1210 Wien 1210, Austria" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4923-0993" />
<meta name="citation_author" content="Kathrin Anna Otte" />
<meta name="citation_author_institution" content="Institut für Populationsgenetik, Vetmeduni Vienna, Veterinärplatz 1, 1210 Wien 1210, Austria" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5026-4168" />
<meta name="citation_author" content="Christian Schlötterer" />
<meta name="citation_author_institution" content="Institut für Populationsgenetik, Vetmeduni Vienna, Veterinärplatz 1, 1210 Wien 1210, Austria" />
<meta name="citation_author_email" content="christian.schloetterer@vetmeduni.ac.at" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4710-6526" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=C. Alkan;citation_author=J. M. Kidd;citation_author=T. Marques-bonet;citation_author=G. Aksay;citation_author=F. Hormozdiari;citation_title=Personalized copy-number and segmental duplication maps using next-generation sequencing.;citation_pages=1061-1067;citation_volume=41;citation_year=2010;citation_issue=10" />
<meta name="citation_reference" content="citation_journal_title=Proc. Biol. Sci.;citation_title=Evolutionary genomics of Culex pipiens: global and local adaptations associated with climate, life-history traits and anthropogenic factors.;citation_author=H. Asgharian;citation_author=P. L. Chang;citation_author=S. Lysenkov;citation_author=V. A. Scobeyeva;citation_author=W. K. Reisen;citation_author=S. V. Nuzhdin;citation_volume=282;citation_year=2015;citation_issue=1810" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=E. Axelsson;citation_author=A. Ratnakumar;citation_author=M.-L. Arendt;citation_author=K. Maqbool;citation_author=M. T. Webster;citation_title=The genomic signature of dog domestication reveals adaptation to a starch-rich diet.;citation_pages=360-364;citation_volume=495;citation_year=2013;citation_issue=7441;citation_pmid=23354050;citation_doi=10.1038/nature11837" />
<meta name="citation_reference" content="citation_journal_title=Cancer Inform.;citation_author=R. Bao;citation_author=L. Huang;citation_author=J. Andrade;citation_author=W. Tan;citation_author=W. A. Kibbe;citation_title=Review of current methods, applications, and data management for the bioinformatics analysis of whole exome sequencing.;citation_pages=67-83;citation_volume=13;citation_year=2014;citation_pmid=25288881" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=H. Bastide;citation_author=A. Betancourt;citation_author=V. Nolte;citation_author=R. Tobler;citation_author=P. Stöbe;citation_title=A genome-wide, fine-scale map of natural pigmentation variation in Drosophila melanogaster.;citation_pages=e1003534;citation_volume=9;citation_year=2013;citation_issue=6;citation_pmid=23754958;citation_doi=10.1371/journal.pgen.1003534" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=A. O. Bergland;citation_author=E. L. Behrman;citation_author=K. R. O’Brien;citation_author=P. S. Schmidt;citation_author=D. A. Petrov;citation_title=Genomic evidence of rapid and stable adaptive oscillations over seasonal time scales in Drosophila.;citation_pages=e1004775;citation_volume=10;citation_year=2014;citation_issue=11;citation_pmid=25375361;citation_doi=10.1371/journal.pgen.1004775" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=S. Boitard;citation_author=R. Kofler;citation_author=P. Françoise;citation_author=D. Robelin;citation_author=C. Schlötterer;citation_title=Pool-hmm: a Python program for estimating the allele frequency spectrum and detecting selective sweeps from next generation sequencing of pooled samples.;citation_pages=337-340;citation_volume=13;citation_year=2013;citation_issue=2;citation_pmid=23311589;citation_doi=10.1111/1755-0998.12063" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=I. H. Cheeseman;citation_author=M. McDew-White;citation_author=A. P. Phyo;citation_author=K. Sriprawat;citation_author=F. Nosten;citation_title=Pooled sequencing and rare variant association tests for identifying the determinants of emerging drug resistance in malaria parasites.;citation_volume=32;citation_year=2015;citation_issue=4;citation_pmid=25534029;citation_doi=10.1093/molbev/msu397" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=J. F. Degner;citation_author=J. C. Marioni;citation_author=A. A. Pai;citation_author=J. K. Pickrell;citation_author=E. Nkadori;citation_title=Effect of read-mapping biases on detecting allele-specific expression from RNA-sequencing data.;citation_volume=25;citation_year=2009;citation_issue=24;citation_pmid=19808877;citation_doi=10.1093/bioinformatics/btp579" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=L. Ding;citation_author=T. J. Ley;citation_author=D. E. Larson;citation_author=C. A. Miller;citation_author=D. C. Koboldt;citation_title=Clonal evolution in relapsed acute myeloid leukaemia revealed by whole-genome sequencing.;citation_pages=506-510;citation_volume=481;citation_year=2012;citation_issue=7382;citation_pmid=22237025;citation_doi=10.1038/nature10738" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=L. Ferretti;citation_author=S. E. Ramos-Onsins;citation_author=M. Pérez-Enciso;citation_title=Population genomics from pool sequencing.;citation_pages=5561-5576;citation_volume=22;citation_year=2013;citation_issue=22;citation_doi=10.1111/mec.12522" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=M. A. Field;citation_author=V. Cho;citation_author=T. D. Andrews;citation_author=C. C. Goodnow;citation_title=Reliably detecting clinically important variants requires both combined variant calls and optimized filtering strategies.;citation_pages=e0143199;citation_volume=10;citation_year=2015;citation_issue=11;citation_pmid=26600436;citation_doi=10.1371/journal.pone.0143199" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Futschik;citation_author=C. Schlötterer;citation_title=The next generation of molecular markers from massively parallel sequencing of pooled DNA samples.;citation_volume=186;citation_year=2010;citation_issue=1;citation_pmid=20457880;citation_doi=10.1534/genetics.110.114397" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=M. Gautier;citation_author=J. Foucaud;citation_author=K. Gharbi;citation_author=T. Cézard;citation_author=M. Galan;citation_title=Estimation of population allele frequencies from next-generation sequencing data: pool- vs. individual-based genotyping.;citation_pages=3766-3779;citation_volume=22;citation_year=2013;citation_issue=14;citation_doi=10.1111/mec.12360" />
<meta name="citation_reference" content="citation_title=Principles of Population Genetics.;citation_author=D. L. Hartl;citation_author=A. G. Clark;citation_year=1997" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=S. Hoffmann;citation_author=C. Otto;citation_author=S. Kurtz;citation_author=C. M. Sharma;citation_author=P. Khaitovich;citation_title=Fast mapping of short sequences with mismatches, insertions and deletions using index structures.;citation_pages=e1000502-e1000502;citation_volume=5;citation_year=2009;citation_issue=9;citation_pmid=19750212;citation_doi=10.1371/journal.pcbi.1000502" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=R. Kofler;citation_author=C. Schlötterer;citation_title=A guide for the design of evolve and resequencing studies.;citation_volume=31;citation_year=2014;citation_issue=2;citation_pmid=24214537;citation_doi=10.1093/molbev/mst221" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. Kofler;citation_author=P. Orozco-terWengel;citation_author=N. De Maio;citation_author=R. V. Pandey;citation_author=V. Nolte;citation_title=PoPoolation: a toolbox for population genetic analysis of next generation sequencing data from pooled individuals.;citation_pages=e15925-e15925;citation_volume=6;citation_year=2011;citation_issue=1;citation_pmid=21253599;citation_doi=10.1371/journal.pone.0015925" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=R. Kofler;citation_author=R. V. Pandey;citation_author=C. Schlötterer;citation_title=PoPoolation2: identifying differentiation between populations using sequencing of pooled DNA samples (Pool-Seq).;citation_volume=27;citation_year=2011;citation_issue=24;citation_pmid=22025480;citation_doi=10.1093/bioinformatics/btr589" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=R. Kofler;citation_author=A. J. Betancourt;citation_author=C. Schlötterer;citation_title=Sequencing of pooled DNA samples (Pool-Seq) uncovers complex dynamics of transposable element insertions in Drosophila melanogaster.;citation_pages=e1002487-e1002487;citation_volume=8;citation_year=2012;citation_issue=1;citation_pmid=22291611;citation_doi=10.1371/journal.pgen.1002487" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=R. Kofler;citation_author=T. Hill;citation_author=V. Nolte;citation_author=A. Betancourt;citation_author=C. Schlötterer;citation_title=The recent invasion of natural Drosophila simulans populations by the P-element.;citation_volume=112;citation_year=2015;citation_issue=21;citation_pmid=25964349;citation_doi=10.1073/pnas.1500758112" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=R. Kofler;citation_author=V. Nolte;citation_author=C. Schlötterer;citation_title=The impact of library preparation protocols on the consistency of allele frequency estimates in Pool-Seq data.;citation_pages=118-122;citation_volume=16;citation_year=2015;citation_issue=1;citation_pmid=26014582" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. Lamichhaney;citation_author=A. M. Barrio;citation_author=N. Rafati;citation_author=G. Sundström;citation_author=C.-J. Rubin;citation_title=Population-scale sequencing reveals genetic differentiation due to local adaptation in atlantic herring.;citation_volume=109;citation_year=2012;citation_issue=47;citation_pmid=23134729;citation_doi=10.1073/pnas.1216128109" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=G. I. Lang;citation_author=D. P. Rice;citation_author=M. J. Hickman;citation_author=E. Sodergren;citation_author=G. M. Weinstock;citation_title=Pervasive genetic hitchhiking and clonal interference in forty evolving yeast populations.;citation_pages=571-574;citation_volume=500;citation_year=2013;citation_issue=7464;citation_pmid=23873039;citation_doi=10.1038/nature12344" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=B. Langmead;citation_author=S. L. Salzberg;citation_title=Fast gapped-read alignment with Bowtie 2.;citation_pages=357-359;citation_volume=9;citation_year=2012;citation_issue=4;citation_pmid=22388286;citation_doi=10.1038/nmeth.1923" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_volume=25;citation_year=2009;citation_issue=14;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate long-read alignment with Burrows-Wheeler transform.;citation_volume=26;citation_year=2010;citation_issue=5;citation_pmid=20080505;citation_doi=10.1093/bioinformatics/btp698" />
<meta name="citation_reference" content="citation_journal_title=Brief. Bioinform.;citation_author=H. Li;citation_author=N. Homer;citation_title=A survey of sequence alignment algorithms for next-generation sequencing.;citation_volume=11;citation_year=2010;citation_issue=5;citation_pmid=20460430;citation_doi=10.1093/bib/bbq015" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=H. Li;citation_author=J. Ruan;citation_author=R. Durbin;citation_title=Mapping short DNA sequencing reads and calling variants using mapping quality scores.;citation_volume=18;citation_year=2008;citation_issue=11;citation_pmid=18714091;citation_doi=10.1101/gr.078212.108" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The sequence alignment/map format and SAMtools.;citation_volume=25;citation_year=2009;citation_issue=16;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=R. Li;citation_author=C. Yu;citation_author=Y. Li;citation_author=T.-W. Lam;citation_author=S.-M. Yiu;citation_title=Soap2: an improved ultrafast tool for short read alignment.;citation_volume=25;citation_year=2009;citation_issue=15;citation_pmid=19497933;citation_doi=10.1093/bioinformatics/btp336" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J.-Q. Lim;citation_author=C. Tennakoon;citation_author=P. Guan;citation_author=W.-K. Sung;citation_title=Batalign: an incremental method for accurate alignment of sequencing reads.;citation_volume=43;citation_year=2015;citation_issue=16;citation_pmid=26170239;citation_doi=10.1093/nar/gkv533" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=G. Lunter;citation_author=M. Goodson;citation_title=Stampy: a statistical algorithm for sensitive and fast mapping of Illumina sequence reads.;citation_volume=21;citation_year=2011;citation_issue=6;citation_pmid=20980556;citation_doi=10.1101/gr.111120.110" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=S. Marco-Sola;citation_author=M. Sammeth;citation_author=R. Guigó;citation_author=P. Ribeca;citation_title=The GEM mapper: fast, accurate and versatile alignment by filtration.;citation_pages=1185-1188;citation_volume=9;citation_year=2012;citation_issue=12;citation_pmid=23103880;citation_doi=10.1038/nmeth.2221" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=N. E. Martins;citation_author=V. G. Faria;citation_author=V. Nolte;citation_author=C. Schlötterer;citation_author=L. Teixeira;citation_title=Host adaptation to viruses relies on few genes with different cross-resistance properties.;citation_volume=111;citation_year=2014;citation_issue=43;citation_pmid=25298533;citation_doi=10.1073/pnas.1418561111" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=B. Nevado;citation_author=S. Ramos-Onsins;citation_author=M. Perez-Enciso;citation_title=Resequencing studies of nonmodel organisms using closely related reference genomes: optimal experimental designs and bioinformatics approaches for population genomics.;citation_pages=1764-1779;citation_volume=23;citation_year=2014;citation_issue=7;citation_doi=10.1111/mec.12693" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=R. Nielsen;citation_author=J. S. Paul;citation_author=A. Albrechtsen;citation_author=Y. S. Song;citation_title=Genotype and SNP calling from next-generation sequencing data.;citation_pages=443-451;citation_volume=12;citation_year=2011;citation_issue=6;citation_pmid=21587300;citation_doi=10.1038/nrg2986" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=V. Nolte;citation_author=R. V. Pandey;citation_author=R. Kofler;citation_author=C. Schlötterer;citation_title=Genome-wide patterns of natural variation reveal strong selective sweeps and ongoing genomic conflict in Drosophila mauritiana.;citation_pages=99-110;citation_volume=23;citation_year=2012;citation_pmid=23051690" />
<meta name="citation_reference" content="citation_journal_title=Genome Med.;citation_author=J. O’Rawe;citation_author=T. Jiang;citation_author=G. Sun;citation_author=Y. Wu;citation_author=W. Wang;citation_title=Low concordance of multiple variant-calling pipelines: practical implications for exome and genome sequencing.;citation_pages=28;citation_volume=5;citation_year=2013;citation_issue=3;citation_pmid=23537139;citation_doi=10.1186/gm432" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=P. Orozco-terWengel;citation_author=M. Kapun;citation_author=V. Nolte;citation_author=R. Kofler;citation_author=T. Flatt;citation_title=Adaptation of Drosophila to a novel laboratory environment reveals temporally heterogeneous trajectories of selected alleles.;citation_pages=4931-4941;citation_volume=21;citation_year=2012;citation_issue=20;citation_pmid=22726122;citation_doi=10.1111/j.1365-294X.2012.05673.x" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=N. Palmieri;citation_author=V. Nolte;citation_author=J. Chen;citation_author=C. Schlötterer;citation_title=Genome assembly and annotation of a Drosophila simulans strain from Madagascar.;citation_pages=372-381;citation_volume=15;citation_year=2015;citation_issue=2;citation_pmid=24961367;citation_doi=10.1111/1755-0998.12297" />
<meta name="citation_reference" content="citation_title=R: A Language and Environment for Statistical Computing.;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=C. Rellstab;citation_author=S. Zoller;citation_author=A. Tedder;citation_author=F. Gugerli;citation_author=M. C. Fischer;citation_title=Validation of SNP allele frequencies determined by pooled next-generation sequencing in natural populations of a non-model plant species.;citation_pages=e80422;citation_volume=8;citation_year=2013;citation_issue=11;citation_pmid=24244686;citation_doi=10.1371/journal.pone.0080422" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=C.-J. Rubin;citation_author=M. C. Zody;citation_author=J. Eriksson;citation_author=J. R. Meadows;citation_author=E. Sherwood;citation_title=Whole-genome resequencing reveals loci under selection during chicken domestication.;citation_pages=587-591;citation_volume=464;citation_year=2010;citation_issue=7288;citation_pmid=20220755;citation_doi=10.1038/nature08832" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=C. Schlötterer;citation_author=R. Tobler;citation_author=R. Kofler;citation_author=V. Nolte;citation_title=Sequencing pools of individuals—mining genome-wide polymorphism data without big funding.;citation_pages=749-763;citation_volume=15;citation_year=2014;citation_issue=11;citation_pmid=25246196" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=K. Schneeberger;citation_author=S. Ossowski;citation_author=C. Lanz;citation_author=T. Juul;citation_author=A. H. Petersen;citation_title=SHOREmap: simultaneous mapping and mutation identification by deep sequencing.;citation_pages=550-551;citation_volume=6;citation_year=2009;citation_issue=8;citation_pmid=19644454;citation_doi=10.1038/nmeth0809-550" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=F. J. Sedlazeck;citation_author=P. Rescheneder;citation_author=A. von Haeseler;citation_title=NextGenMap: fast and accurate read mapping in highly polymorphic genomes.;citation_volume=29;citation_year=2013;citation_issue=21;citation_pmid=23975764;citation_doi=10.1093/bioinformatics/btt468" />
<meta name="citation_reference" content="citation_journal_title=Brief. Bioinform.;citation_author=H. Thorvaldsdóttir;citation_author=J. T. Robinson;citation_author=J. P. Mesirov;citation_title=Integrative Genomics Viewer (IGV): high-performance genomics data visualization and exploration.;citation_volume=14;citation_year=2013;citation_pmid=22517427;citation_doi=10.1093/bib/bbs017" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=R. Tobler;citation_author=S. U. Franssen;citation_author=R. Kofler;citation_author=P. Orozco-terWengel;citation_author=V. Nolte;citation_title=Massive habitat-specific genomic response in D. melanogaster populations during experimental evolution in hot and cold environments.;citation_volume=31;citation_year=2014;citation_pmid=24150039;citation_doi=10.1093/molbev/mst205" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=T. L. Turner;citation_author=E. C. Bourne;citation_author=E. J. VonWettberg;citation_author=T. T. Hu;citation_author=S. V. Nuzhdin;citation_title=Population resequencing reveals local adaptation of Arabidopsis lyrata to serpentine soils.;citation_pages=260-263;citation_volume=42;citation_year=2010;citation_issue=3;citation_pmid=20101244;citation_doi=10.1038/ng.515" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=T. D. Wu;citation_author=S. Nacu;citation_title=Fast and SNP-tolerant detection of complex variants and splicing in short reads.;citation_volume=26;citation_year=2010;citation_issue=7;citation_pmid=20147302;citation_doi=10.1093/bioinformatics/btq057" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Suitability of Different Mapping Algorithms for Genome-Wide Polymorphism Scans with Pool-Seq Data" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11.cover-source.jpg" />
<meta name="twitter:description" content="The cost-effectiveness of sequencing pools of individuals (Pool-Seq) provides the basis for the popularity and widespread use of this method for many research questions, ranging from unraveling the genetic basis of complex traits, to the clonal evolution of cancer cells. Because the accuracy of Pool-Seq could be affected by many potential sources of error, several studies have determined, for example, the influence of sequencing technology, the library preparation protocol, and mapping parameters. Nevertheless, the impact of the mapping tools has not yet been evaluated. Using simulated and real Pool-Seq data, we demonstrate a substantial impact of the mapping tools, leading to characteristic false positives in genome-wide scans. The problem of false positives was particularly pronounced when data with different read lengths and insert sizes were compared. Out of 14 evaluated algorithms novoalign, bwa mem and clc4 are most suitable for mapping Pool-Seq data. Nevertheless, no single algorithm is sufficient for avoiding all false positives. We show that the intersection of the results of two mapping algorithms provides a simple, yet effective, strategy to eliminate false positives. We propose that the implementation of a consistent Pool-Seq bioinformatics pipeline, building on the recommendations of this study, can substantially increase the reliability of Pool-Seq results, in particular when libraries generated with different protocols are being compared." />
<meta name="og-title" property="og:title" content="Suitability of Different Mapping Algorithms for Genome-Wide Polymorphism Scans with Pool-Seq Data" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/11/3507" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The cost-effectiveness of sequencing pools of individuals (Pool-Seq) provides the basis for the popularity and widespread use of this method for many research questions, ranging from unraveling the genetic basis of complex traits, to the clonal evolution of cancer cells. Because the accuracy of Pool-Seq could be affected by many potential sources of error, several studies have determined, for example, the influence of sequencing technology, the library preparation protocol, and mapping parameters. Nevertheless, the impact of the mapping tools has not yet been evaluated. Using simulated and real Pool-Seq data, we demonstrate a substantial impact of the mapping tools, leading to characteristic false positives in genome-wide scans. The problem of false positives was particularly pronounced when data with different read lengths and insert sizes were compared. Out of 14 evaluated algorithms novoalign, bwa mem and clc4 are most suitable for mapping Pool-Seq data. Nevertheless, no single algorithm is sufficient for avoiding all false positives. We show that the intersection of the results of two mapping algorithms provides a simple, yet effective, strategy to eliminate false positives. We propose that the implementation of a consistent Pool-Seq bioinformatics pipeline, building on the recommendations of this study, can substantially increase the reliability of Pool-Seq results, in particular when libraries generated with different protocols are being compared." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11.cover-source.jpg" />
    <title>Suitability of Different Mapping Algorithms for Genome-Wide Polymorphism Scans with Pool-Seq Data | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"WfEa-CYUmC5MJRPPwvNXUDHQTowz4Is1Sw904DoLmIM","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460473","apath":"\/ggg\/6\/11\/3507.atom","pisa":"ggg;6\/11\/3507","ac":{"\/ggg\/6\/11\/3507.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/11\/3507.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/11\/3507"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/11\/3507"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/11\/3507.full":true,"\/content\/6\/11\/3507":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022sqlkzHRLxbW3dg1sGgUqhD6nWellYApUVD3OHEKhO6fbrN1ce+ZNuSqEI1sCDFBr1Gd6xAo2Q6trGNlwg2APvgKPn++vWmaXL2Hr3CsYbDj4P5tNw4rh4Pp4t59srhuYWqSo5KMb3imkvEpA7tOWdUPQ64oNnA\\\\\\\/gdHbMCkd0Mf7JBcMyNxihH6IQ5pV9u4ec4x22P2VibrMUlqcb5U3+eCFu9uyIKceHjEeaH3LR+5DXTXJMVLBDfF0JEZ74QcQeGfZtweY2FF4tUWwOuYDJcDvjkqslGwmseG7ZvSf2cvnvsQfuz3aMc4aubeWwajivtMygGQ13q2Hj2\\\\\\\/bEP\\\\\\\/I8iuNBaTr9lPyr1me7nLoLSRXrB7qQO\\\\\\\/mDPU1SxP+B3j9X11SYJkZObLcgQLql0ankzAc2LsZtbsy7bnMRJ3RVQUqWlelFZOb6Mnaw8r25A7u2lT\\\\\\\/tnotfy1kRoOPVYBhuVXDeiJRgHk1VuMUS+11hHx3YybEYPxPBuObpL\\\\\\\/LPcVHcOWlLdBByoD8jOxAAPWv1l5l8EJ036UH5vwpu79\\\\\\\/SpOvTYGLpnX\\\\\\\/OfeDX1gaaJc0fV\\\\\\\/St4Dj14iaQLPamtwJo4V0EZXAGmOkbjc+UaC3Nw\\\\\\\/ycT1b4g65XMFGVPVTFQU5P0F1IWzR6RCrjeAbDKbJz3MOzcyAy7L6KT\\\\\\\/2Al3BuoQwqpLNvUMFTMy2eJAxjG7Al4B0syBUua8j5m0KaitJWh2sugVESOpj\\\\\\\/501Ko7COMyPO\\\\\\\/r6wxok2WwaZjNfhkoMO2UKMoSowF1LZS6yG53IDYSCu9MsvPpSj0jUi0OTbxgMPHzhljPPkeu6kP1EdX+XPYlCbiN008OiUEaZPmbMLGu9C+fbMh8kKXNKWfYZeFH1gRZu44uQBI6fEycvAv1p9TefYL0uN+51IOeI1ukmUppSp0OsOqqkJP9BZfXuNzlWgGt++VTRlvRjrLWsmHLAY4uw3pM\\\\\\\/oNUpYCD3mfFK5A+Ll8+Pvb1j0OuFootaPsf9x9iyD9xn3hLMcaBr+xPtEp0KrD6jlGuSDZFX\\\\\\\/iv2rK1yCYaySuJp5aipyaF0Q4WJnTDddeV75xNS6Xg4UFb\\\\\\\/yHL06BvEn4O55VEhjRi376AWZe3xwxfzhix6WR8OvHos5Uzs64pCZMO2aat0KDLs\\\\\\\/z5o+7\\\\\\\/LepUChOiKXbP9xf8ZKnpjIKKFY9BEwEId5uWTU\\\\\\\/Yad50T0oShAlBbpBUISjOwsh9bULsOjCLSgzFIk8VmVfaKqmEgMIKgrLZh5sP\\\\\\\/H01s9bpTnXsQOD3tPX5rn\\\u0022,\\\u0022iv\\\u0022:\\\u0022SYlzdQxcIWJunnO1gky3Sg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b9457b661d18e83e05871cf568f6fcd4b052f746c676f47859dc416b5dfa0a5d\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022cKNwsUjlKJkGOPF+VoISeQhGd\\\\\\\/kGGW1hpZb3rT+My7Hr\\\\\\\/gau6f0MjwrSBW5f7p88xepFYW24qK2wqh9T4dC0+V1h+s3l3p4vdq7wS4C5sxXPI7\\\\\\\/OCcbOvItEB+eJNeTjT44qTwzM3W5PwQ5IUbCJsSBIkcpWGsg7F89ebA15U7eQ8vEovKr65fHQ9ploHyPafep2fNbmhG5hOUhHQFibSV4z\\\\\\\/wtzQll9HZ3qsU3Ne9RmhdJc9rHJVd1FxCClv0PGe2IXW+7uuhXi8xdT3QY8Xdiw+LLKzsBSeMyqxiJZj1+rnavHPzJy8kMC+eUimN7cReYt59NXp2Us34VmAX5Md5VK+qh57MjPUS\\\\\\\/i5s1CTzBd3EV99WSmR86DsX+N7VZxD0k4kUqurbUxfSlQns3FPLOwjXj2f\\\\\\\/vVVZDsYGuty0qcP9jV\\\\\\\/yvsBvwfaNXK4tzDryjtfIKgdoq2aVLUkK67QrEGz55DBloDcIfQ\\\\\\\/uivlIvvlu7QcmKYZbBKUpyRQoykMQEq15zZJMR6bUgPWN2xWT64Zmuepo+fW2YA+upMrcl\\\\\\\/lAJ8xm6B\\\\\\\/And1hUU2ygHcbKQnCpf3i9mgPvOZhBlzXMGQdzTg2+akl60RXl4UtHo5kdT3TDBwq+9TtCDXIo6R0aaOxGtXHxi1kjeRUYJU947dQaJ6ki9THH20CTs\\\\\\\/S4HlPmqtrLc9vP2C5KPJUXSLL4iDB0hwxUyt1qiLsWvIQZrqPPgLy31zoz4fU1LlOyfB05ewhGXrAwXpHXaMzOHoG+OdtDXKNwF5KX8KocsKXTDtN7QSwm6mAwCUVF183QMs9uKbPLJWFKkGpqd2AaSPeiNDW617pHcIBfsshveiGdnjptfltEU8CnMmQhZP2CodPem2cDtroRfKmxHi5eoISn7oKeeOjlA9ftFgzo63vBkAqFla3nu6TqEb99qb7JQ60um\\\\\\\/TTEuxMneGnDMs7W70drJN0Lw+8B8NfKtm7GAx6sasncx4UE03Wqilmb7ckitysV9R1aoXghQTca4MsPVjojUiD+MTyfMxJqRHS+SFmMYozXsx34BPzNmDcvAEnHlTA3oGFdz4t2EZmPkpl+amc9f0lxuBto8pPxKr8G7R7AfhcIvBX62OX0ori86OoPEV3Exrc1ctzRf7WqLq7rEyEKt2XjURqyt1EfnVPT1VxtyUGbV++5vSiIlDdmLtFQYsszkeDTChUUm6ULRXg\\\\\\\/zqnuMt\\\\\\\/3ntmkJg52iqXQCtyKAkzFRGQJf6QGO12XYCy4WA7NeL3sKr8JkEU\\\\\\\/nRQyyp7fXtsqk7SQvVSBSG8fLeIE72JAwBMAzhCexhm5lYv3W3ep6t1m101Y7ygfmHn7R5Z8N2vqVkP+8oToFKBtPRu59Wab1Lm+ioJzzxn5LDrnKRXp7wrCBGlZQmrPIyypp5r3F70qBHFlxs+jNMJsnj78qr2PIEDtLQOkJUl8WsxvufPIUYhYII7ilDi9Gc1TlZQafNxxOqoeDg3fLNTvIlsv8AoGP2jg8pf8icjvqkP3DJgdo3Nsk0EhgsRIpzes+bJB8jJrdtWAqiT629bSI2pVXcOtBBjyyelCo68iUtVfo3LF\\\\\\\/oETZaCCV9V9U5NpSdTmd1XBDIoshw0bjL3gYIRBwHHMgnecU2nnNSZOFXM3Bb\\\\\\\/LfFlze8EWPHyijB5aCPbjhyHmh2rVCWn0mdKpQS4kSaJIFD5NKZ1Q9Bq5bWXP+ZGZ17VGsE2wSDYYNKpk3ZK\\\\\\\/RqWnLwaN\\\\\\\/Sj4R\\\\\\\/JqVs4fWx70xKY0vLpK8v7Y29u382gJzoa5SGz8Hy3X2de4bWT870taHSMTrvvuLBRpev0aVRwvoAGYd2ehp+nNFTkRe\\\\\\\/iHuq\\\\\\\/xDrQAaCwnwBffsdycIP82VVMBUsNgKugsGJ+VkrObTb\\\\\\\/sE4Oj0oiac9AS0UentoJnuOniipJEbuh2uH1LSzdNa+ZEbhA10xvf68NISp4o9Okuwe7haZ1MRIqkg5RWc57FcSbVVd8Ynd4zwBZyEFohC27+6eUOsJogKB+lGxJRRd\\\\\\\/FXAF2IKFOZ2HpMQ5Td8wOKhOJv\\\\\\\/pJt1Y5rtncifKSnxq4XX5YieGEFYB67RUqpe9YzmJQr6Hpz1vnr\\\\\\\/ZjCIHzdh4GY6DuaSdY4UOSaFY\\\\\\\/sf9zpCOqdWEDVuHwvbRGjrgqh04\\\\\\\/8h71waOqV8oUBbLmubXiwPAuO94uMS5twWk5ANU8jMSiwA2vCwQ6tlVQC\\\\\\\/L9gZOsJY+hVwjkS+q1lQjVb0WPAbiUKpjdTdKwdUqGXuDuD4zMmF1kQ1aqULeNPwb4AY9SvFz\\\\\\\/L+W6z+4MdN4AW+ke2Arswu6ajGEF+qErxcI+ysopVwwFQHXjO2IK9XXNViVrr3P9FT27O+8YstLlefNXo2OVDZlkC4ScwbffIgeiL4kCQhajKwZNo6emmz1KJTcoRLDlbm\\\\\\\/zYuvfiJdWXvgQy5qvoFg+qO167T1mT+IyabXuulIq9MAiT3PEtUCTi2FLrQ9jl7e6sRyNsaKCXMi6YDfRJXX5qNRiKKZY8YmFytwPFQH1Oj7IWkgb0c2Yo2atCTUIzma9OJFg\\\\\\\/X8faKxW0Jnfu5CX9HVEa35ZASXmHjAT1ikS0aybkoU2PGNp3\\\\\\\/7+5cTQCQSqOyYIQfu8\\\\\\\/cPYB14plGDb3NLtQvesJqCixqeuf0hCMpvOmdIHp81\\\\\\\/7ieK0CyeyIUC1xVj0BY14w6bwBa1g0f1oVRlXMkqIqwXOY4PpOy9aEERVIMaSjyftuaHwGtqmxI63kFFhZrUWPLT8RY1SodjIHGXtbuSRNvhJ2NeV9sz3JhNdmuj\\\\\\\/3a+nQl0YLBLGI487hkShhWyUJo4JcCZcpveeg6lUBNshBnsem17ACgrhUvusqbNTtafYZqhwrGUgu+5C4NYBi3s+c4AeXSvvURkLoBDpgw5YyB55IYEkcPTKoUVmv4mR77w+3tWVIGM+tRHx+pWm7iosADBDO9+hZrmPd2RNgXlTaSPVJ+QCGr12mYAo8TYcIrv6VdHCHix7FbH71E7ruFYvx9AgjKkgus3pmkuCXGPN+7Rs6p9f\\\\\\\/nU2okHLZTObRSGfiwYJ26xSez6no2j3nY745K8XAxBNOKehbpEuXz74HWtPg3jC48jeeKNSbR38yfqFeLAFXKsypHvzRsmYpyM17p0MwGY1x1InquCwrydsO2qzUD5Nv05+uhG5EnhK7M80SxnZMGwel4qOoTdvBBSvNZh8lRVP0rGkQonRsRdYm7\\\\\\\/mK1xhvtSW06zz213zUBelhjoP2W2UcpdlFuqoEOvFm\\\\\\\/yLZ9f67+xwkdH9m\\\\\\\/Bl7djPafB7HeILGV6apQVcsa9EtE08uioFGLnM6DgvYIVe1yxOJB\\\\\\\/urzA+oYFa9KdXxTOlXj5MURh6+JJDZ4MikQfNT1SUdFZsy6VQCRgBJFbDBH1\\\\\\\/RsL5dzXIQ+Hk11fzBCEIguwh30zw9p2uLnEljmR5kJhpYsb+IQ+bX2DURnoPeNyySR7KkYhpusstz\\\\\\\/NVQUN8wVNobkz6bvnV27v1HWSUQSXyueY5EXalBjf1RkHS6xUCkA2CY8DBwYGOV6ESe4OvJ3wL5ItpADD7k39kaAb0mB+UHlzrxVjYEebnb\\\\\\\/RPVn6qvn10f0MyFl4TdCiV8ylh0YEfQI1HH4FZXG6Q9SlPgzXGJaZCftZ9SR\\\\\\\/Atckmz0fXBVYhmKDH33IXCMCsdlw86wS10uER0\\\\\\\/m3miHGFYnTmYLamVYhH1aKnKWsREctILktoJdvAWYuPkRt3r4lRnAnJQMNIm+YHfEiFVtcMVNIKamF8pb9ktca9dqvdrgDDhHXQted4U2rXtcXgaf8VvdqN5d5itcfUgvhwMuxHRuPYzOl5njYrutNPPUZDCWgUl08Y+sJbKmw9y56EcfNzNgRloMnsYuTLhtXWJ7sJd1Xv51iNNr4dAxRZL9JWRxnv6s\\\\\\\/5srMYx2wfD1WPRLE6JvG9fjY2UtqPMXmItr7OLsfhEyejdN3oV6a059RpITQ4s7CxKciIIq2O1xqnQjSpMBgpCyH4twbKDaUWB1vXW5+02p81584zsQ+C0jmTijwXn2xlJkM3ODu6qcgANr+N31qX8Xdqb2xvMBVzT8bezDUst\\\\\\\/QdLajuuvxliUn8fhUTS+oCL9Csy9di2AZ\\\\\\\/v9UjRA\\\\\\\/myTQDX635QLf\\\\\\\/j3CTmBOZPFY9lUKbkxMaNfo91U7+hd6CkUVrJxn0Gx+IyFlK4Q0It3tP88muTZv185MAoHuPQ1UGH+fb\\\\\\\/wugkJPeRYmiFcYWKpNl1jQO5MnYhdOzbkbMvhvX3ButeokTeLdekr8CnEqMLgjs6Ew79CwYTufCj9TCLn4cEsm2JMeQy1jfHUgZrOuRZUixJ\\\\\\\/H6dvvIyIYlWoCo\\\\\\\/rT9ki585+xRAQU8456E3uAn\\\\\\\/fTIANYSbhBzrrFPR4tU34qyZEJTeza+qIxKY3YWwXMug5VkPbyHHA+ycXaslWzCaecbBT+TR3gm5Z4RoAATkn3nPzkaNbRbpenH7kf+lT83VZqlzJDtrHXowOPuOmF2mS6q0gH61CB5maEeIO01VHhP0skCZ0B4JUDtZZ3qSQ\\\\\\\/TmsHbQXcAI\\\\\\\/\\\\\\\/4Kh92xWBtu4Oqcw6jsmSA+nrzySkikovoTFMArqSNtrwzREzyrOEQ7nqtoitOE8k4CxrRREtzJ2BxOs1hY\\\\\\\/9BxN+v6PM5ndzsnxLdNpXXw4Tf2M0FXoVq8WkbXHKlzvu+MCKPD3ePjVUZIXZ6vMO0cYuBC60CHF5e9MkMsZqovfEDK1NMaf7FhfSx6u5nqhaGhinWuUHMXoJjwJxanW6oofXEhurPAJoXR9EqyuNl8H+bAZ\\\\\\\/8T\\\\\\\/ElLgZGii7QoezkkJRLv30dzHNAmIJ6VNVT88dXdvoCw34MP7j\\\\\\\/snlFKfTFgfFdGWnzsmtdfeptcRrf45DpukM3YyR5h8wyO0Z1EKu\\\\\\\/auUE3E2lza6DlRgGES8nptpvG1XCFwVQPjk4jki6t8mvSNmV8Dji50d3ESkfjy2ZjwZhv8sGlo3Swy+qThKCYuoC8Jqi+F4ZmMTQdjJZkKTivHUluxOzxN1c3o5eNaaBNzlTifSEEpIV3qDdVNq0eNx81XDSqpd7gG+1eHgCiY3HPiQt5As+gWfEgm0S4dIdseBvRlw9YsyuL0Q3MJ4\\\\\\\/njzMxgdSe52XYk3FGkqDGHBh4GXz4PW6skubcAe5wCIEtgcHw15tQRthNNEJKzxKZcDzIxFDP9Ogo7QuQIJZyImV1rnggO+R8lBkn0ZEHkKpCiioaOiUU65v80PeMY\\\\\\\/XWZggII9YgIvYa3I9+P5henYQJPHH72BYpj3g9ToN6vNhnvvqhgMmWIHSUYdJEDBAv481z220yZt0qLm9bw93hZ4bQrxgCLsXATpHmxfraeBiP88RebSqpc\\\\\\\/j618ie5ANPP3rFXagl2OOTeaMRnk8c4EERBxTXnwrFEVrxHpipgS+P3hmgZKwh65ub94onApnnfPHJghVpsVcXaj9gfu8CZuYpcN72xO\\\\\\\/ht7FQP9zo\\\\\\\/4LlYUqy95V\\\\\\\/9w0euD95Ta7XOKJA54yROwdnot\\\\\\\/HGTcOBk+VsVtPiFuUgA1ZfhnnaRbJXYN6Q7BmHvp8kTZpR0LQ\\\\\\\/knVyO3NPNvWH2uN8PM9ur4J58jDwK06ke7oQnaLNyJOYiDfF5aPIptRJ3OWT6IBXGkfCXV\\\\\\\/ha0PY\\\\\\\/c31tB9SWvtzLF1DOI96jhvPpMmZomDL76fpw8ravzpbfDhNjcfV1t64J8ZO0EsYqsz+luImA6qp1z0W2r1u\\\\\\\/nYOHtSDlF28p+crZnEH\\\\\\\/vkXXoF79sN0DWyghA5afzRuF8KI8wS23jo6KusSBuje7uEgimpVKcjgL7AyZHDKPdRVbM+hjtbDs07kJvpJ8sXWYEVkLS484fDFDMB+4QtFVrtmaMgvf2wRRUe0s4fppS0MikOPwqgNemX3Xzyow1Qu1dQPmWLMzP8tXr68qPnldyS+Lrk1ShFaffvqVsXCzbjS\\\\\\\/PoaWFdcAy0D6YrLHrkvyR2yGhawobbj\\\\\\\/k69aRSUVWCvPGW3ZUwBKvNVODkpqorlbDckQv1IlGgyRyVOQsq6sYnRXLTRXHu2dQyPMlQ+WK42Ct4+MnBnJYD4vldtHHocnBSeTptRNOqgZaYXfLFXADUdZ893Z+QVPRAy2pd\\\\\\\/ntbivO9AHclxYO\\\\\\\/2EhFZ9KGHntD49t+\\\\\\\/6Udc6ICO9piyhniVatoMY6M4e3A9XyCvbNznxqrhtoeY\\\\\\\/7V657KH3sT9TpZs8NHCYMbp98Jm0Ne6zIzHCvTkFqxoWdv8\\\\\\\/eL6SdFXYQ2eLCm23u7wIGyha8+Kpj024m1PH3AZKO9p2\\\\\\\/YeFZ6aMM0YL+teRL+RGkRI+wlB4cMZoBzb5Rl3j2rOrUr\\\\\\\/fnuyBOJzI22jpBVXJDeNiYG3qriBidvisVVRjz7p82vdH5rLuCVzOZwJqWvM7re3lfisRqzp39foUkg8ev2YUwIjUFTq0C6oMGMmfCDyOvNBSiodT7sTIusDjlTO+8Z3aOgPjFScc3mETzlSTkUWdeKXb+kRUtpPrGc\\\\\\\/nfn9q5Y9OJlF63ydNuKUfz7plrDwfy8Y2j0pHwjz\\\\\\\/LNuyNSYNHL1MOR9thjpE1pVcYS1sbXcNM2c\\\\\\\/wAK5XkdQgShDQnUO2mbUMAGJ7XnIYfnOEMH0Wun4nUCYiK57kFlCj55ckuzGg3xcgnNW\\\\\\\/5iK5QzrWKfOUMp0oIxWZOVl+gzP1jeVLQT5CF65ilpR6xNnlL1VQ3Yg63CTDgJ9bDGY4XOA0FmSEHgCfL27BFSBoZW+rN3SQd9ZPjvrsDC5y65gEDCUM373jUDwipiROOZPQHNmXdpcOYDbL7pMh5Nz5ZZWl2zXKt4A7KCMRNNcoJWrPTHhr\\\\\\\/HCgsOcM49mdSl5l7abQIZWfC8VxdKVjZ8GZ\\\\\\\/+Ej\\\\\\\/NY+1VcOW6PoIEpD02MBV33VszhKk\\\\\\\/ZQlB2vXLBBjfKvVyTcASD3bcw1KYN3D6HG7OFEcYt2A9Ya\\\\\\\/bocQ\\\\\\\/SnKvdCdHmu+o1eHY6gBar3ztzE3o5eOyf1PY+sxwuMRValdf8TGQ9tNUme4xDl2R+SPs7KX8p\\\\\\\/L96KWNTAMzbKejzBGutpUZs1xxNg927EPpe0B5dKcRTWVUSZVmfGUzz57t2yz8YMSIhK38tR4erwcQ5NkGdQvsfrFKHmoz3W5RE5lBv6A3O2Htr\\\\\\\/0q3Pxb501vCFA+iPQD5+E8nhsdnEfiqNzgaGN3adI1kFOXDw7bbbX9oqh5SbyPGi\\\\\\\/LxDQwIMuu\\\\\\\/sKTs077nd3gITzmru0oDRlevCjvi5Qv\\\\\\\/rGKUlIj1T6NIMe16AgUygl1k2hYqZB5JOZE6Gfzo1ssrWLz2B084\\\\\\\/qy6SEPpLDiUow3bI\\\\\\\/\\\\\\\/mOSVQnARwIiOCBq+LL0PGLbFk3B1NCbIjMmON2fohrA7sdR7OVBCimDcX2OpGC+NVlnpIdVsCMobwe2BG\\\\\\\/8C9\\\\\\\/bNl3FADoi5Fxa7c2zyXR1GEVG3lwrinmGC+eJ9bwyoqG2\\\\\\\/c7q8PWtz7SMh9qbrlZ4GM71DnQLn3uwXKQDPKV+VYATs8bMffIHdo43nHCxYKdm3xlIk52ZcJMelro\\\\\\\/h3KqJs776uFGXDfgfPYZ1cvOvNWkDQqELcp\\\\\\\/faFe3PFTXMTslRPqZoew\\\\\\\/157489nZEIc96ZXtvnWMNderD5z0vC0FjAJlNa3GK50d04krKsmmOKVTui6BwHJ26bCC+4jD1EWP+0plqEEIHDKopdZLFJNBU5q0Bd7QLxECPI4zsMeqFvAJy6yeXWo2tqECdin0xYXtSUIIsJMwNn7YPp3dpr2afRmSlXbGElTLJgf3t87b+WRribl1hiUTZebDvMUIXXUjnX4yR9LhFU\\\\\\\/BWBufL6k4h4stlRZfg2CeA2vDorI17ETx9uJ2GLlA7s1hybGUlJArmu3CHjH+izqe\\\\\\\/OLbwPQB1rJsWfuPWpliPKfFvzVFO5uwDxdNBwnDm4OBRm5O8PyxuaauDVlR5QCg+sgJv2yp8FM7vCg4ALzBK3wsAovwXP\\\\\\\/T2DwpaFJBqI9\\\\\\\/EKPBCM+ZB\\\\\\\/mw4MPs5JvVxzVMV0Loc8fbleXWrTE5G2tDXjKqsh4yNv6G+0f4y18IlgDCHfWSiEZGOczOkeTu3iOYONr6Tk7z0GAYLmrcm\\\\\\\/6X\\\\\\\/OkJjJe2mT\\\\\\\/evrfTl1vaY9Ram0IAGei7UFQtnCk4YrjNUOrcozpsxIqRkaGf1bMyRQdlKYuzqxWMvDjduGZstWc1Tg4VbPP6\\\\\\\/O+Re7zAlb4fAr0dL0CR+lF1oOuJ5skR0c2m5Erx+WOd2IFQI5rpMLwjyH4BGGepbjy5Be8oxItXNTeD6wBGdQDjdBe6+RWwcFyznzhB4COjCqK5+1+qbKLcU\\\\\\\/GwwihTL0s1F3QAEQPcoo98ydWBLRqEZKcvSzW4AHbQOK\\\\\\\/Zh9Q7\\\\\\\/fiW0fCtaA1vIz\\\\\\\/5PX92zQGLnzNG7XK5EfPJIkllrzTGtqjKqhz7wKGe50AQERhY+PPAm+L167IdxWGuvhEByrL+wvPjBbeVaDvbEmOx\\\\\\\/AnoroGCHU8roY43A0dZNBfClfpXVAJ0tfGHqz26a4fWZ7K\\\\\\\/hN4It1IZaRTLlu\\\\\\\/ETK1UHarY+8H6l9yZo3DgUZlEsK0YUQoXFBEA\\\\\\\/SkcRJGIGDIzDinr0D0Bmq0hF7FEL7zDyVhBqOXPt3zhABq\\\\\\\/mIN1i1GMVVJkPCyrSV9\\\\\\\/fipf0QnpGeIgTYKpksZKDET97BLXuo5qY1lXtqCRAPGUS17jxux66j4B\\\\\\\/JqNj1bsk1khjlpQB6KRnOybLXcMTqSVs\\\\\\\/5a5HaAjhQcOnU5R8Zs8dDRt69X3AC+1M+aCTpOaGV39g8aztGOXfqsg3hHMj\\\\\\\/9oBA2bsDCIb3yJSq4KUYq3HpFkn6V5ZXkHhbnSTQ3oyi15QiCtR\\\\\\\/4U5jgrqIPu0pPbKUalkopka7piU4ela0IozVm8lXJ1xM2G9Z3\\\\\\\/kgARRKG+jhaJR6kTooVPXAyf03U4A7HOYd5XU5gt5kWU7pjKcH8hOXZMRqll5earMjovl1pmf9q2\\\\\\\/UXZH\\\\\\\/0CjE7E0B8K6OWmKxhpEcyG2WiKH5SkphkHXlYRON1UGjSqPXuTjMW4Nk02Uhv4Y3SvvvHPBWrv8CjHWHQOfejKAsrhHGG7euuhGQLjYSQbqfweK5wZRK2PurpaXM5nCw5Cne+jp3oTL9GRN17cHnURpCFQ\\\\\\\/Ng5o7k\\\\\\\/4hQje49Wb5\\\\\\\/jTgH4Jo0ZALrBnI4s\\\\\\\/37cQ8MJpjpscsdJx3Jwtl2mltGrUZbK12DDSantRUGqOHlhLgi2kWlJJldSCARoAGZrMJkYu5pvC+HUwQFHZcO+di73z9dCRjGl3+89\\\\\\\/WN3iZgUGRTYvXnLjqxOPf9m5hjy4Dqo9VLkY7c1eLInNajEa0ESBDXRcj+T+mggLe8zrxpa\\\\\\\/h2KXh\\\\\\\/2tG\\\\\\\/I2vSo3gOeO2IeaMpOUcbB8Z+unwnI6vG6EaRwWWT3bgqyDTXuCMNxSkQ6wnwqnd\\\\\\\/wNhNm12b7CAL8DBhk8W6M5okLn1PFy8GaJEA\\\\\\\/DrEcRcv\\\\\\\/\\\\\\\/imV2mleIPCm9y8ya7VmKT5OvYZPzpLnWRsU20OznOHvVL55YZr00dIwVkkdgJjaNAKTVU6jE8K1xEi7DF5hES+yIIPE7iWpkFL\\\\\\\/vfbDm9ZkvP+9WvhdwXlu4N9AHrB5J4p1mlTMLs68hHYqAZv8pAi5HThs0uYP2Z5yb0DMe++1mCqvQLtil70ouabptcC\\\\\\\/5v1kPjWs6Ylp5oZq0htsnsQIOgFCbbqjkwdQP1YrV74mRomob09V6SmlmFjKgcKLBn\\\\\\\/UWP1JL1UmblmDq8ICjkvEs3wpbYEK2\\\\\\\/Z2O5gl42bVM2jtgvyqZoksBNNMHuOGEhzwd2PU0D0jAnZHgZMwWxVoQcFThmRn3sRSIK7TwCJSmiA\\\\\\\/oQtmG2ZzMCZtKO+H20ONOaDlgmZw85kyB7FmgGNyIUdg5GQ7vtMkrHzyRPZgEArAuTmeYH2SR3T1kvyXrFu8gjMuo8sbXQl\\\\\\\/1b5x\\\\\\\/BEXv8JDkpupgRwW5Ia1Nfm9Jx1nZ\\\\\\\/JYujKGmP6pMKeo\\\\\\\/tqBFmkl9fcsTKaqePNqO\\\\\\\/gCwVoR8UfASCm835gO2o+ZerET\\\\\\\/DFDCV46GdC5\\\\\\\/q6VS2ETZr7MoJvG84I0hdfH6HVlkVUc\\\\\\\/aMTtB0GrSYGqNZjrsUxBbN9nnp8XuaALU\\\\\\\/LtG3yTkUxlPyaxtjD\\\\\\\/Yw3\\\\\\\/LI5YTm1jis7gTmztLXRbO16811JnTKqJMaj5BiLM4PhMqFpZWWYiKHbBMIHbCz6SjiMQ9hlsI8WevFQk0K9vmJUCSG+JqQe0q\\\\\\\/ZKN1FHOJh99kRXvM8FFpVGJNAFPoGy94Ah1rm748\\\\\\\/IXnG3DqgKLIF2KnqP\\\\\\\/Bx+eisZdAtDrbcudBynaHLBBxigLqnVcTT+fgMuahCn1e2zoJgp3RiU2c9nzqGffaFFvAuuw448F1nlUGRxxNXPdT8DMCj+W7996o5PCj6VhtMonAbaVXWZmfhFrIkXjuosh2vv9ITciuY6uNJlWhKdBPrReQo8lb\\\\\\\/7d0kDy7f55t\\\\\\\/nHmkWzxjyC6QWpVWxnhvD\\\\\\\/xJ6falFMK2Q5cObRVERfCYninopwQndGlWH+I166XdNsaDjyDvNoNGJGo6Yf\\\\\\\/J5Kf5UIFUwhYjVqAQIMP30tsKNL6ZMwyyw4p04P2bSMvHDZMkXj+vdQXvh8bz0UKswGM8CYtsVpVAgrlDK37wPzkY2gxYORJ210BFwXoiLiT2J3fb67opeO8NnEN23DNSwfjHoR+RqhMhRQlcq7SHNcml5uFVLncWJJBg7Zsv0yr2DF9vDw2je1lGmUvJPNQ4Io7tGBapwSkCiiGpYu958YTUQc5UZcHJbgEmv4hrRGBb+7Y7KmXqJgRF5Hgh8ItNJF8GyXS4jTu0JghnFUcj0S5Q7zG2Lp5wxj6mQT+ER5kvggCFiUIJnGyPIEnSjQXwaKCr\\\\\\\/7mRuNrr9ZA\\\\\\\/kzyxL9hEZWjKk3YwOCAdceJA\\\\\\\/LD1CQV6\\\\\\\/s3iSawDx86nL32zaCSzNhOhrOEud6Krxd6BZ28CmVEYhc1+UnmiVuVFImc+0pSvO9NRXwg+I7ZDGKwefb+uXcvIbqRVE0JGZxkxSoEDuel1EsblIWAhMKuQ6SPURQwKH3rWEn7C25k1gbitUgLaS9wE2Dn2H7b7yXzPc7TbH0ULcG0gs9GZHyNbzGgrFYVIzsuftwiy6JqLH7CEvIbfjyUnTWo3gPLoqWvZI+taXD+ftw3Rb20YErZ82HNk4FxUFTIWkSCnTldU3Ws7vUYJs8rNISD3BueDJ5GSkI3n2\\\\\\\/Ou\\\\\\\/zXvTCLJwhjWFhnC4Pec5FsnIfdQy+T6bLkHmnNRygjykTNqC0FJXnQbpELovn762MfZ7EE2eGCV5JswiYBFo8F9EGUjwI3FY7WgY\\\\\\\/b\\\\\\\/IZTEdWw4Anx1iTNkZMp\\\\\\\/vQjVl8tO7EvGq33E6Q1ewOX5DsjH+LV5cpwcoY9qAEp+o5UVycao7WYvO14dkfnebCAW+teC4KDnTnMptKzbjbkFgDCIy+Oj5a+MS3BO\\\\\\\/5ndG2wM5bBv+v9iGUAmd\\\\\\\/y4R0Zx1oNAYLbmQuBs2riU5KfFEnRNxO5AvPIXI8ZmES8CFeseohg+k9pXu1fGZYq5mfO4Ry78pBtpO8A4vU9Tv7hBs0gc1zZ1qoY\\\\\\\/QzOBkJ18H6OsAA+gq\\\\\\\/WGdq\\\\\\\/U1sBOkm\\\\\\\/Y\\\\\\\/\\\\\\\/FadOKGZnbn0XP+iU60TEFlkCPmyrdi7g0k3uppQeOqGgROrGbVtCMcqo4dIQXQa3K7Xj0b5haHVhqQKv51zUMsJ4unpIJc9yE9+GyZDKmqSWVVD2xJPYEd167ED1N1Fa5BUBijuI0H4lOeKCSTE+UW5PerU69r+CBqpN+g0qZBDa\\\\\\\/vSOUu\\\\\\\/joP+cTOd91uMa97iDihKu6PMRhtfpFFGp\\\\\\\/SbTGAGcgIR9MjtvAKtVmVBbZQB6nJWunxuf54kRqXj\\\\\\\/KwUCgJdbHxf2gcn3P18\\\\\\\/5eAcQMvenlpczvjjU8FeWe\\\\\\\/3kvfns\\\\\\\/Xyw3BrFl9o1qQrl7esC0ajkCYPQ+5DEo61Fx+xfYXrUlf8uHXpTBHZxfVsZhtz801xXxydPIz\\\\\\\/CPECyoVly6alwXZPZ2FlufaCGBcLeFMiRRTriUttBbJ2G52t4ztJwSpDBuxRIs3P2IxDtZl\\\\\\\/HmUzZ5z114ytl\\\\\\\/\\\\\\\/r7ViEIfZNWbhuNSyGfs7fMy3CGcj\\\\\\\/QVbZIivkQDyYS6LiUYgsTE0tTpPsRP4kIyYUZ+SbahSGc9UGsIXdemWD4CkmKI1T7JRZ4cCfQy79TYZLDrTujphlJzl3qlxLjfUpHNAoq5QaMZyT\\\\\\\/NgMH9RGEXYdReq3dXZP4QMj8\\\\\\\/6l6Ah0xaY1TyU7dBFCMVeWlNJcHwDCUqQeeYAcTPmi1B+zR7eoKfAdfjl4ypMvWuRRyAm9BK8jDmWymWN1NfMifjiSVIbJ5LrOaOIfV5JnwAs3gJOp+zNFPTQ+kRiXZKq3NeiyANNdVXy7XjZFLZH+jTxm6qJmc18vQvhQBxsYzmgllH4lU8rdsXOonV+LI2ARWlk9ms+WAJu8mA0IR0+SMls0da5Vnpca\\\\\\\/enBAAHXvcFYuv2rxgMM6LATr5cUtKSS4fbIZWor\\\\\\\/X7yg\\\\\\\/A1g6oK0bZwky6uOr2ikF122o8tM8Qvml3LVQMX5qtihmfmpKUhFTm30qJ0dqH5l9JfIrSyGv92qqr5cmgl2xWILehkvs2C+IYe1mYWtSyIMTPI9\\\\\\\/XPCKqeyQorOBtpSnW1eeMULbdsONCnz1SzdGeui+XMFyeiv8V7j4G4W59vlW73EioQ3JPwLxEhnQi+AKkbKTYBUKqmptLap4Z0b3XUS+oJVJpzmKmmtQ\\\\\\\/3xWJb6WZENHhK1JtiCrscK5\\\\\\\/4MwwPXDbzetRE5\\\\\\\/ZMxkotftPOqGcpqdDCAVAQz2WQyhsHWRRxwBtFs5n1b3qE1Naxzjp3ZpthC7Fodv0V1jW1cA+tXNGX3t2vAevLQ\\\\\\\/o5S1OlOTx3WtdkUI1hEUEycBdo7vyfw8xQaHawhxr47ZNXHvL8w3QqV8BQTJIYlTzhZ1yKuJPZT3k8a1MtZ3S8bW7HQbuLSIfIddVT2xWcrpJZKpFj4egGwczRKcGwOknl\\\\\\\/CWUF\\\\\\\/2otqDB77UWaZ8DMgmjRyiblSLukcXI304aw5HmGMgmcpr88uaGv23n+1oEhkRG60sBN6IPDdLWqe9\\\\\\\/pZ1Lalx+8SGpn7GGXso3Z5v187dZWRhtrjvoc3HfgQBeYsrj5cz1y1RZ+wmlBJuKT+rBUx1RRSwzQxzs8NsqfUTHt4z4PqTuR\\\\\\\/Brsya3i04KYEtkk3+JfkLiH\\\\\\\/3O7xPFMmTqMosu8n\\\\\\\/PL1cAK5BNvRsT1l9U7JbHnvckwUqdGvtRKQIwd8bKFYl5oAGuR7fDdR21JcYG1xaWflsW2huVqRFT\\\\\\\/l7\\\\\\\/Mg6QsAs5FbKz8jjUQImU26mgRu4iaIpwIpSL53NB9X2x4TYF9Ojrh7w8vgrW+jqmCM41hegg3eTTdd\\\\\\\/CzXjEZlZt8ELZ2\\\\\\\/xjKYmlRoh6xQTCEW8lN7Be9YgbknLq+3vFb5xjayF+hK1yEVxv1SiFWD39VqlZnEgEckgKghCUBpn6K5m2EMULgkSc397OwTySy3dlQOrQhLA+M6NnwcZ0Znb9LMyhSOMHvnZVEGxk5mTtdjOFFUFCd9ZhR2pGOAa7sIhY1Wm4Oj5\\\\\\\/GJ7s+ACukn8qCzh8cCl3\\\\\\\/gkXp9Z1J4GCBQdkEE5PrluYj9lUgCqQzH7hbxNVywwkoupzDabdEuFe+xTGg6T4DUsJPli6NCPIIuvHcSRRU5I9Al9I+JOfGNPRV6GeZBlrchaMYg5M9AQnRxZ4MphPXVXFPXwoIKMdKcB4Ys6fNOvEBKW8nVYQiuf\\\\\\\/VJn+SOyJsDMCJKbpKOoHbuq9xxp2gCAfH\\\\\\\/l\\\\\\\/D23+SOGNNxG1WIxC+QrZKY4uoOA9jLAlOROEQEIYABA+Pge4m7gquIpv\\\\\\\/HEsaxisJ1HJykJagOmQNmYKxWG4cxLPIUWIlIMTSp1G1AdAVGu+Z2Wbb\\\\\\\/dtTEsN8y55AhS7kGzT6gevtxFwEdZHjkt1fDGpkFUMsFz28KK271I3WNOi8UbRqHR7iC48V2ZF3OQ+s\\\\\\\/vGV9lL8I4iGLrmFtfjDbeayOalP+g0qLfByrmPwT0E50LtdC6LqJrlzXmCAlnicZvN31Cy+Gf51u9XPRyikjyvHi0o6Z\\\\\\\/VK2rR9xRCTyzRgGJxiUOPSLl6DgnJ7HC+sBn7GPjloE5qfZooiDAmCPbJvDUrE++LEEgsbHRPXW0e6g7tWCpNwzjNe+Cm9kpOHzkE2yZx8ZZYpXi\\\\\\\/YIB3eTx2FVBBomG6HxMP28z64fX\\\\\\\/14CnKxb02xz0EBUFA4P4QXnkD9mjxN+LDt\\\\\\\/BSdRxDc\\\\\\\/m3xV4QrHdyQPXzN7t4ffR5+I9a5569\\\\\\\/cGLsZuvO3IhFC+oo0\\\\\\\/JJt1aiE3lR33yZg2N6F9w5xsHep7I9Abj\\\\\\\/s4Tu7utPMvQ7rJU9HLz524f0XgabDnm1eUyiVMcHYD0BT5AOtAR7S7jTHyo+4vuOVXQMbgQhMJzFvmRXU8aqzfTugvx4xkRi\\\\\\\/0BJbbqXouBrumcBCNQ28zdZ70XQW5axfEQpmidRgSQlap2UBbGcf3lxiKXjNShMTMI0rJmm9GoDnQVdPxsxTNAXzX\\\\\\\/RBgP\\\\\\\/70NzQh7w03lE1QTwK6Zbdbo5gIq8c3egugwZDcnXtcvE7Yfiz7WOinDNO1donzxrMo1yDDoosr\\\\\\\/KaCzm4dLTaRDv\\\\\\\/CK5wXHGBYVDasidWAS\\\\\\\/rLrP5XMc8rl+EAfa\\\\\\\/lLs+f8T58lj7xh4llSEhU6xK7tJ5plj4wp0UVIczRVrpXm\\\\\\\/jCSgU7s5x\\\\\\\/M+NlJNvxdkDaCpx+4Gmyw1NiMpoJq9xU63Op7Sghuz0ljfmMGtPH1E06ImWZXlYiOyy4iudyAAJ9eryMDvBso8D0cO7Opm6IvK\\\\\\\/4FDWVDWIBCR9AlAdhGlqS269P4wugxsd+qOD4PyiwjK9LjMb92tGQhrG+SIcGWeWJmGzWoIiU8ZR2q9kK2mdztKl9gPEvC6WNNo63naa7RdDbhwltzmyr24ZlyMrOh5eNKHdGLpguGWYEj8IhSjKY6lrdGD1BMiEB70reA8GDY7s9oHoZMBzEFyZAqZL9xU2Cq3Aq3Kgs1KePYOEXLBRZE+YtOZyzp+8icXxLMQ6db+wvpGaQ\\\\\\\/ZBrwnmcPkQinlOBCpSJZjI\\\\\\\/k7Mhd45mwqNX5LaLilRDInu0D2\\\\\\\/L9r\\\\\\\/JJz4Ce1MDeObEQncqIjHrvJZN7vjaF\\\\\\\/CjEZCGE0+D+1NAsfw3wf+fYP+vVHfl7eh\\\\\\\/KlqL5WzE1SKwhsII9rbJMn5cQ+cl0I+dTUj3nusaZnSri2EdPyeCEzs63Pseq1MRzPR1gTAmJ7HYVKQTcCSndEFWupadj8ALJU5jrlcoErSzYxoFnUps8oBMK9hrkxtAhNWo2E78CAcDw9QYYVY\\\\\\\/\\\\\\\/BnjCjP8GXIdrU\\\\\\\/7AjwxZY8yAlk7zUMQ0RojbYgAAQik29IQDUNKl2Z8OUI+0RM1Pd0Ucd4aTVRjqn9qkFAKZgb7viKL7Gv45LZakRWYxE3FJWFaAsjiFBgMsxuKrkeXAq3lkradj3XgD+2dMzQ8ZMFEXDGr5b9vnHpOqS3EPzSNNzuq6NyK+jUw1prvJJKUdaqDg\\\\\\\/JDGL974kCW37WAzFxtBKjWds9m+NyqZwhM7wUiBkBUsOB40fE4ZJCPAV0SMxhh9TTPUNPp\\\\\\\/5QyNQoRgSXG+6ggbXwneitz\\\\\\\/s5SVuVj20FaA3iB93Qcs5DlWxNJqKWP2vfEm1dzcBgQ9v0r3o4WrfPxqnodLWbYjL\\\\\\\/CXJk+IHV0ruajKQjdc+cDozOEIVzkTEXpOr\\\\\\\/CQqusoWqh53tas3kDYoYR\\\\\\\/Ow+kSLlDATfro2XYlQK089xhxoBwAY3qcJ5rwXLl6fujEwsV52Mm12Ceyeli5\\\\\\\/\\\\\\\/uhr6uFALl\\\\\\\/j1gHteN0wgxDWAF15Oq5ix3pXb7d3QE3cWq7bP0iLTsoC4F\\\\\\\/03Gecjnp0RYWCVxP\\\\\\\/NvXZ6iRptokTobw0lYZjt\\\\\\\/tXSSVezgdFJgpumqDB2OFjBFVbNFUgDL37WGX4YR\\\\\\\/IUnycLIgDGw6DSobyREmjZgr62bgizAz9NYkGOzbIsUg2ki9nh7aXiV8Hv3+eaEKSfpeQSVje0McOBUpyPhinMCBdfPpR4bt9VFRSwMzniqmap\\\\\\\/deFth6oLbN9uGcqQ\\\\\\\/FN9jcjA7faKRJAFZcKV19K2cReMyAsaiCDcozuIRrzRfCvM2uW1zlXenor8TDIe1YclrEKc\\\\\\\/CknRDDMvBfCHzUEAe4p5T+TmR3VvaIA0qOQkCpq4AXq0ShICHZt4CN3CaodqCrSNCFPrq+MmMWkNnjL1C\\\\\\\/WAh5glgvq3iD6+GdgiO\\\\\\\/gN+gOsQvi6hLl3Wq+usyaPfFWGZMORM8KnuvzDQy4c5LtTbTVgxe7\\\\\\\/I3+Act++arZtvxuqRfSrkkulsbPtOf\\\\\\\/HCZfKZKiNnoKYR5qF4JzwY4oSWBObPVaAXNJABGxLXgifB9+vY0lkmSZcrAONRDMIiE1Znc\\\\\\\/w06Xp2BKBft+Oes1AddpLMxkxxDSeNf2BykJHjM1i9LrljTu6f5wiXnDNPgBS6xOoQhepX\\\\\\\/MgMWKOgekhCltgsWdGSqS9wTd3gY4nNCC1VV7PwBPgBWcpVGZ6QKvCoWvf+oKLTyr5TLhr0DbFsnoQ2l+hbc8vkKwi8kpy9\\\\\\\/DVgYO3LhINbA+A3o6ZM0x\\\\\\\/Gskau+WqyuSSGD7zmV3lqSA6N5NCsTZVIi+Ghz3l6HWgdakSbemGrwmhIpVeBuDFU20FIkoY0m5vxxGK4RsrTuwuKOjOixaHZCz5rfzsjYwhkrio1NDBJ0e56yc\\\\\\\/3KUIYagDlTIvjyFmGvPs6hY5wes39\\\\\\\/RnRqx6K1SBy3qUauMaPkZbdZFM0PJOcSOcflmHjh2D5QIJPREXvNu+tXVgQSg7X2vLk8tjnfaWF5Pc3zEoZqplBTQs8JobRJMIsFxBjd6WdDsc1mJI1Co2epL\\\\\\\/r+qgR2EyS5BGghk52kEzCi7BNG6FjSm1SRXiE\\\\\\\/7O2Yxg4D\\\\\\\/96Pmrin85TbJV8ntkWasSHZDOcGR7Il7Dvo9EHsjI6j8n4074PfWQXx+5AofbGsHC1VhSyJy34ABWMsZFguawoO86GUgWXBX6kT3yf2H3zyE9lrFxjPgEh5Lu2ZbesXErZzZms9turhOqL9UAXtDuyI6WFQ4Qn35211uMtEpHqqfj4ur+TGKNt4doVM1OxuMUv1LdryBW\\\\\\\/ATQUEZl7p8s6aN8fNxmGwzOVPfPdLnqdDF3NE1ovmm5RGDryrQDbe\\\\\\\/8\\\\\\\/9u6AUzMsVKYht8w\\\\\\\/CAcTitBuZ0ejgB5GyYVZ1ah1q7vEeRQAFHDsXjTpEFRPiO\\\\\\\/7WWed5sHmDbOvIujBs5XVFTMfCwxauGoDeMDI2Sy5T\\\\\\\/iEaCzTP1gmlw+sRd9RJgANV8Kl0l8v\\\\\\\/\\\\\\\/XTVFBQTgs9uJE8i+PYzlOVECo4O4zp2p+7C8hICPg00x2jNWLeXzF3sc\\\\\\\/SCmbEzqm4eAP7WcB+aUP7wab3GeUDuwHc2a+IM15fWYYjeftBLM6QqroS7dE8dNVBPouajUTN\\\\\\\/IEXyJ791aWFVlDj9mcPmZgx\\\\\\\/E2MWjwC+vr54Tfz6owEnnyoWHQmtP\\\\\\\/uANJjYlNcHmrS6iNS3UgCH3l+SFMRcHNOMgOc2iCDSZtmuIA7hedEEa6uFc3vq+yqPdXLsLJQfgKzozyrUCusrcw81awUM\\\\\\\/zFREVVMAzxcNAYiMyoHB2EwRYaYNrJp92uQFYSpFhIS27vfvxhoG+iSO8IKrkivbORooZQ6SbVuYIGpy\\\\\\\/ENIgq8VolUanuLI6wrerpD\\\\\\\/2oic3W6RFEIaIKHaR665tyF23hS\\\\\\\/MZGeaoKfjUqGz1\\\\\\\/4SZSk6AxWxRDfsOswL2HVgjYqjHWRoycD8omzXa9LLNAeea5W1xtAx1QfSpdB3vwJCA7cyXIWyn\\\\\\\/h+kWrSO7601qnsn6IVo523Y50ZekyXJOfkm0lDnA6up+51qcMKRTPdJAdlvTkuM4JbejbBrLLYanxJSmhuMmPlm8iZZo43cUNxzBbjHZSfb9beVEfA9\\\\\\\/RhjJjHSBZ6X8SoSEEnDYSApWcMj1Y9iWlBj2BKsW5CHXcJnODPv3CUnyFJtPhfJBiazVOlOiuc9sdp4EAqdezww0\\\\\\\/sdv0ILt+BxEgsmrtT5q9NrcXlblvr+CwcK\\\\\\\/ai4H7pwBoU56kVNPwSmbcdPmWXsnGZI+kF2eREsdBV0t+VUyATF4oHjmpVnPabOMU3qBTpacU3I3L8sYhgAh\\\\\\\/S\\\\\\\/LXcs0ZNJrNHH9X66GSIz5IvEI1NrF\\\\\\\/cXhuFx8uHbF0jrfaLn+lZTv\\\\\\\/VDpwW2H4IZOTWEk7TA\\\\\\\/AilJFJm84Q3y8v5sBbErDznO0NoIh4WVaAsKcG8EwDiQ5iGGv7XMhDEdp6pczupSWEgOUKyP6wW\\\\\\\/PvfVJ+zW7JRcw2sy3dOl9d3UjStTE4vr8O1Sl2\\\\\\\/kmsF3+0hYC4\\\\\\\/kf32gZYGq2IMlkK1LNqI\\\\\\\/6qcvOLhYSDIcxcco+rVGvzTfwvSSNuwMg8hcKTU\\\\\\\/IXvaP2Uu8aNGAOzmv321HycjWmBYSFH5meeGylE81cQhdBwb7BlN35fFUMwn4ZA3oabY072ijJ+S9wsZ8EMWrLVFMUL9rIXul80IdG6xg7xuW6rf9c4vKXKW5PnYf4GeccIAXiTYAYtddECAoHVViuJ5+GRHCsQ+L\\\\\\\/cChchbIJd4G5WJAhq0jGXgroUD6gyRcXq1XKiUtxGdP7WTO0skFiHSnIngjbjzB8626f62emmm5wed\\\\\\\/PFCD2qMz8ImCdYIT0CLsveFjGMIEg89kc8rSKtvEkzoGUGycmK2K3XBtqSqHuGRu+mlSoj\\\\\\\/LrXGkVrZOylkEGkJVI9qrYgENku27VslL7A6mgz4gTE5Z+gBdMzy3bEpFqJsun0fYGFnd9DnUH2IZrnxHa6zJPrPt1edj6cZT37DNhGF2RcdII9FK+8b4+2dnuo0wofInkoZ73sg4WcOUDq\\\\\\\/GDjEiRi38BzATEokB2qvldvavz7PXC16eqDW1EL4mnfwTBoEpqianVAXZAsmXI0qvNmRdA+2SAiChV7FZuCnikDzz4jWYJCQTg3NDZjjqVyVYMa9SeufIteTzeSU8O7Nvu+3oAn2zouqw+Ij7KRX3\\\\\\\/8c6hHgYgxinkLwHA+vyFaTneI\\\\\\\/CTjltouVRnugpKBdSq8+BTfB0+yXtQYiztWRglyHorZpcdnZrv65e4Q1mMzdphrIzNt9TOalJQGNmGXjSrc4EakGJA3VeZuryyFu3ppYGzayIJq\\\\\\\/rjfgfOmbWIxVVy02oTkI6r2OIIe1RASF7Y1Udvm+6UJY7EHNOWt8tNIG6h4oyIxVcs6puZRpgzUqpE3luw6i9m+qbgXd6q9qNzXv3bVv7RY8qUMOpJjMM0a82jnyZRhYIKxkQ2Fj\\\\\\\/wkYD5coly5HZXOuv2xQYyNsrwtZDU3bEHeL16uVCLZhQgI7lK9zpFFC7rou1Jca5pS5FxniK5N1Iq+nQod2ZaYXjWLVmDPUoWP6Gz21qx7Wh4gj1tD38NpRVbaMP7MhaWclG0ls6LVFLT7hdvHJ+BFy9rHgZKhNYQGRzt9RCy631CmHunA3k4guw1lFchxFqn4KW2sZzqqGJIRXuFMKFBKfy1343ecSw7E7ko3eGYH+RlpHX5lZsTCVXeyqny+Ww0mlle5d9Cijvo89IUbj\\\\\\\/AFrXs2OyKD4xNPTrWonRjvcJePES4u6IdtkQQAVzH025YgfYj57IATb+1uIW9sNpSJi1uXCqhlLu\\\\\\\/MWYQafgZ\\\\\\\/FvfyFJE6\\\\\\\/pdYeKLKfgxP9D6aRbaqGx42vD53m460fpSnN4WBipSwqO4+MynG3YZFtUwhvlWN2xvRMa+7cHc6YZVFG56ifbIdDL4po73Bs7ERBl31\\\\\\\/nS2uP+knMpiZJ5CB6CesEB3hCnvokYheyGW0sOO3Z57IS0PStoQ+MTRs6BSgYpTenEx\\\\\\\/UOT1l6gkF1i5aSonePYDV7FPVZxNSkwm0jLDl\\\\\\\/fTjHrV5rPyI5YsBQ5EjEHdDBTUj+mOHztKX96pPaRbNa7iomNfJlxhaACKxriPTrq8Dbar6\\\\\\\/XztIkF1eYpWEQXziLO\\\\\\\/2j1C3e35sXU5fwFTdS3H45VhMNZ4xBMka9++35gFcwbIj4wBYElCP7V\\\\\\\/RkyuoipGiiZrQ\\\\\\\/rYaxpcTIRAT+uSyiQaYDmv9Y3QAYKplyoaL5oUgVifEe8korCNMLYySxJraAm9ChfVDsbqAak14s2V+N+k9oMfPS8SssE+u1U858L5gxKIcxHH\\\\\\\/V1Ug72IG2pGw3kGu27l6CX8hzqaRM+aIKmQ8wm6hDZa\\\\\\\/+qfEP9b4vFlkAv1am9mwE6XFB0hGiQiGDZyruLXKCr7rHMCI3GthldwBuw5ciczGK5Qq+NjGkrOjG1mD4QFEYnTXw6HQ3YI8v+qQSnhUG9dRQGqVTbyeUn8rX9NsXOL5FNh03\\\\\\\/6Jy9\\\\\\\/0Rfcw7OUh5hjH7RJvKzGQW\\\\\\\/gBJW9EQlhGw2rmybMDoIOSMcSBbzk9VkjisY9FNu5C1WiTTNpV1z88Vtc0SopLcxHv42uiVMPGIlma7fih7M3I6nUDnITVAoR7duTlcdf5yz+QJNEQhX6s7x7PbotGmICGgIMy2nq5Xc7mlfKygPVdAKUoz\\\\\\\/uHR+c1no7z7M+WlnlX3yDN6sCT3AMI1HSgr0CfIcLu2l5n8kVdrMN5t92wQS3J3NXkacuJe9erT+\\\\\\\/F\\\\\\\/ep\\\\\\\/IcO8ZTiXppJvkxcdPs0RRh4t+JIaWBsR+E3hTJHRzW3x0sksD12U9KehYrS00JRDZZewPGQDR19UIo6kQDiz97wYga8peNt2fr73Orw3mjpu\\\\\\\/C6LrmpR420bpf4TdRhKGRwzAXPj7mnKCsUNU5GoyU+g43B9OrMyD1vasC\\\\\\\/5s7\\\\\\\/E20lcqp+69MKGtjseLFgfY06J8mTf4jQM5KuNMeSmQoVOMDTNgtTKwnAs\\\\\\\/TLlbdH3\\\\\\\/yXgStHL51j1p6tdNF\\\\\\\/Ek9JbmU7MXWWnJUCgDTfJeV6eb2N0uLD4K6TW1530dIDVDBnPChHiUJyufkKOhGfKxPPLATpa3VMnDJk2xSOLi9R82RZbLMWF3\\\\\\\/8h04iIYmzSuegUXQDJQpIXLei2EbmEwoQTFuQrpiyyrK8zrLj9MyjA9dtJpmYCQ6bGBUNI1YyzmVeDkWWKKM8JmIXWFSh2YiCON6S29m4F1wippf9ecKMP1cZl8arfChXePxrAjNbzAFLFrg+KsLTuCUB42om7nXBQkxmvsmwOu\\\\\\\/9eMcYE6r1FQdvNTVDEJ5efNxaMEaJU\\\\\\\/Y3+wmvcvdGTcnkcTn3lyIQmIh8VyQ3FC344pXvisWFEHGWVDwm0ps0lnfsmCf71paZfrxwPbvFNpQA7\\\\\\\/m+kyfqfyrVFZVzkQD3e0OPRo2PJXAk822oOypVL763o4sBpENhtpQt9\\\\\\\/RpFeErDfTWqGRQxFbEpf7qLiedWcvnbx4RQFdfvUaz\\\\\\\/Rf+Kr4tk6\\\\\\\/PuT4qEo0Z\\\\\\\/jPPpYL2BQs7yL6uFr\\\\\\\/NMAAPIiz8gHveMitqZLDZEs6Gp+jb\\\\\\\/tQTDwSaZVEBxa16NhczO9BhYHYP9821+zHZvyTJWO729k+kUnOZz+Izoekgy3tmftf4LchrVWeHPcJR6B1AsSeh4TUUfcESf42AxOoifIW5bf14RwrNlSjXRRjxIlpLZUvV7J7wRu8Pg0wQad5YpgsRiimhI3lOrEd1vxU0\\\\\\\/ZIJUa2ebM0vuxF7uvbrxcAlWROeu8w79KSsqg54OquK1Q5dww6lZXu0ESm9HCsvksH\\\\\\\/ntlf6Pe0mRmHzZ1NWMSHFCx1qPhj9U2IN79j+K1jDV58Eeg7c3GjgQHFp3TNyEiRuI2O3iavKzh6bitdYv+WMFoPyLJnBJvlXjYk8Kf\\\\\\\/eLEl0O0k1TMyEr\\\\\\\/An73BBBCT17Jcv54NUDbw2D0FejU6C6ijzNUm51\\\\\\\/F36UqTzbtJg+3Q0tYzYbVqbpc8y0XUocUugvAM0vz2DCvhPEna6VEdM\\\\\\\/\\\\\\\/dlRW4iQFyKWKYwNxnBG1Hfy2YK2UgW7cwmfwyxYbwbnOlx\\\\\\\/pz3ZRWtgBpyTTD8WwnnLHHy8abXQx8jqM1pVMT12wvXK\\\\\\\/cW0NyYYWnA5RUE8iYJjPQcEO2ynjq8xBFwaMNorNtrSrdrig41msLpY6tbjAfso3O9s9kZAANRjShX2xCvwyH5ETuNee4TKfFyI0Ij3GQt29afwNDRyQe0+v9soHc6QwHWoyUnc2KD2U0Luza1VyNfW7rs6Pm6h6YV0m48xdGChWhclWTh5uiDaWYxWV2Nod0yqQxPC0kStGiS3lWcq8JKrwZLs2CMvyJu1ZDLRGU68epvJF92t\\\\\\\/wsVSKwC3Hgxw4I9X7DynizHplhDyWwF3u9lP2xKtL5Wo0xo0TW5+X8ZNiWnauDwRAKXUVAC3Nj0ByVrLNqOHVyZAPN81LE5Gg28lSSdTEg0oohXgF00X6JIVB\\\\\\\/4\\\\\\\/oPYfx7Gyxo5YQk\\\\\\\/60NQK9iwYIVh4EAg\\\\\\\/n9sXP6dYzT78ofrYkF2sok5GabVNx6IlWPvIEqrXU6yGD1u46HZ8V6MGIqK4NLOlKoMwMmEoyMWbfiAsCJHujuamXRTGf8BED4gwaPLqsxYAHdg9LzCl6LosLcJUMtAHrSm0DHQipJQyNguzCEzlesog9C\\\\\\\/acF7CGuS2N6x99yRwuqKWjYb+Cbvg6ltczT4ftvNMQvoInm9lVaViIPjMisVxjgrUpfyp2pLkcUwuT8W\\\\\\\/ZsJqHWK+M+EEWWyTW8u6SQAzFUApgB4+1JG++L2e46Fns43vSTMCP6WEr\\\\\\\/GDwro\\\\\\\/y5y76aeqZznsU\\\\\\\/wB\\\\\\\/9K89Va66c3NX\\\\\\\/V+B5hOOQkxDiK\\\\\\\/UgnMaw\\\\\\\/tRwK8osFqTthRfMeriZ+YUYH6LBjyxz\\\\\\\/8901y\\\\\\\/J2J8ZRLzWwWEVvERzpg2ZXbiSEjbbVafxtRz0COdOaUY7cp7XeEGyoOb9T9qGm9fqJvle5a3EdoJ4OZ9XsS4i4209MFglzTORe1oveYAdqabWDn2fOuo4siigHrBfpyePXr4wkCerDidNaXbmAyX3ifXag\\\\\\\/CSD9dYPy5\\\\\\\/kZh0oTgSg1Ywl1kmG6jajVQEM9f4PuNVqqjfhWGz+EB72KBeHZI3beFNPsrcOQX0C4GYJNiOHDrt4B5nBde0FaGkHdYSYHfMYotMGq4YtXKJHM+1EeswQJKG3cGP38qCq64+Qtw9MFhlVkv96e0aKbTVWWmnZPvdJhZXdyXWxYL1OasdMeXjuj6oFSAB\\\\\\\/+uKAXJJah6Fzw6Uub3khQHs7lHCU2kUGwFLPIcUBcriEBmTdLA+pYyNDbzEVPthHbJxI2on7nY4ZMR1xAUAd9ThkLCPMiFc46wZInaX6HybjeF5mMZJchO688SkggZNs5OtMRTQfdhtMJhJYUpucpEUF2nphvcGxFCge0EgWCeOwZGjcv3WTlr\\\\\\\/xY+SBPKvov5fa4ZaTMr1VJn+GkE+FinFRdSD8mwigLfwKkPF0qcRiaLxjpkpj9LasNeOu86u3wTGXe1jtgJIJT3317\\\\\\\/wk\\\\\\\/NHAMUebRXWdbixBmbB3mqOKx0D5Uv8\\\\\\\/7PPUN2T99ZiA9\\\\\\\/JCzTdPwYJoC\\\\\\\/9DCofNQKNoXiEDzCx+fh1w0g1PZPbtO+RbWfm4D6TrOYf41jpfqH6JnjmfX6SKzekmBhYQ8m+xkqZPP8Kv56tu4LbE37KEUxG0jM6UI9Pqe0I2vXL2P7Fn5Kfc0pOgUQuRrCAOABixUbxOGdGHeKzGSvchLX2aG5YzXmmLiEnEUmjaTj+Lq9E1e7ZQcQy2\\\\\\\/c+pBJR6e8rBaCQnJUXGymP1jyN5gyiMcDd0J++Hn6Lx1iZUSYON61Cet3UE2axV\\\\\\\/51w0NC0vrBdB35g9ZYrfwDnkGk41toLLRs4h8l+ZefxNNnWvbeRuU\\\\\\\/dKBHKNvwtpjBMAWldViEUuH5O6JFQFa\\\\\\\/mGt3Fr+6ElanE8KGWOm\\\\\\\/+worsdvDILdRyYPyL+8\\\\\\\/NX2p3W08J5yYHVQZJR0EzOD756elnkkxdBE5fcd9knmwAGds8zEWapSYmVKt3zFRUPYCmCrsuPHpv7pgvHQEt6AhHTdCgxmdysg2m3HPHJkAuOAKYX\\\\\\\/hoz23qFVEKqBpQXbfDtSaQwDkb8ep53AkMy9N4rG4sjJtEwcKRimuVDi8gx\\\\\\\/8xvG\\\\\\\/xntyJAoHVGEeMImFJrw7WnxUuqH1IauoaR9j0SG0g0c3cWWlLsPjoJgsDw3O+h9JAnOTqyjSd5aY2IbS36Lvt6XWhmJlQV+z\\\\\\\/CHw64Fdr1cEELd2Dp6hjVpNv6wz8p+x0YKKtYMSobwRO5z\\\\\\\/qt+wxRXV0d94RumfsJukyipIT7N88ccwq9iPHEIU1N1ZXvBr\\\\\\\/5buZb6qJXHUi51JJIArdnFIwK0XNOBc4DJftq6ru7ojeJcefjDLHhI9Of2ZvHeoZXaVUwAUbX5bele4yD+oYpbxrim6vUTVOs4vhuSSBKEgko6tOeuEBdRtMhoK+tmvbN77T40fD0mnlJa\\\\\\\/exEvs5S6F4moHgjpQ+wxyjJ95fUUEH451N9Af2DvOIzEezfh0jODqhgdAwtl\\\\\\\/BLU7U4b8QHYPnDDbJyFWPAuHGwW5GOpLafuFT94pwP5+IErPbgubWgT3JH43BZg9sMYeKBZfNAj91akfm592qqUReBLLzkB3lzvRemyV80lltbWKSR9jdNiP5tHGhfxUu\\\\\\\/pqtG8NyJJcS\\\\\\\/oO+J6sXcglUjK9xUJh1MkPozeuXRFUI7tYE7pdJkfbSRmc\\\\\\\/mpgLQVUwezJBKdWp2fM8h0fS\\\\\\\/Y9fP+omik3h0RH0hDjAsFQmul6dzg8N05+\\\\\\\/5r9KKm\\\\\\\/PSj1p6\\\\\\\/0oYon+sIU\\\\\\\/uTypPaq4X9G0PKBBhGv1UL9nAaTWOs5uty6RTxn4ItxtJO8j+LSCK6XCn+nkxCwrXqX0sv\\\\\\\/hShVcSqOLJB7HpjNyF4yTWIfWAOtlCZyUz0F99VS2ojDpeHDWe1u4SsVS8rT\\\\\\\/RMw1srjdJPi+3q5jdocX2Y4ztQza7dFygC3i56Y+f1CGI0tA5lRTFSs9iUblS4HLjiDOAAXn\\\\\\\/G8HRSFRSC3bm9qI+j\\\\\\\/gytIUwOKJkmCGEl\\\\\\\/yxwb4FjzYlvdnEz7urJy4DVPR7FoBmHqqHbA9\\\\\\\/q0llTWASF4lND6vx3H+w2xIB1DCN\\\\\\\/fdnPJBGZcZCOkdeyDyNnA5LPio\\\\\\\/oyX6sBwBztpDDYdREcnxCghjPs1Lc9jkfA1Iz0YFBy12qw5SXHiwKmi1xCyB2jF4WNLWf9csDb9v8k59VLGoo4UkL7iPymAoEThF9Oq686PvrQBrxEqCyupnM+TWfeY2t0hNFhrAI14iKiueFUt5yl92LB2SZniwR1XQd0zp4GwELSWtued4X6r9bJSR6O0TtjM+92sOGUwjvyY1cVGm7ZmaEgdDK8sR6uLo+0KS2wyKSh39AH\\\\\\\/phg\\\\\\\/4s8vjKZtdunptK5L0DwztQqF+Vo1kzEqNQyF2skUmVxphbtA\\\\\\\/71qZdkmPKSjUo0DEjfAjOxOaUrVE2h5zHvesHTW\\\\\\\/QtpN4zZUPOydliTehLyvVrlxEtr5+c\\\\\\\/Ghu8qcC2RsPieFfx8zbla6c12E0CQKhPoXM5Y4I5vQhmeaGy7nzA6AYUxLH4VzgtkTQTs3JsnKHvHLDoeH+oG9Jk1fwjoc75yrzv4egfka7ccxeP\\\\\\\/NQU4bVFriy12v2u\\\\\\\/LB8WuVBTRflecc27O\\\\\\\/nnKwa8YZGWVA0N8zcsA\\\\\\\/5F3A6fvvP2buXjLye1qH7sR+If3jFlb7sSppJDsXdPbyQSe3Fp95OvNXx4wRNfb39Eb3OkrUmmEZOVJZcBILhcz\\\\\\\/E+sC8QKiYWghwfsXVto2KDytt6G\\\\\\\/\\\\\\\/jwnR51Bp0r4QwkNTrS9xClTq3nheZR5gJke64FZF2n0mJhHLfi53Ee33igNKb2F58u4AuxPbAWWPdFmLZuJ8l24aiM+ij6\\\\\\\/gTRnlmfLP93CDF\\\\\\\/57FpNdjzJ+s3eyTTM3+HNdEtxEmLiqZ\\\\\\\/isIejfvk\\\\\\\/bXG\\\\\\\/bfQZCRV4oKSUIL0pZW3F1jsHsstO+FEIvV1e70zfIj2IseAVg7uu3M12BvBXZ+FlQRFS2cAt73Y\\\\\\\/w7pFZWsLJbT4EPBQY8NTMIRPKGAuq5rw4QMpBhGbdOqE8SAIRsXolosr1pHTOaMCUgLONiIgehGksfplhxLklCKh9hXDcVOO9cW\\\\\\\/bv\\\\\\\/jQ4ySLFm0h98TeU3A8Yw5Wpa1mA9BQaVGg4Czc+g3WtGJIFoaI4b201E20sdNL2qsSwolG21bgAUi\\\\\\\/4ahhi4383xB4bDwclZ4EYvXkYC+JNbVSAYTvPFzp+P+hhgO4+00W0wIinWRXsXWoUDC\\\\\\\/eQNfsPi3eGIHP9CHPpKo0V4Jqsi41kb6UmZdCY2xg5E\\\\\\\/eALu8rW+wC6IJTkZ3Nm0lgPBnADjNsny8zm5FWMQeWvPimlVpeXPsWO\\\\\\\/iDb\\\\\\\/sNxNwVUvaNYNWD2giCgtCQNxvAhIOEuX89JFVSoKQCc6lipsc2fC0oAIVxui8v9TKt7pNBAgN3zPFTQKv\\\\\\\/0MxXwUMQu5Ht3LS11ecAkahgeKxtAEJ8ryIoj57wAwiSnTzx5xgbHjjkiCzMieF\\\\\\\/qC6uglbYb6hF0q4lp\\\\\\\/b9pVZGMLlaHMKWOMoByvSoyPnioTH6Yi\\\\\\\/44cjJOD8B0XyVq6tBWTZlW9Qsqc\\\\\\\/0x7wYhMbKGs50LQr5n0DXmUOEH9XxLYSmISJlrjBL5sG4JY8sOD1wQAUuW4KGK1+3Mc9YjAbBoACXOzW2yv4u1gg2e6WdhgYfgArpr3ZPyHor5H32hsKbtkdWREHw8JTbgoLa6iIQkH8ZE6KAvN\\\\\\\/tbztOmMVeSDEIvAjx3WOvXPQfoRbaJy5XpgyLy0lonz2MTTRjTnKrS+sMSUHCNd4mn\\\\\\\/tcD+TTm1zqb0By8Z6yDpbkahEnS\\\\\\\/MGQ\\\\\\\/0LSukyE+Wq3yJMVRAzh3Yu3wo3aVceD1F73pQczhF3pdMt+8Ll671QDwzKTq983C4RuNC9+A0KG\\\\\\\/wnkTXMjnZZzUjHjMr6hXXx1P6iv1lBIlH2PJs16kbBqNtI8icomACFumbIyy5b+n3+pBrfrh3WUj\\\\\\\/X6uz4sO3syowBeFNgKlXJUuQrl4G3w1yuUPrT0ZgSqPQZZ4PzRqy4SNx8PN8DqyCTqkc10gr1X\\\\\\\/wKyJyj5TfHOvFpeJLj4U6+dXFvV7HK9HEBQ2+eDigBinRfeSRHwVanEIkjZZzNX2KwhN6RQ6H1b5OSYASGSMoC\\\\\\\/IznZm1xXaVSY5OObtUp0Laz7XLvKhFu9OjWp\\\\\\\/GVsS8hhWL\\\\\\\/12cXx\\\\\\\/clFjNOnmJ7a2C0h9agKL+0aP4xMysBBkZNGZ3oqjCkehXAJKWQn8hXFdeNgELZq+vOLGthyQhA8P4ntZNOvqP6Ic34hvWKBjiCMAz6Ki1uzrQiYub8Oz\\\\\\\/F4NFsunqGrhMpOfreIfocvdCE3QMviRacQn8h2\\\\\\\/kVYoFrzsrZRUi6geWS+GYxAmHqxyN4pJ76R88C3rVjzosy95Cf0fRlHkkAW58YOuj2Ev5ONRwFVgsAvI1j2ALb4232C3v7J0usKhWoeX\\\\\\\/fTnzZtH6bsISxTxAA6CNRwiMa6HvI8zsykCRQhYnLujMIvqf1vu279eAc4qdcdIjabeKmhqhbNFaVwELRCqx33oBClsQSmy6Ti3t2EFno8rC4XuClvuXO835xMZyxPnAWRfjhcKG8NBB58F+zSMHjBWmGOFDWiSi0RSdqdB+RpQzLpSoF2ewrOHIK0poST4ipKb2VmPE04K4vND6Ra6UfihgDJHlblmYsUl7LTXVFOViC9jM42IEiS9nTWhvIa5T80Nm4KH8KVzeul237ZgJ2lkQwb2TYhYXod7NGvBhK1gjtSppikw+GI+QTwWnVhzzS88qWn0u+Me\\\\\\\/8BgwToHCFVBiPlTBeXEHDjjanVQnFwJT7pP4WpawAF83cLkL0cbpZR1WZCjJj7v2VgICoWf+jUZE2KyMP2b5AKbzbmt0felgi4j+TWyh5K6GyxnmcjlxepM6P1zC9ZZucDldDwYAZdhkVKSFnaFDxfNZPcRHSzmzCzld1C8ffVubEiq4hqiorrKOwAqFnjCt\\\\\\\/w+gOoMudIyHdHwCwhIIbP5qv41y+WF656V30sRCm\\\\\\\/kcoTM\\\\\\\/FC2\\\\\\\/7oJivpWFJI\\\\\\\/aykRGNmR7EgqHXmzl2lvOsJ5Akd8KNL8YIieEdtKWfT95CYz4zlT5FMekpyhPTDxf7KCJbmZO19k6+BwrquPtEAR\\\u0022,\\\u0022iv\\\u0022:\\\u0022EmqoqKZhZ0N+PgbmFqmk3w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a0285f16263d4e8d10397d3b7f41f4357c6cc6f74470c52032610dd750a893cd\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022pOAIWKYis2miFhDUvvzfTkfO1gS2Wly5kkpN4o+vggQ9c3Eparh0Uu1MF\\\\\\\/lzK4hWkt1F4CTAE0dbmNwrODZfKF\\\\\\\/l2vIP1Aj72QjwhhiT6KxrnwKr81s9wG3pGnO5zVAn7gqJMokhEPQsH0tNAQ\\\\\\\/KHDXzNAm9QjlZUnGfoVK\\\\\\\/c01YaVGRCS5DjmY+ELg12uq95hDs1q9ueWtU7PZvZl4+q0gya9pvyzcW1jU1Mjxc3BqUf2Kz7ZbxT\\\\\\\/tVOImQ\\\\\\\/L3HMRWlsb1lrXhUixFtO\\\\\\\/sXSiGaGCL5Y4+aeAPGtxl556Ilf+SczQyRJR2gQaZk4mJBaTE0sSpAz3AyLMXBvkQP0QK32HT2YUU1Rhp4oAWQd9JYKWEMh\\\\\\\/+SBO3i16\\\\\\\/jutVpINFTlHIVHW8kxe8mKSVef16\\\\\\\/czoh+9DAtdtMzouPThLzBT6pPd\\\\\\\/pUednUlDupN6VBiOEnrrQl\\\\\\\/\\\\\\\/xGbv9qg5zyvtrd5ZqndM1CY4g1\\\\\\\/AHfj6D6xXkevLvtt+b8cM9zpTkWGtINQ7HP8xBUGLdhaGDRnmMi0MhWjLGSPSMCFz1L\\\\\\\/HPUqcWx7qHJ55KzNXu4Dc3T2sIJrM2Z3C8Ktwz756K3urcqLIgOb8vTQ4BA2p\\\\\\\/Reiw0yS77Yy63VWFJg0qAPa9zWSXcHSk9f62EeZM7aJ+cPDrNoJQlBg9F3+TB7UU8a80uLygu7w2zW4OKFJ5SchBQ9shzLMhdoI8x\\\\\\\/enSn4IxaLUpwssc3BVBOQDPTvU2bF7iAr3o+vc8BgeC+Xo6m9JrWz9I1P73HlJaeoWPW\\\\\\\/S+KsFzZ7TkAppenFhFL8YFrMqcSh1aX10MC2M9FUPYid3rkZkYxB+lVmIDdIkXMVwQDcF8FUVJudk+SexB9lYsOe8TmopYiavJ7BXlPc1GiYTa7ioVa16awNqnjVlBhVJmW5F6BHXpUVZKTNdXajWw9Soe9X2V+\\\\\\\/G2y1NXjCBS0VhRSJ4JACYeXexlrOTY93XiOrSuo\\\\\\\/8HoHXlevAjAuC5HyxyatdfWElCkkIXMYReDwpxox\\\\\\\/CAQs6ZTjNjZgWOVI21YzEfdn4Gf5\\\\\\\/MkSA4v2cYJRkzsxRRa+ykXTjygPmfYc95tdVlPpJs2OumCJyETPp1e\\\\\\\/cZI\\\\\\\/aidcH09PryV8QpK2Xk8FFWBrNnpTK3KQSZ1FIL9fG19CLFUmzPVdcfy8pO2rK\\\\\\\/Z4pzU1JGRzKQTfyTDM1KzUlS2YGzBMQ\\\\\\\/bLcoR7W+I1t7WrOrJ4F6mEpkH3VeOM0zKapCqU+6dvBdrbsIL8MxR47+rtAsHc2kVTtyFzNZXA2VpoZTLyaN3dm56M8\\\\\\\/To8IYZE2K5vwItOlAWRxSdJS373KMBv7rNXr7NPoul1Z4VyhPLkckUB3\\\\\\\/wI9IstpEnL7s9GqHGi+SZpGSkuf7coEiE2ZOrX5HKLQnc6fH81nQb+XngBC5K\\\\\\\/SrkaUaNcStx4yklimrsE1O\\\\\\\/7cc97Qm2d7aNT0sXkL40HHYjp84yk2hknL7QgEuAdDSPNo46mz+Y+eB63R8B5FIIP+Afn4KxlNwl3Ff\\\\\\\/zMAJy2YXyPrao7g2zR2X3AGJrsCCTJxugNPkaTPH5Q1lxxRz2iqLeTMMBCeswRk\\\\\\\/ZzhBeNNNPNffONWXxyEfMqI534Razz8z6PMN55UmWv0Rv8kOvJyGG9p\\\\\\\/ZXSaMwsXaI38qwdgN3egzNkQRgOZC4sVAUj4EMt0uXT2Nj+3\\\\\\\/+837U6Pmag7cQsuyqDwhOqhe0pxGx1UgtSFv5h9ZXV0fIb+003rm8pdaJ9\\\\\\\/cqQ7dnejH7iCtF3h1zh8PwM2IHUqOSh4LNVpXgMz1HZhQxUoMFpQe7v6F8si2zBreZn+plL6GM0ejdRMgaCXwQ\\\\\\\/Qq6ijHkOzMR7mcAXcITzCV0JS2+dTLfbldD8LUVklWoHXUqOHIdnfoc2ZrNQPl\\\\\\\/J5f4ugbCrDmpd3jVPLnJEDZ4NJ8SCo3sHenIA6ChWYRAy+gUMt9pNHCOZhaIF8NgDMNomaQyutRePj1ncL5J2SycHGwtqXkjAk9M0tYztZw4Ikz6CW6XMhd9G64dA=\\\u0022,\\\u0022iv\\\u0022:\\\u0022mC\\\\\\\/u1JAM7T3n+jxQAj5viw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225826da6dd8d07cd16f2c17bf6cb80851afe8b58ff465c14b30b3d8edbdb9e03f\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460473 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/11/3507" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-eGsvDuLjp282R5tqyJrMyhMsfHq4OIBU83_9YUVCFrg" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/11/3507" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-VqURaW9hrTvoG843p9y26h83BhJrjEStQV8yeTABHUY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/11/3497" title="Marker-Based Estimates Reveal Significant Nonadditive Effects in Clonally Propagated Cassava (Manihot esculenta): Implications for the Prediction of Total Genetic Value and the Selection of Varieties" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/11/3517" title="A Pedigree-Based Map of Recombination in the Domestic Dog Genome" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460473" id="node460473" data-pisa="ggg;6/11/3507" data-pisa-master="ggg;g3.116.034488" data-apath="/ggg/6/11/3507.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Suitability of Different Mapping Algorithms for Genome-Wide Polymorphism Scans with Pool-Seq Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-9960-7248" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Robert Kofler</span>, <span class="highwire-citation-author" data-delta="1">Anna Maria Langmüller</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-4923-0993" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Pierre Nouhaud</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-5026-4168" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Kathrin Anna Otte</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-4710-6526" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Christian Schlötterer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3507-3515;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.034488 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460473" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Robert Kofler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut für Populationsgenetik, Vetmeduni Vienna, Veterinärplatz 1, 1210 Wien 1210, Austria</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BKofler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kofler%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARobert%2BKofler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9960-7248" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Robert Kofler</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Anna Maria Langmüller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut für Populationsgenetik, Vetmeduni Vienna, Veterinärplatz 1, 1210 Wien 1210, Austria</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Vienna Graduate School of Population Genetics, 1210, Austria</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anna%2BMaria%2BLangm%C3%BCller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Langm%C3%BCller%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnna%2BMaria%2BLangm%25C3%25BCller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Pierre Nouhaud </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut für Populationsgenetik, Vetmeduni Vienna, Veterinärplatz 1, 1210 Wien 1210, Austria</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pierre%2BNouhaud%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nouhaud%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APierre%2BNouhaud%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4923-0993" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Pierre Nouhaud</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Kathrin Anna Otte </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut für Populationsgenetik, Vetmeduni Vienna, Veterinärplatz 1, 1210 Wien 1210, Austria</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kathrin%2BAnna%2BOtte%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Otte%20KA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKathrin%2BAnna%2BOtte%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5026-4168" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kathrin Anna Otte</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Christian Schlötterer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut für Populationsgenetik, Vetmeduni Vienna, Veterinärplatz 1, 1210 Wien 1210, Austria</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christian%2BSchl%C3%B6tterer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schl%C3%B6tterer%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChristian%2BSchl%25C3%25B6tterer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-4710-6526" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Christian Schlötterer</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:christian.schloetterer@vetmeduni.ac.at" class="" data-icon-position="" data-hide-link-title="0">christian.schloetterer@vetmeduni.ac.at</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/11/3507" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460473" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460473/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/11/3507.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460473" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460473/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/11/3507.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460473" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460473/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/11/3507.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460473" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460473/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/11/3507.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460473" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460473/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">The cost-effectiveness of sequencing pools of individuals (Pool-Seq) provides the basis for the popularity and widespread use of this method for many research questions, ranging from unraveling the genetic basis of complex traits, to the clonal evolution of cancer cells. Because the accuracy of Pool-Seq could be affected by many potential sources of error, several studies have determined, for example, the influence of sequencing technology, the library preparation protocol, and mapping parameters. Nevertheless, the impact of the mapping tools has not yet been evaluated. Using simulated and real Pool-Seq data, we demonstrate a substantial impact of the mapping tools, leading to characteristic false positives in genome-wide scans. The problem of false positives was particularly pronounced when data with different read lengths and insert sizes were compared. Out of 14 evaluated algorithms novoalign, bwa mem and clc4 are most suitable for mapping Pool-Seq data. Nevertheless, no single algorithm is sufficient for avoiding all false positives. We show that the intersection of the results of two mapping algorithms provides a simple, yet effective, strategy to eliminate false positives. We propose that the implementation of a consistent Pool-Seq bioinformatics pipeline, building on the recommendations of this study, can substantially increase the reliability of Pool-Seq results, in particular when libraries generated with different protocols are being compared.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/pool-seq" class="hw-term hw-article-keyword hw-article-keyword-pool-seq" rel="nofollow">Pool-Seq</a></li><li class="kwd"><a href="/keyword/bioinformatics" class="hw-term hw-article-keyword hw-article-keyword-bioinformatics" rel="nofollow">bioinformatics</a></li><li class="kwd"><a href="/keyword/next-generation-sequencing" class="hw-term hw-article-keyword hw-article-keyword-next-generation-sequencing" rel="nofollow">Next Generation Sequencing</a></li><li class="kwd"><a href="/keyword/mapping-algorithm" class="hw-term hw-article-keyword hw-article-keyword-mapping-algorithm" rel="nofollow">mapping algorithm</a></li><li class="kwd"><a href="/keyword/drosophila" class="hw-term hw-article-keyword hw-article-keyword-drosophila" rel="nofollow">Drosophila</a></li></ul><p id="p-4">Sequencing pools of individuals (Pool-Seq) is a cost efficient approach to generating genome-wide polymorphism data, which is enjoying increasing popularity (reviewed in <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Schlötterer <em>et al.</em> 2014</a>). For example, Pool-Seq was used to unravel the genetic basis of complex traits (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bastide <em>et al.</em> 2013</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Cheeseman <em>et al.</em> 2015</a>), identify loci contributing to local adaptation (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Lamichhaney <em>et al.</em> 2012</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Turner <em>et al.</em> 2010</a>), trace beneficial loci during experimental evolution (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Lang <em>et al.</em> 2013</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Orozco-terWengel <em>et al.</em> 2012</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Tobler <em>et al.</em> 2014</a>), identify positively selected loci in populations (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bergland <em>et al.</em> 2014</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Kofler <em>et al.</em> 2012</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Nolte <em>et al.</em> 2012</a>), find genes selected during domestication (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Axelsson <em>et al.</em> 2013</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Rubin <em>et al.</em> 2010</a>), study the invasion of transposable elements (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kofler <em>et al.</em> 2015a</a>), investigate clonal evolution in cancer (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Ding <em>et al.</em> 2012</a>), and to identify causative mutations in forward genetic screens (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Schneeberger <em>et al.</em> 2009</a>). With this rapid gain in popularity, it is important to ensure the reliable analysis of Pool-Seq data. Several studies have investigated various aspects that potentially affect the accuracy of Pool-Seq, including the sequencing platform (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Rellstab <em>et al.</em> 2013</a>), reference genome (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Nevado <em>et al.</em> 2014</a>), parameters used for aligning reads (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Kofler <em>et al.</em> 2011a</a>), sequencing depth (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Ferretti <em>et al.</em> 2013</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Kofler and Schlötterer 2014</a>), pool size (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Futschik and Schlötterer 2010</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gautier <em>et al.</em> 2013</a>), and library preparation protocol (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kofler <em>et al.</em> 2015b</a>) used.</p><p id="p-5">However, until now, the impact of the mapping algorithm used for aligning Pool-Seq data has not been studied in sufficient detail. Here, we show that the mapping algorithm can have a profound effect, leading to erroneous signals of allele frequency differences between libraries. We compared systematically the performance of 14 different alignment algorithms using both simulated and real Pool-Seq data. Of the algorithms tested, clc4, novoalign, and bwa mem consistently produced the most reliable results with Pool-Seq data. Nevertheless, no single alignment algorithm avoids all artifacts, but, by intersecting the results of two alignment tools, the vast majority of artifactual outliers can be avoided.</p><div class="section materials-methods" id="sec-1"><h2 class="">Material and Methods</h2><div id="sec-2" class="subsection"><h3>Alignment algorithms</h3><p id="p-6">We tested seven semiglobal alignment algorithms, where the entire read is required to match, and seven local alignment algorithms, where only a part of the read needs to match (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). For tools that support semiglobal, as well as local, alignments, we evaluated the suitability of both algorithms (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). We also included gsnap (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Wu and Nacu 2010</a>) into our study, despite the fact that this tool was designed for aligning RNA-Seq data (<em>i.e.</em>, alignments with large gaps to allow for spliced introns). We also aimed to include gem (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Marco-Sola <em>et al.</em> 2012</a>), batalign (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Lim <em>et al.</em> 2015</a>), stampy (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Lunter and Goodson 2011</a>), and soap2 (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Li <em>et al.</em> 2009b</a>) into our study but were not able to run these tools on our computational infrastructure (Mac Pro; batalign: did not respond, gem: compilation failed, stampy: compilation failed due to missing files, soap2: segmentation fault while indexing the reference genome). If possible, we used default parameters for all tools, and deviated from these settings only when deemed necessary to ensure an unbiased comparison of the alignment algorithms (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). With Bowtie2, we set the maximum fragment length of paired ends (–X) to 1500. For bwa, we used version 0.7.4 for the mem and bwasw algorithm, and version 0.6.2 for the aln algorithm. This was necessary as bwa aln 0.7.4 reports a segmentation fault when aligning some data sets (<em>e.g.</em>, the <em>Drosophila simulans</em> libraries), whereas the mem algorithm was not available for bwa version 0.6.2. For clc4, we interleaved the sequences of the two fastq files (-i), activated the paired end mode (-p), set the orientation of the paired ends to forward followed by backward (fb), and measured the distance between paired ends from start-to-start (ss). As the performance of clc4 is highly sensitive to the provided minimum distance (min) and maximum distance (max) between paired ends, we provided the most suitable setting for each alignment (simulated data, read length 50 and inner distance 100: <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-1.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-2.gif"/></noscript></span></span> read length 100 and inner distance 100: <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-3.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-4.gif"/></noscript></span></span> read length 100 and inner distance 300: <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-5.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-6.gif"/></noscript></span></span> <em>D. simulans</em> libraries, read length 76: <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-7.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-8.gif"/></noscript></span></span> read length 120: <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-9.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-10.gif"/></noscript></span></span>). For mrfast, we used paired end mapping (-pe), provided a minimum fragment size of 10 (−min), a maximum fragment size of 400 (−max; for the simulated data with an inner distance of 300, −max 700 was used), a maximum number of mismatches of 6 (-e), and required that only the best position of a read should be reported (–best). We specified bam as output format (-b) for ngm, and performed a sensitive search (–sensitive; the default is unclear). For novoalign, we provided sam as output (-o SAM), set the quality encoding of fastq files to Sanger (-o STDFQ), required that a random position is reported for ambiguously mapped reads (-r Random), and provided suitable estimates for the insert size (mean) and the SD of the insert size (-i mean SD; simulated data: <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-11.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-12.gif"/></noscript></span></span> <em>D. simulans</em> libraries, read length 76: <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-13.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-14.gif"/></noscript></span></span> read length 120: <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-15.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-16.gif"/></noscript></span></span>). For segemehl, we set the maximum insert size to 1500 (-I). For gsnap, we used sam as output format (-A sam). Only for the <em>D. simulans</em> libraries was the maximum number of allowed mismatches set to 1 (-m 1), as gsnap encountered an error using these data and default settings. (Problem sequence; we iteratively removed five problem sequences but still encountered the error). Alignments with bwa aln were done on a Hadoop cluster (Pandey and Schlötterer, 2013) <a href="http://journals.plos.org/plosone/article?id=10.1371%2Fjournal.pone.0072614">http://journals.plos.org/plosone/article?id=10.1371%2Fjournal.pone.0072614</a>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469728/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469728/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Overview of the mapping algorithms used in this work</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>Data sets</h3><p id="p-9">We tested the performance of the different alignment algorithms using both simulated and real data.</p><p id="p-10">Simulated paired end data were generated for populations having SNPs with known positions and allele frequencies. This was accomplished in four steps. We first obtained the <em>Drosophila melanogaster</em> reference chromosome 2R (r6.03; <a href="http://flybase.org/">http://flybase.org/</a>), removed all characters other than A, T, C, or G, and extracted the first 2 Mbp. This small subsequence (the chassis) acted as a basis for introducing variants. Second, we generated two modified versions of the chassis: (i) we introduced a SNP with a random, not-reference, allele every 100 bp into the chassis (<span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-17.gif"/></noscript></span></span>chassis with SNPs), and (ii) we introduced an indel at a random position with a random Poisson distributed length (<span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-18.gif"/></noscript></span></span> zero length indels were discarded and Poisson sampling was repeated; insertions had a random sequence), between all pairs of adjacent SNPs into the chassis with SNPs (<span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-19.gif"/></noscript></span></span>chassis with SNPs and indels). Third, we generated two sequences serving as templates for simulating paired ends: one consisting of the chassis and the chassis with SNPs (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>), and another consisting of the chassis and the chassis with SNPs and indels (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). Finally, uniformly distributed paired end reads (equal 5′ distance between consecutive paired ends; uniform base quality of 40) were simulated from these template sequences (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2C</a>). Note that SNPs identified from these data have known positions (each 100 bp) and known allele frequencies (<span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-20.gif"/></noscript></span></span>). Paired end reads were simulated with SimulaTE (<a href="https://sourceforge.net/projects/simulates/">https://sourceforge.net/projects/simulates/</a>; R. V. Pandey <em>et al.</em> personal communications), and the number of reads was selected such that a genomic coverage of 200 resulted (<em>generate-reads_paired-end-uniformdistribution.py</em>; ∼2 million paired ends for a read length of 100, and 4 million for a read length of 50).</p><p id="p-11">We tested the performance of the different alignment algorithms for real data using paired end reads from a <em>D. simulans</em> population that was collected in 2008 in Northern Portugal (Póvoa de Varzim; provided by P. Orozco-terWengel). We established 250 isofemale lines from the population, used one female from each isofemale line, and extracted genomic DNA from the pooled flies as described (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Orozco-terWengel <em>et al.</em> 2012</a>). From this DNA, we generated two Illumina sequencing libraries. The first was prepared using the Paired-End DNA Sample Preparation Kit (Illumina, San Diego, CA) following fragmentation of the DNA using a nebulizer, and size selection using an agarose gel. The library was sequenced on two lanes of an Illumina GAIIx, resulting in 14.3 and 24.7 million 2 × 76 bp paired end reads after trimming [median insert size 232 bp; SD of the insert size 25 bp; estimated with Picard v1.128 (<a href="http://picard.sourceforge.net">http://picard.sourceforge.net</a>) after mapping the reads with bwa aln (0.6.2) (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Li and Durbin 2009</a>)].</p><p id="p-12">The second library was prepared with barcoded adapters using a protocol based on the NEBNextDNA Library Prep Master Mix Set reagents (E6040L) following shearing pooled genomic DNA with a Covaris S2 device (Covaris, Woburn, MA), and size selection with AMPureXP beads (Beckman Coulter, Brea, CA). The library was sequenced on one lane of an Illumina HiSeq 2500 using 2 × 120 bp reads (median insert size 396 bp; SD of the insert size 110 bp; 84.5 million paired end reads after trimming).</p><p id="p-13">The quality encoding of all reads was converted to Sanger (offset = 33), and low quality regions of reads were trimmed with ReadTools (<a href="https://github.com/magicDGS/ReadTools%13disable-zipped-output%13minimum-length50%13no-5p-trim%13quality-threshold18">https://github.com/magicDGS/ReadTools–disable-zipped-output–minimum-length50–no-5p-trim–quality-threshold18</a>; per default the quality is converted to Sanger encoding). ReadTools provides a fast implementation of the trimming algorithm described in <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Kofler <em>et al.</em> (2011a)</a>.</p><p id="p-14">We tested whether intersecting of mappers preserves the targets of selection using the data published by <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Martins <em>et al.</em> (2014)</a>. We obtained Illumina paired end data (2 × 100 bp) for four populations infected with C-virus for 20 generations (VirSys; accession numbers ERS409784-ERS409787), and for four control populations (ContSys; accession numbers ERS409780-ERS409783).</p></div><div id="sec-4" class="subsection"><h3>Data analysis</h3><p id="p-15">The simulated reads were mapped to the chassis (see above), the <em>D. simulans</em> libraries were mapped to the reference genome of strain M252 (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Palmieri <em>et al.</em> 2015</a>) (v1.1; we included the sequences of <em>Lactobacillus brevis</em>, <em>Acetobacter pasteurianus</em>, and two Wolbachia strains; GenBank accession numbers <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=CP000416.1&amp;atom=%2Fggg%2F6%2F11%2F3507.atom">CP000416.1</a>, AP011170.1, AE017196.1, and CP001391.1), and the data from <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Martins <em>et al.</em> (2014)</a> were mapped to the reference genome of <em>D. melanogaster</em> (v6.03; we again included the sequences of <em>L. brevis</em>, <em>A. pasteurianus</em>, and two Wolbachia strains). If not mentioned otherwise, mapped reads were filtered for mapping quality (-q 20), and proper pairs (-f 0x002 -F 0x004 -F 0x008; except for the analysis of single end reads), with samtools (v1.2) (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Li <em>et al.</em> 2009a</a>). Mapped reads were converted to mpileup files with samtools (v1.2), and the parameters -B -Q 0. SNPs were called using a minimum allele count of 2. The number of true SNPs (every 100th position), the number of false SNPs (not at every 100th position), the frequency of the reference allele (only for true positive SNPs), and the number of extreme outlier SNPs (where the estimated allele frequency deviates by &gt; 0.4 from the true frequency 0.5), were computed using custom Python scripts (<em>snp-caller.py</em>, <em>stat-snp.py</em>). For computing allele frequency differences between samples, mpileup files were created with samtools (v1.2; -B -Q 0), the mpileup files were converted to sync files with PoPoolation2 [revision 196; <em>mpileup2sync.jar</em> –fastq-type sanger; the minimum quality (–min-qual) was set to 0 for simulated reads, and to 20 for <em>D. simulans</em> libraries; (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Kofler <em>et al.</em> 2011b</a>)], and <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-21.gif"/></noscript></span></span> or Fisher exact test <em>P</em>-values (−log10 transformed) were computed with PoPoolation2 (revision 196; <em>fst-sliding.pl</em> –min-count 2 –min-coverage 10 –max-coverage 500 –window-size 1 –step-size 1 –suppress-noninformative –pool-size 400 –min-covered-fraction 1.0; <em>fisher-test.pl</em> –min-count 2 –min-coverage 10 –max-coverage 500 –window-size 1 –step-size 1 –min-covered-fraction 1.0). The outlier quantiles of <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-22.gif"/></noscript></span></span> and <em>P</em>-values (Fisher exact test; −log10(<em>P</em>-values)] were calculated with Python scripts (<em>fst-fractionwise.py</em>). Results of two mapping algorithm were intersected with a custom script (<em>merge-teststat.py</em>).</p><p id="p-16">We evaluated the performance of the following quality filtering methods: (i) a minimum mapping quality of 40 (using samtools v1.2; parameters -B -Q 0 -q 40), (ii) a minimum allele count of 10 (using PoPoolation2; revision 196; <em>fisher-test.pl</em> –min-count 10 –min-coverage 10 –max-coverage 500 –window-size 1 –step-size 1 –min-covered-fraction 1.0), (iii) a minimum base quality of 30 (using PoPoolation2; <em>mpileup2sync.jar</em> –min-qual 30), (iv) the 10% of the SNPs with the most pronounced strand bias were removed (strand-bias was computed as <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-23.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-24.gif"/></noscript></span></span> is the frequency of reads mapping to the forward strand at a given site, irrespective of the allele), and (v) sites not called by FreeBayes were removed [FreeBayes v1.0.2-6-g3ce827d (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Garrison and Marth 2012</a>); parameters -X -K -C 1 -F 0.01 (input data are pooled, ignore multi-allele SNPs, minimum allele count of 1, minimum allele frequency of 0.01)].</p><p id="p-17">Differentiation between evolved and control populations for the data from <a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Martins <em>et al.</em> (2014)</a> was assessed with the Cochran-Mantel-Haenszel test (CMH) implemented in PoPoolation2 (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Kofler <em>et al.</em> 2011b</a>) (parameters: –min-count 2 –min-coverage 10 –max-coverage 500).</p><p id="p-18">Aligned reads were inspected visually using IGV (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Thorvaldsdóttir <em>et al.</em> 2013</a>), and statistical analyses was performed using the R programming language (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">R Core Team 2012</a>).</p></div><div id="sec-5" class="subsection"><h3>Data availability</h3><p id="p-19">The short reads have been made available at the European Nucleotide Archive (ENA; <a href="http://www.ebi.ac.uk/ena">http://www.ebi.ac.uk/ena</a>; PRJEB13602), and the scripts used in this work, as well as the simulated reads, are available at Dryad (<a href="http://datadryad.org/doi:10.5061/dryad.2g3s4">http://datadryad.org/doi:10.5061/dryad.2g3s4</a>).</p></div></div><div class="section results" id="sec-6"><h2 class="">Results</h2><p id="p-20">Genome-wide polymorphism scans with Pool-Seq data are becoming increasingly used in population genomic research. Typically, these studies use genome-wide Pool-Seq data to identify marked outlier loci in pairwise comparisons between population samples. For example, loci contributing to local adaptation are identified by significantly different allele frequencies between populations (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Lamichhaney <em>et al.</em> 2012</a>; <a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Turner <em>et al.</em> 2010</a>). This focus on outlier loci makes genome-wide scans susceptible to technical problems that could generate outlier artifacts. We found that the mapping algorithms for aligning Pool-Seq data may be an important source of outlier artifacts (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). Comparing allele frequencies between two Pool-Seq libraries prepared from identical genomic DNA, but with different insert size and read length, we found a substantial number of outlier loci, despite the fact that no differences between the libraries were expected (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1, A and B</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/11/3507/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Manhattan plots indicating the significance of allele frequency differences between Pool-Seq libraries when the same genomic DNA is sequenced. Two Illumina paired-end sequencing libraries with different read length and insert sizes were prepared from a pool of 250 D. simulans individuals. Reads were mapped to the reference genome, and the significance of differences in allele frequencies between the two libraries were computed (Fisher&#x2019;s exact test). Although no significant allele frequency differences were expected, we found pronounced outlier peaks using bwa aln (A) or novoalign(g) (B) for mapping the reads. Importantly, outlier peaks found with these two alignment algorithms are at different genomic sites. Hence, intersecting the results of these two algorithms by plotting the lowest P-value obtained at each site removes the vast majority of outlier peaks (C)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-864650000" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Manhattan plots indicating the significance of allele frequency differences between Pool-Seq libraries when the same genomic DNA is sequenced. Two Illumina paired-end sequencing libraries with different read length and insert sizes were prepared from a pool of 250 &lt;em&gt;D. simulans&lt;/em&gt; individuals. Reads were mapped to the reference genome, and the significance of differences in allele frequencies between the two libraries were computed (Fisher&#x2019;s exact test). Although no significant allele frequency differences were expected, we found pronounced outlier peaks using bwa aln (A) or novoalign(g) (B) for mapping the reads. Importantly, outlier peaks found with these two alignment algorithms are at different genomic sites. Hence, intersecting the results of these two algorithms by plotting the lowest &lt;em&gt;P&lt;/em&gt;-value obtained at each site removes the vast majority of outlier peaks (C).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/11/3507/F1.medium.gif" width="431" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/11/3507/F1.medium.gif" width="431" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/11/3507/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/11/3507/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469655" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-21" class="first-child">Manhattan plots indicating the significance of allele frequency differences between Pool-Seq libraries when the same genomic DNA is sequenced. Two Illumina paired-end sequencing libraries with different read length and insert sizes were prepared from a pool of 250 <em>D. simulans</em> individuals. Reads were mapped to the reference genome, and the significance of differences in allele frequencies between the two libraries were computed (Fisher’s exact test). Although no significant allele frequency differences were expected, we found pronounced outlier peaks using bwa aln (A) or novoalign(g) (B) for mapping the reads. Importantly, outlier peaks found with these two alignment algorithms are at different genomic sites. Hence, intersecting the results of these two algorithms by plotting the lowest <em>P</em>-value obtained at each site removes the vast majority of outlier peaks (C).</p><div class="sb-div caption-clear"></div></div></div><p id="p-22">To overcome this problem, we set out to identify alignment algorithms that are most suitable for genome-wide outlier scans using Pool-Seq data. We tested seven semiglobal alignment algorithms, where the entire read is required to match [bowtie2(g), bwa aln, clc4(g), mrfast, ngm(g), novoalign(g), and segemehl], and seven local alignment algorithms, where only a part of the read needs to match [bwa sw, bwa mem, clc4(l), gsnap, ngm(l), and novoalign(l); for an overview see <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>] (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Alkan <em>et al.</em> 2010</a>; <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">CLC bio 2015</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Hoffmann <em>et al.</em> 2009</a>; <a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Langmead and Salzberg 2012</a>; <a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">Li and Durbin, 2009</a>, <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">2010</a>; <a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">Novocraft 2014</a>; <a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">Sedlazeck <em>et al.</em> 2013</a>; <a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">Wu and Nacu 2010</a>). With several tools, like ngm or bowtie2, supporting both semiglobal and local alignments, we indicate the pertinent algorithm in brackets [<em>e.g.</em>: ngm(g): semiglobal alignment, ngm(l): local alignment].</p><p id="p-23">We first tested the overall performance of the alignment algorithm using simulated data sets. We generated template sequences with SNPs and indels (random position and length) at known positions, and then simulated uniformly distributed paired ends from these templates such that true SNPs are spaced exactly 100 bp apart, and have a population frequency of 0.5 (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>). Note that indels are in linkage disequilibrium with SNPs to identify biased allele frequency estimates resulting from mapping of reads with indels.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/11/3507/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Overview of simulated Pool-Seq data sets. Based on a 2&#xA0;Mbp region of D. melanogaster chromosome 2R, we simulated a pair of sequences, with one sequence having a SNP (red) every 100&#xA0;bp (A), and a pair of sequences with one sequence having, in addition to the SNPs, an indel (blue) with random position and length between adjacent SNPs (B). Using these sequences as templates, we simulated uniformly distributed paired ends (gray; C) resulting in SNPs with known positions and frequency (f = 0.5)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-864650000" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Overview of simulated Pool-Seq data sets. Based on a 2&#xA0;Mbp region of &lt;em&gt;D. melanogaster&lt;/em&gt; chromosome 2R, we simulated a pair of sequences, with one sequence having a SNP (red) every 100&#xA0;bp (A), and a pair of sequences with one sequence having, in addition to the SNPs, an indel (blue) with random position and length between adjacent SNPs (B). Using these sequences as templates, we simulated uniformly distributed paired ends (gray; C) resulting in SNPs with known positions and frequency (f = 0.5).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/11/3507/F2.medium.gif" width="440" height="277"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/11/3507/F2.medium.gif" width="440" height="277"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/11/3507/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/11/3507/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469581" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-24" class="first-child">Overview of simulated Pool-Seq data sets. Based on a 2 Mbp region of <em>D. melanogaster</em> chromosome 2R, we simulated a pair of sequences, with one sequence having a SNP (red) every 100 bp (A), and a pair of sequences with one sequence having, in addition to the SNPs, an indel (blue) with random position and length between adjacent SNPs (B). Using these sequences as templates, we simulated uniformly distributed paired ends (gray; C) resulting in SNPs with known positions and frequency (f = 0.5).</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">We evaluated the mapping algorithms with three different paired end data sets: (i) a data set representing optimal conditions (2 × 100 bp paired ends; insert size <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-25.gif"/></noscript></span></span> bp; error rate of 0%; no indels; <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2A</a>), (ii) a data set with indels and variation of the distance between paired ends (2 × 100 bp paired ends; insert size <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-26.gif"/></noscript></span></span> bp; error rate of 0%; indels; <a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2B</a>), and a dataset with indels and a high error rate (polymorphism) of 5% (2 × 100 bp paired ends with an insert size of <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-27.gif"/></noscript></span></span> bp; error rate of 5%; indels; <a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2B</a>). For all data sets, a coverage of 200 per site was targeted (∼2 million paired ends per data set). We evaluated the performance of the mapping algorithms based on two criteria: the number of true positive SNPs, and the number of extreme outlier loci with highly inaccurate allele frequency estimates (<span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-28.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-29.gif"/></noscript></span></span> frequency should be <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-30.gif"/></noscript></span></span>).</p><p id="p-26">We compared the performance of the mapping algorithms with and without filtering for quality criteria, such as paired end reads and mapping quality [<span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-31.gif"/></noscript></span></span> a low mapping quality suggest that the read is ambiguously mapped (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Li <em>et al.</em> 2008</a>)], and found that filtering consistently leads to reduced numbers of false positive SNPs and more accurate allele frequency estimates (Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034488/-/DC1/TableS1.pdf">Table S1</a>; note that, in the absence of sequencing errors, false positive SNPs are an artifact of the alignment). This observation is in agreement with previous work showing that quality filtering can reduce the number of false positive SNPs (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Li <em>et al.</em> 2008</a>). We note, however, that quality filtering also leads to fewer true positive SNPs (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034488/-/DC1/TableS1.pdf">Table S1</a>).</p><p id="p-27">Quality filtering also affected the coverage distribution. Fewer sites had a higher coverage than simulated in filtered data (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034488/-/DC1/FigureS1.pdf">Figure S1</a>), which is likely due to smaller numbers of ambiguously mapped reads that stochastically accumulate in some genomic regions. For mrfast, quality filtering resulted in a severe shift of the coverage distribution, halving the average coverage (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034488/-/DC1/FigureS1.pdf">Figure S1</a>). The distribution of mapping qualities differed between mapping algorithms (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034488/-/DC1/FigureS2.pdf">Figure S2</a>), which is likely due to distinct algorithms for computing mapping qualities. Since the accuracy of allele frequency estimates was substantially better for filtered data sets, we rely on quality filtered reads for the remaining manuscript. Summarizing the results for all three simulated data sets, we found that gsnap, novoalign(l), clc4(l), and clc4(g) showed the best performance, while mrfast, bowtie2(g), and bwa mem showed the worst (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034488/-/DC1/TableS2.pdf">Table S2</a>; for results with unfiltered data, see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034488/-/DC1/TableS3.pdf">Table S3</a>). The average reference allele frequency of most alignment algorithms was above 0.5 indicating a bias toward the reference allele [<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034488/-/DC1/TableS2.pdf">Table S2</a>; see also <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Degner <em>et al.</em> 2009</a>; <a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Kofler <em>et al.</em> 2011a</a>]. After quality filtering, mrfast had a substantial bias against the reference allele (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034488/-/DC1/TableS3.pdf">Table S3</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/11/3507/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Suitability of mapping algorithms for performing genome wide polymorphism scans with Pool-Seq data. Ideally, a mapping algorithm should enable the identification of all truly positive SNPs (green; 19.999 were simulated), while avoiding the identification of extreme outlier SNPs, with highly inaccurate allele frequency estimates (red;  or ). Algorithms are sorted according to performance, with the best performing algorithm shown at the top (maximizing the true positives, and minimizing the number of outliers). We tested the algorithm with three different data sets. Best case: 2&#xA0;&#xD7;&#xA0;100&#xA0;bp paired ends with an insert size of 100&#xA0;bp, indel - insert size: 2&#xA0;&#xD7;&#xA0;100&#xA0;bp paired ends with an insert size of 100&#xA0;bp, and indels between the SNPs, indel - error rate: 2&#xA0;&#xD7;&#xA0;100&#xA0;bp paired ends with an insert size of 100&#xA0;bp, indels between the SNPs and an error rate of 5%." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-864650000" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Suitability of mapping algorithms for performing genome wide polymorphism scans with Pool-Seq data. Ideally, a mapping algorithm should enable the identification of all truly positive SNPs (green; 19.999 were simulated), while avoiding the identification of extreme outlier SNPs, with highly inaccurate allele frequency estimates (red; &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-32&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; or &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-33&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;). Algorithms are sorted according to performance, with the best performing algorithm shown at the top (maximizing the true positives, and minimizing the number of outliers). We tested the algorithm with three different data sets. Best case: 2&#xA0;&#xD7;&#xA0;100&#xA0;bp paired ends with an insert size of 100&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-34&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;&#xA0;bp, indel - insert size: 2&#xA0;&#xD7;&#xA0;100&#xA0;bp paired ends with an insert size of 100&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-35&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;&#xA0;bp, and indels between the SNPs, indel - error rate: 2&#xA0;&#xD7;&#xA0;100&#xA0;bp paired ends with an insert size of 100&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-36&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;&#xA0;bp, indels between the SNPs and an error rate of 5%.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/11/3507/F3.medium.gif" width="402" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/11/3507/F3.medium.gif" width="402" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/11/3507/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/11/3507/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469701" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-28" class="first-child">Suitability of mapping algorithms for performing genome wide polymorphism scans with Pool-Seq data. Ideally, a mapping algorithm should enable the identification of all truly positive SNPs (green; 19.999 were simulated), while avoiding the identification of extreme outlier SNPs, with highly inaccurate allele frequency estimates (red; <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/F3/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/F3/embed/mml-math-32.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/F3/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/F3/embed/mml-math-33.gif"/></noscript></span></span>). Algorithms are sorted according to performance, with the best performing algorithm shown at the top (maximizing the true positives, and minimizing the number of outliers). We tested the algorithm with three different data sets. Best case: 2 × 100 bp paired ends with an insert size of 100<span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/F3/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/F3/embed/mml-math-34.gif"/></noscript></span></span> bp, indel - insert size: 2 × 100 bp paired ends with an insert size of 100<span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/F3/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/F3/embed/mml-math-35.gif"/></noscript></span></span> bp, and indels between the SNPs, indel - error rate: 2 × 100 bp paired ends with an insert size of 100<span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/F3/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/F3/embed/mml-math-36.gif"/></noscript></span></span> bp, indels between the SNPs and an error rate of 5%.</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">Next, we compared allele frequency estimates between samples—an approach that is typically used to identify loci responsible for local adaption. We investigated the sensitivity of the alignment algorithm to (i) differences of the inner distance between paired ends (inner distances <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-37.gif"/></noscript></span></span> bp <em>vs.</em> <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-38.gif"/></noscript></span></span> bp), (ii) differences in read length (read length 100 <em>vs.</em> 50 bp) and (iii) differences in the error rates (error rates 1 <em>vs.</em> 5%). Uniformly distributed paired ends were simulated from the template sequences having SNPs and indels (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2B</a>). Allele frequency differences between samples were measured using <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-39.gif"/></noscript></span></span> Values of <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-40.gif"/></noscript></span></span> range from 0 to 1, where 0 indicates no differentiation between samples (populations), and 1 indicates complete differentiation (fixation for alternative alleles) (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hartl and Clark 1997</a>). As all paired ends have a uniform genomic distribution, and were derived from the same template sequences, only small allele frequency differences are expected between samples. A perfect alignment algorithm would detect all positive SNPs (<span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-41.gif"/></noscript></span></span>), and yield a low <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-42.gif"/></noscript></span></span> for all SNPs (<span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-43.gif"/></noscript></span></span>). Based on the simulated data bwa sw mem, novoalign(g), and novoalign(l) showed the best performance, whereas mrfast, bowtie2(l), and ngm(g) performed worst (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034488/-/DC1/TableS4.pdf">Table S4</a>; for allele frequency differences with false positive SNPs, see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034488/-/DC1/TableS5.pdf">Table S5</a>). We noted substantial allele frequency differences when the same data were mapped as paired end and as single end reads, and then compared against each other (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034488/-/DC1/TableS7.pdf">Table S7</a>). ngm(g) and ngm(l) were most suitable for such comparisons between paired and single end reads.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/11/3507/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of allele frequency differences between simulated Pool-Seq data sets with different mapping algorithms. We simulated different paired end Pool-Seq libraries, mapped the reads, and compared the allele frequencies between the libraries using  With this procedure, we evaluated the sensitivity of the alignment algorithm to differences in the distance between paired ends (id), differences in the read length (rl), and differences in the error rates (e). As all libraries were derived from identical template sequences (templates with SNPs and indels), no significant allele frequency differences were expected (). We estimated the number of truly positive SNPs for which allele frequencies could be compared (TP), and the lowest -values in the 0.1% quantile with the most differentiated SNPs. Algorithms are sorted according to performance, with the best performing algorithm shown at the top (maximizing the true positives, and minimizing the  in the outlier quantile). id100, rl100, e1%: 2&#xA0;&#xD7;&#xA0;100&#xA0;bp paired ends, insert size &#xA0;bp, error rate 1%; id300: 2&#xA0;&#xD7;&#xA0;100&#xA0;bp paired ends, insert size &#xA0;bp, error rate 1%; rl50: 2&#xA0;&#xD7;&#xA0;50&#xA0;bp paired ends, insert size &#xA0;bp, error rate 1%; e5%: 2&#xA0;&#xD7;&#xA0;100&#xA0;bp paired ends, insert size &#xA0;bp, error rate 5%." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-864650000" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparison of allele frequency differences between simulated Pool-Seq data sets with different mapping algorithms. We simulated different paired end Pool-Seq libraries, mapped the reads, and compared the allele frequencies between the libraries using &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-44&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; With this procedure, we evaluated the sensitivity of the alignment algorithm to differences in the distance between paired ends (id), differences in the read length (rl), and differences in the error rates (e). As all libraries were derived from identical template sequences (templates with SNPs and indels), no significant allele frequency differences were expected (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-45&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;). We estimated the number of truly positive SNPs for which allele frequencies could be compared (TP), and the lowest &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-46&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;-values in the 0.1% quantile with the most differentiated SNPs. Algorithms are sorted according to performance, with the best performing algorithm shown at the top (maximizing the true positives, and minimizing the &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-47&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; in the outlier quantile). id100, rl100, e1%: 2&#xA0;&#xD7;&#xA0;100&#xA0;bp paired ends, insert size &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-48&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;&#xA0;bp, error rate 1%; id300: 2&#xA0;&#xD7;&#xA0;100&#xA0;bp paired ends, insert size &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-49&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;&#xA0;bp, error rate 1%; rl50: 2&#xA0;&#xD7;&#xA0;50&#xA0;bp paired ends, insert size &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-50&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;&#xA0;bp, error rate 1%; e5%: 2&#xA0;&#xD7;&#xA0;100&#xA0;bp paired ends, insert size &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-51&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;&#xA0;bp, error rate 5%.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/11/3507/F4.medium.gif" width="398" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/11/3507/F4.medium.gif" width="398" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/11/3507/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/11/3507/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469695" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-30" class="first-child">Comparison of allele frequency differences between simulated Pool-Seq data sets with different mapping algorithms. We simulated different paired end Pool-Seq libraries, mapped the reads, and compared the allele frequencies between the libraries using <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/F4/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/F4/embed/mml-math-44.gif"/></noscript></span></span> With this procedure, we evaluated the sensitivity of the alignment algorithm to differences in the distance between paired ends (id), differences in the read length (rl), and differences in the error rates (e). As all libraries were derived from identical template sequences (templates with SNPs and indels), no significant allele frequency differences were expected (<span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/F4/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/F4/embed/mml-math-45.gif"/></noscript></span></span>). We estimated the number of truly positive SNPs for which allele frequencies could be compared (TP), and the lowest <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/F4/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/F4/embed/mml-math-46.gif"/></noscript></span></span>-values in the 0.1% quantile with the most differentiated SNPs. Algorithms are sorted according to performance, with the best performing algorithm shown at the top (maximizing the true positives, and minimizing the <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/F4/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/F4/embed/mml-math-47.gif"/></noscript></span></span> in the outlier quantile). id100, rl100, e1%: 2 × 100 bp paired ends, insert size <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/F4/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/F4/embed/mml-math-48.gif"/></noscript></span></span> bp, error rate 1%; id300: 2 × 100 bp paired ends, insert size <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/F4/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/F4/embed/mml-math-49.gif"/></noscript></span></span> bp, error rate 1%; rl50: 2 × 50 bp paired ends, insert size <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/F4/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/F4/embed/mml-math-50.gif"/></noscript></span></span> bp, error rate 1%; e5%: 2 × 100 bp paired ends, insert size <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/F4/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/F4/embed/mml-math-51.gif"/></noscript></span></span> bp, error rate 5%.</p><div class="sb-div caption-clear"></div></div></div><p id="p-31">Simulated data may not capture all the properties of real data, such as reads having different lengths (after trimming), variable base qualities along reads, and biases in sequencing errors. Therefore, we also evaluated the performance of different alignment algorithms based on <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-52.gif"/></noscript></span></span> between samples using real data. We used two libraries with different read length and insert size prepared from the same genomic DNA (library 1: 2 × 76 bp paired ends, median insert size = 232 bp; library 2: 2 × 120 bp paired ends, median insert size = 396; both prepared from pooled <em>D. simulans</em> flies; see <em>Materials and Methods</em>), trimmed low quality regions from the 3′-ends of reads, and compared allele frequency differences between the samples using <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-53.gif"/></noscript></span></span> As both libraries were prepared from the same genomic DNA, only small allele frequency differences were expected between the samples (<span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/embed/mml-math-54.gif"/></noscript></span></span>). Novoalign(l), novoalign(g), and bwa sw showed the best performance ,while mrfast, segemehl, and ngm(l) performed worst (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/11/3507/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of allele frequency differences between real Pool-Seq data sets with different mapping algorithms. We compared allele frequencies between two paired end libraries with different read length and insert size that were prepared from the same genomic DNA (pooled D. simulans flies). We determined the lowest -values in different quantiles with the most differentiated SNPs. Algorithms are sorted according to performance, with best the performing algorithm shown at the top (minimizing the  in the 0.001% outlier quantile). mrfast generated an invalid output file with these data (an uniform read length was reported despite these reads having varying read lengths)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-864650000" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Comparison of allele frequency differences between real Pool-Seq data sets with different mapping algorithms. We compared allele frequencies between two paired end libraries with different read length and insert size that were prepared from the same genomic DNA (pooled &lt;em&gt;D. simulans&lt;/em&gt; flies). We determined the lowest &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-55&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;-values in different quantiles with the most differentiated SNPs. Algorithms are sorted according to performance, with best the performing algorithm shown at the top (minimizing the &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-56&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; in the 0.001% outlier quantile). mrfast generated an invalid output file with these data (an uniform read length was reported despite these reads having varying read lengths).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/11/3507/F5.medium.gif" width="423" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/11/3507/F5.medium.gif" width="423" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/11/3507/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/11/3507/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469511" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-32" class="first-child">Comparison of allele frequency differences between real Pool-Seq data sets with different mapping algorithms. We compared allele frequencies between two paired end libraries with different read length and insert size that were prepared from the same genomic DNA (pooled <em>D. simulans</em> flies). We determined the lowest <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/F5/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/F5/embed/mml-math-55.gif"/></noscript></span></span>-values in different quantiles with the most differentiated SNPs. Algorithms are sorted according to performance, with best the performing algorithm shown at the top (minimizing the <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/F5/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3507/F5/embed/mml-math-56.gif"/></noscript></span></span> in the 0.001% outlier quantile). mrfast generated an invalid output file with these data (an uniform read length was reported despite these reads having varying read lengths).</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">In summary, when comparing the results of the previous evaluations, we conclude that novoalign(l), novoalign(g), clc4(l), and bwa mem are the most suitable alignment algorithm for Pool-Seq data, whereas mrfast, ngm(l), ngm(g), and bowtie2(g) did not perform as well (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469556/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469556/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Comparison of alignment algorithms for Pool-Seq data: summary across data sets</span><div class="sb-div caption-clear"></div></div></div><p id="p-36">Despite novoalign(g) being one of the most suitable algorithms for Pool-Seq data, a substantial number of artifactual outlier peaks can still be found when comparing the allele frequency between the <em>D. simulans</em> libraries (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). The comparison of different mappers indicated that outlier artifacts are frequently specific to the alignment algorithm (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034488/-/DC1/FigureS3.pdf">Figure S3</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034488/-/DC1/FigureS4.pdf">Figure S4</a>). We reasoned therefore that an intersection of two mappers, recording for every SNP only the least significant result found by any mapper, could overcome this problem. Intersecting the results of bwa and novoalign (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1, A and B</a>), the number of outlier peaks could be substantially reduced (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1C</a>). We tested whether intersecting the results of two mappers results in a more efficient removal of outlier peaks than quality filtering approaches. We evaluated the impact of filtering for (i) mapping quality, (ii) minor allele count, (iii) base quality, (iv) strand-bias, and (v) SNPs called by FreeBayes, a SNP caller well-suited for Pool-Seq data (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Garrison and Marth 2012</a>). We found that, of the approaches tested, intersecting the results of two mappers led to the most pronounced reduction of outlier peaks, while the vast majority of the SNPs were retained (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034488/-/DC1/FigureS10.pdf">Figure S10</a>).</p><p id="p-37">We also tested whether intersecting the results of different mappers preserves the targets of selection using data from an experimental evolution study for C-virus resistance in <em>D. melanogaster</em> (<a id="xref-ref-37-4" class="xref-bibr" href="#ref-37">Martins <em>et al.</em> 2014</a>), and found that the most differentiated loci identified by <a id="xref-ref-37-5" class="xref-bibr" href="#ref-37">Martins <em>et al.</em> (2014)</a> were retained (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034488/-/DC1/FigureS5.pdf">Figure S5</a>). Hence, intersecting the results of different mappers is an efficient strategy for minimizing the number of artifacts, while preserving the targets of selection.</p><p id="p-38">To identify the most suitable combination of mapping algorithms, we used the data from the pooled <em>D. simulans</em> flies, computed all pairwise intersections of the algorithms, and benchmarked them using the number of SNPs and the 0.001% quantile of most differentiated SNPs (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034488/-/DC1/TableS8.pdf">Table S8</a>). ngm(l) combined with bowtie2(g) yielded the least pronounced outlier peaks, with ∼4.15 million shared SNPs [<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034488/-/DC1/TableS8.pdf">Table S8</a>; for Manhattan plots see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034488/-/DC1/FigureS6.pdf">Figure S6</a>]. We note, however, that the best combination of alignment algorithms depends on the threshold—with the 0.01% quantile, novoalign(l) and bowtie2(g) are the best combination (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034488/-/DC1/TableS9.pdf">Table S9</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034488/-/DC1/FigureS7.pdf">Figure S7</a>).</p><div id="sec-7" class="subsection"><h3>Conclusions</h3><p id="p-39">Here, we performed a comprehensive analysis of different alignment algorithms for Pool-Seq data. The evaluation of alignment algorithms is complicated by several issues. First, mapping quality is computed differently between algorithms (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034488/-/DC1/FigureS2.pdf">Figure S2</a>). Thus, the fraction of reads filtered by requiring a certain minimum quality (we used 20) varies among the alignment tools. The fraction of filtered reads will affect both the number of identified true positive SNPs, and the accuracy of the allele frequency estimates; more mapped reads result in a higher number of true SNPs, but the number of ambiguously mapped reads is also increased, which distorts allele frequency estimates. The tradeoff between optimizing the recovery of true SNPs and accuracy of the allele frequency estimates is particularly pronounced for segemehl: no reads could be quality filtered since all reads have a mapping quality of 255, resulting in the highest number of true positive SNPs but poor allele frequency estimates (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034488/-/DC1/TableS2.pdf">Table S2</a>). Despite this complication, we considered quality filtering of reads essential, as this substantially improves allele frequency estimates from Pool-Seq data (for unfiltered results, see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034488/-/DC1/TableS3.pdf">Table S3</a>). Interestingly, the best performing algorithms (<em>e.g.</em>, novoalign and clc) identified the highest number of true positive SNPs, and yielded the most accurate allele frequency estimates (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034488/-/DC1/TableS2.pdf">Table S2</a>), which suggests that the superior performance of these tools is robust with respect to the tradeoff introduced by quality filtering. Furthermore, we found that the top performing algorithm consistently had the highest ratio of true positive SNPs to false positive SNPs, irrespective of the mapping quality threshold used (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034488/-/DC1/FigureS11.pdf">Figure S11</a>).</p><p id="p-40">The choice of alignment parameters is a challenge for the comparison of different mapping algorithms. Whenever feasible, we used default parameters, and modified them only when we considered it necessary to ensure an unbiased comparison (<em>e.g.</em>, when the error rate exceeded the number of allowed mismatches, or when the insert size was larger than the maximum insert size; see <em>Materials and Methods</em>). We note, however, that the performance of each of these algorithms may be improved by fine-tuning the parameters. For example, the performance of bwa aln was substantially improved by using parameters optimized for Pool-Seq data (<a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">Kofler <em>et al.</em> 2011a)</a> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034488/-/DC1/TableS10.pdf">Table S10</a>). While the optimization of mapping parameters for all 14 algorithms is clearly beyond the scope of this manuscript, we made all data, including the simulated ones, publicly available to allow testing of the performance of different mappers and parameters with these data sets.</p><p id="p-41">We did not consider low frequency alleles for the simulated data for three reasons: (1) mapping errors will have the strongest effect with balanced allele frequencies; (2) the identification of low frequency variants in real Pool-seq data are challenging, as sequencing errors can not be reliably distinguished from base substitutions (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Schlötterer <em>et al.</em>, 2014</a>); and (3) low frequency variants will not significantly contribute to any measure of sample differentiation.</p><p id="p-42">Out of the 14 algorithms tested, clc4(g), novoalign(g), bwa mem, clc4(l), and novoalign(l) are the most suitable for Pool-Seq data. The superior performance of novoalign is in agreement with previous work, which found that novoalign yields highly accurate alignments and SNP calls (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bao <em>et al.</em> 2014</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Li and Homer 2010</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Nielsen <em>et al.</em> 2011</a>).</p><p id="p-43">The most striking influence of different alignment algorithms was noted for experimental data differing in insert size and read length. Comparing different libraries from the same genomic DNA, we identified substantial outliers, some of them clustering in peaks, which indicate allele frequency differences at multiple neighboring sites. Since such peaks are a typical signal in genome-wide outlier scans, such as Pool-genome-wide association (GWAS) or evolve and resequence (E&amp;R) studies, these artifacts may lead to false conclusions. Similar artifacts were also seen when the data were mapped as single reads (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034488/-/DC1/FigureS8.pdf">Figure S8</a>), suggesting that this is not an artifact of paired end mapping. Assuming that true allele frequency differences between samples should be identified with most alignment tools, whereas artifacts should be found with only a few algorithms, we propose intersecting multiple alignment algorithms. We noticed a clear improvement when intersecting two alignment algorithms, but, depending on the evaluation criteria, different pairs of algorithms perform best. These results are consistent with other studies, which also found that the combination of mapping algorithms and/or variant calling pipelines may yield superior results (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Bao <em>et al.</em> 2014</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Field <em>et al.</em> 2015</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">O’Rawe <em>et al.</em> 2013</a>).</p><p id="p-44">Our approach to intersect algorithms is based on the least significant allele frequency differences between two samples. It is straightforward to extend this approach to studies that rely on multiple samples, such as replicated Pool-GWAS experiments or E&amp;R studies (for example, see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034488/-/DC1/FigureS5.pdf">Figure S5</a>), provided that it is feasible to collapse allele frequency differences between multiple samples into a single representative measure [<em>e.g.</em> <em>P</em>-value from a cmh-test (<a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Orozco-terWengel <em>et al.</em> 2012</a>)]. In this case, again the least significant value found by any mapper may be used. However, this strategy cannot be applied to Pool-Seq data from single populations (<em>e.g.</em>, <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Asgharian <em>et al.</em> 2015</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Boitard <em>et al.</em> 2013</a>; <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Nolte <em>et al.</em> 2012</a>). One possibility to avoid mapping artifacts for single population Pool-Seq data may be to filter SNPs with incongruent allele frequency estimates among multiple mappers. Given that most artifacts were observed when libraries with different insert sizes and read lengths were compared (cf. <a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034488/-/DC1/FigureS9.pdf">Figure S9</a>), we recommend using a single consistent sequencing strategy for all Pool-Seq libraries, whenever possible. We additionally propose to use a single consistent mapping pipeline for all Pool-Seq data, as mixing samples aligned with different tools, algorithms, parameters, or even versions, of the same tool, leads to elevated levels of outlier peaks (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034488/-/DC1/TableS10.pdf">Table S10</a>).</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-45">We thank all members of the Institute of Population Genetics for feedback and support. This work was supported by the European Research Council grant ”ArchAdapt” and Austrian Science Funds (FWF-W1225).</p><p id="p-46">Author contributions: R.K. and C.S. conceived the study. A.M.L., R.K., P.N., and K.A.O. analyzed the data. R.K. developed the scripts. R.K., A.M.L., and C.S. wrote the article.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-6"><p id="p-47">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034488/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.034488/-/DC1</a>.</p></li><li class="fn" id="fn-7"><p id="p-48"><em>Communicating editor: J. Ross-Ibarra</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-05-20"><span class="received-label">Received </span>May 20, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-08-22"><span class="accepted-label">Accepted </span>August 22, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Kofler <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-2" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3507.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alkan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kidd</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marques-bonet</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aksay</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hormozdiari</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Personalized copy-number and segmental duplication maps using next-generation sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">41</span>(<span class="cit-issue">10</span>): <span class="cit-fpage">1061</span>–<span class="cit-lpage">1067</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D41%26rft.spage%253D1061%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3507.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Asgharian</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lysenkov</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scobeyeva</span>  <span class="cit-name-given-names">V. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reisen</span>  <span class="cit-name-given-names">W. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nuzhdin</span>  <span class="cit-name-given-names">S. V.</span></span></li></ol><cite> <span class="cit-pub-date">2015</span> <span class="cit-article-title">Evolutionary genomics of <em>Culex pipiens</em>: global and local adaptations associated with climate, life-history traits and anthropogenic factors.</span> <abbr class="cit-jnl-abbrev">Proc. Biol. Sci.</abbr> <span class="cit-vol">282</span>(<span class="cit-issue">1810</span>): 20150728.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3507.3" data-doi="10.1038/nature11837"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Axelsson</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ratnakumar</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arendt</span>  <span class="cit-name-given-names">M.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maqbool</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Webster</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The genomic signature of dog domestication reveals adaptation to a starch-rich diet.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">495</span>(<span class="cit-issue">7441</span>): <span class="cit-fpage">360</span>–<span class="cit-lpage">364</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D495%26rft.spage%253D360%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11837%26rft_id%253Dinfo%253Apmid%252F23354050%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11837&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23354050&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3507.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000316650500041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3507.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bao</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andrade</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kibbe</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Review of current methods, applications, and data management for the bioinformatics analysis of whole exome sequencing.</span> <abbr class="cit-jnl-abbrev">Cancer Inform.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">67</span>–<span class="cit-lpage">83</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCancer%2BInform.%26rft.volume%253D13%26rft.spage%253D67%26rft_id%253Dinfo%253Apmid%252F25288881%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=25288881&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3507.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3507.5" data-doi="10.1371/journal.pgen.1003534"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bastide</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Betancourt</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nolte</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tobler</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stöbe</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A genome-wide, fine-scale map of natural pigmentation variation in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">e1003534</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003534%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003534%26rft_id%253Dinfo%253Apmid%252F23754958%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003534&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23754958&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3507.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3507.6" data-doi="10.1371/journal.pgen.1004775"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bergland</span>  <span class="cit-name-given-names">A. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Behrman</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Brien</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic evidence of rapid and stable adaptive oscillations over seasonal time scales in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">e1004775</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004775%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004775%26rft_id%253Dinfo%253Apmid%252F25375361%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004775&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25375361&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3507.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3507.7" data-doi="10.1111/1755-0998.12063"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boitard</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kofler</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Françoise</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robelin</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlötterer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Pool-hmm: a Python program for estimating the allele frequency spectrum and detecting selective sweeps from next generation sequencing of pooled samples.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">13</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">337</span>–<span class="cit-lpage">340</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D13%26rft.spage%253D337%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12063%26rft_id%253Dinfo%253Apmid%252F23311589%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12063&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23311589&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3507.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3507.8" data-doi="10.1093/molbev/msu397"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheeseman</span>  <span class="cit-name-given-names">I. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDew-White</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phyo</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sriprawat</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nosten</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Pooled sequencing and rare variant association tests for identifying the determinants of emerging drug resistance in malaria parasites.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">32</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">1080</span>–<span class="cit-lpage">1090</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu397%26rft_id%253Dinfo%253Apmid%252F25534029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjMyLzQvMTA4MCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzUwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-3" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-web" id="cit-6.11.3507.9"><div class="cit-metadata unstructured">CLC bio, 2015 CLC Assembly Cell. Available at: <a href="http://www.clcbio.com/products/clc-assembly-cell/">http://www.clcbio.com/products/clc-assembly-cell/</a>. Accessed: January 18, 2016.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3507.10" data-doi="10.1093/bioinformatics/btp579"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Degner</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marioni</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pai</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pickrell</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nkadori</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Effect of read-mapping biases on detecting allele-specific expression from RNA-sequencing data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>(<span class="cit-issue">24</span>): <span class="cit-fpage">3207</span>–<span class="cit-lpage">3212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp579%26rft_id%253Dinfo%253Apmid%252F19808877%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMjQvMzIwNyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzUwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3507.11" data-doi="10.1038/nature10738"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ley</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larson</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koboldt</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Clonal evolution in relapsed acutemyeloid leukaemia revealed by whole-genome sequencing.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">481</span>(<span class="cit-issue">7382</span>): <span class="cit-fpage">506</span>–<span class="cit-lpage">510</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDing%26rft.auinit1%253DL.%26rft.volume%253D481%26rft.issue%253D7382%26rft.spage%253D506%26rft.epage%253D510%26rft.atitle%253DClonal%2Bevolution%2Bin%2Brelapsed%2Bacute%2Bmyeloid%2Bleukaemia%2Brevealed%2Bby%2Bwhole-genome%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10738%26rft_id%253Dinfo%253Apmid%252F22237025%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10738&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22237025&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3507.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000299471800040&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3507.12" data-doi="10.1111/mec.12522"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ferretti</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramos-Onsins</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Enciso</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Population genomics from pool sequencing.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>(<span class="cit-issue">22</span>): <span class="cit-fpage">5561</span>–<span class="cit-lpage">5576</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D5561%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12522&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000329786700006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3507.13" data-doi="10.1371/journal.pone.0143199"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Field</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cho</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andrews</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodnow</span>  <span class="cit-name-given-names">C. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Reliably detecting clinically important variants requires both combined variant calls and optimized filtering strategies.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">e0143199</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0143199%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0143199%26rft_id%253Dinfo%253Apmid%252F26600436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0143199&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26600436&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3507.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3507.14" data-doi="10.1534/genetics.110.114397"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Futschik</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlötterer</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The next generation of molecular markers from massively parallel sequencing of pooled DNA samples.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">207</span>–<span class="cit-lpage">218</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.114397%26rft_id%253Dinfo%253Apmid%252F20457880%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg2LzEvMjA3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNTA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-web" id="cit-6.11.3507.15"><div class="cit-metadata unstructured">Garrison, E., and G. Marth, 2012 Haplotype-based variant detection from short-read sequencing. arXiv:1207.3907v2 [q-bio.GN].</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3507.16" data-doi="10.1111/mec.12360"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gautier</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foucaud</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gharbi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cézard</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Estimation of population allele frequencies from next-generation sequencing data: pool- <em>vs.</em> individual-based genotyping.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>(<span class="cit-issue">14</span>): <span class="cit-fpage">3766</span>–<span class="cit-lpage">3779</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D3766%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12360%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12360&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000321552800011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-book" id="cit-6.11.3507.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hartl</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">A. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-source"><em>Principles of Population Genetics</em>.</span> <span class="cit-publ-name">Sinauer Associates</span>, <span class="cit-publ-loc">Sunderland, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-2" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3507.18" data-doi="10.1371/journal.pcbi.1000502"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Otto</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kurtz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharma</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khaitovich</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast mapping of short sequences with mismatches, insertions and deletions using index structures.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000502</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DHoffmann%26rft.auinit1%253DS.%26rft.volume%253D5%26rft.issue%253D9%26rft.spage%253De1000502%26rft.epage%253De1000502%26rft.atitle%253DFast%2Bmapping%2Bof%2Bshort%2Bsequences%2Bwith%2Bmismatches%252C%2Binsertions%2Band%2Bdeletions%2Busing%2Bindex%2Bstructures.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1000502%26rft_id%253Dinfo%253Apmid%252F19750212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1000502&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19750212&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3507.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3507.19" data-doi="10.1093/molbev/mst221"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kofler</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlötterer</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A guide for the design of evolve and resequencing studies.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">31</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">474</span>–<span class="cit-lpage">483</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst221%26rft_id%253Dinfo%253Apmid%252F24214537%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjMxLzIvNDc0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNTA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3507.20" data-doi="10.1371/journal.pone.0015925"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kofler</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Orozco-terWengel</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Maio</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pandey</span>  <span class="cit-name-given-names">R. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nolte</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span><span class="cit-comment">a</span> <span class="cit-article-title">PoPoolation: a toolbox for population genetic analysis of next generation sequencing data from pooled individuals.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">e15925</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DKofler%26rft.auinit1%253DR.%26rft.volume%253D6%26rft.issue%253D1%26rft.spage%253De15925%26rft.epage%253De15925%26rft.atitle%253DPoPoolation%253A%2Ba%2Btoolbox%2Bfor%2Bpopulation%2Bgenetic%2Banalysis%2Bof%2Bnext%2Bgeneration%2Bsequencing%2Bdata%2Bfrom%2Bpooled%2Bindividuals.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0015925%26rft_id%253Dinfo%253Apmid%252F21253599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0015925&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21253599&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3507.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3507.21" data-doi="10.1093/bioinformatics/btr589"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kofler</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pandey</span>  <span class="cit-name-given-names">R. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlötterer</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span><span class="cit-comment">b</span> <span class="cit-article-title">PoPoolation2: identifying differentiation between populations using sequencing of pooled DNA samples (Pool-Seq).</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>(<span class="cit-issue">24</span>): <span class="cit-fpage">3435</span>–<span class="cit-lpage">3436</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr589%26rft_id%253Dinfo%253Apmid%252F22025480%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjcvMjQvMzQzNSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzUwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3507.22" data-doi="10.1371/journal.pgen.1002487"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kofler</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Betancourt</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlötterer</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Sequencing of pooled DNA samples (Pool-Seq) uncovers complex dynamics of transposable element insertions in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">e1002487</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DKofler%26rft.auinit1%253DR.%26rft.volume%253D8%26rft.issue%253D1%26rft.spage%253De1002487%26rft.epage%253De1002487%26rft.atitle%253DSequencing%2Bof%2Bpooled%2BDNA%2Bsamples%2B%2528Pool-Seq%2529%2Buncovers%2Bcomplex%2Bdynamics%2Bof%2Btransposable%2Belement%2Binsertions%2Bin%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002487%26rft_id%253Dinfo%253Apmid%252F22291611%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002487&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22291611&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3507.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3507.23" data-doi="10.1073/pnas.1500758112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kofler</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nolte</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Betancourt</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlötterer</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span><span class="cit-comment">a</span> <span class="cit-article-title">The recent invasion of natural <em>Drosophila simulans</em> populations by the P-element.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">112</span>(<span class="cit-issue">21</span>): <span class="cit-fpage">6659</span>–<span class="cit-lpage">6663</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1500758112%26rft_id%253Dinfo%253Apmid%252F25964349%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEyLzIxLzY2NTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzExLzM1MDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3507.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kofler</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nolte</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlötterer</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span><span class="cit-comment">b</span> <span class="cit-article-title">The impact of library preparation protocols on the consistency of allele frequency estimates in Pool-Seq data.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">16</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">118</span>–<span class="cit-lpage">122</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D16%26rft.spage%253D118%26rft_id%253Dinfo%253Apmid%252F26014582%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=26014582&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3507.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3507.25" data-doi="10.1073/pnas.1216128109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lamichhaney</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrio</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rafati</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sundström</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rubin</span>  <span class="cit-name-given-names">C.-J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Population-scale sequencing reveals genetic differentiation due to local adaptation in atlantic herring.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>(<span class="cit-issue">47</span>): <span class="cit-fpage">19345</span>–<span class="cit-lpage">19350</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1216128109%26rft_id%253Dinfo%253Apmid%252F23134729%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzQ3LzE5MzQ1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNTA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3507.26" data-doi="10.1038/nature12344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lang</span>  <span class="cit-name-given-names">G. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rice</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickman</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sodergren</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weinstock</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Pervasive genetic hitchhiking and clonal interference in forty evolving yeast populations.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">500</span>(<span class="cit-issue">7464</span>): <span class="cit-fpage">571</span>–<span class="cit-lpage">574</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D500%26rft.spage%253D571%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12344%26rft_id%253Dinfo%253Apmid%252F23873039%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12344&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23873039&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3507.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000323625900033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-3" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3507.27" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast gapped-read alignment with Bowtie 2.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">357</span>–<span class="cit-lpage">359</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3507.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-3" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3507.28" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>(<span class="cit-issue">14</span>): <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTQvMTc1NCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzUwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-2" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3507.29" data-doi="10.1093/bioinformatics/btp698"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Fast and accurate long-read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">589</span>–<span class="cit-lpage">595</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp698%26rft_id%253Dinfo%253Apmid%252F20080505%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNi81LzU4OSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzUwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3507.30" data-doi="10.1093/bib/bbq015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Homer</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A survey of sequence alignment algorithms for next-generation sequencing.</span> <abbr class="cit-jnl-abbrev">Brief. Bioinform.</abbr> <span class="cit-vol">11</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">473</span>–<span class="cit-lpage">483</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BBioinform.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbib%252Fbbq015%26rft_id%253Dinfo%253Apmid%252F20460430%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYmliIjtzOjU6InJlc2lkIjtzOjg6IjExLzUvNDczIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNTA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3507.31" data-doi="10.1101/gr.078212.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Mapping short DNA sequencing reads and calling variants using mapping quality scores.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">1851</span>–<span class="cit-lpage">1858</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.078212.108%26rft_id%253Dinfo%253Apmid%252F18714091%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxOC8xMS8xODUxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNTA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3507.32" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">a</span> <span class="cit-article-title">The sequence alignment/map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>(<span class="cit-issue">16</span>): <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTYvMjA3OCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzUwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3507.33" data-doi="10.1093/bioinformatics/btp336"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lam</span>  <span class="cit-name-given-names">T.-W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yiu</span>  <span class="cit-name-given-names">S.-M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">b</span> <span class="cit-article-title">Soap2: an improved ultrafast tool for short read alignment.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>(<span class="cit-issue">15</span>): <span class="cit-fpage">1966</span>–<span class="cit-lpage">1967</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp336%26rft_id%253Dinfo%253Apmid%252F19497933%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTUvMTk2NiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzUwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3507.34" data-doi="10.1093/nar/gkv533"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lim</span>  <span class="cit-name-given-names">J.-Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tennakoon</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guan</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sung</span>  <span class="cit-name-given-names">W.-K.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Batalign: an incremental method for accurate alignment of sequencing reads.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">43</span>(<span class="cit-issue">16</span>): <span class="cit-fpage">e107</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv533%26rft_id%253Dinfo%253Apmid%252F26170239%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0My8xNi9lMTA3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNTA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3507.35" data-doi="10.1101/gr.111120.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lunter</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodson</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Stampy: a statistical algorithm for sensitive and fast mapping of Illumina sequence reads.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">936</span>–<span class="cit-lpage">939</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.111120.110%26rft_id%253Dinfo%253Apmid%252F20980556%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIxLzYvOTM2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNTA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3507.36" data-doi="10.1038/nmeth.2221"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marco-Sola</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sammeth</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guigó</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ribeca</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The GEM mapper: fast, accurate and versatile alignment by filtration.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">1185</span>–<span class="cit-lpage">1188</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DMarco-Sola%26rft.volume%253D9%26rft.spage%253D1185%26rft.atitle%253DThe%2BGEM%2Bmapper%253A%2Bfast%252C%2Baccurate%2Band%2Bversatile%2Balignment%2Bby%2Bfiltration.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2221%26rft_id%253Dinfo%253Apmid%252F23103880%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2221&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23103880&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3507.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312093500020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3507.37" data-doi="10.1073/pnas.1418561111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martins</span>  <span class="cit-name-given-names">N. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faria</span>  <span class="cit-name-given-names">V. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nolte</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlötterer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Teixeira</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Host adaptation to viruses relies on few genes with different cross-resistance properties.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">111</span>(<span class="cit-issue">43</span>): <span class="cit-fpage">15597</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1418561111%26rft_id%253Dinfo%253Apmid%252F25298533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTExLzQzLzE1NTk3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNTA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3507.38" data-doi="10.1111/mec.12693"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nevado</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramos-Onsins</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perez-Enciso</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Resequencing studies of nonmodel organisms using closely related reference genomes: optimal experimental designs and bioinformatics approaches for population genomics.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">23</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">1764</span>–<span class="cit-lpage">1779</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D23%26rft.spage%253D1764%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12693%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12693&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000333333500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3507.39" data-doi="10.1038/nrg2986"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paul</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albrechtsen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Y. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genotype and SNP calling from next-generation sequencing data.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">443</span>–<span class="cit-lpage">451</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DNielsen%26rft.auinit1%253DR.%26rft.volume%253D12%26rft.issue%253D6%26rft.spage%253D443%26rft.epage%253D451%26rft.atitle%253DGenotype%2Band%2BSNP%2Bcalling%2Bfrom%2Bnext-generation%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2986%26rft_id%253Dinfo%253Apmid%252F21587300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2986&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21587300&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3507.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3507.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nolte</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pandey</span>  <span class="cit-name-given-names">R. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kofler</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlötterer</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-wide patterns of natural variation reveal strong selective sweeps and ongoing genomic conflict in <em>Drosophila mauritiana</em>.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">99</span>–<span class="cit-lpage">110</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft.volume%253D23%26rft.spage%253D99%26rft_id%253Dinfo%253Apmid%252F23051690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23051690&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3507.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-3" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-web" id="cit-6.11.3507.41"><div class="cit-metadata unstructured">Novocraft, 2014  NovoAlign. Available at: <a href="http://www.novocraft.com/products/novoalign/">http://www.novocraft.com/products/novoalign/</a>. Accessed: December 10, 2015.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3507.42" data-doi="10.1186/gm432"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">O’Rawe</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Low concordance of multiple variant-calling pipelines: practical implications for exome and genome sequencing.</span> <abbr class="cit-jnl-abbrev">Genome Med.</abbr> <span class="cit-vol">5</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">28</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BMed.%26rft.volume%253D5%26rft.spage%253D28%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgm432%26rft_id%253Dinfo%253Apmid%252F23537139%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gm432&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23537139&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3507.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3507.43" data-doi="10.1111/j.1365-294X.2012.05673.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Orozco-terWengel</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kapun</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nolte</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kofler</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flatt</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Adaptation of <em>Drosophila</em> to a novel laboratory environment reveals temporally heterogeneous trajectories of selected alleles.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">21</span>(<span class="cit-issue">20</span>): <span class="cit-fpage">4931</span>–<span class="cit-lpage">4941</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DOrozco-Terwengel%26rft.auinit1%253DP.%26rft.volume%253D21%26rft.issue%253D20%26rft.spage%253D4931%26rft.epage%253D4941%26rft.atitle%253DAdaptation%2Bof%2BDrosophila%2Bto%2Ba%2Bnovel%2Blaboratory%2Benvironment%2Breveals%2Btemporally%2Bheterogeneous%2Btrajectories%2Bof%2Bselected%2Balleles.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2012.05673.x%26rft_id%253Dinfo%253Apmid%252F22726122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2012.05673.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22726122&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3507.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309888200005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3507.44" data-doi="10.1111/1755-0998.12297"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Palmieri</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nolte</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlötterer</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genome assembly and annotation of a <em>Drosophila simulans</em> strain from Madagascar.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">15</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">372</span>–<span class="cit-lpage">381</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D15%26rft.spage%253D372%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12297%26rft_id%253Dinfo%253Apmid%252F24961367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12297&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24961367&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3507.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-book" id="cit-6.11.3507.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Core Team</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-source"><em>R: A Language and Environment for Statistical Computing</em>.</span> <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3507.46" data-doi="10.1371/journal.pone.0080422"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rellstab</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zoller</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tedder</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gugerli</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">M. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Validation of SNP allele frequencies determined by pooled next-generation sequencing in natural populations of a non-model plant species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">e80422</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De80422%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0080422%26rft_id%253Dinfo%253Apmid%252F24244686%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0080422&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24244686&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3507.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3507.47" data-doi="10.1038/nature08832"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rubin</span>  <span class="cit-name-given-names">C.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zody</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eriksson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meadows</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherwood</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Whole-genome resequencing reveals loci under selection during chicken domestication.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">464</span>(<span class="cit-issue">7288</span>): <span class="cit-fpage">587</span>–<span class="cit-lpage">591</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DRubin%26rft.auinit1%253DC.%2BJ.%26rft.volume%253D464%26rft.issue%253D7288%26rft.spage%253D587%26rft.epage%253D591%26rft.atitle%253DWhole-genome%2Bresequencing%2Breveals%2Bloci%2Bunder%2Bselection%2Bduring%2Bchicken%2Bdomestication.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08832%26rft_id%253Dinfo%253Apmid%252F20220755%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08832&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20220755&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3507.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275974200047&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3507.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schlötterer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tobler</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kofler</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nolte</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Sequencing pools of individuals—mining genome-wide polymorphism data without big funding.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">15</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">749</span>–<span class="cit-lpage">763</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D15%26rft.spage%253D749%26rft_id%253Dinfo%253Apmid%252F25246196%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=25246196&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3507.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3507.49" data-doi="10.1038/nmeth0809-550"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schneeberger</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ossowski</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lanz</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Juul</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petersen</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">SHOREMAP: simultaneous mapping and mutation identification by deep sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">6</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">550</span>–<span class="cit-lpage">551</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DSchneeberger%26rft.auinit1%253DK.%26rft.volume%253D6%26rft.issue%253D8%26rft.spage%253D550%26rft.epage%253D551%26rft.atitle%253DSHOREmap%253A%2Bsimultaneous%2Bmapping%2Band%2Bmutation%2Bidentification%2Bby%2Bdeep%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth0809-550%26rft_id%253Dinfo%253Apmid%252F19644454%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth0809-550&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19644454&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3507.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268493700004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-3" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3507.50" data-doi="10.1093/bioinformatics/btt468"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sedlazeck</span>  <span class="cit-name-given-names">F. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rescheneder</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">von Haeseler</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">NextGenMap: fast and accurate read mapping in highly polymorphic genomes.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">29</span>(<span class="cit-issue">21</span>): <span class="cit-fpage">2790</span>–<span class="cit-lpage">2791</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt468%26rft_id%253Dinfo%253Apmid%252F23975764%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjkvMjEvMjc5MCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzUwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3507.51" data-doi="10.1093/bib/bbs017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thorvaldsdóttir</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mesirov</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Integrative Genomics Viewer (IGV): high-performance genomics data visualization and exploration.</span> <abbr class="cit-jnl-abbrev">Brief. Bioinform.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">178</span>–<span class="cit-lpage">192</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BBioinform.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbib%252Fbbs017%26rft_id%253Dinfo%253Apmid%252F22517427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYmliIjtzOjU6InJlc2lkIjtzOjg6IjE0LzIvMTc4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNTA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3507.52" data-doi="10.1093/molbev/mst205"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tobler</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franssen</span>  <span class="cit-name-given-names">S. U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kofler</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Orozco-terWengel</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nolte</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Massive habitat-specific genomic response in <em>D. melanogaster</em> populations during experimental evolution in hot and cold environments.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">364</span>–<span class="cit-lpage">375</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst205%26rft_id%253Dinfo%253Apmid%252F24150039%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjMxLzIvMzY0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNTA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3507.53" data-doi="10.1038/ng.515"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bourne</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">VonWettberg</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">T. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nuzhdin</span>  <span class="cit-name-given-names">S. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Population resequencing reveals local adaptation of <em>Arabidopsis lyrata</em> to serpentine soils.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">260</span>–<span class="cit-lpage">263</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DTurner%26rft.auinit1%253DT.%2BL.%26rft.volume%253D42%26rft.issue%253D3%26rft.spage%253D260%26rft.epage%253D263%26rft.atitle%253DPopulation%2Bresequencing%2Breveals%2Blocal%2Badaptation%2Bof%2BArabidopsis%2Blyrata%2Bto%2Bserpentine%2Bsoils.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.515%26rft_id%253Dinfo%253Apmid%252F20101244%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.515&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20101244&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3507.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274912400017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3507.54" data-doi="10.1093/bioinformatics/btq057"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nacu</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Fast and SNP-tolerant detection of complex variants and splicing in short reads.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">873</span>–<span class="cit-lpage">881</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq057%26rft_id%253Dinfo%253Apmid%252F20147302%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNi83Lzg3MyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzUwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/11/3507.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/11/3497" title="Marker-Based Estimates Reveal Significant Nonadditive Effects in Clonally Propagated Cassava (Manihot esculenta): Implications for the Prediction of Total Genetic Value and the Selection of Varieties" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/11/3517" title="A Pedigree-Based Map of Recombination in the Domestic Dog Genome" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/11"><span>Volume 6 Issue 11, November 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/11.cover-source.jpg?itok=hTYn4PRy" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/11/3507.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-owabC0rOSSNnASMjANS33fTUWRqSqivORJuQUgVcviY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/11/3507" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/11/3507" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/11/3507" class="active" data-icon-position="" data-hide-link-title="0">Suitability of Different Mapping Algorithms for Genome-Wide Polymorphism Scans with Pool-Seq Data</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460473" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-r93heefK4udJd_RBifof5kjI_TV4CVa65WEJKa2d7aY" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/11/3507.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/11/3507" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-viiNVHc-A-TfZUhtY4DyWGFpiT5uJ5TuzGs-_ZZb7KY" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/11/3507" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/11/3507.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460473" id="node460473--2" data-pisa="ggg;6/11/3507" data-pisa-master="ggg;g3.116.034488" data-apath="/ggg/6/11/3507.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Suitability of Different Mapping Algorithms for Genome-Wide Polymorphism Scans with Pool-Seq Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-9960-7248" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Robert Kofler</span>, <span class="highwire-citation-author" data-delta="1">Anna Maria Langmüller</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-4923-0993" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Pierre Nouhaud</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-5026-4168" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Kathrin Anna Otte</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-4710-6526" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Christian Schlötterer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3507-3515;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.034488 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460473--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Robert Kofler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut für Populationsgenetik, Vetmeduni Vienna, Veterinärplatz 1, 1210 Wien 1210, Austria</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BKofler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kofler%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARobert%2BKofler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9960-7248" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Robert Kofler</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Anna Maria Langmüller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut für Populationsgenetik, Vetmeduni Vienna, Veterinärplatz 1, 1210 Wien 1210, Austria</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Vienna Graduate School of Population Genetics, 1210, Austria</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anna%2BMaria%2BLangm%C3%BCller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Langm%C3%BCller%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnna%2BMaria%2BLangm%25C3%25BCller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Pierre Nouhaud </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut für Populationsgenetik, Vetmeduni Vienna, Veterinärplatz 1, 1210 Wien 1210, Austria</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pierre%2BNouhaud%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nouhaud%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APierre%2BNouhaud%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4923-0993" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Pierre Nouhaud</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Kathrin Anna Otte </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut für Populationsgenetik, Vetmeduni Vienna, Veterinärplatz 1, 1210 Wien 1210, Austria</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kathrin%2BAnna%2BOtte%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Otte%20KA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKathrin%2BAnna%2BOtte%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5026-4168" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kathrin Anna Otte</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Christian Schlötterer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut für Populationsgenetik, Vetmeduni Vienna, Veterinärplatz 1, 1210 Wien 1210, Austria</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christian%2BSchl%C3%B6tterer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schl%C3%B6tterer%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChristian%2BSchl%25C3%25B6tterer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-4710-6526" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Christian Schlötterer</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:christian.schloetterer@vetmeduni.ac.at" class="" data-icon-position="" data-hide-link-title="0">christian.schloetterer@vetmeduni.ac.at</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/11/3507" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/11/3507&amp;title=Suitability%20of%20Different%20Mapping%20Algorithms%20for%20Genome-Wide%20Polymorphism%20Scans%20with%20Pool-Seq%20Data" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/11/3507&amp;title=Suitability%20of%20Different%20Mapping%20Algorithms%20for%20Genome-Wide%20Polymorphism%20Scans%20with%20Pool-Seq%20Data" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/11/3507&amp;title=Suitability%20of%20Different%20Mapping%20Algorithms%20for%20Genome-Wide%20Polymorphism%20Scans%20with%20Pool-Seq%20Data" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/11/3507&amp;text=Suitability%20of%20Different%20Mapping%20Algorithms%20for%20Genome-Wide%20Polymorphism%20Scans%20with%20Pool-Seq%20Data" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/11/3507&amp;title=Suitability%20of%20Different%20Mapping%20Algorithms%20for%20Genome-Wide%20Polymorphism%20Scans%20with%20Pool-Seq%20Data" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/11/3507&amp;t=Suitability%20of%20Different%20Mapping%20Algorithms%20for%20Genome-Wide%20Polymorphism%20Scans%20with%20Pool-Seq%20Data" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/11/3507&amp;title=Suitability%20of%20Different%20Mapping%20Algorithms%20for%20Genome-Wide%20Polymorphism%20Scans%20with%20Pool-Seq%20Data" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/11/3507&amp;title=Suitability%20of%20Different%20Mapping%20Algorithms%20for%20Genome-Wide%20Polymorphism%20Scans%20with%20Pool-Seq%20Data" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460473/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460473" id="node460473--3" data-pisa="ggg;6/11/3507" data-pisa-master="ggg;g3.116.034488" data-apath="/ggg/6/11/3507.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Suitability of Different Mapping Algorithms for Genome-Wide Polymorphism Scans with Pool-Seq Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-9960-7248" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Robert Kofler</span>, <span class="highwire-citation-author" data-delta="1">Anna Maria Langmüller</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-4923-0993" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Pierre Nouhaud</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-5026-4168" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Kathrin Anna Otte</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-4710-6526" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Christian Schlötterer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3507-3515;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.034488 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460473--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Robert Kofler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut für Populationsgenetik, Vetmeduni Vienna, Veterinärplatz 1, 1210 Wien 1210, Austria</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BKofler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kofler%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARobert%2BKofler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9960-7248" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Robert Kofler</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Anna Maria Langmüller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut für Populationsgenetik, Vetmeduni Vienna, Veterinärplatz 1, 1210 Wien 1210, Austria</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Vienna Graduate School of Population Genetics, 1210, Austria</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anna%2BMaria%2BLangm%C3%BCller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Langm%C3%BCller%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnna%2BMaria%2BLangm%25C3%25BCller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Pierre Nouhaud </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut für Populationsgenetik, Vetmeduni Vienna, Veterinärplatz 1, 1210 Wien 1210, Austria</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pierre%2BNouhaud%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nouhaud%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APierre%2BNouhaud%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4923-0993" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Pierre Nouhaud</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Kathrin Anna Otte </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut für Populationsgenetik, Vetmeduni Vienna, Veterinärplatz 1, 1210 Wien 1210, Austria</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kathrin%2BAnna%2BOtte%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Otte%20KA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKathrin%2BAnna%2BOtte%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5026-4168" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kathrin Anna Otte</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Christian Schlötterer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut für Populationsgenetik, Vetmeduni Vienna, Veterinärplatz 1, 1210 Wien 1210, Austria</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christian%2BSchl%C3%B6tterer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schl%C3%B6tterer%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChristian%2BSchl%25C3%25B6tterer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-4710-6526" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Christian Schlötterer</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:christian.schloetterer@vetmeduni.ac.at" class="" data-icon-position="" data-hide-link-title="0">christian.schloetterer@vetmeduni.ac.at</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3507&amp;rft.epage=3515&amp;rft.atitle=Suitability%20of%20Different%20Mapping%20Algorithms%20for%20Genome-Wide%20Polymorphism%20Scans%20with%20Pool-Seq%20Data&amp;rft.volume=6&amp;rft.issue=11&amp;rft.date=2016-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Kofler%2C+Robert&amp;rft.au=Langm%C3%BCller%2C+Anna+Maria&amp;rft.au=Nouhaud%2C+Pierre&amp;rft.au=Otte%2C+Kathrin+Anna&amp;rft.au=Schl%C3%B6tterer%2C+Christian"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460473/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460473/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460473/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460473/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460473/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460473/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460473/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460473/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460473/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460473/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460473/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460473/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/11/3507" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Material and Methods</a></li><li class="odd"><a href="#sec-6" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/11/3507.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/11/3507.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/11/3507.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/11/3507.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
