<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/11/3671" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/11/3671.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/11/3671.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/11/3671.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/6/11/3671/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/11/3671.atom" />
<meta name="HW.pisa" content="ggg;6/11/3671" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Transcriptome Profiling in Rat Inbred Strains and Experimental Cross Reveals Discrepant Genetic Architecture of Genome-Wide Gene Expression" />
<meta name="DC.Identifier" content="10.1534/g3.116.033274" />
<meta name="DC.Date" content="2016-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Kaisaki et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="To test the impact of genetic heterogeneity on cis - and trans -mediated mechanisms of gene expression regulation, we profiled the transcriptome of adipose tissue in 20 inbred congenic strains derived from diabetic Goto–Kakizaki (GK) rats and Brown–Norway (BN) controls, which contain well-defined blocks (1–183 Mb) of genetic polymorphisms, and in 123 genetically heterogeneous rats of an (GK × BN)F2 offspring. Within each congenic we identified 73–1351 differentially expressed genes (DEGs), only 7.7% of which mapped within the congenic blocks, and which may be regulated in cis . The remainder localized outside the blocks, and therefore must be regulated in trans . Most trans -regulated genes exhibited approximately twofold expression changes, consistent with monoallelic expression. Altered biological pathways were replicated between congenic strains sharing blocks of genetic polymorphisms, but polymorphisms at different loci also had redundant effects on transcription of common distant genes and pathways. We mapped 2735 expression quantitative trait loci (eQTL) in the F2 cross, including 26% predominantly cis -regulated genes, which validated DEGs in congenic strains. A hotspot of &gt;300 eQTL in a 10 cM region of chromosome 1 was enriched in DEGs in a congenic strain. However, many DEGs among GK, BN and congenic strains did not replicate as eQTL in F2 hybrids, demonstrating distinct mechanisms of gene expression when alleles segregate in an outbred population or are fixed homozygous across the entire genome or in short genomic regions. Our analysis provides conceptual advances in our understanding of the complex architecture of genome expression and pathway regulation, and suggests a prominent impact of epistasis and monoallelic expression on gene transcription." />
<meta name="DC.Contributor" content="Pamela J. Kaisaki" />
<meta name="DC.Contributor" content="Georg W. Otto" />
<meta name="DC.Contributor" content="Karène Argoud" />
<meta name="DC.Contributor" content="Stephan C. Collins" />
<meta name="DC.Contributor" content="Robert H. Wallis" />
<meta name="DC.Contributor" content="Steven P. Wilder" />
<meta name="DC.Contributor" content="Anthony C. Y. Yau" />
<meta name="DC.Contributor" content="Christophe Hue" />
<meta name="DC.Contributor" content="Sophie Calderari" />
<meta name="DC.Contributor" content="Marie-Thérèse Bihoreau" />
<meta name="DC.Contributor" content="Jean-Baptiste Cazier" />
<meta name="DC.Contributor" content="Richard Mott" />
<meta name="DC.Contributor" content="Dominique Gauguier" />
<meta name="article:published_time" content="2016-11-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Transcriptome Profiling in Rat Inbred Strains and Experimental Cross Reveals Discrepant Genetic Architecture of Genome-Wide Gene Expression" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;To test the impact of genetic heterogeneity on &lt;i&gt;cis&lt;/i&gt;- and &lt;i&gt;trans&lt;/i&gt;-mediated mechanisms of gene expression regulation, we profiled the transcriptome of adipose tissue in 20 inbred congenic strains derived from diabetic Goto–Kakizaki (GK) rats and Brown–Norway (BN) controls, which contain well-defined blocks (1–183 Mb) of genetic polymorphisms, and in 123 genetically heterogeneous rats of an (GK × BN)F2 offspring. Within each congenic we identified 73–1351 differentially expressed genes (DEGs), only 7.7% of which mapped within the congenic blocks, and which may be regulated in &lt;i&gt;cis&lt;/i&gt;. The remainder localized outside the blocks, and therefore must be regulated in &lt;i&gt;trans&lt;/i&gt;. Most &lt;i&gt;trans&lt;/i&gt;-regulated genes exhibited approximately twofold expression changes, consistent with monoallelic expression. Altered biological pathways were replicated between congenic strains sharing blocks of genetic polymorphisms, but polymorphisms at different loci also had redundant effects on transcription of common distant genes and pathways. We mapped 2735 expression quantitative trait loci (eQTL) in the F2 cross, including 26% predominantly &lt;i&gt;cis&lt;/i&gt;-regulated genes, which validated DEGs in congenic strains. A hotspot of &amp;gt;300 eQTL in a 10 cM region of chromosome 1 was enriched in DEGs in a congenic strain. However, many DEGs among GK, BN and congenic strains did not replicate as eQTL in F2 hybrids, demonstrating distinct mechanisms of gene expression when alleles segregate in an outbred population or are fixed homozygous across the entire genome or in short genomic regions. Our analysis provides conceptual advances in our understanding of the complex architecture of genome expression and pathway regulation, and suggests a prominent impact of epistasis and monoallelic expression on gene transcription.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/11/01" />
<meta name="citation_mjid" content="ggg;6/11/3671" />
<meta name="citation_id" content="6/11/3671" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/11/3671" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/11/3671.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/11/3671.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/11/3671.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.033274" />
<meta name="citation_pmid" content="27646706" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3671" />
<meta name="citation_lastpage" content="3683" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Pamela J. Kaisaki" />
<meta name="citation_author_institution" content="The Wellcome Trust Centre for Human Genetics, University of Oxford, OX3 7BN, United Kingdom" />
<meta name="citation_author" content="Georg W. Otto" />
<meta name="citation_author_institution" content="The Wellcome Trust Centre for Human Genetics, University of Oxford, OX3 7BN, United Kingdom" />
<meta name="citation_author" content="Karène Argoud" />
<meta name="citation_author_institution" content="The Wellcome Trust Centre for Human Genetics, University of Oxford, OX3 7BN, United Kingdom" />
<meta name="citation_author" content="Stephan C. Collins" />
<meta name="citation_author_institution" content="The Wellcome Trust Centre for Human Genetics, University of Oxford, OX3 7BN, United Kingdom" />
<meta name="citation_author" content="Robert H. Wallis" />
<meta name="citation_author_institution" content="The Wellcome Trust Centre for Human Genetics, University of Oxford, OX3 7BN, United Kingdom" />
<meta name="citation_author" content="Steven P. Wilder" />
<meta name="citation_author_institution" content="The Wellcome Trust Centre for Human Genetics, University of Oxford, OX3 7BN, United Kingdom" />
<meta name="citation_author" content="Anthony C. Y. Yau" />
<meta name="citation_author_institution" content="The Wellcome Trust Centre for Human Genetics, University of Oxford, OX3 7BN, United Kingdom" />
<meta name="citation_author" content="Christophe Hue" />
<meta name="citation_author_institution" content="Cordeliers Research Centre, Institut National de la Santé et de la Recherche Médicale Unités Mixtes de Recherche Scientifique 1138, Sorbonne Universities, University Pierre &amp; Marie Curie, University Paris Descartes, Sorbonne Paris Cité, 75006, France" />
<meta name="citation_author" content="Sophie Calderari" />
<meta name="citation_author_institution" content="Cordeliers Research Centre, Institut National de la Santé et de la Recherche Médicale Unités Mixtes de Recherche Scientifique 1138, Sorbonne Universities, University Pierre &amp; Marie Curie, University Paris Descartes, Sorbonne Paris Cité, 75006, France" />
<meta name="citation_author" content="Marie-Thérèse Bihoreau" />
<meta name="citation_author_institution" content="The Wellcome Trust Centre for Human Genetics, University of Oxford, OX3 7BN, United Kingdom" />
<meta name="citation_author" content="Jean-Baptiste Cazier" />
<meta name="citation_author_institution" content="Centre for Computational Biology, University of Birmingham, Edgbaston B15 2TT, United Kingdom" />
<meta name="citation_author" content="Richard Mott" />
<meta name="citation_author_institution" content="University College London Genetics Institute, WC1E 6BT, United Kingdom" />
<meta name="citation_author" content="Dominique Gauguier" />
<meta name="citation_author_institution" content="The Wellcome Trust Centre for Human Genetics, University of Oxford, OX3 7BN, United Kingdom" />
<meta name="citation_author_institution" content="Cordeliers Research Centre, Institut National de la Santé et de la Recherche Médicale Unités Mixtes de Recherche Scientifique 1138, Sorbonne Universities, University Pierre &amp; Marie Curie, University Paris Descartes, Sorbonne Paris Cité, 75006, France" />
<meta name="citation_author_email" content="dominique.gauguier@crc.jussieu.fr" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6156-9530" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. Alberts;citation_author=P. Terpstra;citation_author=Y. Li;citation_author=R. Breitling;citation_author=J. P. Nap;citation_title=Sequence polymorphisms cause many false cis eQTLs.;citation_pages=e622-e622;citation_volume=2;citation_year=2007;citation_issue=7;citation_pmid=17637838;citation_doi=10.1371/journal.pone.0000622" />
<meta name="citation_reference" content="citation_journal_title=Diabetologia;citation_journal_abbrev=Diabetologia;citation_author=K. Argoud;citation_author=S. P. Wilder;citation_author=M. A. McAteer;citation_author=M. T. Bihoreau;citation_author=F. Ouali;citation_title=Genetic control of plasma lipid levels in a cross derived from normoglycaemic Brown Norway and spontaneously diabetic Goto-Kakizaki rats.;citation_pages=2679-2688;citation_volume=49;citation_year=2006;citation_issue=11;citation_pmid=16983556;citation_doi=10.1007/s00125-006-0396-z" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=S. S. Atanur;citation_author=A. G. Diaz;citation_author=K. Maratou;citation_author=A. Sarkis;citation_author=M. Rotival;citation_title=Genome sequencing reveals loci under artificial selection that underlie disease phenotypes in the laboratory rat.;citation_pages=691-703;citation_volume=154;citation_year=2013;citation_pmid=23890820;citation_doi=10.1016/j.cell.2013.06.040" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=R. Breitling;citation_author=Y. Li;citation_author=B. M. Tesson;citation_author=J. Fu;citation_author=C. Wu;citation_title=Genetical genomics: spotlight on QTL hotspots.;citation_pages=e1000232-e1000232;citation_volume=4;citation_year=2008;citation_issue=10;citation_pmid=18949031;citation_doi=10.1371/journal.pgen.1000232" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=K. W. Broman;citation_author=H. Wu;citation_author=S. Sen;citation_author=G. A. Churchill;citation_title=R/qtl: QTL mapping in experimental crosses;citation_pages=889-890;citation_volume=19;citation_year=2003;citation_issue=7;citation_pmid=12724300;citation_doi=10.1093/bioinformatics/btg112" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=H. G. Brunner;citation_author=M. A. van Driel;citation_title=From syndrome families to functional genomics.;citation_pages=545-551;citation_volume=5;citation_year=2004;citation_issue=7;citation_pmid=15211356;citation_doi=10.1038/nrg1383" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. A. Buchner;citation_author=J. H. Nadeau;citation_title=Contrasting genetic architectures in different mouse reference populations used for studying complex traits.;citation_volume=25;citation_year=2015;citation_pmid=25953951;citation_doi=10.1101/gr.187450.114" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=L. Bystrykh;citation_author=E. Weersing;citation_author=B. Dontje;citation_author=S. Sutton;citation_author=M. T. Pletcher;citation_title=Uncovering regulatory pathways that affect hematopoietic stem cell function using 'genetical genomics'.;citation_pages=225-232;citation_volume=37;citation_year=2005;citation_issue=3;citation_pmid=15711547;citation_doi=10.1038/ng1497" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=Y. Chen;citation_author=J. Zhu;citation_author=P. Y. Lum;citation_author=X. Yang;citation_author=S. Pinto;citation_title=Variations in DNA elucidate molecular networks that cause disease.;citation_pages=429-435;citation_volume=452;citation_year=2008;citation_issue=7186;citation_pmid=18344982;citation_doi=10.1038/nature06757" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=E. J. Chesler;citation_author=L. Lu;citation_author=S. Shou;citation_author=Y. Qu;citation_author=J. Gu;citation_title=Complex trait analysis of gene expression uncovers polygenic and pleiotropic networks that modulate nervous system function.;citation_pages=233-242;citation_volume=37;citation_year=2005;citation_issue=3;citation_pmid=15711545;citation_doi=10.1038/ng1518" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=A. Chess;citation_title=Mechanisms and consequences of widespread random monoallelic expression.;citation_pages=421-428;citation_volume=13;citation_year=2012;citation_issue=6;citation_pmid=22585065;citation_doi=10.1038/nrg3239" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=M. Civelek;citation_author=A. J. Lusis;citation_title=Systems genetics approaches to understand complex traits.;citation_pages=34-48;citation_volume=15;citation_year=2014;citation_pmid=24296534;citation_doi=10.1038/nrg3575" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=A. L. Dixon;citation_author=L. Liang;citation_author=M. F. Moffatt;citation_author=W. Chen;citation_author=S. Heath;citation_title=A genome-wide association study of global gene expression.;citation_pages=1202-1207;citation_volume=39;citation_year=2007;citation_issue=10;citation_pmid=17873877;citation_doi=10.1038/ng2109" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. E. Dumas;citation_author=S. P. Wilder;citation_author=M. T. Bihoreau;citation_author=R. H. Barton;citation_author=J. F. Fearnside;citation_title=Direct quantitative trait locus mapping of mammalian metabolic phenotypes in diabetic and normoglycemic rat models.;citation_pages=666-672;citation_volume=39;citation_year=2007;citation_issue=5;citation_pmid=17435758;citation_doi=10.1038/ng2026" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=V. Emilsson;citation_author=G. Thorleifsson;citation_author=B. Zhang;citation_author=A. S. Leonardson;citation_author=F. Zink;citation_title=Genetics of gene expression and its effect on disease.;citation_pages=423-428;citation_volume=452;citation_year=2008;citation_issue=7186;citation_pmid=18344981;citation_doi=10.1038/nature06758" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=B. P. Fairfax;citation_author=S. Makino;citation_author=J. Radhakrishnan;citation_author=K. Plant;citation_author=S. Leslie;citation_title=Genetics of gene expression in primary immune cells identifies cell type-specific master regulators and roles of HLA alleles.;citation_pages=502-510;citation_volume=44;citation_year=2012;citation_issue=5;citation_pmid=22446964;citation_doi=10.1038/ng.2205" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=S. Falcon;citation_author=R. Gentleman;citation_title=Using GOstats to test gene lists for GO term association.;citation_volume=23;citation_year=2007;citation_pmid=17098774;citation_doi=10.1093/bioinformatics/btl567" />
<meta name="citation_reference" content="citation_title=Encyclopedia of Genetics, Genomics, Proteomics and Bioinformatics,;citation_title=The rat as a model physiological system,;citation_author=D. Gauguier;citation_pages=1154-1171;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=D. Gauguier;citation_author=P. Froguel;citation_author=V. Parent;citation_author=C. Bernard;citation_author=M. T. Bihoreau;citation_title=Chromosomal mapping of genetic loci associated with non-insulin dependent diabetes in the GK rat.;citation_pages=38-43;citation_volume=12;citation_year=1996;citation_issue=1;citation_pmid=8528248;citation_doi=10.1038/ng0196-38" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=A. Ghazalpour;citation_author=S. Doss;citation_author=B. Zhang;citation_author=S. Wang;citation_author=C. Plaisier;citation_title=Integrating genetic and network analysis to characterize genes related to mouse weight.;citation_pages=e130-e130;citation_volume=2;citation_year=2006;citation_issue=8;citation_pmid=16934000;citation_doi=10.1371/journal.pgen.0020130" />
<meta name="citation_reference" content="citation_journal_title=The Tohoku journal of experimental medicine;citation_journal_abbrev=Tohoku J Exp Med;citation_author=Y. Goto;citation_author=M. Kakizaki;citation_author=N. Masaki;citation_title=Production of spontaneous diabetic rats by repetition of selective breeding.;citation_pages=85-90;citation_volume=119;citation_year=1976;citation_issue=1;citation_pmid=951706;citation_doi=10.1620/tjem.119.85" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=E. Grundberg;citation_author=K. S. Small;citation_author=Å. Hedman;citation_author=A. C. Nica;citation_author=A. Buil;citation_title=Mapping cis- and trans-regulatory effects across multiple tissues in twins.;citation_pages=1084-1089;citation_volume=44;citation_year=2012;citation_issue=10;citation_pmid=22941192;citation_doi=10.1038/ng.2394" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_title=The Genotype-Tissue Expression (GTEx) pilot analysis: Multitissue gene regulation in humans;citation_pages=648-660;citation_volume=348;citation_year=2015;citation_issue=6235;citation_pmid=25954001;citation_doi=10.1126/science.1262110" />
<meta name="citation_reference" content="citation_journal_title=Heredity (Edinb);citation_author=C. S. Haley;citation_author=S. A. Knott;citation_title=A simple regression method for mapping quantitative trait loci in line crosses using flanking markers.;citation_pages=315-324;citation_volume=69;citation_year=1992;citation_pmid=16718932;citation_doi=10.1038/hdy.1992.131" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. Heinig;citation_author=E. Petretto;citation_author=C. Wallace;citation_author=L. Bottolo;citation_author=M. Rotival;citation_title=A trans-acting locus regulates an anti-viral expression network and type 1 diabetes risk.;citation_pages=460-464;citation_volume=467;citation_year=2010;citation_issue=7314;citation_pmid=20827270;citation_doi=10.1038/nature09386" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=G. J. Huang;citation_author=S. Shifman;citation_author=W. Valdar;citation_author=M. Johannesson;citation_author=B. Yalcin;citation_title=High resolution mapping of expression QTLs in heterogeneous stock mice in multiple tissues.;citation_volume=19;citation_year=2009;citation_pmid=19376938;citation_doi=10.1101/gr.088120.108" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=N. Hubner;citation_author=C. A. Wallace;citation_author=H. Zimdahl;citation_author=E. Petretto;citation_author=H. Schulz;citation_title=Integrated transcriptional profiling and linkage analysis for identification of genes underlying disease.;citation_pages=243-253;citation_volume=37;citation_year=2005;citation_issue=3;citation_pmid=15711544;citation_doi=10.1038/ng1522" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=M. Kanehisa;citation_author=S. Goto;citation_title=KEGG: Kyoto Encyclopedia of Genes and Genomes;citation_pages=27-30;citation_volume=28;citation_year=2000;citation_issue=1;citation_pmid=10592173;citation_doi=10.1093/nar/28.1.27" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=L. Matthews;citation_author=G. Gopinath;citation_author=M. Gillespie;citation_author=M. Caudy;citation_author=D. Croft;citation_title=Reactome knowledgebase of human biological pathways and processes.;citation_volume=37;citation_year=2009;citation_pmid=18981052;citation_doi=10.1093/nar/gkn863" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Monti;citation_author=J. Fischer;citation_author=S. Paskas;citation_author=M. Heinig;citation_author=H. Schulz;citation_title=Soluble epoxide hydrolase is a susceptibility factor for heart failure in a rat model of human disease.;citation_pages=529-537;citation_volume=40;citation_year=2008;citation_issue=5;citation_pmid=18443590;citation_doi=10.1038/ng.129" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=R. Mott;citation_author=W. Yuan;citation_author=P. Kaisaki;citation_author=X. Gan;citation_author=J. Cleak;citation_title=The architecture of parent-of-origin effects in mice.;citation_pages=332-342;citation_volume=156;citation_year=2014;citation_pmid=24439386;citation_doi=10.1016/j.cell.2013.11.043" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=E. Petretto;citation_author=J. Mangion;citation_author=N. J. Dickens;citation_author=S. A. Cook;citation_author=M. K. Kumaran;citation_title=Heritability and tissue specificity of expression quantitative trait loci.;citation_pages=e172-e172;citation_volume=2;citation_year=2006;citation_issue=10;citation_pmid=17054398;citation_doi=10.1371/journal.pgen.0020172" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=E. Petretto;citation_author=R. Sarwar;citation_author=I. Grieve;citation_author=H. Lu;citation_author=M. K. Kumaran;citation_title=Integrated genomic approaches implicate osteoglycin (Ogn) in the regulation of left ventricular mass.;citation_pages=546-552;citation_volume=40;citation_year=2008;citation_issue=5;citation_pmid=18443592;citation_doi=10.1038/ng.134" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=E. E. Schadt;citation_author=S. A. Monks;citation_author=T. A. Drake;citation_author=A. J. Lusis;citation_author=N. Che;citation_title=Genetics of gene expression surveyed in maize, mouse and man.;citation_pages=297-302;citation_volume=422;citation_year=2003;citation_issue=6929;citation_pmid=12646919;citation_doi=10.1038/nature01434" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=W. Shi;citation_author=A. Oshlack;citation_author=G. K. Smyth;citation_title=Optimizing the noise vs. bias trade-off for Illumina whole genome expression BeadChips.;citation_volume=38;citation_year=2010;citation_pmid=20929874;citation_doi=10.1093/nar/gkq871" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=L. C. Solberg;citation_author=A. E. Baum;citation_author=N. Ahmadiyeh;citation_author=K. Shimomura;citation_author=R. Li;citation_title=Sex- and lineage-specific inheritance of depression-like behavior in the rat.;citation_pages=648-662;citation_volume=15;citation_year=2004;citation_issue=8;citation_pmid=15457344;citation_doi=10.1007/s00335-004-2326-z" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=R. Sopko;citation_author=D. Huang;citation_author=N. Preston;citation_author=G. Chua;citation_author=B. Papp;citation_title=Mapping pathways and phenotypes by systematic gene overexpression.;citation_pages=319-330;citation_volume=21;citation_year=2006;citation_issue=3;citation_pmid=16455487;citation_doi=10.1016/j.molcel.2005.12.011" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. Subramanian;citation_author=P. Tamayo;citation_author=V. K. Mootha;citation_author=S. Mukherjee;citation_author=B. L. Ebert;citation_title=Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles.;citation_volume=102;citation_year=2005;citation_pmid=16199517;citation_doi=10.1073/pnas.0506580102" />
<meta name="citation_reference" content="citation_journal_title=Physiol. Genomics;citation_author=K. J. Wallace;citation_author=R. H. Wallis;citation_author=S. C. Collins;citation_author=K. Argoud;citation_author=P. J. Kaisaki;citation_title=Quantitative trait locus dissection in congenic strains of the Goto-Kakizaki rat identifies a region conserved with diabetes loci in human chromosome 1q.;citation_volume=19;citation_year=2004;citation_pmid=15266047;citation_doi=10.1152/physiolgenomics.00114.2004" />
<meta name="citation_reference" content="citation_journal_title=Diabetologia;citation_journal_abbrev=Diabetologia;citation_author=R. H. Wallis;citation_author=K. J. Wallace;citation_author=S. C. Collins;citation_author=M. McAteer;citation_author=K. Argoud;citation_title=Enhanced insulin secretion and cholesterol metabolism in congenic strains of the spontaneously diabetic (Type 2) Goto Kakizaki rat are controlled by independent genetic loci in rat chromosome 8.;citation_pages=1096-1106;citation_volume=47;citation_year=2004;citation_issue=6;citation_pmid=15164172" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. H. Wallis;citation_author=S. C. Collins;citation_author=P. J. Kaisaki;citation_author=K. Argoud;citation_author=S. P. Wilder;citation_title=Pathophysiological, genetic and gene expression features of a novel rodent model of the cardio-metabolic syndrome.;citation_pages=e2962-e2962;citation_volume=3;citation_year=2008;citation_issue=8;citation_pmid=18698428;citation_doi=10.1371/journal.pone.0002962" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=S. P. Wilder;citation_author=M. T. Bihoreau;citation_author=K. Argoud;citation_author=T. K. Watanabe;citation_author=M. Lathrop;citation_title=Integration of the Rat Recombination and EST Maps in the Rat Genomic Sequence and Comparative Mapping Analysis With the Mouse Genome;citation_pages=758-765;citation_volume=14;citation_year=2004;citation_issue=4;citation_pmid=15060020;citation_doi=10.1101/gr.2001604" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=E. G. Williams;citation_author=J. Auwerx;citation_title=The convergence of systems and reductionist approaches in complex trait analysis.;citation_pages=23-32;citation_volume=162;citation_year=2015;citation_pmid=26140590;citation_doi=10.1016/j.cell.2015.06.024" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Zhu;citation_author=B. Zhang;citation_author=E. N. Smith;citation_author=B. Drees;citation_author=R. B. Brem;citation_title=Integrating large-scale functional genomic data to dissect the complexity of yeast regulatory networks.;citation_pages=854-861;citation_volume=40;citation_year=2008;citation_issue=7;citation_pmid=18552845;citation_doi=10.1038/ng.167" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Transcriptome Profiling in Rat Inbred Strains and Experimental Cross Reveals Discrepant Genetic Architecture of Genome-Wide Gene Expression" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11.cover-source.jpg" />
<meta name="twitter:description" content="To test the impact of genetic heterogeneity on cis - and trans -mediated mechanisms of gene expression regulation, we profiled the transcriptome of adipose tissue in 20 inbred congenic strains derived from diabetic Goto–Kakizaki (GK) rats and Brown–Norway (BN) controls, which contain well-defined blocks (1–183 Mb) of genetic polymorphisms, and in 123 genetically heterogeneous rats of an (GK × BN)F2 offspring. Within each congenic we identified 73–1351 differentially expressed genes (DEGs), only 7.7% of which mapped within the congenic blocks, and which may be regulated in cis . The remainder localized outside the blocks, and therefore must be regulated in trans . Most trans -regulated genes exhibited approximately twofold expression changes, consistent with monoallelic expression. Altered biological pathways were replicated between congenic strains sharing blocks of genetic polymorphisms, but polymorphisms at different loci also had redundant effects on transcription of common distant genes and pathways. We mapped 2735 expression quantitative trait loci (eQTL) in the F2 cross, including 26% predominantly cis -regulated genes, which validated DEGs in congenic strains. A hotspot of &gt;300 eQTL in a 10 cM region of chromosome 1 was enriched in DEGs in a congenic strain. However, many DEGs among GK, BN and congenic strains did not replicate as eQTL in F2 hybrids, demonstrating distinct mechanisms of gene expression when alleles segregate in an outbred population or are fixed homozygous across the entire genome or in short genomic regions. Our analysis provides conceptual advances in our understanding of the complex architecture of genome expression and pathway regulation, and suggests a prominent impact of epistasis and monoallelic expression on gene transcription." />
<meta name="og-title" property="og:title" content="Transcriptome Profiling in Rat Inbred Strains and Experimental Cross Reveals Discrepant Genetic Architecture of Genome-Wide Gene Expression" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/11/3671" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="To test the impact of genetic heterogeneity on cis - and trans -mediated mechanisms of gene expression regulation, we profiled the transcriptome of adipose tissue in 20 inbred congenic strains derived from diabetic Goto–Kakizaki (GK) rats and Brown–Norway (BN) controls, which contain well-defined blocks (1–183 Mb) of genetic polymorphisms, and in 123 genetically heterogeneous rats of an (GK × BN)F2 offspring. Within each congenic we identified 73–1351 differentially expressed genes (DEGs), only 7.7% of which mapped within the congenic blocks, and which may be regulated in cis . The remainder localized outside the blocks, and therefore must be regulated in trans . Most trans -regulated genes exhibited approximately twofold expression changes, consistent with monoallelic expression. Altered biological pathways were replicated between congenic strains sharing blocks of genetic polymorphisms, but polymorphisms at different loci also had redundant effects on transcription of common distant genes and pathways. We mapped 2735 expression quantitative trait loci (eQTL) in the F2 cross, including 26% predominantly cis -regulated genes, which validated DEGs in congenic strains. A hotspot of &gt;300 eQTL in a 10 cM region of chromosome 1 was enriched in DEGs in a congenic strain. However, many DEGs among GK, BN and congenic strains did not replicate as eQTL in F2 hybrids, demonstrating distinct mechanisms of gene expression when alleles segregate in an outbred population or are fixed homozygous across the entire genome or in short genomic regions. Our analysis provides conceptual advances in our understanding of the complex architecture of genome expression and pathway regulation, and suggests a prominent impact of epistasis and monoallelic expression on gene transcription." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11.cover-source.jpg" />
    <title>Transcriptome Profiling in Rat Inbred Strains and Experimental Cross Reveals Discrepant Genetic Architecture of Genome-Wide Gene Expression | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"KcrTRzL4Wy-xM5jEpiBpsloofndGtvDVMmBP4n4wSJU","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460451","apath":"\/ggg\/6\/11\/3671.atom","pisa":"ggg;6\/11\/3671","ac":{"\/ggg\/6\/11\/3671.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/11\/3671.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/11\/3671"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/11\/3671"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/11\/3671":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u002248huxk7zkSQOvVcCip3juaujR\\\\\\\/\\\\\\\/Toctdtpmr+oEVadE4QVtUZJTwIWmh1Rw11W7QOF1pNqEuqydb4+zWbhMzdcSdBhnUrxcvR4o+FHWMvRJotME8XTjgypyXZXVDTR3XnrJUs93RoQLa3m2vYCpiI8Q3M01Y9KI880OH\\\\\\\/IEmtj4IkXi4xGfR1fGAZNBhlkRhjTjkB2YbJqd1WGwmMXCSOWSXuyrMxo1bGrLqm5CrmB1QWlGbyUuwYfRGkQSfG9vGu3cXZGlxMIkc0lNNhobs0LsbMtbR19JBwCdbouAZ5OO2e2GYUGRx5q88ueU+ZLI\\\\\\\/NP8DQQgNpUqS0CD7SbMfH\\\\\\\/cajMP9pkR3\\\\\\\/AXSRW35D3eoGmufFXknre7rWbUBwS\\\\\\\/\\\\\\\/XROsPm\\\\\\\/N9B\\\\\\\/B5XvW9yl7FtvnvUiga0WfEzbWe+GE9ZOVTzouf1D6hJSA+7yE+o+N6LjCmlKKhFhO+At8YcLG0OZEuivzafcpO2Ct8JjEZEb1NnRp2O8h0Ss00wO27ecqtk7N6t48XD4KYV1FHpZszL5HQV5wQHKkjbcXwN9\\\\\\\/cP8txhvGWzxuUpiSo2S3LHmCvYBr8PY10pkpcxA5HAnympd1gjW0kWYOhsB4tqDwIlQ3+O4qDzpef2uHqVA0FbhiNtAHVI62WIE12\\\\\\\/tErTeqvdyPork6zCzFucsex8EPqFIvTT9+usYV4BiuNGx2Q1Fd33yuOrheukuINj8BjKBOlEP1shD0610dR\\\\\\\/+VKu7LdOY0JjnqqE\\\\\\\/DN\\\\\\\/ivwOZs4CnB1SCi7B33xaYXTTJfy1QdNsIs8nJohhl8qirtRf5RP2v1\\\\\\\/4TKDK8hFjqYzMmVpq4INO1cfbowGtYAWPdrR\\\\\\\/ubEub2yqf4j9qs6Nl2bcSrgROxTt6\\\\\\\/bNyoKi2CXZlMRGC7gS15d2CEupgS8wwOkZ5OWraChA8vvgvuojVPvycNGmu9QMDIRvObF0KqCWOgp9a1adkzpVBAYidluubhgTb+FcrvD8+wovs9zbXXr5JRJXJ\\\\\\\/vqYajrKZOloG4sGSO9rYuos+KyZ7uUh7Lw\\\\\\\/oJcGweqRJMZ9GSKarMqApAT9mN+FVtNYQ3V1agbpdrZqZvAYBCQSowCDKkFDHezGGaksAbcGixl0Osa\\\\\\\/9xPkAbs4bU\\\\\\\/yO1tgz0tBtxA9dvXGajKMdA8azLnfZW\\\\\\\/8hzVYrplSYzbxNt5+33VS7VWs3E1iLSUQngiDd\\\\\\\/E4WQes5zpc8D9Bnn9nTda1ilmJ99R6DAgevvjlzh0IVe3w0typhbkV+kI\\\\\\\/OG448nSbzCrBQolRq+cQc\\\u0022,\\\u0022iv\\\u0022:\\\u0022kBkrF7J9rymGjHrzv8zz0g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ecd9b8e901af44a9ce232b8446270558d5dbd1665164b7b3f3c4f554cf7d5737\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022JFv9xVtfQqnX2OmF+JN33fjiMwgNK2expAcMAoS3UrUURSD+qUwLNkxeO2sqNJ3HvEtigNq6CK1ezTtDr3JFytV5V6gdaVeNeMktCVYLuhGSjgPXNq2wG5Y2wguEyQHZgVoDxVLcHiETVF\\\\\\\/E4BPrIn0s\\\\\\\/sb42VlQ7ORXMlRdHBiKD38LI1Sg8uDLfgkr0RF0TeQ8r\\\\\\\/m6KXgREUcxZtTAhi973yFBxZRcPAkHH03D7HZdAp\\\\\\\/JfIltoGJRr+oim5lMQZ7b7DmDjYyaOwGz0B5HSjS4qJiBm9QNB01F0bTmOU5A900jqlzb4hoyIBbd5d1dze83eXEZR\\\\\\\/bz7Og7J8RI1lT6Hb2o1aIzxqWA2ARLVLvDpFb0g7ES1EvOqNWhpMjL6ZaJ344rblO3PMeaIy2cRmJrBt1ijyzE24MUMGUHPXnL05Xw3hwdf5\\\\\\\/GdOnzn\\\\\\\/I8Bu34SwMON81pVLr9avaXMXuObWlnsyLL3GBYgGgPsED9zk8WcoVUa9oJwOc5brYk160lzD9A6nY8emYSYELRc3LXccnVP\\\\\\\/K8gjkD5\\\\\\\/MAsZhrmIiDjUCojCjKolhgP9vavT1OEXNAE0VVKvF2ropfezHzSc6XSWgjz0hbU24ufIIb6jHAKxKGwExVbGhQYVXH4J6REXAr9JD2oR3KtTTSI5VN+\\\\\\\/21fLP8WDgd7Qvjv9Edy0F7v6Yxtt4eZD+zXqqo3tCFr0PunzOKTEWcQgdmZ5louNiFRrZzEElEdgAMYYUnlSIeiuYPcFoQdj6rFgkzNTg30ACI\\\\\\\/pDZ1fucG7yIW6BfuOaGQOISf0pyOh4QcCTFIg9WwJPZOejMHA10PTPAi\\\\\\\/c5+54rFNfo\\\\\\\/FZU+Gmk73KALoR8wzZRj4kcjRHtySemS50k3A\\\\\\\/5mQcH5UqjyW1MR+2bZQTcKaDreyH8MlbbOaoSXw5Ogl0y5dCHUihMWAdjguSC9pD0He5nqPXYznGc6KcQpTz5ABazo4E9K1QtcJnOJNvdL6oTnHUdjscmhpfaIHaMjGQ0VE6paljtN1vkVIZGHsx7QodwDGE3Bn9\\\\\\\/id8Kipy7RkbcHPsANi4dq+xpiMkW2xSasOwy2t6Oo+oIseIhujgyEI1JGHx706wBhStVndK3xOqmKVCzxt88qS3LxDzZXqK9Mb9AEQALH19z4EuxHjB0WSAlKTxlxYJSDa9im1xQJ9B\\\\\\\/62hPZN12rNZhzzHotlwGnY5fWIstTE8UPMbsFBpVAczWrlUsjLschmTNPhyJScoYRemYDXuPhHuwIR1ti8IasY62zVIGJyYymQsdj1lzWxEILRN3tG\\\\\\\/EopqnjqzKqPDi0Q5qJqhnkw0XSsBgkBKFSNUYYDxTwMsNRD8\\\\\\\/btqm\\\\\\\/\\\\\\\/I+TQ7HOfi\\\\\\\/v9IitJjhRfJHdG8fBbXEEFNEI1\\\\\\\/SKVcb4QbogrmE12p9vFgh\\\\\\\/b+MvQmGetKz\\\\\\\/71Bm5qVLSXOteKV+LVajPilOLg1QOaKHatIJwxX18VCZtwdkGCLF6UePtEYjHQQ9Ig4PIMlcUDby2Exz\\\\\\\/7Rbu0vA6bTwqKyFBhzva4ug6lhlXEEJ0vRiw3nLqoXSVecPrK39\\\\\\\/jmZsjdIebyjeqshqKgHsEZun09OLlECjlKZysK+owi0qhyFHiIcAj9PE+fLDvPcWWfyLeANQRFjVkpM5htj4rUj5ZC2PTNUljpH7c1SE8LshCslkRKx68j62V502g\\\\\\\/bXRnW+9pemqMzSed95END3e6Tr84Yzr8s18bQKj6FKx1yszH0JuL4+CGHmszv\\\\\\\/InWBfneTUet7r46jQq+ETyyYqU1IQMnMTSwQBTGjU6Gdm\\\\\\\/dc3Ql8G80b+4zkBjTgXUaAUPyTWWjeuwyd+jgsHzWzzoGL5abVRlFP08C3G93w614M58bIsz6nL2zlSgClDQ4ENqxXgNk6D+wzsBe9d3qbP0TB1FMGNfpFX1CIBZJqBgCBDGDIPj7qTozc5bM9ioqN5UZikS3823HRFXJk73xgR4QhB+TcXgCedSEu4HIxuhA2YqAdRgNe8E8y1MT+1Eg4VbALpotqWNHFZrvygTOxIhq4TT6BB9Puw37d8\\\\\\\/hN4mhOcBXov12QijZzhhAfxReIyVibwdv9tAhcKBqjFbLs4ESljoDIVh\\\\\\\/90g0gov3sGXFH9loYnY8Fyx4tht6iN6PYKIIeOHMDeHUQchfZNJRslQCEAWvSzjujs8KMUrDI3S4cw7yG\\\\\\\/V4QxDbORc1+OVFREnDCtqioh4uaq8l77QQe\\\\\\\/FWoZe4RJxZz8gGWDs7a6mQKb1lG6GXWXsncJQqJxHxdtyhymJdQHS2z9wY4xZGFUJyGOaL47IbIvnRKG\\\\\\\/LFmladdt92fyB8vUnbHlKRWY5whcTTnZdrJ+N0LSfj8R6lOMcUm28wB0htIf3pqWo6boUJVoNtJ1gYOKqx6tEVX8h+s68yTdwe0PwxSVAISto6F3xN\\\\\\\/9rCTVoUB6qCmVzqNJs6+ZNK\\\\\\\/zMGXn+sxWXmfmTi8dAZXUXpkzaGFj4yr83xPxJSxiejBWHGbmFcFd8eD0fdON9lWzUydHSEhT77FxFRe4QmNpV3rID6Nm1\\\\\\\/cBpVMDvNIlAoUJ5VqI0IrIWUP6JWmG792H+3oHP1gcRxPursHOvphfyU2wYnvSkpELvj2oLCEvfUEdeUFPal\\\\\\\/ZrwRM8WJMxf3Kw+MasAlhXJIhZMnm4xRLRg+16rpZelruyY7cdzD7lA8c9dRuFQAikr+4lupL\\\\\\\/DUPK45ocZNWdGpHDwT1cboascqw1lqZbDu7K\\\\\\\/zQC0qk4LqFdqLOXClcHNMlZLFT9VD0nkyiMW1b7qnp2e7exyhTV0GYugJiAmA\\\\\\\/2BOMyqQyUekZEMCp5JrRXMv0jLCclKfxUGGuMlYfxpMxschdkLMu0E9qM2R4YXbd6FgE9fK58BA99dK6a07ds1TjQWcvu7neN2hVHAfQhTeBPB1y6mdPBgBJpTQZ2odbJWdc0UV42\\\\\\\/I8\\\\\\\/pVOMmaRKqVmXQIPNbo5d4ApCFNbqToQxpWxsEoyf\\\\\\\/Ie3wGy9FM6+yrQ0Nj6cLNJASUMuwVlArHrv2bg6ipbmWrfizlf+8OiyCIEHNjEFPYZrHJDqIVyJn5xP86TkecufDMAwx\\\\\\\/a+ixY0MBIE4L6qiBLUUQ+ZsSZNmjevSB5SDhjPSvt3I64jy6dEsT1253A5cEr+7PQBycDqs9UV6f0t\\\\\\\/eN9b62VT\\\\\\\/raOokfswR8I9F7AFwrQV7JXGbzqQEtkkJDorYJIfu0O0f7G9\\\\\\\/ks8ZZ0daJaq+T03LKCof6nxvQXrkbx12XznTJg8PRp\\\\\\\/SgGh0Lh1QqPfz3UzXjCoesZTakTtuDoRGmlA0NwwXe18ZlyGWMiGYIa3OUCPuOUg+kVUbwUBL6KDARLz3eD10xoBe8uTQG54+z4QabtHhbsgyuYqjOb71BZcIom8XvfYXGXPbp8A2ypWCmoSCi1WZZuVAC9BQdKlVOn+P7jgo39gf3Qei3fuDvuIGan6BNtk7FlrXveuR+KuWjpWj9UVqAFv39QUO70DcESMIXwzgI96\\\\\\\/GPkL6GAHh8OFUCnj56ftnsq93YkOWLF2Oi1GyGdD8ErBb2eugUyBrkGu338zZ5hIv8Ih0Ml45FuGj2gYp4tgEKDXlJBnHq936Y9g6uqM\\\\\\\/c0Mf1ehMUE+i3YoADi+KL5zdtkELlAtnaVNqI\\\\\\\/y1tttyCCD81b6F926WqGd1z3Y3OCMYifBBz9MD1wFhm1+ufXliTLG6PagTePcflYWI8GXJQaBqNpCk2isb14bD2p3\\\\\\\/nZY3EYDWEmvy05wrFob+ydJhQbqcasu4BODblL5mEB4PrnFp0sl5\\\\\\\/od2aIzPYHo4jDeS9ZkYniM08ko2f5B445\\\\\\\/xh+btm84+DpkTBvSqNpkAnnMjsbFKB3lnIDNJLu1grq+bw6LmupSNRd02Eurw7Kus40BheXKuttn9FKd0OTiQtPnrP1RTM6VtkAN+91HJtHmjwZOU50+i6hVjw0rT2yJpCbhtmc\\\\\\\/gU1iuEVrF5DwfcRzLvUpRwhX4d4ILAdaYyaGUDELa38103PfXaBQhnH9HCc2sBK7eHN3f17YycOteEK+QuQUC1bSPJZav7KwdC4RpydCKTNEPfXFRHTlocRMNiHI\\\\\\\/Zn2jJK3lvGttHBa13cbrmlk1kJT5eDawuW+8OmnqFQxx2HLijcGTkLnJ5mSNepY5rBQeQDFExSjnLI27YAgxldUeTltwbmZG\\\\\\\/LMlurFeh1HM3NxE0Y8wggT\\\\\\\/vhzeKdVOO4mv9HBRtdc9riZLpnM8kEncEC9hc6UYXKYXgBoc9TyoVj\\\\\\\/zAQwAwhmf4e3Oo8GwNOkmWiuPKQVVvLqwpxG05511wHnFVOoGCUlkwIe9TMyE48hpeIYcERaR6tos3DYCeRSwQtJDVYLW0PW7+jxjIkzg6dmHJsc06KeNyqu0qFp1FqE8QDDmjQUzxoX0ftS6585iJl4Jdf3NN0ohXLMSvadPDVGe+2zUUTF\\\\\\\/tjI4tToc4PQPw1hrEruVF1c3labX4N2Toeq51pMV1EWobSIWcF6TMpuqarZ9C+3geBDskRGLg8z9VfA5MQ4HtDebuV2Z8EaxlH6T6+jw49oJb6+0CBNlEnMCJa2Ug6tbO3YwH7kUoLZqlxKRf+mScvBoed8qsLqS6RXaRnLg4+WEmg1C0OrVeb0ZCTlJdZhXgMOB5P4ZO2iIxelaz9RFPm4rIDnDU5X\\\\\\\/peXMU3WekH9t4HnOllkTKq3h\\\\\\\/Ap4wr4U3OSoNtPfj6vucAKMe2ruG3uCzjE1cqHTwWG1qoSXasDA5x\\\\\\\/eZFzLpxwY0XyM02phd9Y+7y7fsVmukj9+QcKGTFOEO9bD5O9hqvJopTaM9pyFgTpKL9WmnRzHjL72MBXX8wh7S6gYw+jW\\\\\\\/mHH4MZgFa0Kr0448rqlf+GDQQepljOVaazF3I\\\\\\\/xJTNKytoGLF9HVMyrX5K0MZTFE\\\\\\\/MttxeDIYVJwFYtLRrNOpHfLMl2HAU0bB0PCvbqcw5QwkwBWmoN2Ux3thCLE5GKH70rZkdDzmfigcfY\\\\\\\/SIGRm4OP87knNzVFtyBeO2gyi1FkBtwhXJMFbho\\\\\\\/xQql85I6S9jm+LgbFb24geoLLeaf8YrUijJOxPh9D8JCoR9jlLYO+ypLnYSW63Q0\\\\\\\/RG3QxzqaJnA018+MAHD3n5b2Xqti7sB8aSs7s5ID+xrZmtKDHas0xahzn0pNAKKc\\\\\\\/d0fO8Mv5ETqHX0zpUv9g1Q6IOueveBu5+t\\\\\\\/kp24g7p0lneSgkVHq5O3E5iqY6ro1BRYF5Ps0FEpMsmGiUnPd33nK3nr7DNL855gBL6dbop0r2UB78EG5dmnHgW+Ywsthek0bgbDqz7z6aF9X5ZZtn5u\\\\\\\/UerJe6c9b4JUcXzf0meq2oqHTFTaYdzZzGo753BiR7j5+TwbstorFzvvgLmX752XF3y4kNHfzxBxYpap23PHcgkFXzwv8QXJYkrojgm01TdM6NszIMb+Az2yyiCugVllhZ0bflM3MVdJ9vsXlk87ggOj4hXUp\\\\\\\/gyIr\\\\\\\/oEdS63DcK9YdaPOyJqnLeaaXUe+jR2\\\\\\\/GhWbxtoukRmZ7Ou4BI\\\\\\\/2NQWFUNZaWk4mahnlVgzB8xdj2yvTtAclNelc8x6D0E\\\\\\\/HgyjCs0Il3UY0voUDrBeEm07LN31zeZp1qIoqTIS6zHsvj7Gda0qPdUTRAzD0E8gKpqwVIpYJEUm1ycPi6v0Whk+GMkSp3zdfNfz41y+gB+fJlI8HPOZzTyaqQ3Dhxi1lu7oqOppXCwudtdkzjJnsDs8D+NjuEl+0jrqUCf5U\\\\\\\/d96Bp0sGQgsKQOFGkXHNV7FvhcEbNnhB9EaPNJPKQKSvgbf7m5b+CHFuDLstZjx0UqouDuLckOMxRiQxy11eeKX+aIRbHBkG\\\\\\\/fynNFAKkHc2bFUNnDSua\\\\\\\/D1uyg4y1VmVwAmpU4w4ODBEO7mFlQCs2NG+fDSuFN8N9xYZrtPiaR5GWbMnmNRm9WHV62barixmoa5AFiBm6l2PRXlmXjViqRAjG6tYbSMOUGOXkFmYHOf+sDC8RiH\\\\\\\/nDV7Q5FvYC3btXfEe2hC4WyilACebpTrqfftNSrsNNFlqtYxY4MvBsNHrejU191yW8XNSN5+jq9grU399FqBG62Q4+RN6vBKoxN9kVIAnC9Kpk4LwhAFJ7vgmipkX5nVVA7CwMUTkQRRhEUd\\\\\\\/R+jS4z07\\\\\\\/ufN0oMYO6Lhl7rqiAy3WELUrKA6dpIrHhpo9O\\\\\\\/tAWFVDLRTdk67OEWy7xVgThjVZTS0PD\\\\\\\/DmpReOFneIaqudCTHalR5\\\\\\\/9k+lCpoGY8kysSmc9ibkwSzBaCFw6Rrb3QBjui\\\\\\\/X+s9SIeCXyOkKOPEOtmedkmxC\\\\\\\/0N+AEFGDQemj0brEzlXhiBlDrEABHQyTGUs0ICVUY\\\\\\\/+1BLoL5BfJLbtsaVP5PSQXsN+oYlK\\\\\\\/bq+UWvBZRAT5fyrQ0FXnXv6pqLzut05hXE2aJVK3pO4YPoaQnw3fwfHw3wcw4YFcKsGOS3oy8mMgyYsbi+eHwwHIPuOt2wd0LEtFSTvdyVAAPK0n8Ky8fqhTYdUCcX6ptGW93ciikWnJAbLmj6uetiNevVaPXPfVmsJMZwlL4jeyfJT7mFNkK25QrsYkFYnmOt3cnt7suJh5xq1o+Ys8AoRVZoUnnixWYRut4KuD6\\\\\\\/vfDbuLY5vXk1v7kPbQFkZrqZnIdidR+hMGJcpm7QcWe0XhaN6GRjpevypPMqFpRaSpLUG\\\\\\\/T7kJZU9U9kVPiZosSU2\\\\\\\/g8\\\\\\\/uSOMDiIzbzA5YzCDJ6NDcqrOek36CELmyJq5QhSkRZqqepVRGfaMoHTaa6dwEbTd9MFTCbzM1ifEVL0j67lSXshWNLvcT4PrMtUCcwkQmXMrGT\\\\\\\/00WlFeYPJsUp3zx5pDJuk+6IwP2kxEYCEuQ\\\\\\\/+AAhOSUy+M05mhNlvCAFMBm6tHEVovhKVtLwUXmD75R2op5SAz\\\\\\\/jcNuB+GgCcsxbuDw2q0BS3reqq4wy2WvsvePI+TX7ELMHkTXRn2wRkUkn4vFZwkxD0Gix1w6qy6heLeyuC9YWnZWTzNfFZzwb6IROmGKwSoDNeqChW66J\\\\\\\/nW+37NNyYY28ZhgZRCB8jI6mJMVHaGJMiuTMbsLwdMbya6srPFopQz9SkVWQGfAQhUtCqU8sDxfZx8c+9\\\\\\\/56jhDaxchLoqIYDSpprRWkUbG7scI5KvlPlE95cQNNOlI9lipXQU72vJ4TpEpHGaJ8rDmjvMtuUatOYS3Fkj8zisgVy2IuKcIUvvyAUmMotWxxIOvZOJ2NGqSmfnhwpPlXGYUNgVV2cKA\\\\\\\/JyAmNScIjjB8WVQwK5Rp2y1wqPM2YV88JHNfAR6pDYDnM2P3T+uGnN1qyx7I+y6DmqexAlc7SVMgILEXUGaMEltOMEiOSi06kLk2coPTlC6ICgutVoq+ssPU4bMAXInIH5BCjOz13fp0eUI57EjzMsx\\\\\\\/wwQ1BnQZAv707P+m2mpsHuMtgUqzXSCLJjkT5Aqi5q7mdU7ggFeMrAA+UIqrLP+CEzyQC9psp72CdySw+wn5Y+MOqdkwCyS7T0uuAt6NeiTq4f14pvECRwKiajl9M8HKDavwpbkHdwHI+tb3fmtfsByXWKEE55yucGp8qtEIXX+q0T6J\\\\\\\/Ui5q1hlBUnGtJN9Bdg9KwQ8XtMM\\\\\\\/v5ja1i6v6VcTW4DRdt4FZUBtRZda0e95XDKRghFkbW1jO\\\\\\\/2lwW1s2Daxn4L\\\\\\\/YYsCjnAlhlBC1YO2FFYBotK5t9BEcIM03d26chnPXb1OBmQi18KAGHRiQ1wSgy3aVijHzfPDikKHwZxdFqTpPGZvopW9efthzQoskH1vT9LV5yy3I01kkJojjgWrp5jBnknv\\\\\\\/tvlY483YB3w562zqWzEcMZHT49siXHNfbytOhPGw35Zv+DjY\\\\\\\/Kx9Vf05jIMfDqhbtFzLH93tOj7AyOuoxS+m\\\\\\\/zGvNrE31efpsgFi2GARZ+sbSNgF1WCIFnKiB+3oqEfhslVcAEq\\\\\\\/cRKD+tuLMKIROoFqftC0+Q3ItscqapMXlZVSLT0uAbrX9g5NoqnYXx7PVq4b+yKmIDujP32mxnMNiKjv9Lhs7xt1Wxp1pvZiyG92eTbmcprSp13Vzlax3soFP0RpcozDJ6kjwFjCRwfVPboQYDaXnZVSteRxaEXG7lDAo0m17uzPgmZXGuMMNoHacUndUU0zDQlP3bR7L7D7WMGil7R\\\\\\\/f2eg2xiJ9hM7bmM3gcFFmtr+pMvxojMMqsjTBY9Oqxhj8JfHfxkUsota6oDtkIdqwI7cO26pbkGKwrtvrvCwtymOH2o+tXAt3jcN30J55AmsQo\\\\\\\/dGgD77IWXeOM3pZ\\\\\\\/wG4ttZqwcA9J62p76Ntx349iQSSmEvDLWjyt0PGgWzAVaeuBWMsx8+Pzc47t7rTpkQ11wbsAsZdH9W7csaI28Y1QEn0I5adPVo+tqB94A7HI\\\\\\\/yT9ylLcdgTFJN\\\\\\\/ZAwrEXahWkmhloGEZ6sYhPtY7mNuVrwUslwigzGQW8bIYSp5LVABd6jDTc57Jl22LLRklENPIqnC05FKELKMh2aMoVAXOvA2uE9ZKtVwnb9nu9kDAe\\\\\\\/iTnoGuZ5LiBdmyFPIdGPIYxZGQ9jLujpIKxAVy5CUX\\\\\\\/nFKcukWEyBAZYQNMW4HpO5FtH8UK3\\\\\\\/aoksnS2uQDW0qHKa4vSBPfTW3BFi9NcL2boHb6NCwH\\\\\\\/QEY0\\\\\\\/sgXDPzRPl0P6A2Zie4u0nnsOFTzK+onBISYNd4HCEnE\\\\\\\/t3VJQaebfPtjWqxzxCVEeSELqSIBT\\\\\\\/zMQr+zgtbXZr36CRxqDrt8WVX+2VuZjUjsb3NyK+P39+EqDrk0elwzeXpCqYbuG3h4i2r89CXch4FD2GQH+2bNN7hxVq0PQTtO5k8\\\\\\\/U0EosfBokRzK6wQX8ONatpfiD3tz+xsVgJSBUfzfAcZyc2KDng1vPa94sT575APPm2LWavOrbIH8wtvr9tRo\\\\\\\/Znrsri0mOk+Tm+F5g2VQLQtuMwrAO2Ewl6SjwvdwaxCDRryP9bx2f963KVA86NK6p2jB73WkuUtL3RhkvCIWT5AAaxD4MAxkPLLfpRkNftQAP5Pa27ZhA0ZRIwINSTZhbEIXuuGDYwqVnhuv2hNtKSYywOFL0GKh0EN8ufzK0dvFTMnDRV5fbqMjuQaCDeSdzDa5JpZgqiAR2hhG36OofvF7C+Pc+Y\\\\\\\/0oH3LN2aMy844xavFluH2poePgUfU4F4HYXRv5ZoBzdEK5Ti6+FebGhp9\\\\\\\/L6HUMy5519NCBNsn4n5\\\\\\\/vyDTH37CCwflF098E0cQusiSU0V2LHygCLRJtnjQW3TPgUd2yInA410tpdzBo+TFPjndfxwsS9xuhgnrd9SWkoGaijkcPCVoizw5wzGn2sZK5MG4sk5RBzR1+P898VQsvoM\\\\\\\/xX4z0\\\\\\\/7JFG2rYiCEPcqgZAKa5+f7zZkUC5b+AHgNRfxYnod95BKkSK7JeambsBF2sZVWWQ6wS5Z+eew8In1SO9CrzgYRfTSS6IVl4tXcMCCIlh3Pd5ZH2hCGjtecUgWC0jtiFsfWskqgkzLE8DZ4CDWzGLj8KqtHZ7HXPG4uMrUPbhY5f+5Oz2p9HJGJyehpgL6DYYOOrUBFO8gBDOsBdkkjoGOPwOAByeT\\\\\\\/8iOSa+QrHc2KPYb5Kt5eSMw7m0B+z+LB5QAlf9vI\\\\\\\/gJBpyJpzkSzmWqfgy3Mwiaud+UeCWm55Gd64bhKqQjfvZP7ZAJaFFDnvesl\\\\\\\/WlP9SThXxlcNjEOWQ0WtNDeNvjexkoLa1dYX2ysiAP\\\\\\\/JdOMw5D7ULRQLymhfPZrpxHcbfNQQGqlAvqEg8RiUyaY2al+pn3CAGsTXoDovN8gVk3L556ynvgBdyZqpZATXuN7JAsZcq9ldRqzsuhSYiA2CRw+Bfjmvif8da5SqMnSANduPseb4hDNgbMCPYw79mfpjK3KeFZvfhQoDenI7ES1MJfBkBv5lPvcFPR4wtkpUSTOSluj91eFi\\\\\\\/wcrd36FLFGT0EWITt+AGYh2X4JYBep58SkM6qWKSFmWPFkLMUFddGpnm+CPhHetjNb3JFWVcGViMbAiwt\\\\\\\/KtAbyAekt180av+utUCelqxYNsSzveEzMug\\\\\\\/8X6tvnxwmvcgGbSx4fNAs0yBjFwmE9ejDAz0MfIp3XU9vlhjmxYgOGAg59quUZQPt0lXnxh7sR5yLqvMMQWXl9ZaZPGaiQDvhKoceqcTfVv\\\\\\\/ab3epUvRhj7jBcGjF8BZ+kubcXiSoY7QDW58Y+z+4sv80AAQcRICkYhyV7zvrk5qQWzaXqV8oh+lF83kPMjJTia5aX67Lgsai0vRQFW+XLqeS8xV85C7EYWAzXfXRaVMMej\\\\\\\/OT49spCQwTGfWuEal8QItiZBSiZSbk3TL9SrhVCH0IQJe5dA+sNksVIBR7QN\\\\\\\/BYFlxDnKr03GY4T0QAD8nLXITWSndZv0oAQeEdJ0bYV\\\\\\\/PgaLERbfB9Jl5eP3ZdmRuTjFgWor5AJwp4hVVZ7qVq5B94JLQ41aauK6Cjm0SiD01bEjr0nMVYWu3+4GSKCjG9sWI+8LwBJP+SfW6HUSl7qZzWeA\\\\\\\/6BJpYy09bGNP27zAdcsNnushlSg0vbiOZOTzgP47lVik9sE\\\\\\\/NNRVtwKgMNc+iJfitQqcATCaZ0J70IgsstUq\\\\\\\/NIaiHfwx\\\\\\\/\\\\\\\/kX\\\\\\\/xuwRt6Zh6zO4qZb17uk49nycYmexC99UFty5kiHcu6iigobtD8Z1yUqlD26pin2YJv6hv2BKNLDbYOxxy5EgrpAm404KGrt6elwBIRTL\\\\\\\/ujiLrK6YhM6oRbPdELVMiI2jVRBpkBLLq\\\\\\\/A4NGkONjoVeRHWzFIged2fiq5onckA7muuHNG5g67TS29i8I+cLAawM\\\\\\\/MZZTdG3C\\\\\\\/HOMrLeBFJw8GI7deSvrbTyGATgUB\\\\\\\/qEsjSoj1lHDF54vgW\\\\\\\/cYYszcCQnEC6JLBmS3M27gB7wlVvfJWbjN9R0tNtGphisQIlvMnyoOTn\\\\\\\/dasgVeBFjCB4Cd7AYuHiH0hl1bqoOQptyBaYdjQWZ8mGGIZE9XGJiloaF0adp1wZs8C7ObQt9zR2enpuoWTxWgJfhmx79Ss2GJUr+4CweubHm1g89MJnbdY4lYSmUEsZzfxLeCIO1m1nzxE\\\\\\\/rBfQIHUbh5k6DqBPbzqdKTrC6XhgTMSQI8xPx7BvAm2\\\\\\\/s6gRLPQgXs1YRJfGwjWdaqOlUviIMqJJlY8DdOXEacA3rSTGjDDpH7S1y5ehMfHBFs6WP4X5bpbKw3h5PebuVRfhOvAAq8jHgCFJLxgTP\\\\\\\/OdjQZmx4cpsd2fK3rx7ORTW6BCQG82tggwE4WDXtQkTe83wOZCGG9\\\\\\\/MhJqIG6De+DpzNvhlLxzwnKMOIartPIh3OiQX56xBi3LRIlFnw9IlRcLiCiqJmwbDM7vjNMAToRm4t9hb6NCTX6S5HkVvCVfmbNHDFnHUrkliClK\\\\\\\/DYtrwfO8hyQoIMt1g1NaBdN0z9YzHqXLg9u1HYpBs1qdJWWcA9A81KMwBaC8hgMMyayYpWE7CSFfVcqhVjtTMwlcfloTuyckD2M+WBffNZHplkFFbz3Z+SVjrl1wmzxfgjiWD65F4eLSoCMXtiRxpyH98JBPGxiIwK4d2mssTxgrzEgslq7ZMCN2j3CItw0GMUyDGwtnQGI0TnN+LASPh6muBKEVxXkaPdlE3QD8ZUrcM6qIsFr4MvMAxkcBb5PBiHLFeawQXBrux059aOPoPlQJFZzg10+Yrj0zD+J7m4SiZNsWF6sCLWelQnfPFOaFV650xO3b6jz9\\\\\\\/\\\\\\\/Tjurzk4uiUJc30ZkMwjnqcb2ItEiohySijO5JeCMok\\\\\\\/Q+cF9GoXuscDjhFItVsY\\\\\\\/fL8Ovf0pdqWZj7CD2owfQSmGRTjIJ4U1EJZF4IMOmCSQ07I4T8psFWdqda6x4xp0GfaFdiQvUGYxs57dWoYoLI027crzY5JTNpLfK55EBHU3zkq2qZW6UVYM6U9maze93fur6k1WKfA2c1ASBOszmggQ5Arp1A1mjMyuTyX5yHHEO9cJUDbwvO7WJ2On1oszt3ui9OSzk27cCCmVRjq29hRHdGp2Q4ByELcY\\\\\\\/AM+GZ8Fk3W169yE4QAJkFY1rNNh\\\\\\\/NZQBTOs9NQfrwsQMn5hkz4YPQSO4OeFu7gK2t96yeSSaGHoS\\\\\\\/Ds2tlRoM0Km+3OmrM3s\\\\\\\/f\\\\\\\/TzwCKsk9o8q6zvPp+0sKJelYO\\\\\\\/ifE6aO\\\\\\\/IjBMQr5d4CeCTekV4vE7M\\\\\\\/m9hGsmUkdBUaKGKBC\\\\\\\/HypKwboLQPjxupCRbWypTJOD29lzOe\\\\\\\/kCGiebcbtwOCUXiYHPga01us9Lf33Hk7+NnujKOGwSPrcqcBqzvwx1XnQe4\\\\\\\/5NmU9g\\\\\\\/0O8GXGD6kUfrtE8Mr0EEORzh1T11YB1GkW2d50Ux4xqU5ePpnygZdneW9riuc8vM3gIbaBj5yYIlp9w8l3c3EiIHE\\\\\\\/G5io+lc1gD6oRZICam0AF3eMpPEWrE8nDwU5YNskXGl2q9Pv+DB7FKeckG\\\\\\\/aHFHnW2JwRDtfbmflwCqcAL1sJbb0MCnD6R3khzyQQ5zDQsgISJwaFIloonoNlv6H+bUjsfOatD4NSnMh+Cjlh9ugSDOalWm7IblHWPs6xjqkRbPSNrbicR\\\\\\\/Wz+vL41eQtyOlIYYLiyN1rcujzw\\\\\\\/J1kPTiu6M6jWBP9A7QIQY5+0VsPDaFDrTuT65pkHqSJjFR6coYTHLVMPdRW\\\\\\\/mcPyNmJ0EFZ\\\\\\\/l7yGtIb2luA3AnCpzLMIfL1azNAYgVtW4wON2oKkQwCn+4zlCnSuaOIKOhg7QvlvYlliygsKPiYL07R0wk\\\\\\\/\\\\\\\/PU2TQC15WfFm9JoYZbJm8EkLncxh9xDnZRVr9PtINlypHiBnGDUZ4reFidpJA90sz7rLYkGSsSTUuuomtlW65QwYC7fKcr++CjOm+ESyG+9\\\\\\\/dfKogZXHE14pUme2vlv2MLD\\\\\\\/UA1uqg8hEub0Gj1A0DLjqDul9CHdefNOrWnYL6Ef0hSa7izEquhYnHQdQeMmEAh+nnTn09YE1PMuC+zOn2yfSoXl5DD2m+RoHGBA212JHL3qLVVvsAk+XLJEsfmLzqjLLHLt6LN7sDlPtX6SjFJ4t+WwYq8onmToA+w+FJnUtajOPv6rbFwkkjVcoqCKwxbSwGzxZybp3sF35MPJLTT9AzRQs6GkbJDS+SN5oKlE\\\\\\\/kjp5MEhwhufTY94JWIb+roxUohCzmleFoseCPk84\\\\\\\/u7x1UXZ6KfQ1hJjOdG2KoJWQLyvU66g0yb1xqM\\\\\\\/jXhrH08s4iwIxw5YptQ8ipnnMKxDBKdkdLrIt7bNmLby0an0m7NVkxxUGCyhQo0CW8tCKtiugcWqSPDuaJiysB3HjJDz4DVOwITGtJWwCzK4WWyWJW\\\\\\\/0\\\\\\\/adqtnqBx3YWbB9mhdu8q3zWaYNnpd6nAtEUSwFvhhVXKdv4Or+bvZ4ayPYgMthkfu8J4MR2obWN49H0t0BXyybv2YaRH08DJIEF70VIAoWzTlaQV4IaNywpwxlCvRp6GcWegXfCSWH83PkC5i+jbYtU5BCCuu6oQNhJlb\\\\\\\/bbb4TSolh8XWtlRDz3b2jAL9b7VxCydO2MKmehv\\\\\\\/D82swsxBfXEEyRPhP\\\\\\\/1M0Is+TbQuiVrs0cbw9Lm\\\\\\\/YDem\\\\\\\/qXZe01dE\\\\\\\/ByD5rRyia\\\\\\\/nmC+9VR6Pt+SqBeLAte5qn5RFgm8Af4uD3qZLftZspEB8ofqrYllyVdNsKUt2JWC5OCUoDx2Y0AoWpWGZ8L2xW8lBc8NoRCGcZtVZiixogQpuBPQNltTUFt\\\\\\\/IjWdKHECQWvk9aNY5YPiIvpR4qh0MiQAf4i+MifrKLMdrrTWhq2xjyUu5iTQEP7mQKf7t72R4Q7I+JKL\\\\\\\/3oOcuIcvt1REW9jsrJusBXR7DKiz6fUBmp3IUxC\\\\\\\/tikkILEnFBOqPW\\\\\\\/U+ua0\\\\\\\/HJTx0DmRSeXJ30fsKo\\\\\\\/bJv+TgH\\\\\\\/XEjK0aMXrCOGgycDXBo2Jj0GLKblWOAQ5Q\\\\\\\/e2+1P9LcAkCoctTJwB+Z49O5u7St+u5ddEZRfGjzqCfiQvJd32BLzvca\\\\\\\/DBXEInvcCEAptBdXXSYaG+eC5KFg82DehpmnSd\\\\\\\/3Az2EWdyt3eD83TixqoMBJN1uu8TvyQkurCi2LAjybD\\\\\\\/nS8Jjr33BCrmoDwjHgMuOFJ+BztJRrqEi3N9ZyesIvq6qIeA7YlAsln161zpCQUX6A\\\\\\\/Xo5rlWjsUoSPSSKa+60jbP5OOHauiGwp84Ae67tQD9d5Ab7xAQMAHNtKec7J9qc1ZkSkCrFVUF1vtDgtadHz0zfnBCEBjaREFg6Y+\\\\\\\/67XVtcxTxHkhvQzWFyfHe3NFT4v9kti\\\\\\\/UwxQs1EX09\\\\\\\/qS+aqgKNemjrvzQ5VYMkME+9a6ORbly13jNnYiIjOFccuKg8Lsgn2Zze26WCF1NtIZ\\\\\\\/BM1zkohFCBTQinJHe+z\\\\\\\/qF5wtCMEZ3v50YLaQmEltUfnWbp16hrEgL+ozdppQWCGIDDihNULKaVcjBtWCODg73lUDeVtwjX27hhsL83E56jl6tI7f8jY8H0ZAidmSUNplZbpTS1NngR2OsZF2Ysfo01PvXZILP4VC63dvIyKdZxomS1Mr96L8R6VrWC8aLg4vpucZ1g8qAZZr7e5+V4VKGjCB4+5+zpUlYcLQTU56bYGurWsUuImvjyDO6ZUHRHUAQWn8UnzDHCGTjMOkuozINVECIP1\\\\\\\/3Jc3qTC5+6ihKRtTS+306ExV4Lg1BUyVXou8oSokmtcHmPLKOcszJxD1jXB+AJ67Hp9FPUf9WEnmA79aGRJnB\\\\\\\/7Mh\\\\\\\/aq3Mbk8G2cF1\\\\\\\/tveZH55Z918ljjajDZOcDan7wNwERGsMphPMH8SI+plCTSTuCitcCTUqFkpG\\\\\\\/5nJ9IV70UgClIVYpYtBa8htIHimZoF4tVHEqMSPch5\\\\\\\/\\\\\\\/tb2zvz8XI4uFkarreeALGNyBCqkrgyKLaICoqWXxhEUpgVI0INrr3paNrdXZD8\\\\\\\/DUDq1ACMmP7izpVAIoCA6pvKHPkCXSYqQbZ5qKMaIIzXPC7BkqtFtEP+wNe6dphb6Kkgl+4jR03lsTw5UyLk3MgNIqKU4FfpI3eE+Bc3XFYjl2d8tCnhkBqz1WDcVWkVuJihGCRXbEDvg4Ac4jVnEmJoWU19jbFxP0UEhI+ji4fxkdAX7t3WeJVpYYh58llORZCzErE2QYFTR7Lh+hC2xukni0zn6jQaP8P1HhxD5ANFKqKX2dahGiOZCr2L9BNiCoPYUw\\\\\\\/EMd3VSL705sQDN2UbXwAp5ObDtVlHr6w\\\\\\\/EnV6\\\\\\\/XsTSQpd\\\\\\\/bosQ6g2YZCpjbAEHAi5J75qCYx8AVkoDrR1sh+XTyUldjMOh3WhzbfFJ6UVlM2GhYiwyyTBJA+ODte+9Bi\\\\\\\/ugdHgMY+PZp6kUKqay8fTZVz55\\\\\\\/RFvZfxrL8mbYv4nsLwEjJm1mhTEPlok7I+UdqdxGArCRfCR3RSU7JzYbMhsaPt7+9nJ4h0kRRi+MyNkm6eStbWpoy1E11q5ZsGLL5C2f8Yk2wJNpjs\\\\\\\/kUh9NcDLCaFRt6Z\\\\\\\/zwzH62iN1mscnw0aipq\\\\\\\/usRxyf9Lra8HBphnt0y7GNFWyNtrTaV3VKaqoYXw+Jjll+JUK7cI5BOnVZ55fLh56H6uthknbUeH2pPBdfJ1zKWffdFoRmXS+l6CX5To0D4Cwa\\\\\\\/oF4Sw4aFN8i8cRpEmGxuK\\\\\\\/ypLDgQM9ASJaxByESbZZfCJcECLBY\\\\\\\/U3vU09QQllH61b2AKW3OJZKQun\\\\\\\/\\\\\\\/LXPcwkbsHNxxAm8vACpsvhlPUaKOHgbTSXnX2xy78aC8yM1m0cFpgnr99tfC8V8zioBczlLzulaDZkSeyWYfz7YAIH59712wfAZ9uuxmn2bFM7jJTuCx1wWJ438jQSNt4q\\\\\\\/ivYgXpp05f5B50x8okeoHMw6FTNOjs5c9muR9lzP4tq+vOwaNNt26DLJ6NBaM93zOIq6TSKbooxhtZ33b4CwcgGhrkwjM6kMXa5KPue4z2lKzKUFBq02oVQCHvXdpSSr3Y2e\\\\\\\/gFD27Y3KfIadOTlfQ6zdQFSChl8Sj6PWeOk77PhPuo9u5O2z3vcwcN5Q+J0e6zPo3o4tjR2oUvzCq+yP72Pyf+MlaW0fplaB77rHLQqc6tS2M8cqmFiHaGnQEWW+2PyLon0VPAK8CproWfPcghasyF0sj\\\\\\\/liz\\\\\\\/CDZ4uXuujlMqsiaswoLC7xXl8Vymb11qmU3xZW\\\\\\\/uW6Be64TXJornyhGahMEd9nV8K3Tea6Qd8cj+3CW9d33YEVQPL8XcFKmDZOsmpuItOrVgZANwse1Me5tbFZos+8qTsdjYgN0Fu25Q2mOhzVJiYkgjFKsrtZNSnp5M2bxsFuuYEiDA7fcqHBjnKHoOIjaz1UyveaDQaNSRt8RygL3LuJoigweMIPDwRUdhAdZBR\\\\\\\/ViHcszSqmMlFbNTwdjagkeiBZpMxGC6AtnfIiWZPHXkzVxzMd\\\\\\\/KMRUeAs\\\\\\\/KOfKZQEqFhjbv45eOGseGKSjSMSWIAYLBk9YGv96AWTa0YpzN2aHFy433PjGeaQyjW81OkjLFMZZTFDgFkBkoHGghozwQ3ghJ6rbLyMr5HNu\\\\\\\/KTD0Dixfua9cnYQKEkJfUfq\\\\\\\/N3yPcxGj2JvZfx9B3\\\\\\\/tNd+LfJZf2CsI40\\\\\\\/3yJ+HD2pl\\\\\\\/gCpijXmYR8jV\\\\\\\/Ymm+QH+uzPGkek8KUmmMENpyiTX\\\\\\\/fMZzyGO2NT5bAU5ySxdA+dhpGWw+ei0CIH9s4Y1EM7DDIEZ3rKzq8WWh+5WGPPOUZTh6wgMM5S1Tx\\\\\\\/dKqaYxYSgCk2\\\\\\\/uo9Lcfem4M\\\\\\\/SOtJ\\\\\\\/Cjlh7Rm\\\\\\\/Oj84uH+zcsDP3JqiypBXQmYuqDyTXfVZhrKUJR2ta1FVykfK8IRuviWu6fot4W9SW7keLQhZEMkadSyYdJY9\\\\\\\/RBL4iu2XVqQvVtoHG+LhjWm1oVDV6KGpF0EZJlACuYqFviMzO4V7tSYRpUjXyrKJWZKu4AfPk91Jf2A9UX\\\\\\\/gDTTEzl4OE81FOhY9NZ1I\\\\\\\/EyNcei35dfKJ7wMS1rw9ZcXx35CcAHgU76LozDihltUNRrX3NisZg0Z\\\\\\\/aNRtqAkrlMhSh63J+gCk8NKcerIDTHNaoMYUD34lD4+JJTc+thkEulprpEjPC0lfZSvLw1u8uvYdUTze5TVPf+cBklawuBY2\\\\\\\/0mPtlnv\\\\\\\/dqN5\\\\\\\/0e2C8stxebzHm9pLM0ewoaQWhcXnxOz90sk9hDrfS58ybSleTNMTRxz8arp\\\\\\\/JxFBzYBZVx9YbKjGBzukL6LvbwDjGjbVUmJMvqhxN70YZ30UGlC9bTyO3FkhLXTzIuLVeqbUc9JT4esOQo6Yiyj49+eYAVF8vaSkf+eWVA+1Mg\\\\\\\/A7FrNVaKomJMscuNh8QNJB9EhUk\\\\\\\/ijPWOJv8vSm3Leoe4eFrGo1\\\\\\\/tNd5+grabJOvziYu8kCxpknGRbdSMk9Xbi3XLpRZPg5cs6jJmxxU19e5svmR5kQ5XcDr6+nWGepEqhAM++ziYI3qKlfoNoVdUyFwUfzAe5UTrlgM5qH8I8iutlarQE3VX+HUdDuB86oOuYjXcEqfsOKXDj7QvQlwI3W31T5or4t8FkNDwjIK5He6fXGdGLxDfWGClneDyW4hRHI6bCzeJi0m5mZ5T1RxkbmkBgFOGFUVKWW1Ibf65u+uVrZhlKQ0vaj8hGeMUT3EQuiyFy1I6\\\\\\\/70jHwdZBfnyVVIq3IV0AJO3eVR3lYtVYELHrULun3OZCzRdWtqFotiFh39CQe7cTka12gNFsPsXd7XoB9\\\\\\\/y1IMbXVMfFoD8waNPQcysrcUINz3DUKrOZEnRgTEMK1xlv19ngT7qenm7rDWQhN5yhLzCq8csTkhd+bBYuQxMNAgOBFArjW0CcgRChk6Rerj9n+tnsHnkC6XgjMTNIHvug0d4qh2AEV7986cIi\\\\\\\/PM2QkssXAwixqoh\\\\\\\/uWPpcd6CHdEr0wJhvPokz4CSrlOwhJGPmGex4Fz0gUGCYGqO7BVexUUpMYzXMwoWP19RgCqaYXE986GbqppPmdPZNUkj+trE0J6y0K9mKl6SRwHOxz1Yx0mhzfVz0jg3chgtwgbafXwWYsfeyBedg7AvpMTepFZmnksWI\\\\\\\/iojoxZGeSQUnIrSqsTuFoa2VoswIVH85OktvNRiEPDH7ehVbGjmDqSLOPY1ockHc5FcRC5fc9oqDfgI5o\\\\\\\/\\\\\\\/tsRnrd9R63VL7CLMAnP6id5fgGs7O5mIuAZCmUodofNQNUlbfGjnPymKyTq3EHUgO5ARcA5oqu+DHkND7ulT0IsZ8A36TenKOwbH3jaLQp5aZefEeCuQdbbBMdk1sFdKHk0pJvx\\\\\\\/jqYGj2gjh9XpQ0rGoGssMr7CsOVpFQQRRr9G+xfFkunKxUFY6dtwykPx8\\\\\\\/5o0m7rjkrDKYI8F4Pe2WQjfqxlV3KJaF60YvIeDVBusWSgkD22aLaqduWnHCtDh3HO\\\\\\\/LIbD1+FWKgI\\\\\\\/1dVlR290t0LzAwIkkBfhh+3Is5Yz\\\\\\\/NYylWJ6OTGoT8gDQ6BvV8F2BvmWKBRVuZsCC7CbFVMM1KLaXJOsuUrIsHu2WIng4DVQp7o3lqJd8xkoocECJAhMgusTZmCq5hK6CAUpYLrv2AkvTaSR\\\\\\\/FCKvdRQ8GG+t3b1DjP8EJBFKbIf0Glh7KJwXVEDHR6hYOog9AzwL4qFm3V5Hv\\\\\\\/39D+E49mqHtz8rr3JKpVBXFYv1wc8PpTb\\\\\\\/myXQNXQSvEEDtI1WM\\\\\\\/sE8W1e\\\\\\\/p2DhLlStv4Jd2o8RHN+L3a8l8qReLDaMWljM7Vs\\\\\\\/hw3ke8lI5H7Y8ftpNksnev8PZsJ+2BFaFHWJT\\\\\\\/If0o0+OeAPQIzVHDu15dWcE0JoR+XMhzOX1CpEaxzRdahS0Odnr7uuES8oaYS4RCDe7LE0ZDKUjn2OTiozWJRkldZPtLURllOwHTyRIbQ2iwiqTSQqRLC4xzbq+ToEvjf5axT\\\\\\\/Yq1Zu1dLZDInxTzktrpV9dfvcgH90L2CUiRhEBjDglhCrCj7xFFK\\\\\\\/LY+6iPm4uZwaBZddJHRL9uwT8wPfCgVsCiLQAuRugChrJIxlpTddLCvyy7lP9R6u1vWhmtMbCtbnB6pk942WK+N+iBKpr\\\\\\\/Zt+avIubnZdQRYbk6MsPYHVDMk7CfTeCU15ipAKqf80Gc2p5LCKwoAeHM7GQa98nInNDI6BnUVwiRGi6Jxna8v8QW+on0kFDf1HAx9ZEfmMdrj4ZqvISAp\\\\\\\/4zpeqRk07SR2IqM\\\\\\\/9Rw0O\\\\\\\/2TV5r9ZZ4EK3pqXdnTGCD9LAhvWpuJAE4y3qsie1GPHntZuWzz86oE2J4\\\\\\\/rZ+iiCMsItZ9HDKvE4nkk0Z8pWv40abos\\\\\\\/9RvpHOieEYFDFLGC8VjsZaamF3ziVIpVy2ekIKZ13lHAh3dhHa7mvWxvaw18gxRf7FuUsLYgFhYk1mOIkeaRzxgH1skk9D\\\\\\\/zXzWRmVQO\\\\\\\/HxZF6LiZ3ewBuX7ztPFsML9W3SEmqQPHHtWf47worOIszUp5vS1id4McX\\\\\\\/Wc4Rw33COGgDlFR\\\\\\\/MQARLLL7UZekdYV42WPXElKJJvRqWzXrdqd6M+svo3wdKxo\\\\\\\/UiBNAbURgYyXOtMPDu7nArYw\\\\\\\/iDzMroxXsAs7FskXsPjs70GHgXvUwl16EnvfioR2hALsr4Np26AC\\\\\\\/0uVAbAc5bNkJAT\\\\\\\/KWcVdOULuAGzysafRSo2bXX5pciXsrv921IRkKOjtJbZ6vDvv\\\\\\\/H5VKh81AsMpF0et5vCjwlTm37qyT+tWQdO31DIYFprdHunA8QqqRSx7Qvq3UIW3XOWbAOLr+pwNXD0IZgJfNDXPUSW1HNNrgqCHLyLvD22Nj5XpdqJazOkPT0PnrTOz1g9qjUr+r86GR1ZT3ntAxmEy5m6XaoG1gjHEoKKvGTqNtySjz7ByjV29VGKj3uF12iKmSSvrHKc7m4gtg4v2O47VgF5+FrB\\\\\\\/E0PSx\\\\\\\/H53wZ02KALrSpnO+cmZrZRnVKQo\\\\\\\/PWMUejA5JxoQnAOkxYnkQqz1fBAeskftR4yS9hzo6No5hGCRfPn8r2ti1A6Vwg0tfRueKNiaFqlV4H\\\\\\\/6Z9unFdRmuBy855wM4aBgBnkDfhWq8RYbW7N1Npz9c9heB5Sz+vAqEDKKTueTmriSM+R7Kh8ggroOnXQ94kXhRACB2WCzKRXesUImMb3fUUXzZyHKWVBngOGyblAIASDiYmpqvpcB\\\\\\\/CqMH16Pohopt\\\\\\\/9yVdeEeOQnOpGhw4pVHzMlfgyTJZd0SUqajbOx5gvmnC\\\\\\\/lUmpwmHqVYs9nCP3WoY4ybByHfaopeFxVROuKEMC5Rc+uziyxK4DW6i2pQiITlupTucE0EqOi7YDumcYHHtt334+c2HJz6cG726+7Ej5EhPhvxjDeUVS4ow1OEeh4Z+h7dsEb7q+v+DpIwonrSiS28+CL8xuYIpxBYQ\\\\\\\/kdxCq6ZyvccRXQbxeT6ODKCSg15wUUcMlsGxa3l2d6srZNHc+sBSg51rR4Hd93ZWn8Yd4m\\\\\\\/l1bSjAY9DPFbXWWgV6RBk8\\\\\\\/tRhknXJFDuP++NdOgyK3H2KmH+9vOUCX+Pcr4kKHNyJocfdenOLCosZBjA5z6o1Wq1RpmbC8mNNyKuo1wEzR7IHVij6DUmzA6Dqma+lfp7WMajzJpdfcn8cZPU1KWPDcXA4Lw00P+lH+2KN8ulsYD8uPuwN8UuP2WyIOvl2Np+Uzd4Xzyt6Geb6aDhf3Thhv54b6B2A0kL9xUW4fW3e6jzITmN7NSBofZYkgS5InoOmzcKKQZSW5IMyfGptNebm7MKq8nsSxng25aGBaar+VFlouwtKrYsT20yUWyVO9j9O6QDbwnw6nWq4m1KgDRKtca3IG4sjpSX4gnMVYrGKaveMl4Npc4+0POou1R+G63IM99s83lgJ7j693Rq\\\\\\\/8RlBHJeBTSxbqzt3xt6vhOY6TCAFpOQZSYk1w19oPxIOK3ghLGDPmpbKsionhUsDr2CvKNWBy5EvPbxF5iCum1yJq+bZWe+NJfcGYsB9TXY+e7HncEg7rUyINNkYrZ+iTIjxWCKF7HcGP7WdomTKBLUyzRpBqb\\\\\\\/nsbK\\\\\\\/jkK5kM3xepk7mjlmoN\\\\\\\/Z67HyceF8pr17J7qKO6sMUdUiz3DgLs4LWgvRRhMkaQjC+2ROfNUTg5f6AqUNObnwn0Oi8nNDJt1jmxXeel8b9h\\\\\\\/Et9GCiE69q6Iqi2pI8viaWTr4DD3ek3tOZX8Bg97WZcAzI4MV3mSJjMQGwBW9tsJrDn851oy1FnZIYyn8csyqV2MvRdw4vYa8MiwAdmsopzzRi6xQ1pexCbORvYYyVI\\\\\\\/dGpPAepXqQCE2plntbQxC4+Bkz\\\\\\\/MLCgsvo2rHiRrDCjXzwU4xHghdCEWvgWa05ZxtU+M1Hp1I7Vdqhj3VKZ2KT\\\\\\\/sNTDx4\\\\\\\/aKeX09MyaCVStarPXvxUiB1YRcIIYN90MLPOIXpPsSJtZQPvHEH5nAzw1j28AtmxCjK6Aj1aLV2eMrVPeSjw7VH+Rzx8q87HJQ21qqlVhfuSfwqRD0kTDD7wKPMlN7WozEv2usMAfIjWf4oTHESoAn1+\\\\\\\/Sga\\\\\\\/yRhKS3pLsiX9UdgA8spQBNeQcUCoqsP1DICEB55ZHkTH3eCKYJB6HuEBY9g7sg\\\\\\\/Gy9BKNKPGRSxXBCp9QlD6tPGflNdBSVXgGudBmCGwQ3qW6apa2r+WfP+pb0nZts3T4m0XHONEVczY33qslM5mscK\\\\\\\/yePyzEwDKn+EhR0U+zvs5twWQWJi28OO\\\\\\\/lveUY9kQ49WDbxodnQwZyVw85jRlJfHNwBBbRyxcEAHXDjsC0NFTlyVGFCsOAcIF0x5b7W5XOsRX1jJ+17hn0+9sgDSnloDfpas5vvkavVcxNVAZp6m5SAbX2CRySLdqIPUfiKGF9KE7gSmkdKJoi+\\\\\\\/4hTNjJBbqeULYI6QgieJ43ZcUqp5sVIrWkYmlqFo3Lo9gXRiigqQN3LZxoTiAMIiokPYpawyDyE2FifibCKYVBm3TbriX6JME+QAlSdomCId4Ix07coXKD39ydd1eJvMW50hdq37r01AaOQSHZVpsUBVLC\\\\\\\/9PVQWGZYJ46+TFBQC2CJyBfFFBnWexzm2EN8JgZ8G8tuW1EknY5Pxf7ACZV\\\\\\\/V6lHk5lWbBxVVkqqfW4QO6jPB8Pi01Kzx6gjvy0L4nCArmXZM1Um\\\\\\\/oaHJbdbmeM7rYcfkkMPtqtqucCOH6\\\\\\\/Yuc7tgIAdKog+h29pzCzxhEQjll0N640H1QOJQXwO8hWf8GB22Cc5+EseUYeVkgsb7h5iaJiqnSZtDmNsIk79O+6wVYAMpR2x+rqZWWhBa+us18O27Zhje6D4WXBg7b\\\\\\\/WZKUvi6wLS9WO1gVmpr06lmQkZfVWLwrOjqV83qu1ScXARUL48nKNrw4Cvu6ZY8wG5qqmWa6tvQomXzST2vvzDaH5hwGjg+MSD6Xg3SQM695FqA6I\\\\\\\/dveO0Mbmj3fpvhbweLLF8YnK+9C5hH2kocW7l3ykpqzpmrP3gWGiFHejnC2\\\\\\\/IgW9bDCFomp8ER9cf1ozNolICUqQ4FKcZiPqofSIS7pTvYefsbseeZ\\\\\\\/CvafLlXlHTU5XB2sPfNNDlnUXp\\\\\\\/tT1nkwsGMfNi7OWTadn30uOaGYe+YRAJv1WijDEI0t7wCELtlBfbpgPbzARVzGmC1CFCevrEXdndAaRdoqa\\\\\\\/rs71099OvG7Mg0YXhuzlynUjZArgpwL4Koywi5LqkzdMRBqEhZws7xePTnkT1GoylJRwpOCluwzo+xbLqH7vyG9mife7Fs6+K1lQ5DiL5L8CDWncWLQov\\\\\\\/zviTeCb64i+otcyFo0j+MAa1\\\\\\\/FZd8S7k3pjiXBZClmQTUQ0vSHWdoqGPbGuIz+hEYG7EFdvpqhOgQnmikaZ\\\\\\\/5EBIj5RAzBvtUfDHchQI+AYborodZxPgTBoIF8qJgTpLR4mwWQYCH0dyxcM2eV9VJVOPGQEh\\\\\\\/Mb4uujCJ0pLRsviLn3fid3DQcQ6PydWHZeIfdvuDzWMFNRvLPFpkvRCrG48mREB56i7Mo62kVIn5c7Vrb5pjERDyja2IQymIKnnidQcpkJ6w6BptL5mtccrQavUPhH5LYxeShp6UCZCZ2abC0aBZ23nxfx0v1FF5dbv0\\\\\\\/DFMATfT9evXlhTBwh5+qmzdtZIapbg\\\\\\\/ec3Xzzg9MKikr3P15t0hRhtL6Rbu6X4GT\\\\\\\/DWU94a2X1D0qHea9UB1B898eevj\\\\\\\/3LwyUicRbBRem1lwHEQnyCfoGbU6jravLQ504CfWhYO0D5f10LPkKRco1rMyCHmUbaIYUyzlnFsfhxTED0qt3OkPxzVdDu+gIC6RGFdOG6BhhrTfFYv9HkmItiotfM+aSMnByH7deuSYgjwcFWg+G7tOiNV4Viy\\\\\\\/1XAGPZAASczr6d2\\\\\\\/ZP7m0ICfDIEeYwkRMDqV\\\\\\\/gf9bZQBr+5WQzeMo3UydZaFbraTT5FAFyIj266bl3TEfu5SUtTWCk1jRWduO0nxhB1irSorMuXG3OJJE473UrJDERlu2KBAY2zUVVKOqwlQrbpx4lYYTIJuY5PhmMQjcAWTO3hV1ALRDZuBodWj7oQn9yiIyyJDzA2GjPbrmpDBolOGTkDCcND0rdQAuJGpihJzg8rQBtKFl4AaQYCdJoVp+KlATInuA7QsdEoOpYeZbhDhnI5PTIsEjvt2K+qUQDKOmL\\\\\\\/K4qZZlTOkpZh9OfmJbu+MpqpEfkpbY2cDaVFew4AJe5LeUMNQUlAnOxMSzTIU3cijuPjBsonTxLIQfITp7IPLCSGoSUc+o5\\\\\\\/dNGgSuk9C\\\\\\\/fGpSvJ8j6rhfR35+dlTjs0+bSITVtRdbcGqsYXqAIUGLnGtlDSWGw0OYIp5bsLVEfnu6GZrSz8Ao5rJZqqxzQA9owmPkGz6Yh4Ezp6Qsqaks5t5jDxxPZwWY4wfTE\\\\\\\/tdLRbjN64HzQcehpnsrdiOLcaKEhHiNns\\\\\\\/102XhdfZYrls9myUttLntpMcFGvB7JW1Zvuf\\\\\\\/Gbj1272skh8wktpQ2hdyQna5E\\\\\\\/2kxwoFYyJWAIZyqC3PRg8DahZ8UnpeszLXlJhy2aVQAoLZsO\\\\\\\/cLsoatgLl3htGJLFhpgXH1mLWp6k1yZqxVZ06wtx\\\\\\\/B3VjPHeOJfJ87Vxyc2S9kKm05EUH2tXKLV9U8T41hYn5Qlkk0sHxPEiEKX17Nd\\\\\\\/2AU\\\\\\\/6Hw2e6+4\\\\\\\/v6GELswcUnwlZnVw3aeLYoNPy26SByeCMx2pIaCnrLxvFKcOikA\\\\\\\/mpN85QVOxUSBgwVZvF1f4Qu3TZMs4l5L3sQTZYJmblf9EBtNaMPvpwgGalPXyHQQdmkVxelJ5AaVlHFdJ1Reo6ESR7LTOcijzVL6T+k81hXamWbRORfvTIKE+1ni58psG1dschtBWkEJQysfdsQAHVm9rCxK5VW\\\\\\\/ssI4auzbpWeYUftB0Ev+EBlR5PJzPJwmEJjOG9WdS1bwogQaj2DBRmJv1yUBmJoxoQQ+GssQ18SCx3u4GaqQaMHi3rFDGVq64iS0xV8uzvbIj6uKWPcCEl6i1uqRKTvT\\\\\\\/jl1\\\\\\\/sqc8CGZ\\\\\\\/V125TyKUd+oBuoPxsXqVyB34K3iD3T22XeBFVAX53gciU9N6FOubdO1kAevPseS\\\\\\\/LnHyJMSdNseRJEJ+nDheDa6ghEX4BLnxhPW5XYoCcf3f7e1AVyDuRA4QckA7q30\\\\\\\/tI93eet2rech8wp3H68eFSEO2F10AQ5ahFZuw+pbpFd8sghXaPvj6lzjrfRvdlQAdZjQQ\\\\\\\/cQ+2NgUNaA3eY2BJRo1fD87mTwHf5VKjkslS7bJecM+TIi\\\\\\\/HywnRa9XSpVOCNLtPAqLL2T6ay9G8EObjh9aRZTixB15gkw5M5PBEzUOPa5sKcroSLeweJQnYzYdcYqoTK+fRsPnD1b7wURjSds9lsvK2hEVE3DcHig0a8nPQzueSKh45ViGuqHXtSScph7IE6h+UjszmPIBmvUTZhj4kaE4Bk5mNklRlilJZGCDCDaTwlaVvUHVBqIjEFAIoTyjw7NFWBAIw\\\\\\\/3sk5DCmTgHxP1giJb0g\\\\\\\/jwi\\\\\\\/OUdJiuutTlCdBygG2WT+B4v8mkoJlawkUxI\\\\\\\/RBHXSUPv0LKPXaoAr2q\\\\\\\/FO6zlHs7jl6IWmB1jM6Etkzmx\\\\\\\/CncDG6MlPvfkY2dpgrQchSbtmJg3r7BUgx2bgx6QbDoxma3iBwGj7PdJ5YQl5E131+l1rQW97R4Vw0RorfdDhxhnnMZIaXkLMxH0aUSemEZILot3DwUCukZnqzAXWi0\\\\\\\/HHYKs78bE\\\\\\\/KU3qpvuo4uBoReN4nGyWT6GafMO5gle1tixnVFQcBdX\\\\\\\/cUHrUCpQY4b\\\\\\\/U6Vg75VeEP8ieVTrLJW94OOnGREZQ\\\\\\\/3VmyJh33ekg5bxtaUcJXgYOhs97dRgUjFlVxPMrBucHIin708DUWkB\\\\\\\/2clhd4rWN1jFYbaWaZFnkWJUxWwPeMNeD51B\\\\\\\/0isabA6L\\\\\\\/5SRo9CBnE+IAvFNiH8RixUdtH5SdVVuiuXbRFfjm4tGkk25VpcdxIKvLmT78dpFFE0jkM4vkzZn4XkIyOOhqoIrlzvbKe7imUk\\\\\\\/A8YIT3La+n3HQnTLHZCuP1nyQPsbxmG0JWqFWAhT9g7xwLQeVt2dzfURFzYRmGih6FIqPzRu58jk+qBsUPX2Nu\\\\\\\/PPaSiShwV8dQpw\\\\\\\/JaLY9GuJKxGaKy6p29lVuigItEwwpgGzFJA+baWk7Us7MLxlrDUKN61ZYgqkEEA6MMFzU057b1RIx0eBiNtmmbaL0fhTrXkj\\\\\\\/2A6W4eB9JaN5w\\\\\\\/Ze2IyOvRCvANsgDMXcsbuUHtiOuDm2w0K3Zjsi3EkAH9RTGin8g471sZq6nT8OYdUXE9FwHdNInVUn+KuHMow0q2z1J7\\\\\\\/DFF9wdPYNIbeQQHDYc8ZL+aby4BsBBFM0DhKy3LzXKscJX0JGduUXgKs5qXc\\\\\\\/71nE5esYRepx\\\\\\\/349YZJ7S3S943ZMYmBM+I10c9YAfsErvYXB1fcAh60M\\\\\\\/G4hKUqdzaaSpXUHFOuao0CYrLsK1D\\\\\\\/JArh3OGb1l3Jkr73a3wUcUsIYWYq73DQ2OT3necptK3sYRoqlVvfR5OIqBSF5pfW03WWX\\\\\\\/sA2X3pTA7ySWdaZN79Vag+MjSoopjZ+4u1IWY14BJamQkvVrfI\\\\\\\/3iM9GVzUAHPokgfuEPOjn9kUHEWKWUEtaQTTxqOGh8NdA5hvEnv5YpJ+\\\\\\\/fedL6mkHMcmZV61gazwdUBkg87yTjjSPSk1Soly5zO6IwgARI7+koTYCP8MiH502JFlLRPeBR2pcIYDp3IH48TYrxSGcDy367\\\\\\\/6V6jCu003TVMlj9IDIc8lzqll5SB0YJYUf\\\\\\\/gFSxNqBlCKcA9+g4J\\\\\\\/a3AdO+CiVTjxL4mBKHAkNgm10aGAyRd44XQNXyaBvyCLOPtpbk1wnveuPUPt9IfjSr2sMRjDzSxbH6B03moF9ZmK1LUPrTMYsfIXM9pfIFj5Zj8Rnfbsc0o+rzgT430L6ZcO209b5dlXeweqkFm2R\\\\\\\/+rm8J2WuHlG5lApXJo6m+MRCB1oXayBL9hY9w4\\\\\\\/9M1hGca5AAPPe6lO7Xf\\\\\\\/1Jwrz4E0K64q9q\\\\\\\/vRh6OOIumU0Jvhn0ZGuHU6qnuMJGdrdDOCkoihvoEQg8cwx1ntSLW2Df88fc0H5bpzQxty+ZeBHdR8bDSfbi1Y4IX+TbCs7a6c0GyyBKqgS8QMcPAjpPoRFWZelhfzvW+g8ZCfLPYbBLS0roJy3qvOevl9+EPl6z5vIrS6GvasM\\\\\\\/RlYaf\\\\\\\/ocMRCmg3yIFIMc6Puht12K6Rf1Yrl59BYwK0V0sXI4MP1DWyJdLJBwz6JdC76h3PMQvgu+W+Fv1Ijlw9LHSOBr+TXflt5Fl42cVONvPoQ6ABcBktYDQPvoOzLF7OQiJBgkNfdKQQwNxZyXVMXrs+tzlDq0M+iU7F+MkcPUF2gthPKSem8w0B\\\\\\\/t3p1IIN\\\\\\\/sB4LtRcypeNrnT2T3zVeHSwQwhsyjElihDHc6qZH9GtmLfAWnvG\\\\\\\/Gz8O8eZ6\\\\\\\/IGP\\\\\\\/D0HQv9+SpSc0b4znbciZ0oaEyqq52hB3Tzkurp149+OeXMCT\\\\\\\/bLEPHSBhOWzZ1T2TLp2SjGP6Q8Y9LKMgwtqti2aC2+xa8XeUSQYKVShJQlVP\\\\\\\/VZ9eMqtqAHJgf7\\\\\\\/ZiT1q+BNt95YzT+ua0G4z609h8gSYLLGxFmdKhJEDeFxqTV9rfcB+MO\\\\\\\/21wZDCOWZT44VwNmqUtH8Rf21ddXqvoGYwZpfwLtsZZwEf7LtJcxzeB5MqP65DSp16u1T50Ku2t1kp5bnzADrtjpaHsFQNM3UlniAxyI8lN7rrzhJjVNUMDnz9n90HNih7uaT0C792yssXxuQv+YC\\\\\\\/bp8\\\\\\\/LvMcseF5eMzmFts7SDOsxzcbxN2AYK4FMpNf5HDGZ7gJklWVBIcAQJ6F2I3V+vNLOXih0qyzu38A1s1WCXB8Xnh6q11y9Nmts7h1QA6BLuds+G\\\\\\\/12xhGAH89YBfYDUcTulPU9b2eya9UMdvyBdSR75H6w3vBWlfxg5vWSAEWUWyfb4Sxorc2pSllmpgXEeDw5yR7JVY4oDOzkVASMjI516X34Yy\\\\\\\/YWTP3e8Qy8z1wqcxtAJ4v3bOU2g0rgNvYk3750Su05iA93hfgwvePzLxOvrhwgIIqhLRzi6YXyVR9mmG\\\\\\\/wUNuMY6TtBa6e9nwOuP09fBYc9PmP9d+T\\\\\\\/qCThOpiwGlG1+WthRNapTnawgoRSXzk8nU8B38ZfzoMBoiJIJFdt3rSk32JqaWAt9jx\\\\\\\/LKMKG9vRjajkw2hDVp9ZYX3sAhrCrhdS+DOkeNb+4VuQi72TqRhUKjtuRih3dWpHjvqkq4d5OXmcq3TrLD53L+Vm1PRqx2TeaX1kRYAoVW4uSbReAY348vPdrXYTsskYME+UyYtjes\\\\\\\/2jmRuiCFwDjU85vmFoYJYrMHOBui0lK5robDJZ2gVhih3J7W4gcCt60PSJpzElsP86RZwmxfabJc5qno6ptYlY\\\\\\\/MVXP5WiHtVoSmyGNmx91RsM0Ly4reI7m5RqoQRAFmwSGvqCc5\\\\\\\/EMDQzA0dQ1CMKuupLATIQ7OOEOnphKieAGd6DxDTp8xTXJFH4dv+wYuVjKcbxi0srH1D2KsqY00Wk7e+afblkeURdXt60RIJJr95Nxs\\\\\\\/1h1Ts3NvfxalhMYH3tL+RVJLzFlhic9tznCnvhsmgvmoRXmvSq7S0rCmncYyn2sDzK9Ypv6FQPK3R8eCDSu8dicGYewm69zo7aQCFGQeLV0aoFvIXAoAcYadkKyuH2Ag2bPYE+7mEJvhjYOZTeq\\\u0022,\\\u0022iv\\\u0022:\\\u0022C8T4DoWkKueTAWPe7a+7fQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022db7b9de3b4a8b8f0717c4b0854459d2401058493ab652b40eea25c71e063ae5b\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022LCxghXVUHmwvzPeCtegFlxqH898VFhcFOucUI4RuoyV90FwiV1NSu69aSioQ25QqYgLM4ETqNABAMTCxDAtVrgoU8HKsw1swNrC\\\\\\\/wc+IAbY2W7qG3HXZqVBAqOeGxSnnHFpV7X5UUw\\\\\\\/mFi2HW188ljQK97mGPJ3PLhMEs2PHJJgi0HW6SgNUxDvPLTreWPqRE5sHmW0V9QqGBeJ7gIwj\\\\\\\/2iyADku+cPR2F1x31pjbE104al1RwNT9IipvrjpNXHN6NYJuiUFGmDqnDEWMjPg3NyIBNG3Kuvjz8AAFK7MSCEskDPoeNSC\\\\\\\/N3cv8BZCGcKtY9\\\\\\\/BOFbXhwuDCvahBb0HKxeBYDiYw7QSf91FdgIHO5wvRqykiR5Eszv3\\\\\\\/GpLkhM6q1a2I7ORM1A2JBzkXfL\\\\\\\/VLLu4E30xQw6Br7ig9uKgmH+iAY6ixyeqPjU836+ULqK+qc5eT4KRN6bDtlUVWFZ0n0v+LdwCAXP4tHLkc8ULJ6uTm9PmmqZJqthvzOjwV3+eVK7At25ydX3M7tqSNDs7XLF2vGWO3N0iFG1UjxzIr+98Nk8JSNO+ArUH3RCR06hJoT8r46Wpz5HYkQWwT8lbIcD8JCkmvLaPj2u5\\\\\\\/3PFUP0XnLcXuaikVYbqClM9\\\\\\\/MgvURJ2UDwaBW+ItEe58zUKsP+0gEtmDgluA5w9q\\\\\\\/HoX69keBdaVvgy5rtBbaZBOrQJkZ51JfzVdqH21gitdAZ5E+GZH1KZD4ovqSz1HDhGRcK7\\\\\\\/hMV7qHmaZJ7haxvmVQ8FIffDvr9zImqVuz6n3sPR+46UuEtJ1haLZyREQaGhZECBB8RGsnaReA2kVd14UEE0qqguYIKyMyJGAmEFE1Cizt0UjZ7Pbc7KSqUPaH0pyOVv1R99RCAA\\\\\\\/hu3GQFSsxuqHmFNNC4lc2NwLnLzW2dgQbrjJjDB\\\\\\\/pR59kUL93cBEPHycXn5xRWG1JBa++BPwE6NjBZDFfxSdNhKnIg0NmAywnsEN2JaadUH0natw5qVyA+\\\\\\\/nZnQwvQnS0uafVA\\\\\\\/gUTE301XxCwu0SRUYEoWMVCoR7BYXxLVQ9kcrIdbx9YAShu8c3jEDyzuvJVJQAvB4ilYe+lpjU81Qn+7iB3SRJP4bNVy5IMUCaCntuhAzCYUhQ5yhjBPfEXgBuAAaSjMCyIdstiJayt5KcKKnHg1fWOdAJCr9rebMM8ftujvMihQeuTpNDJ86jWhNDay1duTpHFa7Blyt0aRHcNyZKBIssPThSraccZMjkTKqnnUJy1wxefIHtRhU47qBIqdUmxjKzsBqcd3efT4kcLUsbeE+ofTBcpNzphTr2Is6fWRyXEs6CFtdzMqUS2jTjQOkOwaKkENiAMuSmNSb4Cy9onar+T7ftLxHDhqjPgQUU1U6oEgKCx106+SRwYM4g+wv0gBE8D77KjqmfDkZutF6852O8TrTd5mR5Scm5C1h6FTlJUa86BOlFPIpW04WZ1f5z2GNQccMkgDvngQi9Ic9OhCrnOIR7YDGQy\\\\\\\/s9nDtzh\\\\\\\/CTacw\\\\\\\/MqP4gX3hw4dtoIwzlBj\\\\\\\/cYnNRlpq8DI8BN2D2x\\\\\\\/Ik2cd8wHqtOLIET\\\\\\\/lPRHbI4FH97kB\\\\\\\/gzt+wIM7XkfZPBbCCGMzqstgRTq4MXiYZph3yFKb7NWukNuALZO9j3ckvE0qIpeTv9RHGgLC7hPO2jv4UwIOre3pZjopTxIlswT+Je7+knwOs5Y2DCCGLVLjnfh51+UOFHOJjqoAA+iQgOF\\\\\\\/mhSL9KlQXJmnJhTEBGJfhirj7Ufv8ZQTu\\\\\\\/0JZjVuCeKLnCD+aZtLNGQQLxo0e2mysZ5YckPB\\\\\\\/f0uihORq574Fg29Mlji9UQYApHI8RxAsgVbVucQrdZsUFOm2GEjzC72zsijDtcEpojBWRlckStbbYsSGY0WmlnWvtlUWJDL9vrTz1i32FUSRLgGyb2\\\\\\\/+FilO\\\\\\\/uhTlJkanLnszv7ToAwXchAOgpAID1R6dbAGtW51E34Gg8P2b\\\\\\\/Uk1Dkg5ScO7ykrVxMuDuq8G3AqNWxxu6szmylo0p\\\\\\\/0rzIRzMs7UeNY=\\\u0022,\\\u0022iv\\\u0022:\\\u0022sFat2YtgRy8fu8Qqg5jXEQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a73a1540ab65f7529d1032faab0f569be948caf698ca89a057e9eb4fff94edf3\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460451 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/11/3671" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-r6XxPwU0nAfb6boXzt2eV-o8UUjdmtc-6hW10flAKHo" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/11/3671" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-2Lkvla-j2q-31T1sQHeqjEhHPXCcMwg0YHmIe6jLz5g" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/11/3663" title="Determinants in the LIN-12/Notch Intracellular Domain That Govern Its Activity and Stability During Caenorhabditis elegans Vulval Development" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/11/3685" title="Non-Mendelian Dominant Maternal Effects Caused by CRISPR/Cas9 Transgenic Components in Drosophila melanogaster" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460451" id="node460451" data-pisa="ggg;6/11/3671" data-pisa-master="ggg;g3.116.033274" data-apath="/ggg/6/11/3671.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Transcriptome Profiling in Rat Inbred Strains and Experimental Cross Reveals Discrepant Genetic Architecture of Genome-Wide Gene Expression</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Pamela J. Kaisaki</span>, <span class="highwire-citation-author" data-delta="1">Georg W. Otto</span>, <span class="highwire-citation-author" data-delta="2">Karène Argoud</span>, <span class="highwire-citation-author" data-delta="3">Stephan C. Collins</span>, <span class="highwire-citation-author" data-delta="4">Robert H. Wallis</span>, <span class="highwire-citation-author" data-delta="5">Steven P. Wilder</span>, <span class="highwire-citation-author" data-delta="6">Anthony C. Y. Yau</span>, <span class="highwire-citation-author" data-delta="7">Christophe Hue</span>, <span class="highwire-citation-author" data-delta="8">Sophie Calderari</span>, <span class="highwire-citation-author" data-delta="9">Marie-Thérèse Bihoreau</span>, <span class="highwire-citation-author" data-delta="10">Jean-Baptiste Cazier</span>, <span class="highwire-citation-author" data-delta="11">Richard Mott</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="12"><a href="http://orcid.org/0000-0001-6156-9530" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Dominique Gauguier</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3671-3683;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.033274 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460451" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Pamela J. Kaisaki </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Wellcome Trust Centre for Human Genetics, University of Oxford, OX3 7BN, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pamela%2BJ.%2BKaisaki%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kaisaki%20PJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APamela%2BJ.%2BKaisaki%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Georg W. Otto </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Wellcome Trust Centre for Human Genetics, University of Oxford, OX3 7BN, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Georg%2BW.%2BOtto%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Otto%20GW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGeorg%2BW.%2BOtto%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Karène Argoud </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Wellcome Trust Centre for Human Genetics, University of Oxford, OX3 7BN, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kar%C3%A8ne%2BArgoud%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Argoud%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKar%25C3%25A8ne%2BArgoud%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Stephan C. Collins </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Wellcome Trust Centre for Human Genetics, University of Oxford, OX3 7BN, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephan%2BC.%2BCollins%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Collins%20SC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStephan%2BC.%2BCollins%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Robert H. Wallis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Wellcome Trust Centre for Human Genetics, University of Oxford, OX3 7BN, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BH.%2BWallis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wallis%20RH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BH.%2BWallis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Steven P. Wilder </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Wellcome Trust Centre for Human Genetics, University of Oxford, OX3 7BN, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Steven%2BP.%2BWilder%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wilder%20SP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASteven%2BP.%2BWilder%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Anthony C. Y. Yau </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Wellcome Trust Centre for Human Genetics, University of Oxford, OX3 7BN, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anthony%2BC.%2BY.%2BYau%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yau%20AC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnthony%2BC.%2BY.%2BYau%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Christophe Hue </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Cordeliers Research Centre, Institut National de la Santé et de la Recherche Médicale Unités Mixtes de Recherche Scientifique 1138, Sorbonne Universities, University Pierre & Marie Curie, University Paris Descartes, Sorbonne Paris Cité, 75006, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christophe%2BHue%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hue%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristophe%2BHue%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Sophie Calderari </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Cordeliers Research Centre, Institut National de la Santé et de la Recherche Médicale Unités Mixtes de Recherche Scientifique 1138, Sorbonne Universities, University Pierre & Marie Curie, University Paris Descartes, Sorbonne Paris Cité, 75006, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sophie%2BCalderari%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Calderari%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASophie%2BCalderari%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Marie-Thérèse Bihoreau </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Wellcome Trust Centre for Human Genetics, University of Oxford, OX3 7BN, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marie-Th%C3%A9r%C3%A8se%2BBihoreau%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bihoreau%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarie-Th%25C3%25A9r%25C3%25A8se%2BBihoreau%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Jean-Baptiste Cazier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Centre for Computational Biology, University of Birmingham, Edgbaston B15 2TT, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-Baptiste%2BCazier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cazier%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJean-Baptiste%2BCazier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Richard Mott </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>University College London Genetics Institute, WC1E 6BT, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Richard%2BMott%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mott%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARichard%2BMott%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Dominique Gauguier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Wellcome Trust Centre for Human Genetics, University of Oxford, OX3 7BN, United Kingdom</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Cordeliers Research Centre, Institut National de la Santé et de la Recherche Médicale Unités Mixtes de Recherche Scientifique 1138, Sorbonne Universities, University Pierre & Marie Curie, University Paris Descartes, Sorbonne Paris Cité, 75006, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dominique%2BGauguier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gauguier%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADominique%2BGauguier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6156-9530" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Dominique Gauguier</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dominique.gauguier@crc.jussieu.fr" class="" data-icon-position="" data-hide-link-title="0">dominique.gauguier@crc.jussieu.fr</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/11/3671" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460451" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460451/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/11/3671.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460451" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460451/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/11/3671.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460451" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460451/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/11/3671.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460451" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460451/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/11/3671.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460451" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460451/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">To test the impact of genetic heterogeneity on <em>cis</em>- and <em>trans</em>-mediated mechanisms of gene expression regulation, we profiled the transcriptome of adipose tissue in 20 inbred congenic strains derived from diabetic Goto–Kakizaki (GK) rats and Brown–Norway (BN) controls, which contain well-defined blocks (1–183 Mb) of genetic polymorphisms, and in 123 genetically heterogeneous rats of an (GK × BN)F2 offspring. Within each congenic we identified 73–1351 differentially expressed genes (DEGs), only 7.7% of which mapped within the congenic blocks, and which may be regulated in <em>cis</em>. The remainder localized outside the blocks, and therefore must be regulated in <em>trans</em>. Most <em>trans</em>-regulated genes exhibited approximately twofold expression changes, consistent with monoallelic expression. Altered biological pathways were replicated between congenic strains sharing blocks of genetic polymorphisms, but polymorphisms at different loci also had redundant effects on transcription of common distant genes and pathways. We mapped 2735 expression quantitative trait loci (eQTL) in the F2 cross, including 26% predominantly <em>cis</em>-regulated genes, which validated DEGs in congenic strains. A hotspot of &gt;300 eQTL in a 10 cM region of chromosome 1 was enriched in DEGs in a congenic strain. However, many DEGs among GK, BN and congenic strains did not replicate as eQTL in F2 hybrids, demonstrating distinct mechanisms of gene expression when alleles segregate in an outbred population or are fixed homozygous across the entire genome or in short genomic regions. Our analysis provides conceptual advances in our understanding of the complex architecture of genome expression and pathway regulation, and suggests a prominent impact of epistasis and monoallelic expression on gene transcription.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/epistasis" class="hw-term hw-article-keyword hw-article-keyword-epistasis" rel="nofollow">epistasis</a></li><li class="kwd"><a href="/keyword/goto%E2%80%93kakizaki" class="hw-term hw-article-keyword hw-article-keyword-goto&#x2013;kakizaki" rel="nofollow">Goto–Kakizaki</a></li><li class="kwd"><a href="/keyword/diabetes-mellitus" class="hw-term hw-article-keyword hw-article-keyword-diabetes-mellitus" rel="nofollow">diabetes mellitus</a></li><li class="kwd"><a href="/keyword/quantitative-trait-locus" class="hw-term hw-article-keyword hw-article-keyword-quantitative-trait-locus" rel="nofollow">quantitative trait locus</a></li><li class="kwd"><a href="/keyword/eqtl" class="hw-term hw-article-keyword hw-article-keyword-eqtl" rel="nofollow">eQTL</a></li></ul><p id="p-4">The analysis of expression quantitative trait loci (eQTL) can provide novel insights into the function of disease-associated genes (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Chen <em>et al.</em> 2008</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Monti <em>et al.</em> 2008</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Petretto <em>et al.</em> 2008</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Heinig <em>et al.</em> 2010</a>) and gene pathways and networks (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Ghazalpour <em>et al.</em> 2006</a>; <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Chen <em>et al.</em> 2008</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Emilsson <em>et al.</em> 2008</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Monti <em>et al.</em> 2008</a>). The regulation of gene expression is orchestrated through complex local (<em>cis</em>-mediated) and distant (mediated <em>in trans</em>) mechanisms. eQTL studies in humans, which until recently were limited to cell systems, whole blood, and biopsies from the most accessible organs (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Dixon <em>et al.</em> 2007</a>; <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Emilsson <em>et al.</em> 2008</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Fairfax <em>et al.</em> 2012</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Grundberg <em>et al.</em> 2012</a>), have now been scaled up to genetic analysis of transcriptome regulation across a broad range of tissues of many healthy individuals (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">GTExConsortium 2015</a>). A key question is whether the predominantly <em>cis</em>-acting genetic architecture of gene regulation observed in studies of outbred populations, such as humans and heterogeneous stocks (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Huang <em>et al.</em> 2009</a>), is the complete picture. These study designs are underpowered to detect <em>trans</em>-regulation, yet the reproducibility of gene expression is governed, in large part, by the extent to which it is controlled <em>in cis</em> as opposed to <em>trans</em>. Therefore experimental designs in which the genetic control of gene expression is forced to be <em>in trans</em> can reveal aspects of regulation that are normally hidden. The use of congenics, in which genetic variation is confined to specific genomic segments but variation in gene expression can be measured genome wide, is key to understanding <em>trans</em> effects.</p><p id="p-5">eQTL experiments in animal models can contribute to improving knowledge of eQTL architecture and elucidating the function of disease susceptibility loci identified in genome-wide association studies (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Williams and Auwerx 2015</a>). A broad range of experimental mammalian systems developed in the laboratory mouse (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Buchner and Nadeau 2015</a>) and rat (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Gauguier 2005</a>) allow the collection of organs from large cohorts of individuals maintained in strictly standardized conditions, thus limiting interindividual phenotype variability, and provide powerful tools for eQTL mapping. The inbred Goto–Kakizaki (GK) rat model of type 2 diabetes mellitus was produced over many generations of breeding rats from an outbred Wistar stock, using glucose intolerance as the sole criterion for selecting breeders (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Goto <em>et al.</em> 1976</a>). This process resulted in the isolation of the GK strain enriched for naturally occurring Wistar polymorphisms that contribute to diabetes and associated phenotypes, which we previously mapped by QTL analysis of pathophysiological phenotypes in F2 crosses between GK rats and Brown–Norway (BN) controls (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Gauguier <em>et al.</em> 1996</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Argoud <em>et al.</em> 2006</a>). Further physiological phenotyping and multitissue transcriptome profiling in a congenic strain designed to contain a large (∼100 Mb) QTL-rich region of the GK rat in a BN background validated QTL effects and suggested that congenics can be efficiently used to dissect out <em>cis</em>- and <em>trans</em>-mediated regulation of gene transcription (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Wallis <em>et al.</em> 2008</a>).</p><p id="p-6">Here we establish fundamental aspects of eQTL architecture and biology in adipose tissue, using three distinct but interrelated genetic settings, namely (i) inbred GK and BN strains, (ii) a panel of 20 congenic strains that carry well-defined blocks of GK and BN haplotypes across a total of 35% of the rat genome, and (iii) a large GK×BN F2 cross, in which we tested congenic transcriptome results. Transcriptome profiling (Illumina bead arrays) was used to detect differentially expressed genes and pathways between congenics and control animals, and to map <em>cis</em>- and <em>trans</em>-regulated eQTLs and eQTL hotspots in the cross. In the F2 cross, we were able to replicate only part of altered transcription regulation of genes and biological pathways detected in congenics. However, we show that the same biological pathways can be regulated by independent genomic loci, suggesting redundant biological function of distinct gene sets. Differential transcriptional regulation in inbred strains and segregating populations supports the important role of gene × gene interaction (<em>i.e.</em>, epistasis) in the control of genome expression.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Animals</h3><p id="p-7">A colony of GK/Ox rats bred at Biomedical Service Unit, University of Oxford, since 1995 from a GK/Par stock and BN rats obtained from a commercial supplier (Charles River Laboratories, Margate, UK) were used to produce a series of 20 BN.GK and GK.BN congenic strains using a genetic marker-assisted breeding strategy (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Wallis <em>et al.</em> 2008</a>). BN.GK congenics were designed to contain GK single genomic blocks introgressed onto the genetic background of the BN strain, whereas the reciprocal GK.BN congenics contained BN genomic blocks transferred onto a GK genetic background (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>, Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033274/-/DC1/TableS1.pdf">Table S1</a>). All congenic rats were genotyped as previously described (<a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">Wallis <em>et al.</em> 2008</a>) to monitor retention of donor alleles across the congenic interval and their elimination from the genetic background. Male GK, BN, and congenic rats were used in all experiments. The F2 cross (<em>n</em> = 123) between rats of the GK/Par colony and normoglycemic BN controls, previously derived to map QTL for glucose tolerance, insulin secretion, adiposity, and metabolomic variables (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Gauguier <em>et al.</em> 1996</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Dumas <em>et al.</em> 2007</a>), was used for eQTL mapping. The cohort consisted of 60 males and 63 females from two reciprocal crosses of 55 F2 rats originating from a GK female and 68 F2 rats originating from a BN female. Rats were allowed free access to tap water and standard laboratory chow pellets (B&amp;K Universal Ltd, Grimston, Hull, UK) and were maintained on a 12-hr light/dark cycle. At 6 months, animals were euthanized after an overnight fast and retroperitoneal fat pads (RFP) were rapidly dissected, snap frozen in liquid nitrogen, and stored at −80°. Animal procedures were carried out under UK Home Office licenses and approved by the ethical review panel of the University of Oxford.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/11/3671/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Schematic representation of the exchanged genomic regions in congenic strains. Red bars show the GK genomic blocks introgressed onto the genomic background of the BN strain in BN.GK congenic series. Blue bars show the BN genomic blocks introgressed onto the genomic background of the GK strain in GK.BN congenic strains. Details of the genomic intervals targeted in each congenic strain are given in Table S1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-17889456" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Schematic representation of the exchanged genomic regions in congenic strains. Red bars show the GK genomic blocks introgressed onto the genomic background of the BN strain in BN.GK congenic series. Blue bars show the BN genomic blocks introgressed onto the genomic background of the GK strain in GK.BN congenic strains. Details of the genomic intervals targeted in each congenic strain are given in Table S1.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/11/3671/F1.medium.gif" width="440" height="227"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/11/3671/F1.medium.gif" width="440" height="227"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/11/3671/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/11/3671/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469689" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-8" class="first-child">Schematic representation of the exchanged genomic regions in congenic strains. Red bars show the GK genomic blocks introgressed onto the genomic background of the BN strain in BN.GK congenic series. Blue bars show the BN genomic blocks introgressed onto the genomic background of the GK strain in GK.BN congenic strains. Details of the genomic intervals targeted in each congenic strain are given in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033274/-/DC1/TableS1.pdf">Table S1</a>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>RNA preparation</h3><p id="p-9">Total RNA was isolated from 100 mg of frozen RFP using the RNeasy 96 Universal Tissue kit (Qiagen, Crawley, UK). Briefly, frozen tissue samples were transferred into cooled RNeasy 96 Universal Tissue plates, and homogenized in QIAzol Lysis Reagent using Qiagen’s Tissue Lyser. Total RNA was purified using a spin technology according to the manufacturer’s guidelines and eluted in 90 µl of RNase-free water. RNA concentrations were determined using a NanoDrop spectrophotometer and RNA integrity was assessed using an Agilent 2100 Bioanalyzer (Agilent Technologies, Waldbronn, Germany).</p></div><div id="sec-4" class="subsection"><h3>Illumina Bead Array hybridization, scanning, and data processing</h3><p id="p-10">Gene transcription profiling of RFP from F2 hybrids and from rats of the GK, BN, and congenic strains (six biological replicates per strain) was performed using Sentrix BeadChip RatRef-12 whole-genome gene expression arrays (Illumina Inc., San Diego, CA), containing 22,523 oligonucleotide probes (replicated on average 30 times), allowing quantification of transcript levels for 21,910 genes. Biological replicates were individually hybridized to the arrays.</p><p id="p-11">Double-stranded cDNA and purified biotin-labeled cRNA were synthesized from 300 ng high-quality total RNA using the Illumina TotalPrep RNA amplification kit (Ambion Inc., Austin, TX). cRNA concentrations were determined using a NanoDrop spectrophotometer, and cRNA quality and integrity were assessed using an Agilent 2100 Bioanalyzer (Agilent Technologies). Hybridizations onto the arrays were carried out using 750 ng of each biotinylated cRNA. BeadChip arrays were scanned on the Illumina Bead Array Reader (Illumina Inc.). Data were analyzed using the Illumina BeadStudio Application software before undergoing comprehensive statistical analysis. Particular attention was given to the following quality control parameters: 0 ≤ G sat ≤ 1; Green 95 Percentile (GP95) for consistency between arrays (around 2000); GP5 background level in range of low 100 or below.</p><p id="p-12">Whole genome sequencing of the GK/Ox strain (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Atanur <em>et al.</em> 2013</a>) identified variants between GK and BN in 757 Illumina oligonucleotides (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033274/-/DC1/TableS2.xlsx">Table S2</a>), which were then excluded prior to array data analysis to avoid detection of spurious gene expression changes due to differences in binding between probes and oligonucleotides (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Alberts <em>et al.</em> 2007</a>). We verified absence of differential expression between GK, BN, and relevant congenics for several such genes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033274/-/DC1/FigureS1.tif">Figure S1</a>). We also withdrew probes that detected only background signal (<em>i.e.</em>, Illumina detection score &lt;0.5 in &gt;50% of samples). Microarray data processing was carried out using normexp background correction and quantile normalization (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Shi <em>et al.</em> 2010</a>).</p><p id="p-13">Microarray experiments were compliant with Minimum Information About a Microarray Experiment standards for reporting microarray experiments, and both protocol details and raw data have been deposited in ArrayExpress (<a href="http://www.ebi.ac.uk/arrayexpress/">http://www.ebi.ac.uk/arrayexpress/</a>) under the accession numbers E-MTAB-969 (F2 hybrids) and E-MTAB-1048 (BN, GK, and congenic strains).</p></div><div id="sec-5" class="subsection"><h3>Genetic mapping of expression QTL in the F2 (GK × BN) cross</h3><p id="p-14">eQTL analysis was performed using the R-qtl software package (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Broman <em>et al.</em> 2003</a>). We used genetic maps constructed in the cross with a combination of microsatellites and single nucleotide polymorphism markers (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Wilder <em>et al.</em> 2004</a>). Genome scans were carried out using the Haley–Knott regression method (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Haley and Knott 1992</a>). To account for effects of sex and lineage on gene expression, we used sex and cross direction as additive covariates in our models (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Solberg <em>et al.</em> 2004</a>). To obtain a genome-wide significance threshold for each transcript, we conducted a permutation test with 1000 permutations. QTL with a genome-scan adjusted <em>P</em>-value &lt;0.05 were retained as significant.</p><p id="p-15">We used a regression model with sex and cross direction as additive covariants:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3671/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3671/embed/mml-math-1.gif"/></noscript></span></span>with <em>β<sub>c</sub>c<sub>i</sub></em> denoting the effect of the cross, <em>β<sub>s</sub>s<sub>i</sub></em> denoting the effect of sex, and <em>β<sub>g</sub>g<sub>i</sub></em> denoting the effect of the genotype.</p></div><div id="sec-6" class="subsection"><h3>Statistical analysis of Illumina array data in the congenic strains</h3><p id="p-16">Differential gene expression in the congenics was assessed by comparing array data in each congenic strain to the relevant parental strain (GK for GK.BN congenics and BN for BN.GK congenics). A linear model was fitted using linear models for microarray data (LIMMA) accounting for effects of experimental batches. Genes were tested for statistical differential expression using a moderated <em>t</em>-statistic and a threshold of <em>P</em> &lt; 0.05.</p></div><div id="sec-7" class="subsection"><h3>Pathway analyses</h3><p id="p-17">To obtain functional categories that are enriched in the eQTL-controlled gene sets in the F2 cross, we used a hypergeometric test on gene ontology terms and KEGG pathways associated with these gene sets against the background of genes with detectable expression (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Falcon and Gentleman 2007</a>). For functional analysis of the congenic transcriptomes, we used gene set enrichment analysis (GSEA) (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Subramanian <em>et al.</em> 2005</a>). We detected pathways in the KEGG and reactome databases that are enriched among upregulated or downregulated genes (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Kanehisa and Goto 2000</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Matthews <em>et al.</em> 2009</a>). We used the standalone java version of GSEA 2.07 on the <em>t</em>-statistics of differential expression with default parameters and <em>n</em> = 1000 permutations.</p></div><div id="sec-8" class="subsection"><h3>Quantitative real-time PCR</h3><p id="p-18">Total RNA was treated with Turbo DNA-free DNase kit, for removal of genomic DNA (Ambion Inc.). First-strand cDNA synthesis was performed using Superscript III First-strand Synthesis Supermix for quantitative real-time PCR (Invitrogen, Paisley, UK). Assays were performed on a Rotor-Gene 3000 system (Corbett Research, Milton, UK) using the QuantiTect SYBR Green PCR kit (Qiagen). Analysis was performed using the standard curve method (Rotor-Gene Software 5.0.47; Corbett Research). This software allowed further verification that a single PCR product signal was quantified. Gene expression was normalized against the expression of either actin or acidic ribosomal phosphoprotein P0 (36B4). Experiments were performed in duplicate with samples prepared from six animals per group. Statistical significance was determined by the two-tailed independent sample <em>t</em>-test or univariate ANOVA, when testing more than two groups. Oligonucleotides designed to test gene expression are given in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033274/-/DC1/TableS3.xlsx">Table S3</a>.</p></div><div id="sec-9" class="subsection"><h3>Data availability</h3><p id="p-19">The authors state that all data necessary for confirming the conclusions presented in the article are represented fully within the article.</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>Differential transcription regulation in adipose tissue in GK and BN strains</h3><p id="p-20">We initially surveyed genome-wide transcriptional patterns in adipose tissue of male GK and BN rats (<em>n</em> = 6 per strain). We identified a total of 1221 genes showing evidence of significant differential expression [false discovery rate (FDR) &lt; 0.05] with an equivalent number of genes upregulated (514) and downregulated (707) in the GK (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033274/-/DC1/TableS4.xlsx">Table S4</a>). The proportion of genes differentially expressed between strains was similar across chromosomes (between 3.1% of genes on chromosome 15 and 6.3% of genes on chromosome 10).</p></div><div id="sec-12" class="subsection"><h3>Transcriptional control by genetic polymorphism blocks in congenic strains</h3><p id="p-21">To establish the effects of blocks of linked genetic polymorphisms on gene transcription, we repeated fat transcriptome profiling in a series of reciprocal BN.GK and GK.BN inbred congenic strains (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033274/-/DC1/TableS4.xlsx">Table S4</a>) using the same Illumina platform. The targeted genomic blocks in congenics collectively covered up to 950 Mb (35%) of the rat genome length and contained &gt;9900 protein coding genes (33.6% of the rat genes). BN.GK congenic strains each contain contiguous GK alleles within genomic blocks ranging in length from 1 to 183 Mb from rat chromosomes 1 (10 congenic lines), 2 (one congenic line), 4 (one congenic line), 5 (two congenic lines), 7 (two congenic lines), 8 (one congenic line), and 10 (one congenic line), independently introgressed onto the genetic background of the BN strain. The reciprocal GK.BN strains contained BN genomic blocks of rat chromosomes 7 (one congenic line), 8 (one congenic line), and 10 (one congenic line) transferred onto a GK genetic background. Thus, within a given congenic line, any difference in expression of a gene from the control line (if it is due to genetic differences) must be caused by polymorphisms within that congenic’s variable block.</p><p id="p-22">In pairwise comparisons between congenics and relevant GK or BN control, using six male rats per strain, we identified a total of 4302 differentially expressed genes at FDR &lt; 0.05 per comparison, ranging from 73 (BN.GK7d) to &gt;1000 (BN.GK1b, 1d, 1h, 1p, and 1u) (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Gene density in congenic intervals had no apparent impact on the number of differentially expressed genes between congenics and controls. For example, congenics BN.GK1b, 1d, 1p, and 1u have the highest numbers of differentially expressed genes yet carry GK genomic blocks containing few genes (24 in BN.GK1d) or many genes (1407 in BN.GK1p), suggesting that transcriptional regulation footprint is related to the function rather than number of genes. A relatively small proportion (7.7% on average) of differentially expressed genes wase localized within the GK congenic blocks themselves, and may correspond to <em>cis</em>-regulated gene expression (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033274/-/DC1/TableS4.xlsx">Table S4</a>). The remainder, which mapped outside the congenic region, unambiguously involve direct or indirect transcription regulatory mechanisms mediated <em>in trans</em>, as illustrated in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033274/-/DC1/FigureS2.tif">Figure S2</a>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469502/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469502/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Overview of fat gene transcription profiling in congenic strains of the GK rat</span><div class="sb-div caption-clear"></div></div></div><p id="p-24">To test the consistency of gene transcription patterns in congenic and control strains, we analyzed the set of 692 genes found significantly differentially expressed between congenic rats and their relevant BN or GK controls, and between GK and BN rats. Of these, 469 (68%) showed consistent direction of expression changes in the two comparisons (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> and <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). Remarkably, when only genes mapped within congenic intervals were considered, we found almost complete concordance in directions (159 of 165; 96%) in the two comparisons (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>), suggesting conservation of <em>cis</em>-mediated control of gene transcription but a lack of conservation of <em>trans</em>-mediated control. The power to detect gene expression changes was independent of whether a gene was inside a congenic block.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/11/3671/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Patterns of gene transcription regulation in white adipose tissue by GK/BN polymorphisms in GK and BN rats and in congenic strains. Conserved gene expression patterns between BN.GK congenic rats and controls and between GK and BN rats are shown for all differentially expressed genes (A) and those localized in congenic intervals (B). Conservation of gene expression changes in congenics targeting overlapping genomic regions is illustrated by plotting expression ratio in strains BN.GK1b and BN.GK1f (C) and in reciprocal strains BN.GK8b and GK.BN8a (D). Full details of differentially expressed genes are given in Table S4." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-17889456" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Patterns of gene transcription regulation in white adipose tissue by GK/BN polymorphisms in GK and BN rats and in congenic strains. Conserved gene expression patterns between BN.GK congenic rats and controls and between GK and BN rats are shown for all differentially expressed genes (A) and those localized in congenic intervals (B). Conservation of gene expression changes in congenics targeting overlapping genomic regions is illustrated by plotting expression ratio in strains BN.GK1b and BN.GK1f (C) and in reciprocal strains BN.GK8b and GK.BN8a (D). Full details of differentially expressed genes are given in Table S4.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/11/3671/F2.medium.gif" width="440" height="338"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/11/3671/F2.medium.gif" width="440" height="338"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/11/3671/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/11/3671/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469656" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-25" class="first-child">Patterns of gene transcription regulation in white adipose tissue by GK/BN polymorphisms in GK and BN rats and in congenic strains. Conserved gene expression patterns between BN.GK congenic rats and controls and between GK and BN rats are shown for all differentially expressed genes (A) and those localized in congenic intervals (B). Conservation of gene expression changes in congenics targeting overlapping genomic regions is illustrated by plotting expression ratio in strains BN.GK1b and BN.GK1f (C) and in reciprocal strains BN.GK8b and GK.BN8a (D). Full details of differentially expressed genes are given in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033274/-/DC1/TableS4.xlsx">Table S4</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">To assess robustness of differential gene expression in congenic rats, we compared transcriptome data in congenic strains that contained overlapping genomic regions on chromosome 1. Expression patterns of genes mapped to shared GK genomic blocks in different congenics showed near complete consistency, as illustrated in <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2C</a> for BN.GK1b and BN.GK1f. Data from reciprocal (BN.GK and GK.BN) congenics, which target the same &gt;40 Mb exchanged genomic segments on chromosomes 7 (BN.GK7a, GK.BN7a, and GK.BN10_7a), 8 (BN.GK8b and GK.BN8a), and 10 (BN.GK10a and GK.BN10_7a) (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033274/-/DC1/TableS1.pdf">Table S1</a>), also supported consistent (but reversed, because of the reciprocity of the congenics) gene expression patterns. They provide strong evidence of robust transcriptional effects of BN and GK alleles at these loci when expressed in contexts of GK and BN genome backgrounds, respectively. Thus, 24 of the 26 differentially expressed genes localized in the shared 40 Mb exchanged region in BN.GK8b and GK.BN8a showed opposite expression trends, demonstrating conserved allelic effects on gene transcription (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2D</a>).</p><p id="p-27">These results were further corroborated in the double congenic GK.BN10_7a, which contains two BN genomic blocks of chromosomes 7 and 10 introgressed onto a GK background. It combines the same regions of chromosome 7 as BN.GK7a and GK.BN7a, and the same region of chromosome 10 as BN.GK10a (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033274/-/DC1/TableS1.pdf">Table S1</a>). Gene transcription changes in GK.BN10_7a were remarkably consistent with BN.GK7a and GK.BN7a data for genes localized in chromosome 7 (<em>e.g.</em>, <em>Pim3</em>, <em>Serhl2</em>, <em>Naprt1</em>, and <em>Rbm9</em>) and BN.GK10a data for genes mapped to chromosome 10 (<em>e.g.</em>, <em>Cd300lg</em>, <em>Cpb</em>, <em>Med13</em>, and <em>Cbx</em>) (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033274/-/DC1/TableS4.xlsx">Table S4</a>). To a lesser extent, consistent transcriptional regulation in these congenics was also observed for genes mapped outside the exchanged genomic interval, demonstrating conserved <em>trans</em>-mediated gene expression patterns.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/11/3671/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Patterns of gene transcription regulation in congenic strains. Conserved allelic effects on genome-wide gene expression regulation in adipose tissue were tested in BN.GK7a and BN.GK10a congenics and in reciprocal strains GK.BN7a and GK.BN10_7a targeting largely overlapping genomic regions. Genomic positions of genes significantly differentially expressed (P &lt; 0.05) between BN.GK congenic and BN controls and between GK.BN congenic and GK controls are plotted along the x-axes and gene expression ratios are shown along the y-axes. Chromosomal location of genes is color coded. Full details of differentially expressed genes, including genes that are localized in the congenic intervals and may be regulated in cis, are given in Table S4." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-17889456" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Patterns of gene transcription regulation in congenic strains. Conserved allelic effects on genome-wide gene expression regulation in adipose tissue were tested in BN.GK7a and BN.GK10a congenics and in reciprocal strains GK.BN7a and GK.BN10_7a targeting largely overlapping genomic regions. Genomic positions of genes significantly differentially expressed (&lt;em&gt;P&lt;/em&gt; &lt; 0.05) between BN.GK congenic and BN controls and between GK.BN congenic and GK controls are plotted along the &lt;em&gt;x&lt;/em&gt;-axes and gene expression ratios are shown along the &lt;em&gt;y&lt;/em&gt;-axes. Chromosomal location of genes is color coded. Full details of differentially expressed genes, including genes that are localized in the congenic intervals and may be regulated in &lt;em&gt;cis&lt;/em&gt;, are given in Table S4.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/11/3671/F3.medium.gif" width="374" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/11/3671/F3.medium.gif" width="374" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/11/3671/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/11/3671/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469555" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-28" class="first-child">Patterns of gene transcription regulation in congenic strains. Conserved allelic effects on genome-wide gene expression regulation in adipose tissue were tested in BN.GK7a and BN.GK10a congenics and in reciprocal strains GK.BN7a and GK.BN10_7a targeting largely overlapping genomic regions. Genomic positions of genes significantly differentially expressed (<em>P</em> &lt; 0.05) between BN.GK congenic and BN controls and between GK.BN congenic and GK controls are plotted along the <em>x</em>-axes and gene expression ratios are shown along the <em>y</em>-axes. Chromosomal location of genes is color coded. Full details of differentially expressed genes, including genes that are localized in the congenic intervals and may be regulated in <em>cis</em>, are given in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033274/-/DC1/TableS4.xlsx">Table S4</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">Importantly, the majority of those genes outside of the congenic block that are differentially expressed exhibit changes very close to twofold up or down (<em>i.e.</em>, +/−1 on the base-2 log scale in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033274/-/DC1/TableS4.xlsx">Table S4</a>), allowing for the statistical uncertainties in the estimated expression levels. In contrast, differentially expressed genes within the congenic block showed unconstrained changes. This suggests that the <em>trans</em> effects are largely confined to transitions between monoallelic and biallelic expression, and thus are both qualitatively and quantitatively distinct from the <em>cis</em> effects.</p><p id="p-30">The congenic series thus provide a powerful experimental system to discriminate <em>cis</em>- and <em>trans</em>-mediated mechanisms of transcription of individual genes driven by polymorphisms present in specific genomic blocks.</p></div><div id="sec-13" class="subsection"><h3>Genomic blocks in congenic strains regulate specific and redundant biological pathways</h3><p id="p-31">To evaluate the biological consequences of coordinated gene transcription changes in congenics, we carried out GSEA. We identified 87 KEGG and 100 reactome pathways altered in at least one congenic strain (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033274/-/DC1/TableS5.xlsx">Table S5</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033274/-/DC1/TableS6.xlsx">Table S6</a>). Analysis of KEGG pathways showed that the two reciprocal congenic series containing GK genomic blocks on a BN background (BN.GK) and BN genomic blocks on a GK background (GK.BN) can be separated, even though several pathways are similarly affected in these two congenic series (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). Concordant KEGG pathway regulation was observed in congenics carrying overlapping GK genomic regions (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033274/-/DC1/TableS5.xlsx">Table S5</a>). For example, significant upregulation of the citrate cycle pathway, as defined by positive enrichment scores (ES) given by GSEA, was replicated in BN.GK1p (ES = 1.97; FDR <em>q</em>-value = 0.02) and 1u (ES = 2.14; FDR <em>q</em>-value = 3.2 × 10<sup>−3</sup>) (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4B</a>), and involved a common set of 12 genes (<em>Acly</em>, <em>Aco1</em>, <em>Aco2</em>, <em>Dlat</em>, <em>Dld</em>, <em>Dlst</em>, <em>Idh1</em>, <em>Idh3b</em>, <em>Pc</em>, <em>Sdha</em>, <em>Sdhc</em>, and <em>Sdhd</em>) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033274/-/DC1/TableS7.xlsx">Table S7</a>) that map outside the congenic intervals and contribute to pathway enrichment among the 20 genes tested in the pathway, suggesting a consistent effect of genetic polymorphisms localized in the shared GK genomic block on distant genes.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/11/3671/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Global effects of GK/BN polymorphisms in congenic strains on biological pathways in white adipose tissue. Hierarchical clustering of differentially regulated pathways in congenic strains illustrate shared and strain specific biological mechanisms regulated by polymorphisms in genomic blocks (A). Pathways downregulated or upregulated in congenics are shown in blue and red cells, respectively. Statistically significant changes are indicated by &#x2212; and +. Enrichment plots of genes contributing to significant alterations of biological pathways in congenics (B&#x2013;E) illustrate conserved patterns of pathway regulation in congenics BN.GK1p and 1u (B) and contrasting patterns in reciprocal congenic strains (BN.GK and GK.BN) targeting overlapping regions of chromosomes 7 (D) and 8 (C, E). Output images of GSEA show the number of genes (vertical lines) upregulated (red bars) or downregulated (blue bars) contributing to overall stimulation or inhibition of the pathways (green curves). Full lists of altered KEGG pathways in congenics are shown in Table S5 and details of differentially expressed genes between congenics and controls are given in Table S4. Details of genes contributing to enrichment of pathways in B&#x2013;E are shown in Table S7." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-17889456" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Global effects of GK/BN polymorphisms in congenic strains on biological pathways in white adipose tissue. Hierarchical clustering of differentially regulated pathways in congenic strains illustrate shared and strain specific biological mechanisms regulated by polymorphisms in genomic blocks (A). Pathways downregulated or upregulated in congenics are shown in blue and red cells, respectively. Statistically significant changes are indicated by &#x2212; and +. Enrichment plots of genes contributing to significant alterations of biological pathways in congenics (B&#x2013;E) illustrate conserved patterns of pathway regulation in congenics BN.GK1p and 1u (B) and contrasting patterns in reciprocal congenic strains (BN.GK and GK.BN) targeting overlapping regions of chromosomes 7 (D) and 8 (C, E). Output images of GSEA show the number of genes (vertical lines) upregulated (red bars) or downregulated (blue bars) contributing to overall stimulation or inhibition of the pathways (green curves). Full lists of altered KEGG pathways in congenics are shown in Table S5 and details of differentially expressed genes between congenics and controls are given in Table S4. Details of genes contributing to enrichment of pathways in B&#x2013;E are shown in Table S7.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/11/3671/F4.medium.gif" width="440" height="342"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/11/3671/F4.medium.gif" width="440" height="342"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/11/3671/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/11/3671/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469623" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-32" class="first-child">Global effects of GK/BN polymorphisms in congenic strains on biological pathways in white adipose tissue. Hierarchical clustering of differentially regulated pathways in congenic strains illustrate shared and strain specific biological mechanisms regulated by polymorphisms in genomic blocks (A). Pathways downregulated or upregulated in congenics are shown in blue and red cells, respectively. Statistically significant changes are indicated by − and +. Enrichment plots of genes contributing to significant alterations of biological pathways in congenics (B–E) illustrate conserved patterns of pathway regulation in congenics BN.GK1p and 1u (B) and contrasting patterns in reciprocal congenic strains (BN.GK and GK.BN) targeting overlapping regions of chromosomes 7 (D) and 8 (C, E). Output images of GSEA show the number of genes (vertical lines) upregulated (red bars) or downregulated (blue bars) contributing to overall stimulation or inhibition of the pathways (green curves). Full lists of altered KEGG pathways in congenics are shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033274/-/DC1/TableS5.xlsx">Table S5</a> and details of differentially expressed genes between congenics and controls are given in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033274/-/DC1/TableS4.xlsx">Table S4</a>. Details of genes contributing to enrichment of pathways in B–E are shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033274/-/DC1/TableS7.xlsx">Table S7</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">As observed at the gene level, contrasting patterns of pathway regulation were found in reciprocal BN.GK and GK.BN congenics, indicating coherent effects of GK and BN alleles in the targeted region, when transferred onto BN and GK genomic backgrounds, respectively. For example, pathways downregulated in GK.BN8a (arachidonic acid metabolism, natural killer cell-mediated cytotoxicity) and GK.BN7a (natural killer cell-mediated cytotoxicity) were upregulated in BN.GK8b and BN.GK7a (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4, C–E</a>). As noted above for the regulation of the citrate cycle in BN.GK1p and 1u, a core set of 6–7 genes contributed to pathway enrichment in reciprocal congenics, in addition to 4–15 strain-specific contributing genes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033274/-/DC1/TableS7.xlsx">Table S7</a>).</p><p id="p-34">Results from pathway analysis also provide evidence of alteration of the same biological mechanisms in BN.GK strains containing different GK genomic regions. For example, the ribosome pathway was systematically upregulated in BN.GK2c (ES = 2.38; FDR <em>q</em>-value &lt; 0.001), 5a (ES = 2.18; FDR <em>q</em>-value &lt; 0.001), 5c (ES = 2.53; FDR <em>q</em>-value &lt; 0.001), 7a (ES = 2.45; FDR <em>q</em>-value &lt; 0.001), and 10a (ES = 2.37; FDR <em>q</em>-value &lt; 0.001) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033274/-/DC1/TableS5.xlsx">Table S5</a>). Even though these strains carry different collections of GK variants (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>), we identified a common set of 25 genes that contributed to pathway enrichment in all five congenics (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033274/-/DC1/TableS8.xlsx">Table S8</a>), demonstrating <em>trans</em> effects of different regions converging on a common biological pathway. Along the same lines, consistent downregulation of natural killer cell-mediated cytotoxicity in GK.BN7a and GK.BN8a (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4, D and E</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033274/-/DC1/TableS7.xlsx">Table S7</a>), which target different chromosomes (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033274/-/DC1/TableS1.pdf">Table S1</a>), involved a common set of 15 genes (<em>Bid</em>, <em>Cd48</em>, <em>Chp2</em>, <em>Fcgr3a</em>, <em>Fyn</em>, <em>Icam2</em>, <em>Klrd1</em>, <em>Lat</em>, <em>Lck</em>, <em>Map2k1</em>, <em>Nras</em>, <em>Plcg1</em>, <em>Prf1</em>, <em>Rac1</em>, and <em>Tnfsf10</em>) among the 40 genes of the pathway (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033274/-/DC1/TableS7.xlsx">Table S7</a>). Interestingly, significant upregulation of this pathway in both BN.GK7a and BN.GK8b (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4, D and E</a>) confirmed the contribution of four genes (<em>Bid</em>, <em>Lat</em>, <em>Lck</em>, and <em>Prf1</em>), but involved a greater number of genes specific to these two BN.GK strains (<em>Hcst</em>, <em>Plcg2</em>, <em>Ppp3ca</em>, <em>Ppp3r1</em>, <em>Ptpn6</em>, <em>Rac2</em>, and <em>Vav1</em>) when compared to GK.BN strains (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033274/-/DC1/TableS7.xlsx">Table S7</a>), suggesting interactions between alleles in the background and in the congenic intervals in the control of this pathway.</p><p id="p-35">These results underline conserved and strain-specific functional consequences of genetic polymorphisms located in specific genomic blocks. They also demonstrate that distinct genomic regions have functionally redundant roles on biological pathways, which may be regulated by different series of genetic polymorphisms or involve epistasis.</p></div><div id="sec-14" class="subsection"><h3>Genetic architecture of fat gene transcription regulation in (GK × BN) F2 hybrids</h3><p id="p-36">To validate architectural features of gene transcription identified in the GK rat and in congenic strains, we mapped eQTLs in white adipose tissue of rats from a GK × BN F2 cross. Genotypes at over 255 framework markers typed in the cross were used to impute allele probabilities and construct a map of 898 marker positions (2.5 cM spacing between markers). Following withdrawal of oligonucleotides containing DNA variants, markers were tested for linkage to 15,822 detectable Illumina array signals in each of the 123 F2 rats. We identified a total of 2735 eQTLs at FDR &lt; 0.05, including 2483 (90.8%) linked to transcripts corresponding to genes unambiguously localized in the rat genome assembly (RGSC3.4, Ensembl release 69) (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033274/-/DC1/TableS9.pdf">Table S9</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/11/3671/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Overview of eQTL features in adipose tissue of GK &#xD7; BN F2 hybrids. Genetic positions of statistically significant eQTLs are plotted against the LOD scores (A). Local and distant eQTLs are illustrated by plotting genetic positions of statistically significant eQTLs and the genomic position of the linked transcripts (B). Data from pairs of eQTL and linked transcripts mapped to the same chromosomes were used to determine the distribution of genomic distances between transcripts and genetic markers showing the strongest evidence of statistically significant linkage in the cross (C) and relationships with the significance of genetic linkages (D). Data from pairs of transcripts and eQTLs mapped to different chromosomes were selected to illustrate distant (trans) effects of genetic loci on gene transcription and trans-acting eQTL hotspots (smaller circles) (E). Chromosomes are color coded on the circle. The colors of the lines illustrate the effects of eQTLs mapped to the same chromosomes on the expression of distant genes. Details of eQTLs are given in Table S4. LOD, Logarithm of the odds." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-17889456" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Overview of eQTL features in adipose tissue of GK &#xD7; BN F2 hybrids. Genetic positions of statistically significant eQTLs are plotted against the LOD scores (A). Local and distant eQTLs are illustrated by plotting genetic positions of statistically significant eQTLs and the genomic position of the linked transcripts (B). Data from pairs of eQTL and linked transcripts mapped to the same chromosomes were used to determine the distribution of genomic distances between transcripts and genetic markers showing the strongest evidence of statistically significant linkage in the cross (C) and relationships with the significance of genetic linkages (D). Data from pairs of transcripts and eQTLs mapped to different chromosomes were selected to illustrate distant (&lt;em&gt;trans&lt;/em&gt;) effects of genetic loci on gene transcription and &lt;em&gt;trans&lt;/em&gt;-acting eQTL hotspots (smaller circles) (E). Chromosomes are color coded on the circle. The colors of the lines illustrate the effects of eQTLs mapped to the same chromosomes on the expression of distant genes. Details of eQTLs are given in Table S4. LOD, Logarithm of the odds.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/11/3671/F5.medium.gif" width="440" height="295"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/11/3671/F5.medium.gif" width="440" height="295"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/11/3671/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/11/3671/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469546" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-37" class="first-child">Overview of eQTL features in adipose tissue of GK × BN F2 hybrids. Genetic positions of statistically significant eQTLs are plotted against the LOD scores (A). Local and distant eQTLs are illustrated by plotting genetic positions of statistically significant eQTLs and the genomic position of the linked transcripts (B). Data from pairs of eQTL and linked transcripts mapped to the same chromosomes were used to determine the distribution of genomic distances between transcripts and genetic markers showing the strongest evidence of statistically significant linkage in the cross (C) and relationships with the significance of genetic linkages (D). Data from pairs of transcripts and eQTLs mapped to different chromosomes were selected to illustrate distant (<em>trans</em>) effects of genetic loci on gene transcription and <em>trans</em>-acting eQTL hotspots (smaller circles) (E). Chromosomes are color coded on the circle. The colors of the lines illustrate the effects of eQTLs mapped to the same chromosomes on the expression of distant genes. Details of eQTLs are given in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033274/-/DC1/TableS4.xlsx">Table S4</a>. LOD, Logarithm of the odds.</p><div class="sb-div caption-clear"></div></div></div><p id="p-38">Results from transcriptome analyses in the F2 showed classical features of eQTL architecture, including local (<em>cis</em>) and distant (<em>trans</em>) genetic control and eQTL hotspots. A total of 1500 eQTLs were linked to transcripts mapped to different chromosomes, which unequivocally indicated the involvement of <em>trans</em>-mediated regulation of gene transcription (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033274/-/DC1/TableS4.xlsx">Table S4</a>). A high proportion of the remaining 1159 eQTLs located to the same chromosomes as the linked transcripts were localized within 5 Mb (<em>n</em> = 698; 60%), 10 Mb (<em>n</em> = 910; 79%), or 20 Mb (<em>n</em> = 1033; 89%) of the linked transcripts, strongly suggesting <em>cis</em>-mediated transcription regulation (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5C</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033274/-/DC1/TableS4.xlsx">Table S4</a>). The most statistically significant of these eQTLs were localized in close vicinity to the linked transcripts (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5D</a>). eQTLs were evenly distributed across the genome, with an average of 4.8% of eQTLs per chromosome (1 Mb average spacing between eQTLs), with the exception of chromosome 1 which showed a large excess (27.1%) of <em>trans</em>-regulating eQTLs (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5B</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033274/-/DC1/TableS9.pdf">Table S9</a>), even though it only covers ∼10% of the rat genome length and only contains 13% of rat genes. This eQTL excess can be explained by a hotspot of 319 eQTLs in a short region of chromosome 1 (49.4–59.4 cM, 90.3–96.9 Mb) (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5E</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033274/-/DC1/TableS4.xlsx">Table S4</a>). Other eQTL hotspots were detected on chromosomes 5 (57.5–62.5 cM, 138.3–146.4 Mb), 7 (70.0–72.5 cM, 113.0–117.0 Mb), and 17 (6.1–8.6 cM, 11.4–14.6 Mb). GK alleles at the eQTL hotspots on chromosomes 1, 7, and 17 were associated with downregulated expression of a high proportion of distant genes (64–73%), suggesting that eQTL clustering may be biologically relevant and reflect genuine mechanisms of gene transcription control driven by the eQTLs.</p></div><div id="sec-15" class="subsection"><h3>eQTLs in outbred F2 hybrids partially replicate transcriptome architecture in inbred strains</h3><p id="p-39">To assess the relevance of eQTLs to genome-wide transcription regulation in inbred strains, we initially compared gene differential expression between GK and BN with transcriptional effects of corresponding eQTLs in the cross. Only 50% of differentially expressed genes between these strains (614 of 1221) corresponded to eQTLs. Even though this only represents a small proportion (22%) of eQTLs, the effect of GK alleles on the direction of gene expression changes was consistent for the majority (86%; <em>n</em> = 531) of these (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>). In addition, allelic effects on gene transcription in the F2 rats and inbred strains were consistent for all 362 <em>cis</em>-regulated eQTLs (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6B</a>) but for only 58% of <em>trans</em>-mediated eQTLs (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6C</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/11/3671/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Correlation analysis of the effects of GK alleles on genome-wide gene expression in F2 hybrids and in inbred strains. The effects of GK alleles at statistically significant eQTLs (P-adjusted &lt; 0.05) in the GK &#xD7; BN F2 cross are plotted against expression ratio of corresponding genes significantly differential expressed (P-adjusted &lt; 0.05) between GK and BN strains for all eQTLs (A) and cis- (B) and trans- (C) mediated eQTLs. The effects of GK alleles at eQTLs on gene expression regulation were compared in the F2 cross and in congenic strains for genome-wide gene expression data (D) and for genes mapped to congenic intervals (E). Details of differentially expressed genes between inbred strains and gene expression at eQTLs are given in Table S4." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-17889456" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Correlation analysis of the effects of GK alleles on genome-wide gene expression in F2 hybrids and in inbred strains. The effects of GK alleles at statistically significant eQTLs (&lt;em&gt;P&lt;/em&gt;-adjusted &lt; 0.05) in the GK &#xD7; BN F2 cross are plotted against expression ratio of corresponding genes significantly differential expressed (&lt;em&gt;P&lt;/em&gt;-adjusted &lt; 0.05) between GK and BN strains for all eQTLs (A) and &lt;em&gt;cis&lt;/em&gt;- (B) and &lt;em&gt;trans&lt;/em&gt;- (C) mediated eQTLs. The effects of GK alleles at eQTLs on gene expression regulation were compared in the F2 cross and in congenic strains for genome-wide gene expression data (D) and for genes mapped to congenic intervals (E). Details of differentially expressed genes between inbred strains and gene expression at eQTLs are given in Table S4.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/11/3671/F6.medium.gif" width="440" height="371"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/6/11/3671/F6.medium.gif" width="440" height="371"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/11/3671/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/11/3671/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469692" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-40" class="first-child">Correlation analysis of the effects of GK alleles on genome-wide gene expression in F2 hybrids and in inbred strains. The effects of GK alleles at statistically significant eQTLs (<em>P</em>-adjusted &lt; 0.05) in the GK × BN F2 cross are plotted against expression ratio of corresponding genes significantly differential expressed (<em>P</em>-adjusted &lt; 0.05) between GK and BN strains for all eQTLs (A) and <em>cis</em>- (B) and <em>trans</em>- (C) mediated eQTLs. The effects of GK alleles at eQTLs on gene expression regulation were compared in the F2 cross and in congenic strains for genome-wide gene expression data (D) and for genes mapped to congenic intervals (E). Details of differentially expressed genes between inbred strains and gene expression at eQTLs are given in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033274/-/DC1/TableS4.xlsx">Table S4</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-41">We then compared genome-wide gene expression data in congenics and in F2 hybrids. The existence of several eQTLs was verified by transcription analysis of the linked genes in congenics and controls (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033274/-/DC1/FigureS1.tif">Figure S1</a>). We identified 1188 genes differentially expressed between congenics and controls that were linked to eQTLs, including 708 (60%) with concordant effects of GK alleles on the direction of gene expression changes in F2 and congenics (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6D</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033274/-/DC1/TableS4.xlsx">Table S4</a>). The resulting 26% replication rate in the F2 cross of differential gene expression in congenics is consistent with gene density in the congenic intervals (33.6% of all rat genes). When only eQTLs for the 358 differentially expressed genes localized in congenic intervals were considered, allelic effects on the direction of gene expression change in F2 and in congenics were consistent in the vast majority of cases (<em>n</em> = 335; 94%) (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6E</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033274/-/DC1/TableS4.xlsx">Table S4</a>). Of note, both <em>cis</em>- and <em>trans</em>-acting eQTLs were equally validated in congenics when the corresponding genes were localized within the genomic blocks of the congenic strains (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033274/-/DC1/FigureS3.tif">Figure S3</a>).</p><p id="p-42">To investigate <em>trans</em>-regulated eQTLs in more detail, we tested if the F2 eQTL hotspot detected in chromosome 1 was replicated in the congenic strain BN.GK1p, which contains GK alleles in this region. Over 36% (98 of 270) of <em>trans</em>-regulated genes in this eQTL hotspot were significantly differentially expressed between BN.GK1p and BN (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033274/-/DC1/TableS10.xlsx">Table S10</a>). Remarkably, for all 98 genes, the effect of GK alleles on the direction of gene expression changes was consistent in F2 hybrids and congenics. Congenic strains BN.GK1u and BN.GK1v, which share large GK haplotypes with BN.GK1p but are distal to this eQTL hotspot, showed more limited differential expression of eQTL genes (14 and 8%, respectively). These data confirm the existence of this <em>trans</em>-eQTL hotspot driven by regulatory elements in the GK region specific to BN.GK1p.</p></div><div id="sec-16" class="subsection"><h3>Congenic transcriptome pathways do not predict F2 eQTL biological function</h3><p id="p-43">To test replication of biological features of fat transcriptomes identified in GK congenics, we repeated pathway enrichment analysis in the F2. Significantly affected pathways in the F2 included diabetes mellitus, digestion and absorption of fat, immunological processes (phagosome, allograft rejection, antigen processing and presentation, autoimmune diseases, cell adhesion molecules, natural killer cell-mediated cytotoxicity, and hematopoietic cell lineage), and metabolism of fatty acids and 11 amino acids (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). Given the importance of the GK rat as a model for diabetes, this suggests that many GK/BN alleles across the genome, and that segregate in the cross, contribute to the metabolic and inflammatory mechanisms described in human diabetes.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469665/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469665/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">KEGG pathways underlying eQTL biological effects in the white adipose tissue transcriptome in the GK × BN F2 cross</span><div class="sb-div caption-clear"></div></div></div><p id="p-45">The vast majority of pathways found altered in congenics, which may be directly or indirectly caused by genetic polymorphisms present in the congenic intervals, were not validated in F2 hybrids (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033274/-/DC1/TableS5.xlsx">Table S5</a>), suggesting that the biological effects of genetic polymorphisms isolated in contiguous genomic blocks in inbred congenics may be counter-regulated by gene × gene interactions, when alleles across the genome segregate in outbred individuals.</p></div></div><div class="section discussion" id="sec-17"><h2 class="">Discussion</h2><p id="p-46">We report the comprehensive genetic analysis of genome-wide fat gene transcription in a series of inbred congenic strains followed by replication in genetically heterogeneous F2 hybrids derived from the same combination of rat strains. We demonstrate conserved and discordant transcript regulation when genetic polymorphisms across the entire genome are fixed homozygous or segregate in a cross, and when they are dissected out in genomic blocks in congenic strains. Comparative analyses of transcriptomes in these systems provide an original experimental framework to assess the extent and impact of epistasis on the regulation of gene expression and biological pathways.</p><p id="p-47">Transcriptome regulation in GK × BN F2 hybrids exhibits typical architectural features of eQTLs described in genetically heterogeneous populations in humans (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Dixon <em>et al.</em> 2007</a>; <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Emilsson <em>et al.</em> 2008</a>; <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Grundberg <em>et al.</em> 2012</a>; <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">GTExConsortium 2015</a>) and in mapping panels in mice (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Schadt <em>et al.</em> 2003</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bystrykh <em>et al.</em> 2005</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Chesler <em>et al.</em> 2005</a>) and rats (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Hubner <em>et al.</em> 2005</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Petretto <em>et al.</em> 2006</a>), including <em>cis</em>- and <em>trans</em>-mediated eQTLs and eQTL clusters controlling the transcription of large numbers of distant genes (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Schadt <em>et al.</em> 2003</a>; <a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">Emilsson <em>et al.</em> 2008</a>). The definition of <em>cis</em>-eQTLs and eQTL hotspots in these genetic contexts is based on arbitrary estimates of physical distances separating genetic markers and linked transcripts (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Petretto <em>et al.</em> 2006</a>). Also, the existence of eQTL hotspots has been questioned (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Breitling <em>et al.</em> 2008</a>) due to the low effect size of <em>trans</em>-regulated eQTLs (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Zhu <em>et al.</em> 2008</a>). Assessment of these features in F2 cohorts is complicated by extensive linkage disequilibrium, preventing the high resolution mapping needed to separate closely linked eQTLs.</p><p id="p-48">In contrast, the genetic structure of congenic strains allows for a cruder measurement of gene differential expression caused by exchanging genetic polymorphisms present in well-defined genomic intervals in an otherwise uniform background, thereby unambiguously distinguishing between <em>cis</em>- and <em>trans</em>-mediated regulations (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Buchner and Nadeau 2015</a>). Differential expression of genes mapped within the genomic regions exchanged in congenics should therefore correspond to <em>cis</em>-regulated effects whereas differential expression of genes mapped outside these regions unambiguously indicates <em>trans</em>-mediated regulation. Perhaps surprisingly, we find that almost all <em>trans</em>-regulation in the congenics takes the form of twofold changes in expression, which we interpret as transitions between monoallelic and biallelic expression. This is reminiscent of expression changes observed in reciprocal crosses of mouse gene knockouts (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Chess 2012</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Mott <em>et al.</em> 2014</a>), and suggests a different mechanism for <em>trans</em> regulation compared to <em>cis</em>. This simple genetic architecture is not observed in the F2 cross, presumably because many <em>trans</em> loci regulate a given gene, smoothing out the individual effects that only become evident in the congenics. We suggest that <em>trans</em> effects in outbred populations, including humans, are accumulations of multiple additive and nonadditive (epistatic) combinations of switches. This hypothesis deserves further investigation; for example, by building a series of combined GK.BN congenics in which the fraction of the genome that is GK accumulates in a known manner. Similarly, analysis of F1 hybrids of congenics (to generate heterozygosity) would demonstrate if dominance effects mask the effects observed in pure congenics.</p><p id="p-49">Congenic strains targeting overlapping genomic regions showed remarkably consistent patterns of gene transcription, and we found predominantly <em>cis</em>-regulated eQTLs and an eQTL hotspot in congenics, despite the 8-yr time-lag required to produce the congenic rats after the F2 hybrids (<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Gauguier <em>et al.</em> 1996</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Wallace <em>et al.</em> 2004</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Wallis <em>et al.</em> 2004</a>). Although the congenic strains collectively target ∼30% of the rat genome, the total number of differentially expressed genes among all congenics was far greater than the number of eQTLs and that of differentially expressed genes between parental GK and BN strains (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>). Even though animals were thoroughly genotyped throughout the backcross breeding and inbreeding stages required for the production of congenics, residual allele contaminants and gene conversions may have remained undetected and may account for abundant gene differential expression in congenics. Concordant transcriptional regulation between congenics, F2, and parental GK and BN strains was proportional (25.9–38.4%) to the fraction of the overall gene density and genomic length of the introgressed regions in the congenic intervals, indicating that <em>trans</em>-mediated mechanisms of gene expression can be efficiently uncovered in congenics. Comparison of transcription patterns in all three experimental systems demonstrated consistent allelic effects on gene expression for a subset of 232 genes (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7</a>). Transcription of a high proportion of these genes (83.6%) was regulated <em>in cis</em> by eQTLs in the cross, as defined by transcripts mapped within 10 Mb of markers showing the strongest evidence of linkage. However, 39.3% of these presumably <em>cis</em>-eQTL genes mapped outside congenic intervals (<em>i.e.</em>, F2 data suggested <em>cis</em>-regulation but congenics demonstrated <em>trans</em>-regulation), indicating additional contribution of <em>trans</em>-mediated control in the expression of these genes. eQTL reports in humans, suggesting that <em>trans</em> genetic variants can regulate multiple transcripts (<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Grundberg <em>et al.</em> 2012</a>), support this observation.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/11/3671/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Summary of transcriptome results among inbred strains and eQTL data in GK &#xD7; BN F2 hybrids. The number of statistically significant eQTLs (green circle) and differentially expressed genes between GK and BN (red circle) and between congenics and controls (blue circle) are reported. Concordant data between the different experimental groups correspond to genes showing consistent allelic effects on the direction of expression changes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-17889456" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Summary of transcriptome results among inbred strains and eQTL data in GK &#xD7; BN F2 hybrids. The number of statistically significant eQTLs (green circle) and differentially expressed genes between GK and BN (red circle) and between congenics and controls (blue circle) are reported. Concordant data between the different experimental groups correspond to genes showing consistent allelic effects on the direction of expression changes.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/11/3671/F7.medium.gif" width="440" height="376"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/6/11/3671/F7.medium.gif" width="440" height="376"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/11/3671/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/11/3671/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469596" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-50" class="first-child">Summary of transcriptome results among inbred strains and eQTL data in GK × BN F2 hybrids. The number of statistically significant eQTLs (green circle) and differentially expressed genes between GK and BN (red circle) and between congenics and controls (blue circle) are reported. Concordant data between the different experimental groups correspond to genes showing consistent allelic effects on the direction of expression changes.</p><div class="sb-div caption-clear"></div></div></div><p id="p-51">We were able to validate only a fraction of <em>trans</em>-regulated eQTLs in congenics and conversely identified an important proportion of differentially expressed genes (83–89%) which were neither differentially expressed between GK and BN strains nor detected as eQTLs in the cross (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7</a>). Our findings suggest that genetic polymorphisms orchestrate gene expression regulation in different structures of gene × gene and gene × environment interactions when fixed homozygous in the genome or when segregating in an F2 cohort. However, due to statistical considerations of methods applied to eQTL mapping and gene differential expression analysis between groups of individuals, a larger F2 cohort would be required to test this hypothesis. Transcriptional changes in our congenic series are crude functional consequences of homozygous genetic variants from the donor strain present in each of the genomic blocks, interacting with specific collections of fixed alleles from the recipient strain in the genetic background. Variations in phenotypic features among congenics (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033274/-/DC1/FigureS4.tif">Figure S4</a>) may also explain incomplete concordance of transcriptome data in these experimental systems. Congenic data suggest that epistatic regulations and gene × environment interactions may have a much more prominent impact on gene transcription than expected by estimates derived from eQTL experiments performed in genetically heterogeneous cohorts.</p><p id="p-52">At pathway level, transcriptome data in the GK × BN F2 cross identified changes in inflammatory and metabolic mechanisms, which are relevant to diabetes pathogenesis and are caused by genome-wide segregation of genetic polymorphisms. In contrast, each congenic transcriptome provided biological signatures of a subset of linked genetic polymorphisms in a specific genomic interval, which can be only partly replicated in F2 hybrids. An important finding in the congenics was that distinct genetic polymorphisms in disjoint genomic blocks control the same biological pathways, suggesting functional redundancy of genetic control. This phenomenon, which was also reported in the budding yeast where about 15% of genes affect growth rate (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Sopko <em>et al.</em> 2006</a>), is proposed as a mechanism to ensure sustained maintenance of essential phenotypes. Our data also provided experimental support to the concept that mammalian syndromes can be caused by mutations at different loci in unrelated genes, which nevertheless share functional relationships with the disease phenotype (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Brunner and van Driel 2004</a>).</p><div id="sec-18" class="subsection"><h3>Conclusions</h3><p id="p-53">Results from our transcriptome analyses demonstrate the power of congenic series to dissect out <em>cis</em>- and <em>trans</em>-mediated mechanisms of gene expression, to attach biological functions to linked polymorphisms in genomic blocks, and to validate eQTL hotspots. Our data underline the importance of systems genetics (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Civelek and Lusis 2014</a>) to enhance knowledge of fundamental mechanisms, including functional epistasis and gene functional redundancy, which contribute to modulating the function of disease susceptibility genes and affect pathophysiological aspects of complex disorders.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-54">This work was supported by a Wellcome Trust Senior Fellowship in Basic Biomedical Science (057733) to D.G., a Wellcome Trust Core Award grant (075491/Z/04), and grants from the European Community’s Sixth and Seventh Framework Programs under grant agreements LSHG-CT-2006-037683 Functional genomic diagnostic tools for coronary artery disease (FGENTCARD) and HEALTH-F4-2010-241504 European large-scale functional genomics in the rat for translational research (EURATRANS), the Agence Nationale pour la Recherche (ANR-08-GENOPAT-030), and the Fondation pour la Recherche Médicale (FRM, INE20091217993). R.M. was supported by Wellcome Trust grants 083573/Z/07/Z and 090532/Z/09/Z. The funding bodies had no role in the design of the study and collection, analysis, and interpretation of data and in writing the manuscript.</p><p id="p-55">Author contributions: P.J.K. and D.G. conceived and designed the experiments. P.J.K., K.A., S.C.C., R.H.W., A.C.Y.Y., C.H., S.C., and M.-T.B. performed the experiments. P.J.K., G.W.O., S.P.W., M.-T.B., J.-B.C., R.M., and D.G. analyzed the data. R.M. and D.G. wrote the manuscript.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-4"><p id="p-56">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033274/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.033274/-/DC1</a>.</p></li><li class="fn" id="fn-5"><p id="p-57"><em>Communicating editor: D. W. Threadgill</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-07-08"><span class="received-label">Received </span>July 8, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-09-14"><span class="accepted-label">Accepted </span>September 14, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Kaisaki <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited. </p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3671.1" data-doi="10.1371/journal.pone.0000622"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alberts</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Terpstra</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Breitling</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nap</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Sequence polymorphisms cause many false cis eQTLs.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e622</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DAlberts%26rft.auinit1%253DR.%26rft.volume%253D2%26rft.issue%253D7%26rft.spage%253De622%26rft.epage%253De622%26rft.atitle%253DSequence%2Bpolymorphisms%2Bcause%2Bmany%2Bfalse%2Bcis%2BeQTLs.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000622%26rft_id%253Dinfo%253Apmid%252F17637838%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0000622&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17637838&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3671.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3671.2" data-doi="10.1007/s00125-006-0396-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Argoud</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilder</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McAteer</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bihoreau</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ouali</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genetic control of plasma lipid levels in a cross derived from normoglycaemic Brown Norway and spontaneously diabetic Goto-Kakizaki rats.</span> <abbr class="cit-jnl-abbrev">Diabetologia</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">2679</span>–<span class="cit-lpage">2688</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDiabetologia%26rft.stitle%253DDiabetologia%26rft.aulast%253DArgoud%26rft.auinit1%253DK.%26rft.volume%253D49%26rft.issue%253D11%26rft.spage%253D2679%26rft.epage%253D2688%26rft.atitle%253DGenetic%2Bcontrol%2Bof%2Bplasma%2Blipid%2Blevels%2Bin%2Ba%2Bcross%2Bderived%2Bfrom%2Bnormoglycaemic%2BBrown%2BNorway%2Band%2Bspontaneously%2Bdiabetic%2BGoto-Kakizaki%2Brats.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00125-006-0396-z%26rft_id%253Dinfo%253Apmid%252F16983556%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00125-006-0396-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16983556&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3671.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3671.3" data-doi="10.1016/j.cell.2013.06.040"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Atanur</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Diaz</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maratou</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sarkis</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rotival</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome sequencing reveals loci under artificial selection that underlie disease phenotypes in the laboratory rat.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">154</span>: <span class="cit-fpage">691</span>–<span class="cit-lpage">703</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D154%26rft.spage%253D691%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2013.06.040%26rft_id%253Dinfo%253Apmid%252F23890820%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2013.06.040&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23890820&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3671.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000322629900020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3671.4" data-doi="10.1371/journal.pgen.1000232"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Breitling</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tesson</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genetical genomics: spotlight on QTL hotspots.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000232</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DBreitling%26rft.auinit1%253DR.%26rft.volume%253D4%26rft.issue%253D10%26rft.spage%253De1000232%26rft.epage%253De1000232%26rft.atitle%253DGenetical%2Bgenomics%253A%2Bspotlight%2Bon%2BQTL%2Bhotspots.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000232%26rft_id%253Dinfo%253Apmid%252F18949031%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000232&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18949031&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3671.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3671.5" data-doi="10.1093/bioinformatics/btg112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">R/qtl: QTL mapping in experimental crosses.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">889</span>–<span class="cit-lpage">890</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DBroman%26rft.auinit1%253DK.%2BW.%26rft.volume%253D19%26rft.issue%253D7%26rft.spage%253D889%26rft.epage%253D890%26rft.atitle%253DR%252Fqtl%253A%2BQTL%2Bmapping%2Bin%2Bexperimental%2Bcrosses%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg112%26rft_id%253Dinfo%253Apmid%252F12724300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIxOS83Lzg4OSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzY3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3671.6" data-doi="10.1038/nrg1383"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brunner</span>  <span class="cit-name-given-names">H. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Driel</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">From syndrome families to functional genomics.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">545</span>–<span class="cit-lpage">551</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DBrunner%26rft.auinit1%253DH.%2BG.%26rft.volume%253D5%26rft.issue%253D7%26rft.spage%253D545%26rft.epage%253D551%26rft.atitle%253DFrom%2Bsyndrome%2Bfamilies%2Bto%2Bfunctional%2Bgenomics.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1383%26rft_id%253Dinfo%253Apmid%252F15211356%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1383&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15211356&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3671.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222435200017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3671.7" data-doi="10.1101/gr.187450.114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buchner</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nadeau</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Contrasting genetic architectures in different mouse reference populations used for studying complex traits.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">775</span>–<span class="cit-lpage">791</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.187450.114%26rft_id%253Dinfo%253Apmid%252F25953951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI1LzYvNzc1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNjcxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3671.8" data-doi="10.1038/ng1497"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bystrykh</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weersing</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dontje</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sutton</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pletcher</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Uncovering regulatory pathways that affect hematopoietic stem cell function using ‘genetical genomics.’</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">225</span>–<span class="cit-lpage">232</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DBystrykh%26rft.auinit1%253DL.%26rft.volume%253D37%26rft.issue%253D3%26rft.spage%253D225%26rft.epage%253D232%26rft.atitle%253DUncovering%2Bregulatory%2Bpathways%2Bthat%2Baffect%2Bhematopoietic%2Bstem%2Bcell%2Bfunction%2Busing%2B%2527genetical%2Bgenomics%2527.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1497%26rft_id%253Dinfo%253Apmid%252F15711547%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1497&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15711547&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3671.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227297800013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3671.9" data-doi="10.1038/nature06757"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lum</span>  <span class="cit-name-given-names">P. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pinto</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Variations in DNA elucidate molecular networks that cause disease.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">452</span>: <span class="cit-fpage">429</span>–<span class="cit-lpage">435</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DChen%26rft.auinit1%253DY.%26rft.volume%253D452%26rft.issue%253D7186%26rft.spage%253D429%26rft.epage%253D435%26rft.atitle%253DVariations%2Bin%2BDNA%2Belucidate%2Bmolecular%2Bnetworks%2Bthat%2Bcause%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06757%26rft_id%253Dinfo%253Apmid%252F18344982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature06757&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18344982&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3671.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000254341300023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3671.10" data-doi="10.1038/ng1518"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chesler</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shou</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Complex trait analysis of gene expression uncovers polygenic and pleiotropic networks that modulate nervous system function.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">233</span>–<span class="cit-lpage">242</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DChesler%26rft.auinit1%253DE.%2BJ.%26rft.volume%253D37%26rft.issue%253D3%26rft.spage%253D233%26rft.epage%253D242%26rft.atitle%253DComplex%2Btrait%2Banalysis%2Bof%2Bgene%2Bexpression%2Buncovers%2Bpolygenic%2Band%2Bpleiotropic%2Bnetworks%2Bthat%2Bmodulate%2Bnervous%2Bsystem%2Bfunction.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1518%26rft_id%253Dinfo%253Apmid%252F15711545%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1518&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15711545&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3671.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227297800014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3671.11" data-doi="10.1038/nrg3239"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chess</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Mechanisms and consequences of widespread random monoallelic expression.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">421</span>–<span class="cit-lpage">428</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DChess%26rft.auinit1%253DA.%26rft.volume%253D13%26rft.issue%253D6%26rft.spage%253D421%26rft.epage%253D428%26rft.atitle%253DMechanisms%2Band%2Bconsequences%2Bof%2Bwidespread%2Brandom%2Bmonoallelic%2Bexpression.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3239%26rft_id%253Dinfo%253Apmid%252F22585065%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3239&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22585065&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3671.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3671.12" data-doi="10.1038/nrg3575"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Civelek</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lusis</span>  <span class="cit-name-given-names">A. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Systems genetics approaches to understand complex traits.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">34</span>–<span class="cit-lpage">48</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D15%26rft.spage%253D34%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3575%26rft_id%253Dinfo%253Apmid%252F24296534%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3575&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24296534&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3671.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3671.13" data-doi="10.1038/ng2109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dixon</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moffatt</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heath</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A genome-wide association study of global gene expression.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">1202</span>–<span class="cit-lpage">1207</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DDixon%26rft.auinit1%253DA.%2BL.%26rft.volume%253D39%26rft.issue%253D10%26rft.spage%253D1202%26rft.epage%253D1207%26rft.atitle%253DA%2Bgenome-wide%2Bassociation%2Bstudy%2Bof%2Bglobal%2Bgene%2Bexpression.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng2109%26rft_id%253Dinfo%253Apmid%252F17873877%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng2109&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17873877&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3671.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3671.14" data-doi="10.1038/ng2026"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dumas</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilder</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bihoreau</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barton</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fearnside</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Direct quantitative trait locus mapping of mammalian metabolic phenotypes in diabetic and normoglycemic rat models.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">666</span>–<span class="cit-lpage">672</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DDumas%26rft.auinit1%253DM.%2BE.%26rft.volume%253D39%26rft.issue%253D5%26rft.spage%253D666%26rft.epage%253D672%26rft.atitle%253DDirect%2Bquantitative%2Btrait%2Blocus%2Bmapping%2Bof%2Bmammalian%2Bmetabolic%2Bphenotypes%2Bin%2Bdiabetic%2Band%2Bnormoglycemic%2Brat%2Bmodels.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng2026%26rft_id%253Dinfo%253Apmid%252F17435758%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng2026&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17435758&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3671.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000245971300024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3671.15" data-doi="10.1038/nature06758"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Emilsson</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorleifsson</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leonardson</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zink</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genetics of gene expression and its effect on disease.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">452</span>: <span class="cit-fpage">423</span>–<span class="cit-lpage">428</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DEmilsson%26rft.auinit1%253DV.%26rft.volume%253D452%26rft.issue%253D7186%26rft.spage%253D423%26rft.epage%253D428%26rft.atitle%253DGenetics%2Bof%2Bgene%2Bexpression%2Band%2Bits%2Beffect%2Bon%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06758%26rft_id%253Dinfo%253Apmid%252F18344981%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature06758&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18344981&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3671.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000254341300022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3671.16" data-doi="10.1038/ng.2205"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fairfax</span>  <span class="cit-name-given-names">B. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Makino</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Radhakrishnan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plant</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leslie</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genetics of gene expression in primary immune cells identifies cell type-specific master regulators and roles of HLA alleles.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">502</span>–<span class="cit-lpage">510</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DFairfax%26rft.auinit1%253DB.%2BP.%26rft.volume%253D44%26rft.issue%253D5%26rft.spage%253D502%26rft.epage%253D510%26rft.atitle%253DGenetics%2Bof%2Bgene%2Bexpression%2Bin%2Bprimary%2Bimmune%2Bcells%2Bidentifies%2Bcell%2Btype-specific%2Bmaster%2Bregulators%2Band%2Broles%2Bof%2BHLA%2Balleles.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2205%26rft_id%253Dinfo%253Apmid%252F22446964%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2205&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22446964&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3671.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3671.17" data-doi="10.1093/bioinformatics/btl567"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falcon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gentleman</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Using GOstats to test gene lists for GO term association.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">257</span>–<span class="cit-lpage">258</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtl567%26rft_id%253Dinfo%253Apmid%252F17098774%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyMy8yLzI1NyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzY3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-book" id="cit-6.11.3671.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gauguier</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The rat as a model physiological system,</span> pp. <span class="cit-fpage">1154</span>–<span class="cit-lpage">1171</span>, in <span class="cit-source"><em>Encyclopedia of Genetics</em>, <em>Genomics</em>, <em>Proteomics and Bioinformatics</em>,</span> edited by <span class="cit-ed"><span class="cit-name-surname">Jorde</span>  <span class="cit-name-given-names">L. B.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Little</span>  <span class="cit-name-given-names">P. F. R.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Dunn</span>  <span class="cit-name-given-names">M. J.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Subramaniam</span>  <span class="cit-name-given-names">S.</span></span>. <span class="cit-edition">Ed. 3</span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Netherlands</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3671.19" data-doi="10.1038/ng0196-38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gauguier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Froguel</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parent</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernard</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bihoreau</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Chromosomal mapping of genetic loci associated with non-insulin dependent diabetes in the GK rat.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">38</span>–<span class="cit-lpage">43</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DGauguier%26rft.auinit1%253DD.%26rft.volume%253D12%26rft.issue%253D1%26rft.spage%253D38%26rft.epage%253D43%26rft.atitle%253DChromosomal%2Bmapping%2Bof%2Bgenetic%2Bloci%2Bassociated%2Bwith%2Bnon-insulin%2Bdependent%2Bdiabetes%2Bin%2Bthe%2BGK%2Brat.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng0196-38%26rft_id%253Dinfo%253Apmid%252F8528248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng0196-38&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8528248&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3671.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996TM55500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3671.20" data-doi="10.1371/journal.pgen.0020130"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ghazalpour</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doss</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plaisier</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Integrating genetic and network analysis to characterize genes related to mouse weight.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e130</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DGhazalpour%26rft.auinit1%253DA.%26rft.volume%253D2%26rft.issue%253D8%26rft.spage%253De130%26rft.epage%253De130%26rft.atitle%253DIntegrating%2Bgenetic%2Band%2Bnetwork%2Banalysis%2Bto%2Bcharacterize%2Bgenes%2Brelated%2Bto%2Bmouse%2Bweight.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0020130%26rft_id%253Dinfo%253Apmid%252F16934000%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0020130&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16934000&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3671.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3671.21" data-doi="10.1620/tjem.119.85"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goto</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kakizaki</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Masaki</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">1976</span> <span class="cit-article-title">Production of spontaneous diabetic rats by repetition of selective breeding.</span> <abbr class="cit-jnl-abbrev">Tohoku J. Exp. Med.</abbr> <span class="cit-vol">119</span>: <span class="cit-fpage">85</span>–<span class="cit-lpage">90</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BTohoku%2Bjournal%2Bof%2Bexperimental%2Bmedicine%26rft.stitle%253DTohoku%2BJ%2BExp%2BMed%26rft.aulast%253DGoto%26rft.auinit1%253DY.%26rft.volume%253D119%26rft.issue%253D1%26rft.spage%253D85%26rft.epage%253D90%26rft.atitle%253DProduction%2Bof%2Bspontaneous%2Bdiabetic%2Brats%2Bby%2Brepetition%2Bof%2Bselective%2Bbreeding.%26rft_id%253Dinfo%253Adoi%252F10.1620%252Ftjem.119.85%26rft_id%253Dinfo%253Apmid%252F951706%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1620/tjem.119.85&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=951706&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3671.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1976BW71200009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3671.22" data-doi="10.1038/ng.2394"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grundberg</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Small</span>  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hedman</span>  <span class="cit-name-given-names">Å.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nica</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buil</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Mapping <em>cis</em>- and <em>trans</em>-regulatory effects across multiple tissues in twins.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">1084</span>–<span class="cit-lpage">1089</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DGrundberg%26rft.auinit1%253DE.%26rft.volume%253D44%26rft.issue%253D10%26rft.spage%253D1084%26rft.epage%253D1089%26rft.atitle%253DMapping%2Bcis-%2Band%2Btrans-regulatory%2Beffects%2Bacross%2Bmultiple%2Btissues%2Bin%2Btwins.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2394%26rft_id%253Dinfo%253Apmid%252F22941192%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2394&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22941192&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3671.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3671.23" data-doi="10.1126/science.1262110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">GTExConsortium</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Human genomics. The Genotype-Tissue Expression (GTEx) pilot analysis: multitissue gene regulation in humans.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">348</span>: <span class="cit-fpage">648</span>–<span class="cit-lpage">660</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DThe%2BGTEx%2BConsortium%26rft.auinit1%253D%2B%26rft.volume%253D348%26rft.issue%253D6235%26rft.spage%253D648%26rft.epage%253D660%26rft.atitle%253DThe%2BGenotype-Tissue%2BExpression%2B%2528GTEx%2529%2Bpilot%2Banalysis%253A%2BMultitissue%2Bgene%2Bregulation%2Bin%2Bhumans%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1262110%26rft_id%253Dinfo%253Apmid%252F25954001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDgvNjIzNS82NDgiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzExLzM2NzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3671.24" data-doi="10.1038/hdy.1992.131"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haley</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knott</span>  <span class="cit-name-given-names">S. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">A simple regression method for mapping quantitative trait loci in line crosses using flanking markers.</span> <abbr class="cit-jnl-abbrev">Heredity (Edinb)</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">315</span>–<span class="cit-lpage">324</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%2B%2528Edinb%2529%26rft.volume%253D69%26rft.spage%253D315%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.1992.131%26rft_id%253Dinfo%253Apmid%252F16718932%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.1992.131&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16718932&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3671.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3671.25" data-doi="10.1038/nature09386"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heinig</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petretto</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wallace</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bottolo</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rotival</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A trans-acting locus regulates an anti-viral expression network and type 1 diabetes risk.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">467</span>: <span class="cit-fpage">460</span>–<span class="cit-lpage">464</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHeinig%26rft.auinit1%253DM.%26rft.volume%253D467%26rft.issue%253D7314%26rft.spage%253D460%26rft.epage%253D464%26rft.atitle%253DA%2Btrans-acting%2Blocus%2Bregulates%2Ban%2Banti-viral%2Bexpression%2Bnetwork%2Band%2Btype%2B1%2Bdiabetes%2Brisk.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09386%26rft_id%253Dinfo%253Apmid%252F20827270%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09386&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20827270&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3671.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282090200043&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3671.26" data-doi="10.1101/gr.088120.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shifman</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johannesson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yalcin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">High resolution mapping of expression QTLs in heterogeneous stock mice in multiple tissues.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1133</span>–<span class="cit-lpage">1140</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.088120.108%26rft_id%253Dinfo%253Apmid%252F19376938%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzYvMTEzMyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzY3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3671.27" data-doi="10.1038/ng1522"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hubner</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wallace</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zimdahl</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petretto</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schulz</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Integrated transcriptional profiling and linkage analysis for identification of genes underlying disease.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">243</span>–<span class="cit-lpage">253</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DHubner%26rft.auinit1%253DN.%26rft.volume%253D37%26rft.issue%253D3%26rft.spage%253D243%26rft.epage%253D253%26rft.atitle%253DIntegrated%2Btranscriptional%2Bprofiling%2Band%2Blinkage%2Banalysis%2Bfor%2Bidentification%2Bof%2Bgenes%2Bunderlying%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1522%26rft_id%253Dinfo%253Apmid%252F15711544%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1522&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15711544&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3671.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227297800015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3671.28" data-doi="10.1093/nar/28.1.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kanehisa</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goto</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">KEGG: Kyoto Encyclopedia of Genes and Genomes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">27</span>–<span class="cit-lpage">30</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DKanehisa%26rft.auinit1%253DM.%26rft.volume%253D28%26rft.issue%253D1%26rft.spage%253D27%26rft.epage%253D30%26rft.atitle%253DKEGG%253A%2BKyoto%2BEncyclopedia%2Bof%2BGenes%2Band%2BGenomes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F28.1.27%26rft_id%253Dinfo%253Apmid%252F10592173%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjc6IjI4LzEvMjciO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzExLzM2NzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3671.29" data-doi="10.1093/nar/gkn863"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Matthews</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gopinath</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gillespie</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caudy</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Croft</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Reactome knowledgebase of human biological pathways and processes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">D619</span>–<span class="cit-lpage">D622</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkn863%26rft_id%253Dinfo%253Apmid%252F18981052%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzNy9zdXBwbF8xL0Q2MTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzExLzM2NzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3671.30" data-doi="10.1038/ng.129"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Monti</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paskas</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heinig</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schulz</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Soluble epoxide hydrolase is a susceptibility factor for heart failure in a rat model of human disease.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">529</span>–<span class="cit-lpage">537</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DMonti%26rft.auinit1%253DJ.%26rft.volume%253D40%26rft.issue%253D5%26rft.spage%253D529%26rft.epage%253D537%26rft.atitle%253DSoluble%2Bepoxide%2Bhydrolase%2Bis%2Ba%2Bsusceptibility%2Bfactor%2Bfor%2Bheart%2Bfailure%2Bin%2Ba%2Brat%2Bmodel%2Bof%2Bhuman%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.129%26rft_id%253Dinfo%253Apmid%252F18443590%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.129&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18443590&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3671.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255366700018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3671.31" data-doi="10.1016/j.cell.2013.11.043"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mott</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaisaki</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gan</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cleak</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The architecture of parent-of-origin effects in mice.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">156</span>: <span class="cit-fpage">332</span>–<span class="cit-lpage">342</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D156%26rft.spage%253D332%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2013.11.043%26rft_id%253Dinfo%253Apmid%252F24439386%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2013.11.043&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24439386&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3671.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000329912200031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3671.32" data-doi="10.1371/journal.pgen.0020172"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Petretto</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mangion</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dickens</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cook</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumaran</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Heritability and tissue specificity of expression quantitative trait loci.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e172</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DPetretto%26rft.auinit1%253DE.%26rft.volume%253D2%26rft.issue%253D10%26rft.spage%253De172%26rft.epage%253De172%26rft.atitle%253DHeritability%2Band%2Btissue%2Bspecificity%2Bof%2Bexpression%2Bquantitative%2Btrait%2Bloci.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0020172%26rft_id%253Dinfo%253Apmid%252F17054398%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0020172&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17054398&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3671.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3671.33" data-doi="10.1038/ng.134"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Petretto</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sarwar</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grieve</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumaran</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Integrated genomic approaches implicate osteoglycin (Ogn) in the regulation of left ventricular mass.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">546</span>–<span class="cit-lpage">552</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DPetretto%26rft.auinit1%253DE.%26rft.volume%253D40%26rft.issue%253D5%26rft.spage%253D546%26rft.epage%253D552%26rft.atitle%253DIntegrated%2Bgenomic%2Bapproaches%2Bimplicate%2Bosteoglycin%2B%2528Ogn%2529%2Bin%2Bthe%2Bregulation%2Bof%2Bleft%2Bventricular%2Bmass.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.134%26rft_id%253Dinfo%253Apmid%252F18443592%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.134&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18443592&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3671.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255366700020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3671.34" data-doi="10.1038/nature01434"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schadt</span>  <span class="cit-name-given-names">E. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Monks</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drake</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lusis</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Che</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Genetics of gene expression surveyed in maize, mouse and man.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">422</span>: <span class="cit-fpage">297</span>–<span class="cit-lpage">302</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSchadt%26rft.auinit1%253DE.%2BE.%26rft.volume%253D422%26rft.issue%253D6929%26rft.spage%253D297%26rft.epage%253D302%26rft.atitle%253DGenetics%2Bof%2Bgene%2Bexpression%2Bsurveyed%2Bin%2Bmaize%252C%2Bmouse%2Band%2Bman.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01434%26rft_id%253Dinfo%253Apmid%252F12646919%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature01434&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12646919&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3671.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181637300037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3671.35" data-doi="10.1093/nar/gkq871"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oshlack</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smyth</span>  <span class="cit-name-given-names">G. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Optimizing the noise <em>vs.</em> bias trade-off for Illumina whole genome expression BeadChips.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">e204</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq871%26rft_id%253Dinfo%253Apmid%252F20929874%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzOC8yMi9lMjA0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNjcxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3671.36" data-doi="10.1007/s00335-004-2326-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Solberg</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baum</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ahmadiyeh</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shimomura</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Sex- and lineage-specific inheritance of depression-like behavior in the rat.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">648</span>–<span class="cit-lpage">662</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DSolberg%26rft.auinit1%253DL.%2BC.%26rft.volume%253D15%26rft.issue%253D8%26rft.spage%253D648%26rft.epage%253D662%26rft.atitle%253DSex-%2Band%2Blineage-specific%2Binheritance%2Bof%2Bdepression-like%2Bbehavior%2Bin%2Bthe%2Brat.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-004-2326-z%26rft_id%253Dinfo%253Apmid%252F15457344%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-004-2326-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15457344&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3671.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223063600008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3671.37" data-doi="10.1016/j.molcel.2005.12.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sopko</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Preston</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chua</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Papp</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Mapping pathways and phenotypes by systematic gene overexpression.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">319</span>–<span class="cit-lpage">330</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DSopko%26rft.auinit1%253DR.%26rft.volume%253D21%26rft.issue%253D3%26rft.spage%253D319%26rft.epage%253D330%26rft.atitle%253DMapping%2Bpathways%2Band%2Bphenotypes%2Bby%2Bsystematic%2Bgene%2Boverexpression.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2005.12.011%26rft_id%253Dinfo%253Apmid%252F16455487%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2005.12.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16455487&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3671.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000235436100003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3671.38" data-doi="10.1073/pnas.0506580102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Subramanian</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tamayo</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mootha</span>  <span class="cit-name-given-names">V. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mukherjee</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ebert</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">15545</span>–<span class="cit-lpage">15550</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0506580102%26rft_id%253Dinfo%253Apmid%252F16199517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAyLzQzLzE1NTQ1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNjcxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3671.39" data-doi="10.1152/physiolgenomics.00114.2004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wallace</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wallis</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Argoud</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaisaki</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Quantitative trait locus dissection in congenic strains of the Goto-Kakizaki rat identifies a region conserved with diabetes loci in human chromosome 1q.</span> <abbr class="cit-jnl-abbrev">Physiol. Genomics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhysiol.%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1152%252Fphysiolgenomics.00114.2004%26rft_id%253Dinfo%253Apmid%252F15266047%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTU6InBoeXNpb2xnZW5vbWljcyI7czo1OiJyZXNpZCI7czo2OiIxOS8xLzEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzExLzM2NzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3671.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wallis</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wallace</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McAteer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Argoud</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Enhanced insulin secretion and cholesterol metabolism in congenic strains of the spontaneously diabetic (Type 2) Goto Kakizaki rat are controlled by independent genetic loci in rat chromosome 8.</span> <abbr class="cit-jnl-abbrev">Diabetologia</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">1096</span>–<span class="cit-lpage">1106</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDiabetologia%26rft.stitle%253DDiabetologia%26rft.aulast%253DWallis%26rft.auinit1%253DR.%2BH.%26rft.volume%253D47%26rft.issue%253D6%26rft.spage%253D1096%26rft.epage%253D1106%26rft.atitle%253DEnhanced%2Binsulin%2Bsecretion%2Band%2Bcholesterol%2Bmetabolism%2Bin%2Bcongenic%2Bstrains%2Bof%2Bthe%2Bspontaneously%2Bdiabetic%2B%2528Type%2B2%2529%2BGoto%2BKakizaki%2Brat%2Bare%2Bcontrolled%2Bby%2Bindependent%2Bgenetic%2Bloci%2Bin%2Brat%2Bchromosome%2B8.%26rft_id%253Dinfo%253Apmid%252F15164172%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=15164172&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3671.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222240100018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3671.41" data-doi="10.1371/journal.pone.0002962"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wallis</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaisaki</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Argoud</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilder</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Pathophysiological, genetic and gene expression features of a novel rodent model of the cardio-metabolic syndrome.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e2962</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DWallis%26rft.auinit1%253DR.%2BH.%26rft.volume%253D3%26rft.issue%253D8%26rft.spage%253De2962%26rft.epage%253De2962%26rft.atitle%253DPathophysiological%252C%2Bgenetic%2Band%2Bgene%2Bexpression%2Bfeatures%2Bof%2Ba%2Bnovel%2Brodent%2Bmodel%2Bof%2Bthe%2Bcardio-metabolic%2Bsyndrome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0002962%26rft_id%253Dinfo%253Apmid%252F18698428%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0002962&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18698428&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3671.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3671.42" data-doi="10.1101/gr.2001604"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wilder</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bihoreau</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Argoud</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watanabe</span>  <span class="cit-name-given-names">T. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lathrop</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Integration of the rat recombination and EST maps in the rat genomic sequence and comparative mapping analysis with the mouse genome.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">758</span>–<span class="cit-lpage">765</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DWilder%26rft.auinit1%253DS.%2BP.%26rft.volume%253D14%26rft.issue%253D4%26rft.spage%253D758%26rft.epage%253D765%26rft.atitle%253DIntegration%2Bof%2Bthe%2BRat%2BRecombination%2Band%2BEST%2BMaps%2Bin%2Bthe%2BRat%2BGenomic%2BSequence%2Band%2BComparative%2BMapping%2BAnalysis%2BWith%2Bthe%2BMouse%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.2001604%26rft_id%253Dinfo%253Apmid%252F15060020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE0LzQvNzU4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNjcxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3671.43" data-doi="10.1016/j.cell.2015.06.024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">E. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auwerx</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The convergence of systems and reductionist approaches in complex trait analysis.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">162</span>: <span class="cit-fpage">23</span>–<span class="cit-lpage">32</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D162%26rft.spage%253D23%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2015.06.024%26rft_id%253Dinfo%253Apmid%252F26140590%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2015.06.024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26140590&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3671.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3671.44" data-doi="10.1038/ng.167"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">E. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drees</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brem</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Integrating large-scale functional genomic data to dissect the complexity of yeast regulatory networks.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">854</span>–<span class="cit-lpage">861</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DZhu%26rft.auinit1%253DJ.%26rft.volume%253D40%26rft.issue%253D7%26rft.spage%253D854%26rft.epage%253D861%26rft.atitle%253DIntegrating%2Blarge-scale%2Bfunctional%2Bgenomic%2Bdata%2Bto%2Bdissect%2Bthe%2Bcomplexity%2Bof%2Byeast%2Bregulatory%2Bnetworks.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.167%26rft_id%253Dinfo%253Apmid%252F18552845%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.167&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18552845&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3671.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257166500020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/11/3671.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/11/3663" title="Determinants in the LIN-12/Notch Intracellular Domain That Govern Its Activity and Stability During Caenorhabditis elegans Vulval Development" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/11/3685" title="Non-Mendelian Dominant Maternal Effects Caused by CRISPR/Cas9 Transgenic Components in Drosophila melanogaster" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/11"><span>Volume 6 Issue 11, November 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/11.cover-source.jpg?itok=hTYn4PRy" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/11/3671" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-SqQbdnS5AGUG3hfsjPHDuBc7uOoUFsct41EFWfKxrhs" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/11/3671" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/11/3671" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/11/3671" class="active" data-icon-position="" data-hide-link-title="0">Transcriptome Profiling in Rat Inbred Strains and Experimental Cross Reveals Discrepant Genetic Architecture of Genome-Wide Gene Expression</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460451" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-S9MadFbXiV4sJw1XH5fpvv8EtcqAvyn5Tjt1HlhtYCs" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/11/3671.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/11/3671" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-ge2tJgAcaQA3dOeUdxlZHM52j7GzFPp0XXQbDzUytHc" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/11/3671" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/11/3671.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460451" id="node460451--2" data-pisa="ggg;6/11/3671" data-pisa-master="ggg;g3.116.033274" data-apath="/ggg/6/11/3671.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Transcriptome Profiling in Rat Inbred Strains and Experimental Cross Reveals Discrepant Genetic Architecture of Genome-Wide Gene Expression</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Pamela J. Kaisaki</span>, <span class="highwire-citation-author" data-delta="1">Georg W. Otto</span>, <span class="highwire-citation-author" data-delta="2">Karène Argoud</span>, <span class="highwire-citation-author" data-delta="3">Stephan C. Collins</span>, <span class="highwire-citation-author" data-delta="4">Robert H. Wallis</span>, <span class="highwire-citation-author" data-delta="5">Steven P. Wilder</span>, <span class="highwire-citation-author" data-delta="6">Anthony C. Y. Yau</span>, <span class="highwire-citation-author" data-delta="7">Christophe Hue</span>, <span class="highwire-citation-author" data-delta="8">Sophie Calderari</span>, <span class="highwire-citation-author" data-delta="9">Marie-Thérèse Bihoreau</span>, <span class="highwire-citation-author" data-delta="10">Jean-Baptiste Cazier</span>, <span class="highwire-citation-author" data-delta="11">Richard Mott</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="12"><a href="http://orcid.org/0000-0001-6156-9530" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Dominique Gauguier</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3671-3683;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.033274 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460451--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Pamela J. Kaisaki </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Wellcome Trust Centre for Human Genetics, University of Oxford, OX3 7BN, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pamela%2BJ.%2BKaisaki%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kaisaki%20PJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APamela%2BJ.%2BKaisaki%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Georg W. Otto </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Wellcome Trust Centre for Human Genetics, University of Oxford, OX3 7BN, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Georg%2BW.%2BOtto%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Otto%20GW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGeorg%2BW.%2BOtto%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Karène Argoud </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Wellcome Trust Centre for Human Genetics, University of Oxford, OX3 7BN, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kar%C3%A8ne%2BArgoud%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Argoud%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKar%25C3%25A8ne%2BArgoud%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Stephan C. Collins </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Wellcome Trust Centre for Human Genetics, University of Oxford, OX3 7BN, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephan%2BC.%2BCollins%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Collins%20SC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStephan%2BC.%2BCollins%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Robert H. Wallis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Wellcome Trust Centre for Human Genetics, University of Oxford, OX3 7BN, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BH.%2BWallis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wallis%20RH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BH.%2BWallis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Steven P. Wilder </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Wellcome Trust Centre for Human Genetics, University of Oxford, OX3 7BN, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Steven%2BP.%2BWilder%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wilder%20SP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASteven%2BP.%2BWilder%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Anthony C. Y. Yau </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Wellcome Trust Centre for Human Genetics, University of Oxford, OX3 7BN, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anthony%2BC.%2BY.%2BYau%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yau%20AC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnthony%2BC.%2BY.%2BYau%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Christophe Hue </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Cordeliers Research Centre, Institut National de la Santé et de la Recherche Médicale Unités Mixtes de Recherche Scientifique 1138, Sorbonne Universities, University Pierre & Marie Curie, University Paris Descartes, Sorbonne Paris Cité, 75006, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christophe%2BHue%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hue%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristophe%2BHue%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Sophie Calderari </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Cordeliers Research Centre, Institut National de la Santé et de la Recherche Médicale Unités Mixtes de Recherche Scientifique 1138, Sorbonne Universities, University Pierre & Marie Curie, University Paris Descartes, Sorbonne Paris Cité, 75006, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sophie%2BCalderari%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Calderari%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASophie%2BCalderari%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Marie-Thérèse Bihoreau </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Wellcome Trust Centre for Human Genetics, University of Oxford, OX3 7BN, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marie-Th%C3%A9r%C3%A8se%2BBihoreau%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bihoreau%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarie-Th%25C3%25A9r%25C3%25A8se%2BBihoreau%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Jean-Baptiste Cazier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Centre for Computational Biology, University of Birmingham, Edgbaston B15 2TT, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-Baptiste%2BCazier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cazier%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJean-Baptiste%2BCazier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Richard Mott </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>University College London Genetics Institute, WC1E 6BT, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Richard%2BMott%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mott%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARichard%2BMott%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Dominique Gauguier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Wellcome Trust Centre for Human Genetics, University of Oxford, OX3 7BN, United Kingdom</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Cordeliers Research Centre, Institut National de la Santé et de la Recherche Médicale Unités Mixtes de Recherche Scientifique 1138, Sorbonne Universities, University Pierre & Marie Curie, University Paris Descartes, Sorbonne Paris Cité, 75006, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dominique%2BGauguier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gauguier%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADominique%2BGauguier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6156-9530" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Dominique Gauguier</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dominique.gauguier@crc.jussieu.fr" class="" data-icon-position="" data-hide-link-title="0">dominique.gauguier@crc.jussieu.fr</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/11/3671" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/11/3671&amp;title=Transcriptome%20Profiling%20in%20Rat%20Inbred%20Strains%20and%20Experimental%20Cross%20Reveals%20Discrepant%20Genetic%20Architecture%20of%20Genome-Wide%20Gene%20Expression" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/11/3671&amp;title=Transcriptome%20Profiling%20in%20Rat%20Inbred%20Strains%20and%20Experimental%20Cross%20Reveals%20Discrepant%20Genetic%20Architecture%20of%20Genome-Wide%20Gene%20Expression" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/11/3671&amp;title=Transcriptome%20Profiling%20in%20Rat%20Inbred%20Strains%20and%20Experimental%20Cross%20Reveals%20Discrepant%20Genetic%20Architecture%20of%20Genome-Wide%20Gene%20Expression" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/11/3671&amp;text=Transcriptome%20Profiling%20in%20Rat%20Inbred%20Strains%20and%20Experimental%20Cross%20Reveals%20Discrepant%20Genetic%20Architecture%20of%20Genome-Wide%20Gene%20Expression" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/11/3671&amp;title=Transcriptome%20Profiling%20in%20Rat%20Inbred%20Strains%20and%20Experimental%20Cross%20Reveals%20Discrepant%20Genetic%20Architecture%20of%20Genome-Wide%20Gene%20Expression" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/11/3671&amp;t=Transcriptome%20Profiling%20in%20Rat%20Inbred%20Strains%20and%20Experimental%20Cross%20Reveals%20Discrepant%20Genetic%20Architecture%20of%20Genome-Wide%20Gene%20Expression" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/11/3671&amp;title=Transcriptome%20Profiling%20in%20Rat%20Inbred%20Strains%20and%20Experimental%20Cross%20Reveals%20Discrepant%20Genetic%20Architecture%20of%20Genome-Wide%20Gene%20Expression" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/11/3671&amp;title=Transcriptome%20Profiling%20in%20Rat%20Inbred%20Strains%20and%20Experimental%20Cross%20Reveals%20Discrepant%20Genetic%20Architecture%20of%20Genome-Wide%20Gene%20Expression" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460451/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460451" id="node460451--3" data-pisa="ggg;6/11/3671" data-pisa-master="ggg;g3.116.033274" data-apath="/ggg/6/11/3671.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Transcriptome Profiling in Rat Inbred Strains and Experimental Cross Reveals Discrepant Genetic Architecture of Genome-Wide Gene Expression</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Pamela J. Kaisaki</span>, <span class="highwire-citation-author" data-delta="1">Georg W. Otto</span>, <span class="highwire-citation-author" data-delta="2">Karène Argoud</span>, <span class="highwire-citation-author" data-delta="3">Stephan C. Collins</span>, <span class="highwire-citation-author" data-delta="4">Robert H. Wallis</span>, <span class="highwire-citation-author" data-delta="5">Steven P. Wilder</span>, <span class="highwire-citation-author" data-delta="6">Anthony C. Y. Yau</span>, <span class="highwire-citation-author" data-delta="7">Christophe Hue</span>, <span class="highwire-citation-author" data-delta="8">Sophie Calderari</span>, <span class="highwire-citation-author" data-delta="9">Marie-Thérèse Bihoreau</span>, <span class="highwire-citation-author" data-delta="10">Jean-Baptiste Cazier</span>, <span class="highwire-citation-author" data-delta="11">Richard Mott</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="12"><a href="http://orcid.org/0000-0001-6156-9530" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Dominique Gauguier</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3671-3683;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.033274 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460451--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Pamela J. Kaisaki </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Wellcome Trust Centre for Human Genetics, University of Oxford, OX3 7BN, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pamela%2BJ.%2BKaisaki%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kaisaki%20PJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APamela%2BJ.%2BKaisaki%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Georg W. Otto </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Wellcome Trust Centre for Human Genetics, University of Oxford, OX3 7BN, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Georg%2BW.%2BOtto%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Otto%20GW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGeorg%2BW.%2BOtto%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Karène Argoud </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Wellcome Trust Centre for Human Genetics, University of Oxford, OX3 7BN, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kar%C3%A8ne%2BArgoud%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Argoud%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKar%25C3%25A8ne%2BArgoud%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Stephan C. Collins </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Wellcome Trust Centre for Human Genetics, University of Oxford, OX3 7BN, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephan%2BC.%2BCollins%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Collins%20SC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStephan%2BC.%2BCollins%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Robert H. Wallis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Wellcome Trust Centre for Human Genetics, University of Oxford, OX3 7BN, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BH.%2BWallis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wallis%20RH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BH.%2BWallis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Steven P. Wilder </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Wellcome Trust Centre for Human Genetics, University of Oxford, OX3 7BN, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Steven%2BP.%2BWilder%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wilder%20SP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASteven%2BP.%2BWilder%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Anthony C. Y. Yau </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Wellcome Trust Centre for Human Genetics, University of Oxford, OX3 7BN, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anthony%2BC.%2BY.%2BYau%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yau%20AC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnthony%2BC.%2BY.%2BYau%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Christophe Hue </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Cordeliers Research Centre, Institut National de la Santé et de la Recherche Médicale Unités Mixtes de Recherche Scientifique 1138, Sorbonne Universities, University Pierre & Marie Curie, University Paris Descartes, Sorbonne Paris Cité, 75006, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christophe%2BHue%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hue%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristophe%2BHue%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Sophie Calderari </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Cordeliers Research Centre, Institut National de la Santé et de la Recherche Médicale Unités Mixtes de Recherche Scientifique 1138, Sorbonne Universities, University Pierre & Marie Curie, University Paris Descartes, Sorbonne Paris Cité, 75006, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sophie%2BCalderari%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Calderari%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASophie%2BCalderari%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Marie-Thérèse Bihoreau </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Wellcome Trust Centre for Human Genetics, University of Oxford, OX3 7BN, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marie-Th%C3%A9r%C3%A8se%2BBihoreau%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bihoreau%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarie-Th%25C3%25A9r%25C3%25A8se%2BBihoreau%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Jean-Baptiste Cazier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Centre for Computational Biology, University of Birmingham, Edgbaston B15 2TT, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-Baptiste%2BCazier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cazier%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJean-Baptiste%2BCazier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Richard Mott </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>University College London Genetics Institute, WC1E 6BT, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Richard%2BMott%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mott%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARichard%2BMott%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Dominique Gauguier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Wellcome Trust Centre for Human Genetics, University of Oxford, OX3 7BN, United Kingdom</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Cordeliers Research Centre, Institut National de la Santé et de la Recherche Médicale Unités Mixtes de Recherche Scientifique 1138, Sorbonne Universities, University Pierre & Marie Curie, University Paris Descartes, Sorbonne Paris Cité, 75006, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dominique%2BGauguier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gauguier%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADominique%2BGauguier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6156-9530" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Dominique Gauguier</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dominique.gauguier@crc.jussieu.fr" class="" data-icon-position="" data-hide-link-title="0">dominique.gauguier@crc.jussieu.fr</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3671&amp;rft.epage=3683&amp;rft.atitle=Transcriptome%20Profiling%20in%20Rat%20Inbred%20Strains%20and%20Experimental%20Cross%20Reveals%20Discrepant%20Genetic%20Architecture%20of%20Genome-Wide%20Gene%20Expression&amp;rft.volume=6&amp;rft.issue=11&amp;rft.date=2016-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Kaisaki%2C+Pamela+J.&amp;rft.au=Otto%2C+Georg+W.&amp;rft.au=Argoud%2C+Kar%C3%A8ne&amp;rft.au=Collins%2C+Stephan+C.&amp;rft.au=Wallis%2C+Robert+H.&amp;rft.au=Wilder%2C+Steven+P.&amp;rft.au=Yau%2C+Anthony+C.+Y.&amp;rft.au=Hue%2C+Christophe&amp;rft.au=Calderari%2C+Sophie&amp;rft.au=Bihoreau%2C+Marie-Th%C3%A9r%C3%A8se&amp;rft.au=Cazier%2C+Jean-Baptiste&amp;rft.au=Mott%2C+Richard&amp;rft.au=Gauguier%2C+Dominique"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460451/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460451/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460451/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460451/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460451/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460451/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460451/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460451/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460451/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460451/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460451/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460451/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/11/3671" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-17" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/11/3671.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/11/3671.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/11/3671.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/11/3671.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
