<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/11/3467" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/11/3467.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/11/3467.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/11/3467.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/6/11/3467/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/11/3467.atom" />
<meta name="HW.pisa" content="ggg;6/11/3467" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Depletion of Shine-Dalgarno Sequences Within Bacterial Coding Regions Is Expression Dependent" />
<meta name="DC.Identifier" content="10.1534/g3.116.032227" />
<meta name="DC.Date" content="2016-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Yang et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Efficient and accurate protein synthesis is crucial for organismal survival in competitive environments. Translation efficiency (the number of proteins translated from a single mRNA in a given time period) is the combined result of differential translation initiation, elongation, and termination rates. Previous research identified the Shine-Dalgarno (SD) sequence as a modulator of translation initiation in bacterial genes, while codon usage biases are frequently implicated as a primary determinant of elongation rate variation. Recent studies have suggested that SD sequences within coding sequences may negatively affect translation elongation speed, but this claim remains controversial. Here, we present a metric to quantify the prevalence of SD sequences in coding regions. We analyze hundreds of bacterial genomes and find that the coding sequences of highly expressed genes systematically contain fewer SD sequences than expected, yielding a robust correlation between the normalized occurrence of SD sites and protein abundances across a range of bacterial taxa. We further show that depletion of SD sequences within ribosomal protein genes is correlated with organismal growth rates, supporting the hypothesis of strong selection against the presence of these sequences in coding regions and suggesting their association with translation efficiency in bacteria." />
<meta name="DC.Contributor" content="Chuyue Yang" />
<meta name="DC.Contributor" content="Adam J. Hockenberry" />
<meta name="DC.Contributor" content="Michael C. Jewett" />
<meta name="DC.Contributor" content="Luís A. N. Amaral" />
<meta name="article:published_time" content="2016-11-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Depletion of Shine-Dalgarno Sequences Within Bacterial Coding Regions Is Expression Dependent" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Efficient and accurate protein synthesis is crucial for organismal survival in competitive environments. Translation efficiency (the number of proteins translated from a single mRNA in a given time period) is the combined result of differential translation initiation, elongation, and termination rates. Previous research identified the Shine-Dalgarno (SD) sequence as a modulator of translation initiation in bacterial genes, while codon usage biases are frequently implicated as a primary determinant of elongation rate variation. Recent studies have suggested that SD sequences within coding sequences may negatively affect translation elongation speed, but this claim remains controversial. Here, we present a metric to quantify the prevalence of SD sequences in coding regions. We analyze hundreds of bacterial genomes and find that the coding sequences of highly expressed genes systematically contain fewer SD sequences than expected, yielding a robust correlation between the normalized occurrence of SD sites and protein abundances across a range of bacterial taxa. We further show that depletion of SD sequences within ribosomal protein genes is correlated with organismal growth rates, supporting the hypothesis of strong selection against the presence of these sequences in coding regions and suggesting their association with translation efficiency in bacteria.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/11/01" />
<meta name="citation_mjid" content="ggg;6/11/3467" />
<meta name="citation_id" content="6/11/3467" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/11/3467" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/11/3467.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/11/3467.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/11/3467.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.032227" />
<meta name="citation_pmid" content="27605518" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3467" />
<meta name="citation_lastpage" content="3474" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Chuyue Yang" />
<meta name="citation_author_institution" content="Department of Chemical and Biological Engineering, Northwestern University, Evanston, Illinois 60208" />
<meta name="citation_author" content="Adam J. Hockenberry" />
<meta name="citation_author_institution" content="Department of Chemical and Biological Engineering, Northwestern University, Evanston, Illinois 60208" />
<meta name="citation_author_institution" content="Interdisciplinary Program in Biological Sciences, Northwestern University, Evanston, Illinois 60208" />
<meta name="citation_author" content="Michael C. Jewett" />
<meta name="citation_author_institution" content="Department of Chemical and Biological Engineering, Northwestern University, Evanston, Illinois 60208" />
<meta name="citation_author_institution" content="Interdisciplinary Program in Biological Sciences, Northwestern University, Evanston, Illinois 60208" />
<meta name="citation_author_institution" content="Northwestern Institute on Complex Systems, Northwestern University, Evanston, Illinois 60208" />
<meta name="citation_author_institution" content="Chemistry of Life Processes Institute, Northwestern University, Evanston, Illinois 60208" />
<meta name="citation_author_email" content="m-jewett@northwestern.edu" />
<meta name="citation_author_email" content="amaral@northwestern.edu" />
<meta name="citation_author" content="Luís A. N. Amaral" />
<meta name="citation_author_institution" content="Department of Chemical and Biological Engineering, Northwestern University, Evanston, Illinois 60208" />
<meta name="citation_author_institution" content="Northwestern Institute on Complex Systems, Northwestern University, Evanston, Illinois 60208" />
<meta name="citation_author_institution" content="Department of Physics and Astronomy, Northwestern University, Evanston, Illinois 60208" />
<meta name="citation_author_email" content="m-jewett@northwestern.edu" />
<meta name="citation_author_email" content="amaral@northwestern.edu" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=D. Agashe;citation_author=N. C. Martinez-Gomez;citation_author=D. A. Drummond;citation_author=C. J. Marx;citation_title=Good codons, bad transcript: large reductions in gene expression and fitness arising from synonymous mutations in a key enzyme.;citation_volume=30;citation_year=2013;citation_pmid=23223712;citation_doi=10.1093/molbev/mss273" />
<meta name="citation_reference" content="citation_journal_title=J. Mol. Biol.;citation_author=A. Borg;citation_author=M. Ehrenberg;citation_title=Determinants of the rate of mRNA translocation in bacterial protein synthesis.;citation_pages=1835-1847;citation_volume=427;citation_year=2015;citation_pmid=25451025;citation_doi=10.1016/j.jmb.2014.10.027" />
<meta name="citation_reference" content="citation_journal_title=Molecular Systems Biology;citation_journal_abbrev=Mol Syst Biol;citation_author=O. Borkowski;citation_author=A. Goelzer;citation_author=M. Schaffer;citation_author=M. Calabre;citation_author=U. Ma der;citation_title=Translation elicits a growth rate-dependent, genome-wide, differential protein production in Bacillus subtilis;citation_pages=870-870;citation_volume=12;citation_year=2016;citation_issue=5;citation_pmid=27193784;citation_doi=10.15252/msb.20156608" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=M. Botzman;citation_author=H. Margalit;citation_title=Variation in global codon usage bias among prokaryotic organisms is associated with their lifestyles.;citation_pages=R109-R109;citation_volume=12;citation_year=2011;citation_issue=10;citation_pmid=22032172;citation_doi=10.1186/gb-2011-12-10-r109" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Y. Chadani;citation_author=T. Niwa;citation_author=S. Chiba;citation_author=H. Taguchi;citation_author=K. Ito;citation_title=Integrated in vivo and in vitro nascent chain profiling reveals widespread translational pausing.;citation_volume=113;citation_year=2016;citation_pmid=26831095;citation_doi=10.1073/pnas.1520560113" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=C. A. Charneski;citation_author=L. D. Hurst;citation_title=Positively charged residues are the major determinants of ribosomal velocity.;citation_pages=e1001508;citation_volume=11;citation_year=2013;citation_pmid=23554576;citation_doi=10.1371/journal.pbio.1001508" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=J. Chen;citation_author=A. Petrov;citation_author=M. Johansson;citation_author=A. Tsai;citation_author=S. E. O’Leary;citation_title=Dynamic pathways of -1 translational frameshifting.;citation_pages=328-332;citation_volume=512;citation_year=2014;citation_pmid=24919156;citation_doi=10.1038/nature13428" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=F. F. V. Chevance;citation_author=S. L. Guyon;citation_author=K. T. Hughes;citation_title=The effects of codon context on in vivo translation speed.;citation_pages=e1004392;citation_volume=10;citation_year=2014;citation_pmid=24901308;citation_doi=10.1371/journal.pgen.1004392" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=D. Chu;citation_author=D. J. Barnes;citation_author=T. Von Der Haar;citation_title=The role of tRNA and ribosome competition in coupling the expression of different mRNAs in Saccharomyces cerevisiae.;citation_volume=39;citation_year=2011;citation_pmid=21558172;citation_doi=10.1093/nar/gkr300" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. Dana;citation_author=T. Tuller;citation_title=The effect of tRNA levels on decoding times of mRNA codons.;citation_volume=42;citation_year=2014;citation_pmid=25056313;citation_doi=10.1093/nar/gku646" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=E. Dekel;citation_author=U. Alon;citation_title=Optimality and evolutionary tuning of the expression level of a protein.;citation_pages=588-592;citation_volume=436;citation_year=2005;citation_issue=7050;citation_pmid=16049495;citation_doi=10.1038/nature03842" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=A. Devaraj;citation_author=K. Fredrick;citation_title=Short spacing between the Shine-Dalgarno sequence and P codon destabilizes codon-anticodon pairing in the P site to promote +1 programmed frameshifting.;citation_pages=1500-1509;citation_volume=78;citation_year=2010;citation_issue=6;citation_pmid=21143320;citation_doi=10.1111/j.1365-2958.2010.07421.x" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=G. D. Diwan;citation_author=D. Agashe;citation_title=The frequency of internal Shine-Dalgarno – like motifs in prokaryotes.;citation_volume=8;citation_year=2016;citation_pmid=27189998;citation_doi=10.1093/gbe/evw107" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=M. dos Reis;citation_author=R. Savva;citation_author=L. Wernisch;citation_title=Solving the riddle of codon usage preferences: a test for translational selection;citation_pages=5036-5044;citation_volume=32;citation_year=2004;citation_issue=17;citation_pmid=15448185;citation_doi=10.1093/nar/gkh834" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=D. A. Drummond;citation_author=C. O. Wilke;citation_title=Mistranslation-induced protein misfolding as a dominant constraint on coding-sequence evolution.;citation_pages=341-352;citation_volume=134;citation_year=2008;citation_issue=2;citation_pmid=18662548;citation_doi=10.1016/j.cell.2008.05.042" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=R. C. Edgar;citation_title=MUSCLE: multiple sequence alignment with high accuracy and high throughput;citation_pages=1792-1797;citation_volume=32;citation_year=2004;citation_issue=5;citation_pmid=15034147;citation_doi=10.1093/nar/gkh340" />
<meta name="citation_reference" content="citation_journal_title=eLife;citation_author=N. Fluman;citation_author=S. Navon;citation_author=E. Bibi;citation_author=Y. Pilpel;citation_title=mRNA-programmed translation pauses in the targeting of E. coli membrane proteins.;citation_pages=1-19;citation_volume=3;citation_year=2014;citation_pmid=25049222;citation_doi=10.7554/eLife.03671" />
<meta name="citation_reference" content="citation_journal_title=eLife;citation_author=J. Gardin;citation_author=R. Yeasmin;citation_author=A. Yurovsky;citation_author=Y. Cai;citation_author=S. Skiena;citation_title=Measurement of average decoding rates of the 61 sense codons in vivo.;citation_pages=1-20;citation_volume=3;citation_year=2014;citation_pmid=25049222;citation_doi=10.7554/eLife.03671" />
<meta name="citation_reference" content="citation_journal_title=Mol. Syst. Biol.;citation_author=H. Gingold;citation_author=Y. Pilpel;citation_title=Determinants of translation efficiency and accuracy.;citation_pages=1-13;citation_volume=7;citation_year=2011;citation_doi=10.1038/msb.2011.80" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. R. Gruber;citation_author=R. Lorenz;citation_author=S. H. Bernhart;citation_author=R. Neuböck;citation_author=I. L. Hofacker;citation_title=The Vienna RNA websuite.;citation_volume=36;citation_year=2008;citation_pmid=18424795;citation_doi=10.1093/nar/gkn188" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. C. Guimaraes;citation_author=M. Rocha;citation_author=A. P. Arkin;citation_title=Transcript level and sequence determinants of protein abundance and noise in Escherichia coli.;citation_volume=42;citation_year=2014;citation_pmid=24510099;citation_doi=10.1093/nar/gku126" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. A. Hussmann;citation_author=S. Patchett;citation_author=A. Johnson;citation_author=S. Sawyer;citation_author=W. H. Press;citation_title=Understanding biases in ribosome profiling experiments reveals signatures of translation dynamics in yeast.;citation_pages=1-25;citation_volume=11;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=T. Ikemura;citation_title=Correlation between the abundance of Escherichia coli transfer RNAs and the occurrence of the respective codons in its protein genes: a proposal for a synonymous codon choice that is optimal for the E. coli translational system.;citation_pages=389-409;citation_volume=151;citation_year=1981;citation_issue=3;citation_pmid=6175758;citation_doi=10.1016/0022-2836(81)90003-6" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=N. T. Ingolia;citation_author=S. Ghaemmaghami;citation_author=J. R. S. Newman;citation_author=J. S. Weissman;citation_title=Genome-wide analysis in vivo of translation with nucleotide resolution using ribosome profiling.;citation_volume=324;citation_year=2009;citation_pmid=19213877;citation_doi=10.1126/science.1168978" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=S. Itzkovitz;citation_author=E. Hodis;citation_author=E. Segal;citation_title=Overlapping codes within protein-coding sequences.;citation_volume=20;citation_year=2010;citation_pmid=20841429;citation_doi=10.1101/gr.105072.110" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=K. Lagesen;citation_author=P. Hallin;citation_author=E. A. Rødland;citation_author=H. H. Stærfeldt;citation_author=T. Rognes;citation_title=RNAmmer: consistent and rapid annotation of ribosomal RNA genes.;citation_volume=35;citation_year=2007;citation_pmid=17452365;citation_doi=10.1093/nar/gkm160" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G.-W. Li;citation_author=E. Oh;citation_author=J. S. Weissman;citation_title=The anti-Shine-Dalgarno sequence drives translational pausing and codon choice in bacteria.;citation_pages=538-541;citation_volume=484;citation_year=2012;citation_issue=7395;citation_pmid=22456704;citation_doi=10.1038/nature10965" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=G.-W. Li;citation_author=D. Burkhardt;citation_author=C. Gross;citation_author=J. S. Weissman;citation_title=Quantifying absolute protein synthesis rates reveals principles underlying allocation of cellular resources.;citation_pages=624-635;citation_volume=157;citation_year=2014;citation_pmid=24766808;citation_doi=10.1016/j.cell.2014.02.033" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=K. Lim;citation_author=Y. Furuta;citation_author=I. Kobayashi;citation_title=Large variations in bacterial ribosomal RNA genes.;citation_volume=29;citation_year=2012;citation_pmid=22446745;citation_doi=10.1093/molbev/mss101" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=X. Liu;citation_author=H. Jiang;citation_author=Z. Gu;citation_author=J. W. Roberts;citation_title=High-resolution view of bacteriophage lambda gene expression by ribosome profiling.;citation_volume=110;citation_year=2013;citation_pmid=23812753;citation_doi=10.1073/pnas.1309739110" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=P. Lu;citation_author=C. Vogel;citation_author=R. Wang;citation_author=X. Yao;citation_author=E. M. Marcotte;citation_title=Absolute protein expression profiling estimates the relative contributions of transcriptional and translational regulation.;citation_pages=117-124;citation_volume=25;citation_year=2007;citation_issue=1;citation_pmid=17187058;citation_doi=10.1038/nbt1270" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=J. Ma;citation_author=A. Campbell;citation_author=S. Karlin;citation_title=Correlations between Shine-Dalgarno Sequences and Gene Features Such as Predicted Expression Levels and Operon Structures;citation_pages=5733-5745;citation_volume=184;citation_year=2002;citation_issue=20;citation_pmid=12270832;citation_doi=10.1128/JB.184.20.5733-5745.2002" />
<meta name="citation_reference" content="citation_journal_title=Cell Rep.;citation_author=F. Mohammad;citation_author=C. J. Woolstenhulme;citation_author=R. Green;citation_author=A. R. Buskirk;citation_title=Clarifying the translational pausing landscape in bacteria by ribosome profiling.;citation_pages=686-694;citation_volume=14;citation_year=2016;citation_pmid=26776510;citation_doi=10.1016/j.celrep.2015.12.073" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. Nakagawa;citation_author=Y. Niimura;citation_author=K.-I. Miura;citation_author=T. Gojobori;citation_title=Dynamic evolution of translation initiation mechanisms in prokaryotes.;citation_volume=107;citation_year=2010;citation_pmid=20308567;citation_doi=10.1073/pnas.1002036107" />
<meta name="citation_reference" content="citation_journal_title=DNA Res.;citation_author=K. Nakahigashi;citation_author=Y. Takai;citation_author=M. Kimura;citation_author=N. Abe;citation_author=T. Nakayashiki;citation_title=Comprehensive identification of translation start sites by tetracycline-inhibited ribosome profiling.;citation_volume=23;citation_year=2016;citation_pmid=27013550;citation_doi=10.1093/dnares/dsw008" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=J. A. Novembre;citation_title=Accounting for Background Nucleotide Composition When Measuring Codon Usage Bias;citation_pages=1390-1394;citation_volume=19;citation_year=2002;citation_issue=8;citation_pmid=12140252;citation_doi=10.1093/oxfordjournals.molbev.a004201" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. B. F. O’Connor;citation_author=G.-W. Li;citation_author=J. S. Weissman;citation_author=J. F. Atkins;citation_author=P. V. Baranov;citation_title=rRNA:mRNA pairing alters the length and the symmetry of mRNA-protected fragments in ribosome profiling experiments.;citation_volume=29;citation_year=2013;citation_pmid=23603333;citation_doi=10.1093/bioinformatics/btt184" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=A. J. Ozin;citation_author=T. Costa;citation_author=A. O. Henriques;citation_author=C. P. M. Jr;citation_title=Alternative Translation Initiation Produces a Short Form of a Spore Coat Protein in Bacillus subtilis;citation_pages=2032-2040;citation_volume=183;citation_year=2001;citation_issue=6;citation_pmid=11222602;citation_doi=10.1128/JB.183.6.2032-2040.2001" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=E. P. C. Rocha;citation_title=Codon usage bias from tRNA’s point of view: redundancy, specialization, and efficient decoding for translation optimization.;citation_volume=14;citation_year=2004;citation_pmid=15479947;citation_doi=10.1101/gr.2896904" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=H. M. Salis;citation_author=E. A. Mirsky;citation_author=C. A. Voigt;citation_title=Automated design of synthetic ribosome binding sites to control protein expression.;citation_pages=946-950;citation_volume=27;citation_year=2009;citation_issue=10;citation_pmid=19801975;citation_doi=10.1038/nbt.1568" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. M. Schrader;citation_author=B. Zhou;citation_author=G.-W. Li;citation_author=K. Lasker;citation_author=W. S. Childers;citation_title=The coding and noncoding architecture of the Caulobacter crescentus genome.;citation_pages=e1004463;citation_volume=10;citation_year=2014;citation_pmid=25078267;citation_doi=10.1371/journal.pgen.1004463" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=I. Shachrai;citation_author=A. Zaslaver;citation_author=U. Alon;citation_author=E. Dekel;citation_title=Cost of unneeded proteins in E. coli is reduced after several generations in exponential growth.;citation_pages=758-767;citation_volume=38;citation_year=2010;citation_issue=5;citation_pmid=20434381;citation_doi=10.1016/j.molcel.2010.04.015" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=P. Shah;citation_author=Y. Ding;citation_author=M. Niemczyk;citation_author=G. Kudla;citation_author=J. B. Plotkin;citation_title=Rate-limiting steps in yeast protein translation.;citation_pages=1589-1601;citation_volume=153;citation_year=2013;citation_pmid=23791185;citation_doi=10.1016/j.cell.2013.05.049" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=P. M. Sharp;citation_author=E. Bailes;citation_author=R. J. Grocock;citation_author=J. F. Peden;citation_author=R. E. Sockett;citation_title=Variation in the strength of selected codon usage bias among bacteria;citation_pages=1141-1153;citation_volume=33;citation_year=2005;citation_issue=4;citation_pmid=15728743;citation_doi=10.1093/nar/gki242" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=J. Shine;citation_author=L. Dalgarno;citation_title=The 3'-Terminal Sequence of Escherichia coli 16S Ribosomal RNA: Complementarity to Nonsense Triplets and Ribosome Binding Sites;citation_pages=1342-1346;citation_volume=71;citation_year=1974;citation_issue=4;citation_pmid=4598299;citation_doi=10.1073/pnas.71.4.1342" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=K. Shiroguchi;citation_author=T. Z. Jia;citation_author=P. a. Sims;citation_author=X. S. Xie;citation_title=Digital RNA sequencing minimizes sequence-dependent bias and amplification noise with optimized single-molecule barcodes.;citation_volume=109;citation_year=2012;citation_pmid=22232676;citation_doi=10.1073/pnas.1118018109" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Stamatakis;citation_title=RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies.;citation_volume=30;citation_year=2014;citation_pmid=24451623;citation_doi=10.1093/bioinformatics/btu033" />
<meta name="citation_reference" content="citation_journal_title=PLOS Comput. Biol.;citation_author=J. Starmer;citation_author=A. Stomp;citation_author=M. Vouk;citation_author=D. Bitzer;citation_title=Predicting Shine-Dalgarno sequence locations exposes genome annotation errors.;citation_pages=454-466;citation_volume=2;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=N. Stoletzki;citation_author=A. Eyre-Walker;citation_title=Synonymous codon usage in Escherichia coli: selection for translational accuracy.;citation_volume=24;citation_year=2007;citation_pmid=17101719;citation_doi=10.1093/molbev/msl166" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=F. Supek;citation_author=N. Skunca;citation_author=J. Repar;citation_author=K. Vlahovicek;citation_author=T. Smuc;citation_title=Translational selection is ubiquitous in prokaryotes.;citation_pages=e1001004-e1001004;citation_volume=6;citation_year=2010;citation_issue=6;citation_pmid=20585573;citation_doi=10.1371/journal.pgen.1001004" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=Y. Taniguchi;citation_author=P. J. Choi;citation_author=G.-W. Li;citation_author=H. Chen;citation_author=M. Babu;citation_title=Quantifying E. coli Proteome and Transcriptome with Single-Molecule Sensitivity in Single Cells;citation_pages=533-538;citation_volume=329;citation_year=2010;citation_issue=5991;citation_pmid=20671182;citation_doi=10.1126/science.1188308" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=T. Tuller;citation_author=Y. Y. Waldman;citation_author=M. Kupiec;citation_author=E. Ruppin;citation_title=Translation efficiency is determined by both codon bias and folding energy.;citation_volume=107;citation_year=2010;citation_pmid=20133581;citation_doi=10.1073/pnas.0909910107" />
<meta name="citation_reference" content="citation_journal_title=ACS Synth. Biol.;citation_author=K. A. Vasquez;citation_author=T. A. Hatridge;citation_author=N. C. Curtis;citation_author=L. M. Contreras;citation_title=Slowing translation between protein domains by increasing affinity between mRNAs and the ribosomal anti-Shine-Dalgarno sequence improves solubility.;citation_pages=133-145;citation_volume=5;citation_year=2015;citation_pmid=26607828" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=S. Vieira-Silva;citation_author=E. P. C. Rocha;citation_title=The systemic imprint of growth and its uses in ecological (meta)genomics.;citation_pages=e1000808-e1000808;citation_volume=6;citation_year=2010;citation_issue=1;citation_pmid=20090831;citation_doi=10.1371/journal.pgen.1000808" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=A. Wagner;citation_title=Energy constraints on the evolution of gene expression.;citation_volume=22;citation_year=2005;citation_pmid=15758206;citation_doi=10.1093/molbev/msi126" />
<meta name="citation_reference" content="citation_journal_title=Proteomics;citation_author=M. Wang;citation_author=C. J. Herrmann;citation_author=M. Simonovic;citation_author=D. Szklarczyk;citation_author=C. von Mering;citation_title=Version 4.0 of PaxDb: protein abundance data, integrated across model organisms, tissues, and cell-lines.;citation_pages=3163-3168;citation_volume=15;citation_year=2015;citation_pmid=25656970;citation_doi=10.1002/pmic.201400441" />
<meta name="citation_reference" content="citation_journal_title=Cell Rep.;citation_author=D. E. Weinberg;citation_author=P. Shah;citation_author=S. W. Eichhorn;citation_author=J. A. Hussmann;citation_author=J. B. Plotkin;citation_title=Improved ribosome-footprint and mRNA measurements provide insights into dynamics and regulation of yeast translation.;citation_pages=1787-1799;citation_volume=14;citation_year=2016;citation_pmid=26876183;citation_doi=10.1016/j.celrep.2016.01.043" />
<meta name="citation_reference" content="citation_journal_title=ACS Synth. Biol.;citation_author=W. R. Whitaker;citation_author=H. Lee;citation_author=A. P. Arkin;citation_author=J. E. Dueber;citation_title=Avoidance of truncated proteins from unintended ribosome binding sites within heterologous protein coding sequences.;citation_pages=249-257;citation_volume=4;citation_year=2014;citation_pmid=24931615" />
<meta name="citation_reference" content="citation_journal_title=Cell Rep.;citation_author=C. J. Woolstenhulme;citation_author=N. R. Guydosh;citation_author=R. Green;citation_author=A. R. Buskirk;citation_title=High-precision analysis of translational pausing by ribosome profiling in bacteria lacking EFP.;citation_pages=13-21;citation_volume=11;citation_year=2015;citation_pmid=25843707;citation_doi=10.1016/j.celrep.2015.03.014" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Depletion of Shine-Dalgarno Sequences Within Bacterial Coding Regions Is Expression Dependent" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11.cover-source.jpg" />
<meta name="twitter:description" content="Efficient and accurate protein synthesis is crucial for organismal survival in competitive environments. Translation efficiency (the number of proteins translated from a single mRNA in a given time period) is the combined result of differential translation initiation, elongation, and termination rates. Previous research identified the Shine-Dalgarno (SD) sequence as a modulator of translation initiation in bacterial genes, while codon usage biases are frequently implicated as a primary determinant of elongation rate variation. Recent studies have suggested that SD sequences within coding sequences may negatively affect translation elongation speed, but this claim remains controversial. Here, we present a metric to quantify the prevalence of SD sequences in coding regions. We analyze hundreds of bacterial genomes and find that the coding sequences of highly expressed genes systematically contain fewer SD sequences than expected, yielding a robust correlation between the normalized occurrence of SD sites and protein abundances across a range of bacterial taxa. We further show that depletion of SD sequences within ribosomal protein genes is correlated with organismal growth rates, supporting the hypothesis of strong selection against the presence of these sequences in coding regions and suggesting their association with translation efficiency in bacteria." />
<meta name="og-title" property="og:title" content="Depletion of Shine-Dalgarno Sequences Within Bacterial Coding Regions Is Expression Dependent" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/11/3467" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Efficient and accurate protein synthesis is crucial for organismal survival in competitive environments. Translation efficiency (the number of proteins translated from a single mRNA in a given time period) is the combined result of differential translation initiation, elongation, and termination rates. Previous research identified the Shine-Dalgarno (SD) sequence as a modulator of translation initiation in bacterial genes, while codon usage biases are frequently implicated as a primary determinant of elongation rate variation. Recent studies have suggested that SD sequences within coding sequences may negatively affect translation elongation speed, but this claim remains controversial. Here, we present a metric to quantify the prevalence of SD sequences in coding regions. We analyze hundreds of bacterial genomes and find that the coding sequences of highly expressed genes systematically contain fewer SD sequences than expected, yielding a robust correlation between the normalized occurrence of SD sites and protein abundances across a range of bacterial taxa. We further show that depletion of SD sequences within ribosomal protein genes is correlated with organismal growth rates, supporting the hypothesis of strong selection against the presence of these sequences in coding regions and suggesting their association with translation efficiency in bacteria." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11.cover-source.jpg" />
    <title>Depletion of Shine-Dalgarno Sequences Within Bacterial Coding Regions Is Expression Dependent | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"_ynLei-scm7xlop2c7Xk6vGiUxzYGnkAnuEvTpF8V-g","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460474","apath":"\/ggg\/6\/11\/3467.atom","pisa":"ggg;6\/11\/3467","ac":{"\/ggg\/6\/11\/3467.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/11\/3467.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/11\/3467"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/11\/3467"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/11\/3467":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022D0alo\\\\\\\/guHjag8vRNgQ0h1J3SpQDeiW3KFZNctJFRsz9uVDxHYGHrbPoe\\\\\\\/vZsrIjQTCTzsE5H5E84cnpbvY8Ecq8NlHhBsmafklZprmEdEYYamOv8wNquKNQ4+m9jGabTx+hgxvQSSRquteH0AqHsC0BUcKAWN0j48aWYMBVtLixSZS2afEhyn43lX548DoQM\\\\\\\/qW2Xpxvn+cAqTVj\\\\\\\/CW8bDWLmUB7GdD4aaKiQh+vYnQ9MF7fKvSnykDyP0U1cOiMWlsU1f\\\\\\\/6EpB2WEw7t0AQUTzVg0KsIg95TuWXp9qhJsisCFctde6RA2NdOWM5W24\\\\\\\/iPvpMd6UQbmnr+sSltuJ3BOPV67wHlVZlsibxc9\\\\\\\/Bvz0KH+dJwzBasXNK\\\\\\\/I57b8XK7jLFBEvcC+VETZj0sfI5Kp8Y09jXgXIqRsV9UWVlnXD8IOCqEEZ1u6F\\\\\\\/fvff2Qm+4qsMRF21\\\\\\\/LPayg4nnUO5ZLbMnboscDoxcy71c3ZmNdUDPnB3rfazbZi+xXBmLYVzEAZHi5gKNTi3OWu2fAHAXeSohglgKuQwRRoh1nfofi0nWUWpASLLP1NmvbUJldKO9mFaU8kRCNB6a7ecUYOB+w0wOMV93lXOgYSBl721zliq0Btt\\\\\\\/0+hwUVM\\\\\\\/5owocXaZ0IRVTv5owkitsHs6Kkpyb\\\\\\\/MoWKSRY8Nq+gjLQ3lI8S0Ng5xzHRIyww\\\\\\\/c0DBEiOBHFJfhpWANjEqk3jhwIPAXhz2Fdznz6L2BIUDcnjgvMUB1XgAd7fm7moI6Ff8qQ5zack2pwMLdwEvZqyPY+WYTeqpBQ4BXJ\\\\\\\/FQAf4vTWMQJDUz9M60OfZTOMzg6E\\\\\\\/OPrL3F5GhTTwPUbqzZ\\\\\\\/+59tGQCjDPlOvu6Q6VRXObo\\\\\\\/N7hxkZ0bDz1Wfc38LCZhiGTACQu2ROa\\\\\\\/5U6JELUlDqszrYjAtlkQZuNphdDY6RzbHvLHMxSzIQJtmENtDg\\\\\\\/qzBAfaP4xxMALBhVqiqWlhY6Kk1VbjP5Blciyi3OITUw3Zdg33RrKnHDyxRMimxp9J3k+Rcd4ToB5Nluj9UlUZAEkXOMyFpVxrdIbBYQDJVeXaR12GmxXbGr7CvtdErrIPG86e0U0pc3KYchftyuaiGREnUQptKI6A4P69gnxnIepw0Zss8fOGTciC1iukDSwE4sltHCGXZiBzdX4\\\\\\\/gOZWephy8iDRG8wX4LPmAGjSMBkh73h97M63F3fbrb0nKzgDGbIq5ICJtiWM5K+hPy6OB3dNLFKhw5HESw07NQg0uh377DjLy\\\\\\\/W2AbxcHzsEO2qmwv1\\\u0022,\\\u0022iv\\\u0022:\\\u0022r5JT1itZht5f5kC1WoE\\\\\\\/Ig==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227eba6a46f12f12de0ad504079211e6723860fa1b9b6beb678e2d2e4ae86147ae\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00221QhrlSHUwaIJFgvV+droxTmXTLv1R+IJzqA2MqDYBqCenolOtjsOkwlyeQLGRp8+JhP2iyIVMidjBFsLeWJwMGIwHFWyOHWWgsdWU0+IUR5RJIKxnrkzFKPc5IjxZjli7p9oRLB\\\\\\\/BtGDCDjnATwweTKe5rLpa1oe\\\\\\\/5q9ygTy5CSu0O\\\\\\\/HrBuD5c2rQThlo\\\\\\\/oC5Kp79qYDboOf3zaJ+sWZBpS33+BnDSpvO51jW04fRGP00+z00+4\\\\\\\/GmMvE2X7LQYeDm1kMiTw7I5\\\\\\\/\\\\\\\/9\\\\\\\/Yd3oD4n1UVKy66qiI8xS55k+O54DIzM18yhWZsn3pCoCWHWULsj6Zh3htY7athE2umx\\\\\\\/9kn3KAgKlpbYzySPW8jJ+l72Qnremni9oNZ\\\\\\\/BuxQ9EbvSk5yhx1jZGL7JcQ8V9mwkOeZBNzUMNdvyN0vWD4Od4L1T2wgFKDfW9\\\\\\\/zsjlaG9gmAG4MxNpUAsF5g7PAJ451jmSuGG8M9R7uAapiMYFy4nUS2LQvgvR+kVZfpXjJcbYG96rOEsyoZxhA+aNbTo4aG3ZwVt6vhDfzcxObO6ZZl4gdFhH6e38xgSIpkelg4NQFafwmR2q4lULCa9L2zeiyNa\\\\\\\/ueJ1lc7JeDyoEN6m4PzfsKB8iIMQlCbG1pgXzGGxaKPFNedIshvWO4BgT+9EEcV9F0NJ7CtevHNwd+\\\\\\\/Emexxthqprj7386Uo0Xq2I8sybUFPiK15KHgud+yARsaoABo9yUm91h1xbGXmcOwEjq3BhZl4rplSRjcfuZNUqUqx0gH0zsrA9LVHRMLMZFgN2DS\\\\\\\/Gs9\\\\\\\/MZJuux83Hcflrr5zkcc+E26bILx6taAzbJy\\\\\\\/\\\\\\\/fkX0FCImmmJnqnPKjfuwh2UXJAMq2GvYomlmx1aGYwr7c3n9z1MYxTieuoPcLk11x97o\\\\\\\/5UDNnGKkzUKDgLGdpt5RoUaRBiPfmJfaUCdHlxYqiVETrEPBRmf1vR+hOomgtM1pTCkz4k6ZNkSQ5avrPBr7NlJXZaMnoj6euK+sKTCpmCajX96pnBRG0pdr\\\\\\\/9EOUP3V1QI4AWVR41pE8PRgEkxOqwilEirLYbeobQ8Ii1+LACgxh9CZ6EQ8bAqHjJzhE0o6cepY4fp87CafqDbikrHc4FJIlQW+BLR\\\\\\\/6m\\\\\\\/eoytKN0AKzL\\\\\\\/suJcXJSuf07Z\\\\\\\/gh+9S2OgdGrdnOF8LCfo+NA2MYqZWnG3XEAfUdMEI\\\\\\\/N0u8k3252rfKw1YiztNbLX3hhdWsdyWySyv4g\\\\\\\/6am2g1vR3rUpMZJrOaq6NT9UR0uAcXAsBouVCyj6kZtq4TarnXmDBUeyM9xWDGIVlYO66RbqFs86DwIVMlVd4ra9IVew5bzJ+49duu5Xwo0YEjlBhu\\\\\\\/veabg5AtGm977Nkk5nDvhdf8l1NaJcz\\\\\\\/VKvGd8uHnTb+N1bsiAvNxHud31h1804HOtLrFsBJWFPgoMtj3H5q0w\\\\\\\/K58BVKa78a+Wiv2MSXBiseLY5rP8D6pVOe9kP6TyepUUdFJ01IoPiheOaqWrhDqigsNt8Dh1Hs7vCYm7S\\\\\\\/2Y0Xpt8VH\\\\\\\/knmWgZE8eViUE1AYK0JGww6ABXSCxdC7nTipfn63roYJVLQ37J6sYT\\\\\\\/ZeT3smNnCKYYJXK\\\\\\\/KmPtXkMJpKERFFfsASSNKncPbxFI6Tk\\\\\\\/k+vr36yWgP+DAJS592YEKxlPIH+5ZLhiQc\\\\\\\/bPQgf7EpMFtgGip1jE5sjwuXqqdlyG6mEZ5e4HUxBnz996YD39VdypyI+MFIC6pdAuIGouLWzXVTOjHfgyJHx4TVcLIFhlrtGH\\\\\\\/7PokcRLgcu39fx5k58+qB6HSyYmelHPSeZ5mQLsO+M84pW0\\\\\\\/hq6Vul42\\\\\\\/yBhmmq6yZsah14XGKyxxdiji5yzdQLSUybhXV0rCsdXPXbyTooVW3U\\\\\\\/eeJUGIbCslJXMJhmnZBm1xbQyHpPA5\\\\\\\/OTVghu7+OlxaXBdw3orfHMr5brCF2B6JZYqap\\\\\\\/cr7TJrmLsXku6jo5aAj\\\\\\\/ydIrWE8r0hWu5AHK2ZjUk6wEHPZsder5qQ8VZ8z4Nyr\\\\\\\/2lZw+cMsO1Wzjp9i+3suWFxAnoDFgI8K6tG6eXScg+Jdbc5FZiV1icAZ76oyQnexhgXFvlnH\\\\\\\/sKUdLFn\\\\\\\/7syiNUXuoopL+L7h9T019mIqDXe\\\\\\\/0oHlYVpEoNt8sYW3oN8qWKyb\\\\\\\/1BJAeHSbqUhpC2PojHph3V6wsACVS3SdTSmxuXRM\\\\\\\/zL+YIZzW5rkaJKbA3Yi32lVEcbV9nTgmO1ypk0OrCKwbCMkNFbouqGrgkfSjwWrUNkmZJMCrbi8ugdVLn1wzLHp8z+tVQ2kYrX76wTZrUKU4nN7vUz334aGcvpTnrrq+FAFiZ95ysvC4Oeqc7lvKrlFpUMmSZZuYhr5GRl14n1u1lQuVgcCXeYLW4X8+d3giTDq568Ddii6k4UzOOO9oSpC6pvmjO+KCkIPWos08mjV8dGe0pSJw9hoB\\\\\\\/HC+irLO\\\\\\\/k+Bg0V4Z9m8BRcNK0veZGxObW7Vb\\\\\\\/Za5mmd6apO3BxLRpcREErOdkMgDdJ\\\\\\\/YycV7dUWZhKOySJ+50ee8pVO6JO6lEtMPlia0H7HLhOAwXvChlGDiBLdaLVkyY\\\\\\\/uKeydEoSdx+C2COi61R74AXZypHRDt4zq66Yh9rvnN3PyAbwvt3+g2flxMiQisY\\\\\\\/JacWOP8f7B4cBGk26RVy542l8iOxLv2ClaKpoLLdN9ZqDM3TBwS1SUM6u\\\\\\\/0gn9Tu82BY3frwjtXD664TYvzBzn57GAzoWdTOYMr46n3oSCJj2Yi0Ln50xc5\\\\\\\/KHk4XVrQLxOHMaXFuUvnbuAt1ivooescCC9u55loccF2hIPoLkSWW6QRhhkTMhvWRlbXagtYXZLD3JakSCN7YhB6L2idLRx3opLIiw5OXrgLxS691B0BtxRBls8WhaLg64GEMHhrBycJsfmwpP6jHYQffubXjm9jmGInHt1trnagC86SFWh50NiYhsmnCASvqsjqiPXJtnMfrXCtDVhS2srEYS+ijqAo9c2lmfl\\\\\\\/QbSWkY8qd4LVb5eh4wYCgdgo3dCMf2MVFerrM7ZdWQIJpxnAOq9ui7Lg49mWA5jT0gYs5b5lQAhWi++J\\\\\\\/uVh36I7oc2qBzXxSgpyZcjyixxIu6dZSaPnUDfqO++O89ByY1Ga8S\\\\\\\/TcAX53ke8wsKiX2wNZA7SQhYbh4npwA8QHWGZf\\\\\\\/U1nVYHuQnkZ+xWwO2l36sj26A2CnhOBvZYz6QptvVzzutlFr1PKHHg\\\\\\\/3J0ealKf4RPT4YUtUWeTraGamumnHtyaFp2I44fX9O5vACudnKisYhrBOdRkLYgJtZgmPBBl2BZp4BmYS8T9NYrWmNv0G3CjRMT3aQWXhTEAGDm0960ImZ1ZRypYoQH\\\\\\\/lL3iMDO8vsNdw5MP\\\\\\\/Oa78S8V6O\\\\\\\/r0P8qkmCtAzVGBjNXluSFm2LPrW\\\\\\\/7FM5utLR9LoEJKJK8NGuIFqVzIHpGAYec\\\\\\\/GUvAASx+5g026qgzHZ3U6h5JtGVctZ3JHfRlZ9EsTuZJaCGE9jiCmZwKEc7sEaG9iqjNJpeJZigxr83NomaHzxToV2iHFA4MX8zeyTRVFgSzLPqYC1Q\\\\\\\/+\\\\\\\/C\\\\\\\/FUE1koCRR47eadUlTimm3tOOk6upq+dI58tVsIY90h4ixn9XbImBqMEyBw2eK\\\\\\\/Sdotmd88Xfvj3sbuVdwxfhTmHn8dkS3WPscV0i6IjPlZmI5XMyRA2vwxWxBblX+Vk0\\\\\\\/zE1VTLO8cpf1+lDe7y8F0mJ2IiDIC438roBf6YRXXxuP9VAsPFQuLYi0EH+zOxeZw+XldC5l3e38CpOL7PmlX5FRYv3NDKrkvKN3UU8n5Jso7Oi0fB4f3dLYuj1ZGR9OIfQpqCEsjmGfpADtPgCxXsWDKgdTY8ncnP97+c5\\\\\\\/EmF4c5Y1+aSLPuhd+oFKs75yfAm9xBRu4RQpY\\\\\\\/qzNrpC+DITrFaDA0iNEGIgz6ANn5BdD0xUWgWwact2M9Z3RNEza0kfcCrvNIiDApT8kmcG9LiWuklW+zbd0p79gpfGzosB72wlj+2pDEFYRi1muiwzmnc+qUUEYxs1EvYOoIBIiG0amu9kzWg98308k4CSv08GWMJT0ijDq8sDow\\\\\\\/BNDzSdJ7hYD60eM2S1ynM4jPbOz++vb\\\\\\\/KvuFMWAUROY2PB5VjaKVF52aBLoFv9\\\\\\\/TPWKDBZC+J3u8aiwfdIJ0HcQWrCQNEigFcrEEOEDwyiCUigLCwF9va+hCsgPhDQ4dEpXbGbEIe1cGSqKCO5C55bHyEx+bOLhPTrUtG5KGZBxkgZwXM95K1rdOGt4stVgHzR9NrdNOPblEwOHeA5S5K7VGkM5pxXBJFEtNBkExF+3YQdaL9sixnRn7FvEOqRcrXZYjDEM\\\\\\\/mA54vcCphBkANj3U2bHcXAEDdIO7vExJNiCxIgqc0ixsAA\\\\\\\/AywmTDAvHQiQ10NeMn00A\\\\\\\/nE5uKTbmh\\\\\\\/DdUFwhL7zc6KK2W4j14myitlqv4PPfYjCsM\\\\\\\/zLCfsz7EBQblON5ldZsx\\\\\\\/tOUoqd0bUCVsidY5knt9FuOxHOT4lNCttv69Vo+A1tEsIj7fF8c+ca1scd8OyRW0mHhZOPZtFn96G8zC\\\\\\\/5QWIwIkk+LK23MsIVmsXEdXXpHyvcV59+laX9IPTe5JN9y8C43ErA2oVebrHtj\\\\\\\/1\\\\\\\/NdUs5oaKTd7O\\\\\\\/97pa75x4R386PDB86xyA8PIgRH1nr3dYfNM039uCAkKRYE83UZbdFPSFD4NoOmRdo76pyi76HxI3zAHCNKDspRVKvc6eOyYsTzp0WGbbkvvrS8R0AglX\\\\\\\/69tojDLNbii+DzbFPhSnQtvfNMO9vC++MYs3GqPZKLL0gZfYf6u5TeHsB\\\\\\\/eMQrU2R+RP9vSEVkZljNDMWUCGIJhFkmYPdKUGXu2U2htSCmlB85DG1C0ns7I9qY+QHARw9o823daXGsmAMWEjUCFphMSAwi9hitfzeI5+1ZBBAuunVUTNRs8r3HdDyQYe4ofcfzeeDlG3DuLl+lwlMvijl5BKgxxsI4DyAfYRNe23K6lJbbVJN30ODOQW70iI22k\\\\\\\/zRMBn\\\\\\\/KQB2Dmv+ut9EqxtvukA71ZbzCLNP94byBe1FOyobQ1hB+up3ky462StHTDsnu6pJjjvGfTd\\\\\\\/YWwAss04ryJxBIIpob79AN7QODmJ53OI1AMEYrMEx1Tb1JAKws1ryUHe4zZ8CFL7OQxun8f7Bdv7IvKRqzWe54Hp\\\\\\\/DDSlccyHhCkyV+m4U2ZECsJGDsSl4GXpO\\\\\\\/HgY0M5PFgLDiU2NHTcMciJjxD7+6bzo0NRJlJkVX0IB50OzBjhhKm7kAg8R\\\\\\\/O2W+zZi9a2Edyk\\\\\\\/UJMm4zjgLvDl6XgnFO+CCXyMjk6jkCjZmGZIZMEaqtf6C588Ftf\\\\\\\/H32ZWgundleom15+or4G3MioA4VHpaxutm99qFszuK0J5ijB9oec5i3OCJtnGWvnJoS18T88hoMYL0gWHbHbgnut6YL9o+C4sw2uQ53lWWIo6SmPx4BrNf5nOf11jLinSdRN\\\\\\\/xTEw51\\\\\\\/qoN3WAbNrZF0zFB4qkKtWQrAAGQb2jKFI\\\\\\\/RXJIAUetmPUrSr15ObP9WRkcE+n496wlkzHk2szfCxjJRnL2y33caAEKfpaYKbCQZU84wbz4pUBsp5BrB4XPbmeKmnb\\\\\\\/tvA77FvIX4AOzTWlYWwoGZwzcj3ZaKvUxMyWi0YYWpfj3LYjpX\\\\\\\/yC13mw1+lflspwugq7gS4vhQ6\\\\\\\/1Ubw3xJWToGlNYq9FHYLfFDx4bW6N1buOlEa8nfhqrbnd2s8kFLU0+HB3Qk30d2\\\\\\\/b6aHiVk59gPKy5SBFm0xU2efHWUOrHxJnJw9Fkv7Rl\\\\\\\/sXdOK1Nb4\\\\\\\/geI6+El5C9vsWQMq3g5OKMelDlTIq69YOOaOF3EQJpjgJoCBuS4xjDf0F3UkmneCIxIGLO6RZKkJQMVRyh64CHlm57urOqVNctn2dtQABC+e0DG4OvuDX4t\\\\\\\/FDKom5aXuDQ1kwldl317I1FsojUK7aP3pG3uJ+9qIn4NECtb4XeOCHemUgkxDOlZoauHpO+qgEdgMiFbQgbWArpYTK7g35MZuQr\\\\\\\/MyXTRwSNEYyitIstpcWu+MIRu7sE1n32afyKG9J75yZFYV47fsBImeuySmYiGqSF2PyZcRt32FBnKAPcvTdrpdsCFuGW73RCloGagGK6bfpIRHRs0C5g6bnXOywchmuitiLMBykXMHu6wOgv\\\\\\\/1E9iVnoKdHRiCpWnzF3t\\\\\\\/6lkz08Qfw5sN4WBi4VYtF6nxojhbnkapod6gGeId6OOgxwOTCEnfeohB7XQ1b1jfMxYBGNfAF0n5pWtnu+4iK0pY0rDFpNZFtVnA5ENTfZT9WOeF1bkHO3dPENVveX1G3V2ljR78aUI+hET0\\\\\\\/QcsTcfZNEVWUsBrvnpZgFekQNaVXi7Oava1UF2RXYzGyemDVInLQo2NC+PkvFINqbfTU7QlqCMiyr02d6zr+qZRNEFn+ruIfnivD787cI6ZzTUXSyWmfXvxVDoi4p6qiKczWhC+TFQ+tGRgTAcbPEpqvZSLIdpYPxlP3e7hsqtOwT8zA\\\\\\\/\\\\\\\/LMoElWjV1ufHT3yHNmk47Rf5DGUPh3qp3GSzE6ou9Jwt8o42MfO7RpBcbC5W+Til02K1vJn\\\\\\\/Nzlg4QvL5CrpVpHoQqbdnaFWLKpZA\\\\\\\/ZNFRSp34kkJzDy8EuVhRzGSuZ8MDDiimE+zJOmJ9Np6nm1KV2d2x30xVdUAvSjqDHH7PvUu54P9A979UKDDhPP37hfinAdrWAfgaIN0RU4uo6NCs+5kYxXXHt\\\\\\\/ivxCAk8251K9UsBJcdE6NzfM7ZkEaWInvqTkIj\\\\\\\/Las5bkfLk8CtK6XZ0rJ17+A2x77ErpXFQptShY0RBQtfdJoesLy4i0nVraXf79\\\\\\\/ttyk6nXMmH+fm9dCfGLgLOI7W8l5nEOIQyPnxU5CnMgOeaN5+gyiUM2\\\\\\\/ndAmPk19Xcxi4d0mbXgONMazS9+9DwRXq6DlgrFCsRSsENEm\\\\\\\/v\\\\\\\/MpDAQsRjplGTxDvUrD2gvV2sR4s\\\\\\\/Ys2bLgqq3g4+2AhnV6Xblu+L4RYADjEJi34R1E9Zm343KUORbLOQ1ZaLC9VKmhMdf\\\\\\\/5l0234Egy6vQr6iwOlFOvd2AT9TTMbcwHvI6\\\\\\\/Q1i0zGp6TD4jRE+a\\\\\\\/\\\\\\\/hlmqadoHkJFLlM9CMHRNZxAdyIMPUqJsh9hm6yzl5NX4qmdoJDFL8pssnpGbaZUm5VCjDhrJcVMQTrb+kllkLJdmXjo0u2l1xIJ+Tp4okxi4azcQPBAnN+y4iCbRKfsNl+UXMB4kixCZdrAJdY32PzPf5CshlaNaXTXzC8nyZdj2mEitahiAVhoepIbI78fFQPBVhwPQCB4Tsh4JjaABi43Pr\\\\\\\/qFxpiXQhFvCw9z4Dp3gDq4UEc9uEbxWvdR96wlJDRmRatYzgn2vSepFfUA9O0yYzZPkgLdKGEZiq13c2XFEVRreGw6xy3y8P\\\\\\\/q4mAuY682F0OIzNHaxcF\\\\\\\/B0HjHFy4797GYmgIul0bCg+fSPfTybFQvO2YM5D6e1omEarcGem6WcuckwDFUs+3tEEnR9An8Gjdd1V82XBCbyDwfMFohHKRs\\\\\\\/LRpwTZBvLBt4KNy+NR2b3yRBYLhwA2ZApNYf9vcCEyII7ijSI5llHBxFQqpAqdYNygVCNJQft3\\\\\\\/6MYDaLCyrMhFB5KvAva8hmhdoLz3ZaZyxNBO2IiJPbxZCZP2i4t\\\\\\\/BZhlYxs+B6eePVVLjPUd5Jb\\\\\\\/C09cgIjigU8tA3+5V8JKe5zSUm1dpuwxpKUgLnWcOkyzpjCpuBl+uC6Hi86Zu6Poqtjtg6tlMK9TNULLm1B\\\\\\\/C9TXzLNUJxSYNDr\\\\\\\/W+Y0jP1k0p7FPok0Kg6X3lc2G8ik9v76ojYCc8JdXwPXPK8cye2e5tKYpiG\\\\\\\/xs6VDf7meP2Z53UQemANkGPLLpht6zZvQe7Tx+Zq3mxCmiMKXmPv8eA5TVimFcjknHZgKr1DUgwdtgO+vp1HLRMI4SwR6EWn24va0Uo2ztCGjoZ2eVk6XBy8a7k56HBoawi+PGshDh\\\\\\\/kQq6v7nQxU1ifJYurHqW96QvinhqD422MHiIzNx6Cu8b8xC3l6Rofiy3l5DbCncOQ1GWqg7Sr9bi6LFXJew5e1pXaCuuOzDXIPiyk+NLMxhA4VXB2bz4FvOH4mR5fITNJtOCO02xEztsK1R61uLv74P3gDpLML0MP9a6vi9CgZ3OWGaJAvwqW7XvBouuw9Eoh3zYJ+oAcLMNDDSqYR5BMmKpgokIn8jXHR0rY5COeLiV42lPquRNSmpSylcllVIkFPZf9sMWynlGArRnzI22de9nQywl0KGelDDGecH1KMdskner7XCgdOl6lk5ZJzQ0Q0KZrJB8CgNhW343CePnke+BLTDfJvugrf+1v1IN4LvSyBF7PePtfgjjwu589ue7F2l2ND9j75xDZ1KDVF8OAedEwxnkVslYSLTu6bwqP\\\\\\\/TpcAzD4E9UUnSJJTbr9inGC7AEOD9O0zgQJfp+LFkJ+9ZAvkuauw7UMus9Kmh3fuKdYeGmD0OrkmhCIJiAXGxqcajwBASbHi0tXaxwekDkGQ2VsKYUcStL8ED8m+bh8Bio8WncKYDOr8OEEuCKzHd7dKGFC4Mks1FxTN1SJu7Yr5uewixT8Y4S3XE+0jJo7oiqEVzvkZAYDDzyQKjvYTAeoIT3Stb82tCw9ZlXhVOYFOmDdPaEz2lb+NBDuG+\\\\\\\/Bwg\\\\\\\/iaHtxVTiiv+VD6avKVIwtb65Y+JtQjvQl\\\\\\\/93H4usjCdMJnblGczjNtN0ZQwot9hS8dMq8c9+FR5UY6XITRt0Ey2GfOmAe1xQrLnuc+oRqBjb3QjNNu92\\\\\\\/gowL4Movy2EFzyMRYvzokiksCm8zvTKo4OMz\\\\\\\/muvqANFj\\\\\\\/VZVWr5syi2j8XMk8VZ\\\\\\\/iODj8LvcRi9AQv3yQ67JJYO0vJ4no8kT221d0V9h0oehXAJLUm75kv48zHmqF7mzj+lHfxNK3nUwWPa8DoajSCVSEj+lgZf8TRn4\\\\\\\/5INRaSVcPgYG2cCG4A6yA3vGV5uKab8Z3H68wer0lze4hjO\\\\\\\/eWc6RBg1H7tJ9Bfj+OviyqQxnlG8hYhp38xzdzQHYgwyf0zO7MtcUd1MD+VPBSeqcnwp+o0o0z0kM4hheu5OrJecLftbNcSdnfFiJzB87Cx26cW1k1JaMF389TTl9Y6RnxwWbMxGw1qTVTj3bFrKhN\\\\\\\/bvJa1mq06wprnDYVQ2PSF+NTWCwhrQDHlfb3ZMO7XUoAPPpuCjxymNkiE3ODwoepL0kveCVjR0zB+ctZafjFovj7YuCrZnRHIfEwi\\\\\\\/XM43XfkL6QKOIO7hZqgnciLDHXngYHp60915jWOmubO+ONyfDwvcjtReNE\\\\\\\/DFFqnIRdB010Tq3EJ2rz8YPZZUBchEmp8YcnyZdSrznODmTJ7VFlojfIzUvgBaatGPjva4p\\\\\\\/B6SfFGM3CEYkrUENswZQkPzPkhfFjHt\\\\\\\/1pdbgTqMcjylJf5YyJm6vA1xNEzX15fd15vFkuQrUnyiUv1xXNJg7V0iktGcUYjDiYpMCxXiGr0nHl+2N9XsFXwxRhpHKZYvljUaWUEWTs6OVadmXHgoH526K\\\\\\\/VqavW4+BLduZh05G1H7EymFjndEsFeU7c56ggSYjeYG2AAgfNwim7uSMqjx50BhsINmtQkE2cagTsU84+sMvwwSNwFZ\\\\\\\/Up9JGEOCWh7pLjFAElsz11w1CPuDs34slg0u5gwKl8Szryk4EvBOrRA7ocVzuzCPhzDGXxYYOfdpr3mrSG1hgancz2f2h7G6cDX0GeSvrJnaGWB77i+hDWNOnzERA46Mi0IgmYuQxXSmAbsW+nB4kyX6ucpvqoIBIsPed0pFCG8SB2\\\\\\\/EAulnO6xDKSxsZtnaVOlQeG+CQU7sHXowMC6gRt\\\\\\\/r+mC3mxwrTvEoyqSvQq7ti6ar+tQ7D\\\\\\\/d5RzSj\\\\\\\/X48nTf0kTZyoQw7UH2dlQzhh+uGHKEfvHySsPv\\\\\\\/+aHIbr0dqS2ll6cfj6sJ0YlUgXWVSaIuvVkWykovzRxs0cp73IcXV4ZhCbpq3RmfX\\\\\\\/RTDf5k3EqTErcBL0zK4x0sTXwteN01iw6IxV78oPFaL18hQfv6deXwkHZDGkY\\\\\\\/lO1Iw8A8XBl5YuabH854y5fYiZk5w+q+yVRr3mvEpKBzBudDDt9U6rQQFsPzOYZ02oFfZtNW\\\\\\\/Jd0JEITDC2bsRpxRtjQm+wd1QXdzS\\\\\\\/zWs1MHlhaY3rYTaogdfI0rZRaXxNo+G0oOyvQXhBaYxMZRqoYzurqzI4vcKmsiJ46RFcC20p3r7lE+Cha7qBL6wjriUwAHOgrOVYkDKhnvndLVPUhnV2nYTWijQuHq4S5mSDa+LIOwx+8+2sjHZwaVSzz7jrhdlaHgkjcIyw4Uv3CBTbDAC+r29NRtkwp2i8FroZOephZ532ROfAa\\\\\\\/rsRJ5jizKiv7VerhxfkKnlJjNnszeozbjxoCM8PQCEaOmfrmjCkRAFEIs4LdaTGhRxbXkASsp65DvV0fVB30QjgmMLJW8MgUP\\\\\\\/ihDPbhPv1GkcyiM9fC\\\\\\\/QSIFz5qX\\\\\\\/YqxruldgkG3cCffcIekXIclOlYAkGRFqc10sInfmezjJ\\\\\\\/y5+VHML5698Pn45jMJlqVCuRjDQn4UldrFSLEBcyux0mJ3+WBj8rTBwGpsPKdXLiv3zxBxvhWlF+4vycnAToQmzDgu6+lW+Mxloxs+nrG+bg3I4nkd\\\\\\\/bYqevFkcFzWapu6YBl63s8Ik4S8eNvQYdBrod1NYtumeisWANWjFLchb+qDhh5P1c12dRjzz3QyrmBjdFl46UMrFNm5UEQh1nLB5v3n9BAOjIGKRhwSP3TtODuKl1D7Q6Lbk9KIqEiIM90FLpzr11\\\\\\\/lveJj9j8jEU0NOZHpc9nf7JCNK4KjWfvBz+8Ga0pJHSw9gD0xv+X1R\\\\\\\/EjxB3fICT54fL\\\\\\\/Civl36ukKnEBPHjRP4KDwlcizXq9eEV9RBMDoB3AmimZGxq1CmMn8EShYBENjiNFbJVg3kIRhlnz2tlXKBVilThBpNQkxvu2zgpIc38tlT2k9mVv46sAg2kb9paXMuZkXr6FKaGHyiCMrtqrsZ1dmj+YKeIbdfVYPXYbwbu3XopEvGBFhVlTUpZU9UAd6IybDSdswWaARzxfTGdFP\\\\\\\/Qkl83Kp2p6UkVga7W\\\\\\\/Itd78uB\\\\\\\/R\\\\\\\/yMl\\\\\\\/bA5nKnDKTuK8WA5krwZ2l0SRireONbiIyex62Q\\\\\\\/SksklmnTL8Y8JGB+wpkC+I\\\\\\\/ACkbyvsap5orSDNHud333+MAbhzLHhZGIr1uqQgbccPeR9Kc6jN4vrGH+ykb3L+Ke\\\\\\\/VH6fgPzKeHdh8YuhU8yJ\\\\\\\/XAUFhhlPBkKwk\\\\\\\/3v6e85+zH4nNyyGzRySXNRqMKzHumzjZ\\\\\\\/DcfCP2d3nFPFOFyQ7Qk09Tcy6FdFkOPXl6KjiJjLMZsxJPqZBsPWS0X4e8U2etn+lIkoY+7ckTzo28o3AgE4INaXaRtGgewYutYwH33MqwJxX0LXSCiF521e2oQTt\\\\\\\/nnVJEheYA23S1a4G6gEVitb2hsPuRv4d2W+E8Ibmzyr9D8fOg5jX6Y3sDHZJR\\\\\\\/9nveizW4ekoc4vW6sGSOdb9ED21iNAJw0EyAUpQiUq0w6HHyeHkWP5dGexpcEQIwV81RTlnDrqhO1XEkAad+YQf5PIp2R2GiU85xssLevrW66A6rUTenyn9Fi4C+bj25iYGNu2iq4XAD4NiQdEIjtFEcTDaF1ZcnLVP25TskWoivzXxco4kDTfOwtcvoOhortgB1mwCoX46RH1mCfgATvGLZniQPHNEu36a2Yv0xnJcBt\\\\\\\/vr5fWEbPGbSq+kuM5p3WRjr3PvWsdXqtjJgowiRT1kqAok8xV2J3\\\\\\\/gcfn71W4kjW06i7SnSW9zX7ZTzYUqAqIkP6yI929MTdsm6OQMwhCeiBuiIGS+ybaaUpUH6KYoE1ms7wTgAilZBA7N+oL+XsJlj5BTVhxUd3KoUfasvAON3E3ac5KvN+yZvdVpSBkVMBmMi6dJEL7bK4QisWusjK90pYmXVprLxOugVNxFc79vSp\\\\\\\/nV6LUTh9aHoxzBSIJ\\\\\\\/QlSDHgMwpZNctJWzdlyP3os5dpOJ1fPgwHrN+YLLGBPiL58X4IQ3NzfqQCr5kvFwjLAq7xNrJmM1VlTQVgMCQ1B1RZCTqI7+f5oMq60uV9H0pqGrJnl1ynrbljmwZRZEsfmsQ8723wbCClYYQJAcvOW+JOPqWIHFM3nysT4tysRGT3yDvvSioHkxcNfV75RDWb3wcbkaTKIuDTPUVLIKPGSOn82ngZFjjTfN2Bp\\\\\\\/Utn1RMEVmzCTrctjIpzkZVKD8fTt\\\\\\\/lHKSlsa8o2z9sw9N\\\\\\\/H9YKvZJArIVFeXIBhELT3loHUuWRm+4JgCQZPG64NGhZjGSJL5JC9Dr7bvvd5mlgHfBO6K6I+kLUgUo4c03wW1QiUQpwCDE8b0Cg1lgm0XwuL4JZ0oRZchZWFXryKepzqYB5iTECp4x1bX\\\\\\\/vHH9yz8\\\\\\\/xIcVTwsq05zH+9koY8gMwgfITDt8fmngWhnXhqrO7eUepAjPgHkpO\\\\\\\/3YAoEcfJH3o57Xq99VpFq\\\\\\\/GbXhEhulMccGuHPxhrdE\\\\\\\/bquNW2WUZkMBikCnsDu3B7iL\\\\\\\/KIDHTaacItGrZpKzSJKbrMdBQFWeIrqvhe1HF+nyyhf9NVvvZgXNfdWHZeHV251wYadv9nVU3G1BVJ4+Qou4ujL9JmvFYP0dWmN1NqBrHrVGUfwCa9WDqa9xwD1ErZl4lV4ceRFlktX6FPIOy3WfcehwCwf4MdviONlL1ZaizKTfBOGMeR7++ZVmzI1j3KQDLRTbrniWHj2ECmJCTBr5W2ygh7IGd+CbiYspCje0UQhuPXTwv3ihSDLpDDj+O\\\\\\\/kCfS5D4gGBaEe1RxQVPCjhmxVPy6eR3siC9kBhKUQuvSUxGIiWuR+iAm\\\\\\\/l\\\\\\\/HYJwvJXNzWYIX3y8P+\\\\\\\/Tl1a+2ot76y2nnPN+vez+DXF9NcWtMmcvpe88t\\\\\\\/sgVPjSL\\\\\\\/cRBhioFd\\\\\\\/eyP8rKQX4vcZLY7K8WX0ciNT8IOopvXs2r+hsxVbPpVCwmbykJNTZUNd+x63pXvw4gskeN4FrmZhDt9NRo48HqpT\\\\\\\/hJqxqF6csJLmjwdUCi4TM8SgDGVUODk+NPy2uHdCyrBkuhUntvmOXm4vQy9uRl3dQTfpBKXbVFxjrt6rAqWDHybA4\\\\\\\/0qCUXvG8TBGDSX4pgct\\\\\\\/tBhLxoM3L90cpUIOWz5CDZrxXx9kOXDuMUnep\\\\\\\/sYKmKZwB7mRozaUQoPXvc4uAzLY3CopHvR1MQf63RubpqR9m1Qh8Zl\\\\\\\/8Wpzfrizk82Y\\\\\\\/+JX0R3UZo4n7YHpUy8Iwm3cJUU+wFuih4rPgMof+KJhK1kk6X4M7rNC7sch+dW6JUDh99k1PDWxF6Bcv06NiIfRzkHL7BJyNFzT44yFYHPEn2cHpkJ5cD+jW9pDeKmsTaYfOPQ8+vU26JvxiUps5IADQdXQRhxMZSW79kFfcdFYIHal4P8QoorrnO5aXwVGSf91adqV4ftsTxMWPmPeNas7CApKH3e1iBrRnE47R8hCtLjb909Dmm7BGjpAo8z032AH4IoOVI2WGFi6EQyoh0GxcNX65+dQL3kN2rxotnc1KmMplTHXW+GG9xpBFHYuclNmteaVdD99DJR9EoLSmpY3yajYMlaI4afsA7gTTKbjefpnQWVB2PGojnRXFuaFpBW6rj1MoRU2sFUqSBc+VkqOQ1BSXcNzIUE7jCIJdwtQT\\\\\\\/i8YYmaQivrm21EtnB2CeFSI02QeU\\\\\\\/eWQkWW\\\\\\\/e7mhw3l+h6UG+K9hHh+skOw+uaR1RK3tnw4Ebvtq4315qPFxN5Fs1eP7ZB1NvNX+nQkmxiVBInVHGqsVvlauwmik3H4BFMvmuIOpGBDZ5gE6ROzW2HEJEZDHGEjUI9odMqOLtuXs6l8+GvmCxd\\\\\\\/so41LvJJnhVVEon0Qt70as7o96dhboKp86JYX6XDM7gsfLtFnAPz9DXPSqM6QfL6egFEPT5Xrfohus7aoIcR+c\\\\\\\/wT4j1+Ln4LPRErxHlmsY+Y5rKnho92kc2SfDXND0KPKHD3teKv1+g+xB7zCXOCKDrBEsbaTTKqhVAmCgrasc1NQ6\\\\\\\/XCqb5ZwWKUa88Ex8F0YqBquFFxvrTEekIs5xM7b7hMneUKJvz4ccyBFI0X2ePND3i5cE\\\\\\\/hv3qq+HCSZ8C287Ma031o+RBklKWFPreIpikMH+n77\\\\\\\/306rPxz7SDx76hbt0CvzQwCG42t3Zk22Zz45mjzbdxACstFA0DQMygv2MIaBlz1etgU\\\\\\\/svK7I8kYUQMg4HHSgFvD664KVgrpkDedWcVUFEbBlB\\\\\\\/R4EJ56y4yr+5FVjd8scTw2B2+OMq4CgoTU7ytbT4aF2BlpmFZTneu2HxUA\\\\\\\/Ekl++soDTl53X1wVhbxi1wJ2EdnHdIGDh5DVAXBhb\\\\\\\/e8TRFFBTspKsLqtqqu+b5i2K\\\\\\\/yxs5r2rJkB2NDXYiq5FDjGVzm\\\\\\\/Vi\\\\\\\/zaAq3Tt814RMGNcyVX+ahYZMLy66eWY7NjLZkT3lerWFvv04c+mKhM38\\\\\\\/h\\\\\\\/PwTNg32g+0YRF3tHON2ZtXrtWydwFsTSHMSFqy4r2MO6rowY+LaVe1YgVsPVCChi6+LFpo5ox5+tcHYbbuNfr0As2zhzYm\\\\\\\/6DHvjzd\\\\\\\/9J+aEMs4xhVCEGuzgkWKcy8RuWwq3kQYWHJ\\\\\\\/vXTgagBBN78hQ5OxBuHX+Ohlty1DO08DL\\\\\\\/uhu5wDsiSGd7UTc6\\\\\\\/GfF2\\\\\\\/ZcIXf3WiW0nH9cITCzGg+juBO21gpwREvBE0UxJWj0MEioj3eYBeQPD4+ImXukZzyZ08GwKmFCX5Rcn6XYR08v0Pi4aOHUxDSgf263lbyWCy3PiqoFfwMZp76YN2dZLHpJzZtUCWKEXA6fGxlDalw9gCN0Z+Vf47qh9Awzbe\\\\\\\/qrt2Hydzt+\\\\\\\/Z16LNMHFheg0pHbbxc6hwZXHVM\\\\\\\/KKDQy0vdi8KBWghuKO0AkEbRkew6RZ1lblM7RyiB0bk6qPhF6eGK\\\\\\\/NDBa1Ok6UMAo+VTTrtwWXBOvC1ltu41n3D8vOknvtDG+qSAZDmzIX4kkC\\\\\\\/lMozXa7monlb\\\\\\\/\\\\\\\/LUTHIgWri7HwCl6iSKhEu5Ai3F+EH15fYOvuads6Q1COd6nUvyUDWjVbrg3Hrr1PcGcvQHkciNrW1l505U89HscXaUf\\\\\\\/JCdustR5BZptEA1Z0XejIscbwp6SJdQACyEM73GrrffSxU4+GWFeSsA4R\\\\\\\/kK4SRrf\\\\\\\/7Gi16EiO5d4Yxqtj6K0UkPmTruCAindLA\\\\\\\/7nfCyJk8GPmPkAzBsj\\\\\\\/7SV3h6lnTGLDY7cDbP67zTde\\\\\\\/qjV0rwivwp3qPSqUWlgs6AiqFP1SznkqetQwTMpeKFCCx3jHvV5zpBQmU1Gwb4xcWJItJD0q5ZDorXe29ECWH5oSsVT6AXgxoZUKdpATffIZTxxUY2VNF0ksDT9aAbCkiLY4JecP+rlnT00JqzK4R3Cuvf81m\\\\\\\/voHmCPYA8RUfIiiZsUFzVuUiLAYWoh+3nMJJ7tzpXlZAnz4o18E6yj2eTlU7Wy0kV3bsyBg7avrx7fCA6Pzy6\\\\\\\/uVeJnxEioN+rO3qsjh+q1HJ4rgOdy5kunGu1\\\\\\\/SRu8YqQYPyXmqbjRubsUuz6cjaJxHrxCNmiyhBN5M4VF8J2gVgtQmVWyls5n7uw2WABvIp2jbmqNZ9JhS6NjFWQuKz9sUmKtxj0\\\\\\\/sbF+TdSwS2i\\\\\\\/jIJlHwTKXQcilx8NtL+7fFniq+Ba39G5ybq8MzFsehGL9MqzUM\\\\\\\/S5xcjhAIXiAwlujiDm5NTpXwXKA5EXzo0y4UiNRUg0swfo9cYCykvW+Hm1aiTyCLGZxdZl6cXYjuqZ8LyItmCkDmeZomUKadYb1QV4aXfEyxPXioAkGkxcME2Jrclol5wcAG5aAMJVmmkFiBfWGo0UqrVQ7F2aLQMY3QSTuOLw4hR79wiEiiwg8ItCjOrFLg5+ZZl7v8dxaYfB+SYwOFkD3z0oNekey\\\\\\\/6sV9kljS+iGZEmhnpACfimII3MWqlSTDeSD1tRCvhQn1a1YQG0eXCpoNZiZmWaM+r6W1PGhZdZW0Xp14RxCIw\\\\\\\/jKkR+Vwt8rbCSLDR\\\\\\\/fr+KjdOdQxFpMv9V8+o3S16BxmbxRmSpb7O+Ay91v4o9yzESAQk3qQ4ej5RasQbh9Ef8e07UaW+7AGWs+iVxEk2a\\\\\\\/eSc8MXJ1RyVRb+ASmc\\\\\\\/g2zpdniQ2Qpr3E2\\\\\\\/XLWKvfymBSFBccXItw7hweC0tm82cLq5aaX2vkLqJdZGgkhK0siEQtFOfAEPGJddEXajXie1gCt4590km5P7sNIlyR\\\\\\\/bY21whH\\\\\\\/ZZ1Pmw68fUDQQR4wLQE6xe6zx\\\\\\\/l68aJMq5Cg+HlOAO5dTXCi\\\\\\\/df+pAOelauH+AX26Uk6pn0XNqNhXfs6Vzf6XvXf9FeLP4\\\\\\\/7U\\\\\\\/UIZenWFiljLo96navXCJk0LE3mnprSeJsXsuR2wyCwTyGRBicGEM\\\\\\\/4IHxeXWMXnY+1lezPe1+e46X7lqW068JNF+RlDJba0Pa3Kx2u6l\\\\\\\/AvIqfNjKZWPB1Z6TQPLcID6+7yiqqFiTHSEl8cU\\\\\\\/Kxl96TKXnm5kpG9\\\\\\\/guFlLM\\\\\\\/y+Mci8DNTPB+eGgCxtVc+OuQVR1OYo8iDw0qXSnBtXlxJBR9iJT\\\\\\\/5SwNm01pY6LjcV3P6aWKw3kKl5C5vzQAefiNEIOIAQs0me9PB0BvOky8pMy52OoEOAZC96lhXnZ0GjgkX2K8B5cO33FAxR8wDJBYxVvsGCe+kmfFMA94gkfci9BL0tv77tHj\\\\\\\/JEGXdUSWYKZX2Ok08+Y3qbFJPHDuFz8kgaNhYqx8T1sq67aa\\\\\\\/MmyHxsM8\\\\\\\/W\\\\\\\/DwViBxJifdbdK\\\\\\\/egVUp72pxl0yrwdh8eOtMu9h73El8G6xhiOPED9+tLsLkudCng0hQtrom0aS7OGl5Env7LbWl0IbDxGBifDKasXxcefWj9UKduiLtmn\\\\\\\/Vm0sqdhSomGIZ1P4GOnedTiOwaUBR39WOTgvsp4uhtVbZSDd+wM9Wsu8h988z321NO5VawhOx9VMuAuMo5M5e8FP+P4c8aw4PdK1Q+zrBM4dchMsiJBOV13Kr0KfTBGSx\\\\\\\/S\\\\\\\/LtU8hdkuv2b9JplfCKr7lgTmlXQsPfUnCGEgEUAV1BUTFUWk5KS5XfIQFlAHxB4PYnmuxlrF9XKGg3hJYC8q89L4zawfbl7a5mrNYC1GU0KykRqdZmaC0IW38h3Tm6Drqw9kRq+\\\\\\\/cn5t6G6PhTShIXW\\\\\\\/9EoKXI1pAv+p3n4VQBmnsW5Q2nBQiSaHmeIZ8nH2blVLauj1i3RgMwswbZAconazQ+JWSaNzPzjqGC+SIzfssbBJ8gJAop+1\\\\\\\/3gmlNRml2iV5ZxO1V01LeNYm1uKeVhr6HBIrh+iugBTYgXb+YOU8Bwx9kDJXKaK+JFT4dV9SxXdZN7isb0Gqx4gVqirRCVS0TRY5rBhVYOPDqvLFrnZuy6RmYIrlFoMsWu9gIBs4Uvb\\\\\\\/aa+VVldWNS8YCg\\\\\\\/OUywxG+FZJjWdMuTJZ3TSq614kHFWGubK62+Bg5P7oeOnghxRKfcXq7kKCXU0enxrhCMF87An2do84h0eeKpwJjpVPZciqHC9nJLVZSVLRr3hbNtTZ+0MyOuO0W1D2cn3M4F8nlipULeG2Vmi98F21XI\\\\\\\/e7iYuYibcnnGVLB0A5A3TfVBHZhpApwSSygIuIWUKd0Lwc5puMQLZDqGhoAd3v1XBMdkF\\\\\\\/RIu1GzDciY3vqPCyoLpIFwV5HDWJmDwP4CXFpu1jl+GAKUGoys0uwBZRj+Xy9iANSuoNODRpzsh2wAX2oXpsG2l8uAuxZqpu+a6aqQGKGYXGATeA90sIUm1T23gaJIlt0tihPIGuMsO1\\\\\\\/fSRHk8gxArKWOkibvqKU7z4ghpjAGtw1\\\\\\\/b1ov+WjtLZeiBxTJeox6rhH2192CZz3KWAgxsz24aZgStRSdpy6nOUwRMXby3k+etLeOWTbLdLiVa1biuGQMXCtp8ujSWadH3WpCwY1fWCOjVVPFHnImpIdr7qMK3+Z1ZN+7WMgYUwuYXk\\\\\\\/9fmQ6DUPTGZmWb26o2I7750miD4NGoqL9nE7Sv4v1OsUSJWhePDtoKW2rifPiU5ejz7tkAYHLQT9WRZBMNh+NnIspDzbN3d\\\\\\\/zcbaXEVzi12DtbubGFHuh11F\\\\\\\/iLkbkrzCmPU5RSySSvxUF4tV9svpnaj5n77+Kg8VAdk5E1b\\\\\\\/ZsA4q5mDcNMmsrwz8je0h3DGFB\\\\\\\/bp6t09v6hb87jQ7c4cWT1gh\\\\\\\/qjWMUG94GzG5VEfzwxCFCQhhP1+tNupXIcXLHz4Gq5XogdUU0mNAXo3oM5JQ6X0tha0dQLNm5t9DoElImarU2qZJmmFcgCmwEI\\\\\\\/QOHzwFhuPafcnsTdlx0TQ41knxR1Lwf0M9eMjQwQ+jmFVOYviDjFZxMRvP8hHxU30otD7sGCcla5tDsMEc8HGCXRMgqhZndDxAyW9yrwKeMfmqrJA7FhhIiPreKnlFjUKmAKukEq4WruWlq2ImHD2lvEBjj7s9BADXSdvR9J8E2+VL7lve4ogbARxmLvu8O5OBcODi6G28vqiXAHucFId+DkdGfq44dBdoG+Rph9m6uv29uvu9mOeKEbz0\\\\\\\/pwa4VEcI27d1o6nYL39R8DZGhJzrcIgs2BO0D0GzS65yda4xokyzlo\\\\\\\/\\\\\\\/7wDfsflYShqJJMFM4aKOREleIdheALnzUHEOOJJgxb6lZ0LqcVLNV\\\\\\\/KMlyp7wdFQZL6yeyB+N2GGJGMyya0Esq7eedcNj4EalMwaqr81M4uyhEOSMGOvrgUKH+iDzb2yhkyFuD8WUyOkZN3Sh6YClAj8r1T6rmS5w5Zwn3D19+e5QAMKiCvHBofjPvK56pY0D\\\\\\\/hDn4AM0SaD5VMv8xkJBbk9gMvk8C04Jf+8Nflnb7NfOuyDyJTbP3VPZmuUy6PjRadMEp6Z5UKi98MzFYdM+NGJ9xwVu6tXmFMO8EunHFQX\\\\\\\/ApIf7MwsHspMDTu0hSwcuM0ViZM9iPMYW9NSX3f69svraPFPFXqKFxJXl0V\\\\\\\/PCc6vzzjJT5YXqy8RexOeKLEV2MX7bhLK0KPyQaC\\\\\\\/+h7aIWyyWZLFu12U9D1qCoQ27kLPY9mfappctYB2Rx7yt\\\\\\\/5ik4Ks8BdVA58jJL0CWlsLeOInj5XuQoT7Wc\\\\\\\/Mc00FRGY3J\\\\\\\/UG8Ocpp5yOYnF+MeAfIIIa4J\\\\\\\/py\\\\\\\/oEFW8XOijXYRLTR\\\\\\\/WwWdZT9Q1i63BfA4P6+QRiJqirjGpe8zeuQb39sQR4hsSgWCIuxJb\\\\\\\/WJxSYhPJLV8oqMqFfF5OcFChQoBxbV\\\\\\\/HQvzGwsdOJLc+AxBKzCVFI1SOvR0v9IK3B\\\\\\\/FJ9bT+9Xp9PyhQijH5WI1cPoU9Wey29MjbHP5gA8kYghuCM7kyMVzO64qQB5j+u59IFOEn4pEBSiDTbXGaz\\\\\\\/XsRoTbVhmfl4EM0bMDhiVHtflBb3bTU8DAeTxd3x6fbJgDD5sgIGiK2yNva5eMPC43z52JsC4AJihii3gYiHxAAHBvtjM+2CJlC2YFjIud5aOLBPymB4W2Bs4mfQaM7NkwE4xVnQx0BQjPH3CTVH8tM24viVHVoaeZwCdrZhosget2cuSCZDCj5\\\\\\\/FJn8JpbXst+BqnU3x+h8dR5Uoug1nY+opx2z3j3F+mKLWUBsGVh5YY+RmHm6JU971gLPfyUiRq2CJxZxPgwSDdifrHwmE6v54cygVPVxxnVFefxu8HhLTsTR8wkct1s70s90\\\\\\\/ewd9zvzPcgyjbWl\\\\\\\/O7rmf38mCCk2oC\\\\\\\/m6JxOYPAbVRs8yZC86NBBeuM3CE\\\\\\\/za4L5xZrAYD4SVFj67MyrUr9Ct8mLnPqkg6\\\\\\\/f55GB53oxKEWKYTznXi\\\\\\\/1lM5P8hufHQEDS0YnrryPQZqYYCJvtcEc\\\\\\\/bgOEkYXa05HKsYfydSqiq4oQfmPs5jxM7M59EBqJj+G\\\\\\\/CwygSw0B5B9HPHZpvAWHmGBBD98aguWm3+D+xe\\\\\\\/Fwcnx5vfST9RfB7PA1upEEiRAIY7Z6KTYTp0YG\\\\\\\/Cs6e\\\\\\\/SMzBvOQHID5nGbusT8gXxUQLhZIhIhBE3T\\\\\\\/viOW2kHzVnFC2PVFhLl7+oY+ldDgkYCnEuaUh1p0Q6ybsmAsEsR23RAWWjvf+WmfGk7GywzaltlkX1Xvv+2RI4Vb5m4\\\\\\\/bg5N6B2WGrdyb\\\\\\\/MzcY4ZemIPSku\\\\\\\/nDgNsV\\\\\\\/hlMS2uqHVTqUotsI7Q92R\\\\\\\/E+RJxykESyb8moyiVnXbg6LJ4ksz7X+DQbQ4PqCNAuAA+1Ovsbjy23SOJ9qpba8VQq8zB8DW+C7Hix7Ki\\\\\\\/s03Ac7Uf4WKKnJCHGrpqrr4WQ4ks8GPZ2vNUFekilavkiid1mQcBpp88w+XCRdXCWn3AIsyH2QW+Vc6nocoExFx78CP4YGZ0HyUP9WhP6J6JztBL1s7HmBLMjxdttJc6LPLmd1htzp4RwyipDuDwn+Pah\\\\\\\/AREn3kQR5dZBBe0nZpZW34aNg4BuoRbTIGDCWxcTNy2qOVAnYthhDXIzHJ9poHAsmKqrddGFKsLZBWGBKkmd1noIHWxbkHKZw+\\\\\\\/4Fss45+4ATgVKn97rbnQHx1UhocDINWhM4oZzHfrToSxbzdLEGR0zY99\\\\\\\/Fzc8eP\\\\\\\/TqfUHlmq1TBWO5KXgTSQ4IM2GvCiOwh9gmbYB9c\\\\\\\/thDiy\\\\\\\/VYRGw\\\\\\\/OunjN76lAPgE6se1TvEetza1oZtnQa4IiRuJYV1GUZGrn2YEd\\\\\\\/SrWg5ykljN6KEodrt8XkLUr4u1tSxANID2ktQyWm\\\\\\\/WUdhjYW8yFBdIZBovHucPrnXYUaXAW4C8a7ezyj5ozuHc8bj44BrOi4RbQH90UFh\\\\\\\/ZUXQ1Iwk1GWHaI5Yi+zxPLz99SsS2In7LI7NgMbe+80GkkGxo+dzOMHvSHv6g5Xx0J3Py3kvq8HN75gEXja+ufRlREwZdFt8mPp\\\\\\\/MjjHF8gMPHQb6GssGWkEI\\\\\\\/383OzluDePWmyl8ls2xmmvLdBkuEAay5rzN46vP8e8LqeYfYzWFfc0fZWrUrwmZ6wBfDToS6FNscSPuPjj3gVsu9pPJLgEVtPuOeQpbm+LvO6EUoYa\\\\\\\/5i6dyyW6h6o7cW7RIhAor6PAgrO5zr17fKPnsBgCk4BTPDkekzplFmGJXbH8nRZgQEE2fM98z6C2Bfi6ff0eke5M\\\\\\\/yIAG2JMptbthss1RsJhDqBXgOoSqChHOniuYJFIcFSJ92NaSnfNPz9Ii0ri9KAjxfH6D9yOHXVBBd+okqiBLZE3f19OZxESVen1k7Dt\\\\\\\/Ok2BPMZQHVWTLlU69gTwPixGvEzNE4IA+msI+BlkOdhPh0ums1s5eXooKfpRXeioyrHo7n2YUja6W4WKoL\\\\\\\/rpxJ39nZFJErDLvHU9a4ZkpPdijVzudC+527V+mV75tl0yE3oOXVbKvgZ29Uw\\\\\\\/QpWcwUr1Ftpeu0smGuKvmK1qIhvkieh3b6iLGKoYjkSYZlrGKh2WvL6yKNBlUhfITg4ijv5Wi6fzDb+l9FCy4bjK7EfwvosjTmAI57jTbFiwZhk0QQx\\\\\\\/3+Tnrx34K2U3B2LDsjuFO\\\\\\\/hO7sQIZdaItHKJ3YEn5sn7XzRC1Ur\\\\\\\/SN3rF3qjCqvHQ\\\\\\\/GaDaTOe2dhsEblte6zotNyoFtIewfWFNTL9IKuOkipUGYLolfE2rzIi+zBszrEJqVcE2aPUWJFzqiOON20VK6exnlkYXIR1SKx2v\\\\\\\/NPa34\\\\\\\/CNnppomCEow3KTzQ1n0HUMWPRZ\\\\\\\/39I2Wbwp8inx4pxuKamhw54PUslTd93lT9NaQAmctqbna7xhPR5KEgZiWWvr3cd8LABvlg8Y\\\\\\\/gQcJKI3rDXT+j6gJX3fzyRqV6jo8gtDLtLHQBArriKs2Xy60iMLX3UrvyvgGZbOh2V8QCnffMVD71LfazObsPV5fWZOjT3MA+AjBEyn+LGkaVT\\\\\\\/+dJ3T0ERf9M5j17Zh996IOaoaU4ykmFSocJshCnG\\\\\\\/ioFMgA0My2dizDlIp0oN2FauQdEWnhNoZIYB95eF5u\\\\\\\/a7+OwlgGVP5ePxVDECyJGlh7Ch+KetmSkDqgH6exlu\\\\\\\/FfUkqUb5Bnr85c0Q7RAnMrmTaf4gh3ocofwQjIqVtCZMC5VrwsfdItPfA+2NpjKPcOEdj\\\\\\\/E6Lrjtr2yfKy1pXew8Gym1DCgHwTA+2b3UVVVSjW3\\\\\\\/LpGxIkGOXYLNvFG822vSSEdfqwbjqKPFlQ0GaRG7UqaqqsXY0Qq5OHy7Ekq6yyeD0oB\\\\\\\/w9T3\\\\\\\/S8v+H80r6jZTiI7zpECmpqHrfXR1AqhSlB3DmgQKWAXAhiUy\\\\\\\/dnpTN1yqzXQL6xgfC\\\\\\\/jYElEjojsF9kt1agAkVdq1Rg41cIPLGap\\\\\\\/DEGbWqquaB+I984CKhY9SQ5MhtHxW\\\\\\\/HX3jufWFUWza3oqVrZWAbFAr0wliy4Rie3HXyJQxsFDhGvN4hjFLklSerTc\\\\\\\/EQ8w4FYkP58guHm88uITb6XJnJzP9z7zZyMzkEw5tKcb+DZ2394+vZTEMvza8G9FZCUMImAcFaAg++vFwo3m\\\\\\\/jGBwCIwI4sUNK6PihotfVhRBXq2krKUp9yKZM9L8zjTLf1rc\\\\\\\/w2TbjzI2x6ghIw0B4xCrvAE9c7x8SO3OU7dmvEvpcA0J7V5f2r0J00ZYSrl4\\\\\\\/RkR1xNFxGqQU8HwfgCBM6H1mIimmF3MCx3h1xytI5HwTOzJz85uez+1T55Z0hIAiSrCqcUdvJ6c52dejrU8Er3JxX2Oszup+P3JHEnjEGCd\\\\\\\/Otj05OkiMOvzISAO4E\\\\\\\/RzLjk2KDOyvw9Zee+gpZL5gT6YBDjq2P5VT4NI8Hz8j0TWxDU8uUWScZr1LuMghFbfCMbIskjkYbJTXOhFDWWbowMz4+exy0b9ta3Wqzn\\\\\\\/T4+0jJlOJgi8Z8ttSeh0UTbFu0RCe52AniYJNpoplTY+k3MG4Tbu8ReVWWGHvsT\\\\\\\/OyDkDEyM9IoUXd9E9Fyyceh2q606zF+5+ptk\\\\\\\/nzt5paxK4HOgkXILDTdeFCqLWYQfF00asF2ZE+b8T3kOUPELXkpY4u4FppBzFe8FecmYjXq\\\\\\\/tTssLasJJVfU2p1AwMcnUTVzjY4ruxfLw8AGsfDIo0cm6t0NScdPH1goPkvnkrpvVB78uFYe7qZmiBqoIe0SrkfqQ1ExLBVVebcby1KQPa2Un+v\\\\\\\/Zj4HnjNxPdJzeDSSYEFDQWr73SYUAeolejdwxagXYiZh3ttFFKHcKrB9EelBCI6QJA1LripCs5BZFzguixFYVn8FznpJbIjXk1pUuZ8+Y9UIKkzhuIJv8n6DXxECmjd+FPF88nICrc0BIMCt41c6RmobDwMyHKYh5TDiSR1h0R8BXJ7yQouGZ1thJIVJsqku5c+ANRBGTHHo1rvdkFm\\\\\\\/v4FzfMXguIsEhPwaQptTTQPqt7xRrvk1oXHsyLAHFzKAzv7Y9+5TqNWRyT8agnNFfmceIx70z2evVts3\\\\\\\/e9JZ4iIb8Mwsf7Am+Nj8rkB9B8Vbf6EUff+G2KljFhBmn2ZwX1p7WZ5+PH3UL0utzWD8UfMnKflxyYmMbb1MNh5iP2AACZHl0roEkdIdBfmTkOUZK0toeTWNozSnZo7zDr3uEGxnzvGrb3IbbhEgwUBU2qa3sbQTCOwBMBIornmnzqy2z9bdRbkkWsW715W+lom4QV6NxfSiB7SnpJRdrWqfm6LIfHdJ\\\\\\\/JIp7r0z0SggdlzFGyflLpZKCAHsaXrjGd5hXgLWP9ePcRvuHuJotLf6LyGo1v3BPrSLPStqkRNZyq\\\\\\\/X4bEF0pEdp9jeAQ5RPrfw7U3JS0CmePoe4MQXiGJsDzbRDXGAaG6hjHcjtWxBsoxgvjqlDStHmIw\\\\\\\/LyL8quAp7nlRR6l31IK\\\\\\\/2Z4N1E9+e9D1kb4PmqhrM9KDabR833u7TlLuWn82CqKo1+fqUfDAxWq25YEQQ4iKbaMNUCR4Jc3Iar+7GiJbyFxtP7eeoagHUOfbiqskLMETbZ66tD1N\\\\\\\/YTqNn1yVBxcFAvc69Q\\\\\\\/bJh+bQpAa+lbAZU4\\\\\\\/tVqOj0W+X2CF62sMyqJdHUAsCO\\\\\\\/hDmpMvppPHpTdx1\\\\\\\/hxwSiFbQBA5naHlF0MJY\\\\\\\/aHkmGDKDFP0CCb+Qrn+vQM\\\\\\\/zIMtl6X8oBDJpgNWFqNSb8aejxNJFakhWNNdAJXZ+mGg1qm02bJk5T1058RC87GfR1LUegWnofrUa0zu7wDjzYDBKNnooRHZiY6rYLQhUvsGWFOuDJtus1EMZzbO1fT6vlXkGeci\\\\\\\/yRdstEKf9f9MnN3JlJMNUZY7R8Mjyc9EB+\\\\\\\/LWI1SgouvJW0SkQXavmGJMH3NaD59obpsSOkrjjy2F8ybjCLNJIFZSHVZx40yQAlwYj8F7Dqyx2oUvhuFQScppflC8K0dvk88noEVPy4rF\\\\\\\/W9xcTD3EkS\\\\\\\/AB9Uza2ZzfDbPylVabnTsEejhWLfBEn4rdhfe6GBc+eloIwSnaF8IHmE1AaMoZvFG6lBraQzNvA8CFa9uNY\\\\\\\/IjgyodXVVabAUnY+gs\\\\\\\/P9MHpHlRpTE4FLqloYbxBeC9igENxQqO2QFd4HIvKmMGDK816FVNUoLxnAWL6deqpjA8UXeID9PrT45n\\\\\\\/aDq4InliBqIbYxRELIh8D+3SndFgjMvUg5zVfcwcle8K7pBLapHILgdn0BJ6VoXQqCA2J0PTALWWb6vC+ZbruZAmcO2o9k18rmgSGg4Itao9W+THKDy7eEsnb27LdaJQjqDqyZy4GeNwFDn5yApkSG8XB8hxD4b6znAcDw6GhZPXWW300jtZ75fYo6wDJa6Z57KvHQPEYUY\\\\\\\/ovOvyvswnWtk53pHZ9Z4aUUcjgBP4+x09PhsY1GUZsT4l+v\\\\\\\/9R15XnJUl6joT3EllYvW1oPyr8AkC4D1eT+6nrT3e6GhDHLSmjK9QrMeKd1HpQCY7Pa2YixpYcPS6Ui8EJ3zW7xitUXltYP4KJ8GH543II0e5jFpELtInNLqueFIUqvKavus+jEHcdo59ruwcfKw500iv2LvvmyaG1trDGhw5StPPG1igvB+sahsyQoIcJvPm0kui1Kc6reOsPuEPdRL8YjTqYmJdMYctrbzSYAmw7Mt9QNe2kimsYxDUEFOSiJC\\\\\\\/29Ecr6gDLtYVbOIY7BdvXkCS5oC2HuDvcLtSe\\\\\\\/qAbI\\\\\\\/o0c34nEKxrsYQVtjEdZ6jVy+dMzk642Bht9jlFy3s9vYcpOUg+0MB77+HtVpqnZdNozkFkZbSOBWkJMZxiTW+ffMjQJbpuvj\\\\\\\/1DiorfnmUvfQamXIkpPThCTmkya1gTCUATYcfUUL1YvWE8TibVHBleImyBZyLoE8BsTnsX60AfAyW\\\\\\\/Q2CnwF49RS70Ve9SLa7F257flrExlcKlKw2bYpONMi3eNGMhJTJcW4o9K74uccruCbH2ftLVoltYxVbsHn8JFzrrZU2PqsxSjrmshntCDUX9u7b9EaCj56TIZSS3b+6iV3Cu9J8CWhdeJXltYTl3bgNUd\\\\\\\/vXiAtlAKvED\\\\\\\/N3HHjKk\\\\\\\/Nzu0y1SoGYaYEUtYkoUuVJvdy4DJlmjiGEmieE3uhmmKgbAWi5jYyg7ySIz4xqWatdJ57PVXbVtuKHWMsqxzWuQzYykcWhL2QhWg0vuXrBFFLm8v+8CFg1pQuY+q+NaiFpLC8ZOIMomI3QJzpDLTgJaHrklpNuZFtIWNMlUEuw+s\\\\\\\/XTOEJZ3\\\\\\\/Td\\\\\\\/w2sDb4TGVqVFQf3yP+Q5knuHWcbhWhdSvCIVVPUnXHFyJmFTMss5NcF7MCpI0JCugKaRyJl3n+u\\\\\\\/I2GBRh7bzpCNHjqd96enz0ODsUt2NJdI6w6jbDwgbQ36mVFSoKKi8pmou2RgOveIiwJKhOSt8Jo4uunsOh2SEqX1WEpI4x9PtV8BQjsjm6y14O\\\\\\\/FFLE\\\\\\\/zTbJFvMAx0IrvpA6C8usZVBnFKsDIrV594\\\\\\\/kn9XmvEKed9yygn\\\\\\\/FnZPZDoTLffuReLT5Q098gJQUAetnbD15yvpcYHyA\\\\\\\/Xj3Ru3di8qlRgs5di79DqpMZBuaUpHGJpn1mctx8OUu7a9Bh54\\\\\\\/zXBNbpUYIWU10tdIVCIjklPlmPS\\\\\\\/nLmR5smzQTqvmf7Gk4JgVqrjADL2W5t4skfOYySN\\\\\\\/nqd\\\\\\\/mWnQLQWJbv3gEhSI85lfyXjXKsk5eeGoIW+DpHQVdw6v3jY6TvI8CImlrbrsP5hLKS7XP668ZX7cRaHkYByxlCGWcaNqrHeUOj+siBNfzCNe9VSqxyZwXBSiIAq7pEEXECmtEMGc2GCvVsD+Xe9VQcCsCwTdUy62UV8kthbhH8HTckcZap29MgT5CpNbC1U7jxcKJ\\\\\\\/8t1lBQAAOdXUNBrOHQU6OceBHw9ud1OiLwCEC5KmLStv0YoxSAtT8GfBZmw+1ieUEcCrc1qQxzpjL+SUkM3xa05I7gLzOoib7Xg4OaEcjn0wZdb1K3Ocktpozorqk91X\\\\\\\/1Q83Y3NIquwi5A6hlolpZJwv6rERZwUYA29ZdNwvpbV2OxHCuudCkkW+FyjY67\\\\\\\/toZd8ZAnBCh6Sza8io79BAA7o\\\\\\\/PndRu1FJHnwc8lpy2Vx1+TQmdvv8XNW+p29o3VWr08awaQfuPBAdwYQjy53\\\\\\\/jaIrrwFYRtc6rvWjD5SXrAs+YuAo+gbrMEgXXqo1hE0MkuY3lWCzKYkpso+781NpO7VrcH5EN24iQUQp0EtvqOZlieF8psvHkC7h5QXR9\\\\\\\/5YOt4yxD1jrXanFj7bSEL84BrVVXzQJbv3qCVJdps\\\\\\\/ky+hsGIiteRgFENJ\\\\\\\/IyJmawIR3Tb37Dh8CSzZPaSAEsD42s+JZrkZ1eioWafLgsjliGstOpnWHsEo14yKUb7VBIGn32yFKcu4YJDQcrzs7gsu0yRTMDZTHtkdmQ3iLoFYfTHPQMzdik8B+in3tqTVP2Owc9HlgxwDbkN+SoWxmJZG3Ry\\\\\\\/8seroreZWuyKds4KRX\\\\\\\/1+A35RkUj25eZFyKwGCNHGuZ3ZjGRIPp0\\\\\\\/88205mwfqnf9zbvwTOmdl\\\\\\\/6BtiawZV63DVKLIv1QHaXFZCqGZZIcodXDQvJmSk+\\\\\\\/FB8XPl4XFk6AEr\\\\\\\/tQIfnvrHjlPApj+7mAe3PhYB843hj5LKqtcCvjXqXd\\\\\\\/g+3BKinLpAPQ3BO6hny3RKOcKKBuz9iPlYNsKbUeUexcEa81c2lh5UHDoa3+BkwenXnCajn4cEe\\\\\\\/YuiT1gikLrB7H2BHPva2Koina1O8k2kINuwR8XQaT7VBmAGlD1UY4Od8FAt3J8Sf0lHpqJwExKYAC0DxSEcN32V+mfsZ0lHH7hLpTGtjMwF6MrBILEAYvhSqQBdwjW1FG2xMZzp88jNWmx2hSU2j9OiooCEETElMpeS8AGZKoL0xblzc1vZYriAO4Urb2qrHZdrMeI0zJ0pURBgDJqlsDNbGG6lv83xp\\\\\\\/kf7G1wmydqlj274MZlWobJu1G3OTm++LkAxwxjKIk5bFfJTLZDczF8odrEV2v7CZx6nGfnjyRiUbS3w7WXszSE9ckS5BCQ9kvlL2siXb8gjEzXiHjsj1MLOlpnHIGTfYjG\\\\\\\/BBWXOHD8FLgrgiWMXkBxwH0xvbZGGx18ftGtgAAGpCu6FXFMt+r+V+t6O7mKFPbZVJXIDmwHVe37PPwUbZ9B2nSbKJIiFsGMpUt6AtGImahUMDdvLXxMvB8huLxMyz8eVwxOucoeKKJEC0dVtkGLT6YGtrVoBiAAYHe12qoRc8D0Ybr1BiNYc47SqebsX+Mf3eb0T+lKKyCoYV2sS2xdIQqzzWIA571UbzymmM9xC4sxlHPUuEGcE+pMZ9DGRyXokJw4X61wBA1KUykKU6BwNBJA9WxaCLnABBIECTX1ZKIuhKcg\\\\\\\/A8QJXIi1q5XA1Q8AcOecSrof348LKBnMsBwUmo\\\\\\\/DHP2g5GLXNWonZ0g2+YZc5QezuyEOFD19YSug8NJiPjaTiVjy4BkUFh4RNr+5sxH8jWFn2dCOXW6k451bmEdmWo2ak6Saqczg5+7G8xgIAfXknvVRMn3jV792AUdNplhpdxRndbuszQqYFhcykN9xkL5xiIzUTmxYSD231tAbsAhmmfG0qkFwvaLHH5tczyie8X03FHlvbH3O41uh5WY2zNT9NZc8kNqMMuPdZuUALQWjF6xiw1Lm59TwKnbweWN\\\\\\\/zcXQM09YiV3LMJqeig9FjAWIL8g==\\\u0022,\\\u0022iv\\\u0022:\\\u0022DFTWrrhjZtLACh52x2Ykqg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e59ba7aec0b590ccbe175e1e58eb86ed5756ccb11172b49bd53cb7b69096c27a\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022O51diTAYQIE30+XJyV7o+Kco939I3NprDFfRBD5SKZcEJOmT88RJeVWeaZTmCi4OZscpmkn+12bMPdF5an0h3x\\\\\\\/NnMDKSPSDRCZgFUpFIkAs3YUGWMuxoD33MTAq84eXtbTejINqekDb3RlMzV6j9Kl\\\\\\\/YRuWqyv34sED0yNtJATn5oeMA5yzX+ZnyYdPi0LyS38u6gM0miZf8mqkq9Pl2B9KTI02BByiu1x0KLAegL31j1MORyA19n+0p626VtktdOPw8nqsPvm+c7\\\\\\\/yg9sEHk0q1+wKalQw6jsJaHBtDadjz3poFUyzfdKg4BqlXZzLGoWDTAYrdxAgy\\\\\\\/yS3pjFBL4fgx68Gg\\\\\\\/SXFsHSVrITEC9QvGLcyTgL7XDfrKlcWLKh0bRoe1UOVokkcOJkITHx2zuhH\\\\\\\/moNlMJpZC+KJDwus5d128iXCjGYM8dzUnM1ClznKvUvIIxaBJ7MsvhVgoidEOAsh+M3nIzPH6bnWvTJVCraEGhMgAPohlYNMTOo0eWiauTyAPuJqwAasu1f2wkWIAzJjhx7rhidVqLx\\\\\\\/arIFj23zuD65CP42AoFVtvIZ1SAjr7qF7a25mF\\\\\\\/kn+BF4tE0WAP8Byg7bY+1bZK+isXQa11R9yk\\\\\\\/xyTajMWKJjub9q8YxL9ulxM3fnTWXJRn4OV7Sk8EpbydlX5dds6pe3LRXrbgwSWS7uZqkG\\\\\\\/qhd4ZLrpQmQPYQrVHfRSe8KJ1mjIoehXegGOxLmy0t3U5isE0MvBUHcbmJAv7qcM8wckCACEpbi5LkUOzPjryCc17DBYNAkn3\\\\\\\/scqmmTDsZ+p9m\\\\\\\/2pLTiaCuchqfBd5GuzrlPm5AWf43gCiwEUw3nFDVtfZbH3XDunT+Rbi7oPwlQy2jcBMahO9Hpr59DulohLbd2IuoMPrCJQXMVSDggcY0y3HosDF6Rhm4jwn4buj30BulW7bC3re23NGowJwZyYavkLF8YqiUvdH8dPAzgGe34PQPduqxbHAyTno473PU4nDExNbLfDzQ67Ihd9En\\\\\\\/asray0oOcgIKSiN27\\\\\\\/1guc8Ue978RfNAbrJYIZMClmrjuaIbjiB8ru7iTr02kAy7dDel9WI4MHk\\\\\\\/AOOvWLYAFwKzPBUqsudr6Du0MYfvh+aIX7MIROCxOAZwRXfwmG\\\\\\\/k83lmnNDWAn32GPwmgrTxOJfCn2uD6azxpb9UhYav0EjEUis2CzPbWcDaDGGNHnpCYgFmOeRSX0nPSGx99cbBoi16QZEe9eNW8824TrsT3jQKbtDc1rikuvp+LqmrGM6TXP0vGGa35jsnaQKQjZNa348W+peRWuq90uxph\\\\\\\/PSAjCY2WwGCyCa3hQOhuHCEiaSxcP3gflss2Rl3Uim5pdZRcCmZnOImRqpIPB+p2c2q0aoEhwkuQ3hf1aabDnHRNyhmdnD4cRKb38kFDA+ClJd+zHfioSvFx0wSpjSaS3sC3CfCahmc\\\\\\\/i\\\\\\\/lMnWdr3MulBpuph+xB3KsHb9EGqpKOst+8xfqEW4rQtFROqbI4elEZcKVHGekHYMUBti0aGxZ3fNpMrMj7mY+ATVq7RteqsIwsS3A\\\\\\\/ZB1eb9IHU4Fuh7cdyZvWb36T9+YthOa2NAXLXHovCKvjMukD7BjW8XG1zV\\\\\\\/ORo1OrHdffnrynQzf21hfySDDJobPEfL7aIb7o2b0vUbSgwf8uvYJQrZHdsH2DKUi\\\\\\\/EAFgkmVU+u4jxzd0Sq1VxI0UgaCW5WGkzE\\\\\\\/vP9U\\\\\\\/nbuMaX2ji5rnntaUSkVpogPry4AKrXXoiGpu4y3vMfPWiICy1cOvshHPLjjWVEFM6z2IeazfryBA33kaJK6YSJDzaC\\\\\\\/AiMlVS60HHib1iJX30V2ZFxhCfwI8CPcZFQNQ4FY1aCFYubl+7gd8R8GXHbQEfkmKUn2iyJ0w7gFpI4af2wO1NSduoisBR\\\\\\\/tL1FOlKKxS0\\\\\\\/5kHNaVpG9p9NtBTyaZYsVV2ySDoCLtXFdw9V2hsMBBV1KE1DeIWWGM8bk1Q+2ttsx0upRKiyBHKTfMmaZVLD5g==\\\u0022,\\\u0022iv\\\u0022:\\\u0022oPHlTczRkCBdPoV5bA3XNw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223c42e69ed00a580f63354463c6bc01b5bbc4a0e1a917c358d70b3d563f566ffa\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460474 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/11/3467" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-vo952U06AQqVchOgUEUz2oelptbiRfbM7MmfeP9-CZs" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/11/3467" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-VZ7R_4bilAExQpb9MTQHxFWMDl9B-eQU95TvOxfNaX0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/11/3455" title="Proper Sterol Distribution Is Required for Candida albicans Hyphal Formation and Virulence" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/11/3475" title="Multiple Transcript Properties Related to Translation Affect mRNA Degradation Rates in Saccharomyces cerevisiae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460474" id="node460474" data-pisa="ggg;6/11/3467" data-pisa-master="ggg;g3.116.032227" data-apath="/ggg/6/11/3467.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Depletion of Shine-Dalgarno Sequences Within Bacterial Coding Regions Is Expression Dependent</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Chuyue Yang</span>, <span class="highwire-citation-author" data-delta="1">Adam J. Hockenberry</span>, <span class="highwire-citation-author" data-delta="2">Michael C. Jewett</span> and <span class="highwire-citation-author" data-delta="3">Luís A. N. Amaral</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3467-3474;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.032227 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460474" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Chuyue Yang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Chemical and Biological Engineering, Northwestern University, Evanston, Illinois 60208</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chuyue%2BYang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yang%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChuyue%2BYang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Adam J. Hockenberry </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Chemical and Biological Engineering, Northwestern University, Evanston, Illinois 60208</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Interdisciplinary Program in Biological Sciences, Northwestern University, Evanston, Illinois 60208</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Adam%2BJ.%2BHockenberry%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hockenberry%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAdam%2BJ.%2BHockenberry%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Michael C. Jewett </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Chemical and Biological Engineering, Northwestern University, Evanston, Illinois 60208</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Interdisciplinary Program in Biological Sciences, Northwestern University, Evanston, Illinois 60208</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Northwestern Institute on Complex Systems, Northwestern University, Evanston, Illinois 60208</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Chemistry of Life Processes Institute, Northwestern University, Evanston, Illinois 60208</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BC.%2BJewett%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jewett%20MC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BC.%2BJewett%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:m-jewett@northwestern.edu" class="" data-icon-position="" data-hide-link-title="0">m-jewett@northwestern.edu</a>
<a href="mailto:amaral@northwestern.edu" class="" data-icon-position="" data-hide-link-title="0">amaral@northwestern.edu</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Luís A. N. Amaral </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Chemical and Biological Engineering, Northwestern University, Evanston, Illinois 60208</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Northwestern Institute on Complex Systems, Northwestern University, Evanston, Illinois 60208</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Physics and Astronomy, Northwestern University, Evanston, Illinois 60208</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lu%C3%ADs%2BA.%2BN.%2BAmaral%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Amaral%20LA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALu%25C3%25ADs%2BA.%2BN.%2BAmaral%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:m-jewett@northwestern.edu" class="" data-icon-position="" data-hide-link-title="0">m-jewett@northwestern.edu</a>
<a href="mailto:amaral@northwestern.edu" class="" data-icon-position="" data-hide-link-title="0">amaral@northwestern.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/11/3467" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460474" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460474/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/11/3467.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460474" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460474/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/11/3467.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460474" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460474/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/11/3467.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460474" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460474/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/11/3467.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460474" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460474/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Efficient and accurate protein synthesis is crucial for organismal survival in competitive environments. Translation efficiency (the number of proteins translated from a single mRNA in a given time period) is the combined result of differential translation initiation, elongation, and termination rates. Previous research identified the Shine-Dalgarno (SD) sequence as a modulator of translation initiation in bacterial genes, while codon usage biases are frequently implicated as a primary determinant of elongation rate variation. Recent studies have suggested that SD sequences within coding sequences may negatively affect translation elongation speed, but this claim remains controversial. Here, we present a metric to quantify the prevalence of SD sequences in coding regions. We analyze hundreds of bacterial genomes and find that the coding sequences of highly expressed genes systematically contain fewer SD sequences than expected, yielding a robust correlation between the normalized occurrence of SD sites and protein abundances across a range of bacterial taxa. We further show that depletion of SD sequences within ribosomal protein genes is correlated with organismal growth rates, supporting the hypothesis of strong selection against the presence of these sequences in coding regions and suggesting their association with translation efficiency in bacteria.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/translation-initiation" class="hw-term hw-article-keyword hw-article-keyword-translation-initiation" rel="nofollow">translation initiation</a></li><li class="kwd"><a href="/keyword/gene-expression" class="hw-term hw-article-keyword hw-article-keyword-gene-expression" rel="nofollow">gene expression</a></li><li class="kwd"><a href="/keyword/growth-regulation" class="hw-term hw-article-keyword hw-article-keyword-growth-regulation" rel="nofollow">growth regulation</a></li></ul><p id="p-4">Translation of mRNA to protein consumes a vast amount of cellular resources, particularly in rapidly growing unicellular organisms (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Dekel and Alon 2005</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Wagner 2005</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Shachrai <em>et al.</em> 2010</a>). Many researchers have hypothesized that efficient (<em>i.e.</em>, fast and accurate) translation is highly advantageous and should therefore leave a recognizable signature on the genome (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Sharp <em>et al.</em> 2005</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Stoletzki and Eyre-Walker 2007</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Drummond and Wilke 2008</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Supek <em>et al.</em> 2010</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Vieira-Silva and Rocha 2010</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Botzman and Margalit 2011</a>).</p><p id="p-5">For decades, researchers have focused on understanding the link between tRNA concentration and translation rates of cognate codons, under the assumption that ribosomal dwell-time on a particular codon is partially determined by diffusion limited tRNA binding and competition between near-cognates (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Ikemura 1981</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">dos Reis <em>et al.</em> 2004</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Rocha 2004</a>). Indeed, multiple lines of evidence strongly support this hypothesis in a multitude of different organisms (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Tuller <em>et al.</em> 2010</a>).</p><p id="p-6">Recently, ribosome profiling (a technique that maps transcriptome-wide ribosome occupancy) has been applied to study whether different codons show variation in translation rates, but researchers have come to conflicting conclusions, even when using the same dataset (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Li <em>et al.</em> 2012</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Dana and Tuller 2014</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Gardin <em>et al.</em> 2014</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hussmann <em>et al.</em> 2015</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Weinberg <em>et al.</em> 2016</a>). One of the most startling findings to emerge from ribosome profiling experiments is the striking degree of heterogeneity in ribosome occupancy across mRNAs, which is punctuated by large peaks suggestive of “pausing” or “stalling” (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Ingolia <em>et al.</em> 2009</a>; <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Li <em>et al.</em> 2012</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Schrader <em>et al.</em> 2014</a>). These pauses, in contrast to known stalling sequences, are orders of magnitude larger than what is expected from basal translation rate variations due to tRNA concentrations, and may instead result from nascent peptide interactions within the ribosomal exit tunnel (such as poly-proline sequences), ribosomal queuing, or <em>trans</em>-interactions between mRNA and ribosomes (<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Li <em>et al.</em> 2012</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Charneski and Hurst 2013</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Shah <em>et al.</em> 2013</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Woolstenhulme <em>et al.</em> 2015</a>; <a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Weinberg <em>et al.</em> 2016</a>).</p><p id="p-7">Using ribosome profiling, <a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">Li <em>et al.</em> (2012)</a> showed that, in bacteria, translational pauses were significantly associated with sequence binding between the anti-Shine-Dalgarno (aSD) sequence of the 16S ribosomal-RNA and the translating message. This binding interaction is important during the process of translation initiation, where the ribosome binds to the 5′ untranslated region (5′-UTR) to facilitate start codon recognition (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). However, the occurrence of these “Shine-Dalgarno” (SD) sequences within coding sequences had not been previously associated with translational pausing (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Shine and Dalgarno 1974</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Salis <em>et al.</em> 2009</a>). SD sequence-mediated pauses have now been documented for several bacterial species and independent ribosomal profiling datasets (<a id="xref-ref-27-5" class="xref-bibr" href="#ref-27">Li <em>et al.</em> 2012</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Liu <em>et al.</em> 2013</a>; <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Schrader <em>et al.</em> 2014</a>). Studies have built on these results by showing SD-associated pauses <em>in vitro</em>, negative effects of SD sequences on protein production in engineered sequences, enhanced solubility of recombinant proteins via rational insertion of SD sequences at protein domain boundaries, and enrichment of SD sequences following transmembrane domains of natural sequences (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Agashe <em>et al.</em> 2013</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Chen <em>et al.</em> 2014</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Chevance <em>et al.</em> 2014</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Fluman <em>et al.</em> 2014</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Vasquez <em>et al.</em> 2015</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/11/3467/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The possible dual impacts of Shine-Dalgarno (SD) sequences on protein synthesis. (A) SD sequences in the 5&#x2032; untranslated region (UTR) of mRNA (messenger RNA) are known to facilitate translation initiation in bacteria via binding to the anti-SD sequence on the 3&#x2032; tail of the 16S ribosomal RNA. (B) Recent research suggests that SD sequences within coding sequences may regulate the rate of translation elongation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-788079424" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The possible dual impacts of Shine-Dalgarno (SD) sequences on protein synthesis. (A) SD sequences in the 5&#x2032; untranslated region (UTR) of mRNA (messenger RNA) are known to facilitate translation initiation in bacteria via binding to the anti-SD sequence on the 3&#x2032; tail of the 16S ribosomal RNA. (B) Recent research suggests that SD sequences within coding sequences may regulate the rate of translation elongation.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/11/3467/F1.medium.gif" width="322" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/11/3467/F1.medium.gif" width="322" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/11/3467/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/11/3467/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469575" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-8" class="first-child">The possible dual impacts of Shine-Dalgarno (SD) sequences on protein synthesis. (A) SD sequences in the 5′ untranslated region (UTR) of mRNA (messenger RNA) are known to facilitate translation initiation in bacteria via binding to the anti-SD sequence on the 3′ tail of the 16S ribosomal RNA. (B) Recent research suggests that SD sequences within coding sequences may regulate the rate of translation elongation.</p><div class="sb-div caption-clear"></div></div></div><p id="p-9">By contrast, recent results have questioned whether the observed SD-associated pauses are actually an experimental artifact resulting from the ribosome profiling protocol, specifically the differential sizes of sequencing fragments (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">O’Connor <em>et al.</em> 2013</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Mohammad <em>et al.</em> 2016</a>). Indeed, the existence of SD-mediated pauses has not been confirmed using several other experimental methods (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Borg and Ehrenberg 2015</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Chadani <em>et al.</em> 2016</a>; <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Mohammad <em>et al.</em> 2016</a>). Thus, remains unclear what role, if any, SD sequences within protein coding genes have in modulating translation speed (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>).</p><p id="p-10">Even though the usage and diversity of SD sequences within the 5′-UTR has been analyzed extensively at the genome-scale (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Ma <em>et al.</em> 2002</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Starmer <em>et al.</em> 2006</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Nakagawa <em>et al.</em> 2010</a>), the occurrence pattern of these important sequence motifs within the coding sequences of diverse species has been largely neglected [though see <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Itzkovitz <em>et al.</em> (2010)</a> for an exception]. Thus, open questions remain as to whether SD sequences are indeed systematically depleted within coding sequences from diverse species and, if so, whether the depletion follows any particular pattern that may provide clues to the evolutionary significance of these sequences.</p><p id="p-11">In order to answer these open questions, we sought to characterize the general occurrence of SD sequences within protein coding genes across a range of bacterial species of known phylogeny. We first present a metric to characterize single mRNA sequences according to their estimated sequence binding propensity with the ribosomal aSD sequence. Using this metric, we show that depletion of SD sequences in coding regions is a hallmark of bacterial genes and that, within a given species, the degree of this depletion is inversely correlated with measured gene expression levels. Finally, we show that variation in SD sequence depletion between different genomes is related to the minimal known doubling time of individual species, suggesting that depletion of SD sequences is driven by evolutionary pressure for greater translation efficiency.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Codon-shuffled null model</h3><p id="p-12">We randomly generated null model genomes that preserve codon usage and primary amino acid sequence at the gene level. For each gene, we constructed a list of all codons used in the original sequence. Given the primary amino acid sequence of the gene, we then randomly selected a codon from the pool of available synonymous codons for that particular amino acid without replacement. The start and stop codons are not affected by this process and thus remain fixed during the shuffling process. We repeated this procedure for every gene within a given genome in order to create one instance of a randomized genome for null model comparison. For statistical comparison using Monte Carlo hypothesis testing, we created 1000 randomized genomes in this manner. Using our metric, we calculated the mean and SD in these randomized genomes for each organism, and then calculated a <em>z</em>-score for the real genome along with the resulting <em>p</em> value, which we report in the main text.</p></div><div id="sec-3" class="subsection"><h3>aSD hybridization</h3><p id="p-13">We predicted thermodynamic interactions between the aSD sequence and each six-nucleotide-long sequence using the RNA cofold method of the ViennaRNA Package 2.0 with default parameters (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Gruber <em>et al.</em> 2008</a>). For this study, we have chosen to use the canonical core aSD sequence of 5′-CCUCCU-3′ for all species, owing to the fact that this core sequence is nearly universally conserved. Further, the 3′-tail of 16s rRNAs is slightly variable and poorly annotated (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Nakagawa <em>et al.</em> 2010</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Lim <em>et al.</em> 2012</a>), making it difficult to empirically determine the precise aSD sequence for each individual species.</p></div><div id="sec-4" class="subsection"><h3>Pax-Db data collection</h3><p id="p-14">We collected the complete bacterial dataset from the Protein Abundance Across Organisms Database (Pax-Db) in August 2015 (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Wang <em>et al.</em> 2015</a>). This resource contains protein abundance measurements for 26 different bacteria. When multiple datasets were available for a particular organism, we chose the “Integrated” dataset, which is the result of Pax-Db curators integrating the various protein abundance data sources based on coverage and quality. The full set of data that we analyzed for each species is available upon request.</p></div><div id="sec-5" class="subsection"><h3>Growth-rate dataset and phylogenetic relatedness</h3><p id="p-15">We obtained growth rate measurements (minimum doubling time, measured in hr) from <a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Vieira-Silva and Rocha (2010)</a>. For each species in their data table, we matched the name of the species provided in the original data source to the species name in a local copy of the NCBI GenBank complete genome sequences. This resulted in 187 matches for bacteria (Archaeal species, which were provided in the original dataset, were ignored for the purposes of this study). Within each of these bacterial genomes, we relied on annotations in the GenBank files to find ribosomal proteins by searching the “product” field for “ribosomal subunit,” or perturbations thereof. Full data, including GenBank files for all relevant organisms and ribosomal protein “locus_tags” used in this study, are available upon request.</p><p id="p-16">To construct a phylogenetic tree from these species, we extracted the 23S and 16S gene sequences using RNAmmer-1.2 (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Lagesen <em>et al.</em> 2007</a>). When multiple sequences were available for a given genome, we randomly chose one of each for alignment. We then individually aligned 23S and 16S sequences using MUSCLE (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Edgar 2004</a>). Finally, we concatenated the 16S and 23S alignments for each organism and constructed a maximum likelihood (ML) tree using RAxML with a partitioned analysis that separately fit rate models for the 16S and 23S sequences. We used a 5′-GTRGAMMA-3′ evolutionary model with 100 rapid bootstrap searches and 20 ML searches and selected the best fitting ML tree. (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Stamatakis 2014</a>).</p></div><div id="sec-6" class="subsection"><h3>Regression analyses</h3><p id="p-17">With one exception noted below, all statistical analyses were performed using the SciPy (version: 0.16.0) and StatsModels (version: 0.6.1) packages in Python.</p><p id="p-18">To control for phylogenetic effects in our growth rates regression analysis, we used the PGLS function from the “caper” package in R, choosing the optimal λ value to transform our input tree via maximum likelihood search.</p></div><div id="sec-7" class="subsection"><h3>Data availability</h3><p id="p-19">The authors state that all data and code necessary for confirming the conclusions presented in the article are available as Supplementary Material (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032227/-/DC1/TableS3.csv">Table S3</a>).</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>Quantifying the occurrence of SD sequences within coding sequences</h3><p id="p-20">We first counted the number of occurrences of the canonical SD motif (5′-AGGAGG-3′) within the coding sequences of the 187 bacterial species compiled by <a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">Vieira-Silva and Rocha (2010)</a>. For each genome, we compared the number of SD sequences found within coding sequences to the number expected by chance using a codon-shuffled null model to control for codon usage bias within each gene (see <em>Materials and Methods</em>). We found that 175 out of 187 genomes contained fewer canonical SD sequences in their coding sequences than expected by chance (154 were significant at <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-1.gif"/></noscript></span></span> Monte Carlo hypothesis testing, <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>).</p><p id="p-21">However, single or multiple base mismatches to the canonical SD sequence are frequently assumed to be functional in translation initiation, and the strength of aSD sequence binding to different hexamer sequences spans a range of values. To quantify the occurrence of SD sequences on a per-gene basis in a manner that encapsulates the full breadth of this heterogeneity, we estimated the free energy of binding between the aSD sequence and each hexamer within the coding region of each mRNA (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>, see <em>Materials and Methods</em> for details). Since the free energy of binding (<span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-2.gif"/></noscript></span></span>) at a particular site is proportional to the logarithm of the ratio of the association and dissociation rate constants of binding, we define the affinity <em>A</em> of a hexamer <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-3.gif"/></noscript></span></span> to the aSD sequence as:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-4.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>We define the aSD binding score <em>S</em> of a gene as:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-5.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>where <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-6.gif"/></noscript></span></span> is the average affinity over a gene’s coding sequence (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). The transformations involved in the definition of <em>S</em> aim to lessen the contribution of weak-binding interactions while amplifying the contributions from the strongest aSD binding sequences.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/11/3467/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Quantifying aSD sequence binding within coding regions. (A) We estimate the free energy of binding for each hexamer within a gene to the core aSD (anti-Shine-Dalgarno) sequence (5&#x2032;-CCUCCU-3&#x2032;). (B) Free energy (top) and affinity (bottom) profiles for a typical E. coli gene (b3055). The affinity profile amplifies the contribution from strongly binding regions within the gene. nt, nucleotides." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-788079424" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Quantifying aSD sequence binding within coding regions. (A) We estimate the free energy of binding for each hexamer within a gene to the core aSD (anti-Shine-Dalgarno) sequence (5&#x2032;-CCUCCU-3&#x2032;). (B) Free energy (top) and affinity (bottom) profiles for a typical &lt;em&gt;E. coli&lt;/em&gt; gene (b3055). The affinity profile amplifies the contribution from strongly binding regions within the gene. nt, nucleotides.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/11/3467/F2.medium.gif" width="421" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/11/3467/F2.medium.gif" width="421" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/11/3467/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/11/3467/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469748" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-22" class="first-child">Quantifying aSD sequence binding within coding regions. (A) We estimate the free energy of binding for each hexamer within a gene to the core aSD (anti-Shine-Dalgarno) sequence (5′-CCUCCU-3′). (B) Free energy (top) and affinity (bottom) profiles for a typical <em>E. coli</em> gene (b3055). The affinity profile amplifies the contribution from strongly binding regions within the gene. nt, nucleotides.</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">We calculated <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-7.gif"/></noscript></span></span> for each of the coding sequences of 187 bacterial species, and define genome aSD binding score <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-8.gif"/></noscript></span></span> We again compared this empirical value to the expected value for a given genome based off a codon-shuffled null model and found that, similar to the previous analysis, 172 out of 187 genomes had average aSD binding scores lower than expected by chance (167 were significant at <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-9.gif"/></noscript></span></span> Monte Carlo hypothesis test, <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>). These results demonstrate that genomes contain significantly fewer SD sequences than would be expected from gene-specific codon usage biases and amino acid sequences.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/11/3467/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Depletion of SD occurrence in genomes compared to expectation from 1000 randomly generated genomes using our codon-shuffled null model. (A) the canonical SD (Shine-Dalgarno) sequence 5&#x2032;-AGGAGG-3&#x2032; is depleted within coding sequences in most genomes (175 of 187). (B) The genome aSD (anti-SD) binding score  is lower for most organisms (172 of 187). Both distributions are centered significantly to the left of 0, showing that the majority of organisms avoid SD sequences according to both metrics." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-788079424" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Depletion of SD occurrence in genomes compared to expectation from 1000 randomly generated genomes using our codon-shuffled null model. (A) the canonical SD (Shine-Dalgarno) sequence 5&#x2032;-AGGAGG-3&#x2032; is depleted within coding sequences in most genomes (175 of 187). (B) The genome aSD (anti-SD) binding score &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-10&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; is lower for most organisms (172 of 187). Both distributions are centered significantly to the left of 0, showing that the majority of organisms avoid SD sequences according to both metrics.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/11/3467/F3.medium.gif" width="332" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/11/3467/F3.medium.gif" width="332" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/11/3467/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/11/3467/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469669" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-24" class="first-child">Depletion of SD occurrence in genomes compared to expectation from 1000 randomly generated genomes using our codon-shuffled null model. (A) the canonical SD (Shine-Dalgarno) sequence 5′-AGGAGG-3′ is depleted within coding sequences in most genomes (175 of 187). (B) The genome aSD (anti-SD) binding score <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/F3/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/F3/embed/mml-math-10.gif"/></noscript></span></span> is lower for most organisms (172 of 187). Both distributions are centered significantly to the left of 0, showing that the majority of organisms avoid SD sequences according to both metrics.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>The occurrence of SD sequences in coding regions correlates negatively with Escherichia coli gene expression data</h3><p id="p-25"><span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-11.gif"/></noscript></span></span> allows us to test whether variation in aSD sequence binding between different genes correlates with gene-level features such as expression level. We obtained five genome-scale expression datasets for <em>E. coli</em> to ensure the robustness of our results (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032227/-/DC1/TableS1.pdf">Table S1</a>) and correlated the gene expression measurements against the calculated aSD binding score for each gene (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>) (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Lu <em>et al.</em> 2007</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Taniguchi <em>et al.</em> 2010</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Shiroguchi <em>et al.</em> 2012</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Li <em>et al.</em> 2014</a>). We observed a highly significant negative relationship in all datasets, indicating that the coding sequences of highly expressed genes contain fewer SD sequences (<span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-12.gif"/></noscript></span></span> for all cases) (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4, A and B</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/11/3467/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="aSD binding scores negatively correlate with gene expression in E. coli. (A) An example dataset showing negative correlation between protein abundance and aSD (anti- Shine-Dalgarno) binding scores for individual E. coli genes ( ). Specifically, coding sequences containing fewer SD sequence motifs have higher protein abundances. (B) Multivariate regression shows that expression changes cannot be fully explained by codon usage bias, and that additional predictive power is offered by  We chose five datasets that provide independent measurements of mRNA (messenger RNA), protein, and translation efficiency levels in order to test the robustness of our findings (Lu et al. 2007; Taniguchi et al. 2010; Shiroguchi et al. 2012; Li et al. 2014)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-788079424" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;aSD binding scores negatively correlate with gene expression in &lt;em&gt;E. coli&lt;/em&gt;. (A) An example dataset showing negative correlation between protein abundance and aSD (anti- Shine-Dalgarno) binding scores for individual &lt;em&gt;E. coli&lt;/em&gt; genes (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-13&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-14&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;). Specifically, coding sequences containing fewer SD sequence motifs have higher protein abundances. (B) Multivariate regression shows that expression changes cannot be fully explained by codon usage bias, and that additional predictive power is offered by &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-15&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; We chose five datasets that provide independent measurements of mRNA (messenger RNA), protein, and translation efficiency levels in order to test the robustness of our findings (Lu &lt;em&gt;et al.&lt;/em&gt; 2007; Taniguchi &lt;em&gt;et al.&lt;/em&gt; 2010; Shiroguchi &lt;em&gt;et al.&lt;/em&gt; 2012; Li &lt;em&gt;et al.&lt;/em&gt; 2014).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/11/3467/F4.medium.gif" width="440" height="222"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/11/3467/F4.medium.gif" width="440" height="222"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/11/3467/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/11/3467/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469598" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-26" class="first-child">aSD binding scores negatively correlate with gene expression in <em>E. coli</em>. (A) An example dataset showing negative correlation between protein abundance and aSD (anti- Shine-Dalgarno) binding scores for individual <em>E. coli</em> genes (<span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/F4/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/F4/embed/mml-math-13.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/F4/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/F4/embed/mml-math-14.gif"/></noscript></span></span>). Specifically, coding sequences containing fewer SD sequence motifs have higher protein abundances. (B) Multivariate regression shows that expression changes cannot be fully explained by codon usage bias, and that additional predictive power is offered by <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/F4/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/F4/embed/mml-math-15.gif"/></noscript></span></span> We chose five datasets that provide independent measurements of mRNA (messenger RNA), protein, and translation efficiency levels in order to test the robustness of our findings (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Lu <em>et al.</em> 2007</a>; <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Taniguchi <em>et al.</em> 2010</a>; <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Shiroguchi <em>et al.</em> 2012</a>; <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Li <em>et al.</em> 2014</a>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">A number of different factors are known to influence protein abundances, including start codon choice, mRNA structural accessibility, and SD sequence usage at translation initiation sites (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Guimaraes <em>et al.</em> 2014</a>). Here, we wish to focus on the elongation phase of translational control to determine what, if any, additional predictive power is conferred by the effect of aSD sequence binding within coding sequences. Prior studies have established that the codon usage bias of individual genes is highly correlated with protein levels (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Tuller <em>et al.</em> 2010</a>). In order to investigate whether the observed correlation between <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-16.gif"/></noscript></span></span> and gene expression is driven solely by codon usage bias, we conducted multivariable linear regression using both <em>S</em> and an established method for quantifying codon usage bias to predict expression levels (<span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-17.gif"/></noscript></span></span>) (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Novembre 2002</a>). If <em>S</em> were solely a consequence of codon usage bias, the adjusted-<span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-18.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-19.gif"/></noscript></span></span>) should decrease when <em>S</em> is included as an independent variable along with <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-20.gif"/></noscript></span></span> On the contrary, we observe that the best model for all datasets includes both <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-21.gif"/></noscript></span></span> and <em>S</em> as predictors of expression (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4B</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032227/-/DC1/TableS1.pdf">Table S1</a>). While the enhancement in predictive power is not additive, this is not uncommon when evaluating models with multiple covarying predictors.</p></div><div id="sec-11" class="subsection"><h3>The occurrence of SD sequences within coding regions correlates negatively with protein abundances in diverse bacterial taxa</h3><p id="p-28">To determine the generality of the previous finding, we expanded our analysis to 26 diverse bacteria for whom protein expression data were previously collected by <a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Wang <em>et al.</em> (2015)</a> (see <em>Materials and Methods</em>). For 19 out of 26 datasets, we observed that <em>S</em> was significantly negatively correlated (<span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-22.gif"/></noscript></span></span>) with protein abundances (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032227/-/DC1/TableS2.pdf">Table S2</a>). As in the previous subsection, we also implemented a multivariate model to determine whether the observed correlation was solely a consequence of codon usage bias. For 23 out of 26 datasets we saw an improved <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-23.gif"/></noscript></span></span> value when <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-24.gif"/></noscript></span></span> is added as a predictor along with estimates of codon usage bias (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/11/3467/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Shine-Dalgarno (SD) sequence depletion is correlated with protein abundances in a diverse set of bacterial taxa. Distribution of differences between the  for models which do and do not contain the S score. For 23 of the 26 organisms, inclusion of aSD (anti-SD) binding score as an independent variable enhances predictive power. The full data table, including organism names and values, is available in Table S2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-788079424" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Shine-Dalgarno (SD) sequence depletion is correlated with protein abundances in a diverse set of bacterial taxa. Distribution of differences between the &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-25&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; for models which do and do not contain the &lt;em&gt;S&lt;/em&gt; score. For 23 of the 26 organisms, inclusion of aSD (anti-SD) binding score as an independent variable enhances predictive power. The full data table, including organism names and values, is available in Table S2.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/11/3467/F5.medium.gif" width="440" height="383"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/11/3467/F5.medium.gif" width="440" height="383"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/11/3467/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/11/3467/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469693" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-29" class="first-child">Shine-Dalgarno (SD) sequence depletion is correlated with protein abundances in a diverse set of bacterial taxa. Distribution of differences between the <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/F5/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/F5/embed/mml-math-25.gif"/></noscript></span></span> for models which do and do not contain the <em>S</em> score. For 23 of the 26 organisms, inclusion of aSD (anti-SD) binding score as an independent variable enhances predictive power. The full data table, including organism names and values, is available in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032227/-/DC1/TableS2.pdf">Table S2</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">We further confirmed the observation that the more complex multivariate model resulted in a better fit to the data by using <em>AIC</em> and <em>BIC</em> to evaluate model fits. For 22 and 18 organisms, respectively, the multivariate model provided a better fit to the data than a linear model based on codon usage bias alone (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032227/-/DC1/FigureS1.pdf">Figure S1</a>).</p></div><div id="sec-12" class="subsection"><h3>Ribosomal protein coding sequences contain fewer SD sequences than other genes</h3><p id="p-31">To overcome the limited availability of bacterial protein expression datasets, we next investigated whether ribosomal protein coding sequences contain fewer SD sequences than other genes within a genome. Ribosomal proteins are essential for all organisms and they are generally expressed at high levels, making them some of the most likely genes to show selection for accurate and efficient translation.</p><p id="p-32">In <em>E. coli</em>, we observed that aSD binding scores for the 58 ribosomal protein genes are significantly lower than that of all other genes (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>). To quantify the magnitude of this difference, we define the normalized SD bias within a genome, <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-26.gif"/></noscript></span></span> as:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-29.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>where <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-30.gif"/></noscript></span></span> is the averaged <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-31.gif"/></noscript></span></span> for ribosomal protein coding genes, and <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-32.gif"/></noscript></span></span> is the averaged <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-33.gif"/></noscript></span></span> for all genes within a genome. When <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-34.gif"/></noscript></span></span> ribosomal protein genes contain fewer SD sequences than would be expected based on the genome-wide average. We opt for this approach for two primary reasons. First, the <em>S</em> values of ribosomal protein coding genes themselves would be heavily influenced by the underlying genomic GC content. Normalizing to the genome-wide average should help to mitigate this effect. Second, research has shown that at higher growth rates, ribosomal protein genes make up an increasingly larger fraction of bacterial proteomes (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Borkowski <em>et al.</em> 2016</a>). Thus, relative differences in <em>S</em> between ribosomal protein coding genes and the genome as a whole should reflect the selective pressure for increased ribosomal protein production during periods of rapid growth.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/11/3467/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Depletion of SD (Shine-Dalgarno) sequences within ribosomal protein coding genes is widespread throughout the bacterial kingdom and associated with organismal growth. (A) Distribution of aSD (anti-SD) binding scores of ribosomal protein coding sequences in E. coli, compared to that of all other protein coding sequences. We characterize SD sequence usage bias in a genome with Equation (3). (B) Distribution of genome SD bias index for 187 bacteria genomes. Ribosomal proteins have significantly lower aSD binding scores, as compared to the rest of the genome, in the majority of bacterial species. (C) SD bias is correlated with minimum generation time in 187 organisms (Spearman-rank:  ). Depletion of internal-SD sequences in ribosomal protein genes is associated with faster growth. The full data table for this analysis, including organism names, growth rate, and B values, is provided as (Table S3)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-788079424" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Depletion of SD (Shine-Dalgarno) sequences within ribosomal protein coding genes is widespread throughout the bacterial kingdom and associated with organismal growth. (A) Distribution of aSD (anti-SD) binding scores of ribosomal protein coding sequences in &lt;em&gt;E. coli&lt;/em&gt;, compared to that of all other protein coding sequences. We characterize SD sequence usage bias in a genome with Equation (3). (B) Distribution of genome SD bias index for 187 bacteria genomes. Ribosomal proteins have significantly lower aSD binding scores, as compared to the rest of the genome, in the majority of bacterial species. (C) SD bias is correlated with minimum generation time in 187 organisms (Spearman-rank: &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-27&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-28&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;). Depletion of internal-SD sequences in ribosomal protein genes is associated with faster growth. The full data table for this analysis, including organism names, growth rate, and &lt;em&gt;B&lt;/em&gt; values, is provided as (Table S3).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/11/3467/F6.medium.gif" width="440" height="374"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/6/11/3467/F6.medium.gif" width="440" height="374"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/11/3467/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/11/3467/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469641" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-33" class="first-child">Depletion of SD (Shine-Dalgarno) sequences within ribosomal protein coding genes is widespread throughout the bacterial kingdom and associated with organismal growth. (A) Distribution of aSD (anti-SD) binding scores of ribosomal protein coding sequences in <em>E. coli</em>, compared to that of all other protein coding sequences. We characterize SD sequence usage bias in a genome with Equation (3). (B) Distribution of genome SD bias index for 187 bacteria genomes. Ribosomal proteins have significantly lower aSD binding scores, as compared to the rest of the genome, in the majority of bacterial species. (C) SD bias is correlated with minimum generation time in 187 organisms (Spearman-rank: <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/F6/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/F6/embed/mml-math-27.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/F6/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/F6/embed/mml-math-28.gif"/></noscript></span></span>). Depletion of internal-SD sequences in ribosomal protein genes is associated with faster growth. The full data table for this analysis, including organism names, growth rate, and <em>B</em> values, is provided as (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032227/-/DC1/TableS3.csv">Table S3</a>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-34">Of the other 187 diverse bacteria spanning different genomic GC contents, growth environments, and growth rates, 173 have <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-35.gif"/></noscript></span></span> suggesting that the vast majority of bacteria have a larger depletion of SD sequences in their ribosomal protein coding genes relative to the genome as a whole (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6B</a>). The systematic depletion of SD sequences in ribosomal protein coding sequences further suggests that these motifs negatively impact gene expression and/or cellular fitness in a wide diversity of bacteria.</p><p id="p-35">Previous studies have shown that the relative codon usage bias of ribosomal genes compared to the rest of the genome is correlated with the minimum observed doubling time for particular species <a id="xref-ref-54-4" class="xref-bibr" href="#ref-54">Vieira-Silva and Rocha (2010)</a>. This finding is mechanistically assumed to be a consequence of the fact that, at rapid growth rates, ribosomal proteins constitute an increasingly large fraction of the proteome; selection for translational accuracy or efficiency within these genes relative to the genome thus likely reflects the evolutionary history driven by growth rate demands. Therefore, we hypothesized that <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-36.gif"/></noscript></span></span> scores may also be related to the growth rate demands of individual species. Indeed, we found that <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-37.gif"/></noscript></span></span> is positively correlated with the minimum known doubling times of this set of 187 bacteria; fewer SD sequences within the ribosomal protein coding sequences relative to the genome is associated with faster maximal growth rates (Spearman-rank: <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-38.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-39.gif"/></noscript></span></span>) (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6C</a>). We further confirmed the robustness of this finding via phylogenetic generalized least squares regression (see <em>Materials and Methods</em>) (<span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-40.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-41.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3467/embed/mml-math-42.gif"/></noscript></span></span>). This finding strongly suggests that SD motifs within coding sequences are detrimental to growth and reproduction, likely via negatively impacting translation.</p></div></div><div class="section discussion" id="sec-13"><h2 class="">Discussion</h2><p id="p-36">Prior research into translation elongation has focused on codon usage as the primary means of modulating elongation speed, but researchers have recently proposed that aSD-mediated sequence interactions are a dominant source of translational pausing in bacteria (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Gingold and Pilpel 2011</a>; <a id="xref-ref-27-6" class="xref-bibr" href="#ref-27">Li <em>et al.</em> 2012</a>). If true, this finding has important consequences for our understanding of the basic mechanisms of translation as well as practical implications for coding sequence design for synthetic biology and biotechnological purposes. By quantifying the usage of SD sequences within coding sequences in a diverse set of bacterial taxa, we have shown a consistent trend whereby SD sequences within coding regions are systematically depleted. Specifically, this effect is strongest in the most highly expressed genes across a variety of genomes. We further show that the level of biased depletion of SD sequences is strongest in organisms capable of very rapid growth where selection for translation efficiency has previously been shown to produce a variety of genome-scale hallmarks (<a id="xref-ref-54-5" class="xref-bibr" href="#ref-54">Vieira-Silva and Rocha 2010</a>).</p><p id="p-37">Recently, <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Diwan and Agashe (2016)</a> published an elegant analysis of “internal-SD-like” sequence usage in prokaryotes. Our results largely confirm the major finding of this study, which showed internal-SD-like sequences are depleted in &gt; 80% of the species analyzed. While their results found a number of species that were exceptions to this rule, we note that many of these exceptions are Archaea, whose translation initiation mechanisms remain elusive and are therefore excluded from our analysis. Further, our results build on these findings in important ways. By developing a metric of <em>S</em>, which is defined at the single-gene level, our analysis provides insight into within-genome variation and the selective pressures governing the usage of internal-SD sequences as it relates to gene expression costs. This within-genome analysis allows us to show that avoidance of SD sequences is highly related to the maximal growth rates of organisms using a method that controls for GC content variation, which <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Diwan and Agashe (2016)</a> found to impose an important constraint on the appearance of internal-SD sequences. Our analysis does not focus on temperature or variation in internal-SD usage with regard to position within genes, but the thorough results of <a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Diwan and Agashe (2016)</a> likely hold within our dataset.</p><p id="p-38">There are several possible limitations to our methodology that readers should be aware of when interpreting our findings. First, our study relies on an assumed aSD sequence of 5′-CCUCCU-3′ to calculate aSD binding strength scores for individual genes. It is possible, and evidence strongly suggests, that in particular lineages the aSD sequence may be slightly altered or extended compared to this canonical sequence (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Lim <em>et al.</em> 2012</a>). Therefore, we may be mischaracterizing the aSD sequence for several species in our dataset, or not encompassing the full breadth of possible sequence interactions. Future work can refine our findings to account for this aSD heterogeneity as more aSD sequences will be empirically determined, but we opt here for a conservative approach likely to be applicable for the majority of organisms in our dataset. Second, while our study relies on the precise definition of coding sequence bounds in existing genome annotations, prior research has shown that these annotations are likely spurious for up to ∼10% of annotated genes (<a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">Schrader <em>et al.</em> 2014</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Nakahigashi <em>et al.</em> 2016</a>). However, reliable N-terminal mapping is currently available for only a small fraction of bacterial genomes; until better computational models are developed to refine translational start site predictions, this will remain a limitation that adds noise to any computational genome-scale analysis, such as the one we perform here.</p><p id="p-39">SD sequences may be avoided within coding sequences for several different, and nonmutually exclusive, reasons. These sequences may: (i) result in erroneous internal translation initiation leading to the production of truncated protein products; (ii) temporarily sequester ribosomes, thus limiting the number available for proper translation initiation; (iii) encourage translational frameshifting; or (iv) substantially slow down translation elongation (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Devaraj and Fredrick 2010</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Chu <em>et al.</em> 2011</a>; <a id="xref-ref-27-7" class="xref-bibr" href="#ref-27">Li <em>et al.</em> 2012</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Whitaker <em>et al.</em> 2014</a>). In all of these cases, we would expect SD sequences within coding sequences to be largely detrimental and thus avoided. In particular, given that the consequences of any of the above explanations is amplified by high mRNA copy numbers, avoidance of these SD sequences would also be expected to manifest particularly in the most highly expressed genes.</p><p id="p-40">Although our results indicate that SD sequences are by and large detrimental, we also wish to clarify that some proportion of the SD sites within coding sequences may serve important functions. Owing to the compact nature of bacterial genomes, the translation initiation site of many genes within operons will occur within the 3′ terminus of the preceding coding sequence. Further, the presence of multiple translation initiation sites may serve a regulatory role for certain proteins, allowing for the production of distinct isoforms depending on the N-terminal sequence or controlling protein folding rates (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Ozin <em>et al.</em> 2001</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Fluman <em>et al.</em> 2014</a>; <a id="xref-ref-41-4" class="xref-bibr" href="#ref-41">Schrader <em>et al.</em> 2014</a>; <a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Vasquez <em>et al.</em> 2015</a>).</p><p id="p-41">One benefit of our large-scale analysis is that exceptions to the rules can point to interesting cases for further study. In <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a> we found three species where <em>S</em> did not appear to enhance predictions of protein abundance: <em>Mycoplasma pneumoniae</em>, <em>Shigella flexneri</em>, and <em>Leptospira interrogans</em>. Although none of these species are known to use noncanonical aSD sequences (<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Lim <em>et al.</em> 2012</a>), all are pathogenic species, suggesting that a possible relationship may exist between ecological strategies, effective population size, and the selection against SD sequences. However, owing to the large number of pathogenic species in this dataset, this finding will require further detailed investigation. Additionally, several species analyzed in <a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6</a> showed an enhancement of SD sequence usage within ribosomal proteins relative to the genome. Nearly all of these cases come from three distinct orders (phyla), pointing to likely mechanistic changes in the aSD interaction in particular clades: <em>Rickettsia</em> (Alphaproteobacteria), Mollicutes (Tenericutes) and Spirochaetes (Spirochaete) (both <em>M. pneumoniae</em> and <em>L. interrogans</em>, mentioned above, fall within one of these orders). Future ribosome profiling experiments on species from within these clades may provide clues on the evolution of the aSD sequence interaction.</p><p id="p-42">The patterns that we observe provide significant insight into the debate surrounding the usage of SD sequences within protein coding genes. Moreover, our results are fully orthogonal to ribosome profiling-based conclusions. It is clear from this bioinformatic analysis that SD sequences are largely avoided across the bacterial kingdom, and that this avoidance is likely due to deleterious effects on translation. Thus, we conclude that even if SD-mediated elongation pausing is an artifact of the ribosomal profiling protocol, as suggested by <a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Mohammad <em>et al.</em> (2016)</a>, care should be taken to avoid SD sequences when designing coding sequences for recombinant protein production applications.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-43">The authors would like to thank Sophia Liu for technical assistance related to data acquisition and helpful comments regarding the manuscript. C.Y. was supported by Northwestern University Undergraduate Research Programs (209WCASSUM133484; 342SUMMER155728). A.J.H. was supported by the National Institutes of Health training grant in Cellular and Molecular Basis of Disease (2-T32GM008061-31) and the Northwestern University Presidential Fellowship. M.C.J. was supported by the National Science Foundation (DMR - 1108350; MCB - 1413563), the David and Lucile Packard Foundation (2011-37152), and the Camille Dreyfus Teacher Scholar Award.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-44">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032227/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.032227/-/DC1</a>.</p></li><li class="fn" id="fn-3"><p id="p-45"><em>Communicating editor: J. M. Comeron</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-06-07"><span class="received-label">Received </span>June 7, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-08-17"><span class="accepted-label">Accepted </span>August 17, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Yang <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.1" data-doi="10.1093/molbev/mss273"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Agashe</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martinez-Gomez</span>  <span class="cit-name-given-names">N. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drummond</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marx</span>  <span class="cit-name-given-names">C. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Good codons, bad transcript: large reductions in gene expression and fitness arising from synonymous mutations in a key enzyme.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">549</span>–<span class="cit-lpage">560</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmss273%26rft_id%253Dinfo%253Apmid%252F23223712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjMwLzMvNTQ5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNDY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.2" data-doi="10.1016/j.jmb.2014.10.027"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Borg</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ehrenberg</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Determinants of the rate of mRNA translocation in bacterial protein synthesis.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">427</span>: <span class="cit-fpage">1835</span>–<span class="cit-lpage">1847</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMol.%2BBiol.%26rft.volume%253D427%26rft.spage%253D1835%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2014.10.027%26rft_id%253Dinfo%253Apmid%252F25451025%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2014.10.027&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25451025&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3467.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.3" data-doi="10.15252/msb.20156608"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Borkowski</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goelzer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaffer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calabre</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma der</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Translation elicits a growth rate-dependent, genome-wide, differential protein production in Bacillus subtilis.</span> <abbr class="cit-jnl-abbrev">Mol. Syst. Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">870</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BSystems%2BBiology%26rft.stitle%253DMol%2BSyst%2BBiol%26rft.aulast%253DBorkowski%26rft.auinit1%253DO.%26rft.volume%253D12%26rft.issue%253D5%26rft.spage%253D870%26rft.epage%253D870%26rft.atitle%253DTranslation%2Belicits%2Ba%2Bgrowth%2Brate-dependent%252C%2Bgenome-wide%252C%2Bdifferential%2Bprotein%2Bproduction%2Bin%2BBacillus%2Bsubtilis%26rft_id%253Dinfo%253Adoi%252F10.15252%252Fmsb.20156608%26rft_id%253Dinfo%253Apmid%252F27193784%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibXNiIjtzOjU6InJlc2lkIjtzOjg6IjEyLzUvODcwIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNDY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.4" data-doi="10.1186/gb-2011-12-10-r109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Botzman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Margalit</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Variation in global codon usage bias among prokaryotic organisms is associated with their lifestyles.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">R109</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DBotzman%26rft.auinit1%253DM.%26rft.volume%253D12%26rft.issue%253D10%26rft.spage%253DR109%26rft.epage%253DR109%26rft.atitle%253DVariation%2Bin%2Bglobal%2Bcodon%2Busage%2Bbias%2Bamong%2Bprokaryotic%2Borganisms%2Bis%2Bassociated%2Bwith%2Btheir%2Blifestyles.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2011-12-10-r109%26rft_id%253Dinfo%253Apmid%252F22032172%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2011-12-10-r109&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22032172&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3467.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.5" data-doi="10.1073/pnas.1520560113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chadani</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Niwa</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chiba</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taguchi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ito</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Integrated in vivo and in vitro nascent chain profiling reveals widespread translational pausing.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">E829</span>–<span class="cit-lpage">E838</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1520560113%26rft_id%253Dinfo%253Apmid%252F26831095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTEzLzcvRTgyOSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzQ2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.6" data-doi="10.1371/journal.pbio.1001508"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charneski</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hurst</span>  <span class="cit-name-given-names">L. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Positively charged residues are the major determinants of ribosomal velocity.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e1001508</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D11%26rft.spage%253De1001508%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1001508%26rft_id%253Dinfo%253Apmid%252F23554576%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1001508&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23554576&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3467.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.7" data-doi="10.1038/nature13428"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johansson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsai</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Leary</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Dynamic pathways of -1 translational frameshifting.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">512</span>: <span class="cit-fpage">328</span>–<span class="cit-lpage">332</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D512%26rft.spage%253D328%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature13428%26rft_id%253Dinfo%253Apmid%252F24919156%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature13428&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24919156&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3467.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340508200039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.8" data-doi="10.1371/journal.pgen.1004392"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chevance</span>  <span class="cit-name-given-names">F. F. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guyon</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hughes</span>  <span class="cit-name-given-names">K. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The effects of codon context on in vivo translation speed.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1004392</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004392%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004392%26rft_id%253Dinfo%253Apmid%252F24901308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004392&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24901308&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3467.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.9" data-doi="10.1093/nar/gkr300"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barnes</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Von Der Haar</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The role of tRNA and ribosome competition in coupling the expression of different mRNAs in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">6705</span>–<span class="cit-lpage">6714</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr300%26rft_id%253Dinfo%253Apmid%252F21558172%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzOS8xNS82NzA1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNDY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.10" data-doi="10.1093/nar/gku646"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dana</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tuller</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The effect of tRNA levels on decoding times of mRNA codons.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">9171</span>–<span class="cit-lpage">9181</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku646%26rft_id%253Dinfo%253Apmid%252F25056313%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0Mi8xNC85MTcxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNDY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.11" data-doi="10.1038/nature03842"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dekel</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alon</span>  <span class="cit-name-given-names">U.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Optimality and evolutionary tuning of the expression level of a protein.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">436</span>: <span class="cit-fpage">588</span>–<span class="cit-lpage">592</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDekel%26rft.auinit1%253DE.%26rft.volume%253D436%26rft.issue%253D7050%26rft.spage%253D588%26rft.epage%253D592%26rft.atitle%253DOptimality%2Band%2Bevolutionary%2Btuning%2Bof%2Bthe%2Bexpression%2Blevel%2Bof%2Ba%2Bprotein.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature03842%26rft_id%253Dinfo%253Apmid%252F16049495%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature03842&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16049495&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3467.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230788800068&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.12" data-doi="10.1111/j.1365-2958.2010.07421.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Devaraj</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fredrick</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Short spacing between the Shine-Dalgarno sequence and P codon destabilizes codon-anticodon pairing in the P site to promote +1 programmed frameshifting.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">78</span>: <span class="cit-fpage">1500</span>–<span class="cit-lpage">1509</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DDevaraj%26rft.auinit1%253DA.%26rft.volume%253D78%26rft.issue%253D6%26rft.spage%253D1500%26rft.epage%253D1509%26rft.atitle%253DShort%2Bspacing%2Bbetween%2Bthe%2BShine-Dalgarno%2Bsequence%2Band%2BP%2Bcodon%2Bdestabilizes%2Bcodon-anticodon%2Bpairing%2Bin%2Bthe%2BP%2Bsite%2Bto%2Bpromote%2B%252B1%2Bprogrammed%2Bframeshifting.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2958.2010.07421.x%26rft_id%253Dinfo%253Apmid%252F21143320%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2958.2010.07421.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21143320&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3467.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.13" data-doi="10.1093/gbe/evw107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Diwan</span>  <span class="cit-name-given-names">G. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Agashe</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The frequency of internal Shine-Dalgarno – like motifs in prokaryotes.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1722</span>–<span class="cit-lpage">1733</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevw107%26rft_id%253Dinfo%253Apmid%252F27189998%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2JlIjtzOjU6InJlc2lkIjtzOjg6IjgvNi8xNzIyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNDY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.14" data-doi="10.1093/nar/gkh834"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">dos Reis</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Savva</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wernisch</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Solving the riddle of codon usage preferences: a test for translational selection.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">5036</span>–<span class="cit-lpage">5044</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DReis%26rft.auinit1%253DM.%2Bd.%26rft.volume%253D32%26rft.issue%253D17%26rft.spage%253D5036%26rft.epage%253D5044%26rft.atitle%253DSolving%2Bthe%2Briddle%2Bof%2Bcodon%2Busage%2Bpreferences%253A%2Ba%2Btest%2Bfor%2Btranslational%2Bselection%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh834%26rft_id%253Dinfo%253Apmid%252F15448185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzMi8xNy81MDM2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNDY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.15" data-doi="10.1016/j.cell.2008.05.042"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Drummond</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilke</span>  <span class="cit-name-given-names">C. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Mistranslation-induced protein misfolding as a dominant constraint on coding-sequence evolution.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">134</span>: <span class="cit-fpage">341</span>–<span class="cit-lpage">352</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DDrummond%26rft.auinit1%253DD.%2BA.%26rft.volume%253D134%26rft.issue%253D2%26rft.spage%253D341%26rft.epage%253D352%26rft.atitle%253DMistranslation-induced%2Bprotein%2Bmisfolding%2Bas%2Ba%2Bdominant%2Bconstraint%2Bon%2Bcoding-sequence%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2008.05.042%26rft_id%253Dinfo%253Apmid%252F18662548%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2008.05.042&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18662548&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3467.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257891700022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.16" data-doi="10.1093/nar/gkh340"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edgar</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">MUSCLE: multiple sequence alignment with high accuracy and high throughput.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">1792</span>–<span class="cit-lpage">1797</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DEdgar%26rft.auinit1%253DR.%2BC.%26rft.volume%253D32%26rft.issue%253D5%26rft.spage%253D1792%26rft.epage%253D1797%26rft.atitle%253DMUSCLE%253A%2Bmultiple%2Bsequence%2Balignment%2Bwith%2Bhigh%2Baccuracy%2Band%2Bhigh%2Bthroughput%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh340%26rft_id%253Dinfo%253Apmid%252F15034147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjMyLzUvMTc5MiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzQ2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.17" data-doi="10.7554/eLife.03671"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fluman</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Navon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bibi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pilpel</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">mRNA-programmed translation pauses in the targeting of E. coli membrane proteins.</span> <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">19</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft.volume%253D3%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.03671%26rft_id%253Dinfo%253Apmid%252F25049222%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/eLife.03671&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25049222&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3467.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.18" data-doi="10.7554/eLife.03671"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gardin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yeasmin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yurovsky</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cai</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skiena</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Measurement of average decoding rates of the 61 sense codons in vivo.</span> <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">20</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft.volume%253D3%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.03671%26rft_id%253Dinfo%253Apmid%252F25049222%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/eLife.03671&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25049222&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3467.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.19" data-doi="10.1038/msb.2011.80"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gingold</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pilpel</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Determinants of translation efficiency and accuracy.</span> <abbr class="cit-jnl-abbrev">Mol. Syst. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BSyst.%2BBiol.%26rft.volume%253D7%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fmsb.2011.80%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/msb.2011.80&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.20" data-doi="10.1093/nar/gkn188"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gruber</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernhart</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neuböck</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hofacker</span>  <span class="cit-name-given-names">I. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The Vienna RNA websuite.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">W70</span>–<span class="cit-lpage">W74</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkn188%26rft_id%253Dinfo%253Apmid%252F18424795%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE0OiIzNi9zdXBwbF8yL1c3MCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzQ2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.21" data-doi="10.1093/nar/gku126"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guimaraes</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rocha</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arkin</span>  <span class="cit-name-given-names">A. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Transcript level and sequence determinants of protein abundance and noise in Escherichia coli.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">4791</span>–<span class="cit-lpage">4799</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku126%26rft_id%253Dinfo%253Apmid%252F24510099%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjQyLzgvNDc5MSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzQ2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hussmann</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patchett</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sawyer</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Press</span>  <span class="cit-name-given-names">W. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Understanding biases in ribosome profiling experiments reveals signatures of translation dynamics in yeast.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.23" data-doi="10.1016/0022-2836(81)90003-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ikemura</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1981</span> <span class="cit-article-title">Correlation between the abundance of Escherichia coli transfer RNAs and the occurrence of the respective codons in its protein genes: a proposal for a synonymous codon choice that is optimal for the E. coli translational system.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">151</span>: <span class="cit-fpage">389</span>–<span class="cit-lpage">409</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DIkemura%26rft.auinit1%253DT.%26rft.volume%253D151%26rft.issue%253D3%26rft.spage%253D389%26rft.epage%253D409%26rft.atitle%253DCorrelation%2Bbetween%2Bthe%2Babundance%2Bof%2BEscherichia%2Bcoli%2Btransfer%2BRNAs%2Band%2Bthe%2Boccurrence%2Bof%2Bthe%2Brespective%2Bcodons%2Bin%2Bits%2Bprotein%2Bgenes%253A%2Ba%2Bproposal%2Bfor%2Ba%2Bsynonymous%2Bcodon%2Bchoice%2Bthat%2Bis%2Boptimal%2Bfor%2Bthe%2BE.%2Bcoli%2Btranslational%2Bsystem.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2836%252881%252990003-6%26rft_id%253Dinfo%253Apmid%252F6175758%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-2836(81)90003-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6175758&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3467.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1981MJ92600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.24" data-doi="10.1126/science.1168978"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ingolia</span>  <span class="cit-name-given-names">N. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ghaemmaghami</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Newman</span>  <span class="cit-name-given-names">J. R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weissman</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genome-wide analysis in vivo of translation with nucleotide resolution using ribosome profiling.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">324</span>: <span class="cit-fpage">218</span>–<span class="cit-lpage">223</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1168978%26rft_id%253Dinfo%253Apmid%252F19213877%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjQvNTkyNC8yMTgiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzExLzM0NjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.25" data-doi="10.1101/gr.105072.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Itzkovitz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hodis</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Segal</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Overlapping codes within protein-coding sequences.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1582</span>–<span class="cit-lpage">1589</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.105072.110%26rft_id%253Dinfo%253Apmid%252F20841429%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyMC8xMS8xNTgyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNDY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.26" data-doi="10.1093/nar/gkm160"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lagesen</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hallin</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rødland</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stærfeldt</span>  <span class="cit-name-given-names">H. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rognes</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">RNAmmer: consistent and rapid annotation of ribosomal RNA genes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">3100</span>–<span class="cit-lpage">3108</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkm160%26rft_id%253Dinfo%253Apmid%252F17452365%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjM1LzkvMzEwMCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzQ2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.27" data-doi="10.1038/nature10965"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">G.-W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oh</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weissman</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The anti-Shine–Dalgarno sequence drives translational pausing and codon choice in bacteria.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">484</span>: <span class="cit-fpage">538</span>–<span class="cit-lpage">541</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLi%26rft.auinit1%253DG.%2BW.%26rft.volume%253D484%26rft.issue%253D7395%26rft.spage%253D538%26rft.epage%253D541%26rft.atitle%253DThe%2Banti-Shine-Dalgarno%2Bsequence%2Bdrives%2Btranslational%2Bpausing%2Band%2Bcodon%2Bchoice%2Bin%2Bbacteria.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10965%26rft_id%253Dinfo%253Apmid%252F22456704%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10965&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22456704&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3467.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000303200400056&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.28" data-doi="10.1016/j.cell.2014.02.033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">G.-W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burkhardt</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gross</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weissman</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Quantifying absolute protein synthesis rates reveals principles underlying allocation of cellular resources.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">624</span>–<span class="cit-lpage">635</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D157%26rft.spage%253D624%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2014.02.033%26rft_id%253Dinfo%253Apmid%252F24766808%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2014.02.033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24766808&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3467.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000335392100012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.29" data-doi="10.1093/molbev/mss101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lim</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Furuta</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kobayashi</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Large variations in bacterial ribosomal RNA genes.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">2937</span>–<span class="cit-lpage">2948</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmss101%26rft_id%253Dinfo%253Apmid%252F22446745%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIyOS8xMC8yOTM3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNDY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.30" data-doi="10.1073/pnas.1309739110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">High-resolution view of bacteriophage lambda gene expression by ribosome profiling.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">11928</span>–<span class="cit-lpage">11933</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1309739110%26rft_id%253Dinfo%253Apmid%252F23812753%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEwLzI5LzExOTI4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNDY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.31" data-doi="10.1038/nbt1270"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vogel</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yao</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marcotte</span>  <span class="cit-name-given-names">E. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Absolute protein expression profiling estimates the relative contributions of transcriptional and translational regulation.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">117</span>–<span class="cit-lpage">124</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DLu%26rft.auinit1%253DP.%26rft.volume%253D25%26rft.issue%253D1%26rft.spage%253D117%26rft.epage%253D124%26rft.atitle%253DAbsolute%2Bprotein%2Bexpression%2Bprofiling%2Bestimates%2Bthe%2Brelative%2Bcontributions%2Bof%2Btranscriptional%2Band%2Btranslational%2Bregulation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt1270%26rft_id%253Dinfo%253Apmid%252F17187058%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt1270&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17187058&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3467.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243491000037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.32" data-doi="10.1128/JB.184.20.5733-5745.2002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karlin</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Correlations between Shine-Dalgarno sequences and gene features such as predicted expression levels and operon structures.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">184</span>: <span class="cit-fpage">5733</span>–<span class="cit-lpage">5745</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DMa%26rft.auinit1%253DJ.%26rft.volume%253D184%26rft.issue%253D20%26rft.spage%253D5733%26rft.epage%253D5745%26rft.atitle%253DCorrelations%2Bbetween%2BShine-Dalgarno%2BSequences%2Band%2BGene%2BFeatures%2BSuch%2Bas%2BPredicted%2BExpression%2BLevels%2Band%2BOperon%2BStructures%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.184.20.5733-5745.2002%26rft_id%253Dinfo%253Apmid%252F12270832%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE4NC8yMC81NzMzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNDY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.33" data-doi="10.1016/j.celrep.2015.12.073"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mohammad</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woolstenhulme</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buskirk</span>  <span class="cit-name-given-names">A. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Clarifying the translational pausing landscape in bacteria by ribosome profiling.</span> <abbr class="cit-jnl-abbrev">Cell Rep.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">686</span>–<span class="cit-lpage">694</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BRep.%26rft.volume%253D14%26rft.spage%253D686%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.celrep.2015.12.073%26rft_id%253Dinfo%253Apmid%252F26776510%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.celrep.2015.12.073&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26776510&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3467.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.34" data-doi="10.1073/pnas.1002036107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nakagawa</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Niimura</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miura</span>  <span class="cit-name-given-names">K.-I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gojobori</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Dynamic evolution of translation initiation mechanisms in prokaryotes.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">6382</span>–<span class="cit-lpage">6387</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1002036107%26rft_id%253Dinfo%253Apmid%252F20308567%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA3LzE0LzYzODIiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzExLzM0NjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.35" data-doi="10.1093/dnares/dsw008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nakahigashi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takai</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kimura</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abe</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakayashiki</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Comprehensive identification of translation start sites by tetracycline-inhibited ribosome profiling.</span> <abbr class="cit-jnl-abbrev">DNA Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">193</span>–<span class="cit-lpage">201</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fdnares%252Fdsw008%26rft_id%253Dinfo%253Apmid%252F27013550%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZG5hcmVzIjtzOjU6InJlc2lkIjtzOjg6IjIzLzMvMTkzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNDY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.36" data-doi="10.1093/oxfordjournals.molbev.a004201"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Novembre</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Accounting for background nucleotide composition when measuring codon usage bias.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1390</span>–<span class="cit-lpage">1394</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DNovembre%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D19%26rft.issue%253D8%26rft.spage%253D1390%26rft.epage%253D1394%26rft.atitle%253DAccounting%2Bfor%2BBackground%2BNucleotide%2BComposition%2BWhen%2BMeasuring%2BCodon%2BUsage%2BBias%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a004201%26rft_id%253Dinfo%253Apmid%252F12140252%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjE5LzgvMTM5MCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzQ2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.37" data-doi="10.1093/bioinformatics/btt184"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">O’Connor</span>  <span class="cit-name-given-names">P. B. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">G.-W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weissman</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atkins</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baranov</span>  <span class="cit-name-given-names">P. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">rRNA:mRNA pairing alters the length and the symmetry of mRNA-protected fragments in ribosome profiling experiments.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">1488</span>–<span class="cit-lpage">1491</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt184%26rft_id%253Dinfo%253Apmid%252F23603333%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjkvMTIvMTQ4OCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzQ2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.38" data-doi="10.1128/JB.183.6.2032-2040.2001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ozin</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costa</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henriques</span>  <span class="cit-name-given-names">A. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jr</span>  <span class="cit-name-given-names">C. P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Alternative translation initiation produces a short form of a spore coat protein in Bacillus subtilis alternative translation initiation produces a short form of a spore coat protein in Bacillus subtilis.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">183</span>: <span class="cit-fpage">2032</span>–<span class="cit-lpage">2040</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DOzin%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D183%26rft.issue%253D6%26rft.spage%253D2032%26rft.epage%253D2040%26rft.atitle%253DAlternative%2BTranslation%2BInitiation%2BProduces%2Ba%2BShort%2BForm%2Bof%2Ba%2BSpore%2BCoat%2BProtein%2Bin%2BBacillus%2Bsubtilis%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.183.6.2032-2040.2001%26rft_id%253Dinfo%253Apmid%252F11222602%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTA6IjE4My82LzIwMzIiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzExLzM0NjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.39" data-doi="10.1101/gr.2896904"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rocha</span>  <span class="cit-name-given-names">E. P. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Codon usage bias from tRNA’s point of view: redundancy, specialization, and efficient decoding for translation optimization.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">2279</span>–<span class="cit-lpage">2286</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.2896904%26rft_id%253Dinfo%253Apmid%252F15479947%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxNC8xMS8yMjc5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNDY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.40" data-doi="10.1038/nbt.1568"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salis</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mirsky</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voigt</span>  <span class="cit-name-given-names">C. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Automated design of synthetic ribosome binding sites to control protein expression.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">946</span>–<span class="cit-lpage">950</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DSalis%26rft.auinit1%253DH.%2BM.%26rft.volume%253D27%26rft.issue%253D10%26rft.spage%253D946%26rft.epage%253D950%26rft.atitle%253DAutomated%2Bdesign%2Bof%2Bsynthetic%2Bribosome%2Bbinding%2Bsites%2Bto%2Bcontrol%2Bprotein%2Bexpression.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1568%26rft_id%253Dinfo%253Apmid%252F19801975%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1568&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19801975&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3467.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271472500026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.41" data-doi="10.1371/journal.pgen.1004463"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schrader</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">G.-W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lasker</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Childers</span>  <span class="cit-name-given-names">W. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The coding and noncoding architecture of the Caulobacter crescentus genome.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1004463</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004463%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004463%26rft_id%253Dinfo%253Apmid%252F25078267%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004463&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25078267&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3467.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.42" data-doi="10.1016/j.molcel.2010.04.015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shachrai</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaslaver</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alon</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekel</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Cost of unneeded proteins in E. coli is reduced after several generations in exponential growth.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">758</span>–<span class="cit-lpage">767</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DShachrai%26rft.auinit1%253DI.%26rft.volume%253D38%26rft.issue%253D5%26rft.spage%253D758%26rft.epage%253D767%26rft.atitle%253DCost%2Bof%2Bunneeded%2Bproteins%2Bin%2BE.%2Bcoli%2Bis%2Breduced%2Bafter%2Bseveral%2Bgenerations%2Bin%2Bexponential%2Bgrowth.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2010.04.015%26rft_id%253Dinfo%253Apmid%252F20434381%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2010.04.015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20434381&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3467.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278950800015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.43" data-doi="10.1016/j.cell.2013.05.049"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shah</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Niemczyk</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kudla</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plotkin</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Rate-limiting steps in yeast protein translation.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">153</span>: <span class="cit-fpage">1589</span>–<span class="cit-lpage">1601</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D153%26rft.spage%253D1589%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2013.05.049%26rft_id%253Dinfo%253Apmid%252F23791185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2013.05.049&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23791185&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3467.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000320722000025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.44" data-doi="10.1093/nar/gki242"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sharp</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bailes</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grocock</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peden</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sockett</span>  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Variation in the strength of selected codon usage bias among bacteria.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">1141</span>–<span class="cit-lpage">1153</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DSharp%26rft.auinit1%253DP.%2BM.%26rft.volume%253D33%26rft.issue%253D4%26rft.spage%253D1141%26rft.epage%253D1153%26rft.atitle%253DVariation%2Bin%2Bthe%2Bstrength%2Bof%2Bselected%2Bcodon%2Busage%2Bbias%2Bamong%2Bbacteria%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgki242%26rft_id%253Dinfo%253Apmid%252F15728743%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjMzLzQvMTE0MSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzQ2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.45" data-doi="10.1073/pnas.71.4.1342"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shine</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dalgarno</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">The 3′-terminal sequence of Escherichia coli 16S ribosomal RNA: complementarity to nonsense triplets and ribosome binding sites.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">1342</span>–<span class="cit-lpage">1346</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DShine%26rft.auinit1%253DJ.%26rft.volume%253D71%26rft.issue%253D4%26rft.spage%253D1342%26rft.epage%253D1346%26rft.atitle%253DThe%2B3%2527-Terminal%2BSequence%2Bof%2BEscherichia%2Bcoli%2B16S%2BRibosomal%2BRNA%253A%2BComplementarity%2Bto%2BNonsense%2BTriplets%2Band%2BRibosome%2BBinding%2BSites%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.71.4.1342%26rft_id%253Dinfo%253Apmid%252F4598299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI3MS80LzEzNDIiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzExLzM0NjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.46" data-doi="10.1073/pnas.1118018109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shiroguchi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">T. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sims</span>  <span class="cit-name-given-names">P. a.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">X. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Digital RNA sequencing minimizes sequence-dependent bias and amplification noise with optimized single-molecule barcodes.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">1347</span>–<span class="cit-lpage">1352</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1118018109%26rft_id%253Dinfo%253Apmid%252F22232676%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA5LzQvMTM0NyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzQ2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.47" data-doi="10.1093/bioinformatics/btu033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stamatakis</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1312</span>–<span class="cit-lpage">1313</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu033%26rft_id%253Dinfo%253Apmid%252F24451623%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo5OiIzMC85LzEzMTIiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzExLzM0NjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Starmer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stomp</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vouk</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bitzer</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Predicting Shine-Dalgarno sequence locations exposes genome annotation errors.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">454</span>–<span class="cit-lpage">466</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BComput.%2BBiol.%26rft.volume%253D2%26rft.spage%253D454%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=000239493900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.49" data-doi="10.1093/molbev/msl166"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stoletzki</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Synonymous codon usage in Escherichia coli: selection for translational accuracy.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">374</span>–<span class="cit-lpage">381</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsl166%26rft_id%253Dinfo%253Apmid%252F17101719%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjI0LzIvMzc0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNDY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.50" data-doi="10.1371/journal.pgen.1001004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Supek</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skunca</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Repar</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vlahovicek</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smuc</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Translational selection is ubiquitous in prokaryotes.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1001004</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DSupek%26rft.auinit1%253DF.%26rft.volume%253D6%26rft.issue%253D6%26rft.spage%253De1001004%26rft.epage%253De1001004%26rft.atitle%253DTranslational%2Bselection%2Bis%2Bubiquitous%2Bin%2Bprokaryotes.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1001004%26rft_id%253Dinfo%253Apmid%252F20585573%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1001004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20585573&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3467.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.51" data-doi="10.1126/science.1188308"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Taniguchi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">G.-W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Babu</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Quantifying E. coli proteome and transcriptome with single-molecule sensitivity in single cells.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">329</span>: <span class="cit-fpage">533</span>–<span class="cit-lpage">538</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTaniguchi%26rft.auinit1%253DY.%26rft.volume%253D329%26rft.issue%253D5991%26rft.spage%253D533%26rft.epage%253D538%26rft.atitle%253DQuantifying%2BE.%2Bcoli%2BProteome%2Band%2BTranscriptome%2Bwith%2BSingle-Molecule%2BSensitivity%2Bin%2BSingle%2BCells%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1188308%26rft_id%253Dinfo%253Apmid%252F20671182%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjkvNTk5MS81MzMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzExLzM0NjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.52" data-doi="10.1073/pnas.0909910107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tuller</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waldman</span>  <span class="cit-name-given-names">Y. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kupiec</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruppin</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Translation efficiency is determined by both codon bias and folding energy.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">3645</span>–<span class="cit-lpage">3650</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0909910107%26rft_id%253Dinfo%253Apmid%252F20133581%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA3LzgvMzY0NSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzQ2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vasquez</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hatridge</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Curtis</span>  <span class="cit-name-given-names">N. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Contreras</span>  <span class="cit-name-given-names">L. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Slowing translation between protein domains by increasing affinity between mRNAs and the ribosomal anti-Shine-Dalgarno sequence improves solubility.</span> <abbr class="cit-jnl-abbrev">ACS Synth. Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">133</span>–<span class="cit-lpage">145</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DACS%2BSynth.%2BBiol.%26rft.volume%253D5%26rft.spage%253D133%26rft_id%253Dinfo%253Apmid%252F26607828%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=26607828&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3467.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.54" data-doi="10.1371/journal.pgen.1000808"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vieira-Silva</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rocha</span>  <span class="cit-name-given-names">E. P. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The systemic imprint of growth and its uses in ecological (meta)genomics.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1000808</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DVieira-Silva%26rft.auinit1%253DS.%26rft.volume%253D6%26rft.issue%253D1%26rft.spage%253De1000808%26rft.epage%253De1000808%26rft.atitle%253DThe%2Bsystemic%2Bimprint%2Bof%2Bgrowth%2Band%2Bits%2Buses%2Bin%2Becological%2B%2528meta%2529genomics.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000808%26rft_id%253Dinfo%253Apmid%252F20090831%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000808&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20090831&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3467.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.55" data-doi="10.1093/molbev/msi126"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wagner</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Energy constraints on the evolution of gene expression.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1365</span>–<span class="cit-lpage">1370</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsi126%26rft_id%253Dinfo%253Apmid%252F15758206%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjIyLzYvMTM2NSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzQ2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.56" data-doi="10.1002/pmic.201400441"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herrmann</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simonovic</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Szklarczyk</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">von Mering</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Version 4.0 of PaxDb: protein abundance data, integrated across model organisms, tissues, and cell-lines.</span> <abbr class="cit-jnl-abbrev">Proteomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">3163</span>–<span class="cit-lpage">3168</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProteomics%26rft.volume%253D15%26rft.spage%253D3163%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fpmic.201400441%26rft_id%253Dinfo%253Apmid%252F25656970%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/pmic.201400441&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25656970&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3467.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.57" data-doi="10.1016/j.celrep.2016.01.043"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weinberg</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shah</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eichhorn</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hussmann</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plotkin</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Improved ribosome-footprint and mRNA measurements provide insights into dynamics and regulation of yeast translation.</span> <abbr class="cit-jnl-abbrev">Cell Rep.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">1787</span>–<span class="cit-lpage">1799</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BRep.%26rft.volume%253D14%26rft.spage%253D1787%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.celrep.2016.01.043%26rft_id%253Dinfo%253Apmid%252F26876183%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.celrep.2016.01.043&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26876183&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3467.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Whitaker</span>  <span class="cit-name-given-names">W. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arkin</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dueber</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Avoidance of truncated proteins from unintended ribosome binding sites within heterologous protein coding sequences.</span> <abbr class="cit-jnl-abbrev">ACS Synth. Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">249</span>–<span class="cit-lpage">257</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DACS%2BSynth.%2BBiol.%26rft.volume%253D4%26rft.spage%253D249%26rft_id%253Dinfo%253Apmid%252F24931615%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24931615&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3467.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3467.59" data-doi="10.1016/j.celrep.2015.03.014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Woolstenhulme</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guydosh</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buskirk</span>  <span class="cit-name-given-names">A. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">High-precision analysis of translational pausing by ribosome profiling in bacteria lacking EFP.</span> <abbr class="cit-jnl-abbrev">Cell Rep.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">13</span>–<span class="cit-lpage">21</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BRep.%26rft.volume%253D11%26rft.spage%253D13%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.celrep.2015.03.014%26rft_id%253Dinfo%253Apmid%252F25843707%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.celrep.2015.03.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25843707&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3467.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/11/3467.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/11/3455" title="Proper Sterol Distribution Is Required for Candida albicans Hyphal Formation and Virulence" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/11/3475" title="Multiple Transcript Properties Related to Translation Affect mRNA Degradation Rates in Saccharomyces cerevisiae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/11"><span>Volume 6 Issue 11, November 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/11.cover-source.jpg?itok=hTYn4PRy" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/11/3467" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-TaU4lInTCT0pGecZiaNF2vgQigV_yIXnmOTXhV-lQfA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/11/3467" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/11/3467" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/11/3467" class="active" data-icon-position="" data-hide-link-title="0">Depletion of Shine-Dalgarno Sequences Within Bacterial Coding Regions Is Expression Dependent</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460474" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-WiZie7kaDTK-g41QlbezdKUI70V8iDxxO21eAol5_M8" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/11/3467.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/11/3467" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-RQptth66wT6Y_5xNSKMiTQB21lAyPrUSQdSDnDzGg2Y" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/11/3467" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/11/3467.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460474" id="node460474--2" data-pisa="ggg;6/11/3467" data-pisa-master="ggg;g3.116.032227" data-apath="/ggg/6/11/3467.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Depletion of Shine-Dalgarno Sequences Within Bacterial Coding Regions Is Expression Dependent</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Chuyue Yang</span>, <span class="highwire-citation-author" data-delta="1">Adam J. Hockenberry</span>, <span class="highwire-citation-author" data-delta="2">Michael C. Jewett</span> and <span class="highwire-citation-author" data-delta="3">Luís A. N. Amaral</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3467-3474;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.032227 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460474--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Chuyue Yang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Chemical and Biological Engineering, Northwestern University, Evanston, Illinois 60208</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chuyue%2BYang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yang%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChuyue%2BYang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Adam J. Hockenberry </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Chemical and Biological Engineering, Northwestern University, Evanston, Illinois 60208</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Interdisciplinary Program in Biological Sciences, Northwestern University, Evanston, Illinois 60208</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Adam%2BJ.%2BHockenberry%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hockenberry%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAdam%2BJ.%2BHockenberry%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Michael C. Jewett </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Chemical and Biological Engineering, Northwestern University, Evanston, Illinois 60208</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Interdisciplinary Program in Biological Sciences, Northwestern University, Evanston, Illinois 60208</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Northwestern Institute on Complex Systems, Northwestern University, Evanston, Illinois 60208</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Chemistry of Life Processes Institute, Northwestern University, Evanston, Illinois 60208</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BC.%2BJewett%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jewett%20MC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BC.%2BJewett%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:m-jewett@northwestern.edu" class="" data-icon-position="" data-hide-link-title="0">m-jewett@northwestern.edu</a>
<a href="mailto:amaral@northwestern.edu" class="" data-icon-position="" data-hide-link-title="0">amaral@northwestern.edu</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Luís A. N. Amaral </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Chemical and Biological Engineering, Northwestern University, Evanston, Illinois 60208</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Northwestern Institute on Complex Systems, Northwestern University, Evanston, Illinois 60208</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Physics and Astronomy, Northwestern University, Evanston, Illinois 60208</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lu%C3%ADs%2BA.%2BN.%2BAmaral%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Amaral%20LA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALu%25C3%25ADs%2BA.%2BN.%2BAmaral%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:m-jewett@northwestern.edu" class="" data-icon-position="" data-hide-link-title="0">m-jewett@northwestern.edu</a>
<a href="mailto:amaral@northwestern.edu" class="" data-icon-position="" data-hide-link-title="0">amaral@northwestern.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/11/3467" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/11/3467&amp;title=Depletion%20of%20Shine-Dalgarno%20Sequences%20Within%20Bacterial%20Coding%20Regions%20Is%20Expression%20Dependent" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/11/3467&amp;title=Depletion%20of%20Shine-Dalgarno%20Sequences%20Within%20Bacterial%20Coding%20Regions%20Is%20Expression%20Dependent" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/11/3467&amp;title=Depletion%20of%20Shine-Dalgarno%20Sequences%20Within%20Bacterial%20Coding%20Regions%20Is%20Expression%20Dependent" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/11/3467&amp;text=Depletion%20of%20Shine-Dalgarno%20Sequences%20Within%20Bacterial%20Coding%20Regions%20Is%20Expression%20Dependent" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/11/3467&amp;title=Depletion%20of%20Shine-Dalgarno%20Sequences%20Within%20Bacterial%20Coding%20Regions%20Is%20Expression%20Dependent" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/11/3467&amp;t=Depletion%20of%20Shine-Dalgarno%20Sequences%20Within%20Bacterial%20Coding%20Regions%20Is%20Expression%20Dependent" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/11/3467&amp;title=Depletion%20of%20Shine-Dalgarno%20Sequences%20Within%20Bacterial%20Coding%20Regions%20Is%20Expression%20Dependent" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/11/3467&amp;title=Depletion%20of%20Shine-Dalgarno%20Sequences%20Within%20Bacterial%20Coding%20Regions%20Is%20Expression%20Dependent" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460474/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460474" id="node460474--3" data-pisa="ggg;6/11/3467" data-pisa-master="ggg;g3.116.032227" data-apath="/ggg/6/11/3467.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Depletion of Shine-Dalgarno Sequences Within Bacterial Coding Regions Is Expression Dependent</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Chuyue Yang</span>, <span class="highwire-citation-author" data-delta="1">Adam J. Hockenberry</span>, <span class="highwire-citation-author" data-delta="2">Michael C. Jewett</span> and <span class="highwire-citation-author" data-delta="3">Luís A. N. Amaral</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3467-3474;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.032227 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460474--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Chuyue Yang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Chemical and Biological Engineering, Northwestern University, Evanston, Illinois 60208</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chuyue%2BYang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yang%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChuyue%2BYang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Adam J. Hockenberry </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Chemical and Biological Engineering, Northwestern University, Evanston, Illinois 60208</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Interdisciplinary Program in Biological Sciences, Northwestern University, Evanston, Illinois 60208</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Adam%2BJ.%2BHockenberry%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hockenberry%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAdam%2BJ.%2BHockenberry%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Michael C. Jewett </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Chemical and Biological Engineering, Northwestern University, Evanston, Illinois 60208</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Interdisciplinary Program in Biological Sciences, Northwestern University, Evanston, Illinois 60208</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Northwestern Institute on Complex Systems, Northwestern University, Evanston, Illinois 60208</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Chemistry of Life Processes Institute, Northwestern University, Evanston, Illinois 60208</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BC.%2BJewett%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jewett%20MC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BC.%2BJewett%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:m-jewett@northwestern.edu" class="" data-icon-position="" data-hide-link-title="0">m-jewett@northwestern.edu</a>
<a href="mailto:amaral@northwestern.edu" class="" data-icon-position="" data-hide-link-title="0">amaral@northwestern.edu</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Luís A. N. Amaral </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Chemical and Biological Engineering, Northwestern University, Evanston, Illinois 60208</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Northwestern Institute on Complex Systems, Northwestern University, Evanston, Illinois 60208</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Physics and Astronomy, Northwestern University, Evanston, Illinois 60208</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lu%C3%ADs%2BA.%2BN.%2BAmaral%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Amaral%20LA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALu%25C3%25ADs%2BA.%2BN.%2BAmaral%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:m-jewett@northwestern.edu" class="" data-icon-position="" data-hide-link-title="0">m-jewett@northwestern.edu</a>
<a href="mailto:amaral@northwestern.edu" class="" data-icon-position="" data-hide-link-title="0">amaral@northwestern.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3467&amp;rft.epage=3474&amp;rft.atitle=Depletion%20of%20Shine-Dalgarno%20Sequences%20Within%20Bacterial%20Coding%20Regions%20Is%20Expression%20Dependent&amp;rft.volume=6&amp;rft.issue=11&amp;rft.date=2016-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Yang%2C+Chuyue&amp;rft.au=Hockenberry%2C+Adam+J.&amp;rft.au=Jewett%2C+Michael+C.&amp;rft.au=Amaral%2C+Lu%C3%ADs+A.+N."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460474/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460474/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460474/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460474/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460474/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460474/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460474/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460474/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460474/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460474/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460474/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460474/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/11/3467" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/11/3467.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/11/3467.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/11/3467.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/11/3467.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
