<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/11/3693" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/11/3693.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/11/3693.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/11/3693.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/6/11/3693/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/11/3693.atom" />
<meta name="HW.pisa" content="ggg;6/11/3693" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genomic Prediction of Resistance to Pasteurellosis in Gilthead Sea Bream (Sparus aurata) Using 2b-RAD Sequencing" />
<meta name="DC.Identifier" content="10.1534/g3.116.035220" />
<meta name="DC.Date" content="2016-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Christos et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Gilthead sea bream ( Sparus aurata ) is a species of paramount importance to the Mediterranean aquaculture industry, with an annual production exceeding 140,000 metric tons. Pasteurellosis due to the Gram-negative bacterium Photobacterium damselae subsp. piscicida (Phdp) causes significant mortality, especially during larval and juvenile stages, and poses a serious threat to bream production. Selective breeding for improved resistance to pasteurellosis is a promising avenue for disease control, and the use of genetic markers to predict breeding values can improve the accuracy of selection, and allow accurate calculation of estimated breeding values of nonchallenged animals. In the current study, a population of 825 sea bream juveniles, originating from a factorial cross between 67 broodfish (32 sires, 35 dams), were challenged by 30 min immersion with 1 × 105 CFU virulent Phdp. Mortalities and survivors were recorded and sampled for genotyping by sequencing. The restriction-site associated DNA sequencing approach, 2b-RAD, was used to generate genome-wide single nucleotide polymorphism (SNP) genotypes for all samples. A high-density linkage map containing 12,085 SNPs grouped into 24 linkage groups (consistent with the karyotype) was constructed. The heritability of surviving days (censored data) was 0.22 (95% highest density interval: 0.11–0.36) and 0.28 (95% highest density interval: 0.17–0.4) using the pedigree and the genomic relationship matrix respectively. A genome-wide association study did not reveal individual SNPs significantly associated with resistance at a genome-wide significance level. Genomic prediction approaches were tested to investigate the potential of the SNPs obtained by 2b-RAD for estimating breeding values for resistance. The accuracy of the genomic prediction models ( r = 0.38–0.46) outperformed the traditional BLUP approach based on pedigree records ( r = 0.30). Overall results suggest that major quantitative trait loci affecting resistance to pasteurellosis were not present in this population, but highlight the effectiveness of 2b-RAD genotyping by sequencing for genomic selection in a mass spawning fish species." />
<meta name="DC.Contributor" content="Christos Palaiokostas" />
<meta name="DC.Contributor" content="Serena Ferraresso" />
<meta name="DC.Contributor" content="Rafaella Franch" />
<meta name="DC.Contributor" content="Ross D. Houston" />
<meta name="DC.Contributor" content="Luca Bargelloni" />
<meta name="article:published_time" content="2016-11-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="Genomic Prediction of Resistance to Pasteurellosis in Gilthead Sea Bream (Sparus aurata) Using 2b-RAD Sequencing" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Gilthead sea bream (&lt;i&gt;Sparus aurata&lt;/i&gt;) is a species of paramount importance to the Mediterranean aquaculture industry, with an annual production exceeding 140,000 metric tons. Pasteurellosis due to the Gram-negative bacterium &lt;i&gt;Photobacterium damselae&lt;/i&gt; subsp. &lt;i&gt;piscicida&lt;/i&gt; (Phdp) causes significant mortality, especially during larval and juvenile stages, and poses a serious threat to bream production. Selective breeding for improved resistance to pasteurellosis is a promising avenue for disease control, and the use of genetic markers to predict breeding values can improve the accuracy of selection, and allow accurate calculation of estimated breeding values of nonchallenged animals. In the current study, a population of 825 sea bream juveniles, originating from a factorial cross between 67 broodfish (32 sires, 35 dams), were challenged by 30 min immersion with 1 × 10&lt;sup&gt;5&lt;/sup&gt; CFU virulent Phdp. Mortalities and survivors were recorded and sampled for genotyping by sequencing. The restriction-site associated DNA sequencing approach, 2b-RAD, was used to generate genome-wide single nucleotide polymorphism (SNP) genotypes for all samples. A high-density linkage map containing 12,085 SNPs grouped into 24 linkage groups (consistent with the karyotype) was constructed. The heritability of surviving days (censored data) was 0.22 (95% highest density interval: 0.11–0.36) and 0.28 (95% highest density interval: 0.17–0.4) using the pedigree and the genomic relationship matrix respectively. A genome-wide association study did not reveal individual SNPs significantly associated with resistance at a genome-wide significance level. Genomic prediction approaches were tested to investigate the potential of the SNPs obtained by 2b-RAD for estimating breeding values for resistance. The accuracy of the genomic prediction models (&lt;i&gt;r&lt;/i&gt; = 0.38–0.46) outperformed the traditional BLUP approach based on pedigree records (&lt;i&gt;r&lt;/i&gt; = 0.30). Overall results suggest that major quantitative trait loci affecting resistance to pasteurellosis were not present in this population, but highlight the effectiveness of 2b-RAD genotyping by sequencing for genomic selection in a mass spawning fish species.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/11/01" />
<meta name="citation_mjid" content="ggg;6/11/3693" />
<meta name="citation_id" content="6/11/3693" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/11/3693" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/11/3693.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/11/3693.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/11/3693.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.035220" />
<meta name="citation_pmid" content="27652890" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="3693" />
<meta name="citation_lastpage" content="3700" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Christos Palaiokostas" />
<meta name="citation_author_institution" content="The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Midlothian, EH25 9RG Scotland, United Kingdom" />
<meta name="citation_author_email" content="christos.palaiokostas@roslin.ed.ac.uk" />
<meta name="citation_author_email" content="luca.bargelloni@unipd.it" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4480-4612" />
<meta name="citation_author" content="Serena Ferraresso" />
<meta name="citation_author_institution" content="Department of Comparative Biomedicine and Food Science, University of Padova, 35020 Legnaro, Italy" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5928-9361" />
<meta name="citation_author" content="Rafaella Franch" />
<meta name="citation_author_institution" content="Department of Comparative Biomedicine and Food Science, University of Padova, 35020 Legnaro, Italy" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1333-0733" />
<meta name="citation_author" content="Ross D. Houston" />
<meta name="citation_author_institution" content="The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Midlothian, EH25 9RG Scotland, United Kingdom" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1805-0762" />
<meta name="citation_author" content="Luca Bargelloni" />
<meta name="citation_author_institution" content="Department of Comparative Biomedicine and Food Science, University of Padova, 35020 Legnaro, Italy" />
<meta name="citation_author_email" content="christos.palaiokostas@roslin.ed.ac.uk" />
<meta name="citation_author_email" content="luca.bargelloni@unipd.it" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4351-2826" />
<meta name="citation_reference" content="citation_journal_title=Aquaculture;citation_author=J. Antonello;citation_author=C. Massault;citation_author=R. Franch;citation_author=C. Haley;citation_author=C. Pellizzari;citation_title=Estimates of heritability and genetic correlation for body length and resistance to fish pasteurellosis in the gilthead sea bream (Sparus aurata L.).;citation_pages=29-35;citation_volume=298;citation_year=2009;citation_doi=10.1016/j.aquaculture.2009.10.022" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=N. A. Baird;citation_author=P. D. Etter;citation_author=T. S. Atwood;citation_author=M. C. Currey;citation_author=A. L. Shiver;citation_title=Rapid SNP discovery and genetic mapping using sequenced RAD markers.;citation_pages=e3376-e3376;citation_volume=3;citation_year=2008;citation_issue=10;citation_pmid=18852878;citation_doi=10.1371/journal.pone.0003376" />
<meta name="citation_reference" content="citation_journal_title=Livest. Sci.;citation_author=S. C. Bishop;citation_author=J. A. Woolliams;citation_title=Genomics and disease resistance studies in livestock.;citation_pages=190-198;citation_volume=166;citation_year=2014;citation_pmid=26339300;citation_doi=10.1016/j.livsci.2014.04.034" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=N. R. Campbell;citation_author=S. E. LaPatra;citation_author=K. Overturf;citation_author=R. Towner;citation_author=S. R. Narum;citation_title=Association mapping of disease resistance traits in rainbow trout using restriction site associated DNA sequencing.;citation_volume=4;citation_year=2014;citation_pmid=25354781;citation_doi=10.1534/g3.114.014621" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda, Md.);citation_author=J. M. Catchen;citation_author=A. Amores;citation_author=P. Hohenlohe;citation_author=W. Cresko;citation_author=J. H. Postlethwait;citation_title=Stacks: Building and Genotyping Loci De Novo from Short-Read Sequences.;citation_pages=171-82;citation_volume=1;citation_year=2011;citation_doi=10.1534/g3.111.000240" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. A. Chistiakov;citation_author=B. Hellemans;citation_author=C. S. Haley;citation_author=A. S. Law;citation_author=C. S. Tsigenopoulos;citation_title=A microsatellite linkage map of the European sea bass Dicentrarchus labrax L.;citation_volume=170;citation_year=2005;citation_pmid=15937133;citation_doi=10.1534/genetics.104.039719" />
<meta name="citation_reference" content="citation_journal_title=Animal genetics;citation_journal_abbrev=Anim Genet;citation_author=D. A. Chistiakov;citation_author=C. S. Tsigenopoulos;citation_author=J. Lagnel;citation_author=Y.-M. Guo;citation_author=B. Hellemans;citation_title=A combined AFLP and microsatellite linkage map and pilot comparative genomic analysis of European sea bass Dicentrarchus labrax L.;citation_pages=623-634;citation_volume=39;citation_year=2008;citation_issue=6;citation_pmid=18828863;citation_doi=10.1111/j.1365-2052.2008.01786.x" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. W. Davey;citation_author=P. A. Hohenlohe;citation_author=P. D. Etter;citation_author=J. Q. Boone;citation_author=J. M. Catchen;citation_title=Genome-wide genetic marker discovery and genotyping using next-generation sequencing.;citation_pages=499-510;citation_volume=12;citation_year=2011;citation_issue=7;citation_pmid=21681211;citation_doi=10.1038/nrg3012" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=J. M. Hickey;citation_author=R. Pong-Wong;citation_author=H. D. Daetwyler;citation_author=M. P. L. Calus;citation_title=Whole-genome regression and prediction methods applied to plant and animal breeding.;citation_volume=193;citation_year=2013;citation_pmid=22745228;citation_doi=10.1534/genetics.112.143313" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=J. Dou;citation_author=X. Li;citation_author=Q. Fu;citation_author=W. Jiao;citation_author=Y. Li;citation_title=Evaluation of the 2b-RAD method for genomic selection in scallop breeding.;citation_pages=19244;citation_volume=6;citation_year=2016;citation_pmid=26754638;citation_doi=10.1038/srep19244" />
<meta name="citation_reference" content="citation_journal_title=The Plant Genome Journal;citation_author=J. B. Endelman;citation_title=Ridge regression and other kernels for genomic selection with R package rrBLUP.;citation_pages=250;citation_volume=4;citation_year=2011;citation_doi=10.3835/plantgenome2011.08.0024" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Gianola;citation_title=Priors in whole-genome regression: the Bayesian alphabet returns.;citation_volume=194;citation_year=2013;citation_pmid=23636739;citation_doi=10.1534/genetics.113.151753" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=M. E. Goddard;citation_author=B. J. Hayes;citation_title=Mapping genes for complex traits in domestic animals and their use in breeding programmes.;citation_pages=381-391;citation_volume=10;citation_year=2009;citation_issue=6;citation_pmid=19448663;citation_doi=10.1038/nrg2575" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=S. Gonen;citation_author=N. R. Lowe;citation_author=T. Cezard;citation_author=K. Gharbi;citation_author=S. C. Bishop;citation_title=Linkage maps of the Atlantic salmon (Salmo salar)genome derived from RAD sequencing.;citation_pages=166;citation_volume=15;citation_year=2014;citation_pmid=24571138;citation_doi=10.1186/1471-2164-15-166" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=K. Kizilkaya;citation_author=D. J. Garrick;citation_title=Extension of the bayesian alphabet for genomic selection.;citation_pages=186-186;citation_volume=12;citation_year=2011;citation_pmid=21605355;citation_doi=10.1186/1471-2105-12-186" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_journal_abbrev=Biometrics;citation_author=C. R. Henderson;citation_title=Best linear unbiased estimation and prediction under a selection model.;citation_pages=423-447;citation_volume=31;citation_year=1975;citation_issue=2;citation_pmid=1174616;citation_doi=10.2307/2529430" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=B. Heringstad;citation_author=G. Klemetsdal;citation_author=T. Steine;citation_title=Selection Responses for Clinical Mastitis and Protein Yield in Two Norwegian Dairy Cattle Selection Experiments;citation_pages=2990-2999;citation_volume=86;citation_year=2003;citation_issue=9;citation_pmid=14507036;citation_doi=10.3168/jds.S0022-0302(03)73897-1" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. D. Houston;citation_author=C. S. Haley;citation_author=A. Hamilton;citation_author=D. R. Guy;citation_author=A. E. Tinch;citation_title=Major quantitative trait loci affect resistance to infectious pancreatic necrosis in Atlantic salmon (Salmo salar).;citation_volume=178;citation_year=2008;citation_pmid=18245341;citation_doi=10.1534/genetics.107.082974" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=R. D. Houston;citation_author=C. S. Haley;citation_author=A. Hamilton;citation_author=D. R. Guy;citation_author=J. C. Mota-Velasco;citation_title=The susceptibility of Atlantic salmon fry to freshwater infectious pancreatic necrosis is largely explained by a major QTL.;citation_pages=318-327;citation_volume=105;citation_year=2010;citation_issue=3;citation_pmid=19935825;citation_doi=10.1038/hdy.2009.171" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=R. D. Houston;citation_author=J. B. Taggart;citation_author=T. Cézard;citation_author=M. Bekaert;citation_author=N. R. Lowe;citation_title=Development and validation of a high density SNP genotyping array for Atlantic salmon (Salmo salar).;citation_pages=90;citation_volume=15;citation_year=2014;citation_pmid=24524230;citation_doi=10.1186/1471-2164-15-90" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Sci.;citation_author=K. Kizilkaya;citation_author=R. L. Fernando;citation_author=D. J. Garrick;citation_title=Genomic prediction of simulated multibreed and purebred performance using observed fifty thousand single nucleotide polymorphism genotypes.;citation_volume=88;citation_year=2010;citation_pmid=19820059;citation_doi=10.2527/jas.2009-2064" />
<meta name="citation_reference" content="citation_journal_title=Aquaculture;citation_author=B. Magarinos;citation_author=N. Couso;citation_author=M. Noya;citation_author=P. Merino;citation_author=A. E. Toranzo;citation_title=Effect of temperature on the development of pasteurellosis in carrier gilthead seabream (Sparus aurata).;citation_pages=17-21;citation_volume=195;citation_year=2001;citation_doi=10.1016/S0044-8486(00)00547-0" />
<meta name="citation_reference" content="citation_journal_title=Animal genetics;citation_journal_abbrev=Anim Genet;citation_author=C. Massault;citation_author=B. Hellemans;citation_author=B. Louro;citation_author=C. Batargias;citation_author=J. K. J. Van Houdt;citation_title=QTL for body weight, morphometric traits and stress response in European sea bass Dicentrarchus labrax.;citation_pages=337-345;citation_volume=41;citation_year=2010;citation_issue=4;citation_pmid=20028379" />
<meta name="citation_reference" content="citation_journal_title=Anim. Genet.;citation_author=C. Massault;citation_author=R. Franch;citation_author=C. Haley;citation_author=D. J. De Koning;citation_author=H. Bovenhuis;citation_title=Quantitative trait loci for resistance to fish pasteurellosis in gilthead sea bream (Sparus aurata).;citation_pages=191-203;citation_volume=42;citation_year=2011;citation_pmid=20946317;citation_doi=10.1111/j.1365-2052.2010.02110.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=T. Moen;citation_author=M. Baranski;citation_author=A. K. Sonesson;citation_author=S. Kjøglum;citation_title=Confirmation and fine-mapping of a major QTL for resistance to infectious pancreatic necrosis in Atlantic salmon (Salmo salar): population-level associations between markers and trait.;citation_pages=368-368;citation_volume=10;citation_year=2009;citation_pmid=19664221;citation_doi=10.1186/1471-2164-10-368" />
<meta name="citation_reference" content="citation_journal_title=Aquaculture;citation_author=M. Noya;citation_author=B. Magariños;citation_author=J. Lamas;citation_title=Interactions between peritoneal exudate cells (PECs) of gilthead seabream (Sparus aurata) and Pasteurella piscicida. A morphological study.;citation_pages=11-21;citation_volume=131;citation_year=1995;citation_doi=10.1016/0044-8486(94)00353-P" />
<meta name="citation_reference" content="citation_journal_title=Aquacult. Res.;citation_author=J. Ødegård;citation_author=M. Baranski;citation_author=B. Gjerde;citation_author=T. Gjedrem;citation_title=Methodology for genetic evaluation of disease resistance in aquaculture species: challenges and future prospects.;citation_pages=103-114;citation_volume=42;citation_year=2011;citation_doi=10.1111/j.1365-2109.2010.02669.x" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=J. Odegård;citation_author=T. Moen;citation_author=N. Santi;citation_author=S. A. Korsvoll;citation_author=S. Kjøglum;citation_title=Genomic prediction in an admixed population of Atlantic salmon (Salmo salar).;citation_pages=402;citation_volume=5;citation_year=2014;citation_pmid=25484890" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=C. Palaiokostas;citation_author=M. Bekaert;citation_author=A. Davie;citation_author=M. E. Cowan;citation_author=M. Oral;citation_title=Mapping the sex determination locus in the Atlantic halibut (Hippoglossus hippoglossus) using RAD sequencing.;citation_pages=566;citation_volume=14;citation_year=2013;citation_pmid=23957753;citation_doi=10.1186/1471-2164-14-566" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=C. Palaiokostas;citation_author=M. Bekaert;citation_author=M. G. Q. Khan;citation_author=J. B. Taggart;citation_author=K. Gharbi;citation_title=Mapping and validation of the major sex-determining region in Nile Tilapia (Oreochromis niloticus L.) using RAD sequencing.;citation_pages=8;citation_volume=8;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=GSE;citation_author=C. Palaiokostas;citation_author=M. Bekaert;citation_author=J. B. Taggart;citation_author=K. Gharbi;citation_author=B. J. McAndrew;citation_title=A new SNP-based vision of the genetics of sex determination in European sea bass (Dicentrarchus labrax). Genetics, selection, evolution;citation_pages=68;citation_volume=47;citation_year=2015;citation_pmid=26337592" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=M. Pauletto;citation_author=L. Carraro;citation_author=M. Babbucci;citation_author=R. Lucchini;citation_author=L. Bargelloni;citation_title=Extending RAD tag analysis to microbial ecology: a comparison between multilocus sequence typing and 2b-RAD to investigate Listeria monocytogenes genetic structure.;citation_pages=823-835;citation_volume=16;citation_year=2016;citation_pmid=26613186;citation_doi=10.1111/1755-0998.12495" />
<meta name="citation_reference" content="citation_journal_title=Mar. Genomics;citation_author=C. Pecoraro;citation_author=M. Babbucci;citation_author=A. Villamor;citation_author=R. Franch;citation_author=C. Papetti;citation_title=Methodological assessment of 2b-RAD genotyping technique for population structure inferences in yellowfin tuna (Thunnus albacares).;citation_pages=43-48;citation_volume=25;citation_year=2016;citation_pmid=26711352;citation_doi=10.1016/j.margen.2015.12.002" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. Pérez;citation_author=G. de Los Campos;citation_title=Genome-wide regression and prediction with the BGLR statistical package.;citation_volume=198;citation_year=2014;citation_pmid=25009151;citation_doi=10.1534/genetics.114.164442" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=J. B. Puritz;citation_author=M. V. Matz;citation_author=R. J. Toonen;citation_author=J. N. Weber;citation_author=D. I. Bolnick;citation_title=Demystifying the RAD fad.;citation_pages=5937-5942;citation_volume=23;citation_year=2014;citation_doi=10.1111/mec.12965" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Rastas;citation_author=L. Paulin;citation_author=I. Hanski;citation_author=R. Lehtonen;citation_author=P. Auvinen;citation_title=Lep-MAP: fast and accurate linkage map construction for large SNP datasets.;citation_volume=29;citation_year=2013;citation_pmid=24078685;citation_doi=10.1093/bioinformatics/btt563" />
<meta name="citation_reference" content="citation_journal_title=J. Stat. Softw.;citation_author=B. J. Smith;citation_title=Boa : an R package for MCMC output convergence.;citation_pages=1-37;citation_volume=21;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=H.-Y. Tsai;citation_author=A. Hamilton;citation_author=A. E. Tinch;citation_author=D. R. Guy;citation_author=K. Gharbi;citation_title=Genome wide association and genomic prediction for growth traits in juvenile farmed Atlantic salmon using a high density SNP array.;citation_pages=969;citation_volume=16;citation_year=2015;citation_pmid=26582102;citation_doi=10.1186/s12864-015-2117-9" />
<meta name="citation_reference" content="citation_journal_title=Mar. Genomics;citation_author=C. S. Tsigenopoulos;citation_author=B. Louro;citation_author=D. Chatziplis;citation_author=J. Lagnel;citation_author=E. Vogiatzi;citation_title=Second generation genetic linkage map for the gilthead sea bream Sparus aurata L.;citation_pages=77-82;citation_volume=18;citation_year=2014;citation_pmid=25307204;citation_doi=10.1016/j.margen.2014.09.008" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=S. Wang;citation_author=E. Meyer;citation_author=J. K. McKay;citation_author=M. V. Matz;citation_title=2b-RAD: a simple and flexible method for genome-wide genotyping.;citation_pages=808-810;citation_volume=9;citation_year=2012;citation_issue=8;citation_pmid=22609625;citation_doi=10.1038/nmeth.2023" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=V. Wimmer;citation_author=T. Albrecht;citation_author=H.-J. Auinger;citation_author=C.-C. Schön;citation_title=Synbreed: a framework for the analysis of genomic prediction data using R.;citation_volume=28;citation_year=2012;citation_pmid=22689388;citation_doi=10.1093/bioinformatics/bts335" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=J. M. Yáñez;citation_author=R. D. Houston;citation_author=S. Newman;citation_title=Genetics and genomics of disease resistance in salmonid species.;citation_pages=415;citation_volume=5;citation_year=2014;citation_pmid=25505486" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Yu;citation_author=G. Pressoir;citation_author=W. H. Briggs;citation_author=I. Vroh Bi;citation_author=M. Yamasaki;citation_title=A unified mixed-model method for association mapping that accounts for multiple levels of relatedness.;citation_pages=203-208;citation_volume=38;citation_year=2006;citation_issue=2;citation_pmid=16380716;citation_doi=10.1038/ng1702" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genomic Prediction of Resistance to Pasteurellosis in Gilthead Sea Bream (Sparus aurata) Using 2b-RAD Sequencing" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11.cover-source.jpg" />
<meta name="twitter:description" content="Gilthead sea bream ( Sparus aurata ) is a species of paramount importance to the Mediterranean aquaculture industry, with an annual production exceeding 140,000 metric tons. Pasteurellosis due to the Gram-negative bacterium Photobacterium damselae subsp. piscicida (Phdp) causes significant mortality, especially during larval and juvenile stages, and poses a serious threat to bream production. Selective breeding for improved resistance to pasteurellosis is a promising avenue for disease control, and the use of genetic markers to predict breeding values can improve the accuracy of selection, and allow accurate calculation of estimated breeding values of nonchallenged animals. In the current study, a population of 825 sea bream juveniles, originating from a factorial cross between 67 broodfish (32 sires, 35 dams), were challenged by 30 min immersion with 1 × 105 CFU virulent Phdp. Mortalities and survivors were recorded and sampled for genotyping by sequencing. The restriction-site associated DNA sequencing approach, 2b-RAD, was used to generate genome-wide single nucleotide polymorphism (SNP) genotypes for all samples. A high-density linkage map containing 12,085 SNPs grouped into 24 linkage groups (consistent with the karyotype) was constructed. The heritability of surviving days (censored data) was 0.22 (95% highest density interval: 0.11–0.36) and 0.28 (95% highest density interval: 0.17–0.4) using the pedigree and the genomic relationship matrix respectively. A genome-wide association study did not reveal individual SNPs significantly associated with resistance at a genome-wide significance level. Genomic prediction approaches were tested to investigate the potential of the SNPs obtained by 2b-RAD for estimating breeding values for resistance. The accuracy of the genomic prediction models ( r = 0.38–0.46) outperformed the traditional BLUP approach based on pedigree records ( r = 0.30). Overall results suggest that major quantitative trait loci affecting resistance to pasteurellosis were not present in this population, but highlight the effectiveness of 2b-RAD genotyping by sequencing for genomic selection in a mass spawning fish species." />
<meta name="og-title" property="og:title" content="Genomic Prediction of Resistance to Pasteurellosis in Gilthead Sea Bream (Sparus aurata) Using 2b-RAD Sequencing" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/11/3693" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Gilthead sea bream ( Sparus aurata ) is a species of paramount importance to the Mediterranean aquaculture industry, with an annual production exceeding 140,000 metric tons. Pasteurellosis due to the Gram-negative bacterium Photobacterium damselae subsp. piscicida (Phdp) causes significant mortality, especially during larval and juvenile stages, and poses a serious threat to bream production. Selective breeding for improved resistance to pasteurellosis is a promising avenue for disease control, and the use of genetic markers to predict breeding values can improve the accuracy of selection, and allow accurate calculation of estimated breeding values of nonchallenged animals. In the current study, a population of 825 sea bream juveniles, originating from a factorial cross between 67 broodfish (32 sires, 35 dams), were challenged by 30 min immersion with 1 × 105 CFU virulent Phdp. Mortalities and survivors were recorded and sampled for genotyping by sequencing. The restriction-site associated DNA sequencing approach, 2b-RAD, was used to generate genome-wide single nucleotide polymorphism (SNP) genotypes for all samples. A high-density linkage map containing 12,085 SNPs grouped into 24 linkage groups (consistent with the karyotype) was constructed. The heritability of surviving days (censored data) was 0.22 (95% highest density interval: 0.11–0.36) and 0.28 (95% highest density interval: 0.17–0.4) using the pedigree and the genomic relationship matrix respectively. A genome-wide association study did not reveal individual SNPs significantly associated with resistance at a genome-wide significance level. Genomic prediction approaches were tested to investigate the potential of the SNPs obtained by 2b-RAD for estimating breeding values for resistance. The accuracy of the genomic prediction models ( r = 0.38–0.46) outperformed the traditional BLUP approach based on pedigree records ( r = 0.30). Overall results suggest that major quantitative trait loci affecting resistance to pasteurellosis were not present in this population, but highlight the effectiveness of 2b-RAD genotyping by sequencing for genomic selection in a mass spawning fish species." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11.cover-source.jpg" />
    <title>Genomic Prediction of Resistance to Pasteurellosis in Gilthead Sea Bream (Sparus aurata) Using 2b-RAD Sequencing | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"OXm2xL_dH7-DKM3MGJt3p-jNN3GUPJwUtzUWP8r4HPM","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460461","apath":"\/ggg\/6\/11\/3693.atom","pisa":"ggg;6\/11\/3693","ac":{"\/ggg\/6\/11\/3693.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/11\/3693.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/11\/3693"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/11\/3693"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/11\/3693":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022qz2hAi943pfqE1XN2oMSTXoX4CoEW6IB4NbGfim5Smx22vmPb9WRpYk6I\\\\\\\/JRNhxuFKfrx5DvMZvtAL72AWoxAEV1f1Tf1vXN6OQTDvq3fOlcXl6CK4OWHsTerSn3lMfmUJADTd5GRpQaiBEWYBy47zRIE56reoW7Ddb7SGnOhpk3h0q4GhsIhNR6hxb8BitfSkHXknCVmI6KY8X6JCC6YZ1ZU09CjiGS7ErlwFJmIuKfK\\\\\\\/qeqHS+I1Dj06tNT6fKrUCdO6S6DrFRyiw+VeYocn8NgLdEtqJzorBX9+itHtSCn9Rs8H5wGCC8h5eKerdYMxmhoui8IeANpXDoGFS1zVJe3t9XNhzfaYvUX2c8jmK4H6yGeG66OGih3HC9LhmFZWjTXzXywlqIep6PqH\\\\\\\/FlSJk2ixcCl7b4Di8GGsat2qYOjyRNBogTd5XDlG6ju94V8pK+9UPA3JUtfBhWHMsWPNro8c4vBjGmmGDtQohB1j0pb4e4E65nug41mQgZ0h9Jhiqr0ypVDs\\\\\\\/hc\\\\\\\/U3+8nwQdLq+ai3Q2afJ2pat04kQ4tmLauzRPpIpAKXztz9N3zdIdaX9hsMuy\\\\\\\/PuUdHXf9FuBsZBLTcCpF9WhatWMvzpNuoj5KC2yjOMTzp7feZ46YEAIuZmML7suQ9fv1eDJjSHaMYlSGM6Fw2qIQP3l5DrmEWjXbkwjBYnrra1\\\\\\\/tURUe2sy5KsmkZNDZeuiEGY5CJCSqHuuO5rXRMxrD0uJTW7h0TxyYzQCk+nlW0O+5lI+WB28Ls5i7VRfoV4mvFg7eAob4g1ggibLWSymPQRv56bAl6V1z0VAxuLblxYCX6xkQkEk1E3K49hp42SRJd\\\\\\\/oKw4yInhP1CRvI290\\\\\\\/dtiH4qd7o35Tioex5rmyZBfTdDz1EFe\\\\\\\/bOK35TJ88izjqoooJYnQo3QSKzodoKbt98DRVfMwoLzP6CIiIpLaJe2EY9Zbkq4kAtlzcjd2aIpiNJCoRDMAKYJy825SdSk54QzkQbXq4kwykWJLd4p0bwFKqiell8m1lvCECecqSR1vqT0PJZUc0M+V9RiyXkEAuI9Wv1DyLagBOmrQHwM\\\\\\\/HTE3mX28SDDlq\\\\\\\/iQ3\\\\\\\/bJUKiZX55j3yHcnwrOP3FIwo1mtp7bbkeLsR2lNXsbKq4ZQTl8qeMRUzWhg3n+QQqtbdieertHZL7fQiWeOpvqcOmxbXH+qQiTl8avKTHy7EEaL0RKxBCdhVJpePkA7TvO6F8c6PSnZQ0xZIaY1fGLPeakmQz41QCOnXId7Y+QARKKmbUNp\\\\\\\/JQ\\\u0022,\\\u0022iv\\\u0022:\\\u0022lZrf2wBA80qymV7Di\\\\\\\/2Ksg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225055ac3b561ee52ac08713a828b88b5b67e91833beaa679fcacfc849369d16d7\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00228D9zao\\\\\\\/mmvJR9c9G2IfqCePcGA9d9sHA7GbRrjyYRGg9nK1M0C3qrtIPd698pfV9HdaclFJG\\\\\\\/dBanpzI0cPsfEoijAlvER0ZFZ53P9\\\\\\\/e9kFETF9cOm\\\\\\\/gwvftcW24wNP76eLrTcjzQYVt8BfcSBHDMpRzH22Rah8QytFLSHPkYLUqZTbkd+CC36KhKlhgA8Rq\\\\\\\/voyfMOYUzzt4ZKITfQPSmGeOr0un9hTe3\\\\\\\/wG\\\\\\\/4jHt6yja+v7pcI9\\\\\\\/5xkD8LKt7lOL620RCHkyFNwrLczAYChe8au\\\\\\\/vxLSoymwBlgiaTSwKwbsNgp76KEr7aHEwn+Dg4JCB36T4hDdfLoEpYxS1M8NskY3wkaLNwEPDiZI4gYNFND+04NUstp5CKUd5oGVM1zTLbobQTp9s4p0QbLjp6GSY8sd15vwHX1yppBxfSSkqzAX8HBf3yEB5kwMkB19Jo5\\\\\\\/5q2ZUKU45LTAXV19df8fYqGdsGhOvKhvE6s3qxoJ6UhJDIy+qICxek7WvAbBmaKxe3Pp14ya5XKhmiFf8GDVP2qD1iXMgAWVpgYfRtcCsz5y599GYzZGo80OyXyEQIlbygX\\\\\\\/j5++xdK2\\\\\\\/88TvwL4ef52eBu6cvkmcqqC8CiOiX8NDO93N4I9sV1WVmZlAjbKMWIqLc9nguvidtgZtdyUKEubocDzZeMoostp07aA+kRC3rVjzxR2tSK2hDhj5vbhHv4AvC0XFf5c74bAldI56Vws9K0KSUlKLPYX9exsWML0n3rm4KhUFEpZz8dG07le4sZe7PCDsblTSLKFV3e8GB0QwBG\\\\\\\/ZLwp7taUjK8J2rD487egm6OayNYvb4Rw0ZnswyQ\\\\\\\/i5yoGN54eSz8VjiYfzmCeRh4kJZtfIdmaWMUxahJM1pyNsg37+Uak7pVgwwrtFeD2RNBRkdKL\\\\\\\/1v\\\\\\\/PRZLMr2m+ChV+UxmD0cw0bTFrdH4KvSS63UPwApZVWHDj9RpTYezFUSXfwGkDUivHqBs7LDWuYuLJwAvX7bjm6fp\\\\\\\/k3ybCh4XwCh420YtykfuyKMhwwdzDkyrmLIc4iu5c8gX5SiYbxl1qB9Riu8biEHTHAPT\\\\\\\/Dgnv2Oo+hqEiWQGDIa7wZyfqOG0FvVtQgmuijJdlEEge3h\\\\\\\/4Pje1CU5ss8XsXlFEwVVA4L4oyagBP523ZG8GQKDqbH9nLtpnwWG91ShQoTxWiDGvkjbxlwNZCmrwEIWwD8p7fvMY2nprdkuiP4yCsSLT9am3AfFUxE9GSZPOHAT9TAqSY4X+RpY0qNqKtXC\\\\\\\/LWe3JbbbLMgx1RWlXAS4SfT96FY\\\\\\\/d5hAfnwGG9hyaT6ru94K8Q+4Aa94Jr4BI58hy0tO\\\\\\\/\\\\\\\/uqYIlMs3We\\\\\\\/jcj0X3vYImegFW9N1smzzSvyFIAKGrAxRgAeVizsCsJwzyZ8clBkJ19h4kCx176uktmZ0vbidj\\\\\\\/HuSEicdnyTMU80egeqpL4qD0amgUIzgIFtH3ZTnR8LZaRrvMloct9yPd4Ujt5GwKmF9JcckB6tpg9HYZFH2qgnCrpbQ0ZGFtllPs8PFPUQLHNd8T2iK2P1wERY22KysGWddI2ZmWQ9qjjHecM5jBAlZaPTB8NGUYIea7Ha84LoGsXA9vW+OFhmngnvZfAmFfGzv\\\\\\\/OT7MIUPXV+MVDDpkaAeIdV5Lr3jeHX3TGF0Ges9vnLwa7YtOUDHzzHfn38Y05KbbRvruNcKSWIfKFUqx4CXlXrDOyBxElN1EcrltE7naSvI41lgOROkkLD9m5hEdqE0V4D+0kmDq9EVxSP4MC\\\\\\\/CIVh5cv+VHplHODC4Le77Ft6OwJYNRYrczh754LJ7\\\\\\\/CjG0p9fyJ5adyIVnbmiXhNetYetQsklfxf\\\\\\\/dPc5+ueEkU6jYWon0MdxXPf4NjWgcurueKW6Cot3eo88lGw7EOP\\\\\\\/BF+jUoxww8bMBmSm41KliFKOk153KapKZHTuvOX4sOvThHOiim6oAGlwWPfxg0C9YsxpimfB9vdcjfF\\\\\\\/7YKktZr04M0Ce7XkNTnLBzx7DSV0WFQLwjuNoPlnoZ3UwxjHrZV37qeJvpv+AN+8ewKh88LxcBdaoXs9kmJFXpX6pnF3o39djW3sNEa4I05RUhw2aLMATtvzhwajnZxq+ZVHY+siC18g2Y6fd3YiguXBw+yxhMSBNZX6WpYeO\\\\\\\/\\\\\\\/FpbA0DnSg\\\\\\\/7hALEuE2yk74y7utVUegGFwyKqY7vETOiYsxurJd8qOwgAdL4uAIiV7+qxasQxDxWhTtnQVULxFKt3MJiTMSPqqXoTav0NisLjouh76JIMyaWecp0juwR38B0+FuQlYDZVVvLIDghHt68+In+DXTzZKPevBorFa1MR78qGXdRb0BuaR4Mo65Xj6PC7foUoWVTS7NQ560OrodhXszST1V1xRT5ih2josBS7qOx6HE5CJUijX0eKXoRFlQ1yR8A8fdMEsRUprVr\\\\\\\/5882OYXmQ8vipK5Ap\\\\\\\/l\\\\\\\/lmcr78izA40417OUAhG33TRkHEuC85ZDT+6lF1jV8W2q8XENsBSpJX7j\\\\\\\/5MuMlP75noW6uVUSQ97\\\\\\\/JNpJt7DJyxX5XgWOYbdP+\\\\\\\/UU2gOH9oLzmcDuCUcAvO3jOzTuYrtikhftFehjV4mVBBWHNSgMaaZXsDMkbWTRUsLZyCtPlMYiOkn3uAaGnOHhIm4w\\\\\\\/2CA8fryXoSWfZREJ4Wq4pbZ3lDi30t9091U4Sb9sFRkV3KV2uWwr1UH0tse+Sgd1RlYisBPb0kRXltFcn0k5168goU1cv2KXFZqfN8PNILMANVs\\\\\\\/bG2BJWNpW8Sw5+zd+0L1PtRZBqZc53oTJIeOojumgQeAL2XZOW+lrT\\\\\\\/qS1KMl07DqF4iWhTwDlSyb7+EEIbFsS6HcBlIV0tWmrcHbVJ\\\\\\\/atMPPtdD+zp2W\\\\\\\/54heOFRceeTblKfAzLudKFP4BcsLKoHfilIskdzV6q0MtcRNq3G6Smo\\\\\\\/n4zvOIzG3sPpwXi5Z4d8r9T2ldCoVuEUJdD862TSnaFQRMtYrU\\\\\\\/yDOW9BXjmVqOGDTcUHKAosFUMl0aWdJ2gMlVx7XufJ8SXb5aYPw0UGAlpMMEHtkoxrG2\\\\\\\/PcTo2uQOaLYcDB0KRzzmCY0kpIMOEGUy4Gwqv0a1NMyfpqQc5umoBiEij953cbVOZUhVyfjEgUiaFIDMZqaqIAmG9FsyknTOK204Fw+TE8Lyo3rz\\\\\\\/d6sdiisKdbR1TtCMJpYyjO5iayjg9cJnkyfPmyn0imZTaPDvIgSEL8hU5REidJvCCwygaxPvwTalWCx1HgYz7Ly4v4uO9c5eOGAYQf2Yv6SyEs12wcrIXW+ddMXGkXxGPLHpIOWIN0DLjSBbASxj7ikLRl1z6rCM18s8jnxD2cLFv5hpdG5NgdIMhieM7+qygZErJhxC13\\\\\\\/g7MJo7NeDQxCb5WNZjwS+tc3SdHaY72xOct7YunD\\\\\\\/LpT3vbHamMtBPxtHU0hp2nBt7YqO6OJgDXqIAYIYOxUB2TxkIn1jHGOBaXH1RkjKpYwYT97WdDBIXjAFpBih+BQg9es57ZKjY3cSvz\\\\\\\/C13cVo9FfXPT8c9fuTT1JFREugjNpduEjzumPjqKafFsCeL4eGPK2ZbiRVjJD6GPo5+KnZUdCdwhKZOBGfOU+IxoEFArHaQDf7429Nx8jb3pgtXxp5T5krfDrqArgBYOwKr2wuWjSaARffOhr6+CKqqDucWTT8IFVaxrdl4bGnel64eLGIRCaoI45wV9oZsKYonBuijaJ\\\\\\\/pWNVyNpW2ZBBsTDy8\\\\\\\/ytzUx8wG0OgXN9OW\\\\\\\/d\\\\\\\/lOlDPnwG3nODiUiTMbPQeEpIaD3\\\\\\\/tGKmjn0YMMvZMDa27pXUOPUt5ijOlVB3yo0pMa2SrOC5ctDOECs4lDwlLfVFKNits0hh+0gNG\\\\\\\/uf0yEs0HoMvKIjVsTbI4KparMaAgm4wprFSWn6IZPirKH94atUmui\\\\\\\/\\\\\\\/+g+o1admwyQLAJiPkR3mvdN7FGQdkAjpaJnP7gXbMZ25RreKLWQ4AMa5g08LQaAbiNFifAs9DflbUis0iSWer7ZPX54hic8NfI5yOxjTb32HMJv2cbKjq4Y9Zik\\\\\\\/LjU8wuM\\\\\\\/AtJOIAKbV5qy1yIyXACTLX4JZwJQ3ephphuzN9cu8hmN2IfxeO7fYi7Z0N2aKiPNIigciIhvURIk6w2n06ZjlOfhYoDzrClxpJd08tMYDqVjKEE761293qdXVSL44ns+Wqcd6xSOHrSv4w0gESj87vzmbJ1EGQRU9HUrmHXsm7UvzH8HDax0BctIbu7Fe9ine52GPzSarh7w8c+YqM6TPj8k5b+JeaVVDvCwIRdWj12g1BB3MgyYjwPvQG7QzXXjmJ++fOYY+v6\\\\\\\/P6KT601sQlXBIPEM0lVPlvJvfLXOgqm18l2Em62JwSzivOJcVJyCMKaJEP3UQsCExdMBaTJcO9FcoNQA2N6FTljiT0Kdwmskb9PTo0w+57saGdfL+fBH\\\\\\\/RGHK+YDPwINtZgtka7C6PanaoAvOtSmAP8VFSJVVA\\\\\\\/whYpMrg\\\\\\\/5fLv691WaGAUlDFzdj1z0w9hKtgL55c2nQLQtv8TXL6CJq5CCiCteARYBWiS\\\\\\\/G3FIV1HsUSM54JQdNbeUjRCV11uQo3\\\\\\\/H6XDjV3nrR55ir+hLLW3VeDjoSGVTqcBi3Yb0vulFTWQa+ktkbIOpiDz5upceO\\\\\\\/D329SKtng0yN7xzIJZYfvynraRVpGuVTz8jPlsPrHUUXWVKOqITA064GEIMb\\\\\\\/02dB+rpkmovKMGUTJL8dJYL8ctKxHBEFWdrHd873IqXBtBISpYMREcqmCFYowGwZ5a0wwQTqsZcVYG5YBqTSNxhmX\\\\\\\/6i3sKVujQKTzDsHj9zjusLapGj9+vJmRkZKF4HeI5EwpGcuDdkgBVagDRth9+yLDQTBkw6IprFfA99MOKE+l\\\\\\\/mkhQDrTWsVUvsCXAWsoTyE1pSvArZw37OuTk5d8l384FpS\\\\\\\/WphtWP2bz\\\\\\\/MGhr56\\\\\\\/jkXAGPhTpJHiTuFZcmG2fqYgNhgUJO0ZdQs6OoUK\\\\\\\/7JcPazFUR2CtonSHgyjCWl\\\\\\\/Te+1kduuaPAho5wNJotIs\\\\\\\/7L9yn0umUgZ8IMsPwFpStkRw3uEdSjgZrtyvKHP\\\\\\\/K0wMbjOb2\\\\\\\/scuhtu8+QM60TwRMYnx0n0yAHqUjqrH9XClpx6mSipbJKLW5Yuir13xvHHJgwrOGv5\\\\\\\/NsJCoXhcVQatrSZ6xD1URj5rocEReB5cJKLSfiUm8i1yOZ\\\\\\\/q1rTo\\\\\\\/S8q4KynKT+3CndfmjNQ5BMWEZe3tpYO0QYAO3fAdWy4ImI181fcIHGj0Gl+d6yvfYPFmIX8EE5GO9QxxOUQj36jng\\\\\\\/q72TQkW51VQdBXZiaTnU7khsHWx5h5JICKkFhADmDxV0EClvmkgwoPqZ4gi2IkcGSFFCydONH5EoJRz7Mq\\\\\\\/mMG+PjxOztKVgl0B8bEvW5BnoI0MHnKqvlsUp5RSodSTFtgpsiCWL7EunLoCbGoSCejnbpCcQyjK2mQWWO+gkMqVi9E35zVk6NdXaZ+v\\\\\\\/XsRdqqQvzD6Z0QMCMggj6rxW13YWlwPtaB+XisaeKIOANXq\\\\\\\/AJlzrjfhsvtX9159w1INTAs3VMuGb0SLRM8xNCXPnDzT0NOQxw2NswY3az2f1UeE5ntc5+i04YMXpHv4c7Sj9zV7CD+lcPNEWTE5+2ZdJO7rfmN4PUT9h\\\\\\\/2iVU\\\\\\\/d99U7\\\\\\\/O0C\\\\\\\/FswJhiY+2Ni2Zom342BvH1B1ofNjGb\\\\\\\/tO6DR8SIq2dvvdjRNapKne6cPG2V5jl3tVbR8lYDmbues7z2t4+\\\\\\\/EjbkmJdIqU\\\\\\\/wX02Ka72FD4H+L+LjTtLUGKYL\\\\\\\/qQEva7l3Fyz3sMHA31mwfJ2ZRbJCAEAxaar76UGY8QnynHqkfWuU8qDOwner8QMkNOR4w6+D2HW+\\\\\\\/d9D22PGWdl4VYCMDx8\\\\\\\/s0eLYrtI8YdcM3cHhi9vkHLwBnb+yQBfGzNwlaOTX8D\\\\\\\/HQWDmfBZmXfPU1NfrgoK93\\\\\\\/m2uTCRib7vzMzZrNRZqV7BDpZrJi0MeNJ\\\\\\\/n0L4dFX\\\\\\\/qQXvYFjZcvkCdHDzPOJf4n3PAIPOJm+a8R31h1lp9IQMenOI50uIEvzK6g+wKbZi6T8rEuius3HxlGXyUzadUI6pmBIVH9wO6SJ5xPEHw29CIz\\\\\\\/152cM+7PHhxAOPumUAo\\\\\\\/15o8MkSWy1k8suF10K2fJbpTWGolOPKBvFKb1nG2g7YvUtwGUAUFm4EqLfPlqbNrMdgZV1wD66X2OIQMxJRGOmzx0QgLTWUQcjqF7vI6b3VFL3LBtJUZrGnEV\\\\\\\/YiZkZNNdkFe09r0lDVdqqUP3hMi7cuzPqAOskNfdBFlxehy5ykAeas1V\\\\\\\/QqfMnZ51MLGReC5rzM33fxgGQc1QH5euPxSuLWKeLYzJ3\\\\\\\/RKbvq1QVGv1XHbgqDEbDBpPsBkEHSx0R0xhKo7jbiUPbvQvb\\\\\\\/v7BYrFWwOp8cjP9EFfPJdbQbEn7YTt5Q8WROnP9+ulRRjCO\\\\\\\/wFnkRzISIRNOfsLDAJS4N6jlHoAxDLtR6hBBuU6oc0mk1b41tLlWt3aXXDxB1p50MzGWtiJ13rAxU2Rt4CcYvGvzzjdqE\\\\\\\/\\\\\\\/ctMR1pwHH+c0xRjCJfE59wiNA1HQyOGtjURlIeNAC9dtPYytNKQQJtyQbCvQzA0OJblwEE6Ntgaup6hSjXvDBbRwLGkuHI6VF10fbBvjQ7rgC4kYHUdn2v6Xk49Gmmi6+BGLtwWGM2VZ1gsruFl71s7sO657aC\\\\\\\/poPgxHniBaSxyDP2l7PXJyRB\\\\\\\/KMiY8MBjE5CQfPi3YJLFsFPElSLRjKq6PTnw8E0p3\\\\\\\/090uu7dpqlPUaAIWCKINcZIGqGBRoHb3szYoAYp9k3Zk0FQ\\\\\\\/zZ+4CD\\\\\\\/Cj1uAuWlN0uhLQExmWEBI4g09kNXa4Tyu1j72zOE4cGhzjwK6peX\\\\\\\/k\\\\\\\/XIbW7Fl5ad3\\\\\\\/iMDmC9quNU3YSRL85wP\\\\\\\/6W0levuUEY5Z\\\\\\\/x9SNAxFJy8tas\\\\\\\/7Aawtptgw1RgVMJV4n8JimX5TgHPc7P\\\\\\\/nNRL57v5AalrMA0qXAGs\\\\\\\/EKAI0W4P6rLWLLaygWrQ07ekpWYrPJdGfUh4oC7qzdjUK+MDV0lnypMTErXnLiBq0iTe8XHjFjnpCMNwxE+BecX0CbPuJAmS0BNXCO6STQiMhMClL22V7vaCqfTneJhM6S3MbOh4jbFn0B7p4EqapkKUfV\\\\\\\/HLeWARiZHUGuD6sweddXBaTT8UpO6ZIbG7PlbLmZbgi0zSw+QX2NYcPkluLi8ug+wUxKa\\\\\\\/l7085+eiPbSAFWAv1Qr1pJ5r3S3jFvwsoLKQUOH0S+cjE6ID7Qeaby4gzHwzjVbetjb2b1e68AJDYj40sKW1WE+Z13ddk+C54pKT72jhvfx1ye3fg5Omuroi6FwojrTkOh48Tm8Jq6fthcjCgjwfM0AEGZ9sMrVczbL6zS\\\\\\\/m+WpCnE7CBVGWSEPEMwYvombiJkc8B9jgUXCxOc\\\\\\\/uyQNl5CdlJ4eeILKSlt3AG34ZpLOxrZxi6X\\\\\\\/96HE4nVX9z+L141dHz3O\\\\\\\/KQ7DjwpUiE+IsL92ttj0Vh11hkTxtVF3cD79A935C0YhlZsGGWx\\\\\\\/qoYMArLMARALsyfgzdc+90HMFPq2Motc0oNQOC54bb\\\\\\\/gTGLLJjrTsRHnYTsp5h2OYqLaTQMdGdcAMssbw+HPq8YKRxkU8VDasLuuZKFDuNbw6GlJlB3IdG5HqNv84uCncHHDEBrh4yVTII6qCNpiqX0oAO\\\\\\\/bBHO4q63rSVVNMi+XkR7F6eNeFwCiEH\\\\\\\/KOFttJQnbZmWpc9uz7yurq\\\\\\\/hU\\\\\\\/aCgTKEFfK3eBsHAUJqCOXJS6wq30A8WPtagL8L\\\\\\\/4JUr8\\\\\\\/i18oEKlFtCXrHKDCmnLrBQbhk7HxOA3vKnu8tHvF9vOsnivTYHiLUXlyicYxZhJHXW+4runCJyXJqcSlajK6ciLQfXrUznIhqVA+EBXI7xhCpMECPYEdy73zQj3GzpWvQxj8k07sRRwusfW4sIWSbIuQTdytbbKgRix3O5lhR0Wntb\\\\\\\/vw1uZy8LeWrN5Vnkjv\\\\\\\/KEl6h8U4ywQB+EIuK4tyJB\\\\\\\/\\\\\\\/I+yphrGf\\\\\\\/lOhnrCYOR+9EYRvMkf+U6xh8Y54MRvfYxiSEvVmkwzplKnTmXzn7nmDPiEZbZib14Gjs0TwMibCaYwA7FW5+NdIu7SCP65Xcp2akpWiAQCVAuABJw3\\\\\\\/slW4wCCuGnXpTZ+KLEiWYmsr4k7UrkXZnpNZjWtfgBA6Kr5I95WvI6n02FKeX3zsS2GxZbIOJcnvETlScLJDVWyh6LHVemXz05vymqy0xo2hSXBHa9FEYb3yZI3XkAv8UJeAxH137MII5w99vJGrgbp1ntacSYwsZDiDbb8ujELfWArdfcRp4Meq60HUJH+dGAEtNrcvv0obftSG6FvzcqbHYw8Ug\\\\\\\/7AsvZlSTpSFW5ryz772SQ9\\\\\\\/KjNtDaN63ut9Lbja0gnn1OL7Q+TsMpU6T8mWQbEuq1VHR71g5w4oBDR8hwrhA3WvL0NnjGRVIxuI1Uxeyr1fTtEfDKpZTO5IBvk\\\\\\\/qGFsZJwVBhT6DKEEi8ldohm0y6kJm51xw1MCWdUpXhL3fxWfOntffrmmciccSdN5OEJDS3YwXqlaPGZnJs6kvalpu8W5VCYa8TWl4EQV5z\\\\\\\/HOSD9SVdl90aGpJP\\\\\\\/wHD2Yb1S5WKeSD4Bm4\\\\\\\/Wz2whwrr13YpJKW3fLzXzgJy5JOQ2CTJmZL0Xuv2xIhor3hPvsPZoqrn1fPOy2Yqwh+DqVla5bJelMBT7nzbZsLTODI8wH2YHrF7p9aKjPAZs29oAxgjvR8vBH2YnfaWF1Vw1gvrUf8VDFg+8zqtJEmL65258ILxqRuZXEPMQUXSj7EiGUs48r++7r8MsWPbXP2S7wHWSa2UrY5MvX24ucmd1qLoGnNeO0gtsIbkHbm1Vrc19zluEVrB6Pxyz3D2vIQKy3PIK96tPumd1bi6DT1seQYi58zMsyIAfLECkMWRxt\\\\\\\/RE\\\\\\\/83hkavZIgiBf1PjXpck1954S7jyBaDwKG582CDi4BG3J5mWj0QTioeGfz4oTKoM+pTI1uqIU\\\\\\\/OP7yQsSkk1PLnrZwCn73ZFz3V7dRDszCMEfEtxTHgft0PQ6ImyuEsQLQkUy1w+NjoIOCL3np+xnuWP0mCdMoyqH5egqKYUUeizCVIfMXWYRUIt5HaHQU4VObWdod5uH\\\\\\\/k1FdzVikFFhffg8gWqjgt+npTiS2C10QNn7dxVJrNgY0r\\\\\\\/Vzj8MyvLPbgS4wzcVCCyGGPz2aYZBQ540yP7QjmfamgSLpGezVpnsbeWxQeTbPyjBX+U1tES\\\\\\\/rl587m0nzlARJAQQbprslEG5l+XzSacuxslAEu9W9m5mUkhsJx17jcNtfu8\\\\\\\/y9g0lHYceI3tkcTEL6lHG5yBqT19kELx61AY+YP6kfVD4VrRyuZEFL2X41R\\\\\\\/Ql0gNXMDLlIOOdVAf1EeqWIaqpp9gpBI1yN68BA1YeXOAiy3a4f1l+k1wtoHDsH96wR6Oz9iSixSHNfji5j4qDkZ7g5Eqg3HO1FoX3fVh2kZLT5bNZ9F\\\\\\\/aD2cR7RkjezzMBvp6JzR6nfrw1FZRf0wxXYD6HpJVfmeskEMsk8n3hsi65lfQj8LLYE1GgjSn1dBzcnXgL1mwZj+ZvPpf4H4hzKGDbNlfGHpUR+BEkeZimCTMmXXu91RMUvIOVH833Xy5Dg5sEmpuEzH3c2dKdXTrUqc5D3peXB+pb2glItycV23Qp+4HCgjFTUMx4ybi7NJy5GVebBOFLdwctN9eiroLA2zGfvtCBROdxeII8FnKcbsXaR8QGGurHMLwybF5FRi0e2MKNR1rFIVO242JG91ytqrdgFMkiIhVsxi\\\\\\\/WTbpdcqcwd8n4bXkGD8tWh\\\\\\\/CCe8drhCml\\\\\\\/JUsIRKJKh\\\\\\\/IZKl63Ubflu8ySalEqM6DReQBdReKToASGQNXesAXX2+MrgOeMGGbR4Wu0mOTw\\\\\\\/PhCke1xBzYjq8rrbSHhg+vzXpLSKI+LneE5094zVLJjH6SPgM9thq5YJVy7fptroPw64fyhwKTVtuTqNGWYnkw2w+0kMME+XfjFX88Dlt8WqLnjNZ7C2lIh06fSQGQd6MdMLEVxClLM8ehY3MilOry3\\\\\\\/Dnjt73flJU+gviyA3kYpUT+SL42mSylSx2di6nMxQaDNBM8DJ40Je6lPi7Lh6ZtMJ3MDPjSXKfvjt+gzwzwKrNUblgHx75jld\\\\\\\/xr0lXdBh3W6wt9UxoQcBJX0xzcyYZwqnR4P0Yc6dQdsuqMF2uBJ9YgkjHyq8qaJeYdPZgy34CNkR7avPNlCMxj1xPWmNnil1fHlFx649Ys4uFWjrtB2P0YAF065JI8BmvXonvjOsmIkadmzCdU2l2yKc8vgsswcaJubXNH96ls8FyPWglMa8V2wPEW7xx3GfQPYPuX+5EeIsXkIB3iYlaMyvATHfolMs5ce0gWgAYEE4lQyxA+5Vba63vWIyGJx31iqFsWOpCgvUqkYOP+OrgMbeLvfKLxQP9md\\\\\\\/UIeF+rgMurv2Mnqu6i+42cyAaIDeTxdzdTYLND4Oen04CUC+WehRgWXxDxUDCSA0Z0AmhsShxAkEPYMvryiNqytst6LvSJNZdhjbTto2KZDMzp0J6v77riLOHIlDAQ4GIq05eHVw4NlK9yV\\\\\\\/JJ4ApIx9QzfLFEGaZPQySueXRkvND1gT2UOc6ZTc3PcXYMFdNDuyZtBQ2NtWDsNxpPBUMlr2f4QPb9\\\\\\\/vmAxpdGq32G6KnOdS+qUokK0lLBXVYzNcP2kYgP8Y5yCglrZ7ffdOOmmWOnJq\\\\\\\/sBQn7tYKxBFBuH4978x8qtTiSCuD3NGEYIcYS1x7m+I09Vk1jm4fFujfORvRf1C1Bbthz9Wr+Y9nbcUjmvUswrAVUrA82ay1E5L+Xsd2pIV4+z7wNhASbTvjmQPQs+ePXttUBztDhpcy8\\\\\\\/FVkXDN2BdhntiYTEVCahO1jvzqoiIhVGxHhGpBbXJwENSn0tAXIiSXMvrkAQdpP1XRCm+KhRXOR++oUWk23k665YzTK8CN3XJXJb4MFBH0Wic3Hsscph5c15AwA3\\\\\\\/OdihIEJVL7Uh8EMjsMize8tTEv0rfLZkvPBq2kK391kbgr3SzXXEde2nbEK4IR3oz6o7l4yjG460\\\\\\\/YCNKH94UMtvDfskdjSlRJ+j13aQJauDCjJlEAuP9ZbpbuuOJ3yA8VX7msl1sDZAbMqkB61mecgM\\\\\\\/xKnqJQ3aT57UJDxZqik44RpEM7\\\\\\\/duJE5fSvngMQ+YT9XAmkdKc1qqZKN9pTlqrnVNIl9MTTSWrWCGjAE\\\\\\\/xlR8wOl3hFkZyCEEYeHnKCtsZpeXTH7HC3fwqPbQUGzkRNr0\\\\\\\/VjCft6530MnN3KHJl4epwVg2COd8i19RzT5k4GbkzgxK0faQFVruWWUAcWxmmd5tWWzGEDEeEoXUmolh9v3Z0wVJwXb2cRC9w2rVddo6kPUWUfLslBhtWGvgrV\\\\\\\/sB6c0U2PmY\\\\\\\/ufA0hR8OE1yJsFr\\\\\\\/hXgn7qXa5gxfrOCEM1APIoHi4NFfnLZMIwJg\\\\\\\/tNpgF9dJkyfSx4LllmvdWdNOpABFQhxMTOGgs3GB1MZAnu7LO9ktQYZ\\\\\\\/kkUKW9V5tAxkvFPUaiiz6EUY9Ln+bLC+IFrQZ5srfftv6DbeZAsNFZJfgfcRzUC\\\\\\\/wbgtF74y0y3MOX43rFtuoBaSew4IgkCtR12VQqbTLvZ3iMenOa5yMlWvo44buJqkw++m38Mdoeh66jSGC7cZ2qcnvF4942M7CiRREOiOnUEoysgIWItyqh9B6mHXjlh+I6vyQsC90BZ6DmD\\\\\\\/9\\\\\\\/V1Ujna9IoIBT1a2EeE4RJBpmJKZV5Ykn\\\\\\\/TCuUAPFnZsNLUa\\\\\\\/bIPsgnPqz25u5rsGbEvNuA58MPhjpkCjm1QkCPiIgECnYStpbN230tGMwBG7lBQWKBB7YSfD65AeDNp66e3t0xvFxA09f+oNKulx6ldWbnQQfih3EDrN2+l2YrjjFITBgpKJchhDrdbGak9S0myz0vlnRyR9sAqiXC1WmbxR+AhnmHRaX5SOl\\\\\\\/lt6rSHtquT2vNBuzze3pcVZZhAs4+Fl8NDLu257wRozXnhull4Z42jeGS8kWi4\\\\\\\/diwIJDGM4VRCfZpj5KGJa15AC5csYKPp\\\\\\\/LOGoZlqTnYzyEOYhdboDf9LpJtckZtoIRbEm6zyfeTexy5molj3N8I4fK2ixCUdMt6ddhn3bRpalyEdVy0biKUp7EF1hDI6oOxxhwY9vRUq7sXEl7yQwK+muGYiuMTnhaHK7Q\\\\\\\/8K2geI6amCEciC7Kk+0MzioVXGxEaVISAhiNLrrcX5RoZIAkR3VzqduWiG1zDTSVyxN3rcBFt1Xvpz3BLxSSX0KuXjGWrcnAH+cOl6JxqJ8KQ\\\\\\\/mgYGk2teMQ03UEyAe\\\\\\\/nVJ926W2byUpy9sdHpqWj8TX0sHrk3zFBrcawpROt5OsK28gAWAH\\\\\\\/lfBt6Ilv46VX2vcbIcELJzugADm9+GVMki2mXPSEUVhWUInTHRpXYOvS3b7pvvHWmmJE4Yme4PNLj\\\\\\\/hHA0BRK0ESX2rJsQYpqNn7yGG0uVu1s1mqxxtm5uNXzR75r3l44cYHJOwXLIncDIV3wLdcjAb8KoNWvv4ZArEUejznMwJxFk6Djp6YiILQHIdl3UPitV2F8QU1h4fYiYOwlH2txDFV3Tz9MVnYaesXukKfMbyqtYqtgSndpZCnHHJt2uYQhP3VkLF44sFzJ\\\\\\\/pTXl90S8TRMQossvHxTXRFvW+YmoHCLLZgZEl6MaNSjGZsimsi4NOf3YJwzp4AicbFfIwUbzV2m1T\\\\\\\/hDvbvurbjzRLGb0j\\\\\\\/AzseDzTNxft4adZC2xnUOAXvOG6uR3hwEwo9cKm+Owkda6ffN9Oyf8qEbM5YcS2KY2aEZB3ozLlJqm9ghV9x2JAI4cqcYp\\\\\\\/mQV9gOKKxV\\\\\\\/eu7XerSvykLwiHOiaI+Z5KyL9v6L5mA+2WS8osBNxJplzU0WrI6fu17ZswMaWvNW\\\\\\\/QSFetUQCpsQo3\\\\\\\/fMzdWhuzMAWL\\\\\\\/xukg+NE2+ktayCNnlYpISQ\\\\\\\/D5quLaOUan6C0KIKuUwZ0PKJ8pQy5mGfxblFioNOzMOXikl1bjhUQF7nbhQsAUjW0dkrS8oa6U1JZLPe5EuAkLaMEmxDJWenP6J3boCpwQux7fZE8MNEzHgwIypWSPJAwEEj6N4H2faAuxhduY+0BeW0aOU0kdWKebADJMbDFFRwnSGEXnhEhdrbZnVY4rtXTSCflYu4a5dmoGLu30cmb8LlSV0nqjnMnqBHUR92pCn\\\\\\\/7Cf3tLQovF0aLupcSEIISZD2A6ehAI01yVJ17y6CWn6dYw3eg5ns6oIOHRAC9+ZAGe9BgOrlY5fgJwU4MWqToQqTMa6jZwDw1cjq9kkylXHdqoRk9NGLJvwBQm9OxNPglT5qyqGhvNidGQigSxaBZKxvenx2EEMuPsppQQFFNPtwaLCthVVexuVmBm2ZvCOah+dRYWzVHlbQAUh0rm4sHE+MQ5VxAhGjRzHhRBpkITt\\\\\\\/sphhPhAI4xXest5gjbzE\\\\\\\/spF1VQRfK5c+b51hTzzeDtaxNLhUPcqxqc1WEMuLbg9dt9fWMkd85Yjm5nuI5wvvWfaHImc6dK0c9u0yRFZAZ\\\\\\\/4KCPfqEMZefQ9YXVJcgEnRNAN5dKkrwCF\\\\\\\/dTmA1tGd1LqXUwTrDStZJo4zBHBEZQGW8GfH7zWAYMTmxVw8bRFMhtxeCN3rsLWpUAzOh5NI6j7NQ5v8r2QUAG8U1yRrh2JlCmtVYVvIpBxL5cvMIdFjEn1CNZZu2gwRf2RUFYYndxpGg+cKEZ4VQizhs8t6S039wiIRX7VGuUkTWjTKO\\\\\\\/UwkuSKf4pEbYAdG6qORoKYKciJcDetBYL+X09MKXs60LnatHv8xGJTQ1vIZdREP7gB\\\\\\\/H173Z0hlP1sPkZcsryG8SMiFSos\\\\\\\/WVV+zb1bN10D7kSgqF5aMH4+z5XwdsBvH8LUz4F+bVs4gah3Bbzmnx3\\\\\\\/g+2h2Sdg1+vYAI\\\\\\\/T9u32ijiej9FC5lA2kEe7g54bUe9C2zAdbfOMuFswVurQzdKeC2Ffw0mqB1qaPmKQqjhl9ype4sbAYeYZO1vQkYIBf6ugl1IVprzDG3X+c1yExULd+0VkgQ\\\\\\\/v6ZsGyFAM793WYmcI+fPremke3GYDEdIk9pYszd54+xjrAsYC3YgV9y\\\\\\\/aaDwGDMOEKo6kQYcQQzqOl6z\\\\\\\/VRjFIJa4p0nkpfwDtqSeBxJyhcBR0LidXaRAmEdkZGqsgfcDnVPzIrwwdIS2FAozp8V0cf05+FVdvZcjxNyocVkktCmU0tO34ixTqO3+Vep2syw6R46lI2s6qMoibPC+l5wsF3L+V7w1e+v+jvylBmdK73QBR+PQ7vA5M3yJWMsS8nZJMqeAmtfoTnTZ303nyxBpTl\\\\\\\/qmtb3i0ZZmg22eBoAtIz3FUenZ3zPvvBtrCwhfFZ8hHpl2zE0D43NF7xACs44bmBSI4uZY2vlL+wSs0Ev7qxgioWUuRw+WO75FD896nWqobVUNznBBk1+HBZNA2TpwT\\\\\\\/TP6yiItPI0frVO9fwkAsPgrVgm0V\\\\\\\/g6tUbd+5ionV9mNAD6HQBhutSacgeSaDzhqsBHpXytklk5ygpuklumuo0JRKl8+QI34V8wYOZGewEXfQZnPxMIif+6xKsU6yTXFvkD\\\\\\\/2zKQ+iCQbNvoAk\\\\\\\/N0LG6Vs7TAImhIwrFVK\\\\\\\/Kyfqb3F1b8foeWoPPCMu38Cl7RA2cLS0khmj9SXCeBpYtBuYzqdWwMZjNgrzUV4vmXlPFawC6qh93fz0\\\\\\\/jyrroESkeH75hLlQUApE6nsJ9zeU9GNEeve9U5ysh3cOQotIp8mV0L3ZD5HYp1z65DuXBKMrdNUv\\\\\\\/OR0nZXdYPReGAvz1p+WJ87ZtL3EQOrMrKmQLLQb4CBdvme\\\\\\\/VPYPrqQWdafixizGaXhW30aOz5um2zK2W5g9OoLW957KZ2tKeehNGbTpPLbb02ZiH+Iy37oMq3YFZXav8GwJMaRX4pyKvsSVNmrDvLZbWQ9TCrKkgXTNFvVzPsxd+FfFR+6xJaiDbUO\\\\\\\/YVHA\\\\\\\/t9Uy5VoqxqkKHSYq3EdUP2PWDieWZIaOYNlHA9dKT+WdPU3iX8qQjwOi3jLdB2HaDpnbwBVZNqkr+CY1YN7h6n9TPaAHH\\\\\\\/jkVS9ziSeCRbApERquRkvNoEJccVneQbP7zultNMTJeycF7\\\\\\\/cdbJ4okd3az0SWhL4v4KMpy0DxrQij1K+yu6evOmG8RYa9PCu5RG7QwDYv87vyy54\\\\\\\/avIStEnNpyFIi5z3dhOagOP8GsxYB2r8NRZ46vk4rX0o9xOrClYU0RdKGxkFuvjEy2oiDoh\\\\\\\/iakZSwgCulCscm8xhgY8rXxZz1aVr5ZDB9K1CFzTNbHbP8i0dB7orMA7lhv866REsMEft03nQtMhLvUMtrbjlRXgLQMCOcxdM2hX2UBV04FAv9eresccfKySEkNFcxjK9b7LX4OegbPsyoT1zk9SC2VIjp5bVrrtyxfS6vNtS7d05un9Aq6By9kmnXDqyYDCCslIEkvRadj5rvww54lh2DOd2TM3aHyEl3xNkNx3rOw5cl6gsAhH7+4v2JFcSfx0pV4pvDLyTrkSXdBsJ+2v0BwcBBXiwwvMgRA09OX5ADzSDTplPJWKykVoOM4KmNJxmPtrgAXWa6VL2zFsUsPepULVazkfRozOFdEKsYwq9uNj1FuD2+lbocC4q6k4wHeuwE2fglvb0apjabdn5+cVEhpZ3bC7JL+msf6In1IKRZVsou1bjsATfwhLkuuIbdaOT+XibxJvxDLwTgXw+0QoCHcavh\\\\\\\/l9lNqhm+kCn9L4xeeoad51n7EURb5pqlf3pJ8J58e3b70GoetVu7Gc3A4WkTtWcdKZi\\\\\\\/ZoasumiGfbB6TgQc1poqQzFmRXa7GhfRCZ43gFI\\\\\\\/E8eE0ngWCaM4+xF9fT5QEqpWXhBbM\\\\\\\/BmTBp5Wc75am\\\\\\\/PqUtrHn8h4D3E\\\\\\\/kGo3+Dxz7\\\\\\\/peVQAytK3xTTzl66KNPljRFlzJjPtNfmVHcnM1HsBpJsKNhMecVBLhqszrm+OvDY1Ou0i\\\\\\\/lYWfk2VIc5Sin0kpJg9SibOlknRP8aMUK1vbR4AaW1VKV\\\\\\\/AjWfdXh4EqPs4IBlQPCRZn\\\\\\\/uCI3KD0Ezr5RUNwt5mA1X0ZJp1hL9ynpDbsNI+2mDcdprRhUFQipeCxyxZaVvkiIIkCjq1J11jVlOIz+a5+9hXVI6PmUEnjNLjWMj81dTVhnJTrwp3S45TxKQBax93lGqJDiOiRWixO99UKBl3nP5v44AHAki\\\\\\\/IXCcw1IpXom7sXQ+FPUcI+dOUOtMz3eZ68NwcpQpXv\\\\\\\/m6RMua5Z3tYO2Q1Clh+IWSb0Ueei39OZhouIKjsZYiNRu6\\\\\\\/GsNk761ItLrMYetYwYO57Phgy1NRBQ14F4ZFPTZZm3Fmr\\\\\\\/YL+qc5rXgIiSt9fI\\\\\\\/zKawH1m5R2N94oquuuw6jOIeFZHH5fIPQoktvXCX3qFmxowrZKhHvq0ub2PAq8NC0t\\\\\\\/ResMtuuHdWIjskxTXTS0OJr6GZTT5xgL+zet5zaXu0iuRltgpJJTekWu6Bd+ZL9D2oaFF3tHpuz6qcr00XJBtT0ZUbn0MsGf6eiVmyF6yMZwulPqxxo2Jojrd5So6pshuNs7Np8tqXFFjsybIOGK0J01ctReJxBawIc6KbjGUcLGY7oUFtsUsstGa+3IeGNH163r81\\\\\\\/ADlOldOSFFDXYxu37IfBhbZ4trAf0sDe5rPzuH\\\\\\\/cfelbO4w4sqYysTE2GGXKGwP3K1xJE54KsQvwaiXvPWMas0gYORX6lK1eJjOXk95HxDuFgGqW13YTKWXsz3GbQUQXqIBChYYCCNhIQFzv8IpivrvnGBRG1AlqikL6EPVlOr8CZMkboCxO29DjzpVqomu1TLprG2yTXkLkp+QFBXKvd0mWIHFvee9aF3DbCtkYBtptYhXH9vTubjinhBTN7ChFStta0DsbdspS1h+M1XSmcBRcharjyihbG06kpKrSBEDENfD56jkDrPHJ79CeZNlQzQRD2xsu8s2K8cOy3WZoNC1ca8pXCQQ2A24sdFkdnF1KMyo6FHJH\\\\\\\/LbinwKoRy7L0zlJ3GGT09y+McWg3ImOVG+GyLQTzPgjwGG\\\\\\\/GXjoNm4zpEhaiyP4H7xDlswHMAImSiZao\\\\\\\/c8VQrapawKsTaMrwvA7FLQWsolC4PT6Uk6nywP29zGu5N40lXf\\\\\\\/UUgz6IxBG7vBav2ut4kyhs+hrJ3Ph+Jh25cPUVFj6rMYtKv3g5USuisseQdls+ETRZkZX6KUAN4qAVCVmRhgpQsJXS9gh9FIE6+YiHCBEdlUSSXhh0RWdPBjW05p57zckCwQpaYjLAx+YNfgJh6Wj26btUpcXbY\\\\\\\/3Tz4dGWcFhiuHmakE9OPmemsmEuJdlsPFQWgZawwas86B+eiukAipKp\\\\\\\/1oxtd6LyA+KQCvF7jOdW8h7hCYGIz3LOpyae\\\\\\\/S\\\\\\\/538XYs1dyb1Roatqn4AnRiFVc\\\\\\\/QM2Pr8SdifU3xjYNPfkchxZnZEVCan529VB3U4u6Txmuq4VSdqIZIf1FYJbgPHV2z8p8HX8xYPBJdueqs2M1KXf9zfcEJy5PK4TuAESO+WW263GsOUmluWBEDpUNaWbZPfxLSEx8yP8k665vg71GTyu33e+3U\\\\\\\/7ydnYRtyJLPoNEdOk0FRIV8NC37BAsXOd6pYmmR2ag+oS\\\\\\\/zp321p1sl4tlkHbTXDjAwPPedWwyg7hUCajBkzLmnr2P\\\\\\\/WGaHmgOZUnjdYWabtRJJhtohVkZxAk3yLNvxYsnyMAT4\\\\\\\/zZl9O6adgqlrTbeztt9I7ClTEgaqbTOCDLFSTCPzNXHkPPAI7SmsIrzYW9yVMn2ChzI2fFg5b7jQ7CTfAL1xR2RiQ\\\\\\\/2r1z4\\\\\\\/epKU0ncHLTDoBNHVZfbPhQxheflM1HH1Vd0wANX8k62\\\\\\\/WtvQVAKly\\\\\\\/7g6lex3e9shJjfiX0uo0BZZpGTNE0X1KwSk5bB4LV7E+hV9eMCJNm1ageJrfg3Bn5v7kYxAY00DVehUWicuboM0Wxud0eBL3dpUh363xKqhPe8uzzmzgP4amD2WBGu7V\\\\\\\/L3vvqmJqQoJGH3dgZ7lFEK8MXn6PYAqhSJLAUMCHL5tbGT\\\\\\\/eTKK3zUaZdHARq\\\\\\\/RLe2bLEeIFlQk+a8VXF49ITTIeawqf7a122BqeRUi45LyFZEkg3l2A0p+goqdI8AWAkvYa0X+311gXvxP5yqxT+ZkKgF0TnYCAgX9GiKau0Uz3JXSVSBJF\\\\\\\/wbPF9EikytoFAvx8EumK6qmwtvtT8AYAxNhCEy8F++M\\\\\\\/1wjqb9mfUsf3inxYgLxhemMWeIFTNNYzKl+ds0upiZ8QMU1r1CG+nWRNXDN6YlpmOJy+gdz7Ii2M74amHsA3zYfFnEZAGi6Avwt0RCLFKEjKIs1GxL99PbuHYUJWBlZ2+lor5KSZxWHZbTQh1F6Ky6E3lryyuJF7OjpirQ\\\\\\\/RzSk9Ubrw7UugASJFeG+djJdlOU1bnoCG9oRAb0PtubrzCfvcdkEnMd957quxOuxevCDUTGYTVjRLpTOSYcpWk3iaOynLikOT26Xa\\\\\\\/XRc2ZWMeLstE5ZRPCSlxXA8vrmlRUhXaV7ksy\\\\\\\/aCUrLF3SSlX0dxDvSm\\\\\\\/ufwhJjfgv4EnN5AZAqwHgbZ5LUq9tJLMnsUxA4utDLIR4HYz6X4xv3ONTIucnUE+kTKmniKb3KfBnUaDh9xHwcjhltiMmBP56BwJLIazZRHQy1+SZm+H\\\\\\\/G9FqzN+foy\\\\\\\/5RRx7wv3OTEZvCrYh9ZsrZh8U7fByhnYeSxQ+nTMElWQkBE0BrG0dPvD2i9ugfnrXWnsbRfAvknYIIM0pz7otyGP8Pf6wB+p9bbHHgJh5kZJppmNiUIWuvIvj2ZVHAD9zLchNOYYK8gU+NVEujQJAT4mQ8MHFc+wPJPN\\\\\\\/4CckqlsVTmGuaIozGwDxgfP05Zp603PRZzw6PYd0AEbTmD8VZ0LGhZ7mNKDfRQ3mESYCQdFR3u3d8lzIfXvg6VvSicpFVTDgS5eIAp77HYNSJlQtBuxYfyxceB5FJcQodmuaFoIQ3\\\\\\\/0xR6rhzA+3s\\\\\\\/LFj9piYxdm0bc9L5+YVGm5FFehgnbi7pMH5v3iPK+t8hd14jWydcCzZ7AAzS3Uev6k1s2+67cxVuLuORLsUIf2vHS0scQmWs8fctg5\\\\\\\/eABeIXp21xJ0hJsI5suEeWEmARR8gAL1SdUwGSlZmmZ0hq7WLpmSKUgAQqUMhhpxYFKTZWICEO9L8dHBvPOlnhTUvy5\\\\\\\/SvhfPnUulIQnQC53w2JjpfjMlxnBw7QwxHtOeS4rDnH9vkxVDyqy0YFDmGokH6Am0GpSOqnre68kFNWUEOenKUpGs9s9nag9e44OHOyrHSMQ7KTlARAZ0m6hyURTpnv1bTYOpuyh4YeH03YqQPZsgcLfduq5AcZx9DupdPLCfKYaUPUQJIfEDMHGiev85gz1vC8mJW0k1EbYW\\\\\\\/ESruN8ezwR4EhrVEloh08h5L3ARiNbsbwuUnkhApn2oS9+4WS8tSIYSKgEY2vua59OsuB9\\\\\\\/0xlzNzyL5iY4MnVrf8p6gAw\\\\\\\/F8+OJjZyjT9XjBUUKrPx5arG0eupZSoAtnqnXRhxpkG86PkHrSlfCMmL9ZI2ytwbDaXZNTMGb3GnHjavaY15IxUrX1daXfiRARBX0L8oP4iWomQan8jN2EZIdcZuvrimGAN8LE\\\\\\\/JyD3z89gbK15iQU7spmPnpamSLqX4+42Qv71g5K2Hrmp6aBwT\\\\\\\/yH637\\\\\\\/18XF9cTL6LBLI4Qy0DsLlB7pKeqpH8PpUmLP\\\\\\\/rais6QMpXysiX\\\\\\\/\\\\\\\/y0BSKFqNKM\\\\\\\/6o295En77oEycUAGwAHc\\\\\\\/XmCRBmLXB\\\\\\\/U627DLu2wWibolyMzb1oClairUfghNKXec25dF3lbIhuPKKw\\\\\\\/BXG1rhGJvOcDM\\\\\\\/UiTG5a0NCjZkp\\\\\\\/RvY4bO5V\\\\\\\/b6GEluBIOwg+bxygXR3gQecdKFKOX27Ee08OzxD7fPR40xadKp+yAEgRqyu92y4hIpVwyA+BnWRQ7biBHae12Su\\\\\\\/d3iOOU75tLhiYhppeGcZ0zIlOUt3gDZivz9e5n3MuF4kbmR5pYlbNOH\\\\\\\/52uKLWosBFzaCDA+gLiduNaqoefwFA43xb5sjuNydfHAYrgmCzbkd+B9eqtbnwG99OOaolTeOzNpFoade2TWpYDL0dYEqr4nQsNbubjOs0HtOJwL9lFHlsPAa4TcoxzUKasGbm9a2+zdjLJkiuMExXHKLCrePs\\\\\\\/c9BpT0h0y4DjzBEoMqwefYuYx+cZTAIj1HMENAen4un5sMfgLH1HTY0aS\\\\\\\/TSdpUgXicPymTqqBh3cT\\\\\\\/f7AMg4hz93jHuWy9NcqWHIEPXOiB29PYH2eeZ6rmepZuygUM7s1H97zV09IPaKl7N\\\\\\\/pmJgN1CY0ifK9LcmvIhtliKbDKPe91FlsP+F1i\\\\\\\/D9QYlpY8xu2V77H3Jk3whGIuWWiwC3a\\\\\\\/ymiwSlj3GyD9F3nNUBo53LL8RoY3Nmhi6WDODANiyXI9GfNLAVftUZSRHUSHiJoJyGSgnbtc+Aon6qa\\\\\\\/wOiSJij4TjX6BWIL6\\\\\\\/UBcfxTpMWVG80\\\\\\\/\\\\\\\/MKZFmp1GSGHU+iZbWBMynW4XDIEwvjCFqoy9PgtIaW6qgWbliPTrRVcmyo6HD\\\\\\\/guozAXHJegJn\\\\\\\/gjCHamV4jXqSHWrqS66xfPuMJGgqrH3uFoGKTxIkf1Dl3LnwhAUu\\\\\\\/sdgYp\\\\\\\/9rlwEbB53ixHjJCJZy2WQqaRGPgj1hA17q69\\\\\\\/e+qrJiuMu6zjqdm0NTlmDEkUkGYLDtxVbl+\\\\\\\/4Q5U1FvA1hpjRQFz08qj1EiHFR9RpBLe4ITtzManrdTKQZ\\\\\\\/mJkN+Lh3ORX+LpoeXyGIU3S1+aCTEJy4zaMKA2tKbx3nZDp0GunG\\\\\\\/Ure8qYWrLf0Rc6rqY+yh2QQh1ykXp9DOAehKnye\\\\\\\/MgIsNFeuQU1CJAB5CsKjMqR4IRL\\\\\\\/ZYgX9TRqyQsfdqhBIwGWM8YfUOfr\\\\\\\/BgjqpgaXJYQTAjvGf\\\\\\\/T0QbvgWEhfSFDr\\\\\\\/Go1SarPF44vVRb\\\\\\\/MKfpX40p\\\\\\\/2YwWKA5xGNXgPDC1qip5GvEDAPP1OUDPmJV6CfJXTWAoX83oD8z7\\\\\\\/Ue9PRvx6LocU9neMtTuXTaJoD0Ehqvg1LdCFnYsPUlWYZ7liJuMOtnWKHa4BdEM1KbzQ95h0a8od4+hHbdaNhk0VbAa1U3ujb8l3YDPyLiUNOrPR9zJtOkCqiVbP8YHAwk6VCE+rUJNzHos+1mH29hhePUyy8ANkgTccruN3Qtwof4b+JkW6F7mqKgtfks28nMMrUYXZ4G3t6MfIjAvS\\\\\\\/U8h3vzAMnhMZpbBKPAITV2D0VPdzpa1p9SBtOUEfeN5E4u+nMprNllqJ140tQUv+TSzlBxQgIaddxWgw8+w9iSQNsEqnKMOZMCrL5yAzzPWMAIjtMVR\\\\\\\/GTlfZoUUxiDrmq0yP5Pe4BC65EuJiR8\\\\\\\/mUDqMda4QXt2MGnOiysR3xGSc0yYibHB0j9kY3of4zn2BBZKZdACPOYodjDsJugnybluSGvUhZ9FUToX6xhB2Vg\\\\\\\/vJBkeCTxEeFt2vtYYaye0+KpJFP3yaPfkAB5OJnAc5SZmP6OZ96ob2\\\\\\\/i+txphLSwxvqFL61gFL3xfY1e9EoDsJ45VR2S4c9jkKElHu6DR5sdPPcpY7jtvCJFgQp5KsAoghLfY7uzeYoQiw\\\\\\\/Zd+pqLcGqFmMFEgLTNdjxudIHw5IUdawp6tpg7IpQKrSPX1lMalROI3FHfOO5bs\\\\\\\/mBG9Dl+8qlWZt2O0uokjxS80ZcNt5H8e9iodRfeh\\\\\\\/oue5IbCAu6A\\\\\\\/b1R0iKbHRM3+uPDlKn0hUc6QQ0eTOQEQ1Fh03iWTC33jgowoTxwc7mpKiUFmtq0Yjnx+3iW+mZp3x+\\\\\\\/gDR\\\\\\\/B+kg\\\\\\\/29wj\\\\\\\/JyHRWjBZeHhC9Sll37uf6UMQ1XIpKfHxaTXMGMlEJSBC9nUCr15fSoVDBOTcj4akts\\\\\\\/pGQBz\\\\\\\/8LgAxWuGsDaWJHnEOOzMYNzgDG4g57XFrK+Cj3kK9vs5z098xfogplDKvZmK4Ll5v5TANIzMPKSvZeOcsdi\\\\\\\/Tsi4b9FcY31OYixLRWrt9ZbN+3jFWl1M0IWpJEtNfb21cZu88DGuQ20mQzgYHvmMXvMpHWJSmt+EoLSyxBFeIiVDr7+fvOPsxZmCfLMN5y\\\\\\\/gR176Y5sUkRHhGInuQVQ6xQ\\\\\\\/SbDJs6e+\\\\\\\/B2tNKlIvWzdAvCL7cPC8DJAT31HRz8W9UR9ENl34Yntl3CAdJ4ugg2HdiKKa0rHwLq1zn1cQV7Xo6SwewJeGRLgQOdfHoHn9g66HaBbroBFmmocKas0rgbybByJrApaAG7P1VKKXdRLfNI0gdA0Ye6HjI4+oE2cvNgCzoAPAIvqcVKu\\\\\\\/8zQ+RGrMtT+oqytGhlElf2fuvDhX95F9mc1GTJ+HPgRAtLzH0ctdTnGCX\\\\\\\/kwJ3j7TlPQou73k5wm4\\\\\\\/bIGy55VqAASI+PC6By22EO49kvV9w1VALg2KYc\\\\\\\/rSUsnxvp7KkV\\\\\\\/EtasTXNMmdsXvYbvP\\\\\\\/xL1OVcH1f2XG9me1gfHO9ThBM+zx\\\\\\\/v1fi8QSNlrCFcRtrwLJ971URsGVHOZGdqI4QPzJmQbHMZxTm7V11N\\\\\\\/xwRRt\\\\\\\/ibjAWE\\\\\\\/UzFZjG3eUqCKxU6vC8oumPAahtIbh5TyeNfmsRbT4yJ2oxiRK4DeoQaQbmtDLfJmDj99vV4wP\\\\\\\/T93prOciEB934j30bn+6A5KCAKzXQua7qUJoEYYy\\\\\\\/WeS7+lebsA7Ehwho0Mrag1aQgh3C2KU3h5g\\\\\\\/CWtRkVhLS9tRWxUddRUt1ZCE5vc60DFZniJqkq30U68MI0FWiKrp+zly11eH83DUnSoYNPu2g7NiURC25Xg20dYRTDeT6vHDcWx34gpL\\\\\\\/blI7Oed1dq2431HuIfn6U5wX8\\\\\\\/6n0M\\\\\\\/rxmQV47YRFlZ9ptNb1tguIoMpChjidLV7ar86Bw0IrkEP7vDIeFz8wykmumfsg8qHYeEtI8HKM4PxyGaq5RUHf7sPTf1eiEX+CHZZOC6papyFV1UWrNE4kiVydCHxAlWAMJfTR3Fcwz45LCss+IJV403GFknoXQ7Nl88VcvndKpJD0YiOLQHzaP5F4X5s8XhSiKMbN8yWJRNMEmLBmCpYWgW5kvGMd5kI10\\\\\\\/zwGJwAFwpnPWU\\\\\\\/b26c4LcBG3M9Wn5MsNVL9ltYBYJoDsktiq50JqzYc6yCnNRLe9qSC6LrB97zSf+Iyk72erYkIvFK0kJpzKDd6xvcLoj0pM1K6vD3JBAHkcFrf1yLnTNp+622YL0vjwll78dR5bihxkvXFc\\\\\\\/RJKtvxttUU7yC7RR96zPzq8conVuZS2mimYMIZz33og26PXLwfhy8L1YWpLFsOW14EJWcAGamf84uzxK\\\\\\\/pu+S0hweVF4lnO4wcUnp8x1Z3ckOzAO6FTgSNz4YD06Gv6elrWqZjnH3J0VjxPcGOh62tA9SG7AmoWTBrk1rhXYRaYmR1njfj3GpmUbDwjAwsYdC\\\\\\\/ty\\\\\\\/DALYUPyquPW973RtFSa56WQWY\\\\\\\/FM9KQL00XIQ0giRV8WB3Og\\\\\\\/xaJ7fuW7agG4iuexwYBwmbUN2i5Ihh1Cd9oewWVNcyR6goZFMYhvgIFblWNsYHJfZeEorUdkiC6FlDcXb0hc7jaKBcnHFMEmPe+CQiXSwEB+m4s8QRWO0TG9ce2C29T5FmxLvrAu6iYa0gC6h9WOwywG3mY\\\\\\\/t8VXrKRoHt3QB2Vrh6B3KqKtHsPOO7ruGtHVAzeN5s1UR+tewsoIB41nKFSjmyJck331JRY9QxZHt9RcwgzjPRz8mDEUUTeo5d25FB2qKA4EFKHI+vQX7uPRmADgiB3J+\\\\\\\/B6ZHBaJxnDDNfgGIlku5pKv\\\\\\\/LueFvpBepyno+6n1ZeYIp4w18C10ILyoG\\\\\\\/8Wvq7evp0a50dSOfNU0gJQb71Snzmrm6iWl8lMNkBSj+oFL3p8+b2v009lImZ8gYRzU7GF958ltua4iwESDflTIC4hgpjyc\\\\\\\/9sZIoqi0G1Wg0CW1lKb6FeDBf18KtK38v6NJE\\\\\\\/+xO\\\\\\\/xdzqNxKc0kaX9NYxP4pkPtmdLBiPtOBOyW2PUqHR9oA532p7mxXnT6GBwXvAFjFwZnNdVoYLC9xZn4ZjKDsnn7i9xHfHulDn3luPVHc\\\\\\\/FhGBnkVxAe4Vg+ESpzrRtRIbZkHYq4S2z2nItxY0nxVEfJ\\\\\\\/oE6HORH6R\\\\\\\/AHAi6UM8QXhjzDYUTEna4e\\\\\\\/ThsHnHQb9oI4XJwDvlVtnQ7jT\\\\\\\/QS2Ew1PBav+R3sBAesTzqhpN5yUIjEc0f8\\\\\\\/nC1Fi1auCVA4cpOCEgzDnr+0a3UhElDqtNmU4x5\\\\\\\/S27NQ1DxnYNLHUAfJruLfp4TmMKhSZ85AXgOVj63EtQMxaBbDhfxJEWZ6\\\\\\\/6gx7dMDLFMlrBkzST1Op9tR0bL\\\\\\\/QDGOMsGCkRvQ9CTzGyDGxTAymGVCceNgDZsDUm8M5SD5drw1u2UfPrhrh4LYqlQ2MjoJuOF\\\\\\\/6RHh2EblGxY0EG0G7ADH\\\\\\\/xmgaFvr4vVyFxq52PSV90pWYM+9Lhvo\\\\\\\/Y8+Kid6JcrR6atK43pm+5FLhyDjIWO8PcEtO8kZjhwCN+Jg7m1Ay9pWMRvx3cOS2UaQUvTXq+E2n7UbhZ6MjnOL6ud9JHlhGhLf7bAMWveaRQtoBH00U8o\\\\\\\/z2LOMRGB4OO1a1xRKX8T\\\\\\\/IFmvBHiYCMOX52zk7cDPwMiE5eNNj0QOx38u2uTT4C4srIpVyU4wMkysFS0VYB48TLlKP\\\\\\\/+JJCVjc2GQEZPDMqWWkJeFaChNYuGAqZMdPdDmlBXujymaCOHWBgHTrg2TnrOy08eVgU\\\\\\\/uD9cnFsMXxWE5Rnw1LliTeLB20g338ibVMZKHiBI6hSCS+zcXeBuwVNhK\\\\\\\/D9WApu5j1Y+Ps9bwyT3oKpAVtsSHDE3wR3dX7btKlNdGTTA26ERcLNUUgQTFxyo3FXIwR0oNCjOVazh5f5d72XBySG06VPjJiK7R4W+f+1H5W\\\\\\\/ooNiN5gI\\\\\\\/nhgs8vEmgRliqYzseaz1JrrthdF6yvzCQ7pXzJ4nRx3HSua\\\\\\\/gLQ8rpwuK2um1wJhDnqXMmF5mZYsIX04f0AyTfXHBhkVDmocaqpOi4vuG\\\\\\\/GAj1n1MOBH9ktQaDfukhqITa+bbiyKjBVvhQy\\\\\\\/YAOSZHBbgATxb2URv5YC++ncZvhRHLK5B6r50b3YxbPLiszPoxtPf8Nc2FbXSGQSDCmCl9z5GKLpx7pN8ksAEon3araQQPpicDnjsXkhKGgnAHCWSj45zdOtfAQohu4gz\\\\\\\/oNCiRA1NFgaQc\\\\\\\/GVZrklvpFNHKAbrZm4pfV30cMcADJMJ4vDOMvmOZAw\\\\\\\/jyqpAkwaVTY3U5moQBAdKPfFYLOgK6dLsmcsAuHU85xJRdnRIAzV94VutOyagODzvnaQUZY+jvkvgzNhnZ1N1\\\\\\\/dLhIR1bycOn+xMKUSvobE9JJPqTBBI7Qya0y8p8CMRZSi2a6lgpwkz0EthgOp8J1zq7G36Ko99S0ijl5FCurSJHWZjo85VPg+vluWHrlr9IeOnNyWat+lZDxBZyKWbSm6chi47sW5miHKQjYfll35So5UfiDVMDYEJT3WqgLO4MLELS2lJE+1mEOUYklZqkxfqLzA1\\\\\\\/BY4UHNfe8NLNRIGTrCUJSv7ebMJZmu\\\\\\\/TASrVPqYmDouFaW\\\\\\\/s7BNUMcrxvFJKdavof9cYsWgYdT0OpVBjUCkDDjtGy06osBF+2zJFqxAJm5X8yna22WtWuoQLHlpkJMJMAtN6R89FzgO3qf3XQqxPirq4XjUP36uL8KfIlruxbowOfb4AOyiPvisXKF4c3b9yLy89G+GwY8DYVsZqzeGa1esnLyEEc2xiHJaMPIbzd744gkohsm3xAJI7hfqe\\\\\\\/BH7DygUeGipJqCrMkdeCcQTIqkZGX9SuQbi8AvzpThOuzivlNJhiFB8\\\\\\\/QUPuItUZ+UTRLmPRbwBnSMt+FJkLcdrx+QbN7QNLobZ+hw3FdobztcLC3lb0mftAzWG8AyhN7SXkjxDPg9q4YdFB73R8WOcNUg06Zcmu8ZrPW+sXkqLG4wY9ZghbvvzwFapMBcrzh83w8gTaGXUGsdRV9lvfM2wFw3P\\\\\\\/aerN66nhF5pPxcVWBkM3MRbkbK1gPpsJ7YLBeI9UqzqGF27dFyjBRbCPN\\\\\\\/QWyhbcmKhbJ4mycDRm9Rkgol8rrjhDmgq4Lj26Arinn0ET+Jhh2HeVmdD52e0jmVVBn25oKhSYhEvCcJAHu1muqelP6+M0hov3neamXaDNoBOWa2LezRONSHdTUr2arwQhus2nIhvkrNTbPd\\\\\\\/mqJll9j6ewMpzoBC8aFdBw\\\\\\\/j0SBThWBuYdzwd90uHlFwRpKr2GL7NpCHd0X4wdWpzn3AleOuJCGP1GgihZU\\\\\\\/NaMEy1c6Hgq0ePQuCsogxKBuA7Y+qnEr3jmWTs\\\\\\\/ajDReoTe\\\\\\\/qkprIOwlPpo9w9KYJac6\\\\\\\/Gm50a+wHXMBXPqmYxzDW2UDDrHOhgSavgcqlUE+a4phrv3sn6JFVx+gwophHsX2fpbhMcThVV02QGQ\\\\\\\/KBZc34H9\\\\\\\/n3VIjrvvNevI+4eRDSIssAfpryFBWpm9V6\\\\\\\/1E3b6Ik0TYJsQsmDWGYzkOhOGCEGkOSeobfURD4fblz1rkF+iSbKGLUsT4psHvj\\\\\\\/Skg1nAFGeV\\\\\\\/014Xp7wL4nOMnW5b6ID9\\\\\\\/avJjGyTmvHcvDByXTkKNcL3mPqC5YnmUC+w9qUqwfdv7CfEtRNbm41D+ed+yxjqFMmP119PT\\\\\\\/mi08DBea2YBx2g9WVx6RlLWrTdPPD47gNVHgjO+Pa2vKensNMxYsHrAO2ZIBvhQTfBSidXrrwMxNejKaOanjqIi340vUWdHIKMlZgbhftgtqWiCubqnuRFmpKGW9uIsnwEtdT7QaL8Xejj4Ok846SA+v3FLp0bmS5BiokIHvh5JsuDuTueP8pa573H5k+F\\\\\\\/l9AkFz23o7uHCmTv+4uBqe\\\\\\\/bljrmk2dZQslgM6uLXBRzrRjFhIdALnhgtzkkWVS4Nxv8YDIJfGB3UvqqMzLtOPAZouFnpVV0LZaA+tvoLy\\\\\\\/ctx99\\\\\\\/uI9JK7y8nmc4tnnl1aHs5BwlToasgBoz+zlY9MuMOkcSg9hrKP67kbtF2BIvPZ8\\\\\\\/cBmAgUW3kdUCQ+SnY9irCuN1kvOaVbVtBK\\\\\\\/H02af6xlOH4sozNDOCRY+kDtvOBqQFSNDKvysIFZfuGySowcih66I3UDP7hCEus0bVDn6jJbaaM\\\\\\\/FIqoQNRoLEspbD8o0UD8XfLuTvuxtGF5qClKIQmsvI9mCMvtSoJmJPhI7B+yGtrDkNIM3CYgRiH5Ex9\\\\\\\/VZ5sSHVgf+c3owvEF8wT5jV1k7hxYEYSTmO\\\\\\\/e0E0KLzuTa9CO6Z\\\\\\\/E6UsNsERkVNU3x5gMHIWYj6Qj6QxX6TLgYwkTIxlr4CCP1k2H\\\\\\\/58bMJKBJiS2W77rKPYiRG4WY1+O0R9TsauqIaNY2S0JwdCOzPm9OSnLFJlzDaXaW27UZYAPkLYkhTVnoWJpjHr9darmD\\\\\\\/eAl3zeJVmc0IwMAO8M40Q55FSgu3tXJnk6tixpAwfkx1c3QNCfbnLd6NvW6glG\\\\\\\/HPM8MkmxQGpvPLpFA1nCuqBR\\\\\\\/1Cn2D6wIFwvF6vSk\\\\\\\/6kwBEyqomjfjlm+M7OwwvBEXzuku0CHDaTvrQH060TRfqdaatLaBdX3Ysp9ZlZn\\\\\\\/EriuHSH+x8XtdeiroQbj8gyAVYAxvRJuYjSV+Qsudhugv7O9zrQwPslgfA\\\\\\\/jpYfmsG9yQcfkrkSZQ3tpEavT9ZHnnDbvSmY1idSYgkALtUGxwnc81cHZAve29\\\\\\\/elt7YgUpvxO8xTlIJkReUiNAnKIzXbTAzBPlldu0gpBLWbLIuyHvxMRXhSy3rqwQ\\\\\\\/FHuY2aCCpyCCxi42MF2V4xzs3T7da2TImpbVX1VU2HWVTYY\\\\\\\/nvoLdRPB4sjmr0b3GDlRP01kKlmx7b4u2iizyy359PzAGVIY+ckBVyL9NLvggtjmRcOFcYItopTuwZJmjikvOM\\\\\\\/PH\\\\\\\/8NdAjeJFHjwWYNntwV2R7uT1QxLVOQnR45+NRDvRGVevox\\\\\\\/AJEDwghSTlo7wBIdNL+BZ4gvhsPtCrRjb1kA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022XKCGAew6dfogm4Zy3Q+xKw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022dab4b6eb7eb3cbde2a397564795dbc40030ba1ed3c190dfe9de5377524268151\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022RiNv46HmyMznthe\\\\\\\/vlo6XonbJVmgqJNIjgZW5hJOD7LnYawFByRN8WU6Si+hdwF6wIuBKhkmlJlJnyale9ugQq8ZMmtAIE2JM35NByoOBQ+XiWasdyAT9Mnge\\\\\\\/FCE49JzsJRm883z+26cqyOqwgfTFFFlZWk9kXwXizAzB4xJ+6nU0pLICAXYZbYxU9Nv5TpFHvETKS6hmPNH7xMlft3YbuG3iRhe2jO9j52okQ\\\\\\\/SBUyXVzj5a2YvbD3mvFqYB3QV4tCp8XcsauZ\\\\\\\/yfUYjpTUBQrToBmd3jDuzwVhWTfLTQoJDMCyNbx9+ukUmD3OXICtLg+7To1DfZjnEkf+vT0PPTagN73nfvukqaj1sk0ESOGjwV7ZKs7vlsbuDhQOqGch7e7YqcPBUGljVgXQSyifTawcFo9VDy4OHg1gsm5S+G\\\\\\\/iH4gjXvaKQU4hKZc7o8vXWk8v9piuUsffsDoqnJRCTGqiDJ21f5rZUqOCp6wsmR7TLnMCGKApyHWyAor6eFJgnB58S9BB5AoXCMkkh4XN60HZgMxvInxVmjiuKwCvncc4anryi2IqzYRE0eN\\\\\\\/z7KaTwGCiZNqIhUI4a8oEL2VxVEyFRr31yf52J4ANBJRirIX022FTrTbh9+wRx2bchlQ4TVUtqWUrWFhGq+E4bl+CF+LbM2xQDuCAngu\\\\\\\/DRTp56tFmZ0amV2Xju2kEESwILPfrkfY6ynbcV4WlYQ1zmKH8yv\\\\\\\/p+ARq5DUJfsBsNjYEGn95+3JFMMgQxlX1Kr9DDwcjwWBUYKBW\\\\\\\/peDJkR01E0FUObiiGyjmYm0EDQdc23xwRIrp9UZpcY0l5wtNAru7pE7aJovut3+g9CYzZZgEPnx6yjeerY6CAMptA1WqWySmGogIAUm4we1bEl+I1F\\\\\\\/GLXrzOf7pRf+zA7lrTxQZrWe39oiPCBDl184smfzcbmxlGQLSmpbqXK5evbJ\\\\\\\/qDnYW+Oce4+p60R3ykKdK3azONBKNF47ZseGekacZN1oCgQRqI9QQtDokrdvJfa\\\\\\\/toJOX30hclCnCyY1Mb5AEaV87LQ\\\\\\\/nWZDeFS1x5yrdaTg9LlqmBcVPZWm1xm1q1GPizuTK9\\\\\\\/Fwl8xPx11q8Vu59Vtu9Hm7f9TKrZCtNO\\\\\\\/tU88QzBIBCsDjDaTe+vn5QYBAmhLQ7LhmmGbY4lRKwoENTSvrxyuZ+Wx8xRKwpFzIpEGTfHlK7XWL4ILwDEhpMZChtFEg\\\\\\\/u8eLOE06zLYrNRWE1HvEH5dPyf2WKUhhsUOrkRRFsaOHRBRjjrGDPYlaqWcE9xTHK3eW1gsk0TfcywOoM7+ziD6At8WEFVnqhbhfB8eQI36B5i9hbTuTbMfF0uTiABaS3rdnqdqCmT\\\\\\\/vl7fbPwhoBDmOM3BmQEzm5w7b8R41D9eR6ek5ZxQotO1zs8gWU7I5a5y1M0bAykZj9uM5xi3+fpN3gilM2mQh6w4LhfVXBAX7QeHu18eYddUaaMWcMtHwblh4xOVXO1axfNupIqQdICaajeqUrZlTfZa\\\\\\\/eEDErer0rqTs8xxHL27XfSd0bym9Egu6HPuBm258e6pk2fRcVfkUfhOoo6i7RK57\\\\\\\/sWcx+aeAilpt4eg4QtgPkRmtTSM5Er51QwxvMZvlRSXBwSGlA8bM3hTmCsWNGB1MRrV6GPndB4ZOfeqdCuyCesM7RoBXDU\\\\\\\/l0\\\\\\\/fz76eoj0LvVn0qsyC+WmXbWi\\\\\\\/B6LD8nGCEiNMowV6fFF7MA0T2X4d7rwFP1VllDYKb\\\\\\\/lZCvoh\\\\\\\/xUxI1UksU3clJIuVDaLQMXaybzTSfW\\\\\\\/TKubwexOQTdF0qbIpCWpU8uotufj8aVGRGg0\\\\\\\/jJTbY77yu\\\\\\\/Rp6BVgVa4PHKM5p9NsD+R25isU6gg8DbrUXeL5CpEdgEjypQJA7JsUR3+URM7gcwy9MLtE+8tbQEnZDga9VmbY960di5bADdrdPOXOX+tfVem9jq2zvo6DMZ5hqEKtt0C5ubVbRMGVp6mHuhm4t1vlVmMHCfkFsoDOPbp76E4IUbgPsNJDYzA==\\\u0022,\\\u0022iv\\\u0022:\\\u00220gT0va6ibaRaOjoZ04MvuQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d054334d0e9468b1644943b62003c4c04864c05521d27c7348f9c92e2f5f67d6\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460461 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/11/3693" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-a3jkuiwTkCrHaBYtAU4zKt1e522plumLOX5VIimAdMY" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/11/3693" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-eMhB2PXMLCQ6ueUMn3n5LVQta6jYdCo7CQcuRAVWeeo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/11/3685" title="Non-Mendelian Dominant Maternal Effects Caused by CRISPR/Cas9 Transgenic Components in Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/11/3701" title="Extensive Epigenetic Changes Accompany Terminal Differentiation of Mouse Hepatocytes After Birth" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460461" id="node460461" data-pisa="ggg;6/11/3693" data-pisa-master="ggg;g3.116.035220" data-apath="/ggg/6/11/3693.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Prediction of Resistance to Pasteurellosis in Gilthead Sea Bream (<em>Sparus aurata</em>) Using 2b-RAD Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-4480-4612" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Christos Palaiokostas</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-5928-9361" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Serena Ferraresso</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-1333-0733" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Rafaella Franch</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-1805-0762" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ross D. Houston</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-4351-2826" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Luca Bargelloni</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3693-3700;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.035220 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460461" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Christos Palaiokostas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Midlothian, EH25 9RG Scotland, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christos%2BPalaiokostas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Palaiokostas%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChristos%2BPalaiokostas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4480-4612" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Christos Palaiokostas</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:christos.palaiokostas@roslin.ed.ac.uk" class="" data-icon-position="" data-hide-link-title="0">christos.palaiokostas@roslin.ed.ac.uk</a>
<a href="mailto:luca.bargelloni@unipd.it" class="" data-icon-position="" data-hide-link-title="0">luca.bargelloni@unipd.it</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Serena Ferraresso </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Comparative Biomedicine and Food Science, University of Padova, 35020 Legnaro, Italy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Serena%2BFerraresso%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ferraresso%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASerena%2BFerraresso%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5928-9361" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Serena Ferraresso</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Rafaella Franch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Comparative Biomedicine and Food Science, University of Padova, 35020 Legnaro, Italy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rafaella%2BFranch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Franch%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARafaella%2BFranch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1333-0733" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Rafaella Franch</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Ross D. Houston </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Midlothian, EH25 9RG Scotland, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ross%2BD.%2BHouston%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Houston%20RD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARoss%2BD.%2BHouston%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1805-0762" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ross D. Houston</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Luca Bargelloni </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Comparative Biomedicine and Food Science, University of Padova, 35020 Legnaro, Italy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Luca%2BBargelloni%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bargelloni%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALuca%2BBargelloni%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-4351-2826" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Luca Bargelloni</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:christos.palaiokostas@roslin.ed.ac.uk" class="" data-icon-position="" data-hide-link-title="0">christos.palaiokostas@roslin.ed.ac.uk</a>
<a href="mailto:luca.bargelloni@unipd.it" class="" data-icon-position="" data-hide-link-title="0">luca.bargelloni@unipd.it</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/11/3693" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460461" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460461/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/11/3693.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460461" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460461/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/11/3693.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460461" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460461/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/11/3693.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460461" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460461/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/11/3693.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460461" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460461/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Gilthead sea bream (<em>Sparus aurata</em>) is a species of paramount importance to the Mediterranean aquaculture industry, with an annual production exceeding 140,000 metric tons. Pasteurellosis due to the Gram-negative bacterium <em>Photobacterium damselae</em> subsp. <em>piscicida</em> (Phdp) causes significant mortality, especially during larval and juvenile stages, and poses a serious threat to bream production. Selective breeding for improved resistance to pasteurellosis is a promising avenue for disease control, and the use of genetic markers to predict breeding values can improve the accuracy of selection, and allow accurate calculation of estimated breeding values of nonchallenged animals. In the current study, a population of 825 sea bream juveniles, originating from a factorial cross between 67 broodfish (32 sires, 35 dams), were challenged by 30 min immersion with 1 × 10<sup>5</sup> CFU virulent Phdp. Mortalities and survivors were recorded and sampled for genotyping by sequencing. The restriction-site associated DNA sequencing approach, 2b-RAD, was used to generate genome-wide single nucleotide polymorphism (SNP) genotypes for all samples. A high-density linkage map containing 12,085 SNPs grouped into 24 linkage groups (consistent with the karyotype) was constructed. The heritability of surviving days (censored data) was 0.22 (95% highest density interval: 0.11–0.36) and 0.28 (95% highest density interval: 0.17–0.4) using the pedigree and the genomic relationship matrix respectively. A genome-wide association study did not reveal individual SNPs significantly associated with resistance at a genome-wide significance level. Genomic prediction approaches were tested to investigate the potential of the SNPs obtained by 2b-RAD for estimating breeding values for resistance. The accuracy of the genomic prediction models (<em>r</em> = 0.38–0.46) outperformed the traditional BLUP approach based on pedigree records (<em>r</em> = 0.30). Overall results suggest that major quantitative trait loci affecting resistance to pasteurellosis were not present in this population, but highlight the effectiveness of 2b-RAD genotyping by sequencing for genomic selection in a mass spawning fish species.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared Data Resources</a></li><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">Genomic Selection</a></li><li class="kwd"><a href="/keyword/2b-rad" class="hw-term hw-article-keyword hw-article-keyword-2b-rad" rel="nofollow">2b-RAD</a></li><li class="kwd"><a href="/keyword/aquaculture" class="hw-term hw-article-keyword hw-article-keyword-aquaculture" rel="nofollow">Aquaculture</a></li><li class="kwd"><a href="/keyword/breeding" class="hw-term hw-article-keyword hw-article-keyword-breeding" rel="nofollow">Breeding</a></li><li class="kwd"><a href="/keyword/high-throughput-sequencing" class="hw-term hw-article-keyword hw-article-keyword-high-throughput-sequencing" rel="nofollow">High-Throughput sequencing</a></li></ul><p id="p-4">Infectious diseases are a major threat to the profitability, sustainability, and welfare status of farmed fish production (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Yáñez <em>et al.</em> 2014</a>). Gilthead sea bream (<em>Sparus aurata</em>) is one of the most important farmed fish in Mediterranean countries, with an annual production of ∼146,000 metric tons (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Federation of European Aquaculture Producers 2014</a>). Pasteurellosis due to <em>Photobacterium damselae</em> subsp. <em>piscicida</em> (Phdp) is one of the primary disease problems faced by the sea bream aquaculture industry. High levels of mortality (∼90–100%) are frequently observed, especially in periods where water temperature rises above 18°, with larvae and juveniles being most susceptible (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Noya <em>et al.</em> 1995</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Magarinos <em>et al.</em> 2001</a>).</p><p id="p-5">Selection for improved genetic resistance in aquaculture breeding schemes is a valuable tool to help prevent or reduce disease outbreaks, especially where effective therapeutic agents or vaccines are lacking (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bishop and Woolliams 2014</a>). Moderate to high heritabilities have been estimated for resistance to many common diseases, indicating that rapid genetic progress can be made through selective breeding (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Ødegård <em>et al.</em> 2011</a>). In addition, recent technological advances in genome-wide sequencing and genotyping technology offer the potential of deriving more accurate estimated breeding values (EBVs) for individual selection candidates, as compared to the classical breeding approach where breeding values are typically estimated at a family level (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Goddard and Hayes 2009</a>). The application of genomic data to breeding is particularly valuable for disease resistance, which is typically expensive or impossible to measure on the selection candidates themselves. While marker-assisted selection for major disease resistance loci has been well documented in Atlantic salmon (<em>Salmo salar</em>) breeding programs (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Houston <em>et al.</em> 2008</a>, <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">2010</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Moen <em>et al.</em> 2009</a>), few successful examples exist for other farmed finfish species. Genomic prediction uses genome-wide markers to estimate breeding values, and can deliver significant improvements in selection accuracy compared to traditional pedigree-based approaches, even for traits with a polygenic architecture (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Odegård <em>et al.</em> 2014</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Tsai <em>et al.</em> 2015</a>, <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">2016</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Vallejo <em>et al.</em> 2016</a>).</p><p id="p-6">Previous studies into the genetic resistance to pasteurellosis in sea bream have detected resistance quantitative trait loci (QTL) using microsatellite markers (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Antonello <em>et al.</em> 2009</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Massault <em>et al.</em> 2010</a>). However, these studies were restricted by the low resolution of microsatellite markers, as compared to the high-density single nucleotide polymorphism (SNP) genotypes offered by SNP arrays (<em>e.g.</em>, <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Houston <em>et al.</em> 2014</a>) or genotyping-by-sequencing approaches (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Davey <em>et al.</em> 2011</a>). Restriction-site associated DNA (RAD) sequencing is a reduced representation high-throughput sequencing technique for the concurrent detection and genotyping of SNP markers in multiplexed samples with a unique nucleotide barcode (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Baird <em>et al.</em> 2008</a>). RAD sequencing and similar genotyping-by-sequencing techniques rely on digestion of the genomic DNA with a restriction enzyme, and subsequent high-depth sequencing of the flanking regions. These techniques have been applied in several studies of aquaculture species to generate high-density linkage maps (<em>e.g.</em>, <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Palaiokostas <em>et al.</em> 2013a</a>,<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">b</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gonen <em>et al.</em> 2014</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Palaiokostas <em>et al.</em> 2015</a>) and perform genome-wide association studies (GWAS) in a cost-efficient manner (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Campbell <em>et al.</em> 2014</a>). A flexible and easily streamlined variation of RAD sequencing, named 2b-RAD sequencing, utilizes type IIB restriction enzymes to cleave genomic DNA upstream and downstream of the target site (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Wang <em>et al.</em> 2012</a>). In theory, 2b-RAD samples all the endonuclease recognition sites for sequencing, circumventing potential biases that may result from the size selection step in the original RAD protocol (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Puritz <em>et al.</em> 2014</a>). 2b-RAD data have also been applied for genetics studies in aquaculture species, for example to test genomic prediction in a limited number of Yesso scallop (<em>Patinopecten yessoensis)</em> families (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Dou <em>et al.</em> 2016</a>).</p><p id="p-7">In this study, we used 2b-RAD sequencing to identify and genotype genome-wide SNPs in juvenile sea bream challenged with virulent Phdp bacteria, and recorded for survival time. A high-density SNP linkage map was constructed and a GWAS was performed to test the association between individual loci and resistance to pasteurellosis. Finally, genomic prediction of resistance was tested using several genomic selection models and marker densities to evaluate its potential in selection for improved resistance to pasteurellosis in sea bream.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Sample collection and preparation</h3><p id="p-8">The experimental population used in the present experiment was part of a larger group of juvenile sea bream that were subjected to an experimental challenge with Phdp to estimate heritability of disease resistance, as reported in <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Antonello <em>et al.</em> (2009)</a>. Fish were provided by the fish farm Valle Ca’ Zuliani (Monfalcone, Italy). All broodstock fish were originally sampled from wild populations. Fertilized eggs were collected on the same day (year 2006), from natural mass spawning events occurring in four different broodstock tanks; therefore all fish had approximately the same age. Each broodstock tank contained 50–60 fish with a sex ratio 3:1 females to males. Approximately 10,000 eggs were collected, pooled, and kept in a separate tank without any size sorting until 110 d old.</p><p id="p-9">All fish were then transferred to the Istituto Zooprofilattico Sperimentale delle Venezie (Legnaro, Italy) for the experimental challenge. Fish were divided into two aerated tanks (A and B), each with 800 liter of recirculating seawater (salinity 35 ppt). Water temperature was maintained at 19°. After 1 wk of acclimation, fish were experimentally infected with a highly virulent strain of Phdp (strain 249/ittio 99), as described in <a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Antonello <em>et al.</em> (2009)</a>. Mortality was monitored daily for 19 d (Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035220/-/DC1/TableS1.txt">Table S1</a>). Mortality levels were nearly identical for both tanks, and only fish from tank A were included in the current study. Fish used in the challenge originated from 67 broodfish (32 sires, 35 dams). As already described in <a id="xref-ref-1-4" class="xref-bibr" href="#ref-1">Antonello <em>et al.</em> (2009)</a> and <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Massault <em>et al.</em> (2011)</a>, parentage analysis was carried out using a panel of nine microsatellite loci (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035220/-/DC1/TableS2.txt">Table S2</a>).</p></div><div id="sec-3" class="subsection"><h3>2b-RAD library preparation and sequencing</h3><p id="p-10">A total of 892 2b-RAD libraries (67 parents and 825 juveniles) were constructed by following the protocol reported by <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Wang <em>et al.</em> (2012)</a>, with some modifications (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Pecoraro <em>et al.</em> 2016</a>). Template DNA for each individual (500 ng) was digested in 6 μl reaction volume using 1 U AlfI (Thermo Fisher Scientific) at 37° for 1 hr, followed by enzyme heat inactivation at 65° for 20 min. The ligation reaction was performed by combining 5 μl of digested DNA with 20 μl of a ligation master mix containing 0.4 μM each of two library-specific adaptors with fully degenerate cohesive ends (5′ -NN- 3′), 0.2 mM adenosine 5′-triphosphate (New England Biolabs), and 1000 U T4 DNA ligase (CABRU, Arcore, Italy). Ligation was carried out at 16° for 3 hr, with subsequent heat inactivation for 10 min at 65°.</p><p id="p-11">Sample-specific barcodes were designed through a Barcode Generator program (<a href="http://comailab.genomecenter.ucdavis.edu/index.php/Barcode_generator">http://comailab.genomecenter.ucdavis.edu/index.php/Barcode_generator</a>). PCR reactions (50 μl) were prepared containing 12 μl of ligated DNA product, 0.2 μM of each primer, 0.3 mM dNTP, 5× Phusion HF buffer, and 2 U Phusion high-fidelity DNA polymerase (New England Biolabs). Each library was PCR amplified using the following conditions: 13 cycles of 95° for 5 sec, 60° for 20 sec, and 72° for 5 sec.</p><p id="p-12">Adaptor and primer sequences were those reported in <a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Wang <em>et al.</em> (2012)</a>. PCR products were purified using the SPRIselect purification kit (Beckman Coulter, Pasadena, CA) and quantified through a Qubit 2.0 Fluorometer (Invitrogen). The quality of all amplicon libraries was checked at 1.8% agarose gel. Additionally, the quality of 10% of randomly selected libraries was also assessed by running them on an Agilent 2100 Bioanalyzer.</p><p id="p-13">Individual libraries were pooled into equimolar amounts by adopting two different multiplexing strategies for parents (24 libraries per pool) and offspring (48 libraries per pool). The quality of each pool was verified on Agilent 2100 Bioanalyzer. Finally, pooled libraries were sequenced on an Illumina HiSeq2500 platform (Illumina, San Diego, CA) using 50 base single-end sequencing (v3 chemistry).</p></div><div id="sec-4" class="subsection"><h3>Genotyping RAD alleles</h3><p id="p-14">Quality and adapters trimming of sequenced reads were performed by running a customized script (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Pauletto <em>et al.</em> 2016</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Pecoraro <em>et al.</em> 2016</a>), obtaining 34-bp long fragments. SNP calling was performed using STACKS v1.23 (Catchen <em>et al.</em> 2013).</p><p id="p-15">For each family/cross, individual genotypes were constructed using components of the STACKS pipeline as follows: (i) for each individual, <em>ustacks</em> program was employed for building loci from all quality control (QC)-passed reads using parameters <em>-m 10 -M 2 -N 3</em> for parents and <em>-m 5 -M 2 -N 3</em> for offspring, (ii) a catalog of loci unique for all families/crosses was constructed by using all parents’ reads on <em>cstacks</em> program, then (iii) each set of parents/offspring per cross was matched separately against such a catalog (<em>sstacks</em> program), followed by genotype assignment by setting the following parameters on the <em>genotypes</em> program: <em>-c–min_hom_sequations 7–max_het_sequations 0.05</em>.</p><p id="p-16">Unique tags created by STACKS were mapped against a draft assembly of <em>S. aurata</em> genome (L. Bargelloni, personal communication). Mapping analysis was carried out by means of CLC Genomic Workbench 7.5, with stringent criteria (length fraction = 0.9, similarity fraction = 0.9, nonspecific match handling = ignore).</p></div><div id="sec-5" class="subsection"><h3>Linkage map construction</h3><p id="p-17">Linkage map construction was performed using Lep-Map v2 (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Rastas <em>et al.</em> 2013</a>). QC was performed for each full sibling family by excluding SNPs with minor allele frequency &lt;0.05 and those deviating from expected Mendelian segregation (<em>P</em> &lt; 0.001). Linkage groups were formed using a minimum LOD threshold value of 8 in the <em>SeparateChromosomes</em> module, allowing a maximum distance between consecutive SNPs of 50 cM. Marker order within each linkage group was performed using the <em>OrderMarkers</em> module, where the likelihood of marker order is computed by using a hidden Markov model (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Rastas <em>et al.</em> 2013</a>). Map distances were calculated in centimorgans, using the Kosambi mapping function.</p></div><div id="sec-6" class="subsection"><h3>Trait definition and heritability estimation</h3><p id="p-18">Heritability of surviving days was estimated with the R/BGLR software (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Pérez and de Los Campos 2014</a>), using both the pedigree-based and the genomic relationship matrix.</p><p id="p-19">The animal model was applied:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3693/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3693/embed/mml-math-1.gif"/></noscript></span></span>where <strong>y</strong> is the vector of recorded phenotypes (days to death; animals surviving at the end of the experiment treated as missing values sampled from corresponding truncated normal distribution with the resulting values being ≥20), <strong><em>μ</em></strong> is the vector of the intercept, <strong>Z</strong> is the incidence matrix relating phenotypes with the random animal effects, <strong>u</strong> is the vector of animal effects ∼<em>N</em>(0, <strong>A</strong><em>σ</em><sub>g</sub><sup>2</sup>) with either <strong>A</strong> corresponding to the pedigree-based relationship matrix or <strong>G</strong> corresponding to the genomic relationship matrix, and <em>σ</em><sub>g</sub><sup>2</sup> corresponding to the additive genetic variance. Finally, <strong>e</strong> is the vector of residuals. The <strong>G</strong> matrix was estimated according to <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">VanRaden (2008)</a> using the <em>kin</em> function of the R/synbreed package (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Wimmer <em>et al.</em> 2012</a>; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035220/-/DC1/FileS1.zip">File S1</a>). The additive genetic variance was estimated by applying Markov Chain Monte Carlo (MCMC) algorithm, using a prior that followed an inverse scaled x2 distribution (df = 5). The MCMC used 10-Million iterations, first 10% of which were discarded, and values were stored every 1000 iterations thereafter. Convergence of the resulting posterior distribution was assessed both visually (inspecting the resulting MCMC plots) and analytically with Geweke’s diagnostic using R/boa v1.1.7 (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Smith 2007</a>).</p><p id="p-20">Heritability for the number of surviving days was estimated using the following formula:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3693/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3693/embed/mml-math-2.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3693/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3693/embed/mml-math-3.gif"/></noscript></span></span> estimated additive genetic variance and <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3693/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3693/embed/mml-math-4.gif"/></noscript></span></span> the residual variance.</p></div><div id="sec-7" class="subsection"><h3>GWAS</h3><p id="p-21">To test the association between individual SNPs and resistance to pasteurellosis (measured as surviving days), a GWAS was performed using R/rrBLUP (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Endelman 2011</a>). The mixed model applied was based on <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Yu <em>et al.</em> (2006)</a> and had the following format:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3693/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3693/embed/mml-math-5.gif"/></noscript></span></span>where <strong>y</strong> is the vector of the phenotypes (surviving days or overall survival), <strong><em>α</em></strong> is the vector of unknown marker effects, <strong>u</strong> is the vector of animal random effects ∼<em>N</em>(0, <strong>G<span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3693/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3693/embed/mml-math-6.gif"/></noscript></span></span></strong>), and <strong>e</strong> is the vector of residuals. The matrix <strong>G</strong> represents the genomic relationship matrix as described above, and <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3693/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3693/embed/mml-math-7.gif"/></noscript></span></span> is the additive genetic variance estimated using REML. <strong>X</strong> and <strong>Z</strong> are incidence matrices relating <strong>y</strong> to <strong><em>α</em></strong> and <strong>u</strong>, respectively. According to the above model, additive SNP effects are treated as fixed effects, with the inclusion of the random animal effect to decrease spurious associations due to (genomic) relationships between the animals (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Yu <em>et al.</em> 2006</a>). The genome-wide significance threshold for the estimated additive SNP effects was calculated using a Bonferroni correction (0.05/<em>N</em>), where <em>N</em> represents the number of QC-filtered SNPs across the entire genome.</p></div><div id="sec-8" class="subsection"><h3>Genomic prediction</h3><p id="p-22">A genomic prediction approach was conducted to quantify the accuracy of the breeding values estimated using the SNP markers to predict the phenotypic trait values (surviving days). SNPs with &gt;15% missing genotypes were removed in order to minimize impact of imputed genotypes, since used software cannot handle missing genotypes. Missing values of the remaining SNPs were imputed using R/synbreed (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Wimmer <em>et al.</em> 2012</a>). Genomic breeding values were estimated using rrBLUP, BayesA, BayesB (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Meuwissen <em>et al.</em> 2001</a>), and BayesC (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Habier <em>et al.</em> 2011</a>) models using the R/BGLR (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Pérez and de Los Campos 2014</a>) software. The above models differ in regard to the prior distribution of the marker effects. Briefly, rrBLUP induces homogenous shrinkage across markers by using the Gaussian distribution, while in BayesA the usage of a scaled-<em>t</em> distribution induces marker size effect shrinkage, allowing for variable marker effect sizes. Models BayesB and BayesC also perform variable selection, with the difference between the two being the usage of a scaled-<em>t</em> or a Gaussian prior density, respectively (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Meuwissen <em>et al.</em> 2001</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">de los Campos <em>et al.</em> 2013</a>). To compare the accuracy of genomic EBVs (GEBVs) to the pedigree-based EBVs, pedigree-based BLUP (PBLUP; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Henderson 1975</a>) was applied to calculate breeding values using the same software.</p><p id="p-23">The general form of the fitted models was the following:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3693/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3693/embed/mml-math-8.gif"/></noscript></span></span>where <strong>y</strong> is the vector of phenotypic records, <strong><em>η</em></strong> is the linear predictor, and <strong>e</strong> is the vector of residuals.</p><p id="p-24">The linear predictor, <strong><em>η</em></strong>, in the case of rrBLUP, BayesA, BayesB, and BayesC, had the following general form:<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3693/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3693/embed/mml-math-9.gif"/></noscript></span></span>where <strong><em>μ</em></strong> is the intercept, <strong>X<sub>1</sub></strong> is the design matrix relating the phenotypes to the markers, and <strong><em>β</em><sub>1</sub></strong> is the vector of marker effects with corresponding priors depending on the model used. Marker coding followed the format where the heterozygotes were coded as 1 and the two alternate homozygotes as 0 and 2.</p><p id="p-25">The linear predictor, <strong><em>η</em></strong>, in the case of pedigree BLUP, had the following form:<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3693/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3693/embed/mml-math-10.gif"/></noscript></span></span>where <strong>u</strong> is the animal random effect vector ∼<em>N</em>(<em>0</em>, <strong>A</strong><em><span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3693/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3693/embed/mml-math-11.gif"/></noscript></span></span></em>) with the matrix <strong>A</strong> representing the pedigree estimated relationship matrix. The parameters of the above models were estimated through MCMC (110,000 iterations; burn-in: 10,000; thin: 100).</p><p id="p-26">Assessment of the accuracy of breeding value predictions was conducted according to the following procedure. The data set was randomly split into a training set (<em>n</em> = 578 animals) and a validation set (<em>n</em> = 200). The above was repeated 100 times, with the obtained prediction accuracies being adjusted for the trait heritability for each tested model. The GEBVs for each replicate of the validation data set were estimated as:<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3693/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3693/embed/mml-math-12.gif"/></noscript></span></span>where <strong>X</strong> is the incidence matrix relating GEBV with SNP genotypes and <strong>u</strong> is the vector of estimated SNP effects from the corresponding training data set.</p><p id="p-27">The accuracy of the estimated GEBV was approximated as:<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3693/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/11/3693/embed/mml-math-13.gif"/></noscript></span></span>where <strong>y</strong> is the vector of recorded phenotypes and <em>h</em> is the square root of the heritability. In all tested scenarios, the heritability estimated using the genomic relationship matrix was used. Reported accuracies for each tested model refer to the mean accuracy of the above-mentioned 100 replicates of validation data sets.</p><p id="p-28">In order to test the predictive ability of varying SNP densities, the above procedure was followed using (i) SNPs spaced &gt;1 cM apart (2614 SNPs) or (ii) SNPs spaced &gt;5 cM apart (705 SNPs) on the linkage map.</p></div><div id="sec-9" class="subsection"><h3>Data availability</h3><p id="p-29">Raw reads were deposited in the European Bioinformatics Institute (EBI) repository under project ID SRP081498. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035220/-/DC1/TableS1.txt">Table S1</a> contains the phenotypic data. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035220/-/DC1/TableS2.txt">Table S2</a> contains the pedigree. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035220/-/DC1/TableS3.txt">Table S3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035220/-/DC1/TableS4.txt">Table S4</a> contain summaries of obtained reads for parental and offspring samples respectively. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035220/-/DC1/TableS5.txt">Table S5</a> contains the location of the linkage map SNPs. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035220/-/DC1/TableS6.txt">Table S6</a> contains the genotypic data. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035220/-/DC1/FileS1.zip">File S1</a> contains the R script used for genomic prediction.</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>Disease challenge</h3><p id="p-30">The challenged population consisted of 75 full sibling families, with a mean family size of 10, originating from a factorial cross between 67 broodfish (32 sires, 35 dams). The largest full-sib family consisted of 114 animals, while the smallest had only two animals (three full-sib families). The overall survival at the end of the pasteurellosis challenge was 4.7%. Observed mortality levels showed three distinct peaks on day 7 (10.4% loss), day 11 (14% loss), and day 15 (5.7% loss), followed by a steady reduction in daily mortality rate (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/11/3693/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of surviving days across disease challenge. Frequency of mortalities per day during 19 d of challenge. Survivors were assigned a value of 20." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1668668718" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Distribution of surviving days across disease challenge. Frequency of mortalities per day during 19 d of challenge. Survivors were assigned a value of 20.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/11/3693/F1.medium.gif" width="440" height="209"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/11/3693/F1.medium.gif" width="440" height="209"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/11/3693/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/11/3693/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469721" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-31" class="first-child">Distribution of surviving days across disease challenge. Frequency of mortalities per day during 19 d of challenge. Survivors were assigned a value of 20.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Genotyping RAD alleles</h3><p id="p-32">The mean number of raw reads was 8.76 million and 4.51 million, while the number of reads passing QC was 7.74 million (88%) and 3.69 million (81%) for parents (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035220/-/DC1/TableS3.txt">Table S3</a>) and offspring (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035220/-/DC1/TableS4.txt">Table S4</a>), respectively. The STACKS catalog consisted of 202,598 unique 2b-RAD loci, of which 73,876 contained at least one SNP in the parents (EBI repository SRP081498). To confirm the identity of loci created by STACKS, the 202,598 tags of the catalog were mapped against an <em>S. aurata</em> draft reference genome assembly (L. Bargelloni, unpublished data). A high percentage of 2b-RAD loci were successfully mapped, with 93.5% of tags showing a unique match to the reference genome. In order to maximize the number of informative SNPs and minimize the amount of missing or erroneous data, RAD-tags that were retrieved in at least 75% of the samples, and that carried only one or two SNPs were retained. A total of 48 animals with missing data &gt;30% were excluded from subsequent analysis. A total of 21,974 putative SNPs were finally used for construction of the genetic map from 777 disease challenged offspring (genotypic missing data &lt;30%) and their corresponding 67 parents.</p></div><div id="sec-13" class="subsection"><h3>Linkage map</h3><p id="p-33">The linkage map consisted of 12,085 SNPs that were grouped into 24 linkage groups, in accordance with gilthead sea bream karyotype, with a total map length of 3899 cM (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>, <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035220/-/DC1/TableS5.txt">Table S5</a>). The remaining SNPs (9889) either failed to pass QC filters, or were not placed on the resulting linkage groups during mapping, and these were discarded. The female and male maps were comparable with total lengths of 3822 and 4010 cM, respectively. The number of SNPs per chromosome ranged from 366 to 607 (mean = 503; SD = 53), while linkage group length ranged from 115 to 202 cM (mean = 162; SD = 26). The correlation between number of SNPs and corresponding chromosome map length was 0.74 (<em>n</em> = 24 linkage groups).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/11/3693/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The Gilthead sea bream linkage map. The heatmap on the right side provides scale of color coding for the size of SNP clusters." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1668668718" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The Gilthead sea bream linkage map. The heatmap on the right side provides scale of color coding for the size of SNP clusters.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/11/3693/F2.medium.gif" width="440" height="308"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/11/3693/F2.medium.gif" width="440" height="308"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/11/3693/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/11/3693/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469653" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-34" class="first-child">The Gilthead sea bream linkage map. The heatmap on the right side provides scale of color coding for the size of SNP clusters.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469745/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469745/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Consensus linkage map</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Heritability estimation and GWAS</h3><p id="p-35">The heritability of surviving days (censored data) was 0.22 (95% highest density interval: 0.11–0.36) and 0.28 (95% highest density interval: 0.17–0.4) using the pedigree and the genomic relationship matrix, respectively, of the 777 disease challenged offspring. No SNPs surpassed the Bonferroni-corrected genome-wide significance threshold (<em>P</em> = 4.1 × 10<sup>−6</sup>; α = 0.05; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035220/-/DC1/TableS6.txt">Table S6</a>). The SNPs with the lowest <em>P</em> values (<em>P</em> &lt; 10<sup>−3</sup>) were located in linkage groups 1–3, 10, 17, 20, and 21 (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/11/3693/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) A Manhattan plot highlighting the association between individual SNPs and surviving days. (B) A QQ plot showing the relationship between the observed and expected &#x2212;log(P) values from the GWAS." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1668668718" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(A) A Manhattan plot highlighting the association between individual SNPs and surviving days. (B) A QQ plot showing the relationship between the observed and expected &#x2212;log(&lt;em&gt;P&lt;/em&gt;) values from the GWAS.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/11/3693/F3.medium.gif" width="440" height="201"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/11/3693/F3.medium.gif" width="440" height="201"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/11/3693/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/11/3693/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469688" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-36" class="first-child">(A) A Manhattan plot highlighting the association between individual SNPs and surviving days. (B) A QQ plot showing the relationship between the observed and expected −log(<em>P</em>) values from the GWAS.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Genomic prediction</h3><p id="p-37">Genomic prediction was tested as a means of obtaining breeding values, and compared to prediction using a pedigree-based approach (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035220/-/DC1/FileS1.zip">File S1</a>). The prediction was conducted using genotype information from 11,239 SNP markers (passed QC filters set for genomic prediction) for the 777 disease challenged animals that were randomly split in training (<em>n</em> = 578) and validation (<em>n</em> = 200) datasets. The application of all the genomic prediction models resulted in higher accuracies than those achieved using PBLUP. Prediction accuracy with PBLUP was 0.3 <em>vs.</em> 0.38–0.46 for the genomic prediction models, with highest accuracy being observed using the BayesA method (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). Prediction accuracy dropped when more sparse SNP marker datasets were used with the last scenario (utilizing SNPs &gt; 5 cM apart), giving similar accuracies with the ones obtained using PBLUP. In the scenario of utilizing only SNPs located &gt;1 cM apart on the linkage map (2614 SNPs), accuracies ranged between 0.3 and 0.36, with the highest accuracy obtained using rrBLUP. For the dataset utilizing only SNPs located &gt;5 cM apart (705 SNPs), accuracies ranged between 0.29 and 0.31, with highest accuracies obtained using rrBLUP and BayesA.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469712/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469712/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Genomic prediction accuracies for surviving days</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-16"><h2 class="">Discussion</h2><p id="p-42">Gilthead sea bream (<em>S. aurata</em>) is a farmed species of paramount importance for Mediterranean aquaculture. While vaccines can offer some protection against pasteurellosis, the low immune competence observed in larval and juvenile stages renders this protection temporary (<a id="xref-ref-1-5" class="xref-bibr" href="#ref-1">Antonello <em>et al.</em> 2009</a>). Breeding for improved genetic resistance offers an additional and complementary tool to combat losses due to this disease. While traditional family-based selective breeding is applied in sea bream, it cannot utilize within-family genetic variation in the trait. Applying genomic information into selective breeding schemes raises the possibility of selecting directly for favorable alleles at major QTL (marker-assisted selection), or incorporating all markers in the prediction of breeding values (genomic selection). As such, genomics-enabled breeding can expedite the rate of genetic gain, and can potentially reduce the need for yearly trait recording. However, to enable these benefits, substantial genomic resources are typically required (<em>e.g.</em>, a high-density SNP genotyping platform), which is currently lacking in sea bream. This is likely to change in the near future as the reference genome sequence and associated genomic tools/data become available. In the meantime, RAD sequencing and similar techniques can readily be applied to generate genome-wide SNP marker datasets, even in the absence of such genomic resources (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Baird <em>et al.</em> 2008</a>).</p><p id="p-43">High-density SNP linkage maps have been constructed for several aquaculture species, and are useful for both QTL positioning and reference genome assembly (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Gonen <em>et al.</em> 2014</a>; <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Palaiokostas <em>et al.</em> 2013a</a>). The most recent linkage map of sea bream consists mainly of microsatellites (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Tsigenopoulos <em>et al.</em> 2014</a>), lacking the necessary resolution for successful implementation of GWAS and genomic prediction. In the current study, we present the first high-density linkage map for this species, consisting of 12,085 SNPs on 24 linkage groups, which is consistent with the karyotype. The genetic map presented here spans 3899 cM, while the map of <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Tsigenopoulos <em>et al.</em> (2014)</a> has a total length of 1769.7 cM, which may reflect the larger number of markers used in the current study. This trend of increase in map distance with increased marker density was observed with previous sea bass (<em>Dicentrarchus labrax</em>) linkage maps (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chistiakov <em>et al.</em> 2005</a>, <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">2008</a>; <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Palaiokostas <em>et al.</em> 2015</a>).</p><p id="p-44">The estimated heritability of resistance was moderate (0.22 and 0.28 for the different models) compared to those previously reported for disease resistance traits in various aquaculture species (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Ødegård <em>et al.</em> 2011</a>). Nonetheless, successful implementation even in the case of low heritability traits in breeding programs is still possible, as demonstrated in livestock (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Heringstad <em>et al.</em> 2003</a>). Also, since heritability of mortality traits are frequency dependent, with maximal values reported at intermediate mortality levels (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Bishop and Woolliams 2014</a>), the low survival rate in the current study may have resulted in an underestimate, and analysis of additional challenge and field data are merited.</p><p id="p-45">The GWAS results pointed to a polygenic or oligogenic genetic architecture for resistance to pasteurellosis, with no genome-wide significant QTL identified, with the lowest <em>P</em> values indicative of putative suggestive QTL on linkage groups 1–3, 10, 17, 20, and 21. Unfortunately, the lack of an integrated linkage map for sea bream does not allow direct comparison of the previously described resistance QTL (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Massault <em>et al.</em> 2010</a>) with the current study. Furthermore no major QTL were identified in the above study, with the largest QTL for surviving days explaining ∼4% of the phenotypic variance. The fact that genomic prediction using all tested models gave similar prediction accuracies, a phenomenon often observed in the study of polygenic traits (<a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Meuwissen <em>et al.</em> 2001</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Kizilkaya <em>et al.</em> 2010</a>), seems to support the hypothesis of polygenic resistance. Nevertheless, since the selected priors will influence the output of the Bayesian models (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Gianola 2013</a>), interpretation of genetic architecture based on these results should be treated with caution. Additionally, the moderate sample size in the current study is likely to limit the statistical power to detect small to medium effect QTL. Further, the low mortality level precludes effective estimation of genetic parameters for survival <em>per se</em>, and the genetic correlation between survival time (days to death) and overall survival is unknown.</p><p id="p-46">The results from the genomic prediction approach were encouraging for practical implementation of selective breeding for genetic resistance in sea bream, with the genomic prediction models outperforming traditional BLUP. The advantage of the genomic-based models was retained also when only SNPs at 1-cM location intervals were used, which may be useful for reducing genotyping costs and improving cost effectiveness. However, due to the limited number of families in the current study, the training and validation sets contain closely related animals, which will increase the accuracy of prediction. Additional testing of genomic prediction at varying marker densities on separate and preferably larger populations would be required to ascertain the appropriate density for commercial application of genomic selection. The benefit of genomic prediction over a pedigree-based approach is likely because of the ability to capture within-family genetic variation. In mass-spawning species such as sea bream, in which family size and structure is difficult to control, this approach is likely to be particularly advantageous. Overall, the current study demonstrates that SNP markers generated via 2b-RAD are effective at capturing the genetic variation in a complex trait in a sea bream breeding population. This approach is likely to be useful in other species with less-developed genomic tools, and provides further evidence that incorporation of genomic selection is likely to result in significant improvement in selection accuracy and genetic gain compared to traditional family selection in aquaculture breeding.</p><div id="sec-17" class="subsection"><h3>Conclusions</h3><p id="p-47">2b-RAD sequencing was applied to investigate genetic resistance to pasteurellosis in gilthead sea bream. The SNP data generated were applied to create the first high-density linkage map for sea bream. Only suggestive QTL were detected, implying that resistance to pasteurellosis has an oligogenic-polygenic architecture for the studied population. Genomic prediction using the 2b-RAD genotype data were effective, with substantial improvement in prediction accuracy over the pedigree-based model. This highlights the utility of genotyping by sequencing for genomic prediction of disease resistance in aquaculture species, and its potential to apply genomic selection in commercial breeding programs.</p></div></div><div class="section" id="sec-18"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-48">This work was partially funded by the research grant “Genomic technologies for the 21st century aquaculture industry: an integrated approach to improve fish growth and health (GenTechAqua),” granted to L.B. by the Fondazione Cassa di Risparmio Padova e Rovigo in the framework of the call “Progetti di Eccellenza 2009–2010”. C.P. and R.H. are supported by funding from the European Union’s Seventh Framework Program (FP7 2007–2013) under grant agreement no. 613611 (FISHBOOST). R.D.H. is supported by Bioscience and Biotechnology Research Council funds BB/J004235/1 and BB/J004324/1.</p><p id="p-49">Author contributions: S.F. and R.F. carried out DNA extractions, 2b-RAD library preparation, and sequence data processing. L.B. conceived the study, contributed to designing the experiments, and analyzed the data. C.P. and R.D.H. designed the quantitative genetic analyses. C.P. conducted the genetic analysis. All authors contributed to drafting the manuscript.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-6"><p id="p-50">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035220/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035220/-/DC1</a>.</p></li><li class="fn" id="fn-7"><p id="p-51"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-07-27"><span class="received-label">Received </span>July 27, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-09-14"><span class="accepted-label">Accepted </span>September 14, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Christos <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3693.1" data-doi="10.1016/j.aquaculture.2009.10.022"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Antonello</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Massault</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franch</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haley</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pellizzari</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Estimates of heritability and genetic correlation for body length and resistance to fish pasteurellosis in the gilthead sea bream (<em>Sparus aurata</em> L.).</span> <abbr class="cit-jnl-abbrev">Aquaculture</abbr> <span class="cit-vol">298</span>: <span class="cit-fpage">29</span>–<span class="cit-lpage">35</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquaculture%26rft.volume%253D298%26rft.spage%253D29%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.aquaculture.2009.10.022%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.aquaculture.2009.10.022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3693.2" data-doi="10.1371/journal.pone.0003376"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baird</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Etter</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atwood</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Currey</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shiver</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Rapid SNP discovery and genetic mapping using sequenced RAD markers.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e3376</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBaird%26rft.auinit1%253DN.%2BA.%26rft.volume%253D3%26rft.issue%253D10%26rft.spage%253De3376%26rft.epage%253De3376%26rft.atitle%253DRapid%2BSNP%2Bdiscovery%2Band%2Bgenetic%2Bmapping%2Busing%2Bsequenced%2BRAD%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0003376%26rft_id%253Dinfo%253Apmid%252F18852878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0003376&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18852878&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3693.3" data-doi="10.1016/j.livsci.2014.04.034"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bishop</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woolliams</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomics and disease resistance studies in livestock.</span> <abbr class="cit-jnl-abbrev">Livest. Sci.</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">190</span>–<span class="cit-lpage">198</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLivest.%2BSci.%26rft.volume%253D166%26rft.spage%253D190%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.livsci.2014.04.034%26rft_id%253Dinfo%253Apmid%252F26339300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.livsci.2014.04.034&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26339300&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3693.4" data-doi="10.1534/g3.114.014621"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">LaPatra</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Overturf</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Towner</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Narum</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Association mapping of disease resistance traits in rainbow trout using restriction site associated DNA sequencing.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">2473</span>–<span class="cit-lpage">2481</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.114.014621%26rft_id%253Dinfo%253Apmid%252F25354781%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjQvMTIvMjQ3MyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzY5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.11.3693.5" data-doi="10.1534/g3.111.000240"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amores</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cresko</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Postlethwait</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Stacks: Building and Genotyping Loci <em>De Novo</em> from Short-Read Sequences.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda, Md.)</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">171</span>–<span class="cit-lpage">82</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%252C%2BMd.%2529%26rft.volume%253D1%26rft.spage%253D171%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.111.000240%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.111.000240&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3693.6" data-doi="10.1534/genetics.104.039719"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chistiakov</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hellemans</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haley</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Law</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsigenopoulos</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A microsatellite linkage map of the European sea bass <em>Dicentrarchus labrax</em> L.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">170</span>: <span class="cit-fpage">1821</span>–<span class="cit-lpage">1826</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.039719%26rft_id%253Dinfo%253Apmid%252F15937133%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3MC80LzE4MjEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzExLzM2OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3693.7" data-doi="10.1111/j.1365-2052.2008.01786.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chistiakov</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsigenopoulos</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lagnel</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">Y.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hellemans</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A combined AFLP and microsatellite linkage map and pilot comparative genomic analysis of European sea bass <em>Dicentrarchus labrax</em> L.</span> <abbr class="cit-jnl-abbrev">Anim. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">623</span>–<span class="cit-lpage">634</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnimal%2Bgenetics%26rft.stitle%253DAnim%2BGenet%26rft.aulast%253DChistiakov%26rft.auinit1%253DD.%2BA.%26rft.volume%253D39%26rft.issue%253D6%26rft.spage%253D623%26rft.epage%253D634%26rft.atitle%253DA%2Bcombined%2BAFLP%2Band%2Bmicrosatellite%2Blinkage%2Bmap%2Band%2Bpilot%2Bcomparative%2Bgenomic%2Banalysis%2Bof%2BEuropean%2Bsea%2Bbass%2BDicentrarchus%2Blabrax%2BL.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2052.2008.01786.x%26rft_id%253Dinfo%253Apmid%252F18828863%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2052.2008.01786.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18828863&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3693.8" data-doi="10.1038/nrg3012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davey</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Etter</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boone</span>  <span class="cit-name-given-names">J. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide genetic marker discovery and genotyping using next-generation sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">499</span>–<span class="cit-lpage">510</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DDavey%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D12%26rft.issue%253D7%26rft.spage%253D499%26rft.epage%253D510%26rft.atitle%253DGenome-wide%2Bgenetic%2Bmarker%2Bdiscovery%2Band%2Bgenotyping%2Busing%2Bnext-generation%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3012%26rft_id%253Dinfo%253Apmid%252F21681211%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21681211&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3693.9" data-doi="10.1534/genetics.112.143313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Whole-genome regression and prediction methods applied to plant and animal breeding.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">327</span>–<span class="cit-lpage">345</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.143313%26rft_id%253Dinfo%253Apmid%252F22745228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvMzI3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNjkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3693.10" data-doi="10.1038/srep19244"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dou</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiao</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Evaluation of the 2b-RAD method for genomic selection in scallop breeding.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">19244</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D6%26rft.spage%253D19244%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep19244%26rft_id%253Dinfo%253Apmid%252F26754638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep19244&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26754638&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3693.11" data-doi="10.3835/plantgenome2011.08.0024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Ridge regression and other kernels for genomic selection with R package rrBLUP.</span> <abbr class="cit-jnl-abbrev">The Plant Genome Journal</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">250</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BGenome%2BJournal%26rft.volume%253D4%26rft.spage%253D250%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2011.08.0024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2011.08.0024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-web" id="cit-6.11.3693.12"><div class="cit-metadata unstructured">Federation of European Aquaculture Producers, 2014 Annual Report. Available at: <a href="http://www.feap.info/default.asp?SHORTCUT=582">http://www.feap.info/default.asp?SHORTCUT=582</a>. Accessed: January 9, 2016.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3693.13" data-doi="10.1534/genetics.113.151753"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Priors in whole-genome regression: the Bayesian alphabet returns.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">573</span>–<span class="cit-lpage">596</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.151753%26rft_id%253Dinfo%253Apmid%252F23636739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzMvNTczIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNjkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3693.14" data-doi="10.1038/nrg2575"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Mapping genes for complex traits in domestic animals and their use in breeding programmes.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">381</span>–<span class="cit-lpage">391</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DGoddard%26rft.auinit1%253DM.%2BE.%26rft.volume%253D10%26rft.issue%253D6%26rft.spage%253D381%26rft.epage%253D391%26rft.atitle%253DMapping%2Bgenes%2Bfor%2Bcomplex%2Btraits%2Bin%2Bdomestic%2Banimals%2Band%2Btheir%2Buse%2Bin%2Bbreeding%2Bprogrammes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2575%26rft_id%253Dinfo%253Apmid%252F19448663%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2575&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19448663&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266095200013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3693.15" data-doi="10.1186/1471-2164-15-166"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gonen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lowe</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cezard</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gharbi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bishop</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Linkage maps of the Atlantic salmon (<em>Salmo salar</em>)genome derived from RAD sequencing.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">166</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D166%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-166%26rft_id%253Dinfo%253Apmid%252F24571138%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-166&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24571138&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3693.16" data-doi="10.1186/1471-2105-12-186"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kizilkaya</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Extension of the bayesian alphabet for genomic selection.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">186</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D12%26rft.spage%253D186%26rft.epage%253D186%26rft.atitle%253DExtension%2Bof%2Bthe%2Bbayesian%2Balphabet%2Bfor%2Bgenomic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-186%26rft_id%253Dinfo%253Apmid%252F21605355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-186&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21605355&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3693.17" data-doi="10.2307/2529430"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">Best linear unbiased estimation and prediction under a selection model.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">423</span>–<span class="cit-lpage">447</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.stitle%253DBiometrics%26rft.aulast%253DHenderson%26rft.auinit1%253DC.%2BR.%26rft.volume%253D31%26rft.issue%253D2%26rft.spage%253D423%26rft.epage%253D447%26rft.atitle%253DBest%2Blinear%2Bunbiased%2Bestimation%2Band%2Bprediction%2Bunder%2Ba%2Bselection%2Bmodel.%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2529430%26rft_id%253Dinfo%253Apmid%252F1174616%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2529430&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1174616&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1975AF09900010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3693.18" data-doi="10.3168/jds.S0022-0302(03)73897-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heringstad</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klemetsdal</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steine</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Selection responses for clinical mastitis and protein yield in two Norwegian dairy cattle selection experiments.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">2990</span>–<span class="cit-lpage">2999</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DHeringstad%26rft.auinit1%253DB.%26rft.volume%253D86%26rft.issue%253D9%26rft.spage%253D2990%26rft.epage%253D2999%26rft.atitle%253DSelection%2BResponses%2Bfor%2BClinical%2BMastitis%2Band%2BProtein%2BYield%2Bin%2BTwo%2BNorwegian%2BDairy%2BCattle%2BSelection%2BExperiments%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.S0022-0302%252803%252973897-1%26rft_id%253Dinfo%253Apmid%252F14507036%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.S0022-0302(03)73897-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14507036&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185137900029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3693.19" data-doi="10.1534/genetics.107.082974"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Houston</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haley</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamilton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guy</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tinch</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Major quantitative trait loci affect resistance to infectious pancreatic necrosis in Atlantic salmon (<em>Salmo salar</em>).</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">1109</span>–<span class="cit-lpage">1115</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.082974%26rft_id%253Dinfo%253Apmid%252F18245341%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OC8yLzExMDkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzExLzM2OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3693.20" data-doi="10.1038/hdy.2009.171"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Houston</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haley</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamilton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guy</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mota-Velasco</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The susceptibility of Atlantic salmon fry to freshwater infectious pancreatic necrosis is largely explained by a major QTL.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">318</span>–<span class="cit-lpage">327</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DHouston%26rft.auinit1%253DR.%2BD.%26rft.volume%253D105%26rft.issue%253D3%26rft.spage%253D318%26rft.epage%253D327%26rft.atitle%253DThe%2Bsusceptibility%2Bof%2BAtlantic%2Bsalmon%2Bfry%2Bto%2Bfreshwater%2Binfectious%2Bpancreatic%2Bnecrosis%2Bis%2Blargely%2Bexplained%2Bby%2Ba%2Bmajor%2BQTL.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2009.171%26rft_id%253Dinfo%253Apmid%252F19935825%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2009.171&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19935825&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3693.21" data-doi="10.1186/1471-2164-15-90"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Houston</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taggart</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cézard</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bekaert</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lowe</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Development and validation of a high density SNP genotyping array for Atlantic salmon (<em>Salmo salar</em>).</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">90</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D90%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-90%26rft_id%253Dinfo%253Apmid%252F24524230%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-90&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24524230&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3693.22" data-doi="10.2527/jas.2009-2064"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kizilkaya</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genomic prediction of simulated multibreed and purebred performance using observed fifty thousand single nucleotide polymorphism genotypes.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">544</span>–<span class="cit-lpage">551</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas.2009-2064%26rft_id%253Dinfo%253Apmid%252F19820059%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas.2009-2064&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19820059&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274040700016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3693.23" data-doi="10.1016/S0044-8486(00)00547-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Magarinos</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Couso</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noya</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merino</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toranzo</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Effect of temperature on the development of pasteurellosis in carrier gilthead seabream (<em>Sparus aurata</em>).</span> <abbr class="cit-jnl-abbrev">Aquaculture</abbr> <span class="cit-vol">195</span>: <span class="cit-fpage">17</span>–<span class="cit-lpage">21</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquaculture%26rft.volume%253D195%26rft.spage%253D17%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0044-8486%252800%252900547-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0044-8486(00)00547-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3693.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Massault</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hellemans</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Louro</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Batargias</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Houdt</span>  <span class="cit-name-given-names">J. K. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">QTL for body weight, morphometric traits and stress response in European sea bass <em>Dicentrarchus labrax</em>.</span> <abbr class="cit-jnl-abbrev">Anim. Genet.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">337</span>–<span class="cit-lpage">345</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnimal%2Bgenetics%26rft.stitle%253DAnim%2BGenet%26rft.aulast%253DMassault%26rft.auinit1%253DC.%26rft.volume%253D41%26rft.issue%253D4%26rft.spage%253D337%26rft.epage%253D345%26rft.atitle%253DQTL%2Bfor%2Bbody%2Bweight%252C%2Bmorphometric%2Btraits%2Band%2Bstress%2Bresponse%2Bin%2BEuropean%2Bsea%2Bbass%2BDicentrarchus%2Blabrax.%26rft_id%253Dinfo%253Apmid%252F20028379%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=20028379&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3693.25" data-doi="10.1111/j.1365-2052.2010.02110.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Massault</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franch</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haley</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Koning</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bovenhuis</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Quantitative trait loci for resistance to fish pasteurellosis in gilthead sea bream (<em>Sparus aurata</em>).</span> <abbr class="cit-jnl-abbrev">Anim. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">191</span>–<span class="cit-lpage">203</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnim.%2BGenet.%26rft.volume%253D42%26rft.spage%253D191%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2052.2010.02110.x%26rft_id%253Dinfo%253Apmid%252F20946317%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2052.2010.02110.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20946317&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3693.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzExLzM2OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3693.27" data-doi="10.1186/1471-2164-10-368"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baranski</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sonesson</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kjøglum</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Confirmation and fine-mapping of a major QTL for resistance to infectious pancreatic necrosis in Atlantic salmon (<em>Salmo salar</em>): population-level associations between markers and trait.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">368</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DMoen%26rft.auinit1%253DT.%26rft.volume%253D10%26rft.spage%253D368%26rft.epage%253D368%26rft.atitle%253DConfirmation%2Band%2Bfine-mapping%2Bof%2Ba%2Bmajor%2BQTL%2Bfor%2Bresistance%2Bto%2Binfectious%2Bpancreatic%2Bnecrosis%2Bin%2BAtlantic%2Bsalmon%2B%2528Salmo%2Bsalar%2529%253A%2Bpopulation-level%2Bassociations%2Bbetween%2Bmarkers%2Band%2Btrait.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-10-368%26rft_id%253Dinfo%253Apmid%252F19664221%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-10-368&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19664221&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3693.28" data-doi="10.1016/0044-8486(94)00353-P"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Noya</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magariños</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lamas</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Interactions between peritoneal exudate cells (PECs) of gilthead seabream (<em>Sparus aurata</em>) and Pasteurella piscicida. A morphological study.</span> <abbr class="cit-jnl-abbrev">Aquaculture</abbr> <span class="cit-vol">131</span>: <span class="cit-fpage">11</span>–<span class="cit-lpage">21</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquaculture%26rft.volume%253D131%26rft.spage%253D11%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0044-8486%252894%252900353-P%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0044-8486(94)00353-P&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1995QN19600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3693.29" data-doi="10.1111/j.1365-2109.2010.02669.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ødegård</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baranski</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gjerde</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gjedrem</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Methodology for genetic evaluation of disease resistance in aquaculture species: challenges and future prospects.</span> <abbr class="cit-jnl-abbrev">Aquacult. Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">103</span>–<span class="cit-lpage">114</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquacult.%2BRes.%26rft.volume%253D42%26rft.spage%253D103%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2109.2010.02669.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2109.2010.02669.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3693.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Odegård</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Santi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korsvoll</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kjøglum</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic prediction in an admixed population of Atlantic salmon (<em>Salmo salar</em>).</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">402</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D5%26rft.spage%253D402%26rft_id%253Dinfo%253Apmid%252F25484890%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=25484890&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3693.31" data-doi="10.1186/1471-2164-14-566"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Palaiokostas</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bekaert</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davie</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cowan</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oral</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span><span class="cit-comment">a</span> <span class="cit-article-title">Mapping the sex determination locus in the Atlantic halibut (<em>Hippoglossus hippoglossus</em>) using RAD sequencing.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">566</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D14%26rft.spage%253D566%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-566%26rft_id%253Dinfo%253Apmid%252F23957753%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-566&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23957753&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3693.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Palaiokostas</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bekaert</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khan</span>  <span class="cit-name-given-names">M. G. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taggart</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gharbi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span><span class="cit-comment">b</span> <span class="cit-article-title">Mapping and validation of the major sex-determining region in Nile Tilapia (<em>Oreochromis niloticus L</em>.) using RAD sequencing.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253D8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3693.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Palaiokostas</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bekaert</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taggart</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gharbi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McAndrew</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A new SNP-based vision of the genetics of sex determination in European sea bass (<em>Dicentrarchus labrax</em>). Genetics, selection, evolution</span>. <abbr class="cit-jnl-abbrev">GSE</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">68</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGSE%26rft.volume%253D47%26rft.spage%253D68%26rft_id%253Dinfo%253Apmid%252F26337592%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=26337592&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3693.34" data-doi="10.1111/1755-0998.12495"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pauletto</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carraro</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Babbucci</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lucchini</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bargelloni</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Extending RAD tag analysis to microbial ecology: a comparison between multilocus sequence typing and 2b-RAD to investigate Listeria monocytogenes genetic structure.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">823</span>–<span class="cit-lpage">835</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D16%26rft.spage%253D823%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12495%26rft_id%253Dinfo%253Apmid%252F26613186%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12495&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26613186&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3693.35" data-doi="10.1016/j.margen.2015.12.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pecoraro</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Babbucci</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villamor</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franch</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Papetti</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Methodological assessment of 2b-RAD genotyping technique for population structure inferences in yellowfin tuna (<em>Thunnus albacares</em>).</span> <abbr class="cit-jnl-abbrev">Mar. Genomics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">43</span>–<span class="cit-lpage">48</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMar.%2BGenomics%26rft.volume%253D25%26rft.spage%253D43%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.margen.2015.12.002%26rft_id%253Dinfo%253Apmid%252F26711352%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.margen.2015.12.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26711352&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3693.36" data-doi="10.1534/genetics.114.164442"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Los Campos</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-wide regression and prediction with the BGLR statistical package.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">483</span>–<span class="cit-lpage">495</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.164442%26rft_id%253Dinfo%253Apmid%252F25009151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzIvNDgzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMS8zNjkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3693.37" data-doi="10.1111/mec.12965"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Puritz</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matz</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toonen</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weber</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bolnick</span>  <span class="cit-name-given-names">D. I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Demystifying the RAD fad.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">5937</span>–<span class="cit-lpage">5942</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D23%26rft.spage%253D5937%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12965%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12965&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000346771900003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3693.38" data-doi="10.1093/bioinformatics/btt563"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rastas</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paulin</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanski</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lehtonen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auvinen</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Lep-MAP: fast and accurate linkage map construction for large SNP datasets.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">3128</span>–<span class="cit-lpage">3134</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt563%26rft_id%253Dinfo%253Apmid%252F24078685%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjkvMjQvMzEyOCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzY5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3693.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Boa : an R package for MCMC output convergence.</span> <abbr class="cit-jnl-abbrev">J. Stat. Softw.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">37</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BStat.%2BSoftw.%26rft.volume%253D21%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-other" id="cit-6.11.3693.40"><div class="cit-metadata unstructured">Tsai, H.-Y., A. Hamilton, A. E. Tinch, D. R. Guy, J. E. Bron <em>et al.</em>, 2016 Genomic prediction of host resistance to sea lice in farmed Atlantic salmon populations. Genet. Sel. Evol. 48: 47</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3693.41" data-doi="10.1186/s12864-015-2117-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsai</span>  <span class="cit-name-given-names">H.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamilton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tinch</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guy</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gharbi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genome wide association and genomic prediction for growth traits in juvenile farmed Atlantic salmon using a high density SNP array.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">969</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D969%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-2117-9%26rft_id%253Dinfo%253Apmid%252F26582102%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-2117-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26582102&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3693.42" data-doi="10.1016/j.margen.2014.09.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsigenopoulos</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Louro</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chatziplis</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lagnel</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vogiatzi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Second generation genetic linkage map for the gilthead sea bream <em>Sparus aurata</em> L.</span> <abbr class="cit-jnl-abbrev">Mar. Genomics</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">82</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMar.%2BGenomics%26rft.volume%253D18%26rft.spage%253D77%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.margen.2014.09.008%26rft_id%253Dinfo%253Apmid%252F25307204%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.margen.2014.09.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25307204&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-other" id="cit-6.11.3693.43"><div class="cit-metadata unstructured">Vallejo, R. L., T. D. Leeds, B. O. Fragomeni, G. Gao, A. G. Hernan- dez <em>et al.</em>, 2016 Evaluation of Genome-Enabled Selection for Bacterial Cold Water Disease Resistance Using Progeny Performance Data in Rainbow Trout: Insights on Genotyping Methods and Genomic Prediction Models. Front. Genet. 7: 96.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3693.44" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3693.45" data-doi="10.1038/nmeth.2023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McKay</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matz</span>  <span class="cit-name-given-names">M. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">2b-RAD: a simple and flexible method for genome-wide genotyping.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">808</span>–<span class="cit-lpage">810</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DWang%26rft.auinit1%253DS.%26rft.volume%253D9%26rft.issue%253D8%26rft.spage%253D808%26rft.epage%253D810%26rft.atitle%253D2b-RAD%253A%2Ba%2Bsimple%2Band%2Bflexible%2Bmethod%2Bfor%2Bgenome-wide%2Bgenotyping.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2023%26rft_id%253Dinfo%253Apmid%252F22609625%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22609625&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000307015700021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3693.46" data-doi="10.1093/bioinformatics/bts335"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wimmer</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albrecht</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auinger</span>  <span class="cit-name-given-names">H.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schön</span>  <span class="cit-name-given-names">C.-C.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Synbreed: a framework for the analysis of genomic prediction data using R.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2086</span>–<span class="cit-lpage">2087</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts335%26rft_id%253Dinfo%253Apmid%252F22689388%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjgvMTUvMjA4NiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTEvMzY5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3693.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yáñez</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Houston</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Newman</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genetics and genomics of disease resistance in salmonid species.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">415</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D5%26rft.spage%253D415%26rft_id%253Dinfo%253Apmid%252F25505486%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=25505486&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-6.11.3693.48" data-doi="10.1038/ng1702"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pressoir</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Briggs</span>  <span class="cit-name-given-names">W. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vroh Bi</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamasaki</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A unified mixed-model method for association mapping that accounts for multiple levels of relatedness.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">203</span>–<span class="cit-lpage">208</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYu%26rft.auinit1%253DJ.%26rft.volume%253D38%26rft.issue%253D2%26rft.spage%253D203%26rft.epage%253D208%26rft.atitle%253DA%2Bunified%2Bmixed-model%2Bmethod%2Bfor%2Bassociation%2Bmapping%2Bthat%2Baccounts%2Bfor%2Bmultiple%2Blevels%2Bof%2Brelatedness.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1702%26rft_id%253Dinfo%253Apmid%252F16380716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1702&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16380716&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F11%2F3693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234953200015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/11/3693.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/11/3685" title="Non-Mendelian Dominant Maternal Effects Caused by CRISPR/Cas9 Transgenic Components in Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/11/3701" title="Extensive Epigenetic Changes Accompany Terminal Differentiation of Mouse Hepatocytes After Birth" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/11"><span>Volume 6 Issue 11, November 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/11.cover-source.jpg?itok=hTYn4PRy" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/11/3693" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-PDe2taWjUjDNyz_ZycI8V7cGWdKyGd1F-TV5FZWLnDc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/11/3693" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/11/3693" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/11/3693" class="active" data-icon-position="" data-hide-link-title="0">Genomic Prediction of Resistance to Pasteurellosis in Gilthead Sea Bream (Sparus aurata) Using 2b-RAD Sequencing</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460461" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-tZcT5SwL2IpkiRIYf-9X57ko785pu1v_LHDzven98DQ" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/11/3693.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/11/3693" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-ykKyrMS3n5AHx8xMnllZkw_7xBb90GPUXDA4Jon5TU0" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/11/3693" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/11/3693.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460461" id="node460461--2" data-pisa="ggg;6/11/3693" data-pisa-master="ggg;g3.116.035220" data-apath="/ggg/6/11/3693.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Prediction of Resistance to Pasteurellosis in Gilthead Sea Bream (<em>Sparus aurata</em>) Using 2b-RAD Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-4480-4612" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Christos Palaiokostas</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-5928-9361" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Serena Ferraresso</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-1333-0733" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Rafaella Franch</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-1805-0762" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ross D. Houston</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-4351-2826" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Luca Bargelloni</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3693-3700;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.035220 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460461--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Christos Palaiokostas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Midlothian, EH25 9RG Scotland, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christos%2BPalaiokostas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Palaiokostas%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChristos%2BPalaiokostas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4480-4612" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Christos Palaiokostas</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:christos.palaiokostas@roslin.ed.ac.uk" class="" data-icon-position="" data-hide-link-title="0">christos.palaiokostas@roslin.ed.ac.uk</a>
<a href="mailto:luca.bargelloni@unipd.it" class="" data-icon-position="" data-hide-link-title="0">luca.bargelloni@unipd.it</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Serena Ferraresso </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Comparative Biomedicine and Food Science, University of Padova, 35020 Legnaro, Italy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Serena%2BFerraresso%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ferraresso%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASerena%2BFerraresso%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5928-9361" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Serena Ferraresso</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Rafaella Franch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Comparative Biomedicine and Food Science, University of Padova, 35020 Legnaro, Italy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rafaella%2BFranch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Franch%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARafaella%2BFranch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1333-0733" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Rafaella Franch</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Ross D. Houston </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Midlothian, EH25 9RG Scotland, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ross%2BD.%2BHouston%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Houston%20RD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARoss%2BD.%2BHouston%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1805-0762" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ross D. Houston</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Luca Bargelloni </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Comparative Biomedicine and Food Science, University of Padova, 35020 Legnaro, Italy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Luca%2BBargelloni%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bargelloni%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALuca%2BBargelloni%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-4351-2826" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Luca Bargelloni</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:christos.palaiokostas@roslin.ed.ac.uk" class="" data-icon-position="" data-hide-link-title="0">christos.palaiokostas@roslin.ed.ac.uk</a>
<a href="mailto:luca.bargelloni@unipd.it" class="" data-icon-position="" data-hide-link-title="0">luca.bargelloni@unipd.it</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/11/3693" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/11/3693&amp;title=Genomic%20Prediction%20of%20Resistance%20to%20Pasteurellosis%20in%20Gilthead%20Sea%20Bream%20%28Sparus%20aurata%29%20Using%202b-RAD%20Sequencing" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/11/3693&amp;title=Genomic%20Prediction%20of%20Resistance%20to%20Pasteurellosis%20in%20Gilthead%20Sea%20Bream%20%28Sparus%20aurata%29%20Using%202b-RAD%20Sequencing" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/11/3693&amp;title=Genomic%20Prediction%20of%20Resistance%20to%20Pasteurellosis%20in%20Gilthead%20Sea%20Bream%20%28Sparus%20aurata%29%20Using%202b-RAD%20Sequencing" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/11/3693&amp;text=Genomic%20Prediction%20of%20Resistance%20to%20Pasteurellosis%20in%20Gilthead%20Sea%20Bream%20%28Sparus%20aurata%29%20Using%202b-RAD%20Sequencing" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/11/3693&amp;title=Genomic%20Prediction%20of%20Resistance%20to%20Pasteurellosis%20in%20Gilthead%20Sea%20Bream%20%28Sparus%20aurata%29%20Using%202b-RAD%20Sequencing" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/11/3693&amp;t=Genomic%20Prediction%20of%20Resistance%20to%20Pasteurellosis%20in%20Gilthead%20Sea%20Bream%20%28Sparus%20aurata%29%20Using%202b-RAD%20Sequencing" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/11/3693&amp;title=Genomic%20Prediction%20of%20Resistance%20to%20Pasteurellosis%20in%20Gilthead%20Sea%20Bream%20%28Sparus%20aurata%29%20Using%202b-RAD%20Sequencing" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/11/3693&amp;title=Genomic%20Prediction%20of%20Resistance%20to%20Pasteurellosis%20in%20Gilthead%20Sea%20Bream%20%28Sparus%20aurata%29%20Using%202b-RAD%20Sequencing" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460461/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460461" id="node460461--3" data-pisa="ggg;6/11/3693" data-pisa-master="ggg;g3.116.035220" data-apath="/ggg/6/11/3693.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Prediction of Resistance to Pasteurellosis in Gilthead Sea Bream (<em>Sparus aurata</em>) Using 2b-RAD Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-4480-4612" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Christos Palaiokostas</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-5928-9361" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Serena Ferraresso</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-1333-0733" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Rafaella Franch</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-1805-0762" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ross D. Houston</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-4351-2826" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Luca Bargelloni</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3693-3700;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.035220 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460461--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Christos Palaiokostas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Midlothian, EH25 9RG Scotland, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christos%2BPalaiokostas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Palaiokostas%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChristos%2BPalaiokostas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4480-4612" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Christos Palaiokostas</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:christos.palaiokostas@roslin.ed.ac.uk" class="" data-icon-position="" data-hide-link-title="0">christos.palaiokostas@roslin.ed.ac.uk</a>
<a href="mailto:luca.bargelloni@unipd.it" class="" data-icon-position="" data-hide-link-title="0">luca.bargelloni@unipd.it</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Serena Ferraresso </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Comparative Biomedicine and Food Science, University of Padova, 35020 Legnaro, Italy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Serena%2BFerraresso%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ferraresso%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASerena%2BFerraresso%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5928-9361" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Serena Ferraresso</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Rafaella Franch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Comparative Biomedicine and Food Science, University of Padova, 35020 Legnaro, Italy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rafaella%2BFranch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Franch%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARafaella%2BFranch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1333-0733" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Rafaella Franch</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Ross D. Houston </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Roslin Institute and Royal (Dick) School of Veterinary Studies, University of Edinburgh, Midlothian, EH25 9RG Scotland, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ross%2BD.%2BHouston%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Houston%20RD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARoss%2BD.%2BHouston%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1805-0762" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ross D. Houston</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Luca Bargelloni </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Comparative Biomedicine and Food Science, University of Padova, 35020 Legnaro, Italy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Luca%2BBargelloni%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bargelloni%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALuca%2BBargelloni%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-4351-2826" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Luca Bargelloni</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:christos.palaiokostas@roslin.ed.ac.uk" class="" data-icon-position="" data-hide-link-title="0">christos.palaiokostas@roslin.ed.ac.uk</a>
<a href="mailto:luca.bargelloni@unipd.it" class="" data-icon-position="" data-hide-link-title="0">luca.bargelloni@unipd.it</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3693&amp;rft.epage=3700&amp;rft.atitle=Genomic%20Prediction%20of%20Resistance%20to%20Pasteurellosis%20in%20Gilthead%20Sea%20Bream%20%28Sparus%20aurata%29%20Using%202b-RAD%20Sequencing&amp;rft.volume=6&amp;rft.issue=11&amp;rft.date=2016-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Palaiokostas%2C+Christos&amp;rft.au=Ferraresso%2C+Serena&amp;rft.au=Franch%2C+Rafaella&amp;rft.au=Houston%2C+Ross+D.&amp;rft.au=Bargelloni%2C+Luca"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460461/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460461/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460461/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460461/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460461/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460461/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460461/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460461/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460461/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460461/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460461/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460461/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/11/3693" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/11/3693.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/11/3693.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/11/3693.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/11/3693.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
