<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/2/413" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/2/413.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/2/413.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/2/413.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/2/413.atom" />
<meta name="HW.pisa" content="ggg;7/2/413" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Forward Genetics by Sequencing EMS Variation-Induced Inbred Lines" />
<meta name="DC.Identifier" content="10.1534/g3.116.029660" />
<meta name="DC.Date" content="2017-02-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Addo-Quaye et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="In order to leverage novel sequencing techniques for cloning genes in eukaryotic organisms with complex genomes, the false positive rate of variant discovery must be controlled for by experimental design and informatics. We sequenced five lines from three pedigrees of ethyl methanesulfonate (EMS)-mutagenized Sorghum bicolor , including a pedigree segregating a recessive dwarf mutant. Comparing the sequences of the lines, we were able to identify and eliminate error-prone positions. One genomic region contained EMS mutant alleles in dwarfs that were homozygous reference sequences in wild-type siblings and heterozygous in segregating families. This region contained a single nonsynonymous change that cosegregated with dwarfism in a validation population and caused a premature stop codon in the Sorghum ortholog encoding the gibberellic acid (GA) biosynthetic enzyme ent -kaurene oxidase. Application of exogenous GA rescued the mutant phenotype. Our method for mapping did not require outcrossing and introduced no segregation variance. This enables work when line crossing is complicated by life history, permitting gene discovery outside of genetic models. This inverts the historical approach of first using recombination to define a locus and then sequencing genes. Our formally identical approach first sequences all the genes and then seeks cosegregation with the trait. Mutagenized lines lacking obvious phenotypic alterations are available for an extension of this approach: mapping with a known marker set in a line that is phenotypically identical to starting material for EMS mutant generation." />
<meta name="DC.Contributor" content="Charles Addo-Quaye" />
<meta name="DC.Contributor" content="Elizabeth Buescher" />
<meta name="DC.Contributor" content="Norman Best" />
<meta name="DC.Contributor" content="Vijay Chaikam" />
<meta name="DC.Contributor" content="Ivan Baxter" />
<meta name="DC.Contributor" content="Brian P. Dilkes" />
<meta name="article:published_time" content="2017-02-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Forward Genetics by Sequencing EMS Variation-Induced Inbred Lines" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;In order to leverage novel sequencing techniques for cloning genes in eukaryotic organisms with complex genomes, the false positive rate of variant discovery must be controlled for by experimental design and informatics. We sequenced five lines from three pedigrees of ethyl methanesulfonate (EMS)-mutagenized &lt;i&gt;Sorghum bicolor&lt;/i&gt;, including a pedigree segregating a recessive dwarf mutant. Comparing the sequences of the lines, we were able to identify and eliminate error-prone positions. One genomic region contained EMS mutant alleles in dwarfs that were homozygous reference sequences in wild-type siblings and heterozygous in segregating families. This region contained a single nonsynonymous change that cosegregated with dwarfism in a validation population and caused a premature stop codon in the &lt;i&gt;Sorghum&lt;/i&gt; ortholog encoding the gibberellic acid (GA) biosynthetic enzyme &lt;i&gt;ent&lt;/i&gt;-kaurene oxidase. Application of exogenous GA rescued the mutant phenotype. Our method for mapping did not require outcrossing and introduced no segregation variance. This enables work when line crossing is complicated by life history, permitting gene discovery outside of genetic models. This inverts the historical approach of first using recombination to define a locus and then sequencing genes. Our formally identical approach first sequences all the genes and then seeks cosegregation with the trait. Mutagenized lines lacking obvious phenotypic alterations are available for an extension of this approach: mapping with a known marker set in a line that is phenotypically identical to starting material for EMS mutant generation.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/02/01" />
<meta name="citation_mjid" content="ggg;7/2/413" />
<meta name="citation_id" content="7/2/413" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/2/413" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/2/413.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/2/413.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/2/413.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.029660" />
<meta name="citation_pmid" content="28040779" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="2" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="413" />
<meta name="citation_lastpage" content="425" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Charles Addo-Quaye" />
<meta name="citation_author_institution" content="Department of Biochemistry, Purdue University, West Lafayette, Indiana 47907" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0598-5980" />
<meta name="citation_author" content="Elizabeth Buescher" />
<meta name="citation_author_institution" content="Department of Biochemistry, Purdue University, West Lafayette, Indiana 47907" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1540-8928" />
<meta name="citation_author" content="Norman Best" />
<meta name="citation_author_institution" content="Department of Biochemistry, Purdue University, West Lafayette, Indiana 47907" />
<meta name="citation_author" content="Vijay Chaikam" />
<meta name="citation_author_institution" content="Department of Biochemistry, Purdue University, West Lafayette, Indiana 47907" />
<meta name="citation_author" content="Ivan Baxter" />
<meta name="citation_author_institution" content="United States Department of Agriculture-Agricultural Research Service, Donald Danforth Plant Science Center, St. Louis, Missouri 63132" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6680-1722" />
<meta name="citation_author" content="Brian P. Dilkes" />
<meta name="citation_author_institution" content="Department of Biochemistry, Purdue University, West Lafayette, Indiana 47907" />
<meta name="citation_author_email" content="bdilkes@purdue.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2799-954X" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=A. Abe;citation_author=S. Kosugi;citation_author=K. Yoshida;citation_author=S. Natsume;citation_author=H. Takagi;citation_title=Genome sequencing reveals agronomically important loci in rice using MutMap.;citation_pages=174-178;citation_volume=30;citation_year=2012;citation_issue=2;citation_pmid=22267009;citation_doi=10.1038/nbt.2095" />
<meta name="citation_reference" content="citation_journal_title=Biophoton. Int.;citation_author=M. D. Abramoff;citation_author=P. J. Magelhaes;citation_author=S. J. Ram;citation_title=Image processing with ImageJ.;citation_pages=36-42;citation_volume=11;citation_year=2004;citation_issue=7" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=K. Ashelford;citation_author=M. E. Eriksson;citation_author=C. M. Allen;citation_author=R. D’Amore;citation_author=M. Johansson;citation_title=Full genome re-sequencing reveals a novel circadian clock mutation in Arabidopsis.;citation_pages=R28-R28;citation_volume=12;citation_year=2011;citation_issue=3;citation_pmid=21429190;citation_doi=10.1186/gb-2011-12-3-r28" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=R. S. Austin;citation_author=D. Vidaurre;citation_author=G. Stamatiou;citation_author=R. Breit;citation_author=N. J. Provart;citation_title=Next-generation mapping of Arabidopsis genes.;citation_pages=715-725;citation_volume=67;citation_year=2011;citation_issue=4;citation_pmid=21518053;citation_doi=10.1111/j.1365-313X.2011.04619.x" />
<meta name="citation_reference" content="citation_journal_abbrev=Plant Biotechnol J;citation_author=C. K. Blomstedt;citation_author=R. M. Gleadow;citation_author=N. O’Donnell;citation_author=P. Naur;citation_author=K. Jensen;citation_title=A combined biochemical screen and TILLING approach identifies mutations in Sorghum bicolor L. Moench resulting in acyanogenic forage production.;citation_pages=54-66;citation_volume=10;citation_year=2012;citation_issue=1;citation_pmid=21880107;citation_doi=10.1111/j.1467-7652.2011.00646.x" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=K. R. Bull;citation_author=A. J. Rimmer;citation_author=O. M. Siggs;citation_author=L. A. Miosge;citation_author=C. M. Roots;citation_title=Unlocking the bottleneck in forward genetics using whole-genome sequencing and identity by descent to isolate causative mutations.;citation_pages=e1003219;citation_volume=9;citation_year=2013;citation_pmid=23382690;citation_doi=10.1371/journal.pgen.1003219" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=A. M. Casa;citation_author=G. Pressoir;citation_author=P. J. Brown;citation_author=S. E. Mitchell;citation_author=W. L. Rooney;citation_title=Community Resources and Strategies for Association Mapping in Sorghum;citation_pages=30-40;citation_volume=48;citation_year=2008;citation_issue=1;citation_doi=10.2135/cropsci2007.02.0080" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Y. Cheng;citation_author=Y.-Y. Teo;citation_author=R. T.-H. Ong;citation_title=Assessing single nucleotide variant detection and genotype calling on whole-genome sequenced individuals.;citation_volume=30;citation_year=2014;citation_issue=12;citation_pmid=24558117;citation_doi=10.1093/bioinformatics/btu067" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=J. Cheung;citation_author=X. Estivill;citation_author=R. Khaja;citation_author=J. R. MacDonald;citation_author=K. Lau;citation_title=Genome-wide detection of segmental duplications and potential assembly errors in the human genome sequence.;citation_pages=R25-R25;citation_volume=4;citation_year=2003;citation_issue=4;citation_pmid=12702206;citation_doi=10.1186/gb-2003-4-4-r25" />
<meta name="citation_reference" content="citation_journal_abbrev=Front Genet;citation_author=P. Cingolani;citation_author=V. M. Patel;citation_author=M. Coon;citation_author=T. Nguyen;citation_author=S. J. Land;citation_title=Using Drosophila melanogaster as a Model for Genotoxic Chemical Mutational Studies with a New Program, SnpSift.;citation_pages=35-35;citation_volume=3;citation_year=2012;citation_pmid=22435069;citation_doi=10.3389/fgene.2012.00035" />
<meta name="citation_reference" content="citation_journal_title=Fly (Austin);citation_author=P. Cingolani;citation_author=A. Platts;citation_author=L. L. Wang;citation_author=M. Coon;citation_author=T. Nguyen;citation_title=A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w1118; iso-2; iso-3.;citation_pages=80-92;citation_volume=6;citation_year=2012;citation_issue=2;citation_pmid=22728672;citation_doi=10.4161/fly.19695" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=S. E. Davidson;citation_author=J. J. Smith;citation_author=C. A. Helliwell;citation_author=A. T. Poole;citation_author=J. B. Reid;citation_title=The pea gene LH encodes ent-kaurene oxidase.;citation_volume=134;citation_year=2004;citation_pmid=14988475;citation_doi=10.1104/pp.103.032706" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=W. F. Dove;citation_author=A. Shedlovsky;citation_author=L. Clipson;citation_author=J. M. Amos-Landgraf;citation_author=R. B. Halberg;citation_title=A strategy to identify dominant point mutant modifiers of a quantitative trait.;citation_pages=1113-1121;citation_volume=4;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Human Molecular Genetics;citation_journal_abbrev=Hum Mol Genet;citation_author=X. Estivill;citation_author=J. Cheung;citation_author=M. A. Pujana;citation_author=K. Nakabayashi;citation_author=S. W. Scherer;citation_title=Chromosomal regions containing high-density and ambiguously mapped putative single nucleotide polymorphisms (SNPs) correlate with segmental duplications in the human genome;citation_pages=1987-1995;citation_volume=11;citation_year=2002;citation_issue=17;citation_pmid=12165560;citation_doi=10.1093/hmg/11.17.1987" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=D. Fredman;citation_author=S. J. White;citation_author=S. Potter;citation_author=E. E. Eichler;citation_author=J. T. Den Dunnen;citation_title=Complex SNP-related sequence variation in segmental genome duplications.;citation_pages=861-866;citation_volume=36;citation_year=2004;citation_issue=8;citation_pmid=15247918;citation_doi=10.1038/ng1401" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=D. M. Goodstein;citation_author=S. Shu;citation_author=R. Howson;citation_author=R. Neupane;citation_author=R. D. Hayes;citation_title=Phytozome: a comparative platform for green plant genomics.;citation_volume=40;citation_year=2012;citation_pmid=22110026;citation_doi=10.1093/nar/gkr944" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=E. A. Greene;citation_author=C. A. Codomo;citation_author=N. E. Taylor;citation_author=J. G. Henikoff;citation_author=B. J. Till;citation_title=Spectrum of Chemically Induced Mutations From a Large-Scale Reverse-Genetic Screen in Arabidopsis;citation_pages=731-740;citation_volume=164;citation_year=2003;citation_issue=2;citation_pmid=12807792" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. T. Hamblin;citation_author=S. E. Mitchell;citation_author=G. M. White;citation_author=J. Gallego;citation_author=R. Kukatla;citation_title=Comparative Population Genetics of the Panicoid Grasses: Sequence Polymorphism, Linkage Disequilibrium and Selection in a Diverse Sample of Sorghum bicolor;citation_pages=471-483;citation_volume=167;citation_year=2004;citation_issue=1;citation_pmid=15166170;citation_doi=10.1534/genetics.167.1.471" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiology;citation_journal_abbrev=Plant Physiol.;citation_author=C. A. Helliwell;citation_author=A. Poole;citation_author=W. J. Peacock;citation_author=E. S. Dennis;citation_title=Arabidopsis ent-Kaurene Oxidase Catalyzes Three Steps of Gibberellin Biosynthesis;citation_pages=507-510;citation_volume=119;citation_year=1999;citation_issue=2;citation_pmid=9952446;citation_doi=10.1104/pp.119.2.507" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=I. M. Henry;citation_author=U. Nagalakshmi;citation_author=M. C. Lieberman;citation_author=K. J. Ngo;citation_author=K. V. Krasileva;citation_title=Efficient genome-wide detection and cataloging of EMS-induced mutations using exome capture and next-generation sequencing.;citation_volume=26;citation_year=2014;citation_pmid=24728647;citation_doi=10.1105/tpc.113.121590" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=I. M. Henry;citation_author=M. S. Zinkgraf;citation_author=A. T. Groover;citation_author=L. Comai;citation_title=A system for dosage-based function genomics in popular.;citation_volume=27;citation_year=2015;citation_pmid=26320226;citation_doi=10.1105/tpc.15.00349" />
<meta name="citation_reference" content="citation_journal_title=EMBO Rep.;citation_author=P. Hunter;citation_title=The paradox of model organisms.;citation_volume=9;citation_year=2008;citation_pmid=18670440;citation_doi=10.1038/embor.2008.142" />
<meta name="citation_reference" content="citation_journal_title=Plant molecular biology;citation_journal_abbrev=Plant Mol Biol;citation_author=H. Itoh;citation_author=T. Tatsumi;citation_author=T. Sakamoto;citation_author=K. Otomo;citation_author=T. Toyomasu;citation_title=A rice semi-dwarf gene, Tan-Ginbozu (D35), encodes the gibberellin biosynthesis enzyme, ent-kaurene oxidase.;citation_pages=533-547;citation_volume=54;citation_year=2004;citation_issue=4;citation_pmid=15316288;citation_doi=10.1023/B:PLAN.0000038261.21060.47" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. Krothapalli;citation_author=E. M. Buescher;citation_author=X. Li;citation_author=E. Brown;citation_author=C. Chapple;citation_title=Forward genetics by genome sequencing reveals that rapid cyanide release deters insect herbivory of Sorghum bicolor.;citation_volume=195;citation_year=2013;citation_pmid=23893483;citation_doi=10.1534/genetics.113.149567" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows–Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The sequence alignment/map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=R. Li;citation_author=Y. Li;citation_author=X. Fang;citation_author=H. Yang;citation_author=J. Wang;citation_title=SNP detection for massively parallel whole-genome resequencing.;citation_volume=19;citation_year=2009;citation_pmid=19420381;citation_doi=10.1101/gr.088013.108" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. Lindner;citation_author=M. T. Raissig;citation_author=C. Sailer;citation_author=H. Shimosato-Asano;citation_author=R. Bruggmann;citation_title=SNP-ratio mapping (SRM): identifying lethal alleles and mutations in complex genetic backgrounds by next-generation sequencing.;citation_volume=191;citation_year=2012;citation_pmid=22649081;citation_doi=10.1534/genetics.112.141341" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=E. S. Mace;citation_author=S. Tai;citation_author=E. K. Gilding;citation_author=Y. Li;citation_author=P. J. Prentis;citation_title=Whole-genome sequencing reveals untapped genetic potential in Africa’s indigenous cereal crop sorghum.;citation_pages=2320;citation_volume=4;citation_year=2013;citation_pmid=23982223" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=R. W. Michelmore;citation_author=I. Paran;citation_author=R. V. Kesseli;citation_title=Identification of markers linked to disease-resistance genes by bulked segregant analysis: a rapid method to detect markers in specific genomic regions by using segregating populations.;citation_pages=9828-9832;citation_volume=88;citation_year=1991;citation_issue=21;citation_pmid=1682921;citation_doi=10.1073/pnas.88.21.9828" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=A. E. Minoche;citation_author=J. C. Dohm;citation_author=H. Himmelbauer;citation_title=Evaluation of genomic high-throughput sequencing data generated on Illumina HiSeq and genome analyzer systems.;citation_pages=R112-R112;citation_volume=12;citation_year=2011;citation_issue=11;citation_pmid=22067484;citation_doi=10.1186/gb-2011-12-11-r112" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=M. Mokry;citation_author=I. J. Nijman;citation_author=A. van Dijken;citation_author=R. Benjamins;citation_author=R. Heidstra;citation_title=Identification of factors required for meristem function in Arabidopsis using a novel next generation sequencing fast forward genetics approach.;citation_pages=256-256;citation_volume=12;citation_year=2011;citation_pmid=21599977;citation_doi=10.1186/1471-2164-12-256" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=M. M. Neff;citation_author=E. Turk;citation_author=M. Kalishman;citation_title=Web-based primer design for single nucleotide polymorphism analysis.;citation_pages=613-615;citation_volume=18;citation_year=2002;citation_issue=12;citation_pmid=12446140;citation_doi=10.1016/S0168-9525(02)02820-2" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=J. C. Nelson;citation_author=S. Wang;citation_author=Y. Wu;citation_author=X. Li;citation_author=G. Antony;citation_title=Single-nucleotide polymorphism discovery by high-throughput sequencing in sorghum.;citation_pages=352-352;citation_volume=12;citation_year=2011;citation_pmid=21736744;citation_doi=10.1186/1471-2164-12-352" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=R. Nielsen;citation_author=J. S. Paul;citation_author=A. Albrechtsen;citation_author=Y. S. Song;citation_title=Genotype and SNP calling from next-generation sequencing data.;citation_pages=443-451;citation_volume=12;citation_year=2011;citation_issue=6;citation_pmid=21587300;citation_doi=10.1038/nrg2986" />
<meta name="citation_reference" content="citation_journal_title=Genome Med.;citation_author=J. O’Rawe;citation_author=T. Jiang;citation_author=G. Sun;citation_author=Y. Wu;citation_author=W. Wang;citation_title=Low concordance of multiple variant-calling pipelines: practical implications for exome and genome sequencing.;citation_pages=28;citation_volume=5;citation_year=2013;citation_issue=3;citation_pmid=23537139;citation_doi=10.1186/gm432" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=R. Ordonio;citation_author=Y. Ito;citation_author=A. Hatakeyama;citation_author=K. Ohmae-Shinohara;citation_author=S. Kasuga;citation_title=Gibberellin deficiency pleiotropically induces culm bending in sorghum: an insight into sorghum semi-dwarf breeding.;citation_pages=5287;citation_volume=4;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. H. Paterson;citation_author=J. E. Bowers;citation_author=R. Bruggmann;citation_author=I. Dubchak;citation_author=J. Grimwood;citation_title=The Sorghum bicolor genome and the diversification of grasses.;citation_pages=551-556;citation_volume=457;citation_year=2009;citation_issue=7229;citation_pmid=19189423;citation_doi=10.1038/nature07723" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=C. Petti;citation_author=K. Hirano;citation_author=J. Stork;citation_author=S. DeBolt;citation_title=Mapping of a cellulose deficient mutant named dwarf1–1 in Sorghum bicolor to the green revolution gene gibberellin20-oxidase reveals a regulatory association between gibberellin and cellulose biosynthesis.;citation_volume=169;citation_year=2015;citation_pmid=26198258;citation_doi=10.1104/pp.15.00928" />
<meta name="citation_reference" content="citation_journal_title=Annals of Botany;citation_journal_abbrev=Ann Bot;citation_author=H. J. Price;citation_author=S. L. Dillon;citation_author=G. Hodnett;citation_author=W. L. Rooney;citation_author=L. Ross;citation_title=Genome Evolution in the Genus Sorghum (Poaceae);citation_pages=219-227;citation_volume=95;citation_year=2005;citation_issue=1;citation_pmid=15596469;citation_doi=10.1093/aob/mci015" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=M. A. Quail;citation_author=I. Kozarewa;citation_author=F. Smith;citation_author=A. Scally;citation_author=P. J. Stephens;citation_title=A large genome center's improvements to the Illumina sequencing system.;citation_pages=1005-1010;citation_volume=5;citation_year=2008;citation_issue=12;citation_pmid=19034268;citation_doi=10.1038/nmeth.1270" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=M. A. Quail;citation_author=M. Smith;citation_author=P. Coupland;citation_author=T. D. Otto;citation_author=S. R. Harris;citation_title=A tale of three next generation sequencing platforms: comparison of Ion Torrent, Pacific Biosciences and Illumina MiSeq sequencers.;citation_pages=341-341;citation_volume=13;citation_year=2012;citation_pmid=22827831;citation_doi=10.1186/1471-2164-13-341" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. R. Quinlan;citation_author=I. M. Hall;citation_title=BEDTools: a flexible suite of utilities for comparing genomic features.;citation_volume=26;citation_year=2010;citation_pmid=20110278;citation_doi=10.1093/bioinformatics/btq033" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=W. R. Rice;citation_title=The Synthesis Paradigm in Genetics;citation_pages=367-371;citation_volume=196;citation_year=2014;citation_issue=2;citation_pmid=24496401;citation_doi=10.1534/genetics.113.160200" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=G. Rizal;citation_author=V. Thakur;citation_author=J. Dionora;citation_author=S. Karki;citation_author=S. Wanchana;citation_title=Two forward genetic screens for vein density mutants in sorghum converge on a cytochrome P450 gene in the brassinosteroid pathway.;citation_pages=257-266;citation_volume=84;citation_year=2015;citation_pmid=26333774;citation_doi=10.1111/tpj.13007" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=K. Schneeberger;citation_author=S. Ossowski;citation_author=C. Lanz;citation_author=T. Juul;citation_author=A. H. Petersen;citation_title=SHOREmap: simultaneous mapping and mutation identification by deep sequencing.;citation_pages=550-551;citation_volume=6;citation_year=2009;citation_issue=8;citation_pmid=19644454;citation_doi=10.1038/nmeth0809-550" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=S. M. Teo;citation_author=Y. Pawitan;citation_author=C. S. Ku;citation_author=K. S. Chia;citation_author=A. Salim;citation_title=Statistical challenges associated with detecting copy number variations with next-generation sequencing.;citation_volume=28;citation_year=2012;citation_issue=21;citation_pmid=22942022;citation_doi=10.1093/bioinformatics/bts535" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=O. Thompson;citation_author=M. Edgley;citation_author=P. Strasbourger;citation_author=S. Flibotte;citation_author=B. Ewing;citation_title=The million mutation project: a new approach to genetics in Caenorhabditis elegans.;citation_volume=23;citation_year=2013;citation_pmid=23800452;citation_doi=10.1101/gr.157651.113" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in plant biology;citation_journal_abbrev=Curr Opin Plant Biol;citation_author=C. Trontin;citation_author=S. Tisné;citation_author=L. Bach;citation_author=O. Loudet;citation_title=What does Arabidopsis natural variation teach us (and does not teach us) about adaptation in plants?;citation_pages=225-231;citation_volume=14;citation_year=2011;citation_issue=3;citation_pmid=21536479;citation_doi=10.1016/j.pbi.2011.03.024" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=H. Tsai;citation_author=V. Missirian;citation_author=K. J. Ngo;citation_author=R. K. Tran;citation_author=S. R. Chan;citation_title=Production of a high-efficiency TILLING population through polyploidization.;citation_volume=161;citation_year=2013;citation_pmid=23417087;citation_doi=10.1104/pp.112.213256" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell Physiol.;citation_author=N. Uchida;citation_author=T. Sakamoto;citation_author=T. Kurata;citation_author=M. Tasaka;citation_title=Identification of EMS-induced causal mutations in a non-reference Arabidopsis thaliana accession by whole genome sequencing.;citation_volume=52;citation_year=2011;citation_pmid=21398646;citation_doi=10.1093/pcp/pcr029" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=R. Williams-Carrier;citation_author=N. Stiffler;citation_author=S. Belcher;citation_author=T. Kroeger;citation_author=D. B. Stern;citation_title=Use of Illumina sequencing to identify transposon insertions underlying mutant phenotypes in high-copy Mutator lines of maize.;citation_pages=167-177;citation_volume=63;citation_year=2010;citation_issue=1;citation_pmid=20409008" />
<meta name="citation_reference" content="citation_journal_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=Z. Xin;citation_author=M. L. Wang;citation_author=N. A. Barkley;citation_author=G. Burow;citation_author=C. Franks;citation_title=Applying genotyping (TILLING) and phenotyping analyses to elucidate gene function in a chemically induced sorghum mutant population.;citation_pages=103-103;citation_volume=8;citation_year=2008;citation_pmid=18854043;citation_doi=10.1186/1471-2229-8-103" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Q. Zhu;citation_author=S. M. Smith;citation_author=M. Ayele;citation_author=L. Yang;citation_author=A. Jogi;citation_title=High-throughput discovery of mutations in tef semi-dwarfing genes by next-generation sequencing analysis.;citation_volume=192;citation_year=2012;citation_pmid=22904035;citation_doi=10.1534/genetics.112.144436" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. Zhu;citation_author=H. G. Mang;citation_author=Q. Sun;citation_author=J. Qian;citation_author=A. Hipps;citation_title=Gene discovery using mutagen-induced polymorphisms and deep sequencing: application to plant disease resistance.;citation_volume=192;citation_year=2012;citation_pmid=22714407;citation_doi=10.1534/genetics.112.141986" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Forward Genetics by Sequencing EMS Variation-Induced Inbred Lines" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2.cover-source.jpg" />
<meta name="twitter:description" content="In order to leverage novel sequencing techniques for cloning genes in eukaryotic organisms with complex genomes, the false positive rate of variant discovery must be controlled for by experimental design and informatics. We sequenced five lines from three pedigrees of ethyl methanesulfonate (EMS)-mutagenized Sorghum bicolor , including a pedigree segregating a recessive dwarf mutant. Comparing the sequences of the lines, we were able to identify and eliminate error-prone positions. One genomic region contained EMS mutant alleles in dwarfs that were homozygous reference sequences in wild-type siblings and heterozygous in segregating families. This region contained a single nonsynonymous change that cosegregated with dwarfism in a validation population and caused a premature stop codon in the Sorghum ortholog encoding the gibberellic acid (GA) biosynthetic enzyme ent -kaurene oxidase. Application of exogenous GA rescued the mutant phenotype. Our method for mapping did not require outcrossing and introduced no segregation variance. This enables work when line crossing is complicated by life history, permitting gene discovery outside of genetic models. This inverts the historical approach of first using recombination to define a locus and then sequencing genes. Our formally identical approach first sequences all the genes and then seeks cosegregation with the trait. Mutagenized lines lacking obvious phenotypic alterations are available for an extension of this approach: mapping with a known marker set in a line that is phenotypically identical to starting material for EMS mutant generation." />
<meta name="og-title" property="og:title" content="Forward Genetics by Sequencing EMS Variation-Induced Inbred Lines" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/2/413" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="In order to leverage novel sequencing techniques for cloning genes in eukaryotic organisms with complex genomes, the false positive rate of variant discovery must be controlled for by experimental design and informatics. We sequenced five lines from three pedigrees of ethyl methanesulfonate (EMS)-mutagenized Sorghum bicolor , including a pedigree segregating a recessive dwarf mutant. Comparing the sequences of the lines, we were able to identify and eliminate error-prone positions. One genomic region contained EMS mutant alleles in dwarfs that were homozygous reference sequences in wild-type siblings and heterozygous in segregating families. This region contained a single nonsynonymous change that cosegregated with dwarfism in a validation population and caused a premature stop codon in the Sorghum ortholog encoding the gibberellic acid (GA) biosynthetic enzyme ent -kaurene oxidase. Application of exogenous GA rescued the mutant phenotype. Our method for mapping did not require outcrossing and introduced no segregation variance. This enables work when line crossing is complicated by life history, permitting gene discovery outside of genetic models. This inverts the historical approach of first using recombination to define a locus and then sequencing genes. Our formally identical approach first sequences all the genes and then seeks cosegregation with the trait. Mutagenized lines lacking obvious phenotypic alterations are available for an extension of this approach: mapping with a known marker set in a line that is phenotypically identical to starting material for EMS mutant generation." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2.cover-source.jpg" />
    <title>Forward Genetics by Sequencing EMS Variation-Induced Inbred Lines | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"v8GCelzfQX3-5XBfuRe2-jacmzou-UC_MXbm0JYLuMA","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460567","apath":"\/ggg\/7\/2\/413.atom","pisa":"ggg;7\/2\/413","ac":{"\/ggg\/7\/2\/413.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/2\/413.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/2\/413"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/2\/413"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/2\/413":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022YNU+d7brXc7ZO3FcAWg39qA8kCAQO1B0uHqc63YWXwWw8t2inTVDdWWJmGjAijEn4IPS5ytMEqF6xZAA\\\\\\\/dL+QbbM1QeDMbhbtGtb+c7JNBCrX4bIPfag0ruRwQi2SspSrLDN55kblDUO4N7EY10XJHsZ1fHZXKoFNwhZEsEAZeKPubKmbN04SNUs73SkUQck8A39ZY4tzsjwjh92NacVgyKEcYlUPct9juvXjkZaTK6LJm9SOxExEvbvMTmcFLV\\\\\\\/l70HS8WlgEjrK9s4Y60tZQ9tRqjURU90QEXwVc0aITkz+X6ckkZ1AStBnpMbbpMCtMnkdGp\\\\\\\/Ig5wfyezR1YWz\\\\\\\/V7LPccoDQ1oPRqNIMniox2sLPObfQestzfOQTO18xWMagsZ3\\\\\\\/CVPIQE12VOYuZrl9mjdwWgIxfwVpE1okcmBPDgY6ARTnrAW12u4MeV3VFql2HieCLniiGTuzT4qR7BtXW3\\\\\\\/Fq1PLErls590azZaQ7PLLIUyZzy+uR0GftDqK5o7hWqSoC6z01Tsekq3lvn5dD5Wgq4gvdqvOcRTqHzlAaF5RJ1fpvuVmiUtRO4V\\\\\\\/hmxMrEId3EH+zODH\\\\\\\/YWJhjluLh9rMx0HVILRWIdU6DJUr9UgIlHecbvWSjSLjCtJ3FxLcA8mkkbCYZrUgP88VEcbSvE08NdFQAZrHBvKbXmrx5POeQL9k+rY9ZjEz+mwlBz66QVr\\\\\\\/FFd3FvL0bmRpjB4L47eaDNnFqP78auRF8C1oUJkNGTSgRSFqDIABRZRSy5fmQO6tK6r+qCxuvSsppJbD0zNhqCEbOnqm+qHD6i4J5GRovHLXzMkuLIds6FaZJapZCrovzfOQkNXmsO3\\\\\\\/Nt8uD0za5PGs0lkKzOZOVWuM5dT+XfuSNOooosxEQRr9m7YED0vn1f2LXHdJECXzMmec0f\\\\\\\/JNQYWIHvVyGkN2Op4CRT\\\\\\\/KPB6F522\\\\\\\/PHJUcY3pj7DUMRI98GS1Njs+B71\\\\\\\/T39DmfFygSsjaVStw\\\\\\\/Wriez7bi8GKHs7yF5o7V4m6TBAWVwDIsx1AIIFVC28WRe6\\\\\\\/1Xf8GHOtIpzkfFTW0Ca0XUwft+cQ754qavB+OKwCxZAlKLJlQPsGZjlZZL3Gd44x9Ttp5ftUAgvtZ99TlDvT0GmTeS2MH4mqh127JuFJk+yosnrA9Jvjq0E4IZ\\\\\\\/EAK2lhBlvht647YOxN08gEqN1vYONvj5ieeSp1l28gOxnl0N+eX0mJO76Nc2n50L1mA0R8\\\\\\\/hUcAlLo\\\\\\\/ctaleaYUqRqhmsGMqEIJRZSiEiclSHYf\\\u0022,\\\u0022iv\\\u0022:\\\u002275slqZMF3t6d4ghUXE5njQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002244917bc6d98a7e303de3e06232bc84131491f8adf24ccc7f398a5896a5a9e34f\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022tzUa+y3uvEhrpXm67JdNal5MwEJnJVFr7pIWJG2DO6gsVj+\\\\\\\/ow87KfkIvP8hH1CpBmWpS5tSy6nFbbXk1q2ihnXbujv8vViqQsho8rQVkSUhysI8aPT\\\\\\\/FtmELj6WEA5e++lScSLRNnuTj7KkIozrtl2HjyZhn3tqQxIh3GonDtxo3QQkYJsBsKHTFGFlZWfSbVfMIN1C44HC6\\\\\\\/rcmnhuOu4MT+ID2PsH5IQvTy4H8\\\\\\\/j3y5Coy6wzmfJD+lXOlkPEOnYX4VcPWAIq\\\\\\\/lBTILurSCLjs0S0\\\\\\\/jefjkPcsiAQ7VcoZyu30x1TD\\\\\\\/skkHDhOH6N7cVCWAPbzIKha5SUZMCVCFXNPnwxIMMGQ31jpxcVzrciLAiJ0Nx56zpmt\\\\\\\/kXB3I8jfO3jdVcYKD\\\\\\\/9Wf1tngIDd0muvEh7Q9IQRhH\\\\\\\/C25RqijVslRxHmNQH\\\\\\\/yHKgp9bqCEO9ZlxYfLaXQLVHPdGhJ\\\\\\\/J9869e82HIR\\\\\\\/5snMhmsWNkMvuf5zM2uE8KWR3sT9gze58rgL6GesBjkcXOut2pKXzosnORZUGEN\\\\\\\/hfbpuqLP\\\\\\\/DWgamj5PqtEb084UPUbaY+H3FysCZdSRX1LkkypinaV90LakispW9Oqkxr\\\\\\\/MLtEArS+aUNLhBj45IxvpSI\\\\\\\/yH6wSdERLSasR92dYoPHgck8sWCz8jZ9iDfgiV0HwWcM3qERscxUuBIQU8BvD4M+OEXJy+HD6Kq6aIkyatPuDVdCcyUWggH73nU4A8DmerHs7dbgNglM0arRVv1uasRVhrFjawyfDp++lcIkmZzY\\\\\\\/dBHRAsXtuRKgNaoYG1qyQb3MOnBrWrRHlTdDSDIGNgxtwLpLEBNo2KXxExw8\\\\\\\/BmctBGCDS+J0hm8AKUkdFGAAiL9MzO3eb3lEhAKitWkbQHJh44mAYlg44TA85NVOZcLxkrZkSQ6F9s5WVgqQIRsjBm7rvDR8FpK3qURilt7Vc4Y4MERAwxu67L8qfEmAczLZqFd35FXTguWYj11IVOkE6HzmvFgB3s8VWTJ2mEccgxjAnnKVOG14txpm8nkOfsgP31iUhzPGzV00zRhzJYG2gN24okcEpG0Tk1DJ2btOnx50fT7zTRnlt5h\\\\\\\/R8RsSBQVxRjVjYwsJd7TXapLh+apQJTlm6DDkfRgJ5MJyiKxV8+eWhsPtWfdHXsf8DkXjk7i09dWl1LmQexfKvILiDis5CWknM40OM2sX0\\\\\\\/7p6rvuMKrtlQAdQUaHZAhuJHcdzX54JWcs95Rn90I2x47gcoBusfCL723IDdz0ntsblAGx0ESHib+9qZRSguCkhXHxTKixTUILcYaxvIzJVlBFJPvmL7qikQyFJPq2b1jOrlI8swfAHBFCR4xit92xUIN93pM9UKnEcxt4\\\\\\\/3BEyV\\\\\\\/\\\\\\\/0qnin\\\\\\\/Fs2VF+jimd67QBRT1pprxIVF38V2PnxJECPcMUB5hpqcuYLn9VA9uiCfKL5b2hdG39Xad0DcByK1dWLk5A8ER5f\\\\\\\/7oZ3JN6sEDKOf6ZtX3TAVagW5yoDwH9dEsaJIaMg6byRkArWdhNFjNPNeRQe+64hXc4z34uLoliyBTN5LBlgZb8zcRp+dFdQ0SCdvubA0tnJYP85ymNvVjR8\\\\\\\/u5T5KneeNFGT7Mko1Epz1uHNjtozuyL6POY8Ejc7d2uB7aa270lgGR+8shSpcqgkul0G4ukWmprVRR5bDFwzSRCzG6r4EPQzL9YLjh5RZqaO9SMIV7daDpvlkaOipDujYtPHE+CHNmU3NpIvyllRvsOmQEDZxbORjAVjN4UtA4P940It+cqhTQ8eInbNiXvLXHuGJRNuOcxfl8A+SzSdZMjFqb2kdMfInoCepzugydmh5jJRaIhDbEYyV3+Qt7rDr9\\\\\\\/jOZo3chr\\\\\\\/qXnh0syUZ5alVfYmK1iK3s2B+TzVKXXnxb2+0QZVwUq2pSwpS0TIZLy14pkcQKgqR1ZRR3FXMVCcJM7eI5xUkyWSt\\\\\\\/L3mB4W29YWGVWq3ZeNWpVczz5DpF4PhveqBeT5p51i3cVItp+enl1K+bEbgXr6ALH44NVJxKFMXS8JS6dlpX3U6mpIuvZWfDgyCdFg8fPNUxbFw4HtB0T0jPIOEQQG0aHzHCQf8I6qrjsBH2yqHjVG861LbP1Q3EMVKHtuxzUlGX+mig0vjZzLfArY3k8q0pu\\\\\\\/PY\\\\\\\/CyBQPcfJAQa1vW+gmi0UvqrkCEJUYhq2jvTMMLoaZUj3LT07wlJ9xoc8yHwimAZvhDt4TolcjD1Idx+dQyjYpwXulsEmN3yUDk0eI\\\\\\\/\\\\\\\/tSVah392pY7fRhkLxvfIjaDiBM3KNc3LDAevxlOCiv6Dn9OvWWVakdvacxeFe3IkTPHGQ1dYlmOGtmJlbDU9Pr+2XbrAtwcbh2qoVUuUlzGbc3LqQcUOMsKj81e1aXecudB06vDkN9\\\\\\\/IRr4TFbe2gEIGKGLoHKJK+pTTY\\\\\\\/uwTgdtn5vXJCzvkLVXqJOIG7H8JVbqjphj2yHLvaKjMvTUtOMLCTEm8ec7uuyPC8F3OU8ntCzBzVbaEteHx\\\\\\\/0STaG7OTK9X3kT88jBCDKatoTPXDI3MD87IFsi2d7flMyfY9sesOXK9TrM+AVYLnjiwDc7m+tr3L4\\\\\\\/shqwkX3riDYXPlva9YDqBOESZ0bgTl9U\\\\\\\/ni3jDgjLSJGofvfoPb3CsL6JJRIOwlsHx70LTty+4AoREFzBPrGYLoRruqB543AAVDzqZDxENELOta5hZiJuq3L57ltYcYvTFnwM6Dd6WPmKxr4DheXtpmr+nsJAVCr4eYxaPPjZrFvzu\\\\\\\/RjZDCEH7wc\\\\\\\/jyMG52yYm2cxjnZiO+uzRgVtXk0W1bMYV9lGSr5U5I\\\\\\\/HPfFSPKELeKUFU0d5QLBi7f8uLDYgZnRy7jSzbFd1F3UPydEMMjHCGbgHMlGOA55QrmJs2n9DPis4ZjtnrYTydR3kNqWVa7qHR422fYx99RcRuvhSh4aRP94FzqclBETjZFWL3ONV830o3umPSkRhVqyTyCxzZMAUeGBp0\\\\\\\/pB+L4q\\\\\\\/7H7AfKPFz6zYF3BvJOU4VcgdEy1x5XszH6I\\\\\\\/wX3nN74zPgnCSdHMJHscj8z1\\\\\\\/fooSi0c9MtES9RZSnL0D0KsLy+VNgT+WCDddwBufGh\\\\\\\/Cm2HDSJu6\\\\\\\/CjbhziGzX3V2TK78uv+zppnFHmWZzdWzh\\\\\\\/14CuZiaESAKphab8My\\\\\\\/dzpucrfG9opYY92ZFTYdn6P13JXyt9I1XBe8denEO9CWjcp63+XfJPI7352nSNb17AazJw57M+BFHygUij4\\\\\\\/xlLlpOaMsitWP+AFTfnLC0MJxFARp00DSucFV8jzODETFL4cy9HPBZMVk1mfttx9Vekk3vahfxyx83Z3LWH4kaouf6EuhR2D2I9jg\\\\\\\/5K6PM0XGnbsawq78Z0xuWx\\\\\\\/DKFaMi2Gahi1qhkYCPdhItV6i1Y3zJeakt3fcz8B9MJEsnk2ruEnmmk9I0O4RlsFwzwWprpA\\\\\\\/fEh8rKJWRjrzSBBpx8JQGZP24nooP0lDMzIL3LvEpzjTyLR2v31dlY4FoM\\\\\\\/uNcIagEZ5b2xk620n6Mit\\\\\\\/dcXVd49OpA3cc+3VD0TC9PwI3Fpg5XzR6nqk40yxHnb8V1mBhh7ZNa91XcwTtCU39dBO1QH7jAqEGRhWoxzOXcVdztuWNnROYYSzW3qvdTU3SNDYkTTqtPmGg3RjGcrPfmvWAeok8Qs7dTEbFjNNXajXjjXE8T2Q\\\\\\\/Di74RS629IqwIThqcyORbtksKdB7AzVSPldjjaE9dInkVFxijv7snOELKgTPj32\\\\\\\/P8wVPXdPAVrJoEeaFh9hI8aNKMGuVQBVkCi1ACTOze7l7NbRxHJbKG\\\\\\\/btvGk976JryIg+udjyV0gw\\\\\\\/T0N0UuAf1vHxxtbGlDpPU6AmyT61Dhnk6SbPA1LdgRsFdY0Jdlq01vyf1NPU2i82hW50Q\\\\\\\/DPImyWvwaaGYwLJyVjjEtRCX8QA1B1\\\\\\\/GKeVZsl5f0vhokSmcnmIlUj8HFVBLEUhSlnPX0CPyvD646Se\\\\\\\/yh1PB3Wp27V5M0pm+QHCeefDZfei0RCGaTv+8V1Y0Q0+iojTVRQwxnCNDO\\\\\\\/8HNjrxICHUm5XPORkmXemTWpi\\\\\\\/\\\\\\\/GXUMa6kXFTpSfGx68WcnSw8p\\\\\\\/FRlA9gYfWTXywZwkdP7wl0yk0VhSVP3LkcjblM6Op+qfzLxJz\\\\\\\/zdKsykOtL7I9ZWPBVUOSSjYir5VGWd4JJJPFjmuB2b8ADoD2Pn6P+xrGxlg21VrJHSJZCeaQ9WTTZ1BBGmDwbEGjN2E7LPXCNSAOWfLYfo+m1AR9v7ZrBMHvKL7JUfbDUJJ\\\\\\\/JRWPWi3F9jkjarqQzrxuLVj17JfEkDIKpwmbfRQ022BCbjk7g8G0sw2rEOvygTY0EwA2u2+EXGi8zFwWMtdaWBNkPYeyyyPB+525Imwc40ZdH0iWznMvN9648QDu\\\\\\\/hM1yXZRA8hqyu3GzheiasZcAyWbepA6LGl0f40waaxneLdPpxCLv4xnvciy3VCoIS\\\\\\\/5b0KpwgIeq+WxGRyBFnCwdRhlBYKSmmMvR0fB0s54rXgg\\\\\\\/\\\\\\\/eIZdjQFX47glIlylhVfcP0hQ7NGD0UjhO9y1XwSY5q4rWh6E1o6alDaJeeo+CPGjBM8ANvmLouNN4ktLS9FGqZ\\\\\\\/4wUTdBX5J8qq9PGwExge3k3GedIbhCpouRF2KcwopCr13N\\\\\\\/6V7vaZkn7kV8UEjk9OQmPEw32LsY+4+ngVJd8fvS40Kd2ynghkk6oklccmFZGtK3MCgpn4+ruLR0JmZwH3KR2+ghxR0DPlP7qG\\\\\\\/FKRJLnskRuPcfPaK1SfF+asSjXXy8RBK6KewVryTGYCzEJfvlT2R8USvHXfmsv9MU6mJ+tRbox3AySex4eB6P6Czy\\\\\\\/TLb8bNS0SPLkrM1+tVhvbfNo\\\\\\\/zkzwFHQROpt7vqfFb7sFErkDbTrUxyoORcun9jcU7cAkdJIOxp00OGIinqCoDcZIKiJsjxqR8jJhFycBsZSVJd1ZY3BDBawS9TerPKO0bnQpxNwVnaO1a+BnkM8lWrhmgD+9tzL+TFcRQSiDHcfAnQXqhjS5vHXt\\\\\\\/+lkh0BvoCPPS8UVCUflfOM0NDcsLLhjyJkoXkUaHgbAfV0copgD+c\\\\\\\/JJ3ker9VGCxtF5jkJut0sAOAe5HebOjV1diM81pIwvivd3ON1KJ2MMAc+dRkkqgxKVXJYbGE5Dq+EBPGnt4kRxNhMF66Pyw1VFNiIOzWqzlrbCNkQ4pCrgmDlUwdH9EDOH0gmaqAiHUavHo7Lc\\\\\\\/276q1KqhN3acyUbkskrRBliR\\\\\\\/8z6gGFIgrGfh3+VcrKoWcFMWLhQ6ZaHJhO6ikC4yyFRSLq+d4KpksCHjvb7b4lBvcZGhLAYakEa5Wx8RneF+Rlf8emx5j+tzMNbmrrDslBMNokiN6jKKn21QQiVod7YhyIJlDFNhZnFV8Ixd4K6Ups+UW4ModlO+vYO2hqs60LFPfQAXuApWkCqM+BJuPBXo3QU65KSbTFMfMArAhEOLF0Qa5VXdsfny2fUlPX17q\\\\\\\/apfG4zwBeUi6\\\\\\\/ggsEz+koDredkBRPW67ykkKM+XsHlMCvJnMgwkBngwWBxEZP+1t1WKE5BiMRHGHEFyA1lcWl7b1D0n\\\\\\\/EnGJYDKi+xrHIWcSzYpZ5hVPpVosre5CtjoF6IuodhXTGy3Ij4w\\\\\\\/gnRN+M1+LyDFHO6ipzWREWhOQgbcoJQIcEc4vP1KBmq6kVVyccxJlOPyocseHVDiZ8M\\\\\\\/tMTrAJ1Jjfwn9bzXeVUauMzU65MmGdDAxHZ6yNS1\\\\\\\/qa\\\\\\\/m7\\\\\\\/MCnlysJO3dk6Ccn\\\\\\\/VvFIoOLDPZ9WwiYDF7pTfqUyDuZZm4+H2ys\\\\\\\/XwNsVEVJ\\\\\\\/QgO6zh2EXlp\\\\\\\/CsS1BzujmcQ5tO5xgZvDXKIcoNgQDwS\\\\\\\/ielJkAko07nEI0lu+QuptuUYY4ziD0sHtGHPzuMZoJ\\\\\\\/g36gl\\\\\\\/No61Px6AZbF6+DqOB+y+ZEyM0i7z34N9UKKxqSo4sQ5KLRMbsYcTYQtg+dQzcMv8xachtmWYnOQkidPgHQSk42uJv+2vfupq3eDMXAcUpKZ\\\\\\\/fiCiNLQFCDnFldQTanmD4+KbRkmsPnjzZ+yb47yj2ij6IlYMzmjRGgQke3xLsD+WLoBaG3Z2VdePH0pYlrcl6lqwu0lAqeye8+ZldsSwrAyqbo+wNVi3KABDSNTS7BiKP8FHkW31N3FSyi6D096y0gagPyGBXrvSBA3q2Pmx3y+QARKYOolCpms27tc12s9sqFIkj0ZwOL65tGJOGyq395X67CHMCkdz0WpDxe5rTlZh0uLmhrkGkwiUn2cGkV960Po73zrhFYrgIBoiQ7Ek8CQSRPCni9dBP6gkUG61RJzt3PoMl06hyQiQZhfYpmzNjfJ1ZPip30Agxcx04FfTKQHOvNIKV0dIfuoMvKYWEQPNTj4Wd1\\\\\\\/gWW0qVfeZ32wD+CHNi9Vwi93gW5rdaeek0LilGo7weipQgBFeZYhPXu3mOhGsR6PJ+TrAeU\\\\\\\/yqzxs\\\\\\\/pI9eJxtkDSbvVI4TefuD3C\\\\\\\/7cZvHr\\\\\\\/ksHHXrKBbQ1+tGd9fg7S3+RD3oi2jMCD7IdyEczGqthDDVU3JHtmJx19ll4pnhEJjTeGE8eyVz6YbFl8f27iESTJJyElJE43SxxY+rjiCu7+V2xm1fgZxLTKAjY\\\\\\\/VwPSg0Hy5cxnsKSHwd9fE1qGGKMrQTHO6Kf8RG957Mo4qibSuAzuVLzig6i13p03nZWjEYqMQDNqbd4zppXN30Wx8bJAUjQM12C2O2BGjSr2MKzknBIQT119E\\\\\\\/31rCA98u9ki8R2fJmq2CJ00oD+\\\\\\\/dGV4WWbkzIuixGYgl99K54YgbYv2tVFsibqTgQ1xRx36DR+x8VlMvfYY8Lt2Gw5T41\\\\\\\/1vd+ZFfoGQzVQfqG2zGdt+C7ek+ldO9PPfjcruOvbx5A\\\\\\\/DoNrGTXuB9hte4qZNsxUCx\\\\\\\/Gr7cxYaPtIv3WTFG7INPdEg1B3uBWonLMeAN5uekMtWpC2mFVyAo5UI6mOHmI1xGnJWxF0jycScVpXJVQ1GcPrU0JCPw0gcKZHLXidmARTr5s95V3SKO2LgvHrUhuD7g888h\\\\\\\/vR\\\\\\\/R5e46tLcLXz5IF9AT2N54IlCctP6kKEBEYduQN6jJvZvjSZiBhBpC6r4hIpAs6MbdlYClBOnXtF7j2oJpkL2c6sOjA+8y0ltLt\\\\\\\/almchdgLv2dTu3vCIUpYNhh9g4xmnxDDQXEfX\\\\\\\/3Q7vW2\\\\\\\/jIulPOeUdFnmm75n9Md8XwLQpEWSdhenL\\\\\\\/HykxAHaKMBw7mFhES0VRAWR331VH0ROJDAxY7k944OMPBAYu+z4oiUkkY3EDZeiA1L9uO5zK6+8lf1pS1LYKqIAw\\\\\\\/VzV92DPokJASPhFj98jMTFNVdnHV\\\\\\\/XSipYbo06jnKD7YXmLr\\\\\\\/zRo47momjJHCRJV7pQqWARht\\\\\\\/61otqWpeLlrE7Rh2GqkbR7cGg2K8kDaf6CJP9+j277JLbJzZLCA+ERWOOYBIaNfGUxMGsQDubGnAYxXoIV7Q61s6+UrMAh3DAeWrAqFccBASUf2Wox8CSGbcdo3ejH8ilUCBxd8fwFV4sSVSArImzKelucLxpw6qYGzKmT16TT8TnilI4DXuA1wa9KV9SysS2BO6kwhYT114Hj9\\\\\\\/FiLX6qRfu5jMwj74Wdqvt\\\\\\\/lbLej6KxjQLLt1fVcAblHE2MLjsyo0jl6QJkfBw6Mt41Z3pry7QWjKWtuNrX\\\\\\\/8Zdz6os14buLqh7bE6MqLA+xVbO1oA0vb5NO0Fl4HngorRgntFXhw\\\\\\\/tcNCcWqzaCnQAmyMEic0dRve\\\\\\\/q3f1bJLX2pzDlLQk3DB8hzcK04dW0mhrdfw7XzBmkSgHPBxMWj8NJeh0GjWLC4bZ6mzV1eYeuTSp4T8v5Kjbtsj3z8aWwYsDTK6xit924ViXDgtzxpPGfc8mvd39fUMOCuJ3h\\\\\\\/9ybUHocwBiOeso\\\\\\\/WA3qYuSsGAGs2F7Etq9N4RanaYaXHyKGiV8X6foRuXMFSwZuA+OwL9BlHn5Cum9TNfymJJkobycwV2yhwMI57tUccW5KJDgRZZDd9wp5n2xD+TQS1CAUMkRlEiv7Dlx3cMpVxJ5bg+y60sfgDBLc0GwZr3LsowFywbrfFv0XGT9zWdJlQyRwjU2U8kKmgap09I4c6IuqxJmkl78tOwDr3EUhqMSwZcbMPPygh6xRlMvTGUgP62fTgVaSImabXtOZL6VFLQ5nlq\\\\\\\/S5TzFt5RFTXVOBR3XswuSI6KYv5rLWpvaCl3SuMG75ivKhd4d1OH6eY5wHPE2xlPayb1HcJqLUrmtbCplV\\\\\\\/7SFR7hHQzZ\\\\\\\/aobywXwjlviyK2Bn2o\\\\\\\/eVe\\\\\\\/QbpROKHWgnNB4fzcIoauVEnCbfWwaRnrsV85g8O5CQ6\\\\\\\/FnVajynLZ6Mjd7OBVFQMgQj1J\\\\\\\/KLiKmqrWwtmcUfZCAZQt1crOpl7NnR46PkvEVe41K0EYNvqkWBiBF7sawkU64Ul8gfDeVjUehh3Nho\\\\\\\/uL0HwQyB1lX5PchXnXuWQdk9JgC\\\\\\\/UnYGapsFnEcw4IiGKVCvClGo+UbXKBd+g353627npna9Wlv2PiGG9mjr0vw18PVouuuzSaUSXWKgMdYqIJBTWoGTCp1MxmAcWSiMnAl92461wuoSIsRnOTMtgUq89\\\\\\\/GbHyxqdQGkvcRd4dJP002brGc79Z\\\\\\\/piJXSmV0We3ytOzDj1PTo2dAroaI+d4s6i7xfgf7pQO0Q0XtwLNJxBPhDt7X\\\\\\\/Zjz5bxT+5r4P\\\\\\\/yyEj5iOXcO8abjoGetNAHa6CTGWr+FDL4vq2vATcdQ7y\\\\\\\/DP22N\\\\\\\/0bTeuCUGbIgmr8kDslpvIV6jfPrukWqdgTY3pl6RaIldPmGeFXZyoDbx+b8nw3nSp56RNf0gziFOYehDqD3LEDPieQDDF9yaoMgS99OsUyZU\\\\\\\/pdmjdjaJqhBh3nG0j+mSm6vxhxyfZ5IqhlPNTCAh6XWuS4B9La6XVqOVrH4j4cvrhGEZPG7XMtoOhgIVTLRCHBEuDBT2MIsURujzQf7J\\\\\\\/NvAn1fUj8nfiTNAKMDKR0LoJdhhIaVrooD\\\\\\\/kHBLlywNJhOzi39sKRmjh3pSIoYeWJ7TkH8gguhgvaB6uxcOTsjOYchzgPWKR7j9WyB8KcNCWkfi816MzzLlmiYSL6pu+SDDuumGCHuTcXkqkkesj9ojCjIX\\\\\\\/bnO\\\\\\\/6TKCb4KRbWx1Wus25tlXdZFNiitXQPsSijMhyDpZv5Fr0q2l4UoYPriVFtEIp0o0nMYiSy7FP3lc\\\\\\\/B5Oo+6T6oJ19TNP9AOOunYm5qZhrNv7pt75Zvtj2WB\\\\\\\/TghM26ByCig+MGZryCKrttDswzXcMpE1D8og574qOu6otKsT\\\\\\\/yuORNgXO0xTUjjU02lLMQLXlcvZruRQRE1faDP3+YjDRLjQYeuo\\\\\\\/VguaArHTKKKBG2HYzkIgXEWlL6FATDdYtRhyKpNWhK\\\\\\\/lngSIeQMpfxiSHwbCAx2MxDyYL1EV6hn7Y5UxmsqiJEZFLT4PwjKc8Q9dCxA6t0\\\\\\\/qQdP6OuL8eD+pFM1PkfAmED0C+8p4exNJOsvsWiznaZSKqz\\\\\\\/WplvhNI24iulpWLda8gve8ccTGjGqOUJWqkZk+JLTRAnmI0bLu7GwawnF4wn8Dv4Ar0aRYyfVmjmSP73R1EpEO8JpG5aVLpQTMdp2zC82SSUqMZiwuG6icfXXHyM4SYwAHrHwwTCoxGCrU1vk4ow1HFOmlSLyIykVyNgAbUC\\\\\\\/L1R2OxzK3XeJjxmf4QjnjSUWwWrVCgyHLp6VXenNAtkSVhwVn+hw3wYeoLeRm8xJiJkT\\\\\\\/Qyc9cOYdkSTvJSB7Xbodc1nRzhcTNgUN\\\\\\\/HHIhJx8WbnVZMcGw4Tbrkyhnenu54s6Uvz9XuKhCHBldOHo0MNo3x4MeB9Y2nleE9zrJxRYxnSGfdjEeu92qsd4XTjGqfZLxco70BoCih\\\\\\\/u5ieE7QKhfUi6\\\\\\\/IuyDtfakOjabiWvpnTCEe17z+4JtH8rf900sSJ86Yit5Kxi8J3n2Y32TcdkK6yFPJNwqmm4+iKKp5Xud8KR2KvTBz5iEIL7usZfVsSX8nvEiATgai33fsc0Nl6r6i1deudKEOI4kIjhvTpwGP6dMmQ8YhAUhjK4kQIKKU7gwvf3\\\\\\\/Y8mzVZ9b+hKrnikzts\\\\\\\/jDlQdBD6UOSMhUkcTv0OgEUSfmRvH8tzyBb0LPiDJMs+0f5O8EPdHNWq86CVghJodZdcij6NpeRgrM3ScelsIEkl10X3iZDexx4tLt\\\\\\\/kZqgwFP1Y9ITpnQ46jpB\\\\\\\/QjVvsrduZPQxbvqTv0nqCiHSAUnmbmGPbR06\\\\\\\/ADQhUR\\\\\\\/oU55tVOnf9HV+8gaHplWg3M9tKWhaKbcuqEqKY1dA0prk1snaWKO8Q7VvjCiLjof70Lx9UUAFVbDN2UqZxxJkDaCcPYKcZVuFHi7RTnP\\\\\\\/jWngZW4zvsI9ozAYtQLBgsaZaHOchLnrh5NbkIglBFLmqcg3qPI6cZiC+i6eGpQXoB\\\\\\\/wjaPqkgGjBY0jfX7uL5w9bOAaJTuXhSGXlXhZKhfBXUKG0ctZU1TpsLn\\\\\\\/7NAACyJ+EaMVrs8Td1OBuBSitXdPoZoxvsOE0cvcUi2CODQ2d9yFlRtn4V78q1zzQOn8fQT+8Y+Is7ohWMOnly9phXME51bfzl7GTSjBPrMtTTA0QPZ\\\\\\\/fdB7LuTJX9\\\\\\\/Ptg8en7PU7ARZ\\\\\\\/8WLhdASYb4z7E3fgfqFm+vNqn\\\\\\\/mBdCQ0Aw05CIDu2IEirntsiK3soPQCgtkOz6E8uD1WfyctpWYPTvTbhKdagiZYHesWvrQx4cfh+NAEqb7TJbxxBDlJzQr\\\\\\\/EAxmz05U\\\\\\\/hIzO\\\\\\\/ykSlx8ZsV8uns5i5hTxje1TMjt6kUaNg7E1JbM9duKHNP+r7K7prj\\\\\\\/9CwWrAj63PgfliqgrOvKcPFFISuvbeCb6OShzB\\\\\\\/iNHzWULLrSdB7a3zdU5A+7xKO7DuLNg+AGqzdgw3gSlGS1TgMeVF3PCX6+43fMxy3KTgW610xwL37mLMFD3U2hrPUTlW0Q8i4ffqTPAd\\\\\\\/FysIWzvDfoxH1z7s6fRcENangEdrh3+fQzFRYh5QBdDdwU2rnelpNeurxcLG+CXxWL89G98XrptyTGwH+6fO+MxSlvGi9UlfYhI2BW6irijWmHpWuNfg1SP9DVTc60YNJrZCiJ4JzUg1BOQQi52LGElvGkmWejl0hVc+wJntSr3Wqn\\\\\\\/VDhYS5PTHAG\\\\\\\/EBwLG3OTRyCqonrskeyXIzEt5u0CqoOXwqbYbSPGaaaQ6LtaoEIKbxmgwpAiKzG8VSEtqfB\\\\\\\/w6QBoUPeFriK9DJ+HQEE0RF62n\\\\\\\/Qdh6y1eQlWM76n0u8NHGYcZPc1Dz8PDLfVR7aiDaYw00h7wYJ6gebxb6ysqVzGCrI9mE3ylAKTgWRIvfr3rcEzXQjmfFCom2lgae9l+KOeDvaszOyHufVAZeuSHdVEtd2j3uSHvowp3EMmYEP\\\\\\\/+gVgfvV9yrZWfkZuCspd+jY+1dNkIsO+7IlCq94604serUeDpSuRVakAeIqPjR6tLnrwe4FDD3J2\\\\\\\/8I2teiJ19S1ERlNQd\\\\\\\/7AKj9sLMDNpbkowUhFTxAWXb1PE0RUISPh9aBfVal53TECdnssolVSjlxOGA3UQOfRCG9tf+QTYLJ4NlKLM1ZTQxX8DXCWFbGSHMMS4AphG0SbWZlu\\\\\\\/BnyzHBgUcJJyebZZ+qbKUiGB0sPcXv7BWUIVTRhgAth7GYQNM0N7myzRoDoXmAV1nfh4uhx+r1k4UzAmHVKvdUM52tt713ChLQs0G0ydWgkERnaxzqI3koN1j4iJetsAfpqlBJmt5hn2qeDWqvx6QxtvJpcT6SaCfUqQgweo2WfqoxEDf4Mo1QlMEpytlS3SM31h7vpFx+SU2krQ1J6eFIKdCrDOfVURdTtRLfDB0YOQFN3SsKM1TVeAgzu+6PIYKjf1i9ksd5XQvS9H6ciBatQz4+YIl3ZeAyWwGeZ5M+6eU4uUKbAr2A+dvkIhF1LL5E220I\\\\\\\/JFBtYOkY049Ps6JLRAiVfz1L7FWiLFxRPAz23Xa5x+DO3Fz+vOEapJ3swXxtO8rxV5V1fvyHg+xvIpTvtCPoVkPll+PvRSQ9lwVi7lfnwrymQf8huHvxekQUG7lc+t3Y9Jgv\\\\\\\/nSuis0fp\\\\\\\/6A5b9UV3Umj+VB4+f6g6m7Skczr2pr8OJ6994XTpQGyCFVPtDVXfQk+1uhpXaEGtxYA9D6umKX+F6rBXskgxSF7k8VXRllDMAB13YBiovzlveDyNoYtrpAbXVHZglajpHWW0M5FyyOa2kR04XLr447pvcKJOFwAqXa1tTXUjuE6iQytp\\\\\\\/P2eT+w0txHR6Ov8MgO6lZ4Khyx4BWWniaYeUJXTKLD0Yxm1NiLGF+bX+S0mbdJcWc5dnNmB\\\\\\\/5NU4Od2MTCwFu9PsQ2iW7bcldebWg0wJbUDhyZvUqQEtZZWjOBka8z65I1FRW28lZ92PRCqu678rOdy1+ou41D3xY2ORMvp5W4fFrdKYjJ+o69Oc4kAwVwIM0A7OBQcTSQy5L8EXSzrbRXvKzGxTV842DgGIQLAJX7bV1vf9yqnlm9X1cE1e\\\\\\\/KVxAArGCTsK1SNlZOnv8RHH6OFQ4dYjoEOsc\\\\\\\/kOUyxWYK+KKrse8Ea6jt8uADATBp2h5qVOp0EFoGHxkWTS3azyHBXzzWdo4kEFueQf3Mk1dQQq5au+kYd7rrswS3UjES26OeOmjYuf8H8Yrjh\\\\\\\/5BFlWdAxyj\\\\\\\/zXjq74CgeEvH0m5IipoOif9Y91zNpis8CCjR8XQxhGYmMqYNIyTFGwn4a77c5BLTwrQBFz0kwtjC1oM5Kw5haBMRpTYyhpygS+llnzC3wuJqy6XmcdNrskRhHuyvH22rp\\\\\\\/aSe2ymy4ufW4dv39EJK0SmM2ux8doZnJfn9xwsjdgLClqDIcimg+AvMaVWEGOYPiHlT8IdIYsFZtqtuOoRAoSJI3VN89+3sZWWwf7m3\\\\\\\/aoDG1beyj2k5sNeBb1ffX9LMCGkyzMm7vy7XVAGOI8wYcsUOmSE0m2hR\\\\\\\/eAaVhvKfDv3vD9Fxxpb8ziFSAgu85uPVmd9XcmsM7biy9OUMN0gA9ZV+bRNs9h3p4OIc3r5KWyFzuZ3TnGcKiTv\\\\\\\/0y\\\\\\\/DqjSfpziLZDOCA6zQBSJzZGP8x2hzR5619iAXlFLntJ8uX2UjyQfNi1cBokgcFGVsFXkuQmI+pUTf6BD9bniZLV1l50Kk7hDCWfWzyc6VkIWn\\\\\\\/OcAnYrOPRTl9kOdlBWhruVhkVkO+5+51E4\\\\\\\/eFD13q++JEOEiqWNXB6rY6xYW2LZ\\\\\\\/HJcSobIkU69Xje5xoAPc\\\\\\\/Q6v4IlO2UleONO+KYM+JpCgPPgDnULydOHRAK7Avd\\\\\\\/FrlVaImsbaL92Yszrkkzc7ylEjrPohhnRz24pv+Zgi8ng+xseAB5\\\\\\\/nU+LGBrDdBUwykP\\\\\\\/Nf\\\\\\\/Y8os9DgNRe9BlqKx5V0P7zOdQ3++zsy6Bzwl22W2kZigEonZ3FLko9BMLGVMK3VU6gTNbTaXHhIvL1T5RtvmEAt5RT+YIuEOTuC3my4QDJR7SubJu0PRW3G0IAVWdC87y32TQ4lddZnc0xV44Ii35OMjrLUToFbtXYJ437jM\\\\\\\/NjyCLHLQjizsLKuhSxXLbJDAlQ3ivQkP2DcX+dVpiV1qSdRz5VGionx3fR2MhxDHR3gUtJK+935cjVmuB6pXbdGQTmd80LMYU+truKAYQqwjISDakmTjNPLAqItWcNUD+Hr6qYaXWt40V8QxT5KgDsZEj9C9t0bZ+WA1Y2YL7qycfPl\\\\\\\/M2X6k7hYm9OkylfoeTf4LxWWFawKs7ZRDqi8NAWqBMKCN68HFF7f0\\\\\\\/W7KtHQH\\\\\\\/ui9mvTBxd87c8owgEhiOgW9UNT1u6EH8SxXlRPYsQQFaBPmuS3f9obDNgoPxtEtaZFoeWRRqsHYPeQUDS2x+2dT6cZ81LbrtZxFfUEsJ9Jekj7mZgnWOgJMxg5e4glTdw00cihquGveNXfZmifcp4Y3MZYbNtjzptdxjuNTl4tcM+kTFQyon2B9+sDkNXaebx9M5Vx8GYmKKxvVvqC8yu8bDXWAAxS+fGCdas3LiwHA0T+8bOfMoDrC2MawDw72BAoemd35+08BKQ63esh7+1TGuaZxPSiQMFRIzr5fkxBsdrlOqiAhxSQqALVGIpsmYVwbmQ9XJQ9Jdrd38YKUlZrb57V2VSbayE11rjmCEi4OlNWfBuyCTIGSGAtH2tgGVngidwp1xCrU+IN7U7oIo4Z7M5nYHChZR4LTN7dwhvJ7S2LVYA\\\\\\\/2+Bq\\\\\\\/MGRDIMuo\\\\\\\/VI43H+yhzmg0rFnSPLBQ\\\\\\\/0pki7EXWM3u7\\\\\\\/eiGiCGCZgVwU90+tO0KPL9zWJzGELc6TdVJKaSevLHOANgme1TXwX+xUcmmCdK\\\\\\\/jPhzlmUfKdqVGrtnIVCK+G1poVa+qnvCSSS7CC38f1mrpKL1gBSsPmDDjXlkpMuCBJWGY\\\\\\\/AkdPRJR3JlXUL3vLMU6QyzHXUBaDaZePwMsGApJhS\\\\\\\/9vFSjN7V+HEvS60bb8COYSTDOfaGbO0P8b7trw4alJJXkqR5YMpoXmPhl723PL7EKr44dznRZFTD7M5yrPduaNYZwR9jA+uIT1ifo6jkkStbPODHVSyCG25raLBpVS6BGx0kVpGkPhzUxu2PFJT10YQYgqej6DG7S9IjNjMyrB5FR6xcp77X+\\\\\\\/n0v3X5z84ZTv7sZO7u814CFLJmqPFOvWDjHrB4tksKMnZnEPU5+jyVe880CFY1LUBkCAJnqCfPo01e8iBzZ3B3q7TiMs2JkFlv3pVh6QqpUS0MgbSbJUpXD9PkblNq\\\\\\\/8nIok51jV+9mCvWpYZoaWetYyKhc\\\\\\\/+h5eOPSgZl0D1+gHE12P7zGtPMLprfUfrVia1B7fE2cMmdF66PCa2C7RoLaRd+YehtYdLOadAd2+gl6BpVh04mcPJnIc4gKfHOifYHjKhggyIhgHDAZaiWcptEzuXgLENLlJ0f43xEmYct5V4VgQyZWtutjXu2OP3G8zBISqoC0WDq0QgSa2mcqltRQ23noswqHqV4FutF0sggI2Z3QNrG0clEUwtWrS8u5N8Ofwo\\\\\\\/V0Pmz0aznbHryFX8NVxfwyAT+bdNjDzC8cnquvhiUlik41qgke0fPo27V7swZjio0vZRRmY\\\\\\\/ukbc5ZPSx8Zv+dLe2fgJxMi3ruI3YhEHvSRfghPQIbbf1DGZxW+bzul\\\\\\\/0Lp0cZ\\\\\\\/vSUKDHuHKQrsqc+WBg+CAfm0pUsFkydBePZt0TwQkSqtTo+hH1XkkLnxrleS3E1AsbPXSt9vdz2qp9li4naLQNMkqoNndCfP3UdvxYrFN+J0fuxUTMixspDqPcbpVALZPjciZWWq4HK9L0a0\\\\\\\/I4VuMUMq66+CgUva\\\\\\\/Bi76hwVJCAg809L+nut\\\\\\\/TvyXqNPTX7iGK6qxz+5pdWVOvvVLVFMrgSOfZG4zX0w74\\\\\\\/5WMCm4aow1PiF3giT5C2+aofaV7cNyStQPuun8IDwWZkRSefnow91buY0Ox\\\\\\\/Cgt1p4O5wkRM3db9XVSj4iNT1wdWPB88\\\\\\\/YsuBzPOya82haSR6NO1WIn0wDthVHp9Rayt3DI7orhBiTjIqnlJdbwYEk77hnpiNUsoBM2YkXNs3UUgQtW+2t9BUaDvbITeaw9jLO3ofvBPEaHZ8NedOoeexQrHJPfiusuODVYcNFQobCcGWhrr9aYD33kncjwZ9no\\\\\\\/YWLzXbABeD8erHgEmqeW0kgndtgo5IP2UTIJA2NlADIySSMkWqDEgi2J2HynFyKtTOogqPV0WhmJRytBLA4dK2OejxgQn2AvPPSv7tLt5gk0nHj2IF0zyTLxsEcsUNUxUMzVqb9M15oo5DZw3OvYTxD2jL5tClR6Sz+zwQUft9XhJ2f8anU9cbSyDC7Uwlrcfy+GIlyJlNw3Jtq\\\\\\\/Z+fhcZZbyn0AhsfiI2lFlXacW8a7XxMAAcHUmqv1VUnyylo5pqdNpjOgjn73TlT7mzZifN90hSCs2obsbuW3DttLETWz7cl+KZ0O+kA6CHmSx46EfRSWyXRVdZ1PTIfTJaV1wiXFyogdRr\\\\\\\/0GgEedlZoLIS9yVAFyca8r7f0+sQXKYdEe7XtoTmE\\\\\\\/A4MXrklM52K16GOdOwi61v+VdqZqhkFW\\\\\\\/uFotjfGsgC52Ai0SCsIiz+sVuzbwe5O+TXvvvYpbXRBQiaVNf+lVctCzZnIgaKJRIVppl7mM4kW1zeegJCH5vlH3g\\\\\\\/e0AnZYtdxxqv\\\\\\\/8BxZ4IHpQDASq\\\\\\\/zpSC7wtPDApAMbbV\\\\\\\/idrbar+6IiLFJBPMHksMGFJ6zFap3I+BWLHVyPFKcd+eLxPHOX5fSsmhgXr5SyQBydOtJjR9XB+dSsSD\\\\\\\/H6VSotyDD7TsCo8oWA84lBs8nUxESXHh0lziWj6\\\\\\\/hl\\\\\\\/wqZJquRf+Oug2+dxSxSXS2eCHut2tNYDcpCQuQOu1\\\\\\\/fbrzQNIH+B6pm7F\\\\\\\/3TuZLicgiyzQORtm6Pl+kpj\\\\\\\/U0MPtr4mERzy7y\\\\\\\/hJ7SG18Fn3o\\\\\\\/1N+zfeEbGU655hugUaTCLZSPFcQ7IsxI0F49dkOM7f7zQLcZ9x7VmlubuEljWkAGQPLGy0pcqXrxqh0OM8pLCtgak4vK9MgRvVPWt17LCrVWrRY9akZD1NPvSvB3pHbWhv4ZNyxR7QA4F6y6C4K5QKoM6PWf2+kug4bg8hu\\\\\\\/kdY321JXIguNziE2FXyqeiXrFF7y17HA4a70qMM28Q94M4eJGn1AsXf\\\\\\\/FNqYKRGaPVIukDIq4KdGVAProUzEOm1Lre\\\\\\\/PWv3rZu7jS9UMjttcVI1INO\\\\\\\/U8SpwSDaT+SbzILqC8qWK9Hm9UZiJUrcWJFoKwBhTot4yxK1DqkUpgpYvwo6JTTe9\\\\\\\/9jGnqo9YyW1YR3BPiyj4Y\\\\\\\/tc+u5lp2mLSQhkhq\\\\\\\/hFvPGz+e8BMO8os0mHkpItRDgOVOfv37xCU9AtzxKYLc7DbZRZu8c7yN7acRsxh2xd8XPrHbANV+jOyRXCgrWv7rxFmTkOxvSbHfdS4+n3toWxVJoMooRemEwuf0ZDfDFsWD0\\\\\\\/WKDtGlAb7ax0OeghS3qkKI7Cm2cWf7k14GYqbNrhmO6GPxJD1Nk9i3EtyFao+U6zmfTrt78jJLSQDSZGgDe0PVubo+vUiUWGmG9i0NrSVPZ1BYJYhvijZYmeOIEKY1KHWV4fUbod4ZPFnCrZBAGcW5Yb2fSjKAyxw3nAklbDtntJGmSLBnG6NFMupDVUgMqD33XvJLx+iep7Nz+FITmxWkC1F67zx2vEqtJ6380vI82RlbX4snHpcZ\\\\\\\/1Ou1BgbIG9YuPlSdXlXackABsxXbqNnFj+iPSfjCtps4J8vZMmo6oFBozetIYBYvEF5YzMkq2fdcuHHb1ZHbL4LHpyOAm4UD1pjJQJS0t6mz2\\\\\\\/L2dR5AjXD2ba8c4xYr79kzx5QCtp6f7OOjynC5\\\\\\\/UPdQ71D9rT05xJPgRoiGZHUobdMl0jeNoe6Xi\\\\\\\/68Lhd7UUWsJkNnuQlXVUOmyS0lCxO0KCHA4rRrcSE9G3XcLqopgbPH8ujLHj9EJXeowCmpSY1vB37SbPxyfXr+TiXIDQMt7Ak7aRI2oXbLu5+HkE4OEbo8pHNm2Mw2Knd2q2X7YqXAfk5r9p5GRMdrwxRWZxJ8HxOfMI2L9aHZYgJuFIgJu3IvAAZdIFkldzBqdTlXy4d+BvaUfrQgVuOeJ7utC6Hk0f0fxuc9Iz7BecnKKJA40eD5KhedSWgy7aS+CSmR9qZKEnZGh7\\\\\\\/oZvD8PX3r07\\\\\\\/cS29AEyMO3n44+xIg9SYAMW5zHqI\\\\\\\/UjNFCebzYUdG+Q3Wnj4ptxvwTxI14xQxs2CiAsZbF5bat9v3G6EM9QJIOLjqkl6vJbymgOvpsan+fBMc2bH7hX8UN8Rd6pze\\\\\\\/Dfnmv6td9PO\\\\\\\/9IT8GLVP47cNOBj0MTx9HgxiyYzhXNzkr\\\\\\\/\\\\\\\/a9\\\\\\\/SmxoK4nNN0IYQSy1p+wpb797vxMHCfGdk5\\\\\\\/XcSICMWsm6VObWOnUBqDgZdT0Dj0w6rRxdH+k7BHHKH9MTL93at6NEaQi3mIlo1P5Zk3jQEUnwwjp+vBAX54Zkfy40zb9z0y4UOkZZV9h7YkiRUXm3tTXkY1epM9GvVh8fgegota49\\\\\\\/3LRPgv6M8eE19U\\\\\\\/vHGEqV8UkYlXgg2ZUhpdYTip9BmVCNfp41yINgyBHbGijwQp3A+UgjlOJsLL7lEt56sdBAbBhb3A\\\\\\\/pXTk5ES7MyLLQIZ9ST2KNPh1XB6FTWBUqtyaRxVMb38ALyWLNPSX2oagqjrtG3QoSlqPKdCNt+AKEJx9QVoSE6r4AMyaCH7\\\\\\\/rAKZ6tYddudgw\\\\\\\/kD6ETO+2giaE1srbMzvYh4WTHATYb5Dw65Y503TNmhpaNVjiyhYOSzudRBrX3FwiKfXNElsFuyGgJ\\\\\\\/eAHMyKdPCKlsxNv7\\\\\\\/fZLUQXY4sl2g+Rmh3PwrOMUiK6kQTymRsyMM9FH4YSoRWBEG6rhE33c7nA61usiGCqv\\\\\\\/GFu\\\\\\\/ozsI3lGDcn1ZT3chrqvsj6MnNFYdNL4cPUfNSlyqpTUcRAOcNgPcNtcBkeT8TgFtu0wbDFa623nDbIVkCaYVPlmXPCNX4XoWs4vi25cFvHXobqEL+74X65Ti9pj7Bt8uu1MW7tPqPCFx5c4htFhj2wjDCyW3nQqgJhEJPpVloXQ2N4UCgBi6hcNqMWyZv4lFVoT2rcR8l1Jb3upPSJyFw1Z0tFMKMruq8VSYsqvQxOAuh4Qar\\\\\\\/yoK+o3ZfnCI74TExpjwbl5QhTZqgEJQysyfYvs6rTGwlMGDMGLwONyerV\\\\\\\/eWLI862laUarydJ7V2DvDmX4YxFxAd7DlSXXMdtR7UTbJcNd2NJAudpu2xYEJK2oGYyg4YAIdWU1t+\\\\\\\/0sSft0\\\\\\\/1jNZJ4WtH9+oYM8PNBSZUAQR7scn1YVUejlrZ4l+67YLFRT2S2cRtJDNw7yKkKx7RAogDapcNBNntxGxleQzP7D9CuMQ5Sk1hv9V2lKqhW6BJA46\\\\\\\/TIqPTksSscl7qvn+wZo+FTbCpGKvbEh1YbZmkD6WYzADc5ldka4kFcOGMf3CDWzFk+sYCTYzEl8ByFBMoe7Z1pG5ujDGEF8Y+asXzaHULMCovBGfU7Vq7xzfgsVr3kjqCj2bB2jaTC9+IocdPa0x0PXK90KXQFv4cN8lJXPvi8z3YU42xBFSexxmhlIqd88ylZK9EfpcWcYrrB\\\\\\\/t8S8DCGGpZweJwoi\\\\\\\/CantrMgxdBWSWJB8NlavL+5B5\\\\\\\/hy0IecKGCczvca5QAuRv95CesNhFozEtcvUOTfLWlYwH\\\\\\\/YV0FuXZK7c6zivSCJ0rZDPO+5sDRPBdUDW5aGNvQrdQD\\\\\\\/c5zCmlprZHi2WXfL\\\\\\\/QCDzEBrjEZ4S1Uvrws76bt8FgJo0732gtPgNiPAmqgn9nvtboEb7vqef1zF2f1U3W2I40pNN26aocjEHIh8FonFa2MJvwZNC7ljREQU5V6Yx\\\\\\\/sqJ6Eb\\\\\\\/n\\\\\\\/D9Ljtrd98tYRzK9AN9ke9YbDUBjfK6Je1UkrAcvr2TaNn7Ee\\\\\\\/lTxAwhB6rgDmmB+zeAVh+zXbtS8ybDYdupw+puzlqh\\\\\\\/tVDeOXeZw3Yf1+DeRq0M+T+io7fvKA6YgjDvjkJTNjVEzSFhdi+nDZrGovqzoOpEklVF0elkqMW52IYk+AIae9YoKdnCdI\\\\\\\/WzevPeKtH+3ct+ewdhXBtxh3I5m4ga5K6n8G88GYRY7+2wnrwxaQo1+yrwmzlfzoYjcu4He2119l3s0dK2jR4w+dm4rYGFtOHD484sL1SZd9i27vGkyvzmJLis0X9X9Wsr5Pt7Or22t5NlNKHT56\\\\\\\/Ubo\\\\\\\/nExKTK18iwh0vKFrHtrzx9IpwsEOErhGlCwwneNEwn0MOtCxDKajV5x1Hfqz9uEliByD450xxJf2\\\\\\\/oRd7Qco8G1gVM8YTWTWDFfbo1cdeCvjWKhWpSk7vkrg+wJaZLPJkNOSvTlcMfr4A4cve7smt\\\\\\\/JoKuphYqQr\\\\\\\/dPf8AwheSz+tKrTKARs8754hEDkDzC9W8PPFsmXG3jb6iSItPeMa28Rl8ezSk6D2MOs6C365lzIfaJA2WvcjxuHn2jNtpZh0jdU39iBFOXoj9\\\\\\\/bsWQrLzTI9tsHIE5bDPE\\\\\\\/BWd4pcv6Ef5fNuZiz\\\\\\\/prMhoKOyeZ3ikhvZX829WjCjGy0WKOuEto9OhCYQlnySrbDYTyKI2VqiNDMuYvEQqKTA7vgwV05DUCOmOqrl9Sv9FAJI6KpXb7Fxlq1AIdZbHRUQr8HtbR+AmyeCrMlWyMMHBizVBvaPZf7faKHYF5KLR7eM8tFP8Q0T\\\\\\\/GNjvSzQIcnVYltbxBK38iMdMhT+qnd4CKMGAc8YJKTM8swm8azVRgWSTeIQ72Ffr5wz9oNu2l4N8GiwTp2o+Qo65WMbP5HElSwrGZYYjz+ktwI0snG2lSBhvYmFT9aYmL07N6s2zR5br8uy\\\\\\\/YQvBptOYravQkXKpWGMkSQ9HCdceqqb7Hm\\\\\\\/WAKXRnFTy0Vn75nyWcFuo6cJygl4Eb2fblgmKZk8V9DUXjWuWrel3EJSJqK7CScTTVDAq3BFBFA0UJzzJSPyGibCcW8BxmC3EIuUH0b1dKsRWi1tPein1W\\\\\\\/H3sxQi4YRRXeFZQt4T5o0YgH3F2UxbZxIU69cYEqV7EmPVptHU3T1Uddh\\\\\\\/ZlIiZGdnC4BLIGQYhatrxGW9gtdVDhMqw4mHN7AbDxWQizQmRCQ5evUyaL6h2N8YXVcunX01ctY3XI5Dgen\\\\\\\/TRCSGvXfIabmsEweLkiXWLJx90IMnEaXgVhKW1bPB21OiZJZ52OC7TAjGVCdALRoOOk3por621oSu\\\\\\\/bOnIqERB6\\\\\\\/ZlVLmcf1ShJ6Je3kOK5yB4eZBoV1jjngL2uAwhh5Hd4Rcg+YrFZ9izpB5YTxl\\\\\\\/ytlayQdbCqeQjuJaN3BwmfmeIM4LHoiP\\\\\\\/34Oi+SeaMRkSwWs3l1prj5rAKMZiKpkQC4wMeJwZ\\\\\\\/von2be4TRUywonm8ohPlll3SzobwtXhTdHAcFIT34A8OqKhxcw08E46eaHSf\\\\\\\/ub4+oORd8XWgXrBBi00hXgLXmOEpngo5ScpoWUOJ4wPBZ+ZPSdgpGCTxX9wBzgp0vgNuMLIpU1v\\\\\\\/\\\\\\\/gat66Blkgx8goIQJbcKYaqWHZJ7Q7GRnkDENtvNfRROIQlRAP2n4gAcUGNW0FXCbFxFfN1iJk5n0c217H1TfyyfKGJIU6WejLZ6TX0FTuSXvFH2G6jx2AUtt3QMh5CggDl7eIAy+1dyKI12M5afkpTy7anpPSoCQcsFj3+c5F6e9MNlFuqUYzmbz34Y52GZlJRGVoZZQBy3dHYy72+SBM6TuUBiz+kj8LUgRvkRftn5tBjGSy7\\\\\\\/aaHH9EoFSrYC23cz10gkrBjhbcGKCdcKyWIXWIUhPUS+VB8hz5ct7hOTgqDAYKyfJY3IFrCOcMnH7TrMY91YzkKPFK76lMPdtv2HqKBjbYCVAIZh8HQO1wuZBtsaah8jbvu1g3RDMSxQwDr1\\\\\\\/AqAus\\\\\\\/juqWKG6fCGsfmfIVJg\\\\\\\/8h+YMo7jz\\\\\\\/eel9wiafyyM58DTUveGyPzP+f12XeYSRK982Itho41Jl1SH8bj7lQdRzbfnfF6S8vYdGA9tOdUf\\\\\\\/Wl9w5pOW2gfcy+YkLbwX4UHBHGPaQ95dD\\\\\\\/IZHu8iep3W75sLbddGG2UZuhJmYsQ77vGqHA6X6xNl1aP03sx6ZV7URudI6sQrffQrJZNcIIdR9yPcShbSwi0xJGmjgT51un1sVgMkRmT7CUfqjczfgdvr6y+7PWrHr9zlX+tecaXtW1ftTIwiV5aCeTid9aQ7cVxKMG6pEWqGknELjjwqBg\\\\\\\/MrTItzLfiZ+EFBqA+LXYD9c5gvU0C3B18FfAKiRjMlKmcScznxeFQ+xrvsZL6OR0a5sgd1ytjtdkskOFDy0m0B\\\\\\\/fdzmW\\\\\\\/TJXuxKVVwg9L03pRsD6TWV4FlLS+Vxhvgnu6lxsDBzfGtfjSaMaCnNUUDeaF29BeOHYuMMK7RQ6y9aXjSzBbn\\\\\\\/t\\\\\\\/VoW6T\\\\\\\/BDiPvGreoEtor1yxA1dEjoplL9QklOB5EDix0gsjHTZclKzhv3WG6TlaH5QftUv4f50+9xEW1ln3cyIv\\\\\\\/Xp5Fhz0DfuJFSrbRckQJL9NpuPsWY1\\\\\\\/zk2Us9Mb6UZXJsmJVZgYAElT\\\\\\\/Sb8mkvSYmQBEbthsqhQgt4IQNAuIUHaJYd0XwIaNk+nm5\\\\\\\/bxOEhequTKVbRJ60uDgFog25yXGs5RSzk7pM+Y9yjRDWd7LXEd3clqMNP6kWj\\\\\\\/yz8TJ8XsbdlkPv8qUh0Tvsgf\\\\\\\/2ZdEZgPBc5xrgUa52wNhAGs0LzWgLvFdD3YfkPnmRZ8HHm7nbYCuVFS6kSQkp3yK6YL1Ga\\\\\\\/RpPbL4PLt8qbv9IkMrrIXif48ejcQ+NKiqxFQ\\\\\\\/VZNpGEGlhI3RMSvF2TiuVmYM3e\\\\\\\/xkq2cbTzEgkY+3c7iyd3VROdhWsYBnHfQK33YSosONOBqZTmRB0NnopeX1+SFDA9c5adSp\\\\\\\/cvU6vxLcGfZLLfbhCYobCfOsPU3lU56GSOWlW7nJDjUfdiS1InzKKQdO4oYn+qJcNyF8erMAP9yM\\\\\\\/tMKeSKjauvbCwt0G5NxT5bdSLp9lbYy7fCLyd39ZHIde964BbIK3H6zWZZ7guPJlW+9rPxcg9sJ11J6U6ofn5Zoeo7MKxXDo7ma0zlgJFZLRT9dq1Gcy9XaDHqFDeGC0xTBGBhmc0TGMLKCur0GeUH31u1k+qjkq30fLxPi7xfO04Rkzwa9yYjek6FU8cztzoQ33\\\\\\\/2saYTd5C+JGnGNuren5VbZoBpPpQnGoboYfBhaFr25y9nxRhxW1mFJLPefb0tC5kvSfw31N6fwtbj+O52qCa3iSbCkTI9CXZsSZTMce8cnM8V18czw2JVbEUAih\\\\\\\/iEJ8HJZZRmSQsCi6MifMrBXjsVK64EbFogk3\\\\\\\/NrjwRHV4uGbFFpjFpfl15TqlOK0QF\\\\\\\/\\\\\\\/zuKz7atBBpZtj7gq3Bq3V1wqbFI6qBnR1p4S2bgi9fj9Sd3oCxEHjtqyUudGNbNHU+PI0wuHqcHgoRon3v7XvqXS\\\\\\\/aRfoMxFguF5ySzATLhGoIHYLSeceHbz0v9n9q9pPVDrS0aVe4SRWdgIh\\\\\\\/7t63Ufv2ZiU4zjSA7V8pMlwGJXKUmuR9OSqpKN16bsTK2TalQ0bW7jfEBtxV6OqHzHa45ZBL2m9R8ie14idR\\\\\\\/F6431PYWeQybnYJ7kLaB0yKQoW1INx1jmCpV22LFqH2rwSrGXWZhCOxWaIbnFzcU0QOcf0\\\\\\\/LKnFFiFOlxykCQ1otbRf6zzjHadUNNdGWuWZz2WfCKzQ7yVgQ6B1775cvmgEQmNyIr84sVN5cARPQBCdDj5p3XgDFz\\\\\\\/1QeIHjrC9SwCTUSua0V+gdRzlU72ABkNSN9TucnL4IWetWnFFpQUK\\\\\\\/+PuBV9FcMS4O4u0rHgeNu9X0lbIrrCkmtkF4LW9jQGtbY4hUd3So4\\\\\\\/s1jN3FTYGEAJSWOZdu8d3HwfL27v4Gho9zx9kLKylZUTTaDd4uqBUjs6RYD8Cx462ODYKEh4AYk+He1f+JQi\\\\\\\/BPzrb8UO5kC1pLOyxOvmTg1CgQZuqoZhOxxOW8TUlnlGtdLWOdZFpreuDyo1Zh7tSnsNvBr4Y3u2pNIEVO0qubJzLOj6GTByqAbRvm19RspnpYbB28nr3DYderAxh3\\\\\\\/uWSeqDhhEaPLaMKfvXJc1qhoU00hcfbZAcKpcTIRW\\\\\\\/+ngWm1vW\\\\\\\/5m79tukuuyO3QkRj58XFibKCDdRw9asmvBos+sw\\\\\\\/lqFWZML\\\\\\\/2LPVU7vXGjBfTz40nbQaSKc9IpmhOTuR4gM+47Nha2xcGaIxeJzPgGgwR40uvXq04ukVGl1Bm7dWJp9meJEwd2yzeJBnUsg1gjrnH5Hfhyn8RRpcdeZlPcBItR4qcq4mTGNPUVoYDwq66xNT\\\\\\\/x1nbVgtE13h0XWT\\\\\\\/YXX\\\\\\\/zttzNeWSJx3oLvCOsgMDXT7Rvfg7JovTCZSj0XMWcx5xq2c\\\\\\\/mGG3rIO91jlppvhUJvUfx0J2eLc80zsStQ8Fjnzt2oWucbkO\\\\\\\/mXVvQNR2Ji21QzQTZ46Sx04JxNFwTyIazUaY7vfaI3I\\\\\\\/T8eA9A2Y3nHLg2L\\\\\\\/xyeKzCkQbZXCOIXY+S8yithuYnUYR1JoRzyZw3v92N58MY8++vP7dS5bO7Feu2nRhCRPVL6MyedqYq2KTxLspDE5yJQGaq2SokUq4zv8x074vmjOkHu2+w83QMEEIGLnnak6UdYsbolKJAxi3nUOfGl0EV1GxqenstocKKlpObkhYMTdctJwd9DjqAwXeL57zS+RbtgH6Q0qHYQ+rdFj5KTKti+2i8KXJ6eRHx7g46yHMmx37mkKPjBNWWS3RDMxWBUHkImqIZH8owffdJbCQZ5Y07MsNQ7aOdZ1e4Qv3VobZIVUshAlHpvF4KpglpZHnM6F5NsqpAFxkBTv2uEduUdAy6C16hZoFzArwR+z7nsA6Ssai5vIwkrAJCmQ3rdTLA11811P4Qa6XTJVmAa+Eydiuy8Dl8UhVULSHVHPVsDkUthxd1TPYaeQ2bjlk8CT5CREGOcdYQf5V\\\\\\\/F8+cW5\\\\\\\/UPCjukT5VuZkh4tpkNKnzzVhMT\\\\\\\/VqtzHIQ6uEdhyyI84rPEMR\\\\\\\/Qwrs+7i+YWSoDtElgsVM4fLGdlgJ1PP7XXz9KJRcc7+GngYgvXW2NsAa9\\\\\\\/KSo7Qlbp4ongKT46N7ZzelhpAhth1QuTUJVsCucEgYwik6qKVKIfP4uXvDJEGlKUX9bXsiPxtY9a3PNSsbncjRNXzAz5SqWbLe1v54De1819SFjXXaTZavIEz2QN76trMdUcJ9z1U64zpxzZY4gTUKVOIqpJHBUgUm4Taun847Es4WH3SN+z3dD+KkbfOUXpKhfOqWF+x9H6+Q8GXoWJjDlSbXwbBRXkoqzx7MnRapuH5j6hylAOEkb9OQPMxsmygkGl4Zc0OI7G5Hq2o5njirRkdTVtzka3dstIuK1Vg29Jgoor+FUEYcESIO62rB7W8j1bdQNgRrSL0kbLUgXOwr23HC5dY+kgAKBnDOVUbGT5r\\\\\\\/KyGKk4yQ4+6WSD\\\\\\\/7hhxrOMK5pgyUr\\\\\\\/U\\\\\\\/+wqb22lU4qNymyWP8N325iow+G0YPoNUYwlQD3TG4S1aoH5dBSVrWo0lnR6ZwhQqz1KEsHQfWtbn122KrrYhBf8beThyDyQaWiMufzUHv8C\\\\\\\/Mt390EbJM9lQcWArmTLF\\\\\\\/LnI3tExV4cF9VkI7eUnKkE7HuMYNZj3wG\\\\\\\/xEJ1eTyEIbMG5MmW4Xb\\\\\\\/YYUBfoudHRx5BZEwKZkpnFDvOeRu1FE8\\\\\\\/UO6PutWi0jI6+mT9Smma6Ty1hbeWZDPwdthr50gHefxbwTSs\\\\\\\/Rpsx5rYhYuB8alK+i7Etn\\\\\\\/Sxb4vUSTJbwMqRzqHosmFinmO9GUi\\\\\\\/ibnqogtjofMNCNv7+MrfjYYJ8DOiuOXmPuVU2iEMs3t0CsdassS0eDI\\\\\\\/itA\\\\\\\/0a1DO6YY+kOnyGl9dwTz9x54vtBCZc08j+406VI\\\\\\\/BLxVZCQM9DXDheythw5bm5v2C+s3MuepM291JNRa2iSuy4PX5foezMbwurZpEkv1Em0Itp\\\\\\\/US\\\\\\\/BCBgF5VkXa58NdT02I47o7yUo2NMTE0WvO\\\\\\\/PKi8BuEoyuVVIRh83xaVfjWgpvv01BfSDEH+lydP8Zs45VNR6cYCUkzT6mbPL5YlDoWBFtgTSGt\\\\\\\/6txeo3oQ2TT2SfKBOz9hev3bomJPg3mqEXZQHeN84mzeY4Ey4iMMWx+DBOTywpS34d0tdgvQJx4+ljO1tBA6qaz2Scp0EhRxqx7Te\\\\\\\/o2Z3Fgw+NeQ1Ie1i8FvqLEXWtSNlnsc5LYlSC+SCqK6Zu0YLNdyYzAA19HTCiI2fhsNqLiG88\\\\\\\/+vxvCOZm8gbNtRTOfEy2AkI4JIFCvOzJqS6oqe+WoNUWEmcyOncx285+dPlcmzqjx1ZX3LBvwXg2+yIn7lsHJQdBYeaKg8A1hCS79t6YMRAaVWh6dfl7I3UZO1tW2q5ryKjY6d+2pj8zoSF8am1XAjkAyko3ToSdk3w4yWvDotHTMJJYT3VvDsNKjarUdDSjbT+UTry5dK8OqItjucVSzvSeGBH7rIBOmafozOmJn3sifyZjUVJjcKAZ8XWV\\\\\\\/mov2AbiAKbaeoEvmp7XeXW\\\\\\\/365CpFM+r0uTn18qK2g43kxj31jEuVxfkyXSoAd7d8q+1RLlSIjWuI2Hb1gx2c0dWODj2MFDgcqMSfPaUFrZs8wVprZ9uQ2M8+SwjoL1zRRFXEZtN8JJSQRGYRXltYCJ4q+sMZq3DXzwTsEExrmSuvy4h9f4\\\\\\\/nq42yiP7NgpUX+afJ45YwSNQbJjssNC4PXx5bZrdbChcuXPh7Nz4Grsh2flVG9G5bWI0cagRTyX1kS5DqkEY4JVZ0R1QdPEDTphhDWiDIPdNLrW5tysb4wlHur0ipOrf+JYIX1Ho0\\\\\\\/6JG9V2Cte51DWG3tROrWQdoyIshV40R\\\\\\\/ai0Qva1FZ6dYZVxnvSQc4yWWT95wPtszsRGbVNcnf3rN4HaJQ1TZzweQA+NbchDKL9abhmTGdDSYwxDcO1LQ\\\\\\\/0T1GlMxpnOdDmKLfHMfwa1Sa1AxKJT+2+JuOG15nL9fA3KxwLUHtSdCpcAQCdMUvApKe3YABkbvugksCeNqbarytx7B00moiap64p0Et290+2hXYI\\\\\\\/xZFNicyig\\\\\\\/oCwZzki1r6JyMXCyiG4StRshNS56CcPJdyUVffPBYL6jbwIlHVcYNJMC+ZQ+i\\\\\\\/kMUn32G1o2bSWXjI0fVcEZhJhoODsOkp2wtdnKs2gVDPwbKkBHo2HWIXa\\\\\\\/4P0DoQueIrBjkQKCJXjkTMS1pCo7G1MdZYzYRKcYbtEVKI4jxFwM1FZeWSMzLh\\\\\\\/c8MYFMt6DGUAFAdKRK0yK29v2Cwm7C\\\\\\\/iuUi1ff4hIY\\\\\\\/lkYtoCtVRctKzH0q8NLYMmSV0izavhQc3BftO07VuKHVNG9PcKBFZ61Mp7BHWiHRuFyEv96GXSp47TkmCtKjWgqpYNBQBy40aNXMH7i\\\\\\\/mLyFf1TsNDScxrEpxp1apmCo4peImjSTj+JdTOLnZhyMoMt6bVUvE\\\\\\\/CoteBcuts9ow5pMBfz3bcr+S7Dxha0lqqxHBOzHqBaaB\\\\\\\/K\\\\\\\/3gD+Ll8bPslEUNcCleJVuxooi\\\\\\\/OI14wU7RFkEDOHl5N7t\\\\\\\/6DqLHmSboQqEhRsVmwsiO45KhyR18iei\\\\\\\/WM5ODpDgU8l8\\\\\\\/szp3tP\\\\\\\/3CnnWPi+F8JFLaaorek0xyJ5xEvKK3lGnHzyfYgekoxSTRX2Rsv\\\\\\\/uqLn8LwDp7j7ED3ZV1Jal1D+P+E+SoLHyagUJo\\\\\\\/t31Gb0kUF95UySFHxUUBh3knmqgfx32OqnRsZLF1EXeodIcpCqvfR+3tPaZSfVQUZche3620mSeGq1JEKhD3UbxzHjKzbdhZPcCHjo1puHWYTZKCjzppxDxdOBaKaF6TVTgOQgfg7U7JxFa1kUBFCHzfuvr2cKhycVEE7X6voGO\\\\\\\/sjp87d4Mn56D\\\\\\\/X1DD7LTRmz+j+LmCFYgAZK9vdTRvwm1vcwOU7IRJmJWh9jePp\\\\\\\/uNRlHmifijKuICAG4BygnFqEjjFh557MrMmZZbNNxxXuZS1bPObMDULXZF7oC8Evd5jq1QecDP4EHz209P1xm+w5boZ7bwkzh3uThH6uq6Kd+ADbwamLTuJQ+6jpihB4kiNrFUoQD9OT09xdaVxM587u3nHAdcVp8gobiCdB7DwiW1EDdpymjNUjvaT80I3Q5FCYQCmJmPtmqW\\\\\\\/VY8TXWxbkOGhLbkk1KEW+RgmzkMTa7lgsvmvlCMP2ucNlV4LLOlENR6tA821Ebd7RdZ9R7IprtX2D6Q7wNftY+YCpczgJ4LRcp5mQJ93pA5BShYui9EUqwIWkTuRcNu\\\\\\\/iXlyXnazXA5HfrxQx0lrM2uQrjn0EZ01E8ytgvqTSnUyOkEvfBRPLJxCpNgKq7Pewymu1uCuTXXle5jNoGBzTenqmVKGkrW2uV2oN+bK2qkgYZiyoitNqRGRDL+qsUSD1bRtwXqklXC5jkQAHBDQElCHv0YR5Rz65hTd2I1o4TbffE+bLM8KjNEv20xcg3LfWVydFD7BX\\\\\\\/a1L5A+umhC1Klnno1h6rmwQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022RpNIckqnECXxMkDV49CbRw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022293fa071a58ca7300a3bd89aca9f08df6f267e5299908b13e9a3cfe59b3b4111\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022nF+tTubfmDVbyTGh6ZzLGR2qQ9IQvnJP33NYr6XkHVXsmBSreBNQpiPpDi5lxtzs6XrHitSn9P9idUxknoUDmfccDklM6YEr22cIt135zaQTXMjX9IMk+HFEwBP1F5guJnyBHHZVAYYG9xkPAjG7xFZGVok3nyzujIKCjRH0R9+gyzAoAQ+OnjwR8jVN2Fajh5mp0w1lBiOUHuLfNifJW4a3SUCXWDTg6fhVqnImc866WfffdFlCGKFPgzMCedNiyj9j0WYKZZ0ZP7XEuHzG2MpA+wkbZhrPW5feGoLfuErY\\\\\\\/cG7SB8rmK2kqSnIPnIgRAanSdgLdwbj9alrMpr9jzp\\\\\\\/kuytBi1ZDn5JdbQj9KK+GW7qWA+bqQ6xkfJzYit7olEdHnysG9oPzHcLFpqLvNuwnTNBgFq202qoEJaVDC75YdGmptMd0wdHgzDAuQ+rJwrU6Ofa0yTK9IImhmnbABa77mG6o599eU\\\\\\\/Lz24kXAr+fZF77\\\\\\\/rRxeNpRi6Q55ZPWiRrdsy7+Me2vrm2jbkbgxprgLRVX0f\\\\\\\/ad7ezEyyV9hgfsemqsEYK\\\\\\\/2BpeJ48CrE8dipkAgNMCxofEqIGsAVKpi8qyV2WWbRyDikBVjZ413ue8jOV\\\\\\\/hwpfVBXoiSJx0bh2zAgENEwbz79Coyz1z2C584wIw9BGmk69Ie3oS9PyvUFNOLTzUBQOJUlpGkPaDi5W7RBnu\\\\\\\/yX0nm8Kv56cc2nE0O9FRPJzarQ51bMHSFv5pPdeXHYN\\\\\\\/m\\\\\\\/8Ct+JL10sliU\\\\\\\/MS9k9wy7bvQeq4DhaIHtrh4gvBhg8\\\\\\\/\\\\\\\/0LF8PGVlTV9Z1c8v8KYf18CGP7V3HirrXd4FUT8jkkWqmHtatVONRg4srv8BDGEEhWxbjiy7YzU5kcriunOq2SsZ8ltqLlPTHqvBWkPXFPLQAdIz9i7w2Z6aaenYPhSpysX6O4Zs+VsQltq6zgqHEQewMTb+rrXb+WS9Ay5aWsJpgN907DQz\\\\\\\/LZolGJ2lZzxOvUdFX02Lezf94DrXZ9zU3bmKoFPiN4Z4KeTSp5KhG50PGia\\\\\\\/3p4MkDI1hGvbWha6chVgI+B8EhrMpvUTK4j41dHecYCiuMbzD1Xa6eR0hi6xxJDJn+MS2SxQbVPesIvRd6xjurn+EhAKVMzEvKSCZgmVcxev87j1SkxymKdHTRKIijW8ApJ2xteSKM3WvZiQU2GrrvXFdVBfKS0ZcJDp5DTk43WrWJYgAAH83SnIIMTtRN3a3ZH3+1UGuLz8LLDT6yZilmR8tGAi2xTOPid2LuT8INJpNvb6ptOyI\\\\\\\/6QK6nJum8UHY2C8TYm5iaCAUcc48zLC9z3sdNVasIEq7zWRtAWJLKwzXPjJiEeZgrzwVCQLBfsCi3HnH4uWnzG4W+YRcp0T5RtRPIGnIdi+c7h7GrJAynG36MYixMZ9mjm+dL3QhgleX0hSvqc5iKMbnlEmiwTJPM3vrl1f1UDN8TMedmb3\\\\\\\/3zvkEsGeOJJNqCRs\\\\\\\/bSHZUgpWaePa2o45IVqegDMXCtB2FY0\\\\\\\/tMdXUFoi8JFxSjVqKygBwKENaKdJQ8sKqolfephr2hxb122f5DffN\\\\\\\/9iEMc+J1KjA6Gw+i5r5IUo1xyl9C5NYn5XALcxdLCyl15XTnpYKAbZSVK1WX7K8ISwOcT6ZjmhIY+ng4z7jMX3nKOfIKhEUpudMbldnmMHK6Qq2zkWcDr\\\\\\\/X3uFyXv6i5r96F3AAAAGjzlvL9PVL0kwEniKQ1jt0Hxxw\\\\\\\/zkc5S4TjeGliLZMnH6AUBN2FXZh8MjMc3wD7SqN15nCb+B\\\\\\\/7ancBzPGDb0n0hvDsBLZny2Q4VbYoLWDFLtV0RiomlwkuuJeNo7L4yOqn8+6BxLhkIlRdyoAw3WI5ZLf4nzlYVSIPpqSX6h\\\\\\\/xtODbRV0I2GiSjfKfp43e3rz6xxyJRlCGZFsWYxdUvV5\\\\\\\/dfepoC1OsyGPlhW1SPwpVrBfENaMCM3ED+pzSuHClNWepCnSCGaLPzouxihz8GQ1vrwJi4qbqZLBsvah7Q==\\\u0022,\\\u0022iv\\\u0022:\\\u0022s6aMugxpP3XFj0PTYVNMQQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228b4d88f7960ba16b87f3bb3141e80d22d98633728bd7f33f4bb7cc176bfe0596\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460567 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/2/413" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-AUJe8MxMr9tKI7vbuT_F-sRqPJTRMhlhkjuliNS6zLU" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/2/413" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Xvxsh_vWVOeIVGjsaolw66un97MPlpQayLiyRYWUErc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/2/399" title="Genetic Causes of Phenotypic Adaptation to the Second Fermentation of Sparkling Wines in Saccharomyces cerevisiae" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/2/427" title="Genomics of Compensatory Adaptation in Experimental Populations of Aspergillus nidulans" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460567" id="node460567" data-pisa="ggg;7/2/413" data-pisa-master="ggg;g3.116.029660" data-apath="/ggg/7/2/413.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Forward Genetics by Sequencing EMS Variation-Induced Inbred Lines</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-0598-5980" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Charles Addo-Quaye</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-1540-8928" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Elizabeth Buescher</span>, <span class="highwire-citation-author" data-delta="2">Norman Best</span>, <span class="highwire-citation-author" data-delta="3">Vijay Chaikam</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-6680-1722" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ivan Baxter</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-2799-954X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Brian P. Dilkes</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">413-425;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.029660 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460567" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Charles Addo-Quaye </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry, Purdue University, West Lafayette, Indiana 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BAddo-Quaye%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Addo-Quaye%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACharles%2BAddo-Quaye%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0598-5980" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Charles Addo-Quaye</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Elizabeth Buescher </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry, Purdue University, West Lafayette, Indiana 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elizabeth%2BBuescher%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Buescher%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AElizabeth%2BBuescher%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1540-8928" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Elizabeth Buescher</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Norman Best </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry, Purdue University, West Lafayette, Indiana 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Norman%2BBest%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Best%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANorman%2BBest%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Vijay Chaikam </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry, Purdue University, West Lafayette, Indiana 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vijay%2BChaikam%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chaikam%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVijay%2BChaikam%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Ivan Baxter </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>United States Department of Agriculture-Agricultural Research Service, Donald Danforth Plant Science Center, St. Louis, Missouri 63132</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ivan%2BBaxter%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baxter%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AIvan%2BBaxter%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6680-1722" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ivan Baxter</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Brian P. Dilkes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry, Purdue University, West Lafayette, Indiana 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brian%2BP.%2BDilkes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dilkes%20BP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABrian%2BP.%2BDilkes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-2799-954X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Brian P. Dilkes</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bdilkes@purdue.edu" class="" data-icon-position="" data-hide-link-title="0">bdilkes@purdue.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/2/413" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460567" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460567/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/2/413.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460567" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460567/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/2/413.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460567" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460567/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/2/413.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460567" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460567/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/2/413.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460567" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460567/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">In order to leverage novel sequencing techniques for cloning genes in eukaryotic organisms with complex genomes, the false positive rate of variant discovery must be controlled for by experimental design and informatics. We sequenced five lines from three pedigrees of ethyl methanesulfonate (EMS)-mutagenized <em>Sorghum bicolor</em>, including a pedigree segregating a recessive dwarf mutant. Comparing the sequences of the lines, we were able to identify and eliminate error-prone positions. One genomic region contained EMS mutant alleles in dwarfs that were homozygous reference sequences in wild-type siblings and heterozygous in segregating families. This region contained a single nonsynonymous change that cosegregated with dwarfism in a validation population and caused a premature stop codon in the <em>Sorghum</em> ortholog encoding the gibberellic acid (GA) biosynthetic enzyme <em>ent</em>-kaurene oxidase. Application of exogenous GA rescued the mutant phenotype. Our method for mapping did not require outcrossing and introduced no segregation variance. This enables work when line crossing is complicated by life history, permitting gene discovery outside of genetic models. This inverts the historical approach of first using recombination to define a locus and then sequencing genes. Our formally identical approach first sequences all the genes and then seeks cosegregation with the trait. Mutagenized lines lacking obvious phenotypic alterations are available for an extension of this approach: mapping with a known marker set in a line that is phenotypically identical to starting material for EMS mutant generation.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/snp" class="hw-term hw-article-keyword hw-article-keyword-snp" rel="nofollow">SNP</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/sorghum" class="hw-term hw-article-keyword hw-article-keyword-sorghum" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Sorghum</em></a></li><li class="kwd"><a href="/keyword/gibberellin" class="hw-term hw-article-keyword hw-article-keyword-gibberellin" rel="nofollow">gibberellin</a></li><li class="kwd"><a href="/keyword/mapping" class="hw-term hw-article-keyword hw-article-keyword-mapping" rel="nofollow">mapping</a></li><li class="kwd"><a href="/keyword/mutant" class="hw-term hw-article-keyword hw-article-keyword-mutant" rel="nofollow">mutant</a></li></ul><p id="p-5">One of the most effective and convincing methods to demonstrate gene function is the molecular identification of allelic variation responsible for induced mutant phenotypes. This gene function discovery approach, originally known as genetics and now sometimes referred to as “forward genetics,” has dramatically expanded our understanding of development, physiology, and biochemistry over the last century (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Rice 2014</a>). Much of this understanding has been generated in genetic model systems, which are characterized by simple laboratory culture, short generation times, and small genome sizes, but limited in their physiology, anatomy, and biochemistry (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hunter 2008</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Trontin <em>et al.</em> 2011</a>). Historically, identification of the genes responsible for mutant phenotypes was accomplished by map-based cloning. Traditional map-based cloning relies on outcrossing, which introduces phenotypic diversity that can confound mutant scoring. Mutant phenotypes with dramatic differences and insensitive to variation introduced by the crossing partner are the most amenable to map-based cloning approaches. Linkage of the mutant phenotype to a chromosomal region is used to generate a list of candidate causative genes. Recombinant genotypes sufficient to recapitulate the mutant phenotype are then used to narrow the candidate gene list. Mapping genetic loci with sufficient precision to identify a single locus requires ruling out polymorphisms introduced by the outcrossing partner and sequencing a limited number of candidates to identify a single remaining causative polymorphism.</p><p id="p-6">While the number of polymorphisms in a mutagenized genome may scale with the overall genome size, the number of genome positions capable of affecting a change in a phenotype does not. EMS mutagenesis induces a few thousand polymorphisms per individual; a relatively small number when compared to the hundreds of thousands that differ between diverse lines. Unlike insertion mutagenesis (<em>e.g.</em>, transposons or T-DNA) or natural variation, chemical mutagens induce changes in phenotype by modifying protein-coding capacity and only very rarely are noncoding sequence mutations <em>in cis</em>-regulatory sequences identified as causative mutations. Essentially, a genome’s coding capacity determines the phenotypically affective mutational target. For example, in a genome with 30 Mbp of protein-coding sequence, a 1/125,000 bp mutation rate would correspond to ∼240 genic mutations. As approximately one-third of all EMS alleles result in synonymous changes (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Ashelford <em>et al.</em> 2011</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Henry <em>et al.</em> 2014</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Thompson <em>et al.</em> 2013</a>) this would mean 160 potentially causative nonsynonymous mutations per line. EMS is an effective way to induce mutations in protein-coding sequence, which results in an observable phenotype.</p><p id="p-7">Most applications of next-generation sequencing (NGS) for gene identification by forward genetics utilize large segregating populations to clone genes by bulked segregant analysis (BSA; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Michelmore <em>et al.</em> 1991</a>). This has been demonstrated in a variety of species including <em>Arabidopsis</em>, Tef (<em>Eragrostis tef</em>), rice, zebrafish, <em>Sorghum</em>, and <em>Drosophila</em> (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abe <em>et al.</em> 2012</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Austin <em>et al.</em> 2011</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Lindner <em>et al.</em> 2012</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Mokry <em>et al.</em> 2011</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Schneeberger <em>et al.</em> 2009</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Uchida <em>et al.</em> 2011</a>; Q. Zhu <em>et al.</em> 2012; Y. Zhu <em>et al.</em> 2012; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Rizal <em>et al.</em> 2015</a>). In organisms in which extensive pedigree data are often kept, as is the case for many crops and model systems, subtraction of preexisting variation and variants present in unaffected siblings would allow the rapid elimination of large genome sections as possible sites for mutant loci. In self-fertile organisms, recurrently selecting heterozygous individuals and selfing them to make advanced single lineage families can drive all loci but the mutant locus to homozygosity. This creates affected and unaffected siblings that vary for the single causative polymorphism only. Such an approach leverages the polymorphism discovery by NGS and the use of pedigree information to identify the causative polymorphism.</p><p id="p-8"><em>Sorghum</em> is a self-fertile diploid panicoid grass that includes crops for food, forage, and biomass products. The adaptation of <em>Sorghum</em> to hot and arid environments makes improved <em>Sorghum</em> a major contributor to global food security. The <em>Sorghum</em> genome of accession BTx623 was sequenced (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Paterson <em>et al.</em> 2009</a>). Substantial genetic variation in <em>Sorghum</em> accessions has been described (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Casa <em>et al.</em> 2008</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hamblin <em>et al.</em> 2004</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Mace <em>et al.</em> 2013</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Nelson <em>et al.</em> 2011</a>). A number of induced variation collections have also been generated (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Blomstedt <em>et al.</em> 2012</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Xin <em>et al.</em> 2008</a>). The fully sequenced genome, diploid inheritance, moderate genome size, and availability of induced mutants makes <em>Sorghum</em> an excellent system to demonstrate mutant gene identification in complex eukaryotes and organisms of human importance. This would expand the reach of these advanced applications of sequencing and mutagenesis technologies beyond genetic model systems.</p><p id="p-9">Next-generation, short-read sequencing technologies are error-prone and this can produce a number of artifacts. These errors, when summed over the length of a complex eukaryotic genome, can result in incorrect inference. Controlling the sources of these errors in the analysis of whole-genome sequencing results may lead to better gene mapping and cloning procedures. Some of these errors result from the inherent error rate in the sequencing technology (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Minoche <em>et al.</em> 2011</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Quail <em>et al.</em> 2008</a>, <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">2012</a>) and these problems can be overcome by sequencing genomes at greater depth. Other artifacts differentially affect specific positions making them error-prone, including alignment errors at insertion or deletion alleles (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">R. Li <em>et al.</em> 2009</a>), SNPs resulting from mistakes in the reference assembly, variation in research material relative to the reference assembly, poor alignment performance (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Cheng <em>et al.</em> 2014</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Fredman <em>et al.</em> 2004</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">O’Rawe <em>et al.</em> 2013</a>), alignment to repetitive or paralogous sequences (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cheung <em>et al.</em> 2003</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Estivill <em>et al.</em> 2002</a>), and the alignment of sequence from DNA encoded by unassembled genomic regions to an incomplete digital reference genome (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Teo <em>et al.</em> 2012</a>). When mutagenesis only induces a few thousand true changes to the genome, the number of false positives can be far greater than the number of mutations. Methods to remove these errors are needed to make gene identification from mutants of complex eukaryotes become routine and reliable.</p><p id="p-10">In this work, we compared the results of whole-genome, short-read sequencing of a <em>Sorghum</em> dwarf mutant to similar data from independently-derived EMS-mutagenized lines. By collecting whole-genome sequence data from two phenotypically unaffected individuals in an EMS population and using the data from a previously published unrelated mutant in <em>dhurrinase2</em> (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Krothapalli <em>et al.</em> 2013</a>), we were able to identify nucleotide positions in the <em>Sorghum</em> genome that resulted in SNP calls in more than one lineage. These positions cannot be the causative mutation for any phenotypes unique to one lineage. Initial SNP calls were not consistent with the result of guanine residue alkylation, the mechanism of EMS mutagenesis. After removal of the shared alleles identified between the three mutants and considering only the coding sequence SNPs, the proportion of G:C to A:T changes, the expected changes from EMS treatment, ranged from 79 to 91% per line. We measured allele frequencies of these SNPs in sequence data from wild-type, heterozygous and, dwarf siblings. The vast majority of these SNPs displayed allele frequencies incongruent with the mutant allele status of the line. Allele frequencies for one region on chromosome 10 segregated with the mutant phenotype, identifying this region as encoding the dwarf mutant. One, and only one, coding sequence change was present in the delimited interval in the dwarf, missing from the wild-type sibling, and heterozygous in the heterozygote sibling’s genome. This polymorphism introduces a premature stop codon in the <em>ent</em>-kaurene oxidase gene of <em>Sorghum</em> (Sobic.010G172700), encoding a putative enzyme in gibberellin biosynthesis. The mutation cosegregated with the dwarf phenotype, which was reversed by GA application, and confirmed the causative nature of this polymorphism. Using EMS-induced SNP variation as molecular markers permits any phenotypically unaffected line to act as an EMS Variation-Induced Lines (EVIL) twin for mapping crosses without introducing the segregation variation of a distinct and diverse genetic background. The use of preexisting data to remove error-prone positions to improve SNP calling and EVIL twins for mapping should improve both the accuracy and sensitivity of forward genetics.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Plant material</h3><p id="p-11">All plant material was grown in the Purdue Horticulture and Landscape Architecture greenhouse in an equal mixture of Turface (Profile Products LLC, Buffalo Grove, IL), potting soil (Conrad Fafard Inc., Agawam, MA), and sand (U.S. Silica, Frederick, MD). For NGS, whole leaves were collected ∼1 wk after germination for a pool of 17 10-2 individuals, in which all individuals planted were of normal size, and 10-dwarf individuals, in which leaves from all dwarfs germinated (13 dwarf individuals) across all individuals were collected and pooled. DNA genotyping leaf tissue was collected ∼6 wk after germination from multiple individuals. DNA was isolated from leaves collected from 12 tall individuals in the 10-3 family, which segregated for tall and dwarf individuals. Leaves from normal-height 12-2 individuals were collected from a population segregating for a dominant tall variant (data not shown). The <em>dhr2-1</em> library was previously described and generated from a single individual (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Krothapalli <em>et al.</em> 2013</a>).</p></div><div id="sec-3" class="subsection"><h3>Cosegregation test for the causative dwarf mutation</h3><p id="p-12">A dCAPS (Derived Cleaved Amplified Polymorphic Sequences; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Neff <em>et al.</em> 2002</a>) was designed for the genomic position 50,477,994 on chromosome 10 using the Washington University in St. Louis dCAPS interface (<a href="http://helix.wustl.edu/dcaps/">http://helix.wustl.edu/dcaps/</a>). Primers for PCR amplification were: dCAPs_BstN1-F (5′-TGTGGAAGAGTATGGGAAGGTT-3′) and dCAPs_BstN1-R (5′-CCTCTCCACGTGCAATTCTT-3′). Post PCR-amplification, 10 μl of the PCR product was digested using five units of <em>Bst</em>N1 (New England Biolabs, Catalog #R0168S) and 1 × NEB Buffer 3.1 in a 25 μl total reaction. Digestions were carried out overnight at room temperature and examined using a 2% TAE agarose gel. <em>Bst</em>N1 digestions only occurred if the PCR amplicon was wild-type for the causative SNP, <em>i.e.</em>, were 44 bases smaller than the uncut amplicon for individuals with the mutation. Individuals were either wild-type (183 bp fragment), dwarf (227 bp fragment), or heterozygous (both 183 and 227 bp fragments).</p></div><div id="sec-4" class="subsection"><h3>NGS</h3><p id="p-13">We generated whole-genome sequencing data for the five EMS-treated <em>Sorghum</em> individuals using Illumina sequencing technology. The individuals 12-2 and 10-3 were sequenced using an Illumina GAIIx instrument, while the 10-2 and 10-d individuals were sequenced using an Illumina HiSeq instrument. Sequenced reads from the 12-2 sample were 80 bases long and single-ends, while the remaining sequenced samples were 100 bases long and paired-ends.</p></div><div id="sec-5" class="subsection"><h3>Sequenced reads mapping</h3><p id="p-14">We used BWA version 0.6.2 (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Li and Durbin 2009</a>) to align the NGS data to the BTx623 <em>Sorghum</em> reference genome sequence (version 2.1; <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Paterson <em>et al.</em> 2009</a>), which was downloaded from Phytozome (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Goodstein <em>et al.</em> 2012</a>). Suffix array coordinates for each query sequence were generated using the BWA <em>aln</em> command with nondefault parameters: “-t 8.” Paired-end alignments were generated using the BWA <em>sampe</em> command with nondefault parameters: “-P -r “@RG\tID:SampleID\tSM:SampleName\tPL:Illumina”.” In the case of the single-end reads, sequence alignments were produced using the BWA <em>samse</em> command. Sequencing data and mapping statistics were obtained using the SAMtools <em>flagstat</em> command with default parameters. Genome coverage statistics were generated from the sequence alignment files using a combination of SAMtools and BEDTools (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Quinlan and Hall 2010</a>). We used the SAMtools <em>view</em> command to uncompressed BAM files and sent the results to standard output. We then used the BEDTools <em>genomeCoverageBed</em> command to estimate the coverage depth from the uncompressed alignment BAM output. The SAMtools <em>view</em> command parameter options were “-u -q 0,” while the BEDTools <em>genomeCoverageBed</em> command parameter options were “-ibam stdin –g.”</p></div><div id="sec-6" class="subsection"><h3>SNP calling and quality filtering</h3><p id="p-15">SNP detection was performed using the <em>mpileup</em> command in the SAMtools package version 0.1.18 (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">H. Li <em>et al.</em> 2009</a>). The nondefault options invoked were “-B -Q 20 -P Illumina -C50 -uf” and the result was piped to the BCFtools <em>view</em> command with nondefault parameters: “-vcg.” These settings require bases to have a quality score of 20 or higher, downgrade the consideration of base calls from poorly mapped reads, and disable the base-alignment quality scoring function in SAMtools. Further quality filtering was performed as follows. Removal of SNPs residing in repetitive regions was performed using the <em>varFilter</em> command within <em>perl</em> script <em>vcfutils.pl varFilter</em> with options “-D100” to remove SNPs with coverage depth &gt;100 or &lt;2. Other default settings for this command also removed SNPs with root mean square quality of &lt;10 and SNPs found within three bases of a gap (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">R. Li <em>et al.</em> 2009</a>). All nucleotide positions for which more than one polymorphism was noted were removed using a custom <em>awk</em> script leaving only biallelic positions. Using the SnpSift program (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Cingolani <em>et al.</em> 2012a</a>), additional filters were applied. Only homozygous variant positions with a minimum SNP quality of 20 were retained. We required no reference base reads and at least one observation of the SNP base call from each strand. EMS mutations result in G:C to A:T mutations, whereas false positives could be any change. Thus, we retained only the alleles that corresponded to G:C to A:T mutations using SnpSift (see Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029660/-/DC1/FileS1.zip">File S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029660/-/DC1/FileS2.zip">File S2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029660/-/DC1/FileS3.zip">File S3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029660/-/DC1/FileS4.zip">FileS4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029660/-/DC1/FileS5.zip">File S5</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029660/-/DC1/FileS9.zip">File S9</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029660/-/DC1/FileS10.zip">File S10</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029660/-/DC1/FileS11.zip">File S11</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029660/-/DC1/FileS12.zip">File S12</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029660/-/DC1/FileS13.zip">File S13</a> for details).</p></div><div id="sec-7" class="subsection"><h3>Detection of false positive shared SNPs</h3><p id="p-16">Since the three individual plants (10, 12-2, and <em>dhr2-1</em>) were obtained from seeds, which were independently treated with EMS, we subtracted all overlapping high-quality SNPs detected in the progenies of these individuals. To remove false positive shared SNPs, we searched for genomic position overlaps in the high-quality homozygous G to A, and C to T SNP positions for the five EMS-mutant individuals. For each of the 10-derived individuals (10-2, 10-3, and 10-d) we subtracted the overlaps with the <em>dhr2-1</em> and 12-2 individuals. Similarly, we subtracted SNPs in <em>dhr2-1</em> with genomic positions overlapping SNP positions in 10-2, 10-3, 10-d, and 12-2. The process was repeated for 12-2 by subtracting SNPs with positions overlapping 10-2, 10-3, 10-d, and <em>dhr2-1</em> SNP positions. We used the BEDTools <em>intersect</em> command (version 2.17.0) with options “-wa” to find common SNP variations detected in different samples. Similarly, we used the BEDTools <em>subtract</em> command with options “–A” to detect pedigree- and sample-specific SNPs for each of the samples.</p></div><div id="sec-8" class="subsection"><h3>Allele frequency determination in EMS variation-induced lines</h3><p id="p-17">For each of the five individuals, we calculated the allele frequencies at all genomic positions corresponding to called variants in each of the other four individuals. To do this, we reran the SAMtools <em>mpileup</em> command and generated a new version of the variant call output files by modifying the BCFtools <em>view</em> command nondefault parameters to “-Ncg.” The BCFtools <em>view</em> command parameter changes resulted in a new variant call output file that contained genotype calls for both variant and not variant sites, in which the genome reference nucleotide base was known. We used the BEDTools <em>intersect</em> command, and the coding region information specified in the <em>Sorghum</em> genome annotation “.gff3” file, to derive a second set of files containing only the coding region allele information. We then used both the genome-wide and coding region files for each individual and calculated allele frequencies. We subtracted SNPs shared among the members of the 10 pedigree. The BEDTools <em>subtract</em> command with options “–A” was used to detect the subset of high-quality homozygous SNPs that were unique to 10-d and absent in 10-2 and 10-3. These positions were examined in the allele frequency plots to find SNPs that cosegregated with the dwarf phenotype.</p></div><div id="sec-9" class="subsection"><h3>Functional annotation of SNP variation</h3><p id="p-18">We used the version 3.1 of the SnpEff program (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Cingolani <em>et al.</em> 2012b</a>) to predict the effect of the SNPs on <em>Sorghum</em> gene function. We downloaded <em>Sorghum</em> version 2.1 whole-genome, proteome, and coding sequences, and the genome annotation “.gff3” files from Phytozome (version 9.1; <a href="http://www.phytozome.net">www.phytozome.net</a>; <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Goodstein <em>et al.</em> 2012</a>). These files were used to generate a custom SnpEff database using the SnpEff <em>build</em> command with parameter options “build -gff3.” This required editing the “snpEff.config” file to create an entry for <em>Sorghum</em> version 2.1. The SnpEff-annotated variant call file was generated using the SnpEff <em>eff</em> command with parameter option “-c snpEff.config.” SNPs that altered the amino acid coding capacity of genes were classified as “MODERATE” if they were a nonsynonymous change or “HIGH” if they resulted in a nonsense codon, splice loss, or loss of the start codon.</p><p id="p-19">Functional information for the genes affected by these SNPs was obtained as follows. We downloaded the latest proteome sequences and the corresponding annotation files for <em>Sorghum</em> BTx623 (version 2.1), maize B73 (version 5b.60), and <em>Arabidopsis thaliana</em> (TAIR release 10) from the Phytozome web portal. We created BLAST proteome databases for each of the three plant species using the <em>makeblastdb</em> program with parameter options “-input_type fasta -dbtype prot.” We used the <em>blastp</em> program (version 2.2.28+) in the BLAST package (Altschul <em>et al.</em> 1990) to align the <em>Sorghum</em> proteome to maize and <em>Arabidopsis</em>. The following <em>blastp</em> nondefault parameter options were selected: “-evalue 1E-05 -num_threads 32 -max_target_seqs 5 -out -outfmt 6 -seg yes.” The sequence annotation for the detected maize and <em>Arabidopsis</em> homologs of the <em>Sorghum</em> genes were appended to the SNP call “.vcf” files, using a custom program (appended as <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029660/-/DC1/FileS6.txt">File S6</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029660/-/DC1/FileS7.txt">File S7</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029660/-/DC1/FileS8.txt">File S8</a>).</p></div><div id="sec-10" class="subsection"><h3>Dwarf mutant complementation with GA<sub>3</sub> in the dark</h3><p id="p-20">An M2 segregating population of mutant and wild-type siblings was grown in the dark for 7 d at 25° with 90% humidity in a Conviron E8 growth chamber (Conviron, Pembina, ND). Seedlings were planted in 72-cell germination flats in a 1:1 mixture of peat germination mix (Conrad Fafard Inc.,) and Turface (Profile Products LLC). Seedlings were allowed to germinate for 2 d (before emergence) and then were treated with 1.75 L per flat of 0.2% ethanol and 0.005% silwet (“no treatment”), or 10 µM GA<sub>3</sub> (Gold Biotechnology, St. Louis, MO), 0.2% ethanol, and 0.005% silwet (“10 µM GA3 treatment”). Seedlings were harvested and photographed after 7 d total in the dark. Mesocotyl length was measured from the root-to-shoot transition zone to the proximal end of the coleoptile using ImageJ (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Abramoff <em>et al.</em> 2004</a>).</p></div><div id="sec-11" class="subsection"><h3>Dwarf mutant complementation with GA<sub>3</sub> in the light</h3><p id="p-21">An M2 segregating population of mutant and wild-type siblings was grown in the light for 14 d under greenhouse conditions: 27° (day) and 21° (night) with 16 h supplemental lighting. Seedlings were grown in media as previously stated. Seedlings were allowed to germinate for 2 d then treated with 1 L per flat of 0.2% ethanol and 0.005% silwet (“no treatment”), or 10 µM GA3, 0.2% ethanol, and 0.005% silwet (“10 µM GA3 treatment”), every third day until harvest. Seedlings were harvested and photographed after 14 d in total. Plant height was measured from the distal end of the mesocotyl to the first leaf collar using ImageJ (Abramoff <em>et al.</em> 2012).</p></div><div id="sec-12" class="subsection"><h3>Data availability</h3><p id="p-22">The Illumina NGS datasets associated with this project have been submitted to the NCBI Sequence Reads Archive. The NCBI SRA project ID is SRP077001 and the BioProject ID number is PRJNA326607. The NCBI BioSample accessions for the individual samples are SAMN05290415, SAMN05290416, SAMN05290417, SAMN05290418, and SAMN05290419. </p></div></div><div class="section results" id="sec-13"><h2 class="">Results</h2><div id="sec-14" class="subsection"><h3>Observation of a dwarf mutant segregating in an M4 generation</h3><p id="p-23"><em>Sorghum</em> BTx623 seeds were mutagenized in a prior study and the resulting offspring selfed for three generations to generate M4 individuals (<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Xin <em>et al.</em> 2008</a>). We screened a limited number of families for aphenotypic EMS-treated derivatives. Two M4 individuals from independent lineages, lines 10 and 12, were selected because they had no obvious mutant phenotypes. When these lines were selfed, the progeny of line 12 were all normal while the M5 progeny of line 10 segregated for extreme dwarfism with reduced leaf lengths and compact internodes (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). Multiple tall M5 siblings were self-pollinated and M6 families were grown and scored for the dwarf phenotype. Dwarfism segregated as a monogenic recessive trait. One M6 family, the selfed progeny of M5 individual 10-2, was replanted and determined to be a homozygous wild-type derivative of line 10. Another M6 family, the selfed progeny of M5 individual 10-3, segregated 3:1 for tall:dwarf and indicated that individual 10-3 was a heterozygote. <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a> shows the pedigree and dwarf phenotype expression of the sequenced lines. We used these materials to test our approach to clone EMS-induced mutants by using all of the EMS-induced variants as molecular markers and to build the bioinformatic analysis pipeline necessary to link NGS and genetic mapping.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/413/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) BTx623 (wild-type) and 10-d mutant plants, exhibiting a dwarf phenotype. (B) Pedigree of the sequenced EMS-treated lines. Independent mutagenesis resulted in the dhr2-1 mutant, the 12-2 line, and the 10 line. Selfing of 10 resulted in sublines 10-2, 10-3, and 10-d. Lines sequenced and represented in this manuscript are indicated in tan. EMS, ethyl methanesulfonate." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-630627784" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(A) BTx623 (wild-type) and 10-d mutant plants, exhibiting a dwarf phenotype. (B) Pedigree of the sequenced EMS-treated lines. Independent mutagenesis resulted in the &lt;em&gt;dhr2-1&lt;/em&gt; mutant, the 12-2 line, and the 10 line. Selfing of 10 resulted in sublines 10-2, 10-3, and 10-d. Lines sequenced and represented in this manuscript are indicated in tan. EMS, ethyl methanesulfonate.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/413/F1.medium.gif" width="440" height="193"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/2/413/F1.medium.gif" width="440" height="193"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/413/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/413/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470355" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-24" class="first-child">(A) BTx623 (wild-type) and 10-d mutant plants, exhibiting a dwarf phenotype. (B) Pedigree of the sequenced EMS-treated lines. Independent mutagenesis resulted in the <em>dhr2-1</em> mutant, the 12-2 line, and the 10 line. Selfing of 10 resulted in sublines 10-2, 10-3, and 10-d. Lines sequenced and represented in this manuscript are indicated in tan. EMS, ethyl methanesulfonate.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Sequence data generation and read mapping</h3><p id="p-25">We sequenced individuals derived from line 10 based on their segregation for the extreme dwarf phenotype. DNA was isolated from a pool of dwarf mutant tissue designated 10-d, a pooled sample of tall heterozygous siblings (the M5 10-3 individual), and a pool of homozygous wild-type siblings from line 10-2. The DNA from each sample was sequenced. In addition, we sequenced another line, 12-2, which showed no gross morphological defects and was independently derived from the same mutagenesis experiment as line 10 (<a id="xref-ref-53-3" class="xref-bibr" href="#ref-53">Xin <em>et al.</em> 2008</a>). Finally, we reanalyzed the sequencing data used to clone the <em>dhurrinase2</em> (<em>dhr2</em>-1) EMS-induced <em>Sorghum</em> mutant, which was found in a different mutant population (<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Krothapalli <em>et al.</em> 2013</a>). A total of 1,036,765,083 Illumina NGS reads were generated from DNA samples derived from the five <em>Sorghum</em> EMS-mutant individuals (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Using the BWA short-read aligner (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Li and Durbin 2009</a>), 91% (942,006,177) of the sequenced reads mapped to the reference BTx623 <em>Sorghum</em> genome (<a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Paterson <em>et al.</em> 2009</a>). Using the alignment results, the estimated median paired-end insert sizes for the 10-2, 10-3, 10-d, and <em>dhr2-1</em> sequenced reads were 249, 307, 260, and 297, respectively. The alignment results also showed that 88% (889,860,334) of the paired-end reads were properly paired within the range of the above insert sizes. <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> shows the summary statistics for the sequencing and alignment to the <em>Sorghum</em> genome assembly version 2.1 available at Phytozome (<a href="http://www.phytozome.net">www.phytozome.net</a>). Between 88 and 94% of the reads mapped to the reference genome. The coverage depths for the homozygous aphenotypic line 10-2, the previously described <em>dhr2-1</em>mutant, and the extreme dwarf 10-d were 45-fold, 43-fold, and 30-fold genome coverage, respectively (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). Two other individuals were sequenced at lower depths. The aphenotypic 12-2 line and the heterozygous sibling 10-3 individuals were sequenced at threefold and sevenfold genome coverage, respectively (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470472/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470472/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Summary of whole-genome sequencing and read mapping for five <em>Sorghum</em> EMS-treated lines</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>SNP detection</h3><p id="p-26">We detected SNPs according to a procedure outlined in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> and described in detail in the <em>Materials and Methods</em> section. In short, alignments were scored for likely deviation from the reference sequence and then sequentially filtered for coverage, homozygosity, and a requirement that reads map to both strands and sequence data quality. Following this, SNPs were further filtered to remove all variants unlikely to be obtained by EMS treatment, which primarily induces G to A mutations (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Greene <em>et al.</em> 2003</a>) that also appear in sequence data as the reverse complement, C to T. <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> summarizes the SNP detection results and filtering steps. We obtained a total of 920,834 likely deviations from the reference sequence in initial SNP calling for the five samples using the SAMtools SNP calling package (<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Li and Durbin 2009</a>). Discarding low-quality SNPs and SNPs of repetitive origin reduced the total by 59% to 376,533. Due to the low coverage and depth of sequencing data from the 12-2 sample, an unusually high number of sites initially scored as variants but derived from singleton reads were detected. These positions were efficiently removed by our procedure. Further filtering to retain only SNPs likely to be homozygous with a minimum Phred quality score of 20 retained 125,308 positions, removing 27% of the initial SNP calls. If we further require all reads overlapping a SNP site to contain the mutant allele and for reads to derive from both forward and reverse DNA strands, only 71,058 high-quality homozygous SNPs remain from the five lines. EMS-induced alleles are almost exclusively G:C to A:T transitions unlike background mutations, which could be any of the eight possible sequence changes. Removing all but the high-quality G:C to A:T SNPs removed 51% of the remaining SNPs. The count of likely EMS-induced changes (high-quality G:C to A:T) ranged from 9668 changes in 10-2 to only 2635 in the low coverage sequence data from 12-2 (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). An average of 6914 high-quality G:C to A:T SNPs were identified per sample.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/413/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Schematic overview of the SNP detection and annotation pipeline for EMS mutant discovery. EMS, ethyl methanesulfonate; NGS, next-generation sequencing; SNP, single nucleotide polymorphism." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-630627784" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Schematic overview of the SNP detection and annotation pipeline for EMS mutant discovery. EMS, ethyl methanesulfonate; NGS, next-generation sequencing; SNP, single nucleotide polymorphism.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/413/F2.medium.gif" width="440" height="325"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/2/413/F2.medium.gif" width="440" height="325"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/413/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/413/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470265" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-27" class="first-child">Schematic overview of the SNP detection and annotation pipeline for EMS mutant discovery. EMS, ethyl methanesulfonate; NGS, next-generation sequencing; SNP, single nucleotide polymorphism.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470514/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470514/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Summary of SNP counts during sequential application of quality filters for five EMS-treated <em>Sorghum</em> lines</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h3>Independent mutagenesis as biological replication for the experimental identification of false positive SNPs</h3><p id="p-29">SNP calling is a statistical likelihood procedure (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Nielsen <em>et al.</em> 2011</a>) for each nucleotide position in the genome. As a result, the number of false positives will scale with the overall genome size. Moreover, chemically mutagenized lines have a low true positive rate (1/125,000–1/500,000 bp) resulting in a relatively higher proportion of false positives. This is in stark contrast to the use of SNP calling for distinguishing natural populations, in which multiple orders of magnitude more true positives are present. We initially attempted to score each SNP as a molecular marker in our sequencing data and found that many adjacent neighboring SNPs were not linked in the three fully sequenced individuals that shared the “10” pedigree. This violates the predicted inheritance of these chromosomes and DNA segments (data not shown). We hypothesized that a high ratio of false positives in the SNP calling procedure was interfering with mapping and cosegregation. To detect positions in the genome that were false positive SNP calls, we identified genomic positions that were scored as variant in more than one independently mutagenized line. We compared SNP calls from the line 10 derivatives (10-2, 10-3, and 10-d), the aphenotypic EMS-treated line 12 (12-2), and the previously published <em>dhr2-1</em> mutant (<a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">Krothapalli <em>et al.</em> 2013</a>). SNPs shared between independent lineages are highly unlikely to be due to independently mutagenized chromosomes. Far more likely, any SNP variation shared between the 10, 12, and <em>dhr2-1</em> pedigrees would be caused by systematic bias in the generation and analysis of sequencing data. These shared SNPs should detect (1) errors in the reference sequence, (2) differences between the mutagenized material and the originally sequenced line, and (3) positions in the genome that frequently return a likely SNP call due to structure or paralogy. Furthermore, only those SNPs present in the 10 lines and not in 12-2 or <em>dhr2-1</em> can be causative for the extreme dwarf locus segregating in that lineage. <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3, A–C</a> displays Venn diagrams showing the overlap in SNP positions between each of the sequenced line 10 derivatives with the independently mutagenized 12-2 and <em>dhr2-1</em> lines. In each case, a large number of SNPs were identified in multiple lineages, indicating that these SNPs are unlikely to be the result of EMS mutagenesis and cannot encode the causative alleles for the extreme dwarf phenotype in line 10. The majority of the shared SNP variation was detected in pairwise comparisons. Only a minority of differences was found when comparing the three lineages of line 10, which indicates that the majority of these shared SNPs are not the result of base calling errors in the reference sequence. <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> provides the filtered common SNP variations for individual samples considering either genome-wide SNP data or only SNPs in coding sequences.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/413/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genome-wide comparison of putative EMS-induced SNPs between pedigrees. Venn diagrams display the number of shared G to A, or C to T SNP positions found in lines 12-2, dhr2-1, and (A) 10-2, (B) 10-3, or (C) 10-d. EMS, ethyl methanesulfonate; SNP, single nucleotide polymorphism." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-630627784" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Genome-wide comparison of putative EMS-induced SNPs between pedigrees. Venn diagrams display the number of shared G to A, or C to T SNP positions found in lines 12-2, dhr2-1, and (A) 10-2, (B) 10-3, or (C) 10-d. EMS, ethyl methanesulfonate; SNP, single nucleotide polymorphism.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/413/F3.medium.gif" width="440" height="357"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/2/413/F3.medium.gif" width="440" height="357"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/413/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/413/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470230" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-30" class="first-child">Genome-wide comparison of putative EMS-induced SNPs between pedigrees. Venn diagrams display the number of shared G to A, or C to T SNP positions found in lines 12-2, dhr2-1, and (A) 10-2, (B) 10-3, or (C) 10-d. EMS, ethyl methanesulfonate; SNP, single nucleotide polymorphism.</p><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470478/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470478/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Summary of likely EMS-induced SNPs</span><div class="sb-div caption-clear"></div></div></div><p id="p-32">The subtraction of common variants selectively enriched for EMS-induced G:C to A:T SNPs. In the initial SNP calling, only 20% of the SNP calls were G:C to A:T transitions. After quality filtration (previous section; <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>), 49% of the high-quality SNPs were G:C to A:T variants (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029660/-/DC1/TableS1.pdf">Table S1</a>). Detection and removal of the SNPs shared between pedigrees resulted in further enrichment, with 79% of all lineage-specific SNPs (28,176 SNPs) encoding G:C to A:T transitions (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029660/-/DC1/TableS1.pdf">Table S1</a>). Thus, roughly one-fifth of SNPs in our stringent sequential SNP filtering procedure were due to false positives arising at reproducible, and therefore detectable, positions in the genome. This proportion was even greater for the coding sequence positions (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>), suggesting that paralogy and failed assembly of repeats may contribute to these error-prone positions. When considering only the coding sequences, we observed that 93% of the SNPs remaining after removal of shared variant positions were G:C to A:T transitions in the line 10 derivatives (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>). This strongly suggests that the overwhelming majority of independently-derived, high-quality SNPs called by this sequential method are homozygous, true-positive EMS-induced variants, and that the common variants were induced by technical or biological processes distinct from EMS mutagenesis.</p></div><div id="sec-18" class="subsection"><h3>EMS variant-induced line mapping of the extreme dwarf mutant to chromosome 10</h3><p id="p-33">If our SNP calling procedure is specific and comprehensive, segregation of the called SNPs within the 10 pedigree should follow Mendelian inheritance and identify the genomic region encoding the dwarf mutant. SNPs linked to the dwarf mutant should be consistently inherited with the recessive trait. SNPs linked to the causative polymorphism will be encoded by their EMS alleles in the 10-d sample, homozygous for the reference allele in the 10-2 wild-type sample, and heterozygous in the 10-3 sample. <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> is a Hormigas plot, so named because each dot representing a SNP resembles an ant, of allele frequencies measured in the wild-type 10-2 at all high-quality G:C to A:T SNP positions identified as homozygotes in 10-d. Only one segment of the 10-2 genome encodes the reference alleles for positions identified as SNPs in the dwarf mutant. This locus on chromosome 10 is predominantly homozygous wild-type in 10-2 and located within a region of chromosome 10 conspicuously lacking homozygous 10-d SNP calls, consistent with segregation and recombination in the heterozygous parent that gave rise to the pool of dwarf plants that were sequenced (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). Thus, based on the sequence data from two libraries, the dwarf mutant can only be located within a relatively narrow window of 5 Mb present on chromosome 10.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/413/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Mapping of the critical region for the 10-d mutant using the EMS mutations in the line and no outcrossing. Hormigas plot of the allele frequencies (y-axis) in the wild-type segregant 10-2 for homozygous polymorphisms detected in 10-d (x-axis). The only segment not displaying EMS-induced variation is present on chromosome 10. EMS, ethyl methanesulfonate." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-630627784" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Mapping of the critical region for the 10-d mutant using the EMS mutations in the line and no outcrossing. Hormigas plot of the allele frequencies (&lt;em&gt;y&lt;/em&gt;-axis) in the wild-type segregant 10-2 for homozygous polymorphisms detected in 10-d (&lt;em&gt;x&lt;/em&gt;-axis). The only segment not displaying EMS-induced variation is present on chromosome 10. EMS, ethyl methanesulfonate.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/413/F4.medium.gif" width="440" height="359"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/2/413/F4.medium.gif" width="440" height="359"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/413/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/413/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470243" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-34" class="first-child">Mapping of the critical region for the 10-d mutant using the EMS mutations in the line and no outcrossing. Hormigas plot of the allele frequencies (<em>y</em>-axis) in the wild-type segregant 10-2 for homozygous polymorphisms detected in 10-d (<em>x</em>-axis). The only segment not displaying EMS-induced variation is present on chromosome 10. EMS, ethyl methanesulfonate.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-19" class="subsection"><h3>The Sorghum ent-kaurene oxidase ortholog is responsible for the extreme dwarf phenotype</h3><p id="p-35">In a classical map-based cloning procedure, a mapping interval is first identified. Once the effort to identify additional recombinants is greater than the effort of sequencing the genes in the region, coding sequence changes are sought. In our case, we already have the sequence of all genes in the genome. Thus, in an effort to identify SNPs likely to alter the line’s phenotype, we tabulated the protein-coding sequence changed by all high-quality SNPs in each line. About 85% (23,997) of the high-quality SNPs were in intergenic regions and 2113 SNPs were within annotated introns (data not shown). In all five lines, a total of 967 SNPs were predicted to alter the coding capacity of an annotated protein-coding gene. Of these, 927 SNPs were missense mutations, 26 were nonsense (stop codon gain) mutations, and an additional 14 encoded stop codon loss or splice site mutations.</p><p id="p-36">A total of 296, 209, and 252 high-quality G:C to A:T SNPs impact protein-coding capacity in the 10-2, 10-3, and 10-d <em>Sorghum</em> individuals, respectively (<a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>). Of these, only those polymorphisms that are homozygous in the 10-d dwarf mutant and not called in either the wild-type sibling or heterozygous sibling could be the causative polymorphism for the dwarf phenotype. Among the 252 SNPs in 10-d, 154 were shared with both 10-2 and 10-3, and an additional 89 were shared with either 10-2 or 10-3 (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). This left six SNPs as possible causal EMS-induced mutations for the dwarf phenotype. Of these six, only one sits within the critical window on chromosome 10 identified by allele frequency mapping of the EMS-induced alleles in the wild-type segregant. The detected EMS-induced G to A mutation occurred at genomic location 50,477,994 on chromosome 10 in the <em>Sorghum</em> genome (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/413/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genome-wide comparison of nonsynonymous SNPs present in the lines derived from EMS lineage 10. Venn diagrams display the number of shared G to A, or C to T, SNP positions found in lines 10-d, 10-2, and 10-3. EMS, ethyl methanesulfonate; SNP, single nucleotide polymorphism." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-630627784" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Genome-wide comparison of nonsynonymous SNPs present in the lines derived from EMS lineage 10. Venn diagrams display the number of shared G to A, or C to T, SNP positions found in lines 10-d, 10-2, and 10-3. EMS, ethyl methanesulfonate; SNP, single nucleotide polymorphism.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/413/F5.medium.gif" width="440" height="343"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/2/413/F5.medium.gif" width="440" height="343"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/413/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/413/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470304" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-37" class="first-child">Genome-wide comparison of nonsynonymous SNPs present in the lines derived from EMS lineage 10. Venn diagrams display the number of shared G to A, or C to T, SNP positions found in lines 10-d, 10-2, and 10-3. EMS, ethyl methanesulfonate; SNP, single nucleotide polymorphism.</p><div class="sb-div caption-clear"></div></div></div><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470443/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470443/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Putative EMS variants with predicted impact on protein function specific to the dwarf mutant</span><div class="sb-div caption-clear"></div></div></div><p id="p-39">We utilized a similar allele frequency analysis focusing on high-quality G:C to A:T SNPs in coding regions to provide additional evidence for the causative polymorphism on chromosome 10. The causative polymorphism for the dwarf mutant will be present as a homozygous mutant allele in 10-d, present in intermediate frequency in the reads from the 10-3 heterozygote, and present as the homozygous reference BTx623 allele in 10-2. The allele frequencies from all high-quality homozygous G:C to A:T SNPs affecting protein-coding changes present in the dwarf mutant, 10-d, are presented as Hormigas plots for both 10-3 and 10-2 in <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>. Among all 252 homozygous coding sequence differences identified in 10-d, exactly one change was homozygous wild-type in 10-3 and heterozygous in 10-2. Again, this corresponded to the G to A mutation at genomic location 50,477,994 on chromosome 10. PCR-based genotyping of this polymorphism was done for 23 dwarf individuals derived from 10-2, as well as 97 dwarfs from six independent families from the initial 10 line. There was complete linkage of the dwarf mutant phenotype and the SNP at 10:50,477994 (data not shown).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/413/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of allele frequencies for nonsynonymous SNPs identifies one and only one SNP that could encode the dwarf mutant. Hormigas plot of the allele frequencies (y-axis) in the heterozygous sibling 10-3 (top panel) and the homozygous wild-type sibling 10-2 (bottom panel) for SNP positions homozygous in the 10-d lineages (x-axis). Het sib, heterozygous sibling; SNP, single nucleotide polymorphism; WT sib, wild-type sibling." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-630627784" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison of allele frequencies for nonsynonymous SNPs identifies one and only one SNP that could encode the dwarf mutant. Hormigas plot of the allele frequencies (&lt;em&gt;y&lt;/em&gt;-axis) in the heterozygous sibling 10-3 (top panel) and the homozygous wild-type sibling 10-2 (bottom panel) for SNP positions homozygous in the 10-d lineages (&lt;em&gt;x&lt;/em&gt;-axis). Het sib, heterozygous sibling; SNP, single nucleotide polymorphism; WT sib, wild-type sibling.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/413/F6.medium.gif" width="440" height="120"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/7/2/413/F6.medium.gif" width="440" height="120"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/413/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/413/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470382" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-40" class="first-child">Comparison of allele frequencies for nonsynonymous SNPs identifies one and only one SNP that could encode the dwarf mutant. Hormigas plot of the allele frequencies (<em>y</em>-axis) in the heterozygous sibling 10-3 (top panel) and the homozygous wild-type sibling 10-2 (bottom panel) for SNP positions homozygous in the 10-d lineages (<em>x</em>-axis). Het sib, heterozygous sibling; SNP, single nucleotide polymorphism; WT sib, wild-type sibling.</p><div class="sb-div caption-clear"></div></div></div><p id="p-41">The sequential filtering that resulted in the identification of this single SNP is presented in <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>. The mutation resides in the <em>Sorghum</em> gene Sobic.010G172700 (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>), which encodes a 508-amino acid protein, and results in a conversion of the 225th codon from a glutamine (CAG) to an amber stop codon (TAG). Sobic.010G172700 encodes cytochrome P450 CYP701A6 and is the <em>Sorghum</em> ortholog of the <em>ent</em>-kaurene oxidases from rice, <em>Arabidopsis</em>, and Pea (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Davidson <em>et al.</em> 2004</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Helliwell <em>et al.</em> 1999</a>). This enzyme is responsible for multiple reactions in the biosynthesis of the plant hormone GA.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/413/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Number of candidate positions after SNP filtering of NGS data for the dwarf mutant in line 10-d. The SNP filtering steps are listed on the left and the number of SNPs remaining the genome after each step is provided on the right. The one and only one SNP remaining after filtering is represented graphically. The gene model of the locus responsible for the dwarf phenotype in 10-d is displayed with exons and introns indicated with the position of the mutation marked and annotated. Chr, chromosome; NGS, next-generation sequencing; SNP, single nucleotide polymorphism." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-630627784" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Number of candidate positions after SNP filtering of NGS data for the dwarf mutant in line 10-d. The SNP filtering steps are listed on the left and the number of SNPs remaining the genome after each step is provided on the right. The one and only one SNP remaining after filtering is represented graphically. The gene model of the locus responsible for the dwarf phenotype in 10-d is displayed with exons and introns indicated with the position of the mutation marked and annotated. Chr, chromosome; NGS, next-generation sequencing; SNP, single nucleotide polymorphism.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/413/F7.medium.gif" width="440" height="318"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/7/2/413/F7.medium.gif" width="440" height="318"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/413/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/413/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470440" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-42" class="first-child">Number of candidate positions after SNP filtering of NGS data for the dwarf mutant in line 10-d. The SNP filtering steps are listed on the left and the number of SNPs remaining the genome after each step is provided on the right. The one and only one SNP remaining after filtering is represented graphically. The gene model of the locus responsible for the dwarf phenotype in 10-d is displayed with exons and introns indicated with the position of the mutation marked and annotated. Chr, chromosome; NGS, next-generation sequencing; SNP, single nucleotide polymorphism.</p><div class="sb-div caption-clear"></div></div></div><p id="p-43">To confirm that disruption of GA biosynthesis was responsible for the dwarf phenotype, we tested whether the dwarf phenotype could be reversed by the exogenous application of GA<sub>3</sub>. Application of 10 μM GA<sub>3</sub> to light-grown seedlings fully restored wild-type growth to the homozygous dwarf phenotype (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8, A and B</a>). Similarly, the elongation of mesocotyls in dark-grown seedlings was restored when dwarf plants were provided 10 μM GA<sub>3</sub> (<a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8C</a>). Application of GA into to the whorl of light-grown plants substantially rescued the dwarf phenotype in adult plants (data not shown), demonstrating that this phenomenon extended beyond seedlings. The dwarfism in the 10-d line was reversible with GA<sub>3</sub> application, consistent with the mutation discovered in Sobic.010G172700 resulting in a disruption in GA biosynthesis. We therefore name the mutation <em>ent-kaurene oxidase dwarf-1</em> (<em>eko-1</em>).</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/413/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Exogenous GA treatment restores growth to eko-1 seedlings. (A) Photograph of 7 d light-grown seedlings of wild-type (+/+) and eko-1 (&#x2212;/&#x2212;) mutants treated with 10 &#x3BC;M GA or mock treatment control. (B) Histogram of mean seedling height and standard error for 7 d light-grown seedlings of wild-type (+/+), heterozygous individuals (+/&#x2212;), and eko-1 (&#x2212;/&#x2212;) mutants treated with 10 &#x3BC;M GA or mock-treated control. (C) Histogram of mean seedling height and standard error for 7 d dark-grown seedlings wild-type (+/+), heterozygous individuals (+/&#x2212;), and eko-1 (&#x2212;/&#x2212;) mutants treated with 10 &#x3BC;M GA or mock-treated controls. GA, gibberellic acid." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-630627784" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Exogenous GA treatment restores growth to eko-1 seedlings. (A) Photograph of 7 d light-grown seedlings of wild-type (+/+) and &lt;em&gt;eko-1&lt;/em&gt; (&#x2212;/&#x2212;) mutants treated with 10 &#x3BC;M GA or mock treatment control. (B) Histogram of mean seedling height and standard error for 7 d light-grown seedlings of wild-type (+/+), heterozygous individuals (+/&#x2212;), and &lt;em&gt;eko-1&lt;/em&gt; (&#x2212;/&#x2212;) mutants treated with 10 &#x3BC;M GA or mock-treated control. (C) Histogram of mean seedling height and standard error for 7 d dark-grown seedlings wild-type (+/+), heterozygous individuals (+/&#x2212;), and &lt;em&gt;eko-1&lt;/em&gt; (&#x2212;/&#x2212;) mutants treated with 10 &#x3BC;M GA or mock-treated controls. GA, gibberellic acid.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/413/F8.medium.gif" width="216" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/7/2/413/F8.medium.gif" width="216" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/413/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/413/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470308" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-44" class="first-child">Exogenous GA treatment restores growth to eko-1 seedlings. (A) Photograph of 7 d light-grown seedlings of wild-type (+/+) and <em>eko-1</em> (−/−) mutants treated with 10 μM GA or mock treatment control. (B) Histogram of mean seedling height and standard error for 7 d light-grown seedlings of wild-type (+/+), heterozygous individuals (+/−), and <em>eko-1</em> (−/−) mutants treated with 10 μM GA or mock-treated control. (C) Histogram of mean seedling height and standard error for 7 d dark-grown seedlings wild-type (+/+), heterozygous individuals (+/−), and <em>eko-1</em> (−/−) mutants treated with 10 μM GA or mock-treated controls. GA, gibberellic acid.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-20" class="subsection"><h3>EMS variation-induced lines for mutant mapping without segregation variance</h3><p id="p-45">We identified the causative <em>eko-1</em> polymorphism using recursively selfed, but still segregating (M4), mutagenized material. The SNP calling approach and removal of false positives by comparing our mutant to other lines improved NGS processing for EMS-mutant cloning, increased our confidence in SNP calls, and permitted the use of EMS-induced variants as molecular markers for mapping (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a> and <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>). In addition, by sequencing pools of pedigreed materials we could better distinguish causative polymorphisms from inconsequential background mutations (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>, <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>, <a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6</a>, and <a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7</a>). Our success suggested to us that a forward genetics crossing scheme and experimental design that takes advantage of comprehensive information available from whole-genome sequencing was worth exploration.</p><p id="p-46">All sequenced samples were derived from EMS-treated BTx623 plants. Two of these, (10-2 and 12-2) were selected because they had no visible morphological defects. As a result, we have discovered thousands of EMS-induced SNPs that can be used as molecular markers in mapping novel genetic variation without impacting whole-plant phenotypes. In addition, the <em>dhr2-1</em> mutant has a recessive and screenable blue/white Cu<sup>+</sup> reduction phenotype (<a id="xref-ref-24-5" class="xref-bibr" href="#ref-24">Krothapalli <em>et al.</em> 2013</a>) that can be used to confirm F1 cross success when the <em>dhr2-1</em> mutant is used as the seed parent. The thousands of EMS mutants segregating in that line could then be used as molecular markers in F2 populations derived from confirmed F1 crosses. In order to maximize the utility of these materials, we identified the SNPs unique to each of the 10-2, 12-2, and <em>dhr2-1</em> lines. This resulted in 2288–8050 SNPs that can be used for mapping (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029660/-/DC1/FileS1.zip">File S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029660/-/DC1/FileS2.zip">File S2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029660/-/DC1/FileS3.zip">File S3</a>, and <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>). The two phenotypically unaffected lines in the BTx623 background enable the mapping of mutants of small effect as well as traits strongly affected by the genetic variation between <em>Sorghum</em> lines. A cross between these phenotypically normal EVIL twins and any independently-derived mutant provides sufficient molecular markers for mapping without also introducing phenotype-altering segregational variance.</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470258/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470258/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Summary of EMS Variation-Induced Lines for mapping in <em>Sorghum</em> without segregation variance</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-21"><h2 class="">Discussion</h2><p id="p-48">While it would be tempting to conclude from these results that any mutant can be cloned just by sequencing a few lines, there are three essential components to our study that were combined to make it efficient and accurate. First, to limit the mapping procedure to actual polymorphisms, it is necessary to remove all error-prone positions in the genome. Second, we were able to impose logical expectations for SNPs linked to the phenotype that ruled out ∼98% of the genome using a pedigree containing a true wild-type (no dwarf progeny), a segregating sibling, and a dwarf mutant. If a segregating pedigree is not available, using the EMS-induced SNPs as markers still has many advantages for mapping. The largest advantage is that, due to their scarcity, far fewer recombinations will be necessary to reduce the possible mutation space to a manageable list. Finally, the nature of the mutagen used allowed us to narrow down the search space to the point where we had one and only one causal locus. The overwhelming majority of the mutations caused by EMS are G to A (with its complement C to T). Furthermore, SNPs affect phenotypic alterations in mutagenesis experiments via disruptions in protein-coding sequences, as very few single SNPs within noncoding portions of the genome can have large effects on phenotype. Thus, the causal SNP for a mutant phenotype in an EMS-mutagenesis experiment can be identified among G:C to A:T changes that alter protein sequence, as previously discussed (<a id="xref-ref-24-6" class="xref-bibr" href="#ref-24">Krothapalli <em>et al.</em> 2013</a>). Other mutagens are more likely to cause insertion/deletion polymorphisms, which can affect gene function while acting in noncoding regions (for example by deleting the promoter). Insertion/deletion polymorphisms are also much more difficult to detect using short-read sequencing, compounding the problem. It is only with all three factors together that we are able to make our assignment of function.</p><p id="p-49">It is also important to note that getting full coverage of the nonerror-prone portion of the genome is an important requirement for this approach, which relies on full coverage of all potential protein-coding mutations. False negative SNP calls will lead to the misidentification of SNPs that are identified as being more likely to be the one and only one remaining SNP in the window. The number of false negative results will increase as the coding sequence length of an organism increases. In addition, genomes with CG bias or with recent polyploidy in their ancestry, where the unique alignment of reads is impeded by a lack of complexity, will also be problematic for this approach. Polyploid genomes in particular, where paralogs do not allow unique assignment of reads to all positions of the coding sequence (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Tsai <em>et al.</em> 2013</a>), will also be difficult to analyze using this approach. Any region of the coding capacity of the genome in which mapping is reduced or SNP calling is prevented by paralogy can invalidate the assumption that all SNP variants are known. In <em>Arabidopsis</em>, simulated experiments estimate that a genome coverage of 20–50 times was optimal for NGS mapping by BSA, depending on line-cross population type (James <em>et al.</em> 2013). For large genomes, this depth may remain cost-prohibitive for whole-genome sequencing. The cost per lineage can be dramatically improved if exome-sequence capture is used instead, and the cost for sequence capture at similar depths of coverage demonstrates that this as a more cost-effective option for larger genomes (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Henry <em>et al.</em> 2015</a>). Beyond cheaper sequencing, informatic strategies to improve SNP call precision that would perform well at lower coverage could help overcome this barrier. Additional approaches to find SNPs within nonuniquely assigned reads that map to paralogs could also reduce the problem of false negatives that limits the application of our approach.</p><p id="p-50">Fully sequenced lines from the wild-type pedigree (10-2 and 12-2) have great utility for future mapping studies. The lines have no obvious phenotypes and a defined set of real polymorphisms, which make them ideal crossing partners for mapping mutants from mutagenesis types that cannot use the methods described here. With reference to a common theme in literature and popular culture, we have named these lines EVIL twins to denote their status as phenotypically identical but genetically different to the wild-type. The advantage of crossing to an EVIL twin line is that the only phenotypes that will be segregating in the population are those that are present in the mutant. This could be especially important for cloning of mutants with small effects, in which the phenotypic diversity of the outcross lines can obscure the signal. In addition, radiation mutagenesis induces a wide variety of deletion/insertion polymorphisms along with SNPs, so many of the logical filters we applied here will not work to reduce the false positive calls, whereas crossing a radiation-derived mutant to a congenic EVIL twin can be used to introduce markers for mapping without causing additional segregation variation. Three of the lines from this study are candidate EVIL twins: 10-2, 12-2, and the <em>dhr2-1</em> line. From our analysis, each of these lines has over 2000 unique SNPs that can be used for mapping (<a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 3</a>).</p><p id="p-51">A number of called variants visible in <a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6</a> are present at frequencies that are not possible for a single sequenced individual. These variant positions depart from the axis of all other SNPs linked to them and appear to defy Mendelian genetics. This pattern was also observed in allele frequency counts from the other genotypes (data not shown). We suspect that these positions represent false positives from error-prone positions that were identified as homozygotes in only one of the lines. When the allele frequencies of these positions are then tallied from the read data and graphed, we observe an allele frequency that is highly unlikely both due to the proportion and the absence of any recombination breakpoints. This is exactly what would be expected if the SNP filtering procedure thresholds an underlying error distribution that is nonrandomly distributed across the genome. The subtraction of error-prone positions from the five sets of sequencing data presented here are capable of identifying many, but not all, of the false positive positions present in the <em>Sorghum</em> genome. Further sequencing experiments by the community will continue to identify positions prone to false positive errors. Indeed, preliminary analysis of a larger set of sequenced lines indicate that additional moderate-frequency shared variants are present in sequence data sets derived from BTx623, and that their discovery was not completely saturated after sequencing 600 mutant lines (C. Addo-Quaye and B. P. Dilkes, unpublished data).</p><p id="p-52">The identified gene responsible for the dwarf allele encodes an enzyme that is the target of paclobutrazol inhibition of GA biosynthesis (Hedden and Graebe 1985). Mutations of the ortholog of this gene are responsible for the <em>Gibberellic Acid 3</em> dwarf mutant of <em>Arabidopsis</em>, the <em>lh</em> dwarf mutant of Pea, and the semidwarfing allele from Tan-Gibozou rice that contributed to the “green revolution” in Japan (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Itoh <em>et al.</em> 2004</a>). Of the maize mutations affected in GA biosynthesis, none have yet been recovered for blocks in these steps. The maize genome encodes two of these enzymes resulting from a tandem duplication expressed at dramatically different levels (Supplemental Material). If both were functional, this would explain why this step has not been identified mutationally in maize.</p><p id="p-53">Recently, other studies have found gene mutations in the GA and brassinosteroid biosynthesis pathways that impact not only plant height but other agronomically important traits in <em>Sorghum</em>. <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Petti <em>et al.</em> (2015)</a> identified a dwarf <em>Sorghum</em> mutant, <em>dwf1-1</em>, that mapped to a frame-shift in a presumed GA20-oxidase. In addition to dwarfism, this also caused male sterility and impacted cellulose biosynthesis. This mutation occurs later in the GA biosynthetic pathway than the <em>ent</em>-kaurene oxidase mutation identified in this study. <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Ordonio <em>et al.</em> (2014)</a> identified four loss-of-function mutations [<em>ent</em>-copalyl diphosphate synthase (CPS; SbCPS1), <em>ent</em>-kaurene synthase (KS; SbKs1), <em>ent</em>-kaurene oxidase (KO; SbKO1), and <em>ent</em>-kaurenoic acid oxidase (KAO; SbKAO1)] early in the GA biosynthetic pathway that lead to severe dwarfism and culm bending in sweet <em>Sorghum</em>. Finally, <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Rizal <em>et al.</em> (2015)</a> found that a mutation in a cytochrome P450 (CYP90D2) in brassinosteroid biosynthesis leads to decreased vein density in <em>Sorghum</em>.</p><p id="p-54">The application of EMS mutagenesis and cost-effective short-read sequencing is driving rapid progress in our understanding of the control of crop plant architecture. Additional improvements in experimental design can further accelerate this and extend the utility of these methods. If we can reliably detect all of the sequence polymorphisms in a given genome, whole-genome sequencing allows us to invert the typical process for mutant identification. When all possible causative SNPs are known, only recombinants distinguishing the putatively causative SNPs are needed. This reduces the number of individuals necessary to identify one and only one potentially causative polymorphism in a line. In addition, by using all of the EMS-induced alleles in a mutagenized line, even those in noncoding sequences, we can remove the need to cross a phenotypically selected mutant to a genetically distinct stock. No incidental segregating variance needs to be introduced and, thus, environmentally labile or adaptively consequential mutant phenotypes that would be masked by background variation are now amenable to cloning. Moreover, because there are few mutations of putative effect, and less segregation variance that might drown out the effects of induced alleles with small phenotypic consequences, we can use this approach to quickly and cost-effectively clone mutations that have moderate to small effects.</p><div id="sec-22" class="subsection"><h3>Conclusions</h3><p id="p-55">Using a novel combination of full genome sequencing, experimental design, and bioinformatics algorithms, we have cloned the gene for an EMS-induced dwarf mutant in <em>Sorghum bicolor</em>. By comparing the sequence of several lines within the pedigree, removing error-prone positions, and focusing on polymorphisms that EMS will induce, we were able to narrow the polymorphisms down to one and only one mutation, an induced stop codon in the cytochrome P450 gene Sobic.010G172700. We have demonstrated that this approach can change the current paradigm for mapping mutants. We have also produced three lines that can be used as EVIL twins for mapping in the BTx623 background without introducing additional phenotypic variation.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-56">This project was supported by a grant from the Bill and Melinda Gates Foundation (grant number OPP 1052924), a United States Department of Agriculture-Agricultural Research Service grant (5070-21000-039-00D) to I.B. and B.P.D., and an Agriculture and Food Research Initiative Competitive grant (2012-67012-19817) to E.B.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-7"><p id="p-57">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029660/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029660/-/DC1</a>.</p></li><li class="fn" id="fn-8"><p id="p-58"><em>Communicating editor: G. M. Coruzzi</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-03-25"><span class="received-label">Received </span>March 25, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-11-20"><span class="accepted-label">Accepted </span>November 20, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Addo-Quaye <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.413.1" data-doi="10.1038/nbt.2095"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abe</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kosugi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yoshida</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Natsume</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takagi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome sequencing reveals agronomically important loci in rice using MutMap.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">174</span>–<span class="cit-lpage">178</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DAbe%26rft.auinit1%253DA.%26rft.volume%253D30%26rft.issue%253D2%26rft.spage%253D174%26rft.epage%253D178%26rft.atitle%253DGenome%2Bsequencing%2Breveals%2Bagronomically%2Bimportant%2Bloci%2Bin%2Brice%2Busing%2BMutMap.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2095%26rft_id%253Dinfo%253Apmid%252F22267009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.2095&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22267009&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F413.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.413.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abramoff</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magelhaes</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ram</span>  <span class="cit-name-given-names">S. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Image processing with ImageJ.</span> <abbr class="cit-jnl-abbrev">Biophoton. Int.</abbr> <span class="cit-vol">11</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">36</span>–<span class="cit-lpage">42</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiophoton.%2BInt.%26rft.volume%253D11%26rft.spage%253D36%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.413.3" data-doi="10.1186/gb-2011-12-3-r28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ashelford</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eriksson</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allen</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">D’Amore</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johansson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Full genome re-sequencing reveals a novel circadian clock mutation in <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">R28</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DAshelford%26rft.auinit1%253DK.%26rft.volume%253D12%26rft.issue%253D3%26rft.spage%253DR28%26rft.epage%253DR28%26rft.atitle%253DFull%2Bgenome%2Bre-sequencing%2Breveals%2Ba%2Bnovel%2Bcircadian%2Bclock%2Bmutation%2Bin%2BArabidopsis.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2011-12-3-r28%26rft_id%253Dinfo%253Apmid%252F21429190%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2011-12-3-r28&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21429190&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F413.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.413.4" data-doi="10.1111/j.1365-313X.2011.04619.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Austin</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vidaurre</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stamatiou</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Breit</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Provart</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Next-generation mapping of Arabidopsis genes.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">715</span>–<span class="cit-lpage">725</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DAustin%26rft.auinit1%253DR.%2BS.%26rft.volume%253D67%26rft.issue%253D4%26rft.spage%253D715%26rft.epage%253D725%26rft.atitle%253DNext-generation%2Bmapping%2Bof%2BArabidopsis%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2011.04619.x%26rft_id%253Dinfo%253Apmid%252F21518053%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2011.04619.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21518053&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F413.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000293753800012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.413.5" data-doi="10.1111/j.1467-7652.2011.00646.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blomstedt</span>  <span class="cit-name-given-names">C. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gleadow</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Donnell</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naur</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A combined biochemical screen and TILLING approach identifies mutations in <em>Sorghum bicolor</em> L. Moench resulting in acyanogenic forage production.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">54</span>–<span class="cit-lpage">66</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPlant%2BBiotechnol%2BJ%26rft.aulast%253DBlomstedt%26rft.auinit1%253DC.%2BK.%26rft.volume%253D10%26rft.issue%253D1%26rft.spage%253D54%26rft.epage%253D66%26rft.atitle%253DA%2Bcombined%2Bbiochemical%2Bscreen%2Band%2BTILLING%2Bapproach%2Bidentifies%2Bmutations%2Bin%2BSorghum%2Bbicolor%2BL.%2BMoench%2Bresulting%2Bin%2Bacyanogenic%2Bforage%2Bproduction.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-7652.2011.00646.x%26rft_id%253Dinfo%253Apmid%252F21880107%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1467-7652.2011.00646.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21880107&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F413.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.2.413.6" data-doi="10.1371/journal.pgen.1003219"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bull</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rimmer</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siggs</span>  <span class="cit-name-given-names">O. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miosge</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roots</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Unlocking the bottleneck in forward genetics using whole-genome sequencing and identity by descent to isolate causative mutations.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003219</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003219%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003219%26rft_id%253Dinfo%253Apmid%252F23382690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003219&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23382690&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F413.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.413.7" data-doi="10.2135/cropsci2007.02.0080"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Casa</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pressoir</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rooney</span>  <span class="cit-name-given-names">W. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Community resources and strategies for association mapping in sorghum.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">30</span>–<span class="cit-lpage">40</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DCasa%26rft.auinit1%253DA.%2BM.%26rft.volume%253D48%26rft.issue%253D1%26rft.spage%253D30%26rft.epage%253D40%26rft.atitle%253DCommunity%2BResources%2Band%2BStrategies%2Bfor%2BAssociation%2BMapping%2Bin%2BSorghum%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2007.02.0080%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2007.02.0080&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000252971400004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.413.8" data-doi="10.1093/bioinformatics/btu067"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">A. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Teo</span>  <span class="cit-name-given-names">Y.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ong</span>  <span class="cit-name-given-names">R. T.-H.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Assessing single nucleotide variant detection and genotype calling on whole-genome sequenced individuals.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">1707</span>–<span class="cit-lpage">1713</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu067%26rft_id%253Dinfo%253Apmid%252F24558117%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMzAvMTIvMTcwNyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi80MTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.413.9" data-doi="10.1186/gb-2003-4-4-r25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheung</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Estivill</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khaja</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">MacDonald</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lau</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Genome-wide detection of segmental duplications and potential assembly errors in the human genome sequence.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">R25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DCheung%26rft.auinit1%253DJ.%26rft.volume%253D4%26rft.issue%253D4%26rft.spage%253DR25%26rft.epage%253DR25%26rft.atitle%253DGenome-wide%2Bdetection%2Bof%2Bsegmental%2Bduplications%2Band%2Bpotential%2Bassembly%2Berrors%2Bin%2Bthe%2Bhuman%2Bgenome%2Bsequence.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2003-4-4-r25%26rft_id%253Dinfo%253Apmid%252F12702206%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2003-4-4-r25&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12702206&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F413.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.413.10" data-doi="10.3389/fgene.2012.00035"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cingolani</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patel</span>  <span class="cit-name-given-names">V. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Land</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">a</span> <span class="cit-article-title">Using <em>Drosophila melanogaster</em> as a model for genotoxic chemical mutational studies with a new program, SnpSift.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">35</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DFront%2BGenet%26rft.aulast%253DCingolani%26rft.auinit1%253DP.%26rft.volume%253D3%26rft.spage%253D35%26rft.epage%253D35%26rft.atitle%253DUsing%2BDrosophila%2Bmelanogaster%2Bas%2Ba%2BModel%2Bfor%2BGenotoxic%2BChemical%2BMutational%2BStudies%2Bwith%2Ba%2BNew%2BProgram%252C%2BSnpSift.%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2012.00035%26rft_id%253Dinfo%253Apmid%252F22435069%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2012.00035&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22435069&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F413.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.413.11" data-doi="10.4161/fly.19695"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cingolani</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Platts</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">b</span> <span class="cit-article-title">A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of <em>Drosophila melanogaster</em> strain w1118; iso-2; iso-3.</span> <abbr class="cit-jnl-abbrev">Fly (Austin)</abbr> <span class="cit-vol">6</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">80</span>–<span class="cit-lpage">92</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFly%2B%2528Austin%2529%26rft.volume%253D6%26rft.spage%253D80%26rft_id%253Dinfo%253Adoi%252F10.4161%252Ffly.19695%26rft_id%253Dinfo%253Apmid%252F22728672%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/fly.19695&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22728672&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F413.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.413.12" data-doi="10.1104/pp.103.032706"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davidson</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Helliwell</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poole</span>  <span class="cit-name-given-names">A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reid</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The pea gene <em>LH</em> encodes <em>ent</em>-kaurene oxidase.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">134</span>: <span class="cit-fpage">1123</span>–<span class="cit-lpage">1134</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.103.032706%26rft_id%253Dinfo%253Apmid%252F14988475%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTM0LzMvMTEyMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi80MTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.2.413.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dove</span>  <span class="cit-name-given-names">W. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shedlovsky</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clipson</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amos-Landgraf</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Halberg</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A strategy to identify dominant point mutant modifiers of a quantitative trait.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1113</span>–<span class="cit-lpage">1121</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D4%26rft.spage%253D1113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.413.14" data-doi="10.1093/hmg/11.17.1987"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Estivill</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheung</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pujana</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakabayashi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scherer</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Chromosomal regions containing high-density and ambiguously mapped putative single nucleotide polymorphisms (SNPs) correlate with segmental duplications in the human genome.</span> <abbr class="cit-jnl-abbrev">Hum. Mol. Genet.</abbr> <span class="cit-vol">11</span>(<span class="cit-issue">17</span>): <span class="cit-fpage">1987</span>–<span class="cit-lpage">1995</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHuman%2BMolecular%2BGenetics%26rft.stitle%253DHum%2BMol%2BGenet%26rft.aulast%253DEstivill%26rft.auinit1%253DX.%26rft.volume%253D11%26rft.issue%253D17%26rft.spage%253D1987%26rft.epage%253D1995%26rft.atitle%253DChromosomal%2Bregions%2Bcontaining%2Bhigh-density%2Band%2Bambiguously%2Bmapped%2Bputative%2Bsingle%2Bnucleotide%2Bpolymorphisms%2B%2528SNPs%2529%2Bcorrelate%2Bwith%2Bsegmental%2Bduplications%2Bin%2Bthe%2Bhuman%2Bgenome%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fhmg%252F11.17.1987%26rft_id%253Dinfo%253Apmid%252F12165560%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiaG1nIjtzOjU6InJlc2lkIjtzOjEwOiIxMS8xNy8xOTg3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzQxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.413.15" data-doi="10.1038/ng1401"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fredman</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Potter</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eichler</span>  <span class="cit-name-given-names">E. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Den Dunnen</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Complex SNP-related sequence variation in segmental genome duplications.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">36</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">861</span>–<span class="cit-lpage">866</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DFredman%26rft.auinit1%253DD.%26rft.volume%253D36%26rft.issue%253D8%26rft.spage%253D861%26rft.epage%253D866%26rft.atitle%253DComplex%2BSNP-related%2Bsequence%2Bvariation%2Bin%2Bsegmental%2Bgenome%2Bduplications.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1401%26rft_id%253Dinfo%253Apmid%252F15247918%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1401&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15247918&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F413.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222974000019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.413.16" data-doi="10.1093/nar/gkr944"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goodstein</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howson</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neupane</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Phytozome: a comparative platform for green plant genomics.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">D1178</span>–<span class="cit-lpage">D1186</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr944%26rft_id%253Dinfo%253Apmid%252F22110026%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjExOiI0MC9EMS9EMTE3OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi80MTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.413.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Greene</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Codomo</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">N. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henikoff</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Till</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Spectrum of chemically induced mutations from a large-scale reverse-genetic screen in Arabidopsis.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">164</span>: <span class="cit-fpage">731</span>–<span class="cit-lpage">740</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGreene%26rft.auinit1%253DE.%2BA.%26rft.volume%253D164%26rft.issue%253D2%26rft.spage%253D731%26rft.epage%253D740%26rft.atitle%253DSpectrum%2Bof%2BChemically%2BInduced%2BMutations%2BFrom%2Ba%2BLarge-Scale%2BReverse-Genetic%2BScreen%2Bin%2BArabidopsis%26rft_id%253Dinfo%253Apmid%252F12807792%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY0LzIvNzMxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzQxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.413.18" data-doi="10.1534/genetics.167.1.471"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hamblin</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gallego</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kukatla</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Comparative population genetics of the panicoid grasses: sequence polymorphism, linkage disequilibrium and selection in a diverse sample of <em>Sorghum bicolor</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">167</span>: <span class="cit-fpage">471</span>–<span class="cit-lpage">483</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHamblin%26rft.auinit1%253DM.%2BT.%26rft.volume%253D167%26rft.issue%253D1%26rft.spage%253D471%26rft.epage%253D483%26rft.atitle%253DComparative%2BPopulation%2BGenetics%2Bof%2Bthe%2BPanicoid%2BGrasses%253A%2BSequence%2BPolymorphism%252C%2BLinkage%2BDisequilibrium%2Band%2BSelection%2Bin%2Ba%2BDiverse%2BSample%2Bof%2BSorghum%2Bbicolor%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.167.1.471%26rft_id%253Dinfo%253Apmid%252F15166170%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY3LzEvNDcxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzQxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.413.19" data-doi="10.1104/pp.119.2.507"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Helliwell</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poole</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peacock</span>  <span class="cit-name-given-names">W. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dennis</span>  <span class="cit-name-given-names">E. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Arabidopsis <em>ent</em>-kaurene oxidase catalyzes three steps of gibberellin biosynthesis.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">119</span>: <span class="cit-fpage">507</span>–<span class="cit-lpage">510</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiology%26rft.stitle%253DPlant%2BPhysiol.%26rft.aulast%253DHelliwell%26rft.auinit1%253DC.%2BA.%26rft.volume%253D119%26rft.issue%253D2%26rft.spage%253D507%26rft.epage%253D510%26rft.atitle%253DArabidopsis%2Bent-Kaurene%2BOxidase%2BCatalyzes%2BThree%2BSteps%2Bof%2BGibberellin%2BBiosynthesis%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.119.2.507%26rft_id%253Dinfo%253Apmid%252F9952446%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxMTkvMi81MDciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzIvNDEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.413.20" data-doi="10.1105/tpc.113.121590"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henry</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagalakshmi</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lieberman</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ngo</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krasileva</span>  <span class="cit-name-given-names">K. V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Efficient genome-wide detection and cataloging of EMS-induced mutations using exome capture and next-generation sequencing.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">1382</span>–<span class="cit-lpage">1397</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.113.121590%26rft_id%253Dinfo%253Apmid%252F24728647%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjI2LzQvMTM4MiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi80MTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.413.21" data-doi="10.1105/tpc.15.00349"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henry</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zinkgraf</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Groover</span>  <span class="cit-name-given-names">A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Comai</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A system for dosage-based function genomics in popular.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2370</span>–<span class="cit-lpage">2383</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.15.00349%26rft_id%253Dinfo%253Apmid%252F26320226%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjI3LzkvMjM3MCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi80MTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.413.22" data-doi="10.1038/embor.2008.142"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hunter</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The paradox of model organisms.</span> <abbr class="cit-jnl-abbrev">EMBO Rep.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">717</span>–<span class="cit-lpage">720</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BRep.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fembor.2008.142%26rft_id%253Dinfo%253Apmid%252F18670440%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NToiZW1ib3IiO3M6NToicmVzaWQiO3M6NzoiOS84LzcxNyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi80MTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.413.23" data-doi="10.1023/B:PLAN.0000038261.21060.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Itoh</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tatsumi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sakamoto</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Otomo</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toyomasu</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A rice semi-dwarf gene, <em>Tan-Ginbozu</em> (<em>D35</em>), encodes the gibberellin biosynthesis enzyme, <em>ent</em>-kaurene oxidase.</span> <abbr class="cit-jnl-abbrev">Plant Mol. Biol.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">533</span>–<span class="cit-lpage">547</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BMol%2BBiol%26rft.aulast%253DItoh%26rft.auinit1%253DH.%26rft.volume%253D54%26rft.issue%253D4%26rft.spage%253D533%26rft.epage%253D547%26rft.atitle%253DA%2Brice%2Bsemi-dwarf%2Bgene%252C%2BTan-Ginbozu%2B%2528D35%2529%252C%2Bencodes%2Bthe%2Bgibberellin%2Bbiosynthesis%2Benzyme%252C%2Bent-kaurene%2Boxidase.%26rft_id%253Dinfo%253Adoi%252F10.1023%252FB%253APLAN.0000038261.21060.47%26rft_id%253Dinfo%253Apmid%252F15316288%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/B:PLAN.0000038261.21060.47&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15316288&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F413.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223358600005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.413.24" data-doi="10.1534/genetics.113.149567"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krothapalli</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buescher</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chapple</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Forward genetics by genome sequencing reveals that rapid cyanide release deters insect herbivory of <em>Sorghum bicolor</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">195</span>: <span class="cit-fpage">309</span>–<span class="cit-lpage">318</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.149567%26rft_id%253Dinfo%253Apmid%252F23893483%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk1LzIvMzA5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzQxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.413.25" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows–Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTQvMTc1NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi80MTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.413.26" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The sequence alignment/map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTYvMjA3OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi80MTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.413.27" data-doi="10.1101/gr.088013.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">SNP detection for massively parallel whole-genome resequencing.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1124</span>–<span class="cit-lpage">1132</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.088013.108%26rft_id%253Dinfo%253Apmid%252F19420381%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzYvMTEyNCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi80MTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.413.28" data-doi="10.1534/genetics.112.141341"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lindner</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raissig</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sailer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shimosato-Asano</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bruggmann</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">SNP-ratio mapping (SRM): identifying lethal alleles and mutations in complex genetic backgrounds by next-generation sequencing.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">191</span>: <span class="cit-fpage">1381</span>–<span class="cit-lpage">1386</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.141341%26rft_id%253Dinfo%253Apmid%252F22649081%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5MS80LzEzODEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzIvNDEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.413.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mace</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tai</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilding</span>  <span class="cit-name-given-names">E. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prentis</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Whole-genome sequencing reveals untapped genetic potential in Africa’s indigenous cereal crop sorghum.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">2320</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D4%26rft.spage%253D2320%26rft_id%253Dinfo%253Apmid%252F23982223%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23982223&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F413.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.413.30" data-doi="10.1073/pnas.88.21.9828"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Michelmore</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paran</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kesseli</span>  <span class="cit-name-given-names">R. V.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Identification of markers linked to disease-resistance genes by bulked segregant analysis: a rapid method to detect markers in specific genomic regions by using segregating populations.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">9828</span>–<span class="cit-lpage">9832</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DMichelmore%26rft.auinit1%253DR.%2BW.%26rft.volume%253D88%26rft.issue%253D21%26rft.spage%253D9828%26rft.epage%253D9832%26rft.atitle%253DIdentification%2Bof%2Bmarkers%2Blinked%2Bto%2Bdisease-resistance%2Bgenes%2Bby%2Bbulked%2Bsegregant%2Banalysis%253A%2Ba%2Brapid%2Bmethod%2Bto%2Bdetect%2Bmarkers%2Bin%2Bspecific%2Bgenomic%2Bregions%2Bby%2Busing%2Bsegregating%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.88.21.9828%26rft_id%253Dinfo%253Apmid%252F1682921%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODgvMjEvOTgyOCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi80MTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.413.31" data-doi="10.1186/gb-2011-12-11-r112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Minoche</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dohm</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Himmelbauer</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Evaluation of genomic high-throughput sequencing data generated on Illumina HiSeq and genome analyzer systems.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">R112</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DMinoche%26rft.auinit1%253DA.%2BE.%26rft.volume%253D12%26rft.issue%253D11%26rft.spage%253DR112%26rft.epage%253DR112%26rft.atitle%253DEvaluation%2Bof%2Bgenomic%2Bhigh-throughput%2Bsequencing%2Bdata%2Bgenerated%2Bon%2BIllumina%2BHiSeq%2Band%2Bgenome%2Banalyzer%2Bsystems.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2011-12-11-r112%26rft_id%253Dinfo%253Apmid%252F22067484%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2011-12-11-r112&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22067484&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F413.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.413.32" data-doi="10.1186/1471-2164-12-256"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mokry</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nijman</span>  <span class="cit-name-given-names">I. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Dijken</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benjamins</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heidstra</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Identification of factors required for meristem function in Arabidopsis using a novel next generation sequencing fast forward genetics approach.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">256</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DMokry%26rft.auinit1%253DM.%26rft.volume%253D12%26rft.spage%253D256%26rft.epage%253D256%26rft.atitle%253DIdentification%2Bof%2Bfactors%2Brequired%2Bfor%2Bmeristem%2Bfunction%2Bin%2BArabidopsis%2Busing%2Ba%2Bnovel%2Bnext%2Bgeneration%2Bsequencing%2Bfast%2Bforward%2Bgenetics%2Bapproach.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-256%26rft_id%253Dinfo%253Apmid%252F21599977%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-256&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21599977&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F413.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.413.33" data-doi="10.1016/S0168-9525(02)02820-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neff</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turk</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kalishman</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Web-based primer design for single nucleotide polymorphism analysis.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">613</span>–<span class="cit-lpage">615</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DNeff%26rft.auinit1%253DM.%2BM.%26rft.volume%253D18%26rft.issue%253D12%26rft.spage%253D613%26rft.epage%253D615%26rft.atitle%253DWeb-based%2Bprimer%2Bdesign%2Bfor%2Bsingle%2Bnucleotide%2Bpolymorphism%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0168-9525%252802%252902820-2%26rft_id%253Dinfo%253Apmid%252F12446140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0168-9525(02)02820-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12446140&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F413.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000179433300007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.413.34" data-doi="10.1186/1471-2164-12-352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Antony</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Single-nucleotide polymorphism discovery by high-throughput sequencing in sorghum.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">352</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DNelson%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D12%26rft.spage%253D352%26rft.epage%253D352%26rft.atitle%253DSingle-nucleotide%2Bpolymorphism%2Bdiscovery%2Bby%2Bhigh-throughput%2Bsequencing%2Bin%2Bsorghum.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-352%26rft_id%253Dinfo%253Apmid%252F21736744%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21736744&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F413.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.413.35" data-doi="10.1038/nrg2986"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paul</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albrechtsen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Y. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genotype and SNP calling from next-generation sequencing data.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">443</span>–<span class="cit-lpage">451</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DNielsen%26rft.auinit1%253DR.%26rft.volume%253D12%26rft.issue%253D6%26rft.spage%253D443%26rft.epage%253D451%26rft.atitle%253DGenotype%2Band%2BSNP%2Bcalling%2Bfrom%2Bnext-generation%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2986%26rft_id%253Dinfo%253Apmid%252F21587300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2986&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21587300&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F413.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.413.36" data-doi="10.1186/gm432"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">O’Rawe</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Low concordance of multiple variant-calling pipelines: practical implications for exome and genome sequencing.</span> <abbr class="cit-jnl-abbrev">Genome Med.</abbr> <span class="cit-vol">5</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">28</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BMed.%26rft.volume%253D5%26rft.spage%253D28%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgm432%26rft_id%253Dinfo%253Apmid%252F23537139%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gm432&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23537139&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F413.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.413.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ordonio</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ito</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hatakeyama</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ohmae-Shinohara</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kasuga</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Gibberellin deficiency pleiotropically induces culm bending in sorghum: an insight into sorghum semi-dwarf breeding.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">5287</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D4%26rft.spage%253D5287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.413.38" data-doi="10.1038/nature07723"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowers</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bruggmann</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dubchak</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grimwood</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The <em>Sorghum bicolor</em> genome and the diversification of grasses.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">457</span>: <span class="cit-fpage">551</span>–<span class="cit-lpage">556</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DPaterson%26rft.auinit1%253DA.%2BH.%26rft.volume%253D457%26rft.issue%253D7229%26rft.spage%253D551%26rft.epage%253D556%26rft.atitle%253DThe%2BSorghum%2Bbicolor%2Bgenome%2Band%2Bthe%2Bdiversification%2Bof%2Bgrasses.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07723%26rft_id%253Dinfo%253Apmid%252F19189423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07723&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19189423&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F413.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262852200034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.413.39" data-doi="10.1104/pp.15.00928"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Petti</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirano</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stork</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DeBolt</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Mapping of a cellulose deficient mutant named <em>dwarf1–1</em> in <em>Sorghum bicolor</em> to the green revolution gene <em>gibberellin20-oxidase</em> reveals a regulatory association between gibberellin and cellulose biosynthesis.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">169</span>: <span class="cit-fpage">705</span>–<span class="cit-lpage">716</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.15.00928%26rft_id%253Dinfo%253Apmid%252F26198258%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNjkvMS83MDUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzIvNDEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.2.413.40" data-doi="10.1093/aob/mci015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dillon</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hodnett</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rooney</span>  <span class="cit-name-given-names">W. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ross</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genome evolution in the genus <em>Sorghum</em> (Poaceae).</span> <abbr class="cit-jnl-abbrev">Ann. Bot. (Lond.)</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">219</span>–<span class="cit-lpage">227</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2BBotany%26rft.stitle%253DAnn%2BBot%26rft.aulast%253DPRICE%26rft.auinit1%253DH.%2BJ.%26rft.volume%253D95%26rft.issue%253D1%26rft.spage%253D219%26rft.epage%253D227%26rft.atitle%253DGenome%2BEvolution%2Bin%2Bthe%2BGenus%2BSorghum%2B%2528Poaceae%2529%26rft_id%253Dinfo%253Adoi%252F10.1093%252Faob%252Fmci015%26rft_id%253Dinfo%253Apmid%252F15596469%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW5uYm90IjtzOjU6InJlc2lkIjtzOjg6Ijk1LzEvMjE5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzQxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.413.41" data-doi="10.1038/nmeth.1270"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quail</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kozarewa</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scally</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A large genome center’s improvements to the Illumina sequencing system.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1005</span>–<span class="cit-lpage">1010</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DQuail%26rft.auinit1%253DM.%2BA.%26rft.volume%253D5%26rft.issue%253D12%26rft.spage%253D1005%26rft.epage%253D1010%26rft.atitle%253DA%2Blarge%2Bgenome%2Bcenter%2527s%2Bimprovements%2Bto%2Bthe%2BIllumina%2Bsequencing%2Bsystem.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1270%26rft_id%253Dinfo%253Apmid%252F19034268%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1270&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19034268&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F413.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261212700012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.413.42" data-doi="10.1186/1471-2164-13-341"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quail</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coupland</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Otto</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A tale of three next generation sequencing platforms: comparison of Ion Torrent, Pacific Biosciences and Illumina MiSeq sequencers.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">341</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DQuail%26rft.auinit1%253DM.%26rft.volume%253D13%26rft.spage%253D341%26rft.epage%253D341%26rft.atitle%253DA%2Btale%2Bof%2Bthree%2Bnext%2Bgeneration%2Bsequencing%2Bplatforms%253A%2Bcomparison%2Bof%2BIon%2BTorrent%252C%2BPacific%2BBiosciences%2Band%2BIllumina%2BMiSeq%2Bsequencers.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-341%26rft_id%253Dinfo%253Apmid%252F22827831%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-341&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22827831&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F413.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.413.43" data-doi="10.1093/bioinformatics/btq033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quinlan</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">I. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">BEDTools: a flexible suite of utilities for comparing genomic features.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">841</span>–<span class="cit-lpage">842</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq033%26rft_id%253Dinfo%253Apmid%252F20110278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNi82Lzg0MSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi80MTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.413.44" data-doi="10.1534/genetics.113.160200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rice</span>  <span class="cit-name-given-names">W. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The synthesis paradigm in genetics.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">196</span>: <span class="cit-fpage">367</span>–<span class="cit-lpage">371</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DRice%26rft.auinit1%253DW.%2BR.%26rft.volume%253D196%26rft.issue%253D2%26rft.spage%253D367%26rft.epage%253D371%26rft.atitle%253DThe%2BSynthesis%2BParadigm%2Bin%2BGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.160200%26rft_id%253Dinfo%253Apmid%252F24496401%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk2LzIvMzY3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzQxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.413.45" data-doi="10.1111/tpj.13007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rizal</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thakur</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dionora</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karki</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wanchana</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Two forward genetic screens for vein density mutants in sorghum converge on a cytochrome P450 gene in the brassinosteroid pathway.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">257</span>–<span class="cit-lpage">266</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D84%26rft.spage%253D257%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.13007%26rft_id%253Dinfo%253Apmid%252F26333774%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.13007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26333774&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F413.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.413.46" data-doi="10.1038/nmeth0809-550"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schneeberger</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ossowski</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lanz</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Juul</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petersen</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">SHOREmap: simultaneous mapping and mutation identification by deep sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">550</span>–<span class="cit-lpage">551</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DSchneeberger%26rft.auinit1%253DK.%26rft.volume%253D6%26rft.issue%253D8%26rft.spage%253D550%26rft.epage%253D551%26rft.atitle%253DSHOREmap%253A%2Bsimultaneous%2Bmapping%2Band%2Bmutation%2Bidentification%2Bby%2Bdeep%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth0809-550%26rft_id%253Dinfo%253Apmid%252F19644454%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth0809-550&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19644454&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F413.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268493700004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.413.47" data-doi="10.1093/bioinformatics/bts535"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Teo</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pawitan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ku</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chia</span>  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salim</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Statistical challenges associated with detecting copy number variations with next-generation sequencing.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>(<span class="cit-issue">21</span>): <span class="cit-fpage">2711</span>–<span class="cit-lpage">2718</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts535%26rft_id%253Dinfo%253Apmid%252F22942022%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjgvMjEvMjcxMSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi80MTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.413.48" data-doi="10.1101/gr.157651.113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edgley</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strasbourger</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flibotte</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ewing</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The million mutation project: a new approach to genetics in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1749</span>–<span class="cit-lpage">1762</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.157651.113%26rft_id%253Dinfo%253Apmid%252F23800452%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyMy8xMC8xNzQ5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzQxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.413.49" data-doi="10.1016/j.pbi.2011.03.024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trontin</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tisné</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bach</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loudet</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">What does Arabidopsis natural variation teach us (and does not teach us) about adaptation in plants?</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">225</span>–<span class="cit-lpage">231</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bplant%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BPlant%2BBiol%26rft.aulast%253DTrontin%26rft.auinit1%253DC.%26rft.volume%253D14%26rft.issue%253D3%26rft.spage%253D225%26rft.epage%253D231%26rft.atitle%253DWhat%2Bdoes%2BArabidopsis%2Bnatural%2Bvariation%2Bteach%2Bus%2B%2528and%2Bdoes%2Bnot%2Bteach%2Bus%2529%2Babout%2Badaptation%2Bin%2Bplants%253F%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pbi.2011.03.024%26rft_id%253Dinfo%253Apmid%252F21536479%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pbi.2011.03.024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21536479&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F413.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.413.50" data-doi="10.1104/pp.112.213256"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsai</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Missirian</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ngo</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tran</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Production of a high-efficiency TILLING population through polyploidization.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">161</span>: <span class="cit-fpage">1604</span>–<span class="cit-lpage">1614</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.112.213256%26rft_id%253Dinfo%253Apmid%252F23417087%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTYxLzQvMTYwNCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi80MTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.413.51" data-doi="10.1093/pcp/pcr029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Uchida</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sakamoto</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kurata</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tasaka</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Identification of EMS-induced causal mutations in a non-reference <em>Arabidopsis thaliana</em> accession by whole genome sequencing.</span> <abbr class="cit-jnl-abbrev">Plant Cell Physiol.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">716</span>–<span class="cit-lpage">722</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fpcp%252Fpcr029%26rft_id%253Dinfo%253Apmid%252F21398646%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGNlbGxwaHlzIjtzOjU6InJlc2lkIjtzOjg6IjUyLzQvNzE2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzQxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.2.413.52"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Williams-Carrier</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stiffler</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Belcher</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kroeger</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stern</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Use of Illumina sequencing to identify transposon insertions underlying mutant phenotypes in high-copy <em>Mutator</em> lines of maize.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">167</span>–<span class="cit-lpage">177</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DWilliams-Carrier%26rft.auinit1%253DR.%26rft.volume%253D63%26rft.issue%253D1%26rft.spage%253D167%26rft.epage%253D177%26rft.atitle%253DUse%2Bof%2BIllumina%2Bsequencing%2Bto%2Bidentify%2Btransposon%2Binsertions%2Bunderlying%2Bmutant%2Bphenotypes%2Bin%2Bhigh-copy%2BMutator%2Blines%2Bof%2Bmaize.%26rft_id%253Dinfo%253Apmid%252F20409008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=20409008&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F413.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000279406400013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.413.53" data-doi="10.1186/1471-2229-8-103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xin</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barkley</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burow</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franks</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Applying genotyping (TILLING) and phenotyping analyses to elucidate gene function in a chemically induced sorghum mutant population.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">103</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DXin%26rft.auinit1%253DZ.%26rft.volume%253D8%26rft.spage%253D103%26rft.epage%253D103%26rft.atitle%253DApplying%2Bgenotyping%2B%2528TILLING%2529%2Band%2Bphenotyping%2Banalyses%2Bto%2Belucidate%2Bgene%2Bfunction%2Bin%2Ba%2Bchemically%2Binduced%2Bsorghum%2Bmutant%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-8-103%26rft_id%253Dinfo%253Apmid%252F18854043%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-8-103&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18854043&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F413.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.2.413.54" data-doi="10.1534/genetics.112.144436"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ayele</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jogi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">a</span> <span class="cit-article-title">High-throughput discovery of mutations in tef semi-dwarfing genes by next-generation sequencing analysis.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">819</span>–<span class="cit-lpage">829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.144436%26rft_id%253Dinfo%253Apmid%252F22904035%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkyLzMvODE5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzQxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.2.413.55" data-doi="10.1534/genetics.112.141986"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mang</span>  <span class="cit-name-given-names">H. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qian</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hipps</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">b</span> <span class="cit-article-title">Gene discovery using mutagen-induced polymorphisms and deep sequencing: application to plant disease resistance.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">139</span>–<span class="cit-lpage">146</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.141986%26rft_id%253Dinfo%253Apmid%252F22714407%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkyLzEvMTM5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzQxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/2/413.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/2/399" title="Genetic Causes of Phenotypic Adaptation to the Second Fermentation of Sparkling Wines in Saccharomyces cerevisiae" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/2/427" title="Genomics of Compensatory Adaptation in Experimental Populations of Aspergillus nidulans" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/2"><span>Volume 7 Issue 2, February 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/2.cover-source.jpg?itok=m4uMC43I" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/2/413" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-jEO6Ifp3ydeUf6RflTp_72cKsA1xA6HuRhGSC6xNrkA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/2/413" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/2/413" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/2/413" class="active" data-icon-position="" data-hide-link-title="0">Forward Genetics by Sequencing EMS Variation-Induced Inbred Lines</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460567" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-Eg7bDOsuVAjEamKK_AYOLgkR8Msthd2zlvh1kAq9qec" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/2/413.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/2/413" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-YZpLlA02GenXC3yOim4ZdF27a57R8YV2XCo-koQmUS4" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/2/413" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/2/413.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460567" id="node460567--2" data-pisa="ggg;7/2/413" data-pisa-master="ggg;g3.116.029660" data-apath="/ggg/7/2/413.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Forward Genetics by Sequencing EMS Variation-Induced Inbred Lines</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-0598-5980" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Charles Addo-Quaye</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-1540-8928" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Elizabeth Buescher</span>, <span class="highwire-citation-author" data-delta="2">Norman Best</span>, <span class="highwire-citation-author" data-delta="3">Vijay Chaikam</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-6680-1722" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ivan Baxter</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-2799-954X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Brian P. Dilkes</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">413-425;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.029660 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460567--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Charles Addo-Quaye </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry, Purdue University, West Lafayette, Indiana 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BAddo-Quaye%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Addo-Quaye%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACharles%2BAddo-Quaye%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0598-5980" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Charles Addo-Quaye</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Elizabeth Buescher </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry, Purdue University, West Lafayette, Indiana 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elizabeth%2BBuescher%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Buescher%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AElizabeth%2BBuescher%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1540-8928" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Elizabeth Buescher</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Norman Best </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry, Purdue University, West Lafayette, Indiana 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Norman%2BBest%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Best%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANorman%2BBest%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Vijay Chaikam </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry, Purdue University, West Lafayette, Indiana 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vijay%2BChaikam%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chaikam%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVijay%2BChaikam%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Ivan Baxter </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>United States Department of Agriculture-Agricultural Research Service, Donald Danforth Plant Science Center, St. Louis, Missouri 63132</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ivan%2BBaxter%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baxter%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AIvan%2BBaxter%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6680-1722" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ivan Baxter</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Brian P. Dilkes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry, Purdue University, West Lafayette, Indiana 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brian%2BP.%2BDilkes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dilkes%20BP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABrian%2BP.%2BDilkes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-2799-954X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Brian P. Dilkes</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bdilkes@purdue.edu" class="" data-icon-position="" data-hide-link-title="0">bdilkes@purdue.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/2/413" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/2/413&amp;title=Forward%20Genetics%20by%20Sequencing%20EMS%20Variation-Induced%20Inbred%20Lines" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/2/413&amp;title=Forward%20Genetics%20by%20Sequencing%20EMS%20Variation-Induced%20Inbred%20Lines" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/2/413&amp;title=Forward%20Genetics%20by%20Sequencing%20EMS%20Variation-Induced%20Inbred%20Lines" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/2/413&amp;text=Forward%20Genetics%20by%20Sequencing%20EMS%20Variation-Induced%20Inbred%20Lines" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/2/413&amp;title=Forward%20Genetics%20by%20Sequencing%20EMS%20Variation-Induced%20Inbred%20Lines" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/2/413&amp;t=Forward%20Genetics%20by%20Sequencing%20EMS%20Variation-Induced%20Inbred%20Lines" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/2/413&amp;title=Forward%20Genetics%20by%20Sequencing%20EMS%20Variation-Induced%20Inbred%20Lines" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/2/413&amp;title=Forward%20Genetics%20by%20Sequencing%20EMS%20Variation-Induced%20Inbred%20Lines" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460567/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460567" id="node460567--3" data-pisa="ggg;7/2/413" data-pisa-master="ggg;g3.116.029660" data-apath="/ggg/7/2/413.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Forward Genetics by Sequencing EMS Variation-Induced Inbred Lines</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-0598-5980" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Charles Addo-Quaye</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-1540-8928" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Elizabeth Buescher</span>, <span class="highwire-citation-author" data-delta="2">Norman Best</span>, <span class="highwire-citation-author" data-delta="3">Vijay Chaikam</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-6680-1722" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ivan Baxter</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-2799-954X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Brian P. Dilkes</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">413-425;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.029660 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460567--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Charles Addo-Quaye </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry, Purdue University, West Lafayette, Indiana 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BAddo-Quaye%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Addo-Quaye%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACharles%2BAddo-Quaye%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0598-5980" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Charles Addo-Quaye</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Elizabeth Buescher </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry, Purdue University, West Lafayette, Indiana 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elizabeth%2BBuescher%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Buescher%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AElizabeth%2BBuescher%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1540-8928" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Elizabeth Buescher</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Norman Best </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry, Purdue University, West Lafayette, Indiana 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Norman%2BBest%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Best%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANorman%2BBest%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Vijay Chaikam </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry, Purdue University, West Lafayette, Indiana 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vijay%2BChaikam%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chaikam%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVijay%2BChaikam%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Ivan Baxter </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>United States Department of Agriculture-Agricultural Research Service, Donald Danforth Plant Science Center, St. Louis, Missouri 63132</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ivan%2BBaxter%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baxter%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AIvan%2BBaxter%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6680-1722" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ivan Baxter</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Brian P. Dilkes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry, Purdue University, West Lafayette, Indiana 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brian%2BP.%2BDilkes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dilkes%20BP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABrian%2BP.%2BDilkes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-2799-954X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Brian P. Dilkes</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bdilkes@purdue.edu" class="" data-icon-position="" data-hide-link-title="0">bdilkes@purdue.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=413&amp;rft.epage=425&amp;rft.atitle=Forward%20Genetics%20by%20Sequencing%20EMS%20Variation-Induced%20Inbred%20Lines&amp;rft.volume=7&amp;rft.issue=2&amp;rft.date=2017-02-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Addo-Quaye%2C+Charles&amp;rft.au=Buescher%2C+Elizabeth&amp;rft.au=Best%2C+Norman&amp;rft.au=Chaikam%2C+Vijay&amp;rft.au=Baxter%2C+Ivan&amp;rft.au=Dilkes%2C+Brian+P."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460567/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460567/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460567/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460567/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460567/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460567/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460567/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460567/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460567/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460567/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460567/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460567/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/2/413" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-21" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/2/413.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/2/413.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/2/413.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/2/413.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
