<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/2/625" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/2/625.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/2/625.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/2/625.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/2/625.atom" />
<meta name="HW.pisa" content="ggg;7/2/625" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="upSET, the Drosophila homologue of SET3, Is Required for Viability and the Proper Balance of Active and Repressive Chromatin Marks" />
<meta name="DC.Identifier" content="10.1534/g3.116.037788" />
<meta name="DC.Date" content="2017-02-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 McElroy et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Chromatin plays a critical role in faithful implementation of gene expression programs. Different post-translational modifications (PTMs) of histone proteins reflect the underlying state of gene activity, and many chromatin proteins write, erase, bind, or are repelled by, these histone marks. One such protein is UpSET, the Drosophila homolog of yeast Set3 and mammalian KMT2E (MLL5). Here, we show that UpSET is necessary for the proper balance between active and repressed states. Using CRISPR/Cas-9 editing, we generated S2 cells that are mutant for [upSET][1] . We found that loss of UpSET is tolerated in S2 cells, but that heterochromatin is misregulated, as evidenced by a strong decrease in H3K9me2 levels assessed by bulk histone PTM quantification. To test whether this finding was consistent in the whole organism, we deleted the [upSET][1] coding sequence using CRISPR/Cas-9, which we found to be lethal in both sexes in flies. We were able to rescue this lethality using a tagged [upSET][1] transgene, and found that UpSET protein localizes to transcriptional start sites (TSS) of active genes throughout the genome. Misregulated heterochromatin is apparent by suppressed position effect variegation of the wm4 allele in heterozygous [upSET][1] -deleted flies. Using nascent-RNA sequencing in the [upSET][1] -mutant S2 lines, we show that this result applies to heterochromatin genes generally. Our findings support a critical role for UpSET in maintaining heterochromatin, perhaps by delimiting the active chromatin environment.

 [1]: http://flybase.org/reports/FBgn0036398.html" />
<meta name="DC.Contributor" content="Kyle A. McElroy" />
<meta name="DC.Contributor" content="Youngsook L. Jung" />
<meta name="DC.Contributor" content="Barry M. Zee" />
<meta name="DC.Contributor" content="Charlotte I. Wang" />
<meta name="DC.Contributor" content="Peter J. Park" />
<meta name="DC.Contributor" content="Mitzi I. Kuroda" />
<meta name="article:published_time" content="2017-02-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="upSET, the Drosophila homologue of SET3, Is Required for Viability and the Proper Balance of Active and Repressive Chromatin Marks" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Chromatin plays a critical role in faithful implementation of gene expression programs. Different post-translational modifications (PTMs) of histone proteins reflect the underlying state of gene activity, and many chromatin proteins write, erase, bind, or are repelled by, these histone marks. One such protein is UpSET, the &lt;i&gt;Drosophila&lt;/i&gt; homolog of yeast Set3 and mammalian KMT2E (MLL5). Here, we show that UpSET is necessary for the proper balance between active and repressed states. Using CRISPR/Cas-9 editing, we generated S2 cells that are mutant for &lt;i&gt;upSET&lt;/i&gt;. We found that loss of UpSET is tolerated in S2 cells, but that heterochromatin is misregulated, as evidenced by a strong decrease in H3K9me2 levels assessed by bulk histone PTM quantification. To test whether this finding was consistent in the whole organism, we deleted the &lt;i&gt;upSET&lt;/i&gt; coding sequence using CRISPR/Cas-9, which we found to be lethal in both sexes in flies. We were able to rescue this lethality using a tagged &lt;i&gt;upSET&lt;/i&gt; transgene, and found that UpSET protein localizes to transcriptional start sites (TSS) of active genes throughout the genome. Misregulated heterochromatin is apparent by suppressed position effect variegation of the &lt;i&gt;w&lt;sup&gt;m4&lt;/sup&gt;&lt;/i&gt; allele in heterozygous &lt;i&gt;upSET&lt;/i&gt;-deleted flies. Using nascent-RNA sequencing in the &lt;i&gt;upSET&lt;/i&gt;-mutant S2 lines, we show that this result applies to heterochromatin genes generally. Our findings support a critical role for UpSET in maintaining heterochromatin, perhaps by delimiting the active chromatin environment.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/02/01" />
<meta name="citation_mjid" content="ggg;7/2/625" />
<meta name="citation_id" content="7/2/625" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/2/625" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/2/625.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/2/625.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/2/625.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.037788" />
<meta name="citation_pmid" content="28064188" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="2" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="625" />
<meta name="citation_lastpage" content="635" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Kyle A. McElroy" />
<meta name="citation_author_institution" content="Division of Genetics, Brigham and Women’s Hospital, Boston, Massachusetts 02115" />
<meta name="citation_author_institution" content="Department of Genetics, Harvard Medical School, Boston, Massachusetts 02115" />
<meta name="citation_author_institution" content="Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138" />
<meta name="citation_author" content="Youngsook L. Jung" />
<meta name="citation_author_institution" content="Division of Genetics, Brigham and Women’s Hospital, Boston, Massachusetts 02115" />
<meta name="citation_author_institution" content="Center for Biomedical Informatics, Harvard Medical School, Boston, Massachusetts 02115" />
<meta name="citation_author" content="Barry M. Zee" />
<meta name="citation_author_institution" content="Division of Genetics, Brigham and Women’s Hospital, Boston, Massachusetts 02115" />
<meta name="citation_author_institution" content="Department of Genetics, Harvard Medical School, Boston, Massachusetts 02115" />
<meta name="citation_author" content="Charlotte I. Wang" />
<meta name="citation_author_institution" content="Division of Genetics, Brigham and Women’s Hospital, Boston, Massachusetts 02115" />
<meta name="citation_author_institution" content="Department of Genetics, Harvard Medical School, Boston, Massachusetts 02115" />
<meta name="citation_author" content="Peter J. Park" />
<meta name="citation_author_institution" content="Division of Genetics, Brigham and Women’s Hospital, Boston, Massachusetts 02115" />
<meta name="citation_author_institution" content="Center for Biomedical Informatics, Harvard Medical School, Boston, Massachusetts 02115" />
<meta name="citation_author" content="Mitzi I. Kuroda" />
<meta name="citation_author_institution" content="Division of Genetics, Brigham and Women’s Hospital, Boston, Massachusetts 02115" />
<meta name="citation_author_institution" content="Department of Genetics, Harvard Medical School, Boston, Massachusetts 02115" />
<meta name="citation_author_email" content="mkuroda@genetics.med.harvard.edu" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=A. A. Alekseyenko;citation_author=S. Peng;citation_author=E. Larschan;citation_author=A. A. Gorchakov;citation_author=O. K. Lee;citation_title=A sequence motif within chromatin entry sites directs MSL establishment on the Drosophila X chromosome.;citation_pages=599-609;citation_volume=134;citation_year=2008;citation_issue=4;citation_pmid=18724933;citation_doi=10.1016/j.cell.2008.06.033" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=A. A. Alekseyenko;citation_author=E. M. Walsh;citation_author=X. Wang;citation_author=A. R. Grayson;citation_author=P. T. Hsi;citation_title=The oncogenic BRD4-NUT chromatin regulator drives aberrant transcription within large topological domains;citation_pages=1507-1523;citation_volume=29;citation_year=2015;citation_issue=14;citation_pmid=26220994;citation_doi=10.1101/gad.267583.115" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. Ali;citation_author=H. Rincon-Arano;citation_author=W. Zhao;citation_author=S. B. Rothbart;citation_author=Q. Tong;citation_title=Molecular basis for chromatin binding and regulation of MLL5.;citation_volume=110;citation_year=2013;citation_pmid=23798402;citation_doi=10.1073/pnas.1310156110" />
<meta name="citation_reference" content="citation_journal_title=Cell Rep.;citation_author=A. R. Bassett;citation_author=C. Tibbit;citation_author=C. P. Ponting;citation_author=J. L. Liu;citation_title=HIGHLY EFFICIENT TARGETED MUTAGENESIS OF DROSOPHILA WITH THE CRISPR/CAS9 SYSTEM.;citation_pages=220-228;citation_volume=4;citation_year=2013;citation_pmid=23827738;citation_doi=10.1016/j.celrep.2013.06.020" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. Boeke;citation_author=C. Regnard;citation_author=W. Cai;citation_author=J. Johansen;citation_author=K. M. Johansen;citation_title=Phosphorylation of SU(VAR)3-9 by the chromosomal kinase JIL-1.;citation_pages=e10042-e10042;citation_volume=5;citation_year=2010;citation_issue=4;citation_pmid=20386606;citation_doi=10.1371/journal.pone.0010042" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=E. P. M. Candido;citation_author=R. Reeves;citation_author=J. R. Davie;citation_title=Sodium butyrate inhibits histone deacetylation in cultured cells.;citation_pages=105-113;citation_volume=14;citation_year=1978;citation_issue=1;citation_pmid=667927;citation_doi=10.1016/0092-8674(78)90305-7" />
<meta name="citation_reference" content="citation_journal_title=Anal. Chem.;citation_author=F. Capuano;citation_author=M. Mulleder;citation_author=R. Kok;citation_author=H. J. Blom;citation_author=M. Ralser;citation_title=Cytosine DNA methylation is found in Drosophila melanogaster but absent in Saccharomyces cerevisiae, Schizosaccharomyces pombe, and other yeast species.;citation_pages=3697-3702;citation_volume=86;citation_year=2014;citation_pmid=24640988;citation_doi=10.1021/ac500447w" />
<meta name="citation_reference" content="citation_journal_title=The international journal of biochemistry &amp; cell biology;citation_journal_abbrev=Int J Biochem Cell Biol;citation_author=F. Cheng;citation_author=J. Liu;citation_author=S. H. Zhou;citation_author=X. N. Wang;citation_author=J. F. Chew;citation_title=RNA interference against mixed lineage leukemia 5 resulted in cell cycle arrest.;citation_pages=2472-2481;citation_volume=40;citation_year=2008;citation_issue=11;citation_pmid=18573682;citation_doi=10.1016/j.biocel.2008.04.012" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=O. V. Demakova;citation_author=G. V. Pokholkova;citation_author=T. D. Kolesnikova;citation_author=S. A. Demakov;citation_author=E. N. Andreyeva;citation_title=The SU(VAR)3–9/HP1 complex differentially regulates the compaction state and degree of underreplication of X chromosome pericentric heterochromatin in Drosophila melanogaster.;citation_volume=175;citation_year=2007;citation_pmid=17151257;citation_doi=10.1534/genetics.106.062133" />
<meta name="citation_reference" content="citation_journal_title=Chromosoma;citation_journal_abbrev=Chromosoma;citation_author=H. Deng;citation_author=W. Zhang;citation_author=X. Bao;citation_author=J. N. Martin;citation_author=J. Girton;citation_title=The JIL-1 kinase regulates the structure of Drosophila polytene chromosomes.;citation_pages=173-182;citation_volume=114;citation_year=2005;citation_issue=3;citation_pmid=15986206;citation_doi=10.1007/s00412-005-0006-8" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. Deng;citation_author=X. Bao;citation_author=W. Zhang;citation_author=J. Girton;citation_author=J. Johansen;citation_title=Reduced levels of Su(var)3–9 but not Su(var)2–5 (HP1) counteract the effects on chromatin structure and viability in loss-of-function mutants of the JIL-1 histone H3S10 kinase.;citation_volume=177;citation_year=2007;citation_pmid=17660558;citation_doi=10.1534/genetics.107.075143" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=L. W. Deng;citation_author=I. Chiu;citation_author=J. L. Strominger;citation_title=MLL 5 protein forms intranuclear foci, and overexpression inhibits cell cycle progression.;citation_volume=101;citation_year=2004;citation_pmid=14718661;citation_doi=10.1073/pnas.2036345100" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=A. Ebert;citation_author=G. Schotta;citation_author=S. Lein;citation_author=S. Kubicek;citation_author=V. Krauss;citation_title=Su(var) genes regulate the balance between euchromatin and heterochromatin in Drosophila;citation_pages=2973-2983;citation_volume=18;citation_year=2004;citation_issue=23;citation_pmid=15574598;citation_doi=10.1101/gad.323004" />
<meta name="citation_reference" content="citation_journal_title=Cold Spring Harbor Perspectives in Biology;citation_journal_abbrev=Cold Spring Harb. Perspect. Biol.;citation_author=S. C. Elgin;citation_author=G. Reuter;citation_title=Position-Effect Variegation, Heterochromatin Formation, and Gene Silencing in Drosophila;citation_pages=a017780-a017780;citation_volume=5;citation_year=2013;citation_issue=8;citation_pmid=23906716;citation_doi=10.1101/cshperspect.a017780" />
<meta name="citation_reference" content="citation_journal_title=Oncogene;citation_journal_abbrev=Oncogene;citation_author=B. M. Emerling;citation_author=J. Bonifas;citation_author=C. P. Kratz;citation_author=S. Donovan;citation_author=B. R. Taylor;citation_title=MLL5, a homolog of Drosophila trithorax located within a segment of chromosome band 7q22 implicated in myeloid leukemia.;citation_pages=4849-4854;citation_volume=21;citation_year=2002;citation_issue=31;citation_pmid=12101424;citation_doi=10.1038/sj.onc.1205615" />
<meta name="citation_reference" content="citation_journal_title=Cell Rep;citation_author=F. Ferrari;citation_author=A. Plachetka;citation_author=A. A. Alekseyenko;citation_author=Y. L. Jung;citation_author=F. Ozsolak;citation_title=“Jump start and gain” model for dosage compensation in Drosophila based on direct sequencing of nascent transcripts.;citation_pages=629-636;citation_volume=5;citation_year=2013;citation_pmid=24183666;citation_doi=10.1016/j.celrep.2013.09.037" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=M. B. Gerstein;citation_author=J. Rozowsky;citation_author=K. K. Yan;citation_author=D. Wang;citation_author=C. Cheng;citation_title=Comparative analysis of the transcriptome across distant species.;citation_pages=445-448;citation_volume=512;citation_year=2014;citation_pmid=25164755;citation_doi=10.1038/nature13424" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. J. Gratz;citation_author=A. M. Cummings;citation_author=J. N. Nguyen;citation_author=D. C. Hamm;citation_author=L. K. Donohue;citation_title=Genome engineering of Drosophila with the CRISPR RNA-guided Cas9 nuclease.;citation_volume=194;citation_year=2013;citation_pmid=23709638;citation_doi=10.1534/genetics.113.152710" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. J. Gratz;citation_author=F. P. Ukken;citation_author=C. D. Rubinstein;citation_author=G. Thiede;citation_author=L. K. Donohue;citation_title=Highly specific and efficient CRISPR/Cas9-catalyzed homology-directed repair in Drosophila.;citation_volume=196;citation_year=2014;citation_pmid=24478335;citation_doi=10.1534/genetics.113.160713" />
<meta name="citation_reference" content="citation_journal_title=Blood;citation_author=M. Heuser;citation_author=D. B. Yap;citation_author=M. Leung;citation_author=T. R. de Algara;citation_author=A. Tafech;citation_title=Loss of MLL5 results in pleiotropic hematopoietic defects, reduced neutrophil immune function, and extreme sensitivity to DNA demethylation.;citation_volume=113;citation_year=2009;citation_pmid=18854576;citation_doi=10.1182/blood-2008-06-162263" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=J. W. Ho;citation_author=Y. L. Jung;citation_author=T. Liu;citation_author=B. H. Alver;citation_author=S. Lee;citation_title=Comparative analysis of metazoan chromatin organization.;citation_pages=449-452;citation_volume=512;citation_year=2014;citation_pmid=25164756;citation_doi=10.1038/nature13415" />
<meta name="citation_reference" content="citation_journal_title=Methods Enzymol.;citation_author=B. E. Housden;citation_author=S. Lin;citation_author=N. Perrimon;citation_title=Cas9-based genome editing in Drosophila.;citation_pages=415-439;citation_volume=546;citation_year=2014;citation_pmid=25398351;citation_doi=10.1016/B978-0-12-801185-0.00019-2" />
<meta name="citation_reference" content="citation_journal_title=Science Signaling;citation_journal_abbrev=Sci Signal;citation_author=B. E. Housden;citation_author=A. J. Valvezan;citation_author=C. Kelley;citation_author=R. Sopko;citation_author=Y. Hu;citation_title=Identification of potential drug targets for tuberous sclerosis complex by synthetic screens combining CRISPR-based knockouts with RNAi;citation_pages=rs9-rs9;citation_volume=8;citation_year=2015;citation_issue=393;citation_pmid=26350902;citation_doi=10.1126/scisignal.aab3729" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=S. A. Jacobs;citation_author=S. Khorasanizadeh;citation_title=Structure of HP1 chromodomain bound to a lysine 9-methylated histone H3 tail.;citation_volume=295;citation_year=2002;citation_pmid=11859155;citation_doi=10.1126/science.1069473" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=Y. Jin;citation_author=Y. Wang;citation_author=D. L. Walker;citation_author=H. Dong;citation_author=C. Conley;citation_title=JIL-1: a novel chromosomal tandem kinase implicated in transcriptional regulation in Drosophila.;citation_pages=129-135;citation_volume=4;citation_year=1999;citation_issue=1;citation_pmid=10445035;citation_doi=10.1016/S1097-2765(00)80195-1" />
<meta name="citation_reference" content="citation_journal_title=J. Cell Biol.;citation_author=Y. Jin;citation_author=Y. Wang;citation_author=J. Johansen;citation_author=K. M. Johansen;citation_title=JIL-1, a chromosomal kinase implicated in regulation of chromatin structure, associates with the male specific lethal (MSL) dosage compensation complex.;citation_volume=149;citation_year=2000;citation_pmid=10831604;citation_doi=10.1083/jcb.149.5.1005" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=P. V. Kharchenko;citation_author=M. Y. Tolstorukov;citation_author=P. J. Park;citation_title=Design and analysis of ChIP-seq experiments for DNA-binding proteins.;citation_pages=1351-1359;citation_volume=26;citation_year=2008;citation_issue=12;citation_pmid=19029915;citation_doi=10.1038/nbt.1508" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=P. V. Kharchenko;citation_author=A. A. Alekseyenko;citation_author=Y. B. Schwartz;citation_author=A. Minoda;citation_author=N. C. Riddle;citation_title=Comprehensive analysis of the chromatin landscape in Drosophila melanogaster.;citation_pages=480-485;citation_volume=471;citation_year=2011;citation_issue=7339;citation_pmid=21179089;citation_doi=10.1038/nature09725" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=T. Kim;citation_author=S. Buratowski;citation_title=Dimethylation of H3K4 by Set1 recruits the Set3 histone deacetylase complex to 5' transcribed regions.;citation_pages=259-272;citation_volume=137;citation_year=2009;citation_issue=2;citation_pmid=19379692;citation_doi=10.1016/j.cell.2009.02.045" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=T. Kim;citation_author=Z. Xu;citation_author=S. Clauder-Munster;citation_author=L. M. Steinmetz;citation_author=S. Buratowski;citation_title=Set3 HDAC mediates effects of overlapping noncoding transcription on gene induction kinetics.;citation_pages=1158-1169;citation_volume=150;citation_year=2012;citation_issue=6;citation_pmid=22959268;citation_doi=10.1016/j.cell.2012.08.016" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=S. K. Koya;citation_author=V. H. Meller;citation_title=Modulation of heterochromatin by male specific lethal proteins and roX RNA in Drosophila melanogaster males.;citation_pages=e0140259;citation_volume=10;citation_year=2015;citation_pmid=26468879;citation_doi=10.1371/journal.pone.0140259" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=B. Langmead;citation_author=C. Trapnell;citation_author=M. Pop;citation_author=S. L. Salzberg;citation_title=Ultrafast and memory-efficient alignment of short DNA sequences to the human genome.;citation_pages=R25-R25;citation_volume=10;citation_year=2009;citation_issue=3;citation_pmid=19261174;citation_doi=10.1186/gb-2009-10-3-r25" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=A. Lemak;citation_author=A. Yee;citation_author=H. Wu;citation_author=D. Yap;citation_author=H. Zeng;citation_title=Solution NMR structure and histone binding of the PHD domain of human MLL5.;citation_pages=e77020;citation_volume=8;citation_year=2013;citation_pmid=24130829;citation_doi=10.1371/journal.pone.0077020" />
<meta name="citation_reference" content="citation_journal_title=EMBO J.;citation_author=B. Y. Lu;citation_author=C. P. Bishop;citation_author=J. C. Eissenberg;citation_title=Developmental timing and tissue specificity of heterochromatin-mediated silencing.;citation_pages=1323-1332;citation_volume=15;citation_year=1996;citation_pmid=8635465" />
<meta name="citation_reference" content="citation_journal_title=Cold Spring Harbor Perspectives in Biology;citation_journal_abbrev=Cold Spring Harb. Perspect. Biol.;citation_author=J. C. Lucchesi;citation_author=M. I. Kuroda;citation_title=Dosage Compensation in Drosophila;citation_pages=a019398-a019398;citation_volume=7;citation_year=2015;citation_issue=5;citation_pmid=25934013;citation_doi=10.1101/cshperspect.a019398" />
<meta name="citation_reference" content="citation_journal_title=Blood;citation_author=V. Madan;citation_author=B. Madan;citation_author=U. Brykczynska;citation_author=F. Zilbermann;citation_author=K. Hogeveen;citation_title=Impaired function of primitive hematopoietic cells in mice lacking the Mixed-Lineage-Leukemia homolog MLL5.;citation_volume=113;citation_year=2009;citation_pmid=18952892;citation_doi=10.1182/blood-2008-02-142638" />
<meta name="citation_reference" content="citation_journal_title=EMBnet.journal;citation_author=M. Martin;citation_title=Cutadapt removes adapter sequences from high-throughput sequencing reads.;citation_pages=10-12;citation_volume=17;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Biotechniques;citation_author=D. F. B. Miller;citation_author=S. L. Holtzman;citation_author=T. C. Kaufman;citation_title=Customized microinjection glass capillary needles for P-element transformations in Drosophila melanogaster.;citation_pages=366-375;citation_volume=33;citation_year=2002;citation_pmid=12188189" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=W. W. Pijnappel;citation_author=D. Schaft;citation_author=A. Roguev;citation_author=A. Shevchenko;citation_author=H. Tekotte;citation_title=The S. cerevisiae SET3 complex includes two histone deacetylases, Hos2 and Hst1, and is a meiotic-specific repressor of the sporulation gene program;citation_pages=2991-3004;citation_volume=15;citation_year=2001;citation_issue=22;citation_pmid=11711434;citation_doi=10.1101/gad.207401" />
<meta name="citation_reference" content="citation_journal_title=EMBO J.;citation_author=J. S. Platero;citation_author=T. Hartnett;citation_author=J. C. Eissenberg;citation_title=Functional analysis of the chromo domain of HP1.;citation_pages=3977-3986;citation_volume=14;citation_year=1995;citation_pmid=7664737" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=N. C. Riddle;citation_author=A. Minoda;citation_author=P. V. Kharchenko;citation_author=A. A. Alekseyenko;citation_author=Y. B. Schwartz;citation_title=Plasticity in patterns of histone modifications and chromosomal proteins in Drosophila heterochromatin.;citation_volume=21;citation_year=2011;citation_pmid=21177972;citation_doi=10.1101/gr.110098.110" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=H. Rincon-Arano;citation_author=J. Halow;citation_author=J. J. Delrow;citation_author=S. M. Parkhurst;citation_author=M. Groudine;citation_title=UpSET recruits HDAC complexes and restricts chromatin accessibility and acetylation at promoter regions.;citation_pages=1214-1228;citation_volume=151;citation_year=2012;citation_pmid=23177352;citation_doi=10.1016/j.cell.2012.11.009" />
<meta name="citation_reference" content="citation_journal_title=Fly (Austin);citation_author=H. Rincon-Arano;citation_author=S. M. Parkhurst;citation_author=M. Groudine;citation_title=UpSET-ing the balance.;citation_pages=1-8;citation_volume=7;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. D. Robinson;citation_author=D. J. McCarthy;citation_author=G. K. Smyth;citation_title=edgeR: a bioconductor package for differential expression analysis of digital gene expression data.;citation_volume=26;citation_year=2010;citation_pmid=19910308;citation_doi=10.1093/bioinformatics/btp616" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=C. S. Ross-Innes;citation_author=R. Stark;citation_author=A. E. Teschendorff;citation_author=K. A. Holmes;citation_author=H. R. Ali;citation_title=Differential oestrogen receptor binding is associated with clinical outcome in breast cancer.;citation_pages=389-393;citation_volume=481;citation_year=2012;citation_issue=7381;citation_pmid=22217937;citation_doi=10.1038/nature10730" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Shin;citation_author=T. Liu;citation_author=A. K. Manrai;citation_author=X. S. Liu;citation_title=CEAS: cis-regulatory element annotation system.;citation_volume=25;citation_year=2009;citation_pmid=19689956;citation_doi=10.1093/bioinformatics/btp479" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=A. Spierer;citation_author=F. Begeot;citation_author=P. Spierer;citation_author=M. Delattre;citation_title=SU(VAR)3-7 links heterochromatin and dosage compensation in Drosophila.;citation_pages=e1000066-e1000066;citation_volume=4;citation_year=2008;citation_issue=5;citation_pmid=18451980;citation_doi=10.1371/journal.pgen.1000066" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=S. Takayama;citation_author=J. Dhahbi;citation_author=A. Roberts;citation_author=G. Mao;citation_author=S. J. Heo;citation_title=Genome methylation in D. melanogaster is found at specific short motifs and is independent of DNMT2 activity.;citation_volume=24;citation_year=2014;citation_pmid=24558263;citation_doi=10.1101/gr.162412.113" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Biol.;citation_author=A. L. Torres-Machorro;citation_author=L. G. Clark;citation_author=C. S. Chang;citation_author=L. Pillus;citation_title=The set3 complex antagonizes the MYST acetyltransferase Esa1 in the DNA damage response.;citation_volume=35;citation_year=2015;citation_pmid=26303527;citation_doi=10.1128/MCB.00298-15" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. Wang;citation_author=S. K. Kurdistani;citation_author=M. Grunstein;citation_title=Requirement of Hos2 Histone Deacetylase for Gene Activity in Yeast;citation_pages=1412-1414;citation_volume=298;citation_year=2002;citation_issue=5597;citation_pmid=12434058;citation_doi=10.1126/science.1077790" />
<meta name="citation_reference" content="citation_journal_title=Chromosoma;citation_author=C. Wang;citation_author=Y. Li;citation_author=W. Cai;citation_author=X. Bao;citation_author=J. Girton;citation_title=Histone H3S10 phosphorylation by the JIL-1 kinase in pericentric heterochromatin and on the fourth chromosome creates a composite H3S10phK9me2 epigenetic mark.;citation_pages=273-280;citation_volume=123;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Nat. Struct. Mol. Biol.;citation_author=C. I. Wang;citation_author=A. A. Alekseyenko;citation_author=G. LeRoy;citation_author=A. E. Elia;citation_author=A. A. Gorchakov;citation_title=Chromatin proteins captured by ChIP-mass spectrometry are linked to dosage compensation in Drosophila.;citation_pages=202-209;citation_volume=20;citation_year=2013;citation_pmid=23295261;citation_doi=10.1038/nsmb.2477" />
<meta name="citation_reference" content="citation_journal_title=Haematologica;citation_author=H. Yun;citation_author=F. Damm;citation_author=D. Yap;citation_author=A. Schwarzer;citation_author=A. Chaturvedi;citation_title=Impact of MLL5 expression on decitabine efficacy and DNA methylation in acute myeloid leukemia.;citation_volume=99;citation_year=2014;citation_pmid=24895338;citation_doi=10.3324/haematol.2013.101386" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=B. M. Zee;citation_author=A. A. Alekseyenko;citation_author=K. A. McElroy;citation_author=M. I. Kuroda;citation_title=Streamlined discovery of cross-linked chromatin complexes and associated histone modifications by mass spectrometry.;citation_volume=113;citation_year=2016;citation_pmid=26831069;citation_doi=10.1073/pnas.1522750113" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=B. M. Zee;citation_author=A. B. Dibona;citation_author=A. A. Alekseyenko;citation_author=C. A. French;citation_author=M. I. Kuroda;citation_title=The oncoprotein BRD4-NUT generates aberrant histone modification patterns.;citation_pages=e0163820;citation_volume=11;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Blood;citation_author=Y. Zhang;citation_author=J. Wong;citation_author=M. Klinger;citation_author=M. T. Tran;citation_author=K. M. Shannon;citation_title=MLL5 contributes to hematopoietic stem cell fitness and homeostasis.;citation_volume=113;citation_year=2009;citation_pmid=18818388;citation_doi=10.1182/blood-2008-05-159905" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="upSET, the Drosophila homologue of SET3, Is Required for Viability and the Proper Balance of Active and Repressive Chromatin Marks" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2.cover-source.jpg" />
<meta name="twitter:description" content="Chromatin plays a critical role in faithful implementation of gene expression programs. Different post-translational modifications (PTMs) of histone proteins reflect the underlying state of gene activity, and many chromatin proteins write, erase, bind, or are repelled by, these histone marks. One such protein is UpSET, the Drosophila homolog of yeast Set3 and mammalian KMT2E (MLL5). Here, we show that UpSET is necessary for the proper balance between active and repressed states. Using CRISPR/Cas-9 editing, we generated S2 cells that are mutant for [upSET][1] . We found that loss of UpSET is tolerated in S2 cells, but that heterochromatin is misregulated, as evidenced by a strong decrease in H3K9me2 levels assessed by bulk histone PTM quantification. To test whether this finding was consistent in the whole organism, we deleted the [upSET][1] coding sequence using CRISPR/Cas-9, which we found to be lethal in both sexes in flies. We were able to rescue this lethality using a tagged [upSET][1] transgene, and found that UpSET protein localizes to transcriptional start sites (TSS) of active genes throughout the genome. Misregulated heterochromatin is apparent by suppressed position effect variegation of the wm4 allele in heterozygous [upSET][1] -deleted flies. Using nascent-RNA sequencing in the [upSET][1] -mutant S2 lines, we show that this result applies to heterochromatin genes generally. Our findings support a critical role for UpSET in maintaining heterochromatin, perhaps by delimiting the active chromatin environment.

 [1]: http://flybase.org/reports/FBgn0036398.html" />
<meta name="og-title" property="og:title" content="upSET, the Drosophila homologue of SET3, Is Required for Viability and the Proper Balance of Active and Repressive Chromatin Marks" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/2/625" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Chromatin plays a critical role in faithful implementation of gene expression programs. Different post-translational modifications (PTMs) of histone proteins reflect the underlying state of gene activity, and many chromatin proteins write, erase, bind, or are repelled by, these histone marks. One such protein is UpSET, the Drosophila homolog of yeast Set3 and mammalian KMT2E (MLL5). Here, we show that UpSET is necessary for the proper balance between active and repressed states. Using CRISPR/Cas-9 editing, we generated S2 cells that are mutant for [upSET][1] . We found that loss of UpSET is tolerated in S2 cells, but that heterochromatin is misregulated, as evidenced by a strong decrease in H3K9me2 levels assessed by bulk histone PTM quantification. To test whether this finding was consistent in the whole organism, we deleted the [upSET][1] coding sequence using CRISPR/Cas-9, which we found to be lethal in both sexes in flies. We were able to rescue this lethality using a tagged [upSET][1] transgene, and found that UpSET protein localizes to transcriptional start sites (TSS) of active genes throughout the genome. Misregulated heterochromatin is apparent by suppressed position effect variegation of the wm4 allele in heterozygous [upSET][1] -deleted flies. Using nascent-RNA sequencing in the [upSET][1] -mutant S2 lines, we show that this result applies to heterochromatin genes generally. Our findings support a critical role for UpSET in maintaining heterochromatin, perhaps by delimiting the active chromatin environment.

 [1]: http://flybase.org/reports/FBgn0036398.html" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2.cover-source.jpg" />
    <title>upSET, the Drosophila homologue of SET3, Is Required for Viability and the Proper Balance of Active and Repressive Chromatin Marks | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"LHuo4ne-NFVXj4GY30jZI7sNYykOdFZ2nYjErIIxZuk","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460568","apath":"\/ggg\/7\/2\/625.atom","pisa":"ggg;7\/2\/625","ac":{"\/ggg\/7\/2\/625.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/2\/625.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/2\/625"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/2\/625"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/2\/625":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022HVVw+yQ2YPm+vg1ej8f8StUTQHeK9Svr3TfdYiJ4tgOHZTZBRgQnRHiTyRswJXMg6VtLhrspw3hXv8NazWc95kvrsWtDGfGJccke4MoDIu+TaAhV48wvs7dhnAq0ss0O2cKj8vwjklDOveJ0W+eGHMWXqozq\\\\\\\/YjD+f1ujPLIdlFgjmOvD5naxLh7XsJKvsXEi6jz3r3bwNbIRCKHwn+a7jkuMADSWw+gSt0aMY\\\\\\\/OMvzwhJCWuDn0JBsn8+utONfui+5oQ6ZjXb7yW4mikVGy2u4XCDO\\\\\\\/28bzmCTUIKh83bLW4GHFsKu9tn0xN7ao\\\\\\\/XzHCa32pKkMX5oXqQ\\\\\\\/oWE8CQeSIZOkMBeku8oeUPT0y7YkoRdy2iRzE5T4hkpuQ0xr+A81zHuedAZoTrf9bk6xctrIqSGJdd+GAsB6ljaqmS2JwDRWUucCEWlmk\\\\\\\/DkOGZhbVsELBpdssOMBSwL89\\\\\\\/x3JH36k02EFy+VCvr4jfiqO6XzEqahtXdUJEaXC3w+fHbrzclEgdlun8BGoLzYNV5XCJH6Yf2yXD2vNLqQp9toZ9OmQ7h0HtJ7tg1nwql7gxKBeTwWrDHJw2Ep+xVTYCRqo1nEXyISM5CxpRmCSGN36F3C47hv+rboXjJ0VCj4UnjAz3hGxm5WyI5sOUOAXaLTGQhuCCCH1VLXEZ7HCBQPT9+JxjefEXTyEHLEmaVNdRFOyeljIWXv6hk6JHMi1qgjjRdGw9Ol2KV1JGxVtwRmo6LTsKUEU2F\\\\\\\/U1wo+JGoUqs0j0D\\\\\\\/y9T5tpEcYqfCmI96y8eVTmnqnbWAcP1AWJexDeTmVZsSe4HK7jAh2onQb32YIILjx2jwfyPxST+AZoS2F2W2ycfltxkAie7LAkYgBM3pmLkrypmnxQC\\\\\\\/XRJ0NFqXhhmuKGyabyR+3wofNOIPYZ8KnS2jQEkjVcL2oRNYhZSc46biF7pv5hYyModfYAR+V4Tz47TXSO6MfI62lZVrMH1rW0mOFVfi6vZWJLKwLmQEX41WGc5kQcIk83ZGc+jxN1azv23Z3c4wtzEjyazZa34myRp3EIWGg\\\\\\\/mh3HyDk3HAobC3ikle3krEgTSC+UNRXUoEVvljaRYa08KJUFBk6rCBW8tRX8kVRqx04FWsdxxdXfkMhhV2TTymj2\\\\\\\/urVYRsE3wCsqCp2n51yWZ+T6y0fVPAiZ+MbUepUJJ9rVKxwOA5foF+nL7C8ahLtgMDFk+02h1oAvB86srJqB1qHohQWui4e91oAX6LEVffB3Dm9A8viowHoYpZu77WNqWNLxX\\\u0022,\\\u0022iv\\\u0022:\\\u0022oi2hE3DBR8eyaK2OfJWG0Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227e10db1e3b8d5ae712f9f8581fb921a865209e38b69f44d7b4efbee5a46b4b17\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022yRx8PwG496md9gBR4sj4FEXX1KS1lRMU4G8q5\\\\\\\/m\\\\\\\/GRDrB5JCtDsk7O61EGJ7BVtjLT5d0a6L+T5CNmGbbR1oOfpZv9Pd41vkvhqnMF2+nReMsPWGsVAEPpBiaHBkr+6decyMowxE9vfR1eDTEifPoN5goqgFRJWzw\\\\\\\/7Hj6r76UkvzXDgRJ2VxwuTwYRYM8aEm5+Qj09FsOKRspIbf7mZKcezmrPdnHRwri8rYfZ1AKpzw80ZsknsD1upBEJmqOub\\\\\\\/Cgnw+7+3BzGKrpJf+t38Yb6pUx09JnpqDqrzHsO8G7EsmcXpZawsARxTDeC5FZTZipYEez9jrUy1yKiODPPGLBasESb+fgqfOqDyjOdGC1i0tcQRavYrf9D10AMlGYyMft8aPPS\\\\\\\/vdb60Z3Lk+9fItXwxXMwIeR5xznEp10atDMCnzRR3rjYMHA8uGxyxVzlZrG3IVNHieCOAdSaOA+mPR5qJvJnQP6MEyYE1nETxCOjf6o+bSdoZhStwB+CxhNx4pXEKI3Qxnm6F1qxeDf6YPYgUdF\\\\\\\/1gLGgYno9HcNYwMTY4DFevylQSGpzJo1AnBtoyd30hFXyu7Pq1ZlwWudQOHgnH\\\\\\\/MrJsWs0xfUcHJSWlWZh81XrQm4tvgi5\\\\\\\/hmqCDCuOnMsHBfRVKA55A5CmK6FdwMO3GgrV7ddCFnwITHMGy75DMyH52LH92DKFQXe9d60MINT92RvMcI5o6ZE\\\\\\\/l8OQL\\\\\\\/k84gVpRtW9aEywTQ1YZNQ0nj42Ag0WjvOgYbmhS9pOWMsKJQV0cMIkjsFTjXyCdLgVuFvuiQVf0ODqP5rt30juphwt1gYCRC16aPPnywcSckJCt5Yk+u28M+\\\\\\\/3UQ6D0Ee+grd0AVThQaCRFSiJgWWRTEO9CI4qK6kvuoq13QGqS4SSsXQmk8eMqpCZ2LPiUZxMI138iALul2iNBrLijLeAiLaCMxdxRwXyUWxEyD8qD27LqjyFdMeVePHLRQjBjyPZ9ylDELdx3+jC20SXqNI\\\\\\\/bAIf21\\\\\\\/NNaX7t0rIrLL46kXnVfKQWHjeShPuxOhFtTpkrNK9SpeeXt7TUWjft6dEy6zT7A0VIp8\\\\\\\/yCj8es2GosLGNwSW7Gs3jdupfVWQx18iWutB6xV0evkDdNBjgsd5+b16MfHTUJeTN\\\\\\\/j4TQkc0wW6\\\\\\\/wMWameJsOxCdyNTydxvQStGruNk7OoK0Xg+HXLqrbEjw1k38A+k3SnbsPQ23EAXLN+6nzSruir17E9YA1J5MSwdDvWx8xfd9fqJ+o1HYXyVknPXOq3o6LED4bOOHTiYC0oU1xVo+CF5YEmEimvQxwUNmzCoHS2Q3+Zlsd0WDNM\\\\\\\/wbReR7Kd9A+MKTd8am8MVDlpgmptF9LRH\\\\\\\/D0FbOczrdKtOIe\\\\\\\/EgaGueOjHDZI2U355u2d2EST05h1wh\\\\\\\/gchqxaSIoAtb74pwKkTyuC62CnQyauYUfXNgxockAJDYOrsgRilEi9YeusdwhgFK6leDRQtS1jnOuDR7jl5uhoWTRky3Jb0tlcPFC2OPHYtag3kYXjCrJ2FmJdas+DvYz2V9tkfJ2tBMPo8ght0oaAMBvMNQxkNkkWcqCbAtUMPWE3xBIN6CHch915ckx\\\\\\\/1GeqOkaySKWMvUsf6CfVkyczlmWQ3bne5l3Su4E\\\\\\\/ngkd60absn\\\\\\\/ROxeSm36\\\\\\\/K3W\\\\\\\/Ep6mr2\\\\\\\/b2eUBJBhB3Zp28zXjYCfSyXWhhyemylnHTUU3lHFYL6fnPZgXo0j7wNJU4ZXC3fykgKS8KTfixD\\\\\\\/4JNKZ+mHy342LD23u9Kc2yKQxdvbpZCAvOYkLv9AmyLR4Bv8NR5jkj5Ct7vQpYONmle58CyEe03+FQkldNSpPevaVUluOw5ByOihHqicWdwjw5cSHb3Le4qcSR33LlxTFzssTfN24arVW7IJrKAhMFPM2oszhMV\\\\\\\/eZkcRjh1xvIG0IVV3wBAu6QKLvk9X0qZrZeCdi02MFix\\\\\\\/tOa63o0dR6rHJnzwiJScHzC02B8rM7JlPpRQLTo3JnB1A\\\\\\\/GVBOhY6Agn8MuwKDymIwNIkOqBR9cGvWPTKb5AA4UGSOdY33J3GfIY2qQSrvffjXemqVifa9oOgAcXFLxvmjxH8V+QbQdB6MREMxKleVD9QFdjF9y+8qtneNVtDMhzPxLLIKcDIaAaj8j+Cs4gpAE8J2r4DNWM6RBMJy\\\\\\\/rb60VGQu21vVhChUiMgr+r7c\\\\\\\/ANGhhC6WhJpOybXupVSRdmj6BvUdccMzgBrrq5soGKuSWjrRKvCvlbDoLps4+yctpRrRSpjiM74vJbZ6fD10eBnsmLr9lk3kkTB25FzUYJRjA5u0ofMLenuprNjw1EWEsEEtcjHa\\\\\\\/q\\\\\\\/b8LKdzlA5DjtEaEU1Myf\\\\\\\/To2vpSS9SwzGUocq1veJUpcCmF5+cYTlwEUMSisRyizcsNucEQRfnWTHX72KQi1lJL\\\\\\\/D9Ge6hm95VRYDka7aBDsVZMyqhOJBnG+ez80lbGKSJUSh50iGmNSj7jZ3NKVAwYgiNgiq4vUqubkFDAJb9syv5mZwVol\\\\\\\/kODOZ2+U4KnIe1Cgt323TzYRKvQo5U+Xqvxg358mZNLsPqJFU4gcjYraf9QfbNhXM3MWabD1UOsYV9DvEXDwE1fldHu9fIIkSljcTUiiMDXQayK0M\\\\\\\/J2w4hC9\\\\\\\/xcZK9tLU40k3wA3g3azNU7H+qEPAfDDJOHkDT+Tr4wWmzHRBuMH\\\\\\\/G90JIvQiZAchZnuGRTzgC29H7OklWGdYkv7Cwqp7ReB13Ii9RCB0BsNojho8uBxQ5WypkMcb31mqHYFvkvHmR8Q5ruegWk9ozl4CLrczYF+T01dxIPf\\\\\\\/\\\\\\\/ttBKbvfERdXV76KZ+1TUOFR5dsRfDD8BpNr5Zac0iNIIMrHcF1mgepjXdMPizxnvvTxxA05IgCQ2BWMiupOPuWC5v9raIuYJZF\\\\\\\/Zk6iENti5UN4fP5hitLuUKKKPQoH9hTyWYpxwWLMO6nHWLD4+pD8pz4CPujwYkmFU+pmwt4011M6JrzVhDhRlT7emTtau+38lc82BYXRwZb2nd+uK2lzk0W4tfLSuYMrm54WfzAr2OCvMZ1\\\\\\\/pxXakVOaN4MvHBrCPm35Q2AissPYiTdQp5nUyImykDrDibUVHD9gbyfyKqq7ANRgzKqBMTwlLojXFKulP6IDrigPzw8ozY4AfwmSiu63iu1DhDYv3qfCH\\\\\\\/VydG42hiad7\\\\\\\/f6X37fn6BSeOgl8e8SLsNKUjbbfSU31E3di+Piv4DTj9CC3WCLx1neTSW2blBLWHtkGQnOXLmKDgVEJk8r3wSU3tDb90SfH5VzUhZnuIhne8JHMv4jsNTZgMJx3DtWiXBuOvRGAVyfV1DYkgQ33f6HR9T4h80wSkV\\\\\\\/SrqYup5nr7119rXvNz7qM4w8yAEbE9cJfe3QSR819v\\\\\\\/C06mkhDsO7qFR9NZRuiIMe9nrLLbo\\\\\\\/iBiY8Grs95TMlmRgcvTnhqDzYucUfVVwuGc32VY\\\\\\\/YKHH80wP2pjduednbhf7TP0SWfdUNIz3fhrr0u+GhOkPu1iJN5n7AB9OMjUMQizoA6r33Ps9M5yl91K4BLDmOYKui0fmJi2BNsBsW7NIjS+VPStaCNjZ4BkqgvqAH7r0RJHpaAh1IuZkJZhMVzWmH3euUpC5ZObWF5vV8M118QpG6Al9Wd6X9VxqVbQpHAUSzQRjTog5sdGjj2267btTs6\\\\\\\/pEE7eBqNyUPTWp3OYUpuiNN5SK4lY8MGprywEKa7trZBoQdmhrEpZqBiaXXjxQ4VPXcgVvXHDUf7SXfxEWGl0RJe\\\\\\\/1agC+M4ggahsTyQ3IDXn4yja+94PI4i7UghvHSWfbXkwskZ0+2X7pMZo3UDRyb7CR9r\\\\\\\/ev62Ew3S4yuwzbpdcfUJ5LBd9VSzBiRiMPD\\\\\\\/mNfaBjmZYdNfg8ZYNZQV4ZNHn5ckGZL34wJ\\\\\\\/bvjWeN5MwqPOGKASaLUYX+5qCIJdxqLPqLSRQ5KNq64ATGwEmXLJiJN4Dk8Kwvxwj34DQ9vCeX+D8zvGopBURhDZcQ8pFp7tzdppL8gafKJ690nNUp4AF5ME5J4aQEjktk+nPUwhvXDNKFt+buLOqCWYpYzkgo4JeHE6Mba\\\\\\\/mIpOW+otMmcSgOQw7pKmyKeIE4J5fD51XMgnkhBOUfynKSXwXBrDAd5PoS+lxtgsuQTMTcJlu18YQ54yQs5VmMQ94UMA2rYuAP0lvcirt96odcNUaFDGUeANq7TUCNGdf9DpEXfoQvMEGdx6yjHQqZJAsfRMNATHfuLbMXqEjSWQC\\\\\\\/nKjqHUwc9rsXiU+0sENP92aFcAz1b0c5xydkNmmRxQgjuETqLQqHjn3q3pnv9BLDWy\\\\\\\/qzYQKCI53hL+Sx2FdFKYk0WDC08EGaAkPF6QtXqlKGzEgI1gjf9kPT5z6RCsRL4tr\\\\\\\/zk4N\\\\\\\/P7bWkkg0ZBia4FthTwv6tEds1zjrAwpq44qpg6wDTRu0mWOe45GCfYL+122LfTKj7vHv7HSfvFBN0+bHwpDWS7KYz7nf2GwKkdCx2UHpEWBzoEKlnZk9p6M2eWtsZL9JrrhAR1wTtI3DmSq3AHQ8gmvW+PruG2O5gng8AFvErs1Dr9HdvQyDCsT3kRLocH\\\\\\\/mtiRtm+D2qbZ7YSL4HB4yqBp0PJXbD9Wyt0fY\\\\\\\/Lqb7t9JjQkYRpHBupWBkWTIxrizxEpgqQjcUaN6+DhGu2NwHl+R0yZx6o\\\\\\\/e83s9ql3pnM4gFHs9cmd\\\\\\\/o+lG4NNfrV3xGsfhmHd5FJTgXnj8YBtEGqC7tNFZiLiuWlPYfYsOAvJgvQwDajk+GoEiMMgKGr3M26TSHbxxmtQHxxvaeO+NMlhae8UaVGZcxmPzuoQ8CIXH11SYvjHJhQ\\\\\\\/uddQ4Kgr3L95ycm6AIGSS0xlUbYFoQkl5ckSAfBDFdMnY9iNaEIp5QGHWvTuuyD8MmFOnssSHMjCbeN3tpfWo3LFc73w7VsWCmQYcjiV5BtFjs8OviqdqLhGQ805joNYY+lrmHc6u7EP1c8KgXcQBo5Kn1ywT+GrsZVtQML66f+zuaDEjeg\\\\\\\/XkyPA3mv6WwORNMq81L\\\\\\\/m2e5xligXsUa06D766k42XHbxnpoVqFS\\\\\\\/XNzI9fitaDgM3ljb1Yq\\\\\\\/Db0Vzpa2VnquL1r4xc7a3tnIPxaV5uNtcl09474gz3gWydzYzCfJCUtJ+AtVNWzoUewcovdAaVrbHDor20n1NY5oHW+KY+Ictheh2aaWpUqdF2SZ4mzKofHNIbdXDbOopAby23FGEWPFu5JWCykNRi\\\\\\\/xjpkDI6oqwLEweeiU7s54qGWi3XOk6gsrWxlXN+niqCxJMaU8ToU5VdeC\\\\\\\/VZEht4AphPYS5imZPSZHO11RFvKgdimgLTrvzKGTWauAlwqB1mMWy6GOY09k\\\\\\\/HqObrWA0R4nFV8aRK0V\\\\\\\/fnpPW+B2nDeDi7iVjixtC+DCwAgG3jN\\\\\\\/IQZ\\\\\\\/S60uQ17KiQoJyM6rnH6piLR2Tu3KN8uTtK3uS3ZYi26ffbf5kQ3r8p08D\\\\\\\/\\\\\\\/XGhyNie5DdmZGo\\\\\\\/JIlcMu0pnHeoxs6N7OHvXIoOcJ0BhsKtJ76lH2J5UdfI8jakC2JEpSOSrOxDIrS\\\\\\\/crG4QqTHzdO95k1xSXFN5PdYa2v91JQg\\\\\\\/tl6XSl\\\\\\\/PYwh1k9EuimEFBfjEN8TsIlY9NO41qYBbaexAL4VfH5ZrMHRSEvsxAF3O4K5mZhn8Om\\\\\\\/Sux3IjT5n1GzLsKNbO7ff6f14rdvB23FZVJBLroM+onzkmAZ4XYg\\\\\\\/BWOZYzwAUq0oQn5YCcS6rBdP1KFNybm9JnuxSFjuZGFqqo1hrQBRacMIbshp\\\\\\\/UqpJO1e79nVftuQk4z95GgDCL9bFc2KdI0eH0Vf+RRlEwLB01uEqC2rsuEBvMiEl1YD8n2ewKS5F4tbxqIGpiyPvr\\\\\\\/\\\\\\\/Oy1uE\\\\\\\/Uoq1cqybW2m8mtxO3HLmD6GxX9nH4fsIV3PQXUCyL\\\\\\\/UGIY3+i3gZfjiXzsWqbQuaNWUb6xwIU8VjgAHqkhUiOnDl\\\\\\\/w6UVw0J1rd0nfCQQqo9NjnZVFjvcz40nb0yzeOpGgOofcbELPgFWJd4QT93N6f1TL07ylr24U2FcOVlYXGChJndnIFzaw\\\\\\\/7nT8GpKwcJ6jpq7Tn+WkZxt2nuU60l5f601\\\\\\\/YRPDMZu9taafvEeADiIMYts5tfKNog4rwo7AbkkMOzo6Jycua2S+hF\\\\\\\/LA2FdzmqecgOxozQ6nzZsBgtz\\\\\\\/gYQxmFr+Mh27qGDMOpqo6A9A4MtNAgiV9QRl\\\\\\\/Z0iHCcC9yjUhpXScN8BXkVmJrDFOK\\\\\\\/Z6584HdeUf9yOUMPdmtIfcrnji8Usu8+dTaFmamnlng9vcHnBWJ7AP6YzvFVZoCgBXdbtfWlSsJ\\\\\\\/ZGgHUiHytZtZoD\\\\\\\/jNr1BsYk7C5Tmc78FOrsZ7yaCDYJE\\\\\\\/Sc80JoXVOwyIpEdvtE1zoJ0eSesgm8ZCFWS+pJ1hK4KYjFS8qq+50ByFfq9yh1J4\\\\\\\/4qqrThRsxKEqosGrtE18czWywfRt6LfIaOayg0NLJlVZITMWB5at8ymYRnF\\\\\\\/pvq57XDquyefLCR1hRGhu7TsWSKNqgSXsEN0qVjD0hqUIvKSOmYOduHeO8\\\\\\\/pW\\\\\\\/BsLPwp1C3efLv\\\\\\\/JY025lcNMefgu+RFIxYuIIJh7NdVY71CFxKnPsV+yoCtrCXOjxZnKdnA9W4GvbRq38jnLOTPLISIKx9ANLKXY6oEIhawongR\\\\\\\/GKycJx\\\\\\\/Fh+Z2gE+gYNCz50TwJGnwZFFSBYL6uyA05OaAzNxDpW+AvcrKkS3cN7paNOzHr1hjYHTKROQSLAvqkOKF\\\\\\\/G7sB\\\\\\\/yk9FBedu+HZcHsu4xQk+jyEoIwxDwUTT5uM\\\\\\\/w9X4+dT6Quo4YKCvH+MWBZvH1mqJ16alti01EBckapqVc3aj3lSH4SUJ+NU81VhblsOH9w3owAF1NhZp8b2lVsoDR6XTxLmNZLOqgRarAKqgpV6tS\\\\\\\/uZcie9L7o9DStZhuZpHryqfOC9bzBkaKNPy4BDRMfCmBkFAjJMxOUXCg4KiEGNXb2JKSONnfv+pg7Ezk9Db7cbTD2ppI8X8TDmCFpP3C2vPxvV2Mv865yNuwdueAoE2hc\\\\\\\/FkniEp6vOZAkwjyFBkho4\\\\\\\/Q0P\\\\\\\/wWgOOq\\\\\\\/kdQM8bk7e+FdqK4ZoWrDsuSHzxU\\\\\\\/qp3KyL8DumBCiJ0A8AG8wx\\\\\\\/aVtHz5mqoAy++5adrRmNi82W1J6X9fc2fyMoeCJVDMPGQz415HJEA4Gl76tnTfBVcSo3kRranqnKlH\\\\\\\/NymhIYbR4PpQ7jS3Vt8qI1kwj01bsq3i9dbsfv8Q6g3RdxUHLcolaFjz869ulVRHE56OaGCkNIRUwl4QAWqt5fm5NOKKS9vvcsxucKZG6qSAY8r5CX87TRMPIV7UOmuLmDMNwE8RMy+UTDeTZgUxBLQC1RKUKFm7RjNqE54vtRP8myJuiGGAK9dnLtwLTQBWlhsmcePA7lrCD1xbAEBoUe5SbKk9FmcehYHJb6t8bA4XhOEw3MEeIaYS620qCrydmq0ibsqhsyzhd7\\\\\\\/V7le4lKt+9F+YNvEaErwWUiILDFEeUcCsWZRrsFVFDUre80lA28+wGPSqM6N6+UNixBfmpljwVsWMwpyOvrJQLpcP1Ov9SOQfRtnxL0cBkHTeIi0SBAQMvb+WCDoKDxb3khRvmw\\\\\\\/JZLNviVV0avNY\\\\\\\/10yv+YGgH8f8OuJrzt9+l41c1QghGoaPvm3IjNGPVNlQgBeult4aBQXB0mPYb29BYdZ0QtPx2S3DR5d2ybxfgw8XUikxoiFvx3MRbQLvZE6GfCiCq\\\\\\\/l0iicLbs1b5MKTX\\\\\\\/9a5Su19lB5CuRyY+LPwBYqjl6YT6TMcGnZnhxBflPyKjZjJ1ByMEsRQYXE4XReo6BBFIYYsDo1Ai3tmSAqq8LGkO+7gNDsX6P7WT2PsGtoDOR3+Gu2pA8f9L\\\\\\\/3tyoB452\\\\\\\/jLchmSsAAwtZJsb\\\\\\\/l\\\\\\\/v+MB2qMofLROifj2ztozVMmkdABiY9jrUNzy5ky+Aw\\\\\\\/gRx+4PRnFA0hd9yonArpqs9NJ6p5ORVOw7IVS4tsd1fmHgZt12SoOmxyvWEID6\\\\\\\/ysJpKQQwTPISnxmLkGysLjQ\\\\\\\/NzQX6plwZ2IYb5j+9Giv2SKHvGM73ggVBDABaNP0MjPtxfEQP99zPgug6IkjNhLzL+5eVXXUAa\\\\\\\/OWO3GNUGZIBi7S9MQIOfyl48QKrj7r2i62uXSfPoBtDgd+OckOZg5SzDPR03j2GyYkpRoupQus5X+uEWZeYdJvPNAHt2ADsYYUqMCPBpey7yd5dPnpwIr1x+k\\\\\\\/O2j1cmel1wnUQj6EXnxzYGBGOUdg4nFEMB33TkF4vDj8+E7\\\\\\\/3IqxF2mr+EMgHE8UQJuurOyO5gxIUKOOsfqU7hTS0gFtb0tqxYeiKxjwqAUcM7pdakkr77A6TdCx4nCa9gR9q9h\\\\\\\/xUzPs3ToJVUH5\\\\\\\/m3jdSTdYnAPWVq0z8Bgmt3yQ8NYT9sOA0DIz3diIFv74qVqifftOVz5KYYDQ9G1L3WU9yVJKtp71C6FvmNDHOmi6L87006bBHGgos72WtT9r5mx0LcQXB9rvu94QBPM5\\\\\\\/u9\\\\\\\/8p2OXskQSql04KFG2\\\\\\\/WB6UlUkCGIqQmx1ebAQc7maPEM5cQnzoKxTP0m3C7YYr2qjOuylHEJ4Xbpk50trWWLJVwTlsswABG6VUdkWj674vczrke3uyARsF2ELkA\\\\\\\/4H1Q4KBHVXvjAhCmCC+zviHErv3DhYy+sXCH3w6GTt9FdFHXpHzQkOVjxlueyD19nMmWFohsV8hWNpfGfTyrfUyRYHh0vequC18XBY9P+JWNiZXC\\\\\\\/ztlejERe765STb9avefqI06iZPiPAqEGuPRueUtrHqamV44xFYL2dco2rKVtBifoKnWElS6qe22\\\\\\\/U6bsD\\\\\\\/fl+3iA7cl1PRVTkxKQF2HurO4+oZc9EANIRdtL3gai+6aaoJYgv+Egd10i59JMYsuvz9ZqK6G3z5iZSTWxhszegaW0EJD8px0hNzvCK+2XtY+nY1qBC58D1DScYd0dkI1We74jIuiubK20KpN1x+mYKPjnVNm1uVXjxAEztCzw6p8PBwdl3W4fTByPCfldvXIolhyH5+y3qT4DDbgefCgvBUxzMjeCW1mcnmc6143JDG3GGx+P69chliIW8K+zeBghsysJ55B5jKrI+IwBDmGumWZRE0pVgTuzJk7M9\\\\\\\/bEDnfjjRWLCMoPI\\\\\\\/8vowvCkTf6sHRnuCtEL0ktG2TiRY1lBCcuut+EurkueX0pzxlGOgG\\\\\\\/kasy8vqpQHjhCgTS50\\\\\\\/SZS6wMJZRODOtjJ3s5nKfCg3fsk9MDtFdzvqZb5+63F4QjBct\\\\\\\/UyXZnnvOY5H5MXB9PBcgG7R1ryFA0pAaHPieWpDQ6CfjZiURXB6YcJW7y9e9s0nkLT9oC2R31Mz4e0hRREN4qKpnC\\\\\\\/BwvlRHco9Gc+UEtyD6HY9bmEigbIWVorXx2ALiqEyS0bo3MpVTFFqBsex6rXW\\\\\\\/V2LrJZI078APU4PJ456GK9ahrywd\\\\\\\/si16\\\\\\\/rszvXyYmcygXNslGvKPjJhJMxxiZaGvpn2aLs9ixR2DISWLobC2212E1Hdsclt5P6beDmRyJMf79itCVPGw1+we105geFl6GzNVhutgpK8z7aJKNZdcfAeSBJUclMCwqXyoA+5WDwS5YY6pUs5OetDPWV08hiFfP\\\\\\\/L9LaDeGfz1i8nxYJRTY9OvaxF5qngY0ofvszlZ1RLnenKdQa4KPr3ZtRJorYijcjXJ6C0pnrVGQZg8+z5FOaL1KGaTOblsrv6j3AQCFrGqfz\\\\\\\/Dn6RHhPLnvzTsJfdyUbmVM0s3qYWIh1mbkB82oEUD5W5Gno+STaOKIM7P3ENilNRT2pbqWMXyhDCvnzn1c45m1MG8zbnJNGeF0Ny7Fr8h7IREiwIC5zxNMB7bYw5cPpZTap\\\\\\\/3S9MsbP8KqyHB1jZJqb91DuqKCMPKKxZNi+t4wC32jIQJ9ieWm7D+iaUc63HTFMIi+cggB1P9ZYF3Lo2iicKgP9WGFWIYxEYzu6TD0GO348wv5SPxMiHpvKchtWxeWHTz+8UBRXaqfblvF6Qr06usKdz4\\\\\\\/4Icp09kxAhH4KNnzVBa\\\\\\\/KtM2I0u9ck2nap3ATDp3nyEMGRWeuYTN4hMTg6Om2ZE7dB63zDhQYTvq6Xise+czQtBXBWUEl3lBA+t\\\\\\\/xl+5HC9e\\\\\\\/fl+KZKfc52On+93ik9SkwKi1\\\\\\\/l7fs1ter+AsbBE+AwqdbyT8ysxgzNWt6a8BJn48o4ALi8Lq5FhDKK1yhKGSNQY2H40CLgULEdcg2G0KiJHY7mEc1W1k8TQZ4WzrHh1aoDW2pCIzPsm5C8qIxZuaQG+gkvWg5MKpAPZWhQOQCSHxF8UFOiPDKexz22dFRG8izmw73k+8Up6ePs7ZL47Dv4VIq2sfAZpcQHNxIqTOc3OEhyMHB5s3cVKD4Wqcss4X4Y5eVy6+VmBhP9XxC3c4u7pBJqNpPcEoEacDhIU878oj22N4u11u5H4PDJqphghZDN696RyXj0IIDNZAviaX6fbJpOFFGRoTnApR8TsIyXhY8uoNlm1NRvDplM25vVBAPeCkg2YzVW+4FbiTKxrpUwYoXb4NFBno8OdvZk86xcEUPIHFSLIVSJX7tY2T7XYDw7JTOwohHZ8LblkT5Ks918mEoN9qepNypZqbu5N9qwEqAL4IvtQXZt2nrhNWPxIaCBMhJIzHTurakpO0HLyE0jBCBbBS\\\\\\\/NUWpXLN6fHOBxf5rU1GAOAafArNyvowDWHIajHefNsY9KXHG7h3z0x9r7gEVs2UTN9NPElp6b2OLGtMxrFRET1qNA9mrXKKFspYJbsEnctMV8GzkbVQzDQggX7vKaSy1Rpp0COhWGgZNkS6nmE8ajLIZLlWMk0bUnTDTsicP2rLJtDhLK1SIOS+nWe9GmOfooUt6pQGJtOjgpBGByUaEXoQ116J42uq6POXwDftRuaogL4vclT4cZyWbej4mcCNwUcG+xyM0bEJKstpQUnl8Ya0m\\\\\\\/5lJLZrnOWuXNGe2JvGtbdY6NaEntL4Dlq+8AHsK7GC0fKLRG4lzLLdnKC5MPyNnzo89iJFg4UgpD1NkXOjBD0eXPkPdbMOtriWVfZg5\\\\\\\/By\\\\\\\/AiyV1gR1kI\\\\\\\/bUlI6KJa0w08iFJXSCnT2pyQhuoOS9T4ggYwLJYgbzII5EedVWkWYiWsJAU5xHszpo3+KEYyiBczb5pT1jSFpJXW6Usrg0FDt5Z4gpIiUb\\\\\\\/7tsbo+XB4mBsSy8ymrr5Kx+UTR9LlznIAlcNkeTfmJC8EW5Pwjvi1yYoVLbLnze9zNCxowni7ScRgCr64uSO1r+eRhvojAox9htPXgOve4p8xWbWV9JDUPcXOdjaH69vV82XGexvsQLhxHbQu8H8ZikjSEdjRSWUCWQJ9J6pDZnHXC6ZquFfvxclypr+8BD6hLNstHYbRECg0QAIybTuekDwXpFSBQBbMLLD3K1WMqEsUw4vI6LSaqj4bFS\\\\\\\/yxyrjMVBg29PuvLIl13OcTrxifZduPzmABQ4vgcvQ9UTHDbG4IrEP96TdKOx\\\\\\\/JSxaIL\\\\\\\/Vss\\\\\\\/niDNW9Ee8juPzCyiuz6MQTW6cR77jfjdKvHspiSA1E0tehtIoTTIgHhy\\\\\\\/1bwYGM5Up4Mvm9zu0eySqXDbE5\\\\\\\/ncjFyTBgs7\\\\\\\/eCbsS1yJv\\\\\\\/H1Bdno+IF5bzVT07b9WblC7NAaTdb1QhvHGbk\\\\\\\/WDP\\\\\\\/1f7345m1gFobDotxWAR9VXK2ohFyDSHHZOSt2UELDjOHGjJwq5LY85H1jcRPbGUnIYERix1TR7nkoc7JpFgi9BdOnDWqv7cm\\\\\\\/YpEO3ESS9\\\\\\\/sEKzbLPWqnYHNnXQBjkekMlWmJmwzd\\\\\\\/fe7zcDbIhpaXEkS9h2ZCtChex1h3EoEeAmKiYc3Yq0wG2k+hqU\\\\\\\/lxpXIPSoEZeYXdi+16mgn\\\\\\\/FGLFZI8mj0+Zg4GD2GS9QTjRlJ+l\\\\\\\/otYhS2enDVcFV1P21oAFytngJNRC\\\\\\\/PUmSMVeAmzcackViZlr90SAJt7r6JsQc8h5zSU3\\\\\\\/SOO0CagkrVdSmxXOQszmb9uGWHHaloEbAGuRj7qyJekeLc+GzqRx3wSw0jynTkfUtX9STuuroiZJnLB23X9KTxBb9Tsnus8KJ5GBvcmGHDYfSVMvsa6XDyU3RHK4MVblGNXVNgKWXhzbeZSuwwKdv+kXr5hmVbAaF4eMAApfvTW1KhC53niw2QCrAkj0XBbu\\\\\\\/YyTThnYROJ4vAWot9rrMIFX9Y\\\\\\\/74aWBju9zJzWgwMvwJOVEF8D3msLCk8hfMg3jRNfNJSbJk1iKqKjCcX+yqWjErZynf4MY9CMqPReMrG9UKJcLnKnYFCbVPKrp3Dm\\\\\\\/UdwSfrmk0vCfwaZq\\\\\\\/Hz8zsDnO\\\\\\\/2pQGpedIZsFWvfRrvvU3qBJ0\\\\\\\/Hek8rdDoOPwUOD\\\\\\\/xkhadXqi09vdlvIo\\\\\\\/n\\\\\\\/NQOhxc9PNqBPzT5ow9q3l3WbFlJIJA2JWvfNXa9909frQaA6\\\\\\\/+blllUVmEI7JgrZ4g3AZyQa0qWiOuV1x592MkJM4XyQUhFigGPMOehqGNU8ivhdl2BbK1br7Wznkvut5ZtJi+uuAoiYtN9MgNma8Wdb1xzCQTa26Y2qJQlcJ18hiRIxgyKgg39xA\\\\\\\/sPkgypUCOHzhL8gkD1eezRpaaU7HccKk2QGf++IZKFoQASbqt2qEKKcLVvTTUSyRLXQXVhMC5uUJEV9HSFcFQb4jw54OQauSr3SBoyHD005PloJSYLLyUgQovvaLxRhYRPQVkISqWpWlbtP\\\\\\\/+o4Yhbrw9NhelQNWtS+DO4xKSu8lp\\\\\\\/6BzjxG3J0Ps+yGUOOviYpC+8o4RwaJ\\\\\\\/\\\\\\\/PJJOh3\\\\\\\/3rrFpU+QSbvcwB4qSSGmdtPIMRR7qARxYjb6Dn6yygayApaB1Q9cFJqXlUa94WVxTAxAGGsdZi0Oo8Cvz8rNF2PlNJbEZdIS\\\\\\\/\\\\\\\/rcxLDrIZ4eUWVaBy9rjYPrpkk97hxnO3hLL8wRGQWwG8yXDcLUeFeTWxCvLFBZq8nsTje96\\\\\\\/kqRuPJunHOkzsagL2rPSIJ4aBt8gL5jIgrzkL27MbLxe0+bEUk\\\\\\\/VnGZQVB75UAS+pn5fGXwoQANaKbmrWGYZ6vlieB2OFQwoEtCV6LDhMgQUmQSWNZtXumuZbs4+roPIzFPhQPCxXtPXg8\\\\\\\/xle7UWHyK\\\\\\\/JJpDHmqlPNlXgIE8pFFSXf+NKteg5lPqmgyG0kXbtqX8q8rzHIyUyO5FIr+06KBDjQBC4yjWZg8zxMkHjd2mAU5d7b414XCwhMS5tVWuQluKTYM95YXRs0Rqtg7DHvohnvV2VSzTn\\\\\\\/vyKWW2zEa30R3wfvXOa6FPcOFSRxJqJzrN0oTI2DV9YLlLZYWCWCnCJPoQCc8YCt0gj+YJUe1ccrSVBKtXvhFRycMGewAfK5Zyv0hCmnlvY3I9dxQmpgt05pQUALb\\\\\\\/Ue8HbNLBJMYkVRwu7KeWFzYPf3W5vz6xjAtQqSF6wXJyiGChUUFkGLMUiBA955BUjpNymOc18Vf9vDEHAvy\\\\\\\/WA3ZPG7sm40kBJxnpC5H67PJzA\\\\\\\/0nxPQb2V\\\\\\\/ohjgMWaL66yJi+PtnzwssR4Etwx+ZOoNdmSA\\\\\\\/FJh\\\\\\\/u0eeZlJU6D5JH0byhOEfKEBqXIE1x16akj7qY3OTc\\\\\\\/pet495mWdVEVO39dmGtvy4UA+2+xKdXRwGxNZLxAp2fkVEwVYCVMw25E0eOzMFjYzq9vcczo2QNTADIXnuAs0aPIKtoji5z09w9hsfNUIVyNXd3CHAhMuTAXPkCH6xOERKOR0y1rmR\\\\\\\/1c1Fd8CRuSqp0\\\\\\\/aokYiKjiDnXd6wgWkpyTkCcGGByGO\\\\\\\/wud8Cp1RB+O2iy5onYJs9zOG+jSLH5aVps7TeQBoDWz4iYNhSgkInVBp4rm4L0VdbLit6+IeObg+55hlNtXxrGD2zdyVAVAbOPl5tc6eVqvJn5YuKt\\\\\\\/CSEaDI+tl6pxkMhEDQF5BushHjcf+HV1z4jBIRhm5Dte6mkxvFxX6OAOLUYBiBZsEeZ\\\\\\\/EUajRtO5U1TLiaqZi6lZAfXFkIXtJtMjAbvkgGmgtqJemWvL3+YgP73Oz\\\\\\\/1QD6anhMP7cFAPcpbrX1dFIKIzLefyzd4Jz3Mm7Rt5pHplRn4J+jZp28UH\\\\\\\/pmAm3kpFRUrQ52gzbNx55TwmcMP+jaVXi7osogRmaK0AXP1v669LA7d4UPdqLpposMkORHH7P1a0AZzibHTrKMDocAhJlEHCzIapHaHiHNvqlM1C9sq67H4SEyyWlk\\\\\\\/boDOkZLqko2t15z+6W4zpx0i5ADTH78HnuH4uIY+scQInok0qzq9AvulKyGC\\\\\\\/IuUnEajgnWzx4Z2Ylv+IACH07QDpstAkgX9OiB7NbMRvmfWDgkMocbVrxa7zhoAcS0+Gkohg5V0k8PPV7N1NWz7ZuFBXRD2WnWg8HAogF2l6UNegiEAEjJqusXENlSmkR0cyaCOx5ulxhUnFkqM9d74EfooGgpFpmjw0uYaWOKSuPiy73cuTPBZLSqDia13t5SJJ64yyhCNt7g0DKbs+4XZQgbr3g0cmOJbUcSjqoQGyOFJnWV\\\\\\\/jvEvVSW3twG0TKTGoLU6tBGU3QBK+Kqj122w9fVwUaWqdehgvH5wm+EyHEiRxDFUd\\\\\\\/N4TFy5kQ1+WuqqC0tVRPCqMsZSLWd68BWCT2gJpW2QKnnSMB2io+7OGr+TjmzTDf\\\\\\\/1XtTd0z34HTZjd+eGHPqpzGkNFzz2iciyZ6mcgL9JeioDLqb9Qx3jWzgx4Xk\\\\\\\/NQn08TlcmQz6mbknV6ngJZhoEdDONdAlsOMaRyf4DEm4KIKo9oYEPnHJEd2SHaeU9LRr1PbhXxeBsV4dVMADw91kdefZealJdws+U9m84mHDVt9mU1aUcIb2x98m+iYEup8fstkTij5\\\\\\\/ID6IVSSeDWyvhCT7Bp5oSpmQq6F3iArpRUSqmj\\\\\\\/EkyifhVyMuL37SRO5tuvq\\\\\\\/LcFjWDl7n7WyiobPotrJzs4NHDXVbQaPihcCQbTnVyLGJqaezX1qmSaP+6F4GCZageYfwtwQIqvD+tnJcmeFTVYyflqqK6qPPgZgOrn61REQd+zH6c5MMGTxUZFbqMOW2B1r795Ij0979ZiggBX3HXsABlYf\\\\\\\/uYp0\\\\\\\/+9aqurYFhdVIrt\\\\\\\/SW3qucnUMMax37cTOwU6LC8Sb6+JBM8i24m6PzCGNhGwmrCH9aGdWm60WI54DsHbZEUTRTUCD+kq2vlgzRn+VweS8HXzMoDBxfwpGJH7v6xfo99wZzuy5u+Dh5l4kQjRlkhjRXXSw+ZWU6+SFurKb8uZY7G4JMa9f5ptGfB+gdequBBTCBnjttzP0LkfQcqjQ9O27X9Um44sF+p488oDuJQnk7hntWxFiEs6plENB7PgqmBrx4gKEnwpCBSgHlDcHgsX0TzZdQCZMF5iThC+k3\\\\\\\/UnobOH526ZxdaAJbObGSu4IsaigKib9lam3FyXr4RkYdTxxo5s\\\\\\\/oLY2JLsl9vx9E7PhgfuvL3h3M14NUopYnvsX2xI\\\\\\\/hZ+ZuPYgKoa6E+lW+8WZdw6GxjWw26Zo6FfjZinb\\\\\\\/6VXBV0X2M94uwqsFLSp6ZnKV3UDYMQV9vXYyAjwBiSC4uRlOrpZpoTzZk9yKU2TH9GgNLn3iGXH9Olm+kKQNbLTkQ\\\\\\\/+rqa1y3\\\\\\\/z9GKNR2SfDAUdASZPHDQN0zSvhGRmWijnLEsQ5TtHj+ZcUoittCMVpx+BcXJBfhc4srxsOUh+EcVYC+AlaWUbeZ\\\\\\\/SgY3d3hfryIT++8I0YFpqHGH2RbaosB92nRrfdQJunLTnDtmd\\\\\\\/EpAnJ7iuipP3Jl3mDnE7cHR2xjCo+QfPVeOLeBB0SSdWAs814QUDwVrRGngSxMxDyNjMFA8ZSj+BWPBG37jfVo4frYVTeJjzQ4AV8JkQOkURAMNrRXY6L1bZ66kTsyEY5H2l9LbEzvTyLtD8wZHyTrF+cvXK0LcKauP0MPE3sYaiCM3tA7Uvsw987XCWVZ6+qBGMInXa4ybkzfUVLGFV6d0Q+dX8jGhKdhsD8AaNAG1XfGCC0NOsw8p3zfrCZ9Sq7TD8EE4UQgYs0UNVIFEejmWD77nQaC8ZmUOBHSEhbQxDKuz7+SGxUfmCWcYd072bEKK8jJBJeaYS9Up15zV\\\\\\\/0pA2Lro0\\\\\\\/qVwkIirtwPiIqk5n\\\\\\\/0RvjpnlwJf\\\\\\\/Xed0SGqRpon9v3J9OJ3ZjbNzPhBWqPCDvww+\\\\\\\/1MSYiyaY71V1U9ohaAXdjL6M4pMcJq3ZUstAYZemHId+xOtnfXyyb1NlmanoO94r0CjkPBfIIVQjBPO7j\\\\\\\/KGMNH6K1EPQQTaZTLVRH3VCPwtCi95FDM9MSZ\\\\\\\/kj5J5JBRTuTuU+6T72aShNCIYPHGb3tm1hxlw9XTm2H7XBlnSV7ymfp32rLByqj0iwn3rc62\\\\\\\/1k+ey6Va3AicKpufwOkmozGwv4T4WzfwJMoueDsHjcxRweU6lRSNZ5S+qQ45rvUCIHgcSKbqtHMYljoYfEtr5X8huXfCrTOYGcUuVOxsYB10PF\\\\\\\/+AzX4\\\\\\\/9mE3xQK\\\\\\\/ybOWs2UxjREFoM1SsPdeair1wYc26EcGRFm+gsWEELRm9UOo+15+vznCseafImanFSFITiOCFvLmbMhqhU0MZOIR9JF1rJE4xSX8EyDstFpJZ\\\\\\\/GI8oZMt7VfLVMAlEgQsWkZekxKM0nXQ4gzE+UnsnkDnBhmKkpV+Bgdsi9hHHkL+ruhzSxRi13LmRYUXNOD8bdVdEAtLSfv\\\\\\\/IEWvXyKfOfdJKPASsPX7noNojLURq+TdMmVbI4gkpNN1hgLCmb0DJstHLisE0IrNFQO0UwpIcsftehMxcaWpzPWRX\\\\\\\/lPcv8ZaF9cR32l13FOfVaKaZ9Y+bzdBzRcFpMZIAsUTaL+h+eWbrIJ\\\\\\\/aCTXzOicXye+9V1gE\\\\\\\/PjJklEK03MzhLLuGzRPGN1FfpTt1hzHGCgX6K5mp6s\\\\\\\/2GT0y1ROrIq+Gmr+r+bY6dQkug5YI8E30FhxddO64tkde5K+ASulFx4AJX2KNkfn7YYgUbTSLbTVZEesrS+5wRHiV7QisWqb9O2+ct65oIwe42qiH\\\\\\\/qCveGFInu2SAKzxOsktgXpat3yRaD53ZU2INM0iLcX42JzQGR7QiCE\\\\\\\/5fHOtZIWqcC8NCk66oxd4RPF+Y9NaX38FiS5DZRU74Trd9GEYf5WU5XzhQ2hL1qQVAlNU5UgwBUC4eiia8nJvKtSp7WIOzE4jEGBb+p7JTc9t5bWRtHqemarZpHh5SKwpc9\\\\\\\/Xyz31V9FOoU6FH+EbIc9gqQZ1wjkQ+wknJxg9zCUoGZm+OhuiEy3NZIoFFotrIrfab5xQgfaMeqBDiraS9HXQ9SdClb3Cop3+n49QrWXhc22OEEoF2dDSAYl\\\\\\\/91xu6Cbk1aUswXzycGLfODnUFTr4vtUEV0TeFK0A07gUusQMT4cRJBX7OUwud1JaEgVGraSMfrXUo21NNYtBBtfRv5Fgd6rjbsxCgbVKZCsCsoENwGa3m9fOLuiPFDf+4H6yE4n3qnz1+aYiQJSDTe++eU2zZ08QEirrFA5ek35tAO34n4\\\\\\\/PPPkUagS0oSiwatw89nQrlxGLgz9VXAzgcSTmKiknNhRcWCZfOcD5Z8envRuKP1g9exZLx3NSy78fx+G82s6148IdSox\\\\\\\/gR\\\\\\\/26B6vYZxRL6WHtGfmnji5\\\\\\\/UVN0n8gEfBvvKPjvD3krmUll0nqi5VAey3cFWyiI4ZRTq3MC6MgiowD2VE7FfpeT9JbiVhcjDmXgYuFBp6TttwL63nORbs428EILumVrlRFyx7RmOo1qJPE9xqM3OQaptxe0NN6KQKGlweeOVpiIfUhSWL0Pq5LYTF0hdc3dyZjD3wWJBEjZUbYK1J3820cJYaTQaOvMIq97xzQLVCKw\\\\\\\/ACSRjjUmSh4QNBuQzMV+CKtGjeyS3SV3s9JsvpKfYngrxYyEdFc5zPkadYinD8+5OoNkOJGl5KmMFj0xd5hMzF2uTX5C3V8cGBinTXlnhRYXSLqroptUyN3z2ycR5J\\\\\\\/PgfyGKQ\\\\\\\/Y4giqkZP2VCdp9q6j\\\\\\\/FDbG+OxzRBGkqBd5IRMg3BWobCF9h0ax2U\\\\\\\/gUQ02UByb6\\\\\\\/kU1CznAsXZNskFJzjBVjn89Ft3HAQFhULTtaQdBIQEHv80hBWJiUkeLRpjwn\\\\\\\/M+N3dlGMo13Az+b1dxcW9sANwBaOQIAZ84eSbfD6g\\\\\\\/eRh8IaHadktKKsQZH9VFJ\\\\\\\/hiDBQJlLlHZ110FEwbGbNZSrj3MWi59wBNBUj2vUiPMkUv0tQ\\\\\\\/fxZuLtaiJJVi+HK1y7YOPPeewYUBnwG6H9Dk6\\\\\\\/QlOUv80ndlzYKwAd9nR69p3jlWW\\\\\\\/UDxuwtuQo2D8Y2b1NmrTOVBWNBgZEKhKKb91I74fVmFYEkyWsRzu4zFg6yERNVrwH0SiwmKJsBGTHtHT0Entvv08boW3zkcPFnJTLObqRfx18y\\\\\\\/nyHYw67GRpIDpv28p6oeMJzF89ZiD+RUqZzn8Pt\\\\\\\/1Dbx0MDTHzfuySKe0UvZOVceQkvIHHnVobbuEjyCzaTV8n+ckolU9rsAzhS0TK84IL1+PlPL+NcwATv0V83B+k9udBUw4Q1MlRJ2CuimTR\\\\\\\/sGTxjnQvB8ZIdxxbY+Gp17J479EmDYdC6MxIBvBOoMJj94St1fv8kPIGF1RomsDGBbog0ayLVA6pvWMj6er6wLMIma192DppiI\\\\\\\/KenNtkh+1i1T71tj\\\\\\\/Otg28rJ\\\\\\\/qmnUfQXiPk8LCx5\\\\\\\/nsK\\\\\\\/BJI0gOON8Vlh7\\\\\\\/L\\\\\\\/btkdROx1OTOAHKDC55Y\\\\\\\/FgE\\\\\\\/DoM\\\\\\\/pMQxvQSRUhWBzr9abrml3PmbT9dyK0sgLmJd8NpLn9pohEBGPZBYS\\\\\\\/aqXu9cdxWjFAS2WJ1gPHsUiXcHiOBoUDwPx97Ge+7duRLOGx6fWnDtesZ4ZO\\\\\\\/XmpvzKSEMvZjiO6PFpizjGjtrTksS\\\\\\\/oolRWQSEiXdH3UwJ7u\\\\\\\/4X1fW6No19Xp3bEZBoFDHLNq\\\\\\\/KNrID1DFhGzhuOS0a1wGpSTT2p8OUXRVv3WZCny+hQx4P+p5j1Tv2ZYdxDU0aC+JrQKc4NsBJwPiCvNs4f4eO6rmDhE5lr3zgB5Pgg+bvb15huHk6Gsroih3onRWSNsKhSjUtfY4R4zS2Cssl9q8lqVO3z91z91pijF7gAjNv\\\\\\\/TbOblR\\\\\\\/ogkPc3PeYYKu38C6xSC4Hrk0kU8T9b0\\\\\\\/eOABdAU7adQHQ0JRfR3rvvyP+ii4zvMOjCECcrRoiR891iK5L6lKOdPmFbBxnf59edMb7GtwQ+Q\\\\\\\/gHn+2MDBW+DxK53EVI+kyAGLEJpYL8ODIPcJxwaoJq0pninmOCmUwkOdomf120wqeWZqZ180KsmTaKCJIlD7jBH+Y2QOMfImjiTxZRPsklc4iWqi8Efhd54XHDFw6KfSg9OqhLA5skpP7MoT7nfdzqA170WRhXGPHtl8Qd+UHwRlPHWDcE\\\\\\\/a+vT+964raDHtuxEfUPUFJDsbXGnlBaIODBnp0FLQFvtrKSfD5n4wOMdyYxYCf+wUsaIgn7tIeB603rpeDqQjJrtgWH9vNrkLNoNqjUnORFMhUwrM9o2GTWQFtrRxfmGY49qzg3rktSoAJTj9prQIn9MRMEwtcINgp1LXyYv+0MWQSpldcwqD2mhVibIhBozvC0vwW\\\\\\\/QaRAik0GQXf6fNxN1MEjHCFZGHCHGI+6xENMRzFAJKubJLN0tILnEKzKZiRJ4D+rnKq9yW9rDgBJB3suT5T6yJfFtpDWPQS8BA9weYg6x0OmBecsR7QAbP3kWIEIJg4LO9RMsOjwLue0W6X2bDU\\\\\\\/933nqOer08nmFLmiZ86ZQEzYVVcmXn85zHJPRsF57IxDWh6LBQO1hLg9ULqH6hcky+I+SXQP\\\\\\\/UrIqHyPWHMZq2vhy7\\\\\\\/DX6q\\\\\\\/gXlN9BLYM27PNP49mfhRPRJSxjnvtfbPmo4x2K9gm5tdFZDj0Pzpsn0bn1J8\\\\\\\/IVjPsuXbrver20zHxf32IVra2Tp9tqWwo0rD1cne599Ax+c7WjVe4W8+dJX8jotMJjVI8maJUIZRd+qtMjpSGXwUDGuHII5UaL\\\\\\\/yN140xIpjITCmzO6JoX8qcUFMq8NAgpAotMo3xxuiOTVQs\\\\\\\/XvKnw4pTnIlssAO+pqBVLsKW44RlEqY1c94BaFYNxhWH7V8uCz96qOyGtZY7JCGl9DeYXubUObpA6ngV8slAyflWmBnf4E2tDP5UMDscz55MjoGISmwgrV0kZ\\\\\\\/hFpqL32iB8NWCCv2UAYloIB2XhAoIzLXuZ164jDyob3PF\\\\\\\/nSyMWV38CjznFoegp6iUeBQYcW64iGs\\\\\\\/gI+ggGyYrnDK0Ozft1iYBa6pmKo8igj1Vz1czCQRzaONXuYHR82bZ3\\\\\\\/jQ\\\\\\\/lA5duO1bnKYEeHacBWLBOoTgc3f9w1\\\\\\\/NNkZPv2fElO5TnxuT6EhLulPdYXUNWV7Q1XcmTHpeNYNrIvWm9icM1Vhh3geI+Jg+S4636+yNjl9ZsfoykTN8jSGCjLzyaopHv1BiOPttE2+zVMy7YVfW1XD\\\\\\\/4I3BC6PKAvs\\\\\\\/TjeeL+vzgdhGxwPjSeJyJDHHcH95+aJn4Z3jTniWJtekkL3IN4IXqpkvmEwQb0NOY70lKQKD7720GI7mOi6bkayTUkvWrc8ofFxUybAawdgkXetltEBZA8Awe+brOCITjy4CX24bO7fyOVt6AIFGq8wYI16MFqerFN\\\\\\\/bnT7VwkovH64SEudullAI11QHBwDLHP4THnENsx6J3mDwqmW1z6dOQUMKP+JMOX6HlTBMeoO7WQrrb7xG3sAZL03cNvXDk6kzBuF\\\\\\\/gjQsqOr2r2pJImiwrJ5ODg16gMJydSpRjktu1QLV79bL9skEl\\\\\\\/Auy3jaFCwx53i1YfL6l6Qxn\\\\\\\/0lJWgxQinxc4ljtMNrRHvEtakofFBQaqDz8+pveb867TOHMKwIIXMrXjSu\\\\\\\/jSAKZYQL9wJUg5Nwk3sP7z8wkpl6TINdBdtjzFOxc10j0v1xlmc4HiP0hiV4TXEABfeRuoUh3T2Kq9BU7Mmv4JnpBa8QTs88RWNbewAAqvEFeNkJXTWGeonoA8E7JlpI0IfW2RBQiQ4OXlkHI1FOKmYSQ+QqGiIaqD5rWA1lG4EeYEvdCsmHM9ZrbYS406rij7AxxV1zZsOqfWzTJtGghs+ooyqemNtRMig2vB6vl3wzALPOhTTLJhqtkBDL7LPHvLrnz7KGuOxLWwBhCI0QLnvjNX66bJVhOfToy87A8JWBa19GOE9mmrfMSjEuYtYKK0b6OOFzjl70O0T\\\\\\\/auffvcOIyPb0W9ADsvgnpIzCb7t4\\\\\\\/UgREADYdFUCryczkO9p612hMc7h4hgFngII9Tcn06fpVwEqB\\\\\\\/XUcecN6Y967iWNWjj2GDh46vLHn0c75ZObXkEzbQgcK4TQ+fzbBnIJT3TnjH4cZ9Lewj5D7U6ikTVVX71WQS0utTiMxlYqmE\\\\\\\/6Z+mgaMaoxhmhYdcr9E4jrmW7hmme8BqhNw3G4d9jNirpH2KQGlFUY6b5+BVUYkTuAvveFL\\\\\\\/tRuz2dMyTOh9+CaRwPvUPTHqNqypO7ZEHK\\\\\\\/rBq0Wygpoq92U7\\\\\\\/5faAeVhDjonqzhLZPEBuMQa2s62dPL6pcjakulT1CTC\\\\\\\/tEaAe8L+MK21soL+KaFfFercumNKKMgJMPcIb4RSGLiSBk1qxeb6q9cx\\\\\\\/GT+5VAMNXTJ4fhGFvbr5oOP0hXNfeBz0SIR9d\\\\\\\/aRQO+j7acnbYzMy\\\\\\\/vER91WfaxDB\\\\\\\/S8SNO2Var45c8e1T8Y1hbn4fwV1ye1YzssuV1NRFPeX3F5clk0ikWwRWAsuKFYdZNPxLZ5S7DYAgnXdQwysnr3swpwG3lGIwUfoyHUZz35QWQmxrOIx8ZyfcxjSw3BmOoJH0siLNkbr7zRfQnBvpN4PRFcAsh1wdO\\\\\\\/qB9Y7EpBELuQldPuaLM2eHMm8YLUx\\\\\\\/9Xvoxp2VIoMTS08mj9+4UIeVRrafg17yNyfzZCYhjAwIte9ZQJXMvGPqv8ZbL0+wR8FiG1YMUQj42VAxCeRabqp6hnrNPfeE+nBhmwf6BSi+uwgHOrGxEJpXexaHk8\\\\\\\/htERbQO5Rhl4z36YcOxnIPpq0Dit2RXTVpIQemEXpJfR4Y56ln\\\\\\\/B4s9FajEGx3B9UBTIM\\\\\\\/znbwH7Bo7gNU9\\\\\\\/uYXofBXkvnXpZI4anZCnqMSzGrYPoltxV0Cfqa+sw1CftUk0NuH7siGI25\\\\\\\/b+sXKUsDPlN\\\\\\\/V2yK81WZzzW+kfh8PxKvtb2232L\\\\\\\/CMiWyQBh5zBlq4iFiRk3uKdfTYOECslUPu1684n9puSkP60NI\\\\\\\/97zHUpBrcHMLkOLn1cgHnbkKTQr2IjWm9b74pCu0\\\\\\\/gwh2kRlPKYVcFCgB02qnp9LOrn0bCmFy2ACq4sNSD0Ap4NjO3X2zcWbDM0vgFheNQ+zWrFDK9KPi0uj+wBk3VvFZW5Tvljk\\\\\\\/xMe9WPejcSdVqRIQQ736pUautLHOhICHNlrjeqWI6Mxxgwd069m1i+KzpkVbJ702O1h3cAjhNd6+DbuLLs1lekivIOZkynrcDIwIImAHv8W1pDyFiIKAHYC107hQqggB4X2qUnLKWpP4DXxTxliZpj2ysOTJdiwTC5xeSd2ulUUWcnvoAzUUp8cvxow85NVM0CLyqg2gcoXt0ZvSsNVeqyCvey4r\\\\\\\/gHdIJqTmHVbqpnOJrP79ho\\\\\\\/URNLHSloNpmyXC7SFsb\\\\\\\/44LzYf4FRX41l8uUKGVNmEHnJzJuFjAOsd7abb1sxOs2+KpsSuKlPQRnGq2A29wg6t9Z38jKYAzmbtuI8WD8ZJcFarNLNtLpXPc\\\\\\\/vu94+ceSpJCEagOubLv+V3wDYoi2j1pQYBceofMnyps8G7Myaguw5P7QVtZ93dTQTL15HfBKULO+xvH0CShZ0a1Wz1Jg0azUISSVs0uFK9fWwX0Ge2SARVxZ3H2AZ162NUAqZmBGni0lhm9q79bHyAFAkwvZBhm3qe9WW1lC1ndFpWcU1+Y03rJONGwVUwYem\\\\\\\/aK6oAoamoTf9Go+F2xqlPwjTHaeTuSWAKuqv+DxsraJGDbnP30QuTwdEMrfwPRSyCwZ6X5cczV\\\\\\\/g88LOXzsc8T+zAVyQyupIKH4oAF90Aan7ZyiSAPyYpx6AHh3hRc4Gr+O4wiTXDAsRLtPi5lki\\\\\\\/8ywcK+vamJDOyBdyx2XsGOYHqYoOE52aVwvaNZiVFPJUZciNVY+XBi4HkHF37k8Q5osvmYsbcdw+nWYGlP06BCYIVgtSCovl4lxo0weqpX+tJ+1HCEYw5yPwAFjBnPhr1HUQXyVEswG53W\\\\\\\/P3mT7+8PPo5JhN0G6pKwPKTGwg3YcYhm8vJzk3yZvvR2+gnHEyvCVTVmCP+LPenZEKgF1MtkP+uYpQ56jTWC9mZFoiN9YYW4Y5pZwUFLd+IsV0SjvDTIjYqrokqamhlvAM+NgBWq0h\\\\\\\/55SRj5Dz5KeMXbMmFdvdYybfTRNSGBc1muYvCmn00TGeX60NZy6ZARKEwJRP0FdKhCHKxosE0DPczp1bMlMQxQbjefGMOt1sxgv1IJ59EfW\\\\\\\/zQhGJKM32ivcukOy4n+HhYNOu1UtKM42v5xx0GEM9u8HTaxliwUiLSx0BXivY0gxQk0nMdYskyBeb4qF5U1lCdhBa31smiw1UOWAN48+xQZGpeg557VjmSrcSe8l1n\\\\\\\/udNUf7+3pjcsLfP46Noi\\\\\\\/W384y8weQWIjUbSGMdoVL18fUDoEM\\\\\\\/FHmKe2RLkS2heIbwHrHhUOzvw8x1vKxyAUO60YyHDtxOgbzOGG4kxbG0DG5DB1WPdRkAnxFU2bQbb1oxeCReAIUOH\\\\\\\/My5UQTxvWif40izD0y+qSQxPTzc++RMQ2Ai3w9upR+NVNfHzEOR7BY\\\\\\\/NrKlURwRzP3ahq+ZP8ggpYn8qzDVCKnj\\\\\\\/KiCFatsLE7cduEyD8HVqcgLrxqjn1JOvVaYELbhdtIDgcD0udTrvfpMVEzwrAoph35ulasGpvwiCE1eflkg5CSDWfQtZ999OTyaSN\\\\\\\/y9Y06ple4rSY\\\\\\\/N5nVvHOKyausKkEd27jv1n\\\\\\\/zjbd9WOK\\\\\\\/+Ckiuc5KYN3WvFiJZjKg1mxgh5xXAiLwdWdsVtEQJtACh4N6xC\\\\\\\/k\\\\\\\/5Z+hKBH+YeTu4yJAcPS8OJaMhDLFYUoU0rO2W+ySYHBFuqcpwVhUnpArn6WU\\\\\\\/kp2P94tsh\\\\\\\/VzVmvZIPbixgNLIThZXvElE+qpZUaKALQp2DskqnPeOxFAWzud16BHRWqetju2B8dwkBP1Y6tZv4wtDq0Jayf2Gg\\\\\\\/HW4aCwdB8XKFcS8xYGIK3\\\\\\\/lVuD3HlidP8nXsygS0xqgZL1\\\\\\\/AJpgoHMOp2LRPJmqAgY1paOsALITJoBOS3kPxa9jS3bPFsv2AcgXsb0PriJhnXpRWBLny+1wmO4CABPtlGLZMXd9wy0+GvI84ZILAt9xVEjm4DSdwAqibDOAvtyCabMq0TjZ1byvopAxX7neD3Sk46cBUpbpWMIErpSa+HQ5PSIxL5hZohN1plIU5\\\\\\\/GegUCwGdZlpHNmFW+fNplkRj7+8a7AQPP+CRV77TvnGg6DFIFY0zVdTMpzOO6m+K554QnKbgqb9ANZxe0AHPXm+DzsAsA4PO\\\\\\\/jV3SbhzX0pIV5ufJzg8hFGzci7cL5AR7Jh+z8h+38TtEUOvLnBha2kcSbuGmq\\\\\\\/h3UNYtyucpnvqTyQRrpdyu3d5fkHcIeYqbL1ApZfi6HA12efD1wJ+85T556We\\\\\\\/zD5AUVbuQmLL0Tt0JOF9ZB97zGT3saELw88qBZxxF3XDmXgJAViWXgRybq2R3ltDrGNV0wxvIVEM1ZWcGMNbeagh2GqAN90G5x\\\\\\\/N4yc2hbv07ThGw7xxIYbHRsbuWoaFoE9bQ7rJbFJ7RyOoHZAT3Gdx0uEd5+FAurwFwfC1kOM6poVkuDyI6n2IfG7OgUhYvdF+sueuca5JbwUPrqwpE4+kkhMb\\\\\\\/WkDngSk\\\\\\\/zqBnYCQbnYSd2lk582cAHSH60belEH\\\\\\\/ULTE6n+QRKmWjZw8vllzTd0eJJhUi8RwB56l9Cp5sk9uuL+E9yQRzKwG62UDLWRQ1IbVr57uDiiqvhEV+4tFAZ4UiEN\\\\\\\/bI9lKsc+aS6jlkX0eaCy2kDwzsKNK2EhX7Z0VXan+gmEmGt06N1GGfz+xs9mSasbljtgcZaZefVeiroHuQj+zPfMLa3MwqXMyOo\\\\\\\/M3e2jPbKaYqvD8sWVVpyxvby9xKQV26fo3xEYji11vJBCnOFDY7yOQRpd0j44sYfIDn2PYD9KUFXnKsVxZe6PrnBhr4GprfFU0E\\\\\\\/E9VK67H2qqhWvvgTzkCh8FFZAGdSkg22BXSD1FZePTI\\\\\\\/9yG7E9RvN9tu1gfY8zsn7uSu+1DzMxCFI9MA32yCbgDq6zGqjOedFyADP7c2UPEttdYovlyrgJUw+Jn2QYkkBmVEziYpby152G2jDr\\\\\\\/j3fIN37pkhpKFaB+vM+iiGu1+u17G8uzivCyqwpZm7tNNyUbvlDOEIJ3dszFsp\\\\\\\/CKIsa57wO4+jDtWJjBBET5drNGk0e51qpp7qaQDI39M3eiIbOxV3S\\\\\\\/7oxDrauCKZSSYL7qtBJvS+dpEIPZw+e3rx89yOAx8Q2bVUQcaqUbBAoHjvqi6U5pb7pQNrG\\\\\\\/PGAE8gq0YrwaJsak41+CeQIhYRILN9zKXlmv03v3BmAqkfTpRMFb4dFwwa1VffnqSr5kA7IrBpCr9wEj1Lw\\\\\\\/cTmiWvHm74e\\\\\\\/Xus3QqV8BvyVOwbUemrh5JjIcKOEG96KzkBYHpPLrfbGcOOOrhlmVtU692dE6oOv6EHb6WoeWkwr+yQctjHSpUAaiIU7mfBqhmQEsEDIBaONA5vUtRwlyehGaeSRUVIIHUuO5aZBMnZvr1Gw7yJod30ta9Hb\\\\\\\/eHCrZ0Ls+90+Akgcot92gL\\\\\\\/yGTHblFUkOI6wymSqUzz35lVVuI0TtfJxo16h1jQ76I3Jklxhnk29XpoZYxKG0ksDyDAL7mtfReyuggHRzfDaO8KF63sfILmy6RvFIzYZi4cVEyw8Km+q8TOUAVytcCHS5d18HsGTXFgu+g3XLi3WUjAUC4JVraDGmfpZqrM978p+GwBAgbPR3zthJ5wmUiaGBQ4Kmgks+sR6JTIzUS+IlvelT\\\\\\\/EEFSqQtImR6qU8JUytqR5uPEaV+Af\\\\\\\/qdwYZPnfiZ0vayKObVtEB4YRGugrgSsJ5ySotGN7\\\\\\\/ToKSz45NKjDYB7UXElRbUFivXlY0t53QkPapK9\\\\\\\/wVFSEK4JNq6p1qdrz8b+peQtCPoIOSLudSq8VbpjyCxG0i1yVUJD\\\\\\\/N2zBXC50t429G5BD+1Ae9jpz\\\\\\\/hBt8FUNuBbSmtFSUwj8dwOvjFM\\\\\\\/oqSyn\\\\\\\/xFPzSnj7cs9PIt95ENtlPLusk1CfFDHsxESX+agE91j+h+Pye\\\\\\\/8LJtcdPVzlZs6zVcNd2xfGApJbz80NS1eXKETiAJOtmG7K5fEuDvAjln4VQz8eG9JWZ7a+U2V47\\\\\\\/N2Zn1MJ39asDoSBnuawhdapv4ZwI7LjEA4tyaLAP1rqY7c+GQ6kVGQY7qM4Yw3z8N+ELnn43NdxoCm6hZz6GyY7lqmmWHY1ewdK0zWpNFGLvrJj8ad1X6wkKqzQO+YJcuQvEAJb0K17aiCCAcsIPnPMAOFCp9+Ekx7UWt6V1b6utMBwM4wfA6Cu7eInNc3bAqpMiKxrUy9IKJDSmMeZcJXQY8+Sobf4DwkB6dZMPUkaEGxJ9hnxhAVPNcCXOK7FJv7ceGd+M+ZPkCkx7UEOnfQ0IKOMGJbxDCPmFhW4VecyXPJuVEvp31fehtZkye1Nou7v7AQCh92OHyeqWaOybZlDDQMLxxjjju79tavr529u9gwEZZt88fq3zD8q4ATvrf5mmt61S7qyLCiLC4bCdAdTHhFi4ravDBbwQW7iAH3z4ozl5qDZeqrPkFy08wiG0J8hUoAuVRZwe2DuqjAvteBK4Cs2DgtgzUJszuwxQmdOgaltxBBQzN4XglLqTjCPT4iTzP2RgwDWEZsfDcSVOIY3TTDgn5KE1muLRohv5Q71wtnPp0LgGRLM1hvF36Z60QDt0VQOzTMydm484zoypsV1xrK+Pq4PdYNnqwbnm1Swx+kdaBM1Fg\\\\\\\/CZdS87tvvu\\\\\\\/d\\\\\\\/GKiIqsbeEworF\\\\\\\/2eN+ehYDC05w5nweNjgZePrVneFI+8pqGbLeyz2Jo5of7fGfKEVfFC+0hWYZ7048qwUSiNkfr\\\\\\\/HkA\\\\\\\/UdrWRlRCixOC0Kw7Pnqw4QkIIaITt\\\\\\\/n8GQnvg72wD8c7tT21OVFudBPW3B63ObhHP0s18JRLgS3lpgroJt+Qo90gRbL9Dz0TZERji7TnkSKfOW4mLawKygPML8\\\\\\\/RYrQ\\\\\\\/WKFNvHZrVCO9dYDJOs55+xdJKVL5on94R+WTtH6Nna4PtH0q+tsc79Uuwm+XdFP8yrGnXr3wU7e0eaekejPODo3wjN11kvGr9eK4379SuXq6hRwWhqHO77USjFsamzU50RcPjOIwnL9A\\\\\\\/MCLo64JLZLUsg1fheYrU1KcDNG6lfsxcvopZMdtOKrAnrMW8BkbV1RFO039WCIH3qMeG03tQQUsHspHFj+V+ONjxUyukI88zHzOhpGLnFdHi5RWrvuqnRFS0BxtbgdMBz\\\\\\\/9Mf0BYz0llq1ATUkXKsNq4c8hLLyFNMM8xiPc4uYpC7C+06w8aqdyDxobAp0XEa87YlzIg8GEM69U5BDoT4itTlFb3RHvFP879Mo1EfP6tfErwu6woiH\\\\\\\/yMixZfV\\\\\\\/z9k4RLUBU\\\\\\\/ypahIZH+K0r8SXPC1Els+595AnHWlHqtEddM5sIPYfrBmE42DNPNRawdenNr77sW5ghodgcHBx7Si6CvaBXV7r9M090W2OHbTVymkvrRtvPreTdPFs2i\\\\\\\/p4QCXcp4bPfO4j34OpR9\\\\\\\/yKr9JfGIZN9HHPp6\\\\\\\/LSxCPyGrZ7lmVEge1ziHCa9Oz7ERyMsQuP0IG\\\\\\\/Ra\\\\\\\/NLSriVKI349fg2j4OCmqjTJ3vKfIzJ5QxqQpEjKBs07UAiZzHrgvLMV84PO5QdIdz6RoERx5FuGMrST2KR278oJ6TOoTZaiCqNdCM5DG5bszpfBzwRst90fEdRbeCOwQTCzfLx43o5CFbGIWd0tkF9mk2yUX\\\\\\\/ZpE9AvUiGPSELcOR6uqm6Ie85YQhds6s48g==\\\u0022,\\\u0022iv\\\u0022:\\\u00221PP6136DM3x7lsip68C66Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222595e1c9b8993668f73f51ec25fc7fb94690ae0437d87d6dbe3e1c753a1272b5\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022HQ+TU2piYbusCE9OkPjeKmE092CFJPltwmNBn3rYGvSpR\\\\\\\/d5BtHEfd4adfCbuX0aqRg4ZDFSplGlXhWebefzgqb1sGwTJtUY07JxN3apWEfwf8jXxpu5Kngtq61ixyO+x4LOC7IO4MLfug0K2dEgbDX+4eXbWnBisHzIfR2T89M3GJT\\\\\\\/tFakOt20Baw1p\\\\\\\/pLGRFDJX4WGBXin\\\\\\\/7OolB4oUg8XJ0TO12sTzUbFeI3+7quUX31fHyIrx10laaBp2KksK1TosDdNE3+ppb+XrBwb+DbwwmLX2AWLlevlDrvDNo2FChNsZVrv6seO9J9EHIETA8uBMFKc9g93lE0ju2q1ZuEj2DoIrcOobxNTC3g\\\\\\\/RdjTcG2B\\\\\\\/ix39UEFCD38Esbs7frnilTaKKakUsVvhescJbO6F++NriTczs6aemXehdYv8UJH0QicEIXKgBguyD0G0hGJIxcWBdVEiakTV1zjsG6rwocrKva\\\\\\\/CdfiCkAEIaoMpu3Dz\\\\\\\/UvIMdf+DYVpr\\\\\\\/pDvgqwGdLCORAeZkiFthOy7tiF2QoaWrx83uEiKbeLjYRmwOtMRGxHBl70EnOxUn5emuk6CM7FU2ymHBN+36W58EDlSZhjKolmFxsbfRT2vS1bSn3jDMvWaXfLDN2XLK9D7NcP7Smj1ag5oNQzBa6z9kEIc7IAW7GDtEQGuH9Aeh\\\\\\\/a8B4s4x3VywHTO5\\\\\\\/zxQlypuB+BmAraCpRSIRIh6jaUyJbnrIQ4W+IeNI0jEaFG4Ye4r7910heFVztuRj+ujthVTxLtLn0jBDY1duhiqz5SMeFdzQykIjR15GBMxzw4mZzsdemClHpQNKbClM3+Zpl0uw41kKUj7iaBnL9SjgoxSZ5\\\\\\\/tJ3ysX1Kss3giDiengQaJOnj9BRltrBQoa839ieL1i38Y\\\\\\\/TMLBIxX\\\\\\\/EEHCTtJ4GdviV6UWQN+1svJjBfoRdWXRsDOomvpz\\\\\\\/BANUm66pmuiuOZw5PiJ+QDxeX1AfWeOGf4YpsIE+HKBOij\\\\\\\/Uyzwp8Aayrg6Qym\\\\\\\/\\\\\\\/YF6kX4MbzVNX4XFAQKFMjdvmAxt2h2tU+Gi6wOXMRvpM6H1Mb4cOXklJaqS1N2nEfUyZXWRZBdvR0owkslpLFz\\\\\\\/Ozg\\\\\\\/iMunh6pR3AcglO1gwf8OKLuK0+2Wii9EOtX8iMS1LywW3sWBbDVz4kBWXFu9pkXJoUL5dldmh8QPk2ycEN8r2S\\\\\\\/RLmVKq61J3bFgRuKu7FKqSOsa51Rsk7rKqQBXgCLQUUC7M\\\\\\\/ltI996fNUx6JlNWSgpHrupRegS\\\\\\\/ICdlb7bX+TGrPMru0JBdAfzoQ+Wez+Dhb1Xvxq7bnvlVMu4hyBkL1cJQYE4EefN6\\\\\\\/GjCpuo5Vm5TBtB3P\\\\\\\/j30mOzyC9NggiTnUF1YwEHatJuLKlBka6FZwZhWzjEry1kEOFf3HDM9OEaGQwtB2XpxnAsWiWqAv4mDyxA12jCndlt64sZmDdzWq7c+YpqBITCal7NsKmyjmuO1CbLq7zTQDbvvOWOVU1m6ri+IBTn45NILGLEe0YqwAKXuElKTKeJ4qUuxVs9rpgVeItvfMpR6o3m4x5hDaKnUcq8I+7yNT1HS\\\\\\\/VciMEHYtqVt4VgKA6Hez\\\\\\\/+T\\\\\\\/94+p0QQHUkiIYFwNm3k5nDGWOYnaWuL7sl4HLkDPt0Oj6NGmvcNoZpC78pv3hWwHTvEZnUPzVf5Bi1aH1I6UMC2HOkO2YgFYZLAUartR4QL8CMOZ8JOrKwTypPjHNX35e8OZxbd1ZOiPrx3+Imbbild3Wo\\\\\\\/gCywahryylkN\\\\\\\/ii+E3bS89W7naP2qq2h5xmvsSBhP34KmcjseD2TMckrcTvprQExjEKUPd\\\\\\\/CpkNfLfTKfG58khMwu7soiugCzKRYUlITHyeiC+6rMH1nFpWyK1PywK0mjC5e\\\\\\\/dbHUGx\\\\\\\/e8uQ5gezibZ\\\\\\\/DRmuWtOj4I6TIzzbuqA9MG6njqG6\\\\\\\/W773g47IDPkTSbsYoCdmBrNwnfa1TkvvTt5uFOM9lLUmD\\\\\\\/R0JDq4vofN8bAXFJO5dS\\\\\\\/gijg5Dw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022RKOYv4LK3nB\\\\\\\/mlApMYTIGw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ec42a3bb7431c2aa863cf75e296c4a05d1d5ab5fd102da5222703995bf1e5279\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460568 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/2/625" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-rcIGGEDPZAXVmcDydUMBbmHCnHwwHDPbR5HrqAico1Y" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/2/625" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-tgbuZ0r63qHVrsDie7lbUeYfsEK02mhZBMWSV77N4_c" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/2/617" title="Predictability of Genetic Interactions from Functional Gene Modules" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/2/637" title="Comparative High-Density Linkage Mapping Reveals Conserved Genome Structure but Variation in Levels of Heterochiasmy and Location of Recombination Cold Spots in the Common Frog" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460568" id="node460568" data-pisa="ggg;7/2/625" data-pisa-master="ggg;g3.116.037788" data-apath="/ggg/7/2/625.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title"><em>upSET</em>, the <em>Drosophila</em> homologue of SET3, Is Required for Viability and the Proper Balance of Active and Repressive Chromatin Marks</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Kyle A. McElroy</span>, <span class="highwire-citation-author" data-delta="1">Youngsook L. Jung</span>, <span class="highwire-citation-author" data-delta="2">Barry M. Zee</span>, <span class="highwire-citation-author" data-delta="3">Charlotte I. Wang</span>, <span class="highwire-citation-author" data-delta="4">Peter J. Park</span> and <span class="highwire-citation-author" data-delta="5">Mitzi I. Kuroda</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">625-635;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.037788 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460568" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Kyle A. McElroy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Genetics, Brigham and Women’s Hospital, Boston, Massachusetts 02115</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Harvard Medical School, Boston, Massachusetts 02115</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kyle%2BA.%2BMcElroy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McElroy%20KA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKyle%2BA.%2BMcElroy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Youngsook L. Jung </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Genetics, Brigham and Women’s Hospital, Boston, Massachusetts 02115</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Center for Biomedical Informatics, Harvard Medical School, Boston, Massachusetts 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Youngsook%2BL.%2BJung%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jung%20YL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYoungsook%2BL.%2BJung%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Barry M. Zee </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Genetics, Brigham and Women’s Hospital, Boston, Massachusetts 02115</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Harvard Medical School, Boston, Massachusetts 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Barry%2BM.%2BZee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zee%20BM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABarry%2BM.%2BZee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Charlotte I. Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Genetics, Brigham and Women’s Hospital, Boston, Massachusetts 02115</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Harvard Medical School, Boston, Massachusetts 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charlotte%2BI.%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20CI&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACharlotte%2BI.%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Peter J. Park </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Genetics, Brigham and Women’s Hospital, Boston, Massachusetts 02115</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Center for Biomedical Informatics, Harvard Medical School, Boston, Massachusetts 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BJ.%2BPark%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Park%20PJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APeter%2BJ.%2BPark%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Mitzi I. Kuroda </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Genetics, Brigham and Women’s Hospital, Boston, Massachusetts 02115</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Harvard Medical School, Boston, Massachusetts 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mitzi%2BI.%2BKuroda%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kuroda%20MI&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMitzi%2BI.%2BKuroda%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mkuroda@genetics.med.harvard.edu" class="" data-icon-position="" data-hide-link-title="0">mkuroda@genetics.med.harvard.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/2/625" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460568" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460568/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/2/625.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460568" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460568/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/2/625.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460568" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460568/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/2/625.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460568" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460568/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/2/625.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460568" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460568/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Chromatin plays a critical role in faithful implementation of gene expression programs. Different post-translational modifications (PTMs) of histone proteins reflect the underlying state of gene activity, and many chromatin proteins write, erase, bind, or are repelled by, these histone marks. One such protein is UpSET, the <em>Drosophila</em> homolog of yeast Set3 and mammalian KMT2E (MLL5). Here, we show that UpSET is necessary for the proper balance between active and repressed states. Using CRISPR/Cas-9 editing, we generated S2 cells that are mutant for <em><a href="http://flybase.org/reports/FBgn0036398.html">upSET</a></em>. We found that loss of UpSET is tolerated in S2 cells, but that heterochromatin is misregulated, as evidenced by a strong decrease in H3K9me2 levels assessed by bulk histone PTM quantification. To test whether this finding was consistent in the whole organism, we deleted the <em><a href="http://flybase.org/reports/FBgn0036398.html">upSET</a></em> coding sequence using CRISPR/Cas-9, which we found to be lethal in both sexes in flies. We were able to rescue this lethality using a tagged <em><a href="http://flybase.org/reports/FBgn0036398.html">upSET</a></em> transgene, and found that UpSET protein localizes to transcriptional start sites (TSS) of active genes throughout the genome. Misregulated heterochromatin is apparent by suppressed position effect variegation of the <em>w<sup>m4</sup></em> allele in heterozygous <em><a href="http://flybase.org/reports/FBgn0036398.html">upSET</a></em>-deleted flies. Using nascent-RNA sequencing in the <em><a href="http://flybase.org/reports/FBgn0036398.html">upSET</a></em>-mutant S2 lines, we show that this result applies to heterochromatin genes generally. Our findings support a critical role for UpSET in maintaining heterochromatin, perhaps by delimiting the active chromatin environment.</p></div><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/drosophila" class="hw-term hw-article-keyword hw-article-keyword-drosophila" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Drosophila</em></a></li><li class="kwd"><a href="/keyword/chromatin" class="hw-term hw-article-keyword hw-article-keyword-chromatin" rel="nofollow">chromatin</a></li><li class="kwd"><a href="/keyword/heterochromatin" class="hw-term hw-article-keyword hw-article-keyword-heterochromatin" rel="nofollow">heterochromatin</a></li><li class="kwd"><a href="/keyword/position-effect-variegation" class="hw-term hw-article-keyword hw-article-keyword-position-effect-variegation" rel="nofollow">position effect variegation</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/upset" class="hw-term hw-article-keyword hw-article-keyword-upset" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">upSET</em></a></li><li class="kwd"><a href="/keyword/set3" class="hw-term hw-article-keyword hw-article-keyword-set3" rel="nofollow">SET3</a></li><li class="kwd"><a href="/keyword/mll5" class="hw-term hw-article-keyword hw-article-keyword-mll5" rel="nofollow">MLL5</a></li></ul><p id="p-3">Chromatin, the environment in which DNA is packaged, and transcription, the molecular dance to faithfully express the genic content of DNA, are intimately linked. As such, chromatin proteins exert a strong influence on the timing, patterning, and level, of gene expression. This influence can be in the form of direct binding to histones/nucleosomes or DNA, the post-translational modification (PTM) of histones, or DNA modification. One family of chromatin-associated proteins that post-translationally modify histones is the SET domain-containing proteins. The SET domain catalyzes methylation of histone tails. Different SET domain-containing proteins create unique histone modifications, which are associated with different forms of active and repressed chromatin environments.</p><p id="p-4">Of the family of SET domain proteins, one paralog is a notable exception to this paradigm. The <em>Drosophila</em> protein UpSET and its homologs Set3 in yeast and KMT2E (henceforth referred to as MLL5) in mammals are not known to have histone-modifying activity (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Pijnappel <em>et al.</em> 2001</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Madan <em>et al.</em> 2009</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Rincon-Arano <em>et al.</em> 2012</a>). In fact, conserved catalytically important residues in the SET domain are mutated in UpSET/Set3/MLL5, suggesting that protein function must not rely on the competence of the SET domain (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Madan <em>et al.</em> 2009</a>). Rather than catalyzing histone methylation, these proteins have been characterized in yeast and flies to form complexes with histone deacetylases (HDACs) (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Pijnappel <em>et al.</em> 2001</a>; <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Rincon-Arano <em>et al.</em> 2012</a>).</p><p id="p-5">Set3 is a nonessential gene in yeast (<a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Pijnappel <em>et al.</em> 2001</a>), with a mutant phenotype of defective transcription kinetics when cells are metabolically challenged (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Wang <em>et al.</em> 2002</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Kim <em>et al.</em> 2012</a>). Set3 complex (Set3C) was also recently tied to the DNA damage response operating under a model of altered histone acetylation dynamics (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Torres-Machorro <em>et al.</em> 2015</a>). MLL5 in mammals has been tied to several different cellular processes, including hematopoesis (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Heuser <em>et al.</em> 2009</a>; <a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Madan <em>et al.</em> 2009</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Zhang <em>et al.</em> 2009</a>), cell cycle progression (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Deng <em>et al.</em> 2004</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Cheng <em>et al.</em> 2008</a>), oncogenesis (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Emerling <em>et al.</em> 2002</a>), and DNA methylation (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Yun <em>et al.</em> 2014</a>), though its exact mechanistic role or binding partners in these diverse functions have not been fully resolved. In <em>Drosophila</em>, <em>upSET<sup>e00365/e00365</sup></em> flies were described to be viable, but with a female fertility defect due to derepression of transposable elements in the ovary (<a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Rincon-Arano <em>et al.</em> 2012</a>).</p><p id="p-6">We previously identified the UpSET protein (CG9007) as a top interactor with the MSL3 protein by a cross-linked affinity purification technique (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Wang <em>et al.</em> 2013</a>). MSL3 is a chromodomain protein that is a core constituent of the Male-specific lethal (MSL) dosage compensation complex in <em>Drosophila</em>. The five genetically defined MSL proteins (Male-specific Lethal-1, -2, -3, Maleless, and Males-absent-on-first), and two redundant noncoding RNAs (<em>RNA on X -1 and -2</em>) localize specifically to the male X to create a unique chromatin environment, and boost expression of active genes (for review, see <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Lucchesi and Kuroda 2015</a>). The chromatin environment that is created by the MSL complex is catalyzed by the Males-absent-on-first (MOF) protein, which acetylates histone 4 on lysine 16 (H4K16ac). This modification has a very stereotypical pattern near TSSs on autosomes, and across the female genome. In contrast, on the male X, H4K16ac is enriched on gene bodies, reflecting the localization of the MSL complex, and its putative function in transcriptional elongation.</p><p id="p-7">The potential interaction between UpSET, a member of an HDAC complex, and MSL3, a member of the MSL complex that makes an acetyl mark, led us to create <em><a href="http://flybase.org/reports/FBgn0036398.html">upSET</a></em> mutant S2 cell lines using the CRISPR/Cas-9 system. We surveyed bulk histone PTM levels in these cells to test for a global effect on the H4K16ac dosage compensation-associated mark, but instead observed that the relative amounts of the heterochromatin mark histone H3 lysine 9 dimethyl (H3K9me2) were strongly reduced. To investigate this potential heterochromatin association, we created an <em><a href="http://flybase.org/reports/FBgn0036398.html">upSET</a></em> deleted <em>Drosophila</em> fly line using the CRISPR/Cas-9 system for genome engineering coupled with a homologous recombination donor. We observed <em>upSET<sup>DsRed+{</sup></em><sup>Δ</sup><em><sup>upSET}/DsRed+{</sup></em><sup>Δ</sup><em><sup>upSET}</sup></em> to be lethal in both males and females, confirming a role independent of, or broader than, regulation of male-specific dosage compensation, and in contrast to a previous report of homozygous mutant viability (<a id="xref-ref-42-4" class="xref-bibr" href="#ref-42">Rincon-Arano <em>et al.</em> 2012</a>). <em><a href="http://flybase.org/reports/FBgn0036398.html">upSET</a></em><sup>Δ</sup><em><sup>DsRed{</sup></em><sup>Δ</sup><em><sup>upSET}/+</sup></em> heterozygotes exhibit a suppressor of variegation phenotype, suggesting that heterochromatic silencing is influenced by UpSET dosage. Furthermore, we also detected perturbation of heterochromatin gene expression in <em><a href="http://flybase.org/reports/FBgn0036398.html">upSET</a></em> mutant cell lines using nascent-RNA sequencing. Together with previous studies, our results suggest that heterochromatin and heterochromatin-embedded genes are particularly sensitive to a balance of chromatin-modifying activities, regulated in part by the UpSET protein.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Generating upSET mutant S2 cells and flies</h3><p id="p-8">S2 cells and mutant S2 lines were maintained at 26° in Schneider’s medium supplemented with 10% FBS and 1% antibiotic/antimycotic (Gibco). Mutant S2 cells were generated using the CRISPR/Cas-9 system essentially as described (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Housden <em>et al.</em> 2015</a>). Guide RNA sequences were obtained from the <em>Drosophila</em> RNAi Screening Center (DRSC)’s sgRNA design tool (<a href="http://www.flyrnai.org/crispr2">www.flyrnai.org/crispr2</a>). Oligonucleotides of the appropriate gRNA sequence were ordered from IDT (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037788/-/DC1/TableS1.docx">Table S1</a>), with additional bases to allow for ligation into the <em>Bbs</em>I site of the pL018 plasmid (a gift from N. Perrimon), which also expresses Cas-9. S2 cells were transfected using Effectene (Qiagen) using 40 ng of an actin::RFP marker plasmid (a gift from T. Wu), and 360 ng total of the appropriate pL018 construct, either singly or in combination with other pL018 constructs. At 4 d after transfection, single cells in the top 10% of RFP<sup>+</sup> cells (typically top ∼3% of total population) were sorted by FACS into conditioned medium in 96-well plates. Colony growth from single S2 cells was observed after 2–3 wk. Independent lines were expanded and tested for mutations by HRMA (high resolution melt assay) using Precision Melt Supermix (Bio-Rad). Lines scoring well in the HRMA had the gRNA target region amplified by PCR using flanking primers (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037788/-/DC1/TableS1.docx">Table S1</a>), and the resulting product was subcloned into the pCR4Blunt-TOPO vector (Invitrogen). To identify the molecular lesion, five bacterial colonies per S2 line were sequenced by Genewiz. Lines with mutations that introduce frameshifts or deletions of the start codon and adjacent sequence were used for subsequent experiments.</p><p id="p-9"><em><a href="http://flybase.org/reports/FBgn0036398.html">upSET</a></em>-directed gRNA sequences were as follows:</p><ul class="list-simple " id="list-1"><li id="list-item-1"><p id="p-10">upset1: 5′-AACCGAGTCGTGACTGGACATGG-3′</p></li><li id="list-item-2"><p id="p-11">upset3: 5′-AGGCGCGATGCCGTCTGATTAGG-3′</p></li><li id="list-item-3"><p id="p-12">upset5: 5′-TGGCCAGGCGCAGTAGTAATAGG-3′</p></li><li id="list-item-4"><p id="p-13">upset7: 5′-ACAGCAGATCAGCCTACCGCAGG-3′</p></li></ul><p id="p-14">Mutant flies were generated by injecting gRNA constructs, and a homologous recombination donor, into <em>w</em>; <em>w<sup>−</sup>{nos-cas9}/CyO</em> embryos [a gift from N. Perrimon, see also <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Housden <em>et al.</em> (2014)</a> for general guidelines for <em>Drosophila</em> CRISPR/Cas-9]. Injections were made using glass capillary needles through the intact chorion essentially as described (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Miller <em>et al.</em> 2002</a>). The gRNA constructs were designed as above, and oligos were ligated into the pU6-BbsI-chiRNA plasmid (#45946; addgene) as described (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Gratz <em>et al.</em> 2013</a>). The homologous recombination donor was constructed using 1 kb of genomic sequence flanking the 5′ and 3′ gRNA cut sites inserted into the pDsRed-attP plasmid (#51019; addgene) (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Gratz <em>et al.</em> 2014</a>), which expresses the synthetic marker 3×P3-DsRed in the adult eye. Genome (R6.12) coordinates, 5′ homology arm: 3L:14006860..14007859; 3′ homology arm: 3L:14017877..14018876. Adults resulting from the injections were outcrossed to <em>yw</em> flies, and their progeny were screened for the fluorescent DsRed marker. DsRed-positive progeny were crossed to <em>yw</em>; <em>TM3/TM6</em> flies to generate balanced stocks of <em>yw</em>; +; <em>DsRed<sup>+</sup>{</em>Δ<em>upSET}/TM3</em> or <em>yw</em>; +; <em>DsRed<sup>+</sup>{</em>Δ<em>upSET}/TM6</em>.</p><p id="p-15">In order to mitigate any interference of the DsRed marker with eye phenotype scoring in position effect variegation experiments, we excised the 3×P3-DsRed cassette using Cre recombination with the flanking loxP sites present in the pDsRed-attP vector. Cre recombinase was introduced from the <em>yw</em>; <em>MKRS</em>,<em>{hs-FLP}/TM6B</em>,<em>{Crew}</em>,<em>Tb</em> line (#1501; Bloomington). <em>yw</em>; <em>DsRed<sup>+</sup>{</em>Δ<em>upSET}/TM6B</em>,<em>{Crew}Tb</em> progeny were crossed to a third chromosome balancer line to establish balanced stocks of <em>yw</em>; Δ<em>DsRed{</em>Δ<em>upSET}/TM3</em> or <em>TM6</em>. Successful mobilization of the DsRed cassette was confirmed in all crosses by visual inspection.</p></div><div id="sec-3" class="subsection"><h3>Cloning and transgenesis for the upSET-BioTAP allele</h3><p id="p-16">The UpSET-BioTAP allele was constructed using the pRedET recombineering system (GeneBridges K002). The genomic region of <em><a href="http://flybase.org/reports/FBgn0036398.html">upSET</a></em> was transferred to the pFly (aka pGS-mw) vector, and injected into flies for site-specific integration at 53B2 on the second chromosome (#9736; BestGene stock). The resulting flies, <em>yw</em>; <em>UpSET-BioTAP/UpSET-BioTAP</em>, were crossed into the <em>DsRed<sup>+</sup>{</em>Δ<em>upSET}</em> background. <em>DsRed<sup>+</sup>{</em>Δ<em>upSET}</em>-homozygous flies carrying one or two copies of the UpSET-BioTAP allele were used for one-step ChIP (see below).</p><p id="p-17">Presence of WT <em><a href="http://flybase.org/reports/FBgn0036398.html">upSET</a></em> or the BioTAP-tagged-<em><a href="http://flybase.org/reports/FBgn0036398.html">upSET</a></em> transgene was assessed by PCR from genomic DNA isolated from two to three female flies of the specific genotype. A 544 bp product from wild type (WT), and a 1210 bp product from the BioTAP-tagged <em><a href="http://flybase.org/reports/FBgn0036398.html">upSET</a></em> construct are obtained when using the following primer pair:</p><ul class="list-simple " id="list-2"><li id="list-item-5"><p id="p-18">KAM201: 5′-gctgcacatgtttgatgataagc-3′</p></li><li id="list-item-6"><p id="p-19">KAM202: 5′-gtgcaagctcatactttatgcgc-3′</p></li></ul></div><div id="sec-4" class="subsection"><h3>Bulk histone purification and mass spectrometry</h3><p id="p-20">Bulk histones were salt-acid extracted from S2 cell lines using 2M NaCl and 0.4N H<sub>2</sub>SO<sub>4</sub> following cell lysis with RIPA buffer. Histone proteins were precipitated using trichloroacetic acid (TCA), and resuspended in 25 mM sodium bicarbonate. Resuspended histone proteins were treated with propionic anhydride (Sigma Aldrich) for bottom-up peptide analysis by liquid chromatography–mass spectrometry as described in a previous report (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Zee <em>et al.</em> 2016a</a>,<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">b</a>).</p></div><div id="sec-5" class="subsection"><h3>Small scale one-step ChIP-seq from BioTAP-tagged embryos</h3><p id="p-21">To assess the genomic localization of BioTAP-tagged proteins from a small scale of embryos, immunoprecipitation using only the proteinA moieties of the tag were performed. Using a protocol essentially described elsewhere (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Alekseyenko <em>et al.</em> 2008</a>), 0.1 g of embryos was collected, and disrupted using a motorized pestle. Formaldehyde was added to 1% final concentration, and incubated for 15 min at room temperature. Following quenching of the reaction with glycine and washing, fixed material was sonicated in RIPA buffer using a Bioruptor, four cycles of 30 sec on/30 sec off on the high setting. Sonicated material was supplemented with TritonX-100 to 1%, Sodium DOC to 0.1%, and NaCl to 140 mM, and debris was cleared by centrifugation. Chromatin was aliquoted and stored at −80° until IP. For ChIP, 20–30 μl of IgG agarose slurry per IP were washed in RIPA buffer, and incubated with chromatin overnight. Bound immunocomplexes were washed 5× with RIPA (140 mM NaCl; 10 mM Tris, pH 8; 1 mM EDTA, pH 8; 1% Triton; 0.1% SDS; and 0.1% sodium deoxycholate), once with LiCl buffer (250 mM LiCl; 10 mM Tris pH 8; 1 mM EDTA, pH 8; 0.5% NP40; and 0.5% sodium deoxycholate), twice with TE, and finally resuspended in TE. Input and IPs were treated for 30 min with RNase at 37°, then overnight with the addition of proteinase K and SDS (0.5% final), and crosslinks were reversed for 6 hr at 65°. IP samples were supplemented with NaCl to 140 mM final, and both IP and input samples were extracted with an equal volume of 25:24:1 phenol:chloroform:isoamyl alcohol. To maximize recovery in IPs, the organic fraction was extracted with TEN140 (TE + 140 mM NaCl), and pooled with the initial aqueous phase. All samples were then extracted with an equal volume of 24:1 chloroform:isoamyl alcohol, and precipitated overnight at −80° with sodium acetate and ethanol, in the presence of glycogen. The entirety of the precipitated IP-DNA and ∼200 ng of input DNA were used to create high-throughput sequencing Illumina libraries using the NEBNext ChIP-seq kit (NEB 6240). Prior to library amplification, size selection was achieved using a 2% agarose gel (50111; Lonza). Sequencing was performed at the Tufts Genomics Core.</p></div><div id="sec-6" class="subsection"><h3>ChIP-seq analysis</h3><p id="p-22">The adaptor sequences were trimmed with Cutadapt ver. 1.2.1 (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Martin 2011</a>). The reads were aligned to the <em>Drosophila</em> genome (dm3 assembly) using Bowtie ver. 12.0 (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Langmead <em>et al.</em> 2009</a>) with a unique mapping option (-m 1). Only uniquely aligned reads were used for analysis. The input normalized fold enrichment profiles were generated using the <em>get.smoothed.enrichment.mle</em> function of the SPP R package (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Kharchenko <em>et al.</em> 2008</a>), with a step size of 20 bp, and Gaussian kernel bandwidth of 150 bp. The profiles were normalized by the background scaling method. For metagene plots, the regions in the gene body except for 500 bp margins of 5′-end and 3′-end were scaled and averaged after merging two replicates. Only genes &gt;1.5 kb in length, and &gt;1 kb away from adjacent genes, were included in the metagene analysis. To estimate gene expression, RNA-seq samples profiled by modENCODE consortium were used for S2 cell and 14–16 hr embryos (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Gerstein <em>et al.</em> 2014</a>). FPKM = 1 was used as a threshold for expressed genes. To detect significantly enriched peaks, the <em>get.broad.enrichment.clusters</em> function of the SPP R package was used, with a window size of 1 kb and <em>z</em>-score threshold of 3. For downstream analysis based on peaks, we used the significant peaks from the second UpSET-BioTAP replicate, which has a better signal-to-noise ratio. The genomic annotation for UpSET was performed using CEAS (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Shin <em>et al.</em> 2009</a>). For chromatin annotation, the chromatin segmentations were obtained from previous studies of S2 (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Kharchenko <em>et al.</em> 2011</a>) and embryos (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Ho <em>et al.</em> 2014</a>).</p></div><div id="sec-7" class="subsection"><h3>Position effect variegation of the w<sup>m4</sup> allele</h3><p id="p-23">Virgin females of the genotype <em>w<sup>m4</sup>/w<sup>m4</sup></em>; +; + were crossed to males of the genotype <em>yw</em>; Δ<em>DsRed{</em>Δ<em>upSET}/TM3</em>,<em>Sb or yw</em>; +; +. Resulting progeny of this cross were sorted to the appropriate third chromosome genotype by balancer chromosome markers. These flies were maintained at 24° for 3 d, after which variegation of eye pigmentation was assessed. The extent of eye pigmentation was scored into three classes.</p></div><div id="sec-8" class="subsection"><h3>Nascent-RNA-seq from S2 cells</h3><p id="p-24">Nascent-RNA sequencing was done using a urea-based method (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Ferrari <em>et al</em>. 2013</a>) essentially as described (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Alekseyenko <em>et al.</em> 2015</a>). In short, 1 × 10<sup>7</sup> S2 cells were collected by centrifugation at 300 × <em>g</em> at 4°, and homogenized in CKS buffer + SUPERase•In RNase inhibitor (Ambion AM2696) + ProteaseArrest (786-108; G-Biosciences) with three strokes through a 25 G needle. Nuclei were collected by centrifugation, and resuspended in CF buffer + RNasin. NUN buffer was added and samples were vortexed ∼30 sec until a wispy, filamentous precipitate was apparent. This precipitate was spun down and washed three times with NUN buffer. Samples were then treated with proteinaseK in CF buffer + 0.5% SDS at 55° for 30 min. Samples were then passed through a 25 G needle five times to disrupt the chromatin, and incubated an additional 30 min at 55°. Samples were extracted twice with 25:24:1 phenol:chloroform:isoamyl alcohol, once with 24:1 chloroform:isoamyl alcohol, and ethanol precipitated overnight in the presence of glycogen (AM9510; Ambion). Resulting nucleic acids were treated with RNase-free TURBO DNase (AM2238; Ambion) for 30 min at 37°, with proteinase K + SDS for an additional 5 min at 37°, and then extracted once each with 25:24:1 phenol:chloroform:isoamyl alcohol, and 24:1 chloroform:isoamyl alcohol. Nascent-RNA was ethanol precipitated overnight except without the addition of glycogen. Illumina sequencing libraries were constructed using the NEBNext Ultra Directional RNA Library kit (NEB 7420).</p></div><div id="sec-9" class="subsection"><h3>Nascent-RNA-seq analysis</h3><p id="p-25">The reads were mapped as described above. The tag density profiles were generated using <em>get.smoothed.tag.density</em> function of SPP R package, with a Gaussian kernel of 100 bp, and a step size of 10 bp after library size normalization. To compare the enrichment values between the mutant and control, the reads were separated to sense and antisense transcripts. The fold change, and significantly changed regions in transcription between conditions, were determined using EdgeR (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Robinson <em>et al.</em> 2010</a>) after TMM (trimmed mean normalization method) (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Ross-Innes <em>et al.</em> 2012</a>). Heterochromatic and euchromatic genes were defined using the heterochromatin, and euchromatin boundary information for each chromosome obtained from a previous study based on H3K9me2 enrichment levels (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Riddle <em>et al.</em> 2011</a>). To access the significance for the portions of upregulated genes between groups, a bootstrap method was used (<em>n</em> = 1000).</p></div><div id="sec-10" class="subsection"><h3>Data availability</h3><p id="p-26">Cell and fly lines are available upon request. Mass Spectrometry raw data files are available through Harvard Dataverse (<a href="https://dataverse.harvard.edu/">https://dataverse.harvard.edu/</a>) at doi: 10.7910/DVN/LFGYUV. Gene expression (nascent-RNA) and ChIP-seq data are available at GEO accession number GSE90703.</p></div></div><div class="section results" id="sec-11"><h2 class="">Results</h2><div id="sec-12" class="subsection"><h3>CRISPR-engineered S2 cell lines tolerate inactivating upSET mutations</h3><p id="p-27">In order to assess the molecular effects of the loss of UpSET, specifically in male cells, we generated S2 cell lines stably carrying <em><a href="http://flybase.org/reports/FBgn0036398.html">upSET</a></em> mutations. S2 cells are a male <em>Drosophila</em> cell line which is highly polyploid. We reasoned that these cells may tolerate perturbed chromatin states better than the whole organism, given their tolerance of their nondiploid genome. The ploidy of the genome introduces its own challenges for genome engineering, yet we saw that mutations typically went to fixation when we used the CRISPR/Cas-9 system (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Housden <em>et al.</em> 2015</a>). We cotransfected S2 cells with an RFP-expressing marker plasmid, and a plasmid coexpressing both the Cas-9 protein and one or more of several different guide RNAs directed toward the <em><a href="http://flybase.org/reports/FBgn0036398.html">upSET</a></em> gene (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). To isolate single clones, RFP-positive transfected cells were sorted into 96-well plates by FACS. Following regrowth from these single cells, we identified lines with putative mutations by high-resolution melt assays (HRMA) (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bassett <em>et al.</em> 2013</a>). Statistically significant hits were further analyzed by Sanger sequencing to identify the nature of the molecular lesions at the gRNA target site. In this way, we were able to isolate three <em><a href="http://flybase.org/reports/FBgn0036398.html">upSET</a></em> clonal mutant lines that lack a wild-type UpSET open reading frame (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1, B and C</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/625/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Generation of upSET mutant S2 lines. (A) Schematic for the upSET gene locus. Noncoding exons are in gray, while coding exons are in light orange. Locations of guide RNA targets for Cas9 are indicated. (B) Molecular lesions generated around the upSET gRNA #1 target site, located just downstream of the upSET start codon (underlined). In the S2 reference sequence, the gRNA target is in capital letters, and the PAM is highlighted. Clone G3 has a homozygous 67&#xA0;bp deletion, removing the start codon, and 10 additional coding base pairs, as well as 54&#xA0;bp of the adjacent sequence. Clone A7 has two separate 7&#xA0;bp deletions, both resulting in frameshift mutations, and predicted truncations in the protein product. (C) As in (B), in the S2 reference sequence, the gRNA target is in capital letters and the PAM is highlighted. The molecular lesion generated around the upSET gRNA #7 target site in clone B2 carries a 1&#xA0;bp deletion, resulting in a frameshift after 367 amino acids of the wild-type UpSET protein sequence, and a predicted 436 amino acid product." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-285027449" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Generation of &lt;em&gt;upSET&lt;/em&gt; mutant S2 lines. (A) Schematic for the &lt;em&gt;upSET&lt;/em&gt; gene locus. Noncoding exons are in gray, while coding exons are in light orange. Locations of guide RNA targets for Cas9 are indicated. (B) Molecular lesions generated around the &lt;em&gt;upSET&lt;/em&gt; gRNA #1 target site, located just downstream of the &lt;em&gt;upSET&lt;/em&gt; start codon (underlined). In the S2 reference sequence, the gRNA target is in capital letters, and the PAM is highlighted. Clone G3 has a homozygous 67&#xA0;bp deletion, removing the start codon, and 10 additional coding base pairs, as well as 54&#xA0;bp of the adjacent sequence. Clone A7 has two separate 7&#xA0;bp deletions, both resulting in frameshift mutations, and predicted truncations in the protein product. (C) As in (B), in the S2 reference sequence, the gRNA target is in capital letters and the PAM is highlighted. The molecular lesion generated around the &lt;em&gt;upSET&lt;/em&gt; gRNA #7 target site in clone B2 carries a 1&#xA0;bp deletion, resulting in a frameshift after 367 amino acids of the wild-type UpSET protein sequence, and a predicted 436 amino acid product.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/625/F1.medium.gif" width="440" height="151"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/2/625/F1.medium.gif" width="440" height="151"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/625/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/625/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470283" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-28" class="first-child">Generation of <em>upSET</em> mutant S2 lines. (A) Schematic for the <em>upSET</em> gene locus. Noncoding exons are in gray, while coding exons are in light orange. Locations of guide RNA targets for Cas9 are indicated. (B) Molecular lesions generated around the <em>upSET</em> gRNA #1 target site, located just downstream of the <em>upSET</em> start codon (underlined). In the S2 reference sequence, the gRNA target is in capital letters, and the PAM is highlighted. Clone G3 has a homozygous 67 bp deletion, removing the start codon, and 10 additional coding base pairs, as well as 54 bp of the adjacent sequence. Clone A7 has two separate 7 bp deletions, both resulting in frameshift mutations, and predicted truncations in the protein product. (C) As in (B), in the S2 reference sequence, the gRNA target is in capital letters and the PAM is highlighted. The molecular lesion generated around the <em>upSET</em> gRNA #7 target site in clone B2 carries a 1 bp deletion, resulting in a frameshift after 367 amino acids of the wild-type UpSET protein sequence, and a predicted 436 amino acid product.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Bulk histone PTM analysis in upSET mutant S2 cells reveals a perturbed chromatin state</h3><p id="p-29">To explore the potential role of UpSET in chromatin and gene expression, we sought to obtain a comprehensive assessment of all histone post-translational modifications that were altered by the loss of <em><a href="http://flybase.org/reports/FBgn0036398.html">upSET</a></em>. We isolated bulk histones from S2 cells, and all three <em><a href="http://flybase.org/reports/FBgn0036398.html">upSET</a></em> mutant cell lines, using a salt/acid extraction method (<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Zee <em>et al.</em> 2016a</a>). To quantitatively recover histone peptides with their PTM state intact for mass spectrometry analysis, histones were derivatized in solution with a protecting group that allows recovery of histone peptides from reverse phase chromatography. Using the mass difference and relative elution order between the protecting group and various modifications (acetylation, methylation, etc.), we were able to quantify the relative abundance of a given histone PTM with respect to all observable modified forms within the same tryptic peptide backbone within each sample. As a positive control for our methodology, we also assessed the histone modifications in S2 cells treated with the general HDAC inhibitor, sodium butyrate (S2but), which should result in global accumulation of acetylation (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Candido <em>et al.</em> 1978</a>).</p><p id="p-30">We observed a slight increase in H4-monoacetylation on the peptide that contains lysines at residues 5, 8, 12, and 16 in <em><a href="http://flybase.org/reports/FBgn0036398.html">upSET</a></em> mutant cells, and a larger increase in the butyrate-treated cells (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). We obtained similar results in replicates (Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037788/-/DC1/FigureS1.pptx">Figure S1A</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037788/-/DC1/FileS1.docx">File S1</a>). We also found an increase in H3K4me1, as compared to unmodified H3K4 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037788/-/DC1/FigureS2.pptx">Figure S2</a>). However, a comprehensive view of the modification state at this residue was not possible, due to the hydrophillic nature of the H3K4me2/me3 peptides, which precludes their recovery during desalting prior to liquid chromatography-mass spectrometry.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/625/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of histone post-translational modifications from bulk histones in upSET mutant cells. (A) Relative quantification (1.0&#xA0;=&#xA0;100%) of H4K5K8K12K16 mono-, di-, tri-, and tetra-acetyl patterns in upSET mutant cell lines (G3, B2, and A7), butyrate treated cells (S2but), and the parental S2 line. The patterns of the mutant or butyrate treated cells differ from the parental cell line. Sodium butyrate inhibits deacetylases, resulting in accumulation of acetylation marks. (B) Relative quantification (1.0&#xA0;=&#xA0;100%) of H3K9K14 PTM patterns in upSET mutant cell lines (G3, B2, and A7), butyrate treated cells (S2but), and the parental S2 line. The H3K9me2/me3 marks are depleted from upSET mutant S2 cells, suggestive of an effect on heterochromatin." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-285027449" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Analysis of histone post-translational modifications from bulk histones in &lt;em&gt;upSET&lt;/em&gt; mutant cells. (A) Relative quantification (1.0&#xA0;=&#xA0;100%) of H4K5K8K12K16 mono-, di-, tri-, and tetra-acetyl patterns in &lt;em&gt;upSET&lt;/em&gt; mutant cell lines (G3, B2, and A7), butyrate treated cells (S2but), and the parental S2 line. The patterns of the mutant or butyrate treated cells differ from the parental cell line. Sodium butyrate inhibits deacetylases, resulting in accumulation of acetylation marks. (B) Relative quantification (1.0&#xA0;=&#xA0;100%) of H3K9K14 PTM patterns in &lt;em&gt;upSET&lt;/em&gt; mutant cell lines (G3, B2, and A7), butyrate treated cells (S2but), and the parental S2 line. The H3K9me2/me3 marks are depleted from &lt;em&gt;upSET&lt;/em&gt; mutant S2 cells, suggestive of an effect on heterochromatin.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/625/F2.medium.gif" width="397" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/2/625/F2.medium.gif" width="397" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/625/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/625/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470268" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-31" class="first-child">Analysis of histone post-translational modifications from bulk histones in <em>upSET</em> mutant cells. (A) Relative quantification (1.0 = 100%) of H4K5K8K12K16 mono-, di-, tri-, and tetra-acetyl patterns in <em>upSET</em> mutant cell lines (G3, B2, and A7), butyrate treated cells (S2but), and the parental S2 line. The patterns of the mutant or butyrate treated cells differ from the parental cell line. Sodium butyrate inhibits deacetylases, resulting in accumulation of acetylation marks. (B) Relative quantification (1.0 = 100%) of H3K9K14 PTM patterns in <em>upSET</em> mutant cell lines (G3, B2, and A7), butyrate treated cells (S2but), and the parental S2 line. The H3K9me2/me3 marks are depleted from <em>upSET</em> mutant S2 cells, suggestive of an effect on heterochromatin.</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">Unexpectedly, the largest relative changes in modifications on total histones in the <em><a href="http://flybase.org/reports/FBgn0036398.html">upSET</a></em> mutant cells were in H3K9me2 and H3K9me3 levels, which were greatly diminished compared to wild-type S2 cells, with a concomitant increase in monomethylated and unmethylated H3K9 (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). We obtained similar results in replicates (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037788/-/DC1/FigureS1.pptx">Figure S1B</a>). In contrast, cells treated with S2but also experienced a drop in H3K9me2/3 levels, however with a concomitant increase in H3K9K14 monomethylation or diacetylation. H3K9me2 is a modification known to be enriched in heterochromatin (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Ebert <em>et al.</em> 2004</a>). The HP1a protein, which is critical for heterochromatin formation, interfaces with this mark via its chromodomain (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Platero <em>et al.</em> 1995</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Jacobs and Khorasanizadeh 2002</a>). These data suggest an as-yet undetermined role for UpSET in heterochromatin.</p></div><div id="sec-14" class="subsection"><h3>upSET is an essential gene in Drosophila</h3><p id="p-33">The previous characterization of <em><a href="http://flybase.org/reports/FBgn0036398.html">upSET</a></em> mutant flies utilized the only two lines then available, which carry a <em>P</em>-element and Minos insertion in the <em><a href="http://flybase.org/reports/FBgn0036398.html">upSET</a></em> gene, respectively (<a id="xref-ref-42-5" class="xref-bibr" href="#ref-42">Rincon-Arano <em>et al.</em> 2012</a>). However, the insertions carried by both of these lines leave the coding sequence largely intact. While Western blotting suggested no residual protein, given the possibility that those alleles could in fact be hypomorphic instead of complete loss-of-function, we sought to create an <em><a href="http://flybase.org/reports/FBgn0036398.html">upSET</a></em> deletion allele with the coding sequence removed from the genome. To accomplish this, we turned to the versatile CRISPR/Cas-9 genome engineering system. We coinjected <em>w</em>; <em>w<sup>−</sup>{nos-cas9}/CyO</em> embryos, which express Cas-9 in the germ line, with two guide RNA constructs, and a homologous recombination donor marked with 3xP3-DsRed (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). We isolated balanced flies that carried the DsRed marker, and confirmed the loss of the entire predicted <em><a href="http://flybase.org/reports/FBgn0036398.html">upSET</a></em> coding sequence by PCR. We found that homozygosity for the <em><a href="http://flybase.org/reports/FBgn0036398.html">upSET</a></em> deletion is lethal, with a low escape rate. The few escapers that were recovered were sickly, and did not reproduce with <em>yw</em> mates. We were able to rescue this lethality with an UpSET-BioTAP transgene (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>). PCR from rescued individuals confirmed that no wild-type <em><a href="http://flybase.org/reports/FBgn0036398.html">upSET</a></em> DNA remained (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3C</a>). Taken together, these results suggest that, contrary to previous reports, <em><a href="http://flybase.org/reports/FBgn0036398.html">upSET</a></em> is an essential gene in <em>Drosophila</em>, and that the previous alleles are hypomorphs, rather than <em>bona fide</em> complete loss-of-function.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/625/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="upSET is an essential gene in Drosophila. (A) The upSET gene, as depicted in Figure 1, with the location of guide RNA targets (blue triangles), homologous recombination donor arms (green rectangles), and the structure of the DsRed donor construct indicated. The BioTAP-tag insertion in a separate transgenic rescue construct shown below is indicated in crimson, with flanking PCR primers (red, not to scale). (B) upSET is homozygous lethal, but is rescued by the UpSET-BioTAP transgene (compare counts in columns three and four to columns seven and eight). Cross scheme is shown above the table. Table contains the expected ratio of adult progeny per genotype, as well as the observed count and ratio of these genotypes. (C) PCR analysis reveals that homozygous escapers lack the 544 bp wild-type product. The rescued flies produce only the band corresponding to the tagged construct." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-285027449" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;upSET&lt;/em&gt; is an essential gene in &lt;em&gt;Drosophila&lt;/em&gt;. (A) The &lt;em&gt;upSET&lt;/em&gt; gene, as depicted in Figure 1, with the location of guide RNA targets (blue triangles), homologous recombination donor arms (green rectangles), and the structure of the DsRed donor construct indicated. The BioTAP-tag insertion in a separate transgenic rescue construct shown below is indicated in crimson, with flanking PCR primers (red, not to scale). (B) &lt;em&gt;upSET&lt;/em&gt; is homozygous lethal, but is rescued by the UpSET-BioTAP transgene (compare counts in columns three and four to columns seven and eight). Cross scheme is shown above the table. Table contains the expected ratio of adult progeny per genotype, as well as the observed count and ratio of these genotypes. (C) PCR analysis reveals that homozygous escapers lack the 544 bp wild-type product. The rescued flies produce only the band corresponding to the tagged construct.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/625/F3.medium.gif" width="440" height="317"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/2/625/F3.medium.gif" width="440" height="317"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/625/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/625/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470384" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-34" class="first-child"><em>upSET</em> is an essential gene in <em>Drosophila</em>. (A) The <em>upSET</em> gene, as depicted in <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>, with the location of guide RNA targets (blue triangles), homologous recombination donor arms (green rectangles), and the structure of the DsRed donor construct indicated. The BioTAP-tag insertion in a separate transgenic rescue construct shown below is indicated in crimson, with flanking PCR primers (red, not to scale). (B) <em>upSET</em> is homozygous lethal, but is rescued by the UpSET-BioTAP transgene (compare counts in columns three and four to columns seven and eight). Cross scheme is shown above the table. Table contains the expected ratio of adult progeny per genotype, as well as the observed count and ratio of these genotypes. (C) PCR analysis reveals that homozygous escapers lack the 544 bp wild-type product. The rescued flies produce only the band corresponding to the tagged construct.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>UpSET-BioTAP localizes to TSS of active genes by ChIP-seq</h3><p id="p-35">Our previous attempts to determine the localization of UpSET using the BioTAP-tagged transgene were unsuccessful in cell culture due to poor stability of the tagged protein in chromatin preparations (data not shown). We reasoned that more of the tagged protein would be incorporated into chromatin in the <em><a href="http://flybase.org/reports/FBgn0036398.html">upSET</a></em>-deleted background, and so we prepared chromatin for ChIP from 0.1 g of mixed 12–24 hr embryos carrying the UpSET-BioTAP transgene in the homozygous <em>DsRed<sup>+</sup>{</em>Δ<em>upSET}</em> background. We immunoprecipated UpSET-BioTAP using the protein-A moiety, and sequenced the resulting material.</p><p id="p-36">In agreement with previous localization data generated by UpSET Dam-ID in Kc cells (<a id="xref-ref-42-6" class="xref-bibr" href="#ref-42">Rincon-Arano <em>et al.</em> 2012</a>), we observed UpSET-BioTAP to localize to active genes by ChIP-seq (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4, A and B</a>). More specifically, in the BioTAP data, there is enrichment for UpSET-BioTAP ChIP peaks in regions carrying the chromatin signatures of TSS proximal and active elongation states (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Ho <em>et al.</em> 2014</a>) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037788/-/DC1/FigureS3.pptx">Figure S3</a>). This is further supported by comparing the overlap of UpSET-BioTAP ChIP peaks with different genome feature annotations. UpSET-BioTAP peak regions display a &gt;2-fold enrichment over the genomic background for promoter and 5′ UTR regions, and are also enriched for coding exons (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4C</a>). Conversely, intron and intergenic regions are depleted from UpSET-BioTAP peaks when compared to the whole genome. When comparing the UpSET-BioTAP dataset to those publically available in the modENCODE project, we observe the highest correlation with Pol II-datasets (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037788/-/DC1/FigureS4.pptx">Figure S4</a>), consistent with enrichment at the TSS of genes in the active chromatin context.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/625/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="UpSET-BioTAP localizes to TSSs of active genes in embryos. (A) Two representative genome browser views of 36&#xA0;kb regions of the X and 2L chromosomes displaying UpSET-BioTAP ChIP-seq tracks, the corresponding peak call used for downstream analysis, gene annotations, and gene orientations. There are no apparent differences between UpSET-BioTAP on the X chromosome and binding on the autosomes. The y-axes for these ChIP-seq tracks are on a log2-scale. (B) Metagene profile for UpSET-BioTAP binding. TSS, transcription start site; TES, 3&#x2032; terminal site. Gene bodies from TSS+500&#xA0;bp to TES&#x2212;500 bp are scaled. The 1&#xA0;kb upstream and downstream, and 500&#xA0;bp into the gene on either end are unscaled. UpSET-BioTAP shows a strong preference for binding at the TSS of active genes. The y-axis for this metagene is a linear scale. (C) UpSET-BioTAP peaks, as highlighted in (A), are enriched for genome regions annotated to be promoters, 5&#x2032;UTR, and coding exons. Intron and intergenic regions are depleted compared to the whole genome background." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-285027449" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;UpSET-BioTAP localizes to TSSs of active genes in embryos. (A) Two representative genome browser views of 36&#xA0;kb regions of the X and 2L chromosomes displaying UpSET-BioTAP ChIP-seq tracks, the corresponding peak call used for downstream analysis, gene annotations, and gene orientations. There are no apparent differences between UpSET-BioTAP on the X chromosome and binding on the autosomes. The &lt;em&gt;y&lt;/em&gt;-axes for these ChIP-seq tracks are on a log2-scale. (B) Metagene profile for UpSET-BioTAP binding. TSS, transcription start site; TES, 3&#x2032; terminal site. Gene bodies from TSS+500&#xA0;bp to TES&#x2212;500 bp are scaled. The 1&#xA0;kb upstream and downstream, and 500&#xA0;bp into the gene on either end are unscaled. UpSET-BioTAP shows a strong preference for binding at the TSS of active genes. The &lt;em&gt;y&lt;/em&gt;-axis for this metagene is a linear scale. (C) UpSET-BioTAP peaks, as highlighted in (A), are enriched for genome regions annotated to be promoters, 5&#x2032;UTR, and coding exons. Intron and intergenic regions are depleted compared to the whole genome background.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/625/F4.medium.gif" width="440" height="279"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/2/625/F4.medium.gif" width="440" height="279"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/625/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/625/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470501" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-37" class="first-child">UpSET-BioTAP localizes to TSSs of active genes in embryos. (A) Two representative genome browser views of 36 kb regions of the X and 2L chromosomes displaying UpSET-BioTAP ChIP-seq tracks, the corresponding peak call used for downstream analysis, gene annotations, and gene orientations. There are no apparent differences between UpSET-BioTAP on the X chromosome and binding on the autosomes. The <em>y</em>-axes for these ChIP-seq tracks are on a log2-scale. (B) Metagene profile for UpSET-BioTAP binding. TSS, transcription start site; TES, 3′ terminal site. Gene bodies from TSS+500 bp to TES−500 bp are scaled. The 1 kb upstream and downstream, and 500 bp into the gene on either end are unscaled. UpSET-BioTAP shows a strong preference for binding at the TSS of active genes. The <em>y</em>-axis for this metagene is a linear scale. (C) UpSET-BioTAP peaks, as highlighted in (A), are enriched for genome regions annotated to be promoters, 5′UTR, and coding exons. Intron and intergenic regions are depleted compared to the whole genome background.</p><div class="sb-div caption-clear"></div></div></div><p id="p-38">We sought to examine whether we could detect an X-specific localization pattern for UpSET-BioTAP as compared to the autosomal pattern, although, in mixed-sex embryos, any X-specific localization signal in males would be dampened. We did not detect any difference between UpSET-BioTAP localization at the TSS, or throughout the gene body, between X-linked and autosomal genes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037788/-/DC1/FigureS5.pptx">Figure S5</a>).</p></div><div id="sec-16" class="subsection"><h3>Heterozygous loss of upSET influences position effect variegation</h3><p id="p-39">To test whether the critical role for UpSET protein may be related to the maintenance of heterochromatin, we tested whether heterozygous loss of <em><a href="http://flybase.org/reports/FBgn0036398.html">upSET</a></em> would influence position effect variegation (PEV) of the <em>w<sup>m4</sup></em> allele seen in the eyes of adult flies. The <em>w<sup>m4</sup></em> allele carries an inversion on the X chromosome placing the <em><a href="http://flybase.org/reports/FBgn0003996.html">white</a></em> locus adjacent to heterochromatin, resulting in the apparent spreading of silencing into the locus and a variegating white-eyed phenotype (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>) (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Elgin and Reuter 2013</a>). Defects in heterochromatin components lead to expression of <em>white<sup>+</sup></em> in a larger fraction of cells, thus larger sectors of red eye pigmentation. Loss-of-function mutants for the core components of heterochromatin score strongly in these assays and are collectively called suppressors of variegation [for example, Su(var)3–9, the primary enzyme responsible for H3K9 dimethylation in pericentric heterochromatin]. We scored the eye sectoring phenotypes of hemizygous <em>w<sup>m4</sup></em> males heterozygous for Δ<em>DsRed{</em>Δ<em>upSET}</em> compared to hemizygous <em>w<sup>m4</sup></em> males wild-type for the third chromosome obtained from a parallel cross. We observed that heterozygous loss of <em><a href="http://flybase.org/reports/FBgn0036398.html">upSET</a></em> results in suppression of variegation, that is, a larger number of flies with a greater extent of red pigmentation (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5, B and C</a>). This effect was also evident in female progeny, though there is a higher incidence of suppressed variegation in the control cross (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037788/-/DC1/FigureS6.pptx">Figure S6</a>). Similarly, we observed suppression of PEV of a HS-lacZ reporter inserted on the Y chromosome (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Lu <em>et al.</em> 1996</a>) in heterozygous <em>DsRed<sup>+</sup>{</em>Δ<em>upSET}</em> larval salivary glands (data not shown). Suppression of PEV of a different <em><a href="http://flybase.org/reports/FBal0191837.html">w<sup>+</sup></a></em> reporter has also been reported (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Rincon-Arano <em>et al.</em> 2013</a>). These <em>in vivo</em> findings, along with our bulk analysis of histone PTMs in S2 cells, support the conclusion that UpSET plays a role in heterochromatin maintenance.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/625/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Heterozygous upSET mutants display a Suppressor of variegation [Su(var)] phenotype. (A) Schematic for the wm4 allele (heterochromatin indicated in gray, euchromatin in blue, white locus in black). The wm4 line has a white variegating phenotype due to juxtaposition of the white locus adjacent to heterochromatin on the X chromosome. Loss of repression of white, resulting in eyes with increased numbers of red pigmented cells, is thought to reflect destabilization of heterochromatin. (B) Representative images of the three classes of eye pigmentation into which flies were sorted. Class A has the highest proportion of red pigmentation, Class B has intermediate levels, and Class C is largely unpigmented. (C) Cross scheme for testing of the effect of heterozygous loss of upSET on position effect variegation. Male flies were scored at 3&#xA0;d posteclosion into the three classes described above. In comparison to control males from a parallel cross, heterozygous loss of upSET leads to an increase in red pigmentation, skewing the population toward suppressed variegation (55.6% Class A, 42.8% B, 1.6% C for heterozygous &#x394;DsRed{&#x394;upSET} vs. 0% Class A, 59.3% B, 40.7% C for control males). Total number of progeny scored is indicated to the right of the pie chart." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-285027449" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Heterozygous &lt;em&gt;upSET&lt;/em&gt; mutants display a Suppressor of variegation [Su(var)] phenotype. (A) Schematic for the &lt;em&gt;w&lt;sup&gt;m4&lt;/sup&gt;&lt;/em&gt; allele (heterochromatin indicated in gray, euchromatin in blue, &lt;em&gt;white&lt;/em&gt; locus in black). The &lt;em&gt;w&lt;sup&gt;m4&lt;/sup&gt;&lt;/em&gt; line has a &lt;em&gt;white&lt;/em&gt; variegating phenotype due to juxtaposition of the &lt;em&gt;white&lt;/em&gt; locus adjacent to heterochromatin on the X chromosome. Loss of repression of &lt;em&gt;white&lt;/em&gt;, resulting in eyes with increased numbers of red pigmented cells, is thought to reflect destabilization of heterochromatin. (B) Representative images of the three classes of eye pigmentation into which flies were sorted. Class A has the highest proportion of red pigmentation, Class B has intermediate levels, and Class C is largely unpigmented. (C) Cross scheme for testing of the effect of heterozygous loss of &lt;em&gt;upSET&lt;/em&gt; on position effect variegation. Male flies were scored at 3&#xA0;d posteclosion into the three classes described above. In comparison to control males from a parallel cross, heterozygous loss of &lt;em&gt;upSET&lt;/em&gt; leads to an increase in red pigmentation, skewing the population toward suppressed variegation (55.6% Class A, 42.8% B, 1.6% C for heterozygous &#x394;&lt;em&gt;DsRed{&lt;/em&gt;&#x394;&lt;em&gt;upSET}&lt;/em&gt; &lt;em&gt;vs.&lt;/em&gt; 0% Class A, 59.3% B, 40.7% C for control males). Total number of progeny scored is indicated to the right of the pie chart.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/625/F5.medium.gif" width="440" height="339"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/2/625/F5.medium.gif" width="440" height="339"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/625/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/625/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470459" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-40" class="first-child">Heterozygous <em>upSET</em> mutants display a Suppressor of variegation [Su(var)] phenotype. (A) Schematic for the <em>w<sup>m4</sup></em> allele (heterochromatin indicated in gray, euchromatin in blue, <em>white</em> locus in black). The <em>w<sup>m4</sup></em> line has a <em>white</em> variegating phenotype due to juxtaposition of the <em>white</em> locus adjacent to heterochromatin on the X chromosome. Loss of repression of <em>white</em>, resulting in eyes with increased numbers of red pigmented cells, is thought to reflect destabilization of heterochromatin. (B) Representative images of the three classes of eye pigmentation into which flies were sorted. Class A has the highest proportion of red pigmentation, Class B has intermediate levels, and Class C is largely unpigmented. (C) Cross scheme for testing of the effect of heterozygous loss of <em>upSET</em> on position effect variegation. Male flies were scored at 3 d posteclosion into the three classes described above. In comparison to control males from a parallel cross, heterozygous loss of <em>upSET</em> leads to an increase in red pigmentation, skewing the population toward suppressed variegation (55.6% Class A, 42.8% B, 1.6% C for heterozygous Δ<em>DsRed{</em>Δ<em>upSET}</em> <em>vs.</em> 0% Class A, 59.3% B, 40.7% C for control males). Total number of progeny scored is indicated to the right of the pie chart.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h3>Changes in transcription correlate with altered chromatin state</h3><p id="p-41">In order to assess whether global transcription might be affected by the altered chromatin states in <em><a href="http://flybase.org/reports/FBgn0036398.html">upSET</a></em> mutant cell lines, we utilized a urea-based method to sequence the nascently transcribed RNA associated with Pol II. We elected to isolate nascent RNA in order to limit our observations to changes in transcription, rather than in the steady state cytosolic pool of mRNA. Indeed, we observed that aberrant transcription occurred in all three <em><a href="http://flybase.org/reports/FBgn0036398.html">upSET</a></em> mutant S2 cell lines in comparison to the parental S2 cell line.</p><p id="p-42">Statistical analysis to identify upregulated or downregulated genes revealed striking heterogeneity in the nascent-RNA-sequencing transcriptional profiles between the three <em><a href="http://flybase.org/reports/FBgn0036398.html">upSET</a></em>-mutant cell lines. Therefore, we broadened our analysis to identify differential trends for groups of genes rather than individual loci, based on their previously determined chromatin environment or genomic location, for example, in heterochromatin <em>vs.</em> euchromatin, or X-linked <em>vs.</em> autosomal. To do so, we counted the numbers of genes falling above and below the no-fold-change (log<sub>2</sub>FC = 0) line per grouping (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6, A–C</a>). When comparing X-linked <em>vs.</em> autosomal genes, there was a weak trend toward upregulation, though this failed to reach statistical significance in all three <em><a href="http://flybase.org/reports/FBgn0036398.html">upSET</a></em> mutant cell lines (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6, D–F</a>, second bar). The most striking trend, which was consistent for all three <em><a href="http://flybase.org/reports/FBgn0036398.html">upSET</a></em> mutant cell lines, was upregulation of heterochromatin genes, as defined by the presence of H3K9me2 over the gene in wild-type S2 cells (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6, D–F</a>, third bar). While the fold change for individual genes generally was not statistically significant, taken as a whole, the distribution of the heterochromatin genes skewed toward increased expression in a statistically significant manner.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/625/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Increased transcription of heterochromatin and X-linked genes in upSET mutant cell lines. (A) Scatter plot showing fold-change for individual genes based on their nascent-RNA levels in the G3 upSET mutant cell line on the y-axis vs. wild type S2 expression level on the x-axis. Autosomal euchromatin genes are highlighted in blue against all other genes. Euchromatin or heterochromatin annotations were based on H3K9me2 levels in S2 cells. (B) As in (A), except with X chromosome euchromatin genes highlighted. (C) As in (A and B), except with heterochromatin genes highlighted. (D) Summary of (A&#x2013;C) with number of genes falling above and below the zero-fold change line from the G3 upSET mutant cell line. The general trends of increased expression on the X chromosome and of heterochromatin genes are statistically significant (P&#xA0;" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-285027449" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Increased transcription of heterochromatin and X-linked genes in &lt;em&gt;upSET&lt;/em&gt; mutant cell lines. (A) Scatter plot showing fold-change for individual genes based on their nascent-RNA levels in the G3 &lt;em&gt;upSET&lt;/em&gt; mutant cell line on the &lt;em&gt;y&lt;/em&gt;-axis &lt;em&gt;vs.&lt;/em&gt; wild type S2 expression level on the &lt;em&gt;x&lt;/em&gt;-axis. Autosomal euchromatin genes are highlighted in blue against all other genes. Euchromatin or heterochromatin annotations were based on H3K9me2 levels in S2 cells. (B) As in (A), except with X chromosome euchromatin genes highlighted. (C) As in (A and B), except with heterochromatin genes highlighted. (D) Summary of (A&#x2013;C) with number of genes falling above and below the zero-fold change line from the G3 &lt;em&gt;upSET&lt;/em&gt; mutant cell line. The general trends of increased expression on the X chromosome and of heterochromatin genes are statistically significant (&lt;em&gt;P&lt;/em&gt;&#xA0;&lt;&#xA0;0.001). (E) As in (D), except using data comparing the A7 &lt;em&gt;upSET&lt;/em&gt; mutant cell line to S2 cells. The same general trends are observed as in (D). (F) As in (D and E), except using data comparing the B2 &lt;em&gt;upSET&lt;/em&gt; mutant cell line to S2 cells. The same general trends are observed as in (D and E); however, upregulation of the X chromosome was not observed in the B2 line (&lt;em&gt;P&lt;/em&gt;&#xA0;=&#xA0;0.19).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/625/F6.medium.gif" width="440" height="434"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/7/2/625/F6.medium.gif" width="440" height="434"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/625/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/625/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470395" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-43" class="first-child">Increased transcription of heterochromatin and X-linked genes in <em>upSET</em> mutant cell lines. (A) Scatter plot showing fold-change for individual genes based on their nascent-RNA levels in the G3 <em>upSET</em> mutant cell line on the <em>y</em>-axis <em>vs.</em> wild type S2 expression level on the <em>x</em>-axis. Autosomal euchromatin genes are highlighted in blue against all other genes. Euchromatin or heterochromatin annotations were based on H3K9me2 levels in S2 cells. (B) As in (A), except with X chromosome euchromatin genes highlighted. (C) As in (A and B), except with heterochromatin genes highlighted. (D) Summary of (A–C) with number of genes falling above and below the zero-fold change line from the G3 <em>upSET</em> mutant cell line. The general trends of increased expression on the X chromosome and of heterochromatin genes are statistically significant (<em>P</em> &lt; 0.001). (E) As in (D), except using data comparing the A7 <em>upSET</em> mutant cell line to S2 cells. The same general trends are observed as in (D). (F) As in (D and E), except using data comparing the B2 <em>upSET</em> mutant cell line to S2 cells. The same general trends are observed as in (D and E); however, upregulation of the X chromosome was not observed in the B2 line (<em>P</em> = 0.19).</p><div class="sb-div caption-clear"></div></div></div><p id="p-44">In summary, we found that the loss of UpSET has a consistent effect on heterochromatin composition and function, based on multiple assays in flies and in cell culture. Since it has been proposed that MSL proteins play a role in activation of autosomal heterochromatin genes in males (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Koya and Meller 2015</a>), it is possible that the heterochromatin phenotype that we observe is due, at least in part, to disruption of a specific interaction between UpSET and the MSL complex. However, given the previous study showing derepression of silent genes and repetitive transposable elements in ovaries and female Kc cells (<a id="xref-ref-42-7" class="xref-bibr" href="#ref-42">Rincon-Arano <em>et al.</em> 2012</a>), along with our finding that UpSET is an essential gene, it is likely that UpSET plays a broad role in maintaining the balance between active and silent marks in both sexes.</p></div></div><div class="section discussion" id="sec-18"><h2 class="">Discussion</h2><p id="p-45">Chromatin and gene expression are intimately linked at the molecular level. Proteins that create and maintain chromatin domains therefore are critical for transcriptional fidelity of gene expression programs. Here, we have further investigated one such chromatin protein, the SET-domain containing protein UpSET. Previous characterizations of this protein and its homologs SET3 in yeast, and MLL5 in mammals, have shown that it assembles into a complex with histone deacetylase activity (<a id="xref-ref-39-4" class="xref-bibr" href="#ref-39">Pijnappel <em>et al.</em> 2001</a>). Furthermore, the PHD finger of UpSET has been shown to interact with the histone post-translational modification H3K4me2/3 (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Ali <em>et al.</em> 2013</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Lemak <em>et al.</em> 2013</a>), which results in recruitment of the HDAC complex to TSS-proximal locations. Once there, the HDAC complex restricts the spread of activating marks which prevents the improper activation of neighboring genes (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Kim and Buratowski 2009</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Kim <em>et al.</em> 2012</a>; <a id="xref-ref-42-8" class="xref-bibr" href="#ref-42">Rincon-Arano <em>et al.</em> 2012</a>). Our results are largely compatible with this model, yet suggest that this role may be particularly important for the maintenance of the heterochromatin environment.</p><p id="p-46">The original characterization of UpSET in <em>Drosophila</em> made use of a <em>P</em>-element insertion line, which left the coding region of the gene intact, and was described as homozygous viable with female sterility. Additionally, many of the experiments were performed in female cultured cells and in female tissues (<a id="xref-ref-42-9" class="xref-bibr" href="#ref-42">Rincon-Arano <em>et al.</em> 2012</a>). Interestingly, our laboratory independently discovered UpSET as one of the most enriched proteins in crosslinked MSL3 purifications. This led us to seek whether UpSET plays a role in dosage compensation, a male-specific process in <em>Drosophila</em>.</p><p id="p-47">To our surprise, we found that precise deletion of the <em><a href="http://flybase.org/reports/FBgn0036398.html">upSET</a></em> locus was lethal in both males and females. That we were able to rescue this lethality with a tagged UpSET transgene suggests that it is due specifically to the loss of UpSET. Furthermore, it suggests that the previously utilized <em>P</em>-element allele may be hypomorphic, and still provide enough UpSET protein for viability, but then result in maternal effect lethality as characterized. Using our tagged allele, we determined the genomic localization of UpSET protein in mixed embryos, confirming, while also refining, the previous result for UpSET localization to active genes by Dam-ID (<a id="xref-ref-42-10" class="xref-bibr" href="#ref-42">Rincon-Arano <em>et al.</em> 2012</a>).</p><p id="p-48">In parallel, we created S2 male cell lines that carried <em><a href="http://flybase.org/reports/FBgn0036398.html">upSET</a></em> mutations that introduce frameshifts to the UpSET open reading frame. The cell culture system proved invaluable for assessing the molecular impact of the loss of UpSET. We observed that loss of UpSET had a profound impact on the state of chromatin, which in broad strokes was consistent across all three cell lines. Consistent with a role related to deacetylation, we saw a modest increase in acetylated histone H4 in bulk histones, but the more striking change was that H3K9me2 levels in bulk histones were reduced in all three lines. In addition, our nascent-RNA-sequencing showed increases in transcription of genes embedded in heterochromatin regions in all three lines. These molecular findings were further supported by our analysis of <em><a href="http://flybase.org/reports/FBgn0036398.html">upSET</a></em>-deficient flies when we tested the impact of the heterozygous <em><a href="http://flybase.org/reports/FBgn0036398.html">upSET</a></em> deletion on the position effect variegation phenotype of the <em>w<sup>m4</sup></em> allele. Our findings showed suppressed <em><a href="http://flybase.org/reports/FBgn0003996.html">white</a></em> variegation, suggesting a loss of heterochromatin stability allowing the <em><a href="http://flybase.org/reports/FBgn0003996.html">white</a></em> locus to become expressed more readily.</p><p id="p-49">Interestingly, there has long been an, as yet unexplained, relationship between heterochromatin and the X chromosome in <em>Drosophila</em>. The X chromosome is observed to be less compact in polytene chromosome preparations, and its morphology is particularly sensitive to mutations in heterochromatin components such as Su(var)3–7, Su(var)3–9, and HP1a (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Demakova <em>et al.</em> 2007</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Spierer <em>et al.</em> 2008</a>). Loss of these core heterochromatin factors leads to a swollen X chromosome (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Demakova <em>et al.</em> 2007</a>), whereas their overexpression can lead to enhanced compaction, as compared to changes in autosomes (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Spierer <em>et al.</em> 2008</a>). Furthermore, Jil-1 kinase, which is enriched ∼2-fold on the male X in an MSL-dependent manner, is thought to prevent the spread of heterochromatin by catalyzing the H3S10ph modification (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Jin <em>et al.</em> 1999</a>, <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">2000</a>). Jil-1 has a complex interplay with heterochromatin components (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Ebert <em>et al.</em> 2004</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Deng <em>et al.</em> 2005</a>, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">2007</a>), with evidence for roles in phosphorylation of Su(var)3–9 (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Boeke <em>et al.</em> 2010</a>), and for establishing a composite H3S10phK9me2 epigenetic mark (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Wang <em>et al.</em> 2014</a>). Indeed, using <em>Jil-1</em> mutant larvae, it has been observed that H3K9me2 spreads from pericentric heterochromatin into the euchromatic gene arms, with a marked increase on the X chromosome in both sexes (A. Plachetka, A. Alekseyenko, and M.I.K., modENCODE, unpublished observations). Intriguingly this spread appeared to skip over gene bodies, suggesting additional non-Jil-1 mechanisms exist for protecting genes from the spread of heterochromatin.</p><p id="p-50">The unique function of UpSET in heterochromatin in <em>Drosophila</em> may make sense in terms of evolutionary history, since the mammalian homolog MLL5 has been implicated in establishing proper DNA methylation. Canonical (5-methylcytosine) DNA methylation is a repressed state found in higher eukaryotes, but is found at only very low levels in <em>Drosophila</em> (&lt;1%) and is of uncertain significance in flies (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Capuano <em>et al.</em> 2014</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Takayama <em>et al.</em> 2014</a>). However, the exact molecular role for MLL5 with respect to DNA methylation has not been elucidated (<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Yun <em>et al.</em> 2014</a>). One can posit that this derived trait in mammals is an extension of the repressive role of SET3 in yeast, with UpSET functioning in an intermediate manner. This expanding alternative repressive role may explain how this family of atypical catalytically inactive SET-domain proteins has perdured through evolutionary time.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-51">We are grateful to B. Housden and R. Binari of N. Perrimon’s laboratory and Sonny Nguyen of T. Wu’s laboratory for advice and reagents, R. Tomaino for liquid chromatography-mass spectrometry, S. Terrizzi for fluorescence-activated cell sorting, J. Abed for work adapting nascent-RNA-seq to <em>Drosophila</em> cells, and H. Kang and A. Alekseyenko for generous help in the last phases of this work. We thank members of the M.I.K. and W. Bender laboratories for helpful discussion and critical reading of the manuscript. This work was supported by the National Institutes of Health (NIH) (GM45444 and GM101958 to M.I.K.). K.A.M. was supported by the Joint Training Program in Molecules, Cells, and Organisms (T32GM007598 from the NIH).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-52">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037788/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.037788/-/DC1</a>.</p></li><li class="fn" id="fn-2"><p id="p-53"><em>Communicating editor: H. K. Salz</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-11-23"><span class="received-label">Received </span>November 23, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-12-15"><span class="accepted-label">Accepted </span>December 15, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 McElroy <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.1" data-doi="10.1016/j.cell.2008.06.033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alekseyenko</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larschan</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gorchakov</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">O. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A sequence motif within chromatin entry sites directs MSL establishment on the <em>Drosophila</em> X chromosome.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">134</span>: <span class="cit-fpage">599</span>–<span class="cit-lpage">609</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DAlekseyenko%26rft.auinit1%253DA.%2BA.%26rft.volume%253D134%26rft.issue%253D4%26rft.spage%253D599%26rft.epage%253D609%26rft.atitle%253DA%2Bsequence%2Bmotif%2Bwithin%2Bchromatin%2Bentry%2Bsites%2Bdirects%2BMSL%2Bestablishment%2Bon%2Bthe%2BDrosophila%2BX%2Bchromosome.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2008.06.033%26rft_id%253Dinfo%253Apmid%252F18724933%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2008.06.033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18724933&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F625.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258665800014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.2" data-doi="10.1101/gad.267583.115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alekseyenko</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walsh</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grayson</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hsi</span>  <span class="cit-name-given-names">P. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The oncogenic BRD4-NUT chromatin regulator drives aberrant transcription within large topological domains.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">1507</span>–<span class="cit-lpage">1523</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DAlekseyenko%26rft.auinit1%253DA.%2BA.%26rft.volume%253D29%26rft.issue%253D14%26rft.spage%253D1507%26rft.epage%253D1523%26rft.atitle%253DThe%2Boncogenic%2BBRD4-NUT%2Bchromatin%2Bregulator%2Bdrives%2Baberrant%2Btranscription%2Bwithin%2Blarge%2Btopological%2Bdomains%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.267583.115%26rft_id%253Dinfo%253Apmid%252F26220994%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjI5LzE0LzE1MDciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzIvNjI1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.3" data-doi="10.1073/pnas.1310156110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ali</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rincon-Arano</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rothbart</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tong</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Molecular basis for chromatin binding and regulation of MLL5.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">11296</span>–<span class="cit-lpage">11301</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1310156110%26rft_id%253Dinfo%253Apmid%252F23798402%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEwLzI4LzExMjk2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzYyNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.4" data-doi="10.1016/j.celrep.2013.06.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bassett</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tibbit</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ponting</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Highly efficient targeted mutagenesis of <em>Drosophila</em> with the CRISPR/Cas9 system.</span> <abbr class="cit-jnl-abbrev">Cell Rep.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">220</span>–<span class="cit-lpage">228</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BRep.%26rft.volume%253D4%26rft.spage%253D220%26rft.atitle%253DHIGHLY%2BEFFICIENT%2BTARGETED%2BMUTAGENESIS%2BOF%2BDROSOPHILA%2BWITH%2BTHE%2BCRISPR%252FCAS9%2BSYSTEM.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.celrep.2013.06.020%26rft_id%253Dinfo%253Apmid%252F23827738%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.celrep.2013.06.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23827738&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F625.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000321901900020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.5" data-doi="10.1371/journal.pone.0010042"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boeke</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Regnard</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cai</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johansen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johansen</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Phosphorylation of SU(VAR)3–9 by the chromosomal kinase JIL-1.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e10042</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBoeke%26rft.auinit1%253DJ.%26rft.volume%253D5%26rft.issue%253D4%26rft.spage%253De10042%26rft.epage%253De10042%26rft.atitle%253DPhosphorylation%2Bof%2BSU%2528VAR%25293-9%2Bby%2Bthe%2Bchromosomal%2Bkinase%2BJIL-1.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0010042%26rft_id%253Dinfo%253Apmid%252F20386606%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0010042&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20386606&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F625.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.6" data-doi="10.1016/0092-8674(78)90305-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Candido</span>  <span class="cit-name-given-names">E. P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reeves</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davie</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1978</span> <span class="cit-article-title">Sodium butyrate inhibits histone deacetylation in cultured cells.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">105</span>–<span class="cit-lpage">113</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DCandido%26rft.auinit1%253DE.%2BP.%26rft.volume%253D14%26rft.issue%253D1%26rft.spage%253D105%26rft.epage%253D113%26rft.atitle%253DSodium%2Bbutyrate%2Binhibits%2Bhistone%2Bdeacetylation%2Bin%2Bcultured%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252878%252990305-7%26rft_id%253Dinfo%253Apmid%252F667927%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(78)90305-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=667927&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F625.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1978FA32400011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.7" data-doi="10.1021/ac500447w"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Capuano</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mulleder</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kok</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blom</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ralser</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Cytosine DNA methylation is found in <em>Drosophila melanogaster</em> but absent in <em>Saccharomyces cerevisiae</em>, <em>Schizosaccharomyces pombe</em>, and other yeast species.</span> <abbr class="cit-jnl-abbrev">Anal. Chem.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">3697</span>–<span class="cit-lpage">3702</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnal.%2BChem.%26rft.volume%253D86%26rft.spage%253D3697%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fac500447w%26rft_id%253Dinfo%253Apmid%252F24640988%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/ac500447w&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24640988&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F625.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.8" data-doi="10.1016/j.biocel.2008.04.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chew</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">RNA interference against mixed lineage leukemia 5 resulted in cell cycle arrest.</span> <abbr class="cit-jnl-abbrev">Int. J. Biochem. Cell Biol.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">2472</span>–<span class="cit-lpage">2481</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2Binternational%2Bjournal%2Bof%2Bbiochemistry%2B%2526%2Bcell%2Bbiology%26rft.stitle%253DInt%2BJ%2BBiochem%2BCell%2BBiol%26rft.aulast%253DCheng%26rft.auinit1%253DF.%26rft.volume%253D40%26rft.issue%253D11%26rft.spage%253D2472%26rft.epage%253D2481%26rft.atitle%253DRNA%2Binterference%2Bagainst%2Bmixed%2Blineage%2Bleukemia%2B5%2Bresulted%2Bin%2Bcell%2Bcycle%2Barrest.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.biocel.2008.04.012%26rft_id%253Dinfo%253Apmid%252F18573682%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.biocel.2008.04.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18573682&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F625.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259134100018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.9" data-doi="10.1534/genetics.106.062133"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Demakova</span>  <span class="cit-name-given-names">O. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pokholkova</span>  <span class="cit-name-given-names">G. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kolesnikova</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Demakov</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andreyeva</span>  <span class="cit-name-given-names">E. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The SU(VAR)3–9/HP1 complex differentially regulates the compaction state and degree of underreplication of X chromosome pericentric heterochromatin in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">175</span>: <span class="cit-fpage">609</span>–<span class="cit-lpage">620</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.062133%26rft_id%253Dinfo%253Apmid%252F17151257%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc1LzIvNjA5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzYyNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.10" data-doi="10.1007/s00412-005-0006-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Deng</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bao</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Girton</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The JIL-1 kinase regulates the structure of <em>Drosophila</em> polytene chromosomes.</span> <abbr class="cit-jnl-abbrev">Chromosoma</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">173</span>–<span class="cit-lpage">182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChromosoma%26rft.stitle%253DChromosoma%26rft.aulast%253DDeng%26rft.auinit1%253DH.%26rft.volume%253D114%26rft.issue%253D3%26rft.spage%253D173%26rft.epage%253D182%26rft.atitle%253DThe%2BJIL-1%2Bkinase%2Bregulates%2Bthe%2Bstructure%2Bof%2BDrosophila%2Bpolytene%2Bchromosomes.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00412-005-0006-8%26rft_id%253Dinfo%253Apmid%252F15986206%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00412-005-0006-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15986206&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F625.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231635600005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.11" data-doi="10.1534/genetics.107.075143"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Deng</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bao</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Girton</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johansen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Reduced levels of Su(var)3–9 but not Su(var)2–5 (HP1) counteract the effects on chromatin structure and viability in loss-of-function mutants of the JIL-1 histone H3S10 kinase.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">79</span>–<span class="cit-lpage">87</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.075143%26rft_id%253Dinfo%253Apmid%252F17660558%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTc3LzEvNzkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzIvNjI1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.12" data-doi="10.1073/pnas.2036345100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Deng</span>  <span class="cit-name-given-names">L. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chiu</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strominger</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">MLL 5 protein forms intranuclear foci, and overexpression inhibits cell cycle progression.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">757</span>–<span class="cit-lpage">762</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.2036345100%26rft_id%253Dinfo%253Apmid%252F14718661%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIxMDEvMy83NTciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzIvNjI1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.13" data-doi="10.1101/gad.323004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ebert</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schotta</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lein</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kubicek</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krauss</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Su(var) genes regulate the balance between euchromatin and heterochromatin in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">2973</span>–<span class="cit-lpage">2983</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DEbert%26rft.auinit1%253DA.%26rft.volume%253D18%26rft.issue%253D23%26rft.spage%253D2973%26rft.epage%253D2983%26rft.atitle%253DSu%2528var%2529%2Bgenes%2Bregulate%2Bthe%2Bbalance%2Bbetween%2Beuchromatin%2Band%2Bheterochromatin%2Bin%2BDrosophila%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.323004%26rft_id%253Dinfo%253Apmid%252F15574598%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjE4LzIzLzI5NzMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzIvNjI1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.14" data-doi="10.1101/cshperspect.a017780"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elgin</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reuter</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Position-effect variegation, heterochromatin formation, and gene silencing in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Cold Spring Harb. Perspect. Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">a017780</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCold%2BSpring%2BHarbor%2BPerspectives%2Bin%2BBiology%26rft.stitle%253DCold%2BSpring%2BHarb.%2BPerspect.%2BBiol.%26rft.aulast%253DElgin%26rft.auinit1%253DS.%2BC.%2BR.%26rft.volume%253D5%26rft.issue%253D8%26rft.spage%253Da017780%26rft.epage%253Da017780%26rft.atitle%253DPosition-Effect%2BVariegation%252C%2BHeterochromatin%2BFormation%252C%2Band%2BGene%2BSilencing%2Bin%2BDrosophila%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fcshperspect.a017780%26rft_id%253Dinfo%253Apmid%252F23906716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6ImNzaHBlcnNwZWN0IjtzOjU6InJlc2lkIjtzOjExOiI1LzgvYTAxNzc4MCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi82MjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.15" data-doi="10.1038/sj.onc.1205615"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Emerling</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonifas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kratz</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donovan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title"><em>MLL5</em>, a homolog of <em>Drosophila trithorax</em> located within a segment of chromosome band 7q22 implicated in myeloid leukemia.</span> <abbr class="cit-jnl-abbrev">Oncogene</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">4849</span>–<span class="cit-lpage">4854</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOncogene%26rft.stitle%253DOncogene%26rft.aulast%253DEmerling%26rft.auinit1%253DB.%2BM.%26rft.volume%253D21%26rft.issue%253D31%26rft.spage%253D4849%26rft.epage%253D4854%26rft.atitle%253DMLL5%252C%2Ba%2Bhomolog%2Bof%2BDrosophila%2Btrithorax%2Blocated%2Bwithin%2Ba%2Bsegment%2Bof%2Bchromosome%2Bband%2B7q22%2Bimplicated%2Bin%2Bmyeloid%2Bleukemia.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.onc.1205615%26rft_id%253Dinfo%253Apmid%252F12101424%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.onc.1205615&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12101424&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F625.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000176716300015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.16" data-doi="10.1016/j.celrep.2013.09.037"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ferrari</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plachetka</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alekseyenko</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jung</span>  <span class="cit-name-given-names">Y. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ozsolak</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">“Jump start and gain” model for dosage compensation in Drosophila based on direct sequencing of nascent transcripts.</span> <abbr class="cit-jnl-abbrev">Cell Rep</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">629</span>–<span class="cit-lpage">636</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BRep%26rft.volume%253D5%26rft.spage%253D629%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.celrep.2013.09.037%26rft_id%253Dinfo%253Apmid%252F24183666%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.celrep.2013.09.037&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24183666&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F625.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.17" data-doi="10.1038/nature13424"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gerstein</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rozowsky</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">K. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Comparative analysis of the transcriptome across distant species.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">512</span>: <span class="cit-fpage">445</span>–<span class="cit-lpage">448</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D512%26rft.spage%253D445%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature13424%26rft_id%253Dinfo%253Apmid%252F25164755%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature13424&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25164755&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F625.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340840600037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.18" data-doi="10.1534/genetics.113.152710"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gratz</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cummings</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamm</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donohue</span>  <span class="cit-name-given-names">L. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome engineering of <em>Drosophila</em> with the CRISPR RNA-guided Cas9 nuclease.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">1029</span>–<span class="cit-lpage">1035</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.152710%26rft_id%253Dinfo%253Apmid%252F23709638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5NC80LzEwMjkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzIvNjI1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.19" data-doi="10.1534/genetics.113.160713"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gratz</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ukken</span>  <span class="cit-name-given-names">F. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rubinstein</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thiede</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donohue</span>  <span class="cit-name-given-names">L. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Highly specific and efficient CRISPR/Cas9-catalyzed homology-directed repair in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">196</span>: <span class="cit-fpage">961</span>–<span class="cit-lpage">971</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.160713%26rft_id%253Dinfo%253Apmid%252F24478335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk2LzQvOTYxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzYyNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.20" data-doi="10.1182/blood-2008-06-162263"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heuser</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yap</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leung</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Algara</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tafech</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Loss of MLL5 results in pleiotropic hematopoietic defects, reduced neutrophil immune function, and extreme sensitivity to DNA demethylation.</span> <abbr class="cit-jnl-abbrev">Blood</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">1432</span>–<span class="cit-lpage">1443</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBlood%26rft_id%253Dinfo%253Adoi%252F10.1182%252Fblood-2008-06-162263%26rft_id%253Dinfo%253Apmid%252F18854576%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6ImJsb29kam91cm5hbCI7czo1OiJyZXNpZCI7czoxMDoiMTEzLzcvMTQzMiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi82MjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.21" data-doi="10.1038/nature13415"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ho</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jung</span>  <span class="cit-name-given-names">Y. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alver</span>  <span class="cit-name-given-names">B. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Comparative analysis of metazoan chromatin organization.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">512</span>: <span class="cit-fpage">449</span>–<span class="cit-lpage">452</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D512%26rft.spage%253D449%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature13415%26rft_id%253Dinfo%253Apmid%252F25164756%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature13415&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25164756&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F625.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340840600038&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.22" data-doi="10.1016/B978-0-12-801185-0.00019-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Housden</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perrimon</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Cas9-based genome editing in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Methods Enzymol.</abbr> <span class="cit-vol">546</span>: <span class="cit-fpage">415</span>–<span class="cit-lpage">439</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BEnzymol.%26rft.volume%253D546%26rft.spage%253D415%26rft_id%253Dinfo%253Adoi%252F10.1016%252FB978-0-12-801185-0.00019-2%26rft_id%253Dinfo%253Apmid%252F25398351%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/B978-0-12-801185-0.00019-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25398351&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F625.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.23" data-doi="10.1126/scisignal.aab3729"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Housden</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valvezan</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelley</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sopko</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Identification of potential drug targets for tuberous sclerosis complex by synthetic screens combining CRISPR-based knockouts with RNAi.</span> <abbr class="cit-jnl-abbrev">Sci. Signal.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">rs9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2BSignaling%26rft.stitle%253DSci%2BSignal%26rft.aulast%253DHousden%26rft.auinit1%253DB.%2BE.%26rft.volume%253D8%26rft.issue%253D393%26rft.spage%253Drs9%26rft.epage%253Drs9%26rft.atitle%253DIdentification%2Bof%2Bpotential%2Bdrug%2Btargets%2Bfor%2Btuberous%2Bsclerosis%2Bcomplex%2Bby%2Bsynthetic%2Bscreens%2Bcombining%2BCRISPR-based%2Bknockouts%2Bwith%2BRNAi%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscisignal.aab3729%26rft_id%253Dinfo%253Apmid%252F26350902%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoic2lndHJhbnMiO3M6NToicmVzaWQiO3M6OToiOC8zOTMvcnM5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzYyNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.24" data-doi="10.1126/science.1069473"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jacobs</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khorasanizadeh</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Structure of HP1 chromodomain bound to a lysine 9-methylated histone H3 tail.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">295</span>: <span class="cit-fpage">2080</span>–<span class="cit-lpage">2083</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1069473%26rft_id%253Dinfo%253Apmid%252F11859155%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTUvNTU2Mi8yMDgwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzYyNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.25" data-doi="10.1016/S1097-2765(00)80195-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walker</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conley</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">JIL-1: a novel chromosomal tandem kinase implicated in transcriptional regulation in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">129</span>–<span class="cit-lpage">135</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DJin%26rft.auinit1%253DY.%26rft.volume%253D4%26rft.issue%253D1%26rft.spage%253D129%26rft.epage%253D135%26rft.atitle%253DJIL-1%253A%2Ba%2Bnovel%2Bchromosomal%2Btandem%2Bkinase%2Bimplicated%2Bin%2Btranscriptional%2Bregulation%2Bin%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1097-2765%252800%252980195-1%26rft_id%253Dinfo%253Apmid%252F10445035%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1097-2765(00)80195-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10445035&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F625.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000081669300014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.26" data-doi="10.1083/jcb.149.5.1005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johansen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johansen</span>  <span class="cit-name-given-names">K. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">JIL-1, a chromosomal kinase implicated in regulation of chromatin structure, associates with the male specific lethal (MSL) dosage compensation complex.</span> <abbr class="cit-jnl-abbrev">J. Cell Biol.</abbr> <span class="cit-vol">149</span>: <span class="cit-fpage">1005</span>–<span class="cit-lpage">1010</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCell%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.149.5.1005%26rft_id%253Dinfo%253Apmid%252F10831604%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjEwOiIxNDkvNS8xMDA1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzYyNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.27" data-doi="10.1038/nbt.1508"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kharchenko</span>  <span class="cit-name-given-names">P. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tolstorukov</span>  <span class="cit-name-given-names">M. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Design and analysis of ChIP-seq experiments for DNA-binding proteins.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">1351</span>–<span class="cit-lpage">1359</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DKharchenko%26rft.auinit1%253DP.%2BV.%26rft.volume%253D26%26rft.issue%253D12%26rft.spage%253D1351%26rft.epage%253D1359%26rft.atitle%253DDesign%2Band%2Banalysis%2Bof%2BChIP-seq%2Bexperiments%2Bfor%2BDNA-binding%2Bproteins.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1508%26rft_id%253Dinfo%253Apmid%252F19029915%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1508&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19029915&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F625.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261591300019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.28" data-doi="10.1038/nature09725"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kharchenko</span>  <span class="cit-name-given-names">P. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alekseyenko</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwartz</span>  <span class="cit-name-given-names">Y. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Minoda</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riddle</span>  <span class="cit-name-given-names">N. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Comprehensive analysis of the chromatin landscape in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">471</span>: <span class="cit-fpage">480</span>–<span class="cit-lpage">485</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKharchenko%26rft.auinit1%253DP.%2BV.%26rft.volume%253D471%26rft.issue%253D7339%26rft.spage%253D480%26rft.epage%253D485%26rft.atitle%253DComprehensive%2Banalysis%2Bof%2Bthe%2Bchromatin%2Blandscape%2Bin%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09725%26rft_id%253Dinfo%253Apmid%252F21179089%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09725&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21179089&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F625.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000288702200055&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.29" data-doi="10.1016/j.cell.2009.02.045"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buratowski</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Dimethylation of H3K4 by Set1 recruits the Set3 histone deacetylase complex to 5′ transcribed regions.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">137</span>: <span class="cit-fpage">259</span>–<span class="cit-lpage">272</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DKim%26rft.auinit1%253DT.%26rft.volume%253D137%26rft.issue%253D2%26rft.spage%253D259%26rft.epage%253D272%26rft.atitle%253DDimethylation%2Bof%2BH3K4%2Bby%2BSet1%2Brecruits%2Bthe%2BSet3%2Bhistone%2Bdeacetylase%2Bcomplex%2Bto%2B5%2527%2Btranscribed%2Bregions.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2009.02.045%26rft_id%253Dinfo%253Apmid%252F19379692%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2009.02.045&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19379692&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F625.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265456800019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.30" data-doi="10.1016/j.cell.2012.08.016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clauder-Munster</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steinmetz</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buratowski</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Set3 HDAC mediates effects of overlapping noncoding transcription on gene induction kinetics.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">150</span>: <span class="cit-fpage">1158</span>–<span class="cit-lpage">1169</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DKim%26rft.auinit1%253DT.%26rft.volume%253D150%26rft.issue%253D6%26rft.spage%253D1158%26rft.epage%253D1169%26rft.atitle%253DSet3%2BHDAC%2Bmediates%2Beffects%2Bof%2Boverlapping%2Bnoncoding%2Btranscription%2Bon%2Bgene%2Binduction%2Bkinetics.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2012.08.016%26rft_id%253Dinfo%253Apmid%252F22959268%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2012.08.016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22959268&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F625.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000308928100010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.31" data-doi="10.1371/journal.pone.0140259"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koya</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meller</span>  <span class="cit-name-given-names">V. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Modulation of heterochromatin by male specific lethal proteins and roX RNA in <em>Drosophila melanogaster</em> males.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e0140259</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0140259%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0140259%26rft_id%253Dinfo%253Apmid%252F26468879%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0140259&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26468879&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F625.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.32" data-doi="10.1186/gb-2009-10-3-r25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pop</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Ultrafast and memory-efficient alignment of short DNA sequences to the human genome.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">R25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D10%26rft.issue%253D3%26rft.spage%253DR25%26rft.epage%253DR25%26rft.atitle%253DUltrafast%2Band%2Bmemory-efficient%2Balignment%2Bof%2Bshort%2BDNA%2Bsequences%2Bto%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2009-10-3-r25%26rft_id%253Dinfo%253Apmid%252F19261174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2009-10-3-r25&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19261174&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F625.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.33" data-doi="10.1371/journal.pone.0077020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lemak</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yee</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yap</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Solution NMR structure and histone binding of the PHD domain of human MLL5.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e77020</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De77020%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0077020%26rft_id%253Dinfo%253Apmid%252F24130829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0077020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24130829&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F625.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">B. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bishop</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eissenberg</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Developmental timing and tissue specificity of heterochromatin-mediated silencing.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1323</span>–<span class="cit-lpage">1332</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BJ.%26rft.volume%253D15%26rft.spage%253D1323%26rft_id%253Dinfo%253Apmid%252F8635465%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=8635465&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F625.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.35" data-doi="10.1101/cshperspect.a019398"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lucchesi</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuroda</span>  <span class="cit-name-given-names">M. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Dosage compensation in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Cold Spring Harb. Perspect. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">a019398</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCold%2BSpring%2BHarbor%2BPerspectives%2Bin%2BBiology%26rft.stitle%253DCold%2BSpring%2BHarb.%2BPerspect.%2BBiol.%26rft.aulast%253DLucchesi%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253Da019398%26rft.epage%253Da019398%26rft.atitle%253DDosage%2BCompensation%2Bin%2BDrosophila%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fcshperspect.a019398%26rft_id%253Dinfo%253Apmid%252F25934013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6ImNzaHBlcnNwZWN0IjtzOjU6InJlc2lkIjtzOjExOiI3LzUvYTAxOTM5OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi82MjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.36" data-doi="10.1182/blood-2008-02-142638"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Madan</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madan</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brykczynska</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zilbermann</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hogeveen</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Impaired function of primitive hematopoietic cells in mice lacking the Mixed-Lineage-Leukemia homolog MLL5.</span> <abbr class="cit-jnl-abbrev">Blood</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">1444</span>–<span class="cit-lpage">1454</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBlood%26rft_id%253Dinfo%253Adoi%252F10.1182%252Fblood-2008-02-142638%26rft_id%253Dinfo%253Apmid%252F18952892%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6ImJsb29kam91cm5hbCI7czo1OiJyZXNpZCI7czoxMDoiMTEzLzcvMTQ0NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi82MjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Cutadapt removes adapter sequences from high-throughput sequencing reads.</span> <abbr class="cit-jnl-abbrev">EMBnet.journal</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">10</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBnet.journal%26rft.volume%253D17%26rft.spage%253D10%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">D. F. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holtzman</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaufman</span>  <span class="cit-name-given-names">T. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Customized microinjection glass capillary needles for P-element transformations in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Biotechniques</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">366</span>–<span class="cit-lpage">375</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiotechniques%26rft.volume%253D33%26rft.spage%253D366%26rft_id%253Dinfo%253Apmid%252F12188189%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=12188189&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F625.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177416600019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.39" data-doi="10.1101/gad.207401"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pijnappel</span>  <span class="cit-name-given-names">W. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaft</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roguev</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shevchenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tekotte</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The <em>S. cerevisiae</em> SET3 complex includes two histone deacetylases, Hos2 and Hst1, and is a meiotic-specific repressor of the sporulation gene program.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">2991</span>–<span class="cit-lpage">3004</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DPijnappel%26rft.auinit1%253DW.W.M.%2BP.%26rft.volume%253D15%26rft.issue%253D22%26rft.spage%253D2991%26rft.epage%253D3004%26rft.atitle%253DThe%2BS.%2Bcerevisiae%2BSET3%2Bcomplex%2Bincludes%2Btwo%2Bhistone%2Bdeacetylases%252C%2BHos2%2Band%2BHst1%252C%2Band%2Bis%2Ba%2Bmeiotic-specific%2Brepressor%2Bof%2Bthe%2Bsporulation%2Bgene%2Bprogram%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.207401%26rft_id%253Dinfo%253Apmid%252F11711434%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjE1LzIyLzI5OTEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzIvNjI1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Platero</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartnett</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eissenberg</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Functional analysis of the chromo domain of HP1.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">3977</span>–<span class="cit-lpage">3986</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BJ.%26rft.volume%253D14%26rft.spage%253D3977%26rft_id%253Dinfo%253Apmid%252F7664737%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=7664737&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F625.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995RQ66900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.41" data-doi="10.1101/gr.110098.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Riddle</span>  <span class="cit-name-given-names">N. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Minoda</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kharchenko</span>  <span class="cit-name-given-names">P. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alekseyenko</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwartz</span>  <span class="cit-name-given-names">Y. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Plasticity in patterns of histone modifications and chromosomal proteins in <em>Drosophila</em> heterochromatin.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">147</span>–<span class="cit-lpage">163</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.110098.110%26rft_id%253Dinfo%253Apmid%252F21177972%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIxLzIvMTQ3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzYyNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.42" data-doi="10.1016/j.cell.2012.11.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rincon-Arano</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Halow</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delrow</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parkhurst</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Groudine</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">UpSET recruits HDAC complexes and restricts chromatin accessibility and acetylation at promoter regions.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">151</span>: <span class="cit-fpage">1214</span>–<span class="cit-lpage">1228</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D151%26rft.spage%253D1214%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2012.11.009%26rft_id%253Dinfo%253Apmid%252F23177352%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2012.11.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23177352&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F625.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000311999900012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rincon-Arano</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parkhurst</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Groudine</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">UpSET-ing the balance.</span> <abbr class="cit-jnl-abbrev">Fly (Austin)</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFly%2B%2528Austin%2529%26rft.volume%253D7%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.44" data-doi="10.1093/bioinformatics/btp616"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCarthy</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smyth</span>  <span class="cit-name-given-names">G. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">edgeR: a bioconductor package for differential expression analysis of digital gene expression data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">139</span>–<span class="cit-lpage">140</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp616%26rft_id%253Dinfo%253Apmid%252F19910308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNi8xLzEzOSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi82MjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.45" data-doi="10.1038/nature10730"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ross-Innes</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stark</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Teschendorff</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holmes</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ali</span>  <span class="cit-name-given-names">H. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Differential oestrogen receptor binding is associated with clinical outcome in breast cancer.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">481</span>: <span class="cit-fpage">389</span>–<span class="cit-lpage">393</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DRoss-Innes%26rft.auinit1%253DC.%2BS.%26rft.volume%253D481%26rft.issue%253D7381%26rft.spage%253D389%26rft.epage%253D393%26rft.atitle%253DDifferential%2Boestrogen%2Breceptor%2Bbinding%2Bis%2Bassociated%2Bwith%2Bclinical%2Boutcome%2Bin%2Bbreast%2Bcancer.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10730%26rft_id%253Dinfo%253Apmid%252F22217937%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10730&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22217937&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F625.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000299210600048&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.46" data-doi="10.1093/bioinformatics/btp479"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shin</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manrai</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">X. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">CEAS: <em>cis</em>-regulatory element annotation system.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2605</span>–<span class="cit-lpage">2606</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp479%26rft_id%253Dinfo%253Apmid%252F19689956%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTkvMjYwNSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi82MjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.47" data-doi="10.1371/journal.pgen.1000066"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spierer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Begeot</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spierer</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delattre</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">SU(VAR)3–7 links heterochromatin and dosage compensation in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000066</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DSpierer%26rft.auinit1%253DA.%26rft.volume%253D4%26rft.issue%253D5%26rft.spage%253De1000066%26rft.epage%253De1000066%26rft.atitle%253DSU%2528VAR%25293-7%2Blinks%2Bheterochromatin%2Band%2Bdosage%2Bcompensation%2Bin%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000066%26rft_id%253Dinfo%253Apmid%252F18451980%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000066&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18451980&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F625.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.48" data-doi="10.1101/gr.162412.113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Takayama</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dhahbi</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mao</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heo</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome methylation in <em>D. melanogaster</em> is found at specific short motifs and is independent of DNMT2 activity.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">821</span>–<span class="cit-lpage">830</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.162412.113%26rft_id%253Dinfo%253Apmid%252F24558263%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI0LzUvODIxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzYyNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.49" data-doi="10.1128/MCB.00298-15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Torres-Machorro</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">L. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pillus</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The set3 complex antagonizes the MYST acetyltransferase Esa1 in the DNA damage response.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">3714</span>–<span class="cit-lpage">3725</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.00298-15%26rft_id%253Dinfo%253Apmid%252F26303527%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIzNS8yMS8zNzE0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzYyNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.50" data-doi="10.1126/science.1077790"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kurdistani</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grunstein</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Requirement of Hos2 histone deacetylase for gene activity in yeast.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">298</span>: <span class="cit-fpage">1412</span>–<span class="cit-lpage">1414</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DWang%26rft.auinit1%253DA.%26rft.volume%253D298%26rft.issue%253D5597%26rft.spage%253D1412%26rft.epage%253D1414%26rft.atitle%253DRequirement%2Bof%2BHos2%2BHistone%2BDeacetylase%2Bfor%2BGene%2BActivity%2Bin%2BYeast%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1077790%26rft_id%253Dinfo%253Apmid%252F12434058%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTgvNTU5Ny8xNDEyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzYyNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cai</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bao</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Girton</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Histone H3S10 phosphorylation by the JIL-1 kinase in pericentric heterochromatin and on the fourth chromosome creates a composite H3S10phK9me2 epigenetic mark.</span> <abbr class="cit-jnl-abbrev">Chromosoma</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">273</span>–<span class="cit-lpage">280</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChromosoma%26rft.volume%253D123%26rft.spage%253D273%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.52" data-doi="10.1038/nsmb.2477"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alekseyenko</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">LeRoy</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elia</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gorchakov</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Chromatin proteins captured by ChIP-mass spectrometry are linked to dosage compensation in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Struct. Mol. Biol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">202</span>–<span class="cit-lpage">209</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BStruct.%2BMol.%2BBiol.%26rft.volume%253D20%26rft.spage%253D202%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb.2477%26rft_id%253Dinfo%253Apmid%252F23295261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsmb.2477&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23295261&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F625.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.53" data-doi="10.3324/haematol.2013.101386"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yun</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Damm</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yap</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwarzer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chaturvedi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Impact of MLL5 expression on decitabine efficacy and DNA methylation in acute myeloid leukemia.</span> <abbr class="cit-jnl-abbrev">Haematologica</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">1456</span>–<span class="cit-lpage">1464</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHaematologica%26rft_id%253Dinfo%253Adoi%252F10.3324%252Fhaematol.2013.101386%26rft_id%253Dinfo%253Apmid%252F24895338%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiaGFlbWF0b2wiO3M6NToicmVzaWQiO3M6OToiOTkvOS8xNDU2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzYyNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.54" data-doi="10.1073/pnas.1522750113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zee</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alekseyenko</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McElroy</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuroda</span>  <span class="cit-name-given-names">M. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span><span class="cit-comment">a</span> <span class="cit-article-title">Streamlined discovery of cross-linked chromatin complexes and associated histone modifications by mass spectrometry.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">1784</span>–<span class="cit-lpage">1789</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1522750113%26rft_id%253Dinfo%253Apmid%252F26831069%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTEzLzcvMTc4NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi82MjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zee</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dibona</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alekseyenko</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">French</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuroda</span>  <span class="cit-name-given-names">M. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span><span class="cit-comment">b</span> <span class="cit-article-title">The oncoprotein BRD4-NUT generates aberrant histone modification patterns.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e0163820</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D11%26rft.spage%253De0163820%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.625.56" data-doi="10.1182/blood-2008-05-159905"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klinger</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tran</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shannon</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">MLL5 contributes to hematopoietic stem cell fitness and homeostasis.</span> <abbr class="cit-jnl-abbrev">Blood</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">1455</span>–<span class="cit-lpage">1463</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBlood%26rft_id%253Dinfo%253Adoi%252F10.1182%252Fblood-2008-05-159905%26rft_id%253Dinfo%253Apmid%252F18818388%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6ImJsb29kam91cm5hbCI7czo1OiJyZXNpZCI7czoxMDoiMTEzLzcvMTQ1NSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi82MjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/2/625.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/2/617" title="Predictability of Genetic Interactions from Functional Gene Modules" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/2/637" title="Comparative High-Density Linkage Mapping Reveals Conserved Genome Structure but Variation in Levels of Heterochiasmy and Location of Recombination Cold Spots in the Common Frog" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/2"><span>Volume 7 Issue 2, February 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/2.cover-source.jpg?itok=m4uMC43I" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/2/625" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-UEoVVAn1XB7Q02bJyPMqHOXbMm0Oi_XoqCmDYYSHsCY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/2/625" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/2/625" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/2/625" class="active" data-icon-position="" data-hide-link-title="0">upSET, the Drosophila homologue of SET3, Is Required for Viability and the Proper Balance of Active and Repressive Chromatin Marks</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460568" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-4gN56jPTwRilkCwfOoAVCvNg4j_1vX5X2vqt96iAjuE" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/2/625.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/2/625" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-_FY1VMElaPW9mfzxh4vqyfusfyVZk8gNLIrcLBPcpWU" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/2/625" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/2/625.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460568" id="node460568--2" data-pisa="ggg;7/2/625" data-pisa-master="ggg;g3.116.037788" data-apath="/ggg/7/2/625.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title"><em>upSET</em>, the <em>Drosophila</em> homologue of SET3, Is Required for Viability and the Proper Balance of Active and Repressive Chromatin Marks</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Kyle A. McElroy</span>, <span class="highwire-citation-author" data-delta="1">Youngsook L. Jung</span>, <span class="highwire-citation-author" data-delta="2">Barry M. Zee</span>, <span class="highwire-citation-author" data-delta="3">Charlotte I. Wang</span>, <span class="highwire-citation-author" data-delta="4">Peter J. Park</span> and <span class="highwire-citation-author" data-delta="5">Mitzi I. Kuroda</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">625-635;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.037788 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460568--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Kyle A. McElroy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Genetics, Brigham and Women’s Hospital, Boston, Massachusetts 02115</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Harvard Medical School, Boston, Massachusetts 02115</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kyle%2BA.%2BMcElroy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McElroy%20KA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKyle%2BA.%2BMcElroy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Youngsook L. Jung </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Genetics, Brigham and Women’s Hospital, Boston, Massachusetts 02115</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Center for Biomedical Informatics, Harvard Medical School, Boston, Massachusetts 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Youngsook%2BL.%2BJung%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jung%20YL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYoungsook%2BL.%2BJung%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Barry M. Zee </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Genetics, Brigham and Women’s Hospital, Boston, Massachusetts 02115</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Harvard Medical School, Boston, Massachusetts 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Barry%2BM.%2BZee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zee%20BM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABarry%2BM.%2BZee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Charlotte I. Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Genetics, Brigham and Women’s Hospital, Boston, Massachusetts 02115</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Harvard Medical School, Boston, Massachusetts 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charlotte%2BI.%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20CI&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACharlotte%2BI.%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Peter J. Park </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Genetics, Brigham and Women’s Hospital, Boston, Massachusetts 02115</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Center for Biomedical Informatics, Harvard Medical School, Boston, Massachusetts 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BJ.%2BPark%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Park%20PJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APeter%2BJ.%2BPark%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Mitzi I. Kuroda </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Genetics, Brigham and Women’s Hospital, Boston, Massachusetts 02115</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Harvard Medical School, Boston, Massachusetts 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mitzi%2BI.%2BKuroda%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kuroda%20MI&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMitzi%2BI.%2BKuroda%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mkuroda@genetics.med.harvard.edu" class="" data-icon-position="" data-hide-link-title="0">mkuroda@genetics.med.harvard.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/2/625" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/2/625&amp;title=upSET%2C%20the%20Drosophila%20homologue%20of%20SET3%2C%20Is%20Required%20for%20Viability%20and%20the%20Proper%20Balance%20of%20Active%20and%20Repressive%20Chromatin%20Marks" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/2/625&amp;title=upSET%2C%20the%20Drosophila%20homologue%20of%20SET3%2C%20Is%20Required%20for%20Viability%20and%20the%20Proper%20Balance%20of%20Active%20and%20Repressive%20Chromatin%20Marks" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/2/625&amp;title=upSET%2C%20the%20Drosophila%20homologue%20of%20SET3%2C%20Is%20Required%20for%20Viability%20and%20the%20Proper%20Balance%20of%20Active%20and%20Repressive%20Chromatin%20Marks" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/2/625&amp;text=upSET%2C%20the%20Drosophila%20homologue%20of%20SET3%2C%20Is%20Required%20for%20Viability%20and%20the%20Proper%20Balance%20of%20Active%20and%20Repressive%20Chromatin%20Marks" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/2/625&amp;title=upSET%2C%20the%20Drosophila%20homologue%20of%20SET3%2C%20Is%20Required%20for%20Viability%20and%20the%20Proper%20Balance%20of%20Active%20and%20Repressive%20Chromatin%20Marks" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/2/625&amp;t=upSET%2C%20the%20Drosophila%20homologue%20of%20SET3%2C%20Is%20Required%20for%20Viability%20and%20the%20Proper%20Balance%20of%20Active%20and%20Repressive%20Chromatin%20Marks" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/2/625&amp;title=upSET%2C%20the%20Drosophila%20homologue%20of%20SET3%2C%20Is%20Required%20for%20Viability%20and%20the%20Proper%20Balance%20of%20Active%20and%20Repressive%20Chromatin%20Marks" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/2/625&amp;title=upSET%2C%20the%20Drosophila%20homologue%20of%20SET3%2C%20Is%20Required%20for%20Viability%20and%20the%20Proper%20Balance%20of%20Active%20and%20Repressive%20Chromatin%20Marks" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460568/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460568" id="node460568--3" data-pisa="ggg;7/2/625" data-pisa-master="ggg;g3.116.037788" data-apath="/ggg/7/2/625.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title"><em>upSET</em>, the <em>Drosophila</em> homologue of SET3, Is Required for Viability and the Proper Balance of Active and Repressive Chromatin Marks</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Kyle A. McElroy</span>, <span class="highwire-citation-author" data-delta="1">Youngsook L. Jung</span>, <span class="highwire-citation-author" data-delta="2">Barry M. Zee</span>, <span class="highwire-citation-author" data-delta="3">Charlotte I. Wang</span>, <span class="highwire-citation-author" data-delta="4">Peter J. Park</span> and <span class="highwire-citation-author" data-delta="5">Mitzi I. Kuroda</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">625-635;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.037788 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460568--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Kyle A. McElroy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Genetics, Brigham and Women’s Hospital, Boston, Massachusetts 02115</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Harvard Medical School, Boston, Massachusetts 02115</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kyle%2BA.%2BMcElroy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McElroy%20KA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKyle%2BA.%2BMcElroy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Youngsook L. Jung </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Genetics, Brigham and Women’s Hospital, Boston, Massachusetts 02115</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Center for Biomedical Informatics, Harvard Medical School, Boston, Massachusetts 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Youngsook%2BL.%2BJung%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jung%20YL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYoungsook%2BL.%2BJung%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Barry M. Zee </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Genetics, Brigham and Women’s Hospital, Boston, Massachusetts 02115</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Harvard Medical School, Boston, Massachusetts 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Barry%2BM.%2BZee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zee%20BM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABarry%2BM.%2BZee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Charlotte I. Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Genetics, Brigham and Women’s Hospital, Boston, Massachusetts 02115</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Harvard Medical School, Boston, Massachusetts 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charlotte%2BI.%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20CI&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACharlotte%2BI.%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Peter J. Park </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Genetics, Brigham and Women’s Hospital, Boston, Massachusetts 02115</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Center for Biomedical Informatics, Harvard Medical School, Boston, Massachusetts 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BJ.%2BPark%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Park%20PJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APeter%2BJ.%2BPark%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Mitzi I. Kuroda </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Genetics, Brigham and Women’s Hospital, Boston, Massachusetts 02115</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Harvard Medical School, Boston, Massachusetts 02115</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mitzi%2BI.%2BKuroda%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kuroda%20MI&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMitzi%2BI.%2BKuroda%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mkuroda@genetics.med.harvard.edu" class="" data-icon-position="" data-hide-link-title="0">mkuroda@genetics.med.harvard.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=625&amp;rft.epage=635&amp;rft.atitle=upSET%2C%20the%20Drosophila%20homologue%20of%20SET3%2C%20Is%20Required%20for%20Viability%20and%20the%20Proper%20Balance%20of%20Active%20and%20Repressive%20Chromatin%20Marks&amp;rft.volume=7&amp;rft.issue=2&amp;rft.date=2017-02-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=McElroy%2C+Kyle+A.&amp;rft.au=Jung%2C+Youngsook+L.&amp;rft.au=Zee%2C+Barry+M.&amp;rft.au=Wang%2C+Charlotte+I.&amp;rft.au=Park%2C+Peter+J.&amp;rft.au=Kuroda%2C+Mitzi+I."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460568/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460568/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460568/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460568/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460568/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460568/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460568/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460568/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460568/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460568/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460568/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460568/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/2/625" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/2/625.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/2/625.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/2/625.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/2/625.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
