<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/2/321" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/2/321.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/2/321.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/2/321.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/2/321/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/2/321.atom" />
<meta name="HW.pisa" content="ggg;7/2/321" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genome Report: Identification and Validation of Antigenic Proteins from Pajaroellobacter abortibovis Using De Novo Genome Sequence Assembly and Reverse Vaccinology" />
<meta name="DC.Identifier" content="10.1534/g3.116.036673" />
<meta name="DC.Date" content="2017-02-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Welly et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Epizootic bovine abortion (EBA), or “foothill abortion,” is the leading cause of beef cattle abortion in California and has also been reported in Nevada and Oregon. In the 1970s, the soft-shelled tick Ornithodoros coriaceus , or “pajaroello tick,” was confirmed as the disease-transmitting vector. In 2005, a novel Deltaproteobacterium was discovered as the etiologic agent of EBA (aoEBA), recently named Pajaroellobacter abortibovis . This organism cannot be grown in culture using traditional microbiological techniques; it can only be grown in experimentally-infected severe combined immunodeficient (SCID) mice. The objectives of this study were to perform a de novo genome assembly for P. abortibovis and identify and validate potential antigenic proteins as candidates for future recombinant vaccine development. DNA and RNA were extracted from spleen tissue collected from experimentally-infected SCID mice following exposure to P. abortibovis . This combination of mouse and bacterial DNA was sequenced and aligned to the mouse genome. Mouse sequences were subtracted from the sequence pool and the remaining sequences were de novo assembled at 50x coverage into a 1.82 Mbp complete closed circular Deltaproteobacterial genome containing 2250 putative protein-coding sequences. Phylogenetic analysis of P. abortibovis predicts that this bacterium is most closely related to the organisms of the order Myxococcales, referred to as Myxobacteria. In silico prediction of vaccine candidates was performed using a reverse vaccinology approach resulting in the identification and ranking of the top 10 candidate proteins that are likely to be antigenic. Immunologic testing of these candidate proteins confirmed antigenicity of seven of the nine expressed protein candidates using serum from P. abortibovis immunized mice." />
<meta name="DC.Contributor" content="Bryan T. Welly" />
<meta name="DC.Contributor" content="Michael R. Miller" />
<meta name="DC.Contributor" content="Jeffrey L. Stott" />
<meta name="DC.Contributor" content="Myra T. Blanchard" />
<meta name="DC.Contributor" content="Alma D. Islas-Trejo" />
<meta name="DC.Contributor" content="Sean M. O’Rourke" />
<meta name="DC.Contributor" content="Amy E. Young" />
<meta name="DC.Contributor" content="Juan F. Medrano" />
<meta name="DC.Contributor" content="Alison L. Van Eenennaam" />
<meta name="article:published_time" content="2017-02-01" />
<meta name="article:section" content="Genome Report" />
<meta name="citation_title" content="Genome Report: Identification and Validation of Antigenic Proteins from Pajaroellobacter abortibovis Using De Novo Genome Sequence Assembly and Reverse Vaccinology" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Epizootic bovine abortion (EBA), or “foothill abortion,” is the leading cause of beef cattle abortion in California and has also been reported in Nevada and Oregon. In the 1970s, the soft-shelled tick &lt;i&gt;Ornithodoros coriaceus&lt;/i&gt;, or “pajaroello tick,” was confirmed as the disease-transmitting vector. In 2005, a novel Deltaproteobacterium was discovered as the etiologic agent of EBA (aoEBA), recently named &lt;i&gt;Pajaroellobacter abortibovis&lt;/i&gt;. This organism cannot be grown in culture using traditional microbiological techniques; it can only be grown in experimentally-infected severe combined immunodeficient (SCID) mice. The objectives of this study were to perform a &lt;i&gt;de novo&lt;/i&gt; genome assembly for &lt;i&gt;P. abortibovis&lt;/i&gt; and identify and validate potential antigenic proteins as candidates for future recombinant vaccine development. DNA and RNA were extracted from spleen tissue collected from experimentally-infected SCID mice following exposure to &lt;i&gt;P. abortibovis&lt;/i&gt;. This combination of mouse and bacterial DNA was sequenced and aligned to the mouse genome. Mouse sequences were subtracted from the sequence pool and the remaining sequences were &lt;i&gt;de novo&lt;/i&gt; assembled at 50x coverage into a 1.82 Mbp complete closed circular Deltaproteobacterial genome containing 2250 putative protein-coding sequences. Phylogenetic analysis of &lt;i&gt;P. abortibovis&lt;/i&gt; predicts that this bacterium is most closely related to the organisms of the order Myxococcales, referred to as Myxobacteria. &lt;i&gt;In silico&lt;/i&gt; prediction of vaccine candidates was performed using a reverse vaccinology approach resulting in the identification and ranking of the top 10 candidate proteins that are likely to be antigenic. Immunologic testing of these candidate proteins confirmed antigenicity of seven of the nine expressed protein candidates using serum from &lt;i&gt;P. abortibovis&lt;/i&gt; immunized mice.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/02/01" />
<meta name="citation_mjid" content="ggg;7/2/321" />
<meta name="citation_id" content="7/2/321" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/2/321" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/2/321.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/2/321.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/2/321.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.036673" />
<meta name="citation_pmid" content="28040777" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="2" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genome Report" />
<meta name="citation_firstpage" content="321" />
<meta name="citation_lastpage" content="331" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Bryan T. Welly" />
<meta name="citation_author_institution" content="Department of Animal Science, University of California, Davis, California 95616" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4282-9285" />
<meta name="citation_author" content="Michael R. Miller" />
<meta name="citation_author_institution" content="Department of Animal Science, University of California, Davis, California 95616" />
<meta name="citation_author" content="Jeffrey L. Stott" />
<meta name="citation_author_institution" content="Department of Pathology, Microbiology and Immunology, School of Veterinary Medicine, University of California, Davis, California 95616" />
<meta name="citation_author" content="Myra T. Blanchard" />
<meta name="citation_author_institution" content="Department of Pathology, Microbiology and Immunology, School of Veterinary Medicine, University of California, Davis, California 95616" />
<meta name="citation_author" content="Alma D. Islas-Trejo" />
<meta name="citation_author_institution" content="Department of Animal Science, University of California, Davis, California 95616" />
<meta name="citation_author" content="Sean M. O’Rourke" />
<meta name="citation_author_institution" content="Department of Animal Science, University of California, Davis, California 95616" />
<meta name="citation_author" content="Amy E. Young" />
<meta name="citation_author_institution" content="Department of Animal Science, University of California, Davis, California 95616" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4181-4233" />
<meta name="citation_author" content="Juan F. Medrano" />
<meta name="citation_author_institution" content="Department of Animal Science, University of California, Davis, California 95616" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7113-3183" />
<meta name="citation_author" content="Alison L. Van Eenennaam" />
<meta name="citation_author_institution" content="Department of Animal Science, University of California, Davis, California 95616" />
<meta name="citation_author_email" content="alvaneenennaam@ucdavis.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1562-162X" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=S. F. Altschul;citation_author=W. Gish;citation_author=W. Miller;citation_author=E. W. Myers;citation_author=D. J. Lipman;citation_title=Basic local alignment search tool.;citation_pages=403-410;citation_volume=215;citation_year=1990;citation_issue=3;citation_pmid=2231712;citation_doi=10.1006/jmbi.1990.9999" />
<meta name="citation_reference" content="citation_journal_title=Journal of Veterinary Diagnostic Investigation;citation_journal_abbrev=Journal of Veterinary Diagnostic Investigation;citation_author=M. L. Anderson;citation_author=P. C. Kennedy;citation_author=M. T. Blanchard;citation_author=L. Barbano;citation_author=P. Chiu;citation_title=Histochemical and Immunohistochemical Evidence of a Bacterium Associated with Lesions of Epizootic Bovine Abortion;citation_pages=76-80;citation_volume=18;citation_year=2006;citation_issue=1;citation_pmid=16566260;citation_doi=10.1177/104063870601800110" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=R. K. Aziz;citation_author=D. Bartels;citation_author=A. A. Best;citation_author=M. DeJongh;citation_author=T. Disz;citation_title=The RAST Server: rapid annotations using subsystems technology.;citation_pages=75-75;citation_volume=9;citation_year=2008;citation_pmid=18261238;citation_doi=10.1186/1471-2164-9-75" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=P. Bakke;citation_author=N. Carney;citation_author=W. DeLoache;citation_author=M. Gearing;citation_author=K. Ingvorsen;citation_title=Evaluation of three automated genome annotations for Halorhabdus utahensis.;citation_pages=e6291-e6291;citation_volume=4;citation_year=2009;citation_issue=7;citation_pmid=19617911;citation_doi=10.1371/journal.pone.0006291" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=J. Besemer;citation_author=A. Lomsadze;citation_author=M. Borodovsky;citation_title=GeneMarkS: a self-training method for prediction of gene starts in microbial genomes. Implications for finding sequence motifs in regulatory regions;citation_pages=2607-2618;citation_volume=29;citation_year=2001;citation_issue=12;citation_pmid=11410670;citation_doi=10.1093/nar/29.12.2607" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=H. R. Bigelow;citation_author=D. S. Petrey;citation_author=J. Liu;citation_author=D. Przybylski;citation_author=B. Rost;citation_title=Predicting transmembrane beta-barrels in proteomes;citation_pages=2566-2577;citation_volume=32;citation_year=2004;citation_issue=8;citation_pmid=15141026;citation_doi=10.1093/nar/gkh580" />
<meta name="citation_reference" content="citation_journal_title=Veterinary microbiology;citation_journal_abbrev=Vet Microbiol;citation_author=M. T. Blanchard;citation_author=C. I. Chen;citation_author=M. Anderson;citation_author=M. R. Hall;citation_author=S. W. Barthold;citation_title=Serial passage of the etiologic agent of epizootic bovine abortion in immunodeficient mice.;citation_pages=177-182;citation_volume=144;citation_year=2010;citation_issue=1-2;citation_pmid=20144513;citation_doi=10.1016/j.vetmic.2010.01.002" />
<meta name="citation_reference" content="citation_journal_title=J. Vet. Diagn. Invest.;citation_author=M. T. Blanchard;citation_author=M. L. Anderson;citation_author=B. R. Hoar;citation_author=A. F. Pires;citation_author=P. C. Blanchard;citation_title=Assessment of a fluorescent antibody test for the detection of antibodies against epizootic bovine abortion.;citation_volume=26;citation_year=2014;citation_pmid=25139792;citation_doi=10.1177/1040638714545506" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. Boetzer;citation_author=C. V. Henkel;citation_author=H. J. Jansen;citation_author=D. Butler;citation_author=W. Pirovano;citation_title=Scaf-folding pre-assembled contigs using SSPACE.;citation_volume=27;citation_year=2011;citation_pmid=21149342;citation_doi=10.1093/bioinformatics/btq683" />
<meta name="citation_reference" content="citation_journal_title=J. Vet. Diagn. Invest.;citation_author=R. S. Brooks;citation_author=M. T. Blanchard;citation_author=M. L. Anderson;citation_author=M. R. Hall;citation_author=J. L. Stott;citation_title=Quantitative duplex TaqMan real-time polymerase chain reaction for the assessment of the etiologic agent of epizootic bovine abortion.;citation_volume=23;citation_year=2011;citation_pmid=22362796;citation_doi=10.1177/1040638711425573" />
<meta name="citation_reference" content="citation_journal_title=Vet. Microbiol.;citation_author=R. S. Brooks;citation_author=M. T. Blanchard;citation_author=K. A. Clothier;citation_author=S. Fish;citation_author=M. L. Anderson;citation_title=Characterization of Pajaroellobacter abortibovis, the etiologic agent of epizootic bovine abortion.;citation_pages=73-80;citation_volume=192;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. L. Delcher;citation_author=K. A. Bratke;citation_author=E. C. Powers;citation_author=S. L. Salzberg;citation_title=Identifying bacterial genes and endosymbiont DNA with Glimmer.;citation_volume=23;citation_year=2007;citation_pmid=17237039;citation_doi=10.1093/bioinformatics/btm009" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=I. A. Doytchinova;citation_author=D. R. Flower;citation_title=VaxiJen: a server for prediction of protective antigens, tumour antigens and subunit vaccines.;citation_pages=4-4;citation_volume=8;citation_year=2007;citation_pmid=17207271;citation_doi=10.1186/1471-2105-8-4" />
<meta name="citation_reference" content="citation_journal_title=Vaccine;citation_journal_abbrev=Vaccine;citation_author=R. W. Ellis;citation_title=New technologies for making vaccines.;citation_pages=1596-1604;citation_volume=17;citation_year=1999;citation_issue=13-14;citation_pmid=10194811;citation_doi=10.1016/S0264-410X(98)00416-2" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=M. M. Giuliani;citation_author=J. Adu-Bobie;citation_author=M. Comanducci;citation_author=B. Aricò;citation_author=S. Savino;citation_title=Inaugural Article: A universal vaccine for serogroup B meningococcus.;citation_pages=10834-10839;citation_volume=103;citation_year=2006;citation_issue=29;citation_pmid=16825336;citation_doi=10.1073/pnas.0603940103" />
<meta name="citation_reference" content="citation_journal_abbrev=J Biomed Biotechnol;citation_author=Y. He;citation_author=Z. Xiang;citation_author=H. L. Mobley;citation_title=Vaxign: the first web-based vaccine design program for reverse vaccinology and applications for vaccine development.;citation_pages=297505-297505;citation_volume=2010;citation_year=2010;citation_pmid=20671958" />
<meta name="citation_reference" content="citation_journal_title=Journal of the American Veterinary Medical Association;citation_journal_abbrev=J Am Vet Med Assoc;citation_author=J. A. Howarth;citation_author=J. E. Moulton;citation_author=L. M. Frazier;citation_title=Epizootic bovine abortion characterized by fetal hepatopathy.;citation_pages=441-449;citation_volume=128;citation_year=1956;citation_issue=9;citation_pmid=13306666" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=D. Hyatt;citation_author=G. L. Chen;citation_author=P. F. Locascio;citation_author=M. L. Land;citation_author=F. W. Larimer;citation_title=Prodigal: prokaryotic gene recognition and translation initiation site identification.;citation_pages=119-119;citation_volume=11;citation_year=2010;citation_pmid=20211023;citation_doi=10.1186/1471-2105-11-119" />
<meta name="citation_reference" content="citation_journal_abbrev=Bioinformation;citation_author=K. Imai;citation_author=N. Asakawa;citation_author=T. Tsuji;citation_author=F. Akazawa;citation_author=A. Ino;citation_title=SOSUI-GramN: high performance prediction for sub-cellular localization of proteins in gram-negative bacteria.;citation_pages=417-421;citation_volume=2;citation_year=2008;citation_issue=9;citation_pmid=18795116;citation_doi=10.6026/97320630002417" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. Kearse;citation_author=R. Moir;citation_author=A. Wilson;citation_author=S. Stones-Havas;citation_author=M. Cheung;citation_title=Geneious Basic: an integrated and extendable desktop software platform for the organization and analysis of sequence data.;citation_volume=28;citation_year=2012;citation_pmid=22543367;citation_doi=10.1093/bioinformatics/bts199" />
<meta name="citation_reference" content="citation_journal_title=American journal of veterinary research;citation_journal_abbrev=Am J Vet Res;citation_author=P. C. Kennedy;citation_author=A. P. Casaro;citation_author=P. B. Kimsey;citation_author=R. H. Bon Durant;citation_author=R. B. Bushnell;citation_title=Epizootic bovine abortion: histogenesis of the fetal lesions.;citation_pages=1040-1048;citation_volume=44;citation_year=1983;citation_issue=6;citation_pmid=6870006" />
<meta name="citation_reference" content="citation_journal_title=Journal of Clinical Microbiology;citation_journal_abbrev=J. Clin. Microbiol.;citation_author=D. P. King;citation_author=C. I. Chen;citation_author=M. T. Blanchard;citation_author=B. M. Aldridge;citation_author=M. Anderson;citation_title=Molecular Identification of a Novel Deltaproteobacterium as the Etiologic Agent of Epizootic Bovine Abortion (Foothill Abortion);citation_pages=604-609;citation_volume=43;citation_year=2005;citation_issue=2;citation_pmid=15695652;citation_doi=10.1128/JCM.43.2.604-609.2005" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=B. Langmead;citation_author=S. L. Salzberg;citation_title=Fast gapped-read alignment with Bowtie 2.;citation_pages=357-359;citation_volume=9;citation_year=2012;citation_issue=4;citation_pmid=22388286;citation_doi=10.1038/nmeth.1923" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in genetics &amp; development;citation_journal_abbrev=Curr Opin Genet Dev;citation_author=J. G. Lawrence;citation_title=Common themes in the genome strategies of pathogens.;citation_pages=584-588;citation_volume=15;citation_year=2005;citation_issue=6;citation_pmid=16188434;citation_doi=10.1016/j.gde.2005.09.007" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=L. Li;citation_author=C. J. Stoeckert;citation_author=D. S. Roos;citation_title=OrthoMCL: Identification of Ortholog Groups for Eukaryotic Genomes;citation_pages=2178-2189;citation_volume=13;citation_year=2003;citation_issue=9;citation_pmid=12952885;citation_doi=10.1101/gr.1224503" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Microbiology;citation_journal_abbrev=Nat Rev Microbiol;citation_author=J. P. McCutcheon;citation_author=N. A. Moran;citation_title=Extreme genome reduction in symbiotic bacteria.;citation_pages=13-26;citation_volume=10;citation_year=2012;citation_issue=1;citation_pmid=22064560;citation_doi=10.1038/nrmicro2670" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=N. A. Moran;citation_title=Microbial minimalism: genome reduction in bacterial pathogens.;citation_pages=583-586;citation_volume=108;citation_year=2002;citation_issue=5;citation_pmid=11893328;citation_doi=10.1016/S0092-8674(02)00665-7" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in genetics &amp; development;citation_journal_abbrev=Curr Opin Genet Dev;citation_author=N. A. Moran;citation_author=G. R. Plague;citation_title=Genomic changes following host restriction in bacteria.;citation_pages=627-633;citation_volume=14;citation_year=2004;citation_issue=6;citation_pmid=15531157;citation_doi=10.1016/j.gde.2004.09.003" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=R. H. Waterston;citation_author=K. Lindblad-Toh;citation_author=E. Birney;citation_author=J. Rogers;citation_title=Initial sequencing and comparative analysis of the mouse genome.;citation_pages=520-562;citation_volume=420;citation_year=2002;citation_issue=6915;citation_pmid=12466850;citation_doi=10.1038/nature01262" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=F. Nadalin;citation_author=F. Vezzi;citation_author=A. Policriti;citation_title=GapFiller: a de novo assembly approach to fill the gap within paired reads.;citation_pages=S8;citation_volume=13;citation_year=2012;citation_pmid=23282012;citation_doi=10.1186/1471-2105-13-S14-S8" />
<meta name="citation_reference" content="citation_journal_title=Front. Microbiol.;citation_author=H. Nishida;citation_title=Evolution of genome base composition and genome size in bacteria.;citation_pages=420;citation_volume=3;citation_year=2012;citation_pmid=23230432" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=R. Overbeek;citation_author=R. Olson;citation_author=G. D. Pusch;citation_author=G. J. Olsen;citation_author=J. J. Davis;citation_title=The SEED and the rapid annotation of microbial genomes using subsystems technology (RAST).;citation_volume=42;citation_year=2014;citation_pmid=24293654;citation_doi=10.1093/nar/gkt1226" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. Pizza;citation_author=V. Scarlato;citation_author=V. Masignani;citation_author=M. M. Giuliani;citation_author=B. Aricò;citation_title=Identification of Vaccine Candidates Against Serogroup B Meningococcus by Whole-Genome Sequencing;citation_pages=1816-1820;citation_volume=287;citation_year=2000;citation_issue=5459;citation_pmid=10710308;citation_doi=10.1126/science.287.5459.1816" />
<meta name="citation_reference" content="citation_journal_title=Vaccine;citation_journal_abbrev=Vaccine;citation_author=R. Rappuoli;citation_title=Reverse vaccinology, a genome-based approach to vaccine development.;citation_pages=2688-2691;citation_volume=19;citation_year=2001;citation_issue=17-19;citation_pmid=11257410;citation_doi=10.1016/S0264-410X(00)00554-5" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=G. Sachdeva;citation_author=K. Kumar;citation_author=P. Jain;citation_author=S. Ramachandran;citation_title=SPAAN: a software program for prediction of adhesins and adhesin-like proteins using neural networks.;citation_volume=21;citation_year=2005;citation_pmid=15374866;citation_doi=10.1093/bioinformatics/bti028" />
<meta name="citation_reference" content="citation_journal_title=Genome Announcements;citation_journal_abbrev=Genome Announc;citation_author=M. Sait;citation_author=K. Aitchison;citation_author=N. Wheelhouse;citation_author=K. Wilson;citation_author=F. A. Lainson;citation_title=Genome Sequence of Lawsonia intracellularis Strain N343, Isolated from a Sow with Hemorrhagic Proliferative Enteropathy;citation_pages=e00027-13-e00027-13;citation_volume=1;citation_year=2013;citation_issue=1;citation_pmid=23472224;citation_doi=10.1128/genomeA.00027-13" />
<meta name="citation_reference" content="citation_journal_title=J. Med. Entomol.;citation_author=E. T. Schmidtmann;citation_author=R. B. Bushnell;citation_author=E. C. Loomis;citation_author=M. N. Oliver;citation_author=J. H. Theis;citation_title=Experimental and epizootiologic evidence associating Ornithodoros coriaceus Koch (Acari-Argasidae) with exposure of cattle to epizootic bovine abortion in California.;citation_volume=13;citation_year=1976;citation_pmid=1011232;citation_doi=10.1093/jmedent/13.3.292" />
<meta name="citation_reference" content="citation_journal_title=FEMS Microbiol. Lett.;citation_author=T. A. Tatusova;citation_author=T. L. Madden;citation_title=BLAST 2 Sequences, a new tool for comparing protein and nucleotide sequences.;citation_volume=174;citation_year=1999;citation_pmid=10339815;citation_doi=10.1111/j.1574-6968.1999.tb13575.x" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=T. Tatusova;citation_author=M. DiCuccio;citation_author=A. Badretdin;citation_author=V. Chetvernin;citation_author=E. P. Nawrocki;citation_title=NCBI prokaryotic genome annotation pipeline.;citation_volume=44;citation_year=2016;citation_pmid=27342282;citation_doi=10.1093/nar/gkw569" />
<meta name="citation_reference" content="citation_journal_title=Veterinary parasitology;citation_journal_abbrev=Vet Parasitol;citation_author=M. B. Teglas;citation_author=N. L. Drazenovich;citation_author=J. Stott;citation_author=J. E. Foley;citation_title=The geographic distribution of the putative agent of epizootic bovine abortion in the tick vector, Ornithodoros coriaceus.;citation_pages=327-333;citation_volume=140;citation_year=2006;citation_issue=3-4;citation_pmid=16672178;citation_doi=10.1016/j.vetpar.2006.03.027" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=C. Trapnell;citation_author=D. G. Hendrickson;citation_author=M. Sauvageau;citation_author=L. Goff;citation_author=J. L. Rinn;citation_title=Differential analysis of gene regulation at transcript resolution with RNA-seq.;citation_pages=46-53;citation_volume=31;citation_year=2013;citation_pmid=23222703;citation_doi=10.1038/nbt.2450" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. Untergasser;citation_author=I. Cutcutache;citation_author=T. Koressaar;citation_author=J. Ye;citation_author=B. C. Faircloth;citation_title=Primer3–new capabilities and interfaces.;citation_volume=40;citation_year=2012;citation_pmid=22730293;citation_doi=10.1093/nar/gks596" />
<meta name="citation_reference" content="citation_journal_title=Clin. Vaccine Immunol.;citation_author=E. Watson;citation_author=E. M. Clark;citation_author=M. P. Alberdi;citation_author=N. F. Inglis;citation_author=M. Porter;citation_title=A novel Lawsonia intracellularis autotransporter protein is a prominent antigen.;citation_volume=18;citation_year=2011;citation_pmid=21697340;citation_doi=10.1128/CVI.05073-11" />
<meta name="citation_reference" content="citation_journal_title=Proteins;citation_journal_abbrev=Proteins;citation_author=C. S. Yu;citation_author=Y. C. Chen;citation_author=C. H. Lu;citation_author=J. K. Hwang;citation_title=Prediction of protein subcellular localization.;citation_pages=643-651;citation_volume=64;citation_year=2006;citation_issue=3;citation_pmid=16752418;citation_doi=10.1002/prot.21018" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=N. Y. Yu;citation_author=J. R. Wagner;citation_author=M. R. Laird;citation_author=G. Melli;citation_author=S. Rey;citation_title=PSORTb 3.0: improved protein subcellular localization prediction with refined localization subcategories and predictive capabilities for all prokaryotes.;citation_volume=26;citation_year=2010;citation_pmid=20472543;citation_doi=10.1093/bioinformatics/btq249" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. R. Zerbino;citation_author=E. Birney;citation_title=Velvet: algorithms for de novo short read assembly using de Bruijn graphs.;citation_volume=18;citation_year=2008;citation_pmid=18349386;citation_doi=10.1101/gr.074492.107" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genome Report: Identification and Validation of Antigenic Proteins from Pajaroellobacter abortibovis Using De Novo Genome Sequence Assembly and Reverse Vaccinology" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2.cover-source.jpg" />
<meta name="twitter:description" content="Epizootic bovine abortion (EBA), or “foothill abortion,” is the leading cause of beef cattle abortion in California and has also been reported in Nevada and Oregon. In the 1970s, the soft-shelled tick Ornithodoros coriaceus , or “pajaroello tick,” was confirmed as the disease-transmitting vector. In 2005, a novel Deltaproteobacterium was discovered as the etiologic agent of EBA (aoEBA), recently named Pajaroellobacter abortibovis . This organism cannot be grown in culture using traditional microbiological techniques; it can only be grown in experimentally-infected severe combined immunodeficient (SCID) mice. The objectives of this study were to perform a de novo genome assembly for P. abortibovis and identify and validate potential antigenic proteins as candidates for future recombinant vaccine development. DNA and RNA were extracted from spleen tissue collected from experimentally-infected SCID mice following exposure to P. abortibovis . This combination of mouse and bacterial DNA was sequenced and aligned to the mouse genome. Mouse sequences were subtracted from the sequence pool and the remaining sequences were de novo assembled at 50x coverage into a 1.82 Mbp complete closed circular Deltaproteobacterial genome containing 2250 putative protein-coding sequences. Phylogenetic analysis of P. abortibovis predicts that this bacterium is most closely related to the organisms of the order Myxococcales, referred to as Myxobacteria. In silico prediction of vaccine candidates was performed using a reverse vaccinology approach resulting in the identification and ranking of the top 10 candidate proteins that are likely to be antigenic. Immunologic testing of these candidate proteins confirmed antigenicity of seven of the nine expressed protein candidates using serum from P. abortibovis immunized mice." />
<meta name="og-title" property="og:title" content="Genome Report: Identification and Validation of Antigenic Proteins from Pajaroellobacter abortibovis Using De Novo Genome Sequence Assembly and Reverse Vaccinology" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/2/321" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Epizootic bovine abortion (EBA), or “foothill abortion,” is the leading cause of beef cattle abortion in California and has also been reported in Nevada and Oregon. In the 1970s, the soft-shelled tick Ornithodoros coriaceus , or “pajaroello tick,” was confirmed as the disease-transmitting vector. In 2005, a novel Deltaproteobacterium was discovered as the etiologic agent of EBA (aoEBA), recently named Pajaroellobacter abortibovis . This organism cannot be grown in culture using traditional microbiological techniques; it can only be grown in experimentally-infected severe combined immunodeficient (SCID) mice. The objectives of this study were to perform a de novo genome assembly for P. abortibovis and identify and validate potential antigenic proteins as candidates for future recombinant vaccine development. DNA and RNA were extracted from spleen tissue collected from experimentally-infected SCID mice following exposure to P. abortibovis . This combination of mouse and bacterial DNA was sequenced and aligned to the mouse genome. Mouse sequences were subtracted from the sequence pool and the remaining sequences were de novo assembled at 50x coverage into a 1.82 Mbp complete closed circular Deltaproteobacterial genome containing 2250 putative protein-coding sequences. Phylogenetic analysis of P. abortibovis predicts that this bacterium is most closely related to the organisms of the order Myxococcales, referred to as Myxobacteria. In silico prediction of vaccine candidates was performed using a reverse vaccinology approach resulting in the identification and ranking of the top 10 candidate proteins that are likely to be antigenic. Immunologic testing of these candidate proteins confirmed antigenicity of seven of the nine expressed protein candidates using serum from P. abortibovis immunized mice." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2.cover-source.jpg" />
    <title>Genome Report: Identification and Validation of Antigenic Proteins from Pajaroellobacter abortibovis Using De Novo Genome Sequence Assembly and Reverse Vaccinology | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"OKW-C5pzZHxhsoU5UurVZqzRsoGEw8scXtDrMEEPsNw","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460580","apath":"\/ggg\/7\/2\/321.atom","pisa":"ggg;7\/2\/321","ac":{"\/ggg\/7\/2\/321.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/2\/321.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/2\/321"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/2\/321"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/2\/321":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022hkftcMHr6xNWkLuCeAySApNCmLrHEGMrM4YLv\\\\\\\/BAkE\\\\\\\/B1EJgQoueeUvY9bFQsDjNo8yU2Ck7lbbdDhmYZdWJrwfGMI5eK6MLoDzBQZ8Jnbqn++735MeJHPI2rVIG6qkKC1pytM4YtXhEVS7SjvSrCN0dB61N4LUtOMN1ge4CREeIsAYRDnpeCIezykFt4JPb4yMBFlzzbyq14871MQcEmavJorWNp0rCPLPXWhqXrTnyemZE740EQUazPYhK5n3gqqr+0ZevD2b3+wWpk9rVbi5P3\\\\\\\/GztOkkcEnKtCAArHUfrjvafbKlRn4J+Ta6QwtJ+W+j2CZWzkKU02ZcbPhVBBqC7Pg0I0BuL4XnaT5ve9zL6KYStBtwa\\\\\\\/R74\\\\\\\/52XRnNVtHhPV893Spxrl26uiVGR\\\\\\\/IxYYxOQxNbIfHEvLmfZuDvoe\\\\\\\/4\\\\\\\/77bZuvEV8GWwEUGxLOXrNvZLBWuzxtP65POCSX+B\\\\\\\/BNNaV2kRmnK7Ku9G4rc0DYBBgAyebu9gwTZle\\\\\\\/Xmi7pdHmlCqM04pPrtGywIW0uhdXGW\\\\\\\/zb1FsufVYuWfTmEBQvmlr5HHyUgN0dUNw77w5jDlNT9OwMq2MF1luXAXu2CSVknePbP17ES7Vf2nkwKhjnVaUJDsywOK4K9rXUGKxQ2O9Hl5WBxB+ubjcuw\\\\\\\/4s5lh2R1ZGubrMMvipcKyA4gbILPy8KdHQQiu11x8ZSo94pisnzEIctafHE69kiW6+S52qRKPGHFkNYAEh37cvin5ps9XqcMieBZ2ce8zA9xA7YaCfu4ftcAGJ5VWsT66py0fViLSB0pSx2QtXq8cQQD69c0AW0RiTZJ9INw\\\\\\\/o7zZS4oMbsRKPYQwZzYIpt+5HvYLWJnjsaUCy1F4cQ4n29mGVG95kq7fIf7oEJtgngUg72OQkkgfMEIj2BROHaKVUuBl3FgHHHZPQ4RFxX4a5oOv7x8T\\\\\\\/WWx+FvEzy52rWwxJwKXJooUbGlIoTWq9JrI6a\\\\\\\/VM5bhJc9xXS2UHNEPtSFo6Ll8+ld6hMpbGCCpk0eyPj7eLqUdND3cOjiETeExNPrq8Txg6rvZeHXSGZpqqwB70Lcq8cTrGswZjQiewjOgUuk\\\\\\\/4pNVyLEikpE0g5a+AEYLRdFzIDUYUkz7hpYba9tCYhEHfrNNjwM3\\\\\\\/+t44W45gkUoMNpoHemAv0Ak4q\\\\\\\/sm9hEfPxpjuo7AJFo8v1A31zTGpIZc0P9Hd+9qx9PoWGo6n3o+Qg0zFWRc08Kmm7eONJDN4J7XTRffF1aeGtZKoXNwLytlerZM8c3LbAr\\\u0022,\\\u0022iv\\\u0022:\\\u0022oY8hPrvhRueRdBmLo5z11g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002251b9cab8a8767b1f667f27324611ccd7f5e4e18b64939a348d872eb9265c5ed3\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Xb1r8C8btiDjITZfzR8wcG4o\\\\\\\/7YynaTO+Mg6f4hiNELz809qTDZ1deqQmdjWMkzluXN8NsggGpZXJBc\\\\\\\/KWfFjcG\\\\\\\/jx38adpatFesmhsVIpZVtj9OO7cLcT8TamtcUsBNgBoPP0QUEZaCWbGcqVJLsSe\\\\\\\/esHisnOAh2LoPHSLfFJLosY6OceFyUWKG3+rDQVWkFSLSKhASY5ugIOKJki19BXuG1lNEFK65HCXdcHJKt2uwSZrQIOqodbgBwekYKDGJqps0achzsRk7LIqy5FSHB59uAdoN\\\\\\\/2iSJhghmaQTvG9oS1X2WvYa+VcbHEci0Yvf+EFxkWvBZ\\\\\\\/+XJVUCCeDkO1K8AiJ8sVhgIXpdsfVrfGi5G3emvkO3GE8QNQPKg0ols9rrMez2Nda2UqqlYgdCOTQbTyI18+UlmsYvJMAnnOfyHAr7BPe3aiS8Z1sXwod5+40mwqMQe653rLCBm\\\\\\\/4AjiXFoQbaKqaG05UD2aeGp6nImfGoZIZwPOCboB8ZlxMMAzGhT9Tce\\\\\\\/RnzU+AQojpIPlVfu28PkuoVIokFlFOH66cK38HxT\\\\\\\/NyxXYEBAtd+4k13fjF2tO9JP9D\\\\\\\/Fq7dIqm3FwU7h+Ub1vPmgAgGvPvMxXoE+IFxW5lfbcJFpiue4ATX5DO5h1NmQaxeZ9PJhEXGHuhYo4YqfgxN9Uyy0eJbXMb16iBepiOQJ1rFy57+xOwvsY7Ddeci3PEDadSl6Zcj\\\\\\\/OsFaP94Q7pKRAmUYC8q2knJ5LT4iyA4WsNrX2\\\\\\\/8JMkwY\\\\\\\/lMCj+cEyJj6CW2\\\\\\\/ZoRUCO\\\\\\\/N+Xk0YdIiL1VOTx0UDbSRY9kznzMYeuGhO6je7I\\\\\\\/BLEVmVYOTSoQ\\\\\\\/uVOfyBKm2oRkrpdxzQxhMwyMZY0aNnRjnJuHBYmLN\\\\\\\/\\\\\\\/zkTTT9bBNKBJhSlv01FbxFQhDve9qHirbGFrPSTCZFGGYNeYj\\\\\\\/UqrfZNf5WAIVT6d1zSDlRDl0y923S0wKOOlL0qoJWcIC+DltpZj\\\\\\\/uxkNjg9j0mX2tGEeE5ZhJy80ddu06DrkxOvbojWiOke\\\\\\\/yFeoqZW7vV0UKjp2psLbUfqUAzXyqp+cIDWCcuWHH6eprDOqvGycFCg6z5+AZx3V1s\\\\\\\/MqyC8wfgkfGcMmyxaNUEXKaDPDeAo6KVEXv2DhW4rz8HtLaPKcqZjjJNxvheNZtuiBsKwHw8yEEFCVDePWssaC2JefcijJ0uaKnoFZhvwl6zL5sjUJBhTuGkuk9gX6sBUKEgLUujJj4oc7mPOklfHGntuSvd1XFPXBq44iwi3oldoT3ZTWOzDu2HuqH1yDgLTYGkSEpEQMsuO53WEMxeWNMK4k9eygNBZfGRAvXZxfMpg51kna9TivIXQwn9uWK1HVX0jdqAqbpR8N1j5ShKRyxyFn0uFYB6C3MktApRUeFsxq60zsVBZwAjYeIFSDF\\\\\\\/z2Cxx61GteDr3uztioBNZIADvxsYYdZNuXbhAjQaR9R0lDQPAu99MMI7BRBX\\\\\\\/dIpaA\\\\\\\/SkCDcfK4g4Uu8POb5Mq9cpEuAn2XjY8XIhZX\\\\\\\/C4Re99Kg4ev62ZuUe3FLXzjsDpn7uNFrytxoma5Nc0SxstVJdUUeX4YKV99uIa9loNG6NnqoMkgDeBK5LunYjRznXg0QKW1xCVi9fLffP87gZXFipupALOmuQyMehSZ9QfpuQfR3qXzTW3UpdexsQPOfg6Us3pNxTRX4Aiaa9uct1MIH7LiQbKUTwmLQ4KR7G1TtNAt+vZOXGeUM+2efjWexTwLy9OdVWMKnzbQmseOUyau32sMht1sVo+hWDrenMDUTeibeLQsoeOcrRbqCwstPIpi95tK8FOOWcSHpZ+BUf2x2JckaeC0QFecDQVJK38kTxYVgUOfIJx0f7C+P2g6mrEEIvFnh+D64MvBqqvWYWq+YK\\\\\\\/wzeXSluRBy5DJ4EqHqDeJ3z4krh+W\\\\\\\/+OLKwL7p1a0CbQcS9wVVNjt0HtDABveamzgf9j63MxwzLyLLrS\\\\\\\/qsrh+B7xLqqrQUUZgIAufLH8aGFgVXGKSYO6Ir7P5Gck8C0BF70gmsNJGr5utefsGnncyzK\\\\\\\/B5JMGAlqn7jJiNCs2Gjk5i6fr0sxEKuVciYMPDNBxdCD2TauYX\\\\\\\/hKFxPXaWOtnm29GS016y\\\\\\\/CBtMfufvvsJl0nq0Vxx7rdz794weafv\\\\\\\/ENM\\\\\\\/zhL2AWYk2ZEqIZ1URkAb\\\\\\\/5HbaiUaIRDrAEubRQCuZJ\\\\\\\/dA3yYzZjaJTgpe9vYVzwi2UqAGjjH5etX8+OfSePsxv5fGQokbCXYZHc\\\\\\\/tT82sIBo2YSDRIGiQnDrnT6ILISFGJ3WFFowtS7P1DAjlvfVFVKeUVfSDD2pNYeDY2TaLzjh1GgggcCYCSxorhz4tME5aldK45FYGCkgOjCDTaFzXbrX6ANf\\\\\\\/tK2RKqpbooMsm29seDeCRXaCtCZ54pshfyLhBrJYeAix1EWIO\\\\\\\/KDRcyk3VCQ8ziidCBfVU4sEg9yd+blyzvmtGFts\\\\\\\/ssdxKy\\\\\\\/87hFpwWrUB486E6Hl0uGF971\\\\\\\/Rihu0OjQo\\\\\\\/DoIa+G+oEuRZH4ZefX5J402gKPpUg9RjncjFHCJfMVgmataMSlPnQ14cBk2XvcxoOHsD9jtSG9dxGh6mO11M\\\\\\\/QJcync8dim+1kyh8HsYkn\\\\\\\/TBYkpg9zQswwX5vyGkDI5ESUgMcXqKuVAoOzxb\\\\\\\/Oa03DgNmqEQ73Ze+myXZ9UOzgunz+hwiu3uMp58uliTF2dy3nSNIaTLO35CLTifJ0yJix0qbaJ1XpeZWRKnD+jV6L\\\\\\\/8z5wwHyecPUGQGz4DqJSoJSpr8lnVXC+p5M5NYaogQQXfDy3qSGPG4ZEcc5BV7bCEQJVnTxcHk1875tWNbXDYl+YHFBo75Rja8MzTS4mkuiuGH6YkU6t4BINNPwiA1EOT6o7m4EMB2AShpt2f9e8gvl3IwqZH05ZuWWc0NEIWUZKqJh8nC3gVFIXUOB75aCio6JEIu8aO4jkVb87tLGipul8\\\\\\\/37xPVAHD8KNMmszrGKqyHs6wAJTSli82qSUICWaUj8lpmlE3azmBmGeB0pH4GDWNlL1+PUiwTT9lJQjmEiZUkBbfWlXIl57ODQpFx\\\\\\\/3CmIloY7WbUzNjd9sdKX1n4IvYlK2JlJQe7aX8b+mYnv7pJz4HSlZanJfHvxfEe9X2dLtAYWkMTnGOSqH\\\\\\\/epK+Wn+llInEE\\\\\\\/Sz\\\\\\\/TDcDmvHQvcloJSbBdFekx8BUfcanOsN3faWQoC5OdU7yjAO1OGZWVlSLQ61MtaVBBYEkqIWBo9c61eZBbT+0k5b2A04ymNIAKl\\\\\\\/6Lf63R29t1gxM8sU2E8HLkVcPAuW8K17D\\\\\\\/tg4JlZs\\\\\\\/H7cJCmhQQvIJjtsUVZE+722sJ+AZBp5fArYtVkrAx9eA9PHCvXSM2SI3dzjIvpJB+0Mwlnk5lU9HpSkIWUA5WRtee2WyZ7ffrJyZ910rjGOaY5Yoto3\\\\\\\/EpRlJ3qxSh1hI4G0G6qMdGC2b4QJJdcJcc45kskrGTy8iJWosbHG6\\\\\\\/vegzUeNdmxgJ4z4hINZ7LFJmVRjHhHFAxAljcFBq\\\\\\\/2bGGbwoPOp+CLMdJ4qBNIm7gi5Pw332dtKd7WTP\\\\\\\/PSHC0Uftxm+Mn0ocKjwNa\\\\\\\/wnJWUfoCp\\\\\\\/OIqgpuBBkznWCyjvnSRFAxjFV8ZygFWbWAhS0EOkn2LCFNHA0YjV+wQOO1UdknAFKeZKdH2YU1AmHv24IiufwePE1IbAONW8rluXVLjNolv+2dmXoSPZ\\\\\\\/PXKzoCJIj4XCKufUePVJK0ZElPN6KxoNVcxvCVlRS70x5m6Hl\\\\\\\/ycAPb8Z6uCLofBzbo\\\\\\\/5PysOfrVozT2nzLlztY+ZkaDS04PBQfoKcRTYB34Vt6nwlueZDBoXE0SwEIbN+HgcJYDK8KiHTewx7ZjIf2nUC6DuIK8Da89y4OeIlFIYiJtFFkOCnBX1+YtOINfDLkbF1xtrspxmuOvp69OvFU4JAWHr84OUjCImqwkhJbVbQdel++xLnjstAnr83aw7fafNqmDuBBL31Q6ULTZMndiRLRJxj\\\\\\\/HXQZjgPoDCS6TeNvWKsFKrnGAI9yIUL\\\\\\\/YBTNxALNACdOnkVCTO0Du63QR6F64qf1FiHGCAfSLslmPgRoXiL1aOnzsL2CfuwdWjRln2mx5Cj\\\\\\\/SjWuhZits+byW4cBv9rFFN\\\\\\\/6V0z3AZ39V5yKZodXe8I1k+8ddlqDnfPTJaJmckV+RgEFM+kZrmq3\\\\\\\/L2Py+f\\\\\\\/Ht\\\\\\\/pdBcb05K09gpvhL6R+SPC3ELNDdsPYgqvhYchhfxkhelBF7Xrx+G5fxhoEdk35N5ZrHooiCY2Bmt2qoedCcpYlFZ4UawzvKXH3a+gESbwa2V5RZTQAGqqZ3oJPvBApio7qXXdACFDfdO4ALaLFYGW\\\\\\\/FDvHFEwm44POnUqIoPjsgdPcSr1QLRIcY0aoj9rmvJSOC0n3Y4qF296j+wng6OZTw2dRNGkYrhi76lsjjSm99luzhoyBfoL5\\\\\\\/mIJcE\\\\\\\/AK6CcY5i2NPtV0QuqENwk27rRSn3zJhkZiawLB0OjO93xmteOXmmk33UTMPdBVrEPPoVT9l1xt4he1q\\\\\\\/9RBLjr+dD1rqRI7FFA6vmHpV13cgVE3j8i8TdyZC2te\\\\\\\/yuh4FiRz7BwhMGjhpb8DkjsDWSW6qQ6V5dG1yZDfO8l7G3qxrk1+Laewnz1A9Gjtk84eZCscUi5lRyCqlj\\\\\\\/V7csNn779F9ojBhDk3IeCV6JZ0udI7yoWvq8nE9fYqBR\\\\\\\/+jklK6eu6jNejUM3JP65ZSF5utF4SAcpFfKISmEahviqg3hOo2k1jOy8DPxv\\\\\\\/NnvIVva86m+xRdSNaM7haEN\\\\\\\/2D64NOQQkVMOp9z+GWODIxG+iPGQrbCrMC5U1maQ9jLYqSxulZjcqrsG+Bkr8JPOFqmaS+1bMrsvPkE8CA6TCPhvLtRPpBkvPLDYopTum1diJZ04tbCuis95tVSLhNBfcH8DdiqHsMBan\\\\\\\/O70dlo13g+twwI0pY3XQuEQACKShA+hKhmX2RqLjbteRP6V1uguBdkBEhL8i53cbQ+inOWBDcWFo57CGxEuQZHKkpCUsrDQGhGNe6r7IEI\\\\\\\/QOsQPd1\\\\\\\/CmOK2zp6AfjWaC5ntLWftU\\\\\\\/c5dZrZKlvjMLddfLRTFLFSzgsEH1wTvQEa5C+HsQrjc+YEO8M4dr9\\\\\\\/FxuY+EX0Dt1Sc8+UynWIv0STOc1nnhl8+0F+CkREuJzAjbB8MCHqY38E5aTMl46iK5Is4Z1p4oAmuIBUdz05+kZi4oVFnLzx4Sp1CGo3+xzSM5L9He+YcoMJog5wBqokcMGX1sUt5B9HyT\\\\\\\/oqEPLmoLPGPlzEc5xDVkeHIm0kqy7J9JjpFPwQawY+BJJGhcwUnVJaFDZL90zoOJf1GkEUtdsopFQDSxR0q9wmrNOKR03qN9FCYWpiGzIUymAjEKNeN8sHCEoFid1iNUZQUEPhEnXTwrxxuvDpMEllcRaECElwdftsTaoi2gFdBiVluprATHxBtuzzOMZdMZTOxBcB\\\\\\\/0LjH\\\\\\\/KrvfU162bJ7zUIWZ6K9HR8q\\\\\\\/q+3QL5yYdZscVlrBPa65\\\\\\\/ttDt3bgNY9WcHrKLBtXPU5waEMnUop6yRRK+q5SVutHM7Idb58OL4PRziqvZpB47ew54BVdeaYO7t4TWr+gsIF7oaOrxpGIeh\\\\\\\/4LQRWkdzjQioESjBOH7JF3KSo5elAAhi1uGAdu6CAmzINOCYsO5PnHfrfmy\\\\\\\/4RFig3zH5IQ8G2POMcKblkGkdgeYlu5ldFu8Lv0udeW81dKzxtrlCN3v\\\\\\\/I8I2HpRiztTZEF9ncoc4pQ+KelIS03TzDpE2At8o7N5pdHROGjgjhdvb1rcq03MDVviaf9qbSmvAE2hj0aNRHW\\\\\\\/7k1LJhoeHHlh9gD0Ah\\\\\\\/u+V\\\\\\\/MQTKcmIKomf\\\\\\\/lS\\\\\\\/HJwmS4L\\\\\\\/TQ7ODOV20KlaLWXiPJp6YXSbqn9W2EZeZlJ16USXR7XrHf26+ZBOPKxldZuaG53tTSWcYHl8VkGiAhVZFVj6YVgdDe4i0zf8rArMxaQqRNLeqXNOkDJ+IG4UaUnuDE4hAbqapQRaaKOGtSHj1pPzPKSAZCnJAtY\\\\\\\/M9Moi4d0iKLVz8d7MgmAaPYQPSQAo7KPMNAm7sGqLfVv+JYE6EHfubr9rp4TXJ6PQl6ePv\\\\\\\/1F58AXF\\\\\\\/ZyNDGR2B\\\\\\\/\\\\\\\/D00+vjceMSGEl4adZCGI8L56auhTnqDCnjqJ\\\\\\\/E1VFoxwuNj9T0iUGQlf2H93UIjFg5tX7zbVj8awxjl9aYfpdhBgVhZgjvbLXyn8rq0vY\\\\\\\/VsARqryqU15N1tY5Qs40mRudHcsBJaM93JF\\\\\\\/fBwPVJ1BqM7+hecNqE7qyek3p8dMLEWJF7fl00uR8BpTN0ltdEf9H4XI9nYItJsLMXMQqOcp3FFMbezTtVS09KcC\\\\\\\/5aBrwvm3c9agifpdNojBCbpj0sR9dxD+uySdldx1pG07dnadD+zhhfgRQRMUkML9ddWcLydN87\\\\\\\/26QC98TSLuxXbOqI43sCscY+tn4rmSpt+VUYzr8oe+SXPOSsdP4QfepMuiys\\\\\\\/CIOHudCLK\\\\\\\/QGhkS\\\\\\\/4FR0Nh7qEcF\\\\\\\/FxX9dhGdV3NHlNKY4+PRlkFamLvtGQCp8Uyl6qLrSCtMdXhdl+PeyFu8LMhYWXkRVuFgFYMxiTn3Jdx2\\\\\\\/dk3wSRT5RDN5IMcm+n8eyFAlniwFAOzyPAZ4v7RnFl8AkqSX6ASZVhDpRfo00roYSTGWG6Ms13frv4CitPv1sSnbSLH6blIPDXHaSJnM21lr0JdwTwZ\\\\\\\/oOatI75YM19Sa4KUiaoKSdBjF333XDnnZHirk\\\\\\\/FWxbUraV\\\\\\\/ulOiXFUV6yFa9iluZH1lc8t1QzU9GDT5P80x7aDEs2+OGFkK4pv0Ox+gyNMD1x1psdsBoMKt6cV0mTpIbufXonUKKw9+9d+dyhTzj4Ra6cskP0Yqr48DPIdp3UvZ94bE5qcCBn5zV7nR2qlhiU7T66mNY0TS54JYCNUfBE3OTyfqa2ZMYCrmnV3Nn3cM4uey8NlSMr8uquFQuRjRhh6ADsB++8QLwv9XI4q27Pij1GKbOtS0sYf18IMkm8VEmWJINtc+c6EkIdvDg1B0ilPKtCcSW\\\\\\\/zeaWfC0tjFztwXFAetAWA+UY\\\\\\\/FTVdvLoQXZmiGMZsp8Fabr+Le1pkmAaQD\\\\\\\/e2O1y+X1C62aixBU2QmtKkvLyC2MSOK4uNZXTF1\\\\\\\/EGp1vQpsS6t6AEUq2QL258rcvMmRBnYS8n1kiqxW4A76kQckrvVhGJcgFYLi3++zcxfa\\\\\\\/Hlr+6e\\\\\\\/ikEIYbB4x\\\\\\\/wP2KZTJ8HRWsjRX1irl30TuvVzXE9Mb+l0bszsg\\\\\\\/F7bLJslY5rZOU7feLCsCVI91jbxslhykXWcAuvZhNxZXZxEdHbOvhh8DvOmCMboxXbbRsB439VIEWFdElUNKRYJp62hNFmdCIO0aFLzaPVjXDh+qS3uuu\\\\\\\/rcKDmvTQm\\\\\\\/c\\\\\\\/r45hTs6AIGgusZWlS0Cp4rjGUXDYIG6MaM0X8ofbHeW47B8ejzYvByar7SuYvZc9pCDjbbqHOSmAUQntFN\\\\\\\/VeE5adXT4Bg78uVfP143XOfu0irbufDrFR0gNtTWfdUNtZJ6vGDkitTzxIcvEp8bu6bSBBMs0g60mfaNpVgBoh\\\\\\\/g1DZNI8JD5k0K1l3iXHyuS9ONofoIPds9sjfBgI10S3hthAQviq+wwegU\\\\\\\/fVFD5CQF8OPgkEHzucMwW96jyGXYcpRhqXpNL9X9HNXUCqI6w7VfB55hPdlUhNAUcSZVg8L5XbHGYjMgheZKrzR2o3s4aJp8azzzjtrWoevUxiAAS+qHsqp415eJo4w8vnuUQeGRvMWq\\\\\\\/kZ8Iju1qBDY7NhIiTgsnrxBhJwk07Zd7+iQcFH+2p368kbpyyfZ7JZfpncxjIPh92Rpd5ZGN\\\\\\\/lXCn8F3mzcx+tRizSQX3IRGGfzmp8kaLtMv52rVdm8jZfLcSvFDxi6nt5LRwo8bqB8mcWIHkCwXr\\\\\\\/K\\\\\\\/RzLeMXGVIPZ+5BInRfpSxbk1qT+ZB5MRLn8LwHe\\\\\\\/67viMDX\\\\\\\/z1dTyMzbxPsV9CS1Dc4+cmLwSdu2t0B2Yf62s\\\\\\\/POO5UoID5JoxzdvO1soeFUt\\\\\\\/yGG3EqfGlDZq1+LJdgzjVZhTvo5Z6MmHBsiV1RNTy9q4fr4zxb0qB11DkM0JA8pfYo\\\\\\\/cYZlxXOnuQowMJOHjQGgoBSbWeE9AQkKwZ1c+GW1CreCYd2iRDVlttTP3uoJS7onREZJDVs\\\\\\\/NNmWMhVwQwDuaECRrH3oI4GXUCtlVXuT0COIfoU\\\\\\\/i\\\\\\\/Zpk8zTHG1c3uB93lxmDHmT358izBhaerIubcLzd9BOZahDxZX2a+E4a8dFAJMzTRhq6euUJovbCeJ8gDyz2O9NDghZkn6XlXFn9MllW\\\\\\\/f5CYn3XhhGhM1C9i3T9honBM2QuD0A0EsMyFH06pjKYFZYiMwrp2XF+W+Bwpf7nLBp\\\\\\\/ww+6jS+uDlTRKci0a1\\\\\\\/UXteRBmIEKF0CO8MjKoksRmnAV0Q22RyuA+ZcwQZKUrony79LnRsfdvVLYEsr3fXYOAburWWNd7410dt6RNe3ay2+v\\\\\\\/XanPyqdePLoJNRWy9yqtCQxSdG3nHedv06M38xCSzRNIStL1w7xHwOpEycSzEGlLIOd1L1R+AO6694FjEa7bnfEsnPnrvMBCP+Ax0gEpoLILadPpfuZG5HXQmVcaYyQjDzCYf3s1+PRoI71SEzHCFYYNWYr0NQ2L1SSzO69FifRthaWAIIA6xCfc9IrEkehLqXhHfj7DAhvLURoguB8jfqHNaBqOuFAFK8kx+f1bz7IowtOwN6aWR8zjMzJQ2D1OZa87Vcq2EE\\\\\\\/xF\\\\\\\/M64x4m4Q2LZMUNW+bKURvDLP\\\\\\\/nwH1669i6MbZtNyOAp95M+hLRBt9Je+kkRWGv2nkLHkMlOEsNra9iO+JvVLbpoua4K+5T3AoflFx334S6xW0zQZY8SuTVZA4HuXv\\\\\\\/w+dpPPpG2y5HU\\\\\\\/jdLqKHPeUcPT0ogF9B+O93hSb5zmkiD3ylmUwWnxEQR0P1TiQi4vVKGW5OtiLvU4Zl+\\\\\\\/DeKSl568xwOPH2K3Kl3+Rv7QhUphkpnSPUvSM+tQbBEWi0aucvWEq4K4kSSetsiZF58iITsbX\\\\\\\/la1Fm+yEP2eGqukxcsj\\\\\\\/FTMhupGo8hG7vIAdL9iZ5\\\\\\\/MdrnhtWAzH9N7U\\\\\\\/SnaSdw2MaNl6HZyAtTQ\\\\\\\/3TnVl2tvLrVTduKKAt6Ah1Rxrr+k0\\\\\\\/G50jZaqwDlLa6cVXQ0697D2f6rn5dES5XZ5gSPGrs5nSavdQEDRa8mP2EnrK+fEPj69lF6Zwf7UNNrkQkP2xFPgWzDz87xvulLILO3dh3juaYePiQPHUQE+7QBGlYsaaG9MlwGXmnP7ngJIs6nqfYn8FJz+LfdPZomvdGPGBTo+MM49Kj\\\\\\\/UmC6TvUB8BY089j2WreIFsIYPGW98M4xtrB9fNebZGuPo6k+TTOzMxlFotCbBjeJccYOuOy2pY7UEsudF3YXTuRGnLkeI+uqmVMluHDU6DXbpJggbiwO9H+9olIP1\\\\\\\/vbST4WVEfBu8MIIZpyP\\\\\\\/WGrCtsZKuDb5lCjqnQVXmkpTTuEnrpi0LeeZefK0Cup\\\\\\\/qJP8Zd+F76N5+FsHA4CKs7wrKh6iMjM1e6XA8dybRscKLDOwqFW02\\\\\\\/a1ujZ4fqpNb2Gt\\\\\\\/FPVhqdX+qnluM00AWzAdQdmNYoyS3DBhvLPg+7tetovp2twGgPVBl7hjAbUFsT1kc1DzizoxucVbszPvzu4OzE\\\\\\\/nvF4VvYnS2TGfvljWloI04\\\\\\\/R03CrxnrZiP9P5kcFzgXxz5JOxMrNzTD5bX4R30fQ+GKEXo8dVT8N\\\\\\\/Na\\\\\\\/5nq39Z7zalBrM+\\\\\\\/Yk0+woMmiPPk+LQoxTZVaclpnQpfBZBVAQBz+blNBQ+f1hxepcW7SF\\\\\\\/zNgc8YFRtg6RmxcalTuHQFZW2SLG6JoyI99yDSskP7k0HHGumiQ6hZflPFAiDPJAfRcoQ0ytuhlQ97Raam1bYBkaUoNIGv1G7XVmSfqA28K50JCNCHau8IdzPqBggd1cs1mKuCsymPg3Mog1w3G7iAZQJNM+3JuPXjq855ruoy3Bzez0qhBCcbYVf3WWc2bW\\\\\\\/fljNVMA8mnprEi+GoqRfOv1AMzvPrXSVca5pN2dy3\\\\\\\/ehDLfmYQM0\\\\\\\/y4JlWqOlqXBUonCS11cVO700l7Ej7B\\\\\\\/ywfp2OuV+sK1uVrX9G2kCnCNnOgdtk+1OdGh6vr8Z1ATOQCp1bYRnhmwt9n71wYDG3Q+OBH2GibbON5lnqnhGZMt1NDziTLpM+2o8A9qpGV\\\\\\\/3WeA8xII6pWkZzT+JaFtnbHrqsHNPY4jiHhOWHYMDxeq1kmybFe8gKFIwdQO1H66SULJVrccMIstnpC61eppJpClN4afCxXY6khOZNqTBxLSu6pjf0Avq6Kp+SoqJHOl7ZU\\\\\\\/MxcYCe3T6u2AQJLnxBvObwEe1hr89v7jjTLyi42de4j2hlmGeS1K9hjX1MNwCNxTd1ENqgDeYHnZSmQgcCalac+El2X2PbGQAERsvUciCpB12FYjULuZ57gxyUuwbP1K7IcE2fmhVHifSPpjEW1rMFQzLQ4jxOJOFPCJZ+mbJjxXEaNadXXGPb4nj2LMWQ+6IpBLYshaqZOi\\\\\\\/YRMGpcHdkiQkNh\\\\\\\/VGiEUVlhnfkOCfyVtr78Lfh7XoIjy\\\\\\\/eIgtH6sgnV9kiTBM3FFvCxyM8+A37qdIKb4kwp5CbX+pVeY2+HVX6Wtsu1Z7bu0s5CT3R0ycpmIU+OrVDS5J8JSs4ZjIPkoVXTaAlVH6zSwxb2Ylaa7ApvQplpzZ\\\\\\\/f5\\\\\\\/fQ\\\\\\\/rRwF9SfZWsorIaaEic\\\\\\\/BaeEH5c5UvemIbiU9KR2YlIY390VfBwkIKha3Rz6N+mOWvROTTPzvteYmY2rq5tX2ANLWZo2+hNBT5j9dVCwdjYM+aoJPJPV56pNXJfuwlt7LGxKnGHqTLZlteTQn4jfurQPwweCZxbbfbNKh\\\\\\\/yOsr5zN90C6Nsf0mKqlswxYvv0R72iimUC7w45vAb68gf8KArRnHlr2DXKY+ocXjtL7NMHGRs+5Tvx25hYAFQ8Ozv5A98\\\\\\\/67c53jzJNefyBhz+Fj5uzo6XMd67Ak+PvDX\\\\\\\/r\\\\\\\/KsQwSrTwdeWDzcsDdIve0GzZ9HYybu1SS1qfPUCjufZA9l+WVreSS8djokoAuYYiK1BR\\\\\\\/X62YjXE0aH392a5+ps1De4VGpmKW1295GZ+no5kYC\\\\\\\/ioes2Az0e7eDUHDAKdRrhom9z6JTSkx4X31EQE8AOg30k0S7Vs4L+d96rtHvFY3oZMu0Ws9\\\\\\\/s\\\\\\\/HzTe5nkq+VAqM7MKdtY5kexk7sPjKvbYP8tTd2NY5m705zfI7AEyyBMy1OuZlGTDDdf9cbxvyLzzfsLTbK5QITMEqZNQlMkUpq8Dskj+WjRCcOS0o1tPk26KiyQpoHMuWCtGZp1f45Tf0IQ7nOjNsC8B9sG5a38WH33JAMZVxDnwIcCspIBy6gcbxiusIuwyymzuo\\\\\\\/4yhhCqb\\\\\\\/WvUOzOn8iOfpbKVYez4hfyzlp19MhP4P7g7ZccAwYIzO9JwLpD7Krv6fQD4Cc3\\\\\\\/guYpGV\\\\\\\/Kd5FYzj0Fso8e\\\\\\\/dcLoFwPzS4A\\\\\\\/L47JkXGax7vydK5r8GgFbD8p13mHMMNv8N+d+t0S+ZCkCsHzaWAa2PylBcUOQVpqK1z\\\\\\\/1Bf4TvkMsOdy8TcNrGFAMKc6g4\\\\\\\/V\\\\\\\/XLW\\\\\\\/H+Pn0dI3Pk3Snrmz93KyGl1VNT3j6jvsSnthPX6BRrjW6eoPlCbSBHuFJIBPKMmP83IqnNGDuhJy6T\\\\\\\/\\\\\\\/syoBUv+hMijLQGHkJH++FFXcgMRono6oSyHn0+MksQYNU4flyvIbMPayqo9RHfCN8cGWje5pykfSJ4nf+0Za0I7pj+mz5x\\\\\\\/9dejK5GL2z3XQtFW3FkATHtl6X5YVLekrCgWPlobV9OrLuUjilNn27nF5LIGON1K4n\\\\\\\/3ZTCt\\\\\\\/rBxePLHFpXomcvXWWk+o9UAUne6blUiQigvf3PK8KTp67dOXqyW6q2BgQk0kZoS1GoFNrRC81vK2+tU2u7nTUQi7Jo+xgBnoCAobINQC7syHKfnkcqpT1BUf8I3ZCnogT2+gwHpvVIoDoReSm\\\\\\\/P4NKlbWx8sNBQFOWmfhGsrqLNo5djG6c3JsZ4mCWelURARD0nLmZ+f6G3bn7bjp1UGWUPfLVb3pM+YPMea0VKq2LbzOmZEEQshYWSIJmo6+iy8OZnc9HJ9rjRZQOIdG4m\\\\\\\/JGOPluq2u+rysQ71qDdxjbThQzzmVQKbPJOIc5zZ5lsedtHp9CCyQ7BoHNquPV+imse3Ek6+9Qq6zBf39H6\\\\\\\/M6JYpTptqIlB22WdbtLul2v0Ygka6G9fcsxhhEzxGEr5urs95AdmB3DDxFrJc4AFH7J91BsFjar5Gu7h18Z0pePEKIjNIffNMe1jKJc8o7IDZXfvd5bEXh7XkOCQeh1lAm2TqEePA7brJUWnET7K6elYevKPghnXKDGy+\\\\\\\/Ic2yX1E\\\\\\\/JiOCn6v0z\\\\\\\/uPHe\\\\\\\/9rmixTu1TXm1MEZy1vZAw5xtxRFGt1zfuUmwfe5fCqSEk7FLiGDhn9x8jpoY\\\\\\\/8rD59BDxhII52tbbFiR8ile2WEbIR\\\\\\\/EqDVQfg0\\\\\\\/YnvKbjU87MGuAI3Piagi5TddbbBtNXGAnQpK58Wa\\\\\\\/pdpOrX0rtoF\\\\\\\/zcd\\\\\\\/f81lNgf+kA2hD68mek7au7htpM1SoCGN1FBZOWUsATHS5KaNRD7kBEp193K2sRmqDORCvyDr0nOUTZD59ZRkHkyh0YUjjRGQeP4+LaAaH1lDdP82QtmbjfbuSbBanQnF3dkgoJR8Mu1aT2VeWeDpTLXGHTgU5PhQS+Ytr+AxWzex82q640pmWwg6Rz+XVVplx5poPk9ERoD9G\\\\\\\/4ZANpgodlHHRzrCU\\\\\\\/XrrZSArl3iGfMpSrqRbOMAyRK4WAYyU\\\\\\\/8nLuki23YOnkXfv3bvKpaxRVQqr3akgVHoER0U3GvY3RYDz8btJGt9z+FZ\\\\\\\/hEOCKDyv3YRBME4CCF8JNcTHq7q0RVd+KUAgfhrtvbvbNm\\\\\\\/thG6ZrsYmrmbCCFFvyNZ+kWoSGKdks7t1BUJKbNyQQk6LSIQKKINOXeXLSZwwKd9TRVr+I4hlxzNtIpT3t2wafj8+eSo4MU8KL2xnGyXfGt\\\\\\\/R8OXyIesBIEk2XZ98GSG+sFcZ8QCfQ1B4wjGU8dEQKxHn6y\\\\\\\/bGF3xUuzYw60jvT6zvaErhF60nShRoaUzQPEAcluprdIl9toxmTwUZ112LtzOmAbH47tTqMIyVYcdvVA1eORMIQHuxSb6DYScWmm7YaYkczu2TE1EDGjyRaqm8Gk5Lh3ukY0izGnDTG7xoi9NZon7nscIx5NRSK4z0i+cR26\\\\\\\/l7rCnfO\\\\\\\/daeX0mW5uYnD4BQxM3+6J08kufcskSXtGsVUkqBIJjra8ly\\\\\\\/o8\\\\\\\/bfCZcWtPta0EVP6j+5JG4xmF90XN7qUz5ZocnS6zyApahbSHN27dFxEX58Q+GADARmwjAd3mgpAsSdBuSYk8gs8FJv+XCXDQ1kNZm7fLhD1t0WfFypKQqr44kFgUByk9mfojmy3rfg+QHZ3N1sjNQ+AOiie64BZ8TwZATlYu9IWY85DSSDTVS\\\\\\\/sPpy6Mh3Z8nUxV2Y0ns2xHlL2GSBCt8v2bg0Arz\\\\\\\/QjZ0wLyJczrUxXfdLQqIYOec6SxF2WY\\\\\\\/E0jWx4rQ3xuu1RZsxaBsSfBGgePyCPgLH2dRkAZARHdhwXGZSgBgzEq8XrWx8SIatKitq+pWQdJEH5xszDVR0IgFL8agyxq0CNhHXq8HRkifPlTNKY7f\\\\\\\/JENXvxCt7\\\\\\\/lA4\\\\\\\/oHvWtnLquy4iTg5VetUiaP81R9nZd8JTeT3D5fddu5D4Q2k+QN4DsJDYjEYmr7escbjTwOpU5Cy9M8tG98hCyeX6umWy56crvgsZz\\\\\\\/q6hQyBBmz52uB7AF6xtdwV93OWjXnNZxYXRvOTKymTcJLfhOd1E8oJnWARu8lAnaOtxelX8reqegXy9PPDG2gGv6Izvz1nWBSG7Y\\\\\\\/T8xAHjrD3NWvokl3bgYwINKsrjPEA1AAFreXjNrqjLZMd\\\\\\\/dIPl0+jrijUnNhqer2Ye\\\\\\\/zXVIHInJOk+rY\\\\\\\/x8nFY7KhXDW0vBiOjEA0YcioidTqAMq\\\\\\\/Wp6l\\\\\\\/PhKMKbWCjFHw13\\\\\\\/Y9YdWojCOuAQTd4TWeJiNsCOrB9QQ+UPCZhqLqbz+5HgEtASuw3Rmj+BtdWclhAFJoZo5hd+G1GWkqrxuF\\\\\\\/wWcoEgO2mgy9COhdiR32EYtEE1UxTTqhGjbMlbKsaX1sPeP9ZynulsS80q+JLPz4nBlpz6bKju3IgBSYyF7p3gcrxA+mKOb1D1nLyeWWjx0wZC7K9NFKX5cfBbQn+8hAM81kwZe2iDVv9NGmGGj4e5p7osf27zhEcUz7AjOl8AsiNjFh1vMPhuc3mOsAnEx0P7XW7f9qPFb2iQH7xhC2AX3dTM5F+H8dULvVWWWPaq7ENyUSeI56Vqd7BcIk4cF2RHhavootgKu7W\\\\\\\/klYlLcoXvRwEzJyjvU5Tr3HVHYp4siMYgZu7GHda1aT\\\\\\\/aGsIzjnGUzFfCPYsCsoaQSntlylR39jg87ffL5G3xVJCHon2IrsrhcyJwY4COYt60NKVQnDRt70dZgPtoe607bsHkvN86R4rCdHVrAJtmuPth+0XpYb+xfJL0FZ5Q6hfu0LITm\\\\\\\/J9faVX87BVUwyzg33zHBT7baDMDbsBCqPo6prqpNfTTUThpaXbTLB4KKM4mOgHDrYSz+5JVuWCzfOftkPZamvJfB11P+1J661PWnZ\\\\\\\/g+akSi0qVL17A4Gsy\\\\\\\/xyphgmGz8nfGWJ\\\\\\\/njgPzBdeZ9VkQ\\\\\\\/8ZaUUeMfuQ6MTwpy1b8KFEMPUwl8FHvA92UVZ6Z2eOCkc7MIzbFUN7FEHqjvLlgGMLL5KwWKzsNU0lDeQ1nrb9TkoN4\\\\\\\/qMLjdJTBP22yVc7q7\\\\\\\/lkJC66qnlCycdydaByWlkbaMGKzoB9CKdeV23Clouq4A9K7pe4+binvCAQfPVYp9UJQBppvI57ZJx1pQyofZCHbQt3Kh7zr4oineCVnV3416y0A38WwWbrRaMAwnvsET0\\\\\\\/DquamKACtkz7bSgR5cbh4QTdm8M0LGT1NKCKKSDbBy5ZzNL8pWzRwJWGQg4VjodMUWyH6T4SY9M5Kid\\\\\\\/RhVD3hwTj1s9qOrL+gROvNqCNaLn5uDaKET7hRyWxaJG2fVOFQwwI5fZovEfa9A9v6qNkIsLbICJ459DcM61gfFaK4cJ8Ap2ct3lkUQDZmWC98jywy4X9iZ6I0Rq8reUh4moLHxcf+qf+QA7yJ2nExdrLdE7KiKHH+Ccm2jfOLLXP0IqLDsbRQ8190GVcAD0UZ+iMOLdGUzQHJI1UPE3f1gebm4n3B\\\\\\\/EsOLkFLx\\\\\\\/x42ma7+Oo+ORVojMxIR+X6dPw\\\\\\\/50FLSM0HYZew1iA13oqcjOSv0S\\\\\\\/StPOPTMbCtGZvRqWfp11CYl3+ftZeS7LWfTG9LjNrvfPziTs9bOIcdGz8N4XA8IRktnJSK31dUn+HPt7d2AjZF3WVHjB5XRDCXeUBxKEPfVGa4t03iDeGwQmwy6TY9CrhPFf0T6+Bc7Ny467SqWVA9O\\\\\\\/x0ZiECQteIceIxncEPlTAFsCEVFk8dBvAV\\\\\\\/2LQiwMC\\\\\\\/Y3WPYK\\\\\\\/0CONffbWlWYpvynw6yLr3ZslX5pXVJVHA6lI2uTbpckoTdbjqvgkOGndf4UX4D0xTyz45Xpo\\\\\\\/K+HHZGJb8NQcuwzXExBlWQujL7vOIuGkcc7GnbC6Vgd5WA3fSyd0kyHsLuU3W61vsMN+JigR8cm9QsUWJdXpf78QiZ9AHSBXsWnQhJY1\\\\\\\/E81ugTRrvQ+pSOa20Wr1BWK93m9FKhJmdrkyes9+S2rlpjo+7oo0tpkG6w1rNm47Wg7QXgKjq5oh2mSMXZCZcZXm4yKzk0a9V4A+Pezv8yDNI9jERQuA1IkvEOoaygvouvn1z494rEQ5hJGeONzfpMRGvcnovEXSzOvYltZo3USToGCbPOzCvr82DwTEe+LG0q4fDXwaCDAhLchGLYYw5N59fFLeh1BDw7WQ0KEsqeteLdqpfZBMjI60keDe7IkVuZHm4vprOns9QiQJi3QpGNwu2QvjiOoAYkC3ICwulLLLdUM+qEy3a1K1F5ksHscmpk3SBrnB46BrZQoLWA9MvGMQiat32tbKxAZ3dKIbsvzPl4BfClkjOMGpM4HZWEzvUT85K2fDo4LbFVlky8te6LjDU7BRSZfT6RiIZv3HRvKQjynWAGIwZ54oIM8r\\\\\\\/AciCDrC0xznFyNBCyb+gXMmKRLdVWLPkBpGsCuqQ0HR\\\\\\\/a10XaEu04eBUYJK+nSOmnyFp0LPDJILUiANzLDJsfj3z+xIohCoTnPBY7Gf+N2piKiaYeQZAztGUBQy1TDQx15Ikn4fch1YgahhU8zU7bT62\\\\\\\/XbCAjlcO7eqIu\\\\\\\/vYNM+C1gPeHIje1jZhP+VZT4u67j5XNFtZi5UaJlL\\\\\\\/YMlWz4rCYRISDG6LGBMBAbbv4EIX1XvfcOn8zogeytKUN4Ho1ObGmYtD2FWdKXq9KiICgS3LBf\\\\\\\/P3yevtBE7+PFAqz6AWK7gbHSoCRfIbIs0vcu3vZM5JRTBx+1KmvXg2sP2XsEpmGt9KYvlZwLAb0szFpVJ3TujsEVs1SlLCbrLGEAmICNHMf9W274zSwoMyOWIwwChbEcZnWB+u4F5x02uYTBwNlf6K6UP9zy2QEoUS6RXTZ8yuZCY89Oou3WPqmq3vQ1rYM3Ba4inRfhKOKqov\\\\\\\/0FUGaAgqXwgEymivelna1Clshr3dsMhGFAWopc4Mhs8uoO2p8fDa8G8ZYk9w5mW9M9eHHKvmLz9YgjTYL7reb4PMBmFm2WyZQPeppePVbTD56bEReNbdhH0T5eU9TAU9mrmXoYsHT8PRbTqh5I7XjPo7Uh+uygRgqnQ5qCy0cmYUXr6LvY7FnowMF0QOPB2O8kk+wzlP\\\\\\\/N2HEDhfdjMq1ij9N37+46OXi5Q0+sY+P5PQae7UnXbWkLFrbx21goII26GnaO\\\\\\\/\\\\\\\/X+m3gcFdMx61HjSPd\\\\\\\/\\\\\\\/3E\\\\\\\/osNXVuC\\\\\\\/knE4BDOlVRqzkhZH3JBZSikWcnVA8cnbAz1dAFCS\\\\\\\/\\\\\\\/ikdKmeL4\\\\\\\/MOp5DcIvv3ywvWT\\\\\\\/ei\\\\\\\/dZIKemlUN7HafuEyVjtkDZSLCKhmT4DEUVM+Tzyi+u1VcC8vK1PkjlgvlwW1m86szt0nWLpE4M\\\\\\\/\\\\\\\/+Y+1KgGw5I+tCFsfvnJRGMMANKpeYP2VY\\\\\\\/FXJ7iTx0AWkRR7inyP4W8\\\\\\\/Dmc0KTmKGB0yl8IM5aiy9UhlZKV\\\\\\\/tfmJ1RZI17DO71D6qiab2AS2C0SXo5ohJnH1kevIwhvAgJov9\\\\\\\/HxqC0PUMVyI+DRh0ZDm2Kx+eMhG65X3c1g5HhO+wmmvMjytxZaNXf4N+qKJ3viY9KL5obVbvw\\\\\\\/lDs\\\\\\\/bZDfEpfQFfivkg2Htqyw9WohGeXEgtFpR2O48mPx12PRo1m65zSFh7+KdNq3g6MWGLMui+NMQzGFJqjneX7hhQCW78YCMIYAFds0MNUpZIkcLzCs4uwjgm+7DcNGCHHrbiVoNI9TinSBDBhx37rkyxV54bTLMUbPhFOJQ1\\\\\\\/5ultXe1NNb7KmZAheYgrdVyl+r1Qh7fEgfH\\\\\\\/oqGILpE6b2et4SeStlPk2wsaJRiZXRI\\\\\\\/kvUuHrlZCmrZSuIQd7NOAERprO5cDyZWiX4mf41Mnn328Wz7CY8zJVAAl\\\\\\\/I3IzdQGrV+p2gZ+9j04Q1Q+9xdE4F\\\\\\\/qUYURtW9P4jmRZubrtLP0MAppHvaqIUtXlWpYcj4s0AbmLblOzszMNv\\\\\\\/zwxmO\\\\\\\/zeccPnshkUTwGDvPeQCR9Myak9yczIknicTgZiwZjh5lQ7ifBluVm2sHkd3mIn5Qx5xCZDUg+lyN5c5fhpJ8OAGLQvYbbJJuP6nfoYTMoog57Cg1tm3hA3I91\\\\\\\/XGWpV1hMPbOC5ylV90r5EFj5mAndAOyRM6DVByi59oqf9hXyhEJOxgIIPs5TFQHHiYJmWuyUSItVCSzndCsYdHXduYpf8yBTBcgagD6F+QnFzXvcY95LuF5rZHQrCR4VFGSuAmFr8giUWbOePzQlYu99YxD51VKpc7tERO89M4GfOQWMsAAwBH40hqd80uW3Khx4uLI3GiJB+V8GXTBZTtbAnXgS\\\\\\\/cTZKqSfdacn0QaNezqpA8pfxLgGH3TNrgCvGc3nx5OaqkV3qJobarngEsa6zzPpQfFLiL+dkb0s8d4DAElEi0lxVZvvQvETL6koykrICAH181tNS2297hRySKQFUfUHiyOPwJnLOT3Arn2lZk3sR0ku5J\\\\\\\/UqyhDP3jfeRe1wawGivnFUQVATp0goRTMba+reqMUVWpp86Edztd205ox5R7a\\\\\\\/Qm0vzDgI+A+oCdrgPMKxznDMH0800LbTt8XIpj8MNezxr+57Vg6vzevQBxoh+aKUBuCoffL+d7JEm\\\\\\\/IRQ8Dd8fjKLd4PV62tM7hZDpYv4aqINH\\\\\\\/gc2IUQ4vHTqLURablFKA0k2cnWEKbVsROGM41G7i+p6Q1FpGb9GIaBErw0xP5RYYi\\\\\\\/gyEapBIxe2VO8Hnqm7F1V53szqUM7+bhUcno2r3haq9oKSPrvJu02ftKGS1O6pzKAJ1YOLY2I7aDb5A3EmFSVJf21FleUFJuUxr8LBnZeYArZwyGq4Z4\\\\\\\/BAH0qEOwCEwbXeIfUnZv8fW06fa77lycHmVhbq6UX3AyszWxNA22Ksq9ViV4UPzfVZaqB1VEExd4\\\\\\\/XyC27uG8rUojQDcYCcg6nOPadVsJl5NVbv8wp5mFZlhc8sxtN4qHXD7WjPcKb+cf2NdjODoN82Bs1ZoJC0W42wIeXFCoQjWYmxalKelepMCBfHFtbexHlHMUS1DMZb9nIYaV9OwdSC7D7zckD\\\\\\\/3uCLEe7Bu4g2ZlrYtACyvQGcvnuFby86LR8I45t4X4ODlJJRR4YRigflKruj3bjhUXidil8vds20DI98zaAqd79WjfOxbgk+JyZGxNaRIMR+R1L3e+8L9qTvo5vKIayX95lpSE+NxLMkjs55Nhw6G7pvTamr3UK1pcr2W4GXqakXxR8l9m7Tf1o2tStcT1LldmbkpWLEl7MZvPm7QAYJjJgVurJPysannO2UmwKmfOgYTT6WIv+DLUY5eVwkrb81cePoWQ8QmuVPYvh8yb6dp+IxDZUETDEsb+2UguGf0Dh0C7eKNQUOG6QOTEAQEtix\\\\\\\/BJBu5wli3CQ4\\\\\\\/+HOUPFqhpD+IGqWG9MCdhT0\\\\\\\/xrc1GCgWDRoPCPRM7XNq5kVv2BQTocWaKvaw9jiTyn1Ckez15Eh5OwR9kyzTS4vFW+7l1uJA6gTToYomfTW6ciUuQ51Hbl3yZIJQfA0nBmsYcjeR6cPS\\\\\\\/EaTmYCkhl6mXs825kA2TvMk\\\\\\\/RVB5UCW4MioLrb\\\\\\\/pD\\\\\\\/9uiE9+7FB+d6YMo5lTTzzCaWnB20slGAIBBmQy6eiNmAELYFpl4LqfPULmrkAa49GOlzmt9GbmHJFzF9cT5GNbLZjPRCK\\\\\\\/RCiQ358QNPQhIL9L95cBhyKyyJt6vgWZg4\\\\\\\/EwEls1\\\\\\\/\\\\\\\/\\\\\\\/KZRa\\\\\\\/svFRhIUZ0ph7KCq8iWKgoqwlLigwhCqYFGKLkavc\\\\\\\/+KcbmP\\\\\\\/FvzuZeck+xUN1jiAIoABlPKc1D45LBp5ymKpeYcsc7bqCZk+VTKb5Awm\\\\\\\/BkSGkcXX0EvDu\\\\\\\/86+KYhh5h2\\\\\\\/Od3stsDSE8qrNKpVVRt4kjuqy6og+0NRRrlL7nMtcxLZIkLz+FJZ4To83T4oqf4MXoqM3t\\\\\\\/XZouFuaY6mZgkYN108TjIUpeF0747\\\\\\\/4ctV3md1kq\\\\\\\/ipYKkbsiw+o1L2mdv0OVpcja+5+\\\\\\\/bPun6oLAoNfjsb2ZaYK6FofW1vSUpSNlKiejYee9NZDWGU\\\\\\\/uWxNko34i3wIpjTp2h6JPwKLji8d83AiXFYjmlTF1m6GDmgFnXyb9uhTasmuQ3jYeinIbJ3ZJaH4ueMNKs75Q1jktjAdqdRNRfEztvx2iMrYBeJBcHbpmip8n46s5jMmu+bGZl5E4wJ+0fi2+a6aqP9rwBrK7HkBWeNDAYkvh18X8Afa\\\\\\\/9Fqt4l6GWMiIDUR75eBeBjDtl6ISusdiTl9mQFdFZtPdCUM0Rc8FMpg4LiWL90BbJSiQRYgCswsZ9OQ9g+MBynPW+eTG3TjcSJ1X7oF5lR6eM7FjVdZej5xi6iqk6+jTCsjMm7e69YT6z4WFfuZ3tw3GCN3eHqv1yipZbhhJYm1rtKfMrCRPEgKC9\\\\\\\/4sJ6QOOLHALyDD16td4cpJi4QYgOl1dONlpP2SUy5oi5xg19JhEW0TCk94hN7Pcx6tco0OHz63DUMGciM2lALKbdhrUvPdd\\\\\\\/6Kcmgl1W8S2vBo5aVk\\\\\\\/ugGlGWtFdfwQr0bXeHE3C7coMSvnsWDrrtZOs0Sv9lS\\\\\\\/FByFGNqhY7jVT3fT6VRqRWFKLNuVwsacBJioyU6o\\\\\\\/k4YO6vsIS928Du8jwQ165nqO\\\\\\\/hdoehjoNzrjCnNPcvMgg9tLs8szYzVhCghr7uG2HCldWErJzGuGsTIR3q+8PZVSuYYShaxFIqcoPwOud4ZQhbKPb6EXd0W41++KTOE6Wso4T9m7Jt+Qrb2saYKBDIDQtIar5jx7Dkgfe+3EO5svApHZU4DEKMit45IyOyYJs+XbmuJOdhvxRaGm4apBHkzcAeFzirN9bT7cmeC5P83uSG0N8zQ+V0csICrSNjYW8jrKANy5NCs209vrZNIZyGhZAeHGNT1ehVJLh5mrfYAld9rCkeGBjYkObye3J2E8VuU2Fi2Aw0FFvmXOgxhrn6KCRGbxSwklfoq8C401zR9iCCGn\\\\\\\/PdqQhhu3lkZG765rq6NYeJXlFshLyAnaWn2b8vh43TylLJivRysnADR7OpUVlWVLCsShbYh31TouVs4EWnQn0KKC0x4yLERLJiu0voWSPsaXh1A\\\\\\\/aXac0RD3BzFRL9wcb8gtjiFtyAnQngkO2ALaXGH4aVnL6gczTsWrTUzAW3dXftNjS\\\\\\\/eo\\\\\\\/rwEoN+ThwZ78hJdIwx\\\\\\\/TMUiityYMFey481nLLSivK94ox7FtJRG1TrT8MAxzjCBPAJVZJR\\\\\\\/uOMOnmIfyr5id5aYwZkVP235qgx918geRHJIW70SoQw0PrUXUDkZL7LRgnmQmE5w\\\\\\\/mLIZK87hz8HqRdawe2oIz1vRz8kY6KRJLmvXEKxayi36UIH4bDBpYHVQTtfXp2HsgpvImOgu3dC9Vxmc2IYddgyUZ+eFoQI9AyKzR27SsOVP2+aC2aveAGpTA0VvxembyFK1\\\\\\\/OSLmqaPLSC4peaZmbBi7E+NB\\\\\\\/fjkPchswMO9grU1qOtOrCq5DyFV5DeS6EnANrNhzPm39wrDdhw9+jzD6JekX7AWjNXggBkf3QQEFgAORmRa5oVJlDSfsPWyZrLvD7MSUXk30Acn0NDZvcjwZHO3rcLIJ7nSMqvdcSE27qgrWVorMwGh9EcG12L\\\\\\\/xeKYu6CXsDUdg7KR8uXO8LXKpu6Lhlfr6w3IdD+ORkHPokRToA1MhmAqcuPEmPbo9jernD33GaIXMxnJJ1zks2J4cu9PWo6pKV3EgNZRz89XMQ+6I4G1g0tYJJpBytZ1AMfVouBg2KepTZfNpj2ijUNBno+y6MOvVhZ70kArtndUtuwGS4UpivMhmL34b0asSXFyxQRISobCh4fROt33o+Zmu3iCf4JcjD7rANLvpkNva1cbt7K2jHpaPlqHaF3jMcuzLaRd6k0bTpo+0QfMzKhVX4JGQ4Cam5Iwe\\\\\\\/dT96gYE5gAdFrDW4SusjL6Hnr9laogkyjWnUae8xys5HLhTqihsT+hhC\\\\\\\/jYtmJEet3tqRcsC4V+qKpxNQEhZK0hrIqUPzZoDuhL5pGYF9xie3U+W7RPpaplMW1nXjuKklcomKPMTGrB1OgaABF9HfSJkpdYEniZcRAFLq2zs9uGdvddGxPKs3DCqzxdRwZcUXFHTKwLJsOp1wdPjL37d+E6aZBQ6YmquKHlUFcBpJSMRl2XlkG\\\\\\\/czWl49N4G7HKenqrNjwpXLpZIwudRzU13droMX3sAOGM3DQSFYsgVNITyv3cKSstR\\\\\\\/iJK\\\\\\\/+mmRL5NsLXEePiIbHojTCmttQvow3JyEd2oNfy8TP7hLU6fOZKHXUyodCA76btCpmR+9vA9JvXHYYA9apJUkxSgIhGLu5F1wvMRr30fOnjgW1y1QxVNcqlwaH\\\\\\\/Q9ULAw1845QKj+7bHY\\\\\\\/wz5S5a5HFPu14bb3xAjVvsMLMH0LrjqsZrSgjQkx8ZZ9g9okY1xSRh2ELzamninSd1TsmAyYilo903JMj7cOOUTYzsoBZjob\\\\\\\/IDt45S3Pu+W8podygq3cXu3Q2kbr6yP3iFGnO62rYZ+MN2iRCZYzgrsEDfI\\\\\\\/2nZubW4vDXQLJaAOopnXVlGfwK9Lqbuoz8mzHDhJLwu5\\\\\\\/zuR2MMr8susNCkcX2XAAHhrQgXoXTHY5\\\\\\\/mCuMV8Cd77qG1idKtOfUELOjxZD4i+WzdLs54I94Fkcf3VQr14LZx+V+fSgo5G0ApOwU3cM4QObcRdMn9F3cFm2gbRyRg4yE4cD\\\\\\\/MsRMAmjO6PleKQ1cv0\\\\\\\/AG19t4NPbp8rCnd5s2oBL0V1hx5eb9I95iTVW\\\\\\\/eAw2MU3mHL5iEE5P47LAgQsnYl0\\\\\\\/4d5ALusnxkHQdWtIAlbmgjl49nUuhIKElPvUf8va6q7bkecSyV6Nfat0+JcWjnxfdYkAvctHrRNJ6MZMEHbSyE5yEDFoTH8X34wHRn1gB46\\\\\\\/VtADqx6\\\\\\\/9b5ekXmAKuqtKV30QsQZnpHr7jt3ukDNsPNspNk5p43eOGnixb5zCWVKs3yukuJ5Hc3eZ4G7BhBWDD8kFptwx+LaDxdFxmHa3TSeHLrsGj1+K04BJ\\\\\\\/S1TRBasBtJr1bp5pxcNkLocGltekpwMAYtMlkWPrah2d+EOOgDTzl4g6ZC2HGzPZvMwhsVo6wFm0ga\\\\\\\/L6TyuMSuTWK8SB\\\\\\\/I9XeKyi4hQUPPZ5vSq32hSVMNMKRxZDZznmJo+0lAOavVAcXFcoQdWa3mnNLqYsW0KdHOXkQ3r9\\\\\\\/\\\\\\\/HD29LXPP3lQQ1F2rjta\\\\\\\/i4K22lUhfVQPsONxabIAQhEE3J5M\\\\\\\/XaafzUK9fCZ2IncD8ycn0dv9p0r51M5myRy+OZ6+GeyokCnx0NNlc3IT0bJQVvsn1RQG451eFA0LQaaEQ\\\\\\\/DZJno2RbjbxOu8EwsDd0DaXhVkKQW7MUdgjVMVDDBjxkgM7D6K281GSwy7WPjivYHmYRmOI3AUnUHyaHyzOnWwPj+MkrHPfrVQ08BMoGTx1DMjO\\\\\\\/GMI8iFRx5KuvMx1PwOi+ImqmfiF\\\\\\\/FhuBw7+Fz3ABxmca2ycgWEAXYtE4gx8ajtWAoc4zIqpBFXJ1oA2rrKlA3D1Gsr1VifniWHHHxY\\\\\\\/NCvoxx\\\\\\\/4TIjX2Tc115vcI4R2OCSJJDWorNAnhAECRaIbzbKKYt4ThRux5Sp6RXsl0K3PW9UrT5b9IMPEiSiXcnvun9kLveO2JCy+Ne1dD8pUCVkXdUf+BL10bue7XYtLT\\\\\\\/3TptVaGztsqITsKV0NqtOtBbFlXcASwuVp63S0U+858NUSJZ0Riu03+jfMwpv3YxJWp6inWHtDq5bNt6uYrJ\\\\\\\/Cu74b5ig5qYZYLYsxWfsgT1pcdi87uJvpW8eWHd77GFRb2SrVHNLwZtKoKrKnx0mDuQL\\\\\\\/Mo39tfd2iktxPB35Km3FDTF\\\\\\\/GxSvUxqW5dqCt0AS5iXA6xOinKEqp7onPfyZXGdRyf7ewDI82XlyMza0MkR3WM0kvkUEvw2A\\\\\\\/p50TsBJrTI4LZlLWlxo4qLwi61oIf+gFds6W26lZeMRZ+YEgCUctOspOwo0Uj9QKolImegRcJWVLnZ5SLzltaqwe20LC4OQqfJWmudYjlQAagJhKgGtnaXw1GILuGkruxTxP32mU412FFkuTpWNNKsAWlhHQSzhE6L+QMezGDL2EvFbtEbENgUAofyezJa7cOHsLfGAbFn9NunGINMBUXlBIx5YFeLLMzJGfwiQswyqA4AghM3g+XCTBAOJmle\\\\\\\/6bYxzFYrBM+cbqxyc2bF4v2+xqJKpdzo9qnTfQ\\\\\\\/dOQFnrIk51\\\\\\\/thaZrAoAJC71Q5JiQ96R+KaX79B65wGmSWSJUt4PBudPp7V3J1zcI80GGxYT2Ni8gje2EnGuXMH6GBpNyiu+RDUI5OAqosbc4H4aN98tstz\\\\\\\/vdbpSbzDH37tJeM8UUPkjfsdOlC3nbnJnc3N89TaCa9L2LLTDY3lCi7a6oMln4V5zIJ9eZ1pb8mqNWSBVwJRSP8o9y\\\\\\\/8oqelOkpZnzOcyOFL0+fxomXeaiOLx+fcREQ7CJxZqwHkimb2vHicyfvu0SuMOHQPVl1nWgvKPV3ui3U1\\\\\\\/LBfcVMa8Wme39pD9InmUCjyTgeXVOIvUuDJjAmafpHnETKlqhTQT8nx8CBqaTRV5LWTkh6PXhn0ej0cHaWn33gthFw7QzNDeO8dfviPO8x8pUydjXaf2++\\\\\\\/1yTbgF\\\\\\\/29QdJDkkc+CcZ3YNFgImeji\\\\\\\/x1KqOHkrc6p9K21TtjUfBwtRwcbnWG6pNA4\\\\\\\/ZSM38uDigpdFegZwmGV4tWK2XSiixChXGHFcQmOK6AF66ceqinb57jY6QkxkL6ATxh\\\\\\\/UBNTwsWz0CFCdOpDh7bUrnUIlKwhEeeVNrlQJeRz1gnLlpYmMEddk0uAHEjFYGynPNklTu0eKDnkNcUHE8zXLkapE+bcpZplOBE\\\\\\\/GhS+GFh\\\\\\\/DQ4g+diQLwOxfgxoNFn+tCTrNjnhIyHtWs2bYT\\\\\\\/C1OIBim3vG5CuhsQpxTDTn+aky3md05ByICoriTPxLKd\\\\\\\/Vk5vaLfA\\\\\\\/pEHWv\\\\\\\/h6vLEnUpGOrvvNNlSIHN7RB6R0bjKPFPa1Lrb6wTjI5PkBht5y0EGvpzyA1cwMNFFJnX47VrZGYLaqxRfQis0OosnKGdivkblCg1E\\\\\\\/NEFPEJU56MZ7\\\\\\\/LhN9IANE6TcN3+nUx1+481fi5YchNP7VZbRd3zYoGbJTtcyQlBGn7V9PoHV86y0sOvdN7ir9XO1u55pT1bh6c2qw6Vke8cKaj95wa+Ojj2DueyyJcJDhEE0O59OuS1SUhcoUo+1syEycxp6wjJqKDkJkVdaWzmnOqa8uznz7\\\\\\\/kDEHPKcXsgP8qKeMIC\\\\\\\/gvLWrwohlhdRbo4bNrqwOsalzwoZm2vYNZpKIM+vVRcDOxCYc2OQdXkvR4dlwcSSLGtqqpTS76gKY9tGXUAp0GE8J1Xwy3n5uZhKfoeKAOGY6\\\\\\\/nD15lZ4e9SpFTLuHhGyaNfsziZ7hjcTXAlIg6UbVJ2qu5eimWAZm9HK9cCgdR1\\\\\\\/K36n3qe6JPylWrVFqABnGD2I0hsMgdEYBBoxvsUnHH\\\\\\\/vzPoAfQIyL1fTX\\\\\\\/1B7JzDAhqQxIpfGt\\\\\\\/Jyso+UiBdPvO+F3CmD3BpBPRMrhQvRVNbQPwRLZx7LyUFvlZELNKo1Z6XTQhpw6IS9LAETb8XCo5TDLPIm+C1PURHF4ej4w5qb4xPRClTwglUd4zN4OlRx\\\\\\\/ywuvY8qvFrNtttajkIAjpyptXkTZWHhiR+nNtHqx9M6ojUyibJWmEsyfED32vJeH9zIw16PXtJoJVFdVJ3CwXU9qjKBhm+AXoHeIc6\\\\\\\/OYZiEHFWcJCEFGnj1ox8mIj7DwH3doiiTgwboeImHyMxTExCdLlSLFHnehokAxVxRCg\\\\\\\/OV50bbh4cGwrdgJDyG6qjCkeEiAzqhOBVToZJ+7pKnVTd3Wn40rSbV4q1TMhCSRV8Cd4AvVeLiSvqE6SOyXgMLMqao7GqfGZBOHK5I+PQBDA0gZ5tth7Y6IOYajEZX12DgZ7PGwb3OXh2sw0gTSLZ4SsRtq4gZ8yj26kEdu\\\\\\\/JMQEjcbiOM4r9RWp3maK8xAEByY\\\\\\\/q1kwqMUkfJdWAKz\\\\\\\/JS8EexUXfihmaW6ZJkfq5s4xKlyT\\\\\\\/eeeFPj5qTfAPr7m9jJhoTBf9SRHBl4BpdsRU72qyQzMVcuBOYMdjT80smVrh7eUDEyNVdQ35VQ6KNhRkF14gEpFBO\\\\\\\/DwhCg\\\\\\\/NXFAnZFH7gw0GDtkR+f+GGbULVsOlka+Tg0vI9xMEBR\\\\\\\/cx60X2qOX0OS8FAy\\\\\\\/OaOC3j\\\\\\\/SLip5NSkaBKHO7Y4bZLAW+w5m+Q6MHavSJeiIH5Xrol6o54OLi+CXnVBRyz1f6ogFrlr5DJVfJsSFxpDdfqkHdYLQ5964KmocpsOIK+zWoUQVzh6mH+cJOjKAy5nhwUV9B9\\\\\\\/gmlZZ3AiqDkKHIIGiy3gAziF5HGu+MBFF8a7j2DQ3ZmlZT8tdfhWdhP4H92LdaoqtkF7qKt6b3ZK8yNZdR4uI8QIpvv2ha00f4uufZpNZCx6y6Kiy4p6N4QYK++T8kpMyWLGt7eazLQdFJ90I4c8WJWO5YX\\\\\\\/TWWEJBQftfZ6QHZ5bAIgdjU5+sRnxVvHZZXZHqxE\\\\\\\/dn+z2LkK22OGuOR9ZuFjdiJe80lInorF9LC7PutfM\\\\\\\/\\\\\\\/zoAi+\\\\\\\/vUq5d\\\\\\\/dj3JESBBUGFl5SonuE9KSSLDH37xzhgN+7KzPqJqvlYLIwZEjN5H2fF2d7mMKR2Ezzw\\\\\\\/akO7z92GsUkXpu+\\\\\\\/7xMc8A7IH5b21zGbJjmBwWt8PRDsUMON1DuiGP+9FhrCDr4xE90ypyr2LsILzjhUmKi7Kas18boFncsxnAjDfk6bnxQu65XCR\\\\\\\/+Vs46xzRzWJgskwwngiXn69D822+kvZkH1PbSSvXbjl3NoAqMBtiAPIA7crQUV\\\\\\\/GuK0Je37DEM\\\\\\\/85QzD6jSLk4+xl+S\\\\\\\/QcDIc3Nbee2WaSxSgNAVzcABmiMouLVWbVtKcS3Wd84bXBPYYwcEHRrOrl3\\\\\\\/4bDXxQieJEre4NsEA0MFdQauQpaUciU885xzwSOaMhfJB+93FtUyOkQh\\\\\\\/4ZG2TpSe7bK5pgeR77tEx8gUfRWer6QXfW7bX9LRCq\\\\\\\/5RHv3874ZXlG6L1dMNFKcMle\\\\\\\/ED31mRXCV\\\\\\\/sAwFagrnENoJD07BkOoOXIeQ7nXrVtv3cEVXkEj3T\\\\\\\/cfr23s5Eut\\\\\\\/UptYYnHLgWMECdzP6OmETjxN+5pMJ61\\\\\\\/CqTYyVHMBmCWgtHLQRKLSlhqieDCIKfcUJj\\\\\\\/oLYrPD\\\\\\\/f6l\\\\\\\/gZMwa2CDEzx0M0DKYwW4EGhTNARDbeGqvb0rlfRvRqiv6VScpqNHFnRL0qjkoMgmV\\\\\\\/CRHH2vlWye0OkLayLvGFFhHWpjD1bqGLqDFzMAGPRALbJutyzAjaNkUPdgRewOlgMij7vBsOTaYjxyAbhiQewgwdbgI7EfBiFTqBNHxu9otAMCn+iYgjqrHskwc90MjwVnkSeidAS60+lv2lYCFiGIStxXmLuBK\\\\\\\/3irjSu85qJ0fNmCrYyo0rwzmjCN5sSyHdimZY4DDeKpq2NE6HLYqFt6dbwkdK2Mzl+V5M94VtOKaUYoCPiJqsgODBhtWJcK+oW1hOcd8Uptq00iYqZwkjvUVPeSety4+FWxgh0J1s876UNApP22Fs6XZZBzClWTieruDC2n7giH+LUdOHCzAuLzzoAxtiPf9abI+fyBrnwa54okbpNxI29UhckZXgnSoI9LoGN\\\\\\\/d7vMcExFcBjsekF2v6fi\\\\\\\/fV3qZAKEWPqCGtKte3MM9IIc7fQopdazsuo7v4wlW7p5357WqKyf41nmpFEZPuBhDCBGaNQhhPt8AmVplmbuFV5F1cveTKY+fuK8t45hRrbCSkjW6Znq2ylZEQQgoVM6fcpskZaGyuI7L2b6uRNIMU\\\\\\\/2GY2HgG0A+DYqMUkVQfCgS6h2cAYoFytxktgJtfUwuI1gdTCcp8Et\\\\\\\/44Ayt9r\\\\\\\/KC7qOgM5iTInddnV7IJSurcllHfk1Xj3hX4hIYrLbSQLmPygOGUjqrheGLQlueKaFmR6pTeCwJOiGIwBZ2TvzDCnUV4Eo4Vbhvoqi2rHWY3k2HOm2nv8d7iJbgNZoAzfihgwkUbBb7W+eNuPrQo6lMCMSK7ZtGXuf6hs0lws9rK\\\\\\\/z6XGSBt6fufKbQYSMkyySjT4SfZpZ9r72X6TiZ2evmI3fmXD1ZFps\\\\\\\/A016qfrQetKGZ5aubbq+qoC7Eidi2rCah7X6c\\\\\\\/6fZocvv9PcJvn2ZIjDqTnptigWOEzCNQAEnRA4ZiQiHOmimoS5L4XIKD0491bDargMOSw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022F\\\\\\\/0pU+72ZxsyjJwI0+ZKFg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022944749789d30c17b27406e6104ae055b8cecb878c6a6c9414b42905adc5afa14\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022JtyAiDB\\\\\\\/7je\\\\\\\/\\\\\\\/laQmfVhsCJH9WCJGrBsZSKGmgcWhJZqOFPoI0NaJ8YNkFaggx5hpI1qoR7Yiy1LLYx2U8QJkrkwR6FYTsuI3ewDmZv83ejomNgtSSU+P3qY3ngYxLXQCovxW05NMD0sNz7eipKGiiPo+RhVbCxbanX5JyN1mdl8BAmzhCayLPdUlVRmwIZby0Ioj9ELO+y4fHKgRD5KmBXuuPjeal6A7ItqxrgLBxNoNCslnkbTsG48LmRu2LKgamzapKqTNFkIfD9Ua2mD3Opa5EVpX5ENe2+FY8ybKcCzAF6Wb+WxvG8uW+DgVi\\\\\\\/JS3I9aBzNr+0L8m11s8Sa6qrKDcIf6nYfJA1cIGoEGfh6YW9cFwYzGTbFPMjGfmE\\\\\\\/QpkySIC7dDsJ14pv9CWnmhFvFjLnEgQvQPZ0sWKSzRm0UDC2TFd0TwMN53\\\\\\\/++SVuBlXDOrr5urB98fJqhlWw2XPa4PUcuAQQkL+ufftePdQCE1Rhacz+Te3QjZf+EH4u\\\\\\\/SiXCsRqBgrYU6HGi7QSj8j2x9j5CCVlwx16C8AR3bh8xMzkrLzLbAGRjYhvcmJRwYBlpnUA8zkEFHY0ECbofejP9TEgFcLYScdzRAy+aEJ7mlpa5QQqM4nqiIYBYzrgLenHSm2sRV5mo2i1\\\\\\\/gXzGePXHjBMyAy0LJ5JPoCWQBy2K5yneNyNL6rpip8jYQmZtsLbgLg+obus\\\\\\\/vfipnaLcDSjzdzN6qyXFmo0MSSNIdFQrABRuQFwoUzfWL3ZTHMusFhG++hOKz+7Zpagr9mgKrhuYvkmOcxo\\\\\\\/LUPVH0TX+zXlAWG+yWMnE1Hm3k0UmHRJu8a\\\\\\\/o\\\\\\\/gYnhz8dGpNejaFuGkiklfgqmzAcvocjcoBClG\\\\\\\/TTz6LVIR6NKwaKA3RcoDdN5JYwIEr6Fmo3h6LQgH73PbhFqSq4bfA\\\\\\\/xIxQ2NSj\\\\\\\/+0pTu8hQ4wbeOmjTgce4MyaDU\\\\\\\/5Unn1mlsBDMKw4GW7PuNBQ4spjjVilIlPL4\\\\\\\/kBe21fYLjnsnvtwpRws\\\\\\\/v9A8IVtikCurDMSqAvn4PuNcalYk60wQLekMvpG76d3SqBAFS+q2Y5G7zAQe7qpJ46p2IodSjtoRRU6+Sol7Z9EmyPmVOh8uyiqgNbg\\\\\\\/g5xsIRKtPBJAlUx9d4KGxPBQtpQ5KyJYXBYYKIaCGQOBjboQPn3MNDx8fIlDKRgsChFggQGULMfqx1yiiutYjwKv94IUpBWfCrybNwffpdKOFsNPb6QIsxXCSachREUyJh9ZEK\\\\\\\/FaNf7YgDw\\\\\\\/wnyRpU381vkN3gb+n65\\\\\\\/ig1RljDsLZilAFuAE9QoxF7yfiUVnkB+K9+3fhe6lNBtjPKcex1n02+MpuI\\\\\\\/wyrOCHNpGRTDRMYbnYKsxpXXql\\\\\\\/NprncTu9wmBRYLqcjkdVv0yc43yhymPPC\\\\\\\/As7IBsEBf9pqxlqI2bsEVT0e6t4OlCf1RYD8xylmm3RP\\\\\\\/WNngCzNltomu7IEcxHQTVRNRNuFKtsr+tB1lXXtGX2Tqyv3WapOGYyeSnGPpP7EsHA2\\\\\\\/MoiTvAVSo26TxTjSm+WeX5juoehc1HeRJuWAG8VnLaK4MYKd2Xjm0vd06kFq9HWpyj8ryD9fkSKltJGKlvUu5f\\\\\\\/4\\\\\\\/hauk\\\\\\\/XOSgWMkElY2UX4RlxLeXxpdVHv3BczkWhU2rVhCacnz5Hz7V97+qXqh9XB25\\\\\\\/iIo34Pax19n6MdAHaMDDdU0sObgWhp78sYQDgvIwIMe0dM718RWLU8gc941x0uVzUGnhnysgrq7tYrKtg3sGNiF267E+JwrW9XB7YEFAfytjOGFEQSkC7MleKf\\\\\\\/jGhPVgDgLIvUwyobNGXg6iU8rSGXd\\\\\\\/RiSfRakafDQDT1zcn2\\\\\\\/kfThkBhpZOXJmoEhz8NnVWBo3fR9I6XBlYbRADbKsEYNuttHgsmceNHRBXJ6EUAiYrPQzetjxvt2ZX3TXOQFuaQrxDBRo1Vs2ZgZdUGZSnpKTnlJe1jDzuGI0ZAow\\\\\\\/L6T2cRbHKqBpFFJHOSgjJWIiOTxQ==\\\u0022,\\\u0022iv\\\u0022:\\\u00223K95xKTCaiEpfXMlH+\\\\\\\/h4A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228283859f1c7740da6401ffcc3644c2676e5d30e1e9a3cb5fd1fbbf0928fba9ed\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460580 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genome-report">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/2/321" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-yTyTViBPJAYvYLWsNa1zYRoOxV7u9MDaOm6GPRyGqhE" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/2/321" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form--9cFvPfy3N0CwSNe2cvHu8LGvWdRfFc-4ijsLqw1fG4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/1/319" title="Corrigendum" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/2/333" title="A High Copy Suppressor Screen for Autophagy Defects in Saccharomyces arl1Δ and ypt6Δ Strains" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460580" id="node460580" data-pisa="ggg;7/2/321" data-pisa-master="ggg;g3.116.036673" data-apath="/ggg/7/2/321.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome Report: Identification and Validation of Antigenic Proteins from <em>Pajaroellobacter abortibovis</em> Using <em>De Novo</em> Genome Sequence Assembly and Reverse Vaccinology</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-4282-9285" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Bryan T. Welly</span>, <span class="highwire-citation-author" data-delta="1">Michael R. Miller</span>, <span class="highwire-citation-author" data-delta="2">Jeffrey L. Stott</span>, <span class="highwire-citation-author" data-delta="3">Myra T. Blanchard</span>, <span class="highwire-citation-author" data-delta="4">Alma D. Islas-Trejo</span>, <span class="highwire-citation-author" data-delta="5">Sean M. O’Rourke</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-4181-4233" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Amy E. Young</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0001-7113-3183" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Juan F. Medrano</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0003-1562-162X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Alison L. Van Eenennaam</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">321-331;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.036673 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460580" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Bryan T. Welly </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bryan%2BT.%2BWelly%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Welly%20BT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABryan%2BT.%2BWelly%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4282-9285" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Bryan T. Welly</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Michael R. Miller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BR.%2BMiller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Miller%20MR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BR.%2BMiller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jeffrey L. Stott </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Pathology, Microbiology and Immunology, School of Veterinary Medicine, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeffrey%2BL.%2BStott%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stott%20JL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJeffrey%2BL.%2BStott%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Myra T. Blanchard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Pathology, Microbiology and Immunology, School of Veterinary Medicine, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Myra%2BT.%2BBlanchard%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Blanchard%20MT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMyra%2BT.%2BBlanchard%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Alma D. Islas-Trejo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alma%2BD.%2BIslas-Trejo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Islas-Trejo%20AD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlma%2BD.%2BIslas-Trejo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Sean M. O’Rourke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sean%2BM.%2BO%E2%80%99Rourke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=O%E2%80%99Rourke%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASean%2BM.%2BO%25E2%2580%2599Rourke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Amy E. Young </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amy%2BE.%2BYoung%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Young%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAmy%2BE.%2BYoung%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4181-4233" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Amy E. Young</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Juan F. Medrano </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juan%2BF.%2BMedrano%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Medrano%20JF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJuan%2BF.%2BMedrano%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7113-3183" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Juan F. Medrano</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Alison L. Van Eenennaam </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alison%2BL.%2BVan%2BEenennaam%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Van%20Eenennaam%20AL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlison%2BL.%2BVan%2BEenennaam%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-1562-162X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alison L. Van Eenennaam</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:alvaneenennaam@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">alvaneenennaam@ucdavis.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/2/321" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460580" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460580/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/2/321.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460580" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460580/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/2/321.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460580" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460580/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/2/321.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460580" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460580/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/2/321.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460580" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460580/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Epizootic bovine abortion (EBA), or “foothill abortion,” is the leading cause of beef cattle abortion in California and has also been reported in Nevada and Oregon. In the 1970s, the soft-shelled tick <em>Ornithodoros coriaceus</em>, or “pajaroello tick,” was confirmed as the disease-transmitting vector. In 2005, a novel Deltaproteobacterium was discovered as the etiologic agent of EBA (aoEBA), recently named <em>Pajaroellobacter abortibovis</em>. This organism cannot be grown in culture using traditional microbiological techniques; it can only be grown in experimentally-infected severe combined immunodeficient (SCID) mice. The objectives of this study were to perform a <em>de novo</em> genome assembly for <em>P. abortibovis</em> and identify and validate potential antigenic proteins as candidates for future recombinant vaccine development. DNA and RNA were extracted from spleen tissue collected from experimentally-infected SCID mice following exposure to <em>P. abortibovis</em>. This combination of mouse and bacterial DNA was sequenced and aligned to the mouse genome. Mouse sequences were subtracted from the sequence pool and the remaining sequences were <em>de novo</em> assembled at 50x coverage into a 1.82 Mbp complete closed circular Deltaproteobacterial genome containing 2250 putative protein-coding sequences. Phylogenetic analysis of <em>P. abortibovis</em> predicts that this bacterium is most closely related to the organisms of the order Myxococcales, referred to as Myxobacteria. <em>In silico</em> prediction of vaccine candidates was performed using a reverse vaccinology approach resulting in the identification and ranking of the top 10 candidate proteins that are likely to be antigenic. Immunologic testing of these candidate proteins confirmed antigenicity of seven of the nine expressed protein candidates using serum from <em>P. abortibovis</em> immunized mice.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/epizootic-bovine-abortion" class="hw-term hw-article-keyword hw-article-keyword-epizootic-bovine-abortion" rel="nofollow">epizootic bovine abortion</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/pajaroellobacter-abortibovis" class="hw-term hw-article-keyword hw-article-keyword-pajaroellobacter-abortibovis" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Pajaroellobacter abortibovis</em></a></li><li class="kwd"><a href="/keyword/reverse-vaccinology" class="hw-term hw-article-keyword hw-article-keyword-reverse-vaccinology" rel="nofollow">reverse vaccinology</a></li><li class="kwd"><a href="/keyword/recombinant-vaccine" class="hw-term hw-article-keyword hw-article-keyword-recombinant-vaccine" rel="nofollow">recombinant vaccine</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/ornithodoros-coriaceus" class="hw-term hw-article-keyword hw-article-keyword-ornithodoros-coriaceus" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Ornithodoros coriaceus</em></a></li></ul><p id="p-3">EBA, or “foothill abortion,” has affected the California beef cattle industry for over 60 yr. It is a specific fetal syndrome with distinctive pathology characterized by late-term abortion or birth of weak calves (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Howarth <em>et al.</em> 1956</a>). In the 1970s, the soft-shelled tick <em>Ornithodoros coriaceus</em>, or “pajaroello tick,” was shown to have the same geographic distribution as EBA and was confirmed to be the EBA-transmitting vector (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Schmidtmann <em>et al.</em> 1976</a>). EBA cases have historically been confined to California, Nevada, and Oregon (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Teglas <em>et al.</em> 2006</a>).</p><p id="p-4">Diagnosis is based upon the presence of characteristic gross and microscopic lesions of the aborted fetus, increased fetal serum immunoglobulin concentrations, history of exposure to the tick vector, and elimination of other causes (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Kennedy <em>et al.</em> 1983</a>). The inability to experimentally propagate the etiologic agent resulted in many failed attempts to identify the causative microbe. Suppression hybridization polymerase chain reaction (PCR) ultimately identified the causative agent of EBA (aoEBA) as a novel Deltaproteobacterium closely related to organisms in the order Myxococcales (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">King <em>et al.</em> 2005</a>). Analysis indicated that the most closely related fully-sequenced organism is <em>Sorangium cellulosum</em>; however, updated 16S rRNA sequence analysis groups it into a novel clade with unnamed, uncultured environmental bacteria (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Brooks <em>et al.</em> 2016</a>).</p><p id="p-5">Identification of this Deltaproteobacterium, <em>Pajaroellobacter abortibovis</em>, allowed for the development of new diagnostic methodologies (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Blanchard <em>et al.</em> 2010</a>, <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">2014</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Brooks <em>et al.</em> 2011</a>). The visualization of <em>P. abortibovis</em> through immunohistochemistry and DNA-based diagnostic techniques using PCR have increased diagnostic sensitivity (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Anderson <em>et al.</em> 2006</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Brooks <em>et al.</em> 2016</a>). The inability to culture this bacterium in media necessitated an alternate host for further EBA research. SCID mice have become the preferred host for propagation of <em>P. abortibovis</em>, providing a consistent bacterial load for experimental analysis (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Blanchard <em>et al.</em> 2010</a>).</p><p id="p-6">A live bacterial vaccine consisting of <em>P. abortibovis</em>-infected SCID mouse spleen cells is undergoing field trials with favorable results (J. L. Stott, unpublished data). However, the vaccine can only be administered to nonpregnant cows as the bacteria in the vaccine are live and virulent. A recombinant vaccine would provide the added benefit of being safe for administration to both open and pregnant cattle (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Ellis 1999</a>). Since recombinant vaccine development relies on a thorough understanding of the pathogen’s genomic makeup, sequencing and assembly of the <em>P. abortibovis</em> genome was the first step in the development of a recombinant vaccine for EBA.</p><p id="p-7">Next-generation sequencing (NGS) has increased access to large quantities of sequence data, thereby enabling the successful assembly of many bacterial genomes. Since the advent of NGS, a new field known as “Reverse Vaccinology” (RV), defined as an <em>in silico</em> approach to discover potential vaccine candidates starting with the genome sequence of a bacterial strain, has emerged (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Rappuoli 2001</a>). RV predicts putative antigens based on genomic sequence data and circumvents the limitations of traditional vaccine development for pathogens that are difficult to cultivate. The identification of vaccine candidates through an <em>in silico</em> approach can substantially reduce the immunologic bench experiments required to identify potential vaccine candidates.</p><p id="p-8">The objectives of this study were to assemble and annotate the complete genome of <em>P. abortibovis</em>, use a RV approach to identify vaccine candidates, and perform immunologic western blots to experimentally evaluate the antigenicity of these candidate proteins.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Organism derivation</h3><p id="p-9">Spleen tissue was collected from two EBA-infected BALB/c <em>scid</em> mice for DNA extraction (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Blanchard <em>et al.</em> 2010</a>). Spleens were first pressed through 100 µm Falcon nylon mesh cell strainers (Corning Inc., Corning, NY) with Dulbecco’s phosphate buffered saline (DPBS). The preparations were pelleted by centrifugation at 300 × <em>g</em>, the supernatant harvested, and centrifuged at 1550 × <em>g</em>. The resulting bacterial pellet was suspended in DPBS with 10% DMSO, rate frozen, and stored in liquid nitrogen. DNA was extracted using standard phenol:chloroform methods (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Sambrook and Russell 2006</a>). Spleens from two EBA-infected BALB/c <em>rag</em> mice where minced and stored in RNA<em>later</em> (ThermoFisher Scientific, Rockford, IL). RNA was extracted using the ZR-Duet DNA/RNA mini-prep kit (Zymo Research Corporation, Irvine, CA).</p></div><div id="sec-3" class="subsection"><h3>Ethics statement</h3><p id="p-10">Mice were housed at the University of California, Davis (UCD) and studies were conducted in compliance with the U. S. Federal Animal Welfare Act and the Health Research Extension Act, and overseen by the UCD Institutional Animal Care and Use Committee (Protocol #17358, Approved 12/13/2012).</p></div><div id="sec-4" class="subsection"><h3>Sequencing</h3><p id="p-11">The genomic library was prepared using the NEBNext DNA Library Prep Kit (New England BioLabs, Ipswich, MA) with an average fragment size of 230 bp. The library was sequenced using 100 bp paired-end reads (PE100) on the Illumina HiSeq 2500 (Illumina, San Diego, CA). A mate-pair genomic library was prepared using the Nextera Mate-Pair Protocol (Illumina) with an average fragment size of 2500 bp, and was paired-end sequenced with 100 bp reads on the Illumina HiSeq analyzer. Ribosomal RNA was removed from total RNA using the RiboMinus Invitrogen Eukaryote kit for RNA (Life Technologies, Carlsbad, CA). The RNA-Seq library was prepared using the NEBNext Ultra RNA Library prep kit (New England BioLabs) with an average fragment size of 250 bp, and was paired-end sequenced with 100 bp reads on the Illumina HiSeq analyzer.</p></div><div id="sec-5" class="subsection"><h3>Data processing</h3><p id="p-12">Genomic sequences from the paired-end library were aligned with high stringency to the murine genome (mm10) (Mouse Genome Sequencing Consortium 2002) with Bowtie2 (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Langmead and Salzberg 2012</a>) and unaligned reads were collected. Remaining reads were assembled utilizing a de Bruijn graph genome assembler, Velvet (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Zerbino and Birney 2008</a>), with a k-mer length of 31. The k-mer coverage for the resulting contigs were plotted on a histogram to determine the frequency of coverage (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). The assembly was optimized utilizing only k-mer coverages consistent with the bacterial genome sequence, thereby further reducing the murine read contamination. The final Velvet assembly parameters consisted of an expected coverage of 50, minimum coverage of 25, maximum coverage of 70, and an insert length of 230. Eleven contiguous sequences of assembled reads (contigs) with similar depth of coverage were collected from Velvet, and nucleotide comparisons were made to the NCBI genome database to ensure that contigs were not derived from the mouse genome.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/321/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Frequency of coverage for Velvet assembly with hash length 31. The first peak in the bimodal distribution (&#x201C;1&#x201D;) represents the murine contigs that had lower coverage due to the larger genome size. The smaller peak (&#x201C;2&#x201D;) represents the frequency of assembled contigs originating from P. abortibovis. These contigs were collected for further assembly with the average coverage parameter set to 50." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-333229021" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Frequency of coverage for Velvet assembly with hash length 31. The first peak in the bimodal distribution (&#x201C;1&#x201D;) represents the murine contigs that had lower coverage due to the larger genome size. The smaller peak (&#x201C;2&#x201D;) represents the frequency of assembled contigs originating from &lt;em&gt;P. abortibovis&lt;/em&gt;. These contigs were collected for further assembly with the average coverage parameter set to 50.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/321/F1.medium.gif" width="440" height="186"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/2/321/F1.medium.gif" width="440" height="186"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/321/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/321/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470475" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-13" class="first-child">Frequency of coverage for Velvet assembly with hash length 31. The first peak in the bimodal distribution (“1”) represents the murine contigs that had lower coverage due to the larger genome size. The smaller peak (“2”) represents the frequency of assembled contigs originating from <em>P. abortibovis</em>. These contigs were collected for further assembly with the average coverage parameter set to 50.</p><div class="sb-div caption-clear"></div></div></div><p id="p-14">Mate-pair library reads were aligned to the Velvet contigs using Bowtie2, and the resulting sequence alignment map (SAM) output was used for manual contig scaffolding. To determine scaffold size and validate contig orientation, the stand-alone scaffolder SSpace (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Boetzer <em>et al.</em> 2011</a>) was used. The scaffolded gaps from the assembled contigs were decreased by the stand-alone gap filling program GapFiller (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Nadalin <em>et al.</em> 2012</a>). Correct contig orientation and position within the circular genome was determined by mate-pair mapping events, where one mate mapped to one contig and the other mate mapped to a different contig. The position and orientation of the contigs was confirmed and unassigned base pairs were utilized to fill the sequence gaps.</p><p id="p-15">All contiguous sequences were compared using Blast2Seq (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Tatusova and Madden 1999</a>) to visualize the consistency of assemblies at each stage using different assembly parameters.</p></div><div id="sec-6" class="subsection"><h3>Sanger sequencing</h3><p id="p-16">Unassigned base pairs remaining at the end of the <em>de novo</em> assembly were visualized using Geneious (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Kearse <em>et al.</em> 2012</a>) and segments of DNA were identified that contained regions of assembled sequence with multiple unassigned base pairs (N’s). PCR primers (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>) were designed to span these blocks of unassigned base pairs for an optimal amplicon size of 450 bp using the program Primer3 (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Untergasser <em>et al.</em> 2012</a>). An additional nested primer was designed for one fragment that had a larger amplicon size, and the nested PCR product was utilized for Sanger sequencing.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470264/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470264/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Sequences of primer pairs for Sanger sequencing</span><div class="sb-div caption-clear"></div></div></div><p id="p-18">For PCR analysis, two DNA templates that had been isolated from <em>P. abortibovis</em> were utilized (60 and 400 ng). One microliter of each DNA template was amplified using GoTaq Green Master Mix (Promega, Madison, WI). Cycling conditions consisted of an initial 95° denaturation for 2 min, followed by 35 cycles of 95° for 30 sec, 58° for 30 sec, and 72° for 30 sec, and a final extension of 72° for 5 min. PCR products (15 µl) were purified using the QIAquick PCR purification kit (QIAGEN, Valencia, CA).</p><p id="p-19">Sanger sequencing was performed by Davis Sequencing on an Applied Biosystems 3730 DNA Analyzer (Life Technologies). Chromatogram visualization and trimming of DNA sequence were performed with ChromasPro software (Technelysium, Queensland, Australia) and curation of a consensus sequence through visualization was performed using Geneious software (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Kearse <em>et al.</em> 2012</a>).</p><p id="p-20">The chromatograms displayed a pattern in which overlapping peaks were present following a stretch of homopolymeric base pairs. The template with the largest signal for each region was selected as the consensus sequence and was compared to the NGS sequence reads. This is indicative of “enzyme slippage” that commonly occurs during clonal amplification, such as with Illumina HiSeq NGS sequencing. Visualization of the NGS sequencing assembly supported this hypothesis because the coverage level dropped significantly and sequencing reads were shorter at the homopolymeric sections, likely as a result of the polymerase not being able to extend through these regions. Although it was not possible to confirm the exact number of nucleotides in the homopolymers, evidence-based predictions were used to close the homopolymeric gaps in the assembly.</p></div><div id="sec-7" class="subsection"><h3>Annotation</h3><p id="p-21">The assembled genome for <em>P. abortibovis</em> was annotated using the Rapid Annotation using Subsystem Technology (RAST) (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Aziz <em>et al.</em> 2008</a>). RAST is a fully automated annotation engine designed for annotation of bacterial genomes. Any gene candidate without a function was further searched for similarity with the top 30 closest phylogenetic relatives using BLASTP. Gene candidates that were &lt;90 bp in length were removed from further analyses.</p></div><div id="sec-8" class="subsection"><h3>Comparative genome analysis</h3><p id="p-22">Comparative sequence analysis was performed between <em>P. abortibovis</em> and <em>S. cellulosum</em> using the “sequence-based comparison tool” in the SEED viewer (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Overbeek <em>et al.</em> 2014</a>), which is a BLAST-based tool to compare two genomes. The functional comparison was performed using the “function-based comparison tool” in SEED viewer, which assesses the presence and absence of functional roles that have been linked to subsystems in RAST. The number of genes contributing to each functional category was quantified and the percent loss of genes contributing to that functional category was computed from <em>S. cellulosum</em> to <em>P. abortibovis</em>.</p><p id="p-23">Comparative analyses of specific genes present in <em>P. abortibovis</em> from <em>Myxococcus xanthus</em>, <em>S. cellulosum</em>, and <em>Lawsonia intracellularis</em> were performed using BLAST2seq. To perform the analysis, the genes were extracted from the PATRIC database and nucleotide-based alignment was performed against the assembled <em>P. abortibovis</em> genome. Subsequent amino acid (AA) alignment was performed to identify the presence of proteins with AA sequence variants.</p></div><div id="sec-9" class="subsection"><h3>Reverse vaccinology</h3><div id="sec-10" class="subsection"><h4>Gene prediction:</h4><p id="p-24">Initial protein coding gene prediction was performed by the RAST annotation pipeline, which utilizes Glimmer3 (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Delcher <em>et al.</em> 2007</a>). Gene prediction was also performed using Prodigal (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hyatt <em>et al.</em> 2010</a>), GeneMarkS (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Besemer <em>et al.</em> 2001</a>), and the NCBI prokaryotic genome annotation pipeline (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Tatusova <em>et al.</em> 2013</a>, <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">2016</a>), to test the effect of gene prediction on candidate selection. Thereafter, the reverse vaccinology protocol described below was carried out with proteins predicted by all four gene prediction programs.</p></div><div id="sec-11" class="subsection"><h4>Vaxign:</h4><p id="p-25">A comprehensive reverse vaccinology program, Vaxign (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">He <em>et al.</em> 2010</a>), was utilized for analysis alongside supplementary stand-alone programs. The pipeline includes protein subcellular localization (SCL), transmembrane (TM) helices, adhesin probability, and protein conservation to human, pig, and mouse proteins. Further description of individual programs utilized in Vaxign are in subsequent sections.</p></div><div id="sec-12" class="subsection"><h4>SCL:</h4><p id="p-26">SCL of gram-negative bacteria predicts whether proteins are localized to the cytoplasm, inner membrane, periplasm, outer membrane, or are extracellular (<em>i.e.</em>, secreted). The predictions were determined using PSORTb2.0 in Vaxign and three additional stand-alone software programs: PSORTb3.0 (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Yu <em>et al.</em> 2010</a>), SOSUI-GramN (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Imai <em>et al.</em> 2008</a>), and CELLO (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Yu <em>et al.</em> 2006</a>). No difference was observed between PSORTb2.0 and PSORTb3.0, so results from PSORTb2.0 were excluded from further analysis. A score of 1 was assigned for prediction of outer membrane or extracellular, 0.5 for periplasm, and 0 for cytoplasm and inner membrane. When more than one SCL was predicted, they were scored by summing individual scores (<em>S</em>) and dividing by the number of SCLs predicted (<em>n</em>) <em><span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/321/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/321/embed/mml-math-1.gif"/></noscript></span></span></em> The score for each of the three SCL programs was then summed for final a SCL score for each protein.</p></div><div id="sec-13" class="subsection"><h4>TM helix:</h4><p id="p-27">The Vaxign suite of tools utilizes the PROFtmb (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bigelow <em>et al.</em> 2004</a>), a profile-based hidden Markov model-based software program to predict the number of TM helices in a bacterial protein. A maximum threshold of one TM domain is permitted for vaccine candidates.</p></div><div id="sec-14" class="subsection"><h4>Protein conservation:</h4><p id="p-28">OrthoMCL (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Li <em>et al.</em> 2003</a>) is used by Vaxign to compare the sequence against eukaryotic organisms for protein conservation between each protein and human, mouse, and pig proteins. The default <em>E</em>-value was set to 10<sup>−5</sup> and was also utilized for BLASTP (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Altschul <em>et al.</em> 1990</a>) analysis against the bovine and <em>S. cellulosum</em> genomes. Conservation in eukaryotes and nonpathogenic prokaryotes was used to exclude any conserved protein from being a candidate for a recombinant vaccine. Two of the P. <em>abortibovis</em> candidates were removed for being highly conserved in eukaryotes. Four candidates were filtered based on homology with <em>S. cellulosum</em>.</p></div><div id="sec-15" class="subsection"><h4>Adhesin prediction:</h4><p id="p-29">Vaxign utilizes the program SPAAN (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Sachdeva <em>et al.</em> 2005</a>) for adhesin probability prediction and has a default value of 0.5 for the threshold. SPAAN utilizes the five protein attributes of adhesins, which include the AA frequencies, multiplet frequencies, dipeptide frequencies, charge composition, and hydrophobic composition, to determine the probability that a protein is an adhesin.</p><p id="p-30">An adhesin is a bacterial cell surface component that facilitates adhesion to other cells and is essential for bacterial pathogenesis. Many adhesins are being evaluated for their ability to induce protective immunity. The program SPAAN that is carried out by Vaxign has been validated to be 97.4% accurate, 89% sensitive, and 100% specific at threshold 0.5 (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Sachdeva <em>et al.</em> 2005</a>). Therefore, this criterion was used to filter candidates. Only 2.7% (59 of 2174) of the total genes met the threshold and hence this step reduced the <em>P. abortibovis</em> candidates by almost two thirds, from 29 candidates to 10. After subcellular localization, the adhesin probability was the selection criterion that eliminated the most candidates.</p></div><div id="sec-16" class="subsection"><h4>Protective antigenicity:</h4><p id="p-31">VaxiJen (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Doytchinova and Flower 2007</a>) is a software program that predicts protective antigenicity through an alignment free approach based on AA properties. The default cutoff for protective antigenicity prediction is a probability of 0.4. However, a cutoff of 0.5 is optimal for bacterial prediction.</p></div><div id="sec-17" class="subsection"><h4>RNA sequencing analysis:</h4><p id="p-32">RNA sequencing reads were mapped to the assembled <em>P. abortibovis</em> genome using stringent parameters in Bowtie2 (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Langmead and Salzberg 2012</a>). These parameters allowed one mismatch in the 100 bp read. SAM files were used for input to the visualization program GenomeBrowse (<a href="http://www.goldenhelix.com">www.goldenhelix.com</a>, 2.0.0) and FPKM (fragments per kilobase of transcript per million mapped reads) values were obtained using Cufflinks (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Trapnell <em>et al.</em> 2013</a>).</p></div><div id="sec-18" class="subsection"><h4>Selection criteria:</h4><p id="p-33">The selection criteria utilized for vaccine candidate prediction were: (a) a subcellular localization score of ≥2, (b) ≤1 TM helix, (c) not conserved in any eukaryotes (human, mouse, pig, or cow), (d) not conserved in <em>S. cellulosum</em>, (e) adhesin probability &gt;0.5, and (f) protective antigenicity prediction &gt;0.5. RNA sequence expression with an FPKM &gt; 0 was used for candidate ranking but absence of expression was not used as a criterion for elimination of candidates. The most highly transcribed gene, <em>BCY86_01995</em>, was expressed at a level that was an order of magnitude higher than the next most highly transcribed gene, <em>BCY86_08695</em>.</p></div></div><div id="sec-19" class="subsection"><h3>Immunogenicity testing</h3><p id="p-34">Protein candidates were expressed by GenScript (Piscataway, NJ) utilizing <em>Escherichia coli</em> as an expression host. Codon optimization was performed for optimized expression of protein candidates. Proteins were purified using a 6x His tag. Protein preparations were mixed with sample application buffer lacking 2-mercaptoethanol (2-ME), loaded onto precast 4–20% Mini-Protean TGX gels (BioRad, Hercules, CA) at concentrations ranging from 0.1 to 1.0 μg/well and transferred to PVDF membranes with the Trans-Blot Turbo Mini PVDF Transfer Packs (BioRad). Membranes were blocked with SuperBlock T20 blocking buffer (ThermoFisher Scientific) and all washes were conducted with PBS containing 0.1% Tween 20. Blotted proteins were screened against one of the following: (a) mouse anti-6x His (GenScript), (b) pooled serum from BALB/c mice inoculated with <em>P. abortibovis</em>, (c) serum from a cow following a confirmed EBA abortion, (d) an aborted EBA-positive fetus, (e) commercially available mouse sera (Gemini Biosciences, West Sacramento, CA), (f) serum from a cow with no known <em>P. abortibovis</em> exposure, or (g) commercially available fetal bovine serum (Gemini Biosciences). Sera b, c, and d are referred to as immune sera and e, f, and g as EBA-negative controls. The presence of <em>P. abortibovis</em> antibody was confirmed in serum from <em>P. abortibovis</em>-exposed animals and absence in those with no known exposure by indirect fluorescent antibody test (IFAT) as previously described (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Blanchard <em>et al.</em> 2014</a>). Antibody binding was detected with alkaline phosphatase-conjugated anti-mouse IgG (H&amp;L) (Vector Laboratories, Burlingame, CA) or anti-bovine IgG (H&amp;L) (Jackson ImmunoResearch Laboratories, Inc., West Grove, PA) as appropriate, followed by NBT/BCIP substrate (Roche Diagnostics, Indianapolis, IN). The degree of antibody binding was subjectively assigned by visual appraisal of the developed membrane with a three-level scale. The scale ranged from no detected band (−), to faint bands (±), to distinct bands (+). Binding of anti-6x His identified the bands of interest, and color intensity served as the baseline for evaluating antibody binding when probed with murine or bovine serum samples.</p></div><div id="sec-20" class="subsection"><h3>Data availability</h3><p id="p-35">Data supporting the conclusions of this article are available in GenBank, accession number <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=CP016908.&amp;atom=%2Fggg%2F7%2F2%2F321.atom">CP016908.</a> Gene names are those predicted by the NCBI Prokaryotic Genome Annotation Pipeline (Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036673/-/DC1/TableS1.xlsx">Table S1</a>). <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036673/-/DC1/TableS1.xlsx">Table S1</a> summarizes the RNA-Seq results from the genes predicted by the NCBI Prokaryotic Genome Annotation Pipeline. Raw RNA-Seq data can be accessed through the NCBI Sequence Read Archive (SRA), accession number SRP095186. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036673/-/DC1/FigureS1.tif">Figure S1</a> shows the western blot analyses for the top nine protein candidates.</p></div></div><div class="section results" id="sec-21"><h2 class="">Results</h2><div id="sec-22" class="subsection"><h3>Genome assembly</h3><p id="p-36">Sequencing of the genomic library yielded a total of 243,271,075 100 bp paired-end (PE100) reads consisting of a combination of both mouse and bacterial DNA. To increase the concentration of bacterial reads within our pool of total reads, all reads were aligned to the mouse genome (mm10) (Mouse Genome Sequencing Consortium 2002) and the unmapped putative bacterial reads were collected for further analysis. Despite efforts to enrich for bacterial DNA, which included a variety of DNA extraction methods and cell filtering, 98.4% of the reads from the single-cell suspension mapped to the murine genome leaving only 4,030,414 (1.6%) nonmurine unmapped paired-end reads.</p><p id="p-37">Sequences that did not map to the mouse genome were used as input for the Velvet de Bruijn graph assembler. The optimal assembly, defined as the largest N<sub>50</sub>, for the dataset was a k-mer length of 31 and k-mer coverage equal to ∼50 (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). The optimized Velvet assembly yielded 11 contigs with an average N<sub>50</sub> of 400,733 bp, max contig length of 683,082 bp, and total assembly of 1,815,859 bp (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). The resulting contigs were compared to the NCBI nucleotide database using nblast, and 10 of 11 contigs were found to be most similar to organisms of the class Deltaproteobacteria. One contig was most similar to the mouse genome, suggesting that mouse sequences remained in the data, and this contig was removed from further analysis.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470425/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470425/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Assembly statistics for contiguous sequences at each stage of genome assembly</span><div class="sb-div caption-clear"></div></div></div><p id="p-39">The 10 contigs were manually scaffolded by aligning all 23 million PE100 mate-pair reads (2675 bp insert size) to the assembled Velvet contigs using Bowtie2 and parsing the SAM file to pull out the 1725 mate-pair reads that aligned to two separate contigs. Once it was determined that all of the contigs were present to circularize the genome, the stand-alone scaffolder SSpace was used to confirm the manually curated scaffolds. The 63,642 (0.27%) mate-pair reads that aligned to the Velvet contigs were collected and utilized alongside the original paired-end reads that did not map to the murine genome for SSpace. The scaffolded assembly resulted in one contiguous 1,821,593 bp circular genome starting at the origin of replication (TTATCCACA). The scaffolded genome contained 1741 unknown bp, which equates to 0.096% of the genome or approximately one unknown nucleotide in every 1050 bp. Therefore, the assembled genome meets the standards for NCBI genome completion, which is one unknown nucleotide in each 1000 bp. A supplementary gap filling program, GapFiller, was used to decrease the number of unassigned base pairs and it was successful in producing a 1,821,632 bp genome with 56 unknown bp (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>).</p><p id="p-40">The resulting genome contained three gaps consisting of multiple continuous unknown base pairs (24, 16, and 8 bp). These gaps were amplified using PCR and sequenced by Sanger sequencing. Each of the gaps was the result of a long homopolymeric stretch of DNA sequence which interfered with the Velvet assembly. Through the use of Sanger sequencing and visualization, the genome was further improved to a final size of 1,821,581 bp with 6 unknown bp (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). This sequence has been deposited in GenBank (Accession Number: <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=CP016908&amp;atom=%2Fggg%2F7%2F2%2F321.atom">CP016908</a>).</p></div><div id="sec-23" class="subsection"><h3>Genome annotation</h3><p id="p-41">The Rapid Annotation using Subsystem Technology (RAST) pipeline resulted in the prediction of 2250 coding sequences that were assigned to 229 subsystems (a set of gene functions that are grouped together to implement a specific biological process). There were 47 noncoding RNAs predicted, with 44 of them being tRNAs and one copy each of 5S, 16S, and 23S ribosomal RNAs. The 2250 genes covered 1,544,122 bp of the 1,821,632 bp assembled, which equates to 84.8% of the genome covered by coding genes. The average length of genes predicted was 687 bp and ∼48.5% of the genes were identified from RNA-Seq data of spleen tissue from <em>P. abortibovis</em>-infected SCID mice. The initial prediction was only able to assign a putative function to 942 (42%) of the predicted genes and subsequent BLAST searches were able to assign putative functions to an additional 34 genes for a total of 976 (43%) genes with an assigned function. Hypothetical predicted proteins were generally shorter in length with an average gene length of 378 bp, and of these only 25% were represented in the RNA-Seq data. Of the 954 hypothetical genes that were not expressed, the average gene length was 218 bp (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470218/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470218/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Number of predicted proteins, gene length, and percent expression in predictions from the RAST gene prediction program</span><div class="sb-div caption-clear"></div></div></div><p id="p-43">The protein sequence-based comparison of <em>P. abortibovis</em> and <em>S. cellulosum</em>, the closest nonpathogenic phylogenetic relative to <em>P. abortibovis</em>, showed that 1143 (51%) of the predicted proteins shared &gt;20% AA sequence homology, 465 (21%) of the predicted proteins shared &gt;50% AA homology, and 14 (0.6%) of the predicted proteins shared &gt;80% homology. RAST designates each gene into a hierarchy of categories starting with a specific gene function, which is in a subsystem, that is then in a subcategory and a category. As a result, multiple open reading frames (ORFs) may have the same functional annotation. Of the total 666 <em>P. abortibovis</em> gene functions, 620 (93%) were shared between the two organisms and only 46 functions were identified in <em>P. abortibovis</em> but not in <em>S. cellulosum</em>. The myxobacterium <em>S. cellulosum</em> had 667 unique functions that were not present in <em>P. abortibovis</em> (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/321/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of functional group reduction between S. cellulosum and P. abortibovis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-333229021" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Analysis of functional group reduction between &lt;em&gt;S. cellulosum&lt;/em&gt; and &lt;em&gt;P. abortibovis&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/321/F2.medium.gif" width="440" height="223"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/2/321/F2.medium.gif" width="440" height="223"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/321/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/321/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470473" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-44" class="first-child">Analysis of functional group reduction between <em>S. cellulosum</em> and <em>P. abortibovis</em>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-45"><a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a> displays the total number of ORFs for each subsystem grouped by functional category. This enables the calculation of the percentage of total genes lost in the evolution of <em>S. cellulosum</em> to <em>P. abortibovis</em> for each functional category. Overall, there was a 73% loss of genes in <em>P. abortibovis</em>. The percentage of genes lost (and gained) from each functional category varied widely, ranging from gaining one gene in potassium metabolism to losing all of the genes (100% loss) associated with three functional categories [(1) Phages, Prophages, Transposable elements, and Plasmids; (2) Regulons; and (3) Metabolism of Aromatic Compounds]. Annotations were submitted to the NCBI database using the NCBI Prokaryotic Genome Annotation Pipeline.</p></div><div id="sec-24" class="subsection"><h3>Reverse vaccinology</h3><p id="p-46">All genes predicted for <em>P. abortibovis</em> were analyzed using a reverse vaccinology approach. Since prokaryotic gene prediction has been shown to be highly variable (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bakke <em>et al.</em> 2009</a>), four gene prediction programs were used to analyze the <em>P. abortibovis</em> genome. Prodigal, GeneMark, Glimmer3 (RAST), and NCBI (GeneMarkS+) predicted 1767, 2174, 2250, and 1809 genes, respectively (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). A reverse vaccinology pipeline (discussed below) was used to analyze the gene sets detected by each of the gene prediction programs and the resulting vaccine candidates were subsequently compared. Ultimately, there was no significant difference in the suite of top vaccine candidates predicted using these separate gene sets. Further analysis was performed with the GeneMark gene set as it represented coverage of the largest percentage of the genome.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470496/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470496/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Summary statistics of gene prediction programs</span><div class="sb-div caption-clear"></div></div></div><p id="p-53">The first step in determining which ORFs represented proteins that might be antigenic was to evaluate their SCL. Extracellular or outer membrane ORFs are predicted to be more antigenic due to the accessibility of antibodies binding to these proteins. Initial comparisons were made using the programs PSORTb2.0 and PSORTb3.0; the results indicated a strong positive relationship between the programs with a Pearson’s correlation = 0.98. Therefore, only PSORTb3.0 was used in further comparisons with other (SOSUI and CELLO) SCL prediction programs. The SCL evaluation for the candidates predicted by GeneMark is summarized in <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>, and the predicted outer membrane and extracellular proteins are summarized by a Venn diagram in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>. Variation was observed between the three programs with a Pearson’s correlation of 0.28 between PSORTb and SOSUI, 0.37 between PSORTb and CELLO, and 0.37 between SOSUI and CELLO. The SCL prediction of PSORTb was the most stringent, predicting a total of only 24 extracellular or secreted proteins (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). A score of 1 was assigned for prediction of outer membrane or extracellular, 0.5 for periplasm, and 0 for cytoplasm and inner membrane. The scores of the three stand-alone SCL prediction programs (PSORTb3.0, SOSUI, and CELLO) were summed to obtain a final SCL score between 0 and 3. ORFs with an SCL score of &lt;2 were filtered out from further analysis. This selection criterion eliminated the most ORFs, reducing the total number of ORFS from 2174 to 36 extracellular or outer membrane ORFs.</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470441/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470441/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Top 10 reverse vaccinology candidate list from GeneMark-predicted genes</span><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/321/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of genes that were predicted to be extracellular or outer membrane by the subcellular localization programs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-333229021" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparison of genes that were predicted to be extracellular or outer membrane by the subcellular localization programs.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/321/F3.medium.gif" width="440" height="365"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/2/321/F3.medium.gif" width="440" height="365"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/321/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/321/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470269" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-60" class="first-child">Comparison of genes that were predicted to be extracellular or outer membrane by the subcellular localization programs.</p><div class="sb-div caption-clear"></div></div></div><p id="p-61">The number of TM helices was calculated for each of the extracellular or outer membrane ORFs, and one candidate of the 36 was removed from the analysis because it had two or more TM domains. Next, ORFs that shared homology with eukaryotic proteins (BLAST <em>E</em> &lt; 10<sup>−5</sup>) were removed from the analysis. Of the 2174 total ORFs, 406 (19%) were highly conserved between <em>P. abortibovis</em> and eukaryotes, including two of the remaining 35 extracellular or outer membrane candidates. Four candidates that shared further protein sequence homology with <em>S. cellulosum</em> were also removed from subsequent analysis. Finally, the adhesin probability was computed for all ORFs with only 59 out of 2174 (2.7%) ORFs predicted to be adhesins (<em>P</em><sub>ad</sub> &gt; 0.5). Only 2.7% (59 of 2174) of the total genes met the threshold and hence this step reduced the <em>P. abortibovis</em> candidates by almost two-thirds, from 29 candidates to 10.</p><p id="p-62">The protective antigenicity of each candidate was computed with the VaxiJen score and was filtered on a predictive score of 0.5. The default score of 0.4 was too liberal, with 1487 of 2174 (68%) ORFs predicted to be protective antigens, whereas a cutoff of 0.5 predicted 890 of 2174 (41%) candidates.</p><p id="p-63">The annotated biological functions of these proteins were evaluated to determine if they had sequence homology to proteins known to be antigenic. The top 10 candidates were predicted to be hypothetical proteins with no significant homology to proteins of known function. The last parameter that was used in the selection of vaccine candidates was the FPKM value obtained from RNA-Seq analysis. Nine of the 10 candidates were expressed in our RNA-Seq results (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>) and these FPKM values were used to rank among these top nine candidates.</p></div><div id="sec-25" class="subsection"><h3>Immunological testing</h3><p id="p-64">Nine of 10 vaccine candidate proteins were successfully expressed as 6 × His fusion proteins in <em>E. coli</em>. Protein <em>BCY86_08860</em> was not successfully expressed and was removed from further analysis. Mouse anti-6 × His bound eight of nine expressed proteins (<a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a>). One protein, <em>BCY86_03205</em>, was not detected by immunoblotting when probed with 6 × His antibody and was removed from further analysis. One protein (<em>BCY86_06555</em>) reacted only to serum from control mice, and will not be discussed further. The remaining seven candidate proteins were bound by serum from EBA-infected mice. Three protein candidates, <em>BCY86_00790</em>, <em>BCY86_01995</em>, and <em>BCY86_08695</em>, were specifically bound by serum derived only from EBA-infected mice. Three additional proteins, <em>BCY86_06335</em>, <em>BCY86_03255</em>, and <em>BCY86_08690</em>, were more strongly bound by the serum from EBA-infected mice than the serum from the control mice. Protein <em>BCY86_06085</em> was equally bound by serum derived from both the positive EBA-infected mice and the control mice, and appeared to have an additional protein band bound at ∼100 kDa of unknown origin.</p><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470351/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470351/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6</span> <span class="caption-title">Immunological testing through western blot analysis on the top nine protein candidates</span><div class="sb-div caption-clear"></div></div></div><p id="p-68">Results using serum from mature cows were similar to those of the mouse with the exception of <em>BCY86_08695</em>, which showed no binding to serum from EBA-infected or control cows. Protein <em>BCY86_01995</em> displayed specific binding to serum from only the EBA-infected cow and not the control. Proteins <em>BCY86_00790</em>, BCY86_03255, <em>BCY86_06085</em>, and <em>BCY86_06335</em> were more strongly bound by the serum from the EBA-infected cow than the serum from the control cow. Protein BCY86_08690 showed equivalent binding to serum from both the EBA-infected and the negative cow, suggesting it was not specific.</p><p id="p-69">The serum from EBA-infected fetuses specifically bound proteins <em>BCY86_01995</em>, <em>BCY86_06085</em>, and <em>BCY86_00790</em> while serum from control fetuses did not, whereas <em>BCY86_03255</em> and <em>BCY86_08690</em> showed potential antigenic responses indicated by a weak response in serum from the EBA-positive fetus and no binding to serum from the negative fetus.</p><p id="p-70">In summary, protein <em>BCY86_01995</em> was the most promising candidate, displaying a strong specific immune response in EBA-infected mouse, cow, and bovine fetal serum. Protein <em>BCY86_00790</em> specifically reacted with EBA-infected mouse and bovine fetal serums only and reacted more strongly with infected cow serum than the control cow serum suggesting it is also a good vaccine candidate. Protein <em>BCY86_03255</em> also displayed stronger antibody binding with the EBA-infected serums compared to control serums of both murine and bovine origin.</p></div></div><div class="section discussion" id="sec-26"><h2 class="">Discussion</h2><p id="p-71">The <em>de novo</em> assembly of the <em>P. abortibovis</em> genome was complicated by the presence of eukaryotic DNA from the spleen cells of the SCID mice in which the bacteria were grown. Although the single-cell suspension was enriched for bacterial cells, the bacterium resides intracellularly and cannot be purified from the mouse DNA. We hypothesized that deep coverage NGS and bioinformatics analysis would overcome the eukaryotic DNA contamination and enable the assembly of the complete bacterial genome.</p><p id="p-72">The completion of the circularized Deltaproteobacterial genome confirmed this hypothesis. The resulting sequence showed that <em>P. abortibovis</em> belongs to the order Myxococcales, which includes species with the largest known bacterial genomes. The closest phylogenetic relative to <em>P. abortibovis</em> that has been sequenced is <em>S. cellulosum</em>, a myxobacterium that resides in the soil and has the largest known bacterial genome at 13 Mb. Although these organisms have highly divergent lifestyles, comparative functional analysis revealed high levels of conservation among functional roles as well as protein, AA, and DNA sequences between the two species. Although the assembled <em>P. abortibovis</em> genome is &lt;&lt;13 Mb, research indicates that when bacterial lineages transition from free-living organisms to permanently living in a host (<em>i.e.</em>, obligate pathogens), they rely on the host environment for many essential proteins and thereafter substantially reduce their genome size (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Moran and Plague 2004</a>).</p><p id="p-73"><a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Moran (2002)</a> established that the compounds of intermediate metabolism and AAs can be obtained from host cells and thus are a clear basis for genome reduction in host-associated bacteria. This can clearly be seen in <em>P. abortibovis</em>. When the functional groups (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>) contributing to intermediate metabolism (potassium metabolism, iron metabolism, secondary metabolism, nitrogen metabolism, aromatic metabolism, sulfur metabolism, and phosphorus metabolism) are summed, <em>S. cellulosum</em> has 172 functional roles whereas <em>P. abortibovis</em> has only 12, a 93% reduction in the genes that code for intermediate metabolism. This reduction far exceeds that observed in any other functional group. The genes that benefit free-living organisms and their need to survive in extreme environmental conditions, such as regulatory elements, are also lost. This phenomenon is supported in <em>P. abortibovis</em> with a 94% reduction in the genes responsible for regulatory elements and cell signals. DNA repair enzymes are reduced in <em>P. abortibovis</em> by 54%, from 58 to 27. The loss of DNA repair enzymes has a substantial impact on the genome size reduction and the adenine (A)–thymine (T) bias observed in most host-associated genomes.</p><p id="p-74">In bacteria, genome size and G–C content are correlated (<em>r</em> = 0.46) (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Nishida 2012</a>) and the obligate host-associated bacteria contain genomes with relatively lower G–C content. The genomes with the highest observed A–T enrichment are bacterial symbionts with extremely small genomes. For example, the Candidatus <em>Zinderia insecticola</em> has a 209 kbp genome with a G–C content of only 13.5% (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">McCutcheon and Moran 2012</a>). This relationship is also observed when comparing <em>P. abortibovis</em> (47.2% G–C content) and <em>S. cellulosum</em> (72% G–C content). The existence of an inherent mutational bias from G–C to A–T was observed experimentally and can likely be attributed to DNA damage, such as cytosine deamination and guanine oxidation, which results in G–C being converted to A–T. This further suggests the inability of <em>P. abortibovis</em> to survive outside a host environment. The phenomenon of genome reduction and A–T bias is further supported with the most closely related pathogen to <em>P. abortibovis</em>, <em>L. intracellularis</em>, which contains 1.72 Mbp of genomic DNA. This Deltaproteobacterium contains a 1.46 Mbp circular genome (33% G–C content) and three plasmids that are 27.1, 39.9, and 194 kbp (29, 29, and 33% G–C content, respectively) in size (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Sait <em>et al.</em> 2013</a>).</p><p id="p-75">Although reduced genomes do not have a universal set of genes that are maintained, there are patterns associated with the types of genes that are lost (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Moran 2002</a>). Of the many genomic sequences that have been assembled, only obligate intracellular pathogens and symbionts have a stronger evolutionary force with genome reduction than horizontal gene transfer (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Lawrence 2005</a>). Therefore, it is reasonable to hypothesize that <em>P. abortibovis</em> has evolved to become an obligate eukaryotic host-associated organism that does not contain an environmental reservoir outside eukaryotic organisms.</p><p id="p-76">Understanding the evolution of microbes and the pathogenesis of disease can guide vaccine candidate selection. Reverse vaccinology is the utilization of genomic sequence to make <em>in silico</em> predictions of vaccine candidates. This process has evolved to further refine the selection criteria for a smaller group of candidates to be experimentally tested in immunologic assays. The <em>in silico</em> processing is dependent on the genes predicted and their resulting AA sequences. High variation is generally observed with the prediction of ORFs from a raw nucleotide sequence.</p><p id="p-77">In this study, although the overall variation between predicted genes was high across each gene prediction program (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>), the relative variation was minimized after the prediction of vaccine candidates (<a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a>). Slight differences observed in the prediction of vaccine candidates with each set of genes did not ultimately change the candidates identified by reverse vaccinology.</p><p id="p-78">The main source of candidate filtering in reverse vaccinology was identification of outer membrane and secreted proteins from the prediction of SCL. Selection criteria was stringent to ensure that the properties encoded by the AA sequence represented a true secreted or outer membrane protein. The <em>in silico</em> identification of outer membrane proteins was successfully utilized for antigen discovery in the intracellular Deltaproteobacterial pathogen <em>L. intracellularis</em> (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Watson <em>et al.</em> 2011</a>). Although these proteins were discovered through proteomic analysis, the results of this study support the utilization of subcellular localization to identify antigenic proteins in <em>P. abortibovis</em>.</p><p id="p-79">The next criterion for selection was one TM domain or less for ease of protein expression. The first reverse vaccinology experiment was performed on <em>Neisseria meningitidis</em> serogroup B (MenB), which only utilized <em>in silico</em> analysis for SCL. This experiment attempted to express the proteins encoded by 570 genes and found that the number of TM domains was inversely correlated with the ability to express the protein (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Pizza <em>et al.</em> 2000</a>). This parameter has been adopted for further RV projects and resulted in the removal of one candidate ORF in this study.</p><p id="p-80">The sequence similarity of predicted candidates to proteins in other organisms is an important parameter to consider for vaccine development. The candidate should not have sequence similarity with the host to prevent autoimmunity and to ensure that the host recognizes the protein as foreign so that it can mount an immune response (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">He <em>et al.</em> 2010</a>). Protective antigens generally do not share homology with proteins of any eukaryotic or highly conserved genes.</p><p id="p-81">Reverse vaccinology has proven beneficial for organisms with a high degree of strain divergence because homology analysis can filter proteins likely to display strain-specific immunity (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Pizza <em>et al.</em> 2000</a>). Therefore, the ideal candidate would be conserved in all pathogenic strains and be nonhomologous to any proteins in commensal strains. However, strain-specific proteins can still be included in cocktail vaccines if they trigger a significant immune response in that strain. This has been seen with the inclusion of two proteins (GNA1030 and GNA2091) in the Bexsero vaccine for meningococcal B (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Giuliani <em>et al.</em> 2006</a>).</p><p id="p-82">Protective immunogenicity is the primary goal for any subunit vaccine. The host immune system can mount an adaptive immune response that recognizes a vast spectrum of microbial immunogens with no apparent host-protective antigenicity. Therefore, vaccine design targets those antigens that can induce immune responses that are protective. In this study, the program VaxiJen was used to supplement the other RV programs to better predict whether the candidates will result in a protective immune response.</p><p id="p-83">The FPKM obtained from the Cufflinks analysis was utilized as the main ranking criterion for protein expression. The presence of RNA expression from the candidates also confirms that these predicted proteins are not artifacts of the gene prediction algorithms. Visualization of the RNA-Seq reads also allows for more precise gene boundary assignment and possible reassignment of start and stop sites.</p><p id="p-84">Of the 10 final candidates, one of the proteins was located on the outer membrane and the rest were predicted to be secreted. The probability of a protein being an adhesin was relatively consistent between the candidates (∼0.6) but the protective antigenicity score was highly variable (0.54–1.44), allowing for ranking of candidates based on their protective antigenicity score. Finally, candidates were evaluated by their FPKM value, which designates the level of <em>in vivo</em> murine transcription for that gene. The RNA-Seq experiment was conducted in SCID mice and might not represent the expression levels in the target host (pregnant cow) during infection. The top candidate represents a gene that is highly expressed, being among the top 15 expressed ORFs. This transcript has even coverage of RNA-Seq reads spanning the length of the transcript, indicating that the gene coordinates were properly assigned.</p><p id="p-85">The screening of vaccine candidates against serums of EBA-infected mice and negative control sera provides an initial understanding of the relative antigenicity of the protein candidates. Evaluation of antigen binding to antibodies derived from EBA-infected mice provided a broad picture of the potential for these proteins to induce an immune response, with seven of nine expressed proteins displaying a positive reaction (<a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table 5</a>). The mouse provides the widest range antibody profile, possibly due to the fact that the antibodies are representative of the immune responses from a pool of four mice. The two top candidates from reverse vaccinology, <em>BCY86_01995</em> and <em>BCY86_08695</em>, were bound by murine-derived antibodies. Four additional proteins showed binding to murine-derived antibodies, but some binding to antibodies from naïve control mice was evident. One protein displayed equal binding with the positive and negative serums. This indicates that there is a cross-reactive response of the protein with potential epitopes to other microorganisms.</p><p id="p-86">The antibody-binding profile of the bovine serum better reflects the true potential of these candidate proteins for a recombinant vaccine. The antibody-binding profile was similar between the mouse and the bovine sera with the exception of <em>BCY86_08695</em>, which showed no binding to bovine immune serum antibodies. Protein <em>BCY86_01995</em> demonstrated specific binding to antibody from bovine immune serum. The remaining proteins were bound by antibodies from both immune and naïve cattle, though not necessarily to the same degree. Four proteins were more strongly bound by antibodies from immune bovine serum as compared to the naïve counterpart. These apparent nonspecific reactions can likely be attributed to cross-reactivity of the recombinant proteins with other bacteria. Cows are exposed to many pathogenic and nonpathogenic microorganisms, and therefore have a broad antibody profile. Serum antibodies from aborted fetuses bound three proteins (<em>BCY86_00790</em>, <em>BCY86_01995</em>, and <em>BCY86_06085</em>) with specific antigenic responses and displayed minimal binding with two additional proteins (<em>BCY86_03255</em> and <em>BCY86_08690</em>). The negative control fetal serum did not show binding with any recombinant proteins. This is likely due to the absence of microbe-specific antibodies in a healthy fetus.</p><p id="p-87">The functional immunological testing of these protein candidates provides an initial validation of the reverse vaccinology <em>in silico</em> vaccine candidate prediction approach. The demonstration of an antibody response only addresses the humoral arm of the immune system and evidence of immunogenicity but does not validate actual protection. Robust antibody responses to intracellular pathogens often do not equate to protection. Further immunogenicity studies (ability to induce a specific antibody response in the host) need to be performed to determine the potential of these proteins to be viable as a recombinant vaccine. The results of this study indicate that the seven proteins displaying antigenic responses are promising targets for further studies in the development of a protective subunit recombinant vaccine for EBA.</p><p id="p-88">In summary, protein <em>BCY86_01995</em> displayed specific binding of antibodies derived from all three immune serum types (mature bovine, fetal bovine, and murine), providing evidence for designation as an EBA-specific antigen in the target bovine species. <em>In silico</em> analysis of this protein through the reverse vaccinology pipeline clearly ranked this protein as one of the top two vaccine candidates. Protein <em>BCY86_01995</em> was ranked as the most promising candidate based on RNA-Seq results, which revealed an expression level that was an order of magnitude higher than the next candidate in the list. The other protein ranked in the top two, <em>BCY86_08695</em>, showed a positive response with EBA-specific binding in the mouse but did not exhibit any binding in the cow serum samples. The functional immunological testing of the top two candidates from reverse vaccinology with antigenicity specifically in serum from EBA-infected mice supports this approach as a useful tool for immunologists to narrow their pool of potential vaccine candidates.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-89">This research was funded by grants from the Russell L. Rustici Rangeland and Cattle Research Endowment to the University of California, Davis.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-19"><p id="p-90">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036673/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036673/-/DC1</a>.</p></li><li class="fn" id="fn-20"><p id="p-91"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-10-24"><span class="received-label">Received </span>October 24, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-12-23"><span class="accepted-label">Accepted </span>December 23, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Welly <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.321.1" data-doi="10.1006/jmbi.1990.9999"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gish</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipman</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Basic local alignment search tool.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">215</span>: <span class="cit-fpage">403</span>–<span class="cit-lpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DAltschul%26rft.auinit1%253DS.%2BF.%26rft.volume%253D215%26rft.issue%253D3%26rft.spage%253D403%26rft.epage%253D410%26rft.atitle%253DBasic%2Blocal%2Balignment%2Bsearch%2Btool.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1990.9999%26rft_id%253Dinfo%253Apmid%252F2231712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1990.9999&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2231712&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990ED16700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.321.2" data-doi="10.1177/104063870601800110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kennedy</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blanchard</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barbano</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chiu</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Histochemical and immunohistochemical evidence of a bacterium associated with lesions of epizootic bovine abortion.</span> <abbr class="cit-jnl-abbrev">J. Vet. Diagn. Invest.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">76</span>–<span class="cit-lpage">80</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVeterinary%2BDiagnostic%2BInvestigation%26rft.stitle%253DJournal%2Bof%2BVeterinary%2BDiagnostic%2BInvestigation%26rft.aulast%253DAnderson%26rft.auinit1%253DM.%2BL.%26rft.volume%253D18%26rft.issue%253D1%26rft.spage%253D76%26rft.epage%253D80%26rft.atitle%253DHistochemical%2Band%2BImmunohistochemical%2BEvidence%2Bof%2Ba%2BBacterium%2BAssociated%2Bwith%2BLesions%2Bof%2BEpizootic%2BBovine%2BAbortion%26rft_id%253Dinfo%253Adoi%252F10.1177%252F104063870601800110%26rft_id%253Dinfo%253Apmid%252F16566260%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToic3B2ZGkiO3M6NToicmVzaWQiO3M6NzoiMTgvMS83NiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi8zMjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.321.3" data-doi="10.1186/1471-2164-9-75"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aziz</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bartels</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Best</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DeJongh</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Disz</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The RAST Server: rapid annotations using subsystems technology.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">75</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DAziz%26rft.auinit1%253DR.%2BK.%26rft.volume%253D9%26rft.spage%253D75%26rft.epage%253D75%26rft.atitle%253DThe%2BRAST%2BServer%253A%2Brapid%2Bannotations%2Busing%2Bsubsystems%2Btechnology.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-9-75%26rft_id%253Dinfo%253Apmid%252F18261238%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-9-75&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18261238&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.321.4" data-doi="10.1371/journal.pone.0006291"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bakke</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carney</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DeLoache</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gearing</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ingvorsen</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Evaluation of three automated genome annotations for <em>Halorhabdus utahensis</em>.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e6291</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBakke%26rft.auinit1%253DP.%26rft.volume%253D4%26rft.issue%253D7%26rft.spage%253De6291%26rft.epage%253De6291%26rft.atitle%253DEvaluation%2Bof%2Bthree%2Bautomated%2Bgenome%2Bannotations%2Bfor%2BHalorhabdus%2Butahensis.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0006291%26rft_id%253Dinfo%253Apmid%252F19617911%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0006291&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19617911&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.321.5" data-doi="10.1093/nar/29.12.2607"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Besemer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lomsadze</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borodovsky</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">GeneMarkS: a self-training method for prediction of gene starts in microbial genomes. Implications for finding sequence motifs in regulatory regions.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">2607</span>–<span class="cit-lpage">2618</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DBesemer%26rft.auinit1%253DJ.%26rft.volume%253D29%26rft.issue%253D12%26rft.spage%253D2607%26rft.epage%253D2618%26rft.atitle%253DGeneMarkS%253A%2Ba%2Bself-training%2Bmethod%2Bfor%2Bprediction%2Bof%2Bgene%2Bstarts%2Bin%2Bmicrobial%2Bgenomes.%2BImplications%2Bfor%2Bfinding%2Bsequence%2Bmotifs%2Bin%2Bregulatory%2Bregions%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F29.12.2607%26rft_id%253Dinfo%253Apmid%252F11410670%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIyOS8xMi8yNjA3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzMyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.321.6" data-doi="10.1093/nar/gkh580"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bigelow</span>  <span class="cit-name-given-names">H. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrey</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Przybylski</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rost</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Predicting transmembrane beta-barrels in proteomes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">2566</span>–<span class="cit-lpage">2577</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DBigelow%26rft.auinit1%253DH.%2BR.%26rft.volume%253D32%26rft.issue%253D8%26rft.spage%253D2566%26rft.epage%253D2577%26rft.atitle%253DPredicting%2Btransmembrane%2Bbeta-barrels%2Bin%2Bproteomes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh580%26rft_id%253Dinfo%253Apmid%252F15141026%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjMyLzgvMjU2NiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi8zMjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.321.7" data-doi="10.1016/j.vetmic.2010.01.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blanchard</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barthold</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Serial passage of the etiologic agent of epizootic bovine abortion in immunodeficient mice.</span> <abbr class="cit-jnl-abbrev">Vet. Microbiol.</abbr> <span class="cit-vol">144</span>: <span class="cit-fpage">177</span>–<span class="cit-lpage">182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVeterinary%2Bmicrobiology%26rft.stitle%253DVet%2BMicrobiol%26rft.aulast%253DBlanchard%26rft.auinit1%253DM.%2BT.%26rft.volume%253D144%26rft.issue%253D1-2%26rft.spage%253D177%26rft.epage%253D182%26rft.atitle%253DSerial%2Bpassage%2Bof%2Bthe%2Betiologic%2Bagent%2Bof%2Bepizootic%2Bbovine%2Babortion%2Bin%2Bimmunodeficient%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.vetmic.2010.01.002%26rft_id%253Dinfo%253Apmid%252F20144513%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.vetmic.2010.01.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20144513&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.321.8" data-doi="10.1177/1040638714545506"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blanchard</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoar</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pires</span>  <span class="cit-name-given-names">A. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blanchard</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Assessment of a fluorescent antibody test for the detection of antibodies against epizootic bovine abortion.</span> <abbr class="cit-jnl-abbrev">J. Vet. Diagn. Invest.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">622</span>–<span class="cit-lpage">630</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVet.%2BDiagn.%2BInvest.%26rft_id%253Dinfo%253Adoi%252F10.1177%252F1040638714545506%26rft_id%253Dinfo%253Apmid%252F25139792%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToic3B2ZGkiO3M6NToicmVzaWQiO3M6ODoiMjYvNS82MjIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzIvMzIxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.321.9" data-doi="10.1093/bioinformatics/btq683"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boetzer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henkel</span>  <span class="cit-name-given-names">C. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jansen</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Butler</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pirovano</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Scaf-folding pre-assembled contigs using SSPACE.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">578</span>–<span class="cit-lpage">579</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq683%26rft_id%253Dinfo%253Apmid%252F21149342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNy80LzU3OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi8zMjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.321.10" data-doi="10.1177/1040638711425573"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brooks</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blanchard</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stott</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Quantitative duplex TaqMan real-time polymerase chain reaction for the assessment of the etiologic agent of epizootic bovine abortion.</span> <abbr class="cit-jnl-abbrev">J. Vet. Diagn. Invest.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1153</span>–<span class="cit-lpage">1159</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVet.%2BDiagn.%2BInvest.%26rft_id%253Dinfo%253Adoi%252F10.1177%252F1040638711425573%26rft_id%253Dinfo%253Apmid%252F22362796%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToic3B2ZGkiO3M6NToicmVzaWQiO3M6OToiMjMvNi8xMTUzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzMyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.321.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brooks</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blanchard</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clothier</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fish</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Characterization of <em>Pajaroellobacter abortibovis</em>, the etiologic agent of epizootic bovine abortion.</span> <abbr class="cit-jnl-abbrev">Vet. Microbiol.</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">73</span>–<span class="cit-lpage">80</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVet.%2BMicrobiol.%26rft.volume%253D192%26rft.spage%253D73%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.321.12" data-doi="10.1093/bioinformatics/btm009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Delcher</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bratke</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Powers</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Identifying bacterial genes and endosymbiont DNA with Glimmer.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">673</span>–<span class="cit-lpage">679</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm009%26rft_id%253Dinfo%253Apmid%252F17237039%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyMy82LzY3MyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi8zMjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.321.13" data-doi="10.1186/1471-2105-8-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doytchinova</span>  <span class="cit-name-given-names">I. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flower</span>  <span class="cit-name-given-names">D. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">VaxiJen: a server for prediction of protective antigens, tumour antigens and subunit vaccines.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">4</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DDoytchinova%26rft.auinit1%253DI.%2BA.%26rft.volume%253D8%26rft.spage%253D4%26rft.epage%253D4%26rft.atitle%253DVaxiJen%253A%2Ba%2Bserver%2Bfor%2Bprediction%2Bof%2Bprotective%2Bantigens%252C%2Btumour%2Bantigens%2Band%2Bsubunit%2Bvaccines.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-8-4%26rft_id%253Dinfo%253Apmid%252F17207271%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-8-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17207271&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.321.14" data-doi="10.1016/S0264-410X(98)00416-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ellis</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">New technologies for making vaccines.</span> <abbr class="cit-jnl-abbrev">Vaccine</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1596</span>–<span class="cit-lpage">1604</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVaccine%26rft.stitle%253DVaccine%26rft.aulast%253DEllis%26rft.auinit1%253DR.%2BW.%26rft.volume%253D17%26rft.issue%253D13-14%26rft.spage%253D1596%26rft.epage%253D1604%26rft.atitle%253DNew%2Btechnologies%2Bfor%2Bmaking%2Bvaccines.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0264-410X%252898%252900416-2%26rft_id%253Dinfo%253Apmid%252F10194811%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0264-410X(98)00416-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10194811&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000079262600004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.321.15" data-doi="10.1073/pnas.0603940103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Giuliani</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adu-Bobie</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Comanducci</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aricò</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Savino</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A universal vaccine for serogroup B meningococcus.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">10834</span>–<span class="cit-lpage">10839</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.volume%253D103%26rft.issue%253D29%26rft.spage%253D10834%26rft.epage%253D10839%26rft.atitle%253DInaugural%2BArticle%253A%2BA%2Buniversal%2Bvaccine%2Bfor%2Bserogroup%2BB%2Bmeningococcus.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0603940103%26rft_id%253Dinfo%253Apmid%252F16825336%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAzLzI5LzEwODM0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzMyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.321.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xiang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mobley</span>  <span class="cit-name-given-names">H. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Vaxign: the first web-based vaccine design program for reverse vaccinology and applications for vaccine development.</span> <abbr class="cit-jnl-abbrev">J. Biomed. Biotechnol.</abbr> <span class="cit-vol">2010</span>: <span class="cit-fpage">297505</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BBiomed%2BBiotechnol%26rft.aulast%253DHe%26rft.auinit1%253DY.%26rft.volume%253D2010%26rft.spage%253D297505%26rft.epage%253D297505%26rft.atitle%253DVaxign%253A%2Bthe%2Bfirst%2Bweb-based%2Bvaccine%2Bdesign%2Bprogram%2Bfor%2Breverse%2Bvaccinology%2Band%2Bapplications%2Bfor%2Bvaccine%2Bdevelopment.%26rft_id%253Dinfo%253Apmid%252F20671958%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=20671958&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.321.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Howarth</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moulton</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frazier</span>  <span class="cit-name-given-names">L. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1956</span> <span class="cit-article-title">Epizootic bovine abortion characterized by fetal hepatopathy.</span> <abbr class="cit-jnl-abbrev">J. Am. Vet. Med. Assoc.</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">441</span>–<span class="cit-lpage">449</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bthe%2BAmerican%2BVeterinary%2BMedical%2BAssociation%26rft.stitle%253DJ%2BAm%2BVet%2BMed%2BAssoc%26rft.aulast%253DHowarth%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D128%26rft.issue%253D9%26rft.spage%253D441%26rft.epage%253D449%26rft.atitle%253DEpizootic%2Bbovine%2Babortion%2Bcharacterized%2Bby%2Bfetal%2Bhepatopathy.%26rft_id%253Dinfo%253Apmid%252F13306666%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=13306666&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.321.18" data-doi="10.1186/1471-2105-11-119"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hyatt</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">G. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Locascio</span>  <span class="cit-name-given-names">P. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Land</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larimer</span>  <span class="cit-name-given-names">F. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Prodigal: prokaryotic gene recognition and translation initiation site identification.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">119</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DHyatt%26rft.auinit1%253DD.%26rft.volume%253D11%26rft.spage%253D119%26rft.epage%253D119%26rft.atitle%253DProdigal%253A%2Bprokaryotic%2Bgene%2Brecognition%2Band%2Btranslation%2Binitiation%2Bsite%2Bidentification.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-11-119%26rft_id%253Dinfo%253Apmid%252F20211023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-11-119&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20211023&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.321.19" data-doi="10.6026/97320630002417"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Imai</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Asakawa</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsuji</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akazawa</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ino</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">SOSUI-GramN: high performance prediction for sub-cellular localization of proteins in gram-negative bacteria.</span> <abbr class="cit-jnl-abbrev">Bioinformation</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">417</span>–<span class="cit-lpage">421</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBioinformation%26rft.aulast%253DImai%26rft.auinit1%253DK.%26rft.volume%253D2%26rft.issue%253D9%26rft.spage%253D417%26rft.epage%253D421%26rft.atitle%253DSOSUI-GramN%253A%2Bhigh%2Bperformance%2Bprediction%2Bfor%2Bsub-cellular%2Blocalization%2Bof%2Bproteins%2Bin%2Bgram-negative%2Bbacteria.%26rft_id%253Dinfo%253Adoi%252F10.6026%252F97320630002417%26rft_id%253Dinfo%253Apmid%252F18795116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.6026/97320630002417&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18795116&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.321.20" data-doi="10.1093/bioinformatics/bts199"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kearse</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moir</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stones-Havas</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheung</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Geneious Basic: an integrated and extendable desktop software platform for the organization and analysis of sequence data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1647</span>–<span class="cit-lpage">1649</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts199%26rft_id%253Dinfo%253Apmid%252F22543367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjgvMTIvMTY0NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi8zMjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.321.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kennedy</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casaro</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kimsey</span>  <span class="cit-name-given-names">P. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bon Durant</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bushnell</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Epizootic bovine abortion: histogenesis of the fetal lesions.</span> <abbr class="cit-jnl-abbrev">Am. J. Vet. Res.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">1040</span>–<span class="cit-lpage">1048</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bveterinary%2Bresearch%26rft.stitle%253DAm%2BJ%2BVet%2BRes%26rft.aulast%253DKennedy%26rft.auinit1%253DP.%2BC.%26rft.volume%253D44%26rft.issue%253D6%26rft.spage%253D1040%26rft.epage%253D1048%26rft.atitle%253DEpizootic%2Bbovine%2Babortion%253A%2Bhistogenesis%2Bof%2Bthe%2Bfetal%2Blesions.%26rft_id%253Dinfo%253Apmid%252F6870006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=6870006&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1983QS30000018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.321.22" data-doi="10.1128/JCM.43.2.604-609.2005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blanchard</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aldridge</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Molecular identification of a novel deltaproteobacterium as the etiologic agent of epizootic bovine abortion (foothill abortion).</span> <abbr class="cit-jnl-abbrev">J. Clin. Microbiol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">604</span>–<span class="cit-lpage">609</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BClinical%2BMicrobiology%26rft.stitle%253DJ.%2BClin.%2BMicrobiol.%26rft.aulast%253DKing%26rft.auinit1%253DD.%2BP.%26rft.volume%253D43%26rft.issue%253D2%26rft.spage%253D604%26rft.epage%253D609%26rft.atitle%253DMolecular%2BIdentification%2Bof%2Ba%2BNovel%2BDeltaproteobacterium%2Bas%2Bthe%2BEtiologic%2BAgent%2Bof%2BEpizootic%2BBovine%2BAbortion%2B%2528Foothill%2BAbortion%2529%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJCM.43.2.604-609.2005%26rft_id%253Dinfo%253Apmid%252F15695652%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNtIjtzOjU6InJlc2lkIjtzOjg6IjQzLzIvNjA0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzMyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.321.23" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast gapped-read alignment with Bowtie 2.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">359</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.321.24" data-doi="10.1016/j.gde.2005.09.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lawrence</span>  <span class="cit-name-given-names">J. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Common themes in the genome strategies of pathogens.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Genet. Dev.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">584</span>–<span class="cit-lpage">588</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bgenetics%2B%2526%2Bdevelopment%26rft.stitle%253DCurr%2BOpin%2BGenet%2BDev%26rft.aulast%253DLawrence%26rft.auinit1%253DJ.%2BG.%26rft.volume%253D15%26rft.issue%253D6%26rft.spage%253D584%26rft.epage%253D588%26rft.atitle%253DCommon%2Bthemes%2Bin%2Bthe%2Bgenome%2Bstrategies%2Bof%2Bpathogens.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gde.2005.09.007%26rft_id%253Dinfo%253Apmid%252F16188434%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gde.2005.09.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16188434&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233686200003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.321.25" data-doi="10.1101/gr.1224503"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stoeckert</span>  <span class="cit-name-given-names">C. J.</span>  <span class="cit-name-suffix">Jr.</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roos</span>  <span class="cit-name-given-names">D. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">OrthoMCL: identification of ortholog groups for eukaryotic genomes.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">2178</span>–<span class="cit-lpage">2189</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DLi%26rft.auinit1%253DL.%26rft.volume%253D13%26rft.issue%253D9%26rft.spage%253D2178%26rft.epage%253D2189%26rft.atitle%253DOrthoMCL%253A%2BIdentification%2Bof%2BOrtholog%2BGroups%2Bfor%2BEukaryotic%2BGenomes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.1224503%26rft_id%253Dinfo%253Apmid%252F12952885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjEzLzkvMjE3OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi8zMjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.321.26" data-doi="10.1038/nrmicro2670"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCutcheon</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moran</span>  <span class="cit-name-given-names">N. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Extreme genome reduction in symbiotic bacteria.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Microbiol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">13</span>–<span class="cit-lpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMicrobiology%26rft.stitle%253DNat%2BRev%2BMicrobiol%26rft.aulast%253DMcCutcheon%26rft.auinit1%253DJ.%2BP.%26rft.volume%253D10%26rft.issue%253D1%26rft.spage%253D13%26rft.epage%253D26%26rft.atitle%253DExtreme%2Bgenome%2Breduction%2Bin%2Bsymbiotic%2Bbacteria.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro2670%26rft_id%253Dinfo%253Apmid%252F22064560%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrmicro2670&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22064560&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.321.27" data-doi="10.1016/S0092-8674(02)00665-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moran</span>  <span class="cit-name-given-names">N. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Microbial minimalism: genome reduction in bacterial pathogens.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">583</span>–<span class="cit-lpage">586</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DMoran%26rft.auinit1%253DN.%2BA.%26rft.volume%253D108%26rft.issue%253D5%26rft.spage%253D583%26rft.epage%253D586%26rft.atitle%253DMicrobial%2Bminimalism%253A%2Bgenome%2Breduction%2Bin%2Bbacterial%2Bpathogens.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252802%252900665-7%26rft_id%253Dinfo%253Apmid%252F11893328%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(02)00665-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11893328&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174314800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.321.28" data-doi="10.1016/j.gde.2004.09.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moran</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plague</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Genomic changes following host restriction in bacteria.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Genet. Dev.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">627</span>–<span class="cit-lpage">633</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bgenetics%2B%2526%2Bdevelopment%26rft.stitle%253DCurr%2BOpin%2BGenet%2BDev%26rft.aulast%253DMoran%26rft.auinit1%253DN.%2BA.%26rft.volume%253D14%26rft.issue%253D6%26rft.spage%253D627%26rft.epage%253D633%26rft.atitle%253DGenomic%2Bchanges%2Bfollowing%2Bhost%2Brestriction%2Bin%2Bbacteria.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gde.2004.09.003%26rft_id%253Dinfo%253Apmid%252F15531157%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gde.2004.09.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15531157&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225369300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.2.321.29" data-doi="10.1038/nature01262"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Mouse Genome Sequencing Consortium</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waterston</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lindblad-Toh</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birney</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rogers</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Initial sequencing and comparative analysis of the mouse genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">420</span>: <span class="cit-fpage">520</span>–<span class="cit-lpage">562</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DWaterston%26rft.auinit1%253DR.%2BH.%26rft.volume%253D420%26rft.issue%253D6915%26rft.spage%253D520%26rft.epage%253D562%26rft.atitle%253DInitial%2Bsequencing%2Band%2Bcomparative%2Banalysis%2Bof%2Bthe%2Bmouse%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01262%26rft_id%253Dinfo%253Apmid%252F12466850%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature01262&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12466850&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000179611600053&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.321.30" data-doi="10.1186/1471-2105-13-S14-S8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nadalin</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vezzi</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Policriti</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">GapFiller: a de novo assembly approach to fill the gap within paired reads.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">13</span>(<span class="cit-supplement">Suppl. 14</span>): <span class="cit-fpage">S8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D13%26rft.spage%253DS8%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-13-S14-S8%26rft_id%253Dinfo%253Apmid%252F23282012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-13-S14-S8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23282012&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.321.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nishida</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Evolution of genome base composition and genome size in bacteria.</span> <abbr class="cit-jnl-abbrev">Front. Microbiol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">420</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BMicrobiol.%26rft.volume%253D3%26rft.spage%253D420%26rft_id%253Dinfo%253Apmid%252F23230432%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23230432&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.321.32" data-doi="10.1093/nar/gkt1226"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Overbeek</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olson</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pusch</span>  <span class="cit-name-given-names">G. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olsen</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The SEED and the rapid annotation of microbial genomes using subsystems technology (RAST).</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">D206</span>–<span class="cit-lpage">D214</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt1226%26rft_id%253Dinfo%253Apmid%252F24293654%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0Mi9EMS9EMjA2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzMyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.321.33" data-doi="10.1126/science.287.5459.1816"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pizza</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scarlato</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Masignani</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giuliani</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aricò</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Identification of vaccine candidates against serogroup B meningococcus by whole-genome sequencing.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">287</span>: <span class="cit-fpage">1816</span>–<span class="cit-lpage">1820</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DPizza%26rft.auinit1%253DM.%26rft.volume%253D287%26rft.issue%253D5459%26rft.spage%253D1816%26rft.epage%253D1820%26rft.atitle%253DIdentification%2Bof%2BVaccine%2BCandidates%2BAgainst%2BSerogroup%2BB%2BMeningococcus%2Bby%2BWhole-Genome%2BSequencing%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.287.5459.1816%26rft_id%253Dinfo%253Apmid%252F10710308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyODcvNTQ1OS8xODE2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzMyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.321.34" data-doi="10.1016/S0264-410X(00)00554-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rappuoli</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Reverse vaccinology, a genome-based approach to vaccine development.</span> <abbr class="cit-jnl-abbrev">Vaccine</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">2688</span>–<span class="cit-lpage">2691</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVaccine%26rft.stitle%253DVaccine%26rft.aulast%253DRappuoli%26rft.auinit1%253DR.%26rft.volume%253D19%26rft.issue%253D17-19%26rft.spage%253D2688%26rft.epage%253D2691%26rft.atitle%253DReverse%2Bvaccinology%252C%2Ba%2Bgenome-based%2Bapproach%2Bto%2Bvaccine%2Bdevelopment.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0264-410X%252800%252900554-5%26rft_id%253Dinfo%253Apmid%252F11257410%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0264-410X(00)00554-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11257410&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000167538400080&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.321.35" data-doi="10.1093/bioinformatics/bti028"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sachdeva</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jain</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramachandran</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">SPAAN: a software program for prediction of adhesins and adhesin-like proteins using neural networks.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">483</span>–<span class="cit-lpage">491</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti028%26rft_id%253Dinfo%253Apmid%252F15374866%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyMS80LzQ4MyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi8zMjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.321.36" data-doi="10.1128/genomeA.00027-13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sait</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aitchison</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wheelhouse</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lainson</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome sequence of <em>Lawsonia intracellularis</em> strain N343, isolated from a sow with hemorrhagic proliferative enteropathy.</span> <abbr class="cit-jnl-abbrev">Genome Announc.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">e00027-13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BAnnouncements%26rft.stitle%253DGenome%2BAnnounc%26rft.aulast%253DSait%26rft.auinit1%253DM.%26rft.volume%253D1%26rft.issue%253D1%26rft.spage%253De00027-13%26rft.epage%253De00027-13%26rft.atitle%253DGenome%2BSequence%2Bof%2BLawsonia%2Bintracellularis%2BStrain%2BN343%252C%2BIsolated%2Bfrom%2Ba%2BSow%2Bwith%2BHemorrhagic%2BProliferative%2BEnteropathy%26rft_id%253Dinfo%253Adoi%252F10.1128%252FgenomeA.00027-13%26rft_id%253Dinfo%253Apmid%252F23472224%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiZ2EiO3M6NToicmVzaWQiO3M6MTM6IjEvMS9lMDAwMjctMTMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzIvMzIxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-web" id="cit-7.2.321.37"><div class="cit-metadata unstructured">Sambrook, J., and D. W. Russell, 2006 Purification of nucleic acids by extraction with phenol:chloroform. CSH Protoc. 2006. Available at: <a href="http://cshprotocols.cshlp.org/content/2006/1/pdb.prot4455.long">http://cshprotocols.cshlp.org/content/2006/1/pdb.prot4455.long</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.321.38" data-doi="10.1093/jmedent/13.3.292"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmidtmann</span>  <span class="cit-name-given-names">E. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bushnell</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loomis</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliver</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Theis</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1976</span> <span class="cit-article-title">Experimental and epizootiologic evidence associating <em>Ornithodoros coriaceus</em> Koch (Acari-Argasidae) with exposure of cattle to epizootic bovine abortion in California.</span> <abbr class="cit-jnl-abbrev">J. Med. Entomol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">292</span>–<span class="cit-lpage">299</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMed.%2BEntomol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjmedent%252F13.3.292%26rft_id%253Dinfo%253Apmid%252F1011232%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Nzoiam1lZGVudCI7czo1OiJyZXNpZCI7czo4OiIxMy8zLzI5MiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi8zMjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.321.39" data-doi="10.1111/j.1574-6968.1999.tb13575.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tatusova</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madden</span>  <span class="cit-name-given-names">T. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">BLAST 2 Sequences, a new tool for comparing protein and nucleotide sequences.</span> <abbr class="cit-jnl-abbrev">FEMS Microbiol. Lett.</abbr> <span class="cit-vol">174</span>: <span class="cit-fpage">247</span>–<span class="cit-lpage">250</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BMicrobiol.%2BLett.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1574-6968.1999.tb13575.x%26rft_id%253Dinfo%253Apmid%252F10339815%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc2xlIjtzOjU6InJlc2lkIjtzOjk6IjE3NC8yLzI0NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi8zMjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-book" id="cit-7.2.321.40"><div class="cit-metadata unstructured">Tatusova, T., M. DiCuccio, A. Badretdin, V. Chetvernin, S. Ciufo <em>et al.</em> 2013 Prokaryotic genome annotation pipeline. in <em>The NCBI Handbook</em>, Ed. 2. National Center for Biotechnology Information, Bethesda, MD. Available at: <a href="https://www.ncbi.nlm.nih.gov/books/NBK174280/">https://www.ncbi.nlm.nih.gov/books/NBK174280/</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.321.41" data-doi="10.1093/nar/gkw569"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tatusova</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DiCuccio</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Badretdin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chetvernin</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nawrocki</span>  <span class="cit-name-given-names">E. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">NCBI prokaryotic genome annotation pipeline.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">6614</span>–<span class="cit-lpage">6624</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw569%26rft_id%253Dinfo%253Apmid%252F27342282%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0NC8xNC82NjE0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzMyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.321.42" data-doi="10.1016/j.vetpar.2006.03.027"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Teglas</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drazenovich</span>  <span class="cit-name-given-names">N. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stott</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foley</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The geographic distribution of the putative agent of epizootic bovine abortion in the tick vector, <em>Ornithodoros coriaceus</em>.</span> <abbr class="cit-jnl-abbrev">Vet. Parasitol.</abbr> <span class="cit-vol">140</span>: <span class="cit-fpage">327</span>–<span class="cit-lpage">333</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVeterinary%2Bparasitology%26rft.stitle%253DVet%2BParasitol%26rft.aulast%253DTeglas%26rft.auinit1%253DM.%2BB.%26rft.volume%253D140%26rft.issue%253D3-4%26rft.spage%253D327%26rft.epage%253D333%26rft.atitle%253DThe%2Bgeographic%2Bdistribution%2Bof%2Bthe%2Bputative%2Bagent%2Bof%2Bepizootic%2Bbovine%2Babortion%2Bin%2Bthe%2Btick%2Bvector%252C%2BOrnithodoros%2Bcoriaceus.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.vetpar.2006.03.027%26rft_id%253Dinfo%253Apmid%252F16672178%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.vetpar.2006.03.027&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16672178&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.321.43" data-doi="10.1038/nbt.2450"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hendrickson</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sauvageau</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goff</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rinn</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Differential analysis of gene regulation at transcript resolution with RNA-seq.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">46</span>–<span class="cit-lpage">53</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D31%26rft.spage%253D46%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2450%26rft_id%253Dinfo%253Apmid%252F23222703%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.2450&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23222703&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.321.44" data-doi="10.1093/nar/gks596"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Untergasser</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cutcutache</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koressaar</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faircloth</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Primer3–new capabilities and interfaces.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">e115</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks596%26rft_id%253Dinfo%253Apmid%252F22730293%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0MC8xNS9lMTE1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzMyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.321.45" data-doi="10.1128/CVI.05073-11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Watson</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alberdi</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Inglis</span>  <span class="cit-name-given-names">N. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Porter</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A novel <em>Lawsonia intracellularis</em> autotransporter protein is a prominent antigen.</span> <abbr class="cit-jnl-abbrev">Clin. Vaccine Immunol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1282</span>–<span class="cit-lpage">1287</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClin.%2BVaccine%2BImmunol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FCVI.05073-11%26rft_id%253Dinfo%253Apmid%252F21697340%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoiY2RsaSI7czo1OiJyZXNpZCI7czo5OiIxOC84LzEyODIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzIvMzIxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.321.46" data-doi="10.1002/prot.21018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Y. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hwang</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Prediction of protein subcellular localization.</span> <abbr class="cit-jnl-abbrev">Proteins</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">643</span>–<span class="cit-lpage">651</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProteins%26rft.stitle%253DProteins%26rft.aulast%253DYu%26rft.auinit1%253DC.%2BS.%26rft.volume%253D64%26rft.issue%253D3%26rft.spage%253D643%26rft.epage%253D651%26rft.atitle%253DPrediction%2Bof%2Bprotein%2Bsubcellular%2Blocalization.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fprot.21018%26rft_id%253Dinfo%253Apmid%252F16752418%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/prot.21018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16752418&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239103800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.321.47" data-doi="10.1093/bioinformatics/btq249"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">N. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wagner</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laird</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melli</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rey</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">PSORTb 3.0: improved protein subcellular localization prediction with refined localization subcategories and predictive capabilities for all prokaryotes.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">1608</span>–<span class="cit-lpage">1615</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq249%26rft_id%253Dinfo%253Apmid%252F20472543%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjYvMTMvMTYwOCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi8zMjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.321.48" data-doi="10.1101/gr.074492.107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zerbino</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birney</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Velvet: algorithms for de novo short read assembly using de Bruijn graphs.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">821</span>–<span class="cit-lpage">829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.074492.107%26rft_id%253Dinfo%253Apmid%252F18349386%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE4LzUvODIxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzMyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/2/321.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/1/319" title="Corrigendum" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/2/333" title="A High Copy Suppressor Screen for Autophagy Defects in Saccharomyces arl1Δ and ypt6Δ Strains" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/2"><span>Volume 7 Issue 2, February 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/2.cover-source.jpg?itok=m4uMC43I" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/2/321" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-9n3LhrKu5GGcqW3PfsCeyGu3noUhKPtbz6jETH3g2AY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genome-report" class="" data-icon-position="" data-hide-link-title="0">Genome Report</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/2/321" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/2/321" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/2/321" class="active" data-icon-position="" data-hide-link-title="0">Genome Report: Identification and Validation of Antigenic Proteins from Pajaroellobacter abortibovis Using De Novo Genome Sequence Assembly and Reverse Vaccinology</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460580" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-GR2fh4O7eYfp4d7qd_DM63kouC6GMOyk-9gtGP7RmqQ" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/2/321.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/2/321" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-KHFtDyp9xdUTiuvf7RIY3wmB3-nP7BgnJ3spSdb4kGM" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/2/321" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/2/321.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460580" id="node460580--2" data-pisa="ggg;7/2/321" data-pisa-master="ggg;g3.116.036673" data-apath="/ggg/7/2/321.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome Report: Identification and Validation of Antigenic Proteins from <em>Pajaroellobacter abortibovis</em> Using <em>De Novo</em> Genome Sequence Assembly and Reverse Vaccinology</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-4282-9285" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Bryan T. Welly</span>, <span class="highwire-citation-author" data-delta="1">Michael R. Miller</span>, <span class="highwire-citation-author" data-delta="2">Jeffrey L. Stott</span>, <span class="highwire-citation-author" data-delta="3">Myra T. Blanchard</span>, <span class="highwire-citation-author" data-delta="4">Alma D. Islas-Trejo</span>, <span class="highwire-citation-author" data-delta="5">Sean M. O’Rourke</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-4181-4233" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Amy E. Young</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0001-7113-3183" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Juan F. Medrano</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0003-1562-162X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Alison L. Van Eenennaam</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">321-331;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.036673 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460580--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Bryan T. Welly </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bryan%2BT.%2BWelly%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Welly%20BT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABryan%2BT.%2BWelly%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4282-9285" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Bryan T. Welly</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Michael R. Miller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BR.%2BMiller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Miller%20MR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BR.%2BMiller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jeffrey L. Stott </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Pathology, Microbiology and Immunology, School of Veterinary Medicine, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeffrey%2BL.%2BStott%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stott%20JL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJeffrey%2BL.%2BStott%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Myra T. Blanchard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Pathology, Microbiology and Immunology, School of Veterinary Medicine, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Myra%2BT.%2BBlanchard%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Blanchard%20MT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMyra%2BT.%2BBlanchard%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Alma D. Islas-Trejo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alma%2BD.%2BIslas-Trejo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Islas-Trejo%20AD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlma%2BD.%2BIslas-Trejo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Sean M. O’Rourke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sean%2BM.%2BO%E2%80%99Rourke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=O%E2%80%99Rourke%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASean%2BM.%2BO%25E2%2580%2599Rourke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Amy E. Young </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amy%2BE.%2BYoung%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Young%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAmy%2BE.%2BYoung%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4181-4233" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Amy E. Young</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Juan F. Medrano </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juan%2BF.%2BMedrano%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Medrano%20JF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJuan%2BF.%2BMedrano%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7113-3183" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Juan F. Medrano</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Alison L. Van Eenennaam </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alison%2BL.%2BVan%2BEenennaam%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Van%20Eenennaam%20AL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlison%2BL.%2BVan%2BEenennaam%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-1562-162X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alison L. Van Eenennaam</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:alvaneenennaam@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">alvaneenennaam@ucdavis.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/2/321" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/2/321&amp;title=Genome%20Report%3A%20Identification%20and%20Validation%20of%20Antigenic%20Proteins%20from%20Pajaroellobacter%20abortibovis%20Using%20De%20Novo%20Genome%20Sequence%20Assembly%20and%20Reverse%20Vaccinology" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/2/321&amp;title=Genome%20Report%3A%20Identification%20and%20Validation%20of%20Antigenic%20Proteins%20from%20Pajaroellobacter%20abortibovis%20Using%20De%20Novo%20Genome%20Sequence%20Assembly%20and%20Reverse%20Vaccinology" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/2/321&amp;title=Genome%20Report%3A%20Identification%20and%20Validation%20of%20Antigenic%20Proteins%20from%20Pajaroellobacter%20abortibovis%20Using%20De%20Novo%20Genome%20Sequence%20Assembly%20and%20Reverse%20Vaccinology" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/2/321&amp;text=Genome%20Report%3A%20Identification%20and%20Validation%20of%20Antigenic%20Proteins%20from%20Pajaroellobacter%20abortibovis%20Using%20De%20Novo%20Genome%20Sequence%20Assembly%20and%20Reverse%20Vaccinology" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/2/321&amp;title=Genome%20Report%3A%20Identification%20and%20Validation%20of%20Antigenic%20Proteins%20from%20Pajaroellobacter%20abortibovis%20Using%20De%20Novo%20Genome%20Sequence%20Assembly%20and%20Reverse%20Vaccinology" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/2/321&amp;t=Genome%20Report%3A%20Identification%20and%20Validation%20of%20Antigenic%20Proteins%20from%20Pajaroellobacter%20abortibovis%20Using%20De%20Novo%20Genome%20Sequence%20Assembly%20and%20Reverse%20Vaccinology" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/2/321&amp;title=Genome%20Report%3A%20Identification%20and%20Validation%20of%20Antigenic%20Proteins%20from%20Pajaroellobacter%20abortibovis%20Using%20De%20Novo%20Genome%20Sequence%20Assembly%20and%20Reverse%20Vaccinology" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/2/321&amp;title=Genome%20Report%3A%20Identification%20and%20Validation%20of%20Antigenic%20Proteins%20from%20Pajaroellobacter%20abortibovis%20Using%20De%20Novo%20Genome%20Sequence%20Assembly%20and%20Reverse%20Vaccinology" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460580/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460580" id="node460580--3" data-pisa="ggg;7/2/321" data-pisa-master="ggg;g3.116.036673" data-apath="/ggg/7/2/321.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome Report: Identification and Validation of Antigenic Proteins from <em>Pajaroellobacter abortibovis</em> Using <em>De Novo</em> Genome Sequence Assembly and Reverse Vaccinology</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-4282-9285" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Bryan T. Welly</span>, <span class="highwire-citation-author" data-delta="1">Michael R. Miller</span>, <span class="highwire-citation-author" data-delta="2">Jeffrey L. Stott</span>, <span class="highwire-citation-author" data-delta="3">Myra T. Blanchard</span>, <span class="highwire-citation-author" data-delta="4">Alma D. Islas-Trejo</span>, <span class="highwire-citation-author" data-delta="5">Sean M. O’Rourke</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-4181-4233" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Amy E. Young</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0001-7113-3183" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Juan F. Medrano</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0003-1562-162X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Alison L. Van Eenennaam</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">321-331;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.036673 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460580--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Bryan T. Welly </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bryan%2BT.%2BWelly%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Welly%20BT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABryan%2BT.%2BWelly%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4282-9285" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Bryan T. Welly</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Michael R. Miller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BR.%2BMiller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Miller%20MR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BR.%2BMiller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jeffrey L. Stott </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Pathology, Microbiology and Immunology, School of Veterinary Medicine, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeffrey%2BL.%2BStott%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stott%20JL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJeffrey%2BL.%2BStott%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Myra T. Blanchard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Pathology, Microbiology and Immunology, School of Veterinary Medicine, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Myra%2BT.%2BBlanchard%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Blanchard%20MT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMyra%2BT.%2BBlanchard%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Alma D. Islas-Trejo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alma%2BD.%2BIslas-Trejo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Islas-Trejo%20AD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlma%2BD.%2BIslas-Trejo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Sean M. O’Rourke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sean%2BM.%2BO%E2%80%99Rourke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=O%E2%80%99Rourke%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASean%2BM.%2BO%25E2%2580%2599Rourke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Amy E. Young </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amy%2BE.%2BYoung%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Young%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAmy%2BE.%2BYoung%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4181-4233" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Amy E. Young</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Juan F. Medrano </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juan%2BF.%2BMedrano%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Medrano%20JF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJuan%2BF.%2BMedrano%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7113-3183" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Juan F. Medrano</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Alison L. Van Eenennaam </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alison%2BL.%2BVan%2BEenennaam%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Van%20Eenennaam%20AL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlison%2BL.%2BVan%2BEenennaam%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-1562-162X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alison L. Van Eenennaam</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:alvaneenennaam@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">alvaneenennaam@ucdavis.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=321&amp;rft.epage=331&amp;rft.atitle=Genome%20Report%3A%20Identification%20and%20Validation%20of%20Antigenic%20Proteins%20from%20Pajaroellobacter%20abortibovis%20Using%20De%20Novo%20Genome%20Sequence%20Assembly%20and%20Reverse%20Vaccinology&amp;rft.volume=7&amp;rft.issue=2&amp;rft.date=2017-02-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Welly%2C+Bryan+T.&amp;rft.au=Miller%2C+Michael+R.&amp;rft.au=Stott%2C+Jeffrey+L.&amp;rft.au=Blanchard%2C+Myra+T.&amp;rft.au=Islas-Trejo%2C+Alma+D.&amp;rft.au=O%E2%80%99Rourke%2C+Sean+M.&amp;rft.au=Young%2C+Amy+E.&amp;rft.au=Medrano%2C+Juan+F.&amp;rft.au=Van+Eenennaam%2C+Alison+L."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460580/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460580/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460580/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460580/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460580/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460580/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460580/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460580/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460580/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460580/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460580/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460580/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496044" id="node496044" data-pisa="ggg;g3.119.400871v1" data-pisa-master="ggg;g3.119.400871" data-apath="/ggg/early/2020/03/17/g3.119.400871.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/17/g3.119.400871" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title"><em>De Novo</em> Genome Assembly of the <em>indica</em> Rice Variety IR64 Using Linked-Read Sequencing and Nanopore Sequencing</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496040" id="node496040" data-pisa="ggg;g3.119.400893v1" data-pisa-master="ggg;g3.119.400893" data-apath="/ggg/early/2020/03/12/g3.119.400893.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/12/g3.119.400893" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Understanding the Genetic Domestication History of the Jianchang Duck by Genotyping and Sequencing of Genomic Genes Under Selection</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496037" id="node496037" data-pisa="ggg;g3.120.401071v1" data-pisa-master="ggg;g3.120.401071" data-apath="/ggg/early/2020/03/11/g3.120.401071.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/11/g3.120.401071" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title"><em>De Novo</em> Genome Assembly of the Meadow Brown Butterfly, <em>Maniola jurtina</em></span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genome%20Report" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genome Report</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/2/321" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-21" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-26" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/2/321.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/2/321.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/2/321.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/2/321.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
