<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/2/481" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/2/481.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/2/481.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/2/481.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/2/481/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/2/481.atom" />
<meta name="HW.pisa" content="ggg;7/2/481" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genomic Prediction with Pedigree and Genotype × Environment Interaction in Spring Wheat Grown in South and West Asia, North Africa, and Mexico" />
<meta name="DC.Identifier" content="10.1534/g3.116.036251" />
<meta name="DC.Date" content="2017-02-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Sukumaran et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Developing genomic selection (GS) models is an important step in applying GS to accelerate the rate of genetic gain in grain yield in plant breeding. In this study, seven genomic prediction models under two cross-validation (CV) scenarios were tested on 287 advanced elite spring wheat lines phenotyped for grain yield (GY), thousand-grain weight (GW), grain number (GN), and thermal time for flowering (TTF) in 18 international environments (year-location combinations) in major wheat-producing countries in 2010 and 2011. Prediction models with genomic and pedigree information included main effects and interaction with environments. Two random CV schemes were applied to predict a subset of lines that were not observed in any of the 18 environments (CV1), and a subset of lines that were not observed in a set of the environments, but were observed in other environments (CV2). Genomic prediction models, including genotype × environment (G×E) interaction, had the highest average prediction ability under the CV1 scenario for GY (0.31), GN (0.32), GW (0.45), and TTF (0.27). For CV2, the average prediction ability of the model including the interaction terms was generally high for GY (0.38), GN (0.43), GW (0.63), and TTF (0.53). Wheat lines in site-year combinations in Mexico and India had relatively high prediction ability for GY and GW. Results indicated that prediction ability of lines not observed in certain environments could be relatively high for genomic selection when predicting G×E interaction in multi-environment trials." />
<meta name="DC.Contributor" content="Sivakumar Sukumaran" />
<meta name="DC.Contributor" content="Jose Crossa" />
<meta name="DC.Contributor" content="Diego Jarquin" />
<meta name="DC.Contributor" content="Marta Lopes" />
<meta name="DC.Contributor" content="Matthew P. Reynolds" />
<meta name="article:published_time" content="2017-02-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="Genomic Prediction with Pedigree and Genotype × Environment Interaction in Spring Wheat Grown in South and West Asia, North Africa, and Mexico" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Developing genomic selection (GS) models is an important step in applying GS to accelerate the rate of genetic gain in grain yield in plant breeding. In this study, seven genomic prediction models under two cross-validation (CV) scenarios were tested on 287 advanced elite spring wheat lines phenotyped for grain yield (GY), thousand-grain weight (GW), grain number (GN), and thermal time for flowering (TTF) in 18 international environments (year-location combinations) in major wheat-producing countries in 2010 and 2011. Prediction models with genomic and pedigree information included main effects and interaction with environments. Two random CV schemes were applied to predict a subset of lines that were not observed in any of the 18 environments (CV1), and a subset of lines that were not observed in a set of the environments, but were observed in other environments (CV2). Genomic prediction models, including genotype × environment (G×E) interaction, had the highest average prediction ability under the CV1 scenario for GY (0.31), GN (0.32), GW (0.45), and TTF (0.27). For CV2, the average prediction ability of the model including the interaction terms was generally high for GY (0.38), GN (0.43), GW (0.63), and TTF (0.53). Wheat lines in site-year combinations in Mexico and India had relatively high prediction ability for GY and GW. Results indicated that prediction ability of lines not observed in certain environments could be relatively high for genomic selection when predicting G×E interaction in multi-environment trials.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/02/01" />
<meta name="citation_mjid" content="ggg;7/2/481" />
<meta name="citation_id" content="7/2/481" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/2/481" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/2/481.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/2/481.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/2/481.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.036251" />
<meta name="citation_pmid" content="27903632" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="2" />
<meta name="citation_num_pages" content="15" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="481" />
<meta name="citation_lastpage" content="495" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Sivakumar Sukumaran" />
<meta name="citation_author_institution" content="Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), El Batan, Texcoco CP 56237, Mexico" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4088-4624" />
<meta name="citation_author" content="Jose Crossa" />
<meta name="citation_author_institution" content="Biometrics and Statistics Unit, CIMMYT, El Batan, Texcoco CP 56237, Mexico" />
<meta name="citation_author_email" content="j.crossa@cgiar.org" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9429-5855" />
<meta name="citation_author" content="Diego Jarquin" />
<meta name="citation_author_institution" content="Department of Agronomy and Horticulture, University of Nebraska, Lincoln, Nebraska 68583" />
<meta name="citation_author" content="Marta Lopes" />
<meta name="citation_author_institution" content="Global Wheat Program, CIMMYT, Emex, 06511 Ankara, Turkey" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7698-6019" />
<meta name="citation_author" content="Matthew P. Reynolds" />
<meta name="citation_author_institution" content="Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), El Batan, Texcoco CP 56237, Mexico" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4291-4316" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=K. A. B. Aisawi;citation_author=M. P. Reynolds;citation_author=R. P. Singh;citation_author=M. J. Foulkes;citation_title=The physiological basis of the genetic progress in yield potential of CIMMYT spring wheat cultivars from 1966 to 2009.;citation_pages=1749-1764;citation_volume=55;citation_year=2015;citation_issue=4" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_title=Genomic selection for predicting fusarium head blight resistance in a wheat breeding program.;citation_author=M. P. Arruda;citation_author=P. J. Brown;citation_author=A. E. Lipka;citation_author=A. M. Krill;citation_author=C. Thurber;citation_volume=8;citation_year=2015;citation_issue=3" />
<meta name="citation_reference" content="citation_journal_title=Plant Sci.;citation_author=F. M. Bassi;citation_author=A. R. Bentley;citation_author=G. Charmet;citation_author=R. Ortiz;citation_author=J. Crossa;citation_title=Breeding schemes for the implementation of genomic selection in wheat (Triticum spp.).;citation_pages=23-36;citation_volume=242;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=R. Bernardo;citation_author=J. M. Yu;citation_title=Prospects for Genomewide Selection for Quantitative Traits in Maize;citation_pages=1082-1090;citation_volume=47;citation_year=2007;citation_issue=3;citation_doi=10.2135/cropsci2006.11.0690" />
<meta name="citation_reference" content="citation_title=Climate Change and Crop Production;citation_author=H.-J. Braun;citation_author=G. Atlin;citation_author=T. Payne;citation_title=Multi-location testing as a tool to identify plant response to global climate change;citation_pages=115-138;citation_volume=1;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. Burgueño;citation_author=J. Crossa;citation_author=J. M. Cotes;citation_author=F. San Vicente;citation_author=B. Das;citation_title=Prediction assessment of linear mixed models for multi-environment trials.;citation_pages=944-954;citation_volume=51;citation_year=2011;citation_issue=3;citation_doi=10.2135/cropsci2010.07.0403" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. Burgueño;citation_author=G. de los Campos;citation_author=K. Weigel;citation_author=J. Crossa;citation_title=Genomic prediction of breeding values when modeling genotype × environment interaction using pedigree and dense molecular markers.;citation_pages=707;citation_volume=52;citation_year=2012;citation_issue=2;citation_doi=10.2135/cropsci2011.06.0299" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Crossa;citation_author=L. Campos Gde;citation_author=P. Perez;citation_author=D. Gianola;citation_author=J. Burgueno;citation_title=Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers.;citation_volume=186;citation_year=2010;citation_issue=2;citation_pmid=20813882;citation_doi=10.1534/genetics.110.118521" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=J. Crossa;citation_author=P. Perez;citation_author=J. Hickey;citation_author=J. Burgueno;citation_author=L. Ornella;citation_title=Genomic prediction in CIMMYT maize and wheat breeding programs.;citation_pages=48-60;citation_volume=112;citation_year=2014;citation_issue=1;citation_pmid=23572121;citation_doi=10.1038/hdy.2013.16" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. Crossa;citation_author=G. de los Campos;citation_author=M. Maccaferri;citation_author=R. Tuberosa;citation_author=J. Burgueño;citation_title=Extending the marker × environment interaction model for genomic-enabled prediction and genome-wise association analyses in Durum wheat.;citation_year=2015;citation_doi=10.2135/cropsci2015.04.0260" />
<meta name="citation_reference" content="citation_journal_title=G3;citation_author=J. Crossa;citation_author=D. Jarquín;citation_author=J. Franco;citation_author=P. Pérez-Rodríguez;citation_author=J. Burgueño;citation_title=Genomic prediction of gene bank wheat landraces.;citation_pages=1819-1834;citation_volume=6;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Methods Mol. Biol.;citation_author=G. de los Campos;citation_author=P. Pérez;citation_author=A. I. Vazquez;citation_author=J. Crossa;citation_title=Genome-enabled prediction using the BLR (Bayesian Linear Regression) R-package.;citation_pages=299-320;citation_volume=1019;citation_year=2013;citation_pmid=23756896" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_title=Association mapping and nucleotide sequence variation in five drought tolerance candidate genes in spring wheat.;citation_author=E. A. Edae;citation_author=P. F. Byrne;citation_author=H. Manmathan;citation_author=S. D. Haley;citation_author=M. Moragues;citation_volume=6;citation_year=2013;citation_issue=2" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=E. A. Edae;citation_author=P. F. Byrne;citation_author=S. D. Haley;citation_author=M. S. Lopes;citation_author=M. P. Reynolds;citation_title=Genome-wide association mapping of yield and yield components of spring wheat under contrasting moisture regimes.;citation_pages=791-807;citation_volume=127;citation_year=2014;citation_issue=4;citation_pmid=24408378;citation_doi=10.1007/s00122-013-2257-8" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=N. Heslot;citation_author=D. Akdemir;citation_author=M. E. Sorrells;citation_author=J. L. Jannink;citation_title=Integrating environmental covariates and crop modeling into the genomic selection framework to predict genotype by environment interactions.;citation_pages=463-480;citation_volume=127;citation_year=2014;citation_pmid=24264761;citation_doi=10.1007/s00122-013-2231-5" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=D. Jarquin;citation_author=J. Crossa;citation_author=X. Lacaze;citation_author=P. Du Cheyron;citation_author=J. Daucourt;citation_title=A reaction norm model for genomic selection using high-dimensional genomic and environmental data.;citation_pages=595-607;citation_volume=127;citation_year=2014;citation_issue=3;citation_pmid=24337101;citation_doi=10.1007/s00122-013-2243-1" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=M. S. Lopes;citation_author=M. P. Reynolds;citation_author=Y. Manes;citation_author=R. P. Singh;citation_author=J. Crossa;citation_title=Genetic yield gains and changes in associated traits of CIMMYT spring bread wheat in a “historic” set representing 30 years of breeding.;citation_pages=1123-1131;citation_volume=52;citation_year=2012;citation_issue=3;citation_doi=10.2135/cropsci2011.09.0467" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=M. S. Lopes;citation_author=S. Dreisigacker;citation_author=R. J. Pena;citation_author=S. Sukumaran;citation_author=M. P. Reynolds;citation_title=Genetic characterization of the wheat association mapping initiative (WAMI) panel for dissection of complex traits in spring wheat.;citation_pages=453-464;citation_volume=128;citation_year=2015;citation_issue=3;citation_pmid=25540818;citation_doi=10.1007/s00122-014-2444-2" />
<meta name="citation_reference" content="citation_journal_title=G3;citation_author=M. A. López-Cruz;citation_author=J. Crossa;citation_author=D. Bonnet;citation_author=S. Dreisigacker;citation_author=J. Poland;citation_title=Increased prediction accuracy in wheat breeding trials using a markers × environment interaction genomic selection model.;citation_pages=569-582;citation_volume=5;citation_year=2015;citation_issue=4" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=Y. Manes;citation_author=H. F. Gomez;citation_author=L. Puhl;citation_author=M. Reynolds;citation_author=H. J. Braun;citation_title=Genetic yield gains of the CIMMYT international semi-arid wheat yield trials from 1994 to 2010.;citation_pages=1543-1552;citation_volume=52;citation_year=2012;citation_issue=4;citation_doi=10.2135/cropsci2011.10.0574" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733" />
<meta name="citation_reference" content="citation_journal_title=Hortic. Res.;citation_author=H. Muranty;citation_author=M. Troggio;citation_author=I. B. Sadok;citation_author=M. Al Rifaï;citation_author=A. Auwerkerken;citation_title=Accuracy and responses of genomic selection on key traits in apple breeding.;citation_pages=15060;citation_volume=2;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. Pérez-Rodríguez;citation_author=G. de los Campos;citation_title=Genome-wide regression and prediction with the BGLR statistical package.;citation_volume=198;citation_year=2014;citation_pmid=25009151;citation_doi=10.1534/genetics.114.164442" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=P. Pérez-Rodríguez;citation_author=J. Crossa;citation_author=K. Bondalapati;citation_author=G. De Meyer;citation_author=F. Pita;citation_title=A pedigree-based reaction norm model for prediction of cotton yield in multienvironment trials.;citation_pages=1143-1151;citation_volume=55;citation_year=2015;citation_issue=3" />
<meta name="citation_reference" content="citation_journal_title=Curr. Opin. Plant Biol.;citation_author=M. Reynolds;citation_author=P. Langridge;citation_title=Physiological breeding.;citation_pages=162-171;citation_volume=31;citation_year=2016;citation_pmid=27161822;citation_doi=10.1016/j.pbi.2016.04.005" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Bot.;citation_author=M. Reynolds;citation_author=M. J. Foulkes;citation_author=G. A. Slafer;citation_author=P. Berry;citation_author=M. A. Parry;citation_title=Raising yield potential in wheat.;citation_volume=60;citation_year=2009;citation_issue=7;citation_pmid=19363203;citation_doi=10.1093/jxb/erp016" />
<meta name="citation_reference" content="citation_journal_title=Plant, Cell, and Environment (Print);citation_journal_abbrev=Plant, Cell, and Environment (Print);citation_author=M. Reynolds;citation_author=J. Foulkes;citation_author=R. Furbank;citation_author=S. Griffiths;citation_author=J. King;citation_title=Achieving yield gains in wheat.;citation_pages=1799-1823;citation_volume=35;citation_year=2012;citation_issue=10;citation_pmid=22860982;citation_doi=10.1111/j.1365-3040.2012.02588.x" />
<meta name="citation_reference" content="citation_title=Base SAS 9.2. Procedures Guide: Statistical Procedures.;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=K. D. Sayre;citation_author=S. Rajaram;citation_author=R. A. Fischer;citation_title=Yield Potential Progress in Short Bread Wheats in Northwest Mexico;citation_pages=36-42;citation_volume=37;citation_year=1997;citation_issue=1;citation_doi=10.2135/cropsci1997.0011183X003700010006x" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. Spindel;citation_author=H. Begum;citation_author=D. Akdemir;citation_author=P. Virk;citation_author=B. Collard;citation_title=Genomic selection and association mapping in rice (Oryza sativa): effect of trait genetic architecture, training population composition, marker number and statistical model on accuracy of rice genomic selection in elite, tropical rice breeding lines.;citation_pages=e1004982;citation_volume=11;citation_year=2015;citation_issue=2;citation_pmid=25689273;citation_doi=10.1371/journal.pgen.1004982" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=S. Sukumaran;citation_author=S. Dreisigacker;citation_author=M. Lopes;citation_author=P. Chavez;citation_author=M. P. Reynolds;citation_title=Genome-wide association study for grain yield and related traits in an elite spring wheat population grown in temperate irrigated environments.;citation_pages=353-363;citation_volume=128;citation_year=2015;citation_issue=2" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=S. Sukumaran;citation_author=M. P. Reynolds;citation_author=M. S. Lopes;citation_author=J. Crossa;citation_title=Genome-wide association study for adaptation to agronomic plant density: a component of high yield potential in spring wheat.;citation_pages=2609-2619;citation_volume=55;citation_year=2015;citation_issue=6" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=S. Sukumaran;citation_author=M. Lopes;citation_author=S. Dreisigacker;citation_author=L. Dixon;citation_author=M. Zikhali;citation_title=Identification of earliness per se in spring wheat through a genome wide association study.;citation_pages=2962;citation_volume=56;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. Tester;citation_author=P. Langridge;citation_title=Breeding Technologies to Increase Crop Production in a Changing World;citation_pages=818-822;citation_volume=327;citation_year=2010;citation_issue=5967;citation_pmid=20150489;citation_doi=10.1126/science.1183700" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=G. Velu;citation_author=J. Crossa;citation_author=R. P. Singh;citation_author=Y. Hao;citation_author=S. Dreisigacker;citation_title=Genomic prediction for grain zinc and iron concentrations in spring wheat.;citation_pages=1595-1605;citation_volume=129;citation_year=2016;citation_doi=10.1007/s00122&amp;ndash;016&amp;ndash;2726-y" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=X. Zhang;citation_author=P. Pérez-Rodríguez;citation_author=K. Semaqn;citation_author=Y. Beyene;citation_author=R. Babu;citation_title=Genomic prediction in biparental tropical maize populations in water-stressed and well- watered environments using low-density and GBS SNPs.;citation_pages=291-299;citation_volume=114;citation_year=2014;citation_issue=3" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genomic Prediction with Pedigree and Genotype × Environment Interaction in Spring Wheat Grown in South and West Asia, North Africa, and Mexico" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2.cover-source.jpg" />
<meta name="twitter:description" content="Developing genomic selection (GS) models is an important step in applying GS to accelerate the rate of genetic gain in grain yield in plant breeding. In this study, seven genomic prediction models under two cross-validation (CV) scenarios were tested on 287 advanced elite spring wheat lines phenotyped for grain yield (GY), thousand-grain weight (GW), grain number (GN), and thermal time for flowering (TTF) in 18 international environments (year-location combinations) in major wheat-producing countries in 2010 and 2011. Prediction models with genomic and pedigree information included main effects and interaction with environments. Two random CV schemes were applied to predict a subset of lines that were not observed in any of the 18 environments (CV1), and a subset of lines that were not observed in a set of the environments, but were observed in other environments (CV2). Genomic prediction models, including genotype × environment (G×E) interaction, had the highest average prediction ability under the CV1 scenario for GY (0.31), GN (0.32), GW (0.45), and TTF (0.27). For CV2, the average prediction ability of the model including the interaction terms was generally high for GY (0.38), GN (0.43), GW (0.63), and TTF (0.53). Wheat lines in site-year combinations in Mexico and India had relatively high prediction ability for GY and GW. Results indicated that prediction ability of lines not observed in certain environments could be relatively high for genomic selection when predicting G×E interaction in multi-environment trials." />
<meta name="og-title" property="og:title" content="Genomic Prediction with Pedigree and Genotype × Environment Interaction in Spring Wheat Grown in South and West Asia, North Africa, and Mexico" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/2/481" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Developing genomic selection (GS) models is an important step in applying GS to accelerate the rate of genetic gain in grain yield in plant breeding. In this study, seven genomic prediction models under two cross-validation (CV) scenarios were tested on 287 advanced elite spring wheat lines phenotyped for grain yield (GY), thousand-grain weight (GW), grain number (GN), and thermal time for flowering (TTF) in 18 international environments (year-location combinations) in major wheat-producing countries in 2010 and 2011. Prediction models with genomic and pedigree information included main effects and interaction with environments. Two random CV schemes were applied to predict a subset of lines that were not observed in any of the 18 environments (CV1), and a subset of lines that were not observed in a set of the environments, but were observed in other environments (CV2). Genomic prediction models, including genotype × environment (G×E) interaction, had the highest average prediction ability under the CV1 scenario for GY (0.31), GN (0.32), GW (0.45), and TTF (0.27). For CV2, the average prediction ability of the model including the interaction terms was generally high for GY (0.38), GN (0.43), GW (0.63), and TTF (0.53). Wheat lines in site-year combinations in Mexico and India had relatively high prediction ability for GY and GW. Results indicated that prediction ability of lines not observed in certain environments could be relatively high for genomic selection when predicting G×E interaction in multi-environment trials." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2.cover-source.jpg" />
    <title>Genomic Prediction with Pedigree and Genotype × Environment Interaction in Spring Wheat Grown in South and West Asia, North Africa, and Mexico | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"rz8CzSsyFmIxFCQuELwVvnMI8ChU-fXF1NRAU9a1F2w","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460561","apath":"\/ggg\/7\/2\/481.atom","pisa":"ggg;7\/2\/481","ac":{"\/ggg\/7\/2\/481.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/2\/481.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/2\/481"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/2\/481"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/2\/481":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022NvLsPIE75PVW6CefJMcqjYPcqN\\\\\\\/qdjZhIDyLqBM3buSb2ZkeQnmhszjAXtCfvxDw4roWMFjcjB8h5+JM787DaSDuU9sPB3EXDhFKJZXcqbfoZtuy0N611vAipUHIIMIRwDbgmv6i9XrrdJGyNs5SFm5zKhXTR1DqnVhPRtGBaK40jENs2pYY0JhwHgNsZMteGPlUpLl0YUt16nrIu7vnfmAwJHjtMzkH9dClwaCFZHkdA46+AXtXR9+DuYyfN9i9iZUkmOMY8sbn7ozPgbsoT4vdUV9he0aLI0\\\\\\\/txKu084+kA2TNMtkPq9TDzJaox\\\\\\\/x+uzSq58Fb2Ytp85BxYz4NWajEUVnRYUlR9d+sGwUkUtphw4XePWSTGWhpO154ASkMGM7ZkLAMaHjJ0281Tp3dT2+NDQ4XULL40zeAnkOR5HrRZSirtr\\\\\\\/SQ9poR8LnYdQcbbNn7eVzJNyX\\\\\\\/7cTsERAN8v067r\\\\\\\/wqSi+43aGHAR8DnCbeNWaMQvULoE+KnY1NU4Mm82UB7cbYdd28g\\\\\\\/yQGDp+iuROUd0HtzR4WNrWIypo0UHyXE2EDFmEZRWAbohnp06a3Te5QwUMlgqI9iWzePj35bFSfPKQLr17Jt722vgiUbDES0VIYspaTLfrD710zRJ\\\\\\\/+3FU+7Mcb2MkOkNca2GxhbnOD7lHGKpZAVFVDCCQEABwMhY3IleU1euGLETOzJ6mQAziRxMnVp3M5nOhqG7J41cBNxVEEOiRYlNP4lGZ3e3aIYLXBHltTpyKQd2YLOxMHx4mrT7Rqwt9Yq74xy6eHW5DoA0GTFsLu+iPgPNjvMByt1Xb0ox4KjKyO4iCYONTtuiauzRT9y8UGIgkXdoZfW3jJ4\\\\\\\/yZlA1PBKTbkMcS4G1vwPTht9\\\\\\\/sWzDq3\\\\\\\/WBvmOiyOP7SWVXIkCyRPuVKBg1PJNrjRTJw8AfWfZNYRBM5cTw2BGeom3+704NjO5m\\\\\\\/1TtrCd3Gx26BuvJyWSxLzdGk0VMu4xUjiIHHx\\\\\\\/QKCS05oueaNGAkwd68Yf6qEItVYrBbTBprv49gURQxg+oZhMqioWibTnNYPd4N\\\\\\\/PYEvU+KbJ7TCQs2KmLNKDTL35QR22Ppcu+tKLLio4hNhwQBaodSPLF0l4eWKOev\\\\\\\/hO0oFG5dFcnIY+bK+mY0c0PyOI2z7AwsKvkawEtbMQYSXsY4qaLIbOtOg4HXiwhlKOFGo1fhb0hMNx4AHzMM9c6rg7Y1TukpKYC8nZoO3EvPqxVrIb\\\\\\\/WqQn2yvJ2\\\\\\\/enMw94gzGmfJztRTWWBHCWmgOHnVN\\\\\\\/\\\u0022,\\\u0022iv\\\u0022:\\\u0022vNH406oHBQo+Kn9y6BUuQQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e723da058b0b9c24baf471bf5d8bac962343d4a1f3b4cd3d28db880c32750f79\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022O1hEPXpSsrA4HQkxKgwFUZE1e9BGIH6068C8r1AAFcvlWhczyTzZ9DDPYGzU6XmYSNRdc0gquCiXmxPwBpIa99bIBdisHzKX\\\\\\\/TQoCjxKJsYzF7t5i7eu1spQrI1ZLOWR1DIocf7In74AF5GGgF70dfvP1sRRpH6o6+cWvp7NiqLP6kXYr4fbY2bL1lUJtfGmIGbAhcXHRfmz82lCp1Xbs22DFQkuFOjRESbJFOC9YvPUQp02Iwx7cdn9VSzeW\\\\\\\/Se23k95+pw37vsqM5A6yu0Qx25+78hlLAJS0ZZ7rWAdrPIxo1LJlEss\\\\\\\/uIp9O7hlNt0WTMOg0xeDKbuEs49jl6BDzm6ICDKc0M7wlMRMZYhkfjB7PLkdhcN+2Ydps53908o4Zx075EFZ\\\\\\\/qT7DEE+x5GvAqnQzhO24L20iiCm3MLRxCaDC+0VKQ3+o+rE88gHrU85ddOCmAP\\\\\\\/mwwsnZZLPfN7MeNGDF+E2s1KxsD4MVgsp44BT4jQzMmhpmyllDciX7K7sDdzzowmLLRA6RgKLaikARAyDjh3f3WEpNAtv7bdeKewnlw8FUs3SHrTpGxEkhiOgjppuhPdbTfhJ7Wmt+RqB8+J7tALwS2nhiRHq\\\\\\\/c72CMeFSJZhdFB1nxOuE2EkzeNyksjV2s1j37BOus79CATVSCOKei5xN0gp0fnFGiPlaE3Z7glRN25y0GFNiXKLRfQpFGxCR8Wms58lqYZZ2e5irKEY11QigKvqAQwucLZsV4Rnqs6DS7k6+ztO6CQciS4iXN\\\\\\\/DNOKRKXSKxg64uJYLKCLYhRI8c8IGZ9lxytGhvY\\\\\\\/Q2I\\\\\\\/YaQG74m4ttbSxhvNuqWGZtw0cAf5cVvvHwY+CrrGHo\\\\\\\/jW+0l8joIcoJGGfRN8POSH\\\\\\\/NuCu19iNT42ZRcrub2h4\\\\\\\/ur6IVmJQOUdd0TEy28q2iKzKDcJ8UpJ9gk9uzEI2MTALUwX\\\\\\\/ugMB\\\\\\\/JWG+i\\\\\\\/UBno3cNS7e3OKzlog21OZbPpx8vHd9LTcjAeBhiJG7qgZj65qJ1ExXXilSot2huALCwi79em+eiPSmPBzN1GqQ2sl+\\\\\\\/fwePEFKJ\\\\\\\/b0jqyy4k3uDp2rOz58930f8q8oxs6406GcHMigQ8zsj5jGNoQ20pGebbJNSX64olIPTHA2c2NppTMXhW5ps4melR8OfYMy3d+QDS59R994\\\\\\\/IeL9QzzWJUAHfJ6Qn73GInSlH0ky0VAMxOa16c7td61Pa6SGrUyjT+vwvxO6V\\\\\\\/7\\\\\\\/4rB8BD35fnzUbqebVTILgK6XlsBg3uLzPwVa2l3Ndj8VbQCA529xcSXgeeJG2KRajW0B20gw3xHIP2RjbN+2yXx7P6POAY8dO+bbhrZY4hQRJN0MbQ9\\\\\\\/6HoHfNwhQ1GUUkC6+MtS8nS6bcWxjm2lVON+EqzwWbQEue2sJ1c4fXsbLCOnybI136y8WKdpTN05BA9s6RRmvzJmSjM+ruXhP\\\\\\\/qXcQJeYE7fNLJ4azkhKBR\\\\\\\/wZbzkClON8mFreZ05YCL9\\\\\\\/5lHKDE8TGy+AgYYpV68y81e\\\\\\\/7Vm0Dv2TSkNFr1gjIoshKiyqe4sVIyKI7kI7k5ZBzplo4vgwzE5z80gdVt5wyONBc2QZqEXjEOfwbCNdyflwpl5fwpDle8l0oW6hHsB7acjMWuMrcV6yhjY3eh0VzZtoFu+VgFcPIRlr0zZzqciUoyK7zWsSk39X5Wa7c\\\\\\\/UUl8EjzRiEj1PSxgDCSbzf\\\\\\\/49jrfiD9lVXV8UwsfNRQv\\\\\\\/Qsi9qPlixIhr\\\\\\\/\\\\\\\/0uXSQX2XEFnTGRU3EiaHUj3cg79vy0A8mYiUK+araAtpFdr9lqBWqFEFpjizRyWL3r6z5RpDXCVtEXZ8cLoqzqZK7cbP14yAI4jinx6w95Egw3ZDVVbvQkFUjHXhn7G8NQsVuYy1EVtPi+J3YOpyunvY4TN7blE\\\\\\\/DzQUueXmyQXm1IvrjrniAy1jZv\\\\\\\/zNHpQ3F8ONDBxPumdiyPbNukomCrLp4QK2IIT\\\\\\\/BaB1jFYgQUqmkVidTtqXgw1vEbH09PbCmOfTfC6jfIPfe4Fy+nO9djCaVGrcnuirAF4E1kMk3tRkGU2apd5QRggIOZnL1CgYwMyOBjbDIAFEsBNGRIxrLMUwl7PbaG7P8F4JF4ncTRJqXI\\\\\\\/2F\\\\\\\/GaK8wOsp5X7MO1KFTcKZmLk73e7Wt8yvIg5yBEEADSa3lNxT1qIVHvfb9jvpkxryBbQkKFhdpaGwsAJMdhpKs7yZ2OsNFPIQ2aUZjP+QrmGTeF2G0cEh2wSST99fy7QJZgkq3rHyeCR1YWzoaGceHwlf6kfezwVeToFTRRbzZdFQQmkLRiAUyyOAIW9B4QkTU0HuCddzJHQlrUazac9OHlo6f187mccVX7uMlE2cBj\\\\\\\/kDheN43asycvNHe18JvHNbIaevfbYrRpDIg3txnsq9+1YoB85ZMCN7fXMm4P8FAXsnT3KeG0sOYZ6qj33ULBCqZ1uND1As5HXtqcY9EzHyFzO9rEpyezyy20gm\\\\\\\/89NIoyzbHZ3VA3qYKgDZOb\\\\\\\/q3ALAQieBBV\\\\\\\/4Yd+DR9Bs7\\\\\\\/zEqMQu5muxx9y088MYEfj994\\\\\\\/mroWIzLST2zH5C0CMHHUZR1ddyuckOCBsacsN2NE\\\\\\\/JyPz8iedTZFiwRUyP9G7SFLZpCaYf0bpery\\\\\\\/m54t7Gb+tU9zupZ9sFwXXLhliLv1zZ\\\\\\\/cNU3SEKS7tYazgStLVqRPFcJbPLnFY\\\\\\\/+trPJJd\\\\\\\/4oA2SuUtEd+21gtnIPs+4nd7jkZem+pGUKSNHYYN\\\\\\\/b36GA8+MUynW4Qs1u2S8r1wR4+SgBSJeDtCdPuFHVqx0VZv33Zzz8fTY+bYk8pY6KKVTcuHIltL1KfWtfcwHUW6Fef4v4sr3uUG1z7VR6Xpt3dEcb+QmQVD0rAO2rBFz9otGiIETaSzl2NknOxy9Od3F7a4jDGzdT9wvI+iLX8VznTA+8zESq72IyHXfw5u16vuZWYinI1BDBws95hrwTSfngL3p\\\\\\\/IewD4PagWQbhwMSIvt5WYQbtTI06D\\\\\\\/6EcqDLXL09dmEatLpT9iVv6E6K7hvGaUksQ6RGC2iQBY\\\\\\\/llQxVDO6N5zcZVK8KOE+EGSppQR1W7G0q3JNDFpfFOGHgCIJO5W082ITXtk7glasLbp7TMlbW942EZIBMs6uegU+FbS2C6SeQQ9zKOhEM2wBsBhn35jdzKqpnySx0lCWjOPDxw5GtML9TQNoTOIAXcSF5GIGNeciyQGcTIfwX43kP\\\\\\\/Yy3kiVAiKP7bMjFoeMvk+0glA28J4I0L51D4FsW7QyUYg5EC\\\\\\\/YYipstTsEz2kgqOQrjJyQIlVL2nex58RxoHc+eHY7YXYSEEg6CiPIZwaLahFGF9ACDlSSvjaPZsqMGVIp4tpM7WB5GB0GR7TGwzeplKnCEJ1pGYOdCI9or254ITns7pkh34Vjpm30lJSX9pfAPgwFZ0lGKthzkOkHOPB6epyeeo9gv25cvFnZM+b8gAyC6rsrPzVEvnZIsKGNX7EwsXQsUMNLjyOU2qQJ9pXqjtPDcIwK0aIOE598PO27rrbiAZoL6pyMcK6fAKW3lqW3ZXHm8kp\\\\\\\/OigyM7zcxQ+aklxQSewCncD1URKYBDW15jU89xore3zAaXRWhOeCstZFr1VhbfQdTs4YhiTmY4RVmmkJdFnRUKJ5VflEQmytim0ni\\\\\\\/SctP53RAeltxpWo70soSpoNALCL3YQpY3B6Rxzzbvwywqx95Z076dItqozgCkvgFXdRfpsHrCKFgat9YNVsvwhZ5NcsL85eN9kf\\\\\\\/yZCz8pURI76AZe18nSLnpp96wB7AUjsg4PIJd16PGEFBK2odR5ioEauhKpWbML7t9oCzP\\\\\\\/mdW8FNiqLTGEJNaMt\\\\\\\/sP4HyiyVzmpfXHEcsi3i3H3I+xwPa+JHmzEdKX1+1CrIxqRV0gnPXh1V\\\\\\\/naQW3mBOrWHmmbQcmdCGERZB48hdugmjh8\\\\\\\/wQ77l8OfHnRC+FdtFCYLu1oQk\\\\\\\/YBWvZBqOzNLtfGBXZHIRJWXgGsAz\\\\\\\/+I2h0SCHUIC8SH9EO7sSiYXfKa6srKT0LUjOxN7HtycTVjeqVVVVNXdeiR\\\\\\\/7g8U59AVrNXOPfd86yIntjuzBzF1rmFH6jXaxwvp8X2PGu73A2ICogMYm2POu8UKRelP8aHxFDzAcV4rLjMyeNRJF\\\\\\\/6lWCi7hoWE7+2JuBoxSNT4TryoWFVkaxdn6cvA5OSPgDQXpS0LM10SYrEVH6Ccd90fwnPUdVR21bbNZyqafxn3C2MFYXoYOejMY4n\\\\\\\/G1BSRIIK5XDQ3DTTBw1d1p4hKaB4kYBAHXKIfukDB1V98XojSu1MdLY3ceOudczNr+J4UUcwCpqG4n3mCdwJAt0onJ1jeZjfGBUrRUSQNBunY3LNAiujIWsEPtBfioW75HTj9NVbsxbczzQqG0llLOQbeiKjAVmJBOk6et\\\\\\\/SQUuQuTru7fUTkfJ94lPPycCiuHul5NqfSXRx0CzyoP9acIrBGR4+v8DqAyeW0nVSV++zZQ9vwAASd83mfiFoP84bERIrKHHdiS\\\\\\\/Qyw0RkhEsdFxh41QG1iv59R7TVJdVr6lqm75XWkxpicFEb1jsM5gI7+FPbiTrv\\\\\\\/yZk4ePvU7HhZ4njuiErGGFzvzWZ00Ns4eQaTA4Ygrr0ogv3rrST7AXTESm0yg2aqcty2xZWpv2B3bxULNpzSiXqJgioV3+Kwu7plyzJ1pyu6Jjmw4GCtfdGi\\\\\\\/0MK\\\\\\\/D5JA9zKMHSlUSzMTxlWXCgrtFtsWiRs8QyevCBsSY0kVjqbRjJrINXmElTEkQsi08DculD3vTESnl6YuxWGBY1cE98cjO8NU765Pw6Q3NoHe6bxs7R1XP1LJlYs6g9welv\\\\\\\/lvwhEQxinPxALAlPAZedMvsvEKxaLKig0lN7ITzF3zoahKn5Nmkfe3S9nVzik933oMpQ+bDKOWe0xNmuVKau0howeeWc0FZg5XjDQcLJg4xgoAp4zYMG5u0bDj2skUlhEsb0\\\\\\\/th7gcaLDP3RPVNHAuAvulpT8G\\\\\\\/2WKkvB2gLC6HfK11BFFAEEqDYIQVsM6e1BxfI2guyDrI7Ry68K\\\\\\\/tHa5s\\\\\\\/9sUDcWrSYV4pAQqkk4d3PW+D\\\\\\\/phwlXgPitN+t0kcb3iNuuYqKOm06QaIQYGA54xRwvLL+JKTWdtHb8RwjdEjQB43vUlc0GflQI+QMRjzNAds59J\\\\\\\/JEX96AaNmaPF7m9+km8hCy+kPcN6dgbb4ANCJBQtwMIQovPTh4OelpTgUOSwtPaohq5BfYIZOBvsq+aaVbaWihKcI2myO7x3RQGHCz3IVWHfO12NjD\\\\\\\/yrV1x1Ma0yq\\\\\\\/5C76uSSqXBz9VAidfBkhXG+JfaNfNgWUVWYgKAmyJ+qbIabOjZMPwvTZtwppCUodWeGBE6khhJ5kjnAim+wNRURmfAB5qZBo+hhXVIiYBK\\\\\\\/IvnzZBz58\\\\\\\/AQ2pENFRVXMJrYs+SKij1ZjuWLwgPDjmCf+jNOSag36sB4Gr8EGGugknOd8Io8qsAoUkBDDJVUYDUkqmlWe+Xxl+f7GLMNunVvrZL9WT0zWsEJfHsIT6uBywuKyAnmG0JuvEFiJljUVOAO8IzBiaKAAv88eTivpH+vzu7Y\\\\\\\/0xnIXLbZaNNw+o8I3gO2qrFMMm6Ad5PyU3DItItU\\\\\\\/Bisiq6J2Ru4j1pP6tSe7jkiEIxvW9V4NGUVoH+e8MfZGVBNp6yQgPC5R+s5qTQ2NJQ2V6QES7MatE+ABO7vTih1bzXRHTENa+SyhoeGtSdztWGLpHcNSm0fVvD99\\\\\\\/bljcjhAqye3KLA6\\\\\\\/b1JcFwsgSYvz6Z3sngR5IEQXy4qihqT5oFc9pScSuUP586bb7Oc3nHgB9Z7vwPsJsqUD3Lh4w1kyd0JEo70mIgU\\\\\\\/hYMu30uNCAZ3xomDLPpW6JLMTPLc9t3IxnnXixeXXlygZi1xA17RctCsqhrv6kwCFNH6t9SMT\\\\\\\/y9CHY7KqsjivJVgiWewEP7gEqXVCaC8vYdwunKybjFzyd1osBgjbyt9IXvcMqNa0k36rg6kC74kHJteeWoQUpYtLr7AGFsrMPJJT1gunW0W5ooMKgn0AoYrQGIqM3LXqbMETbgolC5j4cCINsDtYdluniieOmx0ip\\\\\\\/VsilN1HjIjH4z8I9XJJ3AoRe9tK4iTYWlqf+Sc+LSXHFbLvNHLroWbflaK78BJnM7XnvhsNDTuwsrhRzHKaQOLloaqb8SDmV3uo\\\\\\\/0suvVLOWUKME6cRiDiPnZkS5fIGL6yxywXJas8KCOr7TPFjjxIeOhkuEhNma1\\\\\\\/NukZ58Im95svuK24YQTtH\\\\\\\/HO7UZK1IWPrnMMJPh3PHoNsoLQ34h7W3GrSiC2Ldt+wAvjhU9tV5SGIjPNWwEK5C1o3lJJAXspdFm0wXA2WIjIfsf9iP1+zeuCMoAxxZt25plR2CA2NcFM93pVB3QwjICiS8yjE\\\\\\\/eM9u6JSm3Do4InsuNO47eFHknsa6ege5ko4wNjm6aqOVUqKqbzPLEHOTVmXIhS5EZCE47j42tlenA\\\\\\\/r47+\\\\\\\/xBWhlX33OTQ0vOXv+Mi\\\\\\\/Y9UaN9bSnGaDn6q0kRdPgKvxYB4a+IOUHa7Zu81aSUSx8OnNR9oSrCAanXi7S\\\\\\\/sZPgKXV6gbbuSmtU4szO+XKbn\\\\\\\/hU4Ch0X0WFgAzqL2v1d5ZuYpd\\\\\\\/0dwIw9wNiwQZ36oVbR2CwSJaxJN+pNl6FFQHnFENQlGPQCIay2aHU\\\\\\\/tNYcfgaR2CaOJ0BNdmVR+IVfeXPm\\\\\\\/WD2cn73Rl\\\\\\\/03nxvG8Yj+V3azV\\\\\\\/Mwr7AviyNN5W0aYrhA08Ig4JRWiF38Gb25Ysp0yp7S2scMD6S1\\\\\\\/LTgbRvAAA6RgNbxXhe3djYWWGq+P6amriUxroZ8WSZ8BtEmXlHlBeuAmPFdn4ZlpYIUaRomTpd8XCYD5ex2Du0aK8qryzPdWBnhAw1g90KqQZew7CrvEsmsmD4hU9H8dw8Iu6VWqv0W6AC10hJgJEqVgcs75icfSBp8ePMnoI1LHLTMW1ql2gJ+e1ogeYLhIj2Xe6kMMZoJGJR\\\\\\\/SfWSuLqP3ufClvNKbhGL0Ux4MPxk02BNAISL6PTDxDZNuvg0S0DRXogU5LpD\\\\\\\/\\\\\\\/4PmtymWjDKWwGqxcCVz2KIPTAf1qZahkM1ztKjp7V\\\\\\\/qYCM7RhBxYE\\\\\\\/FVlvGvHwQKCq4KLh48L9UNg\\\\\\\/K0O6snC6m\\\\\\\/DVXhzmCk4x7FMam81pzhUvCjEXYE69WKaTdKRYt+geniPx73gmf8KPYIhQfSEKv6Fu7YW4Yv4CyS\\\\\\\/RQW7fFL7NO0R\\\\\\\/i5+lD8JXZfmk3RYqHgkHvyfaNOEJAiBjJqyjrrZHC9vWsVM0QORnKlqY7BgZt\\\\\\\/IXKRMEd5ZStVmxMNWthtR3p7ZkzvB7KoFOL5azubWk8P9n6+3Aj0LN6joBDuV4xuMLXNuovbYzSdpkDm6HkU9bGlE6ORSF2EH3r9yVhAMm77aJ3o\\\\\\\/5yMMrC1A5lixRGgFKNBffVcoaGtM2KsQUoa\\\\\\\/LHNpJoqxyF+TNoNnQ57C8DNR2Ey\\\\\\\/WskvaUIsWzgTpagGJnkyQphd7HV2Dw\\\\\\\/5iuXv4iGMI+yG2pKxmhAHlG0W26e\\\\\\\/VlVyHhgAyyEPzks0ESGdaG5JGR361oVMm7cEEtH9yUh7JBZNpf1wACQAntIcHzzRGB3pCZST+4l5GEL+JMGPXXX+ZK\\\\\\\/EZBPSboBBiwzkgY7gusZ6paSTgbs1685KG91ZyEAeP0LAhV3zgBWSDh6ZkTSOBwJG3Sk8WqNkf7K4pjk0J4tU8w5g29cJWSCAOUfFxZ5pXClBvCcFi1qeBdbz7ehFCRCqxSQHAqKOYgONNrvRDXXrdsEWAEBrfkw6pFf6Jjixburlc+O\\\\\\\/gDJXgPUApXJyebzdnXmWmwErti7e3EACpydBW6HAw9DPuvKobbR13Qrq6pRjhtyBfs1sRSZ9PTeplSsNqkBf3N02ilwErhvDDXL3DQybbi7JPrYfcqurOoBP6qcGlk1iTtfI5ZQ+s5FkmB2gJc\\\\\\\/ANrnX515nXiTPNy37ar1mH0cE02FafCCky8U7Mj4rZucDKty3EeZF8xJXDz2lDRi7SOyR4cgu6GHiAyDXnmCN8o9ounjI1PdiBeZYG6k2oWiMMM4dZ+VdDQTcBckny1\\\\\\\/2gz1iTp1QehzR0FWrcu3GtVzp0SOSAyfh8fdnNasnugxIfpEpNWZHbOZT0gE8KPkBWycHJgn6eXzbck6UN8bBRivmGb2tQv7sAkgNrpNp9\\\\\\\/RROUzhYgbRUgBTnJ3ad+HTy6yqx9BHKHQMr7REVK2KvqvNM15vEVI8WahJ8tuWjbNzPuTHlm9K\\\\\\\/pntiMyWlSa68UGm7jVAsam+tZORTr4EdFCpQqRzy94ep5a0p4wRqgsQOi7YWA0ouseDNETEHMOHzGkYdBp+hhnGHOEovgmdYiyyLywJrpRKvbgJ+QLw59bIQGR7Sq3vWeY6OAa8zpWTMkPGaeSFvZEqNrrXSsHHxwgbcid20WRj6ZPjfOaLE+3uY60rnZsdPxngLM+awHpqZvvXiqDx8ut5jsNXYF1sTO8RGNZBrAPhH\\\\\\\/tY02F0DbFDruvUZPQB1vP02YIDq9eKCi87H\\\\\\\/tYCipVEN61pGtxEo7EajshZC0V71dWJxilVkG1Uv2+PRjPylutlnZRTyQDnQ+KvAs9CwYxH9zSL7aC24pQlwg72kq8uOa8JwRr2XNYeyAsiUk9uusBPYRHvUlW\\\\\\\/13v3CsmXFYQJBQDoaDBS0TwAjz\\\\\\\/5yxQiSXDXpJfg\\\\\\\/Or3kBhYg+BiMbXcz6Sr6T\\\\\\\/YzHxSaUp3k1KV8Mzg1S2auDHRSnGvlQjewUOuA947MDVevmuwi7fkC6cRLf1U3KTta2UeEIHJXSdCXqcExy1lNavp8SrX1Q7IC3hYKuSd4nro8TMnUQB\\\\\\\/gB0pgLYGEOLmofJiOUPXMmSTYvQPAG+ZPGxyhRe9vXNorX3d9zCuY43q0+mvjcoJsU4R8w+ybE6BzQFUZPW4ME+2hekyZw5Fj2Jw25fb4800eG4A3drcGp+IUA6YlTZG7GacNIfUSJN1MTq23AWAe40TBSoad8Y8SP2ZQgIRe27IFb3Zl4xgJgUjJWca6n\\\\\\\/9NPzjHeiQ6cn4t+yENf4i9JcWty6z2sdBXABRU7WdH7ol6Xy+v2DaPQflW2sWM3DwhqjLfej9c151xf33L0T4xbitDaY\\\\\\\/Le+wRYYoBHsWtu1njhbUzcCVSyhuR\\\\\\\/AU2Z6f6bxq0RPloC2AiZaAJpIi3TmrSmax0xew8Jr3L9kUiqN7P13BqiqjzKX2WYYHBeG3qAFVUdOM0Az5nU5XkvrYw6StX5tpCeYvyGzgOLgTEW5tLjuMDnNVNEzGbuTFrla2oXWaIfTRCAgLMO\\\\\\\/f603kMsqW+i\\\\\\\/op5WVGdWcMFFQHgd7BVpo0ck5qLTxypsVZnEfkH6i7jVDrJ\\\\\\\/HPgZVbiFNVsX+pSC0krj+ZuLsx31mzGfrXf+dHxCFT6Pc\\\\\\\/7eEgXhRddO8Eg2yCXMEEJiWXONGIsoNxDmT\\\\\\\/6m8oQKzwoSqkW0jdX1U2\\\\\\\/Ai3POrf4h\\\\\\\/\\\\\\\/nFP\\\\\\\/EXrbxN1ShntYMm1DVRhPZ2szsISAQ5JyHYZ1qZZISlX9UImGKv7+Se3bxwvZrYtEokbV38kpknIdy9hnSqZur6vyS\\\\\\\/KpCG8LVWjOc7NqMCfJtO65Jd92cuM34TR6IQW7tZlSUFkyrpIq8sMpXVNMgmXetJ7isiykgULhBv3\\\\\\\/UTqIws7spT8gnHvtbrpJBgDc47XAJRo4TMxCO3Q+h4g2B50yOlrpm3Pfliu4C86hIg41ojzRaPJhoG8l74jyKCbMiBwuQhLIHgRkA681oY3Ths+K03AzXAOyUzZMO3+8gwdK9qBHfbwPMSjuMuXUAvKC7ap8DY16s\\\\\\\/TehXEtsNPxxOfBR3oWePDJNb2YrbpOYx42WFJlD2St400u6uAhjzmVn50q3HrSE9+Rncg4ZtEAJA7d4VNxwvfhCDmAFNnft9VHWH3ZKFGK3RzI+Prm7a2KQcncg8LGDndUohJj8QOixf1CQhQ\\\\\\\/LW7+vYHWm3bcJuCtT2NkWQIgB4+uW9KGa+XEKnmXnyMQC0h3mmSWzgbEcTJnLOmTVzzt0PKNg4MVNqMlk2Q5fhalGFMqJnUO8eaXXgD\\\\\\\/FUsNmcCfsaab8\\\\\\\/WkKUZzAXZsu0OVoX+27RKdeBVn0Tke3sQZlE94ctvRdnlnnZIaBzFvZt6JqQdt5xXWKkkwrEQ2wKUiwEF9eedfsUKuNuu3D7XIMciGUqnbVv60X6vVtLNpff7wg0qg82nPvqK0W6NTg5FPUpxUq+hA8kHhzCv8ffZwIwaoyXHfreLmvbgQJvCl45MLkQV++r+BhBlvLHiL\\\\\\\/5QAkDHgtTg52nzIr4jJF9G7qljfImae8+Wf\\\\\\\/I\\\\\\\/FFDO+Wm58oUFvd1GtOEgWmjPAbHkuYQZRAA3gdY\\\\\\\/0Sxr94ZcxpyVxWQY0h9V9lW78RYYGzxCavFKVZ5C\\\\\\\/JpZFJTtnXssRhXrw27XZluFCHFjFgbN29cXvyDdjI\\\\\\\/9TKe2uB\\\\\\\/F0WwFxPQJZFGljYn0hJM5kAQOWaT5ZWs03C36vB\\\\\\\/M4FgdKPYcCODlNTPtDlE4FAAoZyVnyW0u9Kc3xUMREuZIm5wAIKZFzKhQOQ7Sk9NL46b0Tbi0d9aKykpGdRBLTYtiFCsgmlJpv2idRiBX0mHxoS6V+I2sN2qDr8VJP6WxuKa8s6+8xCB4n39HPEJXkQb\\\\\\\/69OEZVD1Y2VNDyBzT1TNzd8MvPTf0LFF91wAbu5wYKbKIEFU7jIMOxFICSWwHglr4xmqEW5nexNDjyGL4uhD8N2YM\\\\\\\/PTBVbnB8JlVxduwh4rwGaVMJJ07T2I106CxW1QW9rH9HaF2FFspH80pt27Idwev5ZfhZtns0k998ZN8m6i5y5TYIvscDYQ+8LHAjEzcG3Z5cED5JQQ7GJ\\\\\\\/0BjwbCPFFRb\\\\\\\/BiGwGFxyclj\\\\\\\/aXw3VexxCECCMGMw0ODh5BhRjB4XJwKHnfPKeIhZOuVog8w2AM\\\\\\\/D3FAYbQZ5FPvZsjCLVKi2SaYkwMyp4Go\\\\\\\/TArly75afvLBmeBLeg54deWhxs+vBHTCUi4Jd\\\\\\\/XOewcV1uuEolKcigcT+XLNlFK\\\\\\\/ZMWJ8M2VdHsLMyzEXZw08K5Wmp5MZorqbZC5\\\\\\\/Y5U0JH4HI8NpP3NraVnDP68iCGKUfiomONg3vYOQvIADFtkgRToBqtv9NExILMDxB9N4AGeNsdsFw\\\\\\\/DaCX\\\\\\\/zMSopiIbxriJP+cMK8l5ofkeFH0sbgbqjlImKUApkLE3MSgg\\\\\\\/UB57hbInwhwLvucJdHf\\\\\\\/hxaZHLL1+xl8lFTQYpnC14ezBuoPzTZjNLnalNA+5Y2Q3ph8trqT1cv5Qjsjzm4vBqkoQADYHqhBAHOP3aT1z0JNmsLrfz\\\\\\\/gUnfFCFgq1MFDx1hTlwErtZTDGCc+xkvhD2HRV1lK\\\\\\\/d2xy9kN+FI+CSQ4e5FihAyjeHx0oskTibSr2V4b\\\\\\\/dBqlqRg2P\\\\\\\/dcCXFPROJum9GxYrRt2Onixt6GtUaiHeSARexeKXuzFstDLzwWokjYhQGlw6S6hVVjnmUaCNLymeqW6\\\\\\\/UyvdVQ1cjiY3GXfYr\\\\\\\/CUh+rdMCj2acPhUpasePZ+MQivr\\\\\\\/pXZra6eO5Idrc3uiwvhzhVGXGyYu3f49OkDucEXSvDSwk7nDQyWIq14unhIbWZj\\\\\\\/ILstFyWJ93xMfBXz0Ngve\\\\\\\/NzVo0+NAY8gYKFNpn18cm4kq7cleLVfxTasJpYeKgH4N\\\\\\\/zm\\\\\\\/vBCw\\\\\\\/WUBKgHU4dI3r\\\\\\\/1db57jeSpc5OgDNUIa\\\\\\\/7eBVK5mOtDhE4FBiZuGltG7HhgiCRpzczy4eHx3joQFsvZV\\\\\\\/lO5PMUiruCATY8SYS08+OGnrAgyDFoVl7VSdJllU51o8Z72VlG4OOs15H9hH1Rgc4sCBHL9Vmz7VHckxDqxHJslEpjVNAo5jsKcidsZhjclco+TFovxpovxOf+GH9u3sJdHglYl2l771SLut+EVY8AtYDTgqD2vB67gzUGpM+e6WMucc9cWKSuwktlKapI35o3lKY18BQ18JCWhe7foM1iDjKQayOK6GNElzCxWf9V1G07fC0Gntr8hASLiWuWd3+wTuPNBUvwmL9NXBHTLgOJXrLqs\\\\\\\/R3PIb\\\\\\\/slO\\\\\\\/tB2HaPLa6usYpoJrsh23sqb\\\\\\\/vw9fMqGhmmyfS4gSKz+DrRUxQGuBwpjHusDCPKTY6dH5MzJc0BSo+\\\\\\\/BnarHdEqV4kf7bF+v2zi48WoMLFw54C2Gt7lIQTLZOmo1DIjCcNBqKjfXo2tHkE31zkC6fk3ToAhS243T9nd+B\\\\\\\/eqKltFoZfsbrrth7HRtl6Q4rtzrA4lL9O4kxC3BjiQ2NjPPESsMip2VEnupxbW+JhGWHPH+QP3oIe\\\\\\\/PXObSPvyFgTZ5MnOTJV50l64Z73pf5esoAnsDE\\\\\\\/k\\\\\\\/6+TyX5Mm3kRo3Mqu0rOTPvvkwqQngXUu8VQc1EoMuITLrWLZcKS2MdY+Oc+5vcFybeByTH074XTRNch0A8EoarCGMOHZyyzMn9cjfGfsPb+Ddf5jBe2aR4pLybMj7CvmunKjU82Bqu3eONgM+82B80ls\\\\\\\/iuexzsj+PmBWxnGMZMT7g004SzzWUvUj6XEoxgWrZtUez1siibh7hjJDhTgUG3IcNg3\\\\\\\/ygRd6Qxik5+fZ3jYmmuM9674pJ9yhyFAY8s50n0S79glzQEvNh1Sh9ffCSYuTRg8y0yiJmw6lvDJZ4DQL686rVcXV\\\\\\\/RuLUYrAN++w9ME1LBwEsvfnIETAZEiAdhZBWobt0\\\\\\\/JPEG5UV0\\\\\\\/onmwWIZNtF3mgooHK3FU7b7jg6Wwj1wVzNUeeeB7+lyVCahCjmTpcd+Sa2dX\\\\\\\/Zm6llX+4ke+1RSLSfxG8Qdp21VHVvZ5X4+Xb2VLlLQoJ+dJ12x0eCnzKdRwS+\\\\\\\/FJ7Tmad5H9+3IlH5bQ0buWpr32f8u+xDw272L5igAbGWgCz\\\\\\\/o0Dc1NWDh0B6kgcbcunjQvh2fjNzjxeq3m4\\\\\\\/aqRjb6s1kNDXDAZ2kKtm18TKJBpVZ4UFgzRhv66znaDRiBOG9xWKPCZLI5fNF5ODBZ16g2gCCeTPQcIED3NKz2rnRQN1cfFeXUklmG2kIJZ1Nwan1fbr9ZmoK4smhWkEVS0rVvnw41YnaOA+rBs4TlFEdAvKv4MMl7AIYtnuUyeitYNFLqS1oyQWVIUvxNkw82ZfYKU1m5UzfBaWA1wvciPvCBLAATphR\\\\\\\/lXE8xH98bjGIfWzxagRZrcTLzrtnMK0rhUuYmf5G07yZ8rRN8m5D5GaN3ZEPaP\\\\\\\/1vqgeG3SFYlvgESzYuN\\\\\\\/EScPgczxj0xcx6ilZ7xD6mSa4hQW7oMYmcLR\\\\\\\/BlsnSeYBn8O2R1HdsQJdRA941B8RKYRXHpstfazdHzHIwRvfZCdhh4wjtVLO+FAphS50CRie1DEEpZ33dYjdt1ZcXa6hImd\\\\\\\/HZOKZzYSwUw1nr3eurQJgqyXfo4lNaq7VBMA4LLGvNiw46LvYGdsWc6G3hikNkmGleaIvmgRHxxicivPmmL4Joj4EGHCAt82ydOCFV3iNt5j3d8pbnwh5J+zLB29nX02khWZ5fQvtoySFeEnztcHJr4xb0XO\\\\\\\/+SEbexc\\\\\\\/AWeLWL\\\\\\\/gq1\\\\\\\/a1FjbNKlVktKGpMsC3CCs9JVKq+oX8MAXdhQTTbRD23dUtsFMUrdYJ\\\\\\\/ArawhYQsjhtBsoKWnoGhMhnABa0k+Pe99tna8SeSxkbUkgISsPKyajG\\\\\\\/ePyau5XcsN9lL740r7yssBadqQgF\\\\\\\/4F0e6TEanAeYxuNVExYLbSZJMQ8ISCJkLC2x1ZAmuehlEvALhlbroCyqo+Nm1pO3igQEY64jhoa\\\\\\\/5trdjQ0LsWfk56vsoYgFYGoTw0mEIMehyHGHEc8eALXZSNFBjwEfZFn3Pi0p869usX314IHwj79VyuRmyDHQMWHFUlXQgPpDW0jfm3I0gTkCJLQT1TTTHsA+aG3Hq6W9bg5+JfoRYIqQTmuDOM6pFQ1mtKBEHQdFHD+eSwzGocHRQ3YgV\\\\\\\/n94JMu0FvccDQ63raK8tCJ3d\\\\\\\/qg4RRU+9uDsexbIiH2BZuLjdPZSQhizS5QqSUOVmsmDug4dMfCx+w022OYYFZ0oY3gW5nqb5vJB+vINB9NwbAQRILOqA+7nVoEJkmBGQFZodnGao71o7Otsh7HWtS6daCO0YcV5ThQykrIrqOUE9l0Zv9PQf5BQwaQXs4jCbFaEefclqthLvLSFmSiWGnUMy9yH3OAb13k+VMT7JACgHLl35uxDUOQkVNPZ7pCfpDBlemYb8xapDoKegujSqyZManIq806TgQrQgRhsYhYxBptmYB+GJhhXhzwU+V5KwSrPUT3uJ9ASEqSVKfiGvOtzlOS4dbzFRQNMSx8YEDUqU5IhFMAX2JOfGNzx3i3zNhX3YeL4cIBKLphq2t1or1tljee5h8C+HfgHLiAbUWnVe5RpwTtCCGLPq36irQ4bLvAgsqtTq6D8L\\\\\\\/OZJEniyAGEqX+X\\\\\\\/VCF2Qoat7JQ322tvM8m521HsxgMkp9671\\\\\\\/Lh+dFrcmE3uOiyQdEJV4kAqRJzor1N5JKmDWUGMzFL5HdrCgXs83frCSiCQCO3K+jrX1gqgDd2fcZObTiH34O0XOeAOSU22gb5WytkBwDUgjyQvqg6ZbY2OYmAY8fLuxPlZz6X\\\\\\\/DnCtxkiKfnWFy3sandJxyraEeBI8BoYWrgn3W1la5\\\\\\\/VzoqG4YoPbQWwNuLxtLlsYhX4wnm0kngVW2rmjMPpAIR+cWLo0UNeUalnzPiJyly+pwe\\\\\\\/A\\\\\\\/F5KxO3kFYzvnNtRKz8Acrk5dNIMfE9zWJjEuzOWQoGy6Em1CxlaHAc6wrhigOc9z4mSZee4o3kYtcETOrqWrnUgOXC1VTRu\\\\\\\/taOLZsV3GzYeL70MmUi21jL2rxK1UxNssxVahSvvsZey3cHBuDQHekRSSyhYNaEKY4YxkwfBun2LaX1Bs9G1g3B4rYK\\\\\\\/TlkFXX8lUU+BQatgOaHHgjdbnpAEVja5kzOWgptOTDtP9ZaIVt6wqjKub\\\\\\\/j8gErq1Z+01EcbZsxFzUo9QO04vAYj34cNbj3Lqo7eODGTdg0\\\\\\\/BOeXP\\\\\\\/U6RLVvK6ey0BEuJcWpavn1TbU2zhJbeaLd3Jg+6sS9Sy35F0qUzBNLzFpslqRSBQvQkibzRne19JdEvFiwCE+3J7ANdOJJmFO2vWhH+4lZorP3A7bDvlP\\\\\\\/5pc90Ne6WdSvKRQPDl\\\\\\\/yKPviSoTimdZWqW8gUtAfdN9+c64My9nfMEadk65RVi3I8L\\\\\\\/a0ffHtUNjWn60ZKF20fo7nqbB\\\\\\\/7Z9Lf8SQAAZZgIl2YTxBENEIXw11tyCs+\\\\\\\/gXtd5fPf7fwhHkwQC9LM03TEf0\\\\\\\/HNnOCCSySpsiHFXf7ByaDZCplD6iWY+X5P3BLEolb8HiXM2xCoDWWCvJnuuUesH8hkrAXI1P5KEwtCHkMXqGNut6iDg3m0rTdvqpiZRocOezexDSeFw7GiA52L2M6+ZgvISzaSGSjKDmWgxJL3exQ+W3Xbsb9\\\\\\\/vwp+L8fXcWdrA4G1XyLU76TnfAwr7EK00FKG7u2+cNsG4ZLez36GUTuWKnHgDrMd85DXoD4XTY748UZR+SBC6DhKLouNB73+qoNwyJ\\\\\\\/CPiCnvbV9+ffN05pPZ4R1XPHuOyTq7cSU+SLp+lFI8ImsEwzdYpDfYADdZmsh1zmSOnFOhBsYW5deeNJyYkyAYW\\\\\\\/5CF78oO+96e5fndST1fqzTxRLUBlKg2\\\\\\\/jUdjcWEA\\\\\\\/qXkmkmlNOXl6jiGL3cbuC3IREoTjskedmFT5IDa7Nk1kgR+lSLxCSKzVBOrQP2OQfLg+nhVTTxWF5E2t4M0RL2FouUwr5Jwh48xOYsQN9rUgdzEDhgk89IOs0+odO69txKTj\\\\\\\/vnKQmBVCXh0U2zz3TOosB9k5T6jEXr07I9DykdkVlKgOQjU4hawqxYw+bonIAUyipqtxSwFu\\\\\\\/P7wkqZAq\\\\\\\/75Bxmn5ubFihRLq7CFWWXJ2aJbmx35Xbho\\\\\\\/UMy92JRQQ67Xbnal\\\\\\\/WQs2OcacpepUTWe6dgENSb+R++mD2k2ywYf7BXmf9J6Jvz1ZiQZjpaQ6yc8\\\\\\\/K5vFZhEfUe1qtTYSuxWWMU6aAAwGBzT17cbGPsPd18Ctt5tog3hD00Iaz0YLyVbpj5mboR2Ikyxig5rKdhTVWdewUqiH17noDnIxIfYODj2ZZcnHCK96b420oqciiPz6NPRUIw2wzKNo71e3fbT7I3Ao0W72HJjGHDp3Rx+bvpOaOJ1pMTjCkuPgFduWSrUTEYw3jemaQBhtjTSIpQ392k+F4IMpi5XxpJxTMoWIJ0gJE9NdyYcowC\\\\\\\/BI5ii3f3QU0OxeRyFsyPKlDfbJSZ4gXD9ZcDPTBD+27\\\\\\\/4ewRg\\\\\\\/DyonB9K\\\\\\\/GavrplkNNPNLgnYsv\\\\\\\/Swuc8vwsw582n273g0u5MnjLeN8mUXIglP1MQRB\\\\\\\/m9T2h2\\\\\\\/NoAC9eFMl6HP+5j7\\\\\\\/V3TrTZwovnZIajDYojedmwbZMSJ8ZtEwKK1WHi1UelEOJ8zQ3qVYTNkkBB1U+wnF+iWN0ASJgFWlPtwu3vFnzR+TT5T65NZAkpT0FNfQAEsU5Vhd0qjLBZ2ckoIUTblCuam3jJOoRGN582NB0r1Y\\\\\\\/qSvB3LoYiyCTMQlfs5uX2aUxcfYu2TIqg9wY0a7Po9DtiOsUYOmOZpEiIi1\\\\\\\/7mx2zUF7f4mymVskRIdZ9W4kDUkdXcH+s9xfchEkY\\\\\\\/A9+h+gH2pp1SC47CZ8oKynAR9c18Yc8tm6kLN5DZEiZDuu5sXiaM87NDu4px5cN\\\\\\\/txms6uF+o+O14y28yoGHEI0fEmLHNNV4ViwALUutcYe\\\\\\\/hebGw1ypafwZraAJxWmi7R4HQ1Qlli2tcQjRoP+EEJcfaDBHSS+iZWy+rSFPwxZOmNBOAnmHnwU4qGuhzjykNYKaTjPmwf0Iy5jWbVmcH\\\\\\\/qZmnTE6ojA0NCUMUeLLckKL4ZXpLyhXcSywVf0DVz48HOsYCX08gj8v2\\\\\\\/8N1MI0qsvhluIGoW\\\\\\\/xwgiSiTM13RDySFnmlf8jd\\\\\\\/ADm0ZpHrxHSpTjS\\\\\\\/K4EHex46SoLCczAL9SL2CP60BoXHsfXqoXhi9C6dTgcOKkOuDWjT644UhIvWz37AhS6h2ZC8SwWQ6cH3A5x5i8aAInWvBIbTX\\\\\\\/rkqZI7kU+pTtfeONHtn6GAFJPyyUDqxZjKOiuHwxXYmKuAgth2nziOMI+QhHET\\\\\\\/fYeqs72mj7\\\\\\\/jfuAiVYQ3EvCpu9vqzMrbqyd1ABssjk8izIEcYMDLyhH\\\\\\\/XJufk38l7klS4rnfptXUn3CEPKU2qAguPKu2rlHMu9COiHZXfknLaPIgZc0cCUKELFaG\\\\\\\/AG4cpH6VEbnVVln8Do8w6s9LKEZh+qp+ajwVaZ0sDfhU0r0eF0LsXzsAFpnwAP2gsH7F81tn30zSnYSX+j\\\\\\\/n9Hz5JNyxzwJJR6nOjB+sq327G1hklNZSslNR\\\\\\\/7EmIRHpC11h5op77sHyxcEk7KXfv6gpqiFvwnD4Y3CAV831H1U3FbZqaXIXmRIQ4ttlMvudh2bqqmUMizx4vqLL4TzNw7GfJ7zdMZch590+1Ctv+nIzs2842JofABgbn+4o6o49Xr1lPYknIpQJs7NUhLgymcIe6ktm5kU5LyrpTgZkh2vysWIxcUMbBZ0uoYeKw44RaEA3e0tVk0S1BfcwV7AShQbrVLJdniW47CcFkJUJdoAh3M4Z+t7IhnknvSlvRNASxDCe2iG+UPbjLcSDN3CkMsiPklrriaX7YUj+dKEY+GJdcFoZHLYj0Qw7CxRhNKi29GZbcwxC\\\\\\\/9QE2\\\\\\\/6eTfUAIdaPOwJq+A8JzouubuLNT7aWjO6JQmreqpFyoJbY\\\\\\\/j0Yx16NFSuZc9oY6+R1NDy264LdvY7sgY3rnDvPrlImo0betoWFtgRUJY9FF4Rlt7qFa0xdYmv5iuYL4xwDYVnWkl8s3beRwuD0RRwP0ptGv2URfKGa22quJgu75\\\\\\\/4HCUvwT8NYj3tRKNCxBCnma3jx5QuWcgEuFT2hinFge21\\\\\\\/0Lxtl9cYWCiVgx7YBfJLYZHJZsR5xAal8uw2sveWxVCBKzgIAAA39W6C315\\\\\\\/Rb5n+QVuLthRlttSXL4pFWksB0HbptMu\\\\\\\/BEjNPqOrd5wRygvHC+5zoIMBX5isDuf1gpFsip2rUvyhwJB+LCn3g7JT5LRxNtEHZY0EmcG0ZT4ecKOkAeV1XyP\\\\\\\/i7SjbT8Agc2GaYAflSzfxaN\\\\\\\/kaAR32JJwuFUvjbwS59kGsmryB6UlIJc1SAnAmlxpXdb\\\\\\\/aMRrYXUopL9f9jcvM0j4Mm2a81ZZTSdprCoIelnmhjxJcm5cOrTFsJMwcJNHPAPKxmpUHuISJARhG82IK8l0frF0MaREj1bTq69aG6F62EnCrXHjPPilapBFxsnCY1QI0HjBpRZlFgu45ZYCsk0tEN99kR4EAo1PnKXtTq7IVCLs6Ts\\\\\\\/UnQ9CPWfoGA7oczWfI85BOVd5aEzuPN8pU7XL6ZnLigaFyz9Hix4OdI9R2V+9KxmCML4x6o2mtGHd0IEZLuomms3U8h1FvvmLNlHlhgOhLeigpfxVa8sbg+UKsYhqIVyupuZ3d7C+tNXU1Tt6Oi7LjqkLTRzYI4l\\\\\\\/nMum7Qsy2uTzwUrZKWEpqiEZb3icvqj2qr2V2D8q9hGEHPUZ7yNhc+iq3\\\\\\\/PMTq1QaXlVQkRj1IfzrzPbXEXrovvsXywGNk8SC0txXJcH2TKOUMb4pHEO0T99l72LSmFRdYpv2Q\\\\\\\/wVyUDAO6ogLFwaG1Qg80nOiX9eGLx+Tbnf2FPHS0A0xXm142u7Xue3bL9rfR4A67DOk8wgtpeiLuamQjbw1YTFqoxJcab2HWW0KqeFF2cK48yVZgBQw0p0Qo4SFHM5o8dNgcL0ApavAsSY+lzFHRsKuIE\\\\\\\/bfX66En9JmUn+S9GSJw9rP74sglsEbS87uPcCENcotvYECfHQKTb\\\\\\\/ehWzSM9vTLNSa5FiLlCFXOFYqJ1V7L1aLCfuTxZTGKElTu1PErSQ3Jpm4AmwEhfpv9LUjQCHBoTvxZb2MSpkUkKW\\\\\\\/UxbPAIb0yDQ7CsladL7ir2HUR3upG6XBZrbwgubLb98qdZYIx9Uy3EzRoTHjXOempr6X7OLtu\\\\\\\/rL\\\\\\\/6+8nzK5lf+183GUtyxuEBU46TYnucnRyix3MYrYL8Qlc02sQJntsLUtI+cfWI5Y3sgpOn8PdwdVPgrPEA8iY41yUqYN9JOko87RLfYkrknisHADfRQSAG\\\\\\\/dJu\\\\\\\/px9PvEfO8jnSuXPyS5kQwnadLU4hC36CBtdmDJrcmjE21NbAcBiuqqxsQQpVc7jwDREWmO0j2cXQ7g5rOXQaHWyjj9zdwo7eJADZkgrQUH551gpfry8eFzaWTP7t7DpL\\\\\\\/2gMNp97wNIToWMI1Jy7b7iaACIiSklx4HhQgHW5Uvf+6VX8y2fS\\\\\\\/Lb9a\\\\\\\/8gXoc93\\\\\\\/v5DZ0qwxkpZzODsrW0tYhieAIjW5gXyIE6X4qq3Y9jUxKl85ldYnNc2RJ46aWPobPladk0YWbjVQpMlxM8nK8JY+qnJQ5B333Kpy0AaiOrNoc\\\\\\\/TDeaXFGbCenqYORC\\\\\\\/sjU3mZaVawl0Cfm90gbnOGS24V6sJJv8x\\\\\\\/hTUUY5QbJkSM7NSIATPJaE9d9zYL6YzimNicwwSaI8CVf5RgF3P1bzXuqFnBxl1oJ8eI1ucLU6q8sOd0048xcRLdqPnZBLtFD36xY5feXDkWbE5S+oXhmKYYDvHJxSegPBqzo1\\\\\\\/ovq7LmVBWzbp7z5jalQS5sOcJzJrymEYdu0fKoR9uTWHWIaOwxlaxOf6oDRs2d3QN2c3QiCoiqnVf8E47G4tsIA1kIXtxJT1dJzZnTA7AKZ8FJw0ZLCkyhIkl3LfzetdqjjsbubEYfzrQbj9VlSFrHmU\\\\\\\/sFTfCZu8YqqkkB8ZrgHGPVS2uzDcmXH3gAtjpUAr5IgEkOI2EABwZqKFpc7Wer4iEcTX65W6KepxannhI+CYmHBg4KpuCXAtwmO0+3JPguHF\\\\\\\/I\\\\\\\/d39Dnn+zpFPXr9rl7NCIzU6Z63ao452cdh4IY9+RmEAN0cGg2EvoCkKEXtBPFFE7cADJDByDJ\\\\\\\/hVIYcm1R3xiiv4srNDfjJ9VA8rnGy5n0iV4evdA81FcAGvI55GKKo49hzLCJLVbxtWG+yeKfs7vJhEJQbZrkQxdUtUFCUu0\\\\\\\/D+Qxmmk8VW5R\\\\\\\/BOa5QRiqqyMKB0+9l34HSkEMMvkvDJY5EvUVvCrS+MeQ5HEU6x+tXMGNOrbR\\\\\\\/cYukGXTDz3ClJ0fPxMfKHGkmImk4A1JFL8NEBroYAhjgdRUCqHpmmufpKHEuhukQ9b03mYQu1+9wDOr6zKZ8EMbJ5os618VSSOVHUIQexKCD8+RRqOvzC2P9HRFflIMkS1I4S1kuWI0JEhFV0mzy8595paavLaPrQwXbQTYeaE0gJoZ280Gc\\\\\\\/tx\\\\\\\/AyBaaoHdkg1V8awF10V0rRl54tnOIMwKSSly5IP7mplddRbUlENdH4cRW3KMeTx5Yir4wRG6fng2r3isFQziMOWLoW63K2Hxw74ZUhHMeb4nrD98x02rpEWHzGTuoOy5oPL+4tEdzUuIJ\\\\\\\/s7hNNEdhXQUwnUFOI+AFHx6y+tQExMXFHyfmncMnpQB5rHrGBxFpaqZeij13W50IRYcotNl2ZHV6ARNWQHNMcWXQAfjrbRlFxFsDEgr2rJDKdxTdZojy35lhekVGeOqaZyoc9Q7j3D4D+HaCKuK9qLR78UxamA8Kt9MBOdqj\\\\\\\/t+yIE4A+KyTcFyKKbJab70My7XqV75SJzUVjK20ZyH1lsgZnaLEkOzUAUuy6prKNguhkWqzx3rtW\\\\\\\/oqxMuH9U0opqeK4ZMkruhEGsm0wotK0QedtVCum0RQqAlgbuGG7cPFuXAVsnUZbEQy96XAn0q7GF\\\\\\\/ijn4tZLYABJwCs07IMtpvkQPKjbsmU2Yc1Yd6RlRrx2EVXBW6QcKRjki6B29hriHXOgpdvssbbg3vzKa8lIGe7Ve40EbPIyIIw488UlZlxoz95\\\\\\\/165XZPBTOE2xBJ+nmwB\\\\\\\/QmNw9GfO9Ik1290muhKEemj3\\\\\\\/lFqPUJ4LEfZwTvJ2agiX0kW\\\\\\\/DeoODr44rJDSfpVeRMxWdDHKxjsBpOjA8lsVHVYQ0zRNVW5N08im9Dx+Zxu2V5BpgjXf3vNH5Y+7\\\\\\\/coIznLmfvF1MXWGEIIO4lkHH5FtKW1cedcQSTRQ20SbkoSlv8hDhD\\\\\\\/a03zr\\\\\\\/sCDAWhRmAsybucg5B6Rfj4gOr8nv0QJ\\\\\\\/gtH5wLUCFK6Ssvu5ACWKbVKXARnFNPhyD7AzfNBo5uWJEp50zkUxn\\\\\\\/MSSxh1Y2czM\\\\\\\/C5uqzVKHIgZPhD+WeL6iqdTGZotPXfl70eMI\\\\\\\/imz75xKw0mS46A9ZyVOAxxFlgEbP+2xmPPmbU\\\\\\\/fjd4rnurirpYchX4F9RZ6h3naLsXPhHAGLqC285xGOgggg7\\\\\\\/PQLlowal1Q4blCnZS5lR0SK72kBAeHwOr8VBXtKsvQ\\\\\\\/dpYU7t3Sr+UgO2kG\\\\\\\/z9UeexHhC9O2asVGKKb1lwuYEzjbTn6X29VWnSRbyNORrYA6dKpeu5xT2ayr1l1YEcoXJrcK5ii6Td1nH9saJj3qCLgMmM5edMQTm0JoqatH7dRJtSEDUaxhh9kDnJINwmfTDMdRUH8mpbDfKI1bWf2jVaZicEY7utjJqLKaDt7p7vKEfUBulWAnEqAwNj3xqapq4e6ReH63wU\\\\\\\/mXbxCmbYCnKt9aZ9GkChzfzEoO6mbYBj8Bi4+qcHhZsMsPnEvCbWVYG5DqQS+MOY4kMQmyHF+pEDHGPCjmArHbGaSMvYa2F73fHnKnGj7FQAny5Os0f8JhMjrnYd3mWYJ5quXV5MSW0eNivgQ8vPx8m\\\\\\\/0mRAFqCXURVwy65BTWSQDYcDg3xhnG6YHBfnbR4bm3yVyOVvRRlx5DmAVlAAaM4mg59A3EppnCfHmPr9lPYganqlcE1c71iW00Qrs\\\\\\\/u0jeUM6ZcuCPqMahsk8Ni5fvQfWQg5trSveaHoJEpNZVYndnHa3wZAzu4q5y6orvdxoe2y8wXlxkksdPE7kRc8ElIyDDlPBL1rbyEB4fNOuEaVxOwHQOtA6Im74k1XSqfwarX1dT60\\\\\\\/Gm+eaA\\\\\\\/s8KffHcjPXzKpXZbzJ0Qsut1a8HSu9DgInZRdxP0qTkCCrBfhTUoWEgfcYsBG5jifHaNSI26jt9PLTaNE7Riv10EjHFBDukDjIpnhUvf+KJtymSUbSfX3g0Eq8B2FY5HP71vDeyFuFBwjDXKq09FfRYHMuC2FaChElm3KuHz01N\\\\\\\/cl2G\\\\\\\/ljiG3Z4Vzy4AhDI7bEkzU9houGL88HTDLy9F8yVJ4kCjmb3zOpNggTtw2k0TmPikllkcndf4RTjteuRiRxlr7HLtFiaWSDdtExPWWjFFl1zClvXB+IZNHtu5Q2G7xNtZj8QiZJfZX03sq7VeSH\\\\\\\/TccNGBXaT7UmFn\\\\\\\/j+JhtjPlqimgyQG7RfWCzll38N6Cf7DE4qt+5L55UhQzCqO94pmWoKxpPaKx5RdoqJjk2c3a4cGK8Os+xQgL7+o4XQhOz6NEYS5dbj8mNSP33l6jmwT03hisF8w0hnIFzKfP2b\\\\\\\/c64pjq68dbVt5ckhUSVw3y9DrbWYiABRSzUFQORu9vwYiZ4ib7VuLpboTAcfat+ktRTe0vboP6X8rCH+9Lj1w0ALpiHQCD0ORgLL\\\\\\\/HuAZURxOCY293\\\\\\\/5wyR0tRrkf0ItU\\\\\\\/AYwd8fdPL52JLDzWmTcpE\\\\\\\/00nIqvTnYyANwyhyNrKEAPh6IjC6GKNWuBm0d2qGe6c9Du3KSgQAZc95fN\\\\\\\/fOM44SYXuLYlGgvDD\\\\\\\/lXgkI9ggK81XrIDW67Pu2PMRLhb3rjGckX4ke2uNncL6uC4KyaVPvaLOoJXNcaEQCrbf7tagO1MjO097S6kBvE130FyJwBj9OYx3Tyt5YM19uMJ86T3IhwqQcJdEm6wXTa5\\\\\\\/MnbL1IYOG+RRFdWmc7\\\\\\\/VRj8q+SQcs8U8XRKgxAhs64bUeKHc8WP+9lctewrjSMKCPRpE6A6ixr8+dstX8\\\\\\\/yVVYh8MMzrj9wUurTJ8r1zjvb6PuCRdiJi+OeFrMWlgP0Ccxf9PI8yOrd8I6uzRzZ9q1E2fzsXR2OWfxkNuHEDR36GHZ86mYoQ3cTyZE8MRaSNwso5sLhGFU+ZSWg02Vf3pk6ps19j0c5JsAWT4hiwjeHLAa6KrTFFJaA1PmbdAWkauFEaIxtEcuwrJ7gb3jENHMtIFgg7XwzVsUbOVmK3brOQ\\\\\\\/RcUXFMW+8Gr0+sc\\\\\\\/UMIfERGgHNTN2ULIkDGyEsYs0uY4zZqH5OD8uoRfobb0DXtoqY\\\\\\\/o9j9TC2kdqmymUBICtD\\\\\\\/UtI1G1s9JDZH6Cz8X1aPE2eX7bfOOFAy\\\\\\\/xzIWFzShfYQrzfVEIKeloWgqmlZMryCdmLcsgSTnJNFXXyLbM0dabno5\\\\\\\/BD3rIjMUDzoqwuGidraa1kxdP3oG1DdMDvfVlvqrqao453kZNxHkZDqgX+d1SYQdW05MReZVv8w22sT8SfW5j\\\\\\\/WvEEQQs7uy0U1jo\\\\\\\/xCqgfxRYq4X3sA8y6XMEcHxN5gGByirUNpCJUzPB11ZH2C+XehWHZhKf4uxOj9W+pYV+T11e9HCEpoYZitWrFbtJbuOfJxBKGxVKt3cQy+iJlqGnK5iJT8XsP7ccdongXuxPxFvW23rccFZcAwVuuqEQs479UWTtnWXrLb2S+b2L+suNRyAqtuTfqDaEG169Z8dKqQvyBUMTUe9rK22qLApguvdBsYEViKKX3umth\\\\\\\/rww9P9kmj8lucSQyq+zPjETN9+J\\\\\\\/DZi\\\\\\\/fbBQdW8M2sncxWybf2Pr5wLovLGUBqNkURLR2kWtmXMhM9+r\\\\\\\/t9pxdD53d5mRzA10oCVUmcw2tODoJhcrH7bqhvPtDsTPmmWOVXHdq1U8Ib5Xnfi5IfNk1ZPrPQVqfIEybxlOeUWRS51tLDcvykd8J9oQv65FFkTuP51aEX8ZH8PowhrrrJOEqCXgmNwojE2ozEHI4Fz+ff4sq8QuWQrncAm2AslB\\\\\\\/gh8lEz2m5nsBp94gNHm0ROV6o6\\\\\\\/kzq9xPb7GJtDi43F5Y7ljQZnvP3ljbQjqvKMZ3+rD51oMZ8c97YjaPGyoXj4qrsCwfDtCvjgbYcsVLDPRymU5pSViIe7UEOgTRgSaHJS9SbpFKBUmP04fHTrIzN8xX6l2C0I5BJQ4ejTUrEFSgfZ9p0LIMuKCXMLRmcrCOJPkTfA4ebbZDN8hRrMZH+DNFc+X2W4412LaPk\\\\\\\/eAEeEy2n35mjAxe2s2LYTycNgc6FhIJGGEQyo30TOvmGLlkMD0DRz24L390JCwl1aaTQ2jLxKVKht1FO7LUe7OGSF8ZDIn4J4tPLMNivj3AftJz2lwGmog8dSebGF4vPIleHXmRNbbLnLL0BaQCtj8iG7vBzEJNQ07MCRIV7jMKZEOymaYqugRMeTNtZGCESbQRwdyo7MWfso9cjq9vo65fD2I6HjnH4hN1KNewQYGHW3TB8Xjx3LRPmleoSbyA\\\\\\\/OOvWfS0JZjkdp1VG32bjaiN\\\\\\\/iwUEZM+Z9C5qDn\\\\\\\/YnaqYf0+zTjqrm+ltuA5w5+4uC3DS8eRpQMsnuiWieCgKdOYIVSRceuaq91neA0mENdIq39STD+G2lwPFt3fvq74Fmy9jHJDi6IjU2ZYzCFRohXg6pOgl7z7HrJpyjOIiYIV1+raL8TmdzirrX\\\\\\\/p+Spl5l\\\\\\\/LLMdtkLYpKksmFucspnYPbOe8CyapYZBrTbDy9f2SR9w5VFKXYl3H+LVgExZtRUHfI0XwCTm3IAE7\\\\\\\/3iHIunQLN2qqB7E97ItuMfQkMicsI\\\\\\\/ZiqPAloIJmI5CpBIEqMYAYWdahEh3pd9nMCxXPo87Zz9SER9rTFrR5uhyxq7rCzvFmCTxMNFkVNQ5m+wURXKv9PHdcVtDCui8EGE\\\\\\\/dJncilxl3kzFQtMTm\\\\\\\/dQPojxbkCOVEsER0dl+dfo0aABm9plmC65VhM5dQytrb\\\\\\\/L47T9Luiq+APX37PmRNGiNR9k3lX11OXKy7v7LWXvCGqWaBlFPCAGv8FXeIcYCKke8Q8TzWogNrWw1jTFh4PgNCcmtuueqo3W9y+Q4oHiBQxN4fEWToy5lDJyUZ7mL9ycBGTjYJqNBOsz3q4ooVeD5avXm3TIXmOoL04HvtZvnXyoxxTl9XVro97N40nmFYyErxzvpJg07h6aV7W8R8HFlUJgGGcJamrqGTjNJsS9+oYkDzxZuMVjh42XsrZvrHb9XJ0JNrT0io7ZRaPGLHoFyQ\\\\\\\/hQ1FNka1hQTQTECW5XkIoyb0di0WYGLiitvl8lzR5Zntv6glCpemu+vfYJGXW6FZsAlI6VfWp77v8hhIQPg+3C8qQO44DqElr\\\\\\\/TnfEPBFvGwriS7MN\\\\\\\/QZpMXF1Hfo3PWJRrQUNKjoiccxBRhIelZm3XjHPqV3hJvkwb3UQG70Woco3ra\\\\\\\/GSpKaucoRKcR\\\\\\\/WeNUZVxgePvrYh8Cu0uIEapjB4MWzNwTUxsZs5khDIZdHDWSLkC1mDnhLCfIHH7+XpcwluHZf04gsGxPTXLxM34AyhmbXBL2xGQyRZ1HUDRCdokQ4bp210OMOj3u1Vu\\\\\\\/v1jWFZuVn0hkGQvaD21yJLtBmprc9foRZp9raYuWJaD4g+OZ96NXjSJx8vAnWbqgqc8\\\\\\\/QhLF1HFNhjkWHyzlyx8UFMPQtQ3Gr4YIVSgF+NVL6ty4VnDk99mTwZP+N\\\\\\\/uWavAT5wbn5MnMbPZaSjpUMsVWiXymibzTaRcFjolJV7t9VwLT3Xw6caIFVl2jfAsNRWemjgWc3jkg2UHqC8FeFXpeY+YXsSh9JLba0KB5epJEBFgY1NOq85ZYiTUeq9pkOYOcry7wv3sLsy4Yly64Vzr3my7e+c4MWfb0t53NECBNTfznPPFbY4h0Z2rbcKDKbbYi0INMj5aS77ztTAio3lYAe9N2V\\\\\\\/rLm0L+CKQPaVXRBgQNskusqrZCGfLK3I3KJzBA6Qm+Bb\\\\\\\/2dsAzUdtQs9\\\\\\\/G5m0taph5dhqJxXAg1TH1A7Qi3HrQs1sn+GuXuLf5BZT7yv47UWVdEd9IhB6bDmMrYrJvfh4y3UgqBDV1LK+9PkyIYOjOcBO89CEH86Fg8H22+MrH1T827RTXJXL+auTj0IBMAydHGaEMMHXPILj2BZlg1UOuCIbwL9bFmSkg+nAzEPfvYu0zUOnyAOC0S7fJqyINEjeeV0A6DRubgO6OvWBEez\\\\\\\/rViKdAC+gl2f3L13TiMWo43bd5kjgs+i2W+ksp4BrpJqghUGMEh5q85y\\\\\\\/83skGdxW+5Gj2HvPVObCWWE3HI8vFPwmkbON++OWxc0PKPlRms4RJ1fL1X8ye541M9s9zdsxQDTavBTUvPwdX0CBQdpqvmuFED2nwxIMHOGAJycpd7f7IRZodq4\\\\\\\/H7ZIWGEdHNyBg9YsxcBmUB0WNJOonK5CoO4lLNJmVviMCJ9cIyKpSc1GCec\\\\\\\/43WRlV77610ipibIrcJ+ApD6d0scIAGDS+0Idf5bMrkZRBeafUrfNffHUYUm0otnOdYb4ZeCRlVZkJJuWa0sJOOGFG91ICSDbnnZir6P52nLyADcMDJHdeUw53BPLfniL3yQUQMYnZdL1s48jvKbh2nZpMgVgKWhPZH3AKuIeZxAbL\\\\\\\/LzqHsaMvAodqeGnBqGoxonY4mo7188vUjH4ap3nrWKNQ9VcTO7BZGkq3QKf3ynCvYLab1+8j09ZwJKXlPtLhJE\\\\\\\/wFV8x6OWrp03Eso9rCS+P101LpqDbA4hGR9ltRvr8gFs2GxC\\\\\\\/tnYxwDr9pJm05yuTuJdKA04+Qh6g1sOYLxzNabSjd14J5lDRe87\\\\\\\/ozJER5k\\\\\\\/Pq\\\\\\\/Rl6DdPB3YkdSBFvL07GyxlWGwOcjqsRXAGFPrVjvJoqcNtfYdldLyfUEws\\\\\\\/iRW7psDeWJJBsOQUbR4xmDk7yAk2\\\\\\\/lPD5fkVw4h1m+1p7t8eWoZPzZ+D8+YjoL6LVlVlVmoj4He8Qi\\\\\\\/aSlQ5eIsvMaqIMW1wWCO7yBbk8PqJOTFwANf0FHX7hrDWwyC6DEarWPVMl68\\\\\\\/alpgrJzlNyQh9U766DqVccYkcEW2+bq\\\\\\\/L\\\\\\\/53SHU53nRua10WmmWbZ1mdG9JDXqOUylkLjAezFEeP3sxsu69PCcrmDeQ1j4yaulsthSPZPXSNU2HEiLrWFMFTQUpHfcOrU3guD9pf6sqKSMxOjiWMSKPdjWo3xhweM4CvJjmOWC46xGsBhwIooEf2oVp6Yv10b0JIdYar8eVpuEdSgjRnkSaZLyrobizlXq1AQIlT7xGtJpWWzTxvWx5ontvHQLlj706CPo1rrupQWYPukzn6J6Yu7EOcNhCaOjSO01xIIvtU5eQJ\\\\\\\/bMcIQmRc5WpPyxSKDWhwjMb0t7AI2tWjoRyhP2MfsDlSRvztREZI80L9tfx\\\\\\\/4VUUDLpMCCBUJrJfzBaZKdJqc7in3FHQ5GEcHO\\\\\\\/VrATrE2YtmUOTO+e+e4bvqsx9ICIwqH+L4uXcqCBSDAYLkZsNXbWg6bqiTFqIM0ZoczvJjlmwa1HxP1NLY9CCN44Ye6TDTEtyjkmIGYDiAEhh1McdashFlDpU\\\\\\\/cuM0yo3iv2igX7vrAjMY4KocueOm0cSb3KvB6jrfFqtGZo9Tgy2BJfyOiikboZpq2QdERET7SvRanzsQ7PJYgvOfxkvlcH2fbWww6DbnD1wg8YsOjDPam5r+ZDcSjyINTk8wJwgqNa0pq47MB2cn73gkAo2DBbav2KcwkdTDjFMW6jjuq010v6QcB\\\\\\\/GPtKynTe4jD+O1Ff+STrmXfVYKAID0vNhv5u6+Ctd1l3xQPqjNTNHzzZhHAzHdQt60v1eNFUK5r3gdPwlmWv+yKJctQWoe6u1UgwcCZwn\\\\\\\/oBUiiBkooEFIkQ3qp3H9voycUIKqiLY1xRoAc\\\\\\\/GSz5rXzpXpgOMx2RtuBWRwz4QFBzj51S3G2Jcjlbcwwsl0b215ILRon6t8cmC17kvkkRbaP1DDz2F2kBCRN2FnOUokkgk6tkGiHw7lgiUymQI+AW4QcaMtXrGxnGrTiUDHGbiSi+yIHnvsGfIvHDOFpoltwHFi2HXWXvgJeliWxmmnO1uLast\\\\\\\/7qkidQ8Y\\\\\\\/V1dOWQIBMpwEbyAC+7HbJEPo\\\\\\\/+cO17Q4jVCsob8d8kILmpD8\\\\\\\/0hHA9XP7Bzo\\\u0022,\\\u0022iv\\\u0022:\\\u00221TtkpgldiaMdU2zlmGLPhQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022228ca932f15b92acc446329b457a9199ab2552d9ee1d59311a086084a9193588\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00227M\\\\\\\/XIUG0TxVR3NXbPb6y87utI0\\\\\\\/zYbY4I3h\\\\\\\/IqWYrzKI4nlpXpKNbXNJIot+Zt0XLcAjA5Mvb2a7HPA3z0SLOjrqimpyf+17xd0yFtdZvGkkHTylYMZib1TOiW1eoUjH++Sq0Bb6CiHOdis+XG7zBHgC+fdJowE6aozU90FzEnPdODGXMduSUFq9de6YD1wYb4ueUBR2zI46EIbh6wAdMlhRxMl9zoyhxmQz\\\\\\\/QkTUQa9Gh5\\\\\\\/KWKnjhLgyp7fs7iFQ6u6oNjj\\\\\\\/NireOPwa8nNpc+QMlzeYbxN170QUS1ffo+QV82ptmbdUaXBMZbF8vKLHy2QhA2iBC4bh5TyFT9pFuDNdL49pTq0BlCEcZzL6yggUsSLf+ph5g6ve8\\\\\\\/nyLcmfz\\\\\\\/6zIcbBZcMtsPAQgEpn6VCP\\\\\\\/3OFAmJE\\\\\\\/94FXXkp6CDAHH\\\\\\\/54Aicv4N75F\\\\\\\/2TOZquqEsUQhpedrVDH0bZVMIwJoEqmedRZBDpPx6UDZl32r9nNm3WJKlbIkpRWeVgZHP+FgBgN5lIfK1HfNv97e\\\\\\\/4+NOijcmqQHiWr0hOlmPAb\\\\\\\/4yuFnsbP6aftK\\\\\\\/hPTCUFWhM7b3h5DO+5Ll2ne7h8KVPc7YCm2u+pSNe2Ks1rEprw3uz2aKzMWv\\\\\\\/f6sG9mNc3FJRwxcNj+3m6qGTrIiXs4x+uE2El+HI8bW9Ie9iQ6SY62PwqBWXZ7FcMtwC6yKio76r1DxdI1cro065uYXII9pAL8YhSvuB8Ker9eyTVgE+qe5fvqowp9PlMvzDJCUV7tXIEfPv5D\\\\\\\/SnX8WApyneVjpqIzfz2WBOTS8bPE5O0Fgu+CoQjKxToeEpIPtq3jp7QZGVbiqabP60j0PEmRQTZPHrJG935yQNyS1qq3fU1+qpVRPjwoCpw5BGwI2kOFvjwUMBZSuBgSPLGMsixC2MAjLHNzwO9appHO5zgoUHf0ykd50yrQyO+Y901VsCH9j+vYtRlf7XdkHAKSOYUwmnks9P4EIItP\\\\\\\/LDNCdaE647\\\\\\\/RlzrGY2VBFP5c7eOWF3ZsdYvahnwZDGbqVta0pg3uOGgCRapn2ZSkOEi46tO6kWxKNYrlzav9PYrig\\\\\\\/muVlaeWJeLt+d\\\\\\\/24pP0sWI50Y5OgGnZSiBpU4reWlr4jHBftavxEksp\\\\\\\/qEVgbsZLol\\\\\\\/y+RkaJI511PbWzU0DL1b2xcLzbmkWz4O8uu2qHtWySvGzE6hOV00zbkFROy9AxYQD19fWRbwkub4CyhLJSqO+psRRs9nM06AEDj3wwkNwBXSuspV479yYGLhaMCh2v0UwhrbmhCJJbfYvTUCnc6NNlRXyRW2K4lhcivmPvl\\\\\\\/cNIqcBtpS7CE5ZrIh6kUD0+Xi7acumGk69UCGVAvtCMLvJfQ6XlwRzjRul4H3blq3Y4eYwZGz19e1EB2xatxWKb0WrbzIcrLi3vnMchBc1\\\\\\\/WjK8UmaIzVbJXB4uUAU1jqLR\\\\\\\/PlTVvEWB8tIAb5T5QUYq0LqdYeoT+1+bWNUDZ3GjFNL6MGmi6rsz8XtFWl8YVqfH8vSRmK+iqUdss0ZyU\\\\\\\/VWmCMdTNifWnAESyIMTB5Xg3sbIZk\\\\\\\/NMEzVUkmjGWXcZwq67ocsQEY6\\\\\\\/\\\\\\\/j2k4lTwgXpeOGztU\\\\\\\/BSKo27f5pYP8nQDe5KS\\\\\\\/QqJFCGVT\\\\\\\/D4bxxvZLR0i+caxNit\\\\\\\/aLmHH8s98A5m\\\\\\\/sjMfHthMIg96I1v7nhBXOJqRMGYucs+qnQgJXsegvK7WZ60T53Ho2GMqgI7r4ueA3CJnhJqHgOApK42kTasgEcZNmjO4PAvVKLbqmba+7Js9cPuYrJzmtVie6aVOOHQ3jXgGBd55An3YzgwFTmJGMivkQn1yVfZTx33Rsd4H3XvGdQIcEqmklafrda0IedJNWEuy7aQhbiR\\\\\\\/Six5muBTRZcC5EEgegGi3G8vm8d6RaDfxk7GkH7+LWS9bzSqQIpRY6uQfOtxc9sV6ToS0tc86uvAOXQA+x0lmSPL10EFzhSQICPkHgu17SYJQjsYqaC0zsoMQYJGsBo10Jz8VHlip9V8XrjHSvqAAg492I5mlL+T0+907JutoetnmiIHgRD\\\u0022,\\\u0022iv\\\u0022:\\\u00228r6ru0gyiRO7uK+nDbitRQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220fb94a971aa5afbf04da2b0ece9ace364e23d9c91bd65a1063979c095ccc009a\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460561 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/2/481" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-iUI4wUfJgooKCusHfaUXn7o-TefZD5Pe5k98z8Dlk6A" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/2/481" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ziajxERfq2vZNB-uy6Ll6wQi-ibu0clnwrzernufsFA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/2/467" title="Transcriptome Analysis of Drosophila melanogaster Third Instar Larval Ring Glands Points to Novel Functions and Uncovers a Cytochrome p450 Required for Development" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/2/497" title="Genetic and Targeted eQTL Mapping Reveals Strong Candidate Genes Modulating the Stress Response During Chicken Domestication" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460561" id="node460561" data-pisa="ggg;7/2/481" data-pisa-master="ggg;g3.116.036251" data-apath="/ggg/7/2/481.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Prediction with Pedigree and Genotype × Environment Interaction in Spring Wheat Grown in South and West Asia, North Africa, and Mexico</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-4088-4624" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sivakumar Sukumaran</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jose Crossa</span>, <span class="highwire-citation-author" data-delta="2">Diego Jarquin</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-7698-6019" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Marta Lopes</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-4291-4316" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Matthew P. Reynolds</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">481-495;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.036251 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460561" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sivakumar Sukumaran </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), El Batan, Texcoco CP 56237, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sivakumar%2BSukumaran%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sukumaran%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASivakumar%2BSukumaran%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4088-4624" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sivakumar Sukumaran</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jose Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Biometrics and Statistics Unit, CIMMYT, El Batan, Texcoco CP 56237, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jose%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJose%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jose Crossa</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Diego Jarquin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Agronomy and Horticulture, University of Nebraska, Lincoln, Nebraska 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Diego%2BJarquin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jarquin%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADiego%2BJarquin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Marta Lopes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Global Wheat Program, CIMMYT, Emex, 06511 Ankara, Turkey</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marta%2BLopes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lopes%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMarta%2BLopes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7698-6019" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Marta Lopes</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Matthew P. Reynolds </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), El Batan, Texcoco CP 56237, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BP.%2BReynolds%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reynolds%20MP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMatthew%2BP.%2BReynolds%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4291-4316" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Matthew P. Reynolds</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/2/481" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460561" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460561/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/2/481.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460561" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460561/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/2/481.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460561" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460561/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/2/481.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460561" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460561/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Developing genomic selection (GS) models is an important step in applying GS to accelerate the rate of genetic gain in grain yield in plant breeding. In this study, seven genomic prediction models under two cross-validation (CV) scenarios were tested on 287 advanced elite spring wheat lines phenotyped for grain yield (GY), thousand-grain weight (GW), grain number (GN), and thermal time for flowering (TTF) in 18 international environments (year-location combinations) in major wheat-producing countries in 2010 and 2011. Prediction models with genomic and pedigree information included main effects and interaction with environments. Two random CV schemes were applied to predict a subset of lines that were not observed in any of the 18 environments (CV1), and a subset of lines that were not observed in a set of the environments, but were observed in other environments (CV2). Genomic prediction models, including genotype × environment (G×E) interaction, had the highest average prediction ability under the CV1 scenario for GY (0.31), GN (0.32), GW (0.45), and TTF (0.27). For CV2, the average prediction ability of the model including the interaction terms was generally high for GY (0.38), GN (0.43), GW (0.63), and TTF (0.53). Wheat lines in site-year combinations in Mexico and India had relatively high prediction ability for GY and GW. Results indicated that prediction ability of lines not observed in certain environments could be relatively high for genomic selection when predicting G×E interaction in multi-environment trials.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genomic-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-prediction" rel="nofollow">genomic prediction</a></li><li class="kwd"><a href="/keyword/pedigree-based-prediction" class="hw-term hw-article-keyword hw-article-keyword-pedigree-based-prediction" rel="nofollow">pedigree-based prediction</a></li><li class="kwd"><a href="/keyword/wami" class="hw-term hw-article-keyword hw-article-keyword-wami" rel="nofollow">WAMI</a></li><li class="kwd"><a href="/keyword/spring-wheat" class="hw-term hw-article-keyword hw-article-keyword-spring-wheat" rel="nofollow">spring wheat</a></li><li class="kwd"><a href="/keyword/gblup" class="hw-term hw-article-keyword hw-article-keyword-gblup" rel="nofollow">GBLUP</a></li><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">genomic selection</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared Data Resources</a></li></ul><p id="p-3">Wheat is the most widely cultivated cereal crop in the world, and provides 20% of the protein and calories consumed by the world population (FAOSTAT). Several studies have reported that the present rate of genetic gain in spring wheat is &lt;1% yr<sup>−1</sup> (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Aisawi <em>et al.</em> 2015</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Sayre <em>et al.</em> 1997</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Manes <em>et al.</em> 2012</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Lopes <em>et al.</em> 2012</a>); that rate needs to improve to meet future wheat demand (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Reynolds <em>et al.</em> 2012</a>). This can be done through improvements in plant structure and reproduction, and in crop physiology (radiation use efficiency), as well as improved genotyping or phenotyping methods, increased genetic diversity of breeding germplasm, or through the use of complementary genomic selection approaches in plant breeding (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Reynolds <em>et al.</em> 2009</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Tester and Langridge 2010</a>).</p><p id="p-4">Traditional breeders use the pedigree selection method for breeding most crops, which requires several generations of testing and advancing the lines. An alternative method is marker-assisted selection (MAS), where markers associated with genes of major effect are used (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Spindel <em>et al.</em> 2015</a>). The first to propose predicting breeding values of complex traits for unobserved phenotypes using all available high density markers were <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Meuwissen <em>et al.</em> (2001)</a>. This initial study was followed, in plants, by <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bernardo and Yu (2007)</a>, who demonstrated, by simulation, that whole genome regression predicts complex traits more accurately than using only a few markers. These seminal investigations led to the application of different statistical parametric and nonparametric genomic models with pedigree information in different crops (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Crossa <em>et al.</em> 2010</a>, <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">2014</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Jarquin <em>et al.</em> 2014</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Pérez-Rodríguez <em>et al.</em> 2015</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Velu <em>et al.</em> 2016</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">de los Campos <em>et al.</em> 2013</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">de los Campos and Pérez-Rodríguez 2013</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Arruda <em>et al.</em> 2015</a>).</p><p id="p-5">All the initial genomic prediction models were developed for single-environment prediction. However, GS can accelerate genetic gains in wheat breeding, especially when multi-environment testing of lines is routine in their development and release (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Braun <em>et al.</em> 2010</a>). Multi-environment testing is prone to high levels of genotype × environment (G×E) interaction due to varying climatic zones, dynamic weather parameters, and different management factors. <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Burgueño <em>et al.</em> (2012)</a> were the first to use marker- and pedigree-based Best Linear Unbiased Predictor (BLUP) models for assessing G×E under genomic prediction; these models account for correlated environmental structures, and thus predict performance of unobserved phenotypes in several environments. <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Heslot <em>et al.</em> (2014)</a> incorporated crop modeling data for studying genomic G×E, and <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Jarquín <em>et al.</em> (2014)</a> proposed a random effect genomic BLUP (GBLUP) model, where the main effect and the G×E interaction effects of markers and environmental covariates are introduced via covariance structures of markers and environmental covariables in a reaction norm model.</p><p id="p-6">The reaction norm model of <a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Jarquín <em>et al.</em> (2014)</a> has been widely used in multi-environment data of different crops, including wheat data with pedigree and genomic information and their interaction with environments (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Pérez-Rodríguez and de los Campos 2014</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Crossa <em>et al.</em> 2016</a>; <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Velu <em>et al.</em> 2016</a>). This model is flexible and allows the incorporation of highly dimensional environmental covariable data. Furthermore, a similar genomic G×E model was recently developed and used on wheat breeding data, with the novelty that it decomposes the total marker × environment effect into a marker main effect across all the environments, and a marker-specific effect for each environment (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">López-Cruz <em>et al.</em> 2015</a>). A recent study by <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Crossa <em>et al.</em> (2015)</a> showed how the marker × environment model can be used both as a prediction model, by means of shrinkage regression, and/or as a variable selection to estimate marker effects.</p><p id="p-7">Previous genomic and pedigree studies on assessing the prediction ability of G×E have considered a very limited number of environments; they usually included combinations of sites under a few agronomic management systems (<em>i.e.</em>, levels of managed drought and heat stress). In the present study, we assess the genomic prediction ability of several models within the framework of the reaction norm model of <a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">Jarquín <em>et al.</em> (2014)</a> using the Wheat Association Mapping Initiative (WAMI) panel-designed to evaluate G×E for grain yield (GY), while avoiding the confounding effects of extreme phenology (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Lopes <em>et al.</em> 2015</a>). The main objective of this study was to detect prediction ability of different international sites established in different years, with the objective of examining and identifying possible key testing sites to be further used in a genomic-assisted breeding program. A total of 287 spring wheat lines included in the WAMI data were grown in international multi-environment trials in 18 site-year combinations in South and West Asia, North Africa, and Mexico. Traits included in this study are grain yield (GY), grain number (GN), thousand-grain weight (GW) and thermal time for flowering (TTF).</p><p id="p-8">The WAMI panel used in this study is very appropriate for studying genomic and pedigree prediction because its lines were phenotyped under a very diverse set of environments around the world. Also, the WAMI panel has already been studied for several complex traits: adaptation to plant density (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Sukumaran <em>et al.</em> 2015b</a>), grain yield and yield components (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Sukumaran <em>et al.</em> 2015a</a>), drought stress (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Edae <em>et al.</em> 2014</a>), and earliness <em>per se</em> (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Sukumaran <em>et al.</em> 2016</a>).</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>The genetic material</h3><p id="p-9">The WAMI population was assembled from the elite advanced wheat nurseries distributed through the International Wheat Improvement Network (IWIN). It consists of 287 diverse elite lines selected from nurseries bred for high yield potential environments (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Lopes <em>et al.</em> 2015</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Sukumaran <em>et al.</em> 2015a</a>,<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">b</a>).</p></div><div id="sec-3" class="subsection"><h3>Phenotyping and genotyping</h3><p id="p-10">The WAMI population was phenotyped in major wheat-growing areas of India, Pakistan, Nepal, Bangladesh, Iran, Egypt, Sudan, and Mexico. These growing environments are diverse in terms of rainfall, heat stress, drought stress, and solar radiation patterns. Phenotyping was conducted at the following locations: Bangladesh Agricultural Research Institute (BARI), Joydebpur, Bangladesh (BGLD J); Indian Agricultural Research Institute (IARI), Delhi (India D); University of Agricultural Sciences, Dharwad, Karnataka (India H); Indian Institute of Wheat and Barley Research, Karnal, India (India K); Punjab Agricultural University, Ludhiana, India (India L); Darab Hassan Abad, Fars, Iran (Iran D); Banaras Hindu University, Varanasi (India V); National Wheat Research Program, Bhairahawa, Rupandehi (Nepal B); and National Agricultural Research Centre, Islamabad (PAK I). In addition, phenotyping was done under four different treatments at the Norman Borlaug Experiment Station, Cd. Obregon, Sonora, Mexico: irrigated yield potential (Mex I), heat stress (Mex H), drought stress (Mex D), and heat and drought stress (Mex HD) (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> shows the countries, locations, and abbreviations used in this study, as well as the four traits that were recorded and analyzed: GY per square meter, GN per square meter, GW estimated using standard protocols (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Sayre <em>et al.</em> 1997</a>), and TTF estimated based on a base temperature of zero and the sowing date. Minimum and maximum temperatures, and the coordinates of the environments, were described in an earlier publication (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Sukumaran <em>et al.</em> 2016</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/481/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Geographical distribution of the sites where the wheat association mapping initiative (WAMI) panel was grown in 2010&#x2013;2011 in some of the major wheat growing areas. The map was created using ArcGIS Desktop Arcmap software." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-968864526" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Geographical distribution of the sites where the wheat association mapping initiative (WAMI) panel was grown in 2010&#x2013;2011 in some of the major wheat growing areas. The map was created using ArcGIS Desktop Arcmap software.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/481/F1.medium.gif" width="440" height="353"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/2/481/F1.medium.gif" width="440" height="353"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/481/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/481/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470494" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-11" class="first-child">Geographical distribution of the sites where the wheat association mapping initiative (WAMI) panel was grown in 2010–2011 in some of the major wheat growing areas. The map was created using ArcGIS Desktop Arcmap software.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470342/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470342/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Descriptive statistics of the wheat association mapping initiative (WAMI) panel grown in several international environments</span><div class="sb-div caption-clear"></div></div></div><p id="p-13">The WAMI panel was genotyped using 90K Illumina SNPs array (<a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Sukumaran <em>et al.</em> 2015a</a>). From the polymorphic SNPs after using a minor allele frequency cut-off of 5%, 15K SNPs were used for genomic prediction. The population structure associated with the 1B.1R translocation was described in earlier publications (<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Lopes <em>et al.</em> 2015</a>; <a id="xref-ref-32-4" class="xref-bibr" href="#ref-32">Sukumaran <em>et al.</em> 2015a</a>).</p></div><div id="sec-4" class="subsection"><h3>Statistical models</h3><p id="p-14">The Best Linear Unbiased Estimators (BLUEs) were computed for mixed model analysis for each of the traits in each environment. The model used to calculate BLUEs for each environment is<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-1.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-2.gif"/></noscript></span></span> is the phenotypic response value for the specific trait measured on the <em>j</em>th line of the <em>m</em>th incomplete block within the <em>k</em>th replicate, <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-3.gif"/></noscript></span></span> is a fixed effect of the <em>j</em>th wheat line, <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-4.gif"/></noscript></span></span> is the random effect of the <em>k</em>th replicate assumed independent and identically multivariate normally distributed (iid) <em>N</em>(0, <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-5.gif"/></noscript></span></span>) (where <strong><em>I</em></strong> is the identity matrix, and <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-6.gif"/></noscript></span></span> is the variance of replicate), <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-7.gif"/></noscript></span></span> denotes the random effect of the <em>m</em>th incomplete block within the <em>k</em>th replicate assumed independent and identically distributed (iid) with <em>N</em>(0, <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-8.gif"/></noscript></span></span>), where <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-9.gif"/></noscript></span></span> is the variance of the incomplete block, <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-10.gif"/></noscript></span></span> is the random error associated to the trait measured on the <em>j</em>th line of the <em>m</em>th incomplete block within the <em>k</em>th replicate, and assumed iid with <em>N</em>(0, <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-11.gif"/></noscript></span></span>), where <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-12.gif"/></noscript></span></span> denotes the error variance.</p><p id="p-15">Broad-sense heritability (<em>H</em><sup>2</sup>) for each environment was computed on an entry mean basis as <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-13.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-14.gif"/></noscript></span></span> is the wheat line variance, <em>r</em> is the number of replicates, and <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-15.gif"/></noscript></span></span> is the error variance. Heritability estimates across environments were also estimated using the following formula,<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-16.gif"/></noscript></span></span>where <em>s</em> is the number of environments, and <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-17.gif"/></noscript></span></span> is the variance of the wheat line × environment obtained from the combined analyses across environments.</p><p id="p-16">For GS, we used the reaction norm model that is an extension of the random effect Genomic Best Linear Unbiased Predictor (GBLUP) model, where the main effect of lines, the main effect of environments, the main effect of markers, the main effect of pedigree, and their interactions with environments, are modeled using random covariance structures that are functions of marker or pedigree genotypes and environmental covariates (<a id="xref-ref-17-5" class="xref-bibr" href="#ref-17">Jarquín <em>et al.</em> 2014</a>). Brief descriptions of the baseline model, as well as the reaction norm models with G×E, are given below.</p></div><div id="sec-5" class="subsection"><h3>Baseline model</h3><p id="p-17">The response of the phenotypes (<span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-18.gif"/></noscript></span></span>) defined by random baseline model is<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-19.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-20.gif"/></noscript></span></span> is the overall mean, <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-21.gif"/></noscript></span></span> is the random effect of the <em>i</em>th environment, <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-22.gif"/></noscript></span></span> is the random effect of the <em>j</em>th line, <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-23.gif"/></noscript></span></span> is the interaction between the <em>i</em>th environment and the <em>j</em>th line, and <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-24.gif"/></noscript></span></span> is the random error term. All random effects follow a iid multivariate normal distribution such that <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-25.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-26.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-27.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-28.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-29.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-30.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-31.gif"/></noscript></span></span> are the environment, line, and line × environment variances, respectively.</p><p id="p-18">In the model above, the random effect of the line (<span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-32.gif"/></noscript></span></span>) can be replaced by <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-33.gif"/></noscript></span></span> which is an approximation of the genetic value of the <em>j</em>th line from the genomic relationship matrix. Also, the effects of the line (<span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-34.gif"/></noscript></span></span>) can be replaced by <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-35.gif"/></noscript></span></span> which is the additive effect obtained from the pedigree information. In the models described below, we used either <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-36.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-37.gif"/></noscript></span></span> both <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-38.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-39.gif"/></noscript></span></span>, as well as their interactions with environment <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-40.gif"/></noscript></span></span> Full descriptions of the different reaction norm models can be found in <a id="xref-ref-17-6" class="xref-bibr" href="#ref-17">Jarquín <em>et al.</em> (2014)</a> and <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Zhang <em>et al.</em> (2014)</a>, among others. Below, we give a brief description of the different reaction norm models that were fitted using pedigree and genomic information.</p></div><div id="sec-6" class="subsection"><h3>Reaction norm models</h3><p id="p-19">We fitted seven different models (M1–M7) with different components including <em>E</em> = environments, <em>L</em> = line, <em>A</em> = pedigree, <em>G</em> = genomic, <em>AE</em> = pedigree × environment interaction, <em>GE</em> = genomic × environment interaction, and <em>e</em> = residual error.</p></div><div id="sec-7" class="subsection"><h3>M1: Environment and line main effects (Y = E + L + e)</h3><p id="p-20">The response of the phenotypes (<span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-41.gif"/></noscript></span></span>) from the baseline model, but excluding the interaction term, <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-42.gif"/></noscript></span></span> is described as</p><span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-43.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span></div><div id="sec-8" class="subsection"><h3>M2: Environment, line, and pedigree main effects (Y = E + L + A + e)</h3><p id="p-21">By adding the random effect that incorporates pedigree information by means of the numerical relationship matrix (<strong><em>A</em></strong>) to M1, we get model M2, defined as<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-44.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>where <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-45.gif"/></noscript></span></span> is a random additive effect of the line, which, in this case accounts for pedigree-relationships, where <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-46.gif"/></noscript></span></span> contains the pedigree values of all the lines, and is assumed to follow a multivariate normal density with zero mean and covariance matrix <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-47.gif"/></noscript></span></span> where <strong><em>A</em></strong> is the numerical relationship matrix, and <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-48.gif"/></noscript></span></span> is the additive genetic variance. The random effects <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-49.gif"/></noscript></span></span> are correlated such that model M2 allows borrowing of information across lines based on the numerical relationship matrix (<strong><em>A</em></strong>) computed from the pedigree information.</p></div><div id="sec-9" class="subsection"><h3>M3: Environment, line, and genomic main effects (Y = E + L + G + e)</h3><p id="p-22">Model M3 is fitted by adding the genomic random effect of the line <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-50.gif"/></noscript></span></span> to M1, which is an approximation of the genetic value of the <em>j</em>th line, and is defined by the regression on marker covariates <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-51.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-52.gif"/></noscript></span></span> is the genotype of the <em>j</em>th line at the <em>l</em>th marker, and <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-53.gif"/></noscript></span></span> is the effect of the <em>l</em>th marker assuming iid <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-54.gif"/></noscript></span></span> (<em>l</em>=1,…,<em>p</em>), and <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-55.gif"/></noscript></span></span> is the variance of the marker effects. The vector <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-56.gif"/></noscript></span></span> contains the genomic values of all the lines, and is assumed to follow a multivariate normal density with zero mean and covariance matrix <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-57.gif"/></noscript></span></span> where <strong><em>G</em></strong> is the genomic relationship matrix computed as suggested by <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">VanRaden (2008)</a> (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-58.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-59.gif"/></noscript></span></span> as the centered and standardized matrix of molecular markers and <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-60.gif"/></noscript></span></span> the frequency of the <em>l</em>th marker); and <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-61.gif"/></noscript></span></span> is the genomic variance. Thus, model M3 is<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-62.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>with <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-63.gif"/></noscript></span></span> The random effects <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-64.gif"/></noscript></span></span> are correlated such that model M3 allows borrowing information across lines.</p></div><div id="sec-10" class="subsection"><h3>M4: Environment, line, pedigree, and pedigree × environment interaction effects (Y = E + L + A + AE + e)</h3><p id="p-23">By adding the interaction between the additive relationship matrix and environments (<span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-65.gif"/></noscript></span></span>) to model M2, model M4 becomes<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-66.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>where the term <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-67.gif"/></noscript></span></span> is the interaction between the additive value of the <em>i</em>th genotype in the <em>j</em>th environment and <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-68.gif"/></noscript></span></span> Matrices <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-69.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-70.gif"/></noscript></span></span> are the incidence matrices for the effects of the additive genetic values of genotypes and environments, respectively, <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-71.gif"/></noscript></span></span> is the variance component of the interaction term <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-72.gif"/></noscript></span></span>, and “<span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-73.gif"/></noscript></span></span>” stands for Hadamart product between two matrices.</p></div><div id="sec-11" class="subsection"><h3>M5: Environments, lines, genomic, genomic × environment interaction effect (genomic × environment) (Y = E + L + G + GE + e)</h3><p id="p-24">By adding the interaction between markers and environments (<span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-74.gif"/></noscript></span></span>) to model M3, model M5 becomes<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-75.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>where the term <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-76.gif"/></noscript></span></span> is the interaction between the genetic value of the <em>i</em>th genotype in the <em>j</em>th environment; then <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-77.gif"/></noscript></span></span> Matrices <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-78.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-79.gif"/></noscript></span></span> are the incidence matrices for the effects of the genetic values of the genotypes and the environments, respectively, <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-80.gif"/></noscript></span></span> is the variance component of the interaction term <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-81.gif"/></noscript></span></span></p></div><div id="sec-12" class="subsection"><h3>M6: Environment, line, pedigree, and genomic main effects (Y = E + L + A + G + e)</h3><p id="p-25">We added both the pedigree and genomic effects of the lines (<span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-82.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-83.gif"/></noscript></span></span>) to model M1, so that it contains the genomic random vector <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-84.gif"/></noscript></span></span> and the pedigree random vector <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-85.gif"/></noscript></span></span> Therefore, model M6 is</p><span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-86.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span></div><div id="sec-13" class="subsection"><h3>M7: Environment, line, pedigree, genomic, pedigree × environment interaction and genomic × environment interaction effects (M7 = E + L + A + G + AE + GE + e)</h3><p id="p-26">By adding both the interaction between pedigree and environment (<span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-87.gif"/></noscript></span></span>), and the interaction between markers (genomic) and environments (<span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-88.gif"/></noscript></span></span>) to model M6, model M7 becomes<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/481/embed/mml-math-89.gif"/></noscript></span><span class="disp-formula-label">(7)</span></span>All the terms in this model have already been defined above.</p></div><div id="sec-14" class="subsection"><h3>Prediction assessment by cross-validation</h3><p id="p-27">Two distinct cross-validation (CV) schemes were used. The first, CV1, evaluates the prediction ability of models when a set of lines has not been evaluated in any of the environments (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Burgueño <em>et al.</em> 2012</a>). Predictions derived using CV1 are based entirely on phenotypic records of other lines. The second scheme, CV2, evaluates the prediction ability of models when some lines have been evaluated in some environments, but not in others. In CV2 prediction, information from related lines and the correlated environments is used, and prediction assessment benefits from borrowing information between lines within an environment, between lines across environments, and among correlated environments (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Burgueño <em>et al.</em> 2012</a>). Prediction ability is the Pearson correlation coefficient between the observed and predicted values for each genotype.</p><p id="p-28">In both CV1 and CV2, a fivefold cross-validation scheme was used to generate the training (TRN) and testing (TST) sets, and to assess the prediction ability of each testing set. The data were divided randomly into five subsets, with 80% of the lines assigned to the training set and 20% assigned to the testing set. Four subsets were combined to form the training set, and the remaining subset was used as the validation set. Permutation of five subsets led to five possible training and validation data sets. This procedure was repeated 20 times, and a total of 100 runs was performed on each population for each trait-environment combination. The same partitions were analyzed with all models. The average value of the correlations between the phenotype and the genomic estimated breeding values from 100 runs was calculated in each population for each trait-environment combination, and was defined as the prediction ability.</p></div><div id="sec-15" class="subsection"><h3>Software</h3><p id="p-29">The genomic prediction analyses were computed using R, and the models were fitted using the BGLR package (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">de los Campos <em>et al.</em> 2013</a>; <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">de los Campos and Pérez-Rodríguez 2013</a>). The ANOVA was performed in the SAS 9.2 (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">SAS Institute Inc 2010</a>) program, and the boxplots created in R. Tukey’s test for significant differences between the models’ predictions (correlations) were generated in the SAS 9.2 (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">SAS Institute Inc 2010</a>) program.</p></div><div id="sec-16" class="subsection"><h3>Data availability</h3><p id="p-30">All the phenotypic data for each environment and trait, as well as the genomic data, can be downloaded from the link <a href="http://hdl.handle.net/11529/10714">http://hdl.handle.net/11529/10714</a>. </p></div></div><div class="section results" id="sec-17"><h2 class="">Results</h2><div id="sec-18" class="subsection"><h3>Variation in the studied traits</h3><p id="p-31">The WAMI panel was grown in seven countries, comprising a total of 18 environments (site-year combinations). India had the largest number of environments (6) followed by Mexico (4), and Bangladesh, Nepal, Pakistan, Iran, and Sudan with one site each. The ANOVA showed significant differences between the wheat lines and environments (<a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table A1</a>, Appendix A). Environment Mex 110 was the highest yielding environment (7.02 ton/ha), followed by Pak I11 (6.9 ton/ha). The lowest yield was obtained in BGLD J10 (2.2 ton/ha) with a GW value of 30.9. The highest GW value was recorded in Mex I10 (43.4), followed by India L11 (38.3), and the lowest was India H10 (27.6). TTF ranged from 976°D in Mex H10 to 1474°D in Sudan W. The highest GN was recorded in Pak I11 (21920), followed by Iran D10 (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Heritability estimates of individual and combined environments for each trait were also calculated. Trait GW (0.74) had the highest <em>H</em><sup>2</sup> values, followed by GN (0.51), TTF (0.48), and GY (0.41) (<a id="xref-table-wrap-6-2" class="xref-table" href="#T6">Table A1</a>, Appendix A). A box plot of the data at the individual locations indicated TTF had the highest variation and the lowest G × E was observed for GW (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> and <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/481/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Boxplot of the data collected from 18 environments around the world for traits (A) GY and (B) GN. Environments (site-year combinations) are defined in Table 1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-968864526" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Boxplot of the data collected from 18 environments around the world for traits (A) GY and (B) GN. Environments (site-year combinations) are defined in Table 1.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/481/F2.medium.gif" width="440" height="346"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/2/481/F2.medium.gif" width="440" height="346"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/481/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/481/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470259" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-32" class="first-child">Boxplot of the data collected from 18 environments around the world for traits (A) GY and (B) GN. Environments (site-year combinations) are defined in <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/481/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Boxplot of the data collected from 18 environments around the world for traits (A) GW and (B) TTF. Environments (site-year combinations) are defined in Table 1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-968864526" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Boxplot of the data collected from 18 environments around the world for traits (A) GW and (B) TTF. Environments (site-year combinations) are defined in Table 1.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/481/F3.medium.gif" width="440" height="355"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/2/481/F3.medium.gif" width="440" height="355"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/481/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/481/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470334" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-33" class="first-child">Boxplot of the data collected from 18 environments around the world for traits (A) GW and (B) TTF. Environments (site-year combinations) are defined in <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-19" class="subsection"><h3>Prediction ability of different models for international sites</h3><p id="p-34">We used seven models (M1–M7) to predict the lines that were not observed under CV1 and CV2 scenarios. For GY, prediction ability values for individual sites ranged from −0.05 (M1) to 0.52 (M6) (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). The highest value obtained for CV1 was in Mex D10 (0.52). The models’ average prediction abilities were −0.08, 0.11, 0.21, 0.22, 0.22, 0.29, and 0.31 for M1–M7, respectively. Models 6 and 7 had the highest significant prediction ability, followed by models 3, 4, and 5 (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). For CV2, the values ranged from 0.10 (Sudan W10) to 0.54 (India D10, India V10). On average, the CV2 correlation values for each model were 0.31, 0.31, 0.32, 0.32, 0.35, 0.37, and 0.38 for M1–M7, respectively (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). Tukey’s test identified that M5–M7 had significantly higher prediction accuracies than the other models for CV2 (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470323/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470323/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Correlations (mean ± SD) between the observed and predicted values for GY under CV1 and CV2 CV schemes for seven prediction models (M1–M7)</span><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/481/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of boxplot distributions of prediction ability (correlations) for each model (M1&#x2013;M7) for trait GY using two prediction CV scenarios: (A) CV1 and (B) CV2 for GY. Different letters denote significant differences among groups (post hoc nonparametric Tukey&#x2019;s test, P &lt; 0.05). Models: M1 Y = E+L+e; M2 Y= E+L+A+e; M3 Y= E+L+G+e; M4 Y= E+L+A+AE+e; M5 Y= E+L+G+GE+e; M6 Y= E+L+G+A+e; M7 Y= E+L+G+A+GE+AE+e." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-968864526" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison of boxplot distributions of prediction ability (correlations) for each model (M1&#x2013;M7) for trait GY using two prediction CV scenarios: (A) CV1 and (B) CV2 for GY. Different letters denote significant differences among groups (&lt;em&gt;post hoc&lt;/em&gt; nonparametric Tukey&#x2019;s test, &lt;em&gt;P&lt;/em&gt; &lt; 0.05). Models: M1 Y = E+L+e; M2 Y= E+L+A+e; M3 Y= E+L+G+e; M4 Y= E+L+A+AE+e; M5 Y= E+L+G+GE+e; M6 Y= E+L+G+A+e; M7 Y= E+L+G+A+GE+AE+e.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/481/F4.medium.gif" width="271" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/2/481/F4.medium.gif" width="271" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/481/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/481/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470357" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-38" class="first-child">Comparison of boxplot distributions of prediction ability (correlations) for each model (M1–M7) for trait GY using two prediction CV scenarios: (A) CV1 and (B) CV2 for GY. Different letters denote significant differences among groups (<em>post hoc</em> nonparametric Tukey’s test, <em>P</em> &lt; 0.05). Models: M1 Y = E+L+e; M2 Y= E+L+A+e; M3 Y= E+L+G+e; M4 Y= E+L+A+AE+e; M5 Y= E+L+G+GE+e; M6 Y= E+L+G+A+e; M7 Y= E+L+G+A+GE+AE+e.</p><div class="sb-div caption-clear"></div></div></div><p id="p-39">For CV1, 28 environment-model combinations had prediction ability values above 0.30 for GY. Among the sites, when using M7, Mexican environments (Mex I10, Mex D10, and Mex H10) had high prediction ability values (&gt;0.41) for CV1 using M6 and M7. CV2 values were above 0.40 for 52 site-model combinations. For India V10 and India D10, prediction ability was above 0.5 for all models. In the CV1 scenario, 14 sites had the highest values when M6 and M7 were used (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>).</p><p id="p-40">Trait GN mostly followed a similar pattern as that shown for GY but the CV1 values ranged from −0.05 (BGLD J10) to 0.56 (Mex I10). Forty-three environment-model combinations had CV1 values &gt;0.30. Models 6 and 7 had five sites with CV1 values &gt;0.4. Two sites (Sudan W10 and Mex I10) had CV1 values &gt; 0.4 for six models. Models 6 and 7 had CV1 values &gt; 0.50 for four environments (Mex H10, Mex HD10, Sudan W10, and Mex I10). On average, the highest CV1 values were recorded for M5 (0.32) and M7 (0.32) (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). Tukey’s test also grouped M5 and M7 with the highest prediction ability models for CV1 and CV2 scenarios (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470500/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470500/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Correlations (mean ± SD) between the observed and predicted values for GN under CV1 and CV2 schemes for seven prediction models (M1–M7)</span><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/481/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of boxplot distributions of prediction ability (correlations) for each model (M1&#x2013;M7) for trait GN using two prediction CV scenarios: (A) CV1 and (B) CV2 for GY. Different letters denote significant differences among groups (post hoc nonparametric Tukey&#x2019;s test, P &lt; 0.05). Models: M1 Y = E+L+e; M2 Y= E+L+A+e; M3 Y= E+L+G+e; M4 Y= E+L+A+AE+e; M5 Y= E+L+G+GE+e; M6 Y= E+L+G+A+e; M7 Y= E+L+G+A+GE+AE+e." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-968864526" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison of boxplot distributions of prediction ability (correlations) for each model (M1&#x2013;M7) for trait GN using two prediction CV scenarios: (A) CV1 and (B) CV2 for GY. Different letters denote significant differences among groups (&lt;em&gt;post hoc&lt;/em&gt; nonparametric Tukey&#x2019;s test, &lt;em&gt;P&lt;/em&gt; &lt; 0.05). Models: M1 Y = E+L+e; M2 Y= E+L+A+e; M3 Y= E+L+G+e; M4 Y= E+L+A+AE+e; M5 Y= E+L+G+GE+e; M6 Y= E+L+G+A+e; M7 Y= E+L+G+A+GE+AE+e.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/481/F5.medium.gif" width="267" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/2/481/F5.medium.gif" width="267" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/481/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/481/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470328" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-44" class="first-child">Comparison of boxplot distributions of prediction ability (correlations) for each model (M1–M7) for trait GN using two prediction CV scenarios: (A) CV1 and (B) CV2 for GY. Different letters denote significant differences among groups (<em>post hoc</em> nonparametric Tukey’s test, <em>P</em> &lt; 0.05). Models: M1 Y = E+L+e; M2 Y= E+L+A+e; M3 Y= E+L+G+e; M4 Y= E+L+A+AE+e; M5 Y= E+L+G+GE+e; M6 Y= E+L+G+A+e; M7 Y= E+L+G+A+GE+AE+e.</p><div class="sb-div caption-clear"></div></div></div><p id="p-45">For CV2, 36 environment-model combinations had GN values &gt;0.5. The values ranged from 0.08 (Iran D10) to 0.69 (Mex HD10). Five environments (Mex D10, Mex H10, Mex HD10, Sudan W10, and Mex I10) had values &gt;0.5 for all models. On average, the increase in CV2 values from M1 to M7 was 0.04, with M5 and M7 recording the highest average increases (0.43) (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>).</p><p id="p-46">Trait GW had the highest prediction values in CV1 (0.72, Iran D10) and CV2 (0.88, Iran D10) scenarios. Models 6 and 7 had eight environments with prediction values in CV1 &gt;0.5. Thirty-six environment-model combinations had prediction values for CV1 &gt;0.50, with M6 and M7 predicting the values for CV1 of five sites with correlation &gt;0.6. On average, the model with the highest CV1 values was M3, followed by M7. Mex I10 and Mex D10 had the highest CV1 (0.72 for both M6 and M7) and CV2 values (0.88 for M6 and M7) (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). Tukey’s test group showed models M3 and M7 as the most significant models with the highest prediction ability in the CV1 scenario (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). In the CV2 scenario, all models had the same prediction ability (0.63) (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a> and <a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>).</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470333/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470333/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Correlations (mean ± SD) between the observed and predicted values for GW under CV1 and CV2 schemes for seven prediction models (M1–M7)</span><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/481/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of boxplot distributions of prediction ability (correlations) for each model (M1&#x2013;M7) for trait grain weight using two prediction CV scenarios: (A) CV1 and (B) CV2 for trait GW. Different letters denote significant differences among groups (post hoc nonparametric Tukey&#x2019;s test, P &lt; 0.05). Models: M1 Y = E+L+e; M2 Y= E+L+A+e; M3 Y= E+L+G+e; M4 Y= E+L+A+AE+e; M5 Y= E+L+G+GE+e; M6 Y= E+L+G+A+e; M7 Y= E+L+G+A+GE+AE+e." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-968864526" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison of boxplot distributions of prediction ability (correlations) for each model (M1&#x2013;M7) for trait grain weight using two prediction CV scenarios: (A) CV1 and (B) CV2 for trait GW. Different letters denote significant differences among groups (&lt;em&gt;post hoc&lt;/em&gt; nonparametric Tukey&#x2019;s test, &lt;em&gt;P&lt;/em&gt; &lt; 0.05). Models: M1 Y = E+L+e; M2 Y= E+L+A+e; M3 Y= E+L+G+e; M4 Y= E+L+A+AE+e; M5 Y= E+L+G+GE+e; M6 Y= E+L+G+A+e; M7 Y= E+L+G+A+GE+AE+e.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/481/F6.medium.gif" width="267" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/7/2/481/F6.medium.gif" width="267" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/481/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/481/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470414" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-50" class="first-child">Comparison of boxplot distributions of prediction ability (correlations) for each model (M1–M7) for trait grain weight using two prediction CV scenarios: (A) CV1 and (B) CV2 for trait GW. Different letters denote significant differences among groups (<em>post hoc</em> nonparametric Tukey’s test, <em>P</em> &lt; 0.05). Models: M1 Y = E+L+e; M2 Y= E+L+A+e; M3 Y= E+L+G+e; M4 Y= E+L+A+AE+e; M5 Y= E+L+G+GE+e; M6 Y= E+L+G+A+e; M7 Y= E+L+G+A+GE+AE+e.</p><div class="sb-div caption-clear"></div></div></div><p id="p-51">For TTF, the prediction ability in the CV1 scenario ranged from −0.11 (India L11, M1) to 0.44 (India K11, M5). A total of 29 model-site combinations had CV1 values &gt;0.3, with M7 predicting eight sites with correlations &gt;0.3. Mex I10 had five models predicting the sites with &gt;0.3 for CV1, followed by Nepal. On average, M5 had the highest CV1 values (0.28) when compared with other models (<a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>). For CV2, values ranged from 0.40 (Mex H10) to 0.80 (BGLDJ10). All sites had correlation values &gt;0.3, and 85 environment-model combinations had correlation values &gt;0.50. Thirty-five sites (31%) had CV2 values &gt;0.7 (<a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table 5</a>). On average, correlations for M5–M7 for CV1 were 0.28, 0.24, and 0.27, respectively, and prediction ability values for M5–M7 for CV2 were 0.54, 0.52, and 0.53, respectively. Tukey’s test groups indicated that M5–M7 were the best predictive models for CV1, while for CV2 all models had the same prediction ability (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>).</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470481/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470481/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Correlations (mean ± SD) between the observed and predicted values for TTF under CV1 and CV2 schemes for seven prediction models (M1–M7)</span><div class="sb-div caption-clear"></div></div></div><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/481/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of boxplot distributions of prediction ability (correlations) for each model (M1&#x2013;M7) for trait grain number using two prediction CV scenarios (A) CV1 and (B) CV2 for trait thermal time to flowering (TTF). Different letters denote significant differences among groups (post hoc nonparametric Tukey&#x2019;s test, P &lt; 0.05). Models: M1 Y = E+L+e; M2 Y= E+L+A+e; M3 Y= E+L+G+e; M4 Y= E+L+A+AE+e; M5 Y= E+L+G+GE+e; M6 Y= E+L+G+A+e; M7 Y= E+L+G+A+ GE +AE+e." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-968864526" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison of boxplot distributions of prediction ability (correlations) for each model (M1&#x2013;M7) for trait grain number using two prediction CV scenarios (A) CV1 and (B) CV2 for trait thermal time to flowering (TTF). Different letters denote significant differences among groups (&lt;em&gt;post hoc&lt;/em&gt; nonparametric Tukey&#x2019;s test, &lt;em&gt;P&lt;/em&gt; &lt; 0.05). Models: M1 Y = E+L+e; M2 Y= E+L+A+e; M3 Y= E+L+G+e; M4 Y= E+L+A+AE+e; M5 Y= E+L+G+GE+e; M6 Y= E+L+G+A+e; M7 Y= E+L+G+A+ GE +AE+e.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/481/F7.medium.gif" width="265" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/7/2/481/F7.medium.gif" width="265" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/481/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/481/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470298" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-55" class="first-child">Comparison of boxplot distributions of prediction ability (correlations) for each model (M1–M7) for trait grain number using two prediction CV scenarios (A) CV1 and (B) CV2 for trait thermal time to flowering (TTF). Different letters denote significant differences among groups (<em>post hoc</em> nonparametric Tukey’s test, <em>P</em> &lt; 0.05). Models: M1 Y = E+L+e; M2 Y= E+L+A+e; M3 Y= E+L+G+e; M4 Y= E+L+A+AE+e; M5 Y= E+L+G+GE+e; M6 Y= E+L+G+A+e; M7 Y= E+L+G+A+ GE +AE+e.</p><div class="sb-div caption-clear"></div></div></div><p id="p-56">In summary, for the complex trait GY, M6 and M7 with interactions had the highest average prediction ability across environments for CV1 (0.29 and 0.31, respectively), and for CV2 (0.37 and 0.38, respectively). For the less complex trait GW, M3 and M7 showed the highest mean prediction ability for CV1 (0.45), and it was around 0.63 for all models in CV2. For grain number (GN) (which is a GY component and a complex trait), M5 and M7 gave the highest prediction ability for CV1 (0.32) and CV2 (0.43). For trait TTF, M5–M7 (0.28, 0.24, and 0.27, respectively) were the best for CV1; all models performed similarly for CV2 (0.52–0.53).</p></div><div id="sec-20" class="subsection"><h3>Trends in prediction ability <em>vs.</em> heritability</h3><p id="p-57">The best model for GY was M7 and, for CV1, it showed increasing values of environment heritability with their corresponding prediction accuracies, whereas M1 prediction ability was not related to heritability values (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure B1-A</a>, Appendix B). For GY in the CV2 scenario, the best and worst models had similar prediction ability, and showed an increasing trend of up to <em>H</em><sup>2</sup> = 0.50; values decreased thereafter (<a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure B1-B</a>, Appendix B). For trait GW in the CV1 scenario, a positive trend of increased prediction ability with increased <em>H</em><sup>2</sup> values was observed for the best model (M3), which had no interaction terms. The worst model (M1) did not show a response with increased <em>H</em><sup>2</sup> values (<a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure B2-A</a>, Appendix B). For the CV2 scenario, the best and worst models showed increased correlation values, and an increase in <em>H</em><sup>2</sup> values (<a id="xref-fig-9-2" class="xref-fig" href="#F9">Figure B2-B</a>, Appendix B).</p><p id="p-58">For GN, the correlations and <em>H</em><sup>2</sup> values of the best model (M7) showed a positive trend in the CV1 scenario, whereas the basic model (M1) showed no association with <em>H</em><sup>2</sup> values (<a id="xref-fig-10-1" class="xref-fig" href="#F10">Figure B3-A</a>, Appendix B). Similar to GY, the best and basic models showed very close prediction ability and <em>H</em><sup>2</sup> values for environments in the CV2 scenario for GN (<a id="xref-fig-10-2" class="xref-fig" href="#F10">Figure B3-B</a>, Appendix B). For TTF, the best model (M5) did not show greater prediction ability, the positive trend was lower (<em>R</em><sup>2</sup> = 0.12), and the basic model M1 showed no association with <em>H</em><sup>2</sup> values (<em>R</em><sup>2</sup> = 0.08) (<a id="xref-fig-11-1" class="xref-fig" href="#F11">Figure B4-A</a>, Appendix B). In the CV2 scenario, the best and basic model for TTF did not show high association between prediction values and <em>H</em><sup>2</sup> estimates, with some sites with high heritability estimates showing lower prediction values (<a id="xref-fig-11-2" class="xref-fig" href="#F11">Fig. B4-B</a>, Appendix B).</p></div></div><div class="section discussion" id="sec-21"><h2 class="">Discussion</h2><p id="p-59">The WAMI panel has been extensively studied for several complex traits: adaptation to density (<a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Sukumaran <em>et al.</em> 2015b</a>), GY and yield components (<a id="xref-ref-32-5" class="xref-bibr" href="#ref-32">Sukumaran <em>et al.</em> 2015a</a>), drought stress (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Edae <em>et al.</em> 2013</a>, <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">2014</a>), and earliness <em>per se</em> (<a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Sukumaran <em>et al.</em> 2016</a>). Since it was also phenotyped under diverse environments around the world, it is a perfect panel for testing some of the genomic and pedigree selection models. Data from these testing sites were used routinely to select lines for release as varieties, and for crossing them to generate new prebreeding lines (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Reynolds and Langridge 2016</a>). Physiological breeding is aimed at improving wheat productivity through complex physiological traits. These traits are often controlled by genes with small effects; if they can be proven to be of value in the breeding program, they are more effectively selected using genomic selection methods than using MAS.</p><p id="p-60">Several models have been proposed for the genomic prediction schemes; however, it is important to test them on diverse environmental data before using them in the breeding program. Models 6 and 7 were the best models, for they had the highest average prediction ability values for the CV1 and CV2 scenarios for GY among all environments. Here, we evaluated seven models (some with the G×E term), and concluded that these models can predict GY with moderate to high levels of prediction ability, whereas less complex traits, such as GW, can be predicted without including any interaction terms in the model.</p><p id="p-61">The results of this study agree with those of a recent study on Zn and Fe grain concentration in spring wheat (<a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Velu <em>et al.</em> 2016</a>). Models that include G×E interaction terms showed higher prediction accuracies. Also, prediction ability was generally associated with trait heritability, as in earlier reports (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Muranty <em>et al.</em> 2015</a>). The highest prediction ability was for GW, which is a high heritability trait in the WAMI panel (<a id="xref-ref-32-6" class="xref-bibr" href="#ref-32">Sukumaran <em>et al.</em> 2015a</a>). Another observation was that, for some environments, M3 gave high prediction ability for GW in the CV1 scenario, whereas model M2 was the best in CV2. In this study, we evaluated the correlation between genomic- and pedigree-based estimated breeding values, with phenotypic data from field trials. With a reasonable number of molecular markers, and incorporating G×E terms in the models, higher prediction ability was obtained for the “genomic” component when compared to pedigree-based prediction models (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Burgueño <em>et al.</em> 2011</a>). This was also dependent on trait heritability, as GW had higher prediction ability values even when using M3 (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Muranty <em>et al.</em> 2015</a>).</p><p id="p-62">Genotypic values of lines in several environments were predicted using genomic prediction models; when compared across environments, the highest prediction ability was recorded at environments in Cd. Obregon (Mexico) for GY, GN, GW, and TTF. Relatively good climate, as well as optimal management of the Cd. Obregon site, are big factors influencing heritability of yield and prediction ability; sites with high heritability have higher prediction ability. However, our analysis also showed that there is no linear association between heritability and prediction ability values; nevertheless, prediction ability could be a function of <em>H</em><sup>2</sup> values and other parameters (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Spindel <em>et al.</em> 2015</a>). Another factor that could increase genomic prediction ability is incorporating high-dimensional environmental covariates (<a id="xref-ref-17-7" class="xref-bibr" href="#ref-17">Jarquin <em>et al.</em> 2014</a>; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Pérez-Rodríguez <em>et al.</em> 2015</a>). Recent studies on wheat have shown that GS selection could reshape wheat breeding because it produces higher genetic gains than conventional breeding (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bassi <em>et al.</em> 2016</a>).</p><div id="sec-22" class="subsection"><h3>Conclusions</h3><p id="p-63">Genotype × environment prediction models in genomic selection and pedigree-based selection can help accelerate breeding cycles for complex traits such as grain yield in multi-environmental trials. Traditionally, breeders have depended on phenotypic selection for generation advancement. Results of the present study show that GS is a complementary method to phenotypic selection with medium-to-high prediction ability values. Genomic prediction of GY, and other traits in spring wheat lines evaluated in a large and diverse number of international environments, indicated that sites in Mexico and India could be key sites for genomic-assisted breeding. A set of wheat lines not observed in several site-year combinations were predicted with correlations of 0.3–0.5 in Mexico and India (CV1) for models that included genomic and pedigree interaction with environments. When some of these lines were observed in some environments, this correlation increased to 0.45–0.53 (CV2).</p><p id="p-64">For less complex traits, such as GW, the prediction ability of lines not observed in sets of environments increased to about 0.6 for Mexican environments (CV1). Sets of wheat lines observed in some environments, but not in others, were predicted with correlations of up to 0.8 in Mexican and India environments (CV2) for genomic-enabled prediction models including (or not) genomic and pedigree interactions with environments.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-71">The authors acknowledge the staff at research stations where the field experiments were conducted. They also acknowledge the financial support of the International Wheat Yield Partnership, Mexico’s Secretariat of Agriculture, Livestock, Rural Development, Fisheries and Food, and Arcadia Biosciences. We appreciate the help of Jose Manual Mendoza for the use of Arcmap software.</p></div><div class="section app" id="app-1"><h2 class="">APPENDIX A</h2><div class="section" id="sec-23"><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470332/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470332/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table A1</span> <span class="caption-title">ANOVA of the 18 environments for GY, GN, GW, and TTF</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section app" id="app-2"><h2 class="">APPENDIX B</h2><div class="section" id="sec-24"><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/481/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison between heritability values and the correlation between observed and predicted values for the best and worst models in predicting trait GY in different environments for two cross-validation scenarios: (A) CV1 (the best and worst models were M7 and M1, respectively), and (B) CV2 (the best and worst models were M7 and M1, respectively)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-968864526" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparison between heritability values and the correlation between observed and predicted values for the best and worst models in predicting trait GY in different environments for two cross-validation scenarios: (A) CV1 (the best and worst models were M7 and M1, respectively), and (B) CV2 (the best and worst models were M7 and M1, respectively).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure B1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/481/F8.medium.gif" width="398" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure B1" src="https://www.g3journal.org/content/ggg/7/2/481/F8.medium.gif" width="398" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/481/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure B1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/481/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470239" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure B1</span> <p id="p-67" class="first-child">Comparison between heritability values and the correlation between observed and predicted values for the best and worst models in predicting trait GY in different environments for two cross-validation scenarios: (A) CV1 (the best and worst models were M7 and M1, respectively), and (B) CV2 (the best and worst models were M7 and M1, respectively).</p><div class="sb-div caption-clear"></div></div></div><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/481/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison between heritability values and the correlation between observed and predicted values for the best and worst models in predicting trait GW in different environments for two cross-validation scenarios (A) CV1 (the best and worst models were M3 and M1, respectively) and (B) CV2 (the best and worst models were M6 and M1, respectively)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-968864526" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparison between heritability values and the correlation between observed and predicted values for the best and worst models in predicting trait GW in different environments for two cross-validation scenarios (A) CV1 (the best and worst models were M3 and M1, respectively) and (B) CV2 (the best and worst models were M6 and M1, respectively).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure B2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/481/F9.medium.gif" width="382" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure B2" src="https://www.g3journal.org/content/ggg/7/2/481/F9.medium.gif" width="382" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/481/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure B2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/481/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470270" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure B2</span> <p id="p-68" class="first-child">Comparison between heritability values and the correlation between observed and predicted values for the best and worst models in predicting trait GW in different environments for two cross-validation scenarios (A) CV1 (the best and worst models were M3 and M1, respectively) and (B) CV2 (the best and worst models were M6 and M1, respectively).</p><div class="sb-div caption-clear"></div></div></div><div id="F10" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/481/F10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison between heritability values and the correlation between observed and predicted values for the best and worst models in predicting trait GN in different environments for two cross validation scenarios: (A) CV1 (the best and worst models were M6 and M1, respectively) and (B) CV2 (the best and worst models were M6 and M1, respectively)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-968864526" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparison between heritability values and the correlation between observed and predicted values for the best and worst models in predicting trait GN in different environments for two cross validation scenarios: (A) CV1 (the best and worst models were M6 and M1, respectively) and (B) CV2 (the best and worst models were M6 and M1, respectively).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure B3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/481/F10.medium.gif" width="405" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure B3" src="https://www.g3journal.org/content/ggg/7/2/481/F10.medium.gif" width="405" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/481/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure B3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/481/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470338" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure B3</span> <p id="p-69" class="first-child">Comparison between heritability values and the correlation between observed and predicted values for the best and worst models in predicting trait GN in different environments for two cross validation scenarios: (A) CV1 (the best and worst models were M6 and M1, respectively) and (B) CV2 (the best and worst models were M6 and M1, respectively).</p><div class="sb-div caption-clear"></div></div></div><div id="F11" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/481/F11.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison between heritability values and the correlation between observed and predicted values for the best and worst models in predicting trait TTF in different environments for two cross-validation scenarios: (A) CV1 (the best and worst models were M5 and M1, respectively) and (B) CV2 (the best and worst models were M7 and M1, respectively)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-968864526" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparison between heritability values and the correlation between observed and predicted values for the best and worst models in predicting trait TTF in different environments for two cross-validation scenarios: (A) CV1 (the best and worst models were M5 and M1, respectively) and (B) CV2 (the best and worst models were M7 and M1, respectively).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure B4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/481/F11.medium.gif" width="403" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure B4" src="https://www.g3journal.org/content/ggg/7/2/481/F11.medium.gif" width="403" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/481/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure B4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/481/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470489" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure B4</span> <p id="p-70" class="first-child">Comparison between heritability values and the correlation between observed and predicted values for the best and worst models in predicting trait TTF in different environments for two cross-validation scenarios: (A) CV1 (the best and worst models were M5 and M1, respectively) and (B) CV2 (the best and worst models were M7 and M1, respectively).</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-16"><p id="p-72"><em>Communicating editor: E. Huang</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-10-06"><span class="received-label">Received </span>October 6, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-11-24"><span class="accepted-label">Accepted </span>November 24, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Sukumaran <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.481.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aisawi</span>  <span class="cit-name-given-names">K. A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reynolds</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foulkes</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The physiological basis of the genetic progress in yield potential of CIMMYT spring wheat cultivars from 1966 to 2009.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">55</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">1749</span>–<span class="cit-lpage">1764</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D55%26rft.spage%253D1749%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.481.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arruda</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipka</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krill</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thurber</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genomic selection for predicting fusarium head blight resistance in a wheat breeding program.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">3</span>). Available at: <a href="https://dl.sciencesocieties.org/publications/tpg/abstracts/8/3/plantgenome2015.01.0003">https://dl.sciencesocieties.org/publications/tpg/abstracts/8/3/plantgenome2015.01.0003</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.481.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bassi</span>  <span class="cit-name-given-names">F. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bentley</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charmet</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ortiz</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Breeding schemes for the implementation of genomic selection in wheat (<em>Triticum</em> spp.).</span> <abbr class="cit-jnl-abbrev">Plant Sci.</abbr> <span class="cit-vol">242</span>: <span class="cit-fpage">23</span>–<span class="cit-lpage">36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BSci.%26rft.volume%253D242%26rft.spage%253D23%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.481.4" data-doi="10.2135/cropsci2006.11.0690"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Prospects for genome-wide selection for quantitative traits in maize.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">1082</span>–<span class="cit-lpage">1090</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBernardo%26rft.auinit1%253DR.%26rft.volume%253D47%26rft.issue%253D3%26rft.spage%253D1082%26rft.epage%253D1090%26rft.atitle%253DProspects%2Bfor%2BGenomewide%2BSelection%2Bfor%2BQuantitative%2BTraits%2Bin%2BMaize%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2006.11.0690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2006.11.0690&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000247424200020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-book" id="cit-7.2.481.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Braun</span>  <span class="cit-name-given-names">H.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atlin</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Payne</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Multi-location testing as a tool to identify plant response to global climate change</span>, pp. <span class="cit-fpage">115</span>–<span class="cit-lpage">138</span> in <span class="cit-source">Climate Change and Crop Production</span>, Vol. <span class="cit-vol">1</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Reynolds</span>  <span class="cit-name-given-names">M. P.</span></span>. <span class="cit-publ-name">CABI Climate Change Series</span>, <span class="cit-publ-loc">UK</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClimate%2BChange%2Band%2BCrop%2BProduction%26rft.volume%253D1%26rft.spage%253D115%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.481.6" data-doi="10.2135/cropsci2010.07.0403"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cotes</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">San Vicente</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Das</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Prediction assessment of linear mixed models for multi-environment trials.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">51</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">944</span>–<span class="cit-lpage">954</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D51%26rft.spage%253D944%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2010.07.0403%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2010.07.0403&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.481.7" data-doi="10.2135/cropsci2011.06.0299"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic prediction of breeding values when modeling genotype × environment interaction using pedigree and dense molecular markers.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">707</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D707%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.06.0299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.06.0299&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.481.8" data-doi="10.1534/genetics.110.118521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campos Gde</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueno</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">713</span>–<span class="cit-lpage">724</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.118521%26rft_id%253Dinfo%253Apmid%252F20813882%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg2LzIvNzEzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzQ4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.481.9" data-doi="10.1038/hdy.2013.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueno</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ornella</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic prediction in CIMMYT maize and wheat breeding programs.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">112</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">48</span>–<span class="cit-lpage">60</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D112%26rft.spage%253D48%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2013.16%26rft_id%253Dinfo%253Apmid%252F23572121%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2013.16&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23572121&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F481.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.481.10" data-doi="10.2135/cropsci2015.04.0260"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maccaferri</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tuberosa</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Extending the marker × environment interaction model for genomic-enabled prediction and genome-wise association analyses in Durum wheat.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.2135/cropsci2015.04.0260</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2015.04.0260%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2015.04.0260&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.481.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jarquín</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franco</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomic prediction of gene bank wheat landraces.</span> <abbr class="cit-jnl-abbrev">G3</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1834</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft.volume%253D6%26rft.spage%253D1819%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-web" id="cit-7.2.481.12"><div class="cit-metadata unstructured">de los Campos, G., and P. Pérez-Rodríguez, 2013 BGLR: Bayesian Generalized Linear Regression. <em>R package version</em> 1 (3). Available at: <a href="https://cran.r-project.org/web/packages/BGLR/BGLR.pdf">https://cran.r-project.org/web/packages/BGLR/BGLR.pdf</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.481.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vazquez</span>  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome-enabled prediction using the BLR (Bayesian Linear Regression) R-package.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">1019</span>: <span class="cit-fpage">299</span>–<span class="cit-lpage">320</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BMol.%2BBiol.%26rft.volume%253D1019%26rft.spage%253D299%26rft_id%253Dinfo%253Apmid%252F23756896%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23756896&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F481.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.481.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edae</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Byrne</span>  <span class="cit-name-given-names">P. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manmathan</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haley</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moragues</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Association mapping and nucleotide sequence variation in five drought tolerance candidate genes in spring wheat.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">6</span>(<span class="cit-issue">2</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.481.15" data-doi="10.1007/s00122-013-2257-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edae</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Byrne</span>  <span class="cit-name-given-names">P. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haley</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lopes</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reynolds</span>  <span class="cit-name-given-names">M. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-wide association mapping of yield and yield components of spring wheat under contrasting moisture regimes.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">791</span>–<span class="cit-lpage">807</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D791%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2257-8%26rft_id%253Dinfo%253Apmid%252F24408378%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2257-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24408378&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F481.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.481.16" data-doi="10.1007/s00122-013-2231-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akdemir</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Integrating environmental covariates and crop modeling into the genomic selection framework to predict genotype by environment interactions.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">463</span>–<span class="cit-lpage">480</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D463%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2231-5%26rft_id%253Dinfo%253Apmid%252F24264761%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2231-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24264761&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F481.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.481.17" data-doi="10.1007/s00122-013-2243-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jarquin</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lacaze</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Du Cheyron</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daucourt</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A reaction norm model for genomic selection using high-dimensional genomic and environmental data.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">595</span>–<span class="cit-lpage">607</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D595%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2243-1%26rft_id%253Dinfo%253Apmid%252F24337101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2243-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24337101&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F481.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.481.18" data-doi="10.2135/cropsci2011.09.0467"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lopes</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reynolds</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manes</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genetic yield gains and changes in associated traits of CIMMYT spring bread wheat in a “historic” set representing 30 years of breeding.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">1123</span>–<span class="cit-lpage">1131</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D1123%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.09.0467%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.09.0467&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000303367300015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.481.19" data-doi="10.1007/s00122-014-2444-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lopes</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dreisigacker</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pena</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sukumaran</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reynolds</span>  <span class="cit-name-given-names">M. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genetic characterization of the wheat association mapping initiative (WAMI) panel for dissection of complex traits in spring wheat.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">128</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">453</span>–<span class="cit-lpage">464</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D128%26rft.spage%253D453%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-014-2444-2%26rft_id%253Dinfo%253Apmid%252F25540818%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-014-2444-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25540818&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F481.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.481.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">López-Cruz</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonnet</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dreisigacker</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Increased prediction accuracy in wheat breeding trials using a markers × environment interaction genomic selection model.</span> <abbr class="cit-jnl-abbrev">G3</abbr> <span class="cit-vol">5</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">569</span>–<span class="cit-lpage">582</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft.volume%253D5%26rft.spage%253D569%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.481.21" data-doi="10.2135/cropsci2011.10.0574"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Manes</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gomez</span>  <span class="cit-name-given-names">H. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Puhl</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reynolds</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Braun</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genetic yield gains of the CIMMYT international semi-arid wheat yield trials from 1994 to 2010.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">1543</span>–<span class="cit-lpage">1552</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D1543%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.10.0574%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.10.0574&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000305804900009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.481.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzIvNDgxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.481.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muranty</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Troggio</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sadok</span>  <span class="cit-name-given-names">I. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Al Rifaï</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auwerkerken</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Accuracy and responses of genomic selection on key traits in apple breeding.</span> <abbr class="cit-jnl-abbrev">Hortic. Res.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">15060</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHortic.%2BRes.%26rft.volume%253D2%26rft.spage%253D15060%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.481.24" data-doi="10.1534/genetics.114.164442"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-wide regression and prediction with the BGLR statistical package.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">483</span>–<span class="cit-lpage">495</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.164442%26rft_id%253Dinfo%253Apmid%252F25009151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzIvNDgzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzQ4MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.481.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bondalapati</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Meyer</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pita</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A pedigree-based reaction norm model for prediction of cotton yield in multienvironment trials.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">55</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">1143</span>–<span class="cit-lpage">1151</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D55%26rft.spage%253D1143%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.481.26" data-doi="10.1016/j.pbi.2016.04.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reynolds</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Langridge</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Physiological breeding.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">162</span>–<span class="cit-lpage">171</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BPlant%2BBiol.%26rft.volume%253D31%26rft.spage%253D162%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pbi.2016.04.005%26rft_id%253Dinfo%253Apmid%252F27161822%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pbi.2016.04.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27161822&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F481.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.481.27" data-doi="10.1093/jxb/erp016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reynolds</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foulkes</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slafer</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berry</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parry</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Raising yield potential in wheat.</span> <abbr class="cit-jnl-abbrev">J. Exp. Bot.</abbr> <span class="cit-vol">60</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">1899</span>–<span class="cit-lpage">1918</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Ferp016%26rft_id%253Dinfo%253Apmid%252F19363203%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamV4Ym90IjtzOjU6InJlc2lkIjtzOjk6IjYwLzcvMTg5OSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi80ODEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.481.28" data-doi="10.1111/j.1365-3040.2012.02588.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reynolds</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foulkes</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Furbank</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Griffiths</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Achieving yield gains in wheat.</span> <abbr class="cit-jnl-abbrev">Plant Cell Environ.</abbr> <span class="cit-vol">35</span>(<span class="cit-issue">10</span>): <span class="cit-fpage">1799</span>–<span class="cit-lpage">1823</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%252C%2BCell%252C%2Band%2BEnvironment%2B%2528Print%2529%26rft.stitle%253DPlant%252C%2BCell%252C%2Band%2BEnvironment%2B%2528Print%2529%26rft.aulast%253DReynolds%26rft.auinit1%253DM.%26rft.volume%253D35%26rft.issue%253D10%26rft.spage%253D1799%26rft.epage%253D1823%26rft.atitle%253DAchieving%2Byield%2Bgains%2Bin%2Bwheat.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-3040.2012.02588.x%26rft_id%253Dinfo%253Apmid%252F22860982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-3040.2012.02588.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22860982&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F481.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000308395600008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-book" id="cit-7.2.481.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">SAS Institute Inc</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-source"><em>Base SAS 9.2. Procedures Guide: Statistical Procedures</em>.</span> <span class="cit-edition">Ed. 3</span>. <span class="cit-publ-name">SAS Institute Inc.</span>, <span class="cit-publ-loc">Cary, NC</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.481.30" data-doi="10.2135/cropsci1997.0011183X003700010006x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sayre</span>  <span class="cit-name-given-names">K. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rajaram</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Yield potential progress in short bread wheats in northwest Mexico.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">37</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">36</span>–<span class="cit-lpage">42</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DSayre%26rft.auinit1%253DK.%2BD.%26rft.volume%253D37%26rft.issue%253D1%26rft.spage%253D36%26rft.epage%253D42%26rft.atitle%253DYield%2BPotential%2BProgress%2Bin%2BShort%2BBread%2BWheats%2Bin%2BNorthwest%2BMexico%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1997.0011183X003700010006x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1997.0011183X003700010006x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1997WE97700006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.481.31" data-doi="10.1371/journal.pgen.1004982"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spindel</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Begum</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akdemir</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Virk</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collard</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genomic selection and association mapping in rice (<em>Oryza sativa</em>): effect of trait genetic architecture, training population composition, marker number and statistical model on accuracy of rice genomic selection in elite, tropical rice breeding lines.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">e1004982</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1004982%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004982%26rft_id%253Dinfo%253Apmid%252F25689273%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004982&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25689273&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F481.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.481.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sukumaran</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dreisigacker</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lopes</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chavez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reynolds</span>  <span class="cit-name-given-names">M. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span><span class="cit-comment">a</span> <span class="cit-article-title">Genome-wide association study for grain yield and related traits in an elite spring wheat population grown in temperate irrigated environments.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">128</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">353</span>–<span class="cit-lpage">363</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D128%26rft.spage%253D353%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.481.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sukumaran</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reynolds</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lopes</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span><span class="cit-comment">b</span> <span class="cit-article-title">Genome-wide association study for adaptation to agronomic plant density: a component of high yield potential in spring wheat.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">55</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">2609</span>–<span class="cit-lpage">2619</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D55%26rft.spage%253D2609%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.481.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sukumaran</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lopes</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dreisigacker</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dixon</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zikhali</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Identification of earliness per se in spring wheat through a genome wide association study.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">2962</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D56%26rft.spage%253D2962%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.481.35" data-doi="10.1126/science.1183700"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tester</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Langridge</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Breeding technologies to increase crop production in a changing world.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">327</span>(<span class="cit-issue">5967</span>): <span class="cit-fpage">818</span>–<span class="cit-lpage">822</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTester%26rft.auinit1%253DM.%26rft.volume%253D327%26rft.issue%253D5967%26rft.spage%253D818%26rft.epage%253D822%26rft.atitle%253DBreeding%2BTechnologies%2Bto%2BIncrease%2BCrop%2BProduction%2Bin%2Ba%2BChanging%2BWorld%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1183700%26rft_id%253Dinfo%253Apmid%252F20150489%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjcvNTk2Ny84MTgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzIvNDgxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.481.36" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F481.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.481.37" data-doi="10.1007/s00122&amp;ndash;016&amp;ndash;2726-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Velu</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dreisigacker</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomic prediction for grain zinc and iron concentrations in spring wheat.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">1595</span>–<span class="cit-lpage">1605</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122–016–2726-y</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D129%26rft.spage%253D1595%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122%2526ndash%253B016%2526ndash%253B2726-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122&amp;ndash;016&amp;ndash;2726-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.481.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Semaqn</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beyene</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Babu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic prediction in biparental tropical maize populations in water-stressed and well- watered environments using low-density and GBS SNPs.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">114</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">291</span>–<span class="cit-lpage">299</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D114%26rft.spage%253D291%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/2/481.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/2/467" title="Transcriptome Analysis of Drosophila melanogaster Third Instar Larval Ring Glands Points to Novel Functions and Uncovers a Cytochrome p450 Required for Development" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/2/497" title="Genetic and Targeted eQTL Mapping Reveals Strong Candidate Genes Modulating the Stress Response During Chicken Domestication" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/2"><span>Volume 7 Issue 2, February 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/2.cover-source.jpg?itok=m4uMC43I" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/2/481" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-hHP0XwWYhFlOnehO-WCsX0Zo_wkBfMW0yda6q87vzU0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/2/481" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/2/481" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/2/481" class="active" data-icon-position="" data-hide-link-title="0">Genomic Prediction with Pedigree and Genotype × Environment Interaction in Spring Wheat Grown in South and West Asia, North Africa, and Mexico</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460561" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-ZNJ9oDegNbmK7dy7cGCje-AV7X31aXdTV6NMm3hAFLA" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/2/481.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/2/481" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-6IMezn5OO_ZLlc3pA-nZOaqI1srYN4Thbeg8E2XZGBA" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/2/481" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/2/481.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460561" id="node460561--2" data-pisa="ggg;7/2/481" data-pisa-master="ggg;g3.116.036251" data-apath="/ggg/7/2/481.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Prediction with Pedigree and Genotype × Environment Interaction in Spring Wheat Grown in South and West Asia, North Africa, and Mexico</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-4088-4624" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sivakumar Sukumaran</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jose Crossa</span>, <span class="highwire-citation-author" data-delta="2">Diego Jarquin</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-7698-6019" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Marta Lopes</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-4291-4316" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Matthew P. Reynolds</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">481-495;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.036251 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460561--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sivakumar Sukumaran </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), El Batan, Texcoco CP 56237, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sivakumar%2BSukumaran%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sukumaran%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASivakumar%2BSukumaran%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4088-4624" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sivakumar Sukumaran</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jose Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Biometrics and Statistics Unit, CIMMYT, El Batan, Texcoco CP 56237, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jose%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJose%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jose Crossa</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Diego Jarquin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Agronomy and Horticulture, University of Nebraska, Lincoln, Nebraska 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Diego%2BJarquin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jarquin%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADiego%2BJarquin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Marta Lopes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Global Wheat Program, CIMMYT, Emex, 06511 Ankara, Turkey</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marta%2BLopes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lopes%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMarta%2BLopes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7698-6019" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Marta Lopes</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Matthew P. Reynolds </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), El Batan, Texcoco CP 56237, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BP.%2BReynolds%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reynolds%20MP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMatthew%2BP.%2BReynolds%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4291-4316" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Matthew P. Reynolds</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/2/481" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/2/481&amp;title=Genomic%20Prediction%20with%20Pedigree%20and%20Genotype%20%C3%97%20Environment%20Interaction%20in%20Spring%20Wheat%20Grown%20in%20South%20and%20West%20Asia%2C%20North%20Africa%2C%20and%20Mexico" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/2/481&amp;title=Genomic%20Prediction%20with%20Pedigree%20and%20Genotype%20%C3%97%20Environment%20Interaction%20in%20Spring%20Wheat%20Grown%20in%20South%20and%20West%20Asia%2C%20North%20Africa%2C%20and%20Mexico" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/2/481&amp;title=Genomic%20Prediction%20with%20Pedigree%20and%20Genotype%20%C3%97%20Environment%20Interaction%20in%20Spring%20Wheat%20Grown%20in%20South%20and%20West%20Asia%2C%20North%20Africa%2C%20and%20Mexico" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/2/481&amp;text=Genomic%20Prediction%20with%20Pedigree%20and%20Genotype%20%C3%97%20Environment%20Interaction%20in%20Spring%20Wheat%20Grown%20in%20South%20and%20West%20Asia%2C%20North%20Africa%2C%20and%20Mexico" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/2/481&amp;title=Genomic%20Prediction%20with%20Pedigree%20and%20Genotype%20%C3%97%20Environment%20Interaction%20in%20Spring%20Wheat%20Grown%20in%20South%20and%20West%20Asia%2C%20North%20Africa%2C%20and%20Mexico" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/2/481&amp;t=Genomic%20Prediction%20with%20Pedigree%20and%20Genotype%20%C3%97%20Environment%20Interaction%20in%20Spring%20Wheat%20Grown%20in%20South%20and%20West%20Asia%2C%20North%20Africa%2C%20and%20Mexico" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/2/481&amp;title=Genomic%20Prediction%20with%20Pedigree%20and%20Genotype%20%C3%97%20Environment%20Interaction%20in%20Spring%20Wheat%20Grown%20in%20South%20and%20West%20Asia%2C%20North%20Africa%2C%20and%20Mexico" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/2/481&amp;title=Genomic%20Prediction%20with%20Pedigree%20and%20Genotype%20%C3%97%20Environment%20Interaction%20in%20Spring%20Wheat%20Grown%20in%20South%20and%20West%20Asia%2C%20North%20Africa%2C%20and%20Mexico" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460561/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460561" id="node460561--3" data-pisa="ggg;7/2/481" data-pisa-master="ggg;g3.116.036251" data-apath="/ggg/7/2/481.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Prediction with Pedigree and Genotype × Environment Interaction in Spring Wheat Grown in South and West Asia, North Africa, and Mexico</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-4088-4624" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sivakumar Sukumaran</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jose Crossa</span>, <span class="highwire-citation-author" data-delta="2">Diego Jarquin</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-7698-6019" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Marta Lopes</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-4291-4316" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Matthew P. Reynolds</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">481-495;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.036251 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460561--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sivakumar Sukumaran </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), El Batan, Texcoco CP 56237, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sivakumar%2BSukumaran%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sukumaran%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASivakumar%2BSukumaran%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4088-4624" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sivakumar Sukumaran</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jose Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Biometrics and Statistics Unit, CIMMYT, El Batan, Texcoco CP 56237, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jose%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJose%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jose Crossa</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Diego Jarquin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Agronomy and Horticulture, University of Nebraska, Lincoln, Nebraska 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Diego%2BJarquin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jarquin%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADiego%2BJarquin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Marta Lopes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Global Wheat Program, CIMMYT, Emex, 06511 Ankara, Turkey</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marta%2BLopes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lopes%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMarta%2BLopes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7698-6019" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Marta Lopes</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Matthew P. Reynolds </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), El Batan, Texcoco CP 56237, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BP.%2BReynolds%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reynolds%20MP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMatthew%2BP.%2BReynolds%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4291-4316" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Matthew P. Reynolds</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=481&amp;rft.epage=495&amp;rft.atitle=Genomic%20Prediction%20with%20Pedigree%20and%20Genotype%20%C3%97%20Environment%20Interaction%20in%20Spring%20Wheat%20Grown%20in%20South%20and%20West%20Asia%2C%20North%20Africa%2C%20and%20Mexico&amp;rft.volume=7&amp;rft.issue=2&amp;rft.date=2017-02-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Sukumaran%2C+Sivakumar&amp;rft.au=Crossa%2C+Jose&amp;rft.au=Jarquin%2C+Diego&amp;rft.au=Lopes%2C+Marta&amp;rft.au=Reynolds%2C+Matthew+P."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460561/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460561/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460561/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460561/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460561/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460561/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460561/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460561/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460561/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460561/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460561/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460561/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/2/481" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-17" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-21" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">APPENDIX A</a></li><li class="odd"><a href="#app-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">APPENDIX B</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/2/481.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/2/481.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/7/2/481.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
