<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/2/427" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/2/427.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/2/427.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/2/427.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/2/427.atom" />
<meta name="HW.pisa" content="ggg;7/2/427" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genomics of Compensatory Adaptation in Experimental Populations of Aspergillus nidulans" />
<meta name="DC.Identifier" content="10.1534/g3.116.036152" />
<meta name="DC.Date" content="2017-02-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Dettman et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Knowledge of the number and nature of genetic changes responsible for adaptation is essential for understanding and predicting evolutionary trajectories. Here, we study the genomic basis of compensatory adaptation to the fitness cost of fungicide resistance in experimentally evolved strains of the filamentous fungus Aspergillus nidulans . The original selection experiment tracked the fitness recovery of lines founded by an ancestral strain that was resistant to fludioxonil, but paid a fitness cost in the absence of the fungicide. We obtained whole-genome sequence data for the ancestral A. nidulans strain and eight experimentally evolved strains. We find that fludioxonil resistance in the ancestor was likely conferred by a mutation in histidine kinase nikA , part of the two-component signal transduction system of the high-osmolarity glycerol (HOG) stress response pathway. To compensate for the pleiotropic negative effects of the resistance mutation, the subsequent fitness gains observed in the evolved lines were likely caused by secondary modification of HOG pathway activity. Candidate genes for the compensatory fitness increases were significantly overrepresented by stress response functions, and some were specifically associated with the HOG pathway itself. Parallel evolution at the gene level was rare among evolved lines. There was a positive relationship between the predicted number of adaptive steps, estimated from fitness data, and the number of genomic mutations, determined by whole-genome sequencing. However, the number of genomic mutations was, on average, 8.45 times greater than the number of adaptive steps inferred from fitness data. This research expands our understanding of the genetic basis of adaptation in multicellular eukaryotes and lays out a framework for future work on the genomics of compensatory adaptation in A. nidulans ." />
<meta name="DC.Contributor" content="Jeremy R. Dettman" />
<meta name="DC.Contributor" content="Nicolas Rodrigue" />
<meta name="DC.Contributor" content="Sijmen E. Schoustra" />
<meta name="DC.Contributor" content="Rees Kassen" />
<meta name="article:published_time" content="2017-02-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Genomics of Compensatory Adaptation in Experimental Populations of Aspergillus nidulans" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Knowledge of the number and nature of genetic changes responsible for adaptation is essential for understanding and predicting evolutionary trajectories. Here, we study the genomic basis of compensatory adaptation to the fitness cost of fungicide resistance in experimentally evolved strains of the filamentous fungus &lt;i&gt;Aspergillus nidulans&lt;/i&gt;. The original selection experiment tracked the fitness recovery of lines founded by an ancestral strain that was resistant to fludioxonil, but paid a fitness cost in the absence of the fungicide. We obtained whole-genome sequence data for the ancestral &lt;i&gt;A. nidulans&lt;/i&gt; strain and eight experimentally evolved strains. We find that fludioxonil resistance in the ancestor was likely conferred by a mutation in histidine kinase &lt;i&gt;nikA&lt;/i&gt;, part of the two-component signal transduction system of the high-osmolarity glycerol (HOG) stress response pathway. To compensate for the pleiotropic negative effects of the resistance mutation, the subsequent fitness gains observed in the evolved lines were likely caused by secondary modification of HOG pathway activity. Candidate genes for the compensatory fitness increases were significantly overrepresented by stress response functions, and some were specifically associated with the HOG pathway itself. Parallel evolution at the gene level was rare among evolved lines. There was a positive relationship between the predicted number of adaptive steps, estimated from fitness data, and the number of genomic mutations, determined by whole-genome sequencing. However, the number of genomic mutations was, on average, 8.45 times greater than the number of adaptive steps inferred from fitness data. This research expands our understanding of the genetic basis of adaptation in multicellular eukaryotes and lays out a framework for future work on the genomics of compensatory adaptation in &lt;i&gt;A. nidulans&lt;/i&gt;.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/02/01" />
<meta name="citation_mjid" content="ggg;7/2/427" />
<meta name="citation_id" content="7/2/427" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/2/427" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/2/427.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/2/427.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/2/427.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.036152" />
<meta name="citation_pmid" content="27903631" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="2" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="427" />
<meta name="citation_lastpage" content="436" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Jeremy R. Dettman" />
<meta name="citation_author_institution" content="Department of Biology, University of Ottawa, Ontario K1N 6N5, Canada" />
<meta name="citation_author_institution" content="Centre for Advanced Research in Environmental Genomics, University of Ottawa, Ontario K1N 6N5, Canada" />
<meta name="citation_author_email" content="jdettman@uottawa.ca" />
<meta name="citation_author" content="Nicolas Rodrigue" />
<meta name="citation_author_institution" content="Department of Biology, Carleton University, Ottawa, Ontario K1S 5B6, Canada" />
<meta name="citation_author" content="Sijmen E. Schoustra" />
<meta name="citation_author_institution" content="Laboratory of Genetics, Wageningen University, NL-6708 PB, The Netherlands" />
<meta name="citation_author" content="Rees Kassen" />
<meta name="citation_author_institution" content="Department of Biology, University of Ottawa, Ontario K1N 6N5, Canada" />
<meta name="citation_author_institution" content="Centre for Advanced Research in Environmental Genomics, University of Ottawa, Ontario K1N 6N5, Canada" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. R. Andersen;citation_author=J. B. Nielsen;citation_author=A. Klitgaard;citation_author=L. M. Petersen;citation_author=M. Zachariasen;citation_title=Accurate prediction of secondary metabolite gene clusters in filamentous fungi.;citation_volume=110;citation_year=2013;citation_pmid=23248299;citation_doi=10.1073/pnas.1205532110" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=J. B. Anderson;citation_author=J. Funt;citation_author=D. A. Thompson;citation_author=S. Prabhu;citation_author=A. Socha;citation_title=Determinants of divergent adaptation and Dobzhansky-Muller interaction in experimental yeast populations.;citation_pages=1383-1388;citation_volume=20;citation_year=2010;citation_issue=15;citation_pmid=20637622;citation_doi=10.1016/j.cub.2010.06.022" />
<meta name="citation_reference" content="citation_journal_title=Molecular genetics and genomics :  MGG;citation_journal_abbrev=Mol Genet Genomics;citation_author=A. Balazs;citation_author=I. Pocsi;citation_author=Z. Hamari;citation_author=E. Leiter;citation_author=T. Emri;citation_title=AtfA bZIP-type transcription factor regulates oxidative and osmotic stress responses in Aspergillus nidulans.;citation_pages=289-303;citation_volume=283;citation_year=2010;citation_issue=3;citation_pmid=20131067;citation_doi=10.1007/s00438-010-0513-z" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=J. E. Barrick;citation_author=G. Colburn;citation_author=D. E. Deatherage;citation_author=C. C. Traverse;citation_author=M. D. Strand;citation_title=Identifying structural variation in haploid microbial genomes from short-read resequencing data using breseq.;citation_pages=1039;citation_volume=15;citation_year=2014;citation_pmid=25432719;citation_doi=10.1186/1471-2164-15-1039" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. K. Burke;citation_author=J. P. Dunham;citation_author=P. Shahrestani;citation_author=K. R. Thornton;citation_author=M. R. Rose;citation_title=Genome-wide analysis of a long-term evolution experiment with Drosophila.;citation_pages=587-590;citation_volume=467;citation_year=2010;citation_issue=7315;citation_pmid=20844486;citation_doi=10.1038/nature09352" />
<meta name="citation_reference" content="citation_journal_title=Fly (Austin);citation_author=P. Cingolani;citation_author=A. Platts;citation_author=L. L. Wang;citation_author=M. Coon;citation_author=T. Nguyen;citation_title=A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w(1118); iso-2; iso-3.;citation_pages=80-92;citation_volume=6;citation_year=2012;citation_pmid=22728672;citation_doi=10.4161/fly.19695" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=T. M. Conrad;citation_author=A. R. Joyce;citation_author=M. K. Applebee;citation_author=C. L. Barrett;citation_author=B. Xie;citation_title=Whole-genome resequencing of Escherichia coli K-12 MG1655 undergoing short-term laboratory evolution in lactate minimal media reveals flexible selection of adaptive mutations.;citation_pages=R118-R118;citation_volume=10;citation_year=2009;citation_issue=10;citation_pmid=19849850;citation_doi=10.1186/gb-2009-10-10-r118" />
<meta name="citation_reference" content="citation_journal_title=Fungal Genet. Biol.;citation_author=P. W. J. de Groot;citation_author=B. W. Brandt;citation_author=H. Horiuchi;citation_author=A. F. J. Ram;citation_author=C. G. de Koster;citation_title=Comprehensive genomic analysis of cell wall genes in Aspergillus nidulans.;citation_pages=S72-S81;citation_volume=46;citation_year=2009;citation_pmid=19585695;citation_doi=10.1016/j.fgb.2008.07.022" />
<meta name="citation_reference" content="citation_journal_title=Fungal genetics and biology :  FG &amp; B;citation_journal_abbrev=Fungal Genet Biol;citation_author=G. Del Sorbo;citation_author=H. J. Schoonbeek;citation_author=M. A. De Waard;citation_title=Fungal transporters involved in efflux of natural toxic compounds and fungicides.;citation_pages=1-15;citation_volume=30;citation_year=2000;citation_issue=1;citation_pmid=10955904;citation_doi=10.1006/fgbi.2000.1206" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=C. P. De Souza;citation_author=S. B. Hashmi;citation_author=A. H. Osmani;citation_author=P. Andrews;citation_author=C. S. Ringelberg;citation_title=Functional analysis of the Aspergillus nidulans kinome.;citation_pages=e58008;citation_volume=8;citation_year=2013;citation_pmid=23505451;citation_doi=10.1371/journal.pone.0058008" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=J. R. Dettman;citation_author=N. Rodrigue;citation_author=A. H. Melnyk;citation_author=A. Wong;citation_author=S. F. Bailey;citation_title=Evolutionary insight from whole-genome sequencing of experimentally evolved microbes.;citation_pages=2058-2077;citation_volume=21;citation_year=2012;citation_issue=9;citation_pmid=22332770;citation_doi=10.1111/j.1365-294X.2012.05484.x" />
<meta name="citation_reference" content="citation_journal_title=Toxins (Basel);citation_author=R. Duran;citation_author=J. W. Cary;citation_author=A. M. Calvo;citation_title=Role of the osmotic stress regulatory pathway in morphogenesis and secondary metabolism in filamentous fungi.;citation_pages=367-381;citation_volume=2;citation_year=2010;citation_pmid=22069590;citation_doi=10.3390/toxins2040367" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=S. F. Elena;citation_author=R. E. Lenski;citation_title=Evolution experiments with microorganisms: the dynamics and genetic bases of adaptation.;citation_pages=457-469;citation_volume=4;citation_year=2003;citation_issue=6;citation_pmid=12776215;citation_doi=10.1038/nrg1088" />
<meta name="citation_reference" content="citation_journal_title=MBio.;citation_author=I. V. Ene;citation_author=L. A. Walker;citation_author=M. Schiavone;citation_author=K. K. Lee;citation_author=H. Martin-Yken;citation_title=Cell wall remodeling enzymes modulate fungal cell wall elasticity and osmotic stress resistance.;citation_pages=e00986;citation_volume=6;citation_year=2015;citation_pmid=26220968" />
<meta name="citation_reference" content="citation_journal_title=Molecular Systems Biology;citation_journal_abbrev=Mol Syst Biol;citation_author=M. Filteau;citation_author=V. Hamel;citation_author=M.-C. Pouliot;citation_author=I. Gagnon-Arsenault;citation_author=A. K. Dube;citation_title=Evolutionary rescue by compensatory mutations is constrained by genomic and environmental backgrounds;citation_pages=832-832;citation_volume=11;citation_year=2015;citation_issue=10;citation_pmid=26459777;citation_doi=10.15252/msb.20156444" />
<meta name="citation_reference" content="citation_title=The Genetical Theory of Natural Selection.;citation_author=R. A. Fisher;citation_year=1930" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=K. Furukawa;citation_author=Y. Hoshi;citation_author=T. Maeda;citation_author=T. Nakajima;citation_author=K. Abe;citation_title=Aspergillus nidulans HOG pathway is activated only by two-component signalling pathway in response to osmotic stress.;citation_pages=1246-1261;citation_volume=56;citation_year=2005;citation_issue=5;citation_pmid=15882418;citation_doi=10.1111/j.1365-2958.2005.04605.x" />
<meta name="citation_reference" content="citation_journal_title=Bioscience, Biotechnology, and Biochemistry;citation_journal_abbrev=Bioscience, Biotechnology, and Biochemistry;citation_author=K. Furukawa;citation_author=A. Yoshimi;citation_author=T. Furukawa;citation_author=Y. Hoshi;citation_author=D. Hagiwara;citation_title=Novel reporter gene expression systems for monitoring activation of the Aspergillus nidulans HOG pathway.;citation_pages=1724-1730;citation_volume=71;citation_year=2007;citation_issue=7;citation_pmid=17617716;citation_doi=10.1271/bbb.70131" />
<meta name="citation_reference" content="citation_journal_title=Eukaryot. Cell;citation_author=T. Futagami;citation_author=S. Nakao;citation_author=Y. Kido;citation_author=T. Oka;citation_author=Y. Kajiwara;citation_title=Putative stress sensors WscA and WscB are involved in hypo-osmotic and acidic pH stress tolerance in Aspergillus nidulans.;citation_volume=10;citation_year=2011;citation_pmid=21926329;citation_doi=10.1128/EC.05080-11" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. E. Galagan;citation_author=S. E. Calvo;citation_author=C. Cuomo;citation_author=L. J. Ma;citation_author=J. R. Wortman;citation_title=Sequencing of Aspergillus nidulans and comparative analysis with A. fumigatus and A. oryzae.;citation_pages=1105-1115;citation_volume=438;citation_year=2005;citation_issue=7071;citation_pmid=16372000;citation_doi=10.1038/nature04341" />
<meta name="citation_reference" content="citation_journal_title=Bioscience, Biotechnology, and Biochemistry;citation_journal_abbrev=Bioscience, Biotechnology, and Biochemistry;citation_author=D. Hagiwara;citation_author=Y. Matsubayashi;citation_author=J. Marui;citation_author=K. Furukawa;citation_author=T. Yamashino;citation_title=Characterization of the NikA histidine kinase implicated in the phosphorelay signal transduction of Aspergillus nidulans, with special reference to fungicide responses.;citation_pages=844-847;citation_volume=71;citation_year=2007;citation_issue=3;citation_pmid=17341812;citation_doi=10.1271/bbb.70051" />
<meta name="citation_reference" content="citation_journal_title=Fungal genetics and biology :  FG &amp; B;citation_journal_abbrev=Fungal Genet Biol;citation_author=D. Hagiwara;citation_author=Y. Asano;citation_author=J. Marui;citation_author=A. Yoshimi;citation_author=T. Mizuno;citation_title=Transcriptional profiling for Aspergillusnidulans HogA MAPK signaling pathway in response to fludioxonil and osmotic stress.;citation_pages=868-878;citation_volume=46;citation_year=2009;citation_issue=11;citation_pmid=19596074;citation_doi=10.1016/j.fgb.2009.07.003" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=D. Hagiwara;citation_author=A. Takahashi-Nakaguchi;citation_author=T. Toyotome;citation_author=A. Yoshimi;citation_author=K. Abe;citation_title=NikA/TcsC histidine kinase is involved in conidiation, hyphal morphology, and responses to osmotic stress and antifungal chemicals in Aspergillus fumigatus.;citation_pages=e80881;citation_volume=8;citation_year=2013;citation_pmid=24312504;citation_doi=10.1371/journal.pone.0080881" />
<meta name="citation_reference" content="citation_journal_title=Eukaryot. Cell;citation_author=R. Jaimes-Arroyo;citation_author=F. Lara-Rojas;citation_author=O. Bayram;citation_author=O. Valerius;citation_author=G. H. Braus;citation_title=The SrkA kinase is part of the SakA mitogen-activated protein kinase interactome and regulates stress responses and development in Aspergillus nidulans.;citation_volume=14;citation_year=2015;citation_pmid=25820520;citation_doi=10.1128/EC.00277-14" />
<meta name="citation_reference" content="citation_journal_title=Curr. Opin. Genet. Dev.;citation_author=E. R. Jerison;citation_author=M. M. Desai;citation_title=Genomic investigations of evolutionary dynamics and epistasis in microbial evolution experiments.;citation_pages=33-39;citation_volume=35;citation_year=2015;citation_pmid=26370471;citation_doi=10.1016/j.gde.2015.08.008" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=L. Kang;citation_author=D. D. Aggarwal;citation_author=E. Rashkovetsky;citation_author=A. B. Korol;citation_author=P. Michalak;citation_title=Rapid genomic changes in Drosophila melanogaster adapting to desiccation stress in an experimental evolution system.;citation_pages=233;citation_volume=17;citation_year=2016" />
<meta name="citation_reference" content="citation_title=Experimental Evolution and the Nature of Biodiversity.;citation_author=R. Kassen;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. C. Koboldt;citation_author=Q. Zhang;citation_author=D. E. Larson;citation_author=D. Shen;citation_author=M. D. McLellan;citation_title=VarScan 2: somatic mutation and copy number alteration discovery in cancer by exome sequencing.;citation_volume=22;citation_year=2012;citation_pmid=22300766;citation_doi=10.1101/gr.129684.111" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. Kofler;citation_author=P. Orozco-terWengel;citation_author=N. De Maio;citation_author=R. V. Pandey;citation_author=V. Nolte;citation_title=PoPoolation: a toolbox for population genetic analysis of next generation sequencing data from pooled individuals.;citation_pages=e15925-e15925;citation_volume=6;citation_year=2011;citation_issue=1;citation_pmid=21253599;citation_doi=10.1371/journal.pone.0015925" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=D. J. Kvitek;citation_author=G. Sherlock;citation_title=Reciprocal sign epistasis between frequently experimentally evolved adaptive mutations causes a rugged fitness landscape.;citation_pages=e1002056-e1002056;citation_volume=7;citation_year=2011;citation_issue=4;citation_pmid=21552329;citation_doi=10.1371/journal.pgen.1002056" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=F. Lara-Rojas;citation_author=O. Sánchez;citation_author=L. Kawasaki;citation_author=J. Aguirre;citation_title=Aspergillus nidulans transcription factor AtfA interacts with the MAPK SakA to regulate general stress responses, development and spore functions.;citation_pages=436-454;citation_volume=80;citation_year=2011;citation_issue=2;citation_pmid=21320182;citation_doi=10.1111/j.1365-2958.2011.07581.x" />
<meta name="citation_reference" content="citation_journal_title=Fungal genetics and biology :  FG &amp; B;citation_journal_abbrev=Fungal Genet Biol;citation_author=E. Leiter;citation_author=A. Gonzalez;citation_author=E. Erdei;citation_author=C. Casado;citation_author=L. Kovacs;citation_title=Protein phosphatase Z modulates oxidative stress response in fungi.;citation_pages=708-716;citation_volume=49;citation_year=2012;citation_issue=9;citation_pmid=22750657;citation_doi=10.1016/j.fgb.2012.06.010" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The sequence alignment/map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=A. Long;citation_author=G. Liti;citation_author=A. Luptak;citation_author=O. Tenaillon;citation_title=Elucidating the molecular architecture of adaptation via evolve and resequence experiments.;citation_pages=567-582;citation_volume=16;citation_year=2015;citation_pmid=26347030;citation_doi=10.1038/nrg3937" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. Lynch;citation_author=W. Sung;citation_author=K. Morris;citation_author=N. Coffey;citation_author=C. R. Landry;citation_title=A genome-wide view of the spectrum of spontaneous mutations in yeast.;citation_volume=105;citation_year=2008;citation_pmid=18583475;citation_doi=10.1073/pnas.0803466105" />
<meta name="citation_reference" content="citation_journal_title=Eukaryotic Cell;citation_journal_abbrev=Eukaryot Cell;citation_author=I. Malavazi;citation_author=M. Savoldi;citation_author=S. M. Zingaretti Di Mauro;citation_author=C. F. Martins Menck;citation_author=S. D. Harris;citation_title=Transcriptome Analysis of Aspergillus nidulans Exposed to Camptothecin-Induced DNA Damage;citation_pages=1688-1704;citation_volume=5;citation_year=2006;citation_issue=10;citation_pmid=17030995;citation_doi=10.1128/EC.00167-06" />
<meta name="citation_reference" content="citation_journal_title=Fungal Genet. Biol.;citation_author=M. Miskei;citation_author=Z. Karanyi;citation_author=I. Pocsi;citation_title=Annotation of stress-response proteins in the aspergilli.;citation_pages=S105-S120;citation_volume=46;citation_year=2009;citation_pmid=18703157;citation_doi=10.1016/j.fgb.2008.07.013" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=K. T. Nishant;citation_author=W. Wei;citation_author=E. Mancera;citation_author=J. L. Argueso;citation_author=A. Schlattl;citation_title=The baker’s yeast diploid genome is remarkably stable in vegetative growth and meiosis.;citation_pages=e1001109;citation_volume=6;citation_year=2010;citation_pmid=20838597;citation_doi=10.1371/journal.pgen.1001109" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=H. A. Orr;citation_title=The population genetics of adaptation: the distribution of factors fixed during adaptive evolution.;citation_pages=935-949;citation_volume=52;citation_year=1998;citation_doi=10.2307/2411226" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=H. A. Orr;citation_title=The genetic theory of adaptation: a brief history.;citation_pages=119-127;citation_volume=6;citation_year=2005;citation_issue=2;citation_pmid=15716908;citation_doi=10.1038/nrg1523" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S.-C. Park;citation_author=J. Krug;citation_title=Clonal interference in large populations.;citation_volume=104;citation_year=2007;citation_pmid=17984061;citation_doi=10.1073/pnas.0705778104" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=M. M. Riehle;citation_author=A. F. Bennett;citation_author=A. D. Long;citation_title=Genetic architecture of thermal adaptation in Escherichia coli;citation_pages=525-530;citation_volume=98;citation_year=2001;citation_issue=2;citation_pmid=11149947;citation_doi=10.1073/pnas.98.2.525" />
<meta name="citation_reference" content="citation_journal_title=Journal of evolutionary biology;citation_journal_abbrev=J Evol Biol;citation_author=S. E. Schoustra;citation_author=M. Slakhorst;citation_author=A. J. M. Debets;citation_author=R. F. Hoekstra;citation_title=Comparing artificial and natural selection in rate of adaptation to genetic stress in Aspergillus nidulans.;citation_pages=771-778;citation_volume=18;citation_year=2005;citation_issue=4;citation_pmid=16033548;citation_doi=10.1111/j.1420-9101.2005.00934.x" />
<meta name="citation_reference" content="citation_journal_title=Journal of evolutionary biology;citation_journal_abbrev=J Evol Biol;citation_author=S. E. Schoustra;citation_author=A. J. M. Debets;citation_author=M. Slakhorst;citation_author=R. F. Hoekstra;citation_title=Reducing the cost of resistance; experimental evolution in the filamentous fungus Aspergillus nidulans.;citation_pages=1115-1127;citation_volume=19;citation_year=2006;citation_issue=4;citation_pmid=16780512;citation_doi=10.1111/j.1420-9101.2006.01102.x" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=S. E. Schoustra;citation_author=T. Bataillon;citation_author=D. R. Gifford;citation_author=R. Kassen;citation_title=The properties of adaptive walks in evolving populations of fungus.;citation_pages=e1000250-e1000250;citation_volume=7;citation_year=2009;citation_issue=11;citation_pmid=19956798;citation_doi=10.1371/journal.pbio.1000250" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=S. E. Schoustra;citation_author=D. Punzalan;citation_author=R. Dali;citation_author=H. D. Rundle;citation_author=R. Kassen;citation_title=Multivariate phenotypic divergence due to the fixation of beneficial mutations in experimentally evolved lineages of a filamentous fungus.;citation_pages=e50305-e50305;citation_volume=7;citation_year=2012;citation_issue=11;citation_pmid=23185601;citation_doi=10.1371/journal.pone.0050305" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=B. Szamecz;citation_author=G. Boross;citation_author=D. Kalapis;citation_author=K. Kovacs;citation_author=G. Fekete;citation_title=The genomic landscape of compensatory evolution.;citation_pages=e1001935;citation_volume=12;citation_year=2014;citation_pmid=25157590;citation_doi=10.1371/journal.pbio.1001935" />
<meta name="citation_reference" content="citation_journal_title=Eukaryot. Cell;citation_author=I. Vargas-Perez;citation_author=O. Sanchez;citation_author=L. Kawasaki;citation_author=D. Georgellis;citation_author=J. Aguirre;citation_title=Response regulators SrrA and SskA are central components of a phosphorelay system involved in stress signal transduction and asexual sporulation in Aspergillus nidulans.;citation_volume=6;citation_year=2007;citation_pmid=17630329;citation_doi=10.1128/EC.00085-07" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=H. A. Wichman;citation_author=M. R. Badgett;citation_author=L. A. Scott;citation_author=C. M. Boulianne;citation_author=J. J. Bull;citation_title=Different Trajectories of Parallel Evolution During Viral Adaptation;citation_pages=422-424;citation_volume=285;citation_year=1999;citation_issue=5426;citation_pmid=10411508;citation_doi=10.1126/science.285.5426.422" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=K. Ye;citation_author=M. H. Schulz;citation_author=Q. Long;citation_author=R. Apweiler;citation_author=Z. Ning;citation_title=Pindel: a pattern growth approach to detect break points of large deletions and medium sized insertions from paired-end short reads.;citation_volume=25;citation_year=2009;citation_pmid=19561018;citation_doi=10.1093/bioinformatics/btp394" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genomics of Compensatory Adaptation in Experimental Populations of Aspergillus nidulans" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2.cover-source.jpg" />
<meta name="twitter:description" content="Knowledge of the number and nature of genetic changes responsible for adaptation is essential for understanding and predicting evolutionary trajectories. Here, we study the genomic basis of compensatory adaptation to the fitness cost of fungicide resistance in experimentally evolved strains of the filamentous fungus Aspergillus nidulans . The original selection experiment tracked the fitness recovery of lines founded by an ancestral strain that was resistant to fludioxonil, but paid a fitness cost in the absence of the fungicide. We obtained whole-genome sequence data for the ancestral A. nidulans strain and eight experimentally evolved strains. We find that fludioxonil resistance in the ancestor was likely conferred by a mutation in histidine kinase nikA , part of the two-component signal transduction system of the high-osmolarity glycerol (HOG) stress response pathway. To compensate for the pleiotropic negative effects of the resistance mutation, the subsequent fitness gains observed in the evolved lines were likely caused by secondary modification of HOG pathway activity. Candidate genes for the compensatory fitness increases were significantly overrepresented by stress response functions, and some were specifically associated with the HOG pathway itself. Parallel evolution at the gene level was rare among evolved lines. There was a positive relationship between the predicted number of adaptive steps, estimated from fitness data, and the number of genomic mutations, determined by whole-genome sequencing. However, the number of genomic mutations was, on average, 8.45 times greater than the number of adaptive steps inferred from fitness data. This research expands our understanding of the genetic basis of adaptation in multicellular eukaryotes and lays out a framework for future work on the genomics of compensatory adaptation in A. nidulans ." />
<meta name="og-title" property="og:title" content="Genomics of Compensatory Adaptation in Experimental Populations of Aspergillus nidulans" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/2/427" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Knowledge of the number and nature of genetic changes responsible for adaptation is essential for understanding and predicting evolutionary trajectories. Here, we study the genomic basis of compensatory adaptation to the fitness cost of fungicide resistance in experimentally evolved strains of the filamentous fungus Aspergillus nidulans . The original selection experiment tracked the fitness recovery of lines founded by an ancestral strain that was resistant to fludioxonil, but paid a fitness cost in the absence of the fungicide. We obtained whole-genome sequence data for the ancestral A. nidulans strain and eight experimentally evolved strains. We find that fludioxonil resistance in the ancestor was likely conferred by a mutation in histidine kinase nikA , part of the two-component signal transduction system of the high-osmolarity glycerol (HOG) stress response pathway. To compensate for the pleiotropic negative effects of the resistance mutation, the subsequent fitness gains observed in the evolved lines were likely caused by secondary modification of HOG pathway activity. Candidate genes for the compensatory fitness increases were significantly overrepresented by stress response functions, and some were specifically associated with the HOG pathway itself. Parallel evolution at the gene level was rare among evolved lines. There was a positive relationship between the predicted number of adaptive steps, estimated from fitness data, and the number of genomic mutations, determined by whole-genome sequencing. However, the number of genomic mutations was, on average, 8.45 times greater than the number of adaptive steps inferred from fitness data. This research expands our understanding of the genetic basis of adaptation in multicellular eukaryotes and lays out a framework for future work on the genomics of compensatory adaptation in A. nidulans ." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2.cover-source.jpg" />
    <title>Genomics of Compensatory Adaptation in Experimental Populations of Aspergillus nidulans | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__DbqSGiVjr90FgBIuPlGEAiimgNNAH8S6qbiYFsi8JyQ__E2bjoC6LAI1j30ywBL_JgYRwIhvNl2n0yRyt_snD9IM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"nqt2YwN3Tjna0wQI7cjjrfv9xvcZpAgWviCQRxPTpg8","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460558","apath":"\/ggg\/7\/2\/427.atom","pisa":"ggg;7\/2\/427","ac":{"\/ggg\/7\/2\/427.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/2\/427.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/2\/427"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/2\/427"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/2\/427":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022fO1ATEMif7fAmIrgBJgElxmgvpIeE4D+YpGEXy\\\\\\\/EbjjdkqMsiv7ywPRGfIAH1yWc1BRrNggeNaXbIkiZQLbsGplHUhoqTIrq3Jpeh9oDqZKPCa4EbkzUM7FVX77yQRlEzMdW9xpiBWDO6j08XnVC8XOftKxWZFFckSVhYpGk\\\\\\\/JSXP2H7BhZfz0lExtpMaefG\\\\\\\/SFCj9xveGd2WVfigjJS1CHIPpsyxPS8mDdqpGVyLDadJp0VY9v\\\\\\\/HLYrwUxQOu+uEobJdswr7zRnfc9DVGY3QDdzOBGVZ5YnTBxsWLmU69HVUsReD++J4UQ8uQ3sXJA\\\\\\\/VhuX90uteGrHR7Ds+qjTrzulMJQ6ypt91sFrwetTy5brDTWbzsl1aviHC6+yXoGU+opFZcLhbHJYC0P11f0vsfPzSdRJbn1QLKM7GAy8n0spE0lRDcJiWSeE4rdwiP6SKPg1o22t\\\\\\\/40AKyFbtCDoc6nvuYvSgDTYvxB2ymQzM8COzvdK962L8aOzEmw7pP3CesMWUpEUp\\\\\\\/Oe\\\\\\\/zuCpR301II7gvEf528iKaYz87KTqU6WPOiwxP780zOU0A3fP0S+Ce2JAtRj0Kzn9DwpW7Ub5iGHha6U25OKC3D981iMArwGk1rooqgHH48y0vwGMT3lnthOBS\\\\\\\/ucol5GFOh3KwCsIdJBvqHZM0B+Px5DdtL4Nfgjd0MiQTzFCM2FJ8OFaGsAe4CgJe\\\\\\\/x9OxnRJwv2vJeEqc9sbvQHeVDEQVQSD5YlClTttA7OoybsTF\\\\\\\/8AMT5WyJkxn8Z4OF\\\\\\\/ZF++dstu05UfW3SJRZQlGA8p36QzOWhEalPKXt6M+jhpEIURSm3j5Fa78V7CA\\\\\\\/8bwtrwwp4yOi598UR+dKHyM5FQPAPzq5iJMv\\\\\\\/MYVYmKUrHDQ\\\\\\\/vZdbeSXtDaXi+OF0we\\\\\\\/8VTL39EGdouQ34zKA7m6fuQmdnBJFAa6htZK4dSuZNL4jBvLOT7tx5qj+osAw84c1gQMT09RN9zqgDDA2FInNl9bzBglVyWPT\\\\\\\/RPmpaoUcPv\\\\\\\/768si4ze7ghCgYGlJ1M0XCRBkeJozaQcvF9FjIyA7upfXNBal\\\\\\\/4f\\\\\\\/njjovS4wtWqSpdUAJ2e1IfRKdSEvvj0plbOnnCRoMhk3MzFc1Qx7QGgA6X3pL8wHzl5xFkVM5IKghMcrfBjtyyVRGf2k7maapS8joJZAmBgmRxhGWAHFkcVamWXYg4+aK9mvz5yrXy342AG+z8z6DJ9OVnJ2hoC57qF2mik+ZNpVBWNyAWj2FKKRKun1PBgCvHMlSflmaDXCLKXpFV\\\u0022,\\\u0022iv\\\u0022:\\\u00225d1B6IueD5yA4bUwv3Lf1g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a379109b5bfeb4beee660b17b8496911f0716f7bb0736f5456ae96b8b6dba1b0\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022vVzak+orxikbvD0ziL8Ci9L1tI5JQ+WsBXIBHx2Yp8LDSo5hlq5u1QAiBTLW482zdQYWorf425+wCvlGvhgmKEDeUj1XGo\\\\\\\/f3L+arwP1Xtj8ZYGeLNiUUOwup4Pk2SwjFEdvxY1DWlSSbOzbE0rJd8kNtW9WiV4u1AeRHXWB7AB\\\\\\\/yy3Zk0Xe7LUsUDtyRzTxx2ZedCalv2gj\\\\\\\/TXXV4pGvb1zqcurobJYJzHvKgSz7u+904qhUS6uW05i1\\\\\\\/NWx4aSD\\\\\\\/B7MOiixfl7ExHTf1nPFisdya7Y0Qpda7APJGS+z3YJbGpdTXjuD46ArOFknG76uURI453wIrLjT9U2Z5gb+qFOrqQ8A3zSFTAbh2gNAHSlqO39oXRbzUevamIKlJ78+2q8gkOer+fWu9Y2vc\\\\\\\/0V8665NEK\\\\\\\/xJ3XbEaCvONpV0rcPV4CDy8L8FWank1tm8ZYBJOGaFT+g9r1spQSWScIrwrKiN0Wi0\\\\\\\/hN7FJ0P0VilUELZ6xLm\\\\\\\/b6IdsrxnuBD4ZLY+0m6qgbv0OXstKn14yIbi+up5EZKCCQ63CTIdNRjk+ZPaU3rmCm7QT79Aa24QxbSa4ygzd5y5uUBhljOexzRD8ozy\\\\\\\/sibDWs1AAQaDOvZcA\\\\\\\/YcL1WAhwUHbn5tAxY60I1ZP0L7Jp3fVe3DUs1nVlo5qAzxZ\\\\\\\/0rWYH7DSi0A7op9QECgNroRKeZO8zbReXfiEea6VSvnhAcfnAZEwLNoqdWOJJQkDbUXt1ERi3zbdYSvHqsY2WS5mQqKsUhXOTX69+DQkZ6DtOkRO6+gZgMu4moUkOm7EfqeWKyTaCkLZfZZXu02fkJUGJTa4+NWriImov4n2vu5f79ZhW5cx1yiD7zSuGdrloI+Ht0StY+btKiZujioJ+zT5K34DWFXUnwypNQ++yDgSxi6pJ8QQNnCW4wTdKknAs\\\\\\\/RIszH2X9mcemZVrLRrgISrhyKj8zH88x5ZzPRfn\\\\\\\/tfW5qmh2ZcxJ4nhiUJxui6nXGPPJZhhQefEUaJ+LlaziMY9BlCSNvv5WXxjLa1AcJAx7BcalqHreJ8HgKmn91nrIFp0Mhx2ZmzuWLzMsF33C7XdbPWa7ifUc9dBQRd6bwaRkrUkbQ9UaVJG\\\\\\\/6VxIMP74mpXBSCBLAYA0RLgpisEGRWfCm3aeGZznpGhQx6KLmAMGd8tdncke\\\\\\\/BZDGyLbZ+wFfNnjsmqXukKL9mw47GZxkpKebcBUjh0LuAgqQq6ojCH2Bj6epCo2UeXhrzSFr0yasGPGIBr8PbXB0PXDCvohxuoc2LXCOY28k4\\\\\\\/yfXQ3Oj00Asd5iAaG8JbjUG13UI5SQ0IvYvxzSXzxSiJc0O7YkhOkARaFQDUYu1e0JkHHsSsceIAGNa6iReLpivk8NFcfl9fD82HZG2NA\\\\\\\/eW\\\\\\\/17rwiQtkVqRkplgNs8503IoC6Tu+vVLZwnLmhKLxVLrCKBdv+fuK6qAuvbZLvztEpblZ+yQlH34U3ZH1fOMWJ2O4qTW0hHOUSgQTcfwYXjx8nzb88UrXwp28TJ7qruqMX4NTQyj7xyEHDssIz+qkOm5nwSQSd9TMYcdqVNmwdIlea4aHGwW7VJdd3WaLNFRbpPaHN\\\\\\\/nxhaWA1\\\\\\\/ty8V5\\\\\\\/ZmrIvqNB\\\\\\\/bgobdqYTtNwcrKEgOhgXnYRascjZ1r+hydtdDV4be7h6SqMVMvMOM3RnVZJc3lNst8kYAoN9oWs30H2mG9xmp6HZNNDpcRJbUCsk0YFVmvHi5EFYxJ51YhokL+A9cjvGqRZBSMb8lmYgkxkpbJlzRfBE7zJK83vht2meV6zaXjlhL9\\\\\\\/7JDHkGFlVqytESDdBPm7X34SdGwkj7WII\\\\\\\/XYZtkVls3NetjSxAj\\\\\\\/ZEj2+Vemueu7J\\\\\\\/I5EAwuITstUPKza+djlrqFUdtkcDJsYT6Yf7HQsUEEOVERbmo9s8n0kpxqcAot9E0ieP7t2a3xGXtwTuh3T\\\\\\\/rtZNsm7BxcNn9ZqgprzVLrpdumcQ11lN2eIsj6ztUFrFm7zm\\\\\\\/N+vW7MQU869gKYsS8I12ohG30dMy0D7nqChInfcbKZbUNXTzIgViwRlU9XcEBA5ldtzGwOgVBtVFXKt1FAPxVXm1U4GzB9CNOwGW3hd1VwLhvv0zcz+b3eop4ZXpQp0F6a7zL+c5HeJf6nfpnOZtJRNNYEa1u8cxTmxg9yIciILueaUDwxQMj4W7nKibzss8X51B6ZdE5le0s3Kq8QKA\\\\\\\/2B6NoKOIjIOAp+e0g5QFOX\\\\\\\/yxl2bY2scDqhew2NmXD6c95oSwwkJmTe3CxSPxx07XbVF032N8B+YmTLWFg6UuB4Umjp5evskGKw5x2YRZo69kM7D6yKQXqfRpZGkl870PpsQ1P8c1vCvK7sC9PsRMKwvEnSL6hSOlW\\\\\\\/54irUf0qKCfvnVGFgU+o62atQ5knbrKWx23HHNPlMOROgzCHNShwICdojvrTaCw1UqeZhYUs0uTwgzQ9ERZe5fIe\\\\\\\/Nzvvz0mY67sgb0roGcLH00\\\\\\\/ePi5+HRy\\\\\\\/itWbFBs14CrGoZ0kQxuxSGor7O+73h+wAzaEZQLdLC3gjZq4n0q\\\\\\\/eEmZq80Nbde+eB2fKBL2p1MgwWrG6h4L9S4OrPE0NfIecuSKPB1rdP9sIniAMJsTG1d05\\\\\\\/CO35FryZQGyo0rXPRoud5hfvdSRX6JMGyGBsmh0g+zaGsvdzo90FY+gKEdxriLohtSjjDb4SejnaIMEaQ16BQfgFK95JfnmWLhnutyczCIQdIj+pweTXj43WE+zWVsSIjUFamsW9FbnHr0FqXQwvIQuj6kzbn7yS8Vmyb\\\\\\\/pMgCqu6QR7LhU\\\\\\\/CF2lZLZCpQNwQ\\\\\\\/mJ6YXVFyDvYDZSer7ZyshRSAst4kmhUOo8MJ+4LHwM2syZ3QYQ9CpTzLmA5\\\\\\\/\\\\\\\/uxPGHMOuDUG76uPuj+E8bLnVNoJMabk8nXUBb5KZCsNGPjTRhcJ2wYZziblFzhPdXWepEDn2xRYxue3NrAfqMAzGc3nvFUuPUUq\\\\\\\/j5dvKiepJQLbgQRd3b743HXXfbTIsAr1tmhp4mzq1IRzVBzjHbDKzfaQ6DAK0UkJdl0FtBS2hSMMQTrJcfV+TDBRdc5xbO3tYnFENQ7ZNA\\\\\\\/jvWZBP2Ods8xwcUkEMzk\\\\\\\/nGB0ZlTE0NksrVu\\\\\\\/msjF59PMTUouyV8L78cwLUN3IlV+a6z1RdiWu6F1btzpLe+kaqmsItQHfzGHR1j4oqwC62nEBW51r8intVOGYeGWhUEKPyyK6ly4IPDblA1OUzHERwpoSi1sUxgmDrTMqsPiV4+oTZrveTHcdCFps8R5h+3T+umM8\\\\\\\/cOMVRDt+mMniTb2fkChuDyGA2A8tw7L64NgNVo3mqXo2xnRHq+fm2cx6GEx6xuXpF3QTxPg3y7cLr2TywMsSN68VxAiCvJ6rro2MFTb0aPVjdavPU\\\\\\\/FiQwwHWJCV5TpftER9S1ycSO6XR3WSkOKGOl\\\\\\\/8FxvZZ1Vfz0msSCRJy0U8378Lj6KOK8bshWZhEzbLNqkIsimTK+93xQNCN5lwvpULVkGU8hwilFZAQGnQGWBCo0yn1af8dQ2nJvRmQbrL6XDkt0YGJax7wvNmmUSj10bAGPS2bjgV7MWIvGltB8apWmxRxJtwjauFPB2vNXNRUEH0C5h4UIpRi8dT5twCnhe58HXLzrKP6mH1nSrbcjanKhElBpRCtOAvCfIw1uxX\\\\\\\/ZoHCZFczrcZoi+jA1yr\\\\\\\/2E451WyKyfM4Q4um+RhzSlC38Cbor9Jj3pvIcW4P1mlDMYlWO2xMLDJNTx7bbKtBcNTiR+3Md4lYDIJ8VAjVM9kgcEryJmoIGgkE0qLbhOdeFH8PS3SgKqnusvE8zChP\\\\\\\/MrRMsDFh33W1siTX58Qu49ufASZy5AhFMyFfeMtq10lsKmfJGu8nde1Xj3KVgS1tpDyi0qI9dF\\\\\\\/sm3TWnC8vdC2egL1idWVGQsstgRXxUP4RVR\\\\\\\/2Uxm7F+53h9h5gmuA8cvChe5Fwy74K5zeWbuuTcdyplJL8HWdPvcFDtB1HAVQVVPytLQkYl\\\\\\\/8A4m6My+oSX30iObG4bnsJc4KtaMBRMAahxgE9bZoVm2h8zRwLGK3XHvxqBkt84ArRfu7mUmTRlbPwQE1kjevVrOZPycIbC7MSHZnaKPsw7wJmjD6gLZZPrU02P4p6Roq86ZCuslfdbbT9Bvn9jA338yU0MdVvmWlnP6nclhZsFI6vH2Oe8pnUPSsXTs7wVvs5VeoK0fJ7uLWOkCrmnilSCDPizyxikFReNgmer8CPRx6Lz4nUZcjFaiJTh5b2QVyx7AILQHAvUOWmQyQVSt\\\\\\\/hLB47IQhpeOsm8iR+AD\\\\\\\/+2MI76RWfh+QtOElnDB83lwbY54HunrsDq7CMO9ClosdBY4o46iilGNjzr8Zt2gWC\\\\\\\/OPngBZqYsVYYMppJ3NWKET1RRevlYFPv0pq3Woa3g01QRnuISD2CEtLFa4c2ZQNipqPA2x23P65XoiSLj+3n4xB1vlw4+heYiuGZoubf1G3eOmD+pfNvKpbP8WLNJVbzlqbJW6Ro1fj+X\\\\\\\/\\\\\\\/m\\\\\\\/fAwJTjXwFg9nzEhRfQk419iWh6thAybGj5JapVg5jTnuYy5hrRNv1UrqhiZvETCnnluzoQJZZeIKVS44lO0p2lnzeghgghtN6++5N1YsWsXEyw2YVFXy7\\\\\\\/c+e8n94WnbKC5+QC0zBJ3+bXyIRuY7aa9vV8yIOdLl0Pu7DPHlRCcKL\\\\\\\/N8f8fcRbQHkZlRQ3DWclo9INdzo4\\\\\\\/lSmwPNrat+B7sv6CjO+KvxYwyDd0kyeh23F65uVuQc4Cxrxq\\\\\\\/8K2uQs1JMl88ciJB1OgHVb+ytxdUyxPhyTCmBb\\\\\\\/JQcsJrER6I2jTykqx4egxzFf+CQjU9oqoU7RElkcDiUVVfFiG22PuZF4a+WuPUDxAv00xCQ6DE7SGXARXiiNNFgC\\\\\\\/WxUluut2z0LhL\\\\\\\/lwhNt0EHXVjj0M3FVfteR5Rn5fnPZfgYGMKkX1VSZ+1NueCMHFYyDtsNm4Q0aT8XcI8KMc6vV1TjQ3N2wy\\\\\\\/XZlnW344EC\\\\\\\/Wmt0BLVaxetPjQgkchersEIhhwzvN7DCgVZzk\\\\\\\/fXLlH1+Hc\\\\\\\/B50SFqKbn0p5B4W4dWaLK3NGf0VQdqdlrf6Ms8DarDnzNEDh0EmiqL9Mk79qs3CmugVmG3fJV7bXUe0aQEHwtYwArCv8egvA4F1qE9bUDcKNHEAxAN6i5HNXpEsNoBIAigb2uuGSy+OwotxxEsLaIxaTAx6YI51dB+DCnGJ4lV154t71GFABZDjwt4FekkW9L7fbr3kjoHS1gw8ynlQsIt1Q5rgfNmL5zxyXJ1BSf8DIjG5cfAGdRzYPV3noOcLZO\\\\\\\/qevLPfhskG0mX+YfCJFEFdnlAlJLC9aRCJrKpFOiemFlbn8ajSOmRTavUhVqCj3uTFSO35OYjmOZii\\\\\\\/0jZcc26YGtYk3O0ySNFjjqouVhReHtRPDQgOerooe\\\\\\\/5aRsCm2l80NNc11rgqpvQu5NRg01jzfc7PxxTXR\\\\\\\/M2ZhmeFTbt5cdX6rybxcmayqBKN\\\\\\\/LJI3+Dw\\\\\\\/je8mQRZFiJJ35u\\\\\\\/IWkPZgmFeDMId+eiYtupZvhreOTg8vn01hU4g5DsFq00wZc\\\\\\\/RdFH6mQ955KKhdNAgf3CS1f\\\\\\\/BZhZ\\\\\\\/WK4+LLw6rZUxnKwTe5UrGRFYPwRa1p7a7qi0Oo1+jZgIlYAVFzcFLx5qKqYPYKrvH2B2H5eLb6If6yT\\\\\\\/EqsHZ\\\\\\\/4zk2iPbmr36CC0SJa\\\\\\\/Mtn20pE+VbOzChAfI6G3w0A2iVUKrxrmhngw8KYkODw82n9iCqt1WVLe0ZBY1Wq9M6e9n8kC0oIoI\\\\\\\/ntbxMeFGzQ+JiwY3PTr5S7ZSKTmTcpt1J9spBHolF3Z99N0T86\\\\\\\/u5I1EHA3Su1Q1XzewpBMG2ARfOJadoO0lar8FQOwKrjQxNdt6c4f+WlQ6Z\\\\\\\/QbziASDVdH5XRfwcsHkwsipx0l5tCQRqaPA0RtdukU6sALz1H0sASjZWQHHYrYrTpe8v51EL32q1jjYFqmTHYFXznWFp2PwxpFdlJD339m7o5ydCGcuZzwsurc0XinUS+Lv7wn4VMCwj9ugkmX+2DdttiFvSbJa6X4\\\\\\\/\\\\\\\/TeZv9r3sDYLmMqYTDWJMMikgPKqGQBcvqse88vaowEwnFfCcEcOr+VYrYe88v3Mi7ZDvOO\\\\\\\/eyR5ucJf+MwaUhpbhKM6pVM\\\\\\\/jZGjb+ZmFLqLKw7E1ucTUSf1bky3\\\\\\\/LMo19O1h2dqDOifbTq+eQ4dYpPzunNkNDeA+VZUJ0dCxRF\\\\\\\/67KXVaUnjvFD+L9cSFznVxSJGeQbcHhdY8T49pt\\\\\\\/3hALY4Opb6ElUuS\\\\\\\/AWIlrlhAuFTYDcpweBTqQVlF2D4RYewEVB54bwjQSmAfztdyTvYS805\\\\\\\/J09DmOWhjC1qqqOSZ6uI7W\\\\\\\/jY\\\\\\\/gOi2TeMNo6s7+cNLKZOO0j0tXrZTvtoe45tf9x9pzxgfGvs1hqtpn9pIv5S6QmzGhwsKPvEPDWah+UX6fAWH0yWbrIwmHLqKvxAAB7+zkFAHxnBcy2dtupm230ZMQbkGE9+qtaeS5DSe+c28Kzveb2ldhAZmKxkmd9zrJLj0RbYSOOcwICyH4bzDWqMKR9MRe4cyCxbvRHrqHxTHUetN+WLpTN\\\\\\\/4pigZx\\\\\\\/TicryBTpzHuHHnKRp2MdaituiXxRG\\\\\\\/G\\\\\\\/dBU0U9o2tijiWBKzeGjvk3O5oe4p3DIAEZswqBfA5Uh32vuQoimUbVPTEBVsvdxt7MbC7wmjF3k9DOyJepw\\\\\\\/lPKJpG1ZlTnHPaSLGeOn9krqnhUaZEnkz7fpgrAInq9r+XTNeA0Qx63KAF1AHMmu+RhUqvvat1tGVZTEyqy4e8cDBjME+fmAUXseE\\\\\\\/u3X+qGhY8J2mvKndk1KHRbC4kMKiCdtPgYY7Ninq8CjqN3y0QwxjVvMv3f6KADZIEN1di7eOxfi6oGovZmbGB5TQt8f2ovsaCVQkQtbyB4HLyBBsGT58pHLjxu45gpwmPcKMU3fB+QYOnZv+Rcjlvn97CcuJkhUJ5km\\\\\\\/eL2QM6YlQKbeXtqxxTKDV+o6EPTcqW6zg1zSiR930yoHXcAQk1WWPKEtRrI2SCih0fs21oHQf1Mes91i4JhKgTYhL\\\\\\\/ZohENbHRSaAY2xZawfAwhronPv9ND\\\\\\\/1\\\\\\\/R0Ws+lIuPCIHxLTptlWiQDorhfVv2oc6C2OnrY9QxOx0HVFfGlf\\\\\\\/0QLi0nyRHR9sHEQAcsmWgiobeTv4RuvNsKKi1FGnnzeONtpSaSiQiiDaq3L1PvoctYHS49BZik6CCSvYQiG34oIItQLX1yEN+Y9b+cUi29V5LsSP3LpSQXeSRMn12W4Jf63UOIokVKUMsnVbAq\\\\\\\/pVirN37Iqmndiq\\\\\\\/xDstWeqeF+eM6Z4spjWrejPPNM58tJwJNpOuhEY0rHMzmi2Y05EE9nwm2X5ETE\\\\\\\/oejdpZAYSW6u1HZEqaBKgUJtCYxnGoCI9VxDqnF\\\\\\\/hWNvF4CCCXlTsdjle3y7mnrSO5hhK4wCHon5ScQIcam399jl2Oc5t36Noq8j\\\\\\\/DI\\\\\\\/6zgPQLmek1dlUm22+eSJrJ4jPKPMlUAXp92bQ0\\\\\\\/+SyILwWOVqpOTBMKYqnLAaeM7cP5CF5k4qRXEK3peqDZlSFs5Ux33ylmWrluijQi9D5V4doOOSC6Qy18tOhb0tLLr7txxhQMK43UeIMAkI+NcCiCfQkbg4CB2pGDelkm7Hq6kWXUXFjdr6u7HwO8lF1fnfDcdF3pwrDCO0cisUwQlotxZf7S3nKvS1a80MEySTwDIulgy1W9pfa\\\\\\\/js9un1mrl4DYquPduAoe0\\\\\\\/Usk1zpCgCuC7M4ZpCzBAMPp0t8YQ5rua7EEQV3SPJ5Hr0f9zomuP+DRjvyn2pXTRkzf7OsmaZi2EQCsk6KVA+4LRTJi4OVXTCsHZzoyzncCYpYTAIhOUoAczdsHwa2KVl5jyb6N2o65TuFibdXi323f\\\\\\\/CaI3V2YkReQXZ2mgBHhFqQxguwlxY8SWRBD435t96QlxJr3HmR01Mha6hFN\\\\\\\/hSlwFXFxmea1mA+60SFKXVEDSf8NP8b5\\\\\\\/tBg\\\\\\\/yGYJdggmP8cy3zEoUq25eBSxFrbwx5j4r9kqp1QeG2XgP2lhPWlc6taUxik5wXZPuP\\\\\\\/D76oA0CVoyfCzJfTYdzT3YeZvn+gGQsQPsOiLJb6QpANDjP7kctw7LjWe2HgfDi1IBOQT3tq8l6L\\\\\\\/0pTyXQuLEx+vSePR9ZszRwMtc3i4DCeTja3ffii3lDqkij\\\\\\\/MF+TWB9ijl6S89hVinmL\\\\\\\/w8Ja+aQhE75GAH0xyYazDmhTlJYiHkF0x6LVsa1MBR9ZbmrettSi\\\\\\\/dxmodUotC\\\\\\\/eFLU9mTRu1fyTsY4s648yB3rWi4xesPXpnRxwBW19kFW0mCrNa+b1VznGNrQLDydPdEU9FxLrgCamR+zfdpmuFgOSU9GEm9JxNaPTqX7gDGz82DN462yzE7Vre1yFP3eyoKlX9nmB2XRAJpiXclwa4FyjV7R8S5AGe1aWg2ofxM1O2Hx8r7kuY++IM2TyjxySqxJZfM7FtYKESMKZdBKosGYDjjDt\\\\\\\/9uPkrX3g3JtU\\\\\\\/J1HQ0rRU4jqMyJJWP5bLlQzzObF6QdoF7\\\\\\\/rCZ82+gDQAP2Mz1PM5ppN3ETYaYp1hJ2IUuXY95Cv63smom5ReOfCpnvTctt54\\\\\\\/1R\\\\\\\/l3j5247L7i6hyx3eEPgdvTlviAZFdK3Y5nXRr9hdz1R5cXRnIjzGl2Wna8fWsHzzUGtq2MULlpOqcXeUjJE3MKN43GqLpkcmnDsqmmbkwEc6h6TfPQZmRgpBMHLi63ZN81EKiPyWcwcbpozu7x2G7c3Go\\\\\\\/0K4tMJiadpMJ4l0ORV1\\\\\\\/SqqELqFt5kVuRDaKhJ62bVgQvvmtx+3hGTw6SjiSiMLojzAZBsgX7SLCqAV8YG3p9dfsUUlTld+jlwYhFG5XJdcPYKVguidh\\\\\\\/o5gXu9osRJjD3GcbICQwWjCRwX04ui3PpLMZk4EQkEPgPMGkj0\\\\\\\/1v5FAvGnGQsAj7mpBu8zOzjK4hIKlFF+FhKs9miovUNLaXb3l4KKPIzkbDfz1IEvkiV2O\\\\\\\/MNuvaOyvsKtcCrNOh5tmy4Keps+tqaF0p6ay2sPwUVGJsTBRFwDi8zlXKk1TZrac+Gz2y14E1E3bxOcaYBSHevrerzMtH8FKkP\\\\\\\/EMNZlomzexNDJWxpfbR0vEhDRcqXfLgbMsxIq1c3u\\\\\\\/1cKHWvSWSE+6yAM5Qnavtx0sK8xRlNxYd5FoClaDX4Ic1Mjf4gpYbNvadYaf5pEn9xwN8cu5N+SHxon0fSxe9IGZoHS8eBEcPVwhMZ9rAtvZlFuaYtUE9jBYUZBEgRXiCcSwpJQofGh1lyedJmQAbBzHQun\\\\\\\/X\\\\\\\/EtbN\\\\\\\/uEu5LvQQmfLvYZ+sCM0GUSvSHs0Glsr9za14Lpc8b5izacaeymXeXkWEnpwg7BD6NbAb+CC8hQ\\\\\\\/i8uIWwf5bWadhWYhQ39fU5RBhTzourzKxMgGYIoj9XEbU6lGKLkL\\\\\\\/owKypatoLENNFaTkMSS1gXnLp5816+hyicMkssRrMbuVhBryD0KwqU8fAitwikRL1WmrtVKxa2Vd1VMvcSIrZpezUtOZaLMH\\\\\\\/Gx0xoUPeYC1bapyHatHDLHXbRlDRszuiAS0xxZLWGVQRYSNI88n0xRKE+P+28EJFaUKzsCBeLa+D0UAdm99HsWVD1VC4xvJgpFsnz4yCT32UaBnzDtoqU2N5Mg7vkF3pp0Jz\\\\\\\/vRJd\\\\\\\/q2F5W0rit4pe\\\\\\\/de7vPPvSz3zM3zSQyEjvF1jEZxwJv3RuDrU\\\\\\\/uP\\\\\\\/ocJ5a2B4Gy4OwGreAbtsY\\\\\\\/xc\\\\\\\/LhxJTG1RrBWScaE9sHOC+WPcKaEHmb+R6CGIWtNFvzHGeuX56MosqXdppv7u8Og9NZLELwPqQsqOnzLgZ6rQd0miggZYTXdEyvmnWqYRMhmwM9inmaB5EESygZ8RTqUiVjtUVcPvhY0HR6pa2CvOe507Vigkaqxi6Ln+FPgatLGGJmbSneKXniX0zfhiFIegcH3WcMr2h160LqtOzC05Gj5X6YAkYBcUTMzBDKbV6eeK2eENnCpfpoURpoMy+VU\\\\\\\/OgQGqg6v3BKhXYZ7D4TNEqPve0e+V6\\\\\\\/OLbwNGmNqoiicKrelTv2CY9bUK+PMGnWlFXMm4kq7aXRclPGUb9el4V\\\\\\\/EXRGrprNrK1UZTlCOSuLHhCHefNzpVlKTWUzCdAwObB+gm3s9DkZ1VGdcQbjk0k2AkcBrMp38V95hUjFfQRDxChXSLZsnkZ\\\\\\\/JPQyEMk+9KPF2Cvta3utNXKtrA0EFPbmYTC1yUWDeKyF9XWM31nsAPPv5BxG2dTQWJjvKnq31mmdOzv4R8hh9mULuTNHjFsyL6ChmfccvEeb1lwpT98PGmeE7knavaYl5GIG4nDVeb0KILoa8w2Zed+UupiQzZEZoTUxrdzf6FndUQWM4nfL9mc6T40cpdoMBIiq4nvjo6RcmoVD8pz5pWsB8QfFwgVjGZFexP3qfPggq9kdjLl+XpqbAY5TstUQaWNDcnNWEG6+FAOvLMSb73e4gvl5rbWDKWEDg8TTrJuwXa7SbzFUZsR+EJ4VNFzgBurQQxbsDZEbnAtQflFrRU2XcYn4ll7Fs2N7a8Az5coWbW5LBG6iPsvXRhbEApdL4G2Ij+dvH92inqzpPw4kThWhB5tciMP\\\\\\\/Xzd952FQt9xk6ih4RxsOS82ug8p4308bmXc6QO6hfzb6OdTV65MtmdCjUajEyRIl1PITx0GnWYkjD0G0f+urP9gGpkTtWKS6O\\\\\\\/UVnxGEMpIMS7URGFie+B9X3F5B9HSXm9+ImwpfBp0XJ7NII8u7WZp5KmJk1glX6pJ66s4obXYMYED9ZztnN9zjrd3InUtD1J4im7xDo+JCQfZF+riVqs1jgtXj95O8PfBajBhf\\\\\\\/T7Ngecj+c1xwP8KnQVyx0KoBzQ+FB271LDJea\\\\\\\/tDufyp7lKcJ0+z3yRad5WLx0Ga1QTkikFWSz8srJ2eqt2LW\\\\\\\/eTVIU0el+RFnMW8F\\\\\\\/lFIedu4cmksC6Ceyr85FPoABQX50W68ylGS4\\\\\\\/CBjfRk3Q9j0k576C5PreBeLsfKLyJfbB+4RAzgmSHnMLv4h6QicHQS5cEF5\\\\\\\/xAclZ\\\\\\\/lH97rSGuPfz0osRTgVKpLwmxfycJnKdd5ZWbclpNGWViPJfuinMwBuBtVZUydoRRByPzC9ar01YtIARW5+gXKuaJdgQp50lQv\\\\\\\/oGk0zAId1HzF49iBlxRbGTaW5Nw6EGPWI6Jyr6KdzM9fJ4M1in5LrXYLhj94WWACt0JHtX44SbpQUo1ejP2pys6ETs6Ez\\\\\\\/fKm\\\\\\\/RcVKSvCXtvZtMdbCsX\\\\\\\/n9f5vF9cyHvFxhsYlTpqYxe331yvWoJnBwDKQzRr5UiAhi+z4fO6z0obdaaFh9HWzpOuVTrTxVdbP3ZWkIzszTNp3+MRCOM7wA7YFxrk\\\\\\\/eHTdtqIia5l0YhNtv62stc7\\\\\\\/f\\\\\\\/V4kr7KWFfOnAU+hyTIXmjI0qeafzq3wjAuT8y0AlwkYscCN4kKBdGc5Hs6jsCRcA00bRUaVX\\\\\\\/+o7nBl\\\\\\\/m8XJNP3uFjPLPCBiVUyYeUFYKsbxm\\\\\\\/vwKAXp\\\\\\\/coV+GwnlYTYPvXPDFmcXtHu1Zx3LpFDvyX4RmX1bHAXtF+HIwUisHFYrgKJiFoGGiuY0hqQmQrJia5Iws8I+fFvkmS0O6HAWuUklB\\\\\\\/rb2ktuz+WMHIojOGq0FJKjFQbc09RPtB0EmSAiHYabt1xy6e2fhUtnXL+sQAejXgWQMhPLgqCcbKYb1GbAq4V1ROxafGvwvwiKzcX0V6P+cmxu+5JyZo4JFtrzVJtTZJLKejKMytFI7qcRyKofc3VIG3qbKzl4rcSz6jhi24u35ZxoskQg1uLt\\\\\\\/tmcmUd5eNmR94Fmd1qiN1RVaTm8zdg4anrbWbTym4N11LMCNKryW4mGczDKVdSf8ll7M81RWZ8kOPwBfX6mPYxsczYmA5MFyqN9QMVN+\\\\\\\/9VX69q1kPUEm1eaHitdRFdge9H1QJtG7fJht3q4l3KMNGWpQN9HnORGCLQoH6B3UjHQk9tk7TyzzSF9hvcHLKUpSvplwmBVibJ+jL0jIzRNexr86LcaP8FPwo2JXjdxxB5vtc\\\\\\\/1MYo2qsXI6R4nbTeV3uOed03xZF1Hvbvbd1+3AGfXJA2XQrTZb5WXaJbUC2PIHGbbsQke0ZwB1ffyBrzkxUhbOm2Za4XNSmC0SdHtqhNQDb3RXYYjhxghE5i9ow+Rk4Y2r9AyBPTn6o267eMNZql48ZHow6S86g\\\\\\\/JHLGARpUMJBIcK8htGrnHepf10cnBdnzyhgt4zXcOvJov09pjkpbZEqV\\\\\\\/ZJ28REfupEA2toEB04IhFubAJxF7o0oB0jrUIO8NlUH1iIiWhpSYQwUr0bCatrJxzRYu\\\\\\\/zdXrTMYlsYnBT75viGO6rfNx+2rh4FnxTNHK2vjtcTQiA\\\\\\\/MBZaASCiPC6Liwnygc6dgMwkDmRBSOK5jCAjsgfJDXuQX6nE1wITwJZjSUVah5Xbw3yQ4WMJXVg0ZkzCyqvLqiDzAsEtLOx9CC26g6I8EXm4uESdLujW0fw4Fj5UYcKm3gasNq9nO7v4NQdEkmRCIh7jI5cSD7EVgNNQE3btu8Ey1ugchlQsLAsF3FGEd6cuQvempfg5euHajDsiCJvW5TSUb1QKp2vUhqY9iRZnPs+iSmRtlm9NqRPDwqHX9lzRnP5lujzm2aCBwQ3edKVK6XHv4\\\\\\\/rcqwkXfFEX5PDNLa6qnXBKolvNlmTyfesRyiNrLnfBxKfcIrxBdL4h\\\\\\\/siP0stp2WgFYpBAU6W6GDUmC+q1bSDLHp38OGKek8J5vWh8gUmhAt6e67RapMpjSLI0WxRYGmqXk6tvHO7+AJjP\\\\\\\/8qMwX\\\\\\\/c\\\\\\\/mAL2OvjOv\\\\\\\/kpTZlV5N0Af9SXV0OlgcvqTn2mZkkTKGn6iRea1E+8tY+SjzhSu8jlxu87zXMVN8xBwYYWvqEy6j6I1j9FeHZRi2KM+Xq1GJnwJcGWyway7mFzKy00huEnsXUOqaSxgbP9E2Xt4x0TG5mPBCsPiDkjCtsJIGNKqBfTIXmXnt3S9HzKzNHtoSmv+sVf680+mKmBk7g6EWN\\\\\\\/mB52wSDb0I3HJruvYmZlt48X6QK2PJFtAjaGQFED5JwiwMwqGj2YpJjN2qmdbZwzoQfyrrwCQE4EpqXAu9iILicnV4qLasEIBcHNCqtoHXHZNZMiBEuHnSUX7nPSCmrKPjH5\\\\\\\/DGSx3ac7ERHPACimqTUTk2smsMcvaXpG0rqOyQQoPLx6QH5mWHaLkUvIudIwsobd3OFj2TgflrzvH17hs8CuRrenLGij2+5f5gl1ZcfOGHOtdjZCtd0ZaONCwDp2CvBFwoVv84+EC9kengWtB67r1qykST1YR6W8CLl9x3pMMQQGqlyAxvH4y+9iVx0wi7ONkjhai8YX9KCxvBl8uUXDdYi1EWZNWptBtPFI8BbjXHNzk7iAF6cz1lO2zqyzPdIFfB9+U1O0naoUEIUxSHC9alTTtasxenowJaJbTWbvDpOQTwwqdEsXRJcHHQvFjYL0+e7C3sJ63rFa\\\\\\\/J8JOi7BKMYyqyrhTW9vOtUh+fgXhfdouONlhZUlwb6nplryph5dhcI710Wn54Arbh6KCF3SBxloWqfejTqYmqPlrjyFNTLI9mqe7iN9hKgUvZ\\\\\\\/TZ\\\\\\\/FDUabkTKJ4CHmVoB93q2w45QFclDzGmEjjXMxYI0wAjNpzTr8zt17lJfZ1aYV1zkKenxiqrHCm3wYs3NOz8RQ91xaabSdQRJZ9cqF5OazApGUNK+PZwXGJeRXS75RLhEcMS49V0YBx9hBaRuZ7OtW1GDY7Zj0ibvOkjJyuD2WvGBWCe7KQ3ALKhmqBrKmqo0cPzikJH3FOyqqEsS9+zpLLFVry8qiXmvr\\\\\\\/FdwumCkeA3saKyzw4pHC++rsukyOqSZWqA49eMsVNMExPWBZIzBLk2zONKYvvyAUK6I8jnU98ceL7zENw0hPOsxd8LvIEArvNMQqv8IvKnB7kbUs\\\\\\\/No9LGf5YmGH6ZTp3Xg8dR3JRWsgjnGXp+Qcj4oMdAZRHmRKJGP9qeHE7LGIzPRPSD6XxaEy+0hYSS\\\\\\\/1mmmxyimL8xnYbnQmjA8+IbX5L\\\\\\\/JJaK9xwODsYbrnROAQrK+JfP9j4k4x4Bti8a8iZ4iBg3BlOkkfA3jRX3Os83x\\\\\\\/27IiWxoi+2iXZO25iaODX1AeBPr71v9JSphUMEM9+8auXn0LQsuwO3H4u9E3NFOcEq+U2E8CvYaDUsZWmdOODFwCg6RlVjg97oHOFXjE\\\\\\\/HZvJ4lSfduUc84ojzfUcAJiTKDWKIN97o7U5JHdsQ7BJUdWgtNLkAwSy8s7SxovI\\\\\\\/ULa96UN+dpKj2G6g\\\\\\\/9UMSRbRjgf4wVOw3lUikIx3hJzoUQhxlpkb5YnWI4DXQiqM48CpbMszkQlaQyi+Hv89aPira+9+mW4IL55qJhqn5wbubUyWo5VMQ25QQ1uBPvI+Eam\\\\\\\/4zAzcoiLeqJ1GNHuKFV7ivrjsZ1sqajlglqG6tOGymomSpQRF6xyHOvqpk\\\\\\\/Ngus\\\\\\\/ZHx3CGVL1VvyqX6bbYdQ37OiJzCBihjoO\\\\\\\/\\\\\\\/IRf3ZSuyycPo2tcLA3lElqURoFq0jklvV1YGbPEjNdY4SlA5Xdd4nD+TD1OZuDh4a+9pcQMeD5QxzOXyFsmEs6C7V6dB+jnc6tmJGeNeXkbdRMbOMEEOKztIpEOdr\\\\\\\/vl4ga6BXluOkWs9uMMOxJf43t+41JTTB0Vi7ciHR+ppN8drPUru1TVfAb4rBP0Dttj+\\\\\\\/xCaO80dOJ4XxEheJVpik+tCDwGLzdqD50O84Dg4AH++VUSyJMyAzfnNlNij+53RPR4tG5EK1fCXKr28wpbRYurfFuA0Jzbwm9ndI0n6RcitHE59DdOAm515GSSYJkzSuuqOKddHgFDQS75jqeG5WwJjEBdysWTwBR289tl9EVatTS9eYWuaVoO586qgVzIMdwQ\\\\\\\/sdAxfWJ7h\\\\\\\/YSPT9btu0X63qiWgZQ2eCSe5zeDuR0cuuIU+r5igQZbU\\\\\\\/pbjK\\\\\\\/AN6BTUn51IOAS53StyytawJGUqTtqBoeWTcMqjchY35lXD6RfGI7x6gJSoEHylLcWkbnZ0H14dSkozytwWEcZORZN3ico3MZNa+iMMshRHpzuCWB8HolvtVSlAwvPu8WS9Qr3\\\\\\\/Xb0hnKI3XtThxDiEkHw1If9mVpoEymvWRcmP5l7Se3fligzlJgIwIOmUSWpGhXi6jSAn6cApMr\\\\\\\/MSsh5FQLEOD8HXjY56tSqZ21ZcXuZ+vtZTz\\\\\\\/AjHPx5KeZbn3zwptANubX4VEu3RgEhrBur4\\\\\\\/61KdZ\\\\\\\/jpABELaWzazfi64mRZgh94ZysR8dUjgke1\\\\\\\/ADdcAewvYmfwf8k8YyganQOw\\\\\\\/zXU0R7Ht9one929IInnhRFoDO2MlHUZOTDu69Ii9QdfNsy3UTGtvETuPywHj07EA4mtoOSvVDguCU3iRwhibkGoGg4hIqBKU6vOEnuPCXwc9cg\\\\\\\/cFF+PMPsCTo+gcv0l4QJcnw+sq7K7BjgSE6kdaug9kR6xU4wzo52LwSH+oc4FNboIU23lgTjJtM8zfiNfP3GcB+f+4LM9mRZKqWI6umcy8UxMmZNZJ+8IYOTuQmWW+exNyN4YrRe8L+1HpaKcxpXAkE5XAZhlk8gCiO5H8c2YLe7QB3Fd7ibKrdEvmny+4A1tl1ufubtw6grB77stVRMRJRrIig+s2qzUp8WGJha5WVZo+Hab9Qu9TLZ7rdwjYYFhVropsAZ5EABzYki9cNinn9\\\\\\\/aR8k4dloqPLKyNnaxxC3gNXhMuB3AjR8T1KEaoC6Q7tTZUMo9HrKDFxz5\\\\\\\/sQ9k0jJ3x0TZd\\\\\\\/DVE2rP10rEK6MbkWPH1otk1xZBJYbTT36IZMlwXztR17QVdlUxMUTHk7nokVEAAnRDADKkmcL3KYj83Q5NVKeLJ8+3QPXIq5X2AGE8oBbmqPOFrYQ1Brp+8Nk0YscvpkVQYAjNfvE583BHdEgEDxb\\\\\\\/6zZ1Sg9+CxVFmroAHNT1ullU8tpOfKDNH5aHfnXWN6FjOIwYmEIQKBztk4hAJZcQtvgTXOykYKxJtiLd5oIOxQne4uxikbV11mDOMZNiP8r+raXGWc0w+LqtoaCC6I3eWtnc9UKe5jLzQFSB\\\\\\\/xo+IAjMDLfUn3+OUD364VShphVyBLN5AJgkpoYrob9DAXcpWygKGBFs4qoC20mnpob1ybSpQvjqRRXZ\\\\\\\/O+8PfijQDSQnbENvggkvFdsIdHylq2A0LfSLfwd8TCfoiO0KVxrWPoNaTQpSjSWId0WcWyB3HgMCxOcBDkJJ8yWCNQ2IvjyElAfFp7Xbw\\\\\\\/efONMZU55yVAHn15cr6zU9GFtj\\\\\\\/lamuBLjo2dqGJ2Oz+f6MYC4BPTjpNgVk8rtE\\\\\\\/sbpCNDhSFJsAkBopW2YsvwA9AsbbX3e27t9wOh8orqtH5tVjybk02gyUjVa5+2kLn0HzNTs87YZsjx9HFjh5wPFjT9Z8Sct4YKUBK0ut47iv8C29+bfi5Do6SyZIpyKRVsJxQ1M9Mpi67S6TG4Y6IaYYCyMpE8\\\\\\\/KyQLdnOrubAEfmsyV1qHFl\\\\\\\/SFoYNeeDhHp1\\\\\\\/97Tfzn\\\\\\\/4kYFbwEnJOYwondOB6tD1J0JFc+nL0CI2+f9zbADeEhDxRRKcDDUmsPSpkzFjVH1hY6\\\\\\\/VnWK\\\\\\\/7rC2K45AK3osxSUJVMXMUQ6ymcZk17loVtA8tidJKFulG9B6OzYnkhk3nY7K+EtWZliEMAs5Lf5eLp6JAldPq5kjs\\\\\\\/aEMeqJ+kx\\\\\\\/e0J+cUpMtYuTUAcQulbP9hfOlOuftx0Fm9L8Sm8ojJz9lRt\\\\\\\/vUDi6ReFSx4q9A\\\\\\\/39Mes5SWM+BI7aBOL2C6vKYz+GDdgC8ZXAcvE7GAd6FdMvphGpJBhCxT6qkWF63fcJSq+s+Sua\\\\\\\/oSx8XoXIlBDAidEGmjLNy+4L+KuAUtAOwQaLpjMbck9oSxxwhrrTlvQzFV8DoehTmWI9204vkCuLyioMOqJbOJRWex9C3O7rWXbvDr30l7y6qvW2P5SNYVE7M\\\\\\\/o76M924XRbHLN9WQ\\\\\\\/75VXrbPAqnayOMmMg73ZeQYOz\\\\\\\/Fg0Vq5Rq8PSH85dtMx0aVoqHqzZDcHKcA9sq35sXXQCxkUi\\\\\\\/I2whvLuMl3abVFlUjRW4DVmYWmKLyMVwUcobSSZlXSF\\\\\\\/MwGdGYWi8wuQfaAARjBOFelGrFGCM8PwL4laou5hP4NI5pl2inYBziprK8+M9qBa60fYQ2T0XSiX7d4QZVQpHjmUTgCjAJ+\\\\\\\/qYhgHjob9et\\\\\\\/f8fJ2nObGfpNbi9RbKmKFg14Fy5seqUrPvkwPN+UgpDUAtwlp\\\\\\\/csmnRIQgf5iYt\\\\\\\/wLL5ld7rGRbcH75iGoogg4\\\\\\\/mpUnuCDFPhsESotoCyRlRU1Yqj8OxVhl9Ffply+4IWD0+DNaA9IMzRO0gFaKDQGE+lDrEronTHaUbn8apPN0VF+P9n2G4+v8UpWe5EQtMxgmYqgBsfaW8ScSfjpEEMCBTqFM0TMVm3MdnZI9\\\\\\\/5MD7zkNm\\\\\\\/3EXw7dNXc+EtMcqCxjUMCMNgbOJIMK1FBLGiHYsW\\\\\\\/dWtHhmNvmu2MV1CCRdxg0AbaPQyylXLGg8CW47ApBY+lCdyH0lyySzz\\\\\\\/Njeyl8YMncp5EXx8kZX4FD3CcxLs4Yq4G3OYhqhzialYXIj3blC59R0eC49eMVh3bU49sJyZm47eBtiS0qGvyo\\\\\\\/6UdNSKAWSeu2NyhnQF0fGqwN1oYWzm+yEXjSGYovfq8OHzUeo4DQzijhazl6Q5cAA78CJrbitdEj1cfGedc0NFK6s1uzXiVVxoqrITDKG6fZIJFYLLhtdKCackRT0GZIAKFqGhXP4tY\\\\\\\/OWrAtNUthYzMA6pT6Am0dNShhb7hd\\\\\\\/xwzJ5aqLQTBeLYalyqLFaUDCH+WSIAKmunpvI4v6R+eNCr944SF3ps\\\\\\\/iSvCu01A1dhOQkJ3f7nVsDhXNuEqdAU0JwGgJf0Tbk5UX46jXATHB6\\\\\\\/LjuU\\\\\\\/4TdUJ3odAlU1TRx2ObeP7WR0hRJrGyis33H+jBCmbfthucxN\\\\\\\/diA56zLH4bjd7JjlP601zupnp0Pmm1qajY+SRVBVIlOrg9\\\\\\\/fDN9m5FSBce\\\\\\\/lqmnJ2YYVj6SMRcIwgYUkZrOXhUGIOQvAnoQTvteKXFOmXLeOx\\\\\\\/4jTCjzZ71VYh5PmZEnz9zsngJBUq9N3kL1roR7W2obECCB1UMsejl7\\\\\\\/LvHCj1JcLAlHmqHY+zrylFeKY0C23lzgTiwI4K\\\\\\\/WbR+L6EKD5BCFIOuvqI5igMhqLCJRoLgljnGcnAqqskZAeL4A9nOrNQq6v\\\\\\\/ZTAabT0\\\\\\\/UqkIC0orCHgvOegX\\\\\\\/7NdYxeSORo\\\\\\\/XZF9O0FDhH8Ekk9RlufURSHGcNutcmMSSpetX4RyJyev7D9QCKuFdNCS1u3utxeaT6h1SFcaxikVPvcKdjstxzJECGwH+gKdJvdLNQUSJhvfMWxtnFFeS3XGOJGAm9sfv3FE+14+CKl7\\\\\\\/ZOX2T+9f0edGHfbwDVz5lD3VhuklF0m\\\\\\\/9Ust4KpNEMpVR\\\\\\\/563kUrcK8DZPTTIsmWufUHE7GDP7VUuWQMNd7DnWL9rI6sO\\\\\\\/KNW\\\\\\\/FwrdlqOkv3embTLYcDy6O1KDUNzGg7fJE\\\\\\\/QgWbO40Vgfipy0hZfnMS3E3i4kUJ6fMBrgdc5ZVfLZQSw3ZyrelQgZ+VjnQ6zinOB+c7JpgPPbiub4o+aKHyKBu01YkY+cfFd9szxl71QRU4\\\\\\\/7LEUgggLvthYJYuy1BRCETimqKEpUhRtsyMvUDK85VtP7tW\\\\\\\/4rE80kkqQEl+8odWo4ulCrVwPTovxerysaVV9c1Sr\\\\\\\/6AvwOwryiOaHBi\\\\\\\/UGTsAEv1gNTDveVMWK+hQthH3ibm1IS6YDED\\\\\\\/jW+mBlzJWOlvlx7Z5D0JDbZJySnuqz1t4ZalQHGF0WlrtFOFCbZIb70Hkb5ebimIPX0wl87+IxRo7JxCFKPxijBcnb24F2i3Xyub16OQl+2iqoQTDEp4PpLngteC1RsTTH2GK1vNy7imSsns7\\\\\\\/mvzJKwJLdDjwA+4Zzf1xqxpfggsNz45zzN2wmz7Kc6qWL+7Oe0hMm7gPDGD3f44PVkPdgmZAa6wpqkEtYJAP4jCytQCMqsMRdBrun\\\\\\\/XWDst1muXTt3n+RgbaRAwm1UQ4x\\\\\\\/VxY+NA+rXgZcsHKXAElLeWw+4cPexs\\\\\\\/Lz7yGtb\\\\\\\/UAUSuUF8dwOZHVzJ44a71ttw6uH+QegqEFCYB2eMPNzLOAPWaJTlyMqKSuX72R7Dmq6\\\\\\\/b3KnZaZcOxJCbYfT4Zjy4UOh7pmZ5WeiTD+GF1h9kbAq3oUQJ1iKU0AQfBTI45upGSzEXd+n5NrusVtvnrv77PbBkryKzp7rePBT9jQrLzz+8Ano+oQDn4oGr4DlW+a1w4PaK4APZmswhkXLYEsMPYmi7GF7ZkcZAAwupuAk\\\\\\\/LTnQ1bRbrBU6jYUn4TO+VF4mFqViUQKUbwYWwqWvDu6v13FJ216xkoVlyzZYIqwy6fqlkSfv3D0KIxUzEbaeQdCrhOZYqNgq4jWXjT8EfGwp9jk0BpUgAnAWp55448OrYype10ZQSVVcOxoae7zaz0DCBfzTd2j56+THm8Z4qt4UrxW1jerwnvTM9sgBGU6QULX9vgfedpVXCW+64ZaCLwVkp9NKVpD5QiJaDsOsn2b\\\\\\\/Vmcvx2d6pY2hzMwR2Spqq\\\\\\\/CEYPqJolc9245P05uVtOwYZX7jU5+SBzopDdf1+rrq0lzRL9jlhkExD9NlfXuT6x8fX9Wn3x3drpsp81yxuRlt7BI+1ldr1jpt2YzlD1HGJ+xvXrwk++UUdBZViIbmiNc2B\\\\\\\/WtCz3Bu60vBvGampt8zpoTSBeue7+Fb2zMuG34Iy0ZV8B4PPu2Og5RdDHBOE9HTWuJgQoi4zru1f6AlWx84G3WtzbUkNVGc6p7hjxTiGpXWlkjMLzDP7HMQnv\\\\\\\/I1WGA510MYFnX90BPURJaRDbwfG7VBBaE4tuKoZmpQTXFfz8pJbGwsDxYttNxPmxihN6YEMeKQ6skrsVQ4DVxEp6yjCYX5kKyIFpJPSc2vzK9d5ygM6dB2TdWsElhDomCr729nug7FKrMS9LVBVIpidKG0ozEHOtVT40oj88OorYCJIl3r7mw5iyafWEOsybIrWV6uyB+2Plx4kbK91nZ6bKuQGdgnReeRyTujzMfo9KnFgqmhLp\\\\\\\/WHWI2U98+q4qLvHC0+QnkswWKBvdoF+CwHZ1n4D9iPUg5qDHSvXdN2GXWPvu5wUQx+uurXwgPps4\\\\\\\/G8Az9gAkowegTKvrd4On4DeexwQAjBAX0Qmvm\\\\\\\/BPnUnGoSWy7s2yYGJDuDDm8VL8Cy2Ys5Jq+92yGWRfzh\\\\\\\/UcHzQCVaAhyPhEqo1vWM8fmkJR9mQhD3H\\\\\\\/iF\\\\\\\/GOJXjVSp61LKvA\\\\\\\/7DAajz8MiQQ0x4DYEzjCHOJMKc5zc44Rz5wfJ5aQ0Rik5e7kazDx1xAa3W6JQ4GgVuCyw2dQ3A3LVxh4DBK1iE0ZMXEFhNyfdWYdeNn709MVF0TYqXmbk7RB9O1lPzd+YegfddemWEh3qzCut7ylJAFiEXxRN6m88GTWBGuFRPDrPbBhDb+ckuVAiRnasiDKl6e5JQ050t32uv4uM8wnvPSt6ATXn8oXf+OkaLPVhOanrd6jMKUO8\\\\\\\/Et878eoxy1Mf1PSX96ziRXqMu5Ir5Wiobo7zS9YeFYC69tYevd7LXVtEx8G6UIF3h2lf+tfQGq1T3qsEUznYxY640\\\\\\\/sfQfEVWJefbmIERPT6RUFVhZkIYqPjX9sxZOjqYNqRgMFTe7qe4WecS0JBla7vJAm3ZY0DT4VILsKswvC7wIu7N6Cq\\\\\\\/0hz0vN58KmsDKpWLzzjIiqnLJ\\\\\\\/KlLHt0ARL2k9BxkevwfL5C3dzJiZoaXf68ICiKmQ9dKbFIGEwnLLZq5rpmnZtd\\\\\\\/aDS7LHfGcg1vTtTO9EuK4o7LT5g8krQudBe7A7uW7rI4QygOwksJJgv78m7Dw23vqjXzdBEHHQV1wPufwNcw\\\\\\\/y5e2hSgTvCKkXC7tTDJd5TcMqMBdlfi3pR8kFVCk8O6TphrkiM6HsQ5b1KnNo0Vm1d1gM5JiNgjL7F36iwQl8xxnT6t4TpjqQq0sTtzsiNNwMgfL6EUwRirPmVL4U0Pgis3C21nAuyetwiZviTwU66tuGo2JsCu9Fom2jQPTmFQqzeploCCaJG4MYMmPlieK+d2nbHHRDyvOsHEU6rBEKde2F2qn52GKPEkW8U4frI3ljhs1q6nXGITcGcW2SKf35cxGM0dssOrgfJai+10Z\\\\\\\/ZG2MxBxwfY+3B78OjxGZJhf4DLfOrwwRvBq8MUGYlTSf4\\\\\\\/RpAPeAd+Z+EYhUv2OZF4RnecL6w5FbHZ\\\\\\\/WWjcUCXnHHFNlWtLmZoONIEsHJn7CU20N+MJwZbdx+ljuE74zQPiUkNkVmwEbxN5VHEFhBAByJmvZZCmRs46QUQ\\\\\\\/dE+qDR6bxiAssSYH+VNA+mqq8umj9YNCNVGyxedu2GRfD00lKfF7Drk+7QBbb+X3bf1BPCWE8PFp2zy2v+4d6WKNvHaK8DPDrsq8voBoLlg\\\\\\\/Kn0iE\\\\\\\/SJ\\\\\\\/DjJfJTLGKRUt3AEFp4K7l42JTwvlT65+x5iP086XWMt0YDFBNY9s3WrRGsA0xAksrQmlzqdY8bzd1NcqlqRf7Juum5PcyFGpMXpMOgYwsLDgm9uI5A16gNUIzl9iZ0nuVZUg3K57UaTZFBXPH0jmy0rJrhdXRajUi8vL6SpicI3VNmVrtIZtEAxmb7ydniyPrvAU6ueGcRNfShB6TStiIp1yuwBLTuzwgkTUPdeEo7P7JdiBMDoP3A0SyLjD2s3K\\\\\\\/tPZIEgKvkEArAcw3zUPSnj7DbXMbI3L+q\\\\\\\/QJfLj8eiXATOEIdybEMojMHi9w7OQN+Rh65T\\\\\\\/YBRLZ0qTd9GI85DMfOhYUYM2lQp5oQAHIEZEL+m\\\\\\\/4zca6EplsHteTxqBfMgPiW4TbhQf9S8LcDX4l7LFzC1Osi8BhPRIhW2zyhI2KdKpMuGkZNn+JJJMXV1xINvgV2hvtVPgad5emBmhkOCDLZE99WP7ZRy9MbRAhNxx68zNsnH\\\\\\\/LFpVwz0FW\\\\\\\/URk9xuIMZUvNq9vK8iuhEFFfcyU4RcFB00wKqiffXm6B+JLBn\\\\\\\/wvPFf0thYW2NrqoaCpKKuHeB3BuUuJhLvly64oRbE\\\\\\\/XJvm7CuERBFQtXrjUjCvvDPPWZz3AEYweMXwTjKbuj\\\\\\\/CdelcZZ0OAJrrjBAfGLSyuAcFbySLlDaSU5yr4ArnLSM99zWtmGpYSNE6iIHks8\\\\\\\/RD4SBOOy0093IlLUFetg05M\\\\\\\/R0nfSLX68MaoVZUzGCFNqL+bduQrPk6fZpIU5NUDeHTtmLV28xK7Pns5XSW6pRk+zkVIffTszNtL7p0PP+QPvOu5aedxsJmCJFoAHZSbDB7vbelXytwDInAYK\\\\\\\/TJTT4pXLKQfsWVx6+ic4P1+S6uomBmNyMZUGx+BF48j9l76b\\\\\\\/4\\\\\\\/4MJRzaHYwyjF3FMWlRt4608iYIlSxZZH1BR\\\\\\\/GD6boH+LxxHO1caw8izKCT5iAy4J2o44Z459wF68+CJHDp9WROxg6XUZiC5f1uBdJdZBnJuub+2+rewG5Tk8bwScUbYmhSrkAfP8wc6pdCdDNpLghdPJr360exNhNTK5grLV1gtGi7G+rb\\\\\\\/Aa99wTxQKz\\\\\\\/oGxQy7tusi25ECQrdWpPeskbp+fV3C44jBOcB0IHvu+\\\\\\\/g05QkZVTLA0le4jWNl\\\\\\\/uzh3M7O6plguEEtA915wGvXhXqA3h8jUXc1iHlXynF1Fwjp6EdtJBFv6AbF\\\\\\\/R3AkExjz9i1HJpDftJNmHQWIeaH1SfQlZ\\\\\\\/NVaUCHU8qSWJbtnazldXr1Bzq\\\\\\\/1kIQl9vzRzzw\\\\\\\/quht4be4m4645bL5LiAHIWcXwR0h+9NC+2pR5Qo+pFoSjp5IH6k+Yd6HfX8Bl4euRrJ8RF1vBd+nPsXLD8hFYvagjW1mL5DCf47UyXLp0sgBHKDnrbbRPK3c\\\\\\\/MsDskTjeHK3K+JhHq9\\\\\\\/3krUGQcKRx1C4kO+jAhHu2EnqzpCJLGngexAHvzbrPgZDZo0eQVx5kC5zo2flm\\\\\\\/qKhVtBV8bj93L+DsLjy7qFrSL6033T\\\\\\\/JjzC6Xse6bqPSgihXaxeKmU5IGNthdoJ1gmW6v7uZLie3A7GpQez0+Cz6nDFSpCaQp0jyPAypNkijm53zCX7VBsIO+kZOa5lgP\\\\\\\/ytV4PNQjQIzBdhDATvOF8\\\\\\\/nYG3UHDWbi6ePJZsxIj5oK3GWG2RjAyTRcD\\\\\\\/6XaDfL9MMsF8hNwHZ3kIg5wQLD7IOXNqMq7\\\\\\\/RoJpG4mICD7EjkvJQGeqvtNvt6Jh+Pk\\\\\\\/Nnd\\\\\\\/Flkqj1t6ndZUclztMO28YouCnbGsz2qS35GjzzSWqpLMwIAKlrw8gLS5D8rMyZS46OT14RtPyZH4uQYfV9dXhKeTwu4Fuydc9mCPsBg\\\\\\\/0RYM\\\\\\\/NMUQqF09lG2U3pu2tI+j2vBVi0bjT0Y0qKr1LhQqEgxmmMruxyWIsX0jZ9k4qNe1lYQz3eWXYwBjKJpJ\\\\\\\/Srb\\\\\\\/jE05tPtbX8OpsLMAtJF9HogDZoOpt1gjI0aYaBHSZykffv9+3Xgv86hwvCSzCFM3v6q3eAS4ASgETE7eqwvSmErNuOsFPy+lfAWjrToi\\\\\\\/6bf6od0+FNKu6io2DcRe9KObT+v6KM9wS8LSpWt+exiKkGbjCPxUVY+iEW5WLx7F3HR9VFhHr6mFXy85GhiGENN7YPPkji9iGmeJs+5oJJii1nmhspl\\\\\\\/MMdPxlC8gx+ukh18nO1lxiEC8K3Bzb0f0z9yPHrKTf5o\\\\\\\/DvCrzyGj5j9qn52oEs6BStwpIVJFjnU9BR+iV0OuAd+PlUUCJEyY2Eta3phusiBnXii46AJ7ehy6Urx5d2JHfnGpGYKBpl6boy42WDS7lsUMFrjpamOclWQC\\\\\\\/ojDa6aJMGSYQEK\\\\\\\/L9OXALvJcJU+ZpGRyaK0sd8S60\\\\\\\/3kUuDoRqHi0iBBCRJAR9zRhlONVi1IdjPGJQfXXzR7DWaRyUtg8eDbtzBediLBPLb3OBRIVDXaKV7V4YNbyIRw\\\\\\\/x1zum+1LRUPIwRHAm\\\\\\\/ZCFZAloHNPUEMzynLseQBQfJAfiYVD\\\\\\\/VO+eB8V3dlkK+w0mpHgkif\\\\\\\/IP8iEfNMDPyIeK0NRnoDgU\\\\\\\/eIzODD6zsB4a\\\\\\\/k8evIG2e8RfegjlseIZuJwndmhNe1KbMQg+42Me056Dq+D+3rIrB5WsrQY\\\\\\\/oqFSHlyArq9xyAre3d9mSwFiaBdk0cqmJPM\\\\\\\/VCxdgg+f1Xj8eh9YDCF17DiZlnLvJ83GJGIOPOwlbSnqGDuJburfgOxWT8p2JgvdxKbA5GOGW60kVOCq1DIwhqg9bwHOsM6giDYIYo45JIhEEs8BESs2j1S0tBCqr4wJk032Pba1C9KpNDxt7zd1\\\\\\\/yHYUzkPZLFL5AvfhMi2NkWQc851u0KhEUoeK2NjD34dhz\\\\\\\/2AruYA+RkJ\\\\\\\/6dlq58TP+0rWd0uEY4tyJnvs\\\\\\\/fcRyR2e3ubE1fbEdss3pZ3hIsC98qy2nJWP4rlxTkndrYQ12ucPrMoeLTULuWB9akDPK7TOJeIyBVO8eGgNO5hE6dFLvJOcYcv29M7uldcwIaOsuDa6sDK\\\\\\\/08FxRS+hjcww0ZYVl6ZIR5LX4kco8JFftwbWnAPzJ9zkV6cJa+72ylxaUf2xpJIUMtenDvc5DZnBpcIuQAEyFT2o5MrqgVRZn33Gz9I0Pk2rNhafRfrJaLeEHmrtsCZiOKsy6iLIiHAbxhQdnOETsZl4edihPzemenT31OotJRJtzy7T4okYznBN1tM4N5hKjT54RBki4t7e+P1DhmOc4N7l9AvoN4W5I78tPem9nVrAqNNVGOMI9d4f3mpNxk0TVhHldQAEXjn\\\\\\\/NIQgzFahenRCiwQMfX1kWeR4Ml6gqcf6V\\\\\\\/pUjtSnDAcdg65YzHlII1xyrdVNvWjk9lnUGkgAvAC5eYUXnZW+xECnJpxf1s3bmczWmLWAQeiADWAHUKEAESguxSl270hu5Tl62lfBE4LGqS5KnSg7NsRZqW\\\\\\\/lqUrN2Z1K+XBXEWb652zK3Lt+bz2l9qhPe0M3ZzhdhilCUiqxbpFIQ91JI7nOdXoomZrV7Yt8CjVn8gVQC5doBWVHnHkR2RaXtjD6sh\\\\\\\/oO4PhQLoYIs9Dvdwe4H9rx\\\\\\\/2EgxqFZlCwLnAlOGR3ZTbHkOi1efs2HemKWcQOCQnApXNusyNNwY5GhbxmtBC3W1f8IgIWeER\\\\\\\/0Szv9jRU2GFy3JGIe\\\\\\\/BiKtgvDUKRK4CE1mzhEtFpTyMINS6GQtBY17pw98BSxnInM39di4M2EKSVVDJq8LYNkMRvauHz7BbYYwt79c4SuZ\\\\\\\/azb1vuBu8piucYtP9qGrvZqDEdwZveZAzoYce515Wptm7xV+GkeREtSBos9Y5JAiDupTKdCKsRE55OqnzhaP+508SxgfPidNlqXFDOVoB80rZkyxhpFAPHe8+KjDJJO3fWa97PkHj\\\\\\\/nJKsRdQX+HBR0YP2A\\\\\\\/LGYi6zXiKTB0zgZVcKIo8DwnbnCWUb+h301ugHnerLy1sJgYLH++oEwFoMiYbpzlApo1tFchN4OpRzUceq1QBez+l8CzuKHR0uAl6NxrrbGGqoulR5kdi4wbVymOt1BACmHmz9UvsX61miEs5pke+5F7TU1HSZWeLPxwI\\\\\\\/0dCqJT2mgh09on4PI9tFemXpysEnBLh7cXKIcF2Qv6XkXE3I903gixAuim\\\\\\\/T\\\\\\\/oLTNAOjKsFIgUQOTUFqSfooj+COs793ublmBIFwidb9nZa0Bw2yCQgWXGv6GYiZTA3L30LnT6I9VONQXazvTLOyF29p9yZU9CYILSGoHZuRPl6ThWcsSUC3Zaq3YutrrWue4U3sJNc9I4IKZvEYZHKOFVt9mWwc2L\\\\\\\/dEEC4IY\\\\\\\/D5AcJZhjKZgyekNLiQO1E+EnWw6VGr92ByX97O75hgZ9gpxxsRu+O+qwwgYIyu93bvmjl10c1XjXR1P+PS91tt887wrihwM3XkHsCQfU+k\\\\\\\/Bhsf\\\\\\\/yls084K7bZ5J7by7PxqqDW\\\\\\\/Jseebhv\\\\\\\/+H1kzciI41CF96OxRABXQdfflsOGKXVQhWvn8YsCBrxNHQ5JiPtYSp9hw0pzDCz6ltzamwI\\\\\\\/XWYQ9Xxu6\\\\\\\/jH103tWnnvY5FnoX+urMunXBzsP+Oxvh4ZUjZSUsWn1hItq+RPnfK2P1T90eEtpIsX8WlWeUkApdhXR2\\\\\\\/uZqS7vqSPBiBj+eUbB7Q93vd+QZ8z\\\\\\\/DENWebKgw0Q2dpi49HtCPCpl62bcmrpbCb2DFOQVP2Wp6Tx1vuYmaf650lWBnydYqu2HVp79V3rGuqO2ITN2xpDkMLP7Dh7YUjwIQ71FTXxBT2+wMAs6lHsQPAw5Oui+6vTe+ILqsvYsE8Fnw9EUO98G5przM9UdVm9SxQoOTaNwxVic14zW1CzmInA2s46mufMhys8CZkD4UPLeRjphPujnBtTxwfArMyc9p+HYvrfeHoK8fYQSBKrcvsc1w+dZIKHiXJCML9RbfNn82ssDPg2h5UDeXn0g7DRqjzA8YZ8CUg7xpaZOUzhVKXHxF77yfPFRAJB7jmpNTAPGPr6SkqVNEwxkLnW\\\\\\\/DqJUSa8\\\\\\\/sYgKNqSMpaKpBdywCzUa4bHEhvSXrf37q+7DD88I43AUaPROK\\\\\\\/EwPBoty9njW1AG\\\\\\\/vMycoswtbUq3PYgiQEvbMjUD6UCPajZUsYt7bVhPX2pLnpQWmDtvj5cO2ZCZMIcRAB7afi7j+QH97ymeNhAOX83iz2O7zwwlIIefUEl7ArxcUskp1lXenyk5CsiaRvdbS9\\\\\\\/lNXCLWxvgDnv3MuptdnwZoWQEKe6XoHTKlnD0h7wMSu3tf77VwTEfqi8cDWlpTK515qIx7GZG55Do+yOQ1spTa9wnaSMdlBUiYuImmf79hEaS6DYiWGSEYpnfVR0bT4zHKY8HtQcTm098kvziqIgyQfh3\\\\\\\/eNE6NBqEA55pPg3htNOKzHdtfuXOAegxviS\\\\\\\/2F+iwiH1Qe8Bh6jTX\\\\\\\/bA2ygGRKBPykOiGbfMeB7nuXK\\\\\\\/5TjXdOG8LeNmVB2gX2810A36P7629+9MQOnRhPqvN07OJGCtjVrC9QeyWtNg0mF5gKPzCKmx+x+21zt9pnRInhX8sg8F40d76j\\\\\\\/SWBMy76GF3Oxgi7tmhJ0iX3MPEZHJAx25MPsrt5zLR5uu1a3R\\\\\\\/VDApUhJky0VXl2WU3mrgYy2ZviByM\\\\\\\/t0N\\\\\\\/LiHAKlxObIkkBBlc6a6KAjfKAUx9ySk32lS70Vl3YUBfn6CJAFPUh+iNrDMgf\\\\\\\/YA15UDBxjDz073Qcrli1ahfcbNMi\\\\\\\/jMp+qYuCPcj5HWpBB6ECNkVr3ClIdvXrw6GtsixaUBopi8DJxvuPnGXBod0Md7+\\\\\\\/DoNyQkdMZMI2edHJmjrdNqCtI1aHMYMuCQeQeKSD0vYf2nGJzRxu7wCMP7rvqxtnN\\\\\\\/MPd4WAz4t7QhqET8OmFU6mdwCGQhs9sMTRbSZUaPLZ6mZwC4EWxvvHgEVVCnyViPtBx0hTCBWbluPf00H8wtfi5TKAN2tGTs41zgaInq\\\\\\\/lTOrmo8BUhllzj50\\\\\\\/f8zbU925q1xh2KYKyl+4RUiB32Ke4mLyuRqN0hnebbRJwEu1z0HX\\\\\\\/h1okECHzEfwQlL+tnyp77PKSilOj7PCsUorYevtnHIK+LA5cKyOV3i3LNJR2koB8rzwgewFdSg7evh9t5ESYs9etQZDyjQ8P5rF3E834AzStNc8Lpk2XP4pbS47L1bJ7wPnYcyKvqo1km8BBmGywVC\\\\\\\/eYSQlJOeuEySp1GCBWvD1gQ\\\\\\\/RBMcLkSttQW59oo2TzKtiSHKndgA\\\\\\\/wAnoekOju6SLq05FOpj01L9iW0sf7NQ1jhfK1PD0tUgPZaskMuSe7J7yWqM+r+KxHk\\\\\\\/TgKVOOAGItMi2EuXXVkARkCtMqBxbamgB8p2gRWDrVlDlybvh0KtN0fKnCfhW9nBm7Y0SFi0VHHqJuAntNR8EvN32uqRmuhxKOu6l7R3dx1g6pwz+rN+5sgwP2pvC5CCmSFIdek0p27oawnhCq3xLKvYo7GN89\\\\\\\/Lme3eb4N5DKXWpsckg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022FUuFlfPMavm0pkHKX51JZQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a87679c2b2a57144429b7915fda755a1a895320d7a215fe04da30a266c6b0bbd\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022oXYneXohQY6s14R42f3XhgHh+LuE+\\\\\\\/ZqufZdIc0MrVArDf1QoB2OC90VAZLS5J+d8nCPpkAWkQeoM1fDntyENoXGj3UYXOP4xGTY0o6YOdx3FqP59Z3Wfqu88Mu7CoTxWHmuAXlatUL02UUeCysjXIi3CW40t+kDjwqFkNN3LwUlRcO+sgGV2frlpKp8EN+g7SCo0a+v7DbN1eknSd9eO\\\\\\\/OlUiWksKxcM\\\\\\\/XtPBNARH4QVT9exBZnP\\\\\\\/ncP9TdNwarT8aV7oUKz1nCnlhY797+84Ruf4NX8JGbhsBGu8K8enPGBSctNQulCO8tme5QzT\\\\\\\/6r5vt2GFcG4O+64zyCFycyny3cnElWuTanaQXzod3FX7PxsdPE2GnLy13I9YsueuXAzOHb0p87RgXNZXyzGZhiHf5rjUIIQ3cQLXN\\\\\\\/YQq45pEjYm28bPifX\\\\\\\/qjH0XCLYu4kqIF9k86jy6UMSj8zuQleNR8c\\\\\\\/FF2Mf7b\\\\\\\/iq67uIhLf8YZY\\\\\\\/9RNnwmTwRflowzAyrZjDHdeliB\\\\\\\/9P12I8c6GsK+CcUxqh4ch7Rncuo4PdYUYuGlJivjlA5XA6AXGuxm0DA58KUwhu5QmRDZiv0D0PyLv53Eo52XkYMWHz2XpADEBeukSjGpGWYv\\\\\\\/MCqRMiFwYP3oFl5tY8HSG0OeqQGPgtM5dPqEtkrH9pxJb0j\\\\\\\/eu8O50VB4fJ8vjFyBkPn\\\\\\\/S2Oqbbj3Hi\\\\\\\/VYYxV52z7M0gJcRwPnLM6hA9wLSlZqAeU7WdFRhr6u68Do5EHAGlcQxlQ0EkYvnZZ+W78e+vgWhE8OYQuCr9wqDUmRO5cSaL91zgbT6GV1\\\\\\\/GsjerWA64p7mfVAatlMlnbSZXyC47C7mqw2kirpbCVft4Q4IhPzYQzdevDR\\\\\\\/kSpFDItZ4DYkwLre\\\\\\\/xGOGlWnqEFDrK\\\\\\\/dtrk5XZac3ZhAg4O10ux0SI7UuChoHxhmHpyExRDkX8ueo0dxoxlitI9F4o5TX541jrNFitfA1My5K2z47bi5xtQItA7stiUsXRJROArJt3Iuu5Uxp2NHrlDgZFmIe4R7\\\\\\\/Qj4TrL1sqhGIRAuOKb5cnEd2+RJrGqYMNq5PNhTR3ysk\\\\\\\/7uKWNHiUDDOwju+q8YhPnSVzrWBnM2XA3Lp1HA871e5y8Zr6ygRy677bB1apHCzn7GIA0uqeVLI8zqPbAANJ9IOrIowzTzPltRWf1leFrtED5JqTxxsGNXSxwzf2GvE8izTrZtNwQGrvBsP91WTXcoStDCTScHVAPKZlNPUINaWHB0mumlhQCPHa51Hb11fxHYV8kxt6W9yJGvAq6i2tRtUpl1ztVk63QIZrmEMjbnsoUQUPvh6TVEBYLsy1VcEccTJ8IaijKbbZ4UjC3c4Xsl+09xyvNRZCVhppDCGDNVWLRgBwE5SbxXCQLvvwAHHRdt0nCN90kQbXkbLIZlSWkDBg1TT1krd\\\\\\\/eCqTEdmg2ldigzhnZkSIcvk9cE2cdYbH86l\\\\\\\/ScuoSLAiDe6HrFb\\\\\\\/J1UL2ErrNPwECjvb61FDvFpNIQR06xCBqYYFo4BCjjyYpjy0ezSSOZV5WqD7Yt8UxxrEN1tnx26OOx5lvp19o\\\\\\\/pyzc2im7OPc7eDhQsIArt5e8\\\\\\\/q5n4ObUNOcfyzmE9UuM6JGqI7Fr0G3MJEgkYStCwAKqmnwLn8Y3W4Iit3QDAIZns\\\\\\\/+kpbfISi2fm1L46XJfpR+JIp5oWWo+1Novs8x93BT3JEf\\\\\\\/vSl2OAF4xBovVmvLbdbakzVcC6qqQJ\\\\\\\/MRlABQyq6Kht3t0yk\\\\\\\/eZPkFkfT0D61t4b1CVar3URJ0ZVCvgWOHIWWCGVVQLe8X+WMwGFsvNjHKgREU9ywc1Xm4OjmIcWVEuy4YmAphMXyYpTJ8jQws7Mio\\\\\\\/APZEX+wxnJLmq5ZQtXuvjBZBonLv5BHKNtPO68lIMqSFYTXzV63k43nuu6kTNyQXuKqPNlJTLFx3O484C\\\\\\\/uJlNYO6+XMX8hm4il22UW1lQunDlBncOKvoZlCsRv7lnbzn\\\\\\\/487xt4wAJa6Yg==\\\u0022,\\\u0022iv\\\u0022:\\\u002287A\\\\\\\/Pj84FGQ\\\\\\\/K0tT\\\\\\\/DGybQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022df0689ef7951e05528a80bc9fc5ef43ee4110fee0e25aa4b0122db87f3fc225f\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460558 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/2/427" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-GofLb8NP-cfMzKBqjCi50lUowlDv3HmVl0oH1E7dNn8" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/2/427" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-lv6pKLebAHJhAQ0HEHmPv_9HHpj3cNN4vMimeMtlIyo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/2/413" title="Forward Genetics by Sequencing EMS Variation-Induced Inbred Lines" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/2/437" title="Screening and Analysis of Janelia FlyLight Project Enhancer-Gal4 Strains Identifies Multiple Gene Enhancers Active During Hematopoiesis in Normal and Wasp-Challenged Drosophila Larvae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460558" id="node460558" data-pisa="ggg;7/2/427" data-pisa-master="ggg;g3.116.036152" data-apath="/ggg/7/2/427.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomics of Compensatory Adaptation in Experimental Populations of <em>Aspergillus nidulans</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jeremy R. Dettman</span>, <span class="highwire-citation-author" data-delta="1">Nicolas Rodrigue</span>, <span class="highwire-citation-author" data-delta="2">Sijmen E. Schoustra</span> and <span class="highwire-citation-author" data-delta="3">Rees Kassen</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">427-436;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.036152 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460558" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jeremy R. Dettman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, University of Ottawa, Ontario K1N 6N5, Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Centre for Advanced Research in Environmental Genomics, University of Ottawa, Ontario K1N 6N5, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeremy%2BR.%2BDettman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dettman%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJeremy%2BR.%2BDettman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jdettman@uottawa.ca" class="" data-icon-position="" data-hide-link-title="0">jdettman@uottawa.ca</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Nicolas Rodrigue </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology, Carleton University, Ottawa, Ontario K1S 5B6, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicolas%2BRodrigue%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rodrigue%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANicolas%2BRodrigue%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Sijmen E. Schoustra </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Laboratory of Genetics, Wageningen University, NL-6708 PB, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sijmen%2BE.%2BSchoustra%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schoustra%20SE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASijmen%2BE.%2BSchoustra%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Rees Kassen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, University of Ottawa, Ontario K1N 6N5, Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Centre for Advanced Research in Environmental Genomics, University of Ottawa, Ontario K1N 6N5, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rees%2BKassen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kassen%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARees%2BKassen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/2/427" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460558" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460558/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/2/427.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460558" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460558/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/2/427.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460558" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460558/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/2/427.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460558" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460558/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/2/427.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460558" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460558/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Knowledge of the number and nature of genetic changes responsible for adaptation is essential for understanding and predicting evolutionary trajectories. Here, we study the genomic basis of compensatory adaptation to the fitness cost of fungicide resistance in experimentally evolved strains of the filamentous fungus <em>Aspergillus nidulans</em>. The original selection experiment tracked the fitness recovery of lines founded by an ancestral strain that was resistant to fludioxonil, but paid a fitness cost in the absence of the fungicide. We obtained whole-genome sequence data for the ancestral <em>A. nidulans</em> strain and eight experimentally evolved strains. We find that fludioxonil resistance in the ancestor was likely conferred by a mutation in histidine kinase <em>nikA</em>, part of the two-component signal transduction system of the high-osmolarity glycerol (HOG) stress response pathway. To compensate for the pleiotropic negative effects of the resistance mutation, the subsequent fitness gains observed in the evolved lines were likely caused by secondary modification of HOG pathway activity. Candidate genes for the compensatory fitness increases were significantly overrepresented by stress response functions, and some were specifically associated with the HOG pathway itself. Parallel evolution at the gene level was rare among evolved lines. There was a positive relationship between the predicted number of adaptive steps, estimated from fitness data, and the number of genomic mutations, determined by whole-genome sequencing. However, the number of genomic mutations was, on average, 8.45 times greater than the number of adaptive steps inferred from fitness data. This research expands our understanding of the genetic basis of adaptation in multicellular eukaryotes and lays out a framework for future work on the genomics of compensatory adaptation in <em>A. nidulans</em>.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/fungicide-resistance" class="hw-term hw-article-keyword hw-article-keyword-fungicide-resistance" rel="nofollow">fungicide resistance</a></li><li class="kwd"><a href="/keyword/adaptive-walks" class="hw-term hw-article-keyword hw-article-keyword-adaptive-walks" rel="nofollow">adaptive walks</a></li><li class="kwd"><a href="/keyword/trade-offs" class="hw-term hw-article-keyword hw-article-keyword-trade-offs" rel="nofollow">trade-offs</a></li><li class="kwd"><a href="/keyword/fludioxonil" class="hw-term hw-article-keyword hw-article-keyword-fludioxonil" rel="nofollow">fludioxonil</a></li></ul><p id="p-3">A long-standing question in evolutionary biology concerns the number of genetic changes responsible for adaptation. A common view was that adaptation was underlain by many genes, each having small individual effects on fitness (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Fisher 1930</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Orr 1998</a>). This interpretation initially gained wide acceptance, partly because there was little evidence with which it could be confronted (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Orr 2005</a>). A powerful approach for the study of the genetic basis of adaptation is through microbial selection experiments, especially when coupled with cost-effective whole-genome sequencing (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Elena and Lenski 2003</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Long <em>et al.</em> 2015</a>). Not only can adaptive mutations be tracked in real time as they arise, but the number and nature of genetic changes underlying adaptive evolution can be determined directly.</p><p id="p-4">A compelling result to emerge from this field of work is that much of the fitness gain associated with an adaptive walk (sequential substitution of beneficial mutations by selection) is attributable to a small number of genetic changes (<em>e.g.</em>, <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Wichman <em>et al.</em> 1999</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Riehle <em>et al.</em> 2001</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Conrad <em>et al.</em> 2009</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Schoustra <em>et al.</em> 2009</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Anderson <em>et al.</em> 2010</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Kvitek and Sherlock 2011</a>). A recent review, for example, showed that the number of mutations fixed in microbial selection experiments that have reached a fitness plateau is typically between one and four (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Kassen 2014</a>). This result is consistent with the distribution of fitness effects among beneficial mutations being highly variable, such that in large populations, initial increases in fitness are associated with mutations that have large beneficial effects. Nonetheless, this result likely represents an underestimate because many analyses infer the number of fixation events from fitness trajectories or genetic crosses rather than whole-genome sequencing. Moreover, most microbial experiments are conducted using very large population sizes that violate one of the key assumptions of models of adaptation: that beneficial mutations are rare events that, when they occur, sweep nearly instantaneously to fixation. Consequently, the number of genetic changes underlying adaptive walks may be substantially higher than previous estimates have inferred.</p><p id="p-5">To gain a deeper understanding of the genetic basis of adaptation, we sequenced the genomes of experimentally evolved lines of the filamentous fungus <em>Aspergillus nidulans</em>. These lines were generated during a recently reported selection experiment (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Schoustra <em>et al.</em> 2009</a>) in which 118 selection lines were founded by a common ancestor (strain WG615) that carried a mutation that conferred resistance to the fungicide fludioxonil. As is often the case, a pleiotropic effect of the resistance mutation was a strong fitness cost in the absence of the fungicide (46% reduced growth rate; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Schoustra <em>et al.</em> 2006</a>). To allow for compensatory evolution, the populations were propagated in a fungicide-free environment for 800 generations with two different effective population sizes. The recovery of fitness was tracked during the course of the selection experiment, and all lineages reached a fitness plateau by the end, suggesting that the bulk of fitness recovery had been accomplished. From data on the fitness trajectories, a novel maximum likelihood framework and genetic crosses were used to infer the number of beneficial, compensatory mutations. The main finding was that adaptive walks tended to be rather short, with an overall mean of only 2.20 steps (<a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Schoustra <em>et al.</em> 2009</a>).</p><p id="p-6">The exact molecular mode of action of phenylpyrrole fungicides such as fludioxonil has yet to be elucidated, but they are known to act through the disruption of the HOG response pathway (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Furukawa <em>et al.</em> 2007</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hagiwara <em>et al.</em> 2007</a>, <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">2009</a>, <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">2013</a>). This pathway involves a two-component system and a mitogen-activated protein kinase (MAPK) signaling cascade, in which a histidine kinase serves as a sensor for multiple stressors and transmits the signal downstream via a response regulator to activate the MAPK. Previous work with <em>Aspergillus</em> has shown this stress response pathway to be activated by exposure to fludioxonil (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Furukawa <em>et al.</em> 2007</a>; <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Hagiwara <em>et al.</em> 2009</a>) and that knocking out components of this pathway results in high levels of fludioxonil resistance (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Hagiwara <em>et al.</em> 2007</a>, <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">2013</a>). Genes involved in the HOG MAPK pathway are therefore prime candidates for fludioxonil resistance determinants.</p><p id="p-7">Here, we sequence the 30.5 Mb genomes of the ancestral <em>A. nidulans</em> strain and eight experimentally evolved strains (<a id="xref-ref-45-4" class="xref-bibr" href="#ref-45">Schoustra <em>et al.</em> 2009</a>) to investigate the evolutionary genomics of compensatory adaptation to the fitness cost of fungicide resistance. First, since the genetic basis of the fungicide resistance is not known, we identify candidate mutations for resistance in the ancestral strain. We then identify mutations within the experimentally evolved strains and analyze the characteristics of the mutated genes to determine how they may be related to compensatory, adaptive evolution. Finally, we use these data to determine the relationship between the number of genomic mutations and the predicted number of adaptive steps estimated from fitness data, and we also test hypotheses about how effective population size impacts the number and nature of mutations fixed during adaptation.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Strains</h3><p id="p-8">The ancestral <em>A. nidulans</em> strain is fludioxonil-resistant WG615 (<em>fldA1</em>, <em>wA3</em>, <em>pyroA4</em>, and <em>veA1</em>). When growing in the absence of fungicide, resistance confers a cost of ∼46% relative to the sensitive strain from which WG615 was derived (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Schoustra <em>et al.</em> 2006</a>). All evolved strains were derived from WG615 through 800 generations of experimental evolution and adaptation to a permissive growth environment [complete medium (CM) without fungicide; <a id="xref-ref-45-5" class="xref-bibr" href="#ref-45">Schoustra <em>et al.</em> 2009</a>]. The wild-type sensitive strain was not evolved because, in previous experiments (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Schoustra <em>et al.</em> 2005</a>, <a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">2006</a>), it showed little evidence of adaptation to the permissive growth environment, suggesting that the sensitive strain already resides close to a fitness optimum. Every 80 generations, experimental lines were put through a bottleneck of either ∼500 or 50,000 spores, corresponding to small (S) and large (L) effective population size treatments, respectively. For whole-genome sequencing, we chose four strains from each of the small and large population size treatments. Representative strains were chosen to cover the full range of observed fitness values and predicted step numbers (Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036152/-/DC1/TableS1.pdf">Table S1</a>). The eight evolved strains characterized here (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036152/-/DC1/TableS1.pdf">Table S1</a>) were all put through a single-conidium transfer to ensure that a single haploid genotype was being sequenced. From the results of <a id="xref-ref-45-6" class="xref-bibr" href="#ref-45">Schoustra <em>et al.</em> (2009)</a>, the number of predicted adaptive steps per strain ranged from one to three.</p></div><div id="sec-3" class="subsection"><h3>DNA extraction and sequencing</h3><p id="p-9">Strains were grown on CM agar plates and genomic DNA was extracted using the QIAGEN DNeasy Blood &amp; Tissue Kit. Sequence data were generated using the Illumina Hi-Seq platform with paired-end reads (BC Cancer Agency). Reads generated for the evolved and ancestral strains were 75 and 100 bp in length, respectively.</p></div><div id="sec-4" class="subsection"><h3>Reference-based mapping</h3><p id="p-10">The <em>A. nidulans</em> reference genome was from strain FGSC A4 and is publicly available from NCBI (PRJNA13961). The assembly is divided into scaffolds (NT_106999 to NT_1070015) corresponding roughly to the 16 arms of the eight <em>A. nidulans</em> chromosomes. A modified version of the bioinformatics pipeline described in <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Dettman <em>et al.</em> (2012)</a> was used for sequence data analyses. In brief, reads were trimmed using Popoolation (ver. 1.1, <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Kofler <em>et al.</em> (2011)</a>) with a phred quality threshold of 20 and a minimum retention length of 75% of original read length. Trimmed reads were mapped to the FGSC A4 reference genome using Novoalign (ver. 2.07).</p></div><div id="sec-5" class="subsection"><h3>Coverage</h3><p id="p-11">Coverage along scaffolds was calculated in 1 kb intervals and regions of high or low coverage were identified by bins with &gt;1.8 × or &lt;0.20 × the average coverage of the entire genome of that sample. Regions were confirmed by manual inspection of aligned reads and by results from other insertion/deletion (indel) calling programs (Pindel, ver. 0.2.4, <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Ye <em>et al.</em> 2009</a>; Breseq, ver. 0.16, <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Barrick <em>et al.</em> 2014</a>). Areas of high or missing coverage were reported if they were &gt;500 bp in length, and regions that were within 200 bp of each other were collapsed into one.</p></div><div id="sec-6" class="subsection"><h3>Variant calling</h3><p id="p-12">Single nucleotide polymorphisms (SNPs) and indels relative to the reference were called using Samtools (ver. 0.1.19, <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Li <em>et al.</em> 2009</a>; minimum coverage = 3 ×), VarScan (ver. 2.3.5, <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Koboldt <em>et al.</em> 2012</a>; minimum coverage = 3 ×, <em>P</em> &gt; 0.95), and Pindel (ver. 0.2.4). Results from the different methods were compared for cross-verification of mutation calls. A variant shared by WG615 and all evolved lines indicates that that variant was present in the ancestral WG615 strain at the start of the experiment. Variants that arose during the selection experiment were those that were absent in WG615 but present in an evolved line. For variant calling, we filtered out the regions of low quality assembly in the reference genome, and correspondingly, regions of low mapping quality in the query genomes. Therefore, we may have underestimated the true number of sequence differences between the strains. Sequence data and positional alignments were inspected manually to confirm the called mutations in evolved lines. Significance of Pearson’s correlations were determined using the Student’s <em>t</em>-distribution.</p></div><div id="sec-7" class="subsection"><h3>Annotation and functional analyses</h3><p id="p-13">Genomic variation was annotated using SnpEff (ver. 3.3, <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Cingolani <em>et al.</em> 2012</a>, minimum coverage = 3 ×, minimum base quality = 20) to determine the types of mutation and what genes were affected. Amino acids were classified as hydrophilic uncharged (S, N, T, and Q), aliphatic uncharged (A, G, V, L, and I), nonpolar uncharged (C, M, and P), acidic (D and E), basic (K, R, and H), and aromatic (F, Y, and W). Amino acid changes within and between these classes were considered conservative and radical, respectively. Mutations that caused codon indels, premature stop codons, or frame-shifts were also considered radical. Intergenic mutations were investigated further if they occurred in a 5′-untranslated region or within 100 bp upstream of the transcriptional start site of a gene. Gene functions were inferred, in part, from automated Gene Ontology (GO) information. To summarize the functional information using broad, high-level GO terms, we used the GO Slim sets provided by the <em>Aspergillus</em> Genome Database (<a href="http://www.aspergillusgenome.org">www.aspergillusgenome.org</a>). For enrichment analyses, we aimed to identify the main functional classes, so the results were filtered to retain only the GO Slim terms that were represented by at least 5% of the queried genes. A GO Slim term was considered overrepresented if the observed gene number was ≥ 1.2 times the expected gene number. Statistical significance of overrepresentation was calculated using one-tailed Chi-squared tests and the Benjamini–Hochberg false discovery rate (set to 0.15) correction for multiple comparisons. Given that “Cellular Component,” “Biological Process,” and “Molecular Function” GO Slim annotations were missing for 57, 56, and 54% of genes, respectively, we performed more comprehensive manual annotation of all affected genes using information from other sources, such as similarity and literature searches.</p></div><div id="sec-8" class="subsection"><h3>Data availability</h3><p id="p-14">Sequence data are available from the NCBI Short Read Archive under BioProject PRJNA356622.</p></div></div><div class="section results-discussion" id="sec-9"><h2 class="">Results and Discussion</h2><div id="sec-10" class="subsection"><h3>Sequence data</h3><p id="p-15">Sequence libraries contained an average of 35.8 million reads per sample, providing an expected 89.4 × coverage of the 30.5 Mb <em>A. nidulans</em> genome (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Galagan <em>et al.</em> 2005</a>). To reduce the number of false positive variant calls in downstream analyses, sequence reads were trimmed based on quality criteria. Trimming reduced the average read number and read length by only 4.3 and 1.1%, respectively, indicating that the overall sequence quality was relatively high (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036152/-/DC1/TableS1.pdf">Table S1</a>). All subsequent analyses were based on trimmed reads.</p></div><div id="sec-11" class="subsection"><h3>Coverage distribution</h3><p id="p-16">An average of 93.7% of reads from the ancestor WG615 and the eight evolved lines were successfully mapped to the FGSC A4 reference genome, resulting in an average coverage of 83.1 × (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036152/-/DC1/TableS1.pdf">Table S1</a>). Comparison of fold-coverage among and along scaffolds revealed that WG615 and all descendent, evolved lines shared numerous regions of missing coverage relative to ancestral strain FGSC A4 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036152/-/DC1/TableS2.pdf">Table S2</a>). These absent regions could be up to 32.5 kb in length but still only comprised 1.7% of the reference genome (504.4 kb total). Also, some genome regions of poor alignment and/or low mapping quality were identified (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036152/-/DC1/TableS3.pdf">Table S3</a>). It is not clear if these areas represent genomic regions of poor reference assembly, repetitive regions, or regions of genome plasticity. Regardless, these low quality regions sum to only 82.6 kb, or 0.27% of the reference genome, and therefore have a negligible effect on our mutation detection ability. When filtering for changes that arose during the selection experiment, no evidence for large scale indels or aneuploidies were found.</p></div><div id="sec-12" class="subsection"><h3>Variant calls for ancestral WG615 strain</h3><p id="p-17">Variant calling revealed that the ancestral WG615 strain differed from FGSC A4 reference by 740 SNPs and 456 small indels, confirming that these two strains have moderate sequence divergence (0.004%). WG615 is described as having mutations in the <em>wA</em> (AN8209), <em>pyroA</em> (AN7725), and <em>veA</em> (AN1052) genes, so we examined these genes for quality control purposes. Mutations were confirmed for <em>wA</em> and <em>pyroA</em>, and WG615 and FGSC A4 shared the same <em>veA</em> mutant allele, as predicted.</p></div><div id="sec-13" class="subsection"><h3>Candidate genes for fungicide resistance in ancestral WG615 strain</h3><p id="p-18">To identify putative targets of fludioxonil resistance, we took a two-step strategy. We first filtered sequence differences between the fludioxonil-resistant WG615 strain and the fludioxonil-sensitive FGSC A4 strain to retain only those that caused protein sequence changes, under the assumption that these were the most likely to cause changes in gene function and phenotype. Filtering narrowed the field to 390 mutations, of which 237 had GO annotation information available. We screened for genes involved in the HOG MAPK pathway because they are prime candidates for fludioxonil resistance determinants. Just two of these candidate genes (AN2262 and AN4479) had described osmoregulation functions, and only the latter gene, AN4479, is known to respond to fungicides. Notably, AN4479 encodes <em>nikA</em>, the histidine kinase sensor of the two-component system of the HOG MAPK pathway.</p><p id="p-19">Second, we reviewed our previous genetic analyses (<a id="xref-ref-44-4" class="xref-bibr" href="#ref-44">Schoustra <em>et al.</em> 2006</a>) and found that the resistance mutation was located on the left arm of <em>A. nidulan</em>s linkage group III, between the centromere and <em>cys2</em> (AN4769; also known as sC or <em>sC</em>0). Examination of our genomic data revealed that only 12 genes in this genomic region had predicted protein differences between the sensitive reference and resistant WG615 ancestor. Inspection of annotation information for these 12 candidate genes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036152/-/DC1/TableS4.pdf">Table S4</a>) revealed that only one had known functions related to osmoregulation or fungicide resistance: that one gene was <em>nikA</em>.</p><p id="p-20">Three lines of evidence support that <em>nikA</em> is the target of fludioxonil resistance in WG615. First, <em>nikA</em> is essential for fludioxonil-based growth inhibition (<a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Hagiwara <em>et al.</em> 2007</a>, <a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">2013</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Vargas-Perez <em>et al.</em> 2007</a>) and is responsible for transmitting the phosphorelay signal down the HOG MAPK pathway (<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Hagiwara <em>et al.</em> 2009</a>). Second, <em>nikA</em> mutants with increased resistance to fludioxonil are characterized by strong vegetative growth defects in the absence of the fungicide (<a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">Hagiwara <em>et al.</em> 2007</a>; <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Vargas-Perez <em>et al.</em> 2007</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">De Souza <em>et al.</em> 2013</a>), a fitness trade-off clearly observed in our fludioxonil-resistant strain (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Schoustra <em>et al.</em> 2005</a>, <a id="xref-ref-45-7" class="xref-bibr" href="#ref-45">2009</a>). Third, the WG615 <em>nikA</em> mutation is a nonsynonymous substitution causing a radical amino acid change (K→N) at residue 854, which is within the highly conserved, signal transduction histidine kinase domain. Although formal genetic analyses are needed for full confirmation, the evidence points to the <em>nikA</em> mutation as the determinant of fludioxonil resistance.</p></div><div id="sec-14" class="subsection"><h3>Derived mutations in compensatory evolution lines</h3><p id="p-21">We identified 144 SNPs and indels that differed between the ancestor and the compensatory evolution lines after 800 generations of selection (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036152/-/DC1/TableS5.pdf">Table S5</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036152/-/DC1/TableS6.pdf">Table S6</a>). An average of 16.75 SNPs and 1.25 small indels were detected in the ∼30.5 Mb genome of each evolved line. Derived mutations were relatively evenly distributed throughout the genome (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>), with only a few locations containing multiple mutations clustered together. Only one derived structural variant was called: a ∼103 kb inversion was predicted in strain 25S (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036152/-/DC1/TableS6.pdf">Table S6B</a>). Substitution rates per chromosome did not differ drastically and ranged from 3.32 to 11.66 × 10<sup>−10</sup> per nucleotide per generation (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036152/-/DC1/TableS7.pdf">Table S7</a>). The overall substitution rate was 7.58 × 10<sup>−10</sup> per nucleotide per generation, similar to rates reported for other fungi such as yeast (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Lynch <em>et al.</em> 2008</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Nishant <em>et al.</em> 2010</a>). Approximately 50% of the <em>A. nidulans</em> genome is coding, yet over 57% of the observed mutations (83/144) were located within coding regions (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). This difference between observed and expected proportions in coding regions is not statistically significant (Chi-square, <em>P</em> &lt; 0.194), but the overrepresentation of coding mutations is consistent with positive selection in our experiment.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470296/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470296/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Types and numbers of derived mutations in evolved lines</span><div class="sb-div caption-clear"></div></div></div><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/427/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Cumulative number of derived mutations and their locations within the A. nidulans genome. Data for all eight evolved lines were combined. Chromosome (Chr.) ends are indicated by solid lines and centromere positions are indicated by dashed lines. Chromosome numbers and associated genome sequence scaffolds (FGSC A4) are listed at the top." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1317270871" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Cumulative number of derived mutations and their locations within the &lt;em&gt;A. nidulans&lt;/em&gt; genome. Data for all eight evolved lines were combined. Chromosome (Chr.) ends are indicated by solid lines and centromere positions are indicated by dashed lines. Chromosome numbers and associated genome sequence scaffolds (FGSC A4) are listed at the top.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/427/F1.medium.gif" width="440" height="217"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/2/427/F1.medium.gif" width="440" height="217"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/427/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/427/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470309" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-23" class="first-child">Cumulative number of derived mutations and their locations within the <em>A. nidulans</em> genome. Data for all eight evolved lines were combined. Chromosome (Chr.) ends are indicated by solid lines and centromere positions are indicated by dashed lines. Chromosome numbers and associated genome sequence scaffolds (FGSC A4) are listed at the top.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Genes mutated during compensatory adaptation</h3><p id="p-24">The HOG MAPK pathway in <em>A. nidulans</em> senses and responds to multiple forms of potentially harmful stimuli, such as osmotic stress, oxidative stress, DNA damage, and fungicides (<a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Furukawa <em>et al.</em> 2007</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Miskei <em>et al.</em> 2009</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Balazs <em>et al.</em> 2010</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Jaimes-Arroyo <em>et al.</em> 2015</a>). If the <em>nikA</em> resistance-conferring mutation disrupts the normal function of the HOG MAPK signaling pathway, there are likely to be pleiotropic effects on several other physiological processes and phenotypes (<em>e.g.</em>, <a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">Vargas-Perez <em>et al.</em> 2007</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Duran <em>et al.</em> 2010</a>). Our observation of a substantial fitness cost associated with resistance is consistent with this idea. Recent studies of compensatory adaptation in yeast (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Szamecz <em>et al.</em> 2014</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Filteau <em>et al.</em> 2015</a>) found that derived genetic changes were often specific to the original functional defect, and that compensatory evolution targeted genes that were functionally related to the disrupted gene. Thus, the obvious prediction was that the compensatory fitness gains observed here in <em>A. nidulans</em> would be caused by secondary modification of the HOG MAPK pathway.</p><p id="p-25">Genome-wide analyses in <em>A. nidulans</em> are hampered by a lack of complete functional annotation for genes, particularly when compared to bacterial genomes or more intensively studied fungi like yeast. For example, the most recent <em>A. nidulans</em> genome snapshot indicates that only ∼11% of ORFs are verified, while the rest remain uncharacterized. Given the high frequency of missing data, our power to detect statistically significant trends was quite low. However, such results are still useful for identifying potential functional pathways under selection and candidate genes to target in future research efforts.</p><p id="p-26">For <em>A. nidulans</em>, the main source for functional information is GO data. We used GO information (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036152/-/DC1/TableS5.pdf">Table S5</a>), summarized using GO Slim Mapper (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036152/-/DC1/TableS8.pdf">Table S8</a>), to identify the main functional categories of genes affected by derived mutations. To determine if particular functional classes were more prevalent than expected by chance, the frequencies of genes belonging to specific GO Slim terms were compared between the mutated gene list and that for the rest of the genome. Despite the fact that less than half of the mutated genes (38/85) had GO information, we were still able to identify three GO Slim terms that were significantly overrepresented (after correction for multiple comparisons) in the mutated gene list (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). The relevance of these three important functional classes, and examples of specific genes, are discussed below. Interestingly, the enriched functional classes were consistent with the HOG MAPK pathway’s role as a stress response pathway with phosphorelay-based transduction of signals to downstream elements.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/427/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Summary of the Gene Ontology (GO) Slim terms that were overrepresented (&gt;1.2-fold) in the list of genes that were mutated during compensatory adaptation. Terms that were significantly overrepresented after correction for multiple comparisons are marked by an asterisk (*)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1317270871" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Summary of the Gene Ontology (GO) Slim terms that were overrepresented (&gt;1.2-fold) in the list of genes that were mutated during compensatory adaptation. Terms that were significantly overrepresented after correction for multiple comparisons are marked by an asterisk (*).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/427/F2.medium.gif" width="440" height="269"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/2/427/F2.medium.gif" width="440" height="269"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/427/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/427/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470499" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-27" class="first-child">Summary of the Gene Ontology (GO) Slim terms that were overrepresented (&gt;1.2-fold) in the list of genes that were mutated during compensatory adaptation. Terms that were significantly overrepresented after correction for multiple comparisons are marked by an asterisk (*).</p><div class="sb-div caption-clear"></div></div></div><div id="sec-16" class="subsection"><h4>Response to stress:</h4><p id="p-28">Genes associated with “response to stress” (GO 6950) were overrepresented 2.1-fold among derived mutations (Chi-square, <em>P</em> &lt; 0.003). We examined the affected genes and found some that were specifically associated with the HOG MAPK pathway itself (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). The most obvious example was the <em>atfA</em> gene (AN2911), which encodes a transcription factor in the HOG MAPK pathway and regulates responses to a broad spectrum of environmental stressors (<a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">Hagiwara <em>et al.</em> 2009</a>; <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Balazs <em>et al.</em> 2010</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Lara-Rojas <em>et al.</em> 2011</a>). The derived <em>atfA</em> mutation was a radical nonsynonymous change (Q→P) of a conserved amino acid within the functionally important basic-region leucine zipper (bZIP) domain. Another mutated gene with a direct role in the central stress response pathway is <em>mpkC</em>, which encodes a second MAPK that is activated by the same upstream MAPKK as <em>hogA</em> (= <em>sakA</em>) MAPK. Overexpression of <em>mpkC</em> has been shown to suppress the high-osmolarity sensitivity observed in <em>hogA</em>-deletion strains (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Furukawa <em>et al.</em> 2005</a>), indicating that these two homologous genes have overlapping roles in a shared pathway. No compensatory mutations were observed in <em>nikA</em> itself.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470315/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470315/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Examples of mutated genes with functions relevant to stress response or membrane and cell wall</span><div class="sb-div caption-clear"></div></div></div><p id="p-30">Several mutated genes (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>) are known to be expressed in <em>Aspergillus</em> at increased levels in response to osmotic stress or stress-inducing chemicals such as menadione and camptothecin (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Malavazi <em>et al.</em> 2006</a>). Camptothecin is a cytotoxic quinoline alkaloid that contains a pyrrole moiety, which is a defining characteristic of phenylpyrrole fungicides such as fludioxonil. This shared chemical structure may explain why both compounds appear to activate similar stress response pathways.</p></div><div id="sec-17" class="subsection"><h4>Signal transduction:</h4><p id="p-31">Genes associated with “signal transduction” (GO 7165) were overrepresented 2.5-fold among derived mutations (Chi-square, <em>P</em> &lt; 0.029), many of which had direct roles in the phosphorylation-based transduction of stress response signals. For example, a radical nonsynonymous mutation (S→F) was observed in <em>ppzA</em>, the catalytic subunit of protein phosphatase Z. This serine/threonine phosphatase is known to modulate the response of fungi to oxidative and other stresses (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Leiter <em>et al.</em> 2012</a>). A radical nonsynonymous mutation (T→I) was also found in the histidine kinase <em>phkB</em>. Homologs of <em>phkB</em> are stress-activated sensor kinases of two-component systems, indicating a role in phosphorylation-based signal transduction, and a recent study with <em>Aspergillus</em> found that <em>phkB</em> expression increased greatly in response to fludioxonil treatment and osmotic shock in a manner dependent on the HOG pathway (<a id="xref-ref-23-4" class="xref-bibr" href="#ref-23">Hagiwara <em>et al.</em> 2013</a>).</p></div><div id="sec-18" class="subsection"><h4>Plasma membrane:</h4><p id="p-32">Genes associated with the “plasma membrane” (GO 5886) were overrepresented 2.3-fold among derived mutations (Chi-square, <em>P</em> &lt; 0.026). Remodeling of the fungal membrane and cell wall components is a common response to a wide range of stressors because it helps maintain cell wall integrity and allow for survival and growth (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Ene <em>et al.</em> 2015</a>). We found derived mutations in a number of genes (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>) with functions in the synthesis and breakdown of fungal cell wall components, such as glucans (<em>agsB</em>, <em>bglM</em>, and AN3504) and chitin (<em>csnA</em> and AN8765; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">de Groot <em>et al.</em> 2009</a>). For example, a nonsynonymous mutation was observed in the main catalytic domain of <em>agsB</em>, which encodes an α-1,3 glucan synthase with roles in osmotic stress tolerance and cell wall integrity (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Futagami <em>et al.</em> 2011</a>). The mutated gene list also contained several membrane-bound transporters with specificities for a range of compounds. Transporters are known to mediate resistance to fungicides and other toxic compounds in many fungi, including <em>Aspergillus</em> (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Del Sorbo <em>et al.</em> 2000</a>).</p></div><div id="sec-19" class="subsection"><h4>Intergenic mutations:</h4><p id="p-33">Intergenic mutations can alter 5′-untranslated regions (5′-UTRs) or promoter sequences, potentially resulting in changes in gene expression that affect fitness. We analyzed the 57 intergenic mutations (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036152/-/DC1/TableS6.pdf">Table S6</a>) and found five that were within 5′-UTRs, and eight that were within potential promoter regions (defined as &lt; 100 bp from the transcription start site of a gene). None of the potentially affected genes had known roles in the HOG MAPK pathway, osmotic stress tolerance, or phenylpyrrole resistance.</p></div></div><div id="sec-20" class="subsection"><h3>Parallel evolution</h3><p id="p-34">Parallel evolution is the repeated evolution of the same genotype or phenotype in independently evolving populations. At the level of fitness, all of the lines we studied evolved in parallel; that is, they all increased in fitness substantially when compared to their ancestor (<a id="xref-ref-45-8" class="xref-bibr" href="#ref-45">Schoustra <em>et al.</em> 2009</a>). Only one of the derived, mutated genes was hit in multiple evolved lines (uncharacterized gene AN2078, predicted RNA polymerase II transcription cofactor activity; strain 9S and 8L, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036152/-/DC1/TableS5.pdf">Table S5</a>), providing little evidence for parallel evolution at the gene level. There may be multiple evolutionary pathways to increased fitness in the permissive environment, yet empirical evidence suggests that compensatory evolution would target pathways that were functionally related to the disrupted pathway that caused the original fitness deficit (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Szamecz <em>et al.</em> 2014</a>; <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Filteau <em>et al.</em> 2015</a>).</p><p id="p-35">Current understanding of the HOG pathway [KEGG (Kyoto Encyclopedia of Genes and Genomes), <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Furukawa <em>et al.</em> 2005</a>; <a id="xref-ref-22-5" class="xref-bibr" href="#ref-22">Hagiwara <em>et al.</em> 2009</a>] indicates that it involves 22 of the 10,667 protein-coding genes in <em>A. nidulans</em>. We observed derived mutations in only two of these genes (mpkC and atfA), but the probability of 2/87 derived mutations occurring randomly in this gene subset is extremely small (∼3.57 × 10<sup>−6</sup>). Thus, compensatory adaptation was associated with the HOG pathway more commonly than by chance alone.</p><p id="p-36">Previous analyses of multivariate phenotypic divergence among these <em>A. nidulans</em> lines suggested that fitness gains were achieved through distinct phenotypic routes (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Schoustra <em>et al.</em> 2012</a>), but the genetic causes were unknown. The lack of strong parallel evolution provided by our sequence data could indicate that the number of potential genes that allow for compensation is large, or it could be an artifact of our low sample size and lack of complete functional information. Future characterization of mutated genes with unknown function will determine if they interact with the HOG pathway, or if they are generally adaptive and completely independent from the pathway involved with fungicide resistance.</p></div><div id="sec-21" class="subsection"><h3>Comparison of step number for fitness-based and sequence-based estimates</h3><p id="p-37">A general result to emerge from genetic studies of adaptation in microbial experiments has been that just a few mutations are usually responsible for the bulk of adaptation achieved during an adaptive walk (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Kassen 2014</a>). Indeed, by analyzing fitness data with a maximum likelihood-based approach, we previously estimated that these evolved lines accumulated an average of 2.13 (SE = 0.30) fitness-increasing steps during their adaptive walks (<a id="xref-ref-45-9" class="xref-bibr" href="#ref-45">Schoustra <em>et al.</em> 2009</a>). The results from our genomic analyses reveal, at first glance, a strikingly different picture. An average of 18.0 (SE = 2.48) mutations were detected in the genome of each evolved line, or 8.45 times greater than the number of steps inferred from fitness trajectory data. The discrepancy between the number of fitness-based adaptive steps and the total number of sequence-based genomic mutations is likely due to two, nonexclusive reasons.</p><p id="p-38">First, not all mutations observed in the evolved lines are adaptive. Some nonadaptive mutations (neutral, nearly neutral, or even slightly deleterious) may become fixed in the population by chance alone, or may hitch-hike to fixation alongside a beneficial mutation. Hitch-hiking is expected to be especially prevalent because reproduction in this experiment was strictly asexual, leading to complete linkage between beneficial mutations and any others that arose. This effect can be quite pronounced: in experimental evolution studies with yeast, for example, typically less than half of the derived genomic mutations have observed adaptive effects (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Anderson <em>et al.</em> 2010</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Kvitek and Sherlock 2011</a>).</p><p id="p-39">Second, not all adaptive mutations are detectable in fitness assays. Large fitness changes will be identified, but adaptive mutations that confer small fitness increases may be below the detection threshold of the phenotypic assays. Also, if multiple beneficial mutations arise in the same genome and fix together (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Park and Krug 2007</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Jerison and Desai 2015</a>), these would be detected as a single fitness-increasing event. Moreover, fitness measures of mycelial growth rate do not assess all components of total fitness. Mutations that have adaptive effects on other fitness components (<em>e.g.</em>, mycelial density, sporulation) would not be detected. In these cases, fitness-based step numbers would underestimated.</p><p id="p-40">Taken together, these explanations suggest that we should be more conservative in our estimate of the number of mutations substituted by focusing attention only on those mutations that are likely to be adaptive. Experimental determination of the individual fitness effects of all 144 derived <em>A. nidulans</em> mutations, alone and in combination, is a prodigious task and beyond the scope of the present paper, so we inferred potential adaptive effects based on the mutation location and context within genes. When restricted to mutations occurring within protein-coding DNA, evolved lines had an average of 10.38 (SE = 1.27) mutations, corresponding to 4.87 times the estimated step number. Similarly, when restricted to only those mutations that change the protein sequence, evolved lines had an average of 6.88 (SE = 1.04), or 3.23 times the estimated step number. When restricted to radical amino acid changes (mean = 6.00, SE = 0.96), the difference was only 2.82 times.</p><p id="p-41">Reassuringly, there was a positive correlation between inferred step number and the various measures of genomic mutation number for the evolved lines (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> and <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). Given the low statistical power afforded by a sample size of only eight, it was not surprising that these positive relationships were all statistically nonsignificant. Deviation from linearity was mainly caused by the outlier strain 25S, which had a high predicted number of steps but low number of genomic mutations. Examination of the data revealed that strain 25S had no detected nonsynonymous mutations, but was the only strain that had a predicted structural variant: an inversion that encompassed 13 genes, most of which are in a secondary metabolite gene cluster (AN7884 cluster; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Andersen <em>et al.</em> 2013</a>). This inversion could be contributing to fitness increases, but is not counted toward genomic mutation numbers. When outlier strain 25S was removed from the analyses, most mutation number measures were significantly correlated with step number (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/427/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Positive relationship between the estimated step number and number of genomic mutations (SNPs and indels combined) for eight evolved strains (R = 0.56, P &gt; 0.14). Relationship is significant when outlier strain 25S (shown in gray) is removed (R = 0.88, P &lt; 0.009). Indel, insertion/deletion; SNP, single nucleotide polymorphism." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1317270871" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Positive relationship between the estimated step number and number of genomic mutations (SNPs and indels combined) for eight evolved strains (&lt;em&gt;R&lt;/em&gt; = 0.56, &lt;em&gt;P&lt;/em&gt; &gt; 0.14). Relationship is significant when outlier strain 25S (shown in gray) is removed (&lt;em&gt;R&lt;/em&gt; = 0.88, &lt;em&gt;P&lt;/em&gt; &lt; 0.009). Indel, insertion/deletion; SNP, single nucleotide polymorphism.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/427/F3.medium.gif" width="440" height="357"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/2/427/F3.medium.gif" width="440" height="357"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/427/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/427/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470428" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-42" class="first-child">Positive relationship between the estimated step number and number of genomic mutations (SNPs and indels combined) for eight evolved strains (<em>R</em> = 0.56, <em>P</em> &gt; 0.14). Relationship is significant when outlier strain 25S (shown in gray) is removed (<em>R</em> = 0.88, <em>P</em> &lt; 0.009). Indel, insertion/deletion; SNP, single nucleotide polymorphism.</p><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470447/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470447/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Correlations between number of estimated steps and other variables</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-22" class="subsection"><h3>Effect of small <em>vs.</em> large population size</h3><p id="p-45">The original selection experiment investigated the effects of population size on the properties of adaptive walks. Small (S) and large (L) effective population size treatments were imposed by bottlenecks of either 500 or 50,000 individuals, respectively, at every transfer. The effect of genetic drift was exaggerated in S populations, leading to lower mean fitness and a more rapid increase in the variance of fitness during the experiment (<a id="xref-ref-45-10" class="xref-bibr" href="#ref-45">Schoustra <em>et al.</em> 2009</a>). What effect does a 100-fold difference in population size have on the spectrum of genomic changes? While we have relatively little statistical power with which to detect effects, we do find some intriguing trends.</p><p id="p-46">First, the variance in total mutation number for the S group is over twice as large as that for the L group (68.9 and 32.2, respectively), although the difference is not statistically significant (<em>F</em>-test, <em>P</em> &gt; 0.27). Similar patterns are found when each mutation class is analyzed separately. Second, the correlation between fitness-based and sequence-based step number estimates is significantly positive for the L group (<em>R</em> = 0.966, <em>P</em> &lt; 0.034) but not for the S group (<em>R</em> = 0.181, <em>P</em> &gt; 0.81), consistent with more pronounced effects of drift at smaller population sizes. Third, we might expect that, after controlling for differences in total mutation number per line, the S lines should have accrued more putatively neutral mutations than the <em>L</em> lines. To test this prediction, we calculated the proportion of coding SNPs that were synonymous (silent) and found that S lines did have a greater mean than L lines [0.497 (SE = 0.191) and 0.323 (SE = 0.056), respectively]. Similar results were obtained when we compared the proportion of all mutations that were putatively neutral [intergenic + intron + synonymous; <em>S</em> lines = 0.644 (SE = 0.077), L lines = 0.592 (SE = 0.032)]. While these trends are in the predicted direction (<em>S</em> &gt; <em>L</em>), the differences were not significant (one-tailed <em>t</em>-test; <em>P</em> &gt; 0.21 and <em>P</em> &gt; 0.28, respectively), likely due to the low statistical power afforded by only four lines per treatment.</p></div><div id="sec-23" class="subsection"><h3>Conclusions</h3><p id="p-47">Here, we examined the genomic basis of compensatory adaptation to the fitness cost of fungicide resistance in experimentally evolved strains of <em>A. nidulans</em>. Our work reveals three important insights into the genetics of adaptive evolution in this system.</p><p id="p-48">First, fludioxonil resistance in the ancestor was likely conferred by a mutation in the histidine kinase <em>nikA</em>, the sensor part of the two-component signal transduction phosphorelay system of the HOG MAPK stress response pathway.</p><p id="p-49">Second, to compensate for the pleiotropic negative effects of the resistance mutation, the subsequent fitness gains observed in the evolved lines were likely caused by secondary modification of the HOG MAPK pathway activity. Candidate genes for the compensatory fitness increases were significantly overrepresented by stress response functions, and some mutations were within genes specifically associated with the HOG MAPK pathway itself. We have generated a candidate list of adaptive mutations, however, further detailed genetic and phenotypic analyses are required to confirm the function and fitness effects of each mutation.</p><p id="p-50">Third, there was a positive relationship between the predicted number of adaptive steps, estimated from fitness data, and the number of genomic mutations, determined by whole-genome sequencing. It is clear, however, that fitness-based methods underestimate the number of genomic mutations because they miss mutations of small effect and cannot distinguish fixation events involving multiple mutations. On the other hand, sequencing-based methods likely overestimate the number of adaptive mutations, especially when recombination is absent, because many nonadaptive mutations rise to high frequency by hitchhiking alongside beneficial mutations.</p><p id="p-51">Interestingly, the proportion of mutations predicted to be adaptive in our <em>Aspergillus</em> experiment is substantially lower than comparable experiments (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Elena and Lenski 2003</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Dettman <em>et al.</em> 2012</a>; <a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Kassen 2014</a>; <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Long <em>et al.</em> 2015</a>). If we assume, very approximately, that the fitness-based estimates of step number have correctly identified all adaptive mutations, then our sequencing results suggest that, on average, only 12% of the genomic mutations are adaptive. When considering only those mutations that alter the protein sequence, 31% of mutations are adaptive. Such low proportions of adaptive mutations may be due to the higher fraction of the <em>Aspergillus</em> genome that is noncoding, compared to bacterial genomes. Also, the spatially structured, multicellular growth form of filamentous fungi may reduce the efficiency of selection. For example, growing as an interconnected mycelial network may hamper competition among contending mutations, and competitive success may depend upon where within the mycelium the new variant arises. These reasons may explain why the proportion of total mutations we predict to be adaptive in <em>Aspergillus</em> are low compared to those reported for other, typically unicellular, organisms. In most bacterial experiments, for example, the vast majority of mutations tend to be adaptive (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Dettman <em>et al.</em> 2012</a>). The generality of our results cannot be determined because, to date, only a few studies have applied the evolve-and-resequence approach to multicellular eukaryotes (<em>e.g.</em>, <em>Drosophila</em>, <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Burke <em>et al.</em> 2010</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Kang <em>et al.</em> 2016</a>).</p><p id="p-52">Our study with <em>Aspergillus</em> lays out a framework for future research on the genomics of compensatory adaptation. Now that we have identified some clear trends (<em>e.g.</em>, stress response and HOG pathway) based on analyses of only eight evolved lines, it will be interesting to see how these patterns hold when the sample size is expanded to dozens or even hundreds of evolved lines. Such increased sample sizes will mitigate our problems with lack of statistical power, and will also allow us to more thoroughly assess the degree of parallelism in genomic evolution during compensatory adaptation.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-53">We thank two anonymous reviewers for their comments and suggestions. This work was supported by the Canadian Institutes of Health Research and the Natural Sciences and Engineering Research Council of Canada.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-5"><p id="p-54">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036152/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.036152/-/DC1</a>.</p></li><li class="fn" id="fn-6"><p id="p-55"><em>Communicating editor: J. C. Fay</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-10-03"><span class="received-label">Received </span>October 3, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-11-21"><span class="accepted-label">Accepted </span>November 21, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Dettman <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.427.1" data-doi="10.1073/pnas.1205532110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andersen</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klitgaard</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petersen</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zachariasen</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Accurate prediction of secondary metabolite gene clusters in filamentous fungi.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">E99</span>–<span class="cit-lpage">E107</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1205532110%26rft_id%253Dinfo%253Apmid%252F23248299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIxMTAvMS9FOTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzIvNDI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.427.2" data-doi="10.1016/j.cub.2010.06.022"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Funt</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prabhu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Socha</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Determinants of divergent adaptation and Dobzhansky-Muller interaction in experimental yeast populations.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1383</span>–<span class="cit-lpage">1388</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DAnderson%26rft.auinit1%253DJ.%2BB.%26rft.volume%253D20%26rft.issue%253D15%26rft.spage%253D1383%26rft.epage%253D1388%26rft.atitle%253DDeterminants%2Bof%2Bdivergent%2Badaptation%2Band%2BDobzhansky-Muller%2Binteraction%2Bin%2Bexperimental%2Byeast%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2010.06.022%26rft_id%253Dinfo%253Apmid%252F20637622%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2010.06.022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20637622&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000280899900032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.427.3" data-doi="10.1007/s00438-010-0513-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Balazs</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pocsi</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamari</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leiter</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Emri</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">AtfA bZIP-type transcription factor regulates oxidative and osmotic stress responses in <em>Aspergillus nidulans</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Genet. Genomics</abbr> <span class="cit-vol">283</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bgenetics%2Band%2Bgenomics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGenet%2BGenomics%26rft.aulast%253DBalazs%26rft.auinit1%253DA.%26rft.volume%253D283%26rft.issue%253D3%26rft.spage%253D289%26rft.epage%253D303%26rft.atitle%253DAtfA%2BbZIP-type%2Btranscription%2Bfactor%2Bregulates%2Boxidative%2Band%2Bosmotic%2Bstress%2Bresponses%2Bin%2BAspergillus%2Bnidulans.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00438-010-0513-z%26rft_id%253Dinfo%253Apmid%252F20131067%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00438-010-0513-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20131067&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.427.4" data-doi="10.1186/1471-2164-15-1039"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barrick</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Colburn</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deatherage</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Traverse</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strand</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Identifying structural variation in haploid microbial genomes from short-read resequencing data using <em>breseq</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1039</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D1039%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-1039%26rft_id%253Dinfo%253Apmid%252F25432719%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-1039&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25432719&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.427.5" data-doi="10.1038/nature09352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burke</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunham</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shahrestani</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thornton</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rose</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-wide analysis of a long-term evolution experiment with <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">467</span>: <span class="cit-fpage">587</span>–<span class="cit-lpage">590</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBurke%26rft.auinit1%253DM.%2BK.%26rft.volume%253D467%26rft.issue%253D7315%26rft.spage%253D587%26rft.epage%253D590%26rft.atitle%253DGenome-wide%2Banalysis%2Bof%2Ba%2Blong-term%2Bevolution%2Bexperiment%2Bwith%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09352%26rft_id%253Dinfo%253Apmid%252F20844486%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20844486&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282273100038&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.427.6" data-doi="10.4161/fly.19695"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cingolani</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Platts</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w(1118); iso-2; iso-3.</span> <abbr class="cit-jnl-abbrev">Fly (Austin)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">80</span>–<span class="cit-lpage">92</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFly%2B%2528Austin%2529%26rft.volume%253D6%26rft.spage%253D80%26rft_id%253Dinfo%253Adoi%252F10.4161%252Ffly.19695%26rft_id%253Dinfo%253Apmid%252F22728672%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/fly.19695&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22728672&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.427.7" data-doi="10.1186/gb-2009-10-10-r118"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conrad</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joyce</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Applebee</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrett</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Whole-genome resequencing of <em>Escherichia coli</em> K-12 MG1655 undergoing short-term laboratory evolution in lactate minimal media reveals flexible selection of adaptive mutations.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">R118</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DConrad%26rft.auinit1%253DT.%2BM.%26rft.volume%253D10%26rft.issue%253D10%26rft.spage%253DR118%26rft.epage%253DR118%26rft.atitle%253DWhole-genome%2Bresequencing%2Bof%2BEscherichia%2Bcoli%2BK-12%2BMG1655%2Bundergoing%2Bshort-term%2Blaboratory%2Bevolution%2Bin%2Blactate%2Bminimal%2Bmedia%2Breveals%2Bflexible%2Bselection%2Bof%2Badaptive%2Bmutations.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2009-10-10-r118%26rft_id%253Dinfo%253Apmid%252F19849850%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2009-10-10-r118&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19849850&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.427.8" data-doi="10.1016/j.fgb.2008.07.022"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Groot</span>  <span class="cit-name-given-names">P. W. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brandt</span>  <span class="cit-name-given-names">B. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horiuchi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ram</span>  <span class="cit-name-given-names">A. F. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Koster</span>  <span class="cit-name-given-names">C. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Comprehensive genomic analysis of cell wall genes in <em>Aspergillus nidulans</em>.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Biol.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">S72</span>–<span class="cit-lpage">S81</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2BGenet.%2BBiol.%26rft.volume%253D46%26rft.spage%253DS72%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fgb.2008.07.022%26rft_id%253Dinfo%253Apmid%252F19585695%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fgb.2008.07.022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19585695&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.427.9" data-doi="10.1006/fgbi.2000.1206"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Del Sorbo</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schoonbeek</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Waard</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Fungal transporters involved in efflux of natural toxic compounds and fungicides.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Biol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2Bgenetics%2Band%2Bbiology%2B%253A%2B%2BFG%2B%2526%2BB%26rft.stitle%253DFungal%2BGenet%2BBiol%26rft.aulast%253DDel%2BSorbo%26rft.auinit1%253DG.%26rft.volume%253D30%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D15%26rft.atitle%253DFungal%2Btransporters%2Binvolved%2Bin%2Befflux%2Bof%2Bnatural%2Btoxic%2Bcompounds%2Band%2Bfungicides.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Ffgbi.2000.1206%26rft_id%253Dinfo%253Apmid%252F10955904%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/fgbi.2000.1206&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10955904&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000088573400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.427.10" data-doi="10.1371/journal.pone.0058008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De Souza</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hashmi</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Osmani</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andrews</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ringelberg</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Functional analysis of the <em>Aspergillus nidulans</em> kinome.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e58008</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De58008%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0058008%26rft_id%253Dinfo%253Apmid%252F23505451%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0058008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23505451&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.427.11" data-doi="10.1111/j.1365-294X.2012.05484.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dettman</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodrigue</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melnyk</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bailey</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Evolutionary insight from whole-genome sequencing of experimentally evolved microbes.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">2058</span>–<span class="cit-lpage">2077</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DDettman%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D21%26rft.issue%253D9%26rft.spage%253D2058%26rft.epage%253D2077%26rft.atitle%253DEvolutionary%2Binsight%2Bfrom%2Bwhole-genome%2Bsequencing%2Bof%2Bexperimentally%2Bevolved%2Bmicrobes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2012.05484.x%26rft_id%253Dinfo%253Apmid%252F22332770%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2012.05484.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22332770&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302932500003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.427.12" data-doi="10.3390/toxins2040367"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duran</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cary</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calvo</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Role of the osmotic stress regulatory pathway in morphogenesis and secondary metabolism in filamentous fungi.</span> <abbr class="cit-jnl-abbrev">Toxins (Basel)</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">367</span>–<span class="cit-lpage">381</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DToxins%2B%2528Basel%2529%26rft.volume%253D2%26rft.spage%253D367%26rft_id%253Dinfo%253Adoi%252F10.3390%252Ftoxins2040367%26rft_id%253Dinfo%253Apmid%252F22069590%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/toxins2040367&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22069590&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.427.13" data-doi="10.1038/nrg1088"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elena</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lenski</span>  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Evolution experiments with microorganisms: the dynamics and genetic bases of adaptation.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">457</span>–<span class="cit-lpage">469</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DElena%26rft.auinit1%253DS.%2BF.%26rft.volume%253D4%26rft.issue%253D6%26rft.spage%253D457%26rft.epage%253D469%26rft.atitle%253DEvolution%2Bexperiments%2Bwith%2Bmicroorganisms%253A%2Bthe%2Bdynamics%2Band%2Bgenetic%2Bbases%2Bof%2Badaptation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1088%26rft_id%253Dinfo%253Apmid%252F12776215%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1088&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12776215&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000183202600015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.427.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ene</span>  <span class="cit-name-given-names">I. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walker</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schiavone</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">K. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin-Yken</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Cell wall remodeling enzymes modulate fungal cell wall elasticity and osmotic stress resistance.</span> <abbr class="cit-jnl-abbrev">MBio.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e00986</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMBio.%26rft.volume%253D6%26rft.spage%253De00986%26rft_id%253Dinfo%253Apmid%252F26220968%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=26220968&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.427.15" data-doi="10.15252/msb.20156444"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Filteau</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamel</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pouliot</span>  <span class="cit-name-given-names">M.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gagnon-Arsenault</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dube</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Evolutionary rescue by compensatory mutations is constrained by genomic and environmental backgrounds.</span> <abbr class="cit-jnl-abbrev">Mol. Syst. Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">832</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BSystems%2BBiology%26rft.stitle%253DMol%2BSyst%2BBiol%26rft.aulast%253DFilteau%26rft.auinit1%253DM.%26rft.volume%253D11%26rft.issue%253D10%26rft.spage%253D832%26rft.epage%253D832%26rft.atitle%253DEvolutionary%2Brescue%2Bby%2Bcompensatory%2Bmutations%2Bis%2Bconstrained%2Bby%2Bgenomic%2Band%2Benvironmental%2Bbackgrounds%26rft_id%253Dinfo%253Adoi%252F10.15252%252Fmsb.20156444%26rft_id%253Dinfo%253Apmid%252F26459777%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibXNiIjtzOjU6InJlc2lkIjtzOjk6IjExLzEwLzgzMiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi80MjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-book" id="cit-7.2.427.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1930</span> <span class="cit-source"><em>The Genetical Theory of Natural Selection</em>.</span> <span class="cit-publ-name">Oxford University Press</span>, <span class="cit-publ-loc">Oxford, UK</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.427.17" data-doi="10.1111/j.1365-2958.2005.04605.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Furukawa</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoshi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maeda</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakajima</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abe</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title"><em>Aspergillus nidulans</em> HOG pathway is activated only by two-component signalling pathway in response to osmotic stress.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">1246</span>–<span class="cit-lpage">1261</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DFurukawa%26rft.auinit1%253DK.%26rft.volume%253D56%26rft.issue%253D5%26rft.spage%253D1246%26rft.epage%253D1261%26rft.atitle%253DAspergillus%2Bnidulans%2BHOG%2Bpathway%2Bis%2Bactivated%2Bonly%2Bby%2Btwo-component%2Bsignalling%2Bpathway%2Bin%2Bresponse%2Bto%2Bosmotic%2Bstress.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2958.2005.04605.x%26rft_id%253Dinfo%253Apmid%252F15882418%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2958.2005.04605.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15882418&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228975300012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.427.18" data-doi="10.1271/bbb.70131"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Furukawa</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yoshimi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Furukawa</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoshi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hagiwara</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Novel reporter gene expression systems for monitoring activation of the <em>Aspergillus nidulans</em> HOG pathway.</span> <abbr class="cit-jnl-abbrev">Biosci. Biotechnol. Biochem.</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">1724</span>–<span class="cit-lpage">1730</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioscience%252C%2BBiotechnology%252C%2Band%2BBiochemistry%26rft.stitle%253DBioscience%252C%2BBiotechnology%252C%2Band%2BBiochemistry%26rft.aulast%253DFurukawa%26rft.auinit1%253DK.%26rft.volume%253D71%26rft.issue%253D7%26rft.spage%253D1724%26rft.epage%253D1730%26rft.atitle%253DNovel%2Breporter%2Bgene%2Bexpression%2Bsystems%2Bfor%2Bmonitoring%2Bactivation%2Bof%2Bthe%2BAspergillus%2Bnidulans%2BHOG%2Bpathway.%26rft_id%253Dinfo%253Adoi%252F10.1271%252Fbbb.70131%26rft_id%253Dinfo%253Apmid%252F17617716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1271/bbb.70131&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17617716&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.427.19" data-doi="10.1128/EC.05080-11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Futagami</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakao</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kido</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oka</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kajiwara</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Putative stress sensors WscA and WscB are involved in hypo-osmotic and acidic pH stress tolerance in <em>Aspergillus nidulans</em>.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1504</span>–<span class="cit-lpage">1515</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryot.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.05080-11%26rft_id%253Dinfo%253Apmid%252F21926329%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czoxMDoiMTAvMTEvMTUwNCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi80MjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.427.20" data-doi="10.1038/nature04341"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Galagan</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calvo</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cuomo</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wortman</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Sequencing of <em>Aspergillus nidulan</em>s and comparative analysis with <em>A. fumigatus</em> and <em>A. oryzae</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">438</span>: <span class="cit-fpage">1105</span>–<span class="cit-lpage">1115</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGalagan%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D438%26rft.issue%253D7071%26rft.spage%253D1105%26rft.epage%253D1115%26rft.atitle%253DSequencing%2Bof%2BAspergillus%2Bnidulans%2Band%2Bcomparative%2Banalysis%2Bwith%2BA.%2Bfumigatus%2Band%2BA.%2Boryzae.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature04341%26rft_id%253Dinfo%253Apmid%252F16372000%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature04341&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16372000&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234111500040&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.427.21" data-doi="10.1271/bbb.70051"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hagiwara</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matsubayashi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marui</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Furukawa</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamashino</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Characterization of the NikA histidine kinase implicated in the phosphorelay signal transduction of <em>Aspergillus nidulans</em>, with special reference to fungicide responses.</span> <abbr class="cit-jnl-abbrev">Biosci. Biotechnol. Biochem.</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">844</span>–<span class="cit-lpage">847</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioscience%252C%2BBiotechnology%252C%2Band%2BBiochemistry%26rft.stitle%253DBioscience%252C%2BBiotechnology%252C%2Band%2BBiochemistry%26rft.aulast%253DHagiwara%26rft.auinit1%253DD.%26rft.volume%253D71%26rft.issue%253D3%26rft.spage%253D844%26rft.epage%253D847%26rft.atitle%253DCharacterization%2Bof%2Bthe%2BNikA%2Bhistidine%2Bkinase%2Bimplicated%2Bin%2Bthe%2Bphosphorelay%2Bsignal%2Btransduction%2Bof%2BAspergillus%2Bnidulans%252C%2Bwith%2Bspecial%2Breference%2Bto%2Bfungicide%2Bresponses.%26rft_id%253Dinfo%253Adoi%252F10.1271%252Fbbb.70051%26rft_id%253Dinfo%253Apmid%252F17341812%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1271/bbb.70051&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17341812&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.427.22" data-doi="10.1016/j.fgb.2009.07.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hagiwara</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Asano</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marui</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yoshimi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mizuno</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Transcriptional profiling for <em>Aspergillus nidulans</em> HogA MAPK signaling pathway in response to fludioxonil and osmotic stress.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Biol.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">868</span>–<span class="cit-lpage">878</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2Bgenetics%2Band%2Bbiology%2B%253A%2B%2BFG%2B%2526%2BB%26rft.stitle%253DFungal%2BGenet%2BBiol%26rft.aulast%253DHagiwara%26rft.auinit1%253DD.%26rft.volume%253D46%26rft.issue%253D11%26rft.spage%253D868%26rft.epage%253D878%26rft.atitle%253DTranscriptional%2Bprofiling%2Bfor%2BAspergillusnidulans%2BHogA%2BMAPK%2Bsignaling%2Bpathway%2Bin%2Bresponse%2Bto%2Bfludioxonil%2Band%2Bosmotic%2Bstress.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fgb.2009.07.003%26rft_id%253Dinfo%253Apmid%252F19596074%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fgb.2009.07.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19596074&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.427.23" data-doi="10.1371/journal.pone.0080881"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hagiwara</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takahashi-Nakaguchi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toyotome</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yoshimi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abe</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">NikA/TcsC histidine kinase is involved in conidiation, hyphal morphology, and responses to osmotic stress and antifungal chemicals in <em>Aspergillus fumigatus</em>.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e80881</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De80881%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0080881%26rft_id%253Dinfo%253Apmid%252F24312504%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0080881&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24312504&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.427.24" data-doi="10.1128/EC.00277-14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jaimes-Arroyo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lara-Rojas</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bayram</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valerius</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Braus</span>  <span class="cit-name-given-names">G. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The SrkA kinase is part of the SakA mitogen-activated protein kinase interactome and regulates stress responses and development in <em>Aspergillus nidulans</em>.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">495</span>–<span class="cit-lpage">510</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryot.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.00277-14%26rft_id%253Dinfo%253Apmid%252F25820520%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo4OiIxNC81LzQ5NSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi80MjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.427.25" data-doi="10.1016/j.gde.2015.08.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jerison</span>  <span class="cit-name-given-names">E. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Desai</span>  <span class="cit-name-given-names">M. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genomic investigations of evolutionary dynamics and epistasis in microbial evolution experiments.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Genet. Dev.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">33</span>–<span class="cit-lpage">39</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BGenet.%2BDev.%26rft.volume%253D35%26rft.spage%253D33%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gde.2015.08.008%26rft_id%253Dinfo%253Apmid%252F26370471%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gde.2015.08.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26370471&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.427.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aggarwal</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rashkovetsky</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korol</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michalak</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Rapid genomic changes in <em>Drosophila melanogaster</em> adapting to desiccation stress in an experimental evolution system.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">233</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D17%26rft.spage%253D233%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-book" id="cit-7.2.427.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kassen</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-source"><em>Experimental Evolution and the Nature of Biodiversity</em>.</span> <span class="cit-publ-name">Roberts and Company</span>, <span class="cit-publ-loc">Greenwood Village, CO</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.427.28" data-doi="10.1101/gr.129684.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koboldt</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larson</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McLellan</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">VarScan 2: somatic mutation and copy number alteration discovery in cancer by exome sequencing.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">568</span>–<span class="cit-lpage">576</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.129684.111%26rft_id%253Dinfo%253Apmid%252F22300766%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIyLzMvNTY4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzQyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.427.29" data-doi="10.1371/journal.pone.0015925"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kofler</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Orozco-terWengel</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Maio</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pandey</span>  <span class="cit-name-given-names">R. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nolte</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">PoPoolation: a toolbox for population genetic analysis of next generation sequencing data from pooled individuals.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e15925</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DKofler%26rft.auinit1%253DR.%26rft.volume%253D6%26rft.issue%253D1%26rft.spage%253De15925%26rft.epage%253De15925%26rft.atitle%253DPoPoolation%253A%2Ba%2Btoolbox%2Bfor%2Bpopulation%2Bgenetic%2Banalysis%2Bof%2Bnext%2Bgeneration%2Bsequencing%2Bdata%2Bfrom%2Bpooled%2Bindividuals.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0015925%26rft_id%253Dinfo%253Apmid%252F21253599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0015925&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21253599&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.427.30" data-doi="10.1371/journal.pgen.1002056"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kvitek</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherlock</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Reciprocal sign epistasis between frequently experimentally evolved adaptive mutations causes a rugged fitness landscape.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002056</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DKvitek%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D7%26rft.issue%253D4%26rft.spage%253De1002056%26rft.epage%253De1002056%26rft.atitle%253DReciprocal%2Bsign%2Bepistasis%2Bbetween%2Bfrequently%2Bexperimentally%2Bevolved%2Badaptive%2Bmutations%2Bcauses%2Ba%2Brugged%2Bfitness%2Blandscape.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002056%26rft_id%253Dinfo%253Apmid%252F21552329%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002056&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21552329&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.427.31" data-doi="10.1111/j.1365-2958.2011.07581.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lara-Rojas</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sánchez</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawasaki</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aguirre</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title"><em>Aspergillus nidulans</em> transcription factor AtfA interacts with the MAPK SakA to regulate general stress responses, development and spore functions.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">436</span>–<span class="cit-lpage">454</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DLara-Rojas%26rft.auinit1%253DF.%26rft.volume%253D80%26rft.issue%253D2%26rft.spage%253D436%26rft.epage%253D454%26rft.atitle%253DAspergillus%2Bnidulans%2Btranscription%2Bfactor%2BAtfA%2Binteracts%2Bwith%2Bthe%2BMAPK%2BSakA%2Bto%2Bregulate%2Bgeneral%2Bstress%2Bresponses%252C%2Bdevelopment%2Band%2Bspore%2Bfunctions.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2958.2011.07581.x%26rft_id%253Dinfo%253Apmid%252F21320182%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2958.2011.07581.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21320182&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.427.32" data-doi="10.1016/j.fgb.2012.06.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leiter</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gonzalez</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erdei</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casado</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kovacs</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Protein phosphatase Z modulates oxidative stress response in fungi.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Biol.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">708</span>–<span class="cit-lpage">716</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2Bgenetics%2Band%2Bbiology%2B%253A%2B%2BFG%2B%2526%2BB%26rft.stitle%253DFungal%2BGenet%2BBiol%26rft.aulast%253DLeiter%26rft.auinit1%253DE.%26rft.volume%253D49%26rft.issue%253D9%26rft.spage%253D708%26rft.epage%253D716%26rft.atitle%253DProtein%2Bphosphatase%2BZ%2Bmodulates%2Boxidative%2Bstress%2Bresponse%2Bin%2Bfungi.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fgb.2012.06.010%26rft_id%253Dinfo%253Apmid%252F22750657%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fgb.2012.06.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22750657&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.427.33" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The sequence alignment/map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTYvMjA3OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi80MjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.427.34" data-doi="10.1038/nrg3937"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liti</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luptak</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tenaillon</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Elucidating the molecular architecture of adaptation via evolve and resequence experiments.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">567</span>–<span class="cit-lpage">582</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D16%26rft.spage%253D567%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3937%26rft_id%253Dinfo%253Apmid%252F26347030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3937&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26347030&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.427.35" data-doi="10.1073/pnas.0803466105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sung</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morris</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coffey</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landry</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A genome-wide view of the spectrum of spontaneous mutations in yeast.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">9272</span>–<span class="cit-lpage">9277</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0803466105%26rft_id%253Dinfo%253Apmid%252F18583475%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA1LzI3LzkyNzIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzIvNDI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.427.36" data-doi="10.1128/EC.00167-06"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Malavazi</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Savoldi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zingaretti Di Mauro</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martins Menck</span>  <span class="cit-name-given-names">C. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Transcriptome analysis of <em>Aspergillus nidulans</em> exposed to camptothecin-induced DNA damage.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1688</span>–<span class="cit-lpage">1704</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryotic%2BCell%26rft.stitle%253DEukaryot%2BCell%26rft.aulast%253DMalavazi%26rft.auinit1%253DI.%26rft.volume%253D5%26rft.issue%253D10%26rft.spage%253D1688%26rft.epage%253D1704%26rft.atitle%253DTranscriptome%2BAnalysis%2Bof%2BAspergillus%2Bnidulans%2BExposed%2Bto%2BCamptothecin-Induced%2BDNA%2BDamage%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.00167-06%26rft_id%253Dinfo%253Apmid%252F17030995%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo5OiI1LzEwLzE2ODgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzIvNDI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.427.37" data-doi="10.1016/j.fgb.2008.07.013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miskei</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karanyi</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pocsi</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Annotation of stress-response proteins in the aspergilli.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Biol.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">S105</span>–<span class="cit-lpage">S120</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2BGenet.%2BBiol.%26rft.volume%253D46%26rft.spage%253DS105%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fgb.2008.07.013%26rft_id%253Dinfo%253Apmid%252F18703157%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fgb.2008.07.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18703157&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.427.38" data-doi="10.1371/journal.pgen.1001109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nishant</span>  <span class="cit-name-given-names">K. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mancera</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Argueso</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlattl</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The baker’s yeast diploid genome is remarkably stable in vegetative growth and meiosis.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1001109</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D6%26rft.spage%253De1001109%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1001109%26rft_id%253Dinfo%253Apmid%252F20838597%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1001109&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20838597&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.427.39" data-doi="10.2307/2411226"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Orr</span>  <span class="cit-name-given-names">H. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">The population genetics of adaptation: the distribution of factors fixed during adaptive evolution.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">935</span>–<span class="cit-lpage">949</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D52%26rft.spage%253D935%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2411226%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2411226&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000075753100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.427.40" data-doi="10.1038/nrg1523"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Orr</span>  <span class="cit-name-given-names">H. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The genetic theory of adaptation: a brief history.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">119</span>–<span class="cit-lpage">127</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DOrr%26rft.auinit1%253DH.%2BA.%26rft.volume%253D6%26rft.issue%253D2%26rft.spage%253D119%26rft.epage%253D127%26rft.atitle%253DThe%2Bgenetic%2Btheory%2Bof%2Badaptation%253A%2Ba%2Bbrief%2Bhistory.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1523%26rft_id%253Dinfo%253Apmid%252F15716908%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1523&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15716908&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226721300012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.427.41" data-doi="10.1073/pnas.0705778104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">S.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krug</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Clonal interference in large populations.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">18135</span>–<span class="cit-lpage">18140</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0705778104%26rft_id%253Dinfo%253Apmid%252F17984061%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA0LzQ2LzE4MTM1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzQyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.427.42" data-doi="10.1073/pnas.98.2.525"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Riehle</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bennett</span>  <span class="cit-name-given-names">A. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Genetic architecture of thermal adaptation in <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">525</span>–<span class="cit-lpage">530</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DRiehle%26rft.auinit1%253DM.%2BM.%26rft.volume%253D98%26rft.issue%253D2%26rft.spage%253D525%26rft.epage%253D530%26rft.atitle%253DGenetic%2Barchitecture%2Bof%2Bthermal%2Badaptation%2Bin%2BEscherichia%2Bcoli%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.98.2.525%26rft_id%253Dinfo%253Apmid%252F11149947%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo4OiI5OC8yLzUyNSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi80MjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.427.43" data-doi="10.1111/j.1420-9101.2005.00934.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schoustra</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slakhorst</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Debets</span>  <span class="cit-name-given-names">A. J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoekstra</span>  <span class="cit-name-given-names">R. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Comparing artificial and natural selection in rate of adaptation to genetic stress in Aspergillus nidulans.</span> <abbr class="cit-jnl-abbrev">J. Evol. Biol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">771</span>–<span class="cit-lpage">778</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bevolutionary%2Bbiology%26rft.stitle%253DJ%2BEvol%2BBiol%26rft.aulast%253DSchoustra%26rft.auinit1%253DS.%2BE.%26rft.volume%253D18%26rft.issue%253D4%26rft.spage%253D771%26rft.epage%253D778%26rft.atitle%253DComparing%2Bartificial%2Band%2Bnatural%2Bselection%2Bin%2Brate%2Bof%2Badaptation%2Bto%2Bgenetic%2Bstress%2Bin%2BAspergillus%2Bnidulans.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1420-9101.2005.00934.x%26rft_id%253Dinfo%253Apmid%252F16033548%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1420-9101.2005.00934.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16033548&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.427.44" data-doi="10.1111/j.1420-9101.2006.01102.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schoustra</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Debets</span>  <span class="cit-name-given-names">A. J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slakhorst</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoekstra</span>  <span class="cit-name-given-names">R. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Reducing the cost of resistance; experimental evolution in the filamentous fungus <em>Aspergillus nidulans</em>.</span> <abbr class="cit-jnl-abbrev">J. Evol. Biol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1115</span>–<span class="cit-lpage">1127</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bevolutionary%2Bbiology%26rft.stitle%253DJ%2BEvol%2BBiol%26rft.aulast%253DSchoustra%26rft.auinit1%253DS.%2BE.%26rft.volume%253D19%26rft.issue%253D4%26rft.spage%253D1115%26rft.epage%253D1127%26rft.atitle%253DReducing%2Bthe%2Bcost%2Bof%2Bresistance%253B%2Bexperimental%2Bevolution%2Bin%2Bthe%2Bfilamentous%2Bfungus%2BAspergillus%2Bnidulans.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1420-9101.2006.01102.x%26rft_id%253Dinfo%253Apmid%252F16780512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1420-9101.2006.01102.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16780512&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239154100010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.427.45" data-doi="10.1371/journal.pbio.1000250"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schoustra</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bataillon</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gifford</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kassen</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The properties of adaptive walks in evolving populations of fungus.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1000250</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DSchoustra%26rft.auinit1%253DS.%2BE.%26rft.volume%253D7%26rft.issue%253D11%26rft.spage%253De1000250%26rft.epage%253De1000250%26rft.atitle%253DThe%2Bproperties%2Bof%2Badaptive%2Bwalks%2Bin%2Bevolving%2Bpopulations%2Bof%2Bfungus.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1000250%26rft_id%253Dinfo%253Apmid%252F19956798%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1000250&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19956798&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.427.46" data-doi="10.1371/journal.pone.0050305"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schoustra</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Punzalan</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dali</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rundle</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kassen</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Multivariate phenotypic divergence due to the fixation of beneficial mutations in experimentally evolved lineages of a filamentous fungus.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e50305</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DSchoustra%26rft.auinit1%253DS.%2BE.%26rft.volume%253D7%26rft.issue%253D11%26rft.spage%253De50305%26rft.epage%253De50305%26rft.atitle%253DMultivariate%2Bphenotypic%2Bdivergence%2Bdue%2Bto%2Bthe%2Bfixation%2Bof%2Bbeneficial%2Bmutations%2Bin%2Bexperimentally%2Bevolved%2Blineages%2Bof%2Ba%2Bfilamentous%2Bfungus.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0050305%26rft_id%253Dinfo%253Apmid%252F23185601%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0050305&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23185601&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.427.47" data-doi="10.1371/journal.pbio.1001935"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Szamecz</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boross</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kalapis</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kovacs</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fekete</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The genomic landscape of compensatory evolution.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">e1001935</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D12%26rft.spage%253De1001935%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1001935%26rft_id%253Dinfo%253Apmid%252F25157590%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1001935&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25157590&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.427.48" data-doi="10.1128/EC.00085-07"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vargas-Perez</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanchez</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawasaki</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Georgellis</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aguirre</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Response regulators SrrA and SskA are central components of a phosphorelay system involved in stress signal transduction and asexual sporulation in <em>Aspergillus nidulans</em>.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1570</span>–<span class="cit-lpage">1583</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryot.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.00085-07%26rft_id%253Dinfo%253Apmid%252F17630329%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo4OiI2LzkvMTU3MCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi80MjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.427.49" data-doi="10.1126/science.285.5426.422"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wichman</span>  <span class="cit-name-given-names">H. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Badgett</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scott</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boulianne</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bull</span>  <span class="cit-name-given-names">J. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Different trajectories of parallel evolution during viral adaptation.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">285</span>: <span class="cit-fpage">422</span>–<span class="cit-lpage">424</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DWichman%26rft.auinit1%253DH.%2BA.%26rft.volume%253D285%26rft.issue%253D5426%26rft.spage%253D422%26rft.epage%253D424%26rft.atitle%253DDifferent%2BTrajectories%2Bof%2BParallel%2BEvolution%2BDuring%2BViral%2BAdaptation%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.285.5426.422%26rft_id%253Dinfo%253Apmid%252F10411508%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyODUvNTQyNi80MjIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzIvNDI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.427.50" data-doi="10.1093/bioinformatics/btp394"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schulz</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Apweiler</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ning</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Pindel: a pattern growth approach to detect break points of large deletions and medium sized insertions from paired-end short reads.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2865</span>–<span class="cit-lpage">2871</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp394%26rft_id%253Dinfo%253Apmid%252F19561018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMjEvMjg2NSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi80MjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/2/427.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/2/413" title="Forward Genetics by Sequencing EMS Variation-Induced Inbred Lines" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/2/437" title="Screening and Analysis of Janelia FlyLight Project Enhancer-Gal4 Strains Identifies Multiple Gene Enhancers Active During Hematopoiesis in Normal and Wasp-Challenged Drosophila Larvae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/2"><span>Volume 7 Issue 2, February 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/2.cover-source.jpg?itok=m4uMC43I" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/2/427" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form--dIGaWCFWB8g3Q-8EDpqKIcZG5WHsdVE3uxwcfQikgc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/2/427" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/2/427" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/2/427" class="active" data-icon-position="" data-hide-link-title="0">Genomics of Compensatory Adaptation in Experimental Populations of Aspergillus nidulans</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460558" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-pacMV2PPZyz78ZWVtGz5NGAmS66N6GOGxULbDjssrtM" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/2/427.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/2/427" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-ZGwX4Rc3xzkLwH9SLUh8DLKPhOzN10KCttey5vZzlVU" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/2/427" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/2/427.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460558" id="node460558--2" data-pisa="ggg;7/2/427" data-pisa-master="ggg;g3.116.036152" data-apath="/ggg/7/2/427.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomics of Compensatory Adaptation in Experimental Populations of <em>Aspergillus nidulans</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jeremy R. Dettman</span>, <span class="highwire-citation-author" data-delta="1">Nicolas Rodrigue</span>, <span class="highwire-citation-author" data-delta="2">Sijmen E. Schoustra</span> and <span class="highwire-citation-author" data-delta="3">Rees Kassen</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">427-436;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.036152 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460558--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jeremy R. Dettman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, University of Ottawa, Ontario K1N 6N5, Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Centre for Advanced Research in Environmental Genomics, University of Ottawa, Ontario K1N 6N5, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeremy%2BR.%2BDettman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dettman%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJeremy%2BR.%2BDettman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jdettman@uottawa.ca" class="" data-icon-position="" data-hide-link-title="0">jdettman@uottawa.ca</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Nicolas Rodrigue </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology, Carleton University, Ottawa, Ontario K1S 5B6, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicolas%2BRodrigue%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rodrigue%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANicolas%2BRodrigue%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Sijmen E. Schoustra </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Laboratory of Genetics, Wageningen University, NL-6708 PB, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sijmen%2BE.%2BSchoustra%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schoustra%20SE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASijmen%2BE.%2BSchoustra%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Rees Kassen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, University of Ottawa, Ontario K1N 6N5, Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Centre for Advanced Research in Environmental Genomics, University of Ottawa, Ontario K1N 6N5, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rees%2BKassen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kassen%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARees%2BKassen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/2/427" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/2/427&amp;title=Genomics%20of%20Compensatory%20Adaptation%20in%20Experimental%20Populations%20of%20Aspergillus%20nidulans" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/2/427&amp;title=Genomics%20of%20Compensatory%20Adaptation%20in%20Experimental%20Populations%20of%20Aspergillus%20nidulans" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/2/427&amp;title=Genomics%20of%20Compensatory%20Adaptation%20in%20Experimental%20Populations%20of%20Aspergillus%20nidulans" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/2/427&amp;text=Genomics%20of%20Compensatory%20Adaptation%20in%20Experimental%20Populations%20of%20Aspergillus%20nidulans" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/2/427&amp;title=Genomics%20of%20Compensatory%20Adaptation%20in%20Experimental%20Populations%20of%20Aspergillus%20nidulans" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/2/427&amp;t=Genomics%20of%20Compensatory%20Adaptation%20in%20Experimental%20Populations%20of%20Aspergillus%20nidulans" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/2/427&amp;title=Genomics%20of%20Compensatory%20Adaptation%20in%20Experimental%20Populations%20of%20Aspergillus%20nidulans" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/2/427&amp;title=Genomics%20of%20Compensatory%20Adaptation%20in%20Experimental%20Populations%20of%20Aspergillus%20nidulans" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460558/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460558" id="node460558--3" data-pisa="ggg;7/2/427" data-pisa-master="ggg;g3.116.036152" data-apath="/ggg/7/2/427.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomics of Compensatory Adaptation in Experimental Populations of <em>Aspergillus nidulans</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jeremy R. Dettman</span>, <span class="highwire-citation-author" data-delta="1">Nicolas Rodrigue</span>, <span class="highwire-citation-author" data-delta="2">Sijmen E. Schoustra</span> and <span class="highwire-citation-author" data-delta="3">Rees Kassen</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">427-436;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.036152 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460558--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jeremy R. Dettman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, University of Ottawa, Ontario K1N 6N5, Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Centre for Advanced Research in Environmental Genomics, University of Ottawa, Ontario K1N 6N5, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeremy%2BR.%2BDettman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dettman%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJeremy%2BR.%2BDettman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jdettman@uottawa.ca" class="" data-icon-position="" data-hide-link-title="0">jdettman@uottawa.ca</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Nicolas Rodrigue </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology, Carleton University, Ottawa, Ontario K1S 5B6, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicolas%2BRodrigue%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rodrigue%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANicolas%2BRodrigue%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Sijmen E. Schoustra </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Laboratory of Genetics, Wageningen University, NL-6708 PB, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sijmen%2BE.%2BSchoustra%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schoustra%20SE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASijmen%2BE.%2BSchoustra%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Rees Kassen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, University of Ottawa, Ontario K1N 6N5, Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Centre for Advanced Research in Environmental Genomics, University of Ottawa, Ontario K1N 6N5, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rees%2BKassen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kassen%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARees%2BKassen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=427&amp;rft.epage=436&amp;rft.atitle=Genomics%20of%20Compensatory%20Adaptation%20in%20Experimental%20Populations%20of%20Aspergillus%20nidulans&amp;rft.volume=7&amp;rft.issue=2&amp;rft.date=2017-02-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Dettman%2C+Jeremy+R.&amp;rft.au=Rodrigue%2C+Nicolas&amp;rft.au=Schoustra%2C+Sijmen+E.&amp;rft.au=Kassen%2C+Rees"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460558/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460558/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460558/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460558/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460558/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460558/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460558/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460558/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460558/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460558/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460558/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460558/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/2/427" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/2/427.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/2/427.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/2/427.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/2/427.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
