<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/2/617" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/2/617.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/2/617.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/2/617.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/2/617/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/2/617.atom" />
<meta name="HW.pisa" content="ggg;7/2/617" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Predictability of Genetic Interactions from Functional Gene Modules" />
<meta name="DC.Identifier" content="10.1534/g3.116.035915" />
<meta name="DC.Date" content="2017-02-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Young and Marcotte. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Characterizing genetic interactions is crucial to understanding cellular and organismal response to gene-level perturbations. Such knowledge can inform the selection of candidate disease therapy targets, yet experimentally determining whether genes interact is technically nontrivial and time-consuming. High-fidelity prediction of different classes of genetic interactions in multiple organisms would substantially alleviate this experimental burden. Under the hypothesis that functionally related genes tend to share common genetic interaction partners, we evaluate a computational approach to predict genetic interactions in Homo sapiens , Drosophila melanogaster , and Saccharomyces cerevisiae . By leveraging knowledge of functional relationships between genes, we cross-validate predictions on known genetic interactions and observe high predictive power of multiple classes of genetic interactions in all three organisms. Additionally, our method suggests high-confidence candidate interaction pairs that can be directly experimentally tested. A web application is provided for users to query genes for predicted novel genetic interaction partners. Finally, by subsampling the known yeast genetic interaction network, we found that novel genetic interactions are predictable even when knowledge of currently known interactions is minimal." />
<meta name="DC.Contributor" content="Jonathan H. Young" />
<meta name="DC.Contributor" content="Edward M. Marcotte" />
<meta name="article:published_time" content="2017-02-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Predictability of Genetic Interactions from Functional Gene Modules" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Characterizing genetic interactions is crucial to understanding cellular and organismal response to gene-level perturbations. Such knowledge can inform the selection of candidate disease therapy targets, yet experimentally determining whether genes interact is technically nontrivial and time-consuming. High-fidelity prediction of different classes of genetic interactions in multiple organisms would substantially alleviate this experimental burden. Under the hypothesis that functionally related genes tend to share common genetic interaction partners, we evaluate a computational approach to predict genetic interactions in &lt;i&gt;Homo sapiens&lt;/i&gt;, &lt;i&gt;Drosophila melanogaster&lt;/i&gt;, and &lt;i&gt;Saccharomyces cerevisiae&lt;/i&gt;. By leveraging knowledge of functional relationships between genes, we cross-validate predictions on known genetic interactions and observe high predictive power of multiple classes of genetic interactions in all three organisms. Additionally, our method suggests high-confidence candidate interaction pairs that can be directly experimentally tested. A web application is provided for users to query genes for predicted novel genetic interaction partners. Finally, by subsampling the known yeast genetic interaction network, we found that novel genetic interactions are predictable even when knowledge of currently known interactions is minimal.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/02/01" />
<meta name="citation_mjid" content="ggg;7/2/617" />
<meta name="citation_id" content="7/2/617" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/2/617" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/2/617.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/2/617.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/2/617.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.035915" />
<meta name="citation_pmid" content="28007839" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="2" />
<meta name="citation_num_pages" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="617" />
<meta name="citation_lastpage" content="624" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Jonathan H. Young" />
<meta name="citation_author_institution" content="Institute for Computational Engineering and Sciences, The University of Texas at Austin, Texas 78712" />
<meta name="citation_author_institution" content="Center for Systems and Synthetic Biology, The University of Texas at Austin, Texas 78712" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3365-2702" />
<meta name="citation_author" content="Edward M. Marcotte" />
<meta name="citation_author_institution" content="Center for Systems and Synthetic Biology, The University of Texas at Austin, Texas 78712" />
<meta name="citation_author_institution" content="Department of Molecular Biosciences, The University of Texas at Austin, Texas 78712" />
<meta name="citation_author_email" content="marcotte@icmb.utexas.edu" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=A. Ashworth;citation_author=C. J. Lord;citation_author=J. S. Reis-Filho;citation_title=Genetic interactions in cancer progression and treatment.;citation_pages=30-38;citation_volume=145;citation_year=2011;citation_issue=1;citation_pmid=21458666;citation_doi=10.1016/j.cell.2011.03.020" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Genomics Hum. Genet.;citation_author=A. Baryshnikova;citation_author=M. Costanzo;citation_author=C. L. Myers;citation_author=B. Andrews;citation_author=C. Boone;citation_title=Genetic interaction networks: toward an understanding of heritability.;citation_pages=111-133;citation_volume=14;citation_year=2013;citation_pmid=23808365;citation_doi=10.1146/annurev-genom-082509-141730" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. B.;citation_author=Y. Benjamini;citation_author=Y. Hochberg;citation_title=Controlling the false discovery rate: a practical and powerful approach to multiple testing.;citation_pages=289-300;citation_volume=57;citation_year=1995" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. Costanzo;citation_author=A. Baryshnikova;citation_author=J. Bellay;citation_author=Y. Kim;citation_author=E. D. Spear;citation_title=The Genetic Landscape of a Cell;citation_pages=425-431;citation_volume=327;citation_year=2010;citation_issue=5964;citation_pmid=20093466;citation_doi=10.1126/science.1180823" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=A. J. Enright;citation_author=S. Van Dongen;citation_author=C. A. Ouzounis;citation_title=An efficient algorithm for large-scale detection of protein families;citation_pages=1575-1584;citation_volume=30;citation_year=2002;citation_issue=7;citation_pmid=11917018;citation_doi=10.1093/nar/30.7.1575" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Pharmacol. Toxicol.;citation_author=F. Fece de la Cruz;citation_author=B. V. Gapp;citation_author=S. M. Nijman;citation_title=Synthetic lethal vulnerabilities of cancer.;citation_pages=513-531;citation_volume=55;citation_year=2015;citation_pmid=25340932;citation_doi=10.1146/annurev-pharmtox-010814-124511" />
<meta name="citation_reference" content="citation_journal_title=N. Engl. J. Med.;citation_author=P. C. Fong;citation_author=D. S. Boss;citation_author=T. A. Yap;citation_author=A. Tutt;citation_author=P. Wu;citation_title=Inhibition of poly (ADP-ribose) polymerase in tumors from BRCA mutation carriers.;citation_volume=361;citation_year=2009;citation_pmid=19553641;citation_doi=10.1056/NEJMoa0900212" />
<meta name="citation_reference" content="citation_journal_title=Phys. Rep.;citation_author=S. Fortunato;citation_title=Community detection in graphs.;citation_pages=75-174;citation_volume=486;citation_year=2010;citation_doi=10.1016/j.physrep.2009.11.002" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=C. S. Greene;citation_author=A. Krishnan;citation_author=A. K. Wong;citation_author=E. Ricciotti;citation_author=R. A. Zelaya;citation_title=Understanding multicellular function and disease with human tissue-specific networks.;citation_pages=569-576;citation_volume=47;citation_year=2015;citation_pmid=25915600;citation_doi=10.1038/ng.3259" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=G. T. Hart;citation_author=I. Lee;citation_author=E. M. Marcotte;citation_title=A high-accuracy consensus map of yeast protein complexes reveals modular nature of gene essentiality.;citation_pages=236-236;citation_volume=8;citation_year=2007;citation_pmid=17605818;citation_doi=10.1186/1471-2105-8-236" />
<meta name="citation_reference" content="citation_journal_title=Comput. Sci. Eng.;citation_author=J. D. Hunter;citation_title=Matplotlib: a 2D graphics environment.;citation_pages=90-95;citation_volume=9;citation_year=2007;citation_doi=10.1109/MCSE.2007.55" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=I. Lee;citation_author=Z. Li;citation_author=E. M. Marcotte;citation_title=An improved, bias-reduced probabilistic functional gene network of baker's yeast, Saccharomyces cerevisiae.;citation_pages=e988-e988;citation_volume=2;citation_year=2007;citation_issue=10;citation_pmid=17912365;citation_doi=10.1371/journal.pone.0000988" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=I. Lee;citation_author=B. Lehner;citation_author=T. Vavouri;citation_author=J. Shin;citation_author=A. G. Fraser;citation_title=Predicting genetic modifier loci using functional gene networks.;citation_volume=20;citation_year=2010;citation_pmid=20538624;citation_doi=10.1101/gr.102749.109" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=I. Lee;citation_author=U. M. Blom;citation_author=P. I. Wang;citation_author=J. E. Shim;citation_author=E. M. Marcotte;citation_title=Prioritizing candidate disease genes by network-based boosting of genome-wide association data.;citation_volume=21;citation_year=2011;citation_pmid=21536720;citation_doi=10.1101/gr.118992.110" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=R. Mani;citation_author=R. P. S. Onge;citation_author=J. L. Hartman;citation_author=G. Giaever;citation_author=F. P. Roth;citation_title=Defining genetic interaction.;citation_volume=105;citation_year=2008;citation_pmid=18305163;citation_doi=10.1073/pnas.0712255105" />
<meta name="citation_reference" content="citation_journal_title=PLOS Comput. Biol.;citation_title=An integrative multi-network and multi-classifier approach to predict genetic interactions.;citation_author=G. Pandey;citation_author=B. Zhang;citation_author=A. N. Chang;citation_author=C. L. Myers;citation_author=J. Zhu;citation_volume=6;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=J. Mach. Learn. Res.;citation_author=F. Pedregosa;citation_author=G. Varoquaux;citation_author=A. Gramfort;citation_author=V. Michel;citation_author=B. Thirion;citation_title=Scikit-learn: machine learning in Python.;citation_pages=2825-2830;citation_volume=12;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=Y. Qi;citation_author=Y. Suhail;citation_author=Y. -Y. Lin;citation_author=J. D. Boeke;citation_author=J. S. Bader;citation_title=Finding friends and enemies in an enemies-only network: a graph diffusion kernel for predicting novel genetic interactions and co-complex membership from yeast genetic interactions.;citation_volume=18;citation_year=2008;citation_pmid=18832443;citation_doi=10.1101/gr.077693.108" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. Ruepp;citation_author=B. Waegele;citation_author=M. Lechner;citation_author=B. Brauner;citation_author=I. Dunger-Kaltenbach;citation_title=CORUM: the comprehensive resource of mammalian protein complexes–2009.;citation_volume=38;citation_year=2010;citation_pmid=19884131;citation_doi=10.1093/nar/gkp914" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=P. Shannon;citation_author=A. Markiel;citation_author=O. Ozier;citation_author=N. S. Baliga;citation_author=J. T. Wang;citation_title=Cytoscape: A Software Environment for Integrated Models of Biomolecular Interaction Networks;citation_pages=2498-2504;citation_volume=13;citation_year=2003;citation_issue=11;citation_pmid=14597658;citation_doi=10.1101/gr.1239303" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. Shin;citation_author=S. Yang;citation_author=E. Kim;citation_author=C. Y. Kim;citation_author=H. Shim;citation_title=FlyNet: a versatile network prioritization server for the Drosophila community.;citation_volume=43;citation_year=2015;citation_pmid=25943544;citation_doi=10.1093/nar/gkv453" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=C. Stark;citation_author=B.-J. Breitkreutz;citation_author=T. Reguly;citation_author=L. Boucher;citation_author=A. Breitkreutz;citation_title=BioGRID: a general repository for interaction datasets;citation_pages=D535-D539;citation_volume=34;citation_year=2006;citation_issue=suppl_1;citation_pmid=16381927;citation_doi=10.1093/nar/gkj109" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=T. Wang;citation_author=J. J. Wei;citation_author=D. M. Sabatini;citation_author=E. S. Lander;citation_title=Genetic screens in human cells using the CRISPR-Cas9 system.;citation_volume=343;citation_year=2014;citation_pmid=24336569;citation_doi=10.1126/science.1246981" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. S. Wong;citation_author=G. C. Choi;citation_author=C. H. Cui;citation_author=G. Pregernig;citation_author=P. Milani;citation_title=Multiplexed barcoded CRISPR-Cas9 screening enabled by CombiGEM.;citation_volume=113;citation_year=2016;citation_pmid=26864203;citation_doi=10.1073/pnas.1517883113" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. L. Wong;citation_author=L. V. Zhang;citation_author=A. H. Tong;citation_author=Z. Li;citation_author=D. S. Goldberg;citation_title=Combining biological networks to predict genetic interactions.;citation_volume=101;citation_year=2004;citation_pmid=15496468;citation_doi=10.1073/pnas.0406614101" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=C. Wu;citation_author=I. MacLeod;citation_author=A. I. Su;citation_title=BioGPS and MyGene.info: organizing online, gene-centric information.;citation_pages=D561-D565;citation_volume=41;citation_year=2012;citation_pmid=23175613" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=W. Zhong;citation_author=P. W. Sternberg;citation_title=Genome-wide prediction of C. elegans genetic interactions.;citation_pages=1481-1484;citation_volume=311;citation_year=2006;citation_issue=5766;citation_pmid=16527984;citation_doi=10.1126/science.1123287" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Predictability of Genetic Interactions from Functional Gene Modules" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2.cover-source.jpg" />
<meta name="twitter:description" content="Characterizing genetic interactions is crucial to understanding cellular and organismal response to gene-level perturbations. Such knowledge can inform the selection of candidate disease therapy targets, yet experimentally determining whether genes interact is technically nontrivial and time-consuming. High-fidelity prediction of different classes of genetic interactions in multiple organisms would substantially alleviate this experimental burden. Under the hypothesis that functionally related genes tend to share common genetic interaction partners, we evaluate a computational approach to predict genetic interactions in Homo sapiens , Drosophila melanogaster , and Saccharomyces cerevisiae . By leveraging knowledge of functional relationships between genes, we cross-validate predictions on known genetic interactions and observe high predictive power of multiple classes of genetic interactions in all three organisms. Additionally, our method suggests high-confidence candidate interaction pairs that can be directly experimentally tested. A web application is provided for users to query genes for predicted novel genetic interaction partners. Finally, by subsampling the known yeast genetic interaction network, we found that novel genetic interactions are predictable even when knowledge of currently known interactions is minimal." />
<meta name="og-title" property="og:title" content="Predictability of Genetic Interactions from Functional Gene Modules" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/2/617" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Characterizing genetic interactions is crucial to understanding cellular and organismal response to gene-level perturbations. Such knowledge can inform the selection of candidate disease therapy targets, yet experimentally determining whether genes interact is technically nontrivial and time-consuming. High-fidelity prediction of different classes of genetic interactions in multiple organisms would substantially alleviate this experimental burden. Under the hypothesis that functionally related genes tend to share common genetic interaction partners, we evaluate a computational approach to predict genetic interactions in Homo sapiens , Drosophila melanogaster , and Saccharomyces cerevisiae . By leveraging knowledge of functional relationships between genes, we cross-validate predictions on known genetic interactions and observe high predictive power of multiple classes of genetic interactions in all three organisms. Additionally, our method suggests high-confidence candidate interaction pairs that can be directly experimentally tested. A web application is provided for users to query genes for predicted novel genetic interaction partners. Finally, by subsampling the known yeast genetic interaction network, we found that novel genetic interactions are predictable even when knowledge of currently known interactions is minimal." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2.cover-source.jpg" />
    <title>Predictability of Genetic Interactions from Functional Gene Modules | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bed9eK4HgZgeq1SWIHZ3nUSsO_Lg6SY0VT6zRKv2uw4__AOTOtpTA1sp4jw4ldY-8h8LueAv-V2MjVmPlkIZ--AU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"kksEnrKJvvG3S3lmEAKQH1VhpR2XDtYsw1kjOWme1bA","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460586","apath":"\/ggg\/7\/2\/617.atom","pisa":"ggg;7\/2\/617","ac":{"\/ggg\/7\/2\/617.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/2\/617.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/2\/617"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/2\/617"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/2\/617.supplemental":true,"\/content\/7\/2\/617":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022R\\\\\\\/ud6DNNbi\\\\\\\/zl1ZuCteakDH\\\\\\\/5VgU+rgp+jWrfLn0p0TvfF41ITIEhYeWAbbh3VXyF7gW2LWqVbCi3I4GkpIAVNNnVk7UH+8ayDRMmtGqmm\\\\\\\/isnWQf1NqIgmLOooAriuHJBNxHSWeB+X5q3QWgrI\\\\\\\/C\\\\\\\/BKzEqoBFofOHjce7jgsHsT0U5+VZGQJGFWoIwW3nGF3O70Na2j+jcfAeOFifH052g5bWMEz3dsBZBVTukL1gUn1Dr81bJ3zID1jIEACZCEkT0OvU5kZMdgZkj45KRxOE+OHtHqJeSl\\\\\\\/Ey7tblE9WrUYtPn5IT3yjOHTVyWh1aRgVi2joybPkKh0LQS3AM4o6Hy2krM3eBlsxgk4pVxDCQiG2CnxoF+uN0zGcL1xM5SQoCe4XbIBuH4E7YhQJBAOQWvj+5cMwKkaJFWYAzWquuJnTOVzpzKY3SYsN4M9RLj8wuu415pXDZxSmNgLQcwP448lgR\\\\\\\/Y5b2iaBzLruKcJ8jrB3b+eYjMQ2Yjosn4ZN9V90y0zE8NmCx2eiVbQihPU9fAgwB1hwn\\\\\\\/79jAGeC3FldPo755R6Zj2nKE6bBXS+05t0VyM03cQ99wojh0YcIPJ5TjE6EaMbca0DbAFvM+BnJw2yMQEGLAZaT+D\\\\\\\/2tL+wcAteaTkm9M2SzGT8190vlO9EJ6kuZdUs2XuEqKMb2DTDZH2jaKyHW5tRjUA99pFDIgGKm6ZAOF5Q\\\\\\\/okWHFCFUNayyH\\\\\\\/k3FXQkw7\\\\\\\/0zWrSAkQMmEZgBaSOH\\\\\\\/xRVXGo7MdOXv1En0oqvQDKoxyivsW\\\\\\\/54Z4BIHVCqr\\\\\\\/bV9xhiuZDkeWLguv34xyyneoZYuywPBwPMCQSU2BtfTwQE6orCvfujQ4Ip6EH54KJS+BB7GZB96tDltuWdDe\\\\\\\/i+1RfxgTh1O7UZX5z+veWHMrUBsy8L1ZSdfv0WdLXgegAt8jgY2buPzvEQ8pDj3RfNwf++Gund6FXN0WYrVDGCEv9Z4LC1AoLH8mbl9oeGibBA3D1C78S6B6CO2yHDbNFNr1h50OMO9CuwMLHqpWgVNfcb0Xxox\\\\\\\/DURi9IOXplGLDDZqWi9Oh0Z2dTLKV0PR64tLcn+tDTvpV5ozwBXzWtfoRRX\\\\\\\/kksEo7qCzDOyzI\\\\\\\/6QEfx488nIvAMNQtBVHVdok\\\\\\\/WoxhfJai2p4BGroI19Q\\\\\\\/J9Z6t7tm6uBsAfvWdlg0fQASzrg388oLHJCrvG1EfhOHYPaaUTamuPL3a4+c06HF5A5R8aoVK724j\\\\\\\/ZfrVAGY0XzT\\\\\\\/E+oC1mXDO2VgFQCuSakBmUFec\\\u0022,\\\u0022iv\\\u0022:\\\u0022CWTC4I5DqmdRJOkMTY9gSQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229f6f1944fbc36f21153de5111bfe6a401c636fd0874b8332e54d026b5058c706\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022LsHxQdqce3ShJRmpReJJL+3humDZ7CL1PBJY19ChFy\\\\\\\/QF2JBUk3HVunuUWl4aN7vGvG7pOHtaW+ny2c05QEdAm+gi8jMD+c7TXlLHrT+E1RoTidJ8nKiGFaNbAvBVPpxAvoDHbbUgKQkCcfukYlw1rAmhfeXhbFqY7h4sdxHddPUOF0IXCY+Dle3f33KUTqOt0Qzsw4jjWpibzH7Scwm6daCEvxvaGjaA9Jc1nMd35977PtEfD853w72BesZqcUJEItgFUlvw8rTbQeKbJWtnWCLHcI9z3c1L8iRudgOr+Hps5uKKism7nonCshAXt0ydY8F1cC0qYEWFmKRecJg0VKG53a93ReWkNxryBshlQSpJI6AIkqbazW1EIn5YksmIugSa8Irzn1H7mumkXjG7PL8D08g9HvEwnmhsvYUG9FurWRXr7QND0374rZad4CE4FmDiITl1nGohvvJltZ+ijVLfZ3IavCrd3icxUKC1zHWMi3aZMf6TddDTtzvvmvujSfUUZSK9fMz1RAvLBRoVsAzu3P4M\\\\\\\/EOOqCONDUzKYZ+g+nY\\\\\\\/WL5tEkIu61jsw7bULLIp2tEWnhSqSf8whGaFrJE2chzr5D94Lgr4C9wprb4R5BDiGrxK7Ocz2t3bmSWqZ48KOWp1QKt8vpTn50tVokbXndOCRJXyNqnPjYFSPDYk8PnTdAgUNBPJiR\\\\\\\/hPQKFKRjA3jymI8bkBOvZaTxvq\\\\\\\/NaE+yCSa4rdw639Wtj8p7YcTdcd4stt58R3YzlzvrTAvrOE9pF\\\\\\\/OtlyFc2jwA7ZeUJCky1gSOY5luRSqoLiWV\\\\\\\/\\\\\\\/WT399mdbQqsCECfW96Rz1Gm1jdNcCwc7iidKjx2NM3Rqdw+fg5YDnfGahveri2f9MhhZb0z4FzGHlpB4hZuSWEXxeHOSNe7VT42YnI7h\\\\\\\/PoFRv01HdbppM1aYmX3CparN6mQSsELKvWcLjSpvu9bFMuMY9sYizezF\\\\\\\/czAPg7qziI0j8D6dyiG2R8CFySXfQBHQpFLK\\\\\\\/pONgQFyEC3FCs4uTDgpGcIJC\\\\\\\/kIJ\\\\\\\/98mmj6m4yyUPcYd7Y1tmpAVDdypGmJbI7jAu4aIWp5Fxvr4lpz+8yTnyEcNXI46S9LsliWiLoe2Nbj+808ycgpC5h2f0YAGcfwN+oeAJq5wU9YIbA4Z3a\\\\\\\/BpHwNtDe5XyJTwB8PUS3r3UnIsVnHhyZ618hTuUONQTfGkjoVM+XK0yPbDqPDsMVINIDGZyCR9HuW+YTZF6bqkETsjjJrZ96ZZBIkKfbg1PenqkxyVu7dKIiF0fWuItiq3I1Q9k8lDCbghgudP1kYz0dJHYZm1DsxM8v3R0hzBDs7K+OIjPrPfisXzZgZ0jo02VZ54e1Kc7Tu83lJuBapswNgpjfTSzJWeQXt0g0Nmye9zJ2baVvHNGuMZd7\\\\\\\/foL1kL4pLUv61PXceXe8bNh3XNljysAF2RhPR5jlfRx6TTuv\\\\\\\/XUsUbR8276rauBYq3WEC7iJ5ItVfw2Wbjh0T+ojcfV5o0LDtoO+PZkv\\\\\\\/z2V8lTwsXKfu7Qg\\\\\\\/wZI29B1v2VtK\\\\\\\/Ue0VrPfYLTLojnI+8iIZDbUwbWx6biJSQqnmQjJCron5\\\\\\\/1\\\\\\\/lS29NRq83m5NyrnDAT8q5cTrOtx8rxJrEYqO40Wr3qVC6SJbi2hY0MoKMvwiQ5sU7jXP1WppyGWoYIGWvXc\\\\\\\/DkyNfKWuz\\\\\\\/8rc5Ipe7ecqvIzDFBkLyYGwQ9BCnlytmwG+VV1OsIbFx9kxStZL2Ir0DOPZBDLxkahbtSbCUjS3UqpnTJYKLPVViVOYrctngGqVhEqLeS+bKuAv4heGvONfHdyYallBNU54IVhESY3fuglklwvRtVdBdy54IFyW54r8oBm07ElfRlRaqo7MCahHBaFYo1KWgjRuzgHwLb8izHT6DRUtBxpRPTgI2+ooKFnFc8BtiqBcRuiHLJhjBTavLBmNxmpnCc\\\\\\\/mKB7Cwvje4RB62LGl5XUchvltfi51KIoN9Uq3lnoRGfnzdXJIb0SbXshiUvGpHxlmhzU+OSJyQFoxtui761166RokP\\\\\\\/XLMczNKwfVa1HQzAzANb8KOrO3EhNRN2sPfnxDPMdemGby5O8366anwsJx5IpS3w0bVral+eT93B5K\\\\\\\/8ryZb5RfxXfCrZJ\\\\\\\/5xPpEs7swM6r3hy8+yaG7iNkBbURULEM2OVzXIAwVse2JfOZkk0xmE6G3LzLhvlVT8G8LGhRa2LTLXrN1L1XVBo\\\\\\\/1GT9hZXADTeUY6iejH5ph1QlhuS6s75c2aweNjE5mjPZs2IUXrk09RJrHBcJ+AW3HCjVhRpnKGE3kAS4oaag54AQU7OFRkqsn6W72gv2MmpBDZHo68QLsALUUPl+ve4egtf0WibjuKNqvptF2uVkuC2QtBBb5DS6nDOzozx\\\\\\\/9PSAtNeLZxwvdrCQCaXq06yx5RgqRQnz7FELWsXTSKXY2A9VvfuOIAHh7YTm\\\\\\\/nP6MwOxbonsaabEBxAxGx4nYpgiRXtPk2CAElhCQ1JMWW6Hn7vNMn01+bVrabMezzeHoCjVn5asZMsXrlY7Mctdj\\\\\\\/AIZ96JFXFXUzmdSPIjrUCu6Cd1aQYaVM+vyBSqHCtiEpvqqEfoZXsO0v\\\\\\\/ofs\\\\\\\/f5hH1OTKxBlKhAtp8KIAq5Qip1Lr2Q1m\\\\\\\/Xv9QYUHdF1ZXZH\\\\\\\/BJTZzhGcoFW7Fg1jVlkjqAcFK+9Fqbx3CS\\\\\\\/A3tydEcCz9vvKFQtrxPkecbEx9Nnd6Jo2zuNPJcI02ygti5yRYYj+NruffQNMh2FVaNWwbubJ8toyedH2tl7WjMT4tyEInQivOfKr83SpImAFN4swhHPNMkLZW9SoIQFe0l0MpZozV4783rHhHyiZeLILMgTHuHRDqAXsMRVU6JLs0sYiQH197fH\\\\\\\/K8osMpRcG4O0z0PtqFthkQqHOuL9pfnxhB2L1cf0if\\\\\\\/xHGIOEZnCadEL0t631R6znhScMv9+zrMY8tTu0cmBaEaMRClE0Jn0SNxovsQNbuf\\\\\\\/0ns57T7lxfoWAH8+ucIPLNXbBICZQdPEialKmwoMTqI1lCzDAm6douOmK\\\\\\\/mANLs7KkJgeSnZh9EFyrJRwKnEMOiPJR1iAou3D\\\\\\\/qlUsIrqvCmRxZ5lB4Miu27vZ42JO7fBBaKdkk9bS\\\\\\\/4BDnh7bP04\\\\\\\/CY2cnG3SYNWs8dCRYV\\\\\\\/pc+fl0Hfqesv7Pa8r\\\\\\\/10jw+Ff3CNimLbqknRrjhZF787rrdDvXsOHCzSZhtX9BP8SnU3Gnlga3al5OkVX\\\\\\\/USGUdgCXn3oqmgALqIH77PhrEMTLA6LDXDFXz+8h6xJU7g84rEy79n6ufulCFj7tLAoDWf24DDvblh7kmVMal9NIRCKYBc6KGM6D\\\\\\\/sahFI9NiMqVnC\\\\\\\/V95OW9QWA91nV3UydlsFZhrbVl6Ydq1aFoD8xEjTZC5EeATozh526PU6d+76DJu4Pmpc6mwWvqayrWEN7Ppnn\\\\\\\/LG7lHUWEFOQGWr2rPIldao8x3uZKSHE8kvpZwgge\\\\\\\/7vMIEOzpnc84Sj9kwc6W8sjHhSc8Qn+eTl5vp3q3cqirUVkV847eyO9unB+4eBhi5lOnPG5mtH6gKDb3oQMvGgIBK5xZw\\\\\\\/4CMyWcGQgYpKGmXK1HFGak6\\\\\\\/Ow\\\\\\\/Q29ydNu3hXkCPsRblD8t7yfSINkxh4Pt0Jc2Fi4ktEL7rAc5CU8k48hfPPsQiEKW8fdj1HR5kIaG5twylz4BL191ZeGTGBupXwoTq4QHDDoKUlxGkqSwNWcgRThdGWh+i42vMmIGbuEpiRFtd8YCXNwqcsHmNPxQu36qpvsmc279MjF2jOS463QKs5rafV5k8Jkf1BpKdd+a7YZvpXZHyyZV+\\\\\\\/ye\\\\\\\/ytg36oIwngVGCpaykMGBOVcVyB4hyWI8fUMweJwmDF8eIlH62Ptu1DJ2BUSVuF1mCrLTBZVxwDePxsxcZFTef+K37M8Eza5Sa+rfTAZDM597TiPZuiROuEOgSZwYWkhoO5s87wtFeupNM\\\\\\\/MVTXGv19Lkrw3+1X+JRO9E7iC0h9f6LAkMlKZVWHWfCcmWRz29wT0\\\\\\\/fQyijdxMy25PLmmuvcQgHfR8zs3lYbrv+VxCRFKrsWt7uhxbjS\\\\\\\/g\\\\\\\/uUzXBXKOyFnudAmtGIhIm9O5jfe8dARKKlLhZrc98QnghvpWUXz11VpAAJFOr5smuyiO9ccARfvH5qChNDjVye4FDdCeQffjYxqWHWMApqSa4g1kLetBLZSGIDgnkP7RTsIUgIrxbR0b6vVWSZ\\\\\\\/pVKwf0DLKDtyk5WeKLXUwwNB8S2lBUl8a92B765XPj7\\\\\\\/CdM9hYS1PEPf\\\\\\\/BvnxtmuakbeDBxDZ8RGW5mX+\\\\\\\/2bjhlk6yIQ+FpR5cp7nvnaikZffccdLApsh706RadlNhIODqgvdL0j82SILij6lpC1IvTgCFd6pZ54HWLHuqFFWh6H3c\\\\\\\/DyEg0bL7jDwz0aKIFnDPquLeceXr5mFMystaWnUoZ8r7b6luN52iOhloBz8fCpwCJ3at1WKe8KpqGRqFGfbsSPuJvT\\\\\\\/\\\\\\\/SL5T46cXBbbI558zO3\\\\\\\/FQpnZKg4JsF0cWU7YlNs6GC6\\\\\\\/sNfVzYE4MQIzeY4bk7Id4zsTUew+j38j5Ym807xrDbzZtmTfuE\\\\\\\/NNtm4BOci2aTrp4MSSjhIAIOtMEWjmbPU0QHs38edmSstQbJwg78DuCrxixByIi3Va3oCcBmOkDMImU+IXVcYpFQdou3M8Q08AyGtYJWDxsgl+6R3UeFZYjHR1bJhHFAkN2LpbVxy\\\\\\\/LAIFvmdcaepD4TemFQQ\\\\\\\/Uoo8cci2c6S9O6lDkruMRu9k3eVew7JXn9u5uZaM6nf+rlZ28O\\\\\\\/UxZAoJSsaNw3R9wML5RcGVjdJ+ShHDGvdVzZSHCnC1gu1LMmRLlHbGVjCy539Hexd36h375lzkyVbR5raskemGhsjSYkvBJANv3FLp+xWKgLjarHC4Evs9wF4ZLiXUcw9m4KWdSmLxG8UWaSmF8AeIoenWdQHXi21dP1Nv9E+z6+V2O6EuuNj\\\\\\\/LeA97edTHG71DRXKBJmzokbZ5KNL8NWQTO7iSfUxcs4AmcizCjoB1okdYk2GdqWOx4iBfOZZ4S8nwTVM88H41xR+C7gptfeAIKhUdXQpLizdWR3o60QeCZnm0Jp9ycLr+BA9rQwzJG4rEKzibj3d0t2mOfSJlDZ5oXslh08SbeZe8gZaJjqedObwlKILgKYuD5RWi+mvw5yhrk9oEbwRcHn0pJw9BEigFhSVJ9m0Fm\\\\\\\/moZTY1r64bpkkCEJePEE7BmRcQxwnCtnZjDAgr8IpHTggV9aJ\\\\\\\/tBGldg53R1Z8hckyj+3OPaOqEEGDuuVDmsOLGFLEMz+OPFotHokXpyN2mOzhsRLgChP96yg3PXIFP0ONWqPFNBVU7oltWSNwdcNGJobeQvQx1rLVIil4QPaGNPyNaxmse4KtSnHdQcFrjJe5Yp5o6+K+dFkI7+p+0CMaAbj4+Sdnvhblmol33RlQyT\\\\\\\/t8UUceeAz1+y+yRh7yzRqz0JLLC7DET4IQf5SIwthvIV9SgCxNBMkl6QU\\\\\\\/SJnkQzxlaVRF9hWYjMiLO0tZUtLPnItBU3B6agtVM93xCFOEZXH9DOGJcsB9Tf32FPl3dU\\\\\\\/s2BbWR6e7sadk8k444U7tNUlhQAnODjw7kiC5iiebNu4X4doa4a3wJ1AlCxfch1tMccWywgcCkXsf4HH62caAGonGTAqBUUt8IjtFVMYsOcchKLi76wTX9t0Rsi0ss8L88x8U5j8BoUm4tWOIuRpH95EIvNZyTIt2Kn0C7u6eT9lLiPag9oR+A697hAaIJSB4bQhWF0TDtvhKM6ppKQbHtWRvQqizAniQOGQIjkCuapPxhY6xd9+8yaWhyMlAsxa8WoKMCDIdDFQ11juuv0ubobl\\\\\\\/m6ULNoanyPknhyJ8ChkRiwgMZDfy8l04LcxtpyN49KLE2ZzzGrcGSXRrH8iAMRzyveX4UErc7dFfGzHIevmlDw0L4zVnSwzD45fHs1E270\\\\\\\/csXOXV7HphlW0xlxkJE4lSgS5VPyPO141oyrGp033D4IxmO\\\\\\\/wCTUTWUZWJFYP+PYmBd9oF3LHH7dvN0UkJnV9wAKRWcPIcUUt2D2d\\\\\\\/CXWQgkwlLl7FakINcI6hNqzs47JlexMApsEKgchgmG7y3KEja0m4MyG4GOBkrGuBNgA3b24gQAR8bgeS9H2k6fuH57gxzHiGHNe+iDEF1TeKbN8JCeS\\\\\\\/rfEnsnOXYXaZ7cVwf6rrv8QO3z1LcLaaUHuaTkI0yVJTrB485Pp3JoUpiqjuVz5lJ\\\\\\\/d+LFQWTH4EN5pwirjwsVXQL+4iWSvCHrQoMEcek044h35UKhxQfqGhlNFEfxGnR2MZtnefawnCr2MSUDMx\\\\\\\/+6kBfrvoVQEzIqdvcG2wRDDcjPslGlLOikuTliIcK2XY7QFM8g8fLVq3fskiR6+TTmU9VSWroXv9iEtHEj\\\\\\\/BM+mOYCb+yfh4Wl1t0LtmyGob2HxCzhaD3qWaBF6HMhM6DkYjy8ti6LEtqcmVdMg2uxRZm0fEeSr0y36sJ5rPLBxDECZk2DXgj1y2J1boPUgw7Ddsz3Qr6irxnwqiuEaZgk3ntxrm7N+pouTVBJpBc1ZsVhSoRMF7jqUtyqDy8rVjD4jvqtuHOfviL\\\\\\\/wYVnB1EUu0G3C6BVk0GsElRePNfMcXVXSLeZaFrXHeWIcImeUFS61A256VZmhGaVTxYFzRrFNfOJI5PZ3c9wpW6ky5ryJqitBOw2+MI6Vehy52OQPpPhrWPE2+351fseAYnoUwVNKdthNNWld5kASVHgzlgv8xJpbb6kj3UW3jgjjT7nRZSOthyXMlmVYjfVeq6F6X5R\\\\\\\/\\\\\\\/cFRYiE6iC8zu587k2LsQAflsrSVUfPh2YjG3bLI2hKY+Xz0wd7aYL+6DSLSmXiyXQyG+Q4Cx+kTlU7lO310A4xQWCZ7q1MJE1riJ86db9TCSeRHeL+JwBv\\\\\\\/BiiiOn+6VBdPOiMKiwh+1YrpgKjA2cMzVUG7YixKDS7saXTF8pHmDN6ATAOa3eWeS\\\\\\\/XrIjPs0zUvnNl7XggFzAhhYmoSD21BF\\\\\\\/+x8WEQy1+wmS\\\\\\\/PTQImt0\\\\\\\/mmiosQuLOX+DSyco2DwHBlxStzYt+SghD48CcI\\\\\\\/7kb3xGFTNXjQtEadGThXf7vEI7KjMR8vaapmshada7B4b3yIMZ6oEupd0tTayKz3dlkz+ZNnu9sNcYe46SgZCe3r+5mYY9aYb6yFdOo3xcjCg08xgNe9gcm9zwQPoU4Y6rJvlqERaOagMknEQ84F+zItUatAJSwzXuENdXvxE51PuSRSO7BSSY+V1XyKNlWwK4wOthqSbOx4UvqAkVGNmV6anh\\\\\\\/it6bJT0sHWnj55eUIsC7VX5Y14NJBSLAwhRGcrpMGFrn81Zakyx8+AbIgyJneqPm6hLGtxXxHBvli2f550nXDVY\\\\\\\/DC26QL7Ve577pl4McXDI0NDwHFkEurvT6LJHAh3YWB3qsx1XIUrnQTy2ps7wzgr48kdvl3SUTDAT4GSVwUG2kROKHbc\\\\\\\/kS3F66ksmTAMK\\\\\\\/BEXieohtRXpmw6ZeLBvSV3AXYr2q8nEFWc0FqoougUmsdgtXN86MAxNPDGidKZmMGyV2KgfW\\\\\\\/p29II589xPbo0iONKPyq3I4eB82mqlltt5fugh9AjSfJ9TGDwQ0MX78xGbOGMR1SUqFF3TTPA5Od3EIkj2t6vh6sUSbciK9BPtqIGSNySIrT7fsDFyaViL2He4WwiNyu8fI4Tm+j6tuOquMID5HmSpRr2RkBg5EmDnrwZUsz\\\\\\\/eeX3jdt9BZxY0nPc\\\\\\\/sQ6JI3eT9qAH+Glb4dXbviJvfdPh3s1OXEiRjddSttvlHWr2UKyK0nPtbnxLt749lALhtS0eRIPCHllGnxFHNJJAlgZZysyw25RV+kUlRcXJYkZlTkrdcV6pczTQcA+EnFpoGc9sjdNQf0srluOpsL5MOGSVcctiBjIm6AYNsiprELmSsX\\\\\\\/8GDe1E+OEwILThaCFl4b9ntw+1QfSy1Qkd6dylZ8BUFQkTjYAJE9oUBo3pF+ChBE9zsrZ\\\\\\\/3lugc0WvQ\\\\\\\/Dp5btJn\\\\\\\/Ycu7gjibsr\\\\\\\/MPZC1Dywgein8dMWJ67n\\\\\\\/NQAwV2sR1YLrPUgoJrM9OCiE+HGgz7mNkOy0TlPBe35VeyfZwAMsAMeyOrV1EJAIP9jEABKvcW5rGWn12KCqvLxsRevXmeZY2nyVthS86g8KrbzNzcayRRXw+X2JBx3YAn0WhvL7BfkbpkdRCAzbx7kofuLmJgCIUlS61WkQoMglk54EpMNo8T+2snf7WUgzjT4lKx3AmY++ebaEcI0AVlUWakLCqQdBdMbV7bQtEviQb++BmwG7HxCYgazOu75O\\\\\\\/1X3V+UwRlXqAdqlMRqX8insyzO5mAmtIssT1pzUDNDv5yGcyDV8W4iYNuWfcSsZFiB3WGHE\\\\\\\/J3HpeE58dlwi1ItwZLijE4fBqI\\\\\\\/MaEnIesdOZIfbLXHbkV2wMctCn4oJPi\\\\\\\/MF6rtVuue3\\\\\\\/pU5pS6RqrnLkLRkdVCEIjjzE4h3L4C9\\\\\\\/D+\\\\\\\/QlmjkT9eT7yoq2BE36ZkwA+\\\\\\\/Fpcg8gRhgLambHjnjp+JKG6fadDtUdAKic+qxVazQC9CqdjArgXDkuN+kbpVhu0eOczR1amHycHy+zuq5Ew+Dr342\\\\\\\/wdjhq416czQ06iz4N\\\\\\\/MkDINpielOn0NYwe5hI1hmGp9nzj1XItSbEoqyc7rbJnHtoqs+RFP5Bd6RBhqWi4Ark\\\\\\\/4fT4WSNr1cntaeQsUsBRuZDGLH\\\\\\\/Y0Cqi5h01hkvt1A\\\\\\\/+Bx4OO5zoCoXbeIt0UbVv06Ismy\\\\\\\/6PVHR6LJ3z\\\\\\\/FacXIfydD9K3T2wVgbTpoEE3IkpNPseAzbym22XVJK6R3wJ2lgxbW3i1frWqhYB8UlcZZzwCdf+YkMrq4ZaT0\\\\\\\/I3Ma9H5O7YXk58jF6ePIaOQUkML6f9OV80tWAV5iF+xszll5C6pDgEwZP0LO0epZfUHB+ZhJgTqK45hQ+Z4oAeGiVfuH9nzAJEQNe\\\\\\\/9abGAP5ke+ifNhrX\\\\\\\/+GA5ZOel5gSP7WtOq1Z7dK9WVtznetJ76ES7vOsiW8JSOLjluaKxLweSS2B7aZh40AIT+HQZkVeYmT18BR0Jw7vfiOzSPU90ZwkM4USSgPvEVthHfA1Jesx8yKltna6i7yrp\\\\\\\/HbKGyn+XAfD1Kebo6+I31h9X9fH6qg4aQ7fxdBLHaLNTUAeqyR0sPKRYeyEEMAUSQZX0rPFe33d9\\\\\\\/tBPxBhKNHGzIH2DUqFzuSewfFt9BBKlTPbSjoOJcPw03wMZC3pvaOlOeABb9LuIjRVeyRI9WIlOFU5ouhu1\\\\\\\/Ay\\\\\\\/jar4KD8LBR8KTKwue1YADhIUabwDgAK3O3Z4Lse8f0vh173v0B27JIap1IzkBV0DLdVEl+xeZj8P0w6NLykBCg0brLhr9S6ZQF\\\\\\\/1e6pADn5EFfbVWgpzQfzzmrOHxRspB6l3XR\\\\\\\/oS9mssOVRv57HKvcDpl7Ko0SGpZEkMtx7cdp4BBFhZ4jdNb3PtQnEVxO8RlFJ1bs8XinWKKcrkLPvXzVsCK5O7u1THK70l1VhWUKviPKL3DetJeE7+7hOYVyoudR4arwACQbVSUnGFVlZO14bh6TuEwlpsVVbsF9aa0TCzh1882XkSwmZriJm7SeQaOlwAsyaM1cK88HLHWI3Ei5cxvq\\\\\\\/L4Ky\\\\\\\/c32YbAZWsfy13fd7VZXNx86AMo7yQJoDg6rlGu4Jw6m5qn+GKMZaWnh3k3gmTVINMvAIT2K3JznJ8LQ0trCmdF1g3CiK9dtN2Mw5QBTBGQaio6R4TTQaYPB\\\\\\\/Qpns02mm7KC8iSaRPQKsXWQyA+x3DVAVNBSHpCQMOUONyzmbr0i\\\\\\\/PGSY3p5\\\\\\\/LZmk33ljBvMXFmlgygS\\\\\\\/6IyidHzdzQR+E3mfbFA5VrvYfN1lxWlmrWh8ZvclEAlQ0vFM\\\\\\\/oprNCfd7451kMUQoLUJ8LqxSz7e7iIcP6Zbnda9AX\\\\\\\/hRLJz\\\\\\\/D\\\\\\\/reOgXMtZxnFgOZ8pX7KOolnptleuAcKmce132e2annAg7qzgRguk9RfCpN5XP9fppA4gcKcXHZFW99DYS+y2Sy8I94Wp8xabuIZjybF0VBDIruATnq6mY\\\\\\\/UXEZIVPbYS0Ul5YyP2LSGm+b+X\\\\\\\/h9vdR\\\\\\\/Q7VeXA4oTl3OopQKpgLcLnxILoEJahhZ+\\\\\\\/u77QGBdVH7l5HlHGQtnnkEEmeq\\\\\\\/Uk3eQl39dXQjjFNJ2raJ53BG+r1bP3yyxwNEALdNRmq0BNoJ3x\\\\\\\/6VZvvwYVZgp56KtjGkPQlm2y7vSV6AHnGVx+CzOOe22g\\\\\\\/tSy+j5p4OIa8NIPKqa4LYn3Ys0+0HXKfojTBDAy9WQ0QBYDESqG4gQ1i1uZpNYzmPWkgs5+AFT9R55kzV\\\\\\\/\\\\\\\/Olo0ZuFZRFvRgoUw4gFUSJo9WpapXqWNIbjuOLAD3giXWYXSYuBQiAip2spOZcWg9Dn0wh6O17Sq+P+B46LQybSV7x8nHnMIOxFx3RNr+qIvtVybci42HTRXX71SXhL8+PYLac4ppbMTv7eBjg0\\\\\\\/hJKeN8la4EYNPo\\\\\\\/V7cVWRPVkGP+uT8vJyhEEKTOJeTP+1ubtaikmpvTx9IgI3zSq\\\\\\\/XPLslIXlMCd1mu5Cjq4\\\\\\\/dx+SDoFSqlTYcmEJkJiZHloj2vK3a8xOy7941pEkkFz4t7knXx9jP1PmaI9N6SQQ4bbxXXY6KpajDAAx2seticuxbQFwMd\\\\\\\/J4+KXMGRxsPVMkE29Oi0xo3qmA5dKpZPyLjsYb3\\\\\\\/tU5qqw6ouyzlks7kawaqRTMx5o70MytJP2KyQRLsgD4B+hpvp\\\\\\\/VC0pxrszd7PUyg0nO44cL9pfN3GtaQJdosn6FPl+Q8XzH5L47LAWLzk0s8\\\\\\\/CIM2rRbfWo0InZzCMiyKb7ewgcqpOFkvRRFgrunAqhRIt66QnHqRTZIMXIILJCeCCO1VpEhObAVEEQNt4ZuY6EvZ79oSGGf15nuSlhl5ZyzbFQ03jewjnacqlMgIWK\\\\\\\/oJeER+IERyxGYyFRFXMqAIvMse0WkOYH6oRtOUlORBMObf82egjBJOB+oThWjL1DfJaplkNLb6VjnS2zoelb+RxyByWhYPH8FgpgQm62ClqDnbmECwFu01aMcF2TzxiwBbzVfMl9ahxeSRpT\\\\\\\/2Bt7z5HHp5ur8zaUw1Uas0KsGZ+FoSbt1KpS2bTjh2SruhueJtGulFNPpyuASGpzjSEW0498I5XTOTsSm1qJe1U6rHgQT3ZZ3\\\\\\\/g9ni3t\\\\\\\/eU50zYJAy7ND8\\\\\\\/n7L6AnNLmbKZKuLvD4p4ODWok2W2hnY9OyeyfSzTcVYQdOxYN17nhKhxWL+9v0ZSYD1rG4LlLyQn5W3pRrLb4OmRYlWQEF8o8pW+k9svhvH0461ZVMRIS\\\\\\\/\\\\\\\/zbLJa\\\\\\\/VvY4fear215Eddskw\\\\\\\/2UlGAslFBUZ7r+O+g5d1hqUgoLqynG+77rPHu7cmreWguTmsG17TmKMafpq4+LmR6Iat7dgJeukkLe881Ug7r4cw9\\\\\\\/bLrEcInE72jbySx1KpfzLn0vQ9nz+fGHp3z99tjGj064UKRHcFdlUECUJ\\\\\\\/Cr\\\\\\\/7CCfqiOaDJrqX9DOxx59tM+FMGMoieXj2e\\\\\\\/5xTiAcs1TRIDzjZmNc1r\\\\\\\/n8P+mnjaQ7VOu1Hd++6Js7Tp0dv+2JV9Im+p9lnqpNHytc0AHXNyEwCWN3Osp\\\\\\\/Du8dj7KiD5LoD2GY+1L5LSVRvDOePb7Z1NDwwPJFS8Ck8rbr4Sm4m26cu\\\\\\\/\\\\\\\/OOFyratfS2NDRiUE4BG0w+GfznkNDhH\\\\\\\/oKjitDEEEnF6NOt+4evI9HIpvXZhUFcTj5fqgBLFPZOoBfEjVCk3u7zjiOT+Uf1Xid5ewUMkzRl3oWTSIqEB33OfWRKWtZarVp2NkhJNfjZodwoylUFQroqvW6IVldzTuaxfe0o3Gcb0BN+AxpvF9jN336Hlg\\\\\\\/PhH18npEUr7ILjnY4\\\\\\\/j55FoYsu1vJ+pahiZhucc1Z4umNuggCtYXmXoxByN0btUlFAbJXJGkX2zJgzd4b7x9vmr9RgD4GXZU8ilcygVdpgP4KxFmjKPSBQjiaHSp3I\\\\\\\/P9caW6ehBLscHsSkskrlQBDZfnyfr9Abj22eYN8wsJtLzC3th3H3V2L0aVfjmRsLloZ5B9HJHOoTY\\\\\\\/YvnJAPTmQRZgIyryNWf7M65xZ7v3v95j7ZBG4PnetekgEzePqDS3u1PXcjYDVOk+c6S7D\\\\\\\/siwi8Cr9\\\\\\\/X3VWfVg55nt05qurb5Q4+nbMo7pgVfOWvLYd7Y1QsVe1XPnwj9SMwZ3qfc0y2IW3cj\\\\\\\/H+cb2xBxj8mNsOPoEmXMAz+Hz5uejVrfWjM3dPodH2JnP1ICUwi\\\\\\\/cah4Q2mbIXXsCrvpZmVzlDeTTsaYQABztdAhOIrRFnRdRCs6V0sIp5PW4MkPpHejwmWLttdGoZ7jyV7lJUS5XUK3XURTuYmPL7WvjiZMKWnqAoFHvVESTCUmqw8j\\\\\\\/Opjg+G5BUJyT1pCzkHvm8AVMtrZzPXwqCcr0SBOInY78pIG7bhiMbSoYiRYahdhGX3mxClCcpF\\\\\\\/tqlP9qzsOnt4jvogwNnw4ts2yT+tlCuivPA0H3mZmrRR5y3He1kCWSpG9q8Xvmi8WNeDrm+OAY\\\\\\\/TOnHiZCQZ6Be3d7EiYS9QZ6ws6Mna4cDYJ7n1Vr9+KpTVsk3DWNluuC5oAJ2WLg9Nq8PJdT0sd\\\\\\\/lyv2r0fYKObjZbqb5UXYMY8INLxAlilaIQFjCH9jYq5rmwAqQiANesu3I73qP3yOTFQOYRr6K72rZM8VJhWEyiaeEETNAbrkdC+9CIjt+Wv6wMYg8l23A1wQpcysqxT3S2SVjG9f8AvORPWzELcPpLDehKAycgnz+N2tO+tOjOAa\\\\\\\/iUBJb2UEeQ+gbjOtjXZjUGT9j8Y2lUKgDK8amolqaquH2T7CUdU3psUhcNtzB2oNYWddKYo4NIlb03NpqCAgivf9\\\\\\\/Yb2Ps3OLSgbJ6eHEQPXDj94DqiedIcQaj7lG9pvQTVM0kJpMv9jF20FCXyF0WoFqh+2pgz644CS5WJALxLlCsDUufOpwjKrV\\\\\\\/rjUKsWuRH5s0UT3HcIWQ36vIo6JP0yr34soE02NkVTGYCs8Q2P88iYTv7Sb3MRnJqG+u9CkKTHlcT1FqrIjlN4vdRL\\\\\\\/mtsFJybJqnCm4qWF3GIL+RIiCkzZMX3ym2Tc74D5yJXGbIPG3Jn9WS6EB8i1EWgWlN+cGHEy3Wa0l\\\\\\\/ZTul5rUTl1KVIAWgqcf2v89WlMTYZryZH0aNpSQT413Rp6r78jDEKJ873u5pQ3oL3sEw+IpwECgbVv8s4COq2etAOcUM9GH8QYj\\\\\\\/EvghvuuLq8imGB9PK7yXBsB\\\\\\\/ZUIlLXa+9pxsk8fshtLvJIqAKSrXMidSQyvWHcvsfCSDqVjyx\\\\\\\/\\\\\\\/h6lkQqeqvJBSC5YfMtOmolAol9lB1RQMvxKPQeO+bwLogCI5vF5z0UbpqyxRr7fTkI3mFyw4sobnoxypgmbOaAxoO0lM0k9jblQG\\\\\\\/k7pwpZN2r2nMGRPmlUbyK9t6tQO5RBs2CMGVz9SXn+1q2azytgp6QTP8q4LP+XZz\\\\\\\/mVct3Aek8seA9kE5j4v5KDVwSQNpZHYD2LBGZvo7oD4ANMzPjRpPqRWQBgHs9BmpPtSJqsiu1sFNjsDmJWE5M46s70H5DRdIaXwAAQrmejm2zWA3uvttl+OcvjvDBUtjbwSp6DPuG56GLU3y2TnLsAMewBpoeCLTG50mDilswGYSLlba1zLnREZB9LaslkNj688P0Wd\\\\\\\/P8H6ixsqnTSKrmnvs128aa1vZwLvbvQkXebqgdl9t2giUOamjp7pqlyNkd9tbkrt1SBvNe5DBIWYpZQqScSXaPLhck6SlgzzYaeXus1uX5egWS5zvRsa5CwnMTq7wBBbGlXBqtNyF6+A5VbJJFagOl+Q8IxrFEY2ex8Klf4geXrzwj52whJ3B9C3KGUVou\\\\\\\/fo16bmFZUKA8nZqx+cR9WI+63lo6edphFCBxnKpAk70upUXgye4p0xEGxkHPClFTnE096d3TC328o6I4yEW2uDDfBBc4\\\\\\\/s1HkLndTbWPD8gSIXp0FPys1xvzN73IV8pw2Jmxtizc2U0Gf9eQ9B1epgjtsbSVnwB4Tyv0KZLkP\\\\\\\/iqf8pHtYkDMl8ZtMh2EeMKxJeZpKKaZe3uGIzvlMSFbfk0o2auCKLX7gAurODB6qIgHdjWyu06qUwysk8HUhRg2mepMhJAHkr3Ozg\\\\\\\/6mgv9+gAmtr26MHbIfqkqj2nb2XSFEEgcxZnwvtFCm\\\\\\\/hkKrBCthKx1wF2n1rzaMCd1486smy54x4ffVbkVkAtFbxGQhjXc5Zv\\\\\\\/aWNpPij\\\\\\\/MiC2+piG+3RJjHIRz1dbmV612l7\\\\\\\/urK\\\\\\\/jmB0r9WldFDUZaTVU1iWDF5e+KNyGp8MAZYd\\\\\\\/a\\\\\\\/Mqy2x48Hbbf5EiJPYdpaMiXAe8475xeZXLAswChgTcQZzTS7ykHVVHGmma1Tvigfuz7RYMQa8oI8L9HIMifEE5o3YE+rWLjv8IIPLvbCjW+DPb9pMZ8rYmYs6TLBG8vfT+KFATgpU4czJcRn6hYTkBKxW1Kk0pjx8a1dmPf8g8qvURATcyz5VR9XGjoOAFE0g+2RFS54xDxFtqYBs8iHYhO4qAf\\\\\\\/wDrMHZ4Y1EVSysZh5m2yAMm0bZdyidK9J5AancLBUw+xUnKIzBKVPCBvJH9OHfkb88QdywMmrDCG+XtO+yh8Jn0WC3AavBZ8r7x7qTwgSh5mdGoC\\\\\\\/i8PtmPr8ULichA636QJRDiCIgjrhU5WDOrbUBcayq4HM2qGLUxvmxr0K6fHucYa5nyWywRFoWPE1NXOMAVc4Jhu5WsYjWs0snEJj4DE7bmm0xUgtOpgr3CvT+q4Jasp\\\\\\\/X6Nrw6RKpWYwRFEPPCpTcteS9ohPFEG\\\\\\\/6uUdZmBhgX8NDSuKhJPS07qhNb3ud85doJZyBgY5Pqlu+znx+douhRlGFtpFfvfx5PcTFKCeboLii0DSa6Z0fdkq9XJRexNOVuzr6kyoZWc8+KuSSIBbEm0VqWh73v6rs3NGCJDO38f3ViWkPGTEddGJdQVkkMjGVtIhe+RAoR4i0gV1dsvOgPfc3xq7ItgOK6KNS2J4lAHCTJTEEh9eJI9o0IYX1GalylkBGbLuUGCQugQgq0Si2KUI694TqR6576o2d0ZQi1+Ex+znduGYRGq9IQZA8Wn4fCY4Rt3xwpxi8usxkJp5QpkPR4iCupHrrypWHFW029KX36Lsc8WAZdD4WmMwq7uiRobKAK3X3LNaHB1IOUdrgO7UuV\\\\\\\/KUefnUflmWaFE5+asvBUrYuY4mSFtrOXBERXoXfq\\\\\\\/rVpuNFfJH6XOveLxuQiE03R\\\\\\\/3xINEVofvop4WTBhKRH9uUVkmxQXwkyczi\\\\\\\/Pnmx32k8KBAAY483vk7fnm8tC5Xd7srrNtBJHW1f4\\\\\\\/IdVv7uHTdpfRFfq4XzPvesOTpJSA53tzyrOlZQ2tWqBjy7\\\\\\\/IC4g7qeBL+xqValZ5Y6Fa7kpi9x7+IO7lW3B+13VEhvgX38U0Uyp+WqWOO\\\\\\\/gdYIFeLyH\\\\\\\/kr3cBN0QJ1Pylum1eZZkjEZfWMNOT5alM8KXRN7M\\\\\\\/Om\\\\\\\/2w4kMsX+obSjMYuIBNUI5yxPfjXcSAcCqRo0s83EtOUKeHOX1\\\\\\\/lZ1XJgJYunKKHbGuufB58TPlCnbzi7bnsaa3fi\\\\\\\/O0K1RZGVQsLKfoAllCf+2+iRNb0L8ao0zio1kxalMWroaCRs\\\\\\\/FfLjIPJnVjEf7Z8FT3r2TEuHKlVYdrxmsX0zlzFVQcslShomO+v3uROPRpvehgZdhQB5V5A9K9ignONPEr30WAH0dDtPJRds5fewdQ\\\\\\\/NwzimepI7nbYdwTlnr0BhWt0HBXTlfMmQp5NKD2TWdmVnSp56I0BoRlRe\\\\\\\/GWgXODeCc5KKSWhmMf60ntx9vyicSnM7EaKNmOHuhgV7RWQztjBmwAa2Upuw3sXw9y8E52Mfjuw6uDOh38TAqmDbTV\\\\\\\/vmA505Yyk9kOw5my7wEQY+NR0nS9WAJzWU38DlzOS1KLbboVzsWd3FymETpCUIKlLvb6z0hMWDq01upsqYqA7AQExKKzK2vc1tzgTO4oEgeYuDCw+RPMYBvRAmDUqOR8M9ur9Je6cFq4lcfrFo8Wsq8NWiO4R35+zOnasN10h6b2Xwrt88RqdASJRbxcc4pi64D0LHmiOSfF7xMqBWsUxmTs3DfV\\\\\\\/sxyQxDeBakZFOpgSRPjOGIoROhJhWaDvqP8Qzvu28Q2YsMQtiWXuEQ2PBlmJliZORK7IGySuQgQDbnXE1Q4WwjVgNenAjLecgiq8u5stnylOOxnPBHwa9TMof6VHTxTnwIWH2d6Xls5ehp6MNowey3gOrkzRD\\\\\\\/H1W8kNFj3TA30TOxmr4zR9Hy1M5NLSaQL7gqawueGyGUURiI3vF66X6G4aycrH8TsxCgDaPoZ5mLOUWjs\\\\\\\/sdNGOH5dQchANez36of91GF+d9\\\\\\\/KUPfo+cOy3\\\\\\\/6UlivQQq7i28RMn7iMBUxZOxwluV6uvTi0M1hd73eOLtKh72F9DVlfCZL9C6C\\\\\\\/u5gGi9xCw+Xg5izySNMkRk8990o9hHSE0wEGLZLyrhUSni9TpsIUGl1S3crZ1nxLe+eGDaICusU82GmlVpoLvtqCPkqFnwIgDdvY1G0AxrlH2kOLMlhVvm8aXz91oUXlyq4BobZGINGNfIL2vx0IQvjjcFpy50NpBqdYAt2UtTkbJTf9ElzfZPScvNjQcpV8dqtlnC2hG\\\\\\\/q3uc4jRaHI6CCaXQ9ssAC3356SUw8tGTJNbvpT5zbP\\\\\\\/+MEjnFejoDNYqIYhZS8\\\\\\\/KJ\\\\\\\/8pK407hx8WPYki7+k7L\\\\\\\/E5OyIolcJiJ4yhwjVdCKkKXbjzalxIOKgUq4b1ebxRvxzT1Zmlj1aIVZe2kxTayrz0QbgxyWe4RLUjqaeWLgWpLQHeN1qIi6DPQ1Im45GWeSKJ7UEbEUcNI2BTgr6gV+uAkA+AE3R\\\\\\\/zYLqs9cx9ZIkkDByjw8qCN7oWcSfwdtcs+sIw5r9rzTOnrThAr0HQoPIegWVTKCojsGOpd2c+J8HmOmGtqpcLKbgzprpw\\\\\\\/u1\\\\\\\/mjuFqzniyMC4+9ybEfK3JJ21ehEiITFEqhsM1V5Ij5aVRhA9b\\\\\\\/E5ck\\\\\\\/yv2PZh8JCU4K1A77ZZdA6hJDrvkyXIFTkIi8d2ANlOUgeEHoq4o8V7N2wyNwenA97Cbr8UrWyDcjjecbMvA6BFMH\\\\\\\/AP6VosTUwU6teYZwKxHhQS6MqLB1p5O4WcjRBqgz09ClzKsSk24B0R4hdywd+onI9MBtE+rCJB+Gx7IRmRrW4Gsq1No8iKy2+LTWrPiW8fqNyAdcmCMlSigiTFjcuXgG6cnvyMvUmP9nkukw2CVxI4k2Wd5JgkqDHTcqZmdhpAzHAW5NGF373sJUECSKILurafTSNz9Gx+h0eLME1n16UmQq7ulLmaAPKM54V4IqWESmUWBDNQHxmumdJgA5wIwfpWuhxOYhOU4bK93XJvi+rvnsL6xRcr8wvJdV44MrfqqsfNSBJ\\\\\\\/tw\\\\\\\/dem5ghIxhR6foDG8z6XeRpqzPPqrm4+z6O\\\\\\\/WETuLow0AVRQzvDI+zf+hByw0Jy7Q512l43UCHWb5ZbNytpDHYnjiLEF1t8BIBx7A5mzMdPoTulYadMyEtYCLKbKjkAREeUmN6FE80\\\\\\\/+f2r6d8qtLOfK1QrgRgVfoYHk6LExA11ra9jBwnnMOABpnACqMV6wtZACcXdq3gJsYwbFOeN8W+s2a\\\\\\\/v9BAG1+h1GH7n\\\\\\\/1LmvUlcCfQ9g\\\\\\\/fHYZ\\\\\\\/g8j5N4t3eOwWPC7Jzj8vwydvEpRNwCem4dnNRl5+\\\\\\\/SpGq\\\\\\\/bGaNqWs5KLYvcNe5l5bjUHtJZzoebfNb6VR7+OD1PLUsg33KQEWqfabzUV2Y3j469uqxx9HrQ9LbZGU3\\\\\\\/lFr9wpV7jTkgxnNfDryojeV4jJfXjLErhowkFaFqqgXqfpmTJmsDIR4zpOJocRx38UlszV6D5Y\\\\\\\/BkBade3I5ho4OxgoANkfhTwmRE1GB5roWQCkxNp2rPm+J6gNGu3jPd5TMKiejIrHQTu+l8sMibdW5Qsy8NFEDIOzfhPdiIHCRf52cFA4OxPhEkYlVLYZE\\\\\\\/gewix7e6caRbmqViZmz9Yn0SnZapfFk5x5NPKeRrEXVHSTc04pAulraUH2l4RNLnfwMDkwmHLIgF6Xz5BGoIlBTO0rLJG0nud1mGvVtSRAxai\\\\\\\/6GPt8lHzr7ySwmBsRpANt+SR\\\\\\\/tSFm3hAmB1MENBNV\\\\\\\/nBtAS8XpUzHg19MNZ7iop+ND3Vpw5a1VLaZfl9WgCgHYc\\\\\\\/NCSvmRszyBItLPoqDws4zSEg7eY0bCpccDSL\\\\\\\/nax\\\\\\\/deY6At7vXm\\\\\\\/3Gf18UBEMvsIsqDr5Y6VjK46xXVyn5YY17zjDeoIy9Tw8z7Fyy0YpBjuTS4yWj+CLD3QZEicz6IPywidHgej5siv9y8Zq+z9kUun\\\\\\\/6MNizz4Nerxf+KyAdTGmdFP+hiXzXLEqmMKgolS4TO3wFu8T1vxWur6VYTK8cLauR4UWnKo2uaXyPOu9JTHqKAWjSpOXhBp7kvMqelHGTSDhRUq4qCLKdZqt1BC97mo30bmqmEf925oXEY\\\\\\\/C5pXhxx5jzaMr\\\\\\\/2HZugBXWeqRjtAimY6822RXyP1cHRhX+0ucDGddJv7N+NQWTgG\\\\\\\/vYl2YHJcd6QJ\\\\\\\/Yq\\\\\\\/v9NZtJdnoIvBYNVb6kWZnMEjdTlv7VXfzhg+jqrzfvzII+aAiy9ofMEM03Mj81NIv1eF3I9Iem\\\\\\\/Ykl1sJqoBUYniIJGTERQOxiP029M9GIV0C7kdBYcyUOV\\\\\\\/5orTTFugt199+wtwblC7DZsf1R0bnoDr8vrOKUpTQsHme+FIkURoeqSH6rc0UuLQclQ4pma20HPq0TbzR53vKXgLnZkenAImiME80L8T58zlTGDgCTeQWKaWV\\\\\\\/zKniQ5aDJJquGVT+jKPYANRu8EmfaqKRRoP3zN83NJK+PVTYWKVYvEbpXNRgKnHPIOpLaFT18zzmmymIfUCPjz4skVW1Z+\\\\\\\/2NYyNr881UWi++upQxjrjoNJfvofS+KFQXM\\\\\\\/T7PX9156Q+bubxUaOBvMd6fReFbGZeFKpRwhE7U2VdTQwCfOIQexTim1nNQ5I6CAPzWmUk58Grdm0BwQIsAM9mre0DxCpA9N6Aw\\\\\\\/vs1xIAxGHVa+Ea3SwBLFVTw9oEYimRlkIIDDfufV3eWIJkt\\\\\\\/Q\\\\\\\/nhKvz07GmSyfFqlhgl8kGaB8871gfnXGn\\\\\\\/j2itcM\\\\\\\/u4hG9Dzx6zlm6hPq34T6im34w3T60fQlmvdPR6o\\\\\\\/QFUJQNpSQT0UhaqHjLXA3jWalY1sw3b2YuzHsDmdXe7WIEOodgIsiynE583kMkFpF3LSAzze72OgBi9sKEY5aRUS8PXwJiRKPOuuyF1Jtxnbz+SdrpBhSPaA3r6d+TfBThWLMtol3hZO1nF2paK0mroseBlieYYvkQaXAiJWsoCkcRtvL2Pkugo5BOYXwzczCOMM66ugz2jUTtr4mglnttsLtgzuxRrjxsgwI7p0NwNvSZD9iClYMSOU6sc\\\\\\\/psJWG4SWn1GFAmSm4fniWQ8HcO8dNU7rKGeTMIBk8plEPJwK3QiKZzVFdTa++LaxfGk7nfgRZpf\\\\\\\/qITst8dB4EF3vPoP3ZVxDmGpNSvJxe70Iq2muYsb5hraJ61yKnAf1o8AbYhZUX2CI9Cg9QV5igpwwQ\\\\\\\/7TCQ2UgIbiPNMwbQ7Kpug50c6qbgYSqigiMtLgzw+T3wl7d\\\\\\\/ubVmUU9y2tH8HKhGJSlodoGn4TAfJWb7qpwZ1sQSA1os6hCcRYPX2oPgi0htwIpqLrxNn1v3XlpuIONcIZtn23GtLPOvEFBBqTC4hEqvEr90I7UTMTtTYOGlkyQ9DzEyqCj1sMXqncPRPSD6Sp8+4NndNO34GG7wWeuGTgY+cHuuFFJPNU0WOLW8s8ebg2x9xbG32EwsxYgrZlHw81\\\\\\\/gibvKj20zREcovU7CMPv+5430KMT60OjdTlMcdwFB3LrG0BE+sdhBbP3+YjQXxFsxnhN6IJq06y8eifqgY0yBmE0zSVk9v35Wp4HaZOgKvKbB3ZOlyOEJ8HT7ce9myKEXuNFGlTdyqmGeSHEpHkYPI7xTHMMAqvq8v4e6H58ZvXq3K0OzNpB4eTzXexqnSc\\\\\\\/tgz\\\\\\\/UA0qqzwwXm7Bb7zfZXRNfNL7nXbap7OaX4dELh4KT\\\\\\\/ZSWFJ7yBQjCOZFwsYqOZoeC5vGsqXkn0BlstYe0ZTO1QBJrzMd9MNBUfWFtTe0X6IvaHp7qVbDG4LBAeeSpteM9N33lJOWq1aECW8mbIZ0e4rvoyzjVvfM+QfA0dO1EerVjANlLnADuOtOA0hePet1N7q90rS9vi5z15TqlL9cztJoAMTgcGiYUa1SI3mYws4bfZZYG7BWwBTPZTc5Y2m4\\\\\\\/1DM8fWShRO0LHLqg5yF92+mKEhszAQtLBAGV568oHBnTqPJ5MuhIFaI1\\\\\\\/8sgLZZtmrC1RmgsT5dZKJZMSlFXn8gLg5m7JbChADJn+bP0sHid7aPwDKyEZ5uJWB1KMviKKzI2gkfrPZm\\\\\\\/\\\\\\\/EbZXXeFOVg6bozSImK0O6QA7W61qQiXeoqrzN26zH3IpDVujIgCwn+zdSw28njBDSxiTbwN+dVCwncoSfdUc8HUbRtT\\\\\\\/+9OUbJhIwnT1cjXKYfK9GxPqV1ahDfYZaCVd4XjWdQfo8oVls0ynz\\\\\\\/Gz8AqTbai4nZN0TNoU0bE+ynX0Udvuzz1UUQ7t3TO7mZrSrOfI9in8UrxA6rgHmBmUE1ZGn7KxJFiHqbEMfrXliqZgPK4Q4z2mjnp1XKgQLua4+pe2MJbhQnkmxkZJT7H2CSTlzO6Su4bMW\\\\\\\/E4Q197J6GxOjxPlolDzag10el5AemzWzdoKEuo4FWzOiNUebFrBQxLArx49g37SqB\\\\\\\/rMl1wSh3yDZiWYzBsXlcUtjsYs4aPK8ixqhUqhhNO50eIMa9OUhC+tRfetHv\\\\\\\/RIeUlHE8ibV+Mu4gx9K4w9nhsPy\\\\\\\/6WK\\\\\\\/nrItXFJefBtRWcPMRJdsyugicWB3e8deeB03M6gMvJTrKkdYrC0+7MVVA+TDSW2i+vL3GXIWYUlhbdSK0qeSziXIvvqn6i4kOTX+HlqGE9wFCTeWZxKHUwMWc4VgUrdJAIxExQ91MmvYrWbWsoJ9FvHIoU+utxlgZVMstsfbm9Fpay0KTLq8JVV6Ct56SGDEQCvXaT8HQ4fPxE95bdkS+L6BxH8LLUgw4WnnQArMKnfL\\\\\\\/bIfVONtLC3xGUh\\\\\\\/cn\\\\\\\/ZX4EWl\\\\\\\/SY2fuMSOTHpMHFSLxOs6JlfKTMUzgkEezZWZ4O9P0uSAEIjbxwn361vWBz5Ijcc8A28Tlc5bF2dweBbrDPh7iqfJbuS9pX6qgIZDvdGGrkW16yAV7fNYypJItKpbw6acje7iB1UzRO1gr6Lw7TZYFVGGH47jj\\\\\\\/9kQ++wA+Lgz9faT4\\\\\\\/9rjCflB4HzYP\\\\\\\/PPH07+n6a8euRgNTwPdNuvAkB4MzXqmhkV9ybq78k8bhs1mmEmXZulBlW2dN1fkWNy8RsRE\\\\\\\/KN595bstI\\\\\\\/bXcTihEtcZ8JbRgPklnSBrerR6zhcGwbM7ta9+LsecmK3fa9YRGwM4DYfrpvsmovC+K14Wz9XRrAnwTBLfkWfPIfQ9I4oUe8VsjG3XCTWw4rdjDf3RoPX614bBLRMPXQOEUdWpBf0mA79PaCK5U1qAjap91ZNLMsGMiC1h9NnmTLnxL19NEjmVuAz4Eu30cXrNw4PKiNtN4d7nvZFxbMIIpn3l8rCbUL8vVWtBkiyTG\\\\\\\/qi5TkMZHd0kEUGwaTYh7rPvVs67B7VgUeeMUMzKI0g+SLbR7cxFwNiuaO93TNgPh07e7jqHOk9yYe49rjaq0PXQ3Aw4tSy4tSBlC8KfRsjd+7jlgTLWK\\\\\\\/0gU2bxPiRoSn8okj3PICbtKbOcYgINKzZegNFDiMxSDDnEM6A700NMb5fG1JUlfDKgcw8F2KiFXcggh+N+UrP3cUdoycsMAPXXjaw7HzsDEo665Y\\\\\\\/fY5gOVhoC3IFaAxWp9603ylzs9aW4ZKMD4n+bfNHBxyQh9weoJnxSHj4d7Ihf3TlTQ\\\\\\\/u3SHVZ6dkXH+aN2HMBZRpjLa3kf+fzjSAJI2kpnv\\\\\\\/z98W+Al4kIlhsgovb56JO3AnD\\\\\\\/+WNGE1c9EDtid8dbD1LzZ9AgqVibNsxUMtfhXYjtdVY8i2GCzIBFLj6wWB2sxVnghN7ID0jRoE7nqJSDE8MihpDSu1HvBkuc0XxqkHcR4NgsK2GxKQaCnhhKYK6MKg7gXdmvjYwCdmKb5DuQzkJJy3zBUNzwpmF+MKhRqCBTrMpybikqVW+IFHjaG3JurbALEvbiK4YH5IIlX68lPiAXpYWk8uo4aoHDRKRomuTMR6MTlIEUhSG8L5qxvvXYc3xzfYRZdJw\\\\\\\/Qxk8zCyymvE\\\\\\\/K5WdZ3Htbfmo9Qai2fzz4W4oXm7lC7Q+BJSiWy8ympjkptar7DbDsaXi+58fyCdx5Cl+a\\\\\\\/3VE+PJtLkFifQ7\\\\\\\/92up4X0065lGxhwQT+yiXK30jOr4YXlfry3RMDKPBrCX3Wc3MdFZ9YLxLj476yPew11SVSIO0liqWhVoJs4kFehtiKa69Z8NlS\\\\\\\/UDkbdyEPhi+DH3wp9SabdHpN3soFobwLAmBNLDeA0eb7hHvaMPGktj0CQoeC7i96uhhBoslsepOtxXm5tlS5UUk2eQl4cGfyMzQAYEUBZXcG2ShM+NhpDKY2ly0Q+ElqL7PxRX37OVYaKFdl+yz+xuDei18aLV1OlmbiViKqh\\\\\\\/f73xGnrGVN6zZtyNBZ4KZ+ISsSdelvMDjtnhU6amuUWLJ+V90wXi6SpWRzmSxAGY5aNYZFLIJr27nKvrgWq5X9uqhXUL5tk8xy7kBZgloQXWv9v0MNeMm+d6VGgn31UNuhbtmGDQo8YEyOTgixnCgQ2JXrLkovyRGMf\\\\\\\/+oWP+PrOiul15q1GHRRcVFn\\\\\\\/ViYWAfrTL6LIzq\\\\\\\/3BLz\\\\\\\/PmVd9FpA5esTMv6jGyq2W48AbWAhD+c+RMowb3SJtJetaRTDmxiUExOB7OgHunet0GLAA1tafPRnVxC8k+6yc\\\\\\\/mL5sXFZE6RibTYa0lQ8jgaqTslC7gNBLcmt\\\\\\\/JV20YL2ZCzhyvEQPTQ6hQzOLpkLqIAceG\\\\\\\/XkqyBbLRrrr+fdjfrxYE2iF5g5\\\\\\\/jULplhmKDGjO7JnFklHkVS1nnLh6GdhefHvlcxGNkL+bzhjSz+sDJ73G0TkdytAVyt8ChZqYC6TaAtkgzKJ6p\\\\\\\/D+wDliJo4PsNX30qZSN8o44XQECUOK9gue2nYcVaRDyVtHtMxVRRglAwJIyElmZZ4e5unljH+aADsaw7VjoSqt228XcR+KDoeZJuY3cKMHu8zIuWIMNLqUWvY6HJTtjFa1kQw90REe6VU6e9OKof2qWCEOwU1kzgCWLEbvI\\\\\\\/odYjzceROc8yIiO0EKSyP2ZB7ebs8n6uvka7BzU5NDfZosnGE8ZCacQ69KcZaf8X81HZ66SzLft++jlc6fvspZP8vWhTe16b3hSIKJeRBHzsfW9ceLNz3euhp1tmL5kyHeJy4mK1c0YdiOCXjMBs7BuTtPmlgltb9eutCCUvj+QKoMzC1yH26CH2TdItN+M\\\\\\\/LlV86Dfk+xdHgv0Er3K2IR0fKPpO1Ty3mmPxDpv27bcC+K0Cs0YasHROJ9xpvSmv8Me9znFr4rKnzxhkCvxXXXopIjTh8xPNpOKyLbcik2lOSLQYGNFN9TUdcrk7ciEz72EdqVG1pe8+VwIfmMwuLb76iEhULBWarthaslrmjZ5MJcW5dD1HI3TVW71FvolHhEU7DnJDhLebXLpSaKwKPrrqlPmNijz4q4Z24reVNAXimxLb8Qwlt0fHZjYIsh+ZWw8ibmZI8S8qjESXijg2zKaKb+aFmrXfNRT5Od8TF0f\\\\\\\/Hszdsxdeu9pweHIohtxxqu7ncWizhqDyQ0O\\\\\\\/vqimMF76HVRAML6wC0EkDN36eP2gStcHtUUMuU2fRi2Z1vNdfELfRyzfa\\\\\\\/E4UFJsadB1uwTMNe8Jf\\\\\\\/lw4nOuSoBmK1qPFuR+L8m5BiwSB8HySEKyG+EYlDu70n0zcHkqSaLzqHE1XNyGMdBMp7E\\\\\\\/67FDNu0wAidvvjahrUcRORhR20dnRRvsjnsS4OLh4aDVtXJ4XZlAr0TGsX1E4BLR9CPPYuxerqrpUAam67Q8L2XDOaAZYXZrW3JA44D8TJhw3bzsvIxEt5pwVFkLVXdB5S\\\\\\\/pNc9cSK+XvVr31vC1MTjNW\\\\\\\/RpVGYhhepjxYs7IFftkFZWive\\\\\\\/niTLxwmVdk6C6L+hZi1y0Kg2VeDUYqp4lJ\\\\\\\/VgVya9Rtv1SLx8jlSHRrlLOnUvDY7KNd2y\\\\\\\/\\\\\\\/UNga2Ix5S8DFP0W+uMknTU\\\\\\\/vZNIxz2qZJtftTSF\\\\\\\/64nnBlZg5nEFNaA9IRJFl7QyAmeQNWKZmS2d1ypx8Pm8DwiRNXk7v9hzsEu+S+7AWujSnyb6TqqPdu3uAgHvO1\\\\\\\/P4JRF14tvXFW4judIGyUyN5yj6niHpKmIvg4uGplCAAKLLZ6sAbC4BROWQt2J8gazAvs9ILjZg00yg5so3Mvp84bkKe1kBGU9XYpr\\\\\\\/PpdxpxKYJsb4MYN0eCzxiW8CHou05YczhrLYh8H1jQ7+6KHKGncWOXMk2BHNqe2\\\\\\\/ago+TBkU6tNF8PpbFCmPhuhgcdffb91lDxVUU6F\\\\\\\/J8WVWlK+5j\\\\\\\/mNH2hq7v7UBhB\\\\\\\/tFDGvsrKyn4M4yqYLoKf0tSCvMdX0hKXCmw17u8rk7+RFh6+iz\\\\\\\/FTE1OiOilPy33XsDb9RzoRivP+y5d3IwX35kyF7kEPKoosr0J8TSAW\\\\\\\/jlGe545jn9JTKp5PSd8IPk2faGN+Rbln7A90o0qjTkW0+RGTKjzADGvwXDW2a3sMJOSWVOF3pm4Mhw6cT3xgBn+rYD+yMF1TW45yiAzQr8c2o4YtOOFEuS\\\\\\\/5mxzxW2g0+paamnKybEGL+LpHDBEsNwG\\\\\\\/mMscEzfd0u\\\\\\\/Pu1MIYUxEIUxmvo+QbIjrBtmvHATNKEIwVqkEP3UVVhNJ1o7WPfwzWhDtYyd8jSlmMoCy0aIStJ49TnDd6qTwrqEFBN5MMTtj5dM5rHE6pQSOHZCFJqLshptg\\\\\\\/l+9Dn2VmFv7wxDk7hKS+FuYGTwC7fI6qsIZ+rWhMYoZhpVbjaN4YSgIWRzVg1CqsA4IgO4EM03dIeN+MqlZZGyRznxe+FhSYv16hjUsdBy1K\\\\\\\/yWTQ\\\\\\\/IMWbMORo6xjTEmQUOF1fbfGlhSUV1a2iDW2finf0Bz\\\\\\\/vMxgqLYndyRZJcoTKL7YZx44DWTMuJt7+fysu6R3mVr5Jq4UrORNR3wl1mbLf1iH4\\\\\\\/eaIMRNjuPLvhfaCDOSc3uD4ds\\\\\\\/UyoNMNaUyOVKVzrBp8jOWw6e+D96GIZUS0KXTqucujDKrBjDEPHBVddiCMQatcEnFNBiSB5ywx2ZER2HxrBs5ZeTA+IA8mMYbAtA8Gy9vZ3KPLOB8NXvuQf6Wh4HbcykCzklPIsaBGb1qHPH9MlFYvuXdvmm4qGdabWgrbeAGMqxlSw68u79T2fJk5MG2e92Rst50NnyL+0iWfBVG5Po0TYXT\\\\\\\/OH2HEft2Tc63e0a7cTC4k9hY3+tPuFFW6HP9oLXaiNQOJzKh89hjNqfdOf1+yrlr2qK3RGyPOiN2QQNzSGcCafZkSr0GtCpswEu+ewDPKXMO9lvwY+GHVP7COGa2iieg8EBk+U\\\\\\\/McFOGpiQvrxJ4LXijvXdSlGHXYGT2TTnYicqqC09ChO\\\\\\\/iRARXxt6Un559\\\\\\\/oNhKLcfaGwrCnG2D7FvshkluxQZyJ1NOa9PRD\\\\\\\/1duXyif98VU9rG\\\\\\\/jAi2M671TEBvPwe7iXr+v+k6LK+i2G8xCnjJPCNB0PWtKiB46nwh16C4FSrNqp5ygeRdB5e9Dflm2v4juqN1J84J6IntjEHXykRNpbRrqHZH2vvAyi92JRu2Aif4IJcQ2jcCjNRrMqPGNWMTB+aDe2tqT8rk5SuSGA6x+bD5hHfF7RT1Nk1DOhy5tDRcyYrL99Yi8SeWeCp32uFeJAdvFd4SPDGoG8tP3E6cdtaXX3lMBPPiNyZDsMwzMJKMyqczkUJEf9CYXJFnb9rV4PWb5JFFtReD9uyb+gSRCYLk8SE19tYrYcOPxrmJAiRWAN1SrsR9hdyt3dIagnPJ1Yzd+sSoaF2UnI4\\\\\\\/n74XnAdBqrmnpqedjAvHfir793yYPNLl5vDb6gSJAI0eanpeU9HJicJp2nJqGx4dG5WUD3GqLxDP6vSkKzNH\\\\\\\/OSO7JS+Y9KDmYXuEgPtazRyTT8jRBWqTM7suFi\\\\\\\/\\\\\\\/l\\\\\\\/oZAR\\\\\\\/7AP5CGuwDnGUT\\\\\\\/ajoIu+6LKksQIF8OwuQgj9BPPkucxBSf+ibtJLItDW7HE7FRWaWJjZYTJcQnylm2\\\\\\\/DMfNdEu7Axn18ct99BEgIH6Rctduh5PhpK4IKOet4qLkfNBdgkRZhEd8NwrcEeIVv3bHViPMWVI0e8CIJQTXyubxyl5SJXMlYEzqdkYayt2iCUwS9Q89NP3o\\\\\\\/m1hn6j\\\\\\\/BZv7sfURqonHpbcuL9sEU9h2grj2zZQOYLWo1MNo2x7XNwn60YUj1q\\\\\\\/UvrlH6lLMCj2anD0qS+Dk88AXPxVMkKUxCWMMiSmgwjw8YDlsN\\\\\\\/1X4a\\\\\\\/687H\\\\\\\/rrteofogHAbGJrb15OJUjRwpxWI+JYB4fsQdrtWHHsMIDNNrhbjsdzAlQt8W+fGnydjPLLhvbjGz+mgGVzs8eiC2mwupy3Rxo1MmwChP70xRI\\\\\\\/XgxCRP1bySE1wBrdML3bm98Ue6wyBd1kkczN5r6dGrYtDcOtQIPosXQiYeBjuIlrWDDHeKqVtJGJbXRCYUAleyZfS7yCCSU+cHF8VuCzggA4xgAhBQj8fzIBP9qn+7JjSvKeMjimvOvT7CovgkroRPb7N2TyubL+Vx\\\\\\\/DR8P4ynVdVf+sdUng9nZdFoBhgrlZoaoLzLZNtmmW+W5H+nhJJdZPvB852LaQpOWt8a0qs1qyguPO\\\\\\\/URNNsJzTs17uM1hHMDUk3cmugyljhTFUfRqK8yt437e3qWZhxGTlhrZSgmfuvBCrMaw+tNGaS7gWAJr0\\\\\\\/+jryq8MwGVGYhWbEqNHZEtKAsM0EkfqvyZQni+coOzbZetLH3Ic0\\\\\\\/NM044h8XO4kvdP6kD87\\\\\\\/Xf0Tw1bb0scyVQ2nBjsjyM7h7rUKGuAq\\\\\\\/Qmv\\\\\\\/MjAH09gUeAsOOFfbpoDF6RFteIgkfAW7GbP6WJwGLaiZ2U8aXRfLjomAFdWCc3TJUxS7Cy+4b2DH5Oh5s3FMLTjfbD9mMHUyJzYolUHpAQocarLlO2Ccv2a6BF2WS+12Ox1FnyzOif0nBPbbtYst6F9Rbd9+lRV1OfTrDUg3ORCRswXhwZMSZpMSlKBchwL3wn2Um57B2NXia2lPnujaqyDcybGm1HwFyVJX3Em31fehgqwamLkimeRhjBZuieeR+jbiKgyV8BMnhBoHqJMEZs12aPOFyVobqW\\\\\\\/jBkTUis163ftMSJFpOiUjnJVDnYe1KMGMDQxvJcHh188GwPLuvNYYuDvTjAJyMN81\\\\\\\/hpqNKpzIQcS+iVwsU20HwV10lwONhcdPdipTMG\\\\\\\/1\\\\\\\/pjGlvxQoTv6gxIzb5A\\\\\\\/7vc0+HlS+Jkk0wBAwfPpsj+JAvPlGYRg4ovyod7N4pMMAVD4lOl4DVpj9L3tjaIvLihNREGFbhSK8oGiRbC1O9zRFf163YLJFNNKpKJG2l\\\\\\\/3xGAAIGWPawb4L+it+j5RAlBeAA20xU92izi3qzW+DcmrrfUq0xZhkJ4dPowxcqZCKE+sNq0m5c9vm812Xd1rLonAhC1tAdUrF0YkkgRw3v64dc6bKB4IWtca5SGp1a9tPXdRfK\\\\\\\/BxqoNswObtNrzb1g3Xu3dFaVnKm\\\\\\\/mpiCgvSABREZF9jCf4hw0Lj6U2k6SHuv8FqIOL97XRSTm\\\\\\\/Snj4OY\\\\\\\/0TtWSaZAq091FIUMgwmzWlG+IdaGlsdhmOSmFd5qfmUxMJa2I+3IJyldftP6Dk2UAVIbVisB7FEaOf7WAyBYhFh4ywvFes0UFZ3bBn6ITLZrR92CYEY\\\\\\\/JDFocvI5MsphNW20mKV7hvWl3Eo176x6zL0pG1Vz0wPnJl6ZTEAwxsNf0\\\u0022,\\\u0022iv\\\u0022:\\\u0022yaekR1zqKse9ijZPgWP+oA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c21e0677a9ab044b1806baeb2d284591e8440e4b2417ac298a7fedc813fb0778\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00229EyIatNQ3cAitIk6MRPljnlxK05SdISVXPWOSxsrWB5mVZt8IoKoRpmzLZ64ZBO8sjeGQzdQ6ox71I\\\\\\\/EPHpS0PXw5h4YI42qBXim751G5v+2d2eC8ML0upTiQ4R3URstJARvbd0PjOcQj+f2w0Mdz2Us9E+qXMcXxu8Hy8hQn2Io0QHXBMmloX3D9SAyvt+8M+eQkJHQcuA\\\\\\\/SUKEPV4ZpyrO+Ymv+7zq3E4a0u3zPrIHxOLpKFQFV73G+e67IiH4RRYefRQl3yqQ\\\\\\\/Z47gp1ydextOjwzbSYrxezAcHJ1o3O5SgYJT1bujwXVLubqoRql7l6Ad35PylFFdxWbAZiGar2iIRlOc4h+386EEx3v9StkvlaXXV15m4GNjifG6Or\\\\\\\/oPZsFJHDSNoRf7L8yOZF\\\\\\\/+jaFZKuptijXv\\\\\\\/Dw4BIWhN23htQi6369hIoT1PMcoszEvLevwGAWYgbIYgaSBZU3ps8M1qqpu7xsFIE16KFNtFa6khxmJWiJljCiBx281WoVC1l37TvEYG9FYuxBEDjmutPmmsO5LEiCCqNM6Ca3zDcynGCQ8wHEJSht+OUdp53J7a5qjs04THd2+lMbc\\\\\\\/v3BrWyoqmU6S\\\\\\\/sGXlgxrUzalPA4t0Ts+VN4JZ61NyYZ08\\\\\\\/+gARptol+ZotpFcZqo3D7Lf0sUml3fmvkuYr7R9ncoDraDfmQqe6VZ7G9N8Blh9KGsph3KW0\\\\\\\/FFn3VrzxS43KDHuV\\\\\\\/njLVubEwa\\\\\\\/tF8PTUxzVmIP1B\\\\\\\/x+aRKXSabu5okmgA1WbZsSepdRegkyRUPmKkmoIitL5a4lJ8Udz3uxs4\\\\\\\/RNezoPUngjLlKmd3YfQHAALVpFitvB0+0jZKEmkApeS1MRZlzELFNMVRCz+d11KUO6WE\\\\\\\/gnvcfzCwqXTH1kecJ6GMHo3x+A4mLUcKAZ7rXpiLOiuQDBw5rNT+IY0Awd4uZZyr0c9+vseZLzBEHNNeQihRuvib8smWvQb6EIsaWfikBRn1vaqYkVDjGnu+SRexH3uDdRs03g9IlFjXqxLq1UBwa9OWGAXMvKw\\\\\\\/NxYTxbRjFTunCMB6EgLmD9uxiex4KL+p0h2fkUDTTFv5\\\\\\\/LeqiFXPlHXcf9vKnv7Yuxcf+PeSWOnb2Ej7LTvYGVjNEKG71BvRLt7L3U5Z\\\\\\\/XxdoWvPyBS7lL6RYH456Pgka6mjTqlgqwQjPi3iRXTgUJg2P+nbKnzAiGrV9FTPLCyEyKIgGCYQYP9fJTCI41y4iEbeqOXyXjVp1d2iMzctSylTZX88xlZX3XjmyRJ9YZF3B3Y2MHxOXmsJO0BO0Bmoojx6YJ\\\\\\\/Hhye00WMmvvcWgR1FRTp3ODkgqfL+gCE4NH+2fNdqTrcSErIY2W\\\\\\\/jCBQb6BG9ztqV2rLkt99VVQdOZyevRuhEMYwJcX82\\\\\\\/4e+Xhmb2GQQ2SU01die6wmB4xCikjxyLg3q5VEt8UkVvFErOxlxMzybmPEBueLHrje+H\\\\\\\/l\\\\\\\/0VJOhI28WPS9OI\\\\\\\/MvXvHPy7fAmay3Z\\\\\\\/f\\\\\\\/7WF9pLVhC7e2jt6qIK3Q+09quGphHntVMCL2IBvAah0e0FU7qydDfmfVxGPpPJVUeGFOleMSJc1Y8QWYgCH0Xj9r9r+wCsUtG1cNRsuso0HbReb3mYrf1M1qelH1MaRiWu59zZ8vWyqNuwoOz3pS+zlPhk34KpWk++caYiPf2d4R1+fdHWqG\\\\\\\/hQ4EaU4uoA5FsK2OhbN0M4BrgQeLWlBU\\\\\\\/BzxIf\\\\\\\/qC7xJprJMtV6AMbBtEcfveVEAuroN4FbwXV+5Lf7itdHC6vwEwGvthH+vssSnVKEWsXKhHyQGJOygWhkyh9ZgC+jPpYAjNDGF\\\\\\\/3xlrHa7oc2GgSUnl+FKbPKmyyAsn28hQ+V+0UaXJsOJW4ZL9abPWrTGeQ6N7+a333KLkTOwKSaZH16mPptKK3hkQc4nKWyXWeC3HUOdOGD0JbcCxVmnG0KohTNKnkn7yZiOiDdksJ6KiDNxz6Ddo46rQN2zPSXY+Y9mnEUX4s1xe4UrvEzQ42R4TNtz7iJnAN2Rge8lQ6EHtLA=\\\u0022,\\\u0022iv\\\u0022:\\\u00228xB3rN+6JSmxVxT+yz6Fjw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002221620fc121ec9f9421a7f731f6885d5311d1f6a28fc5bcb8918d620920672012\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460586 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/2/617" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-TYx50JcFJULZScLGMqztRemCBPPfQnPT6iNc57Zmsj8" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/2/617" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Q-TsqLElfu1jOHNcPwHO8ub1Os17Wk0KzgJGZBjfxrs" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/2/607" title="The Bright Fluorescent Protein mNeonGreen Facilitates Protein Expression Analysis In Vivo" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/2/625" title="upSET, the Drosophila homologue of SET3, Is Required for Viability and the Proper Balance of Active and Repressive Chromatin Marks" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460586" id="node460586" data-pisa="ggg;7/2/617" data-pisa-master="ggg;g3.116.035915" data-apath="/ggg/7/2/617.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Predictability of Genetic Interactions from Functional Gene Modules</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-3365-2702" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jonathan H. Young</span> and <span class="highwire-citation-author" data-delta="1">Edward M. Marcotte</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">617-624;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.035915 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460586" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jonathan H. Young </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Computational Engineering and Sciences, The University of Texas at Austin, Texas 78712</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Systems and Synthetic Biology, The University of Texas at Austin, Texas 78712</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BH.%2BYoung%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Young%20JH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJonathan%2BH.%2BYoung%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3365-2702" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jonathan H. Young</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Edward M. Marcotte </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Systems and Synthetic Biology, The University of Texas at Austin, Texas 78712</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Molecular Biosciences, The University of Texas at Austin, Texas 78712</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Edward%2BM.%2BMarcotte%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Marcotte%20EM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEdward%2BM.%2BMarcotte%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:marcotte@icmb.utexas.edu" class="" data-icon-position="" data-hide-link-title="0">marcotte@icmb.utexas.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/2/617" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460586" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460586/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/2/617.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460586" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460586/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/2/617.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460586" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460586/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/2/617.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460586" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460586/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/2/617.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460586" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460586/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Characterizing genetic interactions is crucial to understanding cellular and organismal response to gene-level perturbations. Such knowledge can inform the selection of candidate disease therapy targets, yet experimentally determining whether genes interact is technically nontrivial and time-consuming. High-fidelity prediction of different classes of genetic interactions in multiple organisms would substantially alleviate this experimental burden. Under the hypothesis that functionally related genes tend to share common genetic interaction partners, we evaluate a computational approach to predict genetic interactions in <em>Homo sapiens</em>, <em>Drosophila melanogaster</em>, and <em>Saccharomyces cerevisiae</em>. By leveraging knowledge of functional relationships between genes, we cross-validate predictions on known genetic interactions and observe high predictive power of multiple classes of genetic interactions in all three organisms. Additionally, our method suggests high-confidence candidate interaction pairs that can be directly experimentally tested. A web application is provided for users to query genes for predicted novel genetic interaction partners. Finally, by subsampling the known yeast genetic interaction network, we found that novel genetic interactions are predictable even when knowledge of currently known interactions is minimal.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/epistasis" class="hw-term hw-article-keyword hw-article-keyword-epistasis" rel="nofollow">epistasis</a></li><li class="kwd"><a href="/keyword/gene-network" class="hw-term hw-article-keyword hw-article-keyword-gene-network" rel="nofollow">gene network</a></li><li class="kwd"><a href="/keyword/synthetic-lethality" class="hw-term hw-article-keyword hw-article-keyword-synthetic-lethality" rel="nofollow">synthetic lethality</a></li><li class="kwd"><a href="/keyword/data-mining" class="hw-term hw-article-keyword hw-article-keyword-data-mining" rel="nofollow">data mining</a></li><li class="kwd"><a href="/keyword/drug-target" class="hw-term hw-article-keyword hw-article-keyword-drug-target" rel="nofollow">drug target</a></li></ul><p id="p-3">Determining the genetic interactions in an organism provides a basis for understanding how the role of a gene is influenced by the action of any other gene. By definition, two or more genes interact when combining variants of each gene produces a significantly pronounced phenotype when compared to the phenotypes of individual variants (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Mani <em>et al.</em> 2008</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Baryshnikova <em>et al.</em> 2013</a>). The applications of exploiting such interactions extend to drug target discovery. Strategies such as targeting genes that interact with cancer-specific mutations have been proposed and reviewed extensively (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Ashworth <em>et al.</em> 2011</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Fece de la Cruz <em>et al.</em> 2015</a>) and have led to clinical trials (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Fong <em>et al.</em> 2009</a>). Because experimental determination of genetic interactions involves examining all possible pairs from a group of genes, practical difficulties arise when a comprehensive interaction map of an entire organism is desired. Multicellular organisms present the challenge of various differentiated cell types, each having potentially differing genetic interactions. Moreover, there are different kinds of genetic interactions, ranging from those based on growth effects to other phenotypic effects. There exists a need to either reduce the search space for testing genetic interactions or to reliably predict them. Here, we evaluate a computational approach to predict and validate different types genetic interactions across multiple organisms.</p><p id="p-4">Previous studies to predict genetic interactions leveraged existing sources of biological information. Integration of biological features in yeast (<em>i.e.</em>, gene coexpression, protein interaction and function) and their associated network topological properties guided the training of probabilistic decision trees to predict synthetic sick or lethal (SSL) interactions (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Wong <em>et al.</em> 2004</a>). In a similar vein, an ensemble classifier was trained on a set of 152 genetic interaction-independent features to predict SSL in yeast (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Pandey <em>et al.</em> 2010</a>). Compiling multiple biological features has also been extended to more than one organism. By considering the orthologous gene pairs among yeast, fly, and worm, features such as functional annotation were used to train a logistic regression model to predict a genome-wide map of genetic interactions (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Zhong and Sternberg 2006</a>). Alternatively, studies have also explored network-based approaches for genetic interaction prediction. Novel SSL interactions were predicted by way of a diffusion kernel on a network of known SSL gene pairs (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Qi <em>et al.</em> 2008</a>). Interrogating functional gene networks that were constructed from integration of biological data from literature have proven useful in predicting modifier genes in yeast and worm (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Lee <em>et al.</em> 2010</a>). Many of these approaches have focused on a single genetic interaction type in a single organism.</p><p id="p-5">Here, we examine an algorithm to predict multiple types of genetic interactions across diverse organisms based on the hypothesis that genes strongly participating in shared functions also share common genetic interaction partners. Our approach relies on a functional gene network for a given organism and knowledge of known genetic interactions of a particular type. We tested our approach on three organisms, human (<em>Homo sapiens</em>), fly (<em>Drosophila melanogaster</em>), and yeast (<em>Saccharomyces cerevisiae</em>), and found predictability across different types of genetic interactions. We also investigated how some interactions are enriched in yeast and human gene modules, specifically protein complexes, and the degree to which genetic interactions need to be experimentally determined before enrichment can be found.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><p id="p-6">For various classes of genetic interactions in human, fly, and yeast, a list of genes and each of their known genetic interaction partners were assembled. A gene and its known interaction partners are collectively referred to as a “seed set.” Seed sets with only a single interacting gene pair were filtered out. Receiver operating characteristic (ROC) analysis was performed to quantify whether the interaction partners of any given gene are clustered in the organism’s functional gene network. Specifically, for every group of interaction partners of a gene, a score vector consists of entries that are sums of functional network edge weights between each gene in the network to the interaction partners. Because there are no self-edges in the network, leave-one-out cross-validation is carried out on the known interaction partners. An accompanying label vector indicates whether each gene in the network is indeed an interaction partner. The two vectors yield an ROC curve and the corresponding area under the curve (AUC). A seed set’s AUC is the measure of how tightly connected the interaction partners are in the functional network, and therefore how predictive the seed set is for novel interactions (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Lee <em>et al.</em> 2010</a>). None of the known genetic interactions used for prediction were contained in the functional gene network. There were no fly or human genetic interactions incorporated into their respective functional gene networks. The yeast functional network did not include any genetic interactions discovered after 2006; every interacting gene pair before that year in BioGRID was excluded.</p><p id="p-7">Enrichment of genetic interactions within yeast and human protein complexes was calculated with a binomial model defined as <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/617/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/617/embed/mml-math-1.gif"/></noscript></span></span> where the background probability, <em>p</em>, equals the proportion of all possible gene pairs that are genetically interacting. The number of trials, <em>n</em>, is the number of possible gene pairs in the complex, and <em>k</em> equals the number of interacting pairs in the protein complex.</p><div id="sec-2" class="subsection"><h3>Statistical analysis</h3><p id="p-8">If <em>k</em> is the number of genetic interactions within a protein complex, then the corresponding <em>p</em>-value is <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/617/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/617/embed/mml-math-2.gif"/></noscript></span></span>, according to a binomial model previously described, with control of FDR at 5% using the Benjamini–Hochberg procedure (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Benjamini and Hochberg 1995</a>). Seed sets with AUC ≥ 0.9 were considered highly predictive of novel genetic interactions.</p></div><div id="sec-3" class="subsection"><h3>Data availability</h3><p id="p-9">All genetic interactions were downloaded from version 3.4.130 of BioGRID (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Stark <em>et al.</em> 2006</a>). Organism-specific functional gene networks were downloaded for human (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Lee <em>et al.</em> 2011</a>), fly (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Shin <em>et al.</em> 2015</a>), and yeast (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Lee <em>et al.</em> 2007</a>). Previous studies served as sources of protein complexes for yeast and human (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Hart <em>et al.</em> 2007</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Ruepp <em>et al.</em> 2010</a>). Python code using the Matplotlib (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Hunter <em>et al.</em> 2007</a>), scikit-learn (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Pedregosa <em>et al.</em> 2011</a>), and <em>mygene</em> (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Wu <em>et al.</em> 2012</a>) libraries is available at <a href="https://bitbucket.org/youngjh/genetic_interact">https://bitbucket.org/youngjh/genetic_interact</a>. All network visualizations were produced in Cytoscape (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Shannon <em>et al.</em> 2003</a>). A supplementary web page at <a href="http://marcottelab.org/Genetic_Interact">http://marcottelab.org/Genetic_Interact</a> allows users to query a gene of interest. If the gene has known genetic interaction partners that are predictive, then the functional network cluster is displayed. Raw data files listing the seed sets with AUC ≥ 0.7 are also available.</p></div></div><div class="section results" id="sec-4"><h2 class="">Results</h2><p id="p-10">We sought to determine whether clusters of functionally related genes, for example, genes <em>A–E</em> in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>, are predictive of genetic interactions. In this example, genes <em>A</em> and <em>C–E</em> are known to share genetic interactions with gene <em>X</em>, and our hypothesis would suggest gene <em>B</em> as a novel interaction partner of <em>X</em>. Our method identifies predictive clusters by leave-one-out cross-validation and ROC analysis; when applied to the network in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>, each of genes <em>A</em> and <em>C–E</em> are individually withheld as known interaction partners one at a time, and predicted back with high recall. Subsequently, gene <em>B</em> is a novel high-confidence predicted interaction partner of <em>X</em>. The approach described here was evaluated for several classes of phenotypic and growth-based genetic interactions in human, fly, and yeast.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/617/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genetic interaction prediction. Dashed edges indicate known genetic interactions. Solid edges connect genes that participate in the same biological process, with log likelihood scores (LLS) as edge weights reflecting the degree of confidence in the genes&#x2019; shared functionality. Genes A, C&#x2013;E are genetic interaction partners of gene X and members of a functional net cluster; then the remaining cluster member, gene B, is a predicted interaction partner of gene X as well. Candidate clusters are evaluated by first assigning scores to each gene in the network by summing the edge weights, as shown in the first row of the matrix.  denotes the LLS between genes g and A. The second row is populated with binary labels indicating whether the gene is a known interaction partner of X. In this fashion, an ROC curve is constructed to yield an AUC." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2027503916" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Genetic interaction prediction. Dashed edges indicate known genetic interactions. Solid edges connect genes that participate in the same biological process, with log likelihood scores (LLS) as edge weights reflecting the degree of confidence in the genes&#x2019; shared functionality. Genes &lt;em&gt;A&lt;/em&gt;, &lt;em&gt;C&#x2013;E&lt;/em&gt; are genetic interaction partners of gene &lt;em&gt;X&lt;/em&gt; and members of a functional net cluster; then the remaining cluster member, gene &lt;em&gt;B&lt;/em&gt;, is a predicted interaction partner of gene &lt;em&gt;X&lt;/em&gt; as well. Candidate clusters are evaluated by first assigning scores to each gene in the network by summing the edge weights, as shown in the first row of the matrix. &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-3&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; denotes the LLS between genes &lt;em&gt;g&lt;/em&gt; and &lt;em&gt;A&lt;/em&gt;. The second row is populated with binary labels indicating whether the gene is a known interaction partner of &lt;em&gt;X&lt;/em&gt;. In this fashion, an ROC curve is constructed to yield an AUC.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/617/F1.medium.gif" width="223" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/2/617/F1.medium.gif" width="223" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/617/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/617/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470509" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-11" class="first-child">Genetic interaction prediction. Dashed edges indicate known genetic interactions. Solid edges connect genes that participate in the same biological process, with log likelihood scores (LLS) as edge weights reflecting the degree of confidence in the genes’ shared functionality. Genes <em>A</em>, <em>C–E</em> are genetic interaction partners of gene <em>X</em> and members of a functional net cluster; then the remaining cluster member, gene <em>B</em>, is a predicted interaction partner of gene <em>X</em> as well. Candidate clusters are evaluated by first assigning scores to each gene in the network by summing the edge weights, as shown in the first row of the matrix. <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/617/F1/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/617/F1/embed/mml-math-3.gif"/></noscript></span></span> denotes the LLS between genes <em>g</em> and <em>A</em>. The second row is populated with binary labels indicating whether the gene is a known interaction partner of <em>X</em>. In this fashion, an ROC curve is constructed to yield an AUC.</p><div class="sb-div caption-clear"></div></div></div><div id="sec-5" class="subsection"><h3>The human functional gene network is predictive for phenotypic genetic interactions</h3><p id="p-12">As shown in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>, our method demonstrated high performance in predicting phenotypic enhancing and suppressing human gene pairs. In these interactions, a double mutant has an enhanced or suppressed phenotype (other than growth) in comparison to either of the single mutants. The plots for phenotypic enhancement and suppression in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a> display the performance of seed sets, each of which are defined as a group of known phenotypic enhancing or suppressing partners of a particular gene. There are 57 phenotypic enhancement seed sets, of which 30 have an AUC ≥ 0.9. Similarly, 36 of 66 phenotypic suppression seed sets have an AUC ≥ 0.9. The AUC is the area under the ROC curve that measures how well the known interaction partners rank in our leave-one-out cross-validation scheme. Those that are not predictive are the ones with an AUC ≈ 0.5, indicating that their predictability is no better than random. For the most part, seed sets are either at least moderately predictive, or not at all. The sharp drop in AUC to 0.5 in <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2A</a> does not appear to correspond with genes that simply have no edges above the background in the functional network. It does not appear that all random AUCs are the result of a small set size, as seed set sizes over two also produce random AUCs. As shown in Supplemental Material, <a href="http://www.g3journal.org/content/suppl/2016/12/21/g3.116.035915.DC1/Figure_S1.pdf">Figure S1</a>, low incident edge weight does not correlate with low AUC.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/617/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Predictive functional net clusters yield novel phenotypic enhancing and suppressing human gene pairs. (A) Each horizontal bar represents the set of known genetic interaction partners of a specific human gene; each of these sets is referred to as a &#x201C;seed set.&#x201D; High AUC scores indicate that the interaction partners participate together in a cluster in HumanNet, the human functional gene network. Therefore, other members of the cluster are predicted as novel interaction partners. (B) Shown are two examples of well-defined HumanNet clusters that are highly predictive for phenotypic enhancement (left) and suppression (right), with the known interactions from the seed set denoted by the boxed genes and dashed edges." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2027503916" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Predictive functional net clusters yield novel phenotypic enhancing and suppressing human gene pairs. (A) Each horizontal bar represents the set of known genetic interaction partners of a specific human gene; each of these sets is referred to as a &#x201C;seed set.&#x201D; High AUC scores indicate that the interaction partners participate together in a cluster in HumanNet, the human functional gene network. Therefore, other members of the cluster are predicted as novel interaction partners. (B) Shown are two examples of well-defined HumanNet clusters that are highly predictive for phenotypic enhancement (left) and suppression (right), with the known interactions from the seed set denoted by the boxed genes and dashed edges.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/617/F2.medium.gif" width="440" height="427"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/2/617/F2.medium.gif" width="440" height="427"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/617/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/617/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470274" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-13" class="first-child">Predictive functional net clusters yield novel phenotypic enhancing and suppressing human gene pairs. (A) Each horizontal bar represents the set of known genetic interaction partners of a specific human gene; each of these sets is referred to as a “seed set.” High AUC scores indicate that the interaction partners participate together in a cluster in HumanNet, the human functional gene network. Therefore, other members of the cluster are predicted as novel interaction partners. (B) Shown are two examples of well-defined HumanNet clusters that are highly predictive for phenotypic enhancement (left) and suppression (right), with the known interactions from the seed set denoted by the boxed genes and dashed edges.</p><div class="sb-div caption-clear"></div></div></div><p id="p-14">Shown in <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2B</a> are illustrative seed sets with high predictability that form well-defined clusters in the human functional gene network, HumanNet. For clarity, only functional network edges with log likelihood scores (LLS) above 3.0 are shown. Furthermore, HumanNet genes are shown only if they connect to at least two of the known genetic interaction partners. The seed set consisting of the SNW domain containing one gene (<em>SNW1</em>) that phenotypically enhances members of the SMAD family and nuclear receptor coactivators yielded an AUC of 0.91. The prediction is that <em>SNW1</em> also phenotypically enhances with other members of the SMAD family along with members of the forkhead box. In the phenotypic suppression case, we find that known phenotypic suppressors of caspase 2 are tightly functionally linked with members of the BCL2-like family, among other genes. With a resulting AUC of 0.90, these BCL2-like genes are expected to participate in phenotypic suppression with caspase 2.</p></div><div id="sec-6" class="subsection"><h3>Fly phenotypic enhancement and suppression interactions are predicted from functional net clusters</h3><p id="p-15">Similar to the human case, the fly functional network FlyNet is particularly predictive of phenotypic enhancement and suppression, as shown in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>. A larger proportion of the seed sets are predictive than in the human case. For phenotypic enhancement, 322 out of 449 seed sets had AUC ≥ 0.9 and 398 phenotypic suppression seed sets (out of 518) met the same threshold. <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a> shows a well-defined gene cluster (AUC = 0.94) containing phenotypic enhancement interaction partners of <em>seven up</em>. From this cluster, genes involved in the <em>sevenless</em> signaling and the <em>Drosophila</em> <em>epidermal growth factor receptor</em> signal transduction pathways achieved high recall, and neighbor genes also involved in the same signaling pathways are expected to phenotypically enhance <em>seven up</em>. Turning to phenotypic suppression, several <em>Enhancer of split</em> genes are tightly clustered (AUC = 0.98) with known phenotypic suppressors of <em>hairy</em> that include the <em>achaete-scute</em> complex, thereby implicating them as additional, novel phenotypic suppressors of <em>hairy</em>.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/617/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="FlyNet predictability for phenotypic enhancing and suppressing genetic interactions. (A) Each horizontal bar represents a single fly gene that is known to interact with a number of other genes. (B) Predictive seed gene sets are shown for phenotypic enhancement (left) and suppression (right)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2027503916" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;FlyNet predictability for phenotypic enhancing and suppressing genetic interactions. (A) Each horizontal bar represents a single fly gene that is known to interact with a number of other genes. (B) Predictive seed gene sets are shown for phenotypic enhancement (left) and suppression (right).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/617/F3.medium.gif" width="440" height="421"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/2/617/F3.medium.gif" width="440" height="421"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/617/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/617/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470233" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-16" class="first-child">FlyNet predictability for phenotypic enhancing and suppressing genetic interactions. (A) Each horizontal bar represents a single fly gene that is known to interact with a number of other genes. (B) Predictive seed gene sets are shown for phenotypic enhancement (left) and suppression (right).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-7" class="subsection"><h3>High-confidence predictability is found in human, fly, and yeast</h3><p id="p-17">The full range of various genetic interaction classes that were analyzed from BioGRID are listed in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. Genetic interactions were generally based on phenotypic effects or growth and lethality measurements. Each entry in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> lists the number of predictive seed sets having AUC ≥ 0.9 of out the total examined. For human, our method performed well primarily for phenotypic enhancement and suppression, as described above, but did not offer predictability for the dosage lethality and synthetic growth defect and rescue interactions determined to date. For fly, most of the known interactions fall into the phenotypic enhancement and suppression categories, for which high predictability was observed. In both human and fly, several classes of interactions have not been extensively determined and thus were untested in our prediction scheme.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470319/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470319/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Predictive power of functional networks across different genetic interactions</span><div class="sb-div caption-clear"></div></div></div><p id="p-19">Our method also performed well in most of the interaction categories for <em>S. cerevisiae</em> (<a href="http://www.g3journal.org/content/suppl/2016/12/21/g3.116.035915.DC1/Figure_S2.pdf">Figure S2</a> and <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). Notably, negative and positive genetic interactions fared poorly as few predictive seed sets were identified, even though most of the experimentally determined interactions in yeast fall into these categories.</p></div><div id="sec-8" class="subsection"><h3>Predictability observed in unbiased screens of eukaryotes</h3><p id="p-20">The negative and positive genetic interaction screens for yeast are nearly exhaustive and therefore largely unbiased. We investigated the performance on the yeast data using different AUC cutoffs and found that predictability increased as the AUC threshold was relaxed (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). This suggests that our method predicts positive and negative interactions even in yeast from unbiased data, albeit to more modest levels.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/617/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Performance in unbiased yeast screens as a function of AUC cutoff. For both negative and positive genetic interactions in S. cerevisiae, the proportion of seed gene sets exceeding a set AUC threshold ranging from 0.1 to 0.9 is shown. Nonrandom performance is observed and increases as the AUC threshold is relaxed." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2027503916" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Performance in unbiased yeast screens as a function of AUC cutoff. For both negative and positive genetic interactions in &lt;em&gt;S. cerevisiae&lt;/em&gt;, the proportion of seed gene sets exceeding a set AUC threshold ranging from 0.1 to 0.9 is shown. Nonrandom performance is observed and increases as the AUC threshold is relaxed.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/617/F4.medium.gif" width="324" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/2/617/F4.medium.gif" width="324" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/617/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/617/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470359" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-21" class="first-child">Performance in unbiased yeast screens as a function of AUC cutoff. For both negative and positive genetic interactions in <em>S. cerevisiae</em>, the proportion of seed gene sets exceeding a set AUC threshold ranging from 0.1 to 0.9 is shown. Nonrandom performance is observed and increases as the AUC threshold is relaxed.</p><div class="sb-div caption-clear"></div></div></div><p id="p-22">Given the sparsity of currently known genetic interactions in higher eukaryotes, we analyzed a genome-scale CRISPR screen against four human cancer cell lines that uncovered essential genes specific to each cell line (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Wang <em>et al.</em> 2014</a>). The screen was not previously included among the BioGRID datasets in our study and, unlike other data on higher eukaryotes, is unbiased in the sense that there was no selection of candidate genetic interaction partners. Rather, the characteristic mutations of each cell line determined the specific hits. As shown in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>, our method found moderate predictability from three of the four cell lines, especially for KBM7, with an AUC of 0.74.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/617/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Predictability observed in unbiased screens of higher eukaryotes. A CRISPR-Cas9 screen against four human cancer cell lines uncovered cell line-specific essential genes (Wang et al. 2014). (A) The specific hits for each cell line were used as the seed sets, and we found moderate predictability in three of the four cell lines, especially for KBM7. (B) Shown is the predictive functional net cluster induced by KBM7-specific hits, which are the boxed genes. Solid edges indicate connections in the human functional network, HumanNet. For clarity, only edges above a LLS weight cutoff of 3.0 and genes with more than one interaction to the hits are shown." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2027503916" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Predictability observed in unbiased screens of higher eukaryotes. A CRISPR-Cas9 screen against four human cancer cell lines uncovered cell line-specific essential genes (Wang &lt;em&gt;et al.&lt;/em&gt; 2014). (A) The specific hits for each cell line were used as the seed sets, and we found moderate predictability in three of the four cell lines, especially for KBM7. (B) Shown is the predictive functional net cluster induced by KBM7-specific hits, which are the boxed genes. Solid edges indicate connections in the human functional network, HumanNet. For clarity, only edges above a LLS weight cutoff of 3.0 and genes with more than one interaction to the hits are shown.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/617/F5.medium.gif" width="375" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/2/617/F5.medium.gif" width="375" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/617/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/617/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470225" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-23" class="first-child">Predictability observed in unbiased screens of higher eukaryotes. A CRISPR-Cas9 screen against four human cancer cell lines uncovered cell line-specific essential genes (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Wang <em>et al.</em> 2014</a>). (A) The specific hits for each cell line were used as the seed sets, and we found moderate predictability in three of the four cell lines, especially for KBM7. (B) Shown is the predictive functional net cluster induced by KBM7-specific hits, which are the boxed genes. Solid edges indicate connections in the human functional network, HumanNet. For clarity, only edges above a LLS weight cutoff of 3.0 and genes with more than one interaction to the hits are shown.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h3>Protein complexes inform trends of genetic interaction predictability</h3><p id="p-24">With genetic interactions predicted across multiple organisms, it was natural to investigate their evolutionary conservation. In particular, if a protein complex were enriched in genetic interactions then perhaps a homologous protein complex would also exhibit similar enrichment. We found enrichment of various types of interactions within yeast protein complexes, but none thus far for human. Therefore, instead the problem shifted to identifying the degree to which genetic interactions must be determined in order to find enrichment, and therefore predictability. Using yeast as a test case, simulations successively withheld increasing proportions of genetic interactions, with enrichment within yeast protein complexes computed at each point. The interaction types considered were negative and positive genetic, and synthetic growth defect and lethality. We note that according to BioGRID, negative and positive genetic interactions are reserved for high-throughput assays with growth scores. As shown in <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>, when withholding genes with a genetic interaction degree (the number of interacting partners of a certain gene) of &gt;5, corresponding to withholding &gt;90% of synthetic growth defect and &gt;80% of synthetic lethality pairs, then an immediate drop-off in enrichment resulted. No such behavior was observed for negative and positive genetic interactions, for which enrichment linearly decreased as a function of the withheld proportion. Similarly, when removing interacting pairs at random, there was a steady decrease in the number of significantly enriched complexes among all types. Finally, when withholding pairs under a degree cutoff, there was also no point beyond which enrichment failed to be found (<a href="http://www.g3journal.org/content/suppl/2016/12/21/g3.116.035915.DC1/Figure_S3.pdf">Figure S3</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/617/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Predictability of genetic interactions can be found even when known interactions are sparse. By successively withholding known yeast genetic interactions according to each gene&#x2019;s interaction degree (e.g., number of interaction partners), enrichment and therefore predictability is still detectable when information of known interactions is minimal. This effect is especially pronounced for synthetic growth defect and lethality, provided genes possess sufficiently high interaction degree." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2027503916" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Predictability of genetic interactions can be found even when known interactions are sparse. By successively withholding known yeast genetic interactions according to each gene&#x2019;s interaction degree (&lt;em&gt;e.g.&lt;/em&gt;, number of interaction partners), enrichment and therefore predictability is still detectable when information of known interactions is minimal. This effect is especially pronounced for synthetic growth defect and lethality, provided genes possess sufficiently high interaction degree.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/617/F6.medium.gif" width="440" height="404"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/7/2/617/F6.medium.gif" width="440" height="404"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/617/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/617/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470495" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-25" class="first-child">Predictability of genetic interactions can be found even when known interactions are sparse. By successively withholding known yeast genetic interactions according to each gene’s interaction degree (<em>e.g.</em>, number of interaction partners), enrichment and therefore predictability is still detectable when information of known interactions is minimal. This effect is especially pronounced for synthetic growth defect and lethality, provided genes possess sufficiently high interaction degree.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-10"><h2 class="">Discussion</h2><p id="p-26">Our results demonstrate that various classes of genetic interactions in different organisms can be successfully predicted based on the hypothesis that functional gene clusters tend to share genetic interaction partners. For <em>S. cerevisiae</em> in particular, predictability was obtained whether the genetic interaction type was based on growth effects or nongrowth phenotype-based measurements (<em>i.e.</em>, phenotypic suppression). Interestingly, our method did not yield predictability for negative and positive genetic interactions, which happen to be the interaction types for which most of the pairs have been tested (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Costanzo <em>et al.</em> 2010</a>). As BioGRID reserves negative and positive genetic interactions for high-throughput assays with growth scores, a threshold is applied to determine the presence of an interaction. The choice of this threshold may affect the predictability of the algorithm we have evaluated here.</p><p id="p-27">In many of the genetic interaction datasets analyzed in our study, a subset of the organism’s genome was selected for screening as candidate genetic interaction partners of a chosen gene. In contrast, the yeast negative and positive genetic interactions and the human CRISPR dataset we analyzed are unbiased in that nearly the entire genome was screened, and therefore no decision was taken to include or exclude genes as candidate genetic interaction partners. Using the unbiased data, our method found predictability of genetic interactions, albeit to a lesser degree than in other datasets. We note that this result does not confirm or rule out potential bias in analyzing data from other organisms, given that the genetic interactions are sparser and hence have the potential for ascertainment bias elevating the prediction scores. Moreover, our method would likely benefit from increased coverage of the functional gene networks.</p><p id="p-28">While the range of predictable genetic interaction classes for human and fly were limited to phenotypic enhancement and suppression, we believe that this is probably due to the sparsity of known genetic interactions for these organisms. In this study, the source of known genetic interactions, BioGRID, had over 150,000 yeast gene pairs but only ∼2800 pairs for fly and ∼1500 for human. As shown in <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>, many types of genetic interactions could simply not be tested for fly and human.</p><p id="p-29">This sparseness of experimentally determined genetic interactions, especially in human, led to the lack of enrichment in gene modules such as protein complexes. In our simulations of withholding genetic interacting pairs, we expected that regardless of the interaction type, there would be a point after which no enrichment would be found. Thus, it was surprising that negative and positive genetic interactions exhibited a linear decrease in enrichment, regardless of how the pairs were withheld (by degree or at random). On the other hand, the enrichment signal in synthetic growth defect and lethality is sensitive to the interaction degree, as there was a steep drop-off when most of the interaction pairs were withheld. In the negative and positive genetic networks, there appears to sufficient genetic interaction density such that even when high numbers of interacting pairs are withheld, enrichment under a binomial model can still be found. By extrapolating to the human case, a modest increase in the number of screened human gene pairs is likely to dramatically increase the ability to predict additional genetic interactions, especially for synthetic growth defect and lethality, where the genes have multiple interaction partners.</p><p id="p-30">While we did not investigate which classes or properties of functional interactions are predictive of genetic interactions, it would be interesting to further explore such effects. The functional gene network corresponding to a certain organism is an integration of multiple networks from various biological evidence, such as coexpression or protein–protein interactions. It may be informative to hold out certain networks for given evidence classes to determine the resulting effect on the predictability of genetic interactions.</p><p id="p-31">Similar to previous genetic interaction prediction approaches (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Qi <em>et al.</em> 2008</a>; <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Zhong and Sternberg 2006</a>), our algorithm requires knowledge of known experimentally determined genetic interactions. While other studies proceed without such requirements, the assimilation of a host of biologically annotated features are still necessary for their prediction method (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Pandey <em>et al.</em> 2010</a>; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Wong <em>et al.</em> 2004</a>). In contrast to the aforementioned studies, our methodology systematically examined more than one class of genetic interaction and was successfully applied to multiple eukaryotic organisms, thereby generalizing results from a previous study by <a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Lee <em>et al.</em> (2010)</a>. We did not explicitly carry out a comparative evaluation of our method against other approaches, however, and cannot quantitatively comment on the relative performance of different methods. Since the detection of tightly connected sets of nodes in a network is central to our method, further avenues for exploration perhaps include investigating methods, such as graph clustering (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Enright <em>et al.</em> 2002</a>) or community detection algorithms (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Fortunato 2010</a>), though these algorithms lack built-in validation. It would also be interesting to explore using tissue-specific gene networks instead of a single integrated functional gene network for more targeted predictions (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Greene <em>et al.</em> 2015</a>).</p><p id="p-32">As one major goal of any genetic interaction prediction is to at least narrow down the search space for experimentally testing genetically interacting pairs, our predictions are specifically testable experimentally, perhaps through CRISPR-Cas9 for human cells (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Wong <em>et al.</em> 2016</a>). We also contribute to available prediction methodologies for suggesting genetic interactions as candidate therapeutic targets. Ultimately, we demonstrate the power of leveraging knowledge of known genetic interactions and integrated biological information in functional gene networks to predict novel genetic interactions from single-cell to multicellular organisms.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-33">The authors thank Kevin Drew for assistance with web server setup. E.M.M. acknowledges funding from the National Institutes of Health, the National Science Foundation, the Cancer Prevention and Research Institute of Texas, and the Welch Foundation (F1515). </p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-34">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035915/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035915/-/DC1</a>.</p></li><li class="fn" id="fn-3"><p id="p-35"><em>Communicating editor: M. Walhout</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-04-25"><span class="received-label">Received </span>April 25, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-12-14"><span class="accepted-label">Accepted </span>December 14, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Young and Marcotte</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.617.1" data-doi="10.1016/j.cell.2011.03.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ashworth</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lord</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reis-Filho</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genetic interactions in cancer progression and treatment.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">145</span>: <span class="cit-fpage">30</span>–<span class="cit-lpage">38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DAshworth%26rft.auinit1%253DA.%26rft.volume%253D145%26rft.issue%253D1%26rft.spage%253D30%26rft.epage%253D38%26rft.atitle%253DGenetic%2Binteractions%2Bin%2Bcancer%2Bprogression%2Band%2Btreatment.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2011.03.020%26rft_id%253Dinfo%253Apmid%252F21458666%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2011.03.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21458666&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289053900007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.617.2" data-doi="10.1146/annurev-genom-082509-141730"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baryshnikova</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costanzo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andrews</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boone</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genetic interaction networks: toward an understanding of heritability.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genomics Hum. Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">111</span>–<span class="cit-lpage">133</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BGenomics%2BHum.%2BGenet.%26rft.volume%253D14%26rft.spage%253D111%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-genom-082509-141730%26rft_id%253Dinfo%253Apmid%252F23808365%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-genom-082509-141730&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23808365&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000326658500006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.617.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Benjamini</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hochberg</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Controlling the false discovery rate: a practical and powerful approach to multiple testing.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. B.</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BB.%26rft.volume%253D57%26rft.spage%253D289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.617.4" data-doi="10.1126/science.1180823"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Costanzo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baryshnikova</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bellay</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spear</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The genetic landscape of a cell.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">327</span>: <span class="cit-fpage">425</span>–<span class="cit-lpage">431</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DCostanzo%26rft.auinit1%253DM.%26rft.volume%253D327%26rft.issue%253D5964%26rft.spage%253D425%26rft.epage%253D431%26rft.atitle%253DThe%2BGenetic%2BLandscape%2Bof%2Ba%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1180823%26rft_id%253Dinfo%253Apmid%252F20093466%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjcvNTk2NC80MjUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzIvNjE3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.617.5" data-doi="10.1093/nar/30.7.1575"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Enright</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Dongen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ouzounis</span>  <span class="cit-name-given-names">C. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">An efficient algorithm for large-scale detection of protein families.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1575</span>–<span class="cit-lpage">1584</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DEnright%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D30%26rft.issue%253D7%26rft.spage%253D1575%26rft.epage%253D1584%26rft.atitle%253DAn%2Befficient%2Balgorithm%2Bfor%2Blarge-scale%2Bdetection%2Bof%2Bprotein%2Bfamilies%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F30.7.1575%26rft_id%253Dinfo%253Apmid%252F11917018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjMwLzcvMTU3NSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi82MTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.617.6" data-doi="10.1146/annurev-pharmtox-010814-124511"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fece de la Cruz</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gapp</span>  <span class="cit-name-given-names">B. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nijman</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Synthetic lethal vulnerabilities of cancer.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Pharmacol. Toxicol.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">513</span>–<span class="cit-lpage">531</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BPharmacol.%2BToxicol.%26rft.volume%253D55%26rft.spage%253D513%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-pharmtox-010814-124511%26rft_id%253Dinfo%253Apmid%252F25340932%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-pharmtox-010814-124511&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25340932&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.617.7" data-doi="10.1056/NEJMoa0900212"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fong</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boss</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yap</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tutt</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Inhibition of poly (ADP-ribose) polymerase in tumors from BRCA mutation carriers.</span> <abbr class="cit-jnl-abbrev">N. Engl. J. Med.</abbr> <span class="cit-vol">361</span>: <span class="cit-fpage">123</span>–<span class="cit-lpage">134</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN.%2BEngl.%2BJ.%2BMed.%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa0900212%26rft_id%253Dinfo%253Apmid%252F19553641%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa0900212&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19553641&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267748900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.617.8" data-doi="10.1016/j.physrep.2009.11.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fortunato</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Community detection in graphs.</span> <abbr class="cit-jnl-abbrev">Phys. Rep.</abbr> <span class="cit-vol">486</span>: <span class="cit-fpage">75</span>–<span class="cit-lpage">174</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhys.%2BRep.%26rft.volume%253D486%26rft.spage%253D75%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.physrep.2009.11.002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.physrep.2009.11.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.617.9" data-doi="10.1038/ng.3259"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Greene</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krishnan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ricciotti</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zelaya</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Understanding multicellular function and disease with human tissue-specific networks.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">569</span>–<span class="cit-lpage">576</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D47%26rft.spage%253D569%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3259%26rft_id%253Dinfo%253Apmid%252F25915600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3259&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25915600&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.617.10" data-doi="10.1186/1471-2105-8-236"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hart</span>  <span class="cit-name-given-names">G. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marcotte</span>  <span class="cit-name-given-names">E. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A high-accuracy consensus map of yeast protein complexes reveals modular nature of gene essentiality.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">236</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DHart%26rft.auinit1%253DG.%2BT.%26rft.volume%253D8%26rft.spage%253D236%26rft.epage%253D236%26rft.atitle%253DA%2Bhigh-accuracy%2Bconsensus%2Bmap%2Bof%2Byeast%2Bprotein%2Bcomplexes%2Breveals%2Bmodular%2Bnature%2Bof%2Bgene%2Bessentiality.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-8-236%26rft_id%253Dinfo%253Apmid%252F17605818%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-8-236&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17605818&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.617.11" data-doi="10.1109/MCSE.2007.55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hunter</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Matplotlib: a 2D graphics environment.</span> <abbr class="cit-jnl-abbrev">Comput. Sci. Eng.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">90</span>–<span class="cit-lpage">95</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComput.%2BSci.%2BEng.%26rft.volume%253D9%26rft.spage%253D90%26rft_id%253Dinfo%253Adoi%252F10.1109%252FMCSE.2007.55%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1109/MCSE.2007.55&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.617.12" data-doi="10.1371/journal.pone.0000988"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marcotte</span>  <span class="cit-name-given-names">E. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">An improved, bias-reduced probabilistic functional gene network of baker’s yeast, <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e988</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DLee%26rft.auinit1%253DI.%26rft.volume%253D2%26rft.issue%253D10%26rft.spage%253De988%26rft.epage%253De988%26rft.atitle%253DAn%2Bimproved%252C%2Bbias-reduced%2Bprobabilistic%2Bfunctional%2Bgene%2Bnetwork%2Bof%2Bbaker%2527s%2Byeast%252C%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000988%26rft_id%253Dinfo%253Apmid%252F17912365%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0000988&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17912365&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.617.13" data-doi="10.1101/gr.102749.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lehner</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vavouri</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fraser</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Predicting genetic modifier loci using functional gene networks.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1143</span>–<span class="cit-lpage">1153</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.102749.109%26rft_id%253Dinfo%253Apmid%252F20538624%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzgvMTE0MyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi82MTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.617.14" data-doi="10.1101/gr.118992.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blom</span>  <span class="cit-name-given-names">U. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">P. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shim</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marcotte</span>  <span class="cit-name-given-names">E. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Prioritizing candidate disease genes by network-based boosting of genome-wide association data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1109</span>–<span class="cit-lpage">1121</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.118992.110%26rft_id%253Dinfo%253Apmid%252F21536720%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIxLzcvMTEwOSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi82MTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.617.15" data-doi="10.1073/pnas.0712255105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mani</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Onge</span>  <span class="cit-name-given-names">R. P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartman</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giaever</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roth</span>  <span class="cit-name-given-names">F. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Defining genetic interaction.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">3461</span>–<span class="cit-lpage">3466</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0712255105%26rft_id%253Dinfo%253Apmid%252F18305163%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA1LzkvMzQ2MSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi82MTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.617.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pandey</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">A. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">An integrative multi-network and multi-classifier approach to predict genetic interactions.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">6</span>: e1000928.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.617.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pedregosa</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varoquaux</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gramfort</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michel</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thirion</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Scikit-learn: machine learning in Python.</span> <abbr class="cit-jnl-abbrev">J. Mach. Learn. Res.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">2825</span>–<span class="cit-lpage">2830</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMach.%2BLearn.%2BRes.%26rft.volume%253D12%26rft.spage%253D2825%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.617.18" data-doi="10.1101/gr.077693.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Qi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suhail</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">Y. -Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boeke</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bader</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Finding friends and enemies in an enemies-only network: a graph diffusion kernel for predicting novel genetic interactions and co-complex membership from yeast genetic interactions.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1991</span>–<span class="cit-lpage">2004</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.077693.108%26rft_id%253Dinfo%253Apmid%252F18832443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxOC8xMi8xOTkxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzYxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.617.19" data-doi="10.1093/nar/gkp914"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ruepp</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waegele</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lechner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brauner</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunger-Kaltenbach</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">CORUM: the comprehensive resource of mammalian protein complexes–2009.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">D497</span>–<span class="cit-lpage">D501</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkp914%26rft_id%253Dinfo%253Apmid%252F19884131%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzOC9zdXBwbF8xL0Q0OTciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzIvNjE3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.617.20" data-doi="10.1101/gr.1239303"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shannon</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Markiel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ozier</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baliga</span>  <span class="cit-name-given-names">N. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Cytoscape: a software environment for integrated models of biomolecular interaction networks.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">2498</span>–<span class="cit-lpage">2504</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DShannon%26rft.auinit1%253DP.%26rft.volume%253D13%26rft.issue%253D11%26rft.spage%253D2498%26rft.epage%253D2504%26rft.atitle%253DCytoscape%253A%2BA%2BSoftware%2BEnvironment%2Bfor%2BIntegrated%2BModels%2Bof%2BBiomolecular%2BInteraction%2BNetworks%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.1239303%26rft_id%253Dinfo%253Apmid%252F14597658%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxMy8xMS8yNDk4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzYxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.617.21" data-doi="10.1093/nar/gkv453"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">C. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shim</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">FlyNet: a versatile network prioritization server for the <em>Drosophila</em> community.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">W91</span>–<span class="cit-lpage">W97</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv453%26rft_id%253Dinfo%253Apmid%252F25943544%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjQzL1cxL1c5MSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi82MTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.617.22" data-doi="10.1093/nar/gkj109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stark</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Breitkreutz</span>  <span class="cit-name-given-names">B.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reguly</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boucher</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Breitkreutz</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">BioGRID: a general repository for interaction datasets.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">D535</span>–<span class="cit-lpage">D539</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DStark%26rft.auinit1%253DC.%26rft.volume%253D34%26rft.issue%253Dsuppl_1%26rft.spage%253DD535%26rft.epage%253DD539%26rft.atitle%253DBioGRID%253A%2Ba%2Bgeneral%2Brepository%2Bfor%2Binteraction%2Bdatasets%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkj109%26rft_id%253Dinfo%253Apmid%252F16381927%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzNC9zdXBwbF8xL0Q1MzUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzIvNjE3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.617.23" data-doi="10.1126/science.1246981"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sabatini</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genetic screens in human cells using the CRISPR-Cas9 system.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">343</span>: <span class="cit-fpage">80</span>–<span class="cit-lpage">84</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1246981%26rft_id%253Dinfo%253Apmid%252F24336569%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzNDMvNjE2Ni84MCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi82MTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.617.24" data-doi="10.1073/pnas.1517883113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">G. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cui</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pregernig</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Milani</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Multiplexed barcoded CRISPR-Cas9 screening enabled by CombiGEM.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">2544</span>–<span class="cit-lpage">2549</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1517883113%26rft_id%253Dinfo%253Apmid%252F26864203%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTEzLzkvMjU0NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi82MTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.617.25" data-doi="10.1073/pnas.0406614101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">L. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tong</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldberg</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Combining biological networks to predict genetic interactions.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">15682</span>–<span class="cit-lpage">15687</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0406614101%26rft_id%253Dinfo%253Apmid%252F15496468%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAxLzQ0LzE1NjgyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzYxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.617.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">MacLeod</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">A. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">BioGPS and MyGene.info: organizing online, gene-centric information.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">D561</span>–<span class="cit-lpage">D565</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D41%26rft.spage%253DD561%26rft_id%253Dinfo%253Apmid%252F23175613%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23175613&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.617.27" data-doi="10.1126/science.1123287"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sternberg</span>  <span class="cit-name-given-names">P. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genome-wide prediction of <em>C. elegans</em> genetic interactions.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">311</span>: <span class="cit-fpage">1481</span>–<span class="cit-lpage">1484</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DZhong%26rft.auinit1%253DW.%26rft.volume%253D311%26rft.issue%253D5766%26rft.spage%253D1481%26rft.epage%253D1484%26rft.atitle%253DGenome-wide%2Bprediction%2Bof%2BC.%2Belegans%2Bgenetic%2Binteractions.%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1123287%26rft_id%253Dinfo%253Apmid%252F16527984%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMTEvNTc2Ni8xNDgxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzYxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/2/617.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/2/607" title="The Bright Fluorescent Protein mNeonGreen Facilitates Protein Expression Analysis In Vivo" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/2/625" title="upSET, the Drosophila homologue of SET3, Is Required for Viability and the Proper Balance of Active and Repressive Chromatin Marks" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/2"><span>Volume 7 Issue 2, February 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/2.cover-source.jpg?itok=m4uMC43I" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/2/617.supplemental" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-20cD3ZucHM1g1LQpHAbEFgyBBf4vyqIeEmhAuW8TuxM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/2/617" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/2/617" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/2/617" class="active" data-icon-position="" data-hide-link-title="0">Predictability of Genetic Interactions from Functional Gene Modules</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460586" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-ttOu-NP6fUT7EWCHCzhZb2l2XvoqZkxknFhzWqFFtmI" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/2/617.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/2/617" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-N1LFJXtb6WusyQ8bPdjfvk0cQpxnvZeswD4b-dK1xmc" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/2/617" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/2/617.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460586" id="node460586--2" data-pisa="ggg;7/2/617" data-pisa-master="ggg;g3.116.035915" data-apath="/ggg/7/2/617.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Predictability of Genetic Interactions from Functional Gene Modules</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-3365-2702" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jonathan H. Young</span> and <span class="highwire-citation-author" data-delta="1">Edward M. Marcotte</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">617-624;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.035915 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460586--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jonathan H. Young </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Computational Engineering and Sciences, The University of Texas at Austin, Texas 78712</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Systems and Synthetic Biology, The University of Texas at Austin, Texas 78712</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BH.%2BYoung%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Young%20JH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJonathan%2BH.%2BYoung%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3365-2702" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jonathan H. Young</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Edward M. Marcotte </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Systems and Synthetic Biology, The University of Texas at Austin, Texas 78712</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Molecular Biosciences, The University of Texas at Austin, Texas 78712</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Edward%2BM.%2BMarcotte%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Marcotte%20EM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEdward%2BM.%2BMarcotte%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:marcotte@icmb.utexas.edu" class="" data-icon-position="" data-hide-link-title="0">marcotte@icmb.utexas.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/2/617" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/2/617&amp;title=Predictability%20of%20Genetic%20Interactions%20from%20Functional%20Gene%20Modules" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/2/617&amp;title=Predictability%20of%20Genetic%20Interactions%20from%20Functional%20Gene%20Modules" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/2/617&amp;title=Predictability%20of%20Genetic%20Interactions%20from%20Functional%20Gene%20Modules" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/2/617&amp;text=Predictability%20of%20Genetic%20Interactions%20from%20Functional%20Gene%20Modules" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/2/617&amp;title=Predictability%20of%20Genetic%20Interactions%20from%20Functional%20Gene%20Modules" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/2/617&amp;t=Predictability%20of%20Genetic%20Interactions%20from%20Functional%20Gene%20Modules" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/2/617&amp;title=Predictability%20of%20Genetic%20Interactions%20from%20Functional%20Gene%20Modules" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/2/617&amp;title=Predictability%20of%20Genetic%20Interactions%20from%20Functional%20Gene%20Modules" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460586/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460586" id="node460586--3" data-pisa="ggg;7/2/617" data-pisa-master="ggg;g3.116.035915" data-apath="/ggg/7/2/617.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Predictability of Genetic Interactions from Functional Gene Modules</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-3365-2702" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jonathan H. Young</span> and <span class="highwire-citation-author" data-delta="1">Edward M. Marcotte</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">617-624;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.035915 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460586--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jonathan H. Young </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Computational Engineering and Sciences, The University of Texas at Austin, Texas 78712</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Systems and Synthetic Biology, The University of Texas at Austin, Texas 78712</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BH.%2BYoung%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Young%20JH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJonathan%2BH.%2BYoung%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3365-2702" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jonathan H. Young</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Edward M. Marcotte </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Systems and Synthetic Biology, The University of Texas at Austin, Texas 78712</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Molecular Biosciences, The University of Texas at Austin, Texas 78712</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Edward%2BM.%2BMarcotte%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Marcotte%20EM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEdward%2BM.%2BMarcotte%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:marcotte@icmb.utexas.edu" class="" data-icon-position="" data-hide-link-title="0">marcotte@icmb.utexas.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=617&amp;rft.epage=624&amp;rft.atitle=Predictability%20of%20Genetic%20Interactions%20from%20Functional%20Gene%20Modules&amp;rft.volume=7&amp;rft.issue=2&amp;rft.date=2017-02-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Young%2C+Jonathan+H.&amp;rft.au=Marcotte%2C+Edward+M."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460586/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460586/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460586/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460586/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460586/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460586/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460586/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460586/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460586/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460586/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460586/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460586/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/2/617" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-4" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/2/617.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/2/617.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/2/617.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/2/617.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
