<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/2/543" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/2/543.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/2/543.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/2/543.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/2/543.atom" />
<meta name="HW.pisa" content="ggg;7/2/543" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A SNP Based Linkage Map of the Arctic Charr (Salvelinus alpinus) Genome Provides Insights into the Diploidization Process After Whole Genome Duplication" />
<meta name="DC.Identifier" content="10.1534/g3.116.038026" />
<meta name="DC.Date" content="2017-02-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Nugent et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Diploidization, which follows whole genome duplication events, does not occur evenly across the genome. In salmonid fishes, certain pairs of homeologous chromosomes preserve tetraploid loci in higher frequencies toward the telomeres due to residual tetrasomic inheritance. Research suggests this occurs only in homeologous pairs where one chromosome arm has undergone a fusion event. We present a linkage map for Arctic charr ( Salvelinus alpinus ), a salmonid species with relatively fewer chromosome fusions. Genotype by sequencing identified 19,418 SNPs, and a linkage map consisting of 4508 markers was constructed from a subset of high quality SNPs and microsatellite markers that were used to anchor the new map to previous versions. Both male- and female-specific linkage maps contained the expected number of 39 linkage groups. The chromosome type associated with each linkage group was determined, and 10 stable metacentric chromosomes were identified, along with a chromosome polymorphism involving the sex chromosome AC04. Two instances of a weak form of pseudolinkage were detected in the telomeric regions of homeologous chromosome arms in both female and male linkage maps. Chromosome arm homologies within the Atlantic salmon ( Salmo salar ) and rainbow trout ( Oncorhynchus mykiss ) genomes were determined. Paralogous sequence variants (PSVs) were identified, and their comparative BLASTn hit locations showed that duplicate markers exist in higher numbers on seven pairs of homeologous arms, previously identified as preserving tetrasomy in salmonid species. Homeologous arm pairs where neither arm has been part of a fusion event in Arctic charr had fewer PSVs, suggesting faster diploidization rates in these regions." />
<meta name="DC.Contributor" content="Cameron M. Nugent" />
<meta name="DC.Contributor" content="Anne A. Easton" />
<meta name="DC.Contributor" content="Joseph D. Norman" />
<meta name="DC.Contributor" content="Moira M. Ferguson" />
<meta name="DC.Contributor" content="Roy G. Danzmann" />
<meta name="article:published_time" content="2017-02-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="A SNP Based Linkage Map of the Arctic Charr (Salvelinus alpinus) Genome Provides Insights into the Diploidization Process After Whole Genome Duplication" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Diploidization, which follows whole genome duplication events, does not occur evenly across the genome. In salmonid fishes, certain pairs of homeologous chromosomes preserve tetraploid loci in higher frequencies toward the telomeres due to residual tetrasomic inheritance. Research suggests this occurs only in homeologous pairs where one chromosome arm has undergone a fusion event. We present a linkage map for Arctic charr (&lt;i&gt;Salvelinus alpinus&lt;/i&gt;), a salmonid species with relatively fewer chromosome fusions. Genotype by sequencing identified 19,418 SNPs, and a linkage map consisting of 4508 markers was constructed from a subset of high quality SNPs and microsatellite markers that were used to anchor the new map to previous versions. Both male- and female-specific linkage maps contained the expected number of 39 linkage groups. The chromosome type associated with each linkage group was determined, and 10 stable metacentric chromosomes were identified, along with a chromosome polymorphism involving the sex chromosome AC04. Two instances of a weak form of pseudolinkage were detected in the telomeric regions of homeologous chromosome arms in both female and male linkage maps. Chromosome arm homologies within the Atlantic salmon (&lt;i&gt;Salmo salar&lt;/i&gt;) and rainbow trout (&lt;i&gt;Oncorhynchus mykiss&lt;/i&gt;) genomes were determined. Paralogous sequence variants (PSVs) were identified, and their comparative BLASTn hit locations showed that duplicate markers exist in higher numbers on seven pairs of homeologous arms, previously identified as preserving tetrasomy in salmonid species. Homeologous arm pairs where neither arm has been part of a fusion event in Arctic charr had fewer PSVs, suggesting faster diploidization rates in these regions.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/02/01" />
<meta name="citation_mjid" content="ggg;7/2/543" />
<meta name="citation_id" content="7/2/543" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/2/543" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/2/543.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/2/543.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/2/543.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.038026" />
<meta name="citation_pmid" content="27986793" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="2" />
<meta name="citation_num_pages" content="14" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="543" />
<meta name="citation_lastpage" content="556" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Cameron M. Nugent" />
<meta name="citation_author_institution" content="Department of Integrative Biology, University of Guelph, Ontario N1G 2W1, Canada" />
<meta name="citation_author_email" content="nugentc@uoguelph.ca" />
<meta name="citation_author_email" content="rdanzman@uoguelph.ca" />
<meta name="citation_author" content="Anne A. Easton" />
<meta name="citation_author_institution" content="Department of Integrative Biology, University of Guelph, Ontario N1G 2W1, Canada" />
<meta name="citation_author" content="Joseph D. Norman" />
<meta name="citation_author_institution" content="Department of Integrative Biology, University of Guelph, Ontario N1G 2W1, Canada" />
<meta name="citation_author" content="Moira M. Ferguson" />
<meta name="citation_author_institution" content="Department of Integrative Biology, University of Guelph, Ontario N1G 2W1, Canada" />
<meta name="citation_author" content="Roy G. Danzmann" />
<meta name="citation_author_institution" content="Department of Integrative Biology, University of Guelph, Ontario N1G 2W1, Canada" />
<meta name="citation_author_email" content="nugentc@uoguelph.ca" />
<meta name="citation_author_email" content="rdanzman@uoguelph.ca" />
<meta name="citation_reference" content="citation_title=Evolutionary Genetics of Fishes;citation_title=Tetraploidy and the evolution of salmonid fishes;citation_author=F. W. Allendorf;citation_author=G. H. Thorgaard;citation_pages=1-46;citation_year=1984" />
<meta name="citation_reference" content="citation_journal_title=J. Hered.;citation_author=F. W. Allendorf;citation_author=S. Bassham;citation_author=W. A. Cresko;citation_author=M. T. Limborg;citation_author=L. W. Seeb;citation_title=Effects of crossovers between homeologs on inheritance and population genomics in polyploid-derived salmonid fishes.;citation_volume=106;citation_year=2015;citation_pmid=25838153;citation_doi=10.1093/jhered/esv015" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=S. F. Altschul;citation_author=W. Gish;citation_author=W. Miller;citation_author=E. W. Myers;citation_author=D. J. Lipman;citation_title=Basic local alignment search tool.;citation_pages=403-410;citation_volume=215;citation_year=1990;citation_issue=3;citation_pmid=2231712;citation_doi=10.1006/jmbi.1990.9999" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=U. Bergthorsson;citation_author=D. I. Andersson;citation_author=J. R. Roth;citation_title=Ohno’s dilemma: evolution of new genes under continuous selection.;citation_volume=104;citation_year=2007;citation_pmid=17942681;citation_doi=10.1073/pnas.0707158104" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=C. Berthelot;citation_author=F. Brunet;citation_author=D. Chalopin;citation_author=A. Juanchich;citation_author=M. Bernard;citation_title=The rainbow trout genome provides novel insights into evolution after whole-genome duplication in vertebrates.;citation_pages=3657;citation_volume=5;citation_year=2014;citation_pmid=24755649;citation_doi=10.1038/ncomms4657" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=M. S. O. Brieuc;citation_author=C. D. Waters;citation_author=J. E. Seeb;citation_author=K. A. Naish;citation_title=A dense linkage map for chinook salmon (Oncorhynchus tshawytscha) reveals variable chromosomal divergence after an ancestral whole genome duplication event.;citation_pages=447-460;citation_volume=4;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=L. Comai;citation_title=The advantages and disadvantages of being polyploid.;citation_pages=836-846;citation_volume=6;citation_year=2005;citation_issue=11;citation_pmid=16304599;citation_doi=10.1038/nrg1711" />
<meta name="citation_reference" content="citation_journal_title=Genetica;citation_journal_abbrev=Genetica;citation_author=R. G. Danzmann;citation_author=K. Gharbi;citation_title=Gene mapping in fishes: a means to an end.;citation_pages=3-23;citation_volume=111;citation_year=2001;citation_issue=1-3;citation_pmid=11841175;citation_doi=10.1023/A:1013713431255" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=R. G. Danzmann;citation_author=E. A. Davidson;citation_author=M. M. Ferguson;citation_author=K. Gharbi;citation_author=B. F. Koop;citation_title=Distribution of ancestral proto-Actinopterygian chromosome arms within the genomes of 4R-derivative salmonid fishes (Rainbow trout and Atlantic salmon).;citation_pages=557-557;citation_volume=9;citation_year=2008;citation_pmid=19032764;citation_doi=10.1186/1471-2164-9-557" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=J. G. de Boer;citation_author=R. Yazawa;citation_author=W. S. Davidson;citation_author=B. F. Koop;citation_title=Bursts and horizontal evolution of DNA transposons in the speciation of pseudotetraploid salmonids.;citation_pages=422-422;citation_volume=8;citation_year=2007;citation_pmid=18021408;citation_doi=10.1186/1471-2164-8-422" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=1-10;citation_volume=6;citation_year=2011;citation_pmid=21949733;citation_doi=10.1371/journal.pone.0016163" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. C. Glaubitz;citation_author=T. M. Casstevens;citation_author=F. Lu;citation_author=J. Harriman;citation_author=R. J. Elshire;citation_title=TASSEL-GBS: a high capacity genotyping by sequencing analysis pipeline.;citation_pages=e90346;citation_volume=9;citation_year=2014;citation_pmid=24587335;citation_doi=10.1371/journal.pone.0090346" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=S. Gonen;citation_author=N. R. Lowe;citation_author=T. Cezard;citation_author=K. Gharbi;citation_author=S. C. Bishop;citation_title=Linkage maps of the Atlantic salmon (Salmo salar) genome derived from RAD sequencing.;citation_pages=166;citation_volume=15;citation_year=2014;citation_pmid=24571138;citation_doi=10.1186/1471-2164-15-166" />
<meta name="citation_reference" content="citation_journal_title=Genetica;citation_author=S. E. Hartley;citation_title=Chromosomes and constitutive heterochromatin distribution in Arctic charr, Salvelinus alpinus (L.) (Pisces: Salmonidae).;citation_pages=161-166;citation_volume=79;citation_year=1989" />
<meta name="citation_reference" content="citation_journal_title=Biol. Proced. Online;citation_author=R. A. Hermansen;citation_author=T. R. Hvidsten;citation_author=S. R. Sandve;citation_author=D. A. Liberles;citation_title=Extracting functional trends from whole genome duplication events using comparative genomics.;citation_pages=1;citation_volume=18;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Int. J. Biol. Sci.;citation_author=Z. Jiang;citation_author=H. Wang;citation_author=J. J. Michal;citation_author=X. Zhou;citation_author=B. Liu;citation_title=Genome wide sampling sequencing for SNP genotyping: methods, challenges and future development.;citation_pages=100-108;citation_volume=12;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Cytogenetic and genome research;citation_journal_abbrev=Cytogenet Genome Res;citation_author=J. Jurka;citation_author=V. V. Kapitonov;citation_author=A. Pavlicek;citation_author=P. Klonowski;citation_author=O. Kohany;citation_title=Repbase Update, a database of eukaryotic repetitive elements.;citation_pages=462-467;citation_volume=110;citation_year=2005;citation_issue=1-4;citation_pmid=16093699;citation_doi=10.1159/000084979" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. Kasahara;citation_author=K. Naruse;citation_author=S. Sasaki;citation_author=Y. Nakatani;citation_author=W. Qu;citation_title=The medaka draft genome and insights into vertebrate genome evolution.;citation_pages=714-719;citation_volume=447;citation_year=2007;citation_issue=7145;citation_pmid=17554307;citation_doi=10.1038/nature05846" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=H. H. Kazazian;citation_title=Mobile Elements: Drivers of Genome Evolution;citation_pages=1626-1632;citation_volume=303;citation_year=2004;citation_issue=5664;citation_pmid=15016989;citation_doi=10.1126/science.1089670" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=M. Kodama;citation_author=M. S. O. Brieuc;citation_author=R. H. Devlin;citation_author=J. J. Hard;citation_author=K. A. Naish;citation_title=Comparative mapping between coho salmon (Oncorhynchus kisutch) and three other salmonids suggests a role for chromosomal rearrangements in the retention of duplicated regions following a whole genome duplication event.;citation_pages=1717-1730;citation_volume=4;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=E. Küttner;citation_author=J. Nilsson;citation_author=S. Skulason;citation_author=S. Gunnarsson;citation_author=M. M. Ferguson;citation_title=Sex chromosome polymorphisms in Arctic charr and their evolutionary origins.;citation_pages=852-861;citation_volume=54;citation_year=2011;citation_issue=10;citation_pmid=21970434" />
<meta name="citation_reference" content="citation_journal_title=J. Hered.;citation_author=W. A. Larson;citation_author=G. J. McKinney;citation_author=M. T. Limborg;citation_author=M. V. Everett;citation_author=L. W. Seeb;citation_title=Identification of multiple QTL hotspots in sockeye salmon (Oncorhynchus nerka) using genotyping-by-sequencing and a dense linkage map.;citation_pages=122-133;citation_volume=107;citation_year=2015;citation_pmid=26712859" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=S. Lien;citation_author=L. Gidskehaug;citation_author=T. Moen;citation_author=B. J. Hayes;citation_author=P. R. Berg;citation_title=A dense SNP-based linkage map for Atlantic salmon (Salmo salar) reveals extended chromosome homeologies and striking differences in sex-specific recombination patterns.;citation_pages=615-615;citation_volume=12;citation_year=2011;citation_pmid=22182215;citation_doi=10.1186/1471-2164-12-615" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=S. Lien;citation_author=B. F. Koop;citation_author=S. R. Sandve;citation_author=J. R. Miller;citation_author=P. Matthew;citation_title=The Atlantic salmon genome provides insights into rediploidization.;citation_pages=200-205;citation_volume=533;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=M. T. Limborg;citation_author=G. McKinney;citation_author=L. W. Seeb;citation_author=J. E. Seeb;citation_title=Recombination patterns reveal information about centromere location on linkage maps.;citation_pages=655-661;citation_volume=16;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Proc. Biol. Sci.;citation_author=D. J. Macqueen;citation_author=I. A. Johnston;citation_title=A well-constrained estimate for the timing of the salmonid whole genome duplication reveals major decoupling from species diversification.;citation_volume=281;citation_year=2014;citation_pmid=24452024;citation_doi=10.1098/rspb.2013.2881" />
<meta name="citation_reference" content="citation_journal_title=Hereditas;citation_journal_abbrev=Hereditas;citation_author=G. R. A. Margarido;citation_author=A. P. Souza;citation_author=A. A. F. Garcia;citation_title=OneMap: software for genetic mapping in outcrossing species.;citation_pages=78-79;citation_volume=144;citation_year=2007;citation_issue=3;citation_pmid=17663699;citation_doi=10.1111/j.2007.0018-0661.02000.x" />
<meta name="citation_reference" content="citation_journal_title=J. Hered.;citation_author=B. May;citation_author=M. E. Delany;citation_title=Meiotic models to explain classical linkage, pseudolinkage, and chromosomal pairing in tetraploid derivative salmonid genomes: II. Wright is still right.;citation_volume=106;citation_year=2015;citation_pmid=26320244;citation_doi=10.1093/jhered/esv056" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=G. McKinney;citation_author=L. Seeb;citation_author=W. Larson;citation_author=D. Gomez-Uchida;citation_author=M. Limborg;citation_title=An integrated linkage map reveals candidate genes underlying adaptive variation in chinook salmon (Oncorhynchus tshawytscha).;citation_pages=769-783;citation_volume=16;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Int. J. Evol. Biol.;citation_author=O. Mendivil Ramos;citation_author=D. E. Ferrier;citation_title=Mechanisms of gene duplication and translocation and progress towards understanding their relative contributions to animal genome evolution.;citation_pages=1-10;citation_volume=2012;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=H. K. Moghadam;citation_author=M. M. Ferguson;citation_author=R. G. Danzmann;citation_title=Evidence for Hox gene duplication in rainbow trout (Oncorhynchus mykiss): a tetraploid model species.;citation_pages=804-818;citation_volume=61;citation_year=2005;citation_issue=6;citation_pmid=16315109;citation_doi=10.1007/s00239-004-0230-5" />
<meta name="citation_reference" content="citation_journal_title=J. Fish Biol.;citation_author=H. K. Moghadam;citation_author=M. M. Ferguson;citation_author=R. G. Danzmann;citation_title=Linkage variation at the sex-determining locus within Fraser strain Arctic charr (Salvelinus alpinus).;citation_pages=294-301;citation_volume=71;citation_year=2007;citation_doi=10.1111/j.1095-8649.2007.01598.x" />
<meta name="citation_reference" content="citation_journal_title=Journal of Fish Biology;citation_journal_abbrev=Journal of Fish Biology;citation_author=H. K. Moghadam;citation_author=M. M. Ferguson;citation_author=R. G. Danzmann;citation_title=Whole genome duplication: challenges and considerations associated with sequence orthology assignment in Salmoninae.;citation_pages=561-574;citation_volume=79;citation_year=2011;citation_issue=3;citation_pmid=21884100;citation_doi=10.1111/j.1095-8649.2011.03030.x" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=Y. Nakatani;citation_author=H. Takeda;citation_author=Y. Kohara;citation_author=S. Morishita;citation_title=Reconstruction of the vertebrate ancestral genome reveals dynamic genome reorganization in early vertebrates.;citation_volume=17;citation_year=2007;citation_pmid=17652425;citation_doi=10.1101/gr.6316407" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=J. D. Norman;citation_author=R. G. Danzmann;citation_author=B. Glebe;citation_author=M. M. Ferguson;citation_title=The genetic basis of salinity tolerance traits in Arctic charr (Salvelinus alpinus).;citation_pages=81-81;citation_volume=12;citation_year=2011;citation_pmid=21936917" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=J. D. Norman;citation_author=M. Robinson;citation_author=B. Glebe;citation_author=M. M. Ferguson;citation_author=R. G. Danzmann;citation_title=Genomic arrangement of salinity tolerance QTLs in salmonids: A comparative analysis of Atlantic salmon (Salmo salar) with Arctic charr (Salvelinus alpinus) and rainbow trout (Oncorhynchus mykiss).;citation_pages=420-420;citation_volume=13;citation_year=2012;citation_pmid=22916800;citation_doi=10.1186/1471-2164-13-420" />
<meta name="citation_reference" content="citation_journal_title=Physiol. Genomics;citation_author=J. D. Norman;citation_author=M. M. Ferguson;citation_author=R. G. Danzmann;citation_title=Transcriptomics of salinity tolerance capacity in Arctic charr (Salvelinus alpinus): a comparison of gene expression profiles between divergent QTL genotypes.;citation_volume=46;citation_year=2014;citation_pmid=24368751;citation_doi=10.1152/physiolgenomics.00105.2013" />
<meta name="citation_reference" content="citation_title=Evolution by Gene Duplication.;citation_author=S. Ohno;citation_year=1970" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=C. O. Ostberg;citation_author=L. Hauser;citation_author=V. L. Pritchard;citation_author=J. C. Garza;citation_author=K. A. Naish;citation_title=Chromosome rearrangements, recombination suppression, and limited segregation distortion in hybrids between Yellowstone cutthroat trout (Oncorhynchus clarkii bouvieri) and rainbow trout (O. mykiss).;citation_pages=570;citation_volume=14;citation_year=2013;citation_pmid=23968234;citation_doi=10.1186/1471-2164-14-570" />
<meta name="citation_reference" content="citation_journal_title=Biological Reviews (Cambridge; Print);citation_journal_abbrev=Biological Reviews (Cambridge; Print);citation_author=R. Phillips;citation_author=P. Ráb;citation_title=Chromosome evolution in the Salmonidae (Pisces): an update.;citation_pages=1-25;citation_volume=76;citation_year=2001;citation_issue=1;citation_pmid=11325050;citation_doi=10.1017/S1464793100005613" />
<meta name="citation_reference" content="citation_journal_title=Environ. Biol. Fishes;citation_author=R. B. Phillips;citation_author=M. P. Matsuoka;citation_author=K. M. Reed;citation_title=Characterization of charr chromosomes using fluorescence in situ hybridization.;citation_pages=223-228;citation_volume=64;citation_year=2002" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=C. E. Rexroad;citation_author=Y. Palti;citation_author=S. A. Gahr;citation_author=R. L. Vallejo;citation_title=Maximum likelihood estimates of two-locus recombination fractions under some natural inequality restrictions.;citation_pages=1-1;citation_volume=9;citation_year=2008;citation_pmid=18173855" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. Sakamoto;citation_author=R. G. Danzmann;citation_author=K. Gharbi;citation_author=P. Howard;citation_author=A. Ozaki;citation_title=A Microsatellite Linkage Map of Rainbow Trout (Oncorhynchus mykiss) Characterized by Large Sex-Specific Differences in Recombination Rates;citation_pages=1331-1345;citation_volume=155;citation_year=2000;citation_issue=3;citation_pmid=10880492" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=L. Schrader;citation_author=J. W. Kim;citation_author=D. Ence;citation_author=A. Zimin;citation_author=A. Klein;citation_title=Transposable element islands facilitate adaptation to novel environments in an invasive species.;citation_pages=1717-1730;citation_volume=5;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=J. J. Smith;citation_author=S. Kuraku;citation_author=C. Holt;citation_author=T. Sauka-Spengler;citation_author=N. Jiang;citation_title=Sequencing of the sea lamprey (Petromyzon marinus) genome provides insights into vertebrate evolution.;citation_pages=415-421;citation_volume=45;citation_year=2013;citation_pmid=23435085;citation_doi=10.1038/ng.2568" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=S. E. Staton;citation_author=J. M. Burke;citation_title=Evolutionary transitions in the Asteraceae coincide with marked shifts in transposable element abundance.;citation_pages=623;citation_volume=16;citation_year=2015;citation_pmid=26290182;citation_doi=10.1186/s12864-015-1830-8" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=J. S. Taylor;citation_author=J. Raes;citation_title=Duplication and divergence: the evolution of new genes and old ideas.;citation_pages=615-643;citation_volume=38;citation_year=2004;citation_pmid=15568988;citation_doi=10.1146/annurev.genet.38.072902.092831" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=E. R. Timusk;citation_author=M. M. Ferguson;citation_author=H. K. Moghadam;citation_author=J. D. Norman;citation_author=C. C. Wilson;citation_title=Depletion of eIF4G from yeast cells narrows the range of translational efficiencies genome-wide.;citation_pages=68-68;citation_volume=12;citation_year=2011;citation_pmid=21269496;citation_doi=10.1186/1471-2164-12-68" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=H. Y. Tsai;citation_author=D. Robledo;citation_author=N. R. Lowe;citation_author=M. Bekaert;citation_author=J. B. Taggart;citation_title=Construction and annotation of a high density SNP linkage map of the Atlantic salmon (Salmo salar) genome.;citation_pages=2173-2179;citation_volume=6;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=R. K. Waples;citation_author=L. W. Seeb;citation_author=J. E. Seeb;citation_title=Linkage mapping with paralogs exposes regions of residual tetrasomic inheritance in chum salmon (Oncorhynchus keta).;citation_pages=17-28;citation_volume=16;citation_year=2015;citation_pmid=25712438" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=R. A. Woram;citation_author=K. Gharbi;citation_author=T. Sakamoto;citation_author=B. Hoyheim;citation_author=L. Holm;citation_title=Comparative Genome Analysis of the Primary Sex-Determining Locus in Salmonid Fishes;citation_pages=272-280;citation_volume=13;citation_year=2003;citation_issue=2;citation_pmid=12566405;citation_doi=10.1101/gr.578503" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=R. A. Woram;citation_author=C. McGowan;citation_author=J. A. Stout;citation_author=K. Gharbi;citation_author=M. M. Ferguson;citation_title=A genetic linkage map for Arctic char (Salvelinus alpinus): evidence for higher recombination rates and segregation distortion in hybrid versus pure strain mapping parents.;citation_pages=304-315;citation_volume=47;citation_year=2004;citation_issue=2;citation_pmid=15060583;citation_doi=10.1139/g03-127" />
<meta name="citation_reference" content="citation_journal_title=Isozymes;citation_journal_abbrev=Isozymes Curr Top Biol Med Res;citation_author=J. E. Wright;citation_author=K. Johnson;citation_author=A. Hollister;citation_author=B. May;citation_title=Meiotic models to explain classical linkage, pseudolinkage, and chromosome pairing in tetraploid derivative salmonid genomes.;citation_pages=239-260;citation_volume=10;citation_year=1983;citation_pmid=6354984" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=A. Yano;citation_author=R. Guyomard;citation_author=B. Nicol;citation_author=E. Jouanno;citation_author=E. Quillet;citation_title=An immune-related gene evolved into the master sex-determining gene in rainbow trout, Oncorhynchus mykiss.;citation_pages=1423-1428;citation_volume=22;citation_year=2012;citation_issue=15;citation_pmid=22727696;citation_doi=10.1016/j.cub.2012.05.045" />
<meta name="citation_reference" content="citation_journal_title=Evol. Appl.;citation_author=A. Yano;citation_author=B. Nicol;citation_author=E. Jouanno;citation_author=E. Quillet;citation_author=A. Fostier;citation_title=The sexually dimorphic on the Y-chromosome gene (sdY) is a conserved male-specific Y- chromosome sequence in many salmonids.;citation_pages=486-496;citation_volume=6;citation_year=2013;citation_pmid=23745140;citation_doi=10.1111/eva.12032" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A SNP Based Linkage Map of the Arctic Charr (Salvelinus alpinus) Genome Provides Insights into the Diploidization Process After Whole Genome Duplication" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2.cover-source.jpg" />
<meta name="twitter:description" content="Diploidization, which follows whole genome duplication events, does not occur evenly across the genome. In salmonid fishes, certain pairs of homeologous chromosomes preserve tetraploid loci in higher frequencies toward the telomeres due to residual tetrasomic inheritance. Research suggests this occurs only in homeologous pairs where one chromosome arm has undergone a fusion event. We present a linkage map for Arctic charr ( Salvelinus alpinus ), a salmonid species with relatively fewer chromosome fusions. Genotype by sequencing identified 19,418 SNPs, and a linkage map consisting of 4508 markers was constructed from a subset of high quality SNPs and microsatellite markers that were used to anchor the new map to previous versions. Both male- and female-specific linkage maps contained the expected number of 39 linkage groups. The chromosome type associated with each linkage group was determined, and 10 stable metacentric chromosomes were identified, along with a chromosome polymorphism involving the sex chromosome AC04. Two instances of a weak form of pseudolinkage were detected in the telomeric regions of homeologous chromosome arms in both female and male linkage maps. Chromosome arm homologies within the Atlantic salmon ( Salmo salar ) and rainbow trout ( Oncorhynchus mykiss ) genomes were determined. Paralogous sequence variants (PSVs) were identified, and their comparative BLASTn hit locations showed that duplicate markers exist in higher numbers on seven pairs of homeologous arms, previously identified as preserving tetrasomy in salmonid species. Homeologous arm pairs where neither arm has been part of a fusion event in Arctic charr had fewer PSVs, suggesting faster diploidization rates in these regions." />
<meta name="og-title" property="og:title" content="A SNP Based Linkage Map of the Arctic Charr (Salvelinus alpinus) Genome Provides Insights into the Diploidization Process After Whole Genome Duplication" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/2/543" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Diploidization, which follows whole genome duplication events, does not occur evenly across the genome. In salmonid fishes, certain pairs of homeologous chromosomes preserve tetraploid loci in higher frequencies toward the telomeres due to residual tetrasomic inheritance. Research suggests this occurs only in homeologous pairs where one chromosome arm has undergone a fusion event. We present a linkage map for Arctic charr ( Salvelinus alpinus ), a salmonid species with relatively fewer chromosome fusions. Genotype by sequencing identified 19,418 SNPs, and a linkage map consisting of 4508 markers was constructed from a subset of high quality SNPs and microsatellite markers that were used to anchor the new map to previous versions. Both male- and female-specific linkage maps contained the expected number of 39 linkage groups. The chromosome type associated with each linkage group was determined, and 10 stable metacentric chromosomes were identified, along with a chromosome polymorphism involving the sex chromosome AC04. Two instances of a weak form of pseudolinkage were detected in the telomeric regions of homeologous chromosome arms in both female and male linkage maps. Chromosome arm homologies within the Atlantic salmon ( Salmo salar ) and rainbow trout ( Oncorhynchus mykiss ) genomes were determined. Paralogous sequence variants (PSVs) were identified, and their comparative BLASTn hit locations showed that duplicate markers exist in higher numbers on seven pairs of homeologous arms, previously identified as preserving tetrasomy in salmonid species. Homeologous arm pairs where neither arm has been part of a fusion event in Arctic charr had fewer PSVs, suggesting faster diploidization rates in these regions." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2.cover-source.jpg" />
    <title>A SNP Based Linkage Map of the Arctic Charr (Salvelinus alpinus) Genome Provides Insights into the Diploidization Process After Whole Genome Duplication | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__DbqSGiVjr90FgBIuPlGEAiimgNNAH8S6qbiYFsi8JyQ__E2bjoC6LAI1j30ywBL_JgYRwIhvNl2n0yRyt_snD9IM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"ENJ-8GQO8KxEtSegcH4onTpC5uBQyXjeRlbaReuj2KU","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460564","apath":"\/ggg\/7\/2\/543.atom","pisa":"ggg;7\/2\/543","ac":{"\/ggg\/7\/2\/543.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/2\/543.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/2\/543"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/2\/543"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/2\/543":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022x8S08IYh1mJ10rEVIoaUJwuxLCVsm1fYzyB1oF3LoxzdoGAHyiT87TIZsRrlxby\\\\\\\/J\\\\\\\/ZC10n8IATFyJgxrkrTk890YpMHMKS\\\\\\\/3ss3KOjv5AZ7ncTiETfuwAWfBVbje4ccpRsEUvr51aUD+Q9oOxcMxcuu8hhV6Q7SIoAy90ouUfdukOEHA8e7bx7FjB2EyKkQnQb\\\\\\\/rLdCO+5Wop\\\\\\\/X0gkO7QEwYBD69sboFWsUYkH3D\\\\\\\/E+C4aBEj202Ic1qzyes4xaIEsU00Agkqcv+TtTK8H66jP9t4uSy5q\\\\\\\/Ewg5PcMT8BK852ZjRpReLzfgbKZJo9ftDqB2MU4sOvu6NeXAYiWzApwTbbfnJ91i8eC05PzyuA5H0rbrJPVtqKCMA9X49kI3KWT8DXbot8UBBIdKiwoSTrGtg5SlUttDb+Ddpgj8BiwHfgRR9AFDdhufK65KQDw1Tq+HjXI0z6woRIA2xiLWA7TipdRefdL4p0XgLSprrYIiM6AkL+EHUfGMBUV3ua2g7a37VFCHp3z\\\\\\\/dWiy73uM9v0jizniACDQC32+6yoZZy38eNJ5ilBGKOCO2\\\\\\\/lzwIC1HFcT2Hh0NEE\\\\\\\/PLyyqH4\\\\\\\/JptH6UI2hyP\\\\\\\/Afdm7CvFVugX1bQnrRBSgFkMLpt\\\\\\\/Ecvpi4\\\\\\\/qQ0Erq\\\\\\\/aISbs5fTOMKO5IaYl+tMULHymP6VDDl+l2qXEJvalTBMxDq\\\\\\\/NCjTWkZbYRgeIR6KWjNBQqugigk7hrI7wsf5wGYyBPc87u63AUhwYzhP2As2yEhmd7\\\\\\\/\\\\\\\/Ha4yYtczpkXLpNjbNXXVAV5lD19zhgt32qw8xlytxX86qRM+x\\\\\\\/cshLImGrsmP\\\\\\\/dA+fRABIs7k0tUsuiS02TkgooYh5K0t4EKelbMQWc3yC\\\\\\\/Hc\\\\\\\/Z\\\\\\\/x\\\\\\\/cs7khzf3HjhREyJb4sx0X\\\\\\\/XUSiuwdYdRGQ8e4llbiDX5tSEm+k+h957AqxPQqQI6Ocw2H8Pu+4xRauOVF69E6s+iAfHPH3CtLmww1jcBoutOMrD6CGbN4qyjw0yY9DDw850vT81LAHAYZu7w0EOfzmaARmQBg0ssiQyT8F4EQnoJ7FJyFqEx\\\\\\\/HLp1ZYmhlf+YtpQBw6VcoYin9811rpe2WKIcN61Zh+QA8l9WsKETA\\\\\\\/Oyg0GnAH\\\\\\\/1eOMGmb1NjIk05QNR3BqQikYenlp8m4ZIpLqtGrXZ\\\\\\\/U0zxai6ER+rANQnS7rC1YBPZZXP+Q8vg8n9OHp2sxBkwz2xyf1xqc8iRbfkgafYf+yhuGrmINdP9gPtWr1We9IaCPf4B8HSj7Un9+yos+PQLXFsglf\\\u0022,\\\u0022iv\\\u0022:\\\u0022FRhruupmsrnRj6lSvSnJZw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d6fdd3607121b2da580fdea6f0743839727e0b4745d264be13218215c0126abd\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022H3bvr1RAU5TLFtR8bwhtyS7fwriOrqdYWxRB8e\\\\\\\/X7DFcCdCVQ2MmDpdZU3fxLRbyqVQi44l7qVFeayGXKyo2L8g8Pxl155gr3a5FieKwvKT5N1QzdxMPGXlyMycHEJFyGLIZM5CvcleyyrggtXAxyi0pd966SHzudZtLtmb9SJn5YEU0hObXSM+FaV4\\\\\\\/gSBhVv0DbHVWCj2tuzSUowF4Ip106AtaZKdX528ZgeCykzUt2ra+Vu2nF\\\\\\\/RlxBbgLIAiBSaPrsYH\\\\\\\/Idwf6R4UOnyBMBKkLGtHKqs+qZYOc\\\\\\\/H2Ci8UFy+5H0jRCM6Ch6S\\\\\\\/E3K2\\\\\\\/bsuVYptNJ5IoXAES+9cLxAYuqzYQx0RSLc4iggVprCGYJ+l8fXL5EsichqZdq+47TpiasNii1mb2v6kMHhBdPB9pKsXUHtRiD\\\\\\\/Xyci99BSt+gd5RAGUXbHW67dj9iOnU2AjQHJXQlSpZlIlVVwIQX6\\\\\\\/Yz34OgTqeyahKJ8L9WsnJNkR0ySksQgcFa86CQfpZlrdSQ\\\\\\\/inwaJAqj8L9QpX84l5DjzJgXlA0og3+bJweqsfxaLYXACMqgKJDHe0bBHz2iMCnBQS35SPZ0eBlsXnFfvTEiP\\\\\\\/T8Rpx839CKo\\\\\\\/KKr0h7f5ueZx4qY46JvIEk4aG31Wajt1KXLuSNp5RIO\\\\\\\/AhSWUroH9vew0Dh89u2qL8JpaIlJHwenFX+UUBiMMD4vIgCLY5XlJB6YrsIzV\\\\\\\/s2mDnqhUWjz7mEebBETNsxM5\\\\\\\/g2lnmQICIdZE8DmIA1BAl\\\\\\\/dWad9HyP0Lpc7+fwDcxq0Mlfv27EFZS+h7zOmVRvbobtAfaXASeNVfrTFLlQowSN+vc26LOpVvTZ5HTIzVBc5SMDnbLTBQyTBNXDoNh4PyJe9HKUsfIdUdbSA3epmtUcwPrMkmlEtBWMHDia8HHAK7stOfJEvAFZMMvdSK+o7h1BuE9qlOBxaDjACtPDc8vXpXMvMT19R\\\\\\\/27sQnZMsM7JyZClzq4dHlD80qfk\\\\\\\/RD15kW5nwvFxFFHYNz9pRSxzbGTQ\\\\\\\/+Q8btuW5WAUoMEsjNXtiqF+hlcJ0cmRaNkuV8FkW3I648wx4CwukHs0q51mDGifvimGmw7e2bImHeJstUfi1Es1vUIVYOtT92FLPzJSgN68HFw90nwhg6ZhuFLz2cAFZvysixtNIGqKKoRSgt7ty6lUunqshapynN8Q7MHo9ZLgEvk1DFaWWlu87u\\\\\\\/WufeJuyqkyqDT4a3\\\\\\\/YPvX99yaj8\\\\\\\/20KeuomsRi9HR6\\\\\\\/JXKRp7GKC\\\\\\\/lTxuNhawnCY1SKgV\\\\\\\/gA7f59L+hFJNEF0Tk0zvxde6JFfp\\\\\\\/k\\\\\\\/2BwOIvp0FbHVYGt2U6XvsqsQLlbOvMS4A6x6okWclyb0EnTXXjXIP53QblwZukxGMAvGrab4P6NRBnDzw4T+IsV3VEQ\\\\\\\/ovLgFzLwgApBSTrpSkHakEtXezAYwOz3nlX4Vq1xGlfRS4PAS9Y6bxstIj4ZiArJvaFHYMPWMd\\\\\\\/auKvg2eKCXP4s2EfV3sFosY8NoG+jya+5tI5jxJQdJ9yy20aWgnsL7XG2ag+6i+5qEqhKG5V8xGAwiywLJ47kWRDq\\\\\\\/ynLhoN1wLlOhzrHbWiaTysSgcBxvx5LJHjANQdBCtMT1WjsTJezX1PfuzveFtmWe6Lpcsg\\\\\\\/Fk02UrQVwSHYv\\\\\\\/RftcpOCoOkSy4Eno9iKh7FPnouPuEe+h7Zueu0Q7frujB90GxAXe7rMJcrsXVn9bbGq+6k3kk3VU3akFBj8PN\\\\\\\/\\\\\\\/2crA+N49lv9WJPaslOJFmhzUua\\\\\\\/1y+3cm0DBAaL9GijvgQ5aIFbkRocPke3JX6BIa8qHSvWjhHLBhFdwVWAPe+3ScqEt\\\\\\\/K5tPzVGO4dfoXgSJlgLnH2vz69AiMhgtIPyna2+fIusRr3A3zy\\\\\\\/z2uNAkhQkdfFHZX79YuIjnHWueewUbZCO1zVohGBLUd1ANYqdawXpaHtmuoADAPLvVJxDaBvC4Bf0N4\\\\\\\/eNyjS2Mj6oxTAIP90nUW8ZB+0FBaqQppg927JPgQGIboQRL\\\\\\\/i+8weRNqTVfkz\\\\\\\/55iTpFAvydven+EsqveMRlEA3FXAcyOpRD8Ao9ckznBeh87eKiOiTAETsrfB\\\\\\\/LRgxz\\\\\\\/RYxq41P2QJY75Fm4dktOZ1cAMg4FdwCVgw3PlrydCyS1ogN5+AQBsWCVgtdJGmWmFYWxxCKAEluWPI7v1Zn7W8Q\\\\\\\/ZFl0Pl8VRrZZuTAb6PNwM6vlSlSlwarD9aw7kwfxKh1vW7\\\\\\\/QLeVY6Y\\\\\\\/yxtKaN3iGb8K3p6stBbEWc4HZru6Bn4lRcotnhQXlm8QH1YCpwZbQz8HdYsfbLA9ZnRqp7vkVRhHuOV1SAiw352EffYvSZOxVl4Ict3nGL8DxsxxSLwBUYPuA1En6BgZP366anVtLGV4sKUaJEoZ7OmvFRaMRpAtuULQ2UCRte4IiXRnkMCbs+LFBnA3kp2pe\\\\\\\/rmt6hat837Vl0nW367hxq1lvaGtj4bT+2OF4Wkf8t9oK6JzmnKSuoMABX3o0xM5kI0qepzkN4Xd1Cko20D9b3YQklSADImZ4NIiasmILpW1WKejtt\\\\\\\/W9lZD6mYnUy+LwmGxGibHslALauS9IWxbx2n3T4WnuiyziSpFmcCc27CGob7OHb+uutlY\\\\\\\/ZBt4pa6xVdGu7jVXx5PcL3QtEYN5LyFrYPSF57+QfM7ke09oBxCXrFaxlSsFdnHwkNlMXpjKIOEXcJiGQMCoLEvYzQd12nMKpvJluT5xTfzcvaN6TW02vsV+zc6HTmvgfniu0tZSS8uoIeik3sFXnTA3RypNG2TgeBgAfsxu9Hc4fWVxx8skY1tdJwpafsDepZz0h7cGVHdHLZxR3AFXPh6WGQYJJ6Mjt3zx\\\\\\\/tXY0eQIpc67afBjUqoBszrDTSBbwaKJD6r0P8TTFC47xqXsRrq5FcgEsmZh1k9GAs5EtSAYBAzHavbZcFxpT0UjvGCKtWkFnth06v7W9Jpa1dMiS7vL2wIskHs\\\\\\\/vp0a1Rvo8VR3+XmonmA7w\\\\\\\/tYFWO5GBp9Rr5oVywJIbEWlNlDpjM6jhKgHYUjfVI4sOEC51M12OfIN1SQ6TdoAuXP+UMtv1q6LSdIHSbMuYCho78UKFP\\\\\\\/7SCDTHmM6+RqJfH9359k3NBx3FrNaLPoHNUTyyqGNqWmj2nQIPMxYpV8+qaKJ8mqMXaNIfGYGzfq7pFnS6TyUY192SfBxVJwNaClDbQ4vjCwGwTZvPM\\\\\\\/2e7F6ZCkVK52nfqxIpItJ35vOUhSz06jUn+YbMVPkKErKtg1dvqaJBUlL9wUFpSpxrbdJ5fAc2HZBTUPNGUJKxr9FDHtrSmudqRfiBWbFfmFVuaIC3jLeS5M+U\\\\\\\/eCmSCt9UF2eshpwO84nZQbjP6ToOmRyAFHxqNmsK36M3NSWtvl1iv14NjLd6XG2mz+Rz7ivqFuIAk\\\\\\\/jjyqfWRqvrTwRTAR4tGi3vGPtfesVcebgAfdVgnFwZNqH1sJk7qeJmEOwI0GcTEMabDh9FyCU9MhRIeqvqlssH+ffp53opwD\\\\\\\/vMG\\\\\\\/5c+wwq2O2xFG5Dz\\\\\\\/5vEAQSTlNeNURkVXveeKPHNn1yIoHTjAV639LCyi0NJp14rn9j4U0QLJlpu2n0CVqsT\\\\\\\/UBi1wEajyfOXAmPg\\\\\\\/jRkLYNxmXyMTCx2Bcyzu2FzTWYIBs3RJWLwV933ZW7zMNNAgANwpY54ugCASPrnrnYn7slpbHAjzRRx7oDNW5AWh772Rauu\\\\\\\/EiXM8eQbDKs3OhfQf3vlFD+51xiNay14eWyJRmix71+QGEz+OBswpxqkZJz0ktw\\\\\\\/07QFueXDdsbFdw4ITE62cZt3biGv0zGidJBGzdvZPIojWYpQhJJEhK4opPjLHme\\\\\\\/Oy25L3ay\\\\\\\/N8aGGvYyWORVGRdG0lZfBPGLmhp5KLQkWdR3dJm0HrZMkGc+moVjHkbNqk8W0HOxhK1YamD\\\\\\\/lDtRy7YUiNVcChdoFgRrHv4ptiJO38CNwvG3Lhj1gwQBSDz\\\\\\\/YoH8ga9iaMUcSqXtdzu\\\\\\\/EvpXjLefsfjWIwIS42EBvbGuRL1xLrwH9HiZ4mHynLRuCa+zDSzAzn43uXwU293bZW\\\\\\\/e3OhWW3IQNdKBHGE20ME4FIRat8Qk6F1QA\\\\\\\/xQurg3kMNxliC3zE1KqJfc5mo00Y3kH\\\\\\\/Vs6DF2PnuOTaHCVTj0mVLN951a5YYX1JG1kHbkyQsHXneCMn5ypZjqwwfqA+wkve5FLFxa9wmOLa8Z1t9rfw+YqCKi4IHZDKk2+O9XQwyHx5SIUxLE46MBwNFiQrS8cGoofZKhSlaTSXl1WwYe492pFbhZ9W0sBdkTByy5wEiL+HfTK812xQfX2JuhCiLUBroRj+XXT\\\\\\\/bF37RX+g1Y9jjEodvD3p4GC7cjCnGg1D2xA6QeQibc+Jqz8lr7ljwXkTyOtVshHeJon1wAFqgIDR0v36snhV5qwozrEZH560iPSJI20cD5iHhC20rOwDE\\\\\\\/cBHNPN2f1AkCsxWCaCNOWhYy2zJ16q4iKLzYFgV0GGTsRqkJRQFSjF06h5VFihfqyL8jjvdKd85f+IPjeSt3TFmC23FiwqD0brOdyMUtPxXbk3EoTl1RO43hsYZFHomjco32L6Ly7xztn+ucu7hf60C5VNTyPD0pGj\\\\\\\/Jrv+2Omar8Hjxa8erMv6w06PJwfvp55RfbW4P8DtstHlzS\\\\\\\/bYrsHWFr0uqG8+Pk0r90R3ByveUBjh3XozLTCqmXKBM\\\\\\\/H6mNzRaWDcOpoOBJ+3fY0usoJHPza83IWDTnapeTfracCVIw9jAUHTr+SvVPBQWKoWI0KwJEexO8tVTrMlhD6eXl2KqNh6A3vAohAKrdF0gjEJjWNCBe+vmxwJjNI0I2Z1P7aoNnGfYoyZqSYt4RZKMnMLK2lNl\\\\\\\/B7r1GGgepXPUjzNYYC3sIARmhYGODOjXhVi1Vujav9M6cDmS+wXCWwTSgDm2bbb4mehUmund0q6vDGXzxFu20wZQy6B3gQpLv33lrVTd+Bb+DcxEMdgPnjPOozzCy6+CUYimnswLqL\\\\\\\/trhpLYRdSDHMSdpKkMcV1pZn\\\\\\\/B24AeKVJOj64nwWxdhDNF51eg5HoWc1HjSS3Ov6O4rCV8SGQhXGVHTxJ9pejQe37hqfIXWQvLQEtT1cMDkC5G5dbmYA53P1+4GGPgPRA7xtUndnMhgMWZWwfB8LleRZMWd7xoWdFEXczb5n\\\\\\\/fcNqZjqrcXni5\\\\\\\/qOWth04N59y37OXgZAir7N2dRVYdl+fAlDPGDtDCiMd5YJ\\\\\\\/Kwt5CCiiaZzKSc5HA1O4e\\\\\\\/E\\\\\\\/T3+tgGyWniHPBWORD6z1w0SY0118BIXJNyGXvUvtwY4+GKX+iGnhYxhQx0ETN9xL3GIB0ovcj+8iqAo3Ac8I5i+jFn7NwvmjeM7J9Im97uoMJoh1ctGd5ivrm1M9ZIu7z\\\\\\\/zJPh5GW1zyjRMllJdfNmPz+EGIwsULIih79FNYev0HVGklYyvk78\\\\\\\/qj3VrliqGhaJGb1j43W4MnzLga2in2K0RhBgoCTne3y\\\\\\\/yx8ZyOWB0hNCHbgP7BW6LdvnN7HouKv12nIziW9MTh5RAJCyYebit3bly6BXw9yEDpPjOnenyv7vrvUpweRTWI8VVnP\\\\\\\/cuhClGv0hlvNaXdCP8y1PhG8LY6\\\\\\\/dnCvi6O0wNYPWxUuQaAP11gbh7HJuPsK3VXA2HnW1Pw+1GgNt8OsQSf2BrGvBtjOheqi318hIVdeDyXjJzVBnCqnMi70Mc7jKd69T24abA8vWqL4G9Dnpm7GeiFI1TMb9Yo61pOdTH7YtJnL4umUYB9vK8AhqFxZWLBjmmwh7nooFRU3mUvJT0z1vZr1DqL7b2xgmTqUimjB4iSH1u0xANg3HrAQTLRlNot646viON8p+RjEIfsX2Xw8KpYHo2L0PQ8RHna\\\\\\\/91NKG25a\\\\\\\/SItknyK0oOwFL8IrGsnvcABm+FsDGS2S43PBU7O4DeRj0k9XbhGbjFTrT0ueiSpRY1ePUyY5tnT5PXwPS81+129WM\\\\\\\/6caSEj0bNW+yAantmGoM272WMThoZjGNsujD2QWRFuPgjUgzFfZdcqjkI2M\\\\\\\/s0B3obzd7CPWJJ2DuvuqfbJhl\\\\\\\/94rJBahE8TgHb0NVUkKI+4Q95LAeIxcG3RPJqPcUTUaR\\\\\\\/LaV837XDnmLVlnozBOrioS0luW7TmQFeYh7TzuN5Hx5SQ2cKH7VBQfze0S3sLmb8lbERClEcTjK2bV4DeDZalDL6M9qNkUUXRVQkzMBaRoAfR2ARdbu8Sv89pNI7K0XqQz\\\\\\\/UOp2td7FEC1PTSWk5OgOggvAaMBdB6eyPH64jN6rY6u5I6Q2Rqz3SO1N\\\\\\\/CZPVj2PHv3JX3KsT00MpQIeKoslKZ6JpSvk9Qqq5PBiK4d2sqxWlblxJgynDdy7WHvEhnGvFclrA4xXTDiQKXIpcDg0SDjawhQeuVmYvy7RhAzX9jjHkyuitFcmVJr4Xo5omL3v92sCnF5hNbMEz5AT9EMjJ8JXwSj9VBrRDQ+TgyLbZs+V\\\\\\\/JP3EQU8bNh+53UqCuULuQ2xMyhgtCaUoZKJ\\\\\\\/pnxrnuYvPeqkBU2zIu3AptfjgUWaqohrC9p7PxVjlECTbL8i8psN6cRzmCH6ox28Sfl5qnWwaRCKA\\\\\\\/b4d28qiGleulD+j+vE7tYtsEVBiQLM4FUS3pemlokqgzTgB\\\\\\\/EASucLtuEuHXg01dGzc5z5djjpS78MkSMzXJ2DQndIPGMqWjqRi06humziivhA5TdwKr8s2ZIj4ps1Vdk4RTR4wg2ZMKlUZez0FO3VAR2rvC5tKJqHV3KyCG9UD0H3oe8NbuWH\\\\\\\/UlZQhzNcTkDNtNsU+TZNjMHcd5YtwMbYz\\\\\\\/Yfs6IaeBVNF52fgdIpfbJL31pcDKTyHM6V4ikoHTadxanMMXP+QzZglAIia2Ac8IoGHyFeReuVyVTya4\\\\\\\/gnahW5S611bWaBmBKZFF1qpV+urWWoZ0CryTzrP5qL9CdMKs9ykbvHMGM8P7pQl1EIGKvo30VDsFs2tT4GqUdVp4NQdI8N7TtDrnNvC0rYSqw3VPRheNtEhpZaZ4cF8KzXxdCiiYX\\\\\\\/bjlRV\\\\\\\/CpljZGVS+FDKkm7Mz3mBuWgmbQH3IV1ld\\\\\\\/dnBkl8WrguI7C\\\\\\\/PFtQSp950FVO+vic91NkLtUCK7ITXz3zmZc1A17cMSXng2Xw4y+j\\\\\\\/Yz0GFPSCH9h5tqoR8gfymAtjqPS32M+IX1QtJRFQvhb8UvdXbvSVYpuMVJmSCQlUWhqz413d8rWJK294Hy0dwBS8I5KymLAXBrzmJ70UwYnfNm6a3wVHG+lH38lUx4TXoT0wwSXAj\\\\\\\/ve6wT6QfhItazk7WMr\\\\\\\/yTe2BAaY9A1eNT\\\\\\\/OZsvfYJKmm4G39us5MmECmjwkfpmR\\\\\\\/XWn4I4Vk7\\\\\\\/xIoOBrcrbwmtC\\\\\\\/YDdOS3BPTE3PyxI\\\\\\\/gvsqgm8jKl5LtVN8zT\\\\\\\/7O4Ml24q40h8BWAFcgVChPLfKoynrrElaTsGpIEVS4JjHlEvu9gA8XWTkoGsAaDs6uYKs9BgZ5JW6ipTUbOmk\\\\\\\/BcPJRTx14Myc9kQvCQQ\\\\\\\/ywkuZhnRRARJMPUWu7mVbHmA00QfifkwZ0yTEt4SJa9wzlUK7uZG9UZ25idyHOJw1tH0T0crDSjZAP6AFLFwZPPvIrb5ZpEepDWnJW4qYI2ci5gHaS01wE\\\\\\\/1DS5NdSdbdeG8um\\\\\\\/DOJ7\\\\\\\/tOvGn1gB2DxnSxMSZRn8Ty5QxmY1QnzfB7yfVGDg6fmCMGQ0YbuZ+LTT1R0kyXxOZWLvriNFdqwA3w\\\\\\\/VUXbdn1Ud4qlERfhaDg5rMPTX03k70cF2brOD43eVRaoAkwJC1uWgUqdJQ61V01+dhlOkyRviRgJbKbnOgaSa0MF7y+Yf\\\\\\\/h0s3th7U47WtzQqcNOu2DT3t5jJJ38hP7JUPBHngCsfr\\\\\\\/CQO+R049CZ7B8sfCR8TtkWVUWqv7SRNSdKausb8JpdxJxJTu2cMKV+cgCDLuM+w4o2L5I0wU6QX74T9HtIqkjTVDKS2XHHHBAz8f0CSAhpDaXW+Xqw45tiPJX+Ow6\\\\\\\/zIqCDXWPFCbxWZ6C841qnk8LNHo6oWm908bCPgqKnWLDIhx54lFIYsYEyqIwZfp7hDntpRrWgHIHLgklF81T1iKpp1pDy9IUkEnj+xKuS5YKQ\\\\\\\/ks0MROOzIruv3yxLg8t7EM1gkRqHrZVPacuKq6lyAvlu9wpTyDBvjcDhEQikftIkgRnisjHzl3ThGMRKh7nOnmIESc7TuD4kZuGEn+HdVdCy7TMpx4\\\\\\\/HhG2CU6UeD15jnQrLlDkvdkAcOSuEZMvU2w0jgPo8G+Zl+GlVo5TNMH6U7ibAyjx175vuYse4xaopnD0O4jN6h5a+mrswxiXVPozGYQmaPlgdno7mYuc4WOogX\\\\\\\/W63N7+w4ATVPhZKND8C\\\\\\\/KXBfcwYydouE1CK+tqIXpfRfuK6d\\\\\\\/x2gTpIpbL1dv8S3wUYXoT4JLPgG6SdXyCUD3fS\\\\\\\/98Dp1\\\\\\\/dNOGbMZluTya+FiAgG5oNaMl50l10p9hgB5UQMaoddyPn43SL07eiy7F11jQ7jsRsELIKKIRn84P1VH41JHqgLnyaBYYHjInnT6zGpRREkkRTR5RA4t5yWhIiZmJCt+rb+SbKWRFXMJX+iUx3H70sgr7eNPIf68dcDraEOMbj8\\\\\\\/gQ5RmY+OQ0iuDhkojPfURKyXe27PuKukdzDfzQq1ELsPdX5Hz2WBe1B3qWKRsbfdZW7QFuBAeN2l2yfW2SPG2CTw3FXF8J7GCa3ZgMaGnoTzTPanunFgY1nPBdHUTEWMQ6YhuJ+Z8VA8uvPBvoDPGxVjfBBdQShcomO8DhnjK7SwM3PVWM1SERbWSdBvyL+Px6kgDHX4+ZF1aTssgjU1E+TqezpqqIyXAFAiGhTEkAvwgbtpC8\\\\\\\/UGwdfQiICZWGl53ilJ0W6cgUAIWwb\\\\\\\/vsQ3CeyVII\\\\\\\/tvP3dS36rjB+i9iXZjAIhlw6RU7UPpNmNusu9vctAlE8d7wFWXfWLAVobhzT+5j+ahYDYlRRDNAMi7jeGcm67+Pz+obvxwP9Khk2hbsynoXtH8\\\\\\\/agqsRCSSIpnt4uWvYglD5n6UxDIJ4Y1mpyoIv+INcMUAO2yL3bEthIdSi2FRcGwR8zdh7rxfImXBOBmeL+iss0QG8ItQ4AmAQLqYCLGucDIfmjq4lfsiXHd\\\\\\\/8G7\\\\\\\/MQpMm2g\\\\\\\/iCHfZwckHO9BKbre\\\\\\\/\\\\\\\/QL69qL5nuXL8eAHgUEruWY9Iw6ycXS1Q+s5\\\\\\\/UyVKJJLFPMlDJKR+VKRbMFKJv6Xvkrdd\\\\\\\/LNPAkQzHnkW6wKKCr9vE\\\\\\\/EwMMs8y2\\\\\\\/Gx6DNTOr4+jttdebMeMbeTsGI0UonZsA0f5q2srjA2NYKC3sepjFAWCh9KXSzGw8t2yFI9YkoGzXVpCPaKAeDvRR5Y+ahtXjj9xOldn+8cyxEHgyrzpCYr6bFeK7wONGApc\\\\\\\/wTfxSU9dt68d1scsyQcubu6ZXnUJDHFP7jMRaudfOMTsAeu4O4mpryb2VbaVQ2IhDDkc3Bn8Mv4iz2QemtRTL3RXB8DiREMh90iqs\\\\\\\/FI4Yzzmv4g9Dxd5U58s03T3OypYed7VbMk0sPFQ7t8Dpb4rTBtCwdYdCbsaGwHfA57PhMNL\\\\\\\/\\\\\\\/W7oWYEpJfYhKFwowsvV8LCSHSDYzweTGaCB2NQoS6\\\\\\\/DmDlj4z7kDsFxLYVTu5+KgdkQu2IOr0o4cd+gGCF0eZnPNEkDIk5x6abX6f0gp3JBfNytA8jbL4tZBNANBlccfY9gAxqdtT+LAtMBkVwt4qMW7aFH4fZ59sNjXiUrYRSt3D2\\\\\\\/6pDrWGwODINdwTteO5LSchjKcxAMl\\\\\\\/wRCprx7wulhWNrg9WYVfadIFh9xYtA+bbsahjthafA0CBkvzdwSsfuxR84qAKUT1xtNdoDFx\\\\\\\/+PbOl57rgy+9pK+VOe0RSjnd6NyiOu0zQn2H4sMVC\\\\\\\/N3Kna6aHg73KPyG6nzOSnOpdFBh3wN1RQRAmAu1GdyiFYnm\\\\\\\/XYnYsDrX6KB8OQh4EPpq4MD+HOdLFF0KY2BJMpkeCYoyBOxK+K8Y1lkdS42ekdJ4FTr3M2j2m+Qn5BCi4lSFGgoQhZII0V0ko9VA1VQE3fS+DyGi0MqDf2GrCy44zUdfNQKZ6pfw3pFSncsl2r0zV6Ljg4MwOqBIl7VoeT7KiSFx5T335rOQdOV4gkiHdHGcadvZQ0PLar0XxNjfhG35GXxNS2vwnvoXb+IgNjLUYyOGFFZcSCWH44KHFPs3w1b37J7LOH54e30IkNweUOOmb8O2mNlVpYv+B\\\\\\\/zgVh3fjlnEW35sD3o9x04MyKPiubXV51mOMG5zjn6FDOccJmu5CR0t4bl\\\\\\\/IW36n2qbOM+s7R7FsQWUYNpVSiur2RYJ16s8CqpXa1gcRlbPhVZuZj4TrA1qC4KceKqsxXmePnmgEIkVasHhB\\\\\\\/kUWIHyOcqVtgZSRxOqysWzy6PrtgFABO\\\\\\\/ZlCK+glSqLyXivuglDaBb+xpRVDDcVf0rPCtzjZtVKQi4oXP7aSx02zZqmk\\\\\\\/KBgXfqJujZzlSheSmh0qtNsWE3KDjUc2t6SR+Inqe2ycQjVpTEeZhCGnGw51oy6mxWCmZpqUuyV7EGDbQKrLONjuHQX+c8E56F2aooZUfT8Imcz6OcJpKo3Z1Lra5E+Hk55j0k31Rmy\\\\\\\/lKO\\\\\\\/tc\\\\\\\/eOdRYfvr5VYW347uZ2L+H6mo9oqXCSq6n+6hsVPDuLAGbn5jp0AENPASpCrARWXfEIeYpvmVI3KYfNz\\\\\\\/4IS+ahIO8PbnIty5CpIuJFJLi+BkFpECMoL9m6jM+NDEBvwIuR+pFQCUcOrjqzJAKCjTi1a+qnzbRUFAjSbZPoAnV1BQaL8IbA5YiFRV7s3N0Rbt6dnoGBro1rv5Um42Yv6qOmSHl9r079+2VzYVMxk\\\\\\\/mXg+wfC2iIZnwCx71pnIqJAhh1VanuKYprr9yNL\\\\\\\/c6cqYvpnWNonWRdhz9kK7ZeVObkw33t959s70yhaZI5APsCR3yS7JpMn7DI7uIrzUFFPnhAZmNes5XNTSFxlgmOw4GQ12Tx0lnussGifzdqalFZLzs+zqa9XyuH6mI57xftv3Gcuob2eGAJIJLw5p0yu6kibqsL\\\\\\\/1R16T+COHiwLnJvDr+B59AQ0nPwpDoTc\\\\\\\/6g3fOuAei9WiiFJTxZr3pobO28aXajC4HMWRePrrxHjLNS39KvwF76O86VfkTupKsCRI4330T\\\\\\\/lxxcdgAuf8\\\\\\\/pU3TP\\\\\\\/cYxeV2hOPudlEyxdppXzfo5oi0xsuq9+xNwHsUtW1G+7OeoQhOb\\\\\\\/mBfkJu0sqdxN+90t5a8rmuxqnu6h9bKbCbebUyv10H+X\\\\\\\/r0U6E2hoCdPHxmLYnIihPeTn4yAStV30ux5HS4Sqt3K9mvuFq3YjwFbiSTvcIWaJuKyaS+yZW8tBJ6OCjaBFDF3ZRjxyZZSQ3gpHxt7T0pLf5eY9dXwrYQoOzEfp1pkt1vTUAl1a5gR+tTWdx21qnn3Mj6SFJKmj3\\\\\\\/xRGUVnKZVxKbQJjaidojcpX9akjX482Af\\\\\\\/uZYkSn35vXLB3i6WLPgqnn3o+rhnh4U5meay\\\\\\\/C0O2ZSzvgt95pm5KRHo1LFuM8lm\\\\\\\/ZFGrB\\\\\\\/Xw7TixOKR28gFWY\\\\\\\/vlbGeaueIzBYZ6FX+zGKAG5Jbg5liwy9m7IwTz9gpe0rCVChwT7c1J\\\\\\\/T2ZrODjqNYwPFMgu6jP9VUqePMFskjwnHJ6l5vo2zUy35KoivzD9NgDtdxDhHN\\\\\\\/fbE2OgmyKgmN6glYcSQNnQqJnWLJMhGWjbeiVNjE6FY5nARiabR+bQ5B8s8aEQec\\\\\\\/EI1xL+xJD0o3MgP0jiLNsZaOW1tq+2LSq2mpfbjgZX98C0EvQ2L9181Qos2SzvSGmDHLJfNhAvkUT3fxsJkC\\\\\\\/oCe7v+Wc2q3PXLPOLD9V4tqjnKn2MG2yBuVXx4s7FDUVfZYZJJe+EM64hpqlHLSXfDOIwE7c15Q+LEDJrt5bKmhHvxY81jXxfeSFf72m+TEhLPUictdjCskHptGBLKfitDengUmFoRw3iOeEFzCNUpd21zRZ3wW0iGDEBntT6yqCyGdsVdq\\\\\\\/PUt6yizbikY6x7bbJm1M3VR8PCar0kNLJlITbPA+wSzVPpOaSOPj5IJz50Tgeuh4jhGCes95mDVlKAUC7aFLCw1Xe73I+SQ8MosGQ1F7ht4w9665Bbb1j7joe9AM4IuAE0KL\\\\\\\/DH49Vrw4vtHDAFNTaYikjUmmGSYQNTA\\\\\\\/\\\\\\\/2jGN6NnDjhhgJUbLydskTpCP5k0g3nej8ourjtDX0Uok6xGewND6aDMBBwDc+YpOb9eMQoSk6wm72EhM+5ErbM2ZsM8rXs8CbYUkKnZudMh4KrEnHOuLAP61vHOQ0e0e31PlJyPfPZCqpzzWGknkxMbXMThx6I2avTS4ERKkECfpnRQy4LebIAqUu5Y3XJy\\\\\\\/Gy8CFh9488\\\\\\\/0dH0mfTtpUlIe+6n+stmVAczOVtLrODwY+gQU6kq9amGfVuQBXsQRiZvO5Bl9eFThIIYpqmdMHgv0oga3erp4d+Wr3QVDhlncMtU55PcV7RBZeA4o2LsEV4Afyy93K6A+H2OqmrOZQh6CtSECrHF5bw05Xv3Tls\\\\\\\/LdchBpJRaNvlitqfaOQjOJ\\\\\\\/X2d0khaISvRuz9lkEVnvEHiF602\\\\\\\/\\\\\\\/RB4vA24GvtzUyaOW\\\\\\\/fQsELH4pMW21WzM1QjvbL8UM57Mp349WL1HgeN\\\\\\\/jgLe2Ke8BY5j4Ba4yykooyFnJMMMtOO3ZZX71fXDsM+JHqlISzGrrEVkkV7Pu\\\\\\\/iRBnXeVmgX5QS4irEBEC50Lo7rXqxjxh5YK40nXPEIHG8naPt1HgW3OUF9HvcAhlj+9QQfjAn4Hd\\\\\\\/jbVPMvXzy7NOvzD58vhI2On6rLc6YxkYxJk84J++ECNwclv3AfwluCiRMDx2QWqCKTc8kQGTyrFzmfBcmH0K0gxm5aKZNyZias+llW+wsd5ULmR+vVmf\\\\\\\/AnsLad4VLaV5mLM9cGd9F00\\\\\\\/hgJ0BPfMCrvevAnDW2uTYL6akhCTSkZZrYs3P7X3DAPIBrjlZwotu9wyGZqFmGdbcY9XtPWU3gz\\\\\\\/nAPDVn\\\\\\\/tmAWnJVxaqc7ihoO5ZxiJGi2OE9owL7Rx3nGBAgfUEo5wFcD0gle0f4ZcKFYOyNmG3itfOzND8tzciXKxImGNSEXoMUajDcrYtDW2LPXXBrMASnJXBffs+bnzNpsWD5BgS4NimtECrh6nWYdSRpE\\\\\\\/GtA9awT8ZzwYChfX0lTnyKfryvGpNmNDRgF5U+xyNsW+sz\\\\\\\/Xj1CnIcZzrc5RI5xxAvMq\\\\\\\/MQLK3pEJqeAhuuSSngsyQUAlM84sHtBuaWICyz1XXfgGMXoxCeqNvFNI+VwZGDrezV8x9N\\\\\\\/o75p+aH5eRyfgPvyP8v+um27QIHHTzZdDREo1DbL2PZNhzRDUxa5J1Cq+hfwywjsXfPMzSQTmRFtlhNa6nreGcAbaCvm+o8Ky6bCwYQCWiwPQRkrXDqpOnkw5oAnXfTftfg7WE\\\\\\\/zFdx0pJ3Y7bkdeLwN+upCOx\\\\\\\/Ow8efdUyWuo9Yy5Az0sgFsSlJH06xMI\\\\\\\/iFxZ\\\\\\\/Ezoi3iTckFKw2WHzMQTS6mrYsuqKMv5ZvPc6QhRHRhlRsBNCjJzmlmmeS23AXz5GaLWWeYKfu4sS\\\\\\\/DQ6mMLuhU+WV3fp5wK+i6gkc\\\\\\\/hKhKjNnTf+FWhvCiwzAiCB6QG\\\\\\\/\\\\\\\/3+SabdylJIi1Qgo0tvqYwdBRJpAtOA0zaaTcqfyN7\\\\\\\/1tGB33yDA6GtA7h4VU\\\\\\\/YaOxj\\\\\\\/Die\\\\\\\/xLPBqR\\\\\\\/p2mYpq6KgeFmfoJgWPnICfO+kdgNWXe\\\\\\\/yA5Ggc8TmeWVGdTjmQpFvgby0NltqCwjAeEy4kkzG2LROUqINAseoISUgjVRZmpqo2fjKTmod6W9JWl+QBnozhWaWTAA97VsFXjt9\\\\\\\/UrqEJC\\\\\\\/fJs4dph70rpzusypQOZ75\\\\\\\/INqlTqIG3p3tmR7+GYutKn+Dliuq04vDtxTGT0PYyOeNveZBf3fXWAAUKxPKBlOp50+eOB3m1qP8QqAupQnSY8AOsQtI75OiLj0XSTOQ2U5suq+ZshLRtEcF7vtFEUM\\\\\\\/CU8a9O\\\\\\\/nxzuUb8YMigJTzfFJxapwEpN9+w+1ahUivkwhvDpFCQ9InqrFf\\\\\\\/E7ddtWVsMeWNtQX+J4EcCkMGYaSyPMUMgnyLR7bp1R8m1L4c+Pnl8jIc3u2Tp5e7CwShNhyLR6Muu1qfuaxhnzQR9DryFjlc0WnKENRtS8lguhrRnHowmC4odI4qbAlVGlTDBRDDqRl4MUxgz8Y6nwG80c8qzttmC5yy+UD5dHSIZuqSI2jax25VDK6wm5nTI6L95CEOxPtoPBg2madoDf9PzIVlg\\\\\\\/KoK2sqMe7ESfYHN06AMcB5aSPCnSh6ylkJF7fhy4Y3jXQ6s3qzlZGAavS1n0zIu9miIbMz81MPIlgDXmYtOkaXN8raYbfRLhXW43AmbEIq1i5TeYyMQRaek\\\\\\\/gxhyeRFNRAgGL5qC9v0pQapO52sy1iMxU5BwtDdSsaiB7Nl8t8hfnJMn7yoKs3optPwVwUhNrqh75thxGvxq+hYGiNNcmW5DukPC\\\\\\\/ZS5HP3A73cY2KnsJbwd2tpYUFlde1JjnaIEn6RRB+j3wb4afU8aQv+4DonqXjgmaAzLSEvnY+CyaUvtTjkyTBnURU8a34XCh4qs9TxICWv\\\\\\\/RdhCtDau\\\\\\\/WSTbsMNwfivWzL4Da\\\\\\\/EZA8P+eKNYPdQmXQ5bmi1Lih\\\\\\\/iPMeY4L0PuqQ+I5KcKtSDNAJnoLDGx+G9Gfaxa9R6LNKfJNb3k81PhqQ4u+AFoJ6T9MBIgnta64wA6g9y+j8BWIFKPiEmH\\\\\\\/RjDkE8YyjkXU6oVFm3i6gS+\\\\\\\/BgAbbdNOrRc4wx8dKaZGwMHVAt0Oo4mP\\\\\\\/dVT9eflz1GdvFJbx\\\\\\\/JTmwWu4VZVTR+oajNxCs09n+MKPJMMJwnwg\\\\\\\/LCnRPedWvnboTXyotQWm3UmRKEi8lRXU4zPcdH8nRzf\\\\\\\/KYpkk4Xf3M72yQ84VrWpDrM4gNvssbAR2suigrDvTz\\\\\\\/PdNj32GtW2c9ZTPGIoI15AFDyDtYv+NaCsl7b59mhmlChTvdgYrCropv4XHgsVVBHivTk\\\\\\\/K7TwKGok1IxRyo0oFKc0DCFwO6pAsSOmTflw36EcZWCG2NajAaM1Z0WSR9L9sMGRbx5MXxKSFZ4SC4ujVndeE4E2w4\\\\\\\/btmcmFGt9W4mYLpb5KQHOBZqM1WMGlfuqtZcHirRZjoBGi6NCGs10CIPMnjBox2rL4Vlko7lXPwITGHaBcaESAIoMJkovqBRkqVZfTuJheWWFJ76pgvzEpzGDD0KiFWO9\\\\\\\/oJMuBC5NlyiYTnmyBQmbVCxVdlZxBKivxjupuAolZgcscExekju0cIspFaOqioZPihBdue+WYNetp9gr5IKL+GUgCZrpRC3I8aHbtRUc1eqlcVVVEjilk4w9U2xSFx\\\\\\\/UWjgMc0Gmzlns4Db3f5meAN2nCJ1y7R24X5H18NjQRoIhVm0MIdjC0\\\\\\\/CqrJAU\\\\\\\/M1T6as9rQpZOACKOUKsBxyraR4oGFMASpReComh0asO32FRMif38e1rI3Hg3y6WR2FpP1Zi8BN3siZVnwXNEPqI6FVlxDSJSHqCl4vRGEj7cS3kSbY\\\\\\\/GAAPAitx6WWEK+gfL72ldyuPHdoEu3P2WcWCSxog5BiZeqJfKuMni\\\\\\\/eiRKrhRTZ5OPGECRGvfXuRbbE9Vd6sIkA5VUe5AZuDRwMthMg14INQW2Q2QbWkSmO62oYjObTtc7TyitdiX2Ucv429AmjnCgR1ZQo4dY\\\\\\\/\\\\\\\/3CwtcddxatqvPotVh8Tp2KXJBx\\\\\\\/iHz9VI3\\\\\\\/+abEKcSRq1QXRGVwJb1rPKW5z9FVTFSw8RE9V7y8jHJnZdKH8\\\\\\\/fnG9ZWeAZA1HeeHVkGO6ZaAIUKHysvh2bXiY747at7Pdo6olLmKBARjNRAReQ8x\\\\\\\/to3R+F2\\\\\\\/bwjl2oT58n9WQj6xMPRpx2G6fO6xIYo8sQi3oTB\\\\\\\/mZWHnk0AIobOciEmK2v+aEbG51Xgdun+cu1dsyUToxS4t8L7xf9nxAqZSKPe1QHOUhu+va\\\\\\\/+i\\\\\\\/guFTz9oBQW9unlCisjo3CYDU5e6NoGTR9QW8dFc5c+IbexGksvx2JIUh\\\\\\\/NwFYACp7ygfeb+IDg9swF\\\\\\\/2q6\\\\\\\/yGf\\\\\\\/8dVgLERTBMf5nlNtt1VPI4WDGcgVyMD9+vi5UuTu86XF\\\\\\\/k4TxBrpbL7eeENCmWMkj7Vcv3\\\\\\\/LUknWOteHVXjBlQxAxyJ3CIvmSH0pMaKjZm19Xw0AGhU8hARGOx+bzjxueM8ozONtnOdKV4XIpzrxX2boEhsoF1CBIFc1h76PTdCXYKBSd6WvSG1Ir4d4neZRat3eRvfz7qmRAoWlz+AFc6oFObKEfLgNXAPfXHFdV2RxxuJXM9GI2J0fwxad9Gq6MssUOImkijkTqTGd7injB277i1xHunsj4zXV3+0pEhpJ46MeCYllUXGWsMqw1IghL5pRTIzg+KtiPharWiIP5HFe6+S0XRFvLuLTcLbdE+V5Px2h\\\\\\\/3YkkVSYKKZPA2LoCCIisAcAySAfhhIPfuHF6t21pv0F+vMMgmnfDqWxGXq9kgSJBcZKWa1BaaMIWQMNJS4sRMXG9ih6N9UQSXvfKthzI1rW4jemUopx5rr7HFWYvdVJXwNg+wH1ytiQfDoVnSZ7LLcS3Auj8ecPpJgU4MViMTcGmcntYuO4csjNwYTbnsoCWESVrwC3ErfKeUtoZT03z3Vpmpiqe+QHf1NyMPZnuckByLHNUwzKv\\\\\\\/RpEMbL+eXKNiNyKw\\\\\\\/OV1ZhKmwR1q0YgFueMp5VU+2LyE5yff6zGOnZZNs+accPZDbQQo5OqUiryifa15uDPPbyW+TykWnPYCRFdMArZL5ShDYsoWu85s+pxolsNLN4HRkrYWu4pW1w45T6LuCRz12H58Mi2edYss0mptY4dko6jLpLr3hT6mlFTGMvQEcNrR5SuqaBvF82p372RN3wNanz7GZx90Z9Phx6oYtE8AuRn3HUMKzlG+UuwCUpKYVW\\\\\\\/yYG2vsBEhANJ51f0cIhsqtA1iUEP8qIadfhzeTBJiskMv9OMORYK8dUCs6qNioshERf6vKjCVKPEFBgk8+MGwRmYnLqYr6zB0BfvEOwnC\\\\\\\/AH3awqZXQ82sI0j8D2LH2fk6tWweHJXYpgpzwbgU7oejyZ+RMo2FbRLGeREUelEg5q0watXwmvlN0lCTB3q28GBeaLqtrozYUNEp9fbRYirE+LZTBfyAS0FpmTVAhpuIj3LGvwZS2Z46H65RuKFeodWCJmF1cASNU2ZLa5tI+5Iky8QC5BDhOgibu1JIBHPIejtk4wjP1FPG6G+pe+z6Q3MwkdfPOk0XO7GEekcXh6bIkiJOPkwp6jtWvHH8dnCJqyfPLo802kahmmdQhs+YyeMcMURcaJUd8mw51bribrSF81JBoHDuBB1Ckp0t7ggAPj5bGiaha9ilsdDbHROCYIlfg9s7hcPgTkARyzzlnDyQ5JDnhcTosSv9KQAFK7r4EI4OgSXYC4r\\\\\\\/Xbjv0rGUv0GcRdJ\\\\\\\/kxYk0qf3Fmw8AiZ801Dl8z+DCJ7xVxnG7HaS8ycy1ncTgXazn4dBWSMVCw9ksdfebOZXqiXQWuplQ\\\\\\\/rQ1Lcx2KY\\\\\\\/g4z8jp3AKByCqbgk6by7In9NFaH8PiSRrDBKXT8dXJoPqFV1r9Cp8srZ\\\\\\\/ZvCEP7SpC02yPSZazjTROXrBI0Ag6LmQxHou0r\\\\\\\/dwld+hLrFibEzR8h4yv87poL4wJ1\\\\\\\/g3NY4BhuV+s3dXAMR9KiQycG7AyqLfmLN2PpbKFhJzjEM4CSZ9eFIucELE5HiD842UBNGAFQP5lVYqnzJT\\\\\\\/qnnkuVgtgmjAdqWY1J4Q+Fq427E+1lziFEZ7YaHffMqWJ3G0z0pbRQg3RoWZ6f2a6jKL5\\\\\\\/SRxGjRInjPboqt51tPKjRHEQA3zDJLZpbG+M7T1XDBqrWOiREnvepoBSA3aeK40ofG2iiDxm7TXsRrtO79oZChjEpP02m29Lz+mBNmVWrXq7DXzv4vD6HYJOWE3kK22aEBsoO1EBogOE294a3vkReC2rTfdyfpH83F3WtxqqG4AcOZ44+2MwEbMhimvJmAxaRzes2set\\\\\\\/pz1TdXl3td97V9H\\\\\\\/bayGMjmMiRGWS98ncrZ4u1zfjX6QFQjKeQ8rQJJwa\\\\\\\/9s2tuzkB1eA4mkCpnhOEqEcCld3tGbK6GHoxeW7Xk0bUGMgBponfeNPWUmopIhnYRbyk5yj7KLXO3K1gDBbX1bSvNCft15g0xxnwbFzlFkxHkRRdHCU37BXI92UEAJ4Bn+6y7E36FVzNwFNLMyYolRIoMuQfBFhg\\\\\\\/4igZeTGKyLtuF4t4qnFix8OQ5WaWukBnurMRqImciT11aozmDWTIQMALWIvEw4A\\\\\\\/DYL08ND8+B1YrHjK2k+l4JqMtH8LYxjX4CsHUWwFqNLeMYKEBz0NuY9mdeFtT7I6zhMIcfPv9U+KKGA82\\\\\\\/MXOhJBndltUhDHXUnbYIePrBb4fXHyrepw7g1Iuty34uNv98Y9PXRCxvJR9NH4Je3SZhI3oOiQ2MRNzJ2cDLFGUDz\\\\\\\/RM0KBMnygJ4ZY99A2v1KQlPEHbRbPQJoD8zEtUnX2cdUWL+y16j9yoOpO37dKb2JUfh0RtkDJcN9V4QSkVrz3CrNkWZI5x80RdF2+nhFC\\\\\\\/g8mb6WGdj5jztXKm5Nqdg5aIyY+FpAqd99cDVQDLKR2SfhXpcbfA4JJQ+Be+REls7pcYq7rP\\\\\\\/9cUV7brPI7Fgp99bBUtgZtEsL36ywHlpGOtoIs76dc3CsL5rSE+IMEo7FG9ahexYpJJwR6Hd4392uhzNpto\\\\\\\/BeYhOUHahfbVtaByexccwqTcIG9XQkiIrIjGBHWG9cJHLxcbkirwtu52JYINoYB2NXj1wFJqnzJZsV8ppBfVY9f1Maio7a7yn2YYiWWrWF4hcr9f8WPu0RjD97pNtBRTyiuf1YwI1EWRdXU\\\\\\\/rBPgIipzIJ7EL\\\\\\\/eMnNzAg77jrCyOZ5QfjqNC80QAVAUM+63bL4y1ONbh0RociaADNJybAVufbpVplf2bqPFxxzEuW6wYh4\\\\\\\/\\\\\\\/xZQz6kpqtN1PYPiqRRDW7K3gl+tH7yxrCk40Sr9T\\\\\\\/RE\\\\\\\/DtQcyvYDGK4ZqY+v0kl9mq\\\\\\\/85HVDqSdr5u1D\\\\\\\/5WGnA6M4exzSF9mnMl54rKJa4x4hfzc1qWRYPsMkZhVSS\\\\\\\/4SG8J5L\\\\\\\/l62\\\\\\\/eSq+1RzVCcmyDLx7IsZ2hNYjTgTFnjVknQXwDRYgo9pv\\\\\\\/dGcwV6EvukLUllRaWk9LDQGqtAwGU9+1bujFn6xcR2R3Vta9S1Hat5ERwG3ELuuY6Z5z2WdxsjKWCWMlogWMyzPodCf8\\\\\\\/83i8ezds8OqqIbSXCRz9dy7HFjfo2EaUG3m7RrGyacemNmohOBy+HIl+RjGCpYkINrJ4VW2YZf7uc5YdGbQfKQt+Sa\\\\\\\/TnQkQksD8svCv3\\\\\\\/H2TnYv4PD+J8a88tVGOI46Jilc6ZeaWcvuvD1vKhhpnESYW9flM7xTH7KEP11QbTqf+PxL0bex5tGdiPt9cO9fudV6xelvFHJY\\\\\\\/ToAeY6qPTHggasWVxhd7UK9sd4sVyIlPW27BPcpY8C+\\\\\\\/DqxKF3rK6SW1JXDmpLorJgRaNcz1LrT9wm1vHSTlHe+XuiGzoO+5G71OX1X0FWQ4ahzwmZ7gJqoogQkFgWOWGgKqbNIzeqdJWnO3rGDhK4Vk7xeh+rUd4HgOUAtAmfyFEmNQfrvojuWd6Mw+RTWgdKVg7UQh25SeI4pDFxN+UHAn7PUASGkjbFa5FQdtm9XFQ2r9Ilh7MbBxiDUvWpNhgXqGlVEpWg4xSxBDDrGLKVrUu3CtNYAQQn972qdEs+Fh4jd81hIeZPzFH8kjo\\\\\\\/Hju6UnSJPrpgwpETCwftd3t82HdkktqO7Xfe7\\\\\\\/WSE7mmZ1LZmKu2Wf99YoO2IpKPMY6sm\\\\\\\/jCri\\\\\\\/qaA6hhdVc8KLYUizGheO8sVGtCtb1HG\\\\\\\/1ikC0d3Xi8pXJvFSqWfw908MG6nCZgjZwFPzGSMk\\\\\\\/p\\\\\\\/F4eHzHZ7BWEU2sjlc1AINlgj8KrY6nNQ9E6Ttl9QcAqGZaw15vWB4puyIiwcEY\\\\\\\/uf6iGM1ZE9srbu4+rlXkX8pmsuM75CQ2L3Ff4KpBzePQCI3z7GFUDRFwhm\\\\\\\/nuLjdesndwE\\\\\\\/1EYBY2mDBIkpcALsAR3sd1lKY2EPLcU43fwxQHoM8lQLdg4agsyXrJ7AbRdixOrAqTYGChEnxdU8N9DPgNlWeRcKFw9rtGIkZUc6Mt9KQuGzYDlcuElKMd7s6ig7HYg\\\\\\\/qFPLwxsDzs7nbOWxgnuJRi09bapXuP66Xn9kcNctSYNUG+3zLVQbBr8S8qXzWOxIlOZGP6c2BH2reFvBSUtfkermCRkn9R06i6mF50NhEKMXrivPnD6+o8LE2MQkqJUP61dHjmou0B4C\\\\\\\/xo6DEOofivVC3X+rF2K5tADq\\\\\\\/Ce9wX\\\\\\\/wRWlfOHWrdjd63Tn5CRQnYsvbCV73YmE75hSGpeMoDZslibkHvQVrZZBJEPuTUA8dNrEQkZ67GaVod7XRSUO+47lyWkIkWcvkyDsvb5TNi7NgXzfPzZcfGSHIsDIMs1py9JBWF2JMoT0nTu5Sid3InkU585T\\\\\\\/15uHc4VkRUe9BXwrubfFRxy+vuI6\\\\\\\/4WyTtfvRzqxM1DVwXeyqctUYyjzAiJ2DFK\\\\\\\/ObDtb3qXQptnxRgmjISO1COZG+gwo5BCNFTrS6GdMDd2JHEZVmNBN6GvjQHDpOukp4RnRJ3tPKiXFZaIkht64KutjhFfqEXoiY4krGzQn8ZxPF0zmA+NDGgqucN9mAgyeA1ICHsVaEYTMgwMLLSItfAcVxteDRvoA62IkZwtxNEUWv0oFR\\\\\\\/ePCDt1bngFkEA9VYqOZGzFoeVXrA4Hof91LEBGt+ahWIbsiafJJUIceMrSDHkrY\\\\\\\/xTx+25Xb\\\\\\\/bYlxeQ2AtcYsJtbKqvlz8M02fC1kf31VbDwUZ9vkCD2XEvlvBpOBraHFdII49322wzhe+vL4KfNsV9hBbnChpE\\\\\\\/MEr+zCmVK4UWPK8Po4DmmF9JR5Nmdujo\\\\\\\/jFMzQ8Tx\\\\\\\/XoBb8xWCEhgtt4qgaWunz5BG14PS5CZ8BDUA8GXUnAwdmcYbXrL6lCFJQrN8Mx0hp0lExLkLu7lggvXvLMve7IIAZWjztaBOy\\\\\\\/I4H\\\\\\\/DGp9FEDSna4iDa7F7kS9ViOJb2+VAd2qLQW0WCmllvqKLEl16WelE4+297gMNUX7fOOBYEIKRxSLKHAGrZyu1XmOxqZwQk8\\\\\\\/ZL3utTh96u2MSSpMT9NO7zM1wio6lQ7zmRdkVJDG2bGwu5LWL7W9HDBbShJZBXblZM2zH\\\\\\\/\\\\\\\/EFnSVAC65YSMTMPm9vlQSIWlJjvRxW7CA7BbW0wM4EP\\\\\\\/qcx5iHgnjbCqwpji\\\\\\\/F\\\\\\\/nHpaLVCIjoFeRSLNQGCv3bvffVMPAhLe+0tYFrd3kyjazBuTwdj6xJr8NDCO5KfnGOzTzpdWaZ7DL6buMBGShZXsaoJdp3IWGERPFx7wx32Mm3OBwOoUrE5Wj7vk2F3O\\\\\\\/QxFMQ\\\\\\\/MoT6HBTVQ8Z41mHy9pKcRWRgQ8FgCsF6x78IEoZ18PD4NBXZvITrWplAArvgmnOmYk83v2dZE\\\\\\\/UinQYE2mv3kH4o4cMCwDPZGWcq8RQq+62\\\\\\\/Wovk8tOpEGUE6sGy1bjFenzN40u9NNCDbSdPzPE+dUnS7wxF25jF87\\\\\\\/RZoTjZWLSsZIOiaYlOYk0N\\\\\\\/R50l1FYY5AcO7IKTmZSD5cKBH1rgIWAJZmV6VNZrx7W+TaK3EHXPyx53XgBaHvTQKq1dchpWtlRXNHfoFzwZOW8ArXL2hOCPo+0oIJpONqvcPUMEYCcZbWjXX5dDsS+iR6pGts5HOJsldK5uw5vTTNa7cEWdgl1j7WBtxtWzzVj\\\\\\\/v5Y\\\\\\\/jZF2bxj5XujRo9GufEJdcklvOv0vByo+mdT7K0n\\\\\\\/KLaLDs93JI5Ht7ZGdnOHVQCllOroacFZC5\\\\\\\/dXo8\\\\\\\/XpXEtnQkhclQgbZrLvlfQCNqSly9+RLBlbI0VZ7PA\\\\\\\/FHHHBU4WROD5AXCVsbX27sbBP5zAekB3NT3zJBp+U5nlDfTENR9c7HlqtTiDUvGM0+KfBzat8SaaNECYSwITlMr4DB8uOq\\\\\\\/FkPvIMdW9AGTQnj+ewYMxbzIar2RX9epWaQ4UxHB9\\\\\\\/bPSswgU+g0uFys6Dzxii17jPGBYO6wZPURqEoittBxLkSUYTLr7aZuKBoJPu62q0YPnCgxsmR+xX\\\\\\\/DzIldu15fIYnMujdQBLeQoZd9+XVbeDJ0H6iunQtpblgTG\\\\\\\/lXxGvIgixxQHL6toy2v1psLOBfiA\\\\\\\/wgILGMLMqXAgpog+RKgLDg21ZM6fY+6xB56r3DlF3OyMfv+C\\\\\\\/dOtZXAjQv6W5jBPswDuS+nND8mLZK8Zei\\\\\\\/I28fcQOR0rIPw7MCAeA+RRZfRbIwvc8fsCncAQ+wEyy6LQRqZB23Y19eJYNtDfFaxR7OiFZZIPqqXGeZ5dgP+imktCnRhTBTGuUm6+l3vvdKNBPcD1gVjq+6wUVDBsMnuigk8aqPq6vPLL4kS9ozQcgO\\\\\\\/\\\\\\\/Nslv\\\\\\\/oWx57ZYAGAmC\\\\\\\/itliGPbKyLzcvSRkYmrNWPheTYoxgBudbIrqkG7QVDhhRwiLEY\\\\\\\/WEmTpZG4SJYLiaswIQv1Mm0oapFIcFnjMQS3JnYKNdgKRho9iihk5Vix3HeI\\\\\\\/Dv0XvvbkRgfvdCFaoi60qP6x3TaT17ufxbJbJnW8t6Nam9HOfeAzuXaKS1Kvd3SeE+tQH\\\\\\\/Urhgcsn20a8KDZ\\\\\\\/rcU2LQNKsuRz0MSHoeVa9T0ycPCpieLgKwilvapknvRyzrDb7B8h6TuVIuG4uzQNzvUoo8QtUH4KGQvpKJ++QozVgq9epdVX3e3g+X4hRSJUC67ZgV+69j7rSUKKqVAEt8D9kPTlLYSBvh1684untnKIVRGWLhV8u5x97Ap3QKFGbuRCW8MBhpAOvHWWYm3K0a05dq6fQfLcRqCDy65XwTUDSMXHjXlb1upm30U0SnI0Ogaspiw8zPrXlt1VLFc8VRbXoV1xOyekCOVfA3nAvVavTrzQX6LPTiCi72RgId8Tu8DH9Le7H43XnN3QpoZKoDJtos5WNjUprK+vqNzaGEaBpQHOXbLl3SU\\\\\\\/p0VKZ52tFuCZL+n2qqPdHUaBqZw5rFPonSKKYuXh19JF6KtJgYj2v1wA3XTpeGom8KpPqteJ0K62GXhTybwhIcr+Fx5a6ZsbfYtRKnr6MBiq6Z5oi2+TO+npsqAKRcme30yuVaKs8Q6sSI0SvNGne6BLWjzCNAm+a8IeBTUyHRt5wGOgZrmZBffn9boSyL+Ob9Lk7QxCrkmjGzmxcNHUuYl6Y8Cmv3por5eoEcF6lm2VE+2MnPTYPifhZAwAjfheqdCuyrx01nh3H5gxP1HBVCAiR6tD7rT4AxS9vawZfwMr\\\\\\\/W6+oMQTU5acd6nuHUWLPanI+nd4\\\\\\\/ZbsqzQuw2tZl2dsmSNyckEkN0\\\\\\\/P1FbSfg1ddCdLiSjm04QgiYgSAFyO9OtShQtk+vtxSXrAXfA2I6+f2xI76q0\\\\\\\/fY\\\\\\\/Cqx2JD9ndqxx+MZtv\\\\\\\/hzctX2hIUGAZlwNyg+OmfDlgXquR0c7V8DigN0kAeZUdfkgFNfYs+x6ljSwDg4Yyh0gWAoeaP2g5tgWwmh5LZzkhEnJCAx82+VSOAoLaxvWKnB3MPgh3hS\\\\\\\/g9BYn\\\\\\\/7qsHjOHxQhYMCjhXnpmWmO2Z3lHU+GeToQD4MhS8fHqN5NiKQJudTPnu42EHGizx+uZBHYghmVTHlwK0WgQ\\\\\\\/29PouW8MUSuXkL3D1m+LGR6A8Dzo+6bxpMfc7GBBZBhMTfXmYLBTShhwKopc4nk+R1NKGWNHOCqrynDHgFyvYSYuW+XSJm3L4Ta98H2WXzRcDMAE+aBXVA6hg48wVm3QysFZzYmNQtmJ+Cbp+4FqIJBZ3puzBHFf7C+\\\\\\\/2iHF3IayyVvPOgKK1OHpoeQcAksmyq2guglampq4VwoO1FmlxFTXfo7IbWGarLM1SuWATO51WZcp4TnkxK5D4x\\\\\\\/\\\\\\\/4x9WOsOeXGSOsvj2XxpAfY8oLCt8ijVzOuPB6aQLqmnz7I3SHdMNcZciPFAY1aBn8x32EnzdNrSrmfmNrDy0MMR2XzLyN5bpnAoxvAsHrlEqfv0jS8i3CPCxji4ic4vfEcIkNOOMWPiT6a260PyeOml0O0oIySBz4izrfEwt78NLtoLd2BC05IEJuZUsohreAe+HsRiyBS8eCfUjMUMF3NVTRy9X8OKU1PReze+i7mmAZmtKixJjcM7bJylGBq7Vz2kX0WtF20nYWa8VEAG9xTwhlhjh2nXb9rUT4FoD6Me616I2d0wfc1cQEQ8EIthbSgrFbLFEXoalQxlzOcY5rKLaozurXhgv78p3AxGuv06Zu+3WYG7ay4iEgv49Pud2H+oC52azd4jesUJS6Ov6oRaAz1F9aOhXWSYGlyIlaJ6mRfn3\\\\\\\/fPjlrQX7QxZdCRYIXT2MRvUWPMgB7d3DZocvij8OwjGau5u3e1tQhfsbGhfTMe3pbMxeHPMY3rQAqKr+PyzkMdM89sjSfo4H0140yy\\\\\\\/4uxDae6IoQw3v\\\\\\\/JyxKZfZRE8p+c9dvRp9Nt93NwdykSwsWCQiKdE7KWSHKXYG5QTGr7GdN82aOOzUCvEU04bM77BzY5rS0nMLQQJH9qM7a\\\\\\\/nn5IdofnhjERfQxHWsoUx9hJRV3Zk54B7aefksJr1o2BgIjE\\\\\\\/x\\\\\\\/mfW2u7nM1nAYBnb2S8lqNPKs5oSYCiWhW6kFTozyjrtKDOuuMWD\\\\\\\/\\\\\\\/y2C5L3AQQCJ+bXyk1yxI\\\\\\\/iT9zy8skn1fwLfMO4LUeSu45bv2SC\\\\\\\/QtNRkwUzrh74HElyclYp+QUNmS\\\\\\\/kAsZaQxkDfZD8P25rqykNRT5FOY3pNN5qw1JnpuAFTqdcTmGbGXgVnaJ8TiYeoVyV0cC0FmwXgeEX8owp69Ekom+nkhczR4ciEBpqlgy82MPTWQkaYnrkH+CFOI4URhy1FFs4qlFmkCaMj5mankMY+w8kNrfh9FXKPXp4KH4dXcDtLN2L8QQv5qbirjW80DCAoCxxyLeDeLosGEaV0VbltzIuHCmN4NTp66TxKDiwPyo9AImcxCbnzBRggJ7fwX7KbvoiTTZqbbsZhltqoxO1TqlM2\\\\\\\/at4gPosoy11NY9avxEEbj25zQCunebPfeJowfjAtV+WNSryDtmUKRbs+hSRsPEUZxuJZZR2\\\\\\\/V7g\\\\\\\/3XC2moIsx1gTEy8h45\\\\\\\/8AA2Gyb3ywZi5xaZR968VVZ65gGJN+L9GCqScHnM8obAfHgr9i6+lAsfMcxuWNNoyBtpzd7U5A\\\\\\\/MtPsPl4\\\\\\\/+p+mA2AfNzdXVQk+2dGOttgM0QF1T+yqUJyvCMdX74UmuPkxmsrji\\\\\\\/RZtE+orYGahw49LsSBjj7yWpFaEfUOSJteiFU4hXspR2IRjpk6kNWzVueh4D60ubwnq8M9SGZysvbM1C5JXxYb9oRzW5dVEOmAGctqgeeB+pe5aiNk2\\\\\\\/2v\\\\\\\/p0s7HjJf1Q+bzgml6BG8b6+zl4k2OzMLqLMkADjeYQ51gDCrMVtzfFM+QA+AxvXGRTqxUW\\\\\\\/vc3aHnufIk6HDHjWMUOPwVg6M9EvHN\\\\\\\/7aGcTWSyZqtZTR2djGK6Z35LrHocVSIuZHbmdxPzt\\\\\\\/QkmbUlYn3kAwhJ8GjFIhevvwe5mlEAGdJ6BuSUYgJeaBiL9GcJs\\\\\\\/0xYxMBruNhBvKmOO5DqUhEOzwxmoDVt5avZrgz6sCZjSjEZS8SolnvZrtpirK\\\\\\\/ROaQoeUPkIW99IDT4MUQxHoEx+E9NqmTp9TRNNICo80ujkYiBmLz9DfVYBn1B6BJHNdijPD\\\\\\\/Y5rzYVKOo6+j5hqnJhcb\\\\\\\/TbAxkEFvDGG2PJ4ON1iPU0JjHqp92dMsfiPbLNogMsWusc7PIAp\\\\\\\/oTLhlfDG6XibKJOsVN5Mo8y7MI1dpJQifTVJwy3rgifjQUS8AOioNDLZkKtHxixpN9zSouoFxUbCVEf6MJIwL3J\\\\\\\/ti6LupX8htzvzVkT9WlWUg1iXr6nJFNd6sI6LSTEJFvBzQB7kl9NAksbn6bnZsha7hl+wIH6anGpBYyN1+O9DIrUA6XfBcQiUtU86Fozz9dVftyRG\\\\\\\/n1Sc5vf16\\\\\\\/B3JHPTeKCgpcP\\\\\\\/Ys2h9ZxDWDpm51TGwf2UO+7Fc0N5YC7RiLUu7qkIBS0DawNV4Jn12zQjaSP3avuBFDweGJ\\\\\\\/bpAcFIPqUSJMFNAFldQVQyj3IdQqgDdjnIznbiWTqleXsPsB6raqWrf2mpomi7kvCQoXSBjAfbi1koHCZR3zEzj4hZ4gqU+k7U1VjvbtAYTwLZLqxgjlGepoIkeSMWyH+teYkgUpoct3\\\\\\\/T0C2Y1Q+pJvvijHiSe1UaAPef+gDcYUmuEntmqS\\\\\\\/b9Vk8j7VNwXb8RFwnhJ8SqaLVxa\\\\\\\/jJG18NMQggPpeMpS192v6UaL+9DCuXVSPrT9JL9zIiAmawdJWo7zwSnVV6\\\\\\\/iKoXFdixuTC9pdr5M9WRZAucvOmxBn8ldOOtud1k4cNQyilzjo3GAmjbuieW9cWjA3Ccm3ffpQjGKmBCAbAD\\\\\\\/Lej30qBh5fANvs1UGoDbm7SG2a07wpuSCME5kOmoOfJAZ2+A16TDZMnzqEsXVYJvpQnJCORHxpe81m6yCXJUi6Nve0\\\\\\\/WHUIyZLUEHGbM7mB33gPZ7z6CBy3o4wlD5wSRr6n8p171EomFcIMLnmJtgfc38ZCpxtWNe9turOWoPmNpPhuoV5BmqvhFamOYawaHUl0j2PZujbqi1godJWd7Sq1XDpErZ09Kv6QARr4Ppn0PmB\\\\\\\/12mBhkUhQ64Ytogv1QdEWPImoVLxzLhwAAHSGakgX+edy3Fl7FQNW56MPvOt1jikIRIThKdaBiA3BdZUqacpFHHK1xtaQM+QFg4vb7dJvAmsglPY9UfaSEAtAv6KPWLSSqNcZhrWjENH\\\\\\\/c16mBNpua0AjZ1jEmNlVKcfs4f1KfGVhei7yMRtIgKI4kgDDlPDZ5SSY118Vi3rFQeoWfic+d5\\\\\\\/CbBXy72Z5SZcDa3FLDJoJPdAk5DPvUX27RqWzuptQrd0j9lgnbixnzvpb6uEupX88L1eycWrjZdlI\\\\\\\/E9IBQIDEasLkZuim9\\\\\\\/2OPiQLoaBM9oESybjejRsghb0OhA4rZpnxtCT0TAsFsu9S1vLfvA5pbqq\\\\\\\/QcGQOwxjQAZM++TsTR62Cwio6F4\\\\\\\/6DEjoFHZDAARi+Sw1y13C0BYL\\\\\\\/P27H\\\\\\\/STNAeaKrjMy\\\\\\\/AIJz1DC4meOsWPfPJ64WszwaabPrAv7WpnmzFl7k7Vh+1lw\\\\\\\/6bOeCR7kcS1rXhKExkoyx4hkTPyKBuXBqYkwdd50tm1rjTPwfV0E7S27pbCnINFjbGIQuna36bapz1qIKPvISXkpynjJXolwXWVkGZcZwaJlktpkeU0OvsbUAUG0k8oBsq7k+h2KtHncuquCAQme\\\\\\\/lrqS2iLmBosTd8REM+vJjyrPl8GXsjCeKJI15imp6dcxOA8P7JjB6PyCHAw6wZreF815dau3TcRyNZxqNL14IM+mTrklPk+zLBoYisG4V9PTLlU07s3s+dgFypeUQG80V2OAkGopjlLDgfXJogdAe1s6PDbUTHoJ+5V8kX2sMa2otek8RDe0aLQTueGOtWiz\\\\\\\/\\\\\\\/u7MnFlNV17ejKdZeV2Y6SOLG6FsfsrDQsmmn\\\\\\\/wlQuAfXD4SnAY4w9xcFIkitbgcEJESk3w5j5IftOZZhAzdaNVp28cpZNMc\\\\\\\/40E2CfpVWF6NAxkXhxWkHV38KXHwJfWN4\\\\\\\/afKSqSqh28aG73JhAAS6GQGaVtF4KRu\\\\\\\/2tPFkDiaN65Bloe6ReuVeHpcpnX6A2zrelPKW7KPzE20MG\\\\\\\/pGbAxIPAwRVm+lCWkvaBwy+vZ1VkKgunAFE2HyjmD+SJLD4XkGROysLq+GeNYYp3wjRVDRk0k0YnXLNrZ55RYJQ7TvV8+u3ZZuKPyUtebTnaVmHpYVJg03QSyu56U2lzWT\\\\\\\/DF2AH7hzzPq8o9fqijSKcoucBh4skX36Z2VvnZi+i8IUFdNuqI6NUZd4crl\\\\\\\/uibARQioRl+JCxi2iYwyH0Bbh3LBscZzXS7Gn\\\\\\\/2A==\\\u0022,\\\u0022iv\\\u0022:\\\u0022c9TKoyzCe0nxmBegwgSMQw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220f2b5c2cd40b045f93cf2ac2055324f571ca84c0cce2f329239c0fca026a9e94\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022vwo6vlx0mEneY8jnmozFVWfWA0IxGACJem44Sx6wEH+cx7XxZEoe+8VtciE2z2si2fVS0tA3\\\\\\\/3fEzzPSXE3nRqnOpOF8Kw2PfYwiVchi59Gc7p5yRya1vbnmrZmRxwMTBO2XBybXGhLhJfAf9gjTnp0gHjehYctHVDV486agD3r32YE\\\\\\\/dOx+uu31rm9qbczA28pNZsBRglyUp\\\\\\\/ynnQjEVGR1jetWoJs2cI5gD9hnDVBc2TFJgvdXF8TDuVwSwkLSeAsNTP1r\\\\\\\/ns4jdJuBvPhIo74dEMLzqzlDv+o\\\\\\\/nWerW0JDkKoDg62roHUyz1c8wZV25ih18rxBtA38zHreKSH9SnazZuRODWr3DoiHk3gCUayR3f1ElUMZ1RjXVjkhZx1IMklMOcIDyhE1HtBaUFPYd2Cz8KzaZ8tfoT4tccW0ZD9MvmVXF8be9eIkaNpfqpI8ugtFnbhizY9vVUZww9SsUqbe5Ry1M4fYUXIrNKxR944t1T7iY9JxGvJ6QRyzuyjeatCvPinHEomAkWX6uGQpLy6evQx0xAwUzYK+bsQnbYuG99SKBRwCt9tO3Yg04hVjloOhxelpg0VAMMFv+VGji2ykOUCJQDQkU5PUUeg+Xbc1kAKUGcBkr\\\\\\\/pvKdi\\\\\\\/C6y28ocVrvI3b8pKLJdY5TGGEIhAHRvTM8BR2KiAnC2O0QGI\\\\\\\/Lw3z52KpSJUOBHaXKtpYlGsG1fw4LIBuXp+CwuBYvZ\\\\\\\/TGsuWBPsfA7Xf1HYW\\\\\\\/d2QOnuS0kMwTIVfzJb9weZfLWgPbiU3aUzoicdp\\\\\\\/Qi+ZBvrZ9liKxtubKW0koWyJRNhwmZ7wyViyGlSGzWcPzE3+2YECMr+9Zvq2hCwtCnhrxSfRNJZz91FvwAae2ufcnG2L\\\\\\\/3aI0smUTqTbdx573m\\\\\\\/WSpwjpc0HkIHGc+u6myT\\\\\\\/GbLyQ0Tp4hCWkRRYMjYopE7ky13Uaz\\\\\\\/sfZpcvmF28GkQegZheenFVSiPXuqkM4vzWg\\\\\\\/rEczlqbECJ8ReyAo+6TO1pf0zZ0aZaXg8TsP1CpcxNeCxBWIkXTJeevGUFS\\\\\\\/kbzlYrNJGSyQiM055RBrhn0dmR\\\\\\\/04XkCpunJdQNEdQ2TuNgYP4q1tERoJQx0XTFJw9m49H\\\\\\\/\\\\\\\/I1H+TE0aIHuBCquO1OA8YI5GqlH4J9O\\\\\\\/OyWhYXjxokuzrGTFUBFAxjG1LCNoRx5kbmE\\\\\\\/QAcXapciMHbqALBx\\\\\\\/EN1eyfXuYl\\\\\\\/T4H0lmUWFPR45r3ggKvyAn9xrSB8FNvWTvu35T7GmRs4IUUMT988xQmuhRPPTuevBWi5iNTL\\\\\\\/EdBXGa8US\\\\\\\/dv3Oct9KMAGKWJcxaq0Fx+PrzrYG\\\\\\\/KELCB8Jm7o2fxAO0tjUbk30eXsqLx1kWVrcHYy3n0pPLtnEOfyVOG8V4FWZP+5T2cmWx9xcanOBjBly6coobxdJcYY8ApHYzN4VCvBjIkBCnooas3egPV3f\\\\\\\/1PDplG6q5I206iwcPOnH\\\\\\\/g3Lts4Y5ps1BeJI+Gy8ZZrN3W0uEUP1jBTtpz9rqjJgQAWC7LIn9Q2X5tyTh8ir1yOlvyFOHrBBviHzdwPadkaJ6yuIyTPIyR6UD3BlO4zWTN6GCjt1O+J4td\\\\\\\/1lz6leDh2P5cvNPt0lspJEU\\\\\\\/f4BxgunLUQqxagtDpbekaqqqW7J4VRdkiLxJ\\\\\\\/5tL0qYr0YDDIidE7ANURxs3jgAaT319LAZRfIVfSBkMe+tu9U9POt9CgJriiFG6Hx+Kc441V8tJ0EIpnMxnLaKtjBthw8CUF2uPQiUD\\\\\\\/M2kgNAUACbu0p50GE2WeFQC3RnspetPS14gQEuQu3p6K2XdKwOdTnjsLqiWnoUtT0ap3OChZo8bj09gHE5su6\\\\\\\/AgPNI3tUAA2Lnct8tWFnPLaLwtF2z7TGsAqUJpClHyp0fhrZgt\\\\\\\/M1FqurVCAePM5Xt26V1G0VCF6dcdqCow+Qv79pf7xTuw2piO0+ukx+foeb\\\\\\\/+HIYEVVxJiyPGGtCfUDCVDmMFZWlV2oalBzkTuU+zZsoQOn9oiFjpNWw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022OAhWVe+Uu5csD6ezcK88ow==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002295544eced5c3902536a02e8c732400350c38ef12f4a054826037571600b3141e\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460564 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/2/543" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Y0eT4ZQuzVyRF7TZ8aSvUNZsLJ7DIt1DoStuV7yRSdQ" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/2/543" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-mW8aYBCDjDQRA9eX7s2_zre5Fw6nsaXkdEvw5dcmsl8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/2/533" title="Expression Patterns of Atlantic Sturgeon (Acipenser oxyrinchus) During Embryonic Development" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/2/557" title="Fine Mapping of Ur-3, a Historically Important Rust Resistance Locus in Common Bean" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460564" id="node460564" data-pisa="ggg;7/2/543" data-pisa-master="ggg;g3.116.038026" data-apath="/ggg/7/2/543.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A SNP Based Linkage Map of the Arctic Charr (<em>Salvelinus alpinus</em>) Genome Provides Insights into the Diploidization Process After Whole Genome Duplication</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Cameron M. Nugent</span>, <span class="highwire-citation-author" data-delta="1">Anne A. Easton</span>, <span class="highwire-citation-author" data-delta="2">Joseph D. Norman</span>, <span class="highwire-citation-author" data-delta="3">Moira M. Ferguson</span> and <span class="highwire-citation-author" data-delta="4">Roy G. Danzmann</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">543-556;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.038026 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460564" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Cameron M. Nugent </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Integrative Biology, University of Guelph, Ontario N1G 2W1, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cameron%2BM.%2BNugent%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nugent%20CM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACameron%2BM.%2BNugent%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:nugentc@uoguelph.ca" class="" data-icon-position="" data-hide-link-title="0">nugentc@uoguelph.ca</a>
<a href="mailto:rdanzman@uoguelph.ca" class="" data-icon-position="" data-hide-link-title="0">rdanzman@uoguelph.ca</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Anne A. Easton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Integrative Biology, University of Guelph, Ontario N1G 2W1, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anne%2BA.%2BEaston%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Easton%20AA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnne%2BA.%2BEaston%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Joseph D. Norman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Integrative Biology, University of Guelph, Ontario N1G 2W1, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BD.%2BNorman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Norman%20JD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoseph%2BD.%2BNorman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Moira M. Ferguson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Integrative Biology, University of Guelph, Ontario N1G 2W1, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Moira%2BM.%2BFerguson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ferguson%20MM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMoira%2BM.%2BFerguson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Roy G. Danzmann </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Integrative Biology, University of Guelph, Ontario N1G 2W1, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Roy%2BG.%2BDanzmann%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Danzmann%20RG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARoy%2BG.%2BDanzmann%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:nugentc@uoguelph.ca" class="" data-icon-position="" data-hide-link-title="0">nugentc@uoguelph.ca</a>
<a href="mailto:rdanzman@uoguelph.ca" class="" data-icon-position="" data-hide-link-title="0">rdanzman@uoguelph.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/2/543" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460564" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460564/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/2/543.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460564" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460564/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/2/543.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460564" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460564/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/2/543.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460564" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460564/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/2/543.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460564" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460564/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Diploidization, which follows whole genome duplication events, does not occur evenly across the genome. In salmonid fishes, certain pairs of homeologous chromosomes preserve tetraploid loci in higher frequencies toward the telomeres due to residual tetrasomic inheritance. Research suggests this occurs only in homeologous pairs where one chromosome arm has undergone a fusion event. We present a linkage map for Arctic charr (<em>Salvelinus alpinus</em>), a salmonid species with relatively fewer chromosome fusions. Genotype by sequencing identified 19,418 SNPs, and a linkage map consisting of 4508 markers was constructed from a subset of high quality SNPs and microsatellite markers that were used to anchor the new map to previous versions. Both male- and female-specific linkage maps contained the expected number of 39 linkage groups. The chromosome type associated with each linkage group was determined, and 10 stable metacentric chromosomes were identified, along with a chromosome polymorphism involving the sex chromosome AC04. Two instances of a weak form of pseudolinkage were detected in the telomeric regions of homeologous chromosome arms in both female and male linkage maps. Chromosome arm homologies within the Atlantic salmon (<em>Salmo salar</em>) and rainbow trout (<em>Oncorhynchus mykiss</em>) genomes were determined. Paralogous sequence variants (PSVs) were identified, and their comparative BLASTn hit locations showed that duplicate markers exist in higher numbers on seven pairs of homeologous arms, previously identified as preserving tetrasomy in salmonid species. Homeologous arm pairs where neither arm has been part of a fusion event in Arctic charr had fewer PSVs, suggesting faster diploidization rates in these regions.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/diploidization" class="hw-term hw-article-keyword hw-article-keyword-diploidization" rel="nofollow">diploidization</a></li><li class="kwd"><a href="/keyword/duplicated-genes" class="hw-term hw-article-keyword hw-article-keyword-duplicated-genes" rel="nofollow">duplicated genes</a></li><li class="kwd"><a href="/keyword/epigenetic-modification" class="hw-term hw-article-keyword hw-article-keyword-epigenetic-modification" rel="nofollow">epigenetic modification</a></li><li class="kwd"><a href="/keyword/linkage-map" class="hw-term hw-article-keyword hw-article-keyword-linkage-map" rel="nofollow">linkage map</a></li><li class="kwd"><a href="/keyword/salmonid-fishes" class="hw-term hw-article-keyword hw-article-keyword-salmonid-fishes" rel="nofollow">salmonid fishes</a></li><li class="kwd"><a href="/keyword/transmission-genetics" class="hw-term hw-article-keyword hw-article-keyword-transmission-genetics" rel="nofollow">transmission genetics</a></li><li class="kwd"><a href="/keyword/transposition" class="hw-term hw-article-keyword hw-article-keyword-transposition" rel="nofollow">transposition</a></li></ul><p id="p-4">Whole genome duplications (WGDs) are rare evolutionary events that drastically alter genomic architecture by producing duplicate copies of every chromosome. The doubling of all loci can be a major driving force of evolution, as there is the potential to produce loci with novel functions through the accumulation of mutations that were formerly selected against (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Ohno 1970</a>). WGDs provide a surfeit of genetic information that can be associated with adaptive innovation and evolutionary change (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Taylor and Raes 2004</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Moghadam <em>et al.</em> 2005</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Nakatani <em>et al.</em> 2007</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Berthelot <em>et al.</em> 2014</a>). WGDs have been important events in the evolutionary history of vertebrate animals (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Smith <em>et al.</em> 2013</a>). Two WGDs occurred in the common ancestor of all vertebrates, and these (referred to as the 1R and 2R WGDs) yielded a karyotype of between 40 and 52 chromosomes from the protovertebrate karyotype of 10–13 chromosomes (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Nakatani <em>et al.</em> 2007</a>). A third (3R) WGD occurred ∼400 million yr ago (MYA) in the ancestor of teleost fish, while a fourth salmonid specific WGD (Ss4R) took place ∼96 MYA in the ancestor of salmonid fishes (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Allendorf and Thorgaard 1984</a>; <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Berthelot <em>et al.</em> 2014</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Macqueen and Johnston 2014</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Lien <em>et al.</em> 2016</a>). Subsequent chromosome fusions and fissions caused changes in chromosome numbers and genomic architecture in specific vertebrate lineages (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Kasahara <em>et al.</em> 2007</a>; <a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Nakatani <em>et al.</em> 2007</a>).</p><p id="p-5">Following WGD, the genome undergoes the process of diploidization, where it reverts from a tetraploid (4<em>n</em>) to a diploid (2<em>n</em>) state. Specifically, chromosome pairs that shared a common genetic ancestor prior to WGD (termed homeologs), diverge from one another due to genomic rearrangements, gene deletion, pseudogenization, and mutation (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Comai 2005</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bergthorsson <em>et al.</em> 2007</a>). Salmonids are in the midst of the diploidization process, in that some regions of the genome have diverged into two pairs of diploid loci, while in other regions, residual tetrasomy occurs as the result of multivalent formation and recombination among homeologues during meiosis (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Berthelot <em>et al.</em> 2014</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Allendorf <em>et al.</em> 2015</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">May and Delany 2015</a>; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Lien <em>et al.</em> 2016</a>). For example, 52% of genes in the rainbow trout (<em>Oncorhynchus mykiss</em>) genome have diploidized since the Ss4R WGD, while the other 48% of genes have retained both copies, and have yet to revert to a diploid state (<a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Berthelot <em>et al.</em> 2014</a>). Similarly, work on the Atlantic salmon (<em>Salmo salar</em>) genome has found that 55% of genes have been retained as two functional copies since the Ss4R WGD (<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Lien <em>et al.</em> 2016</a>). Chromosomal architecture appears to play a role in determining which genomic regions undergo homeologous recombination during meiosis in these fishes (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Brieuc <em>et al.</em> 2014</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Lien <em>et al.</em> 2011</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Kodama <em>et al.</em> 2014</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Waples <em>et al.</em> 2015</a>). All recombining pairs of homeologs appear to include one large chromosome, either a metacentric produced through the Robertsonian fusion of two chromosome arms, or a fused acrocentric resulting from the tandem fusion of two smaller acrocentric chromosomes. This suggests that the large size of these chromosomes may provide the stability necessary for homeologous recombination (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Brieuc <em>et al.</em> 2014</a>; <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Kodama <em>et al.</em> 2014</a>; <a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">Lien <em>et al.</em> 2016</a>). Furthermore, the telomeric regions of homeologs show the slowest rates of diploidization within several salmonid species, as illustrated by a relatively high number of paralogous loci (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Brieuc <em>et al.</em> 2014</a>; <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Waples <em>et al.</em> 2015</a>; <a id="xref-ref-25-5" class="xref-bibr" href="#ref-25">Lien <em>et al.</em> 2016</a>).</p><p id="p-6">The development of detailed linkage maps has shown that lineage specific changes in chromosomal architecture have taken place in salmonids following WGD. Maps are available for species of <em>Salmo</em> and <em>Oncorhynchus</em> (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Rexroad <em>et al.</em> 2008</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gonen <em>et al.</em> 2014</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Limborg <em>et al.</em> 2015</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">McKinney <em>et al.</em> 2015</a>; <a id="xref-ref-51-3" class="xref-bibr" href="#ref-51">Waples <em>et al.</em> 2015</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Tsai <em>et al.</em> 2016</a>). In the karyotypes of certain species such as Atlantic salmon, a large number of chromosomes are the product of fusions (<em>e.g.</em>, 42/58) (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Phillips and Ráb 2001</a>). Variation in genomic architecture may cause the rate of diploidization to vary across the genomic landscape of the different taxa, and which regions of the genome undergo residual tetrasomy (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Lien <em>et al.</em> 2011</a>; <a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">Brieuc <em>et al.</em> 2014</a>; <a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Kodama <em>et al.</em> 2014</a>; <a id="xref-ref-25-6" class="xref-bibr" href="#ref-25">Lien <em>et al.</em> 2016</a>). Similar studies of taxa with more basal karyotypes, such as the <em>Salvelinus</em> species, would improve our understanding of the role that genomic rearrangements have in the diploidization process. For instance, karyotype data suggests that only 20 of the 78 chromosomes (2<em>n</em> = 78) in Arctic charr (<em>Salvelinus alpinus</em>) are metacentric, suggesting that fewer Robertsonian fusion events have occurred in their evolutionary history (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hartley 1989</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Phillips <em>et al.</em> 2002</a>). The study of genomic evolution in Arctic charr is currently limited by the low resolution of available genetic linkage maps, and the paucity of known molecular markers (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Woram <em>et al.</em> 2004</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Norman <em>et al.</em> 2011</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Timusk <em>et al.</em> 2011</a>). Expanding the genomic resources of Arctic charr through the addition of several thousand SNPs will make it possible to track the genomic rearrangements that have shaped the modern Arctic charr karyotype.</p><p id="p-7">The initial aims of this study were to (1) increase the number of known genetic markers in the Arctic charr genome using genotype by sequencing (GBS); (2) create a second generation genetic linkage map of the Arctic charr genome using newly identified SNP markers, and integrate the revised map to previous versions primarily based on microsatellite loci; and (3) identify the chromosome type associated with each linkage group, and use this information to test for the existence of acrocentric-acrocentric homeologous pairs in the genome. These are of interest because a lack of homeologous recombination could cause diploidization to occur faster in homeologous pairs of this type. Using the data produced to meet the above goals, we also aimed to (4) compare genomic data from Arctic charr to the genomes of rainbow trout and Atlantic salmon to identify chromosome arm homologies across these three species, allowing the characterization of conserved genomic rearrangements and fusion/fission events unique to the Arctic charr lineage; and, also, (5) identify putative duplicate loci, and assess their distribution across the Arctic charr genome. As we analyzed sequence data, it became apparent that there were signatures of significant transposon activity within the Arctic charr genome. We therefore used this opportunity to characterize transposable element (TE) activity in the Arctic charr genome, and see how TE distribution is affected by residual tetrasomy, chromosome architecture, and the uneven distribution of duplicate loci throughout the genome.</p><p id="p-8">Transposon activity is associated with important evolutionary transitions, adaptation to novel environments, and extensive changes in genome evolution (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">de Boer <em>et al.</em> 2007</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Schrader <em>et al.</em> 2014</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Staton and Burke 2015</a>). In fact, the estimated time of radiation of <em>Salmoninae</em> into <em>Salmo</em>, <em>Oncorhynchus</em>, and <em>Salvelinus</em> (14–23 MYA) (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Macqueen and Johnston 2014</a>) coincides with a known spike in TE activity (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">de Boer <em>et al.</em> 2007</a>). TEs cause sequence deletion or duplication due to unequal homologous recombination and segmental duplication, and they facilitate genomic rearrangements (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Kazazian 2004</a>). Genomic regions with an accumulation of TEs appear to evolve faster than the rest of the genome (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Schrader <em>et al.</em> 2014</a>). Therefore, TE activity might be influenced by the rate of residual tetrasomy in particular chromosomal regions, or vice versa. Reduced presence of TEs has been observed in the duplicated regions of Atlantic salmon, chum salmon (<em>O. keta</em>), and chinook salmon (<em>O. tshawytscha</em>) that lag behind in the diploidization process (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Larson <em>et al.</em> 2015</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">McKinney <em>et al.</em> 2015</a>; <a id="xref-ref-51-4" class="xref-bibr" href="#ref-51">Waples <em>et al.</em> 2015</a>; <a id="xref-ref-25-7" class="xref-bibr" href="#ref-25">Lien <em>et al.</em> 2016</a>). Therefore, our final goal was to characterize TE activity in the Arctic charr genome, and determine if there is a relationship between residual tetrasomy and TE activity in the telomeric regions of homeologs that undergo homeologous recombination.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Source mapping panel</h3><p id="p-9">The analysis utilized 85 full-siblings, and their parents, from a single family of Fraser strain Arctic charr obtained from the Coastal Zones Research Institute (CZRI), Shippagan, NB Canada. The Fraser strain originated from collections of fish from the Fraser River, Labrador, Canada, in the 1980s. The family was produced on November 6, 2012, and reared communally until March 11, 2014, at which time each fish was PIT tagged, weighed (to the nearest gram), measured (fork length), and samples of adipose fin were removed for DNA analysis. DNA was extracted using a commercial kit (Qiagen DNeasy Blood &amp; Tissue), as per manufacturer’s instructions, and treated with RNase A to remove any RNA. The samples were then quantified using a Qubit Fluorometer to ensure that all DNA concentrations exceeded 50 ng/µl.</p></div><div id="sec-3" class="subsection"><h3>Sequencing analysis</h3><p id="p-10">The DNA samples were submitted for GBS (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Elshire <em>et al.</em> 2011</a>) at the Cornell Institute of Biotechnology. DNA from each progeny was added to a single well of a 96-well plate, while the parents were analyzed in triplicate, to increase sequencing depth and provide the information necessary for linkage mapping based on the inheritance of SNP alleles. The four grandparents of the family were also added to a single well each. Samples were digested with the restriction enzyme <em>Eco</em>T22I, and unique barcode sequence adapters (4–8 bp in length) were ligated to each of the DNA samples (three for each parent) such that the DNA sequence data could be assigned to a specific individual or parental subsample. After the barcodes were added, sequencing primers, and the samples from all 95 wells containing DNA samples (and a single blank control well) were pooled. Paired end sequencing primers with oligonucleotides that allow binding to the sequencing flowcell were then added to the pooled samples. Polymerase chain reaction (PCR) was then used to amplify the DNA fragment pool, and the resulting DNA products were analyzed for fragment size. The DNA samples were then sequenced on an Illumina Hisequation 2000 high-throughput sequencing instrument, and replicated across two flowcells. The sequencing process produced 100 bp single-end reads. GBS sequence reads are available in the NCBI sequence read archive (<a href="http://www.ncbi.nlm.nih.gov/sra">www.ncbi.nlm.nih.gov/sra</a>) under the BioProject accession number #SRP026259 and BioSample accession numbers #SAMN06165956 and #SAMN06165957.</p></div><div id="sec-4" class="subsection"><h3>SNP identification from raw sequence data</h3><p id="p-11">Raw sequence reads were analyzed using the UNEAK pipeline, part of the Tassel 3.0 software package produced by the maize genetics laboratory at Cornell University (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Glaubitz <em>et al.</em> 2014</a>). The UNEAK pipeline allows for the identification of SNPs in species where a reference genome is not available. The 5′ sequence barcodes were used to define individual-specific reads, but were trimmed prior to sequence analysis. Alignment of all the sequence data was conducted to produce a master “tag” list for the dataset. Tags are unique sequences of up to 64 bp in length observed across multiple reads. Further alignment of the master tag list identified tag pairs with a single base pair mismatch, and these were considered SNPs (with a sequencing error tolerance rate parameter of 0.03) (for more information, see <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Glaubitz <em>et al.</em> 2014</a>). Note the program only considered tag pairs with a 1 bp mismatch to be SNPs, so any 64 bp read with 2+ SNPs would be excluded. The number of times each tag is observed in the sequencing data from each individual is used to determine the individual’s genotype for a particular SNP.</p></div><div id="sec-5" class="subsection"><h3>SNP filtering</h3><p id="p-12">The inventory list of SNP genotypes identified within the UNEAK pipeline was analyzed manually to remove markers where: (1) data were available for &lt;75/85 progeny; (2) the genotype for one or both of the parents was missing. Markers where SNP inheritance displayed significant segregation distortion (<em>i.e.</em>, SNPs with <em>G</em> values &gt;6.693, <em>P</em> &lt; 0.01) were identified and removed using LINKMFEX (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Danzmann 2016</a>).</p></div><div id="sec-6" class="subsection"><h3>Microsatellite genotyping</h3><p id="p-13">To anchor the newly identified SNP markers to previous Arctic charr linkage maps, genotypes at 102 microsatellite loci (Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038026/-/DC1/FileS1.xlsx">File S1</a>) from known locations across the Arctic charr genome were determined for all progeny and parents using established genotyping methods (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Moghadam <em>et al.</em> 2007</a>; <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Timusk <em>et al.</em> 2011</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Norman <em>et al.</em> 2014</a>).</p></div><div id="sec-7" class="subsection"><h3>Linkage mapping</h3><p id="p-14">The high quality SNPs selected for mapping, and the microsatellite marker anchors, were assessed for genetic linkage using LINKMFEX (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Danzmann 2016</a>). The SNPs were split into three categories: heterozygous male, heterozygous female, and double heterozygote (DH) SNPs where both parents were heterozygotes. The SNPs where only a single parent was heterozygous had high information content, given that all progeny genotypes are informative for linkage mapping. Double heterozygous SNP markers are informative in only about half of the progeny, given that linkage phases cannot be assigned in heterozygous progeny. Markers that are heterozygous in only one parent are problematic in that SNP mapping locations cannot be compared between parents. Data from additional mapping panels will be required to compare map orders between the sexes for these markers. Double heterozygous SNP markers were assigned to specific linkage groups, but were not added to specific map locations.</p><p id="p-15">Linkage groups were identified using a logarithm of odds (LOD) threshold of 10 for linkage group assessment. We first created male-parent- and female-parent-specific linkage maps using the SNPs that were heterozygous in only one parent using LINKMFEX. In order to integrate the male and female linkage maps with one another, SNPs where both parents were heterozygous were added to each of the two datasets, and analyzed in LINKMFEX for genetic linkage with a LOD=10 threshold. It was then possible to identify overlapping sets of male- and female-specific linkage groups.</p><p id="p-16">To determine the location of unlinked markers, repeated analysis was performed using descending LOD scores (LOD = 6 down to LOD = 3) using LINKMFEX, and LOD = 3 additions were accepted if they created joinings of two or more linkage groups that are known to be homeologous to one another. For some markers, a pseudolinkage (see below) of known homeologous chromosome arms was detected within the range of LOD = 3.0 to LOD = 5.0, and were therefore accepted. Linkage groups were named according to historical designations based on previous microsatellite marker assignments (<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Woram <em>et al.</em> 2004</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Norman <em>et al.</em> 2011</a>; <a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">Timusk <em>et al.</em> 2011</a>). For each linkage group, marker order was determined in OneMap using the record algorithm (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Margarido <em>et al.</em> 2007</a>). The record algorithm was selected for ordering because it consistently gave the shortest map distances [<em>i.e.</em>, smallest number of adjacent double cross-over (DCO) alignment points] out of the three possible OneMap ordering functions (ug, rcd, and record). This option also gave shorter map lengths than those produced by LINKMFEX. Male and female marker orderings were determined separately for each linkage group. As mentioned above, markers heterozygous in both parents were not ordered. OneMap marker ordering was further refined using the LINKMFEX program Adjacent-DCO-Count_Ripple-Check. This program was used to identify and reorder marker placements causing adjacent DCOs in the ordering. Adjacent double cross-overs are biologically unlikely in salmonids, given the high levels of chromatid interference detected during meiosis (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Sakamoto <em>et al.</em> 2000</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Danzmann and Gharbi 2001</a>; <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Allendorf <em>et al.</em> 2015</a>) in these species. The revised marker ordering minimized the number of adjacent DCOs in the dataset. Final map distances were calculated using the MAPDIS-V program in LINKMFEX, and selecting the option to ignore adjacent DCO events. We chose this option, as we considered that remaining adjacent DCOs may be due to errors in genotyping calls.</p></div><div id="sec-8" class="subsection"><h3>sdY marker</h3><p id="p-17">The progeny were genotyped for the sexually dimorphic gene (sdY) located on the Y-chromosome, using the PCR and agarose gel visualization methods described in <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Yano <em>et al.</em> (2012)</a> with two modifications. First, we substituted insulin-like growth factor binding protein 5 (IGFBP5) as a positive control, using primers we developed: DQ206713-F3 (CCACCAGCTAATTACTGCAA) and DQ206713-R3 (GTAGAATTTGGCTGGCCCTA). Second, the following PCR temperature cycling conditions was used: denaturation for 5 min at 95°, followed by five cycles of 95° for 1 min, 58° for 30 sec, and 72° for 30 sec, then 30 cycles of 95° for 30 sec, 58° for 30 sec, and 72° for 30 sec, followed by a final 10 min at 72°. The sdY marker was validated based on conformity of the sdY genotypes to phenotypic assessment of the individual’s sex in this population.</p></div><div id="sec-9" class="subsection"><h3>Comparison with rainbow trout and Atlantic salmon genomes</h3><p id="p-18">SNP sequences were compared to the rainbow trout draft genome, and the Atlantic salmon genome (<a id="xref-ref-5-5" class="xref-bibr" href="#ref-5">Berthelot <em>et al.</em> 2014</a>; <a id="xref-ref-25-8" class="xref-bibr" href="#ref-25">Lien <em>et al.</em> 2016</a>), using the following BLASTn parameter settings (-word_size 11 -gapopen 5 -evalue 0.00001 -gapextend 2 -reward 2 -penalty -3) (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Altschul <em>et al.</em> 1990</a>). Blast hits were filtered, and the hits with the lowest <em>e</em>-value were used for subsequent homology identification. In the case of equivalent <em>e</em>-values, all Blast hits were retained, and considered equal “top hits” for the given SNP.</p></div><div id="sec-10" class="subsection"><h3>Moveable genetic elements and homeologies</h3><p id="p-19">Following Blast comparison of the Arctic charr markers to the rainbow trout and Atlantic salmon genomes, SNPs that aligned to adjacent regions of a single rainbow trout, and Atlantic salmon chromosome arm, allowed for the identification of arm homologies across the species. Within the SNP clusters that displayed consistent homology to a given Atlantic salmon, or rainbow trout, chromosome arm, individual markers sometimes showed homologies to disparate regions of the genome. Two hypotheses about the cause of these disparate SNPs were tested: (1) the SNPs lie within moveable, and/or highly repetitive DNA sequences; (2) the SNPs may be aligning to a homeologous chromosome arm with highly similar sequences. To test for the existence of repetitive sequences and moveable DNA elements, BLASTn (same parameters as above) was used to compare the linkage map SNPs to Repbase Update’s database of known vertebrate moveable and repetitive DNA elements (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Jurka <em>et al.</em> 2005</a>). The distribution of TE Blast hits in the linkage map was used to assess TE activity in the Arctic charr genome. TE activity in each linkage group was determined by assessing the proportion of markers in each linkage group that displayed significant Blast alignments to TEs.</p></div><div id="sec-11" class="subsection"><h3>Genomic architecture and residual tetrasomy</h3><p id="p-20">Paralogous sequence variants (PSVs) were identified to assess the distribution of duplicate loci through the Arctic charr genome. To do this, fixed heterozygote SNPs were used as PSV markers. These SNPs were heterozygous in both parents, and 100% of the progeny. The lack of homozygous progeny suggests that these are duplicate, monomorphic, loci with a single base-pair difference, causing them to appear heterozygous in all individuals. Since these PSVs lack recombinant progeny, their location relative to the linkage map SNPs cannot be determined. However, their linkage group affinities can be inferred based upon comparative homologies. We tentatively assigned these PSVs to Arctic charr linkage groups by aligning the PSVs and linkage map SNPs to the Atlantic salmon genome using BLASTn, and comparing their top hit locations. Two characteristics were assessed: (1) the distribution of PSVs between, and within, chromosomes of different genomic architectures, including Acrocentric Homeolog Pairs (AHPs), and High Residual Tetrasomy Arms (HRTAs); (2) the distribution of PSVs along chromosome arms relative to the centromere or telomere (see <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). AHPs were defined as homeologous pairs of Arctic charr chromosome arms where neither arm in the pair is fused with another chromosome arm. HRTAs are defined as pairs of homeologous chromosomes that have higher levels of duplicate loci than the rest of the genome in multiple salmonid species (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Danzmann <em>et al.</em> 2008</a>; <a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Lien <em>et al.</em> 2011</a>, <a id="xref-ref-25-9" class="xref-bibr" href="#ref-25">2016</a>; <a id="xref-ref-6-5" class="xref-bibr" href="#ref-6">Brieuc <em>et al.</em> 2014</a>), and likely form multivalents during meiosis due to crossing-over between their homeologous arms (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Sakamoto <em>et al.</em> 2000</a>)</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/543/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Visual representation of how Atlantic salmon chromosome arms were divided into quarters to assess Arctic charr PSV and map marker BLASTn hit distributions. Circles represent centromeres." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-148801367" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Visual representation of how Atlantic salmon chromosome arms were divided into quarters to assess Arctic charr PSV and map marker BLASTn hit distributions. Circles represent centromeres.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/543/F1.medium.gif" width="440" height="290"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/2/543/F1.medium.gif" width="440" height="290"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/543/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/543/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470424" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-21" class="first-child">Visual representation of how Atlantic salmon chromosome arms were divided into quarters to assess Arctic charr PSV and map marker BLASTn hit distributions. Circles represent centromeres.</p><div class="sb-div caption-clear"></div></div></div><p id="p-22">The distribution of duplicate loci in the genome was assessed using the Blast TopHit dataset, which consisted of markers in the Arctic charr linkage map and PSVs. The linkage map markers represent diploid loci, and the PSVs represent tetraploid loci. These markers were assigned to Atlantic salmon chromosome arm based on their top BLASTn hit locations (determined by lowest observed <em>e</em>-value). For markers with BLASTn hits of equal <em>e</em>-values to a single chromosome arm, only a single hit per chromosome arm was retained in the dataset. In the case of markers (both PSVs and linkage map markers) with equal BLASTn hit locations on two Atlantic salmon chromosome arms, the top Blast hit to each chromosome arm was retained in the dataset. Markers with equal top Blast hits to three or more chromosome arms, and markers with no Blast hits in the Atlantic salmon genome, were removed from the dataset.</p><p id="p-23">To further assess whether any of the apparent single copy SNP markers may be duplicate copies of one another, we performed a BLASTn analysis of all SNPs against all SNPs (non-PSVs) in the database (see parameter settings above). Duplicate pairs exceeding 95% identity, and a 95% overlap in length, and occurring on separate chromosome arms, were considered potential homeologs of one another. Duplicates mapping to the same linkage groups were considered regional marker duplicates, unless they mapped adjacent to one another, indicating some type of tandem duplication.</p><p id="p-24">We tested if duplicate loci are preserved in a higher frequency on HRTAs. Using Atlantic salmon as a reference, the HRTA homeolog pairs are represented by: Ssa02p/Ssa05q, Ssa11qa/Ssa26, Ssa16qa/Ssa17qa, Ssa03q/Ssa06p, Ssa12qa/Ssa02q, Ssa07q/Ssa17qb, and Ssa04p/Ssa08q. Using a contingency chi-square test, the number of linkage map and PSV top BLASTn hits on the HRTAs was compared to the number of hits on all other chromosome arms.</p><p id="p-25">An additional contingency chi-square test was performed to test if Arctic charr AHPs preserve duplicate loci in the same manner as the rest of the genome. Putative AHPs were identified based on homologies in the Arctic charr genome, and their respective Atlantic salmon (<a id="xref-ref-25-10" class="xref-bibr" href="#ref-25">Lien <em>et al.</em> 2016</a>), and rainbow trout (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Danzmann <em>et al.</em> 2008</a>; <a id="xref-ref-5-6" class="xref-bibr" href="#ref-5">Berthelot <em>et al.</em> 2014</a>), homeologies. Based on this information, the following AHPs were identified in Arctic charr: AC02/AC36, AC05/AC29, AC19/AC32, and AC30/AC31. A contingency chi-square test compared the number of linkage map and PSV top BLASTn hits on acrocentric homeolog chromosome arms to the number of hits on all other chromosome arms.</p><p id="p-26">The base pair hit location (s.start) for each marker in the Blast TopHit dataset was used to assess the PSV and linkage map markers’ distributions along Atlantic salmon chromosome arms. Each Atlantic salmon chromosome arm was divided equally into four quarters based on the known base-pair start and end locations (<a id="xref-ref-25-11" class="xref-bibr" href="#ref-25">Lien <em>et al.</em> 2016</a>). Quarter 1 was closest to the centromere, and quarter 4 was telomeric (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). All marker hits were then assigned to a chromosome arm quarter, and data from all the chromosome arms were merged. A chi-square test (goodness-of-fit) was performed to see if the frequency of linkage map SNPs and PSVs varied across different chromosome arm quarters. In addition to the above tests, BLASTn was used to compare the PSVs to Repbase update’s list of vertebrate TEs. The number of TE hits in PSVs was then compared to the number of TE hits in the linkage map SNPs using a contingency chi-square test.</p><p id="p-27">We tested if PSVs in Arctic charr aligned more toward the telomeres of Atlantic salmon chromosome arms compared to linkage map SNPs. Tetrasomy is more readily preserved near the telomeres of chromosomes and most cross-overs in multivalent chromosome formations occur toward the telomeres (<a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Sakamoto <em>et al.</em> 2000</a>). Therefore, residual tetrasomic inheritance would likely persist in these regions for a longer period of time following the Ss4R WGD, and highly similar duplicate loci are expected to be found in high frequency closer to the telomeres of chromosome arms (<a id="xref-ref-6-6" class="xref-bibr" href="#ref-6">Brieuc <em>et al.</em> 2014</a>; <a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Allendorf <em>et al.</em> 2015</a>).</p></div><div id="sec-12" class="subsection"><h3>Data availability</h3><p id="p-28">Details on data used in this study can be found in Materials and Methods under the Sequencing Analysis section, and in the supplementary files.</p></div></div><div class="section results-discussion" id="sec-13"><h2 class="">Results and Discussion</h2><div id="sec-14" class="subsection"><h3>SNP identification and linkage mapping</h3><p id="p-29">GBS of the Arctic charr family produced 4 × 10<sup>8</sup> sequence reads (roughly 4.7 million reads per progeny). Using the UNEAK pipeline (Tassel 3.0), 19,418 SNPs were then identified. Following the SNP filtering process, and the addition of microsatellite markers, the mapping dataset consisted of 4536 markers (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038026/-/DC1/FileS1.xlsx">File S1</a> for the DNA sequence corresponding to each SNP marker).</p><p id="p-30">The linkage mapping process produced 39 linkage groups containing 4508 markers (4405 SNPs, 1 sdY, and 102 SSR), while 28 markers (24 SNP and 4 SSR) remained unlinked. Separate male and female maps were produced (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038026/-/DC1/FileS2.xlsx">File S2</a>). A total of 1538 markers was ordered in the male map, spanning a distance of 2808.5 cM, while 1709 markers were ordered in the female map, covering 4302.7 cM (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> and <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Markers that were heterozygous in both parents were not ordered, as mentioned previously, but 1283 of these were assigned to linkage groups (linkage group assignments are found in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038026/-/DC1/FileS2.xlsx">File S2</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/543/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Visual representation of Arctic charr linkage groups. Female linkage groups are shown in red, and male linkage groups are shown in blue. Each point along the length of the line represents a single marker, or zero recombination cluster of several markers." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-148801367" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Visual representation of Arctic charr linkage groups. Female linkage groups are shown in red, and male linkage groups are shown in blue. Each point along the length of the line represents a single marker, or zero recombination cluster of several markers.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/543/F2.medium.gif" width="440" height="278"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/2/543/F2.medium.gif" width="440" height="278"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/543/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/543/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470299" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-31" class="first-child">Visual representation of Arctic charr linkage groups. Female linkage groups are shown in red, and male linkage groups are shown in blue. Each point along the length of the line represents a single marker, or zero recombination cluster of several markers.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470295/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470295/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">List of Arctic charr linkage groups; the number of markers in each linkage group and the map distance covered (centiMorgans) by each male and female linkage group</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Microsatellite anchors</h3><p id="p-36">Of the 102 microsatellite markers genotyped, 98 were successfully added to the linkage map (four remained unlinked). Five markers were duplicated and mapped to two linkage groups. Of 39 linkage groups, 36 contained one or more microsatellite markers, allowing the new SNP-based linkage groups to be aligned with the microsatellite-based linkage maps (<a id="xref-ref-53-3" class="xref-bibr" href="#ref-53">Woram <em>et al.</em> 2004</a>; <a id="xref-ref-49-4" class="xref-bibr" href="#ref-49">Timusk <em>et al.</em> 2011</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Norman <em>et al.</em> 2012</a>). The identity of three linkage groups without microsatellite markers (AC25, AC30, and AC31) was determined based on BLASTn search results, and previously identified arm homologies among the salmonid species compared.</p></div><div id="sec-16" class="subsection"><h3>Chromosome type and salmonid arm homologies</h3><p id="p-37">A comparison of the Arctic charr linkage map to the Atlantic salmon (<a id="xref-ref-25-12" class="xref-bibr" href="#ref-25">Lien <em>et al.</em> 2016</a>) and rainbow trout genomes (<a id="xref-ref-5-7" class="xref-bibr" href="#ref-5">Berthelot <em>et al.</em> 2014</a>) based on the BLASTn analysis is presented in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>. The top Blast hit data for each marker are available in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038026/-/DC1/FileS2.xlsx">File S2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038026/-/DC1/FileS3.xlsx">File S3</a>. Nine linkage groups (AC01, -03, -06, -08, -13, -14, -15, -18, and -27) were identified as metacentric chromosomes based on homologies to two salmonid chromosome arms. Two potential split metacentric groupings were identified (AC04 and AC20). One set involves the sex-linkage group AC04, and has been identified as possessing a fusion polymorphism in Arctic charr (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Moghadam <em>et al.</em> 2007</a>). Our data also suggest that a two acrocentric <em>vs.</em> one metacentric polymorphism exists for AC04 (see below). The q-arm in this chromosome set involves a fusion of two ancestral salmonid chromosome arms that are homologous to Omy02q and Omy25, while the p-arm is homologous to Omy24. None of these arms are homeologous to one another, and are therefore unlikely to show pseudolinkage affinities. The second set of chromosomes (AC20 group) involves a small (AC20a) and a large (AC20b) acrocentric arm, where the large arm appears to be composed of two fused ancestral salmonid arms that are homologous to Omy12p/q and Ots09/q. Therefore, it is more likely that AC20b represents an entire metacentric chromosome, and AC20a represents a separate acrocentric arm, rather than AC20a/20b representing a large metacentric chromosome comprised of joined homeologous arms (<a id="xref-ref-53-4" class="xref-bibr" href="#ref-53">Woram <em>et al.</em> 2004</a>). This would support the suggestion that there are 10 stable metacentrics in North American Arctic charr (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Hartley 1989</a>; <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Phillips <em>et al.</em> 2002</a>), with the AC04 polymorphism generating an additional metacentric in some individuals. The homologous chromosome arm of AC20a in rainbow trout is Omy13q, and, since Omy12q/Omy13q are homeologs, they may show pseudolinkage affinities. Twenty-seven linkage groups appear to be acrocentric, including two (AC04 and AC17) that result from a tandem fusion of qa and qb chromosome arms. AC17 appears to include segments homologous to both Omy16q and 20q. The total chromosome arm number (NF) observed was 100, containing 52 haploid ancestral arm segments.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470474/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470474/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Salmonid chromosome arm homologies referenced to Arctic charr linkage group homologies</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h3>Sex-linkage group polymorphism in AC04</h3><p id="p-43">Previous work has shown AC04 to be polymorphic, and taking the form of either a single linkage group (type 1) or two unlinked linkage groups (type 2) (<a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Moghadam <em>et al.</em> 2007</a>). The mapping parents in this study are both type 2 individuals; microsatellite markers associated with AC04 are found on both AC04p and AC04q (<a id="xref-ref-53-5" class="xref-bibr" href="#ref-53">Woram <em>et al.</em> 2004</a>; <a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">Moghadam <em>et al.</em> 2007</a>; <a id="xref-ref-49-5" class="xref-bibr" href="#ref-49">Timusk <em>et al.</em> 2011</a>). Previous research identified three salmonid chromosome arms homologous with AC04 (<a id="xref-ref-49-6" class="xref-bibr" href="#ref-49">Timusk <em>et al.</em> 2011</a>). Two of these arms are homologous with AC04q, and the third is homologous with AC04p (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). AC04p also contains the sex-determining gene, sdY (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Yano <em>et al.</em> 2013</a>). Although two separate, type 2 AC04 linkage groups are observed in the current linkage map, the polymorphic nature of this linkage group, and the evidence presented here, is indicative of a fusion/fission polymorphism for the sex-determining chromosome of Arctic charr. Affinity of the sdY marker to the Slml-family of TEs supports the suggestion that the reported sex-linkage difference between North American (AC04) and European (AC01/21) Arctic charr may be the result of a translocation through TE movements (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Woram <em>et al.</em> 2003</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Küttner <em>et al.</em> 2011</a>).</p></div><div id="sec-18" class="subsection"><h3>Pseudolinkage</h3><p id="p-44">Two pairs of homeologous linkage groups (AC01q/21 and AC13q/34) were detected as possessing a weak pseudolinkage to one another (LOD ≥ 3–5). Interestingly, pseudolinkage was detected between both homeologous pairs of linkage groups in both the male and female parents (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). Previously, pseudolinkage was thought to occur only within male meiosis, with rare reports of female pseudolinkage (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Ostberg <em>et al.</em> 2013</a>; <a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Allendorf <em>et al.</em> 2015</a>). Our identification of pseudolinkage in a female confirms that multivalents are also likely formed during female meioses.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470292/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470292/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Allele counts of the two homeolog pairs displaying pseudolinkage within females</span><div class="sb-div caption-clear"></div></div></div><p id="p-46">Pseudolinkage is a phenomenon arising due to segregation of gametes following preferential pairing of homeologous chromosome arms during meiosis I (<a id="xref-ref-2-5" class="xref-bibr" href="#ref-2">Allendorf <em>et al.</em> 2015</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">May and Delany 2015</a>). Preferential as opposed to random pairing typically occurs in cases of hybridization, where the homeolog pairs provided by one parent may be more closely related to one another due to their species-specific ancestry. Therefore, within the hybrid, the homeologs from one parent may therefore be more likely to pair with one another and recombine. Crossing-over tends not to occur between homeologous markers located very close to the centromere. If preferential pairing occurs at meiosis I, this leads to a significant excess of nonparental genotypes being produced following meiosis II, because alternate disjunction occurs as the multivalents separate at meiosis I [see <a id="xref-ref-2-6" class="xref-bibr" href="#ref-2">Allendorf <em>et al.</em> (2015)</a> and <a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">May and Delany (2015)</a> for a more detailed explanation of these models]. This produces a statistical linkage between the two homeologous chromosome arms characterized by a significant excess of nonparental gametes, and is a characteristic feature of hybrid salmonids.</p><p id="p-47">Random pairing of homeologous pairs may, however, still occur as the genome undergoes the process of diploidization within species. This may involve the random formation of both bivalents, and multivalents, between the homeologous pairs, such that gametic expectation models cannot be precisely defined. This may result in varying levels of exchange among alleles for loci that are located proximal and distal to chiasmata junctions along the length of randomly paired homeologous chromosomes [see <a id="xref-ref-44-4" class="xref-bibr" href="#ref-44">Sakamoto <em>et al.</em> (2000)</a>, for an explanation]. We have developed models to explain this weaker form of pseudolinkage, and will present these in a subsequent publication.</p></div><div id="sec-19" class="subsection"><h3>Homeologies</h3><p id="p-48">Arm homologies identified in Atlantic salmon, rainbow trout, and chinook salmon (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>) with Arctic charr, were used to identify homeologous arm pairings within Arctic charr (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). Lower levels of duplicate marker regions were detected for certain arms, but can nonetheless be inferred based upon the recent extensive survey of duplicate gene copies in Atlantic salmon (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hermansen <em>et al.</em> 2016</a>) (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>). All seven homeologous pairs in Arctic charr identified as HRTAs in other species contain one chromosome arm that has undergone a chromosome fusion. One pair of HRTA identified in Atlantic salmon (Ssa09qc/20qb) (<a id="xref-ref-25-14" class="xref-bibr" href="#ref-25">Lien <em>et al.</em> 2016</a>) did not show a high number of duplicates in either rainbow trout or chinook salmon. Similarly a HRTA region in both rainbow trout and chinook salmon (Omy01q and Ots06q/Omy23 and Ots01q), was not identified as an HRTA in Atlantic salmon (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>). Four AHPs, where neither homeolog has undergone a fusion event since the Ss4R, were identified in Arctic charr (<a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>). AHP likely undergo minimal residual tetrasomic inheritance, given that AHP multivalent pairings are unlikely due to structural instability (<a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">May and Delany 2015</a>). We found support for this prediction in that the number of PSVs aligning to AHPs was significantly lower compared to the number aligning to the other chromosome arms (<em>P</em> &lt; 0.0001). Of the linkage map markers that successfully aligned to the Atlantic salmon genome, 20% (861/4290) aligned to HRTA chromosome arms, while only 14.0% of PSVs (158/1130) had top hit locations on AHPs. These results suggest that fewer duplicate loci exist on AHPs, which might be indicative of more rapid diploidization relative to the rest of the genome (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>).</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470310/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470310/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Reference table of homeologous chromosome pairs in Arctic charr</span><div class="sb-div caption-clear"></div></div></div><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470301/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470301/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Reference table for the analysis of Arctic charr duplicate loci distribution</span><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/543/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="BLASTn hit locations in the Atlantic salmon genome for Arctic charr PSVs, and linkage map markers; 40.8% (462/1130) of PSVs had their top hit locations on HRTAs, while only 22.1% (950/4290) of the linkage map SNPs had their top Blast hit locations on HRTAs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-148801367" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;BLASTn hit locations in the Atlantic salmon genome for Arctic charr PSVs, and linkage map markers; 40.8% (462/1130) of PSVs had their top hit locations on HRTAs, while only 22.1% (950/4290) of the linkage map SNPs had their top Blast hit locations on HRTAs.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/543/F3.medium.gif" width="440" height="278"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/2/543/F3.medium.gif" width="440" height="278"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/543/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/543/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470377" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-51" class="first-child">BLASTn hit locations in the Atlantic salmon genome for Arctic charr PSVs, and linkage map markers; 40.8% (462/1130) of PSVs had their top hit locations on HRTAs, while only 22.1% (950/4290) of the linkage map SNPs had their top Blast hit locations on HRTAs.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-20" class="subsection"><h3>Duplicate loci</h3><p id="p-52">The distribution of PSVs throughout the Arctic charr genome appeared to be nonrandom, with certain regions preserving higher numbers of duplicate loci. BLASTn alignment of PSVs, and linkage map SNPs to the Atlantic salmon genome, was performed to compare the locations of linkage map markers and PSVs. The top BLASTn hit locations (based on lowest <em>e</em>-values) were used to assess the distribution of SNPs throughout the genome. Direct comparison of the markers through linkage mapping was not possible, given that PSVs lack any type of segregation pattern, and therefore cannot be mapped. For markers with BLASTn hits of equal <em>e</em>-values on a single chromosome arm, only a single hit was used per chromosome arm in the dataset. In the case of markers (both linkage map markers and PSVs) with BLASTn hit locations of equal <em>e</em>-values on two Atlantic salmon chromosomes, a BLASTn hit to each chromosome arm was retained in the dataset, as these markers represented potential duplicate loci. Markers with equal BLASTn hits to three or more locations were excluded from the dataset.</p><p id="p-53">The analysis identified 1130 PSV Blast hits in the Atlantic salmon genome (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>), in addition to the 4429 SNPs assigned to the linkage map. If the PSV distribution throughout the genome is random, and chromosome arm size was the only factor influencing their distribution, then the proportion of PSVs aligning to each chromosome would be similar to the proportion of linkage map SNPs aligning to each chromosome arm. To assess the effect of genomic architecture on the preservation of duplicate loci, chromosome arms were binned into two categories: HRTAs (listed in <a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table 5</a>) and non-HRTA (other) salmonid chromosome arms. A contingency test showed higher numbers of PSVs aligned to the HRTAs than expected (<em>P</em> &lt; 0.0001) (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). Of the 14 chromosome arms in the HRTA category, 12 had higher proportions of PSV Blast hits than linkage map marker Blast hits (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). The two HRTA arms with lower numbers of PSV BLASTn hits (Ssa04p and Ssa17qb) had homeologs with high numbers of PSV hits (Ssa08q and Ssa07q, respectively). This suggests that the lower number of PSV hits on the HRTA arms may be due to more PSVs aligning to their homeologs because of small sequence differences. High numbers of PSVs are observed in regions that preserved residual tetrasomy longer after the Ss4R than other parts of the genome, which may be indicative of historically reduced diploidization rates (<a id="xref-ref-6-8" class="xref-bibr" href="#ref-6">Brieuc <em>et al.</em> 2014</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/543/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The top BLASTn hit locations of Arctic charr linkage map SNPs, and Arctic charr PSVs, across Atlantic salmon chromosome arms. The data are shown in the proportion of hits from a category (Map SNPs or PSVs) in order to account for the bias of Atlantic salmon chromosome size. Arms with a * are HRTAs identified in Lien et al. (2016)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-148801367" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The top BLASTn hit locations of Arctic charr linkage map SNPs, and Arctic charr PSVs, across Atlantic salmon chromosome arms. The data are shown in the proportion of hits from a category (Map SNPs or PSVs) in order to account for the bias of Atlantic salmon chromosome size. Arms with a * are HRTAs identified in Lien &lt;em&gt;et al.&lt;/em&gt; (2016).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/543/F4.medium.gif" width="440" height="271"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/2/543/F4.medium.gif" width="440" height="271"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/543/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/543/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470286" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-54" class="first-child">The top BLASTn hit locations of Arctic charr linkage map SNPs, and Arctic charr PSVs, across Atlantic salmon chromosome arms. The data are shown in the proportion of hits from a category (Map SNPs or PSVs) in order to account for the bias of Atlantic salmon chromosome size. Arms with a * are HRTAs identified in <a id="xref-ref-25-15" class="xref-bibr" href="#ref-25">Lien <em>et al.</em> (2016)</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-55">Several other Atlantic salmon arms displayed high numbers of PSV BLASTn hits, notably Ssa09qb, Ssa09qc, Ssa10qb, and Ssa22. The Arctic charr homologs of these arms are AC08p/q, AC04p, AC04qb, and AC11, respectively. All of these arms, or their homeologs in Arctic charr, have undergone a fusion. Interestingly, these arms also share homology to rainbow trout chromosome arms with high numbers of duplicates (<a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>). Previously, it had been thought that one metacentric chromosome must be present in a homeolog pair to provide the stability necessary for homeologous pairing, and multivalent formation (<a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">Kodama <em>et al.</em> 2014</a>). However, <a id="xref-ref-25-16" class="xref-bibr" href="#ref-25">Lien <em>et al.</em> (2016)</a> have recently shown that metacentric structures are not a requirement for homeologous pairing, given that two HRTA pairs in Atlantic salmon (Ssa11qa/Ssa26 and Ssa16qb/Ssa17qa) preserve high sequence similarity without the presence of a metacentric, but, in both cases, one of the arms has undergone a fusion event. This suggests that fused acrocentrics, as well as metacentric chromosomes, provide the structural stability necessary for homeologous recombination.</p><p id="p-56">The conservation of duplicate loci on HRTAs does not appear to be due to current chromosome structure, because the HRTAs do not display homologous chromosome arm fusions across species (with the exception of AC03/Ssa07/Omy21/Ots15). Therefore, the slower diploidization in these regions may be attributed to some aspect of their evolutionary past. For instance, certain fusions may have arisen in the common ancestor of all these salmonid lineages following the Ss4R. This could have provided the HRTA homeolog pairs with the ability to form multivalents during meiosis, and undergo residual tetrasomy, thereby slowing diploidization rates. A large number of species-specific fusion/fission events since the more recent divergence of salmonids (<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Macqueen and Johnston 2014</a>) could disjoin previous chromosome structures, and also explain why the seven HRTA homeolog pairs preserve duplicate loci in multiple species, despite the large variation in karyotypic structures across current salmonid species.</p><p id="p-57">To search for possible duplicate SNP positions that could have arisen from the Ss4R event, we reciprocally BLASTn aligned all linkage map SNP markers against each other, and retained those duplicate pairs that shared ≥95% identity to one another, as well as retaining ≥95% of an overlap in their length distributions. Surprisingly, of the 362 duplicate pairs identified, only 18 pairs were interchromosomal duplicate pairs. Of these 18 pairs, only three were considered to be possible WGD paralogs, as the other 15 pairs involved either one or both SNPs with TE-specific sequence (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038026/-/DC1/FileS4.xlsx">File S4</a>). One of the potential paralogs that was not associated with TE, involved markers from identified homeolog pair of AC03q/37. Two of the potential paralogs not associated with TEs involved markers from the linkage groups AC06p and AC32, which is not an identified homeolog region in the Arctic charr map. However, these two paralog sets show homology to the Ssa02p/05q paralogs, and therefore may indicate a small region of previously unidentified homeology.</p><p id="p-58">The vast majority of duplicate pair markers appear to be intrachromosomal duplicates (95.1%) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038026/-/DC1/FileS4.xlsx">File S4</a>). To assess whether chromosome structure may have had an influence on the distribution of these duplicates, we compared the proportion of duplicated SNP markers between metacentric <em>vs.</em> acrocentric type linkage groups. Duplicates within AC04p and AC04q were included in the metacentric grouping, as well as duplicates within AC17, given that this acrocentric appears to be composed of fused chromosome arms. No differences were detected in the distribution of duplicates between the chromosome types (<em>P</em> &gt; 0.05) (<a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a>).</p><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470374/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470374/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6</span> <span class="caption-title">Arctic charr linkage map location of duplicate pair SNPs that shared ≥95% identity with one another</span><div class="sb-div caption-clear"></div></div></div><p id="p-61">Duplicate marker pairs do not map next to one another within linkage groups, suggesting that they are not tandem repeats. These marker duplicates may be part of larger segmental duplicate blocks that can vary in size from 1 to 400 kb (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Mendivil Ramos and Ferrier 2012</a>), but alignment of these regions to more complete scaffold contigs would be needed to test this idea. The reason so few apparent WGD paralog regions were detected is likely due to the high stringency in which duplicated regions were identified. Interchromosomal 4R paralogs have average identity levels ranging from 86 to 90% (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Moghadam <em>et al.</em> 2011</a>; <a id="xref-ref-5-9" class="xref-bibr" href="#ref-5">Berthelot <em>et al.</em> 2014</a>), with average levels up to 96% for protein-coding duplicate regions (<a id="xref-ref-5-10" class="xref-bibr" href="#ref-5">Berthelot <em>et al.</em> 2014</a>). Hence, an analysis using lowered stringency cut-offs will likely reveal higher frequencies of WGD paralog regions than those reported here.</p><p id="p-62">Many of the duplicate markers appear to align to identical locations within both the rainbow trout and Atlantic salmon genomes, but in opposite strand orientations (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038026/-/DC1/FileS4.xlsx">File S4</a>). Upon closer inspection of these duplicates, we observed that several reads aligning to each contig cluster were longer in either the 5′ or 3′ direction, and these reads spanned an <em>Eco</em>T22I cut-site on either side. Both the 3′ side of the forward strand, and 5′ side of the reverse strand were bordered by the 5 bp signature of a DNA strand cut by EcoT22I. This suggests that an internal cut-site may have been skipped due to an epigenetic modification, and that an additional EcoT22I cut-site, 20–30 bp away, flanked the uncut site.</p><p id="p-63">We queried all 425,391,431 reads obtained from the fish used in this study, and determined that the presence of uncut EcoT22I sites averaged 2.8%, while those found in the paired duplicates having extended reads ranged from 4 to 6% supporting the suggestion that these regions may be more epigenetically modified. However, many of these reads also appeared as chimeric religations, highlighting that enzymes susceptible to epigenetic modification may lead to genotyping errors (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Jiang <em>et al.</em> 2016</a>).</p></div><div id="sec-21" class="subsection"><h3>Distribution of Arctic charr duplicated markers in relation to the Atlantic salmon genome</h3><p id="p-64">BLASTn alignments of the Arctic charr duplicated markers were not uniformly distributed along the length of Atlantic salmon chromosomes from the <a id="xref-ref-25-17" class="xref-bibr" href="#ref-25">Lien <em>et al.</em> (2016)</a> assembly. Significantly more Arctic charr PSVs aligned to the 4th quarter (telomeric ends) of chromosome arms (<em>P</em> &lt; 0.0001) (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). This suggests that PSVs are preserved near the telomeres in the regions of chromosomes that undergo residual tetrasomy (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Wright <em>et al.</em> 1983</a>; <a id="xref-ref-2-7" class="xref-bibr" href="#ref-2">Allendorf <em>et al.</em> 2015</a>; <a id="xref-ref-29-5" class="xref-bibr" href="#ref-29">May and Delany 2015</a>). Previous studies in salmonids have also observed that duplicate loci are present in higher frequencies near the telomeres (<a id="xref-ref-44-5" class="xref-bibr" href="#ref-44">Sakamoto <em>et al.</em> 2000</a>; <a id="xref-ref-6-9" class="xref-bibr" href="#ref-6">Brieuc <em>et al.</em> 2014</a>; <a id="xref-ref-21-5" class="xref-bibr" href="#ref-21">Kodama <em>et al.</em> 2014</a>; <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Larson <em>et al.</em> 2015</a>; <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">McKinney <em>et al.</em> 2015</a>; <a id="xref-ref-51-5" class="xref-bibr" href="#ref-51">Waples <em>et al.</em> 2015</a>). There was also a significant reduction of linkage map markers with top Blast hits in the 4th quarter of chromosomes, which suggests that residual tetrasomy may generate high numbers of duplicate loci that cause telomeric regions to be under-represented with markers (<a id="xref-ref-2-8" class="xref-bibr" href="#ref-2">Allendorf <em>et al.</em> 2015</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/543/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of the top Blast hits of Arctic charr linkage map SNPs and PSVs across Atlantic salmon chromosome arms. q1 is the quarter closest to the centromere, q4 is the quarter closest to the telomere." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-148801367" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Distribution of the top Blast hits of Arctic charr linkage map SNPs and PSVs across Atlantic salmon chromosome arms. q1 is the quarter closest to the centromere, q4 is the quarter closest to the telomere.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/543/F5.medium.gif" width="440" height="288"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/2/543/F5.medium.gif" width="440" height="288"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/543/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/543/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470530" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-65" class="first-child">Distribution of the top Blast hits of Arctic charr linkage map SNPs and PSVs across Atlantic salmon chromosome arms. q1 is the quarter closest to the centromere, q4 is the quarter closest to the telomere.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-22" class="subsection"><h3>Transposon activity</h3><p id="p-66">Recent studies of salmonid genomes have observed lower instances of sequence homology to TEs within genomic regions characterized as preserving tetrasomy (<a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">McKinney <em>et al.</em> 2015</a>; <a id="xref-ref-51-6" class="xref-bibr" href="#ref-51">Waples <em>et al.</em> 2015</a>; <a id="xref-ref-25-18" class="xref-bibr" href="#ref-25">Lien <em>et al.</em> 2016</a>). We assessed TE distributions through BLASTn alignments to the Atlantic salmon genome to determine if chromosomes homologous to HRTA had lower levels of TE alignments. Of the 4405 linkage map SNPs, 1608 (36.5%) had homology with transposable elements in Repbase Update’s list of known vertebrate TEs (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Jurka <em>et al.</em> 2005</a>). This was a lower proportion of TE hits than expected as repetitive elements comprise 58–60% of the Atlantic salmon genome, and suggests that our dataset underrepresents the proportion of repetitive elements in Arctic charr. We detected a small but significant reduction in TE activity between HRTAs (30.9% of SNPs on these arms had significant TE hits), and non-HRTA chromosome arms (35.8%) (<em>P</em> = 0.0009) (<a id="xref-table-wrap-7-1" class="xref-table" href="#T7">Table 7</a>). Using the Atlantic salmon genome as a scaffold, we detected no significant difference in the frequency of TE hits between the telomeric and centromeric regions of chromosomes (<em>P</em> = 0.1322) (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>).</p><div id="T7" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470313/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470313/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 7</span> <span class="caption-title">TE distribution of the SNPs aligning to the Atlantic salmon chromosome arms, and PSVs markers not found in the linkage map</span><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/543/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Proportion of Arctic charr Map SNPs and PSVs with TE hits across the length of chromosome arms. SNPs were grouped based on their alignment to Atlantic salmon chromosome arms. q1 is the quarter closest to the centromere, q4 is the quarter closest to the telomere. Atlantic salmon chromosome arms were grouped according to whether they were HRTAs (A) or non-HRTAs (B)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-148801367" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Proportion of Arctic charr Map SNPs and PSVs with TE hits across the length of chromosome arms. SNPs were grouped based on their alignment to Atlantic salmon chromosome arms. q1 is the quarter closest to the centromere, q4 is the quarter closest to the telomere. Atlantic salmon chromosome arms were grouped according to whether they were HRTAs (A) or non-HRTAs (B).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/543/F6.medium.gif" width="440" height="395"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/7/2/543/F6.medium.gif" width="440" height="395"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/543/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/543/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470316" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-68" class="first-child">Proportion of Arctic charr Map SNPs and PSVs with TE hits across the length of chromosome arms. SNPs were grouped based on their alignment to Atlantic salmon chromosome arms. q1 is the quarter closest to the centromere, q4 is the quarter closest to the telomere. Atlantic salmon chromosome arms were grouped according to whether they were HRTAs (A) or non-HRTAs (B).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-23" class="subsection"><h3>Conclusion</h3><p id="p-69">We have presented a SNP-based linkage map of the Arctic charr genome, which is comprised of 4508 markers spanning 39 linkage groups. The map was used to identify the chromosome type of each linkage group, and the homologous chromosome arms in other salmonid species. Using data from the Atlantic salmon genome, we have identified putative homeologous arm pairs in Arctic charr. Based on the distribution of PSV, we suggest that genomic architecture is influencing diploidization rate in the Arctic charr genome, with higher levels of duplicate loci being preserved on HRTAs and lower numbers of duplicate loci preserved on AHPs. Transposon activity was also quantified, but we failed to detect a strong influence of genomic architecture on TE distribution. Pseudolinkage was also detected in both the male and female parents, and this involved two HRTA homeolog pairs (AC01q/21 and AC13q/34). This map also characterized the genome of a salmonid species with a more basal karyotype, and shows how these differences in genomic architecture have influenced diploidization.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-70">We thank Claude Pelletier from the Coastal Zones Research Institute, Shippagan, New Brunswick, for providing the mapping family. The project was overseen and coordinated by the Coastal Zones Research Institute and we thank Andre Dumas and Rodrigue Yossa for their important role in this regard. This work was funded by Natural Sciences and Engineering Research Council of Canada (NSERC) (Discovery Grant Program), and the Atlantic Innovation Fund/Atlantic Canada Opportunities Agency for the project “Aquaculture development and profitable commercialization of Arctic charr in Canada.”</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-17"><p id="p-71">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038026/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.038026/-/DC1</a>.</p></li><li class="fn" id="fn-18"><p id="p-72"><em>Communicating editor: R. Houston</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-07-29"><span class="received-label">Received </span>July 29, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-12-04"><span class="accepted-label">Accepted </span>December 4, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Nugent <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-book" id="cit-7.2.543.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Allendorf</span>  <span class="cit-name-given-names">F. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorgaard</span>  <span class="cit-name-given-names">G. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Tetraploidy and the evolution of salmonid fishes</span>, pp. <span class="cit-fpage">1</span>–<span class="cit-lpage">46</span> in <span class="cit-source"><em>Evolutionary Genetics of Fishes</em></span>, edited by <span class="cit-ed"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">B. J.</span></span>. <span class="cit-publ-name">Plenum Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.543.2" data-doi="10.1093/jhered/esv015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Allendorf</span>  <span class="cit-name-given-names">F. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bassham</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cresko</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Limborg</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">L. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Effects of crossovers between homeologs on inheritance and population genomics in polyploid-derived salmonid fishes.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">217</span>–<span class="cit-lpage">227</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BHered.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252Fesv015%26rft_id%253Dinfo%253Apmid%252F25838153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamhlcmVkIjtzOjU6InJlc2lkIjtzOjk6IjEwNi8zLzIxNyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi81NDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.543.3" data-doi="10.1006/jmbi.1990.9999"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gish</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipman</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Basic local alignment search tool.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">215</span>: <span class="cit-fpage">403</span>–<span class="cit-lpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DAltschul%26rft.auinit1%253DS.%2BF.%26rft.volume%253D215%26rft.issue%253D3%26rft.spage%253D403%26rft.epage%253D410%26rft.atitle%253DBasic%2Blocal%2Balignment%2Bsearch%2Btool.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1990.9999%26rft_id%253Dinfo%253Apmid%252F2231712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1990.9999&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2231712&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F543.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990ED16700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.543.4" data-doi="10.1073/pnas.0707158104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bergthorsson</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andersson</span>  <span class="cit-name-given-names">D. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roth</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Ohno’s dilemma: evolution of new genes under continuous selection.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">17004</span>–<span class="cit-lpage">17009</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0707158104%26rft_id%253Dinfo%253Apmid%252F17942681%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA0LzQzLzE3MDA0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzU0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.543.5" data-doi="10.1038/ncomms4657"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Berthelot</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brunet</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chalopin</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Juanchich</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernard</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The rainbow trout genome provides novel insights into evolution after whole-genome duplication in vertebrates.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">3657</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D5%26rft.spage%253D3657%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms4657%26rft_id%253Dinfo%253Apmid%252F24755649%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms4657&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24755649&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F543.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.543.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brieuc</span>  <span class="cit-name-given-names">M. S. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waters</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naish</span>  <span class="cit-name-given-names">K. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A dense linkage map for chinook salmon (<em>Oncorhynchus tshawytscha</em>) reveals variable chromosomal divergence after an ancestral whole genome duplication event.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">447</span>–<span class="cit-lpage">460</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D4%26rft.spage%253D447%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.543.7" data-doi="10.1038/nrg1711"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Comai</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The advantages and disadvantages of being polyploid.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">836</span>–<span class="cit-lpage">846</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DComai%26rft.auinit1%253DL.%26rft.volume%253D6%26rft.issue%253D11%26rft.spage%253D836%26rft.epage%253D846%26rft.atitle%253DThe%2Badvantages%2Band%2Bdisadvantages%2Bof%2Bbeing%2Bpolyploid.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1711%26rft_id%253Dinfo%253Apmid%252F16304599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1711&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16304599&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F543.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232990900014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-web" id="cit-7.2.543.8"><div class="cit-metadata unstructured">Danzmann, R. G., 2016 LINKMFEX: linkage analysis package for outcrossed families with male female exchange of the mapping parent. (Click on links to the ‘Software’ and ‘LINKMFEX’ folders.) Available at: <a href="http://www.uoguelph.ca/%7erdanzman/">http://www.uoguelph.ca/∼rdanzman/</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.543.9" data-doi="10.1023/A:1013713431255"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Danzmann</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gharbi</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Gene mapping in fishes: a means to an end.</span> <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">3</span>–<span class="cit-lpage">23</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.stitle%253DGenetica%26rft.aulast%253DDanzmann%26rft.auinit1%253DR.%2BG.%26rft.volume%253D111%26rft.issue%253D1-3%26rft.spage%253D3%26rft.epage%253D23%26rft.atitle%253DGene%2Bmapping%2Bin%2Bfishes%253A%2Ba%2Bmeans%2Bto%2Ban%2Bend.%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1013713431255%26rft_id%253Dinfo%253Apmid%252F11841175%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1013713431255&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11841175&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F543.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173270000002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.543.10" data-doi="10.1186/1471-2164-9-557"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Danzmann</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davidson</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferguson</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gharbi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koop</span>  <span class="cit-name-given-names">B. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Distribution of ancestral proto-Actinopterygian chromosome arms within the genomes of 4R-derivative salmonid fishes (Rainbow trout and Atlantic salmon).</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">557</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DDanzmann%26rft.auinit1%253DR.%2BG.%26rft.volume%253D9%26rft.spage%253D557%26rft.epage%253D557%26rft.atitle%253DDistribution%2Bof%2Bancestral%2Bproto-Actinopterygian%2Bchromosome%2Barms%2Bwithin%2Bthe%2Bgenomes%2Bof%2B4R-derivative%2Bsalmonid%2Bfishes%2B%2528Rainbow%2Btrout%2Band%2BAtlantic%2Bsalmon%2529.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-9-557%26rft_id%253Dinfo%253Apmid%252F19032764%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-9-557&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19032764&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F543.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.543.11" data-doi="10.1186/1471-2164-8-422"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Boer</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yazawa</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davidson</span>  <span class="cit-name-given-names">W. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koop</span>  <span class="cit-name-given-names">B. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Bursts and horizontal evolution of DNA transposons in the speciation of pseudotetraploid salmonids.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">422</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253Dde%2BBoer%26rft.auinit1%253DJ.%2BG.%26rft.volume%253D8%26rft.spage%253D422%26rft.epage%253D422%26rft.atitle%253DBursts%2Band%2Bhorizontal%2Bevolution%2Bof%2BDNA%2Btransposons%2Bin%2Bthe%2Bspeciation%2Bof%2Bpseudotetraploid%2Bsalmonids.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-8-422%26rft_id%253Dinfo%253Apmid%252F18021408%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-8-422&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18021408&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F543.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.543.12" data-doi="10.1371/journal.pone.0016163"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D6%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0016163%26rft_id%253Dinfo%253Apmid%252F21949733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0016163&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21949733&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F543.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.543.13" data-doi="10.1371/journal.pone.0090346"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casstevens</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harriman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">TASSEL-GBS: a high capacity genotyping by sequencing analysis pipeline.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e90346</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De90346%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0090346%26rft_id%253Dinfo%253Apmid%252F24587335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0090346&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24587335&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F543.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.543.14" data-doi="10.1186/1471-2164-15-166"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gonen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lowe</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cezard</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gharbi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bishop</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Linkage maps of the Atlantic salmon (<em>Salmo salar</em>) genome derived from RAD sequencing.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">166</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D166%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-166%26rft_id%253Dinfo%253Apmid%252F24571138%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-166&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24571138&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F543.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.543.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hartley</span>  <span class="cit-name-given-names">S. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Chromosomes and constitutive heterochromatin distribution in Arctic charr, <em>Salvelinus alpinus</em> (L.) (Pisces: <em>Salmonidae</em>).</span> <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">161</span>–<span class="cit-lpage">166</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.volume%253D79%26rft.spage%253D161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.543.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hermansen</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hvidsten</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sandve</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liberles</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Extracting functional trends from whole genome duplication events using comparative genomics.</span> <abbr class="cit-jnl-abbrev">Biol. Proced. Online</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BProced.%2BOnline%26rft.volume%253D18%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.543.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michal</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genome wide sampling sequencing for SNP genotyping: methods, challenges and future development.</span> <abbr class="cit-jnl-abbrev">Int. J. Biol. Sci.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">100</span>–<span class="cit-lpage">108</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BBiol.%2BSci.%26rft.volume%253D12%26rft.spage%253D100%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.543.18" data-doi="10.1159/000084979"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jurka</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kapitonov</span>  <span class="cit-name-given-names">V. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pavlicek</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klonowski</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kohany</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Repbase update, a database of eukaryotic repetitive elements.</span> <abbr class="cit-jnl-abbrev">Cytogenet. Genome Res.</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">462</span>–<span class="cit-lpage">467</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCytogenetic%2Band%2Bgenome%2Bresearch%26rft.stitle%253DCytogenet%2BGenome%2BRes%26rft.aulast%253DJurka%26rft.auinit1%253DJ.%26rft.volume%253D110%26rft.issue%253D1-4%26rft.spage%253D462%26rft.epage%253D467%26rft.atitle%253DRepbase%2BUpdate%252C%2Ba%2Bdatabase%2Bof%2Beukaryotic%2Brepetitive%2Belements.%26rft_id%253Dinfo%253Adoi%252F10.1159%252F000084979%26rft_id%253Dinfo%253Apmid%252F16093699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1159/000084979&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16093699&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F543.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231064600047&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.543.19" data-doi="10.1038/nature05846"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kasahara</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naruse</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sasaki</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakatani</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The medaka draft genome and insights into vertebrate genome evolution.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">447</span>: <span class="cit-fpage">714</span>–<span class="cit-lpage">719</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKasahara%26rft.auinit1%253DM.%26rft.volume%253D447%26rft.issue%253D7145%26rft.spage%253D714%26rft.epage%253D719%26rft.atitle%253DThe%2Bmedaka%2Bdraft%2Bgenome%2Band%2Binsights%2Binto%2Bvertebrate%2Bgenome%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05846%26rft_id%253Dinfo%253Apmid%252F17554307%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature05846&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17554307&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F543.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247030700044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.543.20" data-doi="10.1126/science.1089670"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kazazian</span>  <span class="cit-name-given-names">H. H.</span>  <span class="cit-name-suffix">Jr</span>.</span></li></ol><cite>., <span class="cit-pub-date">2004</span> <span class="cit-article-title">Mobile elements: drivers of genome evolution.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">303</span>: <span class="cit-fpage">1626</span>–<span class="cit-lpage">1632</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DKazazian%26rft.auinit1%253DH.%2BH.%26rft.volume%253D303%26rft.issue%253D5664%26rft.spage%253D1626%26rft.epage%253D1632%26rft.atitle%253DMobile%2BElements%253A%2BDrivers%2Bof%2BGenome%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1089670%26rft_id%253Dinfo%253Apmid%252F15016989%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDMvNTY2NC8xNjI2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzU0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.543.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kodama</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brieuc</span>  <span class="cit-name-given-names">M. S. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Devlin</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hard</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naish</span>  <span class="cit-name-given-names">K. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Comparative mapping between coho salmon (<em>Oncorhynchus kisutch</em>) and three other salmonids suggests a role for chromosomal rearrangements in the retention of duplicated regions following a whole genome duplication event.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1717</span>–<span class="cit-lpage">1730</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D4%26rft.spage%253D1717%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.543.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Küttner</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nilsson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skulason</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gunnarsson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferguson</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Sex chromosome polymorphisms in Arctic charr and their evolutionary origins.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">852</span>–<span class="cit-lpage">861</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DKuttner%26rft.auinit1%253DE.%26rft.volume%253D54%26rft.issue%253D10%26rft.spage%253D852%26rft.epage%253D861%26rft.atitle%253DSex%2Bchromosome%2Bpolymorphisms%2Bin%2BArctic%2Bcharr%2Band%2Btheir%2Bevolutionary%2Borigins.%26rft_id%253Dinfo%253Apmid%252F21970434%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21970434&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F543.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.543.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Larson</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McKinney</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Limborg</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Everett</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">L. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Identification of multiple QTL hotspots in sockeye salmon (<em>Oncorhynchus nerka</em>) using genotyping-by-sequencing and a dense linkage map.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">122</span>–<span class="cit-lpage">133</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BHered.%26rft.volume%253D107%26rft.spage%253D122%26rft_id%253Dinfo%253Apmid%252F26712859%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=26712859&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F543.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.543.24" data-doi="10.1186/1471-2164-12-615"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lien</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gidskehaug</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berg</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A dense SNP-based linkage map for Atlantic salmon (<em>Salmo salar</em>) reveals extended chromosome homeologies and striking differences in sex-specific recombination patterns.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">615</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DLien%26rft.auinit1%253DS.%26rft.volume%253D12%26rft.spage%253D615%26rft.epage%253D615%26rft.atitle%253DA%2Bdense%2BSNP-based%2Blinkage%2Bmap%2Bfor%2BAtlantic%2Bsalmon%2B%2528Salmo%2Bsalar%2529%2Breveals%2Bextended%2Bchromosome%2Bhomeologies%2Band%2Bstriking%2Bdifferences%2Bin%2Bsex-specific%2Brecombination%2Bpatterns.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-615%26rft_id%253Dinfo%253Apmid%252F22182215%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-615&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22182215&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F543.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.543.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lien</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koop</span>  <span class="cit-name-given-names">B. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sandve</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matthew</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The Atlantic salmon genome provides insights into rediploidization.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">533</span>: <span class="cit-fpage">200</span>–<span class="cit-lpage">205</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D533%26rft.spage%253D200%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.543.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Limborg</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McKinney</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">L. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Recombination patterns reveal information about centromere location on linkage maps.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">655</span>–<span class="cit-lpage">661</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D16%26rft.spage%253D655%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.543.27" data-doi="10.1098/rspb.2013.2881"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Macqueen</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">I. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A well-constrained estimate for the timing of the salmonid whole genome duplication reveals major decoupling from species diversification.</span> <abbr class="cit-jnl-abbrev">Proc. Biol. Sci.</abbr> <span class="cit-vol">281</span>: <span class="cit-fpage">20132881</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BBiol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2013.2881%26rft_id%253Dinfo%253Apmid%252F24452024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Nzoicm95cHJzYiI7czo1OiJyZXNpZCI7czoxNzoiMjgxLzE3NzgvMjAxMzI4ODEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzIvNTQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.543.28" data-doi="10.1111/j.2007.0018-0661.02000.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Margarido</span>  <span class="cit-name-given-names">G. R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Souza</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia</span>  <span class="cit-name-given-names">A. A. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Onemap: software for genetic mapping in outcrossing species.</span> <abbr class="cit-jnl-abbrev">Hereditas</abbr> <span class="cit-vol">144</span>: <span class="cit-fpage">78</span>–<span class="cit-lpage">79</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHereditas%26rft.stitle%253DHereditas%26rft.aulast%253DMargarido%26rft.auinit1%253DG.%2BR.%26rft.volume%253D144%26rft.issue%253D3%26rft.spage%253D78%26rft.epage%253D79%26rft.atitle%253DOneMap%253A%2Bsoftware%2Bfor%2Bgenetic%2Bmapping%2Bin%2Boutcrossing%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.2007.0018-0661.02000.x%26rft_id%253Dinfo%253Apmid%252F17663699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.2007.0018-0661.02000.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17663699&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F543.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248257800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.543.29" data-doi="10.1093/jhered/esv056"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">May</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delany</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Meiotic models to explain classical linkage, pseudolinkage, and chromosomal pairing in tetraploid derivative salmonid genomes: II. Wright is still right.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">762</span>–<span class="cit-lpage">766</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BHered.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252Fesv056%26rft_id%253Dinfo%253Apmid%252F26320244%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamhlcmVkIjtzOjU6InJlc2lkIjtzOjk6IjEwNi82Lzc2MiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi81NDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.543.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKinney</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larson</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gomez-Uchida</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Limborg</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">An integrated linkage map reveals candidate genes underlying adaptive variation in chinook salmon (<em>Oncorhynchus tshawytscha</em>).</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">769</span>–<span class="cit-lpage">783</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D16%26rft.spage%253D769%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.543.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mendivil Ramos</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferrier</span>  <span class="cit-name-given-names">D. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Mechanisms of gene duplication and translocation and progress towards understanding their relative contributions to animal genome evolution.</span> <abbr class="cit-jnl-abbrev">Int. J. Evol. Biol.</abbr> <span class="cit-vol">2012</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BEvol.%2BBiol.%26rft.volume%253D2012%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.543.32" data-doi="10.1007/s00239-004-0230-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moghadam</span>  <span class="cit-name-given-names">H. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferguson</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danzmann</span>  <span class="cit-name-given-names">R. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Evidence for <em>Hox</em> gene duplication in rainbow trout (<em>Oncorhynchus mykiss</em>): a tetraploid model species.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">804</span>–<span class="cit-lpage">818</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DMoghadam%26rft.auinit1%253DH.%2BK.%26rft.volume%253D61%26rft.issue%253D6%26rft.spage%253D804%26rft.epage%253D818%26rft.atitle%253DEvidence%2Bfor%2BHox%2Bgene%2Bduplication%2Bin%2Brainbow%2Btrout%2B%2528Oncorhynchus%2Bmykiss%2529%253A%2Ba%2Btetraploid%2Bmodel%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00239-004-0230-5%26rft_id%253Dinfo%253Apmid%252F16315109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00239-004-0230-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16315109&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F543.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233630700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.543.33" data-doi="10.1111/j.1095-8649.2007.01598.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moghadam</span>  <span class="cit-name-given-names">H. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferguson</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danzmann</span>  <span class="cit-name-given-names">R. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Linkage variation at the sex-determining locus within Fraser strain Arctic charr (<em>Salvelinus alpinus</em>).</span> <abbr class="cit-jnl-abbrev">J. Fish Biol.</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">294</span>–<span class="cit-lpage">301</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BFish%2BBiol.%26rft.volume%253D71%26rft.spage%253D294%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1095-8649.2007.01598.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1095-8649.2007.01598.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.543.34" data-doi="10.1111/j.1095-8649.2011.03030.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moghadam</span>  <span class="cit-name-given-names">H. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferguson</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danzmann</span>  <span class="cit-name-given-names">R. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Whole genome duplication: challenges and considerations associated with sequence orthology assignment in Salmoninae.</span> <abbr class="cit-jnl-abbrev">J. Fish Biol.</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">561</span>–<span class="cit-lpage">574</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BFish%2BBiology%26rft.stitle%253DJournal%2Bof%2BFish%2BBiology%26rft.aulast%253DMoghadam%26rft.auinit1%253DH.%2BK.%26rft.volume%253D79%26rft.issue%253D3%26rft.spage%253D561%26rft.epage%253D574%26rft.atitle%253DWhole%2Bgenome%2Bduplication%253A%2Bchallenges%2Band%2Bconsiderations%2Bassociated%2Bwith%2Bsequence%2Borthology%2Bassignment%2Bin%2BSalmoninae.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1095-8649.2011.03030.x%26rft_id%253Dinfo%253Apmid%252F21884100%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1095-8649.2011.03030.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21884100&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F543.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.543.35" data-doi="10.1101/gr.6316407"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nakatani</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takeda</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kohara</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morishita</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Reconstruction of the vertebrate ancestral genome reveals dynamic genome reorganization in early vertebrates.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1254</span>–<span class="cit-lpage">1265</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.6316407%26rft_id%253Dinfo%253Apmid%252F17652425%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE3LzkvMTI1NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi81NDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.543.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Norman</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danzmann</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glebe</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferguson</span>  <span class="cit-name-given-names">M. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The genetic basis of salinity tolerance traits in Arctic charr (<em>Salvelinus alpinus</em>).</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">81</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DNorman%26rft.auinit1%253DJ.%2BD.%26rft.volume%253D12%26rft.spage%253D81%26rft.epage%253D81%26rft.atitle%253DThe%2Bgenetic%2Bbasis%2Bof%2Bsalinity%2Btolerance%2Btraits%2Bin%2BArctic%2Bcharr%2B%2528Salvelinus%2Balpinus%2529.%26rft_id%253Dinfo%253Apmid%252F21936917%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21936917&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F543.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.543.37" data-doi="10.1186/1471-2164-13-420"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Norman</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glebe</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferguson</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danzmann</span>  <span class="cit-name-given-names">R. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic arrangement of salinity tolerance QTLs in salmonids: a comparative analysis of Atlantic salmon (<em>Salmo salar</em>) with Arctic charr (<em>Salvelinus alpinus</em>) and rainbow trout (<em>Oncorhynchus mykiss</em>).</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">420</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DNorman%26rft.auinit1%253DJ.%2BD.%26rft.volume%253D13%26rft.spage%253D420%26rft.epage%253D420%26rft.atitle%253DGenomic%2Barrangement%2Bof%2Bsalinity%2Btolerance%2BQTLs%2Bin%2Bsalmonids%253A%2BA%2Bcomparative%2Banalysis%2Bof%2BAtlantic%2Bsalmon%2B%2528Salmo%2Bsalar%2529%2Bwith%2BArctic%2Bcharr%2B%2528Salvelinus%2Balpinus%2529%2Band%2Brainbow%2Btrout%2B%2528Oncorhynchus%2Bmykiss%2529.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-420%26rft_id%253Dinfo%253Apmid%252F22916800%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-420&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22916800&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F543.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.543.38" data-doi="10.1152/physiolgenomics.00105.2013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Norman</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferguson</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danzmann</span>  <span class="cit-name-given-names">R. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Transcriptomics of salinity tolerance capacity in Arctic charr (<em>Salvelinus alpinus</em>): a comparison of gene expression profiles between divergent QTL genotypes.</span> <abbr class="cit-jnl-abbrev">Physiol. Genomics</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">123</span>–<span class="cit-lpage">137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhysiol.%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1152%252Fphysiolgenomics.00105.2013%26rft_id%253Dinfo%253Apmid%252F24368751%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTU6InBoeXNpb2xnZW5vbWljcyI7czo1OiJyZXNpZCI7czo4OiI0Ni80LzEyMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi81NDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-book" id="cit-7.2.543.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ohno</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span> <span class="cit-source"><em>Evolution by Gene Duplication</em>.</span> <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.543.40" data-doi="10.1186/1471-2164-14-570"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ostberg</span>  <span class="cit-name-given-names">C. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hauser</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">V. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garza</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naish</span>  <span class="cit-name-given-names">K. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Chromosome rearrangements, recombination suppression, and limited segregation distortion in hybrids between Yellowstone cutthroat trout (<em>Oncorhynchus clarkii bouvieri</em>) and rainbow trout (<em>O. mykiss</em>).</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">570</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D14%26rft.spage%253D570%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-570%26rft_id%253Dinfo%253Apmid%252F23968234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-570&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23968234&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F543.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.543.41" data-doi="10.1017/S1464793100005613"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ráb</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Chromosome evolution in the Salmonidae (Pisces): an update.</span> <abbr class="cit-jnl-abbrev">Biol. Rev. Camb. Philos. Soc.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiological%2BReviews%2B%2528Cambridge%253B%2BPrint%2529%26rft.stitle%253DBiological%2BReviews%2B%2528Cambridge%253B%2BPrint%2529%26rft.aulast%253DPhillips%26rft.auinit1%253DR.%26rft.volume%253D76%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D25%26rft.atitle%253DChromosome%2Bevolution%2Bin%2Bthe%2BSalmonidae%2B%2528Pisces%2529%253A%2Ban%2Bupdate.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS1464793100005613%26rft_id%253Dinfo%253Apmid%252F11325050%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S1464793100005613&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11325050&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F543.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.543.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matsuoka</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reed</span>  <span class="cit-name-given-names">K. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Characterization of charr chromosomes using fluorescence in situ hybridization.</span> <abbr class="cit-jnl-abbrev">Environ. Biol. Fishes</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">223</span>–<span class="cit-lpage">228</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEnviron.%2BBiol.%2BFishes%26rft.volume%253D64%26rft.spage%253D223%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.543.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rexroad</span>  <span class="cit-name-given-names">C. E.</span>  <span class="cit-name-suffix">III</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palti</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gahr</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vallejo</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A second generation genetic map for rainbow trout (<em>Oncorhynchus mykiss</em>).</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DZhou%26rft.auinit1%253DY.%26rft.volume%253D9%26rft.spage%253D1%26rft.epage%253D1%26rft.atitle%253DMaximum%2Blikelihood%2Bestimates%2Bof%2Btwo-locus%2Brecombination%2Bfractions%2Bunder%2Bsome%2Bnatural%2Binequality%2Brestrictions.%26rft_id%253Dinfo%253Apmid%252F18173855%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18173855&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F543.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.543.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sakamoto</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danzmann</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gharbi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howard</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ozaki</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">A microsatellite linkage map of rainbow trout (<em>Oncorhynchus mykiss</em>) characterized by large sex-specific differences in recombination rates.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">1331</span>–<span class="cit-lpage">1345</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSakamoto%26rft.auinit1%253DT.%26rft.volume%253D155%26rft.issue%253D3%26rft.spage%253D1331%26rft.epage%253D1345%26rft.atitle%253DA%2BMicrosatellite%2BLinkage%2BMap%2Bof%2BRainbow%2BTrout%2B%2528Oncorhynchus%2Bmykiss%2529%2BCharacterized%2Bby%2BLarge%2BSex-Specific%2BDifferences%2Bin%2BRecombination%2BRates%26rft_id%253Dinfo%253Apmid%252F10880492%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1NS8zLzEzMzEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzIvNTQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.543.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schrader</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ence</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zimin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Transposable element islands facilitate adaptation to novel environments in an invasive species.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1717</span>–<span class="cit-lpage">1730</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D5%26rft.spage%253D1717%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.543.46" data-doi="10.1038/ng.2568"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuraku</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holt</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sauka-Spengler</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Sequencing of the sea lamprey (<em>Petromyzon marinus</em>) genome provides insights into vertebrate evolution.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">415</span>–<span class="cit-lpage">421</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D45%26rft.spage%253D415%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2568%26rft_id%253Dinfo%253Apmid%252F23435085%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2568&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23435085&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F543.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.543.47" data-doi="10.1186/s12864-015-1830-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Staton</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burke</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Evolutionary transitions in the <em>Asteraceae</em> coincide with marked shifts in transposable element abundance.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">623</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D623%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-1830-8%26rft_id%253Dinfo%253Apmid%252F26290182%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-1830-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26290182&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F543.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.543.48" data-doi="10.1146/annurev.genet.38.072902.092831"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raes</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Duplication and divergence: the evolution of new genes and old ideas.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">615</span>–<span class="cit-lpage">643</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DTaylor%26rft.auinit1%253DJ.%2BS.%26rft.volume%253D38%26rft.spage%253D615%26rft.epage%253D643%26rft.atitle%253DDuplication%2Band%2Bdivergence%253A%2Bthe%2Bevolution%2Bof%2Bnew%2Bgenes%2Band%2Bold%2Bideas.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.38.072902.092831%26rft_id%253Dinfo%253Apmid%252F15568988%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genet.38.072902.092831&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15568988&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F543.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226244600019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.543.49" data-doi="10.1186/1471-2164-12-68"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Timusk</span>  <span class="cit-name-given-names">E. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferguson</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moghadam</span>  <span class="cit-name-given-names">H. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Norman</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome evolution in the fish family Salmonidae: generation of a brook charr genetic map and comparisons among charrs (Arctic charr and brook charr) with rainbow trout.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">68</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DPark%26rft.auinit1%253DE.%2BH.%26rft.volume%253D12%26rft.spage%253D68%26rft.epage%253D68%26rft.atitle%253DDepletion%2Bof%2BeIF4G%2Bfrom%2Byeast%2Bcells%2Bnarrows%2Bthe%2Brange%2Bof%2Btranslational%2Befficiencies%2Bgenome-wide.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-68%26rft_id%253Dinfo%253Apmid%252F21269496%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-68&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21269496&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F543.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.543.50"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsai</span>  <span class="cit-name-given-names">H. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robledo</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lowe</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bekaert</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taggart</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Construction and annotation of a high density SNP linkage map of the Atlantic salmon (<em>Salmo salar</em>) genome.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">2173</span>–<span class="cit-lpage">2179</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D6%26rft.spage%253D2173%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.543.51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Waples</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">L. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Linkage mapping with paralogs exposes regions of residual tetrasomic inheritance in chum salmon (<em>Oncorhynchus keta</em>).</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">17</span>–<span class="cit-lpage">28</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D16%26rft.spage%253D17%26rft_id%253Dinfo%253Apmid%252F25712438%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=25712438&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F543.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.543.52" data-doi="10.1101/gr.578503"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Woram</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gharbi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sakamoto</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoyheim</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holm</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Comparative genome analysis of the primary sex-determining locus in salmonid fishes.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">272</span>–<span class="cit-lpage">280</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DWoram%26rft.auinit1%253DR.%2BA.%26rft.volume%253D13%26rft.issue%253D2%26rft.spage%253D272%26rft.epage%253D280%26rft.atitle%253DComparative%2BGenome%2BAnalysis%2Bof%2Bthe%2BPrimary%2BSex-Determining%2BLocus%2Bin%2BSalmonid%2BFishes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.578503%26rft_id%253Dinfo%253Apmid%252F12566405%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEzLzIvMjcyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzU0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.543.53" data-doi="10.1139/g03-127"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Woram</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McGowan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stout</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gharbi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferguson</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A genetic linkage map for Arctic char (<em>Salvelinus alpinus</em>): evidence for higher recombination rates and segregation distortion in hybrid <em>vs.</em> pure strain mapping parents.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">304</span>–<span class="cit-lpage">315</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DWoram%26rft.auinit1%253DR.%2BA.%26rft.volume%253D47%26rft.issue%253D2%26rft.spage%253D304%26rft.epage%253D315%26rft.atitle%253DA%2Bgenetic%2Blinkage%2Bmap%2Bfor%2BArctic%2Bchar%2B%2528Salvelinus%2Balpinus%2529%253A%2Bevidence%2Bfor%2Bhigher%2Brecombination%2Brates%2Band%2Bsegregation%2Bdistortion%2Bin%2Bhybrid%2Bversus%2Bpure%2Bstrain%2Bmapping%2Bparents.%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fg03-127%26rft_id%253Dinfo%253Apmid%252F15060583%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/g03-127&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15060583&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F543.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.543.54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hollister</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">May</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Meiotic models to explain classical linkage, pseudolinkage and chromosome pairing in tetraploid derivative salmonid genomes.</span> <abbr class="cit-jnl-abbrev">Isozymes</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">239</span>–<span class="cit-lpage">260</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIsozymes%26rft.stitle%253DIsozymes%2BCurr%2BTop%2BBiol%2BMed%2BRes%26rft.aulast%253DWright%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D10%26rft.spage%253D239%26rft.epage%253D260%26rft.atitle%253DMeiotic%2Bmodels%2Bto%2Bexplain%2Bclassical%2Blinkage%252C%2Bpseudolinkage%252C%2Band%2Bchromosome%2Bpairing%2Bin%2Btetraploid%2Bderivative%2Bsalmonid%2Bgenomes.%26rft_id%253Dinfo%253Apmid%252F6354984%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=6354984&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F543.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1983RL29500010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.543.55" data-doi="10.1016/j.cub.2012.05.045"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yano</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guyomard</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nicol</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jouanno</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quillet</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">An immune-related gene evolved into the master sex-determining gene in rainbow trout, <em>Oncorhynchus mykiss</em>.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1423</span>–<span class="cit-lpage">1428</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DYano%26rft.auinit1%253DA.%26rft.volume%253D22%26rft.issue%253D15%26rft.spage%253D1423%26rft.epage%253D1428%26rft.atitle%253DAn%2Bimmune-related%2Bgene%2Bevolved%2Binto%2Bthe%2Bmaster%2Bsex-determining%2Bgene%2Bin%2Brainbow%2Btrout%252C%2BOncorhynchus%2Bmykiss.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2012.05.045%26rft_id%253Dinfo%253Apmid%252F22727696%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2012.05.045&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22727696&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F543.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.543.56" data-doi="10.1111/eva.12032"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yano</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nicol</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jouanno</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quillet</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fostier</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The sexually dimorphic on the Y-chromosome gene (<em>sdY</em>) is a conserved male-specific Y- chromosome sequence in many salmonids.</span> <abbr class="cit-jnl-abbrev">Evol. Appl.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">486</span>–<span class="cit-lpage">496</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvol.%2BAppl.%26rft.volume%253D6%26rft.spage%253D486%26rft_id%253Dinfo%253Adoi%252F10.1111%252Feva.12032%26rft_id%253Dinfo%253Apmid%252F23745140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/eva.12032&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23745140&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F543.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/2/543.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/2/533" title="Expression Patterns of Atlantic Sturgeon (Acipenser oxyrinchus) During Embryonic Development" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/2/557" title="Fine Mapping of Ur-3, a Historically Important Rust Resistance Locus in Common Bean" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/2"><span>Volume 7 Issue 2, February 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/2.cover-source.jpg?itok=m4uMC43I" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/2/543" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-d2WN0KqdazB27j0weKhgjJbNShsWaUcr0J-ALhs8LM4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/2/543" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/2/543" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/2/543" class="active" data-icon-position="" data-hide-link-title="0">A SNP Based Linkage Map of the Arctic Charr (Salvelinus alpinus) Genome Provides Insights into the Diploidization Process After Whole Genome Duplication</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460564" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-9N1eXC59C6sPBvwz5uwXbKgZowAnBqqHN0D58lFgk4E" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/2/543.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/2/543" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-Q617hx71FOQfN1ZaiqU1MHnDt0FolEt7ot2L4KCf_ho" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/2/543" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/2/543.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460564" id="node460564--2" data-pisa="ggg;7/2/543" data-pisa-master="ggg;g3.116.038026" data-apath="/ggg/7/2/543.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A SNP Based Linkage Map of the Arctic Charr (<em>Salvelinus alpinus</em>) Genome Provides Insights into the Diploidization Process After Whole Genome Duplication</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Cameron M. Nugent</span>, <span class="highwire-citation-author" data-delta="1">Anne A. Easton</span>, <span class="highwire-citation-author" data-delta="2">Joseph D. Norman</span>, <span class="highwire-citation-author" data-delta="3">Moira M. Ferguson</span> and <span class="highwire-citation-author" data-delta="4">Roy G. Danzmann</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">543-556;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.038026 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460564--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Cameron M. Nugent </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Integrative Biology, University of Guelph, Ontario N1G 2W1, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cameron%2BM.%2BNugent%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nugent%20CM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACameron%2BM.%2BNugent%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:nugentc@uoguelph.ca" class="" data-icon-position="" data-hide-link-title="0">nugentc@uoguelph.ca</a>
<a href="mailto:rdanzman@uoguelph.ca" class="" data-icon-position="" data-hide-link-title="0">rdanzman@uoguelph.ca</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Anne A. Easton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Integrative Biology, University of Guelph, Ontario N1G 2W1, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anne%2BA.%2BEaston%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Easton%20AA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnne%2BA.%2BEaston%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Joseph D. Norman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Integrative Biology, University of Guelph, Ontario N1G 2W1, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BD.%2BNorman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Norman%20JD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoseph%2BD.%2BNorman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Moira M. Ferguson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Integrative Biology, University of Guelph, Ontario N1G 2W1, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Moira%2BM.%2BFerguson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ferguson%20MM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMoira%2BM.%2BFerguson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Roy G. Danzmann </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Integrative Biology, University of Guelph, Ontario N1G 2W1, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Roy%2BG.%2BDanzmann%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Danzmann%20RG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARoy%2BG.%2BDanzmann%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:nugentc@uoguelph.ca" class="" data-icon-position="" data-hide-link-title="0">nugentc@uoguelph.ca</a>
<a href="mailto:rdanzman@uoguelph.ca" class="" data-icon-position="" data-hide-link-title="0">rdanzman@uoguelph.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/2/543" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/2/543&amp;title=A%20SNP%20Based%20Linkage%20Map%20of%20the%20Arctic%20Charr%20%28Salvelinus%20alpinus%29%20Genome%20Provides%20Insights%20into%20the%20Diploidization%20Process%20After%20Whole%20Genome%20Duplication" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/2/543&amp;title=A%20SNP%20Based%20Linkage%20Map%20of%20the%20Arctic%20Charr%20%28Salvelinus%20alpinus%29%20Genome%20Provides%20Insights%20into%20the%20Diploidization%20Process%20After%20Whole%20Genome%20Duplication" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/2/543&amp;title=A%20SNP%20Based%20Linkage%20Map%20of%20the%20Arctic%20Charr%20%28Salvelinus%20alpinus%29%20Genome%20Provides%20Insights%20into%20the%20Diploidization%20Process%20After%20Whole%20Genome%20Duplication" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/2/543&amp;text=A%20SNP%20Based%20Linkage%20Map%20of%20the%20Arctic%20Charr%20%28Salvelinus%20alpinus%29%20Genome%20Provides%20Insights%20into%20the%20Diploidization%20Process%20After%20Whole%20Genome%20Duplication" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/2/543&amp;title=A%20SNP%20Based%20Linkage%20Map%20of%20the%20Arctic%20Charr%20%28Salvelinus%20alpinus%29%20Genome%20Provides%20Insights%20into%20the%20Diploidization%20Process%20After%20Whole%20Genome%20Duplication" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/2/543&amp;t=A%20SNP%20Based%20Linkage%20Map%20of%20the%20Arctic%20Charr%20%28Salvelinus%20alpinus%29%20Genome%20Provides%20Insights%20into%20the%20Diploidization%20Process%20After%20Whole%20Genome%20Duplication" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/2/543&amp;title=A%20SNP%20Based%20Linkage%20Map%20of%20the%20Arctic%20Charr%20%28Salvelinus%20alpinus%29%20Genome%20Provides%20Insights%20into%20the%20Diploidization%20Process%20After%20Whole%20Genome%20Duplication" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/2/543&amp;title=A%20SNP%20Based%20Linkage%20Map%20of%20the%20Arctic%20Charr%20%28Salvelinus%20alpinus%29%20Genome%20Provides%20Insights%20into%20the%20Diploidization%20Process%20After%20Whole%20Genome%20Duplication" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460564/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460564" id="node460564--3" data-pisa="ggg;7/2/543" data-pisa-master="ggg;g3.116.038026" data-apath="/ggg/7/2/543.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A SNP Based Linkage Map of the Arctic Charr (<em>Salvelinus alpinus</em>) Genome Provides Insights into the Diploidization Process After Whole Genome Duplication</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Cameron M. Nugent</span>, <span class="highwire-citation-author" data-delta="1">Anne A. Easton</span>, <span class="highwire-citation-author" data-delta="2">Joseph D. Norman</span>, <span class="highwire-citation-author" data-delta="3">Moira M. Ferguson</span> and <span class="highwire-citation-author" data-delta="4">Roy G. Danzmann</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">543-556;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.038026 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460564--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Cameron M. Nugent </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Integrative Biology, University of Guelph, Ontario N1G 2W1, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cameron%2BM.%2BNugent%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nugent%20CM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACameron%2BM.%2BNugent%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:nugentc@uoguelph.ca" class="" data-icon-position="" data-hide-link-title="0">nugentc@uoguelph.ca</a>
<a href="mailto:rdanzman@uoguelph.ca" class="" data-icon-position="" data-hide-link-title="0">rdanzman@uoguelph.ca</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Anne A. Easton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Integrative Biology, University of Guelph, Ontario N1G 2W1, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anne%2BA.%2BEaston%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Easton%20AA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnne%2BA.%2BEaston%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Joseph D. Norman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Integrative Biology, University of Guelph, Ontario N1G 2W1, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BD.%2BNorman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Norman%20JD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoseph%2BD.%2BNorman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Moira M. Ferguson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Integrative Biology, University of Guelph, Ontario N1G 2W1, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Moira%2BM.%2BFerguson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ferguson%20MM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMoira%2BM.%2BFerguson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Roy G. Danzmann </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Integrative Biology, University of Guelph, Ontario N1G 2W1, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Roy%2BG.%2BDanzmann%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Danzmann%20RG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARoy%2BG.%2BDanzmann%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:nugentc@uoguelph.ca" class="" data-icon-position="" data-hide-link-title="0">nugentc@uoguelph.ca</a>
<a href="mailto:rdanzman@uoguelph.ca" class="" data-icon-position="" data-hide-link-title="0">rdanzman@uoguelph.ca</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=543&amp;rft.epage=556&amp;rft.atitle=A%20SNP%20Based%20Linkage%20Map%20of%20the%20Arctic%20Charr%20%28Salvelinus%20alpinus%29%20Genome%20Provides%20Insights%20into%20the%20Diploidization%20Process%20After%20Whole%20Genome%20Duplication&amp;rft.volume=7&amp;rft.issue=2&amp;rft.date=2017-02-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Nugent%2C+Cameron+M.&amp;rft.au=Easton%2C+Anne+A.&amp;rft.au=Norman%2C+Joseph+D.&amp;rft.au=Ferguson%2C+Moira+M.&amp;rft.au=Danzmann%2C+Roy+G."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460564/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460564/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460564/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460564/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460564/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460564/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460564/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460564/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460564/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460564/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460564/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460564/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/2/543" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/2/543.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/2/543.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/2/543.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/2/543.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
