<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/2/591" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/2/591.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/2/591.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/2/591.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/2/591/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/2/591.atom" />
<meta name="HW.pisa" content="ggg;7/2/591" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Paralog-Specific Functions of RPL7A and RPL7B Mediated by Ribosomal Protein or snoRNA Dosage in Saccharomyces cerevisiae" />
<meta name="DC.Identifier" content="10.1534/g3.116.035931" />
<meta name="DC.Date" content="2017-02-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Palumbo et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Most ribosomal proteins in Saccharomyces cerevisiae are encoded by two paralogs that additively produce the optimal protein level for cell growth. Nonetheless, deleting one paralog of most ribosomal protein gene pairs results in a variety of phenotypes not observed when the other paralog is deleted. To determine whether paralog-specific phenotypes associated with deleting [RPL7A][1] or [RPL7B][2] stem from distinct functions or different levels of the encoded isoforms, the coding region and introns of one paralog, including an intron-embedded snoRNA (small nucleolar RNA) gene, were exchanged with that of the other paralog. Among mutants harboring a single native or chimeric [RPL7][3] allele, expression from the [RPL7A][1] locus exceeded that from the [RPL7B][2] locus, and more [Rpl7a][1] was expressed from either locus than [Rpl7b][2]. Phenotypic differences in tunicamycin sensitivity, [ASH1][4] mRNA localization, and mobility of the Ty1 retrotransposon were strongly correlated with [Rpl7][3] and ribosome levels, but not with the [Rpl7][3] or snoRNA isoform expressed. Although Ty1 RNA is cotranslationally localized, depletion of [Rpl7][3] minimally affected synthesis of Ty1 Gag protein, but strongly influenced Ty1 RNA localization. Unlike the other processes studied, Ty1 cDNA accumulation was influenced by both the level and isoform of [Rpl7][3] or snoRNA expressed. These cellular processes had different minimal threshold values for [Rpl7][3] and ribosome levels, but all were functional when isoforms of either paralog were expressed from the [RPL7A][1] locus or both [RPL7][3] loci. This study illustrates the broad range of phenotypes that can result from depleting ribosomes to different levels.

 [1]: http://www.yeastgenome.org/locus/S000003044/overview
 [2]: http://www.yeastgenome.org/locus/S000006119/overview
 [3]: http://www.yeastgenome.org/locus/S000004947/overview
 [4]: http://www.yeastgenome.org/locus/S000001668/overview" />
<meta name="DC.Contributor" content="Ryan J. Palumbo" />
<meta name="DC.Contributor" content="Gabriele Fuchs" />
<meta name="DC.Contributor" content="Sheila Lutz" />
<meta name="DC.Contributor" content="M. Joan Curcio" />
<meta name="article:published_time" content="2017-02-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Paralog-Specific Functions of RPL7A and RPL7B Mediated by Ribosomal Protein or snoRNA Dosage in Saccharomyces cerevisiae" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Most ribosomal proteins in &lt;i&gt;Saccharomyces cerevisiae&lt;/i&gt; are encoded by two paralogs that additively produce the optimal protein level for cell growth. Nonetheless, deleting one paralog of most ribosomal protein gene pairs results in a variety of phenotypes not observed when the other paralog is deleted. To determine whether paralog-specific phenotypes associated with deleting &lt;i&gt;RPL7A&lt;/i&gt; or &lt;i&gt;RPL7B&lt;/i&gt; stem from distinct functions or different levels of the encoded isoforms, the coding region and introns of one paralog, including an intron-embedded snoRNA (small nucleolar RNA) gene, were exchanged with that of the other paralog. Among mutants harboring a single native or chimeric &lt;i&gt;RPL7&lt;/i&gt; allele, expression from the &lt;i&gt;RPL7A&lt;/i&gt; locus exceeded that from the &lt;i&gt;RPL7B&lt;/i&gt; locus, and more Rpl7a was expressed from either locus than Rpl7b. Phenotypic differences in tunicamycin sensitivity, &lt;i&gt;ASH1&lt;/i&gt; mRNA localization, and mobility of the Ty1 retrotransposon were strongly correlated with Rpl7 and ribosome levels, but not with the Rpl7 or snoRNA isoform expressed. Although Ty1 RNA is cotranslationally localized, depletion of Rpl7 minimally affected synthesis of Ty1 Gag protein, but strongly influenced Ty1 RNA localization. Unlike the other processes studied, Ty1 cDNA accumulation was influenced by both the level and isoform of Rpl7 or snoRNA expressed. These cellular processes had different minimal threshold values for Rpl7 and ribosome levels, but all were functional when isoforms of either paralog were expressed from the &lt;i&gt;RPL7A&lt;/i&gt; locus or both &lt;i&gt;RPL7&lt;/i&gt; loci. This study illustrates the broad range of phenotypes that can result from depleting ribosomes to different levels.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/02/01" />
<meta name="citation_mjid" content="ggg;7/2/591" />
<meta name="citation_id" content="7/2/591" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/2/591" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/2/591.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/2/591.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/2/591.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.035931" />
<meta name="citation_pmid" content="28007835" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="2" />
<meta name="citation_num_pages" content="16" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="591" />
<meta name="citation_lastpage" content="606" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Ryan J. Palumbo" />
<meta name="citation_author_institution" content="Laboratory of Molecular Genetics, Wadsworth Center, New York State Department of Health, Albany, New York 12201" />
<meta name="citation_author_institution" content="Department of Biomedical Sciences, School of Public Health, University at Albany, New York 12201" />
<meta name="citation_author" content="Gabriele Fuchs" />
<meta name="citation_author_institution" content="Department of Biological Sciences, The RNA Institute, University at Albany-State University of New York, New York 12222" />
<meta name="citation_author" content="Sheila Lutz" />
<meta name="citation_author_institution" content="Laboratory of Molecular Genetics, Wadsworth Center, New York State Department of Health, Albany, New York 12201" />
<meta name="citation_author" content="M. Joan Curcio" />
<meta name="citation_author_institution" content="Laboratory of Molecular Genetics, Wadsworth Center, New York State Department of Health, Albany, New York 12201" />
<meta name="citation_author_institution" content="Department of Biomedical Sciences, School of Public Health, University at Albany, New York 12201" />
<meta name="citation_author_email" content="joan.curcio@health.ny.gov" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5361-3909" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=N. Abovich;citation_author=M. Rosbash;citation_title=Two genes for ribosomal protein 51 of Saccharomyces cerevisiae complement and contribute to the ribosomes.;citation_pages=1871-1879;citation_volume=4;citation_year=1984;citation_issue=9;citation_pmid=6092944;citation_doi=10.1128/MCB.4.9.1871" />
<meta name="citation_reference" content="citation_title=Methods in Yeast Genetics.;citation_author=D. C. Amberg;citation_author=D. J. Burke;citation_author=J. N. Strathern;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. H. Askree;citation_author=T. Yehuda;citation_author=S. Smolikov;citation_author=R. Gurevich;citation_author=J. Hawk;citation_title=A genome-wide screen for Saccharomyces cerevisiae deletion mutants that affect telomere length.;citation_volume=101;citation_year=2004;citation_pmid=15161972;citation_doi=10.1073/pnas.0401263101" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=G. Badis;citation_author=C. Saveanu;citation_author=M. Fromont-Racine;citation_author=A. Jacquier;citation_title=Targeted mRNA degradation by deadenylation-independent decapping.;citation_pages=5-15;citation_volume=15;citation_year=2004;citation_issue=1;citation_pmid=15225544;citation_doi=10.1016/j.molcel.2004.06.028" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. A. Baller;citation_author=J. Gao;citation_author=R. Stamenova;citation_author=M. J. Curcio;citation_author=D. F. Voytas;citation_title=A nucleosomal surface defines an integration hotspot for the Saccharomyces cerevisiae Ty1 retrotransposon.;citation_volume=22;citation_year=2012;citation_pmid=22219511;citation_doi=10.1101/gr.129585.111" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology of the Cell;citation_journal_abbrev=Mol. Biol. Cell;citation_author=D. L. Beach;citation_author=K. Bloom;citation_title=ASH1 mRNA Localization in Three Acts;citation_pages=2567-2577;citation_volume=12;citation_year=2001;citation_issue=9;citation_pmid=11553699;citation_doi=10.1091/mbc.12.9.2567" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=D. L. Beach;citation_author=E. D. Salmon;citation_author=K. Bloom;citation_title=Localization and anchoring of mRNA in budding yeast.;citation_pages=569-578;citation_volume=9;citation_year=1999;citation_issue=11;citation_pmid=10359695;citation_doi=10.1016/S0960-9822(99)80260-7" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=A. Ben-Shem;citation_author=N. Garreau de Loubresse;citation_author=S. Melnikov;citation_author=L. Jenner;citation_author=G. Yusupova;citation_title=The structure of the eukaryotic ribosome at 3.0 A resolution.;citation_volume=334;citation_year=2011;citation_pmid=22096102;citation_doi=10.1126/science.1212642" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=E. Bertrand;citation_author=P. Chartrand;citation_author=M. Schaefer;citation_author=S. M. Shenoy;citation_author=R. H. Singer;citation_title=Localization of ASH1 mRNA particles in living yeast.;citation_pages=437-445;citation_volume=2;citation_year=1998;citation_issue=4;citation_pmid=9809065;citation_doi=10.1016/S1097-2765(00)80143-4" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=J. D. Boeke;citation_author=D. Eichinger;citation_author=D. Castrillon;citation_author=G. R. Fink;citation_title=The Saccharomyces cerevisiae genome contains functional and nonfunctional copies of transposon Ty1.;citation_pages=1432-1442;citation_volume=8;citation_year=1988;citation_issue=4;citation_pmid=2837641;citation_doi=10.1128/MCB.8.4.1432" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=C. J. Bonangelino;citation_author=E. M. Chavez;citation_author=J. S. Bonifacino;citation_title=Genomic screen for vacuolar protein sorting genes in Saccharomyces cerevisiae.;citation_volume=13;citation_year=2002;citation_pmid=12134085;citation_doi=10.1091/mbc.02-01-0005" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Biol.;citation_author=M. A. Checkley;citation_author=K. Nagashima;citation_author=S. J. Lockett;citation_author=K. M. Nyswaner;citation_author=D. J. Garfinkel;citation_title=P-body components are required for Ty1 retrotransposition during assembly of retrotransposition-competent virus-like particles.;citation_volume=30;citation_year=2010;citation_pmid=19901074;citation_doi=10.1128/MCB.00251-09" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=B. K. Clarkson;citation_author=W. V. Gilbert;citation_author=J. A. Doudna;citation_title=Functional overlap between eIF4G isoforms in Saccharomyces cerevisiae.;citation_pages=e9114-e9114;citation_volume=5;citation_year=2010;citation_issue=2;citation_pmid=20161741;citation_doi=10.1371/journal.pone.0009114" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=D. Conte;citation_author=M. J. Curcio;citation_title=Fus3 controls Ty1 transpositional dormancy through the invasive growth MAPK pathway.;citation_pages=415-427;citation_volume=35;citation_year=2000;citation_issue=2;citation_pmid=10652102;citation_doi=10.1046/j.1365-2958.2000.01710.x" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=M. J. Curcio;citation_author=D. J. Garfinkel;citation_title=Single-step selection for Ty1 element retrotransposition.;citation_pages=936-940;citation_volume=88;citation_year=1991;citation_issue=3;citation_pmid=1846969;citation_doi=10.1073/pnas.88.3.936" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. J. Curcio;citation_author=D. J. Garfinkel;citation_title=Heterogeneous functional Ty1 elements are abundant in the Saccharomyces cerevisiae genome.;citation_pages=1245-1259;citation_volume=136;citation_year=1994;citation_issue=4;citation_pmid=8013902" />
<meta name="citation_reference" content="citation_journal_title=Microbiol. Spectr.;citation_author=M. J. Curcio;citation_author=S. Lutz;citation_author=P. Lesage;citation_title=The Ty1 LTR-retrotransposon of budding yeast, Saccharomyces cerevisiae.;citation_pages=1-35;citation_volume=2;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Biol.;citation_author=M. J. Curcio;citation_author=A. E. Kenny;citation_author=S. Moore;citation_author=D. J. Garfinkel;citation_author=M. Weintraub;citation_title=S-phase checkpoint pathways stimulate the mobility of the retrovirus-like transposon Ty1.;citation_volume=27;citation_year=2007;citation_pmid=17923678;citation_doi=10.1128/MCB.01095-07" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Dakshinamurthy;citation_author=K. M. Nyswaner;citation_author=P. J. Farabaugh;citation_author=D. J. Garfinkel;citation_title=BUD22 affects Ty1 retrotransposition and ribosome biogenesis in Saccharomyces cerevisiae.;citation_volume=185;citation_year=2010;citation_pmid=20498295;citation_doi=10.1534/genetics.110.119115" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=E. J. Dean;citation_author=J. C. Davis;citation_author=R. W. Davis;citation_author=D. A. Petrov;citation_title=Pervasive and persistent redundancy among duplicated genes in yeast.;citation_pages=e1000113-e1000113;citation_volume=4;citation_year=2008;citation_issue=7;citation_pmid=18604285;citation_doi=10.1371/journal.pgen.1000113" />
<meta name="citation_reference" content="citation_journal_title=Blood;citation_author=K. De Keersmaecker;citation_author=S. O. Sulima;citation_author=J. D. Dinman;citation_title=Ribosomopathies and the paradox of cellular hypo- to hyperproliferation.;citation_volume=125;citation_year=2015;citation_pmid=25575543;citation_doi=10.1182/blood-2014-10-569616" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=S. E. Devine;citation_author=J. D. Boeke;citation_title=Integration of the yeast retrotransposon Ty1 is targeted to regions upstream of genes transcribed by RNA polymerase III.;citation_pages=620-633;citation_volume=10;citation_year=1996;citation_issue=5;citation_pmid=8598291;citation_doi=10.1101/gad.10.5.620" />
<meta name="citation_reference" content="citation_journal_title=J. Mol. Biol.;citation_author=J. D. Dinman;citation_title=Pathways to specialized ribosomes: the Brussels lecture.;citation_pages=2186-2194;citation_volume=428;citation_year=2016;citation_pmid=26764228;citation_doi=10.1016/j.jmb.2015.12.021" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. H. Doh;citation_author=S. Lutz;citation_author=M. J. Curcio;citation_title=Co-translational localization of an LTR-retrotransposon RNA to the endoplasmic reticulum nucleates virus-like particle assembly sites.;citation_pages=e1004219;citation_volume=10;citation_year=2014;citation_pmid=24603646;citation_doi=10.1371/journal.pgen.1004219" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=F. J. Eng;citation_author=J. R. Warner;citation_title=Structural basis for the regulation of splicing of a yeast messenger RNA.;citation_pages=797-804;citation_volume=65;citation_year=1991;citation_issue=5;citation_pmid=2040015;citation_doi=10.1016/0092-8674(91)90387-E" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=S. W. Fewell;citation_author=J. L. Woolford;citation_title=Ribosomal Protein S14 of Saccharomyces cerevisiae Regulates Its Expression by Binding to RPS14B Pre-mRNA and to 18S rRNA;citation_pages=826-834;citation_volume=19;citation_year=1999;citation_issue=1;citation_pmid=9858605;citation_doi=10.1128/MCB.19.1.826" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=S. Ghaemmaghami;citation_author=W. K. Huh;citation_author=K. Bower;citation_author=R. W. Howson;citation_author=A. Belle;citation_title=Global analysis of protein expression in yeast.;citation_pages=737-741;citation_volume=425;citation_year=2003;citation_issue=6959;citation_pmid=14562106;citation_doi=10.1038/nature02046" />
<meta name="citation_reference" content="citation_journal_title=Trends in biochemical sciences;citation_journal_abbrev=Trends Biochem Sci;citation_author=W. V. Gilbert;citation_title=Functional specialization of ribosomes?;citation_pages=127-132;citation_volume=36;citation_year=2011;citation_issue=3;citation_pmid=21242088;citation_doi=10.1016/j.tibs.2010.12.002" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. L. Griffith;citation_author=L. E. Coleman;citation_author=A. S. Raymond;citation_author=S. G. Goodson;citation_author=W. S. Pittard;citation_title=Functional Genomics Reveals Relationships Between the Retrovirus-Like Ty1 Element and Its Host Saccharomyces cerevisiae;citation_pages=867-879;citation_volume=164;citation_year=2003;citation_issue=3;citation_pmid=12871900" />
<meta name="citation_reference" content="citation_journal_title=Genes Dev.;citation_author=B. Haarer;citation_author=S. Viggiano;citation_author=M. A. Hibbs;citation_author=O. G. Troyanskaya;citation_author=D. C. Amberg;citation_title=Modeling complex genetic interactions in a simple eukaryotic genome: actin displays a rich spectrum of complex haploinsufficiencies.;citation_volume=21;citation_year=2007;citation_pmid=17167106;citation_doi=10.1101/gad.1477507" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Protoc;citation_author=L. Haim-Vilmovsky;citation_author=J. E. Gerst;citation_title=m-TAG: a PCR-based genomic integration method to visualize the localization of specific endogenous mRNAs in vivo in yeast.;citation_pages=1274-1284;citation_volume=4;citation_year=2009;citation_issue=9;citation_pmid=19680241;citation_doi=10.1038/nprot.2009.115" />
<meta name="citation_reference" content="citation_journal_title=Virology;citation_journal_abbrev=Virology;citation_author=J. W. J. W. Harger;citation_author=A. A. Meskauskas;citation_author=J. J. Nielsen;citation_author=M. C. M. C. Justice;citation_author=J. D. J. D. Dinman;citation_title=Ty1 retrotransposition and programmed +1 ribosomal frameshifting require the integrity of the protein synthetic translocation step.;citation_pages=216-224;citation_volume=286;citation_year=2001;citation_issue=1;citation_pmid=11448174;citation_doi=10.1006/viro.2001.0997" />
<meta name="citation_reference" content="citation_journal_title=Molecular and biochemical parasitology;citation_journal_abbrev=Mol Biochem Parasitol;citation_author=R. Hernandez;citation_author=F. Diaz-de Leon;citation_author=M. Castaneda;citation_title=Molecular cloning and partial characterization of ribosomal RNA genes from Trypanosoma cruzi.;citation_pages=275-279;citation_volume=27;citation_year=1988;citation_issue=2-3;citation_pmid=3278230;citation_doi=10.1016/0166-6851(88)90047-3" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=H. Innan;citation_author=F. Kondrashov;citation_title=The evolution of gene duplications: classifying and distinguishing between models.;citation_pages=97-108;citation_volume=11;citation_year=2010;citation_issue=2;citation_pmid=20051986;citation_doi=10.1038/nrg2689" />
<meta name="citation_reference" content="citation_journal_title=RNA;citation_author=J. Jakovljevic;citation_author=U. Ohmayer;citation_author=M. Gamalinda;citation_author=J. Talkish;citation_author=L. Alexander;citation_title=Ribosomal proteins L7 and L8 function in concert with six A(3) assembly factors to propagate assembly of domains I and II of 25S rRNA in yeast 60S ribosomal subunits.;citation_volume=18;citation_year=2012;citation_pmid=22893726;citation_doi=10.1261/rna.032540.112" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=H. Ji;citation_author=D. P. Moore;citation_author=M. A. Blomberg;citation_author=L. T. Braiterman;citation_author=D. F. Voytas;citation_title=Hotspots for unselected Ty1 transposition events on yeast chromosome III are near tRNA genes and LTR sequences.;citation_pages=1007-1018;citation_volume=73;citation_year=1993;citation_issue=5;citation_pmid=8388781;citation_doi=10.1016/0092-8674(93)90278-X" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=M. Knop;citation_author=K. Siegers;citation_author=G. Pereira;citation_author=W. Zachariae;citation_author=B. Winsor;citation_title=Epitope tagging of yeast genes using a PCR-based strategy: more tags and improved practical routines.;citation_pages=963-972;citation_volume=15;citation_year=1999;citation_issue=10B;citation_pmid=10407276;citation_doi=10.1002/(SICI)1097-0061(199907)15:10B&lt;963::AID-YEA399&gt;3.0.CO;2-W" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=S. Komili;citation_author=N. G. Farny;citation_author=F. P. Roth;citation_author=P. A. Silver;citation_title=Functional specificity among ribosomal proteins regulates gene expression.;citation_pages=557-571;citation_volume=131;citation_year=2007;citation_issue=3;citation_pmid=17981122;citation_doi=10.1016/j.cell.2007.08.037" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=N. Kondrashov;citation_author=A. Pusic;citation_author=C. R. Stumpf;citation_author=K. Shimizu;citation_author=A. C. Hsieh;citation_title=Ribosome-mediated specificity in Hox mRNA translation and vertebrate tissue patterning.;citation_pages=383-397;citation_volume=145;citation_year=2011;citation_issue=3;citation_pmid=21529712;citation_doi=10.1016/j.cell.2011.03.028" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=R. J. Leer;citation_author=M. M. van Raamsdonk-Duin;citation_author=C. M. Molenaar;citation_author=H. M. Witsenboer;citation_author=W. H. Mager;citation_title=Yeast contains two functional genes coding for ribosomal protein S10;citation_pages=5027-5039;citation_volume=13;citation_year=1985;citation_issue=14;citation_pmid=2991849;citation_doi=10.1093/nar/13.14.5027" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=Z. Li;citation_author=A. G. Paulovich;citation_author=J. L. Woolford;citation_author=J. L. Woolford;citation_title=Feedback inhibition of the yeast ribosomal protein gene CRY2 is mediated by the nucleotide sequence and secondary structure of CRY2 pre- mRNA;citation_pages=6454-6464;citation_volume=15;citation_year=1995;citation_issue=11;citation_pmid=7565797;citation_doi=10.1128/MCB.15.11.6454" />
<meta name="citation_reference" content="citation_journal_title=FEBS Letters;citation_journal_abbrev=FEBS Letters;citation_author=A. Lin;citation_title=Localization of surface peptide from ribosomal protein L7 on 80 S ribosome by biotinylation.;citation_pages=121-124;citation_volume=287;citation_year=1991;citation_issue=1-2;citation_pmid=1879519;citation_doi=10.1016/0014-5793(91)80030-7" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Biol.;citation_author=F. Malagon;citation_author=T. H. Jensen;citation_title=The T body, a new cytoplasmic RNA granule in Saccharomyces cerevisiae.;citation_volume=28;citation_year=2008;citation_pmid=18678648;citation_doi=10.1128/MCB.00684-08" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=V. P. Mauro;citation_author=G. M. Edelman;citation_title=The ribosome filter hypothesis.;citation_volume=99;citation_year=2002;citation_pmid=12221294;citation_doi=10.1073/pnas.192442499" />
<meta name="citation_reference" content="citation_journal_title=Cell cycle (Georgetown, Tex.);citation_journal_abbrev=Cell Cycle;citation_author=V. P. Mauro;citation_author=G. M. Edelman;citation_title=The ribosome filter redux.;citation_pages=2246-2251;citation_volume=6;citation_year=2007;citation_issue=18;citation_pmid=17890902;citation_doi=10.4161/cc.6.18.4739" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=K. L. McCann;citation_author=S. J. Baserga;citation_title=Mysterious Ribosomopathies;citation_pages=849-850;citation_volume=341;citation_year=2013;citation_issue=6148;citation_pmid=23970686;citation_doi=10.1126/science.1244156" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=K. Mizuta;citation_author=T. Hashimoto;citation_author=E. Otaka;citation_title=Yeast ribosomal proteins: XIII. Saccharomyces cerevisiae YL8A gene, interrupted with two introns, encodes a homolog of mammalian L7;citation_pages=1011-1016;citation_volume=20;citation_year=1992;citation_issue=5;citation_pmid=1549461;citation_doi=10.1093/nar/20.5.1011" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Z. Mou;citation_author=A. E. Kenny;citation_author=M. J. Curcio;citation_title=Hos2 and Set3 promote integration of Ty1 retrotransposons at tRNA genes in Saccharomyces cerevisiae.;citation_volume=172;citation_year=2006;citation_pmid=16415356;citation_doi=10.1534/genetics.105.054072" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=L. Mularoni;citation_author=Y. Zhou;citation_author=T. Bowen;citation_author=S. Gangadharan;citation_author=S. J. Wheelan;citation_title=Retrotransposon Ty1 integration targets specifically positioned asymmetric nucleosomal DNA segments in tRNA hotspots.;citation_volume=22;citation_year=2012;citation_pmid=22219510;citation_doi=10.1101/gr.129460.111" />
<meta name="citation_reference" content="citation_journal_title=BMC evolutionary biology [electronic resource];citation_journal_abbrev=BMC Evol Biol;citation_author=C. Münk;citation_author=A. Willemsen;citation_author=I. G. Bravo;citation_title=An ancient history of gene duplications, fusions and losses in the evolution of APOBEC3 mutators in mammals.;citation_pages=71-71;citation_volume=12;citation_year=2012;citation_pmid=22640020;citation_doi=10.1186/1471-2148-12-71" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology of the Cell;citation_journal_abbrev=Mol. Biol. Cell;citation_author=L. Ni;citation_author=M. Snyder;citation_title=A Genomic Study of the Bipolar Bud Site Selection Pattern in Saccharomyces cerevisiae;citation_pages=2147-2170;citation_volume=12;citation_year=2001;citation_issue=7;citation_pmid=11452010;citation_doi=10.1091/mbc.12.7.2147" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=Y. Ohtake;citation_author=R. B. Wickner;citation_title=Yeast virus propagation depends critically on free 60S ribosomal subunit concentration;citation_pages=2772-2781;citation_volume=15;citation_year=1995;citation_issue=5;citation_pmid=7739558;citation_doi=10.1128/MCB.15.5.2772" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=N. Page;citation_author=M. Gerard-Vincent;citation_author=P. Menard;citation_author=M. Beaulieu;citation_author=M. Azuma;citation_title=A Saccharomyces cerevisiae Genome-Wide Mutant Screen for Altered Sensitivity to K1 Killer Toxin;citation_pages=875-894;citation_volume=163;citation_year=2003;citation_issue=3;citation_pmid=12663529" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=J. Parenteau;citation_author=M. Durand;citation_author=G. Morin;citation_author=J. Gagnon;citation_author=J. F. Lucier;citation_title=Introns within ribosomal protein genes regulate the production and function of yeast ribosomes.;citation_pages=320-331;citation_volume=147;citation_year=2011;citation_issue=2;citation_pmid=22000012;citation_doi=10.1016/j.cell.2011.08.044" />
<meta name="citation_reference" content="citation_journal_title=Cell Rep.;citation_author=J. Parenteau;citation_author=M. Lavoie;citation_author=M. Catala;citation_author=M. Malik-Ghulam;citation_author=J. Gagnon;citation_title=Preservation of gene duplication increases the regulatory spectrum of ribosomal protein genes and enhances growth under stress.;citation_pages=2516-2526;citation_volume=13;citation_year=2015;citation_pmid=26686636;citation_doi=10.1016/j.celrep.2015.11.033" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. G. Paulovich;citation_author=J. R. Thompson;citation_author=J. C. Larkin;citation_author=Z. Li;citation_author=J. L. Woolford;citation_title=Molecular genetics of cryptopleurine resistance in Saccharomyces cerevisiae: expression of a ribosomal protein gene family.;citation_pages=719-730;citation_volume=135;citation_year=1993;citation_issue=3;citation_pmid=8293976" />
<meta name="citation_reference" content="citation_journal_title=RNA;citation_author=D. Piekna-Przybylska;citation_author=W. A. Decatur;citation_author=M. J. Fournier;citation_title=New bioinformatic tools for analysis of nucleotide modifications in eukaryotic rRNA.;citation_volume=13;citation_year=2007;citation_pmid=17283215;citation_doi=10.1261/rna.373107" />
<meta name="citation_reference" content="citation_journal_title=EMBO J.;citation_author=C. Presutti;citation_author=S. A. Ciafre;citation_author=I. Bozzoni;citation_title=The ribosomal protein L2 in S. cerevisiae controls the level of accumulation of its own mRNA.;citation_pages=2215-2221;citation_volume=10;citation_year=1991;citation_pmid=2065661" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=K. J. Purzycka;citation_author=M. Legiewicz;citation_author=E. Matsuda;citation_author=L. D. Eizentstat;citation_author=S. Lusvarghi;citation_title=Exploring Ty1 retrotransposon RNA structure within virus-like particles.;citation_volume=41;citation_year=2013;citation_pmid=23093595;citation_doi=10.1093/nar/gks983" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. J. Rattray;citation_author=B. K. Shafer;citation_author=D. J. Garfinkel;citation_title=The Saccharomyces cerevisiae DNA Recombination and Repair Functions of the RAD52 Epistasis Group Inhibit Ty1 Transposition;citation_pages=543-556;citation_volume=154;citation_year=2000;citation_issue=2;citation_pmid=10655210" />
<meta name="citation_reference" content="citation_journal_abbrev=Mob DNA;citation_author=J. K. Risler;citation_author=A. E. Kenny;citation_author=R. J. Palumbo;citation_author=E. R. Gamache;citation_author=M. J. Curcio;citation_title=Host co-factors of the retrovirus-like transposon Ty1.;citation_pages=12-12;citation_volume=3;citation_year=2012;citation_issue=1;citation_pmid=22856544;citation_doi=10.1186/1759-8753-3-12" />
<meta name="citation_reference" content="citation_journal_title=Methods (San Diego, Calif.);citation_journal_abbrev=Methods;citation_author=W. A. Rosche;citation_author=P. L. Foster;citation_title=Determining mutation rates in bacterial populations.;citation_pages=4-17;citation_volume=20;citation_year=2000;citation_issue=1;citation_pmid=10610800;citation_doi=10.1006/meth.1999.0901" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=M. O. Rotenberg;citation_author=M. Moritz;citation_author=J. L. Woolford;citation_title=Depletion of Saccharomyces cerevisiae ribosomal protein L16 causes a decrease in 60S ribosomal subunits and formation of half-mer polyribosomes.;citation_pages=160-172;citation_volume=2;citation_year=1988;citation_issue=2;citation_pmid=3282992;citation_doi=10.1101/gad.2.2.160" />
<meta name="citation_reference" content="citation_journal_title=J. Virol.;citation_author=A. Saha;citation_author=J. A. Mitchell;citation_author=Y. Nishida;citation_author=J. E. Hildreth;citation_author=J. A. Ariberre;citation_title=A trans-dominant form of Gag restricts Ty1 retrotransposition and mediates copy number control.;citation_volume=89;citation_year=2015;citation_pmid=25609815;citation_doi=10.1128/JVI.03060-14" />
<meta name="citation_reference" content="citation_journal_abbrev=RNA Biol;citation_author=S. B. Sandmeyer;citation_author=K. A. Clemens;citation_title=Function of a retrotransposon nucleocapsid protein.;citation_pages=642-654;citation_volume=7;citation_year=2010;citation_issue=6;citation_pmid=21189452;citation_doi=10.4161/rna.7.6.14117" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. T. Scholes;citation_author=M. Banerjee;citation_author=B. Bowen;citation_author=M. J. Curcio;citation_title=Multiple Regulators of Ty1 Transposition in Saccharomyces cerevisiae Have Conserved Roles in Genome Maintenance;citation_pages=1449-1465;citation_volume=159;citation_year=2001;citation_issue=4;citation_pmid=11779788" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=D. T. Scholes;citation_author=A. E. Kenny;citation_author=E. R. Gamache;citation_author=Z. Mou;citation_author=M. J. Curcio;citation_title=Activation of a LTR-retrotransposon by telomere erosion.;citation_volume=100;citation_year=2003;citation_pmid=14673098;citation_doi=10.1073/pnas.2136609100" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=G. Sharon;citation_author=T. J. Burkett;citation_author=D. J. Garfinkel;citation_title=Efficient homologous recombination of Ty1 element cDNA when integration is blocked.;citation_pages=6540-6551;citation_volume=14;citation_year=1994;citation_issue=10;citation_pmid=7523854;citation_doi=10.1128/MCB.14.10.6540" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. S. Sikorski;citation_author=P. Hieter;citation_title=A system of shuttle vectors and yeast host strains designed for efficient manipulation of DNA in Saccharomyces cerevisiae.;citation_pages=19-27;citation_volume=122;citation_year=1989;citation_issue=1;citation_pmid=2659436" />
<meta name="citation_reference" content="citation_journal_title=Current genetics;citation_journal_abbrev=Curr Genet;citation_author=I. Simoff;citation_author=H. Moradi;citation_author=O. Nygard;citation_title=Functional characterization of ribosomal protein L15 from Saccharomyces cerevisiae.;citation_pages=111-125;citation_volume=55;citation_year=2009;citation_issue=2;citation_pmid=19184027;citation_doi=10.1007/s00294-009-0228-z" />
<meta name="citation_reference" content="citation_journal_title=Cell Rep.;citation_author=N. Slavov;citation_author=S. Semrau;citation_author=E. Airoldi;citation_author=B. Budnik;citation_author=A. van Oudenaarden;citation_title=Differential stoichiometry among core ribosomal proteins.;citation_pages=865-873;citation_volume=13;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. Stamenova;citation_author=P. H. Maxwell;citation_author=A. E. Kenny;citation_author=M. J. Curcio;citation_title=Rrm3 protects the Saccharomyces cerevisiae genome from instability at nascent sites of retrotransposition.;citation_volume=182;citation_year=2009;citation_pmid=19414561;citation_doi=10.1534/genetics.109.104208" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=K. K. Steffen;citation_author=V. L. MacKay;citation_author=E. O. Kerr;citation_author=M. Tsuchiya;citation_author=D. Hu;citation_title=Yeast life span extension by depletion of 60s ribosomal subunits is mediated by Gcn4.;citation_pages=292-302;citation_volume=133;citation_year=2008;citation_issue=2;citation_pmid=18423200;citation_doi=10.1016/j.cell.2008.02.037" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. K. Steffen;citation_author=M. A. McCormick;citation_author=K. M. Pham;citation_author=V. L. MacKay;citation_author=J. R. Delaney;citation_title=Ribosome deficiency protects against ER stress in Saccharomyces cerevisiae.;citation_volume=191;citation_year=2012;citation_pmid=22377630;citation_doi=10.1534/genetics.111.136549" />
<meta name="citation_reference" content="citation_journal_title=Methods in enzymology;citation_journal_abbrev=Methods Enzymol;citation_author=F. Storici;citation_author=M. A. Resnick;citation_title=The delitto perfetto approach to in vivo site-directed mutagenesis and chromosome rearrangements with synthetic oligonucleotides in yeast.;citation_pages=329-345;citation_volume=409;citation_year=2006;citation_pmid=16793410;citation_doi=10.1016/j.ymeth.2006.06.011" />
<meta name="citation_reference" content="citation_journal_title=Mob. DNA;citation_author=S. Suresh;citation_author=H. W. Ahn;citation_author=K. Joshi;citation_author=A. Dakshinamurthy;citation_author=A. Kananganat;citation_title=Ribosomal protein and biogenesis factors affect multiple steps during movement of the Saccharomyces cerevisiae Ty1 retrotransposon.;citation_pages=22;citation_volume=6;citation_year=2015;citation_pmid=26664557;citation_doi=10.1186/s13100-015-0053-5" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=Y. J. Tsai;citation_author=H. I. Lee;citation_author=A. Lin;citation_title=Ribosome distribution in HeLa cells during the cell cycle.;citation_pages=e32820-e32820;citation_volume=7;citation_year=2012;citation_issue=3;citation_pmid=22403714" />
<meta name="citation_reference" content="citation_journal_title=RNA;citation_author=S. Vincenti;citation_author=V. De Chiara;citation_author=I. Bozzoni;citation_author=C. Presutti;citation_title=The position of yeast snoRNA-coding regions within host introns is essential for their biosynthesis and for efficient splicing of the host pre-mRNA.;citation_volume=13;citation_year=2007;citation_pmid=17135484;citation_doi=10.1261/rna.251907" />
<meta name="citation_reference" content="citation_journal_title=Cold Spring Harbor Perspectives in Biology;citation_journal_abbrev=Cold Spring Harb. Perspect. Biol.;citation_author=D. N. Wilson;citation_author=J. H. Doudna Cate;citation_title=The Structure and Function of the Eukaryotic Ribosome;citation_pages=a011536-a011536;citation_volume=4;citation_year=2012;citation_issue=5;citation_pmid=22550233;citation_doi=10.1101/cshperspect.a011536" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=K. H. Wolfe;citation_author=D. C. Shields;citation_title=Molecular evidence for an ancient duplication of the entire yeast genome.;citation_pages=708-713;citation_volume=387;citation_year=1997;citation_issue=6634;citation_pmid=9192896;citation_doi=10.1038/42711" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. L. Woolford;citation_author=S. J. Baserga;citation_title=Ribosome Biogenesis in the Yeast Saccharomyces cerevisiae;citation_pages=643-681;citation_volume=195;citation_year=2013;citation_issue=3;citation_pmid=24190922;citation_doi=10.1534/genetics.113.153197" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Molecular cell biology;citation_journal_abbrev=Nat Rev Mol Cell Biol;citation_author=S. Xue;citation_author=M. Barna;citation_title=Specialized ribosomes: a new frontier in gene regulation and organismal biology.;citation_pages=355-369;citation_volume=13;citation_year=2012;citation_issue=6;citation_pmid=22617470;citation_doi=10.1038/nrm3359" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=S. Xue;citation_author=S. Tian;citation_author=K. Fujii;citation_author=W. Kladwang;citation_author=R. Das;citation_title=RNA regulons in Hox 5′ UTRs confer ribosome specificity to gene regulation.;citation_pages=33-38;citation_volume=517;citation_year=2015;citation_pmid=25409156;citation_doi=10.1038/nature14010" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. M. Yarrington;citation_author=S. M. Richardson;citation_author=C. R. Lisa Huang;citation_author=J. D. Boeke;citation_title=Novel transcript truncating function of Rap1p revealed by synthetic codon-optimized Ty1 retrotransposon.;citation_volume=190;citation_year=2012;citation_pmid=22135353;citation_doi=10.1534/genetics.111.136648" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Paralog-Specific Functions of RPL7A and RPL7B Mediated by Ribosomal Protein or snoRNA Dosage in Saccharomyces cerevisiae" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2.cover-source.jpg" />
<meta name="twitter:description" content="Most ribosomal proteins in Saccharomyces cerevisiae are encoded by two paralogs that additively produce the optimal protein level for cell growth. Nonetheless, deleting one paralog of most ribosomal protein gene pairs results in a variety of phenotypes not observed when the other paralog is deleted. To determine whether paralog-specific phenotypes associated with deleting [RPL7A][1] or [RPL7B][2] stem from distinct functions or different levels of the encoded isoforms, the coding region and introns of one paralog, including an intron-embedded snoRNA (small nucleolar RNA) gene, were exchanged with that of the other paralog. Among mutants harboring a single native or chimeric [RPL7][3] allele, expression from the [RPL7A][1] locus exceeded that from the [RPL7B][2] locus, and more [Rpl7a][1] was expressed from either locus than [Rpl7b][2]. Phenotypic differences in tunicamycin sensitivity, [ASH1][4] mRNA localization, and mobility of the Ty1 retrotransposon were strongly correlated with [Rpl7][3] and ribosome levels, but not with the [Rpl7][3] or snoRNA isoform expressed. Although Ty1 RNA is cotranslationally localized, depletion of [Rpl7][3] minimally affected synthesis of Ty1 Gag protein, but strongly influenced Ty1 RNA localization. Unlike the other processes studied, Ty1 cDNA accumulation was influenced by both the level and isoform of [Rpl7][3] or snoRNA expressed. These cellular processes had different minimal threshold values for [Rpl7][3] and ribosome levels, but all were functional when isoforms of either paralog were expressed from the [RPL7A][1] locus or both [RPL7][3] loci. This study illustrates the broad range of phenotypes that can result from depleting ribosomes to different levels.

 [1]: http://www.yeastgenome.org/locus/S000003044/overview
 [2]: http://www.yeastgenome.org/locus/S000006119/overview
 [3]: http://www.yeastgenome.org/locus/S000004947/overview
 [4]: http://www.yeastgenome.org/locus/S000001668/overview" />
<meta name="og-title" property="og:title" content="Paralog-Specific Functions of RPL7A and RPL7B Mediated by Ribosomal Protein or snoRNA Dosage in Saccharomyces cerevisiae" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/2/591" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Most ribosomal proteins in Saccharomyces cerevisiae are encoded by two paralogs that additively produce the optimal protein level for cell growth. Nonetheless, deleting one paralog of most ribosomal protein gene pairs results in a variety of phenotypes not observed when the other paralog is deleted. To determine whether paralog-specific phenotypes associated with deleting [RPL7A][1] or [RPL7B][2] stem from distinct functions or different levels of the encoded isoforms, the coding region and introns of one paralog, including an intron-embedded snoRNA (small nucleolar RNA) gene, were exchanged with that of the other paralog. Among mutants harboring a single native or chimeric [RPL7][3] allele, expression from the [RPL7A][1] locus exceeded that from the [RPL7B][2] locus, and more [Rpl7a][1] was expressed from either locus than [Rpl7b][2]. Phenotypic differences in tunicamycin sensitivity, [ASH1][4] mRNA localization, and mobility of the Ty1 retrotransposon were strongly correlated with [Rpl7][3] and ribosome levels, but not with the [Rpl7][3] or snoRNA isoform expressed. Although Ty1 RNA is cotranslationally localized, depletion of [Rpl7][3] minimally affected synthesis of Ty1 Gag protein, but strongly influenced Ty1 RNA localization. Unlike the other processes studied, Ty1 cDNA accumulation was influenced by both the level and isoform of [Rpl7][3] or snoRNA expressed. These cellular processes had different minimal threshold values for [Rpl7][3] and ribosome levels, but all were functional when isoforms of either paralog were expressed from the [RPL7A][1] locus or both [RPL7][3] loci. This study illustrates the broad range of phenotypes that can result from depleting ribosomes to different levels.

 [1]: http://www.yeastgenome.org/locus/S000003044/overview
 [2]: http://www.yeastgenome.org/locus/S000006119/overview
 [3]: http://www.yeastgenome.org/locus/S000004947/overview
 [4]: http://www.yeastgenome.org/locus/S000001668/overview" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2.cover-source.jpg" />
    <title>Paralog-Specific Functions of RPL7A and RPL7B Mediated by Ribosomal Protein or snoRNA Dosage in Saccharomyces cerevisiae | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"CunWiEI74l45SoBlYJ1AR1s6DEVEfwjCPaXPIlITUnE","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460584","apath":"\/ggg\/7\/2\/591.atom","pisa":"ggg;7\/2\/591","ac":{"\/ggg\/7\/2\/591.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/2\/591.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/2\/591"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/2\/591"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/2\/591":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u002273oBJViFzu47HqrQ0H1oiEoGZDGiZ69rYrjNUL0MgdsWZBy1knVCQgZ4qzzWOX6xYBdlFDHjfROhvCz8Zor3NBPh18NLeFba1CrF04RVYYpkLOmp1BEzgH\\\\\\\/yzp25\\\\\\\/DBLn8EgBROXkuD7D+zXaBrThlu3L9bSdWXH3o3K6CCKY3mmuwBNKzl1oA9g\\\\\\\/DSrrcfmoPXn8ZmHBCKD8NabaDH6mZLioHXF9nHRcgg3JWXdVF9pzI8UBFmOhDcBlrqtn0L\\\\\\\/6tzt21nAxJSAcey4hGpLfXuJGQsQLq3JkYc19a+ja32TPCzTSksl15I0HTOJTkocAcsaFDJXWMgR3CZPfTvwkXQGx9KwMx2Os8V6IN6Cj1qR2EnaODOqlipw5DGZE0+nrjVVjEOUlOokDrvMQmlGYUolYIbAaHfY2dWfCFwetBT0fGFGsOZNZH95yXkX2SFkK\\\\\\\/ZfxgSYAqsyChaV+4BZG0vt\\\\\\\/02XlU9+SJMDqcBatYfOU4CQlfzvPoI30xUkCXwEOwStM0JhW5k7oPTNC\\\\\\\/VvFRGN7\\\\\\\/tCD2bsjolUGXd+hShFfVByy8tlhIzDstQgWBt72e+DSSo0Udt9ewTw6nJPAwyVy3BVCLeDOflKKO+S1hLcvKBBkU7ZavJlrVIDPlo23sjzUAv\\\\\\\/NIWkUtgWgJtpkCMWTEu8niwzSMr3apdviCZerpUjSq3UVHR8BFB69OOkngMsikgwnSqPsMs3tOyfnEW1rqk4+4sIZS2DGh8tJQf\\\\\\\/RjhxqThkvZjvOmj3RBbXHwHSzcZAIVUo+RXbakvhYsEwpWsiK490E\\\\\\\/oZKJpF5vA4gzMBukzyVVZ0ZGk1\\\\\\\/DX8SoO+Hi8XfThkpf+Bk3yCXtAbQOy\\\\\\\/G0b0vbjFqiFd4nngkmZBK5Df1ReJ4X5OxqmMzBOgboHfclj2G1pis9M\\\\\\\/64q4oZIp+uSE4HdOJ8iTFfVR9W85oH9+JqK0HjPRffXVVqk+55lWly3jwc3OdA4Vy1d8fpQHu8\\\\\\\/Q+siL8UFkzkXWesYbUiLsMFZZEhVwE7q5R\\\\\\\/hU\\\\\\\/sGrg8RiEOgfvIasUmdtVlDmbC3Dfcw2DNu9tgZxBKly1M6+mHJYyYXHXUOInSgAW0ugnd+pNJa7naVa+uNoa7686lblPT4XYtp4v06Io9DG\\\\\\\/0DwfN+JSSOFstiddFz50nDon\\\\\\\/cpLX2rhMhEsDINbcJf9ngFlzWYJa3GToR0rGqDWQ6IM6AXbgv16lM81ss0Zd8VKCYtflHzMxnMqx0nP6PeXO3lkFfoufG6\\\\\\\/L+md9Lcbt6bMLvgILMog8k2\\\u0022,\\\u0022iv\\\u0022:\\\u0022+UY4so23S4Yz+NppogKV7A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022183b7b01165ac65ef271ca48d53d1e77cbc7d2d7ad5e87a191bd34b9015b8e21\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u002221vQIDOQ0X0dl3vnh9KbWMe7fIrFk0hLbz6LI+8F3UaAHBcTET55Tpl1gcL96K5SSqv\\\\\\\/Hma0CrL1m1Ijc3tuBl8PXKDl2AKho+Gops30uP1urXmV7wMcA5V06xxjQu7r6B\\\\\\\/\\\\\\\/woPNmNckwBK5zIi30C8gxkihYj8SlZuP3oa6vkYhig\\\\\\\/gKSKoKpL8wVUupiMF6ZV4gvsdptKUgu1EmeGgY1jLRIsB8eQg1mtJtXOokTqBhzACYRXL94FghF4xpRlLeX0BeywBAZvlJz5Ykj2nVcLkKapSpZqBAemXG4ZGyMGsqTaOVF7hWiSSUAKktYfI9oim0Igf\\\\\\\/oecOYv6BjSpE1H7tkllEh2ZNyfdgl0tPHEVk\\\\\\\/Liwtchy3dOrmFaNrwmH0lxJTDaeV99XVhYSQir6VVBUJ2wQeGtGkrFFWrrSULG3cVNxwYIjydhxKnouFfgA80WZzpIhBrOwAeRmTz1k2LVLozgdx8mTd2YzsnwXgfkfm2+7Zgtw1Ao+sNXtPbplm6EBecT5h\\\\\\\/aLH\\\\\\\/\\\\\\\/sZf7Q\\\\\\\/Q2XnwjgP2oY7sqMhMCDWNtI7NtUnnZhbK116WT5n97qovupz5Nzco7wQHX8tZXvb4qV12zBtohN9zaD7JPLKpS5876KlRTgQ2ATLZ+kMiyHlPyvdhZQCBWNlSVPyhEu614ffO3UtHZvCHjAMqj2\\\\\\\/szxK9gviGWwxsjPsw9v6QCwdSEJyPvf9h7opxcqWHezcrbpcBYn2RhjrpUtkBKZuHb7lPsyn0MXGlTqk0UWpVAs7Yp15Oiai\\\\\\\/wQ7qME4m0PNslSmjv\\\\\\\/+DLwJxR+3AeWUdHVzktKXQpWJIsZW30RFl57pSLsV8A0vanZix9bbWd\\\\\\\/BDmG4S+B6uXSVB+6QCnbu+Afo4cx9XqS0TCVukzR9Gg25NtLYn0BGVPZk5a4ALno+zoRAdR5FwLep0BeMn3\\\\\\\/75A8yMrIXqPU\\\\\\\/3ehbRL35zxDQu\\\\\\\/k376xduT2QkCPuiW5TFTG2LDFTt56C2e3duoOdaWv8z8LZOThFS6F3SVkyk6oT\\\\\\\/tZ53BoZwJZi\\\\\\\/XupjvzF0JBo9ndlqclihFUpwNeYpbIFpyklJ9IDP86CZT4g93x8wRY4gmXjbnUoVevGVPnnLXKBF6f+uH7vFR4RNJRaoqMHCrL2lQ0p+wbt+BycsFOchZoPiDlkn\\\\\\\/En7ah\\\\\\\/My8aIKFWN7KR5WmwpdbRBi\\\\\\\/fSTgnG\\\\\\\/uweW7NsWXcLH72oGSQZ7LZr0oUgL2DE7mARtWpaHFiluLW0N7wP07jz1XIgtd1gtWdvIUYfcL8a5+TkajhVPw1mhmX2GI97J5t+YkRF9sFcdxT3aEyNjEgoDLfAD8m1S8qfg2dUpG+kScqqRpNjG3SItIiRUFrQ0u4SPNMkt\\\\\\\/SKUVmzBhL7XEwV+q78G6hN9MCtsNnUPLcKun2qtRyE+c7LIK1buNAuW7dXJvbZmjqF4kG+VMitPfyAsmrMTAdXKkIs2D0j5JxVTrHVXBfFHZNQ+oLO+s0s9DCo\\\\\\\/kusrATjF6XcWCNvSB4\\\\\\\/MAUdv5G9xAw3LsLlASx4uLRYapLbTFnGV1P67U9GJw7824DbDrSnQtdmWIaGzyclwqfReljHKHTfnfM34LVPbyeK06\\\\\\\/jgnCiSU7qf\\\\\\\/xweKAyl1YbNYA62fZszoWOvh8NuL3DkbWZtGRxLqpi6ajvx1XlM3liQM+YN8X0cozx5EeQjETPKpf4MEuZNI3t2pcPA\\\\\\\/QSr8QgbIM\\\\\\\/f7j31h8jHvn2FPApVBWAGDTg2Kxz3n2yKqVsYniaBQWfQqiiadcqRY2GbGfXyNvxJRnjmms84HQUyp9P6mpfvOZJgXC1i+Ky41XSUEvGMSN8N1iyhlvSb21oTZuTRgMpusj3knaMUvd6BHTT8MQ7H4ElqbpdoAFIF+rwIVy2KEMZ0PesQsq22oM6Tor7SfiQlyQTLDFfHVEIX55+xumA9Dcxpu7wznaQmvIAgJ9wtujotFn\\\\\\\/HYIFlZIKc+rQ+aIwF+u9OqENMtbAcmw7ybVKSgdycz0ww9OuUdT6eDj3SF6UaPILEU7xBQG3QDEkYiWsfWJuVs0vE5LnIoypkOGZHlK70F2ZFt5iBlUVybzwSeoqcTnmm96eexlp9OdCMB3vq\\\\\\\/SJCsiZ\\\\\\\/1KRZjFVKn+4YWUIcMcAu\\\\\\\/IzSbAURjPN9oDK9Q6crWEL\\\\\\\/3qGAyxaJnB6XBx6DxZUAVulmqbIvBy9dTJFKboe\\\\\\\/qaIjTTXBVj7BF6Yj+ORRmZCQPxZ+vIr2dFmjcnfmgRX22UeaIyL4PegG3iI6oAVX2k\\\\\\\/T1P4SxxDsdr0ghch1vEfRF+tapmGylt66F5wYVkQ3wtSk8ZA0IKgeyY2ncVo3vtSp7+ghF5G1HBGqfR4pq61JQtxqDBJBnHqiK1FsCPU8XMfi2yaZ6dRvZGQEQstE5PQTM1g+kXpv7HmFnVBM67eqfPTfnjwJq5EzNIvK6h2P3ySfb9j\\\\\\\/jQjTEWDujlo576A+ZqhqF2smCFBZKVKkDaKec+4B9zQvu1nKAQRV0zAy4JJThhgYMuxJVirub5SeV8qbKB9Cx8teHNM5Mhf6giBmJGOQENMpIcJQcf4BWy0tYUlqoatwdFAqqY3OzY+4SAtctY9wzlAjLIeYxZwRqL40zGz0w1sN3Bogf9jUzyIfEgPvk3DM8l0HiP5hTEn6euyj3Aq51blgFLQnjS2d1BJSIPJGDsRWZG\\\\\\\/StO7DXCBTkvLCjg7f4mJ5vcvZ6bPJU2kvSyoEE5dHSLdgk6v4qCHuCJDp9j2pKCK8D9B4YljCL1c4GqzFJ+6O87tu7pocC1\\\\\\\/ZxoZBNpgCjbqVejeucEcCt41WcySBa7m+8E+3QyU5bvNq5+8UhsVN0pQToVeeTMEueZU4NCRYvpizR6Kq8nzptwhAd4VApKIdh1S8TPdleJtNR\\\\\\\/1WAxUyD6qbLlEIum7tcvD+8Q405La8M5867DncKWVsZ+IZq3gQG01kjFpQ6crWcRCvgwE3mf4rHAOEmUGD5MHQDUPE4lLSfWvIDArYD3QldqQtDH5Sa91dgbLt\\\\\\\/MwIWhkuy5avJGm7Iz5ieoy7WbhFAR6bDb+\\\\\\\/OdPbMriB9iqxapuIdJLgaKXjDqF0Hphc0x1kU0\\\\\\\/KZhvAN195jxnn4Kr2+9ryBdI029jHaY\\\\\\\/4Uz13x0WTUdo+wcJgyWWBc\\\\\\\/zPtOMVYnqXK35xtToARHrmyQbvp9rd2MaTD10LNU94FVJPjRPz3sDlAxb5K76qfPeL8pNPZJ1LLi+qQwjmQFfqSP4D19pO+FOH\\\\\\\/\\\\\\\/0mX+u\\\\\\\/iWTWpx04pxIOneC2bJg7Uf39QMyysjE6DHc9qts0U0gBipp1W24Np64u1i7BmaLS8\\\\\\\/dX0NgQzKqOO0bB2RZpkYSp93NAmb4q4eXckU2bek7+HDG\\\\\\\/2oDmJvOPsacTGrKRw99iQBXVbqOr3ihn9dTYoDjEnIOPDeYEFDOfbyfOfcR7h+ts3PXS6rE3d+fJrY1Pm1\\\\\\\/Obrac5ftRMvwCM8Vrq4XqcVoPk3vE9rIopf9YL6ZzouseyBmpqxhha9q284tQKQRwWZblkKRtZ+SbFqqyJxtD2upxNuua6tPZeS24ZlO57mgPimTntendmA+FdD\\\\\\\/VbJLFqs5u4lHrnfyZgYL\\\\\\\/XPZJZjy6ndouiNUxanYXzgkhpb3TRFFeCY2s3sCS3g5jUIYtJKWJACq83lSCDPgKDbVWN2dIu4qC9LVou\\\\\\\/\\\\\\\/uIvB640w9t+nICmnStRQYCgCC5ZTNZuM44HzE7fe8kVIty14dDKR\\\\\\\/2hbZqtoLFoJPJ\\\\\\\/FBQHOlVKcGmxBUlYyXR7FFwBeiCZLOhfNewCzhzpX7S3NPjD22JnYXWvrJOKWBOnCPjbfABRkqwYjBXVO5tU82omhbUuZOg06wDcxB4aZYFXG\\\\\\\/qEQhA+II4nHdlA5eG+W3kNwLCcGfpgk19f+oXN3vI8b02uatE8P0RwKyr09w3QcJbFeaRX5hRSG7q8QgIZ9+q7yXcK+fiJAHjT4uROcOMCwsHavDQbuBy+yT1Pad8G\\\\\\\/zBD1j4kgcHhDk69HQV8VFobLr4f56ZKi\\\\\\\/gGwe6wG\\\\\\\/7PpIRRZGItpaS+YkR2NREm1hdL1M579R\\\\\\\/l6RQvUcve\\\\\\\/8OdGVdLkzZgJHp4kgOpkfOaO5+ooQYp30qwFtSR0\\\\\\\/aMvkL+tRB\\\\\\\/aC\\\\\\\/D78mfb674oHK9Tn7dgl4y2WKVtQtj3ClmpcPCiBLRbm1dgPHRJhkHEu+UunlkdGNr5hAdr0uExOjfaTXAqSSiYWza3mgWcVAGbhV+YKbKQuvcd72yBBTP2Xenkien9UipX0uQ6fC2JZDd7ilK3S0xBxwtwhPFZkOrAYbDwa7LDHHHA8Pqr6WeI+fXHwAuqAEIv8rgfBsFcJVrbgfYO\\\\\\\/vNzKbcY2IZHysDbyt\\\\\\\/JM0juEFoZ4ngIkug0\\\\\\\/FFqfVeD8D\\\\\\\/SjEEQPX4T\\\\\\\/ipGVEVjsqMjZGO+69erIbz2zRrmZHNold983nVWqm8+26q5Doy\\\\\\\/XHAx83CVwm4uwFdXPF4JaWH696RLRzrpaE5XSoNS\\\\\\\/RCbkiHLy+FbRDMptC8jv90AZezZSxUn82+JMHTEyTiklK2qqFI4ydYHEfGmlY8drQWOmvx6\\\\\\\/eh1kQF+5XgtsqGMz17Upq5PPRn2yi1Z8mFT1vUE0pRLMp2af+bxMyBDqSgWFaz+NP5mOyO4WS0T\\\\\\\/0UPgT+tmJIvQ5nkNKxDQeaVetsW4Ay\\\\\\\/Cft3EjALN3I3zF38qc\\\\\\\/tPZ4o2Ba9IKXmW3\\\\\\\/w9wL8xYhqyORwDm9nFLXD5n7AHnXIAeqZTf3nCvARmPWyQgpBEF5kHBscSd\\\\\\\/0J5\\\\\\\/zuq8Ud1Z5aN0NVaLGPltOLmTiSabJEasPhtpt6w1TeY7\\\\\\\/0zJCHs6IB0E4x\\\\\\\/o2ilYg3tQjd\\\\\\\/2GJ6UZati2G0I\\\\\\\/vBrR64YA9CEYWB8bDyYsiJC8JKyIE53iQ0YkRjYWuK043x7dLO9XmBvVQ6oJ6nk9PTIpTQi47FpDvExlR9hluqgvbL5uLKKdoRGpAkLabRae\\\\\\\/lzc7AdhWNxbn5Tdydj2oTBS3Yw2ay\\\\\\\/bBO1+ZSMU9PvX0Ks23bX3RmmpoXubEyKkapTn\\\\\\\/gj56R8oO21rFWpF+0J3oSJyfODQ8NUarkM84VjKNgRkbCV6SG6hO8Q\\\\\\\/0ysm8w0t2gRXgsxx\\\\\\\/A+SqnCGL9yDQYIzmvJIZNGeLtPeVF79\\\\\\\/FhnPPsMhlMUiJMoFTiqH3trLUEVcjaziKcdw4PsAg02C0J4UU3Ox6KfHaeoQKR8i4c5IW1N4ewMFPQJCFEZcehCcUds9mCD8Wbxg\\\\\\\/DsQMQstgtnl5ZxVO4kb9FJzK95NGmvUIwlCo3HnShMh5V9YzEU+m9revOMbgc+nwOVaqTuMmEDyKIsMAsCcof\\\\\\\/NEwtSSDTrAOrCC7AcB6frXMauqREZasheA\\\\\\\/Ll4wAWQ3A6pp0AL5R2+WpKFHROskSXryOt\\\\\\\/8AmectqgsXMQADrHlAJkUHwGx0n9mqj1M63riMl18Xpry0163DEQuvdmr20Ci3vLlKl7UYUZKqQ5WUgORWZQoMjOyUNhbl1rX+CD4zP2fvG2Re0CvuWBWM4\\\\\\\/FyZ2aRmavQ+7loMyCR4ISP6pmdrZbnM7qy2\\\\\\\/5sIcPwztnKmtJ4bgWNyUryhuZrR7p\\\\\\\/t7ONm5YOEJ3P\\\\\\\/r9yr\\\\\\\/9g70d380WVKB+big8CseUHCZFOXehLwHly0+x1+8gn8zcPgCmOgrBX\\\\\\\/twBNp3IqfoVPCvOHYw3MDbWUDlzzSaL1FfRyxKjNcofGAcccv8+LmltvVNp8HBr657E9HVI0wICPWkECnTdmt5XsYTUzK+rRPYa7wrM20Cd6vETThBYKvl6GjBkWUIdd9NvgCksqhdSOoUmxDxmh8yAmRH8y8OfaBg8vtr5Ma8\\\\\\\/9skrh+A8Co7w7p\\\\\\\/SNo7lw+C20\\\\\\\/VMuQrs1Np8v0AW4mu+Di9dN2A31FqJktFkeMvhO0qhpVOR7q8AG4Ql2K1zxAMRIZNBkdCqeWz4DKvjbn5YZdVZ5FVLlVDX+51qRFfMGgtcBPf6JvfUvdUp4+sOzoyjQi2ygBfIAF7SfnyEzvic5jukVcNn0BaQZrTBekGHQSmn2SuACjYAmWJDqWZR20z5Ybhpr+62nhFojhInJ7lQi+Pv\\\\\\\/VoyYbB0md2AVnU9bbZSRKfZGqKCHfuw1WyijtuYndMCYehyV9bNpQqzd9R2ekCZPcFmZFaOoSlWwtr8esNnGiHosF65YJ4WIzxjR+dGzMlAUGNQVGvyVvVZbb904ptd2z+WUYd+18x\\\\\\\/nvdsX5F8fdBVcUb83HOsrNlaoW6CpM0GnKYTzy8LuKcM6OwMskFiU7hasZOlc6vpmpPTEgbw49R21gtRp1EHCZg4jDlUvF43kK0TVhSJNlbnN0Cq6kWCU686UaEKpLWi0JyhY4GYT\\\\\\\/r9ugWDR2v12+E2hiVmDs3boMqF7vH\\\\\\\/QdlXpBIAfFJwF55SVkuvRItvv7Hgd7Hn3ep5L107gl5HR1mPeZ\\\\\\\/d7aHF1Y8hR1mZlhNZtv7cTHuMjGH7OrRe4i719j6GIQ7nUYgnZmBLO8VSuj+uLsAQBn8dGGVGI6tdiq\\\\\\\/C\\\\\\\/Pqzq\\\\\\\/RamY3NheqU09fiJfbBQ9x2nwbgByGm9V0xONsdHFKp02WkjmNWBjMjHds7wpXc3yACmb2oOi51Egr+oSiXIpxdy0OcJjQnNI07MiOyOjms1XoyExH8wRDnYt6zl7leHuibGvLTBcRqCuLJqbZ5m8t9Afk7EkNyDBpTTZ+YGOM0E3AR3yiM8Un9ZZmMGGqNeShnnGZBp+jIC9LTCjejJvkfaJhNDZnzsWAraGSiPcyi+YSjaA3HEAOYst1ACk7PxbtGjcOw04jNnSaOWVpiEdByKzp5qFOgjSssq8Xy5hS4bfK+oPzysGGntdGRtkFXRg571MAo77l+mvMtZiKQvLv8RCkMhuidpEfsvnsgsz0ANvsz8SwRLHjDB2WB\\\\\\\/2qFnfujUPYcMn5mwFXIuJxzrhLNbmsrEMRhu69+zrL96FLYH8wzeqF\\\\\\\/bKgzyFOoug83u75JIbAEKEeNRjCXK1cpkTxFh0\\\\\\\/69UcrlZfEba\\\\\\\/JqcliIkLjIs+jSbSEbsLRZ4XklBMA+vQ4QF8nUQs3+v2fojom0nuQQygVNgICJw\\\\\\\/ovJTVVse15ECitMMyIJh3FjXrW03SIaxuk+OOpdDDnttJfhUDggP7lnpj6M8BI4\\\\\\\/wG6+HfIM0R4E663HWW\\\\\\\/WBHk2fsFjCPJMr2dNwYbCPF5WJdoJRSDaEJMVZNGSluWMdxqwZoX2HdZ41hqR+zaMToDYglt8Lxu2NRMWOqgTyS4\\\\\\\/KrMxCixtT\\\\\\\/pNktm9gGgt7e+Cq+B7phmFMYHRFQGgzf\\\\\\\/KNorclDC1qlXZ0xEPdXGUQijhaIpmGTPXGaSnMPf9ocDbAsQo6TSvOy8jrgp2DhRw01v1baTWNzc1MCaqrWv8R+CC3PG7g49MctEyRmWwVPH\\\\\\\/r1Fbdt0RA5hdGxev5+T64PmzTTE62qA2tdwA6BDF\\\\\\\/tXVjvdjO\\\\\\\/YkfLj1+DRHynw3QkPphnkn2X3BT7x3A+LT7NqFrM+cUjWCv+KyEh0EAvxK4c9IrSyRMPnzGXCBiF5Zbh0gM0MUCMgMmgc\\\\\\\/Dpzlm+JKsBdQKQAEQPcfa\\\\\\\/gVNM0IiTVPNPbgQOR9TtsBeCmcSvf+AI47ZYNt19hUkpol6BdC3gHYaJNQsMZd7cBPIFK9xh0so6ZUs\\\\\\\/oVXzQ6XMg3BChzmJPRNtwh5lIRcv6SCOHtN7vRco9lRCqEZML8eWQJuqUB6LHu2OfunlSXprec6mWrQwTPkRXxXI+JYIc5+AMFTmE0fk6WojDgg4HyE0v3Oma7v\\\\\\\/hGw+WEKhQ6r\\\\\\\/lYbGUu8dXYKh9kecHWIklGiqcYLmKzVxT1RamZZ83cpG78H2MUAw5odX3fKSFlmr0eptEkyGqcwuwEr6OnUmY+vCE3ZX6mpk08MAIhzcdUD1\\\\\\\/\\\\\\\/rALrVJ\\\\\\\/4nxsqvI9ZI6RsNUH7HYJzZj9YKb5+qFGZyxx+Ydu2mro9vJjhqJA1nwbhG3pPZ4YT\\\\\\\/8r6KbTu0jLNG0xo3rgHV4ZEv6VZH7CR9Q\\\\\\\/R6xYVW0WnWkFoLMs53xEKAZRoTmRY5nIxHSYJPS1YtNljLiPtPIzha1kTE20OYLQaOGqfVpaAJk8Au8wlGh\\\\\\\/UshLiLr9ZDyTwnIpYXYCYMf5livZxwz+lkacpVvv+1b2BsPugptRHJdI39xOLehMcPGNdE3Dfp97bCyUTd8HidzeySskKGDP3VwMMbZ+WhqRGvixc\\\\\\\/kL79R0MGNxqXucj8kAx9ysRLkb+3hPGqw7NiOZEcF1uxeNXS7RuwolkR2FhGahDuMRkyZbIc\\\\\\\/O13ZpVhnDwpmwR\\\\\\\/61PzHokBHU6jJigsqwaiHQ4BpTa1bvdNiFybGTkpY0Kap88O3z1RU6QLK6uP\\\\\\\/xvCXmoSsEx3dM79z+o1kTML5hEAAHdk8SwoO3i8InemHMwQhrEOZPiSF+IjuvArtu+leZ7qgctzGyWO3nSscr76j\\\\\\\/BYCd7mKhlWnHmcC4CgG\\\\\\\/GxdsDws9\\\\\\\/6CVTKKtKcKctYykI61KEbzB9rUeFjyOL\\\\\\\/TTviDjiH+Q54rRRE93O3QFQKuQyfbJY\\\\\\\/NC0LysRtnKY+02tSMq8gOLsgikjwgKxW+jYLpSbM5\\\\\\\/fZgo22qN3UAZmutVehPPu2FxGWCKy+apVVWVNJgZnz7QhZqwcJV5BKonWXYTCFAUlRalZtcwZ6POioFk4xkhAH\\\\\\\/dTRdWEbQXNNgbwEQQcyMGkB26piZ8F1dy365P7vFVbuVu5rDP1IoTm295Co\\\\\\\/RkU6HsOQnVv1ymAC1qghnAbjlliqj3RqPghk1XQUIgaES7qoTBY0GyN6No0Yp\\\\\\\/vqOH6YxTgCvrCULr\\\\\\\/fmm\\\\\\\/L1PzYvUaHAaU72z\\\\\\\/PajB6Dl4h+U6Ygt1r9txTAzlCmJK+Y4NDWDDT2dxl3PVP\\\\\\\/5lbmeWAHam1GHH0aoZ17kLKhkikKG7\\\\\\\/7p13BV1K9DPugb9kfVEjwjXwKCyiM4F6qNQSrzad8+nNYIoKMOEUp1UCupXtrYGw30zuz21csoiP\\\\\\\/tfeBjjTGRaoggwCR\\\\\\\/N0dKrJw0nYWDnP+BTnQAwSBjKzONZQsH1touQGXjWmS\\\\\\\/FIA8dcg1Ks3+ygeryXMURzZ5gUPTORx7oyMDkEoF2iqIgnXORNyUJOdAZyRIvdZ2X71cvZk3NFRHrQhvEMYZW3yYPnRnNv2tkoDaa8BcjKOMdlaKDw\\\\\\\/ovsn+EXokYhg+taEEjINR8KD5J+yPV3yVThfoJIgD1xqMb0apEAXW3Doep8TD4BP4WlnuIp9WYKnv1AbfOg0++NQGybYkR1wQMdY74HuRGkiWPO6h1tRnuyMZMPiqrNf6j+805FZUbfBK1jdTcqeDD2\\\\\\\/PdHUJL0iKB\\\\\\\/k7k9MohRw2t23Cx60r5aDSEzY8yUVQI7LKCFWCM7cDpacc9BKfViVmlzSEkt60Sigu3lx4aWLRNfErin7v3QaAImZjgWAsFw5e6k8pcn4M+aYaMr84giZ1yqKRxx2MatehzH5v6USYxEcZ8CfzursV1oj5EOZu\\\\\\\/1fsoj0SIQVIgfDHgnno+\\\\\\\/oRtZLpXJ3C9Er081rOUoeTnqshPaRQKhx+K51sH6WEmQr\\\\\\\/LFhsNkZ2wB10+\\\\\\\/b4Lzzijczyr1Ab1H1qH6kbs4G2ktPxS72cg\\\\\\\/xEr89zpooJ+HaatwvPjBYgG9VOLD+X0G8t1V3Fxm3alFK+gK8J759ghlAtIh1f+Q12PIXaXpUUlWb6UQ7JurBnWtPydgv2durvyjtg\\\\\\\/W0SBBiaWUdeLJp\\\\\\\/SU\\\\\\\/JuVHXzuIeQcZZaG6\\\\\\\/h2bL5mvPrf\\\\\\\/qHLCcrphhCRx8muBr+x7apxrJFhQViWFtMDSoWvh4xDpSQpCnkuR8SqbZpXXikUE13P0sAAawLHmRfleXQHz8FjQ3LEKaNMzAL76p5Lio1wlpIhk89F2KeND+etJ7+WUo5mFBsxgAExB9WCTJ47kRS9vH6r00B5ANYV7+jrki4cIWk75sV2HhqK8YUL0K91Gm9oI0VR\\\\\\\/jL7WPIQS5eZO05nFJVknB1wTIQQxf9VIFrUKeeLTMC5PXZEZg8bvLZ9gyqbrqhce+KuM28jaeoEVeOB981apWnBcjbOGttq9bb4TE6mlvOMveHMZDXDmqDlf57QbjQWi64mo2zU4lBT5a2KWoUz0kyG6t7u5FToyoRiJiq+J8KswRcf3KmjxDoQoq5PhqiOf2Y8Q4d3fBxr+xHqbb3eDHTtYiKxdg9lwlhSPuaP3cGV63kDLpZO5xqGuuaWbZBCwXhEHh8wWAslSWxpM+iCwd4NPXZ1nBM\\\\\\\/Cs8J+CwxvUKj7nJ4q2vZymJRxEbnsTb9FYYb\\\\\\\/DxLsl5FbgD8f6A5oZe9JZ8MRzncEx5+spaNIIumOI8HHTYF6XNjeprgEmeP9Zcm1o2QZHbil73XeOPXUGhzHYRjUxBOpb5NGkGBlGHRBM5TWzU9FQlZxaSOl\\\\\\\/pkCW7GGWxvrcn2hfvw62dzJ7VXa2DJg8e\\\\\\\/QhOudurUHg7AbxhZcquexijQhK92QJZl67VljCPoi8QWIKiJjnPS1mvBOrKzfTWDcaoBJsh4m6jJCLLlx9a3C1TjdBZcjXP2OD8LTlrqFY9qDoPgZdeBAEM7HQg2LbEGbulm9bNUgn+HaGrK5nFE6qn41TdmlelxqzSuY0HjNmhHrHbCI5XBrheaoczOocLFVbMMCfBSDj3\\\\\\\/EIISZLzdwhdGRy9mbpe2UuUZJ+t3FbpQ8Khd+5oeEljpWUauTzb\\\\\\\/Jq\\\\\\\/6jvNYh216TbXd0Rv93I\\\\\\\/oJGhJaVptQQbGBab3t8JhhAiDJWMcreO8ZUoONqSl+GQVCX4kU9Ki\\\\\\\/Z8nqDW3pRgM05YhttvJARyQauiptoet3pENeQDSxxLTgYQywALKNI6ADmbKx\\\\\\\/ZlGE+yH5Vj5pXJlIqEwbFhyzGGzkN3GVhkqMjhIYQyifcJNI3+mJq2eRvVBzzT+xwlO8LwYicoyyWGDjy0mUFIhCS0SgBvpWDfIjh+a0lc6IpO+JSrZB6shR2Ok4aMy68FlRs8sIEpdIPxpEgjkZjqY6PeZNG2OtWrEE3chhSz\\\\\\\/YapmknQi8RCY4gY38XDKSYM0MDEWb3wTY0eRs9JKbr+fbwxea3Qy40wVCfCuHPIQkd1pD\\\\\\\/WK1Fjejcy6x2BgCe9BAEAv5lccd8imyv+OnBqc1iWjSCE0MgtbFyh1ZD1F\\\\\\\/osb4ZU\\\\\\\/sRnFV0pmF2YtcPkbkYSMZCKqDl7RCUOnFfxipB10IDuuUkMwLmit0330ObOS1iA5c9yALJ8Cre9obTxdH7h3tHVK\\\\\\\/yFBiKkiy3AFogL9twPpnmSbSQOyDN0ja2jhZSp0QWt5\\\\\\\/e3SUrfgpZjCdJ6PRNPn8U233NUN7LA5htQVT9qtRDd1ypdpFogKVDnCQLqrc1hYn1Rv\\\\\\\/RZH1kcqmJ\\\\\\\/mb8Pz1FLiqGvO\\\\\\\/H2aNSTKWsCyxAzGIxM5nI9PrFpKjbW0x7soy\\\\\\\/\\\\\\\/ccnsd0EbdbXXwPCQ6GAb\\\\\\\/RNZbbdA61VBLAkDt9PPd7t\\\\\\\/7ZKCvLNAKvX\\\\\\\/He46b4qoPcs6oLEc9duKOk4nyah6byFu9N0ytq0va35Yufphpfc9xlYdVnFSBdpRtJDeoH6WoSZpxXPN6QzfpMxlu5K\\\\\\\/RQGyX9d6Da0PCWFFUvv\\\\\\\/yDtqf2ttmVZzNsEiz+P6CvjQot2vboA8wTzKJai75MKrSVJRmWsoMuns8\\\\\\\/L6j7eVD8Ec0lTf\\\\\\\/fUhSdF3LojugLmjHm3VMYGJLB7f8j7KuuOjZyi\\\\\\\/lJadI+xdOleCG+5yd+Hsn118K2FrHzMc5ghX0COqbddHR+qF2Dpo\\\\\\\/ZAbAMRV21jr1fUqZlzxNcStgsPw1iUKHL0h1nY1HkThvTs9UvZrtxwtz6Ki2F2dfi8D15hbg\\\\\\\/QK6RRiJgcjruRqGg1kjW8blQmOHROufPXyM+\\\\\\\/KrNdcaD4yRirLYKBMFNNNkFG+Ttyd49HLoQEuORXJpG10zEvRzimBgi0RIar2+aM9L2bcKuLgYOF57fBH2qtzbNl7yxY+Hz7vEApF7weTYyvyfkpUkzGrNJCOqJkC0HoSfWGR90fVsCAZUsyLgxsgOqvT5fUu8\\\\\\\/U2k5maSTrSPFkiUT5dQAKMus\\\\\\\/5ShLQVAZQaYYoj6vPmg1F8qsoZI\\\\\\\/BeqXMcPikokdKsz3hzGPupiSorXyvZQwW3vt+ca+IfHqDKreFHdijnG8mf8Yhd55BEEv\\\\\\\/SzNAX6lRZ3jnnLkaWKwW3sgr8a4JbKA53Zy1Abb3r6cdgWWKpqR2uJ7sCTHxNEWo6n7j51qt267jbmZtm7THslwnY8b+leRo7dkTrUZOx9BMqL0MtXZw9KpHxB7XWssn3PIJjD9LwlqcxzZj8iSXRQhi\\\\\\\/kfGmbd+xT+6TSExVyvvoauq79pjq9OrG0xZMQ2styWaThDubDkN5\\\\\\\/i6cC+wLqLEkgT2CGIvmXqMf+ac1rKD2Hyfiw+x+NewKwHfDlnTUe4Xa1tNZh8TyvMYenn6vdTRMvuRWPaixfF9xABSlc8\\\\\\\/4ZYkcwNHXhqrGqeaxNYzhHX1gx66bwpHwNe7gDvQDKST\\\\\\\/tsLnRNuQaANH6EIcms7NXCeuTSuHhQNgwqrRZNnVCK3bvRWgHHEjtGfrF00N6rYwCxvpV7oVr3m+QOQ2QxxePpq7D8TBswThnoOCguTscOi5ld4MD0MhidraLKpgYRuqiijaQg94CfetLcNhehGGQG2Rbvu1UhPciXA+oZbGWFXuLBj5MnkNqfTwhdSu5cOACz3YZzskc4wZ6xIu\\\\\\\/3MQD3ZijQvSa1bZjHivoelUL2rp58xyas7OjL\\\\\\\/uj+wnIX7aOseGajxWzYE4ZndClBGciAPeAfOg1uMJo8SZTYGy9Ps+EJ+TOXi8CdQWLjKSav0rFFxRHhu9OqOUgyqvA5j9XeCO2x6gVZRYYzFHd8QYOa2jGNqaI0Dtg76fU298co5f\\\\\\\/oF2VlorDy9fp54bLhi2pJGWncOuejgDMEwNOg1g9gdEaO5t4DJ0lsPzmv5XRDsfuycJZ2HDlohDAYHG4kU7WsnGdMcbglkp1veABpbOR1rkYO8r\\\\\\\/VCJl8QMy4LbxqZHzEFbkPi88nklO5Dl4QizDvPAXr6GCRzgo5W0pU6zuvAZli8yBrAVaC2o2LWvQjybM2Rx+qn\\\\\\\/F67TD2l5iqe4eKcxEJiJiNlhAqq+oFaqhksh0dhqnMFR7pqFpzJ7lLkKa3hqjjCGUmidxqPSMeJ3HGk0DE1COkMKPeLbFxYjG8QwjfWite8rvpbGuCzBidiWR22RRx03JppJL62eHul7HXT3pbZGGctRMg0QUtDuw6EaEMJBQosFmDzd+YBTq9j0aZtsfYxo84utozBymTvaB5rqEV4iOC7phPPsDZllE1ASWfH+OrS+tf7bJhz82p3EtLvtAy1WV8pfZrVXLkm9xkstIqf7CJTieaIAbXTF5OEUEtq+li+dRYJnVHoPdSbpYb0ceHMQAYb+Fxv0o4bUkZkH0Hz3E84Y5Nshj4j9PauOh1gzMQHvmJOK24b6fQ1LKHn2xL5j8LTMAIhb8AyJy4PUjAQPpaSngor3eSNLLe7wFsOH4RfLcC34f6Fm2llVftbF+5l3qJ3vwBObW9ssqVFITQhh1QtBKTCA\\\\\\\/Ee8kxBcH6JN7WcRQCIaMgbuRQHostej9XG9H7e\\\\\\\/4qB7FF6lIGbbKHH9oUw7oYHII1isqR5Vqyn38SQQ+6RuMwwYjwS9WsxsFre9jqKiI3YmfBUWXmCJnn0WtviNxPrCD3kmTYm8aTZfL1gKE4GmQP4NYry7lyyJgHzWD9uQrGsjhzaeN4\\\\\\\/zSqO6xMPFDzb2ME9E93U5jNx\\\\\\\/AlIdc03ORK33Mor1GYk56kGY+hljS4wSxW+1xRFQGuiJzNmvud\\\\\\\/3yfo6XWAQEEAAT2QDY5sKxE2S+mX4ryxDWItRc4S\\\\\\\/YXTAlYEbBPtn7k+agfXLjR8EVF+zNZKnU90d8XLKwsUS8PY+38ASDmHQoO3PLCgCSTl1ddp\\\\\\\/kv57o6kWvLQNXfeZeDVmgONRjLppJwVFBPvtZqZjfYHlZA42hmImqrzSod9HfLenNLRi167C\\\\\\\/6KVTe5MU0MPu8\\\\\\\/0N3gTI2f5HI\\\\\\\/9W5SFNUDv1EtqKSidqyHLCTTYPZ4VvcQMWxVNOZJkgTmu+AwnQvYk66hk3NDpOfhWpx\\\\\\\/rM93peu\\\\\\\/IIE\\\\\\\/NDTUC7FbVfSAYaufCwpoCjfA6Z7U2SIQ9p0IK5ScmFBE9tHQkkQopmabNPBBWYVOwBAUSiIRMsnYGVMhGIQcN9OFv20v8\\\\\\\/gWHNfwuI9y+MGE28R6igCf1kf7vj7+mngA\\\\\\\/7EuoTur8iUa\\\\\\\/N5A6MGqlp2vwjyOlG4TPMeWrfZZfbxTseMu2f0eKm2kXYY7vMwDXRua7AKgExbfpvvdcrJzVONpBa9hcb5QdMKjxqpLlsOBAdmN2oNFkNbqSKbOvoukD9nKrPRYEdtUUKgR\\\\\\\/7x2wXjvYxhFZmsYL1Mf5iMPATKlTvS54CHKzz2hHZdk1gjfAo9lM3qTQcqqr+MA++TwYN+XZdL2p49aleffdWgREn+OLFTN6u3Vu\\\\\\\/liN4HZ7WzDjUyzrbTtDsgJYlLOYgENY9QeyRtuMYb27HlaCrFPnL1\\\\\\\/MebYO9RZHyM5PWg82ggscaS3+ynYN3Itax0E9DI6EtCJZITeV\\\\\\\/hGteFq3GU6eQlu+jP6pcR0EEYJVI+7aDMNwauX4vJgarMKy7F5COZRxrOLlmPmJfJR2UZIXLhnGlnAwHhiy8wd\\\\\\\/LkDgrpnPNLrPbxlpakuiWKKaMUdlNdLN2St7cHs8uYjzNzdNz8e8p7bz3nFBWabtQRDgMznAEplr9XiVPBdN5+eGR5KDHCJxve8co6tfHl5IOuDMdLm0USwOCh473of4v7tM4bwOB44yKt1VxOA7PNYuq7ctKvC1BWuRUVbt8F9wmmyG51DkcFjfjpnzpBUJh\\\\\\\/pVkcEvMFEQvMkp3+XJAGkEMM+dv1EfP93+BaOJ3pdZn8A06O4JB+sq6gVzKZiZXBWMJmLW4rSaFKykQgvbJirJFef3TrXekGj\\\\\\\/Z658gNgQLUOk5XzmJfXfus7DRAk1WlLktPbDMDkgVatpvyWlwvZUB5mD0WvpCngsisJp3XQu4diVCObF5XFI7jU7p9cFM6K+rIsIlqi3MGI0k30+qYApm1eT68imr+f4KqoaJewctGJ6edX7IQNXzFSa9p1HeAr6bka9jUwbLG6I3gLd0dUdCM0w8HPsjhvtxJ+8gsaSo43UTu64PRy91VPsERo3li4U9JWK+3+R2x20IkvYLHIUERpHRbgPBRC9z2\\\\\\\/SG7ix+L605jVf\\\\\\\/kW\\\\\\\/ouXgeYmQb1Wun4zFUfcK\\\\\\\/mPv9yBvqtEmPLGuuVWiWEMrUn+9Mb2cQIPbdvJJqGi0f6d5vpqAp\\\\\\\/MmsWLGYRn82PmtljRAH06NdAEO6JUUrnBJu4yrn26JzvYWkNezQc7Rna5CNEeG9uOejjjNQseXIcXwyG84ECyaBEtyJql1eipsSaVlWUyWnPB9i22U+d9u\\\\\\\/\\\\\\\/q3Qjh4sMkCkpSRLCgbUPahQ1jX4J7rJQ1NxUFc56k4DZmaVSEYglRFXUZTM8DMrIQGL5+RGXTKQ8PSzLqaYaq1tHP8BRVVNZX07UXxZCyUhOUu9Se8GWIxJRbf7\\\\\\\/SoIHQA9sFF1tIa0STTlb4\\\\\\\/XV8AH4iuuvjxRXlWlvPfUs5HePB5Rq5ic7a96yvaMf6M3r39TerYMwb8Wf+W9ymoHb9CKd9cK8Nmhx8nTfqdrkwhmXKTWLrBEiJqycE5AbwsgR8TIMf2jf6tigVwwgi\\\\\\\/vL0hNVi4kpvPXnsn67lsI0oK6OzJYmZgham6msItGA42dd1VOLyGi7jpnXuhiI73aHz\\\\\\\/VAzJoM9BlRxpMvr1LxpXyI7yKuzKWxr8vLzofLDdtuS4ykD20UR7Ly2UFFd\\\\\\\/zBcuY7RJWAPpCNmuh7EK0KtBUFKgId4aA1aodgiLFh0nk6TyF1zHBDAh6n2e92b60xmv\\\\\\\/0Wsf+wlhzz6u4af+PqxvaE20403b4eZO7up6+onK12PQb1WM\\\\\\\/Bepzy4dck2sSwI75l2PmM9wh15fgxDazl6bXweEdLS87ZkwySNkk4LKpnMaC6i6rd54A3HsQqhQC0RJycGDVGtGD7LNA626Iy+vzRb35+IR0cpovqRafwKRDCagYhT7aGHX5DFwsXtJdLxN2tysqXRocbyxEZwTMx73GZwpRQYajdPdf7gg86l\\\\\\\/gE5NmQbWHEd\\\\\\\/TzZRu\\\\\\\/+hWs+uIv\\\\\\\/Fk1Lslmj4AbLl3C1V6Z1ArLRdxIu4HvInBMkUDqOg9WYJUEZzAYoM9FQOdFkuhQ5Gm\\\\\\\/BxftAmpxcpvPeWYUdfuRcw7Up698uZCaFDb4zcvluaNMDH6psJgywZcurx22k+12foWqtlvY5VPakt\\\\\\\/SyAxBW5gWGYfYagCng\\\\\\\/zQcgBtt86R6el3+FpH+VJLKDhJNtr2Kn7P5fVSW7mLrDaTsbglh\\\\\\\/hR8SmLXfAGu9LNZQ3IqG+zPXLRofVzn7EyqY4Wf01PDYaO5gXv+RPd7wfAP5LX790IfQGO2wgHe1KhIKhOG4d3ab7OIgCk4O49Zt+fDskAaS4sf9mPxFtC4DUkhUzLmzyROAbpPWGyHb+hrGdW8C2GRninLq3NlZhl1HlT372JVMqjSNKfy8fH4FkQebuHzejWjDggPzfqe8UyjuqG2Q96G\\\\\\\/ByhJREzkAooXzmV1mutl5hgPQPWmJ8ip6++Ayksw8uSJaiNu5jm2o3ot1TF9xexAYomQDbgCdqa5fh7kxRAUMiXrLEbQKJRIdtGY2BHS+6jCT+ueMTwSVdhkqg5XVx5g6J+xgHm+SSfOCNIj8vwhKVno+ogZtIV6WOlpYwdnqNwxkQzXgl0NQ2lMqfW3Z9k6LoC9J0VGVaynoFR55c+uk9RSiviih2uZ14kdwEgIIfr5\\\\\\\/uTMxMpJ9Dy8gVCvmllSAFkuaWIlcDNuqEgoheK9d\\\\\\\/a1AMkcBZbqeMmec\\\\\\\/+7O9DU2qsjhInkl5\\\\\\\/06zOzFOnariGi0RDfeJvk8lvz+i3eIaWOMYotAeHsGWc7YyQCcFDd+lEX0z\\\\\\\/7\\\\\\\/\\\\\\\/q5grxWera3nyf4KgHGxQKU6\\\\\\\/dGQ3GC4bL0XI6A4VywpZtxqSXNPF0Xex4SSXM4Wca90egwvCDiD7n+QCN8MoECHFknYm4ERWWcdiAktFzluz6w86+6BFIbJoTtkXm8lfEZqVSB9z69CN6y0\\\\\\\/9w+m9sREnfRkv8nfRNrLtLaoZ3ObUsDTgN9YO\\\\\\\/7DBSKGSh6se8Szn1\\\\\\\/\\\\\\\/pAHupsip582HORCtEDuWXMpe0513ad\\\\\\\/jwzcigGSB0yqhQHE2V6FuVBcXgtbujyXNNDmaTLIFlvCO7RkrdE+PI18oKnKEcmkuMwePnatqR2iH3t2w48Wx+DBUcUYRQyFa8nLGHu2x3svm9d9GI6f3fzGOrzNTK7paO3V5r81vudcG4pFDa0zEuqZOCxXkCvmBh3m0jPwx36BpDMz9zUVHIN4d4v3EerBk1FgYpCDrUuiVfxPflRL8tB+eZTWQgbUxRMgzygTMPyKO6U\\\\\\\/f9RtEeCE1VE1KkIreDA\\\\\\\/a9\\\\\\\/7MgZ+GL3fosyAjENt+wY3RSJ6phLsH0219zvp1OGmDVjWRvi3DRDyt4iG1H+rGr0OXr3WQ8UwuzNUv9ClArzmw8tQWQTMZYAc58alJs2p+jZcU1LGM86uQCJKkP\\\\\\\/\\\\\\\/HchP5o7eL3oaX4GLzP68K9IbJJDHrFlUWSKqLjmedMyMwx0KPUvv8heDkDEXkUy1QUAkesLiDGoODkttvLdLqTmdN93eNXo1+7voRIT6hy78dIYNor1yu3szG7kWGZkheqlAM53yIq9aIwwevvaNCuawCkbl5nq7nZxIeUS2r+qw7dbI6u2buW41PbiriBuac2r3e1gadPPPuOqyAjxhmht9u1VZf8nJ32TeeBg8tMqpndi94BMu3yqAe\\\\\\\/hUt+bXsmN39Izu7zIHlLq0u0xXjMLoPQEO0dTaE0MBvvgelpCeo2rKSTkcnVK+sSNsI6rAkysejI7nS4uHPwhS0ayew0TdDNdZq4xmF+okO9ZoMgWwiW3jCwmcKHSoKo61ilGWqLIXW4m6TzEkua7kYfGtDmn9Sb4pvgGVGkGzzXEo\\\\\\\/2bmK\\\\\\\/8UnIJUwbEsIAzjfu5QtsoWU01CO4kipPV8tp4XyC3B0OjRUsFVklRVOgvH9B8TPEFQfLVFB7sukS+1ra3NrO5juf\\\\\\\/J9AlxShxLI3S9hSN33loWuo21hWbx2qK6zRweGw7sOr7EN6OVYoaDOWmkTLM8Fvpl4qRBvI29sk9pkj4CYM6qxlZgH1wAQP8fOhu4\\\\\\\/6IgoGimCNS093MH3U14UgUqtEVH2ZShWo06AeIg\\\\\\\/AKyyMD0J+tUyCKXg\\\\\\\/BysDFzRl8g5n6FERRxU+ojYtBBYPfivlEoLbn63S3iL+z9LkmTCgE80+HSR4l28v0eHmd9l95Rh5arbDb5SmGj9T8HcRStbBcOtH2Uq+7WthPjB\\\\\\\/XbVTVHaFya0at+z9x2swJWmWdtJvQ+Hsy4Zsbp3MnHl89i4qfAVFTU8ymSbjNtRmCDFD0GcODGO5Araz+Hb+lO1ioCKyR6BUVCwfDwZO\\\\\\\/EpIR2pngM4VtF5yN\\\\\\\/IFsE\\\\\\\/qa5ocMMY2kHexX5yZbgc3j1Ytde4LhncjS0BId6Z\\\\\\\/T9f+DBnImYGoDvoC\\\\\\\/BJaxVBp\\\\\\\/sIg9pLeTjgXRFf7dTIDiMJK39toYglIoggnMvDMg8XCnAKoqX0RCgjpZMcd4TnIhqbkCj0rnb60PCC76KhHCpVPOl5jYy+GmA9EbefHG3R179i\\\\\\\/JLpw722a6kep0y9qxSM8vYwmbHv\\\\\\\/jmksqSvtnJw\\\\\\\/Q9Xf0NmvHWUkD+doG9im7kAC1mTDNctDKpSzn1mt3PqTufAypvhqItvJIdQ4BROACYFEykNF5XRASsLGZCoD0fbNZQKQ5+NqIluZ1bNY8kY0ODyLcjeAtPSdpF2To4Dp5uJDtxsoBAJk+FJZ9x8pU6+W+CTUlHQvGr2Igm+o9ceaEY\\\\\\\/wxJFEeBBsQjk7P9E43YH\\\\\\\/bH72GRL18sdZhZ6KFQl6fnjwde9ZFeFOiefuaI9EWC9veJRG+FDZ6utVKNMtwhS8LAxlCwLeq34vUUlEegzvg6NR1EewtSU\\\\\\\/nRR1FIidfCu\\\\\\\/5C4TPDvj4aSf\\\\\\\/fomNon0CdPh2erIZKEUnr\\\\\\\/eDGPxRGBWhcjqesNKyTADrRGMPygAlpyOZGtlu9ImtRkvJXTAXPiEzFqHxFpuFIgJAuS4vka+U2PChSdBrFDXzRBJL5Tg9BO0HyqDftYkjf2OHYD8D04WWZdjD4SElxdlgnlrLobBahQisvvLOrNCh4iV1zBhgLgId8\\\\\\\/0RYhvdXAwi2fhu1V303nXgAcV1Ek2ZdLY3hQAdu3GcN7zmwGbL0SduqDLmWuXvlC+qMIpIY9MSaprIKZML\\\\\\\/o63F3SSoMf6XaEqyARUdz1b1jhwUNE0vdiETFs\\\\\\\/7HFJq2Ru7CSsJRfRJ1ov+AiFX1UhcJg023g4tyCENcs8k6MYDALSkZxIzOQdoA\\\\\\\/tPXwbdWDCgcIFf6vqK9JxE4xEc93Hfusvu3TqjLuEZpWA6NOalQF0acG4JxVCuAFuCXduT2VRuftY5Gq2A2jEwprYoMWHSd\\\\\\\/NkS2vtvxPFNlLtBKWiVUEfxP6r5CtXW5+EVewRdryQpybBnwOLI6OnmfTdZZkKbWPE\\\\\\\/+lE9N16YrnbLHiQGQ7m4esSnWW3P2nMXWYKF\\\\\\\/rg7gRRkReEKNtk\\\\\\\/6jHgHy7tv0W3F62mKyLVnKN7N2+HxWxF0\\\\\\\/PrkdjvPi9ZBDKLn9axSYaLQYYTf27DezUMv7L6YSZ4\\\\\\\/3Nvh1nc79s8Fn3WF52RzP91XbkVPVjJQQNNqw90infJnIz1PawF9+bxoGrI0mzwlyz6POYcwO+qxU4jk\\\\\\\/G9bjyhDL\\\\\\\/ijVGycRHq1KDeDKOQeNYmPxMHmvJSvoq6oTbVx\\\\\\\/BOU51b1ZEAN2lbIHhWezVlhXqdMdBl9IYbobx1bfptV0057VIm\\\\\\\/W6myPlGcEnZ67uMhEV5TwtZJ+LpV7i\\\\\\\/howMTYqMfaTI2YheAim+u8Cuq0NsE4dtpjOvOCe84dSl7VeApnrvDo\\\\\\\/Z7UaSAajMiEgF0Vwdsu8TfkXPjQFoRJ1ekxPSL\\\\\\\/geKE4O1fLvGOzfJSFbVVhT13BfoGe9oTe7ZhcSjuPJZ3\\\\\\\/wWv77FxeMO0EJDzysIdFFGn7OtZRtNtC\\\\\\\/b7v5EAa\\\\\\\/EP8jT4PZFjL66MVSFpiCGlJCnRsaFZ\\\\\\\/YYlaTq459tjEEG+LW0qXO78THIaPTG0I7qO3ILucNouH9mNg9JvMoW5RhrYfyR2Aljh7pYL7Z6g3pJCFZ678DCskP7GpWGMU69SaZG2Wf8gPeOVDYZ30gLdxMM9C1CQ83TgvMmdA3pMck43jdeHy1RhJJda0t2Po+PqQr28m+nmLGXG3Ifi5arVtvqlnXUPC12l0Ju3DXdkllkd7iVg2yCS\\\\\\\/i56E4Qwd3jO8Rj0XKF5V2sLEKX1UbTZOtXq5Fl0Ei19xNFKAQZntkvXlmu8uJkp5XjOseznVXIraDbCRXPbF2BR9sQfSiBIJutFFlTF5qJxLIw\\\\\\\/FMER7PRbspxiVjJKQAwVQ9PLpAjvmw2sYBRqe9VNx68Q1T3QkHYEJzXUnJE5sgdGEtfSR9Ur\\\\\\\/P9ZLG7j\\\\\\\/nYbaUw1zz3v4l0wtHXtMHTxJBs4k2UnXiGqCpvMt0DKMqkY+dzeK9Pe7K3qOwbjqCh8hko3TNZjN4cNHoNPKFf6YYMQ7P\\\\\\\/ODrhdv7jS5s+YQJaSnc4\\\\\\\/r23NfST4FkmtRKc9Q1pAwmxUn5MhW2QEDsnFgbm24c+jneslzVElnKzWb4gPrHzPAjaTsq8gumvLY2FrDlUh3819FTOW33mwJ\\\\\\\/5\\\\\\\/lVvM5SIziVZQcnpSYrwFWobDCW6e\\\\\\\/+0+LUvJ\\\\\\\/7g05SpQemHx4IZWGpF3uYYolNPRpefQu4uNv9TKbq7JjsiHGkQQXayf0xyMhSiHH4fDNPt5TzsXL7sc6gyc+ly4GHsoTm8+7xsHASMwrTA9R629wQPHlAiEQ4sohle0XhnQntWLs5R+CkCr0pMBZUXMNrHhiu2GkwdvXw9nvjZ5HBffM83gnu6UZz9fu2yOrgz5VmeL3bmK+XYqEbHd66\\\\\\\/czAn2oV7hM\\\\\\\/yabFZc9TfZEguAfBRYhujLhqrOIHbRTcLOfplDj7Aog+hB4WKAgkBV0EIY9QHzt0boJSEQHDKgp5qg7jiHSWv4FIOQOvMZblZkZkZOWUNgm+wk9xpmp7+63d1hzNYKdW7ZrrdDx7TY9o\\\\\\\/sz7rz1X\\\\\\\/3O7r9aceGyyXpWf\\\\\\\/bUsdO4kFEfiw7fMekMFV5zp+\\\\\\\/j5zYMT0Ap9KRXYtR6vngKaAbxGvbzJNaiJ8loUZ1W6790jgyyQgsh8vHjKoMu+R1S2ahdNGGbOcyFwBdkeA6MQL5eO0833t2bIxQnpeb+U4zinmKfeJ\\\\\\\/cKz9UmXTPBUDIV3oH+WhXUn5UIgg7mtbEWzj9rhcTDdU\\\\\\\/2sN\\\\\\\/CiadJPQfGCden7MUohb95ZUqo6isaXRsmnmGLTa9tzClgPlYnQmlJbTp+86isjb3UKxgGeA6kj9em95r7IqT7ULUBBldGJdEVf3oJA0Pk4kSIgewz0hSYw+AP8fOSGdqEL25ZwxrTKbyAh3PxvVcAv0Wy40vjYLtRPQEp7V6REuiUfPJczaMkvrCjtNLiwJ9MMAH0Tw5GGhhrrf8NdsIbCJ0VQ8mKitrfcp4BOnUiJ6AsnW5t5DRZaPGWiAnf4rmjkz4lD918VzWgPQGY7N\\\\\\\/n+77GY7Ipe9x0+Gs0ipf9T19Z7GZdzwBHJ6h+zRDprqBgEIFkVn\\\\\\\/LcReJCxah4ljVUCetOglavGrqNQKaMQaB\\\\\\\/0R\\\\\\\/CtSaOkRN8pzhuA6Hs7U7M3S2txDLO2kRyNvBqPVW9O1exzXdL5cwkEatIF0SBqg\\\\\\\/srbd5ZEQcUDguQdJsoKECVZ6uTcZB+Qwkj5qecfH29Wgbg+ngjXUDllk04OWr0VOKOn9iRfaNe\\\\\\\/twLVhVPCQQzvXCoV18mxLXx57qK49MyP2Vah1nZ\\\\\\\/hzbaJpW8Uxwg8FuIiwK\\\\\\\/sUAz+4cdYpKwQV+Rbuva9ow2ajDNsfYUzaQVBjsFnPyyWUSccRri4qB4QeNx\\\\\\\/PNpzYiv8ccJs6O83iJj\\\\\\\/W6hnNv8xjUNAWMLbFurJs3gkZ+ypUA9ZV5Q2jXFpBmM4mUmZBql9cITXRm8Jw98ZgYSQqqGpY+cmdRXv0nfkFcsb9M\\\\\\\/UXrpnlkKmC1I3HT\\\\\\\/fKGYOdFB8sAFeKpoU7yZ49s8pkEItYlUdP65T9CsLjUZ7yEphRW992FZj77SC61ZhaGsmXICB\\\\\\\/tQ6roJ79O7jWOLkjC3tsOeGmiMPujmkkhJcN+z7CU3oDQYkPjTcpxlURGXr+M7DaUXeBIgM\\\\\\\/KStJbGuFCmxfZmyrkZvAsytENkX35K6XF3aJFc57zBicWfLn7CkKDL89k4pYJQ1D0wB7M8Gr4dr9dBU25TqIUEmwoRcqgeb3Iykw3aAG6jdDz2YpboiX14DyKSvWGJxe+RinImyaRLextUOVa5AbtJ74ufa37Wp9697GYi057qj5e7sOBqdpxf4xuKHU1HQGjaJ9Kj3SPC5K9oSzgsUwSnKxwElVCxHUjKWhVsDiahAR+rq9hvtQmOSnpcLk8ipSIXZsVo7FgVQzmKFF56tTdizld9u+utLbkSyPvcl8FoOx5uJgQ1SZZr746QkKkCf7vKk\\\\\\\/EMg3D0B6pk5sQoaSVQoOjTpO6oyn\\\\\\\/SKEowzRWQxlRkZsRD77268OaGcNgSOjKsFkMDKw65LvWOviXxAsA8qYrMJsKmoE\\\\\\\/\\\\\\\/Ov6TBe3qWHwTe6P7FbEhvD81HRhD\\\\\\\/pOpK74x62oIa57hQKHczlNEwrEXID1AyiZ\\\\\\\/NcJvsd+ItpfyHoW1izweVIdtZHdE\\\\\\\/2X6mQC1L5PX4nRCZseVZZsZ8HCgyYxgzRqGd0sSeimzvR2q07UHwPb7NaVtQnhvs3CYVXyd\\\\\\\/U0Zwc4WqYgJa5W2OB1L78fq4JpsuCRzR30\\\\\\\/vBExbb1s7w+jNF50MCwDign7qKcMy1rr0qcOeZ3LDwNVn3ed60+iFryz9gPLjvU3CZXa1j0hohiAffoB5yYOEWnQ0g04jeBKS1JpI1WvOZlqFVD6gn1rVm\\\\\\\/TTEVQMf6+KBoElJXyTBNxXXWVdIiRdz1oSMtukX3T3LGVc0cVj8FJt0kIaIlGfqVi\\\\\\\/c8LehhSus0peWbTz\\\\\\\/eZG\\\\\\\/IswCdWHEQl8oiLJLLDco4\\\\\\\/w3d2lFeElXT\\\\\\\/y7UBXvZq99vT4aUaT127ErqDtFq5ZXZiZIrRLphGpLhkC9Ej1nJF6E3miGbO+dSolqAnCv3bUZJavJvYvqhozEKq2d55e1JTRdpWFUqlVZuXGS+F98je4cMkOwvcpyxoBMQx4Be6j3iBdaUEbnkN0ie86Cb8V\\\\\\\/FfL7VG1l\\\\\\\/M9VNDyu1HBAkxXQEcBEAUFHo\\\\\\\/eKpMQr6vQ\\\\\\\/YqC21Zg9ou1D2mG41DNdS\\\\\\\/zjYFmkSlOujQkEJnOgQAHSp03xIDtgxrtV14v63dVBI1oSmwmj3IVc2lgl\\\\\\\/p7Pp5t5kiXP86YMQQB0np5tb2rcrPTMMXL+00Q\\\\\\\/bhFNIv+XFTWymS1pGKUMLjwZvCtaGltW4OLfE8\\\\\\\/gefMyRsO02Mm7X2Hp9OnoJtUQInbczTHZbHGxXwgCVYHlSc78ITUChQ0HR+doLlYc0yQ0Gv+yu+MIN90WYODOd2f\\\\\\\/o9EfDmETlYLeU20RoOQc1N0SO5QkKqdszxDfRWGSG12n21u3FDIj2dKa\\\\\\\/C4TYumSaMhSWdm+o3EdWECzizsf5DSAnEluwXNmdVh82lmENfBPXX6+PBnINTXyZV7d5XeJzQnLlJQ7vWfwK8bVLGJUfumJFOi7yjKTCwq1FnH5+O3vseBjp57n3cQMSNmJHkJCPZ\\\\\\\/UFX5irJOGIgW+vrEkJNMZl2Iqu7zbUUvOG1WKdhGm6yotzsQwg27qd+pNLofh\\\\\\\/8tvP+oGIxMoc0VOIsCFdwiL+6Iux\\\\\\\/uKDvmWyRsZ3RZezw4tVa+ElmC1qTOtt3s7Ehug9OYdk\\\\\\\/NrhtRrYiPorngNvOijCGOUJflIQyR9XIUcx9Er8JO75ShtM+TNxr+NSj\\\\\\\/Jk+d5FJyLBG2ggicvUCLvMs+c0tnwjmEbiN\\\\\\\/zJeXqwYZt7FGPzmi716zRSbdGceHyWxmxti89sVKY6Cci36sH1VaOWbJUlau5TG5TK4F\\\\\\\/L+m11NxXEc\\\\\\\/Y0o0Cv+jgohBNWRyW7f096tTpKaO4W5voxzyleGdbJNGxADLt2kLWmqKFvyH9BxsyHRuokDdw6l1x\\\\\\\/U9GkwRsG\\\\\\\/cT4yDSOOmjywHoaTeIJ9aPAmKM0Os2ZU4owl7Kt7hsi+qGtdkkYofB3NChTxpJbdhpMu+IoBBY1O0PmeECnWKwXsAx96EXXtkYPsIefP8hUWY+Lr5egidIWFBk9ykyKuY2z+LxvwLqL4MBju02pfN4MPn+qEqi8YJDyeAU6\\\\\\\/EmUrJmzqRaWkT3knbwwbboCqSIY0uKc2hZe+zmze58etTAEBcPQVSh612eZ0HO21\\\\\\\/RR8HtZhNjlUNAobXixAc9Jiz13aa+\\\\\\\/4HxdUAIXSRARr24CPRgWduknysRgup3d5OSovMjzEaKwpbJhH6\\\\\\\/w3NB5+zqdVC\\\\\\\/1xtGM+qxi4kDxtym5HGxCfEuswsKxDf0OC2kPGrCM167X5BOiZYMAh2hLoFB2JVIg55G\\\\\\\/S9F4LMf5vI8sqb446ALGxNjvne8v2Kq+i+JsmHj5JRa4hcVU9cRltCcw+gC+Sra3nyLH4qf4sQ2F6BwXNWzj+3qAlH\\\\\\\/hekUOKRwl4p+NYjauPqQcH2iwJvhiC\\\\\\\/GAGrCoPMlgDz1Z0xP7FhoYpPz8hJylSdCM4rxoKBEq9UQqUvFBTpUZol5U2TXr0skj5V+acgsT0R\\\\\\\/4KF7yNztZiBqqsxZkuaMNq2HES0vn5Mr\\\\\\\/obEVJuTR4KcHXWEaYbIiE2n1++SAhei+EfeLS9ELf5MuQG4noJlTYymh+PG4VJBC7+J5+JbhQDBGJHIAMFAmAyho\\\\\\\/z\\\\\\\/LwQlwFG0MBoLPdOUN1A9H2YxkdasPJz95c8hoGYEO2Kc9ihYhGkeyn\\\\\\\/\\\\\\\/YBB3aXk0F70UhcEHTeKcisd5YJtqUD2+aPAfX3e0aPXIdkQU9wZkdHnj0mowlJtJu+\\\\\\\/ctl\\\\\\\/KIV+wD4GetXTyjrhZGwRyUlKwTV7uLfHVWJFWDT5QnzdWK8aVZEkm8WzZiIUgIgp4wu1oiP8f3dQuF+QlbKZJ5AnAdnLCKnIllOQRjzy1GkUR1VzTQmh7r+TlvIzk+Jy7Gl\\\\\\\/qalTM0EsP2k\\\\\\\/0M9aZS+4tOr7F8vzSRibdwq5MKr1tf1VHiTXpeepr2qdg0kCMF5+Y+QPEP9YFSFl7x9I2wu3qhbWodjhvTHk97sZtHUedDKn9lWRi4GiqIbrQvbSTdRzp9xtmgQNGWXNMpY\\\\\\\/kPF3wXbzJ+6VIE5t9C2qsGqaoeFbuzSbSsnptAra\\\\\\\/uCX0P+\\\\\\\/M6Y7WPM7oxS46FIDtuhO\\\\\\\/RvgkTnGFt0fPD6ELYsHQvoHIxORAqnalBErr0E9\\\\\\\/nDRBGGv4e22MUeTjG0GTcKUOrPARx8iy0bEWkf9PrGa6e81TzjnsNWmn0xEJo6dVapfXNUCUIuGt6Ddv0ytc9Gx+9n2d6iMGwjhYBoAT+sktchdgItJREcMEBK1AdBtJOa2yYXjNxyyWdseLOyH0gdPIF4vRN89aDaz26lSz6IsxtUJ0rCLf+BJK1bF1CQn14zDRyHwXXVBIg3f\\\\\\\/8IIzqM1drCweu4Ewdrchkz\\\\\\\/LWLhTT\\\\\\\/h8\\\\\\\/gtJo8fpgzvdxMKZFA21sNxhg+781XigAzOTXp5O6sTDg9uUO0JOAEPjHOrN36PIiqCI4MjuONNPnSEmkgAefVS8GPU2u9LHH4ty5FX93d04uyuy7CDOrwiSY77WUliM4\\\\\\\/b7WPut8tsf3vsnY29\\\\\\\/n\\\\\\\/AyOoit5SHVa+99VZIhBeQXfGB2Cu5gN+PBm7R8ipaxVqArlA0gN2DNBJPeXRm\\\\\\\/E9Mc5DnDumkMlXIqMmpDwdkisnU+JgsK1j+IA2f6SrZhLMKFjYp8MfBbSadhkooSM8QexWyENWJx\\\\\\\/Foun5PJL+ueYAWClQ7CVOdZeaz565Gw2fVkB6rSlJJiQC\\\\\\\/1EyfB+bHDFkS5OVXx4\\\\\\\/jgHTaV5eRQkPkYzd7G389zA5ivcsTJJOqUh3cWz8Vv+7dZ7HRo8tS\\\\\\\/hygd36DQcR+HV3gIj6OVcp0paGXayhn\\\\\\\/S55WCevGxhGeBVQp\\\\\\\/sq1ZDx\\\\\\\/THTMCmvGcmpcFjhxMtjMPLVy87sfkUqnO8jilR6mE80xteadnBGz51NYvgE6qDEdEm2+Jq5Wg9TFl9+rT2W6fWv5+NoXln53DmrfE1rocjwgsgCd5PKYNXrdO8\\\\\\\/ZbE7Ex8lsavQhYxh4r\\\\\\\/zpuqRRyFe+IigWmUoE0dO\\\\\\\/iJ7\\\\\\\/wuKNX+rM2DdMH1U+5ou8tAznPxSugp02JdAF+MUzjzIAVZSUS11fz7+E1SlI85B7Wgfkq6sUFnUFLYGno4ZqXdv5Ztw6FCjpFa4T3+OAjfcP2MW0RWZw0usH\\\\\\\/yVVy+VsHxhqqTU6uluJgEICE0s1CWWpFWJTnOSWzvkeZ3xMEo8tfxgBbtkhobPd7k7C9+3jpUqiAYfcKYHphXXBUY37F3jb8fuoAG8FvesJ2XEgbfT84H51U29eiv5redLTR6\\\\\\\/y9DKYtI\\\\\\\/bwFhEpNIN62QFEJHG\\\\\\\/sZGajAhYOVKMrvN2SRfoT1gHWGkw\\\\\\\/S\\\\\\\/SpdYsJdB1pSzz32C1Zdodb4K2BJuPoEzRGq0j2mGAvxCqjFvSWkDRY32CNQrKi4Sim8sqJ+TJBsWiY\\\\\\\/eozjEoIZgJ2gcU8BUEdWjmAaxFsiPmFlcSqKHcS6gHR52nE9OG7D2KTOCPJVUt9wORkHJg0Z4V99gTY9ywbq5B3kfYpOQAQnsgezhlPwCYT5lVnN9LGPgCW61w0OQvCtLrzzQ8+8d7F0Ht84Ur\\\\\\\/Gr\\\\\\\/ZgKEQEWP74KP+drPmXcIpHBdWnxZ1yIDPLi4MY1RH1RCTQj2aagf0EmCbuL3bLNuMSRs8yO+7UgO2ydLqiBtOSKrzDRF2ANrNU2DGMWZGp3hJayIjsE24ZksRF1LRth4\\\\\\\/xLJXxPGbGXZNgIwhSv9Bfaoi5LYYsHXfmdx7ePrx501mRSrmsUiXZlixqRp+bwuoA8lePZiCGYsa+vA85c+TR\\\\\\\/cgG5Aeh6ycjjmhjZEMpK7frh3SVXS58qnN0YihQVLjnRGjpe6sxpzbmz\\\\\\\/Yz1OuSXHzKfijFzhcHELXfVDD8qyuaVWnN\\\\\\\/B\\\\\\\/DZB9\\\\\\\/qx3GVEq3PSVntzieui81atOWjFww+PIsEppyawI2E4TL5bpI2DzDWZD6zOS+Eaec2V3elLrJN3I2gVfDitJMIcH3+1\\\\\\\/MsbaZR3cpRjchZWfaBVwuLL03g7cXWTfl0FPNSg5XCK5DuVrWWCtFPxWzN6MMcWn7vts5VSGU1U4YeHRBz2I6hUjtzu9zmqG18Q+seQZMyxbg7QYFuPOyEbnWS7OtebVEWD3nD\\\\\\\/C6t0\\\\\\\/n1VI23EHES2iP\\\\\\\/gcFsATviLC+2rS5WR6vfX5ZzxI4Za++BgNcjtgaALnCXm5CAjwPBIQyyUvZiq\\\\\\\/yrshBXnHzB\\\\\\\/R\\\\\\\/FZ248Xqg+iIdcWRGfwfGKPX\\\\\\\/25l5cMPDcVtUiCloW6QrKQu\\\\\\\/YyOEurIeiVatodced6P9ae4qxB71LBkI+PmWffA4zCR7zUlGlxL6TkVIz7Q4Tmw35JsL3cDBv36NCX1UuB+bKjaoEmutSeX7aLawfUrjThegj2f80uOLmek1kqqffievHraPkfA2+OJEBBWIx2+VOXV+ETG5YssRRUHGtG3PtZ9UF2X\\\\\\\/oYWSiSkv6U07VURpVccL7\\\\\\\/Q3RNex5NnGce4P4t2Ut75huU2m8bXg0lYHxdzRLEAFIIYk181zlJ7yIPpfh7ay5mBnvFoZCoCr+DKLIZbwOg8o+79qmoVXR9\\\\\\\/SHSTI0nYu3RHpf\\\\\\\/DeFGgdvBZIw9D0fNQ76CJF0qdy+OR2fOahzcWNEfNW1E0nkIS4ocMX8s2V297sCdsTYNrlhhV\\\\\\\/8P3TtZaij05cZuiwZ9QWk7+W14xOKgxsPNcKl3PZqEiLUFlmAPrd06XcWeUlFNZAZqjCcXl0zd0YRXeSpET7SScpHg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022K4EGLtFJaNSAV5lLeHzyJw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229f4f17a3fcf5818b612a4f03814c6197cf2b1b7efaf4fc85b9f739f3d21ca4f0\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022bMi81k2F5rXjxh8YHPfOy69MvNlP9T4r0U7fBguDL8KkDUSd+lDHkOIRet4RYPK1HwI9tLpH7+zqSxE9W02EK9Z0fDtJMekJaYhuFViSAr5++SH9WamIr6oC4t82hbdu6SRKmOR5sIIPIfEqsQxQoYrkj1dyyZg2ijkynOH2o6RAolZ\\\\\\\/MckE\\\\\\\/WKEQTGcpkIFbznUL\\\\\\\/SfIaWbGbhXGaC5lwL\\\\\\\/tfMmRCr88z81oPHTYQwrL0oqUZgkUWHxgy38umDgs7muNhmIUE3sr5CkVWkwqjuaXjYK\\\\\\\/QT02U7tbMj5DwRN5yGUBx3qUCKApvasEMUoXPE7P0DnwirLOf13Y84GSL+N8YesJGrRy70vaTZgvhxGU5DQ3NBDZ7Ajjj9C0jKGs18GIBP8Yc23saFxOjPXYG68wzvItSs07jT1hL9Aw\\\\\\\/CjjSQG5zv8dPepdDuPwBhtnOyEiTeg1tugzAWJAxxcJY6\\\\\\\/MQeOyt3Jg4lKWOjZWylV2Uy7HPVgsqrK\\\\\\\/h4tr5DTaLQEpsICAe3c3BDaa+3GUu3L6uQXElXpYss0gYOxUexCzKCVxTloJpC1xnJy1nQAEvTt36FbW2L8txt\\\\\\\/JEHMuwzhzsDaLhfvS4\\\\\\\/6uq\\\\\\\/wCeG2j5ApFi0mwmhGRKOBWVLAZHLnm8KOhDtG0b\\\\\\\/GZ2OjDVE1FEKhZB15X\\\\\\\/ETgftvb2BGzONbkMGc3jGxv4WROWM3jhPHg7cO2m\\\\\\\/sy46cvRPWWMDlqg3LzFT2x6\\\\\\\/pgJf7vh\\\\\\\/MG\\\\\\\/l2fcXcHJnIv3x7vS3S07LdFdpdu8475\\\\\\\/Rm981q9wh6V4UvmItw3\\\\\\\/Yt1juBWSHTssiiBqn5Ibr8+dQ3uIyvldu1+gZVQP+AU4vMTaQC0FGKUa6NoAMqB9zGZeYElYNs+3g3tT+Vas0Q2R7d4qSOxf7lOIy2yv2M\\\\\\\/RODUPwtJafgaFuzkKcfJ55DyvZc8owlp7XqVth+S\\\\\\\/gWcbLuzkTIJ\\\\\\\/ov4\\\\\\\/xUI0U1tRXtlGPHnqgNtDfD6QsiYzKNfwrr4eq33jwdNqABEqS3L3jZoI\\\\\\\/4Fi0zv\\\\\\\/aDntYS5nl4QJAz6ISSCJ\\\\\\\/+cHlhRQsgQXnOZD8kTn3LXdm95F1loBKkMIAeg1fRVXRjElh6q5gejUbszBaaJrt6GpoGvrKzSwgx8niW6yXyFn3OVqzNHXfrOG3HnDXS+dQthJRtcwkhwTeRLFq4hf8e4GsNYpXqEAawT9FCk+d4oI\\\\\\\/JRTnQEAB8BH9ZhnAZqADntwchEGDGy6JknCoL9nRUxmgY7KgcVZa3bhttX3PwXfCezaxunhzs9RAQf8QJM6ERY+VT6h0ctM5xTkfRJ6YvimaoavH60MvxZGwXo35HpANXTYxJmThLAjHIHAiK0M++HnfU0J0y225RZXDYTVSasz07kqbGmhAbwANqFhNxIpHjgUzmD6scpOh1D\\\\\\\/D7Zr2HAUm+de3tytklOtu9unaekkYVP1zfd0JldgxDjv0wS0W4vzpFeYRg2jcpTIwPQU4zhLANuGrmJz9YtWHFr8UMUU6K\\\\\\\/19X6jvJTtmVOPeLPU\\\\\\\/lAl4X6kDhTowXZW14ZoG7Hg6tjquHE\\\\\\\/aGxBLP9Tw62mYC7g0wZIH2m+\\\\\\\/27LYU6WskWwvNoTsLzO3+v5WybOvrBdInMFNDyjN5nWFqOFJ1wpIUujJgbzjyY6jYaXp2nBpOfpuHVMkrGxoqxnMkw4DAL1aEWQFQYNrAm8WJXi3vLLrzgLg9bOWrHtjfjemXVQlZAYb8xsWO5Y1Bf+shgslQeJMch83rF6WYlmjbtyn84GpKEY6TDsLX3+yx07TLxtQBv2RBN4EhJSPWqrcryyImm688H0exTiEBmJoE4XBiGqN5HlusgFHL57ut5PojMYQRaVkKxn5YXJkAEooVCUTJAEgMpM20f8djdf2fGy9CCFvzqubfaEv7igF+4N9gmfmuBhoPu10S5p2i0WO+QDskVm2\\\\\\\/LmHxlKQxHBabaAJNVcEE7ygLC8dLeyOVv8Nr3M8TPPgExLnY80LFi4+gLM4mqDhnvh3Cmhisiw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022j8IYq1EzM\\\\\\\/U26WJplSm88w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227b74810685f6d8239bc619815bec687a4603b2491e1501e76e843744570b00aa\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460584 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/2/591" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-_yuViaPM3uYVoznyuLna_VupWxLebDLcpK7NSegvbXQ" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/2/591" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-k1htYNHRGIQZNgNG9x7r_ZQLkjienxQx59EhtZ8cBik" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/2/583" title="Single Locus Maintains Large Variation of Sex Reversal in Half-Smooth Tongue Sole (Cynoglossus semilaevis)" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/2/607" title="The Bright Fluorescent Protein mNeonGreen Facilitates Protein Expression Analysis In Vivo" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460584" id="node460584" data-pisa="ggg;7/2/591" data-pisa-master="ggg;g3.116.035931" data-apath="/ggg/7/2/591.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Paralog-Specific Functions of <em>RPL7A</em> and <em>RPL7B</em> Mediated by Ribosomal Protein or snoRNA Dosage in <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Ryan J. Palumbo</span>, <span class="highwire-citation-author" data-delta="1">Gabriele Fuchs</span>, <span class="highwire-citation-author" data-delta="2">Sheila Lutz</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-5361-3909" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>M. Joan Curcio</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">591-606;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.035931 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460584" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ryan J. Palumbo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory of Molecular Genetics, Wadsworth Center, New York State Department of Health, Albany, New York 12201</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biomedical Sciences, School of Public Health, University at Albany, New York 12201</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ryan%2BJ.%2BPalumbo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Palumbo%20RJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARyan%2BJ.%2BPalumbo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gabriele Fuchs </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biological Sciences, The RNA Institute, University at Albany-State University of New York, New York 12222</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gabriele%2BFuchs%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fuchs%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGabriele%2BFuchs%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Sheila Lutz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory of Molecular Genetics, Wadsworth Center, New York State Department of Health, Albany, New York 12201</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sheila%2BLutz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lutz%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASheila%2BLutz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">M. Joan Curcio </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory of Molecular Genetics, Wadsworth Center, New York State Department of Health, Albany, New York 12201</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biomedical Sciences, School of Public Health, University at Albany, New York 12201</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M.%2BJoan%2BCurcio%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Curcio%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AM.%2BJoan%2BCurcio%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5361-3909" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for M. Joan Curcio</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:joan.curcio@health.ny.gov" class="" data-icon-position="" data-hide-link-title="0">joan.curcio@health.ny.gov</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/2/591" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460584" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460584/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/2/591.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460584" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460584/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/2/591.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460584" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460584/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/2/591.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460584" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460584/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/2/591.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460584" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460584/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">Most ribosomal proteins in <em>Saccharomyces cerevisiae</em> are encoded by two paralogs that additively produce the optimal protein level for cell growth. Nonetheless, deleting one paralog of most ribosomal protein gene pairs results in a variety of phenotypes not observed when the other paralog is deleted. To determine whether paralog-specific phenotypes associated with deleting <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> or <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> stem from distinct functions or different levels of the encoded isoforms, the coding region and introns of one paralog, including an intron-embedded snoRNA (small nucleolar RNA) gene, were exchanged with that of the other paralog. Among mutants harboring a single native or chimeric <em><a href="http://www.yeastgenome.org/locus/S000004947/overview">RPL7</a></em> allele, expression from the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> locus exceeded that from the <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> locus, and more <a href="http://www.yeastgenome.org/locus/S000003044/overview">Rpl7a</a> was expressed from either locus than <a href="http://www.yeastgenome.org/locus/S000006119/overview">Rpl7b</a>. Phenotypic differences in tunicamycin sensitivity, <em><a href="http://www.yeastgenome.org/locus/S000001668/overview">ASH1</a></em> mRNA localization, and mobility of the Ty1 retrotransposon were strongly correlated with <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a> and ribosome levels, but not with the <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a> or snoRNA isoform expressed. Although Ty1 RNA is cotranslationally localized, depletion of <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a> minimally affected synthesis of Ty1 Gag protein, but strongly influenced Ty1 RNA localization. Unlike the other processes studied, Ty1 cDNA accumulation was influenced by both the level and isoform of <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a> or snoRNA expressed. These cellular processes had different minimal threshold values for <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a> and ribosome levels, but all were functional when isoforms of either paralog were expressed from the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> locus or both <em><a href="http://www.yeastgenome.org/locus/S000004947/overview">RPL7</a></em> loci. This study illustrates the broad range of phenotypes that can result from depleting ribosomes to different levels.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/ribosomal-protein" class="hw-term hw-article-keyword hw-article-keyword-ribosomal-protein" rel="nofollow">ribosomal protein</a></li><li class="kwd"><a href="/keyword/paralog" class="hw-term hw-article-keyword hw-article-keyword-paralog" rel="nofollow">paralog</a></li><li class="kwd"><a href="/keyword/isoform" class="hw-term hw-article-keyword hw-article-keyword-isoform" rel="nofollow">isoform</a></li><li class="kwd"><a href="/keyword/retrotransposon" class="hw-term hw-article-keyword hw-article-keyword-retrotransposon" rel="nofollow">retrotransposon</a></li><li class="kwd"><a href="/keyword/rna-localization" class="hw-term hw-article-keyword hw-article-keyword-rna-localization" rel="nofollow">RNA localization</a></li></ul><p id="p-5">The <em>Saccharomyces cerevisiae</em> ribosome comprises four ribosomal RNAs and 79 ribosomal proteins (RPs) synthesized in nearly equimolar amounts. Notably, 59 RPs are encoded by paralogous gene pairs that arose from a whole genome duplication and were selectively retained (<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Wolfe and Shields 1997</a>). Paralogs may persist in the genome because they encode redundant functions that balance gene dosage, become specialized for different environmental conditions, or acquire distinct functions by subfunctionalization or neofunctionalization (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Innan and Kondrashov 2010</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Parenteau <em>et al.</em> 2015</a>). Most RP paralogs are redundant for a ribosome-associated function that is essential for optimal cell growth or viability (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Dean <em>et al.</em> 2008</a>; <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Steffen <em>et al.</em> 2012</a>; <a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Woolford and Baserga 2013</a>). The mRNAs transcribed from these genes, although heterogeneous in sequence, especially in untranslated regions (UTRs) (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Leer <em>et al.</em> 1985</a>), typically encode proteins of identical, or nearly identical, length and amino acid sequence. Fitness defects resulting from deletion of one paralog can be suppressed by ectopic expression of the coding region of the other paralog from an active promoter (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abovich and Rosbash 1984</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Rotenberg <em>et al.</em> 1988</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Simoff <em>et al.</em> 2009</a>). Nonetheless, deletion of one paralog or the other of an RP gene pair often results in distinct transcriptional and phenotypic profiles (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Komili <em>et al.</em> 2007</a>). A single paralog of discrete subsets of RP genes is required for bud-site selection (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Ni and Snyder 2001</a>), translational repression, and bud tip localization of the <em><a href="http://www.yeastgenome.org/locus/S000001668/overview">ASH1</a></em> mRNA (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Komili <em>et al.</em> 2007</a>), actin organization (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Haarer <em>et al.</em> 2007</a>), vacuolar carboxypeptidase Y secretion (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Bonangelino <em>et al.</em> 2002</a>), propagation of the M<sub>1</sub> satellite dsRNA of the L-A virus (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Ohtake and Wickner 1995</a>), protection from killer toxin (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Page <em>et al.</em> 2003</a>), telomere length control (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Askree <em>et al.</em> 2004</a>), replicative life span (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Steffen <em>et al.</em> 2008</a>, <a id="xref-ref-74-2" class="xref-bibr" href="#ref-74">2012</a>), and mobility of the Ty1 retrotransposon (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Dakshinamurthy <em>et al.</em> 2010</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Risler <em>et al.</em> 2012</a>). The divergent phenotypes associated with deleting one or the other paralog are not always correlated with the relative abundance of mRNA or RP produced from individual paralogs in a wild-type strain (<a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Komili <em>et al.</em> 2007</a>; <a id="xref-ref-73-2" class="xref-bibr" href="#ref-73">Steffen <em>et al.</em> 2008</a>, <a id="xref-ref-74-3" class="xref-bibr" href="#ref-74">2012</a>). For example, it was argued that <em><a href="http://www.yeastgenome.org/locus/S000004488/overview">RPS18B</a></em> is specifically required for <em><a href="http://www.yeastgenome.org/locus/S000001668/overview">ASH1</a></em> mRNA localization because deletion of <em><a href="http://www.yeastgenome.org/locus/S000004488/overview">RPS18B</a></em> but not <em><a href="http://www.yeastgenome.org/locus/S000002858/overview">RPS18A</a></em> abolished localization despite equivalent levels of epitope-tagged <a href="http://www.yeastgenome.org/locus/S000002858/overview">Rps18a</a> and <a href="http://www.yeastgenome.org/locus/S000004488/overview">Rps18b</a> in a wild-type strain (<a id="xref-ref-38-4" class="xref-bibr" href="#ref-38">Komili <em>et al.</em> 2007</a>). Collectively, these observations led to the “ribosome code” hypothesis, which posits that the divergent RP isoforms are incorporated in different combinations into heterogeneous ribosomes that selectively translate specific subsets of mRNAs (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Mauro and Edelman 2002</a>, <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">2007</a>; <a id="xref-ref-38-5" class="xref-bibr" href="#ref-38">Komili <em>et al.</em> 2007</a>). The specialized functions of heterogeneous ribosomes may be further diversified by selective post-translational modification of RPs and ribosomal RNAs (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Mauro and Edelman 2002</a>, <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">2007</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Gilbert 2011</a>; <a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Xue and Barna 2012</a>).</p><p id="p-6">Discordance between the phenotypic consequences of deleting each paralog, and the relative level of mRNA or RP produced from each paralog in a wild-type strain could result from compensatory changes in the RP level produced from one paralog when the other is deleted. In one study, the longer replicative life span of an <em><a href="http://www.yeastgenome.org/locus/S000005839/overview">rpl20b</a>∆</em> mutant was correlated with a lower level of 60S ribosomal subunits compared to an <em><a href="http://www.yeastgenome.org/locus/S000004855/overview">rpl20a</a>∆</em> mutant, even though <em><a href="http://www.yeastgenome.org/locus/S000004855/overview">RPL20A</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000005839/overview">RPL20B</a></em> mRNA levels are equivalent in a wild-type strain. Thus, a disparity in Rpl20 levels in <em><a href="http://www.yeastgenome.org/locus/S000004855/overview">rpl20a</a>∆</em> <em>vs.</em> <em><a href="http://www.yeastgenome.org/locus/S000005839/overview">rpl20b</a>∆</em> mutants could explain the paralog-specific role of <em><a href="http://www.yeastgenome.org/locus/S000005839/overview">RPL20B</a></em> in replicative life span (<a id="xref-ref-73-3" class="xref-bibr" href="#ref-73">Steffen <em>et al.</em> 2008</a>). Given that RP paralog expression levels are often altered by deletion of the other paralog (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Eng and Warner 1991</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Presutti <em>et al.</em> 1991</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Paulovich <em>et al.</em> 1993</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Li <em>et al.</em><span class="sc"> 1995</span></a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Fewell and Woolford 1999</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Badis <em>et al.</em> 2004</a>), the question of whether paralog-specific phenotypes result from RP isoforms with disparate functions, differences in the level or activity of the RP produced from each paralog, or a combination of these mechanisms has not yet been rigorously tested.</p><p id="p-7">To address this question, we have examined phenotypes associated with expressing <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> or <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> at different levels. <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a> isoforms participate in the earliest steps of 60S precursor rRNA processing (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Jakovljevic <em>et al.</em> 2012</a>), and bind to 25S and 5S rRNAs in the mature 60S ribosomal subunit (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Ben-Shem <em>et al.</em> 2011</a>). Five of the 244 amino acid residues in <a href="http://www.yeastgenome.org/locus/S000003044/overview">Rpl7a</a> and <a href="http://www.yeastgenome.org/locus/S000006119/overview">Rpl7b</a> are divergent, with four amino acid substitutions in <a href="http://www.yeastgenome.org/locus/S000006119/overview">Rpl7b</a> relative to <a href="http://www.yeastgenome.org/locus/S000003044/overview">Rpl7a</a> (A2S, A3T, S16T, and V26I) being in the conserved N-terminal domain that is predicted to be on the surface of the ribosome (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Lin 1991</a>; <a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Tsai <em>et al.</em> 2012</a>). The possibility that <a href="http://www.yeastgenome.org/locus/S000003044/overview">Rpl7a</a> and <a href="http://www.yeastgenome.org/locus/S000006119/overview">Rpl7b</a> have unique functions as components of heterogeneous ribosomes was suggested by the involvement of different ribosome biogenesis factors in assembly of ribosomes containing <a href="http://www.yeastgenome.org/locus/S000003044/overview">Rpl7a</a> <em>vs.</em> <a href="http://www.yeastgenome.org/locus/S000006119/overview">Rpl7b</a> (<a id="xref-ref-38-6" class="xref-bibr" href="#ref-38">Komili <em>et al.</em> 2007</a>). In addition, the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> genes are unique among RP paralogs in that each contains a paralogous C/D box snoRNA gene, <em><a href="http://www.yeastgenome.org/locus/S000007302/overview">snR39</a></em> or <em><a href="http://www.yeastgenome.org/locus/S000006450/overview">snR59</a></em>, that is encoded in the second intron (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>), and likely expressed from the excised intron (<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Vincenti <em>et al.</em> 2007</a>). While divergent in sequence, <a href="http://www.yeastgenome.org/locus/S000007302/overview">snR39</a> and <a href="http://www.yeastgenome.org/locus/S000006450/overview">snR59</a> function redundantly as guide RNAs for 2′-O-methylation of residue A807 in the large subunit rRNA (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Piekna-Przybylska <em>et al.</em> 2007</a>). Deletion of one or both of the introns harboring these snoRNA genes has no effect on cell viability (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Parenteau <em>et al.</em> 2011</a>). Several paralog-specific functions have been assigned to <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em>; however, they are asymmetrically expressed in a wild-type strain (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Ghaemmaghami <em>et al.</em> 2003</a>; <a id="xref-ref-38-7" class="xref-bibr" href="#ref-38">Komili <em>et al.</em> 2007</a>), and the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆</em> mutant has a diminished level of 60S ribosomal subunits, whereas <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">rpl7b</a>∆</em> has a wild-type polysome profile (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Jakovljevic <em>et al.</em> 2012</a>). Thus, it remains to be determined whether <a href="http://www.yeastgenome.org/locus/S000003044/overview">Rpl7a</a> or <a href="http://www.yeastgenome.org/locus/S000006119/overview">Rpl7b</a> has unique functions irrespective of its expression level.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/591/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Schematic of the RPL7A and RPL7B loci used in this study. (A) RPL7A and RPL7B coding regions are interrupted by two introns. Encoded in the second intron is a snoRNA paralog, snR39 or snR59. (B) Construction of chimeragenic rpl7a&#x2206;::RPL7B rpl7b&#x2206; (left column), and rpl7a&#x2206; RPL7B&#x2206;::RPL7A (right column) mutants from the wild-type strain in three sequential steps. First, the RPL7A or RPL7B CRI was replaced with the GAL1-I-SceI-hygB-KlURA3 delitto perfetto cassette. Second, replacement of the delitto perfetto cassette with the RPL7B or RPL7A CRI, respectively, generated homogenic strains harboring the same CRI at both RPL7 loci. Third, the CRI of the RPL7B or RPL7A locus in homogenic strains was replaced with the kanMX4 marker to generate chimeragenic strains. Turquoise represents sequences derived from the RPL7A gene, and green represents sequences derived from the RPL7B gene. Solid, colored lines: noncoding locus DNA flanking the CRI; dashed, colored lines: introns; solid, dark-colored boxes: exons; solid, light-colored boxes: intron-encoded snoRNA genes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1111785762" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Schematic of the &lt;em&gt;RPL7A&lt;/em&gt; and &lt;em&gt;RPL7B&lt;/em&gt; loci used in this study. (A) &lt;em&gt;RPL7A&lt;/em&gt; and &lt;em&gt;RPL7B&lt;/em&gt; coding regions are interrupted by two introns. Encoded in the second intron is a snoRNA paralog, &lt;em&gt;snR39&lt;/em&gt; or &lt;em&gt;snR59&lt;/em&gt;. (B) Construction of chimeragenic &lt;em&gt;rpl7a&#x2206;&lt;/em&gt;::&lt;em&gt;RPL7B rpl7b&#x2206;&lt;/em&gt; (left column), and &lt;em&gt;rpl7a&#x2206; RPL7B&#x2206;&lt;/em&gt;::&lt;em&gt;RPL7A&lt;/em&gt; (right column) mutants from the wild-type strain in three sequential steps. First, the &lt;em&gt;RPL7A&lt;/em&gt; or &lt;em&gt;RPL7B&lt;/em&gt; CRI was replaced with the &lt;em&gt;GAL1-I-SceI-hygB-KlURA3 delitto perfetto&lt;/em&gt; cassette. Second, replacement of the &lt;em&gt;delitto perfetto&lt;/em&gt; cassette with the &lt;em&gt;RPL7B&lt;/em&gt; or &lt;em&gt;RPL7A&lt;/em&gt; CRI, respectively, generated homogenic strains harboring the same CRI at both &lt;em&gt;RPL7&lt;/em&gt; loci. Third, the CRI of the &lt;em&gt;RPL7B&lt;/em&gt; or &lt;em&gt;RPL7A&lt;/em&gt; locus in homogenic strains was replaced with the &lt;em&gt;kanMX4&lt;/em&gt; marker to generate chimeragenic strains. Turquoise represents sequences derived from the &lt;em&gt;RPL7A&lt;/em&gt; gene, and green represents sequences derived from the &lt;em&gt;RPL7B&lt;/em&gt; gene. Solid, colored lines: noncoding locus DNA flanking the CRI; dashed, colored lines: introns; solid, dark-colored boxes: exons; solid, light-colored boxes: intron-encoded snoRNA genes.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/591/F1.medium.gif" width="440" height="255"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/2/591/F1.medium.gif" width="440" height="255"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/591/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/591/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470227" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-8" class="first-child">Schematic of the <em>RPL7A</em> and <em>RPL7B</em> loci used in this study. (A) <em>RPL7A</em> and <em>RPL7B</em> coding regions are interrupted by two introns. Encoded in the second intron is a snoRNA paralog, <em>snR39</em> or <em>snR59</em>. (B) Construction of chimeragenic <em>rpl7a∆</em>::<em>RPL7B rpl7b∆</em> (left column), and <em>rpl7a∆ RPL7B∆</em>::<em>RPL7A</em> (right column) mutants from the wild-type strain in three sequential steps. First, the <em>RPL7A</em> or <em>RPL7B</em> CRI was replaced with the <em>GAL1-I-SceI-hygB-KlURA3 delitto perfetto</em> cassette. Second, replacement of the <em>delitto perfetto</em> cassette with the <em>RPL7B</em> or <em>RPL7A</em> CRI, respectively, generated homogenic strains harboring the same CRI at both <em>RPL7</em> loci. Third, the CRI of the <em>RPL7B</em> or <em>RPL7A</em> locus in homogenic strains was replaced with the <em>kanMX4</em> marker to generate chimeragenic strains. Turquoise represents sequences derived from the <em>RPL7A</em> gene, and green represents sequences derived from the <em>RPL7B</em> gene. Solid, colored lines: noncoding locus DNA flanking the CRI; dashed, colored lines: introns; solid, dark-colored boxes: exons; solid, light-colored boxes: intron-encoded snoRNA genes.</p><div class="sb-div caption-clear"></div></div></div><p id="p-9">Because selective pressure to suppress retrotransposon activity in the genome can drive neofunctionalization of gene paralogs involved in retrotransposon control (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Münk <em>et al.</em> 2012</a>), we examined the role of <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> in regulating Ty1, a long terminal repeat (LTR)-retrotransposon present in ∼30 copies in haploid <em>S. cerevisiae</em> strains. Ty1 elements contain two overlapping ORFs: <em>GAG</em>, which encodes the capsid protein, and <em>POL</em>, which is expressed by translational frameshifting, and encodes enzymatic mobility proteins. Retromobility involves localized translation of Ty1 RNA in cytoplasmic foci known as retrosomes, assembly of Ty1 RNA and proteins into virus-like particles (VLPs), reverse transcription of Ty1 RNA within VLPs, transport of cDNA to the nucleus, and introduction of the cDNA into the host genome by nonhomologous integration or homologous recombination [reviewed in <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Curcio <em>et al.</em> (2015)</a>]. Deletion of <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> results in lower levels of Ty1 retrosomes, cDNA, and retromobility (<a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Risler <em>et al.</em> 2012</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Doh <em>et al.</em> 2014</a>).</p><p id="p-10">In this study, the coding region and introns (CRI) of <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> were exchanged to express <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a> and snoRNA isoforms at different levels. Expression of either <em><a href="http://www.yeastgenome.org/locus/S000004947/overview">RPL7</a></em> CRI from the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> locus resulted in faster cell growth and higher levels of <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a> and 60S ribosomal subunits than expression from the <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> locus. Moreover, the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> CRI is expressed more highly from either locus than the <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> CRI. Divergent growth rates, <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a> levels and 60S/40S ribosomal subunit ratios in strains expressing a single native or chimeric allele of <em><a href="http://www.yeastgenome.org/locus/S000004947/overview">RPL7</a></em> correlated strongly with differences in sensitivity to tunicamycin, efficiency of <em><a href="http://www.yeastgenome.org/locus/S000001668/overview">ASH1</a></em> mRNA and Ty1 retrotransposon RNA localization, and rate of Ty1 retromobility. Another phenotype, Ty1 cDNA accumulation, correlated with both the level and isoforms of <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a> and snoRNA expressed. These cellular processes required different minimal levels of <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a> or the <em><a href="http://www.yeastgenome.org/locus/S000004947/overview">RPL7</a></em>-encoded snoRNA, but all phenotypic defects resulting from either <em><a href="http://www.yeastgenome.org/locus/S000004947/overview">RPL7</a></em> CRI being expressed from the <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> locus, or from the <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> CRI being expressed from the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> locus were suppressed by expressing the <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> CRI from both loci. Thus, <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a> and snoRNA isoforms encoded by <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> have redundant, dosage-dependent functions. Overall, the findings suggest that cellular processes have different minimal threshold-values for the level of <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a> or <a href="http://www.yeastgenome.org/locus/S000007302/overview">snR39</a>/<a href="http://www.yeastgenome.org/locus/S000006450/overview">59</a>, and, consequently, 60S ribosomal subunits that are required, and that variations in 60S ribosomal subunit levels underlie the phenotypic diversity that results from deleting one <em><a href="http://www.yeastgenome.org/locus/S000004947/overview">RPL7</a></em> paralog or the other.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Yeast strains, media, and plasmids</h3><p id="p-11">All synthetic complete (SC) drop-out media contained 2% glucose unless otherwise noted. The strains used in this study are derivatives of strain BY4741 (Open Biosystems), and genotypes are listed in Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035931/-/DC1/TableS1.docx">Table S1</a>. Strain JC3212 harbors a chromosomal Ty1<em>his3AI[∆1]</em>-3114 element (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Mou <em>et al.</em> 2006</a>). Strain JC3807 is a derivative of JC3212 harboring Ty1(<em>GAG:GFP</em>)-3566 (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Scholes <em>et al.</em> 2003</a>). Oligonucleotide primers used to amplify PCR products for gene replacement are listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035931/-/DC1/TableS2.docx">Table S2</a>. PCR products used for gene replacement were amplified with Phusion High Fidelity Polymerase (New England Biolabs). Each gene replacement was confirmed by two independent PCR analyses.</p><p id="p-12">Plasmid pGSHU, a gift from Michael Resnick, harbors the <em><a href="http://www.yeastgenome.org/locus/S000000224/overview">GAL1</a><sub>P</sub> -I-SceI-hygB-K.l.URA3</em> counter-selectable <em>delitto perfetto</em> cassette (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Storici and Resnick 2006</a>). A PCR-amplified <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆</em>::<em><a href="http://www.yeastgenome.org/locus/S000000224/overview">GAL1</a><sub>P</sub> -I-SceI-hygB-K.l.URA3</em> cassette was generated with primers PJ813 and PJ814 using plasmid pGHU as a template. The cassette was introduced into strain JC3212 by one-step gene replacement, and hygromycin-resistant (Hyg<sup>R</sup>) Ura<sup>+</sup> isolates were selected and tested for the presence of the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆</em>::<em><a href="http://www.yeastgenome.org/locus/S000000224/overview">GAL1</a><sub>P</sub> -I-SceI-hygB-K.l.URA3</em> allele. An <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆</em>::<em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> cassette, amplified by PCR using genomic DNA of an <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆ <a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> strain as a template, and primers PJ757 and PJ759, was introduced into <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆</em>::<em><a href="http://www.yeastgenome.org/locus/S000000224/overview">GAL1</a><sub>P</sub> -I-SceI-hygB-K.l.URA3</em> strains JC5896 and JC5898 by one-step gene replacement. The <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> allele was deleted from this strain using an <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">rpl7b</a>∆</em>::<em>kanMX</em> cassette that was PCR-amplified using primers PJ463 and PJ464, and genomic DNA from an <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">rpl7b</a>∆</em>::<em>kanMX</em> derivative of strain BY4741 as a template. A <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">rpl7b</a>∆</em>::<em><a href="http://www.yeastgenome.org/locus/S000000224/overview">GAL1</a><sub>P</sub> -I-SceI-hygB-K.l.URA3</em> cassette was amplified with primers PJ842 and PJ843 using plasmid pGHU as a template, and introduced into strain JC3212 by one-step gene replacement. An <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">rpl7b</a>∆</em>::<em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> cassette, generated by PCR with primers PJ845 and PJ846 and genomic DNA from an <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a> <a href="http://www.yeastgenome.org/locus/S000006119/overview">rpl7b</a>∆</em> strain as a template, was used to replace the <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">rpl7b</a>∆</em>::<em><a href="http://www.yeastgenome.org/locus/S000000224/overview">GAL1</a><sub>P</sub> -I-SceI-hygB-K.l.URA3</em> allele in strains JC5900 and JC5901. The <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> allele was deleted from this strain using an <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆</em>::<em>kanMX</em> cassette that was PCR-amplified using primers PJ459 and PJ460 and genomic DNA from the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆</em>::<em>kanMX</em> derivative of strain BY4741 as a template. An <em><a href="http://www.yeastgenome.org/locus/S000002233/overview">rpl31a</a>∆</em>::<em><a href="http://www.yeastgenome.org/locus/S000000224/overview">GAL1</a><sub>P</sub>-I-SceI-hygB-K.l.URA3</em> cassette, amplified with primers PJ893 and PJ894 using plasmid pGHU as a template, was introduced into strain JC3212 by gene replacement. An <em><a href="http://www.yeastgenome.org/locus/S000002233/overview">rpl31a</a>∆</em>::<em><a href="http://www.yeastgenome.org/locus/S000004398/overview">RPL31B</a></em> allele, generated by PCR amplification with primers PJ895 and PJ896 using genomic DNA of an <em><a href="http://www.yeastgenome.org/locus/S000002233/overview">rpl31a</a>∆ <a href="http://www.yeastgenome.org/locus/S000004398/overview">RPL31B</a></em> strain as a template, was used to replace the <em><a href="http://www.yeastgenome.org/locus/S000002233/overview">rpl31a</a>∆</em>::<em><a href="http://www.yeastgenome.org/locus/S000000224/overview">GAL1</a><sub>P</sub>-I-SceI-hygB-K.l.URA3</em> allele in strains JC6130 and JC6131. The <em><a href="http://www.yeastgenome.org/locus/S000004398/overview">RPL31B</a></em> allele in this strain was deleted by one-step gene replacement using a DNA fragment that was PCR-amplified with primers PJ471 and PJ472, and genomic DNA from the <em><a href="http://www.yeastgenome.org/locus/S000004398/overview">rpl31b</a>∆</em>::<em>kanMX</em> derivative of strain BY4741 as a template. The <em><a href="http://www.yeastgenome.org/locus/S000003394/overview">tif4631</a>∆</em>::<em>kanMX</em> derivative of strain JC3212 was made by gene replacement using a cassette that was PCR-amplified using primers PJ371 and PJ372, and the <em><a href="http://www.yeastgenome.org/locus/S000003394/overview">tif4631</a>∆</em>::<em>kanMX</em> derivative of strain BY4741 as a template. Strains harboring the <em><a href="http://www.yeastgenome.org/locus/S000004494/overview">rad52</a></em>::<em>hisG-URA3-hisG</em> allele were constructed by one-step gene replacement using a DNA fragment isolated from plasmid pBDG542, as described previously (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Curcio and Garfinkel 1994</a>).</p><p id="p-13">Derivatives of yeast strain JC3709 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035931/-/DC1/TableS2.docx">Table S2</a>) harboring <em>9xMYC</em>-tagged alleles of <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> were made by using PCR-amplified <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a>:9xMyc-K.l.TRP1</em> and <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a>:9xMyc-K.l.TRP1</em> cassettes for one-step gene replacement. The <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a>:9xMyc-K.l.TRP1</em> cassette was amplified using primers PJ564 and PJ565, and plasmid pYM6 (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Knop <em>et al.</em> 1999</a>) as a template. The <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a>:9xMyc-K.l.TRP1</em> cassette was amplified using primers PJ566 and PJ567, and plasmid pYM6 as a template. The corresponding untagged paralog was deleted in each strain by one-step gene replacement using a PCR-amplified <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆</em>::<em>kanMX</em> or <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">rpl7b</a>∆</em>::<em>kanMX</em> cassette, respectively. Construction of these deletion cassettes is described above.</p><p id="p-14">pBJC1058, a <em><a href="http://www.yeastgenome.org/locus/S000000523/overview">LEU2</a>-<a href="http://www.yeastgenome.org/locus/S000001896/overview">CEN6</a></em> plasmid that contains the pLTR<sub>P</sub>:Gag<sub>1–401</sub>:GFP:<em><a href="http://www.yeastgenome.org/locus/S000005446/overview">ADH1</a></em><sub>TER</sub> cassette, has been described previously (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Doh <em>et al.</em> 2014</a>).</p><p id="p-15">Plasmid pGTy1(<em>synGAG</em>) consists of pRS416 carrying a <em><a href="http://www.yeastgenome.org/locus/S000000224/overview">GAL1</a><sub>P</sub></em>-driven Ty1 element that encodes functional <em>GAG</em> and <em>POL</em> ORFs, but carries silent nucleotide substitutions in the 5′ UTR and <em>GAG</em> ORF, and lacks the 3′ LTR. Plasmid pGTy1(<em>synGAG</em>) is a derivative of plasmid pBJC838 (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Stamenova <em>et al.</em> 2009</a>). The transcriptional start site of Ty1 (nucleotide 241 of Ty1-H3; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Boeke <em>et al.</em> 1988</a>) is fused to <em><a href="http://www.yeastgenome.org/locus/S000000224/overview">GAL1</a><sub>P</sub></em> at a <em>Xho</em>I site. In addition, the 3′ LTR and <em>his3AI</em> marker gene were deleted by removal of a DNA fragment from the internal <em>Bgl</em>II site to the end of the element (nucleotides 5564–5918 of Ty1-H3). Finally, a sequence-optimized 577 bp <em>Xho</em>I–<em>Hpa</em>I fragment has replaced the <em>Xho</em>I–<em>Hpa</em>I fragment of Ty1-H3 (nucleotides 241–817 of Ty1-H3). The sequence-optimized fragment, synthesized by GenScript, harbors multiple nucleotide changes in the Ty1 RNA 5′ UTR region to reduce predicted secondary structure elements, as well as silent mutations in <em>GAG</em> predicted by the OptimumGene algorithm (GenScript) to optimize gene expression (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035931/-/DC1/FigureS1.tif">Figure S1</a>). The <em><a href="http://www.yeastgenome.org/locus/S000000224/overview">GAL1</a><sub>P</sub></em>:Ty1(<em>synGAG</em>) fragment is carried as an <em>Apa</em>I–<em>Eag</em>I fragment on the <em><a href="http://www.yeastgenome.org/locus/S000000523/overview">LEU2</a>-<a href="http://www.yeastgenome.org/locus/S000001896/overview">CEN6</a></em> vector pRS415.</p><p id="p-16">Plasmid pBJC1198 contains the <em><a href="http://www.yeastgenome.org/locus/S000000224/overview">GAL1</a></em><sub>P</sub>-<em><a href="http://www.yeastgenome.org/locus/S000001668/overview">ASH1</a></em>-MS2L(x12)-<em><a href="http://www.yeastgenome.org/locus/S000005446/overview">ADH1</a><sub>TER</sub></em> cassette on the <em><a href="http://www.yeastgenome.org/locus/S000000523/overview">LEU2</a>-<a href="http://www.yeastgenome.org/locus/S000001896/overview">CEN6</a></em> vector, pRS415. The <em><a href="http://www.yeastgenome.org/locus/S000001668/overview">ASH1</a></em> ORF (from the start codon to the stop codon) was amplified from BY4741 genomic DNA with primers PJ1285, which introduced a 5′ <em>Xho</em>I site, and PJ1286, which introduced a 3′ <em>Bam</em>HI site. The <em>Apa</em>I–<em>Xho</em>I fragment of plasmid pBJC838 containing the <em><a href="http://www.yeastgenome.org/locus/S000000224/overview">GAL1</a><sub>P</sub></em> (<a id="xref-ref-72-2" class="xref-bibr" href="#ref-72">Stamenova <em>et al.</em> 2009</a>), the <em>Xho</em>I–<em>Bam</em>HI fragment containing the <em><a href="http://www.yeastgenome.org/locus/S000001668/overview">ASH1</a></em> ORF, and the <em>Bam</em>HI–<em>Bgl</em>II fragment of pSL-MS2-12X (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Bertrand <em>et al.</em> 1998</a>) were cloned into pRS415 (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Sikorski and Hieter 1989</a>), following digestion with <em>Apa</em>I and <em>Bam</em>HI. An <em>Eag1</em> fragment harboring <em><a href="http://www.yeastgenome.org/locus/S000005446/overview">ADH1</a></em> terminator sequences (<em><a href="http://www.yeastgenome.org/locus/S000005446/overview">ADH1</a><sub>TER</sub></em>) was cloned into the <em>Eag</em>I site of the resulting plasmid. <em><a href="http://www.yeastgenome.org/locus/S000005446/overview">ADH1</a><sub>TER</sub></em> was amplified from plasmid pBJC1058 with primers PJ1169 and PJ944.</p><p id="p-17">Plasmid pMS2-CP-GFP(3X), a gift from Jeffrey Gerst, expresses the MS2 coat binding protein fused to three tandem GFP moieties, and is driven by the <em><a href="http://www.yeastgenome.org/locus/S000004294/overview">MET15</a></em> promoter (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Haim-Vilmovsky and Gerst 2009</a>). Basal expression of pMS2-CP-GFP(x3) in the presence of methionine was sufficient to visualize p<em><a href="http://www.yeastgenome.org/locus/S000001668/overview">ASH1</a>-MS2L</em>(x12) mRNA while minimizing background signal.</p></div><div id="sec-3" class="subsection"><h3>Western blot analyses</h3><p id="p-18">Culture volumes corresponding to 2.5 OD<sub>600</sub> units of cells grown to an OD<sub>600</sub> of 0.6–1.0 at 20° were harvested, and protein was extracted as previously described (<a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Yarrington <em>et al.</em> 2012</a>). Proteins were separated on 10% SDS-Tris gels and transferred to an Immun-Blot PVDF membrane (Bio-Rad). To detect Gag, membranes were incubated with affinity-purified anti-VLP polyclonal antibodies (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Conte and Curcio 2000</a>), at a 1:35,000 dilution in 0.005% nonfat milk in phosphate-buffered saline and 0.05% Tween 20 (PBST), followed by incubation with horseradish peroxidase (HRP)-conjugated goat anti-rabbit secondary polyclonal antibody (Millipore) in 0.005% nonfat milk in PBST. Rpl7a-9xMyc and Rpl7b-9xMyc were detected with a 1:5000 dilution of monoclonal anti-c-Myc antibody (clone 9E10; Sigma), followed by incubation with HRP-conjugated sheep anti-mouse secondary polyclonal antibody (GE Healthcare UK Limited; lot # 357597). Membranes were stripped in 50 mM Tris-HCl pH 7, 2% SDS, 50 mM DTT at 70°. Membranes originally used to detect Gag were incubated in incubated in a 1:2000 dilution of anti-humL7 polyclonal antibodies (Bethyl Labs, Inc.) in PBST, and then HRP-conjugated goat anti-rabbit secondary polyclonal antibodies (Millipore) in PBST. After stripping, the membrane was incubated in a 1:2000 dilution of anti-β-Actin monoclonal antibody (Abcam) in PBST with 2.5% nonfat milk, followed by incubation with a 1:4000 dilution of HRP-conjugated sheep anti-mouse secondary polyclonal antibody (GE Healthcare UK Limited; lot # 357597) in PBST with 2.5% nonfat milk. Membranes originally used to detect Rpl7a-9xMyc and Rpl7b-9xMyc were incubated in a 1:2000 dilution of anti-α-Tubulin polyclonal antibody (Chemicon International) in PBST with 2.5% nonfat milk, followed by incubation in a 1:5000 dilution of HRP-conjugated donkey anti-rat secondary polyclonal antibody (Millipore; lot # 2050222) in PBST with 2.5% nonfat milk. All bands were visualized by incubation of membranes in SuperSignal West Pico chemiluminescent substrate (Pierce) and exposure to film. Films were scanned, and the intensity of nonsaturated bands was determined using ImageJ software. Gag and <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a> values were individually divided by the values for β-Actin, and corrected for differences in dilution.</p></div><div id="sec-4" class="subsection"><h3>Growth assays</h3><p id="p-19">To determine the rate of cell doubling of each strain, cells were scraped from fresh patches on YPD agar, and were resuspended in YPD broth at an OD<sub>600</sub> of &lt;0.01. Three separate cultures of each strain were incubated with shaking at either 30 or 20° to OD<sub>600</sub> ∼0.6. The doubling time (<em>t</em><sub>2</sub>) was calculated for each culture based on the following equation (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Amberg <em>et al.</em> 2005</a>), where <em>t<sub>f</sub></em> is the time in hours that the cultures were incubated, OD<em><sub>f</sub></em> is the OD<sub>600</sub> of the cultures after incubation, and OD<em><sub>i</sub></em> is the OD<sub>600</sub> of the dilutions before incubation:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/591/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/591/embed/mml-math-1.gif"/></noscript></span></span>For serial dilution growth assays, cells from fresh patches on YPD plates were resuspended in YPD broth at an OD<sub>600</sub> of ∼0.3. These suspensions were diluted 10-fold serially, from 1:10 to 1:10,000. A 5 µl aliquot of each dilution was plated onto YPD agar, with or without 5 µg/ml tunicamycin (Sigma), and incubated at 30° for the indicated times.</p></div><div id="sec-5" class="subsection"><h3>Polysome analysis</h3><p id="p-20">Cells were grown to an OD<sub>600</sub> of 0.4–0.6 at 20° in YPD broth, and 2.0 OD<sub>600</sub> units of cells were collected. Cells were washed and lysed by bead beating in 20 mM Tris-HCl pH 8.0, 140 mM KCl, 5 mM MgCl<sub>2</sub>, 0.5 mM DTT, 1% Triton X-100, 1 mg/ml heparin, cOmplete Mini, EDTA-free protease inhibitor cocktail (Roche), and 200 units/ml RNasin (Promega). Debris was pelleted at 4000 rpm at 4° for 5 min. Clarified lysates were transferred to a fresh tube, and centrifuged at 10,000 rpm at 4° for 5 min. Cell extracts were separated by ultracentrifugation at 35,000 rpm in 10–50% (w/v) sucrose gradients containing 20 mM Tris-HCl pH 8.0, 140 mM KCl, 5 mM MgCl<sub>2</sub>, 0.5 mM DTT, and 1 mg/ml heparin for 165 min at 4° in an SW41 rotor. Gradients were fractionated with a Teledyne/ISCO gradient fractionation system. Traces were recorded on a UA6 detector at a sensitivity setting of 0.2. In addition, traces for <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆ <a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆ <a href="http://www.yeastgenome.org/locus/S000006119/overview">rpl7b</a>∆</em>::<em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> strains were collected at a sensitivity setting of 0.1.</p></div><div id="sec-6" class="subsection"><h3>Fluorescence microscopy</h3><p id="p-21">Three independent transformants of strains JC3212, JC5987, JC5989, and JC5914 with plasmids p<em><a href="http://www.yeastgenome.org/locus/S000000224/overview">GAL1</a><sub>P</sub>-<a href="http://www.yeastgenome.org/locus/S000001668/overview">ASH1</a></em>-MS2L(x12) and pMS2-CP-GFP(x3) were grown overnight in SC-HIS-LEU broth at 30°. Aliquots of each culture were inoculated into 10 ml of fresh SC-HIS-LEU broth containing 2% galactose, 2% raffinose, and 2% sucrose, and incubated at 30° to OD<sub>600</sub> of 0.6. A 2.5 OD<sub>600</sub> unit aliquot of each culture was harvested, resuspended in 1 ml of water, and incubated with 2.5 µl Hoechst 33258 for 20 min at room temperature. Cells were washed three times with water prior to visualization.</p><p id="p-22">Detection of Ty1 RNA by fluorescence <em>in situ</em> hybridization was performed as previously described (<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Doh <em>et al.</em> 2014</a>). A Zeiss Axioskop 200 M inverted microscope was used (filter set: 31 for Cy3, and 34 for DAPI) at a magnification of 63×. Photographs were taken with a Q Imagining Camera (or Hamamatsu ORCA ER), and merged and colored with Openlab 4.0.4 software (Improvision).</p></div><div id="sec-7" class="subsection"><h3>Gag-GFP activity assay</h3><p id="p-23">Two independent cultures of strain JC3807 and derivatives, which harbor the Ty1(<em>GAG-GFP</em>)-3566 chromosomal element (<a id="xref-ref-67-2" class="xref-bibr" href="#ref-67">Scholes <em>et al.</em> 2003</a>), and strain BY4741 and derivatives, which do not, were grown in YPD broth overnight at 30°. Duplicate cultures of each strain were diluted to an OD<sub>600</sub> of 0.1 in YPD broth, and grown at 20° for 3 hr. The cells were diluted 1:10 in deionized H<sub>2</sub>O. The geometric mean of the GFP activity in 10,000–20,000 cells of each strain was measured by flow cytometry using a FACSCalibur (Becton Dickinson). The geometric mean of the GFP activity for each strain lacking Ty1(<em>GAG-GFP</em>)-3566 was subtracted from the mean GFP activity in the isogenic strain harboring Ty1(<em>GAG-GFP</em>)-3566 to correct for autofluorescence arising from differences in cell size.</p></div><div id="sec-8" class="subsection"><h3>RNA isolation and quantitative real-time PCR</h3><p id="p-24">Cells of each strain grown overnight in YPD broth at 30° were diluted to an OD<sub>600</sub> of 0.3 in 50 ml YPD broth, and grown to an OD<sub>600</sub> of 0.6 at 20°. Cell harvesting, RNA extraction, and DNase treatment was performed as previously described (<a id="xref-ref-61-3" class="xref-bibr" href="#ref-61">Risler <em>et al.</em> 2012</a>). Total RNA was used as a template for qRT-PCR, which was performed as described previously (<a id="xref-ref-61-4" class="xref-bibr" href="#ref-61">Risler <em>et al.</em> 2012</a>), with the following modifications. Ty1 and <em>snR6</em> RNAs were reverse transcribed with the sequence-specific primers TY5253A and PJ751, respectively. Ty1 and <em>snR6</em> single-stranded cDNAs were amplified via qPCR with PJ748 and PJ1230, and PJ750 and PJ1220, respectively. The average fold change in Ty1 RNA level was calculated according to the 2<sup>−∆∆Ct</sup> method using average Ty1 and <em>snR6</em> cycle thresholds (Ct) from three biological replicates.</p></div><div id="sec-9" class="subsection"><h3>Ty1his3AI retrotransposition assays</h3><p id="p-25">To determine the retromobility rate of the Ty1<em>his3AI-∆1-3114</em> element (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Mou <em>et al.</em> 2006</a>), three biological replicates of each strain were inoculated to a density of 1 × 10<sup>−4</sup> cells/ml in seven cultures of YPD broth for a total of 21 cultures per genotype. Cultures were incubated with shaking at 20° for 3–4 d. A 1 µl aliquot of a 1 × 10<sup>−3</sup> dilution of four randomly selected cultures was plated on SC agar to determine the titer. All seven cultures were plated on SC-HIS agar to determine the number of His<sup>+</sup> prototrophs. The Ty1<em>his3AI-∆1</em> retromobility rate for each genotype was determined by dividing the mean titer into the mutation rate, which was calculated by using the median number of His<sup>+</sup> prototrophs per 1 ml culture in the Lea and Coulson median estimator (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Rosche and Foster 2000</a>).</p><p id="p-26">To determine the frequency of Ty1<em>his3AI-∆1-3114</em> retrotransposition during induction of expression from plasmid pGTy1(<em>synGAG</em>)∆3′LTR, four independent transformants of strains harboring plasmid pRS415, or pGTy1(<em>synGAG</em>)∆3′LTR, were inoculated into SC-LEU broth containing 2% raffinose and 2% sucrose, and incubated overnight at 30° with rolling. Cultures were diluted 1:10 in YP broth containing 2% galactose, 2% raffinose, and 2% sucrose, and incubated at 20° for 3 d. A 1 µl aliquot of each culture was plated on SC-LEU agar to determine the number of cells that retained the plasmid throughout incubation, and the remainder of each culture was plated on SC-LEU-HIS agar to determine the number of His<sup>+</sup> prototrophs. The mean Ty1<em>his3AI</em> retromobility frequency is the mean of the number of His<sup>+</sup>Leu<sup>+</sup> prototrophs per culture divided by the total number of Leu<sup>+</sup> prototrophs per culture.</p><p id="p-27">To determine the retromobility frequency of the Ty1<em>his3AI-∆1-3114</em> element in congenic <em><a href="http://www.yeastgenome.org/locus/S000004494/overview">RAD52</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000004494/overview">rad52</a></em> strains, seven independent cultures of yeast cells of each genotype were inoculated to a density of 1 × 10<sup>−4</sup> cells/ml in YPD broth, and cultures were grown at 20° each genotype were inoculated to a density<sup>−6</sup> dilution of each culture was plated on SC agar to determine the titer. The remaining culture was plated on SC-HIS agar to determine the number of His<sup>+</sup> prototrophs. The retromobility frequency of each culture was calculated as the fraction of total cells that were His<sup>+</sup>. The mean retromobility frequency was determined from seven cultures of each genotype.</p></div><div id="sec-10" class="subsection"><h3>Quantitative Southern blot assay for Ty1 cDNA</h3><p id="p-28">To quantify unintegrated Ty1 cDNA in each strain, total genomic DNA was harvested from cells grown to saturation at 20°. DNA samples were digested with <em>Sph</em>I, and separated on a 1% TBE agarose gel. Southern blot analysis was performed with an antisense Ty1 <em>POL</em> riboprobe <em>in vitro</em>-transcribed using plasmid pJC525 as a template. The intensity of bands corresponding to Ty1 cDNA, and two different genomic DNA bands, was measured with ImageQuant software.</p></div><div id="sec-11" class="subsection"><h3>Data availability</h3><p id="p-29">Strains and plasmids are available upon request. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035931/-/DC1/FigureS1.tif">Figure S1</a> illustrates nucleotide substitutions in Ty1(<em>syn-Gag</em>) relative to Ty1-H3. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035931/-/DC1/TableS1.docx">Table S1</a> contains genotypes of strains used in this study. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035931/-/DC1/TableS2.docx">Table S2</a> contains oligonucleotide primers used in this study. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035931/-/DC1/TableS3.docx">Table S3</a> lists the chromosomal location and orientation of Ty1<em><a href="http://www.yeastgenome.org/locus/S000005728/overview">HIS3</a></em> transposition events. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035931/-/DC1/FileS1.docx">File S1</a> contains methods and references for <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035931/-/DC1/TableS3.docx">Table S3</a>. All data necessary for confirming the conclusions presented in the article are represented fully within the article and the associated supplementary files.</p></div></div><div class="section results" id="sec-12"><h2 class="">Results</h2><div id="sec-13" class="subsection"><h3>Construction of chimeric RPL7 alleles</h3><p id="p-30">To determine whether the <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a> or snoRNA isoforms encoded by <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> have evolved distinct functions irrespective of the disparity in their expression, we created chimeric <em><a href="http://www.yeastgenome.org/locus/S000004947/overview">RPL7</a></em> alleles in which the CRI of one paralog, extending from the start codon to the stop codon, and including the <em><a href="http://www.yeastgenome.org/locus/S000007302/overview">snR39</a></em> or <em><a href="http://www.yeastgenome.org/locus/S000006450/overview">snR59</a></em> snoRNA gene, was substituted for the CRI of the other paralog. The resulting chimeric alleles consist of the CRI of one paralog, and the 5′ UTR, 3′ UTR, promoter, and terminator, denoted as the “locus,” of the other. Mutants harboring a chimeric allele were constructed by precisely replacing the CRI of <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> or <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> in strain JC3212 with a counterselectable <em>delitto perfetto</em> cassette (<a id="xref-ref-75-2" class="xref-bibr" href="#ref-75">Storici and Resnick 2006</a>) (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>). Each cassette was then precisely replaced with the <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> or <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> CRI, respectively. The resulting “homogenic” strains harbor the same CRI at both the native allele, and a chimeric <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">rpl7b</a>∆</em>::<em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> or <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆</em>::<em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> allele (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Clarkson <em>et al.</em> 2010</a>). Subsequently, the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> or <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> CRI was deleted at its native locus in homogenic strains to yield “chimeragenic” strains (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>). Since one <em><a href="http://www.yeastgenome.org/locus/S000004947/overview">RPL7</a></em> gene is required for survival (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Mizuta <em>et al.</em> 1992</a>), the viability of chimeragenic strains demonstrates that both chimeric <em><a href="http://www.yeastgenome.org/locus/S000004947/overview">RPL7</a></em> alleles are functional.</p></div><div id="sec-14" class="subsection"><h3>Growth rates and 60S ribosomal subunit abundance in strains with different RPL7 alleles</h3><p id="p-31">The growth rates in <em><a href="http://www.yeastgenome.org/locus/S000004947/overview">RPL7</a></em> deletion, chimeragenic and homogenic strains were measured in YPD broth at 30 and 20°, the latter permissive for retrotransposition of Ty1 (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). The <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a> <a href="http://www.yeastgenome.org/locus/S000006119/overview">rbl7b</a>∆</em> mutant had a wild-type growth rate. Mutants expressing the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> or <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> CRI from the <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> locus exhibited doubling times that were 1.7-fold longer at 30°, and 1.9- to 2.0-fold longer at 20°, when compared to strains expressing the corresponding CRI from the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> locus. These data suggest that, in mutants harboring only one <em><a href="http://www.yeastgenome.org/locus/S000004947/overview">RPL7</a></em> allele, the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> locus is more strongly expressed than the <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> locus. There was also a small increase in doubling time (1.1-fold longer at 30° and 1.2- to 1.3-fold longer at 20°) when the <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> <em>vs.</em> the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> CRI was expressed from either locus, suggesting that higher levels of <a href="http://www.yeastgenome.org/locus/S000003044/overview">Rpl7a</a>/<a href="http://www.yeastgenome.org/locus/S000007302/overview">snR39</a> are expressed from either <em><a href="http://www.yeastgenome.org/locus/S000004947/overview">RPL7</a></em> loci, or that <a href="http://www.yeastgenome.org/locus/S000007302/overview">Rpl7a</a>/<a href="http://www.yeastgenome.org/locus/S000007302/overview">snR39</a> is more active than <a href="http://www.yeastgenome.org/locus/S000006119/overview">Rpl7b</a>/<a href="http://www.yeastgenome.org/locus/S000006450/overview">snR59</a> in 60S ribosomal subunit biogenesis. Increasing the dosage of <a href="http://www.yeastgenome.org/locus/S000006119/overview">Rpl7b</a>/<a href="http://www.yeastgenome.org/locus/S000006450/overview">snR59</a> by expressing the <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> CRI from both loci suppressed the major and minor growth defect of mutants expressing the <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> CRI from the <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> or <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> locus, respectively. In fact, the doubling times of the homogenic <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆</em>::<em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> strain at 20 and 30° were within 5% of those of the wild-type strain, and those of the homogenic <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a> <a href="http://www.yeastgenome.org/locus/S000006119/overview">rpl7b</a>∆</em>::<em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> strain were indistinguishable from wild type. These findings indicate that the <em><a href="http://www.yeastgenome.org/locus/S000004947/overview">RPL7</a></em> locus or loci expressed have a greater effect on cell doubling time than the particular <em><a href="http://www.yeastgenome.org/locus/S000004947/overview">RPL7</a></em> CRI that is expressed.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/591/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The RPL7 locus is the primary determinant of cell growth rate, Rpl7 level and tunicamycin resistance. (A) The rate of cell doubling &#xB1;&#xA0;SD in three biological replicates of each strain of the indicated genotype grown in YPD broth at 20 or 30&#xB0; to an OD600 of 0.6. (B) Western blot analysis of Rpl7 and Ty1 Gag, which runs as a doublet, in rpl7 deletion and chimeragenic strains. Each lane from left to right represents a twofold serial dilution of whole cell lysate. Gag was detected with anti-VLP antisera, and Rpl7 isoforms were detected with polyclonal anti-human L7 antibodies. Act1 was detected with a monoclonal anti-Actin antibody as a loading control. (C) The relative tunicamycin resistance of each strain of the indicated genotype grown to an OD600 of 0.6 in YPD broth at 30&#xB0;. A 5&#xA0;&#xB5;l aliquot of each culture, and 1:10 serial dilutions were plated on YPD agar, with or without 5&#xA0;&#xB5;g/ml tunicamycin. Plates were incubated at 30&#xB0; for 1&#xA0;d (YPD) or 5&#xA0;d (YPD with tunicamycin)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1111785762" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The &lt;em&gt;RPL7&lt;/em&gt; locus is the primary determinant of cell growth rate, Rpl7 level and tunicamycin resistance. (A) The rate of cell doubling &#xB1;&#xA0;SD in three biological replicates of each strain of the indicated genotype grown in YPD broth at 20 or 30&#xB0; to an OD&lt;sub&gt;600&lt;/sub&gt; of 0.6. (B) Western blot analysis of Rpl7 and Ty1 Gag, which runs as a doublet, in &lt;em&gt;rpl7&lt;/em&gt; deletion and chimeragenic strains. Each lane from left to right represents a twofold serial dilution of whole cell lysate. Gag was detected with anti-VLP antisera, and Rpl7 isoforms were detected with polyclonal anti-human L7 antibodies. Act1 was detected with a monoclonal anti-Actin antibody as a loading control. (C) The relative tunicamycin resistance of each strain of the indicated genotype grown to an OD&lt;sub&gt;600&lt;/sub&gt; of 0.6 in YPD broth at 30&#xB0;. A 5&#xA0;&#xB5;l aliquot of each culture, and 1:10 serial dilutions were plated on YPD agar, with or without 5&#xA0;&#xB5;g/ml tunicamycin. Plates were incubated at 30&#xB0; for 1&#xA0;d (YPD) or 5&#xA0;d (YPD with tunicamycin).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/591/F2.medium.gif" width="334" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/2/591/F2.medium.gif" width="334" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/591/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/591/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470350" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-32" class="first-child">The <em>RPL7</em> locus is the primary determinant of cell growth rate, Rpl7 level and tunicamycin resistance. (A) The rate of cell doubling ± SD in three biological replicates of each strain of the indicated genotype grown in YPD broth at 20 or 30° to an OD<sub>600</sub> of 0.6. (B) Western blot analysis of Rpl7 and Ty1 Gag, which runs as a doublet, in <em>rpl7</em> deletion and chimeragenic strains. Each lane from left to right represents a twofold serial dilution of whole cell lysate. Gag was detected with anti-VLP antisera, and Rpl7 isoforms were detected with polyclonal anti-human L7 antibodies. Act1 was detected with a monoclonal anti-Actin antibody as a loading control. (C) The relative tunicamycin resistance of each strain of the indicated genotype grown to an OD<sub>600</sub> of 0.6 in YPD broth at 30°. A 5 µl aliquot of each culture, and 1:10 serial dilutions were plated on YPD agar, with or without 5 µg/ml tunicamycin. Plates were incubated at 30° for 1 d (YPD) or 5 d (YPD with tunicamycin).</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">Polysome profiles of the <em><a href="http://www.yeastgenome.org/locus/S000004947/overview">RPL7</a></em> chimeragenic and deletion strains were compared to determine whether differences in cell doubling times are reflected in altered 60S ribosomal subunit levels (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). Cells were grown at 20° because differences in doubling time were more pronounced at this temperature (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>). Even the wild-type strain had higher 40S and 60S ribosomal subunit peaks than 80S or polysome peaks at this suboptimal temperature (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>). Nonetheless, mutants with either <em><a href="http://www.yeastgenome.org/locus/S000004947/overview">RPL7</a></em> CRI at the <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> locus only had greatly diminished levels of 60S ribosomal subunits, 80S monosomes, and polysomes relative to the wild-type strain (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3A</a>). In addition, halfmer peaks, in which 40S ribosomal subunits that are not joined with 60S subunits are associated with mRNA, were evident as shoulders on the 80S monosome and disome peaks (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3B</a>), indicating that the ratio of 60S to 40S ribosomal subunits is markedly reduced in these mutants. In contrast, the polysome profiles of mutants with either <em><a href="http://www.yeastgenome.org/locus/S000004947/overview">RPL7</a></em> CRI at the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> locus more closely resembled that of the wild-type strain, and halfmer polysomes were not detected (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3C</a>). Compared to 40S ribosomal subunits, there was a significantly lower level of 60S ribosomal subunits, and reduced 80S monosomes and polysomes when the <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> CRI <em>vs.</em> the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> CRI was present at the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> locus (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3, C and E</a>). These discrepancies paralleled the modest difference in growth rate between these mutants. Expression of the <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> CRI from both <em><a href="http://www.yeastgenome.org/locus/S000004947/overview">RPL7</a></em> loci suppressed the 60S/40S ribosomal subunit imbalance of the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆</em>::<em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a>∆</em> mutant (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3, C<strong>–</strong>E</a>). The polysome profiles of homogenic<em> <a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a> <a href="http://www.yeastgenome.org/locus/S000006119/overview">rpl7b</a>∆</em>::<em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆</em>::<em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> strains were both similar to that of the wild-type strain, although there was a minor reduction in light polysomes in both (<a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3D</a>), and a decrease in the ratio of disomes to 80S monosomes in the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆</em>::<em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> (<a id="xref-fig-3-9" class="xref-fig" href="#F3">Figure 3E</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/591/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Primarily the RPL7 locus, but also the Rpl7 and snR39/59 isoforms expressed, contributes to differences in polysome profiles. Polysome profiles of cell extracts from strains of the indicated genotype grown in YPD broth at 20&#xB0;, and fractionated in 10&#x2013;50% (w/v) sucrose gradients. Absorbance units at 254&#xA0;nm from the top to the bottom of the gradient for each genotype are shown in each graph. The same trace of the RPL7A RPL7B strain is displayed in black in (A), (C), and (D) to facilitate comparisons between panels. (A) Polysome profiles of the mutants with the RPL7B (red), or RPL7A (blue), CRI at the RPL7B locus are shown, and the genotypes indicated. Traces of the two mutants and the RPL7A RPL7B strain were overlaid by aligning the lowest point of each trace between the 80S monosome peak and the disome peak. The sensitivity setting of the UV monitor was 0.2. The arrow points to a minor peak of halfmer polyribosomes on the right shoulder of the 80S peak in rpl7a&#x2206; RPL7B and rpl7a&#x2206; rpl7b&#x2206;::RPL7A mutants. (B) Reanalysis of the rpl7a&#x2206; RPL7B and rpl7a&#x2206; rpl7b&#x2206;::RPL7A mutants, with the UV monitor at a more sensitive setting of 0.1, and profiles normalized to the 80S peak for better resolution of the halfmer polyribosome peaks, which are indicated by arrows. The 40S peak cannot be resolved at this sensitivity setting. (C, D) Polysome profiles of mutants of the indicated genotypes with the RPL7A (red), or RPL7B (blue), CRI at the RPL7A locus. In each panel, traces of three strains were overlaid by aligning the lowest point of each trace between the 80S monosome peak and disome peak. The sensitivity setting of the UV monitor was 0.2. (E) A graph of relative levels of 60S ribosomal subunits, 80S monosomes, disomes, and polysomes in three biological replicates of each strain of the indicated genotype. To normalize for differences in lysis efficiency and lysate amount, the heights of 40S, 60S, 80S, disome, and highest polysome peaks in polysome profiles were measured, and ratios indicated on the y-axis were calculated. Statistically significant differences in the indicated ratios between strains of different genotypes, as determined by a two-way ANOVA test, are indicated with asterisks. *P&#xA0;" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1111785762" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Primarily the &lt;em&gt;RPL7&lt;/em&gt; locus, but also the Rpl7 and snR39/59 isoforms expressed, contributes to differences in polysome profiles. Polysome profiles of cell extracts from strains of the indicated genotype grown in YPD broth at 20&#xB0;, and fractionated in 10&#x2013;50% (w/v) sucrose gradients. Absorbance units at 254&#xA0;nm from the top to the bottom of the gradient for each genotype are shown in each graph. The same trace of the &lt;em&gt;RPL7A RPL7B&lt;/em&gt; strain is displayed in black in (A), (C), and (D) to facilitate comparisons between panels. (A) Polysome profiles of the mutants with the &lt;em&gt;RPL7B&lt;/em&gt; (red), or &lt;em&gt;RPL7A&lt;/em&gt; (blue), CRI at the &lt;em&gt;RPL7B&lt;/em&gt; locus are shown, and the genotypes indicated. Traces of the two mutants and the &lt;em&gt;RPL7A RPL7B&lt;/em&gt; strain were overlaid by aligning the lowest point of each trace between the 80S monosome peak and the disome peak. The sensitivity setting of the UV monitor was 0.2. The arrow points to a minor peak of halfmer polyribosomes on the right shoulder of the 80S peak in &lt;em&gt;rpl7a&#x2206; RPL7B&lt;/em&gt; and &lt;em&gt;rpl7a&#x2206; rpl7b&#x2206;&lt;/em&gt;::&lt;em&gt;RPL7A&lt;/em&gt; mutants. (B) Reanalysis of the &lt;em&gt;rpl7a&#x2206; RPL7B&lt;/em&gt; and &lt;em&gt;rpl7a&#x2206; rpl7b&#x2206;&lt;/em&gt;::&lt;em&gt;RPL7A&lt;/em&gt; mutants, with the UV monitor at a more sensitive setting of 0.1, and profiles normalized to the 80S peak for better resolution of the halfmer polyribosome peaks, which are indicated by arrows. The 40S peak cannot be resolved at this sensitivity setting. (C, D) Polysome profiles of mutants of the indicated genotypes with the &lt;em&gt;RPL7A&lt;/em&gt; (red), or &lt;em&gt;RPL7B&lt;/em&gt; (blue), CRI at the &lt;em&gt;RPL7A&lt;/em&gt; locus. In each panel, traces of three strains were overlaid by aligning the lowest point of each trace between the 80S monosome peak and disome peak. The sensitivity setting of the UV monitor was 0.2. (E) A graph of relative levels of 60S ribosomal subunits, 80S monosomes, disomes, and polysomes in three biological replicates of each strain of the indicated genotype. To normalize for differences in lysis efficiency and lysate amount, the heights of 40S, 60S, 80S, disome, and highest polysome peaks in polysome profiles were measured, and ratios indicated on the &lt;em&gt;y&lt;/em&gt;-axis were calculated. Statistically significant differences in the indicated ratios between strains of different genotypes, as determined by a two-way ANOVA test, are indicated with asterisks. *&lt;em&gt;P&lt;/em&gt;&#xA0;&lt;&#xA0;0.05, **&lt;em&gt;P&lt;/em&gt;&#xA0;&lt;&#xA0;0.01, ***&lt;em&gt;P&lt;/em&gt;&#xA0;&lt;&#xA0;0.001, ****&lt;em&gt;P&lt;/em&gt;&#xA0;&lt;&#xA0;0.0001.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/591/F3.medium.gif" width="330" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/2/591/F3.medium.gif" width="330" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/591/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/591/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470307" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-34" class="first-child">Primarily the <em>RPL7</em> locus, but also the Rpl7 and snR39/59 isoforms expressed, contributes to differences in polysome profiles. Polysome profiles of cell extracts from strains of the indicated genotype grown in YPD broth at 20°, and fractionated in 10–50% (w/v) sucrose gradients. Absorbance units at 254 nm from the top to the bottom of the gradient for each genotype are shown in each graph. The same trace of the <em>RPL7A RPL7B</em> strain is displayed in black in (A), (C), and (D) to facilitate comparisons between panels. (A) Polysome profiles of the mutants with the <em>RPL7B</em> (red), or <em>RPL7A</em> (blue), CRI at the <em>RPL7B</em> locus are shown, and the genotypes indicated. Traces of the two mutants and the <em>RPL7A RPL7B</em> strain were overlaid by aligning the lowest point of each trace between the 80S monosome peak and the disome peak. The sensitivity setting of the UV monitor was 0.2. The arrow points to a minor peak of halfmer polyribosomes on the right shoulder of the 80S peak in <em>rpl7a∆ RPL7B</em> and <em>rpl7a∆ rpl7b∆</em>::<em>RPL7A</em> mutants. (B) Reanalysis of the <em>rpl7a∆ RPL7B</em> and <em>rpl7a∆ rpl7b∆</em>::<em>RPL7A</em> mutants, with the UV monitor at a more sensitive setting of 0.1, and profiles normalized to the 80S peak for better resolution of the halfmer polyribosome peaks, which are indicated by arrows. The 40S peak cannot be resolved at this sensitivity setting. (C, D) Polysome profiles of mutants of the indicated genotypes with the <em>RPL7A</em> (red), or <em>RPL7B</em> (blue), CRI at the <em>RPL7A</em> locus. In each panel, traces of three strains were overlaid by aligning the lowest point of each trace between the 80S monosome peak and disome peak. The sensitivity setting of the UV monitor was 0.2. (E) A graph of relative levels of 60S ribosomal subunits, 80S monosomes, disomes, and polysomes in three biological replicates of each strain of the indicated genotype. To normalize for differences in lysis efficiency and lysate amount, the heights of 40S, 60S, 80S, disome, and highest polysome peaks in polysome profiles were measured, and ratios indicated on the <em>y</em>-axis were calculated. Statistically significant differences in the indicated ratios between strains of different genotypes, as determined by a two-way ANOVA test, are indicated with asterisks. *<em>P</em> &lt; 0.05, **<em>P</em> &lt; 0.01, ***<em>P</em> &lt; 0.001, ****<em>P</em> &lt; 0.0001.</p><div class="sb-div caption-clear"></div></div></div><p id="p-35">It was not possible to determine whether the 60S to 40S ribosomal subunit ratio differed in mutants with the <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> <em>vs.</em> <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> CRI at the <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> locus, since the level of 40S subunits was so low in both strains (<a id="xref-fig-3-10" class="xref-fig" href="#F3">Figure 3A</a>). Nonetheless, the polysome data from mutants expressing <em><a href="http://www.yeastgenome.org/locus/S000004947/overview">RPL7</a></em> CRI from the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> locus suggests that a reduced ratio of 60S to 40S ribosomal subunits is strongly correlated with differences in the growth rate of <em><a href="http://www.yeastgenome.org/locus/S000004947/overview">rpl7</a></em> deletion and chimeragenic strains.</p><p id="p-36">To determine whether doubling time and 60S ribosomal subunit level differences among strains expressing only one native or chimeric <em><a href="http://www.yeastgenome.org/locus/S000004947/overview">RPL7</a></em> allele were reflected in differences in <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a> levels, western blot analysis of twofold serial dilutions of cell lysate with a human anti-L7 antibody that recognizes the identical epitope in <a href="http://www.yeastgenome.org/locus/S000003044/overview">Rpl7a</a> and <a href="http://www.yeastgenome.org/locus/S000006119/overview">Rpl7b</a> was performed. <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a> levels relative to level of a control protein, <a href="http://www.yeastgenome.org/locus/S000001855/overview">Act1</a> were measured by densitometry using bands at nonsaturating intensities. The <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a>/<a href="http://www.yeastgenome.org/locus/S000001855/overview">Act1</a> ratio was &gt;2-fold higher when the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> CRI was expressed from the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> locus <em>vs.</em> the <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> locus (0.045 <em>vs.</em> 0.020, respectively), or when the <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> CRI was expressed from the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> locus <em>vs.</em> the <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> locus (0.029 <em>vs.</em> 0.012, respectively) (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>). In addition, there was a modestly higher level of <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a> when the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> CRI <em>vs.</em> the <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> CRI was expressed from either the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> locus (0.045 <em>vs.</em> 0.029) or the <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> locus (0.020 <em>vs.</em> 0.012). Therefore, differences in growth rates and relative 60S ribosomal subunit levels among strains expressing a single native or chimeric <em><a href="http://www.yeastgenome.org/locus/S000004947/overview">RPL7</a></em> allele may result, at least partially, from different <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a> levels. Together these findings indicate that <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a> <a href="http://www.yeastgenome.org/locus/S000006119/overview">rpl7b</a>∆</em> &gt; <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆</em>::<em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a>∆</em> &gt; <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆ <a href="http://www.yeastgenome.org/locus/S000006119/overview">rpl7b</a>∆</em>::<em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a> &gt; <a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆ <a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> for rate of growth, ratio of 60S to 40S ribosomal subunits, and <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a> level.</p></div><div id="sec-15" class="subsection"><h3>Negative correlation between tunicamycin resistance and Rpl7 and snR39/59 expression</h3><p id="p-37">Deletion of <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em>, but not <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em>, results in resistance to tunicamycin, an inhibitor of protein N-glycosylation in the endoplasmic reticulum (ER), and inducer of the ER stress response (<a id="xref-ref-74-4" class="xref-bibr" href="#ref-74">Steffen <em>et al.</em> 2012</a>). We asked whether resistance to tunicamycin is a function of expressing the <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> CRI or simply correlated with <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a> levels by plating serial dilutions of <em><a href="http://www.yeastgenome.org/locus/S000004947/overview">rpl7</a></em> deletion and chimeragenic strains on YPD agar, with or without 5 µg/ml tunicamycin (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2C</a>). Mutants with either <em><a href="http://www.yeastgenome.org/locus/S000004947/overview">RPL7</a></em> CRI at the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> locus were sensitive to tunicamycin, while mutants with either <em><a href="http://www.yeastgenome.org/locus/S000004947/overview">RPL7</a></em> CRI at the <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> locus were resistant (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2C</a>). There was a small decrease in tunicamycin-resistance in the mutant with the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> <em>vs.</em> the <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> CRI at the <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> locus (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2C</a>), which parallels the slightly faster growth and higher <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a> level in the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆ <a href="http://www.yeastgenome.org/locus/S000006119/overview">rpl7b</a>∆:<a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> mutant (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2, A and B</a>). Together, the data suggest that tunicamycin resistance is correlated with <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a> and <a href="http://www.yeastgenome.org/locus/S000007302/overview">snR39</a>/<a href="http://www.yeastgenome.org/locus/S000006450/overview">59</a> levels, and not a specific function of <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> isoforms.</p></div><div id="sec-16" class="subsection"><h3>ASH1 mRNA localization in mutants with an RPL7A or rpl7a∆::RPL7B allele</h3><p id="p-38">The <em><a href="http://www.yeastgenome.org/locus/S000001668/overview">ASH1</a></em> mRNA transcript localizes to a focus at the tip of the budding daughter cell during bud growth, and migrates toward the bud neck as the cell progresses through M-phase (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Beach <em>et al.</em> 1999</a>). Deletion of <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> but not <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> significantly reduces the percentage of budded cells with bud-localized <em><a href="http://www.yeastgenome.org/locus/S000001668/overview">ASH1</a></em> mRNA foci (<a id="xref-ref-38-8" class="xref-bibr" href="#ref-38">Komili <em>et al.</em> 2007</a>). To determine whether the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> isoforms or locus contribute to <em><a href="http://www.yeastgenome.org/locus/S000001668/overview">ASH1</a></em> mRNA localization to the bud, an mRNA consisting of the <em><a href="http://www.yeastgenome.org/locus/S000001668/overview">ASH1</a></em> ORF and 12 MS2 stem-loops [<em><a href="http://www.yeastgenome.org/locus/S000001668/overview">ASH1</a>-MS2L(x12)</em>] was expressed in <em><a href="http://www.yeastgenome.org/locus/S000004947/overview">rpl7</a></em> chimeragenic and deletion strains. Coexpression of an MS2 coat protein, which binds to the MS2L(x12) RNA sequences, fused to three GFP moieties [MS2-CP-GFP(x3)] was used to detect <em><a href="http://www.yeastgenome.org/locus/S000001668/overview">ASH1</a>-MS2L(x12)</em> RNA by fluorescent microscopy of live cells. As expected, a lower percentage of budded <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆ <a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> cells had a bud-localized <em><a href="http://www.yeastgenome.org/locus/S000001668/overview">ASH1</a>-MS2L(x12)</em> mRNA focus compared to budded <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a> <a href="http://www.yeastgenome.org/locus/S000006119/overview">rpl7b</a>∆</em> or wild-type cells (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). An equivalently low level of bud-localized <em><a href="http://www.yeastgenome.org/locus/S000001668/overview">ASH1</a>-MS2L(x12)</em> mRNA was seen when <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> was expressed from the <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> locus (<em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆ <a href="http://www.yeastgenome.org/locus/S000006119/overview">rpl7b</a>∆</em>::<em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em>, <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>), indicating that <em><a href="http://www.yeastgenome.org/locus/S000001668/overview">ASH1</a></em> mRNA localization is inefficient when either <em><a href="http://www.yeastgenome.org/locus/S000004947/overview">RPL7</a></em> CRI is expressed from the <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> locus. In contrast, expression of either CRI from the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> locus resulted in the same percentage of budded cells exhibiting bud-localized <em><a href="http://www.yeastgenome.org/locus/S000001668/overview">ASH1</a>-MS2L(x12)</em> mRNA as in the wild-type strain. (Compare <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a> <a href="http://www.yeastgenome.org/locus/S000006119/overview">rpl7b</a>∆</em>, <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆</em>::<em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a>∆</em>, and <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a> <a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em>, <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>.) Together, these data suggest that either the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> or <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> CRI, when expressed from the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> locus results in sufficient levels of <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a> and <a href="http://www.yeastgenome.org/locus/S000007302/overview">snR39</a>/<a href="http://www.yeastgenome.org/locus/S000006450/overview">59</a> to support wild-type levels of <em><a href="http://www.yeastgenome.org/locus/S000001668/overview">ASH1</a></em> mRNA localization.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/591/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="An epitope-tagged RPL7B allele suppresses the growth and Ty1 retromobility defect of an rpl7a&#x2206; mutant. (A) Western blot analysis of Rpl7a-9xMyc or Rpl7b-9xMyc in whole cell lysate from strains, without or with deletions of RPL7B or RPL7A, respectively. Rpl7a-9xMyc and Rpl7b-9xMyc were detected with anti-c-Myc antibody, and &#x3B1;-Tubulin was detected with an anti-&#x3B1;-Tubulin antibody as a loading control. (B) The rate of cell doubling of three biological replicates of each strain of the indicated genotype grown in YPD broth at 20&#xB0; to an OD600 of 0.4&#x2013;0.6. Error bars represent SD. (C) The fold-change in retrotransposition rate is the retrotransposition rate of a chromosomal Ty1his3AI element in each strain of the indicated genotype relative to that in the RPL7A RPL7B strain. Error bars represent SE." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1111785762" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;An epitope-tagged &lt;em&gt;RPL7B&lt;/em&gt; allele suppresses the growth and Ty1 retromobility defect of an &lt;em&gt;rpl7a&#x2206;&lt;/em&gt; mutant. (A) Western blot analysis of Rpl7a-9xMyc or Rpl7b-9xMyc in whole cell lysate from strains, without or with deletions of &lt;em&gt;RPL7B&lt;/em&gt; or &lt;em&gt;RPL7A&lt;/em&gt;, respectively. Rpl7a-9xMyc and Rpl7b-9xMyc were detected with anti-c-Myc antibody, and &#x3B1;-Tubulin was detected with an anti-&#x3B1;-Tubulin antibody as a loading control. (B) The rate of cell doubling of three biological replicates of each strain of the indicated genotype grown in YPD broth at 20&#xB0; to an OD&lt;sub&gt;600&lt;/sub&gt; of 0.4&#x2013;0.6. Error bars represent SD. (C) The fold-change in retrotransposition rate is the retrotransposition rate of a chromosomal Ty1&lt;em&gt;his3AI&lt;/em&gt; element in each strain of the indicated genotype relative to that in the &lt;em&gt;RPL7A RPL7B&lt;/em&gt; strain. Error bars represent SE.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/591/F4.medium.gif" width="196" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/2/591/F4.medium.gif" width="196" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/591/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/591/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470506" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-39" class="first-child">An epitope-tagged <em>RPL7B</em> allele suppresses the growth and Ty1 retromobility defect of an <em>rpl7a∆</em> mutant. (A) Western blot analysis of Rpl7a-9xMyc or Rpl7b-9xMyc in whole cell lysate from strains, without or with deletions of <em>RPL7B</em> or <em>RPL7A</em>, respectively. Rpl7a-9xMyc and Rpl7b-9xMyc were detected with anti-c-Myc antibody, and α-Tubulin was detected with an anti-α-Tubulin antibody as a loading control. (B) The rate of cell doubling of three biological replicates of each strain of the indicated genotype grown in YPD broth at 20° to an OD<sub>600</sub> of 0.4–0.6. Error bars represent SD. (C) The fold-change in retrotransposition rate is the retrotransposition rate of a chromosomal Ty1<em>his3AI</em> element in each strain of the indicated genotype relative to that in the <em>RPL7A RPL7B</em> strain. Error bars represent SE.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h3>Ty1 retromobility is a function of robust Rpl7 and Rpl31 expression</h3><p id="p-40">Ty1 retromobility is strongly reduced in the absence of <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em>, or a specific paralog of a few other RP gene pairs (<a id="xref-ref-61-5" class="xref-bibr" href="#ref-61">Risler <em>et al.</em> 2012</a>). Retrotransposons have the potential to damage the host genome when their rate of mobility or location of insertion into the genome is not tightly controlled; therefore, we considered the possibility that differences between <a href="http://www.yeastgenome.org/locus/S000006119/overview">Rpl7b</a>/<a href="http://www.yeastgenome.org/locus/S000006450/overview">snR59</a> and <a href="http://www.yeastgenome.org/locus/S000003044/overview">Rpl7a</a>/<a href="http://www.yeastgenome.org/locus/S000007302/overview">snR39</a> isoforms might confer a selective advantage on the cell by repressing Ty1 retromobility while maintaining ribosome-related functions. To determine whether Ty1 retromobility is a paralog-specific function of <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em>, we used a chromosomal Ty1<em>his3AI</em> element to measure retromobility in <em><a href="http://www.yeastgenome.org/locus/S000004947/overview">RPL7</a></em> deletion strains. The <em>his3AI</em> retrotranscript indicator gene contains an antisense intron interrupting the <em><a href="http://www.yeastgenome.org/locus/S000005728/overview">HIS3</a></em> coding sequence. The <em>his3AI</em> reporter is located in the 3′ UTR of the chromosomal Ty1 element in the opposite transcriptional orientation; ergo, it can be spliced from the Ty1 transcript. Cells in which Ty1<em>his3AI</em> RNA is spliced and reverse transcribed, and the resulting Ty1<em><a href="http://www.yeastgenome.org/locus/S000005728/overview">HIS3</a></em> cDNA is inserted into the genome by integration or recombination, are detected as His<sup>+</sup> prototrophs. The rate of His<sup>+</sup> prototroph formation per cell per generation is directly proportional to the rate of Ty1<em>his3AI</em> retromobility (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Curcio and Garfinkel 1991</a>). At 20°, a permissive temperature for Ty1 retrotransposition, Ty1<em>his3AI</em> retromobility in the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆ <a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> mutant was &lt;8% of that in the wild-type strain, but in an <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a> <a href="http://www.yeastgenome.org/locus/S000006119/overview">rpl7b</a>∆</em> mutant, the rate of retromobility was equivalent to that in a wild-type strain (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Thus, Ty1 retromobility is a paralog-specific function of <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470217/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470217/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Effect of <em>RPL7</em> and <em>RPL31</em> alleles on Ty1<em>his3AI</em> retromobility</span><div class="sb-div caption-clear"></div></div></div><p id="p-42">The rate of Ty1<em>his3AI</em> retromobility in the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆</em>::<em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a>∆</em> mutant was similar to that of the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a> <a href="http://www.yeastgenome.org/locus/S000006119/overview">rpl7b</a>∆</em> strain, suggesting that <a href="http://www.yeastgenome.org/locus/S000003044/overview">Rpl7a</a>/<a href="http://www.yeastgenome.org/locus/S000007302/overview">snR39</a> isoforms do not play a specialized role in Ty1 retrotransposition. This conclusion is also supported by the observation that homogenic mutants expressing either the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> or the <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> CRI at both loci had high levels Ty1 retromobility that were comparable to each other, and similar to the wild-type strain. In contrast, the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆ <a href="http://www.yeastgenome.org/locus/S000006119/overview">rpl7b</a>∆</em>::<em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> mutant had a strongly reduced retromobility rate similar to that of the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆ <a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> mutant (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Thus, expression of either <a href="http://www.yeastgenome.org/locus/S000003044/overview">Rpl7a</a>/<a href="http://www.yeastgenome.org/locus/S000007302/overview">snR39</a> or <a href="http://www.yeastgenome.org/locus/S000006119/overview">Rpl7b</a>/<a href="http://www.yeastgenome.org/locus/S000006450/overview">snR59</a> isoforms from the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> locus, but not the <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> locus, resulted in efficient Ty1 retromobility.</p><p id="p-43">To further test the idea that the frequency of Ty1 retromobility is correlated with the level of <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a>, we capitalized on an interesting observation: when <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> is tagged with a <em>9xMYC</em> epitope at the 3′ end of the CRI, the level of <a href="http://www.yeastgenome.org/locus/S000006119/overview">Rpl7b</a>:9xMyc increased upon deletion of <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>). Addition of the <em>9xMYC</em> tag to <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> did not change the doubling time of cells with an <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> allele, but it substantially decreased the doubling time of an <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆</em> mutant (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a>). Thus, in contrast to the ∼2-fold difference in doubling time in an <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆ <a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> mutant relative to the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a> <a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> strain (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2A</a> and <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5B</a>), there was a very slight difference in doubling time between the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆ <a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a>:9xMYC</em> mutant and <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a> <a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a>:9xMYC</em> strain (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5B</a>), consistent with the compensatory increase in <a href="http://www.yeastgenome.org/locus/S000006119/overview">Rpl7b</a>:9xMyc protein when <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> was deleted. Also, there was no significant difference in the frequency of Ty1<em>his3AI</em> retromobility between the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆ <a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a>:9xMYC</em> mutant and <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a> <a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a>:9xMYC</em> strain (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5C</a>), which contrasts starkly with the &gt;10-fold difference in Ty1<em>his3AI</em> retromobility between <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆ <a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a> <a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> strains (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). The data suggest that increased expression of <a href="http://www.yeastgenome.org/locus/S000006119/overview">Rpl7b</a>:9xMyc upon deletion of <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> suppresses the retromobility defect that normally results from deleting <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em>, and strongly support the idea that the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> paralog is required for retromobility because of the level of <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a> expression rather than the expression of the <a href="http://www.yeastgenome.org/locus/S000003044/overview">Rpl7a</a> or <a href="http://www.yeastgenome.org/locus/S000007302/overview">snR39</a> isoform.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/591/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="ASH1 mRNA localization requires a high dosage of Rpl7 and snR39/59. Expression of ASH1-MS2L(x12) RNA was induced in strains of the indicated genotype cotransformed with plasmid pGAL1PASH1-MS2L(x12), and plasmid pMS2-CP-GFP(x3), by growth in galactose-containing medium. The ASH1 ORF fragment in ASH1-MS2L(x12) RNA contains three sequence motifs&#x2014;E1, E2 and E3&#x2014;that target the mRNA for localization to the bud tip (Beach and Bloom 2001). Cells were incubated with Hoechst 33258 dye, which stains the yeast cell wall (Hernandez et al. 1988). Cells from two or three biological replicates were analyzed for a total of three independent experiments. Numerical values represent the average percent of budded cells with bud-localized ASH1-MS2L(x12) mRNA foci; n represents the total number of budded cells counted. White arrows point to ASH1-MS2L(x12) RNA foci in budded cells." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1111785762" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;ASH1&lt;/em&gt; mRNA localization requires a high dosage of Rpl7 and snR39/59. Expression of &lt;em&gt;ASH1&lt;/em&gt;-MS2L(x12) RNA was induced in strains of the indicated genotype cotransformed with plasmid pGAL1&lt;sub&gt;P&lt;/sub&gt;&lt;em&gt;ASH1&lt;/em&gt;-MS2L(x12), and plasmid pMS2-CP-GFP(x3), by growth in galactose-containing medium. The &lt;em&gt;ASH1&lt;/em&gt; ORF fragment in &lt;em&gt;ASH1&lt;/em&gt;-MS2L(x12) RNA contains three sequence motifs&#x2014;E1, E2 and E3&#x2014;that target the mRNA for localization to the bud tip (Beach and Bloom 2001). Cells were incubated with Hoechst 33258 dye, which stains the yeast cell wall (Hernandez &lt;em&gt;et al.&lt;/em&gt; 1988). Cells from two or three biological replicates were analyzed for a total of three independent experiments. Numerical values represent the average percent of budded cells with bud-localized &lt;em&gt;ASH1&lt;/em&gt;-MS2L(x12) mRNA foci; &lt;em&gt;n&lt;/em&gt; represents the total number of budded cells counted. White arrows point to &lt;em&gt;ASH1&lt;/em&gt;-MS2L(x12) RNA foci in budded cells.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/591/F5.medium.gif" width="303" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/2/591/F5.medium.gif" width="303" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/591/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/591/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470344" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-44" class="first-child"><em>ASH1</em> mRNA localization requires a high dosage of Rpl7 and snR39/59. Expression of <em>ASH1</em>-MS2L(x12) RNA was induced in strains of the indicated genotype cotransformed with plasmid pGAL1<sub>P</sub><em>ASH1</em>-MS2L(x12), and plasmid pMS2-CP-GFP(x3), by growth in galactose-containing medium. The <em>ASH1</em> ORF fragment in <em>ASH1</em>-MS2L(x12) RNA contains three sequence motifs—E1, E2 and E3—that target the mRNA for localization to the bud tip (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Beach and Bloom 2001</a>). Cells were incubated with Hoechst 33258 dye, which stains the yeast cell wall (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Hernandez <em>et al.</em> 1988</a>). Cells from two or three biological replicates were analyzed for a total of three independent experiments. Numerical values represent the average percent of budded cells with bud-localized <em>ASH1</em>-MS2L(x12) mRNA foci; <em>n</em> represents the total number of budded cells counted. White arrows point to <em>ASH1</em>-MS2L(x12) RNA foci in budded cells.</p><div class="sb-div caption-clear"></div></div></div><p id="p-45">The <em><a href="http://www.yeastgenome.org/locus/S000002233/overview">RPL31A</a></em> gene, but not <em><a href="http://www.yeastgenome.org/locus/S000004398/overview">RPL31B</a></em>, was also identified as essential for efficient Ty1 retromobility (<a id="xref-ref-61-6" class="xref-bibr" href="#ref-61">Risler <em>et al.</em><span class="sc"> 2012</span></a>). <em><a href="http://www.yeastgenome.org/locus/S000002233/overview">RPL31A</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000004398/overview">RPL31B</a></em> ORFs are not interrupted by introns, and encode isoforms that differ at only one amino acid residue. The rate of Ty1<em>his3AI</em> retromobility in an <em><a href="http://www.yeastgenome.org/locus/S000002233/overview">rpl31a</a>∆ <a href="http://www.yeastgenome.org/locus/S000004398/overview">RPL31B</a></em> mutant was 5% of that in a wild-type strain (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). Ty1<em>his3AI</em> retromobility in a chimeragenic strain expressing the <em><a href="http://www.yeastgenome.org/locus/S000004398/overview">RPL31B</a></em> ORF from the <em><a href="http://www.yeastgenome.org/locus/S000002233/overview">RPL31A</a></em> locus was measured to determine whether Rpl31 dosage or the <a href="http://www.yeastgenome.org/locus/S000002233/overview">Rpl31a</a> isoform is required for Ty1 retromobility. The <em><a href="http://www.yeastgenome.org/locus/S000002233/overview">rpl31a</a>∆</em>::<em><a href="http://www.yeastgenome.org/locus/S000004398/overview">RPL31B</a>∆</em> strain had a similar rate of Ty1 retromobility to that of the wild-type strain, and a 23-fold higher rate than the <em><a href="http://www.yeastgenome.org/locus/S000002233/overview">rpl31a</a>∆ <a href="http://www.yeastgenome.org/locus/S000004398/overview">RPL31B</a></em> mutant (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>), indicating that expression of either Rpl31 isoform from the <em><a href="http://www.yeastgenome.org/locus/S000002233/overview">RPL31A</a></em> locus is sufficient for Ty1 retromobility. Since deletion of <em><a href="http://www.yeastgenome.org/locus/S000002233/overview">RPL31A</a></em> but not <em><a href="http://www.yeastgenome.org/locus/S000004398/overview">RPL31B</a></em> substantially retards cell doubling (<a id="xref-ref-74-5" class="xref-bibr" href="#ref-74">Steffen <em>et al.</em> 2012</a>), a higher dosage of <a href="http://www.yeastgenome.org/locus/S000004398/overview">Rpl31b</a> when expressed from the <em><a href="http://www.yeastgenome.org/locus/S000002233/overview">RPL31A</a></em> <em>vs.</em> <em><a href="http://www.yeastgenome.org/locus/S000004398/overview">RPL31B</a></em> locus is the most plausible cause of more efficient Ty1 retromobility.</p></div><div id="sec-18" class="subsection"><h3>Minor effects of Rpl7 and snR39/59 levels on Ty1 RNA translation</h3><p id="p-46">Depletion of 60S ribosomal subunits in mutants with low levels of <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a> could inhibit Ty1 retromobility by selectively reducing the rate of Ty1 RNA translation. However, Gag levels were not significantly different among chimeragenic and deletion mutants harboring one <em><a href="http://www.yeastgenome.org/locus/S000004947/overview">RPL7</a></em> allele (<a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2B</a>). The <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆ <a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> mutant had 59% as much Gag as the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a> <a href="http://www.yeastgenome.org/locus/S000006119/overview">rpl7b</a>∆</em> mutant, despite substantial differences in doubling time and 60S subunit levels between these strains (<a id="xref-fig-2-10" class="xref-fig" href="#F2">Figure 2, A and B</a>). As an independent method of measuring relative Gag levels in the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆ <a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> mutant, we quantified Gag:GFP produced from a chromosomal Ty1(<em>GAG:GFP</em>) element (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>, top). The <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆ <a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> mutant had ∼75% of the GFP activity in the wild-type strain or <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a> <a href="http://www.yeastgenome.org/locus/S000006119/overview">rpl7b</a>∆</em> mutant (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6A</a>, bottom). Thus, even the mutant with the strongest depletion of <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a> and 60S ribosomal subunits has only a minor defect in Gag accumulation, indicating that inhibition of Ty1 RNA translation may not be the major cause of the retromobility defect in the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆ <a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> mutant.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/591/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Diminished Ty1 retromobility in the rpl7a&#x2206; RPL7B mutant is not a result of limiting Gag levels. (A) A schematic of the chromosomal Ty1(GAG:GFP)-3566 element is shown (top). The GFP ORF is fused at the C-terminal end of the GAG ORF after codon 401 in a chromosomal Ty1 element (Scholes et al. 2003). The mean fluorescence in 10,000&#x2013;20,000 cells of each genotype, corrected for autofluorescence by subtracting the mean fluorescence in each congenic strain lacking Ty1(GAG:GFP)-3566 from the mean fluorescence in each strain harboring Ty1(GAG:GFP)-3566, was determined, and the value relative to that in the RPL7A RPL7B strain is reported (bottom panel). Error bars represent the SD of mean fluorescence calculated from two independent isolates of each genotype. (B) Schematic of plasmid pGTy1(syn-GAG), which carries a modified Ty1 element that lacks the 3&#x2032; LTR, and has numerous silent nucleotide substitutions in the GAG ORF. Ty1(syn-GAG) RNA is defective for retromobility in cis, but encodes wild-type Gag and Gag-Pol proteins required in trans for retromobility of the chromosomal Ty1his3AI element, which is diagrammed below plasmid pGTy1(syn-GAG). (C) Western blot analysis of RPL7A RPL7B, rpl7a&#x2206; RPL7B, and tif4631&#x2206; TIF4632 strains harboring either the vector pRS415 or plasmid pGTy1(syn-GAG) and grown in galactose medium at 20&#xB0;. The membrane was probed with anti-VLP antisera to detect Ty1 Gag (top), or with an anti-&#x3B1;-Tubulin antibody to detect &#x3B1;-Tubulin (bottom) as a loading control. (D) Effect of pGTy1(syn-GAG) expression on Ty1his3AI retromobility in different genetic backgrounds. RPL7A RPL7B, rpl7a&#x2206; RPL7B and tif4631&#x2206; TIF4632 strains harboring a chromosomal Ty1his3AI element, and the vector (pRS415), or plasmid pGTy1(syn-GAG) grown in galactose-containing medium at 20&#xB0; to induce expression of Ty1 proteins, and the Ty1 retromobility frequency was determined by measuring the frequency of His+ prototrophs formed. Error represents the SD of four biological replicates." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1111785762" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Diminished Ty1 retromobility in the &lt;em&gt;rpl7a&#x2206; RPL7B&lt;/em&gt; mutant is not a result of limiting Gag levels. (A) A schematic of the chromosomal Ty1(&lt;em&gt;GAG:GFP&lt;/em&gt;)-3566 element is shown (top). The &lt;em&gt;GFP&lt;/em&gt; ORF is fused at the C-terminal end of the &lt;em&gt;GAG&lt;/em&gt; ORF after codon 401 in a chromosomal Ty1 element (Scholes &lt;em&gt;et al.&lt;/em&gt; 2003). The mean fluorescence in 10,000&#x2013;20,000 cells of each genotype, corrected for autofluorescence by subtracting the mean fluorescence in each congenic strain lacking Ty1(&lt;em&gt;GAG:GFP&lt;/em&gt;)-3566 from the mean fluorescence in each strain harboring Ty1(&lt;em&gt;GAG:GFP&lt;/em&gt;)-3566, was determined, and the value relative to that in the &lt;em&gt;RPL7A RPL7B&lt;/em&gt; strain is reported (bottom panel). Error bars represent the SD of mean fluorescence calculated from two independent isolates of each genotype. (B) Schematic of plasmid pGTy1(&lt;em&gt;syn-GAG&lt;/em&gt;), which carries a modified Ty1 element that lacks the 3&#x2032; LTR, and has numerous silent nucleotide substitutions in the &lt;em&gt;GAG&lt;/em&gt; ORF. Ty1(&lt;em&gt;syn-GAG&lt;/em&gt;) RNA is defective for retromobility in &lt;em&gt;cis&lt;/em&gt;, but encodes wild-type Gag and Gag-Pol proteins required in &lt;em&gt;trans&lt;/em&gt; for retromobility of the chromosomal Ty1&lt;em&gt;his3AI&lt;/em&gt; element, which is diagrammed below plasmid pGTy1(&lt;em&gt;syn-GAG&lt;/em&gt;). (C) Western blot analysis of &lt;em&gt;RPL7A RPL7B&lt;/em&gt;, &lt;em&gt;rpl7a&#x2206; RPL7B&lt;/em&gt;, and &lt;em&gt;tif4631&#x2206; TIF4632&lt;/em&gt; strains harboring either the vector pRS415 or plasmid pGTy1(&lt;em&gt;syn-GAG&lt;/em&gt;) and grown in galactose medium at 20&#xB0;. The membrane was probed with anti-VLP antisera to detect Ty1 Gag (top), or with an anti-&#x3B1;-Tubulin antibody to detect &#x3B1;-Tubulin (bottom) as a loading control. (D) Effect of pGTy1(&lt;em&gt;syn-GAG&lt;/em&gt;) expression on Ty1&lt;em&gt;his3AI&lt;/em&gt; retromobility in different genetic backgrounds. &lt;em&gt;RPL7A RPL7B&lt;/em&gt;, &lt;em&gt;rpl7a&#x2206; RPL7B&lt;/em&gt; and &lt;em&gt;tif4631&#x2206; TIF4632&lt;/em&gt; strains harboring a chromosomal Ty1&lt;em&gt;his3AI&lt;/em&gt; element, and the vector (pRS415), or plasmid pGTy1(&lt;em&gt;syn-GAG&lt;/em&gt;) grown in galactose-containing medium at 20&#xB0; to induce expression of Ty1 proteins, and the Ty1 retromobility frequency was determined by measuring the frequency of His&lt;sup&gt;+&lt;/sup&gt; prototrophs formed. Error represents the SD of four biological replicates.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/591/F6.medium.gif" width="208" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/7/2/591/F6.medium.gif" width="208" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/591/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/591/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470279" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-47" class="first-child">Diminished Ty1 retromobility in the <em>rpl7a∆ RPL7B</em> mutant is not a result of limiting Gag levels. (A) A schematic of the chromosomal Ty1(<em>GAG:GFP</em>)-3566 element is shown (top). The <em>GFP</em> ORF is fused at the C-terminal end of the <em>GAG</em> ORF after codon 401 in a chromosomal Ty1 element (<a id="xref-ref-67-3" class="xref-bibr" href="#ref-67">Scholes <em>et al.</em> 2003</a>). The mean fluorescence in 10,000–20,000 cells of each genotype, corrected for autofluorescence by subtracting the mean fluorescence in each congenic strain lacking Ty1(<em>GAG:GFP</em>)-3566 from the mean fluorescence in each strain harboring Ty1(<em>GAG:GFP</em>)-3566, was determined, and the value relative to that in the <em>RPL7A RPL7B</em> strain is reported (bottom panel). Error bars represent the SD of mean fluorescence calculated from two independent isolates of each genotype. (B) Schematic of plasmid pGTy1(<em>syn-GAG</em>), which carries a modified Ty1 element that lacks the 3′ LTR, and has numerous silent nucleotide substitutions in the <em>GAG</em> ORF. Ty1(<em>syn-GAG</em>) RNA is defective for retromobility in <em>cis</em>, but encodes wild-type Gag and Gag-Pol proteins required in <em>trans</em> for retromobility of the chromosomal Ty1<em>his3AI</em> element, which is diagrammed below plasmid pGTy1(<em>syn-GAG</em>). (C) Western blot analysis of <em>RPL7A RPL7B</em>, <em>rpl7a∆ RPL7B</em>, and <em>tif4631∆ TIF4632</em> strains harboring either the vector pRS415 or plasmid pGTy1(<em>syn-GAG</em>) and grown in galactose medium at 20°. The membrane was probed with anti-VLP antisera to detect Ty1 Gag (top), or with an anti-α-Tubulin antibody to detect α-Tubulin (bottom) as a loading control. (D) Effect of pGTy1(<em>syn-GAG</em>) expression on Ty1<em>his3AI</em> retromobility in different genetic backgrounds. <em>RPL7A RPL7B</em>, <em>rpl7a∆ RPL7B</em> and <em>tif4631∆ TIF4632</em> strains harboring a chromosomal Ty1<em>his3AI</em> element, and the vector (pRS415), or plasmid pGTy1(<em>syn-GAG</em>) grown in galactose-containing medium at 20° to induce expression of Ty1 proteins, and the Ty1 retromobility frequency was determined by measuring the frequency of His<sup>+</sup> prototrophs formed. Error represents the SD of four biological replicates.</p><div class="sb-div caption-clear"></div></div></div><p id="p-48">To further determine whether the retromobility defect of the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆ <a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> mutant is a direct result of limiting levels of Ty1 proteins, the ability of Ty1 proteins supplied in <em>trans</em> to suppress the retromobility defect was tested. Ty1 proteins were expressed from a <em><a href="http://www.yeastgenome.org/locus/S000000224/overview">GAL1</a></em>-driven modified Ty1 element whose transcript cannot be used as a template for retrotransposition. The plasmid-borne pGTy1(<em>syn-GAG</em>) element lacks a 3′ LTR, and has multiple nucleotide substitutions in the 5′ UTR and 5′ terminus of the <em>GAG</em> ORF that are predicted to disrupt RNA secondary structures and other <em>cis</em>-acting sequences required for retrotransposition (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Curcio <em>et al.</em> 2015</a>), but not the amino acid sequence of Gag (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6B</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035931/-/DC1/FigureS1.tif">Figure S1</a>). As a positive control for suppression of a Ty1 RNA translation defect, pGTy1(<em>syn-GAG</em>) was also expressed in a <em><a href="http://www.yeastgenome.org/locus/S000003394/overview">tif4631</a></em>∆ strain. In the absence of <em><a href="http://www.yeastgenome.org/locus/S000003394/overview">TIF4631</a></em>, which encodes the translation initiation factor eIF4G1, the amount of Ty1 Gag is substantially reduced (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6C</a>, <em><a href="http://www.yeastgenome.org/locus/S000003394/overview">tif4631</a>∆ <a href="http://www.yeastgenome.org/locus/S000003017/overview">TIF4632</a></em> strain with vector), presumably because eIF4G1 is required to initiate translation from the highly structured 5′ UTR of Ty1 RNA (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Purzycka <em>et al.</em><span class="sc"> 2013</span></a>). In the <em><a href="http://www.yeastgenome.org/locus/S000003394/overview">tif4631</a></em>∆ strain, galactose-mediated induction of pGTy1(<em>syn-GAG</em>) increased the amount of Gag (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6C</a>, compare <em><a href="http://www.yeastgenome.org/locus/S000003394/overview">tif4631</a>∆ <a href="http://www.yeastgenome.org/locus/S000003017/overview">TIF4632</a></em> strain with pGTy1(<em>syn-Gag</em>) plasmid to that with vector). Accordingly, retromobility of the chromosomal Ty1<em>his3AI</em> element increased to 75% of that of the wild-type strain with a vector only (<a id="xref-fig-6-6" class="xref-fig" href="#F6">Figure 6D</a>). In contrast, expression of pGTy1(<em>syn-GAG</em>) in the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆ <a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> strain increased Gag levels [<a id="xref-fig-6-7" class="xref-fig" href="#F6">Figure 6C</a>, compare <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆ <a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> strain with pGTy1(<em>syn-GAG</em>) to that with vector], but it only increased Ty1<em>his3AI</em> retromobility to 4.2% of that in the wild-type strain with a vector only (<a id="xref-fig-6-8" class="xref-fig" href="#F6">Figure 6D</a>). Together, the data suggest that Gag is not the only limiting factor for retrotransposition in the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆ <a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> mutant.</p></div><div id="sec-19" class="subsection"><h3>High levels of Rpl7 and 60S ribosomal subunits are required for Ty1 RNA localization</h3><p id="p-49">When cells are grown at the permissive temperature for Ty1 retrotransposition, Ty1 RNA is cotranslationally localized to a discrete cytoplasmic focus known as the retrosome, the presumptive site of VLP assembly (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Malagon and Jensen 2008</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Checkley <em>et al.</em> 2010</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Sandmeyer and Clemens 2010</a>; <a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">Doh <em>et al.</em> 2014</a>). Ty1 RNA does not localize in retrosomes in the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆ <a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> mutant (<a id="xref-ref-24-5" class="xref-bibr" href="#ref-24">Doh <em>et al.</em> 2014</a>). To determine whether this localization defect is specific to deleting <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em>, fluorescent <em>in situ</em> hybridization (FISH) of Ty1 RNA was performed in strains bearing a deletion of each <em><a href="http://www.yeastgenome.org/locus/S000004947/overview">RPL7</a></em> allele (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7A</a>). Consistent with the previous finding, the percentage of cells with at least one Ty1 RNA focus in an <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆ <a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> strain was 0.4%; in contrast, 41.6% of <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a> <a href="http://www.yeastgenome.org/locus/S000006119/overview">rpl7b</a>∆</em> cells had a Ty1 RNA focus, which was similar to the level in the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a> <a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> strain. Therefore, some feature of <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em>, but not <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em>, is required for Ty1 RNA localization to retrosomes. In the mutant with the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> CRI expressed from the <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> locus, the percentage of cells with a Ty1 RNA focus was 2.8%, indicating that expression of either CRI from the <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> locus is not sufficient for Ty1 RNA localization. Notably, strains expressing the <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> CRI from the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> locus also had a very low percentage of cells with a Ty1 RNA focus (1.6%). Lack of Ty1 RNA localization in these three Rpl7-deficient mutants is not a consequence of reduced Ty1 RNA levels (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7B</a>). Efficient retrosome formation in the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a> <a href="http://www.yeastgenome.org/locus/S000006119/overview">rpl7b</a>∆</em> mutant, but not the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆</em>::<em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a>∆</em> mutant, raises the possibility that a very high level of <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a> or <a href="http://www.yeastgenome.org/locus/S000007302/overview">snR39</a>/<a href="http://www.yeastgenome.org/locus/S000006450/overview">59</a>—one which results in no growth defect or 60S ribosomal subunit deficiency—is necessary for Ty1 RNA to localize to retrosomes. Alternatively, retrosome formation could be a distinct function of the <a href="http://www.yeastgenome.org/locus/S000003044/overview">Rpl7a</a>/<a href="http://www.yeastgenome.org/locus/S000007302/overview">snR39</a> isoforms, which must be expressed from the more robust <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> locus to perform this function efficiently. To differentiate between these possibilities, retrosomes in the homogenic strain expressing the <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> CRI from both <em><a href="http://www.yeastgenome.org/locus/S000004947/overview">RPL7</a></em> loci were quantified. The percentage of cells with a Ty1 RNA retrosome increased 18-fold relative to the strain expressing the <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> CRI from the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> locus alone, (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7A</a>), indicating that <a href="http://www.yeastgenome.org/locus/S000006119/overview">Rpl7b</a>/<a href="http://www.yeastgenome.org/locus/S000006450/overview">snR59</a> supports retrosome formation when expressed at high levels, and, therefore, that Ty1 RNA localization is not a distinct function of the <a href="http://www.yeastgenome.org/locus/S000003044/overview">Rpl7a</a>/<a href="http://www.yeastgenome.org/locus/S000007302/overview">snR39</a> isoforms. The finding that Ty1 RNA efficiently localizes to retrosomes only in mutants with growth rates similar to the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a> <a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> strain suggests that Ty1 RNA localization is very sensitive to reductions in 60S ribosomal subunits.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/591/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A high dosage of Rpl7 or snR39/59 is required for localization of Ty1 RNA to retrosomes. (A) Fluorescent in situ hybridization was used to determine the number of cells harboring a cytoplasmic focus of Ty1 RNA, or retrosome. Ty1 RNA (red) was detected with a Cy3-labeled oligonucleotide probe. Nuclei (blue) were stained with DAPI. The percentage of cells with one or more Ty1 RNA retrosome is indicated. &#x201C;n&#x201D; is the number of cells scored for the presence of an RNA focus. (B) Quantitative real-time PCR of total RNA to determine the amount of Ty1 RNA relative to the amount of snR6 RNA in strains of the indicated genotypes. Error bars represent the SD of three biological replicates analyzed in triplicate." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1111785762" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;A high dosage of Rpl7 or snR39/59 is required for localization of Ty1 RNA to retrosomes. (A) Fluorescent &lt;em&gt;in situ&lt;/em&gt; hybridization was used to determine the number of cells harboring a cytoplasmic focus of Ty1 RNA, or retrosome. Ty1 RNA (red) was detected with a Cy3-labeled oligonucleotide probe. Nuclei (blue) were stained with DAPI. The percentage of cells with one or more Ty1 RNA retrosome is indicated. &#x201C;&lt;em&gt;n&lt;/em&gt;&#x201D; is the number of cells scored for the presence of an RNA focus. (B) Quantitative real-time PCR of total RNA to determine the amount of Ty1 RNA relative to the amount of &lt;em&gt;snR6&lt;/em&gt; RNA in strains of the indicated genotypes. Error bars represent the SD of three biological replicates analyzed in triplicate.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/591/F7.medium.gif" width="440" height="371"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/7/2/591/F7.medium.gif" width="440" height="371"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/591/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/591/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470251" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-50" class="first-child">A high dosage of Rpl7 or snR39/59 is required for localization of Ty1 RNA to retrosomes. (A) Fluorescent <em>in situ</em> hybridization was used to determine the number of cells harboring a cytoplasmic focus of Ty1 RNA, or retrosome. Ty1 RNA (red) was detected with a Cy3-labeled oligonucleotide probe. Nuclei (blue) were stained with DAPI. The percentage of cells with one or more Ty1 RNA retrosome is indicated. “<em>n</em>” is the number of cells scored for the presence of an RNA focus. (B) Quantitative real-time PCR of total RNA to determine the amount of Ty1 RNA relative to the amount of <em>snR6</em> RNA in strains of the indicated genotypes. Error bars represent the SD of three biological replicates analyzed in triplicate.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-20" class="subsection"><h3>Rpl7 or snR39/59 isoforms influence Ty1 cDNA accumulation in a dosage-dependent manner</h3><p id="p-51">Given that Ty1 retrotransposes efficiently, but retrosomes rarely form in a mutant with modestly reduced <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a> and 60S subunit levels, we determined whether Ty1 cDNA accumulation, a step in retrotransposition between retrosome formation and transposition into the genome, is regulated by <em><a href="http://www.yeastgenome.org/locus/S000004947/overview">RPL7</a></em> paralogs. Unintegrated Ty1 cDNA levels in paralog deletion strains were measured relative to genomic copies of Ty1 via quantitative Southern blot analysis (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Scholes <em>et al.</em> 2001</a>). Deletion of <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> decreased Ty1 cDNA levels to 8% of the wild-type strain, whereas deletion of <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> had no effect on cDNA accumulation, demonstrating that Ty1 cDNA accumulation is a paralog-specific function of <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a>). When the <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> CRI was substituted for the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> CRI at the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> locus, cDNA levels were reduced to 30% of that in the wild-type strain. On the other hand, substituting <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> for the <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> CRI at the <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> locus increased cDNA levels from 8 to 36% of that in the wild-type strain. Equivalent levels of Ty1 cDNA in the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆ <a href="http://www.yeastgenome.org/locus/S000006119/overview">rpl7b</a>∆</em>::<em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆</em>::<em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a> <a href="http://www.yeastgenome.org/locus/S000006119/overview">rpl7b</a>∆</em> mutants was not predicted by the substantially different <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a> and 60S ribosomal subunits levels between these mutants. Higher cDNA levels resulting from expression of <a href="http://www.yeastgenome.org/locus/S000003044/overview">Rpl7a</a>/<a href="http://www.yeastgenome.org/locus/S000007302/overview">snR39</a>, rather than <a href="http://www.yeastgenome.org/locus/S000006119/overview">Rpl7b</a>/<a href="http://www.yeastgenome.org/locus/S000006450/overview">snR59</a> from both the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> loci, suggested that <a href="http://www.yeastgenome.org/locus/S000003044/overview">Rpl7a</a>/<a href="http://www.yeastgenome.org/locus/S000007302/overview">snR39</a> isoforms could have a specialized role in Ty1 cDNA accumulation. To determine whether this function is unique to <a href="http://www.yeastgenome.org/locus/S000003044/overview">Rpl7a</a>/<a href="http://www.yeastgenome.org/locus/S000007302/overview">snR39</a>, or can be performed by <a href="http://www.yeastgenome.org/locus/S000006119/overview">Rpl7b</a>/<a href="http://www.yeastgenome.org/locus/S000006450/overview">snR59</a> at a higher dosage, cDNA levels in homogenic strains were measured (<a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8</a>). Expression of the <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> CRI from both loci increased the relative level of cDNA from 30 to 83% compared to expression from the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> locus only. Thus, a mutant expressing <a href="http://www.yeastgenome.org/locus/S000006119/overview">Rpl7b</a>/<a href="http://www.yeastgenome.org/locus/S000006450/overview">snR59</a> with a polysome profile that is comparable to the wild-type strain has efficient cDNA accumulation, indicating that cDNA accumulation is not a distinct function of <a href="http://www.yeastgenome.org/locus/S000003044/overview">Rpl7a</a>/<a href="http://www.yeastgenome.org/locus/S000007302/overview">snR39</a>. Nonetheless, equivalent levels of Ty1 cDNA in a mutant with low levels of <a href="http://www.yeastgenome.org/locus/S000003044/overview">Rpl7a</a>, and a severe 60S ribosomal subunit defect, as well as a mutant with higher levels of <a href="http://www.yeastgenome.org/locus/S000006119/overview">Rpl7b</a>, and a mild 60S subunit defect, suggests that <a href="http://www.yeastgenome.org/locus/S000003044/overview">Rpl7a</a> or <a href="http://www.yeastgenome.org/locus/S000007302/overview">snR39</a> is more active than <a href="http://www.yeastgenome.org/locus/S000006119/overview">Rpl7b</a> or <a href="http://www.yeastgenome.org/locus/S000006450/overview">snR59</a> at some aspect of Ty1 cDNA synthesis or stability.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/591/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Rpl7 and snR39/59 isoforms expressed affect Ty1 cDNA accumulation in a dosage-and isoform-dependent manner. (A) Schematic of a chromosomal Ty1 element, with the positions of SphI sites in Ty1 and flanking genomic DNA indicated. The position of a riboprobe used in Southern blot analysis is indicated. (B) Southern blot analyses of three biological replicates of genomic DNA from each strain. Numbers correspond to strain genotypes indicated below in (C). The amount of unintegrated Ty1 cDNA is measured relative to genomic Ty1 elements in SphI-digested total DNA. The size of each Ty1:genomic DNA junction band is determined by the distance between the internal Ty1 SphI site and the nearest SphI site in flanking genomic DNA. The 1.7 kb Ty1 cDNA band is the fastest-migrating band because of the lack of flanking genomic DNA. The relative Ty1 cDNA level is the average ratio of the 32P-signal in the unintegrated Ty1 cDNA band (cDNA) relative to the average of two different Ty1:genomic DNA junction bands (G1 and G2) in three samples. Unmarked lanes: molecular weight marker. (C) Average (&#xB1;SE) Ty1 cDNA level in each mutant relative to the RPL7A RPL7B strain. The spt3&#x2206; strain, in which Ty1 RNA and cDNA are markedly decreased, was analyzed as a negative control." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1111785762" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Rpl7 and snR39/59 isoforms expressed affect Ty1 cDNA accumulation in a dosage-and isoform-dependent manner. (A) Schematic of a chromosomal Ty1 element, with the positions of &lt;em&gt;Sph&lt;/em&gt;I sites in Ty1 and flanking genomic DNA indicated. The position of a riboprobe used in Southern blot analysis is indicated. (B) Southern blot analyses of three biological replicates of genomic DNA from each strain. Numbers correspond to strain genotypes indicated below in (C). The amount of unintegrated Ty1 cDNA is measured relative to genomic Ty1 elements in &lt;em&gt;Sph&lt;/em&gt;I-digested total DNA. The size of each Ty1:genomic DNA junction band is determined by the distance between the internal Ty1 &lt;em&gt;Sph&lt;/em&gt;I site and the nearest &lt;em&gt;Sph&lt;/em&gt;I site in flanking genomic DNA. The 1.7 kb Ty1 cDNA band is the fastest-migrating band because of the lack of flanking genomic DNA. The relative Ty1 cDNA level is the average ratio of the &lt;sup&gt;32&lt;/sup&gt;P-signal in the unintegrated Ty1 cDNA band (cDNA) relative to the average of two different Ty1:genomic DNA junction bands (G1 and G2) in three samples. Unmarked lanes: molecular weight marker. (C) Average (&#xB1;SE) Ty1 cDNA level in each mutant relative to the &lt;em&gt;RPL7A RPL7B&lt;/em&gt; strain. The &lt;em&gt;spt3&#x2206;&lt;/em&gt; strain, in which Ty1 RNA and cDNA are markedly decreased, was analyzed as a negative control.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/591/F8.medium.gif" width="440" height="422"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/7/2/591/F8.medium.gif" width="440" height="422"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/591/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/591/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470354" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-52" class="first-child">Rpl7 and snR39/59 isoforms expressed affect Ty1 cDNA accumulation in a dosage-and isoform-dependent manner. (A) Schematic of a chromosomal Ty1 element, with the positions of <em>Sph</em>I sites in Ty1 and flanking genomic DNA indicated. The position of a riboprobe used in Southern blot analysis is indicated. (B) Southern blot analyses of three biological replicates of genomic DNA from each strain. Numbers correspond to strain genotypes indicated below in (C). The amount of unintegrated Ty1 cDNA is measured relative to genomic Ty1 elements in <em>Sph</em>I-digested total DNA. The size of each Ty1:genomic DNA junction band is determined by the distance between the internal Ty1 <em>Sph</em>I site and the nearest <em>Sph</em>I site in flanking genomic DNA. The 1.7 kb Ty1 cDNA band is the fastest-migrating band because of the lack of flanking genomic DNA. The relative Ty1 cDNA level is the average ratio of the <sup>32</sup>P-signal in the unintegrated Ty1 cDNA band (cDNA) relative to the average of two different Ty1:genomic DNA junction bands (G1 and G2) in three samples. Unmarked lanes: molecular weight marker. (C) Average (±SE) Ty1 cDNA level in each mutant relative to the <em>RPL7A RPL7B</em> strain. The <em>spt3∆</em> strain, in which Ty1 RNA and cDNA are markedly decreased, was analyzed as a negative control.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-21" class="subsection"><h3>Unaltered mechanism of cDNA integration in the rpl7a∆::RPL7B rpl7b∆ mutant</h3><p id="p-53">The rate of Ty1 retromobility in the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆</em>::<em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a> <a href="http://www.yeastgenome.org/locus/S000006119/overview">rpl7b</a>∆</em> mutant was similar to that of the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a> <a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> strain (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>), despite a low levels of Ty1 retrosomes, and a 70% reduction in unintegrated Ty1 cDNA. Therefore, we determined whether Ty1 cDNA is introduced into the genome of the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆</em>::<em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a> <a href="http://www.yeastgenome.org/locus/S000006119/overview">rpl7b</a>∆</em> mutant via a mechanism that is different than that of the wild-type strain, which might uncover evidence for a specialized function <a href="http://www.yeastgenome.org/locus/S000006119/overview">Rpl7b</a>/<a href="http://www.yeastgenome.org/locus/S000006450/overview">snR59</a> in retromobility. Ty1 cDNA integration is normally much more efficient than recombination with genomic Ty1 sequences. Accordingly, the homologous recombination factor, <a href="http://www.yeastgenome.org/locus/S000004494/overview">Rad52</a>, is dispensable for retromobility, and deletion of <em><a href="http://www.yeastgenome.org/locus/S000004494/overview">RAD52</a></em> increases Ty1 retromobility by stimulating cDNA synthesis (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>) (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Sharon <em>et al.</em> 1994</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Rattray <em>et al.</em> 2000</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Curcio <em>et al.</em> 2007</a>). Deletion of <em><a href="http://www.yeastgenome.org/locus/S000004494/overview">RAD52</a></em> also increased retromobility in the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆</em>::<em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a> <a href="http://www.yeastgenome.org/locus/S000006119/overview">rpl7b</a>∆</em> strain, suggesting that cDNA recombination is rare and integration remains the major mechanism of retromobility (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). A second possibility is that the target site specificity of cDNA integration is altered when only <a href="http://www.yeastgenome.org/locus/S000006119/overview">Rpl7b</a>/<a href="http://www.yeastgenome.org/locus/S000006450/overview">snR59</a> is expressed. Therefore, independent Ty1<em><a href="http://www.yeastgenome.org/locus/S000005728/overview">HIS3</a></em> integration sites in the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆</em>::<em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a> <a href="http://www.yeastgenome.org/locus/S000006119/overview">rpl7b</a>∆</em> mutant were compared to those in a wild-type strain. Following selection for His<sup>+</sup> prototrophs, the location of Ty1<em><a href="http://www.yeastgenome.org/locus/S000005728/overview">HIS3</a></em> was determined by sequencing Ty1<em><a href="http://www.yeastgenome.org/locus/S000005728/overview">HIS3</a></em> cDNA:genomic DNA junctions amplified by TAIL PCR (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035931/-/DC1/TableS3.docx">Table S3</a>). Integration is usually targeted to a ∼750-bp window upstream of Pol III transcribed genes (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Ji <em>et al.</em> 1993</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Devine and Boeke 1996</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Baller <em>et al.</em> 2012</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Mularoni <em>et al.</em> 2012</a>). In the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a> <a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> strain, 12 of 17 Ty1<em><a href="http://www.yeastgenome.org/locus/S000005728/overview">HIS3</a></em> retromobility events integrated within 475 bp upstream of a Pol III-transcribed gene, and one was 150 bp downstream of a Pol III-transcribed gene. In the remaining four isolates, Ty1<em><a href="http://www.yeastgenome.org/locus/S000005728/overview">HIS3</a></em> cDNA was present within a tandem array of Ty1 cDNAs, so genomic sequences flanking these arrays could not be identified by TAIL PCR. A similar integration profile was observed for 15 independent Ty1<em><a href="http://www.yeastgenome.org/locus/S000005728/overview">HIS3</a></em> cDNA integration events in the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆</em>::<em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a> <a href="http://www.yeastgenome.org/locus/S000006119/overview">rpl7b</a>∆</em> mutant. The two strains had equivalent fractions of Ty1<em><a href="http://www.yeastgenome.org/locus/S000005728/overview">HIS3</a></em> elements integrated upstream of Pol III-transcribed genes (11/15 <em>vs.</em> 12/17) and within multimeric arrays (4/15 <em>vs.</em> 4/17). One Ty1<em><a href="http://www.yeastgenome.org/locus/S000005728/overview">HIS3</a></em> element integrated into an ORF &gt;6 kb away from the nearest Pol III-transcribed gene, but this low fraction of ORF disruption is not inconsistent with that observed in a study of &gt;150,000 Ty1<em><a href="http://www.yeastgenome.org/locus/S000005728/overview">HIS3</a></em> insertions in a wild-type strain (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Baller <em>et al.</em> 2012</a>). We conclude that Ty1 target specificity is generally preserved in the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆</em>::<em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a> <a href="http://www.yeastgenome.org/locus/S000006119/overview">rpl7b</a>∆</em> mutant, and, therefore, that expression of <a href="http://www.yeastgenome.org/locus/S000006119/overview">Rpl7b</a>/<a href="http://www.yeastgenome.org/locus/S000006450/overview">snR59</a> alone does not result in an altered mechanism of Ty1 retromobility.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470250/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470250/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Effect of <em>RAD52</em> disruption on Ty1<em>his3AI</em> retromobility</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-22"><h2 class="">Discussion</h2><div id="sec-23" class="subsection"><h3>Different Rpl7 or snoRNA levels underlie paralog-specific functions of RPL7A and RPL7B</h3><p id="p-55">The question of whether the diverse phenotypic consequences of deleting one paralog or the other of an RP gene pair can be explained by differences in RP gene dosage, or constitute evidence of “specialized” ribosomes containing distinct subsets of RP isoforms has been widely debated (<a id="xref-ref-38-9" class="xref-bibr" href="#ref-38">Komili <em>et al.</em> 2007</a>; <a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Parenteau <em>et al.</em> 2011</a>, <a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">2015</a>; <a id="xref-ref-74-6" class="xref-bibr" href="#ref-74">Steffen <em>et al.</em> 2012</a>; <a id="xref-ref-82-2" class="xref-bibr" href="#ref-82">Xue and Barna 2012</a>; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Slavov <em>et al.</em> 2015</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Dinman 2016</a>). Based on the mislocalization of <a href="http://www.yeastgenome.org/locus/S000006119/overview">Rpl7b</a>, but not <a href="http://www.yeastgenome.org/locus/S000003044/overview">Rpl7a</a>, when certain ribosome biogenesis factors were depleted, it was proposed that <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a> isoforms are incorporated into heterogenous ribosomes that selectively translate specific mRNAs, and give rise to paralog-specific functions of <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> (<a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Mauro and Edelman 2002</a>, <a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">2007</a>; <a id="xref-ref-38-10" class="xref-bibr" href="#ref-38">Komili <em>et al.</em> 2007</a>; <a id="xref-ref-82-3" class="xref-bibr" href="#ref-82">Xue and Barna 2012</a>). However, phenotypic comparisons of strains expressing either <a href="http://www.yeastgenome.org/locus/S000003044/overview">Rpl7a</a>/<a href="http://www.yeastgenome.org/locus/S000007302/overview">snR39</a> or <a href="http://www.yeastgenome.org/locus/S000006119/overview">Rpl7b</a>/<a href="http://www.yeastgenome.org/locus/S000006450/overview">snR59</a> from the robust <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> locus, or weaker <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> locus, suggest that paralog-specific functions of <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> can be explained by differences in the level of <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a> or <em><a href="http://www.yeastgenome.org/locus/S000004947/overview">RPL7</a></em>-encoded snoRNA expression, and the effect of depleting these gene products on 60S ribosomal subunit levels. For example, the efficiencies of <em><a href="http://www.yeastgenome.org/locus/S000001668/overview">ASH1</a></em> mRNA localization, and Ty1 retromobility in mutants expressing either <em><a href="http://www.yeastgenome.org/locus/S000004947/overview">RPL7</a></em> CRI from the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> locus, were similar to each other, and to the wild-type strain, whereas mutants with either <em><a href="http://www.yeastgenome.org/locus/S000004947/overview">RPL7</a></em> CRI at the <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> locus had marked defects in these processes. Because the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆ <a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆ <a href="http://www.yeastgenome.org/locus/S000006119/overview">rpl7b</a>∆</em>::<em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> mutants have severe reductions in 60S ribosomal subunits that lead to halfmer ribosomes, and the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆</em>::<em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a> <a href="http://www.yeastgenome.org/locus/S000006119/overview">rpl7b</a>∆</em> and <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a> <a href="http://www.yeastgenome.org/locus/S000006119/overview">rpl7b</a>∆</em> mutants have minor reductions in 60S ribosomal subunits; these observations indicate that <em><a href="http://www.yeastgenome.org/locus/S000001668/overview">ASH1</a></em> mRNA localization and Ty1 retromobility are correlated with the dosage of <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a> and <a href="http://www.yeastgenome.org/locus/S000007302/overview">snR39</a>/<a href="http://www.yeastgenome.org/locus/S000006450/overview">59</a>, and not with the specific isoforms expressed. The Ty1 retromobility phenotypes of strains expressing an epitope-tagged <a href="http://www.yeastgenome.org/locus/S000006119/overview">Rpl7b</a>, the level of which increased when <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> is deleted, support this view, and further suggest that only the dosage of single <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a> isoform need be altered to produce different phenotypic outcomes. Dosage of the encoded product(s) also explains the paralog-specific regulation of Ty1 retromobility by <em><a href="http://www.yeastgenome.org/locus/S000002233/overview">RPL31A</a></em>. Together, these findings support the hypothesis that phenotypic variation in complex cellular processes can result from different degrees of ribosomal subunit imbalance.</p><p id="p-56">Other phenotypes examined here differed depending on which CRI was expressed from a particular locus. There was greater resistance to tunicamycin when <a href="http://www.yeastgenome.org/locus/S000006119/overview">Rpl7b</a>/<a href="http://www.yeastgenome.org/locus/S000006450/overview">snR59</a> <em>vs.</em> <a href="http://www.yeastgenome.org/locus/S000003044/overview">Rpl7a</a>/<a href="http://www.yeastgenome.org/locus/S000007302/overview">snR39</a> was expressed from the <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> locus, and Ty1 RNA localization was significantly more robust when <a href="http://www.yeastgenome.org/locus/S000003044/overview">Rpl7a</a>/<a href="http://www.yeastgenome.org/locus/S000007302/overview">snR39</a> was expressed from the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> locus. These phenotypic differences do not necessarily result from distinct, or even specialized, functions of the <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a> or snoRNA isoforms, because <a href="http://www.yeastgenome.org/locus/S000003044/overview">Rpl7a</a> is more highly expressed from either locus than <a href="http://www.yeastgenome.org/locus/S000006119/overview">Rpl7b</a>. The reason for this expression difference was not investigated, but it likely reflects the negative regulatory effects that <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> introns have on <em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> mRNA expression (<a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">Parenteau <em>et al.</em> 2011</a>). Growth rates and polysome profiles indicated that mutants expressing <a href="http://www.yeastgenome.org/locus/S000003044/overview">Rpl7a</a>/<a href="http://www.yeastgenome.org/locus/S000007302/overview">snR39</a> have a less severe deficiency in 60S ribosomal subunits that parallels their higher <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a> levels. Thus, a different extent of 60S subunit depletion is the likely explanation for the phenotypic differences associated with disparate <em><a href="http://www.yeastgenome.org/locus/S000004947/overview">RPL7</a></em> CRI at the same locus. The importance of <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a> and snoRNA dosage and irrelevance of isoform in these phenotypes was further illustrated by the effects of increasing the dosage of <a href="http://www.yeastgenome.org/locus/S000006119/overview">Rpl7b</a>/<a href="http://www.yeastgenome.org/locus/S000006450/overview">snR59</a>. Tunicamycin resistance was suppressed by expressing <a href="http://www.yeastgenome.org/locus/S000006119/overview">Rpl7b</a>/<a href="http://www.yeastgenome.org/locus/S000006450/overview">snR59</a> from the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> locus, and the Ty1 RNA localization defect was rescued by expressing <a href="http://www.yeastgenome.org/locus/S000006119/overview">Rpl7b</a>/<a href="http://www.yeastgenome.org/locus/S000006450/overview">snR59</a> from both <em><a href="http://www.yeastgenome.org/locus/S000004947/overview">RPL7</a></em> loci. The dependence of the phenotypes above on the dosage of <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a> or snoRNA strongly suggests that neither <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a> nor <a href="http://www.yeastgenome.org/locus/S000007302/overview">snR39</a>/<a href="http://www.yeastgenome.org/locus/S000006450/overview">59</a> isoforms have distinct functions in the relevant cellular processes.</p><p id="p-57">Ty1 cDNA accumulation was the single phenotype studied here that was not strictly correlated with the extent of 60S ribosomal subunit depletion. Similar levels of Ty1 cDNA in a <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆ <a href="http://www.yeastgenome.org/locus/S000006119/overview">rpl7b</a>∆</em>::<em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> mutant, which has a severe 60S ribosomal subunit biogenesis defect, and an <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆</em>::<em><a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a> <a href="http://www.yeastgenome.org/locus/S000006119/overview">rpl7b</a>∆</em> mutant, which has only a modest reduction in 60S ribosomal subunits, suggests that <a href="http://www.yeastgenome.org/locus/S000003044/overview">Rpl7a</a> or <a href="http://www.yeastgenome.org/locus/S000007302/overview">snR39</a> could have a specialized role in promoting cDNA synthesis or stability. Nonetheless, this is not a unique role, since expressing <a href="http://www.yeastgenome.org/locus/S000006119/overview">Rpl7b</a>/<a href="http://www.yeastgenome.org/locus/S000006450/overview">snR59</a> from both <em><a href="http://www.yeastgenome.org/locus/S000004947/overview">RPL7</a></em> loci rescued the defect in cDNA accumulation in mutants expressing <a href="http://www.yeastgenome.org/locus/S000006119/overview">Rpl7b</a>/<a href="http://www.yeastgenome.org/locus/S000006450/overview">snR59</a> from either single locus. The finding that defective cDNA accumulation is correlated with a depletion of <a href="http://www.yeastgenome.org/locus/S000003044/overview">Rpl7a</a>/<a href="http://www.yeastgenome.org/locus/S000007302/overview">snR39</a> or <a href="http://www.yeastgenome.org/locus/S000006119/overview">Rpl7b</a>/<a href="http://www.yeastgenome.org/locus/S000006450/overview">snR59</a>, albeit to substantially different levels depending on the isoforms, argues against the idea that control of cDNA accumulation is an extraribosomal function of <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a> isoforms. Rather, these observations are consistent with <a href="http://www.yeastgenome.org/locus/S000003044/overview">Rpl7a</a> being more active than <a href="http://www.yeastgenome.org/locus/S000006119/overview">Rpl7b</a> in a specific ribosome-associated function. Numerous host cofactors and signaling pathways influence Ty1 cDNA levels (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Curcio <em>et al.</em> 2015</a>), and, thus, the translation of an mRNA that encodes a Ty1 cofactor could be more efficient in a strain expressing <a href="http://www.yeastgenome.org/locus/S000003044/overview">Rpl7a</a>/<a href="http://www.yeastgenome.org/locus/S000007302/overview">snR39</a>. The N-terminal domain of <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a>, which contains four of the five amino acid substitutions that differentiate the <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a> isoforms and forms an α-helix located on the surface of the ribosome (<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Wilson and Doudna Cate 2012</a>), could differentially recruit a factor that influences the efficiency of translation of specific mRNAs. Three of the four N-terminal substitutions in <a href="http://www.yeastgenome.org/locus/S000003044/overview">Rpl7a</a> are serine and threonine residues, suggesting the possible involvement of signaling via post-translational modification of <a href="http://www.yeastgenome.org/locus/S000003044/overview">Rpl7a</a>.</p><p id="p-58">Although evidence for neofunctionalization of <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a>, <a href="http://www.yeastgenome.org/locus/S000007302/overview">snR39</a>/<a href="http://www.yeastgenome.org/locus/S000006450/overview">59</a> or Rpl31 isoforms was not uncovered, it remains possible that these isoforms have distinct, dosage-independent functions in other processes not tested here, and that <em><a href="http://www.yeastgenome.org/locus/S000004947/overview">RPL7</a></em> and <em>RPL31</em> paralogs are differentially expressed as environmental conditions vary (<a id="xref-ref-54-4" class="xref-bibr" href="#ref-54">Parenteau <em>et al.</em> 2011</a>, <a id="xref-ref-55-3" class="xref-bibr" href="#ref-55">2015</a>). The chimeragenic and homogenic <em><a href="http://www.yeastgenome.org/locus/S000004947/overview">RPL7</a></em> strains described here provide a useful tool for testing the isoform specificity of other cellular processes under a variety of environmental conditions. It is also possible that isoforms encoded by RP gene pairs other than <em><a href="http://www.yeastgenome.org/locus/S000004947/overview">RPL7</a></em> paralogs have unique functions in specialized ribosomes. Nonetheless, this study highlights the importance of considering the relative levels of each paralog’s protein products in the absence of the corresponding paralog when assigning RP isoform-specific functions. Other studies have used the rate of cell growth and polysome profiling in paralog-deletion strains as informative proxies of relative RP levels that are incorporated into functional ribosomes (<a id="xref-ref-73-4" class="xref-bibr" href="#ref-73">Steffen <em>et al.</em> 2008</a>, <a id="xref-ref-74-7" class="xref-bibr" href="#ref-74">2012</a>), and these findings also support the idea that RP dosage and levels of functional ribosomes underlie many complex cellular phenotypes in yeast.</p></div><div id="sec-24" class="subsection"><h3>The effects of Rpl7 depletion on intermediate steps in Ty1 retromobility</h3><p id="p-59">Uncovering the mechanism by which <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a> or the <em><a href="http://www.yeastgenome.org/locus/S000004947/overview">RPL7</a></em>-encoded snoRNAs regulate Ty1 retromobility was not the primary goal of this study, but several mechanistic conclusions can be drawn from our findings. First, Ty1 mRNA is not likely to be translated preferentially on ribosomes containing <a href="http://www.yeastgenome.org/locus/S000003044/overview">Rpl7a</a> or <a href="http://www.yeastgenome.org/locus/S000007302/overview">snR39</a>-modified rRNA, since equivalent levels of Gag accumulate when either the <em><a href="http://www.yeastgenome.org/locus/S000004947/overview">RPL7</a></em> CRI was expressed from the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> locus. Second, depletion of <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a>, even to the point of a severe deficiency in 60S ribosomal subunits, caused a &lt;2-fold decrease in the level of Gag, and overexpressing Ty1 proteins only weakly rescued the Ty1 retromobility defect of an <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆ <a href="http://www.yeastgenome.org/locus/S000006119/overview">RPL7B</a></em> mutant; therefore, reduced Gag is not likely to be the major cause of the decrease in retromobility in <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">rpl7a</a>∆</em> mutants. Many RP genes influence Ty1 retromobility (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Harger <em>et al.</em> 2001</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Griffith <em>et al.</em> 2003</a>; <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Dakshinamurthy <em>et al.</em> 2010</a>; <a id="xref-ref-61-7" class="xref-bibr" href="#ref-61">Risler <em>et al.</em> 2012</a>; <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Suresh <em>et al.</em> 2015</a>), and several others that are superfluous for Ty1 Gag accumulation have been characterized (<a id="xref-ref-76-2" class="xref-bibr" href="#ref-76">Suresh <em>et al.</em> 2015</a>). Third, Ty1 retromobility was more sensitive to <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a> or <a href="http://www.yeastgenome.org/locus/S000007302/overview">snR39</a>/<a href="http://www.yeastgenome.org/locus/S000006450/overview">59</a> depletion than Gag accumulation, suggesting that <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a> or <a href="http://www.yeastgenome.org/locus/S000007302/overview">snR39</a>/<a href="http://www.yeastgenome.org/locus/S000006450/overview">59</a> might regulate retrotransposition at a post-translational level. However, it is important to note that our studies have not addressed the effect of <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a> depletion on translational frameshifting from the <em>GAG</em> to <em>POL</em> ORF, the efficiency of which impacts the level of Ty1 retrotransposition (<a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">Curcio <em>et al.</em> 2015</a>). Fourth, localization of Ty1 RNA to retrosomes, and cDNA accumulation, had very low thresholds for <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a> and <a href="http://www.yeastgenome.org/locus/S000007302/overview">snR39</a>/<a href="http://www.yeastgenome.org/locus/S000006450/overview">59</a> levels, suggesting that <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a> or <em><a href="http://www.yeastgenome.org/locus/S000004947/overview">RPL7</a></em>-encoded snoRNAs might directly influence Ty1 RNA localization. While retrotransposition was efficient even in a mutant with reduced retrosomes and cDNA, the findings suggest a model in which <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a> or <a href="http://www.yeastgenome.org/locus/S000007302/overview">snR39</a>/<a href="http://www.yeastgenome.org/locus/S000006450/overview">59</a> control the cotranslational localization of Ty1 RNA in retrosomes. <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a> or ribosomal RNA modifications made by <a href="http://www.yeastgenome.org/locus/S000007302/overview">snR39</a>/<a href="http://www.yeastgenome.org/locus/S000006450/overview">59</a> could facilitate the interaction between Gag and Ty1 RNA on translating ribosomes, which is critical for retrosome formation (<a id="xref-ref-24-6" class="xref-bibr" href="#ref-24">Doh <em>et al.</em> 2014</a>). Another possibility supported by our findings is that depletion of <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a>, like that of a few other 60S ribosomal subunit proteins, increases expression of a truncated form of Gag (p22-Gag) that potently inhibits retromobility and reduces retrosome formation (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Saha <em>et al.</em> 2015</a>; <a id="xref-ref-76-3" class="xref-bibr" href="#ref-76">Suresh <em>et al.</em> 2015</a>). It is not obvious how a shortage of 60S ribosomal subunits would increase the expression of the internal Ty1 transcript that encodes p22-Gag, but it could be via an indirect effect on cell metabolism or the cell cycle.</p></div><div id="sec-25" class="subsection"><h3>Variable phenotypic consequences of reduced expression of Rpl7</h3><p id="p-60">The results presented here suggest that different cellular pathways have varied thresholds for 60S ribosomal subunit depletion. For example, the dosage of <a href="http://www.yeastgenome.org/locus/S000006119/overview">Rpl7b</a>/<a href="http://www.yeastgenome.org/locus/S000006450/overview">snR59</a> produced from the <em><a href="http://www.yeastgenome.org/locus/S000003044/overview">RPL7A</a></em> locus, which results in a moderately reduced level of 60S ribosomal subunits, and slower growth relative to the wild-type strain, is sufficient for wild-type levels of <em><a href="http://www.yeastgenome.org/locus/S000001668/overview">ASH1</a></em> mRNA localization, and Ty1 retromobility, but insufficient for Ty1 RNA localization. Depletion of other 60S ribosomal subunits by deletion of different RP gene paralogs, or 60S subunit biogenesis factors, also quantitatively alters distinct cellular processes, including replicative life span and translation of the L-A virus mRNA (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Ohtake and Wickner 1995</a>; <a id="xref-ref-73-5" class="xref-bibr" href="#ref-73">Steffen <em>et al.</em> 2008</a>). Therefore, phenotypic variation in many cellular processes might be linked to varying degrees of ribosomal subunit imbalance. The asymmetric regulation and expression of many RP gene pairs also supports the idea that differences in ribosome levels could underlie paralog-specific phenotypes (<a id="xref-ref-54-5" class="xref-bibr" href="#ref-54">Parenteau <em>et al.</em> 2011</a>, <a id="xref-ref-55-4" class="xref-bibr" href="#ref-55">2015</a>).</p><p id="p-61">In humans, mutations in RP genes, most of which are present in a single copy, have been linked to a plethora of inherited developmental abnormalities, and somatic mutations in RP genes have been found in specific cancers. Tissue-specific defects that characterize specific mutations in various RP genes may reflect unique functions of ribosomes lacking a particular RP, or imbalances in ribosomal subunits that affect some cellular pathways more strongly than others, and each of these cellular pathways may be more or less critical for differentiation in specific cell types (<a id="xref-ref-82-4" class="xref-bibr" href="#ref-82">Xue and Barna 2012</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">McCann and Baserga 2013</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">De Keersmaecker <em>et al.</em> 2015</a>). In mice, ribosomopathy-associated tissue-specific abnormalities are, in some cases, linked to impaired translation of specific subsets of mRNAs (<a id="xref-ref-82-5" class="xref-bibr" href="#ref-82">Xue and Barna 2012</a>; <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">McCann and Baserga 2013</a>). For example, depletion of Rpl38 in certain tissues blocks translation of the mRNAs of a specific subset of homeobox genes during mouse embryogenesis; these mRNAs have specific <em>cis</em>-acting regulatory sequences in their 5′ UTRs that govern their dependence on Rpl38 (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Kondrashov <em>et al.</em> 2011</a>; <a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Xue <em>et al.</em> 2015</a>). What remains to be determined in human ribosomopathies and mouse models is whether the depletion of a specific RP changes the composition or level of functional ribosomes (<a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">McCann and Baserga 2013</a>). Because <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a> is required for an early step in ribosome biogenesis in yeast, and there is a tight correlation between <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a> and 60S subunit levels (<a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Jakovljevic <em>et al.</em> 2012</a>), it is likely that the phenotypes associated with <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a> and snoRNA depletion in yeast are a result of a reduced level of functional ribosomes, rather than an increased percentage of ribosomes lacking <a href="http://www.yeastgenome.org/locus/S000004947/overview">Rpl7</a>. Thus, this study supports a model in which subtle differences in ribosomal subunit biogenesis caused by depletion of a single RP result in varied phenotypic consequences, perhaps because translation of specific subsets of mRNAs have different sensitivities to imbalances between ribosomal subunits (<a id="xref-ref-52-3" class="xref-bibr" href="#ref-52">Ohtake and Wickner 1995</a>; <a id="xref-ref-35-4" class="xref-bibr" href="#ref-35">Jakovljevic <em>et al.</em> 2012</a>; <a id="xref-ref-74-8" class="xref-bibr" href="#ref-74">Steffen <em>et al.</em> 2012</a>). Characterization of global translation patterns in yeast strains that are homogenic or chimeragenic for a single RP paralog, or harbor a single native allele, could provide a systematic means of identifying classes of mRNAs that are differentially sensitive to ribosomal subunit imbalances.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-62">We are grateful to Patrick Maxwell for sharing experimental findings that initiated this project, to Eric Gamache for technical contributions, to Pooja Flora for figure preparation, to Michael Resnick and Jeffrey Gerst for providing plasmids, and to Randy Morse and Dieter Kressler for thoughtful comments on the manuscript. We thank the Wadsworth Center Applied Genomic Technologies Core for DNA sequencing, the Wadsworth Center DAI Microscope facility for use of fluorescent microscopes, and the Wadsworth Immunology Core for use of the flow cytometer.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-5"><p id="p-63">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035931/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035931/-/DC1</a>.</p></li><li class="fn" id="fn-6"><p id="p-64"><em>Communicating editor: J. Rine</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-09-26"><span class="received-label">Received </span>September 26, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-12-12"><span class="accepted-label">Accepted </span>December 12, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Palumbo <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.1" data-doi="10.1128/MCB.4.9.1871"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abovich</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosbash</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Two genes for ribosomal protein 51 of <em>Saccharomyces cerevisiae</em> complement and contribute to the ribosomes.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1871</span>–<span class="cit-lpage">1879</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DAbovich%26rft.auinit1%253DN%26rft.volume%253D4%26rft.issue%253D9%26rft.spage%253D1871%26rft.epage%253D1879%26rft.atitle%253DTwo%2Bgenes%2Bfor%2Bribosomal%2Bprotein%2B51%2Bof%2BSaccharomyces%2Bcerevisiae%2Bcomplement%2Band%2Bcontribute%2Bto%2Bthe%2Bribosomes.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.4.9.1871%26rft_id%253Dinfo%253Apmid%252F6092944%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjg6IjQvOS8xODcxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzU5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-book" id="cit-7.2.591.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Amberg</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burke</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strathern</span>  <span class="cit-name-given-names">J. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-source"><em>Methods in Yeast Genetics</em>.</span> <span class="cit-publ-name">Cold Spring Harbor Laboratory</span>, <span class="cit-publ-loc">Cold Spring Harbor, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.3" data-doi="10.1073/pnas.0401263101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Askree</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yehuda</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smolikov</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gurevich</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hawk</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A genome-wide screen for <em>Saccharomyces cerevisiae</em> deletion mutants that affect telomere length.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">8658</span>–<span class="cit-lpage">8663</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0401263101%26rft_id%253Dinfo%253Apmid%252F15161972%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAxLzIzLzg2NTgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzIvNTkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.4" data-doi="10.1016/j.molcel.2004.06.028"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Badis</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saveanu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fromont-Racine</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacquier</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Targeted mRNA degradation by deadenylation-independent decapping.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">5</span>–<span class="cit-lpage">15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DBadis%26rft.auinit1%253DG.%26rft.volume%253D15%26rft.issue%253D1%26rft.spage%253D5%26rft.epage%253D15%26rft.atitle%253DTargeted%2BmRNA%2Bdegradation%2Bby%2Bdeadenylation-independent%2Bdecapping.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2004.06.028%26rft_id%253Dinfo%253Apmid%252F15225544%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2004.06.028&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15225544&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F591.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222439100003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.5" data-doi="10.1101/gr.129585.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baller</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stamenova</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Curcio</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voytas</span>  <span class="cit-name-given-names">D. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A nucleosomal surface defines an integration hotspot for the <em>Saccharomyces cerevisiae</em> Ty1 retrotransposon.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">704</span>–<span class="cit-lpage">713</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.129585.111%26rft_id%253Dinfo%253Apmid%252F22219511%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIyLzQvNzA0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzU5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.6" data-doi="10.1091/mbc.12.9.2567"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beach</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bloom</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">ASH1 mRNA localization in three acts.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">2567</span>–<span class="cit-lpage">2577</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Bof%2Bthe%2BCell%26rft.stitle%253DMol.%2BBiol.%2BCell%26rft.aulast%253DBeach%26rft.auinit1%253DD.%2BL.%26rft.volume%253D12%26rft.issue%253D9%26rft.spage%253D2567%26rft.epage%253D2577%26rft.atitle%253DASH1%2BmRNA%2BLocalization%2Bin%2BThree%2BActs%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.12.9.2567%26rft_id%253Dinfo%253Apmid%252F11553699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjEyLzkvMjU2NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi81OTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.7" data-doi="10.1016/S0960-9822(99)80260-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beach</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salmon</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bloom</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Localization and anchoring of mRNA in budding yeast.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">569</span>–<span class="cit-lpage">578</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DBeach%26rft.auinit1%253DD.%2BL.%26rft.volume%253D9%26rft.issue%253D11%26rft.spage%253D569%26rft.epage%253D578%26rft.atitle%253DLocalization%2Band%2Banchoring%2Bof%2BmRNA%2Bin%2Bbudding%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0960-9822%252899%252980260-7%26rft_id%253Dinfo%253Apmid%252F10359695%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0960-9822(99)80260-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10359695&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F591.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000080859700015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.8" data-doi="10.1126/science.1212642"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ben-Shem</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garreau de Loubresse</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melnikov</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jenner</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yusupova</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The structure of the eukaryotic ribosome at 3.0 A resolution.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">334</span>: <span class="cit-fpage">1524</span>–<span class="cit-lpage">1529</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1212642%26rft_id%253Dinfo%253Apmid%252F22096102%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzQvNjA2Mi8xNTI0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzU5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.9" data-doi="10.1016/S1097-2765(00)80143-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bertrand</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chartrand</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaefer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shenoy</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singer</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Localization of ASH1 mRNA particles in living yeast.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">437</span>–<span class="cit-lpage">445</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DBertrand%26rft.auinit1%253DE.%26rft.volume%253D2%26rft.issue%253D4%26rft.spage%253D437%26rft.epage%253D445%26rft.atitle%253DLocalization%2Bof%2BASH1%2BmRNA%2Bparticles%2Bin%2Bliving%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1097-2765%252800%252980143-4%26rft_id%253Dinfo%253Apmid%252F9809065%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1097-2765(00)80143-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9809065&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F591.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000076678900004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.10" data-doi="10.1128/MCB.8.4.1432"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boeke</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eichinger</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Castrillon</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fink</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">The <em>Saccharomyces cerevisiae</em> genome contains functional and nonfunctional copies of transposon Ty1.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1432</span>–<span class="cit-lpage">1442</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DBoeke%26rft.auinit1%253DJ%2BD%26rft.volume%253D8%26rft.issue%253D4%26rft.spage%253D1432%26rft.epage%253D1442%26rft.atitle%253DThe%2BSaccharomyces%2Bcerevisiae%2Bgenome%2Bcontains%2Bfunctional%2Band%2Bnonfunctional%2Bcopies%2Bof%2Btransposon%2BTy1.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.8.4.1432%26rft_id%253Dinfo%253Apmid%252F2837641%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjg6IjgvNC8xNDMyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzU5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.11" data-doi="10.1091/mbc.02-01-0005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bonangelino</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chavez</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonifacino</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Genomic screen for vacuolar protein sorting genes in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">2486</span>–<span class="cit-lpage">2501</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.02-01-0005%26rft_id%253Dinfo%253Apmid%252F12134085%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjEzLzcvMjQ4NiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi81OTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.12" data-doi="10.1128/MCB.00251-09"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Checkley</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagashima</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lockett</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nyswaner</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garfinkel</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">P-body components are required for Ty1 retrotransposition during assembly of retrotransposition-competent virus-like particles.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">382</span>–<span class="cit-lpage">398</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.00251-09%26rft_id%253Dinfo%253Apmid%252F19901074%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjg6IjMwLzIvMzgyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzU5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.13" data-doi="10.1371/journal.pone.0009114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clarkson</span>  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilbert</span>  <span class="cit-name-given-names">W. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doudna</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Functional overlap between eIF4G isoforms in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e9114</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DClarkson%26rft.auinit1%253DB.%2BK.%26rft.volume%253D5%26rft.issue%253D2%26rft.spage%253De9114%26rft.epage%253De9114%26rft.atitle%253DFunctional%2Boverlap%2Bbetween%2BeIF4G%2Bisoforms%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0009114%26rft_id%253Dinfo%253Apmid%252F20161741%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0009114&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20161741&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F591.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.14" data-doi="10.1046/j.1365-2958.2000.01710.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conte</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Curcio</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Fus3 controls Ty1 transpositional dormancy through the invasive growth MAPK pathway.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">415</span>–<span class="cit-lpage">427</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DConte%26rft.auinit1%253DD.%26rft.volume%253D35%26rft.issue%253D2%26rft.spage%253D415%26rft.epage%253D427%26rft.atitle%253DFus3%2Bcontrols%2BTy1%2Btranspositional%2Bdormancy%2Bthrough%2Bthe%2Binvasive%2Bgrowth%2BMAPK%2Bpathway.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2958.2000.01710.x%26rft_id%253Dinfo%253Apmid%252F10652102%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2958.2000.01710.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10652102&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F591.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000085242300016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.15" data-doi="10.1073/pnas.88.3.936"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Curcio</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garfinkel</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Single-step selection for Ty1 element retrotransposition.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">936</span>–<span class="cit-lpage">940</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DCurcio%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D88%26rft.issue%253D3%26rft.spage%253D936%26rft.epage%253D940%26rft.atitle%253DSingle-step%2Bselection%2Bfor%2BTy1%2Belement%2Bretrotransposition.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.88.3.936%26rft_id%253Dinfo%253Apmid%252F1846969%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo4OiI4OC8zLzkzNiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi81OTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Curcio</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garfinkel</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Heterogeneous functional Ty1 elements are abundant in the <em>Saccharomyces cerevisiae</em> genome.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">136</span>: <span class="cit-fpage">1245</span>–<span class="cit-lpage">1259</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DCurcio%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D136%26rft.issue%253D4%26rft.spage%253D1245%26rft.epage%253D1259%26rft.atitle%253DHeterogeneous%2Bfunctional%2BTy1%2Belements%2Bare%2Babundant%2Bin%2Bthe%2BSaccharomyces%2Bcerevisiae%2Bgenome.%26rft_id%253Dinfo%253Apmid%252F8013902%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjEzNi80LzEyNDUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzIvNTkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Curcio</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lutz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lesage</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The Ty1 LTR-retrotransposon of budding yeast, <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Microbiol. Spectr.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">35</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiol.%2BSpectr.%26rft.volume%253D2%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.18" data-doi="10.1128/MCB.01095-07"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Curcio</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kenny</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garfinkel</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weintraub</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">S-phase checkpoint pathways stimulate the mobility of the retrovirus-like transposon Ty1.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">8874</span>–<span class="cit-lpage">8885</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.01095-07%26rft_id%253Dinfo%253Apmid%252F17923678%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyNy8yNC84ODc0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzU5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.19" data-doi="10.1534/genetics.110.119115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dakshinamurthy</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nyswaner</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farabaugh</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garfinkel</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">BUD22 affects Ty1 retrotransposition and ribosome biogenesis in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">1193</span>–<span class="cit-lpage">1205</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.119115%26rft_id%253Dinfo%253Apmid%252F20498295%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4NS80LzExOTMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzIvNTkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.20" data-doi="10.1371/journal.pgen.1000113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dean</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Pervasive and persistent redundancy among duplicated genes in yeast.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000113</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DDean%26rft.auinit1%253DE.%2BJ.%26rft.volume%253D4%26rft.issue%253D7%26rft.spage%253De1000113%26rft.epage%253De1000113%26rft.atitle%253DPervasive%2Band%2Bpersistent%2Bredundancy%2Bamong%2Bduplicated%2Bgenes%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000113%26rft_id%253Dinfo%253Apmid%252F18604285%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000113&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18604285&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F591.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.21" data-doi="10.1182/blood-2014-10-569616"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De Keersmaecker</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sulima</span>  <span class="cit-name-given-names">S. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dinman</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Ribosomopathies and the paradox of cellular hypo- to hyperproliferation.</span> <abbr class="cit-jnl-abbrev">Blood</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">1377</span>–<span class="cit-lpage">1382</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBlood%26rft_id%253Dinfo%253Adoi%252F10.1182%252Fblood-2014-10-569616%26rft_id%253Dinfo%253Apmid%252F25575543%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6ImJsb29kam91cm5hbCI7czo1OiJyZXNpZCI7czoxMDoiMTI1LzkvMTM3NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi81OTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.22" data-doi="10.1101/gad.10.5.620"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Devine</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boeke</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Integration of the yeast retrotransposon Ty1 is targeted to regions upstream of genes transcribed by RNA polymerase III.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">620</span>–<span class="cit-lpage">633</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DDevine%26rft.auinit1%253DS%2BE%26rft.volume%253D10%26rft.issue%253D5%26rft.spage%253D620%26rft.epage%253D633%26rft.atitle%253DIntegration%2Bof%2Bthe%2Byeast%2Bretrotransposon%2BTy1%2Bis%2Btargeted%2Bto%2Bregions%2Bupstream%2Bof%2Bgenes%2Btranscribed%2Bby%2BRNA%2Bpolymerase%2BIII.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.10.5.620%26rft_id%253Dinfo%253Apmid%252F8598291%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6ODoiMTAvNS82MjAiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzIvNTkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.23" data-doi="10.1016/j.jmb.2015.12.021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dinman</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Pathways to specialized ribosomes: the Brussels lecture.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">428</span>: <span class="cit-fpage">2186</span>–<span class="cit-lpage">2194</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMol.%2BBiol.%26rft.volume%253D428%26rft.spage%253D2186%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2015.12.021%26rft_id%253Dinfo%253Apmid%252F26764228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2015.12.021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26764228&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F591.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.24" data-doi="10.1371/journal.pgen.1004219"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doh</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lutz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Curcio</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Co-translational localization of an LTR-retrotransposon RNA to the endoplasmic reticulum nucleates virus-like particle assembly sites.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1004219</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004219%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004219%26rft_id%253Dinfo%253Apmid%252F24603646%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004219&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24603646&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F591.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.25" data-doi="10.1016/0092-8674(91)90387-E"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eng</span>  <span class="cit-name-given-names">F. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warner</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Structural basis for the regulation of splicing of a yeast messenger RNA.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">797</span>–<span class="cit-lpage">804</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DEng%26rft.auinit1%253DF.%2BJ.%26rft.volume%253D65%26rft.issue%253D5%26rft.spage%253D797%26rft.epage%253D804%26rft.atitle%253DStructural%2Bbasis%2Bfor%2Bthe%2Bregulation%2Bof%2Bsplicing%2Bof%2Ba%2Byeast%2Bmessenger%2BRNA.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252891%252990387-E%26rft_id%253Dinfo%253Apmid%252F2040015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(91)90387-E&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2040015&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F591.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991FP51600010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.26" data-doi="10.1128/MCB.19.1.826"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fewell</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woolford</span>  <span class="cit-name-given-names">J. L.</span>  <span class="cit-name-suffix">Jr</span>.</span></li></ol><cite>., <span class="cit-pub-date">1999</span> <span class="cit-article-title">Ribosomal protein S14 of <em>Saccharomyces cerevisiae</em> regulates its expression by binding to RPS14B pre-mRNA and to 18S rRNA.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">826</span>–<span class="cit-lpage">834</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DFewell%26rft.auinit1%253DS.%2BW.%26rft.volume%253D19%26rft.issue%253D1%26rft.spage%253D826%26rft.epage%253D834%26rft.atitle%253DRibosomal%2BProtein%2BS14%2Bof%2BSaccharomyces%2Bcerevisiae%2BRegulates%2BIts%2BExpression%2Bby%2BBinding%2Bto%2BRPS14B%2BPre-mRNA%2Band%2Bto%2B18S%2BrRNA%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.19.1.826%26rft_id%253Dinfo%253Apmid%252F9858605%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjg6IjE5LzEvODI2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzU5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.27" data-doi="10.1038/nature02046"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ghaemmaghami</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huh</span>  <span class="cit-name-given-names">W. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bower</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howson</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Belle</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Global analysis of protein expression in yeast.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">425</span>: <span class="cit-fpage">737</span>–<span class="cit-lpage">741</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGhaemmaghami%26rft.auinit1%253DS.%26rft.volume%253D425%26rft.issue%253D6959%26rft.spage%253D737%26rft.epage%253D741%26rft.atitle%253DGlobal%2Banalysis%2Bof%2Bprotein%2Bexpression%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02046%26rft_id%253Dinfo%253Apmid%252F14562106%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02046&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14562106&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F591.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185924500046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.28" data-doi="10.1016/j.tibs.2010.12.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gilbert</span>  <span class="cit-name-given-names">W. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Functional specialization of ribosomes?</span> <abbr class="cit-jnl-abbrev">Trends Biochem. Sci.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">127</span>–<span class="cit-lpage">132</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bbiochemical%2Bsciences%26rft.stitle%253DTrends%2BBiochem%2BSci%26rft.aulast%253DGilbert%26rft.auinit1%253DW.%2BV.%26rft.volume%253D36%26rft.issue%253D3%26rft.spage%253D127%26rft.epage%253D132%26rft.atitle%253DFunctional%2Bspecialization%2Bof%2Bribosomes%253F%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tibs.2010.12.002%26rft_id%253Dinfo%253Apmid%252F21242088%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tibs.2010.12.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21242088&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F591.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000288824300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Griffith</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coleman</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raymond</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodson</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pittard</span>  <span class="cit-name-given-names">W. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Functional genomics reveals relationships between the retrovirus-like Ty1 element and its host <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">164</span>: <span class="cit-fpage">867</span>–<span class="cit-lpage">879</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGriffith%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D164%26rft.issue%253D3%26rft.spage%253D867%26rft.epage%253D879%26rft.atitle%253DFunctional%2BGenomics%2BReveals%2BRelationships%2BBetween%2Bthe%2BRetrovirus-Like%2BTy1%2BElement%2Band%2BIts%2BHost%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Apmid%252F12871900%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY0LzMvODY3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzU5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.30" data-doi="10.1101/gad.1477507"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haarer</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Viggiano</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hibbs</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Troyanskaya</span>  <span class="cit-name-given-names">O. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amberg</span>  <span class="cit-name-given-names">D. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Modeling complex genetic interactions in a simple eukaryotic genome: actin displays a rich spectrum of complex haploinsufficiencies.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">148</span>–<span class="cit-lpage">159</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BDev.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1477507%26rft_id%253Dinfo%253Apmid%252F17167106%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6ODoiMjEvMi8xNDgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzIvNTkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.31" data-doi="10.1038/nprot.2009.115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haim-Vilmovsky</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerst</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">m-TAG: a PCR-based genomic integration method to visualize the localization of specific endogenous mRNAs <em>in vivo</em> in yeast.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1274</span>–<span class="cit-lpage">1284</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DHaim-Vilmovsky%26rft.auinit1%253DL.%26rft.volume%253D4%26rft.issue%253D9%26rft.spage%253D1274%26rft.epage%253D1284%26rft.atitle%253Dm-TAG%253A%2Ba%2BPCR-based%2Bgenomic%2Bintegration%2Bmethod%2Bto%2Bvisualize%2Bthe%2Blocalization%2Bof%2Bspecific%2Bendogenous%2BmRNAs%2Bin%2Bvivo%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2009.115%26rft_id%253Dinfo%253Apmid%252F19680241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2009.115&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19680241&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F591.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.32" data-doi="10.1006/viro.2001.0997"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harger</span>  <span class="cit-name-given-names">J. W. J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meskauskas</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Justice</span>  <span class="cit-name-given-names">M. C. M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dinman</span>  <span class="cit-name-given-names">J. D. J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Ty1 retrotransposition and programmed +1 ribosomal frameshifting require the integrity of the protein synthetic translocation step.</span> <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">286</span>: <span class="cit-fpage">216</span>–<span class="cit-lpage">224</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DHarger%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D286%26rft.issue%253D1%26rft.spage%253D216%26rft.epage%253D224%26rft.atitle%253DTy1%2Bretrotransposition%2Band%2Bprogrammed%2B%252B1%2Bribosomal%2Bframeshifting%2Brequire%2Bthe%2Bintegrity%2Bof%2Bthe%2Bprotein%2Bsynthetic%2Btranslocation%2Bstep.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fviro.2001.0997%26rft_id%253Dinfo%253Apmid%252F11448174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/viro.2001.0997&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11448174&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F591.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000170061600022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.33" data-doi="10.1016/0166-6851(88)90047-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hernandez</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Diaz-de Leon</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Castaneda</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Molecular cloning and partial characterization of ribosomal RNA genes from <em>Trypanosoma cruzi</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biochem. Parasitol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">275</span>–<span class="cit-lpage">279</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2Bbiochemical%2Bparasitology%26rft.stitle%253DMol%2BBiochem%2BParasitol%26rft.aulast%253DHern%25C3%25A1ndez%26rft.auinit1%253DR.%26rft.volume%253D27%26rft.issue%253D2-3%26rft.spage%253D275%26rft.epage%253D279%26rft.atitle%253DMolecular%2Bcloning%2Band%2Bpartial%2Bcharacterization%2Bof%2Bribosomal%2BRNA%2Bgenes%2Bfrom%2BTrypanosoma%2Bcruzi.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0166-6851%252888%252990047-3%26rft_id%253Dinfo%253Apmid%252F3278230%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0166-6851(88)90047-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3278230&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F591.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1988L772700019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.34" data-doi="10.1038/nrg2689"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Innan</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kondrashov</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The evolution of gene duplications: classifying and distinguishing between models.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">97</span>–<span class="cit-lpage">108</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DInnan%26rft.auinit1%253DH.%26rft.volume%253D11%26rft.issue%253D2%26rft.spage%253D97%26rft.epage%253D108%26rft.atitle%253DThe%2Bevolution%2Bof%2Bgene%2Bduplications%253A%2Bclassifying%2Band%2Bdistinguishing%2Bbetween%2Bmodels.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2689%26rft_id%253Dinfo%253Apmid%252F20051986%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2689&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20051986&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F591.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273688200004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.35" data-doi="10.1261/rna.032540.112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jakovljevic</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ohmayer</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gamalinda</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Talkish</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alexander</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Ribosomal proteins L7 and L8 function in concert with six A(3) assembly factors to propagate assembly of domains I and II of 25S rRNA in yeast 60S ribosomal subunits.</span> <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1805</span>–<span class="cit-lpage">1822</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.032540.112%26rft_id%253Dinfo%253Apmid%252F22893726%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjEwOiIxOC8xMC8xODA1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzU5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.36" data-doi="10.1016/0092-8674(93)90278-X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ji</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blomberg</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Braiterman</span>  <span class="cit-name-given-names">L. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voytas</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Hotspots for unselected Ty1 transposition events on yeast chromosome III are near tRNA genes and LTR sequences.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">1007</span>–<span class="cit-lpage">1018</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DJi%26rft.auinit1%253DH.%26rft.volume%253D73%26rft.issue%253D5%26rft.spage%253D1007%26rft.epage%253D1018%26rft.atitle%253DHotspots%2Bfor%2Bunselected%2BTy1%2Btransposition%2Bevents%2Bon%2Byeast%2Bchromosome%2BIII%2Bare%2Bnear%2BtRNA%2Bgenes%2Band%2BLTR%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252893%252990278-X%26rft_id%253Dinfo%253Apmid%252F8388781%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(93)90278-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8388781&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F591.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993LF06100016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.37" data-doi="10.1002/(SICI)1097-0061(199907)15:10B&lt;963::AID-YEA399&gt;3.0.CO;2-W"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Knop</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siegers</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pereira</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zachariae</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winsor</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Epitope tagging of yeast genes using a PCR-based strategy: more tags and improved practical routines.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">963</span>–<span class="cit-lpage">972</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DKnop%26rft.auinit1%253DM.%26rft.volume%253D15%26rft.issue%253D10B%26rft.spage%253D963%26rft.epage%253D972%26rft.atitle%253DEpitope%2Btagging%2Bof%2Byeast%2Bgenes%2Busing%2Ba%2BPCR-based%2Bstrategy%253A%2Bmore%2Btags%2Band%2Bimproved%2Bpractical%2Broutines.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F%2528SICI%25291097-0061%2528199907%252915%253A10B%253C963%253A%253AAID-YEA399%253E3.0.CO%253B2-W%26rft_id%253Dinfo%253Apmid%252F10407276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/(SICI)1097-0061(199907)15:10B&lt;963::AID-YEA399&gt;3.0.CO;2-W&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10407276&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F591.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000081644400007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.38" data-doi="10.1016/j.cell.2007.08.037"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Komili</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farny</span>  <span class="cit-name-given-names">N. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roth</span>  <span class="cit-name-given-names">F. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silver</span>  <span class="cit-name-given-names">P. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Functional specificity among ribosomal proteins regulates gene expression.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">131</span>: <span class="cit-fpage">557</span>–<span class="cit-lpage">571</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DKomili%26rft.auinit1%253DS.%26rft.volume%253D131%26rft.issue%253D3%26rft.spage%253D557%26rft.epage%253D571%26rft.atitle%253DFunctional%2Bspecificity%2Bamong%2Bribosomal%2Bproteins%2Bregulates%2Bgene%2Bexpression.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2007.08.037%26rft_id%253Dinfo%253Apmid%252F17981122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2007.08.037&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17981122&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F591.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250618500022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.39" data-doi="10.1016/j.cell.2011.03.028"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kondrashov</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pusic</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stumpf</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shimizu</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hsieh</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Ribosome-mediated specificity in Hox mRNA translation and vertebrate tissue patterning.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">145</span>: <span class="cit-fpage">383</span>–<span class="cit-lpage">397</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DKondrashov%26rft.auinit1%253DN.%26rft.volume%253D145%26rft.issue%253D3%26rft.spage%253D383%26rft.epage%253D397%26rft.atitle%253DRibosome-mediated%2Bspecificity%2Bin%2BHox%2BmRNA%2Btranslation%2Band%2Bvertebrate%2Btissue%2Bpatterning.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2011.03.028%26rft_id%253Dinfo%253Apmid%252F21529712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2011.03.028&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21529712&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F591.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000290022900008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.40" data-doi="10.1093/nar/13.14.5027"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leer</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Raamsdonk-Duin</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Molenaar</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Witsenboer</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mager</span>  <span class="cit-name-given-names">W. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Yeast contains two functional genes coding for ribosomal protein S10.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">5027</span>–<span class="cit-lpage">5039</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DLeer%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D13%26rft.issue%253D14%26rft.spage%253D5027%26rft.epage%253D5039%26rft.atitle%253DYeast%2Bcontains%2Btwo%2Bfunctional%2Bgenes%2Bcoding%2Bfor%2Bribosomal%2Bprotein%2BS10%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F13.14.5027%26rft_id%253Dinfo%253Apmid%252F2991849%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIxMy8xNC81MDI3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzU5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.41" data-doi="10.1128/MCB.15.11.6454"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paulovich</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woolford</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woolford</span>  <span class="cit-name-given-names">J. L.</span>  <span class="cit-name-suffix">Jr</span>.</span></li></ol><cite>., <span class="cit-pub-date">1995</span> <span class="cit-article-title">Feedback inhibition of the yeast ribosomal protein gene CRY2 is mediated by the nucleotide sequence and secondary structure of CRY2 pre-mRNA.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">6454</span>–<span class="cit-lpage">6464</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DLi%26rft.auinit1%253DZ.%26rft.volume%253D15%26rft.issue%253D11%26rft.spage%253D6454%26rft.epage%253D6464%26rft.atitle%253DFeedback%2Binhibition%2Bof%2Bthe%2Byeast%2Bribosomal%2Bprotein%2Bgene%2BCRY2%2Bis%2Bmediated%2Bby%2Bthe%2Bnucleotide%2Bsequence%2Band%2Bsecondary%2Bstructure%2Bof%2BCRY2%2Bpre-%2BmRNA%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.15.11.6454%26rft_id%253Dinfo%253Apmid%252F7565797%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIxNS8xMS82NDU0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzU5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.42" data-doi="10.1016/0014-5793(91)80030-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Localization of surface peptide from ribosomal protein L7 on 80 S ribosome by biotinylation.</span> <abbr class="cit-jnl-abbrev">FEBS Lett.</abbr> <span class="cit-vol">287</span>: <span class="cit-fpage">121</span>–<span class="cit-lpage">124</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BLetters%26rft.stitle%253DFEBS%2BLetters%26rft.aulast%253DLin%26rft.auinit1%253DA.%26rft.volume%253D287%26rft.issue%253D1-2%26rft.spage%253D121%26rft.epage%253D124%26rft.atitle%253DLocalization%2Bof%2Bsurface%2Bpeptide%2Bfrom%2Bribosomal%2Bprotein%2BL7%2Bon%2B80%2BS%2Bribosome%2Bby%2Bbiotinylation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0014-5793%252891%252980030-7%26rft_id%253Dinfo%253Apmid%252F1879519%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0014-5793(91)80030-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1879519&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F591.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.43" data-doi="10.1128/MCB.00684-08"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Malagon</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">T. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The T body, a new cytoplasmic RNA granule in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">6022</span>–<span class="cit-lpage">6032</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.00684-08%26rft_id%253Dinfo%253Apmid%252F18678648%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyOC8xOS82MDIyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzU5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.44" data-doi="10.1073/pnas.192442499"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mauro</span>  <span class="cit-name-given-names">V. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edelman</span>  <span class="cit-name-given-names">G. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">The ribosome filter hypothesis.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">12031</span>–<span class="cit-lpage">12036</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.192442499%26rft_id%253Dinfo%253Apmid%252F12221294%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTkvMTkvMTIwMzEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzIvNTkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.45" data-doi="10.4161/cc.6.18.4739"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mauro</span>  <span class="cit-name-given-names">V. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edelman</span>  <span class="cit-name-given-names">G. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The ribosome filter redux.</span> <abbr class="cit-jnl-abbrev">Cell Cycle</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">2246</span>–<span class="cit-lpage">2251</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2Bcycle%2B%2528Georgetown%252C%2BTex.%2529%26rft.stitle%253DCell%2BCycle%26rft.aulast%253DMauro%26rft.auinit1%253DV.%2BP.%26rft.volume%253D6%26rft.issue%253D18%26rft.spage%253D2246%26rft.epage%253D2251%26rft.atitle%253DThe%2Bribosome%2Bfilter%2Bredux.%26rft_id%253Dinfo%253Adoi%252F10.4161%252Fcc.6.18.4739%26rft_id%253Dinfo%253Apmid%252F17890902%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/cc.6.18.4739&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17890902&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F591.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251085600010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.46" data-doi="10.1126/science.1244156"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCann</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baserga</span>  <span class="cit-name-given-names">S. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Mysterious ribosomopathies.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">341</span>: <span class="cit-fpage">849</span>–<span class="cit-lpage">850</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMcCann%26rft.auinit1%253DK.%2BL.%26rft.volume%253D341%26rft.issue%253D6148%26rft.spage%253D849%26rft.epage%253D850%26rft.atitle%253DMysterious%2BRibosomopathies%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1244156%26rft_id%253Dinfo%253Apmid%252F23970686%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDEvNjE0OC84NDkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzIvNTkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.47" data-doi="10.1093/nar/20.5.1011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mizuta</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hashimoto</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Otaka</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Yeast ribosomal proteins: XIII. <em>Saccharomyces cerevisiae</em> YL8A gene, interrupted with two introns, encodes a homolog of mammalian L7.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1011</span>–<span class="cit-lpage">1016</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DMizuta%26rft.auinit1%253DK.%26rft.volume%253D20%26rft.issue%253D5%26rft.spage%253D1011%26rft.epage%253D1016%26rft.atitle%253DYeast%2Bribosomal%2Bproteins%253A%2BXIII.%2BSaccharomyces%2Bcerevisiae%2BYL8A%2Bgene%252C%2Binterrupted%2Bwith%2Btwo%2Bintrons%252C%2Bencodes%2Ba%2Bhomolog%2Bof%2Bmammalian%2BL7%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F20.5.1011%26rft_id%253Dinfo%253Apmid%252F1549461%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjIwLzUvMTAxMSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi81OTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.48" data-doi="10.1534/genetics.105.054072"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mou</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kenny</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Curcio</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Hos2 and Set3 promote integration of Ty1 retrotransposons at tRNA genes in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">2157</span>–<span class="cit-lpage">2167</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.054072%26rft_id%253Dinfo%253Apmid%252F16415356%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Mi80LzIxNTciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzIvNTkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.49" data-doi="10.1101/gr.129460.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mularoni</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gangadharan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wheelan</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Retrotransposon Ty1 integration targets specifically positioned asymmetric nucleosomal DNA segments in tRNA hotspots.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">693</span>–<span class="cit-lpage">703</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.129460.111%26rft_id%253Dinfo%253Apmid%252F22219510%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIyLzQvNjkzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzU5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.50" data-doi="10.1186/1471-2148-12-71"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Münk</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willemsen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bravo</span>  <span class="cit-name-given-names">I. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">An ancient history of gene duplications, fusions and losses in the evolution of APOBEC3 mutators in mammals.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">71</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DMunk%26rft.auinit1%253DC.%26rft.volume%253D12%26rft.spage%253D71%26rft.epage%253D71%26rft.atitle%253DAn%2Bancient%2Bhistory%2Bof%2Bgene%2Bduplications%252C%2Bfusions%2Band%2Blosses%2Bin%2Bthe%2Bevolution%2Bof%2BAPOBEC3%2Bmutators%2Bin%2Bmammals.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-12-71%26rft_id%253Dinfo%253Apmid%252F22640020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-12-71&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22640020&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F591.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.51" data-doi="10.1091/mbc.12.7.2147"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ni</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snyder</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">A genomic study of the bipolar bud site selection pattern in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">2147</span>–<span class="cit-lpage">2170</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Bof%2Bthe%2BCell%26rft.stitle%253DMol.%2BBiol.%2BCell%26rft.aulast%253DNi%26rft.auinit1%253DL.%26rft.volume%253D12%26rft.issue%253D7%26rft.spage%253D2147%26rft.epage%253D2170%26rft.atitle%253DA%2BGenomic%2BStudy%2Bof%2Bthe%2BBipolar%2BBud%2BSite%2BSelection%2BPattern%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.12.7.2147%26rft_id%253Dinfo%253Apmid%252F11452010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjEyLzcvMjE0NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi81OTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.52" data-doi="10.1128/MCB.15.5.2772"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ohtake</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wickner</span>  <span class="cit-name-given-names">R. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Yeast virus propagation depends critically on free 60S ribosomal subunit concentration.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">2772</span>–<span class="cit-lpage">2781</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DOhtake%26rft.auinit1%253DY.%26rft.volume%253D15%26rft.issue%253D5%26rft.spage%253D2772%26rft.epage%253D2781%26rft.atitle%253DYeast%2Bvirus%2Bpropagation%2Bdepends%2Bcritically%2Bon%2Bfree%2B60S%2Bribosomal%2Bsubunit%2Bconcentration%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.15.5.2772%26rft_id%253Dinfo%253Apmid%252F7739558%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjE1LzUvMjc3MiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi81OTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Page</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerard-Vincent</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Menard</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beaulieu</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Azuma</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">A <em>Saccharomyces cerevisiae</em> genome-wide mutant screen for altered sensitivity to K1 killer toxin.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">163</span>: <span class="cit-fpage">875</span>–<span class="cit-lpage">894</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPage%26rft.auinit1%253DN.%26rft.volume%253D163%26rft.issue%253D3%26rft.spage%253D875%26rft.epage%253D894%26rft.atitle%253DA%2BSaccharomyces%2Bcerevisiae%2BGenome-Wide%2BMutant%2BScreen%2Bfor%2BAltered%2BSensitivity%2Bto%2BK1%2BKiller%2BToxin%26rft_id%253Dinfo%253Apmid%252F12663529%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTYzLzMvODc1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzU5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.54" data-doi="10.1016/j.cell.2011.08.044"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parenteau</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durand</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morin</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gagnon</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lucier</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Introns within ribosomal protein genes regulate the production and function of yeast ribosomes.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">147</span>: <span class="cit-fpage">320</span>–<span class="cit-lpage">331</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DParenteau%26rft.auinit1%253DJ.%26rft.volume%253D147%26rft.issue%253D2%26rft.spage%253D320%26rft.epage%253D331%26rft.atitle%253DIntrons%2Bwithin%2Bribosomal%2Bprotein%2Bgenes%2Bregulate%2Bthe%2Bproduction%2Band%2Bfunction%2Bof%2Byeast%2Bribosomes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2011.08.044%26rft_id%253Dinfo%253Apmid%252F22000012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2011.08.044&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22000012&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F591.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000296199300011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.55" data-doi="10.1016/j.celrep.2015.11.033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parenteau</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lavoie</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Catala</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malik-Ghulam</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gagnon</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Preservation of gene duplication increases the regulatory spectrum of ribosomal protein genes and enhances growth under stress.</span> <abbr class="cit-jnl-abbrev">Cell Rep.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">2516</span>–<span class="cit-lpage">2526</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BRep.%26rft.volume%253D13%26rft.spage%253D2516%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.celrep.2015.11.033%26rft_id%253Dinfo%253Apmid%252F26686636%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.celrep.2015.11.033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26686636&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F591.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paulovich</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larkin</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woolford</span>  <span class="cit-name-given-names">J. L.</span>  <span class="cit-name-suffix">Jr</span>.</span></li></ol><cite>., <span class="cit-pub-date">1993</span> <span class="cit-article-title">Molecular genetics of cryptopleurine resistance in <em>Saccharomyces cerevisiae</em>: expression of a ribosomal protein gene family.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">135</span>: <span class="cit-fpage">719</span>–<span class="cit-lpage">730</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPaulovich%26rft.auinit1%253DA.%2BG.%26rft.volume%253D135%26rft.issue%253D3%26rft.spage%253D719%26rft.epage%253D730%26rft.atitle%253DMolecular%2Bgenetics%2Bof%2Bcryptopleurine%2Bresistance%2Bin%2BSaccharomyces%2Bcerevisiae%253A%2Bexpression%2Bof%2Ba%2Bribosomal%2Bprotein%2Bgene%2Bfamily.%26rft_id%253Dinfo%253Apmid%252F8293976%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTM1LzMvNzE5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzU5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.57" data-doi="10.1261/rna.373107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Piekna-Przybylska</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Decatur</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fournier</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">New bioinformatic tools for analysis of nucleotide modifications in eukaryotic rRNA.</span> <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">305</span>–<span class="cit-lpage">312</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.373107%26rft_id%253Dinfo%253Apmid%252F17283215%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjg6IjEzLzMvMzA1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzU5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Presutti</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ciafre</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bozzoni</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">The ribosomal protein L2 in <em>S. cerevisiae</em> controls the level of accumulation of its own mRNA.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">2215</span>–<span class="cit-lpage">2221</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BJ.%26rft.volume%253D10%26rft.spage%253D2215%26rft_id%253Dinfo%253Apmid%252F2065661%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=2065661&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F591.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.59" data-doi="10.1093/nar/gks983"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Purzycka</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legiewicz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matsuda</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eizentstat</span>  <span class="cit-name-given-names">L. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lusvarghi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Exploring Ty1 retrotransposon RNA structure within virus-like particles.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">463</span>–<span class="cit-lpage">473</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks983%26rft_id%253Dinfo%253Apmid%252F23093595%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjg6IjQxLzEvNDYzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzU5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.60"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rattray</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shafer</span>  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garfinkel</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The <em>Saccharomyces cerevisiae</em> DNA recombination and repair functions of the RAD52 epistasis group inhibit Ty1 transposition.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">154</span>: <span class="cit-fpage">543</span>–<span class="cit-lpage">556</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DRattray%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D154%26rft.issue%253D2%26rft.spage%253D543%26rft.epage%253D556%26rft.atitle%253DThe%2BSaccharomyces%2Bcerevisiae%2BDNA%2BRecombination%2Band%2BRepair%2BFunctions%2Bof%2Bthe%2BRAD52%2BEpistasis%2BGroup%2BInhibit%2BTy1%2BTransposition%26rft_id%253Dinfo%253Apmid%252F10655210%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU0LzIvNTQzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzU5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.61" data-doi="10.1186/1759-8753-3-12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Risler</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kenny</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palumbo</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gamache</span>  <span class="cit-name-given-names">E. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Curcio</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Host co-factors of the retrovirus-like transposon Ty1.</span> <abbr class="cit-jnl-abbrev">Mob. DNA</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DMob%2BDNA%26rft.aulast%253DRisler%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D3%26rft.issue%253D1%26rft.spage%253D12%26rft.epage%253D12%26rft.atitle%253DHost%2Bco-factors%2Bof%2Bthe%2Bretrovirus-like%2Btransposon%2BTy1.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1759-8753-3-12%26rft_id%253Dinfo%253Apmid%252F22856544%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1759-8753-3-12&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22856544&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F591.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.62" data-doi="10.1006/meth.1999.0901"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rosche</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foster</span>  <span class="cit-name-given-names">P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Determining mutation rates in bacterial populations.</span> <abbr class="cit-jnl-abbrev">Methods</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">4</span>–<span class="cit-lpage">17</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2B%2528San%2BDiego%252C%2BCalif.%2529%26rft.stitle%253DMethods%26rft.aulast%253DRosche%26rft.auinit1%253DW.%2BA.%26rft.volume%253D20%26rft.issue%253D1%26rft.spage%253D4%26rft.epage%253D17%26rft.atitle%253DDetermining%2Bmutation%2Brates%2Bin%2Bbacterial%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fmeth.1999.0901%26rft_id%253Dinfo%253Apmid%252F10610800%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/meth.1999.0901&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10610800&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F591.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000084627600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.63" data-doi="10.1101/gad.2.2.160"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rotenberg</span>  <span class="cit-name-given-names">M. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moritz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woolford</span>  <span class="cit-name-given-names">J. L.</span>  <span class="cit-name-suffix">Jr</span>.</span></li></ol><cite>., <span class="cit-pub-date">1988</span> <span class="cit-article-title">Depletion of <em>Saccharomyces cerevisiae</em> ribosomal protein L16 causes a decrease in 60S ribosomal subunits and formation of half-mer polyribosomes.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">160</span>–<span class="cit-lpage">172</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DRotenberg%26rft.auinit1%253DM%2BO%26rft.volume%253D2%26rft.issue%253D2%26rft.spage%253D160%26rft.epage%253D172%26rft.atitle%253DDepletion%2Bof%2BSaccharomyces%2Bcerevisiae%2Bribosomal%2Bprotein%2BL16%2Bcauses%2Ba%2Bdecrease%2Bin%2B60S%2Bribosomal%2Bsubunits%2Band%2Bformation%2Bof%2Bhalf-mer%2Bpolyribosomes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.2.2.160%26rft_id%253Dinfo%253Apmid%252F3282992%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6NzoiMi8yLzE2MCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi81OTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.64" data-doi="10.1128/JVI.03060-14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saha</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nishida</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hildreth</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ariberre</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A trans-dominant form of Gag restricts Ty1 retrotransposition and mediates copy number control.</span> <abbr class="cit-jnl-abbrev">J. Virol.</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">3922</span>–<span class="cit-lpage">3938</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.03060-14%26rft_id%253Dinfo%253Apmid%252F25609815%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijg5LzcvMzkyMiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi81OTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.65" data-doi="10.4161/rna.7.6.14117"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sandmeyer</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clemens</span>  <span class="cit-name-given-names">K. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Function of a retrotransposon nucleocapsid protein.</span> <abbr class="cit-jnl-abbrev">RNA Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">642</span>–<span class="cit-lpage">654</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DRNA%2BBiol%26rft.aulast%253DSandmeyer%26rft.auinit1%253DS.%2BB.%26rft.volume%253D7%26rft.issue%253D6%26rft.spage%253D642%26rft.epage%253D654%26rft.atitle%253DFunction%2Bof%2Ba%2Bretrotransposon%2Bnucleocapsid%2Bprotein.%26rft_id%253Dinfo%253Adoi%252F10.4161%252Frna.7.6.14117%26rft_id%253Dinfo%253Apmid%252F21189452%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/rna.7.6.14117&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21189452&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F591.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000288490500003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.66"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scholes</span>  <span class="cit-name-given-names">D. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banerjee</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Curcio</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Multiple regulators of Ty1 transposition in <em>Saccharomyces cerevisiae</em> have conserved roles in genome maintenance.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">159</span>: <span class="cit-fpage">1449</span>–<span class="cit-lpage">1465</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DScholes%26rft.auinit1%253DD.%2BT.%26rft.volume%253D159%26rft.issue%253D4%26rft.spage%253D1449%26rft.epage%253D1465%26rft.atitle%253DMultiple%2BRegulators%2Bof%2BTy1%2BTransposition%2Bin%2BSaccharomyces%2Bcerevisiae%2BHave%2BConserved%2BRoles%2Bin%2BGenome%2BMaintenance%26rft_id%253Dinfo%253Apmid%252F11779788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1OS80LzE0NDkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzIvNTkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.67" data-doi="10.1073/pnas.2136609100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scholes</span>  <span class="cit-name-given-names">D. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kenny</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gamache</span>  <span class="cit-name-given-names">E. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mou</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Curcio</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Activation of a LTR-retrotransposon by telomere erosion.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">15736</span>–<span class="cit-lpage">15741</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.2136609100%26rft_id%253Dinfo%253Apmid%252F14673098%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAwLzI2LzE1NzM2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzU5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.68" data-doi="10.1128/MCB.14.10.6540"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sharon</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burkett</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garfinkel</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Efficient homologous recombination of Ty1 element cDNA when integration is blocked.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">6540</span>–<span class="cit-lpage">6551</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DSharon%26rft.auinit1%253DG%26rft.volume%253D14%26rft.issue%253D10%26rft.spage%253D6540%26rft.epage%253D6551%26rft.atitle%253DEfficient%2Bhomologous%2Brecombination%2Bof%2BTy1%2Belement%2BcDNA%2Bwhen%2Bintegration%2Bis%2Bblocked.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.14.10.6540%26rft_id%253Dinfo%253Apmid%252F7523854%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIxNC8xMC82NTQwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzU5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.69"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sikorski</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hieter</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">A system of shuttle vectors and yeast host strains designed for efficient manipulation of DNA in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">122</span>: <span class="cit-fpage">19</span>–<span class="cit-lpage">27</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSikorski%26rft.auinit1%253DR.%2BS.%26rft.volume%253D122%26rft.issue%253D1%26rft.spage%253D19%26rft.epage%253D27%26rft.atitle%253DA%2Bsystem%2Bof%2Bshuttle%2Bvectors%2Band%2Byeast%2Bhost%2Bstrains%2Bdesigned%2Bfor%2Befficient%2Bmanipulation%2Bof%2BDNA%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Apmid%252F2659436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTIyLzEvMTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzIvNTkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.70" data-doi="10.1007/s00294-009-0228-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simoff</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moradi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nygard</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Functional characterization of ribosomal protein L15 from <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Curr. Genet.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">111</span>–<span class="cit-lpage">125</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bgenetics%26rft.stitle%253DCurr%2BGenet%26rft.aulast%253DSimoff%26rft.auinit1%253DI.%26rft.volume%253D55%26rft.issue%253D2%26rft.spage%253D111%26rft.epage%253D125%26rft.atitle%253DFunctional%2Bcharacterization%2Bof%2Bribosomal%2Bprotein%2BL15%2Bfrom%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00294-009-0228-z%26rft_id%253Dinfo%253Apmid%252F19184027%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00294-009-0228-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19184027&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F591.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265092000002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.71"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Slavov</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Semrau</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Airoldi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Budnik</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Oudenaarden</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Differential stoichiometry among core ribosomal proteins.</span> <abbr class="cit-jnl-abbrev">Cell Rep.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">865</span>–<span class="cit-lpage">873</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BRep.%26rft.volume%253D13%26rft.spage%253D865%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.72" data-doi="10.1534/genetics.109.104208"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stamenova</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maxwell</span>  <span class="cit-name-given-names">P. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kenny</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Curcio</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Rrm3 protects the <em>Saccharomyces cerevisiae</em> genome from instability at nascent sites of retrotransposition.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">711</span>–<span class="cit-lpage">723</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.104208%26rft_id%253Dinfo%253Apmid%252F19414561%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzMvNzExIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzU5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.73" data-doi="10.1016/j.cell.2008.02.037"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Steffen</span>  <span class="cit-name-given-names">K. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">MacKay</span>  <span class="cit-name-given-names">V. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kerr</span>  <span class="cit-name-given-names">E. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsuchiya</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Yeast life span extension by depletion of 60s ribosomal subunits is mediated by Gcn4.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">133</span>: <span class="cit-fpage">292</span>–<span class="cit-lpage">302</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DSteffen%26rft.auinit1%253DK.%2BK.%26rft.volume%253D133%26rft.issue%253D2%26rft.spage%253D292%26rft.epage%253D302%26rft.atitle%253DYeast%2Blife%2Bspan%2Bextension%2Bby%2Bdepletion%2Bof%2B60s%2Bribosomal%2Bsubunits%2Bis%2Bmediated%2Bby%2BGcn4.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2008.02.037%26rft_id%253Dinfo%253Apmid%252F18423200%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2008.02.037&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18423200&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F591.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255052000018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.74" data-doi="10.1534/genetics.111.136549"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Steffen</span>  <span class="cit-name-given-names">K. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCormick</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pham</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">MacKay</span>  <span class="cit-name-given-names">V. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delaney</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Ribosome deficiency protects against ER stress in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">191</span>: <span class="cit-fpage">107</span>–<span class="cit-lpage">118</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.136549%26rft_id%253Dinfo%253Apmid%252F22377630%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkxLzEvMTA3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzU5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.75" data-doi="10.1016/j.ymeth.2006.06.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Storici</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resnick</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The <em>delitto perfetto</em> approach to <em>in vivo</em> site-directed mutagenesis and chromosome rearrangements with synthetic oligonucleotides in yeast.</span> <abbr class="cit-jnl-abbrev">Methods Enzymol.</abbr> <span class="cit-vol">409</span>: <span class="cit-fpage">329</span>–<span class="cit-lpage">345</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Benzymology%26rft.stitle%253DMethods%2BEnzymol%26rft.aulast%253DStorici%26rft.auinit1%253DF.%26rft.volume%253D409%26rft.spage%253D329%26rft.epage%253D345%26rft.atitle%253DThe%2Bdelitto%2Bperfetto%2Bapproach%2Bto%2Bin%2Bvivo%2Bsite-directed%2Bmutagenesis%2Band%2Bchromosome%2Brearrangements%2Bwith%2Bsynthetic%2Boligonucleotides%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ymeth.2006.06.011%26rft_id%253Dinfo%253Apmid%252F16793410%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ymeth.2006.06.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16793410&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F591.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238354800019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.76" data-doi="10.1186/s13100-015-0053-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Suresh</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ahn</span>  <span class="cit-name-given-names">H. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joshi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dakshinamurthy</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kananganat</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Ribosomal protein and biogenesis factors affect multiple steps during movement of the <em>Saccharomyces cerevisiae</em> Ty1 retrotransposon.</span> <abbr class="cit-jnl-abbrev">Mob. DNA</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMob.%2BDNA%26rft.volume%253D6%26rft.spage%253D22%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13100-015-0053-5%26rft_id%253Dinfo%253Apmid%252F26664557%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13100-015-0053-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26664557&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F591.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.77"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsai</span>  <span class="cit-name-given-names">Y. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">H. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Ribosome distribution in HeLa cells during the cell cycle.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e32820</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DTsai%26rft.auinit1%253DY.%2BJ.%26rft.volume%253D7%26rft.issue%253D3%26rft.spage%253De32820%26rft.epage%253De32820%26rft.atitle%253DRibosome%2Bdistribution%2Bin%2BHeLa%2Bcells%2Bduring%2Bthe%2Bcell%2Bcycle.%26rft_id%253Dinfo%253Apmid%252F22403714%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22403714&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F591.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.78" data-doi="10.1261/rna.251907"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vincenti</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Chiara</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bozzoni</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Presutti</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The position of yeast snoRNA-coding regions within host introns is essential for their biosynthesis and for efficient splicing of the host pre-mRNA.</span> <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">138</span>–<span class="cit-lpage">150</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.251907%26rft_id%253Dinfo%253Apmid%252F17135484%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjg6IjEzLzEvMTM4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzU5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.79" data-doi="10.1101/cshperspect.a011536"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">D. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doudna Cate</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The structure and function of the eukaryotic ribosome.</span> <abbr class="cit-jnl-abbrev">Cold Spring Harb. Perspect. Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">a011536</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCold%2BSpring%2BHarbor%2BPerspectives%2Bin%2BBiology%26rft.stitle%253DCold%2BSpring%2BHarb.%2BPerspect.%2BBiol.%26rft.aulast%253DWilson%26rft.auinit1%253DD.%2BN.%26rft.volume%253D4%26rft.issue%253D5%26rft.spage%253Da011536%26rft.epage%253Da011536%26rft.atitle%253DThe%2BStructure%2Band%2BFunction%2Bof%2Bthe%2BEukaryotic%2BRibosome%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fcshperspect.a011536%26rft_id%253Dinfo%253Apmid%252F22550233%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6ImNzaHBlcnNwZWN0IjtzOjU6InJlc2lkIjtzOjExOiI0LzUvYTAxMTUzNiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi81OTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.80" data-doi="10.1038/42711"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">K. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shields</span>  <span class="cit-name-given-names">D. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Molecular evidence for an ancient duplication of the entire yeast genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">387</span>: <span class="cit-fpage">708</span>–<span class="cit-lpage">713</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DWolfe%26rft.auinit1%253DK.%2BH.%26rft.volume%253D387%26rft.issue%253D6634%26rft.spage%253D708%26rft.epage%253D713%26rft.atitle%253DMolecular%2Bevidence%2Bfor%2Ban%2Bancient%2Bduplication%2Bof%2Bthe%2Bentire%2Byeast%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F42711%26rft_id%253Dinfo%253Apmid%252F9192896%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/42711&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9192896&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F591.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XD86900055&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.81" data-doi="10.1534/genetics.113.153197"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Woolford</span>  <span class="cit-name-given-names">J. L.</span>  <span class="cit-name-suffix">Jr</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baserga</span>  <span class="cit-name-given-names">S. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Ribosome biogenesis in the yeast <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">195</span>: <span class="cit-fpage">643</span>–<span class="cit-lpage">681</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWoolford%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D195%26rft.issue%253D3%26rft.spage%253D643%26rft.epage%253D681%26rft.atitle%253DRibosome%2BBiogenesis%2Bin%2Bthe%2BYeast%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.153197%26rft_id%253Dinfo%253Apmid%252F24190922%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk1LzMvNjQzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzU5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.82" data-doi="10.1038/nrm3359"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xue</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barna</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Specialized ribosomes: a new frontier in gene regulation and organismal biology.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Mol. Cell Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">355</span>–<span class="cit-lpage">369</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMolecular%2Bcell%2Bbiology%26rft.stitle%253DNat%2BRev%2BMol%2BCell%2BBiol%26rft.aulast%253DXue%26rft.auinit1%253DS.%26rft.volume%253D13%26rft.issue%253D6%26rft.spage%253D355%26rft.epage%253D369%26rft.atitle%253DSpecialized%2Bribosomes%253A%2Ba%2Bnew%2Bfrontier%2Bin%2Bgene%2Bregulation%2Band%2Borganismal%2Bbiology.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrm3359%26rft_id%253Dinfo%253Apmid%252F22617470%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrm3359&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22617470&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F591.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.83" data-doi="10.1038/nature14010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xue</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fujii</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kladwang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Das</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">RNA regulons in Hox 5′ UTRs confer ribosome specificity to gene regulation.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">517</span>: <span class="cit-fpage">33</span>–<span class="cit-lpage">38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D517%26rft.spage%253D33%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature14010%26rft_id%253Dinfo%253Apmid%252F25409156%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature14010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25409156&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F591.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.591.84" data-doi="10.1534/genetics.111.136648"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yarrington</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richardson</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lisa Huang</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boeke</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Novel transcript truncating function of Rap1p revealed by synthetic codon-optimized Ty1 retrotransposon.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">523</span>–<span class="cit-lpage">535</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.136648%26rft_id%253Dinfo%253Apmid%252F22135353%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzIvNTIzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzU5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/2/591.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/2/583" title="Single Locus Maintains Large Variation of Sex Reversal in Half-Smooth Tongue Sole (Cynoglossus semilaevis)" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/2/607" title="The Bright Fluorescent Protein mNeonGreen Facilitates Protein Expression Analysis In Vivo" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/2"><span>Volume 7 Issue 2, February 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/2.cover-source.jpg?itok=m4uMC43I" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/2/591" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-OVw_FdMAEQWuh-A3vYGJjbHDW1cAF66a3JfxpRdmcEI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/2/591" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/2/591" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/2/591" class="active" data-icon-position="" data-hide-link-title="0">Paralog-Specific Functions of RPL7A and RPL7B Mediated by Ribosomal Protein or snoRNA Dosage in Saccharomyces cerevisiae</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460584" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-TYxproTU_HWFM3KgH0TyGSgTqnUeD6tgYAUFzrLTU60" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/2/591.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/2/591" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-eAOwCCp5ENSm8JxtEUCn7khhvHeJjhkWv32IvUZuDLA" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/2/591" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/2/591.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460584" id="node460584--2" data-pisa="ggg;7/2/591" data-pisa-master="ggg;g3.116.035931" data-apath="/ggg/7/2/591.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Paralog-Specific Functions of <em>RPL7A</em> and <em>RPL7B</em> Mediated by Ribosomal Protein or snoRNA Dosage in <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Ryan J. Palumbo</span>, <span class="highwire-citation-author" data-delta="1">Gabriele Fuchs</span>, <span class="highwire-citation-author" data-delta="2">Sheila Lutz</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-5361-3909" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>M. Joan Curcio</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">591-606;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.035931 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460584--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ryan J. Palumbo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory of Molecular Genetics, Wadsworth Center, New York State Department of Health, Albany, New York 12201</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biomedical Sciences, School of Public Health, University at Albany, New York 12201</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ryan%2BJ.%2BPalumbo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Palumbo%20RJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARyan%2BJ.%2BPalumbo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gabriele Fuchs </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biological Sciences, The RNA Institute, University at Albany-State University of New York, New York 12222</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gabriele%2BFuchs%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fuchs%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGabriele%2BFuchs%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Sheila Lutz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory of Molecular Genetics, Wadsworth Center, New York State Department of Health, Albany, New York 12201</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sheila%2BLutz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lutz%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASheila%2BLutz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">M. Joan Curcio </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory of Molecular Genetics, Wadsworth Center, New York State Department of Health, Albany, New York 12201</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biomedical Sciences, School of Public Health, University at Albany, New York 12201</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M.%2BJoan%2BCurcio%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Curcio%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AM.%2BJoan%2BCurcio%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5361-3909" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for M. Joan Curcio</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:joan.curcio@health.ny.gov" class="" data-icon-position="" data-hide-link-title="0">joan.curcio@health.ny.gov</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/2/591" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/2/591&amp;title=Paralog-Specific%20Functions%20of%20RPL7A%20and%20RPL7B%20Mediated%20by%20Ribosomal%20Protein%20or%20snoRNA%20Dosage%20in%20Saccharomyces%20cerevisiae" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/2/591&amp;title=Paralog-Specific%20Functions%20of%20RPL7A%20and%20RPL7B%20Mediated%20by%20Ribosomal%20Protein%20or%20snoRNA%20Dosage%20in%20Saccharomyces%20cerevisiae" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/2/591&amp;title=Paralog-Specific%20Functions%20of%20RPL7A%20and%20RPL7B%20Mediated%20by%20Ribosomal%20Protein%20or%20snoRNA%20Dosage%20in%20Saccharomyces%20cerevisiae" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/2/591&amp;text=Paralog-Specific%20Functions%20of%20RPL7A%20and%20RPL7B%20Mediated%20by%20Ribosomal%20Protein%20or%20snoRNA%20Dosage%20in%20Saccharomyces%20cerevisiae" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/2/591&amp;title=Paralog-Specific%20Functions%20of%20RPL7A%20and%20RPL7B%20Mediated%20by%20Ribosomal%20Protein%20or%20snoRNA%20Dosage%20in%20Saccharomyces%20cerevisiae" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/2/591&amp;t=Paralog-Specific%20Functions%20of%20RPL7A%20and%20RPL7B%20Mediated%20by%20Ribosomal%20Protein%20or%20snoRNA%20Dosage%20in%20Saccharomyces%20cerevisiae" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/2/591&amp;title=Paralog-Specific%20Functions%20of%20RPL7A%20and%20RPL7B%20Mediated%20by%20Ribosomal%20Protein%20or%20snoRNA%20Dosage%20in%20Saccharomyces%20cerevisiae" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/2/591&amp;title=Paralog-Specific%20Functions%20of%20RPL7A%20and%20RPL7B%20Mediated%20by%20Ribosomal%20Protein%20or%20snoRNA%20Dosage%20in%20Saccharomyces%20cerevisiae" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460584/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460584" id="node460584--3" data-pisa="ggg;7/2/591" data-pisa-master="ggg;g3.116.035931" data-apath="/ggg/7/2/591.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Paralog-Specific Functions of <em>RPL7A</em> and <em>RPL7B</em> Mediated by Ribosomal Protein or snoRNA Dosage in <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Ryan J. Palumbo</span>, <span class="highwire-citation-author" data-delta="1">Gabriele Fuchs</span>, <span class="highwire-citation-author" data-delta="2">Sheila Lutz</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-5361-3909" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>M. Joan Curcio</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">591-606;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.035931 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460584--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ryan J. Palumbo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory of Molecular Genetics, Wadsworth Center, New York State Department of Health, Albany, New York 12201</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biomedical Sciences, School of Public Health, University at Albany, New York 12201</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ryan%2BJ.%2BPalumbo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Palumbo%20RJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARyan%2BJ.%2BPalumbo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gabriele Fuchs </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biological Sciences, The RNA Institute, University at Albany-State University of New York, New York 12222</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gabriele%2BFuchs%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fuchs%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGabriele%2BFuchs%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Sheila Lutz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory of Molecular Genetics, Wadsworth Center, New York State Department of Health, Albany, New York 12201</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sheila%2BLutz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lutz%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASheila%2BLutz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">M. Joan Curcio </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory of Molecular Genetics, Wadsworth Center, New York State Department of Health, Albany, New York 12201</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biomedical Sciences, School of Public Health, University at Albany, New York 12201</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M.%2BJoan%2BCurcio%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Curcio%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AM.%2BJoan%2BCurcio%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5361-3909" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for M. Joan Curcio</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:joan.curcio@health.ny.gov" class="" data-icon-position="" data-hide-link-title="0">joan.curcio@health.ny.gov</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=591&amp;rft.epage=606&amp;rft.atitle=Paralog-Specific%20Functions%20of%20RPL7A%20and%20RPL7B%20Mediated%20by%20Ribosomal%20Protein%20or%20snoRNA%20Dosage%20in%20Saccharomyces%20cerevisiae&amp;rft.volume=7&amp;rft.issue=2&amp;rft.date=2017-02-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Palumbo%2C+Ryan+J.&amp;rft.au=Fuchs%2C+Gabriele&amp;rft.au=Lutz%2C+Sheila&amp;rft.au=Curcio%2C+M.+Joan"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460584/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460584/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460584/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460584/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460584/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460584/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460584/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460584/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460584/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460584/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460584/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460584/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/2/591" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-22" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/2/591.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/2/591.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/2/591.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/2/591.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
