<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/2/449" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/2/449.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/2/449.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/2/449.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/2/449.atom" />
<meta name="HW.pisa" content="ggg;7/2/449" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Alternative Oxidase Transcription Factors AOD2 and AOD5 of Neurospora crassa Control the Expression of Genes Involved in Energy Production and Metabolism" />
<meta name="DC.Identifier" content="10.1534/g3.116.035402" />
<meta name="DC.Date" content="2017-02-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Qi et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="In Neurospora crassa , blocking the function of the standard mitochondrial electron transport chain results in the induction of an alternative oxidase (AOX). AOX transfers electrons directly from ubiquinol to molecular oxygen. AOX serves as a model of retrograde regulation since it is encoded by a nuclear gene that is regulated in response to signals from mitochondria. The N. crassa transcription factors AOD2 and AOD5 are necessary for the expression of the AOX gene. To gain insight into the mechanism by which these factors function, and to determine if they have roles in the expression of additional genes in N. crassa , we constructed strains expressing only tagged versions of the proteins. Cell fractionation experiments showed that both proteins are localized to the nucleus under both AOX inducing and noninducing conditions. Furthermore, chromatin immunoprecipitation and high throughput sequencing (ChIP-seq) analysis revealed that the proteins are bound to the promoter region of the AOX gene under both conditions. ChIP-seq also showed that the transcription factors bind to the upstream regions of a number of genes that are involved in energy production and metabolism. Dependence on AOD2 and AOD5 for the expression of several of these genes was verified by quantitative PCR. The majority of ChIP-seq peaks observed were enriched for both AOD2 and AOD5. However, we also observed occasional sites where one factor appeared to bind preferentially. The most striking of these was a conserved sequence that bound large amounts of AOD2 but little AOD5. This sequence was found within a 310 bp repeat unit that occurs at several locations in the genome." />
<meta name="DC.Contributor" content="Zhigang Qi" />
<meta name="DC.Contributor" content="Kristina M. Smith" />
<meta name="DC.Contributor" content="Erin L. Bredeweg" />
<meta name="DC.Contributor" content="Natasa Bosnjak" />
<meta name="DC.Contributor" content="Michael Freitag" />
<meta name="DC.Contributor" content="Frank E. Nargang" />
<meta name="article:published_time" content="2017-02-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Alternative Oxidase Transcription Factors AOD2 and AOD5 of Neurospora crassa Control the Expression of Genes Involved in Energy Production and Metabolism" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;In &lt;i&gt;Neurospora crassa&lt;/i&gt;, blocking the function of the standard mitochondrial electron transport chain results in the induction of an alternative oxidase (AOX). AOX transfers electrons directly from ubiquinol to molecular oxygen. AOX serves as a model of retrograde regulation since it is encoded by a nuclear gene that is regulated in response to signals from mitochondria. The &lt;i&gt;N. crassa&lt;/i&gt; transcription factors AOD2 and AOD5 are necessary for the expression of the AOX gene. To gain insight into the mechanism by which these factors function, and to determine if they have roles in the expression of additional genes in &lt;i&gt;N. crassa&lt;/i&gt;, we constructed strains expressing only tagged versions of the proteins. Cell fractionation experiments showed that both proteins are localized to the nucleus under both AOX inducing and noninducing conditions. Furthermore, chromatin immunoprecipitation and high throughput sequencing (ChIP-seq) analysis revealed that the proteins are bound to the promoter region of the AOX gene under both conditions. ChIP-seq also showed that the transcription factors bind to the upstream regions of a number of genes that are involved in energy production and metabolism. Dependence on AOD2 and AOD5 for the expression of several of these genes was verified by quantitative PCR. The majority of ChIP-seq peaks observed were enriched for both AOD2 and AOD5. However, we also observed occasional sites where one factor appeared to bind preferentially. The most striking of these was a conserved sequence that bound large amounts of AOD2 but little AOD5. This sequence was found within a 310 bp repeat unit that occurs at several locations in the genome.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/02/01" />
<meta name="citation_mjid" content="ggg;7/2/449" />
<meta name="citation_id" content="7/2/449" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/2/449" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/2/449.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/2/449.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/2/449.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.035402" />
<meta name="citation_pmid" content="27986792" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="2" />
<meta name="citation_num_pages" content="18" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="449" />
<meta name="citation_lastpage" content="466" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Zhigang Qi" />
<meta name="citation_author_institution" content="Department of Biological Sciences, University of Alberta, Edmonton, Alberta T6G 2E9, Canada" />
<meta name="citation_author" content="Kristina M. Smith" />
<meta name="citation_author_institution" content="Department of Biochemistry and Biophysics, Oregon State University, Corvallis, Oregon 97331-4003" />
<meta name="citation_author" content="Erin L. Bredeweg" />
<meta name="citation_author_institution" content="Department of Biochemistry and Biophysics, Oregon State University, Corvallis, Oregon 97331-4003" />
<meta name="citation_author" content="Natasa Bosnjak" />
<meta name="citation_author_institution" content="Department of Biological Sciences, University of Alberta, Edmonton, Alberta T6G 2E9, Canada" />
<meta name="citation_author" content="Michael Freitag" />
<meta name="citation_author_institution" content="Department of Biochemistry and Biophysics, Oregon State University, Corvallis, Oregon 97331-4003" />
<meta name="citation_author" content="Frank E. Nargang" />
<meta name="citation_author_institution" content="Department of Biological Sciences, University of Alberta, Edmonton, Alberta T6G 2E9, Canada" />
<meta name="citation_author_email" content="frank.nargang@ualberta.ca" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1383-4720" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=Y. Arthanari;citation_author=C. Heintzen;citation_author=S. Griffiths-Jones;citation_author=S. K. Crosthwaite;citation_title=Natural antisense transcripts and long non-coding RNA in Neurospora crassa.;citation_pages=e91353;citation_volume=9;citation_year=2014;citation_issue=3;citation_pmid=24621812;citation_doi=10.1371/journal.pone.0091353" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=U. Baumgartner;citation_author=B. Hamilton;citation_author=M. Piskacek;citation_author=H. Ruis;citation_author=H. Rottensteiner;citation_title=Functional Analysis of the Zn2Cys6Transcription Factors Oaf1p and Pip2p: DIFFERENT ROLES IN FATTY ACID INDUCTION OF {beta}-OXIDATION INSACCHAROMYCES CEREVISIAE;citation_pages=22208-22216;citation_volume=274;citation_year=1999;citation_issue=32;citation_pmid=10428786;citation_doi=10.1074/jbc.274.32.22208" />
<meta name="citation_reference" content="citation_title=Genetic control of the biogenesis of cyanide insensitive respiration in Neurospora crassa;citation_author=H. Bertrand;citation_author=A. Argan;citation_author=N. A. Szakacs;citation_pages=495-507;citation_year=1983" />
<meta name="citation_reference" content="citation_title=Mitochondrial Function and Biogenetics, Topics in Current Genetics;citation_author=R. B. Bevan;citation_author=B. F. Lang;citation_title=Mitochondrial genome evolution: the origin of mitochondria and of eukaryotes;citation_pages=1-35;citation_volume=Vol. 8;citation_year=2004" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=C. Borghouts;citation_author=A. Werner;citation_author=T. E. Elthon;citation_author=H. D. Osiewacz;citation_title=Copper-Modulated Gene Expression and Senescence in the Filamentous Fungus Podospora anserina;citation_pages=390-399;citation_volume=21;citation_year=2001;citation_issue=2;citation_pmid=11134328;citation_doi=10.1128/MCB.21.2.390-399.2001" />
<meta name="citation_reference" content="citation_journal_title=Eukaryot. Cell;citation_author=E. Bovier;citation_author=C. H. Sellem;citation_author=A. Humbert;citation_author=A. Sainsard-Chanet;citation_title=Genetic and functional investigation of Zn(2)Cys(6) transcription factors RSE2 and RSE3 in Podospora anserina.;citation_volume=13;citation_year=2014;citation_issue=1;citation_pmid=24186951;citation_doi=10.1128/EC.00172-13" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=G. Burger;citation_author=M. W. Gray;citation_author=L. Forget;citation_author=B. F. Lang;citation_title=Strikingly bacteria-like and gene-rich mitochondrial genomes throughout jakobid protists.;citation_volume=5;citation_year=2013;citation_issue=2;citation_pmid=23335123;citation_doi=10.1093/gbe/evt008" />
<meta name="citation_reference" content="citation_journal_title=Physiologia Plantarum (Copenhagen. 1948);citation_journal_abbrev=Physiologia Plantarum (Copenhagen. 1948);citation_author=M. S. Chae;citation_author=F. E. Nargang;citation_title=Investigation of regulatory factors required for alternative oxidase production in Neurospora crassa.;citation_pages=407-418;citation_volume=137;citation_year=2009;citation_issue=4;citation_pmid=19493307;citation_doi=10.1111/j.1399-3054.2009.01239.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. S. Chae;citation_author=C. C. Lin;citation_author=K. E. Kessler;citation_author=C. E. Nargang;citation_author=L. L. Tanton;citation_title=Identification of an alternative oxidase induction motif in the promoter region of the aod-1 gene in Neurospora crassa.;citation_volume=175;citation_year=2007;citation_pmid=17237510;citation_doi=10.1534/genetics.106.068635" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. S. Chae;citation_author=C. E. Nargang;citation_author=I. A. Cleary;citation_author=C. C. Lin;citation_author=A. T. Todd;citation_title=Two Zinc-Cluster Transcription Factors Control Induction of Alternative Oxidase in Neurospora crassa;citation_pages=1997-2006;citation_volume=177;citation_year=2007;citation_issue=4;citation_pmid=18073419;citation_doi=10.1534/genetics.107.078212" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=A. Chakravarty;citation_author=J. M. Carlson;citation_author=R. S. Khetani;citation_author=R. H. Gross;citation_title=A novel ensemble learning method for de novo computational identification of DNA binding sites.;citation_pages=249-249;citation_volume=8;citation_year=2007;citation_pmid=17626633;citation_doi=10.1186/1471-2105-8-249" />
<meta name="citation_reference" content="citation_journal_title=Oxid. Med. Cell. Longev.;citation_author=F. M. da Cunha;citation_author=N. Q. Torelli;citation_author=A. J. Kowaltowski;citation_title=Mitochondrial retrograde signaling: triggers, pathways, and outcomes.;citation_pages=482582;citation_volume=2015;citation_year=2015" />
<meta name="citation_reference" content="citation_title=Nitrogen metabolism in filamentous fungi;citation_author=M. A. Davis;citation_author=K. H. Wong;citation_pages=325-338;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Methods Enzymol.;citation_author=R. H. Davis;citation_author=F. J. De Serres;citation_title=Genetic and microbiological research techniques for Neurospora crassa.;citation_pages=79-143;citation_volume=17;citation_year=1970;citation_doi=10.1016/0076-6879(71)17168-6" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. T. Descheneau;citation_author=I. A. Cleary;citation_author=F. E. Nargang;citation_title=Genetic evidence for a regulatory pathway controlling alternative oxidase production in Neurospora crassa.;citation_volume=169;citation_year=2005;citation_pmid=15466423;citation_doi=10.1534/genetics.104.034017" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=E. Dufour;citation_author=J. Boulay;citation_author=V. Rincheval;citation_author=A. Sainsard-Chanet;citation_title=A causal link between respiration and senescence in Podospora anserina.;citation_volume=97;citation_year=2000;citation_pmid=10759557;citation_doi=10.1073/pnas.070501997" />
<meta name="citation_reference" content="citation_title=Genetics and Biogenesis of Chloroplasts and Mitochondria;citation_title=Assembly of the cyanide-insensitive respiratory pathway in Neurospora crassa;citation_author=D. L. Edwards;citation_author=J. H. Chalmers;citation_author=H. J. Guzik;citation_author=J. T. Warden;citation_pages=865-872;citation_year=1976" />
<meta name="citation_reference" content="citation_journal_title=Methods in enzymology;citation_journal_abbrev=Methods Enzymol;citation_author=M. Elstner;citation_author=C. Andreoli;citation_author=T. Klopstock;citation_author=T. Meitinger;citation_author=H. Prokisch;citation_title=The mitochondrial proteome database: MitoP2.;citation_pages=3-20;citation_volume=457;citation_year=2009;citation_pmid=19426859;citation_doi=10.1016/S0076-6879(09)05001-0" />
<meta name="citation_reference" content="citation_journal_title=Molecular aspects of medicine;citation_journal_abbrev=Mol Aspects Med;citation_author=H. J. Forman;citation_author=H. Zhang;citation_author=A. Rinna;citation_title=Glutathione: overview of its protective roles, measurement, and biosynthesis.;citation_pages=1-12;citation_volume=30;citation_year=2009;citation_issue=1–2;citation_pmid=18796312;citation_doi=10.1016/j.mam.2008.08.006" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=E. Gladyshev;citation_author=N. Kleckner;citation_title=Direct recognition of homology between double helices of DNA in Neurospora crassa.;citation_pages=3509;citation_volume=5;citation_year=2014;citation_pmid=24699390" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiology;citation_journal_abbrev=Plant Physiol.;citation_author=A. G. Good;citation_author=W. L. Crosby;citation_title=Anaerobic Induction of Alanine Aminotransferase in Barley Root Tissue;citation_pages=1305-1309;citation_volume=90;citation_year=1989;citation_issue=4;citation_pmid=16666927;citation_doi=10.1104/pp.90.4.1305" />
<meta name="citation_reference" content="citation_journal_title=Methods in molecular biology (Clifton, N.J.);citation_journal_abbrev=Methods Mol Biol;citation_author=J. Guo;citation_author=G. Huang;citation_author=J. Cha;citation_author=Y. Liu;citation_title=Biochemical methods used to study the gene expression and protein complexes in the filamentous fungus Neurospora crassa.;citation_pages=189-200;citation_volume=638;citation_year=2010;citation_pmid=20238270" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. C. Hoppins;citation_author=N. E. Go;citation_author=A. Klein;citation_author=S. Schmitt;citation_author=W. Neupert;citation_title=Alternative splicing gives rise to different isoforms of the Neurospora crassa Tob55 protein that vary in their ability to insert β−barrel proteins into the outer mitochondrial membrane.;citation_volume=177;citation_year=2007;citation_pmid=17660559;citation_doi=10.1534/genetics.107.075051" />
<meta name="citation_reference" content="citation_journal_title=Biochemical Journal;citation_journal_abbrev=Biochem. J.;citation_author=W. Huh;citation_author=S. Kang;citation_title=Characterization of the gene family encoding alternative oxidase from Candida albicans;citation_pages=595-604;citation_volume=356;citation_year=2001;citation_issue=2;citation_pmid=11368790;citation_doi=10.1042/bj3560595" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=D. Jain;citation_author=S. Baldi;citation_author=A. Zabel;citation_author=T. Straub;citation_author=P. B. Becker;citation_title=Active promoters give rise to false positive ‘Phantom Peaks’ in ChIP-seq experiments.;citation_volume=43;citation_year=2015;citation_issue=14;citation_pmid=26117547;citation_doi=10.1093/nar/gkv637" />
<meta name="citation_reference" content="citation_journal_title=Biochim. Biophys. Acta;citation_author=S. M. Jazwinski;citation_title=The retrograde response: when mitochondrial quality control is not enough.;citation_pages=400-409;citation_volume=1833;citation_year=2013;citation_issue=2;citation_doi=10.1016/j.bbamcr.2012.02.010" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=I. V. Karpichev;citation_author=G. M. Small;citation_title=Global Regulatory Functions of Oaf1p and Pip2p (Oaf2p), Transcription Factors That Regulate Genes Encoding Peroxisomal Proteins in Saccharomyces cerevisiae;citation_pages=6560-6570;citation_volume=18;citation_year=1998;citation_issue=11;citation_pmid=9774671;citation_doi=10.1128/MCB.18.11.6560" />
<meta name="citation_reference" content="citation_journal_title=Molecular and cellular biochemistry;citation_journal_abbrev=Mol Cell Biochem;citation_author=G. Kikuchi;citation_author=K. Hiraga;citation_title=The mitochondrial glycine cleavage system. Unique features of the glycine decarboxylation.;citation_pages=137-149;citation_volume=45;citation_year=1982;citation_issue=3;citation_pmid=6750353" />
<meta name="citation_reference" content="citation_journal_title=FEMS Microbiol. Lett.;citation_author=K. Kirimura;citation_author=T. Matsui;citation_author=S. Sugano;citation_author=S. Usami;citation_title=Enhancement and repression of cyanide-insensitive respiration in Aspergillus niger.;citation_volume=141;citation_year=1996;citation_pmid=8768530;citation_doi=10.1111/j.1574-6968.1996.tb08393.x" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=C. Kramer;citation_author=J. J. Loros;citation_author=J. C. Dunlap;citation_author=S. K. Crosthwaite;citation_title=Role for antisense RNA in regulating circadian clock function in Neurospora crassa.;citation_pages=948-952;citation_volume=421;citation_year=2003;citation_issue=6926;citation_pmid=12607002;citation_doi=10.1038/nature01427" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=U. K. Laemmli;citation_title=Cleavage of structural proteins during the assembly of the head of bacteriophage T4.;citation_pages=680-685;citation_volume=227;citation_year=1970;citation_issue=5259;citation_pmid=5432063;citation_doi=10.1038/227680a0" />
<meta name="citation_reference" content="citation_journal_title=Methods Enzymol.;citation_author=A. M. Lambowitz;citation_title=Preparation and analysis of mitochondrial ribosomes.;citation_pages=421-433;citation_volume=54;citation_year=1979" />
<meta name="citation_reference" content="citation_journal_title=The international journal of biochemistry &amp; cell biology;citation_journal_abbrev=Int J Biochem Cell Biol;citation_author=G. Lenaz;citation_author=M. L. Genova;citation_title=Structural and functional organization of the mitochondrial respiratory chain: a dynamic super-assembly.;citation_pages=1750-1772;citation_volume=41;citation_year=2009;citation_issue=10;citation_pmid=19711505;citation_doi=10.1016/j.biocel.2009.04.003" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_volume=25;citation_year=2009;citation_issue=14;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=X. Liao;citation_author=R. A. Butow;citation_title=RTG1 and RTG2: two yeast genes required for a novel path of communication from mitochondria to the nucleus.;citation_pages=61-71;citation_volume=72;citation_year=1993;citation_issue=1;citation_pmid=8422683;citation_doi=10.1016/0092-8674(93)90050-Z" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=Z. Liu;citation_author=R. A. Butow;citation_title=A Transcriptional Switch in the Expression of Yeast Tricarboxylic Acid Cycle Genes in Response to a Reduction or Loss of Respiratory Function;citation_pages=6720-6728;citation_volume=19;citation_year=1999;citation_issue=10;citation_pmid=10490611;citation_doi=10.1128/MCB.19.10.6720" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=Z. Liu;citation_author=R. A. Butow;citation_title=Mitochondrial retrograde signaling.;citation_pages=159-185;citation_volume=40;citation_year=2006;citation_pmid=16771627;citation_doi=10.1146/annurev.genet.40.110405.090613" />
<meta name="citation_reference" content="citation_journal_title=Microbiology and Molecular Biology Reviews;citation_journal_abbrev=Microbiol. Mol. Biol. Rev.;citation_author=S. MacPherson;citation_author=M. Larochelle;citation_author=B. Turcotte;citation_title=A Fungal Family of Transcriptional Regulators: the Zinc Cluster Proteins;citation_pages=583-604;citation_volume=70;citation_year=2006;citation_issue=3;citation_pmid=16959962;citation_doi=10.1128/MMBR.00015-06" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=B. Magasanik;citation_author=C. A. Kaiser;citation_title=Nitrogen regulation in Saccharomyces cerevisiae.;citation_pages=1-18;citation_volume=290;citation_year=2002;citation_issue=1–2;citation_pmid=12062797;citation_doi=10.1016/S0378-1119(02)00558-9" />
<meta name="citation_reference" content="citation_journal_title=Microbiology and Molecular Biology Reviews;citation_journal_abbrev=Microbiol. Mol. Biol. Rev.;citation_author=G. A. Marzluf;citation_title=Genetic regulation of nitrogen metabolism in the fungi;citation_pages=17-32;citation_volume=61;citation_year=1997;citation_issue=1;citation_pmid=9106362" />
<meta name="citation_reference" content="citation_journal_title=Funct. Plant Biol.;citation_author=A. E. McDonald;citation_title=Alternative oxidase: an inter-kingdom perspective on the function and regulation of this broadly distributed “cyanide-resistant” terminal oxidase.;citation_pages=535-552;citation_volume=35;citation_year=2008;citation_doi=10.1071/FP08025" />
<meta name="citation_reference" content="citation_journal_title=Comp. Biochem. Physiol. Part D;citation_author=A. E. McDonald;citation_author=G. C. Vanlerberghe;citation_title=Origins, evolutionary history, and taxonomic distribution of alternative oxidase and plastoquinol terminal oxidase.;citation_pages=357-364;citation_volume=1;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Plant molecular biology;citation_journal_abbrev=Plant Mol Biol;citation_author=A. E. McDonald;citation_author=S. Amirsadeghi;citation_author=G. C. Vanlerberghe;citation_title=Prokaryotic orthologues of mitochondrial alternative oxidase and plastid terminal oxidase.;citation_pages=865-876;citation_volume=53;citation_year=2003;citation_issue=6;citation_pmid=15082931;citation_doi=10.1023/B:PLAN.0000023669.79465.d2" />
<meta name="citation_reference" content="citation_journal_title=Journal of Experimental Biology;citation_journal_abbrev=J. Exp. Biol.;citation_author=A. E. McDonald;citation_author=G. C. Vanlerberghe;citation_author=J. F. Staples;citation_title=Alternative oxidase in animals: unique characteristics and taxonomic distribution;citation_pages=2627-2634;citation_volume=212;citation_year=2009;citation_issue=Pt 16;citation_pmid=19648408;citation_doi=10.1242/jeb.032151" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=C. Meisinger;citation_author=A. Sickmann;citation_author=N. Pfanner;citation_title=The mitochondrial proteome: from inventory to function.;citation_pages=22-24;citation_volume=134;citation_year=2008;citation_issue=1;citation_pmid=18614007;citation_doi=10.1016/j.cell.2008.06.043" />
<meta name="citation_reference" content="citation_journal_title=Fungal Genet. Newsl.;citation_author=R. L. Metzenberg;citation_title=Vogel’s medium N salts: avoiding the need for ammonium nitrate.;citation_pages=14;citation_volume=50;citation_year=2003" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Plant Biol.;citation_author=A. L. Moore;citation_author=T. Shiba;citation_author=L. Young;citation_author=S. Harada;citation_author=K. Kita;citation_title=Unraveling the heater: new insights into the structure of the alternative oxidase.;citation_pages=637-663;citation_volume=64;citation_year=2013;citation_pmid=23638828;citation_doi=10.1146/annurev-arplant-042811-105432" />
<meta name="citation_reference" content="citation_title=Neurospora crassa as a model organism for mitochondrial biogenesis;citation_author=F. E. Nargang;citation_author=D. Rapaport;citation_pages=107-123;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_author=K. Neimanis;citation_author=J. F. Staples;citation_author=N. P. Huner;citation_author=A. E. McDonald;citation_title=Identification, expression, and taxonomic distribution of alternative oxidases in non-angiosperm plants.;citation_pages=275-286;citation_volume=526;citation_year=2013;citation_issue=2" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=S. Priebe;citation_author=C. Kreisel;citation_author=F. Horn;citation_author=R. Guthke;citation_author=J. Linde;citation_title=FungiFun2: a comprehensive online resource for systematic analysis of gene lists from fungal species.;citation_volume=31;citation_year=2015;citation_issue=3;citation_pmid=25294921;citation_doi=10.1093/bioinformatics/btu627" />
<meta name="citation_reference" content="citation_journal_title=Front. Pharmacol.;citation_author=V. Ribas;citation_author=C. Garcia-Ruiz;citation_author=J. C. Fernandez-Checa;citation_title=Glutathione and mitochondria.;citation_pages=151;citation_volume=5;citation_year=2014;citation_pmid=25024695" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=A. Ruepp;citation_author=A. Zollner;citation_author=D. Maier;citation_author=K. Albermann;citation_author=J. Hani;citation_title=The FunCat, a functional annotation scheme for systematic classification of proteins from whole genomes;citation_pages=5539-5545;citation_volume=32;citation_year=2004;citation_issue=18;citation_pmid=15486203;citation_doi=10.1093/nar/gkh894" />
<meta name="citation_reference" content="citation_journal_title=FEBS Letters;citation_journal_abbrev=FEBS Letters;citation_author=S. Sakajo;citation_author=N. Minagawa;citation_author=A. Yoshimoto;citation_title=Characterization of the alternative oxidase protein in the yeast Hansenula anomala.;citation_pages=310-312;citation_volume=318;citation_year=1993;citation_issue=3;citation_pmid=8440388;citation_doi=10.1016/0014-5793(93)80535-3" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=S. Schmitt;citation_author=U. Ahting;citation_author=L. Eichacker;citation_author=B. Granvogl;citation_author=N. E. Go;citation_title=Role of Tom5 in maintaining the structural stability of the TOM complex of mitochondria.;citation_volume=280;citation_year=2005;citation_pmid=15701639;citation_doi=10.1074/jbc.M413667200" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=E. U. Selker;citation_title=Premeiotic Instability Of Repeated Sequences In Neurospora Crassa;citation_pages=579-613;citation_volume=24;citation_year=1990;citation_issue=1;citation_pmid=2150906;citation_doi=10.1146/annurev.ge.24.120190.003051" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. H. Sellem;citation_author=E. Bovier;citation_author=S. Lorin;citation_author=A. Sainsard-Chanet;citation_title=Mutations in two zinc cluster proteins activate alternative respiratory and gluconeogenic pathways and restore senescence in long-lived respiratory mutants of Podospora anserina.;citation_volume=182;citation_year=2009;citation_pmid=19255367;citation_doi=10.1534/genetics.109.100834" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=E. L. Sherman;citation_author=N. E. Go;citation_author=F. E. Nargang;citation_title=Functions of the small proteins in the TOM complex of Neurospora crassa.;citation_volume=16;citation_year=2005;citation_pmid=15987740;citation_doi=10.1091/mbc.E05-03-0187" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=N.-Q. Shi;citation_author=B. Davis;citation_author=F. Sherman;citation_author=J. Cruz;citation_author=T. W. Jeffries;citation_title=Disruption of the cytochrome c gene in xylose-utilizing yeast Pichia stipitis leads to higher ethanol production.;citation_pages=1021-1030;citation_volume=15;citation_year=1999;citation_issue=11;citation_pmid=10455226;citation_doi=10.1002/(SICI)1097-0061(199908)15:11&lt;1021::AID-YEA429&gt;3.0.CO;2-V" />
<meta name="citation_reference" content="citation_journal_title=Curr. Genet.;citation_author=V. Solotoff;citation_author=R. Moseler;citation_author=U. Schulte;citation_title=Two pentatricopeptide repeat domain proteins are required for the synthesis of respiratory complex I.;citation_pages=19-29;citation_volume=61;citation_year=2015;citation_issue=1" />
<meta name="citation_reference" content="citation_journal_title=Fungal Genet. Newsl.;citation_author=C. Staben;citation_author=B. Jensen;citation_author=M. Singer;citation_author=J. Pollock;citation_author=M. Schechtman;citation_title=Use of bacterial hygromycin B resistance gene as a dominant selectable marker in Neurospora crassa transformation.;citation_pages=79-81;citation_volume=36;citation_year=1989" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=L. D. Stein;citation_author=C. Mungall;citation_author=S. Shu;citation_author=M. Caudy;citation_author=M. Mangone;citation_title=The Generic Genome Browser: A Building Block for a Model Organism System Database;citation_pages=1599-1610;citation_volume=12;citation_year=2002;citation_issue=10;citation_pmid=12368253;citation_doi=10.1101/gr.403602" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=Y. Suzuki;citation_author=S. L. Murray;citation_author=K. H. Wong;citation_author=M. A. Davis;citation_author=M. J. Hynes;citation_title=Reprogramming of carbon metabolism by the transcriptional activators AcuK and AcuM in Aspergillus nidulans.;citation_pages=942-964;citation_volume=84;citation_year=2012;citation_issue=5;citation_pmid=22500966;citation_doi=10.1111/j.1365-2958.2012.08067.x" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiology;citation_journal_abbrev=Plant Physiol.;citation_author=K. J. Talbot;citation_author=P. J. Russell;citation_title=Nuclear Buoyant Density Determination and the Purification and Characterization of Wild-Type Neurospora Nuclei Using Percoll Density Gradients;citation_pages=704-708;citation_volume=70;citation_year=1982;citation_issue=3;citation_pmid=16662561;citation_doi=10.1104/pp.70.3.704" />
<meta name="citation_reference" content="citation_journal_title=Fungal genetics and biology :  FG &amp; B;citation_journal_abbrev=Fungal Genet Biol;citation_author=L. L. Tanton;citation_author=C. E. Nargang;citation_author=K. E. Kessler;citation_author=Q. Li;citation_author=F. E. Nargang;citation_title=Alternative oxidase expression in Neurospora crassa.;citation_pages=176-190;citation_volume=39;citation_year=2003;citation_issue=2;citation_pmid=12781676;citation_doi=10.1016/S1087-1845(03)00002-1" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=A. Tissieres;citation_author=H. K. Mitchell;citation_author=F. A. Haskins;citation_title=STUDIES ON THE RESPIRATORY SYSTEM OF THE POKY STRAIN OF NEUROSPORA;citation_pages=423-433;citation_volume=205;citation_year=1953;citation_issue=1;citation_pmid=13117920" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. K. Watters;citation_author=T. A. Randall;citation_author=B. S. Margolin;citation_author=E. U. Selker;citation_author=D. R. Stadler;citation_title=Action of Repeat-Induced Point Mutation on Both Strands of a Duplex and on Tandem Duplications of Various Sizes in Neurospora;citation_pages=705-714;citation_volume=153;citation_year=1999;citation_issue=2;citation_pmid=10511550" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=R. Worsley Hunt;citation_author=W. W. Wasserman;citation_title=Non-targeted transcription factors motifs are a systemic component of ChIP-seq datasets.;citation_pages=412;citation_volume=15;citation_year=2014;citation_issue=7;citation_pmid=25070602;citation_doi=10.1186/s13059-014-0412-4" />
<meta name="citation_reference" content="citation_journal_title=Biochimica et Biophysica Acta. Protein Structure and Molecular Enzymology;citation_journal_abbrev=Biochimica et Biophysica Acta. Protein Structure and Molecular Enzymology;citation_author=H. Yukioka;citation_author=S. Inagaki;citation_author=R. Tanaka;citation_author=K. Katoh;citation_author=N. Miki;citation_title=Transcriptional activation of the alternative oxidase gene of the fungus Magnaporthe grisea by a respiratory-inhibiting fungicide and hydrogen peroxide.;citation_pages=161-169;citation_volume=1442;citation_year=1998;citation_issue=2-3;citation_pmid=9804939" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=Y. Zhang;citation_author=T. Liu;citation_author=C. A. Meyer;citation_author=J. Eeckhoute;citation_author=D. S. Johnson;citation_title=Model-based analysis of ChIP-Seq (MACS).;citation_pages=R137-R137;citation_volume=9;citation_year=2008;citation_issue=9;citation_pmid=18798982;citation_doi=10.1186/gb-2008-9-9-r137" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Alternative Oxidase Transcription Factors AOD2 and AOD5 of Neurospora crassa Control the Expression of Genes Involved in Energy Production and Metabolism" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2.cover-source.jpg" />
<meta name="twitter:description" content="In Neurospora crassa , blocking the function of the standard mitochondrial electron transport chain results in the induction of an alternative oxidase (AOX). AOX transfers electrons directly from ubiquinol to molecular oxygen. AOX serves as a model of retrograde regulation since it is encoded by a nuclear gene that is regulated in response to signals from mitochondria. The N. crassa transcription factors AOD2 and AOD5 are necessary for the expression of the AOX gene. To gain insight into the mechanism by which these factors function, and to determine if they have roles in the expression of additional genes in N. crassa , we constructed strains expressing only tagged versions of the proteins. Cell fractionation experiments showed that both proteins are localized to the nucleus under both AOX inducing and noninducing conditions. Furthermore, chromatin immunoprecipitation and high throughput sequencing (ChIP-seq) analysis revealed that the proteins are bound to the promoter region of the AOX gene under both conditions. ChIP-seq also showed that the transcription factors bind to the upstream regions of a number of genes that are involved in energy production and metabolism. Dependence on AOD2 and AOD5 for the expression of several of these genes was verified by quantitative PCR. The majority of ChIP-seq peaks observed were enriched for both AOD2 and AOD5. However, we also observed occasional sites where one factor appeared to bind preferentially. The most striking of these was a conserved sequence that bound large amounts of AOD2 but little AOD5. This sequence was found within a 310 bp repeat unit that occurs at several locations in the genome." />
<meta name="og-title" property="og:title" content="Alternative Oxidase Transcription Factors AOD2 and AOD5 of Neurospora crassa Control the Expression of Genes Involved in Energy Production and Metabolism" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/2/449" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="In Neurospora crassa , blocking the function of the standard mitochondrial electron transport chain results in the induction of an alternative oxidase (AOX). AOX transfers electrons directly from ubiquinol to molecular oxygen. AOX serves as a model of retrograde regulation since it is encoded by a nuclear gene that is regulated in response to signals from mitochondria. The N. crassa transcription factors AOD2 and AOD5 are necessary for the expression of the AOX gene. To gain insight into the mechanism by which these factors function, and to determine if they have roles in the expression of additional genes in N. crassa , we constructed strains expressing only tagged versions of the proteins. Cell fractionation experiments showed that both proteins are localized to the nucleus under both AOX inducing and noninducing conditions. Furthermore, chromatin immunoprecipitation and high throughput sequencing (ChIP-seq) analysis revealed that the proteins are bound to the promoter region of the AOX gene under both conditions. ChIP-seq also showed that the transcription factors bind to the upstream regions of a number of genes that are involved in energy production and metabolism. Dependence on AOD2 and AOD5 for the expression of several of these genes was verified by quantitative PCR. The majority of ChIP-seq peaks observed were enriched for both AOD2 and AOD5. However, we also observed occasional sites where one factor appeared to bind preferentially. The most striking of these was a conserved sequence that bound large amounts of AOD2 but little AOD5. This sequence was found within a 310 bp repeat unit that occurs at several locations in the genome." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2.cover-source.jpg" />
    <title>Alternative Oxidase Transcription Factors AOD2 and AOD5 of Neurospora crassa Control the Expression of Genes Involved in Energy Production and Metabolism | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"SGIG5ShVOJ4vUrqoJplZrciP2AluqL45dkY11ckng7M","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460549","apath":"\/ggg\/7\/2\/449.atom","pisa":"ggg;7\/2\/449","ac":{"\/ggg\/7\/2\/449.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/2\/449.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/2\/449"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/2\/449"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/2\/449":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022icEIvqnUQn6uO08yecGmfyHJ7Rq0sI+dPtS\\\\\\\/x5P+KWrFzZ3LF0+qFMqUQr3+yOLUDBLb13Ohr5iHVl10fynsS+aUDxzAWS8M+iLAv5G1AwG+m1h4WEEfHFaIxEJBK2Q40gvboviYOkvEdAXlsamNjtOXeX1dz8fzsQnxbYcuyT0qCsgJkVrfN53olS8bKGjFOLXDFQqHEvNP2Cmm3CCrz\\\\\\\/CmxKaRTiTM5tiEqDAqireL7AGn6TI9P0J2sLA+QbcHq\\\\\\\/2nVRInRl7QiC5UIEv8wN\\\\\\\/+oDkmL3GGeUYf39YMBhdIP\\\\\\\/fMM\\\\\\\/1LMFUrluX+HhIe+MM7gkpktBan0OUkmN\\\\\\\/WqvaoFU8h8OHLmyR\\\\\\\/N7\\\\\\\/Lm\\\\\\\/GgaZ2kFIYPMG6cA7ywobHRiBAfmmo+5s+\\\\\\\/bLAzUyDrE\\\\\\\/Vb8ky0MfIz885tfmOQklJKpJ69Izwc8L5OnOnNziQ7Yy7X1sfQ4TOVtlA1zZpY9qpH6abs2yxzeW7MME5lYoM0uJ4We6L\\\\\\\/fBx1NV2UCIbCnSB76QbLy+GHvt3ZmFlMJi1ymOCFk7flVPkZp6fN99koav5xzflmdfu6HH66O3fPyOgGHTX6ErgvLol9aiV7EVUb9ijV2IYU\\\\\\\/qVxcTbn08iYr\\\\\\\/lYYhhlHzEt5H9GDDIGtuEKtBmR4yArLlOdKfH7OZSMBLs9afJ3dRHNDlXlGzGUS4LOXDKWLiILKwWNhz7LPDTLM0J7oGSrsAXEy6r0iW5UIOsImDUSRncZQHimZ1m6V7BNTM4YLLFi4W7KH5FPk3DcoCoUUAmpE1WMP9pv8jp\\\\\\\/eUb+9jFGWEtXO9NHoSSIx2v6IPeRQGDaQDgNrwPJ2AXBsWzclTNBAX0gYfyY+S3DVfn3IobrMsE2166rW7sRfOhJmzL3ZzDZ1Xw2At66o8CUV4+aEnACWIrlam+22XARvy7OrirY8a7vl9bWqenUGnnlXS8Ozf14g4szwS+ZzvZ4UJKvi4pjXYrsAU043c9OewwCg5RaZdwInxTTIZbAVMiyhsyOgV9Yo\\\\\\\/Kn2dQIBd6xPH\\\\\\\/a9wZBV8YL7rnraaSIADzX\\\\\\\/YR1kobAPeS6PAaViOqaYEgGfiqg8akEktxb247xIq+Cf0cJOPo9Q2xydNq51E9B3FEey\\\\\\\/uQzA9vXOx7yKwkU4nbVHJ7HeVChFlRYMda\\\\\\\/G645\\\\\\\/xj6hMxq0FAbqs91IaBUINU6zJViaiHgfHVHbMR+KKaCjYnLFqp3b6J6Jae9VG2NfkYL\\\\\\\/LNmdx2naOLHwo0CT5Nfy2u29fiATlBnCJ8tGNYom6W5SigtLq5\\\u0022,\\\u0022iv\\\u0022:\\\u00220N9eSDSd6MiF\\\\\\\/tdWa08HbQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022daee7b61581d975fc36e897202230492d5669e0cc26c17c78f01c098cb29217e\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022O3q\\\\\\\/4YCG7eYwGx+D3e820gsG7pWeA8DNHCmrjxHuHSUAI0V0zPcLWEImSeOllvbeq9YXb7EpfQAetxXntR9IcD+FO9Q7F28daqUMwmvjDbTHYolxTnfyphnslBIYOC7ITC8JTWKM7XZwVv2nz5cILOLss96ZHd8PNxh8OQsash8SdavJElcosPUXXRRjcHdNiPO7geS3Gr9IR91kVEWk\\\\\\\/Sug6p7XAlXISm5eNrUPR\\\\\\\/LDIHEYDm\\\\\\\/BvnK6dxLXYPVjAu5QdcWy7mO1mtioVqctQg6o7\\\\\\\/ZPHoDp28xOb+8n4DhBEnD1I\\\\\\\/TVJv4lxm5TWeIldwYLAMZL8inYZTeD0CD9eX1gm8Uqdhmmdpauy\\\\\\\/R1icsYMAR4bAPINuWI0Sd\\\\\\\/KUobqCpj0Fv9xqhsDuvcC17J9w4bTPLT\\\\\\\/Kf7XiZpCPkpMv2izj8tq9Q3t7BiXSvZdL5jE0W6\\\\\\\/467taggJSRES+HckqoNoxpeBcr3bcLnAyPgxqSsWwNeVGIhK\\\\\\\/MeF2YT8EgeS7UEeZwFble68FNXjPhl7GJ3QmxvXS36Nkcv7tWiiSoOE69b5edB7NOdJC41q4MF69q1d2p11rJBwCYNCcana7oms4k3eJVZY9y\\\\\\\/sJPcUCaBc1FDQuIIgDjs8Z\\\\\\\/jyxAsf+I3\\\\\\\/H0ccB4NpWLgxfPTGySxziUiCFP4wARRH4\\\\\\\/KR0C8JFxXoOE3LNSJpe0PRPFtDpZSP1aAlTXuT7YRgzXTf22TsQzl7UI9gyEH0y0xr3z4bVVSbw8aBw6hAYSaIBxcTJ23qakPc+ZInR8gEVFsLpPP8Ak2bzm6ZbIuBdwuQlLB9SEMyjcFJ+4YVZfGQkYG0BPdfic+1jQXooaGviJ1qSKFDtWSFuoi+kfKSToO\\\\\\\/B9O7tThx4ROBEI4zz9H\\\\\\\/NSYjtATtOCdcHURhmv6cRU6uaEgpnbDutd985sRMA2ELaLqkSpQCzH1zPW0pl2rsFeEj20NJLyd\\\\\\\/PGFGiBVgs7j69HAthLIq3OQFaHp5QxEPGbNpLusBZtLgaB34Vowow2fnCKmfcJaKTxcSrdbaGxwdS3zEVBXm4wXz7Ik1ZWFNFQFwrLynstg331OU1Ysjg3mMY0NPCkLVwaozwiIE3ozdMwh6keFau8C0VMsgYpbodKj3SsH2\\\\\\\/0AViq6PuryTIiye0BaFlMXc4UfArjSNMLsMBzXCVlNTmPNG\\\\\\\/1jwNTsABX2MMdv4X9Gr2k1o7Er3VvAYZCGkDZANRawlGGu5UPQy1VXS8fEHGRCTtPgmX5l+oACViW9O7VpP7rc+v8OLtpZT2kyzH3R41YYTwrl2S3CksZf2KA89w86+Uea+Ej0tByyevYOo1yoY45hTB91vd6Qd6\\\\\\\/CLl4XyBJcLDxoVAerBaz\\\\\\\/TXV0fbnK9huhY12wrsdDfzw21ZsVBaiAPxeyIr+\\\\\\\/59jq9ljCnf2VEW57l5FDfxvZz8cdeCEk7D\\\\\\\/nzdtIEcThiUbsu4y6CYLrrUz0ogHLTs6Fc1WmNd2869pBUfZbTJTdCDJPdaIjmRdqRInmbOhsTtotkMwcrIiiApVQ+DDVg51Do5gjnSsnFowvmR+9tzHOZiQ6t57GacSowKBeIM1BABqQgMZORGIT1xeJ3CqC9Yw5ND2qxDg6ktaYTnU8MsUQ6qDjnvlrBkGh6Y2XimIbmyO7x0djRVfJvcXnMYCa1+aBYtwsxR3BPF2EQ4zBbwhKFr\\\\\\\/zg1BIKwA2pwRLcDwEE+xkz59et0PMmYvMr46uK9dbivkRfRgE3zDNG76aLG9LoQ40HCL2nrTR2I05uhC5CV4rlKGaS\\\\\\\/fbosNe2ie9hvsoPYjqxhQwGb2rYxfBuw+0EBmzCc4Ine0tunydL+5aT2a\\\\\\\/0GXa\\\\\\\/wluSmkspKPKEowp6SoxL7MMkK6VAbnOYuFCDcQkMSc2579s1VyJtZ2D7IQAvydtI7KW3C7CULkSb4smTRFMEUETKwkDUvLuKCq1PrmpIPtEmiuphFJdpbVQCWYA6V2KomnIYc93bkNYdjl+lPs9U5Zrcwsk8KtT4TsMiCgdNuv5RoogJjS4mflFJKFrDoS1pJMlUjtFpWmYu\\\\\\\/dAxkE1+3dfzID8BeCb1lF4MtwY\\\\\\\/hdBtbkkm2xaIgvOdiQVrIP2TlgKxYl3wNbI1ZwTZHemPMqpeAev\\\\\\\/axa13JyxNTn2jypr5f6Ru0MUv2AJCjfyFWa0SGWG\\\\\\\/OFQqeGB74B628E8LHQHWWO6a+EF4f23x6tjjEOsplsIci2x76uVQZCbNUQx3+b+KMdGOxJh6OGJXsw5FZJ8BI3+akzOGqZh7pTLU1oyBNLzld9qbVlgxeBPYQ7OIhqVvj2JbqjvcBmxFEORQDTYlWYG2iWop82yh2OuP8GUm1GD4vpVqErXsfLov53hn1QV8t0RUARi0Oo4uVG+7vEQfnD4KOsWVqzLnVX\\\\\\\/I+OwJUUbte8P9Z7zpdPr3ICnWVe9d+BGFBrC1m9jly0TIcT9tq0bOwRf7i7NFht3QRzg7nF1\\\\\\\/ZiMcyiDn3rRV9We8rZh6wOBhCALc4qFBzxmou8b+4gY1Pgu71Lpsk07ypAcN826nQ0vHCNr57es\\\\\\\/wk5jbOpmKx8ZS8nqtOk7ocJeA9rQ1FLDHPPuMFwwrJrtwuQpuNypmC8YxqRt5VPKscwwG0rwVvRsncQWTeJLgqmS21Vmnlx65KWQVh6S2kokJMnQksk6looxpj1YURRJv3UuG9gRz25nEZC+syLL68vNpGb93Nm\\\\\\\/dB94ey0ZUrZk6YjhEyOkRYDG0j\\\\\\\/kCzvL5D49vL2OPS+y3LVY7NdwJLtSnYAaxtUfVewLrcUtdq3XtBIHI\\\\\\\/co\\\\\\\/Ou8qpaXEL8l0Ejmschc4EA3t7UIzofeQCoIb\\\\\\\/mLuio+UxumHFnCO1sdGvw+\\\\\\\/5e00tFvLHHHOQFfu8pGzSQ1xGnobKfJXhCZAMCJN2ws\\\\\\\/INeJQ04rlv442JFB5bDWm7DV7fQ9f2A1dHtuelH0Liw3HDJxUJQXyCWo2ta0bInIwprG18ybrh0TLzLlOFmKIlrmIKBOrNQPYxt2wcR7OWNI7O2y7Z0iX+FYuFPmfPQuQy6\\\\\\\/hhmqdGNjS9eEfykGLnFWrn2wjVzDfSuU5VB0ijbm9pL39adE0EzTA58It4ZxK\\\\\\\/iDlpyya7qUl+TbTrQG2\\\\\\\/ZYG1kLZlaRRSSCm8aIVWusqiLdAPKAVJCkfP2786ZkP93bxibkUJvWGiOQHF3tz0YpJ+TOoxRNTgJvD+mo395X20ZkQeHeIYLc+vm\\\\\\\/Ul3b9jPwfUpoZ9ZNwGv8jkI37AEbBz8d9OT8Q0lnmdWdmXFIbKrahxDm8nssvH2\\\\\\\/UfuvkP5YeIGi+i+\\\\\\\/3grdY9mOxS9CEp7N4IEQ3mjJsvHvTvFWXRAkP44rdt1K9OBaPSBx\\\\\\\/LvOsYquPgihJHvIDiFsSsU\\\\\\\/ROxVJv0VGly\\\\\\\/JatLC7XirAmBu+gpQaLFhSpPLUBLD9OD2s19SWPsqIHBrCGT2a+9dNsnnPW+cgJV4uZmPvY0PKeYiYQGWfMnfRI+G\\\\\\\/h+jCSUKRHApH7KRWaZX+tBikrsFRRQ3FZfYQ613g+6cC6biragJtHnrlLKV6DNwf5fHHXmZHQCTdAFpXjhbmwXwSFyb2V6LXEd+M0oRlyIUe+HnRRBZ+WIVvQJovK0YdVWqzFlNhVOEsdNy6l0qFBdn5A5inRfoEuyZfjwOx7QyYVem\\\\\\\/wButhPRoQZlgoXBh4YUPDsVj+lbF3Wdzts+UQghrj8zt5VXntX9xOyYtOczt0n\\\\\\\/1Jbb34+y+KFIwM4iNS1GUHoTIKdnWJTojTW1k\\\\\\\/F0Zzdze1vpvm30V\\\\\\\/7qc4HHGPB3PtJibqQQBsnIXWYCaFI4n9pA7NJePI6ARwfkX0O+jZDelMlx2fwEF7pXDakQ+i7\\\\\\\/QyG++xmk\\\\\\\/K11FrO2dL8UIdWZ+rSN+ME1LlhWVJBaL\\\\\\\/5SKL4Laxg81SCbngPm2kcgK1UsBP4nT8Mxv+QbVkc0+jVXovLJELoCs88z\\\\\\\/yAJ1wmxnBcVnoiknzvnXxUH3+Y4aa8cQkqXABjw2\\\\\\\/ahN1ZhUJFpMEZG8HMWc4NJtY\\\\\\\/LnnUzVcYBPVupCbotaKeX\\\\\\\/239zUyMN7raUtRF0NxK+lnuN4G6x3HiL6PCxlpDDfUx9ln82DkpBLJ0IDOfbjHLxU1eoMIF+5NYLyJV3svROZmDIIT0f7L\\\\\\\/vBxnLA\\\\\\\/M4ezfAzJVLVRRlFgQE7UR47cG5Di2KZeX0L7mQcBb1LcnDF3WqgmetZiOusaHCgaO0W8Zp+8W0K7XBSFB1A7rGPtJFl0eJKttHwsieR4XvCuBmqZ5crVAmf9PWxouHnU+pm8F+DP15AVoaJjUbWLE5X0sOwrhGvM5pfPLaNFN6nkgTo0A1h4KwV7L8tyenuaB1xNXm5Awd3ixR\\\\\\\/C5eEbbaPi2RuU6NklZMln6Tutvxd5SxGI2XqQf\\\\\\\/ybnlMknSYs\\\\\\\/XNpnU1fNBNVIPtFxNj4xKbPbfwHogAQuCSS6UgpOR7pzu9+RiLnCRxZlwPr+q\\\\\\\/1tDfjkFSk8FZ5xiknRcGqQwuCeHqou\\\\\\\/F6O4BJOa\\\\\\\/++FYEnljLVRf18UbjptBTsH1zySGuO7XIeVutLsNkjVAG\\\\\\\/z7TJhflKh5eNO1pP\\\\\\\/prjM86IbN+kqAjXZScA\\\\\\\/5JNlLdae+CY\\\\\\\/ZeRz5Cr3H7FXjorJuYSnBuTb2wFhv5TrXoJNgKj5Raum2t\\\\\\\/6cWpZPvKhCuKduqCcz6rRbUS1vv9dEY7uCCVvVza0ggsO3ewLo2mnEn8SKoBdsGBKWUN4aQpvilwCWGElXNqwPvT6RSS2F3fcQ8MCKtplKp0KENHvjT9ECcWUjJuREOK0ae0\\\\\\\/2Q1DjqjAyCHmKpMuLrh\\\\\\\/sZj\\\\\\\/QFI0CzN2J2AECLBWlKNqXFLapnwrv5xLKM7EhSQ2p2gEITH2A2C84WuUMs7iyiv4V7v5Ih6terbUuM\\\\\\\/6xYnITWt3DjvZtXlMaJIEIYw2kUKWbdSjhTHPHHAL8Are4uKftwC6lw1qKG3HG1yMhCHOECRziS2FzX7RWHZkkVp0UQVbp2Vz58KDkCtVCg1IxpO0K4zfNpqoZS8TzvyQ4iPmZM0fmWi4m1WptVLygQe90BOFAh4+fHw1liVrWuNc58fdxDZ0Z67BOI8+LnYNaGqLM9bCYA04s7q9YKl\\\\\\\/G7pqKzdz1nsxQyCWmiacez8hLhc\\\\\\\/I+070s8pUfLGFpYsCldhXnxOtyExLFP3oWpb4hpd4MwjapUXTv+4phdyH58lq1TNfB5MYQBZb+m8CtKYju9Dhc9GVe1sktG7R2t7oyUkZAkWDHqupxE1zal6MjiF5pFnK\\\\\\\/tf3ZsIP1jyAEKvwlMF\\\\\\\/v2iywTOJQQ3Y7YS74ULVahLmVE1oviG\\\\\\\/vsgFEHyHyVxR3kiSD8ViorW7btWRY7AsOWG1Y6aCE+D13CMlK5pL3uxUMdqiF6FrDCu9kqGycr4pVFifkczP6ASjhtwKOiz3gQ4uZZLJqoB7FIeYDBXf1QsNxhi\\\\\\\/Dvk3Z2Kz7gG8HB\\\\\\\/V79qJPJoFfgTOeZn8ER2fyx5BToEhLRKRtgKowbzX4iqm5av03C4L5ZaoWmSal6ZCRyd3P2sGYZpbMD+ASS6BIpr6R\\\\\\\/NUlhT214TV4PFCUjWBLCQGyOSy7DJU2vqm19jUSlaYy++Spw0QvOJ6jn1+obwjmwr\\\\\\\/v7llemZyRjZFLMCTo9ZemrVFsWecdKoRK0Ywc1G4UkZw1iGdSIDixiD2+N12I50+58SzSKwEw06OPSAk5pO0anA5IENeZ2WABlJetHy\\\\\\\/kx1f\\\\\\\/e+YRr8O7nxf5HeIbwOrNGyd1q4LZQdB\\\\\\\/f4abvNromAxEE5pnLj6oOKAun+Vdj95gw8yCuRyGc\\\\\\\/2sKwcGQWxPWJlbq\\\\\\\/ecDHx\\\\\\\/QMJuf1ZAHAH5oFM9l8JctGi7TgsOtmGRe6kZ8d1qVL09XX9HfNQsbhWqKWFRBDo\\\\\\\/FYSmqKkj5R9WRgZOuM8idrLl3CG470aRwnBM+vFMr6s2uEjur5Q266A8IYKEZIXv\\\\\\\/Dsq8m2FdBkgoztfbbZvtZKoBlTkQVFxlcTStGhnAr8\\\\\\\/nvr4gq+4RC3V1nSvhma8wzj+\\\\\\\/ElRUtNa\\\\\\\/JAlJQ5wgd\\\\\\\/JnLTJik8ktzqeM\\\\\\\/zsdWLaHZzy5ncqWkIiuzKeoddMH4AABXF4yMe1KXGJatW+JEbZDU1IFSRn2ZPsMufThUzTNspd\\\\\\\/V13\\\\\\\/CP8TGfPlr2IZ4X5Df4dAqu5SqrRO\\\\\\\/JSt5QI8uUl6MVJ4HN02dLZo6OKXy0qgGL81gSXfG2fz0f3EY60Sul77nOGUvsTPQMCMGl0gKZSqE5TPuuFhm5ehr0aIm8qg6jU1IvsRNMqQvA0NGiOPZftbc1yVj78m5hYM0RtqDdU3ea7dR+VP5\\\\\\\/clfgB1G17FScoOBn\\\\\\\/ihHmd5OVgjTCaX3\\\\\\\/VYmUCpabHTdgUTu\\\\\\\/7rSq4um54607Mu8KyTbgsSHLRm\\\\\\\/EW8jcy\\\\\\\/UHHZKgAZH2VfqqdY3xfv+HY6z0yGCYhldxZH1vMcjrzei8GLUK94e9Nm6q8rMsNR+pwtZddKwCPLEe2LF9XwC0qTyRfcJunLhO1jWDIBtRHCr0pmRbEn4hDMwAhy9rROmdUxR2tE3R+Zqfzm6b1eTorUdU2282Ce9QPOREjqKj6orUeuP1zuc8xdpnhEt\\\\\\\/jVW7azz5WxKDULLzKyEFPPnwEZd7B9c0rDG1xWK+VvMjQuOWIDVULJ3M\\\\\\\/HnFQA0XSY\\\\\\\/LNiUWuYE\\\\\\\/XWXGPAM317zEfOURhCBgQlp58\\\\\\\/quRkea\\\\\\\/rT4D6w3QLUaDFM9oeIemzbdHPqGSXKErTrELv6Bt3+8EyPokeg2HkV4s+Vne94mGQnmEUICCaTHe3BzQjXRTe9q2dj9\\\\\\\/+J7zvxRaA0VTQjjj8gCzNULah8Iq8JJqlMd8AfZG751siUL8u2iIJ8sByszjJpxtfBllG2Mv0DYyuW1W2s+bYQeEctpcpwkLFvexHVvn5iCE5o+CN0GoI5vqNJYvOIp7oCxnENJNRl0PB4paEyceFCRvzlDYLe4JLvmc7qa0bWY1BYePDI5p3HZ1BlmYqdOR0wcoaTZh7rRcB3\\\\\\\/fSLBnTta0v1EIWvnA11aluYag4RRq1+ld7w5ljsPXj7TwuPVscxreLwfsNaZOBk8FBaCZztYaoStut35b9PP5FRA7X3dG3d1JGMEMxPf7ud\\\\\\\/8VkVhgkEzmprqKIRXNoTNGEPxGgXp7y6Bm2UUAMCmNradGG7G3cSPWbjvj547cULATPOGRo8VKuGoqj7g6S+FDqrac\\\\\\\/FvMKquYCAEatzqP4TU+MZiFwlxU8J0uT4yukVZrf5TkiAm8OE4cr9x4tsFR3QW5JSs537MbMZpUyrjp4bEnRjz+m4LSU2W7Dp7czPp7gB5NSmoTZSzNPY4LIRsDWzh04fDRLH8q7e8HyDy8VELauInOdTn+ZAf2fpyfLKcCkQ4rMVx0a+gg+RfAveHSzI0lvk9o5XgSE3J\\\\\\\/j\\\\\\\/88\\\\\\\/lofvRVK1y7E\\\\\\\/cumbwffRp1a4PpPzNrxcy4NmSTUMdLZCEH7LuF9rJMzlP+W\\\\\\\/KSXo8Sv28aeirShPEnpVWWsMcS5NBj0PjtCcRBogmp6yqeYLE0PpTe7vYz0OV0XsAULyfvCb3MKRArybl\\\\\\\/YYkxjcFyGxhB8b8eRreTRrGIVAZzkyt4qCW5TKV4cC2Nv4jcQUtvGalUguWg\\\\\\\/xBOrfLQcfgLlJPCpp8EzWQ91ST+y3\\\\\\\/PIxRv3VwLd25T50T0YYouDFqC0CDgDJm0h2qG54lUsvIP1gXutHNhHCNQzmIemKBxo7wsDWAu1P274VzO\\\\\\\/RipnEslvtyQRhrhTmnP6VrrlEzp+vQElxLZEuZ0Lt1587riupIhd56fZDraCdvCWOD+ON0ZYDhdKhyshIcZcC3O2ef28z9aZ3ZT0jDIR93qNs0qmwPN2tHPWiYzSHXeb5sIXGFv8PDDidDsIeWEUiM7lKOhoCbHLB\\\\\\\/JQ7vRuzZVw2UjhEJszPSkJMcNJCG4PETEEApZ4TpwbKp8wX6olX51na3UeLeRD06lEUejkQD46UrBfPalQQYEdQIvHqIgwR3SQyuXRoYkXnTekR7tQurS5\\\\\\\/4q8in\\\\\\\/rH9VZ0mK8gXAmlYm47e9BU50DQP4mVfGiai9mDqhf\\\\\\\/3PkPTWN0UjPd\\\\\\\/dfn8Q+5KfkWwh25Jsiw1SR73qT38lMIzXfb8XDF9Ezfxv+M4PsIlBnl5IF1X56XvYsHhLU7FeFeDqRu41quJLs7H8t4RKyTOQ18hOjuEd7pVvxgUZQSLOdwTD+bTuBMStLF9B3MV19owrsu1YqcdJUC30jFB0awbiyeg4wZ4k4Si5MMBcv4jOkBdn5qnaXM6uMJ9kDnoMfwtRc3vy4cdpNWh8ZdshYwjfJK8rxeJ90gEq+rCAwP5vWJOONBKFqBTUksP2KYtizR8yZd3vfk1+fz26DkqUK0kJOAfSHyL3TuF5QVQjBf0e\\\\\\\/Aqs+T7Hj3CuL4wdnfgfnn042JXsB\\\\\\\/dBGG0Ze8gszKYfnnpdfGdd\\\\\\\/uosLFQsrKTCWpI1HG473sZ6dMcovxZrkcKOnhGTNVnlowP1wXWDPynFPGz\\\\\\\/ieNHoCn5M\\\\\\\/WRt9jw4\\\\\\\/xq8dwwEUIyAWconto2ixPlatg0kFKlZ7gFGVHWraxazv2OZrxXDHxUC91zgY4FpzHbXNd8NlgHl0W6X3BFpLFD8j1iI+nZxfBTV+hDTv4scAVUrKRjkWyo650fbh1ACF\\\\\\\/TGvh29A8Rmq7+qy3H9mQExiZP4RxXLl64ywc4Q3TMJJ8n\\\\\\\/RhbYX2fhcPRfNzqMQ0w6MJj8IMLd5IMo\\\\\\\/SMwm4RKY\\\\\\\/sFYSDgm9Wx98pLmsS3Cc5ujCflOMVNe4OiJiHegyK4\\\\\\\/e0kXc3WNCVogbu6OkUNdUGjZ2Yt4Z9SdnO975maOiowhwKVxF3OCJ580Hv8APftuTQvyEM+LxgrRe7dxTii8Km0PftKLExZIb2BKOJ0VUfYBlAoxujdDwCExF4BpnZjRj5cceTl\\\\\\\/14pID+ujX4KbR7Cs0qWF4nvEWdWuAKpbOop33XrPa9eDLfbZpwAkdq92244nedD6tsleTHC38jFvD91RSiNkdxFWfZeRMN0Vbh9u6mY0j4StMT3tH0pOwQSME4JqJ7EY7P4v\\\\\\\/zW617aRrfQoOqyE\\\\\\\/BUUSPjFBWdNLGTzx27sVHH0jrnAOwPZKa1zd6Csp8AjBLBbiNxjxUofv7YtO3GHhJFL7vLGZOog9zdYswyC+4luQLNjmOKeSf4NxQ7uOa1BC5Yc6XFugwRFKyMqgRAjXqHRKYclUVrlDyhd\\\\\\\/bXU5Up9\\\\\\\/qlP\\\\\\\/Bjz4UXIh366nHGGWu0PYbIM1+AuLXJtpalMzUNkIFGoRCBjgsU5yxSzRfvG7Bb9HPQbS+oOi3MU6IDe46xZHf9Q9ePf7U4itWjfT\\\\\\\/RlAyvLMrRcRim3V0V+MUD64xthvbDH2MTmiwoyXQEloAL2gMquY9V34PSvQbx4KUTtcEfvKEAbJtyYKtT17va5M7TY9dpzsdY1SmCEosXHIlqwBlvuOqsDg2dhWHtGIaDXQQQ72l25Ey7a0wgzwMIrH6JOwRXo4yJKaCuSng4D3BUWflCfKfKjuAHyU\\\\\\\/ll3wEN484gZEIq+MONjtzIWnMpM5gSW0ieYniXg+bjzj42lYM384xwRkEZgKJmDFfL61PRBkZnYqaQUwxLMjewQLzWkqcg2YzW70vWXl8dKDQA0S15ME1b6QiBYXI2F6N1Cv737F\\\\\\\/Tz2ARLFHV\\\\\\\/laoo7nY3SBkW45bZxmSe3mA4YOIIy+mLXA2V7DHWsqBrxgbrt1Tf0spCuVSqy1ClOY3YhmQVRWBFdAHm\\\\\\\/ci0+1z+Ba2zQl5goz5+wxIoqxfbPBRgYLGxJuwApgxwj28G4VKuBofzuht9h54pR7mnaoN\\\\\\\/g3lGQUUiqO+TZmRLn0MiuhX2TVgedyMf8y2fV6NIyWAWOOCKqDXxZH95sMKFsahP1OKtuTZjRBHRMGyb3H5CSmLSmtdgv\\\\\\\/6Vl8nFW7JKGOlHkTE462KXk\\\\\\\/6rHGnR94Ylzd5V5WK4OaI3LCRafLQyR2yFPf1kzq9N\\\\\\\/dtLaFOg2SN5jyChTe4oxOQIItdbJWrBNbCZUcVcxFcR7C0qDev9q0gjiXT+JoVJcZ9mlFMdFcY60PVC7vn0o\\\\\\\/Ct8w5QbdUmxrj1QjJVolbYMf7a5wcjZX8NMJ+YSwTU+2AsObhAJr3cKpQG+ncUuHaCgtBNecbeWGwye7txTr4oSxvjXXxpo+aMVbHsgyqZQ635B3zkw920IG9T5S07jD+W3kZpQaUPQTn3KP921k\\\\\\\/fJ93qoSHzolCPfPL4gYZDKHIGrgJGWik4dXRqWS4Vm9rB7EPDbNw84MmNM4XWKM6blIYOiQKBBdBY9tczY2JBUMIvcqoVGQv2C1sM9AOP+\\\\\\\/eOgEJUCi1Zv14\\\\\\\/uj3KHJsTxBZU9XJwvIBL5fapOTGPSqGAA1SzRjdoQl+zO+QwgQPXxI045CWNHInyvA8XBy7my+Z+MpZ5XE+TlcSS6\\\\\\\/0ipJUzJc6h5S\\\\\\\/VLZw\\\\\\\/IUF\\\\\\\/7Qny4irsdw28gcw1P7MsidRxfjIrL0w\\\\\\\/CSSJNg5zYEBen1sWzO1+bCnV2GcP5BEvfHEQ+nLsbETLHogCG66Uk40i1pue2eaTXMpSiY7afPbqIUzypXYOPSrLqVovnbjuDlaAQOwqWWj8KZ9GRtRY3GGg30GYYG2n\\\\\\\/Dky9nFEumyNviVElbIYNwtJ5daIybIpsW8N+reYOhMkHVwPDFzZG9gmkjI9cxW0HdaK3BG1d6aihZohDyxLq4C7+kjywCp4H5i9pAuWVB7hL91ruxs1jCxxgCn5+TdJT0NrHap0zVnDv\\\\\\\/GkODPiYOTuUv0kBSfmV6IZBPYplkYnSEtl\\\\\\\/BLVlFHQjISctTK1iDyet9KnVF1AA\\\\\\\/KLV0DPL4HiEcXj3zI2mWnLK8XL7I4ULbrOILqBBd4OR1drOR0XVeAN9+al9dPyp48z9Z5V7qSn5ehapCDKrfVK1XWXILV0p7P9Wjome0+YxMeDc9WOxNbaxcKaCqDyTctPty7wK6DAa3QiBYUNjrI+tri1a+ry2A2ld7AshgZeadyI9QzQI2WykJhD09zmjc6\\\\\\\/8q40x6Laq8pHsj8xupx0tnAe7tI+aDJg0IkACjK\\\\\\\/HKbSSNlHSKpW2\\\\\\\/uG6wiWuGx\\\\\\\/BhAXwKQ65BbwfepurEREgBaGhmz+YaMpxrp4PelvQDI889FPb+O9YblliX5p2GhsTWiZLoh\\\\\\\/LoebbHZFYhzJjXMFezoSqgj\\\\\\\/OSG7vC2217KLuXDu9ZJ5giCUlxjPWmtHhqQFA7ZmBMOSlXDMdleXHhJ4BVTRoDr8FRoENhHmKNjgpEE7tkSvIJeijCGjKmmZC\\\\\\\/Zgsrpc8BqUyJ+2LtXNUPTY5Wx9i04tTfsytlEDBiyJBTydGuAcMrGbY64bNStmXwlwKP3vj+cPjiRdo7tguSEnUfNHCPCQ9\\\\\\\/4XK3QFfb9XbHEneJtwFftW98sTP5jXDIWqgiV\\\\\\\/1zmxvx\\\\\\\/MdnQEHv7G2lAkWya\\\\\\\/fnEp9537uqxcaGgT\\\\\\\/+vhk8voUmTm\\\\\\\/5WVSCwUaWKd5b5aBjpR9F2ZR9I9+NaPU9fV9+iRBKojxvMCizc4A4wVgi7HTxDxFEhP6qqoSHumNPZglKIchoY+oU7x3g8huv3b8zMJmnGqMD0\\\\\\\/+5fAoIEDPWg+Ds0DPoBoH7HfydKG5gb\\\\\\\/er85vL5UbH7wr4UhuZYoRus37W5NKaVSMzMRoZxEBfuK7+yLUBVi7masR155R\\\\\\\/zVEhPxJgbk2EDNWcG\\\\\\\/cvtSC4KzkkUuboh0Ch90VrY078DToFTKru7r7g4vMvvsGSNpX7Uu0DifmyBWD\\\\\\\/+UJkMT+34bdGhgx2yrFPKlZfA0ryAJahcyUk6ChLg9y+OvDxxKXIOOAJ7Y8ZmYCw\\\\\\\/oOc\\\\\\\/t\\\\\\\/ZA8\\\\\\\/jCXhW7qHX\\\\\\\/WEfzh3T13tqhFptnjQ3bzvLvCSxYa3A9OG0HEXW3PbLh6CPQcj7xRiYTiZVxWqZYpRE6gjR7OBgnlrbqoviLIto9O0iTk89UwhNNEj158wNMbjxUjvZAne\\\\\\\/5mS185A53VuGYaArHrPjCOZIOUcicfcHD4Mw653QdfPOojlnxG9K4IW2ty1W\\\\\\\/S2zIbOkh9hDXMh4tXqd8J9\\\\\\\/H+aKcxRm\\\\\\\/oOHUubIhUf2SrkvlKJ57HodWqdUR7jasmBb0A5fuE5H0aUnD2I+KerTfUZZkSxw7wW4hLBOdWPen1gN5CtKGpBTOxql8tFHTD+W4SPC8HSNx4ux9ZKphjSXKgNjwG27e0nIhThuWCDUV5fVgpJBa2qlM5XvIr4ISjRtqgGtxA0WglgZcYicL23SInEKgVTRWsTJo\\\\\\\/d4P3UX79Y\\\\\\\/WHtcmA6SyEKl32RhzyveSkbXRrTkIL+GugSM2a2b5TRqyGdSfI0DtGDOq0WXqip94zxkACZ82MvkN00tTofYLglLPjF8uuHvC9MLDT3YqqIA5ta2wPffSJNE8y26NBsNkh1dCQMkSN\\\\\\\/6DNICIkS8pPpbRgAMOtA40\\\\\\\/zblXAXQpZFUOtZ\\\\\\\/MPmgyp94odWCpqBflZhaOM5mytKVdzYynzM1zyfEXVahtueMugx2qVnOxRd2mHryyjpdYrvrIW42TT0SgoVGSRI3dFT8S5GsH\\\\\\\/ihYi9a4Lr8hHif\\\\\\\/\\\\\\\/c73rEgvKkl9k+p6nds4fhzjC+sntNbXCZWWve816\\\\\\\/x8wcrEc6HDqG3n+o\\\\\\\/x65RwPCnyJA6DLJ7HxRlJcNgsRkObxDR0ky5qgRaeUvnubb7WMd41tbuAS2TSS2ikoTtEDPvRsHDDA6pvufFPl\\\\\\\/g6x5DUvifaM\\\\\\\/MVyHGf6HHWx5JMsnF7Fynj1Sk2yPNvU36RQ09mPA4ZpcgdIiS+u\\\\\\\/Z1fV1ZKFF1cQFxYtVliWK4LuLEsymLdF44Nnks\\\\\\\/YV+COV5OAfg6yEVCzqGNy520pULziN2Y6vfNJh3qUtYVJvffjisvFIfuQQA9uNIqdLh\\\\\\\/xE8ufGZWz\\\\\\\/lvY3e+7k9w7Iaf9tPYfzjvVWM8E4SMiI6rX4AJ\\\\\\\/hRl6AoEythoHeJd8KsPi3A14sKKw5AWdFjp\\\\\\\/zrsxo02RjZe8U8G1cLAqdG\\\\\\\/XHEVBUoNTBq\\\\\\\/FEEmVljINi5CkZxYO7MF5YTh1S1Uz+IGFjk1hEV1OuoPu\\\\\\\/Ih\\\\\\\/Fmwu0wHoM9ZXsNVD+a\\\\\\\/CtADqTH4Q+MfO8Nh2CWAIBCcVZxawrbivhzwSwL+qxv\\\\\\\/dT5ScFgBL30ZNMjjsxg57c8CQrDLVceIEq0kxLLac\\\\\\\/to8ZF73dbG+ZOGmU0aVSsIvrLzLZTikZuYdI\\\\\\\/KqfBgVWUhvidq7k1UI4WfJaDbIAE4eoojZ97TvQzGZa9wFm592VMU4YKDNXD2LspcUuPGvqs3WUomdTrcSdzk0rSiyJvP+TtBD5l94fu5PBLTT0yt6MZ0sFAAW\\\\\\\/CcqOwqu+R7gqJDuaTWn2nq4rUr5w\\\\\\\/uyBfCGmFt658Yw+AyHOVz3lE5Yu4zLuCJ7TjOfPHVt3rBUkt+iRWXcRqri1VYavxbuod+VGReDfLhUOQuK\\\\\\\/vxNj1pyN1ooLT9zfZR23Y7M4xqmKdwllbvXM0BsMI4Ycl7E2+pqLK6LNiqKOHuLLO2n4eDSuyNDQtvXC3i6Yb7XQpg\\\\\\\/KotSA0qPoNBM23OH5W8SK09v5Rfrsu4O4bccqVLpjN5+pKzI5P2lgzQwvvhdtCzCxHKpJFskJK4R99oJxCWEPzlApcupb7DBZh8WLjH0FTNMy2Iw\\\\\\\/g9txYoMnFYyWXYRABTJ3ic57+Pxn5U4+2xAORh0irrLPAOLj5hQ\\\\\\\/jTwOFBv2is3TDrg0K0RGmbE1Eb3M4WIUBThwh3pRApgm1Xrgapt3MaQ5CqMJVs2HVU40XgQ9gXxiaZlr2P3NhZj0yTXyBj46t6zRfLx7IUkqIB5\\\\\\\/FphbdY\\\\\\\/9Qd65Uf+\\\\\\\/sZoLcnDuIQR\\\\\\\/bjAt5KjfaeBhMaPUOr67GoZLAEI4jSoRJIRh5AHcih1vbPffW7qk4amdsqUvK2wWx4FRxeWItRDUz9NNw+N0OHHXyYElYDeq7AMG4AWk4FR2VHdq5QLJbO9lqZurocxPAaIjwnoHLoymxKfRAkLPRNHQMZEKeSzimFOueWTS8+ly6nQ+\\\\\\\/hkQc\\\\\\\/ycZiUfUxilxgLAr0cct8zLRKrNpTYXFeaxyarmZt0cmJ5YH9PnssK2VuEeBP9fJEwljLFCKoQaZCMvbjDDcMxcsEiUdu4F8DoFitOr5kIkAbLi17hPw8FG7YK0cO+OxZvDvlQaqFrOTEuK+kuBnQD623VwiCC9LxA+W5afTJiMOxC2ILnL9GZPK1u2qWoCyaru1SDiDiffD4nNDuGr4+ooGkRbQWEglD7oF\\\\\\\/cp0KJsHodeyw+ri7BGTJaLZQzFKQ\\\\\\\/VYbm4pvyBJSNxaiXxubl4fAZfAL\\\\\\\/RC0K6oLW1U7jBpKbMh33bArtj0U+mO87fzn+wacoHHAZ5Gcm7yEQOUaHI5wz5SxYO995EJRBJlmz\\\\\\\/+iHghTf7Pt5XEPnWBpZqdGoQ9epwBpeFMr0XdYOxoqxFSDMNyJ7EEywuzSf77rGs3k1AJvYMxXEUuJ177voPB8p8UFXX97I2qKp0cEN7iESnuC9zFhqmAVP1K3YPkboEYqRw0uP\\\\\\\/axKaX+ep9JZU4xgeO7o01g0ymC\\\\\\\/qLOYWzWGmb+46LCmM3c0L4ndXaVlyF91CYYYEglvV4DdbV4NYrlNas38rCSzqTQRVC1kbdrpeZ9LNggPK1TuxwCO5q6g6U77LdXQEx4I05bIbGN9Nd0JfKXODjOKS5FR6RpExiPt1ABR9c5SKk4vjFBcUBby0nE4NpqwwmYWtFhrylbe2uhbrHF64s5\\\\\\\/H2LbMutmvUGqQQAp87KjditntYXzgeAnSM8baFvzfsa1YcYzLOMB9SFzoI30op9GEChzIAkj\\\\\\\/DZ4zTJjsOpWtIwewAigWUlxSpO160nHbsIs01oZJRUqpZA2jfnvRiyigJwKvUg9QZaKTS8Sg8jgm6sMkml1SHxJazWOBdGmy9j4ydlHT0Qd7kPhzKfga5zo941dbdoaG2ZoC2IGrSZjgJtGSgrtboWeqcxDwlYHm9PdzVsOcdw192sgpiYBmsUxw9Sf75B7c2VVdFQLEXTX2igaa3gbodODqNp92bXOr9JKk2vgmkcK0ToXbYPY4CzBSTBz311ideo8z2LJds2AnyHgKuRzJ+rrv0Fw85XCq9gIeAY\\\\\\\/A5mpHVSPfWPR0BARlRa38YMoLRVBtLT6rTRS\\\\\\\/AuBlvyJotoZYXo4d76zW8\\\\\\\/eauO6UK53gMz53XXKaJdNS6nkbD4LJGPmfN2Eje9GQUvfYz+eYFBZEs4RIziQAkXVxGvSNnYPx8okdfbuIGQz9Kmek9+5dPWrFmOPQVomB7nHnsK\\\\\\\/aRY\\\\\\\/RjGvnWUyovhVWTR1\\\\\\\/jjR+N+Od8wtAeeHFzIPaIrZ2NCaoX1a6Zhd\\\\\\\/j677eLE0FE6wqY0DWA4keSx+gxyeBIuJ6Fv7hlc0l8ufSV2sO8k9ALcK0wSz8e0T3M5U9KNI6VYhziiTxGgFzjS3iHJNUnTTmx4kVIC2yoilnoCsIxXjga95YepndAPIQhmYBk8AemdJ5US9zktfYmlnru2gIsxBYHlVGuKjsxvOoJ8MvYdjkr7fvMoNZMoR9cDi6DcfPw4zBs27JerxfI4WTQyP7dETpgMAUiNxIbjFFrY3WgSL5d1CgGQO2ONCCzCGSeEfhqAFX12kFpKCKNxUaZBKq3zD\\\\\\\/KSpEusXKbxccs8l0MbsYfJkIdDqwtT934udefnM0JzQJOz6O\\\\\\\/WHcM\\\\\\\/NFfZ9OnRIUx2ehb0+lCucNDPRJeReoYeUnd4shb\\\\\\\/Z0w+SQpoNP4Xh\\\\\\\/Hkm9+Qc13xMbDsNM5P1F+MdjjQBW7MtNN81e4blPNNLojI48\\\\\\\/qJO3pOqtTALD5fRtN5tswycV55bz5u8wHxn62vzFhJOda\\\\\\\/Nxg\\\\\\\/wXD5b0ZkeS+NFuPiPWkt22HPJv50sRZygTP2hoBOV80YHNYbinF9G\\\\\\\/y9Y9YhHj9WVyepJ2BiWZueti\\\\\\\/38Lfn4Pbby9WjWXYtrAK4C25B5X43IAIFEVIQoROCV9W0\\\\\\\/XHDSIK\\\\\\\/5fObUPzWolqc8IKBiHIro6U0W8wzJb7eafYvky0+NrjGuKA1kP9CiomnyEeewIndiskXUfrjZ9FEq8hlNaE4bFsX2Ww3a3MXxefr3N4X2mDFgvp+YWZyvj193yYR7u8cKT1nWfBsraVepfr7o9Pvu6wMntPimUe8EqaWh8nlZ4hUHbhgpeNq483hwuq+SrLBQyCTm1c0KFGSlzgDbwyi2i30EjDHLHR7VECHuwRSJi5Q1FQ0qyxNLWhZSpt8wXtV2osskU8lOqJUl+TgQxglbvcEvfZd2W2akOgDm3qv5ikq0AzGz5\\\\\\\/3r21af71gAandvSeyothbI47FVKWMCUxLoolrSdl5S8WBslPS0bTazcgRvcAx\\\\\\\/jpYAly8WDt65c1QC6u0uHbdLuf6MV3srvEKK+wbJk2dRSh\\\\\\\/wxkSZZWdve+YAwT4L7WezJNN5DSFvCTyWb4gKkFyObSioBXYn3wXvPbvWmx0obHIdqWv5rxlOdhiNOxTVByO0sHojaocVaoJvXRyZC21P6SDICeFnh8RICW0MaIuygyOqoBT2MZPjD8hFeoVqtd6EWH+YR7sWoEnookFDlJG\\\\\\\/SbKUNXIlOzT2r\\\\\\\/F9+RpRgajkAd\\\\\\\/sbrRxkpa3Rcj3OoJ1llXXgIM6fSUeULKgbSMDUTaOw7SGM9+Ab4j\\\\\\\/C\\\\\\\/Z3eINDrZSANYm8zOIu0wKY+F2S7yCNAcIF+5pzTNW0\\\\\\\/OBuukRIIeXIcKgSQVMm5clkPOwOmclyW8lgbQPa1z8BSLIWi42k\\\\\\\/SvFRU9iCfTEy0B2CQSagpC4M8FuYNBKGrY9CWNq4RACC9KpzqJ92z+FomLul8v0R5oN82YPqnJB8Z3bPnJhAf9OuGZ7pM6KnKpTwxXgk18zLuRxQwS4MYgHEi4dkdphduu\\\\\\\/SQULu62HMJH09QmJ++IYnbpJmssdOJQMQX5PTW4uY6F6TK\\\\\\\/Ogk9eVNmuNhSj4KDmSkaK4nDApLUPw3g1+aH4KifTEO5PqkzLkqVq0fodwSi4wgrS\\\\\\\/iG1Pdf4YzO9iSzKLSBHXnT18\\\\\\\/dCGzgzaEhA9scJy1p9h5+uhYLvmsyWY1eu\\\\\\\/W1yagUHoPCWDJRe3HVhLoXnIJM+G6dX\\\\\\\/+AdopcIx\\\\\\\/qIiJKkJT5VnXUOrjwoq5gzWB9oVbv7\\\\\\\/2\\\\\\\/g2rMbTdu9pZnTcDAw2d7xaKS7xIq+T\\\\\\\/NaNhhl8knn5IBAXlzjAiInRLaeiUDkUQxnW40tjhnEWN1jpsSJ\\\\\\\/xlawpWAsbYkYv01q2VGL+X6uxapa6OpPzVKbmLIdQJ7Wx0IIk29sQzIkVanSoDK7cAzP00OlaDYylgi4zdLTvs9zlI465NHrR81NPJG94FJ1RSIM7XvDjQy\\\\\\\/AoTNsX13mtQIqeIdLTzPYWC1Ns6wKbsPkZztXxCXQpyDjP5+rKyW9Gt\\\\\\\/tFa9nQLw2LevYe1LqnhVV+K1hWJjM16mqyDbZLnWY88OC5Z9V84nXV6lTqkN7tHM4GxW8rH2xpvFNKlXmzMQUl+2FlnXdBDPWJ62M7RIr4k31BWRi8sr8wJqM3a1h4oJ3Dg1aAiScALPPWv4888oTZfOM8xQbKig2xCAVasf0D9jZISpzEm4IS1SZmIF+wVM5JsoIGWw0uJ0Bhimhgk8SlCbbxSsCaR5bsGtGNtDmJiSzjUhbsjSYog90bcAUao+bbkDWt7qNi\\\\\\\/J+pdi3XIRqUS+BZdRnUKJTNYAM8KnWiLVf4EvjI8MZHyH3k9wm0\\\\\\\/ZaM3Y0T+FhlMl5auGTWyyzO0U3zsttcTWuY4bcsivgMxxM1MgFtjFxpfgAeOG41UshI5Zw+bxZe1usnROk4CbJxYH8deDXhixY+n7GS++tNA1sQAIOHhCVk4Pp5pcF\\\\\\\/aKlkeN4hypgtf4DI0NpupF0hlAFzcnNLh3L8kB3lRzuYRub\\\\\\\/36lI37LNaZwg5q5skEUqh8\\\\\\\/LghQM24I89qGPrGIXa92kYRwQ90a6yTU1zG9TdX7n4ZpvghnPN5UQ+TY0knzVjMwRYI5cRi0O\\\\\\\/xXh+2vzNFj3NcdpH\\\\\\\/VDCJUsTfjWMYsY2I\\\\\\\/o7IPEvmKNykI53q2K8Nt3H6XzNFuWRerks0P6m03Z0GUH3WTHfnJs1pj0TCj3wDDcaLeRpOvKEl1RbmPoBza+E9eZ3t45IHMJz86WY26VZl5EsY4rI58QnuIH3vb86GXeuY+N9IoCpl1oRpg5CBkLnklNQyXzjpYV52XLWHlL8V+SizCeRoh4Zn06YoasZaKX\\\\\\\/rvCkQqcVolpB3sSo22jVRyJF3UOmtexjuQQEemcCPSebyduZinC4uOCS86\\\\\\\/FNUrP3VDZ\\\\\\\/WTrvzIGug3FYziFBFa4hrMRefK08gUNE4eqbziyE+IDp3BEBDSuWIZp0gBKqi7P+35wRtkKreHqf3YIy7jT3pUnq9EgOx4245hcmS3lTcqw\\\\\\\/tHT\\\\\\\/5XjMA6cAGN6vcEcf3WJ397E+ZMFvpOhACNXPTQT5FR69LlaX34FcsbO13Fhs7isWcL7PiBIAtxkpIqnjkQqVoZmTBhq3V1USUa\\\\\\\/hEA2AA9oNO5h65H5N8arCsLL6tTiunSWtz1r60LLDm+RitxO79yEw3qx\\\\\\\/dWBwfoPMtDnvcvd5vRWa4X086zuLYPoQ3J5B\\\\\\\/rUAyGuAJQsKDNrwCb6U0zgNOK5vLfbTADB1zrSrSVIz6cQKkiJU4o\\\\\\\/fUTZw7I7M6sNFWzNhQ7WVEQALmSZtpZNsBjb3ANQCYoaJ1boaQFt7d42nyiIA3t5yp277kz\\\\\\\/haOYdz3vhaP959zbC5rnlQIjJEeXB\\\\\\\/b1jArhfPVlr6pcnzo0w0X3wP5o6ovml5FyBOjkYl\\\\\\\/ieSTLO9Ba4zJD9Hg\\\\\\\/3qTcU2g3jaQ78zyMhsF4jkVQk5W5BQ92koAXZ+gZlhkd14nCkFuOeQsEpZ1GIMjqvwIZbHrWn2xkeFy6sXOGnNbfXDIQl1YyKw7obSUEe6BDjkqyk47dRS50XOJfodrSwMN1WoEs9YXokOPdDTqOh3Wu0r8moUEgLlIBtmo+5zx3KAl7XK6xt5m0iByV4Nu6Zc31sZDJn4HBUivLu3cCwU4H+j2ABWLW1zJaIb9YHk1l8oDJNzPOB2nMR4pSTTBdrxkgOCQCvElsg+Hxz1GUyh3tMuUmMgy8oiLCFJ4AW46lpLMPNtVcSIS+UtUbcQMDDmhRJ\\\\\\\/2gmV8nWs7XWkNSa2VTyW4kBj2qgW7sp8oLwaRcckjBDMvKG0pnk+Kpr0N22MuUxvPJRL9lCF1buiAV+Rs8pxqED4oo6ThupW6e3QU3fr18MXu1FEYoMMzH9+5TYRkPTsjAo7jw9rBbwolxdor8uhuw\\\\\\\/przaLGcfyZeBcS5c6QAjFGBeXWZxorJnsT94puE5ZKu4vEWe7gQY7\\\\\\\/whs+zZFT34JQZMq1SNS2tAns2HL9Nu7ICgVKMaB5bMbMpsIxWN9hm3MrUNcbMtIYIjSb1cCaNoyxMifzXUSsTCEQabnOOUJtk3xgHhWzYUAl7KayHVAHeCdyXfrzA4gB+QMkIj3HIH1tyr+Pabkb\\\\\\\/6HnjcW89tSCOpdvwi78qtRoyBVtIYWVFABRH4oAw5j0k9aByY1DYx9n\\\\\\\/KS7z5QXdcTK4L3CrFMYqNOSvi\\\\\\\/lIr346BURy+VNr2ILi7jaALX9Om9vXC\\\\\\\/aR9PZAUrOmn94eWiQwAvMsLX85\\\\\\\/85AqxTu7y8bZP\\\\\\\/ZHnnTN1LTuU+tj8ISXzAvIPiy2cus7+9Jz+RH9HwOx4CyieD2I2PdVG5zQUM3EXHwCRAxnxf\\\\\\\/859aoc8mno\\\\\\\/dRAF21SyCRhD1xT3Xj8vw+5SNeTI5COBX9U6V25Gtcudf5Ol2Qv41W+e377UO1ZVyriO+cTsxqein4DTonmCCIpOjmQL3NIoWCbpQdS1vckv+FnLDFeHSAJCx0XVVf2pVt8vfZ1RYgZ\\\\\\\/jpBmd\\\\\\\/3ZNTFcAw4PDCZ\\\\\\\/1BpXUQ1IujZTpiXoC6eS6JBN7Y1lqOZW4l2yzC4i3d+UZGneqXY\\\\\\\/sCkniUAYirEWc0+YDV5WU58A45812huRDdbPIWbyB39vnKPdkIUAxiIm+9ds8rm2oG+YJpuxhwPJS\\\\\\\/QSu+lDmY3CXDzBvsL6xE3YLOaxv\\\\\\\/UVPV\\\\\\\/SK\\\\\\\/JWtpsK8Ix\\\\\\\/U3KzkIAqd9NF0A7Qs4YIRgTnqPmuY9p2cGDxvIjypACIkMjrawfLSyog+1FTuqfTeBZ6hbl7e4qBiG8YbvyiWweH3P8mIg2pkDdte2tMFSQkX+jzMhGjVrrUp+EH+TRU7CfBacvrAsN5ZXg60tDLrIT2YriUO3bSCyhyPenZJ2yEreIJGNpsQQZskQEBubaTqR4PYGxRnovCJ4ruSpyZAjI37spVOjrE2LHzDq0uWcbU13mz+aDyH5oUsBQr0KS+zV4lBprU\\\\\\\/YCdbjhxzKTeeQ3dVLgFBajVpAEPk4WsC4QzSsQqA2X0+yB0j2pCKWIrJVkh0R+PnPb\\\\\\\/qEcs5s26i7P+GDmXSboFz\\\\\\\/OLFhWT5XTm1xh9DXeGhLBfKSQjlHGF+gfwCyvi2T0IT23OHC3rNVyFyCkqtYpmINiaaXasWR6R+UPGm4g5y\\\\\\\/0y+V5A5V3+Ob2rpjV72r1I8dFHcKiBvfQyJTyqNMFdC4piI\\\\\\\/PAUEer+SgFlFThGrPyfEcpMXY49o7kXTgBA6takakCp6li\\\\\\\/r4w\\\\\\\/IBN+nGAxNB3y1MJnFF3UwulCB+qXtWyPdb8c9Y6Jz1oDsX7f+9HtPk2WPKuAj4zRzOJGn\\\\\\\/0\\\\\\\/nLcDfB6ufThOO2e2\\\\\\\/p6XabaDgN+xd6xPVrx02jkAksLm7ZwnJ2F9J26A6v1L++oWjamiOhQ+agcHqpOaYBrCmbJ6fAXRrFYTnFh5Nmhc94nz20u2OS5G\\\\\\\/bGFmfufyV1YMl6Ip+AUQbtDyPtxSFjGuO\\\\\\\/MjphdSpsSb9O6CNIpW1ZntuNf7aG4FiS7ydV7UyGEbIxYtMSA8P0tMaStN670rb0cnbgbgRQwDILiuxHCY8HvaOIXkTQpY9IUe8hZOMBV3iRkQoTR6IlLg8mY5Vs0LOUTiVQgYuoiAq7qWlFavEZvaINW\\\\\\\/uNbO8msgfcSCTAtmAc9W0LDQulhKdM2ZQBdmuNsSxJddr6udp8TZg1cuqcfC1v9NqxSo1rfA3ZxuguBTkJXkjP+pl3ibSNN4\\\\\\\/GeSgZb6M8EuvpZdrkVa2t6BDFfIaDvs8rN4E6wrcrjp7mzjLlAQSX8yfKL5ACJ1eDXo4DZuFoZdoGGhzvhNQjMzTNBMhBtQACgcMFOZ4R2ywVe9qUYH4Ep7unyf\\\\\\\/\\\\\\\/ZIu4AJKixFRzZ5JjHBlQ5\\\\\\\/v4sGTW7LkNGXcl0Q0Q0SJFbz+8Lp1HZgYh4+Xe+RbvGxoLmndU0tIGlzsTI1WOihwdg+QPH7vNDKqL0exWFmC6RFVKG6iKKWo1NonsOQQQQmTFt3qwT+t7+tPuVzyULDtWq\\\\\\\/MFEfMJgV3zPcA+wAFi04X6F\\\\\\\/AT6ErciVt7jh8HOZbqKiqJEnOHh0dGGSX0eiPgwRLWKMa5LnsU9aJtB7TEFPUbNqTi2+Pp7FjC6leBa2fosymdq0pMv70HnDlmfXPmfR4UzOqpnzdfVnbkSBHoTH1UDOIcsUatwVKwutBRs3LUzbS5zNRkHnVv9kO67h\\\\\\\/pgeYK5VdCIL3eCGTl\\\\\\\/Y6gypS52BzI\\\\\\\/w\\\\\\\/x5nPRajrWWpfcGv+oROoNHq0poXPBO2tjUzhsNVnStSvoVeXtae3bMD+GdEsZwbl3cSDcA+PPOA1WXHSUAUxNHYaR4KzaWJr9bsMCM97x5eDMiE7\\\\\\\/kJbBp8Da1M7SWbpU5gfUohJivyVdy4nPft1g8RLLFePYJMvb+\\\\\\\/3DvNUx1A+UERXIodMOZPlJ1geae9JwmQ9Sd\\\\\\\/7haX0d1DLrcimVyJ+XI7yq3fAvtoq\\\\\\\/\\\\\\\/nJjU33YicbVzrlhu1PBqQ6BfaGZ7hYS+vitfPyZt3zB4H\\\\\\\/QWhliaL8CV3PMkfhhjKKjkEGsZ\\\\\\\/hycRokyWXa7If180XaoMTJtds9pTK0z7uQYB4hnodoG3ugDmf48lX5+LjvWMwmRcnUvc3gxoaGvTkCXHBN6NlaerK5JBcjw1veCQGRcvnqx3ceZ\\\\\\\/i3Lvi2\\\\\\\/i1TubeLNx0anVDXyC1X7PYCO0jbmjrQz2bwRmHgsQae6xA\\\\\\\/lgm0LUE0YejWQricoClO6fv3yZ6AqfMI+RvYe4LHOTksLFDxa6775UwapujcQG8UD2DsDYFGZLzNBINO9ekgiFLjszoUEDTMo7PC7ojjtqf22XFAjKPnFD1UU7cjbCxuzWN50B\\\\\\\/0zpTA1XLGGhErqPvgXoalqZRmSXR465pOx4eK0+lUwstYF4KY1ScHQj2Ib1MgwcP8xwJkMrH0NhK6PLvOWsLGJOZ54nhEQwov29\\\\\\\/kLy2aC\\\\\\\/Ab+4JqcFMSJdbQG4i\\\\\\\/BkrVhZjZK54zQl1NjyqbL1TKpN0oNe\\\\\\\/Y21eBCSRS8kWla41B3efzLzxYORtvvjJnBSJWfNSKXxnxTTnq1Fo63YrAZ0mSG+MGNSCAGxtmq5irVPHQQ3eJTDT5SNpHwNpixZIFbxGVXaCmhdzXA1dXMmfA+YDrW6rnue\\\\\\\/uLlNr6mMvskefudRWHssJ8ZzQSIryC4xKEomLjzXmEXg+UDxy9qi1RuKyc8OylmlGoUQOy1HNW4lt1FUAztgYOBk0EJDu+QLG5Lkm368YssR9FPsG\\\\\\\/4yqmKUkhf3g0XPDgb7m48vn4LQg6YpwE8Z4rJN+gJEhvjX\\\\\\\/HcuEokkW3uvxswofa0xsx7bO4lsjeBYZdjK1KEQWJpUqKQIRjbZRl0T7fP0qhIGJ5jo2LZUj5+TJUorcp78HjdtLF5nx\\\\\\\/7A6G7hPXrh1jVq6Mkhn7EfcOlerMMPrSJn4sREasv6aCKgJJa8mApUdgX6NbfcfrRhh6xYTUaCCn93RFXDn1dGhTaPjisft\\\\\\\/2dv6EjHP3aHgpptwJha22fprYMSvX5e4\\\\\\\/cf2R437qNj0MOiST9b\\\\\\\/gIMaecF5xxQx4lGPFeHBoAJLdEXquydgTkENlVmrfWMdr9792VDI6Xird9lVfOcXla2fYd09LhATxTktvtgMkn9fid4Uz5Hmr\\\\\\\/0K1UUFit1INaCcksNdw7wogYnPPZptGOlrYBfYeOfEJIwvtVHMW9pBzOYpmDrLfSGOSgg5oTqJJ3du9Zze8o7XXgzVd1fIXcJfK9GuVaavsSTHoSHSt9\\\\\\\/HEmKYWbI4oxs0079lxRBQ7IUzAXXsmMrGrZ3XGiqMDinW1DxgD5GGfUAsahoHqeGbqqUjuYAJ0XUzUwIdVuvA0eTMpDRF4mMQ+3X3wjZ\\\\\\\/vgO1Qg6yb4MbcaDrxsc81dqiakWTrUxxOzXiEyMlR\\\\\\\/kUsfFMIJx8dYpJBFWzIDVAonLlGT0eeSwG9Llnjy3KurvU5ded++UGv8a0W1pnD4fyymgUH4+e+oWm\\\\\\\/JWl+On7a1UpwwnKUwhy42+D\\\\\\\/pMhzGlwJQaePFXKlNCC\\\\\\\/m6PTEbabcOIV71kq76n7N9+F9kuaTUM5fKuJWpD3yn5V0KNPTwfAE0cCyeAWiuXQ7UnEgdCgMof2jkWvxN28E9dyfHaJ\\\\\\\/q9iItmeYuFT3bXPkPzf3MwUBTwC1KZC8AyPYOYHI+7C40U2awdiqVMHusQHgxijKEE4BJtk24FApvnoQfQuyWCO7xtByaZk4k790aZhNQTFs\\\\\\\/7Ay14RDfYLqg+9kntYobsBzWqKPr4kLruZzF\\\\\\\/EnnMN0uKBzDXnX\\\\\\\/aznucwhzSP8sqObLCRQzKK9t5uENDlgLPh8VBjkmkpeJYKAMs28WjyiThaERzjWFMG4mDq\\\\\\\/0Oq9RNpx6jza5Mk1Ht\\\\\\\/ro\\\\\\\/FRywIwBjrvPEBHeFmtciJwg1vOdXFQdcrNPpdIbfBOMU6siQ3bW7SrltdkEeipV1t9mTqvEcNMFeWE5JMqwnJqgi36Z2D+j6Otay13RA\\\\\\\/ENZIVFoKGrSfbWTAf6RTzxSJpSZzD6lvRhJ3apXUQlvwg\\\\\\\/vBrRp6siWdQIHAEdTTwXZdLYOO3hEk8Uk5rhchQCnriimyH0kB\\\\\\\/LyZcImH6KS\\\\\\\/GaaWakpZtDSn5tRaJMaGaSpdAhE5dtkbHj4ne+zii7rL1Ef4IJRl+q7xrTV+63WqwpKzVtuoqnMiFuqH4\\\\\\\/Cb06DkG75mtqHOOEXuw1MQpWqt7Hm9kGYx0aknCOi7ESLe+XlQDe+YrCxYh7wZ1sznXZpdM0QWeK0\\\\\\\/44rSMW0cyyG2poronHl4y5x8ix9H++UUcVa2tbPSJW\\\\\\\/C0cYpz4fxmEX9QAcGic1tINDAJYQJcgouGkSiLP07UxYWyrVhzr4aBq1pN6dg8eI1\\\\\\\/PH5sSdba0P8Ja+f\\\\\\\/y1TA0jHsfc86z7lYpj96lUVux\\\\\\\/TP9DyO5QwtGIEMK1hj+iFFcCVReIRzURJLScIWNYWRK16hkzYrBg65qyBrgOPFzxasGq4178Tpw5YAPcrBtnOZJjH\\\\\\\/F\\\\\\\/KqPy5\\\\\\\/sZ0A4P4V9zybO5RSPcqXG8+5cVoOMxOFcOHw+aFfZ\\\\\\\/sscvZ79KZTB+XYba8BU+3TB7wIP5Q1dNTmNqQSHainVwD1HNB2\\\\\\\/rANocXk9TB6yj\\\\\\\/WWdlk8pue2v6ZDbkUb6HFYuylLwd3CR69xgIBsvenH\\\\\\\/5njH9VGF\\\\\\\/V7UdV4uNbYdoL5a6w7BBZm\\\\\\\/4YvL0zxwImwMo2FNj0UW6JUok8xLpvq4h7uRwpZttC\\\\\\\/KJtk8AydPKuCUGZzm9YZE55akmmFm3SNLMEmDDUqvMmR8hF3aqlOsMfgL6ri0sY14xB8zs4fklqNY1YihtK5MGzUZQ4toydcZTRPh805Z3x+eJgNWx9D6kUrpg+\\\\\\\/mGZFpif+rK302w06GsGdNheu8FfpEqSk+IMQl\\\\\\\/+eSC+40DTy6tHh5S+8877OxhYvZM9hJzGNu5+Qu0+gkBNuAOtr8tinuL5UKpT9zfuhUXAkwOq+LZXkMh29dB\\\\\\\/jaii8Mem6FFLN6iMBjxjbOUAptin0A9ZJm1BWpWu+fI7qfVWfN+SXlwe8\\\\\\\/SJdzo9BZ7+2W4KaXnpFSY3xwSRcwnnJTxg5zUKzZslOIxy\\\\\\\/oEhqqa1megyO2iJVEGDD3l+OO30stRjg3bs9fFKvFaaGDgHfeHuqsvf5BJ1zw4p+SU7tw22j\\\\\\\/FiGHkqrSIKQTrdTHDJodMK+gr34e82FRL2T1R2M81RAKCRw4n0r5fa0seghld5oVcMyjNLFkHJ2UBEtsqqjFyMn5jninsD3h1y1\\\\\\\/oCdtknuwwrMQyfO+4Wdwhi2hRmPRvXpCHCaVTwMW038tGmGcEC6RJOfLCdGpYxvNRKJAc\\\\\\\/rsvrdp4jvBDGZfBPj20PfsD2AVd0yUskDtVrljFWRj\\\\\\\/jWNJVLJTuTwnFhBF0fgqkTQSwibIfZI6BzuHhTidnJu3OtKTpb0XtNflOTyrS5B4Nf+47EtrizVNizApeBaNON1J3E3wOdslME3vnC6Ia9OUyRpB3KKEp5WZvmfnee28Sbg2uz+k0aTbYUhY\\\\\\\/FSZUmtOVQ1BgYl2euV\\\\\\\/6rk\\\\\\\/vz88CkKEQQSWBtloCAUdg5ICyQk80SZWb+ibV8bLfbzE9et6CIKE3g5cXg4OmtpLTO4rzSc8sttGfnBm1jUFDoDjFiyrN0JuqTETfCd9l7pX0uro4itP+DVbMJkUbY\\\\\\\/BsSvrSeDYSLG642MZitet1cLhRRZBsg4PBv8hHtLbgTBdjaMZ8NUdylN\\\\\\\/\\\\\\\/iNzrfvUDVGAijKWdDUJS5b2qDoYGRoT\\\\\\\/uIYJCJehEFVEUPLwcqQt5zAg2lm01yVM2O\\\\\\\/a4XiLU02gFavKjKusrTlLIgUcPQgQgC6LBWTuaoY+SpROaXML\\\\\\\/7y3W3TejOqC6B+J1u5vscgpA+iVRda8RfBrj\\\\\\\/lrMIkJB6LMlgvB+lx\\\\\\\/1Pe7fobOBlfh2f6qKjFmlbeInHqHb3Mjx1tRE6eo9eZiuQq2FazaFxhnsnAUD\\\\\\\/+K\\\\\\\/R0\\\\\\\/+XUxAUugNmyHy5Yk8PTuMbZQ14hwFu2aOSsNBt3FLqhDuwekuWETw2Xz6TF2pEJytd5OJFdmOzYE9tkTVKCbE1LPZ6LoEWUVBHwB7JjiYJCuUaQwRbCgqny6c9qESak2TtCt+pcRdtOmg7Wct0OVwl7T+BNUJN4X1j2pchuLcnK+DNxvLSnWIqNIAgFjzV9Tx\\\\\\\/evqd5qAGjZEIV22iz\\\\\\\/7k4uvUcKHtb18487ktKtBz6ceMpXGfjT9erJGG+JusES5\\\\\\\/ZyFMQIXDQcQt4G2JpsPMC+bnewqSh7hb48fLZrZK3tVikmBCYUh99S2QtYdDTQya9pmVmrw0V8sdflNQavKq091XYTUVWQk1uUgra\\\\\\\/wYKqJduG11MjJMPoISX6eoXXWdvujPSFAl39Z8mXaHeFPDgHeyv5ExPUHUAot8bNQxLwXQZuzGFR5oXsA\\\\\\\/+0rJhfae7PBm04W4EgoP8kuTjaBhuUmeSb8iTi1ZDlikg4qH2bSvesABegAblMvEh1zwBN0feLNcEBA0CSNOMi9yjIsVinF3TO01JP\\\\\\\/MA1S0peS4rq6jO8U2NseQIaIweRTIcblDpBiSLE3UfWFERB0ANt0CIjUZhTRQsA+QjCqObtgFwaaw\\\\\\\/CLbNb4hL0IyZqEUm0TztZbIqRVUFYnkxn+\\\\\\\/eB\\\\\\\/\\\\\\\/eXy0PO9r38JBjlfZ\\\\\\\/Riixxnzv82K0\\\\\\\/CPpZ2+tbP3oLJ3JjmZVxj+Muo5fYUFDz9QbTe37crbf\\\\\\\/2MhT+2qq\\\\\\\/fI07iT1GKu9\\\\\\\/USrVdMeMcbvGhI9r4elt0BaxQK1bmkKXIPJHQcz3MxU9+mO3xtYIpD2LIPFqReBmPBBhVuJYl7dYn+\\\\\\\/zX8IcyBxqHULbkeWMM7gYwxLYVTRh+NdYq\\\\\\\/NJY3o2aeO7VRN7TQHzXI\\\\\\\/zszvBdJL5\\\\\\\/koOFbDjaVCA4v3Yx2wiJy3s45u28ZcDUhlPbN1rvif6RzvLqO2RfxFqG8MftbzVezP32CpfMfjVOlcehLDJMyP4CWuW55\\\\\\\/1oN\\\\\\\/V4SS4fXIGLLyeOb\\\\\\\/328CAUOpAiJCEjeTkZu9mSDzGwwE7h4cRqLYaQEjLHyx\\\\\\\/+gZ9PfK4HzdUcuNDWvhSlzPb0QV908+X3fvWN2zHdXr1BWH53KOzVpZNpd0VLWmWq6hdFFKj6G447WwuQLj+pYm6zigh2LraL48BvSHET1U23R\\\\\\\/nc83u2Ft9pExA9BBUbWICZtt1GBd0AGIuG97d0dLp9mhMS0SwvbMnX5KRX3SLpi3qo9rkyYTeaRq0bx0T4lhMWrNdZ7qFwlteK7F9TMqJNP+U4ApcwgpGc3VVK3ywKJna+X8HaPBruM5cZwEGmQ4LH26OHXBB8NzE7KAFUtr1qsUGu9RcEPeSAVVYcu4weP+DJoty+920f3CHRm7o6NV6+d4r+f6TIB8D+K1B+yfQp5EMKYMW9Jl9UZaO1HS5NgF+j9hFEkASbYs8VbfOg3Sm2LVTs0ZkBqHsSWBdZ9i3Ytapjud276Wz9CgxJZgCub6ewX\\\\\\\/\\\\\\\/LvRUkiewpI6XobN71OekHUpBH4X9CZO1Hf+vcOIjFjHPLkW49TZiNb0+WkmtPRU0YdP1p8LfhnEeJyevhAJbNnMmONR9YHCf+3l\\\\\\\/T0ApVtxfc9pRz41oK6UGMy\\\\\\\/JthtEbQewe15Pfvp226WLbqH+IPulE7dth3lyovOsbaOzT3rHUQc84ZFTpY6+4REmfh3SNQljL1tT56vk9YAruaMu75fxkbYgYJZlYGYwz1aAiDK8Fl1WknQzLjv5lDSJMowjio1ztL6Sl7vxcflg0UwN7MUu+za5BrmXHswqaCQJ991iFxfBtFYkrNxqG9EBc0Hg\\\\\\\/JzSumHBM2ut2Abgb7YdMZ7IyG8kXOQavxhqRs1Ims0Mrb+8ajrZVILkWzf0WwvUSTm3kAcli\\\\\\\/ytLe5SS8ACBWib3nbzUFoI4yG9rTBCodvJfjTjLIuqoUYhRqGQ6NJJYzbBH0GKBlqLfn5W9NdGhqFYg9ijt4dQN928y9DKXvC0NNm0baRA9tMa7BP5DZSKCAm77yEld6Nn2HXjolD8uPA7tKQBuej1RJtD9AQrToHcoDheL309TcbidR6RmsIJ\\\\\\\/+mLOssqvmmQC4p9uUIZnWlQuwZw9pIYf+A2JOzZvsOfCB3ka6vyR+RuOhRWYUenJTOaGyuFAbzU9TIXX6jcUpDSgt0MhbSF9CHcSu+8KpJhMP0+EDeu+6NlHXEaCdGGUEC0A\\\\\\\/DqSisGK0kFAO5r9hkxly9CNzNTYrBHx+cY\\\\\\\/OLqjb4g==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Emc2obT1TdEQ4il9PpsHTw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a65ff502660fc936aa1788a6fe0a7b8d6c4f6efd0734ff8f5cdb3e70a0cd4a72\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022+ZY+c0emkMkLjt7CBJe3THmYkJ6M+bz6zXEeJEHY3nSeN7LB3ZAc\\\\\\\/JVQ3HmKALk7SXF6PvArFuydiMaEuqghi1O1AwZijkSimIeh2BlcWDFYlgrz2MabThG7QCmDOpFgsBSE1IjZlugn3RJ9K5Z7Ij492YvwbWyhO7F+degV2IjKEDROrs+roMDuf6JkUBG0dNMiRdABfsmRKuIJMI2Z4GtT7Qoa1CY1lIHde7v50AHYqYso6tFMaotyIDLzIZ5Ki\\\\\\\/gsfd46cxUy3bPtB7iAtF7hQ8WWIJD0sXUh3rpmDCNRaBj+UCWo7J5H8wwRGdNBhCRfM26uSYrhj1\\\\\\\/soLkVF5QiR6dCv3aAZ\\\\\\\/8\\\\\\\/Q6hf7bQIupC7IQISOaMvfFjQzaQSuuIW17teZHmycEvWmWT7EPp3NrFiVPj\\\\\\\/TrmgJNKG1anGGL+zEvg1OrrPqfdEJ205zMXmQl3YeBE7XIduqmDRXkRYctERKEtUkMoV3s6YAubOcUE5XzQ8UL6WdBqG2TC\\\\\\\/JW2CAy8LQdaiuLUCo8e+\\\\\\\/jmh0E3hD8mu046Gdzl9LohRT9R6zWkP\\\\\\\/0gq9FDPQuzaf0AnZeNS+cgyW4HQN6bjKmWtxeU43TWnZVsIu\\\\\\\/xCPsB8wBlG2SZX+\\\\\\\/j8woqgZyfQWVlutE8hX5zKjOuAcXS4tcWhCQKTuIAW0IdvWh+lsdYWBvdAAx+oaYZ8k6xRqDhVfP9ZlmzoVl6ibUpm8vAsCEP231QZn7N2mj0fJumP2YEsWS+R2Z4HqcOVnCNjCZVbFhQbPs0mBdZnX3t8IgL5sxUPYthgVyRi6Qzm5DQcXduRZ1NImKvCI9VoFCZwt31vqFofYaSHPyJjEPnTeY9fNbu9Z6p\\\\\\\/9Sd0JugphdhlISbXww9y9PZsf1497nlSWCb8wyM0JaIUTD+9IMQBSwqDBuZ1U9Km1rSI9RiRwqROrYBzTa9qo1ICO\\\\\\\/Nj6zg5Qzb8CyNNDhMyijccQSwA56eDm5WhqcekmqW87\\\\\\\/mjnZajg++KlOOhFYCC5rtibB4DeEuto5seyWXmhFEM87L1gop9EEJ\\\\\\\/7kTTEn0O7l17Q\\\\\\\/1a4Ng1984YcTIsh1Umy5s8\\\\\\\/pQD8MlF8PUeSOl\\\\\\\/8tyeE1f0p5HwgEaCTimxP+yXx1Zkt7PNId5BD7iCuaWWEfWP7rdJt4RTRQRYSWY9stB\\\\\\\/OoJHoI4UvqrCoU87B5pr\\\\\\\/g2mMKQ67OzCsm1hO06HWG\\\\\\\/+tDJXbdtQEtrTAeQb51sfkfmxv7jcsLtYmEnJ5dJD\\\\\\\/\\\\\\\/+r9b4gNaaFWHi0caB1R0f5s+0bIYnGeSXnhzQ+9m06h56JYkMZMSHjTZx8f3001\\\\\\\/EX\\\\\\\/jQlSZrvC1iZwB59wbdbtThp429TZx8fzctUjNUp8j+i5eMOW\\\\\\\/0Pdrse1OB2bepqzsVwly1ASH5G3b8W7TYbd1C7Lqe7ajogCVXu9VTO+qYogeOE13WSEVc8HdAPUfV7ndK2iR6HOI8izqGXrnr3vrb3GdbMQGYcZnXpVcCC1lKVY2CvqP0fLrFER0lQ6\\\\\\\/PaSwk5mSlq5JtT5nJhWkBl147RaoBDhLjLv3PnX12wzuSdXOeYISgwcaISXDIOxziVg0yvR4yp7rEn3qf0Ge7peXQcyimdq9e60QszT4FaCMX3DhY+ShZnPTneTu2NrtBs7x2+ckqM9R4J2MB+R9Fp\\\\\\\/0Ev4etxrKKZ2Tmc9VL+cU7k4PaENzhXLtT6HHzru\\\\\\\/ZkqI9Nq1Oq9WWnW94QUboa6SrwmBsNmvRh+41Ds\\\\\\\/yNQzuawziSt44Na2XH8ax448ZT0qIHiupoFt8kmPh9VOJaZJRJRv523b2LEo4BoH7LEtRFrJrSmLBuw\\\\\\\/tdQUNsolfdxh\\\\\\\/+aHKoPKJiOpA7SQP+o0mRPSzVTgrmVe7Q3JEi0a\\\\\\\/mmzC8xZY3YUIHhiWECzamqXPVExdEqw5r7nKbQqNSc29MpkyxHaYjk20+DtdK+CbHyA4z\\\\\\\/0H3aslRmfcYuoZ\\\\\\\/jOlUw\\\\\\\/hpx7RhdcJ0mnnXhiwny2ZGKO331Zq168p\\\\\\\/sP4Xug==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Osd9dtw2uYq3ISSB0Um4OA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002295e1898748fd5d087c88358d97e31ffd81448b75698888963a6310a6d2252a50\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460549 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/2/449" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-mJJMF7GkISA8ZkK0xYCMJcl6bczbL9Tt_0evLzdSBJY" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/2/449" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-lS6kZOe2EziBXJfPRSUQhN6fpK6duhPWJGLdP_PCXEU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/2/437" title="Screening and Analysis of Janelia FlyLight Project Enhancer-Gal4 Strains Identifies Multiple Gene Enhancers Active During Hematopoiesis in Normal and Wasp-Challenged Drosophila Larvae" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/2/467" title="Transcriptome Analysis of Drosophila melanogaster Third Instar Larval Ring Glands Points to Novel Functions and Uncovers a Cytochrome p450 Required for Development" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460549" id="node460549" data-pisa="ggg;7/2/449" data-pisa-master="ggg;g3.116.035402" data-apath="/ggg/7/2/449.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Alternative Oxidase Transcription Factors AOD2 and AOD5 of <em>Neurospora crassa</em> Control the Expression of Genes Involved in Energy Production and Metabolism</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Zhigang Qi</span>, <span class="highwire-citation-author" data-delta="1">Kristina M. Smith</span>, <span class="highwire-citation-author" data-delta="2">Erin L. Bredeweg</span>, <span class="highwire-citation-author" data-delta="3">Natasa Bosnjak</span>, <span class="highwire-citation-author" data-delta="4">Michael Freitag</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-1383-4720" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Frank E. Nargang</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">449-466;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.035402 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460549" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zhigang Qi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences, University of Alberta, Edmonton, Alberta T6G 2E9, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhigang%2BQi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Qi%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhigang%2BQi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Kristina M. Smith </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biochemistry and Biophysics, Oregon State University, Corvallis, Oregon 97331-4003</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kristina%2BM.%2BSmith%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Smith%20KM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKristina%2BM.%2BSmith%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Erin L. Bredeweg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biochemistry and Biophysics, Oregon State University, Corvallis, Oregon 97331-4003</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Erin%2BL.%2BBredeweg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bredeweg%20EL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AErin%2BL.%2BBredeweg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Natasa Bosnjak </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences, University of Alberta, Edmonton, Alberta T6G 2E9, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Natasa%2BBosnjak%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bosnjak%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANatasa%2BBosnjak%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Michael Freitag </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biochemistry and Biophysics, Oregon State University, Corvallis, Oregon 97331-4003</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BFreitag%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Freitag%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BFreitag%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Frank E. Nargang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences, University of Alberta, Edmonton, Alberta T6G 2E9, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Frank%2BE.%2BNargang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nargang%20FE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFrank%2BE.%2BNargang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1383-4720" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Frank E. Nargang</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:frank.nargang@ualberta.ca" class="" data-icon-position="" data-hide-link-title="0">frank.nargang@ualberta.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/2/449" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460549" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460549/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/2/449.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460549" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460549/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/2/449.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460549" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460549/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/2/449.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460549" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460549/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/2/449.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460549" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460549/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">In <em>Neurospora crassa</em>, blocking the function of the standard mitochondrial electron transport chain results in the induction of an alternative oxidase (AOX). AOX transfers electrons directly from ubiquinol to molecular oxygen. AOX serves as a model of retrograde regulation since it is encoded by a nuclear gene that is regulated in response to signals from mitochondria. The <em>N. crassa</em> transcription factors AOD2 and AOD5 are necessary for the expression of the AOX gene. To gain insight into the mechanism by which these factors function, and to determine if they have roles in the expression of additional genes in <em>N. crassa</em>, we constructed strains expressing only tagged versions of the proteins. Cell fractionation experiments showed that both proteins are localized to the nucleus under both AOX inducing and noninducing conditions. Furthermore, chromatin immunoprecipitation and high throughput sequencing (ChIP-seq) analysis revealed that the proteins are bound to the promoter region of the AOX gene under both conditions. ChIP-seq also showed that the transcription factors bind to the upstream regions of a number of genes that are involved in energy production and metabolism. Dependence on AOD2 and AOD5 for the expression of several of these genes was verified by quantitative PCR. The majority of ChIP-seq peaks observed were enriched for both AOD2 and AOD5. However, we also observed occasional sites where one factor appeared to bind preferentially. The most striking of these was a conserved sequence that bound large amounts of AOD2 but little AOD5. This sequence was found within a 310 bp repeat unit that occurs at several locations in the genome.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/alternative-oxidase" class="hw-term hw-article-keyword hw-article-keyword-alternative-oxidase" rel="nofollow">alternative oxidase</a></li><li class="kwd"><a href="/keyword/chip-seq" class="hw-term hw-article-keyword hw-article-keyword-chip-seq" rel="nofollow">ChIP-seq</a></li><li class="kwd"><a href="/keyword/aod2" class="hw-term hw-article-keyword hw-article-keyword-aod2" rel="nofollow">AOD2</a></li><li class="kwd"><a href="/keyword/aod5" class="hw-term hw-article-keyword hw-article-keyword-aod5" rel="nofollow">AOD5</a></li><li class="kwd"><a href="/keyword/retrograde-signaling" class="hw-term hw-article-keyword hw-article-keyword-retrograde-signaling" rel="nofollow">retrograde signaling</a></li></ul><p id="p-3">Proper mitochondrial function requires coordinated expression between the mitochondrial and nuclear genomes. Although there is considerable variation in the coding capacity of mitochondrial DNA among different groups of eukaryotes (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bevan and Lang 2004</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Burger <em>et al.</em> 2013</a>), in all organisms the overall number of mitochondrial proteins encoded by the mitochondrial genome is many fold less than the number of mitochondrial proteins encoded by genes housed in the nucleus (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Elstner <em>et al.</em> 2009</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Meisinger <em>et al.</em> 2008</a>). Thus, it is crucial that conditions affecting mitochondrial function or development be communicated to the nucleus so that expression of nuclear genes can be adjusted to meet the demands of changing conditions. The term retrograde regulation refers to changes in the expression of nuclear encoded genes in response to signals received from mitochondria (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Liu and Butow 2006</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Jazwinski 2013</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">da Cunha <em>et al.</em> 2015</a>).</p><p id="p-4">The standard electron transport chain (sETC) of mitochondria contains four large enzyme complexes as well as two smaller electron carriers, ubiquinone and cytochrome <em>c</em>, which shuttle electrons extracted from reduced electron carriers to molecular oxygen. Electron transfer is coupled to the pumping of protons across the mitochondrial inner membrane to establish a gradient that is harnessed by ATP synthase to make ATP (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Lenaz and Genova 2009</a>). Many organisms are able to produce an alternative oxidase (AOX) that provides a branch point from the sETC, which enables direct transfer of electrons from ubiquinol to oxygen, thus bypassing the later stages of the sETC (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Moore <em>et al.</em> 2013</a>). Genes encoding AOX exist in many species of plants, protists, fungi, primitive animals, and α-proteobacteria (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">McDonald <em>et al.</em> 2003</a>, <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">2009</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">McDonald 2008</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">McDonald and Vanlerberghe 2006</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Neimanis <em>et al.</em> 2013</a>). In eukaryotes, AOX genes are housed in the nucleus. Thus, the protein is synthesized in the cytosol and imported into mitochondria where it localizes to the matrix side of the mitochondrial inner membrane.</p><p id="p-5">In most fungi, AOX is undetectable, or present at very low levels, under standard laboratory growth conditions. However, inhibition of the sETC results in induction of AOX. This may occur via the action of chemical inhibitors, such as antimycin A or cyanide, which affect specific complexes of the sETC, or by mutations that decrease the function of sETC components (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Tissieres <em>et al.</em> 1953</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Edwards <em>et al.</em> 1976</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bertrand <em>et al.</em> 1983</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Yukioka <em>et al.</em> 1998</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Huh and Kang 2001</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Dufour <em>et al.</em> 2000</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Borghouts <em>et al.</em> 2001</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Sakajo <em>et al.</em> 1993</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Shi <em>et al.</em> 1999</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Kirimura <em>et al.</em> 1996</a>). Chemicals that specifically block mitochondrial protein synthesis, such as chloramphenicol (Cm), also induce AOX because they inhibit synthesis of mitochondrially encoded subunits of the sETC complexes (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Tanton <em>et al.</em> 2003</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Descheneau <em>et al.</em> 2005</a>). Thus, AOX appears to provide an escape from conditions that block the function of the latter stages of the sETC. This allows continued ATP production via proton pumping at Complex I, and recycling of reduced electron carriers. Induction of AOX in response to a dysfunctional sETC represents a classic example of retrograde regulation.</p><p id="p-6">Although the exact pathway and signals required for fungal AOX induction are not known, studies in <em>Neurospora crassa</em> (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Chae <em>et al.</em> 2007b</a>), <em>Podospora anserina</em> (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Sellem <em>et al.</em> 2009</a>), and <em>Aspergillus nidulans</em> (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Suzuki <em>et al.</em> 2012</a>) have shown that two zinc cluster transcription factors are required for the expression of AOX in response to sETC inhibition. The <em>N. crassa</em> proteins, called AOD2 and AOD5, are known to form a heterodimer that binds an alternative oxidase induction motif (AIM) consisting of two CGG triplets separated by seven nucleotides, found in the promoter region of the AOX-encoding <em>aod-1</em> gene (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Chae <em>et al.</em> 2007a</a>,<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">b</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Chae and Nargang 2009</a>). In addition to their role in AOX expression, the orthologs of AOD2 and AOD5 in <em>P. anserina</em> (RSE2 and RSE3, respectively) and <em>A. nidulans</em> (AcuK and AcuM, respectively) are also known to be required for the expression of phosphoenolpyruvate carboxykinase (PEPCK) and fructose-1,6-bisphosphatase (FBP), two enzymes which are crucial for the process of gluconeogenesis (<a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Sellem <em>et al.</em> 2009</a>; <a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Suzuki <em>et al.</em> 2012</a>). These observations hint at a larger role for the transcription factors in cell growth and metabolism; indeed, a recent microarray study in <em>P. anserina</em> identified 598 genes whose expression is influenced by RSE2 and RSE3 (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bovier <em>et al.</em> 2014</a>).</p><p id="p-7">Here, we further define the mechanism of regulation that AOD2 and AOD5 play in <em>N. crassa</em>. The intracellular location of the transcription factors under conditions that do and do not induce transcription of AOX was examined. AOD2 and AOD5 were found to be constitutively localized to the nucleus. As in the other fungi previously examined, we found that the proteins are required for the expression of PEPCK, but we observed no effect on FBP expression. In addition, we conducted chromatin immunoprecipitation and high throughput sequencing (ChIP-seq), which showed that the proteins bind to the promoters of their target genes in both inducing or noninducing growth conditions. The ChIP-seq study also showed a wider involvement of the factors in controlling aspects of cell metabolism outside of AOX and gluconeogenesis.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Strains and growth of N. crassa</h3><p id="p-8">General handling and growth of <em>N. crassa</em> strains was as described (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Davis and De Serres 1970</a>). Unless otherwise specified, cells were grown using Vogel’s medium (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Davis and De Serres 1970</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Metzenberg 2003</a>) with 44 mM sucrose as the carbon source. Experiments to measure growth rate and transcript levels in different carbon sources were done using synthetic crossing medium, which contains less available nitrogen (10 mM nitrate) compared with Vogel’s medium (25 mM nitrate and 25 mM ammonium). Carbon sources used were 44 mM sucrose, 217 mM glycerol, 217 mM ethanol, or 150 mM sodium acetate. When indicated, cells were grown in the presence of Cm at a final concentration of 2 mg/ml. Growth in the presence of Cm was used to achieve inducing conditions for expression of AOX. Unless stated otherwise, cultures were grown for 18 hr (in the absence of Cm), or 20 hr (in the presence of Cm) at 30°. Strains used in this study are described in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470314/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470314/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1 N.</span> <span class="caption-title"><em>crassa</em> strains used in this study</span><div class="sb-div caption-clear"></div></div></div><p id="p-9">For localization studies and ChIP-seq analysis, strains were developed that expressed only tagged versions of either AOD2 or AOD5. The tags were either triple myc or triple hemagglutinin (HA). Tags were inserted at either the N- or C-terminus of the proteins (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Genes encoding the tagged proteins were individually inserted into a vector (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Staben <em>et al.</em> 1989</a>) carrying hygromycin resistance (HygR). The resulting plasmids were transformed into either an <em>aod-2</em> or <em>aod-5</em> mutant strain. Transformed strains were selected by plating on hygromycin-containing medium. Initial isolates were further purified by single colony isolation on hygromycin-containing medium. To demonstrate that a functional tagged AOD2 or AOD5 protein was present, the transformants were examined for their ability to grow at the wild-type rate in the presence of antimycin A (final concentration 0.5 µg/ml). Finally, they were examined for their ability to produce wild-type levels of AOX when grown in the presence of Cm. The isolates described in <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a> were chosen for further work.</p><p id="p-10">For coimmunoprecipitation experiments, a strain was developed that expressed tagged versions of both AOD2 and AOD5. The double <em>aod-2 aod-5</em> mutant strain DX-13 (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>) was cotransformed with a plasmid carrying an <em>aod-2</em> gene with a triple myc tag at its C-terminus and a plasmid with an <em>aod-5</em> gene with a triple HA tag at its N-terminus. Both plasmids also carried hygromycin resistance. Transformants were selected and analyzed as described above. The double-tagged strain chosen for analysis was AOD2-C-Myc AOD5-N-HA A1 (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>).</p></div><div id="sec-3" class="subsection"><h3>Isolation of nuclei</h3><p id="p-11">The protocol for isolation of nuclei was as described previously (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Talbot and Russell 1982</a>), except that cells were broken while frozen in liquid nitrogen using a mortar and pestle, rather than a French press.</p></div><div id="sec-4" class="subsection"><h3>Isolation of mitochondria</h3><p id="p-12">Mitochondria were isolated as previously described (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Nargang and Rapaport 2007</a>). When highly purified mitochondria were required for analysis in subcellular localization experiments, isolated mitochondria were further purified using sucrose flotation gradients (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Lambowitz 1979</a>).</p></div><div id="sec-5" class="subsection"><h3>Cytosol and microsome isolation</h3><p id="p-13">To isolate cytosol and microsomes (postmitochondrial pellet; PMP) the mitochondrial isolation procedure (see <em>Isolation of mitochondria</em>) was followed. However, after pelleting the mitochondria by centrifugation, 1 ml of the supernatant was processed further. The supernatant sample was centrifuged at 48,000 rpm (27,915 × <em>g</em>) in a Beckman TLA-55 rotor for 1.5 hr at 4°. The pellet was suspended in 150 μl SEMP buffer [0.25 M sucrose, 10 mM MOPS (pH 7.2), 1 mM EDTA, 1 mM PMSF] as the microsomal or PMP fraction, whereas the supernatant was saved as the cytosolic fraction.</p></div><div id="sec-6" class="subsection"><h3>Coimmunoprecipitation</h3><p id="p-14">To extract nuclear proteins, purified nuclei (100 μg protein) were suspended in 60 μl of suspension buffer [25 mM sucrose, 50 mM Tris-HCl (pH 7.5), 5 mM MgCl<sub>2</sub>, 10 mM CaCl<sub>2</sub>] and mixed with 60 μl of 0.4 M KCl containing 1 mM PMSF and protease inhibitors (final concentrations of 2 μg/ml aprotinin, 1 μg/ml leupeptin, and 1 μg/ml pepstatin A). The suspension was gently rocked for 2 hr at 4°. Insoluble material was removed by centrifugation at 13,000 rpm (16,060 × <em>g</em>) for 30 min at 4° in a Sorvall Biofuge fresco centrifuge. The supernatant containing salt-extracted proteins was loaded onto a desalting column (Zeba Spin Desalting column; Thermo Scientific, Rockford, IL) which was placed into a fresh 1.5 ml Eppendorf tube. The desalting column was centrifuged at 1500 × <em>g</em> (4000 rpm) for 2 min at 4° in a Sorvall Biofuge fresco centrifuge. The flow-through was immediately used in coimmunoprecipitation experiments with the Pierce ProFoundTM HA or c-Myc Tag IP/Co-IP kit (Thermo Scientific).</p></div><div id="sec-7" class="subsection"><h3>ChIP-seq</h3><p id="p-15">ChIP-seq was performed on eight separate samples. Strain AOD2-C-HA-8 (expresses C-terminal triple HA–tagged AOD2) was grown in both the presence and absence of Cm. ChIP was done on both samples using an antibody against the HA tag. As controls, wild-type cells (NCN251) were also grown in both the presence and absence of Cm. ChIP was also done on these samples using the same antibody to the HA tag. Similarly, strain AOD5-C-Myc-4 (expresses C-terminal triple myc–tagged AOD5) was grown in the presence and absence of Cm and ChIP was performed on both samples using antibody against the myc tag. Again, control wild-type cultures were grown in the presence and absence of Cm and ChIP was performed using the same antibody against the myc tag. This resulted in four data sets, when controls were subtracted from experimental results.</p><p id="p-16">The protocol for ChIP was as described (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Guo <em>et al.</em> 2010</a>). Cultures were inoculated with 2.5 × 10<sup>8</sup> conidia in 100 ml Vogel’s medium, and were grown for 12 hr at 30° in the absence of Cm, or 14 hr in the presence of Cm. The final isolated ChIP-seq DNA was further treated using a Covaris S2 ultrasonicator to obtain DNA fragments with an average size of 200 bp for use in library construction. The sequence data for determination of binding sites of AOD2 following growth in the presence of Cm were obtained using an Illumina HiScan SQ with read lengths of 51 nucleotides. All other data were obtained using an Illumina HiSeq2000 with read lengths of 101 nucleotides. Covaris treatment, library construction, and DNA sequencing were performed by Delta Genomics (Alberta, Canada).</p><p id="p-17">Reads were mapped to <em>N. crassa</em> Broad Institute Assembly 10 with BWA using default parameters (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Li and Durbin 2009</a>), and peaks were called using MACS2 (Model-based Analysis of ChIP-Seq) software (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Zhang <em>et al.</em> 2008</a>; <a href="https://github.com/taoliu/MACS/">https://github.com/taoliu/MACS/</a>). The <em>q</em> value cutoff was set to 0.05. However, after selecting those peaks in each data set with fourfold or higher enrichment over the corresponding control, the highest <em>q</em> value in the data sets was 0.0022. The online program “Venny” was used to determine peaks in common to different data sets (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Oliveros 2007–2015</a>). Data were further analyzed by visual inspection of peaks in gbrowse (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Stein <em>et al.</em> 2002</a>; <a href="http://ascobase.cgrb.oregonstate.edu/cgi-bin/gb2/gbrowse/ncrassa_public/">http://ascobase.cgrb.oregonstate.edu/cgi-bin/gb2/gbrowse/ncrassa_public/</a>). Functional Catalogue (FunCat) analysis of genes (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Ruepp <em>et al.</em> 2004</a>) was done using FungiFun2 analysis (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Priebe <em>et al.</em> 2015</a>). Searches for consensus DNA binding sites were done using the SCOPE search tool (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Chakravarty <em>et al.</em> 2007</a>).</p></div><div id="sec-8" class="subsection"><h3>RNA isolation and qPCR</h3><p id="p-18">Cultures were grown at 30° with constant shaking for 12 hr (in the absence of Cm) or 14 hr (in the presence of Cm). RNA was isolated using the QIAGEN (Ontario, Canada) RNeasy Plant Mini Kit. RNA integrity was measured using an Agilent 2100 Bioanalyzer system and the Agilent RNA 6000 Nano kit (Agilent Technologies). Only those samples that had an RNA integrity number value of &gt;9 were used for cDNA synthesis. First-strand cDNA was synthesized using SuperScript III Reverse Transcriptase (Invitrogen). Reactions were performed using KAPA SYBR Fast qPCR reagents (KAPA Biosystems) and the StepOnePlus qPCR system (Applied Biosystems, Foster City, CA). Data were analyzed using StepOne software (Applied Biosystems). The β-tubulin gene was the internal control for ΔΔCt quantification. The qPCR data for each gene examined were obtained from four biological replicates, each with three technical replicates.</p></div><div id="sec-9" class="subsection"><h3>General techniques</h3><p id="p-19">Electrophoresis of proteins (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Laemmli 1970</a>), Western blots (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Good and Crosby 1989</a>), and electroporation of conidia to transform <em>N. crassa</em> (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Hoppins <em>et al.</em> 2007</a>) were performed as described previously.</p></div><div id="sec-10" class="subsection"><h3>Data availability</h3><p id="p-20">Strains and constructs are available upon request. Data are available from the SRA under BioProject accession no. PRJNA341311.</p></div></div><div class="section results" id="sec-11"><h2 class="">Results</h2><div id="sec-12" class="subsection"><h3>Localization of AOD2 and AOD5</h3><p id="p-21">Although <em>N. crassa</em> contains two genes encoding AOX, <em>aod-1</em> and <em>aod-3</em>, no conditions have been identified that result in the expression of <em>aod-3</em> (<a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Tanton <em>et al.</em> 2003</a>). Thus, any measurable AOX activity in the organism is derived from the AOD1 protein. The <em>aod-1</em> gene is transcribed at low levels under normal growth conditions. However, growth in the presence of inducers increases expression of both the transcript and the protein many fold (<a id="xref-ref-65-3" class="xref-bibr" href="#ref-65">Tanton <em>et al.</em> 2003</a>; <a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Chae <em>et al.</em> 2007b</a>). Increased expression is dependent on the transcription factors, AOD2 and AOD5. EMSA analysis demonstrated that the two proteins bound as a heterodimer to an AIM upstream of the <em>aod-1</em> gene <em>in vitro</em> (<a id="xref-ref-10-5" class="xref-bibr" href="#ref-10">Chae <em>et al.</em> 2007b</a>). To further understand the mechanism by which these transcription factors induce <em>aod-1</em> expression, we wished to determine if the amount or subcellular location of either AOD2 or AOD5 was affected by growth of cells in AOX-inducing conditions compared with noninducing conditions. To monitor the location of the proteins, we developed strains expressing AOD2 or AOD5 proteins with triple HA or myc tags at either their N- or C-terminus. The tagged strains were grown in the presence (AOX-inducing conditions) or absence (AOX-noninducing conditions) of Cm. Cells were harvested and various subcellular fractions were isolated. Analysis of strains expressing a C-terminal HA-tagged AOD2 (AOD2-HA) and a C-terminal myc-tagged AOD5 (AOD5-myc) revealed that both proteins were only detectable in the nuclear fraction and their localization was not influenced by growth in the presence or absence of Cm (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1, A and B</a>). The levels of the proteins did not appear to change substantially between the two growth conditions. To examine the possibility that the tagged proteins were simply mislocalized due to the location of the tags on the C-terminus, we also examined subcellular fractions from a strain expressing an N-terminal HA-tagged AOD2 (HA-AOD2) and a strain expressing an N-terminal HA-tagged AOD5 (HA-AOD5). Localization results were virtually identical to those observed for the C-terminal tagged proteins (Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035402/-/DC1/FigureS1.ai">Figure S1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/449/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="AOD2 and AOD5 are localized to the nucleus. (A) Strain AOD2-C-HA-8 (2HA) expresses an AOD2 protein with a triple HA tag at its carboxy-terminus. The strain was grown in both the presence and absence of Cm as was the control strain (ct) NCN251. Mycelium was harvested and different cellular fractions were isolated as described in Materials and Methods. Samples from the nuclear (nuclei, 20 &#xB5;g protein), cytosolic (20 &#xB5;g protein), mitochondrial (mito; 20 &#xB5;g protein), and postmitochondrial pellet [(PMP) containing endoplasmic reticulum/microsomes, 20 &#xB5;g protein] fractions were electrophoresed on SDS-PAGE gels along with molecular weight standards. Proteins were blotted to nitrocellulose membrane. The membrane was cut into strips according to the known molecular weights of the proteins to be detected as markers for the different fractions (nuclei: histone H3; cytosol: arginase; mitochondria: Tom70 and AOX; PMP: Kar2) or the HA-tagged AOD2 protein. Antibodies used for detection are shown on the right side of the blots. (B) As in (A), except that strain AOD5-C-Myc-4 (5myc), which expresses AOD5 containing a triple myc tag at its carboxy-terminus, was used." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1992884807" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;AOD2 and AOD5 are localized to the nucleus. (A) Strain AOD2-C-HA-8 (2HA) expresses an AOD2 protein with a triple HA tag at its carboxy-terminus. The strain was grown in both the presence and absence of Cm as was the control strain (ct) NCN251. Mycelium was harvested and different cellular fractions were isolated as described in &lt;em&gt;Materials and Methods&lt;/em&gt;. Samples from the nuclear (nuclei, 20 &#xB5;g protein), cytosolic (20 &#xB5;g protein), mitochondrial (mito; 20 &#xB5;g protein), and postmitochondrial pellet [(PMP) containing endoplasmic reticulum/microsomes, 20 &#xB5;g protein] fractions were electrophoresed on SDS-PAGE gels along with molecular weight standards. Proteins were blotted to nitrocellulose membrane. The membrane was cut into strips according to the known molecular weights of the proteins to be detected as markers for the different fractions (nuclei: histone H3; cytosol: arginase; mitochondria: Tom70 and AOX; PMP: Kar2) or the HA-tagged AOD2 protein. Antibodies used for detection are shown on the right side of the blots. (B) As in (A), except that strain AOD5-C-Myc-4 (5myc), which expresses AOD5 containing a triple myc tag at its carboxy-terminus, was used.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/449/F1.medium.gif" width="440" height="156"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/2/449/F1.medium.gif" width="440" height="156"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/449/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/449/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470390" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-22" class="first-child">AOD2 and AOD5 are localized to the nucleus. (A) Strain AOD2-C-HA-8 (2HA) expresses an AOD2 protein with a triple HA tag at its carboxy-terminus. The strain was grown in both the presence and absence of Cm as was the control strain (ct) NCN251. Mycelium was harvested and different cellular fractions were isolated as described in <em>Materials and Methods</em>. Samples from the nuclear (nuclei, 20 µg protein), cytosolic (20 µg protein), mitochondrial (mito; 20 µg protein), and postmitochondrial pellet [(PMP) containing endoplasmic reticulum/microsomes, 20 µg protein] fractions were electrophoresed on SDS-PAGE gels along with molecular weight standards. Proteins were blotted to nitrocellulose membrane. The membrane was cut into strips according to the known molecular weights of the proteins to be detected as markers for the different fractions (nuclei: histone H3; cytosol: arginase; mitochondria: Tom70 and AOX; PMP: Kar2) or the HA-tagged AOD2 protein. Antibodies used for detection are shown on the right side of the blots. (B) As in (A), except that strain AOD5-C-Myc-4 (5myc), which expresses AOD5 containing a triple myc tag at its carboxy-terminus, was used.</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">Our previous <em>in vitro</em> findings have shown that protein fragments containing the DNA-binding domains of AOD2 and AOD5 act as a heterodimer when binding DNA. To demonstrate that the two full-length proteins associated <em>in vivo</em>, we performed pull-down experiments using proteins extracted from nuclei isolated from a strain expressing differentially tagged versions of both proteins (AOD2-myc and HA-AOD5). We found that either tagged protein would coimmunoprecipitate the other (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>), demonstrating that the two full-length proteins do associate <em>in vivo</em>. Furthermore, the results were virtually identical regardless of whether cultures were grown in the presence or absence of Cm (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). Taken together, these data suggest that the expression of <em>aod-1</em> is mediated by an AOD2/AOD5 heterodimer. Changes in growth conditions do not result in changes of the levels or localization of AOD2 and AOD5. Furthermore, at least a fraction of each protein is associated in a heterodimer under both inducing and noninducing conditions.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/449/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="AOD2 and AOD5 associate with each other in vivo. Control strain NCN251 (ct) and strain AOD2-C-Myc AOD5-N-HA, which expresses both an AOD2 protein carrying a triple myc tag at its carboxy-terminus and an AOD5 protein carrying a triple HA tag at its amino-terminus, were grown both in the presence and absence of Cm. Mycelium was harvested and nuclear proteins were isolated (Input). Immunoprecipitation (IP) was performed with either anti-HA (A) or anti-myc antibodies (B) bound to agarose beads. The beads were washed and bound proteins were eluted as described in Materials and Methods. The elution fractions were subjected to SDS-PAGE and separated proteins were blotted to nitrocellulose membrane. The membranes were probed with the antibodies shown to the right of the panels. Histone 3 was used as a control for loading and purification." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1992884807" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;AOD2 and AOD5 associate with each other &lt;em&gt;in vivo&lt;/em&gt;. Control strain NCN251 (ct) and strain AOD2-C-Myc AOD5-N-HA, which expresses both an AOD2 protein carrying a triple myc tag at its carboxy-terminus and an AOD5 protein carrying a triple HA tag at its amino-terminus, were grown both in the presence and absence of Cm. Mycelium was harvested and nuclear proteins were isolated (Input). Immunoprecipitation (IP) was performed with either anti-HA (A) or anti-myc antibodies (B) bound to agarose beads. The beads were washed and bound proteins were eluted as described in &lt;em&gt;Materials and&lt;/em&gt; &lt;em&gt;Methods&lt;/em&gt;. The elution fractions were subjected to SDS-PAGE and separated proteins were blotted to nitrocellulose membrane. The membranes were probed with the antibodies shown to the right of the panels. Histone 3 was used as a control for loading and purification.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/449/F2.medium.gif" width="440" height="131"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/2/449/F2.medium.gif" width="440" height="131"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/449/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/449/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470266" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-24" class="first-child">AOD2 and AOD5 associate with each other <em>in vivo</em>. Control strain NCN251 (ct) and strain AOD2-C-Myc AOD5-N-HA, which expresses both an AOD2 protein carrying a triple myc tag at its carboxy-terminus and an AOD5 protein carrying a triple HA tag at its amino-terminus, were grown both in the presence and absence of Cm. Mycelium was harvested and nuclear proteins were isolated (Input). Immunoprecipitation (IP) was performed with either anti-HA (A) or anti-myc antibodies (B) bound to agarose beads. The beads were washed and bound proteins were eluted as described in <em>Materials and</em> <em>Methods</em>. The elution fractions were subjected to SDS-PAGE and separated proteins were blotted to nitrocellulose membrane. The membranes were probed with the antibodies shown to the right of the panels. Histone 3 was used as a control for loading and purification.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>AOD2 and AOD5 are required for optimal growth in different carbon sources</h3><p id="p-25">It has been shown in other filamentous fungi, <em>A. nidulans</em> (<a id="xref-ref-63-3" class="xref-bibr" href="#ref-63">Suzuki <em>et al.</em> 2012</a>) and <em>P. anserina</em> (<a id="xref-ref-57-3" class="xref-bibr" href="#ref-57">Sellem <em>et al.</em> 2009</a>), that the orthologs of AOD2 and AOD5 are not only required for the expression of AOX, but also for genes involved in gluconeogenesis. To evaluate this in <em>N. crassa</em>, we measured transcript levels of genes involved in gluconeogenesis in strains lacking either AOD2 or AOD5 following growth in different carbon sources. Since defects in ability to utilize certain carbon sources might affect the growth of the mutants, we also followed the growth rate of the mutant stains under these conditions. We had previously been unable to detect differences in colonial growth rate between controls and mutant <em>aod-2</em> or <em>aod-5</em> cells when tested on solid Vogel’s medium containing sorbose (<a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">Descheneau <em>et al.</em> 2005</a>). Therefore, in the current study, we measured growth in liquid medium containing the different carbon sources. In addition, since the classic retrograde regulation mutants of <em>Saccharomyces cerevisiae</em> have defects in nitrogen metabolism (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Liu and Butow 2006</a>), we used synthetic crossing medium as the source of inorganic salts since it contains a reduced overall nitrogen concentration relative to the standard Vogel’s salts and totally lacks ammonium (see <em>Materials and</em> <em>Methods</em>), which is a preferred nitrogen source for <em>Neurospora</em> (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Marzluf 1997</a>).</p><p id="p-26">When sucrose was the carbon source, the mutant strains grew at ∼50–60% of the rate of the control strain (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). However, when either acetate or glycerol was used as the carbon source, the growth rate of the mutant strains was more severely affected, decreasing to 10–20% of the control on glycerol or acetate and to virtually no growth when ethanol was used as the carbon source. These differences were not due to loss of the ability to produce the AOX AOD1 protein, since a knockout of AOD1 grew at near wild-type rates on all media (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/449/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Effect of carbon source on growth and transcript levels. (A) The control strain (ct) NCN251 and the &#x394;aod-1, &#x394;aod-2, and &#x394;aod-5 knockout strains were grown in low nitrogen-containing medium with either 44 mM sucrose, 217 mM glycerol, 150 mM sodium acetate, or 217 mM ethanol as the carbon source. For the different carbon sources, mycelium was harvested and weighed from these cultures after 20, 72, 72, and 96 hr of growth, respectively. The values shown are the average of three cultures of each strain and carbon source. Error bars show the SEM. (B) The ct NCN251 and the &#x394;aod-2 knockout strain were grown on different carbon sources, as described in (A) (S, sucrose; G, glycerol; A, acetate; E, ethanol), with or without Cm present in the medium as indicated. Mycelium was harvested and total RNA prepared. RT-qPCR was performed for AOD1 mRNA using cDNA generated from the RNA. Error bars shows the SEM. The Student&#x2019;s t-test was used to compare data between two different conditions. p  0.05; n.s., no significant difference. (C) Mitochondria were isolated from the ct NCN251 and the knockout strains &#x394;aod-1 and &#x394;aod-2 grown in the presence or absence of Cm, and in different carbon sources as indicated at the top of the panel. Mitochondrial proteins were subjected to SDS-PAGE and the proteins were transferred to nitrocellulose membrane. The blots were probed with the antibodies indicated on the right. Two different exposure times are shown for AOX (short exposure, long exposure). Tom40 and Tim8 serve as mitochondrial marker loading controls. (D and E) As for (B), except that the RT-qPCR was performed for PEPCK and FBP mRNA, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1992884807" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Effect of carbon source on growth and transcript levels. (A) The control strain (ct) NCN251 and the &#x394;&lt;em&gt;aod-1&lt;/em&gt;, &#x394;&lt;em&gt;aod-2&lt;/em&gt;, and &#x394;&lt;em&gt;aod-5&lt;/em&gt; knockout strains were grown in low nitrogen-containing medium with either 44 mM sucrose, 217 mM glycerol, 150 mM sodium acetate, or 217 mM ethanol as the carbon source. For the different carbon sources, mycelium was harvested and weighed from these cultures after 20, 72, 72, and 96 hr of growth, respectively. The values shown are the average of three cultures of each strain and carbon source. Error bars show the SEM. (B) The ct NCN251 and the &#x394;&lt;em&gt;aod-2&lt;/em&gt; knockout strain were grown on different carbon sources, as described in (A) (S, sucrose; G, glycerol; A, acetate; E, ethanol), with or without Cm present in the medium as indicated. Mycelium was harvested and total RNA prepared. RT-qPCR was performed for AOD1 mRNA using cDNA generated from the RNA. Error bars shows the SEM. The Student&#x2019;s &lt;em&gt;t&lt;/em&gt;-test was used to compare data between two different conditions. &lt;em&gt;p&lt;/em&gt; &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-1&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; 0.05; n.s., no significant difference. (C) Mitochondria were isolated from the ct NCN251 and the knockout strains &#x394;&lt;em&gt;aod-1&lt;/em&gt; and &#x394;&lt;em&gt;aod-2&lt;/em&gt; grown in the presence or absence of Cm, and in different carbon sources as indicated at the top of the panel. Mitochondrial proteins were subjected to SDS-PAGE and the proteins were transferred to nitrocellulose membrane. The blots were probed with the antibodies indicated on the right. Two different exposure times are shown for AOX (short exposure, long exposure). Tom40 and Tim8 serve as mitochondrial marker loading controls. (D and E) As for (B), except that the RT-qPCR was performed for PEPCK and FBP mRNA, respectively.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/449/F3.medium.gif" width="440" height="416"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/2/449/F3.medium.gif" width="440" height="416"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/449/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/449/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470429" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-27" class="first-child">Effect of carbon source on growth and transcript levels. (A) The control strain (ct) NCN251 and the Δ<em>aod-1</em>, Δ<em>aod-2</em>, and Δ<em>aod-5</em> knockout strains were grown in low nitrogen-containing medium with either 44 mM sucrose, 217 mM glycerol, 150 mM sodium acetate, or 217 mM ethanol as the carbon source. For the different carbon sources, mycelium was harvested and weighed from these cultures after 20, 72, 72, and 96 hr of growth, respectively. The values shown are the average of three cultures of each strain and carbon source. Error bars show the SEM. (B) The ct NCN251 and the Δ<em>aod-2</em> knockout strain were grown on different carbon sources, as described in (A) (S, sucrose; G, glycerol; A, acetate; E, ethanol), with or without Cm present in the medium as indicated. Mycelium was harvested and total RNA prepared. RT-qPCR was performed for AOD1 mRNA using cDNA generated from the RNA. Error bars shows the SEM. The Student’s <em>t</em>-test was used to compare data between two different conditions. <em>p</em> <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/449/F3/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/449/F3/embed/mml-math-1.gif"/></noscript></span></span> 0.05; n.s., no significant difference. (C) Mitochondria were isolated from the ct NCN251 and the knockout strains Δ<em>aod-1</em> and Δ<em>aod-2</em> grown in the presence or absence of Cm, and in different carbon sources as indicated at the top of the panel. Mitochondrial proteins were subjected to SDS-PAGE and the proteins were transferred to nitrocellulose membrane. The blots were probed with the antibodies indicated on the right. Two different exposure times are shown for AOX (short exposure, long exposure). Tom40 and Tim8 serve as mitochondrial marker loading controls. (D and E) As for (B), except that the RT-qPCR was performed for PEPCK and FBP mRNA, respectively.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">To determine if the reduced growth on poor carbon sources in the mutant strains was due to an effect on the expression of genes required for gluconeogenesis, we performed qPCR analysis on RNA isolated from an <em>aod-2</em> knockout strain and a control following growth under various conditions. Levels of the transcripts for AOD1, as well as the gluconeogenic enzymes PEPCK and FBP, were determined. As expected, the greatest increase in the <em>aod-1</em> transcript was seen in the cultures grown in the presence of Cm (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>). Slight increases in <em>aod-1</em> mRNA were also seen following growth in poor carbon sources compared with the amounts present in uninduced sucrose cultures, but these were not found to be statistically significant. Furthermore, no AOD1 protein was seen in mitochondria from cultures grown in poor carbon sources (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3C</a>). For PEPCK mRNA (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3D</a>), a small increase due to growth in the presence of Cm was observed, but statistical significance was borderline (<em>p</em> = 0.06). Larger increases were seen following growth in acetate and ethanol. Growth in glycerol as the carbon source had virtually no effect on the level of PEPCK transcript, probably because glycerol enters the gluconeogenic pathway after the action of PEPCK. The mRNA for FBP (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3E</a>) showed no increase in medium containing Cm, but was significantly increased in all three of the poor carbon source cultures compared with the sucrose culture. Interestingly, both AOD1 and PEPCK transcripts were severely reduced in cultures lacking AOD2, but FBP transcript abundance was unaffected by the absence of AOD2. We conclude that expression of the gene encoding PEPCK in <em>N. crassa</em> requires AOD2, and by extension AOD5, but FBP expression does not.</p></div><div id="sec-14" class="subsection"><h3>ChIP-seq analysis of AOD2 and AOD5 binding</h3><p id="p-29">The reduced growth of the <em>aod-2</em> and <em>aod-5</em> knockouts in sucrose medium without Cm suggested a wider role of the two transcription factors than simply gluconeogenesis and <em>aod-1</em> induction. Furthermore, our results suggested that AOD2 and AOD5 were not required for transcription of the gene encoding FBP, in contrast to results in <em>A. nidulans</em> (<a id="xref-ref-63-4" class="xref-bibr" href="#ref-63">Suzuki <em>et al.</em> 2012</a>) and <em>P. anserina</em> (<a id="xref-ref-57-4" class="xref-bibr" href="#ref-57">Sellem <em>et al.</em> 2009</a>). For these reasons, we wished to gain deeper insight into the function and roles of AOD2 and AOD5. We used a ChIP-seq approach to identify binding sites of the two proteins in the <em>Neurospora</em> genome. Four separate experiments were performed to examine the binding of AOD2 and AOD5 in cultures grown in either the presence or absence of Cm. Since antibodies against these proteins were not available, we utilized commercially available antibodies against the HA and myc antigens. This enabled immunoprecipitation from the strains expressing the AOD2-HA– and AOD5-myc–tagged proteins used in the localization experiments described above. To help eliminate false positive results, ChIP was also done using the same antibodies, on a control strain that did not contain tagged proteins. The ChIP-seq experiments and the number of sequence reads from each are shown in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>. The initial data from each experiment following MACS2 (<a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">Zhang <em>et al.</em> 2008</a>) analysis are shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035402/-/DC1/FileS1.xlsx">File S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035402/-/DC1/FileS2.xlsx">File S2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035402/-/DC1/FileS3.xlsx">File S3</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035402/-/DC1/FileS4.xlsx">File S4</a>. Read alignments to the <em>N. crassa</em> genome can be viewed at <a href="http://ascobase.cgrb.oregonstate.edu/cgi-bin/gb2/gbrowse/ncrassa_public/">http://ascobase.cgrb.oregonstate.edu/cgi-bin/gb2/gbrowse/ncrassa_public/</a>. Further investigation was done on all peaks with fourfold or greater enrichment relative to the appropriate control matched for growth condition and antibody.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470435/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470435/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">ChIP experiments</span><div class="sb-div caption-clear"></div></div></div><p id="p-30">Given that AOD2 and AOD5 have been shown to act as a heterodimer and that both proteins were found in the nuclear fraction, regardless of whether cultures were grown in inducing or noninducing conditions, we concentrated on peaks in common to all four experiments as the most likely to be authentic AOD2/AOD5 binding sites. Comparison of the data sets from all four experiments, using the Venn diagram list comparison program Venny (<a id="xref-ref-15-5" class="xref-bibr" href="#ref-15">Oliveros 2007–2015</a>), revealed 70 peaks in common to all four conditions (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Five of the peak sets were found to occur between the 3ʹ ends of two genes. It is conceivable that antisense transcripts arise from these genes in an AOD2/AOD5-dependent fashion, since <em>N. crassa</em> does produce antisense transcripts (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Arthanari <em>et al.</em> 2014</a>) and at least one of these is known to have a regulatory function (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Kramer <em>et al.</em> 2003</a>). However, we chose to focus our investigation on genes with binding sites upstream of their coding regions. Thus, these five sets were not considered further. For the remaining 65 peaks (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>) found in presumptive gene promoters, 32 were associated with one gene in the proper orientation, while in 33 cases, the genes were oriented such that the binding sites were upstream of divergently transcribed genes. For these 33, we cannot assign transcriptional control by AOD2/AOD5 to one of the genes without further investigation.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/449/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genes in common to all four experimental conditions examined by ChIP-seq analysis. Genes in each ChIP that were associated with binding greater than fourfold, compared with the nontagged wild-type control strain in each experimental condition, were identified. These were compared using the online Venn diagram generating program, Venny (Oliveros 2007&#x2013;2015). The individual ChIP experiments are indicated beside the appropriate oval. For further analysis, Venny also lists the genes occurring in any segment of the Venn diagram." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1992884807" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Genes in common to all four experimental conditions examined by ChIP-seq analysis. Genes in each ChIP that were associated with binding greater than fourfold, compared with the nontagged wild-type control strain in each experimental condition, were identified. These were compared using the online Venn diagram generating program, Venny (Oliveros 2007&#x2013;2015). The individual ChIP experiments are indicated beside the appropriate oval. For further analysis, Venny also lists the genes occurring in any segment of the Venn diagram.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/449/F4.medium.gif" width="440" height="358"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/2/449/F4.medium.gif" width="440" height="358"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/449/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/449/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470215" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-31" class="first-child">Genes in common to all four experimental conditions examined by ChIP-seq analysis. Genes in each ChIP that were associated with binding greater than fourfold, compared with the nontagged wild-type control strain in each experimental condition, were identified. These were compared using the online Venn diagram generating program, Venny (<a id="xref-ref-15-6" class="xref-bibr" href="#ref-15">Oliveros 2007–2015</a>). The individual ChIP experiments are indicated beside the appropriate oval. For further analysis, Venny also lists the genes occurring in any segment of the Venn diagram.</p><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470423/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470423/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Peaks in common to all four ChIP-seq experiments with over fourfold enrichment relative to controls</span><div class="sb-div caption-clear"></div></div></div><p id="p-39">Visual inspection of the 65 AOD2/AOD5 binding sites revealed a broad range of quality and characteristics in the peaks. For example, many peaks appeared robust and spanned roughly 0.5–1 kb at the binding site in all four experiments, as for the AOX-encoding <em>aod-1</em> gene (NCU07953) (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>). The peaks at several other sites were relatively smaller and/or were broad, sometimes extending over several kb, as for NCU06230, which encodes a Ser/Thr protein kinase (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a>). The most robust peaks in all four experiments, as judged by visual inspection, were seen in the region between the endosome-associated ubiquitin isopeptidase gene (NCU06939) and the hypothetical protein-encoding gene, NCU06940 (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5C</a>). Though there is variation among the different sets of peaks with respect to amount of binding, in virtually all cases, binding of both proteins was observed following growth in both the presence or absence of Cm.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/449/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Variation in peaks called by MACS2. Snapshots from gbrowse, where peaks from ChIP-seq experiments have been aligned with the N. crassa genome. The ratios of the y-axis in different immunoprecipitations were assigned based on the number of mappable sequence reads obtained in the individual experiments for the HA or myc antibodies. However, no attempt to standardize between the different antibodies was made since their relative efficiencies for immunoprecipitation may differ. (A) Peaks found in the upstream region of the aod-1 gene (encoding AOX, NCU07953). The position of genes in the region is shown at the top. The individual experiments are listed on the left with respect to the tagged protein in the strain used, the growth condition (&#xB1;Cm), and the antibody (Ab) used for immunoprecipitation. The control ChIP-seq experiments were performed on strain NCN251 (expresses no HA- or myc-tagged proteins), exactly as with the strains expressing tagged proteins. (B) As in (A), except the region containing the NCU06229 and NCU06230 genes is shown. (C) As in (A), except the region containing the NCU06939 and NCU06940 genes is shown. In (B) and (C), the y-axis scale was not adjusted so that the difference in height of the peaks relative to those in (A) would be emphasized." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1992884807" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Variation in peaks called by MACS2. Snapshots from gbrowse, where peaks from ChIP-seq experiments have been aligned with the &lt;em&gt;N. crassa&lt;/em&gt; genome. The ratios of the &lt;em&gt;y&lt;/em&gt;-axis in different immunoprecipitations were assigned based on the number of mappable sequence reads obtained in the individual experiments for the HA or myc antibodies. However, no attempt to standardize between the different antibodies was made since their relative efficiencies for immunoprecipitation may differ. (A) Peaks found in the upstream region of the &lt;em&gt;aod-1&lt;/em&gt; gene (encoding AOX, NCU07953). The position of genes in the region is shown at the top. The individual experiments are listed on the left with respect to the tagged protein in the strain used, the growth condition (&#xB1;Cm), and the antibody (Ab) used for immunoprecipitation. The control ChIP-seq experiments were performed on strain NCN251 (expresses no HA- or myc-tagged proteins), exactly as with the strains expressing tagged proteins. (B) As in (A), except the region containing the NCU06229 and NCU06230 genes is shown. (C) As in (A), except the region containing the NCU06939 and NCU06940 genes is shown. In (B) and (C), the &lt;em&gt;y&lt;/em&gt;-axis scale was not adjusted so that the difference in height of the peaks relative to those in (A) would be emphasized.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/449/F5.medium.gif" width="440" height="266"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/2/449/F5.medium.gif" width="440" height="266"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/449/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/449/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470453" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-40" class="first-child">Variation in peaks called by MACS2. Snapshots from gbrowse, where peaks from ChIP-seq experiments have been aligned with the <em>N. crassa</em> genome. The ratios of the <em>y</em>-axis in different immunoprecipitations were assigned based on the number of mappable sequence reads obtained in the individual experiments for the HA or myc antibodies. However, no attempt to standardize between the different antibodies was made since their relative efficiencies for immunoprecipitation may differ. (A) Peaks found in the upstream region of the <em>aod-1</em> gene (encoding AOX, NCU07953). The position of genes in the region is shown at the top. The individual experiments are listed on the left with respect to the tagged protein in the strain used, the growth condition (±Cm), and the antibody (Ab) used for immunoprecipitation. The control ChIP-seq experiments were performed on strain NCN251 (expresses no HA- or myc-tagged proteins), exactly as with the strains expressing tagged proteins. (B) As in (A), except the region containing the NCU06229 and NCU06230 genes is shown. (C) As in (A), except the region containing the NCU06939 and NCU06940 genes is shown. In (B) and (C), the <em>y</em>-axis scale was not adjusted so that the difference in height of the peaks relative to those in (A) would be emphasized.</p><div class="sb-div caption-clear"></div></div></div><p id="p-41">For 33 of the 98 genes listed in <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>, we used a qPCR approach to determine if their expression was dependent on AOD2 and/or AOD5. RNA was isolated from wild-type, Δ<em>aod-2</em>, and Δ<em>aod-5</em> cells, each grown in the presence and absence of Cm. Eleven of the genes examined showed greater than twofold decreased expression in cells lacking either AOD2 or AOD5 in both of the growth conditions (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>), suggesting a positive regulatory role for the transcription factors. Eight of these 11 genes also showed greater than twofold increase in expression in the presence of Cm in wild-type cells (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6A</a>), while three showed little to no change in expression due to the presence of Cm (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6B</a>). Thus, all 11 genes require AOD2 and AOD5 for maximal expression both in the presence and absence of Cm, but for a subset of three, their expression is virtually unaffected by the presence of Cm. Two genes showed a twofold or greater increase in expression in the absence of either AOD2 or AOD5 when cells were grown in the presence of Cm (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6C</a>). For these genes, the transcription factors appear to play a negative role in expression. One gene (NCU04874) showed increased expression in the absence of Cm in cells lacking AOD5 (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6C</a>). Interestingly, NCU04874 represents the <em>aod-3</em> gene. We had previously shown that <em>aod-3</em> encodes a second, potentially functional AOX, but could find no evidence of its expression on Northern blots under any condition tested (<a id="xref-ref-65-4" class="xref-bibr" href="#ref-65">Tanton <em>et al.</em> 2003</a>). Our data show that both AOD2 and AOD5 bind at the promoter region of <em>aod-3</em>, and suggest that at least AOD5 may play a role in repressing transcription. Thus, it appears that AOD2 and AOD5, most likely acting as a heterodimer, can serve as either positive or negative regulatory factors.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/449/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genes tested by qPCR from among the common set of 65. The control strain (ct) NCN251 and the &#x394;aod-2 and &#x394;aod-5 knockout strains were grown in both the presence and absence of Cm. Total RNA was isolated from each of the strains, and mRNA was converted to cDNA. qPCR was performed as described in Materials and Methods. The data for the strains and conditions used are indicated by different colors, as shown on the figure. The results shown are the average of four biological replicates for each strain, each of which was determined using three technical replicates. In each case, the value for the control grown without Cm was set as one and the other data are represented as the fold change relative to that standard. Error bars show the SEM. (A) Genes found to require AOD2 and AOD5 for maximal expression and also show induction by Cm. (B) As in (A), but expression of these genes is not induced twofold or greater by Cm. (C) Genes found to be negatively regulated by AOD2 and/or AOD5. (D) Genes exhibiting only a Cm effect on transcription that does not depend on AOD2 or AOD5. (E) Gene exhibiting induction by Cm that does require AOD2 and AOD5. Expression of this gene in the absence of Cm does not require AOD2 or AOD5." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1992884807" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genes tested by qPCR from among the common set of 65. The control strain (ct) NCN251 and the &#x394;&lt;em&gt;aod-2&lt;/em&gt; and &#x394;&lt;em&gt;aod-5&lt;/em&gt; knockout strains were grown in both the presence and absence of Cm. Total RNA was isolated from each of the strains, and mRNA was converted to cDNA. qPCR was performed as described in &lt;em&gt;Materials and&lt;/em&gt; &lt;em&gt;Methods&lt;/em&gt;. The data for the strains and conditions used are indicated by different colors, as shown on the figure. The results shown are the average of four biological replicates for each strain, each of which was determined using three technical replicates. In each case, the value for the control grown without Cm was set as one and the other data are represented as the fold change relative to that standard. Error bars show the SEM. (A) Genes found to require AOD2 and AOD5 for maximal expression and also show induction by Cm. (B) As in (A), but expression of these genes is not induced twofold or greater by Cm. (C) Genes found to be negatively regulated by AOD2 and/or AOD5. (D) Genes exhibiting only a Cm effect on transcription that does not depend on AOD2 or AOD5. (E) Gene exhibiting induction by Cm that does require AOD2 and AOD5. Expression of this gene in the absence of Cm does not require AOD2 or AOD5.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/449/F6.medium.gif" width="417" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/7/2/449/F6.medium.gif" width="417" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/449/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/449/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470507" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-42" class="first-child">Genes tested by qPCR from among the common set of 65. The control strain (ct) NCN251 and the Δ<em>aod-2</em> and Δ<em>aod-5</em> knockout strains were grown in both the presence and absence of Cm. Total RNA was isolated from each of the strains, and mRNA was converted to cDNA. qPCR was performed as described in <em>Materials and</em> <em>Methods</em>. The data for the strains and conditions used are indicated by different colors, as shown on the figure. The results shown are the average of four biological replicates for each strain, each of which was determined using three technical replicates. In each case, the value for the control grown without Cm was set as one and the other data are represented as the fold change relative to that standard. Error bars show the SEM. (A) Genes found to require AOD2 and AOD5 for maximal expression and also show induction by Cm. (B) As in (A), but expression of these genes is not induced twofold or greater by Cm. (C) Genes found to be negatively regulated by AOD2 and/or AOD5. (D) Genes exhibiting only a Cm effect on transcription that does not depend on AOD2 or AOD5. (E) Gene exhibiting induction by Cm that does require AOD2 and AOD5. Expression of this gene in the absence of Cm does not require AOD2 or AOD5.</p><div class="sb-div caption-clear"></div></div></div><p id="p-43">Examination by qPCR of the expression of 13 genes from <a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a> revealed no obvious effect with regard to the presence or absence of AOD2 or AOD5, nor was an effect seen following growth in the presence or absence of Cm (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035402/-/DC1/FigureS2.ai">Figure S2</a>). Conceivably, unknown conditions may elicit a response in these genes which is mediated by AOD2 and AOD5, with or without contributions from other protein factors.</p><p id="p-44">Unexpectedly, five genes showed less than twofold change in expression in the absence of AOD2 or AOD5, but did show two to fourfold increases in expression when cells were grown in the presence of Cm, regardless of whether AOD2 or AOD5 was present (<a id="xref-fig-6-6" class="xref-fig" href="#F6">Figure 6D</a>). Two of the genes in this category, NCU02474 [encoding TOM5, a component of the mitochondrial TOM complex (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Schmitt <em>et al.</em> 2005</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Sherman <em>et al.</em> 2005</a>)] and NCU08946 (hypothetical protein), were adjacent to divergently transcribed genes (NCU02475 and NCU08947, respectively) whose expression was reduced by loss of AOD2 and AOD5 (<a id="xref-fig-6-7" class="xref-fig" href="#F6">Figure 6, B and E</a>). Whether there is a relationship that affects the expression of these genes that are arranged in divergent orientation is unknown. NCU08947 is unusual because the ChIP-seq peaks appear to be within an intron near the 5′ end of the gene and they were not associated with an AIM. The expression of NCU08947 was not affected by loss of AOD2 or AOD5 in the absence of Cm, but both proteins were required for maximal expression in the presence of Cm (<a id="xref-fig-6-8" class="xref-fig" href="#F6">Figure 6E</a>).</p><p id="p-45">For most of the genes in <a id="xref-fig-6-9" class="xref-fig" href="#F6">Figure 6, A and B</a>, we observed a slight induction of transcription in the presence of Cm, even in the absence of either AOD2 or AOD5. It seemed possible that this might be due to the action of the other remaining transcription factor in the single mutant strains. To test this, we created a Δ<em>aod-2</em> Δ<em>aod-5</em> double mutant strain and examined transcript levels for certain genes requiring AOD2 and AOD5 for expression. There was virtually no change in transcript levels when the single mutants were compared with the double mutants (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7A</a>) for NCU07953 (AOD1) and NCU03408 (hypothetical protein). Although some small differences were found to be statistically significant, the overall changes were often much less than twofold and the biological significance seems questionable. However, the gene NCU10007 (malate synthase) appeared to have a slight reduction of transcription in the double mutant strain (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7B</a>). Overall, the slight induction in the presence of Cm still remained in the double mutant strain, arguing for the involvement of other factors in the response to the induction signal(s) produced by growth in the presence of Cm.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/449/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of transcript levels in a double aod-2 aod-5 mutant strain vs. single mutant strains. The results for the transcripts from the indicated genes, as shown in Figure 6, were compared with the results from the double mutant, as described in the legend to Figure 6. The Student&#x2019;s t-test was used to compare data between two different conditions. (A) Two genes with less than twofold difference between single and double mutants. (B) One gene showing a greater than twofold decrease in expression in the double mutant as compared with the single mutants. p  0.05; n.s., no significant difference." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1992884807" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Comparison of transcript levels in a double &lt;em&gt;aod-2 aod-5&lt;/em&gt; mutant strain &lt;em&gt;vs.&lt;/em&gt; single mutant strains. The results for the transcripts from the indicated genes, as shown in Figure 6, were compared with the results from the double mutant, as described in the legend to Figure 6. The Student&#x2019;s &lt;em&gt;t&lt;/em&gt;-test was used to compare data between two different conditions. (A) Two genes with less than twofold difference between single and double mutants. (B) One gene showing a greater than twofold decrease in expression in the double mutant as compared with the single mutants. &lt;em&gt;p&lt;/em&gt; &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-2&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; 0.05; n.s., no significant difference.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/449/F7.medium.gif" width="440" height="118"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/7/2/449/F7.medium.gif" width="440" height="118"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/449/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/449/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470385" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-46" class="first-child">Comparison of transcript levels in a double <em>aod-2 aod-5</em> mutant strain <em>vs.</em> single mutant strains. The results for the transcripts from the indicated genes, as shown in <a id="xref-fig-6-10" class="xref-fig" href="#F6">Figure 6</a>, were compared with the results from the double mutant, as described in the legend to <a id="xref-fig-6-11" class="xref-fig" href="#F6">Figure 6</a>. The Student’s <em>t</em>-test was used to compare data between two different conditions. (A) Two genes with less than twofold difference between single and double mutants. (B) One gene showing a greater than twofold decrease in expression in the double mutant as compared with the single mutants. <em>p</em> <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/449/F7/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/2/449/F7/embed/mml-math-2.gif"/></noscript></span></span> 0.05; n.s., no significant difference.</p><div class="sb-div caption-clear"></div></div></div><p id="p-47">Previous EMSA data has shown that the AIM (CGGN<sub>7</sub>CGG), found 170 bp upstream of the predicted <em>aod-1</em> start codon, bound AOD2 and AOD5 only when both proteins were present, suggesting that they bind as a heterodimer. The element was also required for maximal expression of the <em>aod-1</em> gene in <em>N. crassa</em> (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Chae <em>et al.</em> 2007a</a>,<a id="xref-ref-10-6" class="xref-bibr" href="#ref-10">b</a>). Thus, we examined the intergenic upstream regions of the 65 peak sets (98 genes) in <a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a> for the presence of this sequence, using the SCOPE search tool (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Chakravarty <em>et al.</em> 2007</a>). One or more AIM sequences were found (<a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 3</a>) for 55 peak sets (84 genes). We also determined the position of the AIM sequence relative to the summit of the 55 ChIP-seq peak sets. Although this assessment is difficult since many peak sets have multiple summits and more than one AIM, we found that there was good concordance of at least one AIM with one summit in the intergenic regions in 46 peak sets (69 genes) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035402/-/DC1/FigureS3.pdf">Figure S3</a> and <a id="xref-table-wrap-3-6" class="xref-table" href="#T3">Table 3</a>). All of the genes for which AOD2 and AOD5 play a positive regulatory role, as shown in <a id="xref-fig-6-12" class="xref-fig" href="#F6">Figure 6, A and B</a>, were found to have an AIM sequence associated with at least one peak summit, with the exception of NCU03408. The latter gene is a special case since it is also associated with another consensus sequence found in a repeated sequence that appears in several places in the genome (see AOD2 binds to a repeat sequence in the genome). For peak sets containing no AIM, or where an AIM exists but does not coincide with a peak, binding of AOD2 and AOD5 generally occurs at a low level, with the exception of some sites that appear to bind AOD2 more efficiently than AOD5 (see AOD2 binds to a repeat sequence in the genome).</p><p id="p-48">The 98 genes in <a id="xref-table-wrap-3-7" class="xref-table" href="#T3">Table 3</a> include 42 hypothetical proteins and two proteins with domains of unknown function. The remaining 54 genes that are associated with a known function or domain were examined for related functions using the FunCat classification (<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Ruepp <em>et al.</em> 2004</a>) in FungiFun2 (<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Priebe <em>et al.</em> 2015</a>) (<a href="https://elbe.hki-jena.de/fungifun/fungifun.php">https://elbe.hki-jena.de/fungifun/fungifun.php</a>). Of those 54, 23 were found to be in seven classes that were over-represented in the sample, relative to known genes in the <em>N. crassa</em> genome, with 11 of the 23 genes listed in two or more functional categories (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035402/-/DC1/TableS1.docx">Table S1</a>). The seven classes fall into three of the main FunCat categories: energy, metabolism, and cellular transport. Eleven of the genes found in the FunCat analysis were among those analyzed by qPCR for expression differences in the absence of AOD2 or AOD5. Seven were found to have altered expression while no effects were observed for four others. However, the appearance of these genes in functionally enriched categories argues that the binding of the transcription factors may be significant, but that proper conditions for altering expression have not yet been discovered.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/449/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Over-represented genes in functional categories. Genes in the group of 65 peak sets common to all four ChIP-seq experiments (Table 3) that have known functions or domains were compared with genes of known function in the N. crassa genome, using FungiFun2 (Priebe et al. 2015). The seven different over-represented functional categories are indicated." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1992884807" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Over-represented genes in functional categories. Genes in the group of 65 peak sets common to all four ChIP-seq experiments (Table 3) that have known functions or domains were compared with genes of known function in the &lt;em&gt;N. crassa&lt;/em&gt; genome, using FungiFun2 (Priebe &lt;em&gt;et al.&lt;/em&gt; 2015). The seven different over-represented functional categories are indicated.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/449/F8.medium.gif" width="440" height="224"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/7/2/449/F8.medium.gif" width="440" height="224"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/449/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/449/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470312" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-49" class="first-child">Over-represented genes in functional categories. Genes in the group of 65 peak sets common to all four ChIP-seq experiments (<a id="xref-table-wrap-3-8" class="xref-table" href="#T3">Table 3</a>) that have known functions or domains were compared with genes of known function in the <em>N. crassa</em> genome, using FungiFun2 (<a id="xref-ref-51-3" class="xref-bibr" href="#ref-51">Priebe <em>et al.</em> 2015</a>). The seven different over-represented functional categories are indicated.</p><div class="sb-div caption-clear"></div></div></div><p id="p-50">Inspection of peaks from other regions of the Venn diagram (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>) where three of the four experiments had peaks in common, where peaks were specific for AOD-2 or AOD-5, or where peaks were specific for the presence or absence of Cm revealed 12 other sets of peaks that appeared robust and/or were in proximity to genes with a relevant metabolic function (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). The expression of the 14 genes that are in the correct orientation relative to these peaks was examined by qPCR to determine the effects of loss of AOD2 or AOD5, and the presence of Cm on expression. We also included the NCU10051 gene (flavohemoglobin) in this study as an example of a gene identified in the <em>P. anserina</em> microarray study that was not found in any of the groupings described here using the criteria chosen. Six genes showed no differences greater than twofold in any pairwise comparisons (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035402/-/DC1/FigureS4.ai">Figure S4</a>). One of the genes in this category was NCU04797, which encodes <em>N. crassa</em> FBP. Nine of the genes examined showed an over twofold expression difference in at least one pairwise comparison of the different conditions examined. Of those, one (NCU00628, hypothetical protein) shows an increase in the absence of AOD2 when grown in the presence of Cm (<a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure 9A</a>), three (NCU04392, NCU04752, NCU09144; all hypothetical proteins) have increases in the absence of AOD5 in the absence of Cm (<a id="xref-fig-9-2" class="xref-fig" href="#F9">Figure 9B</a>), and NCU10051 (flavohemoglobin) shows an increase in expression in the presence of Cm when either AOD2 or AOD5 is absent (<a id="xref-fig-9-3" class="xref-fig" href="#F9">Figure 9C</a>). A single gene (NCU02128, D-arabinitol dehydrogenase) showed decreased expression in the absence of AOD2 (<a id="xref-fig-9-4" class="xref-fig" href="#F9">Figure 9D</a>), while two (NCU04899, malate dehydrogenase and NCU05627, a glucose transporter) showed decreased expression in the absence of either AOD2 or AOD5 (<a id="xref-fig-9-5" class="xref-fig" href="#F9">Figure 9E</a>). One gene (NCU02623, mitochondrial hypoxia response) shows only increased expression in the presence of Cm, but no effects due to presence or absence of AOD2 or AOD5 (<a id="xref-fig-9-6" class="xref-fig" href="#F9">Figure 9F</a>). Interestingly, the gene that showed a decrease in expression in the absence of AOD2 but was not affected by loss of AOD5 (NCU02128) (<a id="xref-fig-9-7" class="xref-fig" href="#F9">Figure 9D</a>) has a peak profile that suggests considerable binding by AOD2, but very little, if any, AOD5 (see AOD2 binds to a repeat sequence in the genome).</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470426/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470426/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Genes examined from other overlapping positions on the Venn diagram</span><div class="sb-div caption-clear"></div></div></div><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/449/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Other genes affected by AOD2 and/or AOD5. Genes from categories of the Venn diagram that were not in the group common to all four ChIP-seq experiments were examined for expression by qPCR, as described in the legend to Figure 6. (A) Increased expression in the absence of AOD2 when grown in the presence of Cm. (B) Increased expression in the absence of AOD5 without CM. (C) Increased expression without AOD2 or AOD5 in the absence of Cm. (D) Decreased expression without AOD2. (E) Decreased expression without AOD2 or AOD5 in the presence of Cm (NCU04899) or decreased expression in the presence of Cm and dependence on AOD2 and AOD5 for expression in the absence of Cm (NCU05627). (F) Cm increases expression but no effect with loss of AOD2 or AOD5." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1992884807" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Other genes affected by AOD2 and/or AOD5. Genes from categories of the Venn diagram that were not in the group common to all four ChIP-seq experiments were examined for expression by qPCR, as described in the legend to Figure 6. (A) Increased expression in the absence of AOD2 when grown in the presence of Cm. (B) Increased expression in the absence of AOD5 without CM. (C) Increased expression without AOD2 or AOD5 in the absence of Cm. (D) Decreased expression without AOD2. (E) Decreased expression without AOD2 or AOD5 in the presence of Cm (NCU04899) or decreased expression in the presence of Cm and dependence on AOD2 and AOD5 for expression in the absence of Cm (NCU05627). (F) Cm increases expression but no effect with loss of AOD2 or AOD5.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 9" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/449/F9.medium.gif" width="440" height="260"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 9" src="https://www.g3journal.org/content/ggg/7/2/449/F9.medium.gif" width="440" height="260"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/449/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/449/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470282" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 9</span> <p id="p-57" class="first-child">Other genes affected by AOD2 and/or AOD5. Genes from categories of the Venn diagram that were not in the group common to all four ChIP-seq experiments were examined for expression by qPCR, as described in the legend to <a id="xref-fig-6-13" class="xref-fig" href="#F6">Figure 6</a>. (A) Increased expression in the absence of AOD2 when grown in the presence of Cm. (B) Increased expression in the absence of AOD5 without CM. (C) Increased expression without AOD2 or AOD5 in the absence of Cm. (D) Decreased expression without AOD2. (E) Decreased expression without AOD2 or AOD5 in the presence of Cm (NCU04899) or decreased expression in the presence of Cm and dependence on AOD2 and AOD5 for expression in the absence of Cm (NCU05627). (F) Cm increases expression but no effect with loss of AOD2 or AOD5.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>AOD2 binds to a repeat sequence in the genome</h3><p id="p-58">The upstream regions of the genes listed in <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a> and <a id="xref-table-wrap-3-9" class="xref-table" href="#T3">Table 3</a> were analyzed for the existence of common sequence motifs in addition to the AIM sequence. A subset of genes was associated with another well conserved upstream sequence of 14 bases. Further investigation, based on searches and alignments of the <em>N. crassa</em> genome with the regions surrounding the 14 bp sequence revealed that 10 peak sets were associated with a 310 bp sequence that is repeated in 11 places throughout the genome with &gt;85% identity (<a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>). There are also smaller subsets of the 310 bp sequence abundant in the genome. Each of the 310 bp sequences contains a set of 78 bp inverted repeats at the ends of the sequence (<a id="xref-fig-10-1" class="xref-fig" href="#F10">Figure 10</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035402/-/DC1/FigureS5.docx">Figure S5</a>) that contain the originally identified 14 bp sequence. The inverted repeats may have functional significance since they are more highly conserved than the region between them (<a id="xref-fig-10-2" class="xref-fig" href="#F10">Figure 10</a>).</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/470356/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/470356/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Sites in the <em>N. crassa</em> genome identified by BLASTN that have &gt;75% identity to the 310 bp region upstream of NCU02128</span><div class="sb-div caption-clear"></div></div></div><div id="F10" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/2/449/F10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Representation of a repeated sequence element in the N. crassa genome that binds AOD2. The long black line represents the 310 bp repeat sequence. The arrows above and below the line represent the inverted repeats at the ends of the sequence. The small open boxes at the ends of the inverted repeat indicate the location of the highly conserved 14 bp sequence found associated with the AOD2-binding peak summits. The sequence of that site is shown. The thin lines at the bottom indicate the extent of the two inverted repeats compared with the rest of the sequence. Numbers give the percentage identity in the indicated region among all 11 of the 310 bp repeat sequences identified." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1992884807" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Representation of a repeated sequence element in the &lt;em&gt;N. crassa&lt;/em&gt; genome that binds AOD2. The long black line represents the 310 bp repeat sequence. The arrows above and below the line represent the inverted repeats at the ends of the sequence. The small open boxes at the ends of the inverted repeat indicate the location of the highly conserved 14 bp sequence found associated with the AOD2-binding peak summits. The sequence of that site is shown. The thin lines at the bottom indicate the extent of the two inverted repeats compared with the rest of the sequence. Numbers give the percentage identity in the indicated region among all 11 of the 310 bp repeat sequences identified.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 10" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/2/449/F10.medium.gif" width="440" height="248"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 10" src="https://www.g3journal.org/content/ggg/7/2/449/F10.medium.gif" width="440" height="248"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/2/449/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 10" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/2/449/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/470236" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 10</span> <p id="p-63" class="first-child">Representation of a repeated sequence element in the <em>N. crassa</em> genome that binds AOD2. The long black line represents the 310 bp repeat sequence. The arrows above and below the line represent the inverted repeats at the ends of the sequence. The small open boxes at the ends of the inverted repeat indicate the location of the highly conserved 14 bp sequence found associated with the AOD2-binding peak summits. The sequence of that site is shown. The thin lines at the bottom indicate the extent of the two inverted repeats compared with the rest of the sequence. Numbers give the percentage identity in the indicated region among all 11 of the 310 bp repeat sequences identified.</p><div class="sb-div caption-clear"></div></div></div><p id="p-64">Repeated sequences in the <em>Neurospora</em> genome are subject to a mutagenic process termed repeat induced point mutation (RIP) (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Selker 1990</a>). Although repeated sequences &lt;0.4 kb are not normally considered good substrates for RIP (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Watters <em>et al.</em> 1999</a>), it has recently been shown that smaller sequences are affected by the process (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Gladyshev and Kleckner 2014</a>). It appears that the repeat units identified in this study have been subject to RIP since the majority of the changes found among the sequences are C:G to T:A transition mutations, which are characteristic of the process (<a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Selker 1990</a>). Of the 80 positions containing a change in at least one of the 11 sequences, only 15 are transversions, one is a deletion of a single base pair, one is an insertion of 2 bp, and 63 are transitions (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035402/-/DC1/FigureS5.docx">Figure S5</a>).</p><p id="p-65">A characteristic of the peak sets associated with this repeat is the binding of relatively large amounts of AOD2 and smaller variable amounts of AOD5 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035402/-/DC1/FigureS6.pdf">Figure S6</a>), as compared with most “typical” peak sets observed in this study, such as for NCU07953 (AOD1) and NCU06940 (hypothetical protein) (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5</a>). One obvious exception to this rule is the repeat sequence in the region between genes NCU09460 and NCU09993, where no binding of either protein was observed (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035402/-/DC1/FigureS6.pdf">Figure S6</a>). The variable amount of AOD5 bound at the other repeat sites is also shown by the fact that five of these peak sets were identified in the original group of 70 that had greater than fourfold enrichment in all four experiments. Another characteristic of the 10 sites that bind predominantly AOD2 is that the AOD2 peak is usually divided into two summits. The AOD2 peak summits at these sequences are always found associated with all or a portion of the highly conserved 14 bp sequence (<a id="xref-fig-10-3" class="xref-fig" href="#F10">Figure 10</a>) that occurs near the ends of the inverted repeats. These AOD2-binding peaks are located in the correct upstream position for at least one of the genes adjacent to the peaks in nine of the 10 cases (<a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table 5</a>).</p><p id="p-66">The qPCR analyses in strains lacking AOD2 or AOD5 that were performed during the investigation of genes described above, included eight genes associated with the nine peak sets occurring in the repeated regions that have at least one gene in the proper orientation. Genes NCU03409, NCU05202, and NCU05203 showed no changes greater than twofold (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035402/-/DC1/FigureS2.ai">Figure S2</a>). NCU02128 was found to depend on AOD2 but not AOD5 for expression (<a id="xref-fig-9-8" class="xref-fig" href="#F9">Figure 9D</a>), though the expression levels changed only two to threefold in the absence of AOD2. For NCU07678, no change in expression that depended on AOD2 or AOD5 was observed, but a small change in response to Cm was seen (<a id="xref-fig-6-14" class="xref-fig" href="#F6">Figure 6D</a>). The greatest changes were seen for NCU03408, where both AOD2 and AOD5 were required for induction (<a id="xref-fig-6-15" class="xref-fig" href="#F6">Figure 6A</a>). A substantial amount of AOD5 is bound at this peak set (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035402/-/DC1/FigureS6.pdf">Figure S6</a>), and it was identified in the group of 65 peaks common to all four experiments. An AIM sequence is also associated with the summit where AOD5 is most abundant (compare <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035402/-/DC1/FigureS3.pdf">Figure S3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035402/-/DC1/FigureS6.pdf">Figure S6</a>). For NCU04392 and NCU09144, small increases in transcription were seen in strains lacking AOD5 (<a id="xref-fig-9-9" class="xref-fig" href="#F9">Figure 9B</a>). The promoters of these two genes appear to bind very little, if any AOD5 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035402/-/DC1/FigureS6.pdf">Figure S6</a>), and it is difficult to imagine a negative regulatory role for AOD5 at these promoters. An alternative explanation might be that if AOD5 is not present, more AOD2 is available for binding and affecting transcription.</p><p id="p-67">Visual inspection of the ChIP-seq data also revealed two additional peak sets (NCU01053 and NCU01224/NCU01225) with an excess of AOD2 binding over AOD5, as compared with the more typical peak sets. These genes are not associated with the repeat sequence, they do not have the conserved 14 bp sequence in their upstream regions, and they do not exhibit the “double summit” pattern of AOD2 binding seen at the repeat sites (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035402/-/DC1/FigureS7.ai">Figure S7</a>). Furthermore, no AIM sequences were found to be associated with these peak sets. All three of these genes were examined by qPCR in the strains lacking AOD2 or AOD5, and no changes greater than twofold were observed (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035402/-/DC1/FigureS2.ai">Figure S2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035402/-/DC1/FigureS4.ai">Figure S4</a>).</p></div></div><div class="section discussion" id="sec-16"><h2 class="">Discussion</h2><p id="p-68">In <em>N. crassa</em>, expression of AOX is greatly increased when the function of the sETC is perturbed by mutations or when chemicals inhibit the proper function of the chain. The induction occurs at the level of transcription and is dependent on the zinc cluster transcription factors AOD2 and AOD5 (<a id="xref-ref-10-7" class="xref-bibr" href="#ref-10">Chae <em>et al.</em> 2007b</a>; <a id="xref-ref-65-5" class="xref-bibr" href="#ref-65">Tanton <em>et al.</em> 2003</a>). In the current study, we have shown that AOD2 and AOD5 are localized to the nucleus and bound to the region upstream of genes, regardless of whether cells are grown under conditions that are noninducing or inducing for AOX expression. Thus, it appears that their activation is achieved by communication of an unknown signal(s) to the nucleus when cells are exposed to specific conditions that affect mitochondrial function. We previously showed that AOD2 and AOD5 form a heterodimer <em>in vitro</em> and now present evidence that this is also the case <em>in vivo</em>, though it is not known what fraction of the population of each protein exists in this state.</p><p id="p-69">The orthologs of AOD2 and AOD5 in <em>A. nidulans</em> and <em>P. anserina</em> have been shown to control expression of the transcripts for the gluconeogenic enzymes PEPCK and FBP, as well as AOX (<a id="xref-ref-63-5" class="xref-bibr" href="#ref-63">Suzuki <em>et al.</em> 2012</a>; <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Bovier <em>et al.</em> 2014</a>). Fluorescence microscopy showed that GFP-tagged orthologs of the proteins in <em>A. nidulans</em> localized to the nucleus regardless of whether cells were grown in glucose or a gluconeogenic carbon source (<a id="xref-ref-63-6" class="xref-bibr" href="#ref-63">Suzuki <em>et al.</em> 2012</a>). Here, we have shown that the expression of <em>N. crassa</em> PEPCK and FBP are increased following growth in gluconeogenic carbon sources. However, we found that only the expression of PEPCK was dependent on AOD2 (and presumably AOD5, though this was not directly tested), while that of FBP was not.</p><p id="p-70">Our analysis of cultures grown in medium with different carbon sources revealed that the PEPCK mRNA was induced fourfold and sixfold by growth in acetate and ethanol as carbon sources, respectively, when compared with levels during growth with sucrose as the carbon source. On the other hand, growth in the presence of Cm had only a slight effect on PEPCK transcript levels, near the border of statistical significance. Conversely, the <em>aod-1</em> mRNA level was not significantly affected by growth in poor carbon sources, but was induced ∼18-fold by growth in the presence of Cm. Despite these differing patterns of expression for PEPCK and AOX, both inductions required AOD2 (and by implication, AOD5). Similar observations were made in <em>P. anserina</em> where the expression of AOX and several other genes was seen to be much higher during growth in the presence of antimycin A compared with when cultures were grown using acetate as a carbon source. However, the opposite was observed for the expression of the genes encoding PEPCK and FBP (<a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">Bovier <em>et al.</em> 2014</a>). A related observation was also made in <em>A. nidulans</em> where mutations in the PAS domains of the AOD2 and AOD5 orthologs led to antimycin A sensitivity, but had only slight effects on ability to utilize gluconeogenic carbon sources (<a id="xref-ref-63-7" class="xref-bibr" href="#ref-63">Suzuki <em>et al.</em> 2012</a>). Various mechanisms are conceivable that might allow different signals to activate the AOD2/AOD5 heterodimer. For example, different signaling molecules might affect one protein and not the other, resulting in differential activation of transcription at specific sites due to surrounding sequence or additional protein factors.</p><p id="p-71">Unexpectedly, we noted a difference in inducibility of the PEPCK transcript during growth in the presence of Cm with sucrose as the carbon source, depending on the growth medium used: about twofold (near the border of statistical significance) in low nitrogen-containing medium, but sevenfold in nitrogen-rich Vogel’s medium. Similarly, Cm induced AOX 18-fold in low nitrogen-containing medium, but ∼30-fold in Vogel’s medium. This suggests that the general nutritional status of a culture may also influence the regulation. Taken together, these data strongly suggest that different signals arising from various conditions of growth can differentially affect the transcription of a number of genes in an AOD2- and AOD5-dependent fashion. Whether the interpretation of different signals can be achieved by AOD2 and AOD5 alone, or if additional factors are required, is not known.</p><p id="p-72">The growth analysis also revealed that deletion mutants of <em>aod-2</em> and <em>aod-5</em> grew at a reduced rate compared with wild-type cells, even when sucrose was the carbon source. This suggested that the proteins may play roles in regulating other pathways, in addition to AOX and gluconeogenesis. The results of the ChIP-seq experiments, where a large number of peaks were identified by MACS2 analysis in each of the four experiments, supported this idea. As described in the <em>Results</em>, we chose to focus on peak sets that fulfilled various criteria, leaving 98 genes as candidates for being controlled by AOD2 and AOD5. Applying these criteria may result in the exclusion of some genes that are legitimate targets, but should help minimize the number of false positives. Inspection of the peaks revealed a wide array of binding characteristics with respect to size and shape of peaks. This may reflect different binding efficiencies which could be due to sequence variation, involvement of additional binding factors, or state of chromatin in different regions of the growing mycelium.</p><p id="p-73">Thirty-three of these 98 genes were analyzed by qPCR in both mutant and control backgrounds in an attempt to validate the role of the proteins as transcription factors for the identified genes. Eleven of the genes chosen for qPCR validation, including <em>aod-1</em>, were regulated in a positive fashion that required both AOD2 and AOD5 (<a id="xref-fig-6-16" class="xref-fig" href="#F6">Figure 6, A and B</a>). However, even among these 11, additional complexities were apparent. For example, eight of the genes demonstrated an obvious response to Cm, while three others did not. Furthermore, for most of those that did, a response to Cm was retained, even in the absence of AOD2 and AOD5, though the overall levels of induction were much lower. Thus, it appears that the promoters of some genes are influenced by both the AOD2/AOD5 heterodimer, as well as other factors that interpret additional signals generated from growth in the presence of Cm. The other three positively controlled genes appear to lack this extra involvement with Cm. These different responses further support the notion that other factors may be involved in the regulation of these genes. Validation by qPCR also revealed that three genes were negatively controlled by AOD2 and/or AOD5, since loss of the proteins resulted in increased transcript levels (<a id="xref-fig-6-17" class="xref-fig" href="#F6">Figure 6C</a>).</p><p id="p-74">The AIM has been shown to be within the promoter region of genes controlled by AOD2/AOD5 or their orthologs (<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Chae <em>et al.</em> 2007a</a>; <a id="xref-ref-63-8" class="xref-bibr" href="#ref-63">Suzuki <em>et al.</em> 2012</a>; <a id="xref-ref-6-5" class="xref-bibr" href="#ref-6">Bovier <em>et al.</em> 2014</a>). The ChIP-seq peaks for 10 of the 11 genes that are positively controlled by the two transcription factors have at least one AIM sequence aligned with at least one summit. The exception is NCU03408. Here, an AIM sequence aligns reasonably well with the AOD5 summits, but not with the largest AOD2 summits. Interestingly, this is a gene associated with the repeat sequence for binding AOD2, and the largest AOD2 summits align with the conserved consensus sequence found within the repeat. Thus, at this site, it appears that a standard AIM and an AOD2-specific binding site occur in close proximity. Of the three negatively regulated genes, NCU04874 (AOD3) and NCU06230 (Ser/Thr kinase) have an AIM sequence at the peak summits, while NCU03749 (hydroxyacyl glutathione hydrolase) does not.</p><p id="p-75">An additional five genes examined by qPCR showed no effect on transcript levels when AOD2 or AOD5 were missing, but did show increased transcript levels in the presence of Cm (<a id="xref-fig-6-18" class="xref-fig" href="#F6">Figure 6D</a>). The fact that the promoters of these genes bind the two transcription factors but show no effects when the factors are missing under the conditions examined, again suggests that other factors may play a role in inducing transcription. In these cases, the role, if any, of AOD2 and AOD5 is unclear. However, the homolog of one of these five genes (NCU11414) was also identified in the <em>P. anserina</em> microarray study (<a id="xref-ref-6-6" class="xref-bibr" href="#ref-6">Bovier <em>et al.</em> 2014</a>) as being upregulated by gain-of-function mutations in the orthologs of AOD2 and AOD5 (RSE2 and RSE3, respectively) and downregulated in strains with deletions of the genes. These findings support a role for AOD2 and AOD5 in the regulation of NCU11414. The relationship identified for two other genes in this category (NCU02474 and NCU08946) with their adjacent, divergently transcribed genes (see <em>Results</em>), points to a complex mechanism of transcriptional control involving AOD2, AOD5, other factors, and possibly different signals resulting from growth in the presence of Cm.</p><p id="p-76">Many genes examined during our validation process failed to show differences in expression in the absence of either protein or in the presence or absence of Cm. Some of these may simply represent false positives, which can be frequent in ChIP-seq studies (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Worsley Hunt and Wasserman 2014</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Jain <em>et al.</em> 2015</a>). On the other hand, many such genes show obvious binding of both proteins in their upstream regions. For example, both NCU04307 (MSF1, mitochondrial tRNA synthetase Phe) and NCU07807 (fructose bisphosphate aldolase) are the only genes in the correct orientation relative to the peaks upstream of their coding sequences. In both cases, the peaks from all four experiments are robust and include an AIM sequence at their summit. It is conceivable that these genes respond in an AOD2/AOD5-dependent manner to other induction signals arising from different stresses. As discussed above, this might also require the action of other proteins.</p><p id="p-77">A relevant question with respect to AOD2 and AOD5, is whether they can act independently of each other as monomers, homodimers, or dimers with other transcription factors. Such interactions, including interactions with other classes of transcription factor, have been described for zinc cluster proteins (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">MacPherson <em>et al.</em> 2006</a>), The Oaf1 zinc cluster protein of <em>S. cerevisiae</em> is known to function both alone or as part of a heterodimer with another zinc cluster protein, Pip2 (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Karpichev and Small 1998</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Baumgartner <em>et al.</em> 1999</a>). Furthermore, it has been observed in <em>P. anserina</em> that loss of RSE3 (the AOD5 ortholog), but not RSE2 (the AOD2 ortholog), results in reduced fertility (<a id="xref-ref-6-7" class="xref-bibr" href="#ref-6">Bovier <em>et al.</em> 2014</a>), implying that RSE3 has at least one function that is independent of RSE2.</p><p id="p-78">ChIP-seq data suggesting robust binding of one protein without the other could be taken as evidence that binding to DNA as independent proteins is possible. Although we cannot directly compare ChIP-seq peak heights obtained with different antibodies, it is apparent from visual inspection of the data that the majority of peaks reflect binding of both proteins. However, it was also apparent that in certain cases the ratios of the two proteins were considerably skewed compared with the typical peak sets observed. While we cannot rule out the possibility that binding of additional proteins at certain promoters may mask the epitope tag of one protein but not the other, it is also possible that the proteins bind without the other at certain sites. For AOD5, the peaks associated with a few genes seem to bind substantially more AOD5 than AOD2; perhaps the best example is NCU06211 (malate dehydrogenase). However, no influence of either AOD2 or AOD5 on transcript levels of the gene could be seen during growth in standard conditions or in the presence of Cm.</p><p id="p-79">Twelve peak sets were identified as binding larger amounts of AOD2 compared with AOD5. Two of these sets had no obvious defining characteristics. However, 10 of the sets were associated with the 310 bp repeat region. In nine of these 10 cases, the repeat region is in a position that corresponds to the upstream region of one or both of the flanking genes. Seven of these genes were examined by qPCR for effects on expression due to Cm and loss of AOD2 or AOD5. A variety of effects were observed so that for the conditions tested it cannot be concluded that AOD2 is required for expression of genes at all these sites. Only one of the 11 repeated regions with &gt;75% identity found in the genome was not associated with AOD2 binding. It is possible that the arrangement of chromatin, or other bound proteins at this locus under the conditions tested, prevents access for AOD2.</p><p id="p-80">The origin of the 310 bp repeat regions and the relevance of AOD2 binding within them are not apparent. BLAST searches of other <em>Neurospora</em> species available at FungiDB (<a href="http://fungidb.org/fungidb/">http://fungidb.org/fungidb/</a>) revealed that <em>N. tetrasperma</em> contained four sequences with high identity to the entire 310 bp region. <em>N. discreta</em> had no sites related to the entire 310 bp sequence but did have two smaller sites with high similarity to portions of the inverted repeat region. Thus, the sequence may have originated in an ancestral species, but is only being maintained in <em>N. crassa</em>. A consensus sequence was identified at the summit of the AOD2 binding sites, but further work is required to determine if it is actually a binding site for the protein.</p><p id="p-81">Analysis of genes by FunCat showed clear enrichment for genes in certain categories. Within our set of 98 genes associated with the 65 peak sets, several are involved with respiration or energy metabolism. In addition, we know of at least one additional gene (NCU08674) in our data set that is categorized simply as a pentatricopeptide repeat protein in the database, but has now been shown to be involved in the assembly of respiratory complexes in <em>N. crassa</em> (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Solotoff <em>et al.</em> 2015</a>). Consideration of the number and type of genes affected by RSE2 and RSE3 in the <em>P. anserina</em> microarray study led to the suggestion that the transcription factors are involved in strategies that contribute to the defense and adaptation of the organism (<a id="xref-ref-6-8" class="xref-bibr" href="#ref-6">Bovier <em>et al.</em> 2014</a>).</p><p id="p-82">Other categories identified by FunCat analysis in the current study involved metabolism of glutamate and glycine. Glutamate, or the related amino acid glutamine, provide the initial nitrogen source for the synthesis of other amino acids, purines, and pyrimidines. Glutamate is typically formed by amination of the TCA cycle intermediate α-ketoglutarate (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Davis and Wong 2010</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Magasanik and Kaiser 2002</a>). Interestingly, when mitochondrial function is low in <em>S. cerevisiae</em>, the production of enzymes necessary for the synthesis of α-ketoglutarate are under the control of the classic retrograde response (RTG) system originally discovered in this organism. Thus, <em>rtg</em> mutants in <em>S. cerevisiae</em> are glutamate auxotrophs in conditions where respiratory activity is low (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Liu and Butow 1999</a>, <a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">2006</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Liao and Butow 1993</a>). In the current study, we observed no evidence of the involvement of AOD2 and AOD5 in the regulation of genes for α-ketoglutarate synthesis, but the upstream region of five genes involved in glutamate metabolism were found to be associated with AOD2/AOD5 binding.</p><p id="p-83">The glycine enzymes identified in the current study are named for the glycine cleavage system, but the system is also known as glycine synthase and can act in the reverse direction to synthesize glycine (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Kikuchi and Hiraga 1982</a>). The complex consists of four proteins housed in the mitochondrion. These proteins convert glycine to CO<sub>2</sub> and NH<sub>3</sub>, while also forming the 1-carbon donor molecule, 5,10-methylenetetrahydrofolate. FunCat analysis revealed that three (NCU02475, NCU06424, and NCU08877) of the four genes encoding these proteins were present in the group of 65 peaks common to all four ChIP-seq experiments. The peaks associated with NCU02475 were quite robust and qPCR analysis confirmed that AOD2 and AOD5 were required for optimal expression. Peak sets associated with the other two genes were very small. Expression of NCU08877 was not affected by the absence of AOD2 or AOD5. NCU06424 expression was not examined. However, all three sets of peaks were associated with AIM sequences at their summits. Both glutamate and glycine are required for the synthesis of glutathione. Glutathione is an important molecule that provides a response to various cellular stresses, including reactive oxygen species in mitochondria (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Forman <em>et al.</em> 2009</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Ribas <em>et al.</em> 2014</a>). Since these damaging species may increase when the sETC is impaired, it is conceivable that AOD2/AOD5 may be involved in increasing production of at least some of the enzymes required for synthesis of these two amino acids, so that substrates for glutathione synthesis are available during oxidative stress.</p><p id="p-84">It has been suggested that malate is an inducing molecule for the AOD2 and AOD5 orthologs in <em>A. nidulans</em> (<a id="xref-ref-63-9" class="xref-bibr" href="#ref-63">Suzuki <em>et al.</em> 2012</a>). Though not identified in our FunCat analysis, we noted that two genes associated with malate metabolism were included in our group of 65 peak sets (NCU03651, malic enzyme and NCU10007, malate synthase) and two others (NCU04899, mitochondrial malate dehydrogenase and NCU06211, cytosolic malate dehydrogenase) appeared among the group with good peak sets that did not fall within the group of 65 peak sets. NCU03651 and NCU10007 showed no response to Cm, but AOD2 and AOD5 were required for their expression both with and without Cm in the growth medium. On the other hand, NCU04899 showed a response to Cm that required the presence of both AOD2 and AOD5.</p><p id="p-85">In conclusion, our data show an involvement of AOD2 and AOD5 in the expression of several genes that play a role in energy metabolism and various metabolic pathways. The factors appear to act predominantly in a positive fashion, but examples of repressing activity were also seen. The majority of genes appear to bind the transcription factors as a heterodimer, but rare cases of preferential binding of one or the other were also seen. The mechanism of function of the proteins is complex, allowing different responses to different conditions.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-86">We are grateful to Kirst King-Jones and Ran Zhuo for assistance and use of equipment for qPCR. We thank Randolph Addison for the antibody to <em>N. crassa</em> Kar2, and Gloria Turner and Kathy Borkovich for antibody to <em>N. crassa</em> arginase. This work was funded by grants from the Natural Sciences and Engineering Research Council of Canada to F.E.N. and National Institutes of Health, National Institute of General Medical Sciences (grant GM RO1-097637) to M.F. </p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-18"><p id="p-87">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035402/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.035402/-/DC1</a>.</p></li><li class="fn" id="fn-19"><p id="p-88"><em>Communicating editor: M. S. Sachs</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-09-11"><span class="received-label">Received </span>September 11, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-11-22"><span class="accepted-label">Accepted </span>November 22, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Qi <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.1" data-doi="10.1371/journal.pone.0091353"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arthanari</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heintzen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Griffiths-Jones</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crosthwaite</span>  <span class="cit-name-given-names">S. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Natural antisense transcripts and long non-coding RNA in Neurospora crassa.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">e91353</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De91353%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0091353%26rft_id%253Dinfo%253Apmid%252F24621812%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0091353&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24621812&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.2" data-doi="10.1074/jbc.274.32.22208"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baumgartner</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamilton</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piskacek</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruis</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rottensteiner</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Functional analysis of the Zn(2)Cys(6) transcription factors Oaf1p and Pip2p. Different roles in fatty acid induction of beta-oxidation in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">274</span>(<span class="cit-issue">32</span>): <span class="cit-fpage">22208</span>–<span class="cit-lpage">22216</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DBaumgartner%26rft.auinit1%253DU.%26rft.volume%253D274%26rft.issue%253D32%26rft.spage%253D22208%26rft.epage%253D22216%26rft.atitle%253DFunctional%2BAnalysis%2Bof%2Bthe%2BZn2Cys6Transcription%2BFactors%2BOaf1p%2Band%2BPip2p%253A%2BDIFFERENT%2BROLES%2BIN%2BFATTY%2BACID%2BINDUCTION%2BOF%2B%257Bbeta%257D-OXIDATION%2BINSACCHAROMYCES%2BCEREVISIAE%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.274.32.22208%26rft_id%253Dinfo%253Apmid%252F10428786%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzQvMzIvMjIyMDgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzIvNDQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-book" id="cit-7.2.449.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bertrand</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Argan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Szakacs</span>  <span class="cit-name-given-names">N. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Genetic control of the biogenesis of cyanide insensitive respiration in <em>Neurospora crassa</em></span>, pp. <span class="cit-fpage">495</span>–<span class="cit-lpage">507</span> in <em>Mitochondria</em>, edited by <span class="cit-ed"><span class="cit-name-surname">Schweyen</span>  <span class="cit-name-given-names">R. J.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Wolf</span>  <span class="cit-name-given-names">K.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Kaudewitz</span>  <span class="cit-name-given-names">F.</span></span>. <span class="cit-publ-name">Walter de Gruyter Co.</span>, <span class="cit-publ-loc">Berlin</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-book" id="cit-7.2.449.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bevan</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lang</span>  <span class="cit-name-given-names">B. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Mitochondrial genome evolution: the origin of mitochondria and of eukaryotes</span>, pp. <span class="cit-fpage">1</span>–<span class="cit-lpage">35</span> in <span class="cit-source"><em>Mitochondrial Function and Biogenetics</em>, <em>Topics in Current Genetics</em></span>, <span class="cit-vol">Vol. 8</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Kohler</span>  <span class="cit-name-given-names">C.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Bauer</span>  <span class="cit-name-given-names">M. F.</span></span>. <span class="cit-publ-name">Springer-Verlag</span>, <span class="cit-publ-loc">Berlin, Heidelberg</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMitochondrial%2BFunction%2Band%2BBiogenetics%252C%2BTopics%2Bin%2BCurrent%2BGenetics%26rft.volume%253D8%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.5" data-doi="10.1128/MCB.21.2.390-399.2001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Borghouts</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Werner</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elthon</span>  <span class="cit-name-given-names">T. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Osiewacz</span>  <span class="cit-name-given-names">H. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Copper-modulated gene expression and senescence in the filamentous fungus <em>Podospora anserina</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">390</span>–<span class="cit-lpage">399</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DBorghouts%26rft.auinit1%253DC.%26rft.volume%253D21%26rft.issue%253D2%26rft.spage%253D390%26rft.epage%253D399%26rft.atitle%253DCopper-Modulated%2BGene%2BExpression%2Band%2BSenescence%2Bin%2Bthe%2BFilamentous%2BFungus%2BPodospora%2Banserina%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.21.2.390-399.2001%26rft_id%253Dinfo%253Apmid%252F11134328%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjg6IjIxLzIvMzkwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzQ0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.6" data-doi="10.1128/EC.00172-13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bovier</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sellem</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Humbert</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sainsard-Chanet</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genetic and functional investigation of Zn(2)Cys(6) transcription factors RSE2 and RSE3 in Podospora anserina.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">13</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">53</span>–<span class="cit-lpage">65</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryot.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.00172-13%26rft_id%253Dinfo%253Apmid%252F24186951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo3OiIxMy8xLzUzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzQ0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.7" data-doi="10.1093/gbe/evt008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burger</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gray</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forget</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lang</span>  <span class="cit-name-given-names">B. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Strikingly bacteria-like and gene-rich mitochondrial genomes throughout jakobid protists.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">5</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">418</span>–<span class="cit-lpage">438</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevt008%26rft_id%253Dinfo%253Apmid%252F23335123%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2JlIjtzOjU6InJlc2lkIjtzOjc6IjUvMi80MTgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzIvNDQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.8" data-doi="10.1111/j.1399-3054.2009.01239.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chae</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nargang</span>  <span class="cit-name-given-names">F. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Investigation of regulatory factors required for alternative oxidase production in Neurospora crassa.</span> <abbr class="cit-jnl-abbrev">Physiol. Plant.</abbr> <span class="cit-vol">137</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">407</span>–<span class="cit-lpage">418</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhysiologia%2BPlantarum%2B%2528Copenhagen.%2B1948%2529%26rft.stitle%253DPhysiologia%2BPlantarum%2B%2528Copenhagen.%2B1948%2529%26rft.aulast%253DChae%26rft.auinit1%253DM.%2BS.%26rft.volume%253D137%26rft.issue%253D4%26rft.spage%253D407%26rft.epage%253D418%26rft.atitle%253DInvestigation%2Bof%2Bregulatory%2Bfactors%2Brequired%2Bfor%2Balternative%2Boxidase%2Bproduction%2Bin%2BNeurospora%2Bcrassa.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1399-3054.2009.01239.x%26rft_id%253Dinfo%253Apmid%252F19493307%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1399-3054.2009.01239.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19493307&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.9" data-doi="10.1534/genetics.106.068635"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chae</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kessler</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nargang</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanton</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span><span class="cit-comment">a</span> <span class="cit-article-title">Identification of an alternative oxidase induction motif in the promoter region of the <em>aod-1</em> gene in <em>Neurospora crassa</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">175</span>: <span class="cit-fpage">1597</span>–<span class="cit-lpage">1606</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.068635%26rft_id%253Dinfo%253Apmid%252F17237510%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3NS80LzE1OTciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzIvNDQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.10" data-doi="10.1534/genetics.107.078212"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chae</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nargang</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cleary</span>  <span class="cit-name-given-names">I. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Todd</span>  <span class="cit-name-given-names">A. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span><span class="cit-comment">b</span> <span class="cit-article-title">Two zinc cluster transcription factors control induction of alternative oxidase in <em>Neurospora crassa</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">1997</span>–<span class="cit-lpage">2006</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DChae%26rft.auinit1%253DM.%2BS.%26rft.volume%253D177%26rft.issue%253D4%26rft.spage%253D1997%26rft.epage%253D2006%26rft.atitle%253DTwo%2BZinc-Cluster%2BTranscription%2BFactors%2BControl%2BInduction%2Bof%2BAlternative%2BOxidase%2Bin%2BNeurospora%2Bcrassa%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.078212%26rft_id%253Dinfo%253Apmid%252F18073419%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny80LzE5OTciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzIvNDQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.11" data-doi="10.1186/1471-2105-8-249"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chakravarty</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlson</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khetani</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gross</span>  <span class="cit-name-given-names">R. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A novel ensemble learning method for de novo computational identification of DNA binding sites.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">249</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DChakravarty%26rft.auinit1%253DA.%26rft.volume%253D8%26rft.spage%253D249%26rft.epage%253D249%26rft.atitle%253DA%2Bnovel%2Bensemble%2Blearning%2Bmethod%2Bfor%2Bde%2Bnovo%2Bcomputational%2Bidentification%2Bof%2BDNA%2Bbinding%2Bsites.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-8-249%26rft_id%253Dinfo%253Apmid%252F17626633%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-8-249&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17626633&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">da Cunha</span>  <span class="cit-name-given-names">F. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Torelli</span>  <span class="cit-name-given-names">N. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kowaltowski</span>  <span class="cit-name-given-names">A. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Mitochondrial retrograde signaling: triggers, pathways, and outcomes.</span> <abbr class="cit-jnl-abbrev">Oxid. Med. Cell. Longev.</abbr> <span class="cit-vol">2015</span>: <span class="cit-fpage">482582</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOxid.%2BMed.%2BCell.%2BLongev.%26rft.volume%253D2015%26rft.spage%253D482582%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-book" id="cit-7.2.449.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">K. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Nitrogen metabolism in filamentous fungi</span>, pp. <span class="cit-fpage">325</span>–<span class="cit-lpage">338</span> in <em>Cellular and Molecular Biology of Filamentous Fungi</em>, edited by <span class="cit-ed"><span class="cit-name-surname">Borkovich</span>  <span class="cit-name-given-names">K. A.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Ebbole</span>  <span class="cit-name-given-names">D. J.</span></span>. <span class="cit-publ-name">ASM Press</span>, <span class="cit-publ-loc">Washington, DC.</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.14" data-doi="10.1016/0076-6879(71)17168-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Serres</span>  <span class="cit-name-given-names">F. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span> <span class="cit-article-title">Genetic and microbiological research techniques for <em>Neurospora crassa</em>.</span> <abbr class="cit-jnl-abbrev">Methods Enzymol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">79</span>–<span class="cit-lpage">143</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BEnzymol.%26rft.volume%253D17%26rft.spage%253D79%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0076-6879%252871%252917168-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0076-6879(71)17168-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.15" data-doi="10.1534/genetics.104.034017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Descheneau</span>  <span class="cit-name-given-names">A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cleary</span>  <span class="cit-name-given-names">I. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nargang</span>  <span class="cit-name-given-names">F. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genetic evidence for a regulatory pathway controlling alternative oxidase production in <em>Neurospora crassa</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">169</span>: <span class="cit-fpage">123</span>–<span class="cit-lpage">135</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.034017%26rft_id%253Dinfo%253Apmid%252F15466423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY5LzEvMTIzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzQ0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.16" data-doi="10.1073/pnas.070501997"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dufour</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boulay</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rincheval</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sainsard-Chanet</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">A causal link between respiration and senescence in <em>Podospora anserina</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">4138</span>–<span class="cit-lpage">4143</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.070501997%26rft_id%253Dinfo%253Apmid%252F10759557%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5Ny84LzQxMzgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzIvNDQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-book" id="cit-7.2.449.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chalmers</span>  <span class="cit-name-given-names">J. H.</span>  <span class="cit-name-suffix">Jr</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guzik</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warden</span>  <span class="cit-name-given-names">J. T.</span></span></li></ol><cite>, <span class="cit-pub-date">1976</span> <span class="cit-article-title">Assembly of the cyanide-insensitive respiratory pathway in Neurospora crassa</span>, pp. <span class="cit-fpage">865</span>–<span class="cit-lpage">872</span> in <span class="cit-source">Genetics and Biogenesis of Chloroplasts and Mitochondria</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Bucher</span>  <span class="cit-name-given-names">T. H.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Neupert</span>  <span class="cit-name-given-names">W.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Sebald</span>  <span class="cit-name-given-names">W.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Werner</span>  <span class="cit-name-given-names">S.</span></span>. <span class="cit-publ-name">Elsevier/North-Holland Biomedical Press</span>, <span class="cit-publ-loc">Amsterdam</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.18" data-doi="10.1016/S0076-6879(09)05001-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elstner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andreoli</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klopstock</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meitinger</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prokisch</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The mitochondrial proteome database: MitoP2.</span> <abbr class="cit-jnl-abbrev">Methods Enzymol.</abbr> <span class="cit-vol">457</span>: <span class="cit-fpage">3</span>–<span class="cit-lpage">20</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Benzymology%26rft.stitle%253DMethods%2BEnzymol%26rft.aulast%253DElstner%26rft.auinit1%253DM.%26rft.volume%253D457%26rft.spage%253D3%26rft.epage%253D20%26rft.atitle%253DThe%2Bmitochondrial%2Bproteome%2Bdatabase%253A%2BMitoP2.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0076-6879%252809%252905001-0%26rft_id%253Dinfo%253Apmid%252F19426859%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0076-6879(09)05001-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19426859&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266544100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.19" data-doi="10.1016/j.mam.2008.08.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Forman</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rinna</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Glutathione: overview of its protective roles, measurement, and biosynthesis.</span> <abbr class="cit-jnl-abbrev">Mol. Aspects Med.</abbr> <span class="cit-vol">30</span>(<span class="cit-issue">1–2</span>): <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Baspects%2Bof%2Bmedicine%26rft.stitle%253DMol%2BAspects%2BMed%26rft.aulast%253DForman%26rft.auinit1%253DH.%2BJ.%26rft.volume%253D30%26rft.issue%253D1-2%26rft.spage%253D1%26rft.epage%253D12%26rft.atitle%253DGlutathione%253A%2Boverview%2Bof%2Bits%2Bprotective%2Broles%252C%2Bmeasurement%252C%2Band%2Bbiosynthesis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.mam.2008.08.006%26rft_id%253Dinfo%253Apmid%252F18796312%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.mam.2008.08.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18796312&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266747800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gladyshev</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleckner</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Direct recognition of homology between double helices of DNA in Neurospora crassa.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">3509</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D5%26rft.spage%253D3509%26rft_id%253Dinfo%253Apmid%252F24699390%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24699390&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.21" data-doi="10.1104/pp.90.4.1305"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Good</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crosby</span>  <span class="cit-name-given-names">W. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Anaerobic induction of alanine aminotransferase in barley root tissue.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">1305</span>–<span class="cit-lpage">1309</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiology%26rft.stitle%253DPlant%2BPhysiol.%26rft.aulast%253DGood%26rft.auinit1%253DA.%2BG.%26rft.volume%253D90%26rft.issue%253D4%26rft.spage%253D1305%26rft.epage%253D1309%26rft.atitle%253DAnaerobic%2BInduction%2Bof%2BAlanine%2BAminotransferase%2Bin%2BBarley%2BRoot%2BTissue%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.90.4.1305%26rft_id%253Dinfo%253Apmid%252F16666927%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiI5MC80LzEzMDUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzIvNDQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cha</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Biochemical methods used to study the gene expression and protein complexes in the filamentous fungus Neurospora crassa.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">638</span>: <span class="cit-fpage">189</span>–<span class="cit-lpage">200</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DGuo%26rft.auinit1%253DJ.%26rft.volume%253D638%26rft.spage%253D189%26rft.epage%253D200%26rft.atitle%253DBiochemical%2Bmethods%2Bused%2Bto%2Bstudy%2Bthe%2Bgene%2Bexpression%2Band%2Bprotein%2Bcomplexes%2Bin%2Bthe%2Bfilamentous%2Bfungus%2BNeurospora%2Bcrassa.%26rft_id%253Dinfo%253Apmid%252F20238270%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=20238270&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.23" data-doi="10.1534/genetics.107.075051"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoppins</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Go</span>  <span class="cit-name-given-names">N. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmitt</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neupert</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Alternative splicing gives rise to different isoforms of the <em>Neurospora crassa</em> Tob55 protein that vary in their ability to insert β−barrel proteins into the outer mitochondrial membrane.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">137</span>–<span class="cit-lpage">149</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.075051%26rft_id%253Dinfo%253Apmid%252F17660559%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc3LzEvMTM3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzQ0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.24" data-doi="10.1042/bj3560595"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huh</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Characterization of the gene family encoding alternative oxidase from <em>Candida albicans</em>.</span> <abbr class="cit-jnl-abbrev">Biochem. J.</abbr> <span class="cit-vol">356</span>: <span class="cit-fpage">595</span>–<span class="cit-lpage">604</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemical%2BJournal%26rft.stitle%253DBiochem.%2BJ.%26rft.aulast%253DHUH%26rft.auinit1%253DW.-K.%26rft.volume%253D356%26rft.issue%253D2%26rft.spage%253D595%26rft.epage%253D604%26rft.atitle%253DCharacterization%2Bof%2Bthe%2Bgene%2Bfamily%2Bencoding%2Balternative%2Boxidase%2Bfrom%2BCandida%2Balbicans%26rft_id%253Dinfo%253Adoi%252F10.1042%252Fbj3560595%26rft_id%253Dinfo%253Apmid%252F11368790%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTA6InBwYmlvY2hlbWoiO3M6NToicmVzaWQiO3M6OToiMzU2LzIvNTk1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzQ0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.25" data-doi="10.1093/nar/gkv637"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jain</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baldi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zabel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Straub</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Becker</span>  <span class="cit-name-given-names">P. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Active promoters give rise to false positive ‘Phantom Peaks’ in ChIP-seq experiments.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">43</span>(<span class="cit-issue">14</span>): <span class="cit-fpage">6959</span>–<span class="cit-lpage">6968</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv637%26rft_id%253Dinfo%253Apmid%252F26117547%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0My8xNC82OTU5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzQ0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.26" data-doi="10.1016/j.bbamcr.2012.02.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jazwinski</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The retrograde response: when mitochondrial quality control is not enough.</span> <abbr class="cit-jnl-abbrev">Biochim. Biophys. Acta</abbr> <span class="cit-vol">1833</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">400</span>–<span class="cit-lpage">409</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochim.%2BBiophys.%2BActa%26rft.volume%253D1833%26rft.spage%253D400%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bbamcr.2012.02.010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bbamcr.2012.02.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.27" data-doi="10.1128/MCB.18.11.6560"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Karpichev</span>  <span class="cit-name-given-names">I. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Small</span>  <span class="cit-name-given-names">G. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Global regulatory functions of Oaf1p and Pip2p (Oaf2p), transcription factors that regulate genes encoding peroxisomal proteins in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">18</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">6560</span>–<span class="cit-lpage">6570</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DKarpichev%26rft.auinit1%253DI.%2BV.%26rft.volume%253D18%26rft.issue%253D11%26rft.spage%253D6560%26rft.epage%253D6570%26rft.atitle%253DGlobal%2BRegulatory%2BFunctions%2Bof%2BOaf1p%2Band%2BPip2p%2B%2528Oaf2p%2529%252C%2BTranscription%2BFactors%2BThat%2BRegulate%2BGenes%2BEncoding%2BPeroxisomal%2BProteins%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.18.11.6560%26rft_id%253Dinfo%253Apmid%252F9774671%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIxOC8xMS82NTYwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzQ0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kikuchi</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hiraga</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title">The mitochondrial glycine cleavage system. Unique features of the glycine decarboxylation.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biochem.</abbr> <span class="cit-vol">45</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">137</span>–<span class="cit-lpage">149</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2Bcellular%2Bbiochemistry%26rft.stitle%253DMol%2BCell%2BBiochem%26rft.aulast%253DKikuchi%26rft.auinit1%253DG.%26rft.volume%253D45%26rft.issue%253D3%26rft.spage%253D137%26rft.epage%253D149%26rft.atitle%253DThe%2Bmitochondrial%2Bglycine%2Bcleavage%2Bsystem.%2BUnique%2Bfeatures%2Bof%2Bthe%2Bglycine%2Bdecarboxylation.%26rft_id%253Dinfo%253Apmid%252F6750353%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=6750353&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1982NW28200002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.29" data-doi="10.1111/j.1574-6968.1996.tb08393.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kirimura</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matsui</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sugano</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Usami</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Enhancement and repression of cyanide-insensitive respiration in <em>Aspergillus niger</em>.</span> <abbr class="cit-jnl-abbrev">FEMS Microbiol. Lett.</abbr> <span class="cit-vol">141</span>: <span class="cit-fpage">251</span>–<span class="cit-lpage">254</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BMicrobiol.%2BLett.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1574-6968.1996.tb08393.x%26rft_id%253Dinfo%253Apmid%252F8768530%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc2xlIjtzOjU6InJlc2lkIjtzOjExOiIxNDEvMi0zLzI1MSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi80NDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.30" data-doi="10.1038/nature01427"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kramer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loros</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunlap</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crosthwaite</span>  <span class="cit-name-given-names">S. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Role for antisense RNA in regulating circadian clock function in Neurospora crassa.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">421</span>(<span class="cit-issue">6926</span>): <span class="cit-fpage">948</span>–<span class="cit-lpage">952</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKramer%26rft.auinit1%253DC.%26rft.volume%253D421%26rft.issue%253D6926%26rft.spage%253D948%26rft.epage%253D952%26rft.atitle%253DRole%2Bfor%2Bantisense%2BRNA%2Bin%2Bregulating%2Bcircadian%2Bclock%2Bfunction%2Bin%2BNeurospora%2Bcrassa.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01427%26rft_id%253Dinfo%253Apmid%252F12607002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature01427&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12607002&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181186900050&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.31" data-doi="10.1038/227680a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Laemmli</span>  <span class="cit-name-given-names">U. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span> <span class="cit-article-title">Cleavage of structural proteins during the assembly of the head of bacteriophage T4.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">227</span>: <span class="cit-fpage">680</span>–<span class="cit-lpage">685</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLaemmli%26rft.auinit1%253DU.%2BK.%26rft.volume%253D227%26rft.issue%253D5259%26rft.spage%253D680%26rft.epage%253D685%26rft.atitle%253DCleavage%2Bof%2Bstructural%2Bproteins%2Bduring%2Bthe%2Bassembly%2Bof%2Bthe%2Bhead%2Bof%2Bbacteriophage%2BT4.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F227680a0%26rft_id%253Dinfo%253Apmid%252F5432063%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/227680a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=5432063&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1970H022600020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lambowitz</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-article-title">Preparation and analysis of mitochondrial ribosomes.</span> <abbr class="cit-jnl-abbrev">Methods Enzymol.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">421</span>–<span class="cit-lpage">433</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BEnzymol.%26rft.volume%253D54%26rft.spage%253D421%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.33" data-doi="10.1016/j.biocel.2009.04.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lenaz</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Genova</span>  <span class="cit-name-given-names">M. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Structural and functional organization of the mitochondrial respiratory chain: a dynamic super-assembly.</span> <abbr class="cit-jnl-abbrev">Int. J. Biochem. Cell Biol.</abbr> <span class="cit-vol">41</span>(<span class="cit-issue">10</span>): <span class="cit-fpage">1750</span>–<span class="cit-lpage">1772</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2Binternational%2Bjournal%2Bof%2Bbiochemistry%2B%2526%2Bcell%2Bbiology%26rft.stitle%253DInt%2BJ%2BBiochem%2BCell%2BBiol%26rft.aulast%253DLenaz%26rft.auinit1%253DG.%26rft.volume%253D41%26rft.issue%253D10%26rft.spage%253D1750%26rft.epage%253D1772%26rft.atitle%253DStructural%2Band%2Bfunctional%2Borganization%2Bof%2Bthe%2Bmitochondrial%2Brespiratory%2Bchain%253A%2Ba%2Bdynamic%2Bsuper-assembly.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.biocel.2009.04.003%26rft_id%253Dinfo%253Apmid%252F19711505%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.biocel.2009.04.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19711505&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.34" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>(<span class="cit-issue">14</span>): <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTQvMTc1NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi80NDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.35" data-doi="10.1016/0092-8674(93)90050-Z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liao</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Butow</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">RTG1 and RTG2: two yeast genes required for a novel path of communication from mitochondria to the nucleus.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">72</span>: <span class="cit-fpage">61</span>–<span class="cit-lpage">71</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DLiao%26rft.auinit1%253DX.%26rft.volume%253D72%26rft.issue%253D1%26rft.spage%253D61%26rft.epage%253D71%26rft.atitle%253DRTG1%2Band%2BRTG2%253A%2Btwo%2Byeast%2Bgenes%2Brequired%2Bfor%2Ba%2Bnovel%2Bpath%2Bof%2Bcommunication%2Bfrom%2Bmitochondria%2Bto%2Bthe%2Bnucleus.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252893%252990050-Z%26rft_id%253Dinfo%253Apmid%252F8422683%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(93)90050-Z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8422683&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993KG95500010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.36" data-doi="10.1128/MCB.19.10.6720"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Butow</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">A transcriptional switch in the expression of yeast tricarboxylic acid cycle genes in response to a reduction or loss of respiratory function.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">19</span>(<span class="cit-issue">10</span>): <span class="cit-fpage">6720</span>–<span class="cit-lpage">6728</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DLiu%26rft.auinit1%253DZ.%26rft.volume%253D19%26rft.issue%253D10%26rft.spage%253D6720%26rft.epage%253D6728%26rft.atitle%253DA%2BTranscriptional%2BSwitch%2Bin%2Bthe%2BExpression%2Bof%2BYeast%2BTricarboxylic%2BAcid%2BCycle%2BGenes%2Bin%2BResponse%2Bto%2Ba%2BReduction%2Bor%2BLoss%2Bof%2BRespiratory%2BFunction%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.19.10.6720%26rft_id%253Dinfo%253Apmid%252F10490611%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIxOS8xMC82NzIwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzQ0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.37" data-doi="10.1146/annurev.genet.40.110405.090613"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Butow</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Mitochondrial retrograde regulation.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">159</span>–<span class="cit-lpage">185</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DLiu%26rft.auinit1%253DZ.%26rft.volume%253D40%26rft.spage%253D159%26rft.epage%253D185%26rft.atitle%253DMitochondrial%2Bretrograde%2Bsignaling.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.40.110405.090613%26rft_id%253Dinfo%253Apmid%252F16771627%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genet.40.110405.090613&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16771627&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243038200007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.38" data-doi="10.1128/MMBR.00015-06"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">MacPherson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larochelle</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turcotte</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A fungal family of transcriptional regulators: the zinc cluster proteins.</span> <abbr class="cit-jnl-abbrev">Microbiol. Mol. Biol. Rev.</abbr> <span class="cit-vol">70</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">583</span>–<span class="cit-lpage">604</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%2Band%2BMolecular%2BBiology%2BReviews%26rft.stitle%253DMicrobiol.%2BMol.%2BBiol.%2BRev.%26rft.aulast%253DMacPherson%26rft.auinit1%253DS.%26rft.volume%253D70%26rft.issue%253D3%26rft.spage%253D583%26rft.epage%253D604%26rft.atitle%253DA%2BFungal%2BFamily%2Bof%2BTranscriptional%2BRegulators%253A%2Bthe%2BZinc%2BCluster%2BProteins%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMMBR.00015-06%26rft_id%253Dinfo%253Apmid%252F16959962%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibW1iciI7czo1OiJyZXNpZCI7czo4OiI3MC8zLzU4MyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi80NDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.39" data-doi="10.1016/S0378-1119(02)00558-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Magasanik</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaiser</span>  <span class="cit-name-given-names">C. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Nitrogen regulation in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">290</span>(<span class="cit-issue">1–2</span>): <span class="cit-fpage">1</span>–<span class="cit-lpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DMagasanik%26rft.auinit1%253DB.%26rft.volume%253D290%26rft.issue%253D1-2%26rft.spage%253D1%26rft.epage%253D18%26rft.atitle%253DNitrogen%2Bregulation%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0378-1119%252802%252900558-9%26rft_id%253Dinfo%253Apmid%252F12062797%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0378-1119(02)00558-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12062797&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000176479500001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marzluf</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Genetic regulation of nitrogen metabolism in the fungi.</span> <abbr class="cit-jnl-abbrev">Microbiol. Mol. Biol. Rev.</abbr> <span class="cit-vol">61</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">17</span>–<span class="cit-lpage">32</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%2Band%2BMolecular%2BBiology%2BReviews%26rft.stitle%253DMicrobiol.%2BMol.%2BBiol.%2BRev.%26rft.aulast%253DMarzluf%26rft.auinit1%253DG.%2BA.%26rft.volume%253D61%26rft.issue%253D1%26rft.spage%253D17%26rft.epage%253D32%26rft.atitle%253DGenetic%2Bregulation%2Bof%2Bnitrogen%2Bmetabolism%2Bin%2Bthe%2Bfungi%26rft_id%253Dinfo%253Apmid%252F9106362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibW1iciI7czo1OiJyZXNpZCI7czo3OiI2MS8xLzE3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzQ0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.41" data-doi="10.1071/FP08025"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Alternative oxidase: an inter-kingdom perspective on the function and regulation of this broadly distributed “cyanide-resistant” terminal oxidase.</span> <abbr class="cit-jnl-abbrev">Funct. Plant Biol.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">535</span>–<span class="cit-lpage">552</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFunct.%2BPlant%2BBiol.%26rft.volume%253D35%26rft.spage%253D535%26rft_id%253Dinfo%253Adoi%252F10.1071%252FFP08025%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1071/FP08025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000258592100002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vanlerberghe</span>  <span class="cit-name-given-names">G. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Origins, evolutionary history, and taxonomic distribution of alternative oxidase and plastoquinol terminal oxidase.</span> <abbr class="cit-jnl-abbrev">Comp. Biochem. Physiol. Part D</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">364</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComp.%2BBiochem.%2BPhysiol.%2BPart%2BD%26rft.volume%253D1%26rft.spage%253D357%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.43" data-doi="10.1023/B:PLAN.0000023669.79465.d2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amirsadeghi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vanlerberghe</span>  <span class="cit-name-given-names">G. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Prokaryotic orthologues of mitochondrial alternative oxidase and plastid terminal oxidase.</span> <abbr class="cit-jnl-abbrev">Plant Mol. Biol.</abbr> <span class="cit-vol">53</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">865</span>–<span class="cit-lpage">876</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BMol%2BBiol%26rft.aulast%253DMcDonald%26rft.auinit1%253DA.%2BE.%26rft.volume%253D53%26rft.issue%253D6%26rft.spage%253D865%26rft.epage%253D876%26rft.atitle%253DProkaryotic%2Borthologues%2Bof%2Bmitochondrial%2Balternative%2Boxidase%2Band%2Bplastid%2Bterminal%2Boxidase.%26rft_id%253Dinfo%253Adoi%252F10.1023%252FB%253APLAN.0000023669.79465.d2%26rft_id%253Dinfo%253Apmid%252F15082931%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/B:PLAN.0000023669.79465.d2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15082931&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220816200010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.44" data-doi="10.1242/jeb.032151"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vanlerberghe</span>  <span class="cit-name-given-names">G. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Staples</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Alternative oxidase in animals: unique characteristics and taxonomic distribution.</span> <abbr class="cit-jnl-abbrev">J. Exp. Biol.</abbr> <span class="cit-vol">212</span>(<span class="cit-issue">Pt 16</span>): <span class="cit-fpage">2627</span>–<span class="cit-lpage">2634</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BExperimental%2BBiology%26rft.stitle%253DJ.%2BExp.%2BBiol.%26rft.aulast%253DMcDonald%26rft.auinit1%253DA.%2BE.%26rft.volume%253D212%26rft.issue%253D16%26rft.spage%253D2627%26rft.epage%253D2634%26rft.atitle%253DAlternative%2Boxidase%2Bin%2Banimals%253A%2Bunique%2Bcharacteristics%2Band%2Btaxonomic%2Bdistribution%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fjeb.032151%26rft_id%253Dinfo%253Apmid%252F19648408%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamV4YmlvIjtzOjU6InJlc2lkIjtzOjExOiIyMTIvMTYvMjYyNyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi80NDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.45" data-doi="10.1016/j.cell.2008.06.043"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meisinger</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sickmann</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pfanner</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The mitochondrial proteome: from inventory to function.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">134</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">22</span>–<span class="cit-lpage">24</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DMeisinger%26rft.auinit1%253DC.%26rft.volume%253D134%26rft.issue%253D1%26rft.spage%253D22%26rft.epage%253D24%26rft.atitle%253DThe%2Bmitochondrial%2Bproteome%253A%2Bfrom%2Binventory%2Bto%2Bfunction.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2008.06.043%26rft_id%253Dinfo%253Apmid%252F18614007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2008.06.043&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18614007&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257513100010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Metzenberg</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Vogel’s medium N salts: avoiding the need for ammonium nitrate.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Newsl.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2BGenet.%2BNewsl.%26rft.volume%253D50%26rft.spage%253D14%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.47" data-doi="10.1146/annurev-arplant-042811-105432"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shiba</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harada</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kita</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Unraveling the heater: new insights into the structure of the alternative oxidase.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Plant Biol.</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">637</span>–<span class="cit-lpage">663</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BPlant%2BBiol.%26rft.volume%253D64%26rft.spage%253D637%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-arplant-042811-105432%26rft_id%253Dinfo%253Apmid%252F23638828%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-arplant-042811-105432&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23638828&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000321699500026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-book" id="cit-7.2.449.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nargang</span>  <span class="cit-name-given-names">F. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rapaport</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Neurospora crassa as a model organism for mitochondrial biogenesis</span>, pp. <span class="cit-fpage">107</span>–<span class="cit-lpage">123</span> in <em>Mitochondria. Practical Protocols</em>, edited by <span class="cit-ed"><span class="cit-name-surname">Leister</span>  <span class="cit-name-given-names">D. L.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Herrmann</span>  <span class="cit-name-given-names">J.</span></span>. <span class="cit-publ-name">Humana Press</span>, <span class="cit-publ-loc">Totowa, NJ</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neimanis</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Staples</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huner</span>  <span class="cit-name-given-names">N. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Identification, expression, and taxonomic distribution of alternative oxidases in non-angiosperm plants.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">526</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">275</span>–<span class="cit-lpage">286</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.volume%253D526%26rft.spage%253D275%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-web no-rev-xref" id="cit-7.2.449.50"><div class="cit-metadata unstructured">Oliveros, J. C., 2007–2015 Venny. An interactive tool for comparing lists with Venn’s diagrams. Accessed June 23, 2015. Available at: <a href="http://bioinfogp.cnb.csic.es/tools/venny/index.html">http://bioinfogp.cnb.csic.es/tools/venny/index.html</a></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.51" data-doi="10.1093/bioinformatics/btu627"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Priebe</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kreisel</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horn</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guthke</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Linde</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">FungiFun2: a comprehensive online resource for systematic analysis of gene lists from fungal species.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">445</span>–<span class="cit-lpage">446</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu627%26rft_id%253Dinfo%253Apmid%252F25294921%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIzMS8zLzQ0NSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi80NDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.52"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ribas</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia-Ruiz</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernandez-Checa</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Glutathione and mitochondria.</span> <abbr class="cit-jnl-abbrev">Front. Pharmacol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">151</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPharmacol.%26rft.volume%253D5%26rft.spage%253D151%26rft_id%253Dinfo%253Apmid%252F25024695%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=25024695&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.53" data-doi="10.1093/nar/gkh894"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ruepp</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zollner</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albermann</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hani</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The FunCat, a functional annotation scheme for systematic classification of proteins from whole genomes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">32</span>(<span class="cit-issue">18</span>): <span class="cit-fpage">5539</span>–<span class="cit-lpage">5545</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DRuepp%26rft.auinit1%253DA.%26rft.volume%253D32%26rft.issue%253D18%26rft.spage%253D5539%26rft.epage%253D5545%26rft.atitle%253DThe%2BFunCat%252C%2Ba%2Bfunctional%2Bannotation%2Bscheme%2Bfor%2Bsystematic%2Bclassification%2Bof%2Bproteins%2Bfrom%2Bwhole%2Bgenomes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh894%26rft_id%253Dinfo%253Apmid%252F15486203%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzMi8xOC81NTM5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzQ0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.54" data-doi="10.1016/0014-5793(93)80535-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sakajo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Minagawa</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yoshimoto</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Characterization of the alternative oxidase protein in the yeast <em>Hansenula anomala</em>.</span> <abbr class="cit-jnl-abbrev">FEBS</abbr> <span class="cit-vol">318</span>: <span class="cit-fpage">310</span>–<span class="cit-lpage">312</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BLetters%26rft.stitle%253DFEBS%2BLetters%26rft.aulast%253DSakajo%26rft.auinit1%253DS.%26rft.volume%253D318%26rft.issue%253D3%26rft.spage%253D310%26rft.epage%253D312%26rft.atitle%253DCharacterization%2Bof%2Bthe%2Balternative%2Boxidase%2Bprotein%2Bin%2Bthe%2Byeast%2BHansenula%2Banomala.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0014-5793%252893%252980535-3%26rft_id%253Dinfo%253Apmid%252F8440388%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0014-5793(93)80535-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8440388&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993KQ17200022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.55" data-doi="10.1074/jbc.M413667200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmitt</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ahting</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eichacker</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Granvogl</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Go</span>  <span class="cit-name-given-names">N. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Role of Tom5 in maintaining the structural stability of the TOM complex of mitochondria.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">280</span>: <span class="cit-fpage">14499</span>–<span class="cit-lpage">14506</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M413667200%26rft_id%253Dinfo%253Apmid%252F15701639%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODAvMTUvMTQ0OTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzIvNDQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.56" data-doi="10.1146/annurev.ge.24.120190.003051"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Selker</span>  <span class="cit-name-given-names">E. U.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Premeiotic instability of repeated sequences in <em>Neurospora crassa</em>.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">579</span>–<span class="cit-lpage">613</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DSelker%26rft.auinit1%253DE.%26rft.volume%253D24%26rft.issue%253D1%26rft.spage%253D579%26rft.epage%253D613%26rft.atitle%253DPremeiotic%2BInstability%2BOf%2BRepeated%2BSequences%2BIn%2BNeurospora%2BCrassa%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ge.24.120190.003051%26rft_id%253Dinfo%253Apmid%252F2150906%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.ge.24.120190.003051&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2150906&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990EN96900024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.57" data-doi="10.1534/genetics.109.100834"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sellem</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bovier</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sainsard-Chanet</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Mutations in two zinc cluster proteins activate alternative respiratory and gluconeogenic pathways and restore senescence in long-lived respiratory mutants of <em>Podospora anserina</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">69</span>–<span class="cit-lpage">78</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.100834%26rft_id%253Dinfo%253Apmid%252F19255367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTgyLzEvNjkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy83LzIvNDQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.58" data-doi="10.1091/mbc.E05-03-0187"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sherman</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Go</span>  <span class="cit-name-given-names">N. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nargang</span>  <span class="cit-name-given-names">F. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Functions of the small proteins in the TOM complex of <em>Neurospora crassa</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">4172</span>–<span class="cit-lpage">4182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E05-03-0187%26rft_id%253Dinfo%253Apmid%252F15987740%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjE2LzkvNDE3MiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi80NDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.59" data-doi="10.1002/(SICI)1097-0061(199908)15:11&lt;1021::AID-YEA429&gt;3.0.CO;2-V"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">N.-Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherman</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cruz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jeffries</span>  <span class="cit-name-given-names">T. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Disruption of the cytochrome c gene in xylose-utilizing yeast <em>Pichia stipitis</em> leads to higher ethanol production.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1021</span>–<span class="cit-lpage">1030</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DShi%26rft.auinit1%253DN.%2BQ.%26rft.volume%253D15%26rft.issue%253D11%26rft.spage%253D1021%26rft.epage%253D1030%26rft.atitle%253DDisruption%2Bof%2Bthe%2Bcytochrome%2Bc%2Bgene%2Bin%2Bxylose-utilizing%2Byeast%2BPichia%2Bstipitis%2Bleads%2Bto%2Bhigher%2Bethanol%2Bproduction.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F%2528SICI%25291097-0061%2528199908%252915%253A11%253C1021%253A%253AAID-YEA429%253E3.0.CO%253B2-V%26rft_id%253Dinfo%253Apmid%252F10455226%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/(SICI)1097-0061(199908)15:11&lt;1021::AID-YEA429&gt;3.0.CO;2-V&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10455226&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000082028500001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.60"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Solotoff</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moseler</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schulte</span>  <span class="cit-name-given-names">U.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Two pentatricopeptide repeat domain proteins are required for the synthesis of respiratory complex I.</span> <abbr class="cit-jnl-abbrev">Curr. Genet.</abbr> <span class="cit-vol">61</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">19</span>–<span class="cit-lpage">29</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BGenet.%26rft.volume%253D61%26rft.spage%253D19%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.61"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Staben</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pollock</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schechtman</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Use of bacterial hygromycin B resistance gene as a dominant selectable marker in <em>Neurospora crassa</em> transformation.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Newsl.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">79</span>–<span class="cit-lpage">81</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2BGenet.%2BNewsl.%26rft.volume%253D36%26rft.spage%253D79%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.62" data-doi="10.1101/gr.403602"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stein</span>  <span class="cit-name-given-names">L. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mungall</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caudy</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mangone</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">The generic genome browser: a building block for a model organism system database.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">12</span>(<span class="cit-issue">10</span>): <span class="cit-fpage">1599</span>–<span class="cit-lpage">1610</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DStein%26rft.auinit1%253DL.%2BD.%26rft.volume%253D12%26rft.issue%253D10%26rft.spage%253D1599%26rft.epage%253D1610%26rft.atitle%253DThe%2BGeneric%2BGenome%2BBrowser%253A%2BA%2BBuilding%2BBlock%2Bfor%2Ba%2BModel%2BOrganism%2BSystem%2BDatabase%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.403602%26rft_id%253Dinfo%253Apmid%252F12368253%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxMi8xMC8xNTk5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzQ0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.63" data-doi="10.1111/j.1365-2958.2012.08067.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Suzuki</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">K. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hynes</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Reprogramming of carbon metabolism by the transcriptional activators AcuK and AcuM in Aspergillus nidulans.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">84</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">942</span>–<span class="cit-lpage">964</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DSuzuki%26rft.auinit1%253DY.%26rft.volume%253D84%26rft.issue%253D5%26rft.spage%253D942%26rft.epage%253D964%26rft.atitle%253DReprogramming%2Bof%2Bcarbon%2Bmetabolism%2Bby%2Bthe%2Btranscriptional%2Bactivators%2BAcuK%2Band%2BAcuM%2Bin%2BAspergillus%2Bnidulans.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2958.2012.08067.x%26rft_id%253Dinfo%253Apmid%252F22500966%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2958.2012.08067.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22500966&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.64" data-doi="10.1104/pp.70.3.704"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Talbot</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Russell</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title">Nuclear buoyant density determination and the purification and characterization of wild-type neurospora nuclei using percoll density gradients.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">70</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">704</span>–<span class="cit-lpage">708</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiology%26rft.stitle%253DPlant%2BPhysiol.%26rft.aulast%253DTalbot%26rft.auinit1%253DK.%2BJ.%26rft.volume%253D70%26rft.issue%253D3%26rft.spage%253D704%26rft.epage%253D708%26rft.atitle%253DNuclear%2BBuoyant%2BDensity%2BDetermination%2Band%2Bthe%2BPurification%2Band%2BCharacterization%2Bof%2BWild-Type%2BNeurospora%2BNuclei%2BUsing%2BPercoll%2BDensity%2BGradients%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.70.3.704%26rft_id%253Dinfo%253Apmid%252F16662561%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo4OiI3MC8zLzcwNCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzcvMi80NDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.65" data-doi="10.1016/S1087-1845(03)00002-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tanton</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nargang</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kessler</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nargang</span>  <span class="cit-name-given-names">F. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Alternative oxidase expression in <em>Neurospora crassa</em>.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Biol.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">176</span>–<span class="cit-lpage">190</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2Bgenetics%2Band%2Bbiology%2B%253A%2B%2BFG%2B%2526%2BB%26rft.stitle%253DFungal%2BGenet%2BBiol%26rft.aulast%253DTanton%26rft.auinit1%253DL.%2BL.%26rft.volume%253D39%26rft.issue%253D2%26rft.spage%253D176%26rft.epage%253D190%26rft.atitle%253DAlternative%2Boxidase%2Bexpression%2Bin%2BNeurospora%2Bcrassa.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1087-1845%252803%252900002-1%26rft_id%253Dinfo%253Apmid%252F12781676%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1087-1845(03)00002-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12781676&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.66"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tissieres</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">H. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haskins</span>  <span class="cit-name-given-names">F. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1953</span> <span class="cit-article-title">Studies on the respiratory system of the poky strain of <em>Neurospora</em>.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">205</span>: <span class="cit-fpage">423</span>–<span class="cit-lpage">433</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DTissieres%26rft.auinit1%253DA.%26rft.volume%253D205%26rft.issue%253D1%26rft.spage%253D423%26rft.epage%253D433%26rft.atitle%253DSTUDIES%2BON%2BTHE%2BRESPIRATORY%2BSYSTEM%2BOF%2BTHE%2BPOKY%2BSTRAIN%2BOF%2BNEUROSPORA%26rft_id%253Dinfo%253Apmid%252F13117920%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czozOiJqYmMiO3M6NToicmVzaWQiO3M6OToiMjA1LzEvNDIzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzQ0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.67"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Watters</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Randall</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Margolin</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Selker</span>  <span class="cit-name-given-names">E. U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stadler</span>  <span class="cit-name-given-names">D. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Action of repeat-induced point mutation on both strands of a duplex and on tandem duplications of various sizes in <em>Neurospora</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">153</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">705</span>–<span class="cit-lpage">714</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWatters%26rft.auinit1%253DM.%2BK.%26rft.volume%253D153%26rft.issue%253D2%26rft.spage%253D705%26rft.epage%253D714%26rft.atitle%253DAction%2Bof%2BRepeat-Induced%2BPoint%2BMutation%2Bon%2BBoth%2BStrands%2Bof%2Ba%2BDuplex%2Band%2Bon%2BTandem%2BDuplications%2Bof%2BVarious%2BSizes%2Bin%2BNeurospora%26rft_id%253Dinfo%253Apmid%252F10511550%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTUzLzIvNzA1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNy8yLzQ0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.68" data-doi="10.1186/s13059-014-0412-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Worsley Hunt</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wasserman</span>  <span class="cit-name-given-names">W. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Non-targeted transcription factors motifs are a systemic component of ChIP-seq datasets.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">15</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">412</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D15%26rft.spage%253D412%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-014-0412-4%26rft_id%253Dinfo%253Apmid%252F25070602%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-014-0412-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25070602&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.69"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yukioka</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Inagaki</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanaka</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Katoh</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miki</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Transcriptional activation of the alternative oxidase gene of the fungus <em>Magnaporthe grisea</em> by a respiratory-inhibiting fungicide and hydrogen peroxide.</span> <abbr class="cit-jnl-abbrev">Biochim. Biophys. Acta</abbr> <span class="cit-vol">1442</span>: <span class="cit-fpage">161</span>–<span class="cit-lpage">169</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.stitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.aulast%253DYukioka%26rft.auinit1%253DH.%26rft.volume%253D1442%26rft.issue%253D2-3%26rft.spage%253D161%26rft.epage%253D169%26rft.atitle%253DTranscriptional%2Bactivation%2Bof%2Bthe%2Balternative%2Boxidase%2Bgene%2Bof%2Bthe%2Bfungus%2BMagnaporthe%2Bgrisea%2Bby%2Ba%2Brespiratory-inhibiting%2Bfungicide%2Band%2Bhydrogen%2Bperoxide.%26rft_id%253Dinfo%253Apmid%252F9804939%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=9804939&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000076888100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-7.2.449.70" data-doi="10.1186/gb-2008-9-9-r137"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eeckhoute</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Model-based analysis of ChIP-Seq (MACS).</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">9</span>): <span class="cit-fpage">R137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DZhang%26rft.auinit1%253DY.%26rft.volume%253D9%26rft.issue%253D9%26rft.spage%253DR137%26rft.epage%253DR137%26rft.atitle%253DModel-based%2Banalysis%2Bof%2BChIP-Seq%2B%2528MACS%2529.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2008-9-9-r137%26rft_id%253Dinfo%253Apmid%252F18798982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2008-9-9-r137&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18798982&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F2%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/2/449.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/2/437" title="Screening and Analysis of Janelia FlyLight Project Enhancer-Gal4 Strains Identifies Multiple Gene Enhancers Active During Hematopoiesis in Normal and Wasp-Challenged Drosophila Larvae" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/2/467" title="Transcriptome Analysis of Drosophila melanogaster Third Instar Larval Ring Glands Points to Novel Functions and Uncovers a Cytochrome p450 Required for Development" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/2"><span>Volume 7 Issue 2, February 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/2.cover-source.jpg?itok=m4uMC43I" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/2/449" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Ollw3FTxlTgwzMyGiooT9BjSYPvs-I2D7QrExb2U1DA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/2/449" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/2/449" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/2/449" class="active" data-icon-position="" data-hide-link-title="0">Alternative Oxidase Transcription Factors AOD2 and AOD5 of Neurospora crassa Control the Expression of Genes Involved in Energy Production and Metabolism</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460549" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-mtbooNKDz_VUE0xQXfm_z7ZOFY0Eg3P3CEjlxyF9mLc" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/2/449.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/2/449" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-n1MR8Hxz2WlSPli1LB9BMxeqzJQJZkl1P75--Xi7I0I" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/2/449" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/2/449.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460549" id="node460549--2" data-pisa="ggg;7/2/449" data-pisa-master="ggg;g3.116.035402" data-apath="/ggg/7/2/449.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Alternative Oxidase Transcription Factors AOD2 and AOD5 of <em>Neurospora crassa</em> Control the Expression of Genes Involved in Energy Production and Metabolism</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Zhigang Qi</span>, <span class="highwire-citation-author" data-delta="1">Kristina M. Smith</span>, <span class="highwire-citation-author" data-delta="2">Erin L. Bredeweg</span>, <span class="highwire-citation-author" data-delta="3">Natasa Bosnjak</span>, <span class="highwire-citation-author" data-delta="4">Michael Freitag</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-1383-4720" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Frank E. Nargang</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">449-466;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.035402 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460549--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zhigang Qi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences, University of Alberta, Edmonton, Alberta T6G 2E9, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhigang%2BQi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Qi%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhigang%2BQi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Kristina M. Smith </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biochemistry and Biophysics, Oregon State University, Corvallis, Oregon 97331-4003</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kristina%2BM.%2BSmith%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Smith%20KM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKristina%2BM.%2BSmith%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Erin L. Bredeweg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biochemistry and Biophysics, Oregon State University, Corvallis, Oregon 97331-4003</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Erin%2BL.%2BBredeweg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bredeweg%20EL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AErin%2BL.%2BBredeweg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Natasa Bosnjak </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences, University of Alberta, Edmonton, Alberta T6G 2E9, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Natasa%2BBosnjak%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bosnjak%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANatasa%2BBosnjak%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Michael Freitag </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biochemistry and Biophysics, Oregon State University, Corvallis, Oregon 97331-4003</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BFreitag%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Freitag%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BFreitag%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Frank E. Nargang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences, University of Alberta, Edmonton, Alberta T6G 2E9, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Frank%2BE.%2BNargang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nargang%20FE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFrank%2BE.%2BNargang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1383-4720" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Frank E. Nargang</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:frank.nargang@ualberta.ca" class="" data-icon-position="" data-hide-link-title="0">frank.nargang@ualberta.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/2/449" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/2/449&amp;title=Alternative%20Oxidase%20Transcription%20Factors%20AOD2%20and%20AOD5%20of%20Neurospora%20crassa%20Control%20the%20Expression%20of%20Genes%20Involved%20in%20Energy%20Production%20and%20Metabolism" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/2/449&amp;title=Alternative%20Oxidase%20Transcription%20Factors%20AOD2%20and%20AOD5%20of%20Neurospora%20crassa%20Control%20the%20Expression%20of%20Genes%20Involved%20in%20Energy%20Production%20and%20Metabolism" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/2/449&amp;title=Alternative%20Oxidase%20Transcription%20Factors%20AOD2%20and%20AOD5%20of%20Neurospora%20crassa%20Control%20the%20Expression%20of%20Genes%20Involved%20in%20Energy%20Production%20and%20Metabolism" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/2/449&amp;text=Alternative%20Oxidase%20Transcription%20Factors%20AOD2%20and%20AOD5%20of%20Neurospora%20crassa%20Control%20the%20Expression%20of%20Genes%20Involved%20in%20Energy%20Production%20and%20Metabolism" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/2/449&amp;title=Alternative%20Oxidase%20Transcription%20Factors%20AOD2%20and%20AOD5%20of%20Neurospora%20crassa%20Control%20the%20Expression%20of%20Genes%20Involved%20in%20Energy%20Production%20and%20Metabolism" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/2/449&amp;t=Alternative%20Oxidase%20Transcription%20Factors%20AOD2%20and%20AOD5%20of%20Neurospora%20crassa%20Control%20the%20Expression%20of%20Genes%20Involved%20in%20Energy%20Production%20and%20Metabolism" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/2/449&amp;title=Alternative%20Oxidase%20Transcription%20Factors%20AOD2%20and%20AOD5%20of%20Neurospora%20crassa%20Control%20the%20Expression%20of%20Genes%20Involved%20in%20Energy%20Production%20and%20Metabolism" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/2/449&amp;title=Alternative%20Oxidase%20Transcription%20Factors%20AOD2%20and%20AOD5%20of%20Neurospora%20crassa%20Control%20the%20Expression%20of%20Genes%20Involved%20in%20Energy%20Production%20and%20Metabolism" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460549/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460549" id="node460549--3" data-pisa="ggg;7/2/449" data-pisa-master="ggg;g3.116.035402" data-apath="/ggg/7/2/449.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Alternative Oxidase Transcription Factors AOD2 and AOD5 of <em>Neurospora crassa</em> Control the Expression of Genes Involved in Energy Production and Metabolism</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Zhigang Qi</span>, <span class="highwire-citation-author" data-delta="1">Kristina M. Smith</span>, <span class="highwire-citation-author" data-delta="2">Erin L. Bredeweg</span>, <span class="highwire-citation-author" data-delta="3">Natasa Bosnjak</span>, <span class="highwire-citation-author" data-delta="4">Michael Freitag</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-1383-4720" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Frank E. Nargang</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">449-466;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.035402 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460549--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zhigang Qi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences, University of Alberta, Edmonton, Alberta T6G 2E9, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhigang%2BQi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Qi%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhigang%2BQi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Kristina M. Smith </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biochemistry and Biophysics, Oregon State University, Corvallis, Oregon 97331-4003</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kristina%2BM.%2BSmith%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Smith%20KM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKristina%2BM.%2BSmith%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Erin L. Bredeweg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biochemistry and Biophysics, Oregon State University, Corvallis, Oregon 97331-4003</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Erin%2BL.%2BBredeweg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bredeweg%20EL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AErin%2BL.%2BBredeweg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Natasa Bosnjak </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences, University of Alberta, Edmonton, Alberta T6G 2E9, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Natasa%2BBosnjak%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bosnjak%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANatasa%2BBosnjak%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Michael Freitag </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biochemistry and Biophysics, Oregon State University, Corvallis, Oregon 97331-4003</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BFreitag%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Freitag%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BFreitag%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Frank E. Nargang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences, University of Alberta, Edmonton, Alberta T6G 2E9, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Frank%2BE.%2BNargang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nargang%20FE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFrank%2BE.%2BNargang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1383-4720" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Frank E. Nargang</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:frank.nargang@ualberta.ca" class="" data-icon-position="" data-hide-link-title="0">frank.nargang@ualberta.ca</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=449&amp;rft.epage=466&amp;rft.atitle=Alternative%20Oxidase%20Transcription%20Factors%20AOD2%20and%20AOD5%20of%20Neurospora%20crassa%20Control%20the%20Expression%20of%20Genes%20Involved%20in%20Energy%20Production%20and%20Metabolism&amp;rft.volume=7&amp;rft.issue=2&amp;rft.date=2017-02-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Qi%2C+Zhigang&amp;rft.au=Smith%2C+Kristina+M.&amp;rft.au=Bredeweg%2C+Erin+L.&amp;rft.au=Bosnjak%2C+Natasa&amp;rft.au=Freitag%2C+Michael&amp;rft.au=Nargang%2C+Frank+E."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460549/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460549/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460549/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460549/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460549/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460549/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460549/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460549/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460549/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460549/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460549/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460549/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/2/449" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/2/449.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/2/449.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/2/449.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/2/449.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
