<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/3/6/959" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/3/6/959.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/3/6/959.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/3/6/959.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/6.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/3/6/959.atom" />
<meta name="HW.pisa" content="ggg;3/6/959" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Resequencing and Comparative Genomics of Stagonospora nodorum: Sectional Gene Absence and Effector Discovery" />
<meta name="DC.Identifier" content="10.1534/g3.112.004994" />
<meta name="DC.Date" content="2013-06-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2013 Syme et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Stagonospora nodorum is an important wheat ( Triticum aestivum ) pathogen in many parts of the world, causing major yield losses. It was the first species in the large fungal Dothideomycete class to be genome sequenced. The reference genome sequence (SN15) has been instrumental in the discovery of genes encoding necrotrophic effectors that induce disease symptoms in specific host genotypes. Here we present the genome sequence of two further S. nodorum strains (Sn4 and Sn79) that differ in their effector repertoire from the reference. Sn79 is avirulent on wheat and produces no apparent effectors when infiltrated onto many cultivars and mapping population parents. Sn4 is pathogenic on wheat and has virulences not found in SN15. The new strains, sequenced with short-read Illumina chemistry, are compared with SN15 by a combination of mapping and de novo assembly approaches. Each of the genomes contains a large number of strain-specific genes, many of which have no meaningful similarity to any known gene. Large contiguous sections of the reference genome are absent in the two newly sequenced strains. We refer to these differences as “sectional gene absences.” The presence of genes in pathogenic strains and absence in Sn79 is added to computationally predicted properties of known proteins to produce a list of likely effector candidates. Transposon insertion was observed in the mitochondrial genomes of virulent strains where the avirulent strain retained the likely ancestral sequence. The study suggests that short-read enabled comparative genomics is an effective way to both identify new S. nodorum effector candidates and to illuminate evolutionary processes in this species." />
<meta name="DC.Contributor" content="Robert Andrew Syme" />
<meta name="DC.Contributor" content="James K. Hane" />
<meta name="DC.Contributor" content="Timothy L. Friesen" />
<meta name="DC.Contributor" content="Richard P. Oliver" />
<meta name="article:published_time" content="2013-06-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Resequencing and Comparative Genomics of Stagonospora nodorum: Sectional Gene Absence and Effector Discovery" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;&lt;i&gt;Stagonospora nodorum&lt;/i&gt; is an important wheat (&lt;i&gt;Triticum aestivum&lt;/i&gt;) pathogen in many parts of the world, causing major yield losses. It was the first species in the large fungal Dothideomycete class to be genome sequenced. The reference genome sequence (SN15) has been instrumental in the discovery of genes encoding necrotrophic effectors that induce disease symptoms in specific host genotypes. Here we present the genome sequence of two further &lt;i&gt;S. nodorum&lt;/i&gt; strains (Sn4 and Sn79) that differ in their effector repertoire from the reference. Sn79 is avirulent on wheat and produces no apparent effectors when infiltrated onto many cultivars and mapping population parents. Sn4 is pathogenic on wheat and has virulences not found in SN15. The new strains, sequenced with short-read Illumina chemistry, are compared with SN15 by a combination of mapping and de novo assembly approaches. Each of the genomes contains a large number of strain-specific genes, many of which have no meaningful similarity to any known gene. Large contiguous sections of the reference genome are absent in the two newly sequenced strains. We refer to these differences as “sectional gene absences.” The presence of genes in pathogenic strains and absence in Sn79 is added to computationally predicted properties of known proteins to produce a list of likely effector candidates. Transposon insertion was observed in the mitochondrial genomes of virulent strains where the avirulent strain retained the likely ancestral sequence. The study suggests that short-read enabled comparative genomics is an effective way to both identify new &lt;i&gt;S. nodorum&lt;/i&gt; effector candidates and to illuminate evolutionary processes in this species.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2013/06/01" />
<meta name="citation_mjid" content="ggg;3/6/959" />
<meta name="citation_id" content="3/6/959" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/3/6/959" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/3/6/959.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/3/6/959.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/3/6/959.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.112.004994" />
<meta name="citation_pmid" content="23589517" />
<meta name="citation_volume" content="3" />
<meta name="citation_issue" content="6" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="959" />
<meta name="citation_lastpage" content="969" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Robert Andrew Syme" />
<meta name="citation_author_institution" content="Australian Centre for Necrotrophic Fungal Pathogens, Curtin University, Department of Environment and Agriculture, Bentley WA 6845, Australia" />
<meta name="citation_author" content="James K. Hane" />
<meta name="citation_author_institution" content="Molecular Pathology and Plant Pathology Laboratory, Centre for Environment and Life Sciences, CSIRO, Floreat WA 6014, Australia" />
<meta name="citation_author" content="Timothy L. Friesen" />
<meta name="citation_author_institution" content="U.S. Department of Agriculture, Agricultural Research Service, Cereal Crops Research Unit, Fargo, North Dakota 58102-2765" />
<meta name="citation_author" content="Richard P. Oliver" />
<meta name="citation_author_institution" content="Australian Centre for Necrotrophic Fungal Pathogens, Curtin University, Department of Environment and Agriculture, Bentley WA 6845, Australia" />
<meta name="citation_author_email" content="richard.oliver@curtin.edu.au" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=N. S. Abeysekara;citation_author=T. L. Friesen;citation_author=B. Keller;citation_author=J. D. Faris;citation_title=Identification and characterization of a novel host-toxin interaction in the wheat-Stagonospora nodorum pathosystem.;citation_pages=117-126;citation_volume=120;citation_year=2009;citation_issue=1;citation_pmid=19816671;citation_doi=10.1007/s00122-009-1163-6" />
<meta name="citation_reference" content="citation_journal_title=Phytopathology;citation_journal_abbrev=Phytopathology;citation_author=N. S. Abeysekara;citation_author=J. D. Faris;citation_author=S. Chao;citation_author=P. E. McClean;citation_author=T. L. Friesen;citation_title=Whole-genome QTL analysis of Stagonospora nodorum blotch resistance and validation of the SnTox4-Snn4 interaction in hexaploid wheat.;citation_pages=94-104;citation_volume=102;citation_year=2012;citation_issue=1;citation_pmid=21864084;citation_doi=10.1094/PHYTO-02-11-0040" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=S. Anders;citation_title=Visualization of genomic data with the Hilbert curve.;citation_volume=25;citation_year=2009;citation_pmid=19297348;citation_doi=10.1093/bioinformatics/btp152" />
<meta name="citation_reference" content="citation_journal_title=Australas. Plant Pathol.;citation_author=E. A. Antoni;citation_author=K. Rybak;citation_author=M. P. Tucker;citation_author=J. K. Hane;citation_author=P. S. Solomon;citation_title=Ubiquity of ToxA and absence of ToxB in Australian populations of Pyrenophora tritici-repentis.;citation_pages=63-68;citation_volume=39;citation_year=2010;citation_doi=10.1071/AP09056" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=J. D. Bendtsen;citation_author=H. Nielsen;citation_author=G. von Heijne;citation_author=S. Brunak;citation_title=Improved prediction of signal peptides: SignalP 3.0.;citation_pages=783-795;citation_volume=340;citation_year=2004;citation_issue=4;citation_pmid=15223320;citation_doi=10.1016/j.jmb.2004.05.028" />
<meta name="citation_reference" content="citation_journal_title=Fungal genetics and biology :  FG &amp; B;citation_journal_abbrev=Fungal Genet Biol;citation_author=R. S. Bennett;citation_author=S. H. Yun;citation_author=T. Y. Lee;citation_author=B. G. Turgeon;citation_author=E. Arseniuk;citation_title=Identity and conservation of mating type genes in geographically diverse isolates of Phaeosphaeria nodorum.;citation_pages=25-37;citation_volume=40;citation_year=2003;citation_issue=1;citation_pmid=12948511;citation_doi=10.1016/S1087-1845(03)00062-8" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=S. Bringans;citation_author=J. K. Hane;citation_author=T. Casey;citation_author=K. C. Tan;citation_author=R. Lipscombe;citation_title=Deep proteogenomics; high throughput gene validation by multidimensional liquid chromatography and mass spectrometry of proteins from the fungal wheat pathogen Stagonospora nodorum.;citation_pages=301-301;citation_volume=10;citation_year=2009;citation_pmid=19772613;citation_doi=10.1186/1471-2105-10-301" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=E. B. Cambareri;citation_author=B. C. Jensen;citation_author=E. Schabtach;citation_author=E. U. Selker;citation_title=Repeat-induced G-C to A-T mutations in Neurospora;citation_pages=1571-1575;citation_volume=244;citation_year=1989;citation_issue=4912;citation_pmid=2544994;citation_doi=10.1126/science.2544994" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=B. L. Cantarel;citation_author=I. Korf;citation_author=S. M. C. Robb;citation_author=G. Parra;citation_author=E. Ross;citation_title=MAKER: an easy-to-use annotation pipeline designed for emerging model organism genomes.;citation_volume=18;citation_year=2008;citation_pmid=18025269;citation_doi=10.1101/gr.6743907" />
<meta name="citation_reference" content="citation_journal_title=Proteomics;citation_journal_abbrev=Proteomics;citation_author=T. Casey;citation_author=P. S. Solomon;citation_author=S. Bringans;citation_author=K. C. Tan;citation_author=R. P. Oliver;citation_title=Quantitative proteomic analysis of G-protein signalling in Stagonospora nodorum using isobaric tags for relative and absolute quantification.;citation_pages=38-47;citation_volume=10;citation_year=2010;citation_issue=1;citation_pmid=19882661;citation_doi=10.1002/pmic.200900474" />
<meta name="citation_reference" content="citation_journal_title=Mycologia;citation_journal_abbrev=Mycologia;citation_author=G. J. Celio;citation_author=M. Padamsee;citation_author=B. T. M. Dentinger;citation_author=R. Bauer;citation_author=D. J. McLaughlin;citation_title=Assembling the Fungal Tree of Life: constructing the Structural and Biochemical Database;citation_pages=850-859;citation_volume=98;citation_year=2006;citation_issue=6;citation_pmid=17486962;citation_doi=10.3852/mycologia.98.6.850" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=C.-G. Chu;citation_author=J. D. Faris;citation_author=S. Xu;citation_author=T. L. Friesen;citation_title=Genetic analysis of disease susceptibility contributed by the compatible Tsn1-SnToxA and Snn1-SnTox1 interactions in the wheat-Stagonospora nodorum pathosystem.;citation_pages=1451-1459;citation_volume=129;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=The Plant Cell Online;citation_journal_abbrev=Plant Cell;citation_author=L. M. Ciuffetti;citation_author=R. P. Tuori;citation_author=J. M. Gaventa;citation_title=A Single Gene Encodes a Selective Toxin Causal to the Development of Tan Spot of Wheat;citation_pages=135-144;citation_volume=9;citation_year=1997;citation_issue=2;citation_pmid=9061946;citation_doi=10.1105/tpc.9.2.135" />
<meta name="citation_reference" content="citation_journal_title=Fungal genetics and biology :  FG &amp; B;citation_journal_abbrev=Fungal Genet Biol;citation_author=A. J. Clutterbuck;citation_title=Genomic evidence of repeat-induced point mutation (RIP) in filamentous ascomycetes.;citation_pages=306-326;citation_volume=48;citation_year=2011;citation_issue=3;citation_pmid=20854921;citation_doi=10.1016/j.fgb.2010.09.002" />
<meta name="citation_reference" content="citation_journal_title=Molecular &amp; general genetics :  MGG;citation_journal_abbrev=Mol Gen Genet;citation_author=R. N. Cooley;citation_author=C. E. Caten;citation_title=Variation in electrophoretic karyotype between strains of Septoria nodorum.;citation_pages=17-23;citation_volume=228;citation_year=1991;citation_issue=1-2;citation_pmid=1886607" />
<meta name="citation_reference" content="citation_journal_title=Phytopathology;citation_journal_abbrev=Phytopathology;citation_author=A. D. Crook;citation_author=T. L. Friesen;citation_author=Z. H. Liu;citation_author=P. S. Ojiambo;citation_author=C. Cowger;citation_title=Novel necrotrophic effectors from Stagonospora nodorum and corresponding host sensitivities in winter wheat germplasm in the southeastern United States.;citation_pages=498-505;citation_volume=102;citation_year=2012;citation_issue=5;citation_pmid=22494247;citation_doi=10.1094/PHYTO-08-11-0238" />
<meta name="citation_reference" content="citation_journal_abbrev=Mol Plant Pathol;citation_author=P. J. G. M. De Wit;citation_author=R. Mehrabi;citation_author=H. A. Van Den Burg;citation_author=I. Stergiopoulos;citation_title=Fungal effector proteins: past, present and future.;citation_pages=735-747;citation_volume=10;citation_year=2009;citation_issue=6;citation_pmid=19849781;citation_doi=10.1111/j.1364-3703.2009.00591.x" />
<meta name="citation_reference" content="citation_title=Phytochem. Bull.;citation_author=J. Doyle;citation_author=J. Doyle;citation_title=A rapid DNA isolation procedure for small quantities of fresh leaf tissue.;citation_pages=11-15;citation_volume=19;citation_year=1987" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=R. C. Edgar;citation_title=MUSCLE: a multiple sequence alignment method with reduced time and space complexity.;citation_pages=113-113;citation_volume=5;citation_year=2004;citation_pmid=15318951;citation_doi=10.1186/1471-2105-5-113" />
<meta name="citation_reference" content="citation_journal_title=Phytopathology;citation_journal_abbrev=Phytopathology;citation_author=J. D. Faris;citation_author=T. L. Friesen;citation_title=Reevaluation of a tetraploid wheat population indicates that the Tsn1-ToxA interaction is the only factor governing Stagonospora nodorum blotch susceptibility.;citation_pages=906-912;citation_volume=99;citation_year=2009;citation_issue=8;citation_pmid=19594309;citation_doi=10.1094/PHYTO-99-8-0906" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. D. Faris;citation_author=Z. Zhang;citation_author=H. Lu;citation_author=S. Lu;citation_author=L. Reddy;citation_title=A unique wheat disease resistance-like gene governs effector-triggered susceptibility to necrotrophic pathogens.;citation_volume=107;citation_year=2010;citation_pmid=20624958;citation_doi=10.1073/pnas.1004090107" />
<meta name="citation_reference" content="citation_journal_title=Molecular plant-microbe interactions :  MPMI;citation_journal_abbrev=Mol Plant Microbe Interact;citation_author=J. D. Faris;citation_author=Z. Zhang;citation_author=J. B. Rasmussen;citation_author=T. L. Friesen;citation_title=Variable expression of the Stagonospora nodorum effector SnToxA among isolates is correlated with levels of disease in wheat.;citation_pages=1419-1426;citation_volume=24;citation_year=2011;citation_issue=12;citation_pmid=21770771;citation_doi=10.1094/MPMI-04-11-0094" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=T. L. Friesen;citation_author=E. H. Stukenbrock;citation_author=Z. Liu;citation_author=S. Meinhardt;citation_author=H. Ling;citation_title=Emergence of a new disease as a result of interspecific virulence gene transfer.;citation_pages=953-956;citation_volume=38;citation_year=2006;citation_issue=8;citation_pmid=16832356;citation_doi=10.1038/ng1839" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=T. L. Friesen;citation_author=S. W. Meinhardt;citation_author=J. D. Faris;citation_title=The Stagonospora nodorum-wheat pathosystem involves multiple proteinaceous host-selective toxins and corresponding host sensitivity genes that interact in an inverse gene-for-gene manner.;citation_pages=681-692;citation_volume=51;citation_year=2007;citation_issue=4;citation_pmid=17573802;citation_doi=10.1111/j.1365-313X.2007.03166.x" />
<meta name="citation_reference" content="citation_journal_title=Cellular microbiology;citation_journal_abbrev=Cell Microbiol;citation_author=T. L. Friesen;citation_author=J. D. Faris;citation_author=P. S. Solomon;citation_author=R. P. Oliver;citation_title=Host-specific toxins: effectors of necrotrophic pathogenicity.;citation_pages=1421-1428;citation_volume=10;citation_year=2008;citation_issue=7;citation_pmid=18384660;citation_doi=10.1111/j.1462-5822.2008.01153.x" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=T. L. Friesen;citation_author=C. G. Chu;citation_author=Z. H. Liu;citation_author=S. S. Xu;citation_author=S. Halley;citation_title=Host-selective toxins produced by Stagonospora nodorum confer disease susceptibility in adult wheat plants under field conditions.;citation_pages=1489-1497;citation_volume=118;citation_year=2009;citation_issue=8;citation_pmid=19266177;citation_doi=10.1007/s00122-009-0997-2" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=R. C. Gentleman;citation_author=V. J. Carey;citation_author=D. M. Bates;citation_author=B. Bolstad;citation_author=M. Dettling;citation_title=Bioconductor: open software development for computational biology and bioinformatics.;citation_pages=R80-R80;citation_volume=5;citation_year=2004;citation_issue=10;citation_pmid=15461798;citation_doi=10.1186/gb-2004-5-10-r80" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=S. B. Goodwin;citation_author=S. B. M’Barek;citation_author=B. Dhillon;citation_author=A. H. J. Wittenberg;citation_author=C. F. Crane;citation_title=Finished genome of the fungal wheat pathogen Mycosphaerella graminicola reveals dispensome structure, chromosome plasticity, and stealth pathogenesis.;citation_pages=1002070;citation_volume=7;citation_year=2011;citation_doi=10.1371/journal.pgen.1002070" />
<meta name="citation_reference" content="citation_title=Bioinformatics (Oxford, England).;citation_author=N. Goto;citation_author=P. Prins;citation_author=M. Nakao;citation_author=R. Bonnal;citation_author=J. Aerts;citation_title=BioRuby: Bioinformatics software for the Ruby programming language.;citation_volume=26;citation_year=2010;citation_pmid=20739307;citation_doi=10.1093/bioinformatics/btq475" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=J. K. Hane;citation_author=R. P. Oliver;citation_title=RIPCAL: a tool for alignment-based analysis of repeat-induced point mutations in fungal genomic sequences.;citation_pages=478-478;citation_volume=9;citation_year=2008;citation_pmid=19014496;citation_doi=10.1186/1471-2105-9-478" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=J. K. Hane;citation_author=R. P. Oliver;citation_title=In silico reversal of repeat-induced point mutation (RIP) identifies the origins of repeat families and uncovers obscured duplicated genes.;citation_pages=655-655;citation_volume=11;citation_year=2010;citation_pmid=21106049;citation_doi=10.1186/1471-2164-11-655" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=J. K. Hane;citation_author=R. G. Lowe;citation_author=P. S. Solomon;citation_author=K. C. Tan;citation_author=C. L. Schoch;citation_title=Dothideomycete plant interactions illuminated by genome sequencing and EST analysis of the wheat pathogen Stagonospora nodorum.;citation_volume=19;citation_year=2007;citation_pmid=18024570;citation_doi=10.1105/tpc.107.052829" />
<meta name="citation_reference" content="citation_title=The Mycota;citation_title=Genomic and comparative analysis of the class Dothideomycetes;citation_author=J. K. Hane;citation_author=A. Williams;citation_author=R. P. Oliver;citation_pages=205-226;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=P. Horton;citation_author=K. J. Park;citation_author=T. Obayashi;citation_author=N. Fujita;citation_author=H. Harada;citation_title=WOLF PSORT: PROTEIN LOCALIZATION PREDICTOR.;citation_pages=W585 587;citation_volume=35;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_abbrev=Mol Plant Pathol;citation_author=S. V. S. Ipcho;citation_author=J. K. Hane;citation_author=E. A. Antoni;citation_author=D. Ahren;citation_author=B. Henrissat;citation_title=Transcriptome analysis of Stagonospora nodorum: gene models, effectors, metabolism and pantothenate dispensability.;citation_pages=531-545;citation_volume=13;citation_year=2012;citation_issue=6;citation_pmid=22145589;citation_doi=10.1111/j.1364-3703.2011.00770.x" />
<meta name="citation_reference" content="citation_title=Bioinformatics.;citation_author=M. Kearse;citation_author=R. Moir;citation_author=A. Wilson;citation_author=S. Stones-Havas;citation_author=M. Cheung;citation_title=Geneious Basic: an integrated and extendable desktop software platform for the organization and analysis of sequence data.;citation_volume=28;citation_year=2012;citation_pmid=22543367;citation_doi=10.1093/bioinformatics/bts199" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=W. J. Kent;citation_title=BLAT - The BLAST-like alignment tool.;citation_volume=12;citation_year=2002;citation_pmid=11932250;citation_doi=10.1101/gr.229202. Article published online before March 2002" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=R. Knight;citation_author=P. Maxwell;citation_author=A. Birmingham;citation_author=J. Carnes;citation_author=J. G. Caporaso;citation_title=PyCogent: a toolkit for making sense from sequence.;citation_pages=R171-R171;citation_volume=8;citation_year=2007;citation_issue=8;citation_pmid=17708774;citation_doi=10.1186/gb-2007-8-8-r171" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. A. Larkin;citation_author=G. Blackshields;citation_author=N. P. Brown;citation_author=R. Chenna;citation_author=P. A. McGettigan;citation_title=Clustal W and Clustal X version 2.0.;citation_volume=23;citation_year=2007;citation_pmid=17846036;citation_doi=10.1093/bioinformatics/btm404" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=L. Li;citation_author=C. J. Stoeckert;citation_author=D. S. Roos;citation_title=OrthoMCL: Identification of Ortholog Groups for Eukaryotic Genomes;citation_pages=2178-2189;citation_volume=13;citation_year=2003;citation_issue=9;citation_pmid=12952885;citation_doi=10.1101/gr.1224503" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Pathog;citation_author=Z. Liu;citation_author=J. D. Faris;citation_author=R. P. Oliver;citation_author=K. C. Tan;citation_author=P. S. Solomon;citation_title=SnTox3 acts in effector triggered susceptibility to induce disease on wheat carrying the Snn3 gene.;citation_pages=e1000581-e1000581;citation_volume=5;citation_year=2009;citation_issue=9;citation_pmid=19806176;citation_doi=10.1371/journal.ppat.1000581" />
<meta name="citation_reference" content="citation_journal_title=PLoS Pathog.;citation_author=Z. Liu;citation_author=Z. Zhang;citation_author=J. D. Faris;citation_author=R. P. Oliver;citation_author=R. Syme;citation_title=The cysteine rich necrotrophic effector SnTox1 produced by Stagonospora nodorum triggers susceptibility of wheat lines harboring Snn1.;citation_pages=1002467;citation_volume=8;citation_year=2012;citation_doi=10.1371/journal.ppat.1002467" />
<meta name="citation_reference" content="citation_journal_abbrev=Mol Plant Pathol;citation_author=R. Luderer;citation_author=M. J. D. De Kock;citation_author=R. H. L. Dees;citation_author=P. De Wit;citation_author=M. Joosten;citation_title=Functional analysis of cysteine residues of ECP elicitor proteins of the fungal tomato pathogen Cladosporium fulvum.;citation_pages=91-95;citation_volume=3;citation_year=2002;citation_issue=2;citation_pmid=20569313;citation_doi=10.1046/j.1464-6722.2001.00095.x" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=L. J. Ma;citation_author=H. C. Van Der Does;citation_author=K. A. Borkovich;citation_author=J. J. Coleman;citation_author=M. J. Daboussi;citation_title=Comparative genomics reveals mobile pathogenicity chromosomes in Fusarium.;citation_pages=367-373;citation_volume=464;citation_year=2010;citation_issue=7287;citation_pmid=20237561;citation_doi=10.1038/nature08850" />
<meta name="citation_reference" content="citation_journal_title=Eur. J. Plant Pathol.;citation_author=A. Malkus;citation_author=Q. Song;citation_author=P. Cregan;citation_author=E. Arseniuk;citation_author=P. P. Ueng;citation_title=Genetic linkage map of Phaeosphaeria nodorum, the causal agent of stagonospora nodorum blotch disease of wheat.;citation_pages=681-690;citation_volume=124;citation_year=2009;citation_doi=10.1007/s10658-009-9454-y" />
<meta name="citation_reference" content="citation_title=G3&amp;#58; Genes|Genomes|Genetics;citation_journal_abbrev=G3;citation_author=V. A. Manning;citation_author=I. Pandelova;citation_author=B. Dhillon;citation_author=L. J. Wilhelm;citation_author=S. B. Goodwin;citation_title=Comparative Genomics of a Plant-Pathogenic Fungus, Pyrenophora tritici-repentis, Reveals Transduplication and the Impact of Repeat Elements on Pathogenicity and Population Divergence;citation_pages=41-63;citation_volume=3;citation_year=2013;citation_issue=1;citation_pmid=23316438;citation_doi=10.1534/g3.112.004044" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. H. McKenna;citation_author=M. Hanna;citation_author=E. Banks;citation_author=A. Sivachenko;citation_author=K. Cibulskis;citation_title=The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.;citation_volume=20;citation_year=2010;citation_pmid=20644199;citation_doi=10.1101/gr.107524.110" />
<meta name="citation_reference" content="citation_journal_title=Australas. Plant Pathol.;citation_author=G. M. Murray;citation_author=J. P. Brennan;citation_title=Estimating disease losses to the Australian wheat industry.;citation_pages=558-570;citation_volume=38;citation_year=2009;citation_doi=10.1071/AP09053" />
<meta name="citation_reference" content="citation_journal_title=Crop Pasture Sci.;citation_author=R. P. Oliver;citation_author=K. Rybak;citation_author=P. S. Solomon;citation_author=M. Ferguson-Hunt;citation_title=Prevalence of ToxA-sensitive alleles of the wheat gene Tsn1 in Australian and Chinese wheat cultivars;citation_pages=348-352;citation_volume=60;citation_year=2009;citation_doi=10.1071/CP08259" />
<meta name="citation_reference" content="citation_journal_title=Annual review of phytopathology;citation_journal_abbrev=Annu Rev Phytopathol;citation_author=R. P. Oliver;citation_author=T. L. Friesen;citation_author=J. D. Faris;citation_author=P. S. Solomon;citation_title=Stagonospora nodorum: from pathology to genomics and host resistance.;citation_pages=23-43;citation_volume=50;citation_year=2012;citation_pmid=22559071;citation_doi=10.1146/annurev-phyto-081211-173019" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. R. Quinlan;citation_author=I. M. Hall;citation_title=BEDTools: a flexible suite of utilities for comparing genomic features.;citation_volume=26;citation_year=2010;citation_pmid=20110278;citation_doi=10.1093/bioinformatics/btq033" />
<meta name="citation_reference" content="citation_title=Nat. Commun.;citation_author=T. Rouxel;citation_author=J. Grandaubert;citation_author=J. Hane;citation_author=C. Hoede;citation_author=A. van de Wouw;citation_title=THE COMPARTMENTALIZED GENOME OF LEPTOSPHAERIA MACULANS: DIVERSIFICATION OF EFFECTORS WITHIN GENOMIC REGIONS AFFECTED BY REPEAT INDUCED POINT MUTATIONS.;citation_pages=art. 202;citation_volume=2;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=E. U. Selker;citation_title=Premeiotic Instability Of Repeated Sequences In Neurospora Crassa;citation_pages=579-613;citation_volume=24;citation_year=1990;citation_issue=1;citation_pmid=2150906;citation_doi=10.1146/annurev.ge.24.120190.003051" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=E. U. Selker;citation_author=E. B. Cambareri;citation_author=B. C. Jensen;citation_author=K. R. Haack;citation_title=Rearrangement of duplicated DNA in specialized cells of Neurospora.;citation_pages=741-752;citation_volume=51;citation_year=1987;citation_issue=5;citation_pmid=2960455;citation_doi=10.1016/0092-8674(87)90097-3" />
<meta name="citation_reference" content="citation_journal_abbrev=Mol Plant Pathol;citation_author=P. S. Solomon;citation_author=R. G. T. Lowe;citation_author=K. C. Tan;citation_author=O. D. C. Waters;citation_author=R. P. Oliver;citation_title=Stagonospora nodorum: cause of stagonospora nodorum blotch of wheat.;citation_pages=147-156;citation_volume=7;citation_year=2006;citation_issue=3;citation_pmid=20507435;citation_doi=10.1111/j.1364-3703.2006.00326.x" />
<meta name="citation_reference" content="citation_journal_title=Eur. J. Plant Pathol.;citation_author=P. S. Solomon;citation_author=T. J. G. Wilson;citation_author=K. Rybak;citation_author=K. Parker;citation_author=R. G. T. Lowe;citation_title=Structural characterisation of the interaction between Triticum aestivum and the dothideomycete pathogen Stagonospora nodorum.;citation_pages=275-282;citation_volume=114;citation_year=2006;citation_doi=10.1007/s10658-005-5768-6" />
<meta name="citation_reference" content="citation_journal_abbrev=Mol Plant Pathol;citation_author=E. H. Stukenbrock;citation_author=B. A. McDonald;citation_title=Geographical variation and positive diversifying selection in the host-specific toxin SnToxA.;citation_pages=321-332;citation_volume=8;citation_year=2007;citation_issue=3;citation_pmid=20507502;citation_doi=10.1111/j.1364-3703.2007.00396.x" />
<meta name="citation_reference" content="citation_journal_title=Metabolomics;citation_author=K. C. Tan;citation_author=R. D. Trengove;citation_author=G. L. Maker;citation_author=R. P. Oliver;citation_author=P. S. Solomon;citation_title=Metabolite profiling identifies the mycotoxin alternariol in the pathogen Stagonospora nodorum.;citation_pages=330-335;citation_volume=5;citation_year=2009;citation_doi=10.1007/s11306-009-0158-2" />
<meta name="citation_reference" content="citation_journal_title=Funct. Plant Biol.;citation_author=K. C. Tan;citation_author=R. P. Oliver;citation_author=P. S. Solomon;citation_author=C. S. Moffat;citation_title=Proteinaceous necrotrophic effectors in fungal virulence.;citation_pages=907-912;citation_volume=37;citation_year=2010;citation_doi=10.1071/FP10067" />
<meta name="citation_reference" content="citation_journal_title=Molecular plant-microbe interactions :  MPMI;citation_journal_abbrev=Mol Plant Microbe Interact;citation_author=K.-C. Tan;citation_author=M. Ferguson-Hunt;citation_author=K. Rybak;citation_author=O. D. C. Waters;citation_author=W. A. Stanley;citation_title=Quantitative variation in effector activity of ToxA isoforms from Stagonospora nodorum and Pyrenophora tritici-repentis.;citation_pages=515-522;citation_volume=25;citation_year=2012;citation_issue=4;citation_pmid=22250581;citation_doi=10.1094/MPMI-10-11-0273" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=V. Ter-Hovhannisyan;citation_author=A. Lomsadze;citation_author=Y. O. Chernoff;citation_author=M. Borodovsky;citation_title=Gene prediction in novel fungal genomes using an ab initio algorithm with unsupervised training.;citation_volume=18;citation_year=2008;citation_pmid=18757608;citation_doi=10.1101/gr.081612.108" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Pathog;citation_author=A. P. Van de Wouw;citation_author=A. J. Cozijnsen;citation_author=J. K. Hane;citation_author=P. C. Brunner;citation_author=B. A. McDonald;citation_title=Evolution of linked avirulence effectors in Leptosphaeria maculans is affected by genomic environment and exposure to resistance genes in host plants.;citation_pages=e1001180-e1001180;citation_volume=6;citation_year=2010;citation_issue=11;citation_pmid=21079787;citation_doi=10.1371/journal.ppat.1001180" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=V. B. Yap;citation_author=H. Lindsay;citation_author=S. Easteal;citation_author=G. Huttley;citation_title=Estimates of the effect of natural selection on protein-coding content.;citation_volume=27;citation_year=2010;citation_pmid=19815689;citation_doi=10.1093/molbev/msp232" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. R. Zerbino;citation_author=E. Birney;citation_title=Velvet: algorithms for de novo short read assembly using de Bruijn graphs.;citation_volume=18;citation_year=2008;citation_pmid=18349386;citation_doi=10.1101/gr.074492.107" />
<meta name="citation_reference" content="citation_title=Studies in Mycology;citation_journal_abbrev=SIM;citation_author=Y. Zhang;citation_author=C. L. Schoch;citation_author=J. Fournier;citation_author=P. W. Crous;citation_author=J. de Gruyter;citation_title=Multi-locus phylogeny of Pleosporales: a taxonomic, ecological and evolutionary re-evaluation;citation_pages=85-102-S5;citation_volume=64;citation_year=2009;citation_issue=1;citation_pmid=20169024;citation_doi=10.3114/sim.2009.64.04" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=Z. Zhang;citation_author=T. L. Friesen;citation_author=S. S. Xu;citation_author=G. Shi;citation_author=Z. Liu;citation_title=Two putatively homoeologous wheat genes mediate recognition of SnTox3 to confer effector-triggered susceptibility to Stagonospora nodorum.;citation_pages=27-38;citation_volume=65;citation_year=2011;citation_issue=1;citation_pmid=21175887;citation_doi=10.1111/j.1365-313X.2010.04407.x" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Resequencing and Comparative Genomics of Stagonospora nodorum: Sectional Gene Absence and Effector Discovery" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/6.cover-source.jpg" />
<meta name="twitter:description" content="Stagonospora nodorum is an important wheat ( Triticum aestivum ) pathogen in many parts of the world, causing major yield losses. It was the first species in the large fungal Dothideomycete class to be genome sequenced. The reference genome sequence (SN15) has been instrumental in the discovery of genes encoding necrotrophic effectors that induce disease symptoms in specific host genotypes. Here we present the genome sequence of two further S. nodorum strains (Sn4 and Sn79) that differ in their effector repertoire from the reference. Sn79 is avirulent on wheat and produces no apparent effectors when infiltrated onto many cultivars and mapping population parents. Sn4 is pathogenic on wheat and has virulences not found in SN15. The new strains, sequenced with short-read Illumina chemistry, are compared with SN15 by a combination of mapping and de novo assembly approaches. Each of the genomes contains a large number of strain-specific genes, many of which have no meaningful similarity to any known gene. Large contiguous sections of the reference genome are absent in the two newly sequenced strains. We refer to these differences as “sectional gene absences.” The presence of genes in pathogenic strains and absence in Sn79 is added to computationally predicted properties of known proteins to produce a list of likely effector candidates. Transposon insertion was observed in the mitochondrial genomes of virulent strains where the avirulent strain retained the likely ancestral sequence. The study suggests that short-read enabled comparative genomics is an effective way to both identify new S. nodorum effector candidates and to illuminate evolutionary processes in this species." />
<meta name="og-title" property="og:title" content="Resequencing and Comparative Genomics of Stagonospora nodorum: Sectional Gene Absence and Effector Discovery" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/3/6/959" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Stagonospora nodorum is an important wheat ( Triticum aestivum ) pathogen in many parts of the world, causing major yield losses. It was the first species in the large fungal Dothideomycete class to be genome sequenced. The reference genome sequence (SN15) has been instrumental in the discovery of genes encoding necrotrophic effectors that induce disease symptoms in specific host genotypes. Here we present the genome sequence of two further S. nodorum strains (Sn4 and Sn79) that differ in their effector repertoire from the reference. Sn79 is avirulent on wheat and produces no apparent effectors when infiltrated onto many cultivars and mapping population parents. Sn4 is pathogenic on wheat and has virulences not found in SN15. The new strains, sequenced with short-read Illumina chemistry, are compared with SN15 by a combination of mapping and de novo assembly approaches. Each of the genomes contains a large number of strain-specific genes, many of which have no meaningful similarity to any known gene. Large contiguous sections of the reference genome are absent in the two newly sequenced strains. We refer to these differences as “sectional gene absences.” The presence of genes in pathogenic strains and absence in Sn79 is added to computationally predicted properties of known proteins to produce a list of likely effector candidates. Transposon insertion was observed in the mitochondrial genomes of virulent strains where the avirulent strain retained the likely ancestral sequence. The study suggests that short-read enabled comparative genomics is an effective way to both identify new S. nodorum effector candidates and to illuminate evolutionary processes in this species." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/6.cover-source.jpg" />
    <title>Resequencing and Comparative Genomics of Stagonospora nodorum: Sectional Gene Absence and Effector Discovery | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__DbqSGiVjr90FgBIuPlGEAiimgNNAH8S6qbiYFsi8JyQ__E2bjoC6LAI1j30ywBL_JgYRwIhvNl2n0yRyt_snD9IM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"q8MI1MYqgTqeIDtGNY5YvEdqtLG492pXQ4aEf11L-qk","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459473","apath":"\/ggg\/3\/6\/959.atom","pisa":"ggg;3\/6\/959","ac":{"\/ggg\/3\/6\/959.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/3\/6\/959.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;3\/6\/959"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;3\/6\/959"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/3\/6\/959.long":true,"\/content\/3\/6\/959":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022v24WCOoHEdpx0X+1DLb1O\\\\\\\/wXtReo\\\\\\\/LccrF2wn8VzhtXMFpnOLWV7GwRgp3rfZc5IlD9pG3ej4G7OOj0wQzB7Bb5sE51VZ94GxmEQ8ch6uTq1nh3+ZAnVGOzvepJ7+JSmex3a1hlHlVjh3fY2V96YnHCLwMzQ1tuWd\\\\\\\/C+fhQXAH68ciPkr7tLp1J2yE91YC+B3PGRtMBLzngg\\\\\\\/ueTygl6CNQw3Ub6NahJyR6y+cS57n6bEcUql4G9yqTCmdpb311vbr7TQXUklEM4JmmFqu4FGKMIfP+W5iTBcdx1kBUZ8LlI+a7jA1JHFx\\\\\\\/0qENVgd3E6LnkzFnT8Ec+9n1YJutFn6qSnYyBBJOxJXFpvwLQsyzZAqL6fv+1XpU9oyMqMLhiXuhBVZIRgzycXXGqKxWUk5+mRP19zZH4ll+crO0ElSQi89ADFGHezIEUP9\\\\\\\/zBMdHKa5Gbe21eAXRjl1301WTT56wbS1YhRP+8s7BeMVLWFuwNomUEXgahhqYgSYwqlMWjFuQ98T8iNW32ZzdqdNj87noFwdPAdSMfahuA+s6OS0xtVqmsSwP8KlIJK4eBEz72rZeH0fQIOtYeQkWWhl2J++AD++XN7higgBXZkgE4EOi4il6PDN2NLhnqc0yAarinnnI6VcD\\\\\\\/7pkrfpVIr16g6E17wqtK4fwL1wWLLTYzqWUSaviLSQXgwwp9xxdgnyn5MF7HpNGy45FFXAuzWowZcJtxyW31fpkFHUnpdVlPH2OWsA4wVW2\\\\\\\/4DCdGRyY\\\\\\\/HyYA8jZnThKzVXtjQ08ZOrDQgdX+v+7R4dQH9XHasXFaAdlI5b9Sr6vSLpmphW1WiAphoqTumlE3Yy4CcT+sh2FHszMdBHthbJVSxXutgoBs3MIgSY8\\\\\\\/q0xVF2Ziq1zrdYir6KfURCsi3xtBgbm8PTXdTwnxi6s9XmrOtGToS+NTKEdURzZV5uWrEGKdXT1Dp4qcZF9qL2rlbbSHni+PSOX2c1BGF0zvB\\\\\\\/i7ylErsgFuYAFAFjlYoYOvCL4hYOnfWf6fa80lYjAfbYQby+vccq09kxvM\\\\\\\/3jXTx9BNn3eldjtZo\\\\\\\/pvXBqAa78Cm9ZhI13Umb6hP57+bGgVdJqZBq1mf8FDQZnXpQYeoa\\\\\\\/WNgmmVAbESs9YQeDuvYguCor\\\\\\\/y3TjW+pe7S21p4u9jN5q96pd\\\\\\\/2aOWGokQi6oCj9s3rjRGiR5j2Bdx7EtMSGOkhC7ZPnl0oWHKvFgyFCwNmAf\\\\\\\/4FnZopfT5JsCeNJNEas7Wlr8MkP0Gkh+rJ2xrAkc2FpqH8uQ\\\u0022,\\\u0022iv\\\u0022:\\\u00223+mD\\\\\\\/KyvcO5vauhswrxO5A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022aab0382bb18759d0368b70f6730459aa10bc97e03932db6229df51a66b179894\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00229P99y1ORwCfskdxNgh92uMKZOQywKIDNfVKYIQpf1gr5kCkWkNL\\\\\\\/TY4YQv3xrQkWzRJSi+XvE7xIcD5VrDkzj5zMWjeoeBfkJY37AFZe00jAsGeI6AGe2NFhstQ1iKwY8V2mVHh4xya26b+zOoc6eqQaGT+k3JONXxOWRcIPB1J6Le2Ct3BXAIsccT2YjPxxre\\\\\\\/k+xuJFXsAFCVd\\\\\\\/fQppO3JAWXSBdTSF6EaKmE3+dtpVgeV4CCYuYxrI5c9dQOFvJ1ZxOUY6j+aCug56LtcmYB4MGok3qxOFf26fmwmagfo8ok+3NcIvylrDOyoVxPqTlITqxk4Q6gCS9uKxTZ4+YDbTjBxVNrFQQSMSO4jy8O3L75cAb8HSvssFIwxu1VOROzPD+u\\\\\\\/3Cy7xLaJH3PWSJw7fMjfOPZy8FEHPLAdn6xZmTbUQ9649eLmScKAo3GW3NpJtNZ2X148GGpeVmquFq2SrYRYCPc\\\\\\\/41eMuXZUz2qk634QFtOQtsdaP3pcz6U7JuJK\\\\\\\/FMJzXkUTWfyYFaErGBjd\\\\\\\/G9CEqI\\\\\\\/S7+QtaEIgPuHq\\\\\\\/oPjKrWra1xQOyvLqVd+Q1DFQKhUA79CYronfoPWqtqEl8EuYEa4OspWGA\\\\\\\/mHDv3ZL7C0Kj\\\\\\\/Rg3XP7clgPcRPsm6Nzn0Bxb3pbjM1AWANNhXjrVCEefNLWPTuhl7A5KahJYm15uVSV+ZnKp6aFGTG54y6uQW5KZ9RDp5x9ZXxs2RKlk1dgs43hK+ofcUcSFEB9xkcyrwmW9yj+5eCRzINcxPKbk4NUsF+gclf2PyW6e0xoh1rkfsdWzocAEN7jIn+oEGCR6TL9CxrqNXMYyWssjp4f4vcH2dSSuHRVKBqugRANWzjuAyQ+O0g6Q0e1bvYBR7WWhJ1EyAPYPr2MhiiJEVe8KPtmkjHVcDy+XcbWJDhMdtvAD4IOj8T40pzWMlbzGNCKkml6aSNwHVGm3Uxf9sFiCrffXyHbXnJy1aJt0Mq03nn\\\\\\\/XC1oAI1v2H+zmQ+FDLv\\\\\\\/0fVT5vLrNYIjYc1m3Y4LN+cYVhTpHUadQMTxRwSv0tvEy18MDTU44W+aFk4qBYMZbSz3t\\\\\\\/ZoXigK72fiQfwqd3sKYdFgGrwlzwaYTAd94cFv+vPcxfCI4zhB\\\\\\\/ly+UwfbUmgHUoIa78tCAFfuBM7oYpM1+qBvsKB24NArih+RImLEz8Y\\\\\\\/ToX1wgiO+E6U9TX38tR4U6FAkmqt07RSVwrh5w9h8W1ydhnZgPiOvOZSVC9E8ipvrbbD7YEvKgvKWQawWlR5JkWD2DsB0psNOCLdY5DeFk5nyojWpr4TmPFL\\\\\\\/lwBlMs6kI1eenRAch2AaoTf1U2kFkcEOnpTadAGZsP5xm+O8Po8QbU9xHMAyovQ3iMi5w9N8ecMCzqmzbcoORw\\\\\\\/XkLhyggLapvhR7NcBi55S9HkxHg02vj087v23HEBbYZZfUQUu6HhtlPs4iJmin\\\\\\\/vo0FSFCoY5uxV3yQ64l7W8gvfdP7JP0vhiZDC8Si3ducKvdECPiUXic03ga4xdMqOe\\\\\\\/2xfNLtGpAaAugrxuEgXSYzSe71YYgjbzMZgQxlno7\\\\\\\/XVjexF+zECHDQdRi+kSk5GSAjgS2PLy7BeRDmZrpPKrX+\\\\\\\/330e3s2ZaKgHI0BQHOAIsOf\\\\\\\/m55Jm7kWaf4qmOTWWhssSy8\\\\\\\/WkWVpGDhEIwlsQUyUrfEYEIN+JE0Y4\\\\\\\/IHCNln4djIdbX6uHhB3Q+22hvnu9rX8x3EjKoxBtdUcNFQhN8z9dWtZJ+9JpBiOMV0bRj+mQqOgGzLbYrejxHTpDA5fuDA3IvffXcWu0Y08kmBdN7qHPJ6rKXg2R8\\\\\\\/uddzF6wK09bUvxrMgWgXtDFzEtBI0cwi2gyei4CEUy5UfHFVw74slLrIfdbKgxq3PAPWTECovKNw0eqWabReoSzX15P10XVXvtg\\\\\\\/e7dJ5ytaRHzjpElVtPnsLyQK8CqC4pUmVxx7yffm0AZJaHqDctGHgCMCEecSNwGISL+hjk4Vyu1ourWlwwUDrKkjnPpVzWHoS\\\\\\\/PMjJeoRebMyLGS9PpiYcGhF5jSKSCtD3KBchKWcJ92+Qc4SGeu+bgyj3wpCXiI9eSVP+BzbggZ\\\\\\\/o+R3eb+jTFMVOEn7ZGUtSZIYv2l3ED5xDi7c7takXob5WmUKdSx3B+\\\\\\\/52RDrbYpCnCgsbt4dt2ogyBohU\\\\\\\/tuNhgIdiYrHnu9EBQw2sTeIVv2eiUYmLuTVVq3YqT\\\\\\\/g2Ny922I1rhiACXU0iafJt0Qj4XZBzbSARhysDcCyRTRQqsqxuZyIM4+sxOroDqEk9mFqbISx9dIRIB69cDbZ2Hooi266UFp1GfK\\\\\\\/fSa3CiKK3TU1S\\\\\\\/\\\\\\\/EmjaPqAv1gwBjcGDx+eUdcvQpvHpSLXZibWGqcFmoEJD+6fv6tH2K8jRrBkTLBh81arLIfIWENVi3+EnRxqgwG7gKOcDiZGXHeNslBbPBq3Htqnzmym1ATsD8oydrRMp583c1y2VI9rcFfbsw9rCYlg\\\\\\\/Uvwtgt59+fZhttM2EiXNXicqdt0qLqY78gENAM7UBtsfmkcDSlxM\\\\\\\/90DLoF+8wPqXB49O4c4zNzMWMF7ZTgsHaqvDxjExUpbLml2nI8BC2Co7f7YMWAHV9zhCSEjBp2Io9J01nYnSYzP3IutvGtLQe\\\\\\\/6ANR\\\\\\\/p+JLFu91KClFZIx\\\\\\\/BzRLDU29b0bKbJ0oXsPmOH9L+canJbGhuie\\\\\\\/hI+bHfu\\\\\\\/HXpWs0slzL9+2MEwF0\\\\\\\/lEW4QTTpRHlswN1S14cgKjIaPBShDf1l4hdHgXJ\\\\\\\/CBKB7uT9Uecm6xwK9m9cNP588dHtP2xG8a9ge3NkW1oUKcnlwMHUnTT7NQb9GdPJBOsm\\\\\\\/th48+NmIjlexYYsuMrW4MaXvC97NSJbVTOXJqUzkv+qMdMOYFwtOlTP5eZl6kc7uyBlv\\\\\\\/aWWS\\\\\\\/\\\\\\\/lNCYiYLYLcTZAUHMr7HjuIhxkrEjMynLSrMS86l6XTRV1HowonmSM+z+8rrBUnBKt2h4f9aVQu2sv0eAc9lN4XFi5FTMIo9c+eBQ652a0qWpKlO9IarVq64nhj9Nxrk2ZVKSRYr2bYro+v3j4k0toJkU3PrfRHZwqJ\\\\\\\/9IFAoSTjE9jpQTzTP9ZlswhNZjwFDr2b3p1HwDo+kVxuNCWfaW+hDOj2t7TBZSQa1H7LExVOFl6dDYiLt84BQBpyDGa0nkfAdvjdi+L79HdRcsWkG8lrmIQMmG80d2Vx\\\\\\\/BzetfiO99yhSyyXMpdhgNnzeVL3ZDpkzKJ1pfH6sMhJPEWZJuWtL8OBLUG\\\\\\\/vXJmk2b7ppnpCPDr3pmxs1UYEaRDWeUxKVFMdhD11ttPXlIGVakcxJofNou9hNHlmDRkzipI3OQ9lV1DBUiC8Wtcv14mGgfZtYB71LW\\\\\\\/QsY+slvI8nvfNFM\\\\\\\/5z8mSfH\\\\\\\/FKPc2xGvR5ly8IDBMiri+ummGGkzlI8k60T1q2\\\\\\\/SZ6TdKlXHqmiCuIR5IQqKaN1cgMAfOYBaEpIXaG0wKKeOe4e13OXdP6fJ2V4qz11schhwhUr7QFSiNtX0SD8YGldCPcL6X77skWa9opbuIxahGMmIqC8fcSELFCsk7AuspNKII3B9h7icxP4FA1Pln+5JaYKLukSARm3uD\\\\\\\/vcWPt3XKMm6H6QzkGzcE6nUBSroWlt1fcmrON5Br\\\\\\\/t8hRHjdNeOcjkO5kFnpAonLK4VNKVqa17gBljnLMXd667jkVEijCcnwoR\\\\\\\/vpyQ3+2SHHwDK9d86fuhsAlp9VpkgMGJP3ZkfrQ0GT6Yu7bfI1C4ZGOzCyKV8iFyNtC4bZFbAD\\\\\\\/3NxfcWs104fyyaXVB+n6WUqTxWPp5AbuZj1UTb3THrg95dsdU+80P7l4yOPPiuwZU+R1Srm25vLVTqGIDtVn720z41M0R8e4ad7p6GoO9+2i+Gv5k9ktLnN\\\\\\\/bMqaDscw9DK97MQaXbJ1vm7otol+0BxYnorgcyjo9cuN9sL5QoGoxk3EgE0h0nl9YrFYy4yYUqGw20qucAVyE29ZcMrVPlxe2tuoTgqGqnRTdqyABV8LJgCOyEnEfKc+k0XsnTDe4Vq40b8E0eQ14F38n4qIANBahA5gqtxOMJHjmp+ZbsV15IphWmDZ7tlDkzVqszJv\\\\\\\/KTmOt92LnF9GioU58Jy1BoKMgYqITqzrfZE+L3r\\\\\\\/tpmhaeW3Oi3GUibG1sRgZBX9fVXKdEiQpnNZ2dvmotFpjZ9xK0NcLzwLw63l4bd3L\\\\\\\/C6bIoVuCmPh+lyPu+XUpdjdUKtBa+QM5lNBy3EAfWloi056B0l6QTm5gDlkpvcN6Ha6ERigE\\\\\\\/xa84nA+tzSSfO2+HArqXsn\\\\\\\/4u4UXP7iVlWa8if708a5mRcmwTHvu1DBpPhxY5jb7RE94bTMksijg7Y9V+42370c96ZmaC0iXX8o7MMDbxR+hGqMq3\\\\\\\/AQy67nZcy2zbjJA3mnqFFhL5Vd6a9pE4AzqUU2OUNM7MT4OliKVxX9S3yC\\\\\\\/Urpgm\\\\\\\/9krtvfqPM0fCwhIpUV3DtR6aqbeJqPQXQKaSriIaVdNBr2YcjdePxhRm6W\\\\\\\/vuussQ2MZWJ5FxJP2no2AsVsmq5xfHgwS\\\\\\\/hXyAhtpQvSTWet9CGSgWzVB\\\\\\\/ZDs6JMH+JLy\\\\\\\/MOaswfOozjPin19o6GbBTpJfj9sjst5xYECHtZMdOaGvMdq14iAWIqgINCVR41oYepHX086jdMHQOBi+QoJQnLV67tqS1z78RcL2PxxvRtmFHwXdIelbEDf7CHPC1lq0\\\\\\\/fAS4LOOrzEyIiGmttrVlFs6tfAVO8MC1vH2cww3\\\\\\\/kaLhs9ERLkNjTpwPALuWTVWcJBsUwfrdv5TZ\\\\\\\/6zzvSDKkz2k5SMFoZvQDPUy8yPDWS7XSOSGzeLNW1Nlj+2YgJSVvFf2HFVRN0oAv7WtlPNMUUtxxX8OtV3f7HPUz50bQEWcmZ2CpCujujsX8XnccM2CU5s9vptW7AsvdeXFO4mUmHOOE0UQRd3VO5BES\\\\\\\/CdNP1lMceTkV9ZzZbikaRaisEYW5\\\\\\\/kCL6HsTN7hBfb+Y2Pwta1LJn3TLOL0Awtsv\\\\\\\/RGPQp7jXGP1iuqtXiv87HxVaZxaW9bH6++DES55PHjBJpUrzs51EaI6NT4fhF7s4Tv\\\\\\\/+EumfcpRN9pZ0HCmqEA2svOcRyx10zHEWzyx7Aiv47e9DdQqnT3gTDG9g3lQ2ySI+t4vGMSRFNGYhX7sw0OFrpfcj70iyg2ChuVi5W6Ur0z+NYmZ4EkhnlATnaR3xvLOXp3MRenPcyQlufRiS6P5XDSH9ANK+0OAScdxS\\\\\\\/B4C+VPflVaWNbeDWJL+tLF3eqdsgMlF4zOE1g8ddLEeRPh1+qZZ4Ikvm8RC7P83ResxYyix2P8WwETOVJAFA59do0WGzqzkUANBX42dydgbGlHypgxh3cOxBiL+2hukcWaWnvQ5jh0LQ2K\\\\\\\/\\\\\\\/S4\\\\\\\/h0jTUk71\\\\\\\/55v2Xpp1foGi2hPFr\\\\\\\/GewQqANvU62q4809klaWr\\\\\\\/4e8SktYQn1vkjviq\\\\\\\/QUpxYOLZfx13KPkyV8\\\\\\\/72ObPPoC+bN8kacLMQncqZkvX83C\\\\\\\/IcN27gFThaynJXjiPDDRikk2IpK3ME4gwjI0DmjOJ2oEVqbRwcaBh4MorppzOEcrbxH8luWPQpaSRIiBa953KzalMJWP1J7huhUIbGL8brzDS5EOxEYUXS7kH6fJ9FtXm6qsQlvKBRoAc\\\\\\\/a2VVWzkWd5t\\\\\\\/KIuGx\\\\\\\/duKBx5h6YpWE9kE118ysd+WXx95qgzinlxWXDmAhaj2nOvU\\\\\\\/KPEVU9\\\\\\\/z\\\\\\\/BcJXC0enLjoinTBF4ylHDww8NbxxFaeq\\\\\\\/09ttI9vaLnmvZAl27JKqFjbxxJbqQ8gl\\\\\\\/Rw5d\\\\\\\/Ke1fxfVC+WZ\\\\\\\/7\\\\\\\/q5L7Qp7kk9LcDSZekkwB5ChhDK1MjRx\\\\\\\/qQWQLTCD6h6EBqTSyud13C6JUgwzCcL9Jsto4w3mns0eUUtlKPBkJUOL+TdzoJUn6VQaI+43bgeqv79zg5G12epfuVzScslIU4mIZfHzlgGhX3yRHw+z4iiGkFZBRGpR7r58z9jSx6MbIZ+gr3Zk4EfLbjwug8YJj\\\\\\\/jwU4Vw8hsiAkTycrFYR5qrpXKTOrXQQWgM4Nk3yS00HSCN+29Rt35ExBIA8Fg0IsCGCg0iQ4ncNzqTYqVtrfiGDQwd01bi9EbH4m5qaADuT9GPWRo5Y0QKGrT0kEwZgXUUxhg43Uunl+VSDqKS4TEwWmfLDi18m8GoNvBzgyu6+GFlv5oiwMsB1sX9+7yzQFZcg6uxxVvZL9UocqCluL3CRCwMrvQ6nX1hf9WxsOcwTIsrmOhw7uwq8IgGU1IzaqoUTElkeZqWTrMdn3LgO\\\\\\\/HuuPDyYBJxbubFfJN4MO43dVwFPkk6f8c3f2tm9UeLHBsTWNMFLsYY4FAHFZPt2H8omd672hVjQrISMlgIMKDkERsPFxZp6WWcU6QXJjXC9r3Do7jMzRDozP37f3xLKQw1TTnSEOW+RvDOzo97GAHTf4NgkiJW0aeTalq00BiUGkbMIiQgo6VXGpi7j8Tfc351VwezSofB5wwqOtnekR2ns8fdaEyogd6k5PnEdKd5W1VdSuTha+12Qh1E0vA0dElSGo0PCHN9XwW8WHwVotG2LI7KcCYwxusYczVcfzjDHGJBAy0TAbYOCOA5oj\\\\\\\/vEshgwIIvrkx3XPe48QzHXMimdaOddr1OOPJT3xkIP0l9Vx+ALc7vXKv6d8976+\\\\\\\/X8C2IuYWHpyXw5qFNu78ZgaoJghPf1bSQMhft6RViAURLDkH\\\\\\\/AO782nGdNiKfnN0hGPSILS5zhxi4xkmXvp\\\\\\\/UErccod58Uf218MG8h6HLbipnGycuizdUiGdTuBUvkY2m1UY9LsW8+m\\\\\\\/qqFV\\\\\\\/fAnmXfdGFM0WY84eSl92DpNRr5fDKTHLBijVllPOGQxp9Tj+v3iD0ngyyuKNg6Ep4zG8Sce7JM\\\\\\\/TG3xvUc8Gm0rF4ZypNeGn1J0clbRyV3e6qy1mHB\\\\\\\/2eoIreU4gU8eVN8uK\\\\\\\/VYjGQgeE010XQ5sbeQxj7mR8+sBZ0ym572sg3\\\\\\\/yTuidfHhmJXm41l0fGYBHGCfCGpK4P+Fmcipd2eSfgWa1syUqRIm1X0P33rGAvFBjZvt9YqIPk\\\\\\\/bdCeLDM29YcNiKs7T8yVUQ7Jueo1XMjj1y0sQiDUwC7DZi8Ee8yp\\\\\\\/lfGkXOKHH96YfgRMxX1K6Y6liIms5nHTtIrgRuRlc6gyYmD4P7K+pfS4tg+2oAIAEDEgpx27oA5P3m++y4sa8DxB75AYYWFEBAKwNtUi7Q4WFWna5lebxhgV2OxE+amr6gJa3C9WmdnNP3xwW4RoutqQBPQ52yFn6rRnGnR6\\\\\\\/1sLnpx1vOTBS3y4\\\\\\\/tlbWejPBuSZw8b\\\\\\\/Jby7GGZQUuOxT9LKy\\\\\\\/mQvpAfvcSbJZaPKXLJEJa9BchqVT56aKJwGuZiKJQRGxiFqNt\\\\\\\/cGjKlf8oks2mDz9azk6tiHY8os3oQ8+OljW3CtiNGDRDyf1mftiZMmn9RGfspplMTXW2mxIdbd5iTJ5yStqfnwgUMI9duLnhFbH7f9Fx5rI2+ODlL1d5ey4c020s2U\\\\\\\/kXBaqe7hYSPfONwxDBHrw+AoGHlnsNOVXLvyD9XmbbUSU+D6Uxi19tTHcoe9csSoN\\\\\\\/IaG\\\\\\\/w7q0NxVulWiTN\\\\\\\/vh5jCVvyY0+wlzCA6nlcNGBC9dnliSdYt4sa4Pq4Hq++Pn+cEz7cdpW87V2FaSz8ov+F4K0MMcOgIzQMPNpNFDrG\\\\\\\/lphNMtB4QPpKx4i5WnLpONqtiA\\\\\\\/Uwyz0FA7RcNhxU6Ay8oAjm13yAlpivhnyiDagQZYgW8LypBptD\\\\\\\/QHAUtv0R20MZIvwhtXJcCyK+elqWpoZ1cp1Im0j9NQxNFk\\\\\\\/tOUjXzD3SCTxOsgbWZBMQxc5vXGg9Ah7UTYL2j5WezRFAu5JZCaBw2OHziIWpMw5IOO\\\\\\\/TFQ0bhFV2GCU3Dn8\\\\\\\/QqRfcrBE4NBm+JWck42rgmqsw5qpykIuUrNKW5YpxHtKvjxvIln9Y9CHiPObBDG0P9wokHhSJNiHSEXhniHSpuCdcnEbNUBsx68luZJ3blpgDAo8HHkud+LjURLF\\\\\\\/KSpadIkQy1Afkd8Nv3Qo9oBIDgnmPalcYTlMiF9c5lK3TjbtbHchEOyt8eAq\\\\\\\/onOddMXL1ono6jtIzJtZtkUgAW2Q3E9GXUAGJwfwIu7nqUep\\\\\\\/k1fgXd8wC\\\\\\\/yOgl+\\\\\\\/u0dioyFUi0884sS28W383jDxQIT2HPlPgeQXzRGPttMXbuzKXltenuZAoEcVUN8dJePTzvwF0XkfhjPuwweEWfs3SN0O6r8BVmGIGIa6kWVK+LNIdxxxYDGLuKRgsdwO2Fvhy8+TiIybl1fzGiZsO0+71+FUneie\\\\\\\/41FgAWxZCEqJV47tdh7l+bTSYrsXSmnS5XqZGj67T0ltJ4EbLkfvxo0cxISzCAKffdqp2iA6pEd3cpeH\\\\\\\/4vqaHJOSp9RO+zXZGbNL4O8ftri9zldnPvQaCGxSFPirtYeTgfxUSOuWQZ5npqW+0\\\\\\\/JPKu6ev1Ggx6blflyxbCvR95WfRGIjJwii4ifSyjm4xw7lPGgSSQ3Nm2lVemxi2iZ3n4xTobMNRmEzs4K\\\\\\\/3E6safqji0q7dJhrd18P49JCvIYqX+b5HwaTVqxzlp\\\\\\\/DeDuzv3f1QZy2T4aOzLx86x9B+wQU3l\\\\\\\/bln39k6VlsJQNsBY2mZJV+cSbn\\\\\\\/CRvmNe5\\\\\\\/zTMD3Amhtpx1N6Ss+Ic3jxZqR\\\\\\\/IzVIG1HMi0gJUhL35Kgfi8bgBP6m0ejxYuZTV8z4sY5NkEqnrptlsVjyWCwObch2XrPRvWka94Jq6iQU4eq3TWKo2DshoF9hjHHtOWmNSPXIiAjA\\\\\\\/15MvsMLhA8YPx5rcE4ojrJxbbGKZDM8PlVgLjPras1nc0Vmqm0Iyyp1xyX+NZQqh3eelpdd\\\\\\\/H1Xp+uqnpmISnUj\\\\\\\/DSUfeyxyCWP+tIf9Y5mtUJ9RR+k7Y2ypsf49RWg7FFyvkWnn8+2QGBdyvaKW\\\\\\\/sUID5957e0bL1hEo8kOHMenchTSl7pt4orxXNRZi0hU6UPJjXD\\\\\\\/6MGOXuUky7w+ezqNMSo+xb7SxU0txMqvYgTSSZ8xqEr6QN5M942u8Og3qruI2k14203KvZjyJ1wZoBqu4\\\\\\\/\\\\\\\/z1LvRVYcZTPxZZyJHShI7vI9Zted5PLM1MliaD9gqwjCRRDIYV4Qw8ozpW9IdIu3K+o7+CohJBeK33rJESby+L2IOvSSAtr\\\\\\\/5IjTKTzQ++DEzLpzMEMARn4m8euRyPNXYIbiVnGRw3lVFLsuIgrGnORnkJn77A67juyHYjCEwB+dq6PLGKfmXor2Ig9NRCL+25QCEqYbKiogAxFiujOgp3cgM2+YcAOn4aJsKfgM+7FUwXYmZy65pVSf8ukPJha1T9P0c3OH3C\\\\\\\/vTbSI5TX44asFunD7Nb3cDaK5mvruAGbsE4QZ7jGflk7pKHvvQCOSPn9An7pGfJumEu\\\\\\\/Nvazpf3tz48cH+6TdCfE6hDVT6ppZ2CfxPOzgjDvyS6UfelWtTFiZqEoukpjNFY77NXC7Xu0av4MoX0qGhiARPlv6feqd496frQoXYE4B0dneO2TYS0Ly3cXLQ6JJWdDkzrgy5J\\\\\\\/5iqFLb\\\\\\\/XVftfUt6tUkFxcBFmuvxOMGZPIYZxqiJGK38O\\\\\\\/vvF5zVskJ1+7J3mrwlNTW9xA\\\\\\\/x4jV1NZ1cwxDyjspFg0LmLm\\\\\\\/V3q5JUpYx63b9Od1K8N1pRDwsyYYPC477vH5rKYDY3kwMPez0ruc2BD7YJxNx+R1gsuwsFrmhc4P2GhzDEC1m1f2vKex02I1taUNMT4g85grdlPfDD6\\\\\\\/OFNU6hMOGgowEZzBnSickSiaH2htchRXxPWHE6hsX7lRU2vfszMulDFY2GzYEfRjLAbtCaMUlN6bZGlhojgO0GtQMH2K3wXE4iHAo3SqE2lLhjJzrqhw8aDZ8+A1YMgZ7ftvC7SNRMPtmGnPMP\\\\\\\/KI\\\\\\\/y40aKAZeX7OCOIhPAgB+m0ClEYLNVJRIS4rsb9dwh3mKOgc6ZO4Od1MsbzrRSWURv57hIP5uL2algJnvOtQApkJ4TuROwCu7IV7UR+P3TCgVujEFfFO\\\\\\\/w29I6YQivYyyHmlCY47T9BYbvDhGn7TNxd3IgdriySvY2n2WhYYbMLEs1XnFCojbHlL5tQKTbvorAQINZgdCIQOH\\\\\\\/r4GFl4YCnGlK1FsoFNRnCZoDasmtzvvc5Mykksq8x\\\\\\\/I9l2S2COdsWIIWQrVtRsmZvsAfdhjMuWXyP9+1pRl2DOB9UXPOM0ffZMTmERWCM0JPEw3BYpMV2hCSsnSAqqO4V6F1zxRlzemdZj1\\\\\\\/CRpEEe2Irn17UtYhgYN6c9H89M8FM+tnqeRbWJPcGRI3kNPeaGYNVjD1vIWbkcxej9lKWh2sK+S2jHaJJiG65fwOcGtY1uDN7vRsFfRGS1qcpPgls1QlUYDaJSS9NAVjnlnVPutjnpdv5QFgtytttxylqTJY\\\\\\\/7BcjUHqtyVhl4MCUTsBCieQXP3XmOWzGQImVPpjNf3ZMP7F2QxO3p+AhJDQ2f86wxUx7RBcNf\\\\\\\/seR6Ieci07sgBzR4Wf6F79TkDCJ5qZUIDfi4Vs4EFxkeIZB4x6uBg7pQrdgsp60xjnzL\\\\\\\/4FjHFGJJhhgdaHJMotFllochR+juX6MbK5eGOUgyPW\\\\\\\/grpo5magStXtFhnd23jLuFuzTrKQs31Sfh9VMGrG9H0MSq4dv1czcvDswaLWPEAmfX1xlHSQN\\\\\\\/mm8XDE4SHnjOBjGmV\\\\\\\/\\\\\\\/63BbkOvL+bIzsIcgofrhs+NXNEp4F\\\\\\\/jo2YYgKcwD+qD6eihDQlG4ZxdhOZw1i8MZQ0HpThlH9gABymgiTrzDHMyreX5jY0zcWuiQy19kN2v+PLtHrz0ggOwpi4vZX3p3ec+GMR7xqssGfDUf3Z3oj6A6NQoJuxLYlo2bTu6aBj0KDSwIonfSdxOzUz6ZW1S3\\\\\\\/ZJ+lcPwW4bSfQnWQH9f0i2crTPFvcyVfqk5WXysmZJjk5UlM2B6xsYOVdl4T0yS4vfh+eWXhihGEzN7Sk6IKO6EmbMrdZD2EDirLujoreRH7SeI0xqFCyj51bBA1vWOAeGh\\\\\\\/NJ084Hfwx+hhUWoBO00sBY6eQTzcqgrFdKggBHxDS2B\\\\\\\/ocrzbpMYQn9KjSyTCIrX+V+An+DMiZVX6hovBWOlleCkI4eTU4zyJ6Gr+u5Tpj0q7bFxfFwx9Gply\\\\\\\/lstJ7QtdPGO7KvTrjqQzbf4Pu6aYNz9scuhmT2io15e+fEhIMpW4kL3aUwJk0PrLxAxjQh9e0Lv3j2zVg93ycIxPa6A8EeNFjIQzNYWhSCuVgtlwziCunQrdJRzwID+DtRFNg83HJ5SpxFYJJx27su6UctDoE82kSW0jF\\\\\\\/UFWJx0tB5jj8zLd0dip8HjuEBMNHVmAyHIIE\\\\\\\/YMaWnehYDuMIgKQQ1QCLxscrwEdd9W8f\\\\\\\/jHjbC2izZqavWNBrkIuLEKeCZ5P5A9ya1tz0WECUAFZCCyJvQsJKLWchtm8tpgaU91F9T1FWsLP+wYxyN8OKP2o0gmN9aH8KimUuf3+JR3geq\\\\\\\/\\\\\\\/FVB\\\\\\\/n3SKL7LKTt9frnsRaf1neHhLGckTpOpK9s1QuAiocWK5Bfzk9Nzn57xVEq0wO5qITeSw1N5LKRYcemKT10dTqzDQbUT3XLQazSWU4X1OahdQUOgeKqafZavUspmG0AwT6GsolZiWxBJCOFOXeiZGUp4Jjt8croE6HWxCXKlkU4WydtBmfcEMhfWQSVcZwX8FDAmIN460cHXKm3XqBxnTdcl81M4IMlaz16kNicuecsgUBWZ9KIOeKXBdWciBhqhQ+tryD2T3DlRA8lVNsdEaMImEXV5IAZLhl8EgIpIpKGpBIjFzuu4JbJyuNFAVzgFcf35xGF\\\\\\\/obSqaEsNuy\\\\\\\/KJeQNTHOODz4cCEXGNM+D6NPSmjmnNJSVVkfC+yKYIHs0TjxP18YzuOzf5kqh+kY5XR0gYJQXS5nmHcEs7dBoGSp3m4DWxVb79BG4k3j5yVNT+quYRClRqmIcYIqcUUf7ma881s7q7mdh9t7T0luLbfuNpAIpLaYv7UE2J\\\\\\\/yD5rs78jokdSOS\\\\\\\/H6Iy\\\\\\\/EfrOedqAST9HP0hD11X06ghrBVXnVhjIx0++1GGzJ69nkUEVvZnxr9pa7HEBlGob0jrKEdIyPP9YfVGPHtB07lWManGzxEuD4zdxAqVDLJAO7LQut9ipneQhsjKE3vd4hsavWvtn+vl3VFvKFjJkU\\\\\\\/dzOufI8OFYs2Eu5gFhq3BTV6rmtGT+Zd3N07sOrL47dlrFjATXURWA6+XCnHI84bzPjvHs8b472ugQPWWZewmIPb09NjdBYOwUxwv7kUETJWgLTqjbLFQml6T6\\\\\\\/QDGFhbouswjYgZtH3LMX\\\\\\\/6AxgOl6fQW9Gcc\\\\\\\/lauE+xfWEPrVoBhEmPhEskuPw3d\\\\\\\/nLcfrEi1sMnYZBZWbvmbMXSjjb2Sb0wqegNP\\\\\\\/ATZMBqXpiUo6sHYAJH5aeJ1KP5lhnRw\\\\\\\/gPDjioAnD39cxEABEW5So3jKO1H6JIPexlN075bA2H7KrI\\\\\\\/F\\\\\\\/1hJu3o4xBIF7B3kKfF08sPE8ioXFlAJfWNCoF470j87iL6McnlTpHOo05MbIXeCfj47r+0OAAyE7Rvk1dX5Xn24vFJYoY4fsxBoP\\\\\\\/Mi3Rv896HBYYxzcctrikf8KbM75SmoEpnMJsb1VK0osiMM3z9kusy7Z+1mp6v3Vx2VyUvih46O1pAMQwPCqoVMrqSt2KV37CNPK9vWjv365uAeBcxGkJoPEVp48j8N7ijtbiN3Zp9RVNRy5vqGG+rhZoNQl46kVVOe1F9iHuRC0pdIUbvvTSesVPq0DV3qGyLXb8IF3Vpl7ExduI5TZwvkkWhtNN8i+z7\\\\\\\/SchpSuYe+uTaDgzvlsfToyLZ5RFB6bZAr5pDx4+D36786C4\\\\\\\/Rzj0jBVNr81ITIZAhxgZ+NiYMfFsqaZMBe+BKZMQV+Qi28d6b7U6d2SjFc4OylCOt2P03geemLra3AdDzjBopJs16SMtC2O7KHCB7YZMEd6OA3bnyUQpL0nqcBMe6cIFyeBz2p4Byx4yGpa2gmH47jSG93Yy2f9kZLQSobrbQNLYihuJYCy5ArlbCR5oYv5MFck6rJg9Ef8P9uRhN9boHujU7YP+IJ2B5AkgeOpxY3UG2AS29jOZb7nQw944WxDWUotMClItrJAeQg5i4uFkh8ggQvxP7AbJ9EaNcOJl\\\\\\\/hUE4ywyWxr4blKqQzEMPr59azdL1q\\\\\\\/2I9UqdIK7D8eVfu33oLU1hO0gu4xOVJ0sMvDm6V7ExOvBz6iLM1WYa+CFH4duRgQhCsMmi1m3sDsmeyKKPLpn30zykHFSE0NBoUvyAidr9tvz5UM51oXxSkrl5q9rhq9rFCmvrDVRG9cdX+xuBbKBppH1kMbeZpyBZ9IzMXkoK9C+Kd0L01GjWVcVN7o14sMZjV45iKAbEX5vMq\\\\\\\/DJTVs0\\\\\\\/MmMRl+Z7kgrsBUjxaWQW4fL\\\\\\\/FVqUdGGc0jBO8oDtQAjIhTb9Y6zflr1C6Rzq9jTMn9++DU0nBUizrD1P09Ra4JliNDmmpOshxnyO6qiHPepN5SlT6YLxPFqVrJ+YZLp+wc+rtkFI0hPmfOqgMi1uNfSwLr7uCUszHgZa8nCFmr+83tVBtkaKdON0VEXN8A7owk2eg7UsQQY+L0W5o+1JMOfTPSw33thCe3+2qwpXL+IzY\\\\\\\/tu1hnv9PWJTwfRiVEO+d+1S9oOD5nrgvwA4mG+sFAyMqu0SNsbFK0ihoklUEXSFZvilqOJf8\\\\\\\/Jc65dPIAj4AwJNWBTZ\\\\\\\/kRJfHQSARXJD91a7ntYYvCxhN4DJRBwkQ1oU98Z+szIN8ro3VaXn0TVeO8NBG+ItcVFNhdoxzUvBzJM34a48j8hm9rIFE+B1JRMwpVttE2Bua\\\\\\\/nZMPGfh4qe\\\\\\\/o4GUnL5D1TIylD5\\\\\\\/MhmZfIVeh6nw8Pp0+sGAAzgHAZY+XBevfodunvfQ\\\\\\\/HORMYfMVWgnbcdf+NOsdDN6gFLe69nbDQ6HrfSiGcVcFVn2VnfkK8YSAbNu0zcsN1Ht8UNrTKIVCKU+gO5OkD8Io\\\\\\\/tqObCKZVFG4GiXZRdeKrjAkYqtv0Hehh6Lqf9CvCeo0t0LoB+1h83xPNxI\\\\\\\/w7Aoeb\\\\\\\/JWgfX\\\\\\\/L7uJo+QVYDZsUrMDjiouvzw58pr2Y1vVQYWhBni3sHV2J0I+YySw\\\\\\\/zwD\\\\\\\/l+rNXXv9IASw1B7EMKpBrAnNEh9jpJs4jPXe02HeG2HoKiiMf+sm3SZRkWgh2pQEQKV3ut9PkViJcfdCjKfPyt3dakS29h+5zmROoauSoP6IJFs0qaCMn9APc2KxAfEaexmIDV1DoLCarBOjctmsutbqupfipbqB99ZalmEqZ9574EdgO8ynOv0idUfbRsQvwU6Ogac0GVMjLXbUukCo\\\\\\\/oPQvRQ37+kTOO6g1R+QKRnXhixpK4d3+8mIUKDFFWFCJqHY7hZ3Tly\\\\\\\/LC+k7LiQSwrRhmGHBGCHIDRIu4106RvkhCxBo5ABzmqJCcuKDZjijMAJ4BJMoeWWRWRgJn6VC81eZqkK52z\\\\\\\/8e36XeIGKqbjUJzknv\\\\\\\/5U4oTFihLyDYbQmskOLTK+jAavHWWyeX18V7oYMKtN8fAlbgZyxtJg6M690oUq8T6B06dB4ppiHXGe48aI3egJ4eaf1jnY74KRkwmmj4yYXH1yxtgo6kHl3evIRsLRIEEmCFLVnvWZI\\\\\\\/Azvbmi+2jjQr1N0l+8Idf5\\\\\\\/oZa319UW39BC2TmrANFG2m7WyGII5V+7F96pTH0Ow7j501nS7PalcAFnZfCOYQuFkJV08Hhx1\\\\\\\/CZ2K4frZxBZ9UMDTNf60dDeM+rX\\\\\\\/F5hy6NZPRC+7IUG5xfJnNz4Y7Zb3z8m3X7rsN2h6TN\\\\\\\/fU+iHxubPF+hkRAXO3IUDlSXdBClqpjhh3+0k63V3AF9SdpB2sMqapn+CqGXYKG5cU4Db4L+YnYEvaeScGW6N1CfrwXY\\\\\\\/9WsoIWNRRhIK4l2M7oQu871tZkCCoNzINYd7Ek7Oumt4T2z2OW0LMqeMIYI\\\\\\\/f+FSma8zPOx+GsNFKEpYfvOzrMLwhz63KucLEwaPrYcftRWoX1JsKuZ3lz9yv3sCXZb3ME2HsOwEtV90owiB8IS5eL29OwDKvYoy4DE\\\\\\\/QLfuX5oB9YJGzfHh3pqwpju6vtzHxo8TUK2vuslsh0jLMNzSNpCBe6iUXAZ69TvxfeYqZ7Dhes8ISUjbVT2EKCljnZG8kyjFWkrUrizKcJZe2if4pHUgmX88BdKthgFQVCl0Iz0uvAdGHAW\\\\\\\/CVaMfgm2YDTyMkszh7oBPvJEdGpnVNJ13J7Ib0coJxvkp2Grr\\\\\\\/vxy4M\\\\\\\/5D\\\\\\\/wTVRNIw9Rwif5HrJiMUjXk6hznKII6Qg1kJjPFvTlt2JD5EztRzK\\\\\\\/xYrywIXXO3Rmo8gGpAO+R2cpqpURFygEg2ddvbPJi8UqEp\\\\\\\/ASUl0eeJzpOkW8Age\\\\\\\/ChIU2f5cl\\\\\\\/VK9zYH8HqyAL84VKMu34uR\\\\\\\/COO7w82lZeXCS6Z2+PgLKH2NapfTvwVqutS9xIJRuApxg0oO7xtRjIM1\\\\\\\/h55eCirziWplH2bMCKj+Es9Ui+lt0vtNdTLpTmstIQlsoE36dQUsUyryYZhmH\\\\\\\/Wl0aqpBq1C4Q6Z\\\\\\\/BfVlwoksW5pVG3R\\\\\\\/wbmjxgSyhTrbOa6OJsR2wdFEn8rSMMQNfmuaWQRpSoWPCLMdkrUVPNZ5\\\\\\\/tN0\\\\\\\/m55fK8mFq+SCIhzXIviTkLWBpx6q3CyJ\\\\\\\/BwPmDP3ZRE59H7OX2nAnQ7JFst7FKfNrUUAJEztEZGJj5QJMGiq\\\\\\\/KEXcZLkt7rtWzajSp9UkIM+w9P3aZz5vpVvnAhnlcGBrvHx1dXhryJ1Wyp1mBBItHkpLsmSDz8vRwZQsiXxkNAuPlXFBuR\\\\\\\/vGfRBbaHEZhDjKtZ4lUILrCO4TlecU9fv+eOH65fZpO+jOurfrv7So+b8aPhgV\\\\\\\/5Y2b\\\\\\\/w9PzSpeM2xv3N6OWwQJYYZYXC6QXVZUVyH7QtkAM0HTOITgcJC7rMxiF9Rbeuqi0df9isOBGjH2bGEmtB\\\\\\\/SOG1mkC6x5z9yWpIxv8X+xD8chGiY257EOUTLoWAGslm1bQrAjR612xCQmWUeb1DA4oG06RJ5K2DU4XWK4SEQSxi0kN4R0WqEREr4uW1nzZYca0C11jEG\\\\\\\/PPlig1uDH33loEJp2zstoHHk+kcbPbYuIYcdoZskdQejupZgu6TCdFa03HxilMHCCMbFzFJN4H\\\\\\\/iUWEeJ4aErY5l2+mEz\\\\\\\/V+JIcD+ohf9LNbjU0nStsNSdNW94IkjluhJyn1fMmKD66mXox8ljRYi2D63PIy+rGRZ03iD1\\\\\\\/AOOtHJnRy\\\\\\\/QHURjTg5nE+VQUlgu3ZZaPLDdv0+H6lKApcawwxvK4kbOtf3danUpR5Xu2gyMBmzZWSdmFj8vqhGx\\\\\\\/4RknBzLhfNunFclZC2LTdbZFu3woWE5LE6m5\\\\\\\/ZxV2PbkQTM2zydam\\\\\\\/c9obpwdMsqZbvfA\\\\\\\/3BRxYUMa4wy6+2wlV8XV2kNkXuQGj7dcQOe6BLwJ1LhzrKMJbIcWZLHKqWlLY+Pg6ACaVT9FYg1+gODq6ZvIUL4Q0uBYKCszdVanEFI4\\\\\\\/7o9L2wTxO2j1RKAzNUU5pyKALBWou0VsanDqZw2mzHIfvY454HkMCaMS8GBPr642BRbF+dzG05db9tm4Wr9rEEO2b9pcVv4xpEFBlYbQMzpYlLViG6TiQSQAPLXrgO8yUt\\\\\\\/7UtvZa\\\\\\\/yplRqTnVpWateYhZJGzxlJAXc\\\\\\\/35Mw1R\\\\\\\/HYMcbkiaOfVrAaB9Dk8HMjKIfuEY9zBhlZUfG+5CP7G3qai9WPxQOziTMXXMwc5l7C3VKWidfoWOaWwGAvwfZJqVe\\\\\\\/XTcwjnTOqQMqiFnFXsyjqhtWV+x02fQBS5KNNNgFe7ReqJHYkgAr6qVYjDvCydlGrXGe3p7uRY5bBuSK79k+QaY\\\\\\\/Tb6cWNm698xpivEAN61m5\\\\\\\/Tez2rHaP+H\\\\\\\/QTKDAbfPL9vTtFM\\\\\\\/KSom8eTd+xIhZjbe3I3VdWMkkr2hlNjCQvDseo5mm2\\\\\\\/O+LWl46mV638xQLAHVlJgpCteq1tFv8QEBkoQ8nBoBU\\\\\\\/KReGVAXqAyGf3kZn8YAqwaIL4Os1pXZbnuF1Phy84mTqfTu1sYirqDibJdwiNgZUlQL3X9kqFwJW+zzdqttICKV8JVr0pBxOVxCViJMruEHyhqnGCDDJ69enwf9a0vjfS0HvvdtWVviH0EI4ivheXuGPTZ+KANzqLOFkmQm55LvoObITdLfVVw42QZ\\\\\\\/xpNRvwgE0dN2mzL+RA\\\\\\\/tjE34\\\\\\\/GiGfrDOL9o8d9qcuv8Wui3J3BB1Qtne27aZCysropudOuFIXLGTu5GRDW8ZNeAEjQ3weyclKdrZsvcu\\\\\\\/S4iKGkGzd0HMuSKnSQLnFt2WCHI57YH319efebVx8ES9wDFhIzQuATHdAh6PCLATS2Pg4zDFqo3fUXa16ZD7GqTESz2jzHmllHmIYJArgGAkHjPXoUYgW03E07qJU+3hw0bsy4DqYkTAL\\\\\\\/ctAxsK4sSmJ9VyG9iXhZyQhDCtgTp1R0x7C8eGNd5iKtpjRq\\\\\\\/q8CpkU8hjcVwVqK20jVg8xVvf3w0xkB\\\\\\\/iH\\\\\\\/3m7vF\\\\\\\/XTMhoTfCHMCXAPllE5zZIHYSd0uwTerdjEzx2Uk74iqKN6ygqnz7CWCUCsjbWEl3+lc6z8pAZvAXnuS\\\\\\\/fsWXooRebffFGxl5+RY0YqkT7PFZm7\\\\\\\/2JdXQ+x\\\\\\\/m1oMdp0mnPQEQYWBqWsvVdZTHSrf1aZYz71IhB3fxyjmcH78gUQaymsACrPK3IjH2B+3eaXmqg0znoy+OVrHcpSUPGAJIh\\\\\\\/41gU+e4tGtXKS0PtYAp9KyjhIJPDEwBQXUdNQEJm+5t1Cx7s0AN8QTNJmsQjUC0BYlMSNiYhdOKHHKg7TQqYpcpubxvUo95rxTSRXkBeCgMYowkuxE2qnZ8hZh9xpQCyWYiOEgKT3k2YMmOBlpS+Cx8T\\\\\\\/zhsR8QS5WtHufZB1+UmhUw0gxL1V\\\\\\\/\\\\\\\/iMphuDqQTfkDyvxjSPJnel500ZU38GviJcpTXG2aFbekzW87mA6gRCdVVVqlCRvICysvERZAMBbXLijg7YMW4DufIEeoLWlWu0m\\\\\\\/Vxf6ltlyvH5X7\\\\\\\/CQwGvoEYqmrMZzRYizSgIJ+Yt1DKRiYOiaT+4\\\\\\\/qNz79RjNUa+GqS694U5vPwfurKuvztxqr5I+A6LPSGf7y8E7EC3Yjf7fppmVTODkN8CXHOGglJdOdQ40H6MYExZ+0GkiAaM6eFnz29ww1+Xm8DfAXgHijNuIWp1CVA3Cpe1eabEImOtvVZButRdUXukrLSY3Y3S3gghv7KRfsjDzYgYUoS4NqhkSwRXwQONfHBXwwVLKoNM5JuoH2NTvuj6El\\\\\\\/YBjTz\\\\\\\/RVGaPXOWScpOOj1z5RZFKnsLH6DjbD1VtmiG+JgskWm3zUE\\\\\\\/P18MSNVPbYNBMd8E2rkacmnObtLl\\\\\\\/eJSl845vjC0Aqfb79qR5T4+S7++E+IFo7ZrFU+JqfR+7wdB3s+0r9qG\\\\\\\/jocSr2WQkFRgN5qsvwQvPz8oMyqBN10BxraBzuLYfUV9n9BPLVd3LuG+8tBZmQNULlrKoo0NvZour4EcEe1UYZB\\\\\\\/cOuysmCAdM\\\\\\\/eHwGWzp1JhAvJ3ghItCyvJGeEhcEOfoR848tisCK3l+e59EuSoD3N2O0NuDWiT+Rg694yWG4Gq2QmitXGWjLbqYwXjvwGcs69erbArjcXlfDlG9emkvot\\\\\\\/8JJyOxtTcB6pM2cBP4KXQQwf6LK\\\\\\\/PCuDR8WgNEEj85oV4PKIN0KOnosKHc4a5Q6nDZ4baSzDWrZB47nQKj79Vmz8b5hgKpJ0wBLss6gWb14lph38xoTSP0lq4LFVrJokt\\\\\\\/Kr+lUygwKF4MIvED18UhV7hLj5bDRLaVP26kY7ops+PMh8gXW\\\\\\\/8rHrTEe0ucjWPhHtvdWPIyaMnDoaXl6Qq9tAiDbJPcr47VSYrL17IW3fel0jG\\\\\\\/+T5DUw3J5tTffDfQFubKd7yYCyNPKuZHoB86MGR2uyk\\\\\\\/UqyeG3mtJT+a3Bhy9HEznMOJkE3Q\\\\\\\/8swerBvxtm704Z+BxbNgtD5TZepqpGaVOVjkU0HCLy15jkF0Pb+r2zIapYVn+ex+IVUWhgGyN+UtmdfegDwScg8eYmCRH6BYVxAtmZ+YOej6VUW58w0oagw7IH+NBCT3A5iztXMD3EF6y6lTqN0yg+hqrzjDaXp0pCCKYH9wPTkNUmk2SApCM2urMGtj8\\\\\\\/+HxfVUy60I8d1ZlZFVzZw\\\\\\\/HDGifwc9gCFrFt028d4vHpACSJ9NWL1chU7cCRDStwXKwF0D9wNw2\\\\\\\/ym6pQEesFJo4k3Gh0qkXu2VgSZk+P56CODETGPu7Ks442nQelmgWW0\\\\\\\/2xkmJQrozp1XP+nm\\\\\\\/zZCt4CWtPxndiBb7Lb8EkInzyQbxx721cNuB83kSze5Q55klGbHvm1ONIt2HDDjBBNqTWXQBWyJU8y5njqu97q7wgA+QIJIhvCs9NaAqrWXLu\\\\\\\/42uPC5LAASxQlr7d220xYPWvk++KUWDplKG9Sq1muYZUNecSf2TK4zSPwVKnz2YnjaXrnHaH+qmPuoZ0Lv3B6zf4l\\\\\\\/HtvLoNxG7RGgI3uqTOEJN8Mp9XMvEwiFgV\\\\\\\/yqL1OwfqPZiBuu6CvvbVADTTE1fh6b8JtTFZXoJg27be03G7+o3vd4EfzsSBu\\\\\\\/UE\\\\\\\/Js1nefwyr2fDzHcLer7l7lSPW6A7QaaRU2dhvdJdYRl4Yi50cS3UbUDK1Xm0rw2KdihFnm\\\\\\\/NPoTxfLu6shexa9yCJrtjIe+sWGnWUXbXIvLPtEgT6aFXZ1tqLM\\\\\\\/KPLhkUawMlP6x9WB8stNAsZ89vDQ8Rl4T78Fcv3HDjA\\\\\\\/Ly9SbEJMIGpAsc27k5JAbUO+bEbvxLZTaTlvFuRLfcQPRoHDqGHWLAdIQWM+X3v0JFbDR9UdUB0STUPKVPbknpkSx1Qlg9e4J4+PPfa1ON08CEENypBxLGLzUkxz7tjArMxzEuOZEW4mph+TNy7XPvHnmhsIoiD6IuBwgg51BcYn4bdkQmtDS083LivtQWg2ufledwZUA0uMasRuZlnGtSi93jyckulqxGXxffBpWf5fD9nzjdSDginIcxhfw9hzdUptAckn1iL4sPGXIZZP8TIcWIfEXmr4tnY6xL+F2bh9VT9BE2weBQsm4bS5Q8yNgIPBIM9QIBhpCPBCdlE5ZteoTS+rKiLkQXfnsa4G7O6VatfAfz6te+upnEYB3P0Otsa\\\\\\\/51NETiDy8J\\\\\\\/8FNQ7t4aDEj\\\\\\\/CLBK0bdbrLGhllGhUwnhE71dGjZIM++u7FwqxdcEyoDgk1P\\\\\\\/052WH7vcG77Y7NVqtIIfDIa6SZGE9n0CkaXZZUBUa3lWrd9iCgNSKdFp1MlyPdQUxPLQ8BZucXfIIF9NnLtL1tfybd2P8aZvVu4Rln8PCP8z3bH6W\\\\\\\/+E3ZCTRkLn+QpVp5E5RYeQM3wLKYJxXUP7TLfh+eOY9Pn2Gmj95FAWm17yzsx3iQBokSZHs9GEFkz\\\\\\\/FBmvjSJUajd6JC+XJsJCcHgpmI4SO184xwFNaX\\\\\\\/vvgmriOYftIDG17QqeuEOtKugUT3YajlOfj15KVxeyC3ADX5aHmQ0rrG0NBIXslA2kMlQGCjMHegPsqIZeYyj5E3D4KsHVyrG7muBeuWbNcmlfa\\\\\\\/avONHcFKR6fBkLIbO9QURir4t+3Cck9zLKVY1lwtFl3bDplh337lfhCZZahzuQydD7bL6aiJP+uj8CsqM56AwMpz4ZVBMxpY3h+3cOtSUku17AsCZlvvcuJ3E3TgsVAJk1vW7mCnQ9XUxI5TWShdIS+St6OXexMW2J7yB5VANzrF+yUb2Y5fzHFKVSxN7sdFJigyAyyuiV4PIUWMrntkr63eQsqeJs34MYWo4dcXGSYD4uON5HzCP+DDnWRfiz7OgH2rC4TpVfAlf3UCqJlQrnhcJhkabPgWo+az88CoDAMGGzsgektSot7CDMr2gxkLgEXKuEGWLxgg5PCHjCTi+xpbSO5l\\\\\\\/xCMYg2W+n3gxvMgVCl6Z5xGpuua9Beiiek3BrMWRJjW1J4cpuzu1GRJxI5MyFr+F7csdFDs8+TSM1sUsMS4Z67+iMAmvO7bCYqonCWLKqKedX+KVRll00T3XAhdH7olMa1Aece9SOFDzIuzZzFaB\\\\\\\/0Lz0S1lC9XFtOawZHCDKCnxOBGik49NkBOco9bidqREP2soYRY8Rqq7zDkv0lvEKovad43SMbanKp2StsWDkwIK3Hka+FVbVm0eWOOp+7U0tJBKyiK1z0UIIcP7o0gyvP5BNKhobnmqDEILJGQHTdb5H87uBCy9y68+IP4BQ4jtkK4u3A1HHbtYF7oq3aXKdXJTsDb5lxGiOIsdwSARGFUa7ENIsMcS8E\\\\\\\/WEYon4Lh2dV1Oa15\\\\\\\/C4SrGdHwC\\\\\\\/sQ3y67jEqAfupm6Z99KIQXlWDhc2ZJFPBPjBWXEnSgpZugVG0Sp1mMaMobxCSP4YFRw8sLOxk8\\\\\\\/GK4GOzqx8KCj4ien4zzH52vah0F4ewuQk1tnNV\\\\\\\/g78ImafSfWrFaxM+kqo8LiXusRdypRZ2gri\\\\\\\/O\\\\\\\/0uVZsqJSHornLlOdfI9EIAfQbGqIvBgF0K2GIan6cfwulTLeZdHSAaStX0JYMGCl9wk20IDfgcUNyz+fNkBIlcs7L0HHLZ5KSstBwe74VV6G1x2PrFp8JkuZ+p29VlkiYOi3RjPf2weocMRmD1lNjtcnB4Mdoy5xy2PWcRGDX3Fvl94zs5oKlnRBdKdmy5\\\\\\\/DzFJ8fthWBxyUhMCSrVJHYZqFWR\\\\\\\/bryTxNyFYMkks90tOKVPo8qyO7YCDAkL8i2nEgszgSISNamM5RXg\\\\\\\/VEmYquROFQsd7u+VZbHddeHxEhScsBdBXsBxlAhynZABHRf3O7LRqI\\\\\\\/ogtEDJYPOj+3g+O6dlGbuLHxel+3KHI0EYBwmMUUi5sJyD1jdpjxEiht1oHavwv2kbXc8Dxo\\\\\\\/0inOYVoXuZz4t+f7bJVMPeUrQzAt0GHJfLSYt6Nk8jzZrjhqdhiQL3GJA2RUc4mILEDLYGY5\\\\\\\/YnA4c4+FDKVRz9fvMnUFk5PbYJuXbtJlYa+B8zfv2YKmL2drodeTzXkW94Ncd72poTy9dsvnGCFkmbxmm1htW9D3\\\\\\\/vSKdQNE300PsAGUgfmzAreCa3xAkpvjUx0VQ9mOY+P+L\\\\\\\/QEH\\\\\\\/Y6MOxZmvOifv5TkpGJIeXsz8TERmJkqlGDjgvBIP0fTCzMv7YlQRgU0n7xgIhdNudXhqL577as4DGfaX3jV3W9vq\\\\\\\/3hlfR29pQ+M65me2C9WSmz9UY+FBLiaDbVGdWOTzbkhNO8KGtBCHjmr1reMHVSixwneR72N9yzK6lVOq+9GM2tYtIXWQ\\\\\\\/cYD+UTIWPPpwKQE5jnd8ogaM5KfTcJ9YSZaeyCLJVG2EYqPowG6Rtj0KUZnfa1+1fcs\\\\\\\/U4eO2IUzSZ\\\\\\\/e8UdPbjIxOfXUZuy3ibm0hGqrZAsALenVds5fQQsOoz2wzPMOu4kLNg2JsqySftI73ZbtInPXkd6FvsJ\\\\\\\/V5lUTnVWdcKS2hESX4cXETryKXJOZpJJwBE4nPvptYluUwEnI6UXVeYen6eBNfwBv5ppJ4H8vaRn61ipde5XZzHEJZnwnYfBwC5tFDPR5\\\\\\\/PB\\\\\\\/2VkQYpFZtkYTH2Z+Ma0eGXUTlPogXNGJ8ETGhOAQ3kmaI85RJ13mJn3Z5uk9+6LB4WwmBHajUkWIdCd9OYNSIJMp3NLuc93CdhlDQ9ciSUxkry94t8JSGlSz8w9jJhMrfdWo76BfrykKkAz6wl2+AGoEZxgiHoGVWcbn21WWUMgQybswuKZ0TeekrYSo4PdkJ1KBnRmuvYGiJJYyYL21\\\\\\\/5\\\\\\\/paDOtQ9l+kWwcsLhtiNke4kxp8PndEVCOMPLSFRnXTtEXthSIPKPOSRpUQwouWkbvsgWLZ5jyFTyPQElny\\\\\\\/1t9PjoVBJ9Ij\\\\\\\/uYxjlW5LQFVGmOHP7eZzfNqMx+IvGE8E8mn1HzhQE3BnMhbab1tOuZf98h2pwwIHke7knPfnEXH\\\\\\\/Khz9vgOYDztkMYo4F9E6Ja89eQiBTEcA4IqAkcLIdM\\\\\\\/lSVPZ0xSgJ3Us+LhDAPtOmrPNNBg3\\\\\\\/8aTQxZoJPjKp1Fvf5zZk0Fhe+Ha8szpeHSITdzWvbZNGcT69DqnBmuTpqv+M1VwZ\\\\\\\/1uZmHeBKwbkA+2Qzew+G3FySP9DMfLqfpC1kMcknmgtrsk\\\\\\\/e3\\\\\\\/ckWVLNYjhvzipFjgQnq7PGsMWrzrRWviDDRNrOch2g6G3qaLZreJjWSLYpDp8sokLxR5y5VnXNCNAe9FqHmVJgh95YkY7mC2FfasEGxSf+Gr0Hr56ROyhYlgBdotYQ9CQpb4CqJpQ+A563BWSrmeXsvl+vwIXs1C2wfks28heyy5Au4r6i3t35NjWzR72yhyqrVikaswCcZ373MmVkmGLCABU0Ht2ZLgEaoFrYhKvveAnympJHtgmgT9QVoFA4Omknyd30ufWh2VZ2rtjbOjTFuoO1sm1T7v6ZFBTxy6\\\\\\\/ib7xpPLmYm+srgb3may5GZ54uXjK1FNKeHmhn5ti2gRBSAEAedoLyfktQLbcJ2RaXD8ZWwzgEqPxV9TFSBZunEtqDnmgVeoq1N6Hvzc35ocRwfSQeV2bmMUQ3gGgiC2ysVYmPqqhwISDVD1su11OjC4rGVBkNAcatw8Rc5sGVqjMMUq\\\\\\\/wEd6w9mT41KGZOgI93HGA2nD4mVk8y1CLbpHsV3j5q2ThgEGYVdhdNQxr\\\\\\\/nqhe4lIRnXSIlzt6wL2rPMzHKKXWOjCCAQef0Z+ukVkHRczWUxkT2Jd5AljfJ91AwnxLBZpKgu+69CtaLj2JB5Z07SH\\\\\\\/HY\\\\\\\/ohh7qoNWJIoXc7A\\\\\\\/uY7HGNoUwQXP69Ne7y4+DiEyovmDBwlVyCm3iaDPKAFfD0ceE0kwha8jleb5w1GAyFDjKXkHZpHbYW2dy4m3tHfV4Omjil8efb+WSIi3nERJcuH\\\\\\\/HEDzcy+PHw5GL34fQTvoEFnJ4MFAtJf5QTR0qoOsEwJU2lMnYxhoEifnE395IoGJIFhJkHXqUe+UbID+gCFe7S6itn5Nmo+70BH4bqCCOUQtgOf9OyqCY9CSBuAzPAcxyGMdjqi2gBRHadxgRn\\\\\\\/Tku5mVWC+LAp8SF23Ufsto4iOu3iQC03GIvCPv\\\\\\\/XwecqXX8T2UJSyhN1VU4\\\\\\\/UId5XVIo1e+BCDjfxuLASsRnvo4dMwkZubgcTOEhJtip8M9HecURni5erz2iDdLSb3NOGpmaSksIBPNS3CvqYuI4JIUc7vUNH1rZvKxZK1v4ndBsx5EiEDwF3Ll2YVDHYYCtOQP2N0z5b8A3xddWZU7d88GxKalIapx+flRiVTAd9RZedOxpZGWdHbyBigHrM6jbsy5ucNfYFMdjjmdoHDyeG1yPhzKE0Yhn+QnKakb898E6eHA+teBXrN+JSA8av\\\\\\\/20yQYeGK4kYUFAH\\\\\\\/4zf975dimUrdpeqozmPEUoQSzVEcBKbJMRTC6u1HNCLkETVJrvxdo2qfVBBXb24sGexVCKCsp3jSn5BXngVai94NDXX1dJ\\\\\\\/flgAl6YlA6D8LpSDsU8mlN2SVdHE\\\\\\\/jY91AeY0pY1XgI7cjNeVUuPFKEXV2Q8Uxw2t+zLTQprqiGzVYrVSBYm54GTWWEA70J0pY6q1xNNP7zALtTokSdtFhAq7idfbLI4lqDJSmQl8VpNu31iFxRN56xLmdel4xeZQ5TS\\\\\\\/Hk0g\\\\\\\/weBjWv6033UleRJI+JUT179kjGBrULYd+z7TQOe5ycpLPvfIiWsMUDVJImJ7dGRKthI9mOXdK1ck4XipC+Dz8A9jFU05Mfja1l9Qkx\\\\\\\/r\\\\\\\/fZmROl4D5x7dN71DGuOou8NMouUJl+U48GypzDE6zHwaaC4gE6bu6YQ7HcEM7Ukv3vI+ResehPO03aiWykUBy8xRBptd6RZBAY7DL6GVouSJKi3154Wu2g0Vuy\\\\\\\/ahw1q3YGscjZ3NQ6vjRWd0CQDgE2qY+oTLE1zw8cXwqIykGEYpdlBoO\\\\\\\/NzBC+uYsptKCsVaNBWTJuFUugexHW1BtWDubEThbTeRkZq8iP0xTs0tDoPVNwb33DWmkZT43tqrVU\\\\\\\/uVwsq\\\\\\\/EA2WJnheKbeo9PEB7N1JbEAr7BV+mB\\\\\\\/5A3K4X1tH4B9hmbhoA1gMZyjCmlEZ1kPTQ0TloVqOKqDT1O2tNmUN8fru6sMFtzZB9z\\\\\\\/RCJAJjOBCPH3tff3uiBMA2jY8DefkbrR4jPiSNYRsQ\\\\\\\/7EDXCVQOoRDdmPvkfTWUyOkoVOXJkMuPxNqBFWOTPJsFE2F4027C3SfT3axzRRGO3BdzKS80Tg76G6ouvD3bE5Wtdxxw0ofEKpJkOh71U9aUGZTmCVWY\\\\\\\/L+Pj\\\\\\\/13Sdb26ReuFBn6yrdZz60I2E\\\\\\\/kcNnNDOO79DuzVbUgaIedYVIEqyY1x1Xu7MXUQ7CfyZAJMlS5o8VyYi0wWhVGsi0PgrCJwpq0bFHUKqiZ7k7GG0eOYxdE30aR\\\\\\\/PeJqGlhJB+RYOIEiOFT6pSbvUcnvXcSKgktxJgVgzk+55X6Ye0i3OCpqRpeheQOzt1oNqpTU58mYTFrez73cQV9cLQ3NUi4K3RUh39PbiVOTPWxiaR52A\\\\\\\/\\\\\\\/i1dPN57HreR\\\\\\\/XiCFDJJgzSuIBxbae7wYdyftTdJ4BTGMta4gGInBPSrQNt7xTB4\\\\\\\/7\\\\\\\/yQnjR5JxV1vmVjaSyCzQd2GoVvuGRbplMOPcf496zPX+2xwQ7M1cxO35Ygw3RC6iJLtlsfrA1KH9ikSDtnzY9GfKptJLJyF\\\\\\\/du6p4WBqLS+FnPup+\\\\\\\/Du+F3mEPn0jRXzo1uIQIsxhUN8tyMyIAe4u3hJ\\\\\\\/kklgQqOoFdAhDgiZ7j0Rf4QnRpwg8LxCQrLhNzdaBLtjYM8k90MSlOX8I5Egg1JTyiexzkVyVUDNF4QuO7u5HhlcYtTEOh0VJySKnIUbZNDtNNBKbZZ410bSA4SQa2gHcIhMpAfeL0w7tsihznfj6+5aTA6BzAxWfCmW0t7qjwJ2nzU3Fd8RSOD5lyFZ5KdU5RWNn3dh03F0NXVyz9MH9fObL4khk17MYXxCuUkoXLHtD+RIWP+Lwwh57IlX9rOM3xBO5+XqLrOSSryhGHw5UWZUFh3lz44EzBMcPVVoNSPbSs7JFZMIHcNGQQ0sSLQQrB0kxr94CI9VsP1ek9JLU7HnprVKSS6tBy+hAzN7I7t6JiqGJmQtFEJKI8LBUMZR5OiDmIYUbAe+GrudweBLXsOT1qOoing4WWvpTySo07jxg+ztUXCgb7eTRyJEeOj6M9cfInmR9wMm22GbNKjdqzOPNvqf\\\\\\\/Wf2SfTArZ+NVjMBh4tqFhnWiGozo3HRXStcvaZyInhxgsHPbD9uV+GmE0Xr\\\\\\\/D7b+sbMvWupsi1auHf3Ekg3eI6vif1kTsIOssUcW1gqYhqI6G2i\\\\\\\/qoUK8J+64jNo8uTyUKx17RhFq\\\\\\\/YMlW0JahsJL9tbZBqDCokxH2fWvZlis\\\\\\\/pMa83iMX+YdJXR1+mz7\\\\\\\/IwYpZZ37rRclwy7d5gF6iFRIuLEQnjz\\\\\\\/NBCRJMLDiHjWXpHWxtizdv5+qUw2Uh+kisDESr7++KXooqdjxxEtvULqa8PcLfgQoVwOlvua1+SOZO3x3btofThFg5DEcjIGdaBHGb9U0Hyyo2Tnbu8CJwBBruPU3YXwkTuX6mfSvILBHnvUfYICTFzuLv8iaHzrlOJ\\\\\\\/YwOlXzWSPrS+XNyJ2yRHosODA8H1SeSFRBMKKgadjy5+KoHxPZfa\\\\\\\/U9IZa4aYZicsvc6QKjmEA8tPFJBeoVphLvkyjZm2\\\\\\\/xz5p209OGLWLiyvO6LOk7wu0l6nrv1n7bpeVNkPz18OcX94M8dgeOkIGVQvVrnWfPdTs77UoZhTGBR+\\\\\\\/8fmywwGOvORybzstcMBZQzZ+3Jyn7BbSyZakginCjOhsuRhnLAsVR\\\\\\\/3xrBkv4VKSR4IFoIgBcB1CAMMAi71ztXl+PYHznvNT0WExscYrwS9ZuWDX3sW8CECOuUO3VbSPqn65rOovYUviGA+OnMkR6jYqZTZ2bLUzql42nhBjKOG1s49GS443mpElMvLGXlUM7sTL\\\\\\\/KL4a+SoBuFKKtPxkyLzV\\\\\\\/4ZAVZ3aVVw6S7WfLpQH7GUEQfU3ptk9OnIZC62NX8+Wo4Rbg\\\\\\\/OL0VI7pRrRwN6vhFzVrMa\\\\\\\/Bsvzmd3AF5Mw5q9VJGr1cIe3uMOvG3zEYm82LqkiYHFmw4MWi7DdxI3OtBnZUBIvYcwjj2CBpgXse\\\\\\\/HqwLJ1cUdNG0aOC7vpMqbFqc3JVgzwK6qyWaJYB85Cr1iCl\\\\\\\/\\\\\\\/JnOyiTR9cqEFUi98Glj7IRaXiF5tQQ\\\\\\\/3cmx4WqDdXULidtYENhAzc1U6Yi+ufesNPh6LXhCvWNVxHnvKrPvYBRX0Q242hAbqgkS3CZl5M+mDAoSoqqj0XuX+DtE+jPTeL3rSNHclZI\\\\\\\/HZCvFwhA2GWDNIEDcayjlzLuTOcCf27Wi2aUppX1smmoP\\\\\\\/mZ7VL4wnZYLBMtbFhGx6Cy9tlmjr4lfsinQ7uuVpr8VsBOe3pIHrqtfWdnPMPIA8PMARJHEtPI6d9G1GrGxaiAT\\\\\\\/8truXx0OTisPwKNiVlmUo9IoO+FD6LA9q8cvUmy6ucSuDewFXjJMBEnLE1w2Fb84V3f7XeeaISXEqGrbu1cJbiK35PHMWhlOQQ\\\\\\\/3lhcmp59wV3CIeeioANEY96Cgi2aSHeHV21p2a3Z\\\\\\\/06v8CnY6Dyn97mp9fn7YFibDpw9T+00FO+wiHep6iie682OzrRh72MB\\\\\\\/yInd2n9uIKDV1LPXMDuNNQibV2Geth8cjLgqG8CU7df+iSJRKCtVWoz58\\\\\\\/BO7eohcnAIkQPUWL7+bGm306fWJM7O5FKWMcY\\\\\\\/7KmHdNoWklBU73iB1BeKvj6IRjUKkpupJ9m0DP0RSdAIvZWFMgGjdHIBa6N7Byc\\\\\\\/ibfCp26CCrhC0hhlA3Z6VEvEwiM+IfSDrpTW6WZhO2mQiw7HDUzF6IdxciFuVek9AvDHMs5P1M0N22y7xCU4l4CvvfXCg\\\\\\\/VUcVTF6gZlhq3olSWiPE96kiGoXSGz5QfjVK6oUmc+\\\\\\\/dSqp00Uw4MG4cpBj2bOngQQTP9xf\\\\\\\/\\\\\\\/iG0Xc\\\\\\\/I1uKxPfI1T\\\u0022,\\\u0022iv\\\u0022:\\\u00223Kx2ZMWpYTaFXysVxc97tw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226598bcbd5b7a95a19107053f6cea0d83b51c371de979efbc510e1343b5be448e\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022XzsAwPeRiFzv0czIScpFmiD9p0899w+v9lb0rB0ZrZblyMrtb+NNiVRtxDS7IvrApN0ndsbKheYYyH4R\\\\\\\/FxyA9HIAyTT20XtZTuinqBW5Xii1JOxSh001PJOzVB+Toon+qT+wC+LUl\\\\\\\/u0zHbPF+zigjy9sv9sDSjDqRs5\\\\\\\/QUKVwPanfgKDcqPQGAcDUjUH2UMmuCfskIuAR2Jukoke2BQzoimrYHJBnk84zb2b7vVHe+lsaggOFlji6\\\\\\\/dyugc8igjPADiG7ecl+\\\\\\\/GVvrD\\\\\\\/enYHsbiD01zEBDQ2DWGpO\\\\\\\/VtQReO9XZQzVDBIqYqlf7K92GHMu0dfGPLu\\\\\\\/6XR1j32n82E9gWBv\\\\\\\/SVjNniicdNSodTOrCuOqTCwDyjxHlzZmOL1SCbqw7RAA41ukM7QRlJ7N3sebY3vrb+Esqt5dADmCgJ9czCNBeBrsy9eKSAikYOdmbJ9TazOiBXsL3UbPs7RCk2d\\\\\\\/1PghmfYxoRl+0zE\\\\\\\/ME8CF1gdQuHjWKUs3FRQoRUshsawr+DahTgAyOPWAPss7\\\\\\\/YXYvvD5ZXjOzcZEQEZwAklJhS3Xtgk3pwSaFLrQ1WTmsmPHIhXQl8cNxNTJfK2DZ0Zec6Qe\\\\\\\/ZA1Dzw2iKgwd6lBPNzsrNf3Tu\\\\\\\/X5huEs80LfqtNB6L9hq2C4PzDoaUm8B5uKV6Y0QGYeOajvL1KlI0frERVJ8g6P1vdUhJs+MNxugnQsAAvzz64Y4QuW\\\\\\\/JF\\\\\\\/\\\\\\\/5RyD\\\\\\\/D31BObC+f2q4tyFinXyPBsSs559A7MeyfYEtJdzQK7QVTt5uf5174QbXDdIQ5dspraWZSJ\\\\\\\/CTKKOz7mF2HNQn7Kurtjx1kTrKYbXI+cThrcSOpCKIAkMVBPK52fdPSewXm4xwDOs207y6PQgHKreaHnvbI704f9+cFaWckbXaqEudz3+x6\\\\\\\/UFtvc3TdQGFQFmqKmkq0+nfjVAZM3Vt+2zgQJ5IHXsrBNuqcmMCmgAuRu\\\\\\\/sW3zQF5GkrcCAY4YCOmyl1Szl8yWU63KtXL7aW7007+NRMm4XaRCYLcbv\\\\\\\/6ZEYdpJUYYHR\\\\\\\/aU8rofCqs8OELbmfk0ZiKQv62O6yXYW+ip7QOVqYQ8NsQjeuom6dFuo24UYANmrAAGZ84eWNfzeI2RkPnQjjWlkp9cb47BcX7xkMHnhX9hGMCu4ItQBc8END9mDv5Vq1XgJo+dSo3mSwb9+i4VMOEVdBqb63dvwHIlIRSE+ramT9wmnl0dcNaZ1I23rIgjc9oaPaYyLtlMm62cJzTTxokzkfb4XEOxiyYuW1dyglIFckAapPjtdwgJ1AOBX1Od8e9BC8j4bBEW5csPi2OqdeFUCjfIOZYCzguDi9tDX554NIWbS0v8GB60sYVZuLrydz1yo7CsfEO6X3B+KY4Jzxg+VMVChLeaDIsMZFMz7lCix0OPWw3rPmy7BF8UV5rluU6\\\\\\\/RRpP4wLXwFn3V8d7+9XB4mY997PvEHpCoOm5jtY6jJFAuOxIJYdoz4LhNtMofYTQIha8dq+1mBk0bTQqRMxsdFquWF3Krq3HRSykNqQo9aztvlm5DFzCr9XVxLfzmL1YektY92HnRUER+oZBgia2u16yswvPx99bMs\\\\\\\/wElZQtsdhB6XbOdF3xklB6jQmHHFN\\\\\\\/+Nb6YYcsnk+DXCJTBevWe2J5W3fezpxCeqSA15qhhR\\\\\\\/diRxApLxROdaHfQCkpToXOvqdszMBlmbvqAPvkz6Hf7P8+0GNkhZbslQwBDnyIMyA5BPyxaajzcGJLudI67tu2E8+6ynNtzW1UH7jardSdtLCIU9+S\\\\\\\/PbGdrFNw4aXLl7BRpq21l9Hc4T0\\\\\\\/dcPIHTLWC7wT0BXw5doFKUUfgFtSBhS6Uvp7LM8Ue3cU7Hr3pwZ9FEYfuJD6BmrQP2lWaN7\\\\\\\/q8VHiM7oqxsYYHmOWNi018e5zsWFLysV0c0YrHon4JC6zQdZXWFyJvzVRNVYHeNY6WGPD7AMcBabJe0bx3U9X4EDbcuXSdUCWeIBkHldqlhYjMnNOtMoDKdjB9tqCkInKHjgMbz8XqmTEFbz9CxhU=\\\u0022,\\\u0022iv\\\u0022:\\\u0022c540PeSNcPJedp8oNU\\\\\\\/Hcg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002267763032dc5ffb1e4650d0a7dc1d9e6e3c6ab1d4d0af8a43c255666e2a3c582d\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459473 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/6/959" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-o8gdWVQwl3GUzeU3oeoNHLZ_EE-U5X6SSw0LdEe3mlw" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/6/959" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-iIA9MtTJvO5aIPEUnvz8zaTrzgd7l0sfQjzxM9naF2g" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/6/949" title="The Use of RelocaTE and Unassembled Short Reads to Produce High-Resolution Snapshots of Transposable Element Generated Diversity in Rice" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/6/971" title="Genetic Polymorphisms in Host Innate Immune Sensor Genes and the Risk of Nasopharyngeal Carcinoma in North Africa" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459473" id="node459473" data-pisa="ggg;3/6/959" data-pisa-master="ggg;g3.112.004994" data-apath="/ggg/3/6/959.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Resequencing and Comparative Genomics of <em>Stagonospora nodorum</em>: Sectional Gene Absence and Effector Discovery</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Robert Andrew Syme</span>, <span class="highwire-citation-author" data-delta="1">James K. Hane</span>, <span class="highwire-citation-author" data-delta="2">Timothy L. Friesen</span> and <span class="highwire-citation-author" data-delta="3">Richard P. Oliver</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">959-969;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.004994 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459473" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Robert Andrew Syme </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Australian Centre for Necrotrophic Fungal Pathogens, Curtin University, Department of Environment and Agriculture, Bentley WA 6845, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BAndrew%2BSyme%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Syme%20RA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BAndrew%2BSyme%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">James K. Hane </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Molecular Pathology and Plant Pathology Laboratory, Centre for Environment and Life Sciences, CSIRO, Floreat WA 6014, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BK.%2BHane%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hane%20JK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BK.%2BHane%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Timothy L. Friesen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>U.S. Department of Agriculture, Agricultural Research Service, Cereal Crops Research Unit, Fargo, North Dakota 58102-2765</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Timothy%2BL.%2BFriesen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Friesen%20TL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATimothy%2BL.%2BFriesen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Richard P. Oliver </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Australian Centre for Necrotrophic Fungal Pathogens, Curtin University, Department of Environment and Agriculture, Bentley WA 6845, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Richard%2BP.%2BOliver%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Oliver%20RP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARichard%2BP.%2BOliver%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:richard.oliver@curtin.edu.au" class="" data-icon-position="" data-hide-link-title="0">richard.oliver@curtin.edu.au</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/6/959" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459473" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459473/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/6/959.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459473" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459473/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/6/959.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459473" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459473/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/6/959.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459473" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459473/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/3/6/959.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459473" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459473/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3"><em>Stagonospora nodorum</em> is an important wheat (<em>Triticum aestivum</em>) pathogen in many parts of the world, causing major yield losses. It was the first species in the large fungal Dothideomycete class to be genome sequenced. The reference genome sequence (SN15) has been instrumental in the discovery of genes encoding necrotrophic effectors that induce disease symptoms in specific host genotypes. Here we present the genome sequence of two further <em>S. nodorum</em> strains (Sn4 and Sn79) that differ in their effector repertoire from the reference. Sn79 is avirulent on wheat and produces no apparent effectors when infiltrated onto many cultivars and mapping population parents. Sn4 is pathogenic on wheat and has virulences not found in SN15. The new strains, sequenced with short-read Illumina chemistry, are compared with SN15 by a combination of mapping and de novo assembly approaches. Each of the genomes contains a large number of strain-specific genes, many of which have no meaningful similarity to any known gene. Large contiguous sections of the reference genome are absent in the two newly sequenced strains. We refer to these differences as “sectional gene absences.” The presence of genes in pathogenic strains and absence in Sn79 is added to computationally predicted properties of known proteins to produce a list of likely effector candidates. Transposon insertion was observed in the mitochondrial genomes of virulent strains where the avirulent strain retained the likely ancestral sequence. The study suggests that short-read enabled comparative genomics is an effective way to both identify new <em>S. nodorum</em> effector candidates and to illuminate evolutionary processes in this species.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/fungal-comparative-genomics" class="hw-term hw-article-keyword hw-article-keyword-fungal-comparative-genomics" rel="nofollow">Fungal comparative genomics</a></li><li class="kwd"><a href="/keyword/fungal-effectors" class="hw-term hw-article-keyword hw-article-keyword-fungal-effectors" rel="nofollow">Fungal effectors</a></li><li class="kwd"><a href="/keyword/phytopathogen" class="hw-term hw-article-keyword hw-article-keyword-phytopathogen" rel="nofollow">Phytopathogen</a></li></ul><p id="p-4"><em>Stagonospora nodorum</em> (teleomorph: <em>Phaeosphaeria nodorum</em>; <em>syn. Septoria</em>) is the causal agent of the wheat (<em>Triticum aestivum</em>) diseases Stagonospora nodorum blotch and glume blotch (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Solomon <em>et al.</em> 2006a</a>,<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">b</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Oliver <em>et al.</em> 2012</a>). <em>S. nodorum</em> is an economically important pathogen in many parts of the world, causing major yield losses and degradation of grain quality (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Murray and Brennan 2009</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Tan <em>et al.</em> 2009</a>).</p><p id="p-5">The genome sequence of a reference strain (SN15) was shotgun-sequenced using Sanger technology and assembled into 107 nuclear scaffolds and the complete mitochondrial genome (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Hane <em>et al.</em> 2007</a>). The nuclear scaffolds totaled 38 Mb and included approximately 5% long interspersed repeats. Of the 15,980 genes initially predicted to be present in the SN15 nuclear assembly, 12,382 have had their annotation manually corrected or presence validated (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bringans <em>et al.</em> 2009</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Casey <em>et al.</em> 2010</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Ipcho <em>et al.</em> 2012</a>).</p><p id="p-6">A major impact stemming from the genome sequence was the identification of necrotrophic effector genes (previously called host-specific or host-selective toxins). These studies revealed that disease symptoms and severity differed in a cultivar- and isolate-specific manner. It therefore has become a priority to sequence more isolates to determine the extent of genetic variation. Necrotrophic effectors have been characterized in <em>S. nodorum</em> by a combination of approaches that typically start with the mapping of disease quantitative trait loci (QTL). More than 20 distinct disease resistance/susceptibility QTL have been mapped in wheat and the number continues to grow (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Friesen <em>et al.</em> 2007</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abeysekara <em>et al.</em> 2009</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Faris and Friesen 2009</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Friesen <em>et al.</em> 2009</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Chu <em>et al.</em> 2010</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Abeysekara <em>et al.</em> 2012</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Crook <em>et al.</em> 2012</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Liu <em>et al.</em> 2012</a>; <a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Oliver <em>et al.</em> 2012</a>). The isolation of effector-active proteins either by direct purification from culture filtrates or by expression of candidate effector genes in microbial systems allows the mapping of sensitivity loci in structured wheat populations. Co-location of effector sensitivity loci and disease QTL has been performed in five cases. Three of the effectors have been cloned and fully characterized. These are <em>SnToxA</em>, <em>SnTox1</em>, and <em>SnTox3</em> (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Friesen <em>et al.</em> 2006</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Liu <em>et al.</em> 2009</a>; <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Liu <em>et al.</em> 2012</a>). Each of these is a small, secreted, and cysteine-rich protein with important disulfide bonds (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Luderer <em>et al.</em> 2002</a>; <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Liu <em>et al.</em> 2009</a>). In addition, two of the effector genes (<em>SnToxA</em> and <em>SnTox3</em>) were found to be adjacent to repetitive DNA or to scaffold termini.</p><p id="p-7">Different strains of <em>S. nodorum</em> produce different suites of effectors. Effector gene absence is a commonly observed allele in the three well-known effectors produced by <em>S. nodorum</em>. Haplotype analysis of worldwide populations of <em>S. nodorum</em> shows 15% of isolates lack <em>SnTox1</em> (<a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Liu <em>et al.</em> 2012</a>), 40% of isolates lack <em>SnTox3</em> (<a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Liu <em>et al.</em> 2009</a>), and 60% of isolates lack <em>SnToxA</em>, (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Stukenbrock and Mcdonald 2007</a>). Some strains produce no apparent effectors. Effector-deficient strains are avirulent on wheat (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Friesen <em>et al.</em> 2006</a>; <a id="xref-ref-42-4" class="xref-bibr" href="#ref-42">Liu <em>et al.</em> 2009</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Zhang <em>et al.</em> 2011</a>; <a id="xref-ref-43-4" class="xref-bibr" href="#ref-43">Liu <em>et al.</em> 2012</a>) but cause disease on related grass weeds.</p><p id="p-8">The three known effectors exist in a number of variant forms expressed from up to 13 different alleles. Differences in the protein sequence modulate effector activity (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Tan <em>et al.</em> 2012</a>). The sequence differences carry the hallmark of accelerated evolution as determined by elevated dN/dS ratios (<a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Stukenbrock and Mcdonald 2007</a>).</p><p id="p-9">The related wheat pathogen <em>Pyrenophora tritici-repentis</em> also produces a battery of effectors. The majority of isolates produce a version of ToxA, here referred to as <em>PtrToxA</em> (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Ciuffetti <em>et al.</em> 1997</a>; <a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Friesen <em>et al.</em> 2006</a>). We presented several lines of evidence that a <em>ToxA</em> gene in <em>S. nodorum</em> was laterally transferred to <em>P. tritici-repentis</em> shortly before 1941 (<a id="xref-ref-23-4" class="xref-bibr" href="#ref-23">Friesen <em>et al.</em> 2006</a>). The <em>SnToxA</em> sequence in <em>S. nodorum</em> was present on a scaffold of 32.4 kb. A central section of 11 kb containing <em>SnToxA</em> was highly homologous and colinear with the <em>PtrToxA</em> containing sequence. The 11 kb also contained a gene for a DNA transposase. Repetitive DNA was found on either side of the 11 kb whereupon an easily recognized level of homology was lost. This finding suggested that the <em>ToxA</em> gene was transferred along with at least 11 kb of DNA and possibly as part of a transposon (<a id="xref-ref-23-5" class="xref-bibr" href="#ref-23">Friesen <em>et al.</em> 2006</a>). The publication of the <em>P. tritici-repentis</em> Pt-1c genome (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Manning <em>et al.</em> 2013</a>) and the sequences of strains of both species that lack <em>ToxA</em> allows us to examine this phenomenon in more detail.</p><p id="p-10">In this study we have used second-generation sequencing to determine the DNA sequence of two further strains of <em>S</em>. <em>nodorum</em>. One strain—Sn4—was isolated from wheat in the United States. It is known to produce a different complement of effectors to SN15. The second strain, Sn79-1087 (hereafter Sn79), was isolated from the grassy weed <em>Agropyron</em> (<a id="xref-ref-23-6" class="xref-bibr" href="#ref-23">Friesen <em>et al.</em> 2006</a>). The Sn79 strain is essentially avirulent on wheat and Sn79 culture filtrates produce no reactions on a diverse panel of wheat cultivars (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/6/959/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The wheat varieties Kulm and Selkirk infected with S. nodorum strains Sn79, SN15, and Sn4." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-510994478" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The wheat varieties Kulm and Selkirk infected with &lt;em&gt;S. nodorum&lt;/em&gt; strains Sn79, SN15, and Sn4.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/6/959/F1.medium.gif" width="440" height="146"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/3/6/959/F1.medium.gif" width="440" height="146"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/6/959/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/6/959/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462955" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-11" class="first-child">The wheat varieties Kulm and Selkirk infected with <em>S. nodorum</em> strains Sn79, SN15, and Sn4.</p><div class="sb-div caption-clear"></div></div></div><p id="p-12">Comparison of the genome sequences of these two additional <em>S. nodorum</em> strains was undertaken to identify further effectors present in Sn4 or SN15 but absent in Sn79 and to explore the genetic differences in each strain that underpins their differing pathogenicity profiles.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Strains, infection, DNA preparation, and sequencing</h3><p id="p-13"><em>S. nodorum</em> strains Sn4 (<a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">Zhang <em>et al.</em> 2011</a>) and Sn79 (<a id="xref-ref-23-7" class="xref-bibr" href="#ref-23">Friesen <em>et al.</em> 2006</a>) were grown in Fries3 with 30 mM glutamate for 1 wk. Spores were harvested and used to infect detached leaves of wheat cultivars Kulm and Selkirk as previously described (<a id="xref-ref-23-8" class="xref-bibr" href="#ref-23">Friesen <em>et al.</em> 2006</a>). For DNA extraction, mycelium was harvested with a milk filter and cells lysed with a bead crusher. The concentrations of DNA extracted using the CTAB protocol (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Doyle and Doyle 1987</a>) were measured to be 364 ng/mL for the Sn4 extraction and 223 ng/mL for Sn79.</p></div><div id="sec-3" class="subsection"><h3>Mapping</h3><p id="p-14">Raw reads were filtered using one round of cutadapt v1.0 (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Martin 2011</a>) using a Phred quality threshold of 28, trimming known Illumina adapters from the 3′ end. Trimmed reads were mapped to the reference sequence using BWA 0.5.7 (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Li and Durbin 2009</a>), allowing for three gap openings and 21-bp gap extensions per read to allow for large indels. Relatively loose mapping parameters were chosen to reduce the probability of false positive low-coverage regions. Reads mapping around identified indels were locally realigned with GATK 1.2.4 (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">McKenna <em>et al.</em> 2010</a>) using the RealignerTargetCreator and IndelRealigner modules to minimize the likelihood of false single-nucleotide polymorphisms (SNPs).</p><p id="p-15">Mapping depth overviews were visualized as Hilbert curves using the HilbertVis v1.15 (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Anders 2009</a>), Rsamtools 1.0, and Bioconductor 2.6 (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Gentleman <em>et al.</em> 2004</a>) packages. Details of the Hilbert curve generation are available in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004994/-/DC1/004994SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004994/-/DC1/FigureS1.pdf">Figure S1</a>. Regions of low coverage and consecutive gene absence were calculated using the genomeCoverageBed and intersectBed utilities of BEDTools 2.12.0 (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Quinlan and Hall 2010</a>). Regions of low coverage were defined as parts of the reference genome with mapped read depth of 5 or less. Scaffold gaps of poly-‘N’ in the reference sequence were removed from the list of low coverage regions as no reads can map to these loci.</p></div><div id="sec-4" class="subsection"><h3>Assembly</h3><p id="p-16">In addition to mapping the reads from Sn4 and Sn79 back to the reference SN15 genome, each genome was assembled <em>de novo</em> with Velvet v1.0.10 (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Zerbino and Birney 2008</a>) using VelvetOptimizer 2.2.0 to find the assembly with optimal N50 length.</p></div><div id="sec-5" class="subsection"><h3>Mitochondrial sequence</h3><p id="p-17">The mitochondrial genomes for Sn4 and Sn79 were considered separately to the nuclear genome. Sequences for the new mitochondrial genomes were inferred from the consensus of reads mapped by BWA to the SN15 mitochondrial genome. Two instances of manual gap removal and contig joining were required to finish the assembly. The velvet assembly was cleaned of mitochondrial sequence by aligning the velvet contigs to the SN15 reference with Mauve 2 using default parameters and removing contigs that aligned with the mitochondrial sequence. The removed mitochondrial contigs were manually checked for major structural changes not present in BWA-mapped consensus to validate the consensus assembly.</p></div><div id="sec-6" class="subsection"><h3>Annotation</h3><p id="p-18">Draft gene models were generated with MAKER2 (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cantarel <em>et al.</em> 2008</a>) using evidence from a SNAP (version 2010-07-28) hidden Markov model generated from the validated SN15 gene structures, a Genemark ES 2.3 hidden Markov model (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Ter-Hovhannisyan <em>et al.</em> 2008</a>) trained on each of the new genome sequences, and mRNA sequences from the set of validated SN15 genes. Complete predicted proteins were clustered with the validated SN15 reference proteins using OrthoMCL 2.0.2 (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Li <em>et al.</em> 2003</a>), using an mcl expansion parameter of 1.5 and a maximum evalue of 1e-5 for the blastp search v2.2.27+.</p></div><div id="sec-7" class="subsection"><h3>Repeats</h3><p id="p-19">Known <em>Stagonospora</em> repeats (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Hane <em>et al.</em> 2007</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Hane and Oliver 2008</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Hane and Oliver 2010</a>); transposons Molly [Genbank:<a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=AJ488502.1&amp;atom=%2Fggg%2F3%2F6%2F959.atom">AJ488502.1</a>], Pixie [Genbank:<a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=AJ488503.1&amp;atom=%2Fggg%2F3%2F6%2F959.atom">AJ488503.1</a>], and Elsa [Genbank:<a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=AJ277966.1&amp;atom=%2Fggg%2F3%2F6%2F959.atom">AJ277966.1</a>]; and simple repeats were identified using RepeatMasker v3.2.8 and crossmatch (from phrap/cross_match/swat ver 1.090518) (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Smit <em>et al.</em> 1996-2004</a>) in the Sn4 and Sn79 genome sequences using default parameters.</p></div><div id="sec-8" class="subsection"><h3>Repeat-induced point mutation (RIP) analysis</h3><p id="p-20">RIP is a fungal-specific process in which duplicated sequences are mutated during meiosis (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Selker <em>et al.</em> 1987</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Selker 1990</a>). In Pezizomycotina, RIP modifies CpN dinucleotides and changes are predominantly in the direction CpA to TpA (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Cambareri <em>et al.</em> 1989</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Clutterbuck 2011</a>). RIPCAL identifies regions likely to have undergone RIP by alignment of similar sequences and scanning for nucleotide changes that are characteristic of RIP (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Hane and Oliver 2010</a>). Thus, RIPcal can be used to identify both the presence and direction of a RIP-like sequence change. <em>S. nodorum</em> SN15 scaffolds were identified that showed sequence similarity to the <em>ToxA</em> region in <em>P. tritici-repentis</em> by blat v34 (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Kent 2002</a>) using parameters minIdentity = 70 and minScore = 200. For each blat matched region, the nucleotide sequence was extracted from each species’ scaffold and aligned with ClustalW v2.1 (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Larkin <em>et al.</em> 2007</a>). A 250-bp nonoverlapping sliding window was moved across the concatenated alignments and RIP-like mutations of CpN → TpN were identified from <em>Pyrenophora</em> to <em>Stagonospora</em> and then from <em>Stagonospora</em> to <em>Pyrenophora</em> using RIPCAL.</p></div><div id="sec-9" class="subsection"><h3>Phylogenetic analysis</h3><p id="p-21"><em>S. nodorum</em> Sn4, Sn79, SN15, <em>P. tritici-repentis</em>, and <em>L. maculans</em> protein sequences for the AFTOL2 (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Celio <em>et al.</em> 2006</a>) genes <em>CDC47</em>, <em>KRR1</em>, <em>CCT4</em>, <em>GCD6</em>, <em>GPI8</em>, <em>MCM3</em>, <em>COX15</em>, <em>RAD3</em>, <em>DBP3</em>, <em>NBP35</em>, <em>CHC1</em>, and <em>RBG2</em> were identified in the resequenced strains using blast. Homologous genes from the three strains were aligned using MUSCLE 3.6 (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Edgar 2004</a>) using default parameters. The alignments were then manually trimmed and concatenated to provide regions present in all five genomes. The concatenated alignments were used to construct phylogenetic trees using the neighbor joining algorithm provided by Geneious (Drummond <em>et al.</em> 2011), using the Jukes-Cantor model for genetic distance and <em>L. maculans</em> as an outgroup. A consensus tree from 20 replicates was built using a support threshold of 50%.</p></div><div id="sec-10" class="subsection"><h3>Effector candidates</h3><p id="p-22">Each of the validated SN15 proteins was assessed against a series of normative assumptions about the characteristics of proteinaceous effectors (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Small proteins (≤30 kD, calculated using BioRuby 1.4.2 (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Goto <em>et al.</em> 2010</a>) were scored. Mean cysteine content as a percentage of total protein length was calculated and proteins were scored if they had cysteine percentage greater than one standard deviation from the mean. Proteins were scored if they were encoded by genes within 5 kb of repeats. Repetitive sequences are likely to result in contig breaks in the final assembly, so scaffold ends were treated as equivalent to repeats for this scoring step. Proteins were scored if they had no blast hits to the National Center for Biotechnology Information nonredundant protein database (NR) after we excluded <em>S. nodorum</em> proteins, using a minimum e-value cutoff of 1e-20. Proteins were scored if WolfPSort v0.2 (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Horton <em>et al.</em> 2007</a>) predicted the protein to be located extracellularly, or SignalP 3.0 (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bendtsen <em>et al.</em> 2004</a>) identified a signal peptide in the sequence. WolfPSort was run with “fungi” as the organism source. SignalP was run with default parameters. Proteins with coding sequence coverage by Sn4 reads of at least 20% were scored, and proteins with coding sequence coverage by Sn79 reads &lt;20% were scored. Evidence of positive selection for each gene was calculated by taking Sn4 SNPs identified in coding regions and measuring dN/dS using the CNFGTR model (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Yap <em>et al.</em> 2010</a>) as implemented in PyCogent 1.5.1 (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Knight <em>et al.</em> 2007</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462845/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462845/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Criteria used to identify effector candidates</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section results" id="sec-11"><h2 class="">Results</h2><div id="sec-12" class="subsection"><h3>Cultivar response</h3><p id="p-24">We chose two strains for resequencing. Sn79 was isolated from Agropyron and produces no significant disease on any wheat cultivar tested (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). Infiltration of Sn79 culture filtrate into wheat cultivars produced no reaction but transformation of ToxA, Tox1, or Tox3 into Sn79 rendered the strain capable of causing disease and necrosis on wheat lines carrying sensitivity loci for the respective effector (<a id="xref-ref-23-9" class="xref-bibr" href="#ref-23">Friesen <em>et al.</em> 2006</a>; <a id="xref-ref-42-5" class="xref-bibr" href="#ref-42">Liu <em>et al.</em> 2009</a>, <a id="xref-ref-43-5" class="xref-bibr" href="#ref-43">2012</a>). Sn4 is a virulent wheat isolate that is known to produce additional effectors compared with SN15 (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Faris <em>et al.</em> 2011</a>). <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a> shows the more pronounced necrosis in response to Sn4 than both SN15 and Sn79 on wheat cultivars Kulm and Selkirk which suggests the presence of effectors in Sn4 that are absent from SN15.</p></div><div id="sec-13" class="subsection"><h3>Sequencing</h3><p id="p-25">DNA sequencing of 35-bp and 75-bp single-end Illumina read libraries yielded 7,097,630 and 13,783,290 reads, respectively, for strain Sn4 and 6,685,623 reads and 15,615,628 reads for Sn79. Assuming the newly sequenced genome sizes are equal to that of SN15 assembly (37.2 Mbp), the sequencing data provided 34-38x coverage.</p><p id="p-26">The reads for each new strain were assembled using Velvet final contigs are available from NCBI as bioproject accessions PRJNA170815 (Sn4) and PRJNA170816 (Sn79). A summary of the genome assembly and mapping statistics is given in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>. As expected, the Illumina-sequenced genomes were more fragmented than the Sanger-sequenced reference strain. Commonly reported metrics of assembly contiguity are N50 and L50 (N50 length). The N50 is the smallest number of sequences that contain 50% of the total assembly length and the L50 is the length of the smallest sequence within this subset. The fragmentation of the short-read assemblies has resulted in L50s of 22 kbp and 17 kbp for Sn4 and Sn79 compared with 1.045 Mbp for SN15 and contig counts of 2559 and 3132 for Sn4 and Sn79 compared to 107 scaffolds for SN15. The assembly sizes of Sn4 and Sn79 are comparable (34.6 Mbp and 33.8 Mbp) but smaller than SN15 (37.2 Mbp) because of reduced representation of their repetitive DNA content.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462849/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462849/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Genome overview. The resequenced genome assemblies are rich in genic regions</span><div class="sb-div caption-clear"></div></div></div><p id="p-29">The process of <em>de novo</em> assembly with short reads is likely to collapse highly similar repeat units and assemble only divergent repeat copies as well as single copy regions. Fragments of known <em>S. nodorum</em> repeats (<a id="xref-ref-32-4" class="xref-bibr" href="#ref-32">Hane <em>et al.</em> 2007</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Hane and Oliver 2008</a>, <a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">2010</a>), were identified in the new assemblies. The short-read assemblies contain 8.5% (Sn4) and 8.4% (Sn79) of the total repeat content of SN15 (data not shown). All but one of the SN15 repeat families was detected as fragments in the new assemblies. The exception was the telomere-associated repeat R22, which was not detected in the Sn4 assembly. The SN15 genome contains 35.48 Mbp of nonrepetitive nuclear DNA, comparable with the assembly sizes of Sn4 (34.6 Mbp) and Sn79 (33.8 Mbp). We therefore tentatively conclude that all three genomes have similar genome sizes.</p></div><div id="sec-14" class="subsection"><h3>Gene calling</h3><p id="p-30">We have accumulated substantial evidence for the validation of 12,383 genes in SN15 (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Bringans <em>et al.</em> 2009</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Casey <em>et al.</em> 2010</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Ipcho <em>et al.</em> 2012</a>) and demoted 3598 genes to “nonvalidated” status (<a id="xref-ref-32-5" class="xref-bibr" href="#ref-32">Hane <em>et al.</em> 2007</a>). Gene calling by GeneMark-ES v2 (<a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Ter-Hovhannisyan <em>et al.</em> 2008</a>) in the new assemblies predicted 14,391 and 14,352 genes in Sn4 and Sn79, respectively. The genes in Sn4 and Sn79 have average feature sizes comparable with the validated SN15 genes and much larger than the discarded SN15 genes (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). Compared with experimentally validated gene models in SN15, the gene models in Sn4 and Sn79 have a slightly larger mean protein size (437aa and 434aa compared with 423aa), a larger median protein size (367aa and 366aa compared with 347aa), fewer exons/gene (2.45 compared with 2.65), and longer mean exon length (515 bp and 510 bp compared with 408 bp). These findings indicate that the new assemblies and gene calls are of comparable accuracy to the SN15 reference and are likely better than the set of low-confidence unvalidated reference genes.</p><p id="p-31">The phylogenetic relationships of the three strains were investigated by comparing 15 single-copy or low-copy genes used routinely for phylogenetic analysis. These genes were aligned together with those of <em>P. tritici-repentis</em> and <em>L. maculans</em>, which constituted an out-group. The phylogenetic tree indicates that each new strain is closely related to SN15 and thus a <em>bona fide S. nodorum</em> sequence. It also indicated that the two wheat pathogens are more closely related to each other than either is to the grass pathogen Sn79 (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). Tree topologies of the individual genes agreed with the tree constructed from the sequence concatenation (data not shown). The three strains are equally distant from both species in the outgroup. This finding is consistent with recent molecular analyses of Pleosporales that place <em>Pyrenophora</em>, <em>Leptosphaeria</em> and <em>Phaeosphaeria</em> (<em>Stagonospora</em>) in three distinct families within the suborder Pleosporinae (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Zhang <em>et al.</em> 2009</a>). Previous authors had placed <em>S. nodorum</em> in the <em>Leptosphaeria</em> genus.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/6/959/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phylogenetic tree constructed from an alignment of the AFTOL proteins CDC47, KRR1, CCT4, GCD6, GPI8, MCM3, COX15, RAD3, DBP3, NBP35, CHC1, and RGB2. Genbank accessions are shown in brackets and consensus support (%) shown at tree nodes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-510994478" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Phylogenetic tree constructed from an alignment of the AFTOL proteins CDC47, KRR1, CCT4, GCD6, GPI8, MCM3, COX15, RAD3, DBP3, NBP35, CHC1, and RGB2. Genbank accessions are shown in brackets and consensus support (%) shown at tree nodes.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/6/959/F2.medium.gif" width="440" height="142"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/3/6/959/F2.medium.gif" width="440" height="142"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/6/959/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/6/959/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462855" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-32" class="first-child">Phylogenetic tree constructed from an alignment of the AFTOL proteins CDC47, KRR1, CCT4, GCD6, GPI8, MCM3, COX15, RAD3, DBP3, NBP35, CHC1, and RGB2. Genbank accessions are shown in brackets and consensus support (%) shown at tree nodes.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Comparative genomics</h3><p id="p-33">The presence and absence of regions in the SN15 genome was analyzed in strains Sn4 and Sn79 via a combination of read-mapping and <em>de novo</em> assembly. <em>De novo</em> assembly was important for the detection of loci that were present in Sn4 or Sn79 but absent in SN15 and thus not detectable by read-mapping.</p><p id="p-34">The new reads were mapped to the genome by BWA. The average mapped read depth in coding regions were 23.2X (Sn4) and 26.2X (Sn79) but was highly variable. Repetitive regions were unreliably accounted for by this procedure, so we only consider low copy number, mainly genic regions here.</p><p id="p-35">Mapping of the reads of each strain to the reference assembly and calculating coverage of each gene’s coding sequence suggested that 180 SN15 genes were absent from Sn4 and 367 SN15 genes were absent from Sn79 (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>) where genes with &lt;5% coding region coverage are defined as absent. Inspection of the pattern of missing genes suggested that in many cases contiguous stretches of genes were absent. Quantification of this phenomenon showed that 57% (Sn4) and 76% (Sn79) of absent genes have a neighboring gene that is also absent. We refer to the contiguous runs of missing genes as sectional gene absence (SGA). The degree of SGA was much greater in Sn79 in both frequency and size. The largest SGA from Sn4 included only five genes, and the largest from Sn79 included 51 genes.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462853/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462853/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title"><em>S</em>. <em>nodorum</em> SN15 genes deemed absent in the newly sequenced strains</span><div class="sb-div caption-clear"></div></div></div><p id="p-37">Genes sectionally absent from Sn79 include all the genes on scaffold 44 (48 genes) and scaffold 45 (51 genes, the largest SGA), and the polyketide synthase gene SNOG_07866 [NCBI Gene: 5975086], which is absent in a cluster of 11 genes at the end of scaffold 11.</p><p id="p-38">This pattern of nonrandom SGA prompted us to explore ways to graphically display this finding. Very long read-depth data can be conveniently visualized as a space filling curve that attempts to preserve proximity during its transformation to an increased number of dimensions. <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> shows Hilbert plots generated using R with the HilbertVis (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Anders 2009</a>) and Bioconductor (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Gentleman <em>et al.</em> 2004</a>) packages (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004994/-/DC1/FigureS1.pdf">Figure S1</a> for more information on Hilbert plots). Only scaffold 2 is shown; scaffolds 1–12 are shown in the supplementary data. White sections in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a> represent those sections of SN15 scaffold 2 that are annotated as coding sequence, whereas B and C represent the mapped read density of Sn4 and Sn79, respectively. White regions in B and C are parts of SN15 scaffold 2 that are covered by Sn4 and Sn79. Large dark boxes 1−3 indicate regions of SN15 scaffold 2 that have no reads mapped from Sn4 and Sn79 and are likely to be absent from the newly sequenced strains.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/6/959/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Hilbert plots of coding sequence and read density of SN15 scaffold 2. (A) White regions correspond to coding regions. S. nodorum Sn4 (B) and Sn79 (C) mapped read density on SN15 scaffold 2. Lighter shades correspond to higher read density. Regions 1 (45&#x2212;46 kbp) and 2 (31 kbp) are absent from both Sn4 and Sn79. Region 3 shows a 48-kbp region absent from Sn79." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-510994478" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Hilbert plots of coding sequence and read density of SN15 scaffold 2. (A) White regions correspond to coding regions. &lt;em&gt;S. nodorum&lt;/em&gt; Sn4 (B) and Sn79 (C) mapped read density on SN15 scaffold 2. Lighter shades correspond to higher read density. Regions 1 (45&#x2212;46 kbp) and 2 (31 kbp) are absent from both Sn4 and Sn79. Region 3 shows a 48-kbp region absent from Sn79.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/6/959/F3.medium.gif" width="440" height="151"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/3/6/959/F3.medium.gif" width="440" height="151"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/6/959/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/6/959/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462949" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-39" class="first-child">Hilbert plots of coding sequence and read density of SN15 scaffold 2. (A) White regions correspond to coding regions. <em>S. nodorum</em> Sn4 (B) and Sn79 (C) mapped read density on SN15 scaffold 2. Lighter shades correspond to higher read density. Regions 1 (45−46 kbp) and 2 (31 kbp) are absent from both Sn4 and Sn79. Region 3 shows a 48-kbp region absent from Sn79.</p><div class="sb-div caption-clear"></div></div></div><p id="p-40">Assembly of the reads <em>de novo</em> allowed us to identify genes not present in the SN15 reference assembly and to estimate the core <em>S. nodorum</em> gene set. Clustering the predicted proteins from the three genomes using OrthoMCL gave a core cluster number estimate of 10,464 (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). The clusters showed a larger conserved gene set between SN15 and Sn4 (430) than between SN15 and Sn79 (246) consistent with the phylogenetic analysis. The large set shared between Sn79 and Sn4 is likely an overestimate as a result of these strains’ shared gene calling procedure. As gene models are improved, the number of genes in this set is likely to fall.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/6/959/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Protein ortholog clusters between the three S. nodorum strains. Proteins were grouped into clusters using orthoMCL (Li et al. 2003). Clusters were regarded as &#x201C;nr-unique&#x201D; if none of its members had blast hits to the nr database at an e-value cutoff of 1e-20." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-510994478" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Protein ortholog clusters between the three &lt;em&gt;S. nodorum&lt;/em&gt; strains. Proteins were grouped into clusters using orthoMCL (Li &lt;em&gt;et al.&lt;/em&gt; 2003). Clusters were regarded as &#x201C;nr-unique&#x201D; if none of its members had blast hits to the nr database at an e-value cutoff of 1e-20.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/6/959/F4.medium.gif" width="440" height="214"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/3/6/959/F4.medium.gif" width="440" height="214"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/6/959/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/6/959/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462865" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-41" class="first-child">Protein ortholog clusters between the three <em>S. nodorum</em> strains. Proteins were grouped into clusters using orthoMCL (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Li <em>et al.</em> 2003</a>). Clusters were regarded as “nr-unique” if none of its members had blast hits to the nr database at an e-value cutoff of 1e-20.</p><div class="sb-div caption-clear"></div></div></div><p id="p-42">The 10,464 protein clusters in the conserved <em>S. nodorum</em> core include many of the products essential to metabolism. Of the protein clusters present in all three strains, 87% had blast hits in NR. A markedly lower percentage of the strain-specific genes (21–50%) had similarity to protein sequences in NR.</p><p id="p-43">Expansion in gene copy number was detected in eight clusters (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). Of the clusters, five contained genes with higher copy numbers in SN15, and three clusters contained genes with greater copy numbers in Sn4. No clusters contained genes expanded in the non-pathogenic strain Sn79. Four of the eight expanded clusters contained genes with no blast hits (e value ≤ 1e–20).</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462848/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462848/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Gene family expansion in the three <em>S. nodorum</em> strains</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>Effector comparison</h3><p id="p-45"><em>S. nodorum</em> Sn79 produces none of the three known <em>S. nodorum</em> effectors: <em>SnTox1</em>, <em>SnToxA</em>, and <em>SnTox3</em> (<a id="xref-ref-23-10" class="xref-bibr" href="#ref-23">Friesen <em>et al.</em> 2006</a>; <a id="xref-ref-42-6" class="xref-bibr" href="#ref-42">Liu <em>et al.</em> 2009</a>, <a id="xref-ref-43-6" class="xref-bibr" href="#ref-43">2012</a>). Effector-containing regions in SN15 were compared with the corresponding regions in Sn4 and Sn79. These known <em>S. nodorum</em> effectors are all absent from the Sn79 assembly (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004994/-/DC1/FigureS2.pdf">Figure S2</a>). In the case of <em>SnTox1</em>, the effector gene and 2 kb of upstream intergenic sequence was absent from Sn79. <em>SnTox3</em> was absent from Sn79 together with the three upstream genes (SNOG_08982 to SNOG_08985; NCBI 5976184 to 5976187). The absent genes included a predicted protein-disulfide isomerase, a short-chain dehydrogenase, and a gene without blast hits. The repetitive regions downstream of <em>SnTox3</em> gene did not assemble in either Sn4 or Sn79.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/6/959/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Effector context in three Stagonospora strains. Black arrows represent effector genes, gray arrows represent other genes, and white lozenges represent repetitive sequence. Gray boxes above and below the SN15 scaffold sections represent sequence present in the assemblies of the resequenced strains. The sequence surrounding SnTox1 is present in all three strains. SnTox3 is absent from Sn79 in a section that includes four genes. The entire SnToxA region is absent in Sn79." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-510994478" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Effector context in three &lt;em&gt;Stagonospora&lt;/em&gt; strains. Black arrows represent effector genes, gray arrows represent other genes, and white lozenges represent repetitive sequence. Gray boxes above and below the SN15 scaffold sections represent sequence present in the assemblies of the resequenced strains. The sequence surrounding &lt;em&gt;SnTox1&lt;/em&gt; is present in all three strains. &lt;em&gt;SnTox3&lt;/em&gt; is absent from Sn79 in a section that includes four genes. The entire &lt;em&gt;SnToxA&lt;/em&gt; region is absent in Sn79.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/6/959/F5.medium.gif" width="440" height="168"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/3/6/959/F5.medium.gif" width="440" height="168"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/6/959/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/6/959/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462934" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-46" class="first-child">Effector context in three <em>Stagonospora</em> strains. Black arrows represent effector genes, gray arrows represent other genes, and white lozenges represent repetitive sequence. Gray boxes above and below the SN15 scaffold sections represent sequence present in the assemblies of the resequenced strains. The sequence surrounding <em>SnTox1</em> is present in all three strains. <em>SnTox3</em> is absent from Sn79 in a section that includes four genes. The entire <em>SnToxA</em> region is absent in Sn79.</p><div class="sb-div caption-clear"></div></div></div><p id="p-47"><em>SnToxA</em> is found on scaffold 55 together with just three more called genes, including a transposase. The remainder of the scaffold comprises sections of gene-free single copy DNA and repetitive elements. The single-copy regions, including a contiguous stretch of 11 kb, were present in Sn4 albeit on different contigs. The 11-kb region was absent in Sn79, but some of the repetitive elements were present on short contigs (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). The presence of <em>ToxA</em> in some strains of both <em>S. nodorum</em> and <em>P. tritici-repentis</em> is suggestive of lateral gene transfer. We earlier reported that the colinear region of 11 kb found in both SN15 and Sn4 is present in <em>P. tritici-repentis</em> with very high levels of similarity (<a id="xref-ref-23-11" class="xref-bibr" href="#ref-23">Friesen <em>et al.</em> 2006</a>). The assembly of <em>P. tritici-repentis</em> Pt-1c using Sanger sequencing aided by optical mapping has recently been published (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Manning <em>et al.</em> 2013</a>). <em>PtrToxA</em> is present on scaffold 4/chromosome 6 of the Pt-1c assembly. Alignment of scaffold 4 to <em>C. heterostrophus</em> indicated that a region of 156 kb including <em>PtrToxA</em> was absent in the corn pathogen but the homologous flanking regions were present in both species (<a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">Manning <em>et al.</em> 2013</a>). A series of SN15 scaffolds [68, 55 (including <em>SnToxA</em>), 51, 46, 64, and 73] appear to be colinear compared with <em>P. tritici-repentis</em> (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). Regions of colinearity were interspersed with repeated sequences inserted into the SN15 genome. Scaffold 46 has 4 such colinear regions interspersed with five repetitive sections. The total length of the colinear DNA in <em>P. tritici-repentis</em> was 72 kb, corresponding to a 350-kb region in SN15 that was inflated by multiple transposon invasions. Read mapping of Sn4 to the SN15 genome indicated that most of the repetitive regions, the 11 kb <em>SnToxA</em> region, and a gene free single copy region on scaffolds 68 and 51 were present in Sn4. In contrast, a single copy region on scaffold 51 with two annotated genes, and the single copy regions on scaffolds 46, 64, and 73 were absent in Sn4. In Sn79 all single copy regions were absent and only traces of the repetitive DNA could be found.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/6/959/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Regions around ToxA in P. tritici-repentis show homology to other SN15 scaffolds. The scaffolds are linked here with blat hits &gt;75% identity. The total size of the expanded predicted laterally transferred DNA region is at least 72 kb." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-510994478" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Regions around &lt;em&gt;ToxA&lt;/em&gt; in &lt;em&gt;P. tritici-repentis&lt;/em&gt; show homology to other SN15 scaffolds. The scaffolds are linked here with blat hits &gt;75% identity. The total size of the expanded predicted laterally transferred DNA region is at least 72 kb.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/6/959/F6.medium.gif" width="440" height="124"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/3/6/959/F6.medium.gif" width="440" height="124"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/6/959/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/6/959/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462930" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-48" class="first-child">Regions around <em>ToxA</em> in <em>P. tritici-repentis</em> show homology to other SN15 scaffolds. The scaffolds are linked here with blat hits &gt;75% identity. The total size of the expanded predicted laterally transferred DNA region is at least 72 kb.</p><div class="sb-div caption-clear"></div></div></div><p id="p-49">The 72 kb of colinear DNA contains 21 genes in <em>P. tritici-repentis</em> but only seven genes in SN15. The hallmarks of RIP have been abundantly found in the SN15 genome but only sparsely in the <em>P. tritici-repentis</em> genome (<a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Hane and Oliver 2008</a>, <a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">2010</a>; <a id="xref-ref-47-4" class="xref-bibr" href="#ref-47">Manning <em>et al.</em> 2013</a>). We extracted and aligned 46 kb of matching regions from the 72-kb <em>ToxA</em> region shown in <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a> and scanned for dinucleotide SNP mutations characteristic of RIP (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>). The dinucleotide changes are predominantly CpA-&gt;TpA mutation, which is characteristic of RIP in Pezizomycotina species. Across most of the region, changes are consistent with the <em>S. nodorum</em> sequence having been subject to RIP. These changes were most likely in the form of RIP leakage. We have previously observed that RIP-like changes, initiated from interspersed repeat regions, are also found in neighboring single-copy regions (<a id="xref-ref-31-5" class="xref-bibr" href="#ref-31">Hane and Oliver 2010</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Van de Wouw <em>et al.</em> 2010</a>). In contrast, one region upstream of the ToxA gene on scaffold 55 had changes characteristic of RIP in the Pt-1c sequence. Subsequences of this region were found in multiple copies in Pt-1c genome, which suggests that the transferred region has been susceptible to RIP in PTR since its lateral transfer, but the majority has remained in single copy and thus has evaded RIP mutation.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/6/959/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Regions of similarity around ToxA in P. tritici-repentis and S. nodorum SN15 identified by blat (Figure 6) were concatenated and aligned. RIP irreversibly converts cytosine bases to thymine, therefore comparisons between two sequences have directionality. The frequency of CpN to TpN mutations across 250-bp nonoverlapping windows has been separated by direction into those with CpN dinucleotides of S. nodorum being converted into TpN in P. tritici-repentis (top) and vice-versa (bottom). CpA to TpA changes that are associated with RIP are indicated by the red line. The location of genes is indicated by black boxes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-510994478" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Regions of similarity around ToxA in &lt;em&gt;P. tritici-repentis&lt;/em&gt; and &lt;em&gt;S. nodorum&lt;/em&gt; SN15 identified by blat (Figure 6) were concatenated and aligned. RIP irreversibly converts cytosine bases to thymine, therefore comparisons between two sequences have directionality. The frequency of CpN to TpN mutations across 250-bp nonoverlapping windows has been separated by direction into those with CpN dinucleotides of &lt;em&gt;S. nodorum&lt;/em&gt; being converted into TpN in &lt;em&gt;P. tritici-repentis&lt;/em&gt; (top) and vice-versa (bottom). CpA to TpA changes that are associated with RIP are indicated by the red line. The location of genes is indicated by black boxes.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/6/959/F7.medium.gif" width="440" height="209"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/3/6/959/F7.medium.gif" width="440" height="209"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/6/959/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/6/959/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462859" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-50" class="first-child">Regions of similarity around ToxA in <em>P. tritici-repentis</em> and <em>S. nodorum</em> SN15 identified by blat (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6</a>) were concatenated and aligned. RIP irreversibly converts cytosine bases to thymine, therefore comparisons between two sequences have directionality. The frequency of CpN to TpN mutations across 250-bp nonoverlapping windows has been separated by direction into those with CpN dinucleotides of <em>S. nodorum</em> being converted into TpN in <em>P. tritici-repentis</em> (top) and vice-versa (bottom). CpA to TpA changes that are associated with RIP are indicated by the red line. The location of genes is indicated by black boxes.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h3>Mating type locus</h3><p id="p-51">The mating type idiotype of SN15 is <em>Mat1-1</em> and of Sn4 and Sn79 is <em>Mat1-2</em>. These idiomorphs have previously been sequenced (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bennett <em>et al.</em> 2003</a>). Sn4 and Sn79 both contained copies of the <em>MAT1-2</em> idiomorph (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a>). The mating type genes show perfect sequence identity to published sequences. There is a 19-bp region present in the Sn79 <em>Mat1-2</em> locus that is absent in both the Sn4 and sn436GA98 sequences.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/6/959/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The MAT locus in 5 Stagonospora strains showing the two idiomorphs. Strains sn435PL98 and SN15 are mating type 1 (MAT1-1) and strains Sn4, Sn79, and sn436GA98 are mating type 2 (MAT1-2). Light gray boxes represent nucleotide sequence, and dark gray boxes connect regions with sequence similarity." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-510994478" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The MAT locus in 5 &lt;em&gt;Stagonospora&lt;/em&gt; strains showing the two idiomorphs. Strains sn435PL98 and SN15 are mating type 1 (&lt;em&gt;MAT1-1&lt;/em&gt;) and strains Sn4, Sn79, and sn436GA98 are mating type 2 (&lt;em&gt;MAT1-2&lt;/em&gt;). Light gray boxes represent nucleotide sequence, and dark gray boxes connect regions with sequence similarity.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/6/959/F8.medium.gif" width="440" height="164"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/3/6/959/F8.medium.gif" width="440" height="164"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/6/959/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/6/959/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462900" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-52" class="first-child">The MAT locus in 5 <em>Stagonospora</em> strains showing the two idiomorphs. Strains sn435PL98 and SN15 are mating type 1 (<em>MAT1-1</em>) and strains Sn4, Sn79, and sn436GA98 are mating type 2 (<em>MAT1-2</em>). Light gray boxes represent nucleotide sequence, and dark gray boxes connect regions with sequence similarity.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-18" class="subsection"><h3>Effector candidate selection</h3><p id="p-53">The resequencing data were combined with data from existing SN15 gene models to predict likely effector proteins, based on a set of normative assumptions about the qualities of effector molecules and their genes both in <em>S. nodorum</em> (<a id="xref-ref-23-12" class="xref-bibr" href="#ref-23">Friesen <em>et al.</em> 2006</a>; <a id="xref-ref-42-7" class="xref-bibr" href="#ref-42">Liu <em>et al.</em> 2009</a>, <a id="xref-ref-43-7" class="xref-bibr" href="#ref-43">2012</a>) and other fungal pathogens (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">De Wit <em>et al.</em> 2009</a>) (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Some criteria were assessed before this sequencing project. These include size, cysteine content, proximity to repetitive elements, and prediction of secretion. Addition of data from this resequencing project allowed us to include criteria reporting gene absence or difference between the three strains.</p><p id="p-54">Because Sn79 lacks detectable wheat effectors, we sought genes that were absent in this strain or highly divergent compared to SN15. Genes with &lt;20% of their coding sequence covered by Sn79 reads (1943 genes) were scored in this criterion. The three known effector genes exist in different isoforms in different strains (<a id="xref-ref-23-13" class="xref-bibr" href="#ref-23">Friesen <em>et al.</em> 2006</a>; <a id="xref-ref-42-8" class="xref-bibr" href="#ref-42">Liu <em>et al.</em> 2009</a>, <a id="xref-ref-43-8" class="xref-bibr" href="#ref-43">2012</a>). To exploit this, we sought genes that were present in Sn4 but contained sequence variation compared withSN15; a total of 6507 genes fit this criterion. Sequence comparison of many effector genes shows evidence of accelerated evolution. We therefore screened for genes with elevated dN/dS ratios when comparing Sn4 and SN15. A total of 729 genes showed evidence of high dN/dS.</p><p id="p-55">Each of the criteria matched between 729 and 9119 genes, providing a filter that reduced 12,382 gene models to 159 candidates that matched at least six of the criteria (<a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>). Known effectors <em>SnTox1</em> and Sn<em>ToxA</em> were ranked first and equal third according to these criteria (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004994/-/DC1/TableS1.xls">Table S1</a>).</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462852/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462852/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Counts of proteins matching the eight criteria from <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a></span><div class="sb-div caption-clear"></div></div></div><p id="p-57">Proteins from the Sn4 genome not observed in SN15 were also scored for likely effector properties including small size (≤30 kD), predicted to be secreted, and cysteine-rich. Of the predicted proteins present in the 981 clusters unique to Sn4, four fulfill all three of the criteria given previously (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004994/-/DC1/FileS2.zip">File S2</a>).</p></div><div id="sec-19" class="subsection"><h3>Mitochondrial genomes</h3><p id="p-58">The mitochondrial genomes of Sn4 and Sn79 were considered separately to the nuclear genome. Sequences for the new mitochondria were assembled by taking the consensus sequence of the BWA-mapped reads of the new strains and removing mitochondrial sequences from the Velvet-assembled contigs.</p><p id="p-59">Alignment of the mitochondrial genomes of Sn4 and SN15 showed entirely conserved gene content and order with no indels larger than 3 bp. Two sections of the SN15 and Sn4 mitochondrial genomes were absent in the Sn79 genome (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004994/-/DC1/FigureS3.pdf">Figure S3</a>). These missing sections included an intronic endonuclease and two downstream open reading frames of unknown function. The total mitochondrial sequence discrepancy was 7.43 kb absent from Sn79 but present in Sn4 and SN15.</p></div></div><div class="section discussion" id="sec-20"><h2 class="">Discussion</h2><p id="p-60">Fungal plant pathogenic species are notoriously variable in their morphology and growth characteristics; pathology studies have traditionally focused on differences in virulence and for some species these have been formalized via the definition of <em>forma specialis</em>, pathotypes or races. Exploration of the genetics of the virulence differences has proved a fruitful way to understand the molecular bases of virulence properties. Genome sequences of fungal plant pathogens have been obtained for a handful of species using Sanger-based technologies (<a id="xref-ref-53-3" class="xref-bibr" href="#ref-53">Oliver <em>et al.</em> 2012</a>). The advent of second-generation sequencing technologies has opened up the possibility of sequencing different strains and thereby directly measuring genetic differences.</p><p id="p-61">Early studies of <em>S. nodorum</em> readily identified significant genotypic differences in strains based on expressed sequence tag, restriction fragment length polymorphism, and chromosome-length polymorphism studies (<a id="xref-ref-53-4" class="xref-bibr" href="#ref-53">Oliver <em>et al.</em> 2012</a>) but these differences were not related to virulence properties. It was not until the discovery of necrotrophic effectors that a rational basis for differentiating strains was elaborated (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Friesen <em>et al.</em> 2008</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Tan <em>et al.</em> 2010</a>; <a id="xref-ref-53-5" class="xref-bibr" href="#ref-53">Oliver <em>et al.</em> 2012</a>). Proteins purified from culture filtrates of different isolates induced necrosis in different wheat cultivars. Sensitivity to the proteins mapped to QTL for disease susceptibility that differed between isolates. One strain, isolated from a wild grass, produced neither observable effectors nor disease on wheat. These strains and the new necrotrophic effector rationale provided sufficient incentive for resequencing.</p><p id="p-62">The process of generating second generation sequencing data for two further strains was affordable but presented significant challenges to analyze the data. It was clear that the new strains genomes differed markedly from the reference SN15 strain. These differences were explored by mapping reads onto the reference and by examining <em>de novo</em> assemblies.</p><p id="p-63">Both methods showed that many genes differed between the strains or were entirely absent. The mapping data suggests that Sn4/Sn79 lacked at least 180 of 367 genes, respectively, compared with SN15 (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>). In addition 6507 Sn4 genes differed in nucleotide coding sequence (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). By any criteria, these are very large strain to strain differences which stretch the definition and technology of resequencing.</p><p id="p-64">The gene complement differences were not randomly located in the genome but were predominantly in the form a contiguous sections containing two to 50 or more absent genes. We refer to this pattern as SGA. These genes include a high proportion without clear homologs in other species. One region absent in Sn79 on scaffold_11:124,000-127,500 comprises a cluster of genes that could well function in synthesizing a secondary metabolite involved in pathogenicity on wheat.</p><p id="p-65">This pattern may resemble the presence in <em>M. graminicola</em> and <em>Fusarium oxysporum</em> of entire chromosomes that are unevenly harbored by different strains (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Ma <em>et al.</em> 2010</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Goodwin <em>et al.</em> 2011</a>). These species contain dispensable chromosomes with relatively low gene density, a high frequency of genes without significant matches to known genes and high repetitive element content. In <em>S. nodorum</em> strain-to-strain SGA sometime comprised entire scaffolds. Pulsed field gel and genetic mapping evidence suggests that there are about 20 chromosomes in <em>S. nodorum</em> (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Cooley and Caten 1991</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Malkus <em>et al.</em> 2009</a>), whereas the SN15 assembly contains 107 scaffolds. The lack of a finished SN15 genome assembly means that it is not currently possible to determine the chromosomal location of the SGA regions.</p><p id="p-66">Clustering the predicted protein datasets of each strain can estimate the core proteins conserved between them. This approach gave a minimum conserved gene set of 10,464 protein clusters present in all three strains and an additional 430 conserved between both wheat pathogenic strains.</p><div id="sec-21" class="subsection"><h3>Mitochondrial sequence</h3><p id="p-67">The mitochondrial genomes of Ascomycete fungi contain a well-conserved set of genes but a variable amount of additional DNA (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Hane <em>et al.</em> 2011</a>). This additional DNA can add 120 kb to the core ca. 40 kb of the genome (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Rouxel <em>et al.</em> 2011</a>). This additional DNA includes genes or unknown function and intronic genes encoding homing endonucleases. The mitochondrial genome of SN15 is 49,761 bp and includes three open reading frames (ORFs) of unknown function (ORF 1, 2, and 3) and four intronic endonucleases. The mitochondrial genome of Sn4 was found to be essentially identical, but the Sn79 genome differed significantly in one region (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004994/-/DC1/FigureS3.pdf">Figure S3</a>) lacking the intron found in <em>atp6</em> and a region containing ORF 1 and 2. This pattern is consistent with the insertion of ORF3 and three introns into the ancestor of all three strains and the subsequent insertions of the <em>atp6</em> intron and the region containing ORF 1 and 2 into the lineage containing Sn4 and SN15. This scenario is consistent with the phylogeny based on nuclear DNA sequences (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). The detection of the <em>atp6</em> intron and the region containing ORF 1 and 2 provides a polar and datable marker for evolutionary studies of the fungus.</p></div><div id="sec-22" class="subsection"><h3>Effector prediction</h3><p id="p-68">The main reason for carrying out this study was to facilitate the identification of effectors from the fungal genomes. Experience from a range of pathogens has generated a set of criteria that can be used to select effector candidates for experimental validation. These criteria (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Tables 1</a> and <a id="xref-table-wrap-5-2" class="xref-table" href="#T5">5</a>) were used to filter and rank 12382 genes so that they can be efficiently prioritized for functional characterization. This process predicted the subsequently verified effector <em>SnTox1</em> that conforms to all eight criteria (<a id="xref-ref-43-9" class="xref-bibr" href="#ref-43">Liu <em>et al.</em> 2012</a>). <em>SnTox3</em> and <em>SnToxA</em> match 5 and 6 of the criteria respectively indicating that we still have an imperfect knowledge of effectors’ properties.</p></div><div id="sec-23" class="subsection"><h3>Context of effector genes</h3><p id="p-69">All three known effector genes were absent from Sn79. Comparisons of the three genomes can be used to infer hypotheses about the evolutionary history of these genes (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>) albeit the short contig lengths limit the conclusions. <em>SnTox1</em> is absent from Sn79 but the flanking sequences are homologous in the other two strains. <em>SnTox3</em> is absent from Sn79 along with a section of at least three genes. Both of these scenarios are consistent with a horizontal gene transfer hypothesis (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Oliver <em>et al.</em> 2008</a>) although gene loss from Sn79 is also a possibility; the source of the genes is unknown.</p><p id="p-70">Multiple studies provide evidence supporting recent horizontal transfer for <em>ToxA</em> from <em>S. nodorum</em> to <em>P. tritici-repentis</em> (<a id="xref-ref-23-14" class="xref-bibr" href="#ref-23">Friesen <em>et al.</em> 2006</a>) and earlier from an unknown source into <em>S. nodorum</em> (<a id="xref-ref-61-3" class="xref-bibr" href="#ref-61">Stukenbrock and McDonald 2007</a>). The new data in this paper and the <em>P. tritici-repentis</em> genome sequence (<a id="xref-ref-47-5" class="xref-bibr" href="#ref-47">Manning <em>et al.</em> 2013</a>) allow us to speculate on the history of this critical region. Sn79 lacks <em>ToxA</em> along with a long section (at least 72 kb) of single copy DNA. This finding suggests that <em>ToxA</em> was transferred into the ancestor of both wheat pathogenic strains along with at least this length of DNA (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6</a>). Subsequently, in some strains such as SN15, this section has been invaded by transposons and subjected to RIP (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7</a>). The most parsimonious hypothesis for the source and structure of the <em>ToxA</em> region in <em>P. tritici-repentis</em> is an <em>S. nodorum</em> strain that had not yet suffered this transposon invasion and RIP. The recent acquisition, the homothallic nature of <em>P. tritici-repentis</em> and the scarcity of RIP have resulted in the monomorphism of <em>PtrToxA</em> (<a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">Tan <em>et al.</em> 2012</a>) and the retention of genes lost to RIP in <em>S. nodorum</em>. The presence of both <em>ToxA</em>-expressing and nonexpressing strains of both species in different proportions in different parts of the world (<a id="xref-ref-61-4" class="xref-bibr" href="#ref-61">Stukenbrock and McDonald 2007</a>) can be accounted for by asexual propagation, sexual crossing and differential selection by wheat cultivars depending on whether they express the recognition gene <em>Tsn1</em> (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Oliver <em>et al.</em> 2009</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Antoni <em>et al.</em> 2010</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Faris <em>et al.</em> 2010</a>). Confirmation and elaboration of these hypotheses must await the assembly of a larger sample of genome sequences from both species.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-71">We thank Bruce McDonald, Eva Stukenbrock, Viola Manning, and Lynda Ciuffetti for many insightful discussions. This research was supported by the Australian Grains Research and Development Corporation. The work was supported by iVEC through the use of advanced computing resources located at iVEC@Murdoch and by an award under the Merit Allocation Scheme on the NCI National Facility at the Australian National University.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004994/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004994/-/DC1</a></p></li><li class="fn" id="fn-8"><p id="p-72"><em>Communicating editor: N. D. Young</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2012-11-28"><span class="received-label">Received </span>November 28, 2012.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2013-04-01"><span class="accepted-label">Accepted </span>April 1, 2013.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2013 Syme <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.1" data-doi="10.1007/s00122-009-1163-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abeysekara</span>  <span class="cit-name-given-names">N. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friesen</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keller</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faris</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Identification and characterization of a novel host-toxin interaction in the wheat-<em>Stagonospora nodorum</em> pathosystem.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">120</span>: <span class="cit-fpage">117</span>–<span class="cit-lpage">126</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DAbeysekara%26rft.auinit1%253DN.%2BS.%26rft.volume%253D120%26rft.issue%253D1%26rft.spage%253D117%26rft.epage%253D126%26rft.atitle%253DIdentification%2Band%2Bcharacterization%2Bof%2Ba%2Bnovel%2Bhost-toxin%2Binteraction%2Bin%2Bthe%2Bwheat-Stagonospora%2Bnodorum%2Bpathosystem.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-009-1163-6%26rft_id%253Dinfo%253Apmid%252F19816671%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-009-1163-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19816671&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F959.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.2" data-doi="10.1094/PHYTO-02-11-0040"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abeysekara</span>  <span class="cit-name-given-names">N. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faris</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chao</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McClean</span>  <span class="cit-name-given-names">P. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friesen</span>  <span class="cit-name-given-names">T. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Whole-genome QTL analysis of <em>Stagonospora nodorum</em> blotch resistance and validation of the SnTox4-Snn4 interaction in hexaploid wheat.</span> <abbr class="cit-jnl-abbrev">Phytopathology</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">94</span>–<span class="cit-lpage">102</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhytopathology%26rft.stitle%253DPhytopathology%26rft.aulast%253DAbeysekara%26rft.auinit1%253DN.%2BS.%26rft.volume%253D102%26rft.issue%253D1%26rft.spage%253D94%26rft.epage%253D104%26rft.atitle%253DWhole-genome%2BQTL%2Banalysis%2Bof%2BStagonospora%2Bnodorum%2Bblotch%2Bresistance%2Band%2Bvalidation%2Bof%2Bthe%2BSnTox4-Snn4%2Binteraction%2Bin%2Bhexaploid%2Bwheat.%26rft_id%253Dinfo%253Adoi%252F10.1094%252FPHYTO-02-11-0040%26rft_id%253Dinfo%253Apmid%252F21864084%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/PHYTO-02-11-0040&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21864084&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F959.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.3" data-doi="10.1093/bioinformatics/btp152"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anders</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Visualization of genomic data with the Hilbert curve.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1231</span>–<span class="cit-lpage">1235</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp152%26rft_id%253Dinfo%253Apmid%252F19297348%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTAvMTIzMSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNi85NTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.4" data-doi="10.1071/AP09056"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Antoni</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rybak</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tucker</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hane</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Solomon</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Ubiquity of ToxA and absence of ToxB in Australian populations of <em>Pyrenophora tritici</em>-repentis.</span> <abbr class="cit-jnl-abbrev">Australas. Plant Pathol.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">63</span>–<span class="cit-lpage">68</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAustralas.%2BPlant%2BPathol.%26rft.volume%253D39%26rft.spage%253D63%26rft_id%253Dinfo%253Adoi%252F10.1071%252FAP09056%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1071/AP09056&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.5" data-doi="10.1016/j.jmb.2004.05.028"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bendtsen</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">von Heijne</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brunak</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Improved prediction of signal peptides: SignalP 3.0.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">340</span>: <span class="cit-fpage">783</span>–<span class="cit-lpage">795</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DBendtsen%26rft.auinit1%253DJ.%2BD.%26rft.volume%253D340%26rft.issue%253D4%26rft.spage%253D783%26rft.epage%253D795%26rft.atitle%253DImproved%2Bprediction%2Bof%2Bsignal%2Bpeptides%253A%2BSignalP%2B3.0.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2004.05.028%26rft_id%253Dinfo%253Apmid%252F15223320%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2004.05.028&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15223320&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F959.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222734300011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.6" data-doi="10.1016/S1087-1845(03)00062-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bennett</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yun</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">T. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turgeon</span>  <span class="cit-name-given-names">B. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arseniuk</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Identity and conservation of mating type genes in geographically diverse isolates of <em>Phaeosphaeria nodorum</em>.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Biol.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">25</span>–<span class="cit-lpage">37</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2Bgenetics%2Band%2Bbiology%2B%253A%2B%2BFG%2B%2526%2BB%26rft.stitle%253DFungal%2BGenet%2BBiol%26rft.aulast%253DBennett%26rft.auinit1%253DR.%2BS.%26rft.volume%253D40%26rft.issue%253D1%26rft.spage%253D25%26rft.epage%253D37%26rft.atitle%253DIdentity%2Band%2Bconservation%2Bof%2Bmating%2Btype%2Bgenes%2Bin%2Bgeographically%2Bdiverse%2Bisolates%2Bof%2BPhaeosphaeria%2Bnodorum.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1087-1845%252803%252900062-8%26rft_id%253Dinfo%253Apmid%252F12948511%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1087-1845(03)00062-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12948511&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F959.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185246300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.7" data-doi="10.1186/1471-2105-10-301"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bringans</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hane</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casey</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipscombe</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Deep proteogenomics; high throughput gene validation by multidimensional liquid chromatography and mass spectrometry of proteins from the fungal wheat pathogen <em>Stagonospora nodorum</em>.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">301</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DBringans%26rft.auinit1%253DS.%26rft.volume%253D10%26rft.spage%253D301%26rft.epage%253D301%26rft.atitle%253DDeep%2Bproteogenomics%253B%2Bhigh%2Bthroughput%2Bgene%2Bvalidation%2Bby%2Bmultidimensional%2Bliquid%2Bchromatography%2Band%2Bmass%2Bspectrometry%2Bof%2Bproteins%2Bfrom%2Bthe%2Bfungal%2Bwheat%2Bpathogen%2BStagonospora%2Bnodorum.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-10-301%26rft_id%253Dinfo%253Apmid%252F19772613%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-10-301&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19772613&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F959.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.8" data-doi="10.1126/science.2544994"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cambareri</span>  <span class="cit-name-given-names">E. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schabtach</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Selker</span>  <span class="cit-name-given-names">E. U.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Repeat-induced G-C to A-T mutations in Neurospora.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">244</span>: <span class="cit-fpage">1571</span>–<span class="cit-lpage">1575</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DCambareri%26rft.auinit1%253DE.%26rft.volume%253D244%26rft.issue%253D4912%26rft.spage%253D1571%26rft.epage%253D1575%26rft.atitle%253DRepeat-induced%2BG-C%2Bto%2BA-T%2Bmutations%2Bin%2BNeurospora%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.2544994%26rft_id%253Dinfo%253Apmid%252F2544994%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyNDQvNDkxMi8xNTcxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy82Lzk1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.9" data-doi="10.1101/gr.6743907"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cantarel</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korf</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robb</span>  <span class="cit-name-given-names">S. M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parra</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ross</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">MAKER: an easy-to-use annotation pipeline designed for emerging model organism genomes.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">188</span>–<span class="cit-lpage">196</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.6743907%26rft_id%253Dinfo%253Apmid%252F18025269%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE4LzEvMTg4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy82Lzk1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.10" data-doi="10.1002/pmic.200900474"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Casey</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Solomon</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bringans</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliver</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Quantitative proteomic analysis of G-protein signalling in <em>Stagonospora nodorum</em> using isobaric tags for relative and absolute quantification.</span> <abbr class="cit-jnl-abbrev">Proteomics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">38</span>–<span class="cit-lpage">47</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProteomics%26rft.stitle%253DProteomics%26rft.aulast%253DCasey%26rft.auinit1%253DT.%26rft.volume%253D10%26rft.issue%253D1%26rft.spage%253D38%26rft.epage%253D47%26rft.atitle%253DQuantitative%2Bproteomic%2Banalysis%2Bof%2BG-protein%2Bsignalling%2Bin%2BStagonospora%2Bnodorum%2Busing%2Bisobaric%2Btags%2Bfor%2Brelative%2Band%2Babsolute%2Bquantification.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fpmic.200900474%26rft_id%253Dinfo%253Apmid%252F19882661%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/pmic.200900474&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19882661&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F959.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.11" data-doi="10.3852/mycologia.98.6.850"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Celio</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Padamsee</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dentinger</span>  <span class="cit-name-given-names">B. T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bauer</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McLaughlin</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Assembling the fungal tree of life: constructing the structural and biochemical database.</span> <abbr class="cit-jnl-abbrev">Mycologia</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">850</span>–<span class="cit-lpage">859</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMycologia%26rft.stitle%253DMycologia%26rft.aulast%253DCelio%26rft.auinit1%253DG.J.%26rft.volume%253D98%26rft.issue%253D6%26rft.spage%253D850%26rft.epage%253D859%26rft.atitle%253DAssembling%2Bthe%2BFungal%2BTree%2Bof%2BLife%253A%2Bconstructing%2Bthe%2BStructural%2Band%2BBiochemical%2BDatabase%26rft_id%253Dinfo%253Adoi%252F10.3852%252Fmycologia.98.6.850%26rft_id%253Dinfo%253Apmid%252F17486962%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToibXljb2xvZ2lhIjtzOjU6InJlc2lkIjtzOjg6Ijk4LzYvODUwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy82Lzk1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chu</span>  <span class="cit-name-given-names">C.-G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faris</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friesen</span>  <span class="cit-name-given-names">T. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genetic analysis of disease susceptibility contributed by the compatible Tsn1-SnToxA and Snn1-SnTox1 interactions in the wheat-Stagonospora nodorum pathosystem.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">1451</span>–<span class="cit-lpage">1459</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D129%26rft.spage%253D1451%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.13" data-doi="10.1105/tpc.9.2.135"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ciuffetti</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tuori</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaventa</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">A single gene encodes a selective toxin causal to the development of tan spot of wheat.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">135</span>–<span class="cit-lpage">144</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BCell%2BOnline%26rft.stitle%253DPlant%2BCell%26rft.aulast%253DCiuffetti%26rft.auinit1%253DL.%2BM.%26rft.volume%253D9%26rft.issue%253D2%26rft.spage%253D135%26rft.epage%253D144%26rft.atitle%253DA%2BSingle%2BGene%2BEncodes%2Ba%2BSelective%2BToxin%2BCausal%2Bto%2Bthe%2BDevelopment%2Bof%2BTan%2BSpot%2Bof%2BWheat%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.9.2.135%26rft_id%253Dinfo%253Apmid%252F9061946%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjc6IjkvMi8xMzUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzYvOTU5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.14" data-doi="10.1016/j.fgb.2010.09.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clutterbuck</span>  <span class="cit-name-given-names">A. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genomic evidence of repeat-induced point mutation (RIP) in filamentous ascomycetes.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Biol.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">306</span>–<span class="cit-lpage">326</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2Bgenetics%2Band%2Bbiology%2B%253A%2B%2BFG%2B%2526%2BB%26rft.stitle%253DFungal%2BGenet%2BBiol%26rft.aulast%253DClutterbuck%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D48%26rft.issue%253D3%26rft.spage%253D306%26rft.epage%253D326%26rft.atitle%253DGenomic%2Bevidence%2Bof%2Brepeat-induced%2Bpoint%2Bmutation%2B%2528RIP%2529%2Bin%2Bfilamentous%2Bascomycetes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fgb.2010.09.002%26rft_id%253Dinfo%253Apmid%252F20854921%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fgb.2010.09.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20854921&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F959.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cooley</span>  <span class="cit-name-given-names">R. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caten</span>  <span class="cit-name-given-names">C. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Variation in electrophoretic karyotype between strains of <em>Septoria nodorum</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Gen. Genet.</abbr> <span class="cit-vol">228</span>: <span class="cit-fpage">17</span>–<span class="cit-lpage">23</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2B%2526%2Bgeneral%2Bgenetics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGen%2BGenet%26rft.aulast%253DCooley%26rft.auinit1%253DR.%2BN.%26rft.volume%253D228%26rft.issue%253D1-2%26rft.spage%253D17%26rft.epage%253D23%26rft.atitle%253DVariation%2Bin%2Belectrophoretic%2Bkaryotype%2Bbetween%2Bstrains%2Bof%2BSeptoria%2Bnodorum.%26rft_id%253Dinfo%253Apmid%252F1886607%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=1886607&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F959.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991GB74400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.16" data-doi="10.1094/PHYTO-08-11-0238"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crook</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friesen</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Z. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ojiambo</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cowger</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Novel necrotrophic effectors from <em>Stagonospora nodorum</em> and corresponding host genes in winter wheat germplasm in the Southeastern U.S.</span> <abbr class="cit-jnl-abbrev">Phytopathology</abbr>. <span class="cit-vol">102</span>: <span class="cit-fpage">498</span>–<span class="cit-lpage">505</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhytopathology%26rft.stitle%253DPhytopathology%26rft.aulast%253DCrook%26rft.auinit1%253DA.%2BD.%26rft.volume%253D102%26rft.issue%253D5%26rft.spage%253D498%26rft.epage%253D505%26rft.atitle%253DNovel%2Bnecrotrophic%2Beffectors%2Bfrom%2BStagonospora%2Bnodorum%2Band%2Bcorresponding%2Bhost%2Bsensitivities%2Bin%2Bwinter%2Bwheat%2Bgermplasm%2Bin%2Bthe%2Bsoutheastern%2BUnited%2BStates.%26rft_id%253Dinfo%253Adoi%252F10.1094%252FPHYTO-08-11-0238%26rft_id%253Dinfo%253Apmid%252F22494247%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/PHYTO-08-11-0238&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22494247&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F959.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.17" data-doi="10.1111/j.1364-3703.2009.00591.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De Wit</span>  <span class="cit-name-given-names">P. J. G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mehrabi</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Den Burg</span>  <span class="cit-name-given-names">H. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stergiopoulos</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fungal effector proteins: past, present and future</span>. <span class="cit-comment">Review</span> <abbr class="cit-jnl-abbrev">Mol. Plant Pathol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">735</span>–<span class="cit-lpage">747</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DMol%2BPlant%2BPathol%26rft.aulast%253DDE%2BWit%26rft.auinit1%253DP.%2BJ.%26rft.volume%253D10%26rft.issue%253D6%26rft.spage%253D735%26rft.epage%253D747%26rft.atitle%253DFungal%2Beffector%2Bproteins%253A%2Bpast%252C%2Bpresent%2Band%2Bfuture.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1364-3703.2009.00591.x%26rft_id%253Dinfo%253Apmid%252F19849781%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1364-3703.2009.00591.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19849781&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F959.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271003900003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-other" id="cit-3.6.959.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doyle</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doyle</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">A rapid DNA isolation procedure for small quantities of fresh leaf tissue.</span> <span class="cit-source">Phytochem. Bull.</span> <span class="cit-vol">19</span>: <span class="cit-fpage">11</span>–<span class="cit-lpage">15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhytochem.%2BBull.%26rft.volume%253D19%26rft.spage%253D11%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.19" data-doi="10.1186/1471-2105-5-113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edgar</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">MUSCLE: a multiple sequence alignment method with reduced time and space complexity.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">113</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DEdgar%26rft.auinit1%253DR.%2BC.%26rft.volume%253D5%26rft.spage%253D113%26rft.epage%253D113%26rft.atitle%253DMUSCLE%253A%2Ba%2Bmultiple%2Bsequence%2Balignment%2Bmethod%2Bwith%2Breduced%2Btime%2Band%2Bspace%2Bcomplexity.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-5-113%26rft_id%253Dinfo%253Apmid%252F15318951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-5-113&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15318951&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F959.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.20" data-doi="10.1094/PHYTO-99-8-0906"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Faris</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friesen</span>  <span class="cit-name-given-names">T. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Reevaluation of a tetraploid wheat population indicates that the Tsnl-ToxA interaction is the only factor governing <em>Stagonospora nodorum</em> blotch susceptibility.</span> <abbr class="cit-jnl-abbrev">Phytopathology</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">906</span>–<span class="cit-lpage">912</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhytopathology%26rft.stitle%253DPhytopathology%26rft.aulast%253DFaris%26rft.auinit1%253DJ.%2BD.%26rft.volume%253D99%26rft.issue%253D8%26rft.spage%253D906%26rft.epage%253D912%26rft.atitle%253DReevaluation%2Bof%2Ba%2Btetraploid%2Bwheat%2Bpopulation%2Bindicates%2Bthat%2Bthe%2BTsn1-ToxA%2Binteraction%2Bis%2Bthe%2Bonly%2Bfactor%2Bgoverning%2BStagonospora%2Bnodorum%2Bblotch%2Bsusceptibility.%26rft_id%253Dinfo%253Adoi%252F10.1094%252FPHYTO-99-8-0906%26rft_id%253Dinfo%253Apmid%252F19594309%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/PHYTO-99-8-0906&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19594309&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F959.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267899100002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.21" data-doi="10.1073/pnas.1004090107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Faris</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reddy</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A unique wheat disease resistance-like gene governs effector-triggered susceptibility to necrotrophic pathogens.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">13544</span>–<span class="cit-lpage">13549</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1004090107%26rft_id%253Dinfo%253Apmid%252F20624958%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA3LzMwLzEzNTQ0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy82Lzk1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.22" data-doi="10.1094/MPMI-04-11-0094"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Faris</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rasmussen</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friesen</span>  <span class="cit-name-given-names">T. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Variable expression of the <em>Stagonospora nodorum</em> effector SnToxA among isolates is correlated with levels of disease in wheat.</span> <abbr class="cit-jnl-abbrev">Mol. Plant Microbe Interact.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1419</span>–<span class="cit-lpage">1426</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bplant-microbe%2Binteractions%2B%253A%2B%2BMPMI%26rft.stitle%253DMol%2BPlant%2BMicrobe%2BInteract%26rft.aulast%253DFaris%26rft.auinit1%253DJ.%2BD.%26rft.volume%253D24%26rft.issue%253D12%26rft.spage%253D1419%26rft.epage%253D1426%26rft.atitle%253DVariable%2Bexpression%2Bof%2Bthe%2BStagonospora%2Bnodorum%2Beffector%2BSnToxA%2Bamong%2Bisolates%2Bis%2Bcorrelated%2Bwith%2Blevels%2Bof%2Bdisease%2Bin%2Bwheat.%26rft_id%253Dinfo%253Adoi%252F10.1094%252FMPMI-04-11-0094%26rft_id%253Dinfo%253Apmid%252F21770771%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/MPMI-04-11-0094&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21770771&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F959.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000297137700002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.23" data-doi="10.1038/ng1839"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Friesen</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stukenbrock</span>  <span class="cit-name-given-names">E. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meinhardt</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ling</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Emergence of a new disease as a result of interspecific virulence gene transfer.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">953</span>–<span class="cit-lpage">956</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DFriesen%26rft.auinit1%253DT.%2BL.%26rft.volume%253D38%26rft.issue%253D8%26rft.spage%253D953%26rft.epage%253D956%26rft.atitle%253DEmergence%2Bof%2Ba%2Bnew%2Bdisease%2Bas%2Ba%2Bresult%2Bof%2Binterspecific%2Bvirulence%2Bgene%2Btransfer.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1839%26rft_id%253Dinfo%253Apmid%252F16832356%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1839&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16832356&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F959.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239325700028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.24" data-doi="10.1111/j.1365-313X.2007.03166.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Friesen</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meinhardt</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faris</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The <em>Stagonospora nodorum</em>-wheat pathosystem involves multiple proteinaceous host-selective toxins and corresponding host sensitivity genes that interact in an inverse gene-for-gene manner.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">681</span>–<span class="cit-lpage">692</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DFriesen%26rft.auinit1%253DT.%2BL.%26rft.volume%253D51%26rft.issue%253D4%26rft.spage%253D681%26rft.epage%253D692%26rft.atitle%253DThe%2BStagonospora%2Bnodorum-wheat%2Bpathosystem%2Binvolves%2Bmultiple%2Bproteinaceous%2Bhost-selective%2Btoxins%2Band%2Bcorresponding%2Bhost%2Bsensitivity%2Bgenes%2Bthat%2Binteract%2Bin%2Ban%2Binverse%2Bgene-for-gene%2Bmanner.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2007.03166.x%26rft_id%253Dinfo%253Apmid%252F17573802%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2007.03166.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17573802&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F959.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249203100012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.25" data-doi="10.1111/j.1462-5822.2008.01153.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Friesen</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faris</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Solomon</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliver</span>  <span class="cit-name-given-names">R. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Host-specific toxins: effectors of necrotrophic pathogenicity.</span> <abbr class="cit-jnl-abbrev">Cell. Microbiol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1421</span>–<span class="cit-lpage">1428</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCellular%2Bmicrobiology%26rft.stitle%253DCell%2BMicrobiol%26rft.aulast%253DFriesen%26rft.auinit1%253DT.%2BL.%26rft.volume%253D10%26rft.issue%253D7%26rft.spage%253D1421%26rft.epage%253D1428%26rft.atitle%253DHost-specific%2Btoxins%253A%2Beffectors%2Bof%2Bnecrotrophic%2Bpathogenicity.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1462-5822.2008.01153.x%26rft_id%253Dinfo%253Apmid%252F18384660%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1462-5822.2008.01153.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18384660&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F959.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000256685800003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.26" data-doi="10.1007/s00122-009-0997-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Friesen</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chu</span>  <span class="cit-name-given-names">C. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Z. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Halley</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Host-selective toxins produced by <em>Stagonospora nodorum</em> confer disease susceptibility in adult wheat plants under field conditions.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">118</span>: <span class="cit-fpage">1489</span>–<span class="cit-lpage">1497</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DFriesen%26rft.auinit1%253DT.%2BL.%26rft.volume%253D118%26rft.issue%253D8%26rft.spage%253D1489%26rft.epage%253D1497%26rft.atitle%253DHost-selective%2Btoxins%2Bproduced%2Bby%2BStagonospora%2Bnodorum%2Bconfer%2Bdisease%2Bsusceptibility%2Bin%2Badult%2Bwheat%2Bplants%2Bunder%2Bfield%2Bconditions.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-009-0997-2%26rft_id%253Dinfo%253Apmid%252F19266177%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-009-0997-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19266177&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F959.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.27" data-doi="10.1186/gb-2004-5-10-r80"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gentleman</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carey</span>  <span class="cit-name-given-names">V. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bates</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bolstad</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dettling</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Bioconductor: open software development for computational biology and bioinformatics.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">R80</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DGentleman%26rft.auinit1%253DR.%2BC.%26rft.volume%253D5%26rft.issue%253D10%26rft.spage%253DR80%26rft.epage%253DR80%26rft.atitle%253DBioconductor%253A%2Bopen%2Bsoftware%2Bdevelopment%2Bfor%2Bcomputational%2Bbiology%2Band%2Bbioinformatics.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2004-5-10-r80%26rft_id%253Dinfo%253Apmid%252F15461798%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2004-5-10-r80&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15461798&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F959.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.28" data-doi="10.1371/journal.pgen.1002070"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goodwin</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">M’Barek</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dhillon</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wittenberg</span>  <span class="cit-name-given-names">A. H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crane</span>  <span class="cit-name-given-names">C. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Finished genome of the fungal wheat pathogen <em>Mycosphaerella graminicola</em> reveals dispensome structure, chromosome plasticity, and stealth pathogenesis.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1002070</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D7%26rft.spage%253D1002070%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002070%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002070&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-other" id="cit-3.6.959.29" data-doi="10.1093/bioinformatics/btq475"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goto</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prins</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakao</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonnal</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aerts</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">BioRuby: Bioinformatics software for the Ruby programming language.</span> <span class="cit-source">Bioinformatics (Oxford, England).</span> <span class="cit-vol">26</span>: <span class="cit-fpage">2617</span>–<span class="cit-lpage">2619</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%2B%2528Oxford%252C%2BEngland%2529.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq475%26rft_id%253Dinfo%253Apmid%252F20739307%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjYvMjAvMjYxNyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNi85NTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.30" data-doi="10.1186/1471-2105-9-478"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hane</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliver</span>  <span class="cit-name-given-names">R. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">RIPCAL: a tool for alignment-based analysis of repeat-induced point mutations in fungal genomic sequences.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">478</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DHane%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D9%26rft.spage%253D478%26rft.epage%253D478%26rft.atitle%253DRIPCAL%253A%2Ba%2Btool%2Bfor%2Balignment-based%2Banalysis%2Bof%2Brepeat-induced%2Bpoint%2Bmutations%2Bin%2Bfungal%2Bgenomic%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-9-478%26rft_id%253Dinfo%253Apmid%252F19014496%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-9-478&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19014496&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F959.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.31" data-doi="10.1186/1471-2164-11-655"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hane</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliver</span>  <span class="cit-name-given-names">R. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">In silico reversal of repeat-induced point mutation (RIP) identifies the origins of repeat families and uncovers obscured duplicated genes.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">655</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DHane%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D11%26rft.spage%253D655%26rft.epage%253D655%26rft.atitle%253DIn%2Bsilico%2Breversal%2Bof%2Brepeat-induced%2Bpoint%2Bmutation%2B%2528RIP%2529%2Bidentifies%2Bthe%2Borigins%2Bof%2Brepeat%2Bfamilies%2Band%2Buncovers%2Bobscured%2Bduplicated%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-11-655%26rft_id%253Dinfo%253Apmid%252F21106049%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-11-655&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21106049&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F959.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.32" data-doi="10.1105/tpc.107.052829"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hane</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lowe</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Solomon</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schoch</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Dothideomycete plant interactions illuminated by genome sequencing and EST analysis of the wheat pathogen <em>Stagonospora nodorum</em>.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">3347</span>–<span class="cit-lpage">3368</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.107.052829%26rft_id%253Dinfo%253Apmid%252F18024570%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIxOS8xMS8zMzQ3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy82Lzk1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-book" id="cit-3.6.959.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hane</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliver</span>  <span class="cit-name-given-names">R. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genomic and comparative analysis of the class Dothideomycetes</span>, pp. <span class="cit-fpage">205</span>–<span class="cit-lpage">226</span> in <span class="cit-source">The Mycota</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Poggeler</span>  <span class="cit-name-given-names">S.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Wostemeyer</span>  <span class="cit-name-given-names">J.</span></span>. <span class="cit-publ-name">Springer-Verlag</span>, <span class="cit-publ-loc">Berlin</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Horton</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Obayashi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fujita</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harada</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">WoLF PSORT: protein localization predictor.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">W585–587</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D35%26rft.spage%253DW585%2B587%26rft.atitle%253DWOLF%2BPSORT%253A%2BPROTEIN%2BLOCALIZATION%2BPREDICTOR.%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.35" data-doi="10.1111/j.1364-3703.2011.00770.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ipcho</span>  <span class="cit-name-given-names">S. V. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hane</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Antoni</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ahren</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henrissat</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Transcriptome analysis of <em>Stagonospora nodorum</em>: gene models, effectors, metabolism and pantothenate dispensability.</span> <abbr class="cit-jnl-abbrev">Mol. Plant Pathol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">531</span>–<span class="cit-lpage">545</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DMol%2BPlant%2BPathol%26rft.aulast%253DIpcho%26rft.auinit1%253DS.%2BV.%26rft.volume%253D13%26rft.issue%253D6%26rft.spage%253D531%26rft.epage%253D545%26rft.atitle%253DTranscriptome%2Banalysis%2Bof%2BStagonospora%2Bnodorum%253A%2Bgene%2Bmodels%252C%2Beffectors%252C%2Bmetabolism%2Band%2Bpantothenate%2Bdispensability.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1364-3703.2011.00770.x%26rft_id%253Dinfo%253Apmid%252F22145589%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1364-3703.2011.00770.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22145589&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F959.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000306003000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-other no-rev-xref" id="cit-3.6.959.36" data-doi="10.1093/bioinformatics/bts199"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kearse</span>  <span class="cit-name-given-names">M</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moir</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stones-Havas</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheung</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Geneious Basic: an integrated and extendable desktop software platform for the organization and analysis of sequence data.</span> <span class="cit-source">Bioinformatics.</span> <span class="cit-vol">28</span>: <span class="cit-fpage">1647</span>–<span class="cit-lpage">1649</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts199%26rft_id%253Dinfo%253Apmid%252F22543367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjgvMTIvMTY0NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNi85NTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.37" data-doi="10.1101/gr.229202. Article published online before March 2002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kent</span>  <span class="cit-name-given-names">W. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">BLAT - The BLAST-like alignment tool.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">656</span>–<span class="cit-lpage">664</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.229202.%2BArticle%2Bpublished%2Bonline%2Bbefore%2BMarch%2B2002%26rft_id%253Dinfo%253Apmid%252F11932250%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEyLzQvNjU2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy82Lzk1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.38" data-doi="10.1186/gb-2007-8-8-r171"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Knight</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maxwell</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birmingham</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carnes</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caporaso</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">PyCogent: a toolkit for making sense from sequence.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">R171</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DKnight%26rft.auinit1%253DR.%26rft.volume%253D8%26rft.issue%253D8%26rft.spage%253DR171%26rft.epage%253DR171%26rft.atitle%253DPyCogent%253A%2Ba%2Btoolkit%2Bfor%2Bmaking%2Bsense%2Bfrom%2Bsequence.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2007-8-8-r171%26rft_id%253Dinfo%253Apmid%252F17708774%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2007-8-8-r171&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17708774&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F959.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.39" data-doi="10.1093/bioinformatics/btm404"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Larkin</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blackshields</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">N. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chenna</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McGettigan</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Clustal W and Clustal X version 2.0.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">2947</span>–<span class="cit-lpage">2948</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm404%26rft_id%253Dinfo%253Apmid%252F17846036%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjMvMjEvMjk0NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNi85NTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.40" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTQvMTc1NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNi85NTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.41" data-doi="10.1101/gr.1224503"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stoeckert</span>  <span class="cit-name-given-names">C. J.</span>  <span class="cit-name-suffix">Jr</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roos</span>  <span class="cit-name-given-names">D. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">OrthoMCL: identification of ortholog groups for eukaryotic genomes.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">2178</span>–<span class="cit-lpage">2189</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DLi%26rft.auinit1%253DL.%26rft.volume%253D13%26rft.issue%253D9%26rft.spage%253D2178%26rft.epage%253D2189%26rft.atitle%253DOrthoMCL%253A%2BIdentification%2Bof%2BOrtholog%2BGroups%2Bfor%2BEukaryotic%2BGenomes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.1224503%26rft_id%253Dinfo%253Apmid%252F12952885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjEzLzkvMjE3OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNi85NTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.42" data-doi="10.1371/journal.ppat.1000581"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faris</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliver</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Solomon</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">SnTox3 acts in effector triggered susceptibility to induce disease on wheat carrying the Snn3 gene.</span> <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000581</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DLiu%26rft.auinit1%253DZ.%26rft.volume%253D5%26rft.issue%253D9%26rft.spage%253De1000581%26rft.epage%253De1000581%26rft.atitle%253DSnTox3%2Bacts%2Bin%2Beffector%2Btriggered%2Bsusceptibility%2Bto%2Binduce%2Bdisease%2Bon%2Bwheat%2Bcarrying%2Bthe%2BSnn3%2Bgene.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1000581%26rft_id%253Dinfo%253Apmid%252F19806176%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1000581&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19806176&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F959.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.43" data-doi="10.1371/journal.ppat.1002467"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faris</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliver</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Syme</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The cysteine rich necrotrophic effector SnTox1 produced by <em>Stagonospora nodorum</em> triggers susceptibility of wheat lines harboring Snn1.</span> <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1002467</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog.%26rft.volume%253D8%26rft.spage%253D1002467%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1002467%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1002467&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.44" data-doi="10.1046/j.1464-6722.2001.00095.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Luderer</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Kock</span>  <span class="cit-name-given-names">M. J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dees</span>  <span class="cit-name-given-names">R. H. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Wit</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joosten</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Functional analysis of cysteine residues of ECP elicitor proteins of the fungal tomato pathogen <em>Cladosporium fulvum</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Plant Pathol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">91</span>–<span class="cit-lpage">95</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DMol%2BPlant%2BPathol%26rft.aulast%253DLuderer%26rft.auinit1%253DR.%26rft.volume%253D3%26rft.issue%253D2%26rft.spage%253D91%26rft.epage%253D95%26rft.atitle%253DFunctional%2Banalysis%2Bof%2Bcysteine%2Bresidues%2Bof%2BECP%2Belicitor%2Bproteins%2Bof%2Bthe%2Bfungal%2Btomato%2Bpathogen%2BCladosporium%2Bfulvum.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1464-6722.2001.00095.x%26rft_id%253Dinfo%253Apmid%252F20569313%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1464-6722.2001.00095.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20569313&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F959.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.45" data-doi="10.1038/nature08850"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Der Does</span>  <span class="cit-name-given-names">H. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borkovich</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coleman</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daboussi</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Comparative genomics reveals mobile pathogenicity chromosomes in Fusarium.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">464</span>: <span class="cit-fpage">367</span>–<span class="cit-lpage">373</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMa%26rft.auinit1%253DL.%2BJ.%26rft.volume%253D464%26rft.issue%253D7287%26rft.spage%253D367%26rft.epage%253D373%26rft.atitle%253DComparative%2Bgenomics%2Breveals%2Bmobile%2Bpathogenicity%2Bchromosomes%2Bin%2BFusarium.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08850%26rft_id%253Dinfo%253Apmid%252F20237561%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08850&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20237561&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F959.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275657100037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.46" data-doi="10.1007/s10658-009-9454-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Malkus</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cregan</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arseniuk</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ueng</span>  <span class="cit-name-given-names">P. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genetic linkage map of <em>Phaeosphaeria nodorum</em>, the causal agent of stagonospora nodorum blotch disease of wheat.</span> <abbr class="cit-jnl-abbrev">Eur. J. Plant Pathol.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">681</span>–<span class="cit-lpage">690</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEur.%2BJ.%2BPlant%2BPathol.%26rft.volume%253D124%26rft.spage%253D681%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10658-009-9454-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10658-009-9454-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-other" id="cit-3.6.959.47" data-doi="10.1534/g3.112.004044"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Manning</span>  <span class="cit-name-given-names">V. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pandelova</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dhillon</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilhelm</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodwin</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Comparative genomics of a plant-pathogenic fungus, Pyrenophora tritici-repentis, reveals transduplication and the impact of repeat elements on pathogenicity and population divergence.</span> <span class="cit-source">G3 (Bethesda)</span> <span class="cit-vol">3</span>: <span class="cit-fpage">41</span>–<span class="cit-lpage">63</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2526%252358%253B%2BGenes%257CGenomes%257CGenetics%26rft.stitle%253DG3%26rft.aulast%253DManning%26rft.auinit1%253DV.%2BA.%26rft.volume%253D3%26rft.issue%253D1%26rft.spage%253D41%26rft.epage%253D63%26rft.atitle%253DComparative%2BGenomics%2Bof%2Ba%2BPlant-Pathogenic%2BFungus%252C%2BPyrenophora%2Btritici-repentis%252C%2BReveals%2BTransduplication%2Band%2Bthe%2BImpact%2Bof%2BRepeat%2BElements%2Bon%2BPathogenicity%2Band%2BPopulation%2BDivergence%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.112.004044%26rft_id%253Dinfo%253Apmid%252F23316438%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjY6IjMvMS80MSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNi85NTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-web" id="cit-3.6.959.48"><div class="cit-metadata unstructured">Martin, M., 2011 Cutadapt removes adapter sequences from high-throughput sequencing reads. EMBnet J. <strong>17:</strong> North America, 17, May. 2011. Available at: <a href="http://journal.embnet.org/index.php/embnetjournal/article/view/200">http://journal.embnet.org/index.php/embnetjournal/article/view/200</a>. Accessed: April 23, 2013.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.49" data-doi="10.1101/gr.107524.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKenna</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanna</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sivachenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cibulskis</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1297</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.107524.110%26rft_id%253Dinfo%253Apmid%252F20644199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTI5NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNi85NTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.50" data-doi="10.1071/AP09053"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brennan</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Estimating disease losses to the Australian wheat industry.</span> <abbr class="cit-jnl-abbrev">Australas. Plant Pathol.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">558</span>–<span class="cit-lpage">570</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAustralas.%2BPlant%2BPathol.%26rft.volume%253D38%26rft.spage%253D558%26rft_id%253Dinfo%253Adoi%252F10.1071%252FAP09053%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1071/AP09053&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000270866400002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-confproc" id="cit-3.6.959.51"><div class="cit-metadata unstructured">Oliver, R. P., P. S. Solomon, J. Hane, M. Ferguson-Hunt, B. A. McDonald <em>et al.</em>, 2008 Multiple host-specific toxins, lateral gene transfer and gene loss in the evolution of cereal Pleosporalean pathogens, in <em>9th International Congress of Plant Pathology</em>, Torino, Italy.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.52" data-doi="10.1071/CP08259"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oliver</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rybak</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Solomon</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferguson-Hunt</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Prevalence of ToxA-sensitive alleles of the wheat gene <em>Tsn1</em> in Australian and Chinese wheat cultivars</span>. <abbr class="cit-jnl-abbrev">Crop Pasture Sci.</abbr> <span class="cit-vol">60</span>: <span class="cit-fpage">348</span>–<span class="cit-lpage">352</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BPasture%2BSci.%26rft.volume%253D60%26rft.spage%253D348%26rft_id%253Dinfo%253Adoi%252F10.1071%252FCP08259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1071/CP08259&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.53" data-doi="10.1146/annurev-phyto-081211-173019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oliver</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friesen</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faris</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Solomon</span>  <span class="cit-name-given-names">P. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title"><em>Stagonospora nodorum</em>: from pathology to genomics and host resistance.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Phytopathol.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">23</span>–<span class="cit-lpage">43</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bphytopathology%26rft.stitle%253DAnnu%2BRev%2BPhytopathol%26rft.aulast%253DOliver%26rft.auinit1%253DR.%2BP.%26rft.volume%253D50%26rft.spage%253D23%26rft.epage%253D43%26rft.atitle%253DStagonospora%2Bnodorum%253A%2Bfrom%2Bpathology%2Bto%2Bgenomics%2Band%2Bhost%2Bresistance.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-phyto-081211-173019%26rft_id%253Dinfo%253Apmid%252F22559071%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-phyto-081211-173019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22559071&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F959.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000310143200003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.54" data-doi="10.1093/bioinformatics/btq033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quinlan</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">I. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">BEDTools: a flexible suite of utilities for comparing genomic features.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">841</span>–<span class="cit-lpage">842</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq033%26rft_id%253Dinfo%253Apmid%252F20110278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNi82Lzg0MSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNi85NTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-other" id="cit-3.6.959.55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rouxel</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grandaubert</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hane</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoede</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van de Wouw</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The compartmentalized genome of <em>Leptosphaeria maculans</em>: diversification of effectors within genomic regions affected by repeat induced Point mutations.</span> <span class="cit-source">Nat. Commun.</span> <span class="cit-vol">2</span>: <span class="cit-fpage">art. 202</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D2%26rft.spage%253Dart.%2B202%26rft.atitle%253DTHE%2BCOMPARTMENTALIZED%2BGENOME%2BOF%2BLEPTOSPHAERIA%2BMACULANS%253A%2BDIVERSIFICATION%2BOF%2BEFFECTORS%2BWITHIN%2BGENOMIC%2BREGIONS%2BAFFECTED%2BBY%2BREPEAT%2BINDUCED%2BPOINT%2BMUTATIONS.%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.56" data-doi="10.1146/annurev.ge.24.120190.003051"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Selker</span>  <span class="cit-name-given-names">E. U.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Premeiotic instability of repeated sequences in <em>Neurospora crassa</em>.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">579</span>–<span class="cit-lpage">613</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DSelker%26rft.auinit1%253DE.%26rft.volume%253D24%26rft.issue%253D1%26rft.spage%253D579%26rft.epage%253D613%26rft.atitle%253DPremeiotic%2BInstability%2BOf%2BRepeated%2BSequences%2BIn%2BNeurospora%2BCrassa%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ge.24.120190.003051%26rft_id%253Dinfo%253Apmid%252F2150906%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.ge.24.120190.003051&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2150906&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F959.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990EN96900024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.57" data-doi="10.1016/0092-8674(87)90097-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Selker</span>  <span class="cit-name-given-names">E. U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cambareri</span>  <span class="cit-name-given-names">E. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haack</span>  <span class="cit-name-given-names">K. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Rearrangement of duplicated DNA in specialized cells of <em>Neurospora</em>.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">741</span>–<span class="cit-lpage">752</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DSelker%26rft.auinit1%253DE.%2BU.%26rft.volume%253D51%26rft.issue%253D5%26rft.spage%253D741%26rft.epage%253D752%26rft.atitle%253DRearrangement%2Bof%2Bduplicated%2BDNA%2Bin%2Bspecialized%2Bcells%2Bof%2BNeurospora.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252887%252990097-3%26rft_id%253Dinfo%253Apmid%252F2960455%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(87)90097-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2960455&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F959.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1987L208200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-web" id="cit-3.6.959.58"><div class="cit-metadata unstructured">Smit, A., R. Hubley, and P. Green, 1996–2004  RepeatMasker Open-3.0. Available at: <a href="http://www.repeatmasker.com">www.repeatmasker.com</a>. Accessed: April 23, 2013.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.59" data-doi="10.1111/j.1364-3703.2006.00326.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Solomon</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lowe</span>  <span class="cit-name-given-names">R. G. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waters</span>  <span class="cit-name-given-names">O. D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliver</span>  <span class="cit-name-given-names">R. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span><span class="cit-comment">a</span> <span class="cit-article-title"><em>Stagonospora nodorum</em>: cause of stagonospora nodorum blotch of wheat.</span> <abbr class="cit-jnl-abbrev">Mol. Plant Pathol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">147</span>–<span class="cit-lpage">156</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DMol%2BPlant%2BPathol%26rft.aulast%253DSolomon%26rft.auinit1%253DP.%2BS.%26rft.volume%253D7%26rft.issue%253D3%26rft.spage%253D147%26rft.epage%253D156%26rft.atitle%253DStagonospora%2Bnodorum%253A%2Bcause%2Bof%2Bstagonospora%2Bnodorum%2Bblotch%2Bof%2Bwheat.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1364-3703.2006.00326.x%26rft_id%253Dinfo%253Apmid%252F20507435%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1364-3703.2006.00326.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20507435&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F959.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.60" data-doi="10.1007/s10658-005-5768-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Solomon</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">T. J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rybak</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parker</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lowe</span>  <span class="cit-name-given-names">R. G. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span><span class="cit-comment">b</span> <span class="cit-article-title">Structural characterisation of the interaction between <em>Triticum aestivum</em> and the dothideomycete pathogen <em>Stagonospora nodorum</em>.</span> <abbr class="cit-jnl-abbrev">Eur. J. Plant Pathol.</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">275</span>–<span class="cit-lpage">282</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEur.%2BJ.%2BPlant%2BPathol.%26rft.volume%253D114%26rft.spage%253D275%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10658-005-5768-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10658-005-5768-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.61" data-doi="10.1111/j.1364-3703.2007.00396.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stukenbrock</span>  <span class="cit-name-given-names">E. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">B. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Geographical variation and positive diversifying selection in the host-specific toxin SnToxA.</span> <abbr class="cit-jnl-abbrev">Mol. Plant Pathol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">321</span>–<span class="cit-lpage">332</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DMol%2BPlant%2BPathol%26rft.aulast%253DStukenbrock%26rft.auinit1%253DE.%2BH.%26rft.volume%253D8%26rft.issue%253D3%26rft.spage%253D321%26rft.epage%253D332%26rft.atitle%253DGeographical%2Bvariation%2Band%2Bpositive%2Bdiversifying%2Bselection%2Bin%2Bthe%2Bhost-specific%2Btoxin%2BSnToxA.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1364-3703.2007.00396.x%26rft_id%253Dinfo%253Apmid%252F20507502%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1364-3703.2007.00396.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20507502&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F959.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000245745400008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.62" data-doi="10.1007/s11306-009-0158-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trengove</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maker</span>  <span class="cit-name-given-names">G. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliver</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Solomon</span>  <span class="cit-name-given-names">P. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Metabolite profiling identifies the mycotoxin alternariol in the pathogen <em>Stagonospora nodorum</em>.</span> <abbr class="cit-jnl-abbrev">Metabolomics</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">330</span>–<span class="cit-lpage">335</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMetabolomics%26rft.volume%253D5%26rft.spage%253D330%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11306-009-0158-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11306-009-0158-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.63" data-doi="10.1071/FP10067"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliver</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Solomon</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moffat</span>  <span class="cit-name-given-names">C. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Proteinaceous necrotrophic effectors in fungal virulence.</span> <abbr class="cit-jnl-abbrev">Funct. Plant Biol.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">907</span>–<span class="cit-lpage">912</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFunct.%2BPlant%2BBiol.%26rft.volume%253D37%26rft.spage%253D907%26rft_id%253Dinfo%253Adoi%252F10.1071%252FFP10067%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1071/FP10067&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.64" data-doi="10.1094/MPMI-10-11-0273"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">K.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferguson-Hunt</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rybak</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waters</span>  <span class="cit-name-given-names">O. D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stanley</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Quantitative variation in effector activity of ToxA Isoforms from <em>Stagonospora nodorum</em> and Pyrenophora tritici-repentis.</span> <abbr class="cit-jnl-abbrev">Mol. Plant Microbe Interact.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">515</span>–<span class="cit-lpage">522</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bplant-microbe%2Binteractions%2B%253A%2B%2BMPMI%26rft.stitle%253DMol%2BPlant%2BMicrobe%2BInteract%26rft.aulast%253DTan%26rft.auinit1%253DK.%2BC.%26rft.volume%253D25%26rft.issue%253D4%26rft.spage%253D515%26rft.epage%253D522%26rft.atitle%253DQuantitative%2Bvariation%2Bin%2Beffector%2Bactivity%2Bof%2BToxA%2Bisoforms%2Bfrom%2BStagonospora%2Bnodorum%2Band%2BPyrenophora%2Btritici-repentis.%26rft_id%253Dinfo%253Adoi%252F10.1094%252FMPMI-10-11-0273%26rft_id%253Dinfo%253Apmid%252F22250581%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/MPMI-10-11-0273&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22250581&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F959.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.65" data-doi="10.1101/gr.081612.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ter-Hovhannisyan</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lomsadze</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chernoff</span>  <span class="cit-name-given-names">Y. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borodovsky</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Gene prediction in novel fungal genomes using an ab initio algorithm with unsupervised training.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1979</span>–<span class="cit-lpage">1990</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.081612.108%26rft_id%253Dinfo%253Apmid%252F18757608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxOC8xMi8xOTc5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy82Lzk1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.66" data-doi="10.1371/journal.ppat.1001180"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van de Wouw</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cozijnsen</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hane</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brunner</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Evolution of linked avirulence effectors in <em>Leptosphaeria maculans</em> is affected by genomic environment and exposure to resistance genes in host plants.</span> <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1001180</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DVan%2Bde%2BWouw%26rft.auinit1%253DA.%2BP.%26rft.volume%253D6%26rft.issue%253D11%26rft.spage%253De1001180%26rft.epage%253De1001180%26rft.atitle%253DEvolution%2Bof%2Blinked%2Bavirulence%2Beffectors%2Bin%2BLeptosphaeria%2Bmaculans%2Bis%2Baffected%2Bby%2Bgenomic%2Benvironment%2Band%2Bexposure%2Bto%2Bresistance%2Bgenes%2Bin%2Bhost%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1001180%26rft_id%253Dinfo%253Apmid%252F21079787%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1001180&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21079787&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F959.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.67" data-doi="10.1093/molbev/msp232"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yap</span>  <span class="cit-name-given-names">V. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lindsay</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Easteal</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huttley</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Estimates of the effect of natural selection on protein-coding content.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">726</span>–<span class="cit-lpage">734</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsp232%26rft_id%253Dinfo%253Apmid%252F19815689%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjI3LzMvNzI2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy82Lzk1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.68" data-doi="10.1101/gr.074492.107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zerbino</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birney</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Velvet: algorithms for de novo short read assembly using de Bruijn graphs.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">821</span>–<span class="cit-lpage">829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.074492.107%26rft_id%253Dinfo%253Apmid%252F18349386%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE4LzUvODIxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy82Lzk1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-other" id="cit-3.6.959.69" data-doi="10.3114/sim.2009.64.04"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schoch</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fournier</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crous</span>  <span class="cit-name-given-names">P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Gruyter</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Multi-locus phylogeny of Pleosporales: a taxonomic, ecological and evolutionary re-evaluation.</span> <span class="cit-source">Stud Mycol</span> <span class="cit-vol">64</span>: <span class="cit-fpage">85</span>–<span class="cit-lpage">102S105</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStudies%2Bin%2BMycology%26rft.stitle%253DSIM%26rft.aulast%253DZhang%26rft.auinit1%253DY.%26rft.volume%253D64%26rft.issue%253D1%26rft.spage%253D85%26rft.epage%253D102-S5%26rft.atitle%253DMulti-locus%2Bphylogeny%2Bof%2BPleosporales%253A%2Ba%2Btaxonomic%252C%2Becological%2Band%2Bevolutionary%2Bre-evaluation%26rft_id%253Dinfo%253Adoi%252F10.3114%252Fsim.2009.64.04%26rft_id%253Dinfo%253Apmid%252F20169024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3114/sim.2009.64.04&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20169024&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F959.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.959.70" data-doi="10.1111/j.1365-313X.2010.04407.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friesen</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Two putatively homoeologous wheat genes mediate recognition of SnTox3 to confer effector-triggered susceptibility to <em>Stagonospora nodorum</em>.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">27</span>–<span class="cit-lpage">38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DZhang%26rft.auinit1%253DZ.%26rft.volume%253D65%26rft.issue%253D1%26rft.spage%253D27%26rft.epage%253D38%26rft.atitle%253DTwo%2Bputatively%2Bhomoeologous%2Bwheat%2Bgenes%2Bmediate%2Brecognition%2Bof%2BSnTox3%2Bto%2Bconfer%2Beffector-triggered%2Bsusceptibility%2Bto%2BStagonospora%2Bnodorum.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2010.04407.x%26rft_id%253Dinfo%253Apmid%252F21175887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2010.04407.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21175887&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F959.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/3/6/959.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/6/949" title="The Use of RelocaTE and Unassembled Short Reads to Produce High-Resolution Snapshots of Transposable Element Generated Diversity in Rice" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/6/971" title="Genetic Polymorphisms in Host Innate Immune Sensor Genes and the Risk of Nasopharyngeal Carcinoma in North Africa" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/3/6"><span>Volume 3 Issue 6, June 2013</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/3/6.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/3/6.cover-source.jpg?itok=CJf4vROV" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 3 (6)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/6/959.long" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-OB0mqqemDofAplxOdb4W-vyObHsl_LCbh0K7qACKhbs" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/3/6/959" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/3/6/959" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/3/6/959" class="active" data-icon-position="" data-hide-link-title="0">Resequencing and Comparative Genomics of Stagonospora nodorum: Sectional Gene Absence and Effector Discovery</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459473" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-ONYpneZPrMUWS_7GMtN5Y4JtD8BjzufmANu3GYIqidw" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/6/959.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/3/6/959" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-DLyhIRNoDsAhn8gNp7rC9luiRRH7fARQW_MZ3CWlVlA" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/3/6/959" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/6/959.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459473" id="node459473--2" data-pisa="ggg;3/6/959" data-pisa-master="ggg;g3.112.004994" data-apath="/ggg/3/6/959.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Resequencing and Comparative Genomics of <em>Stagonospora nodorum</em>: Sectional Gene Absence and Effector Discovery</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Robert Andrew Syme</span>, <span class="highwire-citation-author" data-delta="1">James K. Hane</span>, <span class="highwire-citation-author" data-delta="2">Timothy L. Friesen</span> and <span class="highwire-citation-author" data-delta="3">Richard P. Oliver</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">959-969;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.004994 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459473--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Robert Andrew Syme </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Australian Centre for Necrotrophic Fungal Pathogens, Curtin University, Department of Environment and Agriculture, Bentley WA 6845, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BAndrew%2BSyme%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Syme%20RA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BAndrew%2BSyme%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">James K. Hane </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Molecular Pathology and Plant Pathology Laboratory, Centre for Environment and Life Sciences, CSIRO, Floreat WA 6014, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BK.%2BHane%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hane%20JK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BK.%2BHane%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Timothy L. Friesen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>U.S. Department of Agriculture, Agricultural Research Service, Cereal Crops Research Unit, Fargo, North Dakota 58102-2765</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Timothy%2BL.%2BFriesen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Friesen%20TL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATimothy%2BL.%2BFriesen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Richard P. Oliver </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Australian Centre for Necrotrophic Fungal Pathogens, Curtin University, Department of Environment and Agriculture, Bentley WA 6845, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Richard%2BP.%2BOliver%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Oliver%20RP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARichard%2BP.%2BOliver%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:richard.oliver@curtin.edu.au" class="" data-icon-position="" data-hide-link-title="0">richard.oliver@curtin.edu.au</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/3/6/959" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/3/6/959&amp;title=Resequencing%20and%20Comparative%20Genomics%20of%20Stagonospora%20nodorum%3A%20Sectional%20Gene%20Absence%20and%20Effector%20Discovery" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/3/6/959&amp;title=Resequencing%20and%20Comparative%20Genomics%20of%20Stagonospora%20nodorum%3A%20Sectional%20Gene%20Absence%20and%20Effector%20Discovery" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/3/6/959&amp;title=Resequencing%20and%20Comparative%20Genomics%20of%20Stagonospora%20nodorum%3A%20Sectional%20Gene%20Absence%20and%20Effector%20Discovery" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/3/6/959&amp;text=Resequencing%20and%20Comparative%20Genomics%20of%20Stagonospora%20nodorum%3A%20Sectional%20Gene%20Absence%20and%20Effector%20Discovery" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/3/6/959&amp;title=Resequencing%20and%20Comparative%20Genomics%20of%20Stagonospora%20nodorum%3A%20Sectional%20Gene%20Absence%20and%20Effector%20Discovery" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/3/6/959&amp;t=Resequencing%20and%20Comparative%20Genomics%20of%20Stagonospora%20nodorum%3A%20Sectional%20Gene%20Absence%20and%20Effector%20Discovery" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/3/6/959&amp;title=Resequencing%20and%20Comparative%20Genomics%20of%20Stagonospora%20nodorum%3A%20Sectional%20Gene%20Absence%20and%20Effector%20Discovery" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/3/6/959&amp;title=Resequencing%20and%20Comparative%20Genomics%20of%20Stagonospora%20nodorum%3A%20Sectional%20Gene%20Absence%20and%20Effector%20Discovery" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459473/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459473" id="node459473--3" data-pisa="ggg;3/6/959" data-pisa-master="ggg;g3.112.004994" data-apath="/ggg/3/6/959.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Resequencing and Comparative Genomics of <em>Stagonospora nodorum</em>: Sectional Gene Absence and Effector Discovery</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Robert Andrew Syme</span>, <span class="highwire-citation-author" data-delta="1">James K. Hane</span>, <span class="highwire-citation-author" data-delta="2">Timothy L. Friesen</span> and <span class="highwire-citation-author" data-delta="3">Richard P. Oliver</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">959-969;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.004994 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459473--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Robert Andrew Syme </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Australian Centre for Necrotrophic Fungal Pathogens, Curtin University, Department of Environment and Agriculture, Bentley WA 6845, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BAndrew%2BSyme%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Syme%20RA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BAndrew%2BSyme%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">James K. Hane </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Molecular Pathology and Plant Pathology Laboratory, Centre for Environment and Life Sciences, CSIRO, Floreat WA 6014, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BK.%2BHane%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hane%20JK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BK.%2BHane%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Timothy L. Friesen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>U.S. Department of Agriculture, Agricultural Research Service, Cereal Crops Research Unit, Fargo, North Dakota 58102-2765</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Timothy%2BL.%2BFriesen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Friesen%20TL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATimothy%2BL.%2BFriesen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Richard P. Oliver </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Australian Centre for Necrotrophic Fungal Pathogens, Curtin University, Department of Environment and Agriculture, Bentley WA 6845, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Richard%2BP.%2BOliver%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Oliver%20RP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARichard%2BP.%2BOliver%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:richard.oliver@curtin.edu.au" class="" data-icon-position="" data-hide-link-title="0">richard.oliver@curtin.edu.au</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=959&amp;rft.epage=969&amp;rft.atitle=Resequencing%20and%20Comparative%20Genomics%20of%20Stagonospora%20nodorum%3A%20Sectional%20Gene%20Absence%20and%20Effector%20Discovery&amp;rft.volume=3&amp;rft.issue=6&amp;rft.date=2013-06-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Syme%2C+Robert+Andrew&amp;rft.au=Hane%2C+James+K.&amp;rft.au=Friesen%2C+Timothy+L.&amp;rft.au=Oliver%2C+Richard+P."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459473/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459473/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459473/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459473/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459473/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459473/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459473/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459473/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459473/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459473/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459473/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459473/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/3/6/959" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-20" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/3/6/959.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/3/6/959.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/3/6/959.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/3/6/959.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
