<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/3/6/929" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/3/6/929.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/3/6/929.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/3/6/929.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/6.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/3/6/929.atom" />
<meta name="HW.pisa" content="ggg;3/6/929" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A Nucleosomal Region Important for Ensuring Proper Interactions Between the Transcription Elongation Factor Spt16 and Transcribed Genes in Saccharomyces cerevisiae" />
<meta name="DC.Identifier" content="10.1534/g3.113.005926" />
<meta name="DC.Date" content="2013-06-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2013 Nguyen et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The highly conserved FACT (FAcilitates Chromatin Transactions) histone chaperone assists in the transcription elongation process first by facilitating the removal of histones in front of transcribing RNA polymerase II (Pol II) and then by contributing to nucleosome reassembly in the wake of Pol II passage. Whereas it is well established that FACT localizes across actively transcribed genes, the mechanisms that regulate FACT recruitment to and disengagement from chromatin during transcription still remain to be elucidated. Using the Saccharomyces cerevisiae model system, we previously showed that a histone H3 mutant—H3-L61W—greatly perturbs interactions between the yeast FACT (yFACT) complex and chromatin during transcription, resulting in a pronounced shift in yFACT occupancy toward the 3′ ends of transcribed genes. In the present study we report that two histone H4 mutants—H4-R36A and H4-K31E—alter the association pattern of the yFACT subunit [Spt16][1] across transcribed genes in a fashion similar to that seen for H3-L61W. Interestingly, H4-R36, H4-K31, and H3-L61 are in close proximity to each other on the side of the nucleosome. We also provide evidence that the H4-R36A and H3-L61W mutants impair proper [Spt16][1]−chromatin interactions by perturbing a common process. Collectively, our results suggest that a nucleosomal region encompassing the H4-R36, H4-K31, and H3-L61 residues plays an important role in ensuring proper association of yFACT across transcribed genes.

 [1]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175" />
<meta name="DC.Contributor" content="Hoai-Trang T. Nguyen" />
<meta name="DC.Contributor" content="William Wharton" />
<meta name="DC.Contributor" content="Jennifer A. Harper" />
<meta name="DC.Contributor" content="James R. Dornhoffer" />
<meta name="DC.Contributor" content="Andrea A. Duina" />
<meta name="article:published_time" content="2013-06-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="A Nucleosomal Region Important for Ensuring Proper Interactions Between the Transcription Elongation Factor Spt16 and Transcribed Genes in Saccharomyces cerevisiae" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The highly conserved FACT (FAcilitates Chromatin Transactions) histone chaperone assists in the transcription elongation process first by facilitating the removal of histones in front of transcribing RNA polymerase II (Pol II) and then by contributing to nucleosome reassembly in the wake of Pol II passage. Whereas it is well established that FACT localizes across actively transcribed genes, the mechanisms that regulate FACT recruitment to and disengagement from chromatin during transcription still remain to be elucidated. Using the &lt;i&gt;Saccharomyces cerevisiae&lt;/i&gt; model system, we previously showed that a histone H3 mutant—H3-L61W—greatly perturbs interactions between the yeast FACT (yFACT) complex and chromatin during transcription, resulting in a pronounced shift in yFACT occupancy toward the 3′ ends of transcribed genes. In the present study we report that two histone H4 mutants—H4-R36A and H4-K31E—alter the association pattern of the yFACT subunit Spt16 across transcribed genes in a fashion similar to that seen for H3-L61W. Interestingly, H4-R36, H4-K31, and H3-L61 are in close proximity to each other on the side of the nucleosome. We also provide evidence that the H4-R36A and H3-L61W mutants impair proper Spt16−chromatin interactions by perturbing a common process. Collectively, our results suggest that a nucleosomal region encompassing the H4-R36, H4-K31, and H3-L61 residues plays an important role in ensuring proper association of yFACT across transcribed genes.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2013/06/01" />
<meta name="citation_mjid" content="ggg;3/6/929" />
<meta name="citation_id" content="3/6/929" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/3/6/929" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/3/6/929.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/3/6/929.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/3/6/929.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.113.005926" />
<meta name="citation_pmid" content="23576521" />
<meta name="citation_volume" content="3" />
<meta name="citation_issue" content="6" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="929" />
<meta name="citation_lastpage" content="940" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Hoai-Trang T. Nguyen" />
<meta name="citation_author_institution" content="Biology Department, Hendrix College, Conway, Arkansas 72032" />
<meta name="citation_author" content="William Wharton" />
<meta name="citation_author_institution" content="Biology Department, Hendrix College, Conway, Arkansas 72032" />
<meta name="citation_author" content="Jennifer A. Harper" />
<meta name="citation_author_institution" content="Biology Department, Hendrix College, Conway, Arkansas 72032" />
<meta name="citation_author" content="James R. Dornhoffer" />
<meta name="citation_author_institution" content="Biology Department, Hendrix College, Conway, Arkansas 72032" />
<meta name="citation_author" content="Andrea A. Duina" />
<meta name="citation_author_institution" content="Biology Department, Hendrix College, Conway, Arkansas 72032" />
<meta name="citation_author_email" content="duina@hendrix.edu" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=N. Avvakumov;citation_author=A. Nourani;citation_author=J. Cote;citation_title=Histone chaperones: modulators of chromatin marks.;citation_pages=502-514;citation_volume=41;citation_year=2011;citation_issue=5;citation_pmid=21362547;citation_doi=10.1016/j.molcel.2011.02.013" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=R. Belotserkovskaya;citation_author=S. Oh;citation_author=V. A. Bondarenko;citation_author=G. Orphanides;citation_author=V. M. Studitsky;citation_title=FACT Facilitates Transcription-Dependent Nucleosome Alteration;citation_pages=1090-1093;citation_volume=301;citation_year=2003;citation_issue=5636;citation_pmid=12934006;citation_doi=10.1126/science.1085703" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=N. K. Brewster;citation_author=G. C. Johnston;citation_author=R. A. Singer;citation_title=Characterization of the CP Complex, an Abundant Dimer of Cdc68 and Pob3 Proteins That Regulates Yeast Transcriptional Activation and Chromatin Repression;citation_pages=21972-21979;citation_volume=273;citation_year=1998;citation_issue=34;citation_pmid=9705338;citation_doi=10.1074/jbc.273.34.21972" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=N. K. Brewster;citation_author=G. C. Johnston;citation_author=R. A. Singer;citation_title=A Bipartite Yeast SSRP1 Analog Comprised of Pob3 and Nhp6 Proteins Modulates Transcription;citation_pages=3491-3502;citation_volume=21;citation_year=2001;citation_issue=10;citation_pmid=11313475;citation_doi=10.1128/MCB.21.10.3491-3502.2001" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=E. I. Campos;citation_author=D. Reinberg;citation_title=Histones: annotating chromatin.;citation_pages=559-599;citation_volume=43;citation_year=2009;citation_pmid=19886812;citation_doi=10.1146/annurev.genet.032608.103928" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. Carvalho;citation_author=A. C. Raposo;citation_author=F. B. Martins;citation_author=A. R. Grosso;citation_author=S. C. Sridhara;citation_title=Histone methyltransferase SETD2 coordinates FACT recruitment with nucleosome dynamics during transcription.;citation_volume=41;citation_year=2013;citation_pmid=23325844;citation_doi=10.1093/nar/gks1472" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=V. Cheung;citation_author=G. Chua;citation_author=N. N. Batada;citation_author=C. R. Landry;citation_author=S. W. Michnick;citation_title=Chromatin- and transcription-related factors repress transcription from within coding regions throughout the Saccharomyces cerevisiae genome.;citation_pages=e277-e277;citation_volume=6;citation_year=2008;citation_issue=11;citation_pmid=18998772;citation_doi=10.1371/journal.pbio.0060277" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=J. Dai;citation_author=E. M. Hyland;citation_author=D. S. Yuan;citation_author=H. Huang;citation_author=J. S. Bader;citation_title=Probing nucleosome function: a highly versatile library of synthetic histone H3 and H4 mutants.;citation_pages=1066-1078;citation_volume=134;citation_year=2008;citation_issue=6;citation_pmid=18805098;citation_doi=10.1016/j.cell.2008.07.019" />
<meta name="citation_reference" content="citation_title=Genet. Res. Int.;citation_title=Histone chaperones Spt6 and FACT: similarities and differences in modes of action at transcribed genes.;citation_author=A. A. Duina;citation_volume=134;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=A. A. Duina;citation_author=F. Winston;citation_title=Analysis of a Mutant Histone H3 That Perturbs the Association of Swi/Snf with Chromatin;citation_pages=561-572;citation_volume=24;citation_year=2004;citation_issue=2;citation_pmid=14701730;citation_doi=10.1128/MCB.24.2.561-572.2004" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. A. Duina;citation_author=A. Rufiange;citation_author=J. Bracey;citation_author=J. Hall;citation_author=A. Nourani;citation_title=Evidence that the localization of the elongation factor Spt16 across transcribed genes is dependent upon histone H3 integrity in Saccharomyces cerevisiae.;citation_volume=177;citation_year=2007;citation_pmid=17603125;citation_doi=10.1534/genetics.106.067140" />
<meta name="citation_reference" content="citation_journal_abbrev=Mol Biosyst;citation_author=T. Formosa;citation_title=FACT and the reorganized nucleosome.;citation_pages=1085-1093;citation_volume=4;citation_year=2008;citation_issue=11;citation_pmid=18931784;citation_doi=10.1039/b812136b" />
<meta name="citation_reference" content="citation_journal_title=Biochimica et Biophysica Acta. Protein Structure and Molecular Enzymology;citation_journal_abbrev=Biochimica et Biophysica Acta. Protein Structure and Molecular Enzymology;citation_author=T. Formosa;citation_title=The role of FACT in making and breaking nucleosomes.;citation_pages=247-255;citation_volume=1819;citation_year=2012;citation_issue=3-4;citation_pmid=21807128;citation_doi=10.1016/j.bbagrm.2011.07.009" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO Journal;citation_author=T. Formosa;citation_author=P. Eriksson;citation_author=J. Wittmeyer;citation_author=J. Ginn;citation_author=Y. Yu;citation_title=Spt16-Pob3 and the HMG protein Nhp6 combine to form the nucleosome-binding factor SPN.;citation_pages=3506-3517;citation_volume=20;citation_year=2001;citation_issue=13;citation_pmid=11432837;citation_doi=10.1093/emboj/20.13.3506" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. Formosa;citation_author=S. Ruone;citation_author=M. D. Adams;citation_author=A. E. Olsen;citation_author=P. Eriksson;citation_title=Defects in SPT16 or POB3 (yFACT) in Saccharomyces cerevisiae Cause Dependence on the Hir/Hpc Pathway: Polymerase Passage May Degrade Chromatin Structure;citation_pages=1557-1571;citation_volume=162;citation_year=2002;citation_issue=4;citation_pmid=12524332" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Biol.;citation_author=S. J. Hainer;citation_author=J. A. Martens;citation_title=Identification of histone mutants that are defective for transcription-coupled nucleosome occupancy.;citation_volume=31;citation_year=2011;citation_pmid=21730290;citation_doi=10.1128/MCB.05195-11" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=H. Huang;citation_author=A. M. Maertens;citation_author=E. M. Hyland;citation_author=J. Dai;citation_author=A. Norris;citation_title=HistoneHits: a database for histone mutations and their phenotypes.;citation_volume=19;citation_year=2009;citation_pmid=19218532;citation_doi=10.1101/gr.083402.108" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=A. Jamai;citation_author=A. Puglisi;citation_author=M. Strubin;citation_title=Histone chaperone spt16 promotes redeposition of the original h3-h4 histones evicted by elongating RNA polymerase.;citation_pages=377-383;citation_volume=35;citation_year=2009;citation_issue=3;citation_pmid=19683500;citation_doi=10.1016/j.molcel.2009.07.001" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=C. D. Kaplan;citation_author=L. Laprade;citation_author=F. Winston;citation_title=Transcription Elongation Factors Repress Transcription Initiation from Cryptic Sites;citation_pages=1096-1099;citation_volume=301;citation_year=2003;citation_issue=5636;citation_pmid=12934008;citation_doi=10.1126/science.1087374" />
<meta name="citation_reference" content="citation_journal_title=EMBO J.;citation_author=M. Kim;citation_author=S. H. Ahn;citation_author=N. J. Krogan;citation_author=J. F. Greenblatt;citation_author=S. Buratowski;citation_title=Transitions in RNA polymerase II elongation complexes at the 3′ ends of genes.;citation_volume=23;citation_year=2004;citation_pmid=14739930;citation_doi=10.1038/sj.emboj.7600053" />
<meta name="citation_reference" content="citation_journal_title=Eukaryot. Cell;citation_author=A. Lloyd;citation_author=K. Pratt;citation_author=E. Siebrasse;citation_author=M. D. Moran;citation_author=A. A. Duina;citation_title=Uncoupling of the patterns of chromatin association of different transcription elongation factors by a histone H3 mutant in Saccharomyces cerevisiae.;citation_volume=8;citation_year=2009;citation_pmid=19047363;citation_doi=10.1128/EC.00348-08" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=K. Luger;citation_author=A. W. Mader;citation_author=R. K. Richmond;citation_author=D. F. Sargent;citation_author=T. J. Richmond;citation_title=Crystal structure of the nucleosome core particle at 2.8 A resolution.;citation_pages=251-260;citation_volume=389;citation_year=1997;citation_issue=6648;citation_pmid=9305837;citation_doi=10.1038/38444" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in structural biology;citation_journal_abbrev=Curr Opin Struct Biol;citation_author=K. Luger;citation_author=T. J. Richmond;citation_title=DNA binding within the nucleosome core.;citation_pages=33-40;citation_volume=8;citation_year=1998;citation_issue=1;citation_pmid=9519294;citation_doi=10.1016/S0959-440X(98)80007-9" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=P. B. Mason;citation_author=K. Struhl;citation_title=The FACT Complex Travels with Elongating RNA Polymerase II and Is Important for the Fidelity of Transcriptional Initiation In Vivo;citation_pages=8323-8333;citation_volume=23;citation_year=2003;citation_issue=22;citation_pmid=14585989;citation_doi=10.1128/MCB.23.22.8323-8333.2003" />
<meta name="citation_reference" content="citation_journal_title=Nature structural &amp; molecular biology;citation_journal_abbrev=Nat Struct Mol Biol;citation_author=A. Mayer;citation_author=M. Lidschreiber;citation_author=M. Siebert;citation_author=K. Leike;citation_author=J. Soding;citation_title=Uniform transitions of the general RNA polymerase II transcription complex.;citation_pages=1272-1278;citation_volume=17;citation_year=2010;citation_issue=10;citation_pmid=20818391;citation_doi=10.1038/nsmb.1903" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=C. N. Myers;citation_author=G. B. Berner;citation_author=J. H. Holthoff;citation_author=K. Martinez-Fonts;citation_author=J. A. Harper;citation_title=Mutant versions of the S. cerevisiae transcription elongation factor Spt16 define regions of Spt16 that functionally interact with histone H3.;citation_pages=e20847-e20847;citation_volume=6;citation_year=2011;citation_issue=6;citation_pmid=21673966;citation_doi=10.1371/journal.pone.0020847" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. Orphanides;citation_author=W. H. Wu;citation_author=W. S. Lane;citation_author=M. Hampsey;citation_author=D. Reinberg;citation_title=The chromatin-specific transcription elongation factor FACT comprises human SPT16 and SSRP1 proteins.;citation_pages=284-288;citation_volume=400;citation_year=1999;citation_issue=6741;citation_pmid=10421373;citation_doi=10.1038/22350" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=S. J. Petesch;citation_author=J. T. Lis;citation_title=Overcoming the nucleosome barrier during transcript elongation.;citation_pages=285-294;citation_volume=28;citation_year=2012;citation_issue=6;citation_pmid=22465610;citation_doi=10.1016/j.tig.2012.02.005" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=O. J. Rando;citation_author=F. Winston;citation_title=Chromatin and Transcription in Yeast;citation_pages=351-387;citation_volume=190;citation_year=2012;citation_issue=2;citation_pmid=22345607;citation_doi=10.1534/genetics.111.132266" />
<meta name="citation_reference" content="citation_title=Methods in Yeast Genetics: A Laboratory Course Manual;citation_author=M. D. Rose;citation_author=F. Winston;citation_author=P. Hieter;citation_year=1990" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=S. Ruone;citation_author=A. R. Rhoades;citation_author=T. Formosa;citation_title=Multiple Nhp6 molecules are required to recruit Spt16-Pob3 to form yFACT complexes and to reorganize nucleosomes.;citation_volume=278;citation_year=2003;citation_pmid=12952948;citation_doi=10.1074/jbc.M307291200" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=M. A. Schwabish;citation_author=K. Struhl;citation_title=Evidence for Eviction and Rapid Deposition of Histones upon Transcriptional Elongation by RNA Polymerase II;citation_pages=10111-10117;citation_volume=24;citation_year=2004;citation_issue=23;citation_pmid=15542822;citation_doi=10.1128/MCB.24.23.10111-10117.2004" />
<meta name="citation_reference" content="citation_journal_title=Annual review of biochemistry;citation_journal_abbrev=Annu Rev Biochem;citation_author=L. A. Selth;citation_author=S. Sigurdsson;citation_author=J. Q. Svejstrup;citation_title=Transcript Elongation by RNA Polymerase II.;citation_pages=271-293;citation_volume=79;citation_year=2010;citation_pmid=20367031;citation_doi=10.1146/annurev.biochem.78.062807.091425" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=T. Stuwe;citation_author=M. Hothorn;citation_author=E. Lejeune;citation_author=V. Rybin;citation_author=M. Bortfeld;citation_title=The FACT Spt16 “peptidase” domain is a histone H3–H4 binding module.;citation_volume=105;citation_year=2008;citation_pmid=18579787;citation_doi=10.1073/pnas.0712293105" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=A. P. VanDemark;citation_author=H. Xin;citation_author=L. McCullough;citation_author=R. Rawlins;citation_author=S. Bentley;citation_title=Structural and functional analysis of the Spt16p N-terminal domain reveals overlapping roles of yFACT subunits.;citation_volume=283;citation_year=2008;citation_pmid=18089575;citation_doi=10.1074/jbc.M708682200" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO Journal;citation_author=C. L. White;citation_author=R. K. Suto;citation_author=K. Luger;citation_title=Structure of the yeast nucleosome core particle reveals fundamental changes in internucleosome interactions;citation_pages=5207-5218;citation_volume=20;citation_year=2001;citation_issue=18;citation_pmid=11566884;citation_doi=10.1093/emboj/20.18.5207" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=D. D. Winkler;citation_author=K. Luger;citation_title=The histone chaperone FACT: structural insights and mechanisms for nucleosome reorganization.;citation_volume=286;citation_year=2011;citation_pmid=21454601;citation_doi=10.1074/jbc.R110.180778" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=D. D. Winkler;citation_author=U. M. Muthurajan;citation_author=A. R. Hieb;citation_author=K. Luger;citation_title=Histone chaperone FACT coordinates nucleosome interaction through multiple synergistic binding events.;citation_volume=286;citation_year=2011;citation_pmid=21969370;citation_doi=10.1074/jbc.M111.301465" />
<meta name="citation_reference" content="citation_title=Transcription Regulation;citation_title=Analysis of SPT genes: a genetic approach toward analysis of TFIID, histones, and other transcription factors in yeast;citation_author=F. Winston;citation_pages=1271-1293;citation_year=1992" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=F. Winston;citation_author=C. Dollard;citation_author=S. L. Ricupero-Hovasse;citation_title=Construction of a set of convenient Saccharomyces cerevisiae strains that are isogenic to S288C.;citation_pages=53-55;citation_volume=11;citation_year=1995;citation_issue=1;citation_pmid=7762301;citation_doi=10.1002/yea.320110107" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=H. Xin;citation_author=S. Takahata;citation_author=M. Blanksma;citation_author=L. McCullough;citation_author=D. J. Stillman;citation_title=yFACT induces global accessibility of nucleosomal DNA without H2A-H2B displacement.;citation_pages=365-376;citation_volume=35;citation_year=2009;citation_issue=3;citation_pmid=19683499;citation_doi=10.1016/j.molcel.2009.06.024" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A Nucleosomal Region Important for Ensuring Proper Interactions Between the Transcription Elongation Factor Spt16 and Transcribed Genes in Saccharomyces cerevisiae" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/6.cover-source.jpg" />
<meta name="twitter:description" content="The highly conserved FACT (FAcilitates Chromatin Transactions) histone chaperone assists in the transcription elongation process first by facilitating the removal of histones in front of transcribing RNA polymerase II (Pol II) and then by contributing to nucleosome reassembly in the wake of Pol II passage. Whereas it is well established that FACT localizes across actively transcribed genes, the mechanisms that regulate FACT recruitment to and disengagement from chromatin during transcription still remain to be elucidated. Using the Saccharomyces cerevisiae model system, we previously showed that a histone H3 mutant—H3-L61W—greatly perturbs interactions between the yeast FACT (yFACT) complex and chromatin during transcription, resulting in a pronounced shift in yFACT occupancy toward the 3′ ends of transcribed genes. In the present study we report that two histone H4 mutants—H4-R36A and H4-K31E—alter the association pattern of the yFACT subunit [Spt16][1] across transcribed genes in a fashion similar to that seen for H3-L61W. Interestingly, H4-R36, H4-K31, and H3-L61 are in close proximity to each other on the side of the nucleosome. We also provide evidence that the H4-R36A and H3-L61W mutants impair proper [Spt16][1]−chromatin interactions by perturbing a common process. Collectively, our results suggest that a nucleosomal region encompassing the H4-R36, H4-K31, and H3-L61 residues plays an important role in ensuring proper association of yFACT across transcribed genes.

 [1]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175" />
<meta name="og-title" property="og:title" content="A Nucleosomal Region Important for Ensuring Proper Interactions Between the Transcription Elongation Factor Spt16 and Transcribed Genes in Saccharomyces cerevisiae" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/3/6/929" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The highly conserved FACT (FAcilitates Chromatin Transactions) histone chaperone assists in the transcription elongation process first by facilitating the removal of histones in front of transcribing RNA polymerase II (Pol II) and then by contributing to nucleosome reassembly in the wake of Pol II passage. Whereas it is well established that FACT localizes across actively transcribed genes, the mechanisms that regulate FACT recruitment to and disengagement from chromatin during transcription still remain to be elucidated. Using the Saccharomyces cerevisiae model system, we previously showed that a histone H3 mutant—H3-L61W—greatly perturbs interactions between the yeast FACT (yFACT) complex and chromatin during transcription, resulting in a pronounced shift in yFACT occupancy toward the 3′ ends of transcribed genes. In the present study we report that two histone H4 mutants—H4-R36A and H4-K31E—alter the association pattern of the yFACT subunit [Spt16][1] across transcribed genes in a fashion similar to that seen for H3-L61W. Interestingly, H4-R36, H4-K31, and H3-L61 are in close proximity to each other on the side of the nucleosome. We also provide evidence that the H4-R36A and H3-L61W mutants impair proper [Spt16][1]−chromatin interactions by perturbing a common process. Collectively, our results suggest that a nucleosomal region encompassing the H4-R36, H4-K31, and H3-L61 residues plays an important role in ensuring proper association of yFACT across transcribed genes.

 [1]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/6.cover-source.jpg" />
    <title>A Nucleosomal Region Important for Ensuring Proper Interactions Between the Transcription Elongation Factor Spt16 and Transcribed Genes in Saccharomyces cerevisiae | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"skxNjS_bGJalYrcjflvRFrZnC3dsi0D2X1eGkvueDyU","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459478","apath":"\/ggg\/3\/6\/929.atom","pisa":"ggg;3\/6\/929","ac":{"\/ggg\/3\/6\/929.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/3\/6\/929.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;3\/6\/929"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;3\/6\/929"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/3\/6\/929":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022LoJ4u5aCl3hPa61\\\\\\\/YW5tYJRSdQtJSRCx1cZ03g2X33MlagSCfTSOezkZz\\\\\\\/cFFsJ1hgBpV5UiXrhAtxgL2f\\\\\\\/ia6QR7QkBB4x8zl09iUYeb8es0SA0h9jTWzFl9Sbs16qGmjz0lI1T95wuEfgofEVx434Ib+XaescFdX1XNTEM\\\\\\\/4Y2Am5CpXTLhODVvt\\\\\\\/U2jIPP793rasnjdiSv9qB2RdKNYWnuMVOjm4kah9q3VuPqBQP8vMfFazHqqY25xC0LNlQuW7blg3qqCW3ZJmv1iX26ohW7GXafkZ2Nw+VnU7AHwA71rxX1knj9RMlnR9NWlRNgLXiuxgOaCYNy2bmtKrgREei9eSWK+bE\\\\\\\/7yGJ9Xta6FuVuTpbNoRaXH6eplTcpJ4Gtx3en4wqXTjj59gTBp0c4jXKbao7Pkvm995r+zg2tJmZfxivBgH\\\\\\\/QCAaTiC8qczMr0nBuENVx7VOAHg\\\\\\\/K0cYlzWr58q\\\\\\\/6T4tYNxeTmy7oRhZth8HfWkKN9S3R0FB6Q1+7XN2rR3Q2DI8FrRYLw7eQO+fLMYPHgtUCHFaOzJ\\\\\\\/\\\\\\\/zUAT6g835cpW\\\\\\\/Z8Fbh4UXSbwSrw5O60yAQyf+1OCj7NYyf8FJ0P\\\\\\\/9WYDxdruzCuC1A3k9W\\\\\\\/TMzn6KRxjTPulcw72\\\\\\\/oucnqeIyu3fYR+pSYOwMsGXplDmoliZ+E\\\\\\\/9g4WoFD6P9vVDKPWH86AfFmX3eqznPSvx0SvGi8h2GahIiH\\\\\\\/uQmkDqhNFkBGJcfli4cGHC2Kzi7+WHuxR0\\\\\\\/IP8BdgfIhQB7tWXs35T0WyiJxcwxJJs2J9WCb5NMzpt0Bnl7faflY4kD5sUlekn4CvsJqKTgE5ptQpTqCnuUmaD1HNuvOrYGs9kdNJniU315FxouBT4bKhJ3LKQLyBMhcc88hqGsOafLpa0NNNGykTc5SRu6kQmhemotNjUYrFZdSLAD5\\\\\\\/3TZAqVDA7Eg6o1qmk1oPX8kkv2QDtNq0pW9kl7jvwogvfQr47\\\\\\\/o1oqihgKNr8cL7YEZZUafDhNiI2Sr2b13kPfWxUVAdLDesBizCoOkh7LK\\\\\\\/ACxk136lkbHUOABWEdSbGCFvQRVgfkk0pWj3B1Yaw66tPTypRG6CXMdK0KM5bOnfM+X0dypiw1f8BK3eR0\\\\\\\/9BUrceEenM5Xw8MG6MfbjRGdCkkLY3PoZGi+mpQ1YmL8ST\\\\\\\/fcrS3H46rBH6ihFWTSWEv5FMLNnUPJDShAUtmRpdzYgoQw+MFrtR4Zhhm4X6\\\\\\\/QlCqMHcY8etmo7TGkaSUmhlBq+T+OfNA\\\\\\\/VKb8ueOer\\\\\\\/\\\u0022,\\\u0022iv\\\u0022:\\\u0022Ii617wCI6jZSSccutUFNtg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022042c7160d4bab90cd8049987e6aa23c56732365fb1ec49f26dd7eefe7e267e13\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022YSfyCLBULNL4jbnF1WIcadaA7FcReLcIGiW5iJX+CH4LqFVBnktiQnb9zgv9p2wEk+Hgpr6jeuv1JpOQE7ELxBNntjYzheH\\\\\\\/9ENvTjUdHOO3OoLl1Avv5tdjFnenqxsKRKdH\\\\\\\/hkhU6ws6bT0lLO75NEnKZSrCEQ52AO64Ns7tEey8tDSRHU1QDgguNQWK94Z2O1SWDWL3iMntUmd5rjZwKB919qHGbaWG6+oNqezEh83skW1bQjEnNvHGsc9g++MSKvlXSe9bWbSlEpwn0jS8ikQfRix\\\\\\\/c64XVJfljlErUyFHkBpzTqQl1Z1BzHSm13gvJdtAPkEhn6eHreWc8dIpQKHAUS4yfH2JmW\\\\\\\/fpP9AwmFXfxtIXfnWeuyaassyjOhrgBBM5lQrevQAlSgbon9FWnYzstvL1dVtvYtsVNw+DTIZOfZBu4\\\\\\\/Bx10n3lhRtaOHmh+4uwse1AWrgevkq+FqQzqqRGzPveBSrB\\\\\\\/vb28Qg4dnFAsQpxIac8m3ND1pAQijhUq3N5TLBpEqu5kOxTniyHdyiIX1rSqtX3aVHxuwmroNMt\\\\\\\/2ddO2SuG2PNVrv3960kL4HDxHqA3wKYHiuBVA2cvgfs+MW7k\\\\\\\/tL09hcNkEWHVOqQBJhg4d6IyPjX5MUlVmtgU9zX31KwYaIf4reZYzD6ZRgDoOa6Ge5lxvQ89l8XMoIziXW4W9c4B4geNwkEA\\\\\\\/a3S51FTE+PhfnX0HXr8WIaaakOprkLpnn9ax9wGfSWeDXTm0m24vt9JUn8RdEXFLyxICf+MAWIRgZnffpn12g5nz0m4aZGoPUDst5mBle+3sO+m3YAiOO0bA\\\\\\\/xE0n7IPA3NXQDOyXVfQuBZivET68IxmHf4vId22YqxyMtmJsD6YuRG4FhOsrOdAF5vNVoGQsRxydQE1pnXEq6KaXSA1JJu65DhUHoXN+FAEt9Xx6k08bzzTWnf6\\\\\\\/mgo691avXRj9oNI\\\\\\\/LJROKBcNEZjxbhvFoyBCx+5XldVHrt+9ODMvmrgtpZg\\\\\\\/RlDbb\\\\\\\/w0NUSKVHH8HrgAtoQnvj2f5SakZkFEoiSghR9eT1vTnBaUHjXn2qeaEbLADycYV11je6ncxR\\\\\\\/KzUYI5phJSlT4pqZqvHHFdSGgUflsG7BnSnsdFqU3zW\\\\\\\/L4ujFKpRisqp\\\\\\\/iiVILCkdoSr3rHrs4ICgqMmpPjdFq3RmpgrrnHP4GWR+\\\\\\\/TPLKqMfXxy1y\\\\\\\/yL6Bpyb+ld1TjoiZsz4Fe5k3i+q9dnLKEtfu0dpNr4\\\\\\\/KiAqfso8cvhBTt1IBaThW\\\\\\\/On0\\\\\\\/9yxX7WLnJFtgOVXj8Vnj431oBWPs7rvubbMxJa3N4DdFsOoFYTqIKiFSjiU5JsQ12e8DV18Uug37d+zB13DeeZzhhhmJipoQTBkIJcnPOcy94SsRWV\\\\\\\/vfFLC\\\\\\\/ZFiecbAw3kIfub6iv6AOBCfSAabD8CYskbbRzRk6e6u\\\\\\\/LQrLcg\\\\\\\/EqKzCRNLi4By4Ijru2vhm09pmCS\\\\\\\/xgvQcCYHDkEEN12rAVkdBgaVEr3mIXYHYBpEr7TvSGC154+PUa51vlgXD5O5v7lA4lm2M3l17SNKFGdlcm2mcFTjeYeRbwOMAGrLDvcQZyd9MZPaebXOjeBmHQE8m1wl7Ou\\\\\\\/aOTqkunT7PZwRW62OLwaKsa95KWRbkDFScQAKnJPYt\\\\\\\/NMk\\\\\\\/ttX1BAlRrolGLeFm0SdN1WugzwDyFO8ZCP6qsmOqPA7AJ7BwEP9F0L8vjclCNZVgSiNBo5WseTpO6gFAL7eWIcg6SIoiWAUnPiW0JhXWTbrpjgljn5pU+0W0xYWiCmDQEQQoHnbjw\\\\\\\/OnOjk+5hLJj3BCFvZuBhy5HicQWRZLRKPEF61iJpRtFyVE2I6zb6nLMk2hhLRhZ3R4\\\\\\\/FEih7G+FDFQXlmzHtvb9M2MTqm0OU+InX5phYkzQrmZq6UhlLFJnlo5NumMH55p0UMnMsUlGr3hDwSH3uQKnbsBXCcEzvq2cI\\\\\\\/HEUqXUjFZFtjkqpRIOdVvhXHK112V4iEheppbpFDD8ENrsHxWeA+RoTi6rkzzSRbOc2RQX6UlL60SqPgJ181YajITfQUJ1BDDDUx9KPzVzOYv0bdlKjhBDCUUYZ7yzCqO\\\\\\\/9REd7GMqRNZBcJBL0ov3TLaUeDIl8QgF9R8sVj\\\\\\\/FTqDbRkllx\\\\\\\/GY4L9zZq2XOg5\\\\\\\/OyA1dDkTjEzkXuMcinjKzy7FL5IuYPvJ+mSCJAhTmET6oU1jferdrmAodaHWoxUTsBAE1\\\\\\\/ZdUQokPvglxvLHi+79kBTJCC7N1nkJnvTVO+YtezVt\\\\\\\/SNXNbUZ+PCdZZAFN4dELBh8vbe8Ejdr4c\\\\\\\/jabK1WZX6TdVVWQL1JmMvh5w4LV403rEunUvHeecw6Yfrc1zxBkHsAfkyeBvOmqFFvPvoWTPEywnsvzxVYjpEgOkVp1\\\\\\\/HRFv4\\\\\\\/0SCXDJJMDpRahnKlqfZm3GJvBhV21nOxmDKQveXNjkmdg0LVr4leu6X9xDgPkX\\\\\\\/0iT+i45Q49ZH2+KnnPgadmCJ\\\\\\\/Ck2H0zX2I6i6SEQq\\\\\\\/I2TwfJolDNWQXrkjaUjvyvuSv10Oe1y7NwoIVx8g8Y6P9Qac5H930i5ztFUOwcYI47ofPAN5rDNoePcllDp1IbZtXc\\\\\\\/u9BwP0ygemD3Hw9XkziI7z1otP0o+LkX8uGJ+VfNXeacbQMKqa\\\\\\\/9BKeVaY91vqAGZFiht3kVYB2CTikW8PamMzxL09qZQDdy14QPxl3Blb5Vgn0t08un0Wd1p3l4AY0W+NG5rxsaBAHno7yBY7ZjNFAUrk1fFCU1TcTRxKY+4IbecP9Kch8gnuV5Z34Vyk4CMirTE3u9qIJXqBvOObau6jVYGVj0gGtznHccniv1c351b11M3GfhTyRk\\\\\\\/jw9y06Ne1Qz1DeACuamFEg5pxFfGUw8AceN4fUs3W3L8\\\\\\\/1sd2T+bzfx+myhjQg2pZCxLaL+ky0hyFMn\\\\\\\/bbtmCxIoJ3VsY9W8gcK991UjcYU59G9x8Y3OYvzpOTI26\\\\\\\/\\\\\\\/fNRQbXa3ZLvxjfYzFVxq\\\\\\\/zoEgDzRDrRqYW78aZxQQvoZx8Ovm2tI2MV3SOf3msQyH0Q7MuXTDYTP5wVh2srYlAUFYLHZpwPVzzWktLA\\\\\\\/csryE9ytisU1ml3w9nwTDcH0aHnwHYzKEBDUA6q5bYAbtualO+EgSqL9fPFMrPoR695bSyO\\\\\\\/296Sg34o7RBT0OpenLk\\\\\\\/P4YVXuBBL+gWZRuVNCR4Ec5yynpBJdWbCgrp0WerFbMtNR5DA+qNW8KjTJf0LHOJJEVq3eaAWPfJ3cv0ZBrdPga+70yHIyIgj9jHuImLQ30YwQ7BNtP838ZhLwcDgtiAVBDOuuLSaXLGZT7yNYG4mdZRtgs7LIhTuU2g7sRSz\\\\\\\/ZjWuhllU06tXNwfvcUjWbwunXOjPsrWroGLLC7y6ONid5ZHqZc+yC6sGGTZuHwbb5gTADz9bIciNrcf44SjHdOSvidXLQm5tPrpVdGosvy0NFSTHxOWrSESLryqRbNCITPo189vBbAJITnvnWGj0oc7GsIPprfu2xefCf3k4h3SmcqUMJa2NG7CPsVFScbhzDq\\\\\\\/qyZTIDhR2sCe\\\\\\\/sI2sjR2vTu+tnYZHOdYYDLkGcChW\\\\\\\/HiqKZvGMN17y9981qkuPHctjlAL\\\\\\\/tDHrGhNLXXXPu4GuemjPLS7W\\\\\\\/8uNqCbruItLQcpaR0+Jz7VZ4cviIi4XpKu9RRNTGDy8jvZ4TqFjZR7NY5Vo5d8ptM1\\\\\\\/Fd6QAqvyvt\\\\\\\/\\\\\\\/KPUodvOgx7i6pk7lk1YP+obfsAHuItz5+OT2\\\\\\\/zNG214qqlk81tI2y8E\\\\\\\/514WCjp0MisDlAliKC7oPGzpa0EwjfVhLyJvcOhe+ymm9q5fMbN1GmNqhsHhlX8VMdl90Q57D+tRovxDnc4HaHsLsmp1pPWZVe9trG3K7eoPAlRbPvW8xuefHa25vpCisr9fyDuKEA7iTrbCIS7E5vV0m9Nb8A4QWcOJ3yRRDsPCDO6oPqhi\\\\\\\/FKTfEHGSSR4QfRp31WaBuizl+4zr+COxluvilrdbemQQx206UbbsukbXHxryH3YOZtlHiMRqoxn9EFDjQ+A8j0bKL9bAYYYvsD8DRorRtTLQH+UY13byPDm2ObaAlyaaA7EgmlnMN+Di2A9U2vySbKQkbGbFx9vq2Do8vxW3gfxcAKlKCNEiPjDHm+yXJ7jXev6rluKaEFsp2e97QYL3kFC9gybXQZsEver+NtkQGgw6eFXpgVlBo+5\\\\\\\/FupfiJLi9iJ0zKuBN\\\\\\\/NubatYg\\\\\\\/dJrlRKE8MSrBnG5PXQUKAoukmGdTsF3KPM3N5\\\\\\\/jNSz\\\\\\\/Gf+vd1TcN1QG92uSbWWgsXeAKM52Mv\\\\\\\/KnkYYMNfwFwKcfGq26R838+cHtFDd+2JCDwQS6IxT6CdMKOvIJFc\\\\\\\/k379TmmljdeotRdb\\\\\\\/hpgzg\\\\\\\/Yq8sCJIG\\\\\\\/7nCGAsxduDdPW62kW7DEr0BWgqs292KgCbWHd\\\\\\\/H9bP7P2+dnyFOIjSYFMm2B9Z8XvFYgkY2KVy6CBQlUuJtLCFTTQVyOEGwoWtEgCdTBkhfIF2jAnX4SOYfquyixwYKmysJ5w9dT5DOXuS+kcybxkGBtT9I9Sw4B0TwAx9aTkJoJSEQk7fcpHpPf8JQ3knqlt1ws3daXvejc25PFtizwbuXLxB7qK8TNsW+5aYdyoEAMtSLb7EsN24P96OHZeMfG8uS7pS6UTLY480Y\\\\\\\/xMqLlaU3\\\\\\\/23scv\\\\\\\/6v9kmVCQk\\\\\\\/XvxY8h1oBlLo6UwG8+84TgLcPZvIWHgzuU6Dt2xV+EavYRU7tbzhMGA5i2zCa+1lhl4EMrrm6C9P3Dxkd+rXWVvyEj0m8\\\\\\\/ysbBvTHdDC5UlrgR2dxvky8G+nKivO2q9F5f9dm\\\\\\\/vOr8Hi45ti0J6\\\\\\\/0YesteGjPHtsd4VDZj7cnQwV7\\\\\\\/hg3\\\\\\\/I0u32aM2V49l0qZM+aKl30UIsRdXeiG1UFRGA+xVoTL2K\\\\\\\/78wSgrH66cElWvVz5mvGmulSZox8bvsbTx9VcEaIuPKH7uvnd+w+Qo6DXm1OEe68M6JWpZ\\\\\\\/vXPK4cjN0WSSBxpyDgJTgGIEBLQwgv8E7ZQFm3\\\\\\\/qDSWlfsbvrOB7+Jo46+lCdyh8k7PhguK3pyd2clmuI1bqa0DcV1nBgEitjqfVCIbKAhlHjh4p89vvafeX8WYPs8hbMbgL1LvLqGBLRf+wtJ63X0fnfruFhAR0JyTJmhjwYRtmteoTfasQDTdjG8yV\\\\\\\/lTdptStW9k7mM1XFaEfTOHtxcF9eMrvF6asCIW79Ar12VN21eI1oV0mBvsJbEydwOVQbi\\\\\\\/NmN365NVOiCIuE2MHTar3UUKOeQkUPeGcoUsKkcfgkxZLhSv4KAxEVQE0KxBHAdj8C1fdOd6pMKIAf9ZXFTAABRXftoPOoZhjvTfAGh1UUAr6BX5LPjfTl6yLVOpIX0hMs9D0MZlyErqTnFXWm4Ld10gb9k6xbd224j3aJCRoBu0e4XoVRDYM2HNuacRk+r+fY5\\\\\\\/s1yaYFY7y61MG+qloP4rLBLHyCqwxv0JjTaLkdsjyqIM0EIq5xfRdcy3chsrB4IlI5G2yHnclHcOBD8a80HqChJwqs4lPJMuYyN82fBb\\\\\\\/AezaOZzB19d6X4Xrtwu\\\\\\\/nbHl9zOnWr5zE4zGHAFbslCV6wVUEr2e+uO6ZflD3rTnSBSDKTqa5GMVTtSQlnDneFJO37yKuPmHtJSt9saIILDxysFfsiZTdScbj7jn83fha72wQhalHnhALeqptp0cqCC2tool2ZjbK\\\\\\\/ZEbGbODhKuhxx\\\\\\\/6cU4E76jP+182Luz8a0THpAw9rZ4Nldy2\\\\\\\/UwT1i5OZAr9WMxM1YGlr8m\\\\\\\/hBnvYtznV0GR7HMQhvgAkVVHDSfMDzritEmIftClwQUN5jdzlOnqYkSHYtkBWf4NC+wJst0rGCoKdcZRy0B2HdpavLcb3NsfXcPvRNVm\\\\\\\/aaTAwpQS5Kv2tK3hUrXUdzXgl6+41eS\\\\\\\/QaqYhM+X2eSxkGjM3MUFufo\\\\\\\/9cLg7VJ6b9\\\\\\\/sNecSKQz+q2+buYNUWs9AavTbogZGl9A2B0P4hEpZFWvnImsejbBETyZG\\\\\\\/\\\\\\\/wLShn6jnLpdWEYRjOki2wb5yE9l6hTYu7nLtlwwFCiez4COj5VVKvn1T9YZADuBbDhQ1dQlT8FXiy\\\\\\\/cqY5ENOq6l3wSta3yy1lQgmfE3m+Udl33ssLDxM5XMZ9i9pmxzCik6Xrh6iJJuya3sm0Tyjm5BuU8s2MYomJ01OMQO7F0HW2MjtL9MS0C3rf3pMAQ3hafuBKVNG7BxuKbrHF3XPSdMaZqzjx\\\\\\\/PpAxpHEhmYdAAAguPYoBTlS\\\\\\\/P4SyzePa4kGJe2dUhqOtY6OA4pAv5BrS2QzQgiDnUhtUuacaU60RCZMgA5jWOfKagntGCs6rCHXKUXNnNkZ3umroPZzfE8SLUwkVo5YsfizLc6+M7j1uXsJaEfIaWiCskeb03jVCU+1llrv90dfUn\\\\\\\/SlpFaeof7tL39+1bY7jMFX5GEv5b\\\\\\\/Bipw5Zz5QL8lNLfolaQuJZhijS0R5dGz+3w\\\\\\\/PGPaPo1kABbVJcuYlz0MujlS+upKO45\\\\\\\/BvfgHmtT78P05yJrxuCbIIlpEOQB3NuG+Gkye8vc+k9Leqe7Enq7aYBbIoYyBoWLINcRt72D0wOpeqx1yne9Up0Ap0O2wBI0tq\\\\\\\/5bD8m7ekeZgZ2H5CESrAEqiqpGYCPpVlvL935EEfQlK3p2gL4AeBDpdeX7n5EV64kHW0MTgIgvinRjQ1bC6GPBUNSpQljtpniblLyx2+W2Y9g\\\\\\\/l8Nb18lwKER5jEniRdoBoNwl0HWwfuKhjSEa1UPL2CEk1Wtxl96TD3bfLlv4TtRuwGsoSgdvCbXYVMps+AoS4p81jx4yLTWeSe4dMXR2jfGNurJC9jhOYOoZ7eCuCFyiupLhyYqZ7UVidzrxxRuMycwJjlNw+JWVqie2RZ0KETrLvI99+uS71bBzTDKefO6DQaAOYvZiqxcuaJf55EqJM\\\\\\\/kzuMonkZAZfp2fh\\\\\\\/GdaiENfjE2FfWY\\\\\\\/7qWrZs9LRodRDqTJZD7iiNmKzSLcCovD84abmjcWZT+RizW66UktTldtMcfJLHCjVAD3QUPt1tEGecyrT\\\\\\\/u+f6GuWxt23fk6fj1uTEkT+EVVW2NLQKimYBJKitSISmYZzxC25aKcOq31khgv3eWjCTNLtyfWACi90FhSYP1Vc1GEGky6aEkjV7G9Q9Zpf1\\\\\\\/8PBxW6ZP0oK5x553zAs1sG5HuV9X0HBh5UD6jjw3fGIr2+aVHE6dpmL4LhZQnnMk1BMG2nh7nRA9oF3TpwZ+IrMxIKIrid4owMvK5fD65uaqlPKHMLP4MFgJ\\\\\\\/KefsOoTJJxpJtGSLSAtQlFzSmh8rb43T0\\\\\\\/uD\\\\\\\/bUszvQUEK5pmWO5Uod21+ejNq6zC7SnbeM4n8zuoOX7cN0tnX50e58ytE2qHXclaIa46A8Ghl9vK+Abmwh4HMqMS\\\\\\\/bYeJYhbQFUCG7DbugnrOl4M72p7iyqPFcs4if6tbQkYpilPG73jJOoRaaC9uSV0dzB2wbQFTd5EjZItmAD4l24hUWZ\\\\\\\/vbXb7XAR+NvsAnimN9ByjDbu6ChbuM2wwtEmh8aNhhO0fJPwsjWtdQ1G3BvYn2watoribLbFRWOepRigyXGcTHh2\\\\\\\/MaesqK8ThWVXFs6QCEvqxQoghOT84V0phJCiUxnnYCh9IJUCbkMP38tmfRS4S3eoNkjZTSww+bmqnWusl48WhIUZgtxsODsmL7ACECrc7lS\\\\\\\/Q2YWkYPWhFvMTdLpLU95iG0EtMqrH6dBYwQF4okV5vOv9UVATV01KWLsv7k8dMFeEJADXoPCoLKO3FMYNSeqxrmSpPFCwdA\\\\\\\/4lWRCDL3CMmdajCBG0gtFlHsgFoT2n0P3jvISNbiT2W8EBmp9Nq9jK4mxGtuSLoBDxvh+ZCZs+GE6Rxy9zsmOgKYUKk7keyZjbzGzox+R3NC4jVjEsKezDAsIsK5TGbwBv85s77E6uEw739NkNgbNi0Xdtaw7GFWSZ\\\\\\\/uFJEDCp9thANfHMPm5n4gcUG3XzKuTYIvKSyI6OYRLIa3ol6Pny1e8M6Rqn1b4QXMY00cifmbYhgiCD7+XYGkqqax7awgsOaWcDqc9Cv1Bd8J7Wi7hxI8b4K8OyGqEGr8Zk3O\\\\\\\/DvemjsK5j4fwkd46m0Uq04zy7l0lxfUCsWMzAH0\\\\\\\/awQXDDg\\\\\\\/tXevS1bbuE51Bm0yqrF8tM6F8Vav4zbK0EjG81f03HrWDtvGQK6z\\\\\\\/sSLZH7Gb5l1BNmsKI80eT0MepSxUDxQpRUXDCwswt+0w6RwJiUuim3LK3Ha4k6Pfaqk42L6Y82AMSRXI8ZGMneSeSK4HjbCtlF5mSCdx5bHwl5RnNTFC3uwrBXXvNYLI0FgiIoCudKhGCK7+DeGXe51D2AEZ0fR80qoKD4ts1fo4iW9KpR9ZthGaU8dIbwl9y\\\\\\\/9HkaQ43GU6rWHFLrYG9p\\\\\\\/wliZajJGzn\\\\\\\/OFtw0vHAJIiUEHuDBt2nudQirn1p4dp1XPMHL9uAIPJTUxcpZrM79extNChdo6KnLtKNsx8S+ZWFZAB+SkBWeuxUvKDsSEnV5louyxUfUY31CfXxEm\\\\\\\/NXxqOicTS1MfBtMF24azfpxCQ\\\\\\\/8YxScrtIuhDV5CMde9zmUPs9YYP7xrNHAmEqOb6qD9GXeJellJ9dfYhfVd0H6Xqi9KuWmNmG0AtmYsoGo3hjr9A2s5bgzGM6xW6pCy31ce7v6h9\\\\\\\/ynCECbQSH8xKk3IEo\\\\\\\/of97cwJAJD7WDVTBDqE8rhUctiv5NqMnst8YwKcTw\\\\\\\/UfjOEkh\\\\\\\/b5rHyZzDef3cCorjeAq0+6jwbnqNUzDC\\\\\\\/GPUlaKOv\\\\\\\/xrRLTAJ7AvSARoO2F5WA9vWISJFVGNdz+Z7suo\\\\\\\/oC1pTBa8+6n1kLTYsR6dIPaYeIpi7lCWUdC38fesP7A5bydSghBbSa057sCh8pZNkGamJI6oiqgydig1lJtaRno5\\\\\\\/p3vlbU0JNSGXGPAyQjZRCG+GaeNVhs9ls4p2en0XYYRjuuUQqw+jJ07bS5DvIxxu\\\\\\\/EH16FsDw3PxuifrUiA6TXpYzCgOc9ahKAJb7RaA0PfPVHYl2Qn7E5eLvqH9xRA8VIdyBGtRcWaFWsuAQ5d5VDRJcq\\\\\\\/2ivOoho4x+BPVww8iZjJhgrSuImJFjj\\\\\\\/6i8fM5reOmblgn1fH6Oea7GXlStvYdhFGpqJXXEdJMEfbCHtzPT6zqAK3RFLnUEmmSfQz1THh+ksGpTgO7u97EAT9EmcDk5q2S9WcPgW3RTmj9RtdK357JgojtJyOV5r5iwVOY\\\\\\\/NTfREbWuIQtYe22Tnq\\\\\\\/9SSdISi4HsbGgKiNjDvUs2y7thJ61NDe0he1Ofh+QwSHTYSa29+tJIS11G2Ymf3gm0f0KrYLMovHj\\\\\\\/5XLeBM1MlyPpsbb95v9+eXnJffqXJaz0zfz05snlXyV7Gu67SP3+MA1m8UGg24ZxZWiAaXd0ama\\\\\\\/gV\\\\\\\/S5tk\\\\\\\/hl5wTiPFrQ1NpbKt54hA9opwCIqTvqgPqlefak5ErZhtKcukuhaSEr5dOyxp0+3+tAh5\\\\\\\/4Jz+ypG8DjxatOhywcyAJQDnuQqWe7ID\\\\\\\/ewnnuUCpho3LBBDO3Xt138RUVNKAqhHSvRawjf26Mno1+f3np+XPN4b+WO0f40FtwTBnJeFyBwj27ccDX683nvKKsXTnV4xCLtZ6LyIiAuaRFg75stScr9rU2elnMSgN16Lru9UH6WW0V97Yx0qTkp25BJZK9QzB22gaM6D6EyQj52KCxNJjvJMpaZspaQtDzqSOgiN3ExexDgfagJI8rNCw3xldwPSOj+hibumFOZfICuPqOyjuSVXp2ja99jABZO82ncGyLmp0jj1LDi\\\\\\\/YNRS1r3ME08Agh536xXg9k3oK\\\\\\\/63p3UbKwPfBdvuGVVXCq6PN5ScRbcO2R+xDkInayu0FNearTiSAotOkBIpbPwc5S+ZOgebWol552aBWl4wFU6OrSwGxNzpuB3oPVri\\\\\\\/pq8VN+27u+DDiOQei7zivmGW\\\\\\\/ZoSZO9LLxTv+ujApnZVxX8Rd3cyjK\\\\\\\/3oHHSmQwN\\\\\\\/OHVGCu+RfwEfs8wyEoX3Wn37tSY5XlClL+2PnaITL\\\\\\\/Zriqvz27PZiR5AWzBhz7OeuqmMcGzCJHAcEnm3ypj86OzoJZ2fVj4TRCsVtlhcA+Uy7kIJPI3JywJir1Rq5e1d9lvnRsjfZgK5TmG9HTk0WL7k0FO760iAmsOcNA38LcZFQLWT2d769HPfv95yoDZQXF9BmPSkPDmU+25GH5qRA6U8DnlLJR9TQFAvWg\\\\\\\/xy1ehIVSgc6Qwi5+qluYby8GJja1SP84+jBv8NV8GH0LdL6KRAIJwvpmwecdoADeTBBrIctNv433xWHjMehENwYOIXh3LWdD\\\\\\\/2i1KN+SrUb8XaCIUiktupcAK425h80TaDqb\\\\\\\/2eL4pJOR5Xba9ZojpjEeRdsND1yZoxMMSCqlJ8RxZCuvRcJrLxyd\\\\\\\/arJeC4cxMX81GHutlLboGEKSaGcVmwSCFahoj5fOm3Y7zuirk5wN8w5FgWVM23CYRYQ74YJKH4IdAwJZ8kGme+ChB3N4bh4iWZMp48zuebaoVPpDLPZOoQNNQkiDFFvsJKjmzTxInOhyMDiKX1dJIR+eq8+Ch87B+Uj9B3gtrVl0XsjG2CbFbrusb1eRf\\\\\\\/RKQfdXjBuUNonZhCbYBxPtJUWAOQR6HklQ8Qqn5olCgqFj1rRem5\\\\\\\/qBozNYW6lHcye\\\\\\\/JX3Y8g6w2qPsVddAK6LuaMNU0Sk77BfuRRnFpoL\\\\\\\/IqK7VuuHcO0PGN+929VHbzhvrofob4z0cqUcxGG6c4g\\\\\\\/vLyPm7WfwqaVBRMJv34rmoIPT7Tm41ppXE3uDxluSvjF0dbM18WilsvzGLyXFMHRE+abTkWsX8qN5QjwEBejHHRVG7Nu5eP8xOBzULSlAkeBIFyYCCEo0V9Uhn7p8xm+6E0Rkp3B+m\\\\\\\/HBiw4c\\\\\\\/3GmeU0C30ApsiZ1J6H0jEwWe76Icq7TQQ9yDu\\\\\\\/DhFJqgi\\\\\\\/Z26XiNS6BkgEV4lQfmHQkbELc8ELgr3bO4PlQXCrsTNG3LtP92KHvgtBgMj9kcAEK1SfngVw4GW4HTr2hM9Bxd52i1FePZfL36MXyuFj\\\\\\\/YvGE\\\\\\\/0l7vKO4ehDiNtPI88QgKhqKA3mY0WTTmn1KZ7QXmebJ4G9zEC5EDUzJDNWmtTSUlk7liw3MkF027KNLxZhDdVOsHJS7FGs6j8A6oAfcy6DxSbmPkpZy+BGDe1Y0cJiClgI1dLsfuLu\\\\\\\/WZz8s\\\\\\\/\\\\\\\/euau2CUbDkEvrfuLs0DjovMUpBq00sVRdWrsRTXTaVeLyG51YFWekw86CWuqrFnphXGFdzrcTiT\\\\\\\/BTfp+SWazCOhMlibB51G1wBxBHJoSwO6aY6JIStl+xhAe5yfDV2mJ\\\\\\\/qnVj9oD\\\\\\\/MecLQWSqjOJ7G92W6kOW+\\\\\\\/7G\\\\\\\/LBC4w7POkd0mlm9NCWhn0DCrG19NbaW9jYdNbt6Urwife2gG4SYZzK+Yb2P5lwkTYYIcuW8Hb9ENfXUkKE5\\\\\\\/O+5qzf0EvhGrbgTQ6WcALUErNqYjLQkf9ZzPLx7\\\\\\\/oBVLq1FQKecf8EajMS0kUR5D6m1I7Ey86XspWtUHsu0SmmKQDIjGuqE3V94tN0X9B4Z9S1jYFU93gEedEnec9AsPnXCN3QWMXo7y7d43F3koAGVxtOR+lG1gX0UnWJqtyyfa\\\\\\\/ae4zZ2pSHUBPIgMcNB0WUVQiOZhXBytdXDqjBP0dqj6CZsVPNirKx83xeGG8sPDkE1wBs2yCfsozt1J\\\\\\\/SdBoMTsICxL4ip57jVEG67vsIMxWVIakPmn5XBx2dREwEaQI0Uo5hN7MA9KaL7BAaN96J\\\\\\\/\\\\\\\/ZC0awPa1Bc1imQOV0aLG\\\\\\\/wB7lWS9WkWm7iwGEoyeAvRXdCdehTLAtcCSkpDdNKYpXLSw0vANQBhDxHM22yEsAcEtZSdI8i0tEfn6lh2EFXUbHaIpnkTzR8ineA5PLT7WiL6cjBp7\\\\\\\/56bZw93iBm0DTH9tSmG3cR5lpfZlYh07Aa8k0KDTWJJ7HSdZ5G+Yd7TlMxRGBhSwmeiIuKdC7NthWCki1tJL3RXU26NGjlozVWmb3vsWPsYG1lMMq54BZouc4GBmUWADn5s5wiFDKnaEV3iE0St1AMMvxyTPqqs9D+b82w5KPJU7OjfwPZEy1OJXco\\\\\\\/sxsQKRfLnrNfHVYB7Itwriz4iqNgC2q\\\\\\\/FhWtNmcGZs1Ghp\\\\\\\/knAz58fE5laWHJ04onBIlaa6Vq8kR8HYDNB9X8RjacfzvM7Fvp3HGTaXZSLy4NtwF07k6bYE78RybX0jCZ36TN8TVkv3jRGNmURu9JkBNZ74vPqLBwDp\\\\\\\/eUHLDcGSM7\\\\\\\/XbEWa1hbFBjSJ0QRqlOoiqpKB3LLicQkQtARDz8vQuWUfqANe8gA5KAY1O8diocM8Ryec2Y2cC9fDovO4oWU+NpmFqf5iXxx+3UkUEYPBsg1GHsO6biKYRC60Oyqi6pY0XKbWOO2iIGMyVLqQh0Sn20knu+YjFQW5hWUPd6p8rHJk35h3vcfQ+YeWpRmyt8YJMwYjUhxDlsE+H+F2\\\\\\\/LbNtDyPi5AaKg46EycsO1wzBDI3sgBl282CrQg49SUHS0eF\\\\\\\/w4yLPrEUeUluxutck3K3+WPAdGgt94ZXL2DKkizIlzRipJAOTdk6fwEqvE0qaHqKytYUpahWJWC2mQ7GIL\\\\\\\/fM2SpyvZSCCYJU8ojHfUfiYKvnelB3SEkQUoChm\\\\\\\/qwaRFAONCSld3FHGwZZa+gpEf5lIaQm+v8Xsx+rtVP9VX5dtBwCwwnJ1A7ouR\\\\\\\/8Gsupap8GnmjaVlfFAGO6+6Ctx9HWfwDkDHHpu+rGgFpGmxKHp43kF0+hKmztpQQFXBrIdmsw+vfU\\\\\\\/o0tDVIJMWoaIaAxGUI80syr\\\\\\\/3ptCqG86+GH7Ps+V49+kEtcP0URDoKn4Jx7jl+SxDKXFXS0rHPeuH60af07knJ73MaqMk5Bl2Ze36a9c2Qr8E6RW1UNvu9Y1KMET6sz4eN1r1DN8ee9bnyEvAybAwqrSV2zuP5CGs7t3cmG8zMuHcymppKa8JmJ8fPdRh0L58WDFaXPSz\\\\\\\/9dK3z+nrI9ZE4X1scEwU8QYb82fL1yI2ruyEK+5uEXFt\\\\\\\/jqC4+SL3cZGbm1+hUD830Wfml3Jd9bju6oCf113Oqlaxb4lCI7FcKMB5rzW8OPIj2P0iit07gzFawEqypj3AEgoXqZr0d5Q+fYbm76XsEzv3ekcqEOZHAf4ORXMdfV51Q7MojFijzB51zdop7WNatlQ902CMxRGqU4UOeKXxSpExQ4rBXhKQB5fSTtbjpPZ0TcqSy\\\\\\\/KJOxoBniaKDRC5zG7Oh\\\\\\\/Vh7T9bF8Vi3nmVG9rswJoDkOO46OGPOLGXi7+9pGkKewH2XK5q1\\\\\\\/B6VEhJDLYWytQuXqFsfC+Pm1NCzJLN3WRgFA2f2Jl5EoJ9uE3n3+NURCOV8ETSkv+FtZ98ieg1QvWLIFYXRmgJWGhW294Cx04LxTRMlKMhBmmH39eS2EaRVr7R18dbuLkkUWcfbzVO\\\\\\\/mNKginJwZWVBsXj3szRV4Ed7T8eLbt6SAm0YA+xmb4zRcSIHscDKvHB24V\\\\\\\/rMWM5Y+7l+n\\\\\\\/DnL2sThjqvSIgvhZCAcnWeAXzeJxvYXK+pFMDDbC0W8IS5OmODCFVg\\\\\\\/mP6ZC0OHEHpS8\\\\\\\/aknI\\\\\\\/aPn+GzWW1oqitZVM0NwW7gjy6EIKDUGqeu0Fr9HHxRHtFpcEjSK24XrPFZpbYgErNV7M5gtuJetIjXrMFvebjiUernp9Q7UWl1rGbrH+nZQBqpYbuvutHrqyQVKgi04oJWR3D5OW2tGyWDlj9pf6td5\\\\\\\/754P28nsI66t66IbzeMPOQyu7egp9qvwU5UM\\\\\\\/yI2+ljn6Ghyx\\\\\\\/1\\\\\\\/IVQ51ETys604zkFvdxh\\\\\\\/zdi8vAioyCsx7cTiKYkUJE9c8TP5gttDv2SYadULo04alRJ5pMg6Rf3IQ1ckSmwv35KQ6i9bvYrsassT4zpKSek8F6aPIs70NHKdFd7a\\\\\\\/Se0hL69RxtwUCEcJmJhr8pfoodA4gDUx66rW2NquTOUbjqXZb5bnIKId1fViXXS8OsDveg9JjP6m1HY9TJsOZDXIYi94WXG24GdYZtW0tsfLziI4Bul5oreDicU\\\\\\\/bY+j5OcLOf4QOaYu8qAN\\\\\\\/ATWmNU+yzW3uFwbJg0WNNe50JFDJtDffZCz5fRmyxWjqFusDs+GBc9fTqAMGLdaShTDvH1CQhg68pRGJCO3dE40wuwja1lPXtTHMSJYz0GnhMpE5ub8yaPjSJLj+frk9mXV4d6+fPjdji2YgKPClHByQWA0a5yoHCtImnpXc1HdNSvDH4cgV80CprzLVTl7uatfjdR4nUFf1t62qbKhtsy7AEYzyY+sSGKnf9aJloDg3HKYyUPirbtecKbOtXqYbDgwOrADoBy8pBU+SLT+4bLvOfPZ69iAZ9YlJd8n0WAtYpKzItWLyUdkWw0Ln1q\\\\\\\/BaB4+BsRLa7SFuhV69R5HBNqL\\\\\\\/xjHegjIJDBB2qArDdONFySjrMmjZyNfCXNY4Y1CakXq2T7qajU+rrLjOmez+jsMmIcoaTHlrOZJn7lYjYynO3frASGeej9BcVfhBOm9zd4846BxLEF54+tjsT6r7ca942saLWWha1O\\\\\\\/eQj9MrscN3psjCfxzip01UQnnXI31IAwsSL4bH\\\\\\\/Zj5W65rcqBUpX8opBk0vGvGkCcDkRqVNQaA10vvwYuTzc1+6EYkrwp4jmSbaSfjcPR202wmgB3BG6drCCZKKpygUeIlpVCb2caBGug8u6n6P\\\\\\\/HQBXDy56KKFhXw57QLQ5TbFkN5XYUECOtg6xEUiRT4sHvlcY5llKZUg9VAc0kSkz1VdqVsRIhMXuiFG6QakxOTvz6cbZDZE1QIM\\\\\\\/NB5\\\\\\\/9tJCwPXtpyGBkZx\\\\\\\/eC6K8w+Lnxvv4AwKCwcGgrW1b1fCog36z1VvHqlqsyT+nZGV4ymboMslIBCr94pfCTWwnvzEWeGZGMqU828EvpcR0Qc4zsz\\\\\\\/fxBG39FcaKo3DX\\\\\\\/oTZGamwnfhmdB90H2L5BeHpYI9IKYb+oFRZxd8FsFMoG1CFjbLg2+YKDs0nwslGbVcWMwaq1ZZdMoor7PEhO6h3I2gt2dSYY\\\\\\\/rljMhGR\\\\\\\/250PYMwFiLywL0ruNdP4Zr\\\\\\\/RdCGAr9\\\\\\\/A+5OEFaRwWw4tbtW5zXkVjbUgNi9ZMgffd0k\\\\\\\/+xIfp5F8P+vQppIPULhIz1HEX7Q11ESosstXK5NFv9tkFrhqnlSoYDcWReN9WC1e1FMmIZQy71nBmeSH4npH5NQYZau+H\\\\\\\/F+7uoNdBjnCvmKl40W9WxFSjdWcK3F8BT5xHhJ\\\\\\\/c+YOMh\\\\\\\/koZULG2+oktCW0JINtkT55qhjjb+hcr0VGpDo854En1pdMDTg4XGgdoWjhV8egK0kAca6pWT1ohlmGlF0LZRtUJ5EHwK\\\\\\\/y04wjDexlh0yBtJfclqpwTWcKCdZ\\\\\\\/Xa9DxLW0kpJzMrCzS0PoIrs3CkxiXZtSKyzfhPyw6qRNeoQmsv6dUux8x+5JneTbKZLtNFjAaQFPYLnniJpBtlG9\\\\\\\/e8Fyqdygo9N\\\\\\\/y8znmbcLOn7tthZLIYV9C3CflqOhLYodFdSKvfkIA3csAoTpNQc9uQ30mW3qTT8plTee\\\\\\\/J6O7lw9KzSjKLuQJMfGwCDzMitE5rF39w5lsV8UfsiPqXeGeIbTKbJ0XYFcGhsCkXV\\\\\\\/mG6c9iNY+4JJWVdgEgHPIuSKc3M7zn74OCuU7CqHck2yIEvJCprQJ2oOpRelYgd+qcGLh0f0hO8WukX5UplgNm3p\\\\\\\/7lYyOchGjZETtJ3\\\\\\\/a4JIYIDYOSlv3NL\\\\\\\/Uw7xu7WjAPGcNhItBJ1Ss3En4SSvQRxQ+DT14gjCXmZip2mTycn\\\\\\\/jc00wsh\\\\\\\/H9j8idqXvK0rfwwdMbhzAl5u3IZtI2djjI2ZQZgHJbAxyNSa3stXWfTyZou04qUzhLc0e4mSqBLDx6H8v3BQnttpJAppy2Ga791MxROgEa\\\\\\\/CdhLY+42zg+zYxTdulehsJBuNAaVrlLU2\\\\\\\/Bd\\\\\\\/P2uJ6uycyVkkvJFu0\\\\\\\/nsz1voC2sm17dhL0Whe0zwaWbRL2GMTOv9SjEpEzvKdr9pkXNXU\\\\\\\/4MyBsQ1h\\\\\\\/31+fdV48CX1wm9PtkifY044OtcYgXGhCGdQS9hElMfgxZspFfyLgJChdYADDph9Z5yHFVbpJvQunv6LCQlyiKLsIfJ916dwlTbM+U0I3cwTErt3yR\\\\\\\/Asy45UjFbtZMYNa71ucq7OON5ZPlPRmPhbctSNkbq95zLVYkTVg3kHMsYC\\\\\\\/3k72u1IGaiKrQslDSOuwl9e8si3DU0ANa8QcT5CcdMWe1Q9+n3DBk9\\\\\\\/DVRWx+GF6R\\\\\\\/dR1N900NNS6BzBoUjJt7JBaUg743SjOWzJcWvNS849tBDIZ\\\\\\\/eO5XZ6fV\\\\\\\/KwbxCgbKTbU9cwEjG8prCKRB+luUYHb+IzlGIrjll6sNFXqT6UEtiZ+H7iGgI8ADLcYJ3d3v9ysuaALOMBWgrpiGsYLVllBZ47W8zhabkZdPBhD2f9iZS8kydwHruC5eMYPGan5NICSgwBw\\\\\\\/5LOMd9rcJ6D23UThIOO\\\\\\\/wf9PwhvhACL+6PAV7HjE6xpqGY4mdkk19KWdPwCo9n0SlG2c1Br+Kii9ZUpqnOmBGRkHTpUGcUzgpc8RhS9hxQwlK0Yx60SuHf4o5htATYE7DorFrlpSfQUr\\\\\\\/FXbshPzyKqnGwMCXrzSRnLGqOyiQslDbqMvGbrlHl1gNjC3Nl8I2WzWHeWdcUb049xjz+g9p2B6kwvwXUgt1z84pdoHPPW0L0qlzcEjCECNPv05oe0qABcP091UyjszVApV4Iot8mjeSe1gBNulg0x7nnryok8wT0ViwpYEqfydc3v7menOkZvDRzeUvbilAul8fZBZ\\\\\\\/MVlP9wlHQzk5FTd++J3lpHG5sfkoDxatQsipX84kzib3+9nnOQ7W49zpPQ5pKEjMfF8pWcWHPExOanW1QgF40nLHaTEWqLxM5sPXnWSNHrTUGnN+HCLzCLY1635kpDPXEmA1kzGmdV48eWDhYOeAP8bV5kCRtJafsVZdDjdbur0Rb910OyS8hpXg4\\\\\\\/K3zwkkL4QYHqQC4VHgHQM1lm4zb6tIw\\\\\\\/yz0GW1A75xqUrNvu0n6dUqyNuXOlu4Y8Pxw12qm0Q+czso8eZrMbPiTmomN+\\\\\\\/CTOnbMMJcQsdZ97RQugFSimJyzvY6yd5wnQBYm7pijZWw+nFie9IL7Eqfx4sLqmQ4vq0HsQ8iVfF1Tl4gPEtwwm5jk0J0hxOk2C\\\\\\\/m7MO6hcvHr2qBmAScLf8ac0Gw4RIeferNZXwJmlQtotTkGM2rp7qYDZ9X5PTfcwBGNM\\\\\\\/CN1K+vP5lwKfgZS8pkTxudtVzm9Drj3x6lhXbpdrHpFBBo86PHGcoukwYsfNrIJtWLuaJSkQbXKeIi6As\\\\\\\/qQBpfDKlS1SNFfYkG6zOxhwWmvyanbeMpM8uUVSH3N8hcVzPqnBph2e6n3V+CGfDIgeyHlsuOEWOrBccKz0QPEGAIPWAHUaO65gfgTRmmGdCfTI64bEVVpPZ3Al0hbpsmGqK5Q\\\\\\\/y6WjEYu0SHWj7DKAgPvRz3wnphtD7Ijwriy3NnkN3IzcadoQDZjkFwrMZmJghiPVXBN1mvvnnqm4PJTgPhsF4TRJXbZqRv0yIcR8X3czy8l3apjFQ05WeHxG7+w4\\\\\\\/BO9QM6Pwx2Lmte1vyuPBJj2CbuJ+sht+VwPaJAZ6I0ei9uUuZm1ogy980zjrzibq+WDNcNVo\\\\\\\/rXXVAyXjBHVN2gBJzPFlZzNNdtcV3I94+AZTjxDde1\\\\\\\/XzcdWGCZUrYz0jeNCr3ZJr907mnqDoUktoY2CN3Dom9Txoo8USTfyP5SjKYOY5kHCCd0234LarfcpNx1jBs3yZItXxscmwxgJNFfMcvdFwUo+vNXBbHJ5YFXY6G7uRTYVIwcEi9bqmrfZq6Df0D5\\\\\\\/Qix6Wwh6g0RDqdreXgCJEWDEM9q3Ogvqn9s42zc+B2I6HzgOrwyD1Qop+6YGRIyi+Fz61tYhrVAm7sYb\\\\\\\/KY2s+3Dn7pGJYt1zjZPfs59x9aexux5a8Ya0elkz8\\\\\\\/Jl5zBi\\\\\\\/UXJFbsEy1nNsuzamKVmyyZRAZAFuDYiM6PgByIWknonznBC1\\\\\\\/DlwfPgpu6In39eNvLAiNz2\\\\\\\/JRa1SI1lJAk13Ay78FrP\\\\\\\/CUyc62RFvf2o8RnYgOMIfvMy5MHgcghunyeQz95Ipi\\\\\\\/Vccru9bgMkH1RZAOa\\\\\\\/uvNodlUXFLZ5avVUWBhQTZGJ6r\\\\\\\/ZROa63tdegvRSwuTRw8hDmIGJvsWuO6vBdmRzUWjHPMhzaKVw7d2oFisj7JHC\\\\\\\/C3hQ0AnqM44jCBd5uSp+HkvBfBtNqEQ4QjR6p9uEtW+l\\\\\\\/APyMlvgTv1610WFBx2TKc6lWnmuK4M+kci2CYBDQzD6uRRuwx2vdqguTvabAEVSQnHJG8YS9pc3HMjleJ0UyoE\\\\\\\/3EXBFBaSpIeIFE6Hm4M\\\\\\\/lErO5B9Zf3jEpfJoVeoJ0B9Fb+C+X7ina+mn7wb1XYFkPn5nmlxOc+SdEf5BeWr4zzQP+jM3ENWP2JeLepq04DQE9eKiVrECz8BaaznHYILWykdMJWhHHDQoqTNnOfrIX\\\\\\\/4xpS0hGWv2CmA5TKPoXOwh3hpVV5H2Y0xt0RVRUrrxA+tAyrEHL\\\\\\\/uqqpNmy9ct04225ZgLIBUsFeTs9oAL6gLTUxI5vNsh\\\\\\\/QCMRUSNXN\\\\\\\/FhA7\\\\\\\/p7UZP\\\\\\\/QloGYrsaja2wIGONAMCeyZA2cJmuKSfB3xmKtVk+xsRmMmciLbl1UooZ1beDF6FoBUjCiBs7rjQ2fKxrOvYtXDervLq3eK1\\\\\\\/KSSxuSPwXjrJgLyfOJuCaegpW\\\\\\\/QCo2+M4leNUvG1VCXQE29HeWQ35LisU7i2fyKcmNr+m1p1+2vBdvfN9lnV6YtCIN8IwfD2A\\\\\\\/CCiz\\\\\\\/2+oCZnTwRa2iOyUaKpc3T4lhqVNfufK6BwPpeYCDhBXAH6Y3P8H5i0MKa6PpwJ9Ik6Czgyd\\\\\\\/MMeK7OedRb0naItjaiPa2p23GcwPH75a\\\\\\\/m7pIlkxmp3q\\\\\\\/Liam5klNJU0oMisORTRVUvnnNf\\\\\\\/r\\\\\\\/oCQXtAmXUSIDKLqZdBp+VNLukUGUDiAo0iJHXyHSv2Uun4X9kTJ0B7UKsmAzBj54xLZLecS+7yQllwqPyK1w84IicPZZrfPCjMHIEPUW6h1q\\\\\\\/wIbvGIovSO9jHbzIbWEV7ACW03tTgFmjs2fovA6jCmTEECNTmcTGbCHi509PZFLcAec3m94BpC8Ye1FVI8sgw3mt7QZLTvGu\\\\\\\/BKIAoIK9XaW256hNpXBioife01QheiF9W0tcx1ufqPOvN4AUrYQL9mpJc6FBsbKV4L3QNyFrnOB9oNhsAStdQeCTt27CDnWBkoqTKNFTe\\\\\\\/wCjL4A\\\\\\\/l8zYA41qT4Dg4AngCe6eb622Np+I4tjhTCYlY48+zUCAf8Wb+Cf7rGEyxX3wWJjI9Zzt1NffEAu5e7MQEz9HLE6Ne8hWXxKKS+AAzr9kZMwrXETOd6BG+oPNnk1b0z+aVy7AQEHiFHFmWfDyfsq9AF5D4jTc9EbQiJRo0i0WH6BSSebC+m\\\\\\\/7+ZlYXi7aSEhVv\\\\\\\/Dy\\\\\\\/KmUMAJVwT11OTiRm8DhWI+pUDYrO30LwB715OkuuNEUkdfutyai5mezVyCgeZOgIJFn1NXBVBtOPNo8fRT8oYqdBMDUxc5x2h9HRZj5r4Jibm7YHTfTkxo9rdCstVSMJR9GZT+0Od9Pw1KuEKOFalQC4hXNFy+zrpvbcl4cpRdfkJMkZuHTgsE+c+qki8h\\\\\\\/ZFeYWkPB59hnyzcdyGbnlk4sg8I6Rwis7m1XaXqZ0lZLu2unGSekH4pY4nLGmWyRACH7yylbd50v3p68KV21g0CkXGKjH2S+R6+S4IUCGy9pDm5voGv+3R22ZnRWYdV+Ef+ea5raQkHPp+z3NfPVU45DPp+kJUBpqvavh7u74wQClHADBQk\\\\\\\/Y\\\\\\\/io7BjF4G4HHTFERmp+u5ZbLkY2eOFtib58lWgeHXYkBNFVFRITBpBPfwrmKB20ejZ\\\\\\\/OxpT7sOSP5J\\\\\\\/xNV0FeBzJ4A9NW2K7K9p5hzK9m9BCd1fFa9hLFrgitCWVfKLgd7brXJ2KzvhO+t0hvnfDjIE3yFaJw71UCQ9orDTymh29AonwwUs5opNzSdcy7C1gxVb8TLkv8Y0oB3LtLdRIW50r9\\\\\\\/T+TUf+R5UMfwz8gdN6oZhXqnhAGolptBrZCR5XpZ3xOluLTAmfRqmnsjvXrYPryHhCCFTdOgiSddYVLiYOCbAnMA+OmU62nVjlBsMvFmyf\\\\\\\/W1jxRl+SuCkHDmFWACXAYU2qiG4CVVDixfiSXE+Xx5f5E+YoNGHv481YqeOq\\\\\\\/gIEN5N1\\\\\\\/42f5HAdivfgQG8K1WfZqlzpqRJNBW71jGfgruAObFWGv\\\\\\\/pb09DcEYEWZT39doi7YVLJUSD2F9AZ7iAANQPfuJYicT756jEXeeZ+yplylRrx23K6ekhbjpROpZhKOoDrr0GznQ5wFpPYJN7+CoCMlrQkYm9CLlYx3ozO5TMUDItPoFAT2E1KeuHne9X00wDsycNXnq6DLPnAJDftxcqRFf5GnilFMad5abhsp8ThBvWzv9SPd0fB6OQ2Y7iFCEo45gbHutlmF4kxV0exCbsOpR5S5bcmYygIzYuwKM8S8UIRDyBBen7o84hpL6nYxOzSPeq0hiuHDf2uSxM2npfviCFEfrxhMiOpvnrIJiCKsdMODdd3MvKHbNJgKrukiH7ogR+2oWL5hwzqntGpMzoL9Yp0sEVIB1zOs4NMfIBwK5d8JgQheyG8ZM4ATRVdFoM2s5W5GdkRnyGHv7yZljUs+Wu92ZXR12hfAKYW7+j51Smu0XynqqXn7V\\\\\\\/I0G373vhfSbRegl2tE1OsXXO\\\\\\\/CVXH52zqJlgfEx5K\\\\\\\/nqVbPTZzMgxGPqB93xkjwpH\\\\\\\/p35Tfx\\\\\\\/o+wLPDh2K+GR14IZ5Kq1FMVwa5wcO\\\\\\\/AsnLMGbnde+7D390ZL9kErNxBybbI9zRnLnQgth23VxE3UwJEGToiLOzEfXMh5jPRhc81ix25170BJEKSey6F8MIUPJE4cMbcpocdnS9wp4E1TBR\\\\\\\/DLIqzLfa1sGnwkci2ME5eLTG5DNx5MFF+ipMOWoos0Vtimd+0xRH9MSkd+tMGxIgHtxHS5ZmtHN9Mch4Oehxkqa8hDqkXDfWRI4Ou4OCQ1vgbE2QxFFM9tDLeDP\\\\\\\/0IDo8lrZ2f2Xz2+oO0K3x410oFiuOBHMwvKLqbYARR7RkRfw92Mm\\\\\\\/dK\\\\\\\/+EH03z\\\\\\\/xgiG\\\\\\\/pu3HtadAV06jIiMCQg5afDMIwNasfs3vTS5JTBhUpFpNinwQub2Qzw+ZnNeIKMjYD3vm9ape2WLa7\\\\\\\/oIhCVUeQ0YzW4bzGyt+\\\\\\\/mexvQzM289a5r46uVMPmSbHfGLJh7NM7v8aCCtnZIpSlTlhmOS4OIuhxAL9OP1onhXjcLHFhkqfHR7gVBXn+r7TsU7+2L6389P1kAUCKIwa\\\\\\\/QEAkfD0o4bvXqYCdrxcuAjYoCFf2LZ3jjNqHj6kEUMeMz1VaDSvY97MpKi9XRj9MUbluHhBl2193jTiD7fAkmg0L0NzUxqZkmSzBoL6rDxrBsgJ3pFZd3uyT9l3UkSkgbWrdEPWkZ9oOzcf1EVT6FrXeZA6G9v3y5BiA3zweLdS4FVf8IwXiztF6f1o8Mka7MNo1gIkEUl1Thn1cAjXhVmb7ccvnz0HnVb88PxjwPo8\\\\\\\/LocV2SIbXrv+g4Tr9qg2FxC+Q1LZ8YSKgqOLN8T3swWmduP7J4AKNbijj0JlpkvEqTBym3Eg8ufzhbGH7FVE\\\\\\\/UvuxZ\\\\\\\/GZDIiczZiKp5tidSOFao1k8P5Wj1kg\\\\\\\/RtFjL3GVwm67NfjEc4vlK7sQ+lf7QkR8H709ozv\\\\\\\/Ug4w4aTxtRmA4\\\\\\\/kHIlj+aT8f1pVPiVD\\\\\\\/zFYypjr6p4K7gEniTT9CcEUd1QxlpjyrklJUat9bS5Im1ws6jWI+sxz9HLRrBGtu6lpDunSjdaoTBm3usZwJbIlB7vOadTA7jDKSRk2kSwy8dje1EoebO+Q1AxmyOK4kN0MtJC8l85+B396EHvVpIX07l1esCbp4dp2ZzoFbsuKOKLszuS5OpDWyHm9oknogkr1o4Jp6L9papSDNboGpj\\\\\\\/hbRzA3j7xwgHk94\\\\\\\/oGzDfM3zkJFze3w84QHFIkTx\\\\\\\/VUbKFw\\\\\\\/kRhsWAPh\\\\\\\/sr9nw3zutITbSMnL+rCrqc8Nj6xxTYtbAVWvRA4UFMdf6tLqqUVsOslRoS0j2FGesUlgG6heGM5MhGTxAyP2OeSKSPZJ6GtZWnX+eohiQgFtxj6drNWtWTNdojqBg2OYz8Rrmnh0QUFqITjeMyL2kl7bv1\\\\\\\/MBi8Dy6WTABmUoxKfF+\\\\\\\/8SlbktLFLVe03R2RcLmVzPwjyN+8hDoJAd7gxsoDmNQBuDT6C8fRpWEZI8m7IBOhl5UsAryyaeKmV5UCccGWKWk1C73ZoSn5DV7Ig+0QQk99Met1pCpD\\\\\\\/oB6Gnulj3f2+yMKsgdfsNL97+Sn62wuo+2s\\\\\\\/VmWu3Bn9HAqbdf4xBgDJiZEtdE5laxm2hkMIk1izH9YUXxmgexqKH82iBPiFsBbEV09AU4MLZPi2sNG5s80GrsZ4Tzk0pcNX2LHHqCONInuGmjsNp8uBAYW3u7h1bvH4+bAqms3ELyFDrid2qGVrPE8wRCkVy78GibEOg8vkk+v6kOkmbY9Vy8RiJHw0j4EDUC\\\\\\\/VZf194JSS3QgkGDaf0AhgRyqRiYXlbJx60N81Ru3jInpPGkdLG+EsAYapHF5F4MDbFU2jPKf6dDuR+FyFZEtcJX2+aVpDpeQD1bBb7kWLkD\\\\\\\/Rk3BtneDOMqXrSIrZLoYsuBN4i\\\\\\\/vyBN1apqY6F6Kwgm5ftae6\\\\\\\/xP0uWE+2xg6HTaQ+apVYCLcz5MyBj8UAl1YVD2z9xKEMnsoKhMIMPC1HH8h6NSST6creIbWVQByzA+yPkS1d3ND87wYo8KVB7zcVWkKb0eV7+Nrov1TKew+VL8rpvc8OdFmCBstz4O97EE9SFhnUagrQ1PyL41Q4Lr93ilAiOwLCXWZR2cwSxeU+q0tZFVzxpllolO7bL\\\\\\\/trwX5o\\\\\\\/lDlY8lf3hN62QWh4PB5n4StyQha1LYIJLP23vbGYg2MaBXoGibva1m+Colmp9LNu0ydjsazTFDeYSmwa7WSSy\\\\\\\/4NbleIuXOO6dLsz7vRUa06EBAq+4sJAhHCH0UyuxMW\\\\\\\/lddcE6lEekubKyiBtMfg1zqo7lTVYix88dqEyDktpQrniYZCVuNWkq8uUUkRdq1mHgCZA2gc32CjzaelJurQXno9eSdYdX6p2sazmkwGndUdPxa2OpvAMtPVIMBzSzQs8kcnqRglCS3aay5toVFib2KvWjlXXDvjb8XGEpqUFoWDqbf7dlnbqU1JtxbK7skfygAW5MEZAC7GbmOzfGvv4Tez+u3sHmD0ZzJXMapWVobhUceGZhapW8+Ts32gjeYN83nZGAjI64wwD3A1xJ72TkD2FAvs6vf14tpgDtwmyvwxY\\\\\\\/3EPD6Zvg3NoG+f922vNBtvMTSBY9SuvGxo85j5Wy36ateUMX2iMqy2JX1Yvax\\\\\\\/HR2ESIr\\\\\\\/Yqoue88Hkqm8K3P8anwM6uHVkS5vjUinhTVYpPDqf1a9uVti8cyFbTCa4Ez2adLpoBkz+RlP+PiGe7kn8nyLS8dsUH2VdgU40tny+MXL\\\\\\\/ewp0ZCnRQrzTr4ouEWYFK3D6D4pBC9uYAneSZSlMWwqPfBah8YwR+w11q763cD567FQiAb2qjVLpBGboaK6GOmXG5ypy2c71IBkGBgrHyURj1iDS5R0eQk\\\\\\\/bKCGb125TjxzquGojQ4B4kB\\\\\\\/EtwLfbO8\\\\\\\/n0ld8klaPxC4Is7C24HA9udJGTmKgapvKexkd+vZs4y2sXOHjTNsgeItb6Hbxt+qCcj0seZN+3LyKjFjyDHoM42sIOtujDiPp0l+Ef70TnNreNQ0TODETd5UdGRx2HndcaEZ+2k8UqTMJKQLAJs26gCIcAWQ72Li1lgNrIAEwi4GstekCq8qWTJr6tlZUPsZ9v6eJZJxF9GEh5W\\\\\\\/Jn0ehHkRUJCU62t91DQs2dPWyp9xCPsqI6L1xq9hxrVLzSECtGL+RqQW74oTuImb+1hRePtf1ODtFp171H67zgquzEb7m5qEVTtfZgKGuZ9+UxhBDYQ0MzBXKdRliTOzlF7zho3FwIk87nfRkR4uqB0CIeCIIJBWCs7ZBSgmvkRcKu2u5mfKl7GdPYzASw5kvW0U\\\\\\\/taejKpyeMM9ya4NFUEWkOdI\\\\\\\/C5yn0bK9tc+sLJslM4BlKrPQBVtCPZWfFqfPkAgnzkdvKnrNE5fpW4uDTt3b\\\\\\\/X7yMl2hlJ9+cetpG\\\\\\\/TxbHX5YZn0rcpLJUihrFN7Bcs5\\\\\\\/BoJhhpc418kNubeLeJwlCq1eFVP\\\\\\\/HMu\\\\\\\/xULyVQ+JsJHaCQ6mqwcPwp\\\\\\\/lSTIOB8E5Ef3jdmlsssRSk6ok9KFwKi3mtrgF4hcQl6D4ZGxEDGJSyExN8UvGVpYTn78+HzWJOlL6E9YprXs7FCNv6fYanvnzrxzJpJpdtkURGPQuYkgzX3PiqcuK\\\\\\\/hH56v4LHuxrYERl1Ux3NDctbRUmcr7v8W7SwluSX4tG6IZ7ctjnOvGNrADRog2Y0JmAbOtKt9w93B4ixcMTx31Zvy6WCFVnHuGQY0EV8Hlm6rQptDGaTs8EXW3jKsettjlP4MUgsEZFqpOfs1amfcBf4cc6kctM7sBPFMG2wEt19Iolo3\\\\\\\/seVlZBacbTuoFtsLGx2a37a9cr9pKEYeu69rYnmadtxRAv4RNBguzV964rSiZgs8XGr3+opCf8jW6J4PL078ANtMlxY5nRNsizMG1flfJX+U\\\\\\\/KmN4k\\\\\\\/+SEDtELHtTQ7WQsoal+6bPSKKwD5tCg\\\\\\\/803zXumTDe9FUfipPKVQYQiswuJbsheWZNQHo3359a\\\\\\\/CS6Lu0cxJ0MHYDP9c245hLxC51z+sMFcUOqDHKBajQRinQGOoyD4K+rbbHojBrrQMcBCDQ8PquF7rQ3WDM78DyqaTdi9VZfzf5j+PGF4rrsdG\\\\\\\/fUVoppan5UhzExM\\\\\\\/ROfEJl+bYtQG\\\\\\\/C+\\\\\\\/EfhEpAJONG\\\\\\\/ifQbJIcZKpqZ+aRhCV8VtEJKPrbRnODimbNtxvyC32FsCahWRAn2VYL66vw3Cz4bYCgjUKP2+8ZsUUHrIhTsSFUxJs7THde4lP3r5QROs+u9tHigFazx6tft\\\\\\\/nmuUAgzSrgq0cZNRYM3S84NmxMifyQL8q0HKQxNa1mmi+styKlDG0YnwYNxHotAsw6AAbMl7Novelf0CoUD+xGICEhp1gigTx3YykanP5S6GYSoEWzB2UanhxriYDGL9xD1exE7U8emrlk2t4BuzDLlIS8wnVgGFpOrWO\\\\\\\/ksmSyAjMk\\\\\\\/gHcNqpwtcMyXCzEd\\\\\\\/2sFasIMCVgULXvdij9oJOzvISVJ\\\\\\\/WRKCvH3A4E2wjAy8ks6fhIl5LrIwIp95FXoI5iTJuT6gVXL88h5QbWnsPf7vyeNLplpuSqDhA\\\\\\\/8OLgz7FOBHcg6B1dkkYbw3Q9LSJZQ2lfjTL2sVkTRH8mL9ZYW6TTQ7uxrjihOb6yiUBMVLC8N0ECEq0owrfJhoGVqz9feXXcrLlegt\\\\\\\/ZQxG8g+IOyP6kZ6uETzdxDhVL2K492fkKJrEsGckpavmTJMzzTANa6hyJJ8qY3BQKEz\\\\\\\/2R8PgPY4\\\\\\\/tNWQr7tdpXRQGeMUhw2NG37YV9LqEK7Eo8d9YuyYJkSuZgedEPM+VRz5g1hVAyC0n09H2+96QZLQHOJHkFYeLDhRU6PwSL6LFemqHRWzLqqFkTSc4OSYVmQCgXKGw1+x5C3OTJzVxcb0FyJW+CpWvlZulh1pucluZPbOlV06aIh6ZNZWeo364L5NFnL6EOCw0KtoiTcg\\\\\\\/hwqoH+1\\\\\\\/uz1EtfMl6x0IzuZDUsSgGQVt8355tJPkV8f9ewVJvL1UUechnW8OsnbL8SR3YkUhD262+M7FdiGgmaxOsp721dzNsg7cUXYnu1DyWJ7cb1bn05EkHjhhDYef4jXAWrbwN2wAe7KmdgNh6wwDa7oAHrrvYYSMtnCzWbXddDp2gPEKxa2x9EDeduAA7n+1zMMKg4s\\\\\\\/Wt22yk9f4cHT3S6pSep8lBQLCZPKyAJESDff2KW2uqdFStSa4\\\\\\\/\\\\\\\/PmI1wppQGLtwPmrgTYRE5xKPoMAfPhdo9Wpqlo5pQTS+GVyPIuHY+XiAsOuIP9E2VDqpIy6MFT9hdD02x5Cbs6SVHPwy1IF9lUQRa3m4chLRHo20z2CLi6VJmdGBW0uJWXjPKVokrqG4DxFHjaguvw9kP5onvzu+iqLMlfhCZkPhAbn8x\\\\\\\/ACuJZCntoLiCAB3Qj13bghVhjxGJg9nH9N3s1XenXVaJ+Hf6zLwf\\\\\\\/s6oOGMMXz9D1MVimS3v1Y+\\\\\\\/wvIRKimxJH7Gq5M1u+RDxcVl2vV0Skr9NV7aXSSIq7VSN+m3IotfcnGeSXg1TogtPO\\\\\\\/PR3ZkdSkVVEdQVAMY3NnLKJbgRltK975DwcSjk6M612BKk60mWTJA4O01EQccSmQwXl93DGEWcgUdiwaDw7RS5rz68X2XUMV329Gi8OpJZw2InEx+5jT94HAgBD1qABRr+2bHgGXQXVGxsN\\\\\\\/7DdQkFHpgSCZzgtOXSz29v3SlA1lFuWKI8bojS9xy9T4QburnxbWhtNOSiiGG3ACYB3DKC8lFkPxDTzn+zX1tUKdugYo28R5M5\\\\\\\/MWj4ivi5VdOLvQUAFate2CgZR47zn0cTd8FLwFm0NRav+JeQ0N6FRasbFUAnXLxu6t9GaNqiU+NHqYJswVg9rNJeqA7ZsOzZWNx5LITEFMxJYpwnxlo0qSCUnUdIRFDjL9DCQuhyLmZ+XhYFmMjgY9NnZUc\\\\\\\/QBwpBm8x4JWC98PfwMi4jjQlRO3QpxvgKYgN44HGR1xmx9fpzTZU4S3WfqrvaU\\\\\\\/4r73zo8bFMCwia+2djb4ChgTpte4rDVmkvQzGK\\\\\\\/Lgqm9ylIjmThLgw0WvgaiKfNTvijEhHkMSuVOgy8ugUGQBuoT0CH7aImEwKR7jnBGDitvOAfDNyVzhmAcK9luM1ICarg8THNKTAXepVK8fV1pYNZ7DWx\\\\\\\/rBpITXLGMbq4Ai20aWdEj6Vb2XkXNYYVONgL2Xk+\\\\\\\/2nFLAZmVTj7bg0QXnKiEewSAma8IWk55wGg8OB46m7NrCdkKdRNNYrEYDdDLNhz1xPMqQBEOSaX9AMuAym0Lv8T3nKeOXBeXx2\\\\\\\/PHXghkcAVeE4tkaFR33ToXT0HC6rGf4eD8dJY4Wc4tPA3qL3dFHkRO8MSmxwswpQtEFwkC9RIpHLVVxzJE1fsrHY1YuHsoYRhOIfGuB5UsIsuyHm1nlL87Atd6RwyPVs1wJAvoX2yjcNtWrAfcFF9xx1UqCkKFOqetjOQvnIXmAD+oMRehF65A6T5a1P0wR2aVwY1k70mUDfUc6w6kf2iaaxGNHU9\\\\\\\/kJJf+wTIGdOsy7i2tA+SPL7cErSIKF\\\\\\\/NK1yRwM6gysedVBU6PMm48Rl+2VudRUKUmEx\\\\\\\/h6cmE+9N6aWAZPw0IEP\\\\\\\/UmwWD3bOpy26aFmYsra81YfoTJMrGV1DBX9spMkzWiSSyRUJIxNNxXYbIEnlBBLx5aaMzyGWfsdAjK4pfp6s7KTmRQNuC6t+MOm9PzZPMjQT0GVbSFuhZSvEVDWzJDxmSUz6iQyTSbnshVOFuGxGzvM8RG9Sa6CtXUAJqnWf0hvP4C5fcYCKTwoO1WgUIb3\\\\\\\/UoLmF9YDfI3IhvTo6YtHuBPzQ7MZHfuB7ig+ZTegnWK09xGDv6n4F7gik7ONDKEr36\\\\\\\/+brGcvBgbk6ict9lWvpxwZxmdr2eWamPtDwAitb0R1fQ0C4hqqNGrRlfe1QU+sSLw4NPoWiwtMrQBD+\\\\\\\/KbYhuT6vunOMWSNsoSJHnjk8CStOcBSC6qyn3RxVHmABFi4a7kzHFgHgoY+nObhWRm4eIXiw3tJm1lkH3XNEDh+bxaO0oKdgzmOqp\\\\\\\/bhkVhWuzhVW6nowk2mXSzQidPX7l5zl6wds1Gz8GjhBuWltdFguoKae83\\\\\\\/8oROzTz9UpGZA0QuGAIdNMjgJoqEGQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022IQi82PNA\\\\\\\/6MPQvMpWVKbIw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223344fbb1894409df97cf6c7e44c101671a9902e470cc29272f0f5394791d5833\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022PKpa4hbBtoJtw5kXwk2QTJd82li1iGvOMRIM+kKrUs7+vE+RU0BFQuO1ivAOcEQKe3\\\\\\\/m0jA9A4li5d7TH3EpeFdPFrLUUAf62+FT1GzVRBxaj01O7KpjumcBipQ9N+5oM\\\\\\\/jK0J+YRlZjBkcq5J2HtWLvTh3fkH5LFfufVnjEWSgE+4cyjdaB3p1gVekS\\\\\\\/i3rlNEf2ey6u1lbjXcckYtVoRM1SSHTHhq4OgL5e9oSNtFTd86WDCrM2720uaaVqHMwYQIDiJ8f++ICfnuV618O0CkVzyKK2amL0tYdinUaDrVkbHJLgufuTJZD9nejgd\\\\\\\/G1UxhGzRpClazd5hj+x8OP9lKoHiP8zSnkZzIN3LoWZVlv2sqqTIm03UmlKq0+CIXGOnvlmr0djE7s6hiDmx2roX3cH8NYNDc6+GOP7oXEVMRTcQE4req7EyUbhN2Nxn\\\\\\\/C1bsw2\\\\\\\/aufHAaBxnotvqXZgLS6mE77PdTIHoAAskm3CoKtAYWTMFcI\\\\\\\/hdcQ8\\\\\\\/KUYXRIQA+ZHtgCRL6Xo29C1e2xyTUY8p20oJxcORxW+RCzsKE1RR3Kvh3dHmHtWSI7Ukl1If8DNuMM0m1QkYbrf+S2qAqJKbYMy8l7wI3Y17s\\\\\\\/0el7P03dVscpMp4bSg+pJHnv9u7516Rg1Xcgp\\\\\\\/4WZKxq8CC\\\\\\\/Igysw2Z5nNyFYRfm3fiSGx38NN\\\\\\\/tmShCXxsYALkItaEWFqkPz\\\\\\\/uh9oQEireYj5QuEU3VvAh62Trc1RIDBePDe3+kongKEkRRunzdLaYN3vctgF3K57bibRuTF+lXOpqeRHRxr0jhG\\\\\\\/APnIteIpSlaP6aHkzeZP8ydVUCTtI6yotadELc5epSCrDa6YFSA236Nvg6gqYNYnC3us6D2w8dlM\\\\\\\/EriQVfR7MCT70SqN3+UFCfTnEkemPtHsAJrW9A3mOT4VpxY6xg869cqI3MIhUM6bNCx5s3YwEwDXglIS+o9BMgd9uS+uWgQg2F645n6vWjvBHvlMWbDSpXgskkX8DTRXxlEZ6RmHUaFwBp5Rs9xwzlKlmEMjGJjiK0Ghdp\\\\\\\/A503tV7kxruL92JNpZbSbCd45+jquMJTA0iFEXEoidtZhDBVmezwM03npXhByXSie5gxMjV+2fbnFpuCkk3y7u55P1zkvJme3draSbp6wLbS+L2eQXFxiHvf5SywwrWgkEblKKpEIvv3MOuWT\\\\\\\/eC4W7\\\\\\\/yZ711By5Ywy1s3tkTj8vhgItnrmXko21urc9\\\\\\\/LjZ0+98J9lo5hOSIUFlv1cNtVhqtf3g1ApphqO6OI0cv65wDL75pisZfiTp3pDSiS24SvlmaXWku7LB5gv45vUlPUCkE3tO5fRRs+MNp351TXszw8MLN9gc3\\\\\\\/sfJ8GzgKXKqaTZOuEUJOBMYSziphGKitKwIccZ01J9B4sCLaIYgpWFTV3\\\\\\\/A7E453kn20cEDRjOpqNCv0SDa7eIgmRPXLdZYK9cZRHJ\\\\\\\/ioTszixRaaBj+aoU7pQGGnp5ZeZxDrJurnFpgKL8xg09ThdvkSRWKswmjiNjtmOwugNkYIZjndxQXVy1KUyt8Bxh+CbHDBACIjLCeDjDW+wGtd+WcqoSnfcigYfbGHGyYVUJLrpWNfS7+RmOJPfi2NtQl7\\\\\\\/GHfNrprQPTlQaqblPOxMkohgnw0vdPu+BidCAqwLjMM32Vsi2xV1Fyx20SWm7t3pXyHp4XMaagpv4RUnuI1VD4TzXHkt49hvX2Kl3EgdAho3+2ibboxaNLmxNeAej6VleZD9t3TALasNGgml4Ki0l86tpeqaYlwCDEYaIDhZgbmyMUC6cvDmregaa+jC1HkJrkvex9pPrZdx3G37RY\\\\\\\/SB9tsLVulA8wacCtLg6+YmFxfah7gkDfirRRJcok+edmjm692G+aIIxJnf8CfHoYJ+Z0AV8AuXXsK8keZmw78V0bSydn5v0R0breIc\\\\\\\/MBUeRIMDp1DGaNQrrzcwhqHkakd\\\\\\\/BbtAM3AWJqn+qA3201RqW292I7+SgU+BtwEgekOpIbNf8XA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022LkQAC5dsc8cR3LsM50BOFA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00224fc0bd322607aa2c84ffe48370d3e1495d85be3da9e4f257f4c64ca698284bd6\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459478 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/6/929" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-cQT2yAvfE_CVSx4MfSdorIR9ItCeVBSwOrSV_2GtOl8" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/6/929" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-r5DxZj5FlhT5s3aC4YdGkRUMyzlVZfJ8Y9BgU4u-qIM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/6/927" title="Editorial: HeLa Cell Sequela" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/6/941" title="Resistance to Germline RNA Interference in a Caenorhabditis elegans Wild Isolate Exhibits Complexity and Nonadditivity" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459478" id="node459478" data-pisa="ggg;3/6/929" data-pisa-master="ggg;g3.113.005926" data-apath="/ggg/3/6/929.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Nucleosomal Region Important for Ensuring Proper Interactions Between the Transcription Elongation Factor Spt16 and Transcribed Genes in <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Hoai-Trang T. Nguyen</span>, <span class="highwire-citation-author" data-delta="1">William Wharton</span>, <span class="highwire-citation-author" data-delta="2">Jennifer A. Harper</span>, <span class="highwire-citation-author" data-delta="3">James R. Dornhoffer</span> and <span class="highwire-citation-author" data-delta="4">Andrea A. Duina</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">929-940;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.005926 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459478" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Hoai-Trang T. Nguyen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biology Department, Hendrix College, Conway, Arkansas 72032</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hoai-Trang%2BT.%2BNguyen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nguyen%20HT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHoai-Trang%2BT.%2BNguyen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">William Wharton <span class="nlm-suffix">II</span></div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biology Department, Hendrix College, Conway, Arkansas 72032</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BWharton%2B%3Cspan%2Bclass%3D%22nlm-suffix%22%3EII%3C/span%3E" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wharton%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWilliam%2BWharton%2BII" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jennifer A. Harper </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biology Department, Hendrix College, Conway, Arkansas 72032</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jennifer%2BA.%2BHarper%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Harper%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJennifer%2BA.%2BHarper%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">James R. Dornhoffer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biology Department, Hendrix College, Conway, Arkansas 72032</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BR.%2BDornhoffer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dornhoffer%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BR.%2BDornhoffer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Andrea A. Duina </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biology Department, Hendrix College, Conway, Arkansas 72032</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrea%2BA.%2BDuina%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Duina%20AA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrea%2BA.%2BDuina%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:duina@hendrix.edu" class="" data-icon-position="" data-hide-link-title="0">duina@hendrix.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/6/929" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459478" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459478/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/6/929.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459478" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459478/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/6/929.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459478" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459478/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/3/6/929.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459478" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459478/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">The highly conserved FACT (FAcilitates Chromatin Transactions) histone chaperone assists in the transcription elongation process first by facilitating the removal of histones in front of transcribing RNA polymerase II (Pol II) and then by contributing to nucleosome reassembly in the wake of Pol II passage. Whereas it is well established that FACT localizes across actively transcribed genes, the mechanisms that regulate FACT recruitment to and disengagement from chromatin during transcription still remain to be elucidated. Using the <em>Saccharomyces cerevisiae</em> model system, we previously showed that a histone H3 mutant—H3-L61W—greatly perturbs interactions between the yeast FACT (yFACT) complex and chromatin during transcription, resulting in a pronounced shift in yFACT occupancy toward the 3′ ends of transcribed genes. In the present study we report that two histone H4 mutants—H4-R36A and H4-K31E—alter the association pattern of the yFACT subunit <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> across transcribed genes in a fashion similar to that seen for H3-L61W. Interestingly, H4-R36, H4-K31, and H3-L61 are in close proximity to each other on the side of the nucleosome. We also provide evidence that the H4-R36A and H3-L61W mutants impair proper <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>−chromatin interactions by perturbing a common process. Collectively, our results suggest that a nucleosomal region encompassing the H4-R36, H4-K31, and H3-L61 residues plays an important role in ensuring proper association of yFACT across transcribed genes.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/histones" class="hw-term hw-article-keyword hw-article-keyword-histones" rel="nofollow">histones</a></li><li class="kwd"><a href="/keyword/nucleosomes" class="hw-term hw-article-keyword hw-article-keyword-nucleosomes" rel="nofollow">nucleosomes</a></li><li class="kwd"><a href="/keyword/spt16" class="hw-term hw-article-keyword hw-article-keyword-spt16" rel="nofollow">Spt16</a></li><li class="kwd"><a href="/keyword/yfact" class="hw-term hw-article-keyword hw-article-keyword-yfact" rel="nofollow">yFACT</a></li><li class="kwd"><a href="/keyword/transcription-elongation" class="hw-term hw-article-keyword hw-article-keyword-transcription-elongation" rel="nofollow">transcription elongation</a></li></ul><p id="p-3">Eukaryotic DNA is compacted within the small confines of a cell nucleus through its association with several proteins to form a structure referred to as chromatin. During the initial stages in the formation of chromatin, 147 base pairs of DNA are wrapped around a histone octamer—a protein complex composed of pairs of the four core histone proteins H2A, H2B, H3, and H4—to form a particle known as the nucleosome (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Luger <em>et al.</em> 1997</a>). Studies from many groups in the last few decades have demonstrated that nucleosomes are not static entities with an exclusive role in DNA compaction but rather have additional central roles in the execution and regulation of most of the processes that occur on DNA. One of the processes for which roles for nucleosomes have been extensively characterized is transcription. During the transcription process, nucleosomes can be manipulated in a variety of ways—for example, through covalent modifications of histone proteins, changes in positioning across DNA regions, or replacement of histone core proteins with histone variants—to ensure proper timing and levels of gene expression (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Campos and Reinberg 2009</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Rando and Winston 2012</a>).</p><p id="p-4">A prominent focus of recent research in the gene transcription field has been on the understanding of the dynamic functional and physical relationships that exist between nucleosomes and those proteins that assist RNA polymerase II (Pol II) during the elongation phase of transcription. A major task for a subset of these proteins is to first ensure that nucleosomes are partially or completely dismantled in front of transcribing Pol II to facilitate its progression across transcribed units and then to subsequently reassemble nucleosomes after Pol II passage. The proteins involved in these processes carry out their functions through an interplay of a variety of biochemical activities that include nucleosome remodeling, changes in histone posttranslational modifications, and histone chaperoning (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Selth <em>et al.</em> 2010</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Petesch and Lis 2012</a>). The histone chaperone activities that occur during transcription elongation have been attributed to several distinct factors, with the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003348">Spt6</a> and FACT (FAcilitates Chromatin Transactions) histone chaperones being among the better characterized ones. Although both of these factors promote nucleosome reassembly during transcription elongation, the FACT complex also has a well-established role in promoting the disassembly of nucleosomes during elongation to facilitate Pol II access to the underlying DNA (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Avvakumov <em>et al.</em> 2011</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Duina 2011</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Winkler and Luger 2011</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Formosa 2012</a>).</p><p id="p-5">In the budding yeast <em>Saccharomyces cerevisiae</em>, the FACT complex (yFACT) is composed of two protein subunits, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004534">Pob3</a>, which, with the assistance of the architectural protein Nhp6, can interact directly with nucleosomes and alter their overall structure (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Brewster <em>et al.</em> 1998</a>, <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">2001</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Formosa <em>et al.</em> 2001</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Ruone <em>et al.</em> 2003</a>). According to a current model, the physical interaction between yFACT and a nucleosome gives rise to a reorganized nucleosome containing all eight histone proteins, Nhp6, and the yFACT subunits that is more prone to losing H2A-H2B dimers and thus more easily disassembled by a transcribing Pol II as it travels across a gene (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Formosa 2008</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Xin <em>et al.</em> 2009</a>). The ability of yFACT to tether the histone subunits within the context of the reorganized nucleosome is likely to be relevant to the mechanism by which yFACT is also able to reassemble nucleosomes after Pol II passage, a property of FACT that has been supported by several genetic and biochemical studies (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Orphanides <em>et al.</em> 1999</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Formosa <em>et al.</em> 2002</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Belotserkovskaya <em>et al.</em> 2003</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Kaplan <em>et al.</em> 2003</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Schwabish and Struhl 2004</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Stuwe <em>et al.</em> 2008</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">VanDemark <em>et al.</em> 2008</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Jamai <em>et al.</em> 2009</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Winkler <em>et al.</em> 2011</a>).</p><p id="p-6">The mechanisms that regulate the physical interactions that occur between the FACT complex and transcribed genes remain to be fully elucidated. Studies from a variety of model systems have implicated several factors, including the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000483">Paf1</a> complex, the chromatin remodeler <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000966">Chd1</a>, the histone modifying complex NuA3, and the <em>Drosophila</em> HP1 protein as being at least in some cases responsible for the recruitment of FACT to sites of active transcription (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Duina 2011</a>). A more recent study has provided evidence for a role of SETD2-mediated trimethylation of histone H3 at lysine 36 (H3-K36me3) in FACT recruitment in HeLa cells (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Carvalho <em>et al.</em> 2013</a>). FACT is then thought to travel across genes in conjunction with Pol II (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Mason and Struhl 2003</a>) and eventually dissociate from chromatin at the end of the transcription process. The mechanisms that control the disengagement of FACT from transcribed genes are yet to be defined, but it appears that these are to a certain extent gene-specific because yFACT has been shown to disengage from certain genes at sites downstream from the polyadenylation (pA) sites and at other genes at sites upstream of the pA sites (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Kim <em>et al.</em> 2004</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Mayer <em>et al.</em> 2010</a>).</p><p id="p-7">A more complete understanding of the processes that govern yFACT−chromatin interactions during transcription could be gained by identifying and characterizing any features of the nucleosomes themselves that have roles in controlling the physical interactions between yFACT and transcribed genes. In previous studies, we showed that a specific amino acid substitution within the globular domain of histone H3—H3-L61W—causes a dramatic shift in distribution of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004534">Pob3</a> (yFACT) toward the 3′ ends of several highly transcribed genes as a consequence of lower levels of yFACT abundance at the 5′ ends of these genes and in abnormally elevated levels of yFACT occupancy at the 3′ ends of the same genes (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Duina <em>et al.</em> 2007</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Lloyd <em>et al.</em> 2009</a>). These results suggested that the H3-L61W mutant impairs efficient recruitment of yFACT to genes and also cripples the disengagement of yFACT from chromatin at the end of the transcription process. In the present work, we show that two additional histone mutants, in this case both in histone H4 (H4-R36A and H4-K31E), impart similar defects on <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>−chromatin interactions as those seen in the context of the H3-L61W mutant. H4-R36, H4-K31, and H3-L61 are located in close proximity to each other on the side of the nucleosome, suggesting that these three residues may define a nucleosomal region with a role in directing proper <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>−chromatin interactions across transcribed genes.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Yeast strains, genetic methods, and growth media</h3><p id="p-8">All <em>S. cerevisiae</em> strains listed in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> are <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004071"><em>GAL2</em></a><em><sup>+</sup></em> derivatives of the S288C background (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Winston <em>et al.</em> 1995</a>). Strains yADP67-yADP74 were generated by transforming strain IPY1008 (<em>MAT<strong>a</strong> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">his3</a>-205::<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">HIS3</a>-GFP-LacI <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000523">leu2</a>Δ1::lacO-<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000523">LEU2</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">ura3</a>-52 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002414">trp1</a>Δ63 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000319">lys2</a>Δ202 (<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000214">hht1</a>-<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000213">hhf1</a>)Δ::KanMX (<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004976">hht2</a>-<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004975">hhf2</a>)Δ::clonNAT</em> &lt;pRS317 (H3-WT/H4-WT)&gt;, a generous gift from Inés Pinto) with pRS414-derived plasmids containing cassettes harboring synthetic histones H3 and H4 genes (<em>HHTS</em> and <em>HHFS</em>) expressing either wild-type or mutant histone proteins generated by the laboratory of Jef Boeke (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Dai <em>et al.</em> 2008</a>) followed by loss of the pRS317 plasmid. Note that plasmids pAAD311 and pAAD312 (present in strains yADP71 and yADP67) harbor the H4 base construct and the H3 base construct described by <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Dai <em>et al.</em> (2008)</a>, respectively, and both encode wild-type histone H3 and H4 proteins (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Dai <em>et al.</em> 2008</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462854/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462854/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title"><em>Saccharomyces cerevisiae</em> strains</span><div class="sb-div caption-clear"></div></div></div><p id="p-10">The strategy for generating strains yADP75-yADP78 and yADP82 was as follows: the aforementioned pRS414-derived plasmids were digested with the <em>Bcl</em>VI restriction enzyme, resulting in the release of the cassettes (which, in addition to the <em>HHTS-HHFS</em> genes also include the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747"><em>URA3</em></a> gene) from the plasmid backbones. The digested material was then transformed into strain yAAD859, whose genotype is <em>MAT<strong>a</strong> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">his3</a>Δ200 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000523">leu2</a>Δ1 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">ura3</a>-52 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002414">trp1</a>Δ63 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000319">lys2</a>-128δ (<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004976">hht2</a>-<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004975">hhf2</a>)Δ::<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">HIS3</a></em> (for a description of the generation of the <em>(<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004976">hht2</a>-<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004975">hhf2</a>)Δ::<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">HIS3</a></em> locus, see <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Duina and Winston 2004</a>) and Ura<sup>+</sup> transformants were then selected by plating the transformation reaction on solid medium lacking uracil (SC-URA medium). To screen for successful homologous recombination events resulting in the replacement of the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728"><em>HIS3</em></a> gene with the <em>HHTS-HHFS-<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> cassettes (note that the ends of the cassettes are homologs to regions flanking the <em>(<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004976">hht2</a>-<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004975">hhf2</a>)Δ::<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">HIS3</a></em> locus), the Ura<sup>+</sup> transformants were then screened for an His<sup>−</sup> phenotype. Polymerase chain reactions using primers flanking the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004976">HHT2</a>-<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004975">HHF2</a></em> region provided further support that the desired recombination events in selected Ura<sup>+</sup> His<sup>−</sup> candidates had in fact taken place. The resulting strains—with the genotype <em>MAT<strong>a</strong> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">his3</a>Δ200 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000523">leu2</a>Δ1 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">ura3</a>-52 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002414">trp1</a>Δ63 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000319">lys2</a>-128δ (<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004976">hht2</a>-<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004975">hhf2</a>)Δ::HHTS-HHFS-<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em>, where the synthetic histone-encoding genes express either wild-type histone H4 or one of the two histone H4 mutants, H4-R36A or H4-R36K—were then crossed with preexisting strains and the resulting diploids were sporulated to ultimately give rise to strains yADP75-yADP78 and yADP82. (For a description of the <em>(<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000214">hht1</a>-<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000213">hhf1</a>)Δ::<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000523">LEU2</a>, KanMX4-GAL1pr-<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000911">FLO8</a>-<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">HIS3</a>, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">SPT16</a>-857</em>, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">SPT16</a>-790</em> loci present in the strains, refer to <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Duina and Winston 2004</a>, <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Duina <em>et al.</em> 2007</a>, and <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Cheung <em>et al.</em> 2008</a>). Strains yADP79-81 harbor the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004976">hht2</a>-11</em> allele, which encodes the H3-L61W mutant, described elsewhere (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Duina and Winston 2004</a>).</p><p id="p-11">Techniques for standard genetic experiments have been described by others (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Rose <em>et al.</em> 1990</a>). Details for the preparation of the growth media used in the experiments presented in this study have been provided previously (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Rose <em>et al.</em> 1990</a>). The concentrations of the drugs used in the experiments presented in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> are indicated in the legend of the same figure.</p></div><div id="sec-3" class="subsection"><h3>Genetic screen to identify additional histone mutants that confer H3-L61W−like phenotypes</h3><p id="p-12">The yeast histone H3 and H4 mutant library described in the <em>Results</em> section was provided by Jef Boeke’s laboratory as frozen aliquots in a series of 96-well microtiter plates. For these experiments, the yeast strains were allowed to partially thaw at room temperature and a small sample of each strain was transferred to 150-mm × 15-mm Petri dishes containing solid YPD medium using a sterilized 96-pronged metal replica plater. Plates were incubated at 30° for 2−3 d to allow for cell growth and were then replica-plated to plates containing YPD, YPD + 15mM caffeine, YPD + 3% formamide, and YPD + 150 mM hydroxyurea media, and incubated at 30° to score for sensitivity to caffeine, formamide, and hydroxyurea. In addition, a set of YPD replica plates was incubated at 14° to score for cold-sensitivity. Strains that showed significant growth defects in the presence of one or more of the drugs used or at 14° were then analyzed further.</p></div><div id="sec-4" class="subsection"><h3>Visualization of the location of specific histone residues on the structure of the nucleosome</h3><p id="p-13">The images presented in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> showing the location of the H4-R36, H4-K31, and H3-L61 residues on the structure of the yeast nucleosome core particle were generated using the PyMOL Molecular Graphics System, Version 1.5.0.3 Schrödinger, LLC using the structure information reported by the laboratory of Karolin Luger (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">White <em>et al.</em> 2001</a>) and available at the Research Collaboratory for Structural Bioinformatics protein data bank (PDB ID:1id3).</p></div><div id="sec-5" class="subsection"><h3>Chromatin immunoprecipitation (ChIP) experiments</h3><p id="p-14">ChIP assays were carried out as described previously (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Myers <em>et al.</em> 2011</a>). These experiments were conducted using 1 μL of the following antibodies: rabbit polyclonal antibody specific for the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> protein (a generous gift of Tim Formosa), a mouse monoclonal antibody specific to the yeast Pol II subunit <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001283">Rpb3</a> (Neoclone, no. W0012), and a rabbit polyclonal antibody specific for bulk histone H3 (ab1791; Abcam). The following primers used for the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002976">PMA1</a></em> and NO ORF regions have been previously described (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Myers <em>et al.</em> 2011</a>): 5′<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002976">PMA1</a></em>: OAD394 and OAD395; internal <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002976">PMA1</a></em>: OAD416 and OAD417; 3′<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002976">PMA1</a></em>: OAD383 and OAD384; NO ORF: OAD377 and OAD378. The following primers have not been previously described: 5′<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001543">FBA1</a></em>: OAD419 (5′ CGCTGCTTTAGAAGCTGCTAGA) and OAD420 (5′ CCACCGTTAGAGGTTTGCAAA); internal <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001543">FBA1</a></em>: OAD421 (5′ CAAGGCTTTGCACCCAATCT) and OAD422 (5′ CCGTGACAGTTACCGAAAGCA); 3′<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001543">FBA1</a></em>: OAD423 (5′ GGTCGGCTCTTTTCTTCTGAAG) and OAD424 (5′ AAATAGTGCATGACAAAAGATGAGCTA); 5′<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000224"><em>GAL1</em></a>: OAD442 (5′ CCTGAGTTCAATTCTAGCGCAAA) and OAD443 (5′ TCTTAATTATGCTCGGGCACTTT); 3′<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000224"><em>GAL1</em></a>: OAD444 (5′ CCCTTTGTCCTACTGATTAATTTTGTACT) and OAD445 (5′ TCCTCCTCGCGCTTGTCT); 5′<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004071"><em>GAL2</em></a>: OAD446 (5′ CCAAGCTGGTGAAGACGTGAT) and OAD447 (5′ TTTTGAGATTGTGCGCTTAAATG); and 3′<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004071"><em>GAL2</em></a>: OAD448 (5′ CGAGCAAAATTAAAAACGCAAA) and OAD449 (5′ GATAAGTCTGGTGATGTGGTCCTTT). For each quantitative polymerase chain reaction experiment, control reactions were conducted to ensure linearity of the results and to determine the efficiency of the primer set used.</p></div></div><div class="section results" id="sec-6"><h2 class="">Results</h2><div id="sec-7" class="subsection"><h3>A genetic screen identifies two histone H4 mutants that perturb association of Spt16 with transcribed genes</h3><p id="p-15">In addition to causing a shift in distribution of yFACT toward the 3′ ends of transcribed genes, the H3-L61W mutant confers additional defects, including cold-sensitivity (Cs<sup>−</sup>) and sensitivity to the drugs caffeine, hydroxyurea, and formamide (Caff<sup>s</sup>, HU<sup>s</sup>, and Form<sup>s</sup>, respectively) (<a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Duina and Winston 2004</a>; <a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Myers <em>et al.</em> 2011</a>). In an attempt to identify additional histone mutants that perturb yFACT association with chromatin in a manner similar to that seen in H3-L61W cells, we first screened a yeast library of histone H3 and histone H4 mutants generated and kindly provided to us by the laboratory of Jef Boeke (<a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">Dai <em>et al.</em> 2008</a>) for H3-L61W−like phenotypes (Cs<sup>−</sup>, Caff<sup>s</sup>, HU<sup>s</sup>, and Form<sup>s</sup>—note that many of the strains screened in this study have also been screened previously for some of these phenotypes by others—<a id="xref-ref-8-5" class="xref-bibr" href="#ref-8">Dai <em>et al.</em> 2008</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Huang <em>et al.</em> 2009</a>). This library includes the 486 yeast strains in the S288C background described in the original study (<a id="xref-ref-8-6" class="xref-bibr" href="#ref-8">Dai <em>et al.</em> 2008</a>) that contain integrated synthetic genes expressing histone proteins with single amino acid substitutions at each residue (in most cases, nonalanine residues were changed to alanine and alanine residues were changed to serine, but some other substitutions were also included in the library), plus nearly 100 more mutants that express histone H3 and H4 proteins with additional types of amino acid substitutions that were generated later by the Boeke laboratory. (Note that these and all the other strains used in this study have been engineered to express histones H3 and H4 each from a single gene; see <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>.) From this screen, seven histone mutants were identified as conferring at least a subset of H3-L61W phenotypes and were considered further.</p><p id="p-16">Subsequent genetic manipulations suggested that some of the seven histone mutant strains had likely diploidized at some point during their propagation since crosses between them and haploid partners generated cells that produced a large proportion of inviable spores after induction of meiosis, a phenomenon usually seen in meiotic products derived from triploid cells. To circumvent this problem, we transformed plasmids expressing the same histone mutants (also kindly provided by the Boeke laboratory) into a new host strain, eventually resulting in freshly-generated yeast strains each expressing one of the seven histone mutants from a plasmid. One of the histone mutants appeared unable to support detectable growth when expressed from a plasmid in our host strain background and was not further examined. The phenotypes of the remaining six strains were tested and compared to those seen in the original strains from the histone mutant library. As can be seen in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>, whereas some of the phenotypes were similar between the library strains and the strains expressing the same mutants from plasmids, some were different. For example, the library strain expressing the H4-K31E mutant showed a strong Caff<sup>s</sup> phenotype, whereas the same mutant expressed from a plasmid in our host strain showed essentially no Caff<sup>s</sup> phenotype. These differences may be attributable to (1) elevated expression of the mutant proteins in the plasmid-containing strains caused by the presence of more than one plasmid in these cells, (2) the acquisition of additional spontaneous mutations in the library strains during their propagation, and/or (3) differences in the genetic background between the strain used for the construction of the mutant histone library strain and the host strains we used for the plasmid experiments.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462858/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462858/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Phenotypic analysis of specific histone H3 and histone H4 mutants</span><div class="sb-div caption-clear"></div></div></div><p id="p-20">Strains expressing the six histone mutants from plasmids were then subjected to ChIP assays to determine whether any of them cause abnormal distribution of the yFACT component <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> across the constitutively transcribed gene <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002976">PMA1</a></em>. In these experiments we found that of the six mutants, H4-R36A and H4-K31E cause a pronounced perturbation in <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> distribution across this gene compared with wild-type cells (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1, A−C</a>). Specifically, the H4-R36A and H4-K31E mutants cause a 3′ shift in <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> occupancy at this gene as a result of higher levels of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> over the 3′ end of the gene compared to the 5′ end—a pattern that is the reverse of what normally occurs in histone wild-type cells. Similar perturbations on <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>-chromatin association by the H4-R36A and H4-K31E mutants were also seen at <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001543">FBA1</a></em>, another constitutively expressed gene (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1, D−F</a>), showing that the effects of these histone H4 mutants on <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>−chromatin interactions are not specific for the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002976">PMA1</a></em> gene but are likely to be more widespread.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/6/929/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The H4-R36A and H4-K31E mutants cause pronounced perturbations in the distribution of Spt16 across the PMA1 and FBA1 genes. (A, D) Schematic representations, roughly drawn to scale, of the PMA1 and FBA1 genes and the respective downstream genes. Arrows indicate the direction of transcription for each gene and the numbers correspond to nucleotide positions, with nucleotide 1 corresponding to the first nucleotide of the open reading frame (ORF) of either the PMA1 gene or the FBA1 gene, both of which are represented as shaded rectangles in the diagrams. The colored lines below each diagram represent the regions&#x2014;5&#x2032;, internal, and 3&#x2032;&#x2014;assayed in the ChIP experiments. (B) Results from ChIP experiments in which the occupancy of the Spt16 protein across the PMA1 gene was assayed in strains expressing the indicated histone genes (the strains used in these experiments are yADP67-yADP74). Relative Spt16 occupancy in each case was determined by dividing the %-immunoprecipitation (%IP) value for the specific region of interest to the %IP value for a nontranscribed region on chromosome V (NO ORF region; see Material and Methods for the primers used for this region). For each strain, the data are presented as the mean &#xB1; SEM from at least three independent experiments. As a reference, the NO ORF %IP values for each strain were as follows: yADP67: 0.14 &#xB1; 0.005, yADP68: 0.16 &#xB1; 0.013, yAADP69: 0.27 &#xB1; 0.021, yAADP70: 0.24 &#xB1; 0.025, yADP71: 0.16 &#xB1; 0.029, yADP72: 0.19 &#xB1; 0.046, yADP73: 0.24 &#xB1; 0.053, yADP74: 0.12 &#xB1; 0.018. (C) Data from the experiments described in (B) but represented as the ratio between the %IP value for the 3&#x2032; region of PMA1 and the %IP for the 5&#x2032; region of PMA1. Representation of the data in this fashion facilitates the assessment of the overall effects imparted by each histone mutant on the distribution of Spt16 across the PMA1 gene. (E, F) Effects of the H4-R36A and H4-K31E mutants on the levels of Spt16 occupancy across the FBA1 gene. The data in these panels are presented as described in (B) and (C)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-586614734" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The H4-R36A and H4-K31E mutants cause pronounced perturbations in the distribution of Spt16 across the &lt;em&gt;PMA1&lt;/em&gt; and &lt;em&gt;FBA1&lt;/em&gt; genes. (A, D) Schematic representations, roughly drawn to scale, of the &lt;em&gt;PMA1&lt;/em&gt; and &lt;em&gt;FBA1&lt;/em&gt; genes and the respective downstream genes. Arrows indicate the direction of transcription for each gene and the numbers correspond to nucleotide positions, with nucleotide 1 corresponding to the first nucleotide of the open reading frame (ORF) of either the &lt;em&gt;PMA1&lt;/em&gt; gene or the &lt;em&gt;FBA1&lt;/em&gt; gene, both of which are represented as shaded rectangles in the diagrams. The colored lines below each diagram represent the regions&#x2014;5&#x2032;, internal, and 3&#x2032;&#x2014;assayed in the ChIP experiments. (B) Results from ChIP experiments in which the occupancy of the Spt16 protein across the &lt;em&gt;PMA1&lt;/em&gt; gene was assayed in strains expressing the indicated histone genes (the strains used in these experiments are yADP67-yADP74). Relative Spt16 occupancy in each case was determined by dividing the %-immunoprecipitation (%IP) value for the specific region of interest to the %IP value for a nontranscribed region on chromosome V (NO ORF region; see &lt;em&gt;Material and Methods&lt;/em&gt; for the primers used for this region). For each strain, the data are presented as the mean &#xB1; SEM from at least three independent experiments. As a reference, the NO ORF %IP values for each strain were as follows: yADP67: 0.14 &#xB1; 0.005, yADP68: 0.16 &#xB1; 0.013, yAADP69: 0.27 &#xB1; 0.021, yAADP70: 0.24 &#xB1; 0.025, yADP71: 0.16 &#xB1; 0.029, yADP72: 0.19 &#xB1; 0.046, yADP73: 0.24 &#xB1; 0.053, yADP74: 0.12 &#xB1; 0.018. (C) Data from the experiments described in (B) but represented as the ratio between the %IP value for the 3&#x2032; region of &lt;em&gt;PMA1&lt;/em&gt; and the %IP for the 5&#x2032; region of &lt;em&gt;PMA1&lt;/em&gt;. Representation of the data in this fashion facilitates the assessment of the overall effects imparted by each histone mutant on the distribution of Spt16 across the &lt;em&gt;PMA1&lt;/em&gt; gene. (E, F) Effects of the H4-R36A and H4-K31E mutants on the levels of Spt16 occupancy across the &lt;em&gt;FBA1&lt;/em&gt; gene. The data in these panels are presented as described in (B) and (C).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/6/929/F1.medium.gif" width="440" height="426"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/3/6/929/F1.medium.gif" width="440" height="426"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/6/929/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/6/929/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462927" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-21" class="first-child">The H4-R36A and H4-K31E mutants cause pronounced perturbations in the distribution of Spt16 across the <em>PMA1</em> and <em>FBA1</em> genes. (A, D) Schematic representations, roughly drawn to scale, of the <em>PMA1</em> and <em>FBA1</em> genes and the respective downstream genes. Arrows indicate the direction of transcription for each gene and the numbers correspond to nucleotide positions, with nucleotide 1 corresponding to the first nucleotide of the open reading frame (ORF) of either the <em>PMA1</em> gene or the <em>FBA1</em> gene, both of which are represented as shaded rectangles in the diagrams. The colored lines below each diagram represent the regions—5′, internal, and 3′—assayed in the ChIP experiments. (B) Results from ChIP experiments in which the occupancy of the Spt16 protein across the <em>PMA1</em> gene was assayed in strains expressing the indicated histone genes (the strains used in these experiments are yADP67-yADP74). Relative Spt16 occupancy in each case was determined by dividing the %-immunoprecipitation (%IP) value for the specific region of interest to the %IP value for a nontranscribed region on chromosome V (NO ORF region; see <em>Material and Methods</em> for the primers used for this region). For each strain, the data are presented as the mean ± SEM from at least three independent experiments. As a reference, the NO ORF %IP values for each strain were as follows: yADP67: 0.14 ± 0.005, yADP68: 0.16 ± 0.013, yAADP69: 0.27 ± 0.021, yAADP70: 0.24 ± 0.025, yADP71: 0.16 ± 0.029, yADP72: 0.19 ± 0.046, yADP73: 0.24 ± 0.053, yADP74: 0.12 ± 0.018. (C) Data from the experiments described in (B) but represented as the ratio between the %IP value for the 3′ region of <em>PMA1</em> and the %IP for the 5′ region of <em>PMA1</em>. Representation of the data in this fashion facilitates the assessment of the overall effects imparted by each histone mutant on the distribution of Spt16 across the <em>PMA1</em> gene. (E, F) Effects of the H4-R36A and H4-K31E mutants on the levels of Spt16 occupancy across the <em>FBA1</em> gene. The data in these panels are presented as described in (B) and (C).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-8" class="subsection"><h3>The H4-R36 and H4-K31 residues are located in close proximity to each other and to the H3-L61 residue on the structure of the nucleosome</h3><p id="p-22">The H4-R36A and H4-K31E mutants affect <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> distribution across the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002976">PMA1</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001543">FBA1</a></em> genes in a manner similar to that we have previously reported for the H3-L61W mutant (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Duina <em>et al.</em> 2007</a>). In an attempt to gain insights into how these different mutants cause similar impairments on <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>−chromatin interactions, we mapped the locations of the three wild-type residues onto the yeast core nucleosome structure (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">White <em>et al.</em> 2001</a>) and compared their positions in relation to each other. As shown in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>, the H4-R36, H4-K31, and H3-L61 residues are located in striking vicinity to each other on the side of the nucleosome, with the H3-L61 residue buried toward the interior of the nucleosome and the H4-R36 and H4-K31 residues exposed on the outer surface. The close proximity of these three residues to each other on the structure of the nucleosome and the similar effects of mutations at these residues on the localization of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> across the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002976">PMA1</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001543">FBA1</a></em> genes suggest that the H4-R36, H4-K31, and H3-L61 residues may define a region of the nucleosome with a role in controlling <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>-chromatin interactions during transcription.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/6/929/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The H4-R36, H4-K31, and H3-L61 residues are located in close proximity to each other on the side of the nucleosome. The left panel shows a side-view of the structure of the yeast nucleosome core particle with the histone proteins and the three histone residues of interest color-coded as indicated and the DNA colored in green. The right panel shows a close-up view of the region boxed by the white rectangle in the left panel that includes the H4-R36, H4-K31, and H3-L61 residues." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-586614734" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The H4-R36, H4-K31, and H3-L61 residues are located in close proximity to each other on the side of the nucleosome. The left panel shows a side-view of the structure of the yeast nucleosome core particle with the histone proteins and the three histone residues of interest color-coded as indicated and the DNA colored in green. The right panel shows a close-up view of the region boxed by the white rectangle in the left panel that includes the H4-R36, H4-K31, and H3-L61 residues.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/6/929/F2.medium.gif" width="440" height="253"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/3/6/929/F2.medium.gif" width="440" height="253"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/6/929/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/6/929/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462836" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-23" class="first-child">The H4-R36, H4-K31, and H3-L61 residues are located in close proximity to each other on the side of the nucleosome. The left panel shows a side-view of the structure of the yeast nucleosome core particle with the histone proteins and the three histone residues of interest color-coded as indicated and the DNA colored in green. The right panel shows a close-up view of the region boxed by the white rectangle in the left panel that includes the H4-R36, H4-K31, and H3-L61 residues.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h3>Genetic tests indicate that the H4-R36A and H3-L61W mutants affect cell functions through perturbations of common processes</h3><p id="p-24">The H4-R36 and H3-L61 residues appear to be particularly important in controlling <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> interactions with transcribed genes since specific amino acid substitutions at these location—H4-R36A and H3-L61W—cause the most significant defects in the distribution of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> across transcribed genes (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figures 1</a> and <a id="xref-fig-4-1" class="xref-fig" href="#F4">4</a> and <a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Duina <em>et al.</em> 2007</a>). Based on these observations, we decided to continue our characterization of the nucleosomal region under investigation by directing our studies on the H4-R36A mutant, with a particular focus on its relationship with the better-characterized H3-L61W mutant.</p><p id="p-25">Whereas cells expressing H4-R36A display several growth phenotypes in common with those seen in H3-L61W cells (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>, <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>, and <a id="xref-ref-11-5" class="xref-bibr" href="#ref-11">Duina <em>et al.</em> 2007</a> and <a id="xref-ref-26-4" class="xref-bibr" href="#ref-26">Myers <em>et al.</em> 2011</a>), the mechanistic defects underlying these phenotypes may not necessarily be the same for the two histone mutants. To determine whether the phenotypes conferred by the H4-R36A and H3-L61W mutants are due to perturbations in common or different processes, we performed a series of genetic experiments comparing the phenotypes of H4-R36A and H3-L61W cells in different genetic backgrounds. To facilitate genetic analyses and to eliminate the possibility of artifacts due to changes in gene copy number of histone genes expressed from plasmids, for these and subsequent experiments we generated strains in which the genes expressing the histone mutants are integrated into the genome of a common host strain at their endogenous locations (see <em>Materials and Methods</em> and <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/6/929/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phenotypic assays indicate that the H4-R36A and H3-L61W mutants affect common cellular processes. (A) Cells expressing wild-type or mutant histones and either wild-type or mutant Spt16 proteins, as indicated, were grown to saturation on YPD medium, harvested, and then spotted on the indicated media in 10-fold dilution series, in which the most concentrated spots (left-most spot in each panel) contained ~2 &#xD7; 106 cells. The plates were incubated at 30&#xB0; (except for the plates shown in the second column, which were incubated at 14&#xB0; as indicated) and photographs were taken after the following approximate times of incubation: YPD, 2 d; YPD 14&#xB0;, 13 d; YPD + 15mM caffeine (Caff.), 4 d; YPD + 3% formamide (Form.), 4 d; YPD + 150mM hydroxyurea (HU), 4 d. The strains used in these experiments were yAAD75-yAAD81. (B) The same strains used in the experiments described in (A) but spotted on different media, as indicated, such that the most concentrated spots contained ~4 &#xD7; 106 cells. The plates were incubated at 30&#xB0; and were photographed after the following approximate times of incubation: SC, 2 d; SC&#x2013;lysine (lys), 3 d; SC&#x2013;histidine (his), and with 0.1mM aminotriazole (AT, a competitive inhibitor of the product of the HIS3 gene), 5 d; SC with galactose (GAL), 3 d; SC&#x2013;histidine, with galactose and 0.1mM aminotriazole, 5 d. All the media used in these experiments contained glucose as the carbon source, except where otherwise indicated." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-586614734" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Phenotypic assays indicate that the H4-R36A and H3-L61W mutants affect common cellular processes. (A) Cells expressing wild-type or mutant histones and either wild-type or mutant Spt16 proteins, as indicated, were grown to saturation on YPD medium, harvested, and then spotted on the indicated media in 10-fold dilution series, in which the most concentrated spots (left-most spot in each panel) contained ~2 &#xD7; 10&lt;sup&gt;6&lt;/sup&gt; cells. The plates were incubated at 30&#xB0; (except for the plates shown in the second column, which were incubated at 14&#xB0; as indicated) and photographs were taken after the following approximate times of incubation: YPD, 2 d; YPD 14&#xB0;, 13 d; YPD + 15mM caffeine (Caff.), 4 d; YPD + 3% formamide (Form.), 4 d; YPD + 150mM hydroxyurea (HU), 4 d. The strains used in these experiments were yAAD75-yAAD81. (B) The same strains used in the experiments described in (A) but spotted on different media, as indicated, such that the most concentrated spots contained ~4 &#xD7; 10&lt;sup&gt;6&lt;/sup&gt; cells. The plates were incubated at 30&#xB0; and were photographed after the following approximate times of incubation: SC, 2 d; SC&#x2013;lysine (lys), 3 d; SC&#x2013;histidine (his), and with 0.1mM aminotriazole (AT, a competitive inhibitor of the product of the &lt;em&gt;HIS3&lt;/em&gt; gene), 5 d; SC with galactose (GAL), 3 d; SC&#x2013;histidine, with galactose and 0.1mM aminotriazole, 5 d. All the media used in these experiments contained glucose as the carbon source, except where otherwise indicated.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/6/929/F3.medium.gif" width="440" height="347"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/3/6/929/F3.medium.gif" width="440" height="347"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/6/929/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/6/929/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462928" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-26" class="first-child">Phenotypic assays indicate that the H4-R36A and H3-L61W mutants affect common cellular processes. (A) Cells expressing wild-type or mutant histones and either wild-type or mutant Spt16 proteins, as indicated, were grown to saturation on YPD medium, harvested, and then spotted on the indicated media in 10-fold dilution series, in which the most concentrated spots (left-most spot in each panel) contained ~2 × 10<sup>6</sup> cells. The plates were incubated at 30° (except for the plates shown in the second column, which were incubated at 14° as indicated) and photographs were taken after the following approximate times of incubation: YPD, 2 d; YPD 14°, 13 d; YPD + 15mM caffeine (Caff.), 4 d; YPD + 3% formamide (Form.), 4 d; YPD + 150mM hydroxyurea (HU), 4 d. The strains used in these experiments were yAAD75-yAAD81. (B) The same strains used in the experiments described in (A) but spotted on different media, as indicated, such that the most concentrated spots contained ~4 × 10<sup>6</sup> cells. The plates were incubated at 30° and were photographed after the following approximate times of incubation: SC, 2 d; SC–lysine (lys), 3 d; SC–histidine (his), and with 0.1mM aminotriazole (AT, a competitive inhibitor of the product of the <em>HIS3</em> gene), 5 d; SC with galactose (GAL), 3 d; SC–histidine, with galactose and 0.1mM aminotriazole, 5 d. All the media used in these experiments contained glucose as the carbon source, except where otherwise indicated.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">Consistent with the data shown in <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>, a mutant strain expressing H4-R36A from an integrated gene in our host strain displays several growth phenotypes, including a slow-growth phenotype under permissive conditions and Cs<sup>−</sup>, Caff<sup>s</sup>, Form<sup>s</sup>, and HU<sup>s</sup> phenotypes (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3A</a>). All of these phenotypes are in common with H3-L61W cells (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3A</a> and <a id="xref-ref-11-6" class="xref-bibr" href="#ref-11">Duina <em>et al.</em> 2007</a>; <a id="xref-ref-26-5" class="xref-bibr" href="#ref-26">Myers <em>et al.</em> 2011</a>). As a way to determine whether the H4-R36A mutant confers these phenotypes by perturbing the same cellular processes that are perturbed by the H3-L61W mutant, we asked whether two independent mutations in the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> protein previously shown to suppress H3-L61W-phenotypes (<a id="xref-ref-11-7" class="xref-bibr" href="#ref-11">Duina <em>et al.</em> 2007</a>) are also able to suppress H4-R36A phenotypes. As shown in <a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3A</a>, these mutants—<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>-E857Q and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>-E790K (referred to as <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>-857 and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>-790, respectively)—were found to suppress H4-R36A phenotypes in a manner similar to that seen in the context of the H3-L61W mutant. It is worth noting that H4-R36A appears to be generally a “weaker” mutant than the H3-L61W mutant because although the two mutants share similar phenotypes, those displayed by H4-R36A cells are generally milder and are more easily suppressed by the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> mutants. Nevertheless, the fact that H4-R36A and H3-L61W cells can be suppressed in a similar fashion by the same set of mutations is consistent with the notion that the two histone mutants affect cell functions through perturbations of common processes.</p></div><div id="sec-10" class="subsection"><h3>Comparison of the effects of H4-R36A and H3-L61W on phenotypes indicative of transcription and chromatin defects</h3><p id="p-28">To more fully characterize the H4-R36A mutant and to further compare it with the H3-L61W mutant, we tested cells expressing each of the histone mutants in the context of wild-type or mutant <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> proteins for phenotypes associated with transcription and chromatin defects. In the first set of experiments, we assayed for Spt<sup>−</sup> phenotypes by testing cells harboring the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000319">lys2</a>-128δ</em> allele for their ability to grow in the absence of lysine in the growth medium (for a description of Spt<sup>−</sup> phenotypes, see <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Winston 1992</a>). As previously reported, both H4-R36A and H3-L61W cells displayed an Spt<sup>-</sup> phenotype (<a id="xref-ref-11-8" class="xref-bibr" href="#ref-11">Duina <em>et al.</em> 2007</a>; <a id="xref-ref-8-9" class="xref-bibr" href="#ref-8">Dai <em>et al.</em> 2008</a>), and neither histone mutant strains were found to be significantly suppressed for this phenotype by either <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> mutants (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3B</a>).</p><p id="p-29">In the second set of experiments, we used a reporter construct to indirectly assay for defects in nucleosome reassembly during transcription elongation (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Cheung <em>et al.</em> 2008</a>). In this reporter system, the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">HIS3</a></em> gene is fused to a cryptic promoter within the coding region of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000911">FLO8</a></em> gene and the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000911">FLO8</a></em> promoter has been replaced with the inducible <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000224">GAL1</a></em> promoter. With this reporter, improper nucleosome reassembly and the resulting intragenic cryptic transcription initiation can easily be monitored by testing the ability of cells (deleted for their endogenous <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">HIS3</a></em> gene) to grow on media lacking histidine under conditions in which the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000224">GAL1</a></em> promoter is either activated or repressed. As shown in <a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3B</a>, H4-R36A cells had modest growth on medium lacking histidine and containing glucose and much stronger growth on medium lacking histidine and containing galactose, indicating that cryptic initiation occurs in H4-R36A cells and that this defect is much more pronounced in the context of elevated levels of transcription. [It is worth noting that in another study, cryptic initiation events were not detected at the endogenous <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000911">FLO8</a></em> gene in H4-R36A cells using Northern blotting (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hainer and Martens 2011</a>)—the discrepancy between these results and ours could be due to differences in the level of sensitivity of the assays used in the two studies or might reflect an authentic difference in the effect of the H4-R36A in the context of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000911">FLO8</a>-<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">HIS3</a> vs.</em> that of the endogenous <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000911">FLO8</a></em> gene]. Consistent with previous results (<a id="xref-ref-26-6" class="xref-bibr" href="#ref-26">Myers <em>et al.</em> 2011</a>), H3-L61W cells also grew on media lacking histidine in the presence of either glucose or galactose, indicating that this histone mutant also causes cryptic intragenic transcription at <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000911">FLO8</a>-<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">HIS3</a></em> under both conditions of high transcription levels and of marginal to no transcription (<a id="xref-fig-3-9" class="xref-fig" href="#F3">Figure 3B</a>). The patterns of suppression of the cryptic transcription phenotypes by the two <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> mutants were found to be generally similar for H4-R36A and H3-L61W cells, with the notable exception of the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>-790 mutant, which under conditions of high transcription strongly suppresses the cryptic initiation phenotype of H4-R36A cells but only modestly suppresses the same defect in H3-L61W cells (<a id="xref-fig-3-10" class="xref-fig" href="#F3">Figure 3B</a>).</p><p id="p-30">Taken together, the results from these experiments show that H3-L61W and H4-R36A confer similar phenotypes associated with transcription and chromatin defects and that one of these phenotypes (that seen in the context of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000911">FLO8</a>-<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">HIS3</a></em> reporter construct) can be partially suppressed by the same two <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> mutants, albeit at varying levels of efficacy.</p></div><div id="sec-11" class="subsection"><h3>Evidence that H4-R36A affects the pattern of association of Spt16 across the <em>PMA1</em> and <em>FBA1</em> genes by perturbing the same process that is also perturbed by the H3-L61W mutant</h3><p id="p-31">We sought to determine whether the phenotypic characteristics shared by H4-R36A and H3-L61W cells discussed previously were reflective of a common mechanism by which these two histone mutants perturb <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> association with transcribed genes. To do this, we asked whether the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>-790 and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>-857 mutants, which had been previously shown to partially suppress the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> distribution defect in H3-L61W cells (<a id="xref-ref-11-9" class="xref-bibr" href="#ref-11">Duina <em>et al.</em> 2007</a>), could also suppress the defects in <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> occupancy across the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002976">PMA1</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001543">FBA1</a></em> genes caused by the H4-R36A mutant. As shown in <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>, both <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> mutants are in fact able to partially suppress the H4-R36A-dependent shift in <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> distribution across both genes. Interestingly, the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>-790 has a more pronounced suppressive effect than <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>-857, a finding that parallels what is seen in the context of the H3-L61W mutant (<a id="xref-ref-11-10" class="xref-bibr" href="#ref-11">Duina <em>et al.</em> 2007</a>). It is also noteworthy that H4-R36A and H3-L61W affect <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> occupancy across both genes to similar degrees. In these experiments we also found that an H4-R36K mutation causes some defect in <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> association with chromatin, in particular at the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001543">FBA1</a></em> gene, indicating that an aspect of the arginine residue other than its basicity is important for its participation in this process. Collectively, these results indicate that the H4-R36A and H3-L61W mutants affect <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> association with transcribed genes to a similar extent and that both likely perturb a common process normally required for proper <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>−chromatin interactions during transcription.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/6/929/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The H4-R36A&#x2212;dependent defects in Spt16 distribution across the PMA1 and FBA1 genes are partially suppressed by two Spt16 mutants and an H4-R36K mutant causes some perturbations in Spt16 interactions with the same genes. (A&#x2212;D) Results from Spt16 ChIP experiments performed on the indicated strains are presented using the same methodologies described in Figure 1. The three regions (5&#x2032;, internal, and 3&#x2032;) assayed for each of the two genes are the same as those shown in Figure 1. The strains used in these experiments were yADP75-79 and yADP82. For each strain, the data are presented as the mean &#xB1; SEM from five independent experiments. As a reference, the NO ORF %IP values for each strain were as follows: yADP75: 0.18 &#xB1; 0.050, yADP76: 0.25 &#xB1; 0.038, yADP77: 0.29 &#xB1; 0.049, yADP78: 0.33 &#xB1; 0.075, yADP82: 0.22 &#xB1; 0.043, yADP79: 0.75 &#xB1; 0.257. Note that the mean value for Spt16 binding to the NO ORF region in H3-L61W cells (yADP79) is greater than what is seen in the other strains. Although in these particular experiments the SEM is quite high (making the mean level of Spt16 NO ORF binding in H3-L61W cells not statistically different from the mean level of Spt16 NO ORF binding in wild-type cells), in previous studies we have seen a consistent trend in which Spt16 binding to the NO ORF region in H3-L61W cells appears to be ~2-fold greater than what is normally seen in wild-type cells (Duina et al. 2007; Lloyd et al. 2009; Myers et al. 2011)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-586614734" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The H4-R36A&#x2212;dependent defects in Spt16 distribution across the &lt;em&gt;PMA1&lt;/em&gt; and &lt;em&gt;FBA1&lt;/em&gt; genes are partially suppressed by two Spt16 mutants and an H4-R36K mutant causes some perturbations in Spt16 interactions with the same genes. (A&#x2212;D) Results from Spt16 ChIP experiments performed on the indicated strains are presented using the same methodologies described in Figure 1. The three regions (5&#x2032;, internal, and 3&#x2032;) assayed for each of the two genes are the same as those shown in Figure 1. The strains used in these experiments were yADP75-79 and yADP82. For each strain, the data are presented as the mean &#xB1; SEM from five independent experiments. As a reference, the NO ORF %IP values for each strain were as follows: yADP75: 0.18 &#xB1; 0.050, yADP76: 0.25 &#xB1; 0.038, yADP77: 0.29 &#xB1; 0.049, yADP78: 0.33 &#xB1; 0.075, yADP82: 0.22 &#xB1; 0.043, yADP79: 0.75 &#xB1; 0.257. Note that the mean value for Spt16 binding to the NO ORF region in H3-L61W cells (yADP79) is greater than what is seen in the other strains. Although in these particular experiments the SEM is quite high (making the mean level of Spt16 NO ORF binding in H3-L61W cells not statistically different from the mean level of Spt16 NO ORF binding in wild-type cells), in previous studies we have seen a consistent trend in which Spt16 binding to the NO ORF region in H3-L61W cells appears to be ~2-fold greater than what is normally seen in wild-type cells (Duina &lt;em&gt;et al.&lt;/em&gt; 2007; Lloyd &lt;em&gt;et al.&lt;/em&gt; 2009; Myers &lt;em&gt;et al.&lt;/em&gt; 2011).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/6/929/F4.medium.gif" width="440" height="350"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/3/6/929/F4.medium.gif" width="440" height="350"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/6/929/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/6/929/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462914" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-32" class="first-child">The H4-R36A−dependent defects in Spt16 distribution across the <em>PMA1</em> and <em>FBA1</em> genes are partially suppressed by two Spt16 mutants and an H4-R36K mutant causes some perturbations in Spt16 interactions with the same genes. (A−D) Results from Spt16 ChIP experiments performed on the indicated strains are presented using the same methodologies described in <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>. The three regions (5′, internal, and 3′) assayed for each of the two genes are the same as those shown in <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>. The strains used in these experiments were yADP75-79 and yADP82. For each strain, the data are presented as the mean ± SEM from five independent experiments. As a reference, the NO ORF %IP values for each strain were as follows: yADP75: 0.18 ± 0.050, yADP76: 0.25 ± 0.038, yADP77: 0.29 ± 0.049, yADP78: 0.33 ± 0.075, yADP82: 0.22 ± 0.043, yADP79: 0.75 ± 0.257. Note that the mean value for Spt16 binding to the NO ORF region in H3-L61W cells (yADP79) is greater than what is seen in the other strains. Although in these particular experiments the SEM is quite high (making the mean level of Spt16 NO ORF binding in H3-L61W cells not statistically different from the mean level of Spt16 NO ORF binding in wild-type cells), in previous studies we have seen a consistent trend in which Spt16 binding to the NO ORF region in H3-L61W cells appears to be ~2-fold greater than what is normally seen in wild-type cells (<a id="xref-ref-11-11" class="xref-bibr" href="#ref-11">Duina <em>et al.</em> 2007</a>; <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Lloyd <em>et al.</em> 2009</a>; <a id="xref-ref-26-7" class="xref-bibr" href="#ref-26">Myers <em>et al.</em> 2011</a>).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>The alteration in Spt16 distribution across genes seen in H4-R36A cells appears to be dependent on active transcription</h3><p id="p-33">We have previously shown that the H3-L61W mutant markedly affects the distribution pattern of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> across the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000224">GAL1</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004071">GAL2</a></em> genes but only under conditions in which these genes are being actively transcribed (<em>i.e.</em>, in the presence of galactose), thus indicating that the effects of H3-L61W on <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> interactions with chromatin across genes are dependent on transcription (<a id="xref-ref-11-12" class="xref-bibr" href="#ref-11">Duina <em>et al.</em> 2007</a>). To determine whether H4-R36A also affects <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>-gene interactions in a transcription-dependent fashion, we carried out ChIP experiments in wild-type and H4-R36A cells grown in either glucose or galactose medium and determined the level of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> occupancy across the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000224">GAL1</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004071">GAL2</a></em> genes. As shown in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5B</a>, whereas the background pattern of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> occupancy across the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000224">GAL1</a></em> gene is indistinguishable in wild-type and H4-R36A cells grown under repressive conditions, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> association pattern across <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000224">GAL1</a></em> under activating conditions is significantly affected by the H4-R36A mutant, resulting in an overall shift in <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> occupancy toward the 3′ end of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000224">GAL1</a></em> gene. A similar but more striking effect is also seen at the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004071">GAL2</a></em> gene (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5E</a>). As expected, we detected abnormal <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> association across the constitutively expressed gene <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002976">PMA1</a></em> in H4-R36A cells regardless of the carbon source used in the experiment (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5C</a>). The data from these experiments are consistent with the notion that, similarly to H3-L61W, the H4-R36A mutant causes perturbations in <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> association across transcribed genes in a manner that is dependent on active transcription.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/6/929/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The H4-R36A mutant causes pronounced defects in Spt16 association across the GAL1 and GAL2 genes but only under conditions that activate these genes. (A, D) Schematic representations of the GAL1 and GAL2 genes using the same numbering and labeling systems as those described in Figure 1. (B, C, E) Cells expressing wild-type histone H4 (WT) or the H4-R36A mutant grown in medium containing either glucose (Glu) or galactose (Gal) were subjected to Spt16 ChIP assays, and levels of Spt16 occupancy at the 5&#x2032; and 3&#x2032; locations of the GAL1 and GAL2 genes indicated in (A) and (D) and at the 5&#x2032; and 3&#x2032; locations of PMA1 indicated in Figure 1 were determined. The results are presented using the same methodology as that described in Figure 1. The strains used in these experiments were yADP75 and yADP76. For each strain, the data are presented as the mean &#xB1; SEM from three independent experiments. As a reference, the NO ORF %IP values for each strain were as follows: yADP75: 0.23 &#xB1; 0.072 (Glu) and 0.58 &#xB1; 0.186 (Gal), yADP76: 0.23 &#xB1; 0.054 (Glu) and 0.82 &#xB1; 0.088 (Gal). Note that some of the data shown in this figure correspond to a subset of the data already presented in Figure 4." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-586614734" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The H4-R36A mutant causes pronounced defects in Spt16 association across the &lt;em&gt;GAL1&lt;/em&gt; and &lt;em&gt;GAL2&lt;/em&gt; genes but only under conditions that activate these genes. (A, D) Schematic representations of the &lt;em&gt;GAL1&lt;/em&gt; and &lt;em&gt;GAL2&lt;/em&gt; genes using the same numbering and labeling systems as those described in Figure 1. (B, C, E) Cells expressing wild-type histone H4 (WT) or the H4-R36A mutant grown in medium containing either glucose (Glu) or galactose (Gal) were subjected to Spt16 ChIP assays, and levels of Spt16 occupancy at the 5&#x2032; and 3&#x2032; locations of the &lt;em&gt;GAL1&lt;/em&gt; and &lt;em&gt;GAL2&lt;/em&gt; genes indicated in (A) and (D) and at the 5&#x2032; and 3&#x2032; locations of &lt;em&gt;PMA1&lt;/em&gt; indicated in Figure 1 were determined. The results are presented using the same methodology as that described in Figure 1. The strains used in these experiments were yADP75 and yADP76. For each strain, the data are presented as the mean &#xB1; SEM from three independent experiments. As a reference, the NO ORF %IP values for each strain were as follows: yADP75: 0.23 &#xB1; 0.072 (Glu) and 0.58 &#xB1; 0.186 (Gal), yADP76: 0.23 &#xB1; 0.054 (Glu) and 0.82 &#xB1; 0.088 (Gal). Note that some of the data shown in this figure correspond to a subset of the data already presented in Figure 4.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/6/929/F5.medium.gif" width="440" height="415"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/3/6/929/F5.medium.gif" width="440" height="415"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/6/929/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/6/929/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462842" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-34" class="first-child">The H4-R36A mutant causes pronounced defects in Spt16 association across the <em>GAL1</em> and <em>GAL2</em> genes but only under conditions that activate these genes. (A, D) Schematic representations of the <em>GAL1</em> and <em>GAL2</em> genes using the same numbering and labeling systems as those described in <a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1</a>. (B, C, E) Cells expressing wild-type histone H4 (WT) or the H4-R36A mutant grown in medium containing either glucose (Glu) or galactose (Gal) were subjected to Spt16 ChIP assays, and levels of Spt16 occupancy at the 5′ and 3′ locations of the <em>GAL1</em> and <em>GAL2</em> genes indicated in (A) and (D) and at the 5′ and 3′ locations of <em>PMA1</em> indicated in <a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1</a> were determined. The results are presented using the same methodology as that described in <a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1</a>. The strains used in these experiments were yADP75 and yADP76. For each strain, the data are presented as the mean ± SEM from three independent experiments. As a reference, the NO ORF %IP values for each strain were as follows: yADP75: 0.23 ± 0.072 (Glu) and 0.58 ± 0.186 (Gal), yADP76: 0.23 ± 0.054 (Glu) and 0.82 ± 0.088 (Gal). Note that some of the data shown in this figure correspond to a subset of the data already presented in <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>The H4-R36A mutant confers modest defects in the distribution of Pol II and nucleosomes across the <em>PMA1</em> and <em>FBA1</em> genes</h3><p id="p-35">To further characterize the effects of the H4-R36A mutant on the transcription elongation process, we carried out additional ChIP experiments to determine the effects of this histone mutant on Pol II and bulk nucleosome occupancy across the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002976">PMA1</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001543">FBA1</a></em> genes. In these experiments, we found that the H4-R36A mutant caused a modest reduction in occupancy of both the Pol II subunit <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001283">Rpb3</a> and histone H3 at the 5′ ends of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002976">PMA1</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001543">FBA1</a></em> and had no significant effect on the levels of occupancy of the two proteins at the 3′ ends of the same genes [<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figures 6</a> and <a id="xref-fig-7-1" class="xref-fig" href="#F7">7</a>; note that our results on the effects of H4-R36A on histone H3 occupancy at <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002976">PMA1</a></em> are very similar to those reported in a recent study (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Hainer and Martens 2011</a>)]. Because these effects are rather minor compared with the effects imparted by the H4-R36A on <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> distribution across the same genes, we interpret these results as evidence that the effects of H4-R36A on transcription elongation events are at least to some degree specific to <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>. A similar conclusion was reached for the H3-L61W mutant based on similar studies performed in studies we described previously (<a id="xref-ref-11-13" class="xref-bibr" href="#ref-11">Duina <em>et al.</em> 2007</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/6/929/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Cells expressing the H4-R36A mutant show minimal perturbations in the association of the Pol II component Rpb3 across the PMA1 and FBA1 genes. (A&#x2212;D) Results from ChIP experiments in which the distribution of Rpb3 across the PMA1 and FBA1 genes was assayed in cells expressing either wild-type histone H4 (WT) or the H4-R36A mutant. The strains used in these experiments were yADP75 and yADP76. The results are presented using the same methodology as that described for the experiments shown in Figure 1. For each strain, the data are presented as the mean &#xB1; SEM from three independent experiments. As a reference, the NO ORF %IP values for each strain were as follows: yADP75: 0.03 &#xB1; 0.007, yADP76 and 0.02 &#xB1; 0.005." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-586614734" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Cells expressing the H4-R36A mutant show minimal perturbations in the association of the Pol II component Rpb3 across the &lt;em&gt;PMA1&lt;/em&gt; and &lt;em&gt;FBA1&lt;/em&gt; genes. (A&#x2212;D) Results from ChIP experiments in which the distribution of Rpb3 across the &lt;em&gt;PMA1&lt;/em&gt; and &lt;em&gt;FBA1&lt;/em&gt; genes was assayed in cells expressing either wild-type histone H4 (WT) or the H4-R36A mutant. The strains used in these experiments were yADP75 and yADP76. The results are presented using the same methodology as that described for the experiments shown in Figure 1. For each strain, the data are presented as the mean &#xB1; SEM from three independent experiments. As a reference, the NO ORF %IP values for each strain were as follows: yADP75: 0.03 &#xB1; 0.007, yADP76 and 0.02 &#xB1; 0.005.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/6/929/F6.medium.gif" width="440" height="348"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/3/6/929/F6.medium.gif" width="440" height="348"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/6/929/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/6/929/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462932" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-36" class="first-child">Cells expressing the H4-R36A mutant show minimal perturbations in the association of the Pol II component Rpb3 across the <em>PMA1</em> and <em>FBA1</em> genes. (A−D) Results from ChIP experiments in which the distribution of Rpb3 across the <em>PMA1</em> and <em>FBA1</em> genes was assayed in cells expressing either wild-type histone H4 (WT) or the H4-R36A mutant. The strains used in these experiments were yADP75 and yADP76. The results are presented using the same methodology as that described for the experiments shown in <a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1</a>. For each strain, the data are presented as the mean ± SEM from three independent experiments. As a reference, the NO ORF %IP values for each strain were as follows: yADP75: 0.03 ± 0.007, yADP76 and 0.02 ± 0.005.</p><div class="sb-div caption-clear"></div></div></div><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/6/929/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The H4-R36A causes a modest change in bulk nucleosome distribution across the PMA1 and FBA1 genes. (A&#x2212;D) ChIP experiments were carried out on cells expressing either wild-type histone H4 (WT) or the H4-R36A mutant using an antibody that recognizes bulk histone H3. The strains used in these experiments were yADP75 and yADP76. The results from these experiments are presented using the same methodology as that described in the legend to Figure 1. For each strain, the data are presented as the mean &#xB1; SEM from three independent experiments. As a reference, the NO ORF %IP values for each strain were as follows: yADP75: 1.60 &#xB1; 0.603, yADP76: 2.04 &#xB1; 0.066." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-586614734" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The H4-R36A causes a modest change in bulk nucleosome distribution across the &lt;em&gt;PMA1&lt;/em&gt; and &lt;em&gt;FBA1&lt;/em&gt; genes. (A&#x2212;D) ChIP experiments were carried out on cells expressing either wild-type histone H4 (WT) or the H4-R36A mutant using an antibody that recognizes bulk histone H3. The strains used in these experiments were yADP75 and yADP76. The results from these experiments are presented using the same methodology as that described in the legend to Figure 1. For each strain, the data are presented as the mean &#xB1; SEM from three independent experiments. As a reference, the NO ORF %IP values for each strain were as follows: yADP75: 1.60 &#xB1; 0.603, yADP76: 2.04 &#xB1; 0.066.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/6/929/F7.medium.gif" width="440" height="349"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/3/6/929/F7.medium.gif" width="440" height="349"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/6/929/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/6/929/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462959" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-37" class="first-child">The H4-R36A causes a modest change in bulk nucleosome distribution across the <em>PMA1</em> and <em>FBA1</em> genes. (A−D) ChIP experiments were carried out on cells expressing either wild-type histone H4 (WT) or the H4-R36A mutant using an antibody that recognizes bulk histone H3. The strains used in these experiments were yADP75 and yADP76. The results from these experiments are presented using the same methodology as that described in the legend to <a id="xref-fig-1-10" class="xref-fig" href="#F1">Figure 1</a>. For each strain, the data are presented as the mean ± SEM from three independent experiments. As a reference, the NO ORF %IP values for each strain were as follows: yADP75: 1.60 ± 0.603, yADP76: 2.04 ± 0.066.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-14"><h2 class="">Discussion</h2><p id="p-38">In this work, we have shown that two specific histone H4 mutants—H4-R36A and H4-K31E—cause perturbations in the pattern of association of the transcription elongation factor <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> across transcribed genes. In particular, we found that these histone mutants cause a shift in the distribution of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> toward the 3′ regions of the transcribed genes we investigated. These effects are similar to those we previously reported for cells expressing the histone H3 mutant H3-L61W, and a closer inspection of the effects conferred by H4-R36A indicates that this histone mutant and H3-L61W likely affect <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> association with transcribed genes through perturbations of a common process. These results, combined with the fact that H4-R36, H4-K31, and H3-L61 are located in close proximity to each other on the nucleosome structure, suggest that these histone residues may define a nucleosomal region required for proper <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>-chromatin interactions during the process of transcription elongation.</p><p id="p-39">Whereas it is clear that the H4-R36A, H4-K31E, and H3-L61W mutants perturb normal <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>−chromatin interactions during transcription, the mechanism underlying the shift in <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> distribution toward the 3′ ends of transcribed genes still remains to be elucidated. A simple interpretation of our results is that the histone mutants affect two aspects of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>−gene interactions: first, the mutants impair proper recruitment of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> at the 5′ ends of transcribed genes, and second, the mutants interfere with the proper disengagement of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> over the 3′ regions of the same genes. This model is consistent with our ChIP data showing lower <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> abundance at the 5′ends of transcribed genes and abnormally high levels of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> occupancy at the 3′ ends of the same genes relative to the amount recruited over the corresponding 5′ regions. Whereas it would seem likely that the histone mutants interfere with proper <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> dissociation from chromatin at the 3′ ends of transcribed genes in a direct fashion, the lower levels of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> occupancy at the 5′ regions of the same genes may be reflective of either a direct impairment in <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> recruitment to genes over these regions or an indirect effect stemming from an overall lower concentration of available <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> in the nucleoplasm for gene recruitment due to abnormal retention of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> at the 3′ ends of genes genome-wide. We note that implicit to our model is the assumption that <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> can only be recruited to the 5′ end of transcribed genes and that it does not disengage from chromatin until it has traveled across the entire length of these genes. However, it is also possible that distinct <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> proteins are recruited to and dissociate from chromatin throughout the length of transcribed genes in a dynamic fashion—in this scenario, our histone mutants would be postulated to affect one or more aspects of these dynamic interactions ultimately resulting in abnormal patterns of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> distribution across transcribed genes as we have described.</p><p id="p-40">How might the H4-R36, H4-K31, and H3-L61 residues normally participate in ensuring proper interactions between <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> and chromatin during transcription elongation? It could be envisioned that a region of the nucleosome that includes these residues participates in direct interactions with yFACT and that these interactions are in some way required for proper yFACT disengagement from, and possibly recruitment to, chromatin during the transcription process. The contribution of each of the three histone residues in ensuring proper <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>−chromatin interactions is yet to be defined. Within the context of an intact nucleosome, the H4-R36 residue directly contacts the DNA (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Luger <em>et al.</em> 1997</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Luger and Richmond 1998</a>) and thus it is possible that this DNA−histone interaction is an important feature of the nucleosome required for efficient interactions between yFACT and chromatin. Alternatively, it is conceivable that H4-R36 makes direct interactions with yFACT at some point during yFACT-mediated manipulations of nucleosomes. The H3-L61 residue is buried and therefore unlikely to make direct contacts with yFACT (at least in the context of an intact nucleosome), but a substitution to a tryptophan could cause a structural perturbation within nucleosomes that would in turn lead to abnormal yFACT-chromatin interactions. It is also possible that the H3-L61W mutant exerts its effects by interfering with the ability of the nearby H4-R36 residue to form proper bonds with the DNA or possibly, as discussed previously, with yFACT itself. Finally, H4-K31 is exposed on the surface of the nucleosome and could possibly interact with yFACT directly.</p><p id="p-41">The H4-R36A mutant was among eight histone mutants identified in a recent study that cause reductions in nucleosome occupancy at several highly transcribed genes (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Hainer and Martens 2011</a>). Furthermore, in this and previous studies (<a id="xref-ref-26-8" class="xref-bibr" href="#ref-26">Myers <em>et al.</em> 2011</a>), we have shown that both H4-R36A and H3-L61W cause cryptic intragenic transcription initiation events at the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000911">FLO8</a>:<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">HIS3</a></em> gene [and, in the case of H3-L61W, at the endogenous <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000911">FLO8</a></em> gene as well (<a id="xref-ref-11-14" class="xref-bibr" href="#ref-11">Duina <em>et al.</em> 2007</a>)]. Collectively, these observations suggest that these histone mutants might interfere with one or more steps involved in transcription-coupled nucleosome reassembly. Whether this transcription-coupled nucleosome reassembly defect is related to the perturbations in <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> association across transcribed genes seen in the context of our histone mutants remains an open question. It could be envisioned, for example, that yFACT-mediated nucleosome reassembly is a required step in ensuring proper disengagement of yFACT from the ends of genes and mutations—in the histones or conceivably in other proteins as well—that interfere with this step would cause accumulation of yFACT at the 3′ ends of genes. Alternatively, it could be proposed that the inability of yFACT to properly disengage from the 3′ ends of genes in the histone mutant backgrounds results in the presence of abnormally elevated levels of stalled yFACT-nucleosome complexes over the 3′ ends of genes, which in turn could be responsible for the cryptic intragenic transcription phenotypes we have observed. In a previous study, however, we obtained results in apparent conflict with these hypotheses as we found that among a battery of newly isolated <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> mutants, one of them—<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>-E735G—was among the strongest suppressors of the H3-L61W−dependent <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> 3′ accumulation at <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002976">PMA1</a></em> but was among the weakest suppressor of the cryptic initiation phenotype at <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000911">FLO8</a>:<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">HIS3</a></em> (<a id="xref-ref-26-9" class="xref-bibr" href="#ref-26">Myers <em>et al.</em> 2011</a>); thus, at least in this case, dissociation of yFACT from the 3′ end of a gene and nucleosome reassembly appear not to be functionally linked. However, additional genetic and biochemical experiments need to be carried out to more directly assess a possible functional relationship between transcription-coupled nucleosome reassembly and the processes that regulate yFACT disengagement from the ends of genes after transcription.</p><p id="p-42">Collectively, our results show that specific histone mutants can significantly perturb the association patterns of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> across transcribed genes and suggest that a region of the nucleosome that encompasses the H4-R36, H4-K31, and H3-L61 residues on the side of the nucleosome core particle plays an important role in ensuring proper association of yFACT with transcribed genes. Future studies will be directed toward a more complete characterization of this nucleosomal region and will include studies aimed at identifying other residues in the vicinity of the H4-R36, H4-K31, and H3-L61 residues that may contribute to proper yFACT−gene interactions.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-43">We express our gratitude to Jef Boeke, Junbiao Dai, Tim Formosa, and Inés Pinto for providing some of the reagents used in this study and to Jef Boeke, Sarah Marshall, Joseph Martens, Reine Protacio, and Fred Winston for helpful comments on the manuscript. This material is based upon work supported by the National Science Foundation under grant no. 0919241.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-5"><p id="p-44"><em>Communicating editor: C. S. Hoffman</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2013-02-21"><span class="received-label">Received </span>February 21, 2013.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2013-04-01"><span class="accepted-label">Accepted </span>April 1, 2013.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2013 Nguyen <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.929.1" data-doi="10.1016/j.molcel.2011.02.013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Avvakumov</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nourani</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cote</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Histone chaperones: modulators of chromatin marks.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">502</span>–<span class="cit-lpage">514</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DAvvakumov%26rft.auinit1%253DN.%26rft.volume%253D41%26rft.issue%253D5%26rft.spage%253D502%26rft.epage%253D514%26rft.atitle%253DHistone%2Bchaperones%253A%2Bmodulators%2Bof%2Bchromatin%2Bmarks.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2011.02.013%26rft_id%253Dinfo%253Apmid%252F21362547%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2011.02.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21362547&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F929.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000288150700004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.929.2" data-doi="10.1126/science.1085703"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Belotserkovskaya</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oh</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bondarenko</span>  <span class="cit-name-given-names">V. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Orphanides</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Studitsky</span>  <span class="cit-name-given-names">V. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">FACT facilitates transcription-dependent nucleosome alteration.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">301</span>: <span class="cit-fpage">1090</span>–<span class="cit-lpage">1093</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBelotserkovskaya%26rft.auinit1%253DR.%26rft.volume%253D301%26rft.issue%253D5636%26rft.spage%253D1090%26rft.epage%253D1093%26rft.atitle%253DFACT%2BFacilitates%2BTranscription-Dependent%2BNucleosome%2BAlteration%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1085703%26rft_id%253Dinfo%253Apmid%252F12934006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDEvNTYzNi8xMDkwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy82LzkyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.929.3" data-doi="10.1074/jbc.273.34.21972"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brewster</span>  <span class="cit-name-given-names">N. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">G. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singer</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Characterization of the CP complex, an abundant dimer of Cdc68 and Pob3 proteins that regulates yeast transcriptional activation and chromatin repression.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">273</span>: <span class="cit-fpage">21972</span>–<span class="cit-lpage">21979</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DBrewster%26rft.auinit1%253DN.%2BK.%26rft.volume%253D273%26rft.issue%253D34%26rft.spage%253D21972%26rft.epage%253D21979%26rft.atitle%253DCharacterization%2Bof%2Bthe%2BCP%2BComplex%252C%2Ban%2BAbundant%2BDimer%2Bof%2BCdc68%2Band%2BPob3%2BProteins%2BThat%2BRegulates%2BYeast%2BTranscriptional%2BActivation%2Band%2BChromatin%2BRepression%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.273.34.21972%26rft_id%253Dinfo%253Apmid%252F9705338%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzMvMzQvMjE5NzIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzYvOTI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.929.4" data-doi="10.1128/MCB.21.10.3491-3502.2001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brewster</span>  <span class="cit-name-given-names">N. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">G. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singer</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">A bipartite yeast SSRP1 analog comprised of Pob3 and Nhp6 proteins modulates transcription.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">3491</span>–<span class="cit-lpage">3502</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DBrewster%26rft.auinit1%253DN.%2BK.%26rft.volume%253D21%26rft.issue%253D10%26rft.spage%253D3491%26rft.epage%253D3502%26rft.atitle%253DA%2BBipartite%2BYeast%2BSSRP1%2BAnalog%2BComprised%2Bof%2BPob3%2Band%2BNhp6%2BProteins%2BModulates%2BTranscription%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.21.10.3491-3502.2001%26rft_id%253Dinfo%253Apmid%252F11313475%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyMS8xMC8zNDkxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy82LzkyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.929.5" data-doi="10.1146/annurev.genet.032608.103928"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Campos</span>  <span class="cit-name-given-names">E. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reinberg</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Histones: annotating chromatin.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">559</span>–<span class="cit-lpage">599</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DCampos%26rft.auinit1%253DE.%2BI.%26rft.volume%253D43%26rft.spage%253D559%26rft.epage%253D599%26rft.atitle%253DHistones%253A%2Bannotating%2Bchromatin.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.032608.103928%26rft_id%253Dinfo%253Apmid%252F19886812%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genet.032608.103928&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19886812&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F929.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273580300023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.929.6" data-doi="10.1093/nar/gks1472"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carvalho</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raposo</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martins</span>  <span class="cit-name-given-names">F. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grosso</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sridhara</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Histone methyltransferase SETD2 coordinates FACT recruitment with nucleosome dynamics during transcription.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">2881</span>–<span class="cit-lpage">2893</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks1472%26rft_id%253Dinfo%253Apmid%252F23325844%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjQxLzUvMjg4MSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNi85MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.929.7" data-doi="10.1371/journal.pbio.0060277"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheung</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chua</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Batada</span>  <span class="cit-name-given-names">N. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landry</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michnick</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Chromatin- and transcription-related factors repress transcription from within coding regions throughout the Saccharomyces cerevisiae genome.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e277</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DCheung%26rft.auinit1%253DV.%26rft.volume%253D6%26rft.issue%253D11%26rft.spage%253De277%26rft.epage%253De277%26rft.atitle%253DChromatin-%2Band%2Btranscription-related%2Bfactors%2Brepress%2Btranscription%2Bfrom%2Bwithin%2Bcoding%2Bregions%2Bthroughout%2Bthe%2BSaccharomyces%2Bcerevisiae%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0060277%26rft_id%253Dinfo%253Apmid%252F18998772%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0060277&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18998772&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F929.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.929.8" data-doi="10.1016/j.cell.2008.07.019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dai</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hyland</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bader</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Probing nucleosome function: a highly versatile library of synthetic histone H3 and H4 mutants.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">134</span>: <span class="cit-fpage">1066</span>–<span class="cit-lpage">1078</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DDai%26rft.auinit1%253DJ.%26rft.volume%253D134%26rft.issue%253D6%26rft.spage%253D1066%26rft.epage%253D1078%26rft.atitle%253DProbing%2Bnucleosome%2Bfunction%253A%2Ba%2Bhighly%2Bversatile%2Blibrary%2Bof%2Bsynthetic%2Bhistone%2BH3%2Band%2BH4%2Bmutants.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2008.07.019%26rft_id%253Dinfo%253Apmid%252F18805098%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2008.07.019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18805098&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F929.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259318100025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-book" id="cit-3.6.929.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duina</span>  <span class="cit-name-given-names">A. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Histone chaperones Spt6 and FACT: similarities and differences in modes of action at transcribed genes.</span> <span class="cit-source">Genet. Res. Int.</span> <span class="cit-vol">134</span>: <span class="cit-publ-loc">Article, ID 625201</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.929.10" data-doi="10.1128/MCB.24.2.561-572.2004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duina</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winston</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Analysis of a mutant histone H3 that perturbs the association of Swi/Snf with chromatin.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">561</span>–<span class="cit-lpage">572</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DDuina%26rft.auinit1%253DA.%2BA.%26rft.volume%253D24%26rft.issue%253D2%26rft.spage%253D561%26rft.epage%253D572%26rft.atitle%253DAnalysis%2Bof%2Ba%2BMutant%2BHistone%2BH3%2BThat%2BPerturbs%2Bthe%2BAssociation%2Bof%2BSwi%252FSnf%2Bwith%2BChromatin%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.24.2.561-572.2004%26rft_id%253Dinfo%253Apmid%252F14701730%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjg6IjI0LzIvNTYxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy82LzkyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.929.11" data-doi="10.1534/genetics.106.067140"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duina</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rufiange</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bracey</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nourani</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Evidence that the localization of the elongation factor Spt16 across transcribed genes is dependent upon histone H3 integrity in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">101</span>–<span class="cit-lpage">112</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.067140%26rft_id%253Dinfo%253Apmid%252F17603125%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc3LzEvMTAxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy82LzkyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.929.12" data-doi="10.1039/b812136b"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Formosa</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">FACT and the reorganized nucleosome.</span> <abbr class="cit-jnl-abbrev">Mol. Biosyst.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1085</span>–<span class="cit-lpage">1093</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DMol%2BBiosyst%26rft.aulast%253DFormosa%26rft.auinit1%253DT.%26rft.volume%253D4%26rft.issue%253D11%26rft.spage%253D1085%26rft.epage%253D1093%26rft.atitle%253DFACT%2Band%2Bthe%2Breorganized%2Bnucleosome.%26rft_id%253Dinfo%253Adoi%252F10.1039%252Fb812136b%26rft_id%253Dinfo%253Apmid%252F18931784%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1039/b812136b&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18931784&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F929.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.929.13" data-doi="10.1016/j.bbagrm.2011.07.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Formosa</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The role of FACT in making and breaking nucleosomes.</span> <abbr class="cit-jnl-abbrev">Biochim. Biophys. Acta</abbr> <span class="cit-vol">1819</span>: <span class="cit-fpage">247</span>–<span class="cit-lpage">255</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.stitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.aulast%253DFormosa%26rft.auinit1%253DT.%26rft.volume%253D1819%26rft.issue%253D3-4%26rft.spage%253D247%26rft.epage%253D255%26rft.atitle%253DThe%2Brole%2Bof%2BFACT%2Bin%2Bmaking%2Band%2Bbreaking%2Bnucleosomes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bbagrm.2011.07.009%26rft_id%253Dinfo%253Apmid%252F21807128%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bbagrm.2011.07.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21807128&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F929.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.929.14" data-doi="10.1093/emboj/20.13.3506"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Formosa</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eriksson</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wittmeyer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ginn</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Spt16-Pob3 and the HMG protein Nhp6 combine to form the nucleosome-binding factor SPN.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">3506</span>–<span class="cit-lpage">3517</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJournal%26rft.aulast%253DFormosa%26rft.auinit1%253DT.%26rft.volume%253D20%26rft.issue%253D13%26rft.spage%253D3506%26rft.epage%253D3517%26rft.atitle%253DSpt16-Pob3%2Band%2Bthe%2BHMG%2Bprotein%2BNhp6%2Bcombine%2Bto%2Bform%2Bthe%2Bnucleosome-binding%2Bfactor%2BSPN.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252F20.13.3506%26rft_id%253Dinfo%253Apmid%252F11432837%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMjAvMTMvMzUwNiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNi85MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.929.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Formosa</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruone</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olsen</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eriksson</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Defects in SPT16 or POB3 (yFACT) in <em>Saccharomyces cerevisiae</em> cause dependence on the Hir/Hpc pathway: polymerase passage may degrade chromatin structure.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">162</span>: <span class="cit-fpage">1557</span>–<span class="cit-lpage">1571</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DFormosa%26rft.auinit1%253DT.%26rft.volume%253D162%26rft.issue%253D4%26rft.spage%253D1557%26rft.epage%253D1571%26rft.atitle%253DDefects%2Bin%2BSPT16%2Bor%2BPOB3%2B%2528yFACT%2529%2Bin%2BSaccharomyces%2Bcerevisiae%2BCause%2BDependence%2Bon%2Bthe%2BHir%252FHpc%2BPathway%253A%2BPolymerase%2BPassage%2BMay%2BDegrade%2BChromatin%2BStructure%26rft_id%253Dinfo%253Apmid%252F12524332%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2Mi80LzE1NTciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzYvOTI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.929.16" data-doi="10.1128/MCB.05195-11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hainer</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martens</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Identification of histone mutants that are defective for transcription-coupled nucleosome occupancy.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">3557</span>–<span class="cit-lpage">3568</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.05195-11%26rft_id%253Dinfo%253Apmid%252F21730290%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIzMS8xNy8zNTU3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy82LzkyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.929.17" data-doi="10.1101/gr.083402.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maertens</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hyland</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dai</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Norris</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">HistoneHits: a database for histone mutations and their phenotypes.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">674</span>–<span class="cit-lpage">681</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.083402.108%26rft_id%253Dinfo%253Apmid%252F19218532%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE5LzQvNjc0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy82LzkyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.929.18" data-doi="10.1016/j.molcel.2009.07.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jamai</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Puglisi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strubin</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Histone chaperone spt16 promotes redeposition of the original h3-h4 histones evicted by elongating RNA polymerase.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">377</span>–<span class="cit-lpage">383</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DJamai%26rft.auinit1%253DA.%26rft.volume%253D35%26rft.issue%253D3%26rft.spage%253D377%26rft.epage%253D383%26rft.atitle%253DHistone%2Bchaperone%2Bspt16%2Bpromotes%2Bredeposition%2Bof%2Bthe%2Boriginal%2Bh3-h4%2Bhistones%2Bevicted%2Bby%2Belongating%2BRNA%2Bpolymerase.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2009.07.001%26rft_id%253Dinfo%253Apmid%252F19683500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2009.07.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19683500&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F929.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269278900014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.929.19" data-doi="10.1126/science.1087374"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kaplan</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laprade</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winston</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Transcription elongation factors repress transcription initiation from cryptic sites.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">301</span>: <span class="cit-fpage">1096</span>–<span class="cit-lpage">1099</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DKaplan%26rft.auinit1%253DC.%2BD.%26rft.volume%253D301%26rft.issue%253D5636%26rft.spage%253D1096%26rft.epage%253D1099%26rft.atitle%253DTranscription%2BElongation%2BFactors%2BRepress%2BTranscription%2BInitiation%2Bfrom%2BCryptic%2BSites%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1087374%26rft_id%253Dinfo%253Apmid%252F12934008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDEvNTYzNi8xMDk2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy82LzkyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.929.20" data-doi="10.1038/sj.emboj.7600053"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ahn</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krogan</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greenblatt</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buratowski</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Transitions in RNA polymerase II elongation complexes at the 3′ ends of genes.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">354</span>–<span class="cit-lpage">364</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BJ.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.emboj.7600053%26rft_id%253Dinfo%253Apmid%252F14739930%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czo4OiIyMy8yLzM1NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNi85MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.929.21" data-doi="10.1128/EC.00348-08"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lloyd</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pratt</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siebrasse</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moran</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duina</span>  <span class="cit-name-given-names">A. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Uncoupling of the patterns of chromatin association of different transcription elongation factors by a histone H3 mutant in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">257</span>–<span class="cit-lpage">260</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryot.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.00348-08%26rft_id%253Dinfo%253Apmid%252F19047363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo3OiI4LzIvMjU3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy82LzkyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.929.22" data-doi="10.1038/38444"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Luger</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mader</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richmond</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sargent</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richmond</span>  <span class="cit-name-given-names">T. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Crystal structure of the nucleosome core particle at 2.8 A resolution.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">389</span>: <span class="cit-fpage">251</span>–<span class="cit-lpage">260</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLuger%26rft.auinit1%253DK.%26rft.volume%253D389%26rft.issue%253D6648%26rft.spage%253D251%26rft.epage%253D260%26rft.atitle%253DCrystal%2Bstructure%2Bof%2Bthe%2Bnucleosome%2Bcore%2Bparticle%2Bat%2B2.8%2BA%2Bresolution.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F38444%26rft_id%253Dinfo%253Apmid%252F9305837%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/38444&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9305837&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F929.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XW77200038&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.929.23" data-doi="10.1016/S0959-440X(98)80007-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Luger</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richmond</span>  <span class="cit-name-given-names">T. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">DNA binding within the nucleosome core.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Struct. Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">33</span>–<span class="cit-lpage">40</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bstructural%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BStruct%2BBiol%26rft.aulast%253DLuger%26rft.auinit1%253DK.%26rft.volume%253D8%26rft.issue%253D1%26rft.spage%253D33%26rft.epage%253D40%26rft.atitle%253DDNA%2Bbinding%2Bwithin%2Bthe%2Bnucleosome%2Bcore.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0959-440X%252898%252980007-9%26rft_id%253Dinfo%253Apmid%252F9519294%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0959-440X(98)80007-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9519294&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F929.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000072384400005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.929.24" data-doi="10.1128/MCB.23.22.8323-8333.2003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mason</span>  <span class="cit-name-given-names">P. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Struhl</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The FACT complex travels with elongating RNA polymerase II and is important for the fidelity of transcriptional initiation in vivo.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">8323</span>–<span class="cit-lpage">8333</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DMason%26rft.auinit1%253DP.%2BB.%26rft.volume%253D23%26rft.issue%253D22%26rft.spage%253D8323%26rft.epage%253D8333%26rft.atitle%253DThe%2BFACT%2BComplex%2BTravels%2Bwith%2BElongating%2BRNA%2BPolymerase%2BII%2Band%2BIs%2BImportant%2Bfor%2Bthe%2BFidelity%2Bof%2BTranscriptional%2BInitiation%2BIn%2BVivo%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.23.22.8323-8333.2003%26rft_id%253Dinfo%253Apmid%252F14585989%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyMy8yMi84MzIzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy82LzkyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.929.25" data-doi="10.1038/nsmb.1903"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mayer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lidschreiber</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siebert</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leike</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soding</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Uniform transitions of the general RNA polymerase II transcription complex.</span> <abbr class="cit-jnl-abbrev">Nat. Struct. Mol. Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1272</span>–<span class="cit-lpage">1278</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bstructural%2B%2526%2Bmolecular%2Bbiology%26rft.stitle%253DNat%2BStruct%2BMol%2BBiol%26rft.aulast%253DMayer%26rft.auinit1%253DA.%26rft.volume%253D17%26rft.issue%253D10%26rft.spage%253D1272%26rft.epage%253D1278%26rft.atitle%253DUniform%2Btransitions%2Bof%2Bthe%2Bgeneral%2BRNA%2Bpolymerase%2BII%2Btranscription%2Bcomplex.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb.1903%26rft_id%253Dinfo%253Apmid%252F20818391%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsmb.1903&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20818391&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F929.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282563600021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.929.26" data-doi="10.1371/journal.pone.0020847"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">C. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berner</span>  <span class="cit-name-given-names">G. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holthoff</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martinez-Fonts</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harper</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Mutant versions of the S. cerevisiae transcription elongation factor Spt16 define regions of Spt16 that functionally interact with histone H3.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e20847</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DMyers%26rft.auinit1%253DC.%2BN.%26rft.volume%253D6%26rft.issue%253D6%26rft.spage%253De20847%26rft.epage%253De20847%26rft.atitle%253DMutant%2Bversions%2Bof%2Bthe%2BS.%2Bcerevisiae%2Btranscription%2Belongation%2Bfactor%2BSpt16%2Bdefine%2Bregions%2Bof%2BSpt16%2Bthat%2Bfunctionally%2Binteract%2Bwith%2Bhistone%2BH3.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0020847%26rft_id%253Dinfo%253Apmid%252F21673966%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0020847&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21673966&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F929.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=21673966&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.929.27" data-doi="10.1038/22350"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Orphanides</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">W. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lane</span>  <span class="cit-name-given-names">W. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hampsey</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reinberg</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">The chromatin-specific transcription elongation factor FACT comprises human SPT16 and SSRP1 proteins.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">400</span>: <span class="cit-fpage">284</span>–<span class="cit-lpage">288</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DOrphanides%26rft.auinit1%253DG.%26rft.volume%253D400%26rft.issue%253D6741%26rft.spage%253D284%26rft.epage%253D288%26rft.atitle%253DThe%2Bchromatin-specific%2Btranscription%2Belongation%2Bfactor%2BFACT%2Bcomprises%2Bhuman%2BSPT16%2Band%2BSSRP1%2Bproteins.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F22350%26rft_id%253Dinfo%253Apmid%252F10421373%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/22350&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10421373&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F929.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000081503800049&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.929.28" data-doi="10.1016/j.tig.2012.02.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Petesch</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lis</span>  <span class="cit-name-given-names">J. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Overcoming the nucleosome barrier during transcript elongation.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">285</span>–<span class="cit-lpage">294</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DPetesch%26rft.auinit1%253DS.%2BJ.%26rft.volume%253D28%26rft.issue%253D6%26rft.spage%253D285%26rft.epage%253D294%26rft.atitle%253DOvercoming%2Bthe%2Bnucleosome%2Bbarrier%2Bduring%2Btranscript%2Belongation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2012.02.005%26rft_id%253Dinfo%253Apmid%252F22465610%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2012.02.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22465610&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F929.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305094000005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.929.29" data-doi="10.1534/genetics.111.132266"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rando</span>  <span class="cit-name-given-names">O. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winston</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Chromatin and transcription in yeast.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">351</span>–<span class="cit-lpage">387</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DRando%26rft.auinit1%253DO.%2BJ.%26rft.volume%253D190%26rft.issue%253D2%26rft.spage%253D351%26rft.epage%253D387%26rft.atitle%253DChromatin%2Band%2BTranscription%2Bin%2BYeast%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.132266%26rft_id%253Dinfo%253Apmid%252F22345607%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzIvMzUxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy82LzkyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-book" id="cit-3.6.929.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rose</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winston</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hieter</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-source">Methods in Yeast Genetics: A Laboratory Course Manual</span>. <span class="cit-publ-name">Cold Spring Harbor Laboratory Press</span>, <span class="cit-publ-loc">Cold Spring Harbor, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.929.31" data-doi="10.1074/jbc.M307291200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ruone</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rhoades</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Formosa</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Multiple Nhp6 molecules are required to recruit Spt16-Pob3 to form yFACT complexes and to reorganize nucleosomes.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">278</span>: <span class="cit-fpage">45288</span>–<span class="cit-lpage">45295</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M307291200%26rft_id%253Dinfo%253Apmid%252F12952948%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzgvNDYvNDUyODgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzYvOTI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.929.32" data-doi="10.1128/MCB.24.23.10111-10117.2004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schwabish</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Struhl</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Evidence for eviction and rapid deposition of histones upon transcriptional elongation by RNA polymerase II.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">10111</span>–<span class="cit-lpage">10117</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DSchwabish%26rft.auinit1%253DM.%2BA.%26rft.volume%253D24%26rft.issue%253D23%26rft.spage%253D10111%26rft.epage%253D10117%26rft.atitle%253DEvidence%2Bfor%2BEviction%2Band%2BRapid%2BDeposition%2Bof%2BHistones%2Bupon%2BTranscriptional%2BElongation%2Bby%2BRNA%2BPolymerase%2BII%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.24.23.10111-10117.2004%26rft_id%253Dinfo%253Apmid%252F15542822%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjExOiIyNC8yMy8xMDExMSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNi85MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.929.33" data-doi="10.1146/annurev.biochem.78.062807.091425"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Selth</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sigurdsson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Svejstrup</span>  <span class="cit-name-given-names">J. Q.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Transcript elongation by RNA polymerase II.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Biochem.</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">271</span>–<span class="cit-lpage">293</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bbiochemistry%26rft.stitle%253DAnnu%2BRev%2BBiochem%26rft.aulast%253DSelth%26rft.auinit1%253DL.%2BA.%26rft.volume%253D79%26rft.spage%253D271%26rft.epage%253D293%26rft.atitle%253DTranscript%2BElongation%2Bby%2BRNA%2BPolymerase%2BII.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.biochem.78.062807.091425%26rft_id%253Dinfo%253Apmid%252F20367031%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.biochem.78.062807.091425&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20367031&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F929.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000280225300010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.929.34" data-doi="10.1073/pnas.0712293105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stuwe</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hothorn</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lejeune</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rybin</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bortfeld</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The FACT Spt16 “peptidase” domain is a histone H3–H4 binding module.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">8884</span>–<span class="cit-lpage">8889</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0712293105%26rft_id%253Dinfo%253Apmid%252F18579787%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA1LzI2Lzg4ODQiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzYvOTI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.929.35" data-doi="10.1074/jbc.M708682200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanDemark</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xin</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCullough</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rawlins</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bentley</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Structural and functional analysis of the Spt16p N-terminal domain reveals overlapping roles of yFACT subunits.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">283</span>: <span class="cit-fpage">5058</span>–<span class="cit-lpage">5068</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M708682200%26rft_id%253Dinfo%253Apmid%252F18089575%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEwOiIyODMvOC81MDU4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy82LzkyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.929.36" data-doi="10.1093/emboj/20.18.5207"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suto</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luger</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Structure of the yeast nucleosome core particle reveals fundamental changes in internucleosome interactions.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">5207</span>–<span class="cit-lpage">5218</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJournal%26rft.aulast%253DWhite%26rft.auinit1%253DC.%2BL.%26rft.volume%253D20%26rft.issue%253D18%26rft.spage%253D5207%26rft.epage%253D5218%26rft.atitle%253DStructure%2Bof%2Bthe%2Byeast%2Bnucleosome%2Bcore%2Bparticle%2Breveals%2Bfundamental%2Bchanges%2Bin%2Binternucleosome%2Binteractions%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252F20.18.5207%26rft_id%253Dinfo%253Apmid%252F11566884%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMjAvMTgvNTIwNyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNi85MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.929.37" data-doi="10.1074/jbc.R110.180778"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Winkler</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luger</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The histone chaperone FACT: structural insights and mechanisms for nucleosome reorganization.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">286</span>: <span class="cit-fpage">18369</span>–<span class="cit-lpage">18374</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.R110.180778%26rft_id%253Dinfo%253Apmid%252F21454601%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODYvMjEvMTgzNjkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzYvOTI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.929.38" data-doi="10.1074/jbc.M111.301465"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Winkler</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muthurajan</span>  <span class="cit-name-given-names">U. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hieb</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luger</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Histone chaperone FACT coordinates nucleosome interaction through multiple synergistic binding events.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">286</span>: <span class="cit-fpage">41883</span>–<span class="cit-lpage">41892</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M111.301465%26rft_id%253Dinfo%253Apmid%252F21969370%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODYvNDgvNDE4ODMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzYvOTI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-book" id="cit-3.6.929.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Winston</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Analysis of SPT genes: a genetic approach toward analysis of TFIID, histones, and other transcription factors in yeast</span>, pp. <span class="cit-fpage">1271</span>–<span class="cit-lpage">1293</span> in <span class="cit-source">Transcription Regulation</span><span class="cit-comment">, edited by S. L. McKnight, and K. R. Yamamoto.</span> <span class="cit-publ-name">Cold Spring Harbor Laboratory Press</span>, <span class="cit-publ-loc">Cold Spring Harbor, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.929.40" data-doi="10.1002/yea.320110107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Winston</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dollard</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ricupero-Hovasse</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Construction of a set of convenient Saccharomyces cerevisiae strains that are isogenic to S288C.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">53</span>–<span class="cit-lpage">55</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DWinston%26rft.auinit1%253DF.%26rft.volume%253D11%26rft.issue%253D1%26rft.spage%253D53%26rft.epage%253D55%26rft.atitle%253DConstruction%2Bof%2Ba%2Bset%2Bof%2Bconvenient%2BSaccharomyces%2Bcerevisiae%2Bstrains%2Bthat%2Bare%2Bisogenic%2Bto%2BS288C.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.320110107%26rft_id%253Dinfo%253Apmid%252F7762301%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.320110107&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7762301&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F929.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995QF61400006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.929.41" data-doi="10.1016/j.molcel.2009.06.024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xin</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takahata</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blanksma</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCullough</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stillman</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">yFACT induces global accessibility of nucleosomal DNA without H2A–H2B displacement.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">365</span>–<span class="cit-lpage">376</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DXin%26rft.auinit1%253DH.%26rft.volume%253D35%26rft.issue%253D3%26rft.spage%253D365%26rft.epage%253D376%26rft.atitle%253DyFACT%2Binduces%2Bglobal%2Baccessibility%2Bof%2Bnucleosomal%2BDNA%2Bwithout%2BH2A-H2B%2Bdisplacement.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2009.06.024%26rft_id%253Dinfo%253Apmid%252F19683499%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2009.06.024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19683499&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F929.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269278900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/3/6/929.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/6/927" title="Editorial: HeLa Cell Sequela" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/6/941" title="Resistance to Germline RNA Interference in a Caenorhabditis elegans Wild Isolate Exhibits Complexity and Nonadditivity" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/3/6"><span>Volume 3 Issue 6, June 2013</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/3/6.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/3/6.cover-source.jpg?itok=CJf4vROV" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 3 (6)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/6/929" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-4y6NfxbnVURzeMyUpYCI6llHjSSs0GCMHdOjHzWIfhs" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/3/6/929" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/3/6/929" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/3/6/929" class="active" data-icon-position="" data-hide-link-title="0">A Nucleosomal Region Important for Ensuring Proper Interactions Between the Transcription Elongation Factor Spt16 and Transcribed Genes in Saccharomyces cerevisiae</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459478" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-QRIfg4szMNaeWaFpatG3kAnVeJxs3qx3rsy54MNRLDg" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/6/929.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/3/6/929" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-VXT1Q-HwGyWPhcuDGcMlBe2oQo8PkIwxD5r_DHYcXOU" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/3/6/929" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/6/929.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459478" id="node459478--2" data-pisa="ggg;3/6/929" data-pisa-master="ggg;g3.113.005926" data-apath="/ggg/3/6/929.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Nucleosomal Region Important for Ensuring Proper Interactions Between the Transcription Elongation Factor Spt16 and Transcribed Genes in <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Hoai-Trang T. Nguyen</span>, <span class="highwire-citation-author" data-delta="1">William Wharton</span>, <span class="highwire-citation-author" data-delta="2">Jennifer A. Harper</span>, <span class="highwire-citation-author" data-delta="3">James R. Dornhoffer</span> and <span class="highwire-citation-author" data-delta="4">Andrea A. Duina</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">929-940;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.005926 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459478--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Hoai-Trang T. Nguyen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biology Department, Hendrix College, Conway, Arkansas 72032</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hoai-Trang%2BT.%2BNguyen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nguyen%20HT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHoai-Trang%2BT.%2BNguyen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">William Wharton <span class="nlm-suffix">II</span></div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biology Department, Hendrix College, Conway, Arkansas 72032</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BWharton%2B%3Cspan%2Bclass%3D%22nlm-suffix%22%3EII%3C/span%3E" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wharton%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWilliam%2BWharton%2BII" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jennifer A. Harper </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biology Department, Hendrix College, Conway, Arkansas 72032</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jennifer%2BA.%2BHarper%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Harper%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJennifer%2BA.%2BHarper%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">James R. Dornhoffer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biology Department, Hendrix College, Conway, Arkansas 72032</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BR.%2BDornhoffer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dornhoffer%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BR.%2BDornhoffer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Andrea A. Duina </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biology Department, Hendrix College, Conway, Arkansas 72032</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrea%2BA.%2BDuina%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Duina%20AA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrea%2BA.%2BDuina%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:duina@hendrix.edu" class="" data-icon-position="" data-hide-link-title="0">duina@hendrix.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/3/6/929" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/3/6/929&amp;title=A%20Nucleosomal%20Region%20Important%20for%20Ensuring%20Proper%20Interactions%20Between%20the%20Transcription%20Elongation%20Factor%20Spt16%20and%20Transcribed%20Genes%20in%20Saccharomyces%20cerevisiae" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/3/6/929&amp;title=A%20Nucleosomal%20Region%20Important%20for%20Ensuring%20Proper%20Interactions%20Between%20the%20Transcription%20Elongation%20Factor%20Spt16%20and%20Transcribed%20Genes%20in%20Saccharomyces%20cerevisiae" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/3/6/929&amp;title=A%20Nucleosomal%20Region%20Important%20for%20Ensuring%20Proper%20Interactions%20Between%20the%20Transcription%20Elongation%20Factor%20Spt16%20and%20Transcribed%20Genes%20in%20Saccharomyces%20cerevisiae" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/3/6/929&amp;text=A%20Nucleosomal%20Region%20Important%20for%20Ensuring%20Proper%20Interactions%20Between%20the%20Transcription%20Elongation%20Factor%20Spt16%20and%20Transcribed%20Genes%20in%20Saccharomyces%20cerevisiae" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/3/6/929&amp;title=A%20Nucleosomal%20Region%20Important%20for%20Ensuring%20Proper%20Interactions%20Between%20the%20Transcription%20Elongation%20Factor%20Spt16%20and%20Transcribed%20Genes%20in%20Saccharomyces%20cerevisiae" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/3/6/929&amp;t=A%20Nucleosomal%20Region%20Important%20for%20Ensuring%20Proper%20Interactions%20Between%20the%20Transcription%20Elongation%20Factor%20Spt16%20and%20Transcribed%20Genes%20in%20Saccharomyces%20cerevisiae" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/3/6/929&amp;title=A%20Nucleosomal%20Region%20Important%20for%20Ensuring%20Proper%20Interactions%20Between%20the%20Transcription%20Elongation%20Factor%20Spt16%20and%20Transcribed%20Genes%20in%20Saccharomyces%20cerevisiae" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/3/6/929&amp;title=A%20Nucleosomal%20Region%20Important%20for%20Ensuring%20Proper%20Interactions%20Between%20the%20Transcription%20Elongation%20Factor%20Spt16%20and%20Transcribed%20Genes%20in%20Saccharomyces%20cerevisiae" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459478/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459478" id="node459478--3" data-pisa="ggg;3/6/929" data-pisa-master="ggg;g3.113.005926" data-apath="/ggg/3/6/929.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Nucleosomal Region Important for Ensuring Proper Interactions Between the Transcription Elongation Factor Spt16 and Transcribed Genes in <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Hoai-Trang T. Nguyen</span>, <span class="highwire-citation-author" data-delta="1">William Wharton</span>, <span class="highwire-citation-author" data-delta="2">Jennifer A. Harper</span>, <span class="highwire-citation-author" data-delta="3">James R. Dornhoffer</span> and <span class="highwire-citation-author" data-delta="4">Andrea A. Duina</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">929-940;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.005926 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459478--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Hoai-Trang T. Nguyen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biology Department, Hendrix College, Conway, Arkansas 72032</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hoai-Trang%2BT.%2BNguyen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nguyen%20HT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHoai-Trang%2BT.%2BNguyen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">William Wharton <span class="nlm-suffix">II</span></div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biology Department, Hendrix College, Conway, Arkansas 72032</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BWharton%2B%3Cspan%2Bclass%3D%22nlm-suffix%22%3EII%3C/span%3E" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wharton%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWilliam%2BWharton%2BII" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jennifer A. Harper </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biology Department, Hendrix College, Conway, Arkansas 72032</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jennifer%2BA.%2BHarper%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Harper%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJennifer%2BA.%2BHarper%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">James R. Dornhoffer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biology Department, Hendrix College, Conway, Arkansas 72032</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BR.%2BDornhoffer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dornhoffer%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BR.%2BDornhoffer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Andrea A. Duina </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biology Department, Hendrix College, Conway, Arkansas 72032</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrea%2BA.%2BDuina%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Duina%20AA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrea%2BA.%2BDuina%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:duina@hendrix.edu" class="" data-icon-position="" data-hide-link-title="0">duina@hendrix.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=929&amp;rft.epage=940&amp;rft.atitle=A%20Nucleosomal%20Region%20Important%20for%20Ensuring%20Proper%20Interactions%20Between%20the%20Transcription%20Elongation%20Factor%20Spt16%20and%20Transcribed%20Genes%20in%20Saccharomyces%20cerevisiae&amp;rft.volume=3&amp;rft.issue=6&amp;rft.date=2013-06-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Nguyen%2C+Hoai-Trang+T.&amp;rft.au=Wharton%2C+William&amp;rft.au=Harper%2C+Jennifer+A.&amp;rft.au=Dornhoffer%2C+James+R.&amp;rft.au=Duina%2C+Andrea+A."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459478/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459478/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459478/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459478/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459478/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459478/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459478/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459478/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459478/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459478/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459478/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459478/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/3/6/929" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-6" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/3/6/929.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/3/6/929.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/3/6/929.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
