<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/3/6/993" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/3/6/993.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/3/6/993.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/3/6/993.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/6.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/3/6/993/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/3/6/993.atom" />
<meta name="HW.pisa" content="ggg;3/6/993" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Widespread Selection Across Coding and Noncoding DNA in the Pea Aphid Genome" />
<meta name="DC.Identifier" content="10.1534/g3.113.005793" />
<meta name="DC.Date" content="2013-06-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2013 Bickel et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genome-wide patterns of diversity and selection are critical measures for understanding how evolution has shaped the genome. Yet, these population genomic estimates are available for only a limited number of model organisms. Here we focus on the population genomics of the pea aphid ( Acyrthosiphon pisum ). The pea aphid is an emerging model system that exhibits a range of intriguing biological traits not present in classic model systems. We performed low-coverage genome resequencing of 21 clonal pea aphid lines collected from alfalfa host plants in North America to characterize genome-wide patterns of diversity and selection. We observed an excess of low-frequency polymorphisms throughout coding and noncoding DNA, which we suggest is the result of a founding event and subsequent population expansion in North America. Most gene regions showed lower levels of Tajima’s D than synonymous sites, suggesting that the majority of the genome is not evolving neutrally but rather exhibits significant constraint. Furthermore, we used the pea aphid’s unique manner of X-chromosome inheritance to assign genomic scaffolds to either autosomes or the X chromosome. Comparing autosomal vs. X-linked sequence variation, we discovered that autosomal genes show an excess of low frequency variants indicating that purifying selection acts more efficiently on the X chromosome. Overall, our results provide a critical first step in characterizing the genetic diversity and evolutionary pressures on an aphid genome." />
<meta name="DC.Contributor" content="Ryan D. Bickel" />
<meta name="DC.Contributor" content="Joseph P. Dunham" />
<meta name="DC.Contributor" content="Jennifer A. Brisson" />
<meta name="article:published_time" content="2013-06-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Widespread Selection Across Coding and Noncoding DNA in the Pea Aphid Genome" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genome-wide patterns of diversity and selection are critical measures for understanding how evolution has shaped the genome. Yet, these population genomic estimates are available for only a limited number of model organisms. Here we focus on the population genomics of the pea aphid (&lt;i&gt;Acyrthosiphon pisum&lt;/i&gt;). The pea aphid is an emerging model system that exhibits a range of intriguing biological traits not present in classic model systems. We performed low-coverage genome resequencing of 21 clonal pea aphid lines collected from alfalfa host plants in North America to characterize genome-wide patterns of diversity and selection. We observed an excess of low-frequency polymorphisms throughout coding and noncoding DNA, which we suggest is the result of a founding event and subsequent population expansion in North America. Most gene regions showed lower levels of Tajima’s D than synonymous sites, suggesting that the majority of the genome is not evolving neutrally but rather exhibits significant constraint. Furthermore, we used the pea aphid’s unique manner of X-chromosome inheritance to assign genomic scaffolds to either autosomes or the X chromosome. Comparing autosomal &lt;i&gt;vs.&lt;/i&gt; X-linked sequence variation, we discovered that autosomal genes show an excess of low frequency variants indicating that purifying selection acts more efficiently on the X chromosome. Overall, our results provide a critical first step in characterizing the genetic diversity and evolutionary pressures on an aphid genome.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2013/06/01" />
<meta name="citation_mjid" content="ggg;3/6/993" />
<meta name="citation_id" content="3/6/993" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/3/6/993" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/3/6/993.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/3/6/993.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/3/6/993.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.113.005793" />
<meta name="citation_pmid" content="23589520" />
<meta name="citation_volume" content="3" />
<meta name="citation_issue" content="6" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="993" />
<meta name="citation_lastpage" content="1001" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Ryan D. Bickel" />
<meta name="citation_author_institution" content="University of Nebraska, School of Biological Sciences, Lincoln, Nebraska 68588" />
<meta name="citation_author_email" content="rbickel2@unl.edu" />
<meta name="citation_author" content="Joseph P. Dunham" />
<meta name="citation_author_institution" content="University of Southern California, Molecular &amp; Computational Biology, Los Angeles, California 90089" />
<meta name="citation_author" content="Jennifer A. Brisson" />
<meta name="citation_author_institution" content="University of Nebraska, School of Biological Sciences, Lincoln, Nebraska 68588" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. Achaz;citation_title=Testing for neutrality in samples with sequencing errors.;citation_volume=179;citation_year=2008;citation_pmid=18562660;citation_doi=10.1534/genetics.107.082198" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=P. Andolfatto;citation_title=Adaptive evolution of non-coding DNA in Drosophila.;citation_pages=1149-1152;citation_volume=437;citation_year=2005;citation_issue=7062;citation_pmid=16237443;citation_doi=10.1038/nature04107" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=D. J. Begun;citation_author=A. K. Holloway;citation_author=K. Stevens;citation_author=L. W. Hilllier;citation_author=Y. Poh;citation_title=Population genomics: whole-genome analysis of polymorphism and divergence in Drosophila simulans.;citation_pages=e310-e310;citation_volume=5;citation_year=2007;citation_issue=11;citation_pmid=17988176;citation_doi=10.1371/journal.pbio.0050310" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=B. E. Bernstein;citation_author=E. Birney;citation_author=I. Dunham;citation_author=E. D. Green;citation_author=C. Gunter;citation_title=An integrated encyclopedia of DNA elements in the human genome.;citation_pages=57-74;citation_volume=489;citation_year=2012;citation_issue=7414;citation_pmid=22955616;citation_doi=10.1038/nature11247" />
<meta name="citation_reference" content="citation_journal_title=Genetica;citation_journal_abbrev=Genetica;citation_author=D. Bizzaro;citation_author=M. Mandrioli;citation_author=M. Zanotti;citation_author=M. Giusti;citation_author=G. C. Manicardi;citation_title=Chromosome analysis and molecular characterization of highly repeated DNAs in the aphid Acyrthosiphon pisum (Aphididae, Hemiptera).;citation_pages=197-202;citation_volume=108;citation_year=2000;citation_issue=2;citation_pmid=11138948;citation_doi=10.1023/A:1004128314728" />
<meta name="citation_reference" content="citation_title=Aphids: Their Biology, Natural Enemies &amp; Control;citation_title=Reproduction, cytogenetics and development;citation_author=R. L. Blackman;citation_pages=163-195;citation_year=1987" />
<meta name="citation_reference" content="citation_title=Aphids on the World’s Crops: An Identification and Information Guide;citation_author=R. L. Blackman;citation_author=V. F. Eastop;citation_year=2000" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=C. Braendle;citation_author=M. C. Caillaud;citation_author=D. L. Stern;citation_title=Genetic mapping of aphicarus -- a sex-linked locus controlling a wing polymorphism in the pea aphid (Acyrthosiphon pisum).;citation_pages=435-442;citation_volume=94;citation_year=2005;citation_issue=4;citation_pmid=15674387;citation_doi=10.1038/sj.hdy.6800633" />
<meta name="citation_reference" content="citation_journal_title=BioEssays :  news and reviews in molecular, cellular and developmental biology;citation_journal_abbrev=Bioessays;citation_author=J. A. Brisson;citation_author=D. L. Stern;citation_title=The pea aphid, Acyrthosiphon pisum: an emerging genomic model system for ecological, developmental and evolutionary studies.;citation_pages=747-755;citation_volume=28;citation_year=2006;citation_issue=7;citation_pmid=16850403;citation_doi=10.1002/bies.20436" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=J. A. Brisson;citation_author=D. L. Stern;citation_author=S. V. Nuzhdin;citation_title=Similar patterns of linkage disequilibrium and nucleotide diversity in native and introduced populations of the pea aphid, Acyrthosiphon pisum.;citation_pages=22-22;citation_volume=10;citation_year=2009;citation_pmid=19470181;citation_doi=10.1186/1471-2156-10-22" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=M. C. Caillaud;citation_author=M. Boutin;citation_author=C. Braendle;citation_author=J.-C. Simon;citation_title=A sex-linked locus controls wing polymorphism in males of the pea aphid, Acyrthosiphon pisum (Harris).;citation_pages=346-352;citation_volume=89;citation_year=2002;citation_issue=5;citation_pmid=12399992;citation_doi=10.1038/sj.hdy.6800146" />
<meta name="citation_reference" content="citation_journal_title=Am. Nat.;citation_author=B. Charlesworth;citation_author=J. A. Coyne;citation_author=N. H. Barton;citation_title=The relative rates of evolution of sex-chromosomes and autosomes.;citation_pages=113-146;citation_volume=130;citation_year=1987;citation_doi=10.1086/284701" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Conesa;citation_author=S. Gotz;citation_author=J. M. Garcia-Gomez;citation_author=J. Terol;citation_author=M. Talon;citation_title=Blast2Go: a universal tool for annotation, visualization and analysis in functional genomics research.;citation_volume=21;citation_year=2005;citation_pmid=16081474;citation_doi=10.1093/bioinformatics/bti610" />
<meta name="citation_reference" content="citation_title=Bull. US Dept. Agric.;citation_title=The pea aphid with relation to forage crops.;citation_author=J. J. Davis;citation_pages=276;citation_year=1915" />
<meta name="citation_reference" content="citation_journal_title=BMC evolutionary biology [electronic resource];citation_journal_abbrev=BMC Evol Biol;citation_author=L. J. DeRose-Wilson;citation_author=B. S. Gaut;citation_title=Transcription-related mutations and GC content drive variation in nucleotide substitution rates across the genomes of Arabidopsis thaliana and Arabidopsis lyrata.;citation_pages=66-66;citation_volume=7;citation_year=2007;citation_pmid=17451608;citation_doi=10.1186/1471-2148-7-66" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=H. Ellegren;citation_title=The different levels of genetic diversity in sex chromosomes and autosomes.;citation_pages=278-284;citation_volume=25;citation_year=2009;citation_issue=6;citation_pmid=19481288;citation_doi=10.1016/j.tig.2009.04.005" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=J. Ferrari;citation_author=S. Via;citation_author=C. J. Godfray;citation_title=Population differentiation and genetic variation in performance on eight hosts in the pea aphid complex.;citation_pages=2508-2524;citation_volume=62;citation_year=2008;citation_issue=10;citation_pmid=18647340;citation_doi=10.1111/j.1558-5646.2008.00468.x" />
<meta name="citation_reference" content="citation_journal_title=Proceedings of the Royal Society B: Biological Sciences;citation_journal_abbrev=Proc R Soc B;citation_author=A. Frantz;citation_author=M. Plantegenest;citation_author=J.-C. Simon;citation_title=Temporal habitat variability and the maintenance of sex in host populations of the pea aphid;citation_pages=2887-2891;citation_volume=273;citation_year=2006;citation_issue=1603;citation_pmid=17015368;citation_doi=10.1098/rspb.2006.3665" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. R. Hudson;citation_author=M. Slatkin;citation_author=W. P. Maddison;citation_title=Estimation of levels of gene flow from DNA sequence data.;citation_pages=583-589;citation_volume=132;citation_year=1992;citation_issue=2;citation_pmid=1427045;citation_doi=10.1371/journal.pgen.1005397" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_title=Genome sequence of the pea aphid Acyrthosiphon pisum.;citation_pages=e1000313-e1000313;citation_volume=8;citation_year=2010;citation_issue=2;citation_pmid=20186266;citation_doi=10.1371/journal.pbio.1000313" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=J. Jaquiery;citation_author=S. Stoeckel;citation_author=C. Rispe;citation_author=L. Mieuzet;citation_author=F. Legeai;citation_title=Accelerated evolution of sex chromosomes in aphids, an X0 System.;citation_volume=29;citation_year=2012;citation_pmid=21998277;citation_doi=10.1093/molbev/msr252" />
<meta name="citation_reference" content="citation_journal_title=Proceedings of the Royal Society B: Biological Sciences;citation_journal_abbrev=Proc R Soc B;citation_author=T. E. Leonardo;citation_author=G. T. Muiru;citation_title=Facultative symbionts are associated with host plant specialization in pea aphid populations;citation_pages=S209-S212;citation_volume=270;citation_year=2003;citation_issue=Suppl_2;citation_pmid=14667385;citation_doi=10.1098/rsbl.2003.0064" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate long-read alignment with Burrows-Wheeler Transform.;citation_volume=26;citation_year=2010;citation_pmid=20080505;citation_doi=10.1093/bioinformatics/btp698" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=H. Li;citation_author=W. Stephan;citation_title=Inferring the demographic history and rate of adaptive substitution in Drosophila.;citation_pages=e166-e166;citation_volume=2;citation_year=2006;citation_issue=10;citation_pmid=17040129;citation_doi=10.1371/journal.pgen.0020166" />
<meta name="citation_reference" content="citation_journal_title=Chrom. Res.;citation_author=M. Mandrioli;citation_author=F. Borsatti;citation_title=Analysis of heterochromatic epigenetic markers in the holocentric chromosomes of the aphid Acyrthosiphon pisum.;citation_pages=1015-1022;citation_volume=15;citation_year=2007;citation_pmid=18004669;citation_doi=10.1007/s10577-007-1176-4" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=M. Mandrioli;citation_author=D. Bizzaro;citation_author=M. Giusti;citation_author=G. C. Manicardi;citation_author=U. Bianchi;citation_title=The role of rDNA genes in X chromosome association in the aphid Acyrthosiphon pisum.;citation_pages=381-386;citation_volume=42;citation_year=1999;citation_issue=3;citation_pmid=10382287" />
<meta name="citation_reference" content="citation_journal_title=Annual review of entomology;citation_journal_abbrev=Annu Rev Entomol;citation_author=N. A. Moran;citation_title=The Evolution Of Aphid Life Cycles;citation_pages=321-348;citation_volume=37;citation_year=1992;citation_issue=1;citation_doi=10.1146/annurev.en.37.010192.001541" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Nei;citation_author=F. Tajima;citation_title=DNA polymorphism detectable by restriction endonucleases.;citation_pages=711-716;citation_volume=146;citation_year=1981" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=W. R. Rice;citation_title=Sex chromosomes and the evolution of sexual dimorphism.;citation_pages=735-742;citation_volume=38;citation_year=1984;citation_doi=10.2307/2408385" />
<meta name="citation_reference" content="citation_title=Aphids, Their Biology, Natural Enemies, and Control;citation_title=Dispersion and migration;citation_author=Y. Robert;citation_pages=299-313;citation_year=1987" />
<meta name="citation_reference" content="citation_journal_title=Delaware Agric. Exp. Station Bull.;citation_author=E. D. Sanderson;citation_title=The destructive pea louse in Delaware.;citation_pages=14-24;citation_volume=49;citation_year=1900" />
<meta name="citation_reference" content="citation_journal_title=Proceedings of the Royal Society B: Biological Sciences;citation_journal_abbrev=Proc R Soc B;citation_author=J. C. Simon;citation_author=S. Carre;citation_author=M. Boutin;citation_author=N. Prunier-Leterme;citation_author=B. Sabater-Munoz;citation_title=Host-based divergence in populations of the pea aphid: insights from nuclear markers and the prevalence of facultative symbionts;citation_pages=1703-1712;citation_volume=270;citation_year=2003;citation_issue=1525;citation_pmid=12964998;citation_doi=10.1098/rspb.2003.2430" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=F. Tajima;citation_title=EVOLUTIONARY RELATIONSHIP OF DNA SEQUENCES IN FINITE POPULATIONS;citation_pages=437-460;citation_volume=105;citation_year=1983;citation_issue=2;citation_pmid=6628982" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=F. Tajima;citation_title=Statistical method for testing the neutral mutation hypothesis by DNA polymorphism.;citation_pages=585-595;citation_volume=123;citation_year=1989;citation_issue=3;citation_pmid=2513255" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=K. Thornton;citation_title=libsequence: a C++ class library for evolutionary genetic analysis;citation_pages=2325-2327;citation_volume=19;citation_year=2003;citation_issue=17;citation_pmid=14630667;citation_doi=10.1093/bioinformatics/btg316" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=S. Via;citation_title=Reproductive isolation between sympatric races of pea aphids. I. Gene flow restriction and habitat choice.;citation_pages=1446-1457;citation_volume=53;citation_year=1999;citation_doi=10.2307/2640891" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=S. Via;citation_author=A. C. Bouck;citation_author=S. Skillman;citation_title=Reproductive isolation between divergent races of pea aphids on two hosts. II. Selection against migrants and hybrids in the parental environments.;citation_pages=1626-1637;citation_volume=54;citation_year=2000;citation_issue=5;citation_pmid=11108590;citation_doi=10.1554/0014-3820(2000)054[1626:RIBDRO]2.0.CO;2" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=B. Vicoso;citation_author=B. Charlesworth;citation_title=Evolution on the X chromosome: unusual patterns and processes.;citation_pages=645-653;citation_volume=7;citation_year=2006;citation_issue=8;citation_pmid=16847464;citation_doi=10.1038/nrg1914" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=G. A. Watterson;citation_title=On the number of segregating sites in genetical models without recombination.;citation_pages=256-276;citation_volume=7;citation_year=1975;citation_issue=2;citation_pmid=1145509;citation_doi=10.1016/0040-5809(75)90020-9" />
<meta name="citation_reference" content="citation_journal_title=Genet. Res.;citation_author=A. C. C. Wilson;citation_author=P. Sunnucks;citation_author=D. F. Hales;citation_title=Random loss of X chromosome at male determination in an aphid, Sitobion near fragariae, detected using an X-linked polymorphic microsatellite marker.;citation_pages=233-236;citation_volume=69;citation_year=1997;citation_doi=10.1017/S0016672397002747" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Widespread Selection Across Coding and Noncoding DNA in the Pea Aphid Genome" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/6.cover-source.jpg" />
<meta name="twitter:description" content="Genome-wide patterns of diversity and selection are critical measures for understanding how evolution has shaped the genome. Yet, these population genomic estimates are available for only a limited number of model organisms. Here we focus on the population genomics of the pea aphid ( Acyrthosiphon pisum ). The pea aphid is an emerging model system that exhibits a range of intriguing biological traits not present in classic model systems. We performed low-coverage genome resequencing of 21 clonal pea aphid lines collected from alfalfa host plants in North America to characterize genome-wide patterns of diversity and selection. We observed an excess of low-frequency polymorphisms throughout coding and noncoding DNA, which we suggest is the result of a founding event and subsequent population expansion in North America. Most gene regions showed lower levels of Tajima’s D than synonymous sites, suggesting that the majority of the genome is not evolving neutrally but rather exhibits significant constraint. Furthermore, we used the pea aphid’s unique manner of X-chromosome inheritance to assign genomic scaffolds to either autosomes or the X chromosome. Comparing autosomal vs. X-linked sequence variation, we discovered that autosomal genes show an excess of low frequency variants indicating that purifying selection acts more efficiently on the X chromosome. Overall, our results provide a critical first step in characterizing the genetic diversity and evolutionary pressures on an aphid genome." />
<meta name="og-title" property="og:title" content="Widespread Selection Across Coding and Noncoding DNA in the Pea Aphid Genome" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/3/6/993" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genome-wide patterns of diversity and selection are critical measures for understanding how evolution has shaped the genome. Yet, these population genomic estimates are available for only a limited number of model organisms. Here we focus on the population genomics of the pea aphid ( Acyrthosiphon pisum ). The pea aphid is an emerging model system that exhibits a range of intriguing biological traits not present in classic model systems. We performed low-coverage genome resequencing of 21 clonal pea aphid lines collected from alfalfa host plants in North America to characterize genome-wide patterns of diversity and selection. We observed an excess of low-frequency polymorphisms throughout coding and noncoding DNA, which we suggest is the result of a founding event and subsequent population expansion in North America. Most gene regions showed lower levels of Tajima’s D than synonymous sites, suggesting that the majority of the genome is not evolving neutrally but rather exhibits significant constraint. Furthermore, we used the pea aphid’s unique manner of X-chromosome inheritance to assign genomic scaffolds to either autosomes or the X chromosome. Comparing autosomal vs. X-linked sequence variation, we discovered that autosomal genes show an excess of low frequency variants indicating that purifying selection acts more efficiently on the X chromosome. Overall, our results provide a critical first step in characterizing the genetic diversity and evolutionary pressures on an aphid genome." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/6.cover-source.jpg" />
    <title>Widespread Selection Across Coding and Noncoding DNA in the Pea Aphid Genome | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"KEWMupz5T8RQzCVlqDvxiafeK9sa-X7JMnYrjZh24ow","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459481","apath":"\/ggg\/3\/6\/993.atom","pisa":"ggg;3\/6\/993","ac":{"\/ggg\/3\/6\/993.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/3\/6\/993.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;3\/6\/993"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;3\/6\/993"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/3\/6\/993.abstract":true,"\/content\/3\/6\/993":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022zjfKO68bzpS2VCENyDzKXas7QkQPIwIk+690gAl58xNFZ1sRUW8ksVuzXTcGbj3Iw6tPUpbFFXWpdGecwt\\\\\\\/8ab9PLhq6jwXY63x7nK5YY\\\\\\\/4pWAuhGjL+UclzfBGXPy9i1Bri46ALbpZthVOFEP2EbQGZOrIjeFynuQtZXPemkEkK8+v3LjInrkR7EJJTuEV+LTF6MF6FArnbiIeC7RNbXs\\\\\\\/lMvqPsQsqJdDu66+72kZKOgnF4ES52QFAMxDgFRd3a8hceghJs+Ha4aEps0kRi\\\\\\\/tdp38rhvHDnQKWmrlYH0WIhAmiSDRTI9\\\\\\\/PuSJM\\\\\\\/jsZHgOjxQZzvno\\\\\\\/Qh24hcU2B+dHt5wUV5Mu0fMQESNtMX9ueMFgx6arL6ZmsI9E9rf8nlzBR9OPlJajJTW4dsOnRXzjqws0s5ELZnEm\\\\\\\/NTZGatYPruc\\\\\\\/Xg+fgNCVDPnwYgeir5p8Zq+9k6l9\\\\\\\/Xc8E7gOHlL438FVZ+BzuJIuK6bbeuJPCzou86ktzUo0+hhfNk9fbHDrPKVBuqyiDxPTBKTtpGbXRwRjHKo72KnJDZr91nqlLD07a1VYIMt7L+a4gng0N0gmrehcj3bdqgOeNq0KAfmMv6ehlrauyZvbCoDLt\\\\\\\/NOtISynvweb3EHuSr22cuNeCksrslFM6tE+UR0lSyt2fFxGFr074e6WP+yt74XS9t9fuLqtZisYuTFd8QqYa6O3e3I5Zxv8a2ZafMVqGo+iPMcojiqEWUHRDpFlvlDY9gM8Xj+iWsg5FMUBL0561F8IqVNXQSPxqkGa9zbcp19A+0gL\\\\\\\/XEMEEJG2XOqASoCLAO8pWFWeQSH0hOOlwQnxJxMZSv\\\\\\\/\\\\\\\/Hswm\\\\\\\/RugZDIoaG\\\\\\\/Di95CMka2lnhaA95XKUXPfVYAuH79VZQ6pIfBVFyx8JYKlS\\\\\\\/07+OIjz5CnEBq+V2AarIPwkYALH9sqSO6Rp7xpq0Iv3836xSGOqVnC5bUyydjnczzARJLdoz66UtyAif8vPJUV+rwHNQfW6k0tSMfDKhZGIxoEuUuFeQPVDAcXC8RSaUs8ByCUIJlrGndb9QWbLG3Gdm5QPFgHzjDr9+vuSDBhjtNYU8Jl7Sy3UXBwmetRvNUpht77EBlNMgPoUvPXxiQnCz4j2dZ2vCBo762r0ZE2qqK+mAzUNUGTt6fb3h6RRkXW+SaxsXqFgZ77o2SLnVDjXxom1k10RxAq9AeOEpO\\\\\\\/fMhIklpvd5InVpnDsdJTCPw2icv5MS+D7z0rejNae4nDkav4D\\\\\\\/dbATarXUz3jrAsoFSao18XrAfv0fGO\\\u0022,\\\u0022iv\\\u0022:\\\u0022W0E52gZ0tTg0ziMv09Tk1w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c532c203dae1c5a3c514a53bf3ad541043ec8d399c893bd669fbcaa31bcaccad\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022f76XzUZCjhsGbf3lPrpk3QUYQBOvMDY9m\\\\\\\/4Tlfkxgdir286DjJbdBWTHGa8l0PFDxIu\\\\\\\/JsQQ2vPajgQ6ucECG9zTC4B0z5GMqPFnZ\\\\\\\/wqjobY8DdT0dCkjKCvkrTqacJZJSvDsHmjrWX37SzKoqWreSzhlsYicPk3gRB0Pl\\\\\\\/hbvvSw87NkN3Fozw1Mt0EhldlENqa0gZNmaIXGLo7BV2UJZEbr30MOLGe6waIUhk+o9xw1VpRvspnyrsEvaJShxNjuHTq6q0H34am6y1sdz\\\\\\\/NPZqBaYq1rCwTEr5A3g\\\\\\\/cyd3aS+DhdVTf7bqqCPDPeaG4lOkHpeNTzzo3Jp9CYlNc3ZL9foq6EkaoRSeuK93h0stXDRW6BGmIurT5KzJKXzXHQlJggQnFz5gsgQbxvgw05AiIGlJvdVVi0HxNPJ1dVkw6g3+EUgoXPdde8JeNJ+IEL\\\\\\\/vjpS3Zrt+pGWFBGOvDv3\\\\\\\/HmJTe48QCCmiINozLwBZw7fDPbW8qa9PLS\\\\\\\/gaGcNcod\\\\\\\/areAhVRO25+cArzDxIQy89D\\\\\\\/WNB7R2qRCyMGtOdGK1GhurLid3ZGyCFNlPEqCbU5tbR2fG5YaRl8MPFeXgllPjyRoR7TKH52c62qjkPrPkXwLEEj9mHkmxJhZyaPUpO89lYpbwFk04YthEsTsMqaQvLqiREZGQeYH5ItLLilPD1Pbf4vANol7Qn\\\\\\\/+j7cRQKC6fKuWnHZAbUNNyrcGAJALjvyAjN+6lyg0R1OeWPu3OOJJVaJqru5aBxLKAzoay8tj2xJvQhJ84urbAILe7KYrGM7NSIAKoVglJU5Hn7B445JuedTw2SQGadzKHpyBDgEVhqR7daWQHVaAol5epXsPu\\\\\\\/IGWBQqTqhtKWQGFCJmrQGkUrUHRMKd9IKztggLnaCSXJYhhutgZJVKXzD20+m2gO3JwZh9hohv+3oLnwmmYiafAmp+YhBCevOnu8VoVjsp3L534o+H0AcHRvG4ZkMxOqSD\\\\\\\/cjLAMaqraBJCv4kHLHYHiDE2c+w7P9i+ZFu+IVoODy7ehtovuqQSR50\\\\\\\/pLj26FwVLjm4d4+NFjky3dhCdtvp7SHZCzhzHzqq5wlj7ApKH2lrS0YugxhkD5SsT2pN2cP0cqD1n5JfL80pfHOBpaKBQ485ziTTUEow6JgGEf9LxwiY5mS3ZZYFeP8OiohBCqAmmX6ixBuU8Vv4YBQ\\\\\\\/6oeat6l+\\\\\\\/yF3o7FD057+zTmE5vIE0RJys+1+BvgGdXAH271uQpDMqr6J9SdMhxcpszdpToyicePXiydOAgShkYnU9K16VUOw8s688d7U5P6UOhkpGwszbaXhkEbFOoVKnZ71BvwfX7nGUHuvh9aUW0S5t+koa835zf1KD0MOV6Ivb20Ldbn3vHE0JalRHf3Nwt4sMRGIpGk3dGZJ\\\\\\\/IIGlCdpfBBVGdsd9L0R+Yvi\\\\\\\/mbjJSLj82EfwhjcCEV5h5Vwu30arjtw3cRJyWN8\\\\\\\/acOhMunauHBejnwxlw0mrJsF4uzfdUncXlS9GK5F8VXCU2TktMkmjh7wshBSPIblLu61rIeaB566yKM2EatBOZYwMJ+83iGcABKfjnVWp3bKpQWRhXqGnT\\\\\\\/z5kc+lr67FxDZZ7wFBpOopedMC5ASjM0qUmfu0oyIULBpko7lnWVUN2N+wMGvs34b\\\\\\\/9qnpfxMK4kSdcKI8sYesR\\\\\\\/\\\\\\\/vdhzRj9XfSb9Qjd\\\\\\\/oNgriR3Zuzd7ZJsf0Lxic9f7xZqHo\\\\\\\/IYdiSAUzbka1u9cA36AgREIc5EVQt\\\\\\\/cboBdJRBNly54RK\\\\\\\/Ncq3ljSecSmcsbD2Fv7k43UFyHZRvW1z\\\\\\\/d\\\\\\\/SGuviHbCjaXLLmjIVznrEARK6x2jijVP7mINI7E2CDy4adoOFkaIWw59dXNJ8aqbhkiT\\\\\\\/bZYgDoY4SPPcJt6HKT\\\\\\\/GHHcXILoXK+N\\\\\\\/m8pZJ7YMWBLxuVy\\\\\\\/Kgdt5QvsRtQzPIaifQ1\\\\\\\/N8a9lirovSMdRsBiJmT2Q1qecy9OEH3YvmDSGsMCeoKpSCdsZwkVTM3ui8nnrr8F26iu\\\\\\\/nCmLhDRWJvwtLITjZBoABbPurR22+0InU7SecUJWlEDPaXnGwBhH\\\\\\\/wic2jvIQwH7IQ9p\\\\\\\/tYBGgOrr\\\\\\\/w63waAtHdvsxpuCuXunSgqCB59SiJjN2s\\\\\\\/mRDP6kf0JYGIwaGuqalb9SQNMlhmNoEtWGfh7DYHXdcNLVthWEIARb80kosJsS\\\\\\\/kx+x2PFL3dEpaYTfPIvCJyzyNUytl9EgPuxvJ+0Bf19q+TysD49qGKNJ\\\\\\\/y2+rm+Y8LMuh1UINt0lsB\\\\\\\/2qfuIrx5bkoUus6Pll1e4WWgult29gqhW3DwYJ1I81NaIC9E98GT7gL09HZKu0ItqQXOfM1VRLv2UIy6AmnH4ek+TLtpJ+Pl0LZebu2za+VuYIxRt\\\\\\\/8tWQJrxtSRUXhOmDhfOXuzM0pNz2JJy0jEeU3XQK\\\\\\\/r2aRCdbZQTjH0qvqGBmuRVv9aJH3\\\\\\\/q7YqKiPdwk3n7lpIugwg5cj4YoQMmM5xABsU4kHiFwSW1vGVucaAGfMnmtCfRs7cRYKarn57Eh3j3jPtfvQyg9TivCj3A5dV6yYODy0bpGlP7+8uxrgdNjGFi3ZlJzGE0kd2dmN4T1t1mR08sRXiURNodO0FRV0LMd7gT7S7fU3fF1r26rdGMoas2hGsxCnAcbrG76oBUUflpMJSNGnkw+wE2\\\\\\\/uPP0PeqllPht\\\\\\\/3TTusWJl4w9fkvKv7Z1Bn3SJyQW62vw3H1Do\\\\\\\/oyBO5jQ\\\\\\\/ZFt\\\\\\\/9XpxXQ6OZ707xV9HTjxzh7flRFfDud0nG6\\\\\\\/hhKm2zEO1jq28HrZqWUSnjV40wk5r8qg9D2KBF5ee0UOTn927aaFfn7WKz97nedFmsHCq17ilKz286XHdnCxMg6MX31Ca1QB36Lt03+JXbs2KSriNz\\\\\\\/RyYwJBcngSHSUDWoAJzrEVdKMSrtWJMn7hempE3o2jtwRGv0ZKt8DKXxuN8V5GXOXouJnUTYQQtJdkAfl0WQuasHOwZ3UU9cj2XlW90XUq+7QTY4BvQnDUA2N6oINHJDw+iKssO\\\\\\\/kJ88I7a6SZU2sL4anqBhNMSMoDfRmMTJnJI5\\\\\\\/UgMO3zhZZmlxbe5iVd9feXhlooKj\\\\\\\/b9Pz\\\\\\\/lVbqtDZoXEQpN5EzDIqZ99JP4z7YQ4pemfiuegxZz4d6fvAl8OlGGEGiVivoBP2Kmcye8zCofNFPCCB7AylPKgRME\\\\\\\/+UAqD9wntIt6zCXnsplk6wG6rGdT3lGH4AN0CpgBAvWxUKB8xhUWEqI1RxQcFFRVXpnoxoufqFbAiqqFIs\\\\\\\/971XgeBNvOS02qcgC1qGzWh8epcQvrS64USLyhE1MimVRQJySVNQ9sD21SopUpRVAbckMHEtRQORBA5VyL2C\\\\\\\/o0HTzCBNWuadDM7FAxRjdB79nnxxZ7k8Tn83WnI20yhDnTKWxX+2HFuUsjGDrO\\\\\\\/ufvYsZ08Y8vA6d+EWhXW6GoAx\\\\\\\/dKCUXXz6WsMH4JZM4Nb12nCzJ84QYLG4WYn5SkefyUQsxbHUNp9New5GQGYZvAAE5Gc0EAQ99DfTyfDeEtNFR9y\\\\\\\/2dsxFRgL6YolK\\\\\\\/o9PipkBCF9Ye5lZOhJ8IMAyqhe7pyHqHfamK3A\\\\\\\/ouLGLbvAiX\\\\\\\/LB7rQdmb9qn2MalstgKn9FcdLSSJMO1nFUVB7wKCdHRWHV5l+lgx8B31rNK8tWAfBV3Obq0pFlhLOEIMk50E8rOdZWlqcje66swrKW5jzJA6Jiw8Q+CO85lpm3GTqCOJ20Omyi9HwX3HKckOWOxwBE9lymDId3zECfkcBF0H23Tf5jlPngcSXv\\\\\\\/U53cMY1UyQlA8FbA3TwO8BhXitkzYKfyE3jRpXsUhHKJ4ifcDvY5\\\\\\\/IZCHBwvvk6Fu+D+GWuLuSuYC0v3gCmM1vpMxCc\\\\\\\/CVtZMEbRG9vMM3iZiN+YYoR+yHuCi4jkVI1fvyb4bsUA1eH9X6p5d\\\\\\\/Lx0ycVLGqSMag42p1JNOo3ZlRjTP81T8D4NSXK7CGXG5juS\\\\\\\/U0HDuN9DWIb36dwuj8en0wLNFWMUTnJs1HWfwN5k9hskeMKrMDj0DO82Bz8kurKpX+sPw3S3j31fYZ5zs0Np\\\\\\\/UhgzLMGVngzWJqtQAVuxagH7Rcv\\\\\\\/TpH2iRNN0nD4qxsfy4GDkpwrK+4X70lrGHFytjR7BW3Acdqpj5U83XB0T2o2tLWUEP6l9dvmPk7tRFAB9X548c0k1\\\\\\\/afceNmQcyCaNppkwQ5IYcdsGJPM0dXsMMCQ5n4HZb2TxNR8VUTk6OMO+NCNkwNNzb3TrqrZFKHt+P2I4uT3f7kIhemvFxA94WyEiY\\\\\\\/6Pz6xtTaxDn\\\\\\\/nPFvnhhsa5cVEM3CHea3g3Xu+YRihKhOpSxinLOS8zeMEHRxFsq46N\\\\\\\/j4rdfHTGtYlzPoxxyHGxmsgU+bvpJmaxOjXya01jKkJgmY\\\\\\\/V\\\\\\\/tTtxF3YcVYYXsoEdpljQrN4sC9yZ414DBOV9uFiUr88kAsDBjpTM2hftFf3N+QML6KMyTFQfItSudqweUq4mfmhy5c+\\\\\\\/bZ8d+Cwyvpov52ba6nBe2cLUldcWZjsFbGO9lSIhNJtj9W+HSWLccPb0cWkVRvuYYubDA9DDxZoDSQoL+4Gv+brXfML0QkoZcb2lGNVkVMGsZhnOOt7aWcUgxyt8xae85r0rA1DVW5tjJwIKucweKLUPy\\\\\\\/x4BDNAYPt7WyKWcZe1jbrb0Lib4tArjHtpoo8HOUzJTRe7LiLMBGQ\\\\\\\/AxsuM7v6UqAvmOvIHX\\\\\\\/jXXEWfKrRfj43YhhUdawdxImFFLVY+a2MSUs2YMspiwUua4qeXzXxUboScF37mlZ78ysU3We3SkfNl6x3iVxW\\\\\\\/6P+v8Dr\\\\\\\/DqzFB+p2BSZQ0YpCsYheMvwATvuc9HeiV++sV4DHhqZDI3SyY9BOfIQf14JnlAdlt5Pwhmeh4y7CVqyWO3jbC6UAsr6CSu433AemR53FPKgAvNKwpo3+XqoG1\\\\\\\/VT4ALGRWwR7ayVKndd7gPCU8q8Q\\\\\\\/mt+ZkefkV8QCmLJr9An\\\\\\\/VWxy9X4ekxDWbuKoUQHZrQtJnFCeW2yGGKd3f7OMkEfTdXhe3cQqjdk6wDFKlYhMEsOCVaD6iViBJKX2tPz0jNG1hjlpQatgliNI6ENwcHnfGGjvArWwtg3bw2q+PeRTh7pXT6CXpO6nE\\\\\\\/YnNjMjhO594N6x3OrVE2L0QZ8uglB8iELTtV7tshCJav0IBPIOJnquzHraUXqaBDXDocC66CqNVwVgWAGIw3WZFIwbMUnndGI4+J1K52HGy9qhOro6a\\\\\\\/UjzDsZJdmSTJRjqopd3R6PoiaFSwujCFdDuKCat0PYvi68cMWR3jjOU38VgdhqjYdcBiglNTXg0zzNz+RGob4Rzhp776cPUJiAB4LUlRWt4qIHhAQxdQ8mQpCwdyvbAHaMr+vgmrtcZbu0x26jBfBQ2\\\\\\\/RVZGpGCvdD8ze8Hih944aTMFyIgpaID\\\\\\\/qbWtYRo5Hnmj1RZ0dvFQUuczUH5zfj9BGBDFlYtY3SxvsrD8iVPB4Z\\\\\\\/9C0\\\\\\\/3O45JLjg5q2qY9m1MxNc4rO\\\\\\\/5XgmO\\\\\\\/j1PW6UmzpklEY9W\\\\\\\/PtT1TnIn8Hgwc1XfZalnjgcZuBcvHmSLwKz5q7Y8w1m+jmDQLKR97DiGSgqKtVzRN4N1WGG497iXLyfwZoVW+vppGTJBRB+Q35lTabOyR0sPOtpf5JalxC6TP3YMffrBqhJbW5E2sElzA2N73fGrZr1OaDmDRVxyyCKYoL7a0I3AVaukRHw1qQ1DLY3JzAxe9yrBcN2DLkIFPmrvGjdWAmIQHoiJL57j2IHGHS3RwfpL4KAyPs6On+TY7HIIcOzz04WaLb+rrUo+PuXEndKWfBf9MmU25hbjkZG5r44PmrzUpM7+SeTR0gqRLofvRyFsxtAOU0VbxAp1jxAri7119UsTpk7vuTIVAoRnEkVzegY8J4XuS+rW+H7jLvxYSmVrtm\\\\\\\/rjcymFhsbM7gL\\\\\\\/HNHvt7JDCMb9ALFZjnKQCjaO3wRg\\\\\\\/EksFMuhU7LvaovGTvIl7mxiWMCv8vti5zyByrBnM6YslrxISPTGaBIQidEpLjnkEEGnKxzhHVM1wSaOJ5MWXErVsyH8YoDoyXkz0VgAl\\\\\\\/QO6lmW78s2LdLhQJY2l8wWqPRQzgAnAgo9LdIrW4MFIHh0N1l1fwhtZnfAKMhPXRFEUT0c7V9jjREeRmgdzoH8BHhOsRlTc+5iUi9Z8DcZNqipRfIYlY4MCm++m2ZYshnevwdfnnNNkf79KCCvMYZ8rRPMDYbAJ8ecE6LRjh\\\\\\\/1zGAQL8wFe7ZOx+mTzvRyvBGdv5G9auNjdhbscFmoJ96gDSRe7OGuMky\\\\\\\/Uq9P50U6eAvDU4wmH8ivw1sCONj5amEtOj1yiefCpaBqQOGale1F\\\\\\\/7126fNcpE2iGZmdhXdAyHBHsJO+xaNUI8KrI3669gdcJL5v7fHmYYuSO5K532AAxpvQpPiMvsscQxyOxhbYg2saTLwg+q3mjP1sz6xU6ADxEOtbLZk\\\\\\\/xKDAaOK12D\\\\\\\/dFEl1p1WqLgbnVpgSAWMqWsM69xyNULREUGmwi\\\\\\\/pUChiikqS53D1Y+SowQdH0PeZhe+UuXNKp+lFZzxYbmlkU7Gbt3n9G8Ba\\\\\\\/OhmBIAKFVoZbBCxRf1pCF7PqufD8+iGf794YJp5ZOMdaQTe\\\\\\\/Yc31gGouft+ENdYX1YojqHFOIOI0ypbaoDDkUluQ3QssFmD6SBgh40Gu+mwBh94YlA1zP\\\\\\\/zgkZKPeZp8Vzze+8rU3mrtQjk+YFjsdcShV75f2\\\\\\\/ELBBDgbRWcZKpvkBmUAEnjXdDRlGZOQnxwevSZvE+ZUb\\\\\\\/9Iz65stfu9RP3fcrwc9urvBo2ZEYiZaM47\\\\\\\/ZvfGMoDDaYt\\\\\\\/8c09l1a80L3JeWUB1ephVQQHPR+SmFBBpwYkWio0S70nS959cXYl15b106MXQPvTGs1ZGTq9X8Q8mIAjexyjYCFiyOg5Md8Fv1zL1GwEwymIfthWyv1bbtPUdKSwNto3dldn4mqv7r4KK9ftrhCeRvfN3A8Inzsw1\\\\\\\/Sem7ky0kl9sOuFKJ90Z4wgPZm4H29JR54qJRCs0tld640u1iIqRErFVS11VT4AAxTGwHIq9KybWGecn0FZTYmqOqpeTD1PdAFkrsX\\\\\\\/5rdydal0q2mxgVPzWQtEcdNIbL+4rdajWdMaCt1B3hI6MmKNNKz4NZQYRG4LauJfXWUq0X7z5DCsU2tqD0g0MS7\\\\\\\/U0VWJuHxd3VE9oiFtuo\\\\\\\/8gIP8hEU+5oSBMwjDslFlUzQXOdaZ6gapzcXoWl1uHpoxQnEcTbZc8DkTqeEWLfwtpztuz5oJ6bXxZ5JRcfheQuf61rA1NXAlnxTaWPeW+lAgnaeo24Xs5Qpq01oVjHULJ1nedgoLlfd69csvIDrCCIFdw+UcUbZ0M2xScnGAUzHpbyPtO9f9mfCBhaUNMh9AA4b4seuqMvl\\\\\\\/7TVbhpuS7xZND7mSCr6agMEyBonRoqNFZm8YbqcEdvYu3T1eyx\\\\\\\/Ur2KAe92sBxEncv4qsRwKEDP6B5CFM0PV6dKiv56wjeM1ixf5dHpLn0wHVpo+pCkudmFK5lWsV4\\\\\\\/W9sS7bal6rvxfi8cgQ6wyPSvWA3yjd1KmeNwcwWyoXOtKCNBMj9v4Lj+r3ZiYisd4+YimIFkHR7mTyCbyQueGcNIlrrX8TefELQUG1MltK2VNKdWzBFRx8aUgB7yPO7wiBxIFfONzNxrpwKiRJIDPP95miTb2n0aigkydO4wKbah7V81CUfO3rU8epWMuF2lxn0UB2yUd3p+rRMMOPnCwL\\\\\\\/uDbtBUKkIazF1ckSlUZ\\\\\\\/ozT2p7U002Ds4GNZBsVZz0nJlzPDXi4IeWzOyzOUH8bWcxxBe8ZeN80TIVg4CmgwTwkj0e2ZnQMj1lVmLIn7dNSHSO8AfDq2yji7dV9W5bY9b2oiKjNpbPxercTGSN3DfNZjEL2I7BlPX0bp7dgSYUmqeXVtpAN4e1x1vvXpyB50anPEzSjcGniqIsj5ClpCmJ1Bp3QSEWFb2ogevBxoWU3R67CAunf\\\\\\\/ae7gCtTSPzgUQVnervhd+F0e6QWZgvRUQgD1ueNLcmqM3hS9MB6OuJdJikS+Ky2\\\\\\\/5ppUHYc5YaWw+cilk81+BoC\\\\\\\/FTUNYrin9W1XJcRS3kjSt7\\\\\\\/v9aDFG1L5qT2Hs8Eu7pfwaZnDThOWer8bmyarv15aPjpxV540PHAzlRiuPpuQCfng2XlBwEfxR+a1T+wnVy5O42yPQkGi1PhnzsB98BopWv547SCmEOvkz2iRiSlBwnJ698sQrto8TC4qCEfy1WrkvLJQep5bu5Q8JnJvHnU7+AX2\\\\\\\/pwQFBOHqf+VtNYPv20hL+5B4m5pTxlnBueP3Z4VCchRP0UqYPWkbPpsMNtPGADm\\\\\\\/G8NSZPSlO6IH8h5ogI\\\\\\\/UcrtkfwQX7KcTDvl4LRDCrunO+81U1GnnXVj1ec5kxQX+LZIemB108qRysxfeyEfSy1ru9c+3HfGZg4XVI0CNeB5EUXvVQ1ZpwxVZW\\\\\\\/9qJHGtPZcKfzn2oJxelTXfMxjpbKDnT7oOMHP9oQZgTAon9eUT7z\\\\\\\/jHHwj81OJ9ZNIdaGayN50SZiy36FyUmSa1RVVD2m+bD5o8yJCS6N9ui8RjCEeMbrD17A3pGaGv9NUWpbV3YvznHbOFVquK5Yi66ncJIxu+Q09LXat2PhdMWfIDIvhdm5XClZynPUXcU9g2xOG9gur\\\\\\\/D2cV5ta5Den4awhI5Ft9zMn3RYelLWuLrisE7qhc3VBM9djVgjMP7oNOjRDXis7I3XtAKiGrgzkW\\\\\\\/IKka2oI+lzoDU7Y7uURqXKqYUpTA7rzETcfNt597u4Hw7GHcZEQJtzkWaMbTsdFvNp3gUyEohUxpPhBCMSRqPLfy+ieZOnB7gZwMmHUJxHFfWcLmVV\\\\\\\/CzI8teHzcsQCfqBe5Xm0JtV+zUT+g4uRNUYDyjxehiaI8s\\\\\\\/ToOvFrbttfG+vmrMpLAjtbI5ow3bnE3uGD4ZXsuO6V68NPkfglfgaFy2EP2rs9HCUZDs9SfTXuhApNI1XRM2s5M\\\\\\\/3+RFgohsO44ahG6ewFrl3lrETWivAJP6mOsJLV\\\\\\\/UOgc6zn0eaGhDpLHchYQcW7V6MqK2wesIc4sd+EGO2hAwMWm\\\\\\\/wrtapsTaAmrzGZfPGBKKDiyt5sy5w3edYQCn19wUIxl1FZ9+CQkr5UDF9D7YC1RtH3Rohv0hkcUZ18kV5B0AlnW4ti4WbXggHA701FJrShweTg74USJseVQJ8EfofOrJgxhF88aaxpAzgnIrkaFJA4kI4amWMN\\\\\\\/sJIONd6PL54gve+AotbafurnXdl4qVkByfgz7AdKZCXHERMF5MyExdDF2ji\\\\\\\/p2GHqh7eMZtqk4BacwmXhMvQN4WgWgw9uHxM0gMu\\\\\\\/DQYJjZJuLvtkISAppemO+9EhGoRFRkTgf65mzgfXbDeMIhsZElRYPfMbM5gsmpGcf+6nrkc8+k0EVHv\\\\\\\/wlB+Ljs3AjDozvATiPBSSAccre0HY5E29BGucRL45RGKu0Gup8gF5gBfckYhofr+4pabGcomtXZhcbBsBG49lfUrHTL6D2Cj7lgvvlUBN+Nni+966g1XDyb8149F4Shhu852HMqMTz3olHsyyYZFjuxIHwjbLvXKwpod\\\\\\\/OOjl2hWYY3bJnWEvOXiwvGyOO182gN+0AsJWRTmVutL73zvreS4ZfMjIbS3djgZpEXJ3mqrAMWEwpzQglr6y4R3oFnfPVw7ra4O5xB\\\\\\\/OYXOAITx3Rx2X3TpL7D4QVqmIFjTfTBO5fodxLeLw7x7BzNfb4rkI4JWd\\\\\\\/\\\\\\\/AwCqPSi3P6O4NKcQQ0ZpENS+3f2\\\\\\\/iBkRiAKJs71XvLJyxOBSD6HiEdTWgaT3CXIvkFYR9T0R6fSQ8m+aTLzcWTog8BQBRXIBL\\\\\\\/BhN4oZVn7W4swqLzijJPFjijm8dhdVUk33rfDlnLnYBezqif7GoBXDWNW\\\\\\\/KsTpSsc3JdutxFHPupjGWENsvOxSEgC8lV+KIkGHIiAq\\\\\\\/j8UTA0fPlJUp++4dbkn0+bW+I2cGLp9mQ2NNziy0U3Bs9SU2s6UbMVhhYGKcAdZAJsh9oR+CMdXVXqm63H5asEEdsLIPB01lMmYnZ0iehhPtPvesQB3S3TZAQszUWtnwGyuHu9Spa0RxTluOspnV5zQRBLZheXnd0b2TkqQHfbKM0S4rtU3foNAXauQzLlBWiJ7TToY9ppXBqMYxoatG8IDKmuS3RvPqmn0ZPptuIocP7nXF7rT3kJD5ILthuDl49beegEf4ABToGAYpTnCH9JFKCS5uBYWvFhD0dAQadeVoVcD7wi4Hhb\\\\\\\/JUyS5djrsAx\\\\\\\/UUOsS3xhcrDnvNMvytaHjfnuHX4UXzD6PLYS07Kr3CdtKQDMMRGRsbUNNYuSIzucktbAXbzDA6kWwEfrI81nASNFwLGxT5fDex\\\\\\\/bouqKp9\\\\\\\/ryRkhQz6ZMVSrkukZ5QjOoCywFIdZyfGsYpk5n3AJGmC4ND8p7J4F3Eos2imlBxgNnCPQq1QLMf0ifuIHrHFXhK0Hwmt6\\\\\\\/rNoUE50QeBZlxcifdFvYFKvZdn8pGZzYDOL1tPLdvq3VqU8uA2gHqK2iy\\\\\\\/QgFdJS9aVS4PDleRavcdV\\\\\\\/L5tu1d8Lu67GVigI1un4mGZOl3\\\\\\\/KFqhFZdJ4pII9bB+fYFW6OqV0fdFWCoW+WVBZPOyiow3eILDI1HbjzsYKJOtYrcqlMsEgaSjW1LzVwZIJ8\\\\\\\/PhYJsI1xhAfIOVd70zHf1kR7hm2Y2+BENCLmeSJNPGHNXHGZyDzo7NhpRghgYbYWYs2FFJGpA+TKte\\\\\\\/NfXhQCyZFofwPLPlM1jxAIaiQ703oJqxKZLUDvTVRwKwPgQUk7AIicTE+FzCJvLxxiabr7NJO3UzAZak03KYR+3H\\\\\\\/ztXSm5hbI+21BktPUF5dFi\\\\\\\/EwpMylTDAxeXvafCuoq0XAVneTqbDngxuvlF+tkE4F093+\\\\\\\/H\\\\\\\/gH1SrOmEW4tXkD9eI9o6tVHRNXbghmUrHmodNWBxKD+mV2JRglB4MzybHMwxsIZZMKwKuOzIlaWLDpKg2qDZlHtLd3tvueKbQOrw2ceZdEKRf1+vcUkD1Gl8wlP1MV0mI1B7gOXZkjfjKWwo1EN\\\\\\\/pNXvVet4CE8V0KVFiDW4hFwhwLCdGICAjzuB+CvN+1hp1BHVqbndfzFxOJEXwv85p1TnjJW6Qfz1axBBa2cjmbOVyWHydEj2dbca7YWv\\\\\\\/Zg69ZioKHR7s\\\\\\\/\\\\\\\/HLu\\\\\\\/JwMpHl2Ba5NsFjiI83xlDLdNbjHUEkO1q8cUTscp\\\\\\\/ae1yG8QQOMLnsjpr8M8WHOeSjt7Nom1CnhWzgc7eAq+Qqx9iVD3s4lmOcAQw7CtC1ObGt7IuA9ynXZDvPZZ+E\\\\\\\/c0rccejNcHrR6PoXU6V7XJsicv3Z\\\\\\\/KcSg8HXyU1YIh65Q1OePB4EJpZiop3sFJLLbgOzIykd2\\\\\\\/JGaMBfapnYQiD7lUEqG8qkKz5PdCgvaGCdgnQfKg6H9AQZF+tHEoZ7amqB30ukJ9inSFDdyhzI+Jse3dipOs1dRR6tSN5c9MVcdajRtr9AB+XG2NA\\\\\\\/0T1aoHi1OnIff0neZlToO8l6rplTuWOk2HfYmIxGlLZTAS6bolRYIAM6qU0IP+m0Sl0c8uyjLaDQSXUi7fcDWDncTkcI3MG6FwxCFSEJaNX2yUrXzLwHLfjYyFQZJaSnolu0kD+AY4jscUSXZM4yPXWLzNKN9PUxlWOYXD\\\\\\\/rNJyl27u0NoRvgzV6dctEH\\\\\\\/bHbYt6gdO2gyVsoGlTYxj58ghEg9jKFWRpaEVC87uuaV3S22A6ErCNZ3d4jX3WI8QC6wiooQKlzsKODupbtf3+9PPdLXgV5VScUDIUYPOhrerZzox5xFTlj0ID89GgnGQTvnYKvZUYF2BrbDISqiZTkB7IZcg2wnsbo5IKlmnLZEp\\\\\\\/BxukuOfrISZlxPi5QQIFkpL7l\\\\\\\/du1t+ny9iJsApKTP\\\\\\\/C7tan7Yx0WOF8xOSPGybJ\\\\\\\/78G8kEPjVZGmsjPCXPLHDPg2nt+mmJkp+CFBXg0QOmABk\\\\\\\/aCD3Xa0rpD8RyjhP06UwxzZspdUzJGpFRK09z09vICljA2rRXWMz0jzjJwX2kySI1f17UKUBPqPHsDuS43zl85ry1a1otNrg8UXcDj0sOGc00tBUHTSpycFlVgS50twmhpvhpFNDMHmwXby89UHQ7AgyYibSck3Pr8I\\\\\\\/1MrJ0MB3OMWwGltYaP4sC+jn40jnNq5HI\\\\\\\/6f6bo07ARq2an8XQFvZ5KcOZSsEqjSZVLwSrFLOpUqSnDtcWcM7aLOI3GMm0y3di+Ig1yTYUa6eQDBdBHOfdMUpz1IBCTjL22nwIyVKd3k97d2GvDW7CgHDOkFi2iV4eDWsXQ30MvmnteDJIM8tziYzYl4DKTym9Rg16mg5P4C+sVM\\\\\\\/MYr3Q+\\\\\\\/Dt58OPBm0b0XLPsqCpdUUeo4D0k3Md\\\\\\\/m4tb\\\\\\\/CF6+Z+LG85bjKZK7+buaZ76jUO5letJxnIcyuXBs18g9sQE0hqhpkdfrBLYTRsVY5U+7Kt8l5\\\\\\\/nj0xPwYkwJ901WP6qpTuCLec\\\\\\\/mUOTVTHpDDLZO9MARzplXMC9uHM9LJVHDVdlwIL2jw9dXm9HhsHOdfXcSn5vqbRYhg5gDOnyxaMmVLU35oqs+1Du5YaqoFA7l1vEFhD21IAZ04Fm2QHgcLT5152NgPpswnQTpO9H8mNVaXCfP\\\\\\\/C7VV5Y4bQeoEoEWCvdzQYvqWw76oV1OeDH\\\\\\\/NsjVJlNrW5bIpyUXAYcK+07g+A+EdKPVhFnTo6Fqj2rXDPgFzoVz2X0+9j8t60tmE+PD2++SgDZlLuDv6141Xg9OVnHVfi4hErr8ZxAa0iiyzR1VR+AyrXlCivRbq\\\\\\\/mm5T78eTEpgvq48e8l8FK+mgN+Z9Bcxq06bp5E3v2TNRNR3HuREM0q9OtSU5fdlMtOkX2RwOYcqaC2AdXnMeLxrPsbVyTHisSGmpgb9dDkgAbMPb9utAEDoxVtXnUjGWFLnPgwn0HQcbzXwLYi\\\\\\\/+PjoUoYC0dqaCJTLFJVvuC8rdzYihIOaErqfdPcDsiVY\\\\\\\/xnF4mIzmKBAyup7mGJkGT\\\\\\\/osLmqkB1rYRDYhkPA96WYmF\\\\\\\/jdnn+jQZTE9OJaAbc1207Jb93Q9anoAlKOKmVFKOcqn5ilAo\\\\\\\/PaCSqmMMoWfXn67n7kckiGn+dR0qMV3WBaW7P3q65aSxlfH15PYNNHZgc8q+aQ++qqIkhzrH0FZJ0P8RI3zB6htdZMLdQtipThkDjbWyqGaF2BpqWV6qND5LBHwaQDo1gQN5H9L6UgB+yqCciKdKqHPxq+EeegG177jSB7HSwfQF\\\\\\\/wrKeXq0oQUrJf\\\\\\\/XpRY5Er7Lx6LFrBzKGYfWp1yi60Mf1p5DPiUkzaqg0rK3pcB\\\\\\\/ln1mhmnYh\\\\\\\/zRkX6JTEeh1pz0tQ6gZmdbXlf+gq6ML+DeNfdLdTffZnHSs\\\\\\\/WpB+2X3mqKxh2OojFE1jMKISr3WllSAAsvJIMyLXAzvzOBVseEaTDKxrHaTjLW1zSynXfOmrxDT9V0\\\\\\\/vB7PxqmblyglB+Yp91x8FDWCQHK9O6GuaSdfVXLgKGEYwvt+aCYPvKRnieig3nih7jL3crjIBJKWikeEBi+RrZ9TWkDbrMVK3vot+CoKoaELt+Gmvbod1yNW6QlHULf9wjAFtnvwYOAbexMm6sGyi8NkOZnuCwVIoh5xt8KmVsl2NhU8JzJckhd753tuCY4IaYxuN8KojUDybiUiA6aQV43R2iWeYxTaKwxzELGE8PRYSKjZEfagXuzpejVjen0xEq0HwV0Mpo5D29lY9HjTk5UlAmlMGioE2j+nUO95hMBjw+G0mhJv0xu8uV3jH4zunbTSPlrY3uryaxsnw5BMeBoSxwe\\\\\\\/dnFYEYF1\\\\\\\/IPkbIkfu3wOyQdskpt4aEZN8q4PjsS5WfAhpI4gbBjTV3CvYLmUF\\\\\\\/iSC1hm0\\\\\\\/bN8G5E\\\\\\\/r9XewYVZ8tko1s9loY1tuY+rZnhJl8bSQUXY2CuFJTkNTzq60ITYJEtlEZt7hPw3Bz4iGd8h\\\\\\\/LgJE33n86z4fYYTgZRyIzfhiYxLXbSn9BlxbOHBodNNfoNDNFGwL3rXShim7wzXyeq4T84OvjBcKjHeaZ+vwkSAg3X0tze1yVWUCyVFGtkMYKtOLeluVT6FgW9zp4ytZ66l9gjBA6IbYdabYC61IQnBWQ6LjT3o7Z7unJy85Y0vjQQ8vi+GaBariCdVsIhtFdhnddIlzWELRDlUb+mjbeqGDj3TTstpTXzAANgjQThx9d2ixU+xcnhkDZDobmVEo3IWdVVqqhWl1mIskksj7qPbzmM2pgmeUEjb3E2Aj4zsGb2fPXeGcseRi3Hf09YxYwHRU3dt4vzh1mqakpfnGOMLw+klDi4OL6oA4vrSHTdR+ySmK2w7HtasTqyGkexnf4SDBrGLIF2LalnzTm8Vaf5uYsaPBp9X7bKR\\\\\\\/1CopYYuXaNE+BWtnDXnZS9lkcSHNDfuNUqWh+TPe92T5ZOprfes\\\\\\\/2n7OTegZz65s9tBb9aWFXTj7qcxzNe4M+Xv5q0O1AJuoTPRziOm31S3zaWd4qJ68xL1cX0bUAcLc8FjosXePpx+OgLxnJ8BuIfusSuMnSj7buDVUUSbkLG4KO9lUqwjyb0z+Lh8VqeEpefzrAfNbbu9IIfBAllmpyNipXYtOdC8RxmnOcIVeBlXhg+GaCVlRQPvl4T6CM9pANw5YyD8tjk\\\\\\\/uakLkTpQrsg1FpnugaBXfLudUm7Ve+SzgQH4LQtrnM6nLgJqYuelTPiYgT5jL3aAmfEKRJF4o6hlmji\\\\\\\/eMwPP5x8SZDoZOhtloAMrnceJbsl+MT8xHqxD1cNif1PGl1ofSxkUk3XNav59h76GMyL\\\\\\\/7Nj7ADFv2XZxPhaAytBJipHNmcEeHx\\\\\\\/jyx4gkJLZVcA4x\\\\\\\/27VB2DhR58+H244k9J5gJOc3vciyaTrkoMYT8E3BnbTCWBNJoj8wFwLhe\\\\\\\/vdotMSy6i9oUgGPHfS8L320N5IdGULcI5fQaa3X7WETz+qZwTp5XpgzqEeR7wCXfkH8exQPLIqTYdumY0b9kX5pvD2s82qR2IMWnAbZTUZS3rgSvS1clrwtE0hWwpchDxsdyJcdzkX0fvHxvb\\\\\\\/U39iw+kaDok6hu5XR116vA4jqRNdLlGPKo7lgsh3pCJAMllH\\\\\\\/Tx1KO3dAUvlcGtdgN1eA6FgiAL3sUE3kbtHQLSHEEqp\\\\\\\/XwAun6GsaSAq0RDwhNqJEeVNerxiyqUS494Lflkkr7iK\\\\\\\/bki6cjH3B5E1jsDh5KedtZIVIAd5aHSUfRCrcpJOu7X6RnDF+RVTp32QrgWqTOn7m6QHQmdE7zySnuAf46vdLB2Pj8wnEc9ZrIn\\\\\\\/u9l\\\\\\\/XPYaf\\\\\\\/Xk7ip1Zxt93dFbZp7qPLZJALMFmz363iO7iuhlcBLjICRM955XbMYp7kPrqahjMYomgC9QbU1DrtFdEa254IcgyxtNjr2ktpc7BWm9mkmQgX\\\\\\\/o6UYRoZDizNY9PaT0onlMA\\\\\\\/9QvBOtDicpEgvGqssTeX3D4NUDMFM3XMiTXOfLISFnA3jaTa6aA5fsB27OPQLnNgntD2m5xLQXSZvJj\\\\\\\/xQM\\\\\\\/MBvVwI7zUIjQUHKq3zd8AaO7uUd+pEkEvw2DcTW2BPd54PpYaamTj7SnIAIgA6u\\\\\\\/a706vfukm1KX6uumspJtvt360yqTrLOE2Cz\\\\\\\/q\\\\\\\/tYV8KawYe30IjD6i8U4f+CX9be4lQA3oulM6E6qpoQ1lgKEQrh+q7kCPCp\\\\\\\/Ui6NswWfhHlqrhjNgB2Ts88eOp9+LPzDb\\\\\\\/HriPrr8BH8pPNE3sNt6tlrtQsX4Mq6EbUDWuymTqFOBmPpcoG4azg\\\\\\\/rzI4+sGeuPXW\\\\\\\/eunoLeFfAC1aH9kEGsuFnM30hLsBl9XO96X36ERohokfsuNxLHm9dpUqJ06IE65Kd3IN7TdqjmFWBrlM3Fc2dkfL6ki60jpONfWQeZPBuc1OqNwT\\\\\\\/mQZeErsaXpaEDw5lPnj47qLa6lKyf4+7S5bHDPNKFhT8Yfl36BOL22z9dmXKyIgB2p4H1tuXEw1sblsXhrzAFOcAAdqVBFGIUWKYASN4rRgeAl6Olqxf\\\\\\\/ZjELx8myJpjPNj689sjDiJp6iVoUoJRZ5dOtUeQUQFZ8lhG+TETsg4R10iVHwoY1IPYDR9eVVjWcCrdm39jLiEaEXP6i7fKKzMr5rLmXOfSNDBNX6VHazkR3Q5UQGaEYItRIaf5ijQIIa1gLHx+5LYvI8Ax3jgNMzp0BMKVMgeMJ+r6+ValsRm9uzVv8f9+K5J87TnsZCLBe251OV4tcOCYTjh06t5Ik6mEofDyFl0bOVEotj+Z536gSctQpKdITGMWSlRvariDijni\\\\\\\/zWhw7C3+RPFFmQAWO8OTjCRiFlau6Z1fkCMyHCDmGX4Wn\\\\\\\/0K21z4rqZLMcW\\\\\\\/7TH4cgEESGC6bTle4VUP1Hq2jmsWJgMRkywicosfT8Et7OfP96tH9JHhFMHp6WCVPSPeGqQEfL4qRLWKx6LLxA+YYpgjY1PqRLpNeOGLHkkvXhbbRvtgYTZsUl\\\\\\\/2LDM4tftVBCcTnP49q1ZU6MtGakqqHvVsWMam8oPZ1n2N2FGop5UKjft9Dh1F6MsOutCvpFBSqnc79jYDhFjRveEVXv1n3OeJaFsE20Jkk5EyDrYjJNCIHINjiz9NhT5mNyA0tWwBPw58iwXG2Jtr5wl5i18mwjg4284\\\\\\\/N3ILtlAJzA68nhUQHeFb5I0WPrC8W77rFlFYWVVUPujC3pFQYFpUOWoDQy1dB7fvWcNtK6+Qv+7dnt003g\\\\\\\/1cJbCkkNdvBLE26JaFBY3OXZf3jLASfgVqQsA+7lkYagFWFjqe2CBpbig2kQ74BpWSchotoXS45gUG3zp2fV7KFpot1vHA46bQ6+3mQI312TmUaaa4lv1ecXMS51FJlt9HDpx9Ey1Zovd\\\\\\\/CxqLD+Vy3taSN1Rv03TApsORfsF++DG1kS7qlUYJLaZ98iBKBRt4tz65lnCpv20mVXqQku7MqHJEMMpwqV27ANgEbIQK+RBrMao26XiVDIrCVHkFPHbFDSDQGNQF9tsp8JTROJxKgEoDJllVUC4yVZDu34b6cFNVVS88QH+\\\\\\\/6o0zlI1gXLup1IW4rvQ+ZxHXDZDgCOpauq4slA0PZDFmnP1bYtyM24XmrqMk+q7CiKZxrpZXt1uJi9keOVZ5j5cizvXILLjP0epvQOKq6oytoADtw24jA0IU+JC+kxD0YYEYdTAhXVGHxmlF3PANBH3\\\\\\\/4ZlcliR1cdWzvXGuRUJK4ilp68G1fDq6hVXe0EzUrlcd0Sjqmw2yHYSgVunyZzLVaE6oIsGI20756Z2mpcbJe7iMZb5F+wBMk6OOPyktlFOjPNEQerRLEqVTmN+w5AqNTYXQsVwtPq8GFsEgk+DD9kXQIUfFOFq2PtwJQS0SxY+VO1OJwGdEi+k2tZzjXq3mGP3H1PkqacNAl8JlZ\\\\\\\/il1vRch0zZKzSvEfO2pIwkFuZqAvFEE84Z\\\\\\\/O6x9hqoHsKn7D4ubRtWrVH2S+I+15RQKFvufT6UuAcpS0HNyM9zXTh2qgcz2KOSug3GJgq7ggV0c\\\\\\\/9cL+Pi98a9JemfDGm86gMst3Qu3XcBZgP5fZi7YqtX9mvJyjCsfFPSuQavS8qopX25PcAEL\\\\\\\/8bNABo0EiJNFaQgn4sZvrBNIKEQ29lu\\\\\\\/IjA56A9nSLTGnFsZ6rgMBkeAogSJtuPWR3Ppaup2EykdQ1q0ddPl6fG5waTjDQR3MJ5ju5vnGpdv\\\\\\\/V17KW+5tYu+r+MVCxi8MnxCnrfNR4o1ZEhqYS3b0vHtua1W4CxEa9YLVl3Q68cMm95O5X1+36srfj+dhG7E9MWmnM8SEAZO7x5Ztb05UFGo35RUcSChsYWO5uy9Bz5xwU36G9R6IWTqbiqxKr837OsXqFrMswQkmKqv\\\\\\\/kB2nFvkNouVuOH89IOWaIpuLnajd0n6Uw1z0ldGOYjUigF0pK22PQdRU306BpJ4F66o2HhUQqTcXFZFBBzQvPtxnyr3KnQQcn0ZbGZeCRy8me8yeZrVZ6HgepIz7v9RasEOgs9vu1BJCOx+1BjHHPucWfvboxqqpoEkBl3PpHZX839YxPSqTdryJlTZYcICKegas6DI2vMeQ084e3Elmxq953SNroFKgM\\\\\\\/cC24gkSeLBbry3jTEEU5Ksf7oLi1wEeTJgwzCA48H84V8evyKa5f+TUCONd4lbDNpJ+CQD9sLG3N++ARcbrB2mevfyv7Z8YVfaDvmpM9a7l4Bvuz3T+LEvfXyf5AB4Usfiw8\\\\\\\/YV3qD8yKYmt\\\\\\\/tJVUxRI\\\\\\\/6mLY38Zu8VlB2LCnDHEWV2aQ9INeS6B540nqmMk+OQoV4A5BbXEltIQa7lTRxqAfcYYodPzqFCS5vy7NaA79dDfXqksWltHH3ruin+zVEHJVb7rFVUB3dSJBqsqQXGWMpRSneuW9PglVNuhzFVnMy1wY9npwI0P7AKRnkBUYKVgkZiEd4TihAt+hAFWC8Sqj6vUnT1JzzcduNxfiMwa7sbutVdyYwrvcTHWsE2cIuOSC3uX1zuZjmnzI9I3ZR0Zdk12OcKilIcpg3at5jnI0sO0iFB2F19s44FlJ0NJDujN6RmrhxKavVfjpc\\\\\\\/8AaV9o+RPaVI6o0D32lTEwLN+kJzlRmZsaS1zqt4XN\\\\\\\/4w607yh3I2RBKiNqJFxBs5t\\\\\\\/O6+yrGpod+eWEwii3X72kQM3oryKumaEjX7mKZAf4KfnF\\\\\\\/0\\\\\\\/+it+ycQ1MvPwcjxNCbFthHCjTK+iFwHTyVnIhUHP1FdLdlwywXwF5nkP084oQ422xZiZlcTn1Hz9i2px6HqJWa1zeG2Lnf5bTmdDh4vPso0VE\\\\\\\/YxStFtsH1QBf+fXxEcvLiBWUWGm2gzuvHKVYi9ctM+Zc1Y6NXt2CEJHMiM3qWR8DBptWEXQK2UzIQTNO7W+PIfaxVH4Ov2+G9PiBass5RrmwN+tJT23EPXLzlNGsolgvFYiMf8R\\\\\\\/BY3xhVEeWhsmLHS923PPIGSH30QT6rEMihYMUXOMlhbk33pEwOFOfbwJeb7E05b86LPpkwB84QTve0eEYbWKL\\\\\\\/OCHdo5u1Ft71am0OrWXTVeKS1L7zmgb0OmtONgM43Zjo6vPY+r+irDouPJfl63J6saWiAjIA7Aw2pXHDduBxalMjYnORriyGgcKBm1ke2l8uS5eZAnfTQyS5xyMs3HQgG4e3lRmjAYVYnNqfL86+OTkYjR1zQU6LwIF1v2aN6LbIzIkbvw\\\\\\\/\\\\\\\/GN+owrt4j\\\\\\\/ap3V3Xdfbws+ZY0zIHvQq3JdO\\\\\\\/u+NqQ\\\\\\\/tvbYfOJN05a0RzonhfeJ6jM5EtXu9Xi6S8BVSbPpjLb92h3SJdlg57T9CT8+kCBYrneXwyAxE4Qapl84pLMZ4mCyEIqHhsHF+X+J1dWfa5YXftpkduzxdubfMeB8HbvYMSgr5JVa1zG97OJ3H\\\\\\\/WknUiKby8Uj9XE3eT07d78uvG4mjDDku8iOp9uUyPk9UK3sT9l526qPH4zto9Qi\\\\\\\/fJzH+q6QDlllmiELGNT1oORffROVJB9lERjf+QpXzEv6EyrE+GhXTO0Y2a1xpei7g1e3eOy4R\\\\\\\/Oiybwkk+ffp8oEin0Kgj8mh7YehR2hrUnK\\\\\\\/G+MklrPoZ264LSApsqZ42kEnNHB4f0GkkGky8rdwsZCl0VroS22YTWLFa3pq\\\\\\\/g6N3BAFK4NJiJtMj1X8cibFf0CntrzDFlHa4OvXGhwSV7N5S09I0UZom6x8uTaIRi4\\\\\\\/dv96rGldlCj2SvyLG7D8k5YX0g\\\\\\\/3yQhxbh4726wi5IZ1Cw1y3D+CynBlM6n2yDIvASr52JCbhARyjGsYRaW\\\\\\\/dk59K7BGEkEgXye3dbfFwsgHK1QbeNmk\\\\\\\/OCIS6l6tjv1aaTpxYiQfAGACrbRJB4wjKYvCQENoh0BkaN62ouLJzzqPET5Wy0guBrIOM5fYOXHKAo+LJqInLEetRb1pKTqcjz+oeBP1av2RTNprlXgIQ\\\\\\\/63ZgvzSHjKh3q1ivcS90rbdzoIcZ4gQq5P\\\\\\\/uePWHWqDtTLctt2jR3W9q7F2hXfyKO2c+Ya9d2afvOFyXYflNi4HLjvSwZ9B\\\\\\\/pdZdWG+zumCp1UTsbOpS4rTNtrxHYje0IDVvuTW1+zuzhbCVHi55fN2sGm5DsT0VrS2oCMvrDJidSTcKsSAbIJMV62rPCyy5r9UzUz6hGXtexAQdkDypiXr3SOAeGJKNSLHBGH0f6qpsKW1M287R3Y+MK2D1nLt7qSsOWP0t23ke7eu2bCT9U2GIVxWxJ7rhxQneaZBJqMWT2dougojYWzVjiaNwsR6TXuOKzSPyE0JBR2KbwhritIcyFT2hUzgZ7B3uht8UyHkukZvzpeMyFiaczYZtdFzhe1cGSmDuCCb2MXB2AkvK19l8kyJmssHPpB8\\\\\\\/0sxLvg1L\\\\\\\/5qf5\\\\\\\/i56NDnatJLFo6nKroXN8zMDzy2Qb4mY95yMUdBBuWWw9MfEefhspU3q++C2ZnxKhAdwAY5bUbs2ElfExeSaHS1FRbwDTjDIMaN\\\\\\\/AeyB1bPj0\\\\\\\/iDfuS7Eu7ZfVM0IhjiehEfpl3CRmYoYjPafR0cwQUZwy2l6emVCs4JbDCRwdiDmRGF93t\\\\\\\/HAqvuzZgB\\\\\\\/cT09AJICGatXiW1hdA2tDsG2O3XrV3Ri5iiuYJUQWJTkiGF0M78Dl574Wv0ddpU7h02ZDw9g9YQMf7GnUjXhHgL1dtkH7HfzfaU90\\\\\\\/OdwbPPbUIjC7RcmbFUTfrcNl4zkYhBnWReUHg5nUD12pf90rpUCe02VlLnCu+AwpX\\\\\\\/o+TOcndLPEws8fjqIx6VNCqKDsnjPFfIQO450tYM1KRZzcRJu2z7cs\\\\\\\/mzbz71N1duj6\\\\\\\/ciB920FTZfCrEl\\\\\\\/3C8NybnFUgETD0EFBJXGjnFGKTRAs3\\\\\\\/5CGm\\\\\\\/PlXWlnoD4qUSJXvbgY4mrdCCbKKHDIeT2nyZsZY8nEk8fMYHgRdh7y8WBD0W1xsJj6\\\\\\\/x2Y69vfN0HJ+q1D+dp8bQNBj+OyNoSOLkCDn+UZjSpcNS6u5lB7QHegV5SjoHw17gbiyOpDVUMGXY6CNrnZ\\\\\\\/ACoze6De59dGTYhZ5j+hUYG92v2wdod5mEVN38FeU5WbzbzoDw8ff6UYcOCMY9\\\\\\\/jbzxxqQ8Hrf209MIgSsUAaNKdtA8ROZjtcxaZg8h5XACVn0l3rBLtU+gBe3Zvr0avyA2gETgYHg9IhWMX4GY1kSDd\\\\\\\/tfQ5KGdPNK+kU8NK1fLfDAxSEAWx6jn28bzIZ3GMvsVJIolqXOaPCB0xBAsm1Blak1zjIbEABaIUQaiBge\\\\\\\/OEcEXBlddekaRonI6L7mSpIzIz5dRB7zHc1t75FUVnKfCp86Ih4E2o+ua9LTZCdXQBo5W\\\\\\\/yTmFFyaL8kk1kFRqI7uJCcU0rCkhsmTq6X7lusKcu7Jv9iTPdaZTlkt9uziV0uz9YbWO6JK\\\\\\\/aN5DLPuNQvpP4nj8cJJ7VMcFmdKNpVDnhLkSwLTJKYn14Q+uMJxIcBHKrjEj3DfV5V5gh\\\\\\\/o5w87WjqkTH18JRlY\\\\\\\/UBJwhMqban9JUTnyjrcNa80BoAlsxhDPGlpME6AUI5uu6EAX4+CAj2Q1UAo8fQkd7UHEFaIHkUhk48YnQYvMvgd6TL6YjoL4nQ8MIU6H+ueCC7LBdf+ckMJ1lMPMIXtN8gho4NHysY8NgQukL97dDximgZAgKmPPzMu7psEvVfaD0zYh9MDItmgJeG+wYLNSuqGu32+r6Q2kWvUBdST9B6lcDiWnl0WCJnRoqPFZqdfNFOURG+k2yofoW3gK+JEBjTt7GOmKqdHCSkqBZoqTlwKDe7XPrr2JLOqWH6ZkVMAIrb41pOs\\\\\\\/0ibPlXcQnEyIo1f1AWYJ4Ldi0Ib2C4V+FJnuDpmbmuPzSdcNRFhkXOS8crSzsTls2nq7PQTSN9\\\\\\\/hWfs9zgyRKJ\\\\\\\/bUQ5FOFICYjHY5aNcuTzLPur9BLKXsqacrGFaENDG\\\\\\\/o+iJNoHb1ASk3WraZPNZrOY6Ovm496uCuBAzbRf1CXLuujrgK70li4gLtjmvM5b24U\\\\\\\/TSn0g17HM5EHhJCbS\\\\\\\/VA4hqAmLFXBEQ3EYk20Udo0+NO+3t1UHGx3rjgQxqeG0nMBep1FSYlfiEisL+9nSHS9\\\\\\\/gB7RK2AfDIFEvGkX\\\\\\\/HMbrIMeyQDd4WjStNFmJjlFeVpLvg6RXSiEQaGjBvcDg15xYNBULycTnWrHE1fJfl+0xebtQIRK40ugKVlF0ij6hm3VJQrsOj37AU29Ckv7gjjXSH8St7fG2oYGzFPr1msxrDJVh6c2ITBklkV84XSwgora39GlMEHsB19s\\\\\\\/gY4DdjjbYW9P+GVARSjsFcCa71Y\\\\\\\/r+hb7cNfnMxy3peX2APOmMpY2kk83A2mPjPM7P05E+uGD42xdCy8JKw3NtqUn5AQcB35LTOqiEmasMkmyNKgZ6Ft435TXJKAXBZdoy6onFhSgeDOso9j1Xxfj\\\\\\\/JiqR62GX4GsMxKCY+aLZOXggizA0hAfeXODtsimxoq\\\\\\\/Kfeksm5gUmEgW5PfWPwkiiTcJfFFad4f1UIBdt0qA54v6y0CsH0w1PUgyhrzOChYnBWrZ35uC98UusDTpalY8rOuHqOvzU4y6yJgdkA0h95V2HEk0tCddo71BaMy+s31ZVvxfVy2uTt05jr4WDEs3NodHcxkHHkqQAEE9937OBxQDOTQ8toKduAut9GOz2e3nraEaJUxCrZAfANBv6ZSl75VcdfObvJFkvSjGC3QYlxZgeg+z4NUjSba4QynIlbtbm53z2\\\\\\\/GKdoD2\\\\\\\/Igb9QEocSdtXWKDVXdGwobUPBTAp\\\\\\\/8YLbcowV3\\\\\\\/wTPupIoAvN3gGSEohVGENG0ZyplL5hID7pxOWXKlTosAz5umCz+rLPzLfuwoT2Jsa8B1ksuZiK8q1Kvxlmh\\\\\\\/US7hkrA4xV2QRYCImvi1JZTAiHHkbUPE5U12lEAf\\\\\\\/+ZZ+YYbJQd\\\\\\\/56jzG1ZjAB1z4J2Nd7E8ROmQtfe77kMbKMJnX7nemvS0IYFiAKQkIFVQq+fsQVYwp4trbYyUvqEO1e7csDOPd2Y+eC65D\\\\\\\/Ru49hjE+qxstLW7CTbVwHNe8gTAK01pzIIrxBAsaFTHHYGC8wxrg0Oqgu7tEVzYjVesHICotThJAw77vWwXsFWhzYK\\\\\\\/FJsobUIzdIoggkWCXODBqnKpQ\\\\\\\/OTkFjTryDG+slLY8XQlmKsuBUVIPe6i+ANvl3hHW99451DF9lXoBE88Drf0dl9wNQuuturblXjTLPO3PZzNcoKsu96O6I+1TQ8KnOHg1yVc1DU15p4QfQL\\\\\\\/BAQeYEbnuacJ6CyeumYIbo3a9p3IrHZPU6xR3wo6pQw4L\\\\\\\/6LHG0pptPzF8mfthGW+qqrUFO\\\\\\\/LR9+9UGw8G9mqkAtrbJwTDXqTwDRvuhQ1tVkyU5mjty4IYhTStMwwpAmMnTOvsfyTqp2DAzlMgS3ahbWg1U\\\\\\\/8fFOi3o9\\\\\\\/imJdJ4X6R8tacSJEP+19tg9XQyjuNr2FsJHCdmc3HCWSwQtv7S14pxbaRwbW1XWqFX5w62m0647UE6BTEoOeGSuo8+X8kbP23ny511EYEb9\\\\\\\/ylgg9ASKbAlmPX4zOTuvNzN2aS6WPKwf9GoidJg\\\\\\\/e9VLmbkpkSWaKB\\\\\\\/FKlCPSAbGrjP1kWWfcM99+v6b4sD3dv18kyoNPtU\\\\\\\/PEf4kqx78+Ir+75vmG7BDfVMA5MTrFd7oWEAR4NgzMDA7bAokBKpE\\\\\\\/cm6QnN5sEC8XoQ+YOai7ukskQFuyIGBJVUnyAA3d5MKFXKOjbhhRVmGolO9OVVJs8sS\\\\\\\/u06PbwpcLhymo9gJ6tkg0wrKWdcjsuL9iNjuyb2ywTaVOCGqCoCCpZHB4uAI\\\\\\\/Dw8Zr6IM9cZcl44Cqen6UM1u\\\\\\\/kdZi0uzBBvtmFcSFDbL++3JGnOa+jPI0vm4hB\\\\\\\/Kdf3YSDl0YRoKlA934TKbJhjXT5rxTb42HXsUZVPneir6HambPCsyFEjWyXJVwDaM7TfINO5sHNI8X\\\\\\\/YjrVmnGeiXy6pzwvPJ\\\\\\\/F8FxuS6kQM9Gi9cAiCho7Ql2dsdxId8xRVcxEORaxZNnOySG8Vi11TKlI7Z8W9FlxemVMxZnYqmI9UUJNJ3yiNue9SVbLkLxHW2J81rUK7Fw5zDDVuVWCESfDthDqvUOuhAd5zKwy5kUM7Ywa9ePLAyHdS8jjqEgT\\\\\\\/0DtlwfGMBw2rnmck5FMiAcsOCu2nbbHgex3kwt2a+wflF2GqzP7q+nPlkhhFoPrsoTDeUJKJ6HcOUixwrqVIhnfjYvNbllgY2885cPJw1zZYw7bExg97HvvNM2rr4hOu5reMXHkLcF6NHd5mIyzL2dgl+bAhHxFwHNZgtvQhFbrVbu\\\\\\\/313IzWB9igYmAWJUYnKq\\\\\\\/R4Ujl0PXohpRenNvQ2BQoaSqzS8kTC\\\\\\\/G3jP8Bs3jDKiFaT2aQXIscIP2UkiCgCMlUlN13HAn6Zw\\\\\\\/o9dptwHQ67HtutuANsGvFFhNojgBX1SRGEV\\\\\\\/L3ze8o6JzXcQmk2B5AKBdB\\\\\\\/Z5DoEF+jzoY2Q8o\\\\\\\/H0qA\\\\\\\/tK9vAZ812jyp1Pb\\\\\\\/vqkWmDS+hRyZUBBzsNoa0ahbr3URlnyu7dFRCCKt0gv2da9iw1MA96coTYXw1ALgbxRhf0Erlkx0Xzya\\\\\\\/4dBx0Aq\\\\\\\/XPUBL7N6EeXFnrV5jh644MBtx56Ofu6UrEKk9FnV9GXtR3Dq0o83FIBJxgT2fc\\\\\\\/IKa0e4bjEvg5vzu0DxKhxUYUUM+KKLzGTNZuE+gtVNzCLlWlKYIwUtmBEA33dM0pscjJcVLfn7ZO\\\\\\\/wAjU+vaGfa3+MDhmPjl+orPtUMQ6gHBqnqDBvxEvvsbtXmKuSxd5gkSJMvbGmFBmrvB3UMdHiSP\\\\\\\/V9LM\\\\\\\/o7Sdkz47daS9cyqbE5uHnyTG0dNliYKkK72deQpxQafJ8WR1L6P9EEIzw\\\\\\\/mH8aXuq7z9qSOEtcHHOZFgR4dBclhWJ5NrPjcH5iq2oMG2pk1cD5x5yxLPDrjl3ZH8guZn3xeBJyMqnhdwBb9fk\\\\\\\/faMEkp6ppFmDUiV7vWK+tC1hZd5BRR5J9HUABU3xGkaKU9T5FavpnmgjEDfaQ826PKeR\\\\\\\/AmPsYKq3kdMfmjcMXJts\\\\\\\/OL3C6DMVWx14RVZZ5U4N\\\\\\\/hXagp\\\\\\\/jQKIpin7WrlU1gX+KNpW5hsfy\\\\\\\/Wv4n5HNmgXr4QuEytzl49slDPyq7eXmbiRIhkwBW+Mo48LSaILprPnak\\\\\\\/ooXguKbNjtpkYC08y99A0i8Cg45Ts5J\\\\\\\/EMuRukop3moSDN9RS9nEybTsPltoz5l\\\\\\\/qhJXHVp0HoHT9dJR7+VJw8JLtLhm7mxW3PYCclBOT9uTWofX3iR4CpAwJiZ4x0zfKYNpy7Zi7v7ZJSXNptcwd67klaR1iLax2z82KVUXNkwRa6\\\\\\\/P1zkaKIh6C9XEkMOY22Vi9yzqNWYgexD\\\\\\\/eNsc7LVW0dwpBWDZIEn2Sh3cjfjghMRST6J19exVTy889udQ7\\\\\\\/\\\\\\\/4YHj4Od3OLmcFNn6a6MdA3wcORyibNi\\\\\\\/nZq\\\\\\\/IhP79EUAdg9Db1h0Gv1P0tXchvjPnzCq9oDIWtWx9UUhBrumnJJ9\\\\\\\/keXV4rbdG9vSzVRi45FMtmeh46Y1i3+FiqveLsuLGYA3RadFoKS841ROgi8ymlJFK2Q\\\\\\\/QYlOYcWCrNpLEFi73YZvUdN7Hr9PK\\\\\\\/JD61HIkxGG\\\\\\\/\\\\\\\/ijoZjuePvb0TS\\\\\\\/o+7ykYh7eySAQCPg21kg7T1sUpxMn8YQsa4az9Vjs0vshxR60jH0CpKB5xRMudoyG5REJ\\\\\\\/H32otpOHR8S\\\\\\\/pU0\\\\\\\/ybE63Ce324+zMh+V+4k5fixnNltVvXUWZ+G2tLHL5hDefFj\\\\\\\/8sOyUnVQSeFfb3zf3Pcm0XfXxqB85exhNtv8sLafeyTH9kD0kh3o7lWNpFQyuVADD23O7EjRENWPIca8YKJJMKTlOldkgGFN0+hJ4Kxy\\\\\\\/nx\\\\\\\/YRMm1IoU6\\\\\\\/Nyhm6rvIrsk3sNSyEJC1y5vIVEhNSf2zuy2gVKHivMZlUEYP3jDwgqueWNcYSV48Ej+5AGEChSPBCcHS2MuwQ39f9RyTVnJqtqkyk8RTPW3Xt\\\\\\\/oV+2oGb8Qh7NsZ\\\\\\\/\\\\\\\/6EBLIuHKAfXsPLA6zpADoiI+twMUP4iwjnTjykMaDo\\\\\\\/ZA\\\\\\\/00I9HnMOVn3tkbYz3rEq1RU8Gv3SpJvEoURbCariGoVJsCakp4H0PHOl1zM0MxpWr9ZFV3NExIvkzim4qbjiWRsN7zd2qllbUrQs\\\\\\\/NsJ8kNqfhZRd5Ph+WsaDz3rotmLdqZEDyXGqMuXb5jwM80UVp3kraKBZYjnyOMwls87WL01KV\\\\\\\/g9OJCbd9PH7yMOJAQzSE2D0TXp\\\\\\\/Ati1FEq+mjMjKQO5UgRoiNXBiuylW0LRjUd1xlszJQQYF\\\\\\\/E4XfFR2\\\\\\\/GK8ZlbcKSC1UADPjsAsv3aa+EwslDm2bEBoRggdvP8TK32nxek5U3ZlM+am9Q74WMpAHxkxV7000TY1vLRij73QcoKPdYb8U4aEprJZz7ZMqF1RgN8WytGUlDZhu4UtLj00C8r\\\\\\\/bbLJbhcNW\\\\\\\/OEC2WfxMJtWhO\\\\\\\/AdU2AJOolfV1wqrDzpQnO8xxMsCpAo+XjzLRc3Ku9kj07i63sn7xCcw9wQpLJHUsKguLOKXHYrHqR+MmWyXb2DSidEAXEath9vgpWMA3pUR4VWnZqzVUnZdqGHpsXv3ez\\\\\\\/iHZ4+0+lIVyrbLSBUhcxuCY8HK4htHkkDwoDTVOrh3oWtYMmL6RbwJiuoEVHsNj8je177OT2mXtGHM\\\\\\\/Z52a7LxVkgYFWhiXc\\\\\\\/F6qgzVeADtgg4XRdTLMPFjt+AOrWCVd6BNOvvSdbO2o+WqTP5s24hkl8X0FI8\\\\\\\/m0Gk4vSanKQ8\\\\\\\/na9Hz0xSsNX1ZZ9JOavIRu3YsslSFQQI7hCY39DppiUuwAt76e+\\\\\\\/JJsV5WbdmgV6cSQhT96EeRVoE7QYmquKm9IKuaUUtwO+a4BnTqGru9iu8tzlDXD1TjmN7Yls0HDB9ie2RqQQws8jlIJwrpoWIwTvgh02CN1++31u8rR8CGuqaNV4Fk9DU8Dy\\\\\\\/Kos3gZ+g6IB\\\\\\\/NclobbDvftyiBG3a2idXz9z4Tdr15FhJXqIOFnfcjnCfVwj5xfyStsoZtAXGwy6tNenZbtTNhyScHPG6fIHNnAHn1xyhE9KhaB3AahVnkNMlpTp0mK6IXzJYKi3j8ZC1bzFBe6enOqn+kMWw+1SjyNNZ5U4GSpxLmZtx6qormpSB5grdVl\\\\\\\/4AjQQVaO1lqxoVu3i+YVxXrcMr1L+HmLFVo2PTIdFSEyCgV2YoDqC78A0RpyXQucwN9MoDnIIxsquc01qZB2Oh9s7KXOugD51re7MEMZeo2uPyqvYdoLOFgLkwzD9kz0EKTy0pxxN0bWgsnGoZ+hRdtrVldaUK60y9+BavFBndfvnigBd90ifHvaN\\\\\\\/oOnTjanUZVkWUWC757ofHeNMRelUT6wTUMT5TUU2eEwR8nKdeTY9dl4GXiQqzkaRwcP2JHnFaitX\\\\\\\/jD+5iRCVhk5wOAYB8hnmjIKMfhgDHO7kA8d6hn5EU4WFKjkVqYao+zPvL+Gcg0zW\\\\\\\/FvkIcKl5fGwqoOtOymX1eRKWhUlBchmd13ozMVymhgai2tfDjEjBFsLqJmWodtCMdJmOoSZXJsIw3da9FT8EYK7VtOyhjh8ErBZbHbsI23gczYyIs4YkkqzhM6BU0n28AVao1JESNs8mwEyMFqNGJc00DA+mJ0v8dOlmhHvK0kVCrsLpei4Hc\\\\\\\/d3bAw2jhven6F\\\\\\\/yCJ8IGfDOCIgyD0SWhR1MrppdYKya7DLJWAU0vksaNNf2OfX99kaxAAjqufQzbYiFmz9AGFizfQOlU5blIExanb6wv7tOKJoJjvRInztiCP4wAHGQGqtjYYyb3gV9xllVs+3T9zPefBP8UXTtQxL7Qcfe6vv5R\\\\\\\/KnOPmXDmb9kTbvVQl0Gh7U0YrBf\\\\\\\/EAmuiQbCkwNC2fOS4qWHsMKIsGS0k0MOtEWPJl\\\\\\\/bl7hzd74qWv2aEiS+88PAn9SY8Ffr7Da5+UQwUWiWvJ5Bkz8R4g2VvDGg0JLSBx7BmzgwkuVKTESX7Nd9KQc0dF9c1+XRuJBL+VcGsStP\\\\\\\/mHWTN5atLXRKatRrfUGPoa07dnxe+CN\\\\\\\/WsFXLT6Z\\\\\\\/SkunC1jy1FMKOGfj\\\u0022,\\\u0022iv\\\u0022:\\\u0022CU\\\\\\\/MA8lJC5eLMAEgzs8Fhg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022db18ea8448ba7397a295a21e8e92cf0abec6f0d860c838672f38acd9e0f8f3f6\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00226j8kwx2HrkX1L\\\\\\\/Ne05TGQZh5UfLAiX6Oa2kMIkpM6xyQJs96uFmh9Nubj9MKhzfUc64a5ZM9q\\\\\\\/i++z2wir4Q7htSHk+6+Zu3NrPIc9tW\\\\\\\/AY4N7NFOsPKnyZLKZfXVjpVGvPON2roCZpB+PttMUF8ilIW8F+VCsDx++EW8ARyTl7YUP5IzImlM9vljcI130ys7LyhdZac2lA82kp9iHRCFeBTK59wkoDdRUSao++aDrg81tBeTtjernsoFC\\\\\\\/G0yNexs0Lr2CA4BVQPRhaCF\\\\\\\/InuDqDtO2kc\\\\\\\/sl+RjK04yXcicR6h5vq5\\\\\\\/Jrl0eyqtvzpzwKZCpHFDZCqRAB0ubdk2AO9dMGtfJpcScQNXEQWFGmVpYOX1cSThFWngymLNxBhvPSIXLuipUsKlpCc2vkcjGlfPca139C+BmqaOaPQQP9nOPtA67HlxTqZKKV1OM38qUr5v2WZ+vFJ0DK1hE07yvNebpQNls3zwtPGXmjw\\\\\\\/6OqFJ9+xBS4SZ3uFn5IZ9f79dKPHwhs1ecnYRluBfedbnbIdxHLZyXDz5eS0aDpqci8FzqOhUGMQg8X5Pb6AnMaJ76iQZhzaHwJ9kM9tEcJE3Li2kgtQu\\\\\\\/2pRPFhIdeZ4+q8rg2y9cY4aQS8Pz+41IPGBVuhjDIVZcg3g5ISBBJVNsJPO9wU6zUNAypmX\\\\\\\/szywt9fMfy8N2lrBo87dK8iSMxpIKR788bRsRfK3R3Rmj+dlX8nDleCxtmdUsIEzpzxnrLPnoyIRwA4YgeVj+0k2egOMx7wHURTmcJgDpODVF0sDSolIM1Pr2g8vjQAae3XRFuPp6eVvGFKs2NtoBlaIKDTauONORVC6WHPdPgjFbaObYv5Cab6u0M5mJ7fIg2e8m9Z9t4Z2+WB42WIuY\\\\\\\/YQMk\\\\\\\/wkbzD0+s22vRS31VqEJRp4FwhFAsIZ1w\\\\\\\/iXsfVpFhVd\\\\\\\/8oDNUaKCBnCtJtl83RxMO1kxnm94pnPB3YYH4xs1t0+CPq+f\\\\\\\/wRlSO5T6+8dL3Ct2CGDKQzKK2HEehykXYAuqK4nald9W9qZcrp61Oa3ir6q5pb2OFBlAYD39\\\\\\\/TJAecm6fkDxW+Bml4pidXkNaWVdq1y9Yc6RWFYxFuIBuccC0ZbpQJyDiYG0o2AB87RkMny2ImxPTlhBiFMgz45LnxOIjCnGtCwDzaKlwHXRujS9KCZLPjcdGr+8wusoewgfi6OuTYc4Ir\\\\\\\/KKHBX3hoGeGKehUCjNX4HtU0ysHfPyCmrQDWzsQpTcsetcp7lP+k2p1ZavmPQ3WSLQNpvMPTG8YkWmoq7BwMuXQ2Hr6q60VY9cU2f0Sc75U8LvfdxAEhUC\\\\\\\/EzQHK6Ods\\\\\\\/FM3kkmxHrBOn8AJ4\\\\\\\/61an\\\\\\\/EooVmlQ8pCB6noNWjAr8ego14c8UBgcDxryjRZlXhk9M6irFRwU0CQy9aR5CUg07otSMceskVWx0Y17vb5I5I5c8cA4V9Yv7EMQzdCNP4eI3\\\\\\\/mSBevB1bdJrmIOsFI51SWOV8wbm6qbIFufnaPeSKdtTqpV6Urc2YPSFpzE5OTek22wfjVwFJfk2A9HhlvxE7\\\\\\\/okSIS8wOhcWTy6Pj90xWY+XD34R9U3bVesWHm\\\\\\\/wBQxJ7cAoHDC\\\\\\\/9sOTJl3XJzk3qJBreEPVXO5i7Vov4Jg+ZFJyHVJVvbmUMiBG+9MJ6jz12MmY8YJSBndr5V35ZrsKkAAcsa1XOdA+d19+OWPS4DSr95HFCn1rfp3Zftf+oePXnYiaGuc0lmK3It2N1fX13Y4VlTe5PpWE6TZE5HanV2ZijiXOAMmSY36z8TYm2pUy5U60X1ME9c6WnLUIfmBHsfYL41rvNLM0I8L4StwLffE9mYdP2Zlc5oT6vIgtkmm01Ea1GmXUozp07GeXpLgNr0nWSEHbVWF4SGO0n881vjYMaRBCCxC5oDTFKIRnHWgeORK3wtkBdhBwGBVqtgB8Kfs4k7HYJpnfnZ6s8d0XlkT4NIq3pHDxC+zSnKyeXWU3g26IfkSgl2jz0+UTP8J3Jp5e15Sw9GY0iNArNxmjZXDqxYJEF8=\\\u0022,\\\u0022iv\\\u0022:\\\u0022hFx7tBdeaE2WofPoUt\\\\\\\/BQA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220f0ed9531d3e9187868947f1c5a90d0249b4519683b1e9be8b173e06031f0999\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459481 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/6/993" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Q-_F8JG-8RwY4_VyziObf6XTZgRPxzourL0nfsqOzCI" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/6/993" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ipcoxjl8JLqPjobAYZVDL1kuN5uGjBioSpBk0Ce_Osw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/6/979" title="Identification of Cilia Genes That Affect Cell-Cycle Progression Using Whole-Genome Transcriptome Analysis in Chlamydomonas reinhardtti" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/6/1003" title="Retrospective View of North American Potato (Solanum tuberosum L.) Breeding in the 20th and 21st Centuries" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459481" id="node459481" data-pisa="ggg;3/6/993" data-pisa-master="ggg;g3.113.005793" data-apath="/ggg/3/6/993.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Widespread Selection Across Coding and Noncoding DNA in the Pea Aphid Genome</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Ryan D. Bickel</span>, <span class="highwire-citation-author" data-delta="1">Joseph P. Dunham</span> and <span class="highwire-citation-author" data-delta="2">Jennifer A. Brisson</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">993-1001;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.005793 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459481" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ryan D. Bickel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>University of Nebraska, School of Biological Sciences, Lincoln, Nebraska 68588</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ryan%2BD.%2BBickel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bickel%20RD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARyan%2BD.%2BBickel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:rbickel2@unl.edu" class="" data-icon-position="" data-hide-link-title="0">rbickel2@unl.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Joseph P. Dunham </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>University of Southern California, Molecular & Computational Biology, Los Angeles, California 90089</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BP.%2BDunham%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dunham%20JP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoseph%2BP.%2BDunham%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jennifer A. Brisson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>University of Nebraska, School of Biological Sciences, Lincoln, Nebraska 68588</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jennifer%2BA.%2BBrisson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brisson%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJennifer%2BA.%2BBrisson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/6/993" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459481" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459481/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/6/993.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459481" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459481/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/6/993.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459481" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459481/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/6/993.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459481" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459481/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/3/6/993.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459481" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459481/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Genome-wide patterns of diversity and selection are critical measures for understanding how evolution has shaped the genome. Yet, these population genomic estimates are available for only a limited number of model organisms. Here we focus on the population genomics of the pea aphid (<em>Acyrthosiphon pisum</em>). The pea aphid is an emerging model system that exhibits a range of intriguing biological traits not present in classic model systems. We performed low-coverage genome resequencing of 21 clonal pea aphid lines collected from alfalfa host plants in North America to characterize genome-wide patterns of diversity and selection. We observed an excess of low-frequency polymorphisms throughout coding and noncoding DNA, which we suggest is the result of a founding event and subsequent population expansion in North America. Most gene regions showed lower levels of Tajima’s D than synonymous sites, suggesting that the majority of the genome is not evolving neutrally but rather exhibits significant constraint. Furthermore, we used the pea aphid’s unique manner of X-chromosome inheritance to assign genomic scaffolds to either autosomes or the X chromosome. Comparing autosomal <em>vs.</em> X-linked sequence variation, we discovered that autosomal genes show an excess of low frequency variants indicating that purifying selection acts more efficiently on the X chromosome. Overall, our results provide a critical first step in characterizing the genetic diversity and evolutionary pressures on an aphid genome.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/pea-aphid" class="hw-term hw-article-keyword hw-article-keyword-pea-aphid" rel="nofollow">pea aphid</a></li><li class="kwd"><a href="/keyword/acyrthosiphon" class="hw-term hw-article-keyword hw-article-keyword-acyrthosiphon" rel="nofollow">Acyrthosiphon</a></li><li class="kwd"><a href="/keyword/population-genomics" class="hw-term hw-article-keyword hw-article-keyword-population-genomics" rel="nofollow">population genomics</a></li><li class="kwd"><a href="/keyword/sex-chromosome" class="hw-term hw-article-keyword hw-article-keyword-sex-chromosome" rel="nofollow">sex chromosome</a></li><li class="kwd"><a href="/keyword/selection" class="hw-term hw-article-keyword hw-article-keyword-selection" rel="nofollow">selection</a></li></ul><p id="p-4">The pea aphid (<em>Acyrthosiphon pisum</em>) is an emerging genomic model system that exhibits traits that cannot be studied in most classic model organisms but are common in nature, such as polyphenisms, cyclical parthenogenesis, host−plant specialization, viral transmission, and bacterial symbioses (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Brisson and Stern 2006</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">International Aphid Genomics Consortium 2010</a>). The pea aphid genome, released in 2010 (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">International Aphid Genomics Consortium 2010</a>), was the first for a hemimetabolous insect and revealed that the pea aphid genome harbors surprisingly high levels of gene duplications, exhibits lineage-specific gene losses, and shows evidence of metabolic coordination between bacterial symbionts and the aphid host (<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">International Aphid Genomics Consortium 2010</a>).</p><p id="p-5">To date there has been no genome-wide analysis of genetic variation among pea aphid individuals, although studies examining a small number of loci have been performed (<em>e.g.</em>, <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Brisson <em>et al.</em> 2009</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Ferrari <em>et al.</em> 2008</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Simon <em>et al.</em> 2003</a>). Studies in other organisms have shown that we can infer much about the processes that have shaped genome evolution by using sequence variation (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Begun <em>et al.</em> 2007</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Li and Stephan 2006</a>). The emergence of next-generation sequencing technologies has enabled the investigation of sequence diversity from low genome coverage data for affordable prices. Here we report the first genome-wide analysis of genetic variation in the pea aphid using Illumina sequencing of genomic DNA (gDNA) at low coverage from 21 individuals collected from natural pea aphid populations. gDNA from each aphid clone was individually barcoded to enable us to assign polymorphisms to specific clones (<em>i.e.</em>, these were not pooled samples). We use these data to calculate levels of nucleotide diversity within the species and to assess selection acting on coding and noncoding DNA. We find an excess of low-frequency variants throughout coding and noncoding DNA, suggesting that the majority of the genome is acted upon by purifying selection. Our results achieve a critical step forward for the understanding of aphid biology, provide a valuable resource for those studying evolutionary and population genetic questions in this unique model system, and widen our understanding of genome-level population genetic patterns in insects.</p><p id="p-6">Like most aphids, the pea aphid has a complex life cycle (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Moran 1992</a>), with unique hereditary patterns that are predicted to impact its genome-wide patterns of diversity. The pea aphid alternates between periods of asexual and sexual reproduction. In the spring, asexual females hatch from eggs that have overwintered. These females produce genetically identical daughters (barring spontaneous mutations) through a modified meiosis that lacks recombination and segregation of homologous chromosomes (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Blackman 1987</a>). Multiple generations of asexuality occur during the spring and summer months. In the fall, asexual females asexually produce sexual females and males. Both are genetically identical to their mother except that each male has randomly lost one X chromosome (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Wilson <em>et al.</em> 1997</a>). Both males and sexual females produce gametes with an X chromosome, which through fertilization produces an XX asexual female that hatches from an egg in the spring. <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> follows the X chromosomes through the pea aphid life cycle.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/6/993/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The pea aphid life cycle. In the spring, females hatch and then reproduce asexually throughout the summer. These females are diploid for all chromosomes (represented as XX, AA for the X chromosome and autosomes, respectively). In the fall, females asexually produce sexual females and males. Sexual females are genetically identical to the asexual females and the males are genetically identical except a male inherits only one of his mother&#x2019;s X chromosomes (X, AA). The sexual females and males mate and the female lays fertilized eggs that overwinter. Only sperm with an X chromosome are viable (X, A), so all fertilized eggs are female (XX, AA)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1927319261" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The pea aphid life cycle. In the spring, females hatch and then reproduce asexually throughout the summer. These females are diploid for all chromosomes (represented as XX, AA for the X chromosome and autosomes, respectively). In the fall, females asexually produce sexual females and males. Sexual females are genetically identical to the asexual females and the males are genetically identical except a male inherits only one of his mother&#x2019;s X chromosomes (X, AA). The sexual females and males mate and the female lays fertilized eggs that overwinter. Only sperm with an X chromosome are viable (X, A), so all fertilized eggs are female (XX, AA).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/6/993/F1.medium.gif" width="365" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/3/6/993/F1.medium.gif" width="365" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/6/993/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/6/993/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462904" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-7" class="first-child">The pea aphid life cycle. In the spring, females hatch and then reproduce asexually throughout the summer. These females are diploid for all chromosomes (represented as XX, AA for the X chromosome and autosomes, respectively). In the fall, females asexually produce sexual females and males. Sexual females are genetically identical to the asexual females and the males are genetically identical except a male inherits only one of his mother’s X chromosomes (X, AA). The sexual females and males mate and the female lays fertilized eggs that overwinter. Only sperm with an X chromosome are viable (X, A), so all fertilized eggs are female (XX, AA).</p><div class="sb-div caption-clear"></div></div></div><p id="p-8">We use our sequence data and a novel method to categorize pea aphid genomic scaffolds as X-linked or autosomal. To date, the pea aphid genome lacks a fine-scale linkage map. Without this resource, it has been impossible to assign individual genomic scaffolds to chromosomes. With our method, we assessed the probability that each scaffold is on the X chromosome <em>vs.</em> an autosome. The pea aphid’s unusual sex chromosome inheritance pattern means that each sex always contributes an X chromosome to the offspring of the sexual generation. In contrast, in well-studied XX/XO systems such as <em>Caenorhabditis elegans</em>, the female contributes an X chromosome 100% of the time and the male 50% of the time. In these latter systems, X-linked <em>vs.</em> autosomal genes can be differentially affected at the sequence level by sex-specific reproductive success, sex-biased mutation rates, sex-biased dispersal, demographic processes, and recombination rates (reviewed in <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Ellegren 2009</a>). In pea aphids, both the female and the male always pass an X chromosome to the next generation. This should result, theoretically, in both the X chromosome and the autosomes being affected equally by these processes (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Jaquiery <em>et al.</em> 2012</a>). Indeed, simulations demonstrate that the pea aphid’s unique X-chromosome transmission results in similar effective population sizes and predicted levels of genetic diversity for X chromosomes and autosomes under neutral evolution (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Jaquiery <em>et al.</em> 2012</a>). Here we test these predictions on a genome-wide scale using our predicted X-linked <em>vs.</em> autosomal genes. In conflict with theoretical predictions, we find an excess of low frequency variants on autosomal genes relative to X-linked genes. This pattern may be due to a number of factors (differences in recombination rates, gene density, gene classes present, etc.) that potentially differ between the pea aphid autosomes and X chromosome. We suggest that this pattern may be largely explained by the higher efficiency of purifying selection on the X chromosome because of recessive lethal mutations in the males.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Aphid lines</h3><p id="p-9">We collected pea aphids from alfalfa fields in California, Massachusetts, and New York. Clonal lineages, also known as lines, were genotyped at up to 12 restriction fragment length polymorphic (RFLP) loci to verify that they were genetically unique. Aphid lines were transferred to an incubator that mimicked fall photoperiod (13:11-hr light/dark cycle) until they produced males. All lines selected for analysis produced both winged and wingless males due to heterozygosity at the male wing determination locus, <em>aphicarus</em> (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Braendle <em>et al.</em> 2005</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Caillaud <em>et al.</em> 2002</a>). For each line, we collected from four to ten males of each phenotype, winged or wingless. Because males are produced asexually with no recombination, all males produced by a line within a phenotypic class (winged or unwinged) are genetically identical barring spontaneous mutations. We used a total of 21 unique lines (7 from New York, 4 from Massachusetts, and 10 from California), with winged and wingless males from each line for a total of 42 samples. After sequencing, data from males of the same line were combined such that we had diploid data from 21 lines for subsequent analysis.</p></div><div id="sec-3" class="subsection"><h3>Sample preparation and sequencing</h3><p id="p-10">We prepared samples for sequencing on an Illumina Genome Analyzer as follows. For each male sample, we isolated gDNA, sheared the gDNA by sonication, and selected 300- to 500-bp sized pieces on a 2% agarose gel. We then ligated an adapter sequence containing a three-base barcode and used 15 rounds of amplification using Phusion Taq polymerase (New England Biolabs). Four groups of 10 males with unique bar codes were combined in equal quantities after being quantified on a Qubit fluorometer (Invitrogen). An additional sample was made by combining the two remaining barcoded males. Samples were sequenced on an Illumina Genome Analyzer by the use of 15 lanes of 54nt sequencing. After initial analysis of these data, we determined that samples were not being sequenced in equal quantities. Insufficient amplified material remained, so we returned to the original libraries for each underrepresented male and amplified them individually by polymerase chain reaction. We combined them in equal quantities and sequenced each of four libraries again, two of them with 54nt Illumina sequencing and two of them with 36nt sequencing. The additional sequence data were combined with the original data before filtering and analyses. An additional line (the F1 line of <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Braendle <em>et al.</em> 2005</a>) was independently sequenced using 100-bp, paired-end Illumina sequencing. One lane was run for the winged males and one lane for the wingless males. Details of coverage for each line can be found in Supporting Information, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005793/-/DC1/TableS1.xlsx">Table S1</a>.</p><p id="p-11">Winged and wingless males were sequenced separately for each line for use in association mapping of the <em>api</em> locus, which is not the focus here. We therefore combined the wingless and winged male data to produce a single sequence data set for each line.</p></div><div id="sec-4" class="subsection"><h3>Sequence alignment and filtering</h3><p id="p-12">Version 2.1 of the pea aphid genome was downloaded from <a href="ftp://ftp.ncbi.nih.gov/genomes/Acyrthosiphon_pisum/aps_ref_Acyr_2.1_chrUn.fa.gz">ftp://ftp.ncbi.nih.gov/genomes/Acyrthosiphon_pisum/aps_ref_Acyr_2.1_chrUn.fa.gz</a>. The mitochondrial genome was not used. We aligned sequence reads to the genome using the Burrows-Wheeler Aligner version 0.6.1 using default settings (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Li and Durbin 2010</a>), and alignment pileup format files were generated using Sequence Alignment/Map Tools (SAMtools) version 0.1.17 (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Li and Durbin 2009</a>). For each of the 21 lines, we determined the consensus sequence using the following criteria: only bases with a quality score &gt;20 (99% base call accuracy) were considered. If &gt;80% of the bases at a position were the same, this base was used as the consensus base. If &lt;80% were the same, the base was marked as an ambiguous base (N); approximately 0.001 bases fit into this category (overall = 0.00105, autosomes = 0.00103, X chromosome = 0.00113). These consensus sequences for each line were used for downstream population genetic analyses.</p></div><div id="sec-5" class="subsection"><h3>Identification of X-linked scaffolds</h3><p id="p-13">We sequenced winged and wingless males generated from the F1 clonal line (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Braendle <em>et al.</em> 2005</a>) at high coverage. These two male genotypes have the exact same autosomes but differ in which X chromosome they carry. Thus, their genome sequences should only differ for scaffolds that are located on the X chromosome. Using every genomic position that had a base pair call for both winged and wingless F1 males, we calculated the pairwise difference per base pair between them. Scaffolds on the X chromosomes should have many pairwise differences, whereas scaffolds on autosomes should have no differences.</p><p id="p-14">Multiple factors could affect the pairwise difference value. X-chromosome scaffolds with high diversity should have high pairwise differences, and thus should be easier to identify than X-chromosome scaffolds with low diversity. Both autosomal and X scaffolds may contain errors that appear to be pairwise differences. We have used strict cutoffs to minimize these occurrences (see <em>Sequence Alignment and Filtering</em> section above). Furthermore, autosomal scaffolds may erroneously have differences between the winged and wingless male if a heterozygous base has unequal and opposite read coverage of their two alleles, such that one male has only has reads from one allele and the other male from the other allele. This would appear as a pairwise difference. These issues make it particularly difficult to differentiate low diversity X-linked scaffolds from autosomal scaffolds with high diversity.</p><p id="p-15">We reasoned that if we corrected by the amount of diversity on a scaffold we should be able to better differentiate X-linked from autosomal scaffolds. X scaffolds with low diversity will have low pairwise differences. Autosomes with high diversity are likely to have some heterozygous bases that appear to be pairwise differences, but at a much lower rate than the overall diversity. These two hypothetical scaffolds would have a similar number of pairwise differences per base pair, but different pairwise differences when corrected for diversity levels. We therefore created a corrected differences calculation as follows:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/6/993/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/6/993/embed/mml-math-1.gif"/></noscript></span></span>π (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Nei and Tajima 1981</a>) is an estimator of genetic diversity, calculated as the average pairwise differences per bp within a population. We independently calculated π from our pool of 21 lines. When we calculated π, each line included sequence data from both winged and wingless males and thus included both autosomal and X-linked data (both X chromosomes).</p><p id="p-16">For any scaffold that is on the X chromosome, if the scaffolds observed have a similar number of pairwise differences/bp as the average pairwise differences/bp of the whole population, then the corrected difference would be 0. If a scaffold is located on an autosome, we expect the number of pairwise differences observed to be much lower than π, creating a negative score. This correction also creates three advantageous characteristics: we should have two normally distributed populations of scaffolds, we have a predicted mean of zero for the smaller population of scaffolds (the X chromosome scaffolds), and we should be able to better differentiate X scaffolds with low diversity from autosomal scaffolds with high diversity.</p><p id="p-17">The resulting values fall into two overlapping populations of scaffolds, one centered at 0 and the larger population centered at a negative value, presumably representing scaffolds on the X chromosome and the autosomes, respectively. We found that a Student’s <em>t</em> distribution best fit our data. We used maximum likelihood to estimate the mean and variance of the two populations. We used these data to assign a probability score for each scaffold as belonging to the X-chromosome group or the autosomal group.</p></div><div id="sec-6" class="subsection"><h3>Population genetics calculations</h3><p id="p-18">The sequence data were used to generate Fasta files for all genomic scaffolds &gt;100 kb for each aphid line. For each gene the pea aphid v2.1 annotation was used to categorize the following regions: whole gene (coding, UTRs, and introns), gene plus 5 kb upstream and downstream (the gene plus likely regulatory DNA), Coding sequence (CDS), 5′ untranslated regions (UTRs), 3′ UTRs, introns, 10 kb upstream, and 10 kb downstream. Population genetic statistics were generated using the <em>compute</em> program included with the libsequence library (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hudson <em>et al.</em> 1992</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Thornton 2003</a>). Only sequence sets with at least 50 polymorphisms were included in the analysis for any scaffold, gene, or gene region (CDS, UTRs, introns, upstream, and downstream).</p></div><div id="sec-7" class="subsection"><h3>Scaffold location verification</h3><p id="p-19">Genomic DNA from females, winged males, and wingless males of the F1 line was extracted using the DNeasy Blood &amp; Tissue Kit (QIAGEN). For each scaffold to be verified, polymerase chain reaction primers were designed to amplify a 100- to 300-bp DNA fragment containing an RFLP in the females (primers and restriction enzyme are listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005793/-/DC1/TableS2.xlsx">Table S2</a>). The fragment was amplified from females and each male morph, cut with the relevant restriction enzyme overnight, and run on a 4% agarose gel. Autosomal loci show the pattern of heterozygosity in the females as well as each male because females and males are diploid for all autosomes. X-linked loci show the pattern of female heterozygosity and male homozygosity because males are haploid for the X.</p></div><div id="sec-8" class="subsection"><h3>Gene ontology analysis</h3><p id="p-20">The program Blast2GO was used to blast and annotate the genes identified in pea aphid annotation version 2.1 using default parameters (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Conesa <em>et al.</em> 2005</a>). Fisher exact tests were performed to determine enriched Gene Ontology (GO) terms in gene lists using a false discovery rate (FDR) &lt;0.05.</p></div></div><div class="section results-discussion" id="sec-9"><h2 class="">Results and Discussion</h2><p id="p-21">We sequenced 21 genetically distinct lines of pea aphids using the Illumina platform. In all, we generated more than 278 million sequencing reads, which resulted in an average coverage of 5.7 individuals sequenced per site (of sites with any sequence information) across the genome. Some lines provided more sequence coverage than others, with the relative contribution of each noted in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005793/-/DC1/TableS1.xlsx">Table S1</a>. We sequenced the F1 line to high coverage for the purpose of identifying X-linked <em>vs.</em> autosomal genomic scaffolds (15% of the 5.7 bp average total at each site). All other lines account for between 2% and 6% of the 5.7 bp average total at each site.</p><div id="sec-10" class="subsection"><h3>Aphid samples from California <em>vs.</em> the Northeast show little population differentiation</h3><p id="p-22">Pea aphids are native to Eurasia but have been introduced to North America multiple times over the last 150−200 years (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Blackman and Eastop 2000</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Davis 1915</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Sanderson 1900</a>). They feed on members of the pea family and are found on field crops such as alfalfa (genus <em>Medicago</em>) and clover (genus <em>Trifolium</em>), which are distributed across the United States. During the asexual portion of the pea aphid life cycle, females are wingless or winged. Wingless morphs are largely immobile, whereas winged morphs can disperse from field to field or even travel large distances via passive migratory flights in upper air currents, traveling as far as 1000 km in a single flight (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Robert 1987</a>). The winged morph’s potential for eventual gene flow may or may not be sufficient to eliminate population subdivision in the United States. To test this, we calculated F<sub>st</sub> levels across the genome, comparing 11 pea aphid lines from the Northeast US (New York and Massachusetts) and 10 from California. We observed no structure, with an overall F<sub>st</sub> value of -0.021. We conclude that pea aphid populations in the United States function as a single, panmictic population.</p><p id="p-23">Pea aphids in the northeast demonstrate strong host plant specialization (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Via 1999</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Via <em>et al.</em> 2000</a>), whereas in California both specialists and generalists exist (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Leonardo and Muiru 2003</a>). All pea aphids used in this study were collected from alfalfa, so our conclusions apply specifically to pea aphids that feed on this host plant. F<sub>st</sub> values for comparisons of clover and alfalfa specialists in France were previously found to be around 0.1 (allozymes, F<sub>st</sub> = 0.098; microsatellites, F<sub>st</sub> = 0.104) (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Simon <em>et al.</em> 2003</a>), indicating that, as expected, genetic differences between host plant specialists are much greater than genetic differences within a host plant specialist.</p><p id="p-24">Despite the negligible overall population subdivision observed here, individual genes exhibited high F<sub>st</sub> values. Specifically, 164 genes of the total 36,961 annotated genes (annotation version 2.1) had an F<sub>st</sub> value greater than or equal to 0.5, indicating strong differentiation between the northeast and California populations. All genes and their F<sub>st</sub> values can be found in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005793/-/DC1/TableS3.xlsx">Table S3</a>. One possible explanation for this strong differentiation is selection. The 164 highly differentiated genes comprised significant (FDR &lt; 0.05) GO categories related to localization and transport; oxidation reduction and carbohydrate catabolic processes; and DNA mismatch repair. Pea aphids on the east and west coast of the United States face different challenges from their host plants. Alfalfa in California develops under hot, dry conditions whereas alfalfa in the northeastern United States experiences cooler and wetter climates and a shorter growing season. If selection is responsible for these genes having high F<sub>st</sub> values, it may be because pea aphids on the east and west coast of the United States experience differential selection related to the tendency to disperse, survival during dispersal, or subsequent adaptation to different metabolic products from their host plants.</p><p id="p-25">Alternatively, genes with high F<sub>st</sub> values might be involved in reproductive mode divergence. The pea aphid shows genetic variation for whether and when it alternates between sexual and asexual reproduction or whether it remains parthenogenetic (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Frantz <em>et al.</em> 2006</a>). Pea aphid populations in the northeastern United States must undergo a sexual generation in the fall to produce eggs that can experience diapause through the cold temperatures that cause adult mortality. In contrast, asexual reproduction is potentially more advantageous year-round in the much warmer climate of California. Under any hypothesized scenario for local adaptation, the genes with high F<sub>st</sub> values are excellent candidates for future studies verifying whether they are indeed under selection. The majority of the genome shows no differentiation between the northeastern United States and California aphids. Because there is no systematic differentiation, we treated our samples as a single population for further analyses.</p></div><div id="sec-11" class="subsection"><h3>Diversity and selection across the pea aphid genome</h3><p id="p-26">We documented levels of genetic diversity across the genome using two estimators of genetic diversity: Watterson’s θ (θ<sup>w</sup>) and π. θ<sup>w</sup> estimates genetic diversity as a function of the number of segregating sites (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Watterson 1975</a>), whereas Nei and Tajima’s π uses the number of nucleotide differences per pair of sequences (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Nei and Tajima 1981</a>). Thus, θ<sup>w</sup> gives equal weight to all polymorphisms, whereas π is more strongly impacted by frequent polymorphisms. θ<sup>w</sup> and π for all sites across the genome were 0.0050 and 0.0045, respectively.</p><p id="p-27">We then looked at different potentially functional regions within each gene in the pea aphid genome to determine whether they exhibited markedly different levels of nucleotide diversity. Specifically, we calculated diversity measures for coding regions (CDS) and for the following noncoding regions: 5′ UTRs, 3′ UTRs, introns, the 10-kb upstream intergenic region (IGR), and 10-kb downstream IGR. The coding region is usually one of the most constrained regions in the genome. 5′ and 3′ UTRs are often implicated in posttranscriptional regulation, introns often impact translation and can potentially contain <em>cis</em>-regulatory elements, and IGRs can contain regulatory elements that affect transcription and chromatin state (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bernstein <em>et al.</em> 2012</a>). Values of θ<sup>w</sup> and π for each region are listed in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. The UTRs display the highest levels of diversity, followed by introns and IGRs. The CDS had the lowest diversity suggesting that, unsurprisingly, the coding region is strongly constrained, whereas noncoding regions are not.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462908/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462908/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Coverage and measures of polymorphism in coding and noncoding DNA of the pea aphid</span><div class="sb-div caption-clear"></div></div></div><p id="p-30">Tajima’s D test compares θ<sup>w</sup> and π to identify sequences that are not evolving neutrally (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Tajima 1989</a>). If a region is evolving neutrally, the two diversity measures should be equal (D = 0), whereas selection and demographic factors cause these two estimators to differ (D ≠ 0) (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Tajima 1983</a>). We calculated Tajima’s D for each gene region to determine which regions are under selection (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). Typically synonymous sites are considered neutral and thus would be expected to have a Tajima’s D value of 0. Interestingly, we observed a negative (−0.53) Tajima’s D value for synonymous sites. The low Tajima’s D may be due to the rapid range expansion of the pea aphid in North America since it was introduced approximately 150−200 years ago. This expansion would systematically affect all regions of the genome, causing lower D values (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Tajima 1989</a>). Also, sequencing errors can skew θ<sup>w</sup> estimates toward larger values, which can in turn result in more negative Tajima’s D values (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Achaz 2008</a>). Although we conservatively filtered out poor base pair calls (see <em>Materials and Methods</em>), the low sequencing coverage has likely impacted our results. Therefore, the absolute Tajima’s D values may be impacted by both demographic effects and sequencing errors. We expect these factors to affect each gene region equally. Thus, differences between regions are likely to be the result of difference in relative selection.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/6/993/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Mean Tajima&#x2019;s D values for coding and noncoding DNA. The mean Tajima&#x2019;s D values are given for different loci with bars indicating four SEs. The dotted line represents the neutral expectation. The labels are gene (exon, UTRs and introns), Syn (synonymous coding), Non Syn (non-synonymous coding), UTRs (untranslated regions), Introns, and up IGR (10 kb upstream intergenic region) and down IGR (10 kb downstream intergenic region)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1927319261" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Mean Tajima&#x2019;s D values for coding and noncoding DNA. The mean Tajima&#x2019;s D values are given for different loci with bars indicating four SEs. The dotted line represents the neutral expectation. The labels are gene (exon, UTRs and introns), Syn (synonymous coding), Non Syn (non-synonymous coding), UTRs (untranslated regions), Introns, and up IGR (10 kb upstream intergenic region) and down IGR (10 kb downstream intergenic region).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/6/993/F2.medium.gif" width="440" height="434"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/3/6/993/F2.medium.gif" width="440" height="434"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/6/993/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/6/993/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462838" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-31" class="first-child">Mean Tajima’s D values for coding and noncoding DNA. The mean Tajima’s D values are given for different loci with bars indicating four SEs. The dotted line represents the neutral expectation. The labels are gene (exon, UTRs and introns), Syn (synonymous coding), Non Syn (non-synonymous coding), UTRs (untranslated regions), Introns, and up IGR (10 kb upstream intergenic region) and down IGR (10 kb downstream intergenic region).</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">Synonymous site substitutions typically most closely meet the neutral expectation, so we compared nonsynonymous sites to synonymous sites. As expected, nonsynonymous sites exhibited lower Tajima’s D values, indicating selection (<em>P</em> &lt; 0.001, Mann-Whitney <em>U</em>-test). Additionally, introns, 5′ UTRs and the 10-kb IGR on either side of each gene also showed evidence of purifying selection, with all gene regions except the 3′ UTRs having significantly (<em>P</em> &lt; 0.001, Mann-Whitney <em>U</em>-test) lower Tajima’s D values than synonymous sites (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). <em>Arabidopsis thaliana</em> (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">DeRose-Wilson and Gaut 2007</a>) and <em>Drosophila melanogaster</em> (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Andolfatto 2005</a>) similarly show that most noncoding regions exhibit evidence of purifying selection. However, unlike <em>Drosophila</em>, where UTRs showed the strongest selection of the noncoding regions, pea aphid UTRs exhibit the least constraint of the noncoding DNAs. Upstream and downstream IGR exhibit similar Tajima’s D values as nonsynonymous sites.</p><p id="p-33">To compare individual genes, we examined D values for the entire gene region (UTRs, coding, and introns) and tested whether particular gene functional classes were significantly overrepresented in the lowest or highest 10% of Tajima’s D values. GO terms enriched for the lowest 10% Tajima’s D values (a range of −2.41 to −1.48) relate to DNA binding, including RNA-directed DNA polymerase activity and double-stranded DNA binding (FDR &lt; 0.05). GO terms enriched for the greatest 10% Tajima’s D values (a range of −0.24 to 1.53) mainly involve metabolic functions, such as carbohydrate metabolic process and peptide transporter activity. A full list of GO terms, with their associated genes, is available in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005793/-/DC1/TableS4.xlsx">Table S4</a>. The overrepresented GO terms describing the genes in the top 10% of Tajima’s D values mirror the terms that were significantly enriched for genes highly differentiated between Northeast and California samples (F<sub>st</sub> values &gt;0.5). Almost half (71 of 164 genes) of the highly differentiated genes were also represented in the top 10% of Tajima’s D values. This may be partially explained by our sampling equally from California and the Northeast, but it is also possible that there is population structure or balancing selection at these loci.</p></div><div id="sec-12" class="subsection"><h3>Discovery of X-linked <em>vs.</em> autosomal genomic scaffolds and genes</h3><p id="p-34">One of the challenges of many genome assemblies, particularly in nonmodel organisms that do not have detailed physical or linkage maps, is the large number of scaffolds that are generated and cannot be accurately assigned to a particular chromosome. In the case of the pea aphid, 23,924 scaffolds exist and it is unknown on which of the four chromosomes (one X, three autosomes) they belong. We developed a unique method to identify scaffolds on the X chromosome that takes advantage of the aphid’s method of male production during the life cycle (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>), in which males are genetically identical to their mother except that they inherit only one of her two X chromosomes (females are XX and males XO). We used high coverage genome sequence information from males produced from a single genetic clone (F1 line) heterozygous for the <em>aphicarus</em> (<em>api</em>) locus (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Caillaud <em>et al.</em> 2002</a>), an X-linked locus that controls a male wing polymorphism in the species. Because males are produced by the loss of a single X chromosome in the mother, they will inherit either the winged or wingless allele of <em>api</em>. There is no recombination on the X chromosome when asexual females produce males. Therefore, the wing phenotype is a marker for which X chromosome the male inherits from his mother and the <em>api</em> locus is in complete linkage with the remainder of the X chromosome. These males contain an identical set of autosomes to their mother and to each other.</p><p id="p-35">We compared the genome sequences between winged and wingless brothers of this F1 line to categorize scaffolds as located on the X or autosomes. If a scaffold is located on the X chromosome, the winged and wingless males should have sequence differences due to polymorphisms on the two different X chromosomes. A scaffold located on an autosome should be identical in the winged and wingless male because they inherit the same set of autosomes from their mother. However, in practice, sequence data are not perfect. Sequencing errors can result in base positions that appear to have differences but those differences are not truly present in their genomes. Furthermore, on autosomes, some heterozygous positions may have reads from only one particular allele in one male and only reads from the other allele in the other male, again appearing as a difference. Thus a single informative site does not conclusively show whether a scaffold is X-linked or autosomal. But when we examine a large number of sites, we predict that scaffolds from the X and autosomes will have different patterns.</p><p id="p-36">We examined the largest 1106 genomic scaffolds, each &gt;100 kb in length, which together contain &gt;85% of the genome (461,823,477 bp of the 541,675,471 bp total length). We used this cutoff to have sufficient sequence information for assessing pairwise sequence differences on any given scaffold. We produced a difference score for each scaffold that included the number of sites that differed between winged and wingless males. X scaffolds should have many more differences than autosomal scaffolds. X scaffolds with a large number of polymorphisms should have a larger difference score than X scaffolds with a small number of polymorphisms, and thus are easier to identify as belonging on the X. Unfortunately, autosomal scaffolds with a large number of polymorphisms will have a greater incidence of sequencing the opposite allele from each male than autosomes with a lower number. In some cases, the extremely polymorphic autosomes may have a similar number of differences as X chromosomes with very low levels of polymorphism. Thus, we corrected this score by subtracting the diversity on the scaffold (π, as calculated from all 21 lines) from the number of differences (see <em>Materials and Methods</em> for details). We expected to observe a bimodal frequency distribution of corrected difference scores by scaffold, with negative scores corresponding to autosomal scaffolds (many more polymorphisms than differences between brothers) and scores with a mean of zero corresponding to X-linked scaffolds (similar number of differences between brothers as the average polymorphism on the scaffold). The X chromosome is only one of the four total chromosomes comprising the 540 Mb genome, so we further expected that the autosomal scaffolds would be a much larger percentage of the total scaffolds as compared to the X-linked scaffolds. As predicted, the scaffolds fell into two peaks with autosomal scaffolds in larger numbers than X-linked scaffolds (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/6/993/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The distribution of the corrected pairwise difference and the associated probability that a scaffold is on the X chromosome. The top panel shows the frequency of scaffolds with a given corrected pairwise difference. The lines represent the fitted models. The bottom panel shows the probability of a scaffold being on the X chromosome as a function of the corrected pairwise difference. The dotted lines indicate the 0.05 and 0.95 probability values and the solid line represents the 0.5 probability value." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1927319261" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The distribution of the corrected pairwise difference and the associated probability that a scaffold is on the X chromosome. The top panel shows the frequency of scaffolds with a given corrected pairwise difference. The lines represent the fitted models. The bottom panel shows the probability of a scaffold being on the X chromosome as a function of the corrected pairwise difference. The dotted lines indicate the 0.05 and 0.95 probability values and the solid line represents the 0.5 probability value.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/6/993/F3.medium.gif" width="329" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/3/6/993/F3.medium.gif" width="329" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/6/993/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/6/993/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462861" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-37" class="first-child">The distribution of the corrected pairwise difference and the associated probability that a scaffold is on the X chromosome. The top panel shows the frequency of scaffolds with a given corrected pairwise difference. The lines represent the fitted models. The bottom panel shows the probability of a scaffold being on the X chromosome as a function of the corrected pairwise difference. The dotted lines indicate the 0.05 and 0.95 probability values and the solid line represents the 0.5 probability value.</p><div class="sb-div caption-clear"></div></div></div><p id="p-38">The two distributions overlapped slightly, so we used maximum likelihood to model the distribution to obtain a probability that a given scaffold is on the X. The probability values and how they correspond to the corrected differences per scaffold is visualized in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005793/-/DC1/TableS5.xlsx">Table S5</a> lists the scaffold names and their probability of being on the X.</p><p id="p-39">For further analysis we assigned scaffolds with an “X probability” of &lt;0.5 as autosomal and a probability &gt;0.5 as being X-linked. This resulted in 952 autosomal scaffolds and 154 X-linked scaffolds. These scaffolds housed 26,250 and 2928 genes, respectively.</p><p id="p-40">In males, we expect autosomal DNA to be sequenced twice as often as X DNA because they have a 2:1 ratio in the genome. Predicted autosomal scaffolds had, on average, 18.06 reads per bp, whereas predicted X scaffolds had 10.33, a ratio of 1.74:1. This is consistent with a large number of the scaffolds being correctly identified. The ratio was not exactly 2:1, suggesting some scaffolds may have been misidentified (see below for one explanation).</p><p id="p-41">To verify our method of assigning scaffolds to X or autosomal chromosomes, we independently identified 31 scaffolds on the X and 21 scaffolds on autosomes using RFLP analysis on male <em>vs.</em> female pea aphids (see <em>Materials and Methods</em>). All 27 scaffolds with an X probability &gt;0.5 were verified as X-linked. 21 of 25 scaffolds with an X probability &lt;0.5 were verified as autosomal, whereas the remaining four were verified as X-linked. The verified scaffolds and their X probabilities are illustrated in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> and the RFLP primers for each are listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005793/-/DC1/TableS2.xlsx">Table S2</a>.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/6/993/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The X probability values associated with scaffolds verified via RFLP analysis as X-linked (gray bars) or autosomal (black bars). Counts for the numbers of scaffolds that fall within each X probability value category are shown on the y-axis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1927319261" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The X probability values associated with scaffolds verified via RFLP analysis as X-linked (gray bars) or autosomal (black bars). Counts for the numbers of scaffolds that fall within each X probability value category are shown on the y-axis.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/6/993/F4.medium.gif" width="440" height="298"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/3/6/993/F4.medium.gif" width="440" height="298"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/6/993/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/6/993/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462977" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-42" class="first-child">The X probability values associated with scaffolds verified via RFLP analysis as X-linked (gray bars) or autosomal (black bars). Counts for the numbers of scaffolds that fall within each X probability value category are shown on the y-axis.</p><div class="sb-div caption-clear"></div></div></div><p id="p-43">We observed that miscategorization was most likely among larger scaffolds. We discovered that three scaffolds we examined (not included in the above numbers) were chimeras, a misassembled scaffold containing both X-linked and autosomal DNA. For a chimeric scaffold, an RFLP on one side of the scaffold demonstrated it to be X-linked and an RFLP on the other side demonstrated it as autosomal. These scaffolds have X probabilities of 0.03, 0.13, and 0.83. Based on this observation, we hypothesize that some of the larger scaffolds are chimeras of both autosomal and X-linked DNA incorrectly merged during genome assembly because the larger the scaffold, the greater the probability that an assembly error occurred. Depending on the levels of polymorphism within a given scaffold, it could skew a primarily X-linked chimera toward an autosomal probability score or vice versa. Furthermore, our verification method was based on a single SNP and therefore would verify the scaffold as X-linked or autosomal based on that position, when in fact the scaffold was a mixture of the two DNAs. We conclude that our assignment method works well, but that the genome annotation may have a large number of chimeric scaffolds. Further, we conclude that there is high confidence in our method when it assigns a high X-linked probability value since the scaffolds with high probability values have been verified as truly X-linked. However, a small percentage of X-linked scaffolds or chimeras between X-linked and autosomal scaffolds will be present among the scaffolds identified as autosomal.</p><p id="p-44">The X chromosome has previously been estimated to be 30% of the pea aphid genome based on chromosome photographs (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bizzaro <em>et al.</em> 2000</a>; <a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">Braendle <em>et al.</em> 2005</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Mandrioli <em>et al.</em> 1999</a>). With our data, we can now take a more detailed look at the ratio of X-linked to autosomal DNA. The total length of the combined scaffolds examined here was 461 Mb. The total length of the assembled genome is 540 Mb. The difference in these two genome sizes is because we restricted our analyses to scaffolds larger than 100 kb to have enough data to estimate corrected differences between winged and wingless males. From the scaffolds we examined, the X-linked scaffolds account for 11.7% of the DNA (53,889 kb) and the autosomal scaffolds account for 88.3% of the DNA (407,934 kb), in total harboring 29,178 of the 36,961 genes in the current (version 2.1) pea aphid annotation. We have therefore assigned the majority of the annotated genes to putative X or autosomal locations. Our estimate of 11.7% of the genome being X-linked is different than the 30% previously estimated from visual inspection of chromosome preparations. A large portion of the pea aphid X chromosome appears to be heterochromatic (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Mandrioli and Borsatti 2007</a>). This highly repetitive DNA would not be present in assembled scaffolds, which are primarily euchromatic DNA. Furthermore, some of this discrepancy is likely due to the scaffolds we used or did not use for our analysis. In particular, the large amount of heterochromatic DNA (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Mandrioli and Borsatti 2007</a>) can lead to genome assembly problems, resulting in small genomic scaffold sizes. X chromosome sequences are thus likely over-represented in the smaller scaffolds and therefore underrepresented in the scaffolds we used here. Finally, chimeric scaffolds assigned to autosomes could include large portions of X chromosomal sequence, particularly among the larger scaffolds. These would not be included in the final X-linked DNA count considered here.</p><p id="p-45">We find that the autosomes have greater gene density than the X chromosome (on average a gene every 15,540 bp on autosomal scaffolds and 18,405 bp on X-linked scaffolds). Furthermore, the average gene length is longer on the autosomes (10,203 bp <em>vs.</em> 9,549 bp). Neither genes on the X chromosome nor genes on the autosomes were significantly enriched for any gene ontology categories after multiple comparison correction.</p></div><div id="sec-13" class="subsection"><h3>Sequence diversity differs between the X and autosomes</h3><p id="p-46">In most XX/XY organisms, genetic diversity is expected to be higher on autosomes than the X chromosome because of the higher effective population size of the autosomes (reviewed in <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Ellegren 2009</a>). The pea aphid, with its XX/XO sex determination, presents a different scenario: the effective population sizes of the X chromosome and autosomes are equivalent because a mating female and male each contribute an X chromosome to the next generation (<a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Jaquiery <em>et al.</em> 2012</a>) and thus there is no <em>a priori</em> expectation of lower X diversity. The neutral expectation, therefore, is equal levels of genetic diversity between X chromosomes and autosomes. A previous study in pea aphids found similar genetic diversity on the X compared with autosomes using microsatellite repeats (<a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">Jaquiery <em>et al.</em> 2012</a>). Here we examine the relative genetic diversity on a genome-wide scale. We calculated levels of θ<sup>w</sup> and π for each scaffold and compared autosomal and X-linked chromosomes. We found that autosomal scaffolds had slightly greater θ<sup>w</sup> (0.005028 <em>vs.</em> 0.004833, ratio of 1.04) and π values (0.004459 <em>vs.</em> 0.004414, ratio of 1.01) compared with X-linked scaffolds. To determine whether these differences were significant, we compared the θ<sup>w</sup> and π values for genes (exons plus introns and 10 kb of intergenic regions on each side of the gene) on the autosomes <em>vs.</em> X chromosome. For both θ<sup>w</sup> and π, the autosomal genes had significantly greater values than the X chromosome genes (Mann-Whitney <em>U</em>-test, <em>P</em> &lt; 0.001). The same results were obtained whether we considered X-linked genes as those with an X-probability value of greater than 0.5 and autosomal as less than 0.5 or if we considered the much more stringent criteria of X-linked genes as those with an X-probability value of greater than 0.95 and autosomal as less than 0.05. We conclude that autosomal DNA harbors slightly greater nucleotide diversity than X-linked DNA despite the 1:1 X:A diversity expectation, although the differences are subtle.</p><p id="p-47">Deviations from the 1:1 X:A diversity expectation could be due to the action of natural selection. We therefore investigated whether selection affects X <em>vs.</em> autosomal DNA sequences differentially, again using Tajima’s D as an indicator of selection. We examined whole genomic scaffolds, as well as genes and parts of genes. In all cases, Tajima’s D values were negative and significantly lower for autosomal compared to X-linked genes (<em>P</em> &lt; 0.001, <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>; the same result was obtained with the more stringent criteria). We anticipated that synonymous sites would show similar values of Tajima’s D regardless of their location, again because they are thought to evolve at a nearly neutral rate. Consistent with this expectation, synonymous differences show the smallest difference between X and autosomes (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). In all other comparisons of genes or gene parts, autosomal genes exhibited a larger signal of purifying selection.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/6/993/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Mean Tajima&#x2019;s D values for autosomal and X-linked scaffolds. The mean values are given for genes or gene parts with bars indicating four SEs. The &#x201C;x&#x201D; symbols indicate the mean values of scaffolds predicted to be on the X chromosome and the triangle symbols indicate the mean values of scaffolds predicted to be on autosomes. The dotted line represents the neutral expectation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1927319261" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Mean Tajima&#x2019;s D values for autosomal and X-linked scaffolds. The mean values are given for genes or gene parts with bars indicating four SEs. The &#x201C;x&#x201D; symbols indicate the mean values of scaffolds predicted to be on the X chromosome and the triangle symbols indicate the mean values of scaffolds predicted to be on autosomes. The dotted line represents the neutral expectation.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/6/993/F5.medium.gif" width="440" height="438"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/3/6/993/F5.medium.gif" width="440" height="438"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/6/993/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/6/993/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462891" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-48" class="first-child">Mean Tajima’s D values for autosomal and X-linked scaffolds. The mean values are given for genes or gene parts with bars indicating four SEs. The “x” symbols indicate the mean values of scaffolds predicted to be on the X chromosome and the triangle symbols indicate the mean values of scaffolds predicted to be on autosomes. The dotted line represents the neutral expectation.</p><div class="sb-div caption-clear"></div></div></div><p id="p-49">We compared the Tajima’s D values of the 52 scaffolds that we had independently verified via RFLP analysis as autosomal or X-linked. The results from this subset of scaffolds were very similar to what we detected from the total data set. We observed an average Tajima’s D of -0.64 for X-linked scaffolds and −0.78 for autosomal scaffolds. Further, the differences between them were highly significant (Mann-Whitney <em>U</em>-test, <em>P</em> = 2 × 10<sup>−5</sup>). However, the marginally greater θ<sup>w</sup> and π differences we observed for autosomal relative to X-linked genes across all scaffolds was not supported by polymorphism measure comparisons within these 52 scaffolds (Mann-Whitney <em>U</em>-test, <em>P</em> = 0.93 and 0.40 for θ<sup>w</sup> and π, respectively).</p><p id="p-50">In summary, autosomal genes show marginally higher levels of nucleotide diversity and an excess of low-frequency variants. What could account for the greater number of low-frequency polymorphisms on the autosomes relative to the X chromosome? Recall that pea aphid females have two X chromosomes whereas males have only one. Although the autosomes and the X chromosome have the same effective population size (<a id="xref-ref-21-5" class="xref-bibr" href="#ref-21">Jaquiery <em>et al.</em> 2012</a>), their copy number difference means they respond to selection differently. Recessive X-linked alleles will be exposed to selection every time they are present in males because no other allele is present. Therefore, low-frequency recessive detrimental mutations, which would rarely be homozygous on an autosome, would frequently be under selection if on the X chromosome (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Vicoso and Charlesworth 2006</a>). Selection can fix beneficial and remove deleterious recessive mutations more effectively on the X (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Charlesworth <em>et al.</em> 1987</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Rice 1984</a>). However, because of this effectiveness, selection can remove or fix alleles on the X before they can affect the frequency spectrum of the surrounding polymorphisms in the whole population. Thus, even though selection acts more effectively on the X chromosome, that selection may not have a large effect on the frequency spectrum of the entire region. We suggest that this could account for the pattern we observe here.</p><p id="p-51">Interestingly, there is evidence that the pea aphid X chromosome is evolving faster than the autosomes. <a id="xref-ref-21-6" class="xref-bibr" href="#ref-21">Jaquiery <em>et al.</em> (2012)</a> found marginally significant (<em>P</em> = 0.03) acceleration of evolution of X relative to autosomal genes as measured by dN/dS in two of three comparisons of pea aphid genes to other aphid species for several hundred genes. Consistent with this, we find that pea aphid genes on the X chromosome are less likely to have potential orthologs (hits to the “nr” database via Blast with an Evalue less than E<sup>−20</sup>) than genes on the autosomes (Pearson’s Chi-squared test with Yates’ continuity correction <em>P</em> = 0.003).</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-52">We thank Tayler Schubauer and Cody Wehrkamp for performing the work to verify X-linked <em>vs.</em> autosomal genomic scaffolds. We also thank Chad Brassil for help with modeling. Funding was provided by National Institute of Environmental Health Sciences award 4R00ES017367.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005793/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005793/-/DC1</a></p></li><li class="fn" id="fn-4"><p id="p-53"><em>Communicating editor: J. M. Comeron</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2013-01-31"><span class="received-label">Received </span>January 31, 2013.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2013-04-08"><span class="accepted-label">Accepted </span>April 8, 2013.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2013 Bickel <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.993.1" data-doi="10.1534/genetics.107.082198"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Achaz</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Testing for neutrality in samples with sequencing errors.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">1409</span>–<span class="cit-lpage">1424</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.082198%26rft_id%253Dinfo%253Apmid%252F18562660%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OS8zLzE0MDkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzYvOTkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.993.2" data-doi="10.1038/nature04107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andolfatto</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Adaptive evolution of non-coding DNA in Drosophila.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">437</span>: <span class="cit-fpage">1149</span>–<span class="cit-lpage">1152</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DAndolfatto%26rft.auinit1%253DP.%26rft.volume%253D437%26rft.issue%253D7062%26rft.spage%253D1149%26rft.epage%253D1152%26rft.atitle%253DAdaptive%2Bevolution%2Bof%2Bnon-coding%2BDNA%2Bin%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature04107%26rft_id%253Dinfo%253Apmid%252F16237443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature04107&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16237443&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232660500043&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.993.3" data-doi="10.1371/journal.pbio.0050310"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Begun</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holloway</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hilllier</span>  <span class="cit-name-given-names">L. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poh</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Population genomics: whole-genome analysis of polymorphism and divergence in <em>Drosophila simulans</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e310</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DBegun%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D5%26rft.issue%253D11%26rft.spage%253De310%26rft.epage%253De310%26rft.atitle%253DPopulation%2Bgenomics%253A%2Bwhole-genome%2Banalysis%2Bof%2Bpolymorphism%2Band%2Bdivergence%2Bin%2BDrosophila%2Bsimulans.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0050310%26rft_id%253Dinfo%253Apmid%252F17988176%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0050310&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17988176&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.993.4" data-doi="10.1038/nature11247"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernstein</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birney</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunham</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gunter</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">An integrated encyclopedia of DNA elements in the human genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">489</span>: <span class="cit-fpage">57</span>–<span class="cit-lpage">74</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBernstein%26rft.auinit1%253DB.%2BE.%26rft.volume%253D489%26rft.issue%253D7414%26rft.spage%253D57%26rft.epage%253D74%26rft.atitle%253DAn%2Bintegrated%2Bencyclopedia%2Bof%2BDNA%2Belements%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11247%26rft_id%253Dinfo%253Apmid%252F22955616%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11247&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22955616&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000308347000039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.993.5" data-doi="10.1023/A:1004128314728"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bizzaro</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mandrioli</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zanotti</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giusti</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manicardi</span>  <span class="cit-name-given-names">G. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Chromosome analysis and molecular characterization of highly repeated DNAs in the aphid <em>Acyrthosiphon pisum</em> (Aphididae, Hemiptera).</span> <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">197</span>–<span class="cit-lpage">202</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.stitle%253DGenetica%26rft.aulast%253DBizzaro%26rft.auinit1%253DD.%26rft.volume%253D108%26rft.issue%253D2%26rft.spage%253D197%26rft.epage%253D202%26rft.atitle%253DChromosome%2Banalysis%2Band%2Bmolecular%2Bcharacterization%2Bof%2Bhighly%2Brepeated%2BDNAs%2Bin%2Bthe%2Baphid%2BAcyrthosiphon%2Bpisum%2B%2528Aphididae%252C%2BHemiptera%2529.%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1004128314728%26rft_id%253Dinfo%253Apmid%252F11138948%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1004128314728&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11138948&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000165128000012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-book" id="cit-3.6.993.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blackman</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Reproduction, cytogenetics and development</span>, pp. <span class="cit-fpage">163</span>–<span class="cit-lpage">195</span> in <span class="cit-source"><em>Aphids: Their Biology</em>, <em>Natural Enemies &amp; Control</em></span>, edited by <span class="cit-ed"><span class="cit-name-surname">Minks</span>  <span class="cit-name-given-names">A. K.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Harrewijn</span>  <span class="cit-name-given-names">P.</span></span>. <span class="cit-publ-name">Elsevier</span>, <span class="cit-publ-loc">Amsterdam</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-book" id="cit-3.6.993.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blackman</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eastop</span>  <span class="cit-name-given-names">V. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-source">Aphids on the World’s Crops: An Identification and Information Guide</span>. <span class="cit-publ-name">John Wiley &amp; Sons Ltd.</span>, <span class="cit-publ-loc">Chichester, England</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.993.8" data-doi="10.1038/sj.hdy.6800633"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Braendle</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caillaud</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stern</span>  <span class="cit-name-given-names">D. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genetic mapping of <em>aphicarus</em>—a sex-linked locus controlling a wing polymorphism in the pea aphid (<em>Acyrthosiphon pisum)</em>.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">435</span>–<span class="cit-lpage">442</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DBraendle%26rft.auinit1%253DC.%26rft.volume%253D94%26rft.issue%253D4%26rft.spage%253D435%26rft.epage%253D442%26rft.atitle%253DGenetic%2Bmapping%2Bof%2Baphicarus%2B--%2Ba%2Bsex-linked%2Blocus%2Bcontrolling%2Ba%2Bwing%2Bpolymorphism%2Bin%2Bthe%2Bpea%2Baphid%2B%2528Acyrthosiphon%2Bpisum%2529.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.hdy.6800633%26rft_id%253Dinfo%253Apmid%252F15674387%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.hdy.6800633&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15674387&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227985000009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.993.9" data-doi="10.1002/bies.20436"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brisson</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stern</span>  <span class="cit-name-given-names">D. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The pea aphid, <em>Acyrthosiphon pisum</em>: an emerging genomic model system for ecological, developmental and evolutionary studies.</span> <abbr class="cit-jnl-abbrev">Bioessays</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">747</span>–<span class="cit-lpage">755</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioEssays%2B%253A%2B%2Bnews%2Band%2Breviews%2Bin%2Bmolecular%252C%2Bcellular%2Band%2Bdevelopmental%2Bbiology%26rft.stitle%253DBioessays%26rft.aulast%253DBrisson%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D28%26rft.issue%253D7%26rft.spage%253D747%26rft.epage%253D755%26rft.atitle%253DThe%2Bpea%2Baphid%252C%2BAcyrthosiphon%2Bpisum%253A%2Ban%2Bemerging%2Bgenomic%2Bmodel%2Bsystem%2Bfor%2Becological%252C%2Bdevelopmental%2Band%2Bevolutionary%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbies.20436%26rft_id%253Dinfo%253Apmid%252F16850403%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/bies.20436&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16850403&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239322900009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.993.10" data-doi="10.1186/1471-2156-10-22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brisson</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stern</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nuzhdin</span>  <span class="cit-name-given-names">S. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Patterns of linkage disequilibrium and nucleotide diversity in native and introduced populations of the pea aphid.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DBrisson%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D10%26rft.spage%253D22%26rft.epage%253D22%26rft.atitle%253DSimilar%2Bpatterns%2Bof%2Blinkage%2Bdisequilibrium%2Band%2Bnucleotide%2Bdiversity%2Bin%2Bnative%2Band%2Bintroduced%2Bpopulations%2Bof%2Bthe%2Bpea%2Baphid%252C%2BAcyrthosiphon%2Bpisum.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2156-10-22%26rft_id%253Dinfo%253Apmid%252F19470181%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2156-10-22&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19470181&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.993.11" data-doi="10.1038/sj.hdy.6800146"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Caillaud</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boutin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Braendle</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simon</span>  <span class="cit-name-given-names">J.-C.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">A sex-linked locus controls wing polymorphism in males of the pea aphid, <em>Acyrthosiphon pisum</em> (Harris).</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">346</span>–<span class="cit-lpage">352</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DCaillaud%26rft.auinit1%253DM.%2BC.%26rft.volume%253D89%26rft.issue%253D5%26rft.spage%253D346%26rft.epage%253D352%26rft.atitle%253DA%2Bsex-linked%2Blocus%2Bcontrols%2Bwing%2Bpolymorphism%2Bin%2Bmales%2Bof%2Bthe%2Bpea%2Baphid%252C%2BAcyrthosiphon%2Bpisum%2B%2528Harris%2529.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.hdy.6800146%26rft_id%253Dinfo%253Apmid%252F12399992%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.hdy.6800146&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12399992&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000179334600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.993.12" data-doi="10.1086/284701"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coyne</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barton</span>  <span class="cit-name-given-names">N. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">The relative rates of evolution of sex-chromosomes and autosomes.</span> <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">130</span>: <span class="cit-fpage">113</span>–<span class="cit-lpage">146</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat.%26rft.volume%253D130%26rft.spage%253D113%26rft_id%253Dinfo%253Adoi%252F10.1086%252F284701%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/284701&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1987J340700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.993.13" data-doi="10.1093/bioinformatics/bti610"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conesa</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gotz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia-Gomez</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Terol</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Talon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Blast2Go: a universal tool for annotation, visualization and analysis in functional genomics research.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">3674</span>–<span class="cit-lpage">3676</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti610%26rft_id%253Dinfo%253Apmid%252F16081474%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjEvMTgvMzY3NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNi85OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-other" id="cit-3.6.993.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">J. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1915</span> <span class="cit-article-title">The pea aphid with relation to forage crops.</span> <span class="cit-source">Bull. US Dept. Agric.</span> <span class="cit-fpage">276</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.993.15" data-doi="10.1186/1471-2148-7-66"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DeRose-Wilson</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaut</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Transcription-related mutations and GC content drive variation in nucleotide substitution rates across the genomes of <em>Arabidopsis thaliana</em> and <em>Arabidopsis lyrata</em>.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">66</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DDerose-Wilson%26rft.auinit1%253DL.%2BJ.%26rft.volume%253D7%26rft.spage%253D66%26rft.epage%253D66%26rft.atitle%253DTranscription-related%2Bmutations%2Band%2BGC%2Bcontent%2Bdrive%2Bvariation%2Bin%2Bnucleotide%2Bsubstitution%2Brates%2Bacross%2Bthe%2Bgenomes%2Bof%2BArabidopsis%2Bthaliana%2Band%2BArabidopsis%2Blyrata.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-7-66%26rft_id%253Dinfo%253Apmid%252F17451608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-7-66&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17451608&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.993.16" data-doi="10.1016/j.tig.2009.04.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ellegren</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The different levels of genetic diversity in sex chromosomes and autosomes.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">278</span>–<span class="cit-lpage">284</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DEllegren%26rft.auinit1%253DH.%26rft.volume%253D25%26rft.issue%253D6%26rft.spage%253D278%26rft.epage%253D284%26rft.atitle%253DThe%2Bdifferent%2Blevels%2Bof%2Bgenetic%2Bdiversity%2Bin%2Bsex%2Bchromosomes%2Band%2Bautosomes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2009.04.005%26rft_id%253Dinfo%253Apmid%252F19481288%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2009.04.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19481288&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267587000006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.993.17" data-doi="10.1111/j.1558-5646.2008.00468.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ferrari</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Via</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Godfray</span>  <span class="cit-name-given-names">C. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Population differentiation and genetic variation in performance on eight hosts in the pea aphid complex.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">2508</span>–<span class="cit-lpage">2524</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DFerrari%26rft.auinit1%253DJ.%26rft.volume%253D62%26rft.issue%253D10%26rft.spage%253D2508%26rft.epage%253D2524%26rft.atitle%253DPopulation%2Bdifferentiation%2Band%2Bgenetic%2Bvariation%2Bin%2Bperformance%2Bon%2Beight%2Bhosts%2Bin%2Bthe%2Bpea%2Baphid%2Bcomplex.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2008.00468.x%26rft_id%253Dinfo%253Apmid%252F18647340%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2008.00468.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18647340&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259900800005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.993.18" data-doi="10.1098/rspb.2006.3665"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frantz</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plantegenest</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simon</span>  <span class="cit-name-given-names">J.-C.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Temporal habitat variability and the maintenance of sex in host populations of the pea aphid.</span> <abbr class="cit-jnl-abbrev">Proc. Biol. Sci.</abbr> <span class="cit-vol">273</span>: <span class="cit-fpage">2887</span>–<span class="cit-lpage">2891</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DProc%2BR%2BSoc%2BB%26rft.aulast%253DFrantz%26rft.auinit1%253DA.%26rft.volume%253D273%26rft.issue%253D1603%26rft.spage%253D2887%26rft.epage%253D2891%26rft.atitle%253DTemporal%2Bhabitat%2Bvariability%2Band%2Bthe%2Bmaintenance%2Bof%2Bsex%2Bin%2Bhost%2Bpopulations%2Bof%2Bthe%2Bpea%2Baphid%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2006.3665%26rft_id%253Dinfo%253Apmid%252F17015368%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Nzoicm95cHJzYiI7czo1OiJyZXNpZCI7czoxMzoiMjczLzE2MDMvMjg4NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNi85OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.993.19" data-doi="10.1371/journal.pgen.1005397"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slatkin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maddison</span>  <span class="cit-name-given-names">W. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Estimation of levels of gene flow from DNA-SEQUENCE DATA.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">583</span>–<span class="cit-lpage">589</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHudson%26rft.auinit1%253DR.%2BR.%26rft.volume%253D132%26rft.issue%253D2%26rft.spage%253D583%26rft.epage%253D589%26rft.atitle%253DEstimation%2Bof%2Blevels%2Bof%2Bgene%2Bflow%2Bfrom%2BDNA%2Bsequence%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005397%26rft_id%253Dinfo%253Apmid%252F1427045%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTMyLzIvNTgzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy82Lzk5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.993.20" data-doi="10.1371/journal.pbio.1000313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">International Aphid Genomics Consortium</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome sequence of the pea aphid <em>Acyrthosiphon pisum</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1000313</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.volume%253D8%26rft.issue%253D2%26rft.spage%253De1000313%26rft.epage%253De1000313%26rft.atitle%253DGenome%2Bsequence%2Bof%2Bthe%2Bpea%2Baphid%2BAcyrthosiphon%2Bpisum.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1000313%26rft_id%253Dinfo%253Apmid%252F20186266%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1000313&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20186266&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.993.21" data-doi="10.1093/molbev/msr252"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jaquiery</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stoeckel</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rispe</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mieuzet</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legeai</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Accelerated evolution of sex chromosomes in aphids, an X0 System.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">837</span>–<span class="cit-lpage">847</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsr252%26rft_id%253Dinfo%253Apmid%252F21998277%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjI5LzIvODM3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy82Lzk5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.993.22" data-doi="10.1098/rsbl.2003.0064"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leonardo</span>  <span class="cit-name-given-names">T. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muiru</span>  <span class="cit-name-given-names">G. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Facultative symbionts are associated with host plant specialization in pea aphid populations.</span> <abbr class="cit-jnl-abbrev">Proc. Biol. Sci.</abbr> <span class="cit-vol">270</span>: <span class="cit-fpage">S209</span>–<span class="cit-lpage">S212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DProc%2BR%2BSoc%2BB%26rft.aulast%253DLeonardo%26rft.auinit1%253DT.%2BE.%26rft.volume%253D270%26rft.issue%253DSuppl_2%26rft.spage%253DS209%26rft.epage%253DS212%26rft.atitle%253DFacultative%2Bsymbionts%2Bare%2Bassociated%2Bwith%2Bhost%2Bplant%2Bspecialization%2Bin%2Bpea%2Baphid%2Bpopulations%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frsbl.2003.0064%26rft_id%253Dinfo%253Apmid%252F14667385%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Nzoicm95cHJzYiI7czo1OiJyZXNpZCI7czoxNjoiMjcwL1N1cHBsXzIvUzIwOSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNi85OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.993.23" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTQvMTc1NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNi85OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.993.24" data-doi="10.1093/bioinformatics/btp698"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Fast and accurate long-read alignment with Burrows-Wheeler Transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">589</span>–<span class="cit-lpage">595</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp698%26rft_id%253Dinfo%253Apmid%252F20080505%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNi81LzU4OSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNi85OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.993.25" data-doi="10.1371/journal.pgen.0020166"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephan</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Inferring the demographic history and rate of adaptative substitution in Drosophila.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e166</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DLi%26rft.auinit1%253DH.%26rft.volume%253D2%26rft.issue%253D10%26rft.spage%253De166%26rft.epage%253De166%26rft.atitle%253DInferring%2Bthe%2Bdemographic%2Bhistory%2Band%2Brate%2Bof%2Badaptive%2Bsubstitution%2Bin%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0020166%26rft_id%253Dinfo%253Apmid%252F17040129%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0020166&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17040129&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.993.26" data-doi="10.1007/s10577-007-1176-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mandrioli</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borsatti</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Analysis of heterochromatic epigenetic markers in the holocentric chromosomes of the aphid <em>Acyrthosiphon pisum</em>.</span> <abbr class="cit-jnl-abbrev">Chrom. Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1015</span>–<span class="cit-lpage">1022</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChrom.%2BRes.%26rft.volume%253D15%26rft.spage%253D1015%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10577-007-1176-4%26rft_id%253Dinfo%253Apmid%252F18004669%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10577-007-1176-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18004669&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.993.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mandrioli</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bizzaro</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giusti</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manicardi</span>  <span class="cit-name-given-names">G. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bianchi</span>  <span class="cit-name-given-names">U.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">The role of rDNA genes in X chromosome association in the aphid <em>Acyrthosiphon pisum</em>.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">381</span>–<span class="cit-lpage">386</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DMandrioli%26rft.auinit1%253DM.%26rft.volume%253D42%26rft.issue%253D3%26rft.spage%253D381%26rft.epage%253D386%26rft.atitle%253DThe%2Brole%2Bof%2BrDNA%2Bgenes%2Bin%2BX%2Bchromosome%2Bassociation%2Bin%2Bthe%2Baphid%2BAcyrthosiphon%2Bpisum.%26rft_id%253Dinfo%253Apmid%252F10382287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10382287&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.993.28" data-doi="10.1146/annurev.en.37.010192.001541"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moran</span>  <span class="cit-name-given-names">N. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">The evolution of aphid life cycles.</span> <abbr class="cit-jnl-abbrev">Ann. Rev. Ent.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">321</span>–<span class="cit-lpage">348</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bentomology%26rft.stitle%253DAnnu%2BRev%2BEntomol%26rft.aulast%253DMoran%26rft.auinit1%253DN.%26rft.volume%253D37%26rft.issue%253D1%26rft.spage%253D321%26rft.epage%253D348%26rft.atitle%253DThe%2BEvolution%2BOf%2BAphid%2BLife%2BCycles%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.en.37.010192.001541%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.en.37.010192.001541&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1992GY50200014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.993.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tajima</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1981</span> <span class="cit-article-title">DNA polymorphism detectable by restriction endonucleases.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">146</span>: <span class="cit-fpage">711</span>–<span class="cit-lpage">716</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D146%26rft.spage%253D711%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.993.30" data-doi="10.2307/2408385"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rice</span>  <span class="cit-name-given-names">W. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Sex chromosomes and the evolution of sexual dimorphism.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">735</span>–<span class="cit-lpage">742</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D38%26rft.spage%253D735%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2408385%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2408385&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1984TJ95300003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-book" id="cit-3.6.993.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robert</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Dispersion and migration</span>, pp. <span class="cit-fpage">299</span>–<span class="cit-lpage">313</span> in <span class="cit-source"><em>Aphids</em>, <em>Their Biology</em>, <em>Natural Enemies</em>, <em>and Control</em></span>, edited by <span class="cit-ed"><span class="cit-name-surname">Minks</span>  <span class="cit-name-given-names">A. K.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Harrewijn</span>  <span class="cit-name-given-names">P.</span></span>. <span class="cit-publ-name">Elsevier</span>, <span class="cit-publ-loc">Amsterdam, Netherlands</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.993.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sanderson</span>  <span class="cit-name-given-names">E. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1900</span> <span class="cit-article-title">The destructive pea louse in Delaware.</span> <abbr class="cit-jnl-abbrev">Delaware Agric. Exp. Station Bull.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">14</span>–<span class="cit-lpage">24</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDelaware%2BAgric.%2BExp.%2BStation%2BBull.%26rft.volume%253D49%26rft.spage%253D14%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.993.33" data-doi="10.1098/rspb.2003.2430"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simon</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carre</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boutin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prunier-Leterme</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sabater-Munoz</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Host-based divergence in populations of the pea aphid: insights from nuclear markers and the prevalence of facultative symbionts.</span> <abbr class="cit-jnl-abbrev">Proc. Biol. Sci.</abbr> <span class="cit-vol">270</span>: <span class="cit-fpage">1703</span>–<span class="cit-lpage">1712</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DProc%2BR%2BSoc%2BB%26rft.aulast%253DSimon%26rft.auinit1%253DJ.-%2BC.%26rft.volume%253D270%26rft.issue%253D1525%26rft.spage%253D1703%26rft.epage%253D1712%26rft.atitle%253DHost-based%2Bdivergence%2Bin%2Bpopulations%2Bof%2Bthe%2Bpea%2Baphid%253A%2Binsights%2Bfrom%2Bnuclear%2Bmarkers%2Band%2Bthe%2Bprevalence%2Bof%2Bfacultative%2Bsymbionts%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2003.2430%26rft_id%253Dinfo%253Apmid%252F12964998%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Nzoicm95cHJzYiI7czo1OiJyZXNpZCI7czoxMzoiMjcwLzE1MjUvMTcwMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNi85OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.993.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tajima</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Evolutionary relationship of DNA sequences in finite populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">437</span>–<span class="cit-lpage">460</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DTajima%26rft.auinit1%253DF.%26rft.volume%253D105%26rft.issue%253D2%26rft.spage%253D437%26rft.epage%253D460%26rft.atitle%253DEVOLUTIONARY%2BRELATIONSHIP%2BOF%2BDNA%2BSEQUENCES%2BIN%2BFINITE%2BPOPULATIONS%26rft_id%253Dinfo%253Apmid%252F6628982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTA1LzIvNDM3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy82Lzk5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.993.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tajima</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Statistical method for testing the neutral mutation hypothesis by DNA polymorphism.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">585</span>–<span class="cit-lpage">595</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DTajima%26rft.auinit1%253DF.%26rft.volume%253D123%26rft.issue%253D3%26rft.spage%253D585%26rft.epage%253D595%26rft.atitle%253DStatistical%2Bmethod%2Bfor%2Btesting%2Bthe%2Bneutral%2Bmutation%2Bhypothesis%2Bby%2BDNA%2Bpolymorphism.%26rft_id%253Dinfo%253Apmid%252F2513255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTIzLzMvNTg1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy82Lzk5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.993.36" data-doi="10.1093/bioinformatics/btg316"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thornton</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">libsequence: a C++ class library for evolutionary genetic analysis.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">2325</span>–<span class="cit-lpage">2327</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DThornton%26rft.auinit1%253DK.%26rft.volume%253D19%26rft.issue%253D17%26rft.spage%253D2325%26rft.epage%253D2327%26rft.atitle%253Dlibsequence%253A%2Ba%2BC%252B%252B%2Bclass%2Blibrary%2Bfor%2Bevolutionary%2Bgenetic%2Banalysis%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg316%26rft_id%253Dinfo%253Apmid%252F14630667%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMTkvMTcvMjMyNSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNi85OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.993.37" data-doi="10.2307/2640891"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Via</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Reproductive isolation between sympatric races of pea aphids. I. Gene flow restriction and habitat choice.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">1446</span>–<span class="cit-lpage">1457</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D53%26rft.spage%253D1446%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2640891%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2640891&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000083567100012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.993.38" data-doi="10.1554/0014-3820(2000)054[1626:RIBDRO]2.0.CO;2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Via</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bouck</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skillman</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Reproductive isolation between divergent races of pea aphids on two hosts. II. Selection against migrants and hybrids in the parental environments.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">1626</span>–<span class="cit-lpage">1637</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DVia%26rft.auinit1%253DS.%26rft.volume%253D54%26rft.issue%253D5%26rft.spage%253D1626%26rft.epage%253D1637%26rft.atitle%253DReproductive%2Bisolation%2Bbetween%2Bdivergent%2Braces%2Bof%2Bpea%2Baphids%2Bon%2Btwo%2Bhosts.%2BII.%2BSelection%2Bagainst%2Bmigrants%2Band%2Bhybrids%2Bin%2Bthe%2Bparental%2Benvironments.%26rft_id%253Dinfo%253Adoi%252F10.1554%252F0014-3820%25282000%2529054%255B1626%253ARIBDRO%255D2.0.CO%253B2%26rft_id%253Dinfo%253Apmid%252F11108590%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1554/0014-3820(2000)054[1626:RIBDRO]2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11108590&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000165471000015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.993.39" data-doi="10.1038/nrg1914"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vicoso</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Evolution on the X chromosome: unusual patterns and processes.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">645</span>–<span class="cit-lpage">653</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DVicoso%26rft.auinit1%253DB.%26rft.volume%253D7%26rft.issue%253D8%26rft.spage%253D645%26rft.epage%253D653%26rft.atitle%253DEvolution%2Bon%2Bthe%2BX%2Bchromosome%253A%2Bunusual%2Bpatterns%2Band%2Bprocesses.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1914%26rft_id%253Dinfo%253Apmid%252F16847464%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1914&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16847464&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239118400016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.993.40" data-doi="10.1016/0040-5809(75)90020-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Watterson</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">On the number of segregating sites in genetical models without recombination.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">256</span>–<span class="cit-lpage">275</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DWatterson%26rft.auinit1%253DG.%2BA.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253D256%26rft.epage%253D276%26rft.atitle%253DOn%2Bthe%2Bnumber%2Bof%2Bsegregating%2Bsites%2Bin%2Bgenetical%2Bmodels%2Bwithout%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0040-5809%252875%252990020-9%26rft_id%253Dinfo%253Apmid%252F1145509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0040-5809(75)90020-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1145509&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F6%2F993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1975W295600011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-3.6.993.41" data-doi="10.1017/S0016672397002747"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">A. C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sunnucks</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hales</span>  <span class="cit-name-given-names">D. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Random loss of X chromosome at male determination in an aphid, <em>Sitobion near fragariae</em>, detected using an X-linked polymorphic microsatellite marker.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">233</span>–<span class="cit-lpage">236</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BRes.%26rft.volume%253D69%26rft.spage%253D233%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672397002747%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672397002747&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/3/6/993.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/6/979" title="Identification of Cilia Genes That Affect Cell-Cycle Progression Using Whole-Genome Transcriptome Analysis in Chlamydomonas reinhardtti" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/6/1003" title="Retrospective View of North American Potato (Solanum tuberosum L.) Breeding in the 20th and 21st Centuries" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/3/6"><span>Volume 3 Issue 6, June 2013</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/3/6.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/3/6.cover-source.jpg?itok=CJf4vROV" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 3 (6)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/6/993.abstract" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-m6pRjWMjTdkLwwq-DeFaX0Psy6nKrgHQuRLmsEUG7Ok" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/3/6/993" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/3/6/993" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/3/6/993" class="active" data-icon-position="" data-hide-link-title="0">Widespread Selection Across Coding and Noncoding DNA in the Pea Aphid Genome</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459481" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-ts3h4bu80f8AndWnlIETr9idYl4agKXtQnrIWv1rkaA" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/6/993.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/3/6/993" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-KF_TjDgQvruc8GHMFQX66Ib1-Ek2kPR8nN50E7M_QuI" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/3/6/993" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/6/993.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459481" id="node459481--2" data-pisa="ggg;3/6/993" data-pisa-master="ggg;g3.113.005793" data-apath="/ggg/3/6/993.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Widespread Selection Across Coding and Noncoding DNA in the Pea Aphid Genome</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Ryan D. Bickel</span>, <span class="highwire-citation-author" data-delta="1">Joseph P. Dunham</span> and <span class="highwire-citation-author" data-delta="2">Jennifer A. Brisson</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">993-1001;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.005793 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459481--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ryan D. Bickel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>University of Nebraska, School of Biological Sciences, Lincoln, Nebraska 68588</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ryan%2BD.%2BBickel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bickel%20RD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARyan%2BD.%2BBickel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:rbickel2@unl.edu" class="" data-icon-position="" data-hide-link-title="0">rbickel2@unl.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Joseph P. Dunham </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>University of Southern California, Molecular & Computational Biology, Los Angeles, California 90089</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BP.%2BDunham%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dunham%20JP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoseph%2BP.%2BDunham%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jennifer A. Brisson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>University of Nebraska, School of Biological Sciences, Lincoln, Nebraska 68588</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jennifer%2BA.%2BBrisson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brisson%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJennifer%2BA.%2BBrisson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/3/6/993" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/3/6/993&amp;title=Widespread%20Selection%20Across%20Coding%20and%20Noncoding%20DNA%20in%20the%20Pea%20Aphid%20Genome" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/3/6/993&amp;title=Widespread%20Selection%20Across%20Coding%20and%20Noncoding%20DNA%20in%20the%20Pea%20Aphid%20Genome" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/3/6/993&amp;title=Widespread%20Selection%20Across%20Coding%20and%20Noncoding%20DNA%20in%20the%20Pea%20Aphid%20Genome" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/3/6/993&amp;text=Widespread%20Selection%20Across%20Coding%20and%20Noncoding%20DNA%20in%20the%20Pea%20Aphid%20Genome" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/3/6/993&amp;title=Widespread%20Selection%20Across%20Coding%20and%20Noncoding%20DNA%20in%20the%20Pea%20Aphid%20Genome" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/3/6/993&amp;t=Widespread%20Selection%20Across%20Coding%20and%20Noncoding%20DNA%20in%20the%20Pea%20Aphid%20Genome" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/3/6/993&amp;title=Widespread%20Selection%20Across%20Coding%20and%20Noncoding%20DNA%20in%20the%20Pea%20Aphid%20Genome" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/3/6/993&amp;title=Widespread%20Selection%20Across%20Coding%20and%20Noncoding%20DNA%20in%20the%20Pea%20Aphid%20Genome" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459481/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459481" id="node459481--3" data-pisa="ggg;3/6/993" data-pisa-master="ggg;g3.113.005793" data-apath="/ggg/3/6/993.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Widespread Selection Across Coding and Noncoding DNA in the Pea Aphid Genome</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Ryan D. Bickel</span>, <span class="highwire-citation-author" data-delta="1">Joseph P. Dunham</span> and <span class="highwire-citation-author" data-delta="2">Jennifer A. Brisson</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">993-1001;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.005793 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459481--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ryan D. Bickel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>University of Nebraska, School of Biological Sciences, Lincoln, Nebraska 68588</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ryan%2BD.%2BBickel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bickel%20RD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARyan%2BD.%2BBickel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:rbickel2@unl.edu" class="" data-icon-position="" data-hide-link-title="0">rbickel2@unl.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Joseph P. Dunham </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>University of Southern California, Molecular & Computational Biology, Los Angeles, California 90089</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BP.%2BDunham%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dunham%20JP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoseph%2BP.%2BDunham%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jennifer A. Brisson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>University of Nebraska, School of Biological Sciences, Lincoln, Nebraska 68588</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jennifer%2BA.%2BBrisson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brisson%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJennifer%2BA.%2BBrisson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=993&amp;rft.epage=1001&amp;rft.atitle=Widespread%20Selection%20Across%20Coding%20and%20Noncoding%20DNA%20in%20the%20Pea%20Aphid%20Genome&amp;rft.volume=3&amp;rft.issue=6&amp;rft.date=2013-06-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Bickel%2C+Ryan+D.&amp;rft.au=Dunham%2C+Joseph+P.&amp;rft.au=Brisson%2C+Jennifer+A."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459481/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459481/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459481/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459481/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459481/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459481/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459481/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459481/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459481/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459481/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459481/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459481/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/3/6/993" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/3/6/993.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/3/6/993.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/3/6/993.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/3/6/993.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
