<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/5/1007" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/5/1007.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/5/1007.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/5/1007.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/5/1007.atom" />
<meta name="HW.pisa" content="ggg;5/5/1007" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Rapid and Efficient Identification of Caenorhabditis elegans Legacy Mutations Using Hawaiian SNP-Based Mapping and Whole-Genome Sequencing" />
<meta name="DC.Identifier" content="10.1534/g3.115.017038" />
<meta name="DC.Date" content="2015-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Jaramillo-Lambert et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The production of viable embryos requires the coordination of many cellular processes, including protein synthesis, cytoskeletal reorganization, establishment of polarity, cell migration, cell division, and in Caenorhabditis elegans , eggshell formation. Defects in any of these processes can lead to embryonic lethality. We examined six temperature-sensitive mutants as well as one nonconditional mutant that were previously identified in genetic screens as either embryonic lethal (maternal-effect or zygotic lethal) or eggshell defective. The responsible molecular lesion for each had never been determined. After confirmation of temperature sensitivity and lethality, we performed whole-genome sequencing using a single-nucleotide polymorphism mapping strategy to pinpoint the molecular lesions. Gene candidates were confirmed by RNA interference phenocopy and/or complementation tests and one mutant was further validated by CRISPR (Clustered Regularly Interspaced Short Palidromic Repeats)/Cas9 gene editing. This approach identified new alleles of several genes that had only been previously studied by RNA interference depletion. Our identification of temperature-sensitive alleles for all of these essential genes provides an extremely useful tool for further investigation for the C. elegans community, such as the ability to address mutant phenotypes at various developmental stages and the ability to carry out suppressor/enhancer screens to identify other genes that function in a specific cellular process." />
<meta name="DC.Contributor" content="Aimee Jaramillo-Lambert" />
<meta name="DC.Contributor" content="Abigail S. Fuchsman" />
<meta name="DC.Contributor" content="Amy S. Fabritius" />
<meta name="DC.Contributor" content="Harold E. Smith" />
<meta name="DC.Contributor" content="Andy Golden" />
<meta name="article:published_time" content="2015-05-01" />
<meta name="article:section" content="Mutant Screen Report" />
<meta name="citation_title" content="Rapid and Efficient Identification of Caenorhabditis elegans Legacy Mutations Using Hawaiian SNP-Based Mapping and Whole-Genome Sequencing" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The production of viable embryos requires the coordination of many cellular processes, including protein synthesis, cytoskeletal reorganization, establishment of polarity, cell migration, cell division, and in &lt;i&gt;Caenorhabditis elegans&lt;/i&gt;, eggshell formation. Defects in any of these processes can lead to embryonic lethality. We examined six temperature-sensitive mutants as well as one nonconditional mutant that were previously identified in genetic screens as either embryonic lethal (maternal-effect or zygotic lethal) or eggshell defective. The responsible molecular lesion for each had never been determined. After confirmation of temperature sensitivity and lethality, we performed whole-genome sequencing using a single-nucleotide polymorphism mapping strategy to pinpoint the molecular lesions. Gene candidates were confirmed by RNA interference phenocopy and/or complementation tests and one mutant was further validated by CRISPR (Clustered Regularly Interspaced Short Palidromic Repeats)/Cas9 gene editing. This approach identified new alleles of several genes that had only been previously studied by RNA interference depletion. Our identification of temperature-sensitive alleles for all of these essential genes provides an extremely useful tool for further investigation for the &lt;i&gt;C. elegans&lt;/i&gt; community, such as the ability to address mutant phenotypes at various developmental stages and the ability to carry out suppressor/enhancer screens to identify other genes that function in a specific cellular process.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/05/01" />
<meta name="citation_mjid" content="ggg;5/5/1007" />
<meta name="citation_id" content="5/5/1007" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/5/1007" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/5/1007.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/5/1007.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/5/1007.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.017038" />
<meta name="citation_pmid" content="25740937" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Mutant Screen Report" />
<meta name="citation_firstpage" content="1007" />
<meta name="citation_lastpage" content="1019" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Aimee Jaramillo-Lambert" />
<meta name="citation_author_institution" content="National Institute of Diabetes and Digestive and Kidney Diseases, National Institutes of Health, Bethesda, Maryland 20892" />
<meta name="citation_author" content="Abigail S. Fuchsman" />
<meta name="citation_author_institution" content="National Institute of Diabetes and Digestive and Kidney Diseases, National Institutes of Health, Bethesda, Maryland 20892" />
<meta name="citation_author" content="Amy S. Fabritius" />
<meta name="citation_author_institution" content="National Institute of Diabetes and Digestive and Kidney Diseases, National Institutes of Health, Bethesda, Maryland 20892" />
<meta name="citation_author" content="Harold E. Smith" />
<meta name="citation_author_institution" content="National Institute of Diabetes and Digestive and Kidney Diseases, National Institutes of Health, Bethesda, Maryland 20892" />
<meta name="citation_author" content="Andy Golden" />
<meta name="citation_author_institution" content="National Institute of Diabetes and Digestive and Kidney Diseases, National Institutes of Health, Bethesda, Maryland 20892" />
<meta name="citation_author_email" content="andyg@mail.nih.gov" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. A. Arribere;citation_author=R. T. Bell;citation_author=B. X. H. Fu;citation_author=K. L. Artiles;citation_author=P. S. Hartman;citation_title=Efficient marker-free recovery of custom genetic modifications with CRISPR/Cas9 in Caenorhabditis elegans.;citation_volume=198;citation_year=2014;citation_pmid=25161212;citation_doi=10.1534/genetics.114.169730" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=N. R. Ashcroft;citation_author=M. E. Kosinski;citation_author=D. Wickramasinghe;citation_author=P. J. Donovan;citation_author=A. Golden;citation_title=The four cdc25 genes from the nematode Caenorhabditis elegans.;citation_pages=59-66;citation_volume=214;citation_year=1998;citation_issue=1-2;citation_pmid=9651482;citation_doi=10.1016/S0378-1119(98)00228-5" />
<meta name="citation_reference" content="citation_journal_title=British Medical Bulletin;citation_journal_abbrev=Br Med Bull;citation_author=S. Brenner;citation_title=THE GENETICS OF BEHAVIOUR;citation_pages=269-271;citation_volume=29;citation_year=1973;citation_issue=3;citation_pmid=4807330;citation_doi=10.1093/oxfordjournals.bmb.a071019" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=R. Cassada;citation_author=E. Isnenghi;citation_author=M. Culotti;citation_author=G. von Ehrenstein;citation_title=Genetic analysis of temperature-sensitive embryogenesis mutants in Caenorhabditis elegans.;citation_pages=193-205;citation_volume=84;citation_year=1981;citation_issue=1;citation_pmid=7250492;citation_doi=10.1016/0012-1606(81)90383-3" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=J. S.-C. Chu;citation_author=S.-Y. Chua;citation_author=K. Wong;citation_author=A. M. Davison;citation_author=R. Johnsen;citation_title=High-throughput capturing and characterization of mutations in essential genes of Caenorhabditis elegans.;citation_pages=361;citation_volume=15;citation_year=2014;citation_pmid=24884423;citation_doi=10.1186/1471-2164-15-361" />
<meta name="citation_reference" content="citation_journal_title=Roux’s Arch. Dev. Biol.;citation_author=K. T. R. Denich;citation_author=E. Schierenberg;citation_author=E. Isnenghi;citation_author=R. Cassada;citation_title=Cell-lineage and developmental defects of temperature-sensitive embryonic arrest mutants of the nematode Caenorhabditis elegans.;citation_pages=164-179;citation_volume=193;citation_year=1984;citation_doi=10.1007/BF00848892" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=D. J. Dickinson;citation_author=J. D. Ward;citation_author=D. J. Reiner;citation_author=B. Goldstein;citation_title=Engineering the Caenorhabditis elegans genome using Cas9-triggered homologous recombination.;citation_pages=1028-1034;citation_volume=10;citation_year=2013;citation_pmid=23995389;citation_doi=10.1038/nmeth.2641" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=M. Doitsidou;citation_author=R. J. Poole;citation_author=S. Sarin;citation_author=H. Bigelow;citation_author=O. Hobert;citation_title=C. elegans mutant identification with a one-step whole-genome-sequencing and SNP mapping strategy.;citation_pages=e15435-e15435;citation_volume=5;citation_year=2010;citation_issue=11;citation_pmid=21079745;citation_doi=10.1371/journal.pone.0015435" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. S. Edgar;citation_author=I. Lielausis;citation_title=TEMPERATURE-SENSITIVE MUTANTS OF BACTERIOPHAGE T4D: THEIR ISOLATION AND GENETIC CHARACTERIZATION;citation_pages=649-662;citation_volume=49;citation_year=1964;citation_issue=4;citation_pmid=14156925" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. S. Edgar;citation_author=G. H. Denhardt;citation_author=R. H. Epstein;citation_title=A COMPARATIVE GENETIC STUDY OF CONDITIONAL LETHAL MUTATIONS OF BACTERIOPHAGE T4D;citation_pages=635-648;citation_volume=49;citation_year=1964;citation_issue=4;citation_pmid=14156924" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=L. H. Hartwell;citation_author=J. Culotti;citation_author=J. R. Pringle;citation_author=B. J. Reid;citation_title=Genetic Control of the Cell Division Cycle in Yeast: A model to account for the order of cell cycle events is deduced from the phenotypes of yeast mutants;citation_pages=46-51;citation_volume=183;citation_year=1974;citation_issue=4120;citation_pmid=4587263;citation_doi=10.1126/science.183.4120.46" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=R. M. Hecht;citation_author=S. M. Wall;citation_author=D. F. Schomer;citation_author=J. A. Oró;citation_author=A. H. Bartel;citation_title=DNA replication may be uncoupled from nuclear and cellular division in temperature-sensitive embryonic lethal mutants of Caenorhabditis elegans.;citation_pages=183-191;citation_volume=94;citation_year=1982;citation_issue=1;citation_pmid=7152102;citation_doi=10.1016/0012-1606(82)90081-1" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=L. M. Hereford;citation_author=L. H. Hartwell;citation_title=Sequential gene function in the initiation of Saccharomyces cerevisiae DNA synthesis.;citation_pages=445-461;citation_volume=84;citation_year=1974;citation_issue=3;citation_pmid=4618856;citation_doi=10.1016/0022-2836(74)90451-3" />
<meta name="citation_reference" content="citation_journal_title=Genetic Maps;citation_author=R. K. Herman;citation_author=H. R. Horvitz;citation_author=D. L. Riddle;citation_title=The nematode Caenorhabditis elegans.;citation_pages=183-193;citation_volume=1;citation_year=1980" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=D. Hirsh;citation_author=R. Vanderslice;citation_title=Temperature-sensitive developmental mutants of Caenorhabditis elegans.;citation_pages=220-235;citation_volume=49;citation_year=1976;citation_issue=1;citation_pmid=943345;citation_doi=10.1016/0012-1606(76)90268-2" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. H. Holway;citation_author=C. Hung;citation_author=W. M. Michael;citation_title=Systematic, RNA-interference-mediated identification of mus-101 modifier genes in Caenorhabditis elegans.;citation_volume=169;citation_year=2005;citation_pmid=15654100;citation_doi=10.1534/genetics.104.036137" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=N. Homer;citation_author=B. Merriman;citation_author=S. F. Nelson;citation_title=BFAST: an alignment tool for large scale genome resequencing.;citation_pages=e7767-e7767;citation_volume=4;citation_year=2009;citation_issue=11;citation_pmid=19907642;citation_doi=10.1371/journal.pone.0007767" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. C. Johnsen;citation_author=D. L. Baillie;citation_title=Genetic analysis of a major segment [LGV(left)] of the genome of Caenorhabditis elegans.;citation_pages=735-752;citation_volume=129;citation_year=1991;citation_issue=3;citation_pmid=1752418" />
<meta name="citation_reference" content="citation_journal_title=Mol. General Genet.;citation_author=R. C. Johnsen;citation_author=S. J. Jones;citation_author=A. M. Rose;citation_title=Mutational accessibility of essential genes on chromosome I(left) in Caenorhabditis elegans.;citation_pages=239-252;citation_volume=263;citation_year=2000;citation_doi=10.1007/s004380051165" />
<meta name="citation_reference" content="citation_journal_title=Methods (San Diego, Calif.);citation_journal_abbrev=Methods;citation_author=R. S. Kamath;citation_author=J. Ahringer;citation_title=Genome-wide RNAi screening in Caenorhabditis elegans.;citation_pages=313-321;citation_volume=30;citation_year=2003;citation_issue=4;citation_pmid=12828945;citation_doi=10.1016/S1046-2023(03)00050-1" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=K. J. Kemphues;citation_author=M. Kusch;citation_author=N. Wolf;citation_title=Maternal-effect lethal mutations on linkage group II of Caenorhabditis elegans.;citation_pages=977-986;citation_volume=120;citation_year=1988;citation_issue=4;citation_pmid=3224814" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. Kim;citation_author=T. Ishidate;citation_author=K. S. Ghanta;citation_author=M. Seth;citation_author=D. Conte;citation_title=A Co-CRISPR strategy for efficient genome editing in Caenorhabditis elegans.;citation_volume=197;citation_year=2014;citation_pmid=24879462;citation_doi=10.1534/genetics.114.166389" />
<meta name="citation_reference" content="citation_journal_title=Journal of Cell Science;citation_journal_abbrev=J. Cell Sci.;citation_author=J. Kim;citation_author=I. Kawasaki;citation_author=Y.-H. Shim;citation_title=cdc-25.2, a C. elegans ortholog of cdc25, is required to promote oocyte maturation;citation_pages=993-1000;citation_volume=123;citation_year=2010;citation_issue=6;citation_pmid=20200231;citation_doi=10.1242/jcs.060442" />
<meta name="citation_reference" content="citation_journal_title=Molecules and cells;citation_journal_abbrev=Mol Cells;citation_author=S. Kim;citation_author=D.-H. Park;citation_author=J. Shim;citation_title=Thymidylate synthase and dihydropyrimidine dehydrogenase levels are associated with response to 5-fluorouracil in Caenorhabditis elegans.;citation_pages=344-349;citation_volume=26;citation_year=2008;citation_issue=4;citation_pmid=18612238" />
<meta name="citation_reference" content="citation_journal_title=Virulence;citation_author=K.-Z. Lee;citation_author=M. Kniazeva;citation_author=M. Han;citation_author=N. Pujol;citation_author=J. J. Ewbank;citation_title=The fatty acid synthase fasn-1 acts upstream of WNK and Ste20/GCK-VI kinases to modulate antimicrobial peptide expression in C. elegans epidermis.;citation_pages=113-122;citation_volume=1;citation_year=2010;citation_pmid=21178429;citation_doi=10.4161/viru.1.3.10974" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The Sequence Alignment/Map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=P. M. Meneely;citation_author=R. K. Herman;citation_title=SUPPRESSION AND FUNCTION OF X-LINKED LETHAL AND STERILE MUTATIONS IN CAENORHABDITIS ELEGANS;citation_pages=65-84;citation_volume=97;citation_year=1981;citation_issue=1;citation_pmid=7196363" />
<meta name="citation_reference" content="citation_journal_title=Molecular genetics and genomics :  MGG;citation_journal_abbrev=Mol Genet Genomics;citation_author=R. Minasaki;citation_author=A. Streit;citation_title=MEL-47, a novel protein required for early cell divisions in the nematode Caenorhabditis elegans.;citation_pages=315-328;citation_volume=277;citation_year=2007;citation_issue=3;citation_pmid=17171368;citation_doi=10.1007/s00438-006-0191-z" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. Minevich;citation_author=D. S. Park;citation_author=D. Blankenberg;citation_author=R. J. Poole;citation_author=O. Hobert;citation_title=CloudMap: a cloud-based pipeline for analysis of mutant genome sequences.;citation_volume=192;citation_year=2012;citation_pmid=23051646;citation_doi=10.1534/genetics.112.144204" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=J. Miwa;citation_author=E. Schierenberg;citation_author=S. Miwa;citation_author=G. von Ehrenstein;citation_title=Genetics and mode of expression of temperature-sensitive mutations arresting embryonic development in Caenorhabditis elegans.;citation_pages=160-174;citation_volume=76;citation_year=1980;citation_issue=1;citation_pmid=7380089;citation_doi=10.1016/0012-1606(80)90369-3" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_author=G. Nair;citation_author=T. Walton;citation_author=J. I. Murray;citation_author=A. Raj;citation_title=Gene transcription is coordinated with, but not dependent on, cell divisions during C. elegans embryonic fate specification.;citation_volume=140;citation_year=2013;citation_pmid=23863485;citation_doi=10.1242/dev.098012" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=S. Nakano;citation_author=B. Stillman;citation_author=H. R. Horvitz;citation_title=Replication-coupled chromatin assembly generates a neuronal bilateral asymmetry in C. elegans.;citation_pages=1525-1536;citation_volume=147;citation_year=2011;citation_issue=7;citation_pmid=22177093;citation_doi=10.1016/j.cell.2011.11.053" />
<meta name="citation_reference" content="citation_title=Genetics;citation_author=A. Paix;citation_author=Y. Wang;citation_author=H. E. Smith;citation_author=C.-Y. S. Lee;citation_author=D. Calidas;citation_title=Scalable and versatile genome editing using linear DNAs with micro-homology to Cas9 Sites in Caenorhabditis elegans.;citation_pages=170423;citation_volume=114;citation_year=2014" />
<meta name="citation_reference" content="R Development Core Team, 2014 R: A language and environment for statistical computing. Available at: www.r-project.org/. Accessed March 14, 2015." />
<meta name="citation_reference" content="citation_journal_title=BMC developmental biology [electronic resource];citation_journal_abbrev=BMC Dev Biol;citation_author=C. A. Rappleye;citation_author=A. Tagawa;citation_author=N. Le Bot;citation_author=J. Ahringer;citation_author=R. V. Aroian;citation_title=Involvement of fatty acid pathways and cortical interaction of the pronuclear complex in Caenorhabditis elegans embryonic polarity.;citation_pages=8-8;citation_volume=3;citation_year=2003;citation_pmid=14527340;citation_doi=10.1186/1471-213X-3-8" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. M. Rose;citation_author=D. L. Baillie;citation_title=GENETIC ORGANIZATION OF THE REGION AROUND UNC-15 (I), A GENE AFFECTING PARAMYOSIN IN CAENORHABDITIS ELEGANS;citation_pages=639-648;citation_volume=96;citation_year=1980;citation_issue=3;citation_pmid=7262541" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=E. Schierenberg;citation_author=J. Miwa;citation_author=G. von Ehrenstein;citation_title=Cell lineages and developmental defects of temperature-sensitive embryonic arrest mutants in Caenorhabditis elegans.;citation_pages=141-159;citation_volume=76;citation_year=1980;citation_issue=1;citation_pmid=7380087;citation_doi=10.1016/0012-1606(80)90368-1" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. C. Sigurdson;citation_author=G. J. Spanier;citation_author=R. K. Herman;citation_title=CAENORHABDITIS ELEGANS DEFICIENCY MAPPING;citation_pages=331-345;citation_volume=108;citation_year=1984;citation_issue=2;citation_pmid=6500256" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=S. Smith;citation_author=B. Stillman;citation_title=Purification and characterization of CAF-I, a human cell factor required for chromatin assembly during DNA replication in vitro.;citation_pages=15-25;citation_volume=58;citation_year=1989;citation_issue=1;citation_pmid=2546672;citation_doi=10.1016/0092-8674(89)90398-X" />
<meta name="citation_reference" content="citation_journal_title=Sub-cellular biochemistry;citation_journal_abbrev=Subcell Biochem;citation_author=M. Sokka;citation_author=S. Parkkinen;citation_author=H. Pospiech;citation_author=J. E. Syväoja;citation_title=Function of TopBP1 in genome stability.;citation_pages=119-141;citation_volume=50;citation_year=2010;citation_pmid=20012580;citation_doi=10.1007/978-90-481-3471-7_7" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=L. Timmons;citation_author=D. L. Court;citation_author=A. Fire;citation_title=Ingestion of bacterially expressed dsRNAs can produce specific and potent genetic interference in Caenorhabditis elegans.;citation_pages=103-112;citation_volume=263;citation_year=2001;citation_issue=1-2;citation_pmid=11223248;citation_doi=10.1016/S0378-1119(00)00579-5" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=R. Vanderslice;citation_author=D. Hirsh;citation_title=Temperature-sensitive zygote defective mutants of Caenorhabditis elegans.;citation_pages=236-249;citation_volume=49;citation_year=1976;citation_issue=1;citation_pmid=943346;citation_doi=10.1016/0012-1606(76)90269-4" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=K. Wang;citation_author=M. Li;citation_author=H. Hakonarson;citation_title=ANNOVAR: functional annotation of genetic variants from high-throughput sequencing data.;citation_volume=38;citation_year=2010;citation_pmid=20601685;citation_doi=10.1093/nar/gkq603" />
<meta name="citation_reference" content="citation_title=Identification of suppressors of mbk-2/DYRK by whole-genome sequencing.;citation_author=Y. Wang;citation_author=J. T. Wang;citation_author=D. Rasoloson;citation_author=M. L. Stitzel;citation_author=K. F. O’Connell;citation_pages=231-241;citation_volume=4;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=W. B. Wood;citation_author=R. Hecht;citation_author=S. Carr;citation_author=R. Vanderslice;citation_author=N. Wolf;citation_title=Parental effects and phenotypic characterization of mutations that affect early development in Caenorhabditis elegans.;citation_pages=446-469;citation_volume=74;citation_year=1980;citation_issue=2;citation_pmid=7371984;citation_doi=10.1016/0012-1606(80)90445-5" />
<meta name="citation_reference" content="citation_journal_title=Biochemical genetics;citation_journal_abbrev=Biochem Genet;citation_author=D. A. Wright;citation_author=C. R. Shaw;citation_title=Time of expression of genes controlling specific enzymes in Drosophila embryos.;citation_pages=385-394;citation_volume=4;citation_year=1970;citation_issue=3;citation_pmid=5477232;citation_doi=10.1007/BF00485755" />
<meta name="citation_reference" content="citation_journal_title=Advances in genetics;citation_journal_abbrev=Adv Genet;citation_author=T. R. Wright;citation_title=The genetics of embryogenesis in Drosophila.;citation_pages=261-395;citation_volume=15;citation_year=1970;citation_pmid=4936425;citation_doi=10.1016/S0065-2660(08)60075-9" />
<meta name="citation_reference" content="citation_journal_title=Cell Res.;citation_author=P. Zhao;citation_author=Z. Zhang;citation_author=H. Ke;citation_author=Y. Yue;citation_author=D. Xue;citation_title=Oligonucleotide-based targeted gene editing in C. elegans via the CRISPR/Cas9 system.;citation_pages=247-250;citation_volume=24;citation_year=2014;citation_pmid=24418757;citation_doi=10.1038/cr.2014.9" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Rapid and Efficient Identification of Caenorhabditis elegans Legacy Mutations Using Hawaiian SNP-Based Mapping and Whole-Genome Sequencing" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5.cover-source.jpg" />
<meta name="twitter:description" content="The production of viable embryos requires the coordination of many cellular processes, including protein synthesis, cytoskeletal reorganization, establishment of polarity, cell migration, cell division, and in Caenorhabditis elegans , eggshell formation. Defects in any of these processes can lead to embryonic lethality. We examined six temperature-sensitive mutants as well as one nonconditional mutant that were previously identified in genetic screens as either embryonic lethal (maternal-effect or zygotic lethal) or eggshell defective. The responsible molecular lesion for each had never been determined. After confirmation of temperature sensitivity and lethality, we performed whole-genome sequencing using a single-nucleotide polymorphism mapping strategy to pinpoint the molecular lesions. Gene candidates were confirmed by RNA interference phenocopy and/or complementation tests and one mutant was further validated by CRISPR (Clustered Regularly Interspaced Short Palidromic Repeats)/Cas9 gene editing. This approach identified new alleles of several genes that had only been previously studied by RNA interference depletion. Our identification of temperature-sensitive alleles for all of these essential genes provides an extremely useful tool for further investigation for the C. elegans community, such as the ability to address mutant phenotypes at various developmental stages and the ability to carry out suppressor/enhancer screens to identify other genes that function in a specific cellular process." />
<meta name="og-title" property="og:title" content="Rapid and Efficient Identification of Caenorhabditis elegans Legacy Mutations Using Hawaiian SNP-Based Mapping and Whole-Genome Sequencing" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/5/1007" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The production of viable embryos requires the coordination of many cellular processes, including protein synthesis, cytoskeletal reorganization, establishment of polarity, cell migration, cell division, and in Caenorhabditis elegans , eggshell formation. Defects in any of these processes can lead to embryonic lethality. We examined six temperature-sensitive mutants as well as one nonconditional mutant that were previously identified in genetic screens as either embryonic lethal (maternal-effect or zygotic lethal) or eggshell defective. The responsible molecular lesion for each had never been determined. After confirmation of temperature sensitivity and lethality, we performed whole-genome sequencing using a single-nucleotide polymorphism mapping strategy to pinpoint the molecular lesions. Gene candidates were confirmed by RNA interference phenocopy and/or complementation tests and one mutant was further validated by CRISPR (Clustered Regularly Interspaced Short Palidromic Repeats)/Cas9 gene editing. This approach identified new alleles of several genes that had only been previously studied by RNA interference depletion. Our identification of temperature-sensitive alleles for all of these essential genes provides an extremely useful tool for further investigation for the C. elegans community, such as the ability to address mutant phenotypes at various developmental stages and the ability to carry out suppressor/enhancer screens to identify other genes that function in a specific cellular process." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5.cover-source.jpg" />
    <title>Rapid and Efficient Identification of Caenorhabditis elegans Legacy Mutations Using Hawaiian SNP-Based Mapping and Whole-Genome Sequencing | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"WIecrY3mAYDFZ4C5pCbbUdRc6FIDWtOxB_K2KcW5dNw","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459827","apath":"\/ggg\/5\/5\/1007.atom","pisa":"ggg;5\/5\/1007","ac":{"\/ggg\/5\/5\/1007.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/5\/1007.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/5\/1007"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/5\/1007"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/5\/1007":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022r9vtJFagwe9UbYQYlxZl\\\\\\\/+\\\\\\\/QviNnY5jiOa1GIpPtzk8PMc9BDsTMkEK4AGKc+aHOLBA1kUvc5B\\\\\\\/wcXy+VBiC+esg0DG4BAbU1\\\\\\\/jY3C58u0U1ducB1aFzRfZCjKp7xfZkEDq0DD2kd71JrI8M7HZeSsxrc1tPqkBuhj2BKiXWM54B+adFo1jhrDB3JsahIhAq2dgwEbf+Gw88jhPbMx0TDhoDj9wzJiJQY1RaqzDcoQN4e6hOdP9QZ+ov+C8c4C6WhGVy8XJnUt9zVw\\\\\\\/\\\\\\\/ax2sQpokG9DXxfoFRu2AN4qAVMcZ0EnDnqZYjk21jUHGc1ax8ZGgFoT0rSKKRkD40CnFaDlxDa8CxUgd2sKBWN18XUe+6+FufH2buHTMG9I2AL7KwAeJVCL6rVRJ9IQP4StUeu\\\\\\\/YmKUkQZADTruMs9pMxbUzpLcSoBUV0+xCZp4SBgrdCTfD9t9nWy6B4Pqzna8WfSUMnYOiSBvWCkTY\\\\\\\/\\\\\\\/QTU7OynOnp83okzI9Dm\\\\\\\/nPd\\\\\\\/2aXZetNKYfCnfA\\\\\\\/VlyWwwA2fxqQvSNSay3DsX12ADpsOszWe59D2BBIwzal\\\\\\\/N+uX5MYCN0Nprf18JKe06WuI5q05+N6dr6PlwH40kzgQl03+IEdW4BGMRBQbH7f4HWFY2QDbuZM4nH3EEj0lSaCl+uXLLXK17XQ0Nbmo7D0KFqylBm0K8dcnCgakZA4j\\\\\\\/SzeGds9ICZK9CXgvY+otphRaFQflRt9YmRD22VcAhK8dQLLnuIFRuD22xd1wvju3isn8rNXaPkJZrul39q41giZeDE8Ky32a0ZJmBv1vtaVPBWrnI\\\\\\\/pgGYDMX8u7ZJFH6o+ZUflvAbNE8eV0\\\\\\\/PauBRVkyhR1rW9LTN4oSvPb1OdGa7KVrZ9\\\\\\\/euH6Tb9M9PdzwbQ3Y0i1igvthhln8aowZ0uPy1w8+NunSKHABfNd8oHFvJV7zuGZ47gd9NmP1qaQbwaJKeYf0KqgI0YFrms3DyX040NmPQqZKjsCf+asJX2LGBM9KRWbJFrKtllYeliNi2oU+D5jpi120QHYNDAxw5R4XcQJ9P9wJW8iusFdUqDgVKALd\\\\\\\/U+sTaaGBCPdnNIq3Yr61A+nxJ9Sd9Ewrmt7ADXpl+\\\\\\\/BsySqnSnctCNsPhRbi0aPLat\\\\\\\/4h\\\\\\\/JuGNn1+pyuxKEcVINPko27d5JgMA6WQRA2ts3vPKS1jwZdDuBCjFecy\\\\\\\/EysizTsyWqvL5nm2MA+r9ikziykQWFj1FHsackPyate7jwT+pEjux4IivwZd7FsLu\\\\\\\/PqyyfqdUoAX8dYzRyFN\\\u0022,\\\u0022iv\\\u0022:\\\u0022NmjfjBLhM91uxUfiPpNQyA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002230d26ef55fc1e623f4e3b9811fc20e540c33fc80ea70aeed4d667f8bbda41143\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00229RXOgwoYZtI9ZX7IsVM95NnEx9CspPOfQEjDFAUOtjuiyaB1NYEewEsnjozkFRTvAJPo52wxMEMoskoADLfxK94F+HHLVwy2+VH+GGVvr\\\\\\\/PcEqaqryQxx\\\\\\\/KiqFdofs1ILZJq4zjWO1BOSF0oCQEK9s16AiRistO94fNPwNyjM7vFAAn2gHWkCZOoTyYShfM2L8lgcEHZV1HRHD2lKUfLPojNaYbStZcxbkJsKPADiOJ1h7hUdBGxz+hjzLLDzCoZM\\\\\\\/AoTQUjGhFqgNRB7GKuMABs6OVCJR5\\\\\\\/3uN3smkXQseodbVaB1e90JhWI0pLIiIJcsLHrU5cgcevIS3kpMlsujgI2yq6JCks6kuaFW49TMbhDG0BYsShYnb67JtoVvNaYePW2ttOsKqZSQAPaPJpaL2btfcH7NgPi7lB\\\\\\\/YDMDtkv0Gv8uvM6R+\\\\\\\/RTcXVZwOkucb44KffzD7FUUNWZWR1Vn2yvWprzm4PgQ72U9H3grCEFK75eveHz8Mt3ucSe3X5IJB8zZ4+Lq1uRqZqUKykdxhwwAaVzaVLH6OymETfjtwSh3lyM1AI7EiVPp1vYQJyjBy474gJEaopqHf43o6OeAVhnbOyNZmidOi2q3TB5h8EQwgjaAkqZVch6GP4On1\\\\\\\/VGO39UwIGDllnZYcooxbq\\\\\\\/BNDwEZ7gahVJbed6luCXdd85Vepg7lKxdihf\\\\\\\/Jd787eFuH4DNfExODOqAF6Kuon9Ba+1QenobWkhfreqgfNC4VDh3uJsgTwEDxi+lqwagZQPa8RTZGDVMj6WqjdjupUmgX7JGzyut2oNqdtQvHgA9tX3I+JMXz\\\\\\\/FossvyIzKHV0Br203cIFUlXcSptui+rNvEkQWZNnwqiPsvH7R\\\\\\\/fYzicyN7oB8rVy0R7bi24KRc136nBRTeVhDHXXhKmY42+Gh286BOX6UmSK3yBEPrXw3sW+sN+FqEvIj2ADlI4drVG+ofyialsiXkV5NanMKSIqW+do9AH41RzH7C4iQvCKUOMjp9Axu5wahtRZEJA5UscwJs2PoLniDR\\\\\\\/dLm2y\\\\\\\/+S4L4AJtkUlGOvxVlr5Xzv41g\\\\\\\/vCwrU\\\\\\\/b\\\\\\\/3rvAm8mr8fIRZ0vXcEYAgPAVBzljlpKFpiFu5BCJyTEJmUjA8VmzXJOF7q2IKnEixkIRxMH70ru+F1HyVcar\\\\\\\/lDC7c2Sve+tTKhFrmAzeIOTyFRKOaHL99X3dcNwztgqrpL9o\\\\\\\/7hWGBO4c0mYBNrj9hCaCaNvQXU0y2Uy+UptP1gnStl+uY6BgnxCxmRrIy3OrLlBYaYVkDw0Nq9bVFx92eabio5dggMvzVFBlmirojKxha4QnKdObKE2uLkfY5Iis+uM3Tl38KNx2BP3GYvz2TxN6lG8JRgCHSknMhD4t9KK0N\\\\\\\/I8pzGkRrEysTC\\\\\\\/GaU2SKUi641UYt95mNnBKdHFl88rYgRrUTR2aN5wqU\\\\\\\/aymTXHvWbpkw4tbwfyayvk62Pt7gh9lqYTwt5buZMy+Doavfw6MErtoHVtMRh0KP0jLNA5UobKyy07pxRb7J1qwFN4D9ETapVo\\\\\\\/Xp26p3uY6WfCfE9epX5hXGfwrXHFnGpjYubCO9QzFkUBUf+KB4qLh81qXEfYUyIPUguW+8cR1gWIySebbSctoj6c395w97tZn9r4wXkKSlTNg1x3XlTgCp2PmD5qvzXHZ8z+bOoCB8KeE9Zx53ZqGJU3HO\\\\\\\/CEdebCLrRRMm36Q\\\\\\\/iC7DjDPiOLpBW4EEd4wSMCsk7i1lpdTM2row87w6ODdBVkbHi91jTae6IxJO34JKrL\\\\\\\/1rEbKmugk8SbpE7Irm9c3bMZfHffi18vsD0jgMab6EZSoJACeTzvJWnARnktb\\\\\\\/Nj7r7a5fOJoMPDNInXpGQ4KyIMt4Ra7s+N6kICTMMTBoZFdpVm541hrYUnbtX\\\\\\\/Gckepw0WU\\\\\\\/gbfazVxpMcUd0f5yBqjjXNX1wQ053IukuzbLyScs3eedcKhvDsyEcDwBmftFut6LXO7OM0iQF6B+Z0q5A44N9IDlOYK5PeZ2JNRu7CRsnS0QdD3zNUcTxc3gtvOqoEZzE6NZKeJunQq8GJB\\\\\\\/SMzsHY4EBVhMsVzU6g8m7vxOfE7dw2Od+Iwv+2n7Xs+4+0vWMj7tHovkV4TTtfrMSkEGnghA0ihfpDtAxr7\\\\\\\/tTUt7CEymX9tujBQtx\\\\\\\/Kjza3nyjKxXv\\\\\\\/qt+INGp2Bmv32dCXoSy4KcolXWCN\\\\\\\/JAwL9RpUG1CNPRkla7iH3sa+GOsq0Q+LKItdA5S8i8ywQl2yut+A5xVb9l0l90WEte3U9XNYvuu0obnlpRwig413iHL6aUeSAdsKkP0N+FLJbdxYhrNkAlkykQgfpElcP+Xdnx8Xc7iPxTN+kUHZ1BnoIyPIfMbCNa+mPuD+Bt4UA8Yh35AiU37zF4T4FM6C+CVpiEq1wWrG+AD2X+9xrYR7VaDE5jUXXYp6WhNoEBMiVcMSOa7P\\\\\\\/ryvP6umOeoLORep7Zi209vfTFtoMkRrdQmuYYFYxarSC75ZivAdBML2uFOSOHm3uaUkfje0Y46ZsYU+FZir0s\\\\\\\/2U1Mn6awa4bqN3gDqHRDxqLfzI0VHD19YjLYWSaYOYbIVyb62U82GclcOHC+3OTEWCbqlaJ3FpPo3QrK+HfwieM8ElUHwTJH6Os1OaBCO3o2qKmNEoONHnHCR5M9fOuU4rJwwB\\\\\\\/omu1p2Eqve9DG3iepMs8Xmy6oPCqO5sW+vZjSdGCSOUKYujDbOfE6o5J+P2CheyQQK\\\\\\\/PRdhd4JQOmBHoeAJG\\\\\\\/OFlxY3XUhQVr6u8P6O3CAA62IYl17YZgsoBrBfutfNQJXACM2\\\\\\\/moqqyrNl5SgCMkI19ddcnRKRAr2gtgN9qS6ZzM8nMNHWTU\\\\\\\/8o9YJA2tALbK0Iq4Cx5oJ7QSX\\\\\\\/0Numd\\\\\\\/2sWkYxVpPpgfVNjuXvec3DL6V0w7MQruK554f0QU07b9n+OWNzW98xa90WbUg5dD3diHubanpO0W3HUlnCLj2NXuLcPAxvG5NhfCZa3xpUHkVHscviNw8nqX74gaVmPg5fAMvROu2719N\\\\\\\/zVr7a75G7g8HonUVL+T+kA6QC7W2hATby\\\\\\\/l\\\\\\\/b7liMo7I7c+l5XA34TmMb9gsOno\\\\\\\/1P4NZr69qP+k1hxdU\\\\\\\/GtCoWZo+kLO6Zi4GINUcEqUwwwlUznoRMb6GGsv70y\\\\\\\/whco8XXWC7QL8jy8gfdFwha0sCUyjlQu3ftnJ8KlqB5TSiAVQtyS5h4Pt7HYXXheElkILmz3IAsqRIqaTHCT94ahgvi+YhRiftfFZEyFwBMVqqp7xtAGSY7DsbACzJbI5nWKW+Sm1TQrz0BQiWQm\\\\\\\/kxpELniRJ2awt4hYaQTZNKgdSCD5Fhh6IK4sYTPY+aXNobtyMmGLsmG\\\\\\\/KsHeOYsyTSnGCUFksT8BqD4ECB3T5C9d1m4DgRRIxlTPUTce3ZxGnPj+1teyqZdTJk15XPhKo+TZ0rRMdSCcAtQLU2nQVAkdxcoBFvUISsPSF7JJYk9xvV5BgBBB0VWu8+vPXI7725uW6Yfb7eSgm7nRIgBj9pkd3NfWOEbVDxXMbKAJ1Nb6ZR+tT4Lkf4qY\\\\\\\/orV+6k6x6RgsyPL8IRgEzeI5Asgr5sDAq4VNp32FYbnsm\\\\\\\/djh0G243LCQFa1Uk3mmXnCb2uH8ptnkFFI7ZbJ1VNNvPtxpmIrHjQha1DNEHEDQfjH3JRtiA1iiOrL90LGlq7W2eYYEibD0Krl1i\\\\\\\/8fJ3Al5ISKmEnU2Bbrh1TCGf6R4aVStds3W2xMV1AeU4Idgoy1eEEkDqiaI2NwtMeOtA4csrleQ\\\\\\\/zFol3ql9NEfBEE\\\\\\\/W3yFmEAhlycAu2RpuKU1Be\\\\\\\/JyiP6ml9CkkNqoKakVoa167CTaZp7O9LY0c3LvAZoH4KZbTaH9xbjGwMeYwrTFSc3z0eBqF0+\\\\\\\/0Zpbq8QDnEnaSnzL7+FwAl0xV7YXHzH9YROzo+zI0F1jnGYWfWhnAMlV7LHrXP7PiaVp0dWm\\\\\\\/T\\\\\\\/1lSIc9nFMgHHF+g5WEaPP5oFb6vzbO8UDAX6ya\\\\\\\/Y0\\\\\\\/uAbUxtaTdnBk0sYn3mkW+BZn8mcX6EMX6dJyxiVGT0z95PjNwG6O+lIoOFNW\\\\\\\/JFP\\\\\\\/u6kPtrK5z2w5qACQuGxAn77gCzE+irHt+QQQKccJmhyM1Id8rG2UfQrUlm1UYi4okO5Ucsg4iu8GDBtHT+THyJ1\\\\\\\/GEEs+AzMOBAryYtyFqNUZx5lqwjKXLCqa76K0AUT2ybYfp0Cjft4gzCSH2sU4UPouZc0ju08L6isUYYMFEtS7gr1M7H57BWTncVcdVgR1iou5X1Neh\\\\\\\/zPdxzG1uo0XLuzjVHd+AhkfzSKbi8k5kOVIBVaEJuV46MSYEZEyFMn+p+5QNi\\\\\\\/MeK\\\\\\\/kAGsx3euAiWwIRDdvbcYvy4slr4aENTJhsESGDaWDHhifAqx5ODlj0+os5zGCQLRNds\\\\\\\/+ShyGQheZenCMk5YymE+lIOWUWrbvKF3lauEC1zbxfalixzr8K+ZA7m3CsTpXEy++\\\\\\\/bOPOvu89JEEpWTDmQTcuxgHHm+srJbAmB+WQOrhTLuUP\\\\\\\/utnAUflMTs++XCRK+tgDy2p0aQ37iPnQwDWafbGrDQC6KknqtDnjLIFfdc1kz\\\\\\\/3BiuuiaHI9kmrq2qsPyoqZoJnAvwAl+X7roVNFOymXZQF+vtnJhdGFgMETIdG9lYVaUa47EtpoOt9uL3xmLS5ZnQKPxAY+vPowLfs28TYO09+W1rsIFpQ0tzbt4wQquv3SZ9QSb77KMWlHZz81UeOenziBkvBBYwSFZX5mc\\\\\\\/rTJNfKBCKKFJUqOCeibAkI9gfmesjWXVXmW6ilrneMEO6\\\\\\\/nSF+8nSkKDXFfHIoD7fkZc7l2LsTOZ6RHlz\\\\\\\/7gYty41PFoM1E0kmzodkS8VpRFJqS9PmFxK1JfMcs942\\\\\\\/oM5L7fERERCrKMXWxY1g5dL811yDEkHUbKFwXhss7cCdnUzp7NLtzYulRCIpxmD74M6TcmoUm1llhmZW0ctQ0cjJ+tEDY67nbyfd\\\\\\\/A3IpFxN1rVY6aZYSSmbNmftKKquJPsPCg2lgY0F6d5AxuDLRXZNtYrAE+GVUPJYtvfUG6U1nuxaOdLxT5gKk7ygyQkDrJHWGwYmyIPQqJ4IsuIyc4rgcRS9QZMRrWRZlJGV9zKCjA\\\\\\\/aa9ZuN8ShKFN\\\\\\\/VUWeV35a0D9XXMV99CVZ9llYQ1w9cN4gQADVTvtLz3iwioykRzkpnhUAOHAJuVaD62OhDLFVgyX51mKJiSt9OOpJuoqIhFI\\\\\\\/c7wRIeTuuyB3JJffGAU958GKvR+qp+iuHB6vPj1Gh\\\\\\\/4yr2\\\\\\\/bges5ugQ\\\\\\\/O2PF8d1WBN9Xi1NnsWpzkNcdrxSXVWW36zxhctj+etAVN9fizWL3x1KPKyoUkHy4GrGCNJm+q5bIMcufqt2OCjgDoSAXAnVAWFjMEUC+ICIwI9iE364BI4I+keFIm5jgsPUHeV+sXg0MmkGyILahPlP8f6OKySohFCHeYinGeozn2hV9dYAKln4AAP2cGcvb5qZLJXal8Z7SSeeMLRqTFYBtnqJAr0dJmlYD0j845FxFvMAGGBNWJwGMteX+XBqnGXq4XFzeS4NLWCW4wJXAZum8sXWRqh4e3FV6DS3ItWHFWtOmb9aKJzyJFX8ShubGcK3Y0zvjT1OtoR3LanVAboGlXA0W+8P02PxNYNA3+nqo\\\\\\\/QJE\\\\\\\/iaKxMKm3MRXha\\\\\\\/otJ3EkFz7OENylNy+dukaT0vplzqv1nPby3N6aIV8MsXXaYgiW1BrkYm8d8eFEE0r\\\\\\\/zq2zM9JfioAm1FMF5CZQhVy4AXnygxwY6EPBButjCHP9UsyB1SOwKs333+xULiRRfyYifNLhpnDiltq2p38q5TO0CKNuFR2ZxMfgqv6VG2XxI8+oIGUJ8UOtsZba4K02qG7XOD5ZJwudWqs9WTP5KjXP\\\\\\\/aSwicoRxVorXXHe\\\\\\\/FqpnBDopgy4Dvz73iF5PD6mUx71znhjNr5J8NVtlqtw7Blz+9sLpe6nqhE108FPt65fXp4XpUmUqUHeF3vZX\\\\\\\/WFcPRXpKNRP\\\\\\\/xGrl4o0Dzhmr3kgDkNpNn7XpikRLy9NV63P2KFpxx21nLKq8AUvSuZUtUmlWwksp\\\\\\\/PkameN4FjTqontYN5PGOg7FUKJciKr0DBKSLTMY2Jn5fxox9QKvfp54XKEdFa0v39PIQKKiwIQEb77X2JfAG9jqrc+8vGqLZAYxSbHMGIW07q3u2aJlnr2GDLCw7sMNc3LfJbzkIGDfUffgOg\\\\\\\/2aClAWEZ4kukhCYr9sti8yUlgw0yqeBcyI28WoVYsbm5UDtZv5ntPxwdd7\\\\\\\/kdQiLHY4z+W6JlxY8\\\\\\\/GyACRzA7oDK5SiU3MhhgL7Oj1KhaXPa86tLM27xNWYVvc6NDns80CHCzBvBAqZoMeru2q3A0be+K5PjnlRsDXrCg+q430sNNBR4vhel7Se4gZFQcg\\\\\\\/RMnK8QSIKrypr8Jrnt5YEun4MyfOycFYj4u+9L8F4hi17tiw2zJDddco+XeBvOTFYt5eYMwXuE1cOLQtZxf75M1\\\\\\\/l\\\\\\\/iLlEHhnzSsmu3yx01OcMq0OZ5wo2\\\\\\\/2NN9ngFhWwtubwQrC0TCqcx\\\\\\\/N1NsXs3f3rNZwQDSvTfWaAcyV7yZ9TZNYKI4tXqTkzSKKeEJtgqgq2sPpbOivAcZfWz+\\\\\\\/XqLSrddQ\\\\\\\/bgRa+NEO1VAKgYtwCkO5kaY37eERrD1c4wjjEHCnEVkw8hX9HLGhckO7cSubGT\\\\\\\/3FvD8yiDV+txjWoT9Iik6QdtR1DiqUY2cEdv+j5vw0UTcKI5aH94yBAUhAOVmcmg2y3vNWiC0nA13vAc3RN6eiyre\\\\\\\/04qwP4gNRLXWcL5w81xdgBcHVqgB+xZcqUFcvtXNp0uEIlOAHav3dLlDntVHD7s\\\\\\\/fiUb2BvAhMO+LQBSlOW5EbUQopaPSTgSX5WMEIy2C2UTIMULneFSZbYIjbLb8Gz3kZZJZap1EAT0VZYarA85CLVotXX4e22feK\\\\\\\/S7YmKXy3HgjFH0Fo\\\\\\\/ViS\\\\\\\/7sYFZ4CAql+6Hwvl9LrgkU0b3Ha85JXZohopTX+0dtC17LJbDNqSXF1o0n9eZHlxsje5QL0Fwrm+tK0Al8KOI6OUmjmpXAaVUq6+gj0YqY+Vmx6SgZRccJDMu+jMT0x+b+zKZWsK1woI8t6Su1JtVrEPStZQosKsP\\\\\\\/HhUir7rwq15mM9J4I11jIsyl7mI1+GNU6ZoG+7glLbsAywcRmFCpIRzFNPYXrt\\\\\\\/6\\\\\\\/AxZFenakREXN9ahEVNz\\\\\\\/TL8iTOcfrBG6HYcI\\\\\\\/y9fPxDWL2b6zGPWvZ5itALummWjLh16CN2hDEtWiyXqgqVTQvkUIlogXG3cjKzM65Fq7966yxN2SFKAVA+DaIUuji6ezuTk+YOkb0uw67brCV6ZYcqymtVfgLxtnXkvRCkQvWl6GkpMDqNVBRhUxxWOZjFTVq5044LCQHyCfvp9m0xsMTfytHhjjm1Ac5VV9UCX2FIeE7JPR51lO92J+mA1egoFADyPnIuxKOdS1q+\\\\\\\/v3XIyjNFc6DkFUkfmyUneS4RRZEJ6xeuxt5rBKk\\\\\\\/r8AVrb+N2nXgBn+l2j0iXaKViPR9QjsS6EAv9ulowxT2Oga12RCGxAcb++zoWoJCP6bHg3yKaxMDoJ1uqyA12AZ11k7xG6b\\\\\\\/qgBW+EdBWQC0BvXoYPcylw3bjpxQXbg9IiVAwO+tx31yRVcgnfBN0kF+cY36cxPU+SOCZDEaEJnrkCm68sluRZvIA7f7uoCOycl6F1qwvcM\\\\\\\/Lchsjlv7GuQ56\\\\\\\/5STeCUQQrzZmbpCSCSZlgj1mgNwX6oRLpEVJIV4eLExWFtaEp\\\\\\\/AheO4R1qlEUAfxTPVV07qkXGkRdLlIUp26PEfAWSx5Nk2OwqKIp8SmtVQ0xbRifr\\\\\\\/NyP7w2GGVAqJXFk3DmOxzEsYRDDSnJPjJrNY25JTsDmoKBAUsgE6X2JN0bIW7vbZMjzusvVSTnq7eizBPC26dr\\\\\\\/TIjJCUfuf2uhWb66THG3K9QmZN+auIAqWVqthKYuerjFuUAkbf7hNtB2qWtCtQx4diMrgMvyL12UQ\\\\\\\/CRLGxRcjI6H0YBgUIyyNJYXsfi+ovrIV4n1\\\\\\\/\\\\\\\/5LZMCxDPVoE3DJAnujDhEgRa\\\\\\\/K9bkZzNO43zLQTnUiA2ToTsQBHNXhLQy7ePTcyX03SCwxLrAKqm+DXF7HH\\\\\\\/gkQP8q3o3b7v5OuB1hykp2QT+ISVqP\\\\\\\/2+uYeQ2yPvQJ5l8NMheZVNryoDCXak3oSK7So14XCAQuY2blRX+RqoelpE4J1i6h7iBokuON3aFX00TLhRXpDTua3pp1e8QRHuy7yL3625K2Yal1QjWMRs4Gu4tFNhrHgGneu4aQQRgFpH1sSwq72hY9y62Jqcyr7DgGd1\\\\\\\/UWBmNP7GjQT21TiyoPHttNBiDl8I2y2jloM3x5joBDb94OHloMpPI8u\\\\\\\/zqixCvIIrJxOECW\\\\\\\/6hU1ZRE5QzUFGgUwnqGdZGoVvNetBuLta7bzzRyShbgKoSnQVlCdQ\\\\\\\/xvlxnxnIUoLB6xQbY1rM+Kf0Im7vFulySYTTqKUdIHzMm5LZsQwnJ\\\\\\\/QnQqexFpRFjuPPiFUvOJ5VuXDvo5xj27O1oKBBZJzquE2mpWDwl5XTX555fxKh8o+AraNcGOQtjgw3tsXCDSiEsMlWe0t9\\\\\\\/oyMYyTSnSHQrypTCaBHoGb+u332OF5K1GDxYo9XCj9N8aVHplzW3STVV+1JQhPB2e\\\\\\\/Q3c0vL1aAtgoId9M15DdbxLNg2FB7BlLsm1I4WFNMDwc6qHGYEHp29XIGVzQo5Xsd8bgDSQ5MVTjWhF+APrbNnjmDQqG4JQk9M5QL5P50+JP0st+IduG0MmwMSymL8slfNnYVXQktdlstHWlMsCJ7N0d4hQ8LLyYxhm2jyjwR8YJIXXQDdKeUSEjIZaU93Cg1Oua7vVyRttsljta11xWBOCh7A0kaOmuaTAtXo0RQLcS1osplcir30O1tP0mnO2C7prGCghkJq4X9Adgn24pESw19bCoLLgv8x4ksSPkGGprspBGDjmVFV527J1vj3jqRPAo2iLbt61LddVzmsMqRa0bpZrd++cSHOtSfNtJ4IWkqWCV4rYu8yYOt2Ap6vu2OdPsBq8DIuIo\\\\\\\/qwjXI0wLPfxyg8BbXvJUXjMfJaniTjo4HtOHR0cmxAs33qGJh5LnV120dMNZAt7duXBsj1fwQ5tcdpLbJD8DmsLV0KeKdWeoPsF+GRvJAH7wY9xmNl0zMFYBbJ8D9yNrS1YBFxUNFvu+SL4yXX\\\\\\\/0G4s6bS\\\\\\\/uBHSEJUrwVCKm0ZTH22YtwiAN74TJb7LE7Bh7Nrriy02K\\\\\\\/x6D49WxOv4q+sIeyHI6RUaRt5DEWgI+4AAMUgqS0sJoTQPNrtyFp\\\\\\\/TsVznBlSdW5bBbPpbOPSTGngql3AsuVDvch56gt6FrwjIiyI7X1WtaFq79NS\\\\\\\/l0s1hBSwsXnkGZ4cdnB6Phx52wWRxMfsl3yukR2J1KoXpuQR9tDVOCKdOIiPkiaQfdK46CJjBcK5SkS74jhhuG0If25tSXGyrMNOG\\\\\\\/iq987Olo4URuiIh6g5uwEoYdb3LQXHURQWMyeurwp4KDHnUNiKelZMVCsn9IqUNBG6BqDqZQexyp3mcGPRlzH3zvlaeHnzjRGvfRg5yPjcmsmpos2+f7iK568rYe6iPtWkf7v+f6PbMhuOQ7LaDWJnSIuICYa4FF8xmQoC709iJA22tnnAdqhAxwRUTh4DyzLZG+8tktn\\\\\\\/70K5OIRsKbh\\\\\\\/Pzv06qiujIkYnpy4+p7WbiNYXEXxfH3SLWBCqfR6Vji6F96Vcdg5hguYWnxl7HANqsPmS1wXrcHubw\\\\\\\/Jr793\\\\\\\/Sou99uiT40voiLEZPPBPDYhFOgkTLjDOFUG4vEhTEHR6raoX+ZQD44MZsdGvWsj2aQ1+wXXccpR93uBr68fCpfT2CugU\\\\\\\/cnJh7sIC4tDUcJ4ElkVffIvGfRMiVcqhzeFlpGEky66D3E9IPtCtlWiTeM7I4j3Zz5sWs3rueSX8cxmzgJDmZvL0va11IkaezLct+wdQytv7dzDcLVFJv5AqIqkEhuS8PSx3ipbqh2F3ytMYsymd5IUTz2lwQFR7fTtxLItIRYxkMKDp7DIwACjAlQXRIO6IVS+nAE1Ek3IM3zlSgt2YiH8vl1l3cy4os800\\\\\\\/s6k7ADyX0Gav3WAvcrsv7nj2e67zkeRpVI9jvRiSuQTvmflAOD0gDAxUPMWXwc\\\\\\\/tsdm1sQ0sIidJ3oECeKpjsVlLrvKaqLldgkyNHkh8jnlzQLEa0QnSQVRVRXqfxePRmZrcBfcXWonhvDRjgInl6qgjz3wm4T2cufurzFv5GUauX\\\\\\\/6xxQG2iQOrUkR+mVGl7Cq3NhPFcKRa3I2sfeMWuuULVLVfVIXl0Pwjlu8ajhj2okuRCI6lyKjL7Z7HfMjUqCyksQm3MOmDrdUtlIxgf+QSNS0FH7Mggc3Y7BJURY9VGAJ2czjDyN0u6NzRhrgQlwayQDuY9KqO5BU98deOiH3q8MxME2w9OzmCve2Ltce1MT+8uKomxgDqF2m9SvurrXIwxvaO+2jV4HcG83+dLU2DpMPUXgS6dosFc9BflTFcwvgpXD197g+IcwN4mAMtegkVGHc5SyW5Ir5CVpd9W9SEon57tYDreZvKIjgh5kGlrg17uj421\\\\\\\/oQjii+tQM\\\\\\\/wIUXpyrh4dqBR\\\\\\\/sCuKmH9L4j119N8RSXwbcTgBqJiB87aGmdjSPf0UNLSJK3TOcBzUbuhxufoiLlQZHJF9q7DI2aG0cEUOVj3s2mdtUOq6sg3vmJ9OTCheOQgvoI\\\\\\\/xFKUvgqY6KJ2LOU8gELZV7AT0cuIGjwTyn1LQVbWRKSoe56FRGPOB7mRmMPCsBQ42mczAHUeemCT\\\\\\\/DggBb18XYLD+CCod6tkZYSiy\\\\\\\/K8UXN\\\\\\\/1YLvCPaL0hFSzqjZOw44bH403eR6BMK5E17MPM8Nlap3Xq4pULIRU1Ct3wRtDCBpjDBuOgBt0L5oHGBrQdYpoz8KQ4ZZZWu9u+QKEAxScUr32kxtDoXlFeoND8ssrDLsRT5RFbbfZGDktCvvbZ0N9ZqM7sZtBnEdd\\\\\\\/9\\\\\\\/36l9Z1VfvaBYCa4a4fY58AEOEqOSJallAf0d8TtX0S1t5sIkXO44hAVk\\\\\\\/GzilohoQC45XhPevvr+orRrqpxUlaiPXwhJhpWnBrAwjO9sjA1UghD1PKKoctWwyhHhKMH9pbvraOb3s1Qn\\\\\\\/msitexVkKXZjI62kCD5JGA7SQ0IFWqphH+nH7TZCfXs+foj6s2PSKjq8aXICpcvPPgz66TfZX\\\\\\\/xvB+W5lLTc6Fh5x9JAXJk5hVxTh6g0M1wcKldkJ\\\\\\\/z50SaDJI7OkgQI6QtFQFrGuop7JoqIvG+EbFDeELbWM6XshFhis29ad6\\\\\\\/7iZk1EAQ3aS3tLZ4DzbMeD8AZkUQo8RC\\\\\\\/e+KFxmbv5ZNdDtsZl54BXWCkGqmYkTjoxP+20eaf3SBazqDHKFdPhtfIitpNmGPJd6BCq43UGcjoS3IJi4BM5cCgpTJVVinpBRzp72e\\\\\\\/4i5fGwFH86aUQzu9GvGhuGYs+lZiyBepzyNXZpuvfq2TzE4sgMgroh44Ia8FKrhsxoymNe8yrH+MieFFlSdcWt9lwi4eIYPSXpmGhTvwgQzMLbXEGenXGrxdn0RpaWi2azdY938kGHNgxQWk9MviKrP24o9+BlYG3j7ssv+Sy4CQ6vT\\\\\\\/sT8xknnFuqvZXZ0QVJqKeOXZPQ4QSvQPLsopenOP6ImIlV3QlMTOgDyUvbPJ97EQrtdMXRctSS9xx1nZCSD8bHHuIX0G3gOxbynVQtc8HZQpnu5HqrYaw9dJpWUR\\\\\\\/9iMyM+Fa73RYehUsB7ff\\\\\\\/ATwRqc2TGADaBMhMzXEdfCmLIbZZljpjjpLafBSgAARZLneHNfBdazWMXyM5W2Ts62IwA0a6XeCdAP8rlNO5YVLmbpd3tsTW7T8QEB91wIQ8oe1VrAu09+0geaqegVYourLo24yGvRJy+2KVI3Xs\\\\\\\/Bpxl0+nGJKLQgINNcD3KTcAIvV0rndYI9vvpDqDhM6bnbkBY+7rarRxCLzwkFSbk0zacY9Nqb4mgBi17\\\\\\\/hG76tYEIefVYx0P22paVBgsLBRYFeRxL0y99ol\\\\\\\/tT+80phsgo9j7mV9+E9MiYVyhZa0seicxRI2zdh3Fl+lGKgOnyrYjMcI4fqf1MQ60IjoBd9U6dkE9qElU+2I7CykeoH2rhyqdm84voX5JKhbZFvGuBWLYQfQElE51OwnXlRYOjpqLw\\\\\\\/t2HrRc9z4lP\\\\\\\/2t35G+hHLKgFNxo6a\\\\\\\/IO7I4jMyM9ywzEOTUzQH0J3ID77ba3R3WUCfDl1V9Xhfn39CeB7VHTUj9ByIB7AEzMwwMLjD8NTJtTaQvykKWfxeeuWkkqxO+FVSvO2GKliuFrL1f6V3MOcHyyZbbrfP97KMVm91px4Ea1j57wrnoJfrydhq9rWowMmL2syFaMpMIQ9ezhKWYNvqtsz93x\\\\\\\/kxFGT8ey9HU4+fdWi7\\\\\\\/qEHCEtQxZ7M060Vff\\\\\\\/Uu3qu8B1hnGkVcvrfrHggZBKOeYPgoySvHPpZlDQKeMXwMLoAf086NiTH6Hxg7ABf6nQoHUtIGNzCglJENLTs2P4AAxcxJFX3FCGat1f2YAZIAlR6WQCEG6j2zoOW20ffqKLN1tSctwBJS7JUyq+b8VFdbb6b9GNCqZICsNTDdsrn2yf08d03UHQZmAOXNf8BJk4pt7caKxH4QL4XnxUzDoklXlbiXeCdao9nAsLYjiq\\\\\\\/0jrzAefaU0BXVsCxQj111DqPFVVw+fcGTtZpSCFk6SINYP8qvomCXmY18KTXiELQYtWN7+fgwcZnSxo6ktu+55cd1t5MlsEFUNUdKxJNgztDBVML62ySHGN0\\\\\\\/akY2JnTWgU4FPPSVmVfqfmfnAuUsfH7Nmq\\\\\\\/Fn1lhuRLV4qGMh64Aj2j8\\\\\\\/0tlTMTtX6gvKPzXDPV0f0hAUnJ0bGrVkbnH+RWr+qON9NPJrw1iAlIf7+dydje8MgDT414mIuk\\\\\\\/fhhyEjBCnAf6Ys8MtS2zFv3KkqkQdzw\\\\\\\/kYet5vr9cINJ3jTEGrtIZesojwgJq7illBm\\\\\\\/hkts1KfKeh3NsoBwqdYBtHCjNzCs0ePJsnGCk8HER3SGQ8M89Zf05s5Y552Z8abxnNV6LSQX3u8aAeJjBNT+X0q\\\\\\\/8\\\\\\\/TwAZDfFSlCKhhK2Kdtfv+VerCYM3ICtj4Q4FjiYaLfdDUWDjU+15URcogGvTvyYi2SnAqs\\\\\\\/N3fkb2v0BXCz4GnKP9Ej+Zk8YKuvfERR+F1IRFLIVmaZoKK\\\\\\\/YmeZJFhliIp5FbC20N+HG\\\\\\\/0TnSm\\\\\\\/rkm0UPs6LD1muA4n+IAPJIUEgsM5Zb9CXjLlMCc\\\\\\\/peo6kpRfmsxJ5xXWDVxNJOe2qEMjQB547+joyl+BMvvCcU2Ix8Rcm7+m+4eEgNTP9fgRw5FiLLdmqsZqF+7JC712vaPRugQUdCqlr8IDEKttnah2AVwBH3KGYygGJ5JIqhonxQG74iH6p6\\\\\\\/841kL7je2K808Rq6AfIA5fgFDSjV2IvsRoCqSAPwfuf+C\\\\\\\/X3s0J8MUdz2FY9SYBxv42QCuqzgxjHura9b2N9GNXZkeE30fJxf5wH8t5Vccy8dzF+ONX6zBG9sii+arDr\\\\\\\/KBVBqFwAv3K0LPTroOEHIwLolYruyjGDINxN0FJK+YPvEEdr9KGsbPBz9n6vyl7MdHXjBgT0NTQBl51eQij6NPvXbJa5oJqzDhW8U9AQrrtgbN+r+mI2FNZB5LDGtcZJ9eQZuKS4CoplqDgGcprLAegkdOZPmoim5D5lBDNMIdG6ReCpU9BIqjxLBpGfjStJ6ADRmJ5IP7J4m3zWY\\\\\\\/1U79dg8kmTQufPIA721OVQ1lNTegfwU0EEUUL1OmjpAXnvS1qNXfNW\\\\\\\/wz8KcfSD6\\\\\\\/4MsVuaH50Coy2+S1Vpaw93WJkOE1lpJd5fFh6+1\\\\\\\/CywDuMPZv2rHdjG+t9xsnrwGfOqWlJTdM006QpsLOSzL1g1n+B5co7kaOiAGnu353TEvhDzPRR6M2XsouB4n9xkn0\\\\\\\/RUdrxggOm31cA4Yf3Deh2Pb8xCJt83abxKK2l+gPx4CPoplc03o+yR2tWaHdeAcW\\\\\\\/0Tpp3uWugVYKWupRJl7d9wH0ql\\\\\\\/6TLpUJ8rOb58ARis7MZ1maefEhtQ1rvAJaM3foTRcy0tGKr1so+LEpzBsvk+orZaRmZV2c23Xf4vPnQqAMmZ6dTjkvAUoti4Inv61vzN3dhpWWT8LWX3S8Mt\\\\\\\/6maJnBSp1GPK3k2PHSRsm89jCErxwWUV\\\\\\\/Fm10Fcd7pE0Rksh5dKKWvSpeWO8ePItIXNn5piHXCVtH602fpPHh4pv3k2P1Vl8ntx4SMWTCQbmt+EqThzFlzZEeZoW33Bwmth602ump5h8PKobkFlygiHIJ4QsuJrlhBl7jmxtxK0dkjmNmIBnp2ZRGOyolHXi51EbGZ7eteqoi4sHaCfiNspZzeQd\\\\\\\/PamkeNrre1\\\\\\\/sccVoLL5ebqAw5SRbfliVlFP8f1chRLLbk69pogzQV8XS3P9bwiKR7gRcUkgbs92wiOB1cN8ayA3T59FYyUPus9Pb89IjttiWsFDMozR0THMp5r1S3Eu5Y87Gli9eNnw6crD0EL2ZMa+bzZpjsAwdEVFHcFhrA92IIkvFvGrYKiusB428yhnfwutA1Dt7Jrx1UgHhLF2UNTNEPi5QJHJuprXjPX3aaZRhgf8+2YqU\\\\\\\/+L\\\\\\\/OG0q6INvd1MTIiGpNg9AxL2a68hJPpLBn\\\\\\\/AxA0mxiiiIWJ2dxgW39Q3QnDKbnQ0qf7XHSHU4E8Ms4oyM08E7QCXfxZ5ploqOn\\\\\\\/BFJQGuOGe6AFv9rrm042eiuHJ7k+isj2B3hNTvCDk+Q0hlN6yZ4ysh4SgfkzvR0UUcTtVm+EacTvhLOVhHiYMHx71TjqqCH1AMpx0bu2+Q8AAtT7Eo0PcLz7ncwIJ6v7tboi0y9v4TT7+Kq6lvLx8cJYy6eXRU5TmoBcFqnIyZTX6nziYf6UBdA8jSBwJXhPVAC2Z57B5doOEZwi36LKD+6nkFkqjFCwicvKYR9sUdR0agK7n2nFasjooNFlZKpEjhkzsSnj1Iisb\\\\\\\/SkQJYbtUQ3rL3r1UULUmSDrerCtSGwjLrDJg9Wp8RGqebrPBiRU+WvyiGnYY3JT89423\\\\\\\/97pRo4iNDlKZhtfxPcgPbuM1GcGF0lPkrDjQzujO2G\\\\\\\/G1OqfVuVAiGScfVeu0CQMHUrvzh4XKzQtM9Dy1HjEA5LoNF9wsKNlV0eegik2ByrBhKwv7Tx7meav8KAkFUET3osW\\\\\\\/N\\\\\\\/HDTbOpF1pm3C2NQDVRsAIrgax2t+CzLHpM2SDlfkZQce\\\\\\\/PJTA3S2Rwpu3Rv4\\\\\\\/QOAqIDKKxAJlhgZegECeDuKLTsM+qvq0RPgLJfw5skysF2+zo\\\\\\\/qxTDMpq3WnYktRSo4vcVMsKJdXN1aVdAmQzHT3UegaWEb+3zOmbbCGw0iKDV2\\\\\\\/M5pSqxqm9fqJcB\\\\\\\/d2Cqv9zjDu5FTPTSegqXISaUwj1BJkqIZpqO6ToTfT1mYuB\\\\\\\/nFHDaJiPcwEcUYZ0MBRjEZ3FbDtdMTml9c0KkWJG8rwUbjI8dqRepAMlhGYcJ8cJWANsH8mpu87Gd8cQCjAm3tg\\\\\\\/oBiSLRR6jp\\\\\\\/1j2gUxGzpITwIpoMy+ADWpY4EqpNiVWaG75hjVMR4JqonQwxHW8YUQ9kY27fWxuMfrajoFdFIf\\\\\\\/4hGEjNiUo0bV8YOytUDcMi1qEPy\\\\\\\/JsF84+JRsKPYyVE\\\\\\\/\\\\\\\/UPNOF6JBWHq8vmSrcnoMCpG4tZ4fc3mwLelFpF7hQI61CEZlMmP6bCZQiVdi3BYiVC6zcR6aPneL\\\\\\\/K14LjTw86RUQvVIyyz78kvwyYODUeWhm3OanMhCtwPoVBanFmWR203KUFch4pIm6BY2YdXKbXeAg+C1xfUJjMcudBWriwup8cPbt8RrE1mnL9YruCrtPUMIavXkR08iFWPOrdi9BsywqLw2fmkRA81K7lVMGR\\\\\\\/McFdMEmxhvEU35oZSGdoEC41ilY4iBRPy2bbBh1rOV14MZgFGzzmqcXPi8NE6eRqZF6Rf6npj\\\\\\\/MpGhk0KTK+HBPBbYN1BstWnQMWbUy3uuwk3gmOVmwWXj4AtEcGDurg43ZHODoZ1Pk6mrwscxcF8wI++4\\\\\\\/vf8uDYW4lfjjbPxfmktjKcLwgyRJRt4\\\\\\\/YlYDhrpIiPliivSAgThdMf+u+3Fm4PouoHadRONfLGly+8BUv\\\\\\\/pyKJdzQMKCmi511Ms3u68hd7c+kLM7frGL+FJm1yCbuSdcKlwH73Yz\\\\\\\/RJHyk58qv8Qcgg4D605JWSvdQutO8K93m7T9SzLvy0Ig26qcmV7f8c1P\\\\\\\/6S6wOfUG0Szs3nmmObemFsbJZOd3GUh6w4Clcxt7TRuXhC55vzWkvauUhET9lJbNZ63zU3qhv1Is+Wdb94p6QUyCRzZSsgCh3uk1uEYwW9LeliAfyHkVGq3jI+frY3nse1q0gxsDGQoFEZn4NEBTsiOzWAXQJ+\\\\\\\/nSZ6f\\\\\\\/vPLs8UUH2ZJV4qsj6bLvWdwQsgqP7fHRy6hDbyWWqKIlsg7HiSKyWBvKGAzgbuJowx9okZzjmAAfwTls0a7GC8XrP5ij+xCz+\\\\\\\/+N4LjJUltrpeQ+iuxmAh4JegLuuHpXixR7J95V1V0Eo84k6iNo1u7LqJu6HXTDdvVwrkKiOKMn4ABKvcX+8kYGnic+VKZ++sTvd3HD\\\\\\\/CFQDzzAe4e5x6yGXadrGCPjDz+veSRkWx+rU+5MCzm5q5GbEJjCjec4DY701y5\\\\\\\/2nl5k2L1w2QW9tTrFM2rc446XzCecHRHzOtBV2znixEvwfWrR21WXNk590UgXO3mDzjJZAkvQlSGlDxhJaMSEmB\\\\\\\/MhJkpFa1vgBLONd8hqzL2DgbVkk5c+PRS9pm50xAPflgeOtoxVa\\\\\\\/DpqH6THYykNd\\\\\\\/W+ZqauGw6uyqYbSaOAciyEhC1jyK+hlzzG6ROYZ6+7jrPPDDEiiUYpLHhF41v+WcoNXVeKDKFQ3zSI3cUdLgfjGDgUWTpzxLMCfVyBTRcyg1yJHDiz7Fkzw56Pqhyltan72aStSzyNmSl2MKLz731mtyKQzQEb6vSd5YBXR0sSgxJlCDDQhAVJAznp7BgBzyARKgU+Mqo826kBjO7aEWOpNJs1W9WDZ1OJnamXovy21ill7trI990n5d0\\\\\\\/2ilT2nqI7sfDrulHNAMm7z7vEjqa6YriSiqKzoLgaJdDUdJ0yx4VH319IL++fHL2z3xprUazBLE\\\\\\\/cFt00iXtuA9gMrm6DWkqFjIT+UeIHoM0oTOlqzBh0OwycdxmVmMyROm3AWBc3gNINrpzrBnKmhHrDpdb6u3tNQd44QrugcRT\\\\\\\/7e2LMohKE2xNVi7KKMlYPX\\\\\\\/7FZsHrdX+G7gBaDFPSQNcTLsn6zP3GS3k86y7lf+67j\\\\\\\/dIg3ad34zvBpNGvf73zRdq7Ye3Y6LlqXdmzGPzUcBeWAOn2EvZUhW2OBACb5yydk1ceVhFMnLY8z8i6Aazi06IorXYr5buzuec8h\\\\\\\/18Km1dlwCUA5tCN4jZuzS7jQe4rbGpeD+HQbG7359DfTmCfnY8E+zXkOoN31aquC\\\\\\\/6LayjrRJkZXugNU1+Ohzc3K9YpbURKNvAU4HSfUZyI6J58YU7SdbzhlZ67fTwKkSbi43H6bYCgHs8s0hmh05ydFka90Uu9xVCVl8egF9VlHqG+0HvbfbQOi+BNKdC+pWQRzB7dJBlDu2MwYufSQJHqSS\\\\\\\/sA9IsNzJi3fsRtT1voj1Sc4RhEEVATbktcy\\\\\\\/J7YgrgCjbqpWTFml8We4pM0AYh43C\\\\\\\/9k17G8n9rSAGwZlcJalAHrB0xmcus2mbmzs0nhpXnpHBHkBaL1+3+Ieujq4I73Rlp4tgQBEZhmDm6CmmBsrHeNQhSWXkMXC0uS6DdNvasyH0vCtBFVyJK0AH2cbeo11qRD+h8Jtw9MGFKGovGeCjE90CQVezNkXPL9UEM3psNvPPEcoW8Q6vIDn2NGTYc+XIxtG4GFOg3JL57JSL5J9bwt2Tup4e58GX5wCvy\\\\\\\/iDUgFp5fcYYrVU8NisNqvJb6SU8qBwFoYqhV7AcSinyyC0Mzkl2L5oCEDp7Ts3cZRbcElIj7KkqfuHLuBrMutWw+\\\\\\\/jaVeYDaz3xhXH\\\\\\\/k3abPDiwIgo+QWvX0EuiJWKollLLlUOMKd9G7HALCIBTirkqHhuYZebBSLx\\\\\\\/HGv4ZQ7+iuzE8IeHLZ88MNqOQFrVcLnQhJ4KWBnb47F1LIiTWE8vjfoCke16VzFkYBREnI4Zi2dbv8eEViKhOOr7M\\\\\\\/iKXyEY2tZNy0EswOyl6xaj+\\\\\\\/ry\\\\\\\/zY+zJ5nuWpPnbc\\\\\\\/Q6I11oMNinwV8DG+sFpf9JNlayBB7N53B\\\\\\\/SNgC\\\\\\\/k4\\\\\\\/zlH1zl2Kds296dGsv24g4UNsJS8GSpdsCUnNrnuzYO08Iuyb8DZB+vBkGz\\\\\\\/NkMUCQyybQVLyOqItEkBw\\\\\\\/YkF6Sy0ArzdR09zECk6q59kEB7NzhUYQ+zdtq7xPCkSDNS8MN7vePQevY0zdZ\\\\\\\/v8OvxLKOT+foZP272raJ4BPtQ3nuveOXbc1ec7dPaoUlqz7vIF6vhn+n5dAEIECGhAY4exeIAoYzGD5Djvg+OWdytRTcAJ0BZQubpmYGvmnMxfxV0XaMT6dYICqCnLyw55scRNdDHLogdKLkThMFso4fhubY7gTmISlCqphwFr3Ubl8Nicso7DHyRUdiUUWmOlNgIj2GhfpLd7\\\\\\\/BGQT4+MHSJQyOMl5Eqwe46Sea3NQJwR7cJmABLC8PqiBWM9b+NNr5CJIkrcBRM\\\\\\\/lIoxv8OPiyqrRjbm56Ng\\\\\\\/qNE0WK8WKOb6UzXJtW9kGq4OPQbEkHoMP4YmSruPKzAg5Z9ZDh\\\\\\\/cOKw2JDfEvR8fjdSRefSN8YpVd2R\\\\\\\/BEnMeUM7eFSKssWToFRFVyxJHwftPV8PQgQeaiSE3BvE+v\\\\\\\/99BNaNTyHNE306NSiAmBhUk2w9pk2P+8A+uEuZAOPTfn8iy6jhbNmsqhVZ40zCjMopvWCu3AxLuhDRusn7A9WsaDj7pFheeJ5ie2j6WxZ+poeLhT0jC12s7+7zPc9n1be0TqchQ0Ve+GZf3YKGYu0\\\\\\\/WGajImFuxA17rsw5i1XLo\\\\\\\/TdT9ewDF9ooj75IAnm5qP5HYkJlL6IT7\\\\\\\/ACHbeK2gcRPkbBN9qtvx\\\\\\\/hjxpuVmO1AzkxDRl2yZqNW6nSRqTUDPAOHJ4JEKtbIYR0I8jXKbOqRQG+s690jrpFRS\\\\\\\/2AApWYWjwkSKXKAeBnfPaqHihf1vkx5c2RCYyIOoivmcwZ\\\\\\\/BfudOBmrqdiSMeVNnau5hNM\\\\\\\/o9j0ZTRXnSQ0SPsm\\\\\\\/rAmdYu9bFr41lbjiaNCoujOTqBb8whi+7qsBBDM84YBXy4ZljQOOrHKc4Pq8odxBHMIJqcCXQkZDchqam3MwkN8beUMqqdp+JLzOlf0H+QmZq4LHn3s\\\\\\\/EcE4gDQJOP84SlXA7mk5g0278LT2OKELkQa7TXMfyGGnKJ7pVPMEBrlMxD8oNK7\\\\\\\/PzcXU\\\\\\\/3I6DYQxZewLHFaiFOY0E4gHM18JYYd5IHrCTTfNyT\\\\\\\/ZoduNdizwU8HpQ+Mev6msp6HJ+gBAARbkQcnL8vvs1w9RYIg26TLwyRsGwmrrt50i8LzazsU3ZMFSziCPi4NQMgNgYFO0YwqC+jcdZ1+Bi8zcGJtFMXLStw1ZZANnvSsQ\\\\\\\/Db9tNtat1Va5f0bf9tsQFNVyqt0WFnfr3JkEa9saOCqDeYhoZlYuMXSr64fSSsd9FISXBQ1IMPGMFlBwr+Xak5n3R1SoAFb+Tq9Ty8si+Zg6bNM27E+kxGYkRbdwXYvW9AlDCzO9dnK0RdY083fvo\\\\\\\/+HEiHwMXUZokMgkWWet4ma5iAV+O3LDv2Yzr9d2dgY8mRS0uiGGzo02KtKJiYpGmtgY7Poum+JZZq0v3X2dcDPJR2AZVqFKb7Ds5Y7GzfaJjBBgmloKtxlkddGpnzALccvSDcnEEpplMoZj7Eh\\\\\\\/tC9RUmwBNFn6V7GwsoBHOUFDtxQ9jdGncLLIuQY3JE+bvSVy2Yqdj\\\\\\\/YbdEO0GLrlj1x8cIq4obeZ7a\\\\\\\/DXBWv6l45wB7Gp6BLF2m7kXfHlVU3tuzueux\\\\\\\/QGOTg1w8hx60h5+FdhMT21OOjgeuyX+qIMBiCBlV0Hwef0D5GUlr5XL60eWA45yakI\\\\\\\/Mas6XsoEx1DR0p4joUB3Pydw\\\\\\\/jhMhEnkcqxIsBON0yA0257wjHf0LzRzhjy1DWWDqhc2QCKdC91jMXjJcUNvPReS\\\\\\\/Y\\\\\\\/90ONSEsEyhdP1fROTYdzwAywQTDucUWd39CiAS5tbYmaBeXTNAwxE\\\\\\\/oRN11I751wdJ19E1YnRXTpORLKMN6OTE8PhJ6KcpxECj9udTxCerMr5C999bJJbmjKwQARyx+Z3Wrx3pkCCG45KaX\\\\\\\/Ua55qCX9qCuhWAfTji2nZ8MzG5F3ytLDy9SC3FRWI7AaKzxji9ORO8EqLMzpb6i8yDU2gKhEgIdXc6HqTrhWK18tltNsFKplryQ2deMN+\\\\\\\/OXA06x3c\\\\\\\/LJD19k\\\\\\\/H4DiJVcr0ePWs\\\\\\\/5KybUnE\\\\\\\/QDoChsju4FJQ0Umjqn6isVw1wAsHsgnyygB2C0XsKGRjd0SjOXS9MQ\\\\\\\/4Wqm8JhRBod30j8hikr4puhYujb6q7xDvE0QaQXypZLkcndO0MKiLuuyQfJduuDoJXVArA5kj0ieiIGny1ZuxLsqc8iDK\\\\\\\/t2DWBAUajrpTU3Qu2bZkZTIwEZJqWYFoE7qWAdZtlk0xuj8OUUtxezW\\\\\\\/ovL8Hq\\\\\\\/dpEk7Qu9ZFSeKx\\\\\\\/VyxK6XlguT6flmkUI\\\\\\\/4CAvB0YBw7W6KYz8xR9hQbdFYSvlGBSXLLwS9D1mDMxPW\\\\\\\/BS4WhGX6OXtRE4XkOOHcvazg8pINxofIpFDk\\\\\\\/KGKLHENL68UeAbTxcyMMQlHq1YpUJJ+rhDANhwHvANeEGsk2lPplBjalB281K3FKU5Ui77l4mMBiqnvhcoO2FvfVMxVj3hIoiHYtE8gbpkpyAjfctNq\\\\\\\/rnt1pizH2d79MAaqZvQel3lYf8wnpVHJzps1bbXsZF+PB4rG73fWLuySoQyZg8Xpm3Y\\\\\\\/ZhVI9U36VcI1fhq1ua8y7RxY4C5TcwuBu4qfiriLD4A340mLX7OTqEHo5p8PZ7ybrlL7nXaUu1ahLrV7VOXcrgOteCrt\\\\\\\/cUQC9+4ncizjdfQHls4kw83A0HaMbsk4Wpkhkf28TyX+X882bWKetSDxCe0kwx618Rg\\\\\\\/4j1RKKcVzzdmLSLvz6dsEab\\\\\\\/F8xDwl6Up1+Y8UWFGS1qE7d3YOH7ZJQ6N129ilI4Eb8aBXvriO8Jmi2io3CAE3y3nhatP41AcSIFYOoFPJDhh9t9+F6kXFRM2TSjVMXt0k48H44ACQIXejQpEIT0npvmfnCvUd2jnqDcb826xRQWbXkLa+LnwKDWpwtBwHPX5nZMGAzZx6FMbrkYqgVtWlZGP9QwC8Kritqwy26kyJ2aPO+9iAh0kBaNjttI\\\\\\\/+OSHJJICqqDN8n\\\\\\\/QU8GIEjC2MEcqF8WMVvAPb6aIuku8oTdOZXAxry1E2eOMrtlY9iAIar3oslpew7K5jd6ya7bQ0YXpoCKyAvgkWtnWEQ9QBgn\\\\\\\/NNSvxB6gNYjHDprWGSaVGBz9tLi9NbeA+v9lgkHLl0EJWGvwzl8+bIb73S+1X0q625MoFupCyx+sHLFm7mqIX\\\\\\\/VXj0MO7o2yb24cfVJ19Fz0BCxG7CAospopBSV9zNxMYfKirQ4ogUsi4VITzC3uKRUd2x9XzDf1WHo1m28PYPWrDF\\\\\\\/Gr5yGwSQJY9pITjOm42Zv2Wgd8eYSENRs9OLhMk9SDsnVwuSqrD5jpif\\\\\\\/IxGCJMH\\\\\\\/4shChMIsIv8tJLtDQwR4QqjBnnvvkeWakvZaWwyYqCppeg486m5fSCsB7++dNm3PHjGQpAmRA4R2XVNpa\\\\\\\/6T\\\\\\\/5CPr6+GdRm0fGEgdznT\\\\\\\/hdy3f7uWVFIgCLXV078eUXZ1PFDTKGZdOk9MKFUD1XU+11jytqrVUAHXs0I5ePb83NmbE7X1kLnCQNvvZc56gW84RyWXm87osNFdW6uMJUFlDYhjr\\\\\\\/jsSVou7UpSlqow\\\\\\\/+vTCZurvNEN77Rr49k3Pk3R0LbHcq3TpvsOfDhT4Pxudt4XFg3G4rqpYvraR\\\\\\\/B2w6mJyuzXe4\\\\\\\/jPVuE1yYnjZaRqxLC0V08QlJ3erwxKIsMjeu0e08hTcX1spkh+ZFhLi3u6CJmz0xPjt\\\\\\\/B1f+M+56E\\\\\\\/eukEcqnQQIspnZ9gXGTsb3csRX6zWXn0k6BrBdKfAQ86TrEIyQ+6aJQf2cMVQeuMbkx8sO4fs0kIChCHU6H7BaIiofChCDlBDkH9G9YFZuR+KtMz0qJpXn+zibK\\\\\\\/u12gUamV6Sbu\\\\\\\/\\\\\\\/6GBOOJnESXL0UjM+Op+mj9IlTuAyS2LXzI\\\\\\\/szHCDvmS\\\\\\\/1zW\\\\\\\/x2GAJO9RsoDQZrlViiB+Q9mV7YByEBEHNWf3IUDCdm2UeUiB9gV3Y6Jk1whQsoe2QjR4DUnOSwKq8fcnMC9e6ynYTvb2UaDgrIpmr+qgxt7Lj8YS9RTIh5L7QFfXUKd14Rx3XMc6OWdevCYfXmo5fuC3i19T1RhmnBsEqsDkTWg5Bxwpps0K6FMBYZlKLybI4ibGbqy8sw2CwBK3LIDwggXmSZloFGarZTEkbupT8BJWazpXC6ajU8lsn+eeiJ2rUsNQenvDKrGsScp8zrn+J5CjYuFGQQ55H6HcUy0jYgkougULiq+s0VpJymqIYq\\\\\\\/ElwF0j8X4UqA58tBZmsg1n\\\\\\\/bLt\\\\\\\/+72kKoXAOthHbPBxOr3VdA5UQkfwhcx7BTktJoEL8\\\\\\\/ARWksPzFcKdeJpSNQ0ypK9QQ0K+OY34qBIIbkr00I+pe+58dbV+Iop037kyflN\\\\\\\/6TMsBvTxt5dK1XRBv\\\\\\\/xvL85vJflkX32i5u8qSJzq3LsRjrJWVvDwqzuGTngOSkpnLZ+fYxrYtP0\\\\\\\/L15G5h\\\\\\\/kx2gWCWhreSKNv4O6o+cGMCVYwsxNpTAEKzIc4gaZayEEyVSlL7EqnNyQwDPyxwd9F+zYplXgRzEXtaiXJPa9+OUZAlKD7u8vQAe7YI5CKBerBOkF6cQdPyrBx2ufrh3GJ2ernUbrzMl7r2edb4K3OghRqJZn2CJiJJ1+7kbwEK2\\\\\\\/u7RQGYaVnd743e+Qa9W2D9lfSbljg2WHyscwwNgKS52TpM51NQekKIV89TQeg5XODfJKEt7EiU\\\\\\\/ag+gyA5FshG5fZnnykr7wdfbNe+T32YD98zHqdRxYOu4B8+u5BF66K0R7aAeBNLixst2XhsT6AHHht6HuTZoelEQKXUlpcSj2m0Fi7vkJPs\\\\\\\/UDyf6e7tWz5KxeNYAreexURSTlji3kMCv1vKp11j2AY10XW1Ch2fMBJ8U8MSM8XXU8bPmpPl7H50KLyEk\\\\\\\/ex5JTRYS\\\\\\\/l0osH1WkjK1bDDy2TAUwgJVG+4K3PgZ\\\\\\\/aITX6mjt3fJ5fMk9\\\\\\\/XjM4qBtwWeIsTRzFDtHJOhcDjfft\\\\\\\/XfJirg8putGdRiNinxkCuvjofrQtHdh2bMm3Z5AnrnScqrtNdcFdOE08VHi\\\\\\\/xnsYZnOhWVQVNqSftuY0g\\\\\\\/8Pf3AjKstcYv9j3RH8cPHY3+nqrbsZEAUASrdHa3ii8c5QztZTfLkZsoCWQc8OKoKWxhFfZgLPcG4cWDCRQdQ85JbdmQYT1UG\\\\\\\/awaRRQAM6Jyi6kvmYisIHjaGJQWFAQtwCjbN28T++KtQRbncBW+zdRcxFRWyCHh\\\\\\\/aFgpv+N\\\\\\\/b6FeVRqCYiZZFkz5jc4qjAA5Tx9y4g1DMt1pMHnx5OIv4\\\\\\\/RpfuUtIFgMdtsgb7FWfbPvdBlc\\\\\\\/Al7O1xcJr1o+ER5\\\\\\\/91b1nPjkaHtv3i5cIKP3SYDW3KmARs98tkCUXmsDkN8DO6N+EnNekZLsmYro2NIHWvei4P\\\\\\\/Fkaek+0VzyDTQgpxj3WpO9PZKbowx+TdJgN3dJKgJANMoWBXg3s8qj0UPhJZWgjDXvofr81a7a9yhf0P99kMb2y8zmE+QwHH+qGEs++RorcJrNuW51GFRivZ7UhC+2pCXDuzn+sphdGPjSMXBJFgW8CugSniKKKGw6YSBtZmEkcKxfi5rG7oMGMq5fwg4HB+Q\\\\\\\/7a9EH1fuUKyNCVmWeKoc2YQAKAksj+Ybi5Q21kj7uiD\\\\\\\/1XRbzvvSQLNSP5xwRxYPSoIge\\\\\\\/jqDc000EH5bOlNWzD3QH0e3lQJWogqrdtyb954JOfts7RqG6ljhKGYW9n23MDLHQE4z23UBn7gnQnuf7YgnnSK6rRj15LTmctUCSswRDCE0UDfedkl0M2gjYjEykDFqAp1uiGCw9HlrLOxq7mYxpct4cFsKtbJM13s+Lnf28hLut7zibfGEBoIVp8+YhcBv+IcDq1BUE\\\\\\\/upopjZx1Qqn5SoAz29QdZsLGurbDpWqwC0AgAHBYp7U5yEW1FrjEMOsC3cY4mi8efpKeXSgOzoWcHn4NKTFLkjNoXviddYCDlVQmBq2K1sVIRpqYkJGv0qS0p2FiaAAgd\\\\\\\/n+LM0+2aa5UxbFS9djazu4kFQQmRk9MrA3Vsm06xIPKPcIFTarfaNgzQsOx7lQFWOuIK9t1UFOmULxR7SeTQBiaUqmruM3worTcFSqUnV06IbWWLfN7\\\\\\\/qkPA2CkXCmr\\\\\\\/4tHKtfuwHf25osPq9UV6fp9WJkCfKO6yk8cltKILZ97qWf00VmcJtBNpCX\\\\\\\/o1ffz\\\\\\\/ougBQZap9iQ2cgkDrQGbmwcxx8SzvjKDVoQKhF0HReoOPXsHEYE+Kdm4nKUPANNb8mre1KT5eY9fQlv\\\\\\\/VS2anzzucrFiFS6QzYzBRA3Lw9cld2OPJLI9N4okkWsGT8N\\\\\\\/WSi61aEC5QAs4EeHtT4+XaBv0Uleti\\\\\\\/HMyvKxdUBo0sybG6DCOr51ecorIBPfJC4Y1IObVy63nLjrxrakw+WoM19TxPR51sQZkdrWbYs+8gXq9T8ZC3KW9u2b5IPQ+EMQ4PvUdd6bwNQ2rnxQdxzjrBSWCa2L40rzjuHtBdfnAOWBrtNvxSiouETnCCmF3kDgqWATb9LZpz9rc+VKTnEgSlFOtaFqLsbLkpWTrdjm3k4WL05gHpTQ9S7rpH8p6LqYGobNH0gufYHoQlgwNzmYKeRHSxxd0lPhIYXaHHhzie\\\\\\\/N6TRjNsHNFqk9Daym35iivmEIQyPoZb9LcnhhwrkSOpf2GxW0OpxWOv0u791gHNV5tXN6Wgb2YONXWp6RDzpmhjkb1uz4+sLR5Cnta+M0LHH8iqfEyRbLDH5\\\\\\\/sMV5g1lSK\\\\\\\/bL4od7qHcXW5wweWkW2yNSi6GW8ExJ99Va9JRwY7KWKzVqCOu\\\\\\\/yH6ODEX7\\\\\\\/vaYQGmMULIJBGajeX8Ik2IDUJrb3iev3YbX\\\\\\\/LPA35uM2\\\\\\\/dfcW8Pfv4WEbHFAtYkwhMaA9k8eJ7l3BNZNJOkJ+f7fzxuMNPRKbKtpVQfGgHSLAdHYyXHPL1ddpQbUv8ujHCUO6jhzhOUEFQn+mPHu\\\\\\\/opSA1VSE\\\\\\\/\\\\\\\/LTXicK8H2SAjF4qI8ZVL8LCCtY459XPsw\\\\\\\/AE+F47tKl73kQ1LVGxvvTlVm0QThMOMza5LwC68o\\\\\\\/AavB8ge3G99uB3h8kYVTQFWsnol0I1l7XriTA23868q+QhGy0Wfm0VEzFrRbUcQncSvhtLtYOrj3zqpsc2GW1Safwde\\\\\\\/1V5qil6dAwIqzBpHTHBNUUjaOTPkB640KKcXWGQ\\\\\\\/yt2ZLEPpwx3SKJjPUmK3yMRwKBws5ZjAqXvmYDJeytklzDVrKQkN\\\\\\\/\\\\\\\/2LAoningltDRjxNZYyQfXgRKViDGcpIa4gZphRt9hmY1EWWT6Cs0\\\\\\\/MhIPvphjr7DxDLMNsJF8EWvc0gP2PAynuxGpnCP4XyCOQMYH\\\\\\\/Whv4lLvoCKXXjptBCI+lB9+wfyxWlm1mr5mhgiAB6ikz9rHMWp\\\\\\\/ZqweOJDo2d72ykQTV7EZ9wkPZEmZxBTygS+6E52wyPtFH+pstCiWavYGBGD9Tw\\\\\\\/dgwY+55rF\\\\\\\/KPRAkrdtTSAA2e5wYS99plW1juwJiXGRwXhpuIrVuqsUOMCP4zXYhtY2oqSQlpINkw5x7LvSYko\\\\\\\/SwZqNWx+F34TRwHX897wB8gHGBRMoPLQI+TTjTI3t\\\\\\\/66wqFC9yQCPocFzXW+cCYGpWdkexe73kEJW2lWhl2YIXbG4BCHPbbc9WtNnaGiHaP5t59a97H9cd8DaLs8gkj+SOR3pQ1U3YvVzwv3kW6pWnepQm6wU+Qor50ghflDtA\\\\\\\/tVQ+qezb5wdEUWLGYVq4ew1cNZIAaMyRCM0GZ4u1Jq91R5BlJI3rY9b9ZA8SfPsJ0ovluqkSMv2wfXTLAntbc9gazO+p3vkSyWLMeCSqekrnRmf+kDMvddm9hVNBNbzqjzKiqPR7leJ1rGnu1FAUrz7aiGtDMvXowBIk0RbXwOX\\\\\\\/C+RnH0VAwcilYtLzuLoHYSd1JoytAX54MPHfuQgNCzWarltKjzU124TIgjX3WEaClTGoEjNms7grF+i6VIMe6qlrJV++2jcMNQSNWSQgS2S2qTve5IIYJ2rjTsaape8UZGbyfWCaD6n+JzQH36wnMD8s1fUj9dn4yCUdd+5ciSFd+SyFTxqZGlDSCVQsJGj\\\\\\\/2iwi5xP7QrLKml39CtSolbT+r7U\\\\\\\/x1\\\\\\\/lV6GJnzDXAefURMh+rr8KS7Ho6eaj77\\\\\\\/inU2Gq5S+hvhdjeOGqcugXKrmBOCtMcGGSm9ARI3vhKsTUNrbliO7V6RECoqa+1MeG3kCXSDCASX5Y+lh1u2N6DJgJy03AdjhUk+cc64yrwswvkumFaFD1dVZ2vOgSvj66\\\\\\\/rH5mA8E7cO6GkY2bdIVMFVzuNpUJTjiIh0x3DC3qlWRDrUSpz2qDLLvVAgMfoC0q2L\\\\\\\/XoupmFGq1yHZt+5UrB1vFSHJUv4UMGe+\\\\\\\/sqLnNauAYHLB7s5a0dPHlvIn9bYNvSsQvczRtufmqGHzyDbs7HVLLnmbYTKTBrLSdjk9FapZbZNomVSsC0Aedvv+Hvat7LPzb6HYwXEFcgnvUR40FrPH2Q6hW3eQ6MJd+lWYtke4BZFxHshcwF+Gzr4\\\\\\\/HbANOfVaJr8o2r5jRJGE2a\\\\\\\/iAxhT4X6IOguGupWRnZleChqlWmS3yBKB9C4EkkjiQJR0xZPZv8zHPBe1eG6\\\\\\\/47H3aJSVM1QNsbdBF4xfdliX7C6gf\\\\\\\/udtYM4Zla4O9qTl+B09Z9\\\\\\\/w3UzmHittkV4xCGw+oCZMl+uy3CvfbXKzHMFvZVXrSVl2YxJm8Vi7NvBV07ljDHSnAFUczc3AYQa8gCuE543Qn\\\\\\\/i6Xm6tqyGrtMCtvJDxe3\\\\\\\/JgODJBEKas2LpRTS1NC3WYY8rP3YKNrW5UY3s0eVTtyDUBACTktB9fqOzK8s0cuBzZNkWmizYEUtSlVuy9hr4oVZYa+IH7X3F4ONRRwMTWrrET9e7vh4WYO6bhp+YRonE7ZsXl1xlvQh4nL\\\\\\\/3n\\\\\\\/k9+p\\\\\\\/XoblwvafHpx41nGn0f5WFWoPWcJXdCdVifnEP7oGs4t+CU+z2ARStg4JIk5Dvz6Xi4cTr+2YlBqzm486eDvhjdZbpm\\\\\\\/aZIABS6q3yZObdhWC23v+olEgjS4aI\\\\\\\/wtFQdNrX8uOGOBH3v8zULqHjodGPAe2TgLWdg1FvrsZt8ZuNcopVVEYMRmJ6oioq+YoPtotYqu6PJa9wuh9EPQJ86tqwNlQ9W9g\\\\\\\/\\\\\\\/j4rUqGcfFz2MRBWzawbZwm+KB7I3VrXXEmIg1UYTiAYOuXp+CwpEvhUsohXIYthv2bzTtv7DHmhnanDwoSKpBOMWBGLPLIgq8fjBiUhI7sMw4W8s547tM6DakfRHp\\\\\\\/cFNbkhCm2qJ896pGJLtKvaO72jOIo91zFAUDtGuB63E4eRqL2RSF6Jh4WSZPosKOpecHkXxh211kXZT9pTQDtn8yC1HzJ4VJk8rV6ycek5CFJintMgnRnzH7kR2k9TohV0V+3x7s2d32nNPk2kcMMG5J0eUt+VNSHqvmJ0IRxV4o7tkdhoLg1DHXur3B2q0EYb5R1BxNDFvJx1MRWXTTgji0Pu7lEcRWr+Wd51h\\\\\\\/VyI6usD52I5DEvwW8c6\\\\\\\/mDnEISd0aZtV9zeEpX+NqzTew==\\\u0022,\\\u0022iv\\\u0022:\\\u0022XVt4OpFOv6hIdyEUsUdW6g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022043cd11ce36ee4d4f162af2925ca206af24a4b39309039a7adabb9c5a7376b67\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00223c3VJreZY7KfQzWt\\\\\\\/UPvqnV31IQBzoktJZmBnrjZV9km23AmJnic\\\\\\\/d5i834qQ3JfYciKaChrWlcvv9suYab1PXdg7RbTAMDL57rCcalJHrHRl7TF6NGMYviqAYaF0FoOZnuY70AJgDiPgaL64arwgL2hv0Fqv5ba5VZ3\\\\\\\/F3CwDrBFpHmk5nNBqV+W1CbSKuvnSWMXGIlWsg0iH0iUmfoq+lHMmX3Z0yNJVqdE7EfRXDuU7PgqYSZ\\\\\\\/wPEOPvTkr81rUWDR7URQdKtY3xOVRWRsEcIsa+MUJK4JxqmnsOuVKetMLAKLr591I8fTFECtgHsZeFIQIgsiqafRo\\\\\\\/h3OwFGtINUn\\\\\\\/AU0JXOPpEjCVxDyGiNr32JjsecjoUakO1rYjUml4GFkm3MrkRU4rEnH8aP3g5hxjQw8CfwIbJidjQ+XqAenkoo9TKDZ3fPl9vt\\\\\\\/R+GGwvTd6t2p6b0jORUSinfp6sz0kLUl+g7T6wsvPihPmVKY5UySXzcNr66q40bF5xav0UwebPdOs+kSItdtRcuMbWEkYof+tgQyiVD+kShMGjDrA0C4RGflXl5cKR1jxY2XbugZjne+BWnBOk43VlLJgCZ7DxSh0BQkGHWYb9Qhk+2AXgf8hSLQKUWtXkcsu8HODPJOZTRBuW3wHVvkrRBqIx5Bjc5AXzSawyIfU1HQZdo5PsQkzRPezPt9rL7nBu0anb4stBubFwHhwNAW0tXJpOYNlsEA3jJ4OW3xqDn\\\\\\\/RtQH1qGklXPTzj75Ttj8uIGHjN+QgoYnn9ms8ZrACacqPX2triuflkMzZ6k1zi8m34NbPFRjBnWACQMi+eNZNMSBqIUGRF4gW2JdPWKPyqbjFgSgXuzQiqpfz+izHFkHQN6\\\\\\\/UKP1svM9bZSxRb8HNhvkEbkw23ZyfSZrId1X9CZO7+cKkMqVQ3qLchPYyKrYKNFWS9CttaXbJGbqZepH7U3pCjz546dZiO8Uc9sKoOPNN81x4181OqrDlgs3qq0fV2TQqTqxegfj3HDc9jSuisx8tDvPPuRr51OKLqNSX8TPu4i8flzi4cpPrQh\\\\\\\/fpMWN6X\\\\\\\/baKTS34aHoPUfaLUaOPLUvObYd+PvFtbR5XtG9u1XtXZopeN8FQW9\\\\\\\/vqlTdEjkRQU3yFFh09249GMBpoZs0ecIHTT9EMEqTiEq93o5o7rSjGiFHTdQrQNHoHUfHipJUkWkNk1Rvy7E58YnN0d2LeI9LaQRXRCgx4h02RjDgDyyNefx8680vyJ09Llx2BmtLHfg43V4TNlM7Ll1eZW3UCthjA\\\\\\\/GQsjg4OE0CBT4XDVx291k4SMLpCL8Rv+YMSBgufss6GQoLgnLFSUCvaoX4d\\\\\\\/X6Mhc2mxAsvJpcv2JzORp15dwctybb231OSchEhzzfdZZ6VRLvw6eo7X7y\\\\\\\/OuXnnYGFK7m3m3FaUnELOZ5DDNlTN24aVAFmcG4SrxDNW9AbnN2DGLeF2HOH9cS39QqApVBjFJo723OB47\\\\\\\/\\\\\\\/BDEaWFNoXuDIaDWsKilFHp38h42MG2uS6lOBxW9hDmD47svQ1RTX5IkqV+zsggn4EYW2f1c5xXOS27ZjdMuzWjsmyZppm07pElZRB1u\\\\\\\/ssLZ1Rk6VLaCEdxLYzzhbZHnbPengmrT+1PlDfJzybZOm\\\\\\\/yHNNr\\\\\\\/\\\\\\\/QnGxyxwP9+fBpb3yoHet38jylqgwo+Wp+F4ZPQIdi202DsTZj3WZqHf+8PhkgaJSZre\\\\\\\/3AdvEgbNTMni0I6fWimK8jiuWwZwKA5MScRNmzg2Du0oEjWigBs5idcruYQsZ2JA8Rr5\\\\\\\/xn2zfdq6joa4GhhjrQAhrOpWIwbF9zbV4DdbdCW5fytb+XSqvecvzSjmGqCN7hgy9yGom2nyc8LdFTKadiXEaBa2kWoqPckRRoHdU+ZT\\\\\\\/6+0ykK2iGnUuJGEq5XtzIGP5aAp8iU5fMrtph+iPGx0WhFhJDf\\\\\\\/5aeZYSt\\\\\\\/L\\\\\\\/AvNt7k7M5DXa0yDkJblWWSlFr5myw7lEajppNN7W75MCe7dKlTcAc\\\\\\\/sQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022YA7sjZd3bAJqwGHP8gXt1Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002242f9085aaf302bfa21f523f94cc557fa41db2f590d812d1f2234f8ed08452d82\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459827 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-mutant-screen-report">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/5/1007" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-QRTiA-RTGNybas7rUPGMTbdUXcKIQYA3nWt3S9Nr25E" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/5/1007" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-go0vrVedvRTxWahHmmIRRV79G76AzHUyuLqMwQgoNtg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/5/997" title="Leveraging DNA Damage Response Signaling to Identify Yeast Genes Controlling Genome Stability" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/6/1021" title="Meeting Report on Experimental Approaches to Evolution and Ecology Using Yeast and Other Model Systems" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459827" id="node459827" data-pisa="ggg;5/5/1007" data-pisa-master="ggg;g3.115.017038" data-apath="/ggg/5/5/1007.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Rapid and Efficient Identification of <em>Caenorhabditis elegans</em> Legacy Mutations Using Hawaiian SNP-Based Mapping and Whole-Genome Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Aimee Jaramillo-Lambert</span>, <span class="highwire-citation-author" data-delta="1">Abigail S. Fuchsman</span>, <span class="highwire-citation-author" data-delta="2">Amy S. Fabritius</span>, <span class="highwire-citation-author" data-delta="3">Harold E. Smith</span> and <span class="highwire-citation-author" data-delta="4">Andy Golden</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1007-1019;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.017038 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459827" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Aimee Jaramillo-Lambert </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>National Institute of Diabetes and Digestive and Kidney Diseases, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aimee%2BJaramillo-Lambert%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jaramillo-Lambert%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAimee%2BJaramillo-Lambert%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Abigail S. Fuchsman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>National Institute of Diabetes and Digestive and Kidney Diseases, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abigail%2BS.%2BFuchsman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fuchsman%20AS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAbigail%2BS.%2BFuchsman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Amy S. Fabritius </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>National Institute of Diabetes and Digestive and Kidney Diseases, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amy%2BS.%2BFabritius%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fabritius%20AS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAmy%2BS.%2BFabritius%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Harold E. Smith </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>National Institute of Diabetes and Digestive and Kidney Diseases, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Harold%2BE.%2BSmith%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Smith%20HE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHarold%2BE.%2BSmith%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Andy Golden </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>National Institute of Diabetes and Digestive and Kidney Diseases, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andy%2BGolden%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Golden%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndy%2BGolden%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:andyg@mail.nih.gov" class="" data-icon-position="" data-hide-link-title="0">andyg@mail.nih.gov</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/5/1007" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459827" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459827/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/5/1007.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459827" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459827/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/5/1007.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459827" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459827/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/5/1007.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459827" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459827/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/5/1007.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459827" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459827/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">The production of viable embryos requires the coordination of many cellular processes, including protein synthesis, cytoskeletal reorganization, establishment of polarity, cell migration, cell division, and in <em>Caenorhabditis elegans</em>, eggshell formation. Defects in any of these processes can lead to embryonic lethality. We examined six temperature-sensitive mutants as well as one nonconditional mutant that were previously identified in genetic screens as either embryonic lethal (maternal-effect or zygotic lethal) or eggshell defective. The responsible molecular lesion for each had never been determined. After confirmation of temperature sensitivity and lethality, we performed whole-genome sequencing using a single-nucleotide polymorphism mapping strategy to pinpoint the molecular lesions. Gene candidates were confirmed by RNA interference phenocopy and/or complementation tests and one mutant was further validated by CRISPR (Clustered Regularly Interspaced Short Palidromic Repeats)/Cas9 gene editing. This approach identified new alleles of several genes that had only been previously studied by RNA interference depletion. Our identification of temperature-sensitive alleles for all of these essential genes provides an extremely useful tool for further investigation for the <em>C. elegans</em> community, such as the ability to address mutant phenotypes at various developmental stages and the ability to carry out suppressor/enhancer screens to identify other genes that function in a specific cellular process.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/whole-genome-sequencing" class="hw-term hw-article-keyword hw-article-keyword-whole-genome-sequencing" rel="nofollow">whole-genome sequencing</a></li><li class="kwd"><a href="/keyword/hawaiian-snp-mapping" class="hw-term hw-article-keyword hw-article-keyword-hawaiian-snp-mapping" rel="nofollow">Hawaiian SNP mapping</a></li><li class="kwd"><a href="/keyword/crisprcas9" class="hw-term hw-article-keyword hw-article-keyword-crispr-cas9" rel="nofollow">CRISPR/Cas9</a></li><li class="kwd"><a href="/keyword/essential-genes" class="hw-term hw-article-keyword hw-article-keyword-essential-genes" rel="nofollow">essential genes</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/cdc-25" class="hw-term hw-article-keyword hw-article-keyword-cdc-25" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">cdc-25</em></a></li></ul><p id="p-4">Early studies using the model organism <em>Caenorhabditis elegans</em> sought to elucidate complex biological processes by modeling genetic studies performed in bacteriophage, <em>E. coli</em>, yeast, and <em>Drosophila</em> (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Edgar <em>et al.</em> 1964</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Edgar and Lielausis 1964</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Wright 1970</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Wright and Shaw 1970</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Hartwell <em>et al.</em> 1974</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Hereford and Hartwell 1974</a>). The reasoning behind this approach was to use mutagenesis to accumulate developmental intermediates that could be classified and ordered on the basis of their phenotypes. <em>C. elegans</em> is a particularly useful model for studying developmental processes of metazoans because of its small size, amenable genetics, anatomical simplicity, invariant cell lineage, and transparent body and embryos. The early studies using <em>C. elegans</em> applied this strategy to analyze nervous system development, behavior, and embryonic/zygotic development (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Brenner 1973</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hirsh and Vanderslice 1976</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Miwa <em>et al.</em> 1980</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Schierenberg <em>et al.</em> 1980</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Wood <em>et al.</em> 1980</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Cassada <em>et al.</em> 1981</a>). With the goal of reaching genetic saturation to provide a clear picture of a given developmental pathway, these studies generated about 600 embryonic (emb)/zygotic (zyg) lethal mutants and 54 maternal-effect lethal (mel) mutants. Through linkage analysis and complementation tests, this large data set of mutants was distilled down to approximately 60 <em>emb/zyg</em> genes and 32 <em>mel</em> genes (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Hirsh and Vanderslice 1976</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Miwa <em>et al.</em> 1980</a>; <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Schierenberg <em>et al.</em> 1980</a>; <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Wood <em>et al.</em> 1980</a>; <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Cassada <em>et al.</em> 1981</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Kemphues <em>et al.</em> 1988</a>). Since these early studies, more screens, both conditional and nonconditional, have been conducted, some in hopes of saturating the genome for essential genes (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Rose and Baillie 1980</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Meneely and Herman 1981</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Sigurdson <em>et al.</em> 1984</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Johnsen and Baillie 1991</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Johnsen <em>et al.</em> 2000</a>) and thus the number of genes that can be mutated to reveal essential embryonic phenotypes is now much greater. Mutants identified in these early screens are referred to as legacy mutants.</p><p id="p-5">Several of these early screens focused on isolating temperature-sensitive mutations of developmental processes (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Hirsh and Vanderslice 1976</a>; <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Miwa <em>et al.</em> 1980</a>; <a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Schierenberg <em>et al.</em> 1980</a>; <a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Wood <em>et al.</em> 1980</a>; <a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Cassada <em>et al.</em> 1981</a>; <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Kemphues <em>et al.</em> 1988</a>). In these original screens, temperature-sensitive mutants made it possible to recover and maintain potentially lethal mutations at the permissive temperature and, as a tool, to understand the temporal control of gene function. Another advantage of temperature-sensitive mutants is that temperature-shift experiments could be performed at various times during development revealing new functions for a particular gene during different developmental stages. This is a big advantage over current methods of gene depletion such as RNA interference (RNAi), which usually only uncovers the earliest phenotypes. Temperature-sensitive mutants also are particularly useful in genetic suppressor screens. Suppressor screens allow for an unbiased approach in the identification of second-site extragenic mutations that ameliorate the phenotype of the original mutation. The use of temperature-sensitive alleles has the advantage of being able to perform mutagenesis on homozygous lines of mutations that are otherwise lethal.</p><p id="p-6">The molecular identities of several of the legacy mutants have been determined through traditional cloning methods. However, these methods often are difficult and time-consuming, leaving the identities of hundreds of the legacy mutant alleles unknown. Recent advances in next-generation, whole-genome sequencing (WGS) has made it a rapid and cost-effective technique for the identification of molecular lesions causing a given phenotype (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Minevich <em>et al.</em> 2012</a>). <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Doitsidou <em>et al.</em> (2010)</a> developed a one step protocol that combines WGS and single-nucleotide polymorphism (SNP) mapping that further eases mutation identification in <em>C. elegans</em>. This method backcrosses the mutant of interest (normally generated in the <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> Bristol isolate) to the polymorphic Hawaiian isolate of <em>C. elegans</em> generating recombinants that are sequenced in a single pool. The genomic region linked to the mutation of interest is identified as the region with a decrease in Hawaiian SNPs. This region is then analyzed for molecular lesions.</p><p id="p-7">With the intention of identifying additional reagents that could help expand our knowledge of different aspects of embryonic development including meiotic chromosome segregation and eggshell formation, we performed a literature search for legacy mutants that were previously identified in genetic screens and were preliminarily characterized as temperature sensitive. Of the more than 500 temperature sensitive mutants isolated in early screens, we focused on mutants that were characterized as embryonic lethal or reported to have eggshell defects (osmotic sensitivity, irregular shape, and unshelled or thinly shelled embryos). We chose 12 mutants from publications at least 25 yr old in which the molecular lesion producing the phenotype had not been determined. Of those 12 legacy mutants, only six fit the criteria of being highly penetrant temperature-sensitive mutants. We chose an additional mutant, <em><a href="http://www.wormbase.org/db/get?name=WBGene00003188;class=Gene">mel-2</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00088001;class=Variation">it20</a>)</em>, that was not temperature-sensitive but fit the criteria of being a legacy mutant and maternal-effect embryonic lethal. These seven were subjected to WGS/SNP mapping. The molecular lesion causing the phenotype of each mutant was confirmed by RNAi phenocopy and/or by complementation tests. Our analysis revealed that all seven of these legacy alleles bore mutations in genes that had only been studied previously in nonconditional mutants or by RNAi (<em><a href="http://www.wormbase.org/db/get?name=WBGene00022455;class=Gene">tyms-1</a></em>, <em><a href="http://www.wormbase.org/db/get?name=WBGene00011532;class=Gene">chaf-1</a></em>, <em><a href="http://www.wormbase.org/db/get?name=WBGene00017132;class=Gene">tofu-6</a></em>, <em><a href="http://www.wormbase.org/db/get?name=WBGene00009342;class=Gene">fasn-1</a></em>, <em><a href="http://www.wormbase.org/db/get?name=WBGene00009507;class=Gene">mus-101</a></em>, and <em><a href="http://www.wormbase.org/db/get?name=WBGene00000387;class=Gene">cdc-25.2</a></em>). Two of the temperature-sensitive mutations [<em><a href="http://www.wormbase.org/db/get?name=WBGene00006989;class=Gene">zyg-2</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00000351;class=Variation">b10ts</a>)</em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00001264;class=Gene">emb-10</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00088187;class=Variation">k12ts</a></em>)], thought to define distinct genes, were shown to be allelic; both had mutations in the <em><a href="http://www.wormbase.org/db/get?name=WBGene00009507;class=Gene">mus-101</a></em> gene. The identification of <em><a href="http://www.wormbase.org/db/get?name=WBGene00001283;class=Gene">emb-29</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00145383;class=Variation">g52ts</a>)</em> as an allele of <em><a href="http://www.wormbase.org/db/get?name=WBGene00000387;class=Gene">cdc-25.2</a></em> was further confirmed by CRISPR/Cas9 gene editing where we rescued the mutation and recreated the temperature-sensitive phenotype by editing the single mutation in a wild-type background. We have demonstrated the ease and utility of one-step WGS/SNP mapping to identify legacy mutations. The strategies presented here could be applied to any temperature-sensitive mutant that remains to be identified molecularly.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3><em>C. elegans</em> strains and culture</h3><p id="p-8">The following strains were used: <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> (Bristol), <a href="http://www.wormbase.org/db/get?name=GG36;class=Strain">GG36</a>: <em><a href="http://www.wormbase.org/db/get?name=WBGene00001260;class=Gene">emb-6</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00145372;class=Variation">g36</a>)</em> I, <a href="http://www.wormbase.org/db/get?name=MJ65;class=Strain">MJ65</a>: <em><a href="http://www.wormbase.org/db/get?name=WBGene00001260;class=Gene">emb-6</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00087771;class=Variation">hc65</a>)</em> I, <a href="http://www.wormbase.org/db/get?name=GG43;class=Strain">GG43</a>: <em><a href="http://www.wormbase.org/db/get?name=WBGene00001268;class=Gene">emb-14</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00145371;class=Variation">g43</a>)</em> I, <a href="http://www.wormbase.org/db/get?name=GG20;class=Strain">GG20</a>: <em><a href="http://www.wormbase.org/db/get?name=WBGene00001271;class=Gene">emb-17</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00145360;class=Variation">g20</a>)</em> I, <a href="http://www.wormbase.org/db/get?name=MJ63;class=Strain">MJ63</a>: <em><a href="http://www.wormbase.org/db/get?name=WBGene00001264;class=Gene">emb-10</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00088187;class=Variation">k12</a>)</em> I, DH10: <em><a href="http://www.wormbase.org/db/get?name=WBGene00006989;class=Gene">zyg-2</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00000351;class=Variation">b10</a>)</em> I, <a href="http://www.wormbase.org/db/get?name=GG52;class=Strain">GG52</a>: <em><a href="http://www.wormbase.org/db/get?name=WBGene00001283;class=Gene">emb-29</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00145383;class=Variation">g52</a>)</em> V, <a href="http://www.wormbase.org/db/get?name=YHS25;class=Strain">YHS25</a>: <em><a href="http://www.wormbase.org/db/get?name=WBGene00000387;class=Gene">cdc-25.2</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00091883;class=Variation">ok597</a>∆)</em> (V)<em>/<a href="http://www.wormbase.org/db/get?name=nT1;class=Rearrangement">nT1</a>[<a href="http://www.wormbase.org/db/get?name=WBTransgene00001903;class=Transgene">qIs51</a>]</em> (IV;V), <a href="http://www.wormbase.org/db/get?name=VC2225;class=Strain">VC2225</a>: <em><a href="http://www.wormbase.org/db/get?name=WBGene00003792;class=Gene">npp-6</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00093918;class=Variation">ok2821</a>∆)</em> (I)<em>/<a href="http://www.wormbase.org/db/get?name=hT2;class=Rearrangement">hT2</a>[<a href="http://www.wormbase.org/db/get?name=WBGene00000254;class=Gene">bli-4</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00143617;class=Variation">e937</a>) let-?(<a href="http://www.wormbase.org/db/get?name=WBVar00241187;class=Variation">q782</a>) <a href="http://www.wormbase.org/db/get?name=WBTransgene00001901;class=Transgene">qIs48</a>(Pmyo-2</em>::<em>gfp</em>; <em>Ppes-10</em>::<em>gfp</em>; <em>Pges-1</em>::<em>gfp)]</em> (I; III), <a href="http://www.wormbase.org/db/get?name=CV87;class=Strain">CV87</a>: <em><a href="http://www.wormbase.org/db/get?name=WBGene00019247;class=Gene">syp-4</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00251558;class=Variation">tm2713</a>∆)</em> (I)<em>/<a href="http://www.wormbase.org/db/get?name=hT2;class=Rearrangement">hT2</a>[<a href="http://www.wormbase.org/db/get?name=WBGene00000254;class=Gene">bli-4</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00143617;class=Variation">e937</a>) let-?(<a href="http://www.wormbase.org/db/get?name=WBVar00241187;class=Variation">q782</a>) <a href="http://www.wormbase.org/db/get?name=WBTransgene00001901;class=Transgene">qIs48</a>]</em> (I;III), <a href="http://www.wormbase.org/db/get?name=VC3196;class=Strain">VC3196</a>: <em><a href="http://www.wormbase.org/db/get?name=WBGene00008990;class=Gene">smgl-1</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00093572;class=Variation">ok2423</a>∆)</em> (I)<em>/<a href="http://www.wormbase.org/db/get?name=hT2;class=Rearrangement">hT2</a>[<a href="http://www.wormbase.org/db/get?name=WBGene00000254;class=Gene">bli-4</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00143617;class=Variation">e937</a>) let-?(<a href="http://www.wormbase.org/db/get?name=WBVar00241187;class=Variation">q782</a>) <a href="http://www.wormbase.org/db/get?name=WBTransgene00001901;class=Transgene">qIs48</a>]</em> (I;III), <a href="http://www.wormbase.org/db/get?name=VC1878;class=Strain">VC1878</a>: <em><a href="http://www.wormbase.org/db/get?name=WBGene00003060;class=Gene">lpd-3</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00093316;class=Variation">ok2138</a>∆)</em> I, <a href="http://www.wormbase.org/db/get?name=VC3150;class=Strain">VC3150</a>: <em><a href="http://www.wormbase.org/db/get?name=WBGene00009052;class=Gene">ekl-1</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00296657;class=Variation">ok1197</a>∆)</em> (I)<em>/<a href="http://www.wormbase.org/db/get?name=hT2;class=Rearrangement">hT2</a>[<a href="http://www.wormbase.org/db/get?name=WBGene00000254;class=Gene">bli-4</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00143617;class=Variation">e937</a>) let-?(<a href="http://www.wormbase.org/db/get?name=WBVar00241187;class=Variation">q782</a>) <a href="http://www.wormbase.org/db/get?name=WBTransgene00001901;class=Transgene">qIs48</a>]</em> (I;III), <a href="http://www.wormbase.org/db/get?name=MT20434;class=Strain">MT20434</a>: <em><a href="http://www.wormbase.org/db/get?name=WBGene00011532;class=Gene">chaf-1</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00603944;class=Variation">n5453</a>∆)</em> (I)<em>/<a href="http://www.wormbase.org/db/get?name=hT2;class=Rearrangement">hT2</a>[<a href="http://www.wormbase.org/db/get?name=WBGene00000254;class=Gene">bli-4</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00143617;class=Variation">e937</a>) let-?(<a href="http://www.wormbase.org/db/get?name=WBVar00241187;class=Variation">q782</a>) <a href="http://www.wormbase.org/db/get?name=WBTransgene00001901;class=Transgene">qIs48</a>]</em> (I;III), <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a>: wild-type Hawaiian isolate, <a href="http://www.wormbase.org/db/get?name=KK359;class=Strain">KK359</a>: <em><a href="http://www.wormbase.org/db/get?name=WBGene00003188;class=Gene">mel-2</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00088001;class=Variation">it20</a>) <a href="http://www.wormbase.org/db/get?name=WBGene00006744;class=Gene">unc-4</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00142975;class=Variation">e120</a>)/<a href="http://www.wormbase.org/db/get?name=mnC1;class=Rearrangement">mnC1</a> [<a href="http://www.wormbase.org/db/get?name=WBGene00001072;class=Gene">dpy-10</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00142982;class=Variation">e128</a>) <a href="http://www.wormbase.org/db/get?name=WBGene00006787;class=Gene">unc-52</a>(444)]</em> II, <a href="http://www.wormbase.org/db/get?name=QA137;class=Strain">QA137</a>: <em><a href="http://www.wormbase.org/db/get?name=WBGene00017132;class=Gene">tofu-6</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00275444;class=Variation">yt2</a>)</em> II; <em><a href="http://www.wormbase.org/db/get?name=WBTransgene00017528;class=Transgene">ytEx100</a></em>, AG247: <em><a href="http://www.wormbase.org/db/get?name=WBGene00022455;class=Gene">tyms-1</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00251316;class=Variation">tm2429</a>∆)</em> (I)<em>/<a href="http://www.wormbase.org/db/get?name=hT2;class=Rearrangement">hT2</a> [<a href="http://www.wormbase.org/db/get?name=WBGene00000254;class=Gene">bli-4</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00143617;class=Variation">e937</a>) let-?(<a href="http://www.wormbase.org/db/get?name=WBVar00241187;class=Variation">q782</a>) <a href="http://www.wormbase.org/db/get?name=WBTransgene00001901;class=Transgene">qIs48</a>]</em> (I;III), AG248: <em><a href="http://www.wormbase.org/db/get?name=WBGene00009507;class=Gene">mus-101</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00250726;class=Variation">tm1761</a>∆)</em> (I)<em>/<a href="http://www.wormbase.org/db/get?name=hT2;class=Rearrangement">hT2</a> [<a href="http://www.wormbase.org/db/get?name=WBGene00000254;class=Gene">bli-4</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00143617;class=Variation">e937</a>) let-?(<a href="http://www.wormbase.org/db/get?name=WBVar00241187;class=Variation">q782</a>) <a href="http://www.wormbase.org/db/get?name=WBTransgene00001901;class=Transgene">qIs48</a>]</em> (I;III). All tm deletion alleles were obtained from the National Bioresource Project as heterozygotes and were balanced with <a href="http://www.wormbase.org/db/get?name=hT2;class=Rearrangement">hT2</a>. The presence of the deletion allele was monitored by polymerase chain reaction. All legacy, ok deletion, and tm deletion alleles are available from the <em>Caenorhabditis</em> Genetics Center.</p></div><div id="sec-3" class="subsection"><h3>Quantification of embryonic viability</h3><p id="p-9">Single L4 hermaphrodites of the indicated genotype were placed on individual plates at 15°, 20°, or 24°. Adult worms were transferred to new plates daily until the sperm was exhausted. Percent hatching is calculated as the number of hatched larvae divided by the total number of eggs laid. For all quantifications, the number of broods scored is indicated in the figure legends.</p></div><div id="sec-4" class="subsection"><h3>RNAi phenocopy and complementation tests</h3><div id="sec-5" class="subsection"><h4>RNAi phenocopy:</h4><p id="p-10">Gene candidates were obtained from either the Ahringer RNAi feeding library or the OpenBiosystems RNAi feeding library (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Timmons <em>et al.</em> 2001</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Kamath and Ahringer 2003</a>). Bacteria expressing double-stranded RNA were seeded onto MYOB plates containing 25 μg/mL carbenicillin and 1 mM IPTG and allowed to grow at 22° for 48 hr. L4 hermaphrodites were fed bacteria expressing double-stranded RNA specific to a <em>C. elegans</em> gene (as indicated in text and figures) and scored for the number of dead embryos <em>vs.</em> live progeny. Bacteria expressing <em><a href="http://www.wormbase.org/db/get?name=WBGene00004875;class=Gene">smd-1</a></em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00000405;class=Gene">cdk-1</a></em> were used as negative and positive controls respectively.</p></div><div id="sec-6" class="subsection"><h4>Complementation tests:</h4><p id="p-11">Homozygous (or balanced heterozygotes) L4 hermaphrodites of mutant X were mated to homozygous L4 males of mutant Y at 15°. L4 cross progeny were picked to individual plates, incubated at 24°, and examined for embryonic viability as described previously.</p><p id="p-12"><em><a href="http://www.wormbase.org/db/get?name=WBGene00003188;class=Gene">mel-2</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00088001;class=Variation">it20</a>)</em> rescue: <em><a href="http://www.wormbase.org/db/get?name=WBGene00003188;class=Gene">mel-2</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00088001;class=Variation">it20</a>) <a href="http://www.wormbase.org/db/get?name=WBGene00006744;class=Gene">unc-4</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00142975;class=Variation">e120</a>)/<a href="http://www.wormbase.org/db/get?name=mnC1;class=Rearrangement">mnC1</a></em> hermaphrodites were mated to <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> males. <em><a href="http://www.wormbase.org/db/get?name=WBGene00003188;class=Gene">mel-2</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00088001;class=Variation">it20</a>) <a href="http://www.wormbase.org/db/get?name=WBGene00006744;class=Gene">unc-4</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00142975;class=Variation">e120</a>)</em>/++ heterozygous males were then mated to <em><a href="http://www.wormbase.org/db/get?name=WBGene00017132;class=Gene">tofu-6</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00275444;class=Variation">yt2</a>)</em> hermaphrodites carrying the <em><a href="http://www.wormbase.org/db/get?name=WBTransgene00017528;class=Transgene">ytEx100</a></em> [marked with dominant <em><a href="http://www.wormbase.org/db/get?name=WBGene00004397;class=Gene">rol-6</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00248869;class=Variation">su1006</a>)</em>] rescuing transgene. F1 rollers were picked and their RolUnc progeny scored for hatched progeny [<em><a href="http://www.wormbase.org/db/get?name=WBGene00003188;class=Gene">mel-2</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00088001;class=Variation">it20</a>) <a href="http://www.wormbase.org/db/get?name=WBGene00006744;class=Gene">unc-4</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00142975;class=Variation">e120</a>)</em>; <em><a href="http://www.wormbase.org/db/get?name=WBTransgene00017528;class=Transgene">ytEx100</a></em>].</p></div></div><div id="sec-7" class="subsection"><h3>Whole-Genome Sequencing</h3><p id="p-13">Identification of the molecular lesion responsible for each mutant was determined using a SNP-based mapping strategy combined with WGS (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Doitsidou <em>et al.</em> 2010</a>). Hermaphrodites from mutant strains were crossed with Hawaiian males (strain <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a>). Approximately 200 F2 progeny were singled to fresh 3.5-cm plates and shifted to 24° for 16 hr to reveal the mutant phenotype. The plates were returned to 15° and allowed to produce progeny for one to two generations. The animals from 20 to 50 mutant plates were then pooled and genomic DNA isolated and sheared by sonication. Libraries were prepared using TruSeq reagents and sequenced with a HiSequation 2500 (Illumina, San Diego, CA). Single-read 50-bp sequencing yielded a minimum of 22-fold genome coverage for each library. Variants were identified using a pipeline of BFAST for alignment (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Homer <em>et al.</em> 2009</a>), SAMtools for variant calling (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Li <em>et al.</em> 2009</a>), and ANNOVAR for annotation (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Wang <em>et al.</em> 2010</a>) with <em>C. elegans</em> reference genome version WS220 (<a href="http://www.wormbase.org">www.wormbase.org</a>). Hawaiian SNP density was plotted against chromosome position using R (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">R Development Core Team 2014</a>), and the mapping interval delimited by the absence of Hawaiian SNPs. Mutations within the mapping interval were filtered to remove variants derived from the Hawaiian and pre-mutagenesis strain backgrounds. Candidate genes were defined by homozygous (&gt;85% variant call), nonsynonymous mutations.</p></div><div id="sec-8" class="subsection"><h3>CRISPR/Cas9-mediated single-nucleotide rescue of <em>emb-29(g52ts)</em></h3><p id="p-14">Rescue of the cytosine 716 to thymine mutation (codon 239) in <em><a href="http://www.wormbase.org/db/get?name=WBGene00000387;class=Gene">cdc-25.2</a></em> in <em><a href="http://www.wormbase.org/db/get?name=WBGene00001283;class=Gene">emb-29</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00145383;class=Variation">g52ts</a>)</em> was performed by editing the single nucleotide 716 from T to a wild-type C using the CRISPR/Cas9 technology. An additional silent nucleotide change was made to remove the PAM Cas9 target site to prevent possible recutting of the rescue oligo or the target site (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Kim <em>et al.</em> 2014</a>). This additional change created an <em>Eae</em>I recognition site (underlined to follow), which was used to distinguish the rescued line from <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a>. <em><a href="http://www.wormbase.org/db/get?name=WBGene00000387;class=Gene">cdc-25.2</a></em> was targeted for Cas9 cleavage using the guide sequence (ATGTCTCTCAATGTTTCGG) in pDD162 (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Dickinson <em>et al.</em> 2013</a>), targeting the non-coding strand. <em><a href="http://www.wormbase.org/db/get?name=WBGene00001283;class=Gene">emb-29</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00145383;class=Variation">g52ts</a>)</em> young adults were injected with plasmids containing a dominant <em><a href="http://www.wormbase.org/db/get?name=WBGene00004397;class=Gene">rol-6</a></em> (pRF4, a marker for successful injection), the guide RNA (50 ng/μL), and a rescue oligonucleotide (ATGGGTAGCGGCAATCAATGAGAATATACTTGTCATCAAATTCTTTTTCA<strong>G</strong>AGAGTCGGAAAAAGATGTCTCTCAATGTTT<span class="underline">CGGC</span><strong><span class="underline">C</span></strong><span class="underline">G</span>TGATTCTTCTGAAGCCTTTCGAGGATTCCTTTGCAACAGTTTTGAGGTGG, IDT Coralville, IA; 30 ng/μL). The rescue oligonucleotide contained the nucleotide changes of interest (in bold) and an additional 50 nucleotides of perfect homology on both sides flanking nucleotide 716 and the PAM site (120 ng/μL) (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Paix <em>et al.</em> 2014</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Zhao <em>et al.</em> 2014</a>).</p><p id="p-15">Eight <em><a href="http://www.wormbase.org/db/get?name=WBGene00001283;class=Gene">emb-29</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00145383;class=Variation">g52ts</a>)</em> young adults were injected, placed on a single MYOB plate with <a href="http://www.wormbase.org/db/get?name=OP50;class=Strain">OP50</a> bacteria, and kept at 15° for recovery. After 24 hr, worms were transferred two per fresh plate and allowed to continue laying embryos. All four plates segregated roller F1 progeny. F1 rollers were singled from each of the four plates and their progeny (F2s) were shifted to 24° after hatching. In addition, 100 F2 L1 non-roller worms from each of the four plates containing the injected worms were shifted to 24°. Plates were checked for live F3 progeny. Worms from plates with live progeny were sequenced to confirm the two-nucleotide changes (Macrogen USA, Rockville, MD). These lines were given the allele designation <em><a href="http://www.wormbase.org/db/get?name=WBVar02143996;class=Variation">av38</a></em>.</p></div><div id="sec-9" class="subsection"><h3>CRISPR/Cas9-mediated single-nucleotide recreation of <em>emb-29(g52ts)</em></h3><p id="p-16"><a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> young adult worms were injected as stated previously with the same target plasmid (50 ng/μL) in addition to a repair oligonucleotide to mutate <em><a href="http://www.wormbase.org/db/get?name=WBGene00000387;class=Gene">cdc-25.2</a></em> nucleotide 716 from a C (wild-type) to a T (ATGGGTAGCGGCAATCAATGAGAATATACTTGTCATCAAATTCTTTTTCA<strong>A</strong>AGAGTCGGAAAAAGATGTCTCTCAATGTTT<span class="underline">CGGC</span><strong><span class="underline">C</span></strong><span class="underline">G</span>TGATTCTTCTGAAGCCTTTCGAGGATTCCTTTGCAACAGTTTTGAGGTGG IDT, Coralville, IA; 20 ng/μl) and to mutate the PAM aforementioned site (nucleotide changes in bold, <em>Eae</em>I site underlined). We coinjected with a <em><a href="http://www.wormbase.org/db/get?name=WBGene00001072;class=Gene">dpy-10</a></em> target sequence in pDD162 (25 ng/μL) and a rescue oligonucleotide (20 ng/μL) for co-conversion screening purposes (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Arribere <em>et al.</em> 2014</a>). Seven <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> worms were successfully injected and kept at 15°. Thirty-five F1 rollers were singled and allowed to lay embryos. Of the F1 rollers, eight contained the C681→G silent mutation in the PAM site by restriction digest (confirmed by sequencing), but only one of those eight had the C716→T edit when sequenced. We believe the greater frequency of the silent mutation may be attributable to the Cas9 cut site being only one nucleotide away from the silent mutation, whereas the C716→T site was 34 bases away. To obtain multiple lines, two more <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> young adults were injected with the same constructs as described previously and 34 roller F1s were singled at 15°. Twelve F2 L4 larvae from each of the 34 mothers were singled and shifted up to 24°. After laying embryos, F2s were returned to the permissive temperature (15°) for recovery. After 24 hr, plates were examined for hatching embryos at 24°. In two of the 34 lines ~25% of the animals had a temperature-sensitive embryonic lethal phenotype. Homozygous non-<em><a href="http://www.wormbase.org/db/get?name=WBGene00001072;class=Gene">dpy-10</a></em> embryonic lethal lines were recovered from these lines. Sequencing confirmed the C716→T change and both the lines also contained the C681→G silent mutation. These lines were given the allele designation <em><a href="http://www.wormbase.org/db/get?name=WBVar02143997;class=Variation">av40</a></em>.</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>The selection and characterization of <em>C. elegans</em> mutants for WGS</h3><p id="p-17">Twelve mutants that had been identified previously in genetic screens but whose molecular identity remained unknown were chosen for analysis and WGS (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Each mutant was tested for temperature sensitivity and for embryonic lethality by counting the number of dead embryos <em>vs.</em> live progeny at both the permissive (15°) and restrictive (24°) temperatures (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). One mutant, <em><a href="http://www.wormbase.org/db/get?name=WBGene00003188;class=Gene">mel-2</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00088001;class=Variation">it20</a>)</em>, was not temperature sensitive and was maintained by picking balanced heterozygous <em><a href="http://www.wormbase.org/db/get?name=WBGene00003188;class=Gene">mel-2</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00088001;class=Variation">it20</a>) <a href="http://www.wormbase.org/db/get?name=WBGene00006744;class=Gene">unc-4</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00142975;class=Variation">e120</a>)/<a href="http://www.wormbase.org/db/get?name=mnC1;class=Rearrangement">mnC1</a></em> (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>). These animals were scored for embryonic lethality by picking <a href="http://www.wormbase.org/db/get?name=WBPhenotype%3A0000643;class=Phenotype">Unc</a> homozygotes at 20° and counting the number of dead embryos <em>vs.</em> live progeny produced. For six mutants, embryonic lethality was temperature-sensitive and strongly penetrant at 24° (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1A</a>). Although all mutants were preliminary characterized as embryonic lethal in the original studies, some mutants showed weak penetrance of the embryonic lethal phenotype at the restrictive temperature and others did not appear lethal at any temperature tested (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1A</a>). The lack of an embryonic lethal phenotype could be due to several reasons such as loss of the original mutations during the 30-plus years of maintaining the strain.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465981/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465981/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">List of mutants analyzed in this work</span><div class="sb-div caption-clear"></div></div></div><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/5/1007/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Average percent progeny hatching from mutants evaluated from Table 1. (A) Average percent embryonic viability from N2 and temperature-sensitive mutants. Single L4 hermaphrodites of each mutant strain were placed on individual plates at 15&#xB0; or 24&#xB0;. Adult hermaphrodites were transferred daily until the sperm was exhausted. Percent embryonic viability is the number of hatched larvae divided by the total number of embryos laid. At least 19 hermaphrodites were scored for the viability of their progeny for each mutant at 15&#xB0;. At least 17 hermaphrodites were scored for each mutant at 24&#xB0;. Error bars represent standard deviation of the average percentage of three individual replicate experiments. (B) Average percent embryonic viability from N2 and mel-2(it20). mel-2(it20) animals were marked with unc-4(e120) and single L4 Unc hermaphrodites were picked from balanced animals of the genotype mel-2(it20) unc-4(e120)/mnC1 and placed on individual plates at 20&#xB0;. Adult hermaphrodites were transferred daily until the sperm was exhausted. Percent embryonic viability was determined as stated above. At least 20 hermaphrodites were scored for the viability of their progeny for each strain. mel-2(it20) unc-4(e120) produced zero live progeny from a total of 4791 embryos (N) scored. Error bars represent SD of the average percentage of three individual replicate experiments." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1185382378" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Average percent progeny hatching from mutants evaluated from Table 1. (A) Average percent embryonic viability from N2 and temperature-sensitive mutants. Single L4 hermaphrodites of each mutant strain were placed on individual plates at 15&#xB0; or 24&#xB0;. Adult hermaphrodites were transferred daily until the sperm was exhausted. Percent embryonic viability is the number of hatched larvae divided by the total number of embryos laid. At least 19 hermaphrodites were scored for the viability of their progeny for each mutant at 15&#xB0;. At least 17 hermaphrodites were scored for each mutant at 24&#xB0;. Error bars represent standard deviation of the average percentage of three individual replicate experiments. (B) Average percent embryonic viability from N2 and &lt;em&gt;mel-2(it20)&lt;/em&gt;. &lt;em&gt;mel-2(it20)&lt;/em&gt; animals were marked with &lt;em&gt;unc-4(e120)&lt;/em&gt; and single L4 Unc hermaphrodites were picked from balanced animals of the genotype &lt;em&gt;mel-2(it20) unc-4(e120)/mnC1&lt;/em&gt; and placed on individual plates at 20&#xB0;. Adult hermaphrodites were transferred daily until the sperm was exhausted. Percent embryonic viability was determined as stated above. At least 20 hermaphrodites were scored for the viability of their progeny for each strain. &lt;em&gt;mel-2(it20) unc-4(e120)&lt;/em&gt; produced zero live progeny from a total of 4791 embryos (N) scored. Error bars represent SD of the average percentage of three individual replicate experiments.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/5/1007/F1.medium.gif" width="319" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/5/1007/F1.medium.gif" width="319" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/5/1007/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/5/1007/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466027" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-19" class="first-child">Average percent progeny hatching from mutants evaluated from <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>. (A) Average percent embryonic viability from N2 and temperature-sensitive mutants. Single L4 hermaphrodites of each mutant strain were placed on individual plates at 15° or 24°. Adult hermaphrodites were transferred daily until the sperm was exhausted. Percent embryonic viability is the number of hatched larvae divided by the total number of embryos laid. At least 19 hermaphrodites were scored for the viability of their progeny for each mutant at 15°. At least 17 hermaphrodites were scored for each mutant at 24°. Error bars represent standard deviation of the average percentage of three individual replicate experiments. (B) Average percent embryonic viability from N2 and <em>mel-2(it20)</em>. <em>mel-2(it20)</em> animals were marked with <em>unc-4(e120)</em> and single L4 Unc hermaphrodites were picked from balanced animals of the genotype <em>mel-2(it20) unc-4(e120)/mnC1</em> and placed on individual plates at 20°. Adult hermaphrodites were transferred daily until the sperm was exhausted. Percent embryonic viability was determined as stated above. At least 20 hermaphrodites were scored for the viability of their progeny for each strain. <em>mel-2(it20) unc-4(e120)</em> produced zero live progeny from a total of 4791 embryos (N) scored. Error bars represent SD of the average percentage of three individual replicate experiments.</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">We chose the six mutants with strong temperature-sensitive embryonic lethality, as well as <em><a href="http://www.wormbase.org/db/get?name=WBGene00003188;class=Gene">mel-2</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00088001;class=Variation">it20</a>)</em>, for WGS using the single step SNP mapping strategy of <a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Doitsidou <em>et al.</em> (2010)</a> (WGS/SNP mapping) (<a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">Doitsidou <em>et al.</em> 2010</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Wang <em>et al.</em> 2014</a>). All seven mutants were recessive. After a single outcross into the Hawaiian strain, <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a>, homozygous F2 embryonic-lethal mutants were reisolated and allowed to grow another one to two generations at the permissive temperature (15°). Pooled populations of F3/F4 animals were harvested for WGS and SNP analysis (see the section <em>Materials and Methods</em>). A loss or reduction of Hawaiian SNPs defined the mapping interval containing the mutant allele. A representative graph plotting the Hawaiian SNPs along the chromosome from a population of F2 recombinants is shown in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>. This strategy produced an average mapping interval of 3.0 Mb (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). The mapping interval was then analyzed for missense, nonsense, or splice-site variants in protein coding sequence. An average of five nonsynonymous, candidate mutations were identified per strain with a minimum of two candidates identified for three of the mutants and a maximum of ten candidates identified for one of the mutants (see Tables <a id="xref-fig-2-3" class="xref-fig" href="#F2">2</a>−8). To validate the identity of the molecular lesion responsible for the mutant phenotype, we tested candidates for RNAi phenocopy of the embryonic lethal phenotype or for a failure to complement a deletion allele of the candidate gene. The results of these validation tests are in the following sections.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/5/1007/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Hawaiian SNP mapping of mutation intervals. (A) Representative plot. Shown is chromosome V from emb-29, with the gap in Hawaiian SNPs (red) defining the mapping interval. (B) Summary of Hawaiian SNP mapping results. SNP, single-nucleotide polymorphism." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1185382378" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Hawaiian SNP mapping of mutation intervals. (A) Representative plot. Shown is chromosome V from &lt;em&gt;emb-29&lt;/em&gt;, with the gap in Hawaiian SNPs (red) defining the mapping interval. (B) Summary of Hawaiian SNP mapping results. SNP, single-nucleotide polymorphism.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/5/1007/F2.medium.gif" width="356" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/5/1007/F2.medium.gif" width="356" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/5/1007/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/5/1007/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466031" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-21" class="first-child">Hawaiian SNP mapping of mutation intervals. (A) Representative plot. Shown is chromosome V from <em><a href="http://www.wormbase.org/db/get?name=WBGene00001283;class=Gene">emb-29</a></em>, with the gap in Hawaiian SNPs (red) defining the mapping interval. (B) Summary of Hawaiian SNP mapping results. SNP, single-nucleotide polymorphism.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>emb-6(g36ts) is an allele of tyms-1</h3><p id="p-22"><em><a href="http://www.wormbase.org/db/get?name=WBGene00001260;class=Gene">emb-6</a></em> had been mapped previously to chromosome I at -0.47 cM. The WGS and SNP-based (WGS/SNP) method of mutant mapping also mapped the molecular lesion to chromosome I and to a 2.8-Mb interval. Analysis of that interval produced eight nonsynonymous sequence variants (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). Of the eight candidates, three were reported to have embryonic lethal phenotypes by RNAi (<em><a href="http://www.wormbase.org/db/get?name=WBGene00022455;class=Gene">tyms-1</a></em>, <em><a href="http://www.wormbase.org/db/get?name=WBGene00003792;class=Gene">npp-6</a></em>, and <em><a href="http://www.wormbase.org/db/get?name=WBGene00008990;class=Gene">smgl-1</a></em>) or by analysis of available deletion alleles [<em><a href="http://www.wormbase.org/db/get?name=WBGene00003792;class=Gene">npp-6</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00093918;class=Variation">ok2821</a>∆)</em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00008990;class=Gene">smgl-1</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00093572;class=Variation">ok2423</a>∆)</em>]. To validate the molecular lesion responsible for <em><a href="http://www.wormbase.org/db/get?name=WBGene00001260;class=Gene">emb-6</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00145372;class=Variation">g36ts</a>)</em>, we first performed phenocopy experiments by RNAi knock-down of <em><a href="http://www.wormbase.org/db/get?name=WBGene00022455;class=Gene">tyms-1</a></em>, <em><a href="http://www.wormbase.org/db/get?name=WBGene00003792;class=Gene">npp-6</a></em>, and <em><a href="http://www.wormbase.org/db/get?name=WBGene00008990;class=Gene">smgl-1</a></em>. Depletion of <em><a href="http://www.wormbase.org/db/get?name=WBGene00022455;class=Gene">tyms-1</a></em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00003792;class=Gene">npp-6</a></em> resulted in embryonic lethality with a penetrance of 89% and 99%, respectively (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). RNAi knock-down of <em><a href="http://www.wormbase.org/db/get?name=WBGene00008990;class=Gene">smgl-1</a></em> did not result in embryonic lethality (98% embryonic viability). Because RNAi knock-down of both <em><a href="http://www.wormbase.org/db/get?name=WBGene00022455;class=Gene">tyms-1</a></em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00003792;class=Gene">npp-6</a></em> phenocopied the embryonic lethality of the <em><a href="http://www.wormbase.org/db/get?name=WBGene00001260;class=Gene">emb-6</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00145372;class=Variation">g36ts</a>)</em> mutant, further validation was required.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466096/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466096/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title"><em>emb-6(g36ts)</em> is an allele of <em>tyms-1</em></span><div class="sb-div caption-clear"></div></div></div><p id="p-24">To determine which of these two mutations is responsible for the <em><a href="http://www.wormbase.org/db/get?name=WBGene00001260;class=Gene">emb-6</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00145372;class=Variation">g36ts</a>)</em> embryonic-lethal phenotype, complementation tests were performed. Males of <em><a href="http://www.wormbase.org/db/get?name=WBGene00001260;class=Gene">emb-6</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00145372;class=Variation">g36ts</a>)</em>; <em><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00144038;class=Variation">e1489</a>)</em> were mated to either <em><a href="http://www.wormbase.org/db/get?name=WBGene00003792;class=Gene">npp-6</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00093918;class=Variation">ok2821</a>∆)/<a href="http://www.wormbase.org/db/get?name=hT2;class=Rearrangement">hT2</a></em> or <em><a href="http://www.wormbase.org/db/get?name=WBGene00022455;class=Gene">tyms-1</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00251316;class=Variation">tm2429</a>∆)/<a href="http://www.wormbase.org/db/get?name=hT2;class=Rearrangement">hT2</a></em> hermaphrodites at 15° (<a href="http://www.wormbase.org/db/get?name=hT2;class=Rearrangement">hT2</a> is a translocation balancer with a pharyngeal GFP insertion). Single non-green F1 cross progeny were picked to individual plates, incubated at 24° and scored for the number of dead embryos <em>vs.</em> viable progeny. <em><a href="http://www.wormbase.org/db/get?name=WBGene00001260;class=Gene">emb-6</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00145372;class=Variation">g36ts</a>)</em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00003792;class=Gene">npp-6</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00093918;class=Variation">ok2821</a>∆)</em> complemented each other with 94% of the progeny of F1 <em>trans</em>-heterozygotes hatching at 24° (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a> and <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). We conclude that <em><a href="http://www.wormbase.org/db/get?name=WBGene00003792;class=Gene">npp-6</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00093918;class=Variation">ok2821</a>∆)</em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00001260;class=Gene">emb-6</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00145372;class=Variation">g36ts</a>)</em> are not alleles of the same gene. On the other hand <em><a href="http://www.wormbase.org/db/get?name=WBGene00001260;class=Gene">emb-6</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00145372;class=Variation">g36ts</a>)</em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00022455;class=Gene">tyms-1</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00251316;class=Variation">tm2429</a>∆)</em> fail to complement (0% hatching, <a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a> and <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>) suggesting that <em><a href="http://www.wormbase.org/db/get?name=WBGene00022455;class=Gene">tyms-1</a></em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00001260;class=Gene">emb-6</a></em> are the same gene. The <em><a href="http://www.wormbase.org/db/get?name=WBVar00145372;class=Variation">g36ts</a></em> allele has a G→A transition (Gly42 to Arg) in <em><a href="http://www.wormbase.org/db/get?name=WBGene00022455;class=Gene">tyms-1</a></em>. As further confirmation, we sequenced the <em><a href="http://www.wormbase.org/db/get?name=WBGene00022455;class=Gene">tyms-1</a></em> gene in another <em><a href="http://www.wormbase.org/db/get?name=WBGene00001260;class=Gene">emb-6</a></em> allele, <em><a href="http://www.wormbase.org/db/get?name=WBVar00087771;class=Variation">hc65ts</a></em>, and found a missense mutation in <em><a href="http://www.wormbase.org/db/get?name=WBGene00022455;class=Gene">tyms-1</a></em> (C→T), which changes proline 71 to a serine.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/5/1007/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="emb-6(g36ts) fails to complement tyms-1(tm2429&#x2206;). The following complementation tests were performed: emb-6(g36ts); him-8(e1489) males crossed with tyms-1(tm2429&#x2206;)/hT2, npp-6(ok2831&#x2206;)/hT2, smgl-1(ok2423&#x2206;)/hT2, and emb-6(hc65ts) hermaphrodites. The progeny of single F1 trans-heterozygous hermaphrodites were scored for the number of dead embryos vs. hatched larvae and the results calculated as the average percent embryonic viability at 15&#xB0; or 24&#xB0;C. The broods of at least seven hermaphrodites were scored for each cross at 15&#xB0;. The broods of at least eight hermaphrodites were scored for each cross at 24&#xB0;. F1 trans-heterozygous hermaphrodites from the emb-6(g36ts); him-8(e1489) males crossed with emb-6(hc65ts) produced only one hatched larvae out of the 1906 embryos scored. *P &lt; 0.0001. Error bars indicate SD. N = total number of embryos and hatched larvae scored. At least 1100 embryos were scored for each cross." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1185382378" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;emb-6(g36ts)&lt;/em&gt; fails to complement &lt;em&gt;tyms-1(tm2429&#x2206;)&lt;/em&gt;. The following complementation tests were performed: &lt;em&gt;emb-6(g36ts)&lt;/em&gt;; &lt;em&gt;him-8(e1489)&lt;/em&gt; males crossed with &lt;em&gt;tyms-1(tm2429&#x2206;)/hT2&lt;/em&gt;, &lt;em&gt;npp-6(ok2831&#x2206;)/hT2&lt;/em&gt;, &lt;em&gt;smgl-1(ok2423&#x2206;)/hT2&lt;/em&gt;, and &lt;em&gt;emb-6(hc65ts)&lt;/em&gt; hermaphrodites. The progeny of single F1 &lt;em&gt;trans&lt;/em&gt;-heterozygous hermaphrodites were scored for the number of dead embryos &lt;em&gt;vs.&lt;/em&gt; hatched larvae and the results calculated as the average percent embryonic viability at 15&#xB0; or 24&#xB0;C. The broods of at least seven hermaphrodites were scored for each cross at 15&#xB0;. The broods of at least eight hermaphrodites were scored for each cross at 24&#xB0;. F1 &lt;em&gt;trans&lt;/em&gt;-heterozygous hermaphrodites from the &lt;em&gt;emb-6(g36ts)&lt;/em&gt;; &lt;em&gt;him-8(e1489)&lt;/em&gt; males crossed with &lt;em&gt;emb-6(hc65ts)&lt;/em&gt; produced only one hatched larvae out of the 1906 embryos scored. *&lt;em&gt;P&lt;/em&gt; &lt; 0.0001. Error bars indicate SD. N = total number of embryos and hatched larvae scored. At least 1100 embryos were scored for each cross.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/5/1007/F3.medium.gif" width="440" height="313"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/5/1007/F3.medium.gif" width="440" height="313"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/5/1007/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/5/1007/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466032" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-25" class="first-child"><em>emb-6(g36ts)</em> fails to complement <em>tyms-1(tm2429∆)</em>. The following complementation tests were performed: <em>emb-6(g36ts)</em>; <em>him-8(e1489)</em> males crossed with <em>tyms-1(tm2429∆)/hT2</em>, <em>npp-6(ok2831∆)/hT2</em>, <em>smgl-1(ok2423∆)/hT2</em>, and <em>emb-6(hc65ts)</em> hermaphrodites. The progeny of single F1 <em>trans</em>-heterozygous hermaphrodites were scored for the number of dead embryos <em>vs.</em> hatched larvae and the results calculated as the average percent embryonic viability at 15° or 24°C. The broods of at least seven hermaphrodites were scored for each cross at 15°. The broods of at least eight hermaphrodites were scored for each cross at 24°. F1 <em>trans</em>-heterozygous hermaphrodites from the <em>emb-6(g36ts)</em>; <em>him-8(e1489)</em> males crossed with <em>emb-6(hc65ts)</em> produced only one hatched larvae out of the 1906 embryos scored. *<em>P</em> &lt; 0.0001. Error bars indicate SD. N = total number of embryos and hatched larvae scored. At least 1100 embryos were scored for each cross.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">Even though <em><a href="http://www.wormbase.org/db/get?name=WBGene00008990;class=Gene">smgl-1</a> RNAi</em> did not phenocopy <em><a href="http://www.wormbase.org/db/get?name=WBGene00001260;class=Gene">emb-6</a></em>, a deletion mutant allele of <em><a href="http://www.wormbase.org/db/get?name=WBGene00008990;class=Gene">smgl-1</a></em> was available from the Caenorhabditis Genetics Center that was embryonic lethal. To further rule out <em><a href="http://www.wormbase.org/db/get?name=WBGene00008990;class=Gene">smgl-1</a></em> as a candidate for <em><a href="http://www.wormbase.org/db/get?name=WBGene00001260;class=Gene">emb-6</a></em>, we performed complementation tests between <em><a href="http://www.wormbase.org/db/get?name=WBGene00001260;class=Gene">emb-6</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00145372;class=Variation">g36ts</a>)</em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00008990;class=Gene">smgl-1</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00093572;class=Variation">ok2423</a>∆)</em>. The <em>trans</em>-heterozygotes complemented at 24° (95% hatching, <a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a> and <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>) ruling out <em><a href="http://www.wormbase.org/db/get?name=WBGene00008990;class=Gene">smgl-1</a></em> as an <em><a href="http://www.wormbase.org/db/get?name=WBGene00001260;class=Gene">emb-6</a></em> candidate. Because <em><a href="http://www.wormbase.org/db/get?name=WBGene00022455;class=Gene">tyms-1</a></em> is the only candidate that both phenocopied the <em><a href="http://www.wormbase.org/db/get?name=WBGene00001260;class=Gene">emb-6</a></em> embryonic lethality and failed to complement, we conclude that <em><a href="http://www.wormbase.org/db/get?name=WBGene00001260;class=Gene">emb-6</a></em> is an allele of <em><a href="http://www.wormbase.org/db/get?name=WBGene00022455;class=Gene">tyms-1</a></em>, an ortholog of thymidylate synthase. Thymidylate synthase is involved in the pyrimidine biosynthesis pathway and is a major target of 5-fluorouracil, an anticancer drug (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kim <em>et al.</em> 2008</a>). The Gly42 to Arg mutation found in <em><a href="http://www.wormbase.org/db/get?name=WBGene00001260;class=Gene">emb-6</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00145372;class=Variation">g36ts</a>)</em> and the Pro71 to Ser mutation of <em><a href="http://www.wormbase.org/db/get?name=WBGene00001260;class=Gene">emb-6</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00087771;class=Variation">hc65ts</a>)</em> are located in the pyrimidine hydroxymethylase domain.</p></div><div id="sec-13" class="subsection"><h3>emb-17(g20ts) is an allele of chaf-1</h3><p id="p-27">Previous mapping of <em><a href="http://www.wormbase.org/db/get?name=WBGene00001271;class=Gene">emb-17</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00145360;class=Variation">g20ts</a>)</em> placed it on chromosome I but its position along the chromosome was not determined. WGS/SNP mapping found a reduction in the number of Hawaiian SNPs on chromosome I within a 7.2-Mb interval. This interval contained 10 nonsynonymous sequence variants (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). Of the 10 candidates, four were reported to have some degree of embryonic lethality by RNAi (<em><a href="http://www.wormbase.org/db/get?name=WBGene00004273;class=Gene">rab-10</a></em>, <em><a href="http://www.wormbase.org/db/get?name=WBGene00009052;class=Gene">ekl-1</a></em>, <em><a href="http://www.wormbase.org/db/get?name=WBGene00008990;class=Gene">smgl-1</a></em>, and <em><a href="http://www.wormbase.org/db/get?name=WBGene00011532;class=Gene">chaf-1</a></em>). RNAi clones were available for <em><a href="http://www.wormbase.org/db/get?name=WBGene00009052;class=Gene">ekl-1</a></em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00008990;class=Gene">smgl-1</a></em>. To determine whether either DNA lesion found in <em><a href="http://www.wormbase.org/db/get?name=WBGene00009052;class=Gene">ekl-1</a></em> or <em><a href="http://www.wormbase.org/db/get?name=WBGene00008990;class=Gene">smgl-1</a></em> resulted in the same phenotype as <em><a href="http://www.wormbase.org/db/get?name=WBGene00001271;class=Gene">emb-17</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00145360;class=Variation">g20ts</a>)</em>, we performed RNAi phenocopy experiments. As stated in the previous section, RNAi knock-down of <em><a href="http://www.wormbase.org/db/get?name=WBGene00008990;class=Gene">smgl-1</a></em> did not result in embryonic lethality but RNAi knock-down of <em><a href="http://www.wormbase.org/db/get?name=WBGene00009052;class=Gene">ekl-1</a></em> resulted in 42% embryonic lethality (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465920/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465920/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title"><em>emb-17(g20ts)</em> is an allele of <em>chaf</em>-<em>1</em></span><div class="sb-div caption-clear"></div></div></div><p id="p-29">Previous analysis of deletion alleles of three of the 10 candidates found them to be sterile [<em><a href="http://www.wormbase.org/db/get?name=WBGene00009052;class=Gene">ekl-1</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00296657;class=Variation">ok1197</a>∆)</em>] or embryonic lethal [<em><a href="http://www.wormbase.org/db/get?name=WBGene00008990;class=Gene">smgl-1</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00093572;class=Variation">ok2423</a>∆)</em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00011532;class=Gene">chaf-1</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00603944;class=Variation">n5453</a>∆</em>)]. For the remaining candidates, deletion alleles were either not available (<em><a href="http://www.wormbase.org/db/get?name=WBGene00016603;class=Gene">met-1</a></em>, <em><a href="http://www.wormbase.org/db/get?name=WBGene00019768;class=Gene">M04F3.2</a></em>, <em><a href="http://www.wormbase.org/db/get?name=WBGene00004324;class=Gene">rde-2</a></em>, <em><a href="http://www.wormbase.org/db/get?name=WBGene00010135;class=Gene">F55H12.4</a></em>, and <em><a href="http://www.wormbase.org/db/get?name=WBGene00015017;class=Gene">B0205.1</a></em>) or not characterized as sterile or embryonic lethal (<em><a href="http://www.wormbase.org/db/get?name=WBGene00003060;class=Gene">lpd-3</a></em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00004273;class=Gene">rab-10</a></em>). To further validate the mutant candidate responsible for the <em><a href="http://www.wormbase.org/db/get?name=WBGene00001271;class=Gene">emb-17</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00145360;class=Variation">g20ts</a>)</em> embryonic lethal phenotype, we performed complementation tests between <em><a href="http://www.wormbase.org/db/get?name=WBGene00001271;class=Gene">emb-17</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00145360;class=Variation">g20ts</a>)</em>; <em><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00144038;class=Variation">e1489</a>)</em> males and hermaphrodites carrying deletion alleles of the gene candidates. Single F1 <em>trans</em>-heterozygous cross progeny from these crosses were scored for dead F2 embryos <em>vs.</em> viable progeny at both 15° and 24°. <em><a href="http://www.wormbase.org/db/get?name=WBGene00009052;class=Gene">ekl-1</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00296657;class=Variation">ok1197</a>∆)</em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00008990;class=Gene">smgl-1</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00093572;class=Variation">ok2423</a>∆)</em> both complemented <em><a href="http://www.wormbase.org/db/get?name=WBGene00001271;class=Gene">emb-17</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00145360;class=Variation">g20ts</a>)</em> (97% and 87% hatching of F2 progeny at 24°, <a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a> and <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>), suggesting that neither of these genes are allelic to <em><a href="http://www.wormbase.org/db/get?name=WBGene00001271;class=Gene">emb-17</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00145360;class=Variation">g20ts</a>)</em>. Although <em><a href="http://www.wormbase.org/db/get?name=WBGene00003060;class=Gene">lpd-3</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00093320;class=Variation">ok2143</a>∆)</em> was not characterized to be sterile or embryonic lethal, a deletion allele existed. Therefore, we also performed complementation tests between <em><a href="http://www.wormbase.org/db/get?name=WBGene00001271;class=Gene">emb-17</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00145360;class=Variation">g20ts</a>)</em>; <em><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00144038;class=Variation">e1489</a>)</em> males and <em><a href="http://www.wormbase.org/db/get?name=WBGene00003060;class=Gene">lpd-3</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00093320;class=Variation">ok2143</a>∆)</em> hermaphrodites. <em><a href="http://www.wormbase.org/db/get?name=WBGene00003060;class=Gene">lpd-3</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00093320;class=Variation">ok2143</a>∆)</em> complemented <em><a href="http://www.wormbase.org/db/get?name=WBGene00001271;class=Gene">emb-17</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00145360;class=Variation">g20ts</a>)</em> (98% hatching of F2 progeny at 24°, <a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a> and <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). Only <em>trans</em>-heterozygous F1s from <em><a href="http://www.wormbase.org/db/get?name=WBGene00001271;class=Gene">emb-17</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00145360;class=Variation">g20ts</a>)</em>; <em><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00144038;class=Variation">e1489</a>)</em> males crossed with <em><a href="http://www.wormbase.org/db/get?name=WBGene00011532;class=Gene">chaf-1</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00603944;class=Variation">n5453</a>∆)/<a href="http://www.wormbase.org/db/get?name=hT2;class=Rearrangement">hT2</a></em> hermaphrodites produced dead embryos (4.9% and 2.8% hatching at 24°, <a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 3</a> and <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>). Furthermore, F1 <em>trans</em>-heterozygotes were not healthy at the permissive temperature of 15° (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>), suggesting that these two alleles do not make enough active gene product at the permissive temperature to support development. Taken together these data show that <em><a href="http://www.wormbase.org/db/get?name=WBGene00011532;class=Gene">chaf-1</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00603944;class=Variation">n5453</a>∆)</em> fails to complement <em><a href="http://www.wormbase.org/db/get?name=WBGene00001271;class=Gene">emb-17</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00145360;class=Variation">g20ts</a>)</em>, demonstrating that they are alleles of the same gene that codes for the chromatin assembly factor chromatin assembly factor-1 p150 subunit. Chromatin assembly factor-1 is a three-subunit protein complex (p150, p60, and p48) required for the deposition of histones H3-H4 into nucleosomes during DNA replication (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Nakano <em>et al.</em> 2011</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Smith and Stillman 1989</a>). The <em><a href="http://www.wormbase.org/db/get?name=WBVar00145360;class=Variation">g20ts</a></em> allele is a C→T transition that changes Arg436 to His in the p60 binding domain of the p150 subunit.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/5/1007/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="emb-17(g20ts) fails to complement chaf-1(n5453&#x2206;). Complementation tests were performed by crossing emb-17(g20ts); him-8(e1489) males with lpd-3(ok2138&#x2206;), ekl-1(ok1197&#x2206;)/hT2, smgl-1(ok2423&#x2206;)/hT2, or chaf-1(n5453&#x2206;)/hT2 (Cross 1) hermaphrodites. In addition, emb-17(g20ts); him-5(e1490) males were crossed with chaf-1(n5453&#x2206;)/hT2 hermaphrodites (Cross 2). The progeny of single F1 trans-heterozygous hermaphrodites were scored for the number of dead embryos vs. hatched larvae and the results calculated as the average percent embryonic viability at 15&#xB0; or 24&#xB0;. The broods of at least seven F1 hermaphrodites were scored for each cross at 15&#xB0;. The broods of at least five F1 hermaphrodites were scored for each cross at 24&#xB0;. *P &lt; 0.0001. Error bars indicate standard deviation. At least 900 embryos were scored for each cross." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1185382378" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;emb-17(g20ts)&lt;/em&gt; fails to complement &lt;em&gt;chaf-1(n5453&#x2206;)&lt;/em&gt;. Complementation tests were performed by crossing &lt;em&gt;emb-17(g20ts)&lt;/em&gt;; &lt;em&gt;him-8(e1489)&lt;/em&gt; males with &lt;em&gt;lpd-3(ok2138&#x2206;)&lt;/em&gt;, &lt;em&gt;ekl-1(ok1197&#x2206;)/hT2&lt;/em&gt;, &lt;em&gt;smgl-1(ok2423&#x2206;)/hT2&lt;/em&gt;, or &lt;em&gt;chaf-1(n5453&#x2206;)/hT2&lt;/em&gt; (Cross 1) hermaphrodites. In addition, &lt;em&gt;emb-17(g20ts)&lt;/em&gt;; &lt;em&gt;him-5(e1490)&lt;/em&gt; males were crossed with &lt;em&gt;chaf-1(n5453&#x2206;)/hT2&lt;/em&gt; hermaphrodites (Cross 2). The progeny of single F1 &lt;em&gt;trans&lt;/em&gt;-heterozygous hermaphrodites were scored for the number of dead embryos &lt;em&gt;vs.&lt;/em&gt; hatched larvae and the results calculated as the average percent embryonic viability at 15&#xB0; or 24&#xB0;. The broods of at least seven F1 hermaphrodites were scored for each cross at 15&#xB0;. The broods of at least five F1 hermaphrodites were scored for each cross at 24&#xB0;. *&lt;em&gt;P&lt;/em&gt; &lt; 0.0001. Error bars indicate standard deviation. At least 900 embryos were scored for each cross.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/5/1007/F4.medium.gif" width="440" height="344"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/5/1007/F4.medium.gif" width="440" height="344"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/5/1007/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/5/1007/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466117" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-30" class="first-child"><em>emb-17(g20ts)</em> fails to complement <em>chaf-1(n5453∆)</em>. Complementation tests were performed by crossing <em>emb-17(g20ts)</em>; <em>him-8(e1489)</em> males with <em>lpd-3(ok2138∆)</em>, <em>ekl-1(ok1197∆)/hT2</em>, <em>smgl-1(ok2423∆)/hT2</em>, or <em>chaf-1(n5453∆)/hT2</em> (Cross 1) hermaphrodites. In addition, <em>emb-17(g20ts)</em>; <em>him-5(e1490)</em> males were crossed with <em>chaf-1(n5453∆)/hT2</em> hermaphrodites (Cross 2). The progeny of single F1 <em>trans</em>-heterozygous hermaphrodites were scored for the number of dead embryos <em>vs.</em> hatched larvae and the results calculated as the average percent embryonic viability at 15° or 24°. The broods of at least seven F1 hermaphrodites were scored for each cross at 15°. The broods of at least five F1 hermaphrodites were scored for each cross at 24°. *<em>P</em> &lt; 0.0001. Error bars indicate standard deviation. At least 900 embryos were scored for each cross.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>emb-10(k12ts) and zyg-2(b10ts) are alleles of mus-101</h3><p id="p-31">Mapping data of <em><a href="http://www.wormbase.org/db/get?name=WBGene00001264;class=Gene">emb-10</a></em> placed it on chromosome I at 5.74 cM and <em><a href="http://www.wormbase.org/db/get?name=WBGene00006989;class=Gene">zyg-2</a></em> was previously mapped to chromosome I at 6.24 cM. WGS/SNP mapping placed <em><a href="http://www.wormbase.org/db/get?name=WBGene00001264;class=Gene">emb-10</a></em> in a 0.9-Mb interval on chromosome I and <em><a href="http://www.wormbase.org/db/get?name=WBGene00006989;class=Gene">zyg-2</a></em> within a 5.2-Mb interval on chromosome I. Only two nonsynonymous sequence variants were identified for both <em><a href="http://www.wormbase.org/db/get?name=WBGene00001264;class=Gene">emb-10</a></em> (<em><a href="http://www.wormbase.org/db/get?name=WBGene00009507;class=Gene">mus-101</a></em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00010327;class=Gene">F59C6.8</a></em>, <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>) and <em><a href="http://www.wormbase.org/db/get?name=WBGene00006989;class=Gene">zyg-2</a></em> (<em><a href="http://www.wormbase.org/db/get?name=WBGene00009507;class=Gene">mus-101</a></em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00003068;class=Gene">lrk-1</a></em>, <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>). Previous studies had not tested whether these two mutants might be alleles of the same gene. However, the WGS/SNP mapping analysis found <em><a href="http://www.wormbase.org/db/get?name=WBGene00009507;class=Gene">mus-101</a></em> lesions as candidates for both mutants. In addition, <em><a href="http://www.wormbase.org/db/get?name=WBGene00009507;class=Gene">mus-101</a></em> was the only candidate reported to be embryonic lethal by RNAi. To validate whether the <em><a href="http://www.wormbase.org/db/get?name=WBGene00009507;class=Gene">mus-101</a></em> lesions were responsible for the <em><a href="http://www.wormbase.org/db/get?name=WBGene00001264;class=Gene">emb-10</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00088187;class=Variation">k12ts</a>)</em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00006989;class=Gene">zyg-2</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00000351;class=Variation">b10ts</a>)</em> phenotypes, we performed RNAi phenocopy experiments. However, <em><a href="http://www.wormbase.org/db/get?name=WBGene00009507;class=Gene">mus-101</a></em> RNAi was not embryonic lethal in our hands.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465972/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465972/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title"><em>emb-10(k12ts)</em> is an allele of <em>mus-101</em></span><div class="sb-div caption-clear"></div></div></div><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465987/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465987/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title"><em>zyg-2(b10ts)</em> is an allele of <em>mus-101</em></span><div class="sb-div caption-clear"></div></div></div><p id="p-34">Mutant analysis had previously determined that <em><a href="http://www.wormbase.org/db/get?name=WBGene00009507;class=Gene">mus-101</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00250726;class=Variation">tm1761</a>∆)</em> was sterile and/or lethal. To validate whether the <em><a href="http://www.wormbase.org/db/get?name=WBGene00009507;class=Gene">mus-101</a></em> genetic lesions identified from WGS of <em><a href="http://www.wormbase.org/db/get?name=WBGene00001264;class=Gene">emb-10</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00088187;class=Variation">k12ts</a>)</em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00006989;class=Gene">zyg-2</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00000351;class=Variation">b10ts</a>)</em> were indeed alleles of <em><a href="http://www.wormbase.org/db/get?name=WBGene00009507;class=Gene">mus-101</a></em>, complementation tests were performed. Matings between <em><a href="http://www.wormbase.org/db/get?name=WBGene00001264;class=Gene">emb-10</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00088187;class=Variation">k12ts</a>)</em> males or <em><a href="http://www.wormbase.org/db/get?name=WBGene00006989;class=Gene">zyg-2</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00000351;class=Variation">b10ts</a>)</em>; <em><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> males and <em><a href="http://www.wormbase.org/db/get?name=WBGene00009507;class=Gene">mus-101</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00250726;class=Variation">tm1761</a>∆)/<a href="http://www.wormbase.org/db/get?name=hT2;class=Rearrangement">hT2</a></em> hermaphrodites were performed. Single F1 <em>trans</em>-heterozygous cross progeny were analyzed for the number of dead F2 embryos <em>vs.</em> hatched F2 progeny at 15° and 24°. <em><a href="http://www.wormbase.org/db/get?name=WBGene00009507;class=Gene">mus-101</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00250726;class=Variation">tm1761</a>∆)</em> failed to complement both <em><a href="http://www.wormbase.org/db/get?name=WBGene00001264;class=Gene">emb-10</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00088187;class=Variation">k12ts</a>)</em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00006989;class=Gene">zyg-2</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00000351;class=Variation">b10ts</a>)</em> (1% and 0% hatching at 24°, <a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>, and <a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table 5</a>, and <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>) demonstrating that all three were allelic. In addition, embryonic viability of <em><a href="http://www.wormbase.org/db/get?name=WBGene00009507;class=Gene">mus-101</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00250726;class=Variation">tm1761</a>∆)/<a href="http://www.wormbase.org/db/get?name=WBGene00006989;class=Gene">zyg-2</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00000351;class=Variation">b10ts</a>)</em> F1 <em>trans</em>-heterozygotes at the permissive temperature of 15° was only 3.9% (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>) suggesting that the nonsense allele of <em><a href="http://www.wormbase.org/db/get?name=WBGene00006989;class=Gene">zyg-2</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00000351;class=Variation">b10ts</a>)</em> is stronger than the missense allele of <em><a href="http://www.wormbase.org/db/get?name=WBGene00001264;class=Gene">emb-10</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00088187;class=Variation">k12ts</a>)</em> when combined with the deletion allele of <em><a href="http://www.wormbase.org/db/get?name=WBGene00009507;class=Gene">mus-101</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00250726;class=Variation">tm1761</a>∆)</em>. To further validate that <em><a href="http://www.wormbase.org/db/get?name=WBGene00001264;class=Gene">emb-10</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00088187;class=Variation">k12ts</a>)</em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00006989;class=Gene">zyg-2</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00000351;class=Variation">b10ts</a>)</em> are two alleles of the same gene, we crossed <em><a href="http://www.wormbase.org/db/get?name=WBGene00001264;class=Gene">emb-10</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00088187;class=Variation">k12ts</a>)</em> males with <em><a href="http://www.wormbase.org/db/get?name=WBGene00006989;class=Gene">zyg-2</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00000351;class=Variation">b10ts</a>)</em> hermaphrodites and analyzed the progeny of single F1 <em>trans</em>-heterozygous animals. <em><a href="http://www.wormbase.org/db/get?name=WBGene00001264;class=Gene">emb-10</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00088187;class=Variation">k12ts</a>)</em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00006989;class=Gene">zyg-2</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00000351;class=Variation">b10ts</a>)</em> fail to complement with only 3.6% hatching at 24° (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>, Cross 1). The reciprocal mating [<em><a href="http://www.wormbase.org/db/get?name=WBGene00006989;class=Gene">zyg-2</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00000351;class=Variation">b10ts</a>)</em>; <em><a href="http://www.wormbase.org/db/get?name=WBGene00001867;class=Gene">him-8</a></em> males crossed with <em><a href="http://www.wormbase.org/db/get?name=WBGene00001264;class=Gene">emb-10</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00088187;class=Variation">k12ts</a>)</em> hermaphrodites] also failed to complement (3.5% hatching, Cross 2). These data demonstrate that <em><a href="http://www.wormbase.org/db/get?name=WBGene00001264;class=Gene">emb-10</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00088187;class=Variation">k12ts</a>)</em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00006989;class=Gene">zyg-2</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00000351;class=Variation">b10ts</a>)</em> are temperature-sensitive alleles of <em><a href="http://www.wormbase.org/db/get?name=WBGene00009507;class=Gene">mus-101</a></em>, a member of the Mus101 gene family involved in DNA damage repair and cell-cycle checkpoint control. In <em>C. elegans</em> the <em><a href="http://www.wormbase.org/db/get?name=WBGene00009507;class=Gene">mus-101</a></em> homolog is a 1227 amino acid protein that plays a role in both DNA replication and in the response to DNA damage (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Holway <em>et al.</em> 2005</a>). <em><a href="http://www.wormbase.org/db/get?name=WBGene00001264;class=Gene">emb-10</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00088187;class=Variation">k12ts</a>)</em> is a G→A change resulting in Cys268 to a Tyr in one of the six BRCT domains, which mediate protein−protein and protein−DNA interactions. <em><a href="http://www.wormbase.org/db/get?name=WBGene00006989;class=Gene">zyg-2</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00000351;class=Variation">b10ts</a>)</em> is a G→A transition that changes Trp1062 to a premature stop codon. The 165-bp region removed by the <em><a href="http://www.wormbase.org/db/get?name=WBGene00006989;class=Gene">zyg-2</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00000351;class=Variation">b10ts</a>)</em> premature stop is not predicted to contain a functional domain. However, an oligomerization domain resides in the C-terminus of the human homolog TOPBP1 (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Sokka <em>et al.</em> 2010</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/5/1007/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="emb-10(k12ts) and zyg-2(b10ts) fail to complement mus-101(tm1761&#x2206;) and each other. The following complementation tests were performed: emb-10(k12ts) males crossed with mus-101(tm1761&#x2206;)/hT2 hermaphrodites, zyg-2(b10ts); him-8(e1489) males crossed with mus-101(tm1761&#x2206;)/hT2 hermaphrodites, emb-10(k12ts) males crossed with zyg-2(b10ts) hermaphrodites (Cross 1), and zyg-2(b10ts); him-8(e1489) males crossed with emb-10(k12ts) hermaphrodites (Cross 2). The progeny of single F1 trans-heterozygous hermaphrodites were scored for the number of dead embryos vs. larvae and the results calculated as the average percent embryonic viability at 15&#xB0; or 24&#xB0;. The number of mus-101(tm1761&#x2206;)/emb-10(k12ts) hermaphrodites scored at 15&#xB0; = 12 and 24&#xB0; = 12. The number of mus-101(tm1761&#x2206;)/ zyg-2(b10ts) hermaphrodites scored at 15&#xB0; = 8 and 24&#xB0; = 9. The number of emb-10(k12ts)/zyg-2(b10ts) hermaphrodites scored for Cross 1 at 15&#xB0; = 10 and 24&#xB0; = 12 and for Cross 2 at 15&#xB0; = 3 and 24&#xB0; = 4. N = total number of embryos and hatched larvae scored. *P &lt; 0.0001. **P &lt; 0.05. Error bars indicate standard deviation. At least 854 embryos were scored for each cross." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1185382378" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;emb-10(k12ts)&lt;/em&gt; and &lt;em&gt;zyg-2(b10ts)&lt;/em&gt; fail to complement &lt;em&gt;mus-101(tm1761&#x2206;)&lt;/em&gt; and each other. The following complementation tests were performed: &lt;em&gt;emb-10(k12ts)&lt;/em&gt; males crossed with &lt;em&gt;mus-101(tm1761&#x2206;)/hT2&lt;/em&gt; hermaphrodites, &lt;em&gt;zyg-2(b10ts)&lt;/em&gt;; &lt;em&gt;him-8(e1489)&lt;/em&gt; males crossed with &lt;em&gt;mus-101(tm1761&#x2206;)/hT2&lt;/em&gt; hermaphrodites, &lt;em&gt;emb-10(k12ts)&lt;/em&gt; males crossed with &lt;em&gt;zyg-2(b10ts)&lt;/em&gt; hermaphrodites (Cross 1), and &lt;em&gt;zyg-2(b10ts)&lt;/em&gt;; &lt;em&gt;him-8(e1489)&lt;/em&gt; males crossed with &lt;em&gt;emb-10(k12ts)&lt;/em&gt; hermaphrodites (Cross 2). The progeny of single F1 &lt;em&gt;trans&lt;/em&gt;-heterozygous hermaphrodites were scored for the number of dead embryos &lt;em&gt;vs.&lt;/em&gt; larvae and the results calculated as the average percent embryonic viability at 15&#xB0; or 24&#xB0;. The number of &lt;em&gt;mus-101(tm1761&#x2206;)/emb-10(k12ts)&lt;/em&gt; hermaphrodites scored at 15&#xB0; = 12 and 24&#xB0; = 12. The number of &lt;em&gt;mus-101(tm1761&#x2206;)/ zyg-2(b10ts)&lt;/em&gt; hermaphrodites scored at 15&#xB0; = 8 and 24&#xB0; = 9. The number of &lt;em&gt;emb-10(k12ts)/zyg-2(b10ts)&lt;/em&gt; hermaphrodites scored for Cross 1 at 15&#xB0; = 10 and 24&#xB0; = 12 and for Cross 2 at 15&#xB0; = 3 and 24&#xB0; = 4. N = total number of embryos and hatched larvae scored. *&lt;em&gt;P&lt;/em&gt; &lt; 0.0001. **&lt;em&gt;P&lt;/em&gt; &lt; 0.05. Error bars indicate standard deviation. At least 854 embryos were scored for each cross.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/5/1007/F5.medium.gif" width="440" height="357"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/5/5/1007/F5.medium.gif" width="440" height="357"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/5/1007/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/5/1007/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465994" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-35" class="first-child"><em>emb-10(k12ts)</em> and <em>zyg-2(b10ts)</em> fail to complement <em>mus-101(tm1761∆)</em> and each other. The following complementation tests were performed: <em>emb-10(k12ts)</em> males crossed with <em>mus-101(tm1761∆)/hT2</em> hermaphrodites, <em>zyg-2(b10ts)</em>; <em>him-8(e1489)</em> males crossed with <em>mus-101(tm1761∆)/hT2</em> hermaphrodites, <em>emb-10(k12ts)</em> males crossed with <em>zyg-2(b10ts)</em> hermaphrodites (Cross 1), and <em>zyg-2(b10ts)</em>; <em>him-8(e1489)</em> males crossed with <em>emb-10(k12ts)</em> hermaphrodites (Cross 2). The progeny of single F1 <em>trans</em>-heterozygous hermaphrodites were scored for the number of dead embryos <em>vs.</em> larvae and the results calculated as the average percent embryonic viability at 15° or 24°. The number of <em>mus-101(tm1761∆)/emb-10(k12ts)</em> hermaphrodites scored at 15° = 12 and 24° = 12. The number of <em>mus-101(tm1761∆)/ zyg-2(b10ts)</em> hermaphrodites scored at 15° = 8 and 24° = 9. The number of <em>emb-10(k12ts)/zyg-2(b10ts)</em> hermaphrodites scored for Cross 1 at 15° = 10 and 24° = 12 and for Cross 2 at 15° = 3 and 24° = 4. N = total number of embryos and hatched larvae scored. *<em>P</em> &lt; 0.0001. **<em>P</em> &lt; 0.05. Error bars indicate standard deviation. At least 854 embryos were scored for each cross.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3><em>mel-2(it20)</em> is an allele of <em>tofu-6</em></h3><p id="p-36"><em><a href="http://www.wormbase.org/db/get?name=WBGene00003188;class=Gene">mel-2</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00088001;class=Variation">it20</a>)</em> was originally identified in a screen for maternal-effect lethal mutants on linkage group II and was mapped to the left of <em><a href="http://www.wormbase.org/db/get?name=WBGene00001072;class=Gene">dpy-10</a></em> (<a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">Kemphues <em>et al.</em> 1988</a>). WGS/SNP mapping found a 1.3-Mb interval with a reduction in Hawaiian SNPs on chromosome II (<a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a>). Seven homozygous, nonsynonymous sequence variants were identified within that interval; two variants were identified within the same gene, <em><a href="http://www.wormbase.org/db/get?name=WBGene00020499;class=Gene">T14B4.2</a></em>. Two of the candidates were not reported to have an embryonic lethal phenotype by RNAi (<em><a href="http://www.wormbase.org/db/get?name=WBGene00016117;class=Gene">C25H3.8</a></em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00044315;class=Gene">F41G3.20</a></em>), and one was reported to have a low penetrance of embryonic lethality (<em>cyp-4</em>) and were not investigated further (<a id="xref-table-wrap-6-2" class="xref-table" href="#T6">Table 6</a>). The other three candidates were reported to have a range of embryonic lethality by RNAi (<em><a href="http://www.wormbase.org/db/get?name=WBGene00017132;class=Gene">tofu-6</a></em>, <em><a href="http://www.wormbase.org/db/get?name=WBGene00016966;class=Gene">C56C10.10</a></em>, and <em><a href="http://www.wormbase.org/db/get?name=WBGene00020499;class=Gene">T14B4.2</a></em>).</p><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466023/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466023/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6</span> <span class="caption-title"><em>mel-2(it20)</em> is an allele of <em>tofu-6</em></span><div class="sb-div caption-clear"></div></div></div><p id="p-38">Because <em><a href="http://www.wormbase.org/db/get?name=WBGene00017132;class=Gene">tofu-6</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00275444;class=Variation">yt2</a>)</em> was identified as an allele that caused 100% maternal-effect embryonic lethality in a screen for altered expression of an early embryonic reporter construct (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Minasaki and Streit 2007</a>), it was a likely candidate. A combination of classical mapping, SNP mapping, and sequencing identified the <em><a href="http://www.wormbase.org/db/get?name=WBVar00275444;class=Variation">yt2</a></em> allele as a premature stop codon. An extrachromosomal array (<em><a href="http://www.wormbase.org/db/get?name=WBTransgene00017528;class=Transgene">ytEx100</a></em>) containing a 3245-bp genomic fragment including wild-type <em><a href="http://www.wormbase.org/db/get?name=WBGene00017132;class=Gene">tofu-6</a></em> rescued the embryonic lethality of <em><a href="http://www.wormbase.org/db/get?name=WBGene00017132;class=Gene">tofu-6</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00275444;class=Variation">yt2</a>)</em> (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Minasaki and Streit 2007</a>). To validate whether <em><a href="http://www.wormbase.org/db/get?name=WBGene00003188;class=Gene">mel-2</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00088001;class=Variation">it20</a>)</em> is an allele of <em><a href="http://www.wormbase.org/db/get?name=WBGene00017132;class=Gene">tofu-6</a></em>, we crossed <em><a href="http://www.wormbase.org/db/get?name=WBGene00003188;class=Gene">mel-2</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00088001;class=Variation">it20</a>) <a href="http://www.wormbase.org/db/get?name=WBGene00006744;class=Gene">unc-4</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00142975;class=Variation">e120</a>)/++</em> heterozygous males with rolling <em><a href="http://www.wormbase.org/db/get?name=WBGene00017132;class=Gene">tofu-6</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00275444;class=Variation">yt2</a>)</em>; <em><a href="http://www.wormbase.org/db/get?name=WBTransgene00017528;class=Transgene">ytEx100</a></em> [<em><a href="http://www.wormbase.org/db/get?name=WBTransgene00017528;class=Transgene">ytEx100</a></em> is marked with a dominant marker <em><a href="http://www.wormbase.org/db/get?name=WBGene00004397;class=Gene">rol-6</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00248869;class=Variation">su1006</a>)</em>] hermaphrodites. The RolUnc progeny of F1 rollers were picked and scored for hatched progeny. All <em><a href="http://www.wormbase.org/db/get?name=WBGene00003188;class=Gene">mel-2</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00088001;class=Variation">it20</a>) <a href="http://www.wormbase.org/db/get?name=WBGene00006744;class=Gene">unc-4</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00142975;class=Variation">e120</a>)</em>; <em><a href="http://www.wormbase.org/db/get?name=WBTransgene00017528;class=Transgene">ytEx100</a></em> animals produced live progeny whereas <a href="http://www.wormbase.org/db/get?name=WBPhenotype%3A0000643;class=Phenotype">Unc</a> non-<a href="http://www.wormbase.org/db/get?name=WBPhenotype%3A0000645;class=Phenotype">Rol</a> animals that segregate from these lines exhibit a maternal-effect embryonic lethal phenotype (<a id="xref-table-wrap-6-3" class="xref-table" href="#T6">Table 6</a>). These experiments demonstrate that a transgene carrying <em><a href="http://www.wormbase.org/db/get?name=WBGene00017132;class=Gene">tofu-6</a></em> can rescue <em><a href="http://www.wormbase.org/db/get?name=WBGene00003188;class=Gene">mel-2</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00088001;class=Variation">it20</a>)</em>. From these data we conclude that <em><a href="http://www.wormbase.org/db/get?name=WBGene00003188;class=Gene">mel-2</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00088001;class=Variation">it20</a>)</em> is an allele of <em><a href="http://www.wormbase.org/db/get?name=WBGene00017132;class=Gene">tofu-6</a></em>, a gene involved in cell division timing of the early embryo (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Minasaki and Streit 2007</a>). <a href="http://www.wormbase.org/db/get?name=WBGene00017132;class=Gene">TOFU-6</a> encodes a 367 amino acid protein orthologous to SLIRP (human SRA stem-loop interacting RNA binding protein) with an RRM (RNA recognition motif) domain that spans amino acids 17−87. <em><a href="http://www.wormbase.org/db/get?name=WBGene00003188;class=Gene">mel-2</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00088001;class=Variation">it20</a>)</em> is a C→T transition that changes Gln52 to a stop codon. The premature stop truncates the protein at residue 52 disrupting the RRM domain.</p></div><div id="sec-16" class="subsection"><h3>emb-14(g43ts) is an allele of fasn-1</h3><p id="p-39"><em><a href="http://www.wormbase.org/db/get?name=WBGene00001268;class=Gene">emb-14</a></em> was previously mapped to chromosome I at 1.71 cM. Analysis of the sequence data generated by WGS/SNP mapping of <em><a href="http://www.wormbase.org/db/get?name=WBGene00001268;class=Gene">emb-14</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00145377;class=Variation">g43ts</a>)</em> found a reduction of Hawaiian SNPs on chromosome I within a 2.2 Mb interval. This interval contained seven nonsynonymous sequence variants (<a id="xref-table-wrap-7-1" class="xref-table" href="#T7">Table 7</a>). Of the seven candidates, only one was reported to have an embryonic lethal phenotype by RNAi (<em><a href="http://www.wormbase.org/db/get?name=WBGene00009342;class=Gene">fasn-1</a></em>) and one was reported as lethal or sterile by mutant analysis (<em><a href="http://www.wormbase.org/db/get?name=WBGene00019247;class=Gene">syp-4</a></em>). <em><a href="http://www.wormbase.org/db/get?name=WBGene00009342;class=Gene">fasn-1</a></em> RNAi phenocopied the embryonic lethal phenotype of <em><a href="http://www.wormbase.org/db/get?name=WBGene00001268;class=Gene">emb-14</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00145377;class=Variation">g43ts</a>)</em>, however, no deletion mutant was available for further validation by complementation tests. The deletion allele, <em><a href="http://www.wormbase.org/db/get?name=WBGene00019247;class=Gene">syp-4</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00251558;class=Variation">tm2713</a>∆)</em>, was used for complementation tests with <em><a href="http://www.wormbase.org/db/get?name=WBGene00001268;class=Gene">emb-14</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00145377;class=Variation">g43ts</a>)</em>. <em><a href="http://www.wormbase.org/db/get?name=WBGene00019247;class=Gene">syp-4</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00251558;class=Variation">tm2713</a>∆)/+</em> males were crossed with <em><a href="http://www.wormbase.org/db/get?name=WBGene00001268;class=Gene">emb-14</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00145377;class=Variation">g43ts</a>)</em> homozygous hermaphrodites and the progeny of <em>trans</em>-heterozygous F1 were examined for their ability to hatch at 15° and 24°. All F1 hermaphrodites produced viable progeny (81% hatching, <a id="xref-table-wrap-7-2" class="xref-table" href="#T7">Table 7</a>). To further validate the identity of <em><a href="http://www.wormbase.org/db/get?name=WBGene00001268;class=Gene">emb-14</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00145377;class=Variation">g43ts</a>)</em> seven <a href="http://www.wormbase.org/db/get?name=WBPhenotype%3A0000583;class=Phenotype">Dpy</a> non-<a href="http://www.wormbase.org/db/get?name=WBPhenotype%3A0000643;class=Phenotype">Unc</a> and seven <a href="http://www.wormbase.org/db/get?name=WBPhenotype%3A0000643;class=Phenotype">Unc</a> non-<a href="http://www.wormbase.org/db/get?name=WBPhenotype%3A0000583;class=Phenotype">Dpy</a> recombinants were isolated from <em><a href="http://www.wormbase.org/db/get?name=WBGene00001268;class=Gene">emb-14</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00145377;class=Variation">g43ts</a>)/<a href="http://www.wormbase.org/db/get?name=WBGene00001067;class=Gene">dpy-5</a> <a href="http://www.wormbase.org/db/get?name=WBGene00006752;class=Gene">unc-13</a></em> heterozygotes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017038/-/DC1">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2015/03/04/g3.115.017038.DC1/017038SI.pdf">Figure S1</a>A). After generating homozygous recombinants for the <em><a href="http://www.wormbase.org/db/get?name=WBGene00001067;class=Gene">dpy-5</a></em> or <em><a href="http://www.wormbase.org/db/get?name=WBGene00006752;class=Gene">unc-13</a></em> markers, these recombinants were shifted to 24° to test for the presence of <em><a href="http://www.wormbase.org/db/get?name=WBGene00001268;class=Gene">emb-14</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00145377;class=Variation">g43ts</a>)</em>. Each line was then subjected to polymerase chain reaction and sequencing of the <em><a href="http://www.wormbase.org/db/get?name=WBGene00009342;class=Gene">fasn-1</a></em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00019247;class=Gene">syp-4</a></em> genes and the sequences analyzed for the mutations identified by WGS. All seven <a href="http://www.wormbase.org/db/get?name=WBPhenotype%3A0000583;class=Phenotype">Dpy</a> non-<a href="http://www.wormbase.org/db/get?name=WBPhenotype%3A0000643;class=Phenotype">Unc</a> recombinants carried <em><a href="http://www.wormbase.org/db/get?name=WBGene00001268;class=Gene">emb-14</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00145377;class=Variation">g43ts</a>)</em> (<em>i.e.</em>, exhibited embryonic lethality at 24°). These seven lines also contained the <em><a href="http://www.wormbase.org/db/get?name=WBGene00009342;class=Gene">fasn-1</a></em> mutation (G→A, Ala1425 to Thr) and one of the lines was wild type for <em><a href="http://www.wormbase.org/db/get?name=WBGene00019247;class=Gene">syp-4</a></em> (<a href="http://www.g3journal.org/content/suppl/2015/03/04/g3.115.017038.DC1/017038SI.pdf">Figure S1</a>B). None of the seven <a href="http://www.wormbase.org/db/get?name=WBPhenotype%3A0000643;class=Phenotype">Unc</a> non-<a href="http://www.wormbase.org/db/get?name=WBPhenotype%3A0000583;class=Phenotype">Dpy</a> were embryonic lethal at 24°, suggesting that they did not contain <em><a href="http://www.wormbase.org/db/get?name=WBGene00001268;class=Gene">emb-14</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00145377;class=Variation">g43ts</a>)</em>. In addition, one of the <a href="http://www.wormbase.org/db/get?name=WBPhenotype%3A0000643;class=Phenotype">Unc</a> non-<a href="http://www.wormbase.org/db/get?name=WBPhenotype%3A0000583;class=Phenotype">Dpy</a> (not embryonic lethal) carried the <em><a href="http://www.wormbase.org/db/get?name=WBGene00019247;class=Gene">syp-4</a></em> mutation (<a href="http://www.g3journal.org/content/suppl/2015/03/04/g3.115.017038.DC1/017038SI.pdf">Figure S1</a>B). These genetic data combined with the sequencing data supports the conclusion that <em><a href="http://www.wormbase.org/db/get?name=WBGene00001268;class=Gene">emb-14</a></em> is allelic with the <em><a href="http://www.wormbase.org/db/get?name=WBGene00009342;class=Gene">fasn-1</a></em> gene. <em><a href="http://www.wormbase.org/db/get?name=WBGene00009342;class=Gene">fasn-1</a></em> is the <em>C. elegans</em> fatty acid synthase, which catalyzes the conversion of malonyl CoA into palmitate in the fatty acid biosynthetic pathway. Depletion of maternal <em><a href="http://www.wormbase.org/db/get?name=WBGene00009342;class=Gene">fasn-1</a></em> by RNAi causes a loss of anterior-posterior polarity in the progeny and osmotic sensitivity (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Rappleye <em>et al.</em> 2003</a>). A previously identified allele, <em><a href="http://www.wormbase.org/db/get?name=WBGene00009342;class=Gene">fasn-1</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00317254;class=Variation">fr8</a>)</em>, also has been found to play a role in the innate immune response (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Lee <em>et al.</em> 2010</a>). <em>The <a href="http://www.wormbase.org/db/get?name=WBGene00001268;class=Gene">emb-14</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00145377;class=Variation">g43ts</a>)</em> mutation changes Ala1425 to Thr in a conserved beta-ketoacyl reductase domain, which is needed for reduction of the growing fatty acid chain.</p><div id="T7" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466026/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466026/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 7</span> <span class="caption-title"><em>emb-14(g43ts)</em> is an allele of <em>fasn-1</em></span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h3><em>emb-29(g52ts)</em> is an allele of <em>cdc-25.2</em> and is rescued by CRISPR gene editing</h3><p id="p-41">Previous characterization placed <em><a href="http://www.wormbase.org/db/get?name=WBGene00001283;class=Gene">emb-29</a></em> on chromosome V at −18.34 cM and another group previously implicated <em><a href="http://www.wormbase.org/db/get?name=WBGene00001283;class=Gene">emb-29</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00145383;class=Variation">g52ts</a>)</em> as a complex allele of <em><a href="http://www.wormbase.org/db/get?name=WBGene00000387;class=Gene">cdc-25.2</a></em> (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Nair <em>et al.</em> 2013</a>). Using a different method of WGS, <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Nair <em>et al.</em> (2013)</a> found a Ser239 to Phe change in both <em><a href="http://www.wormbase.org/db/get?name=WBGene00001283;class=Gene">emb-29</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00145383;class=Variation">g52ts</a>)</em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00001283;class=Gene">emb-29</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00000379;class=Variation">b262</a>)</em>. However, no definitive proof such as rescue experiments has ever been reported. Using the method of WGS/SNP mapping, we found a reduction of Hawaiian SNPs in a 1.3-Mb interval on chromosome V, consistent with previous characterization. Within this interval there were two nonsynonymous sequence variants, <em><a href="http://www.wormbase.org/db/get?name=WBGene00003550;class=Gene">nas-32</a></em> (Ala77 to Thr) and <em><a href="http://www.wormbase.org/db/get?name=WBGene00000387;class=Gene">cdc-25.2</a></em> (Ser239 to Phe) (<a id="xref-table-wrap-8-1" class="xref-table" href="#T8">Table 8</a>). Only <em><a href="http://www.wormbase.org/db/get?name=WBGene00000387;class=Gene">cdc-25.2</a></em> was reported to be embryonic lethal by RNAi, but we never observed embryonic lethality by RNAi feeding.</p><div id="T8" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465960/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465960/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 8</span> <span class="caption-title"><em>emb-29(g52ts)</em> is an allele of <em>cdc-25.2</em></span><div class="sb-div caption-clear"></div></div></div><p id="p-43">Further validation by complementation has proven difficult. A deletion allele of <em><a href="http://www.wormbase.org/db/get?name=WBGene00000387;class=Gene">cdc-25.2</a></em> is homozygous sterile (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kim <em>et al.</em> 2010</a>). We performed complementation tests by crossing homozygous <em><a href="http://www.wormbase.org/db/get?name=WBGene00001283;class=Gene">emb-29</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00145383;class=Variation">g52ts</a>)</em> males to <em><a href="http://www.wormbase.org/db/get?name=WBGene00000387;class=Gene">cdc-25.2</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00091883;class=Variation">ok597</a>∆)/<a href="http://www.wormbase.org/db/get?name=hT2;class=Rearrangement">hT2</a></em> hermaphrodites and analyzed single non-green <em>trans</em>-heterozygous progeny for the number of dead F2 embryos <em>vs.</em> live F2 progeny. Similar to <a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Nair <em>et al.</em> (2013)</a>, we found that at 24° only ∼21% of progeny from <em><a href="http://www.wormbase.org/db/get?name=WBGene00001283;class=Gene">emb-29</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00145383;class=Variation">g52ts</a>)/<a href="http://www.wormbase.org/db/get?name=WBGene00000388;class=Gene">cdc-25.2</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00091883;class=Variation">ok597</a>∆)</em> F1s die as embryos but 91% of the surviving progeny are sterile (<a id="xref-table-wrap-8-2" class="xref-table" href="#T8">Table 8</a>). These data suggest that <em><a href="http://www.wormbase.org/db/get?name=WBGene00001283;class=Gene">emb-29</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00145383;class=Variation">g52ts</a>)/<a href="http://www.wormbase.org/db/get?name=WBGene00000387;class=Gene">cdc-25.2</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00091883;class=Variation">ok597</a>∆)</em> complements the embryonic lethal phenotype of <em><a href="http://www.wormbase.org/db/get?name=WBGene00001283;class=Gene">emb-29</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00145383;class=Variation">g52ts</a>)</em> but fails to complement the sterility of <em><a href="http://www.wormbase.org/db/get?name=WBGene00000387;class=Gene">cdc-25.2</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00091883;class=Variation">ok597</a>∆)</em>.</p><p id="p-44">Because there appears to be a complex genetic interaction between these two alleles making the complementation test results ambiguous, we rescued <em><a href="http://www.wormbase.org/db/get?name=WBGene00001283;class=Gene">emb-29</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00145383;class=Variation">g52ts</a>)</em> using the CRISPR/Cas9 gene editing system. <em><a href="http://www.wormbase.org/db/get?name=WBGene00001283;class=Gene">emb-29</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00145383;class=Variation">g52ts</a>)</em> was rescued by a single nucleotide conversion at nucleotide 716 of <em><a href="http://www.wormbase.org/db/get?name=WBGene00000387;class=Gene">cdc-25.2</a></em> with CRISPR/Cas9 gene editing and oligonucleotide-mediated recombination. We recovered three independent rescued lines. While embryonic viability of <em><a href="http://www.wormbase.org/db/get?name=WBGene00001283;class=Gene">emb-29</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00145383;class=Variation">g52ts</a>)</em> was 0% (1039 embryos) at 24°, the single-nucleotide change (T→C) led to an average of 99% hatching (2511 embryos) among the three rescued lines at 24°, similar to <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> (99.8% hatching; 1618 embryos, <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). To further validate that this single nucleotide change in <em><a href="http://www.wormbase.org/db/get?name=WBGene00000387;class=Gene">cdc-25.2</a></em> is the cause of the <em><a href="http://www.wormbase.org/db/get?name=WBGene00001283;class=Gene">emb-29</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00145383;class=Variation">g52</a>)</em> temperature-sensitive embryonic lethal phenotype, we edited the genome of wild-type (<a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a>) animals to recreate the <em><a href="http://www.wormbase.org/db/get?name=WBGene00001283;class=Gene">emb-29</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00145383;class=Variation">g52ts</a>)</em> allele. Although 99.8% (1618 embryos) of <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> progeny hatched at 24°, the percentage hatching of the recreated <em><a href="http://www.wormbase.org/db/get?name=WBGene00001283;class=Gene">emb-29</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00145383;class=Variation">g52ts</a>)</em> mutation (C→T) in three independent lines drastically decreased hatching to 0% (1094 embryos) at this temperature, similar to <em><a href="http://www.wormbase.org/db/get?name=WBGene00001283;class=Gene">emb-29</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00145383;class=Variation">g52ts</a>)</em> (0%; 1039 embryos, <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>). The rescue of <em><a href="http://www.wormbase.org/db/get?name=WBGene00001283;class=Gene">emb-29</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00145383;class=Variation">g52ts</a>)</em> and recreation of the temperature sensitivity by changing only a single nucleotide confirms that embryonic lethality in this line is in fact caused by the C716→T nucleotide change in <em><a href="http://www.wormbase.org/db/get?name=WBGene00000387;class=Gene">cdc-25.2</a></em>. <em><a href="http://www.wormbase.org/db/get?name=WBGene00000387;class=Gene">cdc-25.2</a></em> codes for one of four protein phosphatases (<em><a href="http://www.wormbase.org/db/get?name=WBGene00000386;class=Gene">cdc-25.1</a>-<a href="http://www.wormbase.org/db/get?name=WBGene00000389;class=Gene">cdc-25.4</a></em>) that promote cell cycle progression (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Ashcroft <em>et al.</em> 1998</a>). The Ser239 to Phe change of <em><a href="http://www.wormbase.org/db/get?name=WBGene00001283;class=Gene">emb-29</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00145383;class=Variation">g52ts</a>)</em> falls within a semi-conserved Rhodanese homology domain (amino acids 223-341). Members of the Rhodanese homology domain superfamily activate cell division kinases throughout the cell cycle.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/5/1007/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="emb-29(g52ts) embryonic lethality is caused by a point mutation in cdc-25.2. Percent embryonic viability for control lines N2 (wild type) and emb-29(g52ts) and of converted lines emb-29(g52ts) to emb-29(g52av38[T716C])(rescued) and N2 to cdc-25.2(av40[C716T])(recreated). *P &lt; 0.0001. Error bars indicate standard deviation. N = total number of embryos and hatched larvae scored. At least 860 embryos were scored for each genotype at each temperature." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1185382378" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;emb-29(g52ts)&lt;/em&gt; embryonic lethality is caused by a point mutation in &lt;em&gt;cdc-25.2&lt;/em&gt;. Percent embryonic viability for control lines N2 (wild type) and &lt;em&gt;emb-29(g52ts)&lt;/em&gt; and of converted lines &lt;em&gt;emb-29(g52ts)&lt;/em&gt; to &lt;em&gt;emb-29(g52av38[T716C])&lt;/em&gt;(rescued) and N2 to &lt;em&gt;cdc-25.2(av40[C716T])&lt;/em&gt;(recreated). *&lt;em&gt;P&lt;/em&gt; &lt; 0.0001. Error bars indicate standard deviation. N = total number of embryos and hatched larvae scored. At least 860 embryos were scored for each genotype at each temperature.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/5/1007/F6.medium.gif" width="440" height="336"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/5/5/1007/F6.medium.gif" width="440" height="336"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/5/1007/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/5/1007/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466058" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-45" class="first-child"><em><a href="http://www.wormbase.org/db/get?name=WBGene00001283;class=Gene">emb-29</a>(g52ts)</em> embryonic lethality is caused by a point mutation in <em>cdc-25.2</em>. Percent embryonic viability for control lines N2 (wild type) and <em><a href="http://www.wormbase.org/db/get?name=WBGene00001283;class=Gene">emb-29</a>(g52ts)</em> and of converted lines <em><a href="http://www.wormbase.org/db/get?name=WBGene00001283;class=Gene">emb-29</a>(g52ts)</em> to <em><a href="http://www.wormbase.org/db/get?name=WBGene00001283;class=Gene">emb-29</a>(g52av38[T716C])</em>(rescued) and N2 to <em>cdc-25.2(av40[C716T])</em>(recreated). *<em>P</em> &lt; 0.0001. Error bars indicate standard deviation. N = total number of embryos and hatched larvae scored. At least 860 embryos were scored for each genotype at each temperature.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-18"><h2 class="">Discussion</h2><p id="p-46">Generating and characterizing an allelic series of mutants defective in a particular developmental process is a common strategy to understand how genes control that particular process. From classical work in phage and yeast, this approach has helped define complex biological processes from metabolic pathways to cell cycle division (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Hartwell <em>et al.</em> 1974</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Edgar <em>et al.</em> 1964</a>). Pioneering work in the nematode, <em>C. elegans</em>, also applied a genetic approach to define nervous system development and behavior (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Brenner 1973</a>). In early attempts to understand the genetic control of embryonic development, many groups isolated thousands of mutants with an embryonic lethal phenotype (<a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">Hirsh and Vanderslice 1976</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Vanderslice and Hirsh 1976</a>; <a id="xref-ref-30-5" class="xref-bibr" href="#ref-30">Miwa <em>et al.</em> 1980</a>; <a id="xref-ref-37-5" class="xref-bibr" href="#ref-37">Schierenberg <em>et al.</em> 1980</a>; <a id="xref-ref-45-7" class="xref-bibr" href="#ref-45">Wood <em>et al.</em> 1980</a>; <a id="xref-ref-4-10" class="xref-bibr" href="#ref-4">Cassada <em>et al.</em> 1981</a>). Many of these legacy mutants have only been partially characterized and in many cases the causative molecular lesion remains unknown.</p><p id="p-47">The traditional method for the identification of the phenotype-causing mutation is a long and laborious process. Mutagenized strains contain a substantial number of sequence variants. Even with the identification of sequence variants by WGS, it is often necessary to first map the mutation to a chromosomal interval via three-point mapping or a SNP mapping strategy. Both of these strategies require reiterative mapping to narrow down the relevant interval, which is a time-consuming process. We applied the WGS/SNP mapping strategy originally devised by <a id="xref-ref-8-5" class="xref-bibr" href="#ref-8">Doitsidou <em>et al.</em> (2010)</a> to identify the molecular lesions responsible for the embryonic lethal phenotype of mutants isolated 30 years ago. In this work, we have demonstrated that this is a relatively fast and efficient method to map and identify the molecular lesion of legacy mutants. The entire process, from the first outcross with Hawaiian males, to the picking of F2 progeny, making the library, and WGS can take place in 3−4 wk. The list of candidates can be generated in as little as 2 d. The complementation tests take about 7−10 d once all necessary strains have been obtained. The total protocol can be accomplished in less than 2 mo. We applied the WGS/SNP mapping strategy to seven legacy mutants and found that all seven of the mutants carried a single-point mutation resulting in a nonsynonymous amino acid change that caused the mutant phenotype, six of which were temperature-sensitive.</p><p id="p-48">A recent paper used a non-SNP−based method of WGS to identify the molecular lesion of 64 mutants of essential genes on chromosome I (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Chu <em>et al.</em> 2014</a>). Although five of the mutants we identified had mutations in genes on chromosome I, none of these were identified in the work by <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Chu <em>et al.</em> (2014)</a>. In this work, the newly identified alleles corresponded to genes that have been studied by other groups ranging in processes from cell cycle regulators to components of the fatty-acid biosynthetic pathway. Many of the newly identified mutations are either in functional domains or in conserved regions of the corresponding protein. Because they are essential genes, mutations result in embryonic lethality precluding extensive characterization. With the identification of temperature-sensitive alleles, further studies are possible simply by changing the growth temperature.</p><p id="p-49">In summary, we demonstrated that the WGS/SNP-based mapping strategy can be used to identify the unknown molecular lesions from the hundreds of legacy mutants that were isolated from early studies of <em>C. elegans</em> development, and that ts mutants are ideal targets for this approach. We have identified additional alleles of known genes and alleles of essential genes, which, previously, could only be studied by RNAi. All but one of these alleles is temperature-sensitive. We believe that these new temperature-sensitive alleles of essential genes are a valuable resource for the further investigation of essential developmental processes.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-50">We thank S. Kim for generating and analyzing <em><a href="http://www.wormbase.org/db/get?name=WBGene00001268;class=Gene">emb-14</a></em> recombinants and comments regarding the manuscript. We also thank A. Walters and K. O’Connell for their advice and comments on the manuscript. This research was supported by the Intramural Research Program of the National Institutes of Health (NIH), National Institute of Diabetes and Digestive and Kidney Diseases. Some nematode strains used in this work were provided by the Caenorhabditis Genetics Center, which is funded by NIH Office of Research Infrastructure Programs (P40 OD010440). Some deletion allele strains were provided by the National Bioresource Project in Japan lead by S. Mitani.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-10"><p id="p-51">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017038/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017038/-/DC1</a></p></li><li class="fn" id="fn-11"><p id="p-52"><em>Communicating editor: D. S. Fay</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-01-29"><span class="received-label">Received </span>January 29, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-02-27"><span class="accepted-label">Accepted </span>February 27, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Jaramillo-Lambert <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.1007.1" data-doi="10.1534/genetics.114.169730"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arribere</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">R. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">B. X. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Artiles</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartman</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Efficient marker-free recovery of custom genetic modifications with CRISPR/Cas9 in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">837</span>–<span class="cit-lpage">846</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.169730%26rft_id%253Dinfo%253Apmid%252F25161212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzMvODM3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS81LzEwMDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.1007.2" data-doi="10.1016/S0378-1119(98)00228-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ashcroft</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kosinski</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wickramasinghe</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donovan</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Golden</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">The four cdc25 genes from the nematode <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">214</span>: <span class="cit-fpage">59</span>–<span class="cit-lpage">66</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DAshcroft%26rft.auinit1%253DN.%2BR.%26rft.volume%253D214%26rft.issue%253D1-2%26rft.spage%253D59%26rft.epage%253D66%26rft.atitle%253DThe%2Bfour%2Bcdc25%2Bgenes%2Bfrom%2Bthe%2Bnematode%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0378-1119%252898%252900228-5%26rft_id%253Dinfo%253Apmid%252F9651482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0378-1119(98)00228-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9651482&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F1007.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000075318400008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.1007.3" data-doi="10.1093/oxfordjournals.bmb.a071019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brenner</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1973</span> <span class="cit-article-title">The genetics of behaviour.</span> <abbr class="cit-jnl-abbrev">Br. Med. Bull.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">269</span>–<span class="cit-lpage">271</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBritish%2BMedical%2BBulletin%26rft.stitle%253DBr%2BMed%2BBull%26rft.aulast%253DBRENNER%26rft.auinit1%253DS.%26rft.volume%253D29%26rft.issue%253D3%26rft.spage%253D269%26rft.epage%253D271%26rft.atitle%253DTHE%2BGENETICS%2BOF%2BBEHAVIOUR%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.bmb.a071019%26rft_id%253Dinfo%253Apmid%252F4807330%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/oxfordjournals.bmb.a071019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4807330&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F1007.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1973Q590500016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.1007.4" data-doi="10.1016/0012-1606(81)90383-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cassada</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Isnenghi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Culotti</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">von Ehrenstein</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1981</span> <span class="cit-article-title">Genetic analysis of temperature-sensitive embryogenesis mutants in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">193</span>–<span class="cit-lpage">205</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DCassada%26rft.auinit1%253DR.%26rft.volume%253D84%26rft.issue%253D1%26rft.spage%253D193%26rft.epage%253D205%26rft.atitle%253DGenetic%2Banalysis%2Bof%2Btemperature-sensitive%2Bembryogenesis%2Bmutants%2Bin%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0012-1606%252881%252990383-3%26rft_id%253Dinfo%253Apmid%252F7250492%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0012-1606(81)90383-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7250492&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F1007.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1981LS36300018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.1007.5" data-doi="10.1186/1471-2164-15-361"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chu</span>  <span class="cit-name-given-names">J. S.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chua</span>  <span class="cit-name-given-names">S.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davison</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnsen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">High-throughput capturing and characterization of mutations in essential genes of <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">361</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D361%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-361%26rft_id%253Dinfo%253Apmid%252F24884423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-361&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24884423&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F1007.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-5.5.1007.6" data-doi="10.1007/BF00848892"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Denich</span>  <span class="cit-name-given-names">K. T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schierenberg</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Isnenghi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cassada</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Cell-lineage and developmental defects of temperature-sensitive embryonic arrest mutants of the nematode <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Roux’s Arch. Dev. Biol.</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">164</span>–<span class="cit-lpage">179</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRoux%253Fs%2BArch.%2BDev.%2BBiol.%26rft.volume%253D193%26rft.spage%253D164%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00848892%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00848892&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.1007.7" data-doi="10.1038/nmeth.2641"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dickinson</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ward</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reiner</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldstein</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Engineering the <em>Caenorhabditis elegans</em> genome using Cas9-triggered homologous recombination.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1028</span>–<span class="cit-lpage">1034</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D10%26rft.spage%253D1028%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2641%26rft_id%253Dinfo%253Apmid%252F23995389%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2641&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23995389&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F1007.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000325073800029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-book" id="cit-5.5.1007.8" data-doi="10.1371/journal.pone.0015435"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doitsidou</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poole</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sarin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bigelow</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hobert</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title"><em>C. elegans</em> mutant identification with a one-step whole-genome-sequencing and SNP mapping strategy.</span> <span class="cit-source">PLoS One</span> <span class="cit-vol">5</span>: <span class="cit-fpage">e15435</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DDoitsidou%26rft.auinit1%253DM.%26rft.volume%253D5%26rft.issue%253D11%26rft.spage%253De15435%26rft.epage%253De15435%26rft.atitle%253DC.%2Belegans%2Bmutant%2Bidentification%2Bwith%2Ba%2Bone-step%2Bwhole-genome-sequencing%2Band%2BSNP%2Bmapping%2Bstrategy.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0015435%26rft_id%253Dinfo%253Apmid%252F21079745%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0015435&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21079745&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F1007.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.1007.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edgar</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lielausis</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">1964</span> <span class="cit-article-title">Tempurature-sensitive mutants of bacteriophage T4D: their isolation and genetic characterization.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">649</span>–<span class="cit-lpage">662</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DEdgar%26rft.auinit1%253DR.%2BS.%26rft.volume%253D49%26rft.issue%253D4%26rft.spage%253D649%26rft.epage%253D662%26rft.atitle%253DTEMPERATURE-SENSITIVE%2BMUTANTS%2BOF%2BBACTERIOPHAGE%2BT4D%253A%2BTHEIR%2BISOLATION%2BAND%2BGENETIC%2BCHARACTERIZATION%26rft_id%253Dinfo%253Apmid%252F14156925%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiI0OS80LzY0OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNS8xMDA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.1007.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edgar</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Denhardt</span>  <span class="cit-name-given-names">G. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Epstein</span>  <span class="cit-name-given-names">R. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1964</span> <span class="cit-article-title">A comparative genetic study of conditional lethal mutations of bacteriophage T4D.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">635</span>–<span class="cit-lpage">648</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DEdgar%26rft.auinit1%253DR.%2BS.%26rft.volume%253D49%26rft.issue%253D4%26rft.spage%253D635%26rft.epage%253D648%26rft.atitle%253DA%2BCOMPARATIVE%2BGENETIC%2BSTUDY%2BOF%2BCONDITIONAL%2BLETHAL%2BMUTATIONS%2BOF%2BBACTERIOPHAGE%2BT4D%26rft_id%253Dinfo%253Apmid%252F14156924%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiI0OS80LzYzNSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNS8xMDA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.1007.11" data-doi="10.1126/science.183.4120.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hartwell</span>  <span class="cit-name-given-names">L. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Culotti</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pringle</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reid</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">Genetic control of the cell division cycle in yeast.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">183</span>: <span class="cit-fpage">46</span>–<span class="cit-lpage">51</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DHartwell%26rft.auinit1%253DL.%2BH.%26rft.volume%253D183%26rft.issue%253D4120%26rft.spage%253D46%26rft.epage%253D51%26rft.atitle%253DGenetic%2BControl%2Bof%2Bthe%2BCell%2BDivision%2BCycle%2Bin%2BYeast%253A%2BA%2Bmodel%2Bto%2Baccount%2Bfor%2Bthe%2Border%2Bof%2Bcell%2Bcycle%2Bevents%2Bis%2Bdeduced%2Bfrom%2Bthe%2Bphenotypes%2Bof%2Byeast%2Bmutants%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.183.4120.46%26rft_id%253Dinfo%253Apmid%252F4587263%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czozOiJzY2kiO3M6NToicmVzaWQiO3M6MTE6IjE4My80MTIwLzQ2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS81LzEwMDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-5.5.1007.12" data-doi="10.1016/0012-1606(82)90081-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hecht</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wall</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schomer</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oró</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bartel</span>  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title">DNA replication may be uncoupled from nuclear and cellular division in temperature-sensitive embryonic lethal mutants of <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">183</span>–<span class="cit-lpage">191</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DHecht%26rft.auinit1%253DR.%2BM.%26rft.volume%253D94%26rft.issue%253D1%26rft.spage%253D183%26rft.epage%253D191%26rft.atitle%253DDNA%2Breplication%2Bmay%2Bbe%2Buncoupled%2Bfrom%2Bnuclear%2Band%2Bcellular%2Bdivision%2Bin%2Btemperature-sensitive%2Bembryonic%2Blethal%2Bmutants%2Bof%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0012-1606%252882%252990081-1%26rft_id%253Dinfo%253Apmid%252F7152102%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0012-1606(82)90081-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7152102&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F1007.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.1007.13" data-doi="10.1016/0022-2836(74)90451-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hereford</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartwell</span>  <span class="cit-name-given-names">L. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">Sequential gene function in the initiation of Saccharomyces cerevisiae DNA synthesis.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">445</span>–<span class="cit-lpage">461</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DHereford%26rft.auinit1%253DL.%2BM.%26rft.volume%253D84%26rft.issue%253D3%26rft.spage%253D445%26rft.epage%253D461%26rft.atitle%253DSequential%2Bgene%2Bfunction%2Bin%2Bthe%2Binitiation%2Bof%2BSaccharomyces%2Bcerevisiae%2BDNA%2Bsynthesis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2836%252874%252990451-3%26rft_id%253Dinfo%253Apmid%252F4618856%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-2836(74)90451-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4618856&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F1007.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1974S756200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-5.5.1007.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Herman</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horvitz</span>  <span class="cit-name-given-names">H. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riddle</span>  <span class="cit-name-given-names">D. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">The nematode <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetic Maps</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">183</span>–<span class="cit-lpage">193</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2BMaps%26rft.volume%253D1%26rft.spage%253D183%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.1007.15" data-doi="10.1016/0012-1606(76)90268-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hirsh</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vanderslice</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1976</span> <span class="cit-article-title">Temperature-sensitive developmental mutants of <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">220</span>–<span class="cit-lpage">235</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DHirsh%26rft.auinit1%253DD.%26rft.volume%253D49%26rft.issue%253D1%26rft.spage%253D220%26rft.epage%253D235%26rft.atitle%253DTemperature-sensitive%2Bdevelopmental%2Bmutants%2Bof%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0012-1606%252876%252990268-2%26rft_id%253Dinfo%253Apmid%252F943345%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0012-1606(76)90268-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=943345&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F1007.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1976BH96500016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.1007.16" data-doi="10.1534/genetics.104.036137"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holway</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hung</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michael</span>  <span class="cit-name-given-names">W. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Systematic, RNA-interference-mediated identification of <em>mus-101</em> modifier genes in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">169</span>: <span class="cit-fpage">1451</span>–<span class="cit-lpage">1460</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.036137%26rft_id%253Dinfo%253Apmid%252F15654100%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2OS8zLzE0NTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzUvMTAwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-book" id="cit-5.5.1007.17" data-doi="10.1371/journal.pone.0007767"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Homer</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merriman</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">S. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">BFAST: an alignment tool for large scale genome resequencing.</span> <span class="cit-source">PLoS One</span> <span class="cit-vol">4</span>: <span class="cit-fpage">e7767</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DHomer%26rft.auinit1%253DN.%26rft.volume%253D4%26rft.issue%253D11%26rft.spage%253De7767%26rft.epage%253De7767%26rft.atitle%253DBFAST%253A%2Ban%2Balignment%2Btool%2Bfor%2Blarge%2Bscale%2Bgenome%2Bresequencing.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0007767%26rft_id%253Dinfo%253Apmid%252F19907642%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0007767&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19907642&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F1007.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.1007.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnsen</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baillie</span>  <span class="cit-name-given-names">D. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Genetic analysis of a major segment [LGV(left)] of the genome of <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">735</span>–<span class="cit-lpage">752</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DJohnsen%26rft.auinit1%253DR.%2BC.%26rft.volume%253D129%26rft.issue%253D3%26rft.spage%253D735%26rft.epage%253D752%26rft.atitle%253DGenetic%2Banalysis%2Bof%2Ba%2Bmajor%2Bsegment%2B%255BLGV%2528left%2529%255D%2Bof%2Bthe%2Bgenome%2Bof%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Apmid%252F1752418%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTI5LzMvNzM1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS81LzEwMDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.1007.19" data-doi="10.1007/s004380051165"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnsen</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rose</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Mutational accessibility of essential genes on chromosome I(left) in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Mol. General Genet.</abbr> <span class="cit-vol">263</span>: <span class="cit-fpage">239</span>–<span class="cit-lpage">252</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BGeneral%2BGenet.%26rft.volume%253D263%26rft.spage%253D239%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs004380051165%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s004380051165&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.1007.20" data-doi="10.1016/S1046-2023(03)00050-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kamath</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ahringer</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Genome-wide RNAi screening in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Methods</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">313</span>–<span class="cit-lpage">321</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2B%2528San%2BDiego%252C%2BCalif.%2529%26rft.stitle%253DMethods%26rft.aulast%253DKamath%26rft.auinit1%253DR.%2BS.%26rft.volume%253D30%26rft.issue%253D4%26rft.spage%253D313%26rft.epage%253D321%26rft.atitle%253DGenome-wide%2BRNAi%2Bscreening%2Bin%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1046-2023%252803%252900050-1%26rft_id%253Dinfo%253Apmid%252F12828945%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1046-2023(03)00050-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12828945&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F1007.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000184116700005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.1007.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kemphues</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kusch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolf</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Maternal-effect lethal mutations on linkage group II of <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">120</span>: <span class="cit-fpage">977</span>–<span class="cit-lpage">986</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKemphues%26rft.auinit1%253DK.%2BJ.%26rft.volume%253D120%26rft.issue%253D4%26rft.spage%253D977%26rft.epage%253D986%26rft.atitle%253DMaternal-effect%2Blethal%2Bmutations%2Bon%2Blinkage%2Bgroup%2BII%2Bof%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Apmid%252F3224814%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTIwLzQvOTc3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS81LzEwMDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.1007.22" data-doi="10.1534/genetics.114.166389"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ishidate</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ghanta</span>  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seth</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conte</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A Co-CRISPR strategy for efficient genome editing in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">197</span>: <span class="cit-fpage">1069</span>–<span class="cit-lpage">1080</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.166389%26rft_id%253Dinfo%253Apmid%252F24879462%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Ny80LzEwNjkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzUvMTAwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.1007.23" data-doi="10.1242/jcs.060442"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawasaki</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shim</span>  <span class="cit-name-given-names">Y.-H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title"><em>cdc-25.2</em>, a <em>C. elegans</em> ortholog of <em>cdc25</em>, is required to promote oocyte maturation.</span> <abbr class="cit-jnl-abbrev">J. Cell Sci.</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">993</span>–<span class="cit-lpage">1000</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BCell%2BScience%26rft.stitle%253DJ.%2BCell%2BSci.%26rft.aulast%253DKim%26rft.auinit1%253DJ.%26rft.volume%253D123%26rft.issue%253D6%26rft.spage%253D993%26rft.epage%253D1000%26rft.atitle%253Dcdc-25.2%252C%2Ba%2BC.%2Belegans%2Bortholog%2Bof%2Bcdc25%252C%2Bis%2Brequired%2Bto%2Bpromote%2Boocyte%2Bmaturation%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fjcs.060442%26rft_id%253Dinfo%253Apmid%252F20200231%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToiam9jZXMiO3M6NToicmVzaWQiO3M6OToiMTIzLzYvOTkzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS81LzEwMDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.1007.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">D.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shim</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Thymidylate synthase and dihydropyrimidine dehydrogenase levels are associated with response to 5-fluorouracil in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Cells</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">344</span>–<span class="cit-lpage">349</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecules%2Band%2Bcells%26rft.stitle%253DMol%2BCells%26rft.aulast%253DKim%26rft.auinit1%253DS.%26rft.volume%253D26%26rft.issue%253D4%26rft.spage%253D344%26rft.epage%253D349%26rft.atitle%253DThymidylate%2Bsynthase%2Band%2Bdihydropyrimidine%2Bdehydrogenase%2Blevels%2Bare%2Bassociated%2Bwith%2Bresponse%2Bto%2B5-fluorouracil%2Bin%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Apmid%252F18612238%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18612238&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F1007.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.1007.25" data-doi="10.4161/viru.1.3.10974"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">K.-Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kniazeva</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pujol</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ewbank</span>  <span class="cit-name-given-names">J. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The fatty acid synthase fasn-1 acts upstream of WNK and Ste20/GCK-VI kinases to modulate antimicrobial peptide expression in <em>C. elegans</em> epidermis.</span> <abbr class="cit-jnl-abbrev">Virulence</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">113</span>–<span class="cit-lpage">122</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirulence%26rft.volume%253D1%26rft.spage%253D113%26rft_id%253Dinfo%253Adoi%252F10.4161%252Fviru.1.3.10974%26rft_id%253Dinfo%253Apmid%252F21178429%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/viru.1.3.10974&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21178429&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F1007.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.1007.26" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The Sequence Alignment/Map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505943&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F1007.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268808600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.1007.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meneely</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herman</span>  <span class="cit-name-given-names">R. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1981</span> <span class="cit-article-title">Suppression and function of X-linked lethal and sterile mutations in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">65</span>–<span class="cit-lpage">84</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeneely%26rft.auinit1%253DP.%2BM.%26rft.volume%253D97%26rft.issue%253D1%26rft.spage%253D65%26rft.epage%253D84%26rft.atitle%253DSUPPRESSION%2BAND%2BFUNCTION%2BOF%2BX-LINKED%2BLETHAL%2BAND%2BSTERILE%2BMUTATIONS%2BIN%2BCAENORHABDITIS%2BELEGANS%26rft_id%253Dinfo%253Apmid%252F7196363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6NzoiOTcvMS82NSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNS8xMDA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.1007.28" data-doi="10.1007/s00438-006-0191-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Minasaki</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Streit</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">MEL-47, a novel protein required for early cell divisions in the nematode <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Genet. Genomics</abbr> <span class="cit-vol">277</span>: <span class="cit-fpage">315</span>–<span class="cit-lpage">328</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bgenetics%2Band%2Bgenomics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGenet%2BGenomics%26rft.aulast%253DMinasaki%26rft.auinit1%253DR.%26rft.volume%253D277%26rft.issue%253D3%26rft.spage%253D315%26rft.epage%253D328%26rft.atitle%253DMEL-47%252C%2Ba%2Bnovel%2Bprotein%2Brequired%2Bfor%2Bearly%2Bcell%2Bdivisions%2Bin%2Bthe%2Bnematode%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00438-006-0191-z%26rft_id%253Dinfo%253Apmid%252F17171368%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00438-006-0191-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17171368&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F1007.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.1007.29" data-doi="10.1534/genetics.112.144204"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Minevich</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blankenberg</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poole</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hobert</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">CloudMap: a cloud-based pipeline for analysis of mutant genome sequences.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">1249</span>–<span class="cit-lpage">1269</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.144204%26rft_id%253Dinfo%253Apmid%252F23051646%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Mi80LzEyNDkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzUvMTAwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.1007.30" data-doi="10.1016/0012-1606(80)90369-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miwa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schierenberg</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miwa</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">von Ehrenstein</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">Genetics and mode of expression of temperature-sensitive mutations arresting embryonic development in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">160</span>–<span class="cit-lpage">174</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DMiwa%26rft.auinit1%253DJ.%26rft.volume%253D76%26rft.issue%253D1%26rft.spage%253D160%26rft.epage%253D174%26rft.atitle%253DGenetics%2Band%2Bmode%2Bof%2Bexpression%2Bof%2Btemperature-sensitive%2Bmutations%2Barresting%2Bembryonic%2Bdevelopment%2Bin%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0012-1606%252880%252990369-3%26rft_id%253Dinfo%253Apmid%252F7380089%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0012-1606(80)90369-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7380089&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F1007.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1980JL63000012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.1007.31" data-doi="10.1242/dev.098012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nair</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walton</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raj</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Gene transcription is coordinated with, but not dependent on, cell divisions during <em>C. elegans</em> embryonic fate specification.</span> <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">140</span>: <span class="cit-fpage">3385</span>–<span class="cit-lpage">3394</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fdev.098012%26rft_id%253Dinfo%253Apmid%252F23863485%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czoxMToiMTQwLzE2LzMzODUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzUvMTAwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.1007.32" data-doi="10.1016/j.cell.2011.11.053"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nakano</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stillman</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horvitz</span>  <span class="cit-name-given-names">H. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Replication-coupled chromatin assembly generates a neuronal bilateral asymmetry in <em>C. elegans</em>.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">147</span>: <span class="cit-fpage">1525</span>–<span class="cit-lpage">1536</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DNakano%26rft.auinit1%253DS.%26rft.volume%253D147%26rft.issue%253D7%26rft.spage%253D1525%26rft.epage%253D1536%26rft.atitle%253DReplication-coupled%2Bchromatin%2Bassembly%2Bgenerates%2Ba%2Bneuronal%2Bbilateral%2Basymmetry%2Bin%2BC.%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2011.11.053%26rft_id%253Dinfo%253Apmid%252F22177093%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2011.11.053&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22177093&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F1007.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298403400017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-other" id="cit-5.5.1007.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paix</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">C.-Y. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calidas</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Scalable and versatile genome editing using linear DNAs with micro-homology to Cas9 Sites in <em>Caenorhabditis elegans</em>.</span> <span class="cit-source">Genetics</span> <span class="cit-vol">114</span>: <span class="cit-fpage">170423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D114%26rft.spage%253D170423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-book" id="cit-5.5.1007.34"><div class="cit-metadata unstructured">R Development Core Team, 2014 R: A language and environment for statistical computing. Available at: <a href="http://www.r-project.org/">www.r-project.org/</a>. Accessed March 14, 2015.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.1007.35" data-doi="10.1186/1471-213X-3-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rappleye</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tagawa</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Le Bot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ahringer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aroian</span>  <span class="cit-name-given-names">R. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Involvement of fatty acid pathways and cortical interaction of the pronuclear complex in <em>Caenorhabditis elegans</em> embryonic polarity.</span> <abbr class="cit-jnl-abbrev">BMC Dev. Biol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bdevelopmental%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BDev%2BBiol%26rft.aulast%253DRappleye%26rft.auinit1%253DC.%2BA.%26rft.volume%253D3%26rft.spage%253D8%26rft.epage%253D8%26rft.atitle%253DInvolvement%2Bof%2Bfatty%2Bacid%2Bpathways%2Band%2Bcortical%2Binteraction%2Bof%2Bthe%2Bpronuclear%2Bcomplex%2Bin%2BCaenorhabditis%2Belegans%2Bembryonic%2Bpolarity.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-213X-3-8%26rft_id%253Dinfo%253Apmid%252F14527340%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-213X-3-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14527340&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F1007.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.1007.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rose</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baillie</span>  <span class="cit-name-given-names">D. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">Genetic organization of the region around UNC-15 (I), a gene affecting paramyosin in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">639</span>–<span class="cit-lpage">648</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DRose%26rft.auinit1%253DA.%2BM.%26rft.volume%253D96%26rft.issue%253D3%26rft.spage%253D639%26rft.epage%253D648%26rft.atitle%253DGENETIC%2BORGANIZATION%2BOF%2BTHE%2BREGION%2BAROUND%2BUNC-15%2B%2528I%2529%252C%2BA%2BGENE%2BAFFECTING%2BPARAMYOSIN%2BIN%2BCAENORHABDITIS%2BELEGANS%26rft_id%253Dinfo%253Apmid%252F7262541%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiOTYvMy82MzkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzUvMTAwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.1007.37" data-doi="10.1016/0012-1606(80)90368-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schierenberg</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miwa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">von Ehrenstein</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">Cell lineages and developmental defects of temperature-sensitive embryonic arrest mutants in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">141</span>–<span class="cit-lpage">159</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DSchierenberg%26rft.auinit1%253DE.%26rft.volume%253D76%26rft.issue%253D1%26rft.spage%253D141%26rft.epage%253D159%26rft.atitle%253DCell%2Blineages%2Band%2Bdevelopmental%2Bdefects%2Bof%2Btemperature-sensitive%2Bembryonic%2Barrest%2Bmutants%2Bin%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0012-1606%252880%252990368-1%26rft_id%253Dinfo%253Apmid%252F7380087%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0012-1606(80)90368-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7380087&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F1007.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.1007.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sigurdson</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spanier</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herman</span>  <span class="cit-name-given-names">R. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title"><em>Caenorhabditis elegans</em> deficiency mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">331</span>–<span class="cit-lpage">345</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSigurdson%26rft.auinit1%253DD.%2BC.%26rft.volume%253D108%26rft.issue%253D2%26rft.spage%253D331%26rft.epage%253D345%26rft.atitle%253DCAENORHABDITIS%2BELEGANS%2BDEFICIENCY%2BMAPPING%26rft_id%253Dinfo%253Apmid%252F6500256%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTA4LzIvMzMxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS81LzEwMDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.1007.39" data-doi="10.1016/0092-8674(89)90398-X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stillman</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Purification and characterization of CAF-I, a human cell factor required for chromatin assembly during DNA replication in vitro.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">15</span>–<span class="cit-lpage">25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DSmith%26rft.auinit1%253DS.%26rft.volume%253D58%26rft.issue%253D1%26rft.spage%253D15%26rft.epage%253D25%26rft.atitle%253DPurification%2Band%2Bcharacterization%2Bof%2BCAF-I%252C%2Ba%2Bhuman%2Bcell%2Bfactor%2Brequired%2Bfor%2Bchromatin%2Bassembly%2Bduring%2BDNA%2Breplication%2Bin%2Bvitro.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252889%252990398-X%26rft_id%253Dinfo%253Apmid%252F2546672%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(89)90398-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2546672&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F1007.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1989AG04600004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.1007.40" data-doi="10.1007/978-90-481-3471-7_7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sokka</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parkkinen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pospiech</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Syväoja</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Function of TopBP1 in genome stability.</span> <abbr class="cit-jnl-abbrev">Subcell. Biochem.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">119</span>–<span class="cit-lpage">141</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSub-cellular%2Bbiochemistry%26rft.stitle%253DSubcell%2BBiochem%26rft.aulast%253DSokka%26rft.auinit1%253DM.%26rft.volume%253D50%26rft.spage%253D119%26rft.epage%253D141%26rft.atitle%253DFunction%2Bof%2BTopBP1%2Bin%2Bgenome%2Bstability.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-90-481-3471-7_7%26rft_id%253Dinfo%253Apmid%252F20012580%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-90-481-3471-7_7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20012580&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F1007.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.1007.41" data-doi="10.1016/S0378-1119(00)00579-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Timmons</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Court</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fire</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Ingestion of bacterially expressed dsRNAs can produce specific and potent genetic interference in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">263</span>: <span class="cit-fpage">103</span>–<span class="cit-lpage">112</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DTimmons%26rft.auinit1%253DL.%26rft.volume%253D263%26rft.issue%253D1-2%26rft.spage%253D103%26rft.epage%253D112%26rft.atitle%253DIngestion%2Bof%2Bbacterially%2Bexpressed%2BdsRNAs%2Bcan%2Bproduce%2Bspecific%2Band%2Bpotent%2Bgenetic%2Binterference%2Bin%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0378-1119%252800%252900579-5%26rft_id%253Dinfo%253Apmid%252F11223248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0378-1119(00)00579-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11223248&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F1007.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000167352400011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.1007.42" data-doi="10.1016/0012-1606(76)90269-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vanderslice</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirsh</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1976</span> <span class="cit-article-title">Temperature-sensitive zygote defective mutants of <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">236</span>–<span class="cit-lpage">249</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DVanderslice%26rft.auinit1%253DR.%26rft.volume%253D49%26rft.issue%253D1%26rft.spage%253D236%26rft.epage%253D249%26rft.atitle%253DTemperature-sensitive%2Bzygote%2Bdefective%2Bmutants%2Bof%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0012-1606%252876%252990269-4%26rft_id%253Dinfo%253Apmid%252F943346%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0012-1606(76)90269-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=943346&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F1007.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.1007.43" data-doi="10.1093/nar/gkq603"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hakonarson</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">ANNOVAR: functional annotation of genetic variants from high-throughput sequencing data.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">e164</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq603%26rft_id%253Dinfo%253Apmid%252F20601685%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkq603&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20601685&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F1007.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-book" id="cit-5.5.1007.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rasoloson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stitzel</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Connell</span>  <span class="cit-name-given-names">K. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Identification of suppressors of <em>mbk-2/DYRK</em> by whole-genome sequencing.</span> G3 (Bethesda) <span class="cit-vol">4</span>: <span class="cit-fpage">231</span>–<span class="cit-lpage">241</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.1007.45" data-doi="10.1016/0012-1606(80)90445-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wood</span>  <span class="cit-name-given-names">W. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hecht</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carr</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vanderslice</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolf</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">Parental effects and phenotypic characterization of mutations that affect early development in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">446</span>–<span class="cit-lpage">469</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DWood%26rft.auinit1%253DW.%2BB.%26rft.volume%253D74%26rft.issue%253D2%26rft.spage%253D446%26rft.epage%253D469%26rft.atitle%253DParental%2Beffects%2Band%2Bphenotypic%2Bcharacterization%2Bof%2Bmutations%2Bthat%2Baffect%2Bearly%2Bdevelopment%2Bin%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0012-1606%252880%252990445-5%26rft_id%253Dinfo%253Apmid%252F7371984%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0012-1606(80)90445-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7371984&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F1007.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.1007.46" data-doi="10.1007/BF00485755"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shaw</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span> <span class="cit-article-title">Time of expression of genes controlling specific enzymes in <em>Drosophila</em> embryos.</span> <abbr class="cit-jnl-abbrev">Biochem. Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">385</span>–<span class="cit-lpage">394</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemical%2Bgenetics%26rft.stitle%253DBiochem%2BGenet%26rft.aulast%253DWright%26rft.auinit1%253DD.%2BA.%26rft.volume%253D4%26rft.issue%253D3%26rft.spage%253D385%26rft.epage%253D394%26rft.atitle%253DTime%2Bof%2Bexpression%2Bof%2Bgenes%2Bcontrolling%2Bspecific%2Benzymes%2Bin%2BDrosophila%2Bembryos.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00485755%26rft_id%253Dinfo%253Apmid%252F5477232%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00485755&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=5477232&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F1007.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1970G366500004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.1007.47" data-doi="10.1016/S0065-2660(08)60075-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">T. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span> <span class="cit-article-title">The genetics of embryogenesis in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Adv. Genet.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">261</span>–<span class="cit-lpage">395</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdvances%2Bin%2Bgenetics%26rft.stitle%253DAdv%2BGenet%26rft.aulast%253DWright%26rft.auinit1%253DT.%2BR.%26rft.volume%253D15%26rft.spage%253D261%26rft.epage%253D395%26rft.atitle%253DThe%2Bgenetics%2Bof%2Bembryogenesis%2Bin%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0065-2660%252808%252960075-9%26rft_id%253Dinfo%253Apmid%252F4936425%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0065-2660(08)60075-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4936425&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F1007.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1970I371100006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.1007.48" data-doi="10.1038/cr.2014.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ke</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yue</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xue</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Oligonucleotide-based targeted gene editing in <em>C. elegans</em> via the CRISPR/Cas9 system.</span> <abbr class="cit-jnl-abbrev">Cell Res.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">247</span>–<span class="cit-lpage">250</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BRes.%26rft.volume%253D24%26rft.spage%253D247%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fcr.2014.9%26rft_id%253Dinfo%253Apmid%252F24418757%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/cr.2014.9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24418757&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F1007.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/5/1007.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/5/997" title="Leveraging DNA Damage Response Signaling to Identify Yeast Genes Controlling Genome Stability" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/6/1021" title="Meeting Report on Experimental Approaches to Evolution and Ecology Using Yeast and Other Model Systems" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/5"><span>Volume 5 Issue 5, May 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/5.cover-source.jpg?itok=53KRlKz0" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/5/1007" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-mlABywpqs-LsnACYZ5WaBMMFXn-PJkMsjfkFgLrWGPw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/mutant-screen-report" class="" data-icon-position="" data-hide-link-title="0">Mutant Screen Report</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/5/1007" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/5/1007" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/5/1007" class="active" data-icon-position="" data-hide-link-title="0">Rapid and Efficient Identification of Caenorhabditis elegans Legacy Mutations Using Hawaiian SNP-Based Mapping and Whole-Genome Sequencing</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459827" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-wH2OmhS8HKKbzADz0KbDdyqtjjk6BxA0dAjjXrrc4q0" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/5/1007.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/5/1007" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-anaXramcDOZsbUAg0ecppe9sRftd_yC0uyfjlJ4kjWg" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/5/1007" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/5/1007.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459827" id="node459827--2" data-pisa="ggg;5/5/1007" data-pisa-master="ggg;g3.115.017038" data-apath="/ggg/5/5/1007.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Rapid and Efficient Identification of <em>Caenorhabditis elegans</em> Legacy Mutations Using Hawaiian SNP-Based Mapping and Whole-Genome Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Aimee Jaramillo-Lambert</span>, <span class="highwire-citation-author" data-delta="1">Abigail S. Fuchsman</span>, <span class="highwire-citation-author" data-delta="2">Amy S. Fabritius</span>, <span class="highwire-citation-author" data-delta="3">Harold E. Smith</span> and <span class="highwire-citation-author" data-delta="4">Andy Golden</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1007-1019;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.017038 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459827--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Aimee Jaramillo-Lambert </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>National Institute of Diabetes and Digestive and Kidney Diseases, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aimee%2BJaramillo-Lambert%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jaramillo-Lambert%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAimee%2BJaramillo-Lambert%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Abigail S. Fuchsman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>National Institute of Diabetes and Digestive and Kidney Diseases, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abigail%2BS.%2BFuchsman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fuchsman%20AS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAbigail%2BS.%2BFuchsman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Amy S. Fabritius </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>National Institute of Diabetes and Digestive and Kidney Diseases, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amy%2BS.%2BFabritius%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fabritius%20AS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAmy%2BS.%2BFabritius%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Harold E. Smith </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>National Institute of Diabetes and Digestive and Kidney Diseases, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Harold%2BE.%2BSmith%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Smith%20HE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHarold%2BE.%2BSmith%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Andy Golden </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>National Institute of Diabetes and Digestive and Kidney Diseases, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andy%2BGolden%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Golden%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndy%2BGolden%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:andyg@mail.nih.gov" class="" data-icon-position="" data-hide-link-title="0">andyg@mail.nih.gov</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/5/1007" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/5/1007&amp;title=Rapid%20and%20Efficient%20Identification%20of%20Caenorhabditis%20elegans%20Legacy%20Mutations%20Using%20Hawaiian%20SNP-Based%20Mapping%20and%20Whole-Genome%20Sequencing" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/5/1007&amp;title=Rapid%20and%20Efficient%20Identification%20of%20Caenorhabditis%20elegans%20Legacy%20Mutations%20Using%20Hawaiian%20SNP-Based%20Mapping%20and%20Whole-Genome%20Sequencing" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/5/1007&amp;title=Rapid%20and%20Efficient%20Identification%20of%20Caenorhabditis%20elegans%20Legacy%20Mutations%20Using%20Hawaiian%20SNP-Based%20Mapping%20and%20Whole-Genome%20Sequencing" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/5/1007&amp;text=Rapid%20and%20Efficient%20Identification%20of%20Caenorhabditis%20elegans%20Legacy%20Mutations%20Using%20Hawaiian%20SNP-Based%20Mapping%20and%20Whole-Genome%20Sequencing" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/5/1007&amp;title=Rapid%20and%20Efficient%20Identification%20of%20Caenorhabditis%20elegans%20Legacy%20Mutations%20Using%20Hawaiian%20SNP-Based%20Mapping%20and%20Whole-Genome%20Sequencing" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/5/1007&amp;t=Rapid%20and%20Efficient%20Identification%20of%20Caenorhabditis%20elegans%20Legacy%20Mutations%20Using%20Hawaiian%20SNP-Based%20Mapping%20and%20Whole-Genome%20Sequencing" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/5/1007&amp;title=Rapid%20and%20Efficient%20Identification%20of%20Caenorhabditis%20elegans%20Legacy%20Mutations%20Using%20Hawaiian%20SNP-Based%20Mapping%20and%20Whole-Genome%20Sequencing" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/5/1007&amp;title=Rapid%20and%20Efficient%20Identification%20of%20Caenorhabditis%20elegans%20Legacy%20Mutations%20Using%20Hawaiian%20SNP-Based%20Mapping%20and%20Whole-Genome%20Sequencing" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459827/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459827" id="node459827--3" data-pisa="ggg;5/5/1007" data-pisa-master="ggg;g3.115.017038" data-apath="/ggg/5/5/1007.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Rapid and Efficient Identification of <em>Caenorhabditis elegans</em> Legacy Mutations Using Hawaiian SNP-Based Mapping and Whole-Genome Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Aimee Jaramillo-Lambert</span>, <span class="highwire-citation-author" data-delta="1">Abigail S. Fuchsman</span>, <span class="highwire-citation-author" data-delta="2">Amy S. Fabritius</span>, <span class="highwire-citation-author" data-delta="3">Harold E. Smith</span> and <span class="highwire-citation-author" data-delta="4">Andy Golden</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1007-1019;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.017038 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459827--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Aimee Jaramillo-Lambert </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>National Institute of Diabetes and Digestive and Kidney Diseases, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aimee%2BJaramillo-Lambert%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jaramillo-Lambert%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAimee%2BJaramillo-Lambert%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Abigail S. Fuchsman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>National Institute of Diabetes and Digestive and Kidney Diseases, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abigail%2BS.%2BFuchsman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fuchsman%20AS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAbigail%2BS.%2BFuchsman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Amy S. Fabritius </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>National Institute of Diabetes and Digestive and Kidney Diseases, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amy%2BS.%2BFabritius%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fabritius%20AS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAmy%2BS.%2BFabritius%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Harold E. Smith </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>National Institute of Diabetes and Digestive and Kidney Diseases, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Harold%2BE.%2BSmith%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Smith%20HE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHarold%2BE.%2BSmith%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Andy Golden </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>National Institute of Diabetes and Digestive and Kidney Diseases, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andy%2BGolden%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Golden%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndy%2BGolden%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:andyg@mail.nih.gov" class="" data-icon-position="" data-hide-link-title="0">andyg@mail.nih.gov</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1007&amp;rft.epage=1019&amp;rft.atitle=Rapid%20and%20Efficient%20Identification%20of%20Caenorhabditis%20elegans%20Legacy%20Mutations%20Using%20Hawaiian%20SNP-Based%20Mapping%20and%20Whole-Genome%20Sequencing&amp;rft.volume=5&amp;rft.issue=5&amp;rft.date=2015-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Jaramillo-Lambert%2C+Aimee&amp;rft.au=Fuchsman%2C+Abigail+S.&amp;rft.au=Fabritius%2C+Amy+S.&amp;rft.au=Smith%2C+Harold+E.&amp;rft.au=Golden%2C+Andy"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459827/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459827/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459827/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459827/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459827/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459827/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459827/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459827/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459827/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459827/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459827/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459827/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495856" id="node495856" data-pisa="ggg;10/3/917" data-pisa-master="ggg;g3.119.400936" data-apath="/ggg/10/3/917.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/917" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Mutational Analysis of N-Ethyl-N-Nitrosourea (ENU) in the Fission Yeast <em>Schizosaccharomyces pombe</em></span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495835" id="node495835" data-pisa="ggg;g3.119.400927v1" data-pisa-master="ggg;g3.119.400927" data-apath="/ggg/early/2020/02/21/g3.119.400927.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/02/21/g3.119.400927" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Identification of Suppressors of <em>top-2 </em>Embryonic Lethality in <em>Caenorhabditis elegans</em></span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495548" id="node495548" data-pisa="ggg;10/2/479" data-pisa-master="ggg;g3.119.400933" data-apath="/ggg/10/2/479.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/479" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Identification of the Genetic Requirements for Zinc Tolerance and Toxicity in <em>Saccharomyces cerevisiae</em></span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Mutant%20Screen%20Report" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Mutant Screen Report</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/5/1007" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/5/1007.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/5/1007.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/5/1007.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/5/1007.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
