<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/5/921" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/5/921.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/5/921.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/5/921.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/5/921.atom" />
<meta name="HW.pisa" content="ggg;5/5/921" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Charged/Polar-Residue Scanning of the Hydrophobic Face of Transmembrane Domain 9 of the Yeast Glutathione Transporter, Hgt1p, Reveals a Conformationally Critical Region for Substrate Transport" />
<meta name="DC.Identifier" content="10.1534/g3.115.017079" />
<meta name="DC.Date" content="2015-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Thakur and Bachhawat. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Unraveling the mechanistic workings of membrane transporters has remained a challenging task. We describe a novel strategy that involves subjecting the residues of the hydrophobic face of a transmembrane helix to a charged/polar scanning mutagenesis. TMD9 of the yeast glutathione transporter, [Hgt1p][1], has been identified as being important in substrate binding, and two residues, F523 and Q526, are expected to line the substrate translocation channel while the other face is hydrophobic. The hydrophobic face of TMD9 helix consists of residues A509, V513, L517, L520, I524, and I528, and these were mutated to lysine, glutamine, and glutamic acid. Among the 16 charged mutants created, six were nonfunctional, revealing a surprising tolerance of charged residues in the hydrophobic part of TM helices. Furthermore, the only position that did not tolerate any charged residue was I524, proximal to the substrate binding residues. However, P525, also proximal to the substrate binding residues, did tolerate charged/polar residues, suggesting that mere proximity to the substrate binding residues was not the only factor. The I524K/E/Q mutants expressed well and localized correctly despite lacking any glutathione uptake capability. Isolation of suppressors for all nonfunctional mutants yielded second-site suppressors only for I524K and I524Q, and suppressors for these mutations appeared at G202K/I and G202K/Q, respectively. G202 is in the hydrophilic loop between TMD3 and TMD4. The results suggest that I524 in the hydrophobic face interacts with this region and is also in a conformationally critical region for substrate translocation.

 [1]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003748" />
<meta name="DC.Contributor" content="Anil Thakur" />
<meta name="DC.Contributor" content="Anand K. Bachhawat" />
<meta name="article:published_time" content="2015-05-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Charged/Polar-Residue Scanning of the Hydrophobic Face of Transmembrane Domain 9 of the Yeast Glutathione Transporter, Hgt1p, Reveals a Conformationally Critical Region for Substrate Transport" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Unraveling the mechanistic workings of membrane transporters has remained a challenging task. We describe a novel strategy that involves subjecting the residues of the hydrophobic face of a transmembrane helix to a charged/polar scanning mutagenesis. TMD9 of the yeast glutathione transporter, Hgt1p, has been identified as being important in substrate binding, and two residues, F523 and Q526, are expected to line the substrate translocation channel while the other face is hydrophobic. The hydrophobic face of TMD9 helix consists of residues A509, V513, L517, L520, I524, and I528, and these were mutated to lysine, glutamine, and glutamic acid. Among the 16 charged mutants created, six were nonfunctional, revealing a surprising tolerance of charged residues in the hydrophobic part of TM helices. Furthermore, the only position that did not tolerate any charged residue was I524, proximal to the substrate binding residues. However, P525, also proximal to the substrate binding residues, did tolerate charged/polar residues, suggesting that mere proximity to the substrate binding residues was not the only factor. The I524K/E/Q mutants expressed well and localized correctly despite lacking any glutathione uptake capability. Isolation of suppressors for all nonfunctional mutants yielded second-site suppressors only for I524K and I524Q, and suppressors for these mutations appeared at G202K/I and G202K/Q, respectively. G202 is in the hydrophilic loop between TMD3 and TMD4. The results suggest that I524 in the hydrophobic face interacts with this region and is also in a conformationally critical region for substrate translocation.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/05/01" />
<meta name="citation_mjid" content="ggg;5/5/921" />
<meta name="citation_id" content="5/5/921" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/5/921" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/5/921.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/5/921.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/5/921.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.017079" />
<meta name="citation_pmid" content="25784163" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="921" />
<meta name="citation_lastpage" content="929" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Anil Thakur" />
<meta name="citation_author_institution" content="Institute of Microbial Technology (IMTECH), Sector 39-A Chandigarh 160036, India" />
<meta name="citation_author_institution" content="Indian Institute of Science Education &amp; Research (IISER) Mohali, Knowledge City, S.A.S. Nagar, Punjab 140306, India" />
<meta name="citation_author" content="Anand K. Bachhawat" />
<meta name="citation_author_institution" content="Institute of Microbial Technology (IMTECH), Sector 39-A Chandigarh 160036, India" />
<meta name="citation_author_email" content="anand@iisermohali.ac.in" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. Abramson;citation_author=I. Smirnova;citation_author=V. Kasho;citation_author=G. Verner;citation_author=H. R. Kaback;citation_title=Structure and Mechanism of the Lactose Permease of Escherichia coli;citation_pages=610-615;citation_volume=301;citation_year=2003;citation_issue=5633;citation_pmid=12893935;citation_doi=10.1126/science.1088196" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=S. Arastu-Kapur;citation_author=C. S. Arendt;citation_author=T. Purnat;citation_author=N. S. Carter;citation_author=B. Ullman;citation_title=Second-site suppression of a nonfunctional mutation within the Leishmania donovani inosine-guanosine transporter.;citation_volume=280;citation_year=2005;citation_pmid=15501825;citation_doi=10.1074/jbc.M408224200" />
<meta name="citation_reference" content="citation_journal_title=Journal of receptor and signal transduction research;citation_journal_abbrev=J Recept Signal Transduct Res;citation_author=D. Auerbach;citation_author=B. Galeuchet-Schenk;citation_author=M. O. Hottiger;citation_author=I. Stagljar;citation_title=Genetic approaches to the identification of interactions between membrane proteins in yeast.;citation_pages=471-481;citation_volume=22;citation_year=2002;citation_issue=1-4;citation_pmid=12503635;citation_doi=10.1081/RRS-120014615" />
<meta name="citation_reference" content="citation_journal_title=Biochim. Biophys. Acta;citation_author=A. K. Bachhawat;citation_author=A. Thakur;citation_author=J. Kaur;citation_author=M. Zulkifli;citation_title=Glutathione transporters.;citation_pages=3154-3164;citation_volume=1830;citation_year=2013;citation_pmid=23206830" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=A. Bourbouloux;citation_author=P. Shahi;citation_author=A. Chakladar;citation_author=S. Delrot;citation_author=A. K. Bachhawat;citation_title=Hgt1p, a High Affinity Glutathione Transporter from the Yeast Saccharomyces cerevisiae;citation_pages=13259-13265;citation_volume=275;citation_year=2000;citation_issue=18;citation_pmid=10788431;citation_doi=10.1074/jbc.275.18.13259" />
<meta name="citation_reference" content="citation_journal_title=Ann. Bot. (Lond.);citation_author=C. Curie;citation_author=G. Cassin;citation_author=D. Couch;citation_author=F. Divol;citation_author=K. Higuchi;citation_title=Metal movement within the plant: contribution of nicotianamine and yellow stripe 1-like transporters.;citation_volume=103;citation_year=2009;citation_pmid=18977764;citation_doi=10.1093/aob/mcn207" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=C. Curie;citation_author=Z. Panaviene;citation_author=C. Loulergue;citation_author=S. L. Dellaporta;citation_author=J. F. Briat;citation_title=Maize yellow stripe1 encodes a membrane protein directly involved in Fe(III) uptake.;citation_pages=346-349;citation_volume=409;citation_year=2001;citation_issue=6818;citation_pmid=11201743;citation_doi=10.1038/35053080" />
<meta name="citation_reference" content="citation_journal_title=The Journal of Cell Biology;citation_journal_abbrev=JCB;citation_author=G. L. Davis;citation_author=E. Hunter;citation_title=A charged amino acid substitution within the transmembrane anchor of the Rous sarcoma virus envelope glycoprotein affects surface expression but not intracellular transport;citation_pages=1191-1203;citation_volume=105;citation_year=1987;citation_issue=3;citation_pmid=2821009;citation_doi=10.1083/jcb.105.3.1191" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=P. R. Desai;citation_author=A. Thakur;citation_author=D. Ganguli;citation_author=S. Paul;citation_author=J. Morschhauser;citation_title=Glutathione utilization by Candida albicans requires a functional glutathione degradation (DUG) pathway and OPT7, an unusual member of the oligopeptide transporter family.;citation_volume=286;citation_year=2011;citation_pmid=21994941;citation_doi=10.1074/jbc.M111.272377" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=Y. Elbaz-Alon;citation_author=B. Morgan;citation_author=A. Clancy;citation_author=T. N. Amoako;citation_author=E. Zalckvar;citation_title=The yeast oligopeptide transporter Opt2 is localized to peroxisomes and affects glutathione redox homeostasis.;citation_volume=14;citation_year=2014;citation_pmid=25130273;citation_doi=10.1111/1567-1364.12196" />
<meta name="citation_reference" content="citation_journal_abbrev=Channels (Austin);citation_author=S. M. Elvington;citation_author=M. Maduke;citation_title=Thinking outside the crystal: complementary approaches for examining transporter conformational change.;citation_pages=373-379;citation_volume=2;citation_year=2008;citation_issue=5;citation_pmid=18989097;citation_doi=10.4161/chan.2.5.6903" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=J. Feng;citation_author=J. Zhao;citation_author=J. Li;citation_author=L. Zhang;citation_author=L. Jiang;citation_title=Functional characterization of the PP2C phosphatase CaPtc2p in the human fungal pathogen Candida albicans.;citation_pages=753-764;citation_volume=27;citation_year=2010;citation_issue=9;citation_pmid=20641018;citation_doi=10.1002/yea.1778" />
<meta name="citation_reference" content="citation_journal_title=The FASEB Journal;citation_journal_abbrev=FASEB J.;citation_author=S. Frillingos;citation_author=M. Sahin-Toth;citation_author=J. Wu;citation_author=H. R. Kaback;citation_title=Cys-scanning mutagenesis: a novel approach to structurefunction relationships in polytopic membrane proteins;citation_pages=1281-1299;citation_volume=12;citation_year=1998;citation_issue=13;citation_pmid=9761772" />
<meta name="citation_reference" content="citation_journal_title=The Journal of membrane biology;citation_journal_abbrev=J Membr Biol;citation_author=K. M. Gomolplitinant;citation_author=M. H. Saier;citation_title=Evolution of the oligopeptide transporter family.;citation_pages=89-110;citation_volume=240;citation_year=2011;citation_issue=2;citation_pmid=21347612;citation_doi=10.1007/s00232-011-9347-9" />
<meta name="citation_reference" content="citation_journal_title=Annual review of biophysics and biomolecular structure;citation_journal_abbrev=Annu Rev Biophys Biomol Struct;citation_author=L. Guan;citation_author=H. R. Kaback;citation_title=Lessons from lactose permease.;citation_pages=67-91;citation_volume=35;citation_year=2006;citation_pmid=16689628;citation_doi=10.1146/annurev.biophys.35.040405.102005" />
<meta name="citation_reference" content="citation_journal_title=Methods in enzymology;citation_journal_abbrev=Methods Enzymol;citation_author=C. Guthrie;citation_author=G. R. Fink;citation_title=Getting started with yeast.;citation_pages=3-21;citation_volume=194;citation_year=1991;citation_pmid=2005794;citation_doi=10.1016/S1046-2023(05)80158-6" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=M. Hauser;citation_author=A. M. Donhardt;citation_author=D. Barnes;citation_author=F. Naider;citation_author=J. M. Becker;citation_title=Enkephalins Are Transported by a Novel Eukaryotic Peptide Uptake System;citation_pages=3037-3041;citation_volume=275;citation_year=2000;citation_issue=5;citation_pmid=10652283;citation_doi=10.1074/jbc.275.5.3037" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=H. Inoue;citation_author=T. Kobayashi;citation_author=T. Nozoye;citation_author=M. Takahashi;citation_author=Y. Kakei;citation_title=Rice OsYSL15 is an iron-regulated iron(III)-deoxymugineic acid transporter expressed in the roots and is essential for iron uptake in early growth of the seedlings.;citation_volume=284;citation_year=2009;citation_pmid=19049971;citation_doi=10.1074/jbc.M806042200" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=J. Kaur;citation_author=A. K. Bachhawat;citation_title=Gln-222 in transmembrane domain 4 and Gln-526 in transmembrane domain 9 are critical for substrate recognition in the yeast high affinity glutathione transporter, Hgt1p.;citation_volume=284;citation_year=2009;citation_pmid=19589778;citation_doi=10.1074/jbc.M109.029728" />
<meta name="citation_reference" content="citation_journal_title=Analytical biochemistry;citation_journal_abbrev=Anal Biochem;citation_author=J. Kaur;citation_author=A. K. Bachhawat;citation_title=A modified Western blot protocol for enhanced sensitivity in the detection of a membrane protein.;citation_pages=348-349;citation_volume=384;citation_year=2009;citation_issue=2;citation_pmid=18952039;citation_doi=10.1016/j.ab.2008.10.005" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=J. Kaur;citation_author=C. V. Srikanth;citation_author=A. K. Bachhawat;citation_title=Differential roles played by the native cysteine residues of the yeast glutathione transporter, Hgt1p.;citation_volume=9;citation_year=2009;citation_pmid=19496824;citation_doi=10.1111/j.1567-1364.2009.00529.x" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO J.;citation_author=C. Kumar;citation_author=A. Igbaria;citation_author=B. D’Autreaux;citation_author=A. G. Planson;citation_author=C. Junot;citation_title=Glutathione revisited: a vital function in iron metabolism and ancillary role in thiol-redox control;citation_pages=2044-2056;citation_volume=30;citation_year=2011;citation_issue=10;citation_pmid=21478822;citation_doi=10.1038/emboj.2011.105" />
<meta name="citation_reference" content="citation_journal_title=Annual review of microbiology;citation_journal_abbrev=Annu Rev Microbiol;citation_author=C. J. Law;citation_author=P. C. Maloney;citation_author=D. N. Wang;citation_title=Ins and outs of major facilitator superfamily antiporters.;citation_pages=289-305;citation_volume=62;citation_year=2008;citation_pmid=18537473;citation_doi=10.1146/annurev.micro.61.080706.093329" />
<meta name="citation_reference" content="citation_journal_title=Mol. Plant;citation_author=M. Lubkowitz;citation_title=The oligopeptide transporters: a small gene family with a diverse group of substrates and functions?;citation_volume=4;citation_year=2011;citation_pmid=21310763;citation_doi=10.1093/mp/ssr004" />
<meta name="citation_reference" content="citation_journal_title=Microbiology;citation_journal_abbrev=Microbiology;citation_author=M. A. Lubkowitz;citation_author=L. Hauser;citation_author=M. Breslav;citation_author=F. Naider;citation_author=J. M. Becker;citation_title=An oligopeptide transport gene from Candida albicans;citation_pages=387-396;citation_volume=143;citation_year=1997;citation_issue=Pt 2;citation_pmid=9043116;citation_doi=10.1099/00221287-143-2-387" />
<meta name="citation_reference" content="citation_journal_title=Biochem. J.;citation_author=A. Mandal;citation_author=A. Kumar;citation_author=A. Singh;citation_author=A. M. Lynn;citation_author=K. Kapoor;citation_title=A key structural domain of the Candida albicans Mdr1 protein.;citation_volume=445;citation_year=2012;citation_pmid=22587419;citation_doi=10.1042/BJ20120190" />
<meta name="citation_reference" content="citation_journal_title=Biochem. J.;citation_author=H. Osawa;citation_author=G. Stacey;citation_author=W. Gassmann;citation_title=ScOPT1 and AtOPT4 function as proton-coupled oligopeptide transporters with broad but distinct substrate specificities.;citation_volume=393;citation_year=2006;citation_pmid=16149917;citation_doi=10.1042/BJ20050920" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=M. K. Rawal;citation_author=M. F. Khan;citation_author=K. Kapoor;citation_author=N. Goyal;citation_author=S. Sen;citation_title=Insight into pleiotropic drug resistance ATP-binding cassette pump drug transport through mutagenesis of Cdr1p transmembrane domains.;citation_volume=288;citation_year=2013;citation_pmid=23824183;citation_doi=10.1074/jbc.M113.488353" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=O. Reuss;citation_author=J. Morschhauser;citation_title=A family of oligopeptide transporters is required for growth of Candida albicans on proteins.;citation_pages=795-812;citation_volume=60;citation_year=2006;citation_issue=3;citation_pmid=16629678;citation_doi=10.1111/j.1365-2958.2006.05136.x" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=M. D. Rose;citation_author=G. R. Fink;citation_title=KAR1, a gene required for function of both intranuclear and extranuclear microtubules in yeast.;citation_pages=1047-1060;citation_volume=48;citation_year=1987;citation_issue=6;citation_pmid=3030557;citation_doi=10.1016/0092-8674(87)90712-4" />
<meta name="citation_reference" content="citation_title=Molecular Cloning: A Laboratory Manual;citation_author=J. Sambrook;citation_author=E. F. Fritsch;citation_author=T. Maniatis;citation_year=1989" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=H. Y. Song;citation_author=F. S. Cohen;citation_author=W. A. Cramer;citation_title=Membrane topography of ColE1 gene products: the hydrophobic anchor of the colicin E1 channel is a helical hairpin.;citation_pages=2927-2934;citation_volume=173;citation_year=1991;citation_issue=9;citation_pmid=1708383;citation_doi=10.1128/jb.173.9.2927-2934.1991" />
<meta name="citation_reference" content="citation_journal_title=Current genetics;citation_journal_abbrev=Curr Genet;citation_author=C. V. Srikanth;citation_author=P. Vats;citation_author=A. Bourbouloux;citation_author=S. Delrot;citation_author=A. K. Bachhawat;citation_title=Multiple cis-regulatory elements and the yeast sulphur regulatory network are required for the regulation of the yeast glutathione transporter, Hgt1p.;citation_pages=345-358;citation_volume=47;citation_year=2005;citation_issue=6;citation_pmid=15821937;citation_doi=10.1007/s00294-005-0571-7" />
<meta name="citation_reference" content="citation_journal_title=Biochem. J.;citation_author=A. Thakur;citation_author=A. K. Bachhawat;citation_title=The role of transmembrane domain 9 in substrate recognition by the fungal high-affinity glutathione transporters.;citation_volume=429;citation_year=2010;citation_pmid=20477749;citation_doi=10.1042/BJ20100240" />
<meta name="citation_reference" content="citation_journal_title=Yeast;citation_author=A. Thakur;citation_author=A. K. Bachhawat;citation_title=Mutations in the N-terminal region of the Schizosaccharomyces pombe glutathione transporter pgt1(+) allows functional expression in Saccharomyces cerevisiae.;citation_pages=45-54;citation_volume=30;citation_year=2013;citation_pmid=23280723" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=A. Thakur;citation_author=J. Kaur;citation_author=A. K. Bachhawat;citation_title=Pgt1, a glutathione transporter from the fission yeast Schizosaccharomyces pombe.;citation_volume=8;citation_year=2008;citation_pmid=18662319;citation_doi=10.1111/j.1567-1364.2008.00423.x" />
<meta name="citation_reference" content="citation_journal_title=Microbiology;citation_journal_abbrev=Microbiology;citation_author=M. R. Yen;citation_author=Y. H. Tseng;citation_author=M. H. Saier;citation_title=Maize Yellow Stripe1, an iron-phytosiderophore uptake transporter, is a member of the oligopeptide transporter (OPT) family;citation_pages=2881-2883;citation_volume=147;citation_year=2001;citation_issue=11;citation_pmid=11700339;citation_doi=10.1099/00221287-147-11-2881" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Charged/Polar-Residue Scanning of the Hydrophobic Face of Transmembrane Domain 9 of the Yeast Glutathione Transporter, Hgt1p, Reveals a Conformationally Critical Region for Substrate Transport" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5.cover-source.jpg" />
<meta name="twitter:description" content="Unraveling the mechanistic workings of membrane transporters has remained a challenging task. We describe a novel strategy that involves subjecting the residues of the hydrophobic face of a transmembrane helix to a charged/polar scanning mutagenesis. TMD9 of the yeast glutathione transporter, [Hgt1p][1], has been identified as being important in substrate binding, and two residues, F523 and Q526, are expected to line the substrate translocation channel while the other face is hydrophobic. The hydrophobic face of TMD9 helix consists of residues A509, V513, L517, L520, I524, and I528, and these were mutated to lysine, glutamine, and glutamic acid. Among the 16 charged mutants created, six were nonfunctional, revealing a surprising tolerance of charged residues in the hydrophobic part of TM helices. Furthermore, the only position that did not tolerate any charged residue was I524, proximal to the substrate binding residues. However, P525, also proximal to the substrate binding residues, did tolerate charged/polar residues, suggesting that mere proximity to the substrate binding residues was not the only factor. The I524K/E/Q mutants expressed well and localized correctly despite lacking any glutathione uptake capability. Isolation of suppressors for all nonfunctional mutants yielded second-site suppressors only for I524K and I524Q, and suppressors for these mutations appeared at G202K/I and G202K/Q, respectively. G202 is in the hydrophilic loop between TMD3 and TMD4. The results suggest that I524 in the hydrophobic face interacts with this region and is also in a conformationally critical region for substrate translocation.

 [1]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003748" />
<meta name="og-title" property="og:title" content="Charged/Polar-Residue Scanning of the Hydrophobic Face of Transmembrane Domain 9 of the Yeast Glutathione Transporter, Hgt1p, Reveals a Conformationally Critical Region for Substrate Transport" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/5/921" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Unraveling the mechanistic workings of membrane transporters has remained a challenging task. We describe a novel strategy that involves subjecting the residues of the hydrophobic face of a transmembrane helix to a charged/polar scanning mutagenesis. TMD9 of the yeast glutathione transporter, [Hgt1p][1], has been identified as being important in substrate binding, and two residues, F523 and Q526, are expected to line the substrate translocation channel while the other face is hydrophobic. The hydrophobic face of TMD9 helix consists of residues A509, V513, L517, L520, I524, and I528, and these were mutated to lysine, glutamine, and glutamic acid. Among the 16 charged mutants created, six were nonfunctional, revealing a surprising tolerance of charged residues in the hydrophobic part of TM helices. Furthermore, the only position that did not tolerate any charged residue was I524, proximal to the substrate binding residues. However, P525, also proximal to the substrate binding residues, did tolerate charged/polar residues, suggesting that mere proximity to the substrate binding residues was not the only factor. The I524K/E/Q mutants expressed well and localized correctly despite lacking any glutathione uptake capability. Isolation of suppressors for all nonfunctional mutants yielded second-site suppressors only for I524K and I524Q, and suppressors for these mutations appeared at G202K/I and G202K/Q, respectively. G202 is in the hydrophilic loop between TMD3 and TMD4. The results suggest that I524 in the hydrophobic face interacts with this region and is also in a conformationally critical region for substrate translocation.

 [1]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003748" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5.cover-source.jpg" />
    <title>Charged/Polar-Residue Scanning of the Hydrophobic Face of Transmembrane Domain 9 of the Yeast Glutathione Transporter, Hgt1p, Reveals a Conformationally Critical Region for Substrate Transport | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Y7gQQ2vz1S1gBoG5uPGSgQ-ZNbCwzPFC8Tk9nZp33WY","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459845","apath":"\/ggg\/5\/5\/921.atom","pisa":"ggg;5\/5\/921","ac":{"\/ggg\/5\/5\/921.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/5\/921.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/5\/921"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/5\/921"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/5\/921":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Fy\\\\\\\/avB4uuUZ3723VlLW\\\\\\\/DYero949V86sVFU6sPrBvVKu2uXutPhxYkXrPj65K6SchdRWmdr1aKpMe2Zg3gid\\\\\\\/xKNVMSwy6azLO1rJkovvYc0x2IqZsLeHtXMVLNL02Cuz353EIpvOcA53P5y1EYmuehYkOpgfJXLF3lTa62ozm53VPZrHWvQNpKV6IYf6EauZ4XB0ZF1+GD3bp4zWkbVbAgVXSfs5ZRVfvaSIyWNEO47+dy4ldH81h2TjEYC7sPy9mdFUpzXoxPpsyCt4kNQxnH4bIrwyKof+TuVbJFPOq4dWOL\\\\\\\/gxvbJjvbqqZtqsEU6IvMs1sO6FQHDi50WaE9H72Z+17wEnp8KNI3dR+etTgP98Ux3YckNDPpCHmXKRqpudKa5TI+P7lERf5IrjTkrWhlVtq65zpbIlhDMo0YqTt\\\\\\\/gz+EjsS1\\\\\\\/b2bzH6Uas6HYYXAxsmjMu3ezZm9w4Xm0FKATafCgZb55fsGKBozlp06l3tuDkx2WVtwoKGvlzLiqcAab3Y5VC\\\\\\\/GBSEk0LMdoCZ3ByMyEtOwcF8Bh7WZsHUcOqoqHwgMXglNT+mv6dXtSm6sOZ0+jkLPCRljkGF8SWMq3ZxTVzhpohcwq+IyWTDMQRv5RWSNAoX5GEiEVhUGa8yLeUWxk4tjmelBpygBrvF71kW2pr5e0nky90QNLBwwl3kFcQJsxD1YdamM7qUcCxkncThMWBTr92K0vRLEJsfhZrMF2XRGXn01A1gqcEPt4D63\\\\\\\/UuxNVugIzmz\\\\\\\/0UItd5\\\\\\\/iTW5dGU+abG\\\\\\\/ew8pUcM\\\\\\\/Hr8GgDqD2YDcPyNvgxc85cl3kd8WnMTT5Rfk1akIv+XRKg99gtnmqaGX1dm8A4eQtC1tPD32tw3rW1fQLboEX6f3XGPKkh6\\\\\\\/qBQMH09WsDELWJ8uq0xPRQckpFF3f0qJdJYOBprOZwNek9FaMgk\\\\\\\/r5EGZ8HcozQxotXEkkge2\\\\\\\/HdxyxVSfhhszNgWnUZb5AU6sBBkR9JTua3hQKecK3YBX+1tDw3BIWaABMClZIQjjgdaG3kzYVQlQCAS5br3D85hBEmXVj+yNpSFCZ2xidHKWzocTB\\\\\\\/BkFOvYgTvdJCRAFSoan3VdEHTCioJKnGh8PyPGweUboOjdL7j6+dmZhbNcoWVIn3bq6ij+nZL0rVdbGMaLBtMZS1Rk6TZXxpsJUs4pq\\\\\\\/lLG8q2ZVBOrRUBIFZ7De32XPE1gGgtNAVGkERkULJTcx3Vnbk9XLQckvaXfQTFJKkp+uAp6vEqfF7JyqJBfO4REH8uf9Xd7L6cKF\\\u0022,\\\u0022iv\\\u0022:\\\u0022jS90DgKTz8nbDW3UpE9Zgw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002291bcba4326cbdefe0a2aa35d86a4a33c41fc33066a873f5b5038b6d1f6fd3a9f\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022G\\\\\\\/TqlTWy4mvlNdJQRmvVjDuSRHH50m76lNdBMxKs2OBxHR2Yj8v3awhgW0npCl8Ic+NZDnXdXrQ71j4W\\\\\\\/3vghLBF6q5X3vE6ulBfGCP4dNa8O\\\\\\\/aNdyOWeSaleLmysjh70CQrm6oSdzPJAkWEk\\\\\\\/rjvE\\\\\\\/Tch6jhaXp5DLs3WUdZ2JrcHLjr3Y1CV91H4+XqLlJU0QUwni1TckHdClme9tArwY4O9dzixQFxspCzGx\\\\\\\/fEAkdU8LXgHwdwouUuwGcyuPypImfj4pqReqASEHAahwOYksfaHosPVq+lZ+8Khkk2jFnkl\\\\\\\/ZOHSNHAihJPgJKZXaY4s2p0zJ+SmxpUx4VjOcByJFXus8mwIMkIVscrfPp0JxcMKhX+u0IyAeC4MAuk1CJvdOfL6oDLwhC6XGXYfVDgqS5vNyVRjfhH0gS\\\\\\\/SU6Xq+fZPVhHmoVhqqg3hZ8\\\\\\\/jDcWMvlPuUEQegE61Nrhhe6xFAJUupcY2iHh47zeG5AxgXTvnb5R\\\\\\\/ruTg2cmC7Ws6l2kAcoHgmX+KKOXg2vRcHWsTkGloO7\\\\\\\/iNiaKoUFLT8vaY4EHM9OSM1Eo16RaqDljiyOSp1wMq5i+1zOFk5PsPxPW1dXaphVboYT\\\\\\\/ykMXw0ENypZpsIQQTO\\\\\\\/YA7kl9IRn7rg30uwVHOjR1POH7K9YM0sk1E0fFoxVMCRk4BMQ0OSfQmkhNfWpcuzWBhWL0DRs2y3TNBAG9g6ckfm8SkvTNEyyvb2t3mrIQS4vQ4HqgQPc74Y8C9lkEA3hzVSQ9XCoiiHPGlD7C+L9qPmCPVIas+CTJaekko\\\\\\\/A8cdRG\\\\\\\/N\\\\\\\/A0br3aKYeYymMYeO900lQy4biZqFr9p3llBYq33pXmkpbMSka4eeNPvj1eSfVhcMFV4QQ70YWNda2Gd2fqcp8scxEZrVkylgaADhVAGHZKMaJPIng1RLqWfTxfL8m+IhPtYXIr0\\\\\\\/8una3Hc5miK0El79A2acc51Az7gH2e\\\\\\\/HVEo024tY6E5QMoiL7207u7xaxhNbyVUyjK\\\\\\\/sw5u6ClCqLDQdZ8ZRZMASOzoTI1dCrcSQ6mFW1u1vGinrQTtbQPpd5udthLT7xnwfoEWpmce\\\\\\\/20OQ3RXdAxwSgsJm5lW8GvOzVvEfkWa+4AVN0k6Xn3P\\\\\\\/H9gx1tls3lP+cI0XBubR9fco9ZbpR3F7aaLPgOzIsmN5SMVn9NF37LAE+FjgbiqRp+9UqE0RmcDkRyk9uFitvb35HlL5QqF55L2bkgv3BL0MbHQCDZtMesB1mlC0n0JapdSzHKjuWJiUt6s7uJ+QZldCSLfZ1oapMIaCuDxCEMRyWzDlKvMtCD7sMc3lWq+xuxAIS8e7ZeuI0aSkJvoZKH2xO4a\\\\\\\/p3hx5h5hfUiysiWxv\\\\\\\/749t5Wsz9uqMBXQs5UY7iHGcxABamHJX6IiHu7EsKZ10yNgZnSxgLtRhWdZfAuJ5ZPQk0P5u4GOz5mvHAHmpsV0OfFAh0xypIyifaU6WPa0V1sZD0z2Y71o3M9da\\\\\\\/fR605xiU1iZoKZiQqMjrZ3m3SP25pyd2iHSr\\\\\\\/kw\\\\\\\/imMgcVN74YoTS2S8YyAgUT5rZaO8P0Q+2hIj5la5jmMQ0xSZnPoW\\\\\\\/WH0imjb2qEVQL9vT0OBEBYgkpnO7i+6FWn22PMOu0O1UQmNVGmzwZ4kbMiZx1FlgUbL4AnFNop\\\\\\\/OR2c+lyIapamS8gULs4w4lO4O5aSLbEc8exZVutgeWNIBNhkFdrRpLzoYdMAleRCuik7LED2BNQZTXJzKgWqA8NudwFfbW7eXvi1gFdYrvIty4pp\\\\\\\/tTtvTvxe0qSbetcihF2TUpW6HutqXOcdNVBPtLUmsXPr3Lx1Bmf2CSZMY32Rthts97eZDUJqAWrXd4Eh8hVNRst\\\\\\\/LaJpANK+bZNYepOkD70Xak31wdyB80o25STJDcjFGF10xo3CWTR1y5Q2l3BVkAf2Q8TWtrkE7+ku1oS5wnnBukhwtdsmgAsb4+NaRJeq2c3fjEtTGE1xPUN90sf+eirOldy8Cs052hu39paK0qK6uTLTe0JgoCC7yPiJ4ED3pULKBRiOWLYIFdKENH5Yu+XVxaRqM2BXKF\\\\\\\/IR3VRPxEQpVvVwKh3C2Ktgtsz98OJV6Lk4WtFCYGlZhgoN+RdByzUo4KoZBuxrt4kekFPvwgN9IQLg3f+TPMqyyiKd6UyHNfoRBEGoRUxqOLPDJYMRCN7bcMyVbdjaFkttJVXL3zugNw3os5EbV5LMynrgDrGfJ45SwBUXpNNn0UEPcgXwNosslDLXUTBIiSUWSmhz29Z6FEWb6g+yC6BeTDFTvZWFfl5OH0yPh4XnKp49UFbQSSxhudcDINyIdQ9wXQaa\\\\\\\/9mLHZ5tSRx0dCQIRd9CJJxlS4b2MeH5HatwmKkYY9ccj2GVwQU355E1amyGWPtEBXDHC0+yXp48w9lmIFjBgEnHGKGDCaupIPlEOk7vOzFr4rQmoo8BF4o6m6MRKV5khot+Q6CrdJhmXmiS+EY7HOawNLJS75Npn8KWFmL+kqira4jy20Lu8ZKGd\\\\\\\/TAQJHRREKq8M24malqi02\\\\\\\/3SPTxeftLTQQHhPqiDQcmOIXHYDbDGqbtOnD6ZnFXByG+wSFT+QSSERHYeOzR6VZRT8Ob24PfhKR0+J3FvEtTfywgB+vo9CLFjBFPqzBmEXiySqhNRmPuBqhaFS9exSu1oCA0BRT6cOpfTONLUQcoTmBkyV2V3xlVvqHVLy6f3uy8dIP9ib2lVYYAzwYbAC4\\\\\\\/5B5QeVd24E6G8YdVLYLDGIlL\\\\\\\/rtBmVZNlAQkl395RMEIVw3TbZHENG94k1jPRs+zXp4LyBmquPMw5tlJVZjIkhBtgRb7p6FPTKR8nojSDiIiK1+ll8Gme8PZNgU2KDaPbc9DA8ZO0\\\\\\\/PJO82euJgWVoYLqmKNoboDC2s3MdNIveVLcs2Q4nnUgsEGVNJWsi+QEEOlihlyGChVj\\\\\\\/ANJv2Ws9qqAlFE2W2EOvnVAFnHhA7Zz9oOn0KX\\\\\\\/3GGjDchUveg7nXiC55cuJLBLEdNJXKoYz0j0psn\\\\\\\/PMwaYJPN9ayKaheX49uGqOZRZ\\\\\\\/lXR9PdlR59eRBCNI2KSsnGadXFl2qeIUFg2Yd2Tri\\\\\\\/wDJppfTH5QckK36ltJzTocQ5JPBj+tahkyNcDKgUhuwRw4ZkaNWPv\\\\\\\/BFoCOdATWUw3YzgmJQsy6jRNJrSqa3zaur7zj+ILPmdPPA8tsVnIBrZdnvFBeD73kMogWRRsgjveoAWccYyipo1rzg5Q3RuMeJp51sAVrlJZcubCSp\\\\\\\/JMvpBw7id5L3DUvft9nm4enTogYhrgshRayxfbukXz3n4+kSe20LxzsgBZ2fAETtdY9186xCD0H3cc4taj+2+Jn46sVWqol00WYrSjeh8kygB0ckw4t50LAjOhH3VN8gKNIWny1AnCyc6bq95JGFgm3gtNnQvmjCaHCoiuss4qCBQhj8c4lfsP8PULidDu0Oz2+1G5d33weVRBvqYC5EGEo74NhbdqSPFOK6t+j0\\\\\\\/RQ2WBKG4wkOtUku30ImfZSz1S7daEcvVov+4ZBhD9t7WRcNok9KRHM6865E1Bg3zCkMraPwmKfKg6sNUOH1CBjFRy8WCo0IfEH\\\\\\\/rxcLZ799827KJnFjhAeu0TGNfYVPKMiFwbfYX5ODEZhR\\\\\\\/2ONLCKvEfUjOwvsDBRVF2KveuRC\\\\\\\/Pw4eBakFL2JSQzogec3lg68vQ7jRMH4FCKYfmYgJC4QYsBWUKYWP990ATUyHsSrMmKYIPJca9ixM+clJwsD9\\\\\\\/9KTAZuGiRLCTAbzUKx3SfbI93blbYg7fbJnFHkktsRqdauVa6XaUwM4eZAN0NAZbscSCSh0HlzBy2XVgRtUM1nqkriCyij68H+Y7in2YOQ1XmzUMODXvP1Ncfx9ckC6fVqD\\\\\\\/KITpG9QHsSHvGTiLgbqVQ2RcMJr1JX9iO5+vLTPEJmUZ\\\\\\\/doxMrzX5M9agjPJRUY44GimnQNobGtvTlzDwvqqLLsqanr6jVJGew4dI857D6uJCETf8OpOuXfCdfJOtqDL82JNTWl1j7EqnK7CRjsMbxfQ+8tKPUGpdyblrsvXnHDxIQ1MeIzGhAFiIFXIWw4yJ2CsVKzfGp\\\\\\\/PEZKrWVv7iBgHVXELzdqMZ52Tg6YVXRuaXp8nnHDCYjNM\\\\\\\/cbWl0r8AWcHMohponjH61nn\\\\\\\/GbF2l8YbTdZ0LBDysH\\\\\\\/1vbLXKTnazDt6uY32XJP5+cpAF0\\\\\\\/gYwU1JVjQ8OD+rMv40975H+kojUEzoyCcL63npc5SjAgNJrd6clh1dximiYGanJobANIBQJwLMoBL5xrovXF2fbvdTIasbPmi2ovRAj8fowDMBFgVXN1HAJ7U+H4oNLYr1eWYQtQoEc2B96r7WjTYvglH6zP+ViVsAx4cJVqRHumYVlKMzVrF0qkOCaFIbUMAA4vBchRquExjZXBllnggUzB0uf5lpmrykpgcVEZS+cDAFDjl0BZRxiPLzeXlso3EnHlWiSERNXcTaZuWa2pwfD4iqkhBzNHN2y4bB8Z\\\\\\\/0kGsLOmUbwfDvTbQq69D3XohJPVF8ww852YBxLWINkuJF8F3nMiLvTDN68GgKax72UTK53Fn0n46BIPg+6AOyTXkgC5pbvTElOLo1eaBxGIKpA5bB\\\\\\\/4OgocAnFvx10Q1kSz9c+6wqjI1LwkK\\\\\\\/zgpZnHXlMqibFJoBwJ3ADlPt4i8dV6OteQnJn3RlkX6tJaLH8Ycu1ooTq3eJcC2FGwrTatdYZvDvP9ftRxDgwkXge3x9uC\\\\\\\/vykh4r5UFXUTKOeGuzJKRxqDIdX\\\\\\\/0frJhW2UO1\\\\\\\/6lD8nNrKjR0e9F4waRYASR4x\\\\\\\/hBbPxz2AHLqv4bpJhImcSkqmJAxgZHeyXSrkDonpj2DFaUKIZ0WCc2XhHj9Kdcb7O5Te41QPDYB\\\\\\\/eLVY0ZN7UGtmz5iCmUUSAuLRPL6Tb9GF1Dn5ibmfLFJOIWp5Mv4asX+WORyPhyAyqKa\\\\\\\/sTC\\\\\\\/ib\\\\\\\/WMPhfoHfaP23Iq05V8qzxK3Qqv8+Ya7ZSV3vFBMLhyIhtN2hex42HgsHzCUvH0g7tvpxotGmZwSQMYfV3JjQ3AtazPmkm0r\\\\\\\/3zHITHub5cuyJF4nKWBZl6bZ5EXYz9DBNEAmKSnSlaYd59gAOSQck2nrv6EMOJg+b5BTGCxcmFGxzJuYEYWiva1Ab4RKWWM03Kl0H3+KxAEO+g1QcSpYk6PfudVwbedfDt68xzuYzQL6sgsSOpLguns4WPP8Bybjt\\\\\\\/QSQx3g+5Z\\\\\\\/fd067oXaRFGttbuVmdA9byxoumy\\\\\\\/FUt2pm6KCFEXM8IKApKeW14BMg99\\\\\\\/jidBWgWwJh0F9wge19u7XTqC7yMKhCkCh51YEo03hkkycl7ztMntLQsnfcxNXqVklWdvdxzbcwXwgUVjYyWQCXeRk6Vtv3DjRVOom06I86tcAJBzvB0HGqe4lGmC00K0OrzZtDjNdb26qfeokONB0fLVf3wj0rZHoRpPy+O0DGvhracDpWWyZWPmx+M2vI6v5XG2C6t+nYY16\\\\\\\/2cFzgooHAfwrdV760e6ZOHnMaRuB8NGJMOvTQELFPvyQ43X2wfu0BM5OfKkjC2D\\\\\\\/nzklyYeC0hpB23QJjWuEOtBj\\\\\\\/UIywwUaIAdwXdzZljc0T9BPCYKyyLrUKHMOjRW+n\\\\\\\/JedbdXka11A+I5aARzKRU5gHFV\\\\\\\/VBXiCWGyqlNj37mHWg844hUAgsH2eM0N0uS\\\\\\\/vN\\\\\\\/74a+Dgri\\\\\\\/hRBoa0aPHG1MAVSVK\\\\\\\/fsIk1Dos0hmg0UZQQYCn7GHDVQPlEhRGJSqPm+1w9EjFPFE6bvf7gL+sMDvW\\\\\\\/sTFlkRZHUDdA+34uNwAM2\\\\\\\/E4+zgl3dybATeAKEqDe+i4AByRs8J81pzSdwiB0Prcp\\\\\\\/rYfVElWSpb\\\\\\\/\\\\\\\/zX3k+V6A8AzPS4dCLJfWbQZs\\\\\\\/mtZybigMPXg1mqvaEKqfAa8K2VeMGi43iImLMKJF1P10kW4iXGRkrKv2gBFDwArJ7AXyUvePYigCEX7zeBLok5Xix7ijWWh376IW+W5+vaczhwfOd\\\\\\\/PwDSFRt63HY5WjXxqgwFK5ixgGZ24mpJg0jk8YcSTrpAgNQOLA2XJKfthtfgr+1mGNpgvX4oeIzsNj80hc3UBJCVey\\\\\\\/yHwKvYMceuTZLphe9JPxq5xJkDvgmmVTbyGA84QObJSzB0fE8KM4TlLyoKvGwtEwv5b+R4rn2lYFSBnCbUMnIFq13pis7poqOsmtraoTzMpTlbAbTmYJk52jGyPDsRxFUvcPxuL1NBPJYTMs19oo4\\\\\\\/wS3wJ8WIx5rensq04lLcVDIEY+I9j7QXob7qYfSwpEXYpZd0faDH\\\\\\\/D9A0t5WUIg91b+OJz9oC2ok+ueBt5XA+JYnbQCUv8K+hvffaBxJd207SgggX1zPCKcGiNcYSuq4OsjA4x2ah9Ku6xRTbTxz8qxuufARqYlJoqRDVJQObq\\\\\\\/KqGODi6rcwSHo\\\\\\\/DMEi2gnKsfIGUQS7TBT6AdxrBNOqMzQ4eeP6nMNG4ywYeQRH8alPHedAndLwgr1iX16WL3n2ADi1QNpoCGT9cnDKr488EwyrI5nQW8cTunLPetPpeoA0DIfNxVn2A5b01UnKnOHKlr67zuAvOpMKiXckX6IRG1R6RARHC5UXlY9VJUvia6NMZA+JO2fj9gCqkt15tbkU9YaD5RC+RJhNBNF61tviwKCALt8\\\\\\\/Ml0wfHdIz5AHr0cYDt540wSXa2Hn+bW7BYqmG1eK62pqMZfV1UYV8NE4ZPbElyg3EqKBz3aZoywE75gKZdj9dPoR4xbl3aWaDW2Ub+xUN\\\\\\\/M3ER7z5MPlqnT9BowV94wKcUYyWDtP3Zn6RCjy7JLlBaytWht+CWxbT6CE+m1yyp9qGIs80iHTcLNTq+6i35u6\\\\\\\/y9GvoXMcEfdBbw4UHFa+C7ZEB2Um4MGe6UsVUuB\\\\\\\/4hm3eZKHEqLrHBYC7jF\\\\\\\/LGT4wohPn4Dmy5bgbXxG8llMvdTBD7ypnSCvMHYgGjFmq18o+7p6MpXMye6HUStE23ThXTkB74WCWco5Z+DsC3bj4zuMeDZJ7y+1XSFUp1vtk8RiHNci3PjQ3L6igvi1vkjnbe76e\\\\\\\/iln1xV\\\\\\\/vuqtiOCGe3qxFvybX83bACQojgteJe9ugXNc6bS8z0Sjb2KJn36SAJLW0hQMIExRPBfDZgcaJ+onLVtZEqAdAZrtIoY4GygGtZ5cc34C8CS4Nd3wKomsSnFNvO26bJVebKIv7pXIJUdahobfXdmS3PQ2z7JG5bXA16RaeJY\\\\\\\/muewjffpFg+4QKKsLGl3RJGng3BHPm2E3br84PvSAP\\\\\\\/0v6Eu6TFXqqlHlGJ4xwfzCGX2MIHU8\\\\\\\/xc+GCQVeyDPOAHe+553LsgFd+w+9x2wPVDWoHHrcpefXV\\\\\\\/UH58j\\\\\\\/I5KTvNaw+pSKt+CLHOxq8QWg6n0QYzVUo3v7cdS7ja0mCOeS7EyCj1L1pQzfXF40lAhqlbPHtvsl9Vh5jVLHWhb4NHQzU0XAtcJndNej4Ss3Fwpax1sXcMTjpxJ37zgZYTc4B8kc9qFfqeidYVCfY3cYlJvxBZT\\\\\\\/Mf2WxdT+g1EcUhcZDXIR97FI1r5oh4z1Ln0aDk5+jDFwo9rdvEPK0ElBbjefiQUGh4MnpuWOsllWKTe8nRy2vjdBW2uJZlq6mLyNgQsLiYK4Jar6iLpIIC83jkRwF22Yfx+mjCJYciosk\\\\\\\/XsTUI\\\\\\\/c6VB\\\\\\\/MLpMY3ckN5LUFT1NuSZ17z0UEnf31WDOPX0\\\\\\\/mjh1Pdu4xP49QO+vb9VjaQK9NlkLt+aSlaA6nOPPJI9rkZ7w3ZESNrD45jpgR68mRURgvNJCbJbsLFo\\\\\\\/U4g\\\\\\\/imWU\\\\\\\/NgOWAiFNQH2bwAOl0ARFR6DEyzSwjRRKps7PTpOjlG6FoJ67jXUtGW\\\\\\\/bEc37RxyDPWOxRaQtYXi0IuL8bSXC0XXQ+tdIMgBuC8FhT223wxP9iB6pPWTknup+chsVAUWLjSRl1L1OcMVqFSzMyHS\\\\\\\/8FhsGL7Xg2U9cocOFtJQrtQb5Yvjq6qH7t\\\\\\\/qk\\\\\\\/B9MqlCJ8hvalJ7RH6KrFvoWar5UH7Oj0mgOk6+2YKR5GTN01f2bgvHQ+4gDFH+wxrPcw4l8Zanxi3FX2ev13X+32\\\\\\\/yy9WH8XV5V2h9uszTkhNzCL+Ahm1IuKgFZ4jJahApA\\\\\\\/1f77nj3ZSo3AU5ln0lHPFtArF9\\\\\\\/SI+ldcKTe5IcnDGn9ixlyDLKLmreCkDImP9j6G7a4hBsCcjBAMRhpDg1KYmL71VtDbqILlQy\\\\\\\/FzL7zP1WKNlNLFnFRC9\\\\\\\/TZxER4xIqYhUiTFmzqBssphoOqcJ5xvyhEDd8Ci\\\\\\\/a55j6D9heuprYzbQK27HBFZXBI4BuTSFkbMItjsjDZFAeuWKRfjR0SUdU3lkluWgjpahvLVq9jiNcIrh1kGz29u4j\\\\\\\/685PRdDjCLaGtMYzpKuOsx8hKLu27zrhcziSi5C3szeib5AM6tVbpJBXfcyxtcua08uWFiklJahMETDF6PyeSBlgcyU\\\\\\\/1JMVAZLSQy1ef+7ozVUCobpthBS0RCd7EGdHUUWEi8f0Aef5lPxSfC+kVw8lPgrujjDpPq412L\\\\\\\/sxgwC4f\\\\\\\/e1F14che\\\\\\\/mTgXnhnnMgf55zXHc5IgdQQD0uAEnX3peWvmOMibl5k8DSjKz\\\\\\\/yO2pGxBnX1V0D7ogLKyIW9LE55M7LVHoHcuaYRGCOKVlHhQGcNkTh31gv4Uva3PrEDPgglQQBEj6fTQLnLyGZi1X2nx2MWeAb+O0uWerV\\\\\\\/r4XowKX3dZkRsJofl3w8UiBX034zdDZFIDQfWcg6NMBUTIhfdA9g70UajuR5jRb923amFj3Sg4UzFvElovBJUppDb3xtbhtFCYjnbQNAkWTZhu1auTblvIshH+Td9ZDL0CJjcXqeCkuhACLuFMxlaoQEKpUPRI6XsWng+RbVSr8k8skeAZuHvurNlL1g43sFL1KvWwEi7QRb9+PMjTqvghccJuYwKz\\\\\\\/0JgHJDd2Ylg83SWCfdw4jsoxfZ1aZ31EdsJkrLu7ebTnuHamRuQRCrE05+URcPm3xfgLDkrAoBvnwE8SdowROMgM3ke\\\\\\\/7yxDhaJ3SrcdyK1n\\\\\\\/+J9VCOJ4XO\\\\\\\/EJnkYgkEzIDbQWFKDIb8OWHUGRCDS3zCsIo0MwKbnIeKNWcoXRBjoaIwpDrf4GB3OdI60J9cOoqted8EHZhgCGfSF8rTIbCKgsn2pDukw7Pd2mkZPnGEjwu+Zk0SCTyOVwmwj9g69P85t69rKB6bqFXQqXT3owaL+dXBZuELqmldAhVEt7xN3TCey8ZlujudmbD7NxLVVfOoK0Ddeg+O4WFUmilaHjqyAuPnbry534EyQXyHuhykbeXaG1wK8ul0pAec\\\\\\\/bPZxDT9LlIHAbGkG4tDCcCZV++mUMpsHubWRVysR8a9FaqIxqcbC+juF16QOB808EgRyXdPdU424fHQyUt1M+1WajVkXvDaMZoAQg4G3+l7vHSh8vz0cqiyj0lGkOkAjJrKZYyT1+EQxdC24S\\\\\\\/C8JV6J8nKMTipdojI8soOxsxlQqc4oat7Xky798dZG4qJchLuytPxxDg\\\\\\\/DfdxXchCYIVu+CSiKi+xHx86FykKFDuH7e0xXMP991ZlHzlIKtpR92MY2jZtEpoBvqIiQvZUd0O19rPoJx9iWbccCKtokBA1XkyVG4PBdaD97VuRp8wGrFp\\\\\\\/MKaSqONTku\\\\\\\/nr8s3OALuthJ3DlHsLvsEXHGcNpots4+RT7DNHLc896to+29Phlkxjedbx5fU0TwbuL8W+a6T2hL4tfXuGeFgERDkO0EYwh7ADvE6Ih44DhCRgfVxrnE2p0sLuRVmniIb1Eh72MIjF1Iwlp4Wm7praf\\\\\\\/5RDnD9GTRnd4uZvEbFZvEc7Y9JkPzSJK1eh5QXba\\\\\\\/NJ9WQAwlFTL7VRdHsyHytgrJR8O5fU3q7izDVCDEp9B\\\\\\\/zflgeXV+gY4k8GmryxRzZnekUW+PUekRVUTfSYjg8cW4S7lTbBcdkCk3Vx8LSx+6qZ6aK21I6qV6RrSanxvLtalwDduBRK\\\\\\\/ylj6xnJt5PkBTGq7C7v\\\\\\\/KGCiIhok1rJtoHIwQ1PFElI2oxqVB1U39jFqB628HZBNNuCnBdfYRQAN+auFY4ECtw9u8bDmQMyTqTTaN\\\\\\\/vkFZtcfV6IgMLzrJ6\\\\\\\/apOeJf0Jmk4DneNqv27ULCSEfqO94UaX9GF8ZPDGqU4W7ZJMWPBud7FVH970SwojnAEuVIW9QA9aoLY4gzrhp6KwqqlumgLL70bWJ9NqGQN37GhQJAGKYPnNMNU0fS35c7bf9YGANldjN91P9RH6i3f2u3E2\\\\\\\/oS+OLs7HbJW7N5KwiKzEx1IjXn2oqVBhDGzuCbVpNY0agS0SGk1\\\\\\\/xsB1pq6lXVZftexYqI5dqt07ppBuyIHb\\\\\\\/Bhw95\\\\\\\/LAPISFgqQhO1J+a8IKGTYptM4mt6jzYjxuaproLVfTVWw4etdlFCBYHkAAzqsLLw0czQnYl2R+kgUbaSBGdN03XUofGfBkgN04h\\\\\\\/EwtAcypWsq5WHRkx+Db3lvli6kHBvhp3l+rSEYjL14YAfHDPKGD4I2DrkeEeLeFY8dNgXs1j9R6QytjbuqOdhAAaNyyfyuYcpqqyfH6XsZWnTms\\\\\\\/HTAdmUpOItQ0qNPjbRh5hrurb33bt\\\\\\\/FuMszZhl0bfbeBHSne0NfEXLXd+5AvwSWaJvD7q04e\\\\\\\/J5cyxB3Y0BV0CteTJo2EYPeuuot4Owk9uKOka5qKefRHOSEuqaRHw9dZBTqDF2M3yUCZLYAgokcOm1XaM2f037uAfdflNcS\\\\\\\/mg7bGnWAkDIsn4uWRIiM5U65sjuPY38A6ZCj\\\\\\\/1G1nNgFwDrwu093rX\\\\\\\/1UCNHat2HUTS3u6Z3WwXrqEjxPCQMc02+HfoBchHjNcgrVdvb6xArhQbcXpS9SCS8jTRjABMg9L8yrPerWm7EEUypbWjG6h4FzDAU4Cn6CowWB+h3KcmZP0D8RoHGGtzPTJmxqLa\\\\\\\/OyAAkovEYyJUynBlZBSeTxOWjQBUxdahdKE3fGPeDc2cAv1CmP6dhLt9mBb5AoIO6Nzn1g5UiyvV3Hq0jF9fiJGs0QZVJNiaITrZ2\\\\\\\/oJB+fHlXqdKxWMnq73pcr7G\\\\\\\/IibPGc0zf2V8xvuxFhPF1Xv7v9q2FjtTOxQgNQPp04WKGOm9rlXj4bDfer50do3u9saW2I78gu+ek7MFDzchBoAv3BEC1q3E6SRCp\\\\\\\/Bc\\\\\\\/eHl+66F8dKGjPLC8f3bOdQRgLZ44nDB6gDiLPCa+BO36FgMzrE9VLu++Y5ZuBHRWepme5uLfTOnNQMpf+mIXChlf2Xfu+lM97DVFSjvUvCWxTeX6ovU1FMc+SSOAiTCmAmakWoFNkOHqy5RqDcmlw\\\\\\\/whk+FMQhFsuZygBAq1cSQgPoL3FmlhbzXQFhsmV+h8fV2f88b1bkKccORg6D8wrv\\\\\\\/oquqgAR2rlB9fZj6wN3GWEXKIT7dZLh76UgDQMZ\\\\\\\/hiTDC+329lv3pBDCvLFurGXwayRtvsMnn3KMuNf0+YNdfUYSbdGMBd1cbahA+ZeHMlPj06LdhzfYmwIlYodz6FYl8QPfgnRmUB4CerfO+WZePODSLhfo+2C6Urdw5mvCtiT9yLya\\\\\\\/i7\\\\\\\/b96PnyOQra1WIAwK0dPuUfXpSTvyyAQMNTKluod+CY+t2\\\\\\\/kIHy7AE5eYa+pbt1ldLsIEtN9QfOAZ1vTwN7vvAYHnvyeWt3kr9CHCFAG9igR6DEQNI+96knGt2Btp2topcyBkZs6n1RGqdnuLUCz4YybGb0X6dFSp1eFdMEUtIseBokeRuXIAp\\\\\\\/92voYncVRGdJ9vCgaSs9pHgwcRqSLr2wnYv\\\\\\\/BVl0DP7sKrIKeR2FecEUu1JXcnjHUe1ZIHFYJjR5S9CadvZRBHrfz2ImQJGk8g30LovVfWTRqz+RAAMbZIaZ8zqAG1IDifdXEGAAJQ7rbAB7xCOujMFviucyjMFnTOQQ09ctLOkcB9TsPFFP7hE9wiQucSfOLssHx9nCizk6DM4WmWrA3T+4W5VZNaASVK+Dv+sAz9PD0AjKPDeLgwmFTHln9XuoTgC7bddHjRBCXTAwLoSJPd14Kx8\\\\\\\/UZhFRGUSYHQU1ijPNqB\\\\\\\/nu7oK8xjvFXb26rKengTIC97wEoFAthIgeBA3cuMxqjjJE50wWEbvSVZqpDNV003Ara1uSDEySDMxZsChOcWcfxnquNeWWN2tN1z++F7tfR\\\\\\\/30Bz1\\\\\\\/zK2HVkHdlu324E+3NFXgfxpt1MW1xeZItPIHRvwqBqNdbtJi6+F6O\\\\\\\/Xx+WO2Jt21ZciP5w7xAQE24c+GHhYXgYLh+xb6nqZGTxko3uklynCxVvt82r2sLTg4xYI1Od3yi51Lw1hYGW375f3m9QyVJdtk56SDvF6PuXkvrJbSqGahD9pmFW7ZgdBXUd2Rif\\\\\\\/7mK8k+RRAfqY8kBRvdHzv6fEeStvlMiGPZ0S\\\\\\\/SR2LjYR+tZoeMSrORRQaxnWCHRFqVGF8P0DBRRvq7kNwTjCIY9\\\\\\\/K+ZO+KZKvmCII9h9OTbqLioiGwgy5SE1dX53C\\\\\\\/fUx5HOkS4z2V5YhUr18z4wagWRak5RF8VApE27L2CQilntUyebWPH2e3YSv4EL8XQi7ZjoNNzVq18BsaBjviQNDJ0b8kH69K+RQjv\\\\\\\/gugcw7WKEMuKBZQz7D\\\\\\\/XS+Uf1DuwCYMPeH\\\\\\\/sUubU7wJJxobgLDplHVMPFBQGvtvWOzYa\\\\\\\/bK3CW2z85lwOi+WoWsSeau5UeI5FKf95UqE+GRYrQR5x86kH228y2pTOKamcVdJraUShd6lnQuznmEui0Try\\\\\\\/zmUL0U9KWyMuLmoxv8anLPshGOs8LLXvBrxIxVy2urj\\\\\\\/xVNhPaUbZPsaey5xzIDTGnkdPWYNZ\\\\\\\/vQTgi8DGfMipp0SdOKTs4OJ9HGCXJ+zdAE6+WS8AVrY19wFkPtOG2NByfS8dyLSetH4Da2y48brtZF4puaRUABLTp7kQeclczmUrwt87Y70h1CnUi7AbIPz2R6fFOLuNb3+UGVw2XM9r2crqw\\\\\\\/h9119HvtwL6gvuuLYuOe5meUDJ21I05Z\\\\\\\/vdb0si+Rb9beTbNN3J6GKIxRub4fwm3Nez4m6pR7pq\\\\\\\/ruAuEKlgyBGZrEGcZFeXmW9yFir0yKjdFAANzV8XdKzW3lxdIHxgKn6K1hq7zkKEKCpDsx6hdLw\\\\\\\/cU1k98cEWnbXdZ7CxBDEeLsblppNES\\\\\\\/Rq9G+et91VpzUCZTtn2rPf0XSu1wr0Xh2JoQoSw+EIkIk5DSTgl8aMqyuXUQ6AHYFn7hOoSUf6SbxvHi6vt\\\\\\\/oO+Ttvtk98fLpYKrq2fLRbFNbyUwbrHqKPrS2NI0MbFwFlC16gWTx6eaLLQo0T1l4aeMQtUr0VWDNo9Fhn2e2lBA0a01psmro4Y8yYA\\\\\\\/9giGblF+Hk5IKJ02UJ3wn+ddC+qC8eo5X0jZZ6YuwyOV0gbrpv+Qci8VKEhU\\\\\\\/bPQNTfWFH+QZDZLoKb05mXQ3pakXgj2GTW1yzf5PgMBI0bs1FAoq4uyZXRaPQ9y\\\\\\\/CdcRRWPMfB\\\\\\\/qe6rQi9Orgpl5qge485cmcjBvdz36tKM5YUhvDjm5vQXs2QxVg2sZ5LB4+28qXAPMOK4RQLvdiMWbRPo6PXyNoMYGyhkA\\\\\\\/oeksxxGwrZrzuPJnzSW7j+wArqwjL49760mT8ePwu+otOhkswzZL6uiqPPqaiYjhqi+babiCRtmFBs6LQiUOr52PLsisks1\\\\\\\/QnSuad0bZszDQ0I2poZdx0C+bdEwP+6dvndIz4LyYQy0ZWDYVqUnyPUvOQwqt5Ei3WXp8hJQ4+16woM1GVB8eFEBjNr+3n4PNaxWxJq7SEUuwArdm2\\\\\\\/LtFDCWdz26uckpl8CCiKfN\\\\\\\/2uiT9pnhpVbYfU7d1b2vKNs17nzUf3kJNQBhCLPrX2ePqVNu\\\\\\\/6+3E6jr0ByWvmSa9kEU9ARW9J5g\\\\\\\/6kWdbWss+BvStyCvJbXS4ekyiaIy7U7n9WNxdvWhzUQL3ZVDnMtKLRMCMRs+SC5jq\\\\\\\/QDnnQF+T0GISVjHmHWPOLC4sFn+lnVr+MzEiyELpGZUNx1Wy24ZztVdAi6aTyZFzjOl8S6A9kTUg2LP943t4CghFVQ1mZCHyAciMNTbu3KnoUv4a5tBKIudWUYnuSmOMhsmHvLUKHUpsuj7dpY\\\\\\\/CgZwCeFVETsajd+k\\\\\\\/TfNHkZCOFSnH9XWwbDRLOJ09YA9Qo1MLLPXzWf0hhiqEBhhzEAJjSj8\\\\\\\/Yu7Ng2tfInectxRLOVJKOmkUyaRr7xZroZ\\\\\\\/BJedigZayGsjj9+b+bm0giKrTDPv7CGvmapZzX94XeVQySEwYuq5qXZ8cJMh2LO3VuQ\\\\\\\/u3iWU1JEsVRJRgAuC8YleE3XQrvYHGv22\\\\\\\/jH+fzxS1YAuwrxcaxprrdk4a0Bxukb+PQNLXENrxkLT3937\\\\\\\/cacmQZba6Q5xpSnorQPGwgo4\\\\\\\/zWSM1CPoZsdbVAPSxBiuj8K46hj9i\\\\\\\/xQAcJrvd9PYbLpFrdaLHojF+Oy330rnk9usAfLGM+SsAyDUZsm1+LHmpze1x0RXoRdYZJTSxgmBoN2b\\\\\\\/rtHdjM5meVPaunRYCl8ALIONkQHKwC\\\\\\\/nmds4W0eqYNiFjWp3kFFYU33pwP0W6+g\\\\\\\/5EjjOEJ1mzoMgRaoh+ysdEmeghIb7f8jX4I7Jknp0zHH3AKsryy30d7Ith8utfGIYcfazEyxORTmmRaUbY67qfZTeFuC+rgym+otc5lOb7SPJyK3X+Eho28spY6ITHEkz1pZy6LNWa5A54enhxB2FDYS6Mw+2\\\\\\\/T3P3V\\\\\\\/k2ATLnP7qdpPE0Fbroevi6+gPpwjMmXjXlItOeIaJGmaX0O3uqaYf7FdCFafSvn+gz7u6P\\\\\\\/R7bsYvD\\\\\\\/860sf79pdBbDiAPHt1ZSk4QrHnInENsTt7jP41DIi9c+nf0p\\\\\\\/kr0MvVos0\\\\\\\/HNUPm9Gqz+mL8HssQMp1nbS4GdJ+x5lTzEq0xC7bcpBXIluZIe96wiM+KiuGnu7n7pz7XCQL5fYD3L2aARMX6NU7cSUPHKwHsPFTbxLWMV9B93XpDZTrjZ9tVicD7MWQ9\\\\\\\/HMBd57pdceKxOC4OU7Rvl60vgAanq1rbXXmSX+JPOm6hMdOwgmRm2nUsQ55HHhUXvqs+b7k+jgSkHCAEWQUgVi1sf2BmOgsYenhUDuLvrSiHnNLtve9FfoLTkWDXzCSUvjtVOZyroRyC2VpNmmt+f+QYUt2rVgRedAhMMRQphjf+\\\\\\\/inTylL9FT+ubXBODHo4Rv3g+VBliKRU8s3gtnQPzTH9vnfs1HdRO4cFDcJyXcO9sv9tf+LQPzTvsBDJw9vNT7d5TUHhhi1D38i+VOoscTZEnmL92fAkL6uSSnveWJNALOhXHiiHrkBgCoJ6V3eJoJjBcMWP8XIDr5ooHCt0u+L\\\\\\\/MaItdnwT1GnYdRuQAv2FTYtoo4J3S\\\\\\\/xADZi3FKr6RjBAZ+BTeGeuVrYglkCEI1LzmcHun7jZKIM492sw6mCASi9q0+8tRNZGSROMHasAFdVWtV3tLQdMxhzJ+kVPxzEft4EU7Kf\\\\\\\/MX6xi9UcQrLlFTPg58QtXO1wIvDvFKzttAviQs3xrhNgy35SyhZC8P3l2k7rSPaHXvD47msf\\\\\\\/\\\\\\\/AT\\\\\\\/ZdbFJSgO+G4t1KeoKB8L4xn0Ws4\\\\\\\/I5Bp3hmx0+P\\\\\\\/E6LO3DLZuH60FqOaekWProqkrdmX7xLWCcFRd\\\\\\\/FbV0hLrfF4ehdlFHk4+SutSJ9NOcAIFlrU9pNcjKuiu2o08uAd\\\\\\\/Z4m9KcBA\\\\\\\/Irn\\\\\\\/BSVm8Fi839WU7W17cK7TRzkfGj9GasarZLAEsFB+lMQlOvYeJGCqw\\\\\\\/RR6XRi1DwLmiAW6PvhdyOjKc00ei9CIvOtpOH2RHB15Wg5IfO1PaswYkPRBIVxhtj9OEhC0Oatkm0N4OyS3mvHkSsuSGIuvmaVb666MW\\\\\\\/1IsY03O52ujgOPb80QnDyYGSdXfc3qNL5+SckhLCOTw\\\\\\\/KfqMjlZHXrxj+MxjlQozllO1RrrbK\\\\\\\/KJQOeFUgDYv2gpF0y\\\\\\\/YZCGISsO0VTYc0AHM+Z+4KwXfyKuRDBmY9vXTyF0ZK0dCQaOi6m5jQaF\\\\\\\/JoSUoxCkDKYW7MsqJOLbjrWXly5T2CNH3PeqAxG5gCtF8IKlZT7ZXuY4XCFRxvg20b9tlD08+xNSAyEq6UxWnJmPIjZLCUtgTl7Z4ixy0Gof+vqd+LKwd43C0ygoEXui8wqYN\\\\\\\/GsGrSGTk7RX\\\\\\\/K9EiSV+sgYeaU6WEHoFaxUnGotEV34ljEekzcfpwUcOBvQJq3dQiXnKWoztHStz3ZFaKcZozZiJqWmvQz8Um7bR6Eojm9VDZUk2R2ctbC0i0Pu0cR9UtgQjtJhJ4ieS4X40Ji1JvV1GFQex3iuTEAPajE0sn012T92ds45TH2GUmDZAzs0p042XkeTt0wlgb72zLNCSkSOKzwIhTUGCSgH2so4azC51TQ3DB2envEDXbtB120QQe6p1u6JNILhw9RNVFz+bEnR6XMaitGRzL1G6ooSYO5+dyZWN8MwV4NFcibDigMQLkZR5dOpTwgw0Ycd1AmYWYVCdDd6zURx9K52nPCPgzFNynCNhQ+\\\\\\\/P95adRA+G26Vyg8ib4uLLY\\\\\\\/elesXohyJiQSxWoYR7L446I+wP5XkWwUUMUBZFakFy8kWwukRxta5yrgZUp0HBHbNOTBnxhRal7uSQ9M\\\\\\\/Qfr168YyMXmi5Prc\\\\\\\/BY7BDWYtNcNI\\\\\\\/\\\\\\\/iSn3V2YQZvlDLeH6JholMCeoplgXGnqTEuLpFur9P7fCJgxvYpc1hMbNgLYFIbnx5g5zl3erbHqYX9TPFzuh0rEJ3n+Fq8y0iPbGGDhY3HI75sRBV6sdSqFMf6eKtHs9fwpYn7bXbmIfzf\\\\\\\/OixILjI4xK2S\\\\\\\/5q5agvOyLwsEV7VGDg7rNrCA91DEXBLZ\\\\\\\/FZx1N7evDSJ4+vMfPvHhXa190bIkkh4HLevocBbNt1mIJDXeREak1uchSdcsET+yO4EBEVW2ayvQnBCNoDh+0hJbXAvce0WF8ldaBMtX2PIKNVs9WJ8fBcJ0OxVs5mSnswtQUmVwqeBZIUNHPLFWi6xcxBHK4dyONeq1ovoRG8IsIGMIp7C6p+QNvr589yfuyzEqHjpRWwofcMYWG\\\\\\\/siQmMIT1pdb41wdwh5g+Y8rSdjsN7i9hyuf0ouTF25iew6PEJ9a7mYO0YoPIaUWTKDM7mapz4l0XHDXfm2PL4NVDRicw\\\\\\\/pZX2fV6QRj+DjFsDFVHGEMLydRvedXCA7IJfjj\\\\\\\/Bt3mY3Vx2wg5II4Bm4qLfOh9iIB7698QCYpQYtHWwp4nWNTJvDJkvsiYhXThn0nJioLzCXeK4XHKMDZDvpqEsrebr9tC\\\\\\\/teksmmMQc2QjEIhN6BpMD\\\\\\\/3clslYvnDPfOU+H\\\\\\\/pAFesi5QF4il1NiaOz\\\\\\\/2Xa8aMV92fuXxTEcwp5curMcT7AW8Js6uerOinCtypcbf0NNC\\\\\\\/OEZz2Ar6Lr3HTBfb8BYwYXXgNtFNkjBsQMTJhFw2UBTCpQea+o+blqobmGxQmK5ACsbqGTFZPO2M0VK8NFhe+rUMpCYP8B6e\\\\\\\/Q+U6dz5IEzUnOqkdA0lq8kqovmp6qIQsSTQaP7MGXAs0uIEMJcVjs7V\\\\\\\/nmTQmIccqxzw+FfZiKrt016I4VPBRbj9BvwIT5Hvw0zXKQH7oC57nPT1bfouoD0XJ5XawGj773AqdhGcZ88uOFxFOXmhUnvjjSMHW7c9bZV5twZWij3osaNpCiNSA4dFeRDVDlkflWPlqnSCpFADpCUA8BXSHKXKSaRVBWehH6GwzFHoNkJQ01ec20qpei2UWan+8L20761I7v77xk5L7qYTyO2zxarxIDfQ50ohUzc7HhrS1pGx3i2pUt6tZUfEIshg+hpEPqrRflHzJreiLf9T3KDjM+pQHG0qdbP3uqUjPx9lxtdahQTKcvATSrDSPCfX9LF5WY9UfEEr6w\\\\\\\/zfndhqLeWyVSf2vr4I0qJjQ+H92LXSHlF83wqyudu0t\\\\\\\/XYe+BimM8QLFkDcoIkcvTiNV3BdlLKLmFD+rVRNZxPzlX9BYqlXLIDjMinoryw0Rj3ZZ1CA6zkdhhxvsbIv7FXcyiEHoYdQ2uJxtKdHynL7Yhyo\\\\\\\/N7JhUfB0Xn2yPa8iQFMQViAhntjfS8SnsbtEDT5qD7yvdFwQQir7KT3CDdW4zJhsQltJhKW6w+7HawnfCEvyACVP+9UkF47AxM9T93F6HH8q4NePqcPXvxuEx4CVtYQwSPObQvZbwEyUMS\\\\\\\/8KVdbT9JclGNTruYfETlm6N8+uK65CFUD9tcQXqy7kj0YFVHToIrvNYRrGbmuuqxeQj41R5i1vLkVQWnDLjajiv+o9bC0x3zVxV85zfbWAKmWehJjZbZlLLlBIXyGBneYGyJWvRxaO2y\\\\\\\/+NJ169ZD\\\\\\\/L+u63G754qckQo5Zn2RqeujPCBdnX4fMe11ADdUA\\\\\\\/9aGnp3UBdViRB+2cePk4lp7YovNVLTtzlKsbIp8wGjT6PNYBDsuJRVfbCFQk23j6YAx2uDToankk8x1uA6ieUtnpDrQgem+SizA2un2WBC8bIt7ufzwQktx3\\\\\\\/ZJ4YmNzxDPV8vpJT7RTllXf1a17inxmuYzyaC8jzOGyvge1jyeKWyLcKIjGnMDJB9ddsIEHMQOB9A5c2bE\\\\\\\/kBfo+DmbE3gHNkF2V01i7G6JJjqRzXTxvwHR+2YUtvhX31SC1tCM+stg2Dp335zdOgvKw+BdPZMohFDdnI3keo6g9bz5rPg3\\\\\\\/G1Sii40FRh+iFT05L\\\\\\\/u5eMIdVsaMDl0ElBqdcI\\\\\\\/MCrQpwel25QrklYe+sCpR2psd+mU35TO7Lm0wZNbP+VVeOzNS2GR5jx343lBNFDNwXOOGTVZ95o5NJxbH+fPtdQ6ScCKjsgsiSLdY8FKXtEOrEQ2pSfz1HTxdLphjpz3f+shnPOt05YNkvZdlYdChJ\\\\\\\/0KERFT4LOG+yofA4qxRRKg31BOeI8xITshVHNQFGCEc\\\\\\\/1dD6Je5uEj1diHQ2w1\\\\\\\/D4FdJQ+YzHapiluQqqE8\\\\\\\/2WD05CANiBlQ3m8GEUo+uvsvKcKs\\\\\\\/aW46K5b9BqdN+G1Fer2yO5gC0qgvec0jtMommBoy\\\\\\\/3OgUjo7ndiCCOyWV706rE55ONo4onwK2a4dpzs2JP+4mrNcuPaZFX0VhjUGtpouOuL773ujWGDuJRvyE2\\\\\\\/D\\\\\\\/w+ns40ICCEzOtWACDttVS+D\\\\\\\/4+ZDdiLmt4mLYmqaXyKjJXVjcjwSWtbuOTuBzoUWpvc04TAzHM0QG3BG0syx2kyaGEMRL2XyhiLTUfemLPyu8p9UksReL7QhfDTizTCbKoaJo6brOQJN2qCpwgpJcPkYB6RtimUeU+mu9KOTLnn7LworKAIfQuEFaA7G2H4xSSmKhdQeE8ssFZr6IEnr7pMBAQKdonsoFA6\\\\\\\/3OTjI2Xl0QBCAh1U49qf9RHMfzs0VWErG1mcoeG+s4XXfU3ZpJVnI4w+SC+dt88H\\\\\\\/h9N1J2RFOyAMKmFKtcadsFspUWlED6K0uFixwjcfnn7rV4JMC39c\\\\\\\/tRY7+RNxN57CnUg4IjMzYRGehaxlg+NcKUiiuE72RE04egsTb+Em4HazXfB\\\\\\\/K6VmMLoxRnJv+rHFep00lSQPmJGoMjeALaVP1MIH3Tfot\\\\\\\/g2CxPUGr9PDHHY1OmoWG4sto839y4SYS7GalAdXMLv2lqVEY6CSFHBVl0+k1F+rxRFjy7fEBmgZ\\\\\\\/dAmkmjr2W4irW9rK2oDTNPMWUbmViqJkWIv8pFG0g6vFQfW4QdU4JNQV3e+0JqWxmRoQVZALY4ZLzm45qzWV048btqKT0yaGlRZKQLCFi1w6JI8jQBgB8cPwJzvLc7SNYT0VP04QmprYvncTy\\\\\\\/M1Zulqx800o0FAHgQ7\\\\\\\/TTNCsfeFIB4PR0AYryZXsCPcJ4xcy3RP+a1idp2Ew7oABu7orPALE7aGPvl8YRkS08sxCB58pJsJYTrTNx8ydLtgRcuQGiNU732gcSHVBKOL5V92\\\\\\\/E1T\\\\\\\/CCvvgb09lkYgSAlRo14AgdZv3f5L\\\\\\\/h9JZSfkXdqsh7k6Q4k9MEnVE9jRzE5Kc5Tv4pIidXZaWNeyjol8BgIURJTlDYAUalYaNlvglEcKwsuMUwxMwl5OBRVY7ZoXv93kS6vBwTN\\\\\\\/vY27Ck1Ht\\\\\\\/xYgGwMTorETVHmLziHTk1soqRtNXazYGlvaPu4y2XmJZg9eHX\\\\\\\/d63ogG41F+7p98+xSHYviPFudRel1Km\\\\\\\/C+8ATR4p\\\\\\\/Rud5tkEVRN\\\\\\\/KUQdI3R+zI6NF4496k6bu44YJEjDpzb7qt+hs7+iM041rtB9YaVh8v6j8boZ3OevKNDtzcrrDIGh4k9RYl76EBmgyYCby\\\\\\\/P9bAtCUtWYf\\\\\\\/Rt5hCq+3RBz\\\\\\\/C6rBk+Zaw4SjTKj2uRj72eO6GVRxa\\\\\\\/ONgiLzcrMFASCTD+9MfuTaG6xVKBLQlczPmmrXE9ikbXbIvMaf99Us6V2bY92DijQVMT8DD3plcwvO1SoR2BkkMrz\\\\\\\/0ZxK93bWWEcu8eET0AwIdUgOkeoGT\\\\\\\/NkIVQ+PHkpanKZ4tYh66rUms5A3WbjfdbaLMWLbvMaO82e5B0ATXYAwGgnw8zEgoWPEso8is2UQt4TRDmKm1IyFkjDgMu2mbJSR90P7gpJgDOjEMMmV4ebA4hwbeu5yUPJ5kzTQXLzF6QRz8meRbKH317dSbdHi+WxLa5qwPIi\\\\\\\/URTiQ+op2geN4lOI1Gs2X98cFXv9C0UllotsqYhFM\\\\\\\/7Qr++YA4JIihS9L2iYpmX9pqnBVnE\\\\\\\/27zyNhI75rdQK\\\\\\\/km7Vuv6YF1VLp2YE1al+nuZ3y2daRD4ebAcu56JG7Uq4B\\\\\\\/LW8jfx0UlYdPEo4rTeMJ5ACqlaTVsCigSj\\\\\\\/3147pt95XAqFRNEj2gfl0Si0mbC7z+dReFvoahizm0nrf2ON4BToivSur+EoT\\\\\\\/H78cFvQfGNJjhgjp2WVu4IkoaD9\\\\\\\/k1zRrNWchQg+SNPquGPDfW7LbckhG41Xkyhe\\\\\\\/moAaGlnVhyAZjoBtBpQradNaI8ENFz61tBkWlr\\\\\\\/EDZQG8b7T6U\\\\\\\/4BM6oyPN\\\\\\\/RaQ+s0Mp+B0aCTHSO1AxcVXlukLYhvwAvgR8xbaofViE5EuuP31yrJNkogCHPrcH6rQvvzgdBO1Dh7CR1BPg6ZbA45TH9BdG20cgeeBLGGVHuzRu2ystklmJuJglhSxNncXZBBE53y5V7aJwnDq37+CmplyfTcO0azfXwW+owIhquP0G156hD+olwT5PAI6IoftJWbGvuuPLjWkjGpXOmHV8Yd2mBpKefwNPV\\\\\\\/2jofgXx9nGjU0Do1q6DFrDNESFCo\\\\\\\/4\\\\\\\/QBH3Kp33qNQ\\\\\\\/XC1wExDrGk4c9zofPg3oJGGScWt1B9oNHOXly5kWojvFbeecGdmNDpKY2R3QZxj3gPMhdNpnQ4hPHd66PZLPHzp7\\\\\\\/RS3Sd79iMEGrZa25bNWWjVm+c00O7Dy1Gx6ojjGQ63QLySfDfNdGmdQ6OZQGpo9acEXHOsHQ2vqKghHQ0Q67+Bqg0KbKvjAPL9+LMrheicWGQ0LYfwHmZ3Sr\\\\\\\/z0+w\\\\\\\/t6JOzWsZrQ\\\\\\\/k4LhQ8Ay\\\\\\\/55bvCMfj7Tjocb22CgkSCvrwXUBgMLZ3h1tqta901pwxrlqXSc0+m9NVe+qHSL6m\\\\\\\/snWDg+G9GF85\\\\\\\/fdIq0zV852XBJ78bSd43RTxbssjkh0TS9AhbVjZNoy1m6j6QkpNcSWQDjOaSsWxJC2bRRDkEZvJkzWVz7az+Ivml5XZVKvPa3wpqayNG8fsjTONB\\\\\\\/Kq5rH8bjtv4iZNyynzb\\\\\\\/G+UcQWq0nEboIY7nxIudCyOiedZZrfY5ATGfzNqqzkrEdodtGMw3TIzzatpkt7p2S3y6iUn2LJdDe99FI+S36uEdGi6lDtJjsjUh\\\\\\\/OgTu6482W9rc0rCiC\\\\\\\/FgxKVowQuTcoINSsAQejCWJ0NahhtwXofrDEhSAKQTwz2exkmL4WPp7m+Wjt3Uing3qSPzObYeHsZyBLekDiOrqep3GSewJO2wN7z040yNTSfNcOKNBymg2mjAtDVoYlCZ42eNS2ybx6xIWBir9XA64ywYFYHcSI3z23FJFqSpI4JOY64lc2Wokri97tpG0xi5h97pDf9+ZomJHMjxAg2qKEurnIebDtj+\\\\\\\/3799kTl7TpYXjF3D8ZNoWDJfwzpMlBzORag4gb5sj3U3M5RXxk7EZCDRv5lkwhjqHje6rZW9hmpYHzs\\\\\\\/1qxq9ybBHzLYXkpr6CP6BPrM8GGpcN22FWS1NjpQ9yfBt3fpUpY+3ovtDI2ZljtCOsoGY1X4m\\\\\\\/14XbEzUTzmLWTiG+nW4Rt55OR7C3lzJXAl3z+Nz56CIOnED\\\\\\\/lM5Uw1skygojUKocLn9nL0gRD63ys4LfdFd7P2Ds0\\\\\\\/fkbjOXOPiR0OpAnrq8gfUT2JmAEEtIT3RHIVw4aCSN0Fo6Loh2v4up9gwjy8mHFeFRGcBeOD7XWy9K1dnTMFlIEH96GPPVQ4hney1V6SITfLLTYydLyBloAYd\\\\\\\/5Yrae6oZfNjTCOQQpgtp\\\\\\\/p96T1NUbVJi6mg6+WDBgzjBud26Dk9ov3HREJIkbcOeOXZ4K9ZFIVJ11\\\\\\\/bzwG7Z7GJxLZRFkynQLOjuIYRN56IzDEecvoG7QomhKKgoWYlcdma5fX\\\\\\\/lOIYM5OKylmgVPSN4V+j+lhWqpinMwDYM\\\\\\\/Hl6w6ddG5zdOuEmlyCMA9VU+FlsC1Ij2ErGkMZ38EPgKCu1\\\\\\\/8pyZiKYyWvr7mUZ3jXv6\\\\\\\/SEf0aEpGM5F0FapmGOdiqxHGjBClcObIwF44\\\\\\\/xuhvGJxkv8RUXNi7YH5ORJHq0XtVE4KYeWn61Dh60MCup9g0k8YYhDLR1tIc+fCDrvMh27dOSlRTvcUoLF3BETIAOh2pV9U33EYykEIjcbLHQuT\\\\\\\/qM1mGZQ0QC4JJgw+CPTra5lmkDpGfkzRkEjE4mnAvoBzYoGgeiyrK1r+RYEWv1Lz7PzLd+HSaKOq1UciTfbbpDDHu3uu96D+iqKeWXU1sIGHHgB\\\\\\\/nSHIqLTM4a97YH3e0H3TGzgIp4gX7S5KcjQ8zzgBm1BH5UliF8oqtEc+ChvKCGnAdz+NdUuFviX5qXHqf7zChww\\\\\\\/nNC3UnrLHk5cIDKbT1qgCA+HQRfxz+c3flfJVCPXusSnO\\\\\\\/OfOuynjejhbzfebXEwtolbR+I9vfPO20boI3qAVoogItryyyj9JBGUSzM17ft205OaGTnr9mnPm9jEEZ65zJqiKH3xQE7Bz2XUe8ORYMgtJLIKYHrITU6zDOc2AKO7wkguPiVVU4AgaDKfXiSwP5S+56qn5SQ3T2KAHiJY2NXRy767fh3kvOcPZEkkTzDz3wXvKnxeOdm4dbSNr09UEzndxY6q2rQuNy6z1Fn6kcPkf6zWOvsgZibJDM9jpWLRwFxNG6nSLvyLfh7RhJW82PzfzYp3vOkZon45fj67e+oDQCAF3qh0nlduwgeKVu0Zz5ZD51VyPjsmpwwg5OuDBbf0tnK5Fwmhu\\\\\\\/Ib+VXnU0Q+Gre3qMYVQ7LE8pxH2yZjS0JwYvYWYKrPMudOFeNQBwlbBCNxbLDmju4yN4Avez7CEtfZm67wS\\\\\\\/JOxa3hwOeL0Cxx1ekZl6XJmWinq\\\\\\\/lIWvGdj5\\\\\\\/xSDcZPPnMzmbI9e2cqqzZxjLWTA6iN4HivXfJOYvtSoEL9ftjC03UDQmIKtYNlqhjSerT1c\\\\\\\/fvzUWHBWfX9zkl+ihYs6d3NXh3Ng2pjADcvjltLKyR0OVYPAyK7ssFlN\\\\\\\/d1q4FI1LyVUW6vUF\\\\\\\/FwXuO3mhCjvecjeYq5sTnbQjq0fYurxb\\\\\\\/WGRmp7pNLyIjWl7Eycu\\\\\\\/9Wf9\\\\\\\/dRabWI0O1COyw04di7XEilZQRG6L1D6xgaVMP7Fru0WGTfUz78wBowEcI4CfijO22Dh\\\\\\\/DK2Cbt6GdJB7rok1z6ckyW8DE1+8GqdiYahbqJmkDUsvreRQJKU5uL9Zsb0\\\\\\\/5tLpiMp4AnnU07fgcwprrwvBhxhiawcjALRGh3fq3nK8HX62rpuWvVswecvt1Ta2LesSHx8po\\\\\\\/eM0DgxkwWa0i+AHqM92zeLGvHiZfvlKxq7\\\\\\\/Biz+yp0Rk+PujBDs\\\\\\\/uv3Qy36DihV7yYNzgYeuJ5TImd9ZunnoLtRj7DUec72JOa29And\\\\\\\/T6YAWnLO2aA3e\\\\\\\/vzStRcsnQ2UvWMK6Z7SjE+NY1gmTwreJ1dqm20iCDJ4ZWTfvCNY9vSyDlFWLJ7bCiQeZbvP5xUjtCZMLHw017YjJ2ryJ1wq0+wJ0LsnT+Ehj7x0ExEAezQzgzBhl0nGxbig7gMa5rXbHc\\\\\\\/JnnPYXBEHIqV8o2WCQszL6rHdyrX9KDgiAjVqfeNhtPLZs1Cvxt0alZT3qZTelpUvGDHfY1\\\\\\\/GSDwEG3CztrLECLBVSstS2YuC5s9GaMxpBh57ZM02F9xA0jV0dl8Q0HtKZkGBnt+o7WUxOitOW29AyOn0tN7gCYe9eXVzAfRwNJj1Q2FbzZEBNtwE58AU0VDdwIKV\\\\\\\/lqSIiqX9h2Lnpsw7Vq3uw6HkxbzmwPkzYH+YWn5mf21kWr1mQot6Fc7\\\\\\\/9Niim4KFmCLi01FBpJRPr8hAxSvsMRxK+MxAofgQqb+Y9+9kzrWNxGokZF4b6DggQ38RaIky97iPvvKcrPxP81\\\\\\\/bzwWme++ZlVjc0fsd2BFlzdtBNKyETTLS74BEVYCL6pJBVsTiHHGUIcneuvu69eQbuiH8PYthaVDGq\\\\\\\/zVe4UY8VAf2MYZTq0zxHz\\\\\\\/ZOcNfzmby5H1e+Bp9qgGgtLTRjX3hzN7EZvZ3lwnTlSz2Jwuki2iA0bfFgcq4useMB\\\\\\\/xazplIELGzuaZ0rnmwQ038omdi4yB3b9vnoX7o9Ta5y9XQ+Pkpu19qgDkyQ8Ps1oF\\\\\\\/s3VcqNiEPTVg6Y1y9vBAKhJP21zkSzf4rRstQVhO\\\\\\\/RMjgWzD\\\\\\\/JSBibpmxubfDdknjf6sQHn1Z1\\\\\\\/2G0w29MBk7CZi0o10oXpcInjVzADXHHSoCJpa3tKTW2qia4lxOO1z+hEsC3fTAzXiudcph7OvqM+leV38fCz2XmdYZcZ3Gc\\\\\\\/u6NvmY7rv3conay+Iix8BDYjgOeNVmWwBWACp8RKZDG33X8k3e2ud5fXKspdApPq01mMuRq0qnMVHh90pZP\\\\\\\/8\\\\\\\/w149vJeag0nJ1B5ww0RsXO7GHN2Uh8fw3Huj8ORoIA3rM+a4EVQqlWE4sQWyZZCKrXN\\\\\\\/FBOADWXhRqU\\\\\\\/Z65gwv59tgBG+vu6quzaOyxIYEN6O0ZICdpVxbtE8nTw0mHneCiKb14yEg8lM\\\\\\\/BJr7XxWRyq6RYchuDU3IY9ESrQ6VztMB8Xi1Do8vh+9l3Sm20iTy84L4vP5d9ucV+o4wbNJmQ4V7hdSB6ovLbQYB\\\\\\\/fywKyHSU\\\\\\\/GBOSYIT3HORFxQ+OLJ+zoDbPF\\\\\\\/xt+urqmExg8Ig4gC1hu5IYvB\\\\\\\/svMIR9xudegDOIlasoCkt3WdE8Ijbp8pvWvQvrDA\\\\\\\/f9+32Nxbc5ukMTkIxuaHAT\\\\\\\/4JLAD3FYMSEZAs\\\\\\\/aATgRgH4Tqf3a63xBAhBc6Z29qT53fYavYq4Rpauzj+udc20T6v04mkjNc0E6g\\\\\\\/U38Zhpf2HqdMBYLaS4SQruixnod8t5tnavE9HV1zbEc3GZfzCDTmWHj6Bhr4vui3zLZvxhzP11S6BWyn2TrEUQRbgu\\\\\\\/EMkh5NmsycieWMeqwNc0G\\\\\\\/WOYJ0IH6cN6UwoLAeoBGVAMz738uxfJ9LCy70d\\\\\\\/543hkYtTptnqz9vlU9L4Jil0Q5h0az0J+MpRraXZk36f\\\\\\\/w77y2T090dpAAzXf5vF0R6K7LN06L1hF+fcfLErHvyIbI6UJ\\\\\\\/ZraIKY9cEh1OPEy6GmVxDy0ElSIgpZiJd4dfmMtJkOuHN5ipxhd5jX+OCl7GSo+vH2uDfXSSXB5rLgPJiW03RngRml8ahcm6Qd6yv6V+IH3SpTZr0bw4fDhYB9C1cVWRMQrbLFZMmlmjvY2K+X9S8kCnMn+UtcB7WK8Yv9JZ0aStdhgYGJf+eaSoK42txajD9I5ElnnSqBlBzZfa7GYideda3bXgBpuzQzHAusgbza0rIBrz8b9hHtDDI0TrAKjf\\\\\\\/7BPapJnukAxDQGfOVvL\\\\\\\/OnYXJHxnloKPe\\\\\\\/pZ7pbgrOw8xYAt8bWwxpAa0ZgA3tNwySjJ3+8VvaLCDRhR+qamVwxpNa2hB9jzbkh8cVDoiGjf09+4Co51pB5vXXqRSCpwP40Xk7RUqox2XwkL2J8RYSOimpfhF\\\\\\\/3kfNO7zwpqvKZUpUBeFNis8nMQYpfw+IRLovHCSEw8Bh4LiNxknif8odrcmWtlxVzTvaSvEhPNhBxVZJkjO1mDLbGYiCleLEGnzx442+mN1rBOeXNJiX7jj7dPLxsv0kSX1uzk8PUG68mskSihC1itqEH2nLcDIJUhnBsmCVjAyZWdJ95e95VB0miGvgjT8W7HqTQfqq2M\\\\\\\/CkEQtMaId67duslwrlYvfr0r1RSFNm2ystOwkImLsSwaAoiqMn99qJ9yBHQlYOrlJa3GxAb+FHG278wv+BToLqKMYdxqk5RV7EaWfmMCLuj+tEY\\\\\\\/go\\\\\\\/mhnvzFhACsHRR10svaKfI1UJ+FcO3hjnMHgZ4Z1PpBklbshkgIVZ5VAyc7k3Ye6stmYuNjp8tBqX+\\\\\\\/Imrt5VEQwBfXb4HHzAx+N\\\\\\\/AdjN033aldMnErf\\\\\\\/9p1qSWgcS5qt3hNZzGC3RGNFxhIODgNu15EGFY5LN6250Lh3RPzZDYU4uo838J\\\\\\\/Cu64baaPk89t\\\\\\\/8mmgjW7zL+M1eYIzUm4Fid+KQiY+QnsYB1Rbr6oZEN9j0SWY0fPDMu5hRgnNaoI8C8u9A58nzRGgqJSImjH87XruYtpuW1vpRrb17V5TKMHonhP7jEPd7PRH5T8Dfvhw1aslqwgyNAkGkRyMc4HqtxEyTYf7MiugTUEltPKy\\\\\\\/QxzUIDbcqkDH\\\\\\\/Ehc5Ii\\\\\\\/+\\\\\\\/DiZJSpcJRPxy8x\\\\\\\/+d565C9xMqU0XtnenSm7MGn12Ltvpcpv8Mad8FWoVwdEa1Rrs0EYhwmDYvKr3PxDLq\\\\\\\/tswtr2UWhUAvqMmXyOATrbm2mXOe4MJZEjSKSFbz51gs7nrzw1YVDouV8O1nBov3XN5We247I49JI1btdrsa6hU\\\\\\\/R54Lvc1GiULRv\\\\\\\/S+1VRAtkNzOyxTaAJs2R0IvUV6zs1xHKKrv6EtI5cFjLWlHOBxZXnm9dxxhuDxGWFGEijzVmS3XSBbRwbJBGeGtHKlaF52SqJNR+sUyJcIrPfcpXyRoM6Na3h3m\\\\\\\/4y5ysafVIhSoDJY9diWq01GQAzGsGM9I6J61s4\\\\\\\/Yl8WW18saL+d4MH5ofS5smIDh72bZ\\\\\\\/vnP1mcTCdt\\\\\\\/G90LwboqW\\\\\\\/8NWP6I5bf4hAgVlWCo1bBPsDq1XQcUDKb7KwXeqO9eKTgFDQormvnnijeBecRQ8nMhl4jv9W8APMaQ3+yYVtBTrtFogVYreqqRq6qISX12r2F0E\\\\\\\/ikkxTMv0n8xks7PhQM3eRHLJVp\\\\\\\/Ml1ZvR6hlYTFXrnfeAPzZf10NmmhSUcpVMQ5V7LIuJ7kktygVvbqswyhYAsgnPh\\\\\\\/DuzYgUH2xN1bOj6SRDNva6YB+TwVe6NlIKl63SpstFnET91eX2i2MxeSA09xoQIHdWHaWcNJeXSNyhZ2KtJku+quYD5k5MHxMKMpIodV5FSTCHsdJquPfw2usblVYOuhzRL5rGNNzxcXb5qpu7opokWdxQNlu\\\\\\\/b0hAxS6P8xgrsW2QdsamJPSMVCDcq3LNxmfZ0RrfBRhu\\\\\\\/h07jy8qlLN1QiUGM05Eqsc9y1vwXJk9NBIAxSGcYbPpJ2H1JkTSof2Lo43LGoJa+fnxTRV1oF+nkxquCCN6\\\\\\\/qoqOQyX8T9lwNt50+QIooGm4GuyDWv84za4Od067+\\\\\\\/NpW4MkVj+fq+bd9bja2fz\\\\\\\/EU6aZffbwHaZFpXvasFcQnZE8Pa2Zd1nkK70T\\\\\\\/1EPjR5vWyK+ZPguT34DeUkHJFquQmUXY2KWrJEKZU\\\\\\\/2GW1Dr5D7skPiiJ1EUj9lkHx27eKhiF+Y+R\\\\\\\/HDWwbvdDmRw5rPHd+JzFdtr9PNCYe9ykOMkyleD1uDglmrt0EZbD1FWaoA6f8PIq5GM1EEq8UvRXCQqbve4+7w9Rxwm9Uy204xSViFXrfQXInHHjT\\\\\\\/A==\\\u0022,\\\u0022iv\\\u0022:\\\u0022OiJO3\\\\\\\/ZBDtqja2WS2BNxFA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f81559a81b5451a496d7f47a80097b75c4bd9e009d23544bc72048aec12d3e35\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022xtqHAjkUM7BhO+2LwGcnToukpJ7AQ0pCVUTy+qdW6QAvysp7TeOAzFXAqWXx0dQN0YhsBattvJeP7c1nc0HkXznVsxNMiKH2Lh2roIwGYFuGHZMdxug+PQp12B4ukQK91w1CKp9v9qNyn1cXfSADY8HzEBAuhf6Qy0ogSUYaQSpzaiSJf2LcKyOr5iQs9ZsoGOIr0MA3fHp4t34JNB28loFr8dUMjyot0XWc942XNkHE1oUEqzhsQOrwsjBW9h3bSl3HuqNRc5+Ei9BlxVbUjSirLdMPeR8\\\\\\\/ttkfL30aIZhGMJv5NyeXLD4AdadHULOMTEBFazOE0QR2qc5+fp1HfsGf9CfVYLG34RBG4I9aKyCQlu0zjZx1z4fSTH0Sutw9\\\\\\\/A8okebaRc21\\\\\\\/BjrBu\\\\\\\/ibS0APKJOQSMPsal56Ni0XQJeWTMtQmb0v5HT+P2lh1uqvf3LxknyfG9B2W81tz4kLHPaXYDvUbP27K2Zc+0PMghct+yYYqOpVfXOuUE5Ih1ZEzbzG0OKvyYUidzhIzguwGR4INSB6nyJTU+hkIIANgrwoKdF+NCXvxiA2lR75RPo04lhW9gWn4ELZsuTFjgoecZxasevcRmbBHsXCwI24EBEfdsFFXgIsdgdTd6IOJI2JMtRpAIjXapS9WG90RGMxl90ph20Mdm0+VY37UplA8qC6q8IxQS1sLIhCIctCdTcHGjqNRmoM1igYBYRw7xA7+ioAEzvhyR4gCIMo3UA2pSAfDoi0Nv0\\\\\\\/\\\\\\\/k44I\\\\\\\/0tf16Y9Jdamt6uX4JabpEDrf5D44ynE2zMwyYH85TJF30RrXxMN8QI3OnOKbyRtBbFtQNb8XCXOo6eOwkPqMHnT+H0A2eeaEgugDJJM3MXQuP9MZhOsIVQbMt2CdwQ40tIUb7ptskAQ7TyBA89snez\\\\\\\/0XUq3AQOcfGOtkASY5Ers2I75p8Iz7jBe4rLZy5N8gwm46\\\\\\\/4Df0+YHRjQmD2yO4EsNca6oBcvAQgHvmL8z+YnVV4pYnHVHa5AZrfK1b6kmsQQeh1SPZlN6Rl7gzfzixh7U6oaV\\\\\\\/SXYe6XDSjQdUUrM964sPYex2Bg3fbC8WPK5b9AR28FoKNrLhmr4jjXkFLyeX7S1i9\\\\\\\/DuCOkvyBVeqgAIWHmhYfp1zn1soiYEw65FvLlk\\\\\\\/Uh+L1XOBTnX9P97oTvyXdHljP6YYwhhg67NhWZXK3Or3cWILzx+aeBDmg90LS+NF+A2DQdl\\\\\\\/FgBRWVfC7JMUHvCb55nxO77D+fW+KJToJkFRL7A3FsyAUZPQdJ+AndPq2xXED9VDD\\\\\\\/huTKPnQcgt6UVdu4MfI5MB3bBVVQ65V2l6A4m1LWKyzEuBbzUxZPx4c0VtWqtyH6xyZCHK2w1xO29KewLMKmC9kKIPWj4aW9q1vm7WBVKW3Ska9tAr0y\\\\\\\/KTglDagJFmFeq02pTh5JvIXQ2lYSjWdeSNGMgiaUHNI4l\\\\\\\/2Szhezb7Q7RRnsunwSPMqqXynz39hifqvP+A1QfD59TEijcJumSZ3iJQCXW3zjLTsQYZzzHk61QAK+XJqB\\\\\\\/EIXOfe1k5Rdq+4R5t3bHlIn1KhiRisiRxk2y3dbXD0iRabRnh9H+G\\\\\\\/sIIq2OthUxZ6\\\\\\\/XIAxHDFD5wZmcYJIeFM1PXQCcOgbPIWLlPeVVoGEyHbHwChVRJizx7wMRUMWyll8gz9k+qim32AVaLLBoOB13Yw2A5OgcHgXoU2LPz4+jKpzcvMauvzmBPTQcgY1IXpdqNWB2nPgBou9sHiBqEXn06EcGjKIxsLIQsAeIL3XK4HSTPOB4NVrLsc9tpfG1yP6k4+krjzLvNlCbtFkqiwsyiL+Z7nlmhmNUGtfYlyrZu2mwJ1R1+Q9yKE5Mv4zNazBdtq5V3HvzhpVipN8YSBpP2m7K4rDbSnOJ7r680A4uWA39zNXIAel\\\\\\\/06rQxAyt525Xo9gVBkWjBvFzeuSAbeDS7I2\\\\\\\/i+tS7ex6ln+VzE\\\\\\\/KEa\\\\\\\/N+cDM4mDXv7hhqiMzz5cfwElpE1bE5X3cBnjA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022oiIi\\\\\\\/JwZijfzfdlk9umclA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002269f4415e1110c80b6510b154f751c1e5dec30fa24e748911b4215769db0878f4\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459845 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/5/921" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-4YIPdOeKMt3hGNETegnQF6gYz_WoiJT4B1C793dJz-w" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/5/921" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-dSvDhdQYgxKkCfLmkXpbB0k9Y4H63FrsHyFPbGeWVh0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/5/911" title="A Powerful New Quantitative Genetics Platform, Combining Caenorhabditis elegans High-Throughput Fitness Assays with a Large Collection of Recombinant Strains" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/5/931" title="Mapping Bias Overestimates Reference Allele Frequencies at the HLA Genes in the 1000 Genomes Project Phase I Data" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459845" id="node459845" data-pisa="ggg;5/5/921" data-pisa-master="ggg;g3.115.017079" data-apath="/ggg/5/5/921.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Charged/Polar-Residue Scanning of the Hydrophobic Face of Transmembrane Domain 9 of the Yeast Glutathione Transporter, Hgt1p, Reveals a Conformationally Critical Region for Substrate Transport</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Anil Thakur</span> and <span class="highwire-citation-author" data-delta="1">Anand K. Bachhawat</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">921-929;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.017079 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459845" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Anil Thakur </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Microbial Technology (IMTECH), Sector 39-A Chandigarh 160036, India</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Indian Institute of Science Education & Research (IISER) Mohali, Knowledge City, S.A.S. Nagar, Punjab 140306, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anil%2BThakur%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thakur%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnil%2BThakur%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Anand K. Bachhawat </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Microbial Technology (IMTECH), Sector 39-A Chandigarh 160036, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anand%2BK.%2BBachhawat%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bachhawat%20AK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAnand%2BK.%2BBachhawat%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:anand@iisermohali.ac.in" class="" data-icon-position="" data-hide-link-title="0">anand@iisermohali.ac.in</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/5/921" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459845" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459845/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/5/921.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459845" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459845/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/5/921.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459845" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459845/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/5/921.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459845" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459845/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/5/921.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459845" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459845/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Unraveling the mechanistic workings of membrane transporters has remained a challenging task. We describe a novel strategy that involves subjecting the residues of the hydrophobic face of a transmembrane helix to a charged/polar scanning mutagenesis. TMD9 of the yeast glutathione transporter, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003748">Hgt1p</a>, has been identified as being important in substrate binding, and two residues, F523 and Q526, are expected to line the substrate translocation channel while the other face is hydrophobic. The hydrophobic face of TMD9 helix consists of residues A509, V513, L517, L520, I524, and I528, and these were mutated to lysine, glutamine, and glutamic acid. Among the 16 charged mutants created, six were nonfunctional, revealing a surprising tolerance of charged residues in the hydrophobic part of TM helices. Furthermore, the only position that did not tolerate any charged residue was I524, proximal to the substrate binding residues. However, P525, also proximal to the substrate binding residues, did tolerate charged/polar residues, suggesting that mere proximity to the substrate binding residues was not the only factor. The I524K/E/Q mutants expressed well and localized correctly despite lacking any glutathione uptake capability. Isolation of suppressors for all nonfunctional mutants yielded second-site suppressors only for I524K and I524Q, and suppressors for these mutations appeared at G202K/I and G202K/Q, respectively. G202 is in the hydrophilic loop between TMD3 and TMD4. The results suggest that I524 in the hydrophobic face interacts with this region and is also in a conformationally critical region for substrate translocation.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/glutathione-transporter" class="hw-term hw-article-keyword hw-article-keyword-glutathione-transporter" rel="nofollow">glutathione transporter</a></li><li class="kwd"><a href="/keyword/oligopeptide-transporter-family" class="hw-term hw-article-keyword hw-article-keyword-oligopeptide-transporter-family" rel="nofollow">oligopeptide transporter family</a></li><li class="kwd"><a href="/keyword/charged-residue-scanning" class="hw-term hw-article-keyword hw-article-keyword-charged-residue-scanning" rel="nofollow">charged residue scanning</a></li><li class="kwd"><a href="/keyword/tmds" class="hw-term hw-article-keyword hw-article-keyword-tmds" rel="nofollow">TMDs</a></li><li class="kwd"><a href="/keyword/hydrophobic-face" class="hw-term hw-article-keyword hw-article-keyword-hydrophobic-face" rel="nofollow">hydrophobic face</a></li></ul><p id="p-4">Despite the significant progress in structure determination of membrane transporters, elucidating the dynamics of substrate movement through a transporter or a channel has remained a challenge (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Elvington and Maduke 2008</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Law <em>et al.</em> 2008</a>). Crystal structures provide a static snapshot of the membrane transporter, but the substantial conformational changes that occur during substrate movement are far more difficult to capture. This latter aspect requires additional biochemical, genetic, or spectroscopic inputs to provide insights into the nature of the conformational changes (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Elvington and Maduke 2008</a>).</p><p id="p-5">Over the years, a number of genetic and biochemical techniques have been exploited to obtain insights into the mechanistic workings of membrane transporters. These approaches have provided key support for our understanding, such as in the "alternating access model" of substrate transport seen in LacY permease and other transporters (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abramson <em>et al.</em> 2003</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Guan and Kaback 2006</a>; <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Law <em>et al.</em> 2008</a>). The primary focus of these approaches, however, has been to target the residues involved in substrate binding and substrate access. An aspect that has lagged behind in these studies is the nature of the conformational changes in regions not directly involved in substrate access, and how these other global changes in conformation in the protein contribute toward the transport process.</p><p id="p-6">Here, we have attempted to partially address this lacuna and describe an approach we used to perform “charged/polar residue scanning” of the hydrophobic face of a transmembrane helix of a membrane transporter; we followed this with detailed biochemical and genetic analysis. <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003748">Hgt1p</a> is a high-affinity glutathione transporter of the yeast <em>Saccharomyces cerevisiae</em> (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bourbouloux <em>et al.</em> 2000</a>) that belongs to the relatively uncharacterized Oligopeptide Transporter family (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gomolplitinant and Saier 2011</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hauser <em>et al.</em> 2000</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Lubkowitz 2011</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Lubkowitz <em>et al.</em> 1997</a>). Homologs in other yeasts have also been shown to function in glutathione transport (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bachhawat <em>et al.</em> 2013</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Desai <em>et al.</em> 2011</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Thakur and Bachhawat 2010</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Thakur and Bachhawat 2013</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Thakur <em>et al.</em> 2008</a>) and in some cases oligopeptide transport (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Osawa <em>et al.</em> 2006</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Reuss and Morschhauser 2006</a>), whereas more remote members are transporters of metal–amino acid conjugates (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Curie <em>et al.</em> 2009</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Curie <em>et al.</em> 2001</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Inoue <em>et al.</em> 2009</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Yen <em>et al.</em> 2001</a>). The <em>S. cerevisiae</em> paralogue, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006398">OPT2</a>, also appears to have a role in glutathione homeostasis while functioning at the yeast peroxisomes (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Elbaz-Alon <em>et al.</em> 2014</a>).Very little structure–function information is available for members of this family, and the lack of functionality of a cysteine-free mutant has made it difficult to apply methods, such as the Substituted Cysteine Accessibility Method (SCAM) (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Frillingos <em>et al.</em> 1998</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Kaur <em>et al.</em> 2009</a>). The TMD9 of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003748">Hgt1p</a> has been identified as being important in substrate binding, and two key residues, F523 and Q526, are thought to line the channel on the hydrophilic face of the helix (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Kaur and Bachhawat 2009a</a>; <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Thakur and Bachhawat 2010</a>).On the basis of the helical wheel arrangement of TMD9, it appears that the hydrophobic side of the helix might interact with the lipid matrix and/or other transmembrane segments of the protein. Residues of this face of the TMD9 helix were systematically replaced with lysine, glutamine, and glutamic acid. These replacements in the hydrophobic patch are expected to be deleterious to the protein and would then be subjected to mutagenesis to identify functional suppressors in other parts of the protein, with the premise that critical interaction disrupted by the primary mutation would be compensated by mutations at second sites.</p><p id="p-7">Among the different charged mutants created, only six were nonfunctional, revealing a surprising tolerance of charged residues in the hydrophobic part of TM helices. I524, proximal to the substrate binding residues, was the only position that did not tolerate any charged residues. Suppressor analysis of all the nonfunctional mutants yielded second-site suppressors only in the case of I524K and I524Q, both of which involved a G202Q, G202K, or G202I substitution in the hydrophilic loop of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003748">Hgt1p</a> between TMD3 and TMD4, but G202Q/K/I alone was not deficient in transport activity. Charged and polar residue mutagenesis of P525, another residue close to the substrate binding residues, revealed that mere proximity to these residues was not responsible for the observations with I524. The results suggest that I524 in the hydrophobic face is in a conformationally critical region for substrate translocation and requires the involvement and possible interaction with region G202, near TMD3.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Chemicals and reagents</h3><p id="p-8">All the chemicals used in this study were analytical grade and obtained from commercial sources. Media components were purchased from Difco (Detroit, MI) Sigma Aldrich, (St. Louis, MO), HiMedia, (Mumbai, India), Merck India Ltd (Mumbai, India), and USB Corporation (Cleveland, OH). Oligonucleotides were purchased from Sigma India. Restriction enzymes, Vent DNA polymerase, and other DNA-modifying enzymes were obtained from New England Biolabs (Beverly, MA). DNA sequencing kit (ABI PRISM 310 XL with dye termination cycle sequencing ready reaction kit) was obtained from Perkin Elmer, (Norwalk, CT). Gel-extraction kits and plasmid miniprep columns were obtained from QIAGEN (Valencia, CA) or Sigma (St. Louis, MO). [<sup>35</sup>S] GSH (specific activity 1000 Ci mmol<sup>-1</sup>) was purchased from Bhabha Atomic Research Centre, Mumbai, India. HA-Tag (6E2) mouse monoclonal antibody and horse anti-mouse HRP-linked antibody were bought from Cell Signaling (Danvers, MA). Alexa Flour 488 conjugated goat anti-mouse antibody was obtained from Molecular Probes (Eugene, OR). Hybond ECL (nitrocellulose) membrane and ECL plus Western blotting detection reagents were purchased from Amersham Biosciences (UK).</p></div><div id="sec-3" class="subsection"><h3>Strains, media, and growth conditions</h3><p id="p-9">The <em>Escherichia coli</em> strain DH5α was used as a cloning host. The <em>Saccharomyces cerevisiae</em> strain used in this study was ABC 817 <em>(MATa <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">his3</a>Δ1 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000523">leu2</a>Δ0 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004294">met15</a>Δ</em>-0 <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">ura3</a>Δ0 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003748">hgt1</a>Δ</em>::<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000523">LEU2</a>)</em> (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Bourbouloux <em>et al.</em> 2000</a>). <em>S. cerevisiae</em> was regularly maintained on yeast extract, peptone, and dextrose (YPD) medium. <em>S. cerevisiae</em> synthetic defined minimal medium (SD) contained yeast nitrogen base, ammonium sulfate, and dextrose supplemented with histidine, leucine, and methionine (when required) at 50 mg/liter (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Guthrie and Fink 1991</a>). Glutathione was added as required. Growth, handling of bacteria and yeast, and all the molecular techniques used in the study were according to standard protocols (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Sambrook <em>et al.</em> 1989</a>).</p></div><div id="sec-4" class="subsection"><h3>Site-directed mutagenesis</h3><p id="p-10"><em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003748">HGT1</a></em>, tagged with a Hemagglutinin (HA) tag at the C-terminus, was subcloned downstream of the TEF promoter at the <em>Bam</em> HI and <em>Eco</em> RI sites of a modified p416TEF vector (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Kaur <em>et al.</em> 2009</a>). This construct was used as a template for site-directed mutagenesis for creation of different site-directed mutants of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003748">Hgt1p</a> by splice overlap extension strategy. The mutations, K,Q,E, for each residue were generated using a single mutagenic oligonucleotide exploiting degenerate base pairs at the desired position in the different mutagenic oligonucleotides (<a href="http://www.g3journal.org/content/suppl/2015/03/16/g3.115.017079.DC1/017079SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2015/03/16/g3.115.017079.DC1/TableS1.pdf">Table S1</a>). The PCR products generated with these oligonucleotides were subcloned back into the TEF vector background using appropriate restriction sites for subsequent analyses. The resulting mutants were sequenced to confirm the presence of the desired nucleotides changes and to rule out any undesired mutations introduced during the mutagenic procedure.</p></div><div id="sec-5" class="subsection"><h3>The dual complementation-cum-toxicity plate assay for assessing <em>HGT1</em> functionality</h3><p id="p-11">The complementation-cum-toxicity assay has been described previously (<a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Kaur <em>et al.</em> 2009</a>). The <em>S. cerevisiae <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004294">met15</a></em>Δ <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003748">hgt1</a></em>Δ (ABC 817) was transformed with a single-copy centromeric vector expressing wild-type or different mutants of TMD9 of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003748">HGT1</a></em> expressed downstream of the TEF promoter. Transformants were grown in minimal media containing methionine and other supplements, without uracil, overnight. These cultures were reinoculated in the same media and allowed to grow until they reached the exponential phase. An equal number of cells were harvested, washed with water, and resuspended in sterile water to an OD600 of 0.2. These were serially diluted 1:10, 1:100, and 1:1000; 10 μl of these cell resuspensions were spotted on minimal medium containing different concentrations of glutathione (15, 30, 50, 100, 150, 200 μM) or methionine (200 μM) as sole organic sulfur source. Because <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003748">HGT1</a></em> is under the strong TEF promoter, it is able to confer the ability to grow on glutathione at low concentrations of glutathione. However, at higher concentrations of glutathione, excessive uptake leads to toxicity in growth. Using this growth at low concentrations, and lack of it at higher concentrations, we have been able to grade the transporter defects, which is more informative than a mere growth assay at lower concentrations. The plates were incubated at 30° for 2–3 d and photographs were taken.</p><div id="sec-6" class="subsection"><h4>Hydroxylamine mutagenesis of the pTEF-HGT1:</h4><p id="p-12">The random <em>in vitro</em> mutagenesis of the charged/polar residues of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003748">HGT1</a></em> was done using hydroxylamine and a previously described protocol (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Rose and Fink 1987</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Thakur and Bachhawat 2013</a>). The 10 μg plasmid DNA was incubated in 0.5 ml of hydroxylamine solution (90 mg NaOH, 350 mg hydroxylamine HCl in 5 ml water, pH ∼6.5, freshly made before use). This mixture was incubated at 37° for 24 hr and mutagenized DNA was directly purified using Qiagen column. The purified plasmid DNA concentration was measured by running on agrose gel along with DNA ladder. The randomly mutagenized plasmid was directly transformed in <em>S. cerevisiae <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004294">met15</a>Δ <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003748">hgt1</a>Δ</em> strain and selected on 10 µg glutathione containing plates, and functional transformants were used to prepare plasmid DNA following a retransformation.</p></div></div><div id="sec-7" class="subsection"><h3>Glutathione transport assay</h3><p id="p-13">The <em>S. cerevisiae</em> ABC 817 strain (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004294">met15</a>Δ<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003748">hgt1</a>Δ</em>) was transformed with different plasmid constructs bearing wild-type or <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003748">HGT1</a></em> mutants under TEF promoter and was grown in minimal media containing methionine and other supplements, without uracil, overnight. These cultures were reinoculated in the same media and allowed to grow until they reached the exponential phase. Cells were harvested, washed, and put on ice in a MES-buffered medium until the transport was initiated. Transport experiments were carried out with [35<sup>S</sup>]-GSH as described earlier (<a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">Kaur <em>et al.</em> 2009</a>). The results were expressed as nmol of glutathione-mg.protein-1 min-1. For the measurements of total protein, the 100 μl of the above cell suspension (cell suspension volume used for the transport assay) was boiled with 15% sodium hydroxide for 10 min, followed by neutralization of total cell lysate by addition of hydrochloric acid; 100 μl of this crude cell lysate was incubated with 0.1% Triton X-100 for 10 min and total protein was estimated by using the Bradford reagent (Sigma) using bovine serum albumin as a standard. For saturation kinetics, the initial rate of glutathione uptake was measured at a range of glutathione concentrations from 12.5 μM to 800 μM, with specific activity being kept constant at each concentration. The initial rate of glutathione uptake was determined by measuring the radioactive glutathione accumulated in the cells at 30-sec and 180-sec time points in the ABC 817 strain transformed with different plasmids constructs bearing wild-type and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003748">HGT1</a></em> mutants under TEF promoter or only vector. After subtracting the initial rate of glutathione uptake in vector from the initial rate of glutathione uptake in the different test constructs, the extent of glutathione uptake corresponding to the test construct was determined.</p></div><div id="sec-8" class="subsection"><h3>Preparation of cell extract and Western blot analysis</h3><p id="p-14">Crude cell extracts were prepared and Western blot analysis was done as described previously (<a id="xref-ref-21-5" class="xref-bibr" href="#ref-21">Kaur <em>et al.</em> 2009</a>) using a modified Western blot method (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Kaur and Bachhawat 2009b</a>). Densitometry analysis of the unsaturated band signals was performed using the Scion Image software to quantify the protein expression levels in the different mutants. The resulting signal intensity was normalized with respect to the band surface area (in square pixels) and expressed in arbitrary units. The relative protein expression levels in the mutant <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003748">Hgt1p</a> were represented as percentage expression relative to wild-type <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003748">Hgt1p</a>.</p></div><div id="sec-9" class="subsection"><h3>Cellular localization of the mutants by confocal microscopy</h3><p id="p-15">To localize the different charged residue mutants of TMD9 of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003748">Hgt1p</a> that were created, indirect immunofluorescence was performed using a published protocol modified as described earlier (<a id="xref-ref-21-6" class="xref-bibr" href="#ref-21">Kaur <em>et al.</em> 2009</a>). For staining of ER (endoplasmic reticulum), live yeast cells were incubated with the ER-Tracker TM Red dye (BODIPY TR glibenclamide; Invitrogen, USA) according to the manufacturer’s instructions and by other published literatures (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Feng <em>et al.</em> 2010</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Mandal <em>et al.</em> 2012</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Rawal <em>et al.</em> 2013</a>). Images were obtained with an inverted LSM510 META laser scanning confocal microscope (Carl Zeiss) fitted with a Plan-Apochromat ×100 (numerical aperture, 1.4) oil immersion objective. The 488-nm line of an argon ion laser was directed over an HFT UV/488 beam splitter, and fluorescence was detected using an NFT 490 beam splitter in combination with a BP 505 to 530 band pass filter. ER-Tracker fluorescence was detected at Ex/Em 587/615. Images obtained were processed using Adobe Photoshop version 5.5</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>Analysis of the residues of the hydrophobic face of the TMD9 helix by mutation to charged residues</h3><p id="p-16">The helical wheel diagram of TMD9 shows a topological sidedness. The residues F523 and Q526, shown to be important in substrate binding, are located on the more polar face of the channel and are thought to directly interact with the substrate (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Kaur and Bachhawat 2009a</a>; <a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Thakur and Bachhawat 2010</a>). The hydrophobic face of the helix, in contrast, is expected to interact with the lipid bilayer or with other TMDs. These residues in TMD9 are A509, V513, L517, L520, I524, and I528. To investigate this face and these residues more systematically, we considered a novel strategy. In this approach, each of the residues on this face was mutated to multiple charged residues, lysine, glutamine, or glutamic acid (using a single degenerate oligonucleotide), and nonfunctional mutants would then be subjected to suppressor analysis to obtain further insights (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1, A and B</a>). The residues described above were mutated as indicated and the mutants were subjected to an initial functional characterization using the previously designed sensitive plate assay, termed a complementation-cum-toxicity assay (<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Kaur and Bachhawat 2009a</a>; <a id="xref-ref-21-7" class="xref-bibr" href="#ref-21">Kaur <em>et al.</em> 2009</a>). The assay is based on the ability of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003748">HGT1</a></em> when expressed from the TEF promoter to permit growth on glutathione as a sole sulfur source in an <em>S. cerevisiae</em> strain (ABC 817). This strain is an organic sulfur auxotroph defective in glutathione uptake. At low glutathione concentrations (15 µM), <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003748">HGT1</a></em> expressed under the TEF promoter complements the growth defect in the ABC 817 strain (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Bourbouloux <em>et al.</em> 2000</a>) but results in toxicity when cells are grown in medium containing 50-µM or higher glutathione concentrations (<a id="xref-ref-21-8" class="xref-bibr" href="#ref-21">Kaur <em>et al.</em> 2009</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Srikanth <em>et al.</em> 2005</a>). The reasons for this toxicity have been examined elsewhere (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Kumar <em>et al.</em> 2011</a>), but the dual assay permits one to discern a gradation in functionality. Plasmids bearing either the WT <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003748">HGT1</a></em> or the different mutants as described above were individually transformed into the <em>S. cerevisiae</em> ABC 817 strain, and the transformants were analyzed for their ability to confer complementation and/or toxicity to the cells over a range of glutathione concentrations.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/5/921/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Residues of TMD9 selected for charged residue scanning. (A) The side view of the helix (shown in green). The residues selected for charged residue scanning indicated by their position. The side view of the TMDs was drawn using the PyMol Molecular viewer (version 0.99). The blue boxes represent residues involved in substrate binding. (B) Helical wheel representation of helix 9 of Hgt1p viewed from the exoplasmic surface of the membrane. Amino acid representation by the single letter code. The red arrow points to the residues selected for charged residue scanning situated on the hydrophobic face of the respective transmembrane domains. Helical wheel model of the transmembrane helix 9 of Hgt1p was constructed using the Lasergene software (DNAstar, Madison, WI)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-810909461" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Residues of TMD9 selected for charged residue scanning. (A) The side view of the helix (shown in green). The residues selected for charged residue scanning indicated by their position. The side view of the TMDs was drawn using the PyMol Molecular viewer (version 0.99). The blue boxes represent residues involved in substrate binding. (B) Helical wheel representation of helix 9 of Hgt1p viewed from the exoplasmic surface of the membrane. Amino acid representation by the single letter code. The red arrow points to the residues selected for charged residue scanning situated on the hydrophobic face of the respective transmembrane domains. Helical wheel model of the transmembrane helix 9 of Hgt1p was constructed using the Lasergene software (DNAstar, Madison, WI).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/5/921/F1.medium.gif" width="440" height="269"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/5/921/F1.medium.gif" width="440" height="269"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/5/921/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/5/921/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466112" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-17" class="first-child">Residues of TMD9 selected for charged residue scanning. (A) The side view of the helix (shown in green). The residues selected for charged residue scanning indicated by their position. The side view of the TMDs was drawn using the PyMol Molecular viewer (version 0.99). The blue boxes represent residues involved in substrate binding. (B) Helical wheel representation of helix 9 of Hgt1p viewed from the exoplasmic surface of the membrane. Amino acid representation by the single letter code. The red arrow points to the residues selected for charged residue scanning situated on the hydrophobic face of the respective transmembrane domains. Helical wheel model of the transmembrane helix 9 of Hgt1p was constructed using the Lasergene software (DNAstar, Madison, WI).</p><div class="sb-div caption-clear"></div></div></div><p id="p-18">Based on their inability to complement the glutathione transporter defect, the mutants L517Q, L520Q, L520E, I524E, I524K, and I524Q were nonfunctional. V513K was severely defective but still mildly functional. The remaining mutants (A509Q, A509K, A509E, V513Q, V513E, L517K, L517E, L520K, and I528E) surprisingly had no functional defect and were comparable to WT (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a> and <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). In case of I528, only I528E was constructed, because it was the only mutant plasmid recovered among the 20 different mutants of I528 that we analyzed.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/5/921/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Growth pattern of charged/polar residue mutants of transmembrane domain 9 of Hgt1p when grown on different concentrations of glutathione. Hgt1p and the different charged/polar mutants of TMD9 of Hgt1p expressed under the TEF promoter, and corresponding vector (p416TEF) were transformed into the S. cerevisiae strain ABC817 and evaluated by the complementation-cum-toxicity assay by dilution spotting on minimal media containing glutathione. Transformants were grown in minimal medium containing methionine, harvested, washed, and resuspended in water and serially diluted to give 0.2, 0.02, 0.002, and 0.0002 OD600 of cells; 10 &#x3BC;l of these dilutions was spotted on minimal medium containing different concentrations of glutathione (GSH). The photographs were taken after 3 d of incubation at 30&#xB0;. (B) Relative rate of glutathione uptake in selected mutants of TMD9 mutated to charged residues. Hgt1p and the different mutants of TMD9 of Hgt1p expressed under the TEF promoter were transformed into strain met15&#x394; hgt1&#x394; and percentage transport activity was determined relative to wild-type Hgt1p as described in the experimental procedures." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-810909461" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(A) Growth pattern of charged/polar residue mutants of transmembrane domain 9 of Hgt1p when grown on different concentrations of glutathione. Hgt1p and the different charged/polar mutants of TMD9 of Hgt1p expressed under the TEF promoter, and corresponding vector (p416TEF) were transformed into the &lt;em&gt;S. cerevisiae&lt;/em&gt; strain ABC817 and evaluated by the complementation-cum-toxicity assay by dilution spotting on minimal media containing glutathione. Transformants were grown in minimal medium containing methionine, harvested, washed, and resuspended in water and serially diluted to give 0.2, 0.02, 0.002, and 0.0002 OD&lt;sub&gt;600&lt;/sub&gt; of cells; 10 &#x3BC;l of these dilutions was spotted on minimal medium containing different concentrations of glutathione (GSH). The photographs were taken after 3 d of incubation at 30&#xB0;. (B) Relative rate of glutathione uptake in selected mutants of TMD9 mutated to charged residues. Hgt1p and the different mutants of TMD9 of Hgt1p expressed under the TEF promoter were transformed into strain &lt;em&gt;met15&#x394; hgt1&#x394;&lt;/em&gt; and percentage transport activity was determined relative to wild-type Hgt1p as described in the experimental procedures.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/5/921/F2.medium.gif" width="222" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/5/921/F2.medium.gif" width="222" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/5/921/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/5/921/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466143" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-19" class="first-child">(A) Growth pattern of charged/polar residue mutants of transmembrane domain 9 of Hgt1p when grown on different concentrations of glutathione. Hgt1p and the different charged/polar mutants of TMD9 of Hgt1p expressed under the TEF promoter, and corresponding vector (p416TEF) were transformed into the <em>S. cerevisiae</em> strain ABC817 and evaluated by the complementation-cum-toxicity assay by dilution spotting on minimal media containing glutathione. Transformants were grown in minimal medium containing methionine, harvested, washed, and resuspended in water and serially diluted to give 0.2, 0.02, 0.002, and 0.0002 OD<sub>600</sub> of cells; 10 μl of these dilutions was spotted on minimal medium containing different concentrations of glutathione (GSH). The photographs were taken after 3 d of incubation at 30°. (B) Relative rate of glutathione uptake in selected mutants of TMD9 mutated to charged residues. Hgt1p and the different mutants of TMD9 of Hgt1p expressed under the TEF promoter were transformed into strain <em>met15Δ hgt1Δ</em> and percentage transport activity was determined relative to wild-type Hgt1p as described in the experimental procedures.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466016/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466016/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Summary of the charged residues mutants of <em>HGT1</em></span><div class="sb-div caption-clear"></div></div></div><p id="p-21">To corroborate these growth assays with biochemical assays, the mutants that were nonfunctional and severely defective were evaluated by measuring the rate of <sup>35</sup>S-GSH uptake in the ABC 817 strain. The uptake data correlated well with the plate assay. Among the defective mutants, V513Q displayed some transport activity (∼18%) in agreement with the plate assay. However, the others mutants L517Q, L520Q, L520E, I524E, I524Q, and I524K had an almost complete loss in functional activity, showing very minimal uptake ability in agreement with the plate assay (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>)</p></div><div id="sec-12" class="subsection"><h3>Expression and cell surface targeting of nonfunctional mutants</h3><p id="p-22">To determine whether the nonfunctional mutant proteins are expressed and targeted properly, we first examined their steady-state expression levels by immunoblotting. Equal amounts of the crude protein extracts prepared from the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004294">met15</a>∆<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003748">hgt1</a>∆</em> strain transformed with the different mutants were loaded onto the gel, electroblotted to the membrane, and probed with anti-HA monoclonal antibody. L520Q and L520E showed almost complete loss in protein expression level. V513K and L517Q mutant showed detectable, but significantly lower, expression, whereas the other mutants I524K, I524Q, and I524E showed slightly lower, but otherwise comparable, levels to WT, and their expression ranged between 65% and 92% of wild-type (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/5/921/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Quantification of the total protein expression levels of the different mutants of Hgt1p. Protein extracts were prepared from the S. cerevisiae strain ABC817 transformed with plasmid bearing either the WT or the different charged/polar residues mutants of TMD9 of Hgt1p; 20 &#x3BC;g of total protein was loaded onto SDS-PAGE, resolved, and electroblotted to a nitrocellulose membrane. The data are expressed as percentage protein expression normalized to the wild-type expression level and is mean of the protein expression levels obtained in two independent experiments. A representative blot is shown in the inset. Equal loading of the proteins (20 &#x3BC;g) in each well of the gel was also visually monitored by Coomassie staining and Ponceau S staining of the membrane after transfer (data not shown). (B) Cell surface localization of charged/polar residues mutants of Hgt1p lacking functional activity. The S. cerevisiae strain ABC817 was transformed with plasmids bearing either the WT or the different charged/polar residues mutants of TMD9 of Hgt1p and labeled by indirect immunofluorescence using HA monoclonal antibody as a primary antibody, followed by the Alexa 488, and visualized using confocal microscopy, as described in experimental procedure. Only fluorescence images have been shown." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-810909461" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(A) Quantification of the total protein expression levels of the different mutants of Hgt1p. Protein extracts were prepared from the &lt;em&gt;S. cerevisiae&lt;/em&gt; strain ABC817 transformed with plasmid bearing either the WT or the different charged/polar residues mutants of TMD9 of Hgt1p; 20 &#x3BC;g of total protein was loaded onto SDS-PAGE, resolved, and electroblotted to a nitrocellulose membrane. The data are expressed as percentage protein expression normalized to the wild-type expression level and is mean of the protein expression levels obtained in two independent experiments. A representative blot is shown in the inset. Equal loading of the proteins (20 &#x3BC;g) in each well of the gel was also visually monitored by Coomassie staining and Ponceau S staining of the membrane after transfer (data not shown). (B) Cell surface localization of charged/polar residues mutants of Hgt1p lacking functional activity. The &lt;em&gt;S. cerevisiae&lt;/em&gt; strain ABC817 was transformed with plasmids bearing either the WT or the different charged/polar residues mutants of TMD9 of Hgt1p and labeled by indirect immunofluorescence using HA monoclonal antibody as a primary antibody, followed by the Alexa 488, and visualized using confocal microscopy, as described in experimental procedure. Only fluorescence images have been shown.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/5/921/F3.medium.gif" width="440" height="172"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/5/921/F3.medium.gif" width="440" height="172"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/5/921/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/5/921/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466120" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-23" class="first-child">(A) Quantification of the total protein expression levels of the different mutants of Hgt1p. Protein extracts were prepared from the <em>S. cerevisiae</em> strain ABC817 transformed with plasmid bearing either the WT or the different charged/polar residues mutants of TMD9 of Hgt1p; 20 μg of total protein was loaded onto SDS-PAGE, resolved, and electroblotted to a nitrocellulose membrane. The data are expressed as percentage protein expression normalized to the wild-type expression level and is mean of the protein expression levels obtained in two independent experiments. A representative blot is shown in the inset. Equal loading of the proteins (20 μg) in each well of the gel was also visually monitored by Coomassie staining and Ponceau S staining of the membrane after transfer (data not shown). (B) Cell surface localization of charged/polar residues mutants of Hgt1p lacking functional activity. The <em>S. cerevisiae</em> strain ABC817 was transformed with plasmids bearing either the WT or the different charged/polar residues mutants of TMD9 of Hgt1p and labeled by indirect immunofluorescence using HA monoclonal antibody as a primary antibody, followed by the Alexa 488, and visualized using confocal microscopy, as described in experimental procedure. Only fluorescence images have been shown.</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">The nonfunctional mutants were also studied for subcellular localization by indirect immunofluorescence using anti-HA monoclonal antibody. No signal was observed either in L520Q or in L520E, which probably reflects the almost complete loss of protein expression. In V513K, L517Q, I524Q, I524E, and I524K, a signal was observed at the cellular periphery of the cells, although they also showed a small amount of intracellular signal in addition to the cell surface signal (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>, <a href="http://www.g3journal.org/content/suppl/2015/03/16/g3.115.017079.DC1/FigureS1.pdf">Figure S1</a>). This suggested that these mutants did not carry a significant trafficking defect.</p></div><div id="sec-13" class="subsection"><h3>G202K/I and G202K/Q mutations in the loop region between TMD3 and TMD4 are able to restore function to the nonfunctional I524K and I524Q mutants</h3><p id="p-25">Among all the hydrophobic residues that were evaluated by this charged residue scanning, it was only the charged residues at position I524 (I524K, I524K, and I524Q) that could not be tolerated as seen in functional assays. At all other positions, at least one of the charged residues was tolerated (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>).</p><p id="p-26">The loss of function mutants I524K/Q/E were further subjected to suppressor analysis to examine if these mutants that were otherwise properly expressed and localized might be restored to functionality by compensatory mutations. Mutants were mutagenized <em>in vitro</em> with hydroxylamine treatment and transformed into the yeast ABC 817 strain. The resulting transformants were screened for a functional phenotype by direct selection on minimal media containing 10 μM GSH. For those clones bearing the original mutation, the full gene was sequenced to identify any additional mutations. The I524Q mutation yielded eight suppressors, of which two were revertants, whereas the remaining were either G202K or G202Q (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). The I524K mutations yielded 10 suppressors, of which five were revertants, whereas the remaining were either G202K or G202I (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). It is interesting that in all cases it was the G202 that was mutated to different amino acids (G202Q, G202K, and G202I) (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). G202 is located in the intracellular loop between TMD3 and TMD4, but at the junction of TMD3.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466125/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466125/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Suppressor isolated from hydroxylamine mutagenesis–based intragenic suppressor analysis of mutant<em>s</em></span><div class="sb-div caption-clear"></div></div></div><p id="p-27">We also subjected the remaining nonfunctional mutants to this suppressor analysis. The V513K suppressor involved same-site mutations with conversion to 513N and 513Q. Similarly, the L517Q suppressors yielded a mutation at the same site, 517A. In the case of L520Q and L520E, several suppressors were isolated but they all reverted back to the wild-type sequence. Thus, second-site suppressors were obtained only for I524Q and I524K mutations and were found to involve the G202 residue.</p><p id="p-28">To investigate the I524-G202 interaction in greater detail, we created several additional charged mutants of I524 (I524D, I524R, I524N) as well as G202 (G202D, G202R, G202N, G202E) and evaluated the functional interaction of the different combinations (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). Interestingly, the fresh charged residues at the I524 position that we introduced were all nonfunctional, although some partial functionality was observed with the I524N mutant.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466082/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466082/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Genetic analysis of I524 and G202 mutants and interaction involving I524-G202 in functional activity of the Hgt1p using dual complementation-cum-toxicity assay</span><div class="sb-div caption-clear"></div></div></div><p id="p-30">The interaction of these various combinations as measured by the growth assay is summarized in <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>. Among the various G202 mutations that were made, only G202R and G202Q were able to suppress the I524Q mutation in a manner similar to G202K and suggested that the charge or polarity at this position was critical for suppression. However, in contrast, the I524R mutant (unlike the I524K mutant) could not be suppressed by G202K. Disruption of the activity by this mutation was therefore causing other defects in the functionality. None of the other mutations that included negatively charged residues could restore functionality in these interactions.</p></div><div id="sec-14" class="subsection"><h3>Functional and kinetic analysis of the I524K and I524Q suppressors</h3><p id="p-31">To evaluate the suppressors in greater detail, the mutants I524K G202K, I524K G202I, I524K G202Q, I524Q G202K, and I524Q G202Q were spotted on different concentrations of glutathione and evaluated by the dual complementation-cum-toxicity assay. The suppressors G202K, G202Q, and G202I with their primary mutation I524K and I524Q were able to suppress the phenotypic defect caused by I524Q and I524K in terms of their ability to complement growth on glutathione. All these suppressors exhibited a functional activity almost similar to wild-type in being able to complement at lower concentrations of glutathione. However, the suppressors displayed a slight loss in functional activity in terms of their ability to confer toxicity to the cells at high concentrations of glutathione (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). To corroborate the growth assays with actual transport data, we measured the initial rate of <sup>35</sup>S-glutathione uptake in the ABC 817 strain transformed with the suppressor mutants of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003748">Hgt1p</a>. The uptake was 45–62% as compared to wild-type (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>). These results indicated that the suppressors had regained significant activity.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/5/921/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Growth pattern of I524Q and I524K suppressors. Plasmids bearing the I524KG202K, I524KG202I, I524KG202Q, I524QG202K, and I524QG202Q and the corresponding WT or vector (p416TEF) were transformed into S. cerevisiae strain ABC817 and evaluated by the complementation-cum-toxicity assay by dilution spotting on minimal media containing glutathione. (B) Relative rate of radiolabeled glutathione uptake of I524Q and I524K suppressors. Plasmids bearing the I524K G202K, I524K G202Q and I524K G202I, I524Q G202Q and I524Q G202K, and the corresponding WT or vector (p416TEF) were transformed into S. cerevisiae strain ABC817 and transport rates were determined as described in experimental procedures. The data are represented as percentage rate of glutathione uptake by the suppressors relative to wild-type Hgt1p (WT). (C) Quantification of the total protein expression levels of suppressors of I524Q and I524K. Extracts were prepared from the S. cerevisiae strain ABC817 transformed with plasmids bearing either the WT or the I524Q G202K, I524K G202K, and I524K G202I suppressors, resolved on SDS-PAGE, and electroblotted onto a nitrocellulose membrane. The protein bands were quantified by densitometry scanning. The data are expressed as percentage protein expression normalized to the wild-type expression level and are the mean of the protein expression levels obtained in two independent experiments. A representative blot is also shown. Equal loading of the proteins (20 &#x3BC;g) in each well of the gel was also visually monitored by Coomassie staining and Ponceau S staining of the membrane after transfer (data not shown). (D) Cell surface localization of I524Q and I524K suppressors. The S. cerevisiae strain ABC817 was transformed with plasmids bearing the I524K G202K, I524K G202I, and I524QG202K suppressors of TMD9 of Hgt1p and labeled by indirect immunofluorescence visualized using confocal microscope. Only fluorescence images are shown." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-810909461" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(A) Growth pattern of I524Q and I524K suppressors. Plasmids bearing the I524KG202K, I524KG202I, I524KG202Q, I524QG202K, and I524QG202Q and the corresponding WT or vector (p416TEF) were transformed into &lt;em&gt;S. cerevisiae&lt;/em&gt; strain ABC817 and evaluated by the complementation-cum-toxicity assay by dilution spotting on minimal media containing glutathione. (B) Relative rate of radiolabeled glutathione uptake of I524Q and I524K suppressors. Plasmids bearing the I524K G202K, I524K G202Q and I524K G202I, I524Q G202Q and I524Q G202K, and the corresponding WT or vector (p416TEF) were transformed into &lt;em&gt;S. cerevisiae&lt;/em&gt; strain ABC817 and transport rates were determined as described in experimental procedures. The data are represented as percentage rate of glutathione uptake by the suppressors relative to wild-type Hgt1p (WT). (C) Quantification of the total protein expression levels of suppressors of I524Q and I524K. Extracts were prepared from the &lt;em&gt;S. cerevisiae&lt;/em&gt; strain ABC817 transformed with plasmids bearing either the WT or the I524Q G202K, I524K G202K, and I524K G202I suppressors, resolved on SDS-PAGE, and electroblotted onto a nitrocellulose membrane. The protein bands were quantified by densitometry scanning. The data are expressed as percentage protein expression normalized to the wild-type expression level and are the mean of the protein expression levels obtained in two independent experiments. A representative blot is also shown. Equal loading of the proteins (20 &#x3BC;g) in each well of the gel was also visually monitored by Coomassie staining and Ponceau S staining of the membrane after transfer (data not shown). (D) Cell surface localization of I524Q and I524K suppressors. The &lt;em&gt;S. cerevisiae&lt;/em&gt; strain ABC817 was transformed with plasmids bearing the I524K G202K, I524K G202I, and I524QG202K suppressors of TMD9 of Hgt1p and labeled by indirect immunofluorescence visualized using confocal microscope. Only fluorescence images are shown.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/5/921/F4.medium.gif" width="440" height="348"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/5/921/F4.medium.gif" width="440" height="348"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/5/921/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/5/921/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465967" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-32" class="first-child">(A) Growth pattern of I524Q and I524K suppressors. Plasmids bearing the I524KG202K, I524KG202I, I524KG202Q, I524QG202K, and I524QG202Q and the corresponding WT or vector (p416TEF) were transformed into <em>S. cerevisiae</em> strain ABC817 and evaluated by the complementation-cum-toxicity assay by dilution spotting on minimal media containing glutathione. (B) Relative rate of radiolabeled glutathione uptake of I524Q and I524K suppressors. Plasmids bearing the I524K G202K, I524K G202Q and I524K G202I, I524Q G202Q and I524Q G202K, and the corresponding WT or vector (p416TEF) were transformed into <em>S. cerevisiae</em> strain ABC817 and transport rates were determined as described in experimental procedures. The data are represented as percentage rate of glutathione uptake by the suppressors relative to wild-type Hgt1p (WT). (C) Quantification of the total protein expression levels of suppressors of I524Q and I524K. Extracts were prepared from the <em>S. cerevisiae</em> strain ABC817 transformed with plasmids bearing either the WT or the I524Q G202K, I524K G202K, and I524K G202I suppressors, resolved on SDS-PAGE, and electroblotted onto a nitrocellulose membrane. The protein bands were quantified by densitometry scanning. The data are expressed as percentage protein expression normalized to the wild-type expression level and are the mean of the protein expression levels obtained in two independent experiments. A representative blot is also shown. Equal loading of the proteins (20 μg) in each well of the gel was also visually monitored by Coomassie staining and Ponceau S staining of the membrane after transfer (data not shown). (D) Cell surface localization of I524Q and I524K suppressors. The <em>S. cerevisiae</em> strain ABC817 was transformed with plasmids bearing the I524K G202K, I524K G202I, and I524QG202K suppressors of TMD9 of Hgt1p and labeled by indirect immunofluorescence visualized using confocal microscope. Only fluorescence images are shown.</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">To determine the effect of suppressor mutations in the absence of the primary mutation, the G202Q, G202K, and G202I mutations were subcloned into the wild-type plasmid backbone. The mutants were functionally evaluated using the plate based dual complementation-cum-toxicity assay (<a href="http://www.g3journal.org/content/suppl/2015/03/16/g3.115.017079.DC1/FigureS2.pdf">Figure S2</a>). The individual mutants G202Q G202K and G202I appeared to have an activity similar to wild-type. The protein expression levels of these suppressors with primary mutation were also similar to wild-type and properly localized to the cell surface (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4, C and D</a>).</p><p id="p-34">Because of the low activity of the I524K/E/Q mutations, their kinetics could not be evaluated. However, the restoration of activity by the G202K/I/Q suppressors in these backgrounds provided the opportunity to get some insight into these mutants. We determined the kinetic parameter by measuring the initial rate of glutathione uptake over a range of glutathione concentrations. I524K and I524Q had almost null activity and could not be included for the kinetic study. We determined the kinetic parameters for I524Q G202K. Compared to the <em>Km</em> of the WT 48.1 ± 10.5 μM (<a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Bourbouloux <em>et al.</em> 2000</a>), the <em>Km</em> for the suppressor mutant was significantly higher at 282.8± 44.5 μM (<a href="http://www.g3journal.org/content/suppl/2015/03/16/g3.115.017079.DC1/FigureS3.pdf">Figure S3</a>).</p></div><div id="sec-15" class="subsection"><h3>Charged/polar residue scanning of P525 of TMD9 of Hgt1p</h3><p id="p-35">As I524 was located between the previously identified substrate binding residues F523 and Q526, examination was needed regarding the possibility that the inability of I524 to tolerate charged/polar residues might be due to this proximity to the substrate binding site rather than as a consequence of it being on the hydrophobic face and being involved in interactions critical for the channel functioning. To evaluate this possibility, we decided to examine if the residue, P525 adjacent to I524 and also located between F523 and Q526, could tolerate any charged/polar residues. P525 was mutated to lysine, glutamine, and glutamic acid by site-directed mutagenesis. These mutants were also subjected to an initial functional characterization by growth assays. P525K and P525E showed a severe effect on functional activity of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003748">Hgt1p</a>, although they still retained partial activity, but P525Q had no discernible loss in function and was comparable to WT (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). Thus, polar residues at this position could be tolerated despite the proximity to F523 and Q526.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/5/921/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Functional characterization of P525K, P525Q, and P525E mutants of Hgt1p. The P525K, P525Q, and P525E mutants expressed under the TEF promoter, the WT, and corresponding vector (p416TEF) were transformed into S. cerevisiae strain ABC817 and evaluated by the complementation-cum-toxicity assay by dilution spotting on minimal media containing different concentrations of glutathione." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-810909461" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Functional characterization of P525K, P525Q, and P525E mutants of Hgt1p. The P525K, P525Q, and P525E mutants expressed under the TEF promoter, the WT, and corresponding vector (p416TEF) were transformed into &lt;em&gt;S. cerevisiae&lt;/em&gt; strain ABC817 and evaluated by the complementation-cum-toxicity assay by dilution spotting on minimal media containing different concentrations of glutathione.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/5/921/F5.medium.gif" width="420" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/5/5/921/F5.medium.gif" width="420" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/5/921/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/5/921/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466116" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-36" class="first-child">Functional characterization of P525K, P525Q, and P525E mutants of Hgt1p. The P525K, P525Q, and P525E mutants expressed under the TEF promoter, the WT, and corresponding vector (p416TEF) were transformed into <em>S. cerevisiae</em> strain ABC817 and evaluated by the complementation-cum-toxicity assay by dilution spotting on minimal media containing different concentrations of glutathione.</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">The severely affected mutants P525K and P525E were also subjected to suppressor analysis. Suppressors were screened for a functional phenotype by direct selection on a minimal media containing 10 μM GSH. Plasmids were retrieved, re-examined for their functional activity using the growth-based plate assay, and sequenced. In one clone, P525K had converted to 525Q. All other suppressors for both P525K and P525E had reverted back to the wild-type sequence P525.</p></div></div><div class="section discussion" id="sec-16"><h2 class="">Discussion</h2><p id="p-38">The work described in this manuscript has explored a novel strategy for gaining insight into the mechanisms of membrane transporter functioning. In this approach, the six residues that formed the hydrophobic face of a TMD helix whose polar face was presumed to line the substrate translocation channel was subjected to “charged/polar-residue scanning” mutagenesis followed by genetic suppressor and biochemical analysis.</p><p id="p-39">One of the most striking observations revealed from this study was that membrane transporters are surprisingly tolerant of charged residues, even in the most hydrophobic part of the TMDs. Previous efforts whereby charges were introduced into TM domains of membrane proteins were done either to examine the role in translocation/trafficking (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Davis and Hunter 1987</a>) or to determine if the membrane domain spanned the membrane lipid bilayer with a helical (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Song <em>et al.</em> 1991</a>). In both cases, introduction of charged residues led to significant loss in function of the proteins. In contrast, out of the 16 mutations made to charged/polar residues (to lysine, glutamine, or glutamic acid), only six of these showed a complete loss in activity. In those cases, when an effect was seen, the effect of the replacement depended on both the position and the residue being introduced. Thus, for example, V513Q was functional, but V513K showed very low functionality; in contrast, L517Q was nonfunctional but L517K was functional. The loss of functionality in these cases was shown to be due to loss of protein expression, but it is clear that in the absence of any structural information it is difficult to predict the effects of introducing different charge/polarity at a particular position. Only at one of the six positions, position I524, was there a complete lack of tolerance to any charged/polar residues. I524E, I524K, or I524Q were all completely nonfunctional. However, all other positions on this face were capable of accepting at least one of the charged/polar residues. The I524 residue mutated to charged residues was also the only residue that allowed isolation of intragenic suppressors. This might suggest that other residues are not directly involved in interaction with other domains, and might also be an explanation for why charged/polar residues at these positions are more easily tolerated</p><p id="p-40">Among the six substituted changes that led to complete loss in activity, three of them, L520Q, L520E, and L517Q, appeared to have destabilized the protein because no protein could be detected in L520Q and L520E and a very reduced protein expression was seen in L517Q. It was interesting that charged/polar residue changes at I524 affected the function of the protein but did not affect the protein expression to any significant extent. These mutants were also properly localized to the cell surface. There are thus two possible explanations for the complete loss in activity seen in I524K, I524Q, and I524E. The first one is that considering the proximity to the substrate binding residues; the changes were somehow drastically interfering with the substrate-binding. This could have been resolved by kinetic analysis; however, the activity was too low to allow us to subject these mutants to any kind of kinetic analysis. We thus examined how P525, another residue located between F523 and Q526, could tolerate charged residues. We subjected P525 to charged/polar residue scanning mutagenesis. P525K and P525E were nonfunctional, but P525Q was functional. Thus, charged/polar residues were functional at position P525, suggesting that mere proximity to the substrate binding residues could not account for the behavior of the I524K/Q/E mutants.</p><p id="p-41">That brings us to the second plausible explanation for the complete loss of function of the I524K, I524E and I524Q mutants, namely, that the residue was on the region of the hydrophobic face that was part of the dynamics of channel movement and function. If this was the case, then it should involve interactions with other domains and, if so, we should be able to isolate suppressor mutations at other parts of the protein that might be able to suppress this loss of function. It is interesting in this context that although all the nonfunctional mutants were subjected to suppressor analysis, only second-site suppressor mutations were identified for I524Q and I524K. The position at which the second-site mutations had occurred was at G202, and it is predicted to be located in the loop between TMD3 and TMD4, close to TMD3. Interestingly, neither of these residues (neither I524 nor G202) was conserved in the oligopeptide transporter (OPT) super family. Interestingly, second-site suppressors were not observed even in the search for suppressors of P525K and P525E.</p><p id="p-42">G202Q by itself is not an inhibitory mutation, but in the background of I524Q and I524K it functioned as a genetic suppressor that reverses the incapacitating constraints introduced by the I524Q and I524K mutation. Interestingly, the second-site suppressor was always found at the G202 position for both nonfunctional mutations I524Q and I524K. These studies suggest that TMD9 and TMD3 may be adjacent to, or in close contact with, each other. Sequence analysis of OPT homologs for any evolutionary coupling of G202 and I524 was carried out; however, no such coupling seemed to exist (data not shown). The higher <em>K<sub>m</sub></em> in the suppressor suggests that the interaction has an impact on the substrate binding, but the major role of this interaction seemed to go beyond the effects on substrate binding. An explanation for the G202 with Q/K/I substitutions is that it could shift the local secondary structure probability from loop to helix and may produce helical turn in TMD3. Irrespective of this or other effects of G202 mutations, it is clear that the region plays a critical role in substrate translocation in conjunction with the hydrophobic face of TMD9. More biophysical, biochemical, or crystal structure studies would be required, however, to confirm these events.</p><p id="p-43">In conclusion, we have used a novel strategy that we refer to as charged/polar-residue scanning mutagenesis of the hydrophobic face of TM helices as an approach to investigating membrane transporters and that could be added to the list of approaches currently being used to investigate this class of proteins. Genetic suppressor analysis for the determination of interacting domains is not a new approach, even in membrane transporters, and has been successfully applied previously (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Arastu-Kapur <em>et al.</em> 2005</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Auerbach <em>et al.</em> 2002</a>), but the systematic application of this strategy to the hydrophobic face of the helix reveals a novel way of gaining insight into this class of proteins. The application of this approach to TMD9 of the yeast glutathione transporter, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003748">Hgt1p</a>, has yielded several important insights, and we believe that this approach can be successfully applied to other transporters as well.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-44">This work was supported in part by grant-in-aid projects to A.K.B. from the Departments of Science and Technology and Department of Biotechnology, Government of India, and the JC Bose National Fellowship. A.T. was the recipient of a Research Fellowship from Council of Scientific and Industrial Research, Government of India.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-4"><p id="p-45">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017079/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.017079/-/DC1</a></p></li><li class="fn" id="fn-5"><p id="p-46"><em>Communicating editor: R. A. Sclafani</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-01-30"><span class="received-label">Received </span>January 30, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-03-12"><span class="accepted-label">Accepted </span>March 12, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Thakur and Bachhawat</li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.921.1" data-doi="10.1126/science.1088196"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abramson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smirnova</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kasho</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verner</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaback</span>  <span class="cit-name-given-names">H. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Structure and mechanism of the lactose permease of Escherichia coli.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">301</span>: <span class="cit-fpage">610</span>–<span class="cit-lpage">615</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DAbramson%26rft.auinit1%253DJ.%26rft.volume%253D301%26rft.issue%253D5633%26rft.spage%253D610%26rft.epage%253D615%26rft.atitle%253DStructure%2Band%2BMechanism%2Bof%2Bthe%2BLactose%2BPermease%2Bof%2BEscherichia%2Bcoli%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1088196%26rft_id%253Dinfo%253Apmid%252F12893935%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMDEvNTYzMy82MTAiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzUvOTIxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.921.2" data-doi="10.1074/jbc.M408224200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arastu-Kapur</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arendt</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Purnat</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">N. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ullman</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Second-site suppression of a nonfunctional mutation within the Leishmania donovani inosine-guanosine transporter.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">280</span>: <span class="cit-fpage">2213</span>–<span class="cit-lpage">2219</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M408224200%26rft_id%253Dinfo%253Apmid%252F15501825%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEwOiIyODAvMy8yMjEzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS81LzkyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.921.3" data-doi="10.1081/RRS-120014615"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Auerbach</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galeuchet-Schenk</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hottiger</span>  <span class="cit-name-given-names">M. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stagljar</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Genetic approaches to the identification of interactions between membrane proteins in yeast.</span> <abbr class="cit-jnl-abbrev">J. Recept. Signal Transduct. Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">471</span>–<span class="cit-lpage">481</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Breceptor%2Band%2Bsignal%2Btransduction%2Bresearch%26rft.stitle%253DJ%2BRecept%2BSignal%2BTransduct%2BRes%26rft.aulast%253DAuerbach%26rft.auinit1%253DD.%26rft.volume%253D22%26rft.issue%253D1-4%26rft.spage%253D471%26rft.epage%253D481%26rft.atitle%253DGenetic%2Bapproaches%2Bto%2Bthe%2Bidentification%2Bof%2Binteractions%2Bbetween%2Bmembrane%2Bproteins%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1081%252FRRS-120014615%26rft_id%253Dinfo%253Apmid%252F12503635%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1081/RRS-120014615&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12503635&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000179631000033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.921.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bachhawat</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thakur</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaur</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zulkifli</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Glutathione transporters.</span> <abbr class="cit-jnl-abbrev">Biochim. Biophys. Acta</abbr> <span class="cit-vol">1830</span>: <span class="cit-fpage">3154</span>–<span class="cit-lpage">3164</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochim.%2BBiophys.%2BActa%26rft.volume%253D1830%26rft.spage%253D3154%26rft_id%253Dinfo%253Apmid%252F23206830%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23206830&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.921.5" data-doi="10.1074/jbc.275.18.13259"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bourbouloux</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shahi</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chakladar</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delrot</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bachhawat</span>  <span class="cit-name-given-names">A. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Hgt1p, a high affinity glutathione transporter from the yeast Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">275</span>: <span class="cit-fpage">13259</span>–<span class="cit-lpage">13265</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DBourbouloux%26rft.auinit1%253DA.%26rft.volume%253D275%26rft.issue%253D18%26rft.spage%253D13259%26rft.epage%253D13265%26rft.atitle%253DHgt1p%252C%2Ba%2BHigh%2BAffinity%2BGlutathione%2BTransporter%2Bfrom%2Bthe%2BYeast%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.275.18.13259%26rft_id%253Dinfo%253Apmid%252F10788431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzUvMTgvMTMyNTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzUvOTIxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.921.6" data-doi="10.1093/aob/mcn207"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Curie</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cassin</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Couch</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Divol</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Higuchi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Metal movement within the plant: contribution of nicotianamine and yellow stripe 1-like transporters.</span> <abbr class="cit-jnl-abbrev">Ann. Bot. (Lond.)</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">11</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BBot.%2B%2528Lond.%2529%26rft_id%253Dinfo%253Adoi%252F10.1093%252Faob%252Fmcn207%26rft_id%253Dinfo%253Apmid%252F18977764%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW5uYm90IjtzOjU6InJlc2lkIjtzOjc6IjEwMy8xLzEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzUvOTIxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.921.7" data-doi="10.1038/35053080"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Curie</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Panaviene</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loulergue</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dellaporta</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Briat</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Maize yellow stripe1 encodes a membrane protein directly involved in Fe(III) uptake.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">409</span>: <span class="cit-fpage">346</span>–<span class="cit-lpage">349</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DCurie%26rft.auinit1%253DC.%26rft.volume%253D409%26rft.issue%253D6818%26rft.spage%253D346%26rft.epage%253D349%26rft.atitle%253DMaize%2Byellow%2Bstripe1%2Bencodes%2Ba%2Bmembrane%2Bprotein%2Bdirectly%2Binvolved%2Bin%2BFe%2528III%2529%2Buptake.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35053080%26rft_id%253Dinfo%253Apmid%252F11201743%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35053080&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11201743&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000166434300048&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.921.8" data-doi="10.1083/jcb.105.3.1191"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">G. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hunter</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">A charged amino acid substitution within the transmembrane anchor of the Rous sarcoma virus envelope glycoprotein affects surface expression but not intracellular transport.</span> <abbr class="cit-jnl-abbrev">J. Cell Biol.</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">1191</span>–<span class="cit-lpage">1203</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BCell%2BBiology%26rft.stitle%253DJCB%26rft.aulast%253DDavis%26rft.auinit1%253DG.%2BL.%26rft.volume%253D105%26rft.issue%253D3%26rft.spage%253D1191%26rft.epage%253D1203%26rft.atitle%253DA%2Bcharged%2Bamino%2Bacid%2Bsubstitution%2Bwithin%2Bthe%2Btransmembrane%2Banchor%2Bof%2Bthe%2BRous%2Bsarcoma%2Bvirus%2Benvelope%2Bglycoprotein%2Baffects%2Bsurface%2Bexpression%2Bbut%2Bnot%2Bintracellular%2Btransport%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.105.3.1191%26rft_id%253Dinfo%253Apmid%252F2821009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjEwOiIxMDUvMy8xMTkxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS81LzkyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.921.9" data-doi="10.1074/jbc.M111.272377"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Desai</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thakur</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ganguli</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paul</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morschhauser</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Glutathione utilization by Candida albicans requires a functional glutathione degradation (DUG) pathway and OPT7, an unusual member of the oligopeptide transporter family.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">286</span>: <span class="cit-fpage">41183</span>–<span class="cit-lpage">41194</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M111.272377%26rft_id%253Dinfo%253Apmid%252F21994941%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODYvNDgvNDExODMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzUvOTIxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.921.10" data-doi="10.1111/1567-1364.12196"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elbaz-Alon</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clancy</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amoako</span>  <span class="cit-name-given-names">T. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zalckvar</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The yeast oligopeptide transporter Opt2 is localized to peroxisomes and affects glutathione redox homeostasis.</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">1055</span>–<span class="cit-lpage">1067</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1567-1364.12196%26rft_id%253Dinfo%253Apmid%252F25130273%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc3lyIjtzOjU6InJlc2lkIjtzOjk6IjE0LzcvMTA1NSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvNS85MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.921.11" data-doi="10.4161/chan.2.5.6903"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elvington</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maduke</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Thinking outside the crystal: complementary approaches for examining transporter conformational change.</span> <abbr class="cit-jnl-abbrev">Channels (Austin)</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">373</span>–<span class="cit-lpage">379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DChannels%2B%2528Austin%2529%26rft.aulast%253DElvington%26rft.auinit1%253DS.%2BM.%26rft.volume%253D2%26rft.issue%253D5%26rft.spage%253D373%26rft.epage%253D379%26rft.atitle%253DThinking%2Boutside%2Bthe%2Bcrystal%253A%2Bcomplementary%2Bapproaches%2Bfor%2Bexamining%2Btransporter%2Bconformational%2Bchange.%26rft_id%253Dinfo%253Adoi%252F10.4161%252Fchan.2.5.6903%26rft_id%253Dinfo%253Apmid%252F18989097%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/chan.2.5.6903&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18989097&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.921.12" data-doi="10.1002/yea.1778"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Functional characterization of the PP2C phosphatase CaPtc2p in the human fungal pathogen Candida albicans.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">753</span>–<span class="cit-lpage">764</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DFeng%26rft.auinit1%253DJ.%26rft.volume%253D27%26rft.issue%253D9%26rft.spage%253D753%26rft.epage%253D764%26rft.atitle%253DFunctional%2Bcharacterization%2Bof%2Bthe%2BPP2C%2Bphosphatase%2BCaPtc2p%2Bin%2Bthe%2Bhuman%2Bfungal%2Bpathogen%2BCandida%2Balbicans.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.1778%26rft_id%253Dinfo%253Apmid%252F20641018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.1778&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20641018&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282844800006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.921.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frillingos</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sahin-Toth</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaback</span>  <span class="cit-name-given-names">H. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Cys-scanning mutagenesis: a novel approach to structure function relationships in polytopic membrane proteins.</span> <abbr class="cit-jnl-abbrev">FASEB J.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">1281</span>–<span class="cit-lpage">1299</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BFASEB%2BJournal%26rft.stitle%253DFASEB%2BJ.%26rft.aulast%253DFrillingos%26rft.auinit1%253DS.%26rft.volume%253D12%26rft.issue%253D13%26rft.spage%253D1281%26rft.epage%253D1299%26rft.atitle%253DCys-scanning%2Bmutagenesis%253A%2Ba%2Bnovel%2Bapproach%2Bto%2Bstructure%2596function%2Brelationships%2Bin%2Bpolytopic%2Bmembrane%2Bproteins%26rft_id%253Dinfo%253Apmid%252F9761772%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmFzZWJqIjtzOjU6InJlc2lkIjtzOjEwOiIxMi8xMy8xMjgxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS81LzkyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.921.14" data-doi="10.1007/s00232-011-9347-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gomolplitinant</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saier</span>  <span class="cit-name-given-names">M. H.</span>  <span class="cit-name-suffix">Jr</span>.</span></li></ol><cite>., <span class="cit-pub-date">2011</span> <span class="cit-article-title">Evolution of the oligopeptide transporter family.</span> <abbr class="cit-jnl-abbrev">J. Membr. Biol.</abbr> <span class="cit-vol">240</span>: <span class="cit-fpage">89</span>–<span class="cit-lpage">110</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2Bmembrane%2Bbiology%26rft.stitle%253DJ%2BMembr%2BBiol%26rft.aulast%253DGomolplitinant%26rft.auinit1%253DK.%2BM.%26rft.volume%253D240%26rft.issue%253D2%26rft.spage%253D89%26rft.epage%253D110%26rft.atitle%253DEvolution%2Bof%2Bthe%2Boligopeptide%2Btransporter%2Bfamily.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00232-011-9347-9%26rft_id%253Dinfo%253Apmid%252F21347612%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00232-011-9347-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21347612&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.921.15" data-doi="10.1146/annurev.biophys.35.040405.102005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaback</span>  <span class="cit-name-given-names">H. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Lessons from lactose permease.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Biophys. Biomol. Struct.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">67</span>–<span class="cit-lpage">91</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bbiophysics%2Band%2Bbiomolecular%2Bstructure%26rft.stitle%253DAnnu%2BRev%2BBiophys%2BBiomol%2BStruct%26rft.aulast%253DGuan%26rft.auinit1%253DL.%26rft.volume%253D35%26rft.spage%253D67%26rft.epage%253D91%26rft.atitle%253DLessons%2Bfrom%2Blactose%2Bpermease.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.biophys.35.040405.102005%26rft_id%253Dinfo%253Apmid%252F16689628%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.biophys.35.040405.102005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16689628&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238326400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.921.16" data-doi="10.1016/S1046-2023(05)80158-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guthrie</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fink</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Guide to yeast genetics and molecular biology.</span> <abbr class="cit-jnl-abbrev">Methods Enzymol.</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">3</span>–<span class="cit-lpage">37</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Benzymology%26rft.stitle%253DMethods%2BEnzymol%26rft.aulast%253DSherman%26rft.auinit1%253DF.%26rft.volume%253D194%26rft.spage%253D3%26rft.epage%253D21%26rft.atitle%253DGetting%2Bstarted%2Bwith%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1046-2023%252805%252980158-6%26rft_id%253Dinfo%253Apmid%252F2005794%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1046-2023(05)80158-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2005794&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991FN84200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.921.17" data-doi="10.1074/jbc.275.5.3037"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hauser</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donhardt</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barnes</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naider</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Becker</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Enkephalins are transported by a novel eukaryotic peptide uptake system.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">275</span>: <span class="cit-fpage">3037</span>–<span class="cit-lpage">3041</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DHauser%26rft.auinit1%253DM.%26rft.volume%253D275%26rft.issue%253D5%26rft.spage%253D3037%26rft.epage%253D3041%26rft.atitle%253DEnkephalins%2BAre%2BTransported%2Bby%2Ba%2BNovel%2BEukaryotic%2BPeptide%2BUptake%2BSystem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.275.5.3037%26rft_id%253Dinfo%253Apmid%252F10652283%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEwOiIyNzUvNS8zMDM3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS81LzkyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.921.18" data-doi="10.1074/jbc.M806042200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Inoue</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kobayashi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nozoye</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takahashi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kakei</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Rice OsYSL15 is an iron-regulated iron(III)-deoxymugineic acid transporter expressed in the roots and is essential for iron uptake in early growth of the seedlings.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">284</span>: <span class="cit-fpage">3470</span>–<span class="cit-lpage">3479</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M806042200%26rft_id%253Dinfo%253Apmid%252F19049971%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEwOiIyODQvNi8zNDcwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS81LzkyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.921.19" data-doi="10.1074/jbc.M109.029728"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kaur</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bachhawat</span>  <span class="cit-name-given-names">A. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">a</span> <span class="cit-article-title">Gln-222 in transmembrane domain 4 and Gln-526 in transmembrane domain 9 are critical for substrate recognition in the yeast high affinity glutathione transporter, Hgt1p.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">284</span>: <span class="cit-fpage">23872</span>–<span class="cit-lpage">23884</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M109.029728%26rft_id%253Dinfo%253Apmid%252F19589778%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODQvMzUvMjM4NzIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzUvOTIxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.921.20" data-doi="10.1016/j.ab.2008.10.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kaur</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bachhawat</span>  <span class="cit-name-given-names">A. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">b</span> <span class="cit-article-title">A modified Western blot protocol for enhanced sensitivity in the detection of a membrane protein.</span> <abbr class="cit-jnl-abbrev">Anal. Biochem.</abbr> <span class="cit-vol">384</span>: <span class="cit-fpage">348</span>–<span class="cit-lpage">349</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnalytical%2Bbiochemistry%26rft.stitle%253DAnal%2BBiochem%26rft.aulast%253DKaur%26rft.auinit1%253DJ.%26rft.volume%253D384%26rft.issue%253D2%26rft.spage%253D348%26rft.epage%253D349%26rft.atitle%253DA%2Bmodified%2BWestern%2Bblot%2Bprotocol%2Bfor%2Benhanced%2Bsensitivity%2Bin%2Bthe%2Bdetection%2Bof%2Ba%2Bmembrane%2Bprotein.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ab.2008.10.005%26rft_id%253Dinfo%253Apmid%252F18952039%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ab.2008.10.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18952039&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.921.21" data-doi="10.1111/j.1567-1364.2009.00529.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kaur</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Srikanth</span>  <span class="cit-name-given-names">C. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bachhawat</span>  <span class="cit-name-given-names">A. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Differential roles played by the native cysteine residues of the yeast glutathione transporter, Hgt1p.</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">849</span>–<span class="cit-lpage">866</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1567-1364.2009.00529.x%26rft_id%253Dinfo%253Apmid%252F19496824%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc3lyIjtzOjU6InJlc2lkIjtzOjc6IjkvNi84NDkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzUvOTIxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.921.22" data-doi="10.1038/emboj.2011.105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Igbaria</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">D’Autreaux</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Planson</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Junot</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Glutathione revisited: a vital function in iron metabolism and ancillary role in thiol-redox control.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2044</span>–<span class="cit-lpage">2056</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DKumar%26rft.auinit1%253DC.%26rft.volume%253D30%26rft.issue%253D10%26rft.spage%253D2044%26rft.epage%253D2056%26rft.atitle%253DGlutathione%2Brevisited%253A%2Ba%2Bvital%2Bfunction%2Bin%2Biron%2Bmetabolism%2Band%2Bancillary%2Brole%2Bin%2Bthiol-redox%2Bcontrol%26rft_id%253Dinfo%253Adoi%252F10.1038%252Femboj.2011.105%26rft_id%253Dinfo%253Apmid%252F21478822%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMzAvMTAvMjA0NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvNS85MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.921.23" data-doi="10.1146/annurev.micro.61.080706.093329"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Law</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maloney</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">D. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Ins and outs of major facilitator superfamily antiporters.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Microbiol.</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">305</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bmicrobiology%26rft.stitle%253DAnnu%2BRev%2BMicrobiol%26rft.aulast%253DLaw%26rft.auinit1%253DC.%2BJ.%26rft.volume%253D62%26rft.spage%253D289%26rft.epage%253D305%26rft.atitle%253DIns%2Band%2Bouts%2Bof%2Bmajor%2Bfacilitator%2Bsuperfamily%2Bantiporters.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.micro.61.080706.093329%26rft_id%253Dinfo%253Apmid%252F18537473%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.micro.61.080706.093329&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18537473&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259968000017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.921.24" data-doi="10.1093/mp/ssr004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lubkowitz</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The oligopeptide transporters: a small gene family with a diverse group of substrates and functions?</span> <abbr class="cit-jnl-abbrev">Mol. Plant</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">407</span>–<span class="cit-lpage">415</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPlant%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmp%252Fssr004%26rft_id%253Dinfo%253Apmid%252F21310763%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/mp/ssr004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21310763&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000290813600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.921.25" data-doi="10.1099/00221287-143-2-387"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lubkowitz</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hauser</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Breslav</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naider</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Becker</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">An oligopeptide transport gene from Candida albicans.</span> <abbr class="cit-jnl-abbrev">Microbiology</abbr> <span class="cit-vol">143</span>(<span class="cit-issue">Pt 2</span>): <span class="cit-fpage">387</span>–<span class="cit-lpage">396</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%26rft.stitle%253DMicrobiology%26rft.aulast%253DLubkowitz%26rft.auinit1%253DM.%2BA.%26rft.volume%253D143%26rft.issue%253D2%26rft.spage%253D387%26rft.epage%253D396%26rft.atitle%253DAn%2Boligopeptide%2Btransport%2Bgene%2Bfrom%2BCandida%2Balbicans%26rft_id%253Dinfo%253Adoi%252F10.1099%252F00221287-143-2-387%26rft_id%253Dinfo%253Apmid%252F9043116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/00221287-143-2-387&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9043116&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997WH22500014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.921.26" data-doi="10.1042/BJ20120190"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mandal</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lynn</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kapoor</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A key structural domain of the Candida albicans Mdr1 protein.</span> <abbr class="cit-jnl-abbrev">Biochem. J.</abbr> <span class="cit-vol">445</span>: <span class="cit-fpage">313</span>–<span class="cit-lpage">322</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochem.%2BJ.%26rft_id%253Dinfo%253Adoi%252F10.1042%252FBJ20120190%26rft_id%253Dinfo%253Apmid%252F22587419%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTA6InBwYmlvY2hlbWoiO3M6NToicmVzaWQiO3M6OToiNDQ1LzMvMzEzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS81LzkyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.921.27" data-doi="10.1042/BJ20050920"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Osawa</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stacey</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gassmann</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">ScOPT1 and AtOPT4 function as proton-coupled oligopeptide transporters with broad but distinct substrate specificities.</span> <abbr class="cit-jnl-abbrev">Biochem. J.</abbr> <span class="cit-vol">393</span>: <span class="cit-fpage">267</span>–<span class="cit-lpage">275</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochem.%2BJ.%26rft_id%253Dinfo%253Adoi%252F10.1042%252FBJ20050920%26rft_id%253Dinfo%253Apmid%252F16149917%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTA6InBwYmlvY2hlbWoiO3M6NToicmVzaWQiO3M6OToiMzkzLzEvMjY3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS81LzkyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.921.28" data-doi="10.1074/jbc.M113.488353"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rawal</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khan</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kapoor</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goyal</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Insight into pleiotropic drug resistance ATP-binding cassette pump drug transport through mutagenesis of Cdr1p transmembrane domains.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">288</span>: <span class="cit-fpage">24480</span>–<span class="cit-lpage">24493</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M113.488353%26rft_id%253Dinfo%253Apmid%252F23824183%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODgvMzQvMjQ0ODAiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzUvOTIxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.921.29" data-doi="10.1111/j.1365-2958.2006.05136.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reuss</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morschhauser</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A family of oligopeptide transporters is required for growth of Candida albicans on proteins.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">60</span>: <span class="cit-fpage">795</span>–<span class="cit-lpage">812</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DReu%25C3%259F%26rft.auinit1%253DO.%26rft.volume%253D60%26rft.issue%253D3%26rft.spage%253D795%26rft.epage%253D812%26rft.atitle%253DA%2Bfamily%2Bof%2Boligopeptide%2Btransporters%2Bis%2Brequired%2Bfor%2Bgrowth%2Bof%2BCandida%2Balbicans%2Bon%2Bproteins.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2958.2006.05136.x%26rft_id%253Dinfo%253Apmid%252F16629678%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2958.2006.05136.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16629678&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236767800021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.921.30" data-doi="10.1016/0092-8674(87)90712-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rose</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fink</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">KAR1, a gene required for function of both intranuclear and extranuclear microtubules in yeast.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">1047</span>–<span class="cit-lpage">1060</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DRose%26rft.auinit1%253DM.%2BD.%26rft.volume%253D48%26rft.issue%253D6%26rft.spage%253D1047%26rft.epage%253D1060%26rft.atitle%253DKAR1%252C%2Ba%2Bgene%2Brequired%2Bfor%2Bfunction%2Bof%2Bboth%2Bintranuclear%2Band%2Bextranuclear%2Bmicrotubules%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252887%252990712-4%26rft_id%253Dinfo%253Apmid%252F3030557%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(87)90712-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3030557&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1987G694600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-book" id="cit-5.5.921.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sambrook</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fritsch</span>  <span class="cit-name-given-names">E. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maniatis</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-source">Molecular Cloning: A Laboratory Manual</span>, <span class="cit-publ-name">Cold Spring Harbor Press</span>, <span class="cit-publ-loc">Cold Spring Harbor, New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.921.32" data-doi="10.1128/jb.173.9.2927-2934.1991"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">H. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cohen</span>  <span class="cit-name-given-names">F. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cramer</span>  <span class="cit-name-given-names">W. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Membrane topography of ColE1 gene products: the hydrophobic anchor of the colicin E1 channel is a helical hairpin.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">173</span>: <span class="cit-fpage">2927</span>–<span class="cit-lpage">2934</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DSong%26rft.auinit1%253DH.%2BY.%26rft.volume%253D173%26rft.issue%253D9%26rft.spage%253D2927%26rft.epage%253D2934%26rft.atitle%253DMembrane%2Btopography%2Bof%2BColE1%2Bgene%2Bproducts%253A%2Bthe%2Bhydrophobic%2Banchor%2Bof%2Bthe%2Bcolicin%2BE1%2Bchannel%2Bis%2Ba%2Bhelical%2Bhairpin.%26rft_id%253Dinfo%253Adoi%252F10.1128%252Fjb.173.9.2927-2934.1991%26rft_id%253Dinfo%253Apmid%252F1708383%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTA6IjE3My85LzI5MjciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzUvOTIxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.921.33" data-doi="10.1007/s00294-005-0571-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Srikanth</span>  <span class="cit-name-given-names">C. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vats</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bourbouloux</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delrot</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bachhawat</span>  <span class="cit-name-given-names">A. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Multiple cis-regulatory elements and the yeast sulphur regulatory network are required for the regulation of the yeast glutathione transporter, Hgt1p.</span> <abbr class="cit-jnl-abbrev">Curr. Genet.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">345</span>–<span class="cit-lpage">358</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bgenetics%26rft.stitle%253DCurr%2BGenet%26rft.aulast%253DSrikanth%26rft.auinit1%253DC.%2BV.%26rft.volume%253D47%26rft.issue%253D6%26rft.spage%253D345%26rft.epage%253D358%26rft.atitle%253DMultiple%2Bcis-regulatory%2Belements%2Band%2Bthe%2Byeast%2Bsulphur%2Bregulatory%2Bnetwork%2Bare%2Brequired%2Bfor%2Bthe%2Bregulation%2Bof%2Bthe%2Byeast%2Bglutathione%2Btransporter%252C%2BHgt1p.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00294-005-0571-7%26rft_id%253Dinfo%253Apmid%252F15821937%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00294-005-0571-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15821937&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000229802800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.921.34" data-doi="10.1042/BJ20100240"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thakur</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bachhawat</span>  <span class="cit-name-given-names">A. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The role of transmembrane domain 9 in substrate recognition by the fungal high-affinity glutathione transporters.</span> <abbr class="cit-jnl-abbrev">Biochem. J.</abbr> <span class="cit-vol">429</span>: <span class="cit-fpage">593</span>–<span class="cit-lpage">602</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochem.%2BJ.%26rft_id%253Dinfo%253Adoi%252F10.1042%252FBJ20100240%26rft_id%253Dinfo%253Apmid%252F20477749%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTA6InBwYmlvY2hlbWoiO3M6NToicmVzaWQiO3M6OToiNDI5LzMvNTkzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS81LzkyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.921.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thakur</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bachhawat</span>  <span class="cit-name-given-names">A. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Mutations in the N-terminal region of the Schizosaccharomyces pombe glutathione transporter pgt1(+) allows functional expression in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">45</span>–<span class="cit-lpage">54</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%26rft.volume%253D30%26rft.spage%253D45%26rft_id%253Dinfo%253Apmid%252F23280723%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23280723&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.921.36" data-doi="10.1111/j.1567-1364.2008.00423.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thakur</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaur</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bachhawat</span>  <span class="cit-name-given-names">A. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Pgt1, a glutathione transporter from the fission yeast Schizosaccharomyces pombe.</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">916</span>–<span class="cit-lpage">929</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1567-1364.2008.00423.x%26rft_id%253Dinfo%253Apmid%252F18662319%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc3lyIjtzOjU6InJlc2lkIjtzOjc6IjgvNi85MTYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzUvOTIxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.921.37" data-doi="10.1099/00221287-147-11-2881"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yen</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tseng</span>  <span class="cit-name-given-names">Y. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saier</span>  <span class="cit-name-given-names">M. H.</span>  <span class="cit-name-suffix">Jr</span>.</span></li></ol><cite>., <span class="cit-pub-date">2001</span> <span class="cit-article-title">Maize Yellow Stripe1, an iron-phytosiderophore uptake transporter, is a member of the oligopeptide transporter (OPT) family.</span> <abbr class="cit-jnl-abbrev">Microbiology</abbr> <span class="cit-vol">147</span>: <span class="cit-fpage">2881</span>–<span class="cit-lpage">2883</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%26rft.stitle%253DMicrobiology%26rft.aulast%253DYen%26rft.auinit1%253DM.-R.%26rft.volume%253D147%26rft.issue%253D11%26rft.spage%253D2881%26rft.epage%253D2883%26rft.atitle%253DMaize%2BYellow%2BStripe1%252C%2Ban%2Biron-phytosiderophore%2Buptake%2Btransporter%252C%2Bis%2Ba%2Bmember%2Bof%2Bthe%2Boligopeptide%2Btransporter%2B%2528OPT%2529%2Bfamily%26rft_id%253Dinfo%253Adoi%252F10.1099%252F00221287-147-11-2881%26rft_id%253Dinfo%253Apmid%252F11700339%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/00221287-147-11-2881&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11700339&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000172195600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/5/921.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/5/911" title="A Powerful New Quantitative Genetics Platform, Combining Caenorhabditis elegans High-Throughput Fitness Assays with a Large Collection of Recombinant Strains" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/5/931" title="Mapping Bias Overestimates Reference Allele Frequencies at the HLA Genes in the 1000 Genomes Project Phase I Data" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/5"><span>Volume 5 Issue 5, May 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/5.cover-source.jpg?itok=53KRlKz0" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/5/921" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-urMdPA3nIe_pLM-3fuVKOnYzz4H6uqCcbdRxwtII9A0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/5/921" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/5/921" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/5/921" class="active" data-icon-position="" data-hide-link-title="0">Charged/Polar-Residue Scanning of the Hydrophobic Face of Transmembrane Domain 9 of the Yeast Glutathione Transporter, Hgt1p, Reveals a Conformationally Critical Region for Substrate Transport</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459845" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-G0g65_NTlvrEJdN9ZFFmMYrtOzT7gE3kH-kzW4orWEg" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/5/921.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/5/921" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-E8TIL6tKIk9Ngce8u5pIC763MyDg972TYFptXbWK3Lo" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/5/921" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/5/921.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459845" id="node459845--2" data-pisa="ggg;5/5/921" data-pisa-master="ggg;g3.115.017079" data-apath="/ggg/5/5/921.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Charged/Polar-Residue Scanning of the Hydrophobic Face of Transmembrane Domain 9 of the Yeast Glutathione Transporter, Hgt1p, Reveals a Conformationally Critical Region for Substrate Transport</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Anil Thakur</span> and <span class="highwire-citation-author" data-delta="1">Anand K. Bachhawat</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">921-929;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.017079 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459845--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Anil Thakur </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Microbial Technology (IMTECH), Sector 39-A Chandigarh 160036, India</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Indian Institute of Science Education & Research (IISER) Mohali, Knowledge City, S.A.S. Nagar, Punjab 140306, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anil%2BThakur%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thakur%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnil%2BThakur%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Anand K. Bachhawat </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Microbial Technology (IMTECH), Sector 39-A Chandigarh 160036, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anand%2BK.%2BBachhawat%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bachhawat%20AK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAnand%2BK.%2BBachhawat%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:anand@iisermohali.ac.in" class="" data-icon-position="" data-hide-link-title="0">anand@iisermohali.ac.in</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/5/921" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/5/921&amp;title=Charged/Polar-Residue%20Scanning%20of%20the%20Hydrophobic%20Face%20of%20Transmembrane%20Domain%209%20of%20the%20Yeast%20Glutathione%20Transporter%2C%20Hgt1p%2C%20Reveals%20a%20Conformationally%20Critical%20Region%20for%20Substrate%20Transport" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/5/921&amp;title=Charged/Polar-Residue%20Scanning%20of%20the%20Hydrophobic%20Face%20of%20Transmembrane%20Domain%209%20of%20the%20Yeast%20Glutathione%20Transporter%2C%20Hgt1p%2C%20Reveals%20a%20Conformationally%20Critical%20Region%20for%20Substrate%20Transport" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/5/921&amp;title=Charged/Polar-Residue%20Scanning%20of%20the%20Hydrophobic%20Face%20of%20Transmembrane%20Domain%209%20of%20the%20Yeast%20Glutathione%20Transporter%2C%20Hgt1p%2C%20Reveals%20a%20Conformationally%20Critical%20Region%20for%20Substrate%20Transport" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/5/921&amp;text=Charged/Polar-Residue%20Scanning%20of%20the%20Hydrophobic%20Face%20of%20Transmembrane%20Domain%209%20of%20the%20Yeast%20Glutathione%20Transporter%2C%20Hgt1p%2C%20Reveals%20a%20Conformationally%20Critical%20Region%20for%20Substrate%20Transport" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/5/921&amp;title=Charged/Polar-Residue%20Scanning%20of%20the%20Hydrophobic%20Face%20of%20Transmembrane%20Domain%209%20of%20the%20Yeast%20Glutathione%20Transporter%2C%20Hgt1p%2C%20Reveals%20a%20Conformationally%20Critical%20Region%20for%20Substrate%20Transport" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/5/921&amp;t=Charged/Polar-Residue%20Scanning%20of%20the%20Hydrophobic%20Face%20of%20Transmembrane%20Domain%209%20of%20the%20Yeast%20Glutathione%20Transporter%2C%20Hgt1p%2C%20Reveals%20a%20Conformationally%20Critical%20Region%20for%20Substrate%20Transport" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/5/921&amp;title=Charged/Polar-Residue%20Scanning%20of%20the%20Hydrophobic%20Face%20of%20Transmembrane%20Domain%209%20of%20the%20Yeast%20Glutathione%20Transporter%2C%20Hgt1p%2C%20Reveals%20a%20Conformationally%20Critical%20Region%20for%20Substrate%20Transport" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/5/921&amp;title=Charged/Polar-Residue%20Scanning%20of%20the%20Hydrophobic%20Face%20of%20Transmembrane%20Domain%209%20of%20the%20Yeast%20Glutathione%20Transporter%2C%20Hgt1p%2C%20Reveals%20a%20Conformationally%20Critical%20Region%20for%20Substrate%20Transport" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459845/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459845" id="node459845--3" data-pisa="ggg;5/5/921" data-pisa-master="ggg;g3.115.017079" data-apath="/ggg/5/5/921.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Charged/Polar-Residue Scanning of the Hydrophobic Face of Transmembrane Domain 9 of the Yeast Glutathione Transporter, Hgt1p, Reveals a Conformationally Critical Region for Substrate Transport</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Anil Thakur</span> and <span class="highwire-citation-author" data-delta="1">Anand K. Bachhawat</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">921-929;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.017079 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459845--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Anil Thakur </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Microbial Technology (IMTECH), Sector 39-A Chandigarh 160036, India</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Indian Institute of Science Education & Research (IISER) Mohali, Knowledge City, S.A.S. Nagar, Punjab 140306, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anil%2BThakur%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thakur%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnil%2BThakur%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Anand K. Bachhawat </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Microbial Technology (IMTECH), Sector 39-A Chandigarh 160036, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anand%2BK.%2BBachhawat%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bachhawat%20AK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAnand%2BK.%2BBachhawat%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:anand@iisermohali.ac.in" class="" data-icon-position="" data-hide-link-title="0">anand@iisermohali.ac.in</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=921&amp;rft.epage=929&amp;rft.atitle=Charged/Polar-Residue%20Scanning%20of%20the%20Hydrophobic%20Face%20of%20Transmembrane%20Domain%209%20of%20the%20Yeast%20Glutathione%20Transporter%2C%20Hgt1p%2C%20Reveals%20a%20Conformationally%20Critical%20Region%20for%20Substrate%20Transport&amp;rft.volume=5&amp;rft.issue=5&amp;rft.date=2015-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Thakur%2C+Anil&amp;rft.au=Bachhawat%2C+Anand+K."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459845/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459845/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459845/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459845/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459845/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459845/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459845/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459845/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459845/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459845/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459845/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459845/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/5/921" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/5/921.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/5/921.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/5/921.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/5/921.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
