<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/5/997" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/5/997.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/5/997.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/5/997.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/5/997.atom" />
<meta name="HW.pisa" content="ggg;5/5/997" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Leveraging DNA Damage Response Signaling to Identify Yeast Genes Controlling Genome Stability" />
<meta name="DC.Identifier" content="10.1534/g3.115.016576" />
<meta name="DC.Date" content="2015-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Hendry et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Oncogenesis frequently is accompanied by rampant genome instability, which fuels genetic heterogeneity and resistance to targeted cancer therapy. We have developed an approach that allows precise, quantitative measurement of genome instability in high-throughput format in the Saccharomyces cerevisiae model system. Our approach takes advantage of the strongly DNA damage-inducible gene [RNR3][1] , in conjunction with the reporter synthetic genetic array methodology, to infer mutants exhibiting genome instability by assaying for increased [Rnr3][1] abundance. We screen for genome instability across a set of ~1000 essential and ~4200 nonessential mutant yeast alleles in untreated conditions and in the presence of the DNA-damaging agent methylmethane sulfonate. Our results provide broad insights into the cellular processes and pathways required for genome maintenance. Through comparison with existing genome instability screens, we isolated 130 genes that had not previously been linked to genome maintenance, 51% of which have human homologs. Several of these homologs are associated with a genome instability phenotype in human cells or are causally mutated in cancer. A comprehensive understanding of the processes required to prevent genome instability will facilitate a better understanding of its sources in oncogenesis.

 [1]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328" />
<meta name="DC.Contributor" content="Jason A. Hendry" />
<meta name="DC.Contributor" content="Guihong Tan" />
<meta name="DC.Contributor" content="Jiongwen Ou" />
<meta name="DC.Contributor" content="Charles Boone" />
<meta name="DC.Contributor" content="Grant W. Brown" />
<meta name="article:published_time" content="2015-05-01" />
<meta name="article:section" content="Mutant Screen Report" />
<meta name="citation_title" content="Leveraging DNA Damage Response Signaling to Identify Yeast Genes Controlling Genome Stability" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Oncogenesis frequently is accompanied by rampant genome instability, which fuels genetic heterogeneity and resistance to targeted cancer therapy. We have developed an approach that allows precise, quantitative measurement of genome instability in high-throughput format in the &lt;i&gt;Saccharomyces cerevisiae&lt;/i&gt; model system. Our approach takes advantage of the strongly DNA damage-inducible gene &lt;i&gt;RNR3&lt;/i&gt;, in conjunction with the reporter synthetic genetic array methodology, to infer mutants exhibiting genome instability by assaying for increased Rnr3 abundance. We screen for genome instability across a set of ~1000 essential and ~4200 nonessential mutant yeast alleles in untreated conditions and in the presence of the DNA-damaging agent methylmethane sulfonate. Our results provide broad insights into the cellular processes and pathways required for genome maintenance. Through comparison with existing genome instability screens, we isolated 130 genes that had not previously been linked to genome maintenance, 51% of which have human homologs. Several of these homologs are associated with a genome instability phenotype in human cells or are causally mutated in cancer. A comprehensive understanding of the processes required to prevent genome instability will facilitate a better understanding of its sources in oncogenesis.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/05/01" />
<meta name="citation_mjid" content="ggg;5/5/997" />
<meta name="citation_id" content="5/5/997" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/5/997" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/5/997.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/5/997.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/5/997.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.016576" />
<meta name="citation_pmid" content="25721128" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Mutant Screen Report" />
<meta name="citation_firstpage" content="997" />
<meta name="citation_lastpage" content="1006" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Jason A. Hendry" />
<meta name="citation_author_institution" content="Department of Biochemistry, University of Toronto, Toronto, Ontario, Canada" />
<meta name="citation_author_institution" content="Donnelly Centre, University of Toronto, Toronto, Ontario, Canada" />
<meta name="citation_author" content="Guihong Tan" />
<meta name="citation_author_institution" content="Donnelly Centre, University of Toronto, Toronto, Ontario, Canada" />
<meta name="citation_author_institution" content="Department of Molecular Genetics, University of Toronto, Toronto, Ontario, Canada" />
<meta name="citation_author" content="Jiongwen Ou" />
<meta name="citation_author_institution" content="Department of Biochemistry, University of Toronto, Toronto, Ontario, Canada" />
<meta name="citation_author_institution" content="Donnelly Centre, University of Toronto, Toronto, Ontario, Canada" />
<meta name="citation_author" content="Charles Boone" />
<meta name="citation_author_institution" content="Donnelly Centre, University of Toronto, Toronto, Ontario, Canada" />
<meta name="citation_author_institution" content="Department of Molecular Genetics, University of Toronto, Toronto, Ontario, Canada" />
<meta name="citation_author" content="Grant W. Brown" />
<meta name="citation_author_institution" content="Department of Biochemistry, University of Toronto, Toronto, Ontario, Canada" />
<meta name="citation_author_institution" content="Donnelly Centre, University of Toronto, Toronto, Ontario, Canada" />
<meta name="citation_author_email" content="grant.brown@utoronto.ca" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9002-5003" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=R. Agarwal;citation_author=Z. Tang;citation_author=H. Yu;citation_author=O. Cohen-Fix;citation_title=Two distinct pathways for inhibiting pds1 ubiquitination in response to DNA damage.;citation_volume=278;citation_year=2003;citation_pmid=12947083;citation_doi=10.1074/jbc.M306783200" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=A. Aguilera;citation_author=B. Gomez-Gonzalez;citation_title=Genome instability: a mechanistic view of its causes and consequences.;citation_pages=204-217;citation_volume=9;citation_year=2008;citation_issue=3;citation_pmid=18227811;citation_doi=10.1038/nrg2268" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=D. Alvaro;citation_author=M. Lisby;citation_author=R. Rothstein;citation_title=Genome-wide analysis of Rad52 foci reveals diverse mechanisms impacting recombination.;citation_pages=e228-e228;citation_volume=3;citation_year=2007;citation_issue=12;citation_pmid=18085829;citation_doi=10.1371/journal.pgen.0030228" />
<meta name="citation_reference" content="citation_title=A whole genome screen for minisatellite stability genes in stationary phase yeast cells.;citation_author=B. Alver;citation_author=P. A. Jauert;citation_author=L. Brosnan;citation_author=M. O’Hehir;citation_author=B. Vandersluis;citation_pages=741-756;citation_volume=3;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. P. Andersen;citation_author=Z. W. Nelson;citation_author=E. D. Hetrick;citation_author=D. E. Gottschling;citation_title=A genetic screen for increased loss of heterozygosity in Saccharomyces cerevisiae.;citation_volume=179;citation_year=2008;citation_pmid=18562670;citation_doi=10.1534/genetics.108.089250" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. H. Askree;citation_author=T. Yehuda;citation_author=S. Smolikov;citation_author=R. Gurevich;citation_author=J. Hawk;citation_title=A genome-wide screen for Saccharomyces cerevisiae deletion mutants that affect telomere length.;citation_volume=101;citation_year=2004;citation_pmid=15161972;citation_doi=10.1073/pnas.0401263101" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=L. G. Ball;citation_author=K. Zhang;citation_author=J. A. Cobb;citation_author=C. Boone;citation_author=W. Xiao;citation_title=The yeast Shu complex couples error-free post-replication repair to homologous recombination.;citation_pages=89-102;citation_volume=73;citation_year=2009;citation_issue=1;citation_pmid=19496932;citation_doi=10.1111/j.1365-2958.2009.06748.x" />
<meta name="citation_reference" content="citation_journal_title=Mutation Research;citation_journal_abbrev=Mutation Research;citation_author=D. T. Beranek;citation_title=Distribution of methyl and ethyl adducts following alkylation with monofunctional alkylating agents.;citation_pages=11-30;citation_volume=231;citation_year=1990;citation_issue=1;citation_pmid=2195323;citation_doi=10.1016/0027-5107(90)90173-2" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=A. A. Carmen;citation_author=S. E. Rundlett;citation_author=M. Grunstein;citation_title=HDA1 and HDA3 Are Components of a Yeast Histone Deacetylase (HDA) Complex;citation_pages=15837-15844;citation_volume=271;citation_year=1996;citation_issue=26;citation_pmid=8663039;citation_doi=10.1074/jbc.271.26.15837" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=A. Chabes;citation_author=B. Georgieva;citation_author=V. Domkin;citation_author=X. Zhao;citation_author=R. Rothstein;citation_title=Survival of DNA damage in yeast directly depends on increased dNTP levels allowed by relaxed feedback inhibition of ribonucleotide reductase.;citation_pages=391-401;citation_volume=112;citation_year=2003;citation_issue=3;citation_pmid=12581528;citation_doi=10.1016/S0092-8674(03)00075-8" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. Chang;citation_author=M. Bellaoui;citation_author=C. Boone;citation_author=G. W. Brown;citation_title=A genome-wide screen for methyl methanesulfonate-sensitive mutants reveals genes required for S phase progression in the presence of DNA damage.;citation_volume=99;citation_year=2002;citation_pmid=12482937;citation_doi=10.1073/pnas.262669299" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=E. Cheng;citation_author=J. A. Vaisica;citation_author=J. Ou;citation_author=A. Baryshnikova;citation_author=Y. Lu;citation_title=Genome rearrangements caused by depletion of essential DNA replication proteins in Saccharomyces cerevisiae.;citation_volume=192;citation_year=2012;citation_pmid=22673806;citation_doi=10.1534/genetics.112.141051" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=K. Choi;citation_author=B. Szakal;citation_author=Y. H. Chen;citation_author=D. Branzei;citation_author=X. Zhao;citation_title=The Smc5/6 complex and Esc2 influence multiple replication-associated recombination processes in Saccharomyces cerevisiae.;citation_volume=21;citation_year=2010;citation_pmid=20444977;citation_doi=10.1091/mbc.E10-01-0050" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=R.-Y. Chuang;citation_author=P. L. Weaver;citation_author=Z. Liu;citation_author=T.-H. Chang;citation_title=Requirement of the DEAD-Box Protein Ded1p for Messenger RNA Translation;citation_pages=1468-1471;citation_volume=275;citation_year=1997;citation_issue=5305;citation_pmid=9045610;citation_doi=10.1126/science.275.5305.1468" />
<meta name="citation_reference" content="citation_journal_title=EMBO J.;citation_author=M. B. Davidson;citation_author=Y. Katou;citation_author=A. Keszthelyi;citation_author=T. L. Sing;citation_author=T. Xia;citation_title=Endogenous DNA replication stress results in expansion of dNTP pools and a mutator phenotype.;citation_volume=31;citation_year=2012;citation_pmid=22234187;citation_doi=10.1038/emboj.2011.485" />
<meta name="citation_reference" content="citation_title=G3&amp;#58; Genes|Genomes|Genetics;citation_journal_abbrev=G3;citation_author=A.C. Douglas;citation_author=A.M. Smith;citation_author=S. Sharifpoor;citation_author=Z. Yan;citation_author=T. Durbic;citation_title=Functional Analysis With a Barcoder Yeast Gene Overexpression System;citation_pages=1279-1289;citation_volume=2;citation_year=2012;citation_issue=10;citation_pmid=23050238;citation_doi=10.1534/g3.112.003400" />
<meta name="citation_reference" content="citation_journal_title=BioEssays :  news and reviews in molecular, cellular and developmental biology;citation_journal_abbrev=Bioessays;citation_author=S. J. Elledge;citation_author=Z. Zhou;citation_author=J. B. Allen;citation_author=T. A. Navas;citation_title=DNA damage and cell cycle regulation of ribonucleotide reductase.;citation_pages=333-339;citation_volume=15;citation_year=1993;citation_issue=5;citation_pmid=8343143;citation_doi=10.1002/bies.950150507" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. Falck;citation_author=J. Coates;citation_author=S. P. Jackson;citation_title=Conserved modes of recruitment of ATM, ATR and DNA-PKcs to sites of DNA damage.;citation_pages=605-611;citation_volume=434;citation_year=2005;citation_issue=7033;citation_pmid=15758953;citation_doi=10.1038/nature03442" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. A. Forbes;citation_author=D. Beare;citation_author=P. Gunasekaran;citation_author=K. Leung;citation_author=N. Bindal;citation_title=COSMIC: exploring the world’s knowledge of somatic mutations in human cancer.;citation_volume=43;citation_year=2015;citation_pmid=25355519;citation_doi=10.1093/nar/gku1075" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Cancer;citation_journal_abbrev=Nat Rev Cancer;citation_author=P. A. Futreal;citation_author=L. Coin;citation_author=M. Marshall;citation_author=T. Down;citation_author=T. Hubbard;citation_title=A census of human cancer genes.;citation_pages=177-183;citation_volume=4;citation_year=2004;citation_issue=3;citation_pmid=14993899;citation_doi=10.1038/nrc1299" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. Giaever;citation_author=A. M. Chu;citation_author=L. Ni;citation_author=C. Connelly;citation_author=L. Riles;citation_title=Functional profiling of the Saccharomyces cerevisiae genome.;citation_pages=387-391;citation_volume=418;citation_year=2002;citation_issue=6896;citation_pmid=12140549;citation_doi=10.1038/nature00935" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=M. S. Glickman;citation_author=C. L. Sawyers;citation_title=Converting cancer therapies into cures: lessons from infectious diseases.;citation_pages=1089-1098;citation_volume=148;citation_year=2012;citation_issue=6;citation_pmid=22424221;citation_doi=10.1016/j.cell.2012.02.015" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=J. P. Goldmark;citation_author=T. G. Fazzio;citation_author=P. W. Estep;citation_author=G. M. Church;citation_author=T. Tsukiyama;citation_title=The Isw2 chromatin remodeling complex represses early meiotic genes upon recruitment by Ume6p.;citation_pages=423-433;citation_volume=103;citation_year=2000;citation_issue=3;citation_pmid=11081629;citation_doi=10.1016/S0092-8674(00)00134-3" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=J. C. Harrison;citation_author=J. E. Haber;citation_title=Surviving the breakup: the DNA damage checkpoint.;citation_pages=209-235;citation_volume=40;citation_year=2006;citation_pmid=16805667;citation_doi=10.1146/annurev.genet.40.051206.105231" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=L. H. Hartwell;citation_title=Three Additional Genes Required for Deoxyribonucleic Acid Synthesis in Saccharomyces cerevisiae;citation_pages=966-974;citation_volume=115;citation_year=1973;citation_issue=3;citation_pmid=4580573" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=C. Hoege;citation_author=B. Pfander;citation_author=G. L. Moldovan;citation_author=G. Pyrowolakis;citation_author=S. Jentsch;citation_title=RAD6-dependent DNA repair is linked to modification of PCNA by ubiquitin and SUMO.;citation_pages=135-141;citation_volume=419;citation_year=2002;citation_issue=6903;citation_pmid=12226657;citation_doi=10.1038/nature00991" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=Y. Hu;citation_author=A. Rolfs;citation_author=B. Bhullar;citation_author=T. V. Murthy;citation_author=C. Zhu;citation_title=Approaching a complete repository of sequence-verified protein-encoding clones for Saccharomyces cerevisiae.;citation_volume=17;citation_year=2007;citation_pmid=17322287;citation_doi=10.1101/gr.6037607" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=M. Huang;citation_author=Z. Zhou;citation_author=S. J. Elledge;citation_title=The DNA replication and damage checkpoint pathways induce transcription by inhibition of the Crt1 repressor.;citation_pages=595-605;citation_volume=94;citation_year=1998;citation_issue=5;citation_pmid=9741624;citation_doi=10.1016/S0092-8674(00)81601-3" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. E. Huang;citation_author=A. G. Rio;citation_author=A. Nicolas;citation_author=R. D. Kolodner;citation_title=A genomewide screen in Saccharomyces cerevisiae for genes that suppress the accumulation of mutations.;citation_volume=100;citation_year=2003;citation_pmid=12972632;citation_doi=10.1073/pnas.2035018100" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=I. Iost;citation_author=M. Dreyfus;citation_author=P. Linder;citation_title=Ded1p, a DEAD-box Protein Required for Translation Initiation in Saccharomyces cerevisiae, Is an RNA Helicase;citation_pages=17677-17683;citation_volume=274;citation_year=1999;citation_issue=25;citation_pmid=10364207;citation_doi=10.1074/jbc.274.25.17677" />
<meta name="citation_reference" content="citation_journal_title=Toxicol. Sci.;citation_author=X. Jia;citation_author=W. Xiao;citation_title=Compromised DNA repair enhances sensitivity of the yeast RNR3-lacZ genotoxicity testing system.;citation_volume=75;citation_year=2003;citation_pmid=12805645;citation_doi=10.1093/toxsci/kfg158" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=A. Y. Jong;citation_author=C. L. Kuo;citation_author=J. L. Campbell;citation_title=The CDC8 gene of yeast encodes thymidylate kinase;citation_pages=11052-11059;citation_volume=259;citation_year=1984;citation_issue=17;citation_pmid=6088527" />
<meta name="citation_reference" content="citation_journal_title=Methods (San Diego, Calif.);citation_journal_abbrev=Methods;citation_author=P. Kainth;citation_author=H. E. Sassi;citation_author=L. Pena-Castillo;citation_author=G. Chua;citation_author=T. R. Hughes;citation_title=Comprehensive genetic analysis of transcription factor pathways using a dual reporter gene system in budding yeast.;citation_pages=258-264;citation_volume=48;citation_year=2009;citation_issue=3;citation_pmid=19269327;citation_doi=10.1016/j.ymeth.2009.02.015" />
<meta name="citation_reference" content="citation_journal_title=Eukaryotic Cell;citation_journal_abbrev=Eukaryot Cell;citation_author=L. G. Klinkenberg;citation_author=T. Webb;citation_author=R. S. Zitomer;citation_title=Synergy among Differentially Regulated Repressors of the Ribonucleotide Diphosphate Reductase Genes of Saccharomyces cerevisiae;citation_pages=1007-1017;citation_volume=5;citation_year=2006;citation_issue=7;citation_pmid=16835445;citation_doi=10.1128/EC.00045-06" />
<meta name="citation_reference" content="citation_journal_title=Mutation Research;citation_journal_abbrev=Mutation Research;citation_author=P. D. Lawley;citation_title=Mutagens as carcinogens: development of current concepts.;citation_pages=3-25;citation_volume=213;citation_year=1989;citation_issue=1;citation_pmid=2664491;citation_doi=10.1016/0027-5107(89)90028-6" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=B. Li;citation_author=J. C. Reese;citation_title=Ssn6-Tup1 regulates RNR3 by positioning nucleosomes and affecting the chromatin structure at the upstream repression sequence.;citation_volume=276;citation_year=2001;citation_pmid=11448965;citation_doi=10.1074/jbc.M104220200" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. Lopez-Mosqueda;citation_author=N. L. Maas;citation_author=Z. O. Jonsson;citation_author=L. G. Defazio-Eli;citation_author=J. Wohlschlegel;citation_title=Damage-induced phosphorylation of Sld3 is important to block late origin firing.;citation_pages=479-483;citation_volume=467;citation_year=2010;citation_issue=7314;citation_pmid=20865002;citation_doi=10.1038/nature09377" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=J. Majka;citation_author=A. Niedziela-Majka;citation_author=P. M. Burgers;citation_title=The checkpoint clamp activates Mec1 kinase during initiation of the DNA damage checkpoint.;citation_pages=891-901;citation_volume=24;citation_year=2006;citation_issue=6;citation_pmid=17189191;citation_doi=10.1016/j.molcel.2006.11.027" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=T. Naiki;citation_author=T. Wakayama;citation_author=D. Nakada;citation_author=K. Matsumoto;citation_author=K. Sugimoto;citation_title=Association of Rad9 with Double-Strand Breaks through a Mec1-Dependent Mechanism;citation_pages=3277-3285;citation_volume=24;citation_year=2004;citation_issue=8;citation_pmid=15060150;citation_doi=10.1128/MCB.24.8.3277-3285.2004" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=D. Nakada;citation_author=K. Matsumoto;citation_author=K. Sugimoto;citation_title=ATM-related Tel1 associates with double-strand breaks through an Xrs2-dependent mechanism;citation_pages=1957-1962;citation_volume=17;citation_year=2003;citation_issue=16;citation_pmid=12923051;citation_doi=10.1101/gad.1099003" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=V. M. Navadgi-Patil;citation_author=P. M. Burgers;citation_title=The unstructured C-terminal tail of the 9-1-1 clamp subunit Ddc1 activates Mec1/ATR via two distinct mechanisms.;citation_pages=743-753;citation_volume=36;citation_year=2009;citation_issue=5;citation_pmid=20005839;citation_doi=10.1016/j.molcel.2009.10.014" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Molecular cell biology;citation_journal_abbrev=Nat Rev Mol Cell Biol;citation_author=S. Negrini;citation_author=V. G. Gorgoulis;citation_author=T. D. Halazonetis;citation_title=Genomic instability--an evolving hallmark of cancer.;citation_pages=220-228;citation_volume=11;citation_year=2010;citation_issue=3;citation_pmid=20177397;citation_doi=10.1038/nrm2858" />
<meta name="citation_reference" content="citation_journal_title=DNA repair;citation_journal_abbrev=DNA Repair (Amst);citation_author=F. Osman;citation_author=M. C. Whitby;citation_title=Exploring the roles of Mus81-Eme1/Mms4 at perturbed replication forks.;citation_pages=1004-1017;citation_volume=6;citation_year=2007;citation_issue=7;citation_pmid=17409028;citation_doi=10.1016/j.dnarep.2007.02.019" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell;citation_author=R. D. Paulsen;citation_author=D. V. Soni;citation_author=R. Wollman;citation_author=A. T. Hahn;citation_author=M. C. Yee;citation_title=A genome-wide siRNA screen reveals diverse cellular processes and pathways that mediate genome stability.;citation_pages=228-239;citation_volume=3;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=DNA Repair (Amst.);citation_author=C. D. Putnam;citation_author=E. J. Jaehnig;citation_author=R. D. Kolodner;citation_title=Perspectives on the DNA damage and replication checkpoint responses in Saccharomyces cerevisiae.;citation_pages=974-982;citation_volume=8;citation_year=2009;citation_pmid=19477695;citation_doi=10.1016/j.dnarep.2009.04.021" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=J. Rouse;citation_author=S. P. Jackson;citation_title=Lcd1p recruits Mec1p to DNA lesions in vitro and in vivo.;citation_pages=857-869;citation_volume=9;citation_year=2002;citation_issue=4;citation_pmid=11983176;citation_doi=10.1016/S1097-2765(02)00507-5" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=Y. Sanchez;citation_author=J. Bachant;citation_author=H. Wang;citation_author=F. Hu;citation_author=D. Liu;citation_title=Control of the DNA Damage Checkpoint by Chk1 and Rad53 Protein Kinases Through Distinct Mechanisms;citation_pages=1166-1171;citation_volume=286;citation_year=1999;citation_issue=5442;citation_pmid=10550056;citation_doi=10.1126/science.286.5442.1166" />
<meta name="citation_reference" content="citation_journal_title=Biom. J.;citation_author=N. Sanvisens;citation_author=R. de Llanos;citation_author=S. Puig;citation_title=Function and regulation of yeast ribonucleotide reductase: cell cycle, genotoxic stress, and iron bioavailability.;citation_pages=51-58;citation_volume=36;citation_year=2013;citation_doi=10.4103/2319-4170.110398" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. T. Scholes;citation_author=M. Banerjee;citation_author=B. Bowen;citation_author=M. J. Curcio;citation_title=Multiple Regulators of Ty1 Transposition in Saccharomyces cerevisiae Have Conserved Roles in Genome Maintenance;citation_pages=1449-1465;citation_volume=159;citation_year=2001;citation_issue=4;citation_pmid=11779788" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=M. F. Schwartz;citation_author=J. K. Duong;citation_author=Z. Sun;citation_author=J. S. Morrow;citation_author=D. Pradhan;citation_title=Rad9 phosphorylation sites couple Rad53 to the Saccharomyces cerevisiae DNA damage checkpoint.;citation_pages=1055-1065;citation_volume=9;citation_year=2002;citation_issue=5;citation_pmid=12049741;citation_doi=10.1016/S1097-2765(02)00532-4" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=R. A. Sclafani;citation_author=W. L. Fangman;citation_title=Yeast gene CDC8 encodes thymidylate kinase and is complemented by herpes thymidine kinase gene TK.;citation_pages=5821-5825;citation_volume=81;citation_year=1984;citation_issue=18;citation_pmid=6091111;citation_doi=10.1073/pnas.81.18.5821" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Biol.;citation_author=V. M. Sharma;citation_author=R. S. Tomar;citation_author=A. E. Dempsey;citation_author=J. C. Reese;citation_title=Histone deacetylases RPD3 and HOS2 regulate the transcriptional activation of DNA damage-inducible genes.;citation_volume=27;citation_year=2007;citation_pmid=17296735;citation_doi=10.1128/MCB.02311-06" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. Smith;citation_author=J. Y. Hwang;citation_author=S. Banerjee;citation_author=A. Majeed;citation_author=A. Gupta;citation_title=Mutator genes for suppression of gross chromosomal rearrangements identified by a genome-wide screening in Saccharomyces cerevisiae.;citation_volume=101;citation_year=2004;citation_pmid=15184655;citation_doi=10.1073/pnas.0403093101" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=E. L. Sonnhammer;citation_author=G. Ostlund;citation_title=InParanoid 8: orthology analysis between 273 proteomes, mostly eukaryotic.;citation_pages=D234-D239;citation_volume=43;citation_year=2014;citation_pmid=25429972" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=R. Sopko;citation_author=D. Huang;citation_author=N. Preston;citation_author=G. Chua;citation_author=B. Papp;citation_title=Mapping pathways and phenotypes by systematic gene overexpression.;citation_pages=319-330;citation_volume=21;citation_year=2006;citation_issue=3;citation_pmid=16455487;citation_doi=10.1016/j.molcel.2005.12.011" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=P. C. Stirling;citation_author=M. S. Bloom;citation_author=T. Solanki-Patil;citation_author=S. Smith;citation_author=P. Sipahimalani;citation_title=The complete spectrum of yeast chromosome instability genes identifies candidate CIN cancer genes and functional roles for ASTRA complex components.;citation_pages=e1002057-e1002057;citation_volume=7;citation_year=2011;citation_issue=4;citation_pmid=21552543;citation_doi=10.1371/journal.pgen.1002057" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Molecular cell biology;citation_journal_abbrev=Nat Rev Mol Cell Biol;citation_author=Z. Storchova;citation_author=D. Pellman;citation_title=From polyploidy to aneuploidy, genome instability and cancer.;citation_pages=45-54;citation_volume=5;citation_year=2004;citation_issue=1;citation_pmid=14708009;citation_doi=10.1038/nrm1276" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=Z. Sun;citation_author=J. Hsiao;citation_author=D. S. Fay;citation_author=D. F. Stern;citation_title=Rad53 FHA Domain Associated with Phosphorylated Rad9 in the DNA Damage Checkpoint;citation_pages=272-274;citation_volume=281;citation_year=1998;citation_issue=5374;citation_pmid=9657725;citation_doi=10.1126/science.281.5374.272" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=F. D. Sweeney;citation_author=F. Yang;citation_author=A. Chi;citation_author=J. Shabanowitz;citation_author=D. F. Hunt;citation_title=Saccharomyces cerevisiae Rad9 acts as a Mec1 adaptor to allow Rad53 activation.;citation_pages=1364-1375;citation_volume=15;citation_year=2005;citation_issue=15;citation_pmid=16085488;citation_doi=10.1016/j.cub.2005.06.063" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=H. M. Tang;citation_author=K. L. Siu;citation_author=C. M. Wong;citation_author=D. Y. Jin;citation_title=Loss of yeast peroxiredoxin Tsa1p induces genome instability through activation of the DNA damage checkpoint and elevation of dNTP levels.;citation_pages=e1000697-e1000697;citation_volume=5;citation_year=2009;citation_issue=10;citation_pmid=19851444;citation_doi=10.1371/journal.pgen.1000697" />
<meta name="citation_reference" content="citation_journal_title=Nature Cell Biology;citation_journal_abbrev=Nature Cell Biology;citation_author=J. M. Tkach;citation_author=A. Yimit;citation_author=A. Y. Lee;citation_author=M. Riffle;citation_author=M. Costanzo;citation_title=Dissecting DNA damage response pathways by analysing protein localization and abundance changes during DNA replication stress.;citation_pages=966-976;citation_volume=14;citation_year=2012;citation_issue=9;citation_pmid=22842922;citation_doi=10.1038/ncb2549" />
<meta name="citation_reference" content="citation_journal_title=Methods in molecular biology (Clifton, N.J.);citation_journal_abbrev=Methods Mol Biol;citation_author=A. H. Tong;citation_author=C. Boone;citation_title=Synthetic genetic array analysis in Saccharomyces cerevisiae.;citation_pages=171-192;citation_volume=313;citation_year=2006;citation_pmid=16118434;citation_doi=10.1385/1-59259-958-3:171" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=O. Tsaponina;citation_author=E. Barsoum;citation_author=S. U. Astrom;citation_author=A. Chabes;citation_title=Ixr1 is required for the expression of the ribonucleotide reductase Rnr1 and maintenance of dNTP pools.;citation_pages=e1002061-e1002061;citation_volume=7;citation_year=2011;citation_issue=5;citation_pmid=21573136;citation_doi=10.1371/journal.pgen.1002061" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=B. Vogelstein;citation_author=N. Papadopoulos;citation_author=V. E. Velculescu;citation_author=S. Zhou;citation_author=L. A. Diaz;citation_title=Cancer Genome Landscapes;citation_pages=1546-1558;citation_volume=339;citation_year=2013;citation_issue=6127;citation_pmid=23539594;citation_doi=10.1126/science.1235122" />
<meta name="citation_reference" content="citation_journal_title=Current genetics;citation_journal_abbrev=Curr Genet;citation_author=W. Xiao;citation_author=B. L. Chow;citation_author=L. Rathgeber;citation_title=The repair of DNA methylation damage in Saccharomyces cerevisiae.;citation_pages=461-468;citation_volume=30;citation_year=1996;citation_issue=6;citation_pmid=8939806;citation_doi=10.1007/s002940050157" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=L. Yu;citation_author=L. Pena Castillo;citation_author=S. Mnaimneh;citation_author=T. R. Hughes;citation_author=G. W. Brown;citation_title=A survey of essential gene function in the yeast cell division cycle.;citation_volume=17;citation_year=2006;citation_pmid=16943325;citation_doi=10.1091/mbc.E06-04-0368" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=P. Zegerman;citation_author=J. F. X. Diffley;citation_title=Checkpoint-dependent inhibition of DNA replication initiation by Sld3 and Dbf4 phosphorylation.;citation_pages=474-478;citation_volume=467;citation_year=2010;citation_issue=7314;citation_pmid=20835227;citation_doi=10.1038/nature09373" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO J.;citation_author=Z. Zhang;citation_author=J. C. Reese;citation_title=Ssn6-Tup1 requires the ISW2 complex to position nucleosomes in Saccharomyces cerevisiae;citation_pages=2246-2257;citation_volume=23;citation_year=2004;citation_issue=11;citation_pmid=15116071;citation_doi=10.1038/sj.emboj.7600227" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=Z. Zhou;citation_author=S. J. Elledge;citation_title=Isolation of crt mutants constitutive for transcription of the DNA damage inducible gene RNR3 in Saccharomyces cerevisiae.;citation_pages=851-866;citation_volume=131;citation_year=1992;citation_issue=4;citation_pmid=1516817" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=L. Zou;citation_author=S. J. Elledge;citation_title=Sensing DNA Damage Through ATRIP Recognition of RPA-ssDNA Complexes;citation_pages=1542-1548;citation_volume=300;citation_year=2003;citation_issue=5625;citation_pmid=12791985;citation_doi=10.1126/science.1083430" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Leveraging DNA Damage Response Signaling to Identify Yeast Genes Controlling Genome Stability" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5.cover-source.jpg" />
<meta name="twitter:description" content="Oncogenesis frequently is accompanied by rampant genome instability, which fuels genetic heterogeneity and resistance to targeted cancer therapy. We have developed an approach that allows precise, quantitative measurement of genome instability in high-throughput format in the Saccharomyces cerevisiae model system. Our approach takes advantage of the strongly DNA damage-inducible gene [RNR3][1] , in conjunction with the reporter synthetic genetic array methodology, to infer mutants exhibiting genome instability by assaying for increased [Rnr3][1] abundance. We screen for genome instability across a set of ~1000 essential and ~4200 nonessential mutant yeast alleles in untreated conditions and in the presence of the DNA-damaging agent methylmethane sulfonate. Our results provide broad insights into the cellular processes and pathways required for genome maintenance. Through comparison with existing genome instability screens, we isolated 130 genes that had not previously been linked to genome maintenance, 51% of which have human homologs. Several of these homologs are associated with a genome instability phenotype in human cells or are causally mutated in cancer. A comprehensive understanding of the processes required to prevent genome instability will facilitate a better understanding of its sources in oncogenesis.

 [1]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328" />
<meta name="og-title" property="og:title" content="Leveraging DNA Damage Response Signaling to Identify Yeast Genes Controlling Genome Stability" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/5/997" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Oncogenesis frequently is accompanied by rampant genome instability, which fuels genetic heterogeneity and resistance to targeted cancer therapy. We have developed an approach that allows precise, quantitative measurement of genome instability in high-throughput format in the Saccharomyces cerevisiae model system. Our approach takes advantage of the strongly DNA damage-inducible gene [RNR3][1] , in conjunction with the reporter synthetic genetic array methodology, to infer mutants exhibiting genome instability by assaying for increased [Rnr3][1] abundance. We screen for genome instability across a set of ~1000 essential and ~4200 nonessential mutant yeast alleles in untreated conditions and in the presence of the DNA-damaging agent methylmethane sulfonate. Our results provide broad insights into the cellular processes and pathways required for genome maintenance. Through comparison with existing genome instability screens, we isolated 130 genes that had not previously been linked to genome maintenance, 51% of which have human homologs. Several of these homologs are associated with a genome instability phenotype in human cells or are causally mutated in cancer. A comprehensive understanding of the processes required to prevent genome instability will facilitate a better understanding of its sources in oncogenesis.

 [1]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5.cover-source.jpg" />
    <title>Leveraging DNA Damage Response Signaling to Identify Yeast Genes Controlling Genome Stability | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"us9L4F_1t3Av6dhC5-mRDjs4PUprrY0M5I2bAKm25RY","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459841","apath":"\/ggg\/5\/5\/997.atom","pisa":"ggg;5\/5\/997","ac":{"\/ggg\/5\/5\/997.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/5\/997.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/5\/997"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/5\/997"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/5\/997?cited-by=yes\u0026legid=ggg;5\/5\/997":true,"\/content\/5\/5\/997":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022iESRHluuyRnmPndHPDVPooKk+t+X5Ic\\\\\\\/IO9ZdN+5A7GqdmP2LfKq4KcSLJAIJWrflPSKkJfcl3O3JaqyURUC9ySb1HuvrW0VZlP3xqPQqUWPVpDGDdo9lLhHPOeG7+EPXy+bAZR0ld\\\\\\\/oWJ3oClWw9qVoY8cqksWim8OKwt9njofZ4YZQj\\\\\\\/b4SH5NPOb\\\\\\\/+Om\\\\\\\/xeq6kkvcQsBFeEP2RH4H9nkmPGMoNlkR6wTCeie\\\\\\\/sla38cVqQBy0\\\\\\\/9e0A8zYc3sHDdGChT5\\\\\\\/ZIBXibfzLiJzXSIBYu4XEZxwc9uKhxZptTdEJ2JodGY\\\\\\\/CwxofRK9Xlf3cE6Ns3Oj0B9Q1aFzALFQZuTDj52BYUosXBjeOpWNVVGsKY90t3tf1BIAlqtzzGZ90iDNO1QYkS9OMFMyMJ74snHZS8gw1KCnCKggBOIE0xTFL0fUpAxnwce\\\\\\\/aDBAF5SHI6szUnJa5se9uh2ilWp+2MbmtyTLi2wEq\\\\\\\/1z5+WX8BEet5LwNbpVS\\\\\\\/GHoKxme5hTj5j51\\\\\\\/jf5YfJ+yepwnQY2HhMqQkEXd\\\\\\\/i+034MED7lLmcSIh8sp+8NGXt6vcvuU8Nm7FhLMSF43ucQja12VpqGeVijTAI6uU\\\\\\\/xVWGW1SwHolSL\\\\\\\/PJuuM2bEn2ixqeOm2ztMdGngEyWkXuFlqCHo5b+FkqVmmuzcwXkogqktrEnAUFwEQsXpR\\\\\\\/s1i3yEQ8Yv940qcrvTeAKJzOCN2\\\\\\\/Szatv3G9XC+mSUG87V2ZUL4dYnjOUgdRBaNiYune5SAGqGTfQVGYZrOqvdO\\\\\\\/yn2odm5rB5VAtSsbyO1s4G9mYO+ZM+ucLcLynBsozm\\\\\\\/zMV+TKfHmF58Usg18\\\\\\\/ZcFtkBUOrrcFYmUG0z56pqEeKBkpMqQiylecUtx2WSeFn0C74BqK2KtqTECEK5eArQKlZDriAUGkqGInmmFlkRWqatu4yr+Cvih1iPIahh9wsFSLKpJB3F9nIaquy0ieIE4NnRKeMX4F\\\\\\\/edH8kQ3EzYlP1lpDQ\\\\\\\/vsZW\\\\\\\/koXCqhMEkrJ+DN10dy3zSiJeCLUHxY11wcxASoeDS2457VKOdZb\\\\\\\/nS8ecmJX7EwkTQJ1a9DoQFofvopa5U\\\\\\\/CihGISeNgyxJxPyBPDOB3xFLuBkc+plRfzsgtQAJJGWeQp7Mf002uy70hMMniDCIyHrvw043QahVuwcujbaYilt6MKLY+IdOGGt51A85GHl0Mkk2d9xNOm5W4F86RqYPUT5y3uhNswPZQjmZAI63BDOxfxwCBYsaPoi93y59wORpY8V5w8VpUsNRSZOJAoYy\\\u0022,\\\u0022iv\\\u0022:\\\u0022Ot3jItBV0k6QSdzxvFPLbw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00224ac1c7dbfbbf7aab8ba390a7f6aa6bb58a5f8e69f48c844e5ef3bfbad4a38714\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Io82DY9S36ClicXpLs5hUrycK3a8MCy2Y1B1gnvmFYD3Qxm71Q7Jx+o9r\\\\\\\/HiZztIyunzZQ0ZMIyhN0kCzaEjdgIR0o9humRjF9A2gSESWwhh+VsRZuDn5lAf8fJ6l9lt50GI8yF0kjxaS4hz1ThBiLGn+mOknDDpZUO1\\\\\\\/39aVWn4uJFelSl9i0pg0fQXg+jcAiKHA3ADPPAF4jzZj5iaTGb3gOS7tmD8qzHwsjh776hcfFVtfYW1FS+xRtJa4GllPl3EZUyGlMvBSrgH7FoxFOWTFhW9LRClxbeib9q+4sWFxMXEbVtJnqLKUsjv4cp\\\\\\\/KLamlK24vIyz6vp3nre4CD6Yq28eSSjLwqtTlNEV0hXRBxLFQsiL+Do482JUffaaj1yRpmYlubhJn2Q\\\\\\\/szCNCOd7I\\\\\\\/7YLbd\\\\\\\/t7tefT18Fxh9NsbJDyRjzxMhIpo1ZJwLHzWas+EFHSO4H2L17jMzETY9isq1Xv+mpjhpw831TkZFqsgprmPixfRCQ68GbiHhQok0mLhewqcRaIHnfIkctUoSHE5necq6KupIR4c2D4JAnsvu1Fw0CnqQ0zJKI8s6JjoWKGb\\\\\\\/DopeN\\\\\\\/dSVTHOxnJNcPGeuIHQWIutLECP9ES3WnOBioaNhYzKDoJoOL4o++ukW2ikdxOZNAx3sgsJKKshpOKjV6489JKjciBtxZyGaPOtkKiDLgwW\\\\\\\/vZ1g+lSCbTsEtwtWWR5oOE8Sp8Cuf8+2GRnJEKaNJEw3bOFkaN0h4bVa06+13Yw\\\\\\\/HPQeJP29cFk+gjPPpgdbhoXspHREK0MWWgHN14RQQ7mFOmc20k+\\\\\\\/yUnU1\\\\\\\/ywVglatQpNVKE8WL3w9sOPed4+EIoHjf00f0Jv12KqCvVrdkMG4tPGS5JReJh4O+EgcOJ\\\\\\\/c7wBrPSc1CwUqUnX5\\\\\\\/SjpW\\\\\\\/S2ArvbcyFKZ+UUafiQAPVReIOFlT2B3lCNHbn6DG0Gr3AlMI2k1gPUuOmk\\\\\\\/7Tyxy6KwEXrYETVkUe3YwIwfDHOitFin8SjQzmDykBarpJPWz2VZaamz7T9JsvSJrSctuHpUXtx5n0t1cT5qPeJxVXZDyQptDg9RPB\\\\\\\/dcr8ne7oROYO46XDTmlHCg5KAMkHqnA5r+RENMN4IJ97uUseaxGwA930\\\\\\\/3I4HZXRhLYgNvuR1QVULuLINvWR9NOry1MCBDNNXAAyI23ytJvsf60aBIclr0Ygy9p8PCn2U9VsL0GIibLK+\\\\\\\/B3uiwH6R6kjdhNqtWhQA19ffLo0OpmXHwRL3pDsxO7Nhsslcww4+uVyZRwWrwjacRZhOit1n+R0fbwlV8PV\\\\\\\/vlcVYWAguB+alg9uVnPoExjKp4o5KpwIhwObJ7t29iVAi0FLxIKl\\\\\\\/lz1wIq5PxmRtbHvh4CP3nn0g01hqfIWWvMTwCiWVy9d7c0oUgcW8gi9NoFjgQjmZvxJnlybkMeJivB0vMSK2Ro\\\\\\\/h4QyxJKtx6kEXgr8KJjlw7FgC33LdjQp2pdtfDLsTHIidFQptRrZJymwb32LitAx8Ges5CavgqV4wf+WBMW6SfJUV5uUBy+uohaczer\\\\\\\/c7O3x8GKXDsydUkceEAKnVuWQgXLSckLhdtGbZsbv\\\\\\\/0hOfRq2AKXYv++9GO404mEAR185h4aDy5xzo9v\\\\\\\/Qfkt6qixbMk2WB9gjxBxYQjXge78P2O3UuVrsH33\\\\\\\/P6dz9r0hTt08nR9gHpAQEXkeJ1lgfeGdTSglTpkuhr7oB2GiOJZcHBIi4tavHSm0EontvWLP3p9NXou8aaV9PrvL8ab\\\\\\\/\\\\\\\/CbNFQ6kkOoeCK+V8C2BYCVpvTeJ9EI1NcvYLSpUNvy2pHBVNJuIJUF8Ojg7d2QyBKab7zn6WLKKR\\\\\\\/81uxdXEuk809sx3ZfvZpmyuIBP81pEgtPT0dA9EhfmJIgC3dWzuuKqLonab0yoan0NjU3ZyuNXZrfvs1Vovj0cGnmcW02IFtzlrTQsQIH6zlEzQPXbvFZ4iI7kbXqyiiJ6B3dIi75sYr1rKOw8pqxUE5GBs0RkUNz6NHtfr\\\\\\\/erUTzDyDjib3LfI7sdMEwVTRk1f01uCgiW1l69jeb\\\\\\\/uYkCZ58SGi20NmQ1eTkVuZ3ndy3PDRyhEWT3XU23cmrSBidA9AvIH4RA+mcJAvebW+mCD79Q6EkCvYKxlxW8DP3CjajNTZNrVRinbQbrX8bAod\\\\\\\/TtUD6zHcwtppz6wVXT0ugmb09Qopt8l51e8DEEiC0WCKboXGv+3ujX74DnY0b2hEYawOfLx7O6FIrg84SgQNM1jTiM\\\\\\\/mWSSrx60HBFpMpIAKfPyvAi1OCQPAyHSdf9juI6e98wXOuz8GFzm0bvugfQkt15\\\\\\\/qAzNa83XrEUDgeYnnoDEhX8rC05QJJuT9MkiuqxQihnXy\\\\\\\/TFzwfj8N1DxbFg4hh811fRTUQtWIqACmOmSLph\\\\\\\/GQrsUjsgkE57Km4iDtLMd3AfbcyFCi8BZCHTAIv45ijWVy4rW5pPGUDI72Ij1rrP9xY8hTdteCmyJKs+kUCn9MxEdxIsXPrO\\\\\\\/5YKTilt5GDHtDGwVcoymCabU+GsfZGDkycTpDmws4lHggGQRiqsDGMfEnBmN03y0e48LaietRurxokScacapo5B2MUlkEJ9RvK+jzIBIBsPb5qf4Jrk4JdG\\\\\\\/AvL05xA9x1XSBEBzvEdKUHFSeFawXmqiuS+m0+UGpN7PpBIvGalKHK0oOPan0eDkrN3ydkG+ScmpOJIGgltmJUI\\\\\\\/4xJUErD8fHEbuASBNqbMhWX\\\\\\\/TH4E6kJ9jO1WY0WmcHMrm9v6yvZGuX6HZFFzKDAhnreOyHH4Okx+lIE0KWVMIIVpaQ105D3XaIANfsAAVmhHBIR1HbTOmhNMd6oLozPE46m5x\\\\\\\/5SvayQWQPp9xdapw56qAnxPQS7GeHmeKNJZUIGSC3wPX6SyuspSbluhuy7\\\\\\\/qS57ZXw10nZ1eb2xaRURpW7GoAT6vzYqQT4\\\\\\\/qnUTNoECwEoHSLRtjUCVmoBdU98urO8\\\\\\\/6iRI+V87lvLrwspECWrzOHCXc+NQDzz1ISNTJs0sFePbLqptSRFXLKtd21qfD+UmhfMPfcyCmSeiPEVrmFgHtFTMratdBY\\\\\\\/+7qkNIHaBP2cguFV0kw5fVzB1fUGKM25zbVFbF64LpcqgkLxOXWq7238Yns+KsGpdbfgeEVYtzxV1vVUjzrAni5jClw1eMzR2XLMIbm43ncwvNrWcUc3cRJI5G3cB47SwNfdEq04KlguwBDvxO8epGcnJbrfsoMPbxxlyo+8fFJbW9IQfN3R9pa4rJIhMn3nHNYiLe39okOC02yQuqXxmqlvPXDmErmC+vU\\\\\\\/p8ouQdI0xQ5sMru5TuuDcaQ6t3f4jxsntXo7CksyiOYONNHN\\\\\\\/AElysifdu0M+sQelxZGx3hGFlY5G6kSB\\\\\\\/c8bABB8pX2ix\\\\\\\/fScn7W+IUZQe7Ntp9d\\\\\\\/886fWnDNeK+ISDFFEZ8JTpFEUSFg5bcIPDk3gr5fpYNcLP713h2g5D27Bc0QUZu1MpPz6HvUjCoL4vgbzVkjr8tK\\\\\\\/u\\\\\\\/okWGmEdAWy0pWNXuxCgx\\\\\\\/yu9Ofgd8kls8DX4BilaM715wbNlF\\\\\\\/N6U5YRUv\\\\\\\/3xy4OzNqcrhzqVB2T6phOaYXvUIkaSB78WK681v4Q96EIFbh31acn10ahqCar0MHBe7MqXW9jFIwC7ExHHIA8cbZ0bEMOmOva\\\\\\\/apUhrMsp\\\\\\\/m0CP6Qznlfk6HdzgWILhjFbdi2rKXVLFCNzP2zQT7a2EdGXI8c+rH39inD\\\\\\\/T9tlllIGxWRjkEcohFE9V9zBEEXfBMlV7ZCmFP4sS6N1Do4bAiBIgpgsWEcseazgdNUDInNkcv6JzVwJgnsDBR++Him0EH4QnUuwNAxBihsjGSyEopixGKDMuJi8Khqfz2+l7TgjrXCpIbmFMnRM2eFm4hCNKWqcPuRfJ5l17fxX56Ws9ynJ8x\\\\\\\/7MRVMewX9ysuStav2vqcDLIy96OCW+82NI67RunIi4YCV0cXaNojjcWIXCwjcrXZX7akafYmA8rv8rfvftC7f3ubcQ6ZQsZi1tult8TUwBlonGVCIc9guZeZ5J1MfsvZTMuYeRLin29lLrOUdfQy7+G8vJHxVHGTrM4mbjPkS\\\\\\\/slSrwlYll9YUWjZKiUZ1zLUw0wfqWmYlcc3g9EBnS0HlZ88QyWOiBw\\\\\\\/pczrSB30DIv5miigfqwjW4WRZUW1sU1kn\\\\\\\/z9ta0a4cyaaT22hpG9OHMLNRz6SYLDIvfe51luEh7mG4qELQYk\\\\\\\/ThR4W3ASl5D5G26NhUBa2\\\\\\\/h1Da1AmxNXxzqn5mNfhxXGWDKDqev9OzaCr67iVJCjQjDe+XG87XtBASvpLH8h2oSX2EHcSN9lVBWhvc95cH7io0FCUIrlyytmjwxp\\\\\\\/0\\\\\\\/PWzs4b019eSeLdeKbcw510pHuH6ZYSxsgBXEiTKbr7NwxrC\\\\\\\/OpUhYVMkFP5bUYT5QAEZgL2oHDxlivI22Ws9UREBe1jLMCuo8Mr+d32gid5oyiMMscz2G9VSH1kG9R65HpUMsaQzv8swvEO5lYAqvFIJ7WKDGJQIwrNUcbEqQ8hcyvlHuUzf\\\\\\\/wxVYrilFzEdsH9h39vudP\\\\\\\/FPem3aV\\\\\\\/aZNcsox+rT8au+gJ6ypwa589NtIGa2WQoUwTHuTGERCENgxGSkaVz7nUEF3Sh7SPXYIXg5+PEWYa17vrpQW9rfgGYQu6TGufPZEBwWk7bAWKyoaDxZfVGVeVKrCBPLemOtaQpXd15XTudS9tobe8+m+N+gnx7sLcn98synYkH0klrxdsGIJofBf\\\\\\\/c\\\\\\\/54tz+cdTn2Dr9Q\\\\\\\/JQm4FQrqqQpzox2t7km\\\\\\\/4CpUmvKtlWvoiytBAH6uqSxB9LDD1C9BAIC6smQcyeFa+DlnIlUtoSNhRmNjOJRh9eWnIaHyY1egK3fSBrj9aiUnXqojbV848qXEG3qfb49w00OLJ6zkIIOpfSbFHA26s\\\\\\\/Y0Vc9A8OK6kSY37B6MjbkYG6wtEqogl3vulF1Q52S9cRO7GUsgvVrq+zDIHUOTXBKA2UnBL7JmUwBNQZIWr78XTACmB5i9XMz9PdxlHTnPp8nO+96REnf2g+NPC6l80doiTFdCaUKxliPJKAl99slLySxRDaRNXlllnpGMVGkqwqBL4YiAGI4dvpa2nZEr65Y2mhQUTvrFlLa9L2WXWdoAMbroDbzNQ9EtEm4Z19VRreXXB0EjaLJZJpXu76VpkQAeSpYNnLCSg0bweXB69VAC0ssdSAFZi\\\\\\\/q\\\\\\\/9h8nwJi70P5+\\\\\\\/zIZrqyTXCf6xPUMS9G8NvAzXbrBa24Yx5VtnxrEwb+RgMV6Bwg70IUlqVcAhvOcZC0lIxFcjH7cZTx4dCLgvZINGmmaqmtOOF1Yc8EGj5NTvUq+4hyf0X1XwFDw9r+KEzexgJi5B9oqg6X46AzGzlQfBUdYgrDrkz\\\\\\\/jhqPKuQT+r0nROMBNZYCtRx8pIoZMt+prVULFgeJBL3\\\\\\\/U0M0tdEX8mcqXC6a8zjeEFC57CtASTWaqOF9sjANHnhw+GxCzqywT0knFnlnZx5FuXR4YYYBo+vlL42BI4GAc3oJSLZPfAjc\\\\\\\/CqYlYra\\\\\\\/CrfGNtVIgk8qOpvpnP2CXAD5EwF9qyhSff\\\\\\\/k5BMlKQ0hOMAXA+X4PUwYtllobxDfUucOB12JGXD9qEtvPqKrDwpR6mjR5cL3rA3kPe5GV7bKvVW5WIC38z9+DuryrQ17Vr9q9gOxA5EF3f12sy8B+zwd8jodmed2LgBGQ3i6rnyVfbdUo3Ip2\\\\\\\/K5f3wTvE5sqif+R03wBHy8IxFSB3L7jEHUICDj+SzQBa4DN8VakbWd9O2vUpouAL0aO+nUMEHqoe5JY3Gi7j\\\\\\\/9Q5Q6+C5gke1y40Z1gTSdH2BM8YW97qVHnaI6csFAV8Z0cvEtCPzkvAYGfj6zTfJUNhj+vlgVpZoQn2AYEa76pDmoIoqVCBSniUtsCm6MywjHpJZ7GFmi6exMF7Ll+kdgHr5LPBqgGWqTpkf5bwyYyDLfOpQ93x4tgpRlblLMWwfYi59EytloYubSWzzGLpUCMalU5i556Dqy7KERbxXKe9M9OUnzwu4TZz+k6UHEMNqBwueUOwG32WbNtNuUKq1Tm74Bie9ZDwxpVgGWn9TXOwvsgRu\\\\\\\/oFTkPp+4qok9Q99kA7XD8HGqnIDaw1AuYbNLYbNKfUqcn96srIq0Ks5ss3vAlOOMHahSkMhRRWwB0il6RGrsJozUi9LTCGKh3iL9CaMxrihCtUfSS\\\\\\\/cnLgAEUqyS9NidK9oNOYXig+oSnIKEdsQUtMY\\\\\\\/5X\\\\\\\/gYDSd3luxQT\\\\\\\/g38pStR5EmjcnvCJSk983MXqUzXW3SivwkY9HbNWPBkM8bqFky4kgyBNYTC4xaiBpnETwaU5lr7tgHbEA2fFXcxmOKtCggGbq5ZS\\\\\\\/xZqwaaohhtvK7BiUXty9fYwKYSMtxpHKCkSmhSr31JshNwvK5Z6tRe+N1s+y0jh6XJqc7o3y3vO60uC60tyhiDLT\\\\\\\/ncJ16H8RxyiZQz3R3bhzueQZHjNaZYM0U7YovmrniUsReXEqNbcq0gLNamy6wlJgEWWyGFn+EIPMyzMdystn3eLaQJ6ZAzaBU071XMRMuzkYMLUF4IPDFEl3qa1v33W4Vhj\\\\\\\/8IN7iLqjCRjj3S9YMrYgvIYyIT2QOm\\\\\\\/aGsSotm7d1ad7EYM0CAMJPieA48LmLaz\\\\\\\/qyAL8jcT1wB7jqcB4MYLfquhj4y3Y+QYTaLwotgBPRQDPuWeRYZm9eBGQIpKcUkxhMB4pJ8WIS5S3gKwk+EKI8v\\\\\\\/Beimo3aQJfuMcHBMeFjEt1\\\\\\\/3YewUR0fCrt4p0wma9sGbQzxBSrxvF8\\\\\\\/I5K2FbRyVusIpW2TsXjvVJqLcLTxqlJxkc2GrYluqsf7UEk7FZ57yRCp+2GsdTzBgC0jy4ZWOMiEoxwIn6PzsB0g4EuIAjcFPak42gPlfgM9G2QxRRZDVf+\\\\\\\/oObRpUkhfewiAgPBmNfh0nVEuNLmA9RikLwJRWwsQ7Y50ucpTmqwBEMWqRE2K8XOEXXsFfZBjZfd3yq3Mvz0Ee0dsMSsz9szkTcwTvbehXFNoXjdh1R+Xw2iypgd9T2DhzBUPyjsBGydNQLKodzPWh\\\\\\\/00XFZepX1Yu\\\\\\\/3c1LS\\\\\\\/LTFDgZGPCAqB04YLx91A+eV8TasfheLrKXEoGx\\\\\\\/QUig2ecz3dlBvq9RHT8PRPTCVHthKQI0VDyxyouFgNJdpBULfEkhRSSnc2kuJclWliG7KmDk\\\\\\\/lhV0+cAYyDYI1nGZh6faU4ilMivP25sAe0iuKwVFFz9t2rk7oiP\\\\\\\/7IGhe7kXV1rW02YZ1nHf34wocOMhObwoo82XxIoUs\\\\\\\/kZu2rVmxVAxM3ZRfBXKzK7phxsyiOQtKM3XB+Gqw6wHA2j5H9FXTwB9AeGrqhOdkO7LD2P7RrQtW070M5eG4wcnuuREyXd22cEVvKLhjGKB6VLDlCzCqIfgpzLAKVScOgOpVXRqV+Cxm38\\\\\\\/MKVhJqXv\\\\\\\/bRhyNOM1ZzkQsIEPkZSsCS5oYmXZWAa\\\\\\\/\\\\\\\/EJzOm0Qrzz3qkvWBhGVZZ11fIweCR\\\\\\\/RweKn9nYV0+wvJ7\\\\\\\/6TBbgQy\\\\\\\/U3NzloUBGqp2QmP+KZ3k6+xhOn+JbQDquSF2m2vnpWb0k8WzIvtqfSYL8j70LjzbG8\\\\\\\/wY8zUU7sJRzJ+vVT6JpH7+issaWuSh8Gsy69lMeXa3xg8nB\\\\\\\/+sNs5te7w9qcwiK3FNIJJwCYEGiFHH+2xVzOt1JpZegDl8YfpDlogGfldmH\\\\\\\/4NnI3snkM4aeKvwbJxlhgLGfDo92im5OfcRBDJMsDNnFXxcdSBiJV3n2wIifPNGerKA71AdNfnLA5VOdBbdYxJCI++T0iuv27Z0V8zf\\\\\\\/tjUCoLHbEIyEDOv3Iyb9msQdRohk0BA1Gw6cy2LnfF3P6xSSjzIYUkNpNENKx5k7pwkn+s4ajx3xSu9+oM3ZoEJWgQcFbsOPAe9EfCgUvxq05JfUrq3UoPjlPxg23o8pbRlRkiV71cL9o\\\\\\\/6M7tz8nuLyFLcbO2FjG\\\\\\\/hGdeCHxZlGW5UVe3G5S1vfDDqgZiC5lWIlSG\\\\\\\/pnC56SWgEDSvzs7qCsj3+9Dl7iAdti\\\\\\\/v+c8zUoebTDZoIhi5EQLRC01FE3VeGzL+joWH3RXbc1e8+vlx7UgWkjuWSZtlPq4+WJ5n5biz7n039HRIgGNCYxXNFGUz3e0aVL3sfEnnOFfLIiOQgky2HyReaw9A3GdPN1GBNmcq1jyxPO1zeYASH\\\\\\\/S0moVJB9FNKn97fcuwZZjArpd\\\\\\\/rMbdt5j2VS2bhhJqDlXNqR\\\\\\\/YcMj7YHFnXlyv3IcdBmg1IyWbf3miusTLMCvBgLbMGlpAFfzEHsHom8AsY68oxXgRuV09kV5UeNPPKZwdapZjPcf4DQlMQqJqNk+jUxTHaeb+ehlPRIvFd+6PgAq5qHDeVNxtaFm6x4M0F2J8VHEZjBxmxPVbFBlrUX534E8ogOajF6AViShHt\\\\\\\/MUzhhcawPIHfyk9zKjJpSKUp8E3TRJSZS\\\\\\\/cL0ZPQ7MoyzbWYGdspR0vZWTAbJlX5inMrZI0CMqZR1TXjZVzhWkksePcEHdg4F0ZkGhfPiHU5KOOoiIlok4pthFksw3AwxMA2KNrvvfMJo3xZeqQGLUVUgN8lSFJU8C5KkaaMG0F0u0eCa4vLTdrZjb5nsKbeuL1OKUmUZKLmBW9Yv4FOHXJEf8nQsKwvzkjvkbg+L52BnzWIB0zpk73LClYJDizE8SPxBDJrkY3LZ5Cb4vJEYnQA+kJY1yPetwVp318RYhK1UVVjmeWjkLirOkzxaA3hSZGvvKTparjSxeL9r\\\\\\\/qRFHJS55725O+8Y14azWYI5f7FSkSuFOW4p6n1Imd7mru4bX7+53sdDqa9dl2w1JZfJ4DVid+WtFLIlcDYOac\\\\\\\/9RFxsqGG9JCb8IjEDhmHU4MQe7HpY8Q3p6N0kj1wV34lKp3b8vIZa7t391obuFW4a8zUmjE1Awp\\\\\\\/Zdgry+R+33ddHzYqckqP\\\\\\\/ZhkfKOF2zYQ7zeFuubIrkJc2c1wpyaT3MvO9dFunjqtjo5imYtmlELDhLJUNRkywq5yrqUCa5y2RSqQi5QL9Ijvs5vegFYxT6jD7EjZSrfxMdXUrJQIBgwKpCOud7HSVBCSG+TVBgbxVct37WkINlHD93HYkfK1mRZay5jOkavtY0Xv+nWPIojvFt5SkvQZmN6rOBRefimszcWQkqQenjULyZCcj+AGFYTBPgO0INTgGXFdVt6PZEO05XLwEuY4woIEzpTsxKR6N\\\\\\\/1j8Rh\\\\\\\/lKlWxCvDHxfCskbBY3YuZZgk5NlaLstEXvvZ5a70Ggi\\\\\\\/N1l2uQiQdu3DDSojjFt04fszkMOTvw+KJKryP421ry6KVT2RZlsl0+DchkrVGhR+J6xNDoP60hZlLMxVL6MkkWRUTApbTzI3PUN2IYeuLucLcaJzJRUQcQcYMUWLLI9rEsaZOARCcThUJ3inANCvXaL8kZa5CKIvEjxXL7AJb1ps\\\\\\\/bBEwovztOYebcsblGXhBjzigMv4NU+OYU58NuV\\\\\\\/wWOCJ7oCmpLaI8y6MSZGon86tkbePIoDphvdT\\\\\\\/KG2yZrA89eZU7mrNvlDWkrkM7F69oObO0VPbfJ8CdSGwOa5ENA0bcEHzhliiALViUB6br7+gq4aUmVyOCpsthf0LTW22oTfZCvAKdXXfCNOHgai37WI3bb\\\\\\\/lyUfIs27xQkshA+qdxdsA7u8fF48ZLx5Uy7jNee4P0M2u2EUpzcx9YdoLN7WbbgBX0oQyw2tMLKcZ2IuXam5sulwai7XN3c2GZSWextmpAmx7KQKvCfKaC\\\\\\\/B708zFziPDiEOdmjusfW5o+2oh10qLSfldlqi5yHYy25r16FZKHur2qwUUU3q7IMwukQIrN8Ji2SkzsLWwEAAtVFBzQCcf45kFnRFJb6dDvMBKlJbf0dyP3E63ZQ4GiuF\\\\\\\/a8MwvqfUVk6UUijTupwcQ5ZRmL4WxmjXmhBbfHJPfIhjUzxnCnqMsMFLyDRknjBteBGRvbqIuCccDBXaaM5H3B5Fb+0QyspzgrCT6OSYisMU7Za5etdusDSZlRiVnllEiCMqf3HSvrSxEGXdEVsZnQpPZ6PjzjFagzug62bttowNff5pBp8ZxlIEEmK3j4Rn9dSjEceJ9uqKdvTKKKtdECK2NmqreWwXZH351xfwMtci0+wGufVn95oR5rSgd2Ys7u3UNCXlPtQt4hSuHk\\\\\\\/coZFt0zGQPBVT6VJIccshIUb1GHXwF40asaXReT0IF2aPz7JxYV3RZb36NfXRAkuORSWwCmBSc+YNtKX0zeXPmUQEkDhZhVmWkTtMdXR6z+ddwNzuH8G2pzSSWzFu8OvZ6P\\\\\\\/4vseWzlNnaMur61YsloEOluMGHujqt+\\\\\\\/4xVTG18nGIk9R2dIDw1TowPcUOCXO9trAHuAaYYkGrD2G0KQZkeEiB1TvwCXMDbR2ik\\\\\\\/SHdQq2ZMNTOenqz6ZHJkKpbIzYTcv38+b\\\\\\\/Dgsbm3h0LKHiOsfZaMpXiGCA5ORN94gwc5lDKlnOtSPIpQcQcipCGCpPWHWxzSqh84aBbDxPvS9MP0IfFYHXN12EZryzRNmbUCddap932lxQdZsbH9fwvTCgbCYwfgmiqIi+t8hVeSKNwFHqf2xLptV83pZrFml\\\\\\\/kiLEQg+kPrj4VKGyGuytZUjr4\\\\\\\/Zh7iaOIt2zVP+MsLDkSIFR+VOTBTN8uIsrDElbsPYMUxyeLAuMUfWBsKH1IvtRv1NCLlGcS\\\\\\\/N26+fb5U8muaZzXHDfw9hid4eK\\\\\\\/mSNVWVleTW2g\\\\\\\/QiO0YWDCNj\\\\\\\/4BG4bzQzhWgetPCjmH5Od5DV2cQAzqr8WmEB9vcT8vMgKPWFp+Gx9kJ+uqhzooQBH+VuoOsm38pykYxXphUlo28GpEwDLEl6wNPZBeHFpqBN7t3BXklklELn8t\\\\\\\/7AjXlgvy2mhEOktY93N1OdH6x2cWrHjjjXbuJG1TAGdDdXL89QCpbl\\\\\\\/\\\\\\\/SSBsO5d5FlbZpYmaknTCP74l5yA7iKboGsrji4Ysx7tAfmp7mI4TAydDobOlKhv\\\\\\\/Yj5JbTYGC1u2K3lBJ2MkDo45CKsTwaRZFrrMPjpE28naSU\\\\\\\/6VCfxiH9hZ1HkJp4be1\\\\\\\/\\\\\\\/41eCn5ExpADkfzXbWlrNmlHu+fEeOLUZdc3z7fqRheHiY3+ko9E\\\\\\\/Q7TnunNQixocXCmnioANvwTD1fwYas\\\\\\\/yuvq7LfnshTlBj2Yy4spCShrHuA6PjU74l\\\\\\\/cZf\\\\\\\/Y6lm5W7oAFM4nVhgsVDt6nOwdF\\\\\\\/tEdkQJ5X2VnOeMJrz2bs7LArfFDOjqGv9lqyxJHfpCAaqGgpFIoCw5KyXgDmQ+vTTf3gCeR7jfFNab4NTJBfUXkN28XGxOhDVlD9YurGYFWLe\\\\\\\/nxyiKp1YhENSm0q47\\\\\\\/sHNlXVb9WGYJDVzVD0qHRwE1pSqMSxqunmiyxoVWUBxJmt6r3fmyM7PLIybs7d4vCR0fhJU9SPczpbH\\\\\\\/w1yFsr3SD\\\\\\\/Un0Wa+NejhIogZ1qpDMP78EsqbSAhzwKAqHspo7kv1\\\\\\\/BCqzLs+ed0lXDm05imC9exqNl\\\\\\\/MwL0XjvTfSvuTTvrUbXSSs4l8uhV50CkjmYmZy45EBQnC8B9MFx1BuqfcbsW+0Jp7UEny9YOmbrjpSxX503Ry\\\\\\\/GXLqAwbPehpFy3oFW1k2FVvE+Q6w3eSKZiFgDpESX78L0hboW752qwds\\\\\\\/whu+x8MJxrdmH7QUQxFAnchQPBeK7WdANrLCSHogLe72SUpz9bYEIazdVsEkruuEKuLQDht0j1sNkmNoVK3\\\\\\\/KdwYPwUi2Kr2xg7UO4a62ED2ypXvLStCAFiYbtKQ8BsMJyjCInSJl6cHIJBdgmTvPAg117KrIdas81Mm33wAJRdfhQ6m10lLO4FPnVwKD\\\\\\\/EPtZL7sNX2fnnVo\\\\\\\/JMMAaPV7rEGRyV9HvO4BZ6N\\\\\\\/XX4yBENW3ldGs5ULCed8e6Ne4Sq3ZZLiQHs8jqLbIzAF5hVi2Et\\\\\\\/Lv0m0\\\\\\\/46nHT5yRrONnmWZswa\\\\\\\/sECGTCE\\\\\\\/HPyNDV\\\\\\\/R+FVvvWSjz5HrQArFNQOYO1QlOIJ2ZGiKCQaxxy48KfN1\\\\\\\/qR6C0BhfacisnTPrpAbIROmYhqtmQkZQ93THwWHT1MXkzuO3nALsrPlurVRB9mLyi570PIsQiTFORWavVrGjZyUTD9ydQDVUioPueeEalt6dr9McDShi8ILhGMr1BdwZAepUJcbZ+z9\\\\\\\/Qrrdeq7gx0vRHGRPDf5r4Yj9cWUoS\\\\\\\/UVmBPoFsETINvVdlPlAPLvtZZUx67QzfjKqRUV76wtSD2j7GS00k4+\\\\\\\/0l+1ouV4H5BJSKmL+3cgJi0ObWs84LEdQ0jwSE3ljktWgXu\\\\\\\/T0MMZ2\\\\\\\/p+t4vCsXXsAFaWnf9RuxAx3ecZlXjgPTX5yyUwtDvyaL4kVdd\\\\\\\/8JD0n9MXBHuP9HFLCXtIjZy4RAy0lKLQTagL6pTFtX1n+LcbXdLammj\\\\\\\/AHN4PbuHrvRA9Ntv3Xft3hormLvtExt5Y5sFARF0ACWwFPRHE\\\\\\\/KHUfPNQR8izgFuJrryntDj2JRYTC0P3XZie6eAitD\\\\\\\/kPelPGRdOUtA0d9gJVK97isqO8h4tCv0gl5dIk\\\\\\\/49oTgebDFyiCYcZlKViZJbRDvWu6zubB\\\\\\\/Adruezd8uwtYQhc+lLCXjPVq79rpEusa0focYiOWb4Slwize2Vt52orzDM3JxVsEV3a4rqinWWmbnbvAn5N3AmJRAYwrQ6PAa4dNvJs6k\\\\\\\/tg6A70Uga2wqDRxYOMBYCjDOso20O2JLyGnKjxxQ72KXoIswnRj9g0+BpsuvdgLtp62gkmAI0mCozO5cGnFTEWsj5Xv+r27pAS5e5efvp+EQ7GR88MKRqgysC9fP\\\\\\\/FQxTTnJa5Mlmv2b8Fdf2d0jKL28hMEjI4HO51WbljNKUEjae395F4hcVFxherZFY97Muy8cFdp3aXbkjRCnCrZQtCPN5N0MQKpxvt8Uukrw03WlRC6UPGa1AZB1eFm37svEkRmOBSF+ga\\\\\\\/iUY3s2JcKg6xNl2G7SVX6n7g2bxnUs5+Q9RvO9MPLUjO3D5dlHeK5YRRJz++IQAwNRpnfVyVD040Zh+\\\\\\\/6d37Nor9nWJULmze0zJYGORKE8nZ\\\\\\\/kkD565oMw2zhzNnXO5OrZdDrR\\\\\\\/o6LbohrsktvA\\\\\\\/LTl+CIyJpzPWMtlhKrukWOA1xU6xPkmJZfUA+juXFqM\\\\\\\/fAMxtXjiTh\\\\\\\/oLxWL9cYRoaVVaEt5tq3ok5gkIqUgZ6gEd1myuWhKSyO5TdqczsVGRSD\\\\\\\/nyM7blF0A5\\\\\\\/I1qP2k35ALf634+n3e48GgnYpksGUYi5cJQpX2vYTGsk44Xb+4S50v2fCfHxUu\\\\\\\/KJQDvoGg8O9oBPypCDxgc2TYBgvlydq6ikpIilNUL1gqPZyEl9BecYW\\\\\\\/0QwnSYMYQmCDqEQF4A8HnR8IELYSij\\\\\\\/87ZQYK3fiKjhhGfTmGNtGAQysbijlaO\\\\\\\/CE68ci9UTkJInIPaQvlFlrMqZaV1Z1aYI5xKpjC+VnQNED6zMX0MR4XvSbV4VsMbSJc2bHNAoRqfL2eP29flyOnyYTNlHaCzrmox59KY+3k6MJw9B8OhVyZXTdW85lYiq+hWIGtg\\\\\\\/XasHT8Q\\\\\\\/gCBDA28WYLQ+Ls6eiyQXFC5UT9xjxg3ey6Nf4S5H8Zb9DJsZXhLimbFedD7apU0t8ijuMXOmrmSPpVYOX+zbdccBEZzZ\\\\\\\/RVD+J3wVC88ys2YFqjsdHcuqwXUcbmR9uzCL\\\\\\\/UvAs7LUkueNuErJXq\\\\\\\/n9fLCmNoLRaukKATkCfIt56GX4QQMlSXaI3sfpfghaaPU1RFRI0n7609IzUCbiuQyv7Z9Nc9G1pjITHmaEiSntli6Neg\\\\\\\/Z4HLTdbid8PXDp7hIne5QgJqN4WFFLA0DXFUNU31gxoLfcFkQZ7mya1LAl6Sw54zn7NyT2jcBSSXgdLbZ4QxxQYPBD+q2vAvV\\\\\\\/9+gtjZUWcpF1yXqi9+NqHMeMXnYd1bf7Hqb5Fv+PRAHBV\\\\\\\/GQogn9Kt7oDqoX0oW+al5a8UxqkJkaHF4VWAUaFAoE8qAdNRkBi07tNi9en\\\\\\\/MOMTCd5r4F+8Nt7Im8FINtCaLWP+8MkLjHePfay4KMSx8YTV4EbvwzaSKE7VZ8MmL7cP9WwkjlJTugncB0Izgz+f2N6isdVCd3mN+9B1lXFf8MrrUVhUX210I1N23TgOjBaHGATLrR3vvKK6I3VgKsp\\\\\\\/KFNZADgOLe0r+E1SLVfwWVoKUIVmjFcR+mOVmf6txY4x6OMQtTfIdIfcDhuHgV3KHSkeWfje0l7dUo04O0tWzSLxA\\\\\\\/64l0HuYL4j4Rd+BuZDJyoIvaTixZHLH7+zr9udpC3ICU5ROmclKJ69SgFnr3VKHXmJ5vsHEWc8mNOqlD2FAnBHoyQWC0xi+g07YqmXlt0rj5ke+1477jkrYQmEqqQgK7v0ox1fEMeBTzSRkaMRcqQgmq6AvR+bRTpIU51uo7GV\\\\\\\/6Bcq5mZNq2RTNJenNngmgxicxPP0gSKwBFGdvfK29xL7pUbnuw6q4FdPX\\\\\\\/F6GLXeMKNdfWjDT48Q2CvUSj2M0seSHk4ahUoavqBdLHieylMG8oT6jfrf743kh07G89Qn8fIAtqiriqxtL\\\\\\\/Uz6asZreA4JCF3U3b3rgvnfWiMITOeJ40Ve8vAGp7U3P87FHhS4B+XOE6zFZlHS5CY7TgAeV+tQ\\\\\\\/IDhHNdQjCGUipGl15+OPgd2GZIGyiPzyNuLId55gd4eJnA2vjbF1g9cEDwIkLteoLhesTGUbwKaI0Kew1\\\\\\\/d4M8OCzwaMY6\\\\\\\/psB2dGMHEpLgnMcKoxYpvVzjeGiV\\\\\\\/zE23J3gR6MDirP+xWJn4UsrNT4up2yxgBdgeY8mJItQdAu+CEidQ\\\\\\\/r+u03UdE9ZsVXnD7lxy6UuGI+iO6LREMoBVLYbWIMNFDWu9tqDXY9lu6R1fqdi7fyVvUT1WkepVWWyfjiUNPyW96TZEUNpOhCimMXAy+V40jzqSTZRv6nxJjIt4o0SXr7FDCjx+ekGtdIU9ntiphQ9gUkEjpmfrkm5iQ1sLWREEW+C3nuY0DmPhDIqzjhram0mPL7WjtpxmpFo7AYoY8o3ECWlipAdutGkaaKZtB\\\\\\\/yOx8Nh+mGm72mjYesKe3PuOfiE\\\\\\\/TrIKXxY3GhXphrNNbztjQGU4+Cyq3aGUd7i209SwonriShoXsDK6ZvVvXZKgKVC+u4WjPe+TMJuMr06SYOUQOk2h5uaiaXSK9HrDb8uWzCDMoifZHhH2ucIW8EIypARoHLDhXGpEHt7Eglq1ROeGZWrQIUsXXGvfoUiD9k7UQ94WNOC6FBsj8tcQAoUgANVArJ239SD0h9jGB0echerRMjgox6ozc0JTQuAwBkxX73tuNmsOFwcaEvB3iTQDklGPCziJ3AE3JnFHQQKsEKr+VXK\\\\\\\/SDvJY+pxuYpBQ7dLW59DvhDZAkV+rfOOY9E6iJT4qdjjC1eZkWOia\\\\\\\/FsnNGjS3Z9E74WvqNqYfBAe5oAK0Ew9HtQ4ZRWVB5uR+orIh24hH1fG7ffUOF9cPdOQyNo4ks8K1NJPmPuTpH+ZjN8D93fAgKmY1JXNZzVt53cv\\\\\\\/4ypaj2VHdABx0tk5cGZl0+W4rjem5sorcNdpExWxP7eVcYzDyuJfSAT7mOxAVZJgdLPi+SY0bzdbdhDm0URKKxl4CfJlYRGxRvI4RMuGjv\\\\\\\/VTOc\\\\\\\/IFZUHBCdlFrNeWjtxLE2PmErcPkIO2Lmqj3pCS1f60ExL5Qre9sQmCX0OCp9B4Ec3973l7JcH64k3lf8nJR8tM7qk\\\\\\\/+xSgb7tkAeXxm\\\\\\\/2i+Ia\\\\\\\/5X5Exr2WnRyyM4DaOOwHGdsO7mU9l81IHRsCGA2HeaZSFhkgv\\\\\\\/xG+KUYhD6gRHsjUP+xrqJmtlVoFhDke0LYI\\\\\\\/QwE8ryOQ\\\\\\\/elE\\\\\\\/ntrJzHqBQ79LC552mg9Ndlp5amIvrLT6cCWdWf\\\\\\\/DJ+8DmfR4YZyEW5abVZ+Q04ESa3wsOmbTBUwSXfVwlrp6Jt9AcF6ikYxT1hfBn3jsf5nQfG1Q9s9\\\\\\\/NyjGN20SXtC1mg9K3+It8xfBQEWV0oWQ1hEk8nfSgX5RAEomdkypypbO9NkLVJy8S1xHZEEOuId7rFmjl215j+2eCPvl4Q8aQG\\\\\\\/kMdu7+W5jQBooJ+U9Sp+jgWet7d7i8s+44k0nJMgT\\\\\\\/I4aEo2XWE8Rn0qKZp2PfaZJfcesN5u5cFDIi+0D2EbAxnhMcikNnK0296gk9xmdDKKyFn4Zw3S\\\\\\\/UhEc2aD3X7FNdTjyWbsvOKJ8pz9X2LP5cnSu5KV59Q2mMtLZbnbficN9tw93K4msbazxgdZQX7Bk1g+3gmLtTgyAKUJx7KrDa1byE9OtJIzkxj+NqzETQz6mOif6mQ7BWajicUDlwEDk8vP\\\\\\\/mTlkie7QvRzKf52TsMLTKiGwiyaUuI4hEzGnJnKD4tT3xltJFw0e5P5X+6tTFTNXw\\\\\\\/kwjL\\\\\\\/9N6XKYN4i5aEi3MAdkALy1irq9YKIikZgt03OHSdBAe17UOVxVX6ux6u22l0BR\\\\\\\/FP7F0e+PxNln4n4ogLnegQAIXnHWXODbRD8qm3xDI+UTDBNX+o4NUt6uvBN94CVb4ynMZY9qbxqIsTXV+TdJFS2Ajg5+NZYrOZdqdCY0+R1cUzrk0E\\\\\\\/HjVSCsPpb3rXdrFSWMvIItxNsV4H2yHcznykzUNxnBgdU0i7SUZp5b7sDdwJ81xTtuLfrzbwnJz8riOZT5HzjZP9gnRjwjVC+AfHMqcLVhbHcHLzbKa6yiQsiDNJvvGjaEuqLTRoEzEJxd6iFXz0Tj++txSKJaeGMgY++WzPojIO+WaeOr9d4Kah06jzpUqrpQCVwTLaJpL6XjYv5SKTDSmr0+m8oPsE2GCJ3wf+BAXJLx0SKnhllRxMhRpBH3Qrpl65iY8HXRtKS+O\\\\\\\/vhCN44T0cMcsUC4Cbk7YkO9WiaGwRKhkOHDDq\\\\\\\/IY50dBdEGIMdMH0rNo5FdHwN\\\\\\\/nRO8N26vDqLXNkOEjZ6tifyLhfVKR9Cv8IF6e96+Hk375JmIKDvKO1WyVjoll2zukeaB9IqgUR2ddkR4BFgx3s5FsWsjHigN\\\\\\\/RFnqPEJLBT0hvNAu2tEOzd6IWpSnaVKwedbtstGBXPLkz8LOxpA47DnLHHSED6JXt2Keu2iDxVT3HHz76ipFallqUQ\\\\\\\/5ywfBB1bUOSlsfCTdNQK1CKuxIDT12OH3TUAdfF6lQeVbaShraGxQuz1suCVyg5v1Off9gEvV8LVtECZ1XkFSplcdrRIuw91UHB5aZbTpk32Wyo1KZL7sw\\\\\\\/FamHxRW85pVcpfpRE4m6SgI2HsysNcCTv\\\\\\\/gPqYeV6k7Fs9LWoF5\\\\\\\/3heKgKU69WHzeosJvLep4jPn4Etib5O5gEnpleUs8amaNCi8hYlAf2YBkuVawAzUBIcxt9mupT0M4IFLcD5Px+KPNRwVR0QuBcP8XKT8zVlVXn3\\\\\\\/ijMlbpEO0VO9AZGzvdZfxGijzHUywm\\\\\\\/PFm5iE8rp7EyeMUZm7SkbZWC4rrKzS4E7i9ju29XssGReM0Rc0qqUipqR\\\\\\\/PgR7xnZO6CDRBNougA77uDsLhdj1839yt7Q4Rkb7zSpXQBBDU+Q4H5oF94DOXX+6Tzmo11Nk3Ad\\\\\\\/LbpSnFmyNz\\\\\\\/k440JBbp5q6N9RfU2asOiLe1doUCrj95QxOhKm5w1Doj6+25CgUTVjguE9ZsNxJUiEnC0kezG9hgmCaRKi1lLQyo4nSCYy+62qHZhB3exPIUCH2yesCLMkXj2Y9EqZuJIBrCm\\\\\\\/BwB4trqu1yDIIAxnQY7lwB9XBMAOKS4FxTG5J5L1JQRhvX3VT12ADfKfedAS45hNx7Dcmlwyof+Aei1FN2jGUAFw2jviRmYXnM+WAqG\\\\\\\/t\\\\\\\/oWYndw9qjvaxAp6rGqHVKeKdODqXulbsu4d7aYsXoYtkPrN72DfIBDRINxWHfp9lACuC40VY9BeoXNE5MQ\\\\\\\/d7liIjJ3BsoDtmu6q\\\\\\\/bQXbllMinpfJEWzeOWYIgoI6xaJtbWnz4NInJ7WCUNdVotmJA1Z0+XcFXrZILbHUs9rtSP6R1DxfNz40lMRiE4CWg0zUF7tWR03pk2rohournAiBTBqBJMDG\\\\\\\/QzlynmrzeF4LU1uJvxtWUg2oN\\\\\\\/5sfjONW5UfPERRUk7Z5BJ6+CNvQRMDL04VRnzvGngTAURbFfAj45eHPx3\\\\\\\/c\\\\\\\/whRpnW7FvAIdl7UaeM+Q09KnyiezjHWpsd1QCAaX317sLkjG1jVsMttSdHUKOMccEIsCzjvjUsOL1Mq79+dUEZQzCj56YvFAGcTL8tVz1Ps\\\\\\\/kzNhnicGoDUK0pl9WrtOAN60Hgy+N1i2jBQfbX1p7emj8gR1F47inWdkbNBZNCfRympc\\\\\\\/hx7sb4yOnd3gKdsoHQNf6mYhJTMTMm9SA7gh0ncA54TAD6+9MrvybxmFIWlmlxHB1apOVwIkHavB4QSjKHNtDMHf+MKweF2\\\\\\\/QGd28UnawL+4TxkkSSCdRjbFuu7ioadx7aW7RanH6eAJRrXhw4F04s4OAz\\\\\\\/E0\\\\\\\/0y4N+PquJEBNDRt33ml1yqkS\\\\\\\/HMdIy6whHmmfB\\\\\\\/qkoNcuT9BdGTkQPVJyu3A256oa5k6JR6kmbXsVt7M35L\\\\\\\/c7OQNcIaaLpLr9G3DvKkdsReHUHTXmFwjxa9W2F62SrOs92\\\\\\\/6y6u8PxcDfAj9FXnr3MeY6KisdI9l9M1\\\\\\\/iRbLRCm54dDrHIml2+hM4uRYLmpJaVSw2xD5OQhcrIHIS\\\\\\\/sltMK+OuEaBJyvJrpnKoDDeLz\\\\\\\/Y60had\\\\\\\/qPb2THBsYng1O83noOpbNjMoQQ2EjNDDw7+Q2SLQUThrT2gDnXs0ztIBvXTpzRfS8KDr3euSi1vueIpKNnBzoLBZdTXbQ7ZQ24AxT7XKmyPKE6mxVyg4DSZBvBcBE6Hz\\\\\\\/J+Z1DXiuNjm0Il7GR00+oH0PDhFrViuTn5WN3NFsR1JcY9JOpXOVJtTQhqXaI5bexSbaRt7EdYJ6CfaYBUACiCPTkZJX2Ki1Yn56GBuhbld2KNteJIdzbBR06Ze4tvJZGc7XYl0\\\\\\\/1bsASgnQTTskH9cl9lX7XoMYOhdPIMS0WgJfjVqVOrmEfIu9Ox0dgT4ZwD0DzxScoNljZBQty+8Cr3n9NcCpz5lHOTfAIrubt9gBglhYw+5xnvy3+4Gb\\\\\\\/UjI9akJLvYse5cY5ApffvXuLCrhaA+7yiDG2lzI8Nm6eo75Fk34OXJ0sN8IkGNkDT31PLZO0J7uvNTSwLt++shn\\\\\\\/FCCCW0H5cXLO+wmfB4TvEKaeZ89y1COagPI6z3OHATxqVpOcsTr7Z+x9cPVszbthfC3gquz2hvhzIj+3hg+TToFhTzMAEUnmnLjWBKzTRZp\\\\\\\/NKR21uiai4mNYtJhxs5b5BYqYTTdY8uqhd0OZof9GUNf4hefPjYQSpy787N+PtS0z6l9sjqawbICRe6O54HPyPT5O1+DcWmT\\\\\\\/VlrhOkGqr7tdTH5ELf9nmbgajOzmQE0HV8P7g20kKBKG4mfvs78ZiM6qVdIg\\\\\\\/tJQUi1jXvmB6Rk\\\\\\\/nZ2+d\\\\\\\/wKUyui2ndKZv7ecLdR\\\\\\\/csQ6ygoQG8U+H7ztddWVHK4pP9M+rtbM1q7cPIxvsQNCverz5pdV1D9u32Ukzqi+jKhBHHFqtppJslZGyz0QK4253ykfljoXfu5BkJkanzXj0Mp94HyuDeY3bCv0N\\\\\\\/gtc+7ZtENbYEJYzquPPgNc\\\\\\\/3IQOod8gCBOxsBVpxkUhUSOvOkrZ26pdcpAyLx26954kaU7P\\\\\\\/trtMtm14EPzr9zlPluMUUNocf7VaLwPTSITVnFArL0JvUIJ3Tqs9O5\\\\\\\/JJP9e4yo48loV2+AleRrSpf6q8jz6p5yraklur\\\\\\\/8LCFKi8RRSLR4dxenYGGduJBSRqRjT2xSuA7i3\\\\\\\/\\\\\\\/d7tSHHxlYPul8rMQ0X7iqWtsObnTlXJWgyQsbisufJWKZ4WWWgRmZ3nP4qOrIS+LmM+HNi3pbqjWSbJeKkyi6wHZoU3wv94uMLG5sO0p9zxSNLcWf0wPM02bsX9y5bBdKx9nUmIUuMoX\\\\\\\/U74in0vdDzDH+YETPN9WRfMbr0DTkI5RP\\\\\\\/k1fW37HBOhKuvO6RqfEs60cYVKsG0cy8Yv5cp+F84XIvwmsMKDmd7AXEWAXfIo16M01bDPNpkjBwyY6pJVtI2P+KV94V9qiA9AUyhaLily6petSiPd7oKj+QTH1cbSjiNnxevQJpdApgayAi0zUjmhIU7NT6Q0ihGEfqV8VHxgK0b7bRYH\\\\\\\/Vcv34QvjBbaUzngOpiSjB4ACX4aKHTYg1JyGRxaAm665ynl4Cv+oU7+zgO1PElYlX9OQ+CCOysLUTUtLJVmTpVsPLibLo33tNOe7gi2h51ch5u3v\\\\\\\/J3DWDhdnY3L88NGRhUcs5ibzg+p9KDgHbnUxqWXI4oqDkLmoB2ljxqO\\\\\\\/JugAkvLsVHpWjjpGwnmL2VXXSOLvSLKGt+P6EJPw46fzdGBTTkyRApJleHYNhqwtoe4cr5SmgSeO21QpaevxkDyj9ekyCAWsaYlRmLBm7hR\\\\\\\/V07b6pFbDTTB5LHJNuEsJ4jUuk4sfnvlD1LyHJ07jPtfArA+KI7R6rruLCoQIjgvgrcdO7Pk\\\\\\\/ZeUp9V3dldhjtiF9YY7JuIP6+4WDFrwCEFGPPLh9B7YINEF5GQ9F1H6Iz7PwoUtDECwX8+qky6ze5jLnqHzNHBqUkUUAt0ULyY+3s9Nsiux\\\\\\\/5jHqvRL7GbRgNbvpkSBG5l5FSNdtmZ7iUr0+ETfVihOenAdwZ69p0ya2QnGc9xhm0aes5azNDyPEamPR2uo1NLLPibTnuxnaLfvOB6FoqPgbMuEccY8+msuWN2XDriHK0rtFvUm93TXROm+pgPihriblOiajHa6skCUWxyQD0MMMkP3JgjnK\\\\\\\/4nE3mUxQx2d7EAPxWggS4fc0z9k\\\\\\\/NvLE5On8\\\\\\\/k\\\\\\\/DYoMdt\\\\\\\/fYZEAVZMQBBASmdVT6BeyDtSf5GXE0MlazM4k+sZu7uUxdJO\\\\\\\/w4STQGvl9JI6rVorBT8m8L7kD\\\\\\\/\\\\\\\/YD\\\\\\\/2VfKzpyBu+xzprq+41yYMdKELj38ajRPf\\\\\\\/j2AWq5+8MTcDYvUu4YEyuIu4+OuW4h5aOsVup5ps\\\\\\\/qV9z1AnXpSqS0++cQWfYbntxDlpytcROlypAbfa\\\\\\\/tOmj7tJHi9KUIDJzGeqnYOIAjAKtxf05sQIvWbK1ckYeeDEkYEjfvVKbs6YgmUmFEUgF3jx\\\\\\\/YW5mmg253fpmOl1Fm7abo9WTIZIULs+wyiIh9T66REYreUR7kI5YhwqflU\\\\\\\/P9aB6aIrh19ifqJFYXuhqCh1FEUNw4kBHKOv4Tnkj7gDEsKThUq\\\\\\\/hVN1RpblOv+Ef4Di4CUpfHxZtA7VNLmx1\\\\\\\/1SoTba\\\\\\\/XHNZjVD1nuOLD7skZ1lru\\\\\\\/Ti072ORvt3iJxHAC2XCLdefD6C8G4YgtXa7thMn\\\\\\\/LPa7HEk+fMjmyFgbwdKW\\\\\\\/hyhuj+mcBFkQDM80bvPGTvTGNmUlhV1Gnx9TgiIXDmPmhcWJecr1eYuK5xUrKqkNmqKq8hx2d2ek24gI\\\\\\\/uPXsj9MiuHqhNTF8bWRVnAgyHpnGOgW0qnsL1dcsMwPLP7PFPzRT65KWYInj3YOqSQp8\\\\\\\/A9qXV1g9EXKnt2mrStcxQM+QqasRrzhh9KAIS24PFtOWr28za3RYobbbYiww9t10U+Qc7hZ\\\\\\\/exWpTxaS6Vc\\\\\\\/kR01tnug8x9WJfYwhNtJl3\\\\\\\/faU693mHRziMjxw0eq6xP9ikJX\\\\\\\/gKlI\\\\\\\/x8iXKEr0nU\\\\\\\/RoOgm88FHW363WvV3+\\\\\\\/PdKjR\\\\\\\/gGyrZY8P\\\\\\\/jh5TjjDFlcMLnXzS5LhG7ngNpplzjEKdaDMkn5O3Czr9Nj5SpeTd3A1gSyUe+xJadg1BOKulOUFUSaR8Py7tIp3xlLB8bXYi4EhGmHZfPz3GZ3PiIo6GTdM\\\\\\\/GSt+CSWMcJFsKm4Bqv6UixLx1VkrS9rnB7aqSYakIzAbWQIXzGfaYoCKfB9sTAANFKiK9vHvWcMifC9yB5q9MKojm3\\\\\\\/l5MUYHqdgSmB4PJupLmTRPvh68MCfKOMez3fYtZL4RZ89uKJBAz8xez3zsAmoAPrX39QafOlz7zar\\\\\\\/u2rDgkZYyHJqmPFlKS9DkAQfyVZVTKLpTtVg1wl5nzDCfm4UlHK\\\\\\\/7guo1p5T9K6PZD8YNGNk2l5\\\\\\\/wSqqJZ3b5Jl4DlZT1k5luG4AJZOL8bhY0xeE\\\\\\\/WDlqGauXU\\\\\\\/Iz6Kk5mZHxEqHAbSzrLGJvbpBhV8l\\\\\\\/gx6DmoicoQ4HqZ+Nogr6qTZwiJv0o2yjmJGj3gBRej0gL+XcALBqHsYljKf\\\\\\\/ztBlKGC7PqofeokEIV4zfu0EJfkI2SK5ZPYb\\\\\\\/tFh9C9qWn97vv1NL7by8sc9VnGoVcBcc2o4K0mUXnfCPHBLTNmAPk60A7m2OeMfMQXwU2ImPDP5ezpncPGmecL983L8QiY2fbQbZOEM8+brnsyVfbcSoI5XxNE3HdERkS5R35\\\\\\\/RZflJ7\\\\\\\/NfUQ8YcIoRc7ZIeM3LBf+JKYEIX7Fw0EfsfRrvNrtskWBO1noSxnC\\\\\\\/4ShEOaxosb9gQmm5pcvxs+utEdMLgeR2j+J941MyoUi45t7ywjwiGbdH3O4lGsZP+6t4F2hUw05t5fDvgnxdrdZABbwgwG3XQwPcq3Q6USeCnlbCHgHC48VUGKbfpnNkG7e\\\\\\\/6D9RxdmwZCaxHiFdU3wBNWcbMXyrZpU\\\\\\\/ijsOzOZdJtwk8vFNEvcsrxK41rZVpIwvqCJKb1cHqlAzWnh7SPo\\\\\\\/Ei9DnBcpeaW6jo9lbjuEm6Q3G82+KECRKb5eTwQGGxZhjmlVxYxHmTf\\\\\\\/NaY+gNoXBSNfmCu+59E4myqBhv+uHzFZyMCu8hHkF09hDlQcDaXZIoFUpBT0FIu06yhKMahkCv21+CiR7fLWAvftkoH8CXmjZFJ7kqD\\\\\\\/VUWDXQSSIumXBrcQJ2BTZ2ASot9fO8LPdMtX512zIepZb+z+xBlDolTxdpXjT2CRJ1WBXfumC9yPs\\\\\\\/5lS0TaZzN0zRr4ZlfoF6XYd1qKcP2ZzZ2Odg96XNg+sJ007cD1Sc0VLCzrdVKitkvXZ6SCjyz07gDsXnk+nkmKVQb0efQfkArAuz9ypPUucrELIobiBzAVHUMvQqPYEjKvZtA+nCBbBbSjgc3vKr0x7tulrLftIfs5RUigRT9dpQSUprDP+O9XwRrqbL9Dw4AGulaYX32aSRa0WaXFpggD8WXuV\\\\\\\/z60ZnxT3hGJ9APqcR7UMcZeRBdYUkMV5Oonf7aqUvoRTEVuG2jCwDOAJR1Jc6eNIZRZ2E0hoOSU7UaUq2rhdSqPm9ifgnoAiNp0tSQSXKJsbR7aFbwABzLdB8wkcJjBRw0fuaa0ZcTR2s7h9Go6aw3OdEVTHkDLFJrr8B6JhHzZ79U771PvDxkGRC5RvORqDQvwD8HypV6p3WgclknGNcBopXfGjxAzhwHsO7P16mTnrwaSPMnXsS+mEY4ekxskalTNQnX0pWqm3MILd\\\\\\\/7efOJ1833kI2AgJbmCNF2vqzfsnfAY8JHnrWwyR5AuW6srNkuYhwAk0VKkiQ2r4+RkoDU45nPN2OLIyKYpnG13Ob4ZxTTVu1Fvr3sFJCxrvUucLDtWZM2ajZIDgmtlRRuMyOTtnR+eG0cZX7k9JzeYblchPjJsYvab88+ZuSzM3GKB8MCJCxONOeSszlGyVT++L60rZtVBLHgtT0e7ItEp7vBvxnXsSpQTO0N85Wab\\\\\\\/zLo33BwCLlkvic+kL+i2kNdzSMe7TfgIadKcrjqIIyEkzOTrpi\\\\\\\/uFtUlB43uSHNn3OEKGOKdTSI1pJ8JsNSe64uncp\\\\\\\/BpjN3eG5\\\\\\\/7h1+u74vnSl6ytZCOVkfuWyajlnJ\\\\\\\/e0Z\\\\\\\/cuPdFtDKlLh2rf\\\\\\\/MQ3d\\\\\\\/sn27JJS\\\\\\\/E2Hs6kG87l8+zqd+M7dC5n8\\\\\\\/2HPJedNmxXI5unXkypUbahGIRqleDW9ylToOeNOkhpOrDUG4DQfqXLCFEvh1WU6L0E\\\\\\\/waba4Ih3N7Lxmn\\\\\\\/kyMfH\\\\\\\/yhq1JircIeKthyGg937qEANgocTZNGkTehS75gtn8t+MH\\\\\\\/42KjWAYbQSMUNs8LB1GZ+mnChIVGydkG+3yEW9wzNqSJFi8WuI3CgoxKMULwTMzAGcf7Y+SvsuegxEl56IiXZe\\\\\\\/DjaIeyFTdBOx69QRssOcSRxuKNWSNr2lu4ya9gfWp+PzqzdcRy7g\\\\\\\/a\\\\\\\/0OZIOPZl4LLqeL8NNbvj3X\\\\\\\/Y9vwqJcv8e9moDkL3sO+RgOoybWOivfYzleMA7gW8IQdmMUNfmYMYs9HvaHsxFFv3zrhlu174v2HEnTFBUzU3\\\\\\\/269LMiBmnNf6MTfeT8yJZRq\\\\\\\/RYqBKg7SXEyB28HMLukvcCJiHMpSizQgGEcG0PIwnhxd1KoI3L6plkyCifY6UCGbdQUyLY1fr03G\\\\\\\/m5L6KK7cpPHyrEzb5l8eO+CHfFeB74\\\\\\\/XxT9hkJ0Jzg14Zo7Muicb09huNxlQ8NyerMy7Xwz7eyWy6+tVEWrEexbFql+c88uWpjBIplRSMq7Gp+buaVAn6SM\\\\\\\/z+A+xtwI3hzO+TRAH8v6Z+E68fwppm9QEHuvEaRvT2fBVYTbPVKRFXYCg7W+CjeBXg9wyVBWi39mNtQPJ\\\\\\\/xOte01G5hOto56FBeh47CEYgTMPzUrwCyx4sQlwZwfkOy1vijTGzXTp4JUE4iNmQmYNxSSBtR+dR1B2j6vNvcPSyrIIUwMq40llkD6IzIuo3WKei+Q2Q57EXmMwxLmqUyy4tJLYTrlYMrUOc83eUcpwtaOgEUP++y1zH\\\\\\\/6eDUJ9NRYeqgk+sWbxbY0AZHyhaaNNiYlkDKSQNVLILMqj2D2Fnp0dvPTcfQDGoE6C7AV0zWnZM1qNDoIYjYvApmykDSc5ST38l3ZlKxCt7PVeCkrBhd8ZsrVZOaGQjyHU+X8K56qCg2rORRIYMjvRwHuYdv7bHOrbVKLhQ1cXvA+mgEO2\\\\\\\/AFEvpzBbO6KasFYjD3xyIYjU0kBu\\\\\\\/HRUrc9uyR1el7W0i0Y2KelrCtXxAi1Xvwivpnw7zAJaC7Ln7LBFbD2HcRejwFFp5lYOx4UgqvegSjiJTcmSIp7mFcO44RrYs8L5\\\\\\\/FPjwWUNNnMMEgV7NoSN4k5V6a2myibmf1Zwx3ySa+NcTSuIoygVL7dngpwLQzgnytfh7MheixFPdjAgAJnZKDvKYQNOmrcwUFULeawJsaXaPt03w6ywGvcJ6mspAD9HTYfwAsf1Js4JmWjBVGGLAnVugBuRfYqQkCEQGlqMH1VBqV\\\\\\\/8gmqTgZtgAQYbXgMe05Jmu9M+CRrBU13KVZLgXOPuEktazL5bjTdb1bSaWGEio8egm4E4X7veRL6jsF\\\\\\\/mxZqKSyMmYmT\\\\\\\/GAB3a536hPaqMkNSt6VeSKSZnJ9DLG3UCaIH2XEh+iCQN7WN5NaHHthiosQiFcrOLhHc4n4oHVD7p51wSBF\\\\\\\/mUUS\\\\\\\/Kf7FTMog+x5bNj+OuEYiJhzmTQfks4Ofh5k9go9+igFyt7xbvamGA3XcYE19lqrJr8\\\\\\\/ZKCpEDjf1KRY9H2iqxoyD9eIb9tWcHRShGz\\\\\\\/j1RADXAFjGAoJ4DkfX4FbLMCiZQ8qpx4o+5+I345ZDJQBO4\\\\\\\/G\\\\\\\/qRrLgieQsd8j5E+G3n9DEg98ggVHGAW\\\\\\\/fXIOTd4BiEkHbgDLEk5E2oU2nT3pj73tZcJygmlK4TiSYMwSzyPTcTKVVxH914+Fjf8wG9WdzuVwMzLIQSOoROV3BWmiuaHEMCLfD4Tj+UPhwu+kLvJj3xTnGw6l7C4U\\\\\\\/5QOweaAIm5FRydHc7bx2D6VSC7SmRXLQkLy8LFL2I1hWOTICuppgp\\\\\\\/zN3YkI6h9qap031FzVbO1DJ+I1XrsNiSMdlppN1\\\\\\\/8hVIX5VS3c9BniTP\\\\\\\/lKkyzsGoYXDNhxjHezLmo6gAPYS1snkjQUm2+7BrIFwu+pZeZl9z\\\\\\\/CV2qQwIRfqnMow3gAxow2igxtuKTKzwkA30f1AoYdcnqtHLHNu6sq5YGmrMhJiRdEjIbW9efejgHuYo8g+mDWaq5ip55MVvvBhd4e7svXFWbeAp5OSA3YG3+5uAFsWOvz5AzYKLeSv7L3oaUIP1fYfoaOcCltQGNn4hNQtzgiC9aU\\\\\\\/EdK38wgbby1QkoBMjuFX6DUrEmPgx6\\\\\\\/gOJ2vZ6y3uOuw6Uf2JkEIIcrvIFlV1xUnbmrQ5Qi4hozZXtx2PratuVAySqgD5yMs9W+VwdMedtpRhvSnfBWMqIsjVgj6vl7FQVFJNgZI0F+R4YKzjz+KqdGrHuHur2AA78X1an12nXTr7c7bFC\\\\\\\/S\\\\\\\/EnME8HtyxkpPV0GAZ3CfL8dxbZn8xqvpZxHplu2J8vRT2C+QAJNp+rHoUhoPnsY717trJ2mcIKO0yxNbSePJSKH8x3U30R+yI\\\\\\\/WtumrAYsJnGl2vATGA4Vh0++EurTb+20XzBiYRtx7sqW6moAY9GnERR8v0W68uHajUonTbvSwEQba7lXuhMaskku7Gi6v0xcQRW\\\\\\\/SckmNbL7wt6cF8DaW1kqKcoxfetMvPlYrm2G7IedLEtu8lYPm0OHWiQ2oYKTM43C6p560ZlqmGgsWi\\\\\\\/V6JANxM\\\\\\\/B7RIY3jWF6xxXod75IDv4UGP4n3LktpY4i95rVPDcScVHlLEjggktcxVKjMDFL5l6cqnNHLBOZ0lwT+bP54OiYrG5lO4LYXcWV0mCAGOXSjiP8y+dlwpL8oFOtAAjZAdz+dLoJ2Z12u83gqr5MKUHMc4NfKu0aO\\\\\\\/n\\\\\\\/V6IUSBxsa6sH6lixpnTcHTdmyatg4PoEraqlVNL7hyAHKb3umgk8QuP+zkziYqVkTSCA0Vwcon8mrlwSPEsHDBO+EajifLQ8VL\\\\\\\/rucdh57ZGtCVxB847IgyzJTMkVBaHXXtLGqH+GfOm67GA3ENk\\\\\\\/1unfrPptQrhRzpooDdp2tk6zjMedTlAvsdSCkZXpv+6S3eAhpbww003tOPcwVUbEKY8\\\\\\\/H2+PZyHyYkSa8XKmv8x6\\\\\\\/+hrEdHFvrSIvr95owNLmfqfDv8BIKZ3dSZ2GxZf8uDo1TQhpP1PRasTarlh3TSnTrgI\\\\\\\/xENE\\\\\\\/IUzZBkEEPwfGxddidCRYINUcKWgN1ECWejnu1Jww1l39PqHXIMbCMYROwTlLaNhh4z\\\\\\\/eF3jnHOaqqzdWUT\\\\\\\/B2aTCum2fG6YodTIIhCahRxaUHbLZ9S7u+82AWHPKgLic2EVd6MKEi5unO0TtVYhrirI3+oAqxFGH+KXCbEEmDxGGhs1H4aTV4DBBBbtoHSe0JcJH8AGrbwseEQ4sXA5VZzkPrBln7shmOZDfSl5XgGcYFYMi2mYMKrgNFCd9AlxprJXnUG8tSoosbUj5m1R4NrB7zUbe1047TkjfUaXKs2H00Zw2IJkhQOQK6iwP3mLK2DIwZH1nJ9BAYhiWXkKEnGz+nEmQVc+CXrZDX7SLUUPFsQzbHVKdqBPVJWGcpnvE5vCr38773y2qaQCIvmZ2gS6uChD\\\\\\\/cXXMCMnWMzLay+Xnc3Byg2SqjTkvcc9Tfv0TvfHpyDWN2ABH+rhrfw3wXYfzJi3yx9NNOSzcRRYsWW9n\\\\\\\/dq83yVffm8eZ1jTzbSikTQGhkZwTYlax5DTUIfzOrCm3sOEJ8ArAMhscfIgZSOKCpAPvOIFyLF3X4SjGnL0Dj+eoqItpmMXVOI6JICAXpU0BLKXEllnJGIWSIlepznnS0o+maey0h7eUp3crATxIZVfVGAD1RxfFdEg6H7wwK22dlKfblQeWdZ3J7OlvHDKcp2EMZeo\\\\\\\/FVhKb1W1rgZGQFVVJro1Dl83tB5a1FHB1QlaT7\\\\\\\/cOyawTPWYmHycsO6GoWbXp+Z45XZN9nTrEUWRrvkUnJ5r221hVHUxXhSUzfsuHOkddAFQYPSSA1sWwHqIQjMEK8bAY4S56elIBGJulZ0XGOgv9hzzp8iA9KKEghD3oYHD8g1nRLodua5TcUWJH0DE5CLpMcyF2KWzfmo\\\\\\\/yVfvwHKLPxREMz5menVzzqiQuyS5R19oSNV\\\\\\\/prkt9eoX3760pXoypS9FO7hKsNHB7u1ge+Y6gkDWF+k9ZEvbqAx+JBKfy87B\\\\\\\/8OqwzVNeRzo5uM36FzC+y5q77Nb8qBl18PYrYOT7ZrbYAzGR+9t8U0SPEXpAgeYw7wMh8bGxfhRiitinTZEpZKky8Q==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Y1EiMN+cRWCrUzGX6lbxUw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022cc3020682d6b83cf012aa8a151a20b4a735f0c3ca00c96522846dfd8d4d6f850\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00229iD8Wiea7CXWRyRzYqn34pjLO8Yc5WuNLSsUiCKT+5Vnp2M1jqsE\\\\\\\/k5hYAV2ohC9TfgZke1u3ryD5nztSoYh9sorzPlmtidPo++\\\\\\\/FzTT5t5fhjBq0p5ztxil06ZgBX8++yG3WH\\\\\\\/ZGT+B1mhkO9y6YvB7Fahj+o51pkRZ4LCEaYoi\\\\\\\/hAX1BUECWlA\\\\\\\/Ec7LwZeRUH2L2upiTmUe03hLoHkcXT0QLqj1NgL8J6Sc7KMNAkxoYQnq8GKX0Nb2M3J2FW0Z0VEK4+8kH+vPKnrgYhORtZaL9xevCyHFGPfHTEb3acLF30HvkSV3VFcgLIBBqfVQtW2GPu3Rqv4TZ49FXk32LGxNA8yuHAsmOuXzYE1CzMhhxU+rbkmFyNBoJqDs9QEw6B30gFP5\\\\\\\/RimU7CKD9q9rr7wvAYj1KWY7kxoDqApiDVF81iQW6aD45adnwKdUVOins8Acz\\\\\\\/runwq5n9pOMjJ2N0ohXBzN3eu0RzkJ4hWi4D7f6GAggmXKf573OYD5Ccl+endUlNFmVcM6T91bNc9ZfKlP2++JOQXvQr\\\\\\\/LcIPRmybU+40an1uYCP9Nyc4381wd9dEi+G0K6VSIeVXDuKTyVK+qEtbtzW0iGo0a+RFVDGkGZJTqoWA88HhE4oyixJUIxve8RltA1Ly4XDWOPPxZjJ6AVgmpypmE56SM\\\\\\\/22v\\\\\\\/6EWkTHg5uZjChraQvGn0VoHdbFS9UDlFxTKr1mESNpsFoYGoCLHa3mzylUGgJIZvWkksKV\\\\\\\/etbjVXVabFZpdl8Ni\\\\\\\/qnzvv8Qmm2OpTz0IVACCzpubaaEq7gwiPBPKm2WrLVZztdtgqcQUhgJrN7Up90ey49VRdL7Vx6Wf3N+hGtMs5toOltq0LcN4EbdIB9HFlUEoR985q2lO2fMYij0dETZnVkrZuux\\\\\\\/beRfXCEP+Z+sBvHl+2gKgq2t5MAtZpTvNP9okeWxMBg\\\\\\\/ZA426qyHaFdeIhRoo1uTFsUirjLZZwAz0MKfvY\\\\\\\/oyC+mFYCJ8kUV0PnPzNVDvjoWjTSGv4jrXZUOUDBo0hZaKfJmpfAccVG1j+teEpYAkpLq10JDzt5Yz0W4OBBHdB3yMstLcfw7WhW99mOkhxdyNFNOPW9T8PjvWxTgflLpaMmytp6iSNNaTfVlhX6uKhF5zF5F+TYbncoRWxGywftov\\\\\\\/\\\\\\\/erj24ryYCfY\\\\\\\/AMTPkBBGVVKLobJ3NlJEEdbVCKvw7L7CizaSK8tCPTvfCd6eEmhWtNu\\\\\\\/bAKJYii3f2proiOKxXeWt92yGo0JkP9xNLWnxhR0WL\\\\\\\/XkezFUwlZ7+UZGP2bQrfgsbwSN4yl2zHrlpJKWzUW+5KfhYhHhH2Ghyl97HQEHLQI6Gp78C8+D3hJeeF+WD2LS6xtEx9kB2\\\\\\\/dPz284bAEpXuThrAlofgl\\\\\\\/HDhiiiycuZ0W8lrTOT+2LVPz05yDwnVvCTN6pyNQob3IhDf\\\\\\\/jlzBxZ94dSWqWdBVBVD89ewORXzkKByEWogwHv4ycS3R6D1gwwnRc5GRd+Cw6BUXxJ1I5\\\\\\\/yp+Cub3g568rHXr+sUR1ODLPQABMQx+q13P5f4M4XSBoQMQGrw9u8VwK1v\\\\\\\/riYTrWYbxufvK\\\\\\\/9J7\\\\\\\/ecXdKPu6XV1JLyfzk3hd5LtouJPFXetLZPuGVhtMVg+7IDPNQxOKmd7WJpOpsNwkw3ZQHg3uhkfO+kVKkAagLiDdjGGz0XbJdZLUoiEynlvgd5So+Yu20wyiT1fEzvvJrSWSnspt9kOB\\\\\\\/uEBdKFzsBjO1MDxeXu\\\\\\\/nw50iIWkg16p4UedZ3rnpBq8aGJedSE7fn5OVCvXLArSqEJYhWfRjSgLs8NRWhsSvIlOt0QeTk67Nb6uiHuOcsylA7z6wG53oQulbrG8G399CDDvlCzZqKrhtMXTaEfG\\\\\\\/QAmcNKe6cNt5dKLF9Lu81p8iF7XN6kByal+oQWAFch\\\\\\\/YJgo7rXb6+yUw+K8eb8k71YxfO1ntqMgnLCAFy44vogADrHssi0z54YG6doAFC+9HtLbK7CT0pVxKeGWyjA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Ot6aJXW6gGHkWiyxgTZyXg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e9fac98c3ec76b3be5c78c693a63b04a6aa748bbc73ae4809cddd538f93c85c2\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459841 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-mutant-screen-report">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/5/997" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-VY3H0nS7TJQkvAZ31PmhNYWRTl4_Dek7KSJjgOiDscI" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/5/997" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-w8DwuJsmnXzN2AHxvQChtQFXt79SPfKw8gDYve7VVzE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/5/983" title="Comparative Analysis of Wolbachia Genomes Reveals Streamlining and Divergence of Minimalist Two-Component Systems" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/5/1007" title="Rapid and Efficient Identification of Caenorhabditis elegans Legacy Mutations Using Hawaiian SNP-Based Mapping and Whole-Genome Sequencing" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459841" id="node459841" data-pisa="ggg;5/5/997" data-pisa-master="ggg;g3.115.016576" data-apath="/ggg/5/5/997.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Leveraging DNA Damage Response Signaling to Identify Yeast Genes Controlling Genome Stability</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jason A. Hendry</span>, <span class="highwire-citation-author" data-delta="1">Guihong Tan</span>, <span class="highwire-citation-author" data-delta="2">Jiongwen Ou</span>, <span class="highwire-citation-author" data-delta="3">Charles Boone</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-9002-5003" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Grant W. Brown</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">997-1006;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.016576 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459841" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jason A. Hendry </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry, University of Toronto, Toronto, Ontario, Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Donnelly Centre, University of Toronto, Toronto, Ontario, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jason%2BA.%2BHendry%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hendry%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJason%2BA.%2BHendry%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Guihong Tan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Donnelly Centre, University of Toronto, Toronto, Ontario, Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Molecular Genetics, University of Toronto, Toronto, Ontario, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guihong%2BTan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tan%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGuihong%2BTan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jiongwen Ou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry, University of Toronto, Toronto, Ontario, Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Donnelly Centre, University of Toronto, Toronto, Ontario, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jiongwen%2BOu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ou%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJiongwen%2BOu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Charles Boone </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Donnelly Centre, University of Toronto, Toronto, Ontario, Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Molecular Genetics, University of Toronto, Toronto, Ontario, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BBoone%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Boone%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACharles%2BBoone%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Grant W. Brown </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry, University of Toronto, Toronto, Ontario, Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Donnelly Centre, University of Toronto, Toronto, Ontario, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Grant%2BW.%2BBrown%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brown%20GW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGrant%2BW.%2BBrown%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-9002-5003" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Grant W. Brown</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:grant.brown@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">grant.brown@utoronto.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/5/997" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459841" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459841/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/5/997.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459841" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459841/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/5/997.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459841" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459841/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/5/997.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459841" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459841/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/5/997.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459841" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459841/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Oncogenesis frequently is accompanied by rampant genome instability, which fuels genetic heterogeneity and resistance to targeted cancer therapy. We have developed an approach that allows precise, quantitative measurement of genome instability in high-throughput format in the <em>Saccharomyces cerevisiae</em> model system. Our approach takes advantage of the strongly DNA damage-inducible gene <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">RNR3</a></em>, in conjunction with the reporter synthetic genetic array methodology, to infer mutants exhibiting genome instability by assaying for increased <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> abundance. We screen for genome instability across a set of ~1000 essential and ~4200 nonessential mutant yeast alleles in untreated conditions and in the presence of the DNA-damaging agent methylmethane sulfonate. Our results provide broad insights into the cellular processes and pathways required for genome maintenance. Through comparison with existing genome instability screens, we isolated 130 genes that had not previously been linked to genome maintenance, 51% of which have human homologs. Several of these homologs are associated with a genome instability phenotype in human cells or are causally mutated in cancer. A comprehensive understanding of the processes required to prevent genome instability will facilitate a better understanding of its sources in oncogenesis.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/ribonucleotide-reductase" class="hw-term hw-article-keyword hw-article-keyword-ribonucleotide-reductase" rel="nofollow">ribonucleotide reductase</a></li><li class="kwd"><a href="/keyword/dna-repair" class="hw-term hw-article-keyword hw-article-keyword-dna-repair" rel="nofollow">DNA repair</a></li><li class="kwd"><a href="/keyword/dna-damage" class="hw-term hw-article-keyword hw-article-keyword-dna-damage" rel="nofollow">DNA damage</a></li><li class="kwd"><a href="/keyword/dna-replication" class="hw-term hw-article-keyword hw-article-keyword-dna-replication" rel="nofollow">DNA replication</a></li><li class="kwd"><a href="/keyword/genome-instability" class="hw-term hw-article-keyword hw-article-keyword-genome-instability" rel="nofollow">genome instability</a></li></ul><p id="p-3">Genome maintenance is critical for cell viability and frequently is compromised in cancer. Loss of genome stability can be caused by exposure to sources of DNA damage, or defects in DNA repair or DNA replication genes. Genome instability can result in a diverse set of effects on DNA, including point mutations, small insertions and deletions, chromosome translocation or truncation, loss of heterozygosity, and even changes in ploidy (reviewed in <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Aguilera and Gomez-Gonzalez 2008</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Storchova and Pellman 2004</a>). Cancer cells frequently suffer from increased mutation rates and abnormal karyotypes, indicative of genome instability (reviewed in <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Vogelstein <em>et al.</em> 2013</a>). In fact, genome instability underlies the genetic heterogeneity that stymies targeted cancer therapy (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Glickman and Sawyers 2012</a>; <a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">Vogelstein <em>et al.</em> 2013</a>). Understanding the set of genes that function to maintain genome stability is critical to address the sources of genome instability in cancer. In somatic cancers especially, these sources remain unclear (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Negrini <em>et al.</em> 2010</a>).</p><p id="p-4">Several genome-wide screens assaying genome instability have been conducted with the budding yeast <em>S. cerevisiae</em> (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Alvaro <em>et al.</em> 2007</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Alver <em>et al.</em> 2013</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Andersen <em>et al.</em> 2008</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Askree <em>et al.</em> 2004</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Cheng <em>et al.</em> 2012</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Huang <em>et al.</em> 2003</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Stirling <em>et al.</em> 2011</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Smith <em>et al.</em> 2004</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Yu <em>et al.</em> 2006</a>). Because genome instability has diverse consequences, these screens have used a wide array of assays. Many focus on the consequences of genome instability on DNA, by assaying loss of genetic markers, increased mutation rate, or changes in telomere length (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Andersen <em>et al.</em> 2008</a>; <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Askree <em>et al.</em> 2004</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Huang <em>et al.</em> 2003</a>; <a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Stirling <em>et al.</em> 2011</a>). Others focus on assaying the relocalization of DNA repair or DNA damage checkpoint proteins to nuclear foci, which indicates their active involvement in genome maintenance (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Alvaro <em>et al.</em> 2007</a>; <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Cheng <em>et al.</em> 2012</a>). Another approach is to assay for activation of the DNA damage response (DDR) pathway, a signaling pathway that coordinates DNA repair processes with the cell-cycle (reviewed in <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Harrison and Haber 2006</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Putnam <em>et al.</em> 2009</a>). The DDR causes strong transcriptional induction of the gene <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">RNR3</a></em>, and assaying <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> abundance has been used to identify genome maintenance genes (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Zhou and Elledge 1992</a>). However, the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> abundance screen has yet to be combined with modern reagents and methodology for yeast genetics, such as the nonessential deletion mutant collection (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Giaever <em>et al.</em> 2002</a>), and synthetic genetic array (SGA) technology (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Tong and Boone 2006</a>).</p><p id="p-5">In <em>S. cerevisiae</em>, activation of the DDR begins when the single-stranded binding protein, RPA, coats exposed tracts of single-stranded DNA (ssDNA). Next, ssDNA-RPA is bound by <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002907">Ddc2</a>, in complex with the large phosphatidylinositol 3′ kinase-like kinase <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000340">Mec1</a> (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Rouse and Jackson 2002</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Zou and Elledge 2003</a>). <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000340">Mec1</a> is activated subsequently through interactions with multiple nearby protein assemblies, the most prominent being the 9-1-1 (Mec3-Rad17-Ddc1) clamp (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Majka <em>et al.</em> 2006</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Navadgi-Patil and Burgers 2009</a>). Another large phosphatidylinositol 3′ kinase-like kinase, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000184">Tel1</a>, is uniquely recruited to and activated at double-stand breaks (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Falck <em>et al.</em> 2005</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Nakada <em>et al.</em> 2003</a>). Activated <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000340">Mec1</a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000184">Tel1</a> propagate the damage signal by hyperphosphorylating the signaling adaptor <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002625">Rad9</a>, which mediates phosphorylation of the transducing kinase <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006074">Rad53</a> (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Naiki <em>et al.</em> 2004</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Schwartz <em>et al.</em> 2002</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Sweeney <em>et al.</em> 2005</a>). <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006074">Rad53</a> amplifies the DNA damage signaling through extensive autophosphorylation (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Sun <em>et al.</em> 1998</a>). In addition, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006074">Rad53</a> phosphorylates downstream targets, communicating the presence of DNA damage to diverse cellular processes. Two well-understood effects of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006074">Rad53</a> activation are the inhibition of late origin firing during S-phase (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Zegerman and Diffley 2010</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Lopez-Mosqueda <em>et al.</em> 2010</a>) and prevention of sister chromatid separation in anaphase (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Sanchez <em>et al.</em> 1999</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Agarwal <em>et al.</em> 2003</a>), both of which allow time for DNA repair to occur.</p><p id="p-6">Another well-studied effect of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006074">Rad53</a> activation is the transcriptional induction of the ribonucleotide reductase (RNR) complex, which catalyzes the rate-limiting step in dNTP synthesis. Many repair processes involve a DNA synthesis step and increased dNTP levels promote survival after DNA damage (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Chabes <em>et al.</em> 2003</a>). In <em>S. cerevisiae</em>, the RNR complex is encoded by two small (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003563">RNR2</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003412">RNR4</a></em>) and two large subunits (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000872">RNR1</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">RNR3</a></em>) (reviewed in <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Sanvisens <em>et al.</em> 2013</a>). In a normal cell cycle, two <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000872">Rnr1</a> subunits associate with <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003563">Rnr2</a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003412">Rnr4</a> in an α<sub>2</sub>ββ’ architecture, with <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> abundance being negligible. DDR activation induces the transcription of all four <em>RNR</em> subunits, albeit by different mechanisms and to different extents. <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003563">RNR2</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">RNR3</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003412">RNR4</a></em> contain 13bp damage response elements (DREs) in their promoters, which are bound by the transcriptional repressor <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004166">Crt1</a>/<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004166">Rfx1</a> (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Huang <em>et al.</em> 1998</a>). Active <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006074">Rad53</a> induces transcription of these <em>RNR</em> genes by phosphorylating the kinase <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002259">Dun1</a>, which in turn hyperphosphorylates <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004166">Crt1</a>, causing its dissociation from DNA (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Huang <em>et al.</em> 1998</a>). <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000872">RNR1</a></em> lacks DREs in its promoter and is induced in a Mec1-Rad53−dependent but Dun1-independent manner that involves a different transcription factor, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001515">Ixr1</a> (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Tsaponina <em>et al.</em> 2011</a>).</p><p id="p-7"><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> is an ideal read-out of DDR pathway activation, and by extension genome instability. First, it is a well-characterized transcriptional target of the DNA damage response (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Huang <em>et al.</em> 1998</a>). <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">RNR3</a></em> up-regulation in response to exogenous DNA-damaging agents like methylmethane sulfonate (MMS) has been demonstrated at both the mRNA and protein level and depends on known DDR kinases (<a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">Huang <em>et al.</em> 1998</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Li and Reese 2001</a>; <a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Tsaponina <em>et al.</em> 2011</a>). Second, mutation of several well-characterized DNA repair and replication genes leads to constitutive expression of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">RNR3</a></em>, demonstrating that <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">RNR3</a></em> can be induced by both exogenous (<em>i.e.</em>, environmental) and endogenous (<em>i.e.</em>, genetic perturbation) sources of genome instability (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Tang <em>et al.</em> 2009</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Davidson <em>et al.</em> 2012</a>). Finally, expression of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">RNR3</a></em> is negligible in the absence of perturbation, but it is precipitously induced in response to DNA damage, by far the greatest induction of all the <em>RNR</em> genes (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Elledge <em>et al.</em> 1993</a>). The original <span class="underline">C</span>onstitutive <span class="underline">R</span>NR <span class="underline">T</span>hree (CRT) screen used an <em>RNR3-URA3</em> transcriptional fusion and spontaneous mutagenesis as means to identify mutants causing genome instability (<a id="xref-ref-69-2" class="xref-bibr" href="#ref-69">Zhou and Elledge 1992</a>). We sought to complement this approach using modern yeast genetic tools, incorporating the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> assay into a comprehensive genome-wide screen.</p><p id="p-8">We generated a fluorescent reporter for <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">RNR3</a></em> expression compatible with reporter synthetic genetic array (R-SGA) technology, allowing us to systematically measure <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> abundance across ~5200 yeast mutants (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Kainth <em>et al.</em> 2009</a>). We identified 150 mutants with increased <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> abundance in the absence of exogenous perturbation and, in a second screen, 200 mutants with increased <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> abundance in the presence of the DNA damaging agent MMS. We identified known repressors of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">RNR3</a></em> transcription, such as <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004166">crt1</a>∆</em>, validating our reporter system and saw high enrichment for processes known to impinge upon genome maintenance, such as DNA replication and repair, validating the use of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> abundance as an effective indicator of genome instability. Furthermore, by systematic comparison with existing genome instability datasets, we determined that 133 mutants, representing 130 different genes, 51% of which have human orthologs, are unique to our <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> read-out of genome instability,</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Rnr3 R-SGA</h3><p id="p-9">Two independent crosses of GBY691 (MATα <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">RNR3</a></em>-<em>GFP</em>::<em>HIS3MX <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000523">leu2</a>∆0 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">his3</a>∆1 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004294">met15</a>∆0 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005212">lyp1</a>∆ can1pr</em>::<em>RPL39pr-tdTomato</em>::<em>CaURA3</em>::<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000789">can1</a>∆</em>::<em>STE2pr-LEU2</em>) with the yeast nonessential deletion collection and a set of conditional temperature-sensitive alleles of essential genes (G. Tan and C. Boone, unpublished data) were performed following standard SGA procedures (<a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Tong and Boone 2006</a>). Final arrays were pinned in duplicate on either SD/MSG –his –leu –ura +200 mg/mL G418 (untreated) or YPD supplemented with 0.03% MMS, and grown for 20 hr before fluorescence scanning. Media was poured 48 hr before use, allowing for drying sufficient to prevent condensation on the fluorescence scanner. The Typhoon Trio Variable Mode Imager (GE Healthcare) was used to acquire GFP (488-nm laser, 520/40 BP emission filter) and tdTomato (532-nm laser, 610/30 BP emission filter) fluorescence values. For the essential temperature-sensitive mutants, all growth was conducted at 23°. After fluorescence imaging, colony size data were acquired by individually photographing plates with a Canon PowerShot G2 4.0 megapixel digital camera using Remote Capture software. Data analysis followed essentially what is described in <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Kainth <em>et al.</em> (2009)</a>, with small variations. To summarize, background-subtracted green fluorescent protein (GFP) and tdTomato intensities were computed for each colony from .GEL images using GenePix Pro version 3.0 software. Colony size information was calculated from individual photographs using Qt ColonyImager software, version 1.0.1. Border colonies, small colonies (colony area &lt; 500 pixels), and colonies with aberrantly low tdTomato values (bottom 0.05%) were removed before further analysis. log<sub>2</sub>(Rnr3-GFP/tdTomato) values were calculated and LOESS normalized for each replicate experiment. Using the log<sub>2</sub>(Rnr3-GFP/tdTomato) ratio as a metric for <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> abundance has the advantage that dividing by tdTomato corrects for any colony size-dependent intensity effects. Finally, normalized log<sub>2</sub>(Rnr3-GFP/tdTomato) values were averaged across all replicate experiments and a Z-score calculated (<a href="http://www.g3journal.org/content/suppl/2015/02/26/g3.115.016576.DC1">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2015/02/26/g3.115.016576.DC1/TableS1.xlsx">Table S1</a> and <a href="http://www.g3journal.org/content/suppl/2015/02/26/g3.115.016576.DC1/TableS2.xlsx">Table S2</a>; Raw R-SGA data, <a href="http://www.g3journal.org/content/suppl/2015/02/26/g3.115.016576.DC1">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2015/02/26/g3.115.016576.DC1/FileS1.xlsx">File S1</a>). All analysis was performed in R.</p></div><div id="sec-3" class="subsection"><h3>Gene Ontology (GO) enrichment analysis</h3><p id="p-10">The GO Annotations Database was downloaded on September 23, 2014 (<a href="http://geneontology.org/page/download-annotations">http://geneontology.org/page/download-annotations</a>). A hypergeometric test was used to compute <em>P</em>-values for all GO terms within the biological process, molecular function and cellular component ontologies. GO terms with greater than 2000 associated genes were omitted as they are often too general to be informative. Similarly, for the biological process ontology, GO terms with less than 10 associated genes were omitted to remove very specific and largely redundant terms. <em>P</em>-values were adjusted for multiple hypothesis testing using the Benjamini-Hochberg method with a cut-off of FDR &lt; 0.05.</p></div><div id="sec-4" class="subsection"><h3>Homology analysis</h3><p id="p-11">The inParanoid8 database for <em>Saccharomyces cerevisiae</em> to <em>Homo sapiens</em> homology was downloaded (<a href="http://inparanoid.sbc.su.se/cgi-bin/e.cgi">http://inparanoid.sbc.su.se/cgi-bin/e.cgi</a>) to determine yeast-human homology relationships. The set of human cancer genes casually mutated in cancer were downloaded from the Cancer Gene Consensus (<a href="http://cancer.sanger.ac.uk/cancergenome/projects/census/">http://cancer.sanger.ac.uk/cancergenome/projects/census/</a>).</p></div></div><div class="section results" id="sec-5"><h2 class="">Results</h2><div id="sec-6" class="subsection"><h3>A genome-wide screen for mutants with increased genome instability using Rnr3 abundance</h3><p id="p-12"><em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">RNR3</a></em> exhibits a strong and specific induction in response to genome instability in <em>S. cerevisiae</em>. We sought to adapt <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">RNR3</a></em> expression into a reporter for genome instability that would be amenable to a genome-wide mutant screen. To this end, we adopted the R-SGA methodology (<a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Kainth <em>et al.</em> 2009</a>), which allows for high-throughput acquisition of the abundance of a protein of interest in the context of a yeast mutant collection. We used R-SGA to query <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> abundance across the set of yeast nonessential deletion mutants, containing ~4200 open-reading frame deletions (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Giaever <em>et al.</em> 2002</a>), as well as a set of ~1000 <em>kanMX</em>-marked, temperature-sensitive alleles that cover ~600 essential genes (G. Tan and C. Boone, unpublished data).</p><p id="p-13">To conduct R-SGA, a strain containing endogenous <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">RNR3</a></em> C-terminally tagged with GFP and expressing tdTomato constitutively from the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003725">RPL39</a></em> promoter was mated with both the non-essential gene deletion and conditional temperature-sensitive (TS) essential gene mutant collections using the SGA methodology (<a id="xref-ref-62-3" class="xref-bibr" href="#ref-62">Tong and Boone 2006</a>). The result was a collection of strains harboring <em>RNR3-GFP</em>, <em>RPL39pr-tdTomato</em>, and either a unique nonessential gene deletion (<em>xxxΔ</em>) or a TS essential gene allele (<em>xxx-TS</em>). Using a fluorescence scanner, we assayed the GFP and tdTomato intensities across the mutant strains arrayed as whole colonies on agar plates, and then we calculated a normalized log<sub>2</sub>(GFP/tdTomato) ratio, which is indicative of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> abundance. The screen was conducted in quadruplicate and on both untreated (UT) media and media containing the DNA-damaging agent MMS (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). The average correlation across replicate screens was R = 0.73 and R = 0.95 for the nonessential deletion collection and temperature-sensitive collection, respectively. The correlations compare favorably with previous R-SGA screens, where an average of R = 0.77 was observed across 27 screens (<a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">Kainth <em>et al.</em> 2009</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/5/997/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Assaying Rnr3 abundance across ~5200 yeast mutants, untreated (UT) and in methylmethane sulfonate (MMS). (A) Schematic of the Rnr3 R-SGA screening methodology. ~5200 yeast mutant strains containing RNR3-GFP and RPL39pr-tdTomato were either untreated or treated with MMS before fluorescence scanning and Rnr3 abundance quantification. Potential cellular functions of mutants with increased or decreased Rnr3 are described. (B) Distribution of Rnr3 abundances across all mutants screened in UT. (C) Distribution of Rnr3 abundance in MMS. (D) Overlap between UT and MMS screens for mutants with increased and decreased Rnr3 abundance. Both overlaps are significant. Total number of mutants unique to each condition is shown as a percentage, as well as the total number of mutants with increased and decreased Rnr3 across both conditions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1508284888" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Assaying Rnr3 abundance across ~5200 yeast mutants, untreated (UT) and in methylmethane sulfonate (MMS). (A) Schematic of the Rnr3 R-SGA screening methodology. ~5200 yeast mutant strains containing &lt;em&gt;RNR3-GFP&lt;/em&gt; and &lt;em&gt;RPL39pr-tdTomato&lt;/em&gt; were either untreated or treated with MMS before fluorescence scanning and Rnr3 abundance quantification. Potential cellular functions of mutants with increased or decreased Rnr3 are described. (B) Distribution of Rnr3 abundances across all mutants screened in UT. (C) Distribution of Rnr3 abundance in MMS. (D) Overlap between UT and MMS screens for mutants with increased and decreased Rnr3 abundance. Both overlaps are significant. Total number of mutants unique to each condition is shown as a percentage, as well as the total number of mutants with increased and decreased Rnr3 across both conditions.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/5/997/F1.medium.gif" width="440" height="300"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/5/997/F1.medium.gif" width="440" height="300"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/5/997/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/5/997/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466157" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-14" class="first-child">Assaying Rnr3 abundance across ~5200 yeast mutants, untreated (UT) and in methylmethane sulfonate (MMS). (A) Schematic of the Rnr3 R-SGA screening methodology. ~5200 yeast mutant strains containing <em>RNR3-GFP</em> and <em>RPL39pr-tdTomato</em> were either untreated or treated with MMS before fluorescence scanning and Rnr3 abundance quantification. Potential cellular functions of mutants with increased or decreased Rnr3 are described. (B) Distribution of Rnr3 abundances across all mutants screened in UT. (C) Distribution of Rnr3 abundance in MMS. (D) Overlap between UT and MMS screens for mutants with increased and decreased Rnr3 abundance. Both overlaps are significant. Total number of mutants unique to each condition is shown as a percentage, as well as the total number of mutants with increased and decreased Rnr3 across both conditions.</p><div class="sb-div caption-clear"></div></div></div><p id="p-15">We hypothesized that mutants causing increased <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">RNR3</a></em> expression in the absence of exogenous perturbation (untreated conditions) would unveil two classes of genes: (1) those directly regulating <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">RNR3</a></em> transcription and (2) those involved in the maintenance of genome stability (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>). By further performing the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> R-SGA screen in the presence of MMS (an exogenous source of DNA damage), we aimed to isolate genome maintenance factors that function specifically in tolerance of MMS damage.</p><p id="p-16">We used a Z-score based threshold to identify mutants with the greatest change in <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> abundance, defining those mutants with Z &gt; 2 (corresponding to an <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> abundance two standard deviations above the mean) as having increased <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> and those with Z &lt; −2 as having decreased <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a>. Using these thresholds, we identified 150 mutants (2.9% of those screened) with increased <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> abundance in untreated conditions and 91 with decreased <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> abundance (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>, and <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, <a href="http://www.g3journal.org/content/suppl/2015/02/26/g3.115.016576.DC1/TableS3.xlsx">Table S3</a>, and <a href="http://www.g3journal.org/content/suppl/2015/02/26/g3.115.016576.DC1/TableS4.xlsx">Table S4</a>). In MMS, we identified 200 mutants (3.9%) with increased <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> and 55 mutants with decreased <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1C</a>, <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>, <a href="http://www.g3journal.org/content/suppl/2015/02/26/g3.115.016576.DC1/TableS5.xlsx">Table S5</a>, and <a href="http://www.g3journal.org/content/suppl/2015/02/26/g3.115.016576.DC1/TableS6.xlsx">Table S6</a>). A larger fraction of mutants with increased <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> abundance were nonessential (~66% for UT and MMS); however, a greater relative proportion of all essential mutants screened had increased <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> abundance (<em>e.g.</em>, ~34% of the 150 mutants with increased <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> in UT are essential, although essential genes comprise only ~19% of the mutants screened). A tendency for mutation of essential genes to more readily cause genome instability is consistent with a previous study that similarly used both essential and nonessential mutant collections (<a id="xref-ref-56-3" class="xref-bibr" href="#ref-56">Stirling <em>et al.</em> 2011</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466119/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466119/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Evolutionary conservation of putative novel genome instability genes</span><div class="sb-div caption-clear"></div></div></div><p id="p-18">Altogether, we identified a total of 285 mutants that cause increased <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> abundance and 138 mutants with decreased <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> abundance. Both the UT and MMS screen shared a significant number of mutants with increased and decreased <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> levels. Nevertheless, we identified a large number of mutants unique to each condition (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1D</a>). Indeed, MMS treatment resulted in the identification of an additional 135 mutants with increased and 47 mutants with decreased <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a>, indicating a change in the complement of genes that determine <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> abundance when DNA damage is present.</p></div><div id="sec-7" class="subsection"><h3>Mutants with increased Rnr3 abundance</h3><p id="p-19">We first focused our attention on those mutants that exhibited an increase in <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> abundance, which we expected to fall into two distinct functional classes: (1) those directly regulating <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">RNR3</a></em> transcription and (2) those involved in the maintenance of genome instability.</p><p id="p-20">In a normal cell cycle, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">RNR3</a></em> expression is inhibited by the transcriptional repressor <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004166">Crt1</a>/<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004166">Rfx1</a>, which binds to three DREs located upstream <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">RNR3</a></em>′<em>s</em> promoter (<a id="xref-ref-28-5" class="xref-bibr" href="#ref-28">Huang <em>et al.</em> 1998</a>). Consistent with this, a <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004166">CRT1</a></em> deletion mutant had by far the greatest abundance of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> in our R-SGA assay conducted in untreated conditions (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1B</a>). In addition, several other genes with established connections to <em>RNR</em> transcriptional repression, including <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005831">ISW2</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004966">HDA1</a></em>, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006269">ROX1</a></em> (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Klinkenberg <em>et al.</em> 2006</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Sharma <em>et al.</em> 2007</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Zhang and Reese 2004</a>), possessed an increased <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> abundance upon deletion (<a href="http://www.g3journal.org/content/suppl/2015/02/26/g3.115.016576.DC1/TableS3.xlsx">Table S3</a>). Our data also implicate <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003101">ITC1</a></em>, a binding partner of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005831">ISW2</a></em> (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Goldmark <em>et al.</em> 2000</a>), and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006383">HDA3</a></em>, which forms a complex with <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004966">HDA1</a></em> (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Carmen <em>et al.</em> 1996</a>), in the transcriptional repression of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a>, although this has not been explicitly demonstrated previously. For both complexes, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> induction upon deletion of either of the two constituent members was highly similar, suggesting that R-SGA can yield precise quantitative data on protein abundance. Together, these results confirm that the R-SGA methodology identifies known regulators of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> transcription.</p><p id="p-21">Next, we sought to establish whether <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> R-SGA could recover mutants belonging to our second expected class of hits: genes functioning to maintain genome stability. To take a broad and unbiased approach, we used the GO database, a curated set of terms describing known properties of the protein product of a gene. First, we examined the localization of genes whose mutation causes increased <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> abundance. GO analysis using the cellular component ontology indicated that mutants with increased <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> abundance are significantly enriched for genes that reside within the nucleus (<a href="http://www.g3journal.org/content/suppl/2015/02/26/g3.115.016576.DC1/TableS7.xlsx">Table S7</a>). In fact, 79 of 140 genes with increased <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> abundance have a nuclear localization, suggesting a direct role in genome maintenance. Analysis of the molecular function ontology supported the idea that many of the proteins with elevated <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> abundance could directly influence genome stability. Namely, 31 genes possess direct DNA binding activity and 12 interact with chromatin (<a href="http://www.g3journal.org/content/suppl/2015/02/26/g3.115.016576.DC1/TableS7.xlsx">Table S7</a>). Moreover, several genes contain domains that function to modify DNA topology, such as those possessing “DNA-dependent ATPase activity” (GO:0004003) and “helicase activity” (GO:0004386).</p><p id="p-22">Finally, we examined the biological processes enriched among mutants with increased <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> abundance in untreated conditions (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). These terms overwhelmingly reflected processes that impinge upon genome stability. “DNA replication” (GO:0006260) was the most significantly enriched term, consistent with this biological process posing the greatest risk to genome stability in an unperturbed cell. Functionally similar GO terms “DNA repair” (GO:0006281) and “response to DNA damage stimulus” (GO:00060974) also were highly enriched. In aggregate, the 20 significantly enriched GO biological process terms included 59 unique mutants, covering 42% of all genes with increased <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> abundance.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/5/997/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Significant biological processes present among mutants with increased Rnr3 abundance. (A) Breakdown of all biological process Gene Ontology (GO) terms significantly enriched among genes with increased Rnr3 in either the untreated (UT; green) or methylmethane sulfonate (MMS; yellow) screen. Terms are ordered from top by significance in the UT screen. Pie charts present the fraction of genes with increased Rnr3 annotated with at least one significant GO term. (B) Overlap of genes within the biological processes &#x201C;DNA replication&#x201D; (GO:0006260) and &#x201C;DNA repair&#x201D; (GO:0006281) between UT and MMS. (C) Overlap of nonessential genes with increased Rnr3 abundance with a set of nonessential mutants identified as MMS sensitive (MMSS). Both screens overlap significantly, but MMS has almost twice as many shared genes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1508284888" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Significant biological processes present among mutants with increased Rnr3 abundance. (A) Breakdown of all biological process Gene Ontology (GO) terms significantly enriched among genes with increased Rnr3 in either the untreated (UT; green) or methylmethane sulfonate (MMS; yellow) screen. Terms are ordered from top by significance in the UT screen. Pie charts present the fraction of genes with increased Rnr3 annotated with at least one significant GO term. (B) Overlap of genes within the biological processes &#x201C;DNA replication&#x201D; (GO:0006260) and &#x201C;DNA repair&#x201D; (GO:0006281) between UT and MMS. (C) Overlap of nonessential genes with increased Rnr3 abundance with a set of nonessential mutants identified as MMS sensitive (MMS&lt;sup&gt;S&lt;/sup&gt;). Both screens overlap significantly, but MMS has almost twice as many shared genes.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/5/997/F2.medium.gif" width="440" height="342"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/5/997/F2.medium.gif" width="440" height="342"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/5/997/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/5/997/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465985" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-23" class="first-child">Significant biological processes present among mutants with increased Rnr3 abundance. (A) Breakdown of all biological process Gene Ontology (GO) terms significantly enriched among genes with increased Rnr3 in either the untreated (UT; green) or methylmethane sulfonate (MMS; yellow) screen. Terms are ordered from top by significance in the UT screen. Pie charts present the fraction of genes with increased Rnr3 annotated with at least one significant GO term. (B) Overlap of genes within the biological processes “DNA replication” (GO:0006260) and “DNA repair” (GO:0006281) between UT and MMS. (C) Overlap of nonessential genes with increased Rnr3 abundance with a set of nonessential mutants identified as MMS sensitive (MMS<sup>S</sup>). Both screens overlap significantly, but MMS has almost twice as many shared genes.</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">In addition to uncovering many biological processes associated with genome instability, we also identified several protein complexes associated with the maintenance of genome stability (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). Interestingly, two of these complexes, the Cul8-RING Ubiquitin ligase complex and the StUbL (<em>i.e.</em>, <span class="underline">S</span>UMO-<span class="underline">t</span>argeted <span class="underline">U</span>biquitin <span class="underline">l</span>igase) complex are related to ubiquitin transactions. All three members of lagging strand polymerase, DNA polymerase δ (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002260">POL3</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003766">POL31</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003804">POL32</a></em>), were recovered, as well as both core members of DNA primase <em>(<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001447">PRI1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001528">PRI2</a>)</em>.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/5/997/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Significant protein complexes present among mutants with increased Rnr3 abundance. Protein complexes with significant enrichment in either the untreated or MMS screen are shown. Number of circles corresponds to the number of genes within the complex, filled circles indicates the number of those genes with increased Rnr3 abundance. Color corresponds to the significance of complex enrichment. Two complexes from the MMS screen with insignificant enrichment P-values are indicated in gray, as both their constituent genes were identified [StUbL (i.e., SUMO-targeted Ubiquitin ligase) complex and Primosome complex]. Complexes were derived from the GO cellular component term set (Table S7 and Table S8). MMS, methylmethane sulfonate; GO, Gene Ontology. " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1508284888" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Significant protein complexes present among mutants with increased Rnr3 abundance. Protein complexes with significant enrichment in either the untreated or MMS screen are shown. Number of circles corresponds to the number of genes within the complex, filled circles indicates the number of those genes with increased Rnr3 abundance. Color corresponds to the significance of complex enrichment. Two complexes from the MMS screen with insignificant enrichment &lt;em&gt;P&lt;/em&gt;-values are indicated in gray, as both their constituent genes were identified [StUbL (&lt;em&gt;i.e.&lt;/em&gt;, SUMO-targeted Ubiquitin ligase) complex and Primosome complex]. Complexes were derived from the GO cellular component term set (Table S7 and Table S8). MMS, methylmethane sulfonate; GO, Gene Ontology. &lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/5/997/F3.medium.gif" width="430" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/5/997/F3.medium.gif" width="430" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/5/997/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/5/997/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466025" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-25" class="first-child">Significant protein complexes present among mutants with increased Rnr3 abundance. Protein complexes with significant enrichment in either the untreated or MMS screen are shown. Number of circles corresponds to the number of genes within the complex, filled circles indicates the number of those genes with increased Rnr3 abundance. Color corresponds to the significance of complex enrichment. Two complexes from the MMS screen with insignificant enrichment <em>P</em>-values are indicated in gray, as both their constituent genes were identified [StUbL (<em>i.e.</em>, <span class="underline">S</span>UMO-<span class="underline">t</span>argeted <span class="underline">U</span>biquitin <span class="underline">l</span>igase) complex and Primosome complex]. Complexes were derived from the GO cellular component term set (<a href="http://www.g3journal.org/content/suppl/2015/02/26/g3.115.016576.DC1/TableS7.xlsx">Table S7</a> and <a href="http://www.g3journal.org/content/suppl/2015/02/26/g3.115.016576.DC1/TableS8.xlsx">Table S8</a>). MMS, methylmethane sulfonate; GO, Gene Ontology. </p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-8" class="subsection"><h3>Mutants with increased Rnr3 abundance in MMS</h3><p id="p-26">We sought to expand our effort to understand genome stability to conditions of induced DNA damage. To this end, we repeated the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> R-SGA screen in the context of the carcinogen and DNA damaging agent MMS. MMS is a monofunctional DNA alkylating agent that primarily methylates DNA on <em>N<sup>7</sup></em>-deoxyguanine and <em>N<sup>3</sup></em>-deoxyadenine (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Beranek 1990</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Lawley 1989</a>). The <em>N<sup>3</sup></em>-methyladenine lesion in particular is thought to impede replication fork progression, which can lead to replication fork collapse (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Beranek 1990</a>). We reasoned that mutants with a greater increase in <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> abundance in the presence of MMS would have functions specific to repair of MMS damage.</p><p id="p-27">Results from the MMS <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> R-SGA screen presented some commonalities with the UT <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> R-SGA screen. For example, mutants with increased <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> abundance in MMS were more likely to reside in the nucleus and have biochemical activities relevant to DNA maintenance (<a href="http://www.g3journal.org/content/suppl/2015/02/26/g3.115.016576.DC1/TableS8.xlsx">Table S8</a>). Once again, “DNA replication” (GO:0006260), “DNA repair” (GO:0006821), and “response to DNA damage stimulus” (GO:0006974) were the three most significantly enriched GO biological process terms (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>). However, nearly double the number of genes associated with and the DDR were identified in the presence of MMS. In fact, the MMS screen identified nearly all the DNA repair annotated genes with increased <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> abundance in UT conditions, plus an additional 26 DNA repair annotated genes (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>).</p><p id="p-28">We wanted to assess the degree to which treatment with MMS resulted in the identification of mutants associated specifically with MMS repair. The <em>N<sup>3</sup></em>-methyladenine lesions produced by MMS treatment are addressed through three mechanisms of repair: (1) base excision repair, (2) recombinational repair and (3) postreplication repair (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Xiao <em>et al.</em> 1996</a>). “Base-excision repair” (GO:0006284) and “postreplication repair” (GO:0006301) were significantly enriched terms in both UT and MMS screens; however, more BER and PRR genes were identified in the presence of MMS (6/15 BER genes in MMS <em>vs.</em> 5/15 in UT; 7/11 PRR genes in MMS <em>vs.</em> 4/11 in UT; <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2A</a>). Notably, the base-excision repair gene <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000944">MAG1</a></em> (<em>i.e.</em>, 3-<span class="underline">M</span>ethyl<span class="underline">A</span>denine DNA <span class="underline">G</span>lycosylase), the deletion of which increases the expression of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">RNR3</a></em> in MMS (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Jia and Xiao 2003</a>), was identified specifically in the MMS <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> R-SGA screen. The greater enrichment of postreplication repair was driven in part by identification of the entire E3 ubiquitin ligase complex containing <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004022">RAD5</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002499">UBC13</a></em>, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003055">MMS2</a></em>, which poly-ubquitylates <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000292">proliferating cell nuclear antigen</a> in an early step of error-free postreplication repair (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Hoege <em>et al.</em> 2002</a>). The final mechanism used in repair of MMS damage, “recombinational repair” (GO:0000725), was only enriched in the MMS screen (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2A</a>). Here, we identified all members of the Shu complex (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001394">CSM2</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004368">PSY3</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000998">SHU1</a></em>), which mediates recombinational repair of stalled replication forks (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>) (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Ball <em>et al.</em> 2009</a>). Other mutants found specifically in MMS were <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001441">MPH1</a></em>, which is thought to drive regression of stalled replication forks, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000302">MMS4</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002794">MUS81</a></em>, which play an important role in processing of stalled replication forks for homologous recombination (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Choi <em>et al.</em> 2010</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Osman and Whitby 2007</a>). Finally, we noted a prominent difference between the two screens with regards to the process “mitotic sister chromatid cohesion” (GO:0007064), which was highly enriched only in the presence of MMS (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2A</a>).</p><p id="p-29">As further evidence of the ability of our <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> R-SGA screen to capture mutants involved in MMS repair, we compared mutants from both UT and MMS screens with a set of non-essential deletion mutants previously identified as being MMS sensitive (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2C</a>) (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Chang <em>et al.</em> 2002</a>). Both the UT and MMS <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> R-SGA screen significantly overlapped with the MMS sensitive dataset. However, the MMS <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> R-SGA screen had twice as many mutants in common with the MMS sensitivity screen as did the UT <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> R-SGA screen.</p></div><div id="sec-9" class="subsection"><h3>Comparison with previous genome instability screens</h3><p id="p-30">Several genome instability studies have been conducted in <em>S. cerevisiae</em> (<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Alvaro <em>et al.</em> 2007</a>; <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Alver <em>et al.</em> 2013</a>; <a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Andersen <em>et al.</em> 2008</a>; <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Askree <em>et al.</em> 2004</a>; <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Cheng <em>et al.</em> 2012</a>; <a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Huang <em>et al.</em> 2003</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Scholes <em>et al.</em> 2001</a>; <a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Smith <em>et al.</em> 2004</a>; <a id="xref-ref-56-4" class="xref-bibr" href="#ref-56">Stirling <em>et al.</em> 2011</a>). Many of these studies have made use of the same collection of nonessential deletion mutants and more recently the use of hypomorphic or temperature-sensitive mutant collections similar to the one we used has been reported (<a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Cheng <em>et al.</em> 2012</a>; <a id="xref-ref-56-5" class="xref-bibr" href="#ref-56">Stirling <em>et al.</em> 2011</a>). We used this existing pool of data to conduct a thorough assessment of whether any of the 285 mutants with increased <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> abundance could potentially represent novel genome maintenance genes. We focused on conducting a comparison to studies that directly assayed genome instability, as drug sensitivity and global protein localization studies can capture effects additional to genome maintenance (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Tkach <em>et al.</em> 2012</a>). To this end, we chose four major and orthogonal genome instability screens done in yeast to cross reference with our dataset: (1) 33 nonessential mutants possessing an elevated mutation rate (<a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">Huang <em>et al.</em> 2003</a>), (2) 87 essential mutants with increased <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002907">Ddc2</a> foci (<a id="xref-ref-12-5" class="xref-bibr" href="#ref-12">Cheng <em>et al.</em> 2012</a>), an indication of exposed ssDNA tracts, (3) 89 nonessential mutants with elevated <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004494">Rad52</a> foci (<a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">Alvaro <em>et al.</em> 2007</a>), an indication of double strand DNA breaks, and (4) 692 essential and nonessential mutants with chromosome instability (<a id="xref-ref-56-6" class="xref-bibr" href="#ref-56">Stirling <em>et al.</em> 2011</a>), roughly defined as increased loss of genetic material, as assessed by several different assays. In addition, we collected genes with the GO annotations “DNA repair” (GO:0006281) and “DNA replication” (GO:0006260), to capture mutants with a known association to these processes that haven’t been captured in high-throughput genome instability studies. We systematically annotated each mutant with increased <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> abundance for its presence in any of the four genome instability datasets or two GO terms, and also annotated known transcriptional regulators of <em>RNR</em> (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). </p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/5/997/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Systematic comparison with existing genome instability screens and genome maintenance-related GO terms. (A) Putative novel genome maintenance genes are identified by comparative gene subtraction. The 150 mutants with increased Rnr3 abundance in UT conditions are represented in each row, in order of decreasing Rnr3 abundance. In the top row, yellow rectangles represent essential and blue rectangles represent nonessential mutants. In the following seven rows, mutants are colored green if they are present in the data set listed at left, and gray otherwise. In the bottom row, all previously identified mutants are removed, leaving the putative novel mutants. Note that the majority of novel mutants cluster toward the right, indicating they have a lower Rnr3 abundance. The table on the right gives the number of genes with increased Rnr3 present in each data set, as well as the significance of the overlap between the datasets. (B) Same as in (A), but for the MMS screen. (C) Overlap between the putative novel genome maintenance genes identified in UT and MMS screens. A total of 133 novel mutants are identified. GO, Gene Ontology MMS, methylmethane sulfonate; UT, untreated." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1508284888" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Systematic comparison with existing genome instability screens and genome maintenance-related GO terms. (A) Putative novel genome maintenance genes are identified by comparative gene subtraction. The 150 mutants with increased Rnr3 abundance in UT conditions are represented in each row, in order of decreasing Rnr3 abundance. In the top row, yellow rectangles represent essential and blue rectangles represent nonessential mutants. In the following seven rows, mutants are colored green if they are present in the data set listed at left, and gray otherwise. In the bottom row, all previously identified mutants are removed, leaving the putative novel mutants. Note that the majority of novel mutants cluster toward the right, indicating they have a lower Rnr3 abundance. The table on the right gives the number of genes with increased Rnr3 present in each data set, as well as the significance of the overlap between the datasets. (B) Same as in (A), but for the MMS screen. (C) Overlap between the putative novel genome maintenance genes identified in UT and MMS screens. A total of 133 novel mutants are identified. GO, Gene Ontology MMS, methylmethane sulfonate; UT, untreated.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/5/997/F4.medium.gif" width="440" height="364"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/5/997/F4.medium.gif" width="440" height="364"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/5/997/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/5/997/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466012" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-31" class="first-child">Systematic comparison with existing genome instability screens and genome maintenance-related GO terms. (A) Putative novel genome maintenance genes are identified by comparative gene subtraction. The 150 mutants with increased Rnr3 abundance in UT conditions are represented in each row, in order of decreasing Rnr3 abundance. In the top row, yellow rectangles represent essential and blue rectangles represent nonessential mutants. In the following seven rows, mutants are colored green if they are present in the data set listed at left, and gray otherwise. In the bottom row, all previously identified mutants are removed, leaving the putative novel mutants. Note that the majority of novel mutants cluster toward the right, indicating they have a lower Rnr3 abundance. The table on the right gives the number of genes with increased Rnr3 present in each data set, as well as the significance of the overlap between the datasets. (B) Same as in (A), but for the MMS screen. (C) Overlap between the putative novel genome maintenance genes identified in UT and MMS screens. A total of 133 novel mutants are identified. GO, Gene Ontology MMS, methylmethane sulfonate; UT, untreated.</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">From the pool of mutants with increased <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> in untreated conditions, 84 mutants could be assigned to at least one of the above annotations leaving 66 mutants (44%) as potentially novel genome maintenance genes (<a href="http://www.g3journal.org/content/suppl/2015/02/26/g3.115.016576.DC1/TableS9.xlsx">Table S9</a>). Similarly, 115 mutants with increased <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> abundance in MMS had at least one genome instability annotation, leaving 85 (43%) mutants unannotated (<a href="http://www.g3journal.org/content/suppl/2015/02/26/g3.115.016576.DC1/TableS10.xlsx">Table S10</a>). For both the UT and MMS screen, putative novel mutants had a tendency to be ranked lower than those associated with existing genome instability datasets. Overlap of the two sets of putative novel mutants revealed 18 shared mutants, resulting in an aggregate of 133 novel mutants identified (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4C</a>). Among these genes, GO enrichment produced no significantly enriched terms, suggesting that the putative novel mutants are not functionally cohesive and derive from a wide array of biological processes. Some of the putative novel mutants have inferable connections to genome maintenance, despite not being identified in previous genome instability screens. For example, we identified two alleles of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003818">CDC8</a></em> (<em>cdc8-1 and cdc8-2</em>), which is involved in dTTP synthesis (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Jong <em>et al.</em> 1984</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Sclafani and Fangman 1984</a>), and therefore likely causes genome instability by affecting DNA replication (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Hartwell 1973</a>). Others are less easily linked to genome maintenance, such as <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005730">DED1</a></em> (again supported by two alleles, <em>ded1-95</em> and <em>ded1-199</em>), an ATP-dependent RNA helicase important for translation initiation (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Chuang <em>et al.</em> 1997</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Iost <em>et al.</em> 1999</a>).</p></div><div id="sec-10" class="subsection"><h3>Evolutionary conservation of mutants with increased Rnr3 abundance</h3><p id="p-33">We used the ortholog cluster database generated by the inParanoid algorithm to evaluate the extent to which genes with increased <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> abundance are conserved from yeast to human (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Sonnhammer and Ostlund 2014</a>). The core of the inParanoid algorithm is a reciprocal best-match BLAST search between proteomes of two species of interest, which is used to identify ortholog pairs. In total, the inParanoid algorithm assigns 2386 yeast genes a human ortholog, corresponding to 36% of the yeast genome. We identified 78 (52%) yeast genes with human orthologs among the set of genes with increased <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> abundance in untreated conditions (<a href="http://www.g3journal.org/content/suppl/2015/02/26/g3.115.016576.DC1/TableS11.xlsx">Table S11</a>). Similarly, 106 (53%) of the mutants with increased <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> in MMS had a human ortholog (<a href="http://www.g3journal.org/content/suppl/2015/02/26/g3.115.016576.DC1/TableS12.xlsx">Table S12</a>). Interestingly, considering only our putative novel set of mutants has little effect on the fraction of genes conserved: 33 (50%) novel UT and 44 (52%) novel MMS mutants have human orthologs (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>).</p><p id="p-34">We compared the human orthologs of our genes causing increased <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> abundance to two additional datasets. The first was a set of human genes identified as having increased H2AX phosphorylation following small interfering RNA knock-down (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Paulsen <em>et al.</em> 2009</a>). H2AX is phosphorylated at residue Ser139 in response to double-strand breaks by DNA damage checkpoint kinases, and serves as an indicator of genome instability in human cells (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Paulsen <em>et al.</em> 2009</a>). We found 11 and 17 of the mutants with increased <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> abundance in UT and MMS had a human homolog with increased H2AX phosphorylation (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>, <a href="http://www.g3journal.org/content/suppl/2015/02/26/g3.115.016576.DC1/TableS11.xlsx">Table S11</a>, and <a href="http://www.g3journal.org/content/suppl/2015/02/26/g3.115.016576.DC1/TableS12.xlsx">Table S12</a>), suggesting their role in genome maintenance is conserved. Of these, eight belong to our set of putative novel mutants. In addition, we asked whether any of the human homologs of yeast mutants with increased <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> were present in the Cancer Gene Census, which is a list of genes with mutations that have been causally linked to oncogenesis (Forbes <em>et al.</em> 2014; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Futreal <em>et al.</em> 2004</a>). In doing so, we identified a total of seven mutants present in the cancer census that have increased <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> abundance in yeast, four of which were associated with our putative novel set.</p></div></div><div class="section discussion" id="sec-11"><h2 class="">Discussion</h2><p id="p-35">We have complemented a previous version of the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> assay (<a id="xref-ref-69-3" class="xref-bibr" href="#ref-69">Zhou and Elledge 1992</a>) by using yeast functional genomics reagents and methodologies. Instead of screening mutants generated via spontaneous mutagenesis, we us the nonessential gene deletion collection and conditional temperature-sensitive gene mutant collection. We introduce an <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> reporter into both of these mutant collections using R-SGA. The resultant assay is an extremely high-throughput method to screen for genome instability: from start to finish it entails only an SGA procedure followed by ~1 hr of fluorescence scanning of whole colonies arrayed on agar plates (for the ~4800 mutants in the yeast nonessential deletion collection). Moreover, the method is compatible with drug treatment, insofar as the drug of interest is soluble in agar plates.</p><p id="p-36">The Zhou and Elledge <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> assay identified nine complementation groups (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004166">CRT1</a></em> to <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005600">CRT9</a></em>) exhibiting increased <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> (<a id="xref-ref-69-4" class="xref-bibr" href="#ref-69">Zhou and Elledge 1992</a>). Of these, the associated gene for all but <em>CRT2</em> is known. These genes are the transcriptional regulators <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004166">CRT1</a>/<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004166">RFX1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000316">SSN6</a></em>, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000680">TUP1</a></em>; the RNR subunits <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000872">RNR1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003563">RNR2</a></em>,and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003412">RNR4</a></em>; and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005600">CDC21</a></em>. Our R-SGA <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> screen robustly identified <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004166">CRT1</a></em>/<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004166">RFX1</a></em>. Alleles of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000316">SSN6</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000680">TUP1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000872">RNR1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003563">RNR2</a></em>, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003412">RNR4</a></em> were not present in the mutant collections screened. An allele of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005600">CDC21</a></em> was screened in our R-SGA assay and did not have increased <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> abundance, although another gene in the <em>de novo</em> pyrimidine synthesis pathway, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003818">CDC8</a></em>, was identified.</p><p id="p-37">Although here we focused on mutants causing increased <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a>, we also identified 138 mutants with decreased <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> (<a href="http://www.g3journal.org/content/suppl/2015/02/26/g3.115.016576.DC1/TableS4.xlsx">Table S4</a> and <a href="http://www.g3journal.org/content/suppl/2015/02/26/g3.115.016576.DC1/TableS6.xlsx">Table S6</a>). Interestingly, in the MMS <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> R-SGA screen, we noted that mutants with decreased <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> abundance were significantly enriched for genes associated with the DDR pathway (data not shown). This result makes sense, as <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> induction in the presence of damage depends on an intact DDR and suggests a further class of genes may be identified through screening <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> abundance in the presence of damage.</p><p id="p-38">A number of genome instability screens have been previously conducted in <em>S. cerevisiae</em>. This existing set of screens provides a valuable resource for comparative analysis. For example, after systematic comparison of our <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> R-SGA dataset with four major, orthogonal genome instability studies, as well as GO terms associated with genome maintenance, we conclude that 45% of the 285 mutants we identified have no previous connection to maintenance of the genome. We attribute this substantial number of novel mutants to <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> providing a different measure of genome instability relative to earlier screens. Existing genome instability screens can be roughly divided into two classes, based on the nature of their read-out: (1) those that directly assay alterations in the genetic material (such as mutation of the canavinine reporter, chromosome loss, loss of the MAT locus, telomere length, etc.) and (2) those that infer the presence of DNA damage through monitoring the formation of “damage foci” by fluorescence microscopy. Distinct from these approaches, the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> R-SGA identifies mutants that cause induction of the DDR.</p><p id="p-39">We sought to assess the extent to which insights in yeast can be transferred to human and, in particular, to our understanding of genome instability in cancer. Approximately 52% of the mutants exhibiting increased <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> abundance have human homologs, compared with 36% of all yeast genes. The enrichment for homologs is partly reflective of the fact that core genome maintenance processes, such as DNA replication, are highly conserved (75% of genes associated with the GO term “DNA replication” have human homologs). However, even in considering only the putative novel set of mutants, which lack known connections to replication or repair, we still note that 51% have human homologs. Furthermore, in about 14% of cases, we are able to provide evidence of conservation of the genome instability phenotype from yeast to human, simply by comparing to an existing genome instability screen. Although neither the UT nor the MMS <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> R-SGA screen was significantly enriched for human genome instability genes, we note that, just as in comparisons between genome instability screens in yeast, differences in read-out between human and yeast screens can produce nonoverlapping results. Other factors likely frustrate such a direct approach to assessing phenotypic conservation, such as extent of inactivation of the gene (knock-down <em>vs.</em> deletion and hypomorph) and cell-type specific effects in human. Lastly, we recovered a handful of genes with human homologs that are part of the Cancer Gene Consensus, demonstrating that yeast genome instability studies can be used to provide disease relevant functional annotations to human cancer genes.</p><p id="p-40">More than half of all genes causally implicated in cancer are oncogenes (<a id="xref-ref-64-3" class="xref-bibr" href="#ref-64">Vogelstein <em>et al.</em> 2013</a>). Indeed, gene duplication or hyperactivity is a frequent occurrence in cancer, yet the extent to which this causes genome instability is unknown. Gene overexpression collections now exist in yeast (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Douglas <em>et al.</em> 2012</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Hu <em>et al.</em> 2007</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Sopko <em>et al.</em> 2006</a>), and can be readily combined with the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001328">Rnr3</a> R-SGA method, providing insights into the relationship between oncogenes and genome instability.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-41">We thank Bryan-Joseph San Luis for help with colony size analysis, Brenda Andrews for strains, and David Gallo, Attila Balint, and Raphael Loll-Krippleber for comments on the manuscript. This work was supported by grants from the Canadian Institutes of Health Research (MOP-79368 to G.W.B., and MOP-102629 to C.B.) and the National Institutes of Health (1R01 HG005853 to C.B.).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-42">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.016576/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.016576/-/DC1</a></p></li><li class="fn" id="fn-3"><p id="p-43"><em>Communicating editor: A. Gasch</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-01-07"><span class="received-label">Received </span>January 7, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-02-24"><span class="accepted-label">Accepted </span>February 24, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Hendry <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.1" data-doi="10.1074/jbc.M306783200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Agarwal</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cohen-Fix</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Two distinct pathways for inhibiting pds1 ubiquitination in response to DNA damage.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">278</span>: <span class="cit-fpage">45027</span>–<span class="cit-lpage">45033</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M306783200%26rft_id%253Dinfo%253Apmid%252F12947083%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzgvNDUvNDUwMjciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzUvOTk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.2" data-doi="10.1038/nrg2268"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aguilera</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gomez-Gonzalez</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genome instability: a mechanistic view of its causes and consequences.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">204</span>–<span class="cit-lpage">217</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DAguilera%26rft.auinit1%253DA.%26rft.volume%253D9%26rft.issue%253D3%26rft.spage%253D204%26rft.epage%253D217%26rft.atitle%253DGenome%2Binstability%253A%2Ba%2Bmechanistic%2Bview%2Bof%2Bits%2Bcauses%2Band%2Bconsequences.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2268%26rft_id%253Dinfo%253Apmid%252F18227811%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2268&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18227811&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F997.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253239500013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.3" data-doi="10.1371/journal.pgen.0030228"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alvaro</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lisby</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rothstein</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genome-wide analysis of Rad52 foci reveals diverse mechanisms impacting recombination.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e228</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DAlvaro%26rft.auinit1%253DD.%26rft.volume%253D3%26rft.issue%253D12%26rft.spage%253De228%26rft.epage%253De228%26rft.atitle%253DGenome-wide%2Banalysis%2Bof%2BRad52%2Bfoci%2Breveals%2Bdiverse%2Bmechanisms%2Bimpacting%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0030228%26rft_id%253Dinfo%253Apmid%252F18085829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0030228&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18085829&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F997.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-book" id="cit-5.5.997.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alver</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jauert</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brosnan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Hehir</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vandersluis</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A whole genome screen for minisatellite stability genes in stationary phase yeast cells.</span> G3 (Bethesda) <span class="cit-vol">3</span>: <span class="cit-fpage">741</span>–<span class="cit-lpage">756</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.5" data-doi="10.1534/genetics.108.089250"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andersen</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">Z. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hetrick</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gottschling</span>  <span class="cit-name-given-names">D. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A genetic screen for increased loss of heterozygosity in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">1179</span>–<span class="cit-lpage">1195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.089250%26rft_id%253Dinfo%253Apmid%252F18562670%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OS8zLzExNzkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzUvOTk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.6" data-doi="10.1073/pnas.0401263101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Askree</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yehuda</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smolikov</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gurevich</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hawk</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A genome-wide screen for <em>Saccharomyces cerevisiae</em> deletion mutants that affect telomere length.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">8658</span>–<span class="cit-lpage">8663</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0401263101%26rft_id%253Dinfo%253Apmid%252F15161972%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAxLzIzLzg2NTgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzUvOTk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.7" data-doi="10.1111/j.1365-2958.2009.06748.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ball</span>  <span class="cit-name-given-names">L. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cobb</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boone</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xiao</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The yeast Shu complex couples error-free post-replication repair to homologous recombination.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">89</span>–<span class="cit-lpage">102</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DBall%26rft.auinit1%253DL.%2BG.%26rft.volume%253D73%26rft.issue%253D1%26rft.spage%253D89%26rft.epage%253D102%26rft.atitle%253DThe%2Byeast%2BShu%2Bcomplex%2Bcouples%2Berror-free%2Bpost-replication%2Brepair%2Bto%2Bhomologous%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2958.2009.06748.x%26rft_id%253Dinfo%253Apmid%252F19496932%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2958.2009.06748.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19496932&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F997.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267369600007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.8" data-doi="10.1016/0027-5107(90)90173-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beranek</span>  <span class="cit-name-given-names">D. T.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Distribution of methyl and ethyl adducts following alkylation with monofunctional alkylating agents.</span> <abbr class="cit-jnl-abbrev">Mutat. Res.</abbr> <span class="cit-vol">231</span>: <span class="cit-fpage">11</span>–<span class="cit-lpage">30</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMutation%2BResearch%26rft.stitle%253DMutation%2BResearch%26rft.aulast%253DBeranek%26rft.auinit1%253DD.%2BT.%26rft.volume%253D231%26rft.issue%253D1%26rft.spage%253D11%26rft.epage%253D30%26rft.atitle%253DDistribution%2Bof%2Bmethyl%2Band%2Bethyl%2Badducts%2Bfollowing%2Balkylation%2Bwith%2Bmonofunctional%2Balkylating%2Bagents.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0027-5107%252890%252990173-2%26rft_id%253Dinfo%253Apmid%252F2195323%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0027-5107(90)90173-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2195323&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F997.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990DP39900003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.9" data-doi="10.1074/jbc.271.26.15837"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carmen</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rundlett</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grunstein</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">HDA1 and HDA3 are components of a yeast histone deacetylase (HDA) complex.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">271</span>: <span class="cit-fpage">15837</span>–<span class="cit-lpage">15844</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DCarmen%26rft.auinit1%253DA.%2BA.%26rft.volume%253D271%26rft.issue%253D26%26rft.spage%253D15837%26rft.epage%253D15844%26rft.atitle%253DHDA1%2Band%2BHDA3%2BAre%2BComponents%2Bof%2Ba%2BYeast%2BHistone%2BDeacetylase%2B%2528HDA%2529%2BComplex%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.271.26.15837%26rft_id%253Dinfo%253Apmid%252F8663039%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzEvMjYvMTU4MzciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzUvOTk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.10" data-doi="10.1016/S0092-8674(03)00075-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chabes</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Georgieva</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Domkin</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rothstein</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Survival of DNA damage in yeast directly depends on increased dNTP levels allowed by relaxed feedback inhibition of ribonucleotide reductase.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">391</span>–<span class="cit-lpage">401</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DChabes%26rft.auinit1%253DA.%26rft.volume%253D112%26rft.issue%253D3%26rft.spage%253D391%26rft.epage%253D401%26rft.atitle%253DSurvival%2Bof%2BDNA%2Bdamage%2Bin%2Byeast%2Bdirectly%2Bdepends%2Bon%2Bincreased%2BdNTP%2Blevels%2Ballowed%2Bby%2Brelaxed%2Bfeedback%2Binhibition%2Bof%2Bribonucleotide%2Breductase.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252803%252900075-8%26rft_id%253Dinfo%253Apmid%252F12581528%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(03)00075-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12581528&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F997.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181252200010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.11" data-doi="10.1073/pnas.262669299"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bellaoui</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boone</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">G. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">A genome-wide screen for methyl methanesulfonate-sensitive mutants reveals genes required for S phase progression in the presence of DNA damage.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">16934</span>–<span class="cit-lpage">16939</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.262669299%26rft_id%253Dinfo%253Apmid%252F12482937%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTkvMjYvMTY5MzQiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzUvOTk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.12" data-doi="10.1534/genetics.112.141051"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vaisica</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ou</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baryshnikova</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome rearrangements caused by depletion of essential DNA replication proteins in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">147</span>–<span class="cit-lpage">160</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.141051%26rft_id%253Dinfo%253Apmid%252F22673806%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkyLzEvMTQ3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS81Lzk5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.13" data-doi="10.1091/mbc.E10-01-0050"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Szakal</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Y. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Branzei</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The Smc5/6 complex and Esc2 influence multiple replication-associated recombination processes in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">2306</span>–<span class="cit-lpage">2314</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E10-01-0050%26rft_id%253Dinfo%253Apmid%252F20444977%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIyMS8xMy8yMzA2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS81Lzk5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.14" data-doi="10.1126/science.275.5305.1468"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chuang</span>  <span class="cit-name-given-names">R.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weaver</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">T.-H.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Requirement of the DEAD-Box protein ded1p for messenger RNA translation.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">275</span>: <span class="cit-fpage">1468</span>–<span class="cit-lpage">1471</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DChuang%26rft.auinit1%253DR.-Y.%26rft.volume%253D275%26rft.issue%253D5305%26rft.spage%253D1468%26rft.epage%253D1471%26rft.atitle%253DRequirement%2Bof%2Bthe%2BDEAD-Box%2BProtein%2BDed1p%2Bfor%2BMessenger%2BRNA%2BTranslation%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.275.5305.1468%26rft_id%253Dinfo%253Apmid%252F9045610%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyNzUvNTMwNS8xNDY4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS81Lzk5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.15" data-doi="10.1038/emboj.2011.485"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davidson</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Katou</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keszthelyi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sing</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xia</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Endogenous DNA replication stress results in expansion of dNTP pools and a mutator phenotype.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">895</span>–<span class="cit-lpage">907</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BJ.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Femboj.2011.485%26rft_id%253Dinfo%253Apmid%252F22234187%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czo4OiIzMS80Lzg5NSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvNS85OTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-other" id="cit-5.5.997.16" data-doi="10.1534/g3.112.003400"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Douglas</span>  <span class="cit-name-given-names">A.C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">A.M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharifpoor</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbic</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Functional analysis with a barcoder yeast gene overexpression system.</span> <span class="cit-source">G3 (Bethesda)</span> <span class="cit-vol">2</span>: <span class="cit-fpage">1279</span>–<span class="cit-lpage">1289</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2526%252358%253B%2BGenes%257CGenomes%257CGenetics%26rft.stitle%253DG3%26rft.aulast%253DDouglas%26rft.auinit1%253DA.%2BC.%26rft.volume%253D2%26rft.issue%253D10%26rft.spage%253D1279%26rft.epage%253D1289%26rft.atitle%253DFunctional%2BAnalysis%2BWith%2Ba%2BBarcoder%2BYeast%2BGene%2BOverexpression%2BSystem%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.112.003400%26rft_id%253Dinfo%253Apmid%252F23050238%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjIvMTAvMTI3OSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvNS85OTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.17" data-doi="10.1002/bies.950150507"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elledge</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allen</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Navas</span>  <span class="cit-name-given-names">T. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">DNA damage and cell cycle regulation of ribonucleotide reductase.</span> <abbr class="cit-jnl-abbrev">BioEssays</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">333</span>–<span class="cit-lpage">339</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioEssays%2B%253A%2B%2Bnews%2Band%2Breviews%2Bin%2Bmolecular%252C%2Bcellular%2Band%2Bdevelopmental%2Bbiology%26rft.stitle%253DBioessays%26rft.aulast%253DElledge%26rft.auinit1%253DS.%2BJ.%26rft.volume%253D15%26rft.issue%253D5%26rft.spage%253D333%26rft.epage%253D339%26rft.atitle%253DDNA%2Bdamage%2Band%2Bcell%2Bcycle%2Bregulation%2Bof%2Bribonucleotide%2Breductase.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbies.950150507%26rft_id%253Dinfo%253Apmid%252F8343143%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/bies.950150507&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8343143&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F997.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993LF19300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.18" data-doi="10.1038/nature03442"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falck</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coates</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jackson</span>  <span class="cit-name-given-names">S. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Conserved modes of recruitment of ATM, ATR and DNA-PKcs to sites of DNA damage.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">434</span>: <span class="cit-fpage">605</span>–<span class="cit-lpage">611</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DFalck%26rft.auinit1%253DJ.%26rft.volume%253D434%26rft.issue%253D7033%26rft.spage%253D605%26rft.epage%253D611%26rft.atitle%253DConserved%2Bmodes%2Bof%2Brecruitment%2Bof%2BATM%252C%2BATR%2Band%2BDNA-PKcs%2Bto%2Bsites%2Bof%2BDNA%2Bdamage.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature03442%26rft_id%253Dinfo%253Apmid%252F15758953%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature03442&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15758953&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F997.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228011000036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-5.5.997.19" data-doi="10.1093/nar/gku1075"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Forbes</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beare</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gunasekaran</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leung</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bindal</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">COSMIC: exploring the world’s knowledge of somatic mutations in human cancer.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">43</span><strong>(</strong><span class="cit-comment">Database Issue</span><strong>):</strong> <span class="cit-fpage">D805</span>–<span class="cit-lpage">D811</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku1075%26rft_id%253Dinfo%253Apmid%252F25355519%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gku1075&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25355519&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F997.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.20" data-doi="10.1038/nrc1299"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Futreal</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coin</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marshall</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Down</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hubbard</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A census of human cancer genes.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Cancer</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">177</span>–<span class="cit-lpage">183</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BCancer%26rft.stitle%253DNat%2BRev%2BCancer%26rft.aulast%253DFutreal%26rft.auinit1%253DP.%2BA.%26rft.volume%253D4%26rft.issue%253D3%26rft.spage%253D177%26rft.epage%253D183%26rft.atitle%253DA%2Bcensus%2Bof%2Bhuman%2Bcancer%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrc1299%26rft_id%253Dinfo%253Apmid%252F14993899%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrc1299&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14993899&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F997.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000189345000010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.21" data-doi="10.1038/nature00935"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Giaever</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chu</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ni</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Connelly</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riles</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Functional profiling of the <em>Saccharomyces cerevisiae</em> genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">418</span>: <span class="cit-fpage">387</span>–<span class="cit-lpage">391</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGiaever%26rft.auinit1%253DG.%26rft.volume%253D418%26rft.issue%253D6896%26rft.spage%253D387%26rft.epage%253D391%26rft.atitle%253DFunctional%2Bprofiling%2Bof%2Bthe%2BSaccharomyces%2Bcerevisiae%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature00935%26rft_id%253Dinfo%253Apmid%252F12140549%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature00935&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12140549&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F997.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177009700029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.22" data-doi="10.1016/j.cell.2012.02.015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Glickman</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sawyers</span>  <span class="cit-name-given-names">C. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Converting cancer therapies into cures: lessons from infectious diseases.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">148</span>: <span class="cit-fpage">1089</span>–<span class="cit-lpage">1098</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DGlickman%26rft.auinit1%253DM.%2BS.%26rft.volume%253D148%26rft.issue%253D6%26rft.spage%253D1089%26rft.epage%253D1098%26rft.atitle%253DConverting%2Bcancer%2Btherapies%2Binto%2Bcures%253A%2Blessons%2Bfrom%2Binfectious%2Bdiseases.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2012.02.015%26rft_id%253Dinfo%253Apmid%252F22424221%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2012.02.015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22424221&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F997.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000301889500008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.23" data-doi="10.1016/S0092-8674(00)00134-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goldmark</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fazzio</span>  <span class="cit-name-given-names">T. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Estep</span>  <span class="cit-name-given-names">P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Church</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsukiyama</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The Isw2 chromatin remodeling complex represses early meiotic genes upon recruitment by Ume6p.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">423</span>–<span class="cit-lpage">433</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DGoldmark%26rft.auinit1%253DJ.%2BP.%26rft.volume%253D103%26rft.issue%253D3%26rft.spage%253D423%26rft.epage%253D433%26rft.atitle%253DThe%2BIsw2%2Bchromatin%2Bremodeling%2Bcomplex%2Brepresses%2Bearly%2Bmeiotic%2Bgenes%2Bupon%2Brecruitment%2Bby%2BUme6p.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252800%252900134-3%26rft_id%253Dinfo%253Apmid%252F11081629%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(00)00134-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11081629&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F997.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000090144100008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.24" data-doi="10.1146/annurev.genet.40.051206.105231"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harrison</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haber</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Surviving the breakup: the DNA damage checkpoint.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">209</span>–<span class="cit-lpage">235</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DHopwood%26rft.auinit1%253DD.%2BA.%26rft.volume%253D40%26rft.spage%253D209%26rft.epage%253D235%26rft.atitle%253DSurviving%2Bthe%2Bbreakup%253A%2Bthe%2BDNA%2Bdamage%2Bcheckpoint.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.40.051206.105231%26rft_id%253Dinfo%253Apmid%252F16805667%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genet.40.051206.105231&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16805667&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F997.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243038200009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hartwell</span>  <span class="cit-name-given-names">L. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1973</span> <span class="cit-article-title">Three additional genes required for deoxyribonucleic acid synthesis in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">115</span>: <span class="cit-fpage">966</span>–<span class="cit-lpage">974</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DHartwell%26rft.auinit1%253DL.%2BH.%26rft.volume%253D115%26rft.issue%253D3%26rft.spage%253D966%26rft.epage%253D974%26rft.atitle%253DThree%2BAdditional%2BGenes%2BRequired%2Bfor%2BDeoxyribonucleic%2BAcid%2BSynthesis%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Apmid%252F4580573%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6OToiMTE1LzMvOTY2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS81Lzk5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.26" data-doi="10.1038/nature00991"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoege</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pfander</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moldovan</span>  <span class="cit-name-given-names">G. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pyrowolakis</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jentsch</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">RAD6-dependent DNA repair is linked to modification of PCNA by ubiquitin and SUMO.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">419</span>: <span class="cit-fpage">135</span>–<span class="cit-lpage">141</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHoege%26rft.auinit1%253DC.%26rft.volume%253D419%26rft.issue%253D6903%26rft.spage%253D135%26rft.epage%253D141%26rft.atitle%253DRAD6-dependent%2BDNA%2Brepair%2Bis%2Blinked%2Bto%2Bmodification%2Bof%2BPCNA%2Bby%2Bubiquitin%2Band%2BSUMO.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature00991%26rft_id%253Dinfo%253Apmid%252F12226657%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature00991&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12226657&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F997.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177931200033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.27" data-doi="10.1101/gr.6037607"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rolfs</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhullar</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murthy</span>  <span class="cit-name-given-names">T. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Approaching a complete repository of sequence-verified protein-encoding clones for <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">536</span>–<span class="cit-lpage">543</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.6037607%26rft_id%253Dinfo%253Apmid%252F17322287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE3LzQvNTM2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS81Lzk5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.28" data-doi="10.1016/S0092-8674(00)81601-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elledge</span>  <span class="cit-name-given-names">S. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">The DNA replication and damage checkpoint pathways induce transcription by inhibition of the Crt1 repressor.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">595</span>–<span class="cit-lpage">605</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DHuang%26rft.auinit1%253DM.%26rft.volume%253D94%26rft.issue%253D5%26rft.spage%253D595%26rft.epage%253D605%26rft.atitle%253DThe%2BDNA%2Breplication%2Band%2Bdamage%2Bcheckpoint%2Bpathways%2Binduce%2Btranscription%2Bby%2Binhibition%2Bof%2Bthe%2BCrt1%2Brepressor.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252800%252981601-3%26rft_id%253Dinfo%253Apmid%252F9741624%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(00)81601-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9741624&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F997.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000075851900007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.29" data-doi="10.1073/pnas.2035018100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rio</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nicolas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kolodner</span>  <span class="cit-name-given-names">R. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">A genomewide screen in <em>Saccharomyces cerevisiae</em> for genes that suppress the accumulation of mutations.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">11529</span>–<span class="cit-lpage">11534</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.2035018100%26rft_id%253Dinfo%253Apmid%252F12972632%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAwLzIwLzExNTI5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS81Lzk5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.30" data-doi="10.1074/jbc.274.25.17677"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Iost</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dreyfus</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Linder</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Ded1p, a DEAD-box protein required for translation initiation in <em>Saccharomyces cerevisiae</em>, is an RNA helicase.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">274</span>: <span class="cit-fpage">17677</span>–<span class="cit-lpage">17683</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DIost%26rft.auinit1%253DI.%26rft.volume%253D274%26rft.issue%253D25%26rft.spage%253D17677%26rft.epage%253D17683%26rft.atitle%253DDed1p%252C%2Ba%2BDEAD-box%2BProtein%2BRequired%2Bfor%2BTranslation%2BInitiation%2Bin%2BSaccharomyces%2Bcerevisiae%252C%2BIs%2Ban%2BRNA%2BHelicase%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.274.25.17677%26rft_id%253Dinfo%253Apmid%252F10364207%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzQvMjUvMTc2NzciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzUvOTk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.31" data-doi="10.1093/toxsci/kfg158"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xiao</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Compromised DNA repair enhances sensitivity of the yeast RNR3-lacZ genotoxicity testing system.</span> <abbr class="cit-jnl-abbrev">Toxicol. Sci.</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">82</span>–<span class="cit-lpage">88</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DToxicol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Ftoxsci%252Fkfg158%26rft_id%253Dinfo%253Apmid%252F12805645%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/toxsci/kfg158&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12805645&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F997.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000184931600011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jong</span>  <span class="cit-name-given-names">A. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuo</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">The CDC8 gene of yeast encodes thymidylate kinase.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">259</span>: <span class="cit-fpage">11052</span>–<span class="cit-lpage">11059</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DJong%26rft.auinit1%253DA.%2BY.%26rft.volume%253D259%26rft.issue%253D17%26rft.spage%253D11052%26rft.epage%253D11059%26rft.atitle%253DThe%2BCDC8%2Bgene%2Bof%2Byeast%2Bencodes%2Bthymidylate%2Bkinase%26rft_id%253Dinfo%253Apmid%252F6088527%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNTkvMTcvMTEwNTIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzUvOTk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.33" data-doi="10.1016/j.ymeth.2009.02.015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kainth</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sassi</span>  <span class="cit-name-given-names">H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pena-Castillo</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chua</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hughes</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Comprehensive genetic analysis of transcription factor pathways using a dual reporter gene system in budding yeast.</span> <abbr class="cit-jnl-abbrev">Methods</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">258</span>–<span class="cit-lpage">264</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2B%2528San%2BDiego%252C%2BCalif.%2529%26rft.stitle%253DMethods%26rft.aulast%253DKainth%26rft.auinit1%253DP.%26rft.volume%253D48%26rft.issue%253D3%26rft.spage%253D258%26rft.epage%253D264%26rft.atitle%253DComprehensive%2Bgenetic%2Banalysis%2Bof%2Btranscription%2Bfactor%2Bpathways%2Busing%2Ba%2Bdual%2Breporter%2Bgene%2Bsystem%2Bin%2Bbudding%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ymeth.2009.02.015%26rft_id%253Dinfo%253Apmid%252F19269327%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ymeth.2009.02.015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19269327&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F997.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268359300007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.34" data-doi="10.1128/EC.00045-06"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Klinkenberg</span>  <span class="cit-name-given-names">L. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Webb</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zitomer</span>  <span class="cit-name-given-names">R. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Synergy among differentially regulated repressors of the ribonucleotide diphosphate reductase genes of <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1007</span>–<span class="cit-lpage">1017</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryotic%2BCell%26rft.stitle%253DEukaryot%2BCell%26rft.aulast%253DKlinkenberg%26rft.auinit1%253DL.%2BG.%26rft.volume%253D5%26rft.issue%253D7%26rft.spage%253D1007%26rft.epage%253D1017%26rft.atitle%253DSynergy%2Bamong%2BDifferentially%2BRegulated%2BRepressors%2Bof%2Bthe%2BRibonucleotide%2BDiphosphate%2BReductase%2BGenes%2Bof%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.00045-06%26rft_id%253Dinfo%253Apmid%252F16835445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo4OiI1LzcvMTAwNyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvNS85OTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.35" data-doi="10.1016/0027-5107(89)90028-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lawley</span>  <span class="cit-name-given-names">P. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Mutagens as carcinogens: development of current concepts.</span> <abbr class="cit-jnl-abbrev">Mutat. Res.</abbr> <span class="cit-vol">213</span>: <span class="cit-fpage">3</span>–<span class="cit-lpage">25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMutation%2BResearch%26rft.stitle%253DMutation%2BResearch%26rft.aulast%253DLawley%26rft.auinit1%253DP.%2BD.%26rft.volume%253D213%26rft.issue%253D1%26rft.spage%253D3%26rft.epage%253D25%26rft.atitle%253DMutagens%2Bas%2Bcarcinogens%253A%2Bdevelopment%2Bof%2Bcurrent%2Bconcepts.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0027-5107%252889%252990028-6%26rft_id%253Dinfo%253Apmid%252F2664491%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0027-5107(89)90028-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2664491&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F997.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1989AF65500002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.36" data-doi="10.1074/jbc.M104220200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reese</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Ssn6-Tup1 regulates RNR3 by positioning nucleosomes and affecting the chromatin structure at the upstream repression sequence.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">276</span>: <span class="cit-fpage">33788</span>–<span class="cit-lpage">33797</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M104220200%26rft_id%253Dinfo%253Apmid%252F11448965%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzYvMzYvMzM3ODgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzUvOTk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.37" data-doi="10.1038/nature09377"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lopez-Mosqueda</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maas</span>  <span class="cit-name-given-names">N. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jonsson</span>  <span class="cit-name-given-names">Z. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Defazio-Eli</span>  <span class="cit-name-given-names">L. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wohlschlegel</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Damage-induced phosphorylation of Sld3 is important to block late origin firing.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">467</span>: <span class="cit-fpage">479</span>–<span class="cit-lpage">483</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLopez-Mosqueda%26rft.auinit1%253DJ.%26rft.volume%253D467%26rft.issue%253D7314%26rft.spage%253D479%26rft.epage%253D483%26rft.atitle%253DDamage-induced%2Bphosphorylation%2Bof%2BSld3%2Bis%2Bimportant%2Bto%2Bblock%2Blate%2Borigin%2Bfiring.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09377%26rft_id%253Dinfo%253Apmid%252F20865002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09377&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20865002&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F997.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282090200047&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.38" data-doi="10.1016/j.molcel.2006.11.027"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Majka</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Niedziela-Majka</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgers</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The checkpoint clamp activates Mec1 kinase during initiation of the DNA damage checkpoint.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">891</span>–<span class="cit-lpage">901</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DMajka%26rft.auinit1%253DJ.%26rft.volume%253D24%26rft.issue%253D6%26rft.spage%253D891%26rft.epage%253D901%26rft.atitle%253DThe%2Bcheckpoint%2Bclamp%2Bactivates%2BMec1%2Bkinase%2Bduring%2Binitiation%2Bof%2Bthe%2BDNA%2Bdamage%2Bcheckpoint.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2006.11.027%26rft_id%253Dinfo%253Apmid%252F17189191%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2006.11.027&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17189191&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F997.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243253200010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.39" data-doi="10.1128/MCB.24.8.3277-3285.2004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Naiki</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wakayama</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakada</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matsumoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sugimoto</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Association of Rad9 with double-strand breaks through a Mec1-dependent mechanism.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">3277</span>–<span class="cit-lpage">3285</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DNaiki%26rft.auinit1%253DT.%26rft.volume%253D24%26rft.issue%253D8%26rft.spage%253D3277%26rft.epage%253D3285%26rft.atitle%253DAssociation%2Bof%2BRad9%2Bwith%2BDouble-Strand%2BBreaks%2Bthrough%2Ba%2BMec1-Dependent%2BMechanism%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.24.8.3277-3285.2004%26rft_id%253Dinfo%253Apmid%252F15060150%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjI0LzgvMzI3NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvNS85OTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.40" data-doi="10.1101/gad.1099003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nakada</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matsumoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sugimoto</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">ATM-related Tel1 associates with double-strand breaks through an Xrs2-dependent mechanism.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1957</span>–<span class="cit-lpage">1962</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DNakada%26rft.auinit1%253DD.%26rft.volume%253D17%26rft.issue%253D16%26rft.spage%253D1957%26rft.epage%253D1962%26rft.atitle%253DATM-related%2BTel1%2Bassociates%2Bwith%2Bdouble-strand%2Bbreaks%2Bthrough%2Ban%2BXrs2-dependent%2Bmechanism%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1099003%26rft_id%253Dinfo%253Apmid%252F12923051%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjE3LzE2LzE5NTciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzUvOTk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.41" data-doi="10.1016/j.molcel.2009.10.014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Navadgi-Patil</span>  <span class="cit-name-given-names">V. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgers</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The unstructured C-terminal tail of the 9–1−1 clamp subunit Ddc1 activates Mec1/ATR via two distinct mechanisms.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">743</span>–<span class="cit-lpage">753</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DNavadgi-Patil%26rft.auinit1%253DV.%2BM.%26rft.volume%253D36%26rft.issue%253D5%26rft.spage%253D743%26rft.epage%253D753%26rft.atitle%253DThe%2Bunstructured%2BC-terminal%2Btail%2Bof%2Bthe%2B9-1-1%2Bclamp%2Bsubunit%2BDdc1%2Bactivates%2BMec1%252FATR%2Bvia%2Btwo%2Bdistinct%2Bmechanisms.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2009.10.014%26rft_id%253Dinfo%253Apmid%252F20005839%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2009.10.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20005839&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F997.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000272965400009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.42" data-doi="10.1038/nrm2858"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Negrini</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gorgoulis</span>  <span class="cit-name-given-names">V. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Halazonetis</span>  <span class="cit-name-given-names">T. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genomic instability—an evolving hallmark of cancer.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Mol. Cell Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">220</span>–<span class="cit-lpage">228</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMolecular%2Bcell%2Bbiology%26rft.stitle%253DNat%2BRev%2BMol%2BCell%2BBiol%26rft.aulast%253DNegrini%26rft.auinit1%253DS.%26rft.volume%253D11%26rft.issue%253D3%26rft.spage%253D220%26rft.epage%253D228%26rft.atitle%253DGenomic%2Binstability--an%2Bevolving%2Bhallmark%2Bof%2Bcancer.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrm2858%26rft_id%253Dinfo%253Apmid%252F20177397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrm2858&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20177397&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F997.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274784300016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.43" data-doi="10.1016/j.dnarep.2007.02.019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Osman</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitby</span>  <span class="cit-name-given-names">M. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Exploring the roles of Mus81-Eme1/Mms4 at perturbed replication forks.</span> <abbr class="cit-jnl-abbrev">DNA Repair (Amst.)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1004</span>–<span class="cit-lpage">1017</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2Brepair%26rft.stitle%253DDNA%2BRepair%2B%2528Amst%2529%26rft.aulast%253DOsman%26rft.auinit1%253DF.%26rft.volume%253D6%26rft.issue%253D7%26rft.spage%253D1004%26rft.epage%253D1017%26rft.atitle%253DExploring%2Bthe%2Broles%2Bof%2BMus81-Eme1%252FMms4%2Bat%2Bperturbed%2Breplication%2Bforks.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.dnarep.2007.02.019%26rft_id%253Dinfo%253Apmid%252F17409028%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.dnarep.2007.02.019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17409028&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F997.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paulsen</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soni</span>  <span class="cit-name-given-names">D. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wollman</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hahn</span>  <span class="cit-name-given-names">A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yee</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A genome-wide siRNA screen reveals diverse cellular processes and pathways that mediate genome stability.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">228</span>–<span class="cit-lpage">239</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%26rft.volume%253D3%26rft.spage%253D228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.45" data-doi="10.1016/j.dnarep.2009.04.021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Putnam</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jaehnig</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kolodner</span>  <span class="cit-name-given-names">R. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Perspectives on the DNA damage and replication checkpoint responses in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">DNA Repair (Amst.)</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">974</span>–<span class="cit-lpage">982</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2BRepair%2B%2528Amst.%2529%26rft.volume%253D8%26rft.spage%253D974%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.dnarep.2009.04.021%26rft_id%253Dinfo%253Apmid%252F19477695%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.dnarep.2009.04.021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19477695&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F997.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.46" data-doi="10.1016/S1097-2765(02)00507-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rouse</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jackson</span>  <span class="cit-name-given-names">S. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Lcd1p recruits Mec1p to DNA lesions in vitro and in vivo.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">857</span>–<span class="cit-lpage">869</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DRouse%26rft.auinit1%253DJ.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D857%26rft.epage%253D869%26rft.atitle%253DLcd1p%2Brecruits%2BMec1p%2Bto%2BDNA%2Blesions%2Bin%2Bvitro%2Band%2Bin%2Bvivo.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1097-2765%252802%252900507-5%26rft_id%253Dinfo%253Apmid%252F11983176%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1097-2765(02)00507-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11983176&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F997.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000175244400018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.47" data-doi="10.1126/science.286.5442.1166"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sanchez</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bachant</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Control of the DNA damage checkpoint by chk1 and rad53 protein kinases through distinct mechanisms.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">286</span>: <span class="cit-fpage">1166</span>–<span class="cit-lpage">1171</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSanchez%26rft.auinit1%253DY.%26rft.volume%253D286%26rft.issue%253D5442%26rft.spage%253D1166%26rft.epage%253D1171%26rft.atitle%253DControl%2Bof%2Bthe%2BDNA%2BDamage%2BCheckpoint%2Bby%2BChk1%2Band%2BRad53%2BProtein%2BKinases%2BThrough%2BDistinct%2BMechanisms%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.286.5442.1166%26rft_id%253Dinfo%253Apmid%252F10550056%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyODYvNTQ0Mi8xMTY2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS81Lzk5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.48" data-doi="10.4103/2319-4170.110398"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sanvisens</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Llanos</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Puig</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Function and regulation of yeast ribonucleotide reductase: cell cycle, genotoxic stress, and iron bioavailability.</span> <abbr class="cit-jnl-abbrev">Biom. J.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">51</span>–<span class="cit-lpage">58</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiom.%2BJ.%26rft.volume%253D36%26rft.spage%253D51%26rft_id%253Dinfo%253Adoi%252F10.4103%252F2319-4170.110398%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4103/2319-4170.110398&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scholes</span>  <span class="cit-name-given-names">D. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banerjee</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Curcio</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Multiple regulators of Ty1 transposition in <em>Saccharomyces cerevisiae</em> have conserved roles in genome maintenance.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">159</span>: <span class="cit-fpage">1449</span>–<span class="cit-lpage">1465</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DScholes%26rft.auinit1%253DD.%2BT.%26rft.volume%253D159%26rft.issue%253D4%26rft.spage%253D1449%26rft.epage%253D1465%26rft.atitle%253DMultiple%2BRegulators%2Bof%2BTy1%2BTransposition%2Bin%2BSaccharomyces%2Bcerevisiae%2BHave%2BConserved%2BRoles%2Bin%2BGenome%2BMaintenance%26rft_id%253Dinfo%253Apmid%252F11779788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1OS80LzE0NDkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzUvOTk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.50" data-doi="10.1016/S1097-2765(02)00532-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schwartz</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duong</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morrow</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pradhan</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Rad9 Phosphorylation sites couple Rad53 to the <em>Saccharomyces cerevisiae</em> DNA damage checkpoint.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1055</span>–<span class="cit-lpage">1065</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DSchwartz%26rft.auinit1%253DM.%2BF.%26rft.volume%253D9%26rft.issue%253D5%26rft.spage%253D1055%26rft.epage%253D1065%26rft.atitle%253DRad9%2Bphosphorylation%2Bsites%2Bcouple%2BRad53%2Bto%2Bthe%2BSaccharomyces%2Bcerevisiae%2BDNA%2Bdamage%2Bcheckpoint.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1097-2765%252802%252900532-4%26rft_id%253Dinfo%253Apmid%252F12049741%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1097-2765(02)00532-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12049741&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F997.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000175967100016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.51" data-doi="10.1073/pnas.81.18.5821"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sclafani</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fangman</span>  <span class="cit-name-given-names">W. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Yeast gene CDC8 encodes thymidylate kinase and is complemented by herpes thymidine kinase gene TK.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">5821</span>–<span class="cit-lpage">5825</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DSclafani%26rft.auinit1%253DR.%2BA.%26rft.volume%253D81%26rft.issue%253D18%26rft.spage%253D5821%26rft.epage%253D5825%26rft.atitle%253DYeast%2Bgene%2BCDC8%2Bencodes%2Bthymidylate%2Bkinase%2Band%2Bis%2Bcomplemented%2Bby%2Bherpes%2Bthymidine%2Bkinase%2Bgene%2BTK.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.81.18.5821%26rft_id%253Dinfo%253Apmid%252F6091111%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODEvMTgvNTgyMSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvNS85OTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.52" data-doi="10.1128/MCB.02311-06"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sharma</span>  <span class="cit-name-given-names">V. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tomar</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dempsey</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reese</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Histone deacetylases RPD3 and HOS2 regulate the transcriptional activation of DNA damage-inducible genes.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">3199</span>–<span class="cit-lpage">3210</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.02311-06%26rft_id%253Dinfo%253Apmid%252F17296735%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjI3LzgvMzE5OSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvNS85OTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.53" data-doi="10.1073/pnas.0403093101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hwang</span>  <span class="cit-name-given-names">J. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banerjee</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Majeed</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gupta</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Mutator genes for suppression of gross chromosomal rearrangements identified by a genome-wide screening in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">9039</span>–<span class="cit-lpage">9044</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0403093101%26rft_id%253Dinfo%253Apmid%252F15184655%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAxLzI0LzkwMzkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzUvOTk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sonnhammer</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ostlund</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">InParanoid 8: orthology analysis between 273 proteomes, mostly eukaryotic.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">43</span><strong>(</strong><span class="cit-comment">Database Issue</span><strong>):</strong> <span class="cit-fpage">D234</span>–<span class="cit-lpage">D239</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D43%26rft.spage%253DD234%26rft_id%253Dinfo%253Apmid%252F25429972%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=25429972&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F997.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.55" data-doi="10.1016/j.molcel.2005.12.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sopko</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Preston</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chua</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Papp</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Mapping pathways and phenotypes by systematic gene overexpression.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">319</span>–<span class="cit-lpage">330</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DSopko%26rft.auinit1%253DR.%26rft.volume%253D21%26rft.issue%253D3%26rft.spage%253D319%26rft.epage%253D330%26rft.atitle%253DMapping%2Bpathways%2Band%2Bphenotypes%2Bby%2Bsystematic%2Bgene%2Boverexpression.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2005.12.011%26rft_id%253Dinfo%253Apmid%252F16455487%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2005.12.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16455487&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F997.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000235436100003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.56" data-doi="10.1371/journal.pgen.1002057"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stirling</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bloom</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Solanki-Patil</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sipahimalani</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The complete spectrum of yeast chromosome instability genes identifies candidate CIN cancer genes and functional roles for ASTRA complex components.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002057</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DStirling%26rft.auinit1%253DP.%2BC.%26rft.volume%253D7%26rft.issue%253D4%26rft.spage%253De1002057%26rft.epage%253De1002057%26rft.atitle%253DThe%2Bcomplete%2Bspectrum%2Bof%2Byeast%2Bchromosome%2Binstability%2Bgenes%2Bidentifies%2Bcandidate%2BCIN%2Bcancer%2Bgenes%2Band%2Bfunctional%2Broles%2Bfor%2BASTRA%2Bcomplex%2Bcomponents.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002057%26rft_id%253Dinfo%253Apmid%252F21552543%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002057&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21552543&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F997.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.57" data-doi="10.1038/nrm1276"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Storchova</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pellman</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">From polyploidy to aneuploidy, genome instability and cancer.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Mol. Cell Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">45</span>–<span class="cit-lpage">54</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMolecular%2Bcell%2Bbiology%26rft.stitle%253DNat%2BRev%2BMol%2BCell%2BBiol%26rft.aulast%253DStorchova%26rft.auinit1%253DZ.%26rft.volume%253D5%26rft.issue%253D1%26rft.spage%253D45%26rft.epage%253D54%26rft.atitle%253DFrom%2Bpolyploidy%2Bto%2Baneuploidy%252C%2Bgenome%2Binstability%2Band%2Bcancer.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrm1276%26rft_id%253Dinfo%253Apmid%252F14708009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrm1276&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14708009&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F997.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000187993500014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.58" data-doi="10.1126/science.281.5374.272"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hsiao</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fay</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stern</span>  <span class="cit-name-given-names">D. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Rad53 FHA domain associated with phosphorylated Rad9 in the DNA damage checkpoint.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">281</span>: <span class="cit-fpage">272</span>–<span class="cit-lpage">274</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSun%26rft.auinit1%253DZ.%26rft.volume%253D281%26rft.issue%253D5374%26rft.spage%253D272%26rft.epage%253D274%26rft.atitle%253DRad53%2BFHA%2BDomain%2BAssociated%2Bwith%2BPhosphorylated%2BRad9%2Bin%2Bthe%2BDNA%2BDamage%2BCheckpoint%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.281.5374.272%26rft_id%253Dinfo%253Apmid%252F9657725%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyODEvNTM3NC8yNzIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzUvOTk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.59" data-doi="10.1016/j.cub.2005.06.063"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sweeney</span>  <span class="cit-name-given-names">F. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shabanowitz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hunt</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title"><em>Saccharomyces cerevisiae</em> Rad9 acts as a Mec1 adaptor to allow Rad53 activation.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1364</span>–<span class="cit-lpage">1375</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DSweeney%26rft.auinit1%253DF.%2BD.%26rft.volume%253D15%26rft.issue%253D15%26rft.spage%253D1364%26rft.epage%253D1375%26rft.atitle%253DSaccharomyces%2Bcerevisiae%2BRad9%2Bacts%2Bas%2Ba%2BMec1%2Badaptor%2Bto%2Ballow%2BRad53%2Bactivation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2005.06.063%26rft_id%253Dinfo%253Apmid%252F16085488%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2005.06.063&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16085488&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F997.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231230200021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.60" data-doi="10.1371/journal.pgen.1000697"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siu</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">D. Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Loss of yeast peroxiredoxin Tsa1p induces genome instability through activation of the DNA damage checkpoint and elevation of dNTP levels.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000697</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DTang%26rft.auinit1%253DH.%2BM.%26rft.volume%253D5%26rft.issue%253D10%26rft.spage%253De1000697%26rft.epage%253De1000697%26rft.atitle%253DLoss%2Bof%2Byeast%2Bperoxiredoxin%2BTsa1p%2Binduces%2Bgenome%2Binstability%2Bthrough%2Bactivation%2Bof%2Bthe%2BDNA%2Bdamage%2Bcheckpoint%2Band%2Belevation%2Bof%2BdNTP%2Blevels.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000697%26rft_id%253Dinfo%253Apmid%252F19851444%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000697&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19851444&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F997.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.61" data-doi="10.1038/ncb2549"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tkach</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yimit</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">A. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riffle</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costanzo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Dissecting DNA damage response pathways by analysing protein localization and abundance changes during DNA replication stress.</span> <abbr class="cit-jnl-abbrev">Nat. Cell Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">966</span>–<span class="cit-lpage">976</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BCell%2BBiology%26rft.stitle%253DNature%2BCell%2BBiology%26rft.aulast%253DTkach%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D14%26rft.issue%253D9%26rft.spage%253D966%26rft.epage%253D976%26rft.atitle%253DDissecting%2BDNA%2Bdamage%2Bresponse%2Bpathways%2Bby%2Banalysing%2Bprotein%2Blocalization%2Band%2Babundance%2Bchanges%2Bduring%2BDNA%2Breplication%2Bstress.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncb2549%26rft_id%253Dinfo%253Apmid%252F22842922%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncb2549&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22842922&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F997.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000308668600013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.62" data-doi="10.1385/1-59259-958-3:171"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tong</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boone</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Synthetic genetic array analysis in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">313</span>: <span class="cit-fpage">171</span>–<span class="cit-lpage">192</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DTong%26rft.auinit1%253DA.%2BH.%26rft.volume%253D313%26rft.spage%253D171%26rft.epage%253D192%26rft.atitle%253DSynthetic%2Bgenetic%2Barray%2Banalysis%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1385%252F1-59259-958-3%253A171%26rft_id%253Dinfo%253Apmid%252F16118434%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1385/1-59259-958-3:171&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16118434&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F997.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.63" data-doi="10.1371/journal.pgen.1002061"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsaponina</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barsoum</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Astrom</span>  <span class="cit-name-given-names">S. U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chabes</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Ixr1 is required for the expression of the ribonucleotide reductase Rnr1 and maintenance of dNTP pools.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002061</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DTsaponina%26rft.auinit1%253DO.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253De1002061%26rft.epage%253De1002061%26rft.atitle%253DIxr1%2Bis%2Brequired%2Bfor%2Bthe%2Bexpression%2Bof%2Bthe%2Bribonucleotide%2Breductase%2BRnr1%2Band%2Bmaintenance%2Bof%2BdNTP%2Bpools.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002061%26rft_id%253Dinfo%253Apmid%252F21573136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002061&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573136&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F997.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.64" data-doi="10.1126/science.1235122"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vogelstein</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Papadopoulos</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Velculescu</span>  <span class="cit-name-given-names">V. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Diaz</span>  <span class="cit-name-given-names">L. A.</span>  <span class="cit-name-suffix">Jr</span>.</span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Cancer genome landscapes.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">339</span>: <span class="cit-fpage">1546</span>–<span class="cit-lpage">1558</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DVogelstein%26rft.auinit1%253DB.%26rft.volume%253D339%26rft.issue%253D6127%26rft.spage%253D1546%26rft.epage%253D1558%26rft.atitle%253DCancer%2BGenome%2BLandscapes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1235122%26rft_id%253Dinfo%253Apmid%252F23539594%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzkvNjEyNy8xNTQ2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS81Lzk5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.65" data-doi="10.1007/s002940050157"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xiao</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chow</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rathgeber</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">The repair of DNA methylation damage in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Curr. Genet.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">461</span>–<span class="cit-lpage">468</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bgenetics%26rft.stitle%253DCurr%2BGenet%26rft.aulast%253DXiao%26rft.auinit1%253DW.%26rft.volume%253D30%26rft.issue%253D6%26rft.spage%253D461%26rft.epage%253D468%26rft.atitle%253DThe%2Brepair%2Bof%2BDNA%2Bmethylation%2Bdamage%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs002940050157%26rft_id%253Dinfo%253Apmid%252F8939806%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s002940050157&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8939806&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F997.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996WH82700001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.66" data-doi="10.1091/mbc.E06-04-0368"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pena Castillo</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mnaimneh</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hughes</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">G. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A survey of essential gene function in the yeast cell division cycle.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">4736</span>–<span class="cit-lpage">4747</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E06-04-0368%26rft_id%253Dinfo%253Apmid%252F16943325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIxNy8xMS80NzM2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS81Lzk5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.67" data-doi="10.1038/nature09373"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zegerman</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Diffley</span>  <span class="cit-name-given-names">J. F. X.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Checkpoint-dependent inhibition of DNA replication initiation by Sld3 and Dbf4 phosphorylation.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">467</span>: <span class="cit-fpage">474</span>–<span class="cit-lpage">478</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DZegerman%26rft.auinit1%253DP.%26rft.volume%253D467%26rft.issue%253D7314%26rft.spage%253D474%26rft.epage%253D478%26rft.atitle%253DCheckpoint-dependent%2Binhibition%2Bof%2BDNA%2Breplication%2Binitiation%2Bby%2BSld3%2Band%2BDbf4%2Bphosphorylation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09373%26rft_id%253Dinfo%253Apmid%252F20835227%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09373&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20835227&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F997.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282090200046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.68" data-doi="10.1038/sj.emboj.7600227"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reese</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Ssn6-Tup1 requires the ISW2 complex to position nucleosomes in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">2246</span>–<span class="cit-lpage">2257</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DZhang%26rft.auinit1%253DZ.%26rft.volume%253D23%26rft.issue%253D11%26rft.spage%253D2246%26rft.epage%253D2257%26rft.atitle%253DSsn6-Tup1%2Brequires%2Bthe%2BISW2%2Bcomplex%2Bto%2Bposition%2Bnucleosomes%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.emboj.7600227%26rft_id%253Dinfo%253Apmid%252F15116071%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMjMvMTEvMjI0NiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvNS85OTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.69"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elledge</span>  <span class="cit-name-given-names">S. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Isolation of crt mutants constitutive for transcription of the DNA damage inducible gene RNR3 in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">131</span>: <span class="cit-fpage">851</span>–<span class="cit-lpage">866</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DZhou%26rft.auinit1%253DZ.%26rft.volume%253D131%26rft.issue%253D4%26rft.spage%253D851%26rft.epage%253D866%26rft.atitle%253DIsolation%2Bof%2Bcrt%2Bmutants%2Bconstitutive%2Bfor%2Btranscription%2Bof%2Bthe%2BDNA%2Bdamage%2Binducible%2Bgene%2BRNR3%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Apmid%252F1516817%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTMxLzQvODUxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS81Lzk5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.997.70" data-doi="10.1126/science.1083430"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zou</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elledge</span>  <span class="cit-name-given-names">S. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Sensing DNA damage through ATRIP recognition of RPA-ssDNA complexes.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">300</span>: <span class="cit-fpage">1542</span>–<span class="cit-lpage">1548</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DZou%26rft.auinit1%253DL.%26rft.volume%253D300%26rft.issue%253D5625%26rft.spage%253D1542%26rft.epage%253D1548%26rft.atitle%253DSensing%2BDNA%2BDamage%2BThrough%2BATRIP%2BRecognition%2Bof%2BRPA-ssDNA%2BComplexes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1083430%26rft_id%253Dinfo%253Apmid%252F12791985%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDAvNTYyNS8xNTQyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS81Lzk5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/5/997.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/5/983" title="Comparative Analysis of Wolbachia Genomes Reveals Streamlining and Divergence of Minimalist Two-Component Systems" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/5/1007" title="Rapid and Efficient Identification of Caenorhabditis elegans Legacy Mutations Using Hawaiian SNP-Based Mapping and Whole-Genome Sequencing" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/5"><span>Volume 5 Issue 5, May 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/5.cover-source.jpg?itok=53KRlKz0" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/5/997?cited-by=yes&amp;legid=ggg;5/5/997" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-7Ao__Ifg6jiFgBoG7SU6aQ0_-_oCCDYOFHkxiqEs3PM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/mutant-screen-report" class="" data-icon-position="" data-hide-link-title="0">Mutant Screen Report</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/5/997" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/5/997" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/5/997" class="active" data-icon-position="" data-hide-link-title="0">Leveraging DNA Damage Response Signaling to Identify Yeast Genes Controlling Genome Stability</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459841" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-VhxgpQ1CFTsLXRa4-7sss05JUvEArXqGxFdtnAXMb9o" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/5/997.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/5/997" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-GFwkOa_bLxhk4IfJgAjlEbmkv02zK_zaiygkpfVMjfM" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/5/997" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/5/997.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459841" id="node459841--2" data-pisa="ggg;5/5/997" data-pisa-master="ggg;g3.115.016576" data-apath="/ggg/5/5/997.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Leveraging DNA Damage Response Signaling to Identify Yeast Genes Controlling Genome Stability</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jason A. Hendry</span>, <span class="highwire-citation-author" data-delta="1">Guihong Tan</span>, <span class="highwire-citation-author" data-delta="2">Jiongwen Ou</span>, <span class="highwire-citation-author" data-delta="3">Charles Boone</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-9002-5003" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Grant W. Brown</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">997-1006;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.016576 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459841--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jason A. Hendry </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry, University of Toronto, Toronto, Ontario, Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Donnelly Centre, University of Toronto, Toronto, Ontario, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jason%2BA.%2BHendry%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hendry%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJason%2BA.%2BHendry%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Guihong Tan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Donnelly Centre, University of Toronto, Toronto, Ontario, Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Molecular Genetics, University of Toronto, Toronto, Ontario, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guihong%2BTan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tan%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGuihong%2BTan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jiongwen Ou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry, University of Toronto, Toronto, Ontario, Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Donnelly Centre, University of Toronto, Toronto, Ontario, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jiongwen%2BOu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ou%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJiongwen%2BOu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Charles Boone </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Donnelly Centre, University of Toronto, Toronto, Ontario, Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Molecular Genetics, University of Toronto, Toronto, Ontario, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BBoone%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Boone%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACharles%2BBoone%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Grant W. Brown </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry, University of Toronto, Toronto, Ontario, Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Donnelly Centre, University of Toronto, Toronto, Ontario, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Grant%2BW.%2BBrown%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brown%20GW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGrant%2BW.%2BBrown%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-9002-5003" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Grant W. Brown</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:grant.brown@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">grant.brown@utoronto.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/5/997" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/5/997&amp;title=Leveraging%20DNA%20Damage%20Response%20Signaling%20to%20Identify%20Yeast%20Genes%20Controlling%20Genome%20Stability" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/5/997&amp;title=Leveraging%20DNA%20Damage%20Response%20Signaling%20to%20Identify%20Yeast%20Genes%20Controlling%20Genome%20Stability" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/5/997&amp;title=Leveraging%20DNA%20Damage%20Response%20Signaling%20to%20Identify%20Yeast%20Genes%20Controlling%20Genome%20Stability" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/5/997&amp;text=Leveraging%20DNA%20Damage%20Response%20Signaling%20to%20Identify%20Yeast%20Genes%20Controlling%20Genome%20Stability" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/5/997&amp;title=Leveraging%20DNA%20Damage%20Response%20Signaling%20to%20Identify%20Yeast%20Genes%20Controlling%20Genome%20Stability" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/5/997&amp;t=Leveraging%20DNA%20Damage%20Response%20Signaling%20to%20Identify%20Yeast%20Genes%20Controlling%20Genome%20Stability" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/5/997&amp;title=Leveraging%20DNA%20Damage%20Response%20Signaling%20to%20Identify%20Yeast%20Genes%20Controlling%20Genome%20Stability" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/5/997&amp;title=Leveraging%20DNA%20Damage%20Response%20Signaling%20to%20Identify%20Yeast%20Genes%20Controlling%20Genome%20Stability" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459841/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459841" id="node459841--3" data-pisa="ggg;5/5/997" data-pisa-master="ggg;g3.115.016576" data-apath="/ggg/5/5/997.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Leveraging DNA Damage Response Signaling to Identify Yeast Genes Controlling Genome Stability</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jason A. Hendry</span>, <span class="highwire-citation-author" data-delta="1">Guihong Tan</span>, <span class="highwire-citation-author" data-delta="2">Jiongwen Ou</span>, <span class="highwire-citation-author" data-delta="3">Charles Boone</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-9002-5003" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Grant W. Brown</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">997-1006;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.016576 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459841--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jason A. Hendry </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry, University of Toronto, Toronto, Ontario, Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Donnelly Centre, University of Toronto, Toronto, Ontario, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jason%2BA.%2BHendry%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hendry%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJason%2BA.%2BHendry%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Guihong Tan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Donnelly Centre, University of Toronto, Toronto, Ontario, Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Molecular Genetics, University of Toronto, Toronto, Ontario, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guihong%2BTan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tan%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGuihong%2BTan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jiongwen Ou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry, University of Toronto, Toronto, Ontario, Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Donnelly Centre, University of Toronto, Toronto, Ontario, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jiongwen%2BOu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ou%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJiongwen%2BOu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Charles Boone </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Donnelly Centre, University of Toronto, Toronto, Ontario, Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Molecular Genetics, University of Toronto, Toronto, Ontario, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BBoone%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Boone%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACharles%2BBoone%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Grant W. Brown </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry, University of Toronto, Toronto, Ontario, Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Donnelly Centre, University of Toronto, Toronto, Ontario, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Grant%2BW.%2BBrown%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brown%20GW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGrant%2BW.%2BBrown%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-9002-5003" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Grant W. Brown</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:grant.brown@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">grant.brown@utoronto.ca</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=997&amp;rft.epage=1006&amp;rft.atitle=Leveraging%20DNA%20Damage%20Response%20Signaling%20to%20Identify%20Yeast%20Genes%20Controlling%20Genome%20Stability&amp;rft.volume=5&amp;rft.issue=5&amp;rft.date=2015-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Hendry%2C+Jason+A.&amp;rft.au=Tan%2C+Guihong&amp;rft.au=Ou%2C+Jiongwen&amp;rft.au=Boone%2C+Charles&amp;rft.au=Brown%2C+Grant+W."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459841/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459841/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459841/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459841/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459841/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459841/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459841/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459841/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459841/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459841/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459841/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459841/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495856" id="node495856" data-pisa="ggg;10/3/917" data-pisa-master="ggg;g3.119.400936" data-apath="/ggg/10/3/917.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/917" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Mutational Analysis of N-Ethyl-N-Nitrosourea (ENU) in the Fission Yeast <em>Schizosaccharomyces pombe</em></span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495835" id="node495835" data-pisa="ggg;g3.119.400927v1" data-pisa-master="ggg;g3.119.400927" data-apath="/ggg/early/2020/02/21/g3.119.400927.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/02/21/g3.119.400927" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Identification of Suppressors of <em>top-2 </em>Embryonic Lethality in <em>Caenorhabditis elegans</em></span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495548" id="node495548" data-pisa="ggg;10/2/479" data-pisa-master="ggg;g3.119.400933" data-apath="/ggg/10/2/479.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/479" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Identification of the Genetic Requirements for Zinc Tolerance and Toxicity in <em>Saccharomyces cerevisiae</em></span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Mutant%20Screen%20Report" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Mutant Screen Report</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/5/997" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-5" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/5/997.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/5/997.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/5/997.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/5/997.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
