<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/5/931" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/5/931.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/5/931.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/5/931.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/5/5/931/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/5/931.atom" />
<meta name="HW.pisa" content="ggg;5/5/931" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Mapping Bias Overestimates Reference Allele Frequencies at the HLA Genes in the 1000 Genomes Project Phase I Data" />
<meta name="DC.Identifier" content="10.1534/g3.114.015784" />
<meta name="DC.Date" content="2015-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Brandt et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Next-generation sequencing (NGS) technologies have become the standard for data generation in studies of population genomics, as the 1000 Genomes Project (1000G). However, these techniques are known to be problematic when applied to highly polymorphic genomic regions, such as the human leukocyte antigen ( HLA ) genes. Because accurate genotype calls and allele frequency estimations are crucial to population genomics analyses, it is important to assess the reliability of NGS data. Here, we evaluate the reliability of genotype calls and allele frequency estimates of the single-nucleotide polymorphisms (SNPs) reported by 1000G (phase I) at five HLA genes ( HLA-A , -B , -C , -DRB1 , and -DQB1 ). We take advantage of the availability of HLA Sanger sequencing of 930 of the 1092 1000G samples and use this as a gold standard to benchmark the 1000G data. We document that 18.6% of SNP genotype calls in HLA genes are incorrect and that allele frequencies are estimated with an error greater than ±0.1 at approximately 25% of the SNPs in HLA genes. We found a bias toward overestimation of reference allele frequency for the 1000G data, indicating mapping bias is an important cause of error in frequency estimation in this dataset. We provide a list of sites that have poor allele frequency estimates and discuss the outcomes of including those sites in different kinds of analyses. Because the HLA region is the most polymorphic in the human genome, our results provide insights into the challenges of using of NGS data at other genomic regions of high diversity." />
<meta name="DC.Contributor" content="Débora Y. C. Brandt" />
<meta name="DC.Contributor" content="Vitor R. C. Aguiar" />
<meta name="DC.Contributor" content="Bárbara D. Bitarello" />
<meta name="DC.Contributor" content="Kelly Nunes" />
<meta name="DC.Contributor" content="Jérôme Goudet" />
<meta name="DC.Contributor" content="Diogo Meyer" />
<meta name="article:published_time" content="2015-05-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Mapping Bias Overestimates Reference Allele Frequencies at the HLA Genes in the 1000 Genomes Project Phase I Data" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Next-generation sequencing (NGS) technologies have become the standard for data generation in studies of population genomics, as the 1000 Genomes Project (1000G). However, these techniques are known to be problematic when applied to highly polymorphic genomic regions, such as the human leukocyte antigen (&lt;i&gt;HLA&lt;/i&gt;) genes. Because accurate genotype calls and allele frequency estimations are crucial to population genomics analyses, it is important to assess the reliability of NGS data. Here, we evaluate the reliability of genotype calls and allele frequency estimates of the single-nucleotide polymorphisms (SNPs) reported by 1000G (phase I) at five &lt;i&gt;HLA&lt;/i&gt; genes (&lt;i&gt;HLA-A&lt;/i&gt;, &lt;i&gt;-B&lt;/i&gt;, &lt;i&gt;-C&lt;/i&gt;, &lt;i&gt;-DRB1&lt;/i&gt;, and &lt;i&gt;-DQB1&lt;/i&gt;). We take advantage of the availability of &lt;i&gt;HLA&lt;/i&gt; Sanger sequencing of 930 of the 1092 1000G samples and use this as a gold standard to benchmark the 1000G data. We document that 18.6% of SNP genotype calls in &lt;i&gt;HLA&lt;/i&gt; genes are incorrect and that allele frequencies are estimated with an error greater than ±0.1 at approximately 25% of the SNPs in &lt;i&gt;HLA&lt;/i&gt; genes. We found a bias toward overestimation of reference allele frequency for the 1000G data, indicating mapping bias is an important cause of error in frequency estimation in this dataset. We provide a list of sites that have poor allele frequency estimates and discuss the outcomes of including those sites in different kinds of analyses. Because the HLA region is the most polymorphic in the human genome, our results provide insights into the challenges of using of NGS data at other genomic regions of high diversity.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/05/01" />
<meta name="citation_mjid" content="ggg;5/5/931" />
<meta name="citation_id" content="5/5/931" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/5/931" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/5/931.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/5/931.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/5/931.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.015784" />
<meta name="citation_pmid" content="25787242" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="931" />
<meta name="citation_lastpage" content="941" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Débora Y. C. Brandt" />
<meta name="citation_author_institution" content="Department of Genetics and Evolutionary Biology, University of São Paulo, 05508-090 São Paulo, SP, Brazil" />
<meta name="citation_author" content="Vitor R. C. Aguiar" />
<meta name="citation_author_institution" content="Department of Genetics and Evolutionary Biology, University of São Paulo, 05508-090 São Paulo, SP, Brazil" />
<meta name="citation_author" content="Bárbara D. Bitarello" />
<meta name="citation_author_institution" content="Department of Genetics and Evolutionary Biology, University of São Paulo, 05508-090 São Paulo, SP, Brazil" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7676-9367" />
<meta name="citation_author" content="Kelly Nunes" />
<meta name="citation_author_institution" content="Department of Genetics and Evolutionary Biology, University of São Paulo, 05508-090 São Paulo, SP, Brazil" />
<meta name="citation_author" content="Jérôme Goudet" />
<meta name="citation_author_institution" content="Department of Ecology and Evolution, Biophore, University of Lausanne, CH-1015 Lausanne, Switzerland" />
<meta name="citation_author" content="Diogo Meyer" />
<meta name="citation_author_institution" content="Department of Genetics and Evolutionary Biology, University of São Paulo, 05508-090 São Paulo, SP, Brazil" />
<meta name="citation_author_email" content="diogo@ib.usp.br" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7155-5674" />
<meta name="citation_reference" content="citation_journal_title=Philos. Trans. R. Soc. Lond. B Biol. Sci.;citation_author=K. G. Andersen;citation_author=I. Shylakhter;citation_author=S. Tabrizi;citation_author=S. R. Grossman;citation_author=C. T. Happi;citation_title=Genome-wide scans provide evidence for positive selection of genes implicated in Lassa fever.;citation_volume=367;citation_year=2012;citation_pmid=22312054;citation_doi=10.1098/rstb.2011.0299" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=P. J. Bjorkman;citation_author=M. A. Saper;citation_author=B. Samraoui;citation_author=W. S. Bennett;citation_author=J. L. Strominger;citation_title=Structure of the human class I histocompatibility antigen, HLA-A2.;citation_pages=506-512;citation_volume=329;citation_year=1987;citation_issue=6139;citation_pmid=3309677;citation_doi=10.1038/329506a0" />
<meta name="citation_reference" content="citation_title=Genome Med.;citation_author=S. Boegel;citation_author=M. Löwer;citation_author=M. Schäfer;citation_author=T. Bukur;citation_author=J. de Graaf;citation_title=HLA typing from RNA-Seq sequence reads.;citation_pages=102;citation_volume=4;citation_year=2012;citation_pmid=23259685;citation_doi=10.1186/gm403" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. H. Brown;citation_author=T. S. Jardetzky;citation_author=J. C. Gorga;citation_author=L. J. Stern;citation_author=R. G. Urban;citation_title=Three-dimensional structure of the human class II histocompatibility antigen HLA-DR1.;citation_pages=33-39;citation_volume=364;citation_year=1993;citation_issue=6432;citation_pmid=8316295;citation_doi=10.1038/364033a0" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=S. J. Chapman;citation_author=A. V. S. Hill;citation_title=Human genetic susceptibility to infectious disease.;citation_pages=175-188;citation_volume=13;citation_year=2012;citation_issue=3;citation_pmid=22310894" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Danecek;citation_author=A. Auton;citation_author=G. R. Abecasis;citation_author=C. a. Albers;citation_author=E. Banks;citation_title=The variant call format and VCFtools.;citation_volume=27;citation_year=2011;citation_pmid=21653522;citation_doi=10.1093/bioinformatics/btr330" />
<meta name="citation_reference" content="citation_journal_title=Int. J. Immunogenet.;citation_author=D. De Santis;citation_author=D. Dinauer;citation_author=J. Duke;citation_author=H. A. Erlich;citation_author=C. L. Holcomb;citation_title=16(th) IHIW: review of HLA typing by NGS.;citation_pages=72-76;citation_volume=40;citation_year=2013;citation_pmid=23302098;citation_doi=10.1111/iji.12024" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=J. F. Degner;citation_author=J. C. Marioni;citation_author=A. A. Pai;citation_author=J. K. Pickrell;citation_author=E. Nkadori;citation_title=Effect of read-mapping biases on detecting allele-specific expression from RNA-sequencing data.;citation_volume=25;citation_year=2009;citation_pmid=19808877;citation_doi=10.1093/bioinformatics/btp579" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=R. L. Erlich;citation_author=X. Jia;citation_author=S. Anderson;citation_author=E. Banks;citation_author=X. Gao;citation_title=Next-generation sequencing for HLA typing of class I loci.;citation_pages=42-42;citation_volume=12;citation_year=2011;citation_pmid=21244689;citation_doi=10.1186/1471-2164-12-42" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=P.-A. Gourraud;citation_author=P. Khankhanian;citation_author=N. Cereb;citation_author=S. Y. Yang;citation_author=M. Feolo;citation_title=HLA Diversity in the 1000 Genomes Dataset.;citation_pages=e97282;citation_volume=9;citation_year=2014;citation_pmid=24988075;citation_doi=10.1371/journal.pone.0097282" />
<meta name="citation_reference" content="citation_author=W. Helmberg;citation_author=M. Feolo;citation_author=R. Dunivin;citation_author=D. Hoffman;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=R. D. Hernandez;citation_author=J. L. Kelley;citation_author=E. Elyashiv;citation_author=S. C. Melton;citation_author=A. Auton;citation_title=Classic Selective Sweeps Were Rare in Recent Human Evolution;citation_pages=920-924;citation_volume=331;citation_year=2011;citation_issue=6019;citation_pmid=21330547;citation_doi=10.1126/science.1198878" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=E. M. Hill-Burns;citation_author=S. A. Factor;citation_author=C. P. Zabetian;citation_author=G. Thomson;citation_author=H. Payami;citation_title=Evidence for more than one Parkinson's disease-associated variant within the HLA region.;citation_pages=e27109-e27109;citation_volume=6;citation_year=2011;citation_issue=11;citation_pmid=22096524;citation_doi=10.1371/journal.pone.0027109" />
<meta name="citation_reference" content="citation_title=The major histocompatibility complex database, dbMHC.;citation_author=A. Kitts;citation_author=M. Feolo;citation_author=W. Helmberg;citation_pages=1-29;citation_year=2003" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=T. Lappalainen;citation_author=M. Sammeth;citation_author=M. R. Friedländer;citation_author=P. C. ’t Hoen;citation_author=J. Monlong;citation_title=Transcriptome and genome sequencing uncovers functional variation in humans.;citation_pages=506-511;citation_volume=501;citation_year=2013;citation_pmid=24037378;citation_doi=10.1038/nature12531" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=E. M. Leffler;citation_author=Z. Gao;citation_author=S. Pfeifer;citation_author=L. Ségurel;citation_author=A. Auton;citation_title=Multiple instances of ancient balancing selection shared between humans and chimpanzees.;citation_volume=339;citation_year=2013;citation_pmid=23413192;citation_doi=10.1126/science.1234070;citation_georef=2013044317" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=E. Major;citation_author=K. Rigo;citation_author=T. Hague;citation_author=A. Bérces;citation_author=S. Juhos;citation_title=HLA typing from 1000 genomes whole genome and whole exome Illumina data.;citation_pages=e78410;citation_volume=8;citation_year=2013;citation_pmid=24223151;citation_doi=10.1371/journal.pone.0078410" />
<meta name="citation_reference" content="citation_journal_title=Tissue antigens;citation_journal_abbrev=Tissue Antigens;citation_author=S. G. E. Marsh;citation_author=E. D. Albert;citation_author=W. F. Bodmer;citation_author=R. E. Bontrop;citation_author=B. Dupont;citation_title=Nomenclature for factors of the HLA system, 2010.;citation_pages=291-455;citation_volume=75;citation_year=2010;citation_issue=4;citation_pmid=20356336;citation_doi=10.1111/j.1399-0039.2010.01466.x" />
<meta name="citation_reference" content="citation_journal_title=Annals of human genetics;citation_journal_abbrev=Ann Hum Genet;citation_author=D. Meyer;citation_author=G. Thomson;citation_title=How selection shapes variation of the human major histocompatibility complex: a review.;citation_pages=1-26;citation_volume=65;citation_year=2001;citation_issue=Pt 1;citation_pmid=11415519;citation_doi=10.1046/j.1469-1809.2001.6510001.x" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=R. Nielsen;citation_author=J. S. Paul;citation_author=A. Albrechtsen;citation_author=Y. S. Song;citation_title=Genotype and SNP calling from next-generation sequencing data.;citation_pages=443-451;citation_volume=12;citation_year=2011;citation_issue=6;citation_pmid=21587300;citation_doi=10.1038/nrg2986" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. R. Quinlan;citation_author=I. M. Hall;citation_title=BEDTools: a flexible suite of utilities for comparing genomic features.;citation_volume=26;citation_year=2010;citation_pmid=20110278;citation_doi=10.1093/bioinformatics/btq033" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. Robinson;citation_author=J. A. Halliwell;citation_author=H. McWilliam;citation_author=R. Lopez;citation_author=P. Parham;citation_title=The IMGT/HLA database.;citation_volume=41;citation_year=2013;citation_pmid=23080122;citation_doi=10.1093/nar/gks949" />
<meta name="citation_reference" content="citation_journal_title=Curr. Opin. Immunol.;citation_author=L. M. Sollid;citation_author=W. Pos;citation_author=K. W. Wucherpfennig;citation_title=Molecular mechanisms for contribution of MHC molecules to autoimmune diseases.;citation_pages=24-30;citation_volume=31C;citation_year=2014;citation_pmid=25216261;citation_doi=10.1016/j.coi.2014.08.005" />
<meta name="citation_reference" content="citation_title=Nature;citation_journal_abbrev=Nature;citation_title=An integrated map of genetic variation from 1,092 human genomes.;citation_pages=56-65;citation_volume=491;citation_year=2012;citation_issue=7422;citation_pmid=23128226;citation_doi=10.1038/nature11632" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=T. J. Treangen;citation_author=S. L. Salzberg;citation_title=Repetitive DNA and next-generation sequencing: computational challenges and solutions.;citation_pages=36-46;citation_volume=13;citation_year=2012;citation_issue=1;citation_pmid=22124482;citation_doi=10.1038/nrg3117" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=L. D. Ward;citation_author=M. Kellis;citation_title=Evidence of abundant purifying selection in humans for recently acquired regulatory functions.;citation_volume=337;citation_year=2012;citation_pmid=22956687;citation_doi=10.1126/science.1225057" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Mapping Bias Overestimates Reference Allele Frequencies at the HLA Genes in the 1000 Genomes Project Phase I Data" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5.cover-source.jpg" />
<meta name="twitter:description" content="Next-generation sequencing (NGS) technologies have become the standard for data generation in studies of population genomics, as the 1000 Genomes Project (1000G). However, these techniques are known to be problematic when applied to highly polymorphic genomic regions, such as the human leukocyte antigen ( HLA ) genes. Because accurate genotype calls and allele frequency estimations are crucial to population genomics analyses, it is important to assess the reliability of NGS data. Here, we evaluate the reliability of genotype calls and allele frequency estimates of the single-nucleotide polymorphisms (SNPs) reported by 1000G (phase I) at five HLA genes ( HLA-A , -B , -C , -DRB1 , and -DQB1 ). We take advantage of the availability of HLA Sanger sequencing of 930 of the 1092 1000G samples and use this as a gold standard to benchmark the 1000G data. We document that 18.6% of SNP genotype calls in HLA genes are incorrect and that allele frequencies are estimated with an error greater than ±0.1 at approximately 25% of the SNPs in HLA genes. We found a bias toward overestimation of reference allele frequency for the 1000G data, indicating mapping bias is an important cause of error in frequency estimation in this dataset. We provide a list of sites that have poor allele frequency estimates and discuss the outcomes of including those sites in different kinds of analyses. Because the HLA region is the most polymorphic in the human genome, our results provide insights into the challenges of using of NGS data at other genomic regions of high diversity." />
<meta name="og-title" property="og:title" content="Mapping Bias Overestimates Reference Allele Frequencies at the HLA Genes in the 1000 Genomes Project Phase I Data" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/5/931" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Next-generation sequencing (NGS) technologies have become the standard for data generation in studies of population genomics, as the 1000 Genomes Project (1000G). However, these techniques are known to be problematic when applied to highly polymorphic genomic regions, such as the human leukocyte antigen ( HLA ) genes. Because accurate genotype calls and allele frequency estimations are crucial to population genomics analyses, it is important to assess the reliability of NGS data. Here, we evaluate the reliability of genotype calls and allele frequency estimates of the single-nucleotide polymorphisms (SNPs) reported by 1000G (phase I) at five HLA genes ( HLA-A , -B , -C , -DRB1 , and -DQB1 ). We take advantage of the availability of HLA Sanger sequencing of 930 of the 1092 1000G samples and use this as a gold standard to benchmark the 1000G data. We document that 18.6% of SNP genotype calls in HLA genes are incorrect and that allele frequencies are estimated with an error greater than ±0.1 at approximately 25% of the SNPs in HLA genes. We found a bias toward overestimation of reference allele frequency for the 1000G data, indicating mapping bias is an important cause of error in frequency estimation in this dataset. We provide a list of sites that have poor allele frequency estimates and discuss the outcomes of including those sites in different kinds of analyses. Because the HLA region is the most polymorphic in the human genome, our results provide insights into the challenges of using of NGS data at other genomic regions of high diversity." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5.cover-source.jpg" />
    <title>Mapping Bias Overestimates Reference Allele Frequencies at the HLA Genes in the 1000 Genomes Project Phase I Data | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"uR-aEhqGi9oRVKvi1xD2HdN8LxLAyeZjrYDuYTM6x7k","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459829","apath":"\/ggg\/5\/5\/931.atom","pisa":"ggg;5\/5\/931","ac":{"\/ggg\/5\/5\/931.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/5\/931.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/5\/931"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/5\/931"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/5\/931":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00226ms2OLCMm4QK2PEC+va92qO38CUnT\\\\\\\/xQxXkJ9w2DI+VtmdKeAgD22tTQoP66qZKkwNgB3vGZxvc4boQ9I83rsZVeC3keBYa6dWQVk6Z7oUfGptRvieBzeUbDd\\\\\\\/AwJlqI2GdyE8cO9\\\\\\\/SkPD12a2VxJPzujbTt1RJetCIcfdKa9QpFnpco\\\\\\\/\\\\\\\/XMla3EvjmwXon0enrvV9DW42mlz84ECXSo612O39E9cdV9mn+Utpytq5v\\\\\\\/FOy8Pq0wX3BuRbsGxZvaDW2D7FBsHP170TBaxCfBUBmv7bdLoIMTkp+ZkcGRigAPAQ3L9rWvvRdhKN3cf0PTkUbZj3rFDPfW6XF\\\\\\\/WE1lDYbBhpClXI+mfwAiIzQCHClOIvXh1JUIzjjXJDFI95AqqOPFQiQ1BlQOf\\\\\\\/KHSStGF6wtuhU68FfB8u3OWnAl33BagoKjH4pyp31Am3mO1vbDuZioFFo\\\\\\\/U1ftukaaSY9NeXDfhXZMa+PmCmoiuhEDWJbEVSH4UIjP5gFE2J+Vfn6+sl+XNhyYTXznZOkAnonvmuVeNQPHcemKpDYIeAAdFaaxF0tfU6hwI9tqH6D2xjiAyjUz3gMElekPu+wBzfwM4ueXdAqvLt9+kiFjATSMXHLJ0vUixURvtXZi2TERPUyiIfWpOFfUW7wVyVuoifEBi6DuUpvUYg7U2MBkCE1I8uUIGe0z0UBQSqypoDQCl++0uY33tVxzkci387NuN2PrtFNq7+xULf+Hrrvox4xE3QpXADbJH49MlmK6AGGfWaR2idtWWLv\\\\\\\/P61ZJzN45RlccR9hYM+4DfHklPPPz2rA+wL8YLnlVYvMK2VkBwnsCVoSDmOX9bQCFxbZExC\\\\\\\/qheFsIAYD\\\\\\\/8aJK8hBxxxb2nzDoS0BcRNPt8G7x+nebbB\\\\\\\/M0MVbO+VRjyRN79icdiOUvNErXTroASOT6lN2lMhVrEDPur5tSum0w1uU4b\\\\\\\/S5UyURgZyKVLG6bUfXlUtHtG1TwmBPcXxG4oVmRMe86inPOwraIPDdRKg9W2rfMVEnlIf+4704k0wye\\\\\\\/kGzFOdjSNlm26wL4NMt\\\\\\\/e2atFMh3DCqWr16p0UhzGpOPh8SzI6S1orcgXAyVH6sM9n8wEyKffVd9f88iuWXbNmZWNU7fJbAUM9R\\\\\\\/GEYyHiCFhRgManxZA0Mf7fz+emCGBAwzEKlKj3U+9ScY\\\\\\\/OPrFvvysfJG9l3Pg0FYChqzETdhE4fcRBqgTkS455yLUp50agR8Au8J8YIXwyCoa0cV8Lf1g8D6PlvclIthWqWjbkVQ8eCsYgHB5ok\\\u0022,\\\u0022iv\\\u0022:\\\u0022s\\\\\\\/SLvZ4trMt7X38e+ADnLw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222ad132825b90977edd9e647bd29380acee72a321e741916ed228f383c6a9d535\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ktuVpD6FpJ+PRVItCSXziqN54DpFJvjUVH+duKC209rW6JPMaDyhyXgffAeEaUI0MlccAqJb9sUn5jGkbVpASL3QzNBtNpkxtmf9eshUwF\\\\\\\/M6Z1a3BZZQPPJLNe043a6uYDn2gYD5z9LmQQ4tOOx02mcMSxrzXGHgZ1YwRIB70XYF9OdX2zq49tuqQ3T5UwkiounjhI5Y8LluFErMsr9U3Y2fpsrNcr+krt5gAFlr\\\\\\\/23NW2BbulZGdOh+W4DNptdXFNlq6vv5voNr4N9sqR06gqyh0uDmxQS1\\\\\\\/OLkZF93B17JCXN4ydA4kayN\\\\\\\/9I9KxpUAWqoJb6zp0pkFyWU3TEBQZJKV2iHbbdVG4WdX2rjGad9nRiHI\\\\\\\/65W0MrwqK6yATzL5nitctPlKPGdYtq4DR2BV630xB\\\\\\\/65HhuC12SvPTxyadijyNPX8Hc6c7CKAgzMPCo7XlShmDAcGF6US0v9WL3HnWruxwsO8pYan3W\\\\\\\/SvZIPPdjT+z\\\\\\\/xfV9XuWF9+4ZOig\\\\\\\/VUobzdpfnGJaEL5uRkABXqHva+8Jrzv3BGPWoA33ag\\\\\\\/OiVVHXu2cUT1aLz0qrHgkc61MZavo8W9oBE7uRmrSX\\\\\\\/lBnuj3rOq3JT5RNM\\\\\\\/sPJ7hlAMtSr04Skr32pdJl\\\\\\\/LzF7XqBz9bvHgnIBJj5+mo6vBpB1YDn4YpIFuzz4aMwHp2KnEDFJ8o5FD3lBG1FfObSopOJlnLvYnwMbxy0H+1lsctl3Wmbg5Xtp0diqEBuIplO62LljpCRCgo0rRZu1rBBRmt0QqWq5y3MQjjMV36raFLp2Sdi3Oc6nS1nYMU59i2+08mwWp2f971J1yvcLz\\\\\\\/eIv4qaOW399GEkCLuMNGuu86Em158hT2jGjsRs78AH0ojWUwNByl9SpaYZviaY0x9SakAs8\\\\\\\/0tyEPf4UUS7UPG51Ic\\\\\\\/njqAQ4noR3oyJnjT6SlNicA1AB\\\\\\\/0eZbCOW3xU4\\\\\\\/i5pMpv7tdWl9AhHqdt9BFX7c1AEN5zXOrVFrK7SE9+y5mITyLIPh5ADa2BYGKJJ5BFNNplGT4NZpjWvkSPglTMazyKsP2m22WEWNbpumsESLyfQiiqwghp3L4+HnReYUtIfl9zbxqkcDFa+D\\\\\\\/CZR6iYNoe9A090HbuVHxvkAht9qA4DXGwdZPwEwNNl2owiPQvTG+PHfolTDZlfRlIpiaH6CkQPtbjHHWc0uYfhXzW3X0nds9Hpf\\\\\\\/+AAU3fdP+0OGr4grxg1MEUdp0b05sRvqSkWTIeXYtIaptCxMxEQb2qP8cI3+bQsYJ+bs0AWD+ICXnJEeDOQwwsUuAiNfEKve4qY6yPbLtbMniY19eQv+tWwYAwyquKvzAs3fkh8m6Iw5ZuCh5Dqk9aq7\\\\\\\/x5VJ+Q0jYXih9N2eIdCN25ZVy\\\\\\\/0W0BvZ1ryafLlr32xj4LHiW0\\\\\\\/Hm1WBZ4FJidJlVpvXiTR9j3dRDWanIT0NycpH7WcN\\\\\\\/dL9mSY9wwVtnnCFQ4aBzcgJ13K7nOfdzbGqHsMOoejCzRBCmVCua5pH0Cf7wS1+WMMljo9EBCm7yS7O5S\\\\\\\/O8uckB5D\\\\\\\/0bxA9FEyaa8hieuyIP5TJJuja\\\\\\\/jSk3ibYRvOm2aexPuoeVFlQ1vbADBtX8GKJkHOO4KaU9u73n1DHOfh0D1ZTzBs\\\\\\\/K2+whT5IOdeJqNKSdjcGXlOpKOcwjCxBrREYsTfgOdZCy84SiEXa3UMkaN5wo\\\\\\\/E0CvGh3HJCw+P1laAmABhkz8yzEhhnVHWGoh\\\\\\\/tBLiU24UihraNtei3qgix0iSKYXlKnoaGTd4J\\\\\\\/j\\\\\\\/89oYiWzjW3hCJdIAMQy4AlFQLWQZiKnOjXI8FY7pAeiL8BrQsau\\\\\\\/x3xRUO2DJB31dT9RmFWl2tj7ybqZXqaIdaHgrIV6vtRZoWIoTM9qOaa+GY38ycF5uy0vXLcj3xXpkehdeFCGzCzZM3gfp\\\\\\\/EP2hP4dwH+7u314vqBU8lcaIiYAWuHumPmyp63OnXy9SgS7NJQ9yVqlV5v64qxNDdDE31j6SPZFE8poFGKK4qyk0fQpbTCS9Ll5ExOuC+srnXs+sQ9\\\\\\\/4uwiVclnkhXfsNwt7I4BtItj2j0a5dax+vg6\\\\\\\/Uy76FbTFuU+6+EhLQR4ZaTQzxzKK8r\\\\\\\/XLN6oiE9sEpGhkyq1OkRHN8OaP\\\\\\\/pqL8nkHLkcsj+qtFRMbUzRci\\\\\\\/im\\\\\\\/sLMuef7koLwbh9\\\\\\\/XGeLiqcaQrxif22EbjYkvoMfiAZcMwzOPlSFbmp2PDwUH8dcaG9QLLTAalGGk1z9yPvBRLYHvnv7MthpEIkDr9hDRI267vaH1ksR+fQ4g959+\\\\\\\/i6BUpQj1JXP3G2TEbD3PT0ZHTHTZYalSXPafG\\\\\\\/EfNIFwyO1LgCGjTEQclqn9SjyeEsjguSfTL\\\\\\\/Nd23dlTKquYizm5VnpLUGxeRK9pXLO1Icr\\\\\\\/wXTwzhwJh+ZSoDn6JKyIGN91lH0hzZ60ufhPV7WOCkpb7agtcjLm3YFBkTssRjt6qPoRo5yCglmQE+FVLQgXH3ZlXp0swOmf5\\\\\\\/jvQfSN6+PLbZKESVkHwBs8CqSjmNGljgwl287HxCeyxi9yKd98A1QvY+b2GSRKPrqnBXIXdd8eAIHdcj8xl25npkR0yYfIFytKHdoz\\\\\\\/Mzk1TL0xeFSLTpmD4ycb5lFyHLMKOG7pYfDCT4g1fysDaL3J1dW6BfSA3gjQJ2\\\\\\\/sPwi6M1UhR26o5mLzBYtrtMtyrV25jIRoxQ+mxhwj1YJb4mvdn2xR1FqPfOIBgwoJuLviFGRWI1r6Cay+wmPDIbCibulnxN6mqB6NMM+qz76Pw85gyJKfRtuXXCMNjN\\\\\\\/CTayWzYAq5OCgucUFxgn77wIqz23sx1u+6cigrUGEWQuzKQWiueYrCWmg9PTNHQIzo9\\\\\\\/N+f6KAOCOjeRCxS7aHjPVaTNO3cQUL8inkqrWSJ11NtZBRnk34zLTir7E+dz7zLjMcO2XKc5Rt97eBtfRk37y1wgQ8Ia8vkxAPKyVW7JoOt3PJh8f4NzhqnXKOV+CmYQz2p0LHVg9D2omKVWJg\\\\\\\/JbVs+bbp4FLR6Lg2CLJVVnt\\\\\\\/3ZM4jJ+rgPiKo3R0CmdFVKKwYHGUpFYs2cbYzd9dp\\\\\\\/sqsp4elcc2SSyPlIu5PkyLDFj3Bu6+8bfKtcFAX0PwBs53rH2kBjH5QsWXl2a6N6\\\\\\\/OV4qfYYpjLNukTQEX6alvbbqKozORPyyJVVpnamvHFFjVm2RKJnBvYy2+uvJ8zVwNpuoUoeVE5yxKvtEHMxleespafXjPCkKxOnSvf6CL\\\\\\\/duNkgSwFztAsR74bx1imwhxQEaGWie0gw72qb\\\\\\\/WyKSUaApNDlvWLmkdzalTg\\\\\\\/G4vElqMKkEuV7wyRt\\\\\\\/SsfAWJXmjaBnv4BoqX7meU4lrE0tT25pYP5k7Rsb8bwofpa3t\\\\\\\/j8O8cgy\\\\\\\/N8\\\\\\\/BH8OCtCKAVG00G8rFmPJlFM3Eb4CuSYaNhXCdVDB1yXhe8y\\\\\\\/EbXhNFRCJPwJrFJMOuT1PwZNkCpdYnKrjaSa+XHgVXB31+L3maV8BQuXZo9NIOuHA9skz0iWPq\\\\\\\/0e2m8o97pqv7vqW25YSxB4wXdtaFj75FG4R8nyOkPWnZYCAtDIgMaVvGcL59KUAxvldnLLA+dyS4dr\\\\\\\/oCvply0KYTX2Mw9PBX+qn6FMsZOigBdeF4+gqj9tMpnTbqtfTwAIKQIpB2BHXCz\\\\\\\/K5q77g8II3varPIzydwHDhAx2lZmSI5Ous67cqFbOrRlvVweBXtSG4pXaCdqfPnSe21rcSDfnCo0MEST\\\\\\\/6jCeKqGGmOEgiCJfEDpczhBxV\\\\\\\/VBdLcgSQaC7Ois808Sm+7cZL7NYCU9+BFJ7AkupB3T12a6fkJljDzoD3KUyKd0EODx3gKgTMQP8yFHigrvzjpuuXfYCHxwWhMrYVnQuQJuYqIwZhepkRExIiJpJaMx\\\\\\\/rvd+WueSn+oyweXMF9lGkufpbOmgw8B9ovHdjyQI5e6BRii9naai7Pq\\\\\\\/WA\\\\\\\/doaUy+uEpLPJqc51Gdfv5k7AXjvzEpgprlHjq9YdOtGkeOoX+IvA86AJy7IjRb24IilmFyJbZfLwOOWf+MyeOkWdbyEPR45VJHUDxbvueVc5\\\\\\\/3dpNLvn6uQdxPYZDet86bjjCprq8zWiBk9ro+XbRCPRuq8yveoUOB4ss9NE6MKH+Td5eINrYwk+NmxHUjYPOxEqzSWhhxGCSM5QVoWVRFPjL8fe8tJAWYF1+Y+T9oFXJoM\\\\\\\/yX0gmao2ogJJKMQHLtxYy3a8G3qfppGVdRLAzHfMjcWgm2tPdbELPATavGp4znqW1mNOdos\\\\\\\/gOn7VdtuNTLJW8fvUveL71SL+lMGM6booTPAY++ig646TMjgb8GYmDjBTpByLexaXN13fz3Sx4FeIevxjqPajNlPxon5HiEWd1XNh\\\\\\\/qSHn3zH7Sw0PIZsbjBbk4fkk7zl5fckGuhBBW3dLYGZ45UtwBaNqJMhU3dua\\\\\\\/OwLCj8oWAC+vMlegP3KTsYROYyFY8IBhFWwCvMsid\\\\\\\/UCUJPG\\\\\\\/MpLXU2GF7UeQeROwR0CAJO7XigboR5+rhakg5PN85lmizLDKiZJAIbXqHATUJQ7SXJvxiUMRPnkDd9Nj5hSbjCK94fIK0ilrNh8CvOkeXhY5h8wnDEteyt4v9cwvIMjOW6TT85hUI9Vtgw5FYyMk1Lzsu8kHWupYAGnZRi2szNCbA8qzxmpXPDRxmio2vRkbsBKjrhrp9YJP\\\\\\\/L9xla6QAe3rxh2v5aeSaTBokoHVkgLU88629TeCqRWP4PQgztfrWgdsCgb8BNNfdaNYj\\\\\\\/owKzVdB3z58lSZXgfgb3uwshE1KM532DIAAsZBB5USAO5XfK3PzmI\\\\\\\/VsNAPFTVGvm48lQsLtZ1mbm27\\\\\\\/U5E3klyTsZF79hqtHrjuR07acIuI76uHyzTJz+9gMOmja3prWo6gQRbPpzcb1bnPf4B55zvEHyqr6HTmFcRuqhxWHd4FEGUHAfJnqMErFtx0ne10cnQfOi\\\\\\\/n9FL7xZRY31USzQ381VkvD7bs+PldEBGrRXctLTnaiEKz+4fz9z57PGgTNu5V0pI4dxMA5fmUnbeesSzb5QT4eBujcxBbm6TjW+oVJKnWs6li8rtIkSkdNJvW+q2+j7Ill8XIr+NZ9TTfQKtx8bUoU3LmQy87x444UR0ssIJrQZe5NOkyabgdI9KvTEXudhdStqQycn+UzbpEyGr6IbZr79S7rZ52RkhtN1Ahne5SIVW8fYrr\\\\\\\/+4+0deBf\\\\\\\/X6iJ5MaiVobqRwfU2yMVVTBkHXcMWAQix78txfshCyiFrgTK6IwX8Gtk7WYIf7vRKovwQEbmD5uStfSYE0w\\\\\\\/lYOuPfzXrsGpAhmtZ9BRSRljUkSCbcQUufvd3z4ID6vyO5ya4HYJYHcJ+5ktfWT6ADfpxlL8D9fC6uyskwAnFYwRc3dg43Wd4Tja9lnjGWlv+YVp8sBmZtJOCm0ndoZEhEjmQou5sVMiox2N3o4kZvXQHarVnNB6jnjpSL5fGAcB0awvdWIg3t1l8VN1AKtDkSnPMP3Sz6GQ1lxjrOtrQ5YwZArzrpL9DrmFf9Tit0cSraI5YfjLsoTJ9Y7XPfIeCV\\\\\\\/AJzAYAhJJFYvooMM3GNJOQxYHT0c3J0469m400c+dVGNQfGnCuebvztNkB3z75w0trEILMHFbfNCoMFncw3axSd1KC\\\\\\\/ZZbG5rCC1StHWqTHjt2cngLALpCPWJAvoyW7aCOdr920CoGpWHpeTXK7U45OMQICOTiSKadupR6K5x8Mtr8pyddTNrtReX58z5AX+O4mT1uylj36d7eQGDk\\\\\\\/\\\\\\\/GOftSDAqgVn05UIuQinNHz1+M1bFzgwdFBaHRT7eEB5ljdJa6rYyDDV\\\\\\\/ueDumiUKhPDaOD7lT8wbnD2PTu4mjafX9blo1MhhmHIR2Nw54OkUCIow083v2izjyGnN5hq+KbvKeKOBy8oThou+\\\\\\\/S9pc5e3rr7y6BWiugQbGQ1P1YiGmoBMLRqPrD99OtMAteFHTxghAcuZ\\\\\\\/jeWkv+lfxti1zX1USo0B3k+xAV9ioM66L\\\\\\\/fJJMgUmhhGjglWvXVTfp9N8r6OJQSOpt9XD8lOqT6oHOYngFXSlJhUxbQRa+OkNXASzHSxZ3r60VL3KCoE99zvMm1svvh6aBTKrj0kwwIOKSbvfqsVtneB73L2+DTALoAboWT5hVKJ0OtLLQnnL99eokv3vKVAKMjNI\\\\\\\/ItecoLyHz\\\\\\\/3vyxVgprlN01oMpN6ylxGKiW3m0SnKc8P8FWr\\\\\\\/Z0p5QdCeidxyGJdYZO8ho2wx1GEvbX4H8jQgtSplmItV\\\\\\\/yJve2SXaK0SjOpb1WgynVeHRdG7JS68C9wEYkk82wM5lL++n2WUZIpeKfkTj\\\\\\\/WVglWuuY3NcFotOqEk6dJJPwDaxzf0rdXd8fCf0KIhTgER0XdHddU8k326DDTe+Q8sr2RAnSZcFKm9hnDWaumlnTDQ4WqiyzyGxFz+7+hIyS58m+QwJqzdL4x321+A1PONQDYmh0\\\\\\\/78oU\\\\\\\/esNItlWPvrlPk+nXoSswckVMrMYO7BwVR5qsa1aIXuwcpbuIG3GT7AbailnizJDNsRLlqrrnsPh8foB0nFXmVpN2PvHKSJy0v\\\\\\\/mkP1JYhPH8uCNW8wqE+q8l4\\\\\\\/TBmOvIWstSNWFpW3jAR1+SBeigYqw20Ahe51Ek9aWjF5N4e5OY6+gcMcp5PssWEFl73NgcZ9cw7FoH1c0U2H3ykRqfgY7d56nCBbp4OhsCLTeALhHMau2ZztYp3DJTE1hGK26imb5lFamVLmzYKDlbcNulpcH3d99BtBSORR1Gq\\\\\\\/vNIE4TnO7ViBgSJNJ6oEjofBTcL5bHbld9Emle4A83\\\\\\\/r9NZrPYjuZI1mjyzxq4L6Zabo\\\\\\\/eZoK24t2gkH2zSz1s3xMLhthYhA8DfFKL8EvHeRVWAa1sa53HMl6e8yNrn4F28c4krDc1MG9hsjblEnX5WkNHhcl0QWk4I7Dn4CrIYj7x9n\\\\\\\/PlQgIkSeXwv10rM1nGecH5LQqcEtuI7HGSAZHAh7FxJ6\\\\\\\/Ggkihssx5gYfujx7obRhAGEw+GFlMCFF+xj445iE1ZncwxrcjqqzI7Nj4trBHE5eAvhiXAXtapqKTYr7mG4uXZ93sBEvulXc16qSKgGYKIyqMrMHr\\\\\\\/N+CcHZjekswDOGgFJPEF+XAFbXKHimU7CaG+OTGvVqKGWc2QVp\\\\\\\/To9kxu3T6V\\\\\\\/3cXZkT1Sj0IfkWRnsgpJfUgnV2HHCLLNEzFYfQ4p+XsYEqPUon4zR1ckTpgKleFm6B9Yi4jtU\\\\\\\/Ajkr4rTLyo6OkRpXqP1Up6KbZHFl\\\\\\\/7f8EJF+Br46cDbbVzhfMFGpCpchc+FllCXI3NAptR2tLLf5Hcm03LcsH2wsLR0gHOC73ROh8lj+lRT3dzcao6C+QpZplm4a18y0pD57WsN3hCpK+fsyYRIrJ\\\\\\\/Hz5MZYOenQOWMEzYYvvya\\\\\\\/honJKKINR4LTs9DAOHKOnW\\\\\\\/5rPNfjkJKnnXukgl0zWSf9QhhLiLi0sjDDrsiQ08nBomXHnnXiPMbjX1zyVDoImh1YYqadEurLUcApxp2P0evu4eAnVJ7OfrmwACWjil\\\\\\\/FZAgkERYKn+rS2Q\\\\\\\/9Diz2WHU2yvkOXs51B8Qtxi5iAnhZ+EzCWuLPhVMwmkFXcoXk8yEfcnx73mJhzRuZIbUThU5615TgSsdEEsc9HwUp1eIiq7j1A0fSjshVci5w5bclqpSH\\\\\\\/xWlg5ngQp050bGKvUk\\\\\\\/E\\\\\\\/AmcikhoH0ZsXJYHulJDxW+io1ng2stVuUATG+PE9SMnNm1v8RnTvxck3bfohaiEpOqEWIh\\\\\\\/ckg5XLzQGocCveIo7pNvboPZ6k8BzCqOFe0xYk5TUUL0kQb8hyzTpY2Xc9B9XE3yOK7cv8F4TkCnXcYKHF60pgcC6bXee9Wlt8L51\\\\\\\/MD+4fvuLMwTBiWPCAxG\\\\\\\/MhcCmXNoxOUE29zi4RNJ\\\\\\\/lwzgRKcldYvYZS3sUpKdrTOmbK3B4qpswjAGiA+HFQ1ylYfLaPrhtqULJwidu0fN5MLYAg7OR5+vB\\\\\\\/87DBUmh4oeth2Q0AGoy+pT1+0YPLChDl4ioGjzCv88NHBr5Z\\\\\\\/ejZtGtR9T2UHnrE7caRiwSBaT3ryOfP3LILw3qVjxGDxLWqycqzHBo2D7T47LpmKBiHgGPC2o5k58\\\\\\\/kxIm8MUR1+rh\\\\\\\/XeXtWU8G0fwXmRth1oxSl6mIjy8yRn3jDjuqYYQqIXDWZdSmqRCsFf4u91DPjciaCU2o0BsFufP5Jx6bHY\\\\\\\/n3MAfe0BD6Yd4P0QmvqBKCYt+WG8CL8qia5nyP\\\\\\\/Yi5NhtQIMvBD\\\\\\\/npCT6+uD673LLJu7KREtvkCuumBn09R88waSlH19+FWNav6dPhZT8neExWzqM1MY3WjxIhnPqry0rK9afO\\\\\\\/U31ZHQT\\\\\\\/5M3pq+4oS8mGQJJ1bpLNIo0lCnil6FLJGXo0IJ\\\\\\\/iDdvWN7V1imQosVuzqgDdJcrK9E\\\\\\\/CADUYNp5V0JJoaWYo2nvDBv1Hml9Pc+ofmE8cff3hVCYa\\\\\\\/OZ8sXkxGslKFBoTbpxVzuM97ghPZ0kjjX7j3cHGWxetEvoz+0Q3QP6IVvaaF2p+v28wltw5pJQkAvyHAWlJBVjrZljSetuwF8m9ddxxHc3Nhu7nx2OVD6uSMi7ZjRVNc8\\\\\\\/GlB1wxpm33OJeQitMwGS3Z7yvarwa\\\\\\\/BrEfyeeBdqnxkieLCi5M0H+z6t2i+mdQl8ISCrpwAyKXUjW\\\\\\\/W10YaBWg8ptq6y+HJgR90JGFDzSX88hmqxBRgfMEZ98ZtbntHPvac6VLYW1yuzZdOf7S1VhwVSmgmsyNGd+WaUdZJj\\\\\\\/KhpqDsBSGL7SRbl+ZsEWeDQxBD4OYvdtL+EwU5kjULPKKq0K5zRgjv9orCud9yLcVFjKXR5LQFgbBtWWxzn6nfEG+w6Ybf5kbJfq\\\\\\\/T7KDa4aR7SwuDpn\\\\\\\/HvSEAT3scRRXGxnqlbfCctxOufltRHE6UCI8fhbF0YlMRzuINzBSNF76tEpQQU7pAJrSwCjbY3r+CCdwTP4r0XR9IObEwo32IMEvdawP+CupZn2uIREyty24Dofzy03o+1qWWKoH6KjSujRjWLNKn519bvatQ0VFlWEHFBsjapoeSPM8gPXkPYhaemBia5xQB5LPpdbvmwEsXeTOpTSk+OyT0b5jJ1fFnX17tvKIguzbSQzu1awFprkAksmT8EmTM5+XyevicwYR5hcohSNYLiOD\\\\\\\/jHOUMgQ7jy\\\\\\\/KpaqxbCO5jtDfW+WPzNMcN\\\\\\\/49sgYnSLt42GjsefMd+glc0FetIlax3ZHTIis8RUbpdkhr7DZMPlt8vNgjDrDXmQQ7ETOAow\\\\\\\/xgTfBzC8koRLv+D7JAYtewmt56SBNwJclfpOORFIbqTQYnS1dG31LLdgQ2wNrquil4plxhEXRNnGzb6\\\\\\\/YJemdloYUMsFv+8zNmEjvi6TI7RzvYMKv8SbQXI9tFEAUkh9DVnjaDh9900CQw+ReHaziQskteM3AfLBrhVJYbEoi0q\\\\\\\/lcseeD0CcylmM9S7eIP0+SlvBmpxjPYVD40fVYc5TZHa\\\\\\\/hbO964grm\\\\\\\/nu4YgXTiRFPZ7p7X6h67bXrCefpk+CWgbZQyCP6lhj6ESoM3qnasYsp4VHWGQV0q+Yp8oeWoCIZm\\\\\\\/ungAAxzAy4oM8au4d8Yu5EIE5nAGxl8oMY12Gv55I61DUAJREalOlS10NFi4S8oRkj9fslcTb2TZQdn5KnKumJU0\\\\\\\/ZHnLaPz9BSZU3nMVBa2I\\\\\\\/MSpdBMdKDSBlKxPdC0zvIpqKtpab5hKtYULyLNB\\\\\\\/Hp8Luov7+lhxGTw4\\\\\\\/\\\\\\\/LZOV2bKk0+prF7F2esA9Hak4afLbJsDqgznMaqj3OJLyGw9haECoSIKbNZUeDUhjHXfMm+4iRpPrECwYy9t33vw\\\\\\\/eEBbF0h6Bd9+YDFGzh4RctRG\\\\\\\/96EVrC6+MOqtn1iy7axJWjFKv0+9myEympj1eW5Wym9tddOS5yUk49+R4ZoNQeJokLHFnhvmUe4z8PEbbkx2f3Pi9IS5JJ24+l4RF9nlgzieBSV2SyodfaC7oLaZr8X58buaxixd\\\\\\\/WKE4c3OBhStI4THz1eOJSbnPMp1sEXq0c7pljM9cHIKLx9DpbibY819uMadlCBZC4Nk9qgj9prNaP+mthlONVfEcTjZgQFtpxQIvJ+xSM5hJGhl5AkRpJ2YLC691bjD2cKgFNZTAVGv9dyKFeQYtoGFSCE0ShxKCxv00UB\\\\\\\/zeTXmnPgFzcjpeh7bxmbeI8ilJwNBqv095\\\\\\\/Eeowmppabsk5fPeuzg0n9+QxUkIi9y0LcRIPniuZ2jc8XHolsJdHoxAx0+8qj\\\\\\\/\\\\\\\/iPMgVgL86jL85ZuzcXT20baiJaZvp5Eet9\\\\\\\/NasNZJBt+uQcJbO58DRWiGWtGeKDaQSfihU2H84TdaCvmxW7h5PngpUwcw25Z6XOvlk3mL15RIinvMqvroOXB7EVyllganOBBy\\\\\\\/dpENndWI4Va8wrsLJUeN4xm13EHlGFyQ+nVmAHt7JVbNih+vVWKMcyS5WAxuXMrHfJmErXMvBQ68Wjatvtu9FXAXid4tK1V9bxNl1PPNAq2edTQUxPy0iupVXKm+5AE77ak3VSWfUynnXEy46LqUYL86FZRveq2pID1Hy+TV+pDXIeWN8MBDvyRbie97O0fWnybBE39I2XIfzOdnp+EjENxoY0wSSE\\\\\\\/sr+8Ng\\\\\\\/1kt7g3aZP\\\\\\\/Huicg1bxcmvWABuVmNLngjZqLYjK2F565gLK8KqN1TZebK5qDLProc7QF8GzctUiUXsKO1Fc4KrnV1xmJ65lAdXmRN86RnVFeKU604BVHDfbU82QfF79Hgo7m2F8S7PB5sGr69pxtA+0Pwj3X7t\\\\\\\/b2yulp7QMJr1okWn\\\\\\\/HqjZyrU1WdiKUC+m1TRHIHOdgRdeYSATTOk891hSNWdPO8g+m4bjD5HhgtgFOjrMJP\\\\\\\/sV4RlRuBWEcOmxf8U1ExebhSQBcSUZp6fUvEXUnTxrkewFan+2UyhXYrrf7b33ZYL2lOLm5aRweZg\\\\\\\/LwWEL4piwAQ5QjhBT2hhd\\\\\\\/sSFkOzCT15CLZNIp9OiBVnrGDsJaQKUSTYGkW7y5pOx1eMf3uiPGPK56JwiylMTGSp+qQTm2RF9kssnP5qfG0qtDkfj1OzHw2ntiz7iivShuXwJFd+R63cg946y6BG7vlyai2SxLcO2f++KHmDCm0wK8tdo1CmVIH46EHg2z2meFJ0uMoN4FOFr4PDn4C8bBVhbuV+228+dWPZwuoOi4a4Eow\\\\\\\/NKAeM3oqPKEN9C\\\\\\\/Om\\\\\\\/Yu\\\\\\\/kQK013xw5oK3Tib44GeAelG+hobYjeMZWh2SZZ1p1p3BKneTq5Rj++a7SBC3CccHbPfva2mowNjIOEpWRhXIkiigOZYmtCfiEgAbcPsArPW0moxzowgNBEp7wyV0poKHUMsVekR8QS7CtAkowYPo4\\\\\\\/bApZhglUbUqsOV17wPthxjaEi7c\\\\\\\/RLQdUDWv0lEElnz9RoMeafBYHuGtHl+wQ9KrRz\\\\\\\/1Bv3vN9AtuTUblqGDWFHkO1r3nFhSSs6nmPH8m3X0wu6ntNmEVLWtbhe4nRzsuBGWYt6cXYlvjrvbHnIzeR4l5DWdBG7Gb7toBryk2Ruk8E9NQSYQaAsItqS\\\\\\\/8xhoJO7FmGb2IiSdt2s6f\\\\\\\/vXyQLT+q+UfrxsdqexwwOBvb6vVwDDYL7xeIsW+dYZgjYJaz8vR67\\\\\\\/SisOTOwujsAMNlWO7Xp4G4OShWtCw0dF\\\\\\\/H59ymmOH5sGqfG3HNda28zY3rHBtQ7+ApbUnGe5ayjNZIlikK4RJI\\\\\\\/H\\\\\\\/0xGH9m5BidUg+7Q+BjKMssK7JFcrwBNhlrz\\\\\\\/f\\\\\\\/oIcSkGzw3XTRWbajzD4mJPHoeat4DD97QMYlJO593UprG+hjm4p2f+7DmH+J98IcmiUUcaJH1N7uWZOdSxT\\\\\\\/U+J8OqWWstFjnOR28KWtbUxGltFhzdT7ZaOA6J342\\\\\\\/1uZWqPPgynG7X4RwHTydyMQ1X9gzBNV1uwU0y5o9Gc8V0hVG+oKKsx0O78dErjFE68P8CYNrrSl4BrECb7hd4pT5PXRF26C45UY\\\\\\\/P8yTeT2Ny\\\\\\\/n2hXcXwRm56GTdtGISwhgKnSEXDaqDIsND5l1kU+Awdz2ywNiE18LG\\\\\\\/9vrxCfam7dnpB+rthx\\\\\\\/XqHzRy+234QRtdtaA0mIbxFwVn29LLb9OhHUBVqFQruhnmTXqfsALjuwOMyud\\\\\\\/sfsLBYg96NGXxmt62uSl+7\\\\\\\/QqEaI5lNClDhY1yXh96z9uMc+24emQSIMVw7GvAVqfozaWcqcP65SFGK2Hn9vfKrtjNYr2tACGIxTR\\\\\\\/ci89nhx3m7CgX1o+9X\\\\\\\/nD2jpBK0lnnbYjj9WuBcjiEUi2zL4Xs0kmFYE1zUiQQHGQIs2Nh8Kn5YbvaTvFTWwH5qgwoFYnTJWWJPTJqstywxmCIGuAWW3ObQ9O\\\\\\\/y488PxABaPQbJIRe\\\\\\\/Rt4pm7IpCzlu96MHQQbU3th66kUc30wN5Fi47NMIHPNCOFMXNxRAgYqENuHhjrFF54uYXl3G9C6Tw4n3ChOfUVuB\\\\\\\/8z+OjnnjykiqrtDfTI9vj1x+RdFB3L8lKqtokIUphtrpEj\\\\\\\/Q3MUQ2G9pIUpStfzfxU1mlLm7c3\\\\\\\/+F+xnk\\\\\\\/q8qx3FAlvWgBq5uAhzQfRA5\\\\\\\/Gs9Qm7ss\\\\\\\/Qj+Bb5VepeHVhjHSi+jfgSI54+46MFZcn4sLswdQsyTB7igZWZBze43CzSWNs1lo5ijMD7qNnIH7\\\\\\\/xRY7TcVNd\\\\\\\/+97uA\\\\\\\/jiUANIo6opYnAZYTiI0xMm2pfem7q+1CQsxt4RFQ2SRylSRiD2ap1Gu4X\\\\\\\/qIBeHb+eRW3lbqo1EgB4v1odGyrNl4GnXZKTlnF87zCdy\\\\\\\/2zm+03egi+vTIWsoDzLHVjY9dIEzqzQMDzCiPuCkj+YwkCOgyL2lMfBnZSgCsZH\\\\\\\/AhXZYzvgf\\\\\\\/xBGjAxLtVx3S6LLJcsBQpvvYBPGWt8AFYFpQQSVC2JfMckA4\\\\\\\/QYmHNczW5KMdO5pocsYgBj9+7VSDDPfGrhoQKQ0\\\\\\\/aANGIXYKMD8BS\\\\\\\/XCWwK5RhYRaOOFOpcK5ZAGAIS+UNJWl9TEUeBwSLmrURDjmgR6uXAx2XKSqSp6AThm7HExEDollR+I\\\\\\\/+e6eb3mosIbocuLYmlW9kRwOXj7sMwAdETjZZYhM8g\\\\\\\/UK7ajlyZnFiYksk3coAA8TR3s5u6xfpKlJtPb62oIkqAJV0Kh3TyXVVYbB\\\\\\\/nN9e0Y2nYjXpHFJ2gmgKiGYIdY9mxlVvQ++q8KxQAWi5U0iKT\\\\\\\/3HbPXN6Bz1QFsKy1tFX3E2QMSJpv5uxnan6Ugdot2xI2WSKDD\\\\\\\/wY0mrzOEzQ3DjVKEq3bh9pmZdHuEoWIwxj2toYU1pXCZqE0lK2itybT3e25uDKBUbJxgW8tyI\\\\\\\/b6qQaLLrDi6UnM5U1Af3rnhNyIRDSecpzst3DE+OeGLmPVaE92c9lJlQSasNedXPdbfWex+u5+BuEnjkqX1FQZ\\\\\\\/LPDfVMo2nACgmTt1z4nPtYBnOzzGJFkwDrlSyGQ2JC6tkAh0J6RcmErNUoKFAZo7KweuXYgqgFH0YWjWes138112bXM5ZrgfHZYtfXIG9ak20VVo511qY8BUC7xptpji\\\\\\\/KBowFcMG+HCiQs4122HcVoa5zZioSw2hmOg1Roc\\\\\\\/efUvcoX8whh7bxMI0bev0CSZDJvnvb1v56gxb6umbNxvC8uFrX7+sU7N3JYMXmACjripHBeFMIUV07IUvCYkElSfi8NVkEoxrUjBV4FBMaXi+rvfWtkJmN1+jLUfjDAfToA9nyVcGwqsHUD+yvpilRPQNVQ88soNva3gexiUBtJ4jyayyZvM6mEDe87yjP87kYGoV5S+Kp4NiEt9jfr1k7mZa2YPy\\\\\\\/WgwOILbUqAbUV9KEcg\\\\\\\/+vHZYT6jCTkwCP5KPfWDE2COU9I0WJx02c+dD3bD2YVSLe2OVp2kE4jJqfxBr\\\\\\\/HNsDCboezSvGFlQEwlq4JUZehC+luLmFWn5jWRt2FO75NnBBUfMzCVzZL5z\\\\\\\/cQTuRLH3\\\\\\\/xReIPMRK1o1SK+xGatDhsDRev9rmKgNKDPFVX8+x+uWQllf5vmoav8gfbh4fJtxDIBOiAiphjR7gvGx\\\\\\\/Otjr8yZfA9sWmdu\\\\\\\/qq1U+4aX0psneCh5zw8BysBG7WWPqHpwrs03jf9qisHZXgHVhYfHB0mMDJA\\\\\\\/2cHK4anyYb3SzP1nNZUTseuCp3dnTD8wxXSoXzziQ1sGoyp7r7VftCXgT0UUAAqql2UOnRQzx9aiXaBEbtaGZesZAMFMKWf719447fR6aYBfPQLnzW470aFr3wA0R+yC2wW6HLEXZ0MxXalJ3nh\\\\\\\/JsHDNhx1XK8broBup89XuTP0b3Wglz7vPO6p3zH2gCxnC5+7tYoG5BvlZnrDm03VJ4HODTYYlzq6GmP0dLABfrZH31dL2mkKrUDl9EGoj++sEZ3Hb7NXVZw6ZOtSVOUJJFB3Lmch6LejjArBmBts6hFy1cyIrphaUaOEJHUtVBhKfY37oeARfhrQeqz9eFEM+MGDtGBtBzLd9FwUQna5JLcfszTa1YL50VIk4\\\\\\\/sn7WcpaqMV7sur00bqZ9tlQST6W7W24oiy9m97xi1s1+Ymrxfut2ZziPZiP\\\\\\\/m8wCFLhvTCWoZGlkCU+vf2uuoSFY7X5i1zd7MJbX0qZnnkwj1N3FP0fvudNsevUfPsvWElaijD4jSvwKlBr3zok9XnxgXazC+eVdFm7J9U9xrAKtu5BWRIpbWpTi71k5z+P\\\\\\\/XeECs24zLqTdy92rht1i+X47azuqKZcOZ0C9i2ZV9juGjOb8WQ4zB3r44yipgqob25U78fBCKa9ijnDUWecrucUREy1JMXeZWBQkA+SSpns1vYPxUG+Drm4MJVdMukQqqT0+332yY4vR0shRodySY6mnAR0IqvwtyCeAlkg6FlRD\\\\\\\/lZMPUCWauJmmQCq8CTjS7G\\\\\\\/hfz317TMD8CToytJqGcGJqyNfja6fMcphFsGfJq5La7fSGGn6fM9EMb5PnkOQ1XIWGMUx38F+Vd9jAFWZIKXhHssBDTN6KsasRpkt8ycTf3cwruxKdF5IfmnSHfmjq6kCdvN3s0K6Y3blR1nowbljhkyWYwRRLorQiCpmVRxNihn5k5RAINktbdC\\\\\\\/mNalji0nM33jKhBb0ovpjcTuvRI9nP\\\\\\\/ZtRBL0tEkDeSXGu+6oGX5qzIPU+ygoVLtWZAUtRKT9W9YL+3ynL0\\\\\\\/zHUHLU3c2ya\\\\\\\/sKLZPgq+Zx32AWnc\\\\\\\/9xGQcDmrAQL003q8Rf71eDeFyCRqjI3CiF21MRqY18XAX4mlAM9bLraHhuB9yNcPuYcDKVCX6ggHs4SJCyBBTaYTtDFR2Nc5HTN+c3uZOEyGC0kb8hw+CNYCabDve06Z3e+zPe+xqDh\\\\\\\/meibhU6LPP1aa0SJOEZ8Ff3RJzlptyIF+Hw5ILGdIW10npBrabjdIYByeXSTZEy5X4kPsql+Xc2pdiCm0EvA+oXoGyadejTzYBtYThaXmrDUWENkcEoLTU85b4Rg4AJPoJ9ps8fVeEbj+xrNJDAgTBvexo0BR+By\\\\\\\/8I7VMRWt78APj6pPpwRYXQ1XGMkyIVZnH21bJKrNK7qpN35+W6tfIjBc9SN4jPfiJm2J7uK6TmlJgpF+A1A72ng2DQWNlsMq1x5wQEy65+4QoPlrRSRFhHZbPTrqMoNO8NBoCh5PddUex+aTMga4l7mOXLj1oG3wGKSsQfQFVitdZviwoF6BiCESoD5zqbG7fDU\\\\\\\/7vYcue7\\\\\\\/ynoKI69K+pndEvrNIscSXWCAOrisM0uWEZkH8RUEc14OEOaYO+GRAtyF7jEpHowT+Gtc7YYLyBVObt5yiSYIidrh6Fk15gly0oMtxvUiXEPEMmQ3I1R813jFKyGSsoBTwKx9qmqybARcKdDMCYNatt\\\\\\\/iZkybiWtVjmKLx4rbcLh8ne2D\\\\\\\/3hzo\\\\\\\/OQWd2VZQo5h5FPF1IyQhXV17FBMl0IqeShWKx\\\\\\\/uoBXOMFT6AsltEYU6kpXSHrve2a7m0AKMED1spsZz+rdIDLPr0CUN3IOGJsPXVTgo0saZE\\\\\\\/INI5zcU4iWOuX9kXoGaYenPVfihDmCGKl4JuWVoQP6xq1hH1vto9Qcn7uMETcaLFTVMXMXRRLA6bL\\\\\\\/dXoRaotbCzhlsh47O16mHB2YntH6s2NpzjzCU4jtwM6cA6VrR49mfWYWPNGzcxdvukhmhbaAhdpMzmj1H4PvKqio5fH2i8+rVQTkdB6SYWAsZu8hWUerR74\\\\\\\/6TtTt67i6i9PqzbXRmgKUfqsu0oeyWFdyFED13IZ5PhrBl6msZHDrnGXWPGdMU4yaZVA6behCPpIdqKRmwGY4cJrNRhikHaAR2yn\\\\\\\/xa7xGh1KWGARX7ojMmxzKM6t\\\\\\\/9MKZwzazBrJ2TeSjmuQV4p8yuwKbydD0hDbJq1melkof2jW4V5LM7JpJ+mD+6K0ORubVxHKiCd2z7gNvWJ\\\\\\\/OvGALh1wU5x+paWfIbMBX0hHEq4+9XJmn6bjc39WX8gT9DBvzs4+kTQPAfsifZeo4sXUmyuGcKnEsOv35P\\\\\\\/tKk45MBHXWxVBVtp1SPpjd4WWhI\\\\\\\/yioi9OfOjz7RlvmNEr0fT690HRJN27sP2iR\\\\\\\/P\\\\\\\/n+O0hPjc6Y\\\\\\\/Xk88myO37YaY+lS\\\\\\\/Avb7b2O93FwKzAC7P9lO9ANddd9uTbhEzNAmjRJZ2q6L3gASnopOYn+ws4CkQ9IY26fUO2cQdWyXb3NouUl5u2OlXu0PFvuJMwRyq0TiAZvqMEb+yaS8xIygRr2WmmY3UWdo5fPEl3cJn\\\\\\\/hl98Jdq+yKjtNiU88ewnNgYt9r\\\\\\\/FCMZSoQ9ZliOlBerT7HZ6sAZ9eaD6Ofc7ZkFtqqJU6XwmWil5Mzu0X4f4hPl+uj9AtIhhelfuePre2FO4BzJPb4gdTMnXnbW51\\\\\\\/npF6dneFhUi8RZ2NGBuwhrwBPb44oZ7j9FRA\\\\\\\/J\\\\\\\/8LgOQwbqpmSTX9wxd6oFVk0mM07AirngOZDpIfeqLm0J+xKMOMVKJQNJHFuxJur2BAg3JtemXe9QgvLiVHrKAe8xngfow6aA74yOM1g3oL\\\\\\\/uBD3pfSACd5Fh6mows8BotTeKiZC18Ncsus8mg09aVmJfI1sy7yMLYTaF4OFsH6iWOPgEzrkbbGmD7K2sG3iN+Y4z4fXjfOX1TitD\\\\\\\/BJeDnQ77ThAb9y1V+ZnDcQvxx1Rgid+cWI+imn4jjWoKy64\\\\\\\/i\\\\\\\/6+Mp1UH6RYByc7tXQ\\\\\\\/3M4psW3TUZg7NZGlyn7VuAlVjP51t0Kj751YEpOCHRYiHJgln7DnMiZHmUeJgew1MP7s+4maH2ODrByp\\\\\\\/iE+jgNcH\\\\\\\/wNmba+JvHthijr3+tB3apZ3XJHX5RD416pnJ6NdRqI0YArIPmjNK25CBJpEQK2w9Y3tcJhIMH9GyAKlqZD4sKj4lh5V1bpgxmurBoXz80gZ0qlBHR9Yupbs3hgnCc54AJcqTevuoMIGRG9\\\\\\\/r5ycouRjoA4O5fss+nFrQ9OP\\\\\\\/7ybp4mIS+PWUDhF6aph2jU5LO7nkJJnv8xNLRlIrFg9n11kYVpw3+HgfAqKQC1LC4ghrt+qbwFkSvjKZiP8aonS1vT\\\\\\\/fBgxke4CIvDCcBHQhtoZK0LE0sXr5s\\\\\\\/4wI6E7CmTYTUuTjTu8dPVPamczbke8sLI6LHZgWseu9M7A5wgVUTIJ3UGtj42j1+mHZ6gyF8Af7Sh+WgosYyXGR+sq0ep0tHfd9AwLxhjUQwgkGqRlpFMtPvYUQoZ+48y5OJzGEqmlPCMGEQdkVrwqHfKGO+xw\\\\\\\/gBSIw2u2Vy3BxvPTr8HbNJ\\\\\\\/ekNybjUAb2LWj9aMi8DdnoBjIuGixZ2Q2C7zqqwW7FlOdl2Pig7Y\\\\\\\/rzDfY3dC92be+ie2TFfVpXaXlw83eY7R68OrcO+gOqZjZachcWehWZRYei1gy\\\\\\\/nhJyvUyZ0ADsaBQuL641T9\\\\\\\/VB9Z\\\\\\\/IbRNF\\\\\\\/focHmR8yhk+wWt8\\\\\\\/q4ftk7OAeRvVERzLhEjpthFuJQzAdaMhME0hR7jG8ckrIFWTwZ+lgqUv2IQZZJTmlAVXKVFtNAiQczdupfESz3DekbPkJRP21E8+HBvHn0oQf6fOqK01xTbExIx0+XP3yczUebHlp6\\\\\\\/O7PkOao1aHuX8s62UgJ\\\\\\\/Kh9S\\\\\\\/RzUOXG1IqcANL8D+VvSisF8Ucbz9D00dma5I3MdLv1rTSez\\\\\\\/zoAobeWcOZDspAOAfIyjd12eRrU2yDOrKJ8mZXuI2SwoJc0EbmVvx\\\\\\\/3KyA2j9MoruGJx39i0SKuehC4L57XXAdt9TVUljvj06H3O1uPRjprUGQYTywLfTihQpEZtkpt0X5FFqggw1p2RRZRBFZBT3AEXPgSXi3Ca1WasTv1U7ce5xU9uVkqDweFF+Yud30C8ji7mFnEh\\\\\\\/DzcLpqo0yEBjG2oD9kbkzw8qkSQzvFWy8h9mYY\\\\\\\/2N3\\\\\\\/tOPY7XeBzX0\\\\\\\/Yp0blnf0PdLuG0GnwXv226CsMUJVlRgh\\\\\\\/1r5hgrgxjoMokI42YOL\\\\\\\/eH79dZwygnoreqUyEDWLSNQnVbqiwgi5tT2P8lo\\\\\\\/2CPIsy1Y6tMAFKCrh3045iNi6GKkR6c99T\\\\\\\/jxnLCS1LDjjdj15dwvq5j19uNEnlGscjg8WVkub3Y3leLKwO8Q5ZAosL6Nj0tqS7xU6PV18JIrKtBbeO6jvgeZWIaq+IE4lXdMNZQSymBZ5XjuoReXjHT86DKr+QHQfWU\\\\\\\/hQRcq9pGaS3\\\\\\\/ysTmrJc+me4hHO8wT20qjLMFj2U6ig2pXxkp5bJSA3XnNAfO2Bsn9JpAsGC2dxhkAXqSWylUD1Wwpcc5dVUTzhpYbKnLetW+WZ+B6aNUIaouo1MLApBmygEdsOBqdR7tfVuBZ1mXUdRzdblTX5PLsEq1FSdoYmZ3mCW+x3JIHBJdg8lLh\\\\\\\/8zTocxpVnMqcjHfhjV1PT5bKuEpv8IBNLi6Aku+fWpHJiJNFaD8PuiIpLrV5PbhvDNao63CkOefU8lBn\\\\\\\/q7jYKJY4k009ssRBXYeFNloBbbO96uDuruhx5RUupOlbzsjTd\\\\\\\/7FJeX9CrclZSQAWFRSMwh2JgFFUfELw2\\\\\\\/rWewybc5JfchCpk+QJMZkdla9Jp\\\\\\\/1DF1WRFARs93EUdqsKqc+dlxs+2uPOC\\\\\\\/BfIYht4OMbEQAf8H1\\\\\\\/+rVL71iUHfqKXmh\\\\\\\/Sa5ISxwPgCl3X4iPz+\\\\\\\/MPmkrpH+Vc5uadWXXkJwi1NpUQ0HkfvZUQBv3PdbVFPuwOodUNN7qM0nhKu1074IjS39PSh\\\\\\\/hWLfsVGkXZ7ElHlKKY7COWpwvzgQ0hutM+AK6RvLMABZ2olDlEuvC2EaQR\\\\\\\/QMnh5x0uLQFJBizYeNxX5fxubowAPlWJPa\\\\\\\/i945b5RBDALbbAsUON06yAGjJNfP2HrG1iY0NbXtVb7U9ER5qeFWrcLFFl2x\\\\\\\/pv+LUBbgvs6bRRNRqNnBKLewob3mhOgzF09KK6x4yDyTA8sobgAa4i+CqpdKUIa2FzGHGe6ZBrjI8LSSlaEKHZ8UM\\\\\\\/7FJlZZbsiYnr3qYSjo39FKxhAxDwEfujtjtmylZ8vrKQ\\\\\\\/Y5Xd9E+\\\\\\\/A36DYRCvM1bvgUaW8NKBBQV1USg1b7pkO5Cs5b1Dwy2RaztrMSlzwM180t3rhRHzuqQoT+cTTdXxNZI7mNbqNh5VCp1N+5l3XO9jvnfC8IS+Jl3bb\\\\\\\/1k58PDNXCsIDQsyLVBOpq0AQOQcOYQAHA+5b6pHOVnPwdWZRNekpDyfajwfseKRboCIzcYISix8PYvwzDOZ5a+XhjEjDE6Rm8z2bN3HNpuNarJNaNB1mWIbQ6wgpibeTIa+0z\\\\\\\/rrbj1jrLptE57yPvi9EqByfbPH35qosdlrGrdlxpm6jtlhWL\\\\\\\/agQIh7RS78+eLsAaaq6JQA15eQC+d4gEZxEqmSAMhOCuEwGqU+qmx4v2tBtUed6BtCGnPO8s8AxEsBGsP8tltKlZYfqO0LTZX517e5Fu1wiwWkUwVa8m1dCL6Qduv26pteOKm6DvWRYRRDAEEPbXykMmDHqINEmN5WCwg8EPv8A3nfwUseosWpdDjmsBkNR\\\\\\\/QpZZ2NxeZVIKdvDMDcl0DIu1BlA9qylUfvo6YAcCkJyJCFtvKV6kj7BV9xxOQ3\\\\\\\/jBMztTbC7NQe\\\\\\\/WPTJE\\\\\\\/ysQbYdycVNWaEsSrJgJYaYen0Mj2+F\\\\\\\/aDul54sRYjpgL8DeMWZuBsvba4K3g8b6RQDzsyUEUN+JMXXeFbc6ucfD8BRY631Ai45EzRst1uKBy\\\\\\\/m2fKiEatx05\\\\\\\/lIwA98qg5xrWbgz8JRIUEqw+PT6fH+k1zYAzqBfC5VFrVfaFaQ0p0I1EeYK0dYUbNGv8gMhCCIUKaQCYZZcn9xdCkvNl3DrezkG9iLUxhuMN7ViY7StBwW32LiNn0tgF7gv4LhoXHgQ\\\\\\\/B3btZjDk7vMlYMp9LKB9lU1j06+zMh7CrMWD2XO7P6M0in5k1GN674vHpDRPOrtBKMYhu5Z+FWfLyL9AZCvk0yaQE+h8cgYKPH4UprhR8shEynlOUcZjGmV8gnrjvAJGw7X88IipMqtdgjA1icFUyTI53ruvfwQy+WWwnNK5TEC0Ec9r1HRpDtvP8vdeyG9OK\\\\\\\/zkeA7Mp9lLnsvtjK1IWXXjQ6HegrkKaWzn1VhotQO6XkxDKvXPz1YFdEX74nxBv9z\\\\\\\/aQSD0vXrR\\\\\\\/HHIcT48Ce98fXeN\\\\\\\/ejK\\\\\\\/CdW\\\\\\\/ru1jsdV6FcMx1xooOr3wmnMwP4TXJthZ+EmviNstBE6T5TC7PcTHJSC3bVJdumCFb+w2FSDRicB7L1b\\\\\\\/hpRrcUBTvc3UOoacQzI0KbabB6hFkQPYsYFYKO7Hl8KB+c\\\\\\\/W2tR6xw+z032YSlvSMPmFxn+Q0BGOBkkrx9dLXNVgibFnL5DaCgf5npywUF7nOE9NcVtEUlSVGyYcVpMpinPriSer\\\\\\\/EnS4u6y0Xj9FtzGdhnta6n1eLIhu6sDPK6a70w9TRiQMHaiHVszs+TpOaBhRgLwrrvNeA27z2FDiNngjXk2oZ9rv3u9CKLv9Q4pIM6nefpaW5rBXJgO5Zr51f3D3J6Nvq0V7r91Y54edFHrSJL2RIjaGFEQH+E04XcnR+C0GF38ruO8l71RyVy4rxRoqhRyQHaxcEb8HaOrmlpQ3CIec4gAxVDfS+A4t827lkuGJJb1xe5kWcC8rbaciRJagf0WjRGuZM7b+Cu\\\\\\\/Z7\\\\\\\/KpCkv4l4R49C3haDY6qiDNyr80YaxggeteFTsX5JDkZ7XY6flEIeCydGOJGudOwDRQ+gYkCuhJz9LOTtpgxINoq0C0JQ5kcHWsfPyKNrpPpyOI8r1IsVxNakVhAymjziJd4FXjFw5rNPCK3QRrA1lOZGYdPHg28bn1cIuLITU3WF6PSePgHFas6pQzCPa4MW48Iqa+NL9ikZnVwyJC4EU8cpeKumcBPQ0hp8wr6C8crgVDmNfwo6TY+HeX\\\\\\\/we\\\\\\\/RqtS9rhSfBhbUuAVuIOPJDtu8GstvBo6\\\\\\\/psuBy2REaXTFKxmcpM\\\\\\\/2LyZ+Ngz1xCSphVos2yL\\\\\\\/HbUviXrWI2pnE8DRus5djOrGV6\\\\\\\/S4kRIBIbkPAqfRuc7OedjNRnaETH9WCFJ4XLAntt7VIl\\\\\\\/L+bMD9ZqE0PsdKZvEdqrc+9uONGBkXifoqLS++J7L\\\\\\\/dhBgNPYAVov2SQzGfrPzLH71nn1t5DxJHv1DUtVC1MxLO0lFetd3bkSQmiIznr9aL+EYnJKFazCTuJdWlpdr48fLcyukvhDNPJv7coT9tADvszclUvciVX72gDhtyg03ZdSiehnChnk7h3bEGHBdTc06icE44ih\\\\\\\/kT5I5x5I1zETYq\\\\\\\/fe0yJgyqey7sejw\\\\\\\/zCJgmj1HhjU3E4Ptbg9GeFM3Q062mF8eOgU4Izwczuqqp942rZtvLj7l0dxFkWQ\\\\\\\/eEal8YK6iRp9eEIyuF1reHKIvKNWO1HbUDYbXGM+t4jW9F1q4Z+hw6doMevOWL98Q5zgEQc5d56NsGroOX3NuhT3+bhpqiS+P1ttD\\\\\\\/RDQjfnSbtZZz4sRDd+a+4IwYUfDdl6WWy2lY2\\\\\\\/K7b9DTfeGWyECr2sjkltkoagmg0Y6OVOgvTZL65wNbXw+p\\\\\\\/I8PmuuPiIjgxz6jla52tdUVaK33zwI0eb0rwS9IGXYPyOGhNckawZHWaESUKmlk9dmJPGhRjWuOWIJCUckMh8LvN4qBFlx2a5GfSPqPpJUFc6DD\\\\\\\/wGKGAUlZvlz3x6qSIjlngcGMi\\\\\\\/I9CDUKqMZbjMHxGBgzCs4ptP7U0wtfai5HqXE2OEzhz8uBgarUerSb9Y64gEWhFRIK8mmy6uikfHh+8c7DhGxOgC4BYzNu0DziPmlkdxulb19Rm\\\\\\\/xzqF5wMPfZeBc4rMypy43MSSUiVYjcXPd6gk8ZcIUQT2Kb6b5xcZQ4KpeMPSWiRZgP9+HRD8O5CoooVUiPYCPtneG7gQXc52v4MGl1wD3uYbtiSpxTk0f+I8wQ\\\\\\\/laet8zDD0fDyQJHdgEd2Nw2x1npSWO\\\\\\\/DFWmvNzM6TXKSYajK5aB21hWTeNNdD3UCJ6Jxn00DgJopVHB7F2n9k85+wH64jpzawo75ksZrOfpUUZYUGoTxFy\\\\\\\/mBgFrBoxH7Z3B5xh6VJJ\\\\\\\/5Fl7VYd2Fp8wIsymI3R2JXfNUcI5pF5Y2ThViPgRQ6l9liPp6G1NeQPjuF7dcRIupC0s+WX7mToAiOucIW8rhUdhNcOjhwHWBVC\\\\\\\/uSPLrkp3xh4JAv15lrnH4TAeYyA7F0tJwIAnWYCKewpHjBewTdirJr2Aw8ZAcZHluwwf9Yd0ESKHFR65THuKDIpy3qncK8bUp+qzS8OeIiQR+LEx37r9Az9iJOXItD8BBBOD7Y2GgHBEq2zk1LtyrKj6141nF1NSpsdsS3cxmnnuXMFVHuHF0RkvuXy2riWlQUwBNliAvdevi4usc8Jci\\\\\\\/rXG0Sl4gvnUl2eFTTrduLbHcupF72Xkz8U\\\\\\\/mlRt30RAjA+aK9P7GG\\\\\\\/ErqvjgKaQmhUpe9Y1iA2lWje0ZR0lpeuhixKLpkp1lpqJqSZ4fbhJJezSKhgkBEWKvBB34P3RH0pLB\\\\\\\/CN2wezut\\\\\\\/RcrjRVKFFdyGgE5H6vur40VfZCHBcmNDTylIlEL+xrUdDHn\\\\\\\/lE1UCVO2wKpo31M6dk4t2i5lDqrZgqu7rAZpWhV5rIuQSw9qbsjEych15hYMaDqO5+82kQavAyVCEBWorKPe+ATCFG4pVdtGfmanhSewPwIVICuXh1PqtOODASTgn44j2\\\\\\\/wm9Uzhg8SAykz+ZhzHi3d+MpVOM90TG+CTQrURh+VxkFa6NqSd8MVMXEL8qU3CjaiKr4FqRJ792sW4EvAzhjhM0WxE2xRCPiSiZMkcrFke843dJrqjRyh8BhbAA3w3DpNSskBqmKsk47nIMeEjRjmWv4reyEz\\\\\\\/9RiYIsl\\\\\\\/2zmHhuysKWWGaHPVWgZwmFRLinfo4b2kd0UL+IGZsCuTI1ABwYN6IY8Rnion1HkiakASN\\\\\\\/+iFy91ZNEypxnFaB9OM\\\\\\\/FdlEQu\\\\\\\/vrteoq0LFtsY34\\\\\\\/jR9zpyiclG9c\\\\\\\/\\\\\\\/4FtB55\\\\\\\/JCxBNbUjT27UA2czK+V6n1mUsYcAkgl6bc0Y2UA13gVEk5epZUlX9z7l4NG2Cs6ZkqO9ZL+J662SfuTLHy3J0VJQnhgPMHCJRuHL4gLge\\\\\\\/aYX82vO2yvP30Qzh5ihtbpNYok9K8Nb7ACIEBpm8TFqLVZvp1SRRALEDMqtqUPgaL8Wn\\\\\\\/074eyShyl6zvVRaOJblScNW\\\\\\\/Kqe5vIxY3TDnwVE5AxFkoWkXH+eRUIXms88JXTrs3a0TpNueu39l4CPE6I5qZ\\\\\\\/iFWO1xmSkiQMK9ZFw6IzUzbKpCYzJ4bvHkSxyG8a9\\\\\\\/Lx2Fl2CpNQJmYO+JWEfJTvVElSsV11oCo6evVtxcfle8oJHr+7QyvjF+GlYYWHZZYGuP6u6pY2G8D8K3H2oQYBIY\\\\\\\/5h1T2hj7YE7pP+R3aGK6GHg0NUiGJR\\\\\\\/LC2nxmm7iKQlR5I9ikgcCHHy\\\\\\\/uYWcsbvg+jvdJcG0fyYjTtFmfXD8lsQAdNhggujNdl6w+l1vOzKzHNRi+CHC1IKuR0fWmFwka6OaKepvkbErOPNGTg1yTEprf2HyezqQ2POwN7V2K6KZTP\\\\\\\/9Wbo1wvJ8p9FzU\\\\\\\/RPREmUjLYJTLRgrh+kX\\\\\\\/WRZkd\\\\\\\/emXUCATxClaWhZFmL7ARAGri5ZpsvSq4JIyKO7Ex9mm9ny0JLY7K9v4MR11dh17khGvl00dUXo+5VmaL3EDazrcPl6M\\\\\\\/D4tm9cJErRvqL5owgkHVD5p12Cr16hb4qecsjfnFFTQiHmDs9rLQ8EabOJPmZg\\\\\\\/WNoWUTZGIZ\\\\\\\/m15n\\\\\\\/10EfEqMeTrV\\\\\\\/PVrnTnmLwkKzlikMEQ8nQ4tGFns0TTGIV8SYObMkitoQ5tMXbm+1+bbs9QismrjHtdxZwRBZhpVO2aFsWrfbKGX\\\\\\\/1zg3MgiV125mKgkgmRqnNb3PUHaG1haQ2CJLRs8Dlz\\\\\\\/1KQ5mW2bpFSNU1xAvsRqOr9+ZoiFG3I1B4SZZtYFCLeQKixyXw6ohHoLYP7DX3801J6htEeAJGZWJZUrJQaUWCar9alw8PRykMpAHVhTqPG4gCvwaXP3BOAPqjwHF12k65tvNLs5uM6\\\\\\\/FvWqcm2p5Jq8lK3bNfWWfgNf9F4ZXbBxrg+57r1CykMbkBegsL0JMTQGdJvVvxLFFQnTWnb1QmTJm8G430gqplsG1wo8xcVDGSt4OSgszg9prACK8uth16PNZyeKGZbqzVcP+cZKqTSZn\\\\\\\/A8NzwluURCHqjwLJlA8snFhnabvYSVhbH556D77vPmTlVkmYVcFmW4qFMthponkKmsiHWhpGkDOIKmRKN\\\\\\\/er8ajvv+cbTzg9YIRKrk6VuT6JEbn1KGY73wYVjwh3X0qIAiFqA0z8AUqMa4kuEFFdV42w2gK33ngXv9R\\\\\\\/UsucC9yuHisP1CBRJKLuxBlOtL+gA6vsIzWfONVyc3ckncIu0b6N54Bf7I2noD7izC+RYlUWDwez56eQF\\\\\\\/qMH\\\\\\\/C0TI8awNWfQGQpNvUALSADYV6ipi5jB7039y7IAj67rasyLpA\\\\\\\/kRzQSTeGmNqsHjDqprlY8GH1e+t+Zjx+0bvuvD2j1\\\\\\\/BeCRa75+6LWVApY1zRqK4hbkHJQWm1PxiX3wpkKmkiAI8enj6nm0Rw87Z\\\\\\\/ltns9la33fUGixKoUTp64hqj+LTv9bB1N81AXwbMQi7VoLRnZB1Prhyhz6gKLMpZpCtnU\\\\\\\/sa\\\\\\\/bALTr12LAA8QN8p4fPOE+wvdDvUY+5kLAdIjCERXUo35D+q3hPbtVKeJ0fN+YsF+IjBzo7\\\\\\\/m6YX7bH8S1WLP+\\\\\\\/WugdYzVwgKKSv0bc6o0osg5Ipnqb1CkSXWLu8AU7ZM\\\\\\\/+PHH6XwiUKT+iGDTjQ9XGbH3HPAPg2sa84Ou1l91lSa3CcabxUAhP4gvrWiGNo5B1UYCosEi99nP72DsLjghV7FTh\\\\\\\/tohWiY\\\\\\\/LlbD2\\\\\\\/9S3PACFp4UhME0cDFXCZVUWf8daKbfrxnTrry8Qa\\\\\\\/G1N\\\\\\\/JceGao333qH+WQ2YfdZodEl5R8ObDqZt49X7978wa1XEkdOJpgEBF+Jw6kbqJjzWTzEGr9IwimeHpXKueOcdrmFbGT30OkTZiQXsgWUOBa+mabkJ8yd9GJDwDTVzm6Y6GZ2GsFKqO7BM2fJdvvn7jyc5560EW2b2YPD\\\\\\\/04bZaGAKPw6l9s8OkOEK7Lemb4sVZIFCqgLqQQtQTF6GmP+uM0URSYwtgx4gAITD7OWt4i+SqrxFOzEKLvCBshmhAQa0PBperGQfi8roeKzqwdLsJWBrY+xHuReV7JMSPQ7xOJ5qpTs+SmseJodpffyYCX9dFnRdrPZ9m7s+V0wVDj8tIAIZrVdgwBMy7GB1+fVb\\\\\\\/4MR81cPR8lKn\\\\\\\/\\\\\\\/dkYw52zwhplJDiGDtAh3wZgGAY0KFSOtATGJK+Vs0\\\\\\\/guD0s4p4YUcGplj8oamqCFD4hr+EZDZe4t8X\\\\\\\/H2hIagfosiu3RSZeDf9T63QKPgK9C\\\\\\\/uRoSdQm7CEzWUfLRz4nyj4ODqJ7Sd4m2CgwLQ8F3IQGspnuk9GKIcbxFWA3qxVSLdnixlvZRLDdrxq5WLgfp\\\\\\\/DURnVoYHY2yu\\\\\\\/C5S7Gh11FOdP+\\\\\\\/R\\\\\\\/hZhOZsAMGMd+mMF3cy4clZx1UhDDIZ+zJWymWuwZJ8mz\\\\\\\/oVbkab6wMgMd2kCuZ6m+W3rECAe\\\\\\\/bZUaEYO+NUIzuoSQgXHnuxwXQSGCqyXgJw7uy67ATTQpeJ9dxADNmTEU0W+XKGEpXKqLVNSFVIjQwsWb\\\\\\\/xPj83kPtt00ZVoEMbt5xldGFml4QbDIVuoeHjMBudTZvB9uDa8wYwyclwY3zH\\\\\\\/u4rXqWh7YNYHfQmjQu5fSspWxqzDQ4BGleTCb+OcMYMnIfVv3nEIYfwsrOTgb7ypY7i4rO0DrbJjafDvICwHd1u0tawtD0g8DrZuDrShmYcDPdpoZVZrMhxrasJlKScHgTt88f4BhoN5mibW5hod6FkNj2mo5C\\\\\\\/RnGJuDeLd+0Fz2GTEVJtzrGf9sQr1r4cJxjqzntGWrXDTGwcsK6K9E1co65TYg2e8c2TLJUkewKoWJJ\\\\\\\/SwipLH4kp+kPMS9ZKdMMQ8p5Jl3i4t9uN6Rr65O07rGNOtJhdvTyg5qF4iJp7Grw3Jq7rB5kcB4WFynAO49lzJ4w+aLLvq6pgmc\\\\\\\/re1lzKHe6wHOdwrDqunhxOM6+\\\\\\\/YGxa5UFBk7+GFxEdttCFytxMCXkfICB\\\\\\\/FFQ\\\\\\\/QUJEmRq7LPYCxYCVsRg2bj4MtqIWwdKf3dLfM56TyZ4OfVSd0p+cqPwlw38okBCehoM3cdS+vgn+ejZcoOXGMEEhzLoyEDIMOo6OqnlsNctGaIIbNJFpPuF4hwnB53Oi9t2N7sw7VX27gi2qM3\\\\\\\/ftpiT3BRDMTbQPwyd4TP6ZnkGMrDDMskwJQlxWW+Ci7cTDIeUnKs4L0tczu05dk5coH0+iBLE0doRT5LeXdlPfabgmLVdXeE+MhRzIS5GFpSfs0jYe8P8FschXd506JPo89FoIUdQigcKleCfgKK5vbDg7IPcgRbruVvwLxRL19cJ55J7WJ3OB8iSwauSGkFwqesReKpkWrQbc+bYTf1Ltygm+qNjbrMbZ9YoFVUYdPxuS931KkUB1zaCdbtzZkYGwK+WHR29WuITLr8\\\\\\\/6wb+RSqf70WLeRwl9aB3VJaU9FKso+uX5pO3N3QNotZveXUoUkuiVC3Hq8+DxvvolAulF4VTP2N5YS4QZmXRA8G1SXSsNj9EoBJLbkCc4M3dSixGiIg702DGSnMxJD9d908O\\\\\\\/Imuj+67uHCPovO5ue+kPl+NF4DeiNQBisOg9VKYuuZzeCWY3j6WpBTSZL5R5rJhHQiojmn+wFmgkMHit43T1hEqGahqJ7HRLKt4yJX93NrejP+EsLXmFnP4pa+FBSuil\\\\\\\/AmoKW3yJnbZ0s+1Gts8vr4px\\\\\\\/r2dXWbNdtpQt8xTTj+Ef0tTZstt\\\\\\\/64C4vP9g4CBoXq5fmqXAk5kAh2aD7Cyb11+nxXYGQsILiUpB7ospWLzx7fYGmNCDi5+fwNIXDdfNwkzU9DOZsE75xSBkXEMT9WsDxUyl1wY3sUTzH\\\\\\\/29Flg4a3b2hvLGa0qkYP1bSCmE5zP8pCNGFP+4UzCAh9S89IiCRqa13ARugLQYD1B\\\\\\\/6VDkCA19CTmeOtqMnG5kkjb3Vj9fcVP\\\\\\\/7aXVwxW1HUlgdJ85Pll1kBBwJXQCI+mQFooBQZTkyWp8Z07uhI2MCiiPRvenlRqX3hr\\\\\\\/azJUrxhwQeIBfmk7PL77h5ShR4nLggldsePGfEWsFpqR2N2+BCAAIUAIKBbFon859f9D3IfK\\\\\\\/nOl34MvV6ZXChb07Q2qRXWJQFVF3vUA+SFfFFN\\\\\\\/3NgCH1uJMDrj6\\\\\\\/D6zsTLSdUzDhHXmDRRWNEpM5QVYVcgn27m8EtAHTj1\\\\\\\/14TJZ\\\\\\\/BV7Znyl7Y8TMiwW98DYevQ+zFxzQsiNs9GHSVCFKt0T\\\\\\\/TSPhBejWQC\\\\\\\/u3Ys49q7myogxX0l2locL6Ul9NNo2ah5a\\\\\\\/PB0V7r3zawZn1BB81vSzQKI2gleq\\\\\\\/WoaEFn9GX\\\\\\\/o\\\\\\\/D9+6sNca9ucsotmqf3Tr\\\\\\\/areN2V1qmlrLlw44mxld+I0MoWwzhvDXj+hMeKVKbAaRvbuwc\\\\\\\/o6euXQVisw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022vFwuTegvwsnzSrYD1wDljg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d27607da2c1d5a587d6a9cb969909a58baeaca459a9eeb7776c37e669b5463a4\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022naW+PYnqsxUSPYzIaoD3HNgQKTkl0yvwabokUfeq66D7Re8M\\\\\\\/aXS\\\\\\\/2V8uNT3phuZxEylesGM6Cje5wDq8aFHi7wNFIOP\\\\\\\/GB6+hr4f50AqdaG8+Fm7J4U3Vr4Z2wRWgGDL6z\\\\\\\/iWEwmoHeqXMbSPaxakUNyXMeAYJGPXxKCGp1+doVnAQY\\\\\\\/ncBorNx82fGEI4Dnwgg4s14++hmu\\\\\\\/Kp1GbIyPrUsjGOKlyPfULZ66qW7lEvQ5kEKE9BSoB0tsAgrQHHxq3QTOKD1qcDheNuHnvElkM39NA9fgj+IyzsaSWTnlKlPjUvVejU2bM+dvTdjkU9PmQzpYTIfsh+8YnCy2ocfc8FIt+ccA24uVGJTdjeXyFoTHBPaDnyiiUWXDOIXd8oaD1bLHdk+qEpwrGVT0UJIkT8g6jOXKjwTimX+blj3f2mJbvf12Z5hfNlPRi9f3KV3EBykq023ndgTMn17gUwGo0H+S1hc+Pf0rzQdxafHGxF5yElEiKMHfGwl\\\\\\\/BVCXNTDuX2yISw+e5p2z\\\\\\\/DNYYonQl\\\\\\\/jrZm7x98MjO1hMwzlbj3aGHazrp\\\\\\\/WK3Srs2BxYbHEQSz0CINE4KGxuyACWTx5wp8NFFBmCMH3x9N3wV+g6oT4CV1OURIVxv1Ez5f2XqdQw28Mb4WqYYjgxYNK0yE2fxHPzhJh9ZuKV4sV0wK1LAfeIM58J26vm0Hls8PsYW3A8ekDQNHD\\\\\\\/JsOyo0DUpiA63P6sy0qY+OamMhPQH5\\\\\\\/Esyqj\\\\\\\/NhbGX147woXDJxA05xosW2ks1DOOJdzgQECtuHU0cvEBfPylRpsFbW3vgcH9S86VnoXo5GbeFBU2UMmMsKQaqwpnzpB5APJfFPIoB5TLOnRZLGVLHkiJdpdbHjEQJQhQFA0JxYM+hTQChOZJwrYoaaYYewWD80E06WGdqRj3YI7priUHTdnoIBo0e52MTxHOmUve\\\\\\\/j6MRzlbX9BRqdttY6UdRtwnXjzbYKjc1uc+5asp8I8cPm0+SZfP3Rbtq87FW2RdFUZrufbakPd9DY+OSkBL1SrMmQzyi2kbn24BuMZRwdiG0jUw1a\\\\\\\/YleCACriS+52Nk\\\\\\\/UD+iAz3E815qfT1hayHmIsmkUFvtynPJDK2pW3Lequ0cKGiU82GaExQO53iYMRtihKWqZ\\\\\\\/O7eOkHgGpT6XKvd9RglpykDS5NtUrp3KvueqzTPjvtXJU9gAM6\\\\\\\/mXvlOqzgU7huWceFQkXjZHPQuMZqMZDt7QPjduDbY+AYI2ENNPXI6Sez8vSRux7TNiQHIwYnzxEHgF9FzFWkrHWAzEkuFpGINCXk+nXcg9RaiECeEXPXInEXy2jZeqVOyPsiTFf1OB70CiSuoJlOiSWjT2yBF7afsAoAfYSubR5yQxfbLs+o3dHVOiFNbI7lUYL\\\\\\\/XRICs6Nf9SHpjESO2qXfRX8gj5CPnHb33dL7tmmrnGrAq9F5RajJ\\\\\\\/rqJasMJWZG\\\\\\\/BzfovuyAmjSe2bPVxGaGwJzDWejwbvvqP\\\\\\\/u1oFbPfn7bfwabZ1Z\\\\\\\/PhuNhbsthvIfBhb\\\\\\\/Kj\\\\\\\/YlefFF5MvpPLDqgAX0Sx3Hsu7Q4vKXulvbNSi2sdq1MNK70aglidzmzWuyWhQIL+L0Ae2Q56LgXewmUSw2f8D8GwPZ7TgJCx\\\\\\\/pqoROMJtnjJb7BKWlA5zRH6n5LbwvNqzE4jDWijq8UJ2tFSDr\\\\\\\/U1OApC3P5JDfE08RQDR7jD137583cJnbJ+MIvVhaCnU8EFZ40Rnk9EtIj+kms65qbyLF3uDXb1RPsc\\\\\\\/4Z2WijcHlOvJgaM\\\\\\\/9nAukBCY9XuqMCZDnR3D\\\\\\\/UM694YUrU+sm8avNgTAqFlceKE8BYVmFMlJaU+Gl6MPCN7D2Lw45e+PeZG5pAi4VyNDIRIXYlz53j1G2N57CAB96y5IIzxzlC4eaA4ZLVitHK4bQ\\\\\\\/hEgoI5XB+xz2xzdMnGQPXiZX4ajAc9iyi5PDrRxPBcJ3mDMEXAAy2ZRrTMOBK7ovIvm3bQm0G+xZO8L7LTiUzJuRNqeMeNcqk5CCg==\\\u0022,\\\u0022iv\\\u0022:\\\u00220vVukED6z2CifZ1m79JsIg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00224979bebdf218f28f01cb79d6c79b32bf902582d0d1898ab91b8155ea6bff1e92\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459829 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/5/931" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-mQOnIcSehFxhqmtfeQEHowhnpiW9XmjPse1JAgMTsmI" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/5/931" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Leu4gMum0pKjXUywZ1Gaz4VK7RjjbFfg6YHHPPE8N8A" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/5/921" title="Charged/Polar-Residue Scanning of the Hydrophobic Face of Transmembrane Domain 9 of the Yeast Glutathione Transporter, Hgt1p, Reveals a Conformationally Critical Region for Substrate Transport" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/5/943" title="Discs large 5, an Essential Gene in Drosophila, Regulates Egg Chamber Organization" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459829" id="node459829" data-pisa="ggg;5/5/931" data-pisa-master="ggg;g3.114.015784" data-apath="/ggg/5/5/931.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Mapping Bias Overestimates Reference Allele Frequencies at the <em>HLA</em> Genes in the 1000 Genomes Project Phase I Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Débora Y. C. Brandt</span>, <span class="highwire-citation-author" data-delta="1">Vitor R. C. Aguiar</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-7676-9367" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Bárbara D. Bitarello</span>, <span class="highwire-citation-author" data-delta="3">Kelly Nunes</span>, <span class="highwire-citation-author" data-delta="4">Jérôme Goudet</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-7155-5674" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Diogo Meyer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">931-941;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.015784 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459829" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Débora Y. C. Brandt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics and Evolutionary Biology, University of São Paulo, 05508-090 São Paulo, SP, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=D%C3%A9bora%2BY.%2BC.%2BBrandt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brandt%20DY&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AD%25C3%25A9bora%2BY.%2BC.%2BBrandt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Vitor R. C. Aguiar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics and Evolutionary Biology, University of São Paulo, 05508-090 São Paulo, SP, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vitor%2BR.%2BC.%2BAguiar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Aguiar%20VR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVitor%2BR.%2BC.%2BAguiar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Bárbara D. Bitarello </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics and Evolutionary Biology, University of São Paulo, 05508-090 São Paulo, SP, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=B%C3%A1rbara%2BD.%2BBitarello%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bitarello%20BD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AB%25C3%25A1rbara%2BD.%2BBitarello%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7676-9367" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Bárbara D. Bitarello</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Kelly Nunes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics and Evolutionary Biology, University of São Paulo, 05508-090 São Paulo, SP, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kelly%2BNunes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nunes%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKelly%2BNunes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jérôme Goudet </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Ecology and Evolution, Biophore, University of Lausanne, CH-1015 Lausanne, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J%C3%A9r%C3%B4me%2BGoudet%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Goudet%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJ%25C3%25A9r%25C3%25B4me%2BGoudet%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Diogo Meyer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics and Evolutionary Biology, University of São Paulo, 05508-090 São Paulo, SP, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Diogo%2BMeyer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Meyer%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADiogo%2BMeyer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7155-5674" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Diogo Meyer</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:diogo@ib.usp.br" class="" data-icon-position="" data-hide-link-title="0">diogo@ib.usp.br</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/5/931" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459829" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459829/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/5/931.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459829" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459829/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/5/931.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459829" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459829/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/5/931.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459829" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459829/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/5/931.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459829" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459829/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Next-generation sequencing (NGS) technologies have become the standard for data generation in studies of population genomics, as the 1000 Genomes Project (1000G). However, these techniques are known to be problematic when applied to highly polymorphic genomic regions, such as the human leukocyte antigen (<em>HLA</em>) genes. Because accurate genotype calls and allele frequency estimations are crucial to population genomics analyses, it is important to assess the reliability of NGS data. Here, we evaluate the reliability of genotype calls and allele frequency estimates of the single-nucleotide polymorphisms (SNPs) reported by 1000G (phase I) at five <em>HLA</em> genes (<em>HLA-A</em>, <em>-B</em>, <em>-C</em>, <em>-DRB1</em>, and <em>-DQB1</em>). We take advantage of the availability of <em>HLA</em> Sanger sequencing of 930 of the 1092 1000G samples and use this as a gold standard to benchmark the 1000G data. We document that 18.6% of SNP genotype calls in <em>HLA</em> genes are incorrect and that allele frequencies are estimated with an error greater than ±0.1 at approximately 25% of the SNPs in <em>HLA</em> genes. We found a bias toward overestimation of reference allele frequency for the 1000G data, indicating mapping bias is an important cause of error in frequency estimation in this dataset. We provide a list of sites that have poor allele frequency estimates and discuss the outcomes of including those sites in different kinds of analyses. Because the HLA region is the most polymorphic in the human genome, our results provide insights into the challenges of using of NGS data at other genomic regions of high diversity.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/ngs" class="hw-term hw-article-keyword hw-article-keyword-ngs" rel="nofollow">NGS</a></li><li class="kwd"><a href="/keyword/mapping-bias" class="hw-term hw-article-keyword hw-article-keyword-mapping-bias" rel="nofollow">mapping bias</a></li><li class="kwd"><a href="/keyword/1000-genomes" class="hw-term hw-article-keyword hw-article-keyword-1000-genomes" rel="nofollow">1000 Genomes</a></li><li class="kwd"><a href="/keyword/hla" class="hw-term hw-article-keyword hw-article-keyword-hla" rel="nofollow">HLA</a></li></ul><p id="p-3">Whole-genome resequencing data for large numbers of human individuals, as generated by the 1000 Genomes Project (<a href="http://www.1000genomes.org">www.1000genomes.org</a>), provide unprecedented amounts of information about microevolutionary processes and demographic histories. Such inferences rely on either genotypic or allelic frequency information for each variable position, which constitute the data for downstream analyses and hypothesis testing.</p><p id="p-4">The calling of single-nucleotide polymorphisms (SNPs) and genotypes and the estimation of allele frequencies from next-generation sequencing (NGS) has undergone rapid development, along with likelihood-based and Bayesian methods created to deal with challenges associated to heterogeneity in read quality and coverage (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Nielsen <em>et al.</em> 2011</a>). In Phase I of the 1000 Genomes Project, genotypes were called using a combination of different approaches: first, primary call sets were independently generated by different centers with different sequencing platforms, alignment, and variant calling methods; then, a consensus SNP call set was generated and made publicly available (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">The 1000 Genomes Project Consortium 2012</a>).</p><p id="p-5">The data generated by the 1000 Genomes Project frequently have been used to make inferences about evolutionary processes affecting our species, including the detection of targets of natural selection (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Hernandez <em>et al.</em> 2011</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Ward and Kellis 2012</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Andersen <em>et al.</em> 2012</a>) and understanding the genetic basis of complex phenotypes (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Lappalainen <em>et al.</em> 2013</a>). In addition, the detailed catalog of genetic variation it provides across multiple human populations has been used to understand the processes affecting specific genes.</p><p id="p-6">Among the well-documented targets of selection is the major histocompatibility complex region of the human genome, which harbors the highly polymorphic classical human leukocyte antigen (<em>HLA</em>) class I and II loci. The interest in these loci stems from their strong association to various autoimmune disorders (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Sollid <em>et al.</em> 2014</a>), susceptibility and resistance to infection (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Chapman and Hill 2012</a>), and striking signatures of genetic variation indicating strong balancing selection (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Meyer and Thomson 2001</a>). Such types of investigations can naturally be extended to the analysis of the 1000 Genomes data, which provide a rich resource of population genetic variation within and around <em>HLA</em> genes.</p><p id="p-7">Despite this interest, the use of NGS data for <em>HLA</em> loci is hampered by a major technical hurdle, which is the mapping of short sequence reads to genes that are both highly polymorphic and which constitute a multigene family. The high polymorphism may decrease the probability that short reads will be successfully mapped to the reference genome, in the event that the sequenced individual carries a variant that is highly diverged from that used in the index (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Nielsen <em>et al.</em> 2011</a>). In addition, the fact that many <em>HLA</em> genes have close paralogues increases the chance that a read will map to two or more genomic regions, leading it to be discarded from most sequencing analyses pipelines, and thus decreasing the amount of usable information for genotype calling (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Treangen and Salzberg 2012</a>).</p><p id="p-8">In previous studies authors explored the applicability of NGS to genotype the <em>HLA</em> alleles of an individual, where an allele typically is defined as the haplotype determined by a combination of SNPs within a given <em>HLA</em> gene [<em>e.g.</em>, <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Erlich <em>et al.</em> (2011)</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Major <em>et al.</em> (2013)</a>]. To this end, <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Erlich <em>et al.</em> (2011)</a> proposed NGS methodologies in which different steps—from sample preparation to haplotype level allele calling—were adapted to deal with the issues of high polymorphism and paralogy of <em>HLA</em> genes. In this way, they were able to successfully validate their methodology in a study of 270 samples that had been typed previously by sequence-specific oligonucleotide hybridization, which they treated as a gold standard dataset. The same gold standard dataset was used by <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Major <em>et al.</em> (2013)</a>, who also examined the reliability of calling <em>HLA</em> alleles using NGS, but using the 1000 Genomes alignment data, and showed that this publicly available dataset can be used for this purpose, after appropriate filters (<em>e.g.</em>, coverage) are applied.</p><p id="p-9">Both <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Erlich <em>et al.</em> (2011)</a> and <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Major <em>et al.</em> (2013)</a> were interested in using NGS data to determine <em>HLA</em> alleles. Information regarding <em>HLA</em> alleles is of biomedical relevance because <em>HLA</em> genotypes often are an important covariate to account for in association studies, and <em>HLA</em> typing is critical to hematopoietic transplantation. In this study, however, we evaluate the quality of SNP level genotype calls from the 1000 Genomes at the <em>HLA</em> genes.</p><p id="p-10">The analysis of genotype and allele frequencies for SNPs contained within <em>HLA</em> genes has proven of great value in biomedical and evolutionary studies, and the 1000 Genomes dataset is a resource used recurrently in this context. Examples of the use of <em>HLA</em> SNP data from the 1000 Genomes Project include: (1) In genome-wide association studies (GWAS), SNPs in <em>HLA</em> genes often are associated with phenotypes of interest, and it is useful to understand the prevalence of these variants in additional populations; (2) GWAS studies benefit from knowledge of the haplotype structure surrounding <em>HLA</em> genes, which can be inferred from the dense SNP data of the 1000 Genomes for multiple populations (<em>e.g.</em>, <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Hill-Burns <em>et al.</em> 2011</a>); and (3) when testing for selection, many studies have found strong evidence associated to the HLA region, using the 1000 Genomes as a source of polymorphism data (<em>e.g.</em>, <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Leffler <em>et al.</em> 2013</a>).</p><p id="p-11">All the aforementioned applications of the 1000 Genomes Project SNP data in <em>HLA</em> genes are dependent on the reliability of genotype calls at each SNP. However, no study to date has provided a detailed survey of the reliability of individual genotype calls and allele frequency estimates at the SNPs in <em>HLA</em> genes, despite their frequent usage. We address this issue, discuss likely causes for cases of incorrect genotype calls, and provide a list of reliable sites for the <em>HLA</em> loci in the 1000 Genomes data. As in previous studies (<a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Erlich <em>et al.</em> 2011</a>; <a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">Major <em>et al.</em> 2013</a>), we used a dataset in which individuals had their <em>HLA</em> genes genotyped using Sanger sequencing as a gold standard to benchmark the genotypes called at the 1000 Genomes Project. However, differently from these other studies, which were interested in reconstructing the <em>HLA</em> haplotypes using NGS, here we have deconstructed the haplotypes determined from Sanger sequencing data into SNPs, and compared genotypes at the SNP level to the 1000 Genomes data. We took advantage of the recent availability of a dataset of Sanger sequencing based <em>HLA</em> genotyping of <em>HLA-A</em>, <em>-B</em>, <em>-C</em>, <em>-DQB1</em>, and <em>-DRB1</em> for 930 of the samples from the 1000 Genomes Project (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Gourraud <em>et al.</em> 2014</a>). Our results have implications for other studies that use SNP data from the 1000 Genomes in order to estimate allele frequencies. Because <em>HLA</em> loci are the most polymorphic in the human genome, they most likely represent the worst case scenario for mapping bias and, consequently, allele frequency estimation error.</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><p id="p-12">In this study, we compare NGS genotype calls and allele frequency estimates reported by the 1000 Genomes Project with those obtained in a study which used Sanger sequencing to genotype <em>HLA</em> genes. For the purpose of our analysis we assembled a dataset comprising the intersection of the 1000 Genomes and Sanger sequencing samples, resulting in 930 individuals from 12 populations. <a href="http://www.g3journal.org/content/suppl/2015/03/17/g3.114.015784.DC1/015784SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2015/03/17/g3.114.015784.DC1/FigureS1.pdf">Figure S1</a> summarizes the preprocessing of both datasets, which preceded genotype and allele frequency comparisons.</p><div id="sec-2" class="subsection"><h3>1000 Genomes dataset (1000G)</h3><p id="p-13">SNP genotypes were acquired from the chromosome 6 integrated Variant Call Format (VCF) file from version 3 of the 1000 Genomes Project Phase I data, which is available at <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20110521/">ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20110521/</a> (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">The 1000 Genomes Project Consortium 2012</a>). We selected only SNPs in exons encoding the antigen recognition sites (ARS), which are exons 2 and 3 for <em>HLA</em>-<em>A</em>, -<em>B</em>, and -<em>C</em> (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bjorkman <em>et al.</em> 1987</a>) and exon 2 for <em>HLA-DQB1</em> and <em>-DRB1</em> (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Brown <em>et al.</em> 1993</a>). Sites were selected based on the most inclusive coordinates of the RefSeq database in July 22, 2014 (see <a href="http://www.g3journal.org/content/suppl/2015/03/17/g3.114.015784.DC1/FileS1.pdf">File S1</a>). Both SNP and sample selection were carried out using VCFtools v0.1.12b (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Danecek <em>et al.</em> 2011</a>).</p></div><div id="sec-3" class="subsection"><h3><em>HLA</em> reference panel by <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Gourraud <em>et al.</em> (2014)</a> (PAG2014)</h3><p id="p-14"><a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Gourraud <em>et al.</em> (2014)</a> typed class I <em>HLA-A</em>, <em>-B</em> and <em>-C</em>, and class II <em>HLA-DRB1</em> and <em>-DQB1</em> genes of 1266 individuals from 14 different populations in Africa, Europe, Asia, and America. The <em>HLA</em> sequence-based typing was performed with specific polymerase chain reaction amplification of ARS exons followed by Sanger sequencing. Data are available at the dbMHC Web site (<a href="http://www.ncbi.nlm.nih.gov/gv/mhc/xslcgi.fcgi?cmd=cellsearch">http://www.ncbi.nlm.nih.gov/gv/mhc/xslcgi.fcgi?cmd=cellsearch</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Helmberg <em>et al.</em> 2014</a>).</p><p id="p-15">Data from <a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Gourraud <em>et al.</em> (2014)</a> are available in the form of <em>HLA</em> allele names per individual. Allele naming for <em>HLA</em> genes follows specific rules (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Marsh <em>et al.</em> 2010</a>). To summarize, allele names are composed of a letter indicating the locus, followed by 2−4 numeric fields separated by colons. Each numeric field indicates specific forms of variation: the 1st field distinguishes groups of alleles by serological type, and the following fields distinguish nonsynonymous polymorphisms, synonymous polymorphisms, and noncoding differences, respectively. To obtain SNP genotypes and frequencies from the Sanger sequencing data, we converted all allele names to their associated sequences for ARS encoding exons. Sequences were acquired from the IMGT (<em>i.e.</em>, international ImMunoGeneTics information system) database (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Robinson <em>et al.</em> 2013</a>), which keeps a well-curated repository of all known <em>HLA</em> allele sequences.</p><p id="p-16">Our analysis was restricted to ARS exons because the <em>HLA</em> typing method used by <a id="xref-ref-11-5" class="xref-bibr" href="#ref-11">Gourraud <em>et al.</em> (2014)</a> only probed genetic variation in these specific exons. As a consequence, multiple <em>HLA</em> alleles are compatible with the sequencing results, because the sites that differentiate them are in other exons. This results in what we refer to as an “ambiguous allele call” for an <em>HLA</em> allele (<em>e.g.</em>, the allele is identified as B*35:03, but we cannot establish whether it is B*35:03:01 or B*35:03:02, or a group of alleles is attributed to an individual, such as B*35:02/B*35:03/B*35:04). Ambiguous allele calls also may happen when sequencing has low quality at bases that differentiate two alleles. In addition, there are also genotypic ambiguities, which occur when different pairs of alleles are compatible with the sequencing results. For individuals that bear ambiguous alleles, we created a consensus sequence in which ambiguous sites were reported with both possible alleles (<em>e.g.</em>, A/T, see <a href="http://www.g3journal.org/content/suppl/2015/03/17/g3.114.015784.DC1/FigureS1.pdf">Figure S1</a>). In this way, we incorporate the uncertainty associated to the sequence-based typing into downstream analyses.</p><p id="p-17">Although we cannot rule out technical errors in the Sanger sequencing that generated the PAG2014 data (<a id="xref-ref-11-6" class="xref-bibr" href="#ref-11">Gourraud <em>et al.</em> 2014</a>), we assume that this method provides the most reliable estimate of <em>HLA</em> alleles (and hence SNP genotypes), and will serve as a standard to estimate the reliability of genotype calls and allele frequencies for the 1000 Genomes data (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">De Santis <em>et al.</em> 2013</a>).</p></div><div id="sec-4" class="subsection"><h3>Genotype comparisons</h3><p id="p-18">We initially quantified how well the 1000G and PAG2014 data agreed with respect to genotype calls. Genotypes at each site in each individual were compared between the 1000G data and the PAG2014 data, here considered as a gold standard. In the case of sites with ambiguity (<em>e.g.</em>, T/A) in the PAG2014 data, if one of the two possible alleles matched an allele present in the 1000G, we considered this an allele match and PAG2014 was corrected, by attributing the allele present in the 1000G data to the ambiguous site. After correcting the ambiguous sites in PAG2014, we only considered genotypes to be a match if both alleles in 1000G were present in the PAG2014 data, at that site. Throughout this article, sites are numbered according to their position in the ARS exons coding sequences (1−546 at the class I loci and 1−270 at the class II loci).</p></div><div id="sec-5" class="subsection"><h3>Allele frequency comparisons</h3><p id="p-19">After correcting all possible ambiguities in PAG2014 (as described previously), we calculated allele frequencies for SNPs in both datasets. By comparing the frequency of the reference allele in 1000G to its value in PAG2014, we assessed the accuracy of allele frequency estimation. The reference allele was defined as the allele present in the hg19 build of the reference sequence of the human genome. RefSeq IDs of the reference sequences used for each <em>HLA</em> gene are reported on <a href="http://www.g3journal.org/content/suppl/2015/03/17/g3.114.015784.DC1/FileS1.pdf">File S1</a>.</p><p id="p-20">We computed the error in 1000G frequency estimates per site <em>i</em> (<em>FE<sub>i</sub></em>) as follows:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/931/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/931/embed/mml-math-1.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/931/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/931/embed/mml-math-2.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/931/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/931/embed/mml-math-3.gif"/></noscript></span></span> are the frequency of the reference allele at site <em>i</em> in 1000G and PAG2014, respectively. We also computed the mean absolute error in frequency estimates per gene as a mean of absolute <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/931/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/931/embed/mml-math-4.gif"/></noscript></span></span> for all sites within a gene (MAE):<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/931/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/931/embed/mml-math-5.gif"/></noscript></span></span>where <em>n</em> is the number of SNPs in the gene.</p></div><div id="sec-6" class="subsection"><h3>Coverage in 1000G</h3><p id="p-21">Sequencing coverage per individual per site was calculated from the 1000 Genomes Project phase I BAM files for the low coverage experiments using the genomeCoverageBed program from BEDTools (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Quinlan and Hall 2010</a>). BAM files are available on <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/phase1/data/[sampleID]/alignment/">ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/phase1/data/[sampleID]/alignment/</a>. Only low-coverage BAM files were used to estimate coverage because genotype likelihoods for the data we analyzed (1000 Genomes Project Phase I integrated VCF files) were estimated from this source. Genotype likelihoods were estimated from high coverage exome BAM files only for a minority of sites that were exclusively discovered on the exome experiments, and were not used in the coverage analysis (See <a href="http://www.g3journal.org/content/suppl/2015/03/17/g3.114.015784.DC1/TableS1.pdf">Table S1</a>).</p></div><div id="sec-7" class="subsection"><h3>Testing for mapping bias</h3><p id="p-22">After demonstrating that there is an overestimation of reference allele frequency in the 1000G SNPs (see the section <em>Results</em>), we hypothesized that mapping bias was the underlying cause. To test this hypothesis, we examined whether reads carrying the alternative allele at a SNP are less likely to map to the reference genome than reads carrying the reference allele. First, for each <em>HLA</em> allele present in the PAG2014 dataset, we defined windows of 51 base pairs that were centered on each SNP (25 base pairs upstream and 25 base pairs downstream of the SNP, including non-polymorphic sites). The set of windows centered on a specific SNP was then separated in two groups: (i) those that carry the reference allele at the central site and (ii) those that carry the alternative allele at the central site. Next, all windows were compared with the reference genome (hg19) sequence (the same sequence that was used as an index in the 1000 Genomes Project), and the number of mismatches was counted, excluding the mismatch at the central SNP. If mapping bias was influencing allele frequency estimates, we expected that, for SNP positions with overestimation of the reference allele frequency in the 1000G, the alternative alleles would be flanked by additional alternative alleles (and thus have a greater mismatch count against the reference sequence).</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>Genotypic mismatch frequency</h3><p id="p-23">We found that, on average, 18.6% of genotypes were mismatched between 1000G and PAG2014 when individual genotypes for each site in the five classical <em>HLA</em> genes were compared, and exons with greater nucleotide diversity tend to have a greater proportion of genotype mismatches (<a href="http://www.g3journal.org/content/suppl/2015/03/17/g3.114.015784.DC1/FigureS2.pdf">Figure S2</a>). We also observed that mismatches are specially concentrated on a few sites (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>), with 18.7% of sites concentrating 50% of the mismatches over the five loci we analyzed.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/5/931/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genotype mismatches between the 1000G and PAG2014 datasets. Results per polymorphic site (&#x201C;Position&#x201D;) and per individual (930 in total). Individuals are ordered by number of mismatches (individuals with less mismatches on top). Sites are numbered according to their position in ARS exons coding sequence. Dark squares indicate mismatches between genotypes in the two datasets. ARS, antigen recognition sites; HLA, human leukocyte antigen." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1916747938" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Genotype mismatches between the 1000G and PAG2014 datasets. Results per polymorphic site (&#x201C;Position&#x201D;) and per individual (930 in total). Individuals are ordered by number of mismatches (individuals with less mismatches on top). Sites are numbered according to their position in ARS exons coding sequence. Dark squares indicate mismatches between genotypes in the two datasets. ARS, antigen recognition sites; HLA, human leukocyte antigen.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/5/931/F1.medium.gif" width="430" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/5/931/F1.medium.gif" width="430" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/5/931/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/5/931/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466163" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-24" class="first-child">Genotype mismatches between the 1000G and PAG2014 datasets. Results per polymorphic site (“Position”) and per individual (930 in total). Individuals are ordered by number of mismatches (individuals with less mismatches on top). Sites are numbered according to their position in ARS exons coding sequence. Dark squares indicate mismatches between genotypes in the two datasets. ARS, antigen recognition sites; HLA, human leukocyte antigen.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Reference allele frequency accuracy</h3><p id="p-25">Accuracy of estimation of allele frequencies in 1000G was assessed comparing the observed frequency of the reference allele in the 1000G data with that of PAG2014, for both the global dataset (consisting of a pooled set of all individuals) and for each population separately (see <a href="http://www.g3journal.org/content/suppl/2015/03/17/g3.114.015784.DC1/FigureS3.pdf">Figure S3</a>, <a href="http://www.g3journal.org/content/suppl/2015/03/17/g3.114.015784.DC1/FigureS4.pdf">Figure S4</a>, <a href="http://www.g3journal.org/content/suppl/2015/03/17/g3.114.015784.DC1/FigureS5.pdf">Figure S5</a>, <a href="http://www.g3journal.org/content/suppl/2015/03/17/g3.114.015784.DC1/FigureS6.pdf">Figure S6</a>, and <a href="http://www.g3journal.org/content/suppl/2015/03/17/g3.114.015784.DC1/FigureS7.pdf">Figure S7</a>). We chose a difference of 0.1 between the frequencies on both datasets as a threshold that determines a “large frequency difference.”</p><p id="p-26">For the global dataset (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>) we found that for <em>HLA-A</em> and <em>-C</em> most SNPs have similar frequency estimates for 1000G and PAG2014, with few large deviations (only 9/66 and 8/44 SNPs with absolute difference in frequencies (<span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/931/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/931/embed/mml-math-6.gif"/></noscript></span></span>) larger than 0.1, respectively). The <em>HLA-DQB1</em> locus shows an intermediate proportion of SNPs with large deviations (10/42 SNPs with <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/931/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/931/embed/mml-math-7.gif"/></noscript></span></span>), and <em>HLA-B</em> and <em>HLA-DRB1</em> show the greatest proportion of sites with large frequency differences between 1000G and PAG2014 (23/64 and 15/35 sites with <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/931/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/931/embed/mml-math-8.gif"/></noscript></span></span>). Overall, the mean absolute difference in frequency between SNPs in the 1000G and PAG2014 data are 0.08, and it is greater at the <em>HLA</em> genes with the greatest levels of nucleotide diversity (<em>HLA-B</em>, <em>-DQB1</em> and <em>-DRB1</em> all deviate by <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/931/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/931/embed/mml-math-9.gif"/></noscript></span></span>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/5/931/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="REF allele frequency per site in each HLA gene in the 1000 Genomes (1000G) and Sanger sequencing (PAG2014) datasets. Continuous line indicates the expected relationship (i.e., no difference) between 1000G and PAG2014. Dashed lines indicate a &#xB1;0.1 deviation from the expected frequency (as estimated from PAG2014 dataset). MAE (mean absolute error) defined in the section Materials and Methods. Numbers indicate site position in ARS exons sequence. REF, reference; ARS, antigen recognition sites; HLA, human leukocyte antigen." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1916747938" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;REF allele frequency per site in each &lt;em&gt;HLA&lt;/em&gt; gene in the 1000 Genomes (1000G) and Sanger sequencing (PAG2014) datasets. Continuous line indicates the expected relationship (&lt;em&gt;i.e.&lt;/em&gt;, no difference) between 1000G and PAG2014. Dashed lines indicate a &#xB1;0.1 deviation from the expected frequency (as estimated from PAG2014 dataset). MAE (mean absolute error) defined in the section &lt;em&gt;Materials and Methods&lt;/em&gt;. Numbers indicate site position in ARS exons sequence. REF, reference; ARS, antigen recognition sites; HLA, human leukocyte antigen.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/5/931/F2.medium.gif" width="293" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/5/931/F2.medium.gif" width="293" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/5/931/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/5/931/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466103" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-27" class="first-child">REF allele frequency per site in each <em>HLA</em> gene in the 1000 Genomes (1000G) and Sanger sequencing (PAG2014) datasets. Continuous line indicates the expected relationship (<em>i.e.</em>, no difference) between 1000G and PAG2014. Dashed lines indicate a ±0.1 deviation from the expected frequency (as estimated from PAG2014 dataset). MAE (mean absolute error) defined in the section <em>Materials and Methods</em>. Numbers indicate site position in ARS exons sequence. REF, reference; ARS, antigen recognition sites; HLA, human leukocyte antigen.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">The proportion of genotype mismatches and allele frequency deviations per site are highly correlated (Pearson correlation = 0.86, <em>P</em> &lt; 10<sup>−16</sup>; <a href="http://www.g3journal.org/content/suppl/2015/03/17/g3.114.015784.DC1/FigureS8.pdf">Figure S8</a>). However, some SNPs with a high proportion of genotype mismatches have well-estimated allele frequencies. One example is site 465 at <em>HLA-B</em>, in which 44% of genotypes are mismatched, but <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/931/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/931/embed/mml-math-10.gif"/></noscript></span></span> is only 0.007. Overall, 15 sites have more than 25% mismatched genotypes while showing <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/931/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/931/embed/mml-math-11.gif"/></noscript></span></span> (see <a href="http://www.g3journal.org/content/suppl/2015/03/17/g3.114.015784.DC1/FigureS8.pdf">Figure S8</a>). This is possible when the frequency of genotype errors in which the reference allele is overrepresented is similar to the frequency of errors in which the alternative allele is overrepresented.</p><div id="sec-11" class="subsection"><h4>Allele frequency at the axiom exome genotyping array – Affymetrix:</h4><p id="p-29">Because genotyping arrays constitute an additional frequently used resource to genotype SNPs within <em>HLA</em> genes, playing an important role in GWAS studies, we also have investigated the accuracy of allele frequency estimation from this genotyping technology. We estimated allele frequencies from Axiom Exome data, and we found that those allele frequency estimates are as reliable as the ones from the 1000 Genomes NGS data, at the same SNPs (see <a href="http://www.g3journal.org/content/suppl/2015/03/17/g3.114.015784.DC1/FigureS9.pdf">Figure S9</a>).</p></div></div><div id="sec-12" class="subsection"><h3>Relationship between sequencing coverage and genotypic mismatches</h3><p id="p-30">To investigate whether low sequencing coverage could explain genotype mismatches and deviations from expected allele frequencies, we compared sequencing coverage between mismatched and matched genotypes (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>) and assessed the relationship between coverage and frequency deviation (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/5/931/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Distribution of coverage (x-axis) at matched and mismatched genotypes; y-axis is the square root of the relative frequency (Mann-Whitney U one-tailed test, P &lt; 10&#x2212;16); (B) Relationship between mean coverage (x-axis) and absolute frequency difference (, y-axis) between 1000G and PAG2014 (r = &#x2212;0.11, P = 0.09). All polymorphic sites from HLA-A, -B, -C, -DRB1, and -DQB1 genes are included in both a and b. HLA, human leukocyte antigen." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1916747938" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;(A) Distribution of coverage (x-axis) at matched and mismatched genotypes; y-axis is the square root of the relative frequency (Mann-Whitney &lt;em&gt;U&lt;/em&gt; one-tailed test, P &lt; 10&lt;sup&gt;&#x2212;16&lt;/sup&gt;); (B) Relationship between mean coverage (x-axis) and absolute frequency difference (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-12&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, y-axis) between 1000G and PAG2014 (&lt;em&gt;r&lt;/em&gt; = &#x2212;0.11, &lt;em&gt;P&lt;/em&gt; = 0.09). All polymorphic sites from &lt;em&gt;HLA-A&lt;/em&gt;, &lt;em&gt;-B&lt;/em&gt;, &lt;em&gt;-C&lt;/em&gt;, &lt;em&gt;-DRB1&lt;/em&gt;, and &lt;em&gt;-DQB1&lt;/em&gt; genes are included in both &lt;em&gt;a&lt;/em&gt; and &lt;em&gt;b&lt;/em&gt;. HLA, human leukocyte antigen.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/5/931/F3.medium.gif" width="440" height="166"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/5/931/F3.medium.gif" width="440" height="166"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/5/931/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/5/931/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465961" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-31" class="first-child">(A) Distribution of coverage (x-axis) at matched and mismatched genotypes; y-axis is the square root of the relative frequency (Mann-Whitney <em>U</em> one-tailed test, P &lt; 10<sup>−16</sup>); (B) Relationship between mean coverage (x-axis) and absolute frequency difference (<span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/931/F3/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/931/F3/embed/mml-math-12.gif"/></noscript></span></span>, y-axis) between 1000G and PAG2014 (<em>r</em> = −0.11, <em>P</em> = 0.09). All polymorphic sites from <em>HLA-A</em>, <em>-B</em>, <em>-C</em>, <em>-DRB1</em>, and <em>-DQB1</em> genes are included in both <em>a</em> and <em>b</em>. HLA, human leukocyte antigen.</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">Sites with mismatched genotypes have on average lower sequencing coverage than sites with matched genotypes (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3A</a>; Mann-Whitney <em>U</em> one-tailed test <em>P</em> &lt; 10<sup>−16</sup>). This is the expected relationship if low sequencing coverage explains genotype mismatches between datasets. However, the difference in sequencing coverage between sites with matched and mismatched genotypes is small (mean coverage in matching genotypes is 1.95, and 1.75 in nonmatching genotypes, a difference of 6.2%) and has likely achieved very high significance only due to the large number of observations. Similarly, correlation between allele frequency deviation and sequencing coverage is weak and not significant (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3B</a>; <em>r</em> = −0.11, <em>P</em> = 0.09), although the direction of correlation is in agreement with what would be expected if lower coverage explained larger deviations in frequency estimation. We have also investigated the possible effect of the position of the SNPs relative to exon edges on the allele frequency deviations and found no correlation between those factors (<a href="http://www.g3journal.org/content/suppl/2015/03/17/g3.114.015784.DC1/FigureS10.pdf">Figure S10</a>). We therefore investigated other factors that may account for errors in genotype calling.</p></div><div id="sec-13" class="subsection"><h3>Direction of frequency deviation</h3><p id="p-33">We found that most of the genotype mismatches are caused by miscalling an alternative allele as a reference allele (<a href="http://www.g3journal.org/content/suppl/2015/03/17/g3.114.015784.DC1/TableS2.pdf">Table S2</a>). Furthermore, most deviations in allele frequency estimates are in the direction of an overestimation of reference allele frequencies in the 1000 Genomes data (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). This information is summarized in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>, which shows the location and magnitude of frequency deviations between the 1000G and PAG2014 data.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/5/931/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Difference in reference allele frequency between 1000G and PAG2014, measured by  (see the section Materials and Methods), at each polymorphic site, in each population. Shades of red indicate overestimation of reference allele frequency and shades of blue indicate underestimation of reference allele frequency in 1000G. Full population names are given in Table S3." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1916747938" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Difference in reference allele frequency between 1000G and PAG2014, measured by &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-13&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (see the section &lt;em&gt;Materials and Methods&lt;/em&gt;), at each polymorphic site, in each population. Shades of red indicate overestimation of reference allele frequency and shades of blue indicate underestimation of reference allele frequency in 1000G. Full population names are given in Table S3.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/5/931/F4.medium.gif" width="440" height="437"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/5/931/F4.medium.gif" width="440" height="437"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/5/931/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/5/931/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466022" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-34" class="first-child">Difference in reference allele frequency between 1000G and PAG2014, measured by <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/931/F4/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/931/F4/embed/mml-math-13.gif"/></noscript></span></span> (see the section <em>Materials and Methods</em>), at each polymorphic site, in each population. Shades of red indicate overestimation of reference allele frequency and shades of blue indicate underestimation of reference allele frequency in 1000G. Full population names are given in <a href="http://www.g3journal.org/content/suppl/2015/03/17/g3.114.015784.DC1/TableS3.pdf">Table S3</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-35">The overall shift in the direction of overestimating reference alleles is summarized in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, which shows the number of SNPs with more than 0.1 frequency difference in at least two populations, for each locus. For <em>HLA-A</em>, <em>-B</em>, and <em>-DQB1</em> most sites with large frequency differences between 1000G and PAG2014 are skewed in the direction of overestimating the reference allele [<em>P</em> = 0.057 for <em>HLA-A</em> and <em>P</em> &lt; 10<sup>−4</sup> for <em>HLA-B</em> and <em>-DQB1</em>, binomial test for null hypothesis of equal numbers of deviations in direction of reference (REF) or alternative (ALT)], whereas <em>HLA-C</em> and <em>HLA-DRB1</em> show no evidence for an excess of large deviations in the direction of reference alleles.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466095/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466095/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Number of sites with overestimation of REF or ALT allele frequency in each <em>HLA</em> locus (<span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/931/T1/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/931/T1/embed/mml-math-14.gif"/></noscript></span></span> &gt; 0.1 in 2 or more populations)</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Testing for mapping bias</h3><p id="p-37">We hypothesized that the observed reference allele bias was caused by a lower efficiency in the mapping of reads containing the alternative allele. This is expected under the assumption that the reads carrying the alternative allele on average have more differences with respect to the reference genome (used by the 1000 Genomes Consortium as the index to align NGS reads) than reads carrying the reference allele. In this scenario, some sites would have a stronger bias than others if the alternative alleles in those sites are flanked by additional alternative alleles.</p><p id="p-38">To test this hypothesis, we aligned sequences of all alleles present in PAG2014 to the <em>HLA</em> sequences present in the hg19 build of the reference human genome (the same sequences used for the alignment of reads in the 1000 Genomes Project) and defined windows of 51 base pairs around each SNP. We then quantified the number of differences with respect to the reference genome for windows surrounding (i) REF and (ii) ALT alleles. If REF allele mapping bias is driving errors in frequency estimation, it is expected that sites with an overestimation of REF allele frequency would present the following pattern: windows carrying the REF with fewer differences to the reference genome than sequences centered on the ALT alleles. For sites with well-estimated frequencies, on the other hand, we did not expect such a difference between REF and ALT windows.</p><p id="p-39">To illustrate this effect, <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> shows the results for the two most extreme cases of frequency deviation shown in <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>: site 244 of <em>HLA-DQB1</em> and site 132 of <em>HLA-B</em> (0.56 and 0.52 absolute increase in REF allele frequency in the 1000 Genomes data with respect to PAG2014). In both cases, ALT windows bear more differences to the reference sequence than REF windows.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/5/931/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Number of differences to the reference genome at 1860 51-bp windows centered at sites HLA-B 132 and HLA-DQB1 244 with reference (REF) or alternative (ALT) allele at those sites. Windows were defined from all HLA alleles present in the 930 samples from the PAG2014 dataset. HLA, human leukocyte antigen." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1916747938" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Number of differences to the reference genome at 1860 51-bp windows centered at sites &lt;em&gt;HLA-B&lt;/em&gt; 132 and &lt;em&gt;HLA-DQB1&lt;/em&gt; 244 with reference (REF) or alternative (ALT) allele at those sites. Windows were defined from all &lt;em&gt;HLA&lt;/em&gt; alleles present in the 930 samples from the PAG2014 dataset. HLA, human leukocyte antigen.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/5/931/F5.medium.gif" width="440" height="224"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/5/5/931/F5.medium.gif" width="440" height="224"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/5/931/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/5/931/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465930" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-40" class="first-child">Number of differences to the reference genome at 1860 51-bp windows centered at sites <em>HLA-B</em> 132 and <em>HLA-DQB1</em> 244 with reference (REF) or alternative (ALT) allele at those sites. Windows were defined from all <em>HLA</em> alleles present in the 930 samples from the PAG2014 dataset. HLA, human leukocyte antigen.</p><div class="sb-div caption-clear"></div></div></div><p id="p-41">These results support the hypothesis that these sites with poorly estimated allele frequencies have their ALT alleles residing in haplotypes with substantially more differences with respect to the reference genome than haplotypes centered on the REF allele, thus accounting for the observed bias.</p><p id="p-42">To gain a broader perspective of this issue, we classified SNPs from the <em>HLA</em> loci with REF allele bias (<em>HLA-A</em>, <em>-B</em>, and <em>-DQB1</em>) into three categories: (i) sites at which the REF allele frequency was overestimated, <em>i.e.</em>, <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/931/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/931/embed/mml-math-15.gif"/></noscript></span></span> (“<em>overestimated</em>”); (ii) sites where the REF allele frequency was underestimated, <em>i.e.</em>, <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/931/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/931/embed/mml-math-16.gif"/></noscript></span></span> (“<em>underestimated</em>”); and (iii) sites at which allele frequencies were well estimated (<span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/931/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/931/embed/mml-math-17.gif"/></noscript></span></span>, here referred to as “<em>well estimated</em>”). We compared these three categories of sites with respect to the number of differences relative to the reference genome in REF and ALT windows (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). We found that the <em>overestimated</em> group has significant excess of differences at alternative allele bearing haplotypes. In this group of SNPs, ALT windows have on average 4.4 other differences relative to the reference genome, whereas those centered on the REF allele have 1.9 differences (excess of differences on windows centered on the ALT allele was tested with a one tailed Mann-Whitney <em>U</em> test; <em>P</em> &lt; 10<sup>−16</sup>). Sites with <em>well estimated</em> or <em>underestimated</em> REF allele frequency, on the other hand, do not show a similar excess of differences in the haplotypes bearing the ALT allele, although the difference between REF and ALT windows is statistically significant because of the large sample size (<em>well estimated</em>: ALT mean = 1.7; REF mean = 1.8; one tailed Mann-Whitney <em>U</em> test <em>P</em> &lt; 10<sup>−16</sup>; <em>underestimated</em>: ALT mean = 1.9 ; REF mean = 1.2; one tailed Mann-Whitney <em>U</em>, <em>P</em> &lt; 10<sup>−16</sup>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/5/931/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Number of differences to the reference genome at 51-bp windows centered at each SNP in the HLA-A, -B, and -DQB1 genes. Windows around each SNP were defined from the set of 1860 alleles present in the 930 samples from the PAG2014 dataset. Next, the set of windows was divided in three groups: those centered on SNPs with overestimated, well estimated and underestimated reference allele frequencies (red, yellow and blue boxplots, respectively). Then, each group was divided in two: windows in which the central site contains the reference allele (REF, dark boxplots) and windows centered on an alternative allele (ALT, light colored boxplots). Upper and lower hinges correspond to the 25th and 75th percentiles, horizontal lines represent the median, whiskers are 1.5 times the interquartile range, and outliers are represented by dots. HLA, human leukocyte antigen; SNP, single-nucleotide polymorphism." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1916747938" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Number of differences to the reference genome at 51-bp windows centered at each SNP in the &lt;em&gt;HLA-A&lt;/em&gt;, &lt;em&gt;-B&lt;/em&gt;, and &lt;em&gt;-DQB1&lt;/em&gt; genes. Windows around each SNP were defined from the set of 1860 alleles present in the 930 samples from the PAG2014 dataset. Next, the set of windows was divided in three groups: those centered on SNPs with &lt;em&gt;overestimated&lt;/em&gt;, &lt;em&gt;well estimated&lt;/em&gt; and &lt;em&gt;underestimated&lt;/em&gt; reference allele frequencies (red, yellow and blue boxplots, respectively). Then, each group was divided in two: windows in which the central site contains the reference allele (REF, dark boxplots) and windows centered on an alternative allele (ALT, light colored boxplots). Upper and lower hinges correspond to the 25th and 75th percentiles, horizontal lines represent the median, whiskers are 1.5 times the interquartile range, and outliers are represented by dots. HLA, human leukocyte antigen; SNP, single-nucleotide polymorphism.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/5/931/F6.medium.gif" width="424" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/5/5/931/F6.medium.gif" width="424" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/5/931/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/5/931/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466063" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-43" class="first-child">Number of differences to the reference genome at 51-bp windows centered at each SNP in the <em>HLA-A</em>, <em>-B</em>, and <em>-DQB1</em> genes. Windows around each SNP were defined from the set of 1860 alleles present in the 930 samples from the PAG2014 dataset. Next, the set of windows was divided in three groups: those centered on SNPs with <em>overestimated</em>, <em>well estimated</em> and <em>underestimated</em> reference allele frequencies (red, yellow and blue boxplots, respectively). Then, each group was divided in two: windows in which the central site contains the reference allele (REF, dark boxplots) and windows centered on an alternative allele (ALT, light colored boxplots). Upper and lower hinges correspond to the 25th and 75th percentiles, horizontal lines represent the median, whiskers are 1.5 times the interquartile range, and outliers are represented by dots. HLA, human leukocyte antigen; SNP, single-nucleotide polymorphism.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Impact of biases in frequency estimation to population genetic statistics</h3><p id="p-44">Our analysis was able to identify a subset of SNPs in the <em>HLA</em> genes for which genotype calls and allele frequency estimates from the 1000G showed a high error rate with respect to the PAG2014 dataset. To evaluate the impact of the errors introduced by including these sites in population genetic analyses, we compared the distribution of sample heterozygosity between the sites with low and high error rates. Heterozygosity is defined as <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/931/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/931/embed/mml-math-18.gif"/></noscript></span></span> for biallelic loci, as is the case for the 1000 Genomes Phase I SNPs, because tri- or quad-allelic SNPs were not reported on Phase I.</p><p id="p-45">The removal of sites with poor frequency estimates (<span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/931/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/931/embed/mml-math-19.gif"/></noscript></span></span> in at least two populations) results in a marked change in the distribution of <em>H</em>, with a significant drop in the frequency of sites with large <em>H</em> and a shift in the distribution toward lower values (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>). Note that the <em>H</em> values in <a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7</a> are estimated from the PAG2014 data, implying that the high values of <em>H</em> among “excluded” sites are not due to the deviations in allele frequencies generated by NGS errors, but are the true heterozygosities at those sites. These results therefore document that because sites with high heterozygosity tend to have greater deviations from the “true” frequency (<em>i.e.</em>, based on the PAG2014 dataset), the removal of poorly estimated sites results in a reduction in <em>H</em> values.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/5/931/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Heterozygosity of SNPs at HLA genes estimated from the PAG2014 dataset. Orange bars show distribution of heterozygosity at sites with a high error rate in frequency estimation ( in two or more populations). Blue bars show the distribution of heterozygosity after exclusion of SNPs with high error rate. SNP, single-nucleotide polymorphism; HLA, human leukocyte antigen. " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1916747938" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Heterozygosity of SNPs at &lt;em&gt;HLA&lt;/em&gt; genes estimated from the PAG2014 dataset. Orange bars show distribution of heterozygosity at sites with a high error rate in frequency estimation (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-20&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; in two or more populations). Blue bars show the distribution of heterozygosity after exclusion of SNPs with high error rate. SNP, single-nucleotide polymorphism; HLA, human leukocyte antigen. &lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/5/931/F7.medium.gif" width="429" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/5/5/931/F7.medium.gif" width="429" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/5/931/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/5/931/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466127" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-46" class="first-child">Heterozygosity of SNPs at <em>HLA</em> genes estimated from the PAG2014 dataset. Orange bars show distribution of heterozygosity at sites with a high error rate in frequency estimation (<span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/931/F7/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/931/F7/embed/mml-math-20.gif"/></noscript></span></span> in two or more populations). Blue bars show the distribution of heterozygosity after exclusion of SNPs with high error rate. SNP, single-nucleotide polymorphism; HLA, human leukocyte antigen. </p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>The effect of heterozygosity on allele frequency estimation bias</h3><p id="p-47">We found an overall positive correlation between SNP heterozygosity and the magnitude of error in allele frequency estimates (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8A</a>; Pearson’s correlation = 0.32; <em>P</em> = 1.938 × 10<sup>−7</sup>). This result provides further evidence that sites with greater heterozygosity tend to have poorer estimates for allele frequencies in the 1000G. Also, heterozygosity is even more strongly correlated to the deviation in frequency, considering the direction of the deviation (<a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8B</a>; Pearson’s correlation = 0.59; P &lt; 10<sup>−16</sup>). Together, these results show that <em>HLA</em> SNPs with greater heterozygosities not only have more errors in frequency estimation but also a stronger bias toward overestimation of REF allele frequency.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/5/931/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relationship between SNP heterozygosity (H) and (A) absolute value of deviation (; Pearson&#x2019;s correlation = 0.32; P = 1.938 &#xD7; 10&#x2212;7) or (B) magnitude and direction of deviation (FE; Pearson&#x2019;s correlation = 0.59; P &lt; 10&#x2212;16). SNP, single-nucleotide polymorphism." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1916747938" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Relationship between SNP heterozygosity (H) and (A) absolute value of deviation (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-21&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;; Pearson&#x2019;s correlation = 0.32; &lt;em&gt;P&lt;/em&gt; = 1.938 &#xD7; 10&lt;sup&gt;&#x2212;7&lt;/sup&gt;) or (B) magnitude and direction of deviation (&lt;em&gt;FE&lt;/em&gt;; Pearson&#x2019;s correlation = 0.59; &lt;em&gt;P&lt;/em&gt; &lt; 10&lt;sup&gt;&#x2212;16&lt;/sup&gt;). SNP, single-nucleotide polymorphism.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/5/931/F8.medium.gif" width="440" height="164"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/5/5/931/F8.medium.gif" width="440" height="164"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/5/931/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/5/931/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465978" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-48" class="first-child">Relationship between SNP heterozygosity (H) and (A) absolute value of deviation (<span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/931/F8/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/931/F8/embed/mml-math-21.gif"/></noscript></span></span>; Pearson’s correlation = 0.32; <em>P</em> = 1.938 × 10<sup>−7</sup>) or (B) magnitude and direction of deviation (<em>FE</em>; Pearson’s correlation = 0.59; <em>P</em> &lt; 10<sup>−16</sup>). SNP, single-nucleotide polymorphism.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-17"><h2 class="">Discussion</h2><p id="p-49">The 1000 Genomes Project data were generated by various sequencing centers, which relied on different sequencing platforms, read lengths, aligners and variant and genotype calling algorithms (<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">The 1000 Genomes Project Consortium 2012</a>), creating challenges to an overall assessment of data reliability. In this study, we specifically examine the performance of NGS-based genotype calls and allele frequency estimates for the highly polymorphic and intensely studied classical <em>HLA</em> genes. We took advantage of the possibility of comparing downstream genotype calls from the 1000 Genomes and <em>HLA</em> typing based on Sanger sequencing for the same set of samples to assess data quality and test hypothesis about possible biases.</p><p id="p-50">We show that the 1000 Genomes SNPs called in the <em>HLA</em> genes have many differences at the genotype level, when compared to results obtained using Sanger sequencing. However, considerably high genotype mismatching is possible with only modest deviations in allele frequencies, and we conclude that for the 1000 Genomes data allele frequency estimates for SNPs at <em>HLA</em> genes are considerably more reliable than the individual genotype calls.</p><p id="p-51">Low coverage did not explain the errors in genotypes and allele frequencies in the 1000 Genomes dataset. Instead, we found evidence that read mapping bias was responsible for those errors. Mapping bias is well known for NGS, and highly polymorphic regions such as <em>HLA</em> genes are especially susceptible to its effects (<a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Nielsen <em>et al.</em> 2011</a>), particularly when a single reference genome is used as an index for the alignment of NGS reads. In this situation, many true variants fail to be identified because they are present in haplotypes that differ from the genome used as index, and thus reads generated from these regions are not aligned and are lost. Together, these results suggest that increasing coverage would not improve allele frequency estimates at those sites if a single reference sequence is still used as index. By mapping to multiple genomes [<em>e.g.</em>, using strategies similar to <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Boegel <em>et al.</em> (2012)</a> or <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Dilthey <em>et al.</em> (2014)</a>], it would be possible to improve genotype calling and allele frequency estimates.</p><p id="p-52">In our study, <em>HLA-A</em>, <em>-B</em>, and <em>DQB1</em> show evidence of REF allele mapping bias. The <em>HLA-DRB1</em> locus, on the other hand, did not present REF allele frequency overestimation, a finding that can be explained by the existence of multiple copies of this gene (both pseudogenes and functional copies), which may result in biases/errors that make REF allele bias comparatively less visible (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Degner <em>et al.</em> 2009</a>). The <em>HLA-C</em> locus also shows a weaker REF allele bias, a pattern that may be explained by its lower degree of polymorphism which leads to a decrease in the number of mismatches of reads with respect to the reference genome, thus decreasing the mapping bias.</p><p id="p-53">We provide a list of unreliable SNPs within the <em>HLA</em> genes, defined by us as those with an absolute difference in frequency larger than 0.1 (<span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/931/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5/931/embed/mml-math-22.gif"/></noscript></span></span>) in two or more populations (<a href="http://www.g3journal.org/content/suppl/2015/03/17/g3.114.015784.DC1/TableS4.pdf">Table S4</a>). We show that these unreliable SNPs on average have greater heterozygosities in our gold standard dataset. As a consequence, although filtering out those unreliable sites improves the overall accuracy in allele frequency estimation, it leads to an underestimation of the mean heterozygosity of SNPs in <em>HLA</em> genes, a bias that should be taken into account in downstream analyses. Analyses that require genotype calls at the individual level, including haplotype-based analyses, should be performed with caution when using the data from the 1000 Genomes at <em>HLA</em> genes.</p><p id="p-54">Our results have implications to studies that use SNP data from the 1000 Genomes in other genomic regions with high variability, such as KIR and olfactory receptors. Because <em>HLA</em> loci are the most polymorphic in the human genome, they represent a worst case scenario for mapping bias and subsequent allele frequency estimation errors. We found a significant correlation between SNP heterozygosity and the absolute difference in frequency between 1000 Genomes data and our gold standard. This suggests that in genome-wide studies, SNPs with high heterozygosities, and contained within regions with additional SNPs, have an increased chance of presenting poor frequency estimates.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-55">This research was financially supported by grants from São Paulo Research Foundation (FAPESP) and The Brazilian National Council for Scientific and Technological Development (CNPq). D.Y.C.B. was funded by FAPESP scholarships #2012/22796-9 and #2013/12162-5; V.R.C.A. has a FAPESP grant #2014/12123-2, B.D.B. was funded by #2011/12500-2 (FAPESP) and #152676/2011-2 (CNPq); K.N. has a FAPESP grant #2012/09950-9; and D.M. has a FAPESP research grant #12/18010-0 and a CNPq productivity grant #308167/2012-0.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-2"><p id="p-56">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.015784/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.015784/-/DC1</a></p></li><li class="fn-other" id="fn-3"><p id="p-57">Data available in public repositories: <a href="https://github.com/deboraycb/reliability_hla_1000g">https://github.com/deboraycb/reliability_hla_1000g</a></p></li><li class="fn" id="fn-4"><p id="p-58"><em>Communicating editor: C. R. Marshall</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-12-22"><span class="received-label">Received </span>December 22, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-03-13"><span class="accepted-label">Accepted </span>March 13, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Brandt <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.931.1" data-doi="10.1098/rstb.2011.0299"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andersen</span>  <span class="cit-name-given-names">K. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shylakhter</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tabrizi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grossman</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Happi</span>  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-wide scans provide evidence for positive selection of genes implicated in Lassa fever.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">367</span>: <span class="cit-fpage">868</span>–<span class="cit-lpage">877</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilos.%2BTrans.%2BR.%2BSoc.%2BLond.%2BB%2BBiol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2011.0299%26rft_id%253Dinfo%253Apmid%252F22312054%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoicm95cHRiIjtzOjU6InJlc2lkIjtzOjEyOiIzNjcvMTU5MC84NjgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzUvOTMxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.931.2" data-doi="10.1038/329506a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bjorkman</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saper</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Samraoui</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bennett</span>  <span class="cit-name-given-names">W. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strominger</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Structure of the human class I histocompatibility antigen, HLA-A2.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">329</span>: <span class="cit-fpage">506</span>–<span class="cit-lpage">512</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBjorkman%26rft.auinit1%253DP.%2BJ.%26rft.volume%253D329%26rft.issue%253D6139%26rft.spage%253D506%26rft.epage%253D512%26rft.atitle%253DStructure%2Bof%2Bthe%2Bhuman%2Bclass%2BI%2Bhistocompatibility%2Bantigen%252C%2BHLA-A2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F329506a0%26rft_id%253Dinfo%253Apmid%252F3309677%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/329506a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3309677&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F931.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1987K320600048&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-other" id="cit-5.5.931.3" data-doi="10.1186/gm403"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boegel</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Löwer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schäfer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bukur</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Graaf</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">HLA typing from RNA-Seq sequence reads.</span> <span class="cit-source">Genome Med.</span> <span class="cit-vol">4</span>: <span class="cit-fpage">102</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BMed.%26rft.volume%253D4%26rft.spage%253D102%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgm403%26rft_id%253Dinfo%253Apmid%252F23259685%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gm403&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23259685&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F931.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.931.4" data-doi="10.1038/364033a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jardetzky</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gorga</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stern</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Urban</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Three-dimensional structure of the human class II histocompatibility antigen HLA-DR1.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">364</span>: <span class="cit-fpage">33</span>–<span class="cit-lpage">39</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBrown%26rft.auinit1%253DJ.%2BH.%26rft.volume%253D364%26rft.issue%253D6432%26rft.spage%253D33%26rft.epage%253D39%26rft.atitle%253DThree-dimensional%2Bstructure%2Bof%2Bthe%2Bhuman%2Bclass%2BII%2Bhistocompatibility%2Bantigen%2BHLA-DR1.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F364033a0%26rft_id%253Dinfo%253Apmid%252F8316295%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/364033a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8316295&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F931.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993LK81800048&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.931.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chapman</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">A. V. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Human genetic susceptibility to infectious disease.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">175</span>–<span class="cit-lpage">188</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DChapman%26rft.auinit1%253DS.%2BJ.%26rft.volume%253D13%26rft.issue%253D3%26rft.spage%253D175%26rft.epage%253D188%26rft.atitle%253DHuman%2Bgenetic%2Bsusceptibility%2Bto%2Binfectious%2Bdisease.%26rft_id%253Dinfo%253Apmid%252F22310894%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22310894&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F931.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.931.6" data-doi="10.1093/bioinformatics/btr330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Danecek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albers</span>  <span class="cit-name-given-names">C. a.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The variant call format and VCFtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2156</span>–<span class="cit-lpage">2158</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr330%26rft_id%253Dinfo%253Apmid%252F21653522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjcvMTUvMjE1NiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvNS85MzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.931.7" data-doi="10.1111/iji.12024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De Santis</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dinauer</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duke</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erlich</span>  <span class="cit-name-given-names">H. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holcomb</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">16(th) IHIW: review of HLA typing by NGS.</span> <abbr class="cit-jnl-abbrev">Int. J. Immunogenet.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">72</span>–<span class="cit-lpage">76</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BImmunogenet.%26rft.volume%253D40%26rft.spage%253D72%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fiji.12024%26rft_id%253Dinfo%253Apmid%252F23302098%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/iji.12024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23302098&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F931.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.931.8" data-doi="10.1093/bioinformatics/btp579"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Degner</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marioni</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pai</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pickrell</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nkadori</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Effect of read-mapping biases on detecting allele-specific expression from RNA-sequencing data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">3207</span>–<span class="cit-lpage">3212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp579%26rft_id%253Dinfo%253Apmid%252F19808877%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMjQvMzIwNyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvNS85MzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-other" id="cit-5.5.931.9"><div class="cit-metadata unstructured">Dilthey, A., C. Cox, Z. Iqbal, M. R. Nelson, and G. McVean, 2014 Improved genome inference in the MHC using a population reference graph. bioRxiv. Available from: <a href="http://biorxiv.org/content/early/2014/07/08/006973">http://biorxiv.org/content/early/2014/07/08/006973</a>. Accessed March 20, 2015.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.931.10" data-doi="10.1186/1471-2164-12-42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Erlich</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Next-generation sequencing for HLA typing of class I loci.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">42</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DErlich%26rft.auinit1%253DR.%2BL.%26rft.volume%253D12%26rft.spage%253D42%26rft.epage%253D42%26rft.atitle%253DNext-generation%2Bsequencing%2Bfor%2BHLA%2Btyping%2Bof%2Bclass%2BI%2Bloci.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-42%26rft_id%253Dinfo%253Apmid%252F21244689%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-42&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21244689&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F931.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.931.11" data-doi="10.1371/journal.pone.0097282"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gourraud</span>  <span class="cit-name-given-names">P.-A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khankhanian</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cereb</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">S. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feolo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">HLA Diversity in the 1000 Genomes Dataset.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e97282</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De97282%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0097282%26rft_id%253Dinfo%253Apmid%252F24988075%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0097282&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24988075&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F931.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-other" id="cit-5.5.931.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Helmberg</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feolo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunivin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoffman</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> dbMHC.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.931.13" data-doi="10.1126/science.1198878"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hernandez</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelley</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elyashiv</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melton</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Classic selective sweeps were rare in recent human evolution.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">331</span>: <span class="cit-fpage">920</span>–<span class="cit-lpage">924</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DHernandez%26rft.auinit1%253DR.%2BD.%26rft.volume%253D331%26rft.issue%253D6019%26rft.spage%253D920%26rft.epage%253D924%26rft.atitle%253DClassic%2BSelective%2BSweeps%2BWere%2BRare%2Bin%2BRecent%2BHuman%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1198878%26rft_id%253Dinfo%253Apmid%252F21330547%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzEvNjAxOS85MjAiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzUvOTMxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.931.14" data-doi="10.1371/journal.pone.0027109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill-Burns</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Factor</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zabetian</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomson</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Payami</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Evidence for more than one Parkinson’s disease-associated variant within the HLA region.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e27109</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DHill-Burns%26rft.auinit1%253DE.%2BM.%26rft.volume%253D6%26rft.issue%253D11%26rft.spage%253De27109%26rft.epage%253De27109%26rft.atitle%253DEvidence%2Bfor%2Bmore%2Bthan%2Bone%2BParkinson%2527s%2Bdisease-associated%2Bvariant%2Bwithin%2Bthe%2BHLA%2Bregion.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0027109%26rft_id%253Dinfo%253Apmid%252F22096524%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0027109&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22096524&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F931.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-5.5.931.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kitts</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feolo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Helmberg</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The major histocompatibility complex database, dbMHC.</span> In: National Center for Biotechnology Information NIH, ed. The NCBI Handbook. Bethesda: National Center for Biotechnology Information NIH, p.<span class="cit-fpage">1</span>–<span class="cit-lpage">29</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.931.16" data-doi="10.1038/nature12531"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lappalainen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sammeth</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedländer</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">’t Hoen</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Monlong</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Transcriptome and genome sequencing uncovers functional variation in humans.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">501</span>: <span class="cit-fpage">506</span>–<span class="cit-lpage">511</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D501%26rft.spage%253D506%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12531%26rft_id%253Dinfo%253Apmid%252F24037378%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12531&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24037378&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F931.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000324826300049&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.931.17" data-doi="10.1126/science.1234070"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leffler</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pfeifer</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ségurel</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Multiple instances of ancient balancing selection shared between humans and chimpanzees.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">339</span>: <span class="cit-fpage">1578</span>–<span class="cit-lpage">1582</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1234070%26rft_id%253Dinfo%253Apmid%252F23413192%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzkvNjEyNy8xNTc4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS81LzkzMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.931.18" data-doi="10.1371/journal.pone.0078410"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Major</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rigo</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hague</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bérces</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Juhos</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">HLA typing from 1000 genomes whole genome and whole exome Illumina data.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e78410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De78410%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0078410%26rft_id%253Dinfo%253Apmid%252F24223151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0078410&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24223151&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F931.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.931.19" data-doi="10.1111/j.1399-0039.2010.01466.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marsh</span>  <span class="cit-name-given-names">S. G. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albert</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bodmer</span>  <span class="cit-name-given-names">W. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bontrop</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dupont</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Nomenclature for factors of the HLA system, 2010.</span> <abbr class="cit-jnl-abbrev">Tissue Antigens</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">291</span>–<span class="cit-lpage">455</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTissue%2Bantigens%26rft.stitle%253DTissue%2BAntigens%26rft.aulast%253DMarsh%26rft.auinit1%253DS.%2BG.%26rft.volume%253D75%26rft.issue%253D4%26rft.spage%253D291%26rft.epage%253D455%26rft.atitle%253DNomenclature%2Bfor%2Bfactors%2Bof%2Bthe%2BHLA%2Bsystem%252C%2B2010.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1399-0039.2010.01466.x%26rft_id%253Dinfo%253Apmid%252F20356336%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1399-0039.2010.01466.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20356336&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F931.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275732300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.931.20" data-doi="10.1046/j.1469-1809.2001.6510001.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomson</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">How selection shapes variation of the human major histocompatibility complex: a review.</span> <abbr class="cit-jnl-abbrev">Ann. Hum. Genet.</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAnn%2BHum%2BGenet%26rft.aulast%253DMeyer%26rft.auinit1%253DD.%26rft.volume%253D65%26rft.issue%253DPt%2B1%26rft.spage%253D1%26rft.epage%253D26%26rft.atitle%253DHow%2Bselection%2Bshapes%2Bvariation%2Bof%2Bthe%2Bhuman%2Bmajor%2Bhistocompatibility%2Bcomplex%253A%2Ba%2Breview.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1469-1809.2001.6510001.x%26rft_id%253Dinfo%253Apmid%252F11415519%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1469-1809.2001.6510001.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11415519&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F931.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000168307900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.931.21" data-doi="10.1038/nrg2986"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paul</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albrechtsen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Y. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genotype and SNP calling from next-generation sequencing data.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">443</span>–<span class="cit-lpage">451</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DNielsen%26rft.auinit1%253DR.%26rft.volume%253D12%26rft.issue%253D6%26rft.spage%253D443%26rft.epage%253D451%26rft.atitle%253DGenotype%2Band%2BSNP%2Bcalling%2Bfrom%2Bnext-generation%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2986%26rft_id%253Dinfo%253Apmid%252F21587300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2986&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21587300&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F931.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.931.22" data-doi="10.1093/bioinformatics/btq033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quinlan</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">I. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">BEDTools: a flexible suite of utilities for comparing genomic features.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">841</span>–<span class="cit-lpage">842</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq033%26rft_id%253Dinfo%253Apmid%252F20110278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNi82Lzg0MSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvNS85MzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.931.23" data-doi="10.1093/nar/gks949"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Halliwell</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McWilliam</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lopez</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parham</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The IMGT/HLA database.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">D1222</span>–<span class="cit-lpage">D1227</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks949%26rft_id%253Dinfo%253Apmid%252F23080122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjExOiI0MS9EMS9EMTIyMiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvNS85MzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.931.24" data-doi="10.1016/j.coi.2014.08.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sollid</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pos</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wucherpfennig</span>  <span class="cit-name-given-names">K. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Molecular mechanisms for contribution of MHC molecules to autoimmune diseases.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Immunol.</abbr> <span class="cit-vol">31C</span>: <span class="cit-fpage">24</span>–<span class="cit-lpage">30</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BImmunol.%26rft.volume%253D31%26rft.spage%253D24%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.coi.2014.08.005%26rft_id%253Dinfo%253Apmid%252F25216261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.coi.2014.08.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25216261&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F931.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-other" id="cit-5.5.931.25" data-doi="10.1038/nature11632"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">The 1000 Genomes Project Consortium</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">An integrated map of genetic variation from 1,092 human genomes.</span> <span class="cit-source">Nature</span> <span class="cit-vol">491</span>: <span class="cit-fpage">56</span>–<span class="cit-lpage">65</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DAbecasis%26rft.auinit1%253DG.%2BR.%26rft.volume%253D491%26rft.issue%253D7422%26rft.spage%253D56%26rft.epage%253D65%26rft.atitle%253DAn%2Bintegrated%2Bmap%2Bof%2Bgenetic%2Bvariation%2Bfrom%2B1%252C092%2Bhuman%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11632%26rft_id%253Dinfo%253Apmid%252F23128226%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11632&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23128226&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F931.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000310434500030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.931.26" data-doi="10.1038/nrg3117"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Treangen</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Repetitive DNA and next-generation sequencing: computational challenges and solutions.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">36</span>–<span class="cit-lpage">46</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DTreangen%26rft.auinit1%253DT.%2BJ.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D36%26rft.epage%253D46%26rft.atitle%253DRepetitive%2BDNA%2Band%2Bnext-generation%2Bsequencing%253A%2Bcomputational%2Bchallenges%2Band%2Bsolutions.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3117%26rft_id%253Dinfo%253Apmid%252F22124482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3117&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22124482&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F931.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.931.27" data-doi="10.1126/science.1225057"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ward</span>  <span class="cit-name-given-names">L. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kellis</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Evidence of abundant purifying selection in humans for recently acquired regulatory functions.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">337</span>: <span class="cit-fpage">1675</span>–<span class="cit-lpage">1678</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1225057%26rft_id%253Dinfo%253Apmid%252F22956687%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzcvNjEwMi8xNjc1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS81LzkzMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/5/931.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/5/921" title="Charged/Polar-Residue Scanning of the Hydrophobic Face of Transmembrane Domain 9 of the Yeast Glutathione Transporter, Hgt1p, Reveals a Conformationally Critical Region for Substrate Transport" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/5/943" title="Discs large 5, an Essential Gene in Drosophila, Regulates Egg Chamber Organization" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/5"><span>Volume 5 Issue 5, May 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/5.cover-source.jpg?itok=53KRlKz0" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/5/931" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-BygfynNMUnb9QY6ebPU6_rb_3Adl7s-N6jqw18C3eUk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/5/931" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/5/931" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/5/931" class="active" data-icon-position="" data-hide-link-title="0">Mapping Bias Overestimates Reference Allele Frequencies at the HLA Genes in the 1000 Genomes Project Phase I Data</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459829" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-xbqF8jbYu1x41l9p88thY5D76vGcs0F7tyQg8IyHu_0" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/5/931.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/5/931" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-fuP6HpO_ZjbTqFz3TIeOtK0nV5EJurJq3QmElXeEV4s" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/5/931" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/5/931.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459829" id="node459829--2" data-pisa="ggg;5/5/931" data-pisa-master="ggg;g3.114.015784" data-apath="/ggg/5/5/931.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Mapping Bias Overestimates Reference Allele Frequencies at the <em>HLA</em> Genes in the 1000 Genomes Project Phase I Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Débora Y. C. Brandt</span>, <span class="highwire-citation-author" data-delta="1">Vitor R. C. Aguiar</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-7676-9367" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Bárbara D. Bitarello</span>, <span class="highwire-citation-author" data-delta="3">Kelly Nunes</span>, <span class="highwire-citation-author" data-delta="4">Jérôme Goudet</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-7155-5674" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Diogo Meyer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">931-941;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.015784 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459829--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Débora Y. C. Brandt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics and Evolutionary Biology, University of São Paulo, 05508-090 São Paulo, SP, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=D%C3%A9bora%2BY.%2BC.%2BBrandt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brandt%20DY&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AD%25C3%25A9bora%2BY.%2BC.%2BBrandt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Vitor R. C. Aguiar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics and Evolutionary Biology, University of São Paulo, 05508-090 São Paulo, SP, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vitor%2BR.%2BC.%2BAguiar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Aguiar%20VR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVitor%2BR.%2BC.%2BAguiar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Bárbara D. Bitarello </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics and Evolutionary Biology, University of São Paulo, 05508-090 São Paulo, SP, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=B%C3%A1rbara%2BD.%2BBitarello%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bitarello%20BD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AB%25C3%25A1rbara%2BD.%2BBitarello%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7676-9367" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Bárbara D. Bitarello</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Kelly Nunes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics and Evolutionary Biology, University of São Paulo, 05508-090 São Paulo, SP, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kelly%2BNunes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nunes%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKelly%2BNunes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jérôme Goudet </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Ecology and Evolution, Biophore, University of Lausanne, CH-1015 Lausanne, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J%C3%A9r%C3%B4me%2BGoudet%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Goudet%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJ%25C3%25A9r%25C3%25B4me%2BGoudet%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Diogo Meyer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics and Evolutionary Biology, University of São Paulo, 05508-090 São Paulo, SP, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Diogo%2BMeyer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Meyer%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADiogo%2BMeyer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7155-5674" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Diogo Meyer</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:diogo@ib.usp.br" class="" data-icon-position="" data-hide-link-title="0">diogo@ib.usp.br</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/5/931" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/5/931&amp;title=Mapping%20Bias%20Overestimates%20Reference%20Allele%20Frequencies%20at%20the%20HLA%20Genes%20in%20the%201000%20Genomes%20Project%20Phase%20I%20Data" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/5/931&amp;title=Mapping%20Bias%20Overestimates%20Reference%20Allele%20Frequencies%20at%20the%20HLA%20Genes%20in%20the%201000%20Genomes%20Project%20Phase%20I%20Data" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/5/931&amp;title=Mapping%20Bias%20Overestimates%20Reference%20Allele%20Frequencies%20at%20the%20HLA%20Genes%20in%20the%201000%20Genomes%20Project%20Phase%20I%20Data" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/5/931&amp;text=Mapping%20Bias%20Overestimates%20Reference%20Allele%20Frequencies%20at%20the%20HLA%20Genes%20in%20the%201000%20Genomes%20Project%20Phase%20I%20Data" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/5/931&amp;title=Mapping%20Bias%20Overestimates%20Reference%20Allele%20Frequencies%20at%20the%20HLA%20Genes%20in%20the%201000%20Genomes%20Project%20Phase%20I%20Data" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/5/931&amp;t=Mapping%20Bias%20Overestimates%20Reference%20Allele%20Frequencies%20at%20the%20HLA%20Genes%20in%20the%201000%20Genomes%20Project%20Phase%20I%20Data" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/5/931&amp;title=Mapping%20Bias%20Overestimates%20Reference%20Allele%20Frequencies%20at%20the%20HLA%20Genes%20in%20the%201000%20Genomes%20Project%20Phase%20I%20Data" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/5/931&amp;title=Mapping%20Bias%20Overestimates%20Reference%20Allele%20Frequencies%20at%20the%20HLA%20Genes%20in%20the%201000%20Genomes%20Project%20Phase%20I%20Data" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459829/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459829" id="node459829--3" data-pisa="ggg;5/5/931" data-pisa-master="ggg;g3.114.015784" data-apath="/ggg/5/5/931.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Mapping Bias Overestimates Reference Allele Frequencies at the <em>HLA</em> Genes in the 1000 Genomes Project Phase I Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Débora Y. C. Brandt</span>, <span class="highwire-citation-author" data-delta="1">Vitor R. C. Aguiar</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-7676-9367" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Bárbara D. Bitarello</span>, <span class="highwire-citation-author" data-delta="3">Kelly Nunes</span>, <span class="highwire-citation-author" data-delta="4">Jérôme Goudet</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-7155-5674" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Diogo Meyer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">931-941;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.015784 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459829--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Débora Y. C. Brandt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics and Evolutionary Biology, University of São Paulo, 05508-090 São Paulo, SP, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=D%C3%A9bora%2BY.%2BC.%2BBrandt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brandt%20DY&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AD%25C3%25A9bora%2BY.%2BC.%2BBrandt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Vitor R. C. Aguiar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics and Evolutionary Biology, University of São Paulo, 05508-090 São Paulo, SP, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vitor%2BR.%2BC.%2BAguiar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Aguiar%20VR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVitor%2BR.%2BC.%2BAguiar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Bárbara D. Bitarello </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics and Evolutionary Biology, University of São Paulo, 05508-090 São Paulo, SP, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=B%C3%A1rbara%2BD.%2BBitarello%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bitarello%20BD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AB%25C3%25A1rbara%2BD.%2BBitarello%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7676-9367" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Bárbara D. Bitarello</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Kelly Nunes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics and Evolutionary Biology, University of São Paulo, 05508-090 São Paulo, SP, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kelly%2BNunes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nunes%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKelly%2BNunes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jérôme Goudet </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Ecology and Evolution, Biophore, University of Lausanne, CH-1015 Lausanne, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J%C3%A9r%C3%B4me%2BGoudet%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Goudet%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJ%25C3%25A9r%25C3%25B4me%2BGoudet%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Diogo Meyer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics and Evolutionary Biology, University of São Paulo, 05508-090 São Paulo, SP, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Diogo%2BMeyer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Meyer%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADiogo%2BMeyer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7155-5674" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Diogo Meyer</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:diogo@ib.usp.br" class="" data-icon-position="" data-hide-link-title="0">diogo@ib.usp.br</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=931&amp;rft.epage=941&amp;rft.atitle=Mapping%20Bias%20Overestimates%20Reference%20Allele%20Frequencies%20at%20the%20HLA%20Genes%20in%20the%201000%20Genomes%20Project%20Phase%20I%20Data&amp;rft.volume=5&amp;rft.issue=5&amp;rft.date=2015-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Brandt%2C+D%C3%A9bora+Y.+C.&amp;rft.au=Aguiar%2C+Vitor+R.+C.&amp;rft.au=Bitarello%2C+B%C3%A1rbara+D.&amp;rft.au=Nunes%2C+Kelly&amp;rft.au=Goudet%2C+J%C3%A9r%C3%B4me&amp;rft.au=Meyer%2C+Diogo"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459829/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459829/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459829/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459829/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459829/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459829/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459829/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459829/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459829/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459829/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459829/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459829/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/5/931" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-17" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/5/931.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/5/931.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/5/931.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/5/931.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
