<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/5/741" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/5/741.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/5/741.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/5/741.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/5/741.atom" />
<meta name="HW.pisa" content="ggg;5/5/741" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A Systematic Mutational Analysis of a Histone H3 Residue in Budding Yeast Provides Insights into Chromatin Dynamics" />
<meta name="DC.Identifier" content="10.1534/g3.115.017376" />
<meta name="DC.Date" content="2015-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Johnson et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="10.1534/g3.115.019349" />
<meta name="DC.Description" content="In previous work using the Saccharomyces cerevisiae model system, a mutant version of [histone H3][1]—H3-L61W—was found to confer a variety of abnormal growth phenotypes and defects in specific aspects of the transcription process, including a pronounced alteration in the distribution pattern of the transcription elongation factor [Spt16][2] across transcribed genes and promotion of cryptic transcription initiation within the [FLO8][3] gene. To gain insights into the contribution of the H3-L61 residue to chromatin function, we have generated yeast strains expressing versions of [histone H3][1] harboring all possible natural amino acid substitutions at position 61 (H3-L61X mutants) and tested them in a series of assays. We found that whereas 16 of the 19 H3-L61X mutants support viability when expressed as the sole source of [histone H3][1] in cells, all 19 confer abnormal phenotypes ranging from very mild to severe, a finding that might in part explain the high degree of conservation of the H3-L61 residue among eukaryotes. An examination of the strength of the defects conferred by each H3-L61X mutant and the nature of the corresponding substituted residue provides insights into structural features of the nucleosome required for proper [Spt16][2]−gene interactions and for prevention of cryptic transcription initiation events. Finally, we provide evidence that the defects imparted by H3-L61X mutants on [Spt16][2]−gene interactions and on repression of intragenic transcription initiation are mechanistically related to each other.

 [1]: http://www.yeastgenome.org/locus/S000004976/overview
 [2]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175
 [3]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000911" />
<meta name="DC.Contributor" content="Paige Johnson" />
<meta name="DC.Contributor" content="Virginia Mitchell" />
<meta name="DC.Contributor" content="Kelsi McClure" />
<meta name="DC.Contributor" content="Martha Kellems" />
<meta name="DC.Contributor" content="Sarah Marshall" />
<meta name="DC.Contributor" content="Mary K. Allison" />
<meta name="DC.Contributor" content="Harrison Lindley" />
<meta name="DC.Contributor" content="Hoai-Trang T. Nguyen" />
<meta name="DC.Contributor" content="Jessalyn E. Tackett" />
<meta name="DC.Contributor" content="Andrea A. Duina" />
<meta name="article:published_time" content="2015-05-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="A Systematic Mutational Analysis of a Histone H3 Residue in Budding Yeast Provides Insights into Chromatin Dynamics" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;In previous work using the &lt;i&gt;Saccharomyces cerevisiae&lt;/i&gt; model system, a mutant version of histone H3—H3-L61W—was found to confer a variety of abnormal growth phenotypes and defects in specific aspects of the transcription process, including a pronounced alteration in the distribution pattern of the transcription elongation factor Spt16 across transcribed genes and promotion of cryptic transcription initiation within the &lt;i&gt;FLO8&lt;/i&gt; gene. To gain insights into the contribution of the H3-L61 residue to chromatin function, we have generated yeast strains expressing versions of histone H3 harboring all possible natural amino acid substitutions at position 61 (H3-L61X mutants) and tested them in a series of assays. We found that whereas 16 of the 19 H3-L61X mutants support viability when expressed as the sole source of histone H3 in cells, all 19 confer abnormal phenotypes ranging from very mild to severe, a finding that might in part explain the high degree of conservation of the H3-L61 residue among eukaryotes. An examination of the strength of the defects conferred by each H3-L61X mutant and the nature of the corresponding substituted residue provides insights into structural features of the nucleosome required for proper Spt16−gene interactions and for prevention of cryptic transcription initiation events. Finally, we provide evidence that the defects imparted by H3-L61X mutants on Spt16−gene interactions and on repression of intragenic transcription initiation are mechanistically related to each other.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/05/01" />
<meta name="citation_mjid" content="ggg;5/5/741" />
<meta name="citation_id" content="5/5/741" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/5/741" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/5/741.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/5/741.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/5/741.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.017376" />
<meta name="citation_pmid" content="25711831" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="741" />
<meta name="citation_lastpage" content="749" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Paige Johnson" />
<meta name="citation_author_institution" content="Biology Department, Hendrix College, Conway, Arkansas 72032" />
<meta name="citation_author" content="Virginia Mitchell" />
<meta name="citation_author_institution" content="Biology Department, Hendrix College, Conway, Arkansas 72032" />
<meta name="citation_author" content="Kelsi McClure" />
<meta name="citation_author_institution" content="Biology Department, Hendrix College, Conway, Arkansas 72032" />
<meta name="citation_author" content="Martha Kellems" />
<meta name="citation_author_institution" content="Biology Department, Hendrix College, Conway, Arkansas 72032" />
<meta name="citation_author" content="Sarah Marshall" />
<meta name="citation_author_institution" content="Biology Department, Hendrix College, Conway, Arkansas 72032" />
<meta name="citation_author" content="Mary K. Allison" />
<meta name="citation_author_institution" content="Biology Department, Hendrix College, Conway, Arkansas 72032" />
<meta name="citation_author" content="Harrison Lindley" />
<meta name="citation_author_institution" content="Biology Department, Hendrix College, Conway, Arkansas 72032" />
<meta name="citation_author" content="Hoai-Trang T. Nguyen" />
<meta name="citation_author_institution" content="Biology Department, Hendrix College, Conway, Arkansas 72032" />
<meta name="citation_author" content="Jessalyn E. Tackett" />
<meta name="citation_author_institution" content="Biology Department, Hendrix College, Conway, Arkansas 72032" />
<meta name="citation_author" content="Andrea A. Duina" />
<meta name="citation_author_institution" content="Biology Department, Hendrix College, Conway, Arkansas 72032" />
<meta name="citation_author_email" content="duina@hendrix.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6040-6825" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=N. K. Brewster;citation_author=G. C. Johnston;citation_author=R. A. Singer;citation_title=Characterization of the CP Complex, an Abundant Dimer of Cdc68 and Pob3 Proteins That Regulates Yeast Transcriptional Activation and Chromatin Repression;citation_pages=21972-21979;citation_volume=273;citation_year=1998;citation_issue=34;citation_pmid=9705338;citation_doi=10.1074/jbc.273.34.21972" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=E. I. Campos;citation_author=D. Reinberg;citation_title=Histones: annotating chromatin.;citation_pages=559-599;citation_volume=43;citation_year=2009;citation_pmid=19886812;citation_doi=10.1146/annurev.genet.032608.103928" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=V. Cheung;citation_author=G. Chua;citation_author=N. N. Batada;citation_author=C. R. Landry;citation_author=S. W. Michnick;citation_title=Chromatin- and transcription-related factors repress transcription from within coding regions throughout the Saccharomyces cerevisiae genome.;citation_pages=e277-e277;citation_volume=6;citation_year=2008;citation_issue=11;citation_pmid=18998772;citation_doi=10.1371/journal.pbio.0060277" />
<meta name="citation_reference" content="citation_title=Genet. Res. Int.;citation_author=A. A. Duina;citation_title=Histone chaperones Spt6 and FACT: similarities and differences in modes of action at transcribed genes.;citation_pages=625201;citation_volume=2011;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=A. A. Duina;citation_author=F. Winston;citation_title=Analysis of a Mutant Histone H3 That Perturbs the Association of Swi/Snf with Chromatin;citation_pages=561-572;citation_volume=24;citation_year=2004;citation_issue=2;citation_pmid=14701730;citation_doi=10.1128/MCB.24.2.561-572.2004" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. A. Duina;citation_author=A. Rufiange;citation_author=J. Bracey;citation_author=J. Hall;citation_author=A. Nourani;citation_title=Evidence that the localization of the elongation factor Spt16 across transcribed genes is dependent upon histone H3 integrity in Saccharomyces cerevisiae.;citation_volume=177;citation_year=2007;citation_pmid=17603125;citation_doi=10.1534/genetics.106.067140" />
<meta name="citation_reference" content="citation_journal_abbrev=Mol Biosyst;citation_author=T. Formosa;citation_title=FACT and the reorganized nucleosome.;citation_pages=1085-1093;citation_volume=4;citation_year=2008;citation_issue=11;citation_pmid=18931784;citation_doi=10.1039/b812136b" />
<meta name="citation_reference" content="citation_journal_title=Biochimica et Biophysica Acta. Protein Structure and Molecular Enzymology;citation_journal_abbrev=Biochimica et Biophysica Acta. Protein Structure and Molecular Enzymology;citation_author=T. Formosa;citation_title=The role of FACT in making and breaking nucleosomes.;citation_pages=247-255;citation_volume=1819;citation_year=2012;citation_issue=3-4;citation_pmid=21807128;citation_doi=10.1016/j.bbagrm.2011.07.009" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO J.;citation_author=T. Formosa;citation_author=P. Eriksson;citation_author=J. Wittmeyer;citation_author=J. Ginn;citation_author=Y. Yu;citation_title=Spt16-Pob3 and the HMG protein Nhp6 combine to form the nucleosome-binding factor SPN;citation_pages=3506-3517;citation_volume=20;citation_year=2001;citation_issue=13;citation_pmid=11432837;citation_doi=10.1093/emboj/20.13.3506" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=M. Hondele;citation_author=T. Stuwe;citation_author=M. Hassler;citation_author=F. Halbach;citation_author=A. Bowman;citation_title=STRUCTURAL BASIS OF HISTONE H2A H2B RECOGNITION BY THE ESSENTIAL CHAPERONE FACT.;citation_pages=111-114;citation_volume=499;citation_year=2013;citation_pmid=23698368;citation_doi=10.1038/nature12242" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=F. K. Hsieh;citation_author=O. I. Kulaeva;citation_author=S. S. Patel;citation_author=P. N. Dyer;citation_author=K. Luger;citation_title=Histone chaperone FACT action during transcription through chromatin by RNA polymerase II.;citation_volume=110;citation_year=2013;citation_pmid=23610384;citation_doi=10.1073/pnas.1222198110" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=A. Jamai;citation_author=A. Puglisi;citation_author=M. Strubin;citation_title=Histone chaperone spt16 promotes redeposition of the original h3-h4 histones evicted by elongating RNA polymerase.;citation_pages=377-383;citation_volume=35;citation_year=2009;citation_issue=3;citation_pmid=19683500;citation_doi=10.1016/j.molcel.2009.07.001" />
<meta name="citation_reference" content="citation_journal_title=Eukaryot. Cell;citation_author=A. Lloyd;citation_author=K. Pratt;citation_author=E. Siebrasse;citation_author=M. D. Moran;citation_author=A. A. Duina;citation_title=Uncoupling of the patterns of chromatin association of different transcription elongation factors by a histone H3 mutant in Saccharomyces cerevisiae.;citation_volume=8;citation_year=2009;citation_pmid=19047363;citation_doi=10.1128/EC.00348-08" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in structural biology;citation_journal_abbrev=Curr Opin Struct Biol;citation_author=K. Luger;citation_author=T. J. Richmond;citation_title=DNA binding within the nucleosome core.;citation_pages=33-40;citation_volume=8;citation_year=1998;citation_issue=1;citation_pmid=9519294;citation_doi=10.1016/S0959-440X(98)80007-9" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=K. Luger;citation_author=A. W. Mader;citation_author=R. K. Richmond;citation_author=D. F. Sargent;citation_author=T. J. Richmond;citation_title=Crystal structure of the nucleosome core particle at 2.8 A resolution.;citation_pages=251-260;citation_volume=389;citation_year=1997;citation_issue=6648;citation_pmid=9305837;citation_doi=10.1038/38444" />
<meta name="citation_reference" content="citation_journal_title=Genetics.;citation_author=L. McCullough;citation_author=R. Rawlins;citation_author=A. E. Olsen;citation_author=H. Xin;citation_author=D. J. Stillman;citation_title=Insight into the mechanism of nucleosome reorganization from histone mutants that suppress defects in the FACT histone chaperone.;citation_volume=188;citation_year=2011;citation_pmid=21625001;citation_doi=10.1534/genetics.111.128769" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=C. N. Myers;citation_author=G. B. Berner;citation_author=J. H. Holthoff;citation_author=K. Martinez-Fonts;citation_author=J. A. Harper;citation_title=Mutant versions of the S. cerevisiae transcription elongation factor Spt16 define regions of Spt16 that functionally interact with histone H3.;citation_pages=e20847-e20847;citation_volume=6;citation_year=2011;citation_issue=6;citation_pmid=21673966;citation_doi=10.1371/journal.pone.0020847" />
<meta name="citation_reference" content="citation_title=G3 (Bethesda);citation_author=H. T. Nguyen;citation_author=W. Wharton;citation_author=J. A. Harper;citation_author=J. R. Dornhoffer;citation_author=A. A. Duina;citation_title=A nucleosomal region important for ensuring proper interactions between the transcription elongation factor Spt16 and transcribed genes in Saccharomyces cerevisiae.;citation_volume=3;citation_year=2013;citation_pmid=23576521;citation_doi=10.1534/g3.113.005926" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=O. J. Rando;citation_author=F. Winston;citation_title=Chromatin and Transcription in Yeast;citation_pages=351-387;citation_volume=190;citation_year=2012;citation_issue=2;citation_pmid=22345607;citation_doi=10.1534/genetics.111.132266" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=D. Reinberg;citation_author=R. J. Sims;citation_title=de FACTo nucleosome dynamics.;citation_volume=281;citation_year=2006;citation_pmid=16766522;citation_doi=10.1074/jbc.R600007200" />
<meta name="citation_reference" content="citation_title=Methods in Yeast Genetics: A Laboratory Course Manual;citation_author=M. D. Rose;citation_author=F. Winston;citation_author=P. Hieter;citation_year=1990" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=S. Ruone;citation_author=A. R. Rhoades;citation_author=T. Formosa;citation_title=Multiple Nhp6 molecules are required to recruit Spt16-Pob3 to form yFACT complexes and to reorganize nucleosomes.;citation_volume=278;citation_year=2003;citation_pmid=12952948;citation_doi=10.1074/jbc.M307291200" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. S. Sikorski;citation_author=P. Hieter;citation_title=A system of shuttle vectors and yeast host strains designed for efficient manipulation of DNA in Saccharomyces cerevisiae.;citation_pages=19-27;citation_volume=122;citation_year=1989;citation_issue=1;citation_pmid=2659436" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO J.;citation_author=C. L. White;citation_author=R. K. Suto;citation_author=K. Luger;citation_title=Structure of the yeast nucleosome core particle reveals fundamental changes in internucleosome interactions;citation_pages=5207-5218;citation_volume=20;citation_year=2001;citation_issue=18;citation_pmid=11566884;citation_doi=10.1093/emboj/20.18.5207" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=D. D. Winkler;citation_author=K. Luger;citation_title=The histone chaperone FACT: structural insights and mechanisms for nucleosome reorganization.;citation_volume=286;citation_year=2011;citation_pmid=21454601;citation_doi=10.1074/jbc.R110.180778" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=D. D. Winkler;citation_author=U. M. Muthurajan;citation_author=A. R. Hieb;citation_author=K. Luger;citation_title=Histone chaperone FACT coordinates nucleosome interaction through multiple synergistic binding events.;citation_volume=286;citation_year=2011;citation_pmid=21969370;citation_doi=10.1074/jbc.M111.301465" />
<meta name="citation_reference" content="citation_title=Transcription Regulation;citation_title=Analysis of SPT genes: a genetic approach toward analysis of TFIID, histones, and other transcription factors in yeast;citation_author=F. Winston;citation_pages=1271-1293;citation_year=1992" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=F. Winston;citation_author=C. Dollard;citation_author=S. L. Ricupero-Hovasse;citation_title=Construction of a set of convenient Saccharomyces cerevisiae strains that are isogenic to S288C.;citation_pages=53-55;citation_volume=11;citation_year=1995;citation_issue=1;citation_pmid=7762301;citation_doi=10.1002/yea.320110107" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=H. Xin;citation_author=S. Takahata;citation_author=M. Blanksma;citation_author=L. McCullough;citation_author=D. J. Stillman;citation_title=yFACT induces global accessibility of nucleosomal DNA without H2A-H2B displacement.;citation_pages=365-376;citation_volume=35;citation_year=2009;citation_issue=3;citation_pmid=19683499;citation_doi=10.1016/j.molcel.2009.06.024" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Biol.;citation_author=S. Zheng;citation_author=J. B. Crickard;citation_author=A. Srikanth;citation_author=J. C. Reese;citation_title=A highly conserved region within H2B is important for FACT to act on nucleosomes.;citation_volume=34;citation_year=2014;citation_pmid=24248595;citation_doi=10.1128/MCB.00478-13" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A Systematic Mutational Analysis of a Histone H3 Residue in Budding Yeast Provides Insights into Chromatin Dynamics" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5.cover-source.jpg" />
<meta name="twitter:description" content="In previous work using the Saccharomyces cerevisiae model system, a mutant version of [histone H3][1]—H3-L61W—was found to confer a variety of abnormal growth phenotypes and defects in specific aspects of the transcription process, including a pronounced alteration in the distribution pattern of the transcription elongation factor [Spt16][2] across transcribed genes and promotion of cryptic transcription initiation within the [FLO8][3] gene. To gain insights into the contribution of the H3-L61 residue to chromatin function, we have generated yeast strains expressing versions of [histone H3][1] harboring all possible natural amino acid substitutions at position 61 (H3-L61X mutants) and tested them in a series of assays. We found that whereas 16 of the 19 H3-L61X mutants support viability when expressed as the sole source of [histone H3][1] in cells, all 19 confer abnormal phenotypes ranging from very mild to severe, a finding that might in part explain the high degree of conservation of the H3-L61 residue among eukaryotes. An examination of the strength of the defects conferred by each H3-L61X mutant and the nature of the corresponding substituted residue provides insights into structural features of the nucleosome required for proper [Spt16][2]−gene interactions and for prevention of cryptic transcription initiation events. Finally, we provide evidence that the defects imparted by H3-L61X mutants on [Spt16][2]−gene interactions and on repression of intragenic transcription initiation are mechanistically related to each other.

 [1]: http://www.yeastgenome.org/locus/S000004976/overview
 [2]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175
 [3]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000911" />
<meta name="og-title" property="og:title" content="A Systematic Mutational Analysis of a Histone H3 Residue in Budding Yeast Provides Insights into Chromatin Dynamics" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/5/741" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="In previous work using the Saccharomyces cerevisiae model system, a mutant version of [histone H3][1]—H3-L61W—was found to confer a variety of abnormal growth phenotypes and defects in specific aspects of the transcription process, including a pronounced alteration in the distribution pattern of the transcription elongation factor [Spt16][2] across transcribed genes and promotion of cryptic transcription initiation within the [FLO8][3] gene. To gain insights into the contribution of the H3-L61 residue to chromatin function, we have generated yeast strains expressing versions of [histone H3][1] harboring all possible natural amino acid substitutions at position 61 (H3-L61X mutants) and tested them in a series of assays. We found that whereas 16 of the 19 H3-L61X mutants support viability when expressed as the sole source of [histone H3][1] in cells, all 19 confer abnormal phenotypes ranging from very mild to severe, a finding that might in part explain the high degree of conservation of the H3-L61 residue among eukaryotes. An examination of the strength of the defects conferred by each H3-L61X mutant and the nature of the corresponding substituted residue provides insights into structural features of the nucleosome required for proper [Spt16][2]−gene interactions and for prevention of cryptic transcription initiation events. Finally, we provide evidence that the defects imparted by H3-L61X mutants on [Spt16][2]−gene interactions and on repression of intragenic transcription initiation are mechanistically related to each other.

 [1]: http://www.yeastgenome.org/locus/S000004976/overview
 [2]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175
 [3]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000911" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/5.cover-source.jpg" />
    <title>A Systematic Mutational Analysis of a Histone H3 Residue in Budding Yeast Provides Insights into Chromatin Dynamics | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"573UO-NHQVNv1u5-BqKolCt9bZd3yyZ3WmeeBf1mn6M","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459889","apath":"\/ggg\/5\/5\/741.atom","pisa":"ggg;5\/5\/741","ac":{"\/ggg\/5\/5\/741.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/5\/741.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/5\/741"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/5\/741"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/5\/741":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022rDghZAD5jpRli6apHf1bEoUdqa+bBw+TrjJIrI6Bl\\\\\\\/mio2qxXCai0sjiwHuD8rQPI3B9\\\\\\\/tMmzGBbHFg9RfgtTjJ8sJAOANGBlUbi5E8qzH+PcrgqD+90fz\\\\\\\/EzCfFFEjd4xLNWbfuRr5uoURK3SR4zDTX12W4YRlVQrZWYrO+srmosOkLV7Fa6o4SBJk4j0Sn0G61YVclvM8rJ5l5ESVxikNOd8OmExHzxBihL3G8Io7naNT1KNcUbN+RM\\\\\\\/YSXDV2AmaSu0+EO3qw8hPpEIX6dslx\\\\\\\/OhMMbbm0Q\\\\\\\/k8fWgLhQVHc4XB0dHwioHWuPhC2nfOlS8Aa9nc6B2eKh8GaqrQi9y5KZ0+lx8Yk9zdQ1pjpLRqC34N+wSNtGWevjJbytQ9B4Gup847yuRccPOB8Awr3+CNxU8CUhalFIT8qa4TSP8MHRdx4oosasKrtZ0wGuptMkxNt7gMKpf1iZYTyNEURMcN\\\\\\\/MoIhLyx+ke+A8VZc\\\\\\\/dyuN26Y\\\\\\\/cDa7rbm0AcXvQJIqsxvLrhNtctLlVMqpNWmN4HRX900FpLSC3QnzSe9m5IAOnhYxnYtM\\\\\\\/+5ZJtdRMdUadZQzvXudd4A3UlvHGn078gWnfF3Dm7QFIxwikgHPN8cM\\\\\\\/JbtDlgUBO4RVdnabrmfXImVOPj0x3YV2pCLAo8I997IIPiCa7dD1JkrEA3Xfdbfx\\\\\\\/UBw9ePWXa5xuADgOplEwKJvP1QFUmcvgl5TM\\\\\\\/IowRcN0+7uFmB637sujjUv+mqLNwYI7e3h87H7gtaZYPI9snPO4mKCuHmMCbYaH6pQV\\\\\\\/OK+Vqb4stktXzyyLKtUNjtg2zoLGt5u\\\\\\\/qhvwP+5QCIwiPFFPvZ+otWXC+wfc56BCNvNhqFNv1yvzdgMYwWCIWLMIH7TKBH+XapcVYIzT8lWvoEUUBI0P7GpU9SY\\\\\\\/+rbYHgfjBDQeY0hDm7QLnjmsdBD276xb5CrygqKz86olYur7SYccxYEeUdrjbMf1omraLoCIb1+UGFE7IpKV9xsxZkatcfP\\\\\\\/dj9aEn56XXSrmkoBAGOvgHZQe66Kxv1qWH3J4JPJa6DScTfQF5cCMavwEaDDBamguqYgtqzgShrXxyM0LRGZ7ynamj9UUUZs3oFslUOcAsxN+EddeV5hU66WT81174GzoxiLHpc6eXlg5F+aL4g8JLq\\\\\\\/+b245VjFSXUl5euF33q\\\\\\\/pmWOq2BQsX7qAhpJKsCet\\\\\\\/lCWoq2Lti5e4zmsN0CRwGCHr2DCWpltigLEgtpXK3eBvAFCL1qBxuw09amHTKCW+HpwYXIx\\\\\\\/\\\u0022,\\\u0022iv\\\u0022:\\\u0022Dt\\\\\\\/nCJAgeRfc76Z7C2XJ7A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ca21fa9a323401dd81996112dd21eaeb4ab43464b3caccbc7a0afad9fc735c00\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022jOZYfPYWVBlTBJi\\\\\\\/480b5pXjrKvhQN1pquS0ArvBwjflFUskX2d2nT5wcqsmSyXPaXg8Ur7cbfN915AMHD3nloGEJz5djHvt6pVlfxL5OziEqeTzjHR7cetiOIYSOaJgnTs7DfoNdN0cbo96\\\\\\\/M5bPcQvdUI\\\\\\\/wHPIbKG8SC2Cyw+JruLaAZT8D\\\\\\\/lavnpPPi+GiqBCKRDdeb6LgrjlExVhojm4zMx1GqTgwGEupbLg9HkrWCONF3rq9B6Q45aZXNF\\\\\\\/HZMMJPALxGvl9\\\\\\\/1+4k5zpTORQVQWSvvV1b97zBf3+GCJSdz7lZCdJ9LfUen2jB3Qb2tWkHrhCbOPkBjiN4wSbWq8U1eeTS76PjzW5A3g7QnSw3XRfQyMWaj5APr3oQNgjzIWsc+CoqgqpNElZGbRuU5LW32JWr8uvyA9BZsyf\\\\\\\/4KMMcLL0uqOklA\\\\\\\/J\\\\\\\/MsetTMsMnIUM3aGmdfroh5Al3ZgKdNVlQN6XVRrePz0E7Zfq5Kz0tSswHswlqiCfOMw9r940Si4RDGk13BkEXkzLklbJ3q+PXWNdgze22lcQX7S3n1XJWG0VJ7L3tvj37S1kTUt5MUAHIWCXyPC4vQNiydO0P2PldzpkaNiNMgtKqyrW50f8b4+rwr\\\\\\\/KVwXQ0lQIAJK7tT7\\\\\\\/52El41ct1QUE2NSB7a4puHK8hKzMfyNfcseDExaXDQGoCEypUbKMkKNSq6cMqG\\\\\\\/PtQw83VRHQ5orZIS4gKW319shb\\\\\\\/SClpmbMtlD5ryqPO8SHP5Gl3ZHjkfO7JYjpVjJTavTQr48ty8FAutt6MSc0AGOoinSZXuHqWiDOsOmUVqzLiMPydSyk0Hfp3QDu2+AlX\\\\\\\/ZiEaInpzk7WohjHCt8LIq25ohQkkcXYe0I0lIZXSpAthJHKyFTBSOIhRShl2DvyiVhWX0QOyZskt9C4+Bdn\\\\\\\/7swYlqIJlp3HqPQHRB3S7hDmeqOJWjkIGDVyqIch6XyZ9teJ84oikq1CWpQPsTdS0hYm\\\\\\\/G8N84iAe2vNDZ5Kxg29aHmSxAt3v2h8dEC6ZqR4nlg4lKEyCjeq45XP8zRM984kUeNLAOd+h\\\\\\\/PjgoaklP\\\\\\\/+76xGSNSfb2lGo5LpL3vnfavBuXVCySB6JsOxIuM6a8ClUm5bI8jcjyFCO8xzbv9C9l5wzLd7hb37jTfj6ZH76Ectwyz5QhosZ2UGEl5wzwcIWdlVwlm4Fs+14lGT8LHDbCtq70mloEhoTJ3\\\\\\\/SS5exbRISZiWf3xtc539P7IOC+Rih3GIRk9aAwoyAg+ufHB2zzEvzvHFPwk+7qg+kICGQML75Z05C+Sb\\\\\\\/LTdzB7xYfCKjRz96Rt88x1xK\\\\\\\/Hjpow6fY\\\\\\\/hrJlZ094c28FTa78szJnY\\\\\\\/byWH0qNOM4RxQDf487jKV6zPIBAYkcO\\\\\\\/22nlB1izGF++TKSyvbTqhjGteWGO9k5iwHPgQaxZxY9jM5hc6z3VIjRi\\\\\\\/WV5HL7f1GXsbDqah5BiGHiZtb9TJuLqT0m9iQL3PiQYuwmLGpqqbFpkxN+0xY0hNnMWpiaC18jG5tmiMyhbDrscK1rj\\\\\\\/DCpwhh1SL7aVy8sOMVaSkxjvn0ZnUroGtzYLCx5Nfzf2h1S+0Xm7gZA9rKWegFB+uS4TbE4uc96QqgyCZQIlghe3zIwvBPxIc4CGMNrTKWaYs2x0Jivji6govS6f5n+U\\\\\\\/bxwGWGiZYWrqEhPvu+14P4Dg9bJ\\\\\\\/hZiCAQUrJgsD57DrwEDU56pA+aciXWU32SFGfg9jBk0k0ayTRx5uWBUhg\\\\\\\/Wn6LNJq+xWVZUWUNqm\\\\\\\/7VIpqVK\\\\\\\/oFL+ml5VLGFKfRM6+OyJPVGcM7ks2jhrPds5QQYdQLZoM\\\\\\\/bUdCVH4jL0uu3DlGLri6qGPTAgoG\\\\\\\/R+S2BL0VI2soL1A+p7fm4zAq9k6a4X6y6Hy9Xg0jt9awsUdGa7tfRdTPeJlQo785nY5NT4oPSOxSb43IiCFEV8J6ltISF6WtPrwOFKailpZ6nPKzlQBF7sppuHTIZjOgV8Rvg44LNl0ZRX8Pzp3+uGM2cUgrhhcosQc8oL\\\\\\\/3XWV1XcqDUcan9GVYUSCnmj0CA3323ssVuoVpogOTL5lijEh0N6a5xDS32PEEI6izxBrgDTHNsBDjU66dLHH9kmED55cNevZsWRAbem2HFIF3dFTJ7efs4qClTM3kGLOyp53cKQvQ\\\\\\\/6mpU5jkghle7mzXIgrq7cRSO7JY0UirtbNkb3LuoHRaIPGtl6E6Fr1VdRLj8of3cw0MNRx30\\\\\\\/yz0FXO\\\\\\\/9ds\\\\\\\/IIH+KRMPD2ptffZZoOmJXBKK9SojJCA3YnwyiSW2qFm6RYwMr5Rl29GsZGuPnZYCgqvVQXo4Q9CFX+oe5e8vKr7albaEuI5FvZcC9F4X0f\\\\\\\/U60QBp2xAm5p1doxxRiFada57rTSIuje0ML1F7WhRElSQd0ITFiDa1vxAu7IFR0XfNABJTw22ejdyW7\\\\\\\/vb7oRys60HPSg0Ua\\\\\\\/FYduQ6Hkuml9j4f40JR4O6PcmKVMqBjPR21CC6udh4I+uyo2NjJnxIcZs1kLfz5nlk47H4YdoBACfSeTgcHcI5HxxNS6IhKRmRvCqwPaaT3OQ6jdXdPd2fKXRYPrTUBFxJLBNoTGaiFT3WwCqkDML+KkdDwD3IcBWEMv4YWxBNjbA2o+\\\\\\\/bT7YuqKzUMesLlyNtnzSHGgvs3iHneCLLyDvQhPL\\\\\\\/58wNFuSld6f40hVpTuzOCXW9+qrap+sp7FWl46+w7hNytNRb7AHEYDZP6dfdDLAANDY5b3zl1rwY1b7iFuisrpTTP5L+i7bCXv8f7\\\\\\\/yjziDOBiPENia8IBsZhZ4NWVUJ+xrIcEpPankbVvRIB4QvB7GC0NygL7NwfPSNu86WtQ0fVj+GKNT8BZevpQqlXuC4rXXp2n9JzrBjt0E52be0T+x6m3eAWILZ\\\\\\\/qmyaphRYom+jgxHnF2\\\\\\\/8FPQe3M8d5n6DIZLOSwP9d3CRFxAnm0wsUWx6GK4rW8c5ADKYAufwEvGcGJ90Tlo69T15NrODygAsfoelcCs+9qOaIjYMlQCgs3Ai\\\\\\\/8xfmFOtBKhp8wgJjGBfbiXgGNOYNXwCkRf0\\\\\\\/cFfpGXvwleCecwfq3z44MPq+KD+9u1ui99V\\\\\\\/9NuDbM0B8aBRD0Gbj6Af0Gw8xejgIxuWeorWQP85bn\\\\\\\/qjiDAsJyX\\\\\\\/vujihSwHgbDZSgJ2acya8Bp05+YnAbnXFpJXDzta1gyBMBMf5vYY87S1SiSCvTsCXNgzah6YgeLOxFcXpJnTO+\\\\\\\/YgaaiA+8eLvVglr+S5q2QWKOjT6gaFP+PDt6Ug\\\\\\\/5ghu6YHOUkpzX\\\\\\\/3UK+CKEh2OD\\\\\\\/YKPwbea4i4BZWnbYtN4oS2rhCqU8nIv+xZ90JpAxxlA2AekcO50\\\\\\\/kPT\\\\\\\/N9qsR5\\\\\\\/E7n2\\\\\\\/eSlTabQJVGPIhil0xHYkaE8uh3RV2H8ahShh7n0S5Smn4ACwqi9LHlJHgXCVSl\\\\\\\/ThPggioQ1xFVQrF3XEiLJGLMwZexAwEP0F9IE3CymUv7Ae4xI+GFUA6PlRAuulBtc4fNlba4t\\\\\\\/xyPBzmZTO+313hS1ICpdJD3QnGdff9iUnDMrQAlLyijvjpwKIPdXU\\\\\\\/lxOMVR07\\\\\\\/SfsDnltqdYdCzFsE244hfGbGGIK6OaDND\\\\\\\/695J+3nW\\\\\\\/DvbrlRPEeo3ztUBxVRUTa6wyyoUjTJ33teJuQFx2HAWCNgKXtx5HYVavJZ2T2fVMERHu7j5nQKU3mAZDZOve7rQMmNBI\\\\\\\/W2wXtklg4AZ68bMuGxVxciEzRR5\\\\\\\/uPFlPCXhuKx3qlbaSabC92uJPHfprbX0omFCfeeYMIfBpL+dQI2dgzi7dv9BsOArXSnHAdkI4XDVTKZkELVubLUwtrm\\\\\\\/p9ja\\\\\\\/uuYPESEeChRkjyzYM0ZgjnSza2X46IL6jShycC76N\\\\\\\/JzrgKAtrj\\\\\\\/REUOn05mwYM4gm9bPZ43STEFlFI0baHSA+UG3OR+GGtP6RB4G05UlbfUzG+RBS162nwwD21lT10XbeYL4G4X2Jhr2WjSPIrfZ9C9NnbNdHpi+vFEHlCfzMXZft+s2gIPsNiCXq9Kh+hBuQeZK6HNtV3tzh\\\\\\\/9AudDIBzdo6rD5LadDh7zM3A3E9PNtOOp4ffvzt6TO8oJkoQIZqobhDDU+Bb+p7IvVOYPfhao\\\\\\\/rxaOxKt1fuOvdVQVZLr\\\\\\\/HXchOBhWxkM4wStLYCY1PxmtBivSgXbnvX5teJ\\\\\\\/OGKGqA8QJNxdwnm3Evt+uz1yRSQSTudJ5DhT+bSIqaIlhbSG5gxmbGqWg2wAiqKCXUeTiBf9LP0r+Sc6dBogsV+4RzoEBkpX0G3BkcV8CnjGRSiDtMT+ExkDZiiWFjJXPy0WLS3iV05qS52Q2BWf56bSo5sBeueSMFEduLVoGAyzLuaFYKAExn\\\\\\\/jcotzYXzaxr1YYQp0n7SD777dGUVNEej8I5GcwJKLjAxgoWFfBQl8\\\\\\\/KmXeSg3K2SfjZlz19JA\\\\\\\/pFsrQrlf4gjy\\\\\\\/5o2e54i5tq3ypjaRv5GaQ9CPBudS\\\\\\\/DsKXtJh6ctRLHF9M3sypdAzQ2lBeqDu2YnX1aV6cMHtuLtcd+4MrgPPpp74EsuQ1mtNMaIInWtkV7ABR3Ldqyofcqr3mxR6u9sf0+mhOpz2R7OPQs2Ydo\\\\\\\/18iQ3zjTV0Ce\\\\\\\/upqm81eVxJlKB1EYlgnQq1x9\\\\\\\/YNtz5zSVf4jPws7k2Ma4Q05SZFde6Bt70B\\\\\\\/oh+tNcQLy9n\\\\\\\/IJRohFQp4E8OYVT5AhKEcjzoT+Y2MVmGZtPcYIl6udZVZk3riR9ti6ksQnUMgBls+665utlfjpkArq+2NAtFAChpTjok+FwArZwXhAgtMikzrVzeFwIVxOa9rczOG+QBVLgH765WlNw7YH+Ue7RlMH3YRnc2daXROWxSVHZapbh5RaBXWAFEq2rpfZHu4NI\\\\\\\/dlUJSbGETrffOf75JMefUgKgHBq7cyHHoyRETkKjOzZJHEPPsOog4mOYuHGVOkXieukt2zI9WMr1Un0VAurnspQidIVMSIsss0sthrgwt\\\\\\\/kKYLgfh82ux2w9DtrBPD\\\\\\\/s4zXcLAsFydGIttS422Vz\\\\\\\/LLaKu9kWp5CXU\\\\\\\/WcvpiNUPJtnALJBr4s\\\\\\\/YfrekXXrfydvvGAlY6gLdIEInCKXjnxYQd4XTzuqV0kRq\\\\\\\/4TTTRrh\\\\\\\/Ij4nHw5LHX\\\\\\\/7BCuVXtMWhSgp0eWYKi\\\\\\\/n3wdqC7wclgUFjbTCbqns\\\\\\\/vFcolhxtvn0GDaEj8bw4PpjAHNwNOh5QE0S+fKOToQKrErdBXD7RQKsnbQwBbL\\\\\\\/GmsURFLXOk29eze4EK7pulYI2+2bRTXWEyXpGoJnzIA3n9XHrzmTOAIW1m\\\\\\\/zOkBs\\\\\\\/sVs\\\\\\\/L\\\\\\\/\\\\\\\/AJzPU5nly6j+mi6kSnBIcvyM0\\\\\\\/IqCEtMQ3IAn9W3\\\\\\\/EL\\\\\\\/CgwZpl\\\\\\\/Gb4lPPt1z0K716wyCdsMTLw+jnwovCXdvROxcntQSCIe7weZ6c0zNVDknsKa+y+k0g2vUVP9YQQS1Hapv5kvLx\\\\\\\/y4Ouy7jSg4bovMObwv1Gj8kpqayUVROttGDzCaO8a158CxUvWBMLDgWOc1fnht7HoqzaYoXJx2lkdDEyBql3tDa+1C3zaPZ4xKiYl7O49rqzISXs2almbcLPvbVsqCQ2tPV\\\\\\\/RYJdovPNjpm0unXNENfWg0sAikUwYJXwy6izv6Pt5Xer1lY+EDTmxtlV4ACXM3vFhTLIqI3g46AbiP63zQJPPx+UUbDE\\\\\\\/GADVHnghcqPfjDqJpu0eDnKq3vIJSkwDHTHLLPqiyx5kwpjZoIB3Tzx2DVu1MgzrvZxRNrTxdc\\\\\\\/i6KfID2Gv4JktlSvNSdVwEEXAhbag4iZwMkv+2UYOMRU0RvKSUvQSwzVcdtlh5d\\\\\\\/omYW4wEwrC3uxQrTiJVivDn4MFtHAoCajIPJ2pbr7QuME3k8idlfhz338CRpcsoUUlnXi07cywv20\\\\\\\/1SQ4ZZ5jBfvrkAvx2o8MeTYRcQ9CtExGgSgukDxL3aOACBoNB+JWlrJebLJengqNAn22E7jZdLuyMWvvwU2VL+HNa6YajfQ8biohMfdhpaw5PPRlBgSR5iyZ\\\\\\\/BjjrD1MsAsobVdN6u9WY4g\\\\\\\/NZ2l1O\\\\\\\/fTIwiSiGLcc8zitX77N8MN4sPgqNqJIVESgOwOAauhmIKE1b+1GqjKriN+ZKe72DtOxIeH39T\\\\\\\/1rR0gKpK29ivmOx+bAQ5e857nwHivq039dd7AixwmWwCBDOyZdwJNH20XAoOh1svMxaFP92JlmBCdeVTmkGUCiRWijvREXdaI\\\\\\\/ITNslqgh1m0o4qAgX4QitnhqJq1NDV2G6DEdFC4\\\\\\\/uNsvPniSKV55RjMMC9JeIxF2yUaLPbPBe86nXlwGLl5cn1RYhJvgnmc8crSBFRyhV62J5dS\\\\\\\/kOIcauPFgiqV1wTk2STVDXF0+TsJC\\\\\\\/6x6P3J9QiLTA1ZazLYqlSHhVxKJAVPRQ8zwXVIlYWo9kRAklKW0WPMn\\\\\\\/WjfBQG0KtDis\\\\\\\/HKohr77bIecCi+T+YsMHix20drWs13hOx3nqxR5hAy8Tf10k\\\\\\\/ts7lcA5ticyf0OuF6lQUaXSjWrmw1chOBq\\\\\\\/46Ekpb13YblW5GIf27Q+VGDeYPefS96MStv9j1dVMMnP8c79f4tlOYPC2oY9nwxz3zED2KVc6JHMynTbb4oe\\\\\\\/ZW2+i6EbJ3qWIgSJmHIA5lJpqbE+SdvRQtDJbBitEJZQDwxShQwM1KuSwOcblZjtNyTXAIyP1GyjNZEoLjp2m4zfBlNwsSvC4LJlXa6w9CLgFrXHxMaRg0S\\\\\\\/kRdiHa2+1pr1CwV91FX02RWvY\\\\\\\/YlT2ncei8bTnst\\\\\\\/MFrqtD6kV+G0pAslmw13HJ6ANRC8MQ5Xs\\\\\\\/Gz+dlsNkmSujLrn\\\\\\\/rtd+EY69aMOlniWKlrHs\\\\\\\/QOH1EO8vUrlbDH1MTo+cSovkjyI9SOdjFfVdIrcUofo7kB9C7Zmm1X4O9syXIUGU39u2rxzVfdFzrk1bHOApYAXL2BgxkH8gjSM5IRgkGq4XAVomSLbDCQ8EaLeHEdVfY2JZuGNOMoEiYv0w6otYY3Po4zGPPO3GegdqcMAJikjDtP5ko+\\\\\\\/Jc9yX\\\\\\\/I7QKrEF0C9TFs1b6ls+Jmjo1gl\\\\\\\/RLA4SwBDHLovAM9VISO611fqoPj7FNjrMKSM1BzWaTvOqs\\\\\\\/g1dSb5OkNLnluB\\\\\\\/7EJdufUfmxDOu\\\\\\\/iecLZ1YjvI5Own8hRdo15c+MYCP9uYobJ9mT11HIDt+DPnopMeTDWegfxtGDYYWYjPtmgaWxsYBf\\\\\\\/+RneMG+tdAZb2jasrxfIaWFMskUR\\\\\\\/pLbki20e4Eg3oege\\\\\\\/vBX2CIg+Bx6VSu8Xo9I4NBfBs5kp9jXNlY4e51BytS0HNvzV0YsvIXDxCcOcRkjqpCEWCJOTInel4BdjxsvEOYiqyC79e8q7t7nNdRBTMm49O1jvElap5X7\\\\\\\/WdaatpRjUruWFZSyna5xQjjaPQb\\\\\\\/R7Ju38QMGaVfs7Ym8FFIlAAjbo+ZEIKRubgje1aqO5Dm9gmvhfWYvKJA9wOhzfJB3B87oQGQfSPNUuab9KbcqgbY73BI654xDDnuwST7xkR1uNrPRtibqQR7yRalAyRJpVMu6bjp9gXK10Mv1VxBtHIGpqqmp4XbahMPIAFHgY3MzGSrFI6NiM6E\\\\\\\/qglzU+msh6SRqtOrhmCKKg3RU0YL1DnEAg8QPKewj3b4HH0Yw5v6lgoFfERz4M0Hamy2S99l1EVIZYV14Z\\\\\\\/kiJqpP\\\\\\\/YM4AslL2BE\\\\\\\/JRmAg0RRbNH\\\\\\\/Saklo9lh\\\\\\\/JSXeW7O7S8jlFVqNPEueZZAudEHBc58Mxc2AGclUqlRN+Xw2tVrfwsi4BA1RyMFQlnvGP0+9X3qM\\\\\\\/h\\\\\\\/vt56Fo995BWBs3DZbbrsFJdynK2qcxC0xUdSmduewQIlBp9xuMRS3zOgWHraiqev3D5U2EXMTZ83XDBcCosRPWM9HSASgNFVdEzS\\\\\\\/462b4nRQSG5+JboTPpVkTCW18Xvz7ebwi9Tvv0gxVEy+IlFzbCjlFnkiz4T0zFwyuihcwfxYrZ85Y\\\\\\\/zuJXUcW7tEtQcLJohDF3npsBDekCvGIlt+kEocbck7WJogcOE\\\\\\\/3fQxzxl1U\\\\\\\/j816dicCEVThwFOKg0Lo20q0K+DA988aYJBSjKprDfp9QmzJ0\\\\\\\/Yt6cs06crbykirjA2oK+HlwVe5I\\\\\\\/9suI3GicS21NG+0FVRQwD7B06dumUBWfX0wE0wb1MLF5JyY+zEScxye5lQLwsjzukJqriTumIUFKnRoEAsxIcr65UoSX+DmT9bzFfK0hHeL90nAE4NvlKK+TJ7Ub\\\\\\\/LVuzPQSgRBLzDJ5nxtUNi9ADIN3c83\\\\\\\/VPZsPUT6U3Q2HkAN\\\\\\\/uHr\\\\\\\/nhn3pa8SUOB9gCkMv8t3I3XA2b\\\\\\\/dshdZYhzc2pGzsnIWCWGCjtxBkHxdCHigtVfDuAlebXkvu9BYvsbUSi9sjNsZ6JMXntFKVtRzKeO0NKSJBWc2R419dd\\\\\\\/aYdIfvh4WHavdCMN\\\\\\\/dBg9BG8NpflS9rBaA9riuxOIJzkB907wSIo9uKTYrnKZUeJzI7uFo57KG3xUe1B3Nn1jSUbwGpnbm5amCHX2SVaCV\\\\\\\/8Vxo4bdeMs2m6w7ejRN4PcWXX+GG59B7tUxQgOSa6AaXfZMtHGeFewFA2effQPrJvW1vdep7F1zualM8k1rgDa1qn59vafd0B0l7\\\\\\\/R\\\\\\\/AyN3ES2pj934I8jOGFv\\\\\\\/1ZOqKR3tNQMzEXKKVJgkN+5JhsrQ+m9zqXPu0LA7Yqhx4x3Kk\\\\\\\/IQkJNIPLYhE60FAi8CVxzlW\\\\\\\/CeDpdM59ec3UsWVipOGjq9GJzniFTBmFB7lFNdjwarOBKm7ksv7K5BMYgGsWLfUOkzEqn4kx5TCFWACUUw1DkHPZ1QcFFwNU6ecFo1BtpKHtoKCV3w2Dhmg+iBy1iOvW9shOzrl2e5hw3GzLZkSZEv4GBxPy7N3XkVFMvqmG\\\\\\\/tv95pBDwG6D4eBretwMlIVQMS4wVtmfCDizqXIDnGIbGGvNlzuXEwrDm2xTGkCEba1pENckvC5SAXs\\\\\\\/FPQkfFuWbBAGar1VtZ2\\\\\\\/CzY7cOwd4pmCezl3gRucI1WUpzmXRFLbtqR6xLCatn4juPXBmVCTLxKtbkoSphxCaovkuObgvlAukntPJbRL2JmNWT+m5BeCqpOBxQKNGDvetnLMvAda8TMEfkmTc+ftANWUEdHHCKI0cMSN3pviURitMr0Z3UgqD4SR\\\\\\\/nw5Y6tYHYtu3fxpufWY1SGMHOhVObhAjoaVX9RJCI1i8LoTAbonGPlvBtFkVt8\\\\\\\/zi\\\\\\\/yDie2774OtSuqEBLmuzTKjm0v7ako+zIiT8G+ar0hCqQ1HUsZa3poPavszqWInksJR\\\\\\\/NTctetcFYvrkUhD8dxtRiw+nnHuTwtQH1RweqUFk3Pp+u1fD+\\\\\\\/Z7Fbe\\\\\\\/spZp8WPpMHqT\\\\\\\/AD0c\\\\\\\/kzYTBky52cIwIasUOgp1ITOVofRxwVk+SmCrzkgPmjcrvsEMxE8liz0KCnV83kKqcB2Co0Zy7ZlAdGt0vERiGngTAKuDSGqft\\\\\\\/NJ\\\\\\\/YsZ6F9WwuMXpGLuEEQcRbm0hcLCQHatjBgzMIOCRCVeEvH\\\\\\\/Ygg\\\\\\\/t48d7r9dBJ8175BoKCP511l5uftvkz8OVyEOVpjkwMkYx7KJuL5R+otWR+QNJBmf1r0yqeRI+UDy2Al0ho5OiMuNo3Q3MzeAO9F2\\\\\\\/zyzqpyjwwtPYk9oBM6BdX0rfeSySVhFJz9qOX8IO4rXLeyBAxU\\\\\\\/v6++aVin4DZJMNhzdZmTDrji23K0hGALFag7wgSDO9hKyyvyi1IuR5gwxPdMlDPIM+134YPWI2+dyeCvqMQgrT75kTTxLt5e1QwxxtOZd+bk273FYtY75\\\\\\\/98bZ0wp8TRj+jzR2lbxCmTQdq0Wz7RFGKii4oxLP8tDq0DmKkiRL6kbMlhZ6MXbEM9oYajCrYCBYnPa1J0Io3myTxbOvyDDLwdPCj7DVsWL5WsoemPK4LzgfnUnoprfdCizVGlD6SRW7TNAJP4DpBvBKxuppbUrU8oeUdguTukkZGZ3lDE6eQtCMUBwT\\\\\\\/ifFEsYqVvNz4i27SX04mi50+vKyzTF2yEY9L0psR7MRKtBmL+GN3dLqdZ00LpjYg\\\\\\\/ReJM9Bv395ghhxbK0SJBJM95zhVpoZcQOsTJTm1yIZ4lO1yiYDrmj28cbzl7DovMNJbSi6TpAJ7UQ3ZdNSHKHpZyacQScB0HjWgHmtvQZB5VhW+amroy9Z3f\\\\\\\/obZHzYGy9fnFMHS0bRz7pas7uaK0zbIEnWNAT+5bnx+0ByRcslPO3iIQEsrw0qkaaHhcMGLKkf7MwOzbrebaJdRw8f6TNWkwpR7xipI8vPRSQUXwghDQD7yVTZByi\\\\\\\/nHLu0POxNYHBFmuHMAogGgtWZPOK7IaSTQuM09jb1DT7c64RntFrnDk51qqjZRPo2xwOX14BGx2wJpG3L8ztGN0gp\\\\\\\/17sMlF6EbFCX5U0d8AHahCglSyCVN0n6ntlCjnSOX90Inq0lF8bkmCi5CYo3PGhzBwN77Hzanv\\\\\\\/a9qnRQzQCReaoN1tFXOyO8KAGN3hM7Be3qNBncUmHpUap9hNlMSw9KM0yjfN4lvH\\\\\\\/E1rfDPEmzn1cSkWqz4gZFElaHpOUbMzCTdQTPJkMItHJaWvalJBC\\\\\\\/YvcMk5bRgS1ekxIYGOftQdQDhHjPo+1xAxYKmtQwyPmQ7J483eWJqDV7kCYfDYuRJcZmAXtgnH03HNV4AZYbDm5u9wbe4wghhcYI1B5nq2gRvBwN5J8wyyg\\\\\\\/lXZmzG\\\\\\\/BKr3Yh6GTfawUXT2ugm0y6+gHNy9HdBVKtsNAUN6S9vBMHfuslxwA8iwGAZxFAGruoWuUbpWMQ539Sdm99qP2dvyVE2I8S8KGkBJAQ2BpADke1EL+iPdcjf9Qt89lDiRIst+ks4IobhkS+cFuKcpQkTCni\\\\\\\/\\\\\\\/esVOryAW8OrbyG1feKDg7HX2HLPXNAmN6DVWLowS9acWclbZbAjsFE0AxAmEVQo58F4ZLM\\\\\\\/8sDa8u9cjRyLNrLGaibXoAZ2M7Ub12\\\\\\\/slUl\\\\\\\/upbZ7Kq0WS2PDFijTJRyRYNlrVwhbkwSXKwWw8XSgjwIMui5\\\\\\\/tTygj2XgQfEikxXPOlAjn4DFf0ve5MyOh14cuMCKdReuj0ViivjFEhR9S7IxxTWyB45tlzBq9W6zJLKTEaZwGlQ2eiXYwd2npvh31MNmEjIn0o4hVOM1MtsAxEpXSYhP25QiMRPKr2S9rG0P\\\\\\\/vnY04y5gKoYUQGlvKbEdIu+Y3qZrP+12e7UxP8IFEnQWzJeyUeiOfaFTr15VmcAiJbqZAOtJEYr00p+2sls3Mr\\\\\\\/TlG1GjLncEMpXdKm\\\\\\\/lQ7AoPkp5XSvQ8hoSazWFjxqx0TbJzqo3jtLDiq4EL21i4PS45UDe8hDg3yQE7ZizSmEaJ4rS+CHPtuLVH239kJ5dhyOul0mDUjPSzIdaLYg9Et6bbn9nysqkgPXnp94gY+jq\\\\\\\/e1t59upruUV5+oCZN7YKmP6j2FbMfp\\\\\\\/gSWztGkdhGxdURPDCkkmKiSWj3PZvMypztJ7rIj3GEBw6skyUZHry55OHpayBtC+qMsJAfSLSYdYINnM3m5gAG+zGRMAxwbQkhgukkVZFLqFgqWFVFSgw1Y754+8R2ERMjRsjONth35y\\\\\\\/wFsR0gNVfdqzRsNZjuqQIFwnIUeUTd1YjWCe6RMGBcMmzreKW2J\\\\\\\/\\\\\\\/\\\\\\\/lMk\\\\\\\/bz8KjsR3s7lHgZFhUYGWvOAqeGU\\\\\\\/J8vYaVtkzuL5XVgE7AReYrlYqbWHReyNEROdrao\\\\\\\/T5EQ5qcMd7sDngsJUXFfq9oqlPskD\\\\\\\/whlcrQKxubvIwYDtTlSBHd2S1UIx\\\\\\\/sinn4gBXX0RdGmMHj6wk9IZpLKO+HEnUZ\\\\\\\/fUIUzuyaKXoLGfEYsQyN37st38Zs98ZlxJ\\\\\\\/VA\\\\\\\/+BCo39DKvEDDVO2VMJVapIIQ3MWl+slkR9P33WyEZyAhT0OC1ssqcl8KnAP9ZCW1TxX4gml4NcTqIyHrwP8bK\\\\\\\/GhdflkLYkG+UMdGRRa9tcjJff4aIw\\\\\\\/NDk7FVZaP6hI9Q\\\\\\\/HWkHnT9CkyvfuNvyGlYuWEmn1qOsylmEhXvzgVrE4POso8Q7KYzjE9a6zyVWh2OCgDNn3bv+XsB+hqmpPMZdGSmevzVUnzwxhPvGFhQTuF3QT1dKT+fc0NN1jDybORpjn0zh7BsGof9up\\\\\\\/y\\\\\\\/Q7+GhQqLwuBXp2FN2FMNuOkreTEjmr2dV4E054HRYjWlV12I1J\\\\\\\/WH0+VaSZbJpikCvYW+qF9GbzyVkJ8yWm8C8yiNlCQXutU1MhSE1eDG3VfcbNRfSO63UMwJYv72oj51\\\\\\\/LgFCRKA3vhRQ1owJvrEc6+hy44hWR1dh9GRfYDQGk92FRAI9bEtQN80qmhbTsDhresQDlJIYqxXeeF+D2IWQVujjKuyEhQZ9x5YJs6BPDlOksURpSknIZyw0nBjstqq3WjoDju+O6fcBpmU918ZLRZ2tbIbcM\\\\\\\/EGp2nk8i8CgoM9McK1Vp9YxBmP0sb49KIVbnMjW\\\\\\\/s1lW25C0sAeeni1yNTH9ys4PCifB3XZO8W9V3jO3pPWQl34Ln7K7oDbLpgvCa0TZydd8DQOcKuKgUmi\\\\\\\/LcHJ77uI19JeVbSGPfU2\\\\\\\/7CoJnsUIuKiNqdO+box7pRE9GUb9\\\\\\\/J2RcoEj4TJqDhIuY1Uwjm1HE415\\\\\\\/nN5tHetpl5llpfmhMM5g+tMFTqbQKRgBX9rohxfh6zQj1dplFjq2J2dAUert+jjcrS9P2GcrrIU0cVzf29vhpgkD2o1IIsEjOYiIMISmf2lNmIEwEEq9W1YLRdtn\\\\\\\/fWhaZL+lLyJjjQIAETmUa018L+U5bO8AjhrH8QqJaHiBEqn9XyMig8OdcafKfZKgSlntvUFvGToKVHS\\\\\\\/NklijnWela3T1ta6TH3vaNyT6K4xo9jF5smqsyueWeenFrflk27fgScTxbDLBJbAoxSYdfU4GpXWRm9JkBZPVkTTmnUAFndp6YQ4H\\\\\\\/M5DeWjD8aVTviVbq6\\\\\\\/DEmuesBBHPaSFO1\\\\\\\/uHicxz9C4bIg\\\\\\\/LSj8a\\\\\\\/8fIi5ZUlNFW4XP8eHsbSzBexuJndnjpOBbt8EVJf8H9BJQ+MmtuKMYIuXL0Ak+3MadH+\\\\\\\/7tRH3cHFH+SIGGMgCqsZuzP3ndGwl7sBPUK+rC1q3Bd9SXuoN4T2SrKv0vqMnE1FL19rh5NxYR+hF621U25Le9sfCP2cYOXLhAsJ78wivxOU7rgz4KvHUMxepPsVCWjtl6ro3n9VNzqGeDHUicvabDbEd9G++YG+pBIIAY3A\\\\\\\/BVhiPPC+A2FTBYV2PWh0KyUi5H1iNzY+CVcHrre6djCngt4huP8M60c0Tgq+j865ZcDLHi2GGADcsM0M5OxJKwUrYPSVj4lGiJuiZ17kcjxnIZHl231oNvSVc\\\\\\\/fjzNULySGmor5P\\\\\\\/9qvHfozDL\\\\\\\/FR4KRPk22CFepN70McgR2cbnmBaWPI9acGtiCVtUk9\\\\\\\/MnGxushyLQk7yAfOhh++dWzyIJB+GZ6zcLLx6M3OJvRpP\\\\\\\/QKgZJx\\\\\\\/THgha2EQftyaHmWIEdTvCNgcx8eQfY8GIZjMQFyTKK7h+sDr4c\\\\\\\/md3RXwfdrJdRStLHjpqmEzoLI0UvBrZwzuKW7V8vOhCgLQJrK3fqtamd8rd8Z3yEWLGMIxe8H18KD6d\\\\\\\/l1KAM3Kz2u0XJ1bKq80qscDlb9iv6xdaUjTUkzo5118nW8MlAuP8X+Kf6f88mijF856BqPU4uOvqJIn3q4PjmB4LKhx8dmLIEzcs3sRKtpPMQXKTKST+BAuVT2UR37c7Uu\\\\\\\/U+iZKUhgG5cmrMaPQx6SR+hBDZlaTuRZnMLMZFJgY1JmaGlUaO87Kzca933anpicNwQATRQPu6ye80SPlPPG9Nr5hvXSK7v7LGLJszM78Lh6y\\\\\\\/8CCYvSlef3Qx8+jrvgN1AjvI6l95YU2UKiN2aTRLh7W5gj58UDrnA5LJFAOU3zI1+3Ig32kw+Xm2CdD5JK5e2S69cppJ35vjlW7p4mkaos8ZG0rDChRG7+j31PiHnFjUyb5DzxRPSxwmJu1pXO5M4u2AnstzhrycxYYi+MbYRScOUV3zfy8Qc02WhAo7ORNBi08Js2MgsOgRbEuBNH5Oj64k6JgmLZFHWVLXqB5ScGghuCU4wGQMLj58d4WnOOvUL+3nS+zBIn4mtaigYHw8uGBDBKjbM1STDy2JqKFUcFxnHKDL5G1ftzdfAlzdNrfgKKg3T3b8s\\\\\\\/fz\\\\\\\/N0FAOJErDkxJnAPS2DnY9Za5wazvjous+nbLfS1VOsIz07NtPgeUigWAq4QdF3ASxkrKIxHENsebOBy+GkoY4Fsp8+w1rMkr\\\\\\\/i8xTByvn3frnXp91lAwZQen+GU2\\\\\\\/9m7EZz1uv\\\\\\\/TybXjeqK8QSwz1RQvl\\\\\\\/EPzRxhki7i3qBLOiyHjHOSk8ul\\\\\\\/0VnImEKG7EA9z0oPEJ5AoT45uT+Q\\\\\\\/+mTSc5n7ewxwXzkFRYFJj01qI3WTnrm14G+oJopjtHfnsIvQOg4BNYJnMlHbDCsl8LMG2sg6G9MikRf4HfJeUlx78bF6PDdYuO81cDwkU4gJcG7LaEYojfRc8ZnxzfWtx\\\\\\\/k0wOajyPs5Zuff8PhG0kqsiy1VgRAaOMdcHf14T6N5TR2FwyPGkobz9jQz0ZrjRhNUjUjqS0+4QGfvX2S74yMWO+bZuKqZMH5Pehpy9oNBmGFhqpMTiu3toQiIQEtIBbFwwwOBYTdS\\\\\\\/SvQA9x6W7GkCgm7LOHESf8HbJ2KnvAuzoXi6XRDYapBhpSwCB+zVly1FV0a6V\\\\\\\/yW86WdvRbIcOPdMb4BfxUsLjyhTorv8aDeo4G\\\\\\\/uk1howS5I\\\\\\\/p7sqQllBKidVXqj6XFg2TZjjdx4DTmAN589lXBCS+tgXV5S49U99AAcSlS2oe3vhxPIPzljkH6Sll7UGTtWPlkojo7qsbA6asTJpr9h\\\\\\\/RlW\\\\\\\/v4ml5wZAcWpLeUsFkBJFRsVLK+AtXBycwTW+O4e1ARr+Yu1LLGrLMAKAjatucMstFt2P4PTS9NRdVa3sSrcAGjkLTIsDSTHqB7WxDE2qegQaMrh0D6ilJe5BzGiiQAe+r9fzInY2cPB5uSP1\\\\\\\/gFHmpLo1R08YPRXwTNAod5U1XlFHRblU1hmDF2+fFj1yyBe4PD\\\\\\\/9yy6aJ0iXLS6DKQiUBXl5NDnmn9HqRkIBkvr\\\\\\\/vEnY3D4FmYRQk3etbRLeu6uBcqYw9SLzENvq0il7y8lYemeH2SfWpn5q+qgO0Wec4AxoEcp2FRBiltIvAXbavfSWWqDSiwmg2MIerz7tmVXYmjmYBdaiWzC6bj5xzfoPybHWGpiFTaz1O+beXPunw2OL2GPDrWyHunX+D9TuLUmHRz5NdYnfsdDKl61vKJP39UHvWAg8EFoowFyCyOuL8rDusdmqtnwqtmDlabfRuJHv6AsZwj8Pz1aU+QVD16EmT2VGK\\\\\\\/U+wwOlwiKr\\\\\\\/zFxE8OT9RqaFSxAk0xb08FPjMBw4UyoSjUjXc9ylAjlW1p16kIrROGXuB8edcQAovRJEo8KnaOF7Fyey0Hd0eeOLQTe+M1r2jHEbwrTYe9BE++TJ4D5XnuAIkcLG8qkTc6lbjdSeaNwx2J7o+OImNpTHxrxLeA3QqSjT\\\\\\\/xBoc1ZzsbcHdfBLhPNao+r4zF6AHc9PtfP3djzO1RgKhvKj9GHyUs2ru1X8Huu7pRzNrwdx26qmS0IYKNe8a2EPCSaF2mMow9HV5+nQP2i6Ae1W6i8Wcu4lOm+l\\\\\\\/Re\\\\\\\/iNfDdBCTZkqiRgPHc0dJmdz9hXWWLJkwozrALGaiD68lGTSzFDrSizUmY3ZWdkQlxSJrdT8HIjCBzHbLZzEahEYNwZf3oYO50j3+kmN\\\\\\\/rd01HXpob6Y+uKTif3EXB3Y+gFxsg7OZ6aKGX2Qmzl1VJ9+McgrJs3kWd\\\\\\\/mYPvaG2LrdHoB\\\\\\\/aSXsN5aB\\\\\\\/Qsoe72NndHbNB5fpwhNzREwlZdgMmsvzyiid0g5yJNSWhQ\\\\\\\/Rs\\\\\\\/8LrGYuU+kpGQ8xcs16\\\\\\\/vTEZEVWBgySgZOyhRmwtUTAVUHW9eaaxO6jRdyxier3A0BoZ2Na6lYksRn08K840qgtYPFREljlXvHHcYb1N78EdAh9cPxYr3oy4iSmur28CjV5oILHHx8R6xO74sbxZTsoKQwLCPZodSKuF3A+KCR+4FqFxV7G2ECRWGqXgCtzyuMHuxmU26UR7GTLf4zNgVULvtDvyw60LpG68XlQOCIN6qYBUdEvvbQiaWSWFX0YWL79YlR\\\\\\\/SoObhbpQc3bcLBfUDtcTONHFwt5kvmyHWXzds4FmfegR6Jht\\\\\\\/gI30pJnbExPdHK7HcNS8WyUP3nbE+gtYXTvfpbufoyXIAH+ZWHORrkz769PBZpJ7nIx0HTNETbjKmwqzRoQN+oLxUJHyM0UEulh\\\\\\\/H8UuZWvORPcV4SBfk5NbR2zulHEJrNsv3FGB3fg3qqsK3waa5eQL24chgIkV3HrqeS1rRUgamxgsa03mgjdy4bNREsmmjuq4nMvKunttKQT+D33M4ZBw8H0PA+ASkGUU9BTBYHKXDWxvfqGvIohpnTuExBFRAKcmL+ZuIQb3LN+7AXz6myvEPJGq+cRjf5KXNDd7LpAsP\\\\\\\/omjywVkT5BR92oiD73ahz4CDvI4JQ66dPi7G33YikfERlzoakmZABq2qhQCJ6kLYCEIvy0umiY6cycdaCbGdo7VXaWXLUfyWQIFylXjC9HSRoegESxU53wZnkonsXM0agg59k6h\\\\\\\/+NfLpy1jN\\\\\\\/Q4+vlWeLAxIQ5dDzC\\\\\\\/sa3UnkC8JYbkROX3r0olTtX97o3FldVC18I\\\\\\\/UCqfZ\\\\\\\/u3j2CsuQHbREy5XYEmZnQEKOJI7YJKBlKhIw9EiPUSeuW8WMZDI0wTevQ1iAJ5nHyLDi9\\\\\\\/whX18bkLcmKmKV6w+ZwswKkHZIAHW4glKmxDySUZ40cjKtgPIyJt6xjAjnR3Icz7386dqgIgU8J2OwG8SRzvuP1VJWHyltWRpdmjcyvCFzBa\\\\\\\/\\\\\\\/oma8L1loRw2B7h18FtandNeu\\\\\\\/nIxE47Bg\\\\\\\/MU12LJV0ejca+1X9zkPtiE1DRVG8ljtWhogQ8DnJ6sIwvK9E8BNt7wirMx9ZId3kpgXCuE+rqKPPOho+6z+ftfBLAkDzY1XOGKAzoUr6stO9M+OXH2lpLa5+9c0w8grsFBjCFQ7hUc1tW8F7AG3XnZmimuk8dcHEYqMsqTxGdGYJFpSSQvoQcLnSHl1inX2EFJAhXvVxbXwBYQbcTszPmShEKhmrtwZQCa6EeTvTkATcltA4VysSNF09MQOYS74fWbFE1ajaEj\\\\\\\/lG0EVf5yDjCsHXfUXfUN5ehj\\\\\\\/Uxx+aJlw4tQsIJu46hZ8qX2m\\\\\\\/ZVFSFWURpGMWtqT85RkHqcSmgCkrWBfx\\\\\\\/3x3z0Jx4E+31NFlZhRShak5+p\\\\\\\/4VSLXRvvvFCKXGmL6APjvKfO\\\\\\\/o+d3t9W4QT1bDtO1+eNefANyqRgc\\\\\\\/z1DWwT1RPGjijWguNJ3NUkgNUu3+X6vEoLxRUQ8h5MHXX32QCWxi7vAXnGzRSF315eNhhOBhnlPskgomUbTkzxsFbZU2n9F2I1UNpiux\\\\\\\/Ki4dOmsG4HWOYzLELEBrniyoumAAXum1YNIrh\\\\\\\/W9XsYxU4VYe1Z2dSM+rUm97ghnNYwSsG7+MGL\\\\\\\/iNBagwe9ZaL69\\\\\\\/VyrFTe9X1TKOO74NJts2VONugpzajrV5Zr6uUfoHaI8eG1QGrpOsOa907Wmo\\\\\\\/uG\\\\\\\/CjQTE2GiGHqBKwta+MuaSsyAektAKm2LfNrndHvFbfKJrPCSDv0m24OLiS2t8OdLVw2+2osSGAeI9vwG2kPnWxUn9sCUFcwBk\\\\\\\/+sjnBtDX5SjutcH+9gnjxxaAk12Oha1vqeL\\\\\\\/+2jDaXTx4DoXligfWTGhJFaiheJJibM7Vy\\\\\\\/Kv1THsV1bme68gDNvtq6rppeMc+Xs22CoynRj2e5ew6Em2eQy51hlDeNa4BTFTRxgOUIAkMojJK1wW4K4Qgq9X7D5Dys2UpdJzo0mVl05fgMDuroD5uLeyHBy1yDQHuUhAqzVupKq688vkYVbi8che3UTmvj1QT3ixxrLVZ2UxWgtQ3\\\\\\\/qD97US7TO9zaEfBLaKf9+ks26ZuG36ro\\\\\\\/GQJ0xQ+zKdyScOtrNKe8t9IMv1u8cSAFF1u8Fjen+zUOnsiQWV3Lz7n0yXiqWvvQk1VAzz3N4gEiirir8gEFjGZZP2P0EuRstehEkkPYJkSeZQR0OVILuZ7en8zFrfRQOVTWik0592RRXwRrk1sOvnPBDbZNdDqr3WT3dCg+xExoRi5J+9c5+NK+acIX1V3Who9kpzp0aiKXfiYZkf9XOh9AuQ0sdjhCeGO+2HrkbSfW273tUXoic17yuFn1MFO2uDlwYhWxlAL7khD1rrxjkWoGoQftacK+MN3XUcw4RjRQ5CXeZrE3Y6xT\\\\\\\/JlCEh\\\\\\\/GLjRQdjKNFn6iJwxLFByr+P8++YBgckKqyySeHZlsBpAdVmvfNpqL\\\\\\\/5i4Xk8mSRMhno7J0tCNHcBD4DNGn88BfC2VGBvKvw+QlOPA4BODqDoX1zJcXWEkgbAPdGr0J4ByHeLgTdAXQS5VfeoZ+N\\\\\\\/OLpz0w6SK6G7FU4oBEl7xYNueHiXWoG0LhIieOFbvL+rcnXje1W4MuMlIsERv\\\\\\\/T0IjBkJNP0aJbrF\\\\\\\/y\\\\\\\/WdwkkPmRCblnrfknqsbM5PEDcAA6YazNCi2GvuedppY9Ev++LYOWNTdgpDtmtGPo4zXVrWHLxAvAWEA6GxKx5Wdyq6ODVL\\\\\\\/iOlk\\\\\\\/wPohf4Zhm65fWzNJsrrQao6W9HjMmfdXekChPaiJLYaeoLPNHJ78wv6ExD1kiuR0VsHZ8pmcnRnpIv8eO5AfI++f+rO0smTV9J7jai3O4kRrx+y90tGRcbdOqnJGRD+ZyCfFPb70iZ7eK73v3PU+909z3IzissJumcgjWHZ+adijwWABhOEhOtftMzoAA9VzNXFd19IbcEFWLFlsckCN3ACGNKdDDAsvxkXrPTwoBBeQyNaaFuZwRV+zphN\\\\\\\/4bXc78ul7To6U6elR29uAdckX1NY5mWG\\\\\\\/VBM3tkv\\\\\\\/Y7QhfKvdVduyiEZUSRKZFD+9JslDctNimKEXxTEWHIkCCU9a5iCMfmXvtI1NyE4nfOlOrX1W5eQ1wxEI\\\\\\\/NuV7xv4wtCuHD9U0JMn5vE2UAl6cAkCC+LhLIB6jKLzIhuU1nPvxWZzIfXHdml80qCp+zwM82AkPCoRsfV5HPeW9YfgAEBviCOmqRdXZzI030PtdFDCVbtOKA34ZBX99TuMyjqH5\\\\\\\/+js7QsZMfBrocKglNSopH9QApwhO30kcNx6TgmQoWqz8zBGcQ3Nq1QdIiFymb2zFKuKAA2qJpN8q4JViPxLzpJ8wRUNQNcaFgoM35sBuA1QuZLTCHSXcXSQqA3p7DpXkZ0DqenkYnSGZGZd06zQr3Qyp5\\\\\\\/XrX9zvN+GTvoa2JdGDoSHOTTfett7zLhC4Ma2E0DRuFKFf7CyV3D4EWVd\\\\\\\/l35TFGubTqnfOqDaAIHHx\\\\\\\/aujP\\\\\\\/Zx+fcb7uX6ZFOEKOVfMaSd54gfI9gRfSXWUGxzJH47KvPIDxEij6jFs3SJ\\\\\\\/8tm5OOrF2rIrOshUsx9tseKAHVq+ww4\\\\\\\/VTP3jA1a3pP0H7Z0iBCGBmN997uilnbzmedrQWj7xiVK\\\\\\\/C\\\\\\\/FZ6M9TdvnNWMzvXbYoaLZrPayjknk9uNxV\\\\\\\/5XBkMfAWDLIbM3dqh3GuyUwmZSYfPsPDIP71uch4azhHNSUn08Ni01FPTR4Na0FOBIAuclYn7VbU8TCJz4M9qzXO5A4wvT\\\\\\\/O4bSeLUiOEEJzLpXinx9Bi6Y0K82djVBHol5Ku5qIfzc3K1L9yt7j0ni8A\\\\\\\/DQ4FsiDjtGysg6QQlNhzrmc8YrC4ikF2CMMnZ5ppuT0dEdzVh32lgzdmEytw\\\\\\\/EUDgJthJhi\\\\\\\/PUorCjX5lVEJL373pVaoeVbnWmi0A78Tw8QAuqZjpMbgVkAXOW7DuK3AT1XYyhv6RM6YPLZV+KXPxYPzdh2sd8OefXnS7spciCxjnTM63GdAiK7W2XgR0v\\\\\\\/+y7AcOJ\\\\\\\/dvQnzq5vCxjq5iADgz3dbiWnD3iLTfcJHNkrtbRov6WtmWxzpzellEAxnHG7tqhM6v0qg5K2pHfRhE7FLzaf8Mn1muJ0wmQbbV4Ung7BuBgq4EJ+OfsWFgj8Rk9iaZDg2SZEVvpSsqW10frnRfdLif+R2W+nrOreTO1pHDgRztauFuddji9d4F06MHDNYUHhBHNJ4+UJO5pwU\\\\\\\/WCnEAdUOGofIYcUw82FOGI6rwBkSOumNId9aas0tbr3XHUbM8XWMh90fHy0KOWXM47hOaxEBm+rSUNGkp9Rk\\\\\\\/n1zK2CHHKmPkEbnD57RUWGzakTrIrvOsB\\\\\\\/Er2OnTWbeDV9ZfmejVJomXjlp+7afrXULsdi6Kxy6gtjMaw01zRIBReFMljDjyQbuNk186yyg5TekHfb1rKhiYr0azxfPWeDLo7Ul28JGMGxQNLGCpvphXyXejwrIDEHe7kjlawjCtSxCbfpO71hF2FU0dy4f0D+J2V98P3UiD+As0zwGtyj\\\\\\\/AL6TDTEqc3PTfoeBgUVRiuZYJOaLq6u0BKbbnDRzWsB+Nzo6q9y5W+kH3X9WThJVEQRGjrQAbETHpA47\\\\\\\/b7frk+B23Uj5JAnM4x1aH0nbSPnFtqZfwwMPXD2uh+rwSG+hX3t4jF83dCd\\\\\\\/MqV+9Jpv5P031EEOFzjXtfF8IN\\\\\\\/4cn00ti4eN5iWM1eillK9gDct6MP8G6EPRnP4Oa8P3bnO4bZQrylgFGzlraO24Y1ZyihGBYJzxuAJqM9+19enzXSKW87rnY27QLDJgljsTMn3d\\\\\\\/gL5sHSP1TcoJe4wTWTpbKveSRrSft5KJ3tTvGkTbCRJlxBl\\\\\\\/Sww2SzrE\\\\\\\/U4wdYmCgugDbhr1cJNIQF+cHVyz2sW6p4OO3v2l5UM3bpgMF7j24WGBqHloc32cVfWC6gJtO5MBNOI6kcn70wEiZkuIMX2w7fV\\\\\\\/6bi+G0SdJbBkN+OsZSjdw8+OwjQ1CiCVwNEidycCL7RzqHrRpskryxXcfPvJA61Kuo9KSrqKAuhb4\\\\\\\/HhmUcoS1EJpWG943UVTL0Bj0qTx2y4n8gCm\\\\\\\/8sNYrmDvAt1g0rgo9FhM0Cr+1CIBpEjyOaL3LVpLSA4CI7Vl4p+9byK3OzaBEyRIC0c\\\\\\\/7SA3mE+hk56NM5jKWq4lUJpoE0VyRoUmv\\\\\\\/\\\\\\\/2LJwRHklJBWr8Ohu3X71xRAzlfu05VcRd6thS3e75n0lPzkH3KtjE1FhrisHYQUx3Et0Z8VLRAebhW\\\\\\\/0E8ugP+\\\\\\\/idqB07AdvkLRQK8fXyrvRMpkxKWB049nXmBxJjPYFC0l6qgBSfBen3VhJmiLN+nmsHVCnby\\\\\\\/Zk91OrE8PUcG5M14n3M8QI9EUtQsffdsvWEGUlP+oWNNrp29NipxUdEopEtcnSX9U3EHVHKoVGg7Q3w9iB8ogTRodQ7Day7mLS7iVEKhk7MckrmEw6uhwN5Z1J7pvtg6DkfjNDGO4V6HvwPq5SyT0pViMrr1lvMjn2unbOxkLX8gPsG2cHn47cv53TRLVUaQ5vdtf35RhaTgi9HuEyfVdO\\\\\\\/G\\\\\\\/JgJqCR4opDFnjs4lBleA9dXM25Lo\\\\\\\/EN3pzmF1q3hyIk5ZFWmTfX3NTHJr6TX2cQaOBB4hzFS+7uQh5LXGZCjE6r0nFzK23wc2JUAlyHYfbK3KLE5mLimsnEpwaEKGoVWzUO1A2YZ1iOQ5mAGqzoAm3tV3jShqoj\\\\\\\/q1PEgY+xp2rIlPBnGgYgBm4wKH0v5aM4V\\\\\\\/1kMe7Cwj2YT5x1KGW1\\\\\\\/kGkOJ9sLgtLmJZBFaQ3L\\\\\\\/0gFrgEaq5H6pXa5itmoC5uxdhOceVLqCEQWcGAERGeCjDEdnlJfmuqMRuJl0D4qMVdFdQVjSQfiwVZpXuY1ZB\\\\\\\/zGAD7a92YimADeY2ke4p4o4IRwal1\\\\\\\/BgnErmRqSmyH50O4pVkPUQOopj2BXftunGx1jQzNzO2k2QyKqkRChyu85ykrLojaAK5627tMmnFuCSiPSX\\\\\\\/mqx567NHc0ejETfI1HcKmAjgLMFyaWQ+HrQRRduO4wuu86QXrpkyjNgerP1cbczKp8oQaGtEhgZynB26KCxN5yJ9yYjItQZ+9Nu5WjRWtTog5GAYiEnhAHRof4gbNdsdu+xVfprFeskEwSxSdS6rYOjgcJJJIdYgm3GYeCTMX9Ek7gwV5dfj48wQTLveVXonvytadrVcXgry8GK9u4v7lo57EsxlE8yZ+mf48D4aTqwuon8kQpzQdhAM4aThtmOMljnhiw3l9Do0XdQyYWfCPTwFkBI07dVbgoK4vnb7X0OhnV4oLYvdoGiVZArEBOwX82eXn3tqP7kaHftem8o6ZPZlBUUXKOl7ctwM4msQNh2AhWWFnp0B2W5SwsVVl6hYjaGgNCLldIU9\\\\\\\/gBBSHEGeTD4jfVvni4RTEiloH3ZJWkrdOrTlYg\\\\\\\/rw2VJ2+k3vqJm8lkzxMCkZ10oqPyeymfHlkr8divQi0r6UGZ1MBQr7aLM\\\\\\\/XltHufy3rCKzmbVjPh8ppGftgCCA1jCuWKpB9pvtRi5XadrueTHz8lztPFTo5VqrkZGRd+ZaaJa90YZ3\\\\\\\/Dz38sSczTnHCNMoTxWT2NHkc6Td7GqRd8wkw5w176FE9B6mUIV\\\\\\\/KjvU\\\\\\\/\\\\\\\/xEKDBFZwboFqV3uT\\\\\\\/XEY2LUYHcJ71Q0wg1768qH7W6IVoERQOcQz5E4vQO6DMuGcsKG\\\\\\\/zSus1HvDx8L4rUBomWhUnCUzyIK89T+9n3H97Nb\\\\\\\/BE46XVDe3XSlGb4o+RuPQ8uP6T1iR\\\\\\\/lkFiEWQj+QiKr7u9WLzt+HWz3J138LUrp+YhKHt8+NgXJu7jptCvm+cSALvrt3wTd2Vhb32V2BYkuIjyYMyRzCiBCjzrJwvYjBKtSxFULsgZBW8iY0+PSQKz+VPfqowcQ+JOgxSoHxDyVGAttCoMZwSNTnjugd1sUhjPy1yXN0CTyajX1UGqaBY7+g2CgL5vM9XwzMDnbFrGisE09HSZOAr\\\\\\\/UyCgOdIa\\\\\\\/YyAR9FmpO1TAblHa0+m2UgzUo5L9WxggkpP9UaTbmbgzNld3EuXvgHIpGr5Q3YYZHkB9aUDw+0nSs\\\\\\\/PHeXO+f8C3DopVTN0Dp9qcRUnDjr4WsNn70iniRvgcv8Kp2Z5LTumrqDPaVbEWIKImKTK1e4yWsbaZQFWRkOq6zoVnXXWXm2zhfQM4UA7ObKK2wwaS1xnl84AjANkxDAaf4yMI6X7Qno8XGgKD3UHUbR64oY4tJjkZ3fqUvNg\\\\\\\/6HXNXqL268Dvq4OtZLT62Z3DQfdsxHg+RPM6kRu9VUcixj3TsdXNllBCEbTOncDEPUOUF15FSwV0dYncTNKrkrKIjvH36Nup0mr15qI91UJqtt8aPfP1B6tJdoKKutg\\\\\\\/DDEX\\\\\\\/KLUPhbZZooucVKHvfCywOCQjojSOu9RreSSr7uwL3mRbcfS6UP8XLJ5IZ1huQB7asRQhdno3H27fRTNDwX78vX7tx4LCPPyxFyVm4lyAvWV0FOaqT9BXsfETRFdPEEKhW2zkk+Fhk9BnF+CrcZehIq\\\\\\\/YUV0w01cJCbSwtmDLEH4LIH408E1mXXAno1NHKfSbXrU+jnbyb\\\\\\\/ew2Gw5JAXDvhY+gz\\\\\\\/QOvrUbHYFtPK56lSAqFDqefNpCB3ojB7Fz3YGxwAkju6EmywKemafNnXnzkpI2eVZ8oARl5OPPZxoTqZG8ZZAIka\\\\\\\/Q4XUItrRgJnuWE\\\\\\\/ycAO5WlJVH2sqywQTih2zI\\\\\\\/Dq8uyr2Yx4frgg0fQD6QV0VudTRx\\\\\\\/odgzmy3YxK1o\\\\\\\/+DYyWQ0c1fGILA3yUr+nFO0SIxh82KC89SEKmT+v\\\\\\\/53w+Z16a09qQPawWG2FHjFY5ToXmtLwn09AtVhc+xSr+TxgOVud+kI+8woWQ4lYbQ75w24evXkBHcw+Xr1WPdFvugPgnqAgL\\\\\\\/7Pq+fflkhV8Ea\\\\\\\/j3geHmLw9p+W3SBkG7UYDm3Jd\\\\\\\/aD2rQOXa56O4vmB9upyxWeYpsl21TSg6SiOZzNcCOoAJHN801\\\\\\\/Lejj0QhKA8gZib1MXFJ4M3NEmJuwrZwzt4SVbHHVjGMl70\\\\\\\/g7EK8RBZj5YgVwkDwyoUzz3C\\\\\\\/qUrZpavqDOdx0padqP8lsjvwOoiKvnCD0vEy4Mgz6+EDROvrX1m6OnRUPXX+o9r6+INmzIy6NGEWUjbnBL9AZboxxzymya9OtCsw8h+zGi4iAYAhRA4FaAyJLh6Kh4k5BTNoa\\\\\\\/9TaO9lWj\\\\\\\/Wa0sj8OUvBSE8rbarlVFchhvkdYldTkPJ0WiZMJpjyBpKnRIoJFOddRRy3AvD0xY6DCye0x\\\\\\\/pg9\\\\\\\/ciWOC\\\\\\\/44ouxiIXO5ZwAOS8hgXVsWNfFgPehSj2jhhN+RwKxkfm7tC7XjBWYE0\\\\\\\/OCxFa6IEnoZLdfhVj83HjN2CS\\\\\\\/a2R9tUHRd2GDqLJOsbad3Volvw6FKJJ3znLfmpkNWrw5adxlPoBdSjxzD0pgkzqJ+vb3nLtxEsmwIRGgKYQdq+oH\\\\\\\/TCxhdadKCc7ghfakYRTf0hiRBMP8WQk1ORbyEbnoXylO1MoGqSPR\\\\\\\/ia1HJzCuVr8FNx59xXM5p1Bw\\\\\\\/\\\\\\\/Ee\\\\\\\/NYwDYU1DD6IWuMVNUsZ4Q8Bx8SgcvDRYATagwJDwGuKiMD3NnnQG9v7lCo6DWacjsS8f8UdsygEQCE5WUBa2muJ4GienankdoGiaZe4BlgaSvgUalFlwP8rzyey3YO\\\\\\\/lp97mnNlGOtEv9Cda5RZesHagV+r3Xb\\\\\\\/JYuSAG+MTY7dYJOiT4uvkCzgJ5741\\\\\\\/esYSk35IrsoT9bGU7h0wMpVciLwgjffNgbGMR51JYO2u7yb\\\\\\\/U22jYTFVRgakUTZTkb1eebNW9YWlZM1COItZWUhZTQ+YZ9nndN2OeO7UWkH7rBt0goGIhBZFd8MZSap94PdwCNSo7S66IoRU9apm2Dc6dlX3RjTs0oNQwzSmmtFOHK3dYqKFoAy2RxGgfErx26htldgD+LbG27kpKsGAGiHiGdBU9xd\\\\\\\/OiX9taaNdpdZ+eXxg6xqsX1d0JpMZmPHslgwJZjk6epc0HXm5lMzbrdPm1eTVbVF2KT5Yu+a80\\\\\\\/3cXEbTMgQcXmyndJeg87NROSec\\\\\\\/bUZ2mUHFF\\\\\\\/Jmxl2LSBZwQSsarMjhJ+TeyjI73ucMqxeGZvZJgcecLZqzfXyMLBYBx+O31J9bUo4mHpO8S5P\\\\\\\/Fm236Z19RenonD\\\\\\\/bTdkGqDB+\\\\\\\/sdS3qB2RZmLpwVe9wVlMAePjdXIuLQ+Vx0v676lAclgpzP32zS1eVuA4H2eShT1N3rrJu+\\\\\\\/yKdESglttW8xFjA6UVko+qhBEdhKCc75C8fpnDktsNG6zsVkKIBiib1wbm1pRUHzHLtn3ZYHjr1vyT0kMYAMPUIYJlQ3Vnqy6ZHrMcYd6OhwaIeA5nw6er1\\\\\\\/mPZNmfvb1VpY7cdBK4oREBqIIFz7rJyHEK1lTTtVWr1P6zqgnjCX2OItuURup0\\\\\\\/KCO\\\\\\\/IZj30At8\\\\\\\/NfiIuqrKR6HIp8NLnt6VJqX8iGfJ4QcJKWmiIK+gYnRb1v4tw3X\\\\\\\/bSPWltI+K8LzSbbZ0wjSMxVQzplPoHWtPrT2GYOJj6m3ScngdP1sqehzTJQEMPj\\\\\\\/ZFSa0tFnR70OYdAWPhBUxcmVPu2Ql4gtNvTxTGbJr\\\\\\\/PypRIpi7IZdiLwLYnus1QKG19PffR2b6adtOweuCUd0BZ7\\\\\\\/eCRTpNrp9WRjLA81zW9pDPpxMcNqEoJvpXu5QxiwLxm+K5WwXW3EW4kXJ6ZqK36N5LX1U8Zz\\\\\\\/NqyYYoKyggHNyAL28tiPEueB8dQN0BmuHY6AKl4StB+1845EUE5y5g8srgwQJkhhD93CC4cWBQ8payH4sIfXrQIzqHmGn012zPmhPQepYYqQFGfUGPul44fdoLL9HpjDiwyRSOMY8LAMzYd0p7ZjKIn+iodGhuokrWrw1F5flVrF1D6EEl1\\\\\\\/7eun5WRvf6RlSfFXmh7u37rQG4BWbwCvcIPPOUpbWNoFtDgBCzqMFt6J4bXhmnj\\\\\\\/Jt1sHGcOoj0pS+cyO8qVdWSKhy7Ga8j5gUgIz\\\\\\\/jmA49uyDtUNEpIYSCVBJQlRmZN19vewLot+yWEM+ZLJdVDGDwL9MQ8Zz+92+lo+6kg2BKSNF\\\\\\\/dw3MkYshXdFEPv4uuUWjrTmzljxnomLJmgqbtxRjhJOGl0t1lHvn4akFXoSjhIElHibkaets+vD3Uc9SxRUnf83kTO8lnQjZ5nGPWqfrOY6v2L0U+OMHuiBVOy4c1fyhctSUtZC1SvXExFysAr6DTzk3th\\\\\\\/G8mGhnj5gdlF9B75\\\\\\\/VDYX0LvEEbQx0MOXHnabm3A3P3gOqO\\\\\\\/fVSGdrg\\\\\\\/B6XnUWEBdoWgXf3b8oZkuGYOLCut+cOlD3qfMfA58MyJx4cYVdsgSe9n9jcsa5rTeF3uUllhwLEd\\\\\\\/85FAnOyRaA0hAM0f\\\\\\\/49zXi3KaZKRGvV65c9VDOBspoUW4l9uES\\\\\\\/7eIJ7VrFOWzpAxeePzHRDWM17tJFLORDDOQ596r0zUtNObQdMB\\\\\\\/sXEB6R8\\\\\\\/8I9g+5BxWhR4WCH7bELu3rMB6yu2c8u1S+duzwqPMxnQLUD9jMrqyLOMkSh4ZuZABTC\\\\\\\/PH8Nl7zqMR8xANmCvoAcQAcCEWWIHVKjWO0z7MfT8y34n4MocpJa5wcG6Xb4eXAII7dbmMnxHY6HECpFuJa17W+7ZjTyZqD5\\\\\\\/SnkO7I5o+lrsZ8LwnqNKANi7u+BjhM1u09nUD6z5Uy0W0+ISkijc2LH9v+iuOd0OoHKjtlIZAXgM5xDiwlDhcHZ4xDKI4EthxZlCoI8dyhEFq+0NfCwZakNyLwfJM5wSxGLvviArYdUc9Yx2B87UeFPet09p6y53Lia1tQIdYWO\\\\\\\/O7Yh8vrWb5OETeYQvx6tBVVQC2M7NMYR\\\\\\\/T52z3+gpE4jy56RHzBO5i4gwzBjuGV5ecMt7uGpySIP19PSevvOKdNVksT\\\\\\\/uOgUPpcWb9+oidFOVCs0DPhuWjwySsPAscsMm9nU1UKuU7GlhwW214KNt5OM9Km88Lt5+R9XmvCG4J9vtePPEw4sOzQk1FQJiU34WGjZR9d30DTDT3b6XlYhAQe+i0U9o82tVgvM6\\\\\\\/LxahzIS79qUs9Sbn\\\\\\\/QBaWCXBRyA44ySfLghyWANolVmxMWwxl9a\\\\\\\/DLiszt\\\\\\\/WKb9G46+ffjRwWPGqs2Pb1kr5GDsp20Cwqn7\\\\\\\/Ze51bHGN\\\\\\\/rg4zzFkPU84tkk5t3NjuhtSqq7v0ipic0nV7zClO\\\\\\\/UEwFEVdfX9DDGClpk9yJBlZYsCPXgBWFELOMFppBN+Ae5Y42iaG5X\\\\\\\/Pxj38GYi4J4r8pAq7fTJ3Wk3sB6BbzziEbiHrz\\\\\\\/kBOUxaOP0NtNu19w7A7MpS2bzD3UkDrD6WzDpQjg2MX+ixK2xOXX2hMZE9yief9Zbl3VAvOLnwsINGKOEhb0FPnEUlwh+3F1xkeu2Vmw2+IWuJ79TR+6M8BK0JEJeLfUwZMI3GlxwHBKivQVeT5tD\\\\\\\/nJNTTnodNgNToHfa2qP2oKGdFkBbT5oxwFyCikJI8TI1iDxm7F7FfpMrMqQXLMqTAYdBVaSV6GaTiK7j4ddQUz5\\\\\\\/vJMiCrXY5wU1qi+Jyd64Iv5gtCdj15fN80R2bz\\\\\\\/O74eQzsKbvaXRclIc6qL0Bdz596cZSCt1aOOGPRaiIJNhc9vbozqsCF8fELZBL0rl1+uDKoH34q8nGfCVqTs\\\\\\\/mJDix59ge09tUSVogo5lqevABepe04NzB1NNh9UvULHKxV9oCjjgQHCjhG1cPkbAZxS2nMEOHQ5RaXMV2MwwYcOQaG9kdeNYDMhk3ycIlVc\\\\\\\/ArcRQOugvx8HbCaE4C7fydJs15F1R53baUkub8PdpP0ckN4W7w+NI1PmpOCBn4EjZTl6YjwoV4mMmnBgd2MRZHHT+fBXQafPa2bV03Y\\\\\\\/JMi\\\\\\\/RlXcAukc695p9H9b+VBOytNTHyd7bCyIKAHRrMRiJ9gduyFPwh1Ub2B8L06exRQrYnVUULBO1lE93aYEeEYs1FVL8\\\\\\\/83UoiNxRAMKGpCxdGytqmbiuhsBi9UC+N+DWjfo2bIbjAt\\\\\\\/3e2EoGq2C02HWUWDelZTsywAUCyTwEnouR\\\\\\\/fkQVOIwU\\\\\\\/z5kv+ttU6DEacwT\\\\\\\/aBBS5V1HukWdRQmNQ5r48z5+DeMwIfTinaqXEA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022PehsxTeBth8LCZQwqX78ww==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022004bdab82d1cde9bcbfca4b36af7427eea84992fb66d6f37ac22742c1ec289b2\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022nQyZ7RKrxiaXVt+Jo1Lp+tDfD7LuTI72\\\\\\\/0TsgqDZyi0wzSVrG6S62q6U4rVDP+TiJpzofXS53DOVir8F+Wfwp8\\\\\\\/Mkh5Ng1U5XK6S\\\\\\\/v9v5CGJM\\\\\\\/ppDUf0Ov241P0Eu7PJ1CYiVXukaviwhgEequMlVEbTbijoeFFjbleWGpkJ7eRBcxUULcQBx524qyvOg188HAJAZQEspLED6dUC8PhIG\\\\\\\/6yRi03EY71i9mLtiMTj0PU9QgC9yKicbWU1j7m1N79qna6TjKzIQbUi2RM2nJOBuQkAO\\\\\\\/X64gFU+oOX4Oiyjqi7QZCy1HCLx2fd5FUa3arzpHIIrw7r\\\\\\\/VYuEIvdg96W1foHL8ZrA8gBeYucVjplRp1g1qKdBkNLgRns1S3rKiliKDxpwgR3Nb6NHfvz8uw\\\\\\\/SAZS81WmYAWzpdgA7AlHI+YViLlqX46B5B1ua6KGO9VriShMPW7w783heydAJ2v1FwTmDfsUkrcLT6AcIKdGGVdiF7ysXxGGYpDT1FYhv9WAvmuJkklnkreTVRv+fnsrjvpcPEg8sUSxhyNdHX936ImAHplRl7iV1df99Ll2XLSdVh5JHvDuQmoAP03Zgc7vmH4U8dB9saCaegNu6irpYCekluXYoBT\\\\\\\/X3DnibGfiNoKDZzovfIAPowkzG7u7NBYwctpXrj82csICnjoLMFbS18UL0gUqF0wrTaVgiDD\\\\\\\/jAiGaD0YDYd53ss1NIzSTkMf9ToerynkPTfhClXibmCmJS+YqLMsFogAeDMz1KQrqRKT\\\\\\\/XM7D\\\\\\\/rh2D\\\\\\\/8wmC5BOYvkpC7uwYswBYH\\\\\\\/hWMxbsfZp9Y7YHgJ8GVQZjsr3aIPFBPOzT7N6h3sB23TSxO8ieh3enjanDf\\\\\\\/xYhvQ9hMmV2lkGkeXNY+6wsNAUhCB1hlzq7mnLVzYaijd3PDgjGXQd\\\\\\\/cv0tbbznMH9goVNY30JVT5QVrIM0o\\\\\\\/itBk2M7dByG2hQsscu54fUXx2L382ru85HtH8KD5r4CSktQfiCQg\\\\\\\/tDQl6dF56ThtYMOqmr2+7hpgX5Iayf4lCqI2YxHPwPtqcICd4oTWlSSml7dpolmVEBlisqXkQBoL\\\\\\\/LXruxlZu2VRzJjbn74HM8tU4M2LKhnxl\\\\\\\/bCX6a7exk1vn+o3L60\\\\\\\/YW0p6Fcx5YCrsTXfFIW2bhbJFBp88KYZrCh6DbpSbLjF93kpOFDjOT1D1Wdd0RJ\\\\\\\/kBurgryof3LYOZLua7mo3a33oQzB5Q8JHCh\\\\\\\/qcybcEiOzLJOLczf57SC8U8l5mvJE96FJODmTNa96d2vauZBXCZB0UM9L3Tys8DBahcmAxDaSjqZxKEnzM4ZZXhBvfxRrq8MRMkIssE4Fv4qcqlmvBje3kmZM0+0QAyexMi3asw9PY3RRcG0x\\\\\\\/1lKxYIc2yHqS\\\\\\\/p4d3cyWLerSm1wWcxBCdQ3PLK7R2+Xh+Q2c0d1dZLeVXGbFA6ojxEEeCSRiNFcDAms62RH6ldFwxcqTd+WyAasAKtGBr62qcptSvQVs9bgMiwmtMUUxJKu5xhf00VLAlc+9HMquIcAZ8kOOHBB1iV7xCDXPo0qnK2JsMMBPj7ITaArPVjKxa8l4qriw2NIvEz+CpHe5A3hni2BsYouR8H8iI8UDlK\\\\\\\/GyLQxpz3+uZ8HammNfFxIKRFarZRdpFBxuSQ80YGEElQNwSxmq\\\\\\\/FIDBtMU1CtcDMJ7WllnhaoTfaMHGbtz0etJWZuciIhQam5mpKYxpMmWjXba5tPV4\\\\\\\/U90YN\\\\\\\/TVSg3pJvzwtu1RHfDMbHNNalSLXxWQgUaJlvz2I8XDNks2HIMHLx4E7FKydSdt0ciKj7ynApy76nYQpKgAIRKZ0nRpNGN+oQ\\\\\\\/m8CkjQBvmt7JLbNz+ZlWSGqnnbXrq3+fZlHeBqTCcJNkrk7FBj9KipuVrw\\\\\\\/O8dwqsgg7f6r4xMty7r+CBNWXYC6mg3p1Rof7T3oQD6JhNO0slt797YAlTn6qAyE\\\\\\\/1ivrVqg9g9smGOfrzqpWauUEOWDkq9dfy+pF8hpMHbyHznokFSYA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022kVqs\\\\\\\/uq79GyCCRZBIKyGiw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226e9a83512e978577215a3585d6c35745942acb89d733b146e7cbcd1b81a7fb40\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459889 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/5/741" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-8X2ADZj82Hqp9gKV9bExrGxiUhHUAdHi2ZB3ChiIUy8" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/5/741" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form--9KE6U9d5W7byKprWiCvcOXZa9g2UbpWkdd6JnTemoI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/5/719" title="Drosophila Muller F Elements Maintain a Distinct Set of Genomic Properties Over 40 Million Years of Evolution" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/5/751" title="A Survey of Imprinted Gene Expression in Mouse Trophoblast Stem Cells" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459889" id="node459889" data-pisa="ggg;5/5/741" data-pisa-master="ggg;g3.115.017376" data-apath="/ggg/5/5/741.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Systematic Mutational Analysis of a Histone H3 Residue in Budding Yeast Provides Insights into Chromatin Dynamics</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Paige Johnson</span>, <span class="highwire-citation-author" data-delta="1">Virginia Mitchell</span>, <span class="highwire-citation-author" data-delta="2">Kelsi McClure</span>, <span class="highwire-citation-author" data-delta="3">Martha Kellems</span>, <span class="highwire-citation-author" data-delta="4">Sarah Marshall</span>, <span class="highwire-citation-author" data-delta="5">Mary K. Allison</span>, <span class="highwire-citation-author" data-delta="6">Harrison Lindley</span>, <span class="highwire-citation-author" data-delta="7">Hoai-Trang T. Nguyen</span>, <span class="highwire-citation-author" data-delta="8">Jessalyn E. Tackett</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9"><a href="http://orcid.org/0000-0002-6040-6825" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Andrea A. Duina</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">741-749;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.017376 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459889" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Paige Johnson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biology Department, Hendrix College, Conway, Arkansas 72032</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paige%2BJohnson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Johnson%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaige%2BJohnson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Virginia Mitchell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biology Department, Hendrix College, Conway, Arkansas 72032</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Virginia%2BMitchell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mitchell%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVirginia%2BMitchell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Kelsi McClure </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biology Department, Hendrix College, Conway, Arkansas 72032</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kelsi%2BMcClure%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McClure%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKelsi%2BMcClure%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Martha Kellems </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biology Department, Hendrix College, Conway, Arkansas 72032</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martha%2BKellems%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kellems%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMartha%2BKellems%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Sarah Marshall </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biology Department, Hendrix College, Conway, Arkansas 72032</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sarah%2BMarshall%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Marshall%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASarah%2BMarshall%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Mary K. Allison </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biology Department, Hendrix College, Conway, Arkansas 72032</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mary%2BK.%2BAllison%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Allison%20MK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMary%2BK.%2BAllison%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Harrison Lindley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biology Department, Hendrix College, Conway, Arkansas 72032</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Harrison%2BLindley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lindley%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHarrison%2BLindley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Hoai-Trang T. Nguyen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biology Department, Hendrix College, Conway, Arkansas 72032</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hoai-Trang%2BT.%2BNguyen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nguyen%20HT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHoai-Trang%2BT.%2BNguyen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Jessalyn E. Tackett </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biology Department, Hendrix College, Conway, Arkansas 72032</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jessalyn%2BE.%2BTackett%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tackett%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJessalyn%2BE.%2BTackett%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Andrea A. Duina </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biology Department, Hendrix College, Conway, Arkansas 72032</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrea%2BA.%2BDuina%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Duina%20AA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrea%2BA.%2BDuina%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-6040-6825" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Andrea A. Duina</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:duina@hendrix.edu" class="" data-icon-position="" data-hide-link-title="0">duina@hendrix.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/5/741" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459889" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459889/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/5/741.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459889" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459889/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/5/741.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459889" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459889/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/5/741.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459889" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459889/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/5/741.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459889" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459889/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-corrections" >
  
      
  
  <div class="pane-content">
    <div class="messages highwire-corrections highwire-correction-type-correction"><h3 class="highwire-corrections-title">This article has a correction. Please see:</h3><ul class="highwire-corrections-list"><li class="relation-67986 first last"><a href="/content/5/7/1541" class="" data-icon-position="" data-hide-link-title="0">Corrigendum - July 01, 2015</a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">In previous work using the <em>Saccharomyces cerevisiae</em> model system, a mutant version of <a href="http://www.yeastgenome.org/locus/S000004976/overview">histone H3</a>—H3-L61W—was found to confer a variety of abnormal growth phenotypes and defects in specific aspects of the transcription process, including a pronounced alteration in the distribution pattern of the transcription elongation factor <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> across transcribed genes and promotion of cryptic transcription initiation within the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000911">FLO8</a></em> gene. To gain insights into the contribution of the H3-L61 residue to chromatin function, we have generated yeast strains expressing versions of <a href="http://www.yeastgenome.org/locus/S000004976/overview">histone H3</a> harboring all possible natural amino acid substitutions at position 61 (H3-L61X mutants) and tested them in a series of assays. We found that whereas 16 of the 19 H3-L61X mutants support viability when expressed as the sole source of <a href="http://www.yeastgenome.org/locus/S000004976/overview">histone H3</a> in cells, all 19 confer abnormal phenotypes ranging from very mild to severe, a finding that might in part explain the high degree of conservation of the H3-L61 residue among eukaryotes. An examination of the strength of the defects conferred by each H3-L61X mutant and the nature of the corresponding substituted residue provides insights into structural features of the nucleosome required for proper <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>−gene interactions and for prevention of cryptic transcription initiation events. Finally, we provide evidence that the defects imparted by H3-L61X mutants on <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>−gene interactions and on repression of intragenic transcription initiation are mechanistically related to each other.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/spt16" class="hw-term hw-article-keyword hw-article-keyword-spt16" rel="nofollow">Spt16</a></li><li class="kwd"><a href="/keyword/chromatin" class="hw-term hw-article-keyword hw-article-keyword-chromatin" rel="nofollow">chromatin</a></li><li class="kwd"><a href="/keyword/histone-h3" class="hw-term hw-article-keyword hw-article-keyword-histone-h3" rel="nofollow">histone H3</a></li><li class="kwd"><a href="/keyword/intragenic-transcription-initiation" class="hw-term hw-article-keyword hw-article-keyword-intragenic-transcription-initiation" rel="nofollow">intragenic transcription initiation</a></li><li class="kwd"><a href="/keyword/transcription-elongation" class="hw-term hw-article-keyword hw-article-keyword-transcription-elongation" rel="nofollow">transcription elongation</a></li></ul><p id="p-3">Within the eukaryotic cell nucleus, DNA is condensed through the formation of chromatin, a protein−DNA complex with the nucleosome as its fundamental unit. Nucleosomes are composed of ∼147 base-pairs of DNA wrapped around the histone octamer, itself composed of a <a href="http://www.yeastgenome.org/locus/S000004976/overview">histone H3</a>-H4 tetramer and two H2A-H2B dimers (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Luger <em>et al.</em> 1997</a>). In addition to directing DNA condensation, nucleosomes also are critically important for the regulation of processes that use DNA as template, including gene transcription. A variety of factors have been identified that can regulate specific aspects of transcription by affecting the dynamics of nucleosomes. These include enzymes that modify histone residues through the addition or removal of specific chemical groups, chromatin remodeling complexes that can alter DNA−histone interactions or the stoichiometry of canonical histones and histone variants within individual nucleosomes in an ATP-dependent manner, and histone chaperones that facilitate the assembly and disassembly of nucleosomes through mechanisms that do not rely on ATP hydrolysis (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Campos and Reinberg 2009</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Rando and Winston 2012</a>).</p><p id="p-4">FACT (FAcilitates Chromatin Transcription/Transactions) is a highly conserved histone chaperone complex with roles in several chromatin-based processes. In the budding yeast <em>Saccharomyces cerevisiae</em>, FACT is a heterodimer composed of the proteins <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004534">Pob3</a> that can physically interact with nucleosomes with the assistance of the HMG protein Nhp6 (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Brewster <em>et al.</em> 1998</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Formosa <em>et al.</em> 2001</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Ruone <em>et al.</em> 2003</a>). FACT’s ability to interact with nucleosomes and alter their structure is used by cells to control chromatin environment dynamics during both the initiation and elongation phases of transcription (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Reinberg and Sims 2006</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Duina 2011</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Winkler and Luger 2011</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Formosa 2012</a>). Arguably, the most extensively studied and better-understood aspect of FACT biology relates to its function during transcription elongation. Work by many groups during the course of the last few decades have demonstrated that FACT plays a dual role during elongation, altering nucleosome conformation ahead of transcribing RNA polymerase II (Pol II) to facilitate Pol II progression across genes and assisting in the reassembly of nucleosomes after Pol II passage (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Reinberg and Sims 2006</a>; <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Duina 2011</a>; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Winkler and Luger 2011</a>; <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Formosa 2012</a>). </p><p id="p-5">Several studies have provided insights into the mechanism of action of FACT in these processes. Some of these studies have culminated in a model whereby FACT−nucleosome interactions result in formation of so-called reorganized nucleosomes, in which FACT and all nucleosomal components are tethered together but whose histone subunits are more prone to dissociation during engagement with transcribing Pol II complexes (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Formosa 2008</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Xin <em>et al.</em> 2009</a>). Other studies have supported a mechanism that involves specific interactions between FACT and H2A-H2B dimers as central drivers for alterations in DNA-histone interactions conducive to transcription elongation (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Winkler <em>et al.</em> 2011</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Hondele <em>et al.</em> 2013</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Hsieh <em>et al.</em> 2013</a>). FACT is thought to contribute to the maintenance of the epigenetic code across transcribed genes by reassembling nucleosomes using the same <a href="http://www.yeastgenome.org/locus/S000004976/overview">histone H3</a> and H4 subunits that were present on chromatin ahead of Pol II passage (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Jamai <em>et al.</em> 2009</a>).</p><p id="p-6">A comprehensive model for the mode of action of FACT at transcribed genes requires an understanding of the physical and functional interplay between FACT and histones <em>in vivo</em>. Insights into these relationships have come in part from genetic studies in <em>S. cerevisiae</em> in which the effects of specific histone mutants on FACT function have been assessed. For example, recent work from the Formosa laboratory has shown that specific mutations predicted to disrupt the H2A-H2B dimer:H3-H4 tetramer interface suppress defects in FACT activity, a result consistent with the notion that FACT facilitates dimer-tetramer dissociation (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Mccullough <em>et al.</em> 2011</a>). In another recent study, analyses of yeast cells expressing a <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002632">histone H2B</a> mutant lacking the HBR domain—a conserved region within the N-terminus of the protein––revealed a requirement for this domain in promoting FACT-mediated H2A-H2B eviction from nucleosomes (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Zheng <em>et al.</em> 2014</a>).</p><p id="p-7">We have previously identified specific histone mutants—H3-L61W, H4-R36A, and H4-K31E—that markedly impair the ability of the FACT subunit <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> to properly associate with chromatin (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Duina <em>et al.</em> 2007</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Nguyen <em>et al.</em> 2013</a>). These mutants cause decreased levels of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> occupancy at the 5′ regions of transcribed genes and increased levels at the 3′ regions of the same genes, resulting in an overall shift in <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> distribution toward the 3′ ends of genes. Because H3-L61, H4-R36 and H4-K31 are located near each other on the side of nucleosomes, we have proposed that these residues define a nucleosomal region important for proper <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>−gene interactions, possibly by participating in physical interactions with <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> during transcription (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Nguyen <em>et al.</em> 2013</a>). In this study, we have carried out a systematic mutational analysis of the H3-L61 residue by generating and testing all possible natural amino acid substitutions at this residue (H3-L61X mutants) to determine its contribution in ensuring proper <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>−chromatin interactions. As the H3-L61W mutant also causes cryptic intragenic transcription initiation at the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000911">FLO8</a></em> gene, we have tested the H3-L61X mutants for this defect and have uncovered a correlation between H3-L61X-mediated defects in repression of intragenic transcription initiation and in impairment of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>−gene interactions. Our studies shed new light onto the contribution of the H3-L61 residue to chromatin dynamics, particularly in relation to FACT-mediated processes.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Yeast strains and growth media</h3><p id="p-8">All strains used in this study, listed in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, are <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004071">GAL2</a><sup>+</sup></em> derivatives of the S288C strain background (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Winston <em>et al.</em> 1995</a>). Strains yADP83-yADP88 harbor plasmid pDM9, a centromeric <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em>-marked plasmid containing the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000214">HHT1</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000213">HHF1</a></em> genes described in a previous study (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Duina and Winston 2004</a>). Details for the generation of the H3-L61X mutants are provided below. Standard genetic techniques and details for media preparation have been presented previously (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Rose <em>et al.</em> 1990</a>). Drug-containing plates were prepared using the following concentrations: formamide, 3%; hydroxyurea, 150 mM; and caffeine, 15 mM.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466093/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466093/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title"><em>Saccharomyces cerevisiae</em> strains</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>Generation of strains expressing H3-L61X mutants</h3><p id="p-10">Polymerase chain reaction (PCR) products spanning the entire <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004976">HHT2</a></em> gene and harboring specific mutations at the H3-L61−encoding codon were generated for each of the 19 H3-L61X−encoding genes using the strategy shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>. Generation of <strong><em>a</em></strong> fragments (see <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>) was accomplished using forward primer OAD20 and the appropriate reverse primer L61Xrev (where X is specific for each of the desired mutations—refer to <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> for a list of all primers used for these experiments). Generation of <strong><em>b</em></strong> fragments was accomplished using the corresponding forward primer L61Xfor and reverse primer OAD21. The template for these reactions was genomic DNA originating from an <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004976">HHT2</a></em>-wild-type strain. Amplification reactions were carried out using PrimeSTAR HS DNA Polymerase (Takara, Code No. R010A). In each case, the <strong><em>c</em></strong> fragments were obtained as a result of template-skipping events during PCR amplification reactions that included primers OAD20 and OAD21 (or primers OAD479 and OAD480), the appropriate set of fragments <strong><em>a</em></strong> and <strong><em>b</em></strong> (dissolved in melted low-melting point agarose) as templates, and PrimeSTAR HS DNA Polymerase.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/5/741/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Strategy for the generation of yeast strains expressing histone H3-L61X mutant proteins from the endogenous HHT2 locus. (A) Top: the haploid yeast genome harbors two sets of divergently transcribed genes encoding histones H3 and H4 (histone H4 is encoded by HHF1 and HHF2 and histone H3 is encoded by HHT1 and HHT2&#x2014;the open reading frames are represented by rectangles and the direction of transcription for each gene is indicated by the corresponding arrow). Bottom: the HHT2 gene was replaced with the nutritional markers URA3 and TRP1 through homologous recombination to generate strain yADP106. (B) Two overlapping segments of the wild-type HHT2 gene are amplified by PCR as illustrated in diagram 1. The reverse primer (left-pointing arrow) used to generate PCR product a includes substitution(s) of one, two, or three nucleotides (red circle) within the HHT2 codon encoding L61 to alter its code to specify for one of the other nineteen amino acids. The forward primer (right-pointing arrow) used to generate PCR product b includes the analogous substitution(s) (but in the sense orientation, also indicated by a red circle) as that/those present in the reverse primer used for the generation of product a. PCR products a and b are then used as templates for the PCR reaction shown in diagram 2. Template-switching events during the PCR can produce DNA molecules that span both fragments a and b, which can then serve as templates for the generation of full length PCR product c as shown in diagram 3. (C) Strain yADP106 is co-transformed with PCR products c and with a plasmid harboring the HIS3 nutritional marker. Cells from the transformation reactions are plated on media lacking histidine, and resulting colonies are then screened for 5-FOA resistance and for a Ura-/Trp- phenotype to identify cells that have replaced the URA3 and TRP1 markers with PCR product c through homologous recombination&#x2014;the plasmid is subsequently lost through mitotic cell division. We find that selecting for the most &#x201C;transformable&#x201D; cells based on their His+ phenotype combined with screening for 5-FOA resistance and loss of both URA3 and TRP1 markers greatly increases the success rate for obtaining the desired recombination products compared to direct selection on 5-FOA medium. The resulting hht2 mutant cells are then crossed with cells deleted for HHT1 to give rise to haploid cells harboring the hht2 mutant gene as sole source of histone H3. See the Materials and Methods section for more details on this strategy. PCR, polymerase chain reaction." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2114265090" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Strategy for the generation of yeast strains expressing histone H3-L61X mutant proteins from the endogenous &lt;em&gt;HHT2&lt;/em&gt; locus. (A) Top: the haploid yeast genome harbors two sets of divergently transcribed genes encoding histones H3 and H4 (histone H4 is encoded by &lt;em&gt;HHF1&lt;/em&gt; and &lt;em&gt;HHF2&lt;/em&gt; and histone H3 is encoded by &lt;em&gt;HHT1&lt;/em&gt; and &lt;em&gt;HHT2&lt;/em&gt;&#x2014;the open reading frames are represented by rectangles and the direction of transcription for each gene is indicated by the corresponding arrow). Bottom: the &lt;em&gt;HHT2&lt;/em&gt; gene was replaced with the nutritional markers &lt;em&gt;URA3&lt;/em&gt; and &lt;em&gt;TRP1&lt;/em&gt; through homologous recombination to generate strain yADP106. (B) Two overlapping segments of the wild-type &lt;em&gt;HHT2&lt;/em&gt; gene are amplified by PCR as illustrated in diagram 1. The reverse primer (left-pointing arrow) used to generate PCR product &lt;strong&gt;&lt;em&gt;a&lt;/em&gt;&lt;/strong&gt; includes substitution(s) of one, two, or three nucleotides (red circle) within the &lt;em&gt;HHT2&lt;/em&gt; codon encoding L61 to alter its code to specify for one of the other nineteen amino acids. The forward primer (right-pointing arrow) used to generate PCR product &lt;strong&gt;&lt;em&gt;b&lt;/em&gt;&lt;/strong&gt; includes the analogous substitution(s) (but in the sense orientation, also indicated by a red circle) as that/those present in the reverse primer used for the generation of product &lt;strong&gt;&lt;em&gt;a&lt;/em&gt;&lt;/strong&gt;. PCR products &lt;strong&gt;&lt;em&gt;a&lt;/em&gt;&lt;/strong&gt; and &lt;strong&gt;&lt;em&gt;b&lt;/em&gt;&lt;/strong&gt; are then used as templates for the PCR reaction shown in diagram 2. Template-switching events during the PCR can produce DNA molecules that span both fragments &lt;strong&gt;&lt;em&gt;a&lt;/em&gt;&lt;/strong&gt; and &lt;strong&gt;&lt;em&gt;b&lt;/em&gt;&lt;/strong&gt;, which can then serve as templates for the generation of full length PCR product &lt;strong&gt;&lt;em&gt;c&lt;/em&gt;&lt;/strong&gt; as shown in diagram 3. (C) Strain yADP106 is co-transformed with PCR products &lt;strong&gt;&lt;em&gt;c&lt;/em&gt;&lt;/strong&gt; and with a plasmid harboring the &lt;em&gt;HIS3&lt;/em&gt; nutritional marker. Cells from the transformation reactions are plated on media lacking histidine, and resulting colonies are then screened for 5-FOA resistance and for a Ura&lt;sup&gt;-&lt;/sup&gt;/Trp&lt;sup&gt;-&lt;/sup&gt; phenotype to identify cells that have replaced the &lt;em&gt;URA3&lt;/em&gt; and &lt;em&gt;TRP1&lt;/em&gt; markers with PCR product &lt;strong&gt;&lt;em&gt;c&lt;/em&gt;&lt;/strong&gt; through homologous recombination&#x2014;the plasmid is subsequently lost through mitotic cell division. We find that selecting for the most &#x201C;transformable&#x201D; cells based on their His&lt;sup&gt;+&lt;/sup&gt; phenotype combined with screening for 5-FOA resistance and loss of both &lt;em&gt;URA3&lt;/em&gt; and &lt;em&gt;TRP1&lt;/em&gt; markers greatly increases the success rate for obtaining the desired recombination products compared to direct selection on 5-FOA medium. The resulting &lt;em&gt;hht2&lt;/em&gt; mutant cells are then crossed with cells deleted for &lt;em&gt;HHT1&lt;/em&gt; to give rise to haploid cells harboring the &lt;em&gt;hht2&lt;/em&gt; mutant gene as sole source of histone H3. See the &lt;em&gt;Materials and Methods&lt;/em&gt; section for more details on this strategy. PCR, polymerase chain reaction.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/5/741/F1.medium.gif" width="440" height="331"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/5/741/F1.medium.gif" width="440" height="331"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/5/741/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/5/741/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465944" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-11" class="first-child">Strategy for the generation of yeast strains expressing <a href="http://www.yeastgenome.org/locus/S000004976/overview">histone H3</a>-L61X mutant proteins from the endogenous <em>HHT2</em> locus. (A) Top: the haploid yeast genome harbors two sets of divergently transcribed genes encoding histones H3 and H4 (histone H4 is encoded by <em>HHF1</em> and <em>HHF2</em> and <a href="http://www.yeastgenome.org/locus/S000004976/overview">histone H3</a> is encoded by <em>HHT1</em> and <em>HHT2</em>—the open reading frames are represented by rectangles and the direction of transcription for each gene is indicated by the corresponding arrow). Bottom: the <em>HHT2</em> gene was replaced with the nutritional markers <em>URA3</em> and <em>TRP1</em> through homologous recombination to generate strain yADP106. (B) Two overlapping segments of the wild-type <em>HHT2</em> gene are amplified by PCR as illustrated in diagram 1. The reverse primer (left-pointing arrow) used to generate PCR product <strong><em>a</em></strong> includes substitution(s) of one, two, or three nucleotides (red circle) within the <em>HHT2</em> codon encoding L61 to alter its code to specify for one of the other nineteen amino acids. The forward primer (right-pointing arrow) used to generate PCR product <strong><em>b</em></strong> includes the analogous substitution(s) (but in the sense orientation, also indicated by a red circle) as that/those present in the reverse primer used for the generation of product <strong><em>a</em></strong>. PCR products <strong><em>a</em></strong> and <strong><em>b</em></strong> are then used as templates for the PCR reaction shown in diagram 2. Template-switching events during the PCR can produce DNA molecules that span both fragments <strong><em>a</em></strong> and <strong><em>b</em></strong>, which can then serve as templates for the generation of full length PCR product <strong><em>c</em></strong> as shown in diagram 3. (C) Strain yADP106 is co-transformed with PCR products <strong><em>c</em></strong> and with a plasmid harboring the <em>HIS3</em> nutritional marker. Cells from the transformation reactions are plated on media lacking histidine, and resulting colonies are then screened for 5-FOA resistance and for a Ura<sup>-</sup>/Trp<sup>-</sup> phenotype to identify cells that have replaced the <em>URA3</em> and <em>TRP1</em> markers with PCR product <strong><em>c</em></strong> through homologous recombination—the plasmid is subsequently lost through mitotic cell division. We find that selecting for the most “transformable” cells based on their His<sup>+</sup> phenotype combined with screening for 5-FOA resistance and loss of both <em>URA3</em> and <em>TRP1</em> markers greatly increases the success rate for obtaining the desired recombination products compared to direct selection on 5-FOA medium. The resulting <em>hht2</em> mutant cells are then crossed with cells deleted for <em>HHT1</em> to give rise to haploid cells harboring the <em>hht2</em> mutant gene as sole source of <a href="http://www.yeastgenome.org/locus/S000004976/overview">histone H3</a>. See the <em>Materials and Methods</em> section for more details on this strategy. PCR, polymerase chain reaction.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466033/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466033/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Oligonucleotide Primers</span><div class="sb-div caption-clear"></div></div></div><p id="p-13">The <strong><em>c</em></strong> fragments were then co-transformed along with the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">HIS3</a></em>-marked plasmid pRS413 (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Sikorski and Hieter 1989</a>) into strain yADP106, which contains the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002414">TRP1</a></em> genes in place of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004976">HHT2</a></em> gene (see <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1A</a>). His<sup>+</sup> transformants were then screened for 5-fluoroorotic acid (5-FOA) resistance and inability to grow on media lacking uracil and tryptophan, since these phenotypes indicate successful homologous recombination between the ends of the <strong><em>c</em></strong> fragments and regions flanking the <em>URA3-TRP1</em> genes—the pRS413 plasmid was then lost through mitotic cell division (see <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1C</a>). Integration candidates were then analyzed by sequencing both strands of fragments obtained from PCR reactions using the candidates’ genomic DNA as template and primers OAD476 and OAD477, which amplify a region that includes the entire integrated fragment as well as flanking sequences. For all H3-L61X strains used for subsequent experiments, sequencing of the corresponding OAD476-OAD477−amplified fragments revealed the expected mutations at the H3-L61X codon and no additional mutations. Strains yADP90-yADP105 were obtained through genetic crosses between the H3-L61X strains described above and yADP107. Strains yADP83-yADP88 were obtained through similar genetic crosses but with a version of yADP107 that harbors the pDM9 plasmid.</p></div><div id="sec-4" class="subsection"><h3>Chromatin immunoprecipitation (ChIP) assays</h3><p id="p-14">ChIP assays and real-time quantitative (q)PCR experiments to determine <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> occupancy at the different regions tested in this study were conducted using rabbit polyclonal antibodies specific for <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> (a gift from Tim Formosa) as previously described (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Myers <em>et al.</em> 2011</a>). The primers used for amplification of the specific genomic regions assayed by real-time qPCR were as follows: <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002976">PMA1</a></em> 5′ region, OAD394 and OAD395; <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002976">PMA1</a></em> internal region, OAD416 and OAD417; <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002976">PMA1</a></em> 3′ region, OAD383 and OAD384; <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001543">FBA1</a></em> 5′ region, OAD419 and OAD420; <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001543">FBA1</a></em> internal region, OAD421 and OAD422; <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001543">FBA1</a></em> 3′ region, OAD423 and OAD424; and nontranscribed region on chromosome V (also referred to as NO ORF region), OAD377 and OAD378. The sequences for these primers have been described in previous studies (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Myers <em>et al.</em> 2011</a>; <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Nguyen <em>et al.</em> 2013</a>).</p></div><div id="sec-5" class="subsection"><h3>Visualization of the nucleosomal region of interest and modeling of H3-L61X mutants</h3><p id="p-15">The yeast nucleosome structures shown in <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a> were obtained using the PyMOL Molecular Graphics System, Version 1.5.0.3 Schrödinger, LLC using structural information obtained in a previous study (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">White <em>et al.</em> 2001</a>) and available at the Research Collaboratory for Structural Bioinformatics protein data bank (PDB ID:1id3). In each case, modeling of an amino acid substitution at position 61 of <a href="http://www.yeastgenome.org/locus/S000004976/overview">histone H3</a> was accomplished using the Mutagenesis function in PyMOL and using the rotamer with the highest percentage value.</p></div></div><div class="section results" id="sec-6"><h2 class="">Results</h2><div id="sec-7" class="subsection"><h3>Generation and initial functional evaluation of H3-L61X mutants</h3><p id="p-16">To investigate the effects of amino acid substitutions at position 61 of <a href="http://www.yeastgenome.org/locus/S000004976/overview">histone H3</a> on various cellular processes, we first generated haploid yeast cells carrying targeted mutations at the endogenous <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004976">HHT2</a></em> locus (one of the two <a href="http://www.yeastgenome.org/locus/S000004976/overview">histone H3</a>-encoding genes in the haploid yeast genome) using the strategy shown in <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>. These experiments gave rise to 19 yeast strains, each expressing a version of <a href="http://www.yeastgenome.org/locus/S000004976/overview">histone H3</a> with a different substituted amino acid at position 61 (referred herein as H3-L61X mutants), collectively representing all possible natural amino acid substitutions at this position. The presence of a wild-type copy of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000214">HHT1</a></em>, the other gene encoding <a href="http://www.yeastgenome.org/locus/S000004976/overview">histone H3</a> in yeast, ensured that any strain expressing an H3-L61X mutant unable to support viability on its own could nevertheless be generated and analyzed. Each of the 19 mutant strains was then crossed with a strain deleted for <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000214">HHT1</a></em> to yield spores whose sole source of <a href="http://www.yeastgenome.org/locus/S000004976/overview">histone H3</a> was derived from the H3-L61X encoding gene. Results from these experiments showed that 16 of the 19 H3-L61X mutants are able to support life when present as only source of <a href="http://www.yeastgenome.org/locus/S000004976/overview">histone H3</a> in cells (see <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>), and provided initial evidence that three mutants—H3-L61P, H3-L61R, and H3-L61D—are unable to do so, as viable <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000214">hht1</a>∆ <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004976">hht2</a></em> double mutants were not recovered from their respective crosses. Subsequent viability tests confirmed that these three mutants are indeed unable to support life in the absence of a wild-type source of <a href="http://www.yeastgenome.org/locus/S000004976/overview">histone H3</a> (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/5/741/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The H3-L61P, H3-L61R, and H3-L61D mutants are unable to support viability when expressed as sole source of histone H3 in cells. Cells of the indicated genotypes and carrying a URA3-marked plasmid carrying a wild-type copy of HHT1 (pHHT1-URA3) were grown on nonselective medium for several generations to allow for loss of the plasmid. Following collection, cells were spotted onto SC-ura and 5-FOA plates in a 10-fold dilution series, in which the most concentrated spot (left-most spot on each panel) consisted of &#x223C;240,000 cells. The plates were then placed at 30&#xB0; and photographed after approximately 2 d (SC-ura) or 3 d (5-FOA) of incubation. Absence of growth on the 5-FOA plate reflects an inability of cells to grow in the absence of the plasmid, thus indicating that the corresponding hht2 allele cannot support viability when present as sole source of histone H3. Note that cells expressing the H3-L61P, H3-L61R, or H3-L61D mutant display a 5-FOA&#x2212;sensitive phenotype indistinguishable from that seen in hht2&#x2206; cells, whereas cells expressing wild-type histone H3 or the H3-L61E mutant (which confers viability as sole source of histone H3) show robust growth on 5-FOA medium. The strains used in these experiments were yADP83-yADP88. 5-FOA, 5-fluoroorotic acid." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2114265090" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The H3-L61P, H3-L61R, and H3-L61D mutants are unable to support viability when expressed as sole source of histone H3 in cells. Cells of the indicated genotypes and carrying a &lt;em&gt;URA3&lt;/em&gt;-marked plasmid carrying a wild-type copy of &lt;em&gt;HHT1&lt;/em&gt; (&lt;em&gt;pHHT1-URA3&lt;/em&gt;) were grown on nonselective medium for several generations to allow for loss of the plasmid. Following collection, cells were spotted onto SC-ura and 5-FOA plates in a 10-fold dilution series, in which the most concentrated spot (left-most spot on each panel) consisted of &#x223C;240,000 cells. The plates were then placed at 30&#xB0; and photographed after approximately 2 d (SC-ura) or 3 d (5-FOA) of incubation. Absence of growth on the 5-FOA plate reflects an inability of cells to grow in the absence of the plasmid, thus indicating that the corresponding &lt;em&gt;hht2&lt;/em&gt; allele cannot support viability when present as sole source of histone H3. Note that cells expressing the H3-L61P, H3-L61R, or H3-L61D mutant display a 5-FOA&#x2212;sensitive phenotype indistinguishable from that seen in &lt;em&gt;hht2&#x2206;&lt;/em&gt; cells, whereas cells expressing wild-type histone H3 or the H3-L61E mutant (which confers viability as sole source of histone H3) show robust growth on 5-FOA medium. The strains used in these experiments were yADP83-yADP88. 5-FOA, 5-fluoroorotic acid.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/5/741/F2.medium.gif" width="440" height="177"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/5/741/F2.medium.gif" width="440" height="177"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/5/741/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/5/741/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465990" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-17" class="first-child">The H3-L61P, H3-L61R, and H3-L61D mutants are unable to support viability when expressed as sole source of <a href="http://www.yeastgenome.org/locus/S000004976/overview">histone H3</a> in cells. Cells of the indicated genotypes and carrying a <em>URA3</em>-marked plasmid carrying a wild-type copy of <em>HHT1</em> (<em>pHHT1-URA3</em>) were grown on nonselective medium for several generations to allow for loss of the plasmid. Following collection, cells were spotted onto SC-ura and 5-FOA plates in a 10-fold dilution series, in which the most concentrated spot (left-most spot on each panel) consisted of ∼240,000 cells. The plates were then placed at 30° and photographed after approximately 2 d (SC-ura) or 3 d (5-FOA) of incubation. Absence of growth on the 5-FOA plate reflects an inability of cells to grow in the absence of the plasmid, thus indicating that the corresponding <em>hht2</em> allele cannot support viability when present as sole source of <a href="http://www.yeastgenome.org/locus/S000004976/overview">histone H3</a>. Note that cells expressing the H3-L61P, H3-L61R, or H3-L61D mutant display a 5-FOA−sensitive phenotype indistinguishable from that seen in <em>hht2∆</em> cells, whereas cells expressing wild-type <a href="http://www.yeastgenome.org/locus/S000004976/overview">histone H3</a> or the H3-L61E mutant (which confers viability as sole source of <a href="http://www.yeastgenome.org/locus/S000004976/overview">histone H3</a>) show robust growth on 5-FOA medium. The strains used in these experiments were yADP83-yADP88. 5-FOA, 5-fluoroorotic acid.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-8" class="subsection"><h3>Effects of H3-L61X mutants on Spt16−gene interactions</h3><p id="p-18">Previous studies have shown that the H3-L61W mutant perturbs association of the transcription elongation factor <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> across transcribed genes, causing a marked shift in its distribution toward the 3′ end of transcribed genes (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Duina <em>et al.</em> 2007</a>; <a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">Nguyen <em>et al.</em> 2013</a>). To investigate the effects of the newly generated H3-L61X mutants on <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>−gene interactions, we performed ChIP assays to monitor the binding pattern of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> across two constitutively transcribed genes in the context of the H3-L61X mutants [note that these and all subsequent experiments have been carried out using haploid strains deleted for <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000214">HHT1</a></em> and for <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000213">HHF1</a></em> (one of two genes encoding <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000213">histone H4</a> in yeast); therefore, these strains express <a href="http://www.yeastgenome.org/locus/S000004976/overview">histone H3</a> and H4 each from a single gene, with <a href="http://www.yeastgenome.org/locus/S000004976/overview">histone H3</a> being either wild-type or carrying one of the substitutions at position 61 able to confer viability —refer to <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>].</p><p id="p-19">As shown in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>, all 16 viable H3-L61X mutants cause statistically significant defects in <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> interactions at <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002976">PMA1</a></em> and, with the exception of the H3-L61F mutant, at <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001543">FBA1</a></em> as well. Interestingly, the mutant that confers the strongest defects in these assays is H3-L61W, which is the mutant that was identified and tested in our original studies (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Duina and Winston 2004</a>; <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Duina <em>et al.</em> 2007</a>). As previously shown for H3-L61W (<a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">Duina <em>et al.</em> 2007</a>; <a id="xref-ref-18-5" class="xref-bibr" href="#ref-18">Nguyen <em>et al.</em> 2013</a>), the H3-L61X mutants cause an overall shift in <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> occupancy toward the 3′ ends of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002976">PMA1</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001543">FBA1</a></em> genes (see <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3, C and F</a>). These results underscore the importance of the H3-L61 residue in contributing to proper <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>−gene interactions. The defects conferred by the H3-L61X mutants on <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>−gene interactions range from very mild to severe, and their relative degree of severity is comparable at the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002976">PMA1</a></em> and the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001543">FBA1</a></em> genes (compare <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3, C and F</a>), suggesting that the mechanism that underlies <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>−<a href="http://www.yeastgenome.org/locus/S000004976/overview">histone H3</a> interactions is similar at these genes and that specific H3-L61X mutants interfere with this mechanism similarly at both genes. Since previous work has shown that H3-L61W, the most defective mutant, causes relatively minor perturbations in nucleosome occupancy at the <em>PMA1-LEU1</em> region (<a id="xref-ref-6-5" class="xref-bibr" href="#ref-6">Duina <em>et al.</em> 2007</a>), we consider it unlikely that the H3-L61X mutants mediate their effects on <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>−gene interactions through alteration in overall abundance of nucleosomes across genes.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/5/741/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Effects of H3-L61X mutants on Spt16 occupancy across the PMA1 and FBA1 genes. (A,D). Schematic representations of genomic regions encompassing the PMA1 and FBA1 genes adapted from diagrams used in a previous study (Nguyen et al. 2013). The arrows indicate the direction of transcription of the corresponding genes and the numbers reflect nucleotide positions, with 1 corresponding to the &#x201C;A&#x201D; nucleotide at the beginning of the corresponding PMA1 or FBA1 open reading frames. The 5&#x2032;, internal, and 3&#x2032; locations correspond to the regions assayed in real-time qPCR experiments following the chromatin immunoprecipitation (ChIP) procedure (see Materials and Methods). (B,E) Results from Spt16 ChIP assays expressed as a ratio of the %IP of the indicated PMA1 or FBA1 regions to the %IP of a non-transcribed region on chromosome V (see Materials and Methods). (C,F) Results from the same ChIP experiments that gave rise to the data shown in panels B and E, but expressed as the ratio of the %IP of the 3&#x2032; region to the %IP of the 5&#x2032; region at PMA1 or FBA1. Representation of the ChIP data in this fashion facilitates the determination of the overall 5&#x2032; to 3&#x2032; shift in the distribution of Spt16 across the gene of interest. For the data shown in panels C and F, statistically significant differences in Spt16 distribution were established in all cases (Student&#x2019;s t-tests; P &lt; 0.05), except for the H3-L61F mutant at FBA1 (P = 0.056). For all charts in this figure, the H3-L61X mutants tested are shown on the X-axes and are arranged in increasing order of strength of mutant phenotype based on the values for the bar graphs shown in panel C. The data are expressed as the mean &#xB1; SEM from at least three independent experiments. The strains used in these experiments were yADP89-yADP105." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2114265090" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Effects of H3-L61X mutants on Spt16 occupancy across the &lt;em&gt;PMA1&lt;/em&gt; and &lt;em&gt;FBA1&lt;/em&gt; genes. (A,D). Schematic representations of genomic regions encompassing the &lt;em&gt;PMA1&lt;/em&gt; and &lt;em&gt;FBA1&lt;/em&gt; genes adapted from diagrams used in a previous study (Nguyen &lt;em&gt;et al.&lt;/em&gt; 2013). The arrows indicate the direction of transcription of the corresponding genes and the numbers reflect nucleotide positions, with 1 corresponding to the &#x201C;A&#x201D; nucleotide at the beginning of the corresponding &lt;em&gt;PMA1&lt;/em&gt; or &lt;em&gt;FBA1&lt;/em&gt; open reading frames. The 5&#x2032;, internal, and 3&#x2032; locations correspond to the regions assayed in real-time qPCR experiments following the chromatin immunoprecipitation (ChIP) procedure (see &lt;em&gt;Materials and Methods&lt;/em&gt;). (B,E) Results from Spt16 ChIP assays expressed as a ratio of the %IP of the indicated &lt;em&gt;PMA1&lt;/em&gt; or &lt;em&gt;FBA1&lt;/em&gt; regions to the %IP of a non-transcribed region on chromosome V (see &lt;em&gt;Materials and Methods&lt;/em&gt;). (C,F) Results from the same ChIP experiments that gave rise to the data shown in panels B and E, but expressed as the ratio of the %IP of the 3&#x2032; region to the %IP of the 5&#x2032; region at &lt;em&gt;PMA1&lt;/em&gt; or &lt;em&gt;FBA1&lt;/em&gt;. Representation of the ChIP data in this fashion facilitates the determination of the overall 5&#x2032; to 3&#x2032; shift in the distribution of Spt16 across the gene of interest. For the data shown in panels C and F, statistically significant differences in Spt16 distribution were established in all cases (Student&#x2019;s &lt;em&gt;t&lt;/em&gt;-tests; &lt;em&gt;P&lt;/em&gt; &lt; 0.05), except for the H3-L61F mutant at &lt;em&gt;FBA1&lt;/em&gt; (&lt;em&gt;P&lt;/em&gt; = 0.056). For all charts in this figure, the H3-L61X mutants tested are shown on the X-axes and are arranged in increasing order of strength of mutant phenotype based on the values for the bar graphs shown in panel C. The data are expressed as the mean &#xB1; SEM from at least three independent experiments. The strains used in these experiments were yADP89-yADP105.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/5/741/F3.medium.gif" width="440" height="417"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/5/741/F3.medium.gif" width="440" height="417"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/5/741/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/5/741/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466072" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-20" class="first-child">Effects of H3-L61X mutants on Spt16 occupancy across the <em>PMA1</em> and <em>FBA1</em> genes. (A,D). Schematic representations of genomic regions encompassing the <em>PMA1</em> and <em>FBA1</em> genes adapted from diagrams used in a previous study (<a id="xref-ref-18-6" class="xref-bibr" href="#ref-18">Nguyen <em>et al.</em> 2013</a>). The arrows indicate the direction of transcription of the corresponding genes and the numbers reflect nucleotide positions, with 1 corresponding to the “A” nucleotide at the beginning of the corresponding <em>PMA1</em> or <em>FBA1</em> open reading frames. The 5′, internal, and 3′ locations correspond to the regions assayed in real-time qPCR experiments following the chromatin immunoprecipitation (ChIP) procedure (see <em>Materials and Methods</em>). (B,E) Results from Spt16 ChIP assays expressed as a ratio of the %IP of the indicated <em>PMA1</em> or <em>FBA1</em> regions to the %IP of a non-transcribed region on chromosome V (see <em>Materials and Methods</em>). (C,F) Results from the same ChIP experiments that gave rise to the data shown in panels B and E, but expressed as the ratio of the %IP of the 3′ region to the %IP of the 5′ region at <em>PMA1</em> or <em>FBA1</em>. Representation of the ChIP data in this fashion facilitates the determination of the overall 5′ to 3′ shift in the distribution of Spt16 across the gene of interest. For the data shown in panels C and F, statistically significant differences in Spt16 distribution were established in all cases (Student’s <em>t</em>-tests; <em>P</em> &lt; 0.05), except for the H3-L61F mutant at <em>FBA1</em> (<em>P</em> = 0.056). For all charts in this figure, the H3-L61X mutants tested are shown on the X-axes and are arranged in increasing order of strength of mutant phenotype based on the values for the bar graphs shown in panel C. The data are expressed as the mean ± SEM from at least three independent experiments. The strains used in these experiments were yADP89-yADP105.</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">An analysis of the results from these experiments offers insights into the features of the H3-L61 residue important for promoting proper <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>−gene interactions. First, its hydrophobic nature appears to be critical as hydrophilic amino acid substitutions at position 61 are among the ones causing the strongest defects. This finding is consistent with the observation that whereas the H3-L61F mutant causes minor defects in <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> interactions with <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002976">PMA1</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001543">FBA1</a></em>, H3-L61Y causes much more pronounced perturbations, possibly due to the hydrophilic nature of the hydroxyl group on tyrosine. Second, the finding that substitutions at position 61 with either isoleucine or valine—both of which might be expected to be well tolerated in place of leucine—cause strong defects in these assays could indicate that the precise structural conformation adopted by H3-L61 is required for proper <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>−gene interactions and that isoleucine and valine—both C-beta branched amino acids—are unable to adopt such conformation due their more limited rotational freedom across the polypeptide backbone. Third, the size of the amino acid side chain at position 61 also seems to be relevant as a substitution with the hydrophobic but much bulkier amino acid tryptophan causes the strongest perturbations in <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>−gene interactions. Attributes shared between leucine and the substitutions that confer the smallest defects—methionine, phenylalanine, and cysteine—are hydrophobicity, normal flexibility, and lack of excessive bulk.</p></div><div id="sec-9" class="subsection"><h3>Analysis of growth phenotypes of H3-L61X mutants</h3><p id="p-22">Previous analyses have shown that the H3-L61W mutant displays several abnormal growth phenotypes (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Duina and Winston 2004</a>; <a id="xref-ref-6-6" class="xref-bibr" href="#ref-6">Duina <em>et al.</em> 2007</a>). To more fully characterize the newly generated <a href="http://www.yeastgenome.org/locus/S000004976/overview">histone H3</a> mutants and to assess possible correlations between growth defects and the defects in <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>−gene interactions as determined by the ChIP experiments described in the previous section, we tested the ability of the viable H3-L61X mutants to grow under a variety of conditions using serial dilution spot test assays (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>).</p><p id="p-23">Results from these experiments show that most of the mutants display growth defects at high and low temperatures, as well as other mutant phenotypes. With the exception of the H3-L61I and H3-L61V mutants, there appears to be a general correlation between impaired growth at 37° and perturbations in <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>−gene interactions (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>: compare growth at 37° of the weakest <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>−gene interaction mutants H3-L61M, -L61F, and -L61C with the other H3-L61X mutants). The observation that H3-L61I and H3-L61V mutants do not show a temperature sensitive phenotype at 37° might indicate that although the structural rigidity imparted by isoleucine and valine at position 61 of <a href="http://www.yeastgenome.org/locus/S000004976/overview">histone H3</a> interferes with <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>−interactions, it does not cause cells to become sensitive to elevated temperatures. Conversely, disruptions of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>−gene interactions through introduction of polarity/charge or bulk at position 61 also cause cells to display a temperature sensitive phenotype at 37°. A general correlation between cold sensitivity and defects in <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>−gene interactions can also be seen, as H3-L61X mutants that cause moderate to strong perturbations in <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>−gene interactions show a range of sensitivities to the low temperature of 14° (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/5/741/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phenotypes of cells expressing H3-L61X mutants subjected to a variety of growth conditions. Cells expressing the indicated histone H3 proteins as sole source of histone H3 were grown overnight on YPD medium and subsequently collected and spotted on the indicated plates (for each condition tested, all mutants were spotted on the same plate) in a 10-fold dilution series, with the most concentrated spots (left-most spot in each panel) consisting of &#x223C;120,000 cells. Plates were placed at the indicated temperatures (note that the drug-containing plates were incubated at 30&#xB0;), and photographed after the following approximate incubation times: YPD 30&#xB0;, 2 d; YPD 37&#xB0;, 2 d; YPD 14&#xB0;, 7 and 10 d, as indicated; YPD + formamide (Form), 3 d; YPD + hydroxyurea (HU), 3 d; YPD + caffeine (Caff), 5 d. The YPD 14&#xB0; plate is shown at two different incubation times to allow for assessment of subtle cold-sensitive phenotypes (e.g., for the H3-L61H mutant). Mutants are presented in the same order as that shown in Figure 3. The growth patterns shown in this figure were closely reproduced in an independent experiment for all samples except for H3-L61G, which showed modestly stronger growth on the YPD 30&#xB0; plate. Additional growth experiments showed that the variable growth phenotype conferred by H3-L61G is an inherent property of this mutant. The strains used in these experiments were yADP89-yADP105. YPD, yeast extract peptone dextrose." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2114265090" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Phenotypes of cells expressing H3-L61X mutants subjected to a variety of growth conditions. Cells expressing the indicated histone H3 proteins as sole source of histone H3 were grown overnight on YPD medium and subsequently collected and spotted on the indicated plates (for each condition tested, all mutants were spotted on the same plate) in a 10-fold dilution series, with the most concentrated spots (left-most spot in each panel) consisting of &#x223C;120,000 cells. Plates were placed at the indicated temperatures (note that the drug-containing plates were incubated at 30&#xB0;), and photographed after the following approximate incubation times: YPD 30&#xB0;, 2 d; YPD 37&#xB0;, 2 d; YPD 14&#xB0;, 7 and 10 d, as indicated; YPD + formamide (Form), 3 d; YPD + hydroxyurea (HU), 3 d; YPD + caffeine (Caff), 5 d. The YPD 14&#xB0; plate is shown at two different incubation times to allow for assessment of subtle cold-sensitive phenotypes (&lt;em&gt;e.g.&lt;/em&gt;, for the H3-L61H mutant). Mutants are presented in the same order as that shown in Figure 3. The growth patterns shown in this figure were closely reproduced in an independent experiment for all samples except for H3-L61G, which showed modestly stronger growth on the YPD 30&#xB0; plate. Additional growth experiments showed that the variable growth phenotype conferred by H3-L61G is an inherent property of this mutant. The strains used in these experiments were yADP89-yADP105. YPD, yeast extract peptone dextrose.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/5/741/F4.medium.gif" width="440" height="305"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/5/741/F4.medium.gif" width="440" height="305"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/5/741/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/5/741/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466099" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-24" class="first-child">Phenotypes of cells expressing H3-L61X mutants subjected to a variety of growth conditions. Cells expressing the indicated <a href="http://www.yeastgenome.org/locus/S000004976/overview">histone H3</a> proteins as sole source of <a href="http://www.yeastgenome.org/locus/S000004976/overview">histone H3</a> were grown overnight on YPD medium and subsequently collected and spotted on the indicated plates (for each condition tested, all mutants were spotted on the same plate) in a 10-fold dilution series, with the most concentrated spots (left-most spot in each panel) consisting of ∼120,000 cells. Plates were placed at the indicated temperatures (note that the drug-containing plates were incubated at 30°), and photographed after the following approximate incubation times: YPD 30°, 2 d; YPD 37°, 2 d; YPD 14°, 7 and 10 d, as indicated; YPD + formamide (Form), 3 d; YPD + hydroxyurea (HU), 3 d; YPD + caffeine (Caff), 5 d. The YPD 14° plate is shown at two different incubation times to allow for assessment of subtle cold-sensitive phenotypes (<em>e.g.</em>, for the H3-L61H mutant). Mutants are presented in the same order as that shown in <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>. The growth patterns shown in this figure were closely reproduced in an independent experiment for all samples except for H3-L61G, which showed modestly stronger growth on the YPD 30° plate. Additional growth experiments showed that the variable growth phenotype conferred by H3-L61G is an inherent property of this mutant. The strains used in these experiments were yADP89-yADP105. YPD, yeast extract peptone dextrose.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">Sensitivities of H3-L61X mutants to the drugs formamide, hydroxyurea, and caffeine range from none to very strong and they do not show a clear correlation with impairment in <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>−gene interactions (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a> and <a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a>). In most cases, H3-L61X mutants that show sensitivity to one drug also show sensitivity to the other two drugs suggesting that these sensitivities share a common underlying mechanism.</p></div><div id="sec-10" class="subsection"><h3>Assessment of effects of H3-L61X mutants on phenotypes indicative of transcription and chromatin defects</h3><p id="p-26">We continued the characterization of the newly generated H3-L61X mutants by testing them for phenotypes associated with defects in transcription and chromatin function. In one set of experiments, H3-L61X mutant cells carrying the <em>lys2-128δ</em> allele were assayed for growth on medium lacking lysine as the ability to grow under these conditions—a phenotype referred to as Spt<sup>−</sup>—is indicative of perturbations in transcription and chromatin structure (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Winston 1992</a>). As shown in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>, with the exception of H3-L61F, all other H3-L61X mutants display a range of Spt<sup>−</sup> phenotypes, thus underscoring the importance of L61 of <a href="http://www.yeastgenome.org/locus/S000004976/overview">histone H3</a> in maintaining chromatin integrity. The strength of the Spt<sup>−</sup> phenotypes conferred by the H3-L61X mutants does not correlate with the strength of the defects they impart on <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>−gene interactions (for example, compare the Spt<sup>−</sup> phenotypes of the H3-L61C and H3-L61T mutants in <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a> with the perturbations they impart on <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>−gene interaction in <a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3</a>), suggesting that the two phenotypes are not mechanistically related to each other.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/5/741/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Effects of H3-L61X mutants on growth phenotypes indicative of transcription and chromatin defects. Cells expressing the indicated H3-L61X mutants were treated as described in Figure 4, except that the most concentrated spots consisted of &#x223C;240,000 cells. Plates were placed at 30&#xB0;C and photographed after the following approximate times of incubation: SC, 2 d; SC-lys, 2 d; SC-his, 3 d; SC GAL, 3 d; SC-his GAL, 3 d. The growth patterns shown in this figure were closely reproduced in an independent experiment for all samples except for H3-L61G, which, as was the case in the experiments described in Figure 4, showed some variability: this mutant showed modestly stronger growth on the SC and SC GAL plates and weaker growth on the SC-his plate. The strains used in these experiments were yADP89-yADP105. SC, SC-lys, and SC-his plates contained glucose and SC GAL and SC-his GAL plates contained galactose as carbon and energy source." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2114265090" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Effects of H3-L61X mutants on growth phenotypes indicative of transcription and chromatin defects. Cells expressing the indicated H3-L61X mutants were treated as described in Figure 4, except that the most concentrated spots consisted of &#x223C;240,000 cells. Plates were placed at 30&#xB0;C and photographed after the following approximate times of incubation: SC, 2 d; SC-lys, 2 d; SC-his, 3 d; SC GAL, 3 d; SC-his GAL, 3 d. The growth patterns shown in this figure were closely reproduced in an independent experiment for all samples except for H3-L61G, which, as was the case in the experiments described in Figure 4, showed some variability: this mutant showed modestly stronger growth on the SC and SC GAL plates and weaker growth on the SC-his plate. The strains used in these experiments were yADP89-yADP105. SC, SC-lys, and SC-his plates contained glucose and SC GAL and SC-his GAL plates contained galactose as carbon and energy source.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/5/741/F5.medium.gif" width="440" height="389"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/5/5/741/F5.medium.gif" width="440" height="389"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/5/741/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/5/741/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466114" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-27" class="first-child">Effects of H3-L61X mutants on growth phenotypes indicative of transcription and chromatin defects. Cells expressing the indicated H3-L61X mutants were treated as described in <a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4</a>, except that the most concentrated spots consisted of ∼240,000 cells. Plates were placed at 30°C and photographed after the following approximate times of incubation: SC, 2 d; SC-lys, 2 d; SC-his, 3 d; SC GAL, 3 d; SC-his GAL, 3 d. The growth patterns shown in this figure were closely reproduced in an independent experiment for all samples except for H3-L61G, which, as was the case in the experiments described in <a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4</a>, showed some variability: this mutant showed modestly stronger growth on the SC and SC GAL plates and weaker growth on the SC-his plate. The strains used in these experiments were yADP89-yADP105. SC, SC-lys, and SC-his plates contained glucose and SC GAL and SC-his GAL plates contained galactose as carbon and energy source.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">In a second set of experiments we made use of the <em>FLO8-HIS3</em> reporter gene, which contains the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">HIS3</a></em> coding region fused downstream from a cryptic transcription initiation site within the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000911">FLO8</a></em> coding region (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Cheung <em>et al.</em> 2008</a>), to monitor for intragenic transcription initiation events. Since <em>FLO8-HIS3</em> is under the control of the inducible <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000224">GAL1</a></em> promoter, intragenic transcription initiation events can be easily monitored by assaying cells—otherwise deleted for the endogenous <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">HIS3</a></em> gene—for growth on media lacking histidine in the context of high transcription levels (<em>i.e.</em>, in the presence of galactose) or in the context of little to no transcription (<em>i.e.</em>, in the presence of glucose). As previously reported (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Myers <em>et al.</em> 2011</a>; <a id="xref-ref-18-7" class="xref-bibr" href="#ref-18">Nguyen <em>et al.</em> 2013</a>), the H3-L61W mutant confers a strong His<sup>+</sup> phenotype on media containing either glucose or galactose (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>). Although a few H3-L61X mutant strains behave indistinguishably from wild-type cells on media lacking histidine, most show His<sup>+</sup> phenotypes ranging from mild to strong (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5</a>). Members of this latter class of mutants display stronger growth on galactose-containing medium (SC-his GAL) compared to glucose-containing medium (SC-his), suggesting that the chromatin defects leading to intragenic transcription initiation imparted by these H3-L61X mutants are more pronounced in the context of high levels of transcription (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5</a>). Interestingly, we see a general correlation between H3-L61X mutants’ ability to confer growth on SC-his GAL medium and the degree of the defects they impart on <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>−gene interactions as assessed by the ChIP experiments shown in <a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3</a>. For example, the eight mutants that confer the strongest defects in <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>−gene interactions (H3-L61I, Y, E, H, V, T, K, and W) are also the ones that show the most robust growth on SC-his GAL plates. Taken together, these results suggest that the defects in repression of intragenic transcription initiation and in <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>−gene interactions caused by mutations at position 61 of <a href="http://www.yeastgenome.org/locus/S000004976/overview">histone H3</a> are in some manner mechanistically linked to each other.</p></div></div><div class="section discussion" id="sec-11"><h2 class="">Discussion</h2><p id="p-29">In this work, we have carried out a systematic mutational analysis of the H3-L61 residue to gain insights into its contribution to chromatin function. Our finding that substitutions of H3-L61 to each of the remaining 19 natural amino acids confer defects in one or more of the phenotypes we have assayed highlights the importance of H3-L61 in maintaining chromatin integrity and is consistent with the high degree of evolutionary conservation of this residue across eukaryotic organisms (we note that this level of conservation is not unique to the L61 residue as most other <a href="http://www.yeastgenome.org/locus/S000004976/overview">histone H3</a> residues are also highly conserved). An evaluation of the nature of the defects conferred by each of the H3-L61X mutants indicates that features of H3-L61 important for maintaining proper nucleosome functions include its hydrophobicity, its flexibility across the polypeptide backbone, and its lack of excessive bulk. The aliphatic nature of H3-L61 does not appear to be critical for these functions as a mutation to the aromatic amino acid phenylalanine did not cause strong defects in our assays. We also provide evidence that H3-L61X−dependent defects in <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>−gene interaction and in repression of intragenic transcription initiation are mechanistically linked to each other.</p><p id="p-30">In previous work, we showed that an H4-R36A mutant perturbs <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> association across genes in a fashion similar to that seen in the context of H3-L61W, and genetic evidence indicated that these two histone mutants affect <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>−gene interactions through a common mechanism (<a id="xref-ref-18-8" class="xref-bibr" href="#ref-18">Nguyen <em>et al.</em> 2013</a>). H4-R36 and H3-L61 are adjacent to each other on the side of the nucleosome, with the former being located on the surface making direct contacts with DNA and the latter being buried into the particle ((<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Luger <em>et al.</em> 1997</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Luger and Richmond 1998</a>) and <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6A</a>). How do these histone residues contribute to proper <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>−chromatin interactions, and how might they be functionally linked? Based on our original studies, we considered a model in which H4-R36 is part of a nucleosomal surface that interacts with <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>, and mutations that perturb this surface can lead to abnormal <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> interactions across genes. In the context of this model, we envisioned two possible mechanisms by which H3-L61W could affect H4-R36 function: (i) the π electrons of the aromatic ring of the tryptophan residue could interfere with the ability of H4-R36 to form ionic bonds with other factors, or (ii) the integrity of the hydrophobic microenvironment normally occupied by H3-L61 is required for proper positioning of H4-R36 for interactions with other factors, and the tryptophan residue in H3-L61W disrupts this microenvironment through its bulky nature. The results from the present study do not support the first mechanism, as the associated predictions that substitutions of H3-L61 with the aromatic amino acid phenylalanine should cause strong perturbations in <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>−gene interactions while substitutions with aliphatic hydrophobic residues (<em>e.g.</em>, alanine and isoleucine) should not, were not substantiated. Our results are instead consistent with the second mechanism, as H3-L61X mutants (modeled in <a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6B</a>) that are predicted to disrupt the H3-L61 microenvironment through the introduction of polarity/charge or bulk or by interfering with H3 polypeptide flexibility all cause strong perturbations in <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>−gene interactions. The H3-L61X mutants’ effects on <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>−chromatin interactions are likely to be for the most part specific, as previous work has shown that H3-L61W, the strongest H3-L61X mutant, does not significantly affect Pol II− and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003295">Spt4</a>−gene interactions and only modestly affects <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003348">Spt6</a>−gene interactions (<a id="xref-ref-6-7" class="xref-bibr" href="#ref-6">Duina <em>et al.</em> 2007</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Lloyd <em>et al.</em> 2009</a>). The isolation of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> mutants that can suppress H3-L61W−dependent defects in <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>−chromatin interactions (<a id="xref-ref-6-8" class="xref-bibr" href="#ref-6">Duina <em>et al.</em> 2007</a>; <a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">Myers <em>et al.</em> 2011</a>) further points to a specific relationship between <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> and the H3-L61 residue.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/5/741/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Location of the H3-L61 and H4-R36 residues in the nucleosome and modeling of the H3-L61X mutants. (A) Side-view of the yeast nucleosome core particle with the DNA shown in olive green and the histones color-coded as indicated in the figure. The H3-L61 and H4-R36 residues are shown in stick representation and color-coded as indicated. The blue box represents the region analyzed in panel B. (B) Close-up views of the nucleosomal region indicated by the blue box in panel A showing the wild-type structure (top-left image with blue border) and models of the H3-L61X substitutions. The images with purple borders correspond to the H3-L61X mutants able to confer viability when expressed a sole source of histone H3 in cells and are shown in the same order as that used for the experiments shown in Figure 3, Figure 4, and Figure 5. Models of the H3-L61X mutants unable to support viability in the absence of wild-type histone H3 (see Figure 2) are framed by orange borders. The H3-L61X models were generated using PyMOL and show the orientation of the substituted residues based on the rotamers with greatest frequencies found in other proteins as calculated by the software program." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2114265090" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Location of the H3-L61 and H4-R36 residues in the nucleosome and modeling of the H3-L61X mutants. (A) Side-view of the yeast nucleosome core particle with the DNA shown in olive green and the histones color-coded as indicated in the figure. The H3-L61 and H4-R36 residues are shown in stick representation and color-coded as indicated. The blue box represents the region analyzed in panel B. (B) Close-up views of the nucleosomal region indicated by the blue box in panel A showing the wild-type structure (top-left image with blue border) and models of the H3-L61X substitutions. The images with purple borders correspond to the H3-L61X mutants able to confer viability when expressed a sole source of histone H3 in cells and are shown in the same order as that used for the experiments shown in Figure 3, Figure 4, and Figure 5. Models of the H3-L61X mutants unable to support viability in the absence of wild-type histone H3 (see Figure 2) are framed by orange borders. The H3-L61X models were generated using PyMOL and show the orientation of the substituted residues based on the rotamers with greatest frequencies found in other proteins as calculated by the software program.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/5/741/F6.medium.gif" width="440" height="345"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/5/5/741/F6.medium.gif" width="440" height="345"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/5/741/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/5/741/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466007" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-31" class="first-child">Location of the H3-L61 and H4-R36 residues in the nucleosome and modeling of the H3-L61X mutants. (A) Side-view of the yeast nucleosome core particle with the DNA shown in olive green and the histones color-coded as indicated in the figure. The H3-L61 and H4-R36 residues are shown in stick representation and color-coded as indicated. The blue box represents the region analyzed in panel B. (B) Close-up views of the nucleosomal region indicated by the blue box in panel A showing the wild-type structure (top-left image with blue border) and models of the H3-L61X substitutions. The images with purple borders correspond to the H3-L61X mutants able to confer viability when expressed a sole source of <a href="http://www.yeastgenome.org/locus/S000004976/overview">histone H3</a> in cells and are shown in the same order as that used for the experiments shown in <a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3</a>, <a id="xref-fig-4-8" class="xref-fig" href="#F4">Figure 4</a>, and <a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5</a>. Models of the H3-L61X mutants unable to support viability in the absence of wild-type <a href="http://www.yeastgenome.org/locus/S000004976/overview">histone H3</a> (see <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>) are framed by orange borders. The H3-L61X models were generated using PyMOL and show the orientation of the substituted residues based on the rotamers with greatest frequencies found in other proteins as calculated by the software program.</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">Most H3-L61X mutants cause decreased levels of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> occupancy at the 5′ ends of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002976">PMA1</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001543">FBA1</a></em> while simultaneously elevating the levels of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> occupancy at the 3′ ends of the same genes. In addition, in this study we have provided evidence that the defects in <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>−gene interactions imparted by H3-L61X mutants are correlated with defects in repression of intragenic transcription initiation. Currently, the molecular mechanism that underlies this relationship is unknown. One possibility is that the primary defect conferred by the H3-L61X mutants is to directly interfere with the ability of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> to dissociate from the 3′ ends of genes at the end of the transcription process. Since this 3′ trapping may be occurring throughout the genome (we have previously shown that H3-L61W affects interactions between <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> and all other transcribed genes assayed—a total of six genes (<a id="xref-ref-6-9" class="xref-bibr" href="#ref-6">Duina <em>et al.</em> 2007</a>)), it could lead to reduced levels of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> available for recruitment to transcribed genes, resulting in lower <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> occupancy levels at 5′ ends of genes as well as in defects in Spt16-mediated transcription-coupled nucleosome reassembly over the bodies of genes, which would ultimately lead to intragenic transcription initiation. Alternatively, it is possible that the H3-L61X mutants directly affect chromatin environments over transcribed genes in ways that cause cryptic intragenic transcription initiation, for example, by interfering with proper transcription-coupled nucleosome reassembly, which in turn could in some manner cause a shift in <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> distribution toward the 3′ ends of genes. </p><p id="p-33">We note, however, that cryptic intragenic transcription initiation <em>per se</em> is not sufficient for causing a 3′ shift of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> across genes, since in previous work we have isolated several <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> mutants that cause strong intragenic transcription initiation but that do not confer significant defects in <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> association across the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002976">PMA1</a></em> gene (<a id="xref-ref-17-5" class="xref-bibr" href="#ref-17">Myers <em>et al.</em> 2011</a>). We previously disfavored the possibility that the H3-L61W−dependent defects in <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a>−gene interactions and in repression of intragenic transcription initiation are functionally related to each other in light of the fact that among several <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> mutants we isolated, Spt16-E735G was among the strongest suppressors of the H3-L61W−dependent defect in <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> distribution across <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002976">PMA1</a></em>, but was the weakest suppressor of the intragenic transcription initiation defect at <em>FLO8-HIS3</em> (<a id="xref-ref-17-6" class="xref-bibr" href="#ref-17">Myers <em>et al.</em> 2011</a>). However, Spt16-E735G can cause intragenic transcription initiation at <em>FLO8-HIS3</em> in a <a href="http://www.yeastgenome.org/locus/S000004976/overview">histone H3</a> wild-type background (<a id="xref-ref-17-7" class="xref-bibr" href="#ref-17">Myers <em>et al.</em> 2011</a>), and therefore the apparent lack of suppression of the H3-L61W−dependent defect in repression of intragenic transcription could be attributed to Spt16-E735G’s own ability to promote intragenic transcription initiation.</p><p id="p-34">In conclusion, our systematic mutational analysis has shed new light on the contribution of the H3-L61 residue to proper chromatin function. Future studies will focus on further characterizing the nature of the nucleosomal region involved in controlling <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> interactions with transcribed genes. It will also be of interest to determine the functional outcomes of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> trapping at the 3′ ends of genes due to histone mutants, in particular in relation to possible effects on nucleosome composition and on transcription termination. Collectively, these and related studies will contribute to our understanding of the mechanisms that govern interactions between <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> and nucleosomes during transcription, and possibly, during other chromatin processes that involve <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003175">Spt16</a> function, such as DNA replication and repair.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknoweldgments</h2><p id="p-35">We thank Andres Caro, Karolin Luger, Reine Protacio, and Fred Winston for helpful comments on the manuscript. We also thank Tim Formosa for providing the anti-Spt16 antibodies. This material is based upon work supported by the National Science Foundation under grant no. 1243680.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-3"><p id="p-36"><em>Communicating editor: B. A. Cohen</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-01-13"><span class="received-label">Received </span>January 13, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-02-22"><span class="accepted-label">Accepted </span>February 22, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Johnson <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.741.1" data-doi="10.1074/jbc.273.34.21972"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brewster</span>  <span class="cit-name-given-names">N. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">G. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singer</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Characterization of the CP complex, an abundant dimer of Cdc68 and Pob3 proteins that regulates yeast transcriptional activation and chromatin repression.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">273</span>: <span class="cit-fpage">21972</span>–<span class="cit-lpage">21979</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DBrewster%26rft.auinit1%253DN.%2BK.%26rft.volume%253D273%26rft.issue%253D34%26rft.spage%253D21972%26rft.epage%253D21979%26rft.atitle%253DCharacterization%2Bof%2Bthe%2BCP%2BComplex%252C%2Ban%2BAbundant%2BDimer%2Bof%2BCdc68%2Band%2BPob3%2BProteins%2BThat%2BRegulates%2BYeast%2BTranscriptional%2BActivation%2Band%2BChromatin%2BRepression%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.273.34.21972%26rft_id%253Dinfo%253Apmid%252F9705338%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzMvMzQvMjE5NzIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzUvNzQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.741.2" data-doi="10.1146/annurev.genet.032608.103928"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Campos</span>  <span class="cit-name-given-names">E. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reinberg</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Histones: annotating chromatin.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">559</span>–<span class="cit-lpage">599</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DCampos%26rft.auinit1%253DE.%2BI.%26rft.volume%253D43%26rft.spage%253D559%26rft.epage%253D599%26rft.atitle%253DHistones%253A%2Bannotating%2Bchromatin.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.032608.103928%26rft_id%253Dinfo%253Apmid%252F19886812%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genet.032608.103928&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19886812&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F741.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273580300023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.741.3" data-doi="10.1371/journal.pbio.0060277"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheung</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chua</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Batada</span>  <span class="cit-name-given-names">N. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landry</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michnick</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Chromatin- and transcription-related factors repress transcription from within coding regions throughout the <em>Saccharomyces cerevisiae</em> genome.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e277</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DCheung%26rft.auinit1%253DV.%26rft.volume%253D6%26rft.issue%253D11%26rft.spage%253De277%26rft.epage%253De277%26rft.atitle%253DChromatin-%2Band%2Btranscription-related%2Bfactors%2Brepress%2Btranscription%2Bfrom%2Bwithin%2Bcoding%2Bregions%2Bthroughout%2Bthe%2BSaccharomyces%2Bcerevisiae%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0060277%26rft_id%253Dinfo%253Apmid%252F18998772%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0060277&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18998772&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F741.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-book" id="cit-5.5.741.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duina</span>  <span class="cit-name-given-names">A. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Histone chaperones Spt6 and FACT: similarities and differences in modes of action at transcribed genes.</span> <span class="cit-source">Genet. Res. Int.</span> <span class="cit-vol">2011</span>: <span class="cit-fpage">625201</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BRes.%2BInt.%26rft.volume%253D2011%26rft.spage%253D625201%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.741.5" data-doi="10.1128/MCB.24.2.561-572.2004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duina</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winston</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Analysis of a mutant <a href="http://www.yeastgenome.org/locus/S000004976/overview">histone H3</a> that perturbs the association of Swi/Snf with chromatin.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">561</span>–<span class="cit-lpage">572</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DDuina%26rft.auinit1%253DA.%2BA.%26rft.volume%253D24%26rft.issue%253D2%26rft.spage%253D561%26rft.epage%253D572%26rft.atitle%253DAnalysis%2Bof%2Ba%2BMutant%2BHistone%2BH3%2BThat%2BPerturbs%2Bthe%2BAssociation%2Bof%2BSwi%252FSnf%2Bwith%2BChromatin%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.24.2.561-572.2004%26rft_id%253Dinfo%253Apmid%252F14701730%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjg6IjI0LzIvNTYxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS81Lzc0MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.741.6" data-doi="10.1534/genetics.106.067140"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duina</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rufiange</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bracey</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nourani</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Evidence that the localization of the elongation factor Spt16 across transcribed genes is dependent upon <a href="http://www.yeastgenome.org/locus/S000004976/overview">histone H3</a> integrity in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">101</span>–<span class="cit-lpage">112</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.067140%26rft_id%253Dinfo%253Apmid%252F17603125%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc3LzEvMTAxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS81Lzc0MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.741.7" data-doi="10.1039/b812136b"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Formosa</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">FACT and the reorganized nucleosome.</span> <abbr class="cit-jnl-abbrev">Mol. Biosyst.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1085</span>–<span class="cit-lpage">1093</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DMol%2BBiosyst%26rft.aulast%253DFormosa%26rft.auinit1%253DT.%26rft.volume%253D4%26rft.issue%253D11%26rft.spage%253D1085%26rft.epage%253D1093%26rft.atitle%253DFACT%2Band%2Bthe%2Breorganized%2Bnucleosome.%26rft_id%253Dinfo%253Adoi%252F10.1039%252Fb812136b%26rft_id%253Dinfo%253Apmid%252F18931784%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1039/b812136b&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18931784&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F741.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.741.8" data-doi="10.1016/j.bbagrm.2011.07.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Formosa</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The role of FACT in making and breaking nucleosomes.</span> <abbr class="cit-jnl-abbrev">Biochim. Biophys. Acta</abbr> <span class="cit-vol">1819</span>: <span class="cit-fpage">247</span>–<span class="cit-lpage">255</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.stitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.aulast%253DFormosa%26rft.auinit1%253DT.%26rft.volume%253D1819%26rft.issue%253D3-4%26rft.spage%253D247%26rft.epage%253D255%26rft.atitle%253DThe%2Brole%2Bof%2BFACT%2Bin%2Bmaking%2Band%2Bbreaking%2Bnucleosomes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bbagrm.2011.07.009%26rft_id%253Dinfo%253Apmid%252F21807128%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bbagrm.2011.07.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21807128&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F741.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.741.9" data-doi="10.1093/emboj/20.13.3506"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Formosa</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eriksson</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wittmeyer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ginn</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Spt16-Pob3 and the HMG protein Nhp6 combine to form the nucleosome-binding factor SPN.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">3506</span>–<span class="cit-lpage">3517</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DFormosa%26rft.auinit1%253DT.%26rft.volume%253D20%26rft.issue%253D13%26rft.spage%253D3506%26rft.epage%253D3517%26rft.atitle%253DSpt16-Pob3%2Band%2Bthe%2BHMG%2Bprotein%2BNhp6%2Bcombine%2Bto%2Bform%2Bthe%2Bnucleosome-binding%2Bfactor%2BSPN%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252F20.13.3506%26rft_id%253Dinfo%253Apmid%252F11432837%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMjAvMTMvMzUwNiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvNS83NDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.741.10" data-doi="10.1038/nature12242"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hondele</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stuwe</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hassler</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Halbach</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Structural basis of histone H2A–H2B recognition by the essential chaperone FACT.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">499</span>: <span class="cit-fpage">111</span>–<span class="cit-lpage">114</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D499%26rft.spage%253D111%26rft.atitle%253DSTRUCTURAL%2BBASIS%2BOF%2BHISTONE%2BH2A%2BH2B%2BRECOGNITION%2BBY%2BTHE%2BESSENTIAL%2BCHAPERONE%2BFACT.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12242%26rft_id%253Dinfo%253Apmid%252F23698368%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12242&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23698368&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F741.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000321285600044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.741.11" data-doi="10.1073/pnas.1222198110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hsieh</span>  <span class="cit-name-given-names">F. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kulaeva</span>  <span class="cit-name-given-names">O. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patel</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dyer</span>  <span class="cit-name-given-names">P. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luger</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Histone chaperone FACT action during transcription through chromatin by RNA polymerase II.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">7654</span>–<span class="cit-lpage">7659</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1222198110%26rft_id%253Dinfo%253Apmid%252F23610384%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEwLzE5Lzc2NTQiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzUvNzQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.741.12" data-doi="10.1016/j.molcel.2009.07.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jamai</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Puglisi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strubin</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Histone chaperone spt16 promotes redeposition of the original h3-h4 histones evicted by elongating RNA polymerase.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">377</span>–<span class="cit-lpage">383</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DJamai%26rft.auinit1%253DA.%26rft.volume%253D35%26rft.issue%253D3%26rft.spage%253D377%26rft.epage%253D383%26rft.atitle%253DHistone%2Bchaperone%2Bspt16%2Bpromotes%2Bredeposition%2Bof%2Bthe%2Boriginal%2Bh3-h4%2Bhistones%2Bevicted%2Bby%2Belongating%2BRNA%2Bpolymerase.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2009.07.001%26rft_id%253Dinfo%253Apmid%252F19683500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2009.07.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19683500&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F741.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269278900014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.741.13" data-doi="10.1128/EC.00348-08"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lloyd</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pratt</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siebrasse</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moran</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duina</span>  <span class="cit-name-given-names">A. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Uncoupling of the patterns of chromatin association of different transcription elongation factors by a <a href="http://www.yeastgenome.org/locus/S000004976/overview">histone H3</a> mutant in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">257</span>–<span class="cit-lpage">260</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryot.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.00348-08%26rft_id%253Dinfo%253Apmid%252F19047363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo3OiI4LzIvMjU3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS81Lzc0MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.741.14" data-doi="10.1016/S0959-440X(98)80007-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Luger</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richmond</span>  <span class="cit-name-given-names">T. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">DNA binding within the nucleosome core.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Struct. Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">33</span>–<span class="cit-lpage">40</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bstructural%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BStruct%2BBiol%26rft.aulast%253DLuger%26rft.auinit1%253DK.%26rft.volume%253D8%26rft.issue%253D1%26rft.spage%253D33%26rft.epage%253D40%26rft.atitle%253DDNA%2Bbinding%2Bwithin%2Bthe%2Bnucleosome%2Bcore.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0959-440X%252898%252980007-9%26rft_id%253Dinfo%253Apmid%252F9519294%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0959-440X(98)80007-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9519294&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F741.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000072384400005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.741.15" data-doi="10.1038/38444"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Luger</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mader</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richmond</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sargent</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richmond</span>  <span class="cit-name-given-names">T. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Crystal structure of the nucleosome core particle at 2.8 A resolution.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">389</span>: <span class="cit-fpage">251</span>–<span class="cit-lpage">260</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLuger%26rft.auinit1%253DK.%26rft.volume%253D389%26rft.issue%253D6648%26rft.spage%253D251%26rft.epage%253D260%26rft.atitle%253DCrystal%2Bstructure%2Bof%2Bthe%2Bnucleosome%2Bcore%2Bparticle%2Bat%2B2.8%2BA%2Bresolution.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F38444%26rft_id%253Dinfo%253Apmid%252F9305837%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/38444&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9305837&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F741.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XW77200038&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.741.16" data-doi="10.1534/genetics.111.128769"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCullough</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rawlins</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olsen</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xin</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stillman</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Insight into the mechanism of nucleosome reorganization from histone mutants that suppress defects in the FACT histone chaperone.</span> <abbr class="cit-jnl-abbrev">Genetics.</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">835</span>–<span class="cit-lpage">846</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics.%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.128769%26rft_id%253Dinfo%253Apmid%252F21625001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg4LzQvODM1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS81Lzc0MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.741.17" data-doi="10.1371/journal.pone.0020847"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">C. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berner</span>  <span class="cit-name-given-names">G. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holthoff</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martinez-Fonts</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harper</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Mutant versions of the <em>S. cerevisiae</em> transcription elongation factor Spt16 define regions of Spt16 that functionally interact with <a href="http://www.yeastgenome.org/locus/S000004976/overview">histone H3</a>.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e20847</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DMyers%26rft.auinit1%253DC.%2BN.%26rft.volume%253D6%26rft.issue%253D6%26rft.spage%253De20847%26rft.epage%253De20847%26rft.atitle%253DMutant%2Bversions%2Bof%2Bthe%2BS.%2Bcerevisiae%2Btranscription%2Belongation%2Bfactor%2BSpt16%2Bdefine%2Bregions%2Bof%2BSpt16%2Bthat%2Bfunctionally%2Binteract%2Bwith%2Bhistone%2BH3.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0020847%26rft_id%253Dinfo%253Apmid%252F21673966%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0020847&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21673966&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F741.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=21673966&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-book" id="cit-5.5.741.18" data-doi="10.1534/g3.113.005926"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">H. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wharton</span>  <span class="cit-name-given-names">W.</span>  <span class="cit-name-suffix">2nd</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harper</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dornhoffer</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duina</span>  <span class="cit-name-given-names">A. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A nucleosomal region important for ensuring proper interactions between the transcription elongation factor Spt16 and transcribed genes in <em>Saccharomyces cerevisiae</em>.</span> <span class="cit-source">G3 (Bethesda)</span> <span class="cit-vol">3</span>: <span class="cit-fpage">929</span>–<span class="cit-lpage">940</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.113.005926%26rft_id%253Dinfo%253Apmid%252F23576521%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjMvNi85MjkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzUvNzQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.741.19" data-doi="10.1534/genetics.111.132266"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rando</span>  <span class="cit-name-given-names">O. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winston</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Chromatin and transcription in yeast.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">351</span>–<span class="cit-lpage">387</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DRando%26rft.auinit1%253DO.%2BJ.%26rft.volume%253D190%26rft.issue%253D2%26rft.spage%253D351%26rft.epage%253D387%26rft.atitle%253DChromatin%2Band%2BTranscription%2Bin%2BYeast%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.132266%26rft_id%253Dinfo%253Apmid%252F22345607%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzIvMzUxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS81Lzc0MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.741.20" data-doi="10.1074/jbc.R600007200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reinberg</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sims</span>  <span class="cit-name-given-names">R. J.</span>  <span class="cit-name-suffix">3rd</span>.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">de FACTo nucleosome dynamics.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">281</span>: <span class="cit-fpage">23297</span>–<span class="cit-lpage">23301</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.R600007200%26rft_id%253Dinfo%253Apmid%252F16766522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODEvMzMvMjMyOTciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzUvNzQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-book" id="cit-5.5.741.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rose</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winston</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hieter</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-source">Methods in Yeast Genetics: A Laboratory Course Manual</span>. <span class="cit-publ-name">Cold Spring Harbor Laboratory Press</span>, <span class="cit-publ-loc">Cold Spring Harbor, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.741.22" data-doi="10.1074/jbc.M307291200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ruone</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rhoades</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Formosa</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Multiple Nhp6 molecules are required to recruit Spt16-Pob3 to form yFACT complexes and to reorganize nucleosomes.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">278</span>: <span class="cit-fpage">45288</span>–<span class="cit-lpage">45295</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M307291200%26rft_id%253Dinfo%253Apmid%252F12952948%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzgvNDYvNDUyODgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzUvNzQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.741.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sikorski</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hieter</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">A system of shuttle vectors and yeast host strains designed for efficient manipulation of DNA in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">122</span>: <span class="cit-fpage">19</span>–<span class="cit-lpage">27</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSikorski%26rft.auinit1%253DR.%2BS.%26rft.volume%253D122%26rft.issue%253D1%26rft.spage%253D19%26rft.epage%253D27%26rft.atitle%253DA%2Bsystem%2Bof%2Bshuttle%2Bvectors%2Band%2Byeast%2Bhost%2Bstrains%2Bdesigned%2Bfor%2Befficient%2Bmanipulation%2Bof%2BDNA%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Apmid%252F2659436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTIyLzEvMTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzUvNzQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.741.24" data-doi="10.1093/emboj/20.18.5207"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suto</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luger</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Structure of the yeast nucleosome core particle reveals fundamental changes in internucleosome interactions.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">5207</span>–<span class="cit-lpage">5218</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DWhite%26rft.auinit1%253DC.%2BL.%26rft.volume%253D20%26rft.issue%253D18%26rft.spage%253D5207%26rft.epage%253D5218%26rft.atitle%253DStructure%2Bof%2Bthe%2Byeast%2Bnucleosome%2Bcore%2Bparticle%2Breveals%2Bfundamental%2Bchanges%2Bin%2Binternucleosome%2Binteractions%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252F20.18.5207%26rft_id%253Dinfo%253Apmid%252F11566884%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMjAvMTgvNTIwNyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvNS83NDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.741.25" data-doi="10.1074/jbc.R110.180778"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Winkler</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luger</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The histone chaperone FACT: structural insights and mechanisms for nucleosome reorganization.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">286</span>: <span class="cit-fpage">18369</span>–<span class="cit-lpage">18374</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.R110.180778%26rft_id%253Dinfo%253Apmid%252F21454601%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODYvMjEvMTgzNjkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzUvNzQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.741.26" data-doi="10.1074/jbc.M111.301465"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Winkler</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muthurajan</span>  <span class="cit-name-given-names">U. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hieb</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luger</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Histone chaperone FACT coordinates nucleosome interaction through multiple synergistic binding events.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">286</span>: <span class="cit-fpage">41883</span>–<span class="cit-lpage">41892</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M111.301465%26rft_id%253Dinfo%253Apmid%252F21969370%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODYvNDgvNDE4ODMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzUvNzQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-book" id="cit-5.5.741.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Winston</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Analysis of SPT genes: a genetic approach toward analysis of TFIID, histones, and other transcription factors in yeast</span>, pp. <span class="cit-fpage">1271</span>–<span class="cit-lpage">1293</span> in <span class="cit-source">Transcription Regulation</span>. <span class="cit-publ-name">Cold Spring Harbor Laboratory Press</span>, <span class="cit-publ-loc">Cold Spring Harbor, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.741.28" data-doi="10.1002/yea.320110107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Winston</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dollard</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ricupero-Hovasse</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Construction of a set of convenient <em>Saccharomyces cerevisiae</em> strains that are isogenic to S288C.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">53</span>–<span class="cit-lpage">55</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DWinston%26rft.auinit1%253DF.%26rft.volume%253D11%26rft.issue%253D1%26rft.spage%253D53%26rft.epage%253D55%26rft.atitle%253DConstruction%2Bof%2Ba%2Bset%2Bof%2Bconvenient%2BSaccharomyces%2Bcerevisiae%2Bstrains%2Bthat%2Bare%2Bisogenic%2Bto%2BS288C.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.320110107%26rft_id%253Dinfo%253Apmid%252F7762301%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.320110107&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7762301&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F741.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995QF61400006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.741.29" data-doi="10.1016/j.molcel.2009.06.024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xin</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takahata</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blanksma</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCullough</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stillman</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">yFACT induces global accessibility of nucleosomal DNA without H2A–H2B displacement.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">365</span>–<span class="cit-lpage">376</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DXin%26rft.auinit1%253DH.%26rft.volume%253D35%26rft.issue%253D3%26rft.spage%253D365%26rft.epage%253D376%26rft.atitle%253DyFACT%2Binduces%2Bglobal%2Baccessibility%2Bof%2Bnucleosomal%2BDNA%2Bwithout%2BH2A-H2B%2Bdisplacement.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2009.06.024%26rft_id%253Dinfo%253Apmid%252F19683499%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2009.06.024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19683499&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F5%2F741.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269278900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.5.741.30" data-doi="10.1128/MCB.00478-13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crickard</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Srikanth</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reese</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A highly conserved region within H2B is important for FACT to act on nucleosomes.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">303</span>–<span class="cit-lpage">314</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.00478-13%26rft_id%253Dinfo%253Apmid%252F24248595%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjg6IjM0LzMvMzAzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS81Lzc0MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/5/741.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/5/719" title="Drosophila Muller F Elements Maintain a Distinct Set of Genomic Properties Over 40 Million Years of Evolution" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/5/751" title="A Survey of Imprinted Gene Expression in Mouse Trophoblast Stem Cells" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/5"><span>Volume 5 Issue 5, May 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/5.cover-source.jpg?itok=53KRlKz0" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/5/741" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-NPHmMWuiFIVTt21wuR0Sw-scGfLYr1_Yxee-OXB2nkI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/5/741" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/5/741" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/5/741" class="active" data-icon-position="" data-hide-link-title="0">A Systematic Mutational Analysis of a Histone H3 Residue in Budding Yeast Provides Insights into Chromatin Dynamics</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459889" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-5OiH7hfVE5VqlxIIlXT9AzrhSByVIrIZ3QKOAdeRRxI" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/5/741.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/5/741" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-uM25q8YMUMgavb7uld9B2jyzWmEjXFwY12t-eo_L-bI" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/5/741" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/5/741.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459889" id="node459889--2" data-pisa="ggg;5/5/741" data-pisa-master="ggg;g3.115.017376" data-apath="/ggg/5/5/741.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Systematic Mutational Analysis of a Histone H3 Residue in Budding Yeast Provides Insights into Chromatin Dynamics</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Paige Johnson</span>, <span class="highwire-citation-author" data-delta="1">Virginia Mitchell</span>, <span class="highwire-citation-author" data-delta="2">Kelsi McClure</span>, <span class="highwire-citation-author" data-delta="3">Martha Kellems</span>, <span class="highwire-citation-author" data-delta="4">Sarah Marshall</span>, <span class="highwire-citation-author" data-delta="5">Mary K. Allison</span>, <span class="highwire-citation-author" data-delta="6">Harrison Lindley</span>, <span class="highwire-citation-author" data-delta="7">Hoai-Trang T. Nguyen</span>, <span class="highwire-citation-author" data-delta="8">Jessalyn E. Tackett</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9"><a href="http://orcid.org/0000-0002-6040-6825" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Andrea A. Duina</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">741-749;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.017376 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459889--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Paige Johnson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biology Department, Hendrix College, Conway, Arkansas 72032</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paige%2BJohnson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Johnson%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaige%2BJohnson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Virginia Mitchell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biology Department, Hendrix College, Conway, Arkansas 72032</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Virginia%2BMitchell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mitchell%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVirginia%2BMitchell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Kelsi McClure </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biology Department, Hendrix College, Conway, Arkansas 72032</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kelsi%2BMcClure%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McClure%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKelsi%2BMcClure%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Martha Kellems </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biology Department, Hendrix College, Conway, Arkansas 72032</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martha%2BKellems%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kellems%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMartha%2BKellems%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Sarah Marshall </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biology Department, Hendrix College, Conway, Arkansas 72032</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sarah%2BMarshall%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Marshall%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASarah%2BMarshall%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Mary K. Allison </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biology Department, Hendrix College, Conway, Arkansas 72032</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mary%2BK.%2BAllison%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Allison%20MK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMary%2BK.%2BAllison%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Harrison Lindley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biology Department, Hendrix College, Conway, Arkansas 72032</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Harrison%2BLindley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lindley%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHarrison%2BLindley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Hoai-Trang T. Nguyen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biology Department, Hendrix College, Conway, Arkansas 72032</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hoai-Trang%2BT.%2BNguyen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nguyen%20HT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHoai-Trang%2BT.%2BNguyen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Jessalyn E. Tackett </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biology Department, Hendrix College, Conway, Arkansas 72032</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jessalyn%2BE.%2BTackett%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tackett%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJessalyn%2BE.%2BTackett%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Andrea A. Duina </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biology Department, Hendrix College, Conway, Arkansas 72032</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrea%2BA.%2BDuina%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Duina%20AA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrea%2BA.%2BDuina%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-6040-6825" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Andrea A. Duina</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:duina@hendrix.edu" class="" data-icon-position="" data-hide-link-title="0">duina@hendrix.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/5/741" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/5/741&amp;title=A%20Systematic%20Mutational%20Analysis%20of%20a%20Histone%20H3%20Residue%20in%20Budding%20Yeast%20Provides%20Insights%20into%20Chromatin%20Dynamics" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/5/741&amp;title=A%20Systematic%20Mutational%20Analysis%20of%20a%20Histone%20H3%20Residue%20in%20Budding%20Yeast%20Provides%20Insights%20into%20Chromatin%20Dynamics" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/5/741&amp;title=A%20Systematic%20Mutational%20Analysis%20of%20a%20Histone%20H3%20Residue%20in%20Budding%20Yeast%20Provides%20Insights%20into%20Chromatin%20Dynamics" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/5/741&amp;text=A%20Systematic%20Mutational%20Analysis%20of%20a%20Histone%20H3%20Residue%20in%20Budding%20Yeast%20Provides%20Insights%20into%20Chromatin%20Dynamics" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/5/741&amp;title=A%20Systematic%20Mutational%20Analysis%20of%20a%20Histone%20H3%20Residue%20in%20Budding%20Yeast%20Provides%20Insights%20into%20Chromatin%20Dynamics" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/5/741&amp;t=A%20Systematic%20Mutational%20Analysis%20of%20a%20Histone%20H3%20Residue%20in%20Budding%20Yeast%20Provides%20Insights%20into%20Chromatin%20Dynamics" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/5/741&amp;title=A%20Systematic%20Mutational%20Analysis%20of%20a%20Histone%20H3%20Residue%20in%20Budding%20Yeast%20Provides%20Insights%20into%20Chromatin%20Dynamics" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/5/741&amp;title=A%20Systematic%20Mutational%20Analysis%20of%20a%20Histone%20H3%20Residue%20in%20Budding%20Yeast%20Provides%20Insights%20into%20Chromatin%20Dynamics" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459889/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459889" id="node459889--3" data-pisa="ggg;5/5/741" data-pisa-master="ggg;g3.115.017376" data-apath="/ggg/5/5/741.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Systematic Mutational Analysis of a Histone H3 Residue in Budding Yeast Provides Insights into Chromatin Dynamics</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Paige Johnson</span>, <span class="highwire-citation-author" data-delta="1">Virginia Mitchell</span>, <span class="highwire-citation-author" data-delta="2">Kelsi McClure</span>, <span class="highwire-citation-author" data-delta="3">Martha Kellems</span>, <span class="highwire-citation-author" data-delta="4">Sarah Marshall</span>, <span class="highwire-citation-author" data-delta="5">Mary K. Allison</span>, <span class="highwire-citation-author" data-delta="6">Harrison Lindley</span>, <span class="highwire-citation-author" data-delta="7">Hoai-Trang T. Nguyen</span>, <span class="highwire-citation-author" data-delta="8">Jessalyn E. Tackett</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9"><a href="http://orcid.org/0000-0002-6040-6825" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Andrea A. Duina</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">741-749;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.017376 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459889--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Paige Johnson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biology Department, Hendrix College, Conway, Arkansas 72032</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paige%2BJohnson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Johnson%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaige%2BJohnson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Virginia Mitchell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biology Department, Hendrix College, Conway, Arkansas 72032</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Virginia%2BMitchell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mitchell%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVirginia%2BMitchell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Kelsi McClure </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biology Department, Hendrix College, Conway, Arkansas 72032</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kelsi%2BMcClure%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McClure%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKelsi%2BMcClure%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Martha Kellems </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biology Department, Hendrix College, Conway, Arkansas 72032</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martha%2BKellems%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kellems%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMartha%2BKellems%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Sarah Marshall </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biology Department, Hendrix College, Conway, Arkansas 72032</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sarah%2BMarshall%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Marshall%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASarah%2BMarshall%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Mary K. Allison </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biology Department, Hendrix College, Conway, Arkansas 72032</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mary%2BK.%2BAllison%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Allison%20MK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMary%2BK.%2BAllison%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Harrison Lindley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biology Department, Hendrix College, Conway, Arkansas 72032</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Harrison%2BLindley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lindley%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHarrison%2BLindley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Hoai-Trang T. Nguyen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biology Department, Hendrix College, Conway, Arkansas 72032</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hoai-Trang%2BT.%2BNguyen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nguyen%20HT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHoai-Trang%2BT.%2BNguyen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Jessalyn E. Tackett </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biology Department, Hendrix College, Conway, Arkansas 72032</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jessalyn%2BE.%2BTackett%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tackett%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJessalyn%2BE.%2BTackett%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Andrea A. Duina </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biology Department, Hendrix College, Conway, Arkansas 72032</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrea%2BA.%2BDuina%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Duina%20AA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrea%2BA.%2BDuina%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-6040-6825" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Andrea A. Duina</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:duina@hendrix.edu" class="" data-icon-position="" data-hide-link-title="0">duina@hendrix.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=741&amp;rft.epage=749&amp;rft.atitle=A%20Systematic%20Mutational%20Analysis%20of%20a%20Histone%20H3%20Residue%20in%20Budding%20Yeast%20Provides%20Insights%20into%20Chromatin%20Dynamics&amp;rft.volume=5&amp;rft.issue=5&amp;rft.date=2015-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Johnson%2C+Paige&amp;rft.au=Mitchell%2C+Virginia&amp;rft.au=McClure%2C+Kelsi&amp;rft.au=Kellems%2C+Martha&amp;rft.au=Marshall%2C+Sarah&amp;rft.au=Allison%2C+Mary+K.&amp;rft.au=Lindley%2C+Harrison&amp;rft.au=Nguyen%2C+Hoai-Trang+T.&amp;rft.au=Tackett%2C+Jessalyn+E.&amp;rft.au=Duina%2C+Andrea+A."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459889/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459889/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459889/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459889/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459889/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459889/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459889/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459889/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459889/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459889/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459889/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459889/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/5/741" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-6" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknoweldgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/5/741.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/5/741.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/5/741.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/5/741.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
